<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/5/10/2007" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/5/10/2007.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/5/10/2007.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/5/10/2007.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/5/10/2007.atom" />
<meta name="HW.pisa" content="ggg;5/10/2007" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Second-Generation Linkage Maps for the Pacific Oyster Crassostrea gigas Reveal Errors in Assembly of Genome Scaffolds" />
<meta name="DC.Identifier" content="10.1534/g3.115.019570" />
<meta name="DC.Date" content="2015-10-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2015 Hedgecock et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="The Pacific oyster Crassostrea gigas , a widely cultivated marine bivalve mollusc, is becoming a genetically and genomically enabled model for highly fecund marine metazoans with complex life-histories. A genome sequence is available for the Pacific oyster, as are first-generation, low-density, linkage and gene-centromere maps mostly constructed from microsatellite DNA markers. Here, higher density, second-generation, linkage maps are constructed from more than 1100 coding (exonic) single-nucleotide polymorphisms (SNPs), as well as 66 previously mapped microsatellite DNA markers, all typed in five families of Pacific oysters (nearly 172,000 genotypes). The map comprises 10 linkage groups, as expected, has an average total length of 588 cM, an average marker-spacing of 1.0 cM, and covers 86% of a genome estimated to be 616 cM. All but seven of the mapped SNPs map to 618 genome scaffolds; 260 scaffolds contain two or more mapped SNPs, but for 100 of these scaffolds (38.5%), the contained SNPs map to different linkage groups, suggesting widespread errors in scaffold assemblies. The 100 misassembled scaffolds are significantly longer than those that map to a single linkage group. On the genetic maps, marker orders and intermarker distances vary across families and mapping methods, owing to an abundance of markers segregating from only one parent, to widespread distortions of segregation ratios caused by early mortality, as previously observed for oysters, and to genotyping errors. Maps made from framework markers provide stronger support for marker orders and reasonable map lengths and are used to produce a consensus high-density linkage map containing 656 markers." />
<meta name="DC.Contributor" content="Dennis Hedgecock" />
<meta name="DC.Contributor" content="Grace Shin" />
<meta name="DC.Contributor" content="Andrew Y. Gracey" />
<meta name="DC.Contributor" content="David Van Den Berg" />
<meta name="DC.Contributor" content="Manoj P. Samanta" />
<meta name="article:published_time" content="2015-10-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Second-Generation Linkage Maps for the Pacific Oyster Crassostrea gigas Reveal Errors in Assembly of Genome Scaffolds" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The Pacific oyster &lt;i&gt;Crassostrea gigas&lt;/i&gt;, a widely cultivated marine bivalve mollusc, is becoming a genetically and genomically enabled model for highly fecund marine metazoans with complex life-histories. A genome sequence is available for the Pacific oyster, as are first-generation, low-density, linkage and gene-centromere maps mostly constructed from microsatellite DNA markers. Here, higher density, second-generation, linkage maps are constructed from more than 1100 coding (exonic) single-nucleotide polymorphisms (SNPs), as well as 66 previously mapped microsatellite DNA markers, all typed in five families of Pacific oysters (nearly 172,000 genotypes). The map comprises 10 linkage groups, as expected, has an average total length of 588 cM, an average marker-spacing of 1.0 cM, and covers 86% of a genome estimated to be 616 cM. All but seven of the mapped SNPs map to 618 genome scaffolds; 260 scaffolds contain two or more mapped SNPs, but for 100 of these scaffolds (38.5%), the contained SNPs map to different linkage groups, suggesting widespread errors in scaffold assemblies. The 100 misassembled scaffolds are significantly longer than those that map to a single linkage group. On the genetic maps, marker orders and intermarker distances vary across families and mapping methods, owing to an abundance of markers segregating from only one parent, to widespread distortions of segregation ratios caused by early mortality, as previously observed for oysters, and to genotyping errors. Maps made from framework markers provide stronger support for marker orders and reasonable map lengths and are used to produce a consensus high-density linkage map containing 656 markers.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2015/10/01" />
<meta name="citation_mjid" content="ggg;5/10/2007" />
<meta name="citation_id" content="5/10/2007" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/5/10/2007" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/5/10/2007.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/5/10/2007.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/5/10/2007.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.115.019570" />
<meta name="citation_pmid" content="26248981" />
<meta name="citation_volume" content="5" />
<meta name="citation_issue" content="10" />
<meta name="citation_num_pages" content="13" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2007" />
<meta name="citation_lastpage" content="2019" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Dennis Hedgecock" />
<meta name="citation_author_institution" content="Department of Biological Sciences, University of Southern California, Los Angeles, California 90089-0371" />
<meta name="citation_author_email" content="dhedge@usc.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-3995-646X" />
<meta name="citation_author" content="Grace Shin" />
<meta name="citation_author_institution" content="Department of Biological Sciences, University of Southern California, Los Angeles, California 90089-0371" />
<meta name="citation_author" content="Andrew Y. Gracey" />
<meta name="citation_author_institution" content="Department of Biological Sciences, University of Southern California, Los Angeles, California 90089-0371" />
<meta name="citation_author" content="David Van Den Berg" />
<meta name="citation_author_institution" content="Preventive Medicine, Keck School of Medicine, University of Southern California, Los Angeles, California 90089-9601" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-5404-5218" />
<meta name="citation_author" content="Manoj P. Samanta" />
<meta name="citation_author_institution" content="Systemix Institute, Redmond, Washington 98053" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=A. W. Allard;citation_author=H. L. Alder;citation_title=The effect of incomplete penetrance on the estimation of recombination values.;citation_pages=263-282;citation_volume=15;citation_year=1960;citation_doi=10.1038/hdy.1960.82" />
<meta name="citation_reference" content="citation_journal_title=Integr. Comp. Biol.;citation_author=S. L. Applebaum;citation_author=T.-C. F. Pan;citation_author=D. Hedgecock;citation_author=D. T. Manahan;citation_title=Separating the nature and nurture of energy allocation in response to global change.;citation_volume=54;citation_year=2014;citation_pmid=24907199;citation_doi=10.1093/icb/icu062" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=N. T. J. Bailey;citation_title=The estimation of linkage with differential viability, II and III.;citation_pages=220-228;citation_volume=3;citation_year=1949;citation_doi=10.1038/hdy.1949.14" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=A. D. Ball;citation_author=J. Stapley;citation_author=D. A. Dawson;citation_author=T. R. Birkhead;citation_author=T. Burke;citation_title=A comparison of SNPs and microsatellites as linkage mapping markers: lessons from the zebra finch (Taeniopygia guttata).;citation_pages=218-218;citation_volume=11;citation_year=2010;citation_pmid=20359323;citation_doi=10.1186/1471-2164-11-218" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=N. Bierne;citation_author=S. Launey;citation_author=Y. Naciri-Graven;citation_author=F. Bonhomme;citation_title=Early Effect of Inbreeding as Revealed by Microsatellite Analyses on Ostrea edulis Larvae;citation_pages=1893-1906;citation_volume=148;citation_year=1998;citation_issue=4;citation_pmid=9560403" />
<meta name="citation_reference" content="citation_title=Statistical Analysis of DNA Sequence Data;citation_title=The number of polymorphic DNA clones required to map the human genome;citation_author=D. T. Bishop;citation_author=C. Cannings;citation_author=M. Skolnick;citation_author=J. A. Williamson;citation_year=1983" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=A. Chakravarti;citation_author=L. K. Lasher;citation_author=J. E. Reefer;citation_title=A maximum likelihood method for estimating genome length using genetic linkage data.;citation_pages=175-182;citation_volume=128;citation_year=1991;citation_issue=1;citation_pmid=2060775" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=B. Chevreux;citation_author=T. Pfisterer;citation_author=B. Drescher;citation_author=A. J. Driesel;citation_author=W. E. G. Muller;citation_title=Using the miraEST assembler for reliable and automated mRNA transcript assembly and SNP detection in sequenced ESTs.;citation_volume=14;citation_year=2004;citation_pmid=15140833;citation_doi=10.1101/gr.1917404" />
<meta name="citation_reference" content="citation_title=Aquaculture Genome Technologies;citation_title=Bivalve genomics: complications, challenges, and future perspectives;citation_author=J. P. Curole;citation_author=D. Hedgecock;citation_pages=525-543;citation_year=2007" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=R. A. Dalloul;citation_author=J. A. Long;citation_author=A. V. Zimin;citation_author=L. Aslam;citation_author=K. Beal;citation_title=Multi-platform next-generation sequencing of the domestic turkey (Meleagris gallopavo): genome assembly and analysis.;citation_pages=e1000475;citation_volume=8;citation_year=2010;citation_issue=9;citation_pmid=20838655;citation_doi=10.1371/journal.pbio.1000475" />
<meta name="citation_reference" content="citation_journal_title=Aquaculture;citation_author=L. Dégremont;citation_author=E. Bédier;citation_author=P. Boudry;citation_title=mortality of hatchery-produced Pacific oyster spat (Crassostrea gigas). II. Response to selection for survival and its influence on growth and yield.;citation_pages=21-29;citation_volume=299;citation_year=2010;citation_doi=10.1016/j.aquaculture.2009.11.017" />
<meta name="citation_reference" content="citation_journal_title=Cytogenet. Genome Res.;citation_author=J. B. Dodgson;citation_author=M. E. Delany;citation_author=H. H. Cheng;citation_title=Poultry genome sequences: Progress and outstanding challenges.;citation_pages=19-26;citation_volume=134;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=R. W. Doerge;citation_title=Mapping and analysis of quantitative trait loci in experimental populations.;citation_pages=43-52;citation_volume=3;citation_year=2002;citation_issue=1;citation_pmid=11823790;citation_doi=10.1038/nrg703" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=R. J. Elshire;citation_author=J. C. Glaubitz;citation_author=Q. Sun;citation_author=J. A. Poland;citation_author=K. Kawamoto;citation_title=A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.;citation_pages=e19379-e19379;citation_volume=6;citation_year=2011;citation_issue=5;citation_pmid=21573248;citation_doi=10.1371/journal.pone.0019379" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=L. Fishman;citation_author=A. J. Kelly;citation_author=E. Morgan;citation_author=J. H. Willis;citation_title=A Genetic Map in the Mimulus guttatus Species Complex Reveals Transmission Ratio Distortion due to Heterospecific Interactions;citation_pages=1701-1716;citation_volume=159;citation_year=2001;citation_issue=4;citation_pmid=11779808" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=B. R. Foley;citation_author=C. G. Rose;citation_author=D. E. Rundle;citation_author=W. Leong;citation_author=S. Edmands;citation_title=Postzygotic isolation involves strong mitochondrial and sex-specific effects in Tigriopus californicus, a species lacking heteromorphic sex chromosomes.;citation_pages=391-401;citation_volume=111;citation_year=2013;citation_pmid=23860232;citation_doi=10.1038/hdy.2013.61" />
<meta name="citation_reference" content="citation_journal_title=Marine biotechnology (New York, N.Y.);citation_journal_abbrev=Mar Biotechnol (NY);citation_author=X. Guo;citation_author=Q. Li;citation_author=Q. Z. Wang;citation_author=L. F. Kong;citation_title=Genetic mapping and QTL analysis of growth-related traits in the Pacific oyster.;citation_pages=218-226;citation_volume=14;citation_year=2012;citation_issue=2;citation_pmid=21932055;citation_doi=10.1007/s10126-011-9405-4" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=C. A. Hackett;citation_author=L. B. Broadfoot;citation_title=Effects of genotyping errors, missing values and segregation distortion in molecular marker data on the construction of linkage maps.;citation_pages=33-38;citation_volume=90;citation_year=2003;citation_issue=1;citation_pmid=12522423;citation_doi=10.1038/sj.hdy.6800173" />
<meta name="citation_reference" content="citation_journal_title=Aquaculture;citation_author=D. Hedgecock;citation_author=J. P. Davis;citation_title=Heterosis for yield and crossbreeding of the Pacific oyster Crassostrea gigas.;citation_pages=S17-S29;citation_volume=272;citation_year=2007;citation_doi=10.1016/j.aquaculture.2007.07.226" />
<meta name="citation_reference" content="citation_journal_title=J. Shellfish Res.;citation_author=D. Hedgecock;citation_author=P. M. Gaffney;citation_author=P. Goulletquer;citation_author=X. Guo;citation_author=K. Reece;citation_title=The case for sequencing the Pacific oyster genome.;citation_pages=429-441;citation_volume=24;citation_year=2005;citation_doi=10.2983/0730-8000(2005)24[429:TCFSTP]2.0.CO;2" />
<meta name="citation_reference" content="citation_journal_title=Aquaculture;citation_author=W. K. Hershberger;citation_author=J. A. Perdue;citation_author=J. H. Beattie;citation_title=Genetic selection and systematic breeding in Pacific oyster culture.;citation_pages=237-245;citation_volume=39;citation_year=1984;citation_doi=10.1016/0044-8486(84)90269-2" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=S. Hubert;citation_author=D. Hedgecock;citation_title=Linkage Maps of Microsatellite DNA Markers for the Pacific Oyster Crassostrea gigas;citation_pages=351-362;citation_volume=168;citation_year=2004;citation_issue=1;citation_pmid=15454548;citation_doi=10.1534/genetics.104.027342" />
<meta name="citation_reference" content="citation_journal_title=Aquaculture;citation_author=S. Hubert;citation_author=E. Cognard;citation_author=D. Hedgecock;citation_title=Centromere-mapping in triploid families of the Pacific oyster Crassostrea gigas (Thunberg).;citation_pages=172-183;citation_volume=288;citation_year=2009;citation_doi=10.1016/j.aquaculture.2008.12.006" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=J. Jansen;citation_author=A. G. De Jong;citation_author=J. W. Van Ooijen;citation_title=Constructing dense genetic linkage maps.;citation_pages=1113-1122;citation_volume=102;citation_year=2001;citation_doi=10.1007/s001220000489" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=E. S. Lander;citation_author=D. Botstein;citation_title=Mapping mendelian factors underlying quantitative traits using RFLP linkage maps.;citation_pages=185-199;citation_volume=121;citation_year=1989;citation_issue=1;citation_pmid=2563713" />
<meta name="citation_reference" content="citation_journal_title=Aquaculture;citation_author=C. Langdon;citation_author=F. Evans;citation_author=D. Jacobson;citation_author=M. Blouin;citation_title=Yields of cultured Pacific oysters Crassostrea gigas Thunberg improved after one generation of selection.;citation_pages=227-244;citation_volume=220;citation_year=2003;citation_doi=10.1016/S0044-8486(02)00621-X" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. Launey;citation_author=D. Hedgecock;citation_title=High genetic load in the Pacific oyster Crassostrea gigas.;citation_pages=155-165;citation_volume=159;citation_year=2001" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=R. Durbin;citation_title=Fast and accurate short read alignment with Burrows-Wheeler Transform.;citation_volume=25;citation_year=2009;citation_pmid=19451168;citation_doi=10.1093/bioinformatics/btp324" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=B. Handsaker;citation_author=A. Wysoker;citation_author=T. Fennell;citation_author=J. Ruan;citation_title=The Sequence alignment/map (SAM) format and SAMtools.;citation_volume=25;citation_year=2009;citation_pmid=19505943;citation_doi=10.1093/bioinformatics/btp352" />
<meta name="citation_reference" content="citation_journal_title=Marine biotechnology (New York, N.Y.);citation_journal_abbrev=Mar Biotechnol (NY);citation_author=L. Li;citation_author=X. M. Guo;citation_title=AFLP-based genetic linkage maps of the pacific oyster Crassostrea gigas Thunberg.;citation_pages=26-36;citation_volume=6;citation_year=2004;citation_issue=1;citation_pmid=14564534;citation_doi=10.1007/s10126-003-0001-0" />
<meta name="citation_reference" content="citation_title=Statistical Genomics: Linkage, Mapping, and QTL Analysis;citation_author=B. H. Liu;citation_year=1998" />
<meta name="citation_reference" content="citation_journal_title=Canadian journal of genetics and cytology.  Journal canadien de gÃ©nÃ©tique et de cytologie;citation_journal_abbrev=Can J Genet Cytol;citation_author=A. C. Longwell;citation_author=S. S. Stiles;citation_author=D. G. Smith;citation_title=Chromosome complement of the American oyster Crassostrea virginica, as seen in meiotic and cleaving eggs.;citation_pages=845-856;citation_volume=9;citation_year=1967;citation_issue=4;citation_pmid=5588794;citation_doi=10.1139/g67-090" />
<meta name="citation_reference" content="citation_journal_title=Mol. Breed.;citation_author=M. Lorieux;citation_title=MapDisto: fast and efficient computation of genetic linkage maps.;citation_pages=1231-1235;citation_volume=30;citation_year=2012;citation_doi=10.1007/s11032-012-9706-y" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=M. Lorieux;citation_author=X. Perrier;citation_author=B. Goffinet;citation_author=C. Lanaud;citation_author=D. G. de León;citation_title=Maximum-likelihood models for mapping genetic markers showing segregation distortion. 2. F2 populations.;citation_pages=81-89;citation_volume=90;citation_year=1995;citation_pmid=24173787;citation_doi=10.1007/BF00220999" />
<meta name="citation_reference" content="citation_title=Exotic Species in Mariculture;citation_author=R. Mann;citation_year=1979" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=J. Paps;citation_author=F. Xu;citation_author=G. Zhang;citation_author=P. W. H. Holland;citation_title=Reinforcing the egg-timer: recruitment of novel Lophotrochozoa homeobox genes to early and late development in the Pacific oyster.;citation_volume=7;citation_year=2015;citation_pmid=25631164;citation_doi=10.1093/gbe/evv018" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=L. V. Plough;citation_title=Environmental stress increases selection against and dominance of deleterious mutations in inbred families of the Pacific oyster Crassostrea gigas.;citation_pages=3974-3987;citation_volume=21;citation_year=2012;citation_issue=16;citation_pmid=22747636;citation_doi=10.1111/j.1365-294X.2012.05688.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=L. V. Plough;citation_author=D. Hedgecock;citation_title=Quantitative trait locus analysis of stage-specific inbreeding depression in the Pacific oyster Crassostrea gigas.;citation_volume=189;citation_year=2011;citation_pmid=21940682;citation_doi=10.1534/genetics.111.131854" />
<meta name="citation_reference" content="citation_journal_title=Animal genetics;citation_journal_abbrev=Anim Genet;citation_author=C. Sauvage;citation_author=P. Boudry;citation_author=D.-J. De Koning;citation_author=C. S. Haley;citation_author=S. Heurtebise;citation_title=QTL for resistance to summer mortality and OsHV-1 load in the Pacific oyster (Crassostrea gigas).;citation_pages=390-399;citation_volume=41;citation_year=2010;citation_issue=4;citation_pmid=20096029" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=R. Shah;citation_author=C. R. Cavanagh;citation_author=B. E. Huang;citation_title=Computationally efficient map construction in the presence of segregation distortion.;citation_pages=2585-2597;citation_volume=127;citation_year=2014;citation_pmid=25260690;citation_doi=10.1007/s00122-014-2401-0" />
<meta name="citation_reference" content="citation_journal_title=Mutation Research;citation_journal_abbrev=Mutation Research;citation_author=R. Shen;citation_author=J. B. Fan;citation_author=D. Campbell;citation_author=W. Chang;citation_author=J. Chen;citation_title=High-throughput SNP genotyping on universal bead arrays.;citation_pages=70-82;citation_volume=573;citation_year=2005;citation_issue=1-2;citation_pmid=15829238;citation_doi=10.1016/j.mrfmmm.2004.07.022" />
<meta name="citation_reference" content="citation_journal_title=Plant J.;citation_author=P. Stam;citation_title=Construction of integrated genetic linkage maps by means of a new computer package: JoinMap.;citation_pages=739-744;citation_volume=3;citation_year=1993;citation_doi=10.1111/j.1365-313X.1993.00739.x" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=S. K. Truong;citation_author=R. F. McCormick;citation_author=D. T. Morishige;citation_author=J. E. Mullet;citation_title=Resolution of genetic map expansion caused by excess heterozygosity in plant recombinant inbred populations.;citation_volume=4;citation_year=2014;citation_pmid=25128435;citation_doi=10.1534/g3.114.012468" />
<meta name="citation_reference" content="citation_title=JoinMap 4, Software for the calculation of genetic linkage maps in experimental populations;citation_author=J. W. Van Ooijen;citation_year=2006" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=C. M. Wang;citation_author=C. S. Zhu;citation_author=H. Q. Zhai;citation_author=J. M. Wan;citation_title=Mapping segregation distortion loci and quantitative trait loci for spikelet sterility in rice ( Oryza sativa L.).;citation_pages=97-106;citation_volume=86;citation_year=2005;citation_issue=2;citation_pmid=16356283;citation_doi=10.1017/S0016672305007779" />
<meta name="citation_reference" content="citation_journal_title=J. Shellfish Res.;citation_author=H. Wang;citation_author=G. Zhang;citation_author=X. Lio;citation_author=X. M. Guo;citation_title=Classification of common oysters from North China.;citation_pages=495-503;citation_volume=27;citation_year=2008;citation_doi=10.2983/0730-8000(2008)27[495:COCOFN]2.0.CO;2" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=Y. H. Wu;citation_author=P. R. Bhat;citation_author=T. J. Close;citation_author=S. Lonardi;citation_title=Efficient and accurate construction of genetic linkage maps from the minimum spanning tree of a graph.;citation_pages=e1000212-e1000212;citation_volume=4;citation_year=2008;citation_issue=10;citation_pmid=18846212;citation_doi=10.1371/journal.pgen.1000212" />
<meta name="citation_reference" content="citation_journal_title=IEEE-ACM Trans. Comp. Biol. Bioinform.;citation_author=Y. H. Wu;citation_author=T. J. Close;citation_author=S. Lonardi;citation_title=Accurate construction of consensus genetic maps via integer linear programming.;citation_pages=381-394;citation_volume=8;citation_year=2011;citation_doi=10.1109/TCBB.2010.35" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=G. Zhang;citation_author=X. Fang;citation_author=X. Guo;citation_author=L. Li;citation_author=R. Luo;citation_title=The oyster genome reveals stress adaptation and complexity of shell formation.;citation_pages=49-54;citation_volume=490;citation_year=2012;citation_issue=7418;citation_pmid=22992520;citation_doi=10.1038/nature11413" />
<meta name="citation_reference" content="citation_journal_title=Aquacult. Int.;citation_author=X. X. Zhong;citation_author=Q. Li;citation_author=X. Guo;citation_author=H. Yu;citation_author=L. Kong;citation_title=QTL mapping for glycogen content and shell pigmentation in the Pacific oyster Crassostrea gigas using microsatellites and SNPs.;citation_pages=1877-1899;citation_volume=22;citation_year=2014;citation_doi=10.1007/s10499-014-9789-z" />
<meta name="citation_reference" content="citation_journal_title=BMC genetics [electronic resource];citation_journal_abbrev=BMC Genet;citation_author=C. Zhu;citation_author=Y.-M. Zhang;citation_title=An EM algorithm for mapping segregation distortion loci.;citation_pages=82-82;citation_volume=8;citation_year=2007;citation_pmid=18047652" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=C. Zhu;citation_author=C. Wang;citation_author=Y.-M. Zhang;citation_title=Modeling segregation distortion for viability selection. I. Reconstruction of linkage maps with distorted markers.;citation_pages=295-305;citation_volume=114;citation_year=2007;citation_issue=2;citation_pmid=17119913;citation_doi=10.1007/s00122-006-0432-x" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Second-Generation Linkage Maps for the Pacific Oyster Crassostrea gigas Reveal Errors in Assembly of Genome Scaffolds" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10.cover-source.jpg" />
<meta name="twitter:description" content="The Pacific oyster Crassostrea gigas , a widely cultivated marine bivalve mollusc, is becoming a genetically and genomically enabled model for highly fecund marine metazoans with complex life-histories. A genome sequence is available for the Pacific oyster, as are first-generation, low-density, linkage and gene-centromere maps mostly constructed from microsatellite DNA markers. Here, higher density, second-generation, linkage maps are constructed from more than 1100 coding (exonic) single-nucleotide polymorphisms (SNPs), as well as 66 previously mapped microsatellite DNA markers, all typed in five families of Pacific oysters (nearly 172,000 genotypes). The map comprises 10 linkage groups, as expected, has an average total length of 588 cM, an average marker-spacing of 1.0 cM, and covers 86% of a genome estimated to be 616 cM. All but seven of the mapped SNPs map to 618 genome scaffolds; 260 scaffolds contain two or more mapped SNPs, but for 100 of these scaffolds (38.5%), the contained SNPs map to different linkage groups, suggesting widespread errors in scaffold assemblies. The 100 misassembled scaffolds are significantly longer than those that map to a single linkage group. On the genetic maps, marker orders and intermarker distances vary across families and mapping methods, owing to an abundance of markers segregating from only one parent, to widespread distortions of segregation ratios caused by early mortality, as previously observed for oysters, and to genotyping errors. Maps made from framework markers provide stronger support for marker orders and reasonable map lengths and are used to produce a consensus high-density linkage map containing 656 markers." />
<meta name="og-title" property="og:title" content="Second-Generation Linkage Maps for the Pacific Oyster Crassostrea gigas Reveal Errors in Assembly of Genome Scaffolds" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/5/10/2007" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="The Pacific oyster Crassostrea gigas , a widely cultivated marine bivalve mollusc, is becoming a genetically and genomically enabled model for highly fecund marine metazoans with complex life-histories. A genome sequence is available for the Pacific oyster, as are first-generation, low-density, linkage and gene-centromere maps mostly constructed from microsatellite DNA markers. Here, higher density, second-generation, linkage maps are constructed from more than 1100 coding (exonic) single-nucleotide polymorphisms (SNPs), as well as 66 previously mapped microsatellite DNA markers, all typed in five families of Pacific oysters (nearly 172,000 genotypes). The map comprises 10 linkage groups, as expected, has an average total length of 588 cM, an average marker-spacing of 1.0 cM, and covers 86% of a genome estimated to be 616 cM. All but seven of the mapped SNPs map to 618 genome scaffolds; 260 scaffolds contain two or more mapped SNPs, but for 100 of these scaffolds (38.5%), the contained SNPs map to different linkage groups, suggesting widespread errors in scaffold assemblies. The 100 misassembled scaffolds are significantly longer than those that map to a single linkage group. On the genetic maps, marker orders and intermarker distances vary across families and mapping methods, owing to an abundance of markers segregating from only one parent, to widespread distortions of segregation ratios caused by early mortality, as previously observed for oysters, and to genotyping errors. Maps made from framework markers provide stronger support for marker orders and reasonable map lengths and are used to produce a consensus high-density linkage map containing 656 markers." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10.cover-source.jpg" />
    <title>Second-Generation Linkage Maps for the Pacific Oyster Crassostrea gigas Reveal Errors in Assembly of Genome Scaffolds | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"2wEWBWXVdOEdpraKxtYvIN_da17jHDPcYL-erZbWq5Y","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460153","apath":"\/ggg\/5\/10\/2007.atom","pisa":"ggg;5\/10\/2007","ac":{"\/ggg\/5\/10\/2007.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/5\/10\/2007.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;5\/10\/2007"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;5\/10\/2007"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/5\/10\/2007":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022ub\\\\\\\/GzX5uI5jhXzWi3OZiSidGaOnM+OS2mX2ngUCkn\\\\\\\/2ESaWoJT5T5PMGdB4karwoCrsdoCq44ihQ\\\\\\\/1bc38mkC\\\\\\\/UcxSnVC0bCvsVO\\\\\\\/ZZX5nebWdu2ukhe8F2xwoLlkglIPaThs7h2eUAd3n80zBTZ7AHjNhvm62M6+7eECn1jagT6cJo1LezWh++h3PvvUxLSsIVk3RNDHOsN5a54Fsmf1z1rIcTXX9Kc3EtijzwG8ywmfrrl7av\\\\\\\/LEQNyQjetm2NTuXVRm1uIS20AKEqd5C0ndc0L8HfRuyBGhrLOw3YYdM0YA34vVTw\\\\\\\/rFx53B8VVMsHY0ZhbWymCk\\\\\\\/J0lEDfgYy2QlR1YCNit03Di0QQGzspox1orFXanbHC9KeGtNN7BlDgUD8ncCAUWQIqwJANRwNnDAUOeb\\\\\\\/3A0upgsf2kDV6tdCzCAz6KhjsXrh8LTfwCQNo2wf1iyC8w0qRPYQpGz+jJCZqFkhv0NNa4mfK+z9BrdrHMZBFaVVwN+KMHQNDabcv57JlHJcaPTs+VjZ9AgomPAPv3hxXE\\\\\\\/N1mJmcPdZ4+cGEnPIirdm4sF6OVyMOTLIpASCxzH\\\\\\\/cvwey5rjP\\\\\\\/lLayrYQylBnfApGx63XecrhWqqCtO0J+4FFatQ21g5Q5eEm71U3LWamD9wDUBMu2z\\\\\\\/4hCJM5ma7TqpJylRIqSJ0tYUbOBlIoNScQKeaWlaWOc30Eov4ruO91nV9w8CjhTHcRPa97Vi1t2q6GLKSL23RJ76dN8lAn6twax3t3\\\\\\\/FLquy68kLtzP\\\\\\\/qfGpUUfvMDyBbpp0N8El06d2LWR4WIlUhwDCmPaWfpGQDm4n5+q0sxAa7QZmUR\\\\\\\/lEmAJMkk1l6bewFlEme0NXghKR99Uc8T4ud7MsPt5Q9mgoLUSiArGFvL6HLK2u3TnVdwh6aPv71cnWyUf70OS1DjcUtp4fhacXpsixX1Q92cq15XWDbC4uq35gkAAPcGKKFDI9P3f05ulxwsOsOX3nARRR0meICTBg6S5gJxNckMbnS4Bjw5\\\\\\\/x4zM+4sZ1S0A7IOMNVrVMsjUUZQa237nUU1\\\\\\\/kvPAXnCGJ6uIAmU\\\\\\\/UUGQq2kr1bisBDN0GO3JNEMY6cR63FamtB9dHObjoDTTjTr5GN03yYBdKyv4Z0aWVGnENucf2\\\\\\\/0f5llFFuZ2cQ48UwtgWCFviutq0lE2m4\\\\\\\/0+QJCDsWZ5FSzHXuA4+93uZ58KKMzw6waQOjGIFtvlcJyo9Tnc0YLscXZmBC\\\\\\\/Y9J\\\\\\\/jlOdlmSSfGKACbEq6vLNp8qSqZ01j2b9T3I\\\u0022,\\\u0022iv\\\u0022:\\\u0022u6uXquMtM\\\\\\\/Trlc+2Yi\\\\\\\/m5Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002220b3ee6d1b4ba1a820f30be1ddc7012554f76f32fa7a1ffb5ccf703146e1dfe4\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022+UAZrdVgfoJw4riz7WNanC\\\\\\\/kCRMaasO3m8NPssN\\\\\\\/rZYCRa0EJjzODHGm0qcrOuRBnmvGJkHsqokRNrYioytjfAc10lCkw\\\\\\\/ZZyQsoWqQ0sL44PBR+KMWI19sIXsNBjMX2Ash5AK+Hb8uyKnqByZTC+HpiyNapBBFYiqPP2wm1sgnkjm2t+nA9PMw8fY53WW8aufap2W\\\\\\\/Pn4f5AWZudGLvI6vcuc3uvlt5uWAneXvKkDz1TRYRo77sU838Qs4yD2XDnwpZ4YrDQgR4VbMH6Am7c1PHphBKyw76cybwKoKujCxhN8Je\\\\\\\/7gKIk+TlG66FyHYCylQyCl+1Ve5R6TYPXN9HAP87LgJG51OJ2EkET7maTCSa3cU6Fqf6TlCiDE+uFpk1J9+f57c\\\\\\\/0dyjmmALBvseO4yLgad0G35QlW\\\\\\\/3NKbSy6\\\\\\\/lfq+uxCEgvV0OOPzWGwiY96PoAVW3FQOcXDLQ8uIZfowq2GWyI7UF5pneKHUR1\\\\\\\/\\\\\\\/1fTN2YMIZg4RFxt4CIsKgYde3FxLYYSteImPVqUicWkdIP8174cDf2KN+J5rF48IKr5F8Zhdq\\\\\\\/2+MeXsteJYw45zHWrTHODaR5RwhiG0rX9q0vvRVQSRQ8kKr1woCn6ydpBxgFak+kiRdRZD2YtDpQlwyOu3Loy5LB8SkMAoHkyGiw3o5oFUxXaFRyoDkm\\\\\\\/3o6WdKmyqCPvV8oh7tYj59zdbY7t+2Ia6jnp3cPA3zLJi4NYEmzC8j5cjQQAy0vLjDJ4KpSlYuyX36fke\\\\\\\/dxmoN7qjOD03rBccx+ORob4fuxlJ\\\\\\\/NSro0iPRpsNuhdJWKCbpDLGBELm8RtT1QLUgh6Yy13\\\\\\\/ot4ZGXRWbGkFUVuVPdDh3Sx5iG1I72UXsSokVjXl6tOnABPDDEm6F1L7NI7uIMxEfhn93RwrV2T9G6yUnqfU5CGGh8ut6SBRDBriqJa7JO\\\\\\\/1eFGL9WCcUzSWaw2pXyC5MCuGk0EIgdXCbkAzlUVltm7O6jzMLyBLLzbTRy8k6le\\\\\\\/6NPqH22Kx+dc2unTDgw9TT7hib6TCyyBL\\\\\\\/yVXHFcgD14Zx7W8PHszkYUVzxapyg+sSE1pLQ3N6PoBmsfhbnjm4X+CHAG94cLiC9+KL7zwyPMPpoVk0HjxYujSkDNhIevdahOMd9JVcQjV5YUABfW28OFILVjV\\\\\\\/u4PAdbUfKLoGnODjpC2l0cQnE3GyJcTSg6blTk7xypBDKogO6uGL\\\\\\\/iANKS\\\\\\\/nyCDByfJv2+Oi\\\\\\\/6gG8xKAnIhYv8JVCQZCcxyJ801DkNe1\\\\\\\/OLnIzK6q4h08oRfXDHyvBzfabgOdD6+z\\\\\\\/B\\\\\\\/ThCqDn5w6aOH5wEREAm+2YTRpKM\\\\\\\/BqvnUsYWr07bCPuhviTxNhXZ4ZX0FoEZj0eenEDIHervGJufuL\\\\\\\/t+tdMpGL1JpN1xxOg\\\\\\\/B1+Zei7Qj9JNQuV9xLHigwaMXhA75\\\\\\\/RXX1kTRKB9TEz65CHyKIGYpyoVraZLlFWd\\\\\\\/z0\\\\\\\/YTgI+FAhTKu1Z9g1VtqIsaxZcIqslv517jZ0X5VYTCXxfGjAJamuLCX\\\\\\\/PBoyfl+LGclAo6aPQMqxtx2i3enUFk1CRAAFs7JIXC+xnRFzf77uUL9DcRDmkMXa93iCxL+7+2cmjtuY03HqlqO\\\\\\\/8CwFwNt8aGSOzbSuKoN3qCMEZ6AbGVt33KKswvPMfTpY9Kdp2HFWRRxBcJkxszaBo6xl8vbTAgipQcrNIsCf6LlHbjcr9OWIRkwvvO3UqkndSH\\\\\\\/Vh+lpKlCjZOy7QxO2ERIf\\\\\\\/9cjwHeMuGBzRxIBaP2ekrWOVONkP\\\\\\\/ZMfz8pgMBnnCHLZo172FwEjjLguObrdTq76hR6gjna0FeUSJpb7cq53C0S1cbWub6ET9MmsZZVZTFTSDiMcXUx1fiGT\\\\\\\/hfyP+DyFTo6N9i0zFUAzrKjD4NwraaS7Gm9MaNW6BkWVMda5oC+sc8dHdyqmplynyNc3eyVYYVFWdzZp624lrAEz1g\\\\\\\/hhUxfGqz65Uh6P6i62qreCPzvGrD8WX872Bra8WPqemxfGbAqeRgaFBsqbwhPWzqo9+iN4kJMutj2E6TeS2H5p\\\\\\\/YDW5kkIf0e6Tx16gAbtkUG4RzDB5iM\\\\\\\/g33b57flQCmNLOqdedJMUUUoxIvFN5zjtU4ysbvoyW0YHLQDLmuB3jbNggd2mbLIs3FkGXALtWhaO47IKS4w0Ld9LXCg5ie+VLD+iSMWKtMg1NZj5Dne+tLinfS4or6xGWNKH3PembCp6JS1k+sa8d3pyKTnX7f52Vi2MNcsVsBiLusUtOAX3bEtAdhSn0ro6OoxioQ0xSGV3oHa2LA7p8ckrV3\\\\\\\/rzjs\\\\\\\/tF5Y3ePMe\\\\\\\/hYoz0TPTlrbmRu\\\\\\\/KSj5tF4sugqMBMKtX8MO3T\\\\\\\/3s7d6wKzoeYYSe9+HW96PdboWrRuXs8ztPPjGf6FqimCNdWItpTcMHX1Ov3\\\\\\\/cL9b1NO0k54aJHhDZYtyZDAdyD4GR2VYaNr\\\\\\\/UsZ+7hAzu8hg\\\\\\\/5a11ssXilMdDVoByt090mYNUto0r1u4iEq1pq2KRvIXekmL6iUcnQiYn2lZF72n5fh1H1ywx6+BHVmcF8uPnvuI3QXy8SVRdaORIS22sx2eiOmOZY\\\\\\\/shOdX\\\\\\\/T\\\\\\\/AZhskssa7vChz2W59bBbckTdEaVr5DcFTS1gE\\\\\\\/SZLcy\\\\\\\/UkMQh3679YdhvFkngr3T3ABWuvlBc9x2pmRMiXuGAwL\\\\\\\/VxUyo5EijMjP3Z6Ruaep\\\\\\\/OX5QOemPHB605avG5rxNUZYSjAmk0zjvPeq\\\\\\\/7rtPoetuoAVpO+nS3enJNc2oDuSmwu0ThCGZHEb6SUuFtHt4CxPq3kHFkZpwnQ5uUx1PWNRRZfI1ayezR4VM6uHA33yNgR3AUb+kgXFuwe\\\\\\\/dKtQ4D7ivYWZ+\\\\\\\/+jzkVslMNJZ2x6UMB5wjemaYJU1uiEbf\\\\\\\/3S\\\\\\\/vWZKgwc6d9zhXwOmnwKmZvJYOpCzPJ2tkadPVS6RPe2Sw+en02NBRM1nEBvdXkuqUxgV\\\\\\\/tk4asnPKqH\\\\\\\/L7D1L2AlhU+GkNvFnQgZpbVXVbOR2b63sfvJMNu7eVVyJWNxqxGBzf63a2W8LHKP+ZFZ\\\\\\\/hGZKf239FVIpQkn0PB9gScJ4c7D\\\\\\\/\\\\\\\/qa9soW\\\\\\\/hdV2VSLc6xaZlquz1hAPfH56hFYyYIaeFDLliQdJMdpA0e3yeFJE+bR74miJf6ARPCKyRnY+5REBadtMoeTP3wgcmQgj53ieG21Dc03nlcg\\\\\\\/Clk9JPJ4SkLiRo7+YHpnKlfctBGFoaqQZ1BmZ7+Jjiv9\\\\\\\/h2HeXmG0CR\\\\\\\/v6J3Q7V7qeqWpvCDA2zte9tb6Lbf1VgOekeapnmbFnnZVkCKLmFlLG3QJr4z0VJApi4Ux1b2y1KGwX8aRGbn1QqsJC7P1AO2uSV2IY7U66jBIKEurfY9jdUWXs6sJhVmnecYGB9smllXIJ1oznRgUG08zUYm0bEAWrlZNeFeZmqoCOEL28+kLaPGayYQc5B2ARrqBnKADL9luu8fYkJRmUDxQkiF7cqIRWB08MWnuEWCNEF\\\\\\\/HjbDsURsbeJc5IjBbfxs7F3Cwl9OZWOob11aRL+HezPhm7PwMs8HfjmN+7p6CC\\\\\\\/AyR1Agg9OFJamQTE7rcBs8IPA+aZCIpDqUh0k9fRk2GKy7Y9\\\\\\\/PLuQ+ZZ23YVmvd81S4S0kjhqkLi+\\\\\\\/VMc1IhD+q8LE15a50hV0o1S\\\\\\\/bOXfvZqudkhYYGsYITIQ9oC5bA6HF1Avf\\\\\\\/ctPgLpkVoqBmeV4D+cNyoO+ooSv\\\\\\\/KBlaAUwc7BBaT5bCBAhfU4WT0W0RwC5Fc2dDH0vcUZ18sf1rXqVYmqjAJOzG59d9harwbVcW1sXsW9yMxBkTo7iutBRCs3Os3uHyNjSF3LWzNzVqYOxDz9ngNLK54mUyxLl5yntXlswHcilSXB6z8nsxJXx8KKFc30Za6Jic\\\\\\\/xbbPBdoftSGXdqo8l3YhJQcZxq+Xm7eNz2L7S6O3D4j9c9ErJiqo0VbnMZ5nxoKGgZCv0LZl39TKxvZLCZ+eCwFBL6aTTCe8I1WFY8EmT59PkQQGhNGF8fqcQRFG7S+BK38uTB5glBJJwo\\\\\\\/NGr3uz1ocD7Ejr30zaDgkEPfPkgTqDBGMtNkBhhxSjHK4v4FnIh\\\\\\\/mWreSf6ydC+Ug3FGSyZPt0V4yjyKuPWmvOP4Zav3nEnZ7rzM2Uhayjkw+8WR74WPSwKvO9uV7D1V4Fi144YBFOtY29Dhnb8Cb8fXPm1uUmUuBA9vUpyHP+GQwbteb9Pn5Ue1fZb63GupyZs\\\\\\\/xlunpKZu17syRjw9FYXBIGlVfYj2rHdOWD8LVeuavlvpxVZErP9bp1ys7iLdw\\\\\\\/T50++zqSN1Rjwsc2rkx2DLYLatV\\\\\\\/oUqOR1YWxcIlDdqwxPYug8e+xlA7KgJjC9l6c4YDUJAa2rnaS7vgOOp8DuqCdq7cOFaM7MfGeahTzoOQzjt78SdxoIb+XNsxn6j2bvB\\\\\\\/9nRk22I1AWfCjzeR\\\\\\\/BGWf\\\\\\\/s0ENC2LWRMa8l7\\\\\\\/LsBBXQrRTbCEkNLzw9Z8eqteXJ\\\\\\\/u5sbgcton5e3Hpiz8pL601+x7NlCp+O7x8gKeVXyCPO3CpTVQQz555KkUlPXBPw8P7sc758ocnzYOdcoe7zRhOoWLk3BKmcGyiEr0JJBszW\\\\\\\/2omSObJJ+GtD094SnGsqXOyHUBzjumiNBgRQrbJlN301qEMg071pZ9sGqHs3IyXpXVgOoTPDqicEu3yENUj8ZOO69pWb2swA2eduUVLIj7P3pVI9wvJ4kQZuce0n9rcxOXpc+lTIsWhaTEaGYY2fjgOBMDrKUNsn7fCCE3KUuUBlklm4r7Cs4eSPHp8Nod+QOnbhc8SCNLtjkT+r6boRmWWdpGxfvJWXWWMTR1Lc+eikONneEMdMZpulJFPwIZoRy7\\\\\\\/kbh3xxiJRloZBeBOEHWFoCmnwngl5Mr90xzVrGcVDAnady\\\\\\\/PkuvclIlM09wrIMcUfdyREkH\\\\\\\/wmX8UqpRE7dWhx1JzpI9jnwj2ch090cFudKjj3lNrNQu10TEBEVfMBirb8AYJEHSjRThiFMzCuxAN9Jut6j0tJ4ZJHhjqoFKNb\\\\\\\/MK1Gpyvj48H2eyNLxlHt6S1pfU+7zOXpFTStMKALrR5RgQN1pCk7cLckMPXzP\\\\\\\/5uyxEIkoGtMLG1L\\\\\\\/auLeXgQ\\\\\\\/9Xs5Sm6\\\\\\\/gyLNyudULzczBZJvguaR2c\\\\\\\/BRmIFF9Kq0ANhSO8glqVK0FnokjWqeJ9HsB9I8x+AjVONJhGhCzpj1FzS0A4\\\\\\\/53xa6apxeCjvSFwDWwtHFMQjwaAWzFFoA0LNb3VqhOvrS20FlGfD+bCC3rvNKayAZA1NcactpLpsM\\\\\\\/xgSMNDq6tQomrWXqeXX0YaZdWs55Z8pN5yGERhEXBSf1Z2gAhl0YnwacYd8HdTd6kgZqomgGDabnabAr71Pv6HQf3Y6DQtqRat9zyZxCdg0Oi0Y77FuuXBLr7a1jbbdFyPy7jBHW7pXjbhhHozJ4PANDa9gLoFxQZPWul8fySNBugqS7iwWURCYBROZmDZeTG5CdSSf9bccZBgKErvi1CMHi4gOvYGJUTeVEiatzgEZ9ZLxkHtqGopnUEPrx1uigDK\\\\\\\/CJTlIdhxrKnnOH0aHeEDbbLDtUAp+9pY5bh2iz8mDbHofmSGwGfj\\\\\\\/6dMDPYMy72GrGAPuRFN0OUoP6Xp4+nFktpwDjbwtbTUZVQmt7EUh5zHJXpx93FYzN+oNUul2PXu9oaEl4FUygo2rSkxLIA81pFvJ9pwRQ\\\\\\\/HlYhKngxLLpwMLAYe43bcNlJ7zIHYDWBi41ukWc\\\\\\\/sVuBCRpR5RZO\\\\\\\/NidXZQfFZJjGRVgGeCi2W1ptodCM0oYFnnqH4b0+9Jav0up5uGSlvAk76rBGomlDpRl0p6\\\\\\\/TVntenugwlaD3IL2pZiaVWmjy8Ti+Ju1ZiNAqqsDEnDk0FxponLSdCSh3nDdhaKBl6S9V5yFxwFK2bgbwgoXPoDxUJUp9dcC77fcHMRslUJzgRnnSNTbdkaOiPcy5YDm5kobZph8zFb6vop10ZUk3sKNo6fwdz0ST\\\\\\\/3qniPtdJjBsIHBIc4Z1jE\\\\\\\/shq11Z8IMtOjnvUSc1WcdbZTFbI2bcRjbmBjSRTIzRI\\\\\\\/EhjYm3\\\\\\\/W2ulc53HSgG7R2m7LjjEn2vTjO2onv0LAHysj4GcV384ZsKS\\\\\\\/QB2jBBPLFeCnEMJtutLWnx3kB4yTYos19x7SjkB2h9w+DS61AFdZTckglm8036qHNmL79ocsHu0c2X2bNzsL66MwLy2wj040687Qlcdoy7woUGEYkhDvt4RjMyeBoE32y0DIVlbbxj1uYHDRPu+JnmZ2IqV6KxNnRlBIa9y87S0pbqd5yBA7GlD8DoWIZ8CV7TuTwtV1gX5tAyRpaZXRiIB938qYMddIeyYijN2nfdyqTj9JJRtYadfai0CwAvDvAu0cKiB6srucXn8\\\\\\\/9\\\\\\\/cGzO9bbIKAwdSIwr9hUgHgrTGVWbqXndKHUqrupJ5Wfo9bDyBv\\\\\\\/z1ptGmFSq0YoofTsyXu0lcDZyXt70rHEOSWQc1xfdBJ44Aw\\\\\\\/I\\\\\\\/gTPjDoCBGd+cBouVTVlbYdvqqVekdGwkwOIKYjoKdmE068BqHCx+FucaFoDOYS3iZLE\\\\\\\/gqkUWr1SlfycZ0STOTPZy5bx4eO1tv+z7FOJr7NXogE3k4xPqJq\\\\\\\/Paenv9aOOfT6ydoxwfdGDCd+VW4yfZrBZbYfRAnT\\\\\\\/3czhh40haV3dPs5SEU4a8oP1434XZdZ5baI0gKfrzwRBk6GMtJvcGHify2DSoLoZTQYagIx\\\\\\\/tjiaphcxDowdvmIBe9SO7ErnyyQkDwER7DFNvsoRfHQ\\\\\\\/ZeEr3lyGrnTvuhHRFzbid\\\\\\\/wXLZO+amP7A\\\\\\\/sP0n\\\\\\\/aXBEymNjS21Rm4dlp+hOMnGyLTONYN33cARkusnO5fV7UvVIrA8XN4XzB4Vn1kyXzVB3Bl78AVE9Kz0fuS5AM9USMHhYE1p8DAiiQssAVXooo175rigz6vG8UzpT30Wdg7WPidPKbWfgpTW9sGGQzf+7WKkTQOzHer+vYasjSZsZaXwX\\\\\\\/F+murvRpK3D4gtSgeZoG3AoD2oZJKh4gLi8941922tqAGKlTz1RvtTTO74JKpcDgQ7izI\\\\\\\/9Bhfs+76DbJiWqsTdMBvvZRCq3Cv+00lchIIJ5jZ+9zWSqBSZxIMcDHhm3n+3yWEq2m38EbFmIc6lAlpxWiPCkt78KFO\\\\\\\/lJQrdmYaDJc9J3Qb80ss63Emih9geJnb1S0UuGDF7ps4mDVXt9qkau9+JPGjQGmI3\\\\\\\/xhKEL7SCikMas9p9rU9JdUZU138OaCeTQwrSQrklnOpQ060fEwJYu+AyMFS30aVD+XEFasDbUSm2qyZcn\\\\\\\/L5+1uWejRP6N\\\\\\\/p6JMF6Kzd80oS2kVltRWZKO+\\\\\\\/pihnoyB1Ee3SGmYNkfiobvsmLhtxjUGB9uYmFDFv69ABBGabGelYZLchYMJ70acL07wx5CBmnSPiWcgekLS+qaozWYULTwnswfXRHqLIa\\\\\\\/oFqgafOA2cxh+zjVsSORNGhOlPHI1TE2RIINxAW\\\\\\\/voABD\\\\\\\/h\\\\\\\/5i3fdzKqMTFUtx+3LAHRRU0hhUEFDbanvd\\\\\\\/c9KjVK8XcYM99dPWtsd2ayO+VwIOkbicJxj5nV\\\\\\\/II9rbz7E9eqA9EQSTCOUAB+cm8vKS0MoeTceETNJQAs2HCOhjKPi4nbrfE\\\\\\\/mU5z4F\\\\\\\/FQz53bUAdNU+3Y+CqQVPE97UpTPBbSYNBQW+L5rAHG9LA7sdptZQS52AA+tlYp9v9p9Es8QJDPZ3zFnTEDmy10JZQbETlm96xDTS5YqdNRPMyKXWniMgX3uwibSTayMv5ToMoDR6A6FctE0Ay8SrFlkNEbKVwZ60iRf\\\\\\\/GVYnPiyV7AyK2fDN8tXyNcaUmNKQUjEz0I\\\\\\\/D2bfYF7c3yZ40WWABHpGZuO5zRdkMc0szNTE4Ex3tZmPx+tfd8Kka5Lj6tahZk5hpoNh\\\\\\\/jjFI0o5CvNIz4WeExBiJ\\\\\\\/bhDKRBdtcK\\\\\\\/e0wmWsymH4\\\\\\\/c1\\\\\\\/rGfPrF985HsBML2oPFItBaJoun3zhR0SjoUiKjUWdGh0fdjoHJ5pJrjvxwWs8q5m9+0Pcs21yDEfsJO0E+mlyb6kFTzZ9YAT6bLK8S+EYYYAPUJj\\\\\\\/yKiE\\\\\\\/36nsbrprnZaY4zVSjxJ7puffs0b8PxQlkj4BYtyHxSRToLiZvacgHAvCvle5MmyeXavkJcpJMz6\\\\\\\/fOwRPdJIh2BslqpiG7lgaTvc\\\\\\\/tljUk6\\\\\\\/CwfZkESNYIjHw6NKx7pWE8qL+Gtfehcs6LDB3lIYdQzhOa\\\\\\\/KJKiBtuGgO37Y+eY+zOf4XiO711ZSKbowgGLVm6MLRo0k5X7dht91\\\\\\\/NnksYpYCw79KtSPfRYreuK9thqrdTcgwnEx4BTRutE\\\\\\\/d4dAL1wIUJOUtnMk\\\\\\\/sPcMqYl\\\\\\\/TW4gDo+j9jFaKLt9bKR6BrJYEv16w6fUpHvPDfiWQZuRhpPqcThrTRLhA9NKHUPi3osPerLv6dLvlGiFs23qsRjFBWOK4s1FEhhNFeJYXVt8soiT1A04aZJPpy4\\\\\\\/A8Hs1+WWJjxTR67XsmTNqqt8CPB8cXX70I2RPe97isRYqztf9osLgUikLt9l8V8ug8y9sG0Xj4QGtQJY4E6OfQUgC1RJyFCd5aUB0TeaO2jzn9V4e\\\\\\\/fp+r\\\\\\\/yhAz+2XRulv4R4JRTSxnAe84kVuFPnFaFHv6E7tVh7Ehgs56WGlsydFuJ6rBw9ozKwGJ\\\\\\\/Dbg4piwgQAV3z26zSmOr6s\\\\\\\/Ro8FptxRbg9psU4v0Wor4XN1P9w9ObpocNPhuKUyzBtn7OjncAmxAt1+zKs46ObOA\\\\\\\/qjR\\\\\\\/CpA0IJv25\\\\\\\/p5rYYkfAkgq\\\\\\\/iWjytXr4JQRAzu9Mdp5DTLsCXJ9OWh7JyDTqd3a5VA5n4nwXNwIK4ZaPl30W6ttlOuO6h7t9ou3Nap0F7\\\\\\\/O7aROOh8nd51he6IKLKa3l8ozji2Rn2ye\\\\\\\/T5KhK9jJKWfCY6WW2Oay9xCoen0R0VYf6Sx3\\\\\\\/wE\\\\\\\/RWgOa9TfXdse9l7TzqlKRVqmqJtG5qJ78X2QkZBNBW6KtPuWlDDSKOWnNfO9WuUzXdISnAPhNtMwrJrb2RL7hp+ItN4VcSBVVcVzxFGpCoXL\\\\\\\/AEeCcPvzhPX9B8HDM3ZPyd1wTs5eMnIPGmHxhxRWkapvKVo4yoj2hx9BPQzqloqV234tMVKXd3DQSZZNyJ1KN\\\\\\\/POPZgR09yRnHPkztab\\\\\\\/hAJyET\\\\\\\/yfW69n7xBb1w+QZyyyyhOw1sIdY8uOKdxkzs6xFFe9ZK+SFGDGwTh3YJZlprPqd7pGfwPOMTtCGZY6s7UpKrvaZeLfXYb+5lmB7rdwm\\\\\\\/luDlbkQpe8QfGSymqu7D2YiJ+jMp49XpYG8agE7n7b+64t7sqIb01vL9i9NsUqFJsFbMkXFxxpr9mqQJYJn4XLfVyM5+ymxSZnjAey+hbL1QX8B+ZUE4c5oi4N825uszvCGlfa7RVFXKPFS3JtgPmFaBTm2\\\\\\\/Q44GPJ4uoit1cvnLn9iimIGiYV5qPZht440fG7ynONHc3jDL6iz2wcs\\\\\\\/u4EpsoPEopEIMJ4Qs+f6\\\\\\\/em7AK3OIzp6uGYotpcbkTZ9EeP7+t1\\\\\\\/m5J6X4UM3NEyCrAuZcKqFl2Neiv\\\\\\\/fB9Rj+IUBIrE2INxV2sryrhkXq65EtJbUo8SkBDvCODhvHuBf25ugUgqWkzXYoxr2uN7Am48L\\\\\\\/e2rudjnNzF9J7Oc5AIrNpdf6B0pOL1UQq+K11G2GUM6cNY4X9+RdEraDyEz+BTQvB8WJuZo\\\\\\\/9SCqtmjeh2WeIoz3AAstAbyVc2a3LufMywWq12LCMSr67sZLZZHEeUHSuJZUuBkkqN07b\\\\\\\/jX0Ufj1AM1zaDo8zOb3FA8yv666DlBNM6TBv5AJAHlFBXvH3lejnwG\\\\\\\/iSzp\\\\\\\/I4ejgnSnh38R4WXkT7oDltuBKvn0ogsCFThSH\\\\\\\/seBV8Yjq+sq+XVyLknGtWeEuQZQZeq0BLOynU0+W2FOcVlUHRJ7CV07Wrn4cX6CSXdw74EfSbJ7Y1A8D8TJ3Oc12HF3prSzI0AAMJi8TLDeORP9ZWqc4oQmB3zwaZn0cAZcvvqHvYi6anUaQL2iB+6kJw8YtnSaspgKz\\\\\\\/gux4p60jGjIlgZVbgNVAKMRiPUlR2nPxNRg17JymZsQOKfuUSpNLtsQmipmdyuhEixK9SMMz3WGT4QSBBr3oL0\\\\\\\/Z4u12qBP30WmSGRqa6zOLobrIHWK6kNTUSJHtAcv4\\\\\\\/YtZkPHhbgHH0UgFS5LEUkxKLkPJwGunmlHDC5ULjAkjn0HHK1L4cGobpJNk6th5JSJOgZyWuasGoZPVS8JV+woKqfVxfEp4KUk4C7\\\\\\\/LR4YEGqI7qQXeQvxZ\\\\\\\/ZHQnSxYr2AmvofAFsDfS3uEoM8I78kP8eQDgqle7UAqd8G89K5bJpCUJp3bwP9HWRlkXGj5NJV+6gjX+BVCksx5sQmb1P8LDKWOx783uDQRY6Sf7w\\\\\\\/y2Yh2A9VuwgcohgUSCTvK5vydR5wcBYrLCkUEvXy58s0PhEqJvn71Ct5OyZjYxNOE8RN2Cp513Sc+9ty+Y1OSriCnZkhC6qPNrvojtiQIRjoWvCPJstRYcl9SHQpwChARM4tVOymhPEvAeAVwjAkS5Pk08I66l1awn4p+DsVJgGEKrZKdsmMhwm0ymVJPSQTxolPncoJnCGMS7avA\\\\\\\/2pQWFgvFcKQlqLnYCw+fgKCVU0E3iIBdxck0gX6RWWVjVLQdRW734sgblXpR6CUdIWtt0AKIFH2ilWnzsqEYohTTK1rh3FKD91tGj5IkuvVNVdyScmyReMALFwwCg0yplZv+0kyv0\\\\\\\/YL1jxvQToQRa1KAuaqtRsHC9Q62G7YDRT72My4bCrZEAkWwxtGddE7+9G3ITy+WY9hfYiA0w6nCt0LyLPJnbL1Guqwz76rim0aZyts4Vd5wEZ5sk22u1YUlMvLSlkdPzUxfm+DJfQb9ri+2NI5GYXNJBYb6TeVkMIvzu1FUxSriIuavz\\\\\\\/mxcMux22K\\\\\\\/WQBSem3etC4385thbj+CsfzaPeiuViguYsqsVqp06KIXICUyHEgW+u0sq3M+w5anikmKRRiLpx663JuY9s7Lnc0tfzefU\\\\\\\/jEejzaoX2M71Kb74zmhmClfAHzM+Niw07fNJuKYAao2KQ2JXqfHWwD5RCldjJfl01IOq4z\\\\\\\/QLFokfhPjf+OBgBbDqoVRsAW\\\\\\\/7BCfljDux0M\\\\\\\/0ZM1JdYjW15UApong90p1GHRMog8lFg0U8qyxlpwVgddZl2d3Qhg6Ig4YLQm+yKzfXCpkdoDT3wZDYEht+G47Y\\\\\\\/PsDLNIjHeC8bDcTZzue0Oh8LljN1X5sZ6bc4cqtWEHluANZLUvCiDjk6nUeJVdvxH9P5twRkTnTpRr4EaCk\\\\\\\/kIvRFqSuFdHS+9qQ1v9jas5j7uHfJl\\\\\\\/Y0QJfvujo1bttba+ehg6nFr2ysbX3rCyOuA61mg42hHFG7J6f2cKcachUL1WFlvz+zfo4AG3NLn+Z5uyVVRe56HHjDsfTw\\\\\\\/sRjK\\\\\\\/pY6q+VcsvagnPnhTUtCLhMoVnjC6hUthuQMKi\\\\\\\/m6qqyWU+TNP7Idv78VxYRvfI1wKO9a8WFQ+VtkCa08JuWgBKJjEC1jDypfEw4wenItAm27sNA6iozUXIx4Gq0ZwFVCRNPDJpjkFs3OgdHvkekUgNnXm99CMTh+e0XQzLA5fsck\\\\\\\/WxGTtVSkn5Gie4GgGUpAdH6GSybX3KeR61f6GgoqwNO3e62i9rt5epTGv0Rs9d3+8BSZKVtk9m7F19bwvVB0bfkYdeX6rYij0IGu\\\\\\\/kq2t1gqlIuNEEMiv9VTJ9ZyrsdD7bnj36xuw46AOUM7od1MApZjKAjd1oKQJn+9b6XQnlfNroowdMM159VryoY5eqq7HkeXrkqZXAjxKIPhSRtx\\\\\\\/PtiPZ65wHjrbj\\\\\\\/yZnTlkO4jeCrOdINAT34sAnF6HOwcPygv4tS+NnVt07WQYKKPTkPu6Sh5j5ADGc07LAQ0wvkaK42Y7n09FSzOk2vgAZohTG6TcOWwbX5SJM+hpbzEv8yTpGJMd783ncnKSsJRaISapAPW3fIGgspXSblMKq6pf4Bd+vVohxJXVJqrS+i3yolL825nclvzPFgyt9akDbJaLVniCVE2YatIWpeuFNAXS4n+fITNcqXo5G\\\\\\\/6ArVsvhpcjiPMkhN6qfV9ST8Q2D65luBeqS2rozPDhmqGniQk\\\\\\\/3bjhSO4aTKM5k6em4If50odbQlO6+ThPcvCau8sMtiloNk59Mg85xAmvbGvbkz30+UDvzv94e4JlzGrUH3Hl22C7IJpRVrz0y\\\\\\\/70DOh\\\\\\\/ZAFZLpQpT8DVG8NRPIQ9odopwPXIUGWIwoTOI1vvUEV6lpwejhUsRR6kxt7kfEjgSIOh393iqNMlsMan1UXKUW+stupAmoVI4MqYPizQiGAGSHL1dUdH77eShp57avM+Wu8xaCWw9NWM1GnP7LiDU6BXEi16wSf4pFaEO+O5FZgSz8yfWqsDuMuMEusFkZ7HeWPkrP\\\\\\\/A4N\\\\\\\/vku3OCSzFpfGcVdGsQLPlJCcpsUzGD8Y02jvOnkihe450+tXvdaeHjbxdXAen4gqD4cp6pVH+H+sUUSS0W0dOgyJdLSFR3C8WjuNfaqkh3FmZPDwv5wvTItj9c6n\\\\\\\/4wJ9SnwGzyzt2jYic3h3KplWbVy5HFQKYD\\\\\\\/dtdhgsXdmagnu2lIRdoxb5tVYwtjSxZbHxx5s2goFGGp3wgcoQlvYORFD5SY6XJGOsF1b9E5OLj1clws1VHjQV4FoYxnI6\\\\\\\/6SQXsD7DggXjuhRK8JYx+3oPZpO2bXJZ1PL9iDEar2jbn517YnNq0aXGvjpNRRcDfMz7kT1GDZK0hBLK95+0P8D95b0s1GjmBoAuD78RK9r\\\\\\\/hkEm52bmleQpcWLoZyoWJboofO1x7nxl7usz92Q0iQjZwUMbnoXR+pCP77xaavOMVVzW8ipqMIUq1ZzkVN4YcezvHyEtxfykGRA50Z+4d46RAvNdt30thJeFBFUYMdvxFcDNonNwn2tjawBcMAqKnOa6drM04iwzPYwfAXW9ym0DQMCwqW38JO0xnZL6g+jy1EyXEzlLnm8an2jNtl0D14HAvW\\\\\\\/HyOKrv5LFe9BQW4VETcI1cIBm6jSaWHMpoLK4GTN8iWoGBY0oiaJpfA1e5EjVztq+ME\\\\\\\/osyKy4bUAKCNULdfhVZWlFSnV\\\\\\\/tiJZY7QboF+LxqQJgmNU+2VpBiA+NorPm4UGcBZqiAsDFQm5UO60j5VIO3k19EJzHEV01utG4o0HtAlLDJYfgq1q5h2G4yKVr7hm5GdCa\\\\\\\/rrPh3seXPcnbHZR6M6GIsC\\\\\\\/K9TzCUd0o8XA7LNjgMILw4UazBrJq2NIuA+usJm+lK+Kw6ncPT26k97SGbWZ0h8bhuxMP7cHG6JNqrXbFBskXafSVflp0V6jflX97ArdKDV5Yl1FwgAU3pp+h\\\\\\\/MalfydgT98Es1XznKwwxYrgeM6OUadweRdo6hf8\\\\\\\/XPJsebrPiyKhCw8vS5d9wXh+MEpQInM4KEAokGUGmKj8xHr2wDLxDYKhTSlNZBMaZgLZQwMmZsJ2\\\\\\\/85hmLeXgeQ4tVXhkPksBWBdO6CKSJCissMcIw5a0oEPZJCka\\\\\\\/HGnx2KH+db9zCFvkMGCyuT+RwoMlohJVuTaF1M2Vu5kDQYaokajjooY7cZiPgsIZV0B8ohe9g6SO06+6bK19pXo2CYCKx7ZHgg6LiILt6+lvk0FZ7F2d0Vmi5RoADsZhNstl2FEyoBfoFiSS+IP54SUn815NI+X0nHSDSu6hRBBlq1g49HG5JbTYjpJmfkwREk\\\\\\\/ha\\\\\\\/YJ0RHOYEgTpP3C40WwN0twodUPogFf8nYuZ5HduN2CgoK\\\\\\\/tw1Lu3TBH1Gm2JC2PDHGEboUAV+Lo1x3\\\\\\\/vAxGe1S9G8ylejCRn2IfRrHJ1s3UpeSW1oHPiC50FylEejlPsgKeRzpdb9D3p1ggHWFdBTUzeXWC7M\\\\\\\/7sJzybJ0LE9BenYDSBV0xoJQjnZuxkZ8s\\\\\\\/7kduvZ5PVCsCL4\\\\\\\/Dx7oOaJTSRp1uHN2W\\\\\\\/z4LI7UMSGD1mt\\\\\\\/UXqcw0jvyxvM+suxx4xtQlMh3ZLQRNgze\\\\\\\/n9Q3oH5OoyLs9M0ThSnESgdVJA6JV\\\\\\\/9KUiE8HPBFHku4bLlfWOfjTYIUqw6+ItNVR2lmhwkWw1ZilR\\\\\\\/6HjctQ4u29moNT3G\\\\\\\/BCrvnSxxBfQUnovF6H7lPYH0oE4F5fUYY2I8S5JeU2\\\\\\\/sEddaS71aKPgZtnOvLlq+MO2et0qerYFyrLCBrIfNtLWXPRanx0qh2pdaAPvgNim95dvwtV74jJLfI4NzaMOPu5+hMWsnE8HYm+52Hq7dmvymWnZX4vEDT0cGZIBjx1eHm4a3262PBHZIFhPt91PSSncpHlOmmYGDPUaTS5nJyQ8zqKk9s0+lFdCYMqyM8gzFBCTvq4EhEY4Hq\\\\\\\/34OBK8J77A2otxR6ee2SJZGEJrYAvvKpu0xw2Asfi4JRDCZUHIpLTmKgQbT4A\\\\\\\/qFUnTNpYPfJC4h8bKpP+RuyHe+bXVSwCw6DNeHvZDxxMcnjvt6yFzhKjaQUu+JWWbijb7qVIcT\\\\\\\/TLKy3N+nr3lBXdl6j4zmpbHwquVMZ0vsy0LgMosNUkRihKmGIPiy8jWECtr1wGMtBVxS5PopHQexewKpg7pnb9helV3siuzBNt7gqH2Ebp6dwlHbljf+gnF\\\\\\\/Uop2fgs4lDHhBIJKUJRbRy9MIjgnqo0ZDe164R7HaRhx7S62MaUEGfiB5Mr8yU7STL5OHlf4weuH\\\\\\\/V6corGyZjzL5tKE2I6qG3vWk5RLKX3UbECd7PLUxkLYX7Vn7EtyNzvdA6EEPZbiYhBaA5HffSOM3UUQnNPjj71lqgXe4woHm6IAgLsK9ukHS5Wl7fkLC897KrHEejyvgDmthCuyX8afiavZ\\\\\\\/cTm89T0RPp4nw6UdgDN6eiWmyQ0YpuXR3TY7xCiL2dkEK0tUVH50owIIo9KSGNyaFzxpPYDItygkfBwJ8d\\\\\\\/oEERP30KB8ET4e\\\\\\\/LYJwdGs1Tp6CAeUOWl0p5on29h+cwbxlZDEI5YVXG4OMu3KCT9RiixEB8hp+ZVBbYrbpjWrfTQ\\\\\\\/26h7f4Eg\\\\\\\/3L0ai91W2LBdT4Ft4416q+jUpgmWuAdbCIUjNNKsoUz8NIPMRcpXYKQRQoAA+VVQYnPPXmJTPQkI6ZCssuqMSThURuf9RnuC2y4gMKepKBueeSc3j2SVt5Q6gtW9YSewHSnJUD5AswWmhDGlFym9zRUZorpTbp9OmiYD7AJ+g3aVjJKqsGsywUqO9idFI2PYUVD7Ob56j3fv0BF9ASqcUGlJu9EBg6Jj9SONNsEc6Ij3SVdn+RdlL+0Wfd2e7CP2KghYJF9CWtsydxzLgvWto+5ORbb7j66mY6BJ1N7eR6rNLB7cTp3cJ1r\\\\\\\/gz\\\\\\\/BNOQNy9A5W1Lgvb5WbY4ISAmvHqLL3TwmxwggzwQvn9uRaDV418vEar5xgcUxs20oBqh4rt\\\\\\\/jS0hzspk09aj+nbm9mYThemsiJ\\\\\\\/OzEJtgFIJpsKEQwiAX3VksV5Sb6dX4AkmjqKFIsEqZl+Em98HI7jfPuJtIKY8\\\\\\\/ksQgmDRNHsmfdOkjdAe6q7\\\\\\\/nQND70OUWrsTYl2IgbszqFvXZszYU6WfF7aH\\\\\\\/mLVG88wa3YyOxZJe9X+1cq2zTRdZSj132FiUDgEdJX1uZDmvenKU4u8PkhprbVjpGn2nbjcsfrHOUkH4TfhukA15peLO9CTLNvL6IGN9RTM6INv4B7Se96LEu334bKuTguVemv2TwmHGwj5TqJlxWrhuOqvn9WnJJu7n7n9FvdGzt68PNut2l8+m1n1PwALac964ibsL1rtceZd8So7+tWmvV5UwaDGDK8kjE9+2A3MklFFqp2AVBEzAfwpeFLmgO2BRfFM7Oz5Cm+c\\\\\\\/VSYi5oYCtRltgNwBDcNTC66lVWRdmzoRVJoP9qfJMiWnULjIQ0hHZZI4TYgZB\\\\\\\/jbI6mGOQedROxw5V81ekIiX0NZQvp+SpmfqOr8zK9f+6VxfwrKk99Ex2jZndfer2+awihV4Q6rc8Xkb9Q1Q7PohczYQCQPrnaiUIUfYFL3\\\\\\\/nXYuKHWm0PnjBHqwkD\\\\\\\/saq01lEzgucni+VguxnP1yIzxUfP4pCI4Mip9NCdDGvMj2Wl+xyc\\\\\\\/aUl8Lh22GVzJEVS5cM2ph0keWhDtw6Jmh+J4sFff2R8TJFdUzHzDb1d4HJYOkElRMXwQvNSHGWNJON5fzc8MHOJZHn20YcNF9In6KW6IEHhocwkvGuOZ5xsO0EShPn8gPVRK\\\\\\\/efIcUIYmCQsKxyHOKtYtKB67qwNzQK8Ua3wqQOA\\\\\\\/Bly9eiH+6n2qpVdihsOGlLze\\\\\\\/CQyAmk0l7bwXDeEQzQzhgaNyC71WcILVzwNyPnW4BmdBCj5wZA6jNtHtWX+NkHDR4nozAsdQ8Ee3pnkaWnMzgxbiDxJfrN7wXIy0GCT4h3BJwEtfmMwn3AgR+zjFJYxpGJU5jAj+CV20a96mPauE6fybrJC4I4kfN+VFlyJt5ERrwv\\\\\\\/5AhjUV9pNuYRy5q0SY+mYASq948BwrvjkSaUFeb4m2Q9DUY6KZ+FMK6KBE8GjJZisKXR2Ue7Ix6T9ua8aa\\\\\\\/m66osDH\\\\\\\/9dVJgn4QI1pHRNXMNCrW9szYujuiXIcC4cibhSeKYENk5ofAt7Ded+TWig7JZrbayq\\\\\\\/i52lUFHnYhUFhSxnmATctKCmgxuJL5QwV6FCJQ1t25pl\\\\\\\/WnU1qRWds0IQMmQxupWMCsDfEY9lwPiCqgAkA2KOHUUvoR4uTGFFmnEzO7mTbJ2EcAzjVTBPf6gyygp63wNiz9UNjC56axxNMZOHs1pnxCrjIRBlG+8kBkjOql8flBo2xBDgsVLqJfFDMjWooUcLg7V1mhU9aRqa\\\\\\\/Bocu77QqwgZsvuSEXeReVjghAd76YuKWdk0ZAd\\\\\\\/IBmpekwWNy59570uS7RaHkWUlGjHm59DtzYvG\\\\\\\/zmIznJKaVXyPfEll22\\\\\\\/3netZp1It0Q3iyfRfAMwSDhX1DwolyTpVzJ+15cfxrWYXq+\\\\\\\/+aIwxvX9sHfh4QflmwurKMJkJ6cfPbR2\\\\\\\/sMXZOXwEbFMxZ28PScJSAMo1MGn6USK6h8tTdpvr0Tki2qGm3XEkJMr\\\\\\\/xFOIwpPnaGps8iIFP4qFDexrfoQ0cCzrcuLAqNhGqiypK+nPyHwg2OB7TmnklHmiGqm8RPmCsYpYbKDVapYtWMD\\\\\\\/d9P6R\\\\\\\/voirG3p9pn+4pL63WeK974Y3UGXbnNY+b6h6V0Ne9Rb0nZ2cyhOxXRq1nIukGW7gOfyCNSqUcLqSPIyMxaTpaz5is\\\\\\\/29iXtu7T2VJh1o8UtGtIO7g6MK8hRN3Z1yusYWJMB4UoFIWjV+OgFn+uwK5IpViaTprYlzXJ5yvOgRgSpHSMhKi+YK+jq0PKEuWxKE90yBbHUDTzfB7u72tuhUOqRnKtW\\\\\\\/Ynu5pUCx2c\\\\\\\/1aVR9uiTbddMDDAgFkCJElSvUpImI85KZaYx64lzcYxFC7XKqefwWAKIH2tCUC8\\\\\\\/2UQbXvhV9FK\\\\\\\/\\\\\\\/juZz\\\\\\\/BHrLij\\\\\\\/VvDhzdHxjMenj8jv434HueF3TLcy4sutRYCQje5HazRkvzlSUW6xq4nxvWwBpTMe9bTCDxAfRzwuxiUi\\\\\\\/qpEdBCXb5LydzdKzmTYzL\\\\\\\/PxzOgy0wy31Z40iBlkMu15XOOq1WHnMdJfXT1oouC0S0TkmbGU0ZTmPoZ\\\\\\\/SuHqjyidoCFPV8sXxo8vMopi1O5QFt0nSuXB2uj1baGXRcVpq6QBHopsurne1nmpzsUXXsJPWjl2IKhZFFal1O+1cr0A+JpvbXYKGxfrIT\\\\\\\/9byVUX0oPvnKCI9eVozExMZvOs2HdeqDsNXNMSI03DE6p7v\\\\\\\/qVjFWn6e9ZNJzn9E2+7SS9iAjzeJddlF\\\\\\\/pjs41DnckIc6UkgjTbstnnPErSuEcmpgxz4K11hquWpVtWRf0VqrszHRH\\\\\\\/aRTM2Y1ptTXnY9awOPmCA3VGCspO4R3SeRCvZYrqVJXWjBVA2La34k6\\\\\\\/O1JzfLiMZVGWAk1sdP45\\\\\\\/EZ+LsYNCNUljNIO1ognmP2CL9mId6l2YkTUQ1AY2+DaI78TJiiq2JhJQhIbTXumEPczY1PMALs5fUfujYMX+ZNFyB7To0aeMKZUl7DFln1zXUmAamdfQJAvY5c5E8\\\\\\\/XQAUuT\\\\\\\/PzEpa8ULmt84GcibXH5NjXyuZnXELkJfV3lO8gfrSFyPT3NE6zjgqJDfdpLJuO8fjdVza+AvtBngE18PWhuGr0rz\\\\\\\/+ybgS3wUnCEKLznGbjjb8GdF8RQMFY5fvx5seMylFwr\\\\\\\/5yCVx4EqksyOq2NZVXUWg14jcAkGFXvfUw6gPAEdk3D+PrgRqdo1RU5Y57wvAbQo+IRnt5BRBnvpkyl8z+r578hKCCTndtverpJ3qi0wdCJUI9vz6stHFDlMp\\\\\\\/5pkNGkOqJCtCec3ucNBbs2HxjGZRl\\\\\\\/3CKfYZk0uRIfH5kAYTwikuBtv90OIepc7mv9q6SV9AVM8JudbJ2BCdrAAAYWT4D3fRUZ7WxfNufKN1K6SwD08EUujfLE3Dcq8ISF34MD2+seQ2qr4KZwWj1530PlkqIHRIrEQ5TsWJ3bTkNSRHHJ9tUV7glPhY2OAXx0Tm5WmH5pX7vJvS67gHJH1cUS\\\\\\\/6XP1kdwdNEV6jXSSdyd5yiBj4xUne9dUlwSVyGhusOx2aS+tk6SmxEAHmbkd3tCtqe6lrhJEBp\\\\\\\/6BVT5M4VsLC47h2aR1eicohO2Xj6apBK1qf9COKmzZ8wm3lzfshYpr55hoFcyWB39HP28kzHpZRr6CGZ2Yh8fTUAyrSeohuE+uGNnWd44JM63J7I4ztt\\\\\\\/uxMxG7VFzmv+JoYYnrfQWD54aLn6ycrN9p9pGe69\\\\\\\/oe1pnAHG7rBddm69bJUyV3xoNb5DCcIBw\\\\\\\/21LsHmEyEPOueFwFzwvmJufyVITZ5DKA0GqQrX1IBDagJ5eEg4S+4pa+RFN8UBC62CcL8h0u1wtDgUEiMYi8w6RjG4ZBkYxrNp9Hn4rVgOVBlNCXuM\\\\\\\/iuU0pbSZiKmL4duwiDsZTUtZBwaLFqT\\\\\\\/JNjPLrJJwwJlECKnZyNxlpsVkIs20u9O3IfrdTJVobaqQgbMyeEoKNiGSkf26ntI+KjIVJtp3LMBhLQwPcqkg17B+ZFvtDbQYO3\\\\\\\/SU5GvfWT\\\\\\\/C3d1C1IK6IBmOUCaqwtArIeFPmnhpaz3H8AOS7GaZzjIcNrCD6F3H7cCKtucPqkUCBerNoP5BEud13jnWWk87WX1wRXq7WfJX4NX9IfnrMIxttkEVPZ5Yo3W37OsRHPjOogrXImxNwMM3sDiIOFChRmNwyr6+L3A0VsfqZZNuswZ0gsSlaxF6mJ6xGRW\\\\\\\/hBwjKCXrJ5N0kEAWH\\\\\\\/ohWzUZ9NHTH08B\\\\\\\/0N608NdBAEUFDQySe\\\\\\\/49EO5FcG6+dgk1ekKx+Z48pFKuxvqd7MhDCLxmns1Kkuz3ZgC8NxCllklx9nlPHIg5CbhdB9wQrvvRxmfYh023Pq95wRb23YTg6yJzqhspotZQhz9qvXGNRjgOq+sR666ix\\\\\\\/w+0uCl+MTaJjDGFZwLGd2hHI5HtMQMvl0Cqpksjucfu9VRBQGH\\\\\\\/EgQ7twRDI0ZvkRJEczEp12AJcCaUvMFDzcrWnphdYijrSjINksp+APpLy1e37U6fkZBXiR+bfa4RKi78CQtTD+sXeSoDybe2HbTgLlvSB6tKq7\\\\\\\/6HQmyr1kgAU9H5IGcvlQawY7xo7knrg2gaVFmhhVM9OPSaJDB58AwWVbXug7QTQxOz20TtzIofJinKiLV8ohP\\\\\\\/pzvGWSw2uqpD0xe9QqCCHRTOJ2OBAMUyaM4UVtFXTXyd9lM1ua2\\\\\\\/TabyavJdCJn7L\\\\\\\/N6h+Ya6CBJhL\\\\\\\/yPjIoGm10VKWQPYwTUswjLs\\\\\\\/aPmMQp7i0FnpeP+TZ6yVDtafp6wysjD8DldGP3bO9A+90QWqLMo3aGoc62MWIW15J3wHBGKzL5Lcx8vQh31\\\\\\\/HOhXQY9wC8R3wlm\\\\\\\/xSieQL3FEp2vkrYFVstI1q5dzZkyMVHZukoCKndITBNF01SQOsSrLp81zesul3OIVa9UwtceMqL5Qt46YLVMSws8qKq\\\\\\\/zw93WSBUuCpKUjuRFCDawmwHEQkc\\\\\\\/YAx2HJen68OcdgdzZ9EfqwBbz1K1va4u+QgDf48h\\\\\\\/j66S5x09k0BnIBCiyaEQfeq64m\\\\\\\/rI5tOk2kjGjnwshGoApgxW3tA088bHI5qwA68MMTMxVrXGs6kBysjiML5jBxMjVVDHf5yCx8lFT1kJZN9a80zqYYgdQafUYi0GHWu+Q677d1KzfnyzPMb3\\\\\\\/fyfhzY4DiZm1IaCgLt8JkJCsR1KcTJtbeZczhWhWukXMOo9kLI\\\\\\\/0CktLgQneNfog28wtRr+hiDAbELLN5RYTMfQOOfSPyZimxn6zSaW9JlngRvDNm4x8iAnHWHu5HkGKtjiCvVh2CcXAivtVvlIWMLL0LEMcZFRN9uXFV\\\\\\\/7hz1tVAI\\\\\\\/CW3zwSQLkdH0hBSXBLzbwG9VSFM\\\\\\\/0A9bNyOM1kP1jrJW3poYLK2+0c\\\\\\\/2WOawEba9s8nlW+wSYHLnARN9qWJkz2Drm5Mftqhj7LAUxrLoZFCU29k2A\\\\\\\/mPfco+V2reAFqrPEP3y9T+YdbO\\\\\\\/EuMkPSxFbKaR6\\\\\\\/Ri7CR8vn5OCPrcrRnccZ6RSd0\\\\\\\/8u3x\\\\\\\/CyHXIw3aGFz3C+P5BTpmKCsguiUHU36tt9D1JMqsUuWiLNllIGHD5FiaV\\\\\\\/9eUJqZv7Ouay\\\\\\\/Tpo+2nZlYSV3d1DJ7c4ttdpD1xoKdTsq+J7ebqVvFLVjTvMecOAmxrlfXZCY2u9yfCijF9Z4AkXiYGPGJLG4NUYXlPT3gX5St\\\\\\\/DmLlLPDHYY2DhdhDtgFpjXcI0xlg0W4kOK4Fv\\\\\\\/TDqp1q5N0polwm5VbIMMREdPDQ6sktT1sTpKGgKe\\\\\\\/Wb4k+Vy05G7bWXuUFKhwEJWu0AF9K+b51LrvbEYk+DbwN2UCnKQT\\\\\\\/SC+4NEgTRJezC6khPRO4OEWuD8v5v3xyfHnJNQvjoMgGIvtt57JH9PJDGYEqnlrxOFw6TCXhOlRkzfH3zVARjYbTBRmY6Z7dCUDGWzu7IoBf7KlQDpEwOMFq\\\\\\\/l1JLn3l5uuYDK8iMuJXMA0EcA66Zi+uz\\\\\\\/s+v2Th6trP\\\\\\\/SP0QFztjmxV0LBia3d7v6U+sZn\\\\\\\/H2H6s6+r\\\\\\\/M2szt+v\\\\\\\/KdsONNaQfpPX4dkw2SMcnBh0JNu0pMfQUIzWGrMliRz0ZCN3CcXfDzWkiSsjcx+nu\\\\\\\/R\\\\\\\/NYEbUxbkd57GZwi\\\\\\\/bkw6M6ruARlXEi5QMavMwXm9X90ye\\\\\\\/Bvp89SkXr8f9R3DKiCXy4JrBp\\\\\\\/enVJcNXREbt1EeD8voYQSBTX7JYw9V\\\\\\\/QdIIV8Z99EVLyqC+Qqaud6cg5vHZSFSyZT+ZBRxMw8uebMBBl3C9eHv2rpkgMI6rGliQc79tfrVGT4UUFGZ3yiEAf1JHda9f7SnWK3I2SABrfa2V67nMzl5lrDs79VsmwyS\\\\\\\/NVJDUx2MJjJqLiegsO68T60QKRI8DZgAAaBOorVBE3lzSUXeQlPRIcfDQ0d18ALjz9gx92D3UbIPQlFWVGPFpfZZHjBOHWDz3AHeFWGhgaQW4ElRzVmrRbPVLg3yaPOK1BPZiQTBlqt8bKk2KVvfr7YxEZegHFaNERSlSQmPyjjZ2Y8fAZXadAHot1XP6yqGmllDw2WwUrG5uk9lT483XQUrHXY9qXln0cJwpcKd7CwSvkFcPKTvCJJMRTgP3GNfzInCa3Hk6WYHRJDpobqymIfpid0kE1FudJ+2g+HsfGY35Cx5cz7odw4Yg82mA47b8hiCmbn1p6HFYKJ\\\\\\\/Q8juUC9COxDXzX4ivcTyvkmwkXWOzqzIepQfYJ27DJLBSnxz+2oathQg4so++ubMPIUahpojeNhVzz3xuRUiBbiqXJlpbxS4+jpo2kSZmx\\\\\\\/RopQ6b4kF\\\\\\\/eUBMdSrt07l4gYjkrPAmPcnX\\\\\\\/9T9YThXx00PJJoZ09S9dDZm0JrnhaKeN\\\\\\\/qhCD\\\\\\\/ts7wjtfOlBjUDNP78fd44c9IW+RtwIbgePLQlFtzMUBNUvRraPfITC4rJuBODShDUpwtHyjkg4LRMNxXLEkkrk14LiWsbUbYfWXEKUqU90uVtvuLUn+WScEGN6IcrOF1H6zbojZPpw6c0\\\\\\\/Zwimaqn+v6i2KGZqdSlXS5rmQXi+hS79Y5nO2KWSr2IkCcYWUsj+yztEaCWbhQJW+5BkaOrSEHb159UKI2\\\\\\\/MQ0DDMkRyrxUSZzrH6lMbhv8Tw\\\\\\\/6ASaqg+cvwRmQIHapKZQQD85S0CruYH4c27C5Tx3jk8YVqDVpCOY3bI7vznY8+Juot\\\\\\\/sKHIp6gpCFo8CTLwYmHpQIRgeKCsWdN2Imf3xCg48RwG9YjdqohINahChMX68rtVV\\\\\\\/RK5Gl3UtpYmpq3as7rHnhwuoEcVNdNLU2KLudcPt1WZJVV41AomLR\\\\\\\/gkMnF0qd\\\\\\\/abuz+fPHykbjEj\\\\\\\/QLV6VYVCn99MXU05Opzxu7SQqFsQdaHqa5G+6xNT4ES6rtnF19AOwMPinLRkX2CdusaxhyQ1okILajX2h5EuxRec5EYhIjN0YT\\\\\\\/XwJbj4uq\\\\\\\/vjZ3WwnvOt3gwlmiJ2sS2pVfRxVKv9vihQDM4yRTwdPHIR1ou\\\\\\\/L6ZR4t+ktRNJs6YnH+Y22j1oQQ\\\\\\\/X714kc92JeRadN3xrVmjaHWlaxDKiQF81o68UZmaBmL01I9GtT9j1e8VQRXXGL6A3ggqDA3nXhOfNsgb3RFJl\\\\\\\/f4heFbaNiKlm1kMoubuXirTzrppK9Vly6UAmnIYQ7FoFloLDzWbVSF\\\\\\\/Qz3KJ1RARFDpU63b7AIe1XvkF3XysF8iKnX4zMTHOpUaeERCdD8\\\\\\\/LBvAf3j1tjA\\\\\\\/ds3XwRw4gc\\\\\\\/NKtEGlgsVbrGDu\\\\\\\/azHqJ1y1R8SW1o3B+FN1HTylhHEdvWfcVBJJ4dc1TCzYBfsXrsMLiLB2osdJmrrx+1H+rjTwhZUsrB1vyBHq5H7k4jVWJY\\\\\\\/pwVyxxkvfub7zqDNqB97OUcLWANxqmmjgUfKha+8KYT8U+yBS0NASZaV60j2lfXbFzx8fRhSqVkLT5DXR16wsZcZTpWcLuTd\\\\\\\/gil2vLCaruCb9VI0ZiNjPgr3EQD5XJOudzhB7iLROXee+cnoz1j+cNQ0GPoyvFIQ\\\\\\\/zpdiLr3vmosXwlhHDch4LOdeBw9FAAsBQdk6axVNFsSUbVzHDRiYQUOJxx4kc8bSGQkj2KIIRqwuoCjo8icULT8QoomzlVjYdDo5Kn2N\\\\\\\/eZ6FlnrLyk3Knr1x89hVeixEbBbKnyYJU1G0WslMV0bnusiQifv3PWhyfKv6508WUowJXq53Ag+hHwZCtdhxmIb546I+rTlXjnFYuU02FR8yfOdysK4+1ifETotsBxzxtGLEV2bS87GEZgf9bgCiVyPyAveP3iuIF9pyG3llN\\\\\\\/8V7kfnu9fOwUg7euf7SS5gjPw0hHBaxTwyxnarrGAuJQDrtDIgHBwCvutabhciXZ2vy0f7XmpsiIDDZ\\\\\\\/izeVeHUKplkN2tujqgxFI0ueX8ash3uOMwIpCbuUIOIcbh06ntkgu5ZM1ujEeh2Tu5BX7shg9fZ7gveupuOYT5Jy\\\\\\\/\\\\\\\/aC7zueM+xReoDmEcx8CLtcQbEZpeOmYuYbz9xdPYH+V91n323MS5iuNo1F4MaKGsDGmlhGkDJk1UY8g4KAJ23GrJzagWdZ7Bmcy1JnvApSVoXTrnHWtK4ZVcLqwPOkur+xlpg+edsJIE8xPkMUW8gkC0BOKslViKtqq96KZnt6gvXLyy2DA9cbqzmBlBMlhQ2IPDv8Heo\\\\\\\/pVarGTfMipJVJwgqzMHpYHXCnyjuE9tzFcSN6glsh+MrVd1kxK5Q7uN3bkqtz1W9j+0ZOoRq0CqNNc5AxcdS00Dad6a4PuY0QTlhfwvLkc7XcjZydlwTcLMrz5lZPy7OxHxrq83p\\\\\\\/ZJraNTK1NIIkYj3qoeiVRuc71aHd6Mxz8R5aUKHVtIgFj2fc\\\\\\\/e\\\\\\\/2mbtvdNFCGgab\\\\\\\/7fKQZanfViJzdC60pI9z91S7k5i+vTbFUBZFIHqhwkN6Rrqo1L3DwegGZ12XrR1UEBFZgwMs65HgCMMQYM9+Aox1lKxn1YCdPJMn0QzA8dnQDYCw5tG\\\\\\\/5zEsCZQxVDni3oAd5r5IvD5IV23YAD6o+hP4Bu46bdRYtybSvZR7qxxXn4AfsdhdknuGG+8fO+gJZTdw75IhEQe\\\\\\\/bsjXu4o25qjEtrxiEF\\\\\\\/+7m0CVV\\\\\\\/Q0rHPRvMj2jvDapuAoKgn6KjkrHLgWClxIRpOPZrLCNcyxKBltEvGCFc+yiJ4UoUi\\\\\\\/ZSi9RWF8b7H3f5i2CXtjrB6fWrqYmjmqlqwt7mzpzaSb7HJx\\\\\\\/69v1qduBjOKjCu8fqtb6g\\\\\\\/RPDxMk772U3XBml4dyQedcAEyYUHF04B4fVuv43O3KC8t7z0USDoiGLisw\\\\\\\/rUEEWl\\\\\\\/q1nlM8FxdQFJaL7pVVlxjuPbVlT2nYECM6lGUcy9k7Fvam4gVZbNGHqTP+Va7MU64IUeD6+DA3Ko4fPX8sA3CBqKYfxJGQxJVTvzNSOWq6p\\\\\\\/ADQPc9xxrVGjhCgcFr4CC9jKqRx72B16CSdr9Wsw5jTb6SfMX0itGs50SgPVdM8j8s+f6D6cYQ+pJW3EszXmyB3tmVUM9efWfAb6f4u4lPznGwzVl1teuknq9dTMLQNeF2n+K4QzCdX+cpuapqFzElHwYNiS\\\\\\\/Fa9wkNwBl0erxx94DqkE8gbEGiXqw4xkxiD2EFlS0Y9PQp6QonawUHxPzmcVHR0dV6aFo\\\\\\\/g91\\\\\\\/kQOCcIEQ7hEgHB\\\\\\\/g6rOEdZFe+\\\\\\\/+6jeO9ZQzERtFAMrmbF7\\\\\\\/IpgWP7p93YiY\\\\\\\/GABYdtAjSgnTeD+uUOjIlRpowcT9\\\\\\\/3ggOBwLLoNaKA+kCEzObboWTqT0cAxStKdB3sPefX+ubBCTQzK53rjrtNbjauIcaiPr2OtjChUgWQM6ujWE1nPZ+yBTatOcAI51XVIY4F4kjUCroItNvItAMPXb2tsHRtXib+cKQ20cbz6ZaWWQN6DOpYbX3+PQFHSz7+NI6fO6pTN14jTxnNO2yrQwLuORj1Q0U0foBxneYkWKp4ThfYZPS3xQ2WQdPi6eDJaE0OVLdvaarkTkpjsgues0xY6jCAgIbvq6brVA3pislf8nUjJFSMSjrZb4TEMTSfc+HNzNWISVOrT9iwRdgJ5AUXQhLi\\\\\\\/PiYKg\\\\\\\/y3wdB1aiKZLvDBn+Omx2kSkHG2Vpnxkdzx7iwWd9\\\\\\\/gI4Zb+E6XPFltqv+vYcOjJrnYZD+49CtSQejTtT92wVJHaSOW4wqVEraVXWokqwE58LTU1oHrM9JSW9g3WU2fpqKbrpOJUMA5Y\\\\\\\/deC9DDoqBpQEb\\\\\\\/swEOz1r75WJMNnwhDYeC9ERbkXi1ykHXer8FI\\\\\\\/ioLJvr\\\\\\\/nOff\\\\\\\/DNQHW5X8yxW+7gI4Y0fdAcvfrCxHQ9a0jt11ADK6xhYd3aRTUrr+iN\\\\\\\/Js50eRGFoUbLGWGHrh7E2HmuPazvUZzNzktUjLVjvo70d93EYLuEjYHOP0tc\\\\\\\/Lp8OQ9F3cnGgr7jGgF\\\\\\\/CWDSwSBmR5jAy7PHq3jPbs2bO3tvEx+o9y6Vb88sRQHayYtwAbVtY1ezt258NkXeEjIIhwAYZNojQQONvGU8ur2YxHsSv+MjrOX7rP2yu6LmdLkzXnnZjMSpgfkfqsPTNF6AhTmWHj3H5Nu4FmdVSZFfhgRvgi\\\\\\\/C82QaluP+XyKcqfW9H8INRdbvZr+IoHqBEip1Sp0ac+nvMwKHYZpzlzS+0uef1HRHU42g1N+eAt8KlIOUzqmgRsl\\\\\\\/1L24oPKOOioO6BvH84Ph8ai8aUoszXWAPK48v\\\\\\\/x+jpEzhBU6jh+PJiOdRc5upF0AkY2N9ExAeno5y8JHJzlBGmIkdP+6li0Yp5OqjGFJuZPsHGgEABFHFqm\\\\\\\/DWSY\\\\\\\/J66wQj2ANY87G3o7a2p93+qkewBc0qJ4uMFgQfMC\\\\\\\/ajDCMxbkIHXgYcXyStNyadkmbvehrNSW3SA5fZvbdWDJ3N3OTer7QjJXwLNhs4UeVG3znHER+GHvpPthr5dlWLahedzzz6cDiFCt6Vgdn\\\\\\\/uhho4BeNqrKYCtvcJkVXA63tymsBRjaACe1jxtmlnWUp+FirUgukjlZ9DvPvtW0oB+YuL+GR76G8dSecbU17FJVCWwkftAKdMDKvocfBBM97IkxPbJh2LSgh5oh86wQ7iJg4tGxCzn22IBvVxH4r0YTX8AUd9NbeZ+fE8JzKehT8nWE1JJtat2DMaLf8ghRe2hMwpO7MijuAjWUs7cpSj8n3Ped9zgkTjcQlh2ozTXiPjMS+F4kdQ9Hm9W2yIStVkwCs2qRuylmpFpZbFl+1zX8q70X3SVKbHB5tRsJywurOM\\\\\\\/t\\\\\\\/rPSyFxdG90GQYRzo2HdEhYR6bmlHflmQUjy40CzfFwMJINMTrLvVBPcC\\\\\\\/H6naKqhUEFRZmMAbYzFQ7tXBjyyoIWg3qW7VYVdUYQTTNbzMVd8jaGffmgA0oOETupRV9g+v7kCk7BL8xyo9UVB98ioSh4I6Pr+zrPm6XOmHL637rgju2J9Y1wTu4+gN9Yx6GF6Osm1\\\\\\\/46kYkV9C+JPVpHdSliE2XjfbWBet7Garma+I\\\\\\\/N\\\\\\\/vrU9XnZax8Bh8LY1tt8ulRk+KFGrcsxx4KzVqy5ju0oe0CrgjD8coBzBAci8U6ck1lFpYMhz1MIcAuJF1YSFs2YvOQyDU3goWNr9ypMXwa3Yk8sjwLD2XINnVAWhg3CdzoEjnNKSA66Z3Egyv3tnqB0kC+H\\\\\\\/hZT3mL52C0tGoPc98nRgyiXncqnJuFI0BRopbzi0JBvZmJc5KxmPn6m8UGhPixEk8hlt5ZVhFkCr\\\\\\\/\\\\\\\/XfRx+Jb\\\\\\\/JfWBxpfW4A84Dv1i8PqUw8Q7l+UksD8y3QSEx+Ps0iasrBNN4aBZHfGZQEKcFBhUMn+55M\\\\\\\/ecmbSnSRMxz\\\\\\\/dAzEduzJ3Fp6+eyMoEe5lVzgCvr8\\\\\\\/bXbV+JzshEIUfpUCEmxo3pWek\\\\\\\/xefFdgx71R+xPPvxh+diStTP10xghbaChFWdMZ4MGBTt8bHiaAeYLD2P2eT3gXCFI8yWT1JZP8oaw1Azqv9\\\\\\\/Gm77BGxSAUwm+BOGhpF5\\\\\\\/Dxb1\\\\\\\/UVF3LqGqJ2TFd4ghR24Lvl++PYS2MiG3buj0sET8D\\\\\\\/5lu2vccrdiqgKwMf4PnlZBJRi3asROg2jewe7uvG5dnHr9\\\\\\\/30C2\\\\\\\/VwqnjGTkeI+1VYdzy80qPWerqUN2VvNQ2kIU0JVT7b6QWfim3vsdXW44Ynog6w8GL4D0tAfAl62i8SG2k7DeIR4XyhtIOX8k2DGwyHV6mOS7vcWxIsDi0iE6ci\\\\\\\/2xuxMmyyJT1chIUT76fKH0ecq6Y2owojPcBcRatJ9bBtHtb8r8PeJUrQarfd91XVmcN3vtmPm+Cakg6aZqnUyMPazPIEANfnaPIxlfaW\\\\\\\/huWgmzqoTo9zs\\\\\\\/XFE155hgBgub5btHqjejuoGjt619TfOR7AoxG0Y\\\\\\\/0EytT9bQ8f6rkjY009mJ6Atal0sEOYccU7BaEoYyHvo9ArOSu2L\\\\\\\/p0LVJCwdEZUS+yOArH8KnZi\\\\\\\/X2AgOKeLQFWm2bpCtHrE7ZPHyhx\\\\\\\/tq3CZ5eBdKuQYNAN\\\\\\\/gtWYEOtNCS3CwuBgjNUKtOpSAO9s8hhNgHReb6WMfjQ5c0epJaQyimSY9mJfEksKnAXg9V\\\\\\\/Y4tbWfOQh5IS5gViWNhNOKRs8Vyzc6KngtQKTnlXiCupfOJW1Of0YR9F\\\\\\\/0prdvmOg1a+FmL2BhKANNA3vbkeNoQJnk2KkXcveLojag3SwFmgNSu1iJQhoYtxYmpKMdvBMxCSjuQzowM\\\\\\\/fve+rnDoVxQFz1ELNN5jJT8IVT6qQjshjoYXdwXS3MhSAE3PFyZf3B\\\\\\\/oZF\\\\\\\/26Q\\\\\\\/2\\\\\\\/afO4zRmXiEIDaPPf5WelNyG2Tkxowqo0f11gD9gUoSgaNupNmZ\\\\\\\/JgpE\\\\\\\/FjBDjGXRAxx4zIxj0D3t1ZcDiviSwlh+tv9jl7PLCrTjluzlaP74PYPofXRbLJIS5hoiSAFl50rDqqzR9vHIkOADWGQF3rG8VnwL5GZMbTc1fx9+g7h2678c5yh5o2d1s2YHx+JJBoohQ50OHOX9Ow==\\\u0022,\\\u0022iv\\\u0022:\\\u00226rdaycYoVdRFKwcJ175+Gw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022fd91feaa2ee96ffaba078b2d623d4a897141dda8ed7b07f8813740f03b8aa4a3\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022zH6D+6JmupAT15CshoZF5rCzCvvVD8H6LLFBRYuSFcm2sq0XWa6pLUKgR52cziy4VvXA28Mq082yxAbLZ+KMhzxSq\\\\\\\/q4B6f5dSzIDN3fAAolxHIuMlgpaX4f9nQ59N6ThZVd2LcF+KFbp47pyrzP4XMbAV\\\\\\\/5tVFx7Db2Bek51BlE6usK5l7kYfw+18ZE5x0AXWeDk2ax6liNnZfP3czXeIdCZmi88HuiWpKXPjnA6hDWb7lvQFzfmRbKsyqvJqlLuA1g5zkIYHzC95PG63SWmgP7c4o2uNxyYsFWBQfxNR34VYKKDx6s\\\\\\\/UTLxoOFhyTgGWBed9uu6rSpMlivlkftwYI6r7JuilYrBe6BYldssS3\\\\\\\/KoHZxHTZt4fMgmowfD6Vrdb0IubXU73EPRsfLrX4V1pfGgbGSlgAOsgh+yYNutGZmuCeBaFruqJof5BM0PPq6YS44OjANVIWgqntdsGTRzF20K\\\\\\\/VHAmFhWHAcLvOEcRLAyTaNdbpSTQ5rFQ0S+MiCGdlCmmADLVUuTVhqEWPh18+O8ZVxFKRt8L3KkkZBldhsuMdU\\\\\\\/M4AUQ3hOoek89k4D3d2wxaOjKNFTs32j6KCer\\\\\\\/Qn1RApUMgRNwhPzsGeQCTpt\\\\\\\/0U+KREOcXbgIFNU9jNMmT7KjYd2kC6DcgRJy1lrNZREv9vA8EB1gd5tuIv6ahMhHyiYfzoDK\\\\\\\/Bk9WOU98s+E+NlrLP0G5YZKMi3AxxJqmdDWV77VxZmwWQS0JZP6sGlq+0FeBzkBlCnyhOiNJu4Y1EWJb\\\\\\\/OD\\\\\\\/JpurqyfwlHzVFtSxUC+01CJNe4RKpmSOOr8J8nuWIV09m8PC+3T\\\\\\\/0U3WPYl7O5vyYR9llEkKgU02vinSHEnnXkbmjEU+Q+EwOKU2YUFM22fFqBimMlsuz8rEJwEIrgjd1XU6CPfrelRxDBQdkHaub5X7awvo3zdbM54JA2T0GB2R+C\\\\\\\/J6UKRkWc0cv3gvLvpKuqCxO84YUq7v3eADCLu45ZU8FmP0HQgMX0u75uLCD7APEhnVOCYzSODRlEBVSDlKqfyQCAHQ2Mf8GdKsnzAlszhNnctbfld0c38cWMmnKM1mB8VD0Dvb+aRKrZNi\\\\\\\/SXBlZbdSBXJNlE\\\\\\\/NCLBAty3va3pVwSVxPaI4TtGLsKWNfo0arz56ZNEcuBN29wBB+NPdwEcAXq60NUZcpRtZXrLLIZMqhHbVWw0xpHRbFM\\\\\\\/ghwBqh8xzbm+4MYGF72z+3z4E9QtRyTxy+aEMI\\\\\\\/rh6FIgDo0ixL6ep9gOWxqIWgDrB5fiIr6TpHT2sr9rYZmyUUhDUPYaYV3wFdenyoD4LyONHbWrtEDqSNQYDOJ2MOcsZ6xzDT1fFskY3dsB142bidAy52NJGqSRVcI9xz+M78QESn3zEHaVyo95GDol8sSANnNqrsINSeZ2fW5TNlv3iC+LIOKP7TKI3JjR6\\\\\\\/qTL4K8tcYzd1Y1uWqQw3reUZ1r6wheiT9T4LdJoIVsaKWQjMv50f05ogunoYkA8QYC2JJWNAhminncRiUrb6\\\\\\\/6XfBFgYjLrYN5KtZrx5c5X\\\\\\\/jQs+C2a6QAgD8yHhDJOgiFp8hKDaf3\\\\\\\/zTxTkx9mXNp4LVSMRXhmyHmofr1BsbB8O9ddDRDPcAJ+ZcErnalS7LGa4HSLurq8Sxl9LtnoFUqFGjnNlNzezKYxDy3d5HeMlS\\\\\\\/hw6OjSW9QKQgYB0AKQmFPeJO6lc5Bx+1u3ObjYfiZzXZwZdbSBjeIp9uJQccm7yOQbeaOe6q3f\\\\\\\/3Y7GlyJaC0hYlxtzE\\\\\\\/qC6yEV04ixKkblBuapeE5IiuesuCrVCff55HFERWsUlOHMyElAwlZgKrKpVD3d5LxicrbK6FIieXXAoNwof4Z\\\\\\\/ZWqUY++RznBL1gonwm1SYrbJbcq5XszezaEcgPvF5zZNL0+uGbxZ2ccg+6Htnbd7IRNplyB7EoF4cr1gpVH4V1lPjcwuZRZNmR677AvCjlYp9B\\\\\\\/mOkFwJnGzL9BnDE4gmx1tm78SFM2b4L3ELNdmJ\\\\\\\/RQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022a\\\\\\\/LrR5AABGthxXRmCQIgwA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00221b66da846c9280ff14fcb09ece5abb78c06a5a2bdf1e2f4521a981dd26a69d57\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460153 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/10/2007" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-_3coKGGdALXxZenUaHBPD1iD3_WZK8nhk0wtCbB6R4g" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/10/2007" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Qj1RxQP3yLyBMtmlZAiOZBWcdMim2FBYjqkJjuiUMNM" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/10/1999" title="Fingerprinting Soybean Germplasm and Its Utility in Genomic Research" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/10/2021" title="High-Density Genotypes of Inbred Mouse Strains: Improved Power and Precision of Association Mapping" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460153" id="node460153" data-pisa="ggg;5/10/2007" data-pisa-master="ggg;g3.115.019570" data-apath="/ggg/5/10/2007.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Second-Generation Linkage Maps for the Pacific Oyster <em>Crassostrea gigas</em> Reveal Errors in Assembly of Genome Scaffolds</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-3995-646X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Dennis Hedgecock</span>, <span class="highwire-citation-author" data-delta="1">Grace Shin</span>, <span class="highwire-citation-author" data-delta="2">Andrew Y. Gracey</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-5404-5218" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>David Van Den Berg</span> and <span class="highwire-citation-author" data-delta="4">Manoj P. Samanta</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2007-2019;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.019570 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460153" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Dennis Hedgecock </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biological Sciences, University of Southern California, Los Angeles, California 90089-0371 </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dennis%2BHedgecock%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hedgecock%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADennis%2BHedgecock%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-3995-646X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Dennis Hedgecock</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:dhedge@usc.edu" class="" data-icon-position="" data-hide-link-title="0">dhedge@usc.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Grace Shin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biological Sciences, University of Southern California, Los Angeles, California 90089-0371 </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Grace%2BShin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shin%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGrace%2BShin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Andrew Y. Gracey </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biological Sciences, University of Southern California, Los Angeles, California 90089-0371 </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrew%2BY.%2BGracey%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gracey%20AY&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAndrew%2BY.%2BGracey%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">David Van Den Berg </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Preventive Medicine, Keck School of Medicine, University of Southern California, Los Angeles, California 90089-9601 </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BVan%2BDen%2BBerg%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Den%20Berg%20DV&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADavid%2BVan%2BDen%2BBerg%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-5404-5218" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for David Van Den Berg</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Manoj P. Samanta </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Systemix Institute, Redmond, Washington 98053</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Manoj%2BP.%2BSamanta%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Samanta%20MP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AManoj%2BP.%2BSamanta%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/5/10/2007" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460153" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460153/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/10/2007.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460153" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460153/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/10/2007.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460153" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460153/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/10/2007.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460153" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460153/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/5/10/2007.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460153" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460153/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">The Pacific oyster <em>Crassostrea gigas</em>, a widely cultivated marine bivalve mollusc, is becoming a genetically and genomically enabled model for highly fecund marine metazoans with complex life-histories. A genome sequence is available for the Pacific oyster, as are first-generation, low-density, linkage and gene-centromere maps mostly constructed from microsatellite DNA markers. Here, higher density, second-generation, linkage maps are constructed from more than 1100 coding (exonic) single-nucleotide polymorphisms (SNPs), as well as 66 previously mapped microsatellite DNA markers, all typed in five families of Pacific oysters (nearly 172,000 genotypes). The map comprises 10 linkage groups, as expected, has an average total length of 588 cM, an average marker-spacing of 1.0 cM, and covers 86% of a genome estimated to be 616 cM. All but seven of the mapped SNPs map to 618 genome scaffolds; 260 scaffolds contain two or more mapped SNPs, but for 100 of these scaffolds (38.5%), the contained SNPs map to different linkage groups, suggesting widespread errors in scaffold assemblies. The 100 misassembled scaffolds are significantly longer than those that map to a single linkage group. On the genetic maps, marker orders and intermarker distances vary across families and mapping methods, owing to an abundance of markers segregating from only one parent, to widespread distortions of segregation ratios caused by early mortality, as previously observed for oysters, and to genotyping errors. Maps made from framework markers provide stronger support for marker orders and reasonable map lengths and are used to produce a consensus high-density linkage map containing 656 markers.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/illumina-goldengate-bead-assays" class="hw-term hw-article-keyword hw-article-keyword-illumina-goldengate-bead-assays" rel="nofollow">Illumina GoldenGate bead assays</a></li><li class="kwd"><a href="/keyword/regression-maps" class="hw-term hw-article-keyword hw-article-keyword-regression-maps" rel="nofollow">regression maps</a></li><li class="kwd"><a href="/keyword/maximum-likelihood-maps" class="hw-term hw-article-keyword hw-article-keyword-maximum-likelihood-maps" rel="nofollow">maximum likelihood maps</a></li><li class="kwd"><a href="/keyword/distortion-segregation-ratios" class="hw-term hw-article-keyword hw-article-keyword-distortion-of-segregation-ratios" rel="nofollow">distortion of segregation ratios</a></li><li class="kwd"><a href="/keyword/genotyping-error" class="hw-term hw-article-keyword hw-article-keyword-genotyping-error" rel="nofollow">genotyping error</a></li></ul><p id="p-3">The Pacific oyster <em>Crassostrea gigas</em> is one of the most widely cultivated marine species, having been introduced for this purpose from Asia to all continents but Antarctica (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Mann 1979</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Food and Agriculture Organization 2015</a>). Global annual production is uncertain, owing to taxonomic confusion in reports from China and other countries (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Wang <em>et al.</em> 2008</a>), but is conservatively estimated as 555,914 metric tons in 2013 (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Food and Agriculture Organization 2015</a>), down from 4.6 million metric tons of cupped oysters grouped under the Pacific oyster moniker in 2006. Because of its commercial importance, a number of breeding programs have been initiated over the years (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Hershberger <em>et al.</em> 1984</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Langdon <em>et al.</em> 2003</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Hedgecock and Davis 2007</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Dégremont <em>et al.</em> 2010</a>), substantial genetic and genomic resources have been developed [genetic lines; expressed sequence tag (EST) collections; transcriptomes; bacterial artificial chromosome libraries; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Hedgecock <em>et al.</em> 2005</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Curole and Hedgecock 2007</a>], and recently the genome has been sequenced (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Zhang <em>et al.</em> 2012</a>). The Pacific oyster is also becoming a model species for research on environmental resilience (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Applebaum <em>et al.</em> 2014</a>).</p><p id="p-4">Linkage maps are essential tools useful for mapping quantitative-trait loci (QTL; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Lander and Botstein 1989</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Doerge 2002</a>), marker-assisted selection, positional cloning, and genome assembly. Yet, construction of a single, high-density consensus linkage map for a species, particularly for one such as the Pacific oyster, which has a high load of early lethal or highly deleterious mutations (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Launey and Hedgecock 2001</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Plough and Hedgecock 2011</a>), is a challenging goal.</p><p id="p-5">Previously published linkage maps for the Pacific oyster were based on microsatellite DNA markers (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Hubert and Hedgecock 2004</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Hubert <em>et al.</em> 2009</a>; <a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Plough and Hedgecock 2011</a>), on dominant amplified fragment-length polymorphism (AFLP) markers or combinations of AFLP and microsatellite markers (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Li and Guo 2004</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Guo <em>et al.</em> 2012</a>), and more recently (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Sauvage <em>et al.</em> 2010</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Zhong <em>et al.</em> 2014</a>) on microsatellite markers together with single-nucleotide polymorphisms (SNPs). Although adequate for mapping viability QTL (<a id="xref-ref-39-3" class="xref-bibr" href="#ref-39">Plough and Hedgecock 2011</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Plough 2012</a>), these first-generation maps have rather low marker densities. Here, higher density, second-generation linkage maps are constructed for the Pacific oyster, based largely on assays of a fixed set of 1536 coding (exonic), SNPs in five mapping families.</p><p id="p-6">Variability in marker orders and intermarker distances among individual family maps and between different mapping methods presents formidable challenges to the construction of a consensus linkage map for this species. Harmonizing results from regression (RG) and maximum likelihood (ML) mapping methods, we identify three factors behind uncertainties in marker orders and inflated map distances: (i) markers segregating from only one parent, (ii) markers linked to chromosomal regions impacted by strong, early viability selection, and (iii) genotyping errors. In the end, a single consensus map, constructed from the most reliable set of individual framework maps, is achieved. SNP linkage information has an unexpected bearing on the current genome assembly, insofar as it implies that a large proportion of genome scaffolds may be incorrectly assembled.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Mapping families</h3><p id="p-7">Five full-sib families of Pacific oysters were used for linkage mapping. Two were F<sub>2</sub> families, 2 × 10 (n = 90) and 51 × 35 (n = 108), each derived from an intercross of full-sib F<sub>1</sub> hybrids, which, in turn, were produced by crosses of partially inbred lines (<em>i.e.</em>, 2, 10; 51, 35; <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Hedgecock and Davis 2007</a>). Three other families, F12, F20, and F45 (each, n = 46), were produced by crosses between adult oysters taken in 2006 from the wild population in Dabob Bay, Washington (L. V. Plough, G. Shin, and D. Hedgecock, unpublished data); these random-bred, full-sib families were the G<sub>0</sub> foundation stock for development of inbred lines (<a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">Hedgecock and Davis 2007</a>).</p></div><div id="sec-3" class="subsection"><h3>EST library sequencing and transcriptome assembly</h3><p id="p-8">To construct libraries of expressed DNA sequences, RNA was extracted from 2-yr-old oysters, from G<sub>3</sub> inbred lines 51 and 35, after we subjected small groups to different stressful conditions. The pedigree of all oysters was confirmed by typing microsatellite DNA markers on the inbred oysters and tissue samples from their parents (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Curole and Hedgecock 2007</a>). RNA was pooled within the inbred lines and made into coded cDNA libraries. Four additional larval cDNA libraries, two from 2-hr to 12-hr-old larvae and two from 15-hr to 30-hr-old larvae, were also constructed.</p><p id="p-9">These libraries were Sanger-sequenced by the U.S. DOE Joint Genome Institute (JGI; larval libraries CBSI, CBSN, CBSO, and CBSP; and adult libraries CCBN, CCTP, and CCTS; GenBank accessions <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=HS109651&amp;atom=%2Fggg%2F5%2F10%2F2007.atom">HS109651</a>-<a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=HS248985&amp;atom=%2Fggg%2F5%2F10%2F2007.atom">HS248985</a>) . The adult libraries contributed 126,331 (90.7%) of the 139,335 EST sequences obtained from these libraries. Additional sequence was generated from a putative, nonredundant subset of 11,904 cDNA clones, which were pooled, their inserts fragmented,and sequenced by Illumina Genome Analyzer methods (SRA accession SRR2119182). Further sequence information came from two other larval cDNA libraries, CFPP and CFPS, which were made, respectively, from 6-d- and 18-d-old larvae of a reciprocal hybrid between inbred lines 35 and 51; these libraries were sequenced by 454 GS FLX Titanium methods (GenBank accessions SRX032364 and SRX032365). Sequences from all libraries were assembled, using miraEST (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Chevreux <em>et al.</em> 2004</a>), into a <em>C. gigas</em> transcriptome comprising 52,095 unigenes.</p></div><div id="sec-4" class="subsection"><h3>SNP discovery, marker development, and genotyping</h3><p id="p-10">The JGI ESTs (145,600 sequences) were compared with the transcriptome assembly, using BLAST (blastn, cutoff 1e-50) and the best hit for each EST was aligned with the EST sequence using CLUSTAL. Alignments of cDNA sequences from the two inbred lines 51 and 35 were combined to generate a SAM file for all ESTs. Separately, reads from two additional Illumina RNA-Seq libraries (SRP061799) were compared with the assembled transcriptome, using BWA (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Li and Durbin 2009</a>; <a href="http://bio-bwa.sourceforge.net/">http://bio-bwa.sourceforge.net/</a>). The three SAM output files were combined, using the pileup command in SAMtools (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Li <em>et al.</em> 2009</a>; <a href="http://www.htslib.org/">http://www.htslib.org/</a>) to find 126,392 sites with potential variants (whether true SNPs or read errors). Candidate SNPs were sorted according to the amount of supporting sequence evidence and whether they were segregating within or between the two inbred lines; each alternative allele had to be supported by two or more sequences. Additionally, the unigene contigs were mapped on a draft version of the oyster genome, so that intron-exon boundaries could be identified. The list of high-quality SNPs was further refined to remove candidates located within 50 nucleotides of an intron-exon boundary, so that a 100-nt GoldenGate assay probe centered on the SNP site (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Shen <em>et al.</em> 2005</a>) would be contained entirely within a single exon. A total of 4122, 100-nt probes were selected, after these steps, for design of GoldenGate probes.</p><p id="p-11">From the list of potential probes, we chose the top 1536 SNPs for the final GoldenGate bead array (<a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Shen <em>et al.</em> 2005</a>), which was used to genotype four 96-well plates containing the five mapping families. Subsequently, 27 SNPs were found to have duplicates in the array, 6 SNPs were present in triplicate, and 1 SNP was present in quadruplicate; these repeats were removed from the mapping data sets and the subsequent tally of SNPs and genome scaffolds mapped. Although present in earlier assemblies of the genome, which enabled identification of exon boundaries, seven of the assayed SNPs were not present in the final genome assembly; these retain the “USC” reference numbers from the transcriptome assembly. Otherwise, SNPs were assigned names based on the genome scaffold number and nucleotide position (see <em>Genomic locations and annotations of mapped, coding SNPs</em>).</p><p id="p-12">To determine genotyping error in the GoldenGate bead assay, nine individual samples were run in duplicate on plates processed at the same time. Discounting for missing data and duplicated markers, an average of 1437 SNPs were compared in duplicate samples, a total of 12,937 genotypes; no discrepancies between duplicated genotypes were found.</p><p id="p-13">In addition to SNPs assayed by the GoldenGate assay, we genotyped 66 microsatellite DNA markers that have been mapped in previous studies (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Hubert and Hedgecock 2004</a>, <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Hubert <em>et al.</em> 2009</a>; <a id="xref-ref-39-4" class="xref-bibr" href="#ref-39">Plough and Hedgecock 2011</a>, <a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Plough 2012</a>). We also typed 17 SNPs in families F45 and 51 × 35, which were identified early by the SNP discovery pipeline but genotyped on a Sequenome platform (L. V. Plough, G. Shin, and D. Hedgecock, unpublished data). Across families and markers, a total of 179,263 genotypes were determined in this study and contributed to the linkage maps to be described. Complete genotype data sets for the five families are given in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019851/-/DC1/019570SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019570/-/DC1/TableS1.xlsx">Table S1</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019570/-/DC1/TableS2.xlsx">Table S2</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019570/-/DC1/TableS3.xlsx">Table S3</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019570/-/DC1/TableS4.xlsx">Table S4</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019570/-/DC1/TableS5.xlsx">Table S5</a>.</p></div><div id="sec-5" class="subsection"><h3>Linkage analyses</h3><p id="p-14">Linkage mapping calculations were made with JoinMap 4.1 (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Van Ooijen 2006</a>), using the cross-pollinated coding of genotypes to accommodate the multiple mating types observed in the full-sib families (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>; here, but not in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019570/-/DC1/TableS1.xlsx">Table S1</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019570/-/DC1/TableS2.xlsx">Table S2</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019570/-/DC1/TableS3.xlsx">Table S3</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019570/-/DC1/TableS4.xlsx">Table S4</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019570/-/DC1/TableS5.xlsx">Table S5</a>, coding of markers segregating from only one parent is reversed from JoinMap convention, to reflect our long-standing practice of writing controlled crosses as ♂×♀). Markers missing data for more than 10% of individuals in a family were excluded. Initial groupings of markers were based on independence log of the odds (LOD) tests in steps from a LOD of 1.0 (independence <em>P</em> threshold, 0.001; recombination threshold, 0.25) to a LOD of 10.0 (independence <em>P</em> threshold, 0.0001; recombination threshold, 0.05). The independence LOD is less likely than the linkage LOD test to find spurious linkage, when segregation distortion, such as that observed for oysters (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bierne <em>et al.</em> 1998</a>; <a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Launey and Hedgecock 2001</a>; <a id="xref-ref-39-5" class="xref-bibr" href="#ref-39">Plough and Hedgecock 2011</a>), is present (<a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Van Ooijen 2006</a>). The final grouping LOD is the maximum threshold value passed by all markers in at least one test of independence with other markers in the group. Previously mapped microsatellite markers (<a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">Hubert and Hedgecock 2004</a>; <a id="xref-ref-39-6" class="xref-bibr" href="#ref-39">Plough and Hedgecock 2011</a>) were used to associate the resulting groupings with the 10 identified linkage groups (LG). In each family, sets of markers with identical genotypes were identified in subsequent mapping steps. Since identical markers map to the same location, we used only one marker from each set of identical markers, to increase the efficiency of linkage calculations, and we subsequently positioned initially excluded identical markers at the locus where their representative mapped.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466783/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466783/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Distribution of markers by mating types (♂×♀) in five mapping families of the Pacific oyster <em>Crassostrea gigas</em></span><div class="sb-div caption-clear"></div></div></div><p id="p-15">As observed previously for the Pacific oyster (<a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">Launey and Hedgecock 2001</a>; <a id="xref-ref-39-7" class="xref-bibr" href="#ref-39">Plough and Hedgecock 2011</a>; <a id="xref-ref-38-3" class="xref-bibr" href="#ref-38">Plough 2012</a>), we find widespread distortions of Mendelian segregation ratios in most linkage groups, as reflected by highly significant χ<sup>2</sup> goodness-of-fit tests. The extent of such distortions on maps is summarized by the proportions of markers on a LG that show departures from expected Mendelian segregation ratios at a nominal significance threshold of <em>α</em> ≤ 0.01. A two-way analysis of variance on arcsine square root transformations of these proportions is used to assess variance among families and linkage groups.</p><p id="p-16">Both RG (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Stam 1993</a>) and ML (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Jansen <em>et al.</em> 2001</a>) mapping methods, as implemented in JoinMap 4.1, were used to construct linkage maps. Initial RG linkage maps were made with all markers, using default settings (maximum recombination frequency of 0.4, minimum LOD of 1.0, goodness-of-fit jump threshold for removing loci of 5.0), Kosambi’s mapping function, three-rounds of fitting (a second round attempts to add loci removed in the first round, with a third round forcing all markers onto the map), and a ripple performed after each marker addition. Kosambi’s mapping function is justified by evidence for crossover interference (<a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">Hubert <em>et al.</em> 2009</a>). Occasionally, insufficient linkage or failure to determine linkage phase with default settings required relaxation of thresholds. In three cases (LG 2 in 51 × 35 and LG 8 and LG 10 in F20, the latter two requiring relaxation of linkage thresholds), single markers that appeared to cause poor fit or excessive map lengths were removed before proceeding with mapping. Nearest-neighbor fit—the sum of the absolute values of the differences between observed pairwise and calculated map distances (in centiMorgans, cM) of each marker to its nearest informative neighbors on either side—is used as an indicator of the quality of the map order.</p><p id="p-17">The ML mapping method uses multipoint, ML-based algorithms to determine optimal intermarker distances and marker order and simulated annealing to provide statistical support for marker order. Distance in the ML method is calculated with the Haldane mapping function, which assumes no cross-over interference and may overestimate distances when interference is present. We used the JoinMap default spatial sampling thresholds (0.10, 0.05, 0.03, 0.02, and 0.01) and three rounds of optimization per sample. Default parameters for map-order optimization were used except that chain length was increased from 10,000 to 20,000. Parameters for multipoint estimation of recombination frequency were generally increased (length of burn-in chain, from 10,000 to 20,000; number of Monte Carlo EM cycles, from 4 to 10; chain length per EM cycles, from 1000 to 5000) to ensure convergence.</p><p id="p-18">We attempted to resolve discrepancies in map orders and marker distances between RG and ML maps by focusing, first, on biparentally segregating framework markers and then on a set of well-spaced anchor loci. When suitable congruence between RG and ML maps for anchor loci was achieved, we applied the resulting order as a fixed order to the mapping of framework markers and all markers. Maps of framework markers were used to construct consensus maps and to compare pairwise recombination rates between male and female parents.</p><p id="p-19">The length of the genome was estimated for each family by two methods. First, we calculated <em>s</em>, the average spacing of markers, as the sum of the lengths of the 10 linkage maps within each family divided by the total number of mapped intervals (<em>i.e.</em>, the total number of markers minus the number of linkage groups, 10). Genome length was then estimated as the sum of linkage group lengths plus 20<em>s</em>, to account for intervals beyond the most distal markers on both ends of the 10 linkage groups (following <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Fishman <em>et al.</em> 2001</a>). We also estimated genome length for each family as the sum of map lengths multiplied by (<em>m</em> + 1)/(<em>m</em> − 1), where <em>m</em> is the number of markers per linkage group (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Chakravarti <em>et al.</em> 1991</a>). Finally, for each estimate of map length, we estimated genome coverage as 1- <em>e</em><sup>-2</sup><em><sup>sn</sup></em><sup>/</sup><em><sup>L</sup></em>, where <em>s</em> is average spacing, <em>n</em> is total number of markers assigned to linkage groups, and <em>L</em> is the total map length (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bishop <em>et al.</em> 1983</a>).</p><p id="p-20">A consensus map was made, using MergeMap (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Wu <em>et al.</em> 2011</a>); owing to the explicit statistical support for marker order that the ML method provides, we merged family ML framework maps. Prior to merging, we removed poor-fitting markers from some family maps to improve agreement in map lengths and marker-orders between RG and ML maps. The G<sub>0</sub> families did not have enough bi-parentally segregating markers to provide framework maps for LG 9. In addition, we removed LG 2 and LG 7 maps for G<sub>0</sub> families <em>F20</em> and <em>F45</em> from the final consensus map, because they fit a preliminary merged map poorly; likewise, we removed <em>F12</em> and <em>F20</em> framework maps from the final consensus map for LG 10. Families were weighted in MergeMap, roughly by the number of individuals typed, such that the G<sub>0</sub> full-sib families were weighted 1.0 and the two F<sub>2</sub> families were weighted 2.0. The inverse Haldane mapping function was used to convert ML map units to recombination fractions, which were then converted into Kosambi map units prior to submitting to MergeMap. The consensus maps produced by MergeMap still appeared to have inflated distances, compared to the individual maps, so, after adjusting individual map origins to the appropriate consensus-map position, we performed linear regressions of individual maps onto consensus maps in order to adjust final distances on each consensus linkage map.</p></div><div id="sec-6" class="subsection"><h3>Genomic locations and annotations of mapped, coding SNPs</h3><p id="p-21">The SNP-containing sequences were aligned with the reference genome using BWA-MEM (H. Li; <a href="http://arxiv.org/abs/1303.3997">http://arxiv.org/abs/1303.3997</a>) and each SNP location was identified using a custom script. Loci were named (<em>e.g.</em>, 3-G98282A) according to the genome scaffold to which they aligned (3 in the example), followed by the nucleotide position within that scaffold (98,282 in the example); the position is preceded by the base in the reference genome (G) and followed by the substitution observed in that strand (A).</p></div><div id="sec-7" class="subsection"><h3>Data availability</h3><p id="p-22">Accession numbers to sequences used for SNP discovery are given in “EST library sequencing and transcriptome assembly” and in “SNP discovery, marker development and genotyping.” Supporting information at <a href="http://www.g3journal.org/content/suppl/2015/08/04/g3.115.019570.DC1">http://www.g3journal.org/content/suppl/2015/08/04/g3.115.019570.DC1</a> contains detailed descriptions and links to Excel <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019570/-/DC1/TableS1.xlsx">Table S1</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019570/-/DC1/TableS2.xlsx">Table S2</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019570/-/DC1/TableS3.xlsx">Table S3</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019570/-/DC1/TableS4.xlsx">Table S4</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019570/-/DC1/TableS5.xlsx">Table S5</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019570/-/DC1/TableS6.xlsx">Table S6</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019570/-/DC1/TableS7.xlsx">Table S7</a>, containing the genotype-input files for JoinMap, the mapping of SNPs to scaffolds and to linkage groups, and the consensus linkage map. <a id="xref-table-wrap-6-1" class="xref-table" href="#T6">Tables 6</a> and <a id="xref-table-wrap-7-1" class="xref-table" href="#T7">7</a> provide a roadmap to parameters used in sequential steps of the linkage mapping analyses.</p></div></div><div class="section results" id="sec-8"><h2 class="">Results</h2><div id="sec-9" class="subsection"><h3>Polymorphism of markers and preliminary assignment to LGs</h3><p id="p-23">Each of the mapped 1172 SNPs or microsatellite DNA markers can be classified into one of five possible parental mating types (♂×♀), <em>np</em> × <em>nn</em>, <em>ll</em> × <em>lm</em>, <em>hk</em> × <em>hk</em>, <em>ef</em> × <em>eg</em>, and <em>ab</em> × <em>cd</em>, depending on whether parent-pairs have two, three, or four alleles. The distribution of markers across these categories (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>) reflects the types of markers assayed and levels of marker polymorphism in the populations from which the parents were drawn. There is a preponderance of two-allele types (96.2%), as expected for the bulk of bi-allelic SNP markers. Across the three G<sub>0</sub> families, proportions of two-allele mating types are homogeneous (0.38 for <em>np</em> × <em>nn</em>, 0.40 for <em>ll</em> × <em>lm</em>, and 0.22 for <em>hk</em> × <em>hk</em>; χ<sup>2</sup> = 0.65, 4 df, <em>P</em> = 0.96), but the two F<sub>2</sub> families have many more <em>hk</em> × <em>hk</em> mating types, reflecting derivation from a cross of inbred grandparents (<em>i.e.</em>, <em>hh</em> × <em>kk</em>). The two F<sub>2</sub> families differ significantly from each other, however, with 51 × 35 having a much greater relative proportion of <em>hk</em> × <em>hk</em> mating types than <em>2 × 10</em> (χ<sup>2</sup> = 89.8, 2 df, <em>P</em> &lt; 0.0001); this difference is consistent with the initial selection of SNPs fixed for different alleles in the 51 <em>vs.</em> 35 inbred parent lines. Microsatellite DNA markers account for all of the <em>ab</em> × <em>cd</em> mating types and most of the <em>ef</em> × <em>eg</em> mating types. Some SNPs fall into the <em>ef</em> × <em>eg</em> category, owing to inference of a shared null allele (<em>i.e.</em>, apparent <em>aa</em> × <em>cc</em> mating types produce progeny types, <em>aa</em>, <em>cc</em>, <em>ac</em>, and blanks; such crosses were re-interpreted as <em>aØ</em> × <em>cØ</em>, where <em>Ø</em> is a non-amplifying null allele, yielding <em>aØ</em>, <em>cØ</em>, <em>ac</em>, and <em>ØØ</em> progeny). In keeping with their inbred history, the two F<sub>2</sub> families differ noticeably from the three, random-bred G<sub>0</sub> families, in having almost 100 fewer markers mapped (526 <em>vs.</em> 623) and many fewer <em>ab</em> × <em>cd</em> mating types (1.0 <em>vs.</em> 18.7, on average; <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>).</p><p id="p-24">Markers are initially grouped through pairwise <em>G</em>-tests of independent segregation. Summing across all five mapping families, a total 1085 coding SNPs are grouped, together with enough previously mapped microsatellite DNA markers, 66 in total, to establish consistency with previous linkage maps (<a id="xref-ref-23-4" class="xref-bibr" href="#ref-23">Hubert and Hedgecock 2004</a>; <a id="xref-ref-24-4" class="xref-bibr" href="#ref-24">Hubert <em>et al.</em> 2009</a>; <a id="xref-ref-39-8" class="xref-bibr" href="#ref-39">Plough and Hedgecock 2011</a>). Markers are provisionally mapped, using the regression mapping method, yielding 10 linkage groups in each family, as expected, with total lengths ranging from 508 to 690 cM and an average total map length of 588 cM (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). With a mean of 584 markers per map, including identical loci, these maps have an average marker spacing of 1.04 cM. The 50 linkage groups range in size from 28.7 to 198.4 cM, with an average of 59 cM. Only five linkage groups (one each for LGs 1, 3, 4, 7, and 8) are greater than 85 cM in length, and these appear to be outliers with respect to map lengths obtained in the other families (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>); we return to these excessively long linkage groups in a later section. Genome lengths are estimated for each family by two methods (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Fishman <em>et al.</em> 2001</a>; <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Chakravarti <em>et al.</em> 1991</a>) and range from 524 to 744 cM, with averages of 609 and 623 cM, respectively (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>). Despite variability in genome lengths among families, genome coverages are 85% or 86% in all cases (<a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2</a>).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466800/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466800/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Lengths and numbers of markers for initial regression maps, by linkage group, for five mapping families of the Pacific oyster <em>Crassostrea gigas</em></span><div class="sb-div caption-clear"></div></div></div><p id="p-32">Most SNPs in the GoldenGate assay (823 of 1085, 75.9%) are mapped in two or more families (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>), lending confidence to LG assignments; of these, only two (1255-G326792C and 43940-T130339C) map to different linkage groups in different families. Close inspection of the mapping support for these markers did not uncover any explanation, as it did in a dozen similar cases for which data transcription errors were discovered and corrected. These two markers are dropped from further linkage analyses.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466908/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466908/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Distribution of SNP markers over five mapping families of the Pacific oyster <em>Crassostrea gigas</em></span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h3>Mapping of genome scaffolds</h3><p id="p-34">All but seven of the 1085 mapped SNPs can be located on genome scaffolds (<a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Zhang <em>et al.</em> 2012</a>; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019570/-/DC1/TableS6.xlsx">Table S6</a>). Of the 618 scaffolds with mapped SNPs, 358 contain one SNP, and 260 contain two or more SNPs (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>). Of the scaffolds with two or more SNPs, 100 (38.5%) contain SNPs that unexpectedly map to two or more linkage groups (<a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a>). Cross-classifying scaffolds by the number of SNPs they contain and by the number of linkage groups to which they map reveals a highly significant, positive association between the two factors (contingency χ<sup>2</sup> = 31.84, 3 d.f., for four levels of SNPs per scaffold—2, 3, 4, and &gt;4 SNPs—and two classes of mapping—to one linkage group or to more than one linkage group; <em>P</em> &lt; &lt; 0.0001). The median length of the 100 scaffolds that map to two or more linkage groups, 740 kbp, is significantly longer than the median length of the 160 scaffolds that map to a single linkage group, 403 kbp (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>), and the more SNPs per scaffold, the longer the average multiply-mapped scaffold compared to the average singly-mapped scaffold (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1A</a> inset). Average distance between adjacent SNPs that map to different linkage groups is 273 kbp (<em>n</em> = 118), whereas the average distance between adjacent SNPs that map to the same linkage group is 58.2 kbp (<em>n</em> = 337; <em>t</em> = 11.14, <em>P</em> &lt; &lt; 0.0001, two-tailed test of difference in sample means with unequal variances). Numbers of ambiguous bases (coded as N in the published genome), between adjacent SNPs on a scaffold that either do or do not map to the same linkage group, are also significantly different (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1B</a>). The distribution of the number of Ns between adjacent SNPs mapping to the same linkage group has a mode at zero and a median of 11, whereas between adjacent SNPs that map to different linkage groups, the distribution has a mode at nearly 40,000 and a median just over 25,000 ambiguous bases.</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466811/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466811/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Numbers of mapped, coding SNPs in genome scaffolds of the Pacific oyster <em>Crassostrea gigas</em></span><div class="sb-div caption-clear"></div></div></div><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466787/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466787/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5</span> <span class="caption-title">Number of scaffolds with two or more SNPs that map to one or more than one linkage group, by number of SNPs per scaffold</span><div class="sb-div caption-clear"></div></div></div><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/10/2007/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparisons of scaffolds or adjacent single-nucleotide polymorphisms (SNPs) on a scaffold that either do (unfilled symbols or bars) or do not (filled symbols or bars) map to one linkage group. (A) Distribution of the lengths of scaffolds in these two categories; inset shows average lengths of scaffolds classified by number of SNPs per scaffold. (B) Distributions of the number of ambiguous bases (N in the genome assembly) between adjacent SNPs that either do or do not map to the same linkage group. The number of Ns is transformed to log10(#N+1)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1320023432" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Comparisons of scaffolds or adjacent single-nucleotide polymorphisms (SNPs) on a scaffold that either do (unfilled symbols or bars) or do not (filled symbols or bars) map to one linkage group. (A) Distribution of the lengths of scaffolds in these two categories; inset shows average lengths of scaffolds classified by number of SNPs per scaffold. (B) Distributions of the number of ambiguous bases (N in the genome assembly) between adjacent SNPs that either do or do not map to the same linkage group. The number of Ns is transformed to log&lt;sub&gt;10&lt;/sub&gt;(#N+1).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/10/2007/F1.medium.gif" width="440" height="179"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/5/10/2007/F1.medium.gif" width="440" height="179"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/10/2007/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/10/2007/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466815" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-37" class="first-child">Comparisons of scaffolds or adjacent single-nucleotide polymorphisms (SNPs) on a scaffold that either do (unfilled symbols or bars) or do not (filled symbols or bars) map to one linkage group. (A) Distribution of the lengths of scaffolds in these two categories; inset shows average lengths of scaffolds classified by number of SNPs per scaffold. (B) Distributions of the number of ambiguous bases (N in the genome assembly) between adjacent SNPs that either do or do not map to the same linkage group. The number of Ns is transformed to log<sub>10</sub>(#N+1).</p><div class="sb-div caption-clear"></div></div></div><p id="p-38">Finally, we consider the number of “breakpoints” necessary to account for the order with which same-scaffold SNPs are mapped to linkage groups. For example, a scaffold with three SNPs mapping to two linkage groups (<em>A</em> and <em>B</em>) can have one breakpoint (LG-assignment orders, <em>AAB</em> or <em>ABB</em>) or two breakpoints (<em>ABA</em> or <em>BAB</em>); of the 27 scaffolds in this category (<a id="xref-table-wrap-5-2" class="xref-table" href="#T5">Table 5</a>), 26 have the minimum of one breakpoint and only one has two breakpoints. Similarly, of the 13 scaffolds with four SNPs mapping to two linkage groups, all 13 have one breakpoint (6, <em>AAAB</em> and 7, <em>AABB</em>) and none has an order requiring two breakpoints (<em>AABA</em>) or three breakpoints (<em>ABAB</em>). Indeed, across the 60 informative combinations in <a id="xref-table-wrap-5-3" class="xref-table" href="#T5">Table 5</a> (<em>i.e.</em>, cases in which the number of SNPs &gt; number of LG assignments), only two cases require more than the minimum number of breakpoints.</p></div><div id="sec-11" class="subsection"><h3>Comparison of RG and ML maps</h3><p id="p-39">Provisional linkage maps made by the RG method generally provide poor fits to observed recombination rates among markers (<a id="xref-table-wrap-6-2" class="xref-table" href="#T6">Table 6</a>, nonidentical loci only). Threshold LOD values for grouping range from 3 to 10, reflecting differences in the number of progeny genotyped, such that the two larger F<sub>2</sub> families have average LOD values near 9 and the three smaller G<sub>0</sub> families have average LOD values near 5. In seven of 50 cases (five families × 10 linkage groups), default thresholds for linkage had to be relaxed to permit all markers to be added. For 44 of 50 maps, linkage of all markers could only be achieved by forcing remaining markers onto a third-round RG map, sometimes resulting in large contributions to total χ<sup>2</sup> or negative distances to existing mapped markers; on average, 10.6 makers were forced onto the 44 linkage groups, with a range from one to 42 markers. Maximum values for the nearest-neighbor fit statistic range from a low of 0.2 cM, for LG 2 in 2 × 10, to 621.3 cM, for LG 10 in the same family; indeed, 36 of 50 fit statistics exceed the estimated lengths of their linkage groups (<a id="xref-table-wrap-6-3" class="xref-table" href="#T6">Table 6</a>). Maximum nearest neighbor fit is significantly greater for forced than for unforced maps (252.6 cM <em>vs.</em> 74.0 cM, <em>F<sub>1,48</sub></em> = 4.612, <em>P</em> = 0.037).</p><div id="T6" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466823/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466823/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 6</span> <span class="caption-title">Summary statistics for regression and maximum likelihood maps for all markers grouped in 10 LG, in five families of the Pacific oyster</span><div class="sb-div caption-clear"></div></div></div><p id="p-44">The ML method yields better fitting maps, in most but not all cases; the maximum nearest-neighbor fit statistic per linkage group exceeds map length in only three of 50 cases. However, the ML maps are invariably longer than those produced by the regression method (<a id="xref-table-wrap-6-4" class="xref-table" href="#T6">Table 6</a>). Nine of the 50 ML maps have lengths of 5000+ or 10,000+ cM, owing to markers that have recombination frequencies with flanking markers of 0.5 in one parent but not in the other parent (the ML map is an average of the two parental maps). Often, these markers are segregating from only one parent (mating types, <em>np</em> × <em>nn</em> or <em>ll</em> × <em>lm</em> in <a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>). Excluding these excessively long maps, the average ML map is still three times longer than the corresponding regression map. The orders of markers produced by these maps are, nevertheless, strongly supported by the simulated annealing algorithm implemented in JoinMap (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Jansen <em>et al.</em> 2001</a>), except for closely linked markers. Agreement between the order of markers on RG and ML maps is assessed by the proportion of variance explained by a linear fit (<em>r</em><sup>2</sup>) of RG marker rank order on ML marker rank order. These <em>r</em><sup>2</sup> values range from 0.01 to 1.0, with 24 of 50 exceeding 0.75 but 8 falling below 0.1 (<a id="xref-table-wrap-6-5" class="xref-table" href="#T6">Table 6</a>)</p><p id="p-45">Distortion of Mendelian segregation ratios is widespread over these maps, as observed in previous studies (<a id="xref-ref-28-4" class="xref-bibr" href="#ref-28">Launey and Hedgecock 2001</a>; <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Li and Guo 2004</a>; <a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Sauvage <em>et al.</em> 2010</a>; <a id="xref-ref-39-10" class="xref-bibr" href="#ref-39">Plough and Hedgecock 2011</a>; <a id="xref-ref-38-4" class="xref-bibr" href="#ref-38">Plough 2012</a>; <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Guo <em>et al.</em> 2012</a>). The extent of this distortion is represented in <a id="xref-table-wrap-6-6" class="xref-table" href="#T6">Table 6</a> and <a id="xref-table-wrap-7-2" class="xref-table" href="#T7">Table 7</a> by the proportion of markers on each linkage group with a χ<sup>2</sup> goodness-of-fit test probability less than or equal to 0.01. In the vast majority of cases, markers with distorted segregation ratios are clustered on linkage maps, again, as observed in previous studies (<em>op. cit</em>.). The mean proportion of distorted markers on a linkage group, 0.31, does not vary among families but does vary among linkage groups (<em>F</em><sub>9,36</sub> = 2.404, <em>P</em> = 0.03), ranging from a low of 0.034 on LG 1 to highs of 0.56 on LG 10 and 0.61 on LG 3. There is no significant relationship between proportions of markers distorted and either map lengths or maximum nearest-neighbor fit statistics across linkage groups; but certain distorted markers clearly do contribute to poor fit and inflated map lengths (examples given below).</p><div id="T7" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466727/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466727/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 7</span> <span class="caption-title">Summary statistics for regression and maximum likelihood linkage maps made with bi-parentally inherited, framework markers, in five families of Pacific oyster</span><div class="sb-div caption-clear"></div></div></div><p id="p-48">Having observed effects of markers segregating from only one parent on map orders and lengths, we proceed with linkage maps of framework markers segregating from both parents (mating types <em>hk × hk</em>, <em>ef × eg</em>, and <em>ab × cd</em> in <a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>). For five framework maps, we exclude obviously poorly fitting markers, which are microsatellite DNA markers in seven of 17 cases. Thresholds for regression mapping are relaxed in only three of 47 cases (<a id="xref-table-wrap-7-3" class="xref-table" href="#T7">Table 7</a>; the total number of maps is reduced from 50 because families F12, F20, and F45 had too few two biparentally segregating markers on LG 9 for map construction). Only 11 of 47 regression-method maps are third-round maps, which force an average of 3.2 markers onto those maps. Maximum nearest-neighbor fit still ranges widely, from 0.1 cM to 615.5 cM, but only 10 of 47 fit statistics exceed the estimated lengths of their LGs. As in the analyses of all markers, ML maps generally fit the data on biparentally segregating markers better than do RG maps, although the ML maps are still longer in each case. Excluding two maps that exceed 5000 cM, the average ML map is 1.6 times as long as its regression-method counterpart. The <em>r</em><sup>2</sup> for the linear fit of RG marker order to ML marker order ranges from 0.09 to 1.0, with 29 of 47 being 1.0 and 36 exceeding 0.75. These framework maps were used to make consensus maps (see <em>Consensus linkage maps</em>).</p><p id="p-49">The plausibility of marker positions for two ML maps, for all markers and for biparentally segregating framework markers, are presented in supplementary information (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019570/-/DC1/FigureS1.pdf">Figure S1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019570/-/DC1/FigureS2.pdf">Figure S2</a>), to illustrate factors influencing map lengths and marker orders. In the first example, LG 1 for family 51 × 35, segregation ratios conform to Mendelian expectations, so selection is not a factor. However, markers segregating from only one parent cause moderately large nearest-neighbor fits (<em>cf</em>. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019570/-/DC1/FigureS1.pdf">Figure S1A</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019570/-/DC1/FigureS1.pdf">Figure S1B</a> with <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019570/-/DC1/FigureS1.pdf">Figure S1C</a>) and, at the termini of both parental maps, expansions of map lengths from about 70 cM to more than 200 cM. Still, agreement in the rank order of 66 markers between the RG and ML maps is fairly good, with only a few outliers (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>). Regression of RG on ML marker order is greatly improved for 32 framework markers (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>). In the second example (79 markers grouped for LG 10 of family 2 × 10), severely distorted Mendelian ratios are associated with large jumps in length and regions of very poor fit on ML parental maps (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019570/-/DC1/FigureS2.pdf">Figure S2A</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019570/-/DC1/FigureS2.pdf">Figure S2B</a>). Single-parent segregations also contribute to regions of poor fit and length expansion. Regression of RG on ML marker orders is quite poor for all 79 markers (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>). In attempting to construct the framework marker map, 12 markers with highly distorted segregation ratios had to be removed to achieve reasonable congruence of the remaining 24 marker orders on the RG and ML framework maps (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a>). Still, this reduced ML framework map, though unaffected by selection, has one region of poor fit (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019570/-/DC1/FigureS2.pdf">Figure S2C</a>) and is 2.4× as long as the comparable RG map (<a id="xref-table-wrap-7-4" class="xref-table" href="#T7">Table 7</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/10/2007/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Regression of regression (RG) marker order on maximum likelihood (ML) marker order for linkage group 1 of family 51 &#xD7; 35, from (A) fixed-order maps (RGFO, MLFO; black, filled circles and black, dotted trend line [y = 0.9705x], for all 66 markers; gray, open circles and gray, dashed trend line [y = 1.0196x], for 12 anchor loci used to fix order) and (B) maps of 32 framework markers with no fixed-order." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1320023432" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Regression of regression (RG) marker order on maximum likelihood (ML) marker order for linkage group 1 of family 51 &#xD7; 35, from (A) fixed-order maps (RGFO, MLFO; black, filled circles and black, dotted trend line [&lt;em&gt;y&lt;/em&gt; = 0.9705&lt;em&gt;x&lt;/em&gt;], for all 66 markers; gray, open circles and gray, dashed trend line [&lt;em&gt;y&lt;/em&gt; = 1.0196&lt;em&gt;x&lt;/em&gt;], for 12 anchor loci used to fix order) and (B) maps of 32 framework markers with no fixed-order.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/10/2007/F2.medium.gif" width="440" height="189"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/5/10/2007/F2.medium.gif" width="440" height="189"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/10/2007/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/10/2007/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466917" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-50" class="first-child">Regression of regression (RG) marker order on maximum likelihood (ML) marker order for linkage group 1 of family 51 × 35, from (A) fixed-order maps (RGFO, MLFO; black, filled circles and black, dotted trend line [<em>y</em> = 0.9705<em>x</em>], for all 66 markers; gray, open circles and gray, dashed trend line [<em>y</em> = 1.0196<em>x</em>], for 12 anchor loci used to fix order) and (B) maps of 32 framework markers with no fixed-order.</p><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/10/2007/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Regression of regression (RG) marker order on maximum likelihood (ML) marker order for linkage group 10 of family 2 &#xD7; 10, from (A) maps with 79 markers and (B) maps with 24 framework markers." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1320023432" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Regression of regression (RG) marker order on maximum likelihood (ML) marker order for linkage group 10 of family 2 &#xD7; 10, from (A) maps with 79 markers and (B) maps with 24 framework markers.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/10/2007/F3.medium.gif" width="440" height="190"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/5/10/2007/F3.medium.gif" width="440" height="190"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/10/2007/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/10/2007/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466753" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-51" class="first-child">Regression of regression (RG) marker order on maximum likelihood (ML) marker order for linkage group 10 of family 2 × 10, from (A) maps with 79 markers and (B) maps with 24 framework markers.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h3>Differences in recombination between sexes</h3><p id="p-52">To assess differences in recombination between males and females, we plot sums of adjacent recombination frequencies—the parameter optimized in ML mapping—for maternal against paternal ML framework linkage maps (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). For 32 maps, the sum of adjacent recombination frequencies for the female map exceeds that for the male map, while for 14 maps, the opposite is true; however, assuming an intercept of zero, the slope of the regression, is not significantly different than 1.0.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/10/2007/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of the sums of adjacent recombination frequencies from 46 framework maximum likelihood linkage maps or male and female parents of five Pacific oyster mapping families." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1320023432" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Comparison of the sums of adjacent recombination frequencies from 46 framework maximum likelihood linkage maps or male and female parents of five Pacific oyster mapping families.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/10/2007/F4.medium.gif" width="440" height="409"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/5/10/2007/F4.medium.gif" width="440" height="409"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/10/2007/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/10/2007/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466842" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-53" class="first-child">Comparison of the sums of adjacent recombination frequencies from 46 framework maximum likelihood linkage maps or male and female parents of five Pacific oyster mapping families.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>Consensus linkage maps</h3><p id="p-54">To build a consensus linkage map for the Pacific oyster, we used individual family ML framework maps, with the exceptions described in <em>Materials and Methods</em>. The final consensus linkage map has 656 markers, including 49 previously mapped microsatellite DNA markers, and spans, after adjustment, 890 cM (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019570/-/DC1/TableS7.xlsx">Table S7</a>).</p></div></div><div class="section discussion" id="sec-14"><h2 class="">Discussion</h2><p id="p-55">More than 1100, coding SNPs are placed on a second-generation genetic linkage map for the Pacific oyster <em>Crassostrea gigas</em>, along with 66 microsatellite DNA markers to establish coherence with first-generation maps. On average, the map has a spacing of 1 cM and covers 86% of a genome that is estimated, here, to be ∼616 cM in total length. This observed map length corresponds well with both a cytological estimate of map length in the eastern oyster <em>Crassostrea virginica</em> (1.2 crossovers per bivalent × 50 cM × 10 chromosomes = 600 cM; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Longwell <em>et al.</em> 1967</a>) and with genome-size estimates for the Pacific oyster, from either flow cytometry or assembly of DNA sequences (637 Mb and 559 Mb, respectively; <a id="xref-ref-50-3" class="xref-bibr" href="#ref-50">Zhang <em>et al.</em> 2012</a>), if the 1 cM ≈ 1 Mb rule of thumb from human genetics (<a href="http://ghr.nlm.nih.gov/glossary=centimorgan">http://ghr.nlm.nih.gov/glossary=centimorgan</a>) applies to the oyster.</p><p id="p-56">In contrast to what <a id="xref-ref-23-6" class="xref-bibr" href="#ref-23">Hubert and Hedgecock (2004)</a> reported, no significant differences in map distances are detected across 46 male- and female-parent maps, suggesting no marked difference in underlying male and female recombination rates in this much larger data set. On the other hand, the distribution of SNPs among parents appears to reflect both the breeding history of the mapping families used (F<sub>2</sub> <em>vs.</em> wild-caught) and an ascertainment bias stemming from the initial identification of SNPs fixed for different alleles in the inbred parent lines of the 51 × 35 F<sub>2</sub> family.</p><p id="p-57">This second-generation map is a substantial improvement over previously published, first-generation maps for this species, which were either low-density maps based on microsatellite DNA markers or on small numbers of SNPs or higher-density maps based on dominant, largely nontransferable AFLP markers. A consensus map based on three families and 100 microsatellite DNA markers (<a id="xref-ref-23-7" class="xref-bibr" href="#ref-23">Hubert and Hedgecock 2004</a>; <a id="xref-ref-24-5" class="xref-bibr" href="#ref-24">Hubert <em>et al.</em> 2009</a>) had average lengths of 616 cM and 770 cM for male- and female-based maps, respectively, average spacing of 8−10 cM and genome coverage of 70–80%. <a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">Li and Guo (2004)</a> constructed male and female maps with dominant AFLP markers; the male map had 96 markers in 10 linkage groups, covering 758 cM with an average spacing of 8.8 cM, whereas the female map had 119 markers in 11 linkage groups, covering 1031 cM with an average spacing of 9.5 cM. Later, Guo <em>et al.</em> (2012; see also <a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Zhong <em>et al.</em> 2014</a>) published another map based on a single, full-sib family typed for 64 genomic microsatellite DNA markers, 42 EST-associated microsatellite DNA markers, and 320 dominant AFLP markers; this map spanned 558 cM, had an average spacing of 1.3 cM, and covered 95% of the genome. <a id="xref-ref-40-3" class="xref-bibr" href="#ref-40">Sauvage <em>et al.</em> (2010)</a> built a consensus map of 51 microsatellite markers and 29 EST-derived SNPs typed in three F<sub>2</sub> families, which spanned 1016 cM, had an average spacing of 12.7 cM, and covered 73.6% of the genome. Most first-generation maps are much longer than the second-generation map reported here and, of course, have much lower densities of markers. All of these previous reports note significant distortion of segregation ratios for a substantial proportion of markers, rendering mapping difficult in some cases (<a id="xref-ref-40-4" class="xref-bibr" href="#ref-40">Sauvage <em>et al.</em> 2010</a>).</p><div id="sec-15" class="subsection"><h3>Implications for genome scaffold assembly</h3><p id="p-58">Perhaps, the most surprising finding in this study was the frequency with which markers on the same genome scaffold map to different LGs. Nearly 40% of genome scaffolds with two or more SNPs map to different linkage groups, which strongly suggests that a substantial fraction of the genome scaffolds reported by <a id="xref-ref-50-4" class="xref-bibr" href="#ref-50">Zhang <em>et al.</em> (2012)</a> are incorrectly assembled. The likelihood of misassembly appears to rise with scaffold length, and additional statistical comparisons of inter-marker distances and numbers of ambiguous bases between adjacent SNPs support this inference. Adjacent SNPs mapping to different LGs are 4.7 times farther apart than adjacent SNPs mapping to the same LG, whereas sequences between adjacent SNPs that do or do not map to the same LG have median numbers of 11 and 25,161 ambiguous bases, respectively. Because the minimum number of breakpoints required to explain the order of mapped SNPs is found in 58 of 60 informative cases, our data suggest that misassembled scaffolds comprise large blocks (contigs or super-contigs, see <a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a> in <a id="xref-ref-50-5" class="xref-bibr" href="#ref-50">Zhang <em>et al.</em> 2012</a>) that map to different linkage groups and are separated by long stretches of ambiguous bases. These large scaffolds should be shattered at potential breakpoints and reassembled using linkage information or long-range DNA sequences. That 50% of the genome assembly is found in the largest 401 scaffolds (N50 size of 401.3 kb; <a id="xref-ref-50-6" class="xref-bibr" href="#ref-50">Zhang <em>et al.</em> 2012</a>) will likely prove to be an overestimate.</p><p id="p-59">This conclusion about scaffold assemblies contrasts sharply with the fact that 95% of ESTs were successfully mapped to the draft genome assembly (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019570/-/DC1/TableS6.xlsx">Table S6</a> in <a id="xref-ref-50-7" class="xref-bibr" href="#ref-50">Zhang <em>et al.</em> 2012</a>). Evidently, at the level of contigs and genes, the assembly of the Pacific oyster genome is quite complete and useful, but at the level of scaffolds, the assembly contains many errors. Because the ordering of contigs and genes within scaffolds may not always be accurate, inferences about the clustering of gene families (<em>e.g.</em>, <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Paps <em>et al.</em> 2015</a>) should be regarded as provisional until the assembly of genome scaffolds is either confirmed or improved. High density linkage maps are presently being constructed, using genotyping-by-sequencing methods (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Elshire <em>et al.</em> 2011</a>); these maps should provide critical information for improving future assemblies of the Pacific oyster genome, as they have in other species (<em>e.g.</em>, <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Dalloul <em>et al.</em> 2010</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Dodgson <em>et al.</em> 2010</a>).</p></div><div id="sec-16" class="subsection"><h3>Uncertainties in linkage maps</h3><p id="p-60">In the course of this study, three factors emerged as evident causes of discrepancies in map orders and lengths between RG and ML methods and among families. The first, mentioned above and illustrated by two examples (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019570/-/DC1/FigureS1.pdf">Figure S1</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019570/-/DC1/FigureS2.pdf">Figure S2</a>, <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>, and <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>), is the effect of markers segregating from only one parent, which consistently contribute to inflated ML map lengths and poor agreement between RG and ML maps. Across all 47 linkage groups, the general effect of single-parent markers is reflected in the average improvement in agreement of RG and ML marker orders between maps made with all markers and maps made only from bi-parentally segregating framework markers (mean <em>r</em><sup>2</sup> for all-marker maps, 0.67; mean <em>r</em><sup>2</sup> for framework maps, 0.85; paired sample <em>t</em> = 3.17, <em>P</em> = 0.0014). For this reason, only framework maps are merged into consensus linkage maps.</p><p id="p-61">The second factor causing uncertainty in map construction is distortion of Mendelian inheritance ratios, owing to strong viability selection, as documented in several previous studies of oysters (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Bierne <em>et al.</em> 1998</a>; <a id="xref-ref-28-5" class="xref-bibr" href="#ref-28">Launey and Hedgecock 2001</a>; <a id="xref-ref-39-11" class="xref-bibr" href="#ref-39">Plough and Hedgecock 2011</a>; <a id="xref-ref-38-5" class="xref-bibr" href="#ref-38">Plough 2012</a>). In this study, 45 of 50 linkage groups show evidence of viability selection (<a id="xref-table-wrap-2-5" class="xref-table" href="#T2">Table 2</a>), yet the effect of viability selection on map construction is not a general one. Across all linkage groups, we could find no significant relationship between the proportions of markers distorted and the lengths of maps, the fit of maps to data, or the agreement of RL and ML marker orders. Furthermore, counterexamples to a general effect of viability selection on linkage mapping are evident—linkage groups with no evidence of selection but with uncertainties in map orders and lengths (<em>e.g.</em>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019570/-/DC1/FigureS1.pdf">Figure S1</a> and <a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a>) or linkage groups with most or even all markers showing distorted segregation ratios but having well-behaved maps (LG 3, LG 7, and LG 10 for family 51 × 35, <a id="xref-table-wrap-6-7" class="xref-table" href="#T6">Table 6</a> and <a id="xref-table-wrap-7-5" class="xref-table" href="#T7">Table 7</a>). On the other hand, distorted markers do often contribute to poor fit and extended map lengths (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019570/-/DC1/FigureS2.pdf">Figure S2</a> and <a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3</a>). This is especially the case, if markers segregating from one or the other parent fall into regions of segregation distortion.</p><p id="p-62">Mode of selection appears to make a difference, because massively distorted but well-behaved LGs occur only in the F<sub>2</sub> families, in which associative overdominance—the apparent advantage of heterozygotes, owing to selection against recessive deleterious mutations linked to alternative homozygotes—is a consequence of their breeding history, as is the excess of <em>hk</em>×<em>hk</em> mating types in these families (<a id="xref-table-wrap-1-6" class="xref-table" href="#T1">Table 1</a>; <a id="xref-ref-39-12" class="xref-bibr" href="#ref-39">Plough and Hedgecock 2011</a>). For example, on the ML framework map for LG 10 in family 51 × 35, all 42 markers are highly significantly distorted, but nearest-neighbor fit averages only 0.085 cM and the RG and ML maps are highly correlated (<em>r</em><sup>2</sup> = 0.93; <a id="xref-table-wrap-7-6" class="xref-table" href="#T7">Table 7</a>). The relative fitness of heterozygotes compared to the most abundant homozygotes across the ML map for this linkage group averages 2.03, reaching a peak at 10.3, where genotypic proportions for SNP 1385-A193329G are zero <em>hh</em>, 103 <em>hk</em>, and 5 <em>kk</em>. In this case, the symmetry of the selection on parental types appears not to affect the fit of observed and calculated recombination frequencies and marker orders. In contrast, a framework ML map of 36 markers for the same linkage group, LG 10, in family 2 × 10, has two blocks of distorted markers (12 markers altogether, which had to be removed to produce the framework map recorded in <a id="xref-table-wrap-7-7" class="xref-table" href="#T7">Table 7</a> and <a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3</a>), but nearest-neighbor fit averages 281 cM and orders of markers on the RG and ML maps are poorly correlated (<em>r</em><sup>2</sup> = 0.61). The average relative fitness of heterozygotes compared to the most abundant homozygotes at each marker across this ML map is 0.75, reaching a nadir of ∼0.3, where one homozygote disappears entirely. Here, the gross asymmetry and partial dominance of viability selection is clearly associated with inflation of ML map distances and incongruence of RG and ML marker orders. Thus, the number, location, and phase of loci under viability selection produce idiosyncratic effects on map lengths and marker orders. Detailed analyses of the location and effects of viability mutations in the G<sub>0</sub> families are reported elsewhere (L. V. Plough, G. Shin, and D. Hedgecock, unpublished data).</p><p id="p-63">Distortion of Mendelian segregation ratios is commonly observed in mapping studies, especially in plants and marine molluscs, although the phenomenon may generally be underreported, because distorted markers often are discarded from data sets before linkage analysis. Distortion of segregation ratios can arise in interspecific or intersubspecific crosses, because of Dobzhansky-Muller incompatibilities (<em>e.g.</em>, <a id="xref-ref-16-4" class="xref-bibr" href="#ref-16">Fishman <em>et al.</em> 2001</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Wang <em>et al.</em> 2005</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Foley <em>et al.</em> 2013</a>), but our focus, here, is on distortions of segregation ratios arising from genotype-dependent mortality of progeny from intraspecific crosses of Pacific oysters (<a id="xref-ref-28-6" class="xref-bibr" href="#ref-28">Launey and Hedgecock 2001</a>; <a id="xref-ref-39-13" class="xref-bibr" href="#ref-39">Plough and Hedgecock 2011</a>). Viability loci causing early mortality in oysters appear to act independently of one another, as no statistically significant epistatic interactions are detected (<a id="xref-ref-39-14" class="xref-bibr" href="#ref-39">Plough and Hedgecock 2011</a>; <a id="xref-ref-38-6" class="xref-bibr" href="#ref-38">Plough 2012</a>; L.V. Plough, G. Shin, and D. Hedgecock, unpublished data).</p><p id="p-64">Advice on what to do about segregation distortion in linkage mapping studies ranges from discard distorted markers, to exercise great caution in interpreting results from commonly used computer packages (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Liu 1998</a>), to include such markers, so as to retain genetic and potentially biologically important information (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Shah <em>et al.</em> 2014</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Truong <em>et al.</em> 2014</a>). Theoretical studies on how segregation distortion can bias estimates of recombination date back to <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bailey (1949)</a> and <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Allard and Alder (1960)</a> but yield few generalizations regarding the practice of linkage mapping, particularly now, when dense linkage maps are enabled by high-throughput DNA sequencing. Estimates of recombination are affected by the form of selection (allelic <em>vs.</em> zygotic), by the dominance of viability genes, and by the dominance of markers (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Lorieux <em>et al.</em> 1995</a>; <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Liu 1998</a>), which suggests that impacts of segregation distortion are likely to vary within and among linkage maps, owing to variation in the underlying modes of selection. <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Hackett and Broadfoot (2003)</a> concluded that segregation distortion does not substantially impact map length or marker order, but they simulated only one distorted marker per linkage group, in a doubled-haploid mapping population, which may underestimate the impact of segregation distortion in other mapping populations or species with large numbers of viability loci, such as the oyster. Marker density also appears to be a factor, for example, marker orders in low-density, first-generation linkage maps for the Pacific oyster were reasonably consistent across families and studies (<a id="xref-ref-39-15" class="xref-bibr" href="#ref-39">Plough and Hedgecock 2011</a>; <a id="xref-ref-38-7" class="xref-bibr" href="#ref-38">Plough 2012</a>; <a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">Guo <em>et al.</em> 2012</a>), despite widespread distortion of segregation ratios.</p><p id="p-65">Efficient estimators of maps that take into account distortion or even simultaneously estimate fitness parameters are either computationally burdensome for dense maps or, when made computationally efficient, have so far been developed only for mapping populations common in plant genetics, <em>i.e.</em>, F<sub>2</sub>, doubled-haploid, BC, RIL, and multiparent advanced generation intercross (<em>i.e.</em>, MAGIC) populations (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Wu <em>et al.</em> 2008</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Zhu <em>et al.</em> 2007</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Zhu and Zhang 2007</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Lorieux 2012</a>; <a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Shah <em>et al.</em> 2014</a>). In this study, we have left distorted markers in all analyses, discarding them only from the minority of framework maps in which they prevent a reasonable fit between observed and expected recombination rates or good agreement of marker orders from the regression and maximum likelihood mapping methods. For future linkage or QTL mapping studies with oysters, doubts about marker orders and distances, owing to selection and segregation distortion, can largely be eliminated by taking samples for genotyping in the mid to late larval stages (in addition to the life stage or time-point of interest), since much of the selective mortality occurs later, during metamorphosis (<a id="xref-ref-39-16" class="xref-bibr" href="#ref-39">Plough and Hedgecock 2011</a>).</p><p id="p-66">A third factor likely contributing to discrepancies in map order and lengths is genotyping error (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Hackett and Broadfoot 2003</a>). Microsatellite genotypes for families F12, F20, and F45 were scored in duplicate reactions for at least 5% of individuals and an average of 17 markers, in each family (L. V. Plough, G. Shin, and D. Hedgecock, unpublished data), revealing an average error rate of 2.5%. Such an error rate has a small effect on map accuracy, when marker densities are low (∼10 cM), as in previous linkage maps for the Pacific oyster based on microsatellite DNA markers (<a id="xref-ref-39-17" class="xref-bibr" href="#ref-39">Plough and Hedgecock 2011</a>; <a id="xref-ref-38-8" class="xref-bibr" href="#ref-38">Plough 2012</a>; L.V. Plough, G. Shin, and D. Hedgecock, unpublished data), but can impair the ordering of markers, when the density of markers is 1-2 per cM, as in this study (<a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">Hackett and Broadfoot 2003</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Ball <em>et al.</em> 2010</a>). That microsatellite markers frequently have poor nearest-neighbor fits is most likely explained by their relatively large genotyping error in the context of dense, second-generation SNP maps. The genotyping error of SNP markers, on the other hand, was extremely low (zero observed in 12,937 trials). Still, an observed frequency of zero is compatible with an error rate as high as 3 in 12,937, which yields a 5% chance of observing no errors in 12,937 trials of a binomial distribution. Given this potential error rate and the large number of genotypes determined in this study, nearly 172 thousand, the total number of genotyping errors could have ranged from 19 to 61 (99.9% confidence limits), with an average of 40 errors, and therefore could have contributed to otherwise unexplained problems in mapping.</p></div><div id="sec-17" class="subsection"><h3>A second-generation consensus linkage map for the Pacific oyster</h3><p id="p-67">The consensus linkage map made with MergeMap (<a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Wu <em>et al.</em> 2011</a>) appears too long (890 cM), compared to the estimated length of the genetic map (616 cM) and the size of the genome (637 Mb) estimated by flow cytometry (<a id="xref-ref-50-8" class="xref-bibr" href="#ref-50">Zhang <em>et al.</em> 2012</a>). Still, this map likely represent the best consensus ordering of markers and is presented here for future reference (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019570/-/DC1/TableS7.xlsx">Table S7</a>).</p></div></div><div class="section" id="sec-18"></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-68">We are especially indebted to Daniel S. Rokhsar and Erika Lindquist of the Joint Genome Institute for guidance in the early stages of this project. We thank Jonathan P. Davis (Taylor Shellfish Farms), for rearing the families and providing samples, and Austin Paul, for providing two libraries of Illumina sequencing for SNP discovery. SNP development was aided immensely by the Oyster Genome Project of the Chinese Academy of Sciences, in particular Haigang Qi, Li Li, and Guofan Zhang of the Institute of Oceanology, Chinese Academy of Sciences, Qingdao, Shandong 266071, China, who aligned the JGI EST sequences to an early version of the oyster genome in order to elucidate intron-exon boundaries. We thank Dr. A. Arias Perez for aid in linkage mapping and various bioinformatic analyses. The manuscript was improved by thoughtful and constructive comments from two anonymous reviewers, in particular the suggestion to examine the apparent “breakpoints” in misjoined scaffolds. The authors are grateful for the support of the U.S. Department of Energy Joint Genome Institute (Community Sequencing Project UAE_CSP06-SE20_Hedgecock) and the U.S. Department of Agriculture (National Research Initiative Competitive Grants Program award no. 2009-35205-05052).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-supplementary-material" id="fn-17"><p id="p-69">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019570/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019570/-/DC1</a></p></li><li class="fn" id="fn-18"><p id="p-70"><em>Communicating editor: W. S. Davidson</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2015-06-01"><span class="received-label">Received </span>June 1, 2015.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2015-07-25"><span class="accepted-label">Accepted </span>July 25, 2015.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2015 Hedgecock <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2007.1" data-doi="10.1038/hdy.1960.82"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Allard</span>  <span class="cit-name-given-names">A. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alder</span>  <span class="cit-name-given-names">H. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1960</span> <span class="cit-article-title">The effect of incomplete penetrance on the estimation of recombination values.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">263</span>–<span class="cit-lpage">282</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D15%26rft.spage%253D263%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.1960.82%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.1960.82&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2007.2" data-doi="10.1093/icb/icu062"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Applebaum</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pan</span>  <span class="cit-name-given-names">T.-C. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hedgecock</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manahan</span>  <span class="cit-name-given-names">D. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Separating the nature and nurture of energy allocation in response to global change.</span> <abbr class="cit-jnl-abbrev">Integr. Comp. Biol.</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">284</span>–<span class="cit-lpage">295</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DIntegr.%2BComp.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Ficb%252Ficu062%26rft_id%253Dinfo%253Apmid%252F24907199%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiaWNiaW9sIjtzOjU6InJlc2lkIjtzOjg6IjU0LzIvMjg0IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMC8yMDA3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2007.3" data-doi="10.1038/hdy.1949.14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bailey</span>  <span class="cit-name-given-names">N. T. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1949</span> <span class="cit-article-title">The estimation of linkage with differential viability, II and III.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">220</span>–<span class="cit-lpage">228</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D3%26rft.spage%253D220%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.1949.14%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.1949.14&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2007.4" data-doi="10.1186/1471-2164-11-218"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ball</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stapley</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dawson</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Birkhead</span>  <span class="cit-name-given-names">T. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burke</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A comparison of SNPs and microsatellites as linkage mapping markers: lessons from the zebra finch (<em>Taeniopygia guttata</em>).</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">218</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DBall%26rft.auinit1%253DA.%2BD.%26rft.volume%253D11%26rft.spage%253D218%26rft.epage%253D218%26rft.atitle%253DA%2Bcomparison%2Bof%2BSNPs%2Band%2Bmicrosatellites%2Bas%2Blinkage%2Bmapping%2Bmarkers%253A%2Blessons%2Bfrom%2Bthe%2Bzebra%2Bfinch%2B%2528Taeniopygia%2Bguttata%2529.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-11-218%26rft_id%253Dinfo%253Apmid%252F20359323%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-11-218&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20359323&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2007.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2007.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bierne</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Launey</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Naciri-Graven</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bonhomme</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Early effect of inbreeding as revealed by microsatellite analyses on <em>Ostrea edulis</em> larvae.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">148</span>: <span class="cit-fpage">1893</span>–<span class="cit-lpage">1906</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DBierne%26rft.auinit1%253DN.%26rft.volume%253D148%26rft.issue%253D4%26rft.spage%253D1893%26rft.epage%253D1906%26rft.atitle%253DEarly%2BEffect%2Bof%2BInbreeding%2Bas%2BRevealed%2Bby%2BMicrosatellite%2BAnalyses%2Bon%2BOstrea%2Bedulis%2BLarvae%26rft_id%253Dinfo%253Apmid%252F9560403%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE0OC80LzE4OTMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzEwLzIwMDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-book" id="cit-5.10.2007.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bishop</span>  <span class="cit-name-given-names">D. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cannings</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Skolnick</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williamson</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-article-title">The number of polymorphic DNA clones required to map the human genome</span> in <span class="cit-source">Statistical Analysis of DNA Sequence Data</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Weir</span>  <span class="cit-name-given-names">B. S.</span></span>. <span class="cit-publ-name">Marcel Dekker</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2007.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chakravarti</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lasher</span>  <span class="cit-name-given-names">L. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reefer</span>  <span class="cit-name-given-names">J. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">A maximum likelihood for estimating genome length using genetic linkage data.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">128</span>: <span class="cit-fpage">175</span>–<span class="cit-lpage">182</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DChakravarti%26rft.auinit1%253DA.%26rft.volume%253D128%26rft.issue%253D1%26rft.spage%253D175%26rft.epage%253D182%26rft.atitle%253DA%2Bmaximum%2Blikelihood%2Bmethod%2Bfor%2Bestimating%2Bgenome%2Blength%2Busing%2Bgenetic%2Blinkage%2Bdata.%26rft_id%253Dinfo%253Apmid%252F2060775%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTI4LzEvMTc1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMC8yMDA3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2007.8" data-doi="10.1101/gr.1917404"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chevreux</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pfisterer</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Drescher</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Driesel</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muller</span>  <span class="cit-name-given-names">W. E. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Using the miraEST assembler for reliable and automated mRNA transcript assembly and SNP detection in sequenced ESTs.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">1147</span>–<span class="cit-lpage">1159</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.1917404%26rft_id%253Dinfo%253Apmid%252F15140833%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE0LzYvMTE0NyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTAvMjAwNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-book" id="cit-5.10.2007.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Curole</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hedgecock</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Bivalve genomics: complications, challenges, and future perspectives</span>, pp. <span class="cit-fpage">525</span>–<span class="cit-lpage">543</span> in <span class="cit-source">Aquaculture Genome Technologies</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Z. J.</span></span>. <span class="cit-publ-name">Blackwell Publishing</span>, <span class="cit-publ-loc">Ames, Iowa</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2007.10" data-doi="10.1371/journal.pbio.1000475"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dalloul</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Long</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zimin</span>  <span class="cit-name-given-names">A. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aslam</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beal</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Multi-platform next-generation sequencing of the domestic turkey (<em>Meleagris gallopavo</em>): Genome assembly and analysis.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1000475</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DDalloul%26rft.volume%253D8%26rft.issue%253D9%26rft.spage%253De1000475%26rft.atitle%253DMulti-platform%2Bnext-generation%2Bsequencing%2Bof%2Bthe%2Bdomestic%2Bturkey%2B%2528Meleagris%2Bgallopavo%2529%253A%2Bgenome%2Bassembly%2Band%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.1000475%26rft_id%253Dinfo%253Apmid%252F20838655%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.1000475&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20838655&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2007.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2007.11" data-doi="10.1016/j.aquaculture.2009.11.017"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dégremont</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bédier</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boudry</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">mortality of hatchery-produced Pacific oyster spat (<em>Crassostrea gigas</em>). II. Response to selection for survival and its influence on growth and yield.</span> <abbr class="cit-jnl-abbrev">Aquaculture</abbr> <span class="cit-vol">299</span>: <span class="cit-fpage">21</span>–<span class="cit-lpage">29</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAquaculture%26rft.volume%253D299%26rft.spage%253D21%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.aquaculture.2009.11.017%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.aquaculture.2009.11.017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2007.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dodgson</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Delany</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">H. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Poultry genome sequences: Progress and outstanding challenges.</span> <abbr class="cit-jnl-abbrev">Cytogenet. Genome Res.</abbr> <span class="cit-vol">134</span>: <span class="cit-fpage">19</span>–<span class="cit-lpage">26</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCytogenet.%2BGenome%2BRes.%26rft.volume%253D134%26rft.spage%253D19%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2007.13" data-doi="10.1038/nrg703"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Doerge</span>  <span class="cit-name-given-names">R. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Mapping and analysis of quantitative trait loci in experimental populations.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">43</span>–<span class="cit-lpage">52</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DDoerge%26rft.auinit1%253DR.%2BW.%26rft.volume%253D3%26rft.issue%253D1%26rft.spage%253D43%26rft.epage%253D52%26rft.atitle%253DMapping%2Band%2Banalysis%2Bof%2Bquantitative%2Btrait%2Bloci%2Bin%2Bexperimental%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg703%26rft_id%253Dinfo%253Apmid%252F11823790%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg703&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11823790&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2007.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000173267200016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2007.14" data-doi="10.1371/journal.pone.0019379"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawamoto</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e19379</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DElshire%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253De19379%26rft.epage%253De19379%26rft.atitle%253DA%2Brobust%252C%2Bsimple%2Bgenotyping-by-sequencing%2B%2528GBS%2529%2Bapproach%2Bfor%2Bhigh%2Bdiversity%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0019379%26rft_id%253Dinfo%253Apmid%252F21573248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0019379&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21573248&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2007.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-web" id="cit-5.10.2007.15"><div class="cit-metadata unstructured">Food and Agriculture Organization (FAO), 2015 World aquaculture production of fish, crustaceans, molluscs, etc., by principal species in 2013. Available at: <a href="ftp://ftp.fao.org/fi/stat/summary/a-6.pdf">ftp://ftp.fao.org/fi/stat/summary/a-6.pdf</a>. Accessed August 10, 2015.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2007.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fishman</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kelly</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morgan</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Willis</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">A genetic map in the <em>Mimulus guttatus</em> species complex reveals transmission ratio distortion due to heterospecific interactions.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">159</span>: <span class="cit-fpage">1701</span>–<span class="cit-lpage">1716</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DFishman%26rft.auinit1%253DL.%26rft.volume%253D159%26rft.issue%253D4%26rft.spage%253D1701%26rft.epage%253D1716%26rft.atitle%253DA%2BGenetic%2BMap%2Bin%2Bthe%2BMimulus%2Bguttatus%2BSpecies%2BComplex%2BReveals%2BTransmission%2BRatio%2BDistortion%2Bdue%2Bto%2BHeterospecific%2BInteractions%26rft_id%253Dinfo%253Apmid%252F11779808%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1OS80LzE3MDEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzEwLzIwMDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2007.17" data-doi="10.1038/hdy.2013.61"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Foley</span>  <span class="cit-name-given-names">B. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rose</span>  <span class="cit-name-given-names">C. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rundle</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leong</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Edmands</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Postzygotic isolation involves strong mitochondrial and sex-specific effects in <em>Tigriopus californicus</em>, a species lacking heteromorphic sex chromosomes.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">111</span>: <span class="cit-fpage">391</span>–<span class="cit-lpage">401</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D111%26rft.spage%253D391%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2013.61%26rft_id%253Dinfo%253Apmid%252F23860232%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2013.61&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23860232&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2007.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2007.18" data-doi="10.1007/s10126-011-9405-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Q. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kong</span>  <span class="cit-name-given-names">L. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genetic mapping and QTL analysis of growth-related traits in the Pacific oyster.</span> <abbr class="cit-jnl-abbrev">Mar. Biotechnol. (NY)</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">218</span>–<span class="cit-lpage">226</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMarine%2Bbiotechnology%2B%2528New%2BYork%252C%2BN.Y.%2529%26rft.stitle%253DMar%2BBiotechnol%2B%2528NY%2529%26rft.aulast%253DGuo%26rft.auinit1%253DX.%26rft.volume%253D14%26rft.issue%253D2%26rft.spage%253D218%26rft.epage%253D226%26rft.atitle%253DGenetic%2Bmapping%2Band%2BQTL%2Banalysis%2Bof%2Bgrowth-related%2Btraits%2Bin%2Bthe%2BPacific%2Boyster.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10126-011-9405-4%26rft_id%253Dinfo%253Apmid%252F21932055%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10126-011-9405-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21932055&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2007.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2007.19" data-doi="10.1038/sj.hdy.6800173"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hackett</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Broadfoot</span>  <span class="cit-name-given-names">L. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Effects of genotyping errors, missing values and segregation distortion in molecular marker data on the construction of linkage maps.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">90</span>: <span class="cit-fpage">33</span>–<span class="cit-lpage">38</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DHackett%26rft.auinit1%253DC.%2BA.%26rft.volume%253D90%26rft.issue%253D1%26rft.spage%253D33%26rft.epage%253D38%26rft.atitle%253DEffects%2Bof%2Bgenotyping%2Berrors%252C%2Bmissing%2Bvalues%2Band%2Bsegregation%2Bdistortion%2Bin%2Bmolecular%2Bmarker%2Bdata%2Bon%2Bthe%2Bconstruction%2Bof%2Blinkage%2Bmaps.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.hdy.6800173%26rft_id%253Dinfo%253Apmid%252F12522423%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/sj.hdy.6800173&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12522423&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2007.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000181165800007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2007.20" data-doi="10.1016/j.aquaculture.2007.07.226"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hedgecock</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">J. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Heterosis for yield and crossbreeding of the Pacific oyster <em>Crassostrea gigas</em>.</span> <abbr class="cit-jnl-abbrev">Aquaculture</abbr> <span class="cit-vol">272</span>(<span class="cit-supplement">Suppl. 1</span>)<strong>:</strong> <span class="cit-fpage">S17</span>–<span class="cit-lpage">S29</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAquaculture%26rft.volume%253D272%26rft.spage%253DS17%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.aquaculture.2007.07.226%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.aquaculture.2007.07.226&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000252083500003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2007.21" data-doi="10.2983/0730-8000(2005)24[429:TCFSTP]2.0.CO;2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hedgecock</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gaffney</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goulletquer</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reece</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">The case for sequencing the Pacific oyster genome.</span> <abbr class="cit-jnl-abbrev">J. Shellfish Res.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">429</span>–<span class="cit-lpage">441</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BShellfish%2BRes.%26rft.volume%253D24%26rft.spage%253D429%26rft_id%253Dinfo%253Adoi%252F10.2983%252F0730-8000%25282005%252924%255B429%253ATCFSTP%255D2.0.CO%253B2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2983/0730-8000(2005)24[429:TCFSTP]2.0.CO;2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000231367900013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2007.22" data-doi="10.1016/0044-8486(84)90269-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hershberger</span>  <span class="cit-name-given-names">W. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perdue</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beattie</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title">Genetic selection and systematic breeding in Pacific oyster culture.</span> <abbr class="cit-jnl-abbrev">Aquaculture</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">237</span>–<span class="cit-lpage">245</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAquaculture%26rft.volume%253D39%26rft.spage%253D237%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0044-8486%252884%252990269-2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0044-8486(84)90269-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2007.23" data-doi="10.1534/genetics.104.027342"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hubert</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hedgecock</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Linkage maps of microsatellite DNA markers for the Pacific oyster <em>Crassostrea gigas</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">168</span>: <span class="cit-fpage">351</span>–<span class="cit-lpage">362</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHubert%26rft.auinit1%253DS.%26rft.volume%253D168%26rft.issue%253D1%26rft.spage%253D351%26rft.epage%253D362%26rft.atitle%253DLinkage%2BMaps%2Bof%2BMicrosatellite%2BDNA%2BMarkers%2Bfor%2Bthe%2BPacific%2BOyster%2BCrassostrea%2Bgigas%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.027342%26rft_id%253Dinfo%253Apmid%252F15454548%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTY4LzEvMzUxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMC8yMDA3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2007.24" data-doi="10.1016/j.aquaculture.2008.12.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hubert</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cognard</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hedgecock</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Centromere-mapping in triploid families of the Pacific oyster <em>Crassostrea gigas</em> (Thunberg).</span> <abbr class="cit-jnl-abbrev">Aquaculture</abbr> <span class="cit-vol">288</span>: <span class="cit-fpage">172</span>–<span class="cit-lpage">183</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAquaculture%26rft.volume%253D288%26rft.spage%253D172%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.aquaculture.2008.12.006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.aquaculture.2008.12.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000264622300004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2007.25" data-doi="10.1007/s001220000489"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jansen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Jong</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Ooijen</span>  <span class="cit-name-given-names">J. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Constructing dense genetic linkage maps.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">102</span>: <span class="cit-fpage">1113</span>–<span class="cit-lpage">1122</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D102%26rft.spage%253D1113%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs001220000489%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s001220000489&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000169278600044&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2007.26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lander</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Botstein</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Mapping Mendelian factors underlying quantitative traits using RFLP linkage maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">121</span>: <span class="cit-fpage">185</span>–<span class="cit-lpage">199</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLander%26rft.auinit1%253DE.%2BS.%26rft.volume%253D121%26rft.issue%253D1%26rft.spage%253D185%26rft.epage%253D199%26rft.atitle%253DMapping%2Bmendelian%2Bfactors%2Bunderlying%2Bquantitative%2Btraits%2Busing%2BRFLP%2Blinkage%2Bmaps.%26rft_id%253Dinfo%253Apmid%252F2563713%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTIxLzEvMTg1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMC8yMDA3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2007.27" data-doi="10.1016/S0044-8486(02)00621-X"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langdon</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Evans</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jacobson</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blouin</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Yields of cultured Pacific oysters <em>Crassostrea gigas</em> Thunberg improved after one generation of selection.</span> <abbr class="cit-jnl-abbrev">Aquaculture</abbr> <span class="cit-vol">220</span>: <span class="cit-fpage">227</span>–<span class="cit-lpage">244</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAquaculture%26rft.volume%253D220%26rft.spage%253D227%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0044-8486%252802%252900621-X%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0044-8486(02)00621-X&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000182099100020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2007.28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Launey</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hedgecock</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">High genetic load in the Pacific oyster <em>Crassostrea gigas</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">159</span>: <span class="cit-fpage">155</span>–<span class="cit-lpage">165</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D159%26rft.spage%253D155%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2007.29" data-doi="10.1093/bioinformatics/btp324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fast and accurate short read alignment with Burrows-Wheeler Transform.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1754</span>–<span class="cit-lpage">1760</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp324%26rft_id%253Dinfo%253Apmid%252F19451168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTQvMTc1NCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTAvMjAwNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2007.30" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Handsaker</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wysoker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fennell</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span>, </li><li><span class="cit-auth cit-collab">1000 Genome Project Data Processing Subgroup</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The Sequence alignment/map (SAM) format and SAMtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2078</span>–<span class="cit-lpage">2079</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTYvMjA3OCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTAvMjAwNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2007.31" data-doi="10.1007/s10126-003-0001-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">X. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">AFLP-based genetic linkage maps of the Pacific oyster <em>Crassostrea gigas</em> Thunberg.</span> <abbr class="cit-jnl-abbrev">Mar. Biotechnol. (NY)</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">26</span>–<span class="cit-lpage">36</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMarine%2Bbiotechnology%2B%2528New%2BYork%252C%2BN.Y.%2529%26rft.stitle%253DMar%2BBiotechnol%2B%2528NY%2529%26rft.aulast%253DLi%26rft.auinit1%253DL.%26rft.volume%253D6%26rft.issue%253D1%26rft.spage%253D26%26rft.epage%253D36%26rft.atitle%253DAFLP-based%2Bgenetic%2Blinkage%2Bmaps%2Bof%2Bthe%2Bpacific%2Boyster%2BCrassostrea%2Bgigas%2BThunberg.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10126-003-0001-0%26rft_id%253Dinfo%253Apmid%252F14564534%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10126-003-0001-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14564534&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2007.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-book" id="cit-5.10.2007.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">B. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-source"><em>Statistical Genomics: Linkage</em>, <em>Mapping</em>, <em>and QTL Analysis</em></span>. <span class="cit-publ-name">CRC Press</span>, <span class="cit-publ-loc">Florida</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2007.33" data-doi="10.1139/g67-090"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Longwell</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stiles</span>  <span class="cit-name-given-names">S. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">D. G.</span></span></li></ol><cite>, <span class="cit-pub-date">1967</span> <span class="cit-article-title">Chromosome complement of the American oyster <em>Crassostrea virginica</em> as seen in meiotic and cleaving eggs.</span> <abbr class="cit-jnl-abbrev">Can. J. Genet. Cytol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">845</span>–<span class="cit-lpage">856</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCanadian%2Bjournal%2Bof%2Bgenetics%2Band%2Bcytology.%2B%2BJournal%2Bcanadien%2Bde%2Bg%25C3%25A9n%25C3%25A9tique%2Bet%2Bde%2Bcytologie%26rft.stitle%253DCan%2BJ%2BGenet%2BCytol%26rft.aulast%253DLongwell%26rft.auinit1%253DA.%2BC.%26rft.volume%253D9%26rft.issue%253D4%26rft.spage%253D845%26rft.epage%253D856%26rft.atitle%253DChromosome%2Bcomplement%2Bof%2Bthe%2BAmerican%2Boyster%2BCrassostrea%2Bvirginica%252C%2Bas%2Bseen%2Bin%2Bmeiotic%2Band%2Bcleaving%2Beggs.%26rft_id%253Dinfo%253Adoi%252F10.1139%252Fg67-090%26rft_id%253Dinfo%253Apmid%252F5588794%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1139/g67-090&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=5588794&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2007.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1967A483000019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2007.34" data-doi="10.1007/s11032-012-9706-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lorieux</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">MapDisto: fast and efficient computation of genetic linkage maps.</span> <abbr class="cit-jnl-abbrev">Mol. Breed.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">1231</span>–<span class="cit-lpage">1235</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBreed.%26rft.volume%253D30%26rft.spage%253D1231%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11032-012-9706-y%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11032-012-9706-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2007.35" data-doi="10.1007/BF00220999"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lorieux</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perrier</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goffinet</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lanaud</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de León</span>  <span class="cit-name-given-names">D. G.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Maximum-likelihood models for mapping genetic markers showing segregation distortion. 2. F2 populations.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">90</span>: <span class="cit-fpage">81</span>–<span class="cit-lpage">89</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D90%26rft.spage%253D81%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00220999%26rft_id%253Dinfo%253Apmid%252F24173787%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00220999&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24173787&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2007.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995QF11000011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-book" id="cit-5.10.2007.36"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mann</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1979</span> <span class="cit-source">Exotic Species in Mariculture</span>. <span class="cit-publ-name">MIT Press</span>, <span class="cit-publ-loc">Massachusetts</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2007.37" data-doi="10.1093/gbe/evv018"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Paps</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holland</span>  <span class="cit-name-given-names">P. W. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Reinforcing the egg-timer: recruitment of novel Lophotrochozoa homeobox genes to early and late development in the Pacific oyster.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">677</span>–<span class="cit-lpage">688</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevv018%26rft_id%253Dinfo%253Apmid%252F25631164%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2JlIjtzOjU6InJlc2lkIjtzOjc6IjcvMy82NzciO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzEwLzIwMDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2007.38" data-doi="10.1111/j.1365-294X.2012.05688.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Plough</span>  <span class="cit-name-given-names">L. V.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Environmental stress increases selection against and dominance of deleterious mutations in inbred families of the Pacific oyster, <em>Crassostrea gigas</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">3974</span>–<span class="cit-lpage">3987</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DPlough%26rft.auinit1%253DL.%2BV.%26rft.volume%253D21%26rft.issue%253D16%26rft.spage%253D3974%26rft.epage%253D3987%26rft.atitle%253DEnvironmental%2Bstress%2Bincreases%2Bselection%2Bagainst%2Band%2Bdominance%2Bof%2Bdeleterious%2Bmutations%2Bin%2Binbred%2Bfamilies%2Bof%2Bthe%2BPacific%2Boyster%2BCrassostrea%2Bgigas.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2012.05688.x%26rft_id%253Dinfo%253Apmid%252F22747636%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2012.05688.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22747636&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2007.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000306897500008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2007.39" data-doi="10.1534/genetics.111.131854"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Plough</span>  <span class="cit-name-given-names">L. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hedgecock</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Quantitative trait locus analysis of stage-specific inbreeding depression in the Pacific oyster <em>Crassostrea gigas</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">189</span>: <span class="cit-fpage">1473</span>–<span class="cit-lpage">1486</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.131854%26rft_id%253Dinfo%253Apmid%252F21940682%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4OS80LzE0NzMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzEwLzIwMDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2007.40"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sauvage</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boudry</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Koning</span>  <span class="cit-name-given-names">D.-J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haley</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heurtebise</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">QTL for resistance to summer mortality and OsHV-1 load in the Pacific oyster (<em>Crassostrea gigas</em>).</span> <abbr class="cit-jnl-abbrev">Anim. Genet.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">390</span>–<span class="cit-lpage">399</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnimal%2Bgenetics%26rft.stitle%253DAnim%2BGenet%26rft.aulast%253DSauvage%26rft.auinit1%253DC.%26rft.volume%253D41%26rft.issue%253D4%26rft.spage%253D390%26rft.epage%253D399%26rft.atitle%253DQTL%2Bfor%2Bresistance%2Bto%2Bsummer%2Bmortality%2Band%2BOsHV-1%2Bload%2Bin%2Bthe%2BPacific%2Boyster%2B%2528Crassostrea%2Bgigas%2529.%26rft_id%253Dinfo%253Apmid%252F20096029%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=20096029&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2007.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2007.41" data-doi="10.1007/s00122-014-2401-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shah</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cavanagh</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">B. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Computationally efficient map construction in the presence of segregation distortion.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">127</span>: <span class="cit-fpage">2585</span>–<span class="cit-lpage">2597</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D127%26rft.spage%253D2585%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-014-2401-0%26rft_id%253Dinfo%253Apmid%252F25260690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-014-2401-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25260690&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2007.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2007.42" data-doi="10.1016/j.mrfmmm.2004.07.022"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fan</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Campbell</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">High-throughput SNP genotyping on universal bead arrays.</span> <abbr class="cit-jnl-abbrev">Mutat. Res.</abbr> <span class="cit-vol">573</span>: <span class="cit-fpage">70</span>–<span class="cit-lpage">82</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMutation%2BResearch%26rft.stitle%253DMutation%2BResearch%26rft.aulast%253DShen%26rft.auinit1%253DR.%26rft.volume%253D573%26rft.issue%253D1-2%26rft.spage%253D70%26rft.epage%253D82%26rft.atitle%253DHigh-throughput%2BSNP%2Bgenotyping%2Bon%2Buniversal%2Bbead%2Barrays.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.mrfmmm.2004.07.022%26rft_id%253Dinfo%253Apmid%252F15829238%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.mrfmmm.2004.07.022&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15829238&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2007.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000228887700006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2007.43" data-doi="10.1111/j.1365-313X.1993.00739.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stam</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Construction of integrated genetic linkage maps by means of a new computer package: JoinMap.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">739</span>–<span class="cit-lpage">744</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BJ.%26rft.volume%253D3%26rft.spage%253D739%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-313X.1993.00739.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-313X.1993.00739.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1993LC75800012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2007.44" data-doi="10.1534/g3.114.012468"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Truong</span>  <span class="cit-name-given-names">S. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCormick</span>  <span class="cit-name-given-names">R. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morishige</span>  <span class="cit-name-given-names">D. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mullet</span>  <span class="cit-name-given-names">J. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Resolution of genetic map expansion caused by excess heterozygosity in plant recombinant inbred populations.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">1963</span>–<span class="cit-lpage">1969</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.114.012468%26rft_id%253Dinfo%253Apmid%252F25128435%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjk6IjQvMTAvMTk2MyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTAvMjAwNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-book" id="cit-5.10.2007.45"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Van Ooijen</span>  <span class="cit-name-given-names">J. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-source"><em>JoinMap 4</em>, <em>Software for the calculation of genetic linkage maps in experimental populations</em></span>. <span class="cit-publ-name">Kyazma B. V.</span>, <span class="cit-publ-loc">The Netherlands</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2007.46" data-doi="10.1017/S0016672305007779"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhai</span>  <span class="cit-name-given-names">H. Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wan</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Mapping segregation distortion loci and quantitative trait loci for spikelet sterility in rice (<em>Oryza sativa</em> L.).</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">86</span>: <span class="cit-fpage">97</span>–<span class="cit-lpage">106</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DWang%26rft.auinit1%253DC.%26rft.volume%253D86%26rft.issue%253D2%26rft.spage%253D97%26rft.epage%253D106%26rft.atitle%253DMapping%2Bsegregation%2Bdistortion%2Bloci%2Band%2Bquantitative%2Btrait%2Bloci%2Bfor%2Bspikelet%2Bsterility%2Bin%2Brice%2B%2528%2BOryza%2Bsativa%2BL.%2529.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672305007779%26rft_id%253Dinfo%253Apmid%252F16356283%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672305007779&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16356283&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2007.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2007.47" data-doi="10.2983/0730-8000(2008)27[495:COCOFN]2.0.CO;2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lio</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">X. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Classification of common oysters from North China.</span> <abbr class="cit-jnl-abbrev">J. Shellfish Res.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">495</span>–<span class="cit-lpage">503</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BShellfish%2BRes.%26rft.volume%253D27%26rft.spage%253D495%26rft_id%253Dinfo%253Adoi%252F10.2983%252F0730-8000%25282008%252927%255B495%253ACOCOFN%255D2.0.CO%253B2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2983/0730-8000(2008)27[495:COCOFN]2.0.CO;2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2007.48" data-doi="10.1371/journal.pgen.1000212"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">Y. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bhat</span>  <span class="cit-name-given-names">P. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Close</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lonardi</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient and accurate construction of genetic linkage maps from the minimum spanning tree of a graph.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e1000212</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DWu%26rft.auinit1%253DY.%26rft.volume%253D4%26rft.issue%253D10%26rft.spage%253De1000212%26rft.epage%253De1000212%26rft.atitle%253DEfficient%2Band%2Baccurate%2Bconstruction%2Bof%2Bgenetic%2Blinkage%2Bmaps%2Bfrom%2Bthe%2Bminimum%2Bspanning%2Btree%2Bof%2Ba%2Bgraph.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000212%26rft_id%253Dinfo%253Apmid%252F18846212%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000212&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18846212&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2007.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2007.49" data-doi="10.1109/TCBB.2010.35"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">Y. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Close</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lonardi</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Accurate construction of consensus genetic maps via integer linear programming.</span> <abbr class="cit-jnl-abbrev">IEEE-ACM Trans. Comp. Biol. Bioinform.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">381</span>–<span class="cit-lpage">394</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DIEEE-ACM%2BTrans.%2BComp.%2BBiol.%2BBioinform.%26rft.volume%253D8%26rft.spage%253D381%26rft_id%253Dinfo%253Adoi%252F10.1109%252FTCBB.2010.35%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1109/TCBB.2010.35&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2007.50" data-doi="10.1038/nature11413"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luo</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The oyster genome reveals stress adaptation and complexity of shell formation.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">490</span>: <span class="cit-fpage">49</span>–<span class="cit-lpage">54</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DZhang%26rft.auinit1%253DG.%26rft.volume%253D490%26rft.issue%253D7418%26rft.spage%253D49%26rft.epage%253D54%26rft.atitle%253DThe%2Boyster%2Bgenome%2Breveals%2Bstress%2Badaptation%2Band%2Bcomplexity%2Bof%2Bshell%2Bformation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11413%26rft_id%253Dinfo%253Apmid%252F22992520%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11413&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22992520&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2007.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000309446800030&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2007.51" data-doi="10.1007/s10499-014-9789-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhong</span>  <span class="cit-name-given-names">X. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kong</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">QTL mapping for glycogen content and shell pigmentation in the Pacific oyster <em>Crassostrea gigas</em> using microsatellites and SNPs.</span> <abbr class="cit-jnl-abbrev">Aquacult. Int.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">1877</span>–<span class="cit-lpage">1899</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAquacult.%2BInt.%26rft.volume%253D22%26rft.spage%253D1877%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10499-014-9789-z%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10499-014-9789-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2007.52"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y.-M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">An EM algorithm for mapping segregation distortion loci.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">82</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenetics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenet%26rft.aulast%253DZhu%26rft.auinit1%253DC.%26rft.volume%253D8%26rft.spage%253D82%26rft.epage%253D82%26rft.atitle%253DAn%2BEM%2Balgorithm%2Bfor%2Bmapping%2Bsegregation%2Bdistortion%2Bloci.%26rft_id%253Dinfo%253Apmid%252F18047652%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=18047652&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2007.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2007.53" data-doi="10.1007/s00122-006-0432-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y.-M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Modeling segregation distortion for viability selection I. reconstruction of linkage maps with distorted markers.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">114</span>: <span class="cit-fpage">295</span>–<span class="cit-lpage">305</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DZhu%26rft.auinit1%253DC.%26rft.volume%253D114%26rft.issue%253D2%26rft.spage%253D295%26rft.epage%253D305%26rft.atitle%253DModeling%2Bsegregation%2Bdistortion%2Bfor%2Bviability%2Bselection.%2BI.%2BReconstruction%2Bof%2Blinkage%2Bmaps%2Bwith%2Bdistorted%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-006-0432-x%26rft_id%253Dinfo%253Apmid%252F17119913%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-006-0432-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17119913&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2007.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/5/10/2007.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/10/1999" title="Fingerprinting Soybean Germplasm and Its Utility in Genomic Research" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/10/2021" title="High-Density Genotypes of Inbred Mouse Strains: Improved Power and Precision of Association Mapping" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/5/10"><span>Volume 5 Issue 10, October 2015</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/5/10.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/5/10.cover-source.jpg?itok=JwgNhVLw" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 5 (10)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/10/2007" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-GhADk_rPcifIspS7Gl4OQFSvweMblOHbfSL6teNj9zQ" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/5/10/2007" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/5/10/2007" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/5/10/2007" class="active" data-icon-position="" data-hide-link-title="0">Second-Generation Linkage Maps for the Pacific Oyster Crassostrea gigas Reveal Errors in Assembly of Genome Scaffolds</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460153" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-TFyXzBTNPxuNc9ZKjTniHq4Ogq9XZNPQfdsIZeFweAI" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/10/2007.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/5/10/2007" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-TpWgm-qcOEiJ8oxVpXhr7v7Cq4Us5aYKrAmxzsn16bg" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/5/10/2007" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/10/2007.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460153" id="node460153--2" data-pisa="ggg;5/10/2007" data-pisa-master="ggg;g3.115.019570" data-apath="/ggg/5/10/2007.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Second-Generation Linkage Maps for the Pacific Oyster <em>Crassostrea gigas</em> Reveal Errors in Assembly of Genome Scaffolds</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-3995-646X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Dennis Hedgecock</span>, <span class="highwire-citation-author" data-delta="1">Grace Shin</span>, <span class="highwire-citation-author" data-delta="2">Andrew Y. Gracey</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-5404-5218" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>David Van Den Berg</span> and <span class="highwire-citation-author" data-delta="4">Manoj P. Samanta</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2007-2019;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.019570 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460153--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Dennis Hedgecock </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biological Sciences, University of Southern California, Los Angeles, California 90089-0371 </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dennis%2BHedgecock%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hedgecock%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADennis%2BHedgecock%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-3995-646X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Dennis Hedgecock</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:dhedge@usc.edu" class="" data-icon-position="" data-hide-link-title="0">dhedge@usc.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Grace Shin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biological Sciences, University of Southern California, Los Angeles, California 90089-0371 </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Grace%2BShin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shin%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGrace%2BShin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Andrew Y. Gracey </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biological Sciences, University of Southern California, Los Angeles, California 90089-0371 </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrew%2BY.%2BGracey%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gracey%20AY&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAndrew%2BY.%2BGracey%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">David Van Den Berg </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Preventive Medicine, Keck School of Medicine, University of Southern California, Los Angeles, California 90089-9601 </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BVan%2BDen%2BBerg%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Den%20Berg%20DV&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADavid%2BVan%2BDen%2BBerg%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-5404-5218" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for David Van Den Berg</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Manoj P. Samanta </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Systemix Institute, Redmond, Washington 98053</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Manoj%2BP.%2BSamanta%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Samanta%20MP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AManoj%2BP.%2BSamanta%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/5/10/2007" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/5/10/2007&amp;title=Second-Generation%20Linkage%20Maps%20for%20the%20Pacific%20Oyster%20Crassostrea%20gigas%20Reveal%20Errors%20in%20Assembly%20of%20Genome%20Scaffolds" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/5/10/2007&amp;title=Second-Generation%20Linkage%20Maps%20for%20the%20Pacific%20Oyster%20Crassostrea%20gigas%20Reveal%20Errors%20in%20Assembly%20of%20Genome%20Scaffolds" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/5/10/2007&amp;title=Second-Generation%20Linkage%20Maps%20for%20the%20Pacific%20Oyster%20Crassostrea%20gigas%20Reveal%20Errors%20in%20Assembly%20of%20Genome%20Scaffolds" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/5/10/2007&amp;text=Second-Generation%20Linkage%20Maps%20for%20the%20Pacific%20Oyster%20Crassostrea%20gigas%20Reveal%20Errors%20in%20Assembly%20of%20Genome%20Scaffolds" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/5/10/2007&amp;title=Second-Generation%20Linkage%20Maps%20for%20the%20Pacific%20Oyster%20Crassostrea%20gigas%20Reveal%20Errors%20in%20Assembly%20of%20Genome%20Scaffolds" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/5/10/2007&amp;t=Second-Generation%20Linkage%20Maps%20for%20the%20Pacific%20Oyster%20Crassostrea%20gigas%20Reveal%20Errors%20in%20Assembly%20of%20Genome%20Scaffolds" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/5/10/2007&amp;title=Second-Generation%20Linkage%20Maps%20for%20the%20Pacific%20Oyster%20Crassostrea%20gigas%20Reveal%20Errors%20in%20Assembly%20of%20Genome%20Scaffolds" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/5/10/2007&amp;title=Second-Generation%20Linkage%20Maps%20for%20the%20Pacific%20Oyster%20Crassostrea%20gigas%20Reveal%20Errors%20in%20Assembly%20of%20Genome%20Scaffolds" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460153/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460153" id="node460153--3" data-pisa="ggg;5/10/2007" data-pisa-master="ggg;g3.115.019570" data-apath="/ggg/5/10/2007.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Second-Generation Linkage Maps for the Pacific Oyster <em>Crassostrea gigas</em> Reveal Errors in Assembly of Genome Scaffolds</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-3995-646X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Dennis Hedgecock</span>, <span class="highwire-citation-author" data-delta="1">Grace Shin</span>, <span class="highwire-citation-author" data-delta="2">Andrew Y. Gracey</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-5404-5218" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>David Van Den Berg</span> and <span class="highwire-citation-author" data-delta="4">Manoj P. Samanta</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2007-2019;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.019570 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460153--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Dennis Hedgecock </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biological Sciences, University of Southern California, Los Angeles, California 90089-0371 </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dennis%2BHedgecock%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hedgecock%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADennis%2BHedgecock%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-3995-646X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Dennis Hedgecock</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:dhedge@usc.edu" class="" data-icon-position="" data-hide-link-title="0">dhedge@usc.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Grace Shin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biological Sciences, University of Southern California, Los Angeles, California 90089-0371 </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Grace%2BShin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shin%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGrace%2BShin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Andrew Y. Gracey </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biological Sciences, University of Southern California, Los Angeles, California 90089-0371 </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrew%2BY.%2BGracey%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gracey%20AY&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAndrew%2BY.%2BGracey%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">David Van Den Berg </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Preventive Medicine, Keck School of Medicine, University of Southern California, Los Angeles, California 90089-9601 </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BVan%2BDen%2BBerg%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Den%20Berg%20DV&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADavid%2BVan%2BDen%2BBerg%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-5404-5218" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for David Van Den Berg</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Manoj P. Samanta </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Systemix Institute, Redmond, Washington 98053</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Manoj%2BP.%2BSamanta%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Samanta%20MP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AManoj%2BP.%2BSamanta%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2007&amp;rft.epage=2019&amp;rft.atitle=Second-Generation%20Linkage%20Maps%20for%20the%20Pacific%20Oyster%20Crassostrea%20gigas%20Reveal%20Errors%20in%20Assembly%20of%20Genome%20Scaffolds&amp;rft.volume=5&amp;rft.issue=10&amp;rft.date=2015-10-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Hedgecock%2C+Dennis&amp;rft.au=Shin%2C+Grace&amp;rft.au=Gracey%2C+Andrew+Y.&amp;rft.au=Den+Berg%2C+David+Van&amp;rft.au=Samanta%2C+Manoj+P."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460153/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460153/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460153/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460153/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460153/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460153/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460153/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460153/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460153/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460153/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460153/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460153/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/5/10/2007" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-14" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/5/10/2007.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/5/10/2007.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/5/10/2007.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/5/10/2007.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
