<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/5/10/2177" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/5/10/2177.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/5/10/2177.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/5/10/2177.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/5/10/2177/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/5/10/2177.atom" />
<meta name="HW.pisa" content="ggg;5/10/2177" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Identification and Correction of Sample Mix-Ups in Expression Genetic Data: A Case Study" />
<meta name="DC.Identifier" content="10.1534/g3.115.019778" />
<meta name="DC.Date" content="2015-10-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2015 Broman et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="In a mouse intercross with more than 500 animals and genome-wide gene expression data on six tissues, we identified a high proportion (18%) of sample mix-ups in the genotype data. Local expression quantitative trait loci (eQTL; genetic loci influencing gene expression) with extremely large effect were used to form a classifier to predict an individual’s eQTL genotype based on expression data alone. By considering multiple eQTL and their related transcripts, we identified numerous individuals whose predicted eQTL genotypes (based on their expression data) did not match their observed genotypes, and then went on to identify other individuals whose genotypes did match the predicted eQTL genotypes. The concordance of predictions across six tissues indicated that the problem was due to mix-ups in the genotypes (although we further identified a small number of sample mix-ups in each of the six panels of gene expression microarrays). Consideration of the plate positions of the DNA samples indicated a number of off-by-one and off-by-two errors, likely the result of pipetting errors. Such sample mix-ups can be a problem in any genetic study, but eQTL data allow us to identify, and even correct, such problems. Our methods have been implemented in an R package, R/lineup." />
<meta name="DC.Contributor" content="Karl W. Broman" />
<meta name="DC.Contributor" content="Mark P. Keller" />
<meta name="DC.Contributor" content="Aimee Teo Broman" />
<meta name="DC.Contributor" content="Christina Kendziorski" />
<meta name="DC.Contributor" content="Brian S. Yandell" />
<meta name="DC.Contributor" content="Śaunak Sen" />
<meta name="DC.Contributor" content="Alan D. Attie" />
<meta name="article:published_time" content="2015-10-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Identification and Correction of Sample Mix-Ups in Expression Genetic Data: A Case Study" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;In a mouse intercross with more than 500 animals and genome-wide gene expression data on six tissues, we identified a high proportion (18%) of sample mix-ups in the genotype data. Local expression quantitative trait loci (eQTL; genetic loci influencing gene expression) with extremely large effect were used to form a classifier to predict an individual’s eQTL genotype based on expression data alone. By considering multiple eQTL and their related transcripts, we identified numerous individuals whose predicted eQTL genotypes (based on their expression data) did not match their observed genotypes, and then went on to identify other individuals whose genotypes did match the predicted eQTL genotypes. The concordance of predictions across six tissues indicated that the problem was due to mix-ups in the genotypes (although we further identified a small number of sample mix-ups in each of the six panels of gene expression microarrays). Consideration of the plate positions of the DNA samples indicated a number of off-by-one and off-by-two errors, likely the result of pipetting errors. Such sample mix-ups can be a problem in any genetic study, but eQTL data allow us to identify, and even correct, such problems. Our methods have been implemented in an R package, R/lineup.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2015/10/01" />
<meta name="citation_mjid" content="ggg;5/10/2177" />
<meta name="citation_id" content="5/10/2177" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/5/10/2177" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/5/10/2177.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/5/10/2177.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/5/10/2177.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.115.019778" />
<meta name="citation_pmid" content="26290572" />
<meta name="citation_volume" content="5" />
<meta name="citation_issue" content="10" />
<meta name="citation_num_pages" content="10" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2177" />
<meta name="citation_lastpage" content="2186" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Karl W. Broman" />
<meta name="citation_author_institution" content="Department of Biostatistics and Medical Informatics, University of Wisconsin, Madison, Wisconsin 53706" />
<meta name="citation_author_email" content="kbroman@biostat.wisc.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-4914-6671" />
<meta name="citation_author" content="Mark P. Keller" />
<meta name="citation_author_institution" content="Department of Biochemistry, University of Wisconsin, Madison, Wisconsin 53706" />
<meta name="citation_author" content="Aimee Teo Broman" />
<meta name="citation_author_institution" content="Department of Biostatistics and Medical Informatics, University of Wisconsin, Madison, Wisconsin 53706" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-3783-2807" />
<meta name="citation_author" content="Christina Kendziorski" />
<meta name="citation_author_institution" content="Department of Biostatistics and Medical Informatics, University of Wisconsin, Madison, Wisconsin 53706" />
<meta name="citation_author" content="Brian S. Yandell" />
<meta name="citation_author_institution" content="Department of Statistics, University of Wisconsin, Madison, Wisconsin 53706" />
<meta name="citation_author_institution" content="Department of Horticulture, University of Wisconsin, Madison, Wisconsin 53706" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-8774-9377" />
<meta name="citation_author" content="Śaunak Sen" />
<meta name="citation_author_institution" content="Department of Epidemiology and Biostatistics, University of California, San Francisco, California 94107" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-4519-6361" />
<meta name="citation_author" content="Alan D. Attie" />
<meta name="citation_author_institution" content="Department of Biochemistry, University of Wisconsin, Madison, Wisconsin 53706" />
<meta name="citation_reference" content="citation_journal_title=Journal of Clinical Oncology;citation_journal_abbrev=JCO;citation_author=K. A. Baggerly;citation_author=K. R. Coombes;citation_title=Run Batch Effects Potentially Compromise the Usefulness of Genomic Signatures for Ovarian Cancer;citation_pages=1186-1187;citation_volume=26;citation_year=2008;citation_issue=7;citation_pmid=18309960;citation_doi=10.1200/JCO.2007.15.1951" />
<meta name="citation_reference" content="citation_journal_title=Ann. Appl. Stat.;citation_author=K. A. Baggerly;citation_author=K. R. Coombes;citation_title=Deriving chemosensitivity from cell lines: forensic bioinformatics and reproducible research in high-throughput biology.;citation_pages=1309-1334;citation_volume=3;citation_year=2009;citation_doi=10.1214/09-AOAS291" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=J. Bootkrajang;citation_author=A. Kabán;citation_title=Classification of mislabelled microarrays using robust sparse logistic regression.;citation_volume=29;citation_year=2013;citation_pmid=23418189;citation_doi=10.1093/bioinformatics/btt078" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=K. W. Broman;citation_title=Mapping Quantitative Trait Loci in the Case of a Spike in the Phenotype Distribution;citation_pages=1169-1175;citation_volume=163;citation_year=2003;citation_issue=3;citation_pmid=12663553" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=K. W. Broman;citation_author=H. Wu;citation_author=S. Sen;citation_author=G. A. Churchill;citation_title=R/qtl: QTL mapping in experimental crosses;citation_pages=889-890;citation_volume=19;citation_year=2003;citation_issue=7;citation_pmid=12724300;citation_doi=10.1093/bioinformatics/btg112" />
<meta name="citation_reference" content="citation_journal_title=J. Genet.;citation_author=T. C. Carter;citation_author=D. S. Falconer;citation_title=Stocks for detecting linkage in the mouse, and the theory of their design.;citation_pages=307-323;citation_volume=50;citation_year=1951;citation_pmid=24539711;citation_doi=10.1007/BF02996226" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=E. Choy;citation_author=R. Yelensky;citation_author=S. Bonakdar;citation_author=R. M. Plenge;citation_author=R. Saxena;citation_title=Genetic analysis of human traits in vitro: drug response and gene expression in lymphoblastoid cell lines.;citation_pages=e1000287-e1000287;citation_volume=4;citation_year=2008;citation_issue=11;citation_pmid=19043577;citation_doi=10.1371/journal.pgen.1000287" />
<meta name="citation_reference" content="citation_journal_title=Stat. Appl. Genet. Mol. Biol.;citation_author=C. T. Ekstrøm;citation_author=B. Feenstra;citation_title=Detecting sample misidentifications in genetic association studies.;citation_pages=13;citation_volume=11;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=Biotechnology and Bioengineering;citation_journal_abbrev=Biotechnology and Bioengineering;citation_author=G. R. Grant;citation_author=E. Manduchi;citation_author=A. Pizarro;citation_author=C. J. Stoeckert;citation_title=Maintaining data integrity in microarray data management.;citation_pages=795-800;citation_volume=84;citation_year=2003;citation_issue=7;citation_pmid=14708120;citation_doi=10.1002/bit.10847" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=C. S. Haley;citation_author=S. A. Knott;citation_title=A simple regression method for mapping quantitative trait loci in line crosses using flanking markers.;citation_pages=315-324;citation_volume=69;citation_year=1992;citation_issue=4;citation_pmid=16718932;citation_doi=10.1038/hdy.1992.131" />
<meta name="citation_reference" content="citation_journal_title=Vox sanguinis;citation_journal_abbrev=Vox Sang;citation_author=C. J. J. Huijsmans;citation_author=F. G. C. Heilmann;citation_author=A. G. M. van der Zanden;citation_author=P. M. Schneeberger;citation_author=M. H. A. Hermans;citation_title=Single nucleotide polymorphism profiling assay to exclude serum sample mix-up.;citation_pages=148-153;citation_volume=92;citation_year=2007;citation_issue=2;citation_pmid=17298578;citation_doi=10.1111/j.1423-0410.2006.00871.x" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Diagnostics;citation_journal_abbrev=J. Mol. Diagn.;citation_author=R. Huijsmans;citation_author=J. Damen;citation_author=H. van der Linden;citation_author=M. Hermans;citation_title=Single Nucleotide Polymorphism Profiling Assay to Confirm the Identity of Human Tissues;citation_pages=205-213;citation_volume=9;citation_year=2007;citation_issue=2;citation_pmid=17384212;citation_doi=10.2353/jmoldx.2007.060059" />
<meta name="citation_reference" content="citation_journal_title=Drug discovery today;citation_journal_abbrev=Drug Discov Today;citation_author=S. Imbeaud;citation_author=C. Auffray;citation_title='The 39 steps' in gene expression profiling: critical issues and proposed best practices for microarray experiments.;citation_pages=1175-1182;citation_volume=10;citation_year=2005;citation_issue=17;citation_pmid=16182210;citation_doi=10.1016/S1359-6446(05)03565-8" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=G. Jun;citation_author=M. Flickinger;citation_author=K. N. Hetrick;citation_author=J. M. Romm;citation_author=K. F. Doheny;citation_title=Detecting and estimating contamination of human DNA samples in sequencing and array-based genotype data.;citation_pages=839-848;citation_volume=91;citation_year=2012;citation_issue=5;citation_pmid=23103226;citation_doi=10.1016/j.ajhg.2012.09.004" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=M. P. Keller;citation_author=Y. Choi;citation_author=P. Wang;citation_author=D. B. Davis;citation_author=M. E. Rabaglia;citation_title=A gene expression network model of type 2 diabetes links cell cycle regulation in islets with diabetes susceptibility.;citation_volume=18;citation_year=2008;citation_pmid=18347327;citation_doi=10.1101/gr.074914.107" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=E. S. Lander;citation_author=D. Botstein;citation_title=Mapping mendelian factors underlying quantitative traits using RFLP linkage maps.;citation_pages=185-199;citation_volume=121;citation_year=1989;citation_issue=1;citation_pmid=2563713" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=A. G. Lynch;citation_author=S.-F. Chin;citation_author=M. J. Dunning;citation_author=C. Caldas;citation_author=S. Tavaré;citation_title=Calling sample mix-ups in cancer population studies.;citation_pages=e41815-e41815;citation_volume=7;citation_year=2012;citation_issue=8;citation_pmid=22912679;citation_doi=10.1371/journal.pone.0041815" />
<meta name="citation_reference" content="citation_title=R: A Language and Environment for Statistical Computing;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=E. E. Schadt;citation_author=S. Woo;citation_author=K. Hao;citation_title=Bayesian method to predict individual SNP genotypes from gene  expression data.;citation_pages=603-608;citation_volume=44;citation_year=2012;citation_issue=5;citation_pmid=22484626;citation_doi=10.1038/ng.2248" />
<meta name="citation_reference" content="citation_journal_title=Biotechniques;citation_author=M. Walter;citation_author=A. Honegger;citation_author=R. Schweizer;citation_author=S. Poths;citation_author=M. Bonin;citation_title=Utilization of AFFX spike-in control probes to monitor sample identity throughout Affymetrix GeneChip Array processing.;citation_pages=371-378;citation_volume=48;citation_year=2010;citation_pmid=20569210;citation_doi=10.2144/000113421" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H.-J. Westra;citation_author=R. C. Jansen;citation_author=R. S. N. Fehrmann;citation_author=G. J. te Meerman;citation_author=D. van Heel;citation_title=MixupMapper: correcting sample mix-ups in genome-wide datasets increases power to detect small genetic effects.;citation_volume=27;citation_year=2011;citation_pmid=21653519;citation_doi=10.1093/bioinformatics/btr323" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=S. Xu;citation_author=W. R. Atchley;citation_title=Mapping Quantitative Trait Loci for Complex Binary Diseases Using Line Crosses;citation_pages=1417-1424;citation_volume=143;citation_year=1996;citation_issue=3;citation_pmid=8807312" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=C. Zhang;citation_author=C. Wu;citation_author=E. Blanzieri;citation_author=Y. Zhou;citation_author=Y. Wang;citation_title=Methods for labeling error detection in microarrays based on the effect of data perturbation on the regression model.;citation_volume=25;citation_year=2009;citation_pmid=19661242;citation_doi=10.1093/bioinformatics/btp478" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=M. J. Zilliox;citation_author=R. A. Irizarry;citation_title=A gene expression bar code for microarray data.;citation_pages=911-913;citation_volume=4;citation_year=2007;citation_issue=11;citation_pmid=17906632;citation_doi=10.1038/nmeth1102" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Identification and Correction of Sample Mix-Ups in Expression Genetic Data: A Case Study" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10.cover-source.jpg" />
<meta name="twitter:description" content="In a mouse intercross with more than 500 animals and genome-wide gene expression data on six tissues, we identified a high proportion (18%) of sample mix-ups in the genotype data. Local expression quantitative trait loci (eQTL; genetic loci influencing gene expression) with extremely large effect were used to form a classifier to predict an individual’s eQTL genotype based on expression data alone. By considering multiple eQTL and their related transcripts, we identified numerous individuals whose predicted eQTL genotypes (based on their expression data) did not match their observed genotypes, and then went on to identify other individuals whose genotypes did match the predicted eQTL genotypes. The concordance of predictions across six tissues indicated that the problem was due to mix-ups in the genotypes (although we further identified a small number of sample mix-ups in each of the six panels of gene expression microarrays). Consideration of the plate positions of the DNA samples indicated a number of off-by-one and off-by-two errors, likely the result of pipetting errors. Such sample mix-ups can be a problem in any genetic study, but eQTL data allow us to identify, and even correct, such problems. Our methods have been implemented in an R package, R/lineup." />
<meta name="og-title" property="og:title" content="Identification and Correction of Sample Mix-Ups in Expression Genetic Data: A Case Study" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/5/10/2177" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="In a mouse intercross with more than 500 animals and genome-wide gene expression data on six tissues, we identified a high proportion (18%) of sample mix-ups in the genotype data. Local expression quantitative trait loci (eQTL; genetic loci influencing gene expression) with extremely large effect were used to form a classifier to predict an individual’s eQTL genotype based on expression data alone. By considering multiple eQTL and their related transcripts, we identified numerous individuals whose predicted eQTL genotypes (based on their expression data) did not match their observed genotypes, and then went on to identify other individuals whose genotypes did match the predicted eQTL genotypes. The concordance of predictions across six tissues indicated that the problem was due to mix-ups in the genotypes (although we further identified a small number of sample mix-ups in each of the six panels of gene expression microarrays). Consideration of the plate positions of the DNA samples indicated a number of off-by-one and off-by-two errors, likely the result of pipetting errors. Such sample mix-ups can be a problem in any genetic study, but eQTL data allow us to identify, and even correct, such problems. Our methods have been implemented in an R package, R/lineup." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10.cover-source.jpg" />
    <title>Identification and Correction of Sample Mix-Ups in Expression Genetic Data: A Case Study | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"CGUmw8nPZMQQRTpraHdrn4wZzDItGW7BqEN-22LHxGw","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460044","apath":"\/ggg\/5\/10\/2177.atom","pisa":"ggg;5\/10\/2177","ac":{"\/ggg\/5\/10\/2177.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/5\/10\/2177.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;5\/10\/2177"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;5\/10\/2177"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/5\/10\/2177?uritype=cgi\u0026view=full":true,"\/content\/5\/10\/2177":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022w\\\\\\\/4aaXAVs7tsL3Vd4MTzlzgKqG7R8DLlw\\\\\\\/hHuX7N9nZXSVY4pOYwpcA99Y9NEUHSioveOS6KS+hy3Oobhy2kUdYyaAWcjfK3OjFe5fS6r2N6GgjeXWCdY95113YYoK5bDsPZHhoJpZrxhMPSZl5NGb8n92w11a+V\\\\\\\/K+xGayP0Pw4uqha1oLYQfR0whmHuGVUx7c16A1vAerziSY85Ntjz1w+Mwp1zK4F+jC3FrTODL81yUR38FEdbKdHu8u+1Abo7hCx8KX7YQICJi48k52QDGhDZE0Wf1\\\\\\\/jSgDN8YtasZXYNnC39JT69TjMlMTH4p3eMDr735XPx\\\\\\\/mA2QhrCgUUZX0ftfP6RJTnfdRy7MAEvZl0NsSawsQ9lJj2mboiJB3Dl6PmylX9xY4OwxbNiPWmCPMBWMXNDnShCHkGvFTVpuACZ7ksyVlTS7m20SUyb7IHkwQ8Deh1keivxNUY6US8pXzGqTvgZAPGv+os8VvshNZ4gY8\\\\\\\/Bgdci8UH9u4wumYsJqxuyY3zkkUtCAyVL\\\\\\\/APOLx1b+TBLaOwJbFwft3My3MRoFr3S6rD5t+R7arTj6YDDmPyxJFCUDZh6UCNlUubi\\\\\\\/YWkuxTITSjWMUNts7pkEb+uqc5BjZs2A3kN0gwEpG7l5Ow+Z2JzSOVQwWcH2hlcz7\\\\\\\/tI0Y19pj7UnjVUa09MGWl3e3HhoWUeuToX1NP4XZ7H9KIi8uuc5cFtt8cY5w2llNig+ueO44ZM5M7BpeMDauiF56J6aLddN0PdWW8zL0ZCmZClyiGpR4293tGXliYb\\\\\\\/PVnmy8RB+LQbZW27acP6sP16JDGZACTwfoOC5EB4dc511rUvO\\\\\\\/\\\\\\\/4678KIBfcnWwI60tRApy1QxgyvqW12Ontq+S1Mtyu60Lyck6mZCOv9lijG20BLTTbuZMQJuiRxgQDxg2CUgoZ9FeugJZdSXtPUiYv2UCP+fpV4bfXwcmwtV3\\\\\\\/kSyK91isRiMtlqusPxmCDjdWOHLfEZC4r9MF6\\\\\\\/PzCxhve5JsEjjv4NJxvj\\\\\\\/SBzh7\\\\\\\/hNGphWOEZA1KvmFLf+Ss\\\\\\\/fQNqaUA9NQAZ3t9Pk7qcezG+Y\\\\\\\/ulADH\\\\\\\/pNOLDMSW6tgrGBbfJknwUDj8yZHl5ExTI95ZVBjZXL9U8Q7DG0\\\\\\\/COy7YuTZ7F5x6180+oXDDI250P7bL8TbnLEmIO8WNnNUYILp+qm3ISLDsOkfDSunIaXcEu0Se0rVVqDarmrDnpizpjuGwe9zNnHJC1\\\\\\\/IKGS3aBE5bMy3QCCjEECsKRVgjp3nZeHjzEfeBvNBUvx6\\\u0022,\\\u0022iv\\\u0022:\\\u0022YOmSKeXzHbRjomzzTykS\\\\\\\/Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022ac8d6999dff979ec431ce528522773f7e9716bb5c4418d2c476edea092b79962\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00222o+Ou4+tpr\\\\\\\/mfgNcXH0j2ItPzhOrA2yfSQbd+5yGhi6V5BMp4QcupmhKyhaPqg1emorswtTsDWKbScMfvJGRpjrITWgRMbXiUD3U0TFZYfIi6QbvEgV+wTdD2O0UB2cRXaCTp48vWMIjBHUPTW8gbrXt2KELykkTDbGfc6p2PPPdIhvMq+ihZGp3sN3ePzfdV1WhTmBNzISJPQ48UfQmKg4kBHt0ZQ9Anv734pUclF9hiMqPz+8now7m2xD3gxwMenzpWgn7GzrQANCSC9iL0BC27yejF7KzBO41yfgprbPDFZzIiCnC7yVh1wFx3P6qPmFWNfMjtSYWO4Xfwa8LyCBwimLcfGbxgMfb4WmOaL3TBuwtpH1FucP2WbHuDyyCb2XFe672r6R2+1pIawYPb9QLtAO6+pmEehYZQ5S75mDBenFG7Es8rI0wO2h5Q2xiUWld0LvEJXWEgIAYNl8FczFbHrpWH08XJ2ea3DdHrg+taDNDhDqiV5mZCyUAD9MN1+JcvUkcXAC\\\\\\\/SPaT\\\\\\\/vz2zPeo38jgNFCm6ws7\\\\\\\/PqTl6zskSkOVts+gsCQejhqFcx8ZNCDDi\\\\\\\/WAB4v0AInpPrPp5Czr0HCjgoUO9x2nNiNU3tacIrxiemL3gXeM90iPkSqeIF1RbJt4qtuMlsORVv2hC6ycp60+vB+wdWu5fHWN0creUqvfsJ9p2+F+8TZkTIpQsio2xYfmfPQXwyiTf3VuaW98OCWEJWoruTOceGrJVU+Av2iGU035SiCTmpmu9Dg5FUJRCWNsjti40p\\\\\\\/BR\\\\\\\/thOHJzrx7C7LcuBoxKhQiGxSjKlxt3EkYLCDMh37i+GJ1yEse5Q+4L1JzoAVAKpGGEP8RLGTKruOnYB6BFUQd+LSClH6cDUHL6v797VfoKKQ0S8nSq8z+3gk9ktLBCp5HYcFyl0BcbLujI5\\\\\\\/mvkiC6nIrp42H1s16bZ2FSVyEPEnOqefcFEbhX1C2NnzVCgR7YxwHNWVyi\\\\\\\/C1ntgMuuduHdnpmwTye4OYVKt3kmvnqzNsLJUBSaTeoACN8p241jJnouyUBe5tvZVYtsRSNXxL7XJV76uhHLJe3DehXawvC8AZGGr1+LQ3DwAypEvDJ4iD0k2Z2bqt94jtn9Suu\\\\\\\/8YNqeXxLHrYijruVxZxKYi1qjb27ah1M2ZIX\\\\\\\/12iBkYsMAyIFh+Wby2QXZB5KwSsVkzLlbm4QrtpxWGzlIlO7vW\\\\\\\/8PaNsalHELFb68zZDEVeuvOa5BeINHtu4EZj6cbberrJzMuBSjPW6KwFEArmKepNXmxOTjVNrXU7Wd\\\\\\\/hMME5BNifBMS5c3P5rvxiItPQJoPl1rHDxRBHqiVgC3e9Tjg2JBqksdynPqH\\\\\\\/Zhpt\\\\\\\/\\\\\\\/fjYN2BYh17vXeOPWX9tE\\\\\\\/I5xLMQGOUxgmRjxH62CU2Q7Z3vyElm9noGUBdXOk5iN9cDp5Cfe1R1vEe6\\\\\\\/DajXYRTLVIqtTD+V+2hIDkfcLA\\\\\\\/ZdrTsXLwcBDo92micg6wr9kU+kCUqO1wsMqypCTes4z236bJyZQKNNghPNouXnMk9liDwBypZ1SxLNi85c\\\\\\\/GQzNk5j\\\\\\\/u26xycEd6Cld197UihzP5vgtgwhg5\\\\\\\/7jAsfySJwRZ9+Xp8az8dDd10DYNXvIerSBT3S4KjXKoNZoX5ukTHXE513kt5bPcYx++vmGJt2FQaYukudHeg6Kf0UVP7oQKugSQ7rbxmn3HotiqTdxVfjx5pgHvyKDatY+HI8DNF4rkQOyJZQEe6c\\\\\\\/igCHfFJVqolPVXreWWHTkbrXEgabpvAU0futj91MWmIcdRdrFbaxwwLo9p4Pco9\\\\\\\/vmDhuZdWCuRK7hQCovjIM4edyHscTkvC5IdYykhVoaiFK3es+lM\\\\\\\/6JPQjgWLLVEn6mH0vYLUAUurDvOviCQxfcbY6u1Jw\\\\\\\/hAGCMIv9hZiqzqiNXJGqiRLuPB1C6eVThg6Vn7AsL9DAu2+1tlF46t2Pwb0Kit8PRqC6cn3Kr4QUxkJ36tP8d1iQcyZ5kIcvaxf4y9\\\\\\\/SH4Yac16f2+RfIRxBWu5dduwdHGoBJYKef7JI8FzfNuI4Wr4DAJI5Wuy02dveYZcYHWKcw0QV3BDqEacpLB\\\\\\\/jiDmdMW3ZZ92wfgJ7PfBKbrVta518E39MyXwUNtAGWGVNgbuHStuucdoWDIiCB9aZM+eygwPx80ffdO6gb81IQe4JAQL7N4VyMwIPoQw0MwPTA0TJ\\\\\\\/179WfUvwK8S5cC0O6Igbv3kPZ2CpBNiqGDYBzPFCWLilPtIjjZ6ZmG8GQ4dmbacagpOByCgF41TBuqHoCmouHZ2f5uaENs2B43sn0+zXVW+hKII1nxOEIkHSFL7IK5kgiKCc3riNQjHbHz4lKEBjXRaheoh9+n3f+psANnq\\\\\\\/aaRm0B35e73wGX6BO4ruclubQ5ZYOc0I8MM1Px9KqJpwJcPL4nUWAUBNPJWRhSfr2eFzz7Of6MFP3QJgVuuDaDwtSRnirxCGe4KoeEdLzm3+xFsrQX3X9sq3DWb94DqqNn0bapUzTfsThu9ME9kf0BRCByf7MD6tbjvNgIQB3SjGPXpLV3de3w8sBvGyQK0IqEnfzb\\\\\\\/yIX3pwI5JeET\\\\\\\/VHPo4C763+6EprffQsOoUqcia6+mM9MUJ2mSGEGZGQHnqsKOCYRBb34bY4KftXWyejXXLlXBpS5C\\\\\\\/mMEAvEjXzKMh7yp5omjPYOAXw2c7tbMd3HpKJYIXUk93KP7gDNffPlgkfOWw5WWw0UqK775c2kIvMbyIbsh4CiOIaKFXSXb+Vfqv3cOLeQYVxVVxPf02IADzc1x\\\\\\\/BFCx47n5K3tIfel13wnI9tU8W2IrN9bwcirkOTd0BwKCoXZFe5ACDBvuPBlVYy7Hdm9LZgJYAonOyBugvy8a5zSM2XlsF4d9IGIURBUnRsDJXdicUEnb9Dg0ZDPXe4Ynai19Hj8L7i\\\\\\\/ofO2iGr1L9EpVR0nTeuFkInUCrd+Q2Jd2DNYNl7uYVyPSBg337b+W9HuCq3ZsB+CPUxaW77ttu6NpaYlBw2CPzWp4sX2wPw74OJ82sgKjsZIUy+jMSevoeHl4qZkBQm8sgCNAg1jn+RKXdwlMqTigihVHYJd\\\\\\\/lM5ivIlxvJ64ZQHe3LW\\\\\\\/uwD8DqZQssiUKj6HJkTI0rEGzcAYRFdbUqhX\\\\\\\/0vYBBDggCRMgTrtM+mjI30cmHDuoopEr7hymTWiwVvAc1DTJ1PzsRie+r1w7LwG\\\\\\\/QF79JQ4HGBcISeHevnZEAWdlQCOB9N\\\\\\\/ryG2ws9sdu1nEfVtsorT+lR9eqcuTYxD\\\\\\\/MWtgXBlMkr+qzRgPnulf1WGsUs946l+l2JiUQdkClDQkFrXu8omjjBQ8yPAbr7xHx+IJIY86UP0S1hojO6au05DMSxaNu6Ks3drstO3qcnMghwy6mQ9tlBVx2BE4TEboeQhjn8kNNfDSG2RxieBSzGEHvdMa7nRUoSw0N\\\\\\\/xzuRHOH6jLaJhaCAB7hBvOJD8KfyXdB++bGEsjse\\\\\\\/h7VgPl2KW7Vaw5kbRGir0NBSwhjuYuLLoT6L91p+r2WFxY0fTp7M0Rzk1ewCNYWnwkFokdWv2RBG6XnEtsT6RhxXHuLYX3Bb49x6zON1dvt+uTr7SxXN9it2t1aAgImpVWchQ\\\\\\\/gRDmTeX9vowxZhYBHzv2YIFNmFKdZIBvXBmdU23t7e6GWKCDzbDDeO2dRul66aiHmxFKrw\\\\\\\/qd2mxio05kgNNRhrCvaiKE4MIm55JJiSWRvLis\\\\\\\/a7zS32i2drL7uNVUMR\\\\\\\/OyDc4LnxXQGuK7YxgB2ZM4Ll+X6H71FBqvCeoZ39Nxg9axZNSpJVO2H63al2SsZX2RakAYe0rqjZh+8Id+ChKhqKLqsfc2uk7lZphx2o0m4TnFRrMHVUj90vi6IoJde+A1yzD21tnSgxhD+zrqUiylpZZeBxYsAbn8pQI0QQvz9WNcmg1zxeNueK1\\\\\\\/kyRn2jGcFvhNANs33j31SLywhYxDutOLHWDBxtzf2vpmhc0wKa0\\\\\\\/L3Re4LGP69DOwkmzcD0l0suo9i9xU7o4Kw9NmGJmBEUw6kdGaHK2OVDcbwCrDZKoyzw4qTaQIUrwok5+jU3re84Ha5IgA7EzX1m+VhkRvgp8Tvwg2GLmnB4qvZMnz\\\\\\\/MooIBB4OIWjwQoccQFkV0tiiryBxLIxl1MI5Ui7N0IIEPJ1eeR0tz9GVeFCD\\\\\\\/6sQVaRn9cMIYgKs9XtPFijAHgRAzM4YY1K31rNJaeNXVKJPGR\\\\\\\/e8dzt9qEMOzRgsylnl\\\\\\\/bW+RW+4x6SrQ3V23MU0O6pNXqtHxCnf4SnrzxFt18fSg7SGUA827eLJ68pS3wH1hYJv8etsrJPOddFqD2Yu9JXS2FkcCj+CCMba2C4hv40tfajQbc+all2QDR39f3GRdYcotTYDeYaklodDoSgq2tQYM6ZrbHphrLNlw6J2L\\\\\\\/XnNV8h2767gLG1fX7ZDO8b+rKOiGd5vwqn6RL\\\\\\\/l6gbBuZdvS1H6VEtdaSDsTV\\\\\\\/4FkFiIQz9g8FTl\\\\\\\/fh2iLsBvSgs8Qg\\\\\\\/3pd6o0iPUPpzB6qSYMKNo2zL26AJtNIAq6cnSj\\\\\\\/8ZxsMYOjYPQIt0WB9C3b1QAN2rLrgjKL6KwBRb0TmW37aqBzl32PErvIK2H2jWWZ7sebIXdNKKPwD2g3m7H3EzoyD7NudMoGGdrLLNzlvoKL5M+Lwh4nWAH9rOTsF7ap6ollQve1PvZ7dwtua7dq9Wqe0PB\\\\\\\/B8LzTPnMuFNUTnyFOqDSWvIRNz\\\\\\\/eIN\\\\\\\/\\\\\\\/X6nzsrlor9cHIY2xqsn1sNc4r0pV8OsXYUy3i+WOnw6Is60JSgqtnajNY8em4VO9x7WS5nCR0+pnbVR0H+HjUbLYvrKSCPOaiX477RdvLWcuT1Ljva9KMzEUYa9xrQXnEbmZmMZRB\\\\\\\/0aUmgKy73g1hzgX14aeIlvxzmneoKf6\\\\\\\/L\\\\\\\/NddMViSRFDWrMps28phE1tHOcFQbtLmCRNDgBzJvKFpxK9dlTo5KhPmpskz9lDJBCPFlvNhBQigvPwySOyxppN1hidiRG2mnfcQf3h\\\\\\\/fGxdGiHUnyoHnvqTRdg0+Mu3jKFLSnbrnHL3iV0xkq3ftj+ykcYD6mG\\\\\\\/fzqcR0n8zjRE8xcvp1QplWIWH6oujBlOjrhlxeGGfKWPmCBp7m\\\\\\\/eBBUSzjnxT1rUPVsOdun5d06cUbaFQaTMlpSQen7nDiOJnmP+H\\\\\\\/Bxwzw5YFXYSGPA732KCBzVxo+ZZaj4npnVAqaGSm2IXRCbMoX2WSIVvXAQ8BXbl\\\\\\\/c5TEcdwA4JasS+zSYEQ1aCG3B1yVVmUCXwEPHXWo1x8+ODISsupsKBrEnVpAKmnZXVRLCogQ4NcXMoCqpdo+F1Qdq3mdEZ5N8sk4rG1kIY\\\\\\\/GrNR51Eb3hMySGSg+7rWt7lV1C00DL4ELmzv\\\\\\\/iVxr2mlf+It6tzjoQ\\\\\\\/9eiVLNoLh2MkYfSH90adDff5XEpOyG\\\\\\\/WIN1Nz86MHJRVEpcctgvClL\\\\\\\/SVDXTMqquH6Me0TZxPGTsKiPlrgD0SG1Xo6LbawK9a9Hn1CTR0az4bL5xr9BBrpTWVu\\\\\\\/wvYKzO7sNigBfEI+ip99qb6cnEag5CYofsf4x4Ae0G7nBl0SkNUXmlRZYXMBDKnJh4JlQ3VtPCO9Qv248cdK7dMmukyQMULjbAdI\\\\\\\/lYOjxMilenyKTwKsWz4FQy4l16vL0w5lb2TiAOVAn55u8GCVxJswceGJaTYOCycVuPrCB3NJybqIxMBsALra6Atnlmnd\\\\\\\/uNDjqrXOp07mfc2HYGrzBHBpMqlBRiR2Hc7IPmch\\\\\\\/FnOzs1OvSDog1TL8RL0J8AF0RI3XYy37fsvClDlX9liErys\\\\\\\/dUn0HWEfbSWMqnwHUt94nGWC89gMhuO8qTfv1SQ57QpTB80gqKtOxkC4IRFgMBySpjnqCXcDcpoYK8HMBtGJWhEd7TS7+SRQ0nkeux+xKgsd28w6VGrqYQLO18tdnZQrLMF2Cv1SCWwXfZpcGWgkXBcbB+uymtCz\\\\\\\/d4YLQxDLCS2v45CiE0+0f\\\\\\\/dq0mKBom6aKfzDJqZdzSF9lXnVcQH5Fz88zvTN1YqNjw8eA76CoUaWq31EKx3pDfqOyHXa\\\\\\\/uv2K7BKEHVKqCIThW6dm0t3DtcT8PeGGBem3L4LfUEd77e33D+Ez5ErD2MuGcRAJ3S6ZjkL6EGL1+dnDAllOoDUWt06uwISRFf3XsNsZstdxVCFgfBFNZniSP4EdLCyZFBl2n+K3l9Er9GeFi3X2zlfURAqXRH\\\\\\\/ZoSz8fYiQyhf8jm12vQ9yZtxD+UCjcDSySRgHS4u81aUlHyP\\\\\\\/0yo1A40XAil4++DdhqupB3PDST3KVpAdo4mMV1lkIR\\\\\\\/6QJi1rLfmTii8FxN71rkONsP7Qem8XmT\\\\\\\/tC0SMTyI5cGEdQPniYAdUJ3iqj01k5Acgbz5BVZ\\\\\\\/9qPzbymE\\\\\\\/AZV9gxxJ1nUgw1f1OPTTvg19ubizlUBBFxKP1cphCdtFxjr1+OUh9FbrKJbsaxhJ+qpQNTidT3bdI5LaxNvbXNU3hNxUhZWGw5GO7oyEpGld2w\\\\\\\/OxqKUXxdskOTWWU9eh2uATWEmSn5MllalvhhGS5w9rlacxR\\\\\\\/fQCsIaDG6pCfWuUxUg+Ei2I0QDsLmCJJ+YqJA4w0+sxRBs1paP2LWRfF\\\\\\\/UHIn\\\\\\\/4nOlcx\\\\\\\/a6Sb1\\\\\\\/OtJNNM2sL3fkXhNf\\\\\\\/gZpIUZdj3qmJGeyiDhQcvv2jvDyTBUbry54pgfRqjYDyrNFQ5HqZqtpy69AKaiB4uY2OdKhv9rLbxlRnsNVhRwpOWv0bcoOpAPbMaFGf\\\\\\\/+9x5HdPY1LPmlvmEicznWLYWCoUST7aMYmlYC0JRknEyjuRULtw6xmluxvFYJrzYQRCRjc\\\\\\\/Q0SCuLYrXvoDfaJcAxLm37xaknwzJWlyYWme4KB6EBDobGK5iiALOQxUwFrTepNYzDfWJS0JPXV1DZg4mtd6mpF3RcjOFUfgvCDirJgeZPn06Hn67I6vaXTpShYKmPTWgCbD803GmikndGP8Pg+qyQKT4SZBFoe8fnn1YU3P63scKiHGOsc4MvnK2WBOdOXZegtVi0Ro9whSUpKkI+hXiMRC6sLyMgQFHfIrJHg2mxEtW1lu4MI5lC+PXF2WwMVYbkjdQak5X\\\\\\\/fIGxmo5BOuuLAV55HcpILoAe5i9yVL+kS8csWmhoeOEgV8SVRmeFNjo40lui2f4ouFH85K21nJ1j8\\\\\\\/vmFhXblKG6jI9svn5\\\\\\\/KttW7pGqqAtyENeGPgTBXpDLlCsVAF6YghcJ272gvvK9SkWZ8RM3Vt5p00ip4Ujv0k1sBYzoOawBA1c6Mcw0+TqLBMiKg7qXCK9nM55+OdFV3ktdsWZ1ggvtMwZ920IRB9oBNCXefBQWDEwYCfqjqLuY0x3yF+XLaQwqv3CZgrq2Vx+5xTQfH0vPaU9gWSPjtmeEDORbvIh+ooLcz5KxbcZoew2nMnJWJi4zq7W3nionNrOx+r8OjWHrbGMBknMTmdTlHYoaGvbf1sRNYIOROh43E2GpxPguumLXGcBv2\\\\\\\/\\\\\\\/5MGMJl6B0Utm1n5vB6Hu2aTsWY9R2xbgTB7iqqiuD3t2xoR6Wg+9tpBetkKUcfiR849TO\\\\\\\/ooCOtSi83+OwVMLn4068PkDHfkPEbH\\\\\\\/tI\\\\\\\/2A3A54PPLd74dmlbPl+UhSEeeE6ahQCluC1thT9zrrWxhiArrLj5vtamDMHoCQ3BLl1Cebc\\\\\\\/oj1APnO\\\\\\\/xb7BcMCb3OCxFjlPrTwgcL+Ez1KKUxJ+G6ZlkTmi7P5Rb0Yo8909GCMEqw+PBFPLlELVyAZeo3n9Mo6EibfueHpf\\\\\\\/\\\\\\\/I9CeXiC9afJGvRqSguuTkuHga+glChcgLBR9YGOYSJSRgHvjJmqaqekbbzsTUcjAi1m1iPzze0bbCiK9gA9f2ywI1lFX0IQQEoHNtnulEu04n2u1smXBZJ7PUhUpFq\\\\\\\/YNELBjH9oT87s\\\\\\\/LZPYcDF\\\\\\\/+YoZVZ05cjG9X+6UYbBtewRneJQHeIqnFZYYfrhO2lFEQoM2C0I+J1oJMZ0dxnqHVrxJbSgjicf4o95pAZcB5LE0Vvm5xFvM8lo7XTNzltvlSuHU3Cv\\\\\\\/Hs5ZPpiuVgI1vtyi3YFDPkAphsbMPzfJMv8pn2C15h0pj9I2FWI6xT36wibYciXJfoZe3I0kQODW0BDVQwraiwHSeQgWD5hpjXpIMbwyYsSUO2dDTrk9jOSYd3VRygWr6p\\\\\\\/1Xl0cp7ZGrDcB6Yyrd+GZjoabmjwCSrfIM9C0LIFEfK99bjHp4tI1Yc4FkFXJGK3caHur6fTd6HF3cNulYb4bl+rY3s1fjxpnIZqVyHesqVKU\\\\\\\/PZxy2gDCTPZV2UdG2g6fWRzDwGfGy85qIdjoaHBhFxvt1uvwHFPDkX2KZb+d+Od0FtZQ3vcpHNJkBagtcXA\\\\\\\/YRkZe7TLIFQL\\\\\\\/Wx0\\\\\\\/iSy7cd+WQ34CaldqfDUvvOxmSaLbWYQam6mkK3S0J1wvon01U6lvKhqXArNFC6JgjAFDjIZ0LSEv\\\\\\\/lfXRaOzTA+xVj+hHbK\\\\\\\/\\\\\\\/IG5FKf0awRV4BZL2aBpB2Dcs5v5+jezl25UNJoag0z7lPoa+\\\\\\\/d+Br3JM9p+gWgCaMHg1E+cJrM3hlLfcsO4KdHV0fJH6f4xoGeVxrWlw+Gv9K7CVp7joXWPPvFOQkyDxifmhKf+fYBdTFJjQcCholyKae+fTfbw\\\\\\\/m7CMQ2I72PF9kX60eY89V7yDPcMPXwFPX1hbKheMSKn11K472ygLGJWhwhLbfRK0VUrtxXDlyltRxnEofnNO9PJzsVd5mkpwbLqBhXtRmUYL8ZiCepjAg2Z4mEOBS8rLloKTmhBn2GYvTTqD0BLX9Xq5muYPWpYqxoKRVJWq2iZIKz7jdvAJbtsm9kYlCsjA9Anii4BL5cv10DVpqoOEhUXER0i3Dv1luGz87JgOosjGmNx6HE92TBW75kx6lQhSeQPxM8rSGdyFRoVZCgE0HSiAW2qEvFpmPNwd6dIebqRY0uXdQRfOs9ocICJXBVPRjAWIQqZvHprG22SQluooZMd+UPXI8Efl4yr4dTJhANpA9bWUBXh2PLWNsNpjiRMXZ3iufbYWj8IOb+4wEDSTg+8eV44ZOrVcqYqVynownSSBzYbhWqrQXcQRJNBgTzfIdpqrtVPDunHLtr\\\\\\\/eitPjP1Bypq02q\\\\\\\/7ZNQMmiSA\\\\\\\/tR6UB54zW7avsQgrSxwiRi3RYljovJW37wmAWPyQHU7HSVbnkepP0Gh9tj2fyMlwEb5LabivMU6S5ICPp4OVmK8llC8f0+XEZkR7Jfkx3ohwObJ8GE7dMQ7Rd\\\\\\\/iogso2Bul0LO4GXCKvqh4713GikX\\\\\\\/CpkCp8\\\\\\\/8IhgnkBveL4xkVJJ8MKaP20dDfnNcX\\\\\\\/h2TdDvivMf3rzXuc8z2dgdPiUWyGNIJr5dnXyFStcaYeymYTxTnkmygOm2P8tXocXV1+0\\\\\\\/QfiJlXP7v11+7wCPBHGPTw16UGm65spv0Kp0PW5RQl2Sr0aA6yxiL3W83w4PthEMusJagaXcS0K9V26kFhb9aQNvQewWUkk2jz701qH890zOFfar8fNZ3TkvoNevDjo3\\\\\\\/kQOrv721Jpth8LHRaMUDOOgwSjviaKkaeaAVi7LKE6Smf2ihl9k5WIEr6ZWQ88MoFc7PhvHK7lvQmgLJIdF9sn0DwONy9IxBT+jcVShfolf0zfPjWd5+UFQEDMIhUHBaPY3dZK0vj+YJf3cnbVF3Vd0wAqIFrYTKD+YQ\\\\\\\/xavL1q9tGoFR05XV3s3tbxTQu2bSrQYy3PsGG2JJ6MkTrFIPaOgb9Jbjoqwm37WZJyh\\\\\\\/6X7XxDwW2ZzVlUvEK74SENto0dynLovCOXzrrN0dPS\\\\\\\/g0fCmGichSwL7YIyGqL1DmEkT8OK71LRxdKNaT9OxPPskC6YWLPKLPBwvl\\\\\\\/C4haLs2Il0JHyBZvs0XZhlLaDZXiCZsRIde2ITr1Pm7+JoEaH6+agHaI6ReYgLmQA8aKt6lISCP8Ypfk4EwAQJO4RZuOyWAa4EPkNS8tgaOr5gOliggyxO4RR2O1lS0cprP+PEX3t+YvaIxR74kF6XN2wTHH55wRhSkpbWyssSb17DeUSfgtQT+IlM\\\\\\\/j+VSWyz38bncy+T8G0zlm87U9LNkKkkrF8awnSoPj4ByOzsG0H3Y9aoKicMLlmsGi8mfDjDEzZJB4HYTFV6ipgtLtdXT7xj1TrFimT9hBY8xMEtZeDAQqa8Pmzl2SNSt3rgdD4VHDqskjLUPg67nGAYPuGQfRXWpjOlwo4wf+9AshAEM5PQKBtOaGwYzTNfeyMKAnSY6\\\\\\\/Tou4V1Q7zVY5QIBQOlskHl9Io3JQraC4Qj4V6TLWu1SP\\\\\\\/bKmDhByN9HjMnvAILUqwlXuj6pzzDLDajePOC+gN44EMKScYuoDvBW44Q3vRzQQvmqUjyLOO8Ndt27k5KzOy5zsq61Hhxbn1QpMdgEIhEsJ14T9e9QhCzi+3DSJw3fV8FjZ\\\\\\\/LLmnmeRMw9wpIadp4tN6pFOxCosJ+0ZpJEtWyNEQ9WWRQbct3OVGETzR\\\\\\\/12k5m+gsOzwJ2jIwWA6ctlq97a+vfUOl6ydmnhxc50JijOtPQmat7TFE4p1AQRxSzwvgMHLB\\\\\\\/buQGphuhXCXgaQpqsscZ+Lr7Cfyw9m58ZEtVhyRcU+Yjn2ENYZqaVvd0NQGO2xPovgynPO5tHn8ZHFKMGSt\\\\\\\/\\\\\\\/YLCed654tZDjGTw2vAWwzPXTg1t0EThklsXInyxbxKdiA5YN9W\\\\\\\/9oddilHFrLOsjRXtnLITOEpIKghHcOjojJMIDdUVkduYhiKtiO1hdN615Ls5ntda+JUmrdkXum96vebJfalIvdVP+xo\\\\\\\/XJticKz2dboZ13ZFtDb\\\\\\\/QwaBQQgctN5s0LjImr8\\\\\\\/upv9bFul5Xwsqj2AWTqfe6llKb7xgpWkBOYi3q7vuvrXJz5cckBdGAISQ7Sst9VcUNpriQsOa7WGnj093UII1pU0uof0GCb1LGkxrFemMQaeh8wmYBbh6NNW2dKJW9AwBbVBY6acDQleOR1Ycg+HXyNIsYh5+OWHzpzmZb4WSN+yaFQwo4ayQElK86bO9DrCX9D69vpr3v+ksRH5j+VEBGE0Ko9d6A+Zky3SQbAKKCgwYYZi3xZC+CB8Rh9psix9KenD3GdU2jJIH7UalrvKDGGgxvAIlos\\\\\\\/u2FjesDNKjh6XPYeI62DCG2g+rvV6nx44uWGmCHCOX5um5K6Kip1H2SziHouY4o20qwu6iUYLjHzHmY1BPbSn658UDqN1j2FIRYcuDK+4CtWWtY6PT1kxSxBRvgtQHjpBcNwwsVBF0QqCbZkLno2AS7+bS0INlLE4n6F+DVH6vWLkx\\\\\\\/dM3+BWIubmJGdLdPXbiGVqx6mcxDNKL4N\\\\\\\/I7p9ugP8t9v4bItZOqSUEW5Fr54fxkvQc2pCzKt+t9smP9TbSInHbeRc5eTEhhUjV8gBKyalrlb3mHz6GSnYU7O+Sq9RXMXxBUELTgQfDgLb7wJG29puG3IGA+pScMpaYGjHK8fkM8RpoSfvjduRHatOSX\\\\\\\/xr547FLiZFUjzmmQ\\\\\\\/faDFxty4fUy6ZUvZ5WWzIemBhkN\\\\\\\/WaRf2\\\\\\\/doTQEZObCRf5DCgsRugl4wwgK8b2RrGccx6btPPCiIPQWzu0Y0jUogoZnO455e20aVQtWKo8JDCgrDJzuougvt2v4o\\\\\\\/QbpOm723FWcWSmob8foYR+7SNiSJ8WfdlYi5FJGxaahDqvXgN9K7mPTmzBpt3sWAQ5+EWigq6AhaTstEX88v1er2C\\\\\\\/Qx+iJCA7eSU\\\\\\\/1PPQVD4mLikFFhKxJZjvYLpoCP89i8BZ1UjnK9N+kHXlPLK5Pog\\\\\\\/+yM6CGKnxr4cgeWqowbw4RW8fnWJwFQHBX8jG4t1mAEqTsUABcuac0Id2iWbX4IMHpVzknJOsv05VvPhM14z+xe28iE9xIihp9PMcq6QWPEp6l51xb7I8etU7YnugSTiLmOh4k\\\\\\\/jOATBBGva2SQh4lr83YvcO8UrVjddIyU2bDl3p6kFCvVRqFJW1yNjfVCy9Y2eQlR2cPx0UxIhQcJO99rmBRyMz4d3GgNib+\\\\\\\/01U+hr2TR5LFPJkrud1lWugdcX7WHQ6bgcgATIQFctduzGT2XTx0LbVnMbSnkMwjf3uUJGvrnCThX9p3KEjC+KAto8UsVYfLiZ28xuRSQjCaogCWmAQ3MBKezKsjwDZmA82qaECUD9kUHn7k\\\\\\\/sCEiHajSiIJcxZ6JqLe1\\\\\\\/KDpC95OotQ5ed2vHMehXcwJ6cJ+mOdEAEsbHZnda\\\\\\\/QuIZgfFvH9vaf46MnCT+kDFpH5W4bDVnwSPc3VkyfKUAST9fvxT4NFr7+ucKWbA7wqwJ+gkdDdSII5cXEr2Ke8eL7jaoJxcnM995hBPmnaSckmVc\\\\\\\/yllDQWFyK8210IJNQE\\\\\\\/KgWwfeUbeGF5xb2n5W4CKGBJS+N0aMJ4krWHUmvfFqy1OMuMgB1RR+QEtzoYQw95pge5aNBA\\\\\\\/CtKUTBzBvbnp5e3u4KKbOZ8jgW4SPff5lpMFgKRCJBcq4y5mPUQuEIWPdCjxX2aqCPxhgbCUKvlahdU+B9Pwm6aCLKaZwGke7mvwCRV4kcf58WIdjdcg00vwxLJ+5\\\\\\\/D0GpR01yN5hLwMWMDcFRj9\\\\\\\/RPAaCUXiHmhAzoWA8dCid+2QU9o0fpcBa\\\\\\\/Prw2nzqSm5IdKPAg6z96wScXZa3gQkr9orwCdrhqN6WllGXJGGshLuSq+WNSZiYfXc3D\\\\\\\/DlCWtBWLzkYAbWP3dasWANDWrl39rcKIvQw+3rJK4gYr377Z9BJ14gDUUFcD6rx8AFjg4Dx\\\\\\\/g45wxogfi44AAZdEMTJVd1k1lFseh5+6qHRvgk6\\\\\\\/n\\\\\\\/+gDnC42CTiT1iz98ArRB4gRC1gvEzPP4fWJiNGpjVPyuIpAuM4wGWpmk7hRyvlINqvYCCE9YjKD9RnZewNqn1YXjQUpjux7YAhwaWq\\\\\\\/43qi6yIVXjwnHE3tpPgoxZetlC07YN8Iw6Onv8n875clStkdqjz0MFsfpF+jkFha14grGGCr14Y\\\\\\\/tvXICImFx4Ko0h+82KUvY3zG0T1OygTpzEaRKKP4jseWWFw2Gip6WwIdq0dGAoYMS6c0Zija+wTdIHtN+mAlVsbSHZec9oOVphPU9p6YKojxVTHmsHnAS3GqUYixlno5GfULW0tq13EQSq\\\\\\\/MNm7Z3mEfOd3fw9FsRwLABSD+GKns7Yz+dAzgkJh5hc9L8RIi92aGi2y1xmLjhy1wV75C2ZKAdlt4n4b0kI911EmgwvWgtxs7Hr\\\\\\\/e1z0aFZcZSW2q0KEXQWBu+AuPxFHojMeVtOS6oq0bTs8R2Qm3R0rX4iERs\\\\\\\/h8giWxKIpgfKbkrCCGVsIH1iNZEOVeCJ+Yoyq4hTHwLUzBzlPlbsjLCT6n3\\\\\\\/BRHzivFPmeKh7RcBcL2yXkRdpsLq5OCmGK6F6BlgrE4kNw0738N9r\\\\\\\/7PSPoA+89\\\\\\\/4V1MJGuFE9RPpp2H\\\\\\\/PklFEf22bS6kvuF6YYyn6Xdfuebdee\\\\\\\/ya9G8ArvwIrtoZt2G2abmd9gzDHxSaOVg00bIQDdds5WVS7gIES0pDQS8JXYG4oP40M+ISHq5VQ7A7CC4017IV2iwUhPUyccwFzezu2BLRPlg1BsgP0Enofhofh9mmd+0rKuhKPQN6hIhtPd713Xh2b7Zc1xao913AqQmWOdc14by3IB4n3klb2kBkI2l0USXA4kJhVGCi+BOipvKoAgq4tYFyZfrnqLQP26ZvnZGKUMEGYbDGCV0pM7iQ96L2kcZz8hVcSxOzd8RXKuh\\\\\\\/87Ap4BnKS\\\\\\\/kjpdw7Un9wrWpkoAfudCuv3\\\\\\\/jxI2lNVQh0luH2Ir2VBGpsdnJoMg4bd0gG4Mzjk9mA\\\\\\\/trXR1xGWorg\\\\\\\/5nc1AIS\\\\\\\/kGiO3PMyy58u0ZysNxvF8wRYEwK+csLcfgQFTIcCtRIfLBKo8IrLBmiUn5sKxPzR\\\\\\\/S25Bc+So9fD7kGPx2MdLnjbVbnxYnmriXbru0fXwjwbtR\\\\\\\/6CF91hBY8sCWY\\\\\\\/k6yhCE6TD+u5OaP9QM2laY4zZJ\\\\\\\/WGYv96cowcPFknDCbDOpHxHSbKRbyh9UWJhXiYBO8r\\\\\\\/khMzqA8+FJfV6OSQpo3qpcvXOY8ll1uH00gs1r4OVLG2hE7JXukD8zej7nW3XfU4lm\\\\\\\/pG7ZZZ3cJR3TyvbTGcNIsHO9D\\\\\\\/ZuX8I7XgNmRzy2qj1g7TppM+kl\\\\\\\/8eKxVd7UFuFab45HMGxVpnpyCpyaVOsCdq3if\\\\\\\/h3ZuHEQLsBQHpEdc7Ro+kmL1cQvaekwLhMuixTRadCgR6r9AzTFJZX4KbUMKZHvGiEhhMZf2YPINlYTHpG+Kb9u8C+qV1VPSILXRcfuaZE1PLMsaMsKhvrslWjRATO1pyPEZXY4gvKucDpB1s36N2672NTBFWPtVonTRbDnHq8xgprvmD\\\\\\\/w8nS\\\\\\\/oeWi+SdMR8Ma4Y9D9339MDYAcKF\\\\\\\/wfcmhaFbQ4pnwxK3BpS8\\\\\\\/cvDBhmMp\\\\\\\/hKPfGzAR5namb8QRwvPcZ2rLEdIWWxVWB1V8Be+8aagBUsEvCkFwPkmMu3m11nIeGDAg\\\\\\\/Qu6o74YdE0DUJgmgUBzO3q9sdjax70q6alSgCCvX11ITj5hET2mqgyEEJOSmzKdF08ajG6aw4YGYZHtTXnvTVtAdR\\\\\\\/P2aRA0Wh9Scl8zoEqzLQKlbUN9ujoKJmN4qmF0TxbQXR8D+ZWRL2tmNkpk4v+ZpNGIUiyTHc9YsO97aowgqTDmB8FJZgD+CdapxVFG4rffMohaZ8yoN3N4LQviWBEoN3isJF9867wY2F33rXwXkIAqrYkQBNqgXYGOzh4WPw53YnOlXpdxBWic6jPRAAh8Rt5S4G5Ww9SAeVFmRBLW0ztvno6BSXRKinCGmult3lN+\\\\\\\/R2c8hzb2p0RgTrBFyFwNwwDLUNXoGmXaoCzdN3Mc9P8v2xJPKUrIYVNim2a77y2KxpryV1HkFFe17ZQ5Y1GW2h0Hjc9JLyqlm787dbsWTfM2pdLByPkkAIDqo6dainQl7DwT8\\\\\\\/7aYYKHeTWAqTvQQNoauWhbbYetmXsqjzUEf673jYVg9PsTXTyG4l0KczG0OxIZDtxBxnwiSw2\\\\\\\/RQsfzVTBhakQDqsUr0efgqQhx49eFy7fdDKXX8mxMECgjRPpgNzLfibaqDBDoVxg5t6VK4DqTGUIc7nt9+UFZQFMyJMLS\\\\\\\/rK24ZupTI6HNbMapXUOCXBKJJqzq2KVaKJywVxdbgEjPD3zwKiBhFt\\\\\\\/iSpRKQi\\\\\\\/uH96lIMxgD4zH+TfAfKSubWdwRtoKD4vdcGTORxZtNuJn9PA2Kb+Je5ANMcY4kkE6rZ+TVjlnKtQKhTMYquxXusXw8+7U\\\\\\\/M3p6V1DfSk\\\\\\\/aodyDm8I\\\\\\\/oLdp\\\\\\\/HfPrEsgQBVEqxWzvUDPDANALzxrAGSCJl3tYTQD5\\\\\\\/YrvlgvySzTsClRQxiOygShlqHdDeABCuacXPgYmuz2wHkKvnyizINEkM0E9f3PCdQufFYT8a59Ci\\\\\\\/ePBICBtuzKLxZJy5242N8\\\\\\\/PVzOQV+cUpllF84fTTTTPhOroaY\\\\\\\/9jKYH+wGIoxEaGB2tjPgWhalZk04KcGC716+KrPUSPomwLDdJ9Ttl1q3ljgIlqYwKK5y8+K5HwnbWGTUgQPP4v0jcKNEnswNCRQdo1nwu\\\\\\\/k+bo1JXUszhWyHY1FM+mbK9+hJ6tgBpg0RmJ8WhLCHS0MUWgtPMW6XRoIGGYTepodjfi\\\\\\\/KW2Q8mwIUWGGtzotwW44wLtMs33bwyu5y2CV5hPo89aPxVnuP4M4hgknex5L3MSUCOAxhFL3itgZCKgK9MZULGUwAziUPbiDlcV91b8UXOyjEu2jqRKNgWONrQTioG2puxYl6\\\\\\\/fdqhp9BJWtGfrV6yyGioAQt9dAJgO4OgcVDHYw1BoaoX8aqNNeGdlwL2t0SO+Hrn73l6qQ\\\\\\\/2sZihSY9yy4ticUZbZTB\\\\\\\/BEmTjQHySkrWG12YHF9QrNOGkgMb0TAI2MxpaZVf+s0IZHiyBj\\\\\\\/lLCjgvfi2ifhP7O5NLw3DbgV99lKPj+3s3pTP3CezPU4D5GU6juHJDRsYdWG8HGWwtDpDikLndzyv9jb8fuW7CBH3ocxnICvoJuBHJLe0CnMDOmN8SpgA0CrFGx\\\\\\\/qDDNr58t\\\\\\\/hLkgWYm7ge1Zrh9pU4O5p9te2Q3PEB4tbszaAUGcBrjVL55iJBA3+gU4KVXHd9FY19gy8tqknf\\\\\\\/yETxiMQ\\\\\\\/5W1F9NvGkNwqilY\\\\\\\/0+z\\\\\\\/\\\\\\\/P88pw0NNsfKJG3X0a7K4xecxlz5qbmfJMtxMEWlW59D5FnLGAyuBUGahFE2prWPE1TJBSSGqmn8M+3gv9+IpzYGUDt\\\\\\\/mRQG2\\\\\\\/Ma9GsIwyc+Rh0UyIp18\\\\\\\/yqx3rQ9FAy33orASGF0mTIPUp3qbZ5YiBKe00kC+8iAKbo3UaEhameZzJ3GHeM\\\\\\\/dIK2sVN2Ql\\\\\\\/EdQ135hWoBFxfUB+3CcwqI34MpkttvTf0qe\\\\\\\/BH7ZVw93ZaRvr3en5fu\\\\\\\/aG9dTCd\\\\\\\/b6A8yMVCQrM3UQFOCq6aqGimAmA5o9CEoJyE5VEiGQ9AXpxxed1gEdWDG1lx2yMkIqf8Jy7dQwdvXZAueTYLAGGStWpkbsDx+ZDddBGdk7c4tio0dIb8krrkZWSdAc3IuUO\\\\\\\/OEYDqc0Q8e8nHzmfFdV1crPOzT7KcbhDqqOaJcq+BRJxou5rZdAlkW0p6XwhXKkkJS1i4QmNzWeN\\\\\\\/+k8pmCNR2AkCQEuWJffjEx6lO447r\\\\\\\/bdMpf79gqYW3cZEMewF6eU9cCu4OhQ8e37qTQ51+Bbs4GPBWtW\\\\\\\/vKHzoCJbJR1hGpjXoUmdly9VX3UH8MhkxSmFXquCsG+K5BCNkg6udVNJ3fOWS92mU0r\\\\\\\/fTcDROO2v7OQK+tr9sZYpFg8VB5QDkIQPGEmR2VQufD\\\\\\\/vGAzdtrm3BscHquqYZd4Yktvby2f7q6VQto8gcfLbiwwZwy9xr2kG2MswILG4ymwrO\\\\\\\/NlJSGZYl8NKf5LheeCofWEcHoEOvcfuK7ZeJkV2+vNK5U9G1VOlePbWbaiZdOKfAT+iByweQcWlSKpw\\\\\\\/y8wjpV9VHQj2qWILQesQyQpM9IOOhuazkK26AYgnPGvevFpmsv6NYWqYXaga4WI7OVEroGnRUr\\\\\\\/roSKkY+ktF9E3qwj\\\\\\\/+5H86I7cKpsoA6izih78JYaFcwH8e1fDhgmiU96E\\\\\\\/VI7CQqkSMbjG1X9dcMRm7CUWnC2IU6vjk2x8BMzEQTGRqY66ghKwAuCrZ5wI8CJCKtp\\\\\\\/EYc+e24ANvVyZWsylgPLywx7nFMmoWaYbIgo8NgErEXIvHZnmvPkpCpLdGWAISK4svr4dswiMAlIeDCu44gV89DDGYAq\\\\\\\/ekvkud2zb1Pc0ax4\\\\\\\/HjJkul96VN0nDIwguhj0hx3Vw1bc5oIAk90p8jqJGEEDRuTJkHTOIth8FMtt5AS1h6uEsMkYSosK8pMkrIdBh\\\\\\\/MR4WfJMnH4bLZ7FR4PT5Ht1itJ3x107wUbqUTkzCPEiXBMOrGoyiLaQzLaZUpKlgVP\\\\\\\/g0LjHlgiV0\\\\\\\/FaOqnewjWVI3+V2BIvTBp1WJziYYKaoyIsZocWy4ayNbkDPikXM0o9l0IaBQX7Sy4sZgilH+KzzCSNmJ9RO2o0quuNJ4\\\\\\\/CX3XaoVst1LGR7DFujfKI9ctYYIChlqJFxWIaWfO241baYDiFbtRiKnpJwH3l9yVd4NIaKknIiLgLDAxzZ0DHoTDuTFjDBNtipAwtkSmS9uM\\\\\\\/wbz9tn7bsVdXoUhCkycCeS0nYsFBztDxoRxNe92+funryeWJiHafNQjcoEnV1QqF0tyoHKJFpQVs5PGItlxt3r+LzD4HJtGEBstHPCOHDbshP+AcnooSKPj+glQJQuvK4GKcn6+H1oUHfQGRfLLVfTN5KjOUxz2wePS4p56AmKgARHlz9JsvJSUpjZc7MXPZNlYSqoLa9Nla5IV+2V35ePxqwdPh\\\\\\\/wWomlrw60oYi1rmCZIVjpN65kes9x71kuA7sijaY9ptYBHRzwfrAWV9Jvukxx3S8uuOQbtxMOl1SVOXiPtbVjNIU6dfMfTVno4FILKiIi2bVnTP1F3QllWYnTAobM4WPQ6yDQCDzmb7Q8jM9wJWvEdcdi59XRa14DJ7EKhaCCRJL3tVJ6R4HZoWQrhBMq\\\\\\\/LPfsc87PvNELsfMH50vfDwDeTB4+v491wUQXCNMRvQfvRIcOEse4j9WvSOpmUojpaZYbEMFbsIKPwpl+lPgWgaSFLVInOyo6mv+2yd5gbHhPNB6q8FvowNI9ItKqOkK8dyEj6Ag9NVmHYXrTeREUurEgYb8RbnOOnIKCH274lvQohAIYQ4smN9JVifqR\\\\\\\/8IFty5o8ykYzsHn2EhnZFgV\\\\\\\/NqbIVjN9OP2brp9V5Z9pZupk0129nHozUE6EEUQz5Md+b\\\\\\\/NPxd7aV3tVRd4I5SWXZi\\\\\\\/69xJtB0zelohANh70w\\\\\\\/r1TCwOQGZ2y6Mqq3XZaBiVLqgLRImYi61q3rEefw3QwFA1RECbNaXb2tbsRnHmtLTUNpe5joffmuytaHf8fLlc0hJBQX\\\\\\\/iYiIz2ktehxmsaGQZ0ehTjRNUoA4aWYUEp96FPtEHRHG8TPgc1WfYDJEI8w+ccjHh5GpBN\\\\\\\/+jqXxvs9U7lJLAUMhGtPzAfq3y6pGKEoTU8XbEH8RXKghO+lcC3b7Ig5TKg4VR4HVzU7mHJjx3BFZGkyHvsbAgLkKQvCVHhS5Md7WVDG3ZfDDDADByThUDKtw6C97Ux+HYSEXbxRmgtiewyXoD0MmgXlpJ\\\\\\\/lGalU5vFo3rQS0kJDdrR12Erdz4esDTsjVtpwxkg8PlZW+grdve45lVIABhSeOrLDSTfk1+Qdd6ChxZxFPpupxOD6QwIQDedzm3i0fAexjkTvc3F2c51ROOsAfv6xW+h92layxDEFOWVcJl6GdJ7zU9d2aX3EB+GmsqUaxLT2N0i5kvG1Zol+MbWQ2iu\\\\\\\/MUuh4sBE+Wpsv45Zv5tiRjl95WLLkG+mRH4biCe0e4YgVzlfgCFsSTVkEZQFXAJdQykpT8TX97\\\\\\\/3reQ56SBgpagqT103EsC9KJciZ4\\\\\\\/KrgjurDzXwXVNXcg09UsFNqKIi1XLomXM\\\\\\\/85e4kQg8JTdPOhgtTbkzCIk02pxKaWpGb9ygqu9rYmfxAHRDYCAhR28qu\\\\\\\/dNNOczAp1ZYPLrvS1mLA5uuIjZzEpFtSiWXr078L55pO1y6MICkd7rE4a3mLeMVAqHPl5DuKmhXW7\\\\\\\/+AmUF9vWGpygzJhTACaIS2G9hMvqQcqWWg7WX1pDelHTAgsgIVqybH\\\\\\\/4WSsoZFH\\\\\\\/ahcSp6fgqJlk3e32Nmx5JUu\\\\\\\/DZIUZOXyLDxGI2eh9sJD2MU8ZEVyGjldFqwMvefwnhLV35Uq\\\\\\\/f\\\\\\\/hRAhE7fHQzGCOH3O6cq8CauJsM3AqF24qIhGe3VSDqgEb9vi8+8coQcD2d7lL2tXKMQSxsVcOFwZKTTeKaZLx2RSPi+hMRF3AI6sbSuqvKWyawpnDNfR1uh8p0uonxiOf7gT6Jb+1ucnjlOIgDT9C\\\\\\\/qPKxVeJ4g7yDdSTtweeeTgBH77fVhdDeZ4QIuqV6zQcLURhFgBKQRcs6A4ggROQO5lxlMWCMSmGsyRVhsNqurhaw+Fq2CdkpwvsUtjFmqrypagATHdHStJEVPfpydVnBqtN1vmWfskx3aKgDZbKBWV0wbqQtJv3XtRV2MkIrzq0mWw01xycvGq14PFnBAfC2Jkf5LpxgWa1rAOPGFPhCHN0APrSIpk5DkAtD2o3aFI5rniI260uiItyNUio1BdIa0yN+hlNXgoSXNB4mJY3s+V8loFE37QpwK67B893tU35uLtSyt4kUokIlnj2sOnNt+FUxTt9+xEjlNmWevC1EUB4nDyNJ4\\\\\\\/9pV3hoK7mq0G\\\\\\\/v0\\\\\\\/b5nFbJzG4NHy7p0KpqWaKCeNyDwaa6x2rnNst3GsNjDCaKdxQz+I3pcJ8T2rP9F2Mm0Qd7sVrrYVLa6lLtsYn8rpx9tMUZMU89C+TS4oEBZdF+Mmp0WcPl63PvYDuB0401qq5UQp1Bq3X4sk2lncfJX4SF8SB8qi5fA2aWWxhOeF+4IAOueAPYx1py+RZvcJU3u58ioGcxASnfodlcVorFUrBk\\\\\\\/Hwumq\\\\\\\/zO2XqVlS\\\\\\\/yIwFKn1tEwAojA\\\\\\\/3RZs\\\\\\\/+d5KiX5oZR\\\\\\\/9DhUBPYALSEbNi326S4whIDLfghAUhzN1X\\\\\\\/7WRQ+k1iHYcpF72aVQTq2P6LYHitaSG4xMqpO7P\\\\\\\/W4LsrmfZR\\\\\\\/8UsMbyKnvbmCGGeCoiGqFIm7nZZeOC1nKXxYSClqqJKm2FyaJKch3hCKkLfuNEw2Goykm1WKOYzA9W\\\\\\\/sfUbMwWI7WwTwU0sHmi+cRsnxa4h7ZtmtxX9DZfGjKINcH\\\\\\\/diAwf+qPkKxUZFemTo\\\\\\\/132agYXmnxcGk4s\\\\\\\/4pLatPgkzI82PBuQk1D5TcBEYKeHY0g6fHfq3vxQZRq5F0pCO0RsT7wI2piynoVB1ZO7qRuxg12sJex1Y+G8Ac2GCp0ojVzNG0CeFAF1wu4kurEJC6+SYrrlMQ\\\\\\\/f6EX\\\\\\\/SHm2qdkjwZqqHn70fBEIS5O93CPXDvSa4qrXugD2YdxVum0AJEldAQX04wfXKWazoKaeV3F6YbwP5xRyxoXsEGRgIRIB9NNyJbaTobZV\\\\\\\/Q3l3O4G\\\\\\\/RrXZbPKDKFMoIy7eAOIiJSTM8n5WZ1UmMe9dzpQHMzYsl7L0AFZBQxeMG77XlFzXVgu4yZ6\\\\\\\/T9daAp90fV\\\\\\\/Yiaj6Xu1nygqWYM33jb4vyZ0G+ZSKmCKzLK7LVHUoDDwIkBRJuEPhv1kriE2KUUl5tPKjJdVJoZwk6bTmHBaj00we+G+Tlt+KH9y5F4OUD9x7e7nSv1R9P1p9YXoV6P1bG\\\\\\\/sBce8TMW5lQRrm2TIqTXRJ7RQT\\\\\\\/\\\\\\\/hTjtdgi878s+OYmh5bzC3YyKj2Aix\\\\\\\/acArC5PM9iPAmo1K4\\\\\\\/KKVCpo8qYOkVsttJkj\\\\\\\/Qbgc\\\\\\\/j8p4llO2vXPuaJcgU\\\\\\\/HvPwa2temjWRKLR2GHpT6mfdU5kivds3Ecg+UyYvgug8exGUYs8PRkfGjAGSc\\\\\\\/POOUzSORjVN2etSgKSSmDINL39zwUVQT9SJZnzteIhopVJT8OiPNNy7w29qBfXM4O5maWhDnI\\\\\\\/GMMgCFksvTXCkkGVYoV+FnDQpEB22Np5cQ6STQekd+vCmuxG7qlN48wR2gBPze+dgAmBbzHBju0ZezGGNN8kE56bSDuyp21z\\\\\\\/TCwRXpxwsPu5T+npofKZIKEzrBA4hDLd7nZNlaEoJEoL\\\\\\\/hoV9oH6ixz38ckt+8tftfR1ajznspSxBjU7YJwozyoTVjSfTyw4tSgU3MY0P+9c\\\\\\\/t2zuXZ+X26EWB0uYC\\\\\\\/sCoXYVKsdprezQXx7WRKtH1A0VZh6xzOmFMkPgRd9ksjl6GZsM49+8YUtRtWdbaM8Kr6mjACLAW6wJRkIAKVmFzDHW6n62d5D81WBlGkMsUv7QDDJOAYGgEEZsyshNgsDs5B2ZBznTzDjbxR8l9Z6cvZiD9dQJ+ACY++wBQfSNZKlddiqmGVtuILaM9gGSUFB\\\\\\\/80A7rLUBccWG8+Pm6bkaGhscHjefg1iygGISdt6i48RDNSbylyAi6GF+WKOIFbfusrlxnRcik0kCaXi\\\\\\\/C6m7qp23M9nFBmUXU6D+mYybX0h2s9sBRzb51ZitwFtlVY0Tsy14wPqy9S0PuSsz6sqlBaPSQQrR2NwcO0dijOPUbiylybEtRvTbBQ4Fxl9Yg2TPNVzxd5xNLBn2Asm2dRTviyz2xLn6q8TCioFtTSDGylwjllGKsJ4BQEdSD\\\\\\\/xcjqL45DDGU6UtkCaA5pdI6sxsnM6uZ0Ud1MkCHHtzVW3kNuMXPQxTtLuiUBJrEPwPLCIfn3SoKkcdtc+EzdZgbFEhZygF+hznUfyhFJHnVae1gw5kfP5fZUSraLVZ+59P69gMGzxTT+B9vPo4W9vTYfFDu0rzCgDLeS2vwjKJsmUv\\\\\\\/Vf6DiBBOBZv1YcqUNMDtEtUY6lomD+ydvcbknP4VRucPsQMgl6YovpG9Yx8DDIBw\\\\\\\/\\\\\\\/dj7SH48uaqkQ0wnCelUpv7ZcYDMp682SR5hkOVUz74f4WsfDB2lKHJDmkonSsU0pLk7Lk0U3Z\\\\\\\/mscJ1tm+mZDIANisVum9oVmjfVGHP2vdN7hgE592Gewor3DLF5YdStf2jm5ID4+s6lZl5s9gDQcv8CWlu7LUIXwUMb\\\\\\\/CaAMYwGXYLeOet+f2V5YhsBaA1xjzQfneO9UfsIZMimL88bdvnhNcCTCaNFpJrsVmdNwd\\\\\\\/HFHlsX+tppCCfHhKqYcsgUC8qk6zDInNLKYROb1owgJBzRj5Lj8+pLGnWlnwT8XKoV1X4XYB8KhtsUrHBK3aWeo23TAZvETZ2WPj5a4W0erjTpzar9M3KdKDjkuacNkMs8MlYZJxUFUeAdRWabGhJ+1g0EyV4Ji+B1nPdpKk9tFeCiHKTXxxrru+oRav6Mw4fYtkvo12MffpDZQF1SCZwk5gQDtW\\\\\\\/s12\\\\\\\/0h\\\\\\\/aytbW5yc5cDykJrQadsR37HDuYc7KO4ABILs\\\\\\\/Dv\\\\\\\/+Zj01F3SzHRPkce7VFKkIqM+aOYI4wcdGSgaYlw766fQlREWo7zxCDzJNSCuDMleU1fMO5FvGxbs1ZB1oS2EFlNdz9\\\\\\\/wDroD0EJMhhpx53Qe18PQK5g1WOoR5xllLnwdPnIQBpoZvT1loHGphgPjEQOgOQqRCy68b\\\\\\\/HmAeWkhOnxdJroAfBnyRyLjsj++VKMvsLf9wqEeXRYUM+HF29VEFy0\\\\\\\/mNPtcwBHqvyCTInl3iP5oXlhslwaFrYAtL6S7I9eJRQewdsgPTYnb+aO9RzqlMOU2Rloino7gslR14wupg3mDk6mA0OrHJae4KokwBPr9igAc7g0lTtwijVljbOzlVmmyJeOEgCx2ntzoyGnufaLg1mdA\\\\\\\/nv6dxJL+QFUqn6I5dwC5NA6spQRR3ygT5bK8Ten6FUaQ1sGZzxZ\\\\\\\/oZJJ6wX+Bq4k0EniM3Bc4jnVOsk13+B\\\\\\\/x+LR4xQ8tLS2Bby+fIAIXigmWk0ctQWNNFNQhReKpQbVlAEdxfcB6PnHFWrwi0m1duoOAlLX8E4Yl5k4jj1pDAtMGutmYVLvJrN7bxx8aR3GLcU39lzgKQemVNHQwPPkHplNOTb3Ym8n2jZrn9MvYURcW4EeUWD0ESE4B7sZKyzBPQkyH\\\\\\\/JXSIzHfbXibPFSumd33PChKVFB8ZNHDr\\\\\\\/rH8BcDiWN79ECR3KZzaqjJM3ba5rid7wFbssFRBkwhZvhc41Aw4fG78ZyFKe94VteB\\\\\\\/w3vysuHoAQkifHgAjm5y+MHY9FTX3jVLqO\\\\\\\/dFIXnhvK+ICoqFtYP9O9cxTtnOJ6L89dwRbIp3qgKfQgDH\\\\\\\/ab6JNCxgRU3OIsAAKhGnTDuNRbZE6Ww2LsWM9oSxXLTIQMGuKcd5iT8I6xSnHyQoOPSJsBoMovS28N+yOseiZAl9If2a5ZnkmD3R5A1TsjEMFCM50wYKP7KJUuf7S0nipUHzlMRwCZgvASiMRdJ3LZ93Cgiw1RUJMnYS1M7DgqD3p8ZIbYRjNxx\\\\\\\/YxK9yrNkuFkJgabWzgXtLkG6uQiX0eJhh5Aj1Tmf1KddA8PZlARPiuq01WyigmyXtLe2jIrG9L5KsYcrZdho63BL8Og72r\\\\\\\/JBFztdrlHuFA8AQnTYqXatG8WKLyOqC62zHpJccnvgkbYGHqHkEFz6Ev+n3nkUdg6D3lDVFdE4xXsZ9yDpUtOCX0D\\\\\\\/CPfTR0LP7wu5eoqNEY9v0ePuj7WB9w6yDu7Mlu4xvBXIU1AQTAmGHvSTM1G67cPQAFQj9dfHJ+DTl09m3z2fI0ZB2t2yRJNa8uR5k4LJx+R6gmlokWqi+u4KQ4bPG9tvXRTyxfGWqLv1q7wUwyzTBykyQhkCfPxulFw6t9AE+HPtHc5F9rU\\\\\\\/gJGopmtAEHgmEuxk2JmafBp+nay99u+LOLgxEWdUCPWv64AuvDkdV6CT3cFbugZd6m21KS0JoSG8n1lWfiZZLJ1GLOosre+rsTuJ5xIusu2VuZIIVxHYVYSkbz7z8MQBnxLZjsVW9Y3dUCY0KK2Xt3cjACb1CjTkm7sH2yzIgqrsgkEewIwb8bAGuAmY5iOhmVZuea9S\\\\\\\/anKZCLssmKtlZ9sHF4vSRifWXZEGyA3KQteIRLulbX39T\\\\\\\/Ulmkh6bqJheDNKfD3\\\\\\\/FRHD4Its9\\\\\\\/tT3K4QLy7+wnvzbrJPBLsy+HgeXES6yRTbRWcS5SqTydu9SU2+jzlIHitxO0Krz2ZtnNZ1BFWWHfQCWDtELOaodCVyQyybahUk3+mSfou2z\\\\\\\/lUZkrBJ17RYteL+HdXihOXjRf56Ux2zxg3gOAfttbaOZ0mj1IMqQBfWnLOUFvpUdhuA04CkrrrgbbSz374GanDST5LwkQA6Isg1rhPPdcoLTOxRlayOpAkwNCrroQEe3tnZU1lJeougVWVQRB9ZPFbmdvKhI6yIIeE58vuqTcFY4AYjG4abA9ksnPKlpBuFm1rhP0I0G7c50oabmhA3mz8HpBZjVABjYqhsSs+sMDL8MNFRRV+Ectqbw18SG8ELs\\\\\\\/1+nLUF46AqZOoUw773DIoMSkAb5Kl3EmToRfj2zlg464cKVTyxQF7W0juRBDJ4HN7jHtZBaTkHNvLGl7AaKcjvVs8815sJaBnprJ6OqVkzrqMyMdBhUWcKqrlIDlpDZH\\\\\\\/aCDlsQ38JWE1Fuz\\\\\\\/dWUu97zxhf7FERw40Lk+qxtaXxstRh0f\\\\\\\/Is2m8IRlxsuSRdhl+fLnFeDFZA75RbJfG5R337ytfhiox1FOhH2im3Te82z8ooUfa0bm6UYS3Do2z66sstxGKQ8X\\\\\\\/MKHsNiBY3qunCYHqlrhifEeFj5M6Q5HWwnt+SrE23zQq5Eue3z+g7TYP0UVLuwEwGN8BdhMMMYdCX\\\\\\\/fAG4KSoZ4nIspSjsEDu6E24cfgoomsNK8pNqBoFEXR4zP4OWZOMKlPRoKLqTXhOaV41sPFZ4+q3ObG0vUwa9tdAG966c9sIG8d9c8Y8b3+EKbONrZP326uoEw+WIHqOCvcvGDvYHr5Eqhm3QgozWJs0UuBSnIM7fmK7H8qLlnaLaQl\\\\\\\/RkdAL6\\\\\\\/E04mT5id6uBw6ajqyfd7vs7Kgp8yI5MDWnSKFqIvJNJo8p3Rk7BEfiyhV5n96gSpmN+1\\\\\\\/5a0vS3zdOZ0ZLi+\\\\\\\/Sselydt3OsdxRWV+83fjcB2VUbdqWXGCr5e3+mVHz0nyoKCfuJoSzvTVA08mUmNlM18tkw0FG4M3Kluv95u3aqsyH6X2W6nALn9hBJvHQnl2muV2EIKqe3AOYhRwWmWrMPAi82BNXA0gkrmvsVOCGd\\\\\\\/1TagFZrHasmDHdAxtyvp8bzslOUcLqT9xVpJfmb6ZhPUwJUKK\\\\\\\/EhhlIDLk9vw4ji0QYn3fXEI+rLoqHuUq938PJsCkTAxqnYN3QsyOYfsd1W2HWugpaUfIckeozg21XO63OmPo4ujZV62rKKzpsAwEUubA2c57GsR+5f7bpH3aTtkkoZ0Z18AST18gLbqm0U1XavXAHYvsRX0Kuk4KXizQgO3ZydQCYH9Ynqf33IjJ9XB9aD\\\\\\\/tN3\\\\\\\/PFnXkBDopGbIPKOzjGH46CvtmSD6OhISjoV9dVBfQvZpCbmgy0MRN7R004HBnRGLilyULZJkq8sFYE1QpmZlW+l8P\\\\\\\/HTTRxoCpjDm5usqA9NB95JhWDDdZvEvEnfkWCsh\\\\\\\/ybqPr7JndsR3Q2mlBuiP0tUCw7e4bcY18Jxa503i237+CW7519g8Ou1sN4B5z4jIV57\\\\\\\/W7uzCJeejQ5eInSXw5+dFgv\\\\\\\/9Dr07dzLi9NJN13+EfgYCLIQZQP2qDDrL77Nf8bAYK6+aM+MlGHLq0b3Oks+bBlfjCHoxbdcLhr3P5CnbB9KpUgT89QyRpYvh0fS0ureKr4ze9ZhjCOdbAKLmaK5jTAdsLnFRL5QLvzcr\\\\\\\/Iyu4u3CEp8YN5w+VnFK\\\\\\\/rHRq31sQqENm5332XN2EQ4cHuw9hyXZaWXgkIdU2QlvTP4f91xi9K0\\\\\\\/64aqhEQnCRDzCII5U3JdzwiHQrCYQXDXY1tvRYvB02Tf9tucWf5iHJVJXzUr38wswr84XzNfPHsj5HKXRoxbR9MMyTU0Ybk+NbdBHhMGEGagDqdfTvH\\\\\\\/i6sAjZamgY4of30CbmuN5pSadPf9Hcj+RGreSMkpOUUqdxxJLOkSwT2l9Ipwkg30vCGb4NTdGD+WPjz+zXvWCKPNqnHRx\\\\\\\/TwYsRcsIsWFn\\\\\\\/tfxjy+VDLmRz+zjxyozG\\\\\\\/h8QtleIznPhH4fRVZR10NRYBDup2D3II2XlXquwSyXi8jaX1tWOpZyqc0rr7W0O2wXl0M16vNODXkIGxY+MpbIsVsyzFPNU+a7LLfhaTeLx1Doid97pphNJPXsKwccB8W2BHpkCxix3x2VtQSp\\\\\\\/RL8s0AtwfwEDCA0b7H+MPdbi+a9zjHSdaNgfeDRNS5eNEivbtYbkcoIX2RB7iRZkSX7ClgjA0IJZKtZzuUI5lcPk2ODdhtgyhTA54w0UpdoLPnXKsmoN4oS1oWCVTbjy8\\\\\\\/uevEqi5Gk2X8Z1RsiUeIkXUTT2kM6Jn7khrAZTd0ObL9ejGheE\\\\\\\/Qr5JFl8SjLr5QWSJvsnm5GE5pXaVr8ik4O0iy1nZTM9zFijGPCdY7HoEjvt5vIpwoiy8OsCEEUICRYdSjpbZ+i7lReBunBvJZft+XOd95iielzCpfi7UoiIGtNcQzL+lLcOK15fph7LvFgVlMGDWAwGbvS47ZZOD5LnF+mxGtgZjXkRK0J+gp1ivyVtNmtCAsl7SD1JN9pF0ANRhUV22d1+4XbWEjRiM9mRmfpY09nRzHIaFQCZk1Gm16rwQVP3U\\\\\\\/lkrg5OiNEwpmTzfbOO1nfN8gkl0g3Xq4lOCqiKaOXidbuSNx5A28ZBafbkfwDd1xGESpvCy\\\\\\\/wsbRKTusq78wwP01lR\\\\\\\/3XX3flTA9T3kS7mcZ6HGJiRD8Ec2M9tMEdW+o7bWQROTvs\\\\\\\/j3nubCRKs4oUqnrF9NjmwS5905gHfNAB2+wxw40\\\\\\\/yp5bVfB8s5Of2J19OkffJ34Of8tubW4N7X9yYfDJsn9VYP3jYCORDftpxVcAbKd\\\\\\\/s4NC21YRAZY\\\\\\\/bp0dowHi1iyZPPdzJYV+vbwJ78CFa3yLSbkVoupyFurV24sJ2U1XILbaTtxceGamg2V13T\\\\\\\/gf9kdhXdedhhVyilBzTpWt+ee2aC1aFXwp\\\\\\\/clwPJRHmwTAhOrVwwdzs7zpWCeqUtAjej35Lnj9Zo3kvz9nvNoASL4pYDr9eFc8yH0K8bG7Wh8EdU3tBNo\\\\\\\/yTpowROUT50T4AJvtHc\\\\\\\/TTkKWWEzS8+\\\\\\\/b0mJ66NJgqqF5bTCH6sDDaSLA\\\\\\\/O2KQmX9tGicxrAR0q1ddVpDOZKvTt1KYzQaGwWyCv5snSkeLSnBsETbPPNkjIxLYvsWgS5LSDgeOyp+kD0iTL8XRNH1Ck4nHIX7NAEeXMzpc+JYnE3H6L779spIGHQnf8SG\\\\\\\/4TOLIs2ZP9btsLzp\\\\\\\/bgSuFLAXhUgzZ5dh4P1aHfZyAM2XjP5dAUxuzCPApjzpsvdZFQlC7qNA0StX60NwSVDCuB4D7fX4QtRtcsgSBj3cKEM5djfhxlt\\\\\\\/KwRHDKAz8VFQPWZC5aByZdTvw+9OkW6KHl9oqmTY1BiDWiDTGLlfY3ofo81lLrz+HhI8Y\\\\\\\/AQajvOEFpRkY2HQT+udAwu4Kd39GL6PQkqVH8BfrG1SQHfr+ZhVqHC3cLn01Id1zWKrEJ0sNJWKNbxJSRucrCXW9eV68ckmDeP+A3tTaI4w+T+OFO\\\\\\\/CgegvfCd\\\\\\\/dhZZf3fI1+gwSarjaqM1K8Uai6TgmwBmwte3SZh2XizC779dHsxYL9oaMUXSbGPfrRElt0dJB90mCaXvlGIhx6zQQGLqvxZQxHO9Nv2lcCVHqfuvyAd0A40E8ZeVLvX1mGkYbocPrf7ztMXBlw+Q6XQT4Mv2e1updlHS47YAlu6MJwJxMRDLBVowVmsI9rD8FeL6M2MNqpzG4iWq5PasPafT5hHXAF8CU3945qNw+xjpQC7yt0it7F2pzzF9HttQaV3\\\\\\\/tqmEcWcQExF7EhfcYkhW3UdPzTBoYVrYpL3qwlRRHyIboR\\\\\\\/6QMUQuK0L8wuLrM73yoLZ+p2J98p8Twdfx\\\\\\\/O6fw8+69O0BPAxXFmSqZbQV4UHcDQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022QDBkIo1hhmNe6oshwnOrhw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022dbce276a1a4459517055dc276037ff1472e0134f816fd7c82a8c604438937543\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Gz5BT1\\\\\\\/NN8p64RgxnPgcDEZw7U9\\\\\\\/OnAA4RmYHSeS9Vcq+04lK0kdpYWJsqrrjmw+h0bhgiHtVQglHo7ah6JP0e6FpPZRGEHH5gVqsPtMN\\\\\\\/daFA+EdKm8S7743BwCU4qztmsYj6UmuP35ZgyAmXbfG40M9vfJSJtKmY3UogovsPKWDy5IJGPFnax+oGmqDZaav7NQOxhwrwDaseDH81YWHBDvVWrX8I+X+EGv3v8K+A7YD\\\\\\\/l\\\\\\\/E99PWofN3PE9LwN5M1MuV57slL6S4PpH9s50QKeL85MAWMdveT6hMuBsrolzb\\\\\\\/sSanFFfMHpgW18ZHZdtrt4KTBNNnPEkfpI4RKhbcQ9Y8zt9P\\\\\\\/wTkMI8Vdwnq3iKXd4s2HPt3c0M9EuWDsf6SA3XxSOQSELBxcfDgglA\\\\\\\/rMT\\\\\\\/UHF5g0qNjVqv58jPbm4ZChZDpyOQuaC+jDjAcKsXWcQlpi3l+h+KHutQWjToWQcdu4esciCDYJ\\\\\\\/cU0Id2vCVuTmGyhvv+j3n\\\\\\\/+6Ru69b2Xzve\\\\\\\/6qWo9sTd6Q+cN09g+2t\\\\\\\/0lgcWaHG4mMkf0X8c4d0dadyIAxhKsFkzrEVcoiuWh8iN9jJlADieg3jPTzBFcecehGd+KqdCM180bZT4GdJ10GK5NU4ShbrEEHxsQTWeziM6zfxoblX4lJ0N58Dt6FpXH0Y8yvkHegWdNh+h5tXKnmT2GHFe\\\\\\\/TwYVKwHc9Ejtv0WWNiT0RN0LPnquuwJtKPJzDmxwiIQOYryiRjCi9ow+dOXHCRw+aak+x+aK5yVW8UCRJgin\\\\\\\/iahon5n9vvjM1Ci\\\\\\\/pJ\\\\\\\/pnxd\\\\\\\/chmXR8l414kvrM\\\\\\\/QRJPhL56V26sTjDSVxGJxFLI+1RmwgVyUPMW0ZDC1z0qknC6377j1FUjqOlLI+IenhqXBxyUfLU1Zp94w1xJ6BCKsjqAG+qQdJxaIXR3tb9LNwpOd2z7Wfu\\\\\\\/NiBj+jj+Q26xue3abmwC20djbb80lj32e5wRk\\\\\\\/APXaCOivTRjPckmnhhuBw1cBy5IoCYdCws\\\\\\\/PfkrGOXmxVKSMtrWuc23qGNOyv9HhGmiRh\\\\\\\/jqDVsWVyv5HmG5VxmcmXKz\\\\\\\/6VHQ1qGYZvsSVXEzjHI8irIDA0iwmuHzUYJfisFdYK5\\\\\\\/4kw7+e3EZpRR03aCVKT697pR4tbnVXKCRwsjSmFheLtcTwN2HVnmYWnEdHoXAe7LbyfUs5ExqD6JXkWyECf5ePWs6mm0yMWgN7EyF7ppnh7fmiTga6wwax9Oy9tIAqg0PK5nTN8dlznuq8hLRPZ5bJ8yY29m\\\\\\\/9hNA7xGBTRHe9utkBr+u1dOtHHaVg9vbSOyeNzUblrgFs1HIgz8JMvq8bdT0Cxhvn5NUu9ceBTmMCgGRqa6GOABxrcPNuDDY7DE1XJjBoI2ShDvnRfKZWfDRcuATnuotnm6Efo1h2Lmg7lSUMHh4WpnTNXlXAlWV8OpWOYACUuUV9Rq1VUfGFlsCCCXy0DNp3wEirA5fxBmow8QIrttApq2PAs3vGpYnTZWqZHHtJ9VkOx0BkvDoHdoAUPrZc93NKt5ebIP0G7MBizh6FrULDdS15cNB6PKZHUNGbh7QqT92otK+6f1nrfX8Lo0qjyDFmreka7nkhFHHVjEJllsLwrFK13FsMautmTeEr7Y4Uos1gsPMSScs3JCTV52N0M7Qi6CPyy1gfLtE5gQXOvL7ikPHc4QznUjC7D+5h5wN9IWTnvr+PS3iD+GLbzazWDcnAB2Bx\\\\\\\/Xc0OMiY+9j9bpZdiikAZkiY4A9mTzEfK7afMjkU7tV79rrZkNolkJz0VcujF3j6nyE\\\\\\\/NjmlFjnOUHLkZMMox6dxN5SzoriRiXpjtdACmuXX3GMlueHhEDrInIlPGd3OEUxIwyU8YO6VA7ZwTRdK5XkJ35yYpwfJWO4lif4EQgM0PLYrm7e\\\\\\\/3YGKmRRCmgF9PQZGpNY4D5vVscUetJFml12QJsRW\\\\\\\/+\\\\\\\/W5EktFuUKZ6CEG4synKXOM6qd7kklZsRi1GxbrR1gzfu3hVvNHOn9uyItOmBkqWab1McTbysvWZBLaLzIE+gDVsMc\\\\\\\/HENqtETk\\\u0022,\\\u0022iv\\\u0022:\\\u0022HyD7dDlAhzBRtnX0fD5vhw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002228be05fff25d4614a2f7b8dfeb9e97b960dd63a6acf0977d534ab08e2133d42c\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460044 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/10/2177" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-gY9tuhwZ4hO4iqlamlwjMpN_Cy6culKvN-VdRC0CNLo" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/10/2177" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-8dK_BrDfAdip-oiQej10eyJEUyi-uCLzhYyB6DSTFLE" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/10/2165" title="Similar Efficacies of Selection Shape Mitochondrial and Nuclear Genes in Both Drosophila melanogaster and Homo sapiens" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/10/2187" title="Genetic Networks Required to Coordinate Chromosome Replication by DNA Polymerases α, δ, and ε in Saccharomyces cerevisiae" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460044" id="node460044" data-pisa="ggg;5/10/2177" data-pisa-master="ggg;g3.115.019778" data-apath="/ggg/5/10/2177.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Identification and Correction of Sample Mix-Ups in Expression Genetic Data: A Case Study</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-4914-6671" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Karl W. Broman</span>, <span class="highwire-citation-author" data-delta="1">Mark P. Keller</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0003-3783-2807" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Aimee Teo Broman</span>, <span class="highwire-citation-author" data-delta="3">Christina Kendziorski</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-8774-9377" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Brian S. Yandell</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0003-4519-6361" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Śaunak Sen</span> and <span class="highwire-citation-author" data-delta="6">Alan D. Attie</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2177-2186;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.019778 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460044" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Karl W. Broman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biostatistics and Medical Informatics, University of Wisconsin, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Karl%2BW.%2BBroman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Broman%20KW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKarl%2BW.%2BBroman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-4914-6671" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Karl W. Broman</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:kbroman@biostat.wisc.edu" class="" data-icon-position="" data-hide-link-title="0">kbroman@biostat.wisc.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Mark P. Keller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biochemistry, University of Wisconsin, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mark%2BP.%2BKeller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Keller%20MP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMark%2BP.%2BKeller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Aimee Teo Broman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biostatistics and Medical Informatics, University of Wisconsin, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aimee%2BTeo%2BBroman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Broman%20AT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAimee%2BTeo%2BBroman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-3783-2807" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Aimee Teo Broman</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Christina Kendziorski </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biostatistics and Medical Informatics, University of Wisconsin, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christina%2BKendziorski%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kendziorski%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChristina%2BKendziorski%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Brian S. Yandell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Statistics, University of Wisconsin, Madison, Wisconsin 53706</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Horticulture, University of Wisconsin, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Brian%2BS.%2BYandell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yandell%20BS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABrian%2BS.%2BYandell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8774-9377" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Brian S. Yandell</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Śaunak Sen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Epidemiology and Biostatistics, University of California, San Francisco, California 94107</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=%C5%9Aaunak%2BSen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sen%20%C5%9A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3A%25C5%259Aaunak%2BSen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4519-6361" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Śaunak Sen</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Alan D. Attie </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biochemistry, University of Wisconsin, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alan%2BD.%2BAttie%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Attie%20AD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlan%2BD.%2BAttie%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/5/10/2177" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460044" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460044/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/10/2177.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460044" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460044/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/10/2177.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460044" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460044/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/10/2177.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460044" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460044/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/5/10/2177.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460044" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460044/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">In a mouse intercross with more than 500 animals and genome-wide gene expression data on six tissues, we identified a high proportion (18%) of sample mix-ups in the genotype data. Local expression quantitative trait loci (eQTL; genetic loci influencing gene expression) with extremely large effect were used to form a classifier to predict an individual’s eQTL genotype based on expression data alone. By considering multiple eQTL and their related transcripts, we identified numerous individuals whose predicted eQTL genotypes (based on their expression data) did not match their observed genotypes, and then went on to identify other individuals whose genotypes did match the predicted eQTL genotypes. The concordance of predictions across six tissues indicated that the problem was due to mix-ups in the genotypes (although we further identified a small number of sample mix-ups in each of the six panels of gene expression microarrays). Consideration of the plate positions of the DNA samples indicated a number of off-by-one and off-by-two errors, likely the result of pipetting errors. Such sample mix-ups can be a problem in any genetic study, but eQTL data allow us to identify, and even correct, such problems. Our methods have been implemented in an R package, R/lineup.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/quality-control" class="hw-term hw-article-keyword hw-article-keyword-quality-control" rel="nofollow">quality control</a></li><li class="kwd"><a href="/keyword/microarrays" class="hw-term hw-article-keyword hw-article-keyword-microarrays" rel="nofollow">microarrays</a></li><li class="kwd"><a href="/keyword/genetical-genomics" class="hw-term hw-article-keyword hw-article-keyword-genetical-genomics" rel="nofollow">genetical genomics</a></li><li class="kwd"><a href="/keyword/mislabeling-errors" class="hw-term hw-article-keyword hw-article-keyword-mislabeling-errors" rel="nofollow">mislabeling errors</a></li><li class="kwd"><a href="/keyword/eqtl" class="hw-term hw-article-keyword hw-article-keyword-eqtl" rel="nofollow">eQTL</a></li></ul><p id="p-4">To map the genetic loci influencing a complex phenotype, one seeks to establish an association between genotype and phenotype. In such an effort, the maintenance of the concordance between genotyped and phenotyped samples and data is critical. Sample mislabeling and other sample mix-ups will weaken associations, resulting in reduced power and biased estimates of locus effects. In traditional genetic studies, one has limited ability to detect sample mix-ups and almost no ability to correct such problems. Inconsistencies between subjects’ sex and X chromosome genotypes may reveal some problems, and in family studies, some errors may be revealed through Mendelian inconsistencies at markers, but we will generally be blind to most errors.</p><p id="p-5">In expression genetics studies, in which genome-wide gene expression is assayed along with genotypes at genetic markers, the presence of expression quantitative trait loci (eQTL) with profound effect on gene expression (particularly local eQTL, in which a polymorphism near a gene affects the expression of that gene) provides an opportunity to not just identify but also correct sample mix-ups.</p><p id="p-6">In a mouse intercross with more than 500 animals and genome-wide gene expression data on six tissues, we identified a high proportion (18%) of sample mix-ups in the genotype data. We further identified a small number of mix-ups among the expression arrays in each tissue.</p><p id="p-7">A number of investigators have developed methods for identifying such sample mix-ups (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Westra <em>et al.</em> 2011</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Ekstrøm and Feenstra 2012</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Lynch <em>et al.</em> 2012</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Schadt <em>et al.</em> 2012</a>), and a similar approach was applied by <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Baggerly and Coombes (2008</a>, <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2009</a>) in their forensic bioinformatics analyses of the Duke debacle. We have developed a further approach that is simple but effective. We illustrate its use through a particularly dramatic example.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Mice and genotyping</h3><p id="p-8">C57BL/6J (abbreviated B6 or B) and BTBR <em>T</em><sup>+</sup> <em>tf</em>/J (abbreviated BTBR or R) mice were purchased from the Jackson Laboratory (Bar Harbor, ME) and bred at the University of Wisconsin–Madison. The <em>Lep<sup>ob</sup></em> mutation was introgressed into all strains using heterozygous parents to generate homozygous <em>Lep</em><sup><em>ob/ob</em></sup> offspring. F<sub>2</sub> mice, all <em>Lep</em><sup><em>ob/ob</em></sup>, were the offspring of F<sub>1</sub> parents derived from a cross between BTBR females and B6 males (<a href="http://www.g3journal.org/content/suppl/2015/08/19/g3.115.019778.DC1/019778SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/content/suppl/2015/08/19/g3.115.019778.DC1/FigureS1.pdf">Figure S1</a>). F<sub>2</sub> mice and a small number of parental and F<sub>1</sub> controls were genotyped with the 5K GeneChip (Affymetrix).</p></div><div id="sec-3" class="subsection"><h3>Gene expression microarrays</h3><p id="p-9">Gene expression was assayed with custom two-color, ink-jet microarrays manufactured by Agilent Technologies (Palo Alto, CA). RNA preparations were performed at Rosetta Inpharmatics (Merck &amp; Co.). Six tissues were considered: adipose, gastrocnemius muscle (abbreviated gastroc), hypothalamus (abbreviated hypo), pancreatic islets (abbreviated islet), kidney, and liver. Tissue-specific messenger RNA (mRNA) pools were used for the second channel, and gene expression was quantified as the ratio of the mean log<sub>10</sub> intensity (mlratio). For further details, see <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Keller <em>et al.</em> (2008)</a>.</p></div><div id="sec-4" class="subsection"><h3>Sample mix-ups in the gene expression arrays</h3><p id="p-10">Let <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2177/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2177/embed/mml-math-1.gif"/></noscript></span></span> denote the gene expression measure for sample <em>i</em> at array probe <em>p</em> in tissue <em>s</em>. We first considered each probe and each pair of tissues and calculated the between-tissue correlation across samples, omitting any samples with missing data for that probe in either tissue. We identified the subset of probes, for each tissue pair, with correlation &gt;0.75. With this subset of probes, we then calculated the correlation between sample <em>i</em> in tissue <em>s</em> and sample <em>j</em> in tissue <em>t</em>; call it <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2177/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2177/embed/mml-math-2.gif"/></noscript></span></span>. As an illustration, consider the schematic in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>: for each pair of tissues, we identified the subset of probes with high between-tissue correlation (the shaded region) and then evaluated the correlation between a sample in one tissue and another sample in the other tissue, across that subset of probes.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/10/2177/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Scheme for evaluating the similarity between expression arrays for different tissues. We first consider the expression of each array probe for samples assayed for both tissues (A) and calculate the between-tissue correlation in expression (B). We identify the subset of array probes with correlation &gt;0.75 (shaded region in C) and calculate the correlation in gene expression for one sample in the first tissue and another sample in the second tissue, across these selected probes. This forms a similarity matrix (D), for which darker squares indicate greater similarity. Orange squares indicate missing values (samples assayed in one tissue but not the other)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1379678328" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Scheme for evaluating the similarity between expression arrays for different tissues. We first consider the expression of each array probe for samples assayed for both tissues (A) and calculate the between-tissue correlation in expression (B). We identify the subset of array probes with correlation &gt;0.75 (shaded region in C) and calculate the correlation in gene expression for one sample in the first tissue and another sample in the second tissue, across these selected probes. This forms a similarity matrix (D), for which darker squares indicate greater similarity. Orange squares indicate missing values (samples assayed in one tissue but not the other).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/10/2177/F1.medium.gif" width="440" height="276"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/5/10/2177/F1.medium.gif" width="440" height="276"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/10/2177/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/10/2177/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466870" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-11" class="first-child">Scheme for evaluating the similarity between expression arrays for different tissues. We first consider the expression of each array probe for samples assayed for both tissues (A) and calculate the between-tissue correlation in expression (B). We identify the subset of array probes with correlation &gt;0.75 (shaded region in C) and calculate the correlation in gene expression for one sample in the first tissue and another sample in the second tissue, across these selected probes. This forms a similarity matrix (D), for which darker squares indicate greater similarity. Orange squares indicate missing values (samples assayed in one tissue but not the other).</p><div class="sb-div caption-clear"></div></div></div><p id="p-12">We then summarized the similarity between sample <em>i</em> in tissue <em>s</em> and sample <em>j</em> in the other tissues by the median correlation across tissue pairs that include tissue <em>s</em>, <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2177/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2177/embed/mml-math-3.gif"/></noscript></span></span>. Of course, we considered only pairs of tissues (<em>s</em>,<em>t</em>) for which sample <em>i</em> was measured in tissue <em>s</em> and sample <em>j</em> was measured in tissue <em>t</em>.</p><p id="p-13">Sample mix-ups in tissue <em>s</em> were identified as samples <em>i</em> for which the self similarity, <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2177/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2177/embed/mml-math-4.gif"/></noscript></span></span>, was small, but for which there existed some array with high similarity: <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2177/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2177/embed/mml-math-5.gif"/></noscript></span></span> is large. We then inferred the correct label for sample <em>i</em> in tissue <em>s</em> to be <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2177/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2177/embed/mml-math-6.gif"/></noscript></span></span>. In other words, viewing <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2177/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2177/embed/mml-math-7.gif"/></noscript></span></span> as a similarity matrix, we were looking for rows with a small value on the diagonal, but with some large off-diagonal element in that row. To ensure confidence in the relabeling of such samples, we compared the maximum value in the row to the second-highest value.</p><p id="p-14">To further investigate possible sample duplicates within a tissue, we considered the subset of probes with correlation &gt;0.75 with at least one other tissue and then calculated between-sample correlations, across the chosen subset of probes, within that tissue.</p></div><div id="sec-5" class="subsection"><h3>Sample mix-ups in the DNA samples</h3><p id="p-15">In our investigation of potential sample mix-ups in the DNA samples, we first calculated multipoint genotype probabilities at all markers and at pseudomarker positions between markers. The pseudomarker positions were placed at evenly spaced locations between markers, with a maximum spacing of 0.5 cM between adjacent markers or pseudomarkers. The multipoint genotype probability calculations were performed via a hidden Markov model, with an assumed genotyping error rate of 0.2% and with the Carter-Falconer map function (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Carter and Falconer 1951</a>).</p><p id="p-16">We first considered each tissue, individually, and identified the subset of probes with a strong local eQTL. We considered all array probes with known genomic location and on an autosome, identified the nearest marker or pseudomarker to the location of the probe, and calculated a LOD score (log<sub>10</sub> likelihood ratio) assessing the association between genotype at that location and the gene expression of that probe. The LOD score was calculated by Haley–Knott regression (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Haley and Knott 1992</a>), a quick approximation to standard interval mapping (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Lander and Botstein 1989</a>). Calculations were performed at a single location for each array probe, rather than with a scan of the genome. We chose the subset of probes with LOD &gt;100.</p><p id="p-17">Continuing to focus on one tissue at a time, we considered the set of local eQTL locations and the corresponding probe or probes. (Generally there was a single probe corresponding to a given eQTL location, but in a small number of instances for each tissue, there were a pair of probes at the same eQTL location; for islet, there were three eQTL with three corresponding probes, and for adipose there was one such trio.) For each eQTL position and for each mouse, we took the genotypes with maximal multipoint probability to be the observed eQTL genotype, provided that this exceeded 0.99; if no genotype had probability &gt;0.99, the observed eQTL genotype was treated as missing.</p><p id="p-18">Considering each eQTL in a tissue individually, we then formed a <em>k</em>-nearest neighbor classifier, with <em>k</em> = 40, for predicting eQTL genotype from the expression values for the corresponding probe or probes. For a given mouse, if more than 80% of the 40 nearest neighbors, by Euclidean distance, shared the same observed eQTL genotype, this was taken to be the inferred eQTL genotype for that mouse. If no more than 80% of the 40 nearest neighbors shared a common genotype, the inferred eQTL genotype was treated as missing.</p><p id="p-19">To filter out samples that were clearly incorrect and improve our classifiers, we then calculated the proportion of matches, for each sample, between the observed eQTL genotypes and the corresponding inferred eQTL genotypes, omitted samples for which the proportion of matches was &lt;0.7, and rederived the <em>k</em>-nearest neighbor classifiers with the subset of samples deemed likely correct.</p><p id="p-20">As an illustration, consider the schematic in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>: for each tissue, we identified a subset of array probes with strong local eQTL, we derived classifiers for predicting eQTL genotype from the corresponding expression phenotypes, and then constructed a matrix of inferred eQTL genotypes. As a measure of similarity between a DNA sample and an mRNA sample, we calculated the proportion of matches between the observed eQTL genotypes for the DNA sample and the inferred eQTL genotypes for the mRNA sample.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/10/2177/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Scheme for evaluating the similarity between genotypes and expression arrays. We first identify a set of probes with strong local expression quantitative trait loci (eQTL). For each such eQTL, we use the samples with both genotype and expression data (A) to form a classifier for predicting eQTL genotype from the expression value (B). We then compare the observed eQTL genotypes for one sample to the inferred eQTL genotypes, from the classifiers, for another sample (C). The proportion of matches, between the observed and inferred genotypes, forms a similarity matrix (D), for which darker squares indicate greater similarity. Orange squares indicate missing values (for example, samples with genotype data but no expression data)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1379678328" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Scheme for evaluating the similarity between genotypes and expression arrays. We first identify a set of probes with strong local expression quantitative trait loci (eQTL). For each such eQTL, we use the samples with both genotype and expression data (A) to form a classifier for predicting eQTL genotype from the expression value (B). We then compare the observed eQTL genotypes for one sample to the inferred eQTL genotypes, from the classifiers, for another sample (C). The proportion of matches, between the observed and inferred genotypes, forms a similarity matrix (D), for which darker squares indicate greater similarity. Orange squares indicate missing values (for example, samples with genotype data but no expression data).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/10/2177/F2.medium.gif" width="440" height="314"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/5/10/2177/F2.medium.gif" width="440" height="314"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/10/2177/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/10/2177/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466816" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-21" class="first-child">Scheme for evaluating the similarity between genotypes and expression arrays. We first identify a set of probes with strong local expression quantitative trait loci (eQTL). For each such eQTL, we use the samples with both genotype and expression data (A) to form a classifier for predicting eQTL genotype from the expression value (B). We then compare the observed eQTL genotypes for one sample to the inferred eQTL genotypes, from the classifiers, for another sample (C). The proportion of matches, between the observed and inferred genotypes, forms a similarity matrix (D), for which darker squares indicate greater similarity. Orange squares indicate missing values (for example, samples with genotype data but no expression data).</p><div class="sb-div caption-clear"></div></div></div><p id="p-22">To combine the tissue-specific similarity measures across the six tissues, we simply took the overall proportion of matching genotypes across all eQTL and across all tissues.</p><p id="p-23">As in the investigation of sample mix-ups within the expression arrays, we treated the proportions of matches between observed and inferred eQTL genotypes as a similarity matrix. Problem DNA samples were identified as rows for which the value on the diagonal (the self similarity) was small. In such rows, we inferred the correct label to be that of the maximal off-diagonal value, provided that this maximum was large and was well above the second-largest value.</p></div><div id="sec-6" class="subsection"><h3>QTL analysis</h3><p id="p-24">To characterize the improvement in results after correction of sample mix-ups, we performed QTL analysis with several traits of interest, including the expression traits in each tissue, with the original data and with the corrected data. In the corrected data, we omitted the DNA samples that could not be verified to be correct (that is, those with no corresponding gene expression data.)</p><div id="sec-7" class="subsection"><h4>Insulin:</h4><p id="p-25">We first considered a clinical phenotype of considerable interest: 10-week plasma insulin. QTL analysis was performed by Haley−Knott regression (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Haley and Knott 1992</a>), with log insulin, and with sex included as an interactive covariate (that is, allowing the effects of QTL to be different in the two sexes).</p></div><div id="sec-8" class="subsection"><h4>Agouti and tufted coat:</h4><p id="p-26">We considered two simple Mendelian traits: agouti coat color (due to a single gene on chromosome 2) and tufted coat (due to a single gene on chromosome 17). QTL analysis was performed treating each phenotype as a binary trait (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Xu and Atchley 1996</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Broman 2003</a>). To handle possible marker genotyping errors at the causal loci, we took the observed genotypes to be those with maximal multipoint probability, provided that this exceeded 0.99; if no genotype had probability &gt;0.99, the observed genotype was treated as missing.</p></div><div id="sec-9" class="subsection"><h4>eQTL analyses:</h4><p id="p-27">We considered each of the six tissues individually, and focused on the subset of probes with known genomic location on an autosome or the X chromosome. For hypothalamus tissue, we omitted a batch of 119 poorly behaved arrays, although these had been included in our efforts to identify sample mix-ups.</p><p id="p-28">Expression measures were transformed to normal quantiles. That is, the expression measures were converted to ranks <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2177/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2177/embed/mml-math-8.gif"/></noscript></span></span> and then transformed to <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2177/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2177/embed/mml-math-9.gif"/></noscript></span></span>, where <em>Φ</em><sup>−1</sup> is the inverse of the normal cumulative distribution function.</p><p id="p-29">QTL analysis was performed by Haley−Knott regressions with sex included as an interactive covariate. We considered the maximal peak for each array probe on each chromosome and inferred the presence of a QTL if the LOD score exceeded 5, a 5% genome-wide significance level established by computer simulation. An inferred eQTL was considered a local eQTL if the 2-LOD support interval contained the genomic location of the corresponding array probe; otherwise, it was considered a <em>trans</em>-eQTL.</p></div></div><div id="sec-10" class="subsection"><h3>Software</h3><p id="p-30">All analyses were conducted with R (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">R Development Core Team 2015</a>). QTL analyses were performed with the R package, R/qtl (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Broman <em>et al.</em> 2003</a>). Our methods for identifying sample mix-ups have been assembled as an R package, R/lineup, available at <a href="http://github.com/kbroman/lineup">http://github.com/kbroman/lineup</a> as well as The Comprehensive R Archive Network (CRAN; <a href="http://cran.r-project.org">http://cran.r-project.org</a>).</p></div><div id="sec-11" class="subsection"><h3>Data availability</h3><p id="p-31">The genotype and gene expression microarray data are available at the QTL Archive, now part of the Mouse Phenome Database: <a href="http://phenome.jax.org/db/q?rtn=projects/projdet&amp;reqprojid=532">http://phenome.jax.org/db/q?rtn=projects/projdet&amp;reqprojid=532</a>.</p></div></div><div class="section results" id="sec-12"><h2 class="">Results</h2><p id="p-32">We first became aware of potential problems in the samples through the identification of six duplicate DNA samples and 32 mice whose X chromosome genotypes were incompatible with their sex. We genotyped 554 F<sub>2</sub> mice at 2060 informative SNPs, including 20 on the X chromosome. Three samples were assigned “no call” at all markers and not considered further. Six pairs were seen to be duplicates, with over 98% genotype identity across typed markers (<a href="http://www.g3journal.org/content/suppl/2015/08/19/g3.115.019778.DC1/TableS1.pdf">Table S1</a>).</p><p id="p-33">The F<sub>2</sub> mice were the offspring of F<sub>1</sub> siblings derived by crossing BTBR females to B6 males (<a href="http://www.g3journal.org/content/suppl/2015/08/19/g3.115.019778.DC1/FigureS1.pdf">Figure S1</a>). F<sub>2</sub> females should be homozygous BTBR (RR) or heterozygous (BR) on the X chromosome; F<sub>2</sub> males should be hemizygous B or R. (Note that homozygous and hemizygous genotypes could not be distinguished.) However, 19 females exhibited some homozygous B6 genotypes on the X, and 17 males exhibited some heterozygous genotypes (<a href="http://www.g3journal.org/content/suppl/2015/08/19/g3.115.019778.DC1/FigureS2.pdf">Figure S2</a>). Although four of these males had a single heterozygous genotype that was likely a genotyping error, the 19 females and the other 13 males were clearly indicated to have swapped sex. There were an additional 53 females and 50 males with homozygous RR or hemizygous R genotypes for all markers on the X chromosome, compatible with either sex.</p><p id="p-34">In cleaning the genotype data, we omitted a set of seven samples, including one pair of the sample duplicates, with poorly behaved data. (They showed a high rate of apparent genotyping errors, an unusually large proportion of homozygous genotypes, and an unusually large number of apparent crossovers.) For the other five pairs of duplicates, we omitted one sample from each pair.</p><div id="sec-13" class="subsection"><h3>Sample mix-ups in the gene expression arrays</h3><p id="p-35">For each of six tissues (adipose, gastroc, hypo, islet, kidney, liver), approximately 500 F<sub>2</sub> mice were assayed for gene expression with two-color Agilent arrays with tissue-specific pools (<a href="http://www.g3journal.org/content/suppl/2015/08/19/g3.115.019778.DC1/TableS2.pdf">Table S2</a>). A small number of poorly behaved arrays were omitted. We later discovered a batch of 119 poorly behaved arrays for hypo, but these were included in the analyses described here. There were 527 mice assayed for at least one of the six tissues, but not all mice were assayed for all tissues. In particular, there were 27 mice assayed only for gene expression in kidney, and 43 mice assayed for all tissues except kidney. Furthermore, 27 mice were genotyped but were not subject to gene expression analysis.</p><p id="p-36">To identify potential sample mix-ups among gene expression arrays, we first identified, for each pair of tissues, a subset of array probes with high between-tissue correlations. Consideration of all probes would greatly reduce the apparent correlation between arrays, due to the abundance of unexpressed genes. For example, for Mouse3567, the correlation between gene expression in kidney and in liver, across all 40,572 probes, is 0.32, whereas for the subset of 155 probes with correlation &gt;0.75 between kidney and liver, the correlation is 0.78. (See <a href="http://www.g3journal.org/content/suppl/2015/08/19/g3.115.019778.DC1/FigureS3.pdf">Figure S3</a>.)</p><p id="p-37"><a href="http://www.g3journal.org/content/suppl/2015/08/19/g3.115.019778.DC1/FigureS4.pdf">Figure S4</a> contains density estimates of the between-tissue correlations for all array probes. The densities are organized by tissue, with the panel for each tissue containing the five tissue pairs involving that tissue. There are some small differences among tissue pairs, but the vast majority of between-tissue correlations are between −0.25 and 0.50. <a href="http://www.g3journal.org/content/suppl/2015/08/19/g3.115.019778.DC1/TableS3.pdf">Table S3</a> contains the numbers of probes for each pair of tissues with correlations exceeding 0.70, 0.75, 0.80, and 0.90, respectively. We focused on probes with correlations &gt;0.75, of which there were between 46 and 200 probes per tissue pair.</p><p id="p-38">For each pair of tissues, we calculated the correlations among samples across the subset of correlated probes. For each tissue, we then summarized the similarity between each sample in that tissue and each sample in other tissues by the median correlations, across the tissue pairs that included the target tissue.</p><p id="p-39"><a href="http://www.g3journal.org/content/suppl/2015/08/19/g3.115.019778.DC1/FigureS5.pdf">Figure S5</a> contains histograms of the similarity measures for each tissue, separating the self-self similarities (the diagonal elements) and the self-nonself similarities (the off-diagonal elements). There are a number of clear outliers: small self-self similarities and large self-nonself similarities. The self-nonself similarities follow a bimodal distribution, with the lower mode corresponding to opposite-sex pairs and the upper mode corresponding to same-sex pairs. The chosen probes included a probe in <em>Xist</em> (involved in X chromosome inactivation) and probes on the Y chromosome.</p><p id="p-40">To identify problem samples in each tissue, we considered for each sample, the self similarity <em>vs.</em> the maximum similarity (that is, the values on the diagonal of the similarity matrix and the maximum values in each row). These are displayed in <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/10/2177/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Self similarity (median correlation across tissue pairs) vs. maximum similarity for the expression arrays for each tissue. The diagonal gray line corresponds to equality. Green points are inferred to be sample mix-ups. Gray points correspond to arrays for which the self similarity is maximal. Red points correspond to special cases (see the text). There were 27 samples assayed only for kidney; these have missing self similarity values." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1379678328" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Self similarity (median correlation across tissue pairs) &lt;em&gt;vs.&lt;/em&gt; maximum similarity for the expression arrays for each tissue. The diagonal gray line corresponds to equality. Green points are inferred to be sample mix-ups. Gray points correspond to arrays for which the self similarity is maximal. Red points correspond to special cases (see the text). There were 27 samples assayed only for kidney; these have missing self similarity values.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/10/2177/F3.medium.gif" width="369" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/5/10/2177/F3.medium.gif" width="369" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/10/2177/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/10/2177/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466925" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-41" class="first-child">Self similarity (median correlation across tissue pairs) <em>vs.</em> maximum similarity for the expression arrays for each tissue. The diagonal gray line corresponds to equality. Green points are inferred to be sample mix-ups. Gray points correspond to arrays for which the self similarity is maximal. Red points correspond to special cases (see the text). There were 27 samples assayed only for kidney; these have missing self similarity values.</p><div class="sb-div caption-clear"></div></div></div><p id="p-42">The vast majority of samples in each tissue were indicated to be correctly labeled: the self similarity was the maximum similarity. But for each tissue, there were at least a few samples that were more like some other sample in the other tissues. In each case, we infer the correct label to be that with the maximal similarity. In <a href="http://www.g3journal.org/content/suppl/2015/08/19/g3.115.019778.DC1/FigureS6.pdf">Figure S6</a>, we display the second-highest similarity <em>vs.</em> the maximum similarity for each sample in each tissue. The problem samples (colored green) are generally well away from the diagonal, indicating good support for our ability to infer the correct label.</p><p id="p-43">The red dots in <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a> and <a href="http://www.g3journal.org/content/suppl/2015/08/19/g3.115.019778.DC1/FigureS6.pdf">Figure S6</a> are special cases: The Mouse3188 sample is highlighted as a potential problem in both islet and gastroc (being slightly off the diagonal line), but this is because that sample was involved in array swaps in two different tissues (adipose and hypo). This is the only sample indicated to be mislabeled in multiple tissues. We also highlight Mouse3484 in gastroc, which appeared to be a mixture (more in the paragraphs to follow).</p><p id="p-44">The inferred errors are displayed in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>. For adipose, we identified two problems. The samples for Mouse3583 and Mouse3584 were swapped, and there was a three-way swap among Mouse3187, Mouse3188, and Mouse3200, with the sample labeled Mouse3187 really being Mouse3188, that labeled Mouse3188 really being Mouse3200, and that labeled Mouse3200 really being Mouse3187.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/10/2177/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The messenger RNA sample mix-ups for the six tissues. Double-headed arrows indicate a sample swap. The trio of points in adipose corresponds to a three-way swap. The pink circles with a single-headed arrow, in islet and liver, are sample duplicates. The questionable case in kidney indicates a potential sample mixture arrayed twice." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1379678328" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The messenger RNA sample mix-ups for the six tissues. Double-headed arrows indicate a sample swap. The trio of points in adipose corresponds to a three-way swap. The pink circles with a single-headed arrow, in islet and liver, are sample duplicates. The questionable case in kidney indicates a potential sample mixture arrayed twice.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/10/2177/F4.medium.gif" width="440" height="408"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/5/10/2177/F4.medium.gif" width="440" height="408"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/10/2177/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/10/2177/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466834" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-45" class="first-child">The messenger RNA sample mix-ups for the six tissues. Double-headed arrows indicate a sample swap. The trio of points in adipose corresponds to a three-way swap. The pink circles with a single-headed arrow, in islet and liver, are sample duplicates. The questionable case in kidney indicates a potential sample mixture arrayed twice.</p><div class="sb-div caption-clear"></div></div></div><p id="p-46">For gastroc, there was a single sample swap, between Mouse3655 and Mouse3659. For hypo, there were nine pairs of sample swaps. For islet, the samples Mouse3598 and Mouse3599 were swapped, and the sample labeled Mouse3296 was really a duplicate (or <em>unintended technical replicate</em>) of the Mouse3295 sample. For liver, the sample labeled Mouse3142 really corresponded to Mouse3143 (Mouse3142 was not assayed for gene expression in liver), and the sample labeled Mouse3141 was really a duplicate of the Mouse3136 sample.</p><p id="p-47">For kidney, the samples for Mouse3510 and Mouse3523 were swapped, and Mouse3484 also was seen to be a problem. We believe that the samples for Mouse3484 and Mouse3503 may have been mixed and assayed twice in duplicate (more in the paragraphs to follow). There were 27 samples that were assayed for gene expression only in kidney; for these, the self similarity cannot be calculated. We have limited ability to detect mix-ups for these samples, but none were very close to any sample in other tissues, and so they can, at least provisionally, be assumed to be correctly labeled.</p><p id="p-48">To further illustrate the sample swaps, <a href="http://www.g3journal.org/content/suppl/2015/08/19/g3.115.019778.DC1/FigureS7.pdf">Figure S7</a> contains scatter plots of the gastroc arrays labeled Mouse3655 and Mouse3659 against the arrays in the other tissues with those labels. For each pair of tissues, we plot the array probes with between-tissue correlation &gt;0.75. Mouse3655 in gastroc is correlated with Mouse3659 in other tissues, whereas Mouse3659 in gastroc is correlated with Mouse3655 in other tissues, indicating a clear swap between these samples within gastroc.</p><p id="p-49"><a href="http://www.g3journal.org/content/suppl/2015/08/19/g3.115.019778.DC1/FigureS8.pdf">Figure S8</a> contains similar scatter plots for a pair of inferred duplicates, with the sample labeled Mouse3141 in liver really being a duplicate of the Mouse3136 liver sample. Mouse3136 liver and Mouse3141 liver are each correlated with Mouse3136 in other tissues and not with Mouse3141, and the two samples are extremely highly correlated with each other (see the two central panels in the bottom row). In <a href="http://www.g3journal.org/content/suppl/2015/08/19/g3.115.019778.DC1/FigureS9.pdf">Figure S9</a>, we display the between-sample correlations for samples with these two labels, for all pairs of tissues, with the pairs including liver highlighted in red. The Mouse3136 samples are correlated for all tissue pairs; the Mouse3141 samples are correlated for all tissue pairs not involving liver, and the Mouse3141 liver sample is correlated with all Mouse3136 samples in other tissues.</p><p id="p-50">The Mouse3484 and Mouse3503 samples in kidney appear to be sample duplicates, but these samples are correlated with each of Mouse3484 and Mouse3503 in the other tissues. We’re inclined to believe that the two kidney samples were mixed and arrayed in duplicate, but we are not able to prove this point. <a href="http://www.g3journal.org/content/suppl/2015/08/19/g3.115.019778.DC1/FigureS10.pdf">Figure S10</a> contains scatter plots for the two samples in kidney <em>vs.</em> all tissues; the central panels in the second row from the bottom indicate that the two samples are highly correlated and so likely replicates, but all scatter plots here show strong correlation. <a href="http://www.g3journal.org/content/suppl/2015/08/19/g3.115.019778.DC1/FigureS11.pdf">Figure S11</a> contains the between-sample correlations for both sample labels in all tissue pairs; contrast this with <a href="http://www.g3journal.org/content/suppl/2015/08/19/g3.115.019778.DC1/FigureS9.pdf">Figure S9</a>, for the simple duplicate in liver. Mouse3484 kidney and Mouse3503 kidney are strongly correlated with both samples in the other tissues but not so strongly as Mouse3484 and Mouse3503 are with themselves in the nonkidney pairs. And for tissue pairs not including kidney, Mouse3484 and Mouse3503 are much more weakly correlated.</p><p id="p-51">Because we were unable to resolve the problems with Mouse3484 and Mouse3503 in kidney, these two arrays were omitted from later analyses. The two simple sample duplicates, one in islet and one in liver, were combined and assigned the correct label. The other sample mix-ups were relabeled as inferred in <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>.</p><p id="p-52">Expression of the <em>Xist</em> gene (involved in X chromosome inactivation and so highly expressed in females but not males) and of genes on the Y chromosome is a useful diagnostic for the sex of an mRNA sample. In <a href="http://www.g3journal.org/content/suppl/2015/08/19/g3.115.019778.DC1/FigureS12.pdf">Figure S12</a>, we display, for each tissue, the average expression across four Y chromosome genes <em>vs.</em> the expression of <em>Xist</em>, with the original data and after correction of the sample mix-ups in the expression arrays. Just three of the sample-swaps (one in gastroc and two in hypo) involved opposite-sex pairs. These show up clearly in the left column, with the original data, and are resolved after correction of the sample mix-ups. The unusual pattern of expression in hypo, with a bimodal distribution for the Y chromosome genes in males and a large number of females with relatively low <em>Xist</em> expression, was due to a set of 119 poorly behaved arrays.</p></div><div id="sec-14" class="subsection"><h3>Sample mix-ups in the genotypes</h3><p id="p-53">Having corrected the sample mix-ups among the gene expression arrays, we turned to potential problems in the genotypes. For each tissue, we considered the 36,364 autosomal array probes with known genomic location and identified those with a strong local eQTL, having LOD score &gt;100 for the association between the probe expression measures and genotype at the corresponding location.</p><p id="p-54">For each such probe, we created a <em>k</em>-nearest neighbor classifier (with k = 40), for predicting eQTL genotype from the expression phenotype. For example, in <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>, we display the expression, in islet, of probe 499541 (on chromosome 1) <em>vs.</em> genotype at the nearest marker. At this probe, there are three clear groups of mice, with B6 homozygotes (BB) having high expression, BTBR homozygotes (RR) having low expression, and heterozygotes (BR) intermediate. There are a number of mice whose expression does not match their observed eQTL genotype; the classifier infers a different eQTL genotype. The points highlighted in pink have expression at the boundary between the BB and BR groups and are left unassigned. (To assign an inferred eQTL genotype to a point, we required that 80% of the nearest neighbors had a common eQTL genotype.)</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/10/2177/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Plot of islet expression vs. observed genotype for an example probe. Points are colored by the inferred genotype, based on a k-nearest neighbor classifier, with yellow, green, and blue corresponding to BB, BR, and RR, respectively, where B = B6 and R = BTBR. Salmon-colored points lie at the boundary between two clusters and were not assigned." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1379678328" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Plot of islet expression &lt;em&gt;vs.&lt;/em&gt; observed genotype for an example probe. Points are colored by the inferred genotype, based on a k-nearest neighbor classifier, with yellow, green, and blue corresponding to BB, BR, and RR, respectively, where B = B6 and R = BTBR. Salmon-colored points lie at the boundary between two clusters and were not assigned.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/10/2177/F5.medium.gif" width="439" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/5/10/2177/F5.medium.gif" width="439" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/10/2177/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/10/2177/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466896" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-55" class="first-child">Plot of islet expression <em>vs.</em> observed genotype for an example probe. Points are colored by the inferred genotype, based on a k-nearest neighbor classifier, with yellow, green, and blue corresponding to BB, BR, and RR, respectively, where B = B6 and R = BTBR. Salmon-colored points lie at the boundary between two clusters and were not assigned.</p><div class="sb-div caption-clear"></div></div></div><p id="p-56">For sets of probes mapping to approximately the same genomic location, we considered the probes’ expression jointly. Examples of pairs of probes mapping to the same location are shown in <a href="http://www.g3journal.org/content/suppl/2015/08/19/g3.115.019778.DC1/FigureS13.pdf">Figure S13</a>, with points colored by observed eQTL genotype.</p><p id="p-57">We considered 45–115 eQTL per tissue; their locations on the genetic map of markers is shown in <a href="http://www.g3journal.org/content/suppl/2015/08/19/g3.115.019778.DC1/FigureS14.pdf">Figure S14</a>. The majority of eQTL had a single corresponding probe. There were 3–14 eQTL per tissue with a pair of corresponding probes. For islet, there were three eQTL with three corresponding probes, and for adipose there was one such trio.</p><p id="p-58">For each tissue, we calculated the proportion of matches between the observed eQTL genotypes for each DNA sample and the inferred eQTL genotypes from each mRNA sample, as a measure of similarity between the DNA and mRNA samples. We further calculated a combined measure of similarity as the overall proportion of matches, pooling all six tissues.</p><p id="p-59"><a href="http://www.g3journal.org/content/suppl/2015/08/19/g3.115.019778.DC1/FigureS15.pdf">Figure S15</a> contains histograms of the similarity measures for each tissue, separating the self-self similarities (the diagonal elements) and the self-nonself similarities (the off-diagonal elements). There are a number of clear outliers: small self-self similarities and large self-nonself similarities.</p><p id="p-60">To identify problem DNA samples, we again considered the self similarity <em>vs.</em> the maximum similarity (that is, the values on the diagonal of the similarity matrix <em>vs.</em> the maximum values in each row). <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a> contains a scatterplot of these values. Gray points, with maximum similarity equal to the self similarity, are inferred to be corrected labeled. Green points, with small self similarity but large maximum similarity, are inferred to be incorrect, but are fixable. Red points concern DNA samples for which no corresponding mRNA sample can be found.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/10/2177/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Self-similarity (proportion matches between observed and inferred expression quantitative trait loci genotypes, combined across tissues) vs. maximum similarity for the DNA samples. The diagonal gray line corresponds to equality. Samples with missing self similarity (at bottom) were not intended to have expression assays performed. Gray points correspond to DNA samples that were correctly labeled. Green points correspond to sample mix-ups that are fixable (the correct label can be determined). Red points comprise both samples mix-ups that cannot be corrected as well as samples that may be correct but cannot be verified as no expression data are available." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1379678328" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Self-similarity (proportion matches between observed and inferred expression quantitative trait loci genotypes, combined across tissues) &lt;em&gt;vs.&lt;/em&gt; maximum similarity for the DNA samples. The diagonal gray line corresponds to equality. Samples with missing self similarity (at bottom) were not intended to have expression assays performed. Gray points correspond to DNA samples that were correctly labeled. Green points correspond to sample mix-ups that are fixable (the correct label can be determined). Red points comprise both samples mix-ups that cannot be corrected as well as samples that may be correct but cannot be verified as no expression data are available.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/10/2177/F6.medium.gif" width="435" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/5/10/2177/F6.medium.gif" width="435" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/10/2177/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/10/2177/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466782" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-61" class="first-child">Self-similarity (proportion matches between observed and inferred expression quantitative trait loci genotypes, combined across tissues) <em>vs.</em> maximum similarity for the DNA samples. The diagonal gray line corresponds to equality. Samples with missing self similarity (at bottom) were not intended to have expression assays performed. Gray points correspond to DNA samples that were correctly labeled. Green points correspond to sample mix-ups that are fixable (the correct label can be determined). Red points comprise both samples mix-ups that cannot be corrected as well as samples that may be correct but cannot be verified as no expression data are available.</p><div class="sb-div caption-clear"></div></div></div><p id="p-62">Detailed results for the six tissues, with tissue-specific similarity values, are shown in <a href="http://www.g3journal.org/content/suppl/2015/08/19/g3.115.019778.DC1/FigureS16.pdf">Figure S16</a>. The points are colored as in <a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a>, based on the combined similarity measure. The points with missing self similarity (at the bottom of each panel) were not intended to be assayed for gene expression in that tissue. The tissue-specific results are concordant with the overall conclusions, with two caveats. First, there are a number of green points (corresponding to mislabeled, but fixable, DNA samples), with low maximum similarity in each tissue. These correspond to samples for which gene expression assays were not performed for that tissue, the bulk of which are for the 27 samples that were assayed only for gene expression in kidney and the 43 samples that were assayed for all tissues except kidney. Second, for hypo, the strength of eQTL associations were weaker, and fewer eQTL were considered, than for the other tissues, and so there is less separation between the green and pink points.</p><p id="p-63">In <a href="http://www.g3journal.org/content/suppl/2015/08/19/g3.115.019778.DC1/FigureS17.pdf">Figure S17</a>, we display the second-highest similarity <em>vs.</em> the maximum similarity, for the combined similarity measures accounting for all tissues. The fixable mislabeled samples (in green) are all well away from the diagonal, indicating good support for our ability to infer the correct label.</p><p id="p-64">The inferred mix-ups among the DNA samples are displayed in <a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a> according to the arrangement of the samples on the 96-well genotyping plates. Black dots indicate that the correct DNA sample was placed in the correct well. The blue arrows point from the well in which a DNA sample was supposed to be placed, to the well where it was actually placed. For example, on plate 1631, the sample in well D02 was placed in the correct well but was also placed in well B03. The sample that belonged in B03 was placed in B04, the sample that belonged in B04 was placed in E03, and the sample belonging in E03 was not found (but, as indicated by the green arrowhead, there was no corresponding gene expression data).</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/10/2177/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The DNA sample mix-ups on the seven 96-well plates used for genotyping. Black dots indicate that the correct DNA was put in the well. Blue arrows point from where a sample should have been placed to where it was actually placed; the different shades of blue convey no meaning. Red X&#x2019;s indicate DNA samples that were omitted. Orange arrowheads indicate wells with incorrect samples, but the sample placed there is of unknown origin. Purple and green arrowheads indicate cases in which the sample placed in the well was incorrect, but the DNA that was supposed to be there was not found; with the purple cases, there was corresponding gene expression data, while for the green cases, there was no corresponding gene expression data. Pink circles (e.g., well D02 on plate 1631) indicate sample duplicates. Gray dots indicate that the sample placed in the well cannot be verified, as there was no corresponding gene expression data. Gray circles indicate controls or unused wells." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1379678328" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The DNA sample mix-ups on the seven 96-well plates used for genotyping. Black dots indicate that the correct DNA was put in the well. Blue arrows point from where a sample should have been placed to where it was actually placed; the different shades of blue convey no meaning. Red X&#x2019;s indicate DNA samples that were omitted. Orange arrowheads indicate wells with incorrect samples, but the sample placed there is of unknown origin. Purple and green arrowheads indicate cases in which the sample placed in the well was incorrect, but the DNA that was supposed to be there was not found; with the purple cases, there was corresponding gene expression data, while for the green cases, there was no corresponding gene expression data. Pink circles (&lt;em&gt;e.g.&lt;/em&gt;, well D02 on plate 1631) indicate sample duplicates. Gray dots indicate that the sample placed in the well cannot be verified, as there was no corresponding gene expression data. Gray circles indicate controls or unused wells.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/10/2177/F7.medium.gif" width="351" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/5/10/2177/F7.medium.gif" width="351" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/10/2177/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/10/2177/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466742" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-65" class="first-child">The DNA sample mix-ups on the seven 96-well plates used for genotyping. Black dots indicate that the correct DNA was put in the well. Blue arrows point from where a sample should have been placed to where it was actually placed; the different shades of blue convey no meaning. Red X’s indicate DNA samples that were omitted. Orange arrowheads indicate wells with incorrect samples, but the sample placed there is of unknown origin. Purple and green arrowheads indicate cases in which the sample placed in the well was incorrect, but the DNA that was supposed to be there was not found; with the purple cases, there was corresponding gene expression data, while for the green cases, there was no corresponding gene expression data. Pink circles (<em>e.g.</em>, well D02 on plate 1631) indicate sample duplicates. Gray dots indicate that the sample placed in the well cannot be verified, as there was no corresponding gene expression data. Gray circles indicate controls or unused wells.</p><div class="sb-div caption-clear"></div></div></div><p id="p-66">Although there were many long-range sample swaps, particularly for samples belonging in the eleventh column of plate 1629, the bulk of the errors occurred on plates 1632 and 1630, with a long series of off-by-one and off-by-two errors indicative of single-channel pipetting mistakes.</p><p id="p-67">Let us describe a small portion of the further errors. On plate 1632, the sample belonging in well E07 was placed in the correct well but was also placed in the well below, F07. The sample belonging in well F07 was not found but had no corresponding gene expression data. The sample placed in well G07 was incorrect but had no corresponding gene expression data, and so presumably corresponds to that which should have been in the well above, F07. The sample belonging in well G07 was placed one below, H07. There are then a series of off-by-one errors, except that the sample belonging in well C09 was actually placed in well G01, while the sample belonging in well D09 was placed in both well E01 and on plate 1629 (well C11).</p><p id="p-68">Of the 554 DNA samples that were genotyped, 10 were omitted due to poorly behaved genotypes (including a pair of replicates), 435 were found to be correctly labeled, and 8 were possibly correct but could not be verified due to lack of gene expression assays. However, five samples were duplicates of other samples, 84 were incorrectly labeled but the correct label could be assigned, and 12 were incorrectly labeled and the correct label could not be identified. Thus, at least 18% of the samples were involved in sample mix-ups.</p><p id="p-69">We had initially become suspicious of possible sample mix-ups through the identification of 36 mice whose X chromosome genotypes were inconsistent with their sex. After correction of the sample mix-ups, there were no such discrepancies. Only a small portion of the problems were identified through such sex/genotype incompatibilities, because the majority of sample mix-ups were off-by-one errors in the genotype plates, and the samples were arranged on the plates so that adjacent samples were often the same sex.</p><p id="p-70">The large discrepancies between expression and eQTL genotype seen in <a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a> and <a href="http://www.g3journal.org/content/suppl/2015/08/19/g3.115.019778.DC1/FigureS13.pdf">Figure S13</a> are largely eliminated after correction of the inferred sample mix-ups. <a href="http://www.g3journal.org/content/suppl/2015/08/19/g3.115.019778.DC1/FigureS18.pdf">Figure S18</a> shows the same examples but with the corrected data. <a href="http://www.g3journal.org/content/suppl/2015/08/19/g3.115.019778.DC1/FigureS18.pdf">Figure S18</a>, A−D correspond to the panels in <a href="http://www.g3journal.org/content/suppl/2015/08/19/g3.115.019778.DC1/FigureS13.pdf">Figure S13</a>; the genotypes are now more clearly separated, although some overlap remains and there are a few outliers (most notably, in <a href="http://www.g3journal.org/content/suppl/2015/08/19/g3.115.019778.DC1/FigureS18.pdf">Figure S18B</a>). <a href="http://www.g3journal.org/content/suppl/2015/08/19/g3.115.019778.DC1/FigureS18.pdf">Figure S18</a>E corresponds to <a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5</a>; after correction of the sample mix-ups, there is no overlap between the three genotype groups.</p></div><div id="sec-15" class="subsection"><h3>QTL mapping results</h3><p id="p-71">It should come as no surprise that the correction of the sample mix-ups, particularly the 18% mix-ups in the DNA samples, leads to great improvement in QTL mapping results. <a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure 8</a> contains LOD curves for 10-week insulin level with the original and corrected datasets. With the original data, four chromosomes had LOD score &gt;4; after correction of the sample mix-ups, nine chromosomes have LOD score &gt;4.</p><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/10/2177/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="LOD curves for 10-week insulin level, before (red) and after (blue) correction of the sample mix-ups." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1379678328" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;LOD curves for 10-week insulin level, before (red) and after (blue) correction of the sample mix-ups.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 8" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/10/2177/F8.medium.gif" width="440" height="302"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 8" src="https://www.g3journal.org/content/ggg/5/10/2177/F8.medium.gif" width="440" height="302"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/10/2177/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/10/2177/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466939" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 8</span> <p id="p-72" class="first-child">LOD curves for 10-week insulin level, before (red) and after (blue) correction of the sample mix-ups.</p><div class="sb-div caption-clear"></div></div></div><p id="p-73">Two coat-related traits were recorded for the F<sub>2</sub> mice: agouti and tufted coats. Concerning the agouti coat: BTBR mice have tan coats, whereas B6 mice are black; this is due to a gene on chromosome 2, and the BTBR allele is dominant. Mapping the agouti coat color as a binary phenotype, the LOD score on chromosome 2 increased from 64 to 110 after correction of the sample mix-ups (<a href="http://www.g3journal.org/content/suppl/2015/08/19/g3.115.019778.DC1/FigureS19.pdf">Figure S19A</a>). Although the corrected data still contained inconsistencies between genotype and coat color, the number of inconsistencies decreased from 47 to 7 (<a href="http://www.g3journal.org/content/suppl/2015/08/19/g3.115.019778.DC1/TableS4.pdf">Table S4</a>).</p><p id="p-74">Tufted coat is due to a single gene on chromosome 17, with the BTBR allele (with the tufted phenotype) being recessive to the B6 allele (not tufted). Mapping this phenotype as a binary trait, the LOD score on chromosome 17 increased from 64 to 107 after correction of the sample mix-ups (<a href="http://www.g3journal.org/content/suppl/2015/08/19/g3.115.019778.DC1/FigureS19.pdf">Figure S19B</a>). Although, as with agouti, the corrected data still contained inconsistencies between genotype and phenotype, the number of inconsistencies decreased from 37 to 4 (<a href="http://www.g3journal.org/content/suppl/2015/08/19/g3.115.019778.DC1/TableS5.pdf">Table S5</a>).</p><p id="p-75">Finally the corrected data resulted in a great increase in the numbers of inferred eQTL in the six tissues (<a id="xref-fig-9-1" class="xref-fig" href="#F9">Figure 9</a>). For each array probe with known genomic position, we performed a genome scan, including sex as an interactive covariate (that is, allowing the QTL effect to be different in the two sexes). For each array probe, we counted the number of chromosomes with a peak LOD score above 5. Such a peak, on the chromosome containing the probe, was considered a local eQTL if the 2-LOD support interval contained the probe location; other peaks were called <em>trans</em>-eQTL. The inferred number of local eQTL increased by 7% across tissues (with a somewhat-smaller increase in hypo). The inferred number of <em>trans</em>-eQTL increased by 37% across tissues (although only by 8% in hypo). The modest increases in hypo were due in part to the omission of 119 poorly behaved arrays. The increased numbers of inferred eQTL is also seen with more stringent thresholds; the numbers of eQTL with LOD ≥ 10 are shown in <a href="http://www.g3journal.org/content/suppl/2015/08/19/g3.115.019778.DC1/FigureS20.pdf">Figure S20</a>.</p><div id="F9" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/10/2177/F9.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Numbers of identified local and trans-eQTL with LOD &#x2265; 5, with the original data (red) and after correction of the sample mix-ups (blue), across 37,797 array probes with known genomic location. An expression quantitative trait loci (eQTL) was considered local if the 2-LOD support interval contained the corresponding probe; otherwise it was considered trans." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1379678328" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Numbers of identified local and &lt;em&gt;trans&lt;/em&gt;-eQTL with LOD &#x2265; 5, with the original data (red) and after correction of the sample mix-ups (blue), across 37,797 array probes with known genomic location. An expression quantitative trait loci (eQTL) was considered local if the 2-LOD support interval contained the corresponding probe; otherwise it was considered &lt;em&gt;trans&lt;/em&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 9" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/10/2177/F9.medium.gif" width="440" height="280"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 9" src="https://www.g3journal.org/content/ggg/5/10/2177/F9.medium.gif" width="440" height="280"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/10/2177/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 9" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/10/2177/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466775" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 9</span> <p id="p-76" class="first-child">Numbers of identified local and <em>trans</em>-eQTL with LOD ≥ 5, with the original data (red) and after correction of the sample mix-ups (blue), across 37,797 array probes with known genomic location. An expression quantitative trait loci (eQTL) was considered local if the 2-LOD support interval contained the corresponding probe; otherwise it was considered <em>trans</em>.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-16"><h2 class="">Discussion</h2><p id="p-77">In a mouse intercross with more than 500 animals and gene expression microarray data on six tissues, we identified and corrected sample mix-ups involving 18% of the DNAs, along with a small number of mix-ups in each batch of expression arrays. The QTL mapping results improved markedly after the correction of mix-ups, but it was perhaps most surprising just how strong the results were before the corrections.</p><p id="p-78">To align the expression arrays, we first identified subsets of genes with strong between-tissue correlation in expression and then considered the correlations between samples across these subsets of genes. To align genotypes and expression arrays, we identified transcripts with strong local eQTL, formed predictors of eQTL genotype from expression values, and calculated the proportion of matches between the observed eQTL genotypes for a DNA sample and the predicted eQTL genotypes for an mRNA sample.</p><p id="p-79">This approach applies quite generally: Whenever one has two data matrices, <em>X</em> and <em>Y</em>, whose rows should correspond, one should check that the rows do in fact correspond. The simplest approach is to first identify subsets of associated columns (in which a column of <em>X</em> is associated with a column of <em>Y</em>) and then calculate some measure of similarity between rows of <em>X</em> and rows of <em>Y</em>, across that subset of columns.</p><p id="p-80">Similar approaches have been described by a number of groups. <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Westra <em>et al.</em> (2011)</a> considered a number of public datasets and found an overall rate of 3% sample mix-ups, with one dataset (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Choy <em>et al.</em> 2008</a>) having 23% mix-ups. <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Schadt <em>et al.</em> (2012)</a> showed that, with the tight connection between genotypes and gene expression phenotypes, external eQTL information can, in principle, be used to identify individuals participating in a gene expression study: Genome-wide gene expression is just as revealing of individual identities as genome-wide genotype data. <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Lynch <em>et al.</em> (2012)</a> highlighted issues arising in large tumor studies and focused particularly on a number of experimental design issues, such as plate layout. <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Ekstrøm and Feenstra (2012)</a> considered the identification of sample mix-ups in genome-wide association studies, focusing on a small number of phenotypes, such as blood group data, with strong genotype-phenotype associations. Also relevant is the forensic bioinformatics work of <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Baggerly and Coombes (2008</a>, <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">2009</a>), particularly their efforts to correct mix-ups in data files. Finally, <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Jun <em>et al.</em> (2012)</a> recently described methods for detecting mixtures in DNA samples based on genotype or sequencing data, and there is considerable work on detecting mislabeled microarrays (<em>e.g.</em>, <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Zhang <em>et al.</em> 2009</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bootkrajang and Kabán 2013</a>).</p><p id="p-81">There are a number of opportunities for improvement in our approach. In particular, a number of critical parameters (such as the LOD score for choosing eQTL, and the number of nearest neighbors and the minimum vote in the <em>k</em>-nearest neighbor classifier) were chosen in an <em>ad hoc</em> way. The choice of such parameters influences the variation within and the separation between the self-self and self-nonself distributions of similarity measures, and thus our ability to identify errors. In addition, other classification methods might be used, although the <em>k</em>-nearest neighbor classifier has an important advantage: It works well even in the presence of misclassification error in the “training” data.</p><p id="p-82">Perhaps the most important lesson from this work is the value of investigating aberrations. One should follow up any observed inconsistencies in data, to identify the source. In particular, one should not rely solely on LOD scores or other summary statistics, but also inspect plots of genotype <em>vs.</em> phenotype, such as that in <a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5</a>.</p><p id="p-83">Of course, there are many possible errors that we couldn’t see by these approaches. For example, all of the tissues (including the DNA) for a pair of animals might be swapped, or there may be mix-ups within the clinical phenotypes (such as plasma insulin levels). And some mix-ups are detectable but not correctable.</p><p id="p-84">We have not identified any between-tissue mix-ups in the expression data, but such errors are possible. For that type of error, it may be useful to consider the gene expression bar code developed by <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Zilliox and Irizarry (2007)</a>.</p><p id="p-85">The correction of inferred sample mix-ups, as we have done, may introduce bias toward larger estimated eQTL effects. We believe that, in the current study, there is little risk of such bias, because the data provide strong evidence for specific sample labels. If the correction of sample mix-ups were accompanied by a greater level of uncertainty, one might consider omitting samples rather than assigning the inferred labels, though such an approach could also incur some bias.</p><p id="p-86">Finally, one might ask, following these findings: What is an acceptable error rate in a research study? And what laboratory procedures should be instituted to avoid such errors? There exist procedures to help protect against errors, both for genotypes (<em>e.g.</em>, <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Huijsmans <em>et al.</em> 2007a</a>,<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">b</a>) and for microarrays (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Grant <em>et al.</em> 2003</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Imbeaud and Auffray 2005</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Walter <em>et al.</em> 2010</a>), but they are not always put into practice. However, as the current study indicates, with expression genetic data, one can accommodate a high rate of errors provided that one applies appropriate procedures to detect and correct such errors.</p></div><div class="section" id="sec-17"></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-87">We thank Angie Oler, Mary Rabaglia, Kathryn Schueler, and Donald Stapleton for their work on the underlying project and Amit Kulkarni for providing annotation information for the expression microarrays. This work was supported in part by National Institutes of Health grants GM074244 (to K.W.B.), DK066369 (to A.D.A), and GM012756 (to C. K.).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-supplementary-material" id="fn-2"><p id="p-88">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019778/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019778/-/DC1</a></p></li><li class="fn" id="fn-3"><p id="p-89"><em>Communicating editor: S. I. Wright</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2015-06-15"><span class="received-label">Received </span>June 15, 2015.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2015-08-19"><span class="accepted-label">Accepted </span>August 19, 2015.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2015 Broman <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2177.1" data-doi="10.1200/JCO.2007.15.1951"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baggerly</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coombes</span>  <span class="cit-name-given-names">K. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Run batch effects potentially compromise the usefulness of genomic signatures for ovarian cancer.</span> <abbr class="cit-jnl-abbrev">J. Clin. Oncol.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">1186</span>–<span class="cit-lpage">1187</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BClinical%2BOncology%26rft.stitle%253DJCO%26rft.aulast%253DBaggerly%26rft.auinit1%253DK.%2BA.%26rft.volume%253D26%26rft.issue%253D7%26rft.spage%253D1186%26rft.epage%253D1187%26rft.atitle%253DRun%2BBatch%2BEffects%2BPotentially%2BCompromise%2Bthe%2BUsefulness%2Bof%2BGenomic%2BSignatures%2Bfor%2BOvarian%2BCancer%26rft_id%253Dinfo%253Adoi%252F10.1200%252FJCO.2007.15.1951%26rft_id%253Dinfo%253Apmid%252F18309960%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNvIjtzOjU6InJlc2lkIjtzOjk6IjI2LzcvMTE4NiI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTAvMjE3Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2177.2" data-doi="10.1214/09-AOAS291"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baggerly</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coombes</span>  <span class="cit-name-given-names">K. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Deriving chemosensitivity from cell lines: forensic bioinformatics and reproducible research in high-throughput biology.</span> <abbr class="cit-jnl-abbrev">Ann. Appl. Stat.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">1309</span>–<span class="cit-lpage">1334</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BAppl.%2BStat.%26rft.volume%253D3%26rft.spage%253D1309%26rft_id%253Dinfo%253Adoi%252F10.1214%252F09-AOAS291%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1214/09-AOAS291&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000283528200011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2177.3" data-doi="10.1093/bioinformatics/btt078"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bootkrajang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kabán</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Classification of mislabelled microarrays using robust sparse logistic regression.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">870</span>–<span class="cit-lpage">877</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtt078%26rft_id%253Dinfo%253Apmid%252F23418189%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyOS83Lzg3MCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTAvMjE3Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2177.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Mapping quantitative trait loci in the case of a spike in the phenotype distribution.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">163</span>: <span class="cit-fpage">1169</span>–<span class="cit-lpage">1175</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DBroman%26rft.auinit1%253DK.%2BW.%26rft.volume%253D163%26rft.issue%253D3%26rft.spage%253D1169%26rft.epage%253D1175%26rft.atitle%253DMapping%2BQuantitative%2BTrait%2BLoci%2Bin%2Bthe%2BCase%2Bof%2Ba%2BSpike%2Bin%2Bthe%2BPhenotype%2BDistribution%26rft_id%253Dinfo%253Apmid%252F12663553%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2My8zLzExNjkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzEwLzIxNzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2177.5" data-doi="10.1093/bioinformatics/btg112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">R/qtl: QTL mapping in experimental crosses.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">889</span>–<span class="cit-lpage">890</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DBroman%26rft.auinit1%253DK.%2BW.%26rft.volume%253D19%26rft.issue%253D7%26rft.spage%253D889%26rft.epage%253D890%26rft.atitle%253DR%252Fqtl%253A%2BQTL%2Bmapping%2Bin%2Bexperimental%2Bcrosses%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtg112%26rft_id%253Dinfo%253Apmid%252F12724300%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIxOS83Lzg4OSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTAvMjE3Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2177.6" data-doi="10.1007/BF02996226"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carter</span>  <span class="cit-name-given-names">T. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Falconer</span>  <span class="cit-name-given-names">D. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1951</span> <span class="cit-article-title">Stocks for detecting linkage in the mouse, and the theory of their design.</span> <abbr class="cit-jnl-abbrev">J. Genet.</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">307</span>–<span class="cit-lpage">323</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BGenet.%26rft.volume%253D50%26rft.spage%253D307%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF02996226%26rft_id%253Dinfo%253Apmid%252F24539711%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF02996226&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24539711&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2177.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1951XX17400014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2177.7" data-doi="10.1371/journal.pgen.1000287"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Choy</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yelensky</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bonakdar</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plenge</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saxena</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Genetic analysis of human traits in vitro: drug response and gene expression in lymphoblastoid cell lines.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e1000287</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DChoy%26rft.auinit1%253DE.%26rft.volume%253D4%26rft.issue%253D11%26rft.spage%253De1000287%26rft.epage%253De1000287%26rft.atitle%253DGenetic%2Banalysis%2Bof%2Bhuman%2Btraits%2Bin%2Bvitro%253A%2Bdrug%2Bresponse%2Band%2Bgene%2Bexpression%2Bin%2Blymphoblastoid%2Bcell%2Blines.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000287%26rft_id%253Dinfo%253Apmid%252F19043577%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000287&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19043577&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2177.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2177.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ekstrøm</span>  <span class="cit-name-given-names">C. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feenstra</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Detecting sample misidentifications in genetic association studies.</span> <abbr class="cit-jnl-abbrev">Stat. Appl. Genet. Mol. Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">13</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStat.%2BAppl.%2BGenet.%2BMol.%2BBiol.%26rft.volume%253D11%26rft.spage%253D13%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2177.9" data-doi="10.1002/bit.10847"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grant</span>  <span class="cit-name-given-names">G. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manduchi</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pizarro</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stoeckert</span>  <span class="cit-name-given-names">C. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Maintaining data integrity in microarray data management.</span> <abbr class="cit-jnl-abbrev">Biotechnol. Bioeng.</abbr> <span class="cit-vol">84</span>: <span class="cit-fpage">795</span>–<span class="cit-lpage">800</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiotechnology%2Band%2BBioengineering%26rft.stitle%253DBiotechnology%2Band%2BBioengineering%26rft.aulast%253DGrant%26rft.auinit1%253DG.%2BR.%26rft.volume%253D84%26rft.issue%253D7%26rft.spage%253D795%26rft.epage%253D800%26rft.atitle%253DMaintaining%2Bdata%2Bintegrity%2Bin%2Bmicroarray%2Bdata%2Bmanagement.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fbit.10847%26rft_id%253Dinfo%253Apmid%252F14708120%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/bit.10847&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14708120&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2177.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2177.10" data-doi="10.1038/hdy.1992.131"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haley</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knott</span>  <span class="cit-name-given-names">S. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">A simple regression method for mapping quantitative trait loci in line crosses using flanking markers.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">69</span>: <span class="cit-fpage">315</span>–<span class="cit-lpage">324</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DHaley%26rft.auinit1%253DC.%2BS.%26rft.volume%253D69%26rft.issue%253D4%26rft.spage%253D315%26rft.epage%253D324%26rft.atitle%253DA%2Bsimple%2Bregression%2Bmethod%2Bfor%2Bmapping%2Bquantitative%2Btrait%2Bloci%2Bin%2Bline%2Bcrosses%2Busing%2Bflanking%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.1992.131%26rft_id%253Dinfo%253Apmid%252F16718932%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.1992.131&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16718932&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2177.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1992JQ49200003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2177.11" data-doi="10.1111/j.1423-0410.2006.00871.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huijsmans</span>  <span class="cit-name-given-names">C. J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heilmann</span>  <span class="cit-name-given-names">F. G. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van der Zanden</span>  <span class="cit-name-given-names">A. G. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schneeberger</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hermans</span>  <span class="cit-name-given-names">M. H. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span><span class="cit-comment">a</span> <span class="cit-article-title">Single nucleotide polymorphism profiling assay to exclude serum sample mix-up.</span> <abbr class="cit-jnl-abbrev">Vox Sang.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">148</span>–<span class="cit-lpage">153</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVox%2Bsanguinis%26rft.stitle%253DVox%2BSang%26rft.aulast%253DHuijsmans%26rft.auinit1%253DC.%2BJ.%26rft.volume%253D92%26rft.issue%253D2%26rft.spage%253D148%26rft.epage%253D153%26rft.atitle%253DSingle%2Bnucleotide%2Bpolymorphism%2Bprofiling%2Bassay%2Bto%2Bexclude%2Bserum%2Bsample%2Bmix-up.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1423-0410.2006.00871.x%26rft_id%253Dinfo%253Apmid%252F17298578%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1423-0410.2006.00871.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17298578&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2177.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2177.12" data-doi="10.2353/jmoldx.2007.060059"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huijsmans</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Damen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van der Linden</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hermans</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span><span class="cit-comment">b</span> <span class="cit-article-title">Single nucleotide polymorphism profiling assay to confirm the identity of human tissues.</span> <abbr class="cit-jnl-abbrev">J. Mol. Diagn.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">205</span>–<span class="cit-lpage">213</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BDiagnostics%26rft.stitle%253DJ.%2BMol.%2BDiagn.%26rft.aulast%253DHuijsmans%26rft.auinit1%253DR.%26rft.volume%253D9%26rft.issue%253D2%26rft.spage%253D205%26rft.epage%253D213%26rft.atitle%253DSingle%2BNucleotide%2BPolymorphism%2BProfiling%2BAssay%2Bto%2BConfirm%2Bthe%2BIdentity%2Bof%2BHuman%2BTissues%26rft_id%253Dinfo%253Adoi%252F10.2353%252Fjmoldx.2007.060059%26rft_id%253Dinfo%253Apmid%252F17384212%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2353/jmoldx.2007.060059&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17384212&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2177.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2177.13" data-doi="10.1016/S1359-6446(05)03565-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Imbeaud</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auffray</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">‘The 39 steps’ in gene expression profiling: critical issues and proposed best practices for microarray experiments.</span> <abbr class="cit-jnl-abbrev">Drug Discov. Today</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">1175</span>–<span class="cit-lpage">1182</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDrug%2Bdiscovery%2Btoday%26rft.stitle%253DDrug%2BDiscov%2BToday%26rft.aulast%253DImbeaud%26rft.auinit1%253DS.%26rft.volume%253D10%26rft.issue%253D17%26rft.spage%253D1175%26rft.epage%253D1182%26rft.atitle%253D%2527The%2B39%2Bsteps%2527%2Bin%2Bgene%2Bexpression%2Bprofiling%253A%2Bcritical%2Bissues%2Band%2Bproposed%2Bbest%2Bpractices%2Bfor%2Bmicroarray%2Bexperiments.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1359-6446%252805%252903565-8%26rft_id%253Dinfo%253Apmid%252F16182210%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1359-6446(05)03565-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16182210&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2177.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000231377200011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2177.14" data-doi="10.1016/j.ajhg.2012.09.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jun</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flickinger</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hetrick</span>  <span class="cit-name-given-names">K. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Romm</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doheny</span>  <span class="cit-name-given-names">K. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Detecting and estimating contamination of human DNA samples in sequencing and array-based genotype data.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">839</span>–<span class="cit-lpage">848</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DJun%26rft.auinit1%253DG.%26rft.volume%253D91%26rft.issue%253D5%26rft.spage%253D839%26rft.epage%253D848%26rft.atitle%253DDetecting%2Band%2Bestimating%2Bcontamination%2Bof%2Bhuman%2BDNA%2Bsamples%2Bin%2Bsequencing%2Band%2Barray-based%2Bgenotype%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2012.09.004%26rft_id%253Dinfo%253Apmid%252F23103226%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2012.09.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23103226&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2177.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2177.15" data-doi="10.1101/gr.074914.107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Keller</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Choi</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rabaglia</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A gene expression network model of type 2 diabetes links cell cycle regulation in islets with diabetes susceptibility.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">706</span>–<span class="cit-lpage">716</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.074914.107%26rft_id%253Dinfo%253Apmid%252F18347327%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE4LzUvNzA2IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMC8yMTc3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2177.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lander</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Botstein</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Mapping Mendelian factors underlying quantitative traits using RFLP linkage maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">121</span>: <span class="cit-fpage">185</span>–<span class="cit-lpage">199</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLander%26rft.auinit1%253DE.%2BS.%26rft.volume%253D121%26rft.issue%253D1%26rft.spage%253D185%26rft.epage%253D199%26rft.atitle%253DMapping%2Bmendelian%2Bfactors%2Bunderlying%2Bquantitative%2Btraits%2Busing%2BRFLP%2Blinkage%2Bmaps.%26rft_id%253Dinfo%253Apmid%252F2563713%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTIxLzEvMTg1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMC8yMTc3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2177.17" data-doi="10.1371/journal.pone.0041815"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chin</span>  <span class="cit-name-given-names">S.-F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dunning</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Caldas</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tavaré</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Calling sample mix-ups in cancer population studies.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e41815</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DLynch%26rft.auinit1%253DA.%2BG.%26rft.volume%253D7%26rft.issue%253D8%26rft.spage%253De41815%26rft.epage%253De41815%26rft.atitle%253DCalling%2Bsample%2Bmix-ups%2Bin%2Bcancer%2Bpopulation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0041815%26rft_id%253Dinfo%253Apmid%252F22912679%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0041815&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22912679&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2177.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-book" id="cit-5.10.2177.18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">R Development Core Team</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-source">R: A Language and Environment for Statistical Computing</span>. <span class="cit-publ-name">R Foundation for Statistical Computing</span>, <span class="cit-publ-loc">Vienna, Austria</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2177.19" data-doi="10.1038/ng.2248"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schadt</span>  <span class="cit-name-given-names">E. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Woo</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hao</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Bayesian method to predict individual SNP genotypes from gene expression data.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">603</span>–<span class="cit-lpage">608</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DSchadt%26rft.auinit1%253DE.%2BE.%26rft.volume%253D44%26rft.issue%253D5%26rft.spage%253D603%26rft.epage%253D608%26rft.atitle%253DBayesian%2Bmethod%2Bto%2Bpredict%2Bindividual%2BSNP%2Bgenotypes%2Bfrom%2Bgene%2B%2Bexpression%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2248%26rft_id%253Dinfo%253Apmid%252F22484626%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2248&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22484626&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2177.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2177.20" data-doi="10.2144/000113421"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Walter</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Honegger</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schweizer</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poths</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bonin</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Utilization of AFFX spike-in control probes to monitor sample identity throughout Affymetrix GeneChip Array processing.</span> <abbr class="cit-jnl-abbrev">Biotechniques</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">371</span>–<span class="cit-lpage">378</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiotechniques%26rft.volume%253D48%26rft.spage%253D371%26rft_id%253Dinfo%253Adoi%252F10.2144%252F000113421%26rft_id%253Dinfo%253Apmid%252F20569210%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2144/000113421&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20569210&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2177.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2177.21" data-doi="10.1093/bioinformatics/btr323"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Westra</span>  <span class="cit-name-given-names">H.-J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jansen</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fehrmann</span>  <span class="cit-name-given-names">R. S. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">te Meerman</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Heel</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">MixupMapper: correcting sample mix-ups in genome-wide datasets increases power to detect small genetic effects.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">2104</span>–<span class="cit-lpage">2111</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr323%26rft_id%253Dinfo%253Apmid%252F21653519%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjcvMTUvMjEwNCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTAvMjE3Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2177.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Atchley</span>  <span class="cit-name-given-names">W. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Mapping quantitative trait loci for complex binary diseases using line crosses.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">143</span>: <span class="cit-fpage">1417</span>–<span class="cit-lpage">1424</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DXu%26rft.auinit1%253DS.%26rft.volume%253D143%26rft.issue%253D3%26rft.spage%253D1417%26rft.epage%253D1424%26rft.atitle%253DMapping%2BQuantitative%2BTrait%2BLoci%2Bfor%2BComplex%2BBinary%2BDiseases%2BUsing%2BLine%2BCrosses%26rft_id%253Dinfo%253Apmid%252F8807312%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE0My8zLzE0MTciO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzEwLzIxNzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2177.23" data-doi="10.1093/bioinformatics/btp478"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blanzieri</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Methods for labeling error detection in microarrays based on the effect of data perturbation on the regression model.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2708</span>–<span class="cit-lpage">2714</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp478%26rft_id%253Dinfo%253Apmid%252F19661242%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMjAvMjcwOCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTAvMjE3Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2177.24" data-doi="10.1038/nmeth1102"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zilliox</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Irizarry</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A gene expression bar code for microarray data.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">911</span>–<span class="cit-lpage">913</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DZilliox%26rft.auinit1%253DM.%2BJ.%26rft.volume%253D4%26rft.issue%253D11%26rft.spage%253D911%26rft.epage%253D913%26rft.atitle%253DA%2Bgene%2Bexpression%2Bbar%2Bcode%2Bfor%2Bmicroarray%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth1102%26rft_id%253Dinfo%253Apmid%252F17906632%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth1102&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17906632&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2177.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000250575700013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/5/10/2177.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/10/2165" title="Similar Efficacies of Selection Shape Mitochondrial and Nuclear Genes in Both Drosophila melanogaster and Homo sapiens" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/10/2187" title="Genetic Networks Required to Coordinate Chromosome Replication by DNA Polymerases α, δ, and ε in Saccharomyces cerevisiae" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/5/10"><span>Volume 5 Issue 10, October 2015</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/5/10.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/5/10.cover-source.jpg?itok=JwgNhVLw" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 5 (10)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/10/2177?uritype=cgi&amp;view=full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-JpsomqVD_LmAUxB_pqrnQeFnS6OLx6aFVAhUAs6RLXo" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/5/10/2177" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/5/10/2177" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/5/10/2177" class="active" data-icon-position="" data-hide-link-title="0">Identification and Correction of Sample Mix-Ups in Expression Genetic Data: A Case Study</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460044" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-3pn6nYb18avyksB93QY9E4zytTAg9bW216cIV4HBV7A" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/10/2177.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/5/10/2177" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-VBba4k7A3dkrj-24kNgcXrV8DsWUHW8tPyfgRFMmgz8" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/5/10/2177" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/10/2177.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460044" id="node460044--2" data-pisa="ggg;5/10/2177" data-pisa-master="ggg;g3.115.019778" data-apath="/ggg/5/10/2177.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Identification and Correction of Sample Mix-Ups in Expression Genetic Data: A Case Study</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-4914-6671" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Karl W. Broman</span>, <span class="highwire-citation-author" data-delta="1">Mark P. Keller</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0003-3783-2807" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Aimee Teo Broman</span>, <span class="highwire-citation-author" data-delta="3">Christina Kendziorski</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-8774-9377" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Brian S. Yandell</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0003-4519-6361" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Śaunak Sen</span> and <span class="highwire-citation-author" data-delta="6">Alan D. Attie</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2177-2186;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.019778 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460044--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Karl W. Broman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biostatistics and Medical Informatics, University of Wisconsin, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Karl%2BW.%2BBroman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Broman%20KW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKarl%2BW.%2BBroman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-4914-6671" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Karl W. Broman</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:kbroman@biostat.wisc.edu" class="" data-icon-position="" data-hide-link-title="0">kbroman@biostat.wisc.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Mark P. Keller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biochemistry, University of Wisconsin, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mark%2BP.%2BKeller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Keller%20MP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMark%2BP.%2BKeller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Aimee Teo Broman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biostatistics and Medical Informatics, University of Wisconsin, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aimee%2BTeo%2BBroman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Broman%20AT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAimee%2BTeo%2BBroman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-3783-2807" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Aimee Teo Broman</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Christina Kendziorski </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biostatistics and Medical Informatics, University of Wisconsin, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christina%2BKendziorski%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kendziorski%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChristina%2BKendziorski%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Brian S. Yandell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Statistics, University of Wisconsin, Madison, Wisconsin 53706</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Horticulture, University of Wisconsin, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Brian%2BS.%2BYandell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yandell%20BS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABrian%2BS.%2BYandell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8774-9377" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Brian S. Yandell</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Śaunak Sen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Epidemiology and Biostatistics, University of California, San Francisco, California 94107</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=%C5%9Aaunak%2BSen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sen%20%C5%9A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3A%25C5%259Aaunak%2BSen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4519-6361" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Śaunak Sen</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Alan D. Attie </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biochemistry, University of Wisconsin, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alan%2BD.%2BAttie%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Attie%20AD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlan%2BD.%2BAttie%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/5/10/2177" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/5/10/2177&amp;title=Identification%20and%20Correction%20of%20Sample%20Mix-Ups%20in%20Expression%20Genetic%20Data%3A%20A%20Case%20Study" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/5/10/2177&amp;title=Identification%20and%20Correction%20of%20Sample%20Mix-Ups%20in%20Expression%20Genetic%20Data%3A%20A%20Case%20Study" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/5/10/2177&amp;title=Identification%20and%20Correction%20of%20Sample%20Mix-Ups%20in%20Expression%20Genetic%20Data%3A%20A%20Case%20Study" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/5/10/2177&amp;text=Identification%20and%20Correction%20of%20Sample%20Mix-Ups%20in%20Expression%20Genetic%20Data%3A%20A%20Case%20Study" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/5/10/2177&amp;title=Identification%20and%20Correction%20of%20Sample%20Mix-Ups%20in%20Expression%20Genetic%20Data%3A%20A%20Case%20Study" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/5/10/2177&amp;t=Identification%20and%20Correction%20of%20Sample%20Mix-Ups%20in%20Expression%20Genetic%20Data%3A%20A%20Case%20Study" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/5/10/2177&amp;title=Identification%20and%20Correction%20of%20Sample%20Mix-Ups%20in%20Expression%20Genetic%20Data%3A%20A%20Case%20Study" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/5/10/2177&amp;title=Identification%20and%20Correction%20of%20Sample%20Mix-Ups%20in%20Expression%20Genetic%20Data%3A%20A%20Case%20Study" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460044/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460044" id="node460044--3" data-pisa="ggg;5/10/2177" data-pisa-master="ggg;g3.115.019778" data-apath="/ggg/5/10/2177.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Identification and Correction of Sample Mix-Ups in Expression Genetic Data: A Case Study</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-4914-6671" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Karl W. Broman</span>, <span class="highwire-citation-author" data-delta="1">Mark P. Keller</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0003-3783-2807" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Aimee Teo Broman</span>, <span class="highwire-citation-author" data-delta="3">Christina Kendziorski</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-8774-9377" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Brian S. Yandell</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0003-4519-6361" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Śaunak Sen</span> and <span class="highwire-citation-author" data-delta="6">Alan D. Attie</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2177-2186;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.019778 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460044--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Karl W. Broman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biostatistics and Medical Informatics, University of Wisconsin, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Karl%2BW.%2BBroman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Broman%20KW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKarl%2BW.%2BBroman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-4914-6671" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Karl W. Broman</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:kbroman@biostat.wisc.edu" class="" data-icon-position="" data-hide-link-title="0">kbroman@biostat.wisc.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Mark P. Keller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biochemistry, University of Wisconsin, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mark%2BP.%2BKeller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Keller%20MP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMark%2BP.%2BKeller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Aimee Teo Broman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biostatistics and Medical Informatics, University of Wisconsin, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aimee%2BTeo%2BBroman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Broman%20AT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAimee%2BTeo%2BBroman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-3783-2807" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Aimee Teo Broman</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Christina Kendziorski </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biostatistics and Medical Informatics, University of Wisconsin, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christina%2BKendziorski%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kendziorski%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChristina%2BKendziorski%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Brian S. Yandell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Statistics, University of Wisconsin, Madison, Wisconsin 53706</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Horticulture, University of Wisconsin, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Brian%2BS.%2BYandell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yandell%20BS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABrian%2BS.%2BYandell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8774-9377" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Brian S. Yandell</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Śaunak Sen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Epidemiology and Biostatistics, University of California, San Francisco, California 94107</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=%C5%9Aaunak%2BSen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sen%20%C5%9A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3A%25C5%259Aaunak%2BSen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4519-6361" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Śaunak Sen</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Alan D. Attie </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biochemistry, University of Wisconsin, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alan%2BD.%2BAttie%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Attie%20AD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlan%2BD.%2BAttie%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2177&amp;rft.epage=2186&amp;rft.atitle=Identification%20and%20Correction%20of%20Sample%20Mix-Ups%20in%20Expression%20Genetic%20Data%3A%20A%20Case%20Study&amp;rft.volume=5&amp;rft.issue=10&amp;rft.date=2015-10-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Broman%2C+Karl+W.&amp;rft.au=Keller%2C+Mark+P.&amp;rft.au=Broman%2C+Aimee+Teo&amp;rft.au=Kendziorski%2C+Christina&amp;rft.au=Yandell%2C+Brian+S.&amp;rft.au=Sen%2C+%C5%9Aaunak&amp;rft.au=Attie%2C+Alan+D."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460044/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460044/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460044/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460044/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460044/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460044/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460044/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460044/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460044/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460044/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460044/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460044/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/5/10/2177" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-12" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-16" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/5/10/2177.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/5/10/2177.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/5/10/2177.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/5/10/2177.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
