<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/5/10/2155" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/5/10/2155.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/5/10/2155.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/5/10/2155.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/5/10/2155/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/5/10/2155.atom" />
<meta name="HW.pisa" content="ggg;5/10/2155" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="A Genomic Selection Index Applied to Simulated and Real Data" />
<meta name="DC.Identifier" content="10.1534/g3.115.019869" />
<meta name="DC.Date" content="2015-10-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2015 Ceron-Rojas et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="A genomic selection index (GSI) is a linear combination of genomic estimated breeding values that uses genomic markers to predict the net genetic merit and select parents from a nonphenotyped testing population. Some authors have proposed a GSI; however, they have not used simulated or real data to validate the GSI theory and have not explained how to estimate the GSI selection response and the GSI expected genetic gain per selection cycle for the unobserved traits after the first selection cycle to obtain information about the genetic gains in each subsequent selection cycle. In this paper, we develop the theory of a GSI and apply it to two simulated and four real data sets with four traits. Also, we numerically compare its efficiency with that of the phenotypic selection index (PSI) by using the ratio of the GSI response over the PSI response, and the PSI and GSI expected genetic gain per selection cycle for observed and unobserved traits, respectively. In addition, we used the Technow inequality to compare GSI vs. PSI efficiency. Results from the simulated data were confirmed by the real data, indicating that GSI was more efficient than PSI per unit of time." />
<meta name="DC.Contributor" content="J. Jesus Ceron-Rojas" />
<meta name="DC.Contributor" content="José Crossa" />
<meta name="DC.Contributor" content="Vivi N. Arief" />
<meta name="DC.Contributor" content="Kaye Basford" />
<meta name="DC.Contributor" content="Jessica Rutkoski" />
<meta name="DC.Contributor" content="Diego Jarquín" />
<meta name="DC.Contributor" content="Gregorio Alvarado" />
<meta name="DC.Contributor" content="Yoseph Beyene" />
<meta name="DC.Contributor" content="Kassa Semagn" />
<meta name="DC.Contributor" content="Ian DeLacy" />
<meta name="article:published_time" content="2015-10-01" />
<meta name="article:section" content="Genomic Selection" />
<meta name="citation_title" content="A Genomic Selection Index Applied to Simulated and Real Data" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;A genomic selection index (GSI) is a linear combination of genomic estimated breeding values that uses genomic markers to predict the net genetic merit and select parents from a nonphenotyped testing population. Some authors have proposed a GSI; however, they have not used simulated or real data to validate the GSI theory and have not explained how to estimate the GSI selection response and the GSI expected genetic gain per selection cycle for the unobserved traits after the first selection cycle to obtain information about the genetic gains in each subsequent selection cycle. In this paper, we develop the theory of a GSI and apply it to two simulated and four real data sets with four traits. Also, we numerically compare its efficiency with that of the phenotypic selection index (PSI) by using the ratio of the GSI response over the PSI response, and the PSI and GSI expected genetic gain per selection cycle for observed and unobserved traits, respectively. In addition, we used the Technow inequality to compare GSI &lt;i&gt;vs.&lt;/i&gt; PSI efficiency. Results from the simulated data were confirmed by the real data, indicating that GSI was more efficient than PSI per unit of time.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2015/10/01" />
<meta name="citation_mjid" content="ggg;5/10/2155" />
<meta name="citation_id" content="5/10/2155" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/5/10/2155" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/5/10/2155.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/5/10/2155.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/5/10/2155.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.115.019869" />
<meta name="citation_pmid" content="26290571" />
<meta name="citation_volume" content="5" />
<meta name="citation_issue" content="10" />
<meta name="citation_num_pages" content="10" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genomic Selection" />
<meta name="citation_firstpage" content="2155" />
<meta name="citation_lastpage" content="2164" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="J. Jesus Ceron-Rojas" />
<meta name="citation_author_institution" content="Biometrics and Statistics Unit, International Maize and Wheat Improvement Center (CIMMYT), 06600, México Distrito Federal, México" />
<meta name="citation_author" content="José Crossa" />
<meta name="citation_author_institution" content="Biometrics and Statistics Unit, International Maize and Wheat Improvement Center (CIMMYT), 06600, México Distrito Federal, México" />
<meta name="citation_author_email" content="j.crossa@cgiar.org" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-9429-5855" />
<meta name="citation_author" content="Vivi N. Arief" />
<meta name="citation_author_institution" content="The University of Queensland, School of Agriculture and Food Sciences, St Lucia, QLD 4072, Brisbane, Australia" />
<meta name="citation_author" content="Kaye Basford" />
<meta name="citation_author_institution" content="The University of Queensland, School of Agriculture and Food Sciences, St Lucia, QLD 4072, Brisbane, Australia" />
<meta name="citation_author" content="Jessica Rutkoski" />
<meta name="citation_author_institution" content="Biometrics and Statistics Unit, International Maize and Wheat Improvement Center (CIMMYT), 06600, México Distrito Federal, México" />
<meta name="citation_author_institution" content="International Programs of the College of Agriculture and Life Sciences, Cornell University, Ithaca, New York 14853" />
<meta name="citation_author" content="Diego Jarquín" />
<meta name="citation_author_institution" content="Department of Agronomy and Horticulture, University of Nebraska, Lincoln, Nebraska 68583" />
<meta name="citation_author" content="Gregorio Alvarado" />
<meta name="citation_author_institution" content="Biometrics and Statistics Unit, International Maize and Wheat Improvement Center (CIMMYT), 06600, México Distrito Federal, México" />
<meta name="citation_author" content="Yoseph Beyene" />
<meta name="citation_author_institution" content="Global Maize Program, CIMMYT, Village Market 00621, Nairobi, Kenya" />
<meta name="citation_author" content="Kassa Semagn" />
<meta name="citation_author_institution" content="Global Maize Program, CIMMYT, Village Market 00621, Nairobi, Kenya" />
<meta name="citation_author" content="Ian DeLacy" />
<meta name="citation_author_institution" content="The University of Queensland, School of Agriculture and Food Sciences, St Lucia, QLD 4072, Brisbane, Australia" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=R. Bernardo;citation_author=J. Yu;citation_title=Prospects for Genomewide Selection for Quantitative Traits in Maize;citation_pages=1082-1090;citation_volume=47;citation_year=2007;citation_issue=3;citation_doi=10.2135/cropsci2006.11.0690" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=Y. Beyene;citation_author=K. Semagn;citation_author=S. Mugo;citation_author=A. Tarekegne;citation_author=R. Babu;citation_title=Genetic gains in grain yield through genomic selection in eight bi-parental maize populations under drought stress.;citation_pages=154-163;citation_volume=55;citation_year=2015;citation_doi=10.2135/cropsci2014.07.0460" />
<meta name="citation_reference" content="citation_title=The Mathematical Theory of Quantitative Genetics. Lectures in Biomathematics;citation_author=M. G. Bulmer;citation_year=1980" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=G. de los Campos;citation_author=J. M. Hickey;citation_author=R. Pong-Wong;citation_author=H. D. Daetwyler;citation_author=P. L. Calus;citation_title=Whole-genome regression and prediction methods applied to plant and animal breeding.;citation_volume=193;citation_year=2013;citation_pmid=22745228;citation_doi=10.1534/genetics.112.143313" />
<meta name="citation_reference" content="citation_journal_abbrev=J Anim Breed Genet;citation_author=J. C. M. Dekkers;citation_title=Prediction of response to marker-assisted and genomic selection using selection index theory.;citation_pages=331-341;citation_volume=124;citation_year=2007;citation_issue=6;citation_pmid=18076470;citation_doi=10.1111/j.1439-0388.2007.00701.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. Gianola;citation_title=Priors in whole-genome regression: the Bayesian alphabet returns.;citation_volume=194;citation_year=2013;citation_pmid=23636739;citation_doi=10.1534/genetics.113.151753" />
<meta name="citation_reference" content="citation_journal_title=Genetica;citation_author=M. E. Goddard;citation_title=Genomic selection: prediction of accuracy and maximization of long-term response.;citation_pages=245-257;citation_volume=136;citation_year=2009;citation_pmid=18704696;citation_doi=10.1007/s10709-008-9308-0" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. Habier;citation_author=R. L. Fernando;citation_author=J. C. M. Dekkers;citation_title=The Impact of Genetic Relationship Information on Genome-Assisted Breeding Values;citation_pages=2389-2397;citation_volume=177;citation_year=2007;citation_issue=4;citation_pmid=18073436;citation_doi=10.1534/genetics.107.081190" />
<meta name="citation_reference" content="citation_journal_title=J. Genet.;citation_author=J. B. S. Haldane;citation_title=The combination of linkage values and the calculation of distance between the loci of linked factors.;citation_pages=299-309;citation_volume=8;citation_year=1919" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=B. Hayes;citation_author=P. M. Visscher;citation_author=M. E. Goddard;citation_title=Increased accuracy of artificial selection by using the realized relationship matrix.;citation_pages=47-60;citation_volume=91;citation_year=2009;citation_issue=1;citation_pmid=19220931;citation_doi=10.1017/S0016672308009981" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=E. L. Heffner;citation_author=M. E. Sorrells;citation_author=J. L. Jannink;citation_title=Genomic Selection for Crop Improvement;citation_pages=1-12;citation_volume=49;citation_year=2009;citation_issue=1;citation_doi=10.2135/cropsci2008.08.0512" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=E. L. Heffner;citation_author=J. L. Jannink;citation_author=M. E. Sorrells;citation_title=Genomic selection accuracy using multifamily prediction models in a wheat breeding program.;citation_pages=65-75;citation_volume=4;citation_year=2011;citation_doi=10.3835/plantgenome.2010.12.0029" />
<meta name="citation_reference" content="citation_title=Statistical Genetics and Plant Breeding;citation_title=Selection index and expected genetic advance;citation_author=C. R. Henderson;citation_pages=141-163;citation_year=1963" />
<meta name="citation_reference" content="citation_journal_title=Biometrics;citation_author=O. Kempthorne;citation_author=A. W. Nordskog;citation_title=Restricted selection indices.;citation_pages=10-19;citation_volume=15;citation_year=1959;citation_doi=10.2307/2527598" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=R. Lande;citation_author=R. Thompson;citation_title=Efficiency of marker-assisted selection in the improvement of quantitative traits.;citation_pages=743-756;citation_volume=124;citation_year=1990;citation_issue=3;citation_pmid=1968875" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=C. Y. Lin;citation_title=Index selection for genetic improvement of quantitative characters.;citation_pages=49-56;citation_volume=52;citation_year=1978;citation_pmid=24317409" />
<meta name="citation_reference" content="citation_journal_title=Adv. Agron.;citation_author=A. J. Lorenz;citation_author=S. Chao;citation_author=F. G. Asoro;citation_author=E. L. Heffner;citation_author=T. Hayashi;citation_title=Genomic selection in plant breeding: knowledge and prospects.;citation_pages=77-123;citation_volume=110;citation_year=2011;citation_doi=10.1016/B978-0-12-385531-2.00002-5" />
<meta name="citation_reference" content="citation_journal_title=Am. Stat.;citation_author=R. A. McLean;citation_author=W. L. Sander;citation_author=W. W. Stroup;citation_title=A unified approach to mixed linear models.;citation_pages=54-64;citation_volume=45;citation_year=1991;citation_doi=10.2307/2685241" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. H. E. Meuwissen;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_title=Prediction of Total Genetic Value Using Genome-Wide Dense Marker Maps;citation_pages=1819-1829;citation_volume=157;citation_year=2001;citation_issue=4;citation_pmid=11290733" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=L. Moreau;citation_author=A. Charcosset;citation_author=F. Hospital;citation_author=A. Gallais;citation_title=Marker-Assisted Selection Efficiency in Populations of Finite Size;citation_pages=1353-1365;citation_volume=148;citation_year=1998;citation_issue=3;citation_pmid=9539448" />
<meta name="citation_reference" content="citation_journal_title=Ann. Bot. (Lond.);citation_author=A. Nakaya;citation_author=S. N. Isobe;citation_title=Will genomic selection be a practical method for plant breeding?;citation_volume=110;citation_year=2012;citation_pmid=22645117;citation_doi=10.1093/aob/mcs109" />
<meta name="citation_reference" content="citation_journal_title=Biometrika;citation_journal_abbrev=Biometrika;citation_author=H. D. Patterson;citation_author=R. Thompson;citation_title=Recovery of inter-block information when block sizes are unequal;citation_pages=545-554;citation_volume=58;citation_year=1971;citation_issue=3;citation_doi=10.1093/biomet/58.3.545" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=D. W. Podlich;citation_author=M. Cooper;citation_title=QU-GENE: a simulation platform for quantitative analysis of genetic models;citation_pages=632-653;citation_volume=14;citation_year=1998;citation_issue=7;citation_pmid=9730929;citation_doi=10.1093/bioinformatics/14.7.632" />
<meta name="citation_reference" content="citation_journal_title=Stat. Sci.;citation_author=G. K. Robinson;citation_title=That BLUP is a good thing: the estimation of random effects.;citation_pages=15-51;citation_volume=6;citation_year=1991;citation_doi=10.1214/ss/1177011926" />
<meta name="citation_reference" content="citation_title=Papers on Quantitative Genetics and Related Topics;citation_title=A discriminant function for plant selection;citation_author=H. F. Smith;citation_pages=466-476;citation_year=1936" />
<meta name="citation_reference" content="citation_title=G3&amp;#58; Genes|Genomes|Genetics;citation_journal_abbrev=G3;citation_author=F. Technow;citation_author=A. Bürger;citation_author=A.E. Melchinger;citation_title=Genomic Prediction of Northern Corn Leaf Blight Resistance in Maize with Combined or Separated Training Sets for Heterotic Groups;citation_pages=197-203;citation_volume=3;citation_year=2013;citation_issue=2;citation_pmid=23390596;citation_doi=10.1534/g3.112.004630" />
<meta name="citation_reference" content="citation_journal_title=J. Anim. Sci.;citation_author=K. Togashi;citation_author=C. Y. Lin;citation_author=T. Yamazaki;citation_title=The efficiency of genome-wide selection for genetic improvement of net merit.;citation_volume=89;citation_year=2011;citation_pmid=21512116;citation_doi=10.2527/jas.2009-2606" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=P. M. VanRaden;citation_title=Efficient Methods to Compute Genomic Predictions;citation_pages=4414-4423;citation_volume=91;citation_year=2008;citation_issue=11;citation_pmid=18946147;citation_doi=10.3168/jds.2007-0980" />
<meta name="citation_reference" content="citation_journal_title=BMC Proc;citation_author=K. L. Verbyla;citation_author=P. J. Bowman;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_title=Sensitivity of genomic selection to using different prior distributions.;citation_pages=S5;citation_volume=4;citation_year=2010;citation_pmid=20380759;citation_doi=10.1186/1753-6561-4-S1-S5" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=J. Wang;citation_author=M. van Ginkel;citation_author=D. W. Podlich;citation_author=G. Ye;citation_author=R. Trethowan;citation_title=Comparison of Two Breeding Strategies by Computer Simulation;citation_pages=1764-1773;citation_volume=43;citation_year=2003;citation_issue=5;citation_doi=10.2135/cropsci2003.1764" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="A Genomic Selection Index Applied to Simulated and Real Data" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10.cover-source.jpg" />
<meta name="twitter:description" content="A genomic selection index (GSI) is a linear combination of genomic estimated breeding values that uses genomic markers to predict the net genetic merit and select parents from a nonphenotyped testing population. Some authors have proposed a GSI; however, they have not used simulated or real data to validate the GSI theory and have not explained how to estimate the GSI selection response and the GSI expected genetic gain per selection cycle for the unobserved traits after the first selection cycle to obtain information about the genetic gains in each subsequent selection cycle. In this paper, we develop the theory of a GSI and apply it to two simulated and four real data sets with four traits. Also, we numerically compare its efficiency with that of the phenotypic selection index (PSI) by using the ratio of the GSI response over the PSI response, and the PSI and GSI expected genetic gain per selection cycle for observed and unobserved traits, respectively. In addition, we used the Technow inequality to compare GSI vs. PSI efficiency. Results from the simulated data were confirmed by the real data, indicating that GSI was more efficient than PSI per unit of time." />
<meta name="og-title" property="og:title" content="A Genomic Selection Index Applied to Simulated and Real Data" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/5/10/2155" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="A genomic selection index (GSI) is a linear combination of genomic estimated breeding values that uses genomic markers to predict the net genetic merit and select parents from a nonphenotyped testing population. Some authors have proposed a GSI; however, they have not used simulated or real data to validate the GSI theory and have not explained how to estimate the GSI selection response and the GSI expected genetic gain per selection cycle for the unobserved traits after the first selection cycle to obtain information about the genetic gains in each subsequent selection cycle. In this paper, we develop the theory of a GSI and apply it to two simulated and four real data sets with four traits. Also, we numerically compare its efficiency with that of the phenotypic selection index (PSI) by using the ratio of the GSI response over the PSI response, and the PSI and GSI expected genetic gain per selection cycle for observed and unobserved traits, respectively. In addition, we used the Technow inequality to compare GSI vs. PSI efficiency. Results from the simulated data were confirmed by the real data, indicating that GSI was more efficient than PSI per unit of time." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10.cover-source.jpg" />
    <title>A Genomic Selection Index Applied to Simulated and Real Data | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"udWf3EFoPQbrmL5r1Lr057IE8rtLVh142gRy7sLTJl4","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460152","apath":"\/ggg\/5\/10\/2155.atom","pisa":"ggg;5\/10\/2155","ac":{"\/ggg\/5\/10\/2155.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/5\/10\/2155.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;5\/10\/2155"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;5\/10\/2155"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/5\/10\/2155.short":true,"\/content\/5\/10\/2155":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00220PS1xj5cqtVmSMQargo1X13uChoA+6qhKNjF14CE0V9lXEm+MJEG\\\\\\\/i0+TeQI9lxivzee5FPLh07HTGF5Vz5p34zpzS9Ti1gDrLnCJlap3qmFT0Ry63UtVwKx+GeDuVY5+mPmaQYSmpKIDq6nR\\\\\\\/X6IXQKYvZQidcC\\\\\\\/7GqEpOqwyQwsql8L6fH0AnhtXAjJ7GrQgrKGgjbhv5OLQgl5\\\\\\\/CQpVMG8X0OgTOPJfsVZ5MlxoT0f15+ANl13E3pONfJThUmNXAaTCRg\\\\\\\/zyJrLRMFQwYmevNsyhpL+fhEh1xwITKfVM4jy8nzPvTHPNpxdXwSFu2qEU0LAZP95LXmMhTrD\\\\\\\/sqf5wuM4exUNrA3joNjH\\\\\\\/1JPYoNFbYoMAOlVpLVP7JBcKQITj1kdEVcJZ+1MqKavg0VxQT3zrd0urSWVNwbdryOu5q+d2qDUd2KlND0g\\\\\\\/LC7tAMwKcafQlshafxukpUl8jql+hCIPEgRht5cR4JnRIwMyaBHJisRbCnU5erHenF1CHGyX4PtPrWMGo1lWiAnTfv2NxapEgSZR\\\\\\\/O5xOcjJXvCyrW2gte5GrCZvovlcqGbjeJImmZBvXQ2wQD27xfiNkeE9KlMN599EWJ8jkaamqxGTf5mQAlqmAblG0pR2sFevwt\\\\\\\/b4VPk9\\\\\\\/uRVFTWLG8MBpqtFuCYM+idovPr2oE0XMeBa9w9QhuwPMTwbyQMF4pHyAmzoe8IWCGPQnQyJ5KqL4dd6W476XkVLliSA3fZssdIQWIv6BJUeXiZm6TwYnxHvU5b3M45On98bAqBQUINOqDfTrOQIbdgyjWF4fV77NYbkdp5iEd+nvdpngi+85wfRxOX366SR4VxL4xNJg5Ok6+As6g7EdKxFkIcKxwAd0XV6EwjhvPkbAHEaey0hRgPRJNMFIMyfbuFTefXnJ48sfDtgZX5HNgHJBphFaclTn038HfuteqqfxF\\\\\\\/HOxNdKndXEExI0ns4CDhdjQVGxAZquH+g6qp5OoyiOLR9\\\\\\\/9viDFAGHnrcEpJUyw1F2eQrK0NVKNsww1wijYUQQ6tYs5ykgQJSGxJc30P1rBuZ6gS4yrz27sicIC1u31Sgmpt9pE87qY5NZXQRznubPfUhifJK9yfCGjqa9RLiff0EMK3+aKAadvFU5mp0DGocYsqMWtqs5wJJljB4zTk8LIpJ9yRL5YvLIYofcSquXRMV6QWjBStMzrR6YPQFQZMnRYhXWz9fs7VSyu1oaiEM7puba9YJmsCj0dp6pL75JEcty0BtQL47kTj8g7A2J+8eyFQvv1D\\\u0022,\\\u0022iv\\\u0022:\\\u0022sHk7BTX7V9MfqINOBLjZ4A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00221a9ec5722fdc9d730522a158f985afe886856d342398086bb81fb17b446da69a\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022\\\\\\\/6kAQgulDvn0HPoazk\\\\\\\/IgC6ajkBdbfE0K81HjamP0kl9rw1E2cH2dHHeWwpcg4ZAeQ5qSnEfFOaWRSkdveu8vpju2+C33yGnKfBk\\\\\\\/F8dLH9vb3Rg\\\\\\\/5j1PdlX+GU50uAKb9dV2pw95bgC2C2H7cYtWn8VlJzjuzj4EgxAxHEiBAjtkORaAuMQDaTvkZrVNpEZuWE3u3DXJ3UrPgmmYDqsfLPMFutgEK5qDQx4ZN95slJuboNus0V5+HH5idSfTribLWBd1oiN8EbR\\\\\\\/pDsEjpvCFlzV6pJmRIPRa3J\\\\\\\/4j9EEOSLnOikdwR0Ba203CDEsIaJrNprnQ0QL78uguYHxeeDvrVMr8ZNKcqV9vOih\\\\\\\/B37HUoQWd5i3cV21q+ruiYpxP3qotpNOvS1iVkCOl1ZGISDgaBJyZRDbAd9xmauyKuaeg1Sk0rRwyD69HeSO\\\\\\\/GHtII3ENGz8s5ksVcHfGc1XuRTTHzccRpZBL4Hvl3ZQkMchlKNT3l2KJTlLzUx9fUYgUmCRXlMZoa8j9ndAOJsVTzL81qiB2ip+gS8pGwOE0tV2a52IFfdv3\\\\\\\/SDQs0WiAfk0ZMwwDueHAGNxfEQ6KJt5xXzpVC+M5H8tizp5k62peQNHiXdPUZaa1aHLUFlEHUo+1LaZp69rj3sFXp5DynLdpluC8IVZZ1CGwBHPUemJfiOop\\\\\\\/1wxuFGpY2O3hMdgdVbU4x+ldtX3wx15l87EDZADVWurxaYDNHZFuDYPEdRi2EjJiYUHeszBicvBMu1T6345XC+Ky9icjqsdnkyuekEbigwsch2zOwZGvgqKNihm2KOKcYeIFx\\\\\\\/RQ6nuKTF8MZ66uPVBhtFZ5yblmCSAwg2XP+eXRkEaZ08L5diTNv079jVdSQXJcScL9LnLXyzTas1FQOAwbPb3TCZLp5Ervo6D2G0WUwkDS53CGM\\\\\\\/BhruxNJ9l47aMe1ZErrs\\\\\\\/0CZQL0u1Xsk7rsxPRYqps3ftqqgkYkX0802z1q2jdoaVI2JSLjadVlD06wchUdo\\\\\\\/bmQCRsSaBsZD3R0b\\\\\\\/qxdMBqtu9pAuppiJz6wOI5T5X7qTvKXQsdWM8rrh30nMMNxXXtBF8xopi2UIGGK6pKjjX2L5SSfC3yfOfzsZQ5Bchszmnb+hdU60D3QqcmMDHn10N350uWVvdPbPuGEaD380DC0nMEr3zSGXuxSDF+m\\\\\\\/6kSmvEjIpjWx57G9GGu86QbRnS\\\\\\\/R3rJ0kwWPCWqYvMrslYl7FBzIIE7pQXTSfQp1YWvb79l1iBv\\\\\\\/BtG8s4Ez9RpRgSQpARrhHlOY7LQuG8z1ZOlzNEm2J7v9gH7+51TzybFtiq1XJ+OyydbTn5my5pfmACQRJwqkeeFzabTQ4I1u2QC7iJO2bCXBn8mo7EWwhU4UukeLL613bnwSVUMMGg\\\\\\\/3cgVUsVxyhY28+O\\\\\\\/\\\\\\\/A3PtPIoR8iOByVlLPFSHL6OM8o5JGDneZuSz6k2VpmSVMOa00GO2mhAdnosnzHiyDy3cRKteEfdxcLP10LpMoZ0tFxw5Yy5rSkz3Au2WjPGNbpSNBVTMOjxhSe+pEiMeat9USGRESynfgtKlEZwuBny3L+bPVqjUF\\\\\\\/HSaJxqWovyqmVE5DyGHgYVJksSK6woCj0n\\\\\\\/kZNnBzPCDQWd13LniSrVkpwj8b37WTn6fJmwHPr6hPbThqV54Wy4bPzTM1KiiRPNmbDp\\\\\\\/tX9EejjT7NCYzAJzkvLtsTaZ697o+VtNnk7w+QPyTU9tT0HFycxxpoVXBdXcP+J91pwkzPV1MgNu18CHNgInooRGl8rU+FeybuickpqODVxXjM81u8XIa7VuyITS+QlOTKPOKboGlvojaFdMoGwl45g1vK9ULNf3wNG6RYvp3h2z+hoSwH9Bdnehq8BKUEx9vzt5fwIiFSoMDM4HBkisz3dBncDyICB3IDqe0BQZUKfHx6Ns7rw0qcLyR50mgHfqL3ZmVReh0DtNUU0LVRMqDJwkYxFRMP9y8IjmNVGieB5Vb0XV21L\\\\\\\/jQle1B0fgXHj30PXKumydAfj3xaHF70ZaXZdAccL32f+fazeHxMgZZlRmRisJt4tH+6QlfXZN+6UPrKXAi6I\\\\\\\/H+ylpiD5mWY+uYX+TfvwO4\\\\\\\/QF3rFYZyJb+6u8P39l0bXieFqZOKV8K4YgFpCn6C0f6bLEvdws478vC9pNitxzE0mfGRr1lwBOc6ZlISfYu\\\\\\\/GCqEnWvuLNO5nhjAQgkIVq04M8COxpmTSZCgq6q\\\\\\\/XNFz2fqqXaygmmZct7c2OhjsQVxl\\\\\\\/JsHhaym0Bb11b72VuiI2h0Hb+0Kdf0xPfBpVxapIjbzqaRrTjTpdUfYomd6kbbWesehSM3KI6ps99Lb\\\\\\\/giTdLiumBEmTRqvCg7HSZhtNzV\\\\\\\/T\\\\\\\/Frjd2DCQh+SlAu52yxxtBu7lp5DNMueSjcmMuoXtGKmtBkv\\\\\\\/HD68Esem\\\\\\\/pTv+jtG3yT69BpalN3jiWqIUQ8cE\\\\\\\/JATGycaoQc\\\\\\\/uZVoUee3S6Q02zF\\\\\\\/55Cnuqs0OSRXzJ6TWhmynV2wqD8eHLipNndj0xBEydlDpB2pK46lryooZEK81TRT8eKbLYg9yiX5cvGPve3v2Zt1uA9JEug6ocI8dd4CCkLSitJBVaFUjZm0lE3oeJN3Mh1KDqYo+WeSxuYun9Ku937N4XQ6Hu1yKzb5f0tXqmb25xq45hfNLpCcTjz9fUDlhMTyPWQ8ZSvcm7NEMLnpR4y6YcQ65kmznw5hfUsTRfdEU2SHc+PrwrnJDqvAkFn4XPCj7\\\\\\\/CR9r4SzssrY9uxf1qXVJkk9FaiCsGf\\\\\\\/UkaH3pgPauxvpijHYJMUlL79ZOULyAJrLyq625K\\\\\\\/0Arzjkf59gVA\\\\\\\/2R2wkwgIF7KASgzBk5AvLNKXT\\\\\\\/JkQIccpb7ZhGbFeMwR8PeO899vKe8AYzDywurD1Habd1dhVr1qPXhMTOovY9dQMmUx+vYyMxqEOtXgO5LmlXy9pVawgG5DQC10tLDwGNGPi2\\\\\\\/CBVkuAYy5YhZpkc7JtWeNJ4Nw0twUeMSd8+3XI3m9pQn4lZZFCOaC8RI46MYz4RaUSw8pe0JQBg1zmpmvXxPHW4b50ePZfknZBhHnzO9Ig1Bmp2kV1Cde+7GCxSMfVfVv7gCS2AvRc4cNbXepgaQ72eROoXTqR7XnnEY\\\\\\\/7jArUbxqors62cYTRf2BFaGxbTC9jUy6GvACuHehrBnjY0jS+IN2WucD6E8snIR0OtpLoHOHPEQABdwrNUTKJ6yTevrJJfp0KL+ckleKECFJ7X5P9NOjojfLltBa3cwYut4DBPHsgDMNiW1mrezIxeYixdc0MYS1Q1rDwrYRChM7mwKf0sO6JOo+ewitbGocYjrGJ0LD3TwZ89wV6IBC95Q7ciY6OAS5Nb3mN1oJsF82FV\\\\\\\/yXhRXdWYdbRRBvfS3LXeM2b+qZDqjSJiiamvAtkmhhrx1IGmbt2FYRj30OXLwbPMV4RsB3tjuPthBm+bRQ4pCdaSQAlsPfhNrPGSwLvL32IUA2HUAUnlwIHW+pUr9vXqKQKfkmf4IgRbTZYatAtce5d2qJlScXVy9\\\\\\\/lsXK68igncej4IZPufzztm0Hls7g4l1JMXFAG\\\\\\\/almm8NKJx\\\\\\\/sNVtMcy5e+QkCzKj+4KsKTov4\\\\\\\/k7t3WT3WnAb0cEq3bQEFhj2UGJsQmZkx2pJLja1HwrEOjOHZ1tdFmXmTaTGt+zu+8UF6Gn6xAgjPD2ujAS9iA16jvaJLBVQwYm5NPWVmks8vb0+i7JBW9qoU28+C+u5dKPbwpLI3x5VXy8E3nWLDVRYUgxlaZC95g3kRwIeOfogdVFCjPVG\\\\\\\/YG4rzPv9QbTgg7GgLD74O7RODI5\\\\\\\/xjwutqYBtMsV3lz67jl9W+yXyVLXNh8T1bhpE\\\\\\\/tCLpxgznpRHcA7crqfZ+DGT3RY1n\\\\\\\/YCUAkMiSN1olu0eJdPRvCE3++JT6QIWC9tWKTaefDGnOB9l0V7wnsR8fm7edfnAx1doQXCVIReIM6fs7qqcxA62YDHsP2gR5KQuVBJj2zQCMVjTzxiUwA1+t3gzLs2gRYKxF9XWJD9YPQXUYGPndx1fExmboGGIo\\\\\\\/iJ7eKx5ODmZCoQiaCNsJLQH0k\\\\\\\/yxYSKaUN6gmsKXd6FanxkOtM73av3lGxQKoWUbbfp5XjOmS1n7ifm6OyDqlIF5lDPbyID3PxlnjYubzpbmNmqWA6B08qHNEp8jBElPzaXlNZ58SOp7PR8lVpK2X9P5y9yZcop0yPubTPdGFWcJ0i6rt84vupuThfBrMSiMleKxwFy4EmoKdE1RALUhvQ0deWv7L79PF4gdDnhkuRg5zW9\\\\\\\/iAMGsosUeGQlLO6ztq5QMm23KNUI4HtlwnGptPpUY1uEe1189bKih3rNY7CE9\\\\\\\/z0ey4wi8OMIfcxO6EWNIBBOMLrNOxMcAX5QO5f\\\\\\\/9Yuf0EX2aZmbDK5az4hf7FOrPWdbUfpJS6MnbdHqwtfVDmDz7iSofrBdPdPZfj12SDfdx9Cp9kbA\\\\\\\/FipbH7ooyctlbHkqFbighYZu7NCN75JPalKnpricJOQ\\\\\\\/zx5HPUMZTL6ATyv6Y3t9fpKYhClGtRaCi05Rx3Vn3mdPKLz+N9G4TJEWnKoViJDnl1ZmgFMWhtK2L4WMb241N2GiiTXBIWWkDWQ43Mz1zpG5+XrU32bJAzHg1x6xh8wTynwVSeZEgpEWq3nLbD4JywkgUmI+QwIySS6fuA1PhyueckVeZzSMOfT0kOpu1T8xXq6yl7L593YtnWrqMhHsPRcrRC5p1lR6cGyN\\\\\\\/o94EsN78tHBQphBwQRQBUqIRzwql12aJrxINtLMIyJgu96AbKovQm3dRvltFKLsSHvBj3nQA8UqJVVdZeMEzL9a4\\\\\\\/aa1DUDEzLQBpKJS04qkLSpJwToeC3VfBhS29P8L9vtRVFeoMlDrdzERZg5fV++6V23m4I29VSl2jiQZZrbTDXoyryzJmsJs3Rdgwitxio7NVLlIseCHFJ9pI2L\\\\\\\/Mabb5bjLubmNshytFLS+DcVTqj8LegM1eAF6mGX4iORq8hNVI4j8Dorp2rdYL1fT6UIuUz1u6ipMqdFPvH5\\\\\\\/K3XSX+Bf8fCg4WYtFtklaxupZeY2Wags2V\\\\\\\/eLbqsnK2fTuMbVfTmd8L0uneYOPQ2+bDkMXsv4dQEiXeIiStcfO1G2u6wPVKTWWIalhFfEcHiFtm8pR4P4zDPMplKnALkdkgOVrBhgU7TiNw6\\\\\\\/Nx2qdKq3+Zh1DKayYmDYSDoi3LZTiXHJjdKk5ZnR69caWqlGak\\\\\\\/K8izniG9mUpXrtkdsbfdUN37AkNnwVBDf4rT5TlRukcdN\\\\\\\/2o6PIM+b521yILx8m\\\\\\\/QHD0r+CuSK6YHrID+LXCtOc28bzSP292cVKJK5C7GZvJO5ZI\\\\\\\/kGFiiZeXm+jh+60ait+SGMwJqthsKVPSVXOayl0cwZSFb8D4Uxkxh482PAn4LmRCHVhSmEO7qxQ4TMqNXiaczMK\\\\\\\/fYk1LpogjVWfI6oHUldrZMJJbssggsB\\\\\\\/5oWXdKGydjPGhN+SAjIikMnt8JP6IQvC\\\\\\\/Qp6kxYcjfYVpu5M8JW7JrBeRd6PlhxxkMXy\\\\\\\/NRcrADTj9JhWK1SJbP0vfkEz1\\\\\\\/KfzHy\\\\\\\/bq6QJbDZ\\\\\\\/3P66NbURIPK5f9wjhicr4MROzWWVNldfA8BA6y5P20IyplI0sY3i8t+90bJBVvhzw2XOrj1qW64sFztHYPiXnNNPAh0EECF15lHMhw4nnabC9ZK3M1ITVPXoKCQMM8uo6JLyWr3XmZeySSgRmw1k4\\\\\\\/P+Xc+4F3\\\\\\\/R1p6cmPVezgHDP7I+fIHJbWMGxR9XVc2fTj+6M1zeNsDOroDtDGIPXwrQmBrI6z5N1MRI\\\\\\\/rZk+ECzfbJkqOQ7cwgZCBlo3SBzJyVTmfqxkVn42CaAtvxhAPX1D+QdjRLsgMrX2qk6PFCiYYtCQZhAKnmyntvX7bncPTrhfVXJPYYE\\\\\\\/lDDaP1yc7OoLEWWt6q79tln2Edr9eIytSpH5L0m2puij58uzxGF0MHQEZO8sv+INcz80egltn11DT18Kt7WyiolDm0R3gEDXKU5Jf4h3RnfiikHjA3\\\\\\\/+2v8Y3dkWHEyk8os58JhdQ9W+0D\\\\\\\/Rz09s6UFUx6wCWbG0NdhNKW\\\\\\\/Safdm\\\\\\\/1uiw4ecuxWAXDD8uw0ToLvSYZ4jYBXlFheb46IJLirkiXgXs8Ei\\\\\\\/ir+uneQx+bzHV9E05gV1i7v4eTOYjTxztU5sfl6TRxNg0UYOqjzaOFo3tyrcDLRJLPu+DANHfSRmYVNWm\\\\\\\/GPKdKvcumz9ed\\\\\\\/yyu3++9ihsyr1PShheWowhwYP37P8Am27Rds8cSVcq8I5TAKCckOJcKz24qWQBYg5wnLfHl1DR4RaNNKzNlC\\\\\\\/ACDnak6fbvfpSTQv7yf4OU50QsUp35TzUO8rofavYZf7wA36VzCch8ks2mU9IaWIM3tdXrd\\\\\\\/7vbTxmGsEOT0U2jDM1Zvby3T\\\\\\\/xG2looFZgggqMTMP6jr\\\\\\\/WUf7pkU9CW3qavKXX\\\\\\\/CwmhRfBz2Z0l78l17ImQ5wxO7vvB9SM9MKPZRRoBIZ+KHx80g8c0CMfvgy0TkV42VVZ2TPzSfRt8xsRO3m8moqiIL90VOl2zklktIz1jPXOsYkG4bHwo86DzRQb\\\\\\\/QBrRdYfQSDML2h4ChwSqKB\\\\\\\/xcHtafNw1KolfBXhmRgjChiEu5Kd\\\\\\\/zId7cXYnOi6JoNXzWHOWK1t0Ac60jD0vlsOIV3zJYCcZnbqv+wHwo\\\\\\\/cdeVuq\\\\\\\/DEdWOhw79zhnVvwDO0se0YRQ33YGOyTdN2i8yiH0SAfrPNjeA2LwbtaQNUemuSqlcQrFi3hrtHHKA3boOOXoFpacPNgBNcT90OufPwYMsWfrHsQdBhVzbtKckOGsS6JB102Lewa7QTpjDNQod7qLS0z2WLMfZvjjdUlO8fbuvUxyYEpZstmd7LtvR7Er9KeTnOylkXQdu3bqbj3FS1mMSMkvRJk+fj\\\\\\\/EbM4bP3GUS1ZpHjj7u2UEbTQ\\\\\\\/6LnUN9KW21OBWJ+ChQuWlQrr4u5XsYf5ujBYpXd\\\\\\\/NMzwlMBGVb3R5WBwMVSSOv2xxmfRrFeBeyatIaNnaqpKg1gTwTicvLukQmkU8vHJOBQzTs+Gza2P3mZdCttu\\\\\\\/LwCZdKO2tzE22YnXoLcjg+ThwT8oN5+phk0WYGHnjePLTgGmubJaC6bCmMsrBYYspGBBcqKtryzTIKvlruhCy\\\\\\\/SlMmtijSKwNF9wo4xlMZ5KMzemoqY0sUchwQ\\\\\\\/ktTb1lh4oAU+k7tlkPZVryz18JwoQRBnf5Y4EJ1ebUuNLzOZm08Tavx889ndc4UzA0\\\\\\\/FWWm7ilHRoILQEIZEriFlaTaiP6+\\\\\\\/MLnRKzc07+X3lS2NeEUEZgCuHVfuxpsEPyKuDf1q46vMjOiavpXbv7UkCHIIJNJFbd4DUAosLYmGF3Muuprin9jyIv0hlgM00nlAXEJlGsAZSAdCM8J5aoFeyg7J8WIQDM7chMeAZpCJahTVMYiskbLwVc+kUenu0PEMH0v6U\\\\\\\/xkGmZZ2UMc+mBMT5ZO70YZUHxJ2cWPdduZpbZJIQ+ftmfVbpQ1g+xmWUnpgE7owQnUBn4Y9NpKFL\\\\\\\/wUk+\\\\\\\/X\\\\\\\/d1zPtPZ+0YumNvuUVS+7ColyT9fLjbqOaX1rSA9jNmr3i9BUkWKj3EjZ97wof\\\\\\\/TJpZMLTr4KJ1WJGi3CDfsUUkRtL9Z9zahjQeKNzxpcKpdKOvH9\\\\\\\/jVKraPUWtUyCiH4s5Lzf+TVcLuw5h6lTG6o\\\\\\\/DFXmn4fV+ZfActk+9qaz1Bi26wizaWHfrC3a2vpyHwKnKud5pREeQc\\\\\\\/LyRv2RZFnrfWHKDm0E6sSGwMHgz4GL72zzjxOupMLQ+w28uWIIRY7rGB1zFkBVL7d+xfIbAYk\\\\\\\/QShFzJEGJRkqZvs9O\\\\\\\/rqF2KJCfyk+h\\\\\\\/cbQWw00VMxFFdF7f5gTB0QG62MMcqK+AeiiE39d5iPqOqXjMSo+V\\\\\\\/XvMgvdX7Ezet03By2iP6JkDafCVDetPvWwARyVZhn3ibgQwHET\\\\\\\/7lvtCcTGZ+VUPMmN6self6wIrETlJTFspF\\\\\\\/C+Xq5va6Udy\\\\\\\/kMLuwGriT\\\\\\\/p4yogTGHY33DLBUKjYGSByuFOMXFtILn4gZ4bimNK04EmCkeUNbgGWMZU+1njMGj5lVjJyMuUf13Pr+2N7S2dqkdFDhxJZpd21\\\\\\\/2wT2OhbjHZRX3ek31D+aPrsdvw6Vc3RHhmsFXFIv+iZIj37gfHoXXomBm9fc+kQ\\\\\\\/Yr8TvpQz9Jq5EvATPRDVE\\\\\\\/thY\\\\\\\/Yqp8kCaUI16XGog+fqDYzp6J7jMywz+Hux60pcmbqn5URSm4wILt3EtGfssbDrtq1su69b7XAj5iX4B8S4kUMKo9W\\\\\\\/KWAQG7W340\\\\\\\/hur2ILYOPQyYLQPGC1TyovlJcApphLBDpbyeZ70qAXu2hfA\\\\\\\/O2FcPHbN\\\\\\\/Rhky8TbLd7AT+ktxnP2Ztqhd7uDQRqq9z1Lljbs6iidIFUVEoFTlxhQjb0qFV58\\\\\\\/pGjtjhYhwCGbbmKDVdPVG2Rlu2iPYuoBFIORu2bY3obsvak+0CF13sJk1ZfvmCGNmkJq\\\\\\\/KmGvcPQsPQgnNqIGkKzzz6Fgxbv5+NojJQtVOoBfb9WvrvGJZjwrcF\\\\\\\/fx+PhpVRnF6uNhzGg9D\\\\\\\/ODJ5G4z2fY8e0+W0oh7nwhoxZRF1tHyNe79dxiEN\\\\\\\/8w+XfN1Rg6GUSBbLEZQ1pVpS2c42reF11IhJVJk8+KsDESNnKVjThBFXb12ennP2p3M2KZ6nJIffAVDHcnUV\\\\\\\/HYrlyef2L7v18vxFoz8uW2J50oJEE0nD5OYXjHI\\\\\\\/ONFosT1WnkUN10uMNI\\\\\\\/453JM5EqHpeJZurpl90rM6vAi52pfd4VFOuIidKI\\\\\\\/h9nAWOODMxyDSWxpGjDzSkxJ\\\\\\\/cmoSPA+aG4ZfC82xg\\\\\\\/TEmXysUrwXoBRwfQVaIY\\\\\\\/JLu\\\\\\\/UYe1PRAH4Z16nxrKybkEBRUpRGwnwEKyyJUG+KikTIKylLEExH5j2w5\\\\\\\/3Y2SGWMu4qjZYtEM+QpmGGl097CMRoR2P2BdSCR\\\\\\\/a3aX9EWkg0+56o+bVEoryYBMSTb68c1LhYpsRWIiTZJV2G19Kea+FEZSHkbZxQCmUCizWoolv7HtYV0fPirAQlvk5+LS6uWOlOKUUFh8fGkMW\\\\\\\/v3SrrW49uO5tc5hJFFbWF9xpcRu80C8Gq0Wu6IF3TY2CmoGAb\\\\\\\/vBwyY9atmPu8nfmyE94icOzylIr\\\\\\\/mnIETlG38jfCc8XMAXrDqwt8lxvgTgylImf3dIgpdy1AIkANwMPSbFP35o6WRq3Ss09jK2zK2wpfBhps7gOJ3g3JzHUAsm3zD5X4ZKkmMW81a0KU75az43+tXcjV9ghp9SdenI3i\\\\\\\/X0aWtevyu8XJTnljSiSnLo96AEwFrZDfDwclu6S99x5d3y\\\\\\\/g03Jac22NP4cHSzt9XMdLfmdS8UWrJZRQPkK046AulVvJEqk+qUoK17PREkNc65DGkaqSdvILg39VJb11Uy8io83cPvkQuzYtynLuFgSseK\\\\\\\/tDcRIj3FdXtt3nMRXTg\\\\\\\/Df27qXNk5osp+jvHxGODEIwGXkQUHWS2cffvF7+KjbqloVB8DUrznwZjZ2qJtbmi7a1hOVcT+DtSZRUmGrolb5bT\\\\\\\/7BDAmV74l1B4I9NjCwCDXkR+ycmSV\\\\\\\/w28F7iWP78BkFTg6DmdnrVvyo5GUCIvjYwwSLZiUlTzjQLOGShk94nAPc6ksHcD50jaWMMZpgEZO+F1mu4cgSg4d7JIeXDKzGidaKx3nxXyEOIo70TOhb8umvJtYQRzJ1ehAMxTkytb3v24EyDTLb2d2iOAbl0ps513Rkt1ga\\\\\\\/xzSWae\\\\\\\/D86sPldun1iMA+SwdfgLFnpbPcR1+FLVOa19ECQvjZ6owvH78kZdS9C5MmbpF6EQnCGyt7wq8HQEU56SxbL8vv+c8j2Y\\\\\\\/3LNNmxeyn9vJh1LRJEJ2mQlrrmh1wl8vY5xmBJJtHmRxp8FWKNqhUgC\\\\\\\/l\\\\\\\/OKBkepRXIavTaiUR1sSKIw3wcw0JYaD2UvsrKvP7JARToXXHmd75S9DVWTjBR3AoWhlVvL\\\\\\\/eg0GQvIDIuCq8ByjfNHhkHQqbCGrLZVJ61vNhMybshyV4dQEEEopEQbI0rx77RI9CQkVnIIXXofmDxadXRJxaAskTjO01n\\\\\\\/rBfxKNBQmdrH2uEto8KFKYzkSgE3hf713saGwsT0B+H4u4DgYvG6IWuBjHXM0GADXh2NuSE6czJWkMQM4WJDbvNv\\\\\\\/40ylAHNRVzTeowzTsq41prWtTLhkQ4jnq6kDu6x8evialFDfJdGJtTxcXh13k2lX+6FJ0l6oInd4sgI9v3Rf7HGvPQWthsE0zAiNvFfjnXevx3AJXB5tLVyeAAptsF5xyv1tTdeSzamWaMX7Q0Bj6MpTL8KTFxN5rbSkqKElfzqZtVVQbbiFl9jRqaKYi1o2B+5pLlbU75fmTP3HkDDFTHCk6v1CsQA02BL367hwmx0vtNS1MfA7Z8xQLEl5ls3Bxrjhvr5eENlmqcCEcOmmiZdXAxhJHWUzCY8Dqv1w7elVyNNJtSMAFHmKYAMQ63BoakisMRVry\\\\\\\/URrqxgjV8W8dXwnoqqBN\\\\\\\/DiAfY\\\\\\\/LDsPpOe0G8YL\\\\\\\/3slwAUnlCsg\\\\\\\/qFKdJq92OopEF5xKQYm5EUNRmYIV2VEQZchxmMyUSYggBSiN2yKcuLCJUHq3YBPnbBG8gO2xTSxQcd7jeXM9\\\\\\\/JONOhS3GN1fb37B\\\\\\\/N\\\\\\\/1gHxIWcCRJ0a\\\\\\\/8MAXm51\\\\\\\/3oou0y6ipjiqiKPSGkrwLxyU9TdPU3BZnXzPQozESHxjNVx0HRqrMWXu5kcXQHHtVHcF1cTwXRmjS3doPTYiw\\\\\\\/IS7rXgjfdOsFuU1WSY0PUSlrKguug9Fi+PVe7i4t76h90MQOcpvyqePixnSoHQgHf9ZLRCtZKPbS+YOaWG+Umyf\\\\\\\/BI57NHx\\\\\\\/qTz8K\\\\\\\/n4W7ufkHMelfUJZhbejQpgsXeGJ5KPWa7yTC7TMHIq4VKlxA5+TiCkp3hT+XHdGqGK2O2QpXhkdzSrUQriEdFMpepH\\\\\\\/JYkdMxlzyY3mgPUrSIyuTNPK+0ZHso\\\\\\\/szQmufRDXXYbheI+MekfLsBRUW9F+fFcx7va7jpi\\\\\\\/3bVddbgwiMSfLRcIOurJnMoKLAOcXFTbY4YEhSv5beuNdJ4vPS5v0Vn6U2MORa4zVtelraM+3wBeYIUyzpJPeK44KOVeM2vvu5m+Clnx1J1qWp8DKoiLI\\\\\\\/iVxjbfFT43lusmoM85+kFFsu3KHsdqO08mMq1arXrFcibPisHZoym7rXV1OfWrjbZ0nWhthB4xOznkYKD\\\\\\\/3Fkh1zW5XXiu\\\\\\\/XQsGlga5iGYQTYcNQCwUtvZucyxr+LOC5MenspXN86ncYRLFZIwBwuivZLc56MnzBiJ3xWf89qF\\\\\\\/dwa7Dw8og7iZfMGiDPZSnPl9jkhnXhnwPTZcO38yHOdO3uZBtLTk42k08iYrPkWIebudlis+BvwiuQnQ3CR8EDd68htbvbPokE0TrwEBSqqDhWd96K4fk7A+meahAWAfgNLkDhPKA6WfVKVaTxpT3e4JGfXyMRYoOuloV7X3LLfsj4wzwGnVwsCavI8ok3byI\\\\\\\/Vx2CzgXAaiwXpYSZasEdcWtce9zeYcX8evJrWwCdoB52SSsy4sekWpBjn+xfY9l1ZgnOcEL2fBPdy+D2c6qWeqpSea4eGFlptVd4zDNCjs9I93wq26LGWhKDPzlgaRh0kQSquJ8jg5Kt3HJeZJrpwHCvgLsAPgEj3Pbl2ZP3ZaH8U3VbsA6bXjIh\\\\\\\/0UoITig\\\\\\\/0ZxY9o4fQi8cgBIhVmXfbM12yGn0+SfmPgU\\\\\\\/WVBcoNfKTWLDOINrucPG2fg692XwqZyE3DqUpTTCg75peBa6gcDBifmdHd2dQ0zB0N8FZrZ5BQD5ZTfmOu7htPnpGsSDRPNKMgaPheYSCZhJQzTZlU4ea6ZM+oTAS9i8J16syCeZPecPdUK2skPGqp\\\\\\\/aNZaRV\\\\\\\/Ho\\\\\\\/3xdzb4W5e7MxXGqe2bA+M5sMIdWdd1rB28cN1X9KNk\\\\\\\/fLvyN5gDeXzbl7ZS6qF1Yde8rEVsYqGPxX2y4IxG8UeWgxyk3zJzVgHGaEBMK0oU3ict8UYXsszNzjPWW\\\\\\\/9owtxx5jw4\\\\\\\/dIOt2v7txAE7Y8u1XbctumpQCkjfDNcGd9M+4T5RqlWVM4DD6BNLTzxQEFG5akIZi7mggL8kuu3TxlCB\\\\\\\/6DfARdymMOPXY5jcyABwxL15RPnghtjoHicgJ7Frpl33fakRPyaZfKkdZZmycNTKigv7sjqvxyvqyc2vKl\\\\\\\/7xwzeJ32M2ytKyHJuqop5BfBKOpTRp\\\\\\\/aD1oegmZrKGcgp65kZMC6\\\\\\\/\\\\\\\/FUnhASjGeD4RP+Ud\\\\\\\/DQtsQdtAqEGeWoxsM0xLXKwbo87DYj7WpsDZcQPm5291O+Pv2Sf6bJXjICGqnl6qSXjTBo8IIks1BDuVPegdhKBHfd8YBPituPcF8+5UkUMnsAF36RM1nwaLRVr7LkS1p33r\\\\\\\/9QCF5H0\\\\\\\/zKGglgaVNH0ZL2lv2ngU2GEWb2YS43DvN7LAEZUYaLWe1+OE\\\\\\\/wFNYohBO200R53Rax26Wcz2A7vjHeokdkgBbY4DvYuSwAWGYqcv2clmTdfqotn1KxJ7ekO12AfgEb\\\\\\\/NtWkUjAz\\\\\\\/sl1sMUkuW\\\\\\\/qsngclKarUf8NqYhaNZyXN1zfOG5MuJM6rAr6gqE2JuI4f2ULfyJmXPnQiOA2oDTLTQzAaxHCeQgAFBxVOcHbkdX8PSXdRT2uzmC0bmsg0v5\\\\\\\/vuXPSMRPJilUJr6AVZ1SLKN7becQl1iXUnqX42XhNvRNSFeen5MjZgItrk+n8xwR5Jvggg35lVUuQMBcAesOiSjl4KrqpP6hovZvrzcLad7UeXOo7Z2ukM\\\\\\\/T4dZ3bR9sQEkOlmmKVZQ84xDTQ30Nekru6XSpDk20jxTiSs0Hbo45Og8HCWJyn\\\\\\\/YY8\\\\\\\/1ZL+ErGpTQql7\\\\\\\/KlynNDoaleZDpWL+\\\\\\\/jq\\\\\\\/JndqycAprBzIZTrM3DoIoMhNq4\\\\\\\/HNv1tHTBTVebhOAgRdG0xtI+Nk5ExQzNNXqCMmw1+ririMSeBw+goWnm466VVXGF\\\\\\\/G89kOsYiQjAM4zKN350qccEwOpJDNwHSDR0XOmmBEENhcU4meSgiPWBcC\\\\\\\/+UZVoD4vX8Qj2fkafzNJxbJxrtJOD1pqtWNVEyZamETcmqHO8ZsNMXsqA7vU0sJ5\\\\\\\/8HVJKfxLHRoWMRpKvaERTWlYq9HyHMZb27xMjslJ6+gJ0z0dafQhFD0utGaBBwW2Tzl5s43Et70dihsrU8mHtCOFt5D7MIsyp5eeJWTXAVQvNaB\\\\\\\/RoI9V3rDF4Nx9a6K6rNrt+BX8Iqttb7MTwLQUticaKYqI\\\\\\\/VCww0vC92YG2KJAX0\\\\\\\/2pTMRMOchqPkT5VpkCaODXN5pvjsOncT5Ya5mlLo7RQDCfQHqJYWQECAcXMo6yFEvQQwdGrafTiCKwXlmZ\\\\\\\/J4m0c0QC2FByJd9RpiI0bJkedeb5oAv6b7SumfFaXKiTYUeB8Djd4U+mRg6iO1jVVSss+fYFUiS+3Y7hMP1QWqnkwZWz1Dp7UD\\\\\\\/Z7gqZ+FSYjjNuM\\\\\\\/3BTUb8AbAsk+w8i09BvMKxpcN9lVB61EaC0KemIQm1\\\\\\\/RupcvEpb7qyR8QmVXocScm\\\\\\\/uI7DWpslRqtHullAjAgCxVhnI9bXi8KotWx3Blq9Eyxu7lqnLqnOkX2Gct2rzdM7gnFec+GgaVPt5M7\\\\\\\/RaUPazx9AiakKNQay09Mh6LQ9KC5r8Xa+0SuJEkTflXHZsk2bzBcLR\\\\\\\/NHrqXveKq1H3pB70fYxbIQgT3jB2vtXvuDH7rU1MaWIImRYvdTZ60KuiLORfsqYXw9ZFIQ6PYhA6J2fFA0\\\\\\\/O+dY1LrqVvIE2MNpc\\\\\\\/8kKBftVMMtacuiOac1wA7kIbEcJygdATGHg8GxXz80Oe4+RfP1WxfTnFzdRH8ryyWpfRh0HD542+yEa9M5UqH13dMYqqBR7meRjZDqIcLwKblV2otRwWQk8OeoGrcgFAyIq97\\\\\\\/\\\\\\\/sXHw+cKjfsrdWyw52PIlG7hOG3pLANM95CG7dt7O1xXwZkXn6AJkvpUtr2FUDCssWOUtk\\\\\\\/OXVj5vf6zsDRcqTlFbdz4k6Ln3DqvOAmJYEZldBbeDt1IETZWnc423JZCWAUZRdud\\\\\\\/mypIm+Cyuj18hKpR3LFfKfoCAtvBYooNwG98Vp2GSazd+OiVcbPasVLwJC6VzcpEK6n820ZPE3m\\\\\\\/Lopno6YpubTvB+Zfzy5rAEfQSWEB7aVXhlS8mmfe+R3Cb2jNSrST1nN8M5meDYTN0zOISnMFk3IC3cTZDVUqZHpVTs+nVcAYYPVcIvgaYgeL4ZcBqDz3qU3G3THj+iDePoLymcwiaszu8zezj7\\\\\\\/LL+4k1KLXBPcChh1PGpC6jpKhbGsxz\\\\\\\/mkvG1Blry1bYSIGItK9j4N\\\\\\\/yJWyvx9dzsw+G+IdOGv5UuWpQydlMV\\\\\\\/mg+CrOYnPajmQQ6sdPzP03eps\\\\\\\/Ao9hATNvhS++A1Bw5WvYsM58j1h\\\\\\\/UU\\\\\\\/GWqiMgsxKh3tAA+BI6R9w9RCZFKde1SXfIE4Leax+Io4gUHq1Fe5Pd9bCMgI3I7ld8gKUa+4ZvkMTSFOwZiWX7UeYsCbfz7SiNWIbn6mxG5F0rh7fcKmrigTlCp+tp+O4aMY7ABRU9eEa5d5wBgufqnC+JYg74+\\\\\\\/r9JfyjgacnJ2zX8G+7uGIyESvf9AXOrUCZZGqBrc5MWCjsPFhg886TnTMRNPYroIS2s7vlRPhed8yroisSsZ5eHzZF7abbqGnJh2YNkTQYHbIgoYCGLAYLxJCM5CCFadHk+I3Im9d5CfuE\\\\\\\/HyahY6mRxTW46YREe9g9AQudH0cUGgHMoRY3bQf8LmLVI4liZZH4aI0UK7Gy8HlglCnH+04DWHfMTBHFrID6gcHx\\\\\\\/shSE\\\\\\\/bf0w1zRnjlP07E2YscAJnTf+4e7LGRjdyoDDml7+xrKUCob0F+M6MQ\\\\\\\/chyWmaWerzXsuXjcd1r2ZJ2axFPpeO6tUIXqB60fqih65kB3vs\\\\\\\/qMEitGXkcMplbKMJT9iwEvT7QW3zcgRpc7ToZSIr44T\\\\\\\/hPvjGpE7GzLm9lE7Ry3cj1Ob\\\\\\\/r\\\\\\\/xpu\\\\\\\/vTSllmzXL0JGuGNUYjJpVW2Eigj\\\\\\\/RBjnRokO85bLk4+v1hIhfyG7d52ck9o2Y0ao5FpshzYY+ehpFrZr5\\\\\\\/qPOnYYZzkp4HdqU+Uqdtfm+icOH07zreeaYYuBxnCsp\\\\\\\/o9VXhjQba8+TN6N0b0bdm1ecI8VAwSDhxGkKE9MFmO1jdZqcB\\\\\\\/sMaurU3WLE8+oijMHQ7JC\\\\\\\/if17kwmaTuOzGI\\\\\\\/vfqawYzB4r0gAHRpdMwh1YMh42G1ebJJ\\\\\\\/UwAmc9YFWSUJnylXI97ceMQVC\\\\\\\/wmDeBauJSDiPhdlMcuk7WTunZvj3572p6zkrX2ifgu+iVrpciOxZ4z3Mk5KdygwoIMbdTl58zywaHpid9M4aSXN6S6cRbsnwargF8L8GFZRUHk\\\\\\\/o9OZvbTIyDszzlZ0BCxdkpTqfsJMVcxwKAw7hl1MEofmqMHo3ElTEYiIB2HO9\\\\\\\/LI2Fo740elxusapof7SdJB4h3wWBfpXlWNDEcqq4UPFcx9TZXSG0E5iVw9OTVNAvVNazFRXl3zWA7RndZKonQ5N9IKrm8NyZLz1Iqp4k2lFBgKwTt0cYbAmMpTya7kxx\\\\\\\/YcZE\\\\\\\/EvRsBjVt4QBy3OfT9exfN9pelu5E14VNLWCq1wDWqZVsq0BzsdWhrERL7miNUpZNFfYN1KIKiVqSKIRbX3aki02064ZE2IQ2tdByJ63kCQ9C2BhiYdeAe2TuUrpO6I93zIpIPHgH4V4qFwsMoopzAV9hsfGHDU5wTHqZU5B+tcXCV\\\\\\\/Abx8GfQN5yE1QZkPKg05xndaCMFQvuUGkNmU0hTu9TcXMXXvi7tJssv72ZyE1SEmvK3lF4CzKlWnN0rlAwE4v8ikYw19TWVUS2hqPa02DqJk6FL+LRQLl6ArsOqL2TquFSedo06ZfTf6\\\\\\\/TtkAhNYCynuuG1WmgKgwG0VQdxMK+OeoUo6\\\\\\\/5xJ7nTsTAZ2g7TxPJt9LSd7deNGOR5KdqjBXfFB4CsTHyK7pTaupNbPIRh4IvQ3C0ziXk2iW8ut5K9R1GPV94dPSuDrP2P7ALLp+7wHwVcYI\\\\\\\/U1NiMzJ1dGhgAcygev4bfJvkoXpZgp67K5Kulgjfo6RHunb3hhlAhNoFV6HmEqIFabbrqQxhvFnlluVlZe\\\\\\\/x3vq+4MrygBig\\\\\\\/3PmmDdGwGApoWiBVWos2YLjH\\\\\\\/Pv+AZtZksn+oF1Cen3KtFRssVJLGx+pIReTGimDXNLYwfTGx91TTwRJcayH7+rHa+RIsEMxZkVKBeLDRv\\\\\\\/LFsylu\\\\\\\/6Q0Jq8ERCmHZLKrP8FYNFJqW3VfdR32pI7DDtnaWu1JXCydpx8ffZnriCinTmx6G8ezHK2n\\\\\\\/gJhpSPA5JAQ36bSi9w7tY3n0s0VkgOEMF34POpaarpUDojPzTPOm8OT+qoaCtmy45biwggxNKblx2OVxm5T36qEegxPVe86cBPPFjLnN9GmW9zpNfZ7CP\\\\\\\/DtuK0ghc2u3\\\\\\\/iEXTnioAnMZ2h51S62O\\\\\\\/j4T1M22KR2O80HXunKpmHotuR6zjOY0W2CmdwHFobyHmucWqYSExfsmO9ZhEUfKR5mlD5IUz6TktpgC0sGVkXtj2Tp71ULNfKTUL0m3zh5F4CYSK+Dzs+q37u+UBJQrUGaxv7aqULfkyn7IRHdmuGmG1BKuOHqSsv\\\\\\\/eCN619Obq74TwbN5WILLVbkNyBIu1xxWFd6H4NEXOabeZeYvG8J9KP\\\\\\\/uJrsP\\\\\\\/9j\\\\\\\/wcGL7hpYKMjQ934P4Y+YUQBQhXFOrFWGhx5S31VuCBm4EQKqN+ty+8QA\\\\\\\/8LaCrxl4NmAmz8lgUTmHdHGWS+SdC60Ek6QL1G7L4HB\\\\\\\/awogRFc6W22eAOCY7bqYfcmF3rfBZ0Fg0eHYa1vOcusqa44AzjlqzaErIOd9hFbD2WBWrZG9W7iVYsst+YrB6bC4+sxQPYhRboczqEiOcf34XwX0LU2iyaitQgMQF+oWiDP02kvDRUKGRoCv6pO\\\\\\\/DGpcguq4f9QuCiuhG4UhaE\\\\\\\/LnTxtyPqAgyw0Zhs6z9NKWC\\\\\\\/6dx4vGEDy55P6wNdJekQaNev2XPWeLHujW8KOyJORO36pYZlLMpdJFPMlZKNKeXuHbADy0op0SwLhX9MI1yXmlyJfBM4H9rkdGrlbUPv6QAb4ZuQc+ZbeIM9HNjBCMlzwzRCrZijOFatJmmLE1YnCU2I8aLL3zGO0ETspafuaTGZKp48EGYBab0xq+I5XeThq6wL58zoKnBGdHC2zbS1lv3\\\\\\\/EB9XwDYnIRYcowmyzy8B78w1fUJ9\\\\\\\/ytbVczYlJCNRFtPV7EtdeKT+eGYC3trbonjlslGayw4isffRF9GpuuWuY6EVS9n2eRCT1grvhxWMz1YNtDBPqaNXPERnpzPlwZmI4QgyYjMp2EYxd+2\\\\\\\/IyLmtzdqH2LuXtp1NzohVWK78sbWFkqd2hVq\\\\\\\/EKM6PEQilRv7yY8Rmoa5GJnqO\\\\\\\/+blny+sS+X3zi0LtI96pDg31kvkLD1uRbO\\\\\\\/zhY9GooqleL6A2Y+AmKKLAfEknFhmAYgGUVDaMsi9FEFlMrvGtZsZskaEwszMXyx34vw76xdiqSDu5QS2AK5deh9P7fvKqZaikMRf6rN6vD26SFNNlx7wNypWZICDA\\\\\\\/eC8KBWO77fDSXJphbGJTFYjvKrJsPFvDo6HACyNk0PLvBUSb5CqbtyYEJMBNd94qbubS\\\\\\\/NHj9JjWWOMWbmXW259IzBGArvMJcqVv4YAnCsz08tdDqlanSlxILYpy6qjROdb1n64KyexwnoO9Vqg23VIcz2qIY7F9L1vqn6tFU\\\\\\\/seRWjBwkPNy9QFkjuUCOt+chCV\\\\\\\/lk\\\\\\\/xb89uzZwehBoM5+uKz8SvoKLUS1Hmt+YN\\\\\\\/sObvNrRnp1ToD\\\\\\\/OsBfn4hUcT3rW5vy1xo1AIWzNFAl6bjxvW0eQHH0sy3GncAD4SwN52k3xrtmirtjDTAYc2LeHdGv2oq3rrm+WUHvAEuaUb3wpQ0P8ErTlbznhIxZGM0E+Dj\\\\\\\/zLxKn8WKIVLUpIXuPH3aSCH6UUEKEl6WmCuYET0ZZmLxU7gyINJVYH+tmt2FtkOAZ7UVvVo953343AK9Me4BSjDzh4l2BNQXx15+CGE0hrf+JQOiyqJFc2GHk1DDgglhPz87Sl+UTnkMGodki87FRlUvbhnTnWShk4grEYbx5cNYmiq32xBH54oMUUldDYZmwPNbLOF3tSDKxBw+YswV+kZ6g8PUIeBKEXIbcSaOgQ8Fo4EWVezolzzZ\\\\\\\/qfUYv2oiE3qXox9JtyVZeDf9gJIZHE37vq3pRRdUAfwH0d9JZCzwzkcYwvp+2Fwy+i8YUgeScJi9i2zOWF4e63FmKEoa+jhyldy4ep8wGUSBkOC+nYD24EQ9eWiNj351IuSTG7gjy0\\\\\\\/K4VqQ\\\\\\\/ni1UKiPt4zrsR5aqnixE3eby+cMN9v9Lv7IPp50RmPFBz\\\\\\\/0qDIltN2Q4vEgGW137SI3znD3NcMoLN7YQFd+YpNC7pwdtWekHMpriGbzSUxsblATPdnIs2IqDdA82zPVI1MEz\\\\\\\/cRj0skOZP\\\\\\\/prNj6j2KQQ4ZkCk09GQNGIaObs9KacIq\\\\\\\/ow1xE\\\\\\\/IA9DXO4nZQC2Ft7u21nrrUM8nENnk38OMEDaiQ25e24yaOZuVo\\\\\\\/akrTAvoyaFha9rlU0oEYVH2AEuKippFefEY59fxr0KcFfwxTUR1tnRmnBt6Yd0RS2Xgs\\\\\\\/0JqaqhzGJp\\\\\\\/P7ScyEk7h2kzE2f1kxF6nsMEoaR7TRSGEVPA6D0n4uChPpMzTU2+7OCDyo9A6Ne50P4ihMM82zCTkeeDOXnqQhuVyGznNlTYAu1xKwtulZ9M1vbWEFzw6\\\\\\\/bpfN5XOohjHQpKSwMsGPt5CHt3o5K5TrvVLqLKZi3sd+bAQqN+mkCSpdo6XNhZY1e1NjTJc3Rc7AYxM\\\\\\\/lwFx97fyCbqatDCxx9rcFhzHRqp4TV6G535VlDGKgh8NW3XZtKP0WQrwndGdaFl29gcn\\\\\\\/2JrYr6JoKVYiQSljah3IgKk0OyOMPGbJyyXtQ0vaJ21BhkkK+TeU6riZjoUKsZkzQmAByohRBbl33L2qHQwmqpCXW\\\\\\\/ytGuUqsYtgfzhZJTkNij7ifrztCmBvfAhVPDN7I+LB2hMyElHMVD1kz72BH+ScDFYZdrx+v8rOsm3+HVAbpl6j25+LNmSwI+OswQjRCAe+wVXV+lN64oFGrfWnwc+08EbMHCFTyDMtPDEFMWRLXwaWIyQzAqiWtM37S5zHD7NWVR8vmhiUyGcOLeoFn0isYNZchLqem6oCtprMLNh7RmOaU95g16b8+xWYqTttpnBehUPGIle8SvfGLkg245risDYmNXy\\\\\\\/5mw40O1CHCjHGYhJOBW3IYWyYjinJx+E7QR\\\\\\\/AsmtW8LloeU6g69Ssis7TonBDol\\\\\\\/NNVMcwamnbWps\\\\\\\/Y5eaYirkTEV2i+NvVsFuC6oBB\\\\\\\/oR0FpzTiS\\\\\\\/1waIxgbeFG3aDQ6v5+z\\\\\\\/j++tafO0pKgXNiMIU0CP4nTDYS1XkOx56CSSqYwuV\\\\\\\/lfvdytl\\\\\\\/xbZ7mNaWcvGHz\\\\\\\/3r7vMVtKF+zhleSiJbr5SvGXMXY+ugskpq\\\\\\\/gxEm0bCBAw8XYjfESXT7\\\\\\\/zPL8NDB0dOmGfkATVwXb8LHQ6ZMOg0+EEpxTAQOkRbgI+WuTA0mMH3j2800zElLs34gJbEdQPYr7qDoZkzk7WCaRMscMVoO86oy2auljIKiZz1o3ruWNznzxMg3GGvtXCU8YA0RVkK+d\\\\\\\/HU2Gx65VYUz\\\\\\\/Bi7ymzbg701eszZBjmIJYpeMzgV\\\\\\\/nkrqHG4OM4oaezA0FP\\\\\\\/kSki+w4f0QCIbZZd2VLhRIWm210rqnxT9Q\\\\\\\/kuiNKR+C+9bciWvm920GKOGB+EGk9O0KPw1aqtOJYX2upzcRwgU6MoznwEY8Yw8f3bu70ANnKsD2XfnXIorlQnn48AzJX2no8TNu5KbGv2meZ5Gz10wfGQ4rMdwbQpeoCn1cglkM3bdX6dZ7Pn38jZR7uuN2iD3VP7fTpFlQow7MZxZY1uCtnj8J2bSi9wdTlVuG2w777tHklcUucfL\\\\\\\/iaC0zVQx0KMN7VDdla6T\\\\\\\/yz9YLP4H484seCb7c5H2zTvE4TlQgIUysDlECRsycCRwEF\\\\\\\/aJlYG\\\\\\\/IqtRTdmv1MjJOc+4u13dXkCn7\\\\\\\/KBtl+musIhTnOLdM9NpFwLDnldA9yP0sHkFJaIQA6e\\\\\\\/X4k\\\\\\\/5ewwNdyWmM\\\\\\\/Kni+PhhaWC4RLv8ls9GMDMS2SG39XljXCo576HWuoeYRuyE3eFHpjscOmGROQbYnjpCFpss60ZAydbtfwU+Q8MsshElat4AJaNdSGqPZwYb0Rs9UacdElpHaieqMpq+UUy0DhgY3CR+nmO2iwr5LVuwhGMRBLfhiUQw3mcB07gOy1uFbUVoui8iyHRo5DtIFE6sZUcvJALnH3g1+EjL0ZcJRMfy6xAd0xH+q3vfDTZcQgTnLvCkBGWRI05uS9zrlhX8+IhNHILBLFWXccoYbEB+E0ru5EBvUvWPSvuRymb3X9IIS01SmVQDfYfVt7v3qFfelyZg8C6TG0Acl6LPB7oDDRY8hZpdJCq3d7Hr2Lfl5kGBC6u9hrn42YYybheQUAPflNZixAQftNh1CdDe7aXnX2jyIhtw2cLs9oms8CDaM+t5Z\\\\\\\/JVJ8fo4XrFu8h4zcDlgAohkVUNVFYgcw3SUm5SMvvhbp41PAazhJfVW5wutavJa7ktu12jQh3NJwyzq2ekRCwKvimgqROG4H6TADKl353SKdPAi9jzxQG8\\\\\\\/dhbzyarSx7xDc2gSUpz9SJ948Xls+mM5C0CTmPxQx\\\\\\\/WAGKvcojFRtBho8Ga+SqkTZI0uvcgYsp5Nh3lIKlpdGC7bbePkHtS2lFQQDNmJLpYsxqrrQfblf1Vjo803fRnXgizgUPC7DF663gsWiyv9br2UdsbhQL4T0W95D4N1uu4SsIwvrigCkbCSDK05LKe6cbBiTkha+e\\\\\\\/sKcD0oLXtCmHQcbf3pZxDceJcNUzO5Nwiv3WofmiFQqSyDoTnCy5DWnm3MVIoJLChK4SLGsF6mffxqZn8qoVxsAUzFsZfCpPLDi+F2nZt1EwctMEMefyuYjEH7xIVZdgBQiieWVNRdMVtubO7zEuxGlYSm5FXGIiJ9OmnaQ6XUfvyroK4gzNd7yRsisFm7IXiDF2\\\\\\\/hD4dGRB2GJur0La7ilQ0CJvMPXM2MaN7raRB86oH\\\\\\\/iTPgo8dMAUWxzlyOrPHZD4T4hsTXfKu4pc2f\\\\\\\/4DuMxEHrjlOGBkkakphnieVhVSEbZvz6ElalUe8BVUrcBp1RaZbstfGPRoBGT0j2+ZrW8MyCnzgkFkLbGgzrcuWVapveWZzmxpPR+DB6VTmT0D4rGJ5fKYPuS1UhFhSrVnBLQMZV17aLWRwzFidaGsd0+LSIeNDi+\\\\\\\/d9tFBrhiC\\\\\\\/zrVDSC7tno3Ip+rR98T9A+1K791867J7VGUIdeU7p\\\\\\\/qYzHG8RQGweLPZpZJxM+unW8swdUQvD9WDdqU26fQ0mcjXO6LC74AftN8sdi+GnhmC\\\\\\\/uIzqpfCaxVfQCoLtw9hRylsKRpGELmf86MIRjEdowkCx9fYxnw02yq2HF2FmPfvA2qEH6aVRnephAd6DYK71MN67hzF2eQ1J+yt\\\\\\\/RW31+pwC8xJXpD115qBXZI4RYR7\\\\\\\/yuSLAUgkO1vf\\\\\\\/RHQomANSkynCrheTztQOKW3nYuNZ9z2zhLVSlYQh7G4i4920y5ftmzuI6bUwe81GtyhiHxEbYQJmkyWT9R5e5m4a\\\\\\\/2LTROTmkhlPWRONbodFDaTzWOo9LJsJnR0bXjIJKOly5\\\\\\\/9euSbFXvV8vFvSLtTpEgcR2b+qSBOwzRmXoRX49PT2JHR\\\\\\\/Py97Rba7IZQYjAFTVVumsVSZxcC1Cxp6zHkATythd6nYXNJ1JCbEB3uh9xc2WaZqyG+9fgLztbOAX0BHmKBHHzuXNNihGls4iOcGi9w6v1\\\\\\\/Btc4aYaU56G+yyzzq3S1RqRyw6kVOMxp9VUUz9A552L9aBDxXF7HStVtHc6ewrhCQ6C93fbZZAHQKmppt1Jj8QhJPmuI0ZCzp4r5HeWjIm+hINDNtUEd1REkd7RCi37V\\\\\\\/Au5IZgnbP4e4dMYeaxnILppJPoYuB3wyX8HqYQy+FYhRPKJPvViIqCzSFDkQ0jqZ+P\\\\\\\/34FpkY+DVo2wJ4Yh0+ksxUEywDyesha7KblEYrqVZYUglmihHzFgNyz\\\\\\\/tbYIn9KrW1J+l7a5+LyT72qyyGx0bajcS65qmfF\\\\\\\/\\\\\\\/RuoovKC3fpnFI7k\\\\\\\/3qpjlYJEOcWcQQaTO0hOdb9AfNiB8Lm7SONOVvtgDX7uUlHU\\\\\\\/SkELI2VBDzlKXTp0OVGIdehTWX5g9mtEKueDwcDoUNNLIh7DQCJbqbOBTHFSv9Jl2VricfEgUCF4OXmsEKv4jsbrZ15f7AnIHY9u5yadCPzU42jK2kC\\\\\\\/yRweKr\\\\\\\/dplknKCBXyHSl7\\\\\\\/QRwPiLNreiQ9DxIpZMAPRTBHEDf3DePrWZqDeYLSSI\\\\\\\/QGnwLJlTBFqPNHSMl+\\\\\\\/cKyjsCHeezKTpcYjicqvCf\\\\\\\/zWfF3gJVc\\\\\\\/ssPbYj48lKD6MSXPgVmuqloYA3NrB\\\\\\\/xthhB6jzXn\\\\\\\/FRkrCMvzrvkpQxNKQ0qMJZKpQcOAwYZlTPT37hcqWCRIqLfuelNwN7sWHjoE4LdQIho\\\\\\\/hlcv8bRrUDOd1XObXZazzJY2Jm5vkApROVUO6mFwk\\\\\\\/Z6tvfxNMKzhEq0SayW\\\\\\\/BuJS7l1W8G32AxyYOtBtFJdlSe\\\\\\\/ng0PjNhZ39ZB+zG2zG7jCObCjoQ17ycaNq1+7KDej1gYB0JMe60Pz7l1S9fa7n4qRjRKU9S+2HaZa2KxdY5dq1XIMk+oiFnpw1BPw+Bi3b9zyMWWJcj18vSxpSaSk67cy+WScpnYpAGQzVA7SeiD\\\\\\\/zWzr+\\\\\\\/yuG+UAvWszGjwGufrl22UbdY1vlTG\\\\\\\/pKFl1FoZm1be0l0yHMKRQwDau9M3CLypnSjDUMbM5NhH2580xQx1wH374Dz9S593xc+TofEU51JgBwP3iYleVLxjhM54ubzZ2GGszq4WdHaiNC7Vc8zZWlN0HBoGEubeiUB8TRg4QCupdB7CMv1VyR74i\\\\\\\/zYdAosWV\\\\\\\/8knsgwf1USL2BfHlrVP5OdudDGqZQeFPTfBJcKek9khQ8KQ3veop6nM3Ux0FUgZpcvO+tbr9c25At+y7jHRKY7KdpJC2WnLzmi8SB40YibegNStvR16R4pySfVM61MckiG8H27gRCA3rAcE8sc9it7G8MAZ2nBPAZy29HTqt3mbL7VL7TV9PygZdGD3eEN89lTDbNTcVkUWx0F08XQNk4DQifSNMUHwaICG\\\\\\\/k4AGjQjHkLDGnM9wHLdM41tE5H84hYfD+P2hfCFh+tAMjwD1qk1+zjiZDTuHRUUUlKdXlZ2N7sR4MKPadwuioG43mgljEaGMq4aSteQ1U++AT9\\\\\\\/c4DZyIZKTYQoAhsOE1CPrbgYuE4W0UieieY+k7yX7IQXFKQRyH9q9nNtasNAm4t1cxIseiQkbV5btxeJgkgEHkgk4uPJpcWZ4zaMoFEVoSH+Ma1ck+yG6vAHuQyHVJwSVl8gaVm6imKvgz7xL7rmqkuJe04DJ7Lussae7tg\\\\\\\/\\\\\\\/ePysV9dN0Pok9U\\\\\\\/DLGR687QEr0U4QA6tPyw0k4coG5cmezqa9aTH9dg3xJYpZ+w2ojanmqYOMpDNmwQRd05dQXHz24cSmffKxo\\\\\\\/H5F6b6+sUmDWicSrXsmCpv80Jcc2qoVd8+YiiHu\\\\\\\/+h3L+GeJVaH358IaE1qhnAMP5Lb6dcGYUNtIxhGml3fcO1ztbE+\\\\\\\/n7KxouqK8VgPa2WYWGx6npskiB+HytdINZuJp5WENBWDGng9VpSlTDdjJ\\\\\\\/v4JwwCp2ZiOse4X7rcBBa0gRMEf3E0K2aOL2F+Sv\\\\\\\/nJyrnN3y2C1kBGVEwgcpgW9l5\\\\\\\/7rFwG8GIljBl7QtJx9RuiluZeqrcUoXBYqx1dmAX\\\\\\\/0cRaQBz5swtdc15A3TMUOT4gF4y5nLDzr1Y4isz4PMZg4o5o+8kA2VKIoYnwGzcJoG\\\\\\\/EUVpeLjuEiL1fiSOFhF5W2R\\\\\\\/AYZTLo6KGuTWbGrSdmAOnz1bvmWTk9x9Kf3oyXyErM85XmmXkUwlzSvFphq05IujBWtjqyxUNULL6+78CBu\\\\\\\/UXNJszNS5mJJMrxQfh9kuZvZnMQpsmuB\\\\\\\/L4k1NLc23FTqLXi9CT6KyBU2bud+4KYFSxaNJVt3QVOR\\\\\\\/R1+dxATEjmlYjBHhq4cZQnAu1JPx7dhVF2G9+cEVduNfZkdqpmuGJvnaYrHC2fDEVHrJaaNhV22ns1cYxaPFusha8KDCwhEAHZfKBGyQ+FxCQjFOYd9bhPHV7zUxXHSLuvTUxbyJ5UFNwortCyKf0oEI14D1EE2vORhAM8g+7CIAMhAR94R2doIRY\\\\\\\/jUpZ5sVDVvc6hP5r7IWcgaSdhvxhq82xPLxNq8Yio7ie8kW7IxOu7Cn6EtVoYc4FUVpTOYlb0rL9SsGt7A8l+DXls8o5J6L8673WTnYlKcqFJhJs\\\\\\\/vTZhvhK0SFWL1R\\\\\\\/jiMCM6q1N+5R1C7AKHwIsz7qcfuzOjbLUCv3fJ3XLs3qSF4I1aGBzn4b3eV0iJ4gj2vq6hcjfnIXerhw+aqkJ37tIAUp8lfF0I0lpKpz\\\\\\\/ZcVulbSMZx2xqr2MHc8IEHd\\\\\\\/aZ2Yltdbw3hvXYpw90UEPwNeOEMGBRY0HUAwcSgZKhYNxZERzqEeE5a2hoZj7TibDzmuwvHAdoYuK\\\\\\\/nTOsU59sy87bM0PitjOHuTK2834kyPmxn+mELUZAh6\\\\\\\/Far7Y6aRnfAoUveBzca8sQ8\\\\\\\/HL4oP+QOfKraTLbQJrp3Xi3TNKYMyRFY1I3vWT4kshRfRlTmefPSBaMKevyTmJNkcaOL8AtaGHoLLAX+23MHTnrXVznVLXW1GHCYz4H\\\\\\\/c5n5aCBaYAt9p54Y\\\\\\\/1t4OFFwiLddmwFNEWxkDkMRvTu\\\\\\\/FA5zemZwRvrXlGelmN\\\\\\\/8POna4fabBcp2s2V12DF+T68poQi8G4x5e5lYBNPl0B+7AK+V5K\\\\\\\/WsMgps7xmJtYkkO7vRDBGSNTlIzSdM73+ovbPYtzlj33TEv+SklT+HA+l7khJ3tlgOShCf0RVKmxPYKexli\\\\\\\/9IlyEj5ko4kiDJRjilyZhduYjugEd9HuUjCITd1pY\\\\\\\/mzfpY8HgETlwodiDupprWkMtBRmzXeCrKW9in5q001TG1ZQVIjtwU7DplC3\\\\\\\/z9PlpcIC0k2mrZi+Cdv7fkCBe89SCuqmoJJJZ5+uY2M2eT2XV\\\\\\\/2e2pgr9ZQsawzjpKxBX4dSYFVfP5gO92tTV4e1Oe7N1RGWvMC7PQIGywY85u0DzvkgYL0hmhFSxqmhxCWpAObUStHspHx0MFq1015U1nmpipWtui9NTUzGSQP8m5qzkeOCXhR3rM0UAlK3+LpYs+nklPaFa0QlojzOXs1vuzpqzIhMcM7G5EvcaAbVeFJQtGZ9FilgwNC3Uq9wcrPj2LBI5I6NEWHIQga9Lb6oE08C2saJ5eNctlhoDF0OGSDd\\\\\\\/HiAOB695h5HhBZYvXwBdwk0dhhc\\\\\\\/CkLBLw9oVtemxDXP9a\\\\\\\/z9kzBjjGMfEIgktFxUB2bdN1cxJqhtG+frvvX7xIM4fGVVhliR9oUujg6hYsf1MdxT9ceEa59KHwjlD8UiuuRtd3H\\\\\\\/DbCrkk+O+bALuZxvwrp4PPsTd5ssMX7cGG57aWnoF62RMYM8xVn3M9KDIiMSm3M8jht9Zftp0yBIrLe6x3pUQ9\\\\\\\/4n8ysyCvyOFCW1+x792kQAsl5sDVUD8qx+3AMaU+8Gll9LNl805h6oBWuGnf1FJo4h6LOKADMgHAE8nXIU9F6d8Y+OqCuFm1llFZeKgK7bfgXSD+7ytEFyyQhqhE195GNJqw3t0WqLyn87N4SL1hS8Rv6Msj9JXEaQdw8VnZ2lvsQMt09SpF8A\\\\\\\/R\\\\\\\/IlA6k1526yH3O0izH+Z+TNvM6v1sbmyWwj9QcgYtda8KFpPIdoHtMYUvMGqbu8QqBhuix9G+GiE\\\\\\\/p9m+Z4qE0m\\\\\\\/\\\\\\\/+NEdyhXgdryrx4kdUzgZsrGhW1v2YesoL4qv2ZXhl1A9NUIy5r6eadO69ur+cCvd\\\\\\\/A\\\\\\\/iUTw5gnggAIA4Pz9Zg1ApwxGBXtffPa01vAicjkS\\\\\\\/M0vKxU5Bbok9dsMOos0draWpaIzLBHCDIu9+SO+VYrDyYxF4O9eOeoA34FAEhAFRZzawcn4955PRZkohGrCjR0c8JJHGngZCUSG6c2inBNxMtUQoGv\\\\\\\/fcRWnOSFZqtdZDSl0umG0dsvvh+74tjZaT\\\\\\\/P6pwgtH9Ad5ORNVKYlUsxDBuIVkoLdh9ISTDo1ShhYvJDU9019IOzBAsMVe6J4HohzYV791zoPLos5gcoPZMSk2ezq3Qx7jdaBUuEsnmxMW8f3+8oGMFpWm5auWPCz65lkqRcEwNZVrxgwtVWJ0l2SYMdf69aPwEbCsxRQ44JVTDFU4f44eyTn3kYBxMnZCWkI8EvI2QuyhyHQbPv5ox1tOX9Dq8dl1B4PfvZ83T1jBTkzkO3h1VRMLlYTzxt9N0mcJh2thKlBGkeL\\\\\\\/UJ5ihRpWuWj0yNKbDIJCG1Gsec6u0w6\\\\\\\/CXi1mYpPMfjHza8+xbmBXnAgxSt156CJRCOi1YUL5WyGLPQqObK2Dc53YFBZ\\\\\\\/g6mzacU\\\\\\\/ZrppW7W6WUfeRecW0XnVjqYBxlYAmJ945\\\\\\\/OqNcK\\\\\\\/D7veuJQRQUMF3MZAHEOrF9ed0m1P011BO0TwjGbuNkl3FDE+g3FQ0FdBWy7xwQrHpoiwzCo15Sulrt9R2jiPTVBy5yyULs600rZHq4D\\\\\\\/2pq\\\\\\\/h4duvZBHsO75dsWxGytTarckt19VfZQOj0ZTUcq6dKueJ7I\\\\\\\/QHImEo5oZDcSTCOTHb3Dy+U0fjiSIQjccggzvFCmDLnDxG6F9me1uSGv3HAYObqjxPDERZqHBOYqqgdMnZILwVPpmEtnjr4m6j3PHTylUj7mnjTRFw8MZor06zDCFNc\\\\\\\/KA281ctSo+KB8xn2qMxSOgGysd8j\\\\\\\/q2aDra2vUkQ+5qTPMAsbfJqwbyY6KQGiK\\\\\\\/S9++NF8ywHEqoZsyIQIrC9RK1hdw2wEx5uoMa8iAJc4ArPt2dNYaPcVX+dRdNVs3JrFpr2xra8XZbGwpCBwp9Mb\\\\\\\/LKRAEh4ewQuHkw4KgsZ6sAKWfytLgrcJLUPQBpPXOSI\\\\\\\/PZGDdnCl7daq56DUUVYPNZmWU4krFoVRzCAO2urvYOFh20HK1Lqd72sHLmb\\\\\\\/aDszb6ZfKfz0QwY1UUo6iFseOg0bp8lCkwo4OS8ojZFDb+MaYSVoxutreVQqGVnH0hC1Tpn65MIXA1iTf30tndjOeoJXNaq4sjfI5bkKpQpRE0oTwYLNo4htswTqB5+BICZQB+lQez3fL4Y3geHl22UZ470+R6bujrd1Mkti3uFguSrsUG828TJ6FXm8zbkWY\\\\\\\/ME3jIUf5i42tlgrMA1fZp5rO9hV775MakBZhpVKbFv5sSYhbFXeWGg0H3n2pf5bYse+WySrTMNWULPUKc+0cwxoSMKdcZ63rIpuf+6\\\\\\\/gUmWcmOUSHTgHnpL+TqnWHdk6QmM+w9v+VRLJnVJyqdq2pt88ApYFqRu1aShIq6wRYbgCnFUEKu3I6WI6TTGBmXC5m3OdXuxAMKuLh2w1LQWZoFtV2hq5XDS8q\\\\\\\/QPhQe+MsyFvcsw4h8NExo2tw4wwfIsDi99D2lNbqxpzcsHh7gMBjCk\\\\\\\/\\\\\\\/5BgDYI2ucyR5OZ621J6OETP+2mQ4jqymmc75DMXs5wK08jDG1R3okAZPAZyEEJPotiIBBovURgbUgDoQEF9P7pbBBky2VqoOsN0bM+d9\\\\\\\/4R37JiAtRjnOVFYm4mlvlG1t+66ZEwJgPulrYW3OIsAr5zSHr6sPq+gKJ7hHQoMu0is89r3mtzkK5yRFxgE897SHpGPxr9D8B4N8EPQOrE6qJjtYXa7Lmu21GLmTgUVs23KXv667hZxWrcD+uk001+V9mnOw73s38bPKjf\\\\\\\/HxBBqoe67uLSlwp6hEKxpZrJ\\\\\\\/\\\\\\\/uKUj7B1rQXdeN3o5ywmKpmPHDWK8kUHlTYnDcJvPKeCl34h\\\u0022,\\\u0022iv\\\u0022:\\\u0022BzHkQURqMDifDM903ipsjw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002242272ec4b74ef9d5f3055acc12d178eb78727cc812b59c2802f14b7a60bb3b5c\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022j1VnNmGu2wqpsn2EZxK+TKQ2E1cdOYYaoO6q07cJpyDHmhuzr84yAjn4QDZmtrD9ETTdsI+tV7wlREL65kBpfilhM0VawPa4LwKvJz7xtQU8j30dq4EhvnmCyfTZDm47AmdtvggI6mKq0Uo9\\\\\\\/ZnIBI0qSedd1ne7oFHcD3olaB42g5dtaotZnX70ZbGLLOOTR6hWB9ZRSVOJuLM8hSZJUvU4djkG15FR\\\\\\\/23nV3p4G+N3vaf5rUdimn0gfb99uAso+C3geLnFd1VPNnqEwbWE1FbaBA7FhddKkJt27s+A4fZb25Oz2Sde3zZFYkNNFU\\\\\\\/5T7hIdf2YEGWIwt\\\\\\\/zws3AxqEk+jQ34EUy6q\\\\\\\/AzPTEWpQCt7Bk5N2w9aN6TEHieLF0BJvfsrd8W9qymS7I3rhbbGQtk12xzjo9mlYmDp2LJvAC0Cu6gPeH++w1FaI4QDBol5xHQWH9LbGJa8BeKdCBwPluYukEXnUoEKf9N6jpTn4TiXTA6WxQgDlkuaCp6GyYIEE8heEkRunCDjTxd98CyWrXCaWqqJKQQbmsOhtO4HdMgB68hb7S7H45N0Ye4zUw4nksjcRhNTSbZ\\\\\\\/Ofic73t\\\\\\\/nGgOcMyGOpbpFnDkRETYmG08\\\\\\\/cayORKdE8TgHckUAY2NbtvH5gp2Ghy\\\\\\\/HwZgvpIdMORQsXQwUNj4yJ\\\\\\\/T5jrDLgwexdmKQkI8+3jiKxZWlD+LtSNBb5MmEdhxLkcR7TEOJVzOaMbhK3sOBKrofvMpSVKyvWUo8QQOenBmDnRQWt4GLHU+eQEGof0bZI5iUEQLN\\\\\\\/fh5LWHmhG8ImX3eGBafZtwUyx3qYublgnczWkiMEJwYD72E\\\\\\\/PQJ+I7Ix2RBjZ2Jk439PwOsrHv87kNvy5xyPmd1j1P6OaX7CptSQOWQlfWlXFE0V+w1bR+lVRrRg\\\\\\\/U5hB63TlkFNq619igJ7QSyaBVX6h4+\\\\\\\/MYMizBa8UGoZbmTmz6jph9jSR9IZqwUp1hFMH0zYhf3N\\\\\\\/h9G4cjxFDChN4aCRUVPOjhHCyexbL3zJJRSQ09FSk3ubvlAsmK6Dal5tE0C50STUQDP29OdVpT9alzYWJeYSg2kzwbk\\\\\\\/02f1MA2Ut5OSsKlGg+WRpx2oKNH\\\\\\\/QWMymV6Ryq8ZTNUx2R8I5slcMmlvyPgxKSElGrJ3+ksIbIc74AP35lBmljs9FeiqRrbgQeU0hp63TC6a\\\\\\\/GhmTSIlUr9+dK\\\\\\\/qBMJiIL7G7Yl+Ejya3lJg8HEDOtKPBOK+io8U8cVdSBem0dGsGf55fpwXD+VCbzYaApqEw6b2GYh6YcdZzzrWuy4n9+pLAcWHz1naLiw7WTiQ5AfblvFWctU0mZYv5UiY0\\\\\\\/28p09HqBx+TwVklgisQKtn6Cq\\\\\\\/\\\\\\\/zxnwBlqTZiVsi1zQJy7OVXU423ARaVww3mq4cQWgxLaDPV86eIWwu35Wa0emWuoYCTGHBqPhFQaXTB74ZbZn0lPMnsR3NsMD0mNZA2t4g4pITvFIG\\\\\\\/LdD\\\\\\\/FRo0yZodEeJtKUvw1KSni0gMOppAYiZt4IzN2wXfozFOLple+L7pdAbbYS5SBH04t3BCEDhENgbtoygy2OMI9364hhKR+LbE+4yOOhaphGTBGmaHei5QfpNwPTRidgwjsZJu6Z6O0EUWZR48EDhcyoT1LJOWK6sMmPnvN0ufwB9AgK2RafJdfouKWI5S9OoyYVm+agBE15h\\\\\\\/EWzpfO7xTY1LJOHvMuh6ij3ObJgTc5okhmsxfiCngzhQNB1B5FfWsnc26SIvhGEC9xjAlFwchOPmazJbEczt5Zx5MA6d+8\\\\\\\/nUR+TbQ0UVz0c2UZFQz\\\\\\\/TmD62imReT9E6bfJy6Ca19Qtkpma1gFSb\\\\\\\/NbOHmAPGpFujoclBST5ktpl4w8pacdPivuPC39frvmW9QS2f6idY3nxBzG1xRqWWzZ5Axqe5kDo+Kp3baxKakXlRFu\\\\\\\/zlMWgI+7sfVi4JyS8OJyUpZxkhg7TNEo0yUtfSA\\\\\\\/t9FBQCKel4EzkN5RhffqjDdnS0oIAw57fDoP0Mq7cT+0J2SfMt9NWxaJ6a8=\\\u0022,\\\u0022iv\\\u0022:\\\u0022l7w9wEKuL4Q+i18kkor85g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022b2401e5c59e42089342c8c74973b31f9863f62a381d83d705f06399c25855aeb\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460152 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genomic-selection">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/10/2155" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-32t-H8zcxzLbpbdWCsGBIyGZwEZpfGkUuTjiyLwrhc4" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/10/2155" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-4dAZYKbgPk86CJWnWMHwKWUQ935ZkvVE8DZI84pu8nU" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/10/2137" title="SUMO-Enriched Proteome for Drosophila Innate Immune Response" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/10/2165" title="Similar Efficacies of Selection Shape Mitochondrial and Nuclear Genes in Both Drosophila melanogaster and Homo sapiens" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460152" id="node460152" data-pisa="ggg;5/10/2155" data-pisa-master="ggg;g3.115.019869" data-apath="/ggg/5/10/2155.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Genomic Selection Index Applied to Simulated and Real Data</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">J. Jesus Ceron-Rojas</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>José Crossa</span>, <span class="highwire-citation-author" data-delta="2">Vivi N. Arief</span>, <span class="highwire-citation-author" data-delta="3">Kaye Basford</span>, <span class="highwire-citation-author" data-delta="4">Jessica Rutkoski</span>, <span class="highwire-citation-author" data-delta="5">Diego Jarquín</span>, <span class="highwire-citation-author" data-delta="6">Gregorio Alvarado</span>, <span class="highwire-citation-author" data-delta="7">Yoseph Beyene</span>, <span class="highwire-citation-author" data-delta="8">Kassa Semagn</span> and <span class="highwire-citation-author" data-delta="9">Ian DeLacy</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2155-2164;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.019869 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460152" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">J. Jesus Ceron-Rojas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Biometrics and Statistics Unit, International Maize and Wheat Improvement Center (CIMMYT), 06600, México Distrito Federal, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=J.%2BJesus%2BCeron-Rojas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ceron-Rojas%20JJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJ.%2BJesus%2BCeron-Rojas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">José Crossa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Biometrics and Statistics Unit, International Maize and Wheat Improvement Center (CIMMYT), 06600, México Distrito Federal, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jos%C3%A9%2BCrossa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crossa%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJos%25C3%25A9%2BCrossa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for José Crossa</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a></span></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Vivi N. Arief </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>The University of Queensland, School of Agriculture and Food Sciences, St Lucia, QLD 4072, Brisbane, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vivi%2BN.%2BArief%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Arief%20VN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVivi%2BN.%2BArief%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Kaye Basford </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>The University of Queensland, School of Agriculture and Food Sciences, St Lucia, QLD 4072, Brisbane, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kaye%2BBasford%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Basford%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKaye%2BBasford%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Jessica Rutkoski </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Biometrics and Statistics Unit, International Maize and Wheat Improvement Center (CIMMYT), 06600, México Distrito Federal, México</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>International Programs of the College of Agriculture and Life Sciences, Cornell University, Ithaca, New York 14853 </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jessica%2BRutkoski%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rutkoski%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJessica%2BRutkoski%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Diego Jarquín </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Agronomy and Horticulture, University of Nebraska, Lincoln, Nebraska 68583 </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Diego%2BJarqu%C3%ADn%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jarqu%C3%ADn%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADiego%2BJarqu%25C3%25ADn%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Gregorio Alvarado </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Biometrics and Statistics Unit, International Maize and Wheat Improvement Center (CIMMYT), 06600, México Distrito Federal, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gregorio%2BAlvarado%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Alvarado%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGregorio%2BAlvarado%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Yoseph Beyene </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Global Maize Program, CIMMYT, Village Market 00621, Nairobi, Kenya</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yoseph%2BBeyene%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Beyene%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYoseph%2BBeyene%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Kassa Semagn </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Global Maize Program, CIMMYT, Village Market 00621, Nairobi, Kenya</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kassa%2BSemagn%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Semagn%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKassa%2BSemagn%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Ian DeLacy </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>The University of Queensland, School of Agriculture and Food Sciences, St Lucia, QLD 4072, Brisbane, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ian%2BDeLacy%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=DeLacy%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIan%2BDeLacy%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/5/10/2155" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460152" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460152/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/10/2155.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460152" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460152/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/10/2155.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460152" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460152/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/10/2155.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460152" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460152/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/5/10/2155.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460152" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460152/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">A genomic selection index (GSI) is a linear combination of genomic estimated breeding values that uses genomic markers to predict the net genetic merit and select parents from a nonphenotyped testing population. Some authors have proposed a GSI; however, they have not used simulated or real data to validate the GSI theory and have not explained how to estimate the GSI selection response and the GSI expected genetic gain per selection cycle for the unobserved traits after the first selection cycle to obtain information about the genetic gains in each subsequent selection cycle. In this paper, we develop the theory of a GSI and apply it to two simulated and four real data sets with four traits. Also, we numerically compare its efficiency with that of the phenotypic selection index (PSI) by using the ratio of the GSI response over the PSI response, and the PSI and GSI expected genetic gain per selection cycle for observed and unobserved traits, respectively. In addition, we used the Technow inequality to compare GSI <em>vs.</em> PSI efficiency. Results from the simulated data were confirmed by the real data, indicating that GSI was more efficient than PSI per unit of time.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/genomic-estimated-breeding-value" class="hw-term hw-article-keyword hw-article-keyword-genomic-estimated-breeding-value" rel="nofollow">genomic estimated breeding value</a></li><li class="kwd"><a href="/keyword/net-genetic-merit" class="hw-term hw-article-keyword hw-article-keyword-net-genetic-merit" rel="nofollow">net genetic merit</a></li><li class="kwd"><a href="/keyword/selection-index" class="hw-term hw-article-keyword hw-article-keyword-selection-index" rel="nofollow">selection index</a></li><li class="kwd"><a href="/keyword/selection-response" class="hw-term hw-article-keyword hw-article-keyword-selection-response" rel="nofollow">selection response</a></li><li class="kwd"><a href="/keyword/genomic-selection" class="hw-term hw-article-keyword hw-article-keyword-genomic-selection" rel="nofollow">genomic selection</a></li><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">GenPred</a></li><li class="kwd"><a href="/keyword/shared-data-resource" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resource" rel="nofollow">shared data resource</a></li></ul><p id="p-3">In genomic selection (GS), phenotypic and marker data from the training population are fitted in a statistical model to estimate all available marker effects. These estimates can then be used in subsequent selection cycles to obtain genomic estimated breeding values (GEBVs) that are predictors of the breeding values in the testing population (candidates for selection) for which there is only marker information (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Meuwissen <em>et al.</em> 2001</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Heffner <em>et al.</em> 2009</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Lorenz <em>et al.</em> 2011</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Nakaya and Isobe 2012</a>). In GS, GEBVs are tools for ranking and selecting candidates for selection. <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Bernardo and Yu (2007)</a> and <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Heffner <em>et al.</em> (2011)</a> have shown that selection based on genomic predictions can lead to greater genetic gains per unit of time for complex traits. <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Technow <em>et al.</em> (2013)</a> derived an inequality that depends on GS accuracy and the square root of the heritability of the unobserved trait, which is useful to compare the genomic selection efficiency with the phenotypic efficiency in terms of time.</p><p id="p-4">The standard method for predicting marker effects and breeding values is the ridge-regression best linear unbiased predictor, or its equivalent, the genomic best linear unbiased predictor, which assumes that the effects of all markers have a multivariate normal distribution with mean zero and constant variance (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Meuwissen <em>et al.</em> 2001</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">VanRaden 2008</a>). The difference among the various Bayesian regression methods lies in how they specify the prior distribution of the parameters of interest (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">de los Campos <em>et al.</em> 2013</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Gianola 2013</a>). Methods such as Bayes A and Bayes B assume that the variance of marker effects has an <em>a priori</em> inverse χ<sup>2</sup> distribution (<a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">Meuwissen <em>et al.</em> 2001</a>) that produces shrinkage as well as variable selection. Nevertheless, when the true marker effects have a multivariate normal distribution and the size of the training population and the number of markers is large, all methods produce GEBVs that are highly correlated with the true breeding values of the candidates for selection (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Hayes <em>et al.</em> 2009</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Verbyla <em>et al.</em> 2010</a>).</p><p id="p-5">In the context of molecular marker-assisted selection, <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Lande and Thompson (1990)</a> proposed a selection index that combines marker information with phenotypic information, whereas <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Dekkers (2007)</a> proposed a selection index that combines GEBVs with phenotypic information. Both selection indices were evaluated using simulated data and in both studies the authors found that the estimated selection response was greater than when only phenotypic information was used to estimate it. In the context of GS, <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Togashi <em>et al.</em> (2011)</a> proposed four selection indices similar to Dekkers’ index based on the best linear unbiased predictor theory; however, their results are hypothetical because the authors did not use any data (either simulated or real) to validate these indices. The indices of Togashi are a direct application of the phenotypic selection index (PSI) (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Smith 1936</a>), but they do not explain how to estimate the GS response and the genomic selection index (GSI) expected genetic gain per selection cycle for unobserved traits after the first selection cycle, which is important, because they give information on the genetic gains in the next selection cycle and are the base criteria to compare the efficiency of two or more linear selection indices (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bulmer 1980</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Moreau <em>et al.</em> 1998</a>).</p><p id="p-6">This study had three main objectives: (1) to apply the GSI to two simulated and four real data sets that only use GEBVs for selecting nonphenotyped candidates for selection; (2) to propose a method to estimate the GSI selection response and the GSI expected genetic gain per selection cycle for unobserved traits after the first selection cycle; and (3) to compare GSI efficiency <em>vs.</em> PSI efficiency using simulated and real data.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>PSIs and GSIs</h3><p id="p-7">The objective of any linear selection index, whether phenotypic or genomic, is to predict the net genetic merit <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-1.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-2.gif"/></noscript></span></span> (<em>t</em> = number of traits) is a vector of true breeding values for an individual and <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-3.gif"/></noscript></span></span> is a vector of economic weights. According to <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Kempthorne and Nordskog (1959)</a>, the selection response of any linear selection index can be written as<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-4.gif"/></noscript></span> <span class="disp-formula-label">(1)</span></span>where <em>k</em> is the standardized selection differential (or selection intensity), <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-5.gif"/></noscript></span></span> is the covariance between H and any linear index <em>I</em>, <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-6.gif"/></noscript></span></span> is the variance of <em>I</em>, <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-7.gif"/></noscript></span></span> is the standard deviation of H, <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-8.gif"/></noscript></span></span> is the correlation between H and any linear index <em>I</em>, and <em>L</em> denotes the time required to collect information to evaluate <em>I</em> and complete one selection cycle. The second part of Equation (1) (<span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-9.gif"/></noscript></span></span>) indicates that the genetic change due to selection is proportional to <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-10.gif"/></noscript></span></span> and to <em>k</em>, which is the selection differential in the index in standard deviation units (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Kempthorne and Nordskog 1959</a>). If <em>k</em>, <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-11.gif"/></noscript></span></span>, and <em>L</em> are fixed, <em>R</em> will be maximized when <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-12.gif"/></noscript></span></span> is maximized and the final form of Equation (1) will depend on the particular linear selection index used to select individuals, <em>e.g.</em>, PSI or GSI.</p><div id="sec-3" class="subsection"><h4>PSI and its selection response:</h4><p id="p-8">Let <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-13.gif"/></noscript></span></span> be a vector of phenotypic trait values; the PSI (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Smith 1936</a>) can be written as <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-14.gif"/></noscript></span></span> and its maximized selection response is<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-15.gif"/></noscript></span> <span class="disp-formula-label">(2)</span></span>where <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-16.gif"/></noscript></span></span> denotes the time required for PSI to complete one selection cycle, <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-17.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-18.gif"/></noscript></span></span> is the inverse of the phenotypic covariance matrix (<strong>P</strong>), and <strong>C</strong> is the covariance matrix of true breeding values <strong>a</strong>; <em>k</em> and <strong>w</strong> were defined previously.</p></div><div id="sec-4" class="subsection"><h4>GSI and its selection response:</h4><p id="p-9">The GSI can be written as<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-19.gif"/></noscript></span> <span class="disp-formula-label">(3)</span></span>where <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-20.gif"/></noscript></span></span> is a <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-21.gif"/></noscript></span></span> vector of genomic breeding values for one individual; it can be shown that the maximized GSI selection response is<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-22.gif"/></noscript></span> <span class="disp-formula-label">(4)</span></span>where <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-23.gif"/></noscript></span></span> denotes the time required for GSI to complete one selection cycle; <em>k</em> and <strong>w</strong> were defined previously; <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-24.gif"/></noscript></span></span> (<span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-25.gif"/></noscript></span></span>) is a covariance matrix of additive genomic breeding values <strong><em>γ</em></strong>.</p><p id="p-10">Note that in each selection cycle, matrices <strong>P</strong>, <strong>C</strong>, and <strong><em>Γ</em></strong> change their values as a result of many individuals being eliminated by the selection process.</p></div><div id="sec-5" class="subsection"><h4>Estimating the parameters of the PSI:</h4><p id="p-11">In each selection cycle, we used the restricted maximum likelihood method (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Patterson and Thompson 1971</a>) to estimate the covariance matrix of true breeding values (<strong>C</strong>) and of the residuals (<strong>R</strong>), which were denoted as <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-26.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-27.gif"/></noscript></span></span>, respectively, from where matrix <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-28.gif"/></noscript></span></span> was an estimator of the phenotypic variance-covariance matrix (<strong>P</strong>). We estimated <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-29.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-30.gif"/></noscript></span></span> as <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-31.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-32.gif"/></noscript></span></span>, respectively.</p></div><div id="sec-6" class="subsection"><h4>Estimating the GEBV and the GSI in the l<sup>th</sup> selection cycle:</h4><p id="p-12">Let <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-33.gif"/></noscript></span></span> be the estimator of the vector of marker effects <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-34.gif"/></noscript></span></span> for <em>t</em> traits (<a id="xref-app-1-1" class="xref-app" href="#app-1">Appendix</a>). We obtained the <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-35.gif"/></noscript></span></span> GEBVs (<span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-36.gif"/></noscript></span></span>) in the <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-37.gif"/></noscript></span></span> selection cycle (<span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-38.gif"/></noscript></span></span>number of cycles) as<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-39.gif"/></noscript></span> <span class="disp-formula-label">(5)</span></span>where <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-40.gif"/></noscript></span></span> is the vector of size <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-41.gif"/></noscript></span></span> of the marker effects of the <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-42.gif"/></noscript></span></span> trait in the base population and <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-43.gif"/></noscript></span></span> is a matrix of size <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-44.gif"/></noscript></span></span> of the coded values of marker genotypes in the <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-45.gif"/></noscript></span></span> selection cycle (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Goddard 2009</a>). The estimated GSI (<strong>GSI</strong><sub>E</sub>) values in this cycle were<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-46.gif"/></noscript></span> <span class="disp-formula-label">(6)</span></span>where <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-47.gif"/></noscript></span></span> is the <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-48.gif"/></noscript></span></span> economic weight and <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-49.gif"/></noscript></span></span> was defined in Equation (5). Note that Equation (6) is a vector of size <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-50.gif"/></noscript></span></span> (<em>g</em> = number of genotypes). In practice, <strong>GSI</strong><sub>E</sub> values are ranked to select individual genotypes with optimum GEBV values.</p></div><div id="sec-7" class="subsection"><h4>Estimating the <strong><em>Γ</em></strong> matrix:</h4><p id="p-13">Suppose that <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-51.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-52.gif"/></noscript></span></span> have multivariate normal distribution jointly, with mean <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-53.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-54.gif"/></noscript></span></span>, respectively, and covariance matrix <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-55.gif"/></noscript></span></span>, where <strong>1</strong> is a <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-56.gif"/></noscript></span></span> vector of 1s and <span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-57.gif"/></noscript></span></span> is the additive genomic relationship matrix (<a id="xref-app-1-2" class="xref-app" href="#app-1">Appendix</a>). Then <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-58.gif"/></noscript></span></span> can be estimated as<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-59.gif"/></noscript></span> <span class="disp-formula-label">(7)</span></span>where <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-60.gif"/></noscript></span></span> is the estimated covariance between <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-61.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-62.gif"/></noscript></span></span> in the <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-63.gif"/></noscript></span></span> selection cycle; <em>g</em> is the number of genotypes; <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-64.gif"/></noscript></span></span> was defined in Equation (5); <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-65.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-66.gif"/></noscript></span></span> are the estimated arithmetic means of the values of <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-67.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-68.gif"/></noscript></span></span>; <strong>1</strong> is a <span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-69.gif"/></noscript></span></span> vector of 1s and <span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-70.gif"/></noscript></span></span> is the additive genomic relationship matrix in the <span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-71.gif"/></noscript></span></span> selection cycle (<span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-72.gif"/></noscript></span></span> number of cycles). From Equations (4) and (7), the estimated GSI response is <span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-73.gif"/></noscript></span></span></p></div></div><div id="sec-8" class="subsection"><h3>Criteria for comparing GSI efficiency <em>vs.</em> PSI efficiency</h3><p id="p-14">Assuming that <em>k</em> is the same in both indices, to compare GSI efficiency <em>vs.</em> PSI efficiency in the <span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-74.gif"/></noscript></span></span> selection cycle, we used the ratio<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-75.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-75.gif"/></noscript></span> <span class="disp-formula-label">(8)</span></span>which was proposed by <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Bulmer (1980)</a> and <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Moreau <em>et al.</em> (1998)</a> as a criterion for comparing the efficiency of linear selection indices. In Equation (8), <span class="inline-formula" id="inline-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-76.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-76.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-77.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-77.gif"/></noscript></span></span> are estimators of Equations (2) and (4), respectively, and <span class="inline-formula" id="inline-formula-70"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-78.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-78.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-71"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-79.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-79.gif"/></noscript></span></span> are the maximized estimated correlation (or accuracy) between H and GSI, and between H and PSI, respectively. Using this criterion, if <span class="inline-formula" id="inline-formula-72"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-80.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-80.gif"/></noscript></span></span>, GSI efficiency will be greater than PSI efficiency, if <span class="inline-formula" id="inline-formula-73"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-81.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-81.gif"/></noscript></span></span>, the efficiency of both selection indices will be equal, and if <span class="inline-formula" id="inline-formula-74"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-82.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-82.gif"/></noscript></span></span>, PSI will be more efficient than GSI.</p></div><div id="sec-9" class="subsection"><h3>PSI and GSI expected genetic gain per selection cycle</h3><p id="p-15">Besides Equation (8) for comparing the efficiency of PSI <em>vs.</em> GSI, we used the estimated values of the following two equations:<span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-83.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-83.gif"/></noscript></span> <span class="disp-formula-label">(9)</span></span><span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-84.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-84.gif"/></noscript></span> <span class="disp-formula-label">(10)</span></span>where <strong>E</strong><em><sub>PSI</sub></em> and <strong>E</strong><em><sub>GSI</sub></em> are the expected genetic gain per selection cycle for each trait in the PSI (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Lin 1978</a>) and in the GSI (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Togashi <em>et al.</em> 2011</a>), respectively. All the terms in Equations (9) and (10) were defined and estimated according to Equations (2) and (4), respectively.</p></div><div id="sec-10" class="subsection"><h3>Simulated and real data sets</h3><div id="sec-11" class="subsection"><h4>Simulated data sets (data sets 1 and 2):</h4><p id="p-16"><a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> presents a schematic illustration of the steps followed to generate the simulated data sets. For the simulation, the performance of the <em>F</em><sub>2</sub> or <em>S<sub>n</sub></em> families was evaluated using the selfing generation (<em>F</em><sub>3</sub> or <em>S<sub>n</sub></em><sub>+1</sub>) of the <em>F</em><sub>2</sub> or <em>S<sub>n</sub></em> families, whereas in practice, the <em>F</em><sub>2</sub> or <em>S<sub>n</sub></em> families would be evaluated by crossing them to a tester (or testers).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/10/2155/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Schematic illustration of the steps followed to generate data sets 1 and 2 for the selection process using the phenotypic selection index and the genomic selection index. Dotted lines indicate the process used to simulate the phenotypic data." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1728775229" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Schematic illustration of the steps followed to generate data sets 1 and 2 for the selection process using the phenotypic selection index and the genomic selection index. Dotted lines indicate the process used to simulate the phenotypic data.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/10/2155/F1.medium.gif" width="343" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/5/10/2155/F1.medium.gif" width="343" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/10/2155/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/10/2155/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466838" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-17" class="first-child">Schematic illustration of the steps followed to generate data sets 1 and 2 for the selection process using the phenotypic selection index and the genomic selection index. Dotted lines indicate the process used to simulate the phenotypic data.</p><div class="sb-div caption-clear"></div></div></div><p id="p-18">We simulated eight phenotypic selection cycles [cycle 0 (C0)−cycle 7 (C7)] for PSI (data set 1), and seven GS cycles (C1−C7) for GSI (data set 2), each with four traits (T1, T2, T3, and T4), 500 genotypes and four replicates for each genotype under one possible scenario: 5% of quantitative trait loci (QTL) were in linkage equilibrium.</p><p id="p-19">C0 was the GSI training population, which contained phenotypic and genotypic data; it is the population where we estimated the molecular marker effects (<a id="xref-app-1-3" class="xref-app" href="#app-1">Appendix</a>). In all selection cycles, we selected and intermated the top 10% of individuals (<span class="inline-formula" id="inline-formula-75"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-85.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-85.gif"/></noscript></span></span>). The economic weights used in PSI and GSI for T1, T2, T3, and T4 were 1, −1, 1, and 1, respectively. Selections were based on PSI and GSI values that incorporated all four trait (T1, T2, T3, and T4) means in each selection cycle to predict and select the net genetic merit (<span class="inline-formula" id="inline-formula-76"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-86.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-86.gif"/></noscript></span></span>) of each individual.</p><p id="p-20">Simulated data were generated using QU-GENE software (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Podlich and Cooper 1998</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Wang <em>et al.</em> 2003</a>). Three hundred fifteen QTL and 2500 molecular markers were distributed uniformly across 10 chromosomes to simulate two maize (<em>Zea mays</em> L.) populations. Each QTL and molecular marker was biallelic and the QTL additive values ranged from 0 to 0.5. The 315 QTL were randomly allocated over the 10 chromosomes. Because QU-GENE uses recombination fraction rather than map distance to calculate the probability of crossover events, recombination between adjacent pairs of markers was set at 0.0906, those between a QTL and its flanking markers set at 0.0 and 0.0906, and that between two adjacent QTL set at 0.0. The recombination fraction between 15 random QTL and their flanking markers was set at 0.5, <em>i.e.</em>, complete independence (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Haldane 1919</a>), to simulate linkage equilibrium between 5% of the QTL and their flanking markers.</p><p id="p-21">Each of the four traits (T1, T2, T3, and T4) was affected by a different number of QTL: 300, 100, 60, and 40, respectively. The common QTL affecting the traits generated genotypic correlations of −0.5, 0.4, 0.3, −0.3, −0.2, and 0.1 between T1 and T2, T1 and T3, T1 and T4, T2 and T3, T2 and T4, and T3 and T4, respectively.</p><p id="p-22">The genotypic value of each plant was generated based on its haplotypes and the QTL effects for each trait. For each trait, the phenotypic value for each of four replications of each plant was obtained from QU-GENE software by setting the per-plot heritability of T1, T2, T3, and T4 at 0.4, 0.6, 0.6, and 0.8, respectively.</p><p id="p-23">In cycle C0 (the training population), 500 <em>F</em><sub>2</sub> plants were generated from a cross of two inbred parents. The haplotypes of these parents were randomly generated, but the two parents shared no common alleles. In subsequent cycles (<em>i.e.</em>, C1−C7), 500 plants were generated from a random intercross of the selected 10% of lines from the previous cycle using the PSI and GSI methods. In C0, only PSI was applied. In C1, two selection methods were applied: PSI (data set 1) and GSI (data set 2); the 10% of individuals selected with each method were advanced to the next selection cycle.</p></div><div id="sec-12" class="subsection"><h4>Real data sets 3, 4, 5, and 6:</h4><p id="p-24">These data sets (data sets 3, 4, 5, and 6) correspond to four maize (<em>Zea mays</em> L.) <em>F</em><sub>2</sub> populations. They were used by <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Beyene <em>et al.</em> (2015)</a> and were denoted as JMpop1 DTMA Mexico optimum environment, JMpop1 DTMA Zimbabwe optimum environment, JMpop3 DTMA Mexico optimum environment, and 6x1020 WEMA Africa optimum environment, respectively. These data were used to perform GS in eight biparental populations; field evaluation of a training population (C0), cycle 1 (C1), and cycle 2 (C2) from GS was reported by <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Beyene <em>et al.</em> (2015)</a>.</p><p id="p-25">For each data set, C0 contained genotypic data and four phenotypic traits: grain yield (GY, t/ha), plant height (PHT, cm), ear height (EHT, cm), and anthesis days (AD, d), as well as three sets of markers corresponding to C0 (training population), C1, and C2. The numbers of individuals and molecular markers in each population are shown in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>. Assuming that the breeding objective was to increase GY while decreasing PHT, EHT, and AD, the vectors of economic weights in C0, C1, and C2 for GY, PHT, EHT, and AD, were <span class="inline-formula" id="inline-formula-77"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-87.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-87.gif"/></noscript></span></span> for both indices and the four data sets.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466868/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466868/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Four real maize (<em>Zea mays</em> L.) <span class="inline-formula" id="inline-formula-78"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/T1/embed/mml-math-88.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/T1/embed/mml-math-88.gif"/></noscript></span></span> populations and the number of individuals (<em>i</em>) and molecular markers (<em>m</em>) used in one PSI selection cycle (cycle 0) and in three GSI selection cycles (cycles 0, 1, and 2)</span><div class="sb-div caption-clear"></div></div></div><p id="p-27">In our study, the PSI was applied only in C0 because there were no phenotypic data after that cycle, whereas GSI was applied in C0, C1, and C2. Note that GSI was used in C0 only with the purpose of comparing GSI efficiency <em>vs.</em> PSI efficiency. The top 10% (<em>k</em> = 1.75) was selected in all cycles of the four data sets.</p><p id="p-28">We analyzed the simulated and real data results for all traits in each selection cycle, by using three different criteria: the estimated GSI and PSI selection responses, the estimated expected genetic gain per selection cycle for each trait in the PSI and in the GSI, and the estimated <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Technow <em>et al.</em> (2013)</a> inequality (see <a href="http://www.g3journal.org/content/suppl/2015/08/18/g3.115.019869.DC1/019869SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/content/suppl/2015/08/18/g3.115.019869.DC1/FileS1.pdf">File S1</a> for the last criteria).</p></div><div id="sec-13" class="subsection"><h4>Data repository:</h4><p id="p-29">The simulated phenotypic selection cycles (C0−C7) for PSI (data set 1), and GS cycles (C1−C7) for GSI (data set 2), as well as the real data sets (data sets 3−6) including the phenotype and haplotype data are deposited at <a href="http://hdl.handle.net/11529/10199">http://hdl.handle.net/11529/10199</a>. This repository also has the <a href="http://www.g3journal.org/content/suppl/2015/08/18/g3.115.019869.DC1/FileS1.pdf">File S1</a> cited several times in the text of the paper.</p></div></div><div id="sec-14" class="subsection"><h3>Data availability</h3><p id="p-30">The data repository has the following data: <em>Real_Data_Sets_GSI, Simulated_Data_GSI,</em> and a manuscript<em>: Supplementary Material-2.doc</em> that are described below</p><p id="p-31">File <em>Real_Data_Sets_GSI</em> contains four file data sets: DATA_SET-3, 4, 5 and 6. In addition, each four file data sets contains four excel data. For example, the four excel data for file DATA_SET-3 are: DATA_SET-3_Markers_Cycle-0, 1, 2, and DATA_SET-3_Phenotypic_Cycle-0. The first three excel data contains the marker coded values for cycles 0, 1 and 2, while the excel data DATA_SET-3_Phenotypic_Cycle-0 contains the phenotypic information of cycle 0 (Training population). These four data sets were used to make selection, to estimate the selection response and the genetic expected gains; the results were presented in <a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>.</p><p id="p-32">The other three file data sets: DATA_SET-4, 5 and 6 contains similar information that file DATA_SET-3, but this information correspond to data set 4, 5 and 6 used to make selection, to estimate the selection response and the genetic expected gains; the results were presented in <a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a>.</p><p id="p-33">File <em>Simulated_Data_GSI</em> contains two files: <em>Data_Phenotypes_April-26-15</em> and <em>Haplotypes_GSI_April-26-15</em>. File <em>Data_Phenotypes_April-26-15</em> contains two files: GSI_Phenotypes-05 and PSI_Phenotypes-05. File GSI_Phenotypes-05 contains six excel data sets denoted as C2_GSI_05_Pheno, C3_GSI_05_Pheno, C4_GSI_05_Pheno, C5_GSI_05_Pheno and C6_GSI_05_Pheno, corresponding to the phenotypic simulated information for genomic selection index for cycle 2-7, meanwhile GSI_Phenotypes-05 contains eight excel data sets denoted as <em>C0_Pheno_05, C1_PSI_05_Pheno, C2_PSI_05_Pheno, C3_PSI_05_Pheno, C4_PSI_05_Pheno, C5_PSI_05_Pheno, C6_PSI_05_Pheno, C7_PSI_05_Pheno</em> corresponding to the phenotypic simulated information for phenotypic selection index for cycle 0-7. File Haplotypes_GSI_April-26-15 contains the haplotypes of the markers for cycles 0-7 of GSI.</p><p id="p-34">Finally, the manuscript <em>Supplementary Material-2.doc</em> contain a complete description related with the form we adapted the Technow inequality to the genomic and phenotypic selection index to comparer its efficiency in terms of time. In addition, this manuscript contains <a href="http://www.g3journal.org/content/suppl/2015/08/18/g3.115.019869.DC1/TableS1.pdf">Table S1</a> and <a href="http://www.g3journal.org/content/suppl/2015/08/18/g3.115.019869.DC1/TableS2.pdf">Table S2</a>; the first one contains the results of the simulated data and the second one the results of real data.</p></div></div><div class="section results" id="sec-15"><h2 class="">Results</h2><div id="sec-16" class="subsection"><h3>Simulated data</h3><div id="sec-17" class="subsection"><h4>Correlations between GEBV and the trait true breeding values:</h4><p id="p-35"><a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> shows the correlation between the GEBV and the individual trait true breeding values obtained by the Pearson correlation coefficient. The genomic relationship (<strong>G</strong>) is not incorporated in the correlations. In <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>, each selection cycle contains four columns: the first column (from left to the right) corresponds to the correlation between the GEBV and the T1 true breeding values; the second column corresponds to the correlation between the GEBV and the T2 true breeding values; etc. In this figure, all correlation values tend to decrease after the first selection cycle. In C7, the correlation values between the GEBV and the trait true breeding values were 0.30, 0.21, 0.38, and 0.34, for each of the four traits, respectively, whereas in cycle one (C1) these correlations were 0.40, 0.53, 0.63, and 0.73, for each of the four traits, respectively. In terms of proportions, the correlation values of C7 were only 76%, 40%, 60%, and 46% of the correlation values of C1. That is, the correlation between the GEBV and the trait true breeding values decreased more for traits 2 and 4 than for traits 1 and 3. This can be explained by the number of QTL that affected each trait and the size of the QTL effects on the traits in each selection cycle.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/10/2155/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Correlation between the genomic estimated breeding values (GEBVs) and the true breeding values for four traits in seven selection cycles. For each cycle of selection, the four columns correspond to the correlation between the GEBV and the true breeding values for traits T1, T2, T3, and T4, respectively." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1728775229" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Correlation between the genomic estimated breeding values (GEBVs) and the true breeding values for four traits in seven selection cycles. For each cycle of selection, the four columns correspond to the correlation between the GEBV and the true breeding values for traits T1, T2, T3, and T4, respectively.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/10/2155/F2.medium.gif" width="440" height="242"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/5/10/2155/F2.medium.gif" width="440" height="242"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/10/2155/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/10/2155/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466764" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-36" class="first-child">Correlation between the genomic estimated breeding values (GEBVs) and the true breeding values for four traits in seven selection cycles. For each cycle of selection, the four columns correspond to the correlation between the GEBV and the true breeding values for traits T1, T2, T3, and T4, respectively.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-18" class="subsection"><h4>Correlations between the GSI and the true H values:</h4><p id="p-37"><a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> shows the correlation between the GSI and the true net genetic merit values (<span class="inline-formula" id="inline-formula-79"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-89.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-89.gif"/></noscript></span></span>), for the four simulated traits in seven selection cycles. This correlation is computed as <span class="inline-formula" id="inline-formula-80"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-90.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-90.gif"/></noscript></span></span>, where <strong>C</strong> is the covariance matrix of true breeding values (Equation (1)) and <span class="inline-formula" id="inline-formula-81"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-91.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-91.gif"/></noscript></span></span> was obtained according to Equation (7). In this case, <span class="inline-formula" id="inline-formula-82"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-92.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-92.gif"/></noscript></span></span> incorporated the genomic relationship (<strong>G</strong>) information.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/10/2155/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Correlation between the genomic selection index (GSI), the phenotypic selection index (PSI), and the true net genetic merit (H) values in seven selection cycles. For each cycle of selection, the first column corresponds to the correlation between the GSI estimated values and the H true values (blue), whereas the second column corresponds to the correlation between the PSI estimated values and the H true values (red)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1728775229" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Correlation between the genomic selection index (GSI), the phenotypic selection index (PSI), and the true net genetic merit (H) values in seven selection cycles. For each cycle of selection, the first column corresponds to the correlation between the GSI estimated values and the H true values (blue), whereas the second column corresponds to the correlation between the PSI estimated values and the H true values (red).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/10/2155/F3.medium.gif" width="440" height="237"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/5/10/2155/F3.medium.gif" width="440" height="237"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/10/2155/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/10/2155/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466757" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-38" class="first-child">Correlation between the genomic selection index (GSI), the phenotypic selection index (PSI), and the true net genetic merit (H) values in seven selection cycles. For each cycle of selection, the first column corresponds to the correlation between the GSI estimated values and the H true values (blue), whereas the second column corresponds to the correlation between the PSI estimated values and the H true values (red).</p><div class="sb-div caption-clear"></div></div></div><p id="p-39"><a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a> contains only two columns for each selection cycle: the first column (blue) corresponds to the correlation between the GSI and the true values of H, whereas the second column (red) denotes the correlation between PSI estimated values and H. As expected, the correlation between GSI and H tended to decrease more than the correlation between PSI and H after the third selection cycle. The reason was that the PSI estimated values in each selection cycle were obtained using all phenotypic information of the newly generated population, whereas the GSI estimated values in each selection cycle incorporated only the marker information of the newly generated population. The correlation between GSI and H was 0.71 in C7 and 0.95 in C1, whereas the correlation between PSI and H was 0.83 in C7 and 0.91 in C1.</p></div><div id="sec-19" class="subsection"><h4>Estimated and true selection response of PSI and GSI when their generation interval is ignored:</h4><p id="p-40">The first part of <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a> shows the GSI estimated (<span class="inline-formula" id="inline-formula-83"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-93.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-93.gif"/></noscript></span></span>) and true (<span class="inline-formula" id="inline-formula-84"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-94.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-94.gif"/></noscript></span></span>), and the PSI estimated (<span class="inline-formula" id="inline-formula-85"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-95.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-95.gif"/></noscript></span></span>) and true (<span class="inline-formula" id="inline-formula-86"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-96.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-96.gif"/></noscript></span></span>) selection responses and their ratios: <span class="inline-formula" id="inline-formula-87"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-97.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-97.gif"/></noscript></span></span>/<span class="inline-formula" id="inline-formula-88"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-98.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-98.gif"/></noscript></span></span>and <span class="inline-formula" id="inline-formula-89"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-99.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-99.gif"/></noscript></span></span>/<span class="inline-formula" id="inline-formula-90"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-100.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-100.gif"/></noscript></span></span>, when their generation interval was ignored, for simulated data sets 1 and 2, respectively, for four traits (T1, T2, T3, and T4) and seven GSI and PSI selection cycles. In all selection cycles, <span class="inline-formula" id="inline-formula-91"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-101.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-101.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-92"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-102.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-102.gif"/></noscript></span></span>. In addition, results indicated that, in general, <span class="inline-formula" id="inline-formula-93"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-103.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-103.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-94"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-104.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-104.gif"/></noscript></span></span>, <em>i.e.</em>, <span class="inline-formula" id="inline-formula-95"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-105.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-105.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-96"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-106.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-106.gif"/></noscript></span></span>, underestimated the <span class="inline-formula" id="inline-formula-97"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-107.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-107.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-98"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-108.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-108.gif"/></noscript></span></span> values in all selection cycles.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466909/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466909/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Estimated (<span class="inline-formula" id="inline-formula-99"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/T2/embed/mml-math-109.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/T2/embed/mml-math-109.gif"/></noscript></span></span>) and true (<span class="inline-formula" id="inline-formula-100"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/T2/embed/mml-math-110.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/T2/embed/mml-math-110.gif"/></noscript></span></span>) GSI responses; estimated (<span class="inline-formula" id="inline-formula-101"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/T2/embed/mml-math-111.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/T2/embed/mml-math-111.gif"/></noscript></span></span>) and true (<span class="inline-formula" id="inline-formula-102"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/T2/embed/mml-math-112.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/T2/embed/mml-math-112.gif"/></noscript></span></span>) PSI responses, and the ratios: <span class="inline-formula" id="inline-formula-103"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/T2/embed/mml-math-113.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/T2/embed/mml-math-113.gif"/></noscript></span></span>/<span class="inline-formula" id="inline-formula-104"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/T2/embed/mml-math-114.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/T2/embed/mml-math-114.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-105"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/T2/embed/mml-math-115.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/T2/embed/mml-math-115.gif"/></noscript></span></span>/<span class="inline-formula" id="inline-formula-106"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/T2/embed/mml-math-116.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/T2/embed/mml-math-116.gif"/></noscript></span></span>, when their generation intervals were ignored</span><div class="sb-div caption-clear"></div></div></div><p id="p-42">The average values for all selection cycles of ratios <span class="inline-formula" id="inline-formula-131"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-141.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-141.gif"/></noscript></span></span>/<span class="inline-formula" id="inline-formula-132"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-142.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-142.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-133"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-143.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-143.gif"/></noscript></span></span>/<span class="inline-formula" id="inline-formula-134"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-144.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-144.gif"/></noscript></span></span> were equal to 0.83 and 0.86, respectively, which indicated that <span class="inline-formula" id="inline-formula-135"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-145.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-145.gif"/></noscript></span></span> explained 83% of <span class="inline-formula" id="inline-formula-136"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-146.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-146.gif"/></noscript></span></span> performance, whereas <span class="inline-formula" id="inline-formula-137"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-147.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-147.gif"/></noscript></span></span> explained 86% of <span class="inline-formula" id="inline-formula-138"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-148.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-148.gif"/></noscript></span></span> performance. Then, in terms of mean values, the results indicated that <span class="inline-formula" id="inline-formula-139"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-149.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-149.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-140"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-150.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-150.gif"/></noscript></span></span> were good estimators of <span class="inline-formula" id="inline-formula-141"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-151.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-151.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-142"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-152.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-152.gif"/></noscript></span></span> performance, respectively. The main results here show that for each selection cycle, the estimated PSI response to selection was always higher than the true and estimated GSI response when the generation interval was not incorporated in the estimated selection response.</p></div><div id="sec-20" class="subsection"><h4>Estimated and true selection response of PSI and GSI when their generation interval is included:</h4><p id="p-43">The second part of <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a> shows the GSI estimated (<span class="inline-formula" id="inline-formula-143"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-153.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-153.gif"/></noscript></span></span>) and true (<span class="inline-formula" id="inline-formula-144"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-154.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-154.gif"/></noscript></span></span>), and the PSI estimated (<span class="inline-formula" id="inline-formula-145"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-155.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-155.gif"/></noscript></span></span>) and true (<span class="inline-formula" id="inline-formula-146"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-156.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-156.gif"/></noscript></span></span>) selection response values and their ratios: <span class="inline-formula" id="inline-formula-147"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-157.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-157.gif"/></noscript></span></span>/<span class="inline-formula" id="inline-formula-148"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-158.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-158.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-149"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-159.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-159.gif"/></noscript></span></span>/<span class="inline-formula" id="inline-formula-150"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-160.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-160.gif"/></noscript></span></span>, when their generation interval was included, for simulated data sets 1 and 2 for four traits (T1, T2, T3, and T4) for seven GSI and PSI selection cycles. In this case, the time required to complete one GSI selection cycle was <span class="inline-formula" id="inline-formula-151"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-161.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-161.gif"/></noscript></span></span> years, whereas for one PSI selection cycle it was <span class="inline-formula" id="inline-formula-152"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-162.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-162.gif"/></noscript></span></span> years. According to the ratio values <span class="inline-formula" id="inline-formula-153"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-163.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-163.gif"/></noscript></span></span>/<span class="inline-formula" id="inline-formula-154"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-164.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-164.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-155"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-165.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-165.gif"/></noscript></span></span>/<span class="inline-formula" id="inline-formula-156"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-166.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-166.gif"/></noscript></span></span>, in all selection cycles GSI was more than twice as efficient as PSI. Then, when the generation interval of the estimated GSI and PSI selection response was included in the estimate response to selection, GSI was more efficient than the PSI in all selection cycles for the estimated and true selection responses.</p></div><div id="sec-21" class="subsection"><h4>Expected genetic gains for PSI and GSI in simulated data with and without generation interval:</h4><p id="p-44">Columns 2−9 (from left to right) in <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a> show the PSI estimated expected genetic gains for each trait per selection cycle for the observed traits (Equation 9) and columns 10−17 show the estimated GSI expected genetic gain for each trait per selection cycle for the unobserved traits (Equation 10). Note that the PSI estimated expected genetic gains of columns 2−5 were not divided by 4 (the time required to collect information to evaluate PSI and complete one selection cycle). Similarly, the GSI estimated expected genetic gains of columns 9−13 were not divided by 1.5 (the time required to collect information to evaluate GSI and complete one selection cycle). When the generation interval was not considered, the expected value of PSI for traits in each cycle was always higher than the expected values of the GSI for those traits. However, per unit of time, the expected genetic gains of GSI (columns 14–17 of <a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>) for each cycle and for each trait were always higher than the expected genetic gains of PSI (columns 6−9 of <a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>).</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466846/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466846/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Estimated expected genetic gains obtained using the PSI and the GSI for simulated data sets 1 and 2, respectively, for four traits (T1, T2, T3, and T4), when the generation interval is ignored and when it is included<sup><em><sup>a</sup></em></sup></span><div class="sb-div caption-clear"></div></div></div></div></div><div id="sec-22" class="subsection"><h3>Real data (F2 maize populations)</h3><div id="sec-23" class="subsection"><h4>Estimated expected genetic gains and selection responses for PSI and GSI with generation interval included:</h4><p id="p-47"><a id="xref-table-wrap-4-3" class="xref-table" href="#T4">Table 4</a> shows the estimated PSI and GSI expected genetic gains, and the estimated PSI and GSI selection responses (Equations 2 and 4, respectively) for one PSI selection cycle (C0) and three GSI cycles (C0, C1, and C2) of four maize (<em>Zea mays</em> L.) <span class="inline-formula" id="inline-formula-157"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-167.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-167.gif"/></noscript></span></span> populations and four traits (GY, PHT, EHT, and AD), when their generation interval was included. We aimed to increase GY while decreasing PHT, EHT, and AD by using three sets of markers (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). As for the simulated data, for the PSI and GSI response, the time required to complete one selection cycle was <span class="inline-formula" id="inline-formula-158"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-168.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-168.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-159"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-169.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-169.gif"/></noscript></span></span> years for GSI and PSI, respectively.</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466723/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466723/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Expected genetic gains per selection cycle for the PSI and GSI for cycle 0 and cycles 0, 1, and 2, respectively, for four traits (GY, EHT, PHT, and AD) in four maize (<em>Zea mays</em>) <span class="inline-formula" id="inline-formula-160"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/T4/embed/mml-math-170.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/T4/embed/mml-math-170.gif"/></noscript></span></span> populations when the generation interval was included</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-24" class="subsection"><h4>Estimated expected genetic gains for PSI and GSI:</h4><p id="p-49">In this case, the GSI estimated expected genetic gain for each trait per selection cycle for the unobserved traits in C0 (or training population) were greater than the PSI estimated expected genetic gains for each trait per selection cycle for the observed traits. These results showed a similar tendency to the simulated results when the generation interval was included. That is, in C0, the estimated GSI expected genetic gains were greater than the estimated PSI expected genetic gains. In C1 and C2, it was not possible to compare GSI <em>vs.</em> PSI because there were no phenotypic data in those cycles.</p></div><div id="sec-25" class="subsection"><h4>Estimated PSI and GSI selection response:</h4><p id="p-50">The numbers of individuals and markers used in the four real data sets were lower (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>) than those used in the simulated data; for this reason, the estimated selection values observed in the real data sets (<a id="xref-table-wrap-4-4" class="xref-table" href="#T4">Table 4</a>) were lower than those in the simulated results shown in <a id="xref-table-wrap-4-5" class="xref-table" href="#T4">Table 4</a>. However, in general, the decrease in estimated GSI responses after C0 was similar to the decrease in estimated GSI selection responses after C1 in the simulated data (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>). For the real data sets, in C0, the estimated GSI selection response was higher than the estimated PSI selection response, whereas in C1 and C2, it was not possible to compare GSI <em>vs.</em> PSI because there were no phenotypic data in those cycles.</p></div></div><div id="sec-26" class="subsection"><h3>Additional criteria for comparing PSI <em>vs.</em> GSI</h3><p id="p-51">Besides Equations (8), (9), and (10), we used the <a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">Technow <em>et al.</em> (2013)</a> inequality adapted to the context of PSI and GSI (Supplemental Materials, Equation (S1)) as additional criteria to compare the efficiency of GSI <em>vs.</em> PSI in terms of time. This last criterion corroborated the results obtained with Equations (8), (9), and (10). Results of the last criterion are given in <a href="http://www.g3journal.org/content/suppl/2015/08/18/g3.115.019869.DC1/TableS1.pdf">Table S1</a> and <a href="http://www.g3journal.org/content/suppl/2015/08/18/g3.115.019869.DC1/TableS2.pdf">Table S2</a> for simulated and real data, respectively.</p></div></div><div class="section discussion" id="sec-27"><h2 class="">Discussion</h2><div id="sec-28" class="subsection"><h3>Simulated data</h3><p id="p-52">Our results showed that GSI is more efficient than PSI per unit of time but not in terms of cycle. The average of the PSI and GSI selection responses values for all cycles, and the average of the PSI and GSI expected genetic gains per selection cycle for all cycles for observed and unobserved traits, respectively, were very similar when their generation interval was ignored because in the simulation process 95% of the QTL were in linkage disequilibrium with markers. After C3, the correlation between true and estimated PSI and GSI values was greater for PSI than for GSI. In our simulation, if instead of using 95% of the QTL in linkage disequilibrium, we had used 100% of the QTL in linkage disequilibrium with markers, we would expect the PSI and GSI results to be practically equal under the assumption of a very large number of markers. The importance of this result is that when the generation interval was ignored, PSI efficiency &gt; GSI efficiency, but on average across all cycles, they were similar. When the interval length was used in the PSI and GSI selection responses and in the PSI and GSI expected genetic gain per selection cycle, GSI was always more efficient than PSI in maize population selection for relatively dense molecular markers in an <em>F</em><sub>2</sub> population.</p><p id="p-53">We compared the PSI response with the GSI response considering the time (years) needed for each method to complete a selection cycle assuming that selection intensity is the same in both selection indices. Then, the ratio of the GSI selection response over the PSI selection response (Equation 8) was a good criterion for comparing PSI efficiency <em>vs.</em> GSI efficiency because each selection response included all the information on the genetic gains for each selection index in each selection cycle. In the case of the maize populations, GSI led to greater rates of genetic gain/year than PSI because PSI requires about 4 years to complete each selection cycle, whereas GSI requires about 1.5 years (<a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">Beyene <em>et al.</em> 2015</a>). Thus GSI efficiency was greater than PSI efficiency because the interval of time between selection cycles in GSI is shorter than in PSI. If this factor is not taken into account, the average PSI response for the simulated data were 14% greater than the average GSI response.</p></div><div id="sec-29" class="subsection"><h3>Real data</h3><p id="p-54">In the real data sets, the trend of GSI responses was very similar to those observed in the simulated data when their generation interval was not ignored. That is, GSI responses were higher than PSI responses in C0 for all four data sets (<a id="xref-table-wrap-4-6" class="xref-table" href="#T4">Table 4</a>). One reason for these results may be that markers were in linkage disequilibrium with many QTL of the trait. In that case, GSI was very effective. As shown by <a id="xref-ref-2-4" class="xref-bibr" href="#ref-2">Beyene <em>et al.</em> (2015)</a>, in eight biparental populations, a good genetic gain is expected from rapid cycling of GS in an <em>F</em><sub>2</sub> population with maximum linkage disequilibrium. Note that the estimated selection response for GSI decreased in a manner similar to that of the simulated data after cycle 0. This is because in the real data sets, the estimated selection response depends on the additive genomic variance-covariance matrix (<strong><em>Γ</em></strong>), whose covariance components decreased in each selection cycle.</p></div><div id="sec-30" class="subsection"><h3>The importance of the estimation of matrix <strong><em>Γ</em></strong> in simulated and real data and its effect on GSI correlations, GSI response, and GSI expected genetic gains</h3><p id="p-55">We proposed one way of estimating matrix <span class="inline-formula" id="inline-formula-161"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-171.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-171.gif"/></noscript></span></span> (Equation 7). This method significantly affected (1) the correlation between GSI and the net genetic merit (<span class="inline-formula" id="inline-formula-162"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-172.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-172.gif"/></noscript></span></span>), (2) the estimated GSI response, and (3) the estimated GSI expected genetic gains. The elements of <strong><em>Γ</em></strong> were estimated as <span class="inline-formula" id="inline-formula-163"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-173.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-173.gif"/></noscript></span></span>. Another form of <span class="inline-formula" id="inline-formula-164"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-174.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-174.gif"/></noscript></span></span> estimate is <span class="inline-formula" id="inline-formula-165"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-175.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-175.gif"/></noscript></span></span>, where matrix <span class="inline-formula" id="inline-formula-166"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-176.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-176.gif"/></noscript></span></span> is omitted. In that case, the correlation between the GSI and H would tend to be smaller (data not shown) than that shown in <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>. In addition, we could also expect that the estimated GSI selection responses and the estimated GSI expected genetic gains per selection cycle would be smaller than those shown in <a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2</a>, <a id="xref-table-wrap-3-4" class="xref-table" href="#T3">Table 3</a>, and <a id="xref-table-wrap-4-7" class="xref-table" href="#T4">Table 4</a>.</p><p id="p-56">These results indicate the importance of matrix <span class="inline-formula" id="inline-formula-167"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-177.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-177.gif"/></noscript></span></span> in the estimation of the GSI response, the GSI expected genetic gains per selection cycle, and in the correlation between GSI and H because the use of the genomic relationship matrix increases the accuracy of parameter estimation.</p></div><div id="sec-31" class="subsection"><h3>PSI <em>vs.</em> GSI</h3><p id="p-57">PSI and GSI are predictors of H and both have optimal statistical properties. However, while PSI is a phenotypic predictor of H, GSI is a genomic predictor of H. Based on trait hereditability and genetic architecture, PSI is expected to be more accurate and have a greater selection response per selection cycle than GSI. However, in terms of genetic gain per unit of time, GSI needs one-third of the time required by PSI or less (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Lorenz <em>et al.</em> 2011</a>). Thus, GSI will be more efficient than PSI in GS programs. We have shown (in <a href="http://www.g3journal.org/content/suppl/2015/08/18/g3.115.019869.DC1/FileS1.pdf">File S1</a>) that the Technow inequality corroborated this last argument.</p><p id="p-58">In simulation and empirical studies, GEBVs based solely on individual genotypes have been remarkably accurate. These accuracies depend on the characteristics of the population under selection (<a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">Lorenz <em>et al.</em> 2011</a>). According to Equations (3) and (6), GSI is a linear combination of indices because GEBVs are indices <em>per se</em> (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Robinson 1991</a>; <a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">Togashi <em>et al.</em> 2011</a>) and its main function is to predict the net genetic merit (<span class="inline-formula" id="inline-formula-168"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-178.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-178.gif"/></noscript></span></span>) of the candidate for selection. According to classical best linear unbiased predictor theory (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">McLean <em>et al.</em> 1991</a>; <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Robinson 1991</a>): (a) GSI is the best linear predictor of H; (b) the correlation between GSI and H is maximum; (c) the GEBVs are unique; and (d) <span class="inline-formula" id="inline-formula-169"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-179.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-179.gif"/></noscript></span></span>, <em>i.e.</em>, the expectation of H given GSI<sub>E</sub> is GSI<sub>E</sub>. PSI was constructed with trait phenotypic means to predict and select H; however, <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Henderson (1963)</a> showed that all four points are also true for PSI when matrices <strong>P</strong> and <strong>C</strong> are known.</p><p id="p-59">For the selection objective, GSI requires only the genomic best linear unbiased predictor obtained in the training population (in this case, C0) and the population markers of each selection cycle that are used to obtain the GEBV in each selection cycle. Then, for selection proposes, we only need to construct the estimated GSI as <span class="inline-formula" id="inline-formula-170"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-180.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-180.gif"/></noscript></span></span> (Equation 6) and the <span class="inline-formula" id="inline-formula-171"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-181.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-181.gif"/></noscript></span></span> values are then ranked to select individual genotypes with optimum GEBV values. However, in the present paper, we used the PSI theory originally developed by <a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">Smith (1936)</a> to obtain the GSI selection response and the GSI expected genetic gains per selection cycle. Selection response and expected genetic gains give information on the genetic gains in the next selection cycle and are the base criteria for comparing the efficiency of two or more selection indices (<a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">Bulmer 1980</a>; <a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">Moreau <em>et al.</em> 1998</a>).</p></div><div id="sec-32" class="subsection"><h3>PSI <em>vs.</em> GSI when the generation interval is equal in both indices</h3><p id="p-60">Some of the results shown in <a id="xref-table-wrap-2-5" class="xref-table" href="#T2">Tables 2</a> and <a id="xref-table-wrap-3-5" class="xref-table" href="#T3">Table 3</a> and <a href="http://www.g3journal.org/content/suppl/2015/08/18/g3.115.019869.DC1/TableS1.pdf">Table S1</a> and <a href="http://www.g3journal.org/content/suppl/2015/08/18/g3.115.019869.DC1/TableS2.pdf">Table S2</a> occurred when the PSI generation interval (<em>L<sub>PSI</sub></em>) was greater than the GSI generation interval (<em>L<sub>PSI</sub></em>). What would happen if <em>L<sub>PSI</sub></em> = <em>L<sub>GSI</sub></em>? In this case, if the number of markers is very small, then Equation (4) will give lower values than Equation (2) and PSI efficiency will be greater than GSI efficiency. However, if the number of markers is very large, the PSI and GSI responses will be very similar.</p><p id="p-61">This argument also holds true for the <a id="xref-ref-26-4" class="xref-bibr" href="#ref-26">Technow <em>et al.</em> (2013)</a> inequality and the PSI and GSI expected genetic gain per selection cycle for observed and unobserved traits. For example, note that in <a href="http://www.g3journal.org/content/suppl/2015/08/18/g3.115.019869.DC1/TableS1.pdf">Table S1</a>, we have assumed that <span class="inline-formula" id="inline-formula-172"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-182.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-182.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-173"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-183.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-183.gif"/></noscript></span></span>. Suppose now that <span class="inline-formula" id="inline-formula-174"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-184.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-184.gif"/></noscript></span></span>. In this case, the <a id="xref-ref-26-5" class="xref-bibr" href="#ref-26">Technow <em>et al.</em> (2013)</a> inequality will not hold true because in all selection cycles <span class="inline-formula" id="inline-formula-175"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-185.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-185.gif"/></noscript></span></span>. That is, the <a id="xref-ref-26-6" class="xref-bibr" href="#ref-26">Technow <em>et al.</em> (2013)</a> inequality will change its direction. Finally, it is evident that if <span class="inline-formula" id="inline-formula-176"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-186.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-186.gif"/></noscript></span></span>, PSI will be more efficient than GSI even in the hypothetical case when the number of molecular marker is infinite. In conclusion, GSI will be more efficient than PSI in terms of unit of time only if <span class="inline-formula" id="inline-formula-177"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-187.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-187.gif"/></noscript></span></span>; in this case, the <a id="xref-ref-26-7" class="xref-bibr" href="#ref-26">Technow <em>et al.</em> (2013)</a> inequality is true. In all other cases, PSI will be more efficient than GSI.</p><p id="p-62">In this study, we applied the theory of GSI to simulated and real data and compared its efficiency with PSI efficiency by using three different criteria: the ratio of the GSI response over the PSI response, the PSI and GSI expected genetic gain per selection cycle for observed and unobserved traits, respectively, and the Technow inequality. In all three cases, for simulated and real data, GSI efficiency was higher than PSI efficiency per unit of time in all selection cycles. We thus concluded that GSI is an efficient choice when the purpose of a breeding program is to select individuals using GS.</p></div></div><div class="section app" id="app-1"><h2 class="">Appendix</h2><div class="section" id="sec-33"><h3>Multivariate prediction of molecular marker effects</h3><p id="p-63">In the univariate context, <a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">VanRaden (2008)</a> showed that marker effects in the training population could be estimated as<span class="disp-formula" id="disp-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-188.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-188.gif"/></noscript></span> <span class="disp-formula-label">(A1)</span></span>where <span class="inline-formula" id="inline-formula-178"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-189.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-189.gif"/></noscript></span></span> is the additive genomic relationship matrix, <strong>X</strong> is a matrix with coded marker values on the base population (<em>e.g.</em>, 1, 0, and −1 for genotypes <em>AA</em>, <em>Aa</em>, and <em>aa</em>, respectively); <em>p<sub>j</sub></em> is the allelic frequency of <em>a</em>; <span class="inline-formula" id="inline-formula-179"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-190.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-190.gif"/></noscript></span></span> is a proportional constant (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Habier <em>et al.</em> 2007</a>); <span class="inline-formula" id="inline-formula-180"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-191.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-191.gif"/></noscript></span></span>; <span class="inline-formula" id="inline-formula-181"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-192.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-192.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-182"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-193.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-193.gif"/></noscript></span></span> are the additive and residual variances, respectively, associated with the <em>q</em><sup>th</sup> trait; <span class="inline-formula" id="inline-formula-183"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-194.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-194.gif"/></noscript></span></span> ∼ NMV(<strong>0</strong>, <span class="inline-formula" id="inline-formula-184"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-195.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-195.gif"/></noscript></span></span>) is a vector of observations, where NMV stands for the multivariate normal distribution, <span class="inline-formula" id="inline-formula-185"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-196.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-196.gif"/></noscript></span></span>; and <span class="inline-formula" id="inline-formula-186"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-197.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-197.gif"/></noscript></span></span> is an identity matrix of order <span class="inline-formula" id="inline-formula-187"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-198.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-198.gif"/></noscript></span></span>.</p><p id="p-64">In the multivariate context, to estimate the vector <span class="inline-formula" id="inline-formula-188"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-199.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-199.gif"/></noscript></span></span>, Equation (A1) can be written as<span class="disp-formula" id="disp-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-200.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-200.gif"/></noscript></span> <span class="disp-formula-label">(A2)</span></span>where <span class="inline-formula" id="inline-formula-189"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-201.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-201.gif"/></noscript></span></span>, “⊗” denotes the direct product, <strong>X</strong> was defined in Equation (A1); <span class="inline-formula" id="inline-formula-190"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-202.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-202.gif"/></noscript></span></span>, <strong>R</strong> is the residual covariance matrix, and <strong>C</strong> was defined in the text (Equation 1); <span class="inline-formula" id="inline-formula-191"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-203.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-203.gif"/></noscript></span></span> ∼ NMV(<strong>0</strong>, <strong><em>V</em></strong>) is a vector of size <span class="inline-formula" id="inline-formula-192"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-204.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-204.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-193"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-205.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-205.gif"/></noscript></span></span>; <span class="inline-formula" id="inline-formula-194"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-206.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-206.gif"/></noscript></span></span> is an identity matrix of size <span class="inline-formula" id="inline-formula-195"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-207.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-207.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-196"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-208.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-208.gif"/></noscript></span></span> was defined in Equation (A1). In this case, the estimator of the vector <span class="inline-formula" id="inline-formula-197"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-209.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-209.gif"/></noscript></span></span> is <span class="inline-formula" id="inline-formula-198"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-210.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2155/embed/mml-math-210.gif"/></noscript></span></span>.</p></div></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-supplementary-material" id="fn-6"><p id="p-65">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019869/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019869/-/DC1</a></p></li><li class="fn" id="fn-7"><p id="p-66"><em>Communicating editor: D. J. de Koning</em> </p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2015-06-19"><span class="received-label">Received </span>June 19, 2015.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2015-08-16"><span class="accepted-label">Accepted </span>August 16, 2015.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2015 Ceron-Rojas <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2155.1" data-doi="10.2135/cropsci2006.11.0690"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Prospects for genome-wide selection for quantitative traits in maize.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">1082</span>–<span class="cit-lpage">1090</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DBernardo%26rft.auinit1%253DR.%26rft.volume%253D47%26rft.issue%253D3%26rft.spage%253D1082%26rft.epage%253D1090%26rft.atitle%253DProspects%2Bfor%2BGenomewide%2BSelection%2Bfor%2BQuantitative%2BTraits%2Bin%2BMaize%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2006.11.0690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2006.11.0690&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000247424200020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2155.2" data-doi="10.2135/cropsci2014.07.0460"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beyene</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Semagn</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mugo</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tarekegne</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Babu</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genetic gains in grain yield through genomic selection in eight bi-parental maize populations under drought stress.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">154</span>–<span class="cit-lpage">163</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D55%26rft.spage%253D154%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2014.07.0460%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2014.07.0460&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-book" id="cit-5.10.2155.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bulmer</span>  <span class="cit-name-given-names">M. G.</span></span></li></ol><cite>, <span class="cit-pub-date">1980</span> <span class="cit-source">The Mathematical Theory of Quantitative Genetics. Lectures in Biomathematics</span>. <span class="cit-publ-name">Clarendon Press</span>, <span class="cit-publ-loc">Oxford, United Kingdom</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2155.4" data-doi="10.1534/genetics.112.143313"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pong-Wong</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calus</span>  <span class="cit-name-given-names">P. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Whole-genome regression and prediction methods applied to plant and animal breeding.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">193</span>: <span class="cit-fpage">327</span>–<span class="cit-lpage">345</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.143313%26rft_id%253Dinfo%253Apmid%252F22745228%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkzLzIvMzI3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMC8yMTU1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2155.5" data-doi="10.1111/j.1439-0388.2007.00701.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dekkers</span>  <span class="cit-name-given-names">J. C. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Prediction of response to marker-assisted and genomic selection using selection index theory.</span> <abbr class="cit-jnl-abbrev">J. Anim. Breed. Genet.</abbr> <span class="cit-vol">124</span>: <span class="cit-fpage">331</span>–<span class="cit-lpage">341</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BAnim%2BBreed%2BGenet%26rft.aulast%253DDekkers%26rft.auinit1%253DJ.%2BC.%26rft.volume%253D124%26rft.issue%253D6%26rft.spage%253D331%26rft.epage%253D341%26rft.atitle%253DPrediction%2Bof%2Bresponse%2Bto%2Bmarker-assisted%2Band%2Bgenomic%2Bselection%2Busing%2Bselection%2Bindex%2Btheory.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1439-0388.2007.00701.x%26rft_id%253Dinfo%253Apmid%252F18076470%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1439-0388.2007.00701.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18076470&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2155.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251432800003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2155.6" data-doi="10.1534/genetics.113.151753"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Priors in whole-genome regression: the Bayesian alphabet returns.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">194</span>: <span class="cit-fpage">573</span>–<span class="cit-lpage">596</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.151753%26rft_id%253Dinfo%253Apmid%252F23636739%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk0LzMvNTczIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMC8yMTU1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2155.7" data-doi="10.1007/s10709-008-9308-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genomic selection: prediction of accuracy and maximization of long-term response.</span> <abbr class="cit-jnl-abbrev">Genetica</abbr> <span class="cit-vol">136</span>: <span class="cit-fpage">245</span>–<span class="cit-lpage">257</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetica%26rft.volume%253D136%26rft.spage%253D245%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10709-008-9308-0%26rft_id%253Dinfo%253Apmid%252F18704696%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10709-008-9308-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18704696&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2155.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000265818900005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2155.8" data-doi="10.1534/genetics.107.081190"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Habier</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dekkers</span>  <span class="cit-name-given-names">J. C. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The impact of genetic relationship information on genome-assisted breeding values.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">177</span>: <span class="cit-fpage">2389</span>–<span class="cit-lpage">2397</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHabier%26rft.auinit1%253DD.%26rft.volume%253D177%26rft.issue%253D4%26rft.spage%253D2389%26rft.epage%253D2397%26rft.atitle%253DThe%2BImpact%2Bof%2BGenetic%2BRelationship%2BInformation%2Bon%2BGenome-Assisted%2BBreeding%2BValues%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.081190%26rft_id%253Dinfo%253Apmid%252F18073436%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Ny80LzIzODkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzEwLzIxNTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2155.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haldane</span>  <span class="cit-name-given-names">J. B. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1919</span> <span class="cit-article-title">The combination of linkage values and the calculation of distance between the loci of linked factors.</span> <abbr class="cit-jnl-abbrev">J. Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">299</span>–<span class="cit-lpage">309</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BGenet.%26rft.volume%253D8%26rft.spage%253D299%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=000200614300006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2155.10" data-doi="10.1017/S0016672308009981"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Visscher</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Increased accuracy of artificial selection by using the realized relationship matrix.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">47</span>–<span class="cit-lpage">60</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DHayes%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D91%26rft.issue%253D1%26rft.spage%253D47%26rft.epage%253D60%26rft.atitle%253DIncreased%2Baccuracy%2Bof%2Bartificial%2Bselection%2Bby%2Busing%2Bthe%2Brealized%2Brelationship%2Bmatrix.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672308009981%26rft_id%253Dinfo%253Apmid%252F19220931%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672308009981&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19220931&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2155.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000264226300006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2155.11" data-doi="10.2135/cropsci2008.08.0512"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heffner</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genomic selection for crop improvement.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">12</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DHeffner%26rft.auinit1%253DE.%2BL.%26rft.volume%253D49%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D12%26rft.atitle%253DGenomic%2BSelection%2Bfor%2BCrop%2BImprovement%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2008.08.0512%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2008.08.0512&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000263344300001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2155.12" data-doi="10.3835/plantgenome.2010.12.0029"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heffner</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genomic selection accuracy using multifamily prediction models in a wheat breeding program.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">65</span>–<span class="cit-lpage">75</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D4%26rft.spage%253D65%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome.2010.12.0029%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome.2010.12.0029&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-book" id="cit-5.10.2155.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Henderson</span>  <span class="cit-name-given-names">C. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1963</span> <span class="cit-article-title">Selection index and expected genetic advance</span>, pp. <span class="cit-fpage">141</span>–<span class="cit-lpage">163</span> in <span class="cit-source">Statistical Genetics and Plant Breeding</span>. <span class="cit-publ-name">National Academy of Science-National Research Council</span>, <span class="cit-publ-loc">Washington, DC</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2155.14" data-doi="10.2307/2527598"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kempthorne</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nordskog</span>  <span class="cit-name-given-names">A. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1959</span> <span class="cit-article-title">Restricted selection indices.</span> <abbr class="cit-jnl-abbrev">Biometrics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">10</span>–<span class="cit-lpage">19</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrics%26rft.volume%253D15%26rft.spage%253D10%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2527598%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2527598&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2155.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lande</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Efficiency of marker-assisted selection in the improvement of quantitative traits.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">124</span>: <span class="cit-fpage">743</span>–<span class="cit-lpage">756</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLande%26rft.auinit1%253DR.%26rft.volume%253D124%26rft.issue%253D3%26rft.spage%253D743%26rft.epage%253D756%26rft.atitle%253DEfficiency%2Bof%2Bmarker-assisted%2Bselection%2Bin%2Bthe%2Bimprovement%2Bof%2Bquantitative%2Btraits.%26rft_id%253Dinfo%253Apmid%252F1968875%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTI0LzMvNzQzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMC8yMTU1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2155.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">C. Y.</span></span></li></ol><cite>, <span class="cit-pub-date">1978</span> <span class="cit-article-title">Index selection for genetic improvement of quantitative characters.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">49</span>–<span class="cit-lpage">56</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D52%26rft.spage%253D49%26rft_id%253Dinfo%253Apmid%252F24317409%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=24317409&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2155.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1978EY47400001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2155.17" data-doi="10.1016/B978-0-12-385531-2.00002-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chao</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Asoro</span>  <span class="cit-name-given-names">F. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heffner</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayashi</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genomic selection in plant breeding: knowledge and prospects.</span> <abbr class="cit-jnl-abbrev">Adv. Agron.</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">77</span>–<span class="cit-lpage">123</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAdv.%2BAgron.%26rft.volume%253D110%26rft.spage%253D77%26rft_id%253Dinfo%253Adoi%252F10.1016%252FB978-0-12-385531-2.00002-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/B978-0-12-385531-2.00002-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2155.18" data-doi="10.2307/2685241"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McLean</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sander</span>  <span class="cit-name-given-names">W. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stroup</span>  <span class="cit-name-given-names">W. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">A unified approach to mixed linear models.</span> <abbr class="cit-jnl-abbrev">Am. Stat.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">54</span>–<span class="cit-lpage">64</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BStat.%26rft.volume%253D45%26rft.spage%253D54%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2685241%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2685241&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1991EU94800012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2155.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Prediction of total genetic value using genome-wide dense marker maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">1819</span>–<span class="cit-lpage">1829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMeuwissen%26rft.auinit1%253DT.%2BH.%2BE.%26rft.volume%253D157%26rft.issue%253D4%26rft.spage%253D1819%26rft.epage%253D1829%26rft.atitle%253DPrediction%2Bof%2BTotal%2BGenetic%2BValue%2BUsing%2BGenome-Wide%2BDense%2BMarker%2BMaps%26rft_id%253Dinfo%253Apmid%252F11290733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ny80LzE4MTkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzEwLzIxNTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2155.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moreau</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charcosset</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hospital</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gallais</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Marker-assisted selection efficiency in populations of finite size.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">148</span>: <span class="cit-fpage">1353</span>–<span class="cit-lpage">1365</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMoreau%26rft.auinit1%253DL.%26rft.volume%253D148%26rft.issue%253D3%26rft.spage%253D1353%26rft.epage%253D1365%26rft.atitle%253DMarker-Assisted%2BSelection%2BEfficiency%2Bin%2BPopulations%2Bof%2BFinite%2BSize%26rft_id%253Dinfo%253Apmid%252F9539448%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE0OC8zLzEzNTMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzEwLzIxNTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2155.21" data-doi="10.1093/aob/mcs109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nakaya</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Isobe</span>  <span class="cit-name-given-names">S. N.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Will genomic selection be a practical method for plant breeding?</span> <abbr class="cit-jnl-abbrev">Ann. Bot. (Lond.)</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">1303</span>–<span class="cit-lpage">1316</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BBot.%2B%2528Lond.%2529%26rft_id%253Dinfo%253Adoi%252F10.1093%252Faob%252Fmcs109%26rft_id%253Dinfo%253Apmid%252F22645117%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiYW5uYm90IjtzOjU6InJlc2lkIjtzOjEwOiIxMTAvNi8xMzAzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMC8yMTU1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2155.22" data-doi="10.1093/biomet/58.3.545"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Patterson</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1971</span> <span class="cit-article-title">Recovery of inter-block information when block sizes are unequal.</span> <abbr class="cit-jnl-abbrev">Biometrika</abbr> <span class="cit-vol">58</span>: <span class="cit-fpage">545</span>–<span class="cit-lpage">554</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrika%26rft.stitle%253DBiometrika%26rft.aulast%253DPATTERSON%26rft.auinit1%253DH.%2BD.%26rft.volume%253D58%26rft.issue%253D3%26rft.spage%253D545%26rft.epage%253D554%26rft.atitle%253DRecovery%2Bof%2Binter-block%2Binformation%2Bwhen%2Bblock%2Bsizes%2Bare%2Bunequal%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbiomet%252F58.3.545%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiYmlvbWV0IjtzOjU6InJlc2lkIjtzOjg6IjU4LzMvNTQ1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMC8yMTU1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2155.23" data-doi="10.1093/bioinformatics/14.7.632"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Podlich</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cooper</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">QU-GENE: a simulation platform for quantitative analysis of genetic model.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">632</span>–<span class="cit-lpage">653</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DPodlich%26rft.auinit1%253DD.%2BW.%26rft.volume%253D14%26rft.issue%253D7%26rft.spage%253D632%26rft.epage%253D653%26rft.atitle%253DQU-GENE%253A%2Ba%2Bsimulation%2Bplatform%2Bfor%2Bquantitative%2Banalysis%2Bof%2Bgenetic%2Bmodels%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252F14.7.632%26rft_id%253Dinfo%253Apmid%252F9730929%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIxNC83LzYzMiI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTAvMjE1NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2155.24" data-doi="10.1214/ss/1177011926"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">G. K.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">That BLUP is a good thing: the estimation of random effects.</span> <abbr class="cit-jnl-abbrev">Stat. Sci.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">15</span>–<span class="cit-lpage">51</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStat.%2BSci.%26rft.volume%253D6%26rft.spage%253D15%26rft_id%253Dinfo%253Adoi%252F10.1214%252Fss%252F1177011926%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1214/ss/1177011926&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-book" id="cit-5.10.2155.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">H. F.</span></span></li></ol><cite>, <span class="cit-pub-date">1936</span> <span class="cit-article-title">A discriminant function for plant selection</span>, pp. <span class="cit-fpage">466</span>–<span class="cit-lpage">476</span> in <span class="cit-source">Papers on Quantitative Genetics and Related Topics</span>. <span class="cit-publ-name">Department of Genetics, North Carolina State College</span>, <span class="cit-publ-loc">Raleigh, North Carolina</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-other" id="cit-5.10.2155.26" data-doi="10.1534/g3.112.004630"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Technow</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bürger</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melchinger</span>  <span class="cit-name-given-names">A.E.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span>  <span class="cit-article-title">Genomic prediction of northern corn leaf blight resistance in maize with combined or separated training sets for heterotic groups.</span> <span class="cit-source">G3 (Bethesda)</span> <span class="cit-vol">3</span>: <span class="cit-fpage">197</span>–<span class="cit-lpage">203</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2526%252358%253B%2BGenes%257CGenomes%257CGenetics%26rft.stitle%253DG3%26rft.aulast%253DTechnow%26rft.auinit1%253DF.%26rft.volume%253D3%26rft.issue%253D2%26rft.spage%253D197%26rft.epage%253D203%26rft.atitle%253DGenomic%2BPrediction%2Bof%2BNorthern%2BCorn%2BLeaf%2BBlight%2BResistance%2Bin%2BMaize%2Bwith%2BCombined%2Bor%2BSeparated%2BTraining%2BSets%2Bfor%2BHeterotic%2BGroups%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.112.004630%26rft_id%253Dinfo%253Apmid%252F23390596%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjMvMi8xOTciO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzEwLzIxNTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2155.27" data-doi="10.2527/jas.2009-2606"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Togashi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">C. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yamazaki</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The efficiency of genome-wide selection for genetic improvement of net merit.</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">89</span>: <span class="cit-fpage">2972</span>–<span class="cit-lpage">2980</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAnim.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.2527%252Fjas.2009-2606%26rft_id%253Dinfo%253Apmid%252F21512116%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2527/jas.2009-2606&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21512116&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2155.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2155.28" data-doi="10.3168/jds.2007-0980"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient methods to compute genomic predictions.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">4414</span>–<span class="cit-lpage">4423</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVanRaden%26rft.auinit1%253DP.%2BM.%26rft.volume%253D91%26rft.issue%253D11%26rft.spage%253D4414%26rft.epage%253D4423%26rft.atitle%253DEfficient%2BMethods%2Bto%2BCompute%2BGenomic%2BPredictions%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2007-0980%26rft_id%253Dinfo%253Apmid%252F18946147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2007-0980&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18946147&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2155.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260277200035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2155.29" data-doi="10.1186/1753-6561-4-S1-S5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Verbyla</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowman</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Sensitivity of genomic selection to using different prior distributions.</span> <abbr class="cit-jnl-abbrev">BMC Proc</abbr>. <span class="cit-vol">4</span>(<span class="cit-supplement">Supp. 1</span>)<strong>:</strong> <span class="cit-fpage">S5</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BProc%26rft.volume%253D4%26rft.spage%253DS5%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1753-6561-4-S1-S5%26rft_id%253Dinfo%253Apmid%252F20380759%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1753-6561-4-S1-S5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20380759&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2155.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2155.30" data-doi="10.2135/cropsci2003.1764"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Ginkel</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Podlich</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ye</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trethowan</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Comparison of two breeding strategies by computer simulation.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">1764</span>–<span class="cit-lpage">1773</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DWang%26rft.auinit1%253DJ.%26rft.volume%253D43%26rft.issue%253D5%26rft.spage%253D1764%26rft.epage%253D1773%26rft.atitle%253DComparison%2Bof%2BTwo%2BBreeding%2BStrategies%2Bby%2BComputer%2BSimulation%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2003.1764%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2003.1764&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000185174300026&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/5/10/2155.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/10/2137" title="SUMO-Enriched Proteome for Drosophila Innate Immune Response" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/10/2165" title="Similar Efficacies of Selection Shape Mitochondrial and Nuclear Genes in Both Drosophila melanogaster and Homo sapiens" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/5/10"><span>Volume 5 Issue 10, October 2015</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/5/10.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/5/10.cover-source.jpg?itok=JwgNhVLw" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 5 (10)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/10/2155.short" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-OpNU-x_j07q0LBdwPL40rqZ_nCC3Kw36r0L3bp0Kgng" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genomic-selection" class="" data-icon-position="" data-hide-link-title="0">Genomic Selection</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/5/10/2155" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/5/10/2155" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/5/10/2155" class="active" data-icon-position="" data-hide-link-title="0">A Genomic Selection Index Applied to Simulated and Real Data</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460152" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-7sr7SPP3fAPaK0GUHuasI1YAYpDEXqnnbL_5S5lWnIM" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/10/2155.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/5/10/2155" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-OVirpgLUQDv0QFA1kYt30TOS-hj71Uvf1xz4_9rRbic" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/5/10/2155" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/10/2155.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460152" id="node460152--2" data-pisa="ggg;5/10/2155" data-pisa-master="ggg;g3.115.019869" data-apath="/ggg/5/10/2155.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Genomic Selection Index Applied to Simulated and Real Data</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">J. Jesus Ceron-Rojas</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>José Crossa</span>, <span class="highwire-citation-author" data-delta="2">Vivi N. Arief</span>, <span class="highwire-citation-author" data-delta="3">Kaye Basford</span>, <span class="highwire-citation-author" data-delta="4">Jessica Rutkoski</span>, <span class="highwire-citation-author" data-delta="5">Diego Jarquín</span>, <span class="highwire-citation-author" data-delta="6">Gregorio Alvarado</span>, <span class="highwire-citation-author" data-delta="7">Yoseph Beyene</span>, <span class="highwire-citation-author" data-delta="8">Kassa Semagn</span> and <span class="highwire-citation-author" data-delta="9">Ian DeLacy</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2155-2164;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.019869 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460152--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">J. Jesus Ceron-Rojas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Biometrics and Statistics Unit, International Maize and Wheat Improvement Center (CIMMYT), 06600, México Distrito Federal, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=J.%2BJesus%2BCeron-Rojas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ceron-Rojas%20JJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJ.%2BJesus%2BCeron-Rojas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">José Crossa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Biometrics and Statistics Unit, International Maize and Wheat Improvement Center (CIMMYT), 06600, México Distrito Federal, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jos%C3%A9%2BCrossa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crossa%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJos%25C3%25A9%2BCrossa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for José Crossa</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a></span></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Vivi N. Arief </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>The University of Queensland, School of Agriculture and Food Sciences, St Lucia, QLD 4072, Brisbane, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vivi%2BN.%2BArief%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Arief%20VN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVivi%2BN.%2BArief%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Kaye Basford </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>The University of Queensland, School of Agriculture and Food Sciences, St Lucia, QLD 4072, Brisbane, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kaye%2BBasford%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Basford%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKaye%2BBasford%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Jessica Rutkoski </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Biometrics and Statistics Unit, International Maize and Wheat Improvement Center (CIMMYT), 06600, México Distrito Federal, México</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>International Programs of the College of Agriculture and Life Sciences, Cornell University, Ithaca, New York 14853 </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jessica%2BRutkoski%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rutkoski%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJessica%2BRutkoski%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Diego Jarquín </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Agronomy and Horticulture, University of Nebraska, Lincoln, Nebraska 68583 </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Diego%2BJarqu%C3%ADn%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jarqu%C3%ADn%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADiego%2BJarqu%25C3%25ADn%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Gregorio Alvarado </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Biometrics and Statistics Unit, International Maize and Wheat Improvement Center (CIMMYT), 06600, México Distrito Federal, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gregorio%2BAlvarado%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Alvarado%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGregorio%2BAlvarado%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Yoseph Beyene </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Global Maize Program, CIMMYT, Village Market 00621, Nairobi, Kenya</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yoseph%2BBeyene%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Beyene%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYoseph%2BBeyene%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Kassa Semagn </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Global Maize Program, CIMMYT, Village Market 00621, Nairobi, Kenya</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kassa%2BSemagn%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Semagn%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKassa%2BSemagn%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Ian DeLacy </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>The University of Queensland, School of Agriculture and Food Sciences, St Lucia, QLD 4072, Brisbane, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ian%2BDeLacy%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=DeLacy%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIan%2BDeLacy%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/5/10/2155" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/5/10/2155&amp;title=A%20Genomic%20Selection%20Index%20Applied%20to%20Simulated%20and%20Real%20Data" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/5/10/2155&amp;title=A%20Genomic%20Selection%20Index%20Applied%20to%20Simulated%20and%20Real%20Data" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/5/10/2155&amp;title=A%20Genomic%20Selection%20Index%20Applied%20to%20Simulated%20and%20Real%20Data" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/5/10/2155&amp;text=A%20Genomic%20Selection%20Index%20Applied%20to%20Simulated%20and%20Real%20Data" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/5/10/2155&amp;title=A%20Genomic%20Selection%20Index%20Applied%20to%20Simulated%20and%20Real%20Data" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/5/10/2155&amp;t=A%20Genomic%20Selection%20Index%20Applied%20to%20Simulated%20and%20Real%20Data" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/5/10/2155&amp;title=A%20Genomic%20Selection%20Index%20Applied%20to%20Simulated%20and%20Real%20Data" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/5/10/2155&amp;title=A%20Genomic%20Selection%20Index%20Applied%20to%20Simulated%20and%20Real%20Data" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460152/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460152" id="node460152--3" data-pisa="ggg;5/10/2155" data-pisa-master="ggg;g3.115.019869" data-apath="/ggg/5/10/2155.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Genomic Selection Index Applied to Simulated and Real Data</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">J. Jesus Ceron-Rojas</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>José Crossa</span>, <span class="highwire-citation-author" data-delta="2">Vivi N. Arief</span>, <span class="highwire-citation-author" data-delta="3">Kaye Basford</span>, <span class="highwire-citation-author" data-delta="4">Jessica Rutkoski</span>, <span class="highwire-citation-author" data-delta="5">Diego Jarquín</span>, <span class="highwire-citation-author" data-delta="6">Gregorio Alvarado</span>, <span class="highwire-citation-author" data-delta="7">Yoseph Beyene</span>, <span class="highwire-citation-author" data-delta="8">Kassa Semagn</span> and <span class="highwire-citation-author" data-delta="9">Ian DeLacy</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2155-2164;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.019869 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460152--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">J. Jesus Ceron-Rojas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Biometrics and Statistics Unit, International Maize and Wheat Improvement Center (CIMMYT), 06600, México Distrito Federal, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=J.%2BJesus%2BCeron-Rojas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ceron-Rojas%20JJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJ.%2BJesus%2BCeron-Rojas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">José Crossa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Biometrics and Statistics Unit, International Maize and Wheat Improvement Center (CIMMYT), 06600, México Distrito Federal, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jos%C3%A9%2BCrossa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crossa%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJos%25C3%25A9%2BCrossa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for José Crossa</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a></span></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Vivi N. Arief </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>The University of Queensland, School of Agriculture and Food Sciences, St Lucia, QLD 4072, Brisbane, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vivi%2BN.%2BArief%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Arief%20VN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVivi%2BN.%2BArief%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Kaye Basford </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>The University of Queensland, School of Agriculture and Food Sciences, St Lucia, QLD 4072, Brisbane, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kaye%2BBasford%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Basford%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKaye%2BBasford%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Jessica Rutkoski </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Biometrics and Statistics Unit, International Maize and Wheat Improvement Center (CIMMYT), 06600, México Distrito Federal, México</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>International Programs of the College of Agriculture and Life Sciences, Cornell University, Ithaca, New York 14853 </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jessica%2BRutkoski%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rutkoski%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJessica%2BRutkoski%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Diego Jarquín </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Agronomy and Horticulture, University of Nebraska, Lincoln, Nebraska 68583 </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Diego%2BJarqu%C3%ADn%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jarqu%C3%ADn%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADiego%2BJarqu%25C3%25ADn%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Gregorio Alvarado </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Biometrics and Statistics Unit, International Maize and Wheat Improvement Center (CIMMYT), 06600, México Distrito Federal, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gregorio%2BAlvarado%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Alvarado%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGregorio%2BAlvarado%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Yoseph Beyene </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Global Maize Program, CIMMYT, Village Market 00621, Nairobi, Kenya</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yoseph%2BBeyene%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Beyene%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYoseph%2BBeyene%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Kassa Semagn </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Global Maize Program, CIMMYT, Village Market 00621, Nairobi, Kenya</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kassa%2BSemagn%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Semagn%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKassa%2BSemagn%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Ian DeLacy </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>The University of Queensland, School of Agriculture and Food Sciences, St Lucia, QLD 4072, Brisbane, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ian%2BDeLacy%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=DeLacy%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIan%2BDeLacy%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2155&amp;rft.epage=2164&amp;rft.atitle=A%20Genomic%20Selection%20Index%20Applied%20to%20Simulated%20and%20Real%20Data&amp;rft.volume=5&amp;rft.issue=10&amp;rft.date=2015-10-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Ceron-Rojas%2C+J.+Jesus&amp;rft.au=Crossa%2C+Jos%C3%A9&amp;rft.au=Arief%2C+Vivi+N.&amp;rft.au=Basford%2C+Kaye&amp;rft.au=Rutkoski%2C+Jessica&amp;rft.au=Jarqu%C3%ADn%2C+Diego&amp;rft.au=Alvarado%2C+Gregorio&amp;rft.au=Beyene%2C+Yoseph&amp;rft.au=Semagn%2C+Kassa&amp;rft.au=DeLacy%2C+Ian"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460152/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460152/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460152/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460152/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460152/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460152/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460152/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460152/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460152/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460152/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460152/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460152/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490720" id="node490720" data-pisa="ggg;8/9/3019" data-pisa-master="ggg;g3.118.200430" data-apath="/ggg/8/9/3019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">A Bayesian Decision Theory Approach for Genomic Selection</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490724" id="node490724" data-pisa="ggg;8/9/2889" data-pisa-master="ggg;g3.118.200311" data-apath="/ggg/8/9/2889.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/2889" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Optimising Genomic Selection in Wheat: Effect of Marker Density, Population Size and Population Structure on Prediction Accuracy</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490727" id="node490727" data-pisa="ggg;8/9/3039" data-pisa-master="ggg;g3.118.200435" data-apath="/ggg/8/9/3039.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3039" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">BGGE: A New Package for Genomic-Enabled Prediction Incorporating Genotype × Environment Interaction Models</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genomic%20Selection" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genomic Selection</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/5/10/2155" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-15" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-27" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#app-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Appendix</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/5/10/2155.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/5/10/2155.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/5/10/2155.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/5/10/2155.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
