<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/5/10/1991" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/5/10/1991.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/5/10/1991.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/5/10/1991.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/5/10/1991/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/5/10/1991.atom" />
<meta name="HW.pisa" content="ggg;5/10/1991" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Comparison of Genomic Selection Models to Predict Flowering Time and Spike Grain Number in Two Hexaploid Wheat Doubled Haploid Populations" />
<meta name="DC.Identifier" content="10.1534/g3.115.019745" />
<meta name="DC.Date" content="2015-10-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2015 Thavamanikumar et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Genomic selection (GS) is becoming an important selection tool in crop breeding. In this study, we compared the ability of different GS models to predict time to young microspore (TYM), a flowering time-related trait, spike grain number under control conditions (SGNC) and spike grain number under osmotic stress conditions (SGNO) in two wheat biparental doubled haploid populations with unrelated parents. Prediction accuracies were compared using BayesB, Bayesian least absolute shrinkage and selection operator (Bayesian LASSO / BL), ridge regression best linear unbiased prediction (RR-BLUP), partial least square regression (PLS), and sparse partial least square regression (SPLS) models. Prediction accuracy was tested with 10-fold cross-validation within a population and with independent validation in which marker effects from one population were used to predict traits in the other population. High prediction accuracies were obtained for TYM (0.51–0.84), whereas moderate to low accuracies were observed for SGNC (0.10–0.42) and SGNO (0.27–0.46) using cross-validation. Prediction accuracies based on independent validation are generally lower than those based on cross-validation. BayesB and SPLS outperformed all other models in predicting TYM with both cross-validation and independent validation. Although the accuracies of all models are similar in predicting SGNC and SGNO with cross-validation, BayesB and SPLS had the highest accuracy in predicting SGNC with independent validation. In independent validation, accuracies of all the models increased by using only the QTL-linked markers. Results from this study indicate that BayesB and SPLS capture the linkage disequilibrium between markers and traits effectively leading to higher accuracies. Excluding markers from QTL studies reduces prediction accuracies." />
<meta name="DC.Contributor" content="Saravanan Thavamanikumar" />
<meta name="DC.Contributor" content="Rudy Dolferus" />
<meta name="DC.Contributor" content="Bala R. Thumma" />
<meta name="article:published_time" content="2015-10-01" />
<meta name="article:section" content="Genomic Selection" />
<meta name="citation_title" content="Comparison of Genomic Selection Models to Predict Flowering Time and Spike Grain Number in Two Hexaploid Wheat Doubled Haploid Populations" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Genomic selection (GS) is becoming an important selection tool in crop breeding. In this study, we compared the ability of different GS models to predict time to young microspore (TYM), a flowering time-related trait, spike grain number under control conditions (SGNC) and spike grain number under osmotic stress conditions (SGNO) in two wheat biparental doubled haploid populations with unrelated parents. Prediction accuracies were compared using BayesB, Bayesian least absolute shrinkage and selection operator (Bayesian LASSO / BL), ridge regression best linear unbiased prediction (RR-BLUP), partial least square regression (PLS), and sparse partial least square regression (SPLS) models. Prediction accuracy was tested with 10-fold cross-validation within a population and with independent validation in which marker effects from one population were used to predict traits in the other population. High prediction accuracies were obtained for TYM (0.51–0.84), whereas moderate to low accuracies were observed for SGNC (0.10–0.42) and SGNO (0.27–0.46) using cross-validation. Prediction accuracies based on independent validation are generally lower than those based on cross-validation. BayesB and SPLS outperformed all other models in predicting TYM with both cross-validation and independent validation. Although the accuracies of all models are similar in predicting SGNC and SGNO with cross-validation, BayesB and SPLS had the highest accuracy in predicting SGNC with independent validation. In independent validation, accuracies of all the models increased by using only the QTL-linked markers. Results from this study indicate that BayesB and SPLS capture the linkage disequilibrium between markers and traits effectively leading to higher accuracies. Excluding markers from QTL studies reduces prediction accuracies.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2015/10/01" />
<meta name="citation_mjid" content="ggg;5/10/1991" />
<meta name="citation_id" content="5/10/1991" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/5/10/1991" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/5/10/1991.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/5/10/1991.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/5/10/1991.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.115.019745" />
<meta name="citation_pmid" content="26206349" />
<meta name="citation_volume" content="5" />
<meta name="citation_issue" content="10" />
<meta name="citation_num_pages" content="8" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genomic Selection" />
<meta name="citation_firstpage" content="1991" />
<meta name="citation_lastpage" content="1998" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Saravanan Thavamanikumar" />
<meta name="citation_author_institution" content="CSIRO Agriculture, Canberra ACT 2601, Australia" />
<meta name="citation_author_email" content="saro@gondwanagenomics.com.au" />
<meta name="citation_author" content="Rudy Dolferus" />
<meta name="citation_author_institution" content="CSIRO Agriculture, Canberra ACT 2601, Australia" />
<meta name="citation_author" content="Bala R. Thumma" />
<meta name="citation_author_institution" content="CSIRO Agriculture, Canberra ACT 2601, Australia" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=F. G. Asoro;citation_author=M. A. Newell;citation_author=W. D. Beavis;citation_author=M. P. Scott;citation_author=J.-L. Jannink;citation_title=Accuracy and training population design for genomic selection on quantitative traits in elite North American oats.;citation_pages=132-144;citation_volume=4;citation_year=2011;citation_doi=10.3835/plantgenome2011.02.0007" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=R. Bernardo;citation_title=Molecular Markers and Selection for Complex Traits in Plants: Learning from the Last 20 Years;citation_pages=1649-1664;citation_volume=48;citation_year=2008;citation_issue=5;citation_doi=10.2135/cropsci2008.03.0131" />
<meta name="citation_reference" content="citation_journal_title=Brief. Bioinform.;citation_author=A.-L. Boulesteix;citation_author=K. Strimmer;citation_title=Partial least squares: a versatile tool for the analysis of high-dimensional genomic data.;citation_volume=8;citation_year=2007;citation_pmid=16772269;citation_doi=10.1093/bib/bbl016" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=J. Burgueno;citation_author=G. de los Campos;citation_author=K. Weigel;citation_author=J. Crossa;citation_title=Genomic prediction of breeding values when modeling genotype x environment interaction using pedigree and dense molecular markers.;citation_pages=707-719;citation_volume=52;citation_year=2012;citation_doi=10.2135/cropsci2011.06.0299" />
<meta name="citation_reference" content="citation_journal_title=Aust. J. Agric. Res.;citation_author=K. J. Chalmers;citation_author=A. W. Campbell;citation_author=J. Kretschmer;citation_author=A. Karakousis;citation_author=P. H. Henschke;citation_title=Construction of three linkage maps in bread wheat, Triticum aestivum.;citation_pages=1089-1119;citation_volume=52;citation_year=2001;citation_doi=10.1071/AR01081" />
<meta name="citation_reference" content="citation_journal_title=Stat. Appl. Genet. Mol. Biol.;citation_author=D. Chung;citation_author=S. Keles;citation_title=Sparse partial least squares classification for high dimensional data.;citation_pages=17;citation_volume=9;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=S. A. Clark;citation_author=J. M. Hickey;citation_author=J. H. van der Werf;citation_title=Different models of genetic variation and their effect on genomic evaluation.;citation_pages=18-18;citation_volume=43;citation_year=2011;citation_pmid=21575265;citation_doi=10.1186/1297-9686-43-18" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. Crossa;citation_author=G. de los Campos;citation_author=P. Perez;citation_author=D. Gianola;citation_author=J. Burgueno;citation_title=Prediction of genetic values of quantitative traits in plant breeding using pedigree and molecular markers.;citation_volume=186;citation_year=2010;citation_pmid=20813882;citation_doi=10.1534/genetics.110.118521" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=H. D. Daetwyler;citation_author=R. Pong-Wong;citation_author=B. Villanueva;citation_author=J. A. Woolliams;citation_title=The impact of genetic architecture on genome-wide evaluation methods.;citation_volume=185;citation_year=2010;citation_pmid=20407128;citation_doi=10.1534/genetics.110.116855" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=H. D. Daetwyler;citation_author=U. K. Bansal;citation_author=H. S. Bariana;citation_author=M. J. Hayden;citation_author=B. J. Hayes;citation_title=Genomic prediction for rust resistance in diverse wheat landraces.;citation_pages=1795-1803;citation_volume=127;citation_year=2014;citation_pmid=24965887;citation_doi=10.1007/s00122-014-2341-8" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=J. B. Endelman;citation_title=Ridge Regression and Other Kernels for Genomic Selection with R Package rrBLUP.;citation_pages=250-255;citation_volume=4;citation_year=2011;citation_doi=10.3835/plantgenome2011.08.0024" />
<meta name="citation_reference" content="citation_journal_abbrev=J Anim Breed Genet;citation_author=M. E. Goddard;citation_author=B. J. Hayes;citation_title=Genomic selection.;citation_pages=323-330;citation_volume=124;citation_year=2007;citation_issue=6;citation_pmid=18076469;citation_doi=10.1111/j.1439-0388.2007.00702.x" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=Z. Guo;citation_author=D. M. Tucker;citation_author=J. Lu;citation_author=V. Kishore;citation_author=G. Gay;citation_title=Evaluation of genome-wide selection efficiency in maize nested association mapping populations.;citation_pages=261-275;citation_volume=124;citation_year=2012;citation_issue=2;citation_pmid=21938474;citation_doi=10.1007/s00122-011-1702-9" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. Habier;citation_author=R. L. Fernando;citation_author=J. C. M. Dekkers;citation_title=The Impact of Genetic Relationship Information on Genome-Assisted Breeding Values;citation_pages=2389-2397;citation_volume=177;citation_year=2007;citation_issue=4;citation_pmid=18073436;citation_doi=10.1534/genetics.107.081190" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=B. J. Hayes;citation_author=P. J. Bowman;citation_author=A. C. Chamberlain;citation_author=K. Verbyla;citation_author=M. E. Goddard;citation_title=Accuracy of genomic breeding values in multi-breed dairy cattle populations.;citation_pages=51-51;citation_volume=41;citation_year=2009;citation_pmid=19930712;citation_doi=10.1186/1297-9686-41-51" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=B. J. Hayes;citation_author=P. J. Bowman;citation_author=A. J. Chamberlain;citation_author=M. E. Goddard;citation_title=Invited review: Genomic selection in dairy cattle: Progress and challenges;citation_pages=433-443;citation_volume=92;citation_year=2009;citation_issue=2;citation_pmid=19164653;citation_doi=10.3168/jds.2008-1646" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=E. L. Heffner;citation_author=M. E. Sorrells;citation_title=Genomic Selection for Crop Improvement;citation_pages=1-12;citation_volume=49;citation_year=2009;citation_issue=1;citation_doi=10.2135/cropsci2008.08.0512" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=E. L. Heffner;citation_author=J.-L. Jannink;citation_author=H. Iwata;citation_author=E. Souza;citation_author=M. E. Sorrells;citation_title=Genomic selection accuracy for grain quality traits in biparental wheat populations.;citation_pages=2597-2606;citation_volume=51;citation_year=2011;citation_doi=10.2135/cropsci2011.05.0253" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=J. M. Hickey;citation_author=S. Dreisigacker;citation_author=J. Crossa;citation_author=S. Hearne;citation_author=R. Babu;citation_title=Evaluation of genomic selection training population designs and genotyping strategies in plant breeding programs using simulation.;citation_pages=1476-1488;citation_volume=54;citation_year=2014;citation_doi=10.2135/cropsci2013.03.0195" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=N. Hofheinz;citation_author=D. Borchardt;citation_author=K. Weissleder;citation_author=M. Frisch;citation_title=Genome-based prediction of test cross performance in two subsequent breeding cycles.;citation_pages=1639-1645;citation_volume=125;citation_year=2012;citation_issue=8;citation_pmid=22814724;citation_doi=10.1007/s00122-012-1940-5" />
<meta name="citation_reference" content="citation_journal_title=Brief. Funct. Genomics;citation_author=J. L. Jannink;citation_author=A. J. Lorenz;citation_author=H. Iwata;citation_title=Genomic selection in plant breeding: From theory to practice.;citation_volume=9;citation_year=2010;citation_pmid=20156985;citation_doi=10.1093/bfgp/elq001" />
<meta name="citation_reference" content="citation_journal_title=Plant, Cell, and Environment (Print);citation_journal_abbrev=Plant, Cell, and Environment (Print);citation_author=X. Ji;citation_author=B. Shiran;citation_author=J. Wan;citation_author=D. C. Lewis;citation_author=C. L. D. Jenkins;citation_title=Importance of pre-anthesis anther sink strength for maintenance of grain number during reproductive stage water stress in wheat.;citation_pages=926-942;citation_volume=33;citation_year=2010;citation_issue=6;citation_pmid=20199626;citation_doi=10.1111/j.1365-3040.2010.02130.x" />
<meta name="citation_reference" content="citation_journal_title=Stat. Appl. Genet. Mol. Biol.;citation_author=K. A. Le Cao;citation_author=D. Rossouw;citation_author=C. Robert-Granie;citation_author=P. Besse;citation_title=A sparse PLS for variable selection when integrating omics data.;citation_pages=35;citation_volume=7;citation_year=2008;citation_pmid=19049491;citation_doi=10.2202/1544-6115.1390" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=R. E. Lorenzana;citation_author=R. Bernardo;citation_title=Accuracy of genotypic value predictions for marker-based selection in biparental plant populations.;citation_pages=151-161;citation_volume=120;citation_year=2009;citation_issue=1;citation_pmid=19841887;citation_doi=10.1007/s00122-009-1166-3" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. H. E. Meuwissen;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_title=Prediction of Total Genetic Value Using Genome-Wide Dense Marker Maps;citation_pages=1819-1829;citation_volume=157;citation_year=2001;citation_issue=4;citation_pmid=11290733" />
<meta name="citation_reference" content="citation_journal_title=J. Stat. Softw.;citation_author=B. H. Mevik;citation_author=R. Wehrens;citation_title=The pls Package: Principal component and partial least squares regression in R.;citation_pages=1-24;citation_volume=18;citation_year=2007;citation_doi=10.1360/jos180001" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=L. Ornella;citation_author=S. Sukhwinder;citation_author=P. Perez;citation_author=J. Burgueno;citation_author=R. Singh;citation_title=Genomic prediction of genetic values for resistance to wheat rusts.;citation_pages=136-148;citation_volume=5;citation_year=2012;citation_doi=10.3835/plantgenome2012.07.0017" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=P. Pérez;citation_author=G. de los Campos;citation_title=Genome-wide regression and prediction with the BGLR statistical package.;citation_volume=198;citation_year=2014;citation_issue=2;citation_pmid=25009151;citation_doi=10.1534/genetics.114.164442" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=P. Pérez;citation_author=G. de los Campos;citation_author=J. Crossa;citation_author=D. Gianola;citation_title=Genomic-enabled prediction based on molecular markers and pedigree using the Bayesian linear regression package in R.;citation_pages=106-116;citation_volume=3;citation_year=2010;citation_pmid=21566722;citation_doi=10.3835/plantgenome2010.04.0005" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=J. Poland;citation_author=J. Endelman;citation_author=J. Dawson;citation_author=J. Rutkoski;citation_author=S. Wu;citation_title=Genomic selection in wheat breeding using genotyping-by-sequencing.;citation_pages=103-113;citation_volume=5;citation_year=2012;citation_doi=10.3835/plantgenome2012.06.0006" />
<meta name="citation_reference" content="citation_journal_title=Anim. Prod. Sci.;citation_author=J. E. Pryce;citation_author=H. D. Daetwyler;citation_title=Designing dairy cattle breeding schemes under genomic selection: A review of international research.;citation_pages=107-114;citation_volume=52;citation_year=2012;citation_doi=10.1071/AN11098" />
<meta name="citation_reference" content="citation_title=R: A Language and Environment for Statistical Computing;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=New Phytologist;citation_journal_abbrev=New Phytologist;citation_author=M. D. V. Resende;citation_author=M. F. R. Resende;citation_author=C. P. Sansaloni;citation_author=C. D. Petroli;citation_author=A. A. Missiaggia;citation_title=Genomic selection for growth and wood quality in Eucalyptus: capturing the missing heritability and accelerating breeding for complex traits in forest trees.;citation_pages=116-128;citation_volume=194;citation_year=2012;citation_issue=1;citation_pmid=22309312;citation_doi=10.1111/j.1469-8137.2011.04038.x" />
<meta name="citation_reference" content="citation_journal_title=New Phytologist;citation_journal_abbrev=New Phytologist;citation_author=M. F. R. Resende;citation_author=P. Munoz;citation_author=J. J. Acosta;citation_author=G. F. Peter;citation_author=J. M. Davis;citation_title=Accelerating the domestication of trees using genomic selection: accuracy of prediction models across ages and environments.;citation_pages=617-624;citation_volume=193;citation_year=2012;citation_issue=3;citation_pmid=21973055;citation_doi=10.1111/j.1469-8137.2011.03895.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. F. R. Resende;citation_author=P. Muñoz;citation_author=M. D. V. Resende;citation_author=D. J. Garrick;citation_author=R. L. Fernando;citation_title=Accuracy of genomic selection methods in a standard dataset of loblolly pine (pinus taeda l.).;citation_volume=190;citation_year=2012;citation_pmid=22271763;citation_doi=10.1534/genetics.111.137026" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=C. Riedelsheimer;citation_author=J. Lisec;citation_author=A. Czedik-Eysenberg;citation_author=R. Sulpice;citation_author=A. Flis;citation_title=Genome-wide association mapping of leaf metabolic profiles for dissecting complex traits in maize.;citation_volume=109;citation_year=2012;citation_pmid=22615396;citation_doi=10.1073/pnas.1120813109" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=C. Riedelsheimer;citation_author=J. B. Endelman;citation_author=M. Stange;citation_author=M. E. Sorrells;citation_author=J. L. Jannink;citation_title=Genomic predictability of interconnected biparental maize populations.;citation_volume=194;citation_year=2013;citation_pmid=23535384;citation_doi=10.1534/genetics.113.150227" />
<meta name="citation_reference" content="citation_journal_title=Anim. Front.;citation_author=J. M. Schefers;citation_author=K. A. Weigel;citation_title=Genomic selection in dairy cattle: Integration of DNA testing into breeding programs.;citation_pages=4-9;citation_volume=2;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=J. Spindel;citation_author=H. Begum;citation_author=D. Akdemir;citation_author=P. Virk;citation_author=B. Collard;citation_title=Genomic selection and association mapping in rice (Oryza sativa): effect of trait genetic architecture, training population composition, marker number and statistical model on accuracy of rice genomic selection in elite, tropical rice breeding lines.;citation_pages=e1004982;citation_volume=11;citation_year=2015;citation_pmid=25689273;citation_doi=10.1371/journal.pgen.1004982" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=E. Storlie;citation_author=G. Charmet;citation_title=Genomic selection accuracy using historical data generated in a wheat breeding program.;citation_pages=1-9;citation_volume=6;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Tree Genet. Genomes;citation_author=S. Thavamanikumar;citation_author=S. G. Southerton;citation_author=G. Bossinger;citation_author=B. R. Thumma;citation_title=Dissection of complex traits in forest trees - opportunities for marker-assisted selection.;citation_pages=627-639;citation_volume=9;citation_year=2013;citation_doi=10.1007/s11295-013-0594-z" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=P. M. VanRaden;citation_author=C. P. Van Tassell;citation_author=G. R. Wiggans;citation_author=T. S. Sonstegard;citation_author=R. D. Schnabel;citation_title=Invited Review: Reliability of genomic predictions for North American Holstein bulls;citation_pages=16-24;citation_volume=92;citation_year=2009;citation_issue=1;citation_pmid=19109259;citation_doi=10.3168/jds.2008-1514" />
<meta name="citation_reference" content="citation_title=Asreml 3;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=Plant Biotechnol. J.;citation_author=S. Wang;citation_author=D. Wong;citation_author=K. Forrest;citation_author=A. Allen;citation_author=S. Chao;citation_title=Characterization of polyploid wheat genomic diversity using a high-density 90 000 single nucleotide polymorphism array.;citation_pages=787-796;citation_volume=12;citation_year=2014;citation_pmid=24646323;citation_doi=10.1111/pbi.12183" />
<meta name="citation_reference" content="citation_journal_title=New Phytol.;citation_author=J. W. Westbrook;citation_author=M. F. R. Resende;citation_author=P. Munoz;citation_author=A. R. Walker;citation_author=J. L. Wegrzyn;citation_title=Association genetics of oleoresin flow in loblolly pine: Discovering genes and predicting phenotype for improved resistance to bark beetles and bioenergy potential.;citation_pages=89-100;citation_volume=199;citation_year=2013;citation_pmid=23534834;citation_doi=10.1111/nph.12240" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=Z. Zhang;citation_author=U. Ober;citation_author=M. Erbe;citation_author=H. Zhang;citation_author=N. Gao;citation_title=Improving the accuracy of whole genome prediction for complex traits using the results of genome wide association studies.;citation_pages=e93017;citation_volume=9;citation_year=2014;citation_pmid=24663104;citation_doi=10.1371/journal.pone.0093017" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=Y. Zhao;citation_author=M. F. Mette;citation_author=M. Gowda;citation_author=C. F. H. Longin;citation_author=J. C. Reif;citation_title=Bridging the gap between marker-assisted and genomic selection of heading time and plant height in hybrid wheat.;citation_pages=638-645;citation_volume=112;citation_year=2014;citation_pmid=24518889;citation_doi=10.1038/hdy.2014.1" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Comparison of Genomic Selection Models to Predict Flowering Time and Spike Grain Number in Two Hexaploid Wheat Doubled Haploid Populations" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10.cover-source.jpg" />
<meta name="twitter:description" content="Genomic selection (GS) is becoming an important selection tool in crop breeding. In this study, we compared the ability of different GS models to predict time to young microspore (TYM), a flowering time-related trait, spike grain number under control conditions (SGNC) and spike grain number under osmotic stress conditions (SGNO) in two wheat biparental doubled haploid populations with unrelated parents. Prediction accuracies were compared using BayesB, Bayesian least absolute shrinkage and selection operator (Bayesian LASSO / BL), ridge regression best linear unbiased prediction (RR-BLUP), partial least square regression (PLS), and sparse partial least square regression (SPLS) models. Prediction accuracy was tested with 10-fold cross-validation within a population and with independent validation in which marker effects from one population were used to predict traits in the other population. High prediction accuracies were obtained for TYM (0.51–0.84), whereas moderate to low accuracies were observed for SGNC (0.10–0.42) and SGNO (0.27–0.46) using cross-validation. Prediction accuracies based on independent validation are generally lower than those based on cross-validation. BayesB and SPLS outperformed all other models in predicting TYM with both cross-validation and independent validation. Although the accuracies of all models are similar in predicting SGNC and SGNO with cross-validation, BayesB and SPLS had the highest accuracy in predicting SGNC with independent validation. In independent validation, accuracies of all the models increased by using only the QTL-linked markers. Results from this study indicate that BayesB and SPLS capture the linkage disequilibrium between markers and traits effectively leading to higher accuracies. Excluding markers from QTL studies reduces prediction accuracies." />
<meta name="og-title" property="og:title" content="Comparison of Genomic Selection Models to Predict Flowering Time and Spike Grain Number in Two Hexaploid Wheat Doubled Haploid Populations" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/5/10/1991" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Genomic selection (GS) is becoming an important selection tool in crop breeding. In this study, we compared the ability of different GS models to predict time to young microspore (TYM), a flowering time-related trait, spike grain number under control conditions (SGNC) and spike grain number under osmotic stress conditions (SGNO) in two wheat biparental doubled haploid populations with unrelated parents. Prediction accuracies were compared using BayesB, Bayesian least absolute shrinkage and selection operator (Bayesian LASSO / BL), ridge regression best linear unbiased prediction (RR-BLUP), partial least square regression (PLS), and sparse partial least square regression (SPLS) models. Prediction accuracy was tested with 10-fold cross-validation within a population and with independent validation in which marker effects from one population were used to predict traits in the other population. High prediction accuracies were obtained for TYM (0.51–0.84), whereas moderate to low accuracies were observed for SGNC (0.10–0.42) and SGNO (0.27–0.46) using cross-validation. Prediction accuracies based on independent validation are generally lower than those based on cross-validation. BayesB and SPLS outperformed all other models in predicting TYM with both cross-validation and independent validation. Although the accuracies of all models are similar in predicting SGNC and SGNO with cross-validation, BayesB and SPLS had the highest accuracy in predicting SGNC with independent validation. In independent validation, accuracies of all the models increased by using only the QTL-linked markers. Results from this study indicate that BayesB and SPLS capture the linkage disequilibrium between markers and traits effectively leading to higher accuracies. Excluding markers from QTL studies reduces prediction accuracies." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10.cover-source.jpg" />
    <title>Comparison of Genomic Selection Models to Predict Flowering Time and Spike Grain Number in Two Hexaploid Wheat Doubled Haploid Populations | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"eJFqNtKg1XqcoK-7oZApVS5RQU4U9yEJBH4yqKOUxm8","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460155","apath":"\/ggg\/5\/10\/1991.atom","pisa":"ggg;5\/10\/1991","ac":{"\/ggg\/5\/10\/1991.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/5\/10\/1991.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;5\/10\/1991"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;5\/10\/1991"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/5\/10\/1991":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022yKDxseedpEdV3VROF8glqv0H0oEEqi7GsqBAtAR06ESwmwwM7SGxxOFJIPupVISjlQ6vB6+4AyqjycNnp4XtXTyeNNU1Pc609CpG+zhKkF8+vsX2vHybhI3cwcS\\\\\\\/AyuWbTw6N\\\\\\\/WhKalcWnXrRLf7MFuuX9k06au+YWWXsHD5PKUXAFXjwvIrgbOpF2HOHAqdZQBYxGTVPKxOhrHpm4lcL8wSMe7rUZtwES\\\\\\\/qf4VYnttweB3Y6t4Xxr4DTuHJsYOSsJBAFFok9qqkv1GKqg5XRmQE5uNA3M9cp+cDm\\\\\\\/XCPVF5+TgO808LEkiNyhf\\\\\\\/7WYeobHaRW530\\\\\\\/85GavJm24Q7sk0uVizY7xETa08KNyxSxWM609\\\\\\\/BI7Xu3BlHcJxSLQETwARxtVLSxn3P+mYkEYfaaFDVlg6kEgXG5i+oq3u+9\\\\\\\/kT2Gsi+2MQ\\\\\\\/k0vECBNAROJ\\\\\\\/vDMgb7PQOUDQDYO+U2FTphIPBjQBgPqh2LHJ57nWRk86kMSkyUJNRTY5MCZtuVWLN1ARg2IvwhqW\\\\\\\/GPVZiIZIMqvjvKPqSHF3778sxNV9e8bJ5915Xhp5F0FYNgVeiPJXc4XPLPoDqYsR4jjfA94hS65yuDd3bPg76p3muehSFfZ\\\\\\\/WdUUL6Maz1DG0j3k\\\\\\\/VVOf7rEKrAWbNuTrYigEJWu4O8ZWtYDM8BKKpjVI2D0sh2BWVf7Z3CP8CtnlKJLgIoi3TjA\\\\\\\/jcau9lPEPjZa2pNtlGFFAjsTGpZt0aCOyRBfco5PvL6tL3KY04Y+djZOvp5hoRKXZaYLygqfKe1Pk+9eF2jq86A+Q2SPaSbNQrY0m\\\\\\\/yTqoyRGKswt\\\\\\\/+ZV3K9yNF2bAxX6w6cfSLHFWoFyeFXYxoW3jAmi3m\\\\\\\/Lf2gOkRvSdLywugD2jbBk5mHnwNvcD8hdphMPHBRsrVL7xNol+wcxs5uoGTiHKSS1WPTlxpNFFgL0yBGzCZVE58fZKo6BJ5s4rS9Jqw1A\\\\\\\/H4wCsU6Lc+nU5ZXz5vNmLNu\\\\\\\/C9BbnLT39kJEmhtWzuSBQGjWIwARZstBVW5hbVpD\\\\\\\/xEl8Qw\\\\\\\/RW5v6bQYxPIWjJZSsnRxhdV06AWufPM+1VEfu2Qjnp6yM+D8bKIsBbG2YKper7nMD96GHpzPC\\\\\\\/iXiJ2dgujmsvIA4T+6ASA0fNupY0Vo1lZu92bALbM3fXlvJLR+IRrmnvVaK0UP1TuT8cYyqOX3v5S4em5Mk7JsR5bxuCTtlvujiusy5+lm\\\\\\\/y4mQUJEhDDQOfImUssmk2fHZe0uxKWrlZcwvplje3nxobd1GAmkCd\\\u0022,\\\u0022iv\\\u0022:\\\u0022Ne7goz8XuRmhM36b1+zfbA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00221a7e409c51d6a74e4ee09954f1aa23a7d9bdec0b0652b8c326bed3829df3109d\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022hshp0lvu21xE+vaHw6fo6fNNhjiBJLc\\\\\\\/kVZuiCHIQh0WZ3MD1agJCEw7qL4VmHWXv157YPrj\\\\\\\/TP\\\\\\\/sqYnRc5ypIuYqnQhpPHq+IOsucLJeBh2sbvwsThJy9OSQWUuF6cYOE0\\\\\\\/xoPEHlK2YfG3yqLUu6N6kCuFNapRwi6yyuRMU29Ieo\\\\\\\/0uhZKlB6VfN6vVAQfQBHDfPZ9W1rHtBk4BJdjNY1FlwhnOfwKxzBHd\\\\\\\/JQfPMVitq+u+TXL3WdFhXy+Uc9zrGvndnz+2t\\\\\\\/ZdDc7waaAj+\\\\\\\/Y+C8gvvXUhNmGu923oS\\\\\\\/rMPx9vcjPsaecEwqVR+AUlDsSOq6a9dgvZt8949ZO\\\\\\\/VyfP79YaC+uWJCg+x4UjR\\\\\\\/INyiuhaI4tD9cl7ETi\\\\\\\/OegBzl9hFK+jrV5Q8KblmKvhYuKk+ff+AFXGKHdokq8X8yPOpPFOEM\\\\\\\/O0s4B2L0OE2t2vBv3Etsiw5bFVdehTcPklT\\\\\\\/sY4tbhOVZ+eJMj+R0dDBYDtIqIFRU5pZXAMoNOU1qHtyAlRPl4gKYAthRpeXapgSdRqWGkRJvZrjsUCEE3tDloG3ugG0WKk1iQArs\\\\\\\/arf\\\\\\\/q+0903ja7qtd7Dprqb64bcrDOowa03VXHfDBgG+Ph\\\\\\\/XBFXKoMZH0Qo3vzXGu7lPsWeRUcn2XNYDzvRJ+lq0iC9ZXAcczA9QFYuL5Jhe+FfPDDXkz7QgFhm9Ca8RE\\\\\\\/4SxNFA3r4Gnz5CcG68nWEek\\\\\\\/7wzPNXSr0MAy6uHpCIRREue27jJKu1brKkRAdue9K+NOBEnllFXKEy2DaXVUbyGCbSCQE+ar38P+T6LbmNkmj8O6lI8T4YwasrT3wn7SsKCLtnepeO3a+px2+Cv38A3SLwpZY\\\\\\\/6Yhgv1ioQm4dcUQ0keAwaXEmwvaU9ZBdA7Jy+PwRYM6bl3maiWBgUw5x92DYZzACngLLz2HqMhAln6CrJq2MzYUpjomBCOe8UeBJtamETuUsSN7arCj98X6Gx0CgTBVbDlLE8kbKRdqKtRUEVwgw2+UaPrwbCG8HNm2ZNj0Y72M7Nhbnbm1BG1doLN1bIPNAw+9P7I3xSm4sJgdZwpBL4S9DtzvRa8JhGS9KrTJLEEMBd9Sk57L8zepsAE11Q5W2lxXRRMV9nVOIE\\\\\\\/qtVh0Mzdp4n6WIUMwD+NMi1EXGPy1Xefub5JkRQhqdZMgEPFw1Eelh8H8rm\\\\\\\/8ewH5m5YhW7k9BM9V3iQNSetb91rzMDQ1gxPjk1YIVa2G74l9FUu1ab0izRAewI7fNZ\\\\\\\/K2TMJK7ZvYMu2b67ipBcDXjqxu4BIrrm5xjIknUxc2m7sF9YCMvbdNFr2iOU9gbcnfFKWE7U+DA25w0qawLQudNOMPzWMOCaycLcOMDuu7ab+XpUik4wMSWdvG1J4AFz+L1TduZC8d6NwIIAqZwJtc2sJdueYlDhNYpb531x3fM6S67K6vPiRaVp876yRYcnaAU4ZRhQjZpjwh2NmQlRExth6qIOM2M2C5ynK7gK1Hc3W\\\\\\\/l042Iwb3M+MiUzbgaYP8F+9TJPh+mDYureyPh0adHy6dzMDiuwzuWJXvPGCvb\\\\\\\/sE0EqddujSQyh8QYHXJtlvGi6IxZzPofjPoYHu220F\\\\\\\/NIZwH0Jprg+VELkZaSDRU9WmmjrmwroUvZf85NkHzzV0zGzi3DwbRsjaGdECbeRbFXvMactY7K1F6rKp\\\\\\\/kG4EmrlYn+R52+yMF9mURQCZFoF2euZ846AK7levu5jXgd4BmPWkGCk17yrVdHIzKfGjxsppIsViET9yuelFbLqX6uzbVE\\\\\\\/\\\\\\\/lr4CfG4P82djqj7j8Ph2LwXp5ARh25yQCV42pVbaJdw5zHaPUHaFaSRowVjzXWnlfT\\\\\\\/a0nH0l+pJIKEJRRmsERr5Hf\\\\\\\/YliK701DyNSzPkeml9wEmVZUuhVvozu\\\\\\\/Il8b17g6myZ9UqEADCxYd4XVZbOJjFawSf80RDNOOYkdzI8or1pYaNlti6F+78ZKraghDbu+D6yz1943YK9PajEMpxFvY40+sxXsz2AjzOChlNk5T5Ji3iq1Hgl65rOL20fuZ7JtJXybbir8DVkP6UFgF\\\\\\\/NqbH7tx5wqV3EjZlknVKfKJZ9Sx3mfrlu9WmKtzvwmq7v1hAmI4iqVFBCMSjM0Rx7Tlawa59YupREUeYOKvHnhRftiIjQN09lJHAgj1zNR5+v90df4k1OANvczkXsFYvDvh7+qmz\\\\\\\/UHituOUcpwyE42UcL\\\\\\\/VLrFrhLQWC7BwZMXRd1aReAP+Bn58WMc88bkG0JZEm2OUTliuHonASz2Y+p\\\\\\\/8LYQ1hZLKzap3ZPR6q70Ogcf2JEWgtYmPO7ixgoT7xP6vqZ5N1lsJFChsqC83fc6Cs1aPQICMEQLJODmr7zWeN6Gaiy0+TfYwTmOHVB9n0biX3a3dmCtQkPXGSrjgJqAsyNw\\\\\\\/VN7gk9ad7LkZeQfwkXyuot0uww3XxV7HW6N7peinSIfUbCqCDqFYvDATZOvG22j9is7FGsO9\\\\\\\/pJajE+Zeyu3QN8jOx+6hLU4EUB2P\\\\\\\/9sU1zVF4jQ5uLtQn8FXXkOiKMe9kHBM9Wiv3K8aybGGv28SsmruMBn7BQymUrYOaQ6QX8I4YD5zfLa3PxkBBlY+OC22hXpknimVs3we6aPs\\\\\\\/eIjjQb6w3nXwIIwhZDEAfcMaRraQEd0BAqfwO64zNRMki9wOd9DK+QqlfDuw71GDcJRKaCK0ZiLTipSGBHCAArqQjZaFvkCVwah4hHdJYrub1bVbsQ9480pgtf12p7FV5JOBczkKdMtr\\\\\\\/HzlXC\\\\\\\/eH11ssjd7WiC9XD1smVDauvuFC\\\\\\\/D6Uq5HfF5cMKZ+k3uD74GQtsnbJG7Vf3hILio\\\\\\\/pkEMQUbH8RchGo17zsTbYDtJWBPQnLR5\\\\\\\/JY363EtQIF566Fv2InJCNYAj6c\\\\\\\/j9oTVjcM300mCd\\\\\\\/ffUV5iPlff\\\\\\\/3xCz9ATpur6XeSDOBCkVd3aswBc2xr+UjCWG2ta\\\\\\\/zidxtbcG1RWE4Sntvl690P208eaZHYWCUOODAfXXNxM\\\\\\\/4FjFoMJ9rrtqZvwmowDoBmfsnXnuh0YdmFzIvSJ\\\\\\\/bTofTdMa0wtCbuNYr39n7craeVuCQvyZE\\\\\\\/YHEFv37+9KGgvbbz85YBiWd0eLJepxERwT0yKCw30f0aw8fhpw5cD353MQXSE1eewQLRAXSFEWVe4C88QBU6ZYRrC4cniVex86tmkNrMT4RnTMw+l1YywJX9dhMhN6GwZrcSZtZdFF9qQxpoB+gadibY1s7zr\\\\\\\/eG\\\\\\\/JvR4vIYGyzlJLTqMYaLDWvmjAnOwkDPNzY5kzC15F4V0vNJLnTr47e\\\\\\\/AkIiAYIdCk6h2vipVsJhkLXPNk5I3XzMfvaNh9sbimVmhXiKLYAdgOAfyOsQRrKTh5ccYecrI79pxJnpmnFwIBSXYLadgEauLoQTI\\\\\\\/NdzgoqUGFo\\\\\\\/ila0D0qpLl9oTLkfoE4XgIrlv\\\\\\\/K075pAtNkWSNNwmIJYEkSz3xJqCmGFpgxSI5FBWc5bPxXiqC4nTM6bCpdXCHky8xumrFQMPiw3iSi\\\\\\\/oj2tt9lRDaVgoTf5xqN6lpEm+0GyfAeGGh\\\\\\\/kLzOrDDJ\\\\\\\/CZHVwa8foNaFqBLGc\\\\\\\/qkr6+PDp0pc1rsgNUm5bXmgogNDJGoqkCgWw8hKfueg+ltcQdhMiIfyYjOQa03PBawukcqOOeJ7Fi\\\\\\\/QbKUph2r4QBEtrQ\\\\\\\/G1agaodILBwLP5Lr25wMVtGCj3HW9wws6iBVNNzn\\\\\\\/AN31bc4fpDITKAx4dovI9sKdj0dAraAiZsSVbQSuK5avuegfA8N\\\\\\\/azaht2vuzYW88DCpcvKZGivyIMhhGe4b3ZwJN5cQlCUg\\\\\\\/1fNb5RlljM6enbGHyeKt5Gx3KsFd4+CkfFAFdfPqYN9aKdedX2HguwcHg5AX0ZRoACVU2SFQEYyYSBV8k8jM6UKLZ12kO2IkTRYaCkpmpnuZ3IotJAqzVBs1ScrYVVyji+qRn7JbKPuo6zr2mmf57yIqKdtVZ\\\\\\\/LZC3bewKAM+LSbdhcY\\\\\\\/NQwfMu6qAXAjWL2ix3+I0+NsdZyNfJRRYmJoCqrLpOnTvu7ohbMGIWzWHfRz\\\\\\\/DGkIfcbHl\\\\\\\/Jh8GfhRlbrZEXbzrokdJ\\\\\\\/jJoKbCW+LeZaLeFF1Fvalgt71Jjfi4xkzFdTxxFny9I1ygqARczK2gHxGq7ST4h0aG6yOZSzhWpnUU5s7R3NZsap\\\\\\\/u6C3JHHjXMSt8ShWQM4X8lLmbGMHMQqTItu1gq\\\\\\\/UowaXsGGu+8uLAXzE\\\\\\\/1vUyrGguGZNpocccZBCaDxXTUvB3ioDCt1W9g+jmNa1IEvp84PUXQdvkITkhDT\\\\\\\/U9TFwwRPIxEbZrMu1t0RLm6AwmoXhaYbfxbmDf7u2LBY2FPpqauZ7DI2bDBz1851a1zxo4X7dNEmpHnIHfD1FMZO9DNgNGuGNKL22OFTlg5eJzmLLscK9l25OEsf8HX1TaaM2H9dtz67\\\\\\\/z9fy911TT\\\\\\\/VhyyuEURXX8tzMBZaWrsIXdBTn7+k6OZz3lkgcfgxWEO7mNs7XlIfauEswWaRvw\\\\\\\/hp7mWiMYuuwG\\\\\\\/ppc78bAmNnBVnLanHasUE2MPD5V8hAPjbehFQinwntH9IkYO43UjZRjMkRWPlN8e13zsC5VNX9AdgmKG80ByHlaWOMyOY4zt\\\\\\\/LU1AuIpPPfTypNHFcsVkbk1dH3+lQL1Lyjxgu+Yv0Ew3UyHS8N5FbmLc2lYZ6m2iwNQPpBt+AJLGP71LQHIfNgbAiEw82MqPjtnvnWOeu5e15RC3evhDbXKMRTwU6DB+0LHePr63UCHd0xZARYNAmOw6fKhKjCfdIohSH\\\\\\\/ZcFEqJ726u1VOb\\\\\\\/9GwyXA\\\\\\\/NHmJSVih1PIir\\\\\\\/rmcUVp0KO641h6wLBWPW2qxbC5j+dtx0JcwCOtSEz9eBOg9JrCxC0yGDKrBCO\\\\\\\/QTyYCornmLbafJ3LSZPuCh4CoT96gEuW6V4BwyvHAy4Ug9v5rUUF63D4Xxd7QbiLxzzP5irdx97UNy2+fr4J5FHovjcny1rQtQ\\\\\\\/4BHB3Vern05JGAZ5LCT9gsRN\\\\\\\/Ti8eAbxS5ij+CB+HFPClffFMOuswxwfx1UlciR1omiAas9IaG5\\\\\\\/y09kkWltrwRKnsqba4ZMy3uGcWcRLBE4bmYgxSii0MkJHuWrkRBgx9zj7A5iSxCahM\\\\\\\/hEh9ArSrm3NulV8+JHUfkbp6GoAOnDvK7L9vdbmOaR4hBAp7qNaSkJAk+nlcUHqfwQzhIykR4fgQxVhY6Qufy4L7MvoS7BUD8LxpP92Q5LfKmNWvZS2jZKq5pdkVhdkisnjODQIcgk1zZ1kKLLQ\\\\\\\/hnHAc3c4YpTpeDxJk1hJ\\\\\\\/qScXskEght5s9Q2QwAstR0fmwD6xkkCoYpumUZQss5Ctq7xDYg3THCKxz\\\\\\\/PuQ8Fr73LfFkJUrdg8X4ufqEvAOeh3Cj5apfHWHkHPwEtP+w0XRGKSJbUuciO+e6yGsr5BizGs6jXTwaN4qfMsQiXneoUOQKREP3dTzP\\\\\\\/YvrxvrdTxLEhVG4H7Si9Xl3TWJzjCopgOPUZGNT8n0CGL61rDDdsSlE7lsvVUVxvWJ8QHpvo+4cmyRtuE2ZL+Rb1i22BzLwEsCgsLTMRFRTZRCJ5mGuwOMw434HNSy1a1wj+kWMoY\\\\\\\/v\\\\\\\/WMZ5rVrI9DO6Kn5ccfRUQNDXPsg14fkbT8Au2zgwEXUqvyrtlWfnBQkuT4dFcP8ZfdieRqKYdUByxFKWt68Q9OEEUw2F1lvtBGXgctYufwMQm2ZhkVmoqoSHCluTZWafeqI4px46EwFM4wBm+4zkRk\\\\\\\/biWUDn0V7WnGdf6PYIxQbODIpCEmuBTYEM8rwyXWvWa4HI\\\\\\\/cVdnAysV0o3\\\\\\\/3RomYlv27dsMih\\\\\\\/88kNTd1jN35WuirkHCSiMIeCAUDfhohlXRqurLrF+OIUGo0bvhMOzOsvxcemIFAqHcJSANfURPF2MiQzkVmsZSPS0c2\\\\\\\/JSnNwA3x4QyJd6lHCBDzpRTG4uvtTIEynyol+qmsGWBdoR2HyMXHA0tpdNjufm6ncEh+s1pA48J6hP2Qy4W8syBCBGRhpvlbnF+uDwdJiNdi633IllhoCJrhJZ5Pu4nQligCDqO0rreYv\\\\\\\/GqIDbmtnQGoMp34UoOgXXv9adTmXYafUwrfbgIAhG5YIXPbtix+S1BRFb6gD7XKpT+XqZri8vRZEHsfHdkmz1dHnv4jtTzgecCp+FV2f4PaFiOHUN8jZz5wAXUkLmiMI5eIiVhQafJljsGRiEynGfSulYfLjIEpy4z1E+rW4CzwkGlV3pJuP5YLWAsCiqHd+7nFzPc1NkYZ8jSyOYqgV8pmSE2Mww8pYK\\\\\\\/EwQn761DFjUsFQ\\\\\\\/VzP9gNKcT3mUxR1CL7QGQayJ8XRxds9dHJ7u0PugVap4elyUaAe4PyudR54YgkR3vXPMma3ZaLxIROsqKRncSDR223kQOcRHJW5rgFWRZI43oV41Bw4H98Cf50APUKg1xc3EHS1a2kGc1mljh2qQ8c\\\\\\\/SfGfgbVHnx4Abl9jNptwhnIQy82QlCgSfXjVo3cKWJTw\\\\\\\/SKVmjtAwKOAoE\\\\\\\/v215HpZF4T2Ruat2hkm0HvOLuNA5+RuoBBPKGtEjh7jdBiTXVoIGI00q6++9Kta2VX4TiZEyAvUHjREiSDqb45enA9jmWYoWE0wA12tpNKzfTO35KnbwvjPsCnB8czci3TRwcnbgusqGFbEOSOsBRQJHa0Xg76OeCbyT72lzQ2BflA+XZ2BSH41Fx\\\\\\\/b4NF2csx4T4EcyfbhfxdLHW61hLFSypS5yBaggY+1Rj\\\\\\\/K\\\\\\\/JpN6O4RaVu5Xe\\\\\\\/U3wnURX\\\\\\\/8rBOeQzq6rJxzD4QN\\\\\\\/Kn3Pw5oV0nb0updnwc9l05ogbPBZQ+UsrSUhVpmKOGADN5sTEoowhDTMiEbw5JnhlEDlRsYd8wok7yIEUlAFafvXaocftqSPolmUq84Nwy1W7ETYFxW0xS8gezZxw8Nl++WTFq6jghIPzQwq4hY6POwARolgs8nztrTasT4nEUPhnDK\\\\\\\/t6ogFDlTJ6XkLdB61Crw8New9sfc1ZB0n7XhqbrI4QG+V5jT0QYaPw5lTDRDvBZxjbqQXobi58tuZmm09I9w41vRdb82b0\\\\\\\/O2OJfCAQx7eKd2vNixJqeiscpQurEgO9Lw3lzyTvRxt7Y3DLK8mhpfj7QThbvVtzG49DZwkVPByKZ\\\\\\\/MjiBUQs7E7w7IaIxY6rFtsNpfz9J3kD\\\\\\\/ZhoV3+NZZIk9n+LsUcGj5Ks7KQQdGDI+uyVaw+d5uGx5pmhcpiidMAYlmGTNmPi1vgSuqUm6QrEoDlhbgWKYSL\\\\\\\/wThyt193xngLu2J6bYZ0C7JLEjgBIk+7BnzQ6iG+gmJLopOSWYQXplRD\\\\\\\/pzbDNWsnGroFRoCtdzxQMJcLTVX0eGiT9IwOVeOTWSHTqLD\\\\\\\/hYg+YRIMczgnVk5+E+EG4O+iIiAipuA9heZysu9l6gWdM2BwXlPeR4EuidmYUpX9biMI\\\\\\\/ayDy33U8JHYhFFP2YlwM1N3HxMUoauM7cLxJoJFDz9PS375QM1D5fjuO0BuOM6nhPsYMcK+k8ASl0PaaW6NjGZjXvj5MC7F\\\\\\\/tpsYWq6qIQX9yrbx5KHvhZBCneqq5IJ5mgGJZeNRIpeD\\\\\\\/o8bTDt5oWXK0NoHt8lPCOKI8KdkOKFA7WA28h6elux37mmFExtWgbfWZRCtze67ijTsx2Mp1wpk9dIHiO3eDCEQ6fBCNRY5NsfdWTdGzRPBqgc1zObMg\\\\\\\/PGgRKE7JopPNeOT2dh\\\\\\\/qrhtXaV70RD0qi2YUOCYTmlqwvMv7F6VTtFpHg12xq9PTTRwYBVDRmabZmUPsr2wXrggZWEwDjqKInMsPUEiWPw0QntoKo5jiqbPFlOX\\\\\\\/wzmYXpr0pQBL4SW9Nm5lh0txiX1QG\\\\\\\/htXl74yZw80GfxlLCzfxoLCMeqYiPU+zJhMmZ7+BmSzrdS815yfVYzq0eF+TnsZnOhj0+ia7OanRePNaCVqgc6QBoeJpQZafaC9yiFKvYlSC5VBYZ\\\\\\\/wHnCEV3tZ2IgBTkpiI\\\\\\\/E+jZHphv6tTwWEUGRLKHqKK7en4+EdQeouk4KDxsKxi2UkunUYPqtxKCrcBeOeQMgue4+npxRqwN4NDv0Fk4yRuYLQsg1iTxvLjdgk66NhzQhOuFxcS+ZGCduD9NqMOe8o\\\\\\\/ru5PM1lRUwEVJDFFa0YywEOqwJCuL2GBdWRzbn3KxAg5iKCxYTkwGfL\\\\\\\/\\\\\\\/8do5WTqMRxPcRFBjg1lDHigS+LfGusrZMmPTg578xVjtF+V04rUCnfRel6WG0VfUz34vCHqX5UxCBqqGlBJgn2L\\\\\\\/TIoDa3WHsB8ESP4O\\\\\\\/HOPorAb46SKXVTdyICuiE2q6UJntclHcJ7VthmaRpJiggVM5\\\\\\\/krAyoPXSXgbU\\\\\\\/ZYp+kWznF8O\\\\\\\/wfeyn0ZEVsp3mX4h3\\\\\\\/f1buD2wPc0MwxrmCdjC8qhuNCWPHJJFWI8npXMzyCeuwL1wMeTbRKaYKLbrW\\\\\\\/MD1Dd1qsMSuDiJ0\\\\\\\/NR5ura1kAgQI43Xn1l89jeoD7MB+93IhQMtzDGfIS0N\\\\\\\/k5DaXHM+xKvlu1C3MIRG\\\\\\\/Mg6O0RTGV4LV95cvC7uc8C+DFYGQxPclJt6d1Bt4yW6gapptxdOfTVpTyDlQw7BzJwwNm822Ap6BPuCIKDHshQzDIo3A6aHtpwaLhDXmExUXsEFR6v+94a2BK0\\\\\\\/7LfuYqal9i8z4It73zrF86fBeWqJUW39xPZNgIPASgAol+m5FZn4Ld\\\\\\\/0XXH+OSJyx6\\\\\\\/f+WoOH3VfMS7Nain7AV\\\\\\\/Hy12yBxjBVpuD1+CSIm1pglNsCn7SEn1bChmdM5RwLLxmumZ6Qx6yUF0k6zZBcpr65N4bWdGkBGZ+hserJHb2e224RHO\\\\\\\/uBN54A4wLWHR4OP7tK0gx\\\\\\\/4ALH\\\\\\\/iHCzAoe4oW\\\\\\\/+cgubxdfgwR4L0a0F5W5m5Lilo5HePLi1YwpyWNYJ5V6AdVbDUmk0LN6dHGRch6xQgMm1tEzwu1o56UIbs2aOK85\\\\\\\/cc21mbNtOJy6syleiH1YUiLqcfqVXuifc4gAiP5X3WC7H\\\\\\\/Gu90ISJLJ\\\\\\\/KRgbhU2Lwsff\\\\\\\/aibe2YHQ6Zo4TryF32plIJh0PcDx1W64FXKmU+DFtQPG+a3pwziYEUZoaND3dKgmuNLzk3VDaidZ15SxgC1jdjWlvKpO5pexcXV0En3FRWyWLZAyR82oSCkBmtyv5lzpVVZEMWP1eVm+5IMFbp5lDTNOgX72k2AWVSRaWnQS83iqcBu8SRiBca4+VmS4nWlgturE+ZF5hagNYkokYtKk3lUfjWYP2zhFg3v1oqi8wkqbwSHcpPdR9tJylNKctcnUBeanYL+Sb6ug0XIcKEkMlXEC8jD+nG3RPeIINq8ltOVxhFf96hmNrGdg8uaMByxRgaFnaoPR4s724oHoJSMFMZTuF1iu43J9LE+K491FPmuQ+IKJA38zfm\\\\\\\/DdUlwVPugywcAA7LPZiq0nknxsJDDOHiLMcSjnTiRLdfF\\\\\\\/X36juO0oG2ISb1rlm0vxpgr03XLXUV8RcGwFdw1dP+vRlDhas4Vch\\\\\\\/WkYvZmjSdszq+yaVdctW3eAo6OKEd+SVFfDb23WwER0+xvjh0ul\\\\\\\/UE5STTQEbxcaG6ONZCRllHu3bqQAQ34cGYb7rV\\\\\\\/qwkSURodDt7Y3AXZ3hySgjQ2mG2lv0NAitTALoxrDSzu7lcb9joVv8asVCbhWun46KgJnFXhl1gr4g9XZicvdBJAQfrCVChDy0aAaVsu9l7EUHZBB23KF4uVf3jLaFoRqmm1m6duDSqtKBUZZ4Ip2EpoXjVYzf6VY3ifKaUK0KIIpRWoPVdhSIMT+B55Q+Ts8Eu7SjQ31Bs3L+vFVV48PL0flKrySkdzin2tLy6puntyFYRHq53Rpv3jtlZ0DQUMoEoLAo0h1Dc398YhC6gGZ9r5gTrCsbV8D27tZAE+jX33kTJqIMyaM6RvCfhNRkZ3cSm57Ob5CukfzXWPusGpJ0E+abSoBG4byNOtTMPSxiXo0wb2e2d\\\\\\\/vXGmrp2L7J+abM0k2p4EM3o\\\\\\\/8UyQy6fSILZObXu42vBeGkZmJVAaGfjRjiRnm26OopdBYusxcW5jeuHOUbIeyhfOTlpKcOG9XWMMxLEMImN+wz2WQ4Wzvebw1cNR2wI588nug93V+gfX3PW6xw7UBrh66SnH4WOdoTDuMVG\\\\\\\/kF\\\\\\\/64IYr0i5yYqqC4eZ9UrxgdX4zLrVvLZrutrI9mfgMmA6ofBj\\\\\\\/cjh7TMXmGdRDHOiA+T9GROuXfe7xBjP+9Zv8\\\\\\\/g8PBuomO24ygUun4+icbEbBd2F\\\\\\\/r48Ttm0+oVKl\\\\\\\/4WN1OkOZFCRawtgPZdT\\\\\\\/djNu25z5mqa4MubVakDt0aN7D3KjQtWjzuhNs\\\\\\\/c8ePhO\\\\\\\/dhSZiFswPsB0nm9aCY2FeR6YeHoWfRbNOK03i\\\\\\\/Ef\\\\\\\/sRh2vzGM+YXKpJPXY3t468QPeywo+3K21SD4VPvFMaL7FSReNf3UenszSInVadY7IXkHElWhAAcWSoWd2ef4wFkgyh0X3OaugBlOI0mKUCObl3RtBC7yk3v\\\\\\\/3AY1jU6sTDyARDGwoUzuvE7jQBketO8\\\\\\\/p3H0wglycLsOxSnGNXSQXZfDZSnrE\\\\\\\/q5AmG96\\\\\\\/Qnb5ix8OkcQZl6u3ShhPnqKUGjOu5saTm7he1BoRkVXI0divf9eVWyBlqqiPz67uX9zj199B71gqFlL9Q5IaLg8Je\\\\\\\/yhcJfYUfelfblJGUStMs0MoY9MNlhTqxcobAdplSZWFNKKScivpFOX1mqyfmeoTfgaiRwbppIWZavqC9g2uboD4XAbXLVsFF8UOf2hcXorJp4hn5hw4S6k2xRhpmTO4GQo\\\\\\\/BZQW1kHiKttunzDHf8iFELC3nfMwLp0z8g+kAy53QZTo9bZWWRei5NC70VuYdH4g9PPAVegy7v3mObH+2t9aFjWcatpglG8I1oTnlb3OSkWIAesc2JyDR6U88Be163AVo3B3HJk9ONJ6O005VFcnXaegvQ3vl2UoZpcYHH13tlQDr8QKeWOOUMLGSvg2Z16JJjuXAhYNzFYP039Z3cIMGtKxNofTssDrhXuaQk0ofL12BP9dz2l4kvE9P13mXIrZKYRvRy2gES+td9bh++gRa6CrUTxStab4\\\\\\\/WF+9J7bTyjCDSycWV5XPdLCpIL2D\\\\\\\/vUI6dFhXGA\\\\\\\/\\\\\\\/rAY1dv8d7Im6RCcC4LmQeyiiIkBZiBBnZp\\\\\\\/Qmxz84dHRVXEV7rd8JrK9+8FeaM1n1r7BxEGP9emNIMRWaWtfNIdfgMYfTtcYXxeJ8yYwkbrg\\\\\\\/4eQb\\\\\\\/4ukmHRpZ8wmGZJ3jiW2tsxwsSh0hPRiLkoljQj5\\\\\\\/EEUaVtpllXJgDuhRUYwWphNgmBOLLbvdB9HhXc6safhJ7GH4JGNjdXyXx3pD5vJKvrHeyXyeLFW1SPkjmuuB4iAB1sN\\\\\\\/cbzJHy+8Lf8kLy6zF3yUKhNOdht9d\\\\\\\/aMWpSq\\\\\\\/YlaPiZOgFoPHrsnnROTOhJc1Ntrsq7Te8ghQh2gq4YxrHOTR093\\\\\\\/SHjTub2PyXOo2558uxY5mpIFGd7L0YPrZ1gRDY0LbpQb5lOexDPYhH7Iju51Dky2rDtpA8GDXJhHtSd2jnFugAuwOv5nx8WldcEYrinRQeqBiXLvwbnKGoqULymlC353NG5dR5UzFKJGbxWA9BImOIbMIrKTS3zY9QaxI+7flewTV2HghUL5adq1PHhpu\\\\\\\/ykt\\\\\\\/RzP2Xk7S\\\\\\\/cNW\\\\\\\/tgdAULMqdIYvDyVgi1jVh4ECgFf10IQkYs5lUMFHWx+TwGBxdPAARU8ymHQ0SaOYsNaTGDLqQ0+5pQHyWo9OJt4v2pNgWJlaNe9d1d9kSOPMoMy82JdWC1zx68cbdv5wefJDJJT5bUihl8T1\\\\\\\/aZUaz0dQoen4RKGTY+xFslWuHCUxnMdmbAJHSw0vwtfABrZMu2XrQ4eLo2OZMSf8+iA6GMGhBYd79RVUt4efckw16bsexMbnvBKkik94d7\\\\\\\/5E4ptM7yOPXgl2m3UVS7E1SBQenIcVwFohBApr\\\\\\\/RQv89YiJ5C9M9oWlCkNTLj9t1F6y1K7PfmfuyE6cYunnDXZV4bCKCp7a93jJa58jiIzQccUIG61XkWfa7oy6cmCJw0IAXN3KKnvVDMHtd8KK1yFITxkxxwWdrQ2p7uo65WcK96YyMwlHiLEwK374SCmdk2bj8Ir2q8E6KxHZb2J9NudCG2dzyZ34E88RB1em2z0l1I2xOFFs9NaqLhK0mht2SWJw1wokF8WRE+sGE0zdpQc0uxHDfcTQRrihG2pviiP3cKlGHb48RbZWzua+aLPW3roon759HU9ak42mdwgTNTRdBN\\\\\\\/a5T7TdiTw+gvwjXeR2Wh648Kpit2ZEXaFTrTwUluvphh3YZmImNOia6chEJqS4ZTHoJSZPLD2cYsTfwWk2wkfSQcjOZXkFGYrzvk5sLd\\\\\\\/T7pJ8pxATOhEM67Y\\\\\\\/8aNi0G2+qlV0k+gt4pQOllCwDhFzPTihYrf53T3gzNV3pJQAaaLxTriShQPv05IhqVD5R6K\\\\\\\/dds+i6nzD0Vw3Ppul7eOvrWwRpY5\\\\\\\/u5YH596NXPITARNC1cbUX8lzirysXU7R4wXQRQoMxcgfmjREBXtCcD\\\\\\\/vLDb2pCXf0tRtah0waA8E0frhOs+yasj96xyRHC87iIYbBuLbvlsaq1f34I2HeIFurRMz6v6X1UFVA+nnmcgwTAWJMhGGYI7xmYo8gDw142vyH1hNh7bLRam2Dz6EQnX7NyzLcZ4FoYeAKs5d2FAiK1oSUIdeypudbPSUF6ut5tHw3OgZrHQlst3IBnZmTkbQ1WtXg5L9Qa8D7YZeUr4ZQ2HkCuMH\\\\\\\/MJ3EW1stT4iG6m20BMmLJmfug6ZOovq3p4ITYgsoOGfrkTH0ul0zDUBNViAenP264pAATV+NNkIuycuwp52TP3QEePoyCmrLfarY6FfR\\\\\\\/NYeJpgo3lVAcvpPmhCwFRXC9p6khA3hlhIAL9Hww4yzRgBmaHYpbj+JjWOwo76nKCK+s10WuzkmnpwZZjynkWJ3yMFXVV3WrJfeuscC\\\\\\\/VTajF2pZl\\\\\\\/u4Ek5lR1hx2DBFSmSE4Kcs\\\\\\\/bjyvUB7skifEv4o+CEyxTN\\\\\\\/1grsVG312XxanLuhkiz4moWVgUPgURvsQQBXBHhAvF3gmGpOTqAhyMcj\\\\\\\/eqIh5DaBG9dQbD5G4TtAzl4T\\\\\\\/l+XE6CmWlO3ONG7V+U7JQ6nxhgG0CSdKE\\\\\\\/a4trk\\\\\\\/RVB0ohIKv7gKV3tUFCtB0a0PG+qaYQmd\\\\\\\/wCa115wA6VgqkKP9hhdrT8kT\\\\\\\/D7u6Mn1XoJV+kFh8n+71q+6j9oFnFK8OnWxpDbqu0w\\\\\\\/0V1RQxuJ2bQkifvm1BrrVwBSU6VZe\\\\\\\/7Et53XhMCdmGnQhO7h0PUUfpTQEA4P6+npUJuq7uZ61obX8sP5e0hrPB2BzRSqXNJqSKtuIDt7TSIcZs+sOva+woezzUpIiH53ly2xu5JoDGUXKV1YjFpsWci7qdjP4IBGEjh4gbnmlQJeeW3OKWZ74sr0lWen2A+9Wve6lum7VV4\\\\\\\/\\\\\\\/3RP1l8UsS4fkvbta8FaTtjlu+xoA6MRVM6oMYupqomsA9UD4g3E1NTnEzV9sAPeArt6D+krsjUigzMVHdx0nAPiOhY8LJHq916n2mYjHUfQpRbv0GzmcdSz9GaXMbDtOfXp\\\\\\\/T2Vs+iyPzotzM6wJFbJ9CzbTevJ9Jb5gqLa2YbF3M3GtFPK7DKblWAaR75xoh5X+\\\\\\\/yyUP2wMPFDyRBTA1SEKaiMQabnBVEhlyy41gyftGJdA23uRq2zMgU1D4\\\\\\\/tMDxWmvebr5qdQ9XkkoGXTur+S0\\\\\\\/QjV3RS9rKVohpb7fuGQtEjKlgWMPlSQptIsbbHkOafT736FPdlX5tgOMBbg2eWNx6l4P71DOlYFMuSoF\\\\\\\/\\\\\\\/UZF0MdhmTa1WVJH1OewBPf0UOaN2mccePhX5pnlsVRPrU1cyCteqj26le0uNSh96G\\\\\\\/yXUM3AeQ8u37I2HlptEXfvgCbs5sZh8XJQnTjEAnBTeEu6uirCb9TWJUqHq7LEUhvylbUtitGqD\\\\\\\/5rBCSXGrYHqrUeDo3CZSRZ\\\\\\\/dqQ1ttttENrBw\\\\\\\/5B8P6UGxDzFegscafekG+QwXJ6PGjqB9fENgvSLH7hlSeIUDOYDNWMQMUvxOgD7QH6BTG5pFfblegx+002BcUE4lTrmvIqtNiK+3N4zenbl2jqRwk1ZBDu0mRyXIoDj43s7zG1ss1HUsREmed2awM08kRfGVuXtt+XdpKUlGByZIhqvCGIV2UNZtkMx0B3aFvdSRLMJcKHEIy7L76l8ph0s94V8zkeT6rOXjeXBSjtzW+yqXWu\\\\\\\/A2oVVHWByZtZHvOT+l4DpH8ie5p8JSFd7byXLp652GcOaKfQOvsNGoGR+58g4R8M5BU8fp9DcxmdCcT2jk26RzQjrCVXjoA4RWIPym6gYqcd1LwDPdokhoIwTzxhX14jhHDeSg6R+LyXjqDEiFDCJr\\\\\\\/9\\\\\\\/rTcvOheWP0qyU\\\\\\\/0b2Pbc3chn1UGBSma7tWcqgpFjzuTS6l1kJuex+O5gubDsnFr2CwrwTPZ4I5bAGpIJe44HZagITDg49+gPGdgBQUBG+09QIbCUv2wGrs\\\\\\\/txutXowd4wsATg30p4QhMr8mQLjM4GD37m4LPREB3esy6YoWo2Gne6brSBsK3TGb4LMBX8lRZI08dDkwLSROiXEcyOWx\\\\\\\/0CNiM0voOsh6qt+GYtNYjbBxseWVjFNRq9quFqohag1Dhl0PBwg4CibOHz7CD90F4\\\\\\\/Tih1L\\\\\\\/JPxsSzN6YNpCxU+pW1nCadVQO5dVoD+gbTc8btwyXpvb9QkpCRczumqv4+WNnWpKOifoG2WJfWC60mGC6iLHXhS7YqTWFKwd9prr11JARbpSzEoK+U8HTNdBFMMq4v0bFGHb\\\\\\\/uCsmXXJFpYkIlmGslDQpCds9\\\\\\\/23Ov3abHrciZ+HWeqqHSTuiisCvRF9hOaQEzzH\\\\\\\/m0W5nIKgO\\\\\\\/BWf8P\\\\\\\/7cSUyvk4VGTKFsOrJP88Hu7ytr33ijZ+V\\\\\\\/Aw3fZaIlCy5Qbg\\\\\\\/nSePuU844u7aVb9L3wx58xZXikUkL1tzqgDJh9KeJu1J0ciQMmF8fNqd\\\\\\\/cXhxSkpclV0LDAMeL3SOcgkgYsC+rc8XNI7bL0bJbRZS8Nb7A2lMu7xj6lN6hbGlLI6XZfIJWThp8ygeZRDopf\\\\\\\/JeDw0VqkCNypZ2MXVcED6hl0uQcR4t4eQma4SUwmQ5EBFFQ\\\\\\\/mrYcl84L4DHyUhBgA0\\\\\\\/XUULV\\\\\\\/\\\\\\\/xHRksIefHpQw84qh1FG63co7SI17HjzGLXArdfwM7Vi7jj54hTGpGsUc43HaaGbsQ50dCKXqyHFg+q8i9pIcfUoT9x5bVK2CiOjPIxrg2zjzxNQwgL3rnPcQ90cnOAf97FZJ9y7Ma9nY72WnXMLTpdSG1JdqcLusXtOz+\\\\\\\/EcOxzmlR5dIcuxgRfJzVhkARqOO\\\\\\\/p1PGz8CDdxresthLQuiu9JFpcHEKFSUTZbOnA\\\\\\\/oXnYwCYIH\\\\\\\/WvcjVQTdM\\\\\\\/9aGj2MMxtB7EdRiml0pctoRExsXX4e71CnpfQ5uZHQmcxDuIVlxwusrPWmzgs1tZVggm7daiwLTyTol0Ve5ogVvpRg6bkXYdbugEvn6G1SEBkv0ULVyB\\\\\\\/X75PrbnVadGFOUFrtZjieIfE1bHv6XxBVazKgFFmAJrWYcUpktMpNzq+JibIZlALPkgoWpGcZzX3BProGajO9npJc9pcBaAmQyiIeeyeStNkDsJwdSezENHrxqc+7c+9XIxAHHvtcQYmAJj0vx2MuiI3qX6XX3dWphpDV7BRp7TEe63XQfxt4u2dNFaBW5sVgYtJcSXj3fHFFU7EQn96jJH2++UKXTRiowLPbCVB832nb5doc80KmOTwuB6Oq\\\\\\\/ZkcvgKLAiBX\\\\\\\/Z3aB9j+Wg7Y+XwU\\\\\\\/FymOFZ5OAzK7e+KFxGSGRRmxVbzgeLGxLAMNEu\\\\\\\/OF\\\\\\\/3g3WX6xL867fn8SJG5nAScEqg6Lq1EdFj7yzCFL6+GagETOXGcmxTUwo04stDtFl7SuSGFPDI\\\\\\\/X3Lhl\\\\\\\/pKIvr37ZuuqmXDXVAbeMBX+M1jsljl0KzVgn9ffHslInLom16JrjdbBqkQsflI4a+Mt4H9gL9WyVuyTjYV9I2yIZtEb92eOVJpR86I5+MeGZUoQZc53CbHNFYhApcy55GaBOjSH2FXI+FHSzi\\\\\\\/AUhSDlyUEU7b\\\\\\\/TGJPD83hAoarAY1ign6RmPf\\\\\\\/8EAB55uU7iUwqiUeX4lXUL9mObeXckIqBVaBU+sGpjAb+Z2WnyPUhrlIsoUO1ItQHJJ1H3jf0f9CLZ1a6wwiLv579zMPuwnd4W9oh4yXsPALo4YL9RHmCjcUVT3zrm6a5IQtgddfo5XJN997BYkXYO73bx2ERfDAoDhGz9p0Hcrh1SfDlTWzsSS4Bp5oqH3uTjhpUJ3D9kj9qjP\\\\\\\/dIh5DN2iD6qxpbWc9Y1edbXYHO0nSGffhlnicVMBk6JlMbAuFSt\\\\\\\/D2QAetIuiwo6ovRuXFTX7id2HawFRLFcAxCiZSjLy+1VQEt+\\\\\\\/S8y0hUm2tuz17Cq0Wg8iWNS\\\\\\\/LIf9M\\\\\\\/GsR1TQVNxVIHAtbq6w43XcJ4fS3PFSu\\\\\\\/qm+YuJW5EBbN5udjduQiP1gywWeLTgv9w2xYTEpOykRN5MK8g634TRLBiTHlKZqu63eZuGdVHwQrckpdkwOVa6nn99j0fpAquuo1+GwYw5yFS8AKuRvp+0HZqczBpy6Jh9RRSGUrPSW8zSagJOSQ\\\\\\\/TZUqey\\\\\\\/DdgbowAv3G5KkssbSC3ziBEUqnVeFlDqgO1+flPkFa814n2cIDIIwhQ9NZwwRCbn2ETI59wzhA8qP0iDb\\\\\\\/ivm6rUaOSxPrat8wCTNVe8VWxyCPGlT8g5jVb46HL2OyezfhRBVTw5ZD2AzxIubaMlk9jq6errNWV5\\\\\\\/4atOModB6OpG+e49bWuFjdNfGvcgjSv3st1IfhqfJyspj7lfV5RvQ\\\\\\\/zjZ5RyV4ssgdQzakeJnNEWR03UBVpvTpChBDm9UBmm0OSXy2HGaE5wNqTcli74l6JG14xGHfD0wPG5zUBfOYIipIN3lVnh51+kfM5YpAD1fz\\\\\\\/vbxyr0eaI4uIN2astx+3ehmUzjTN3q7dZSAmpWeMUljRXbtNwe4LZPRztjqDawLJFG0ha3ikjwDLhVhDX3pB5DP34MRL5\\\\\\\/apm6i6Dqc9shyzNOZHORooFtPQQ8\\\\\\\/rUCIN+1ruOwKfhQZGgpCXc3LdDOXw0mo1UbV5dP5lOoorXscFwBteYhnfpGkoRZ53X\\\\\\\/e7sMzbYFZWiOHN0J+9EBlEyPpW6PwzSfWzmME01ETiFJ3NnDexVTstZvbPwhnULwwNcRsSjtoZh90S0fQJBX4n2VRGLPH+drBRtHVJm+KcD7vJFwRnXqJ8SLpSE5JpSSAKSVI4SEGYi78yX41kDuahrcN\\\\\\\/f\\\\\\\/GiXJ+s\\\\\\\/aIt\\\\\\\/B7ls21xob+HbkLavF5P2EtIAtUQeRkV0mMZQIxtJyFsrTB1FTvwAscmz2g6mUBX03TncTqD2yjuASnyIlt4A3xA84n07DJHpcg2Qth16RvgLc5OhkMQDndESo+OdHzvzaz2RUtDG7Xzsls\\\\\\\/2p0o\\\\\\\/Asl9jl81qruPh2uMyiIQpJ5JgcVmKahuoFfNirAsxPuwPB5+yOAfbW1slcX8nE38TPROJzO0wdHrsz1glz+UiA7WhXo0OYdrqF4asN8NxOLlOZ0xdONRLtmk\\\\\\\/0kVvs7pidA5L9kIitGEE8Hh29hxIVp9NULkYc4vd6x\\\\\\\/vJTpps0f4iNYCspB4JXCMtYL4qT8dtNRWWocsvInbpp0j8lfaT2V9o3u0i2B3y5njRiHN9SMtOTfEaDKgCY+u\\\\\\\/AY9HLglHHDeo5gqlYOXNL4P2qF+6nhiMqbTu9umVnTn6RhPPFvqZ0RTgd\\\\\\\/mQVMwkcSGEZzVxnat+R9KWAlBRbscN4VZ7ESet1SQfUfhS17S\\\\\\\/Npx\\\\\\\/qLLhmJzrtQqXx3zBraBMHoI+WB5g2dcKAPAVV8PwlIV5yDMrkf+3U6F+p2aosUJbU0WduFnj73uvcabWBxO6FO5dRSCGL92gDVqLApt+nf1Ve7R9uLj\\\\\\\/T5L5jCbEQVSagMMVwSla\\\\\\\/HkqeI17tnazCyuAXPEvdmcHdX+s8Tknjc1DWSxly1NFF172wnGw6\\\\\\\/saXivdvQgmSQpqPjfljMvzg8cWteejRcXeml88ysQ4Uk1IjMxZ7h6OYLTd7FYMqVRiOYfEeXW\\\\\\\/\\\\\\\/7ZyB12X+OJmESGyJdFMDQn5cls\\\\\\\/vccu8H1MV33JHnA2yETfl6Li290ZtyyvYhUvFXgLn\\\\\\\/Jl2a2ka\\\\\\\/ZxBPETa2KukyeBEPcPnVkgWtmOLO+g9S5zeZGTrkeI\\\\\\\/1l9E0zWyv1QvfCrxQJg5IKKRnA3wUuZQReTklo5g6B0Hp7S5bE2RpYV92zONvfnz9PWgK1uz3pXw6wTPQi\\\\\\\/GbyV4ueSQNt0ssc48x0yvWPWHizaYNWoZmqjVNbntyBprlSfaxAx6tUvTFERMvscy0NDCuFJ5JvYLzEl6ROD5SRRHddaz9KYu8RBwP\\\\\\\/6yVIZiovqH85XxPDC6IYqZ9XILl4t0\\\\\\\/xTc4QhAl5amqf4+\\\\\\\/LIXHT4zdYostK7v2TZqxnbMhnuGUbk2I7B9AvTPP9nyq30TLpr+eNSCC+AXRvS7A1GDsKB0Njk5AjIRP3EdNsZhVNtyuxuVfKYd79koZZehbrGuJKMhTpI8bKYz1y1Y5pRb6NJ9Jz3lUZHwroi2CWmh\\\\\\\/K3TtWqyyWXRYZuqlRtXs71WLAYzHyjt12jJ1YIrxC2sznDLKt4NGHl4gTVLg38nkZauO0PD62aYszZiBqmXyZ5LmvgOJHicM77lj27UWDDRLd8esL3zqjcGJs9N5QghiWB5WLB+Gdf0ftJU+ZxCDi9Mpuajj52sU9XTGqPliehtXWiVOAEhYrHPGDn84oT3iEO8zdg7E0OxID1p7abK8rKTqmoaMuisE4vcetjkZrR7epJ6u+iPsewk\\\\\\\/XpyTvhY1vpgs9JiTglHcIYYVjbgtJ7AMjNNUBHkBEEHK\\\\\\\/DF8CoIGPUJxFyM3x27ARlsTNBx4LMhV33+o9qHvGRR3c7oeH7rLliQOkMCJdshS+rRV+F6RT0vttcCXsqrYh3Nxh1mqbJhzz8Xfg6r2cdxOjmFWIOkfhIkvsdyl3eQap5oLbQASWS055J66LqN9l9bfRkObsC6w7S6p9\\\\\\\/0XKRZ8EADuyjeqjYQJeNcnkM5J34DQ6xfm7VeGPSRdVSad1zgCZfl31cdU15zji98g5RItq+NUEReC4YWfwtd38eZ3QB7JOuuby240do2f3hL1OZlXSaa2u5SFM88x0Ni3YBNuWwYKGd1AdeVztLcYzJVh2lG4TqZhvRRurSFgijA3Qq+slEbA37z7Sq31jsuhM4DzZXe6R0pSEFdXLZaXYkdEoJOayew6Zl6bv+1Fin5PdQx9JNFbDm75HYUoK2uqKdwlc1vftovHrDa5Hhm77PfcVeBN+Mzc1tsjx4Ri0aS0imn2wpqPG1A\\\\\\\/bvhia0mBS+C8RGwPGBiWbv6a3Rzdvpb\\\\\\\/ibDnjTsglF6ZVgSuiKEqtD65rGCsGjnO+t0Orfl3DmNPPvBG5JW2eVoKBvU2+zM\\\\\\\/AnQc66kc0+nCyHuWhVm5EGz6gO6iYVi6Lf5dLzwLGm3VIfxlfd6mgGHIRQNYnagyFYBuh3Ap8+lwC3FbpDYIUHFi\\\\\\\/t0sxML2xqhio7Cfjh1Xce4jgfwAE+B3DJitv7gbIrnKhcvOoZH5eXkqBfvT9kvGcOuHi\\\\\\\/mbmOmzSXOcGTec1U6l4c+2D5ipf0bHmnbJIIiNdfnaegdocy474cNoa7NyySUfOkW509CWVxIRYQJHAOnLc5eMQBIy7xja+dm8dzHS1fo65qZfiURklGUencqP5HiHJJ1Bx2OLi7yyIQX6CO9CDP8b2tRRFlJQ9LalXtxLXU9S\\\\\\\/t8C01etRi0qX2xys9Fng3CmLeDzihPEo9vWg\\\\\\\/\\\\\\\/WW1xCaaPOZBWrnjAFOUOZkOhNa2NjqgCwehEB+uM+Vnr2Af0FWqW2nnllF1zGwhUm8Uoqu1SlOMsmHM6JmApBjL6vw6HFkhuPQBC6e8cnt4xK+pn0y3zF\\\\\\\/yZDdh7dl81hDIQbdmreEucV6Ozm2aZsRZCovpgm6IBrlUBPVWSf4NqbJTHOMSNKcpR2dZjo6Qhva+l4UVzQ87hdKw7HFVFrIrOU6t6L1AMJWYET\\\\\\\/n4m+cUk1i0W2XI7Bx4XNoRvPc+BVTQmy9mytcYCuX7NeqaGT0r58ciz03KboevU+xTlmRgIGA7QjTccfnhlDbCsM8Mv4kWxBW7qEfAKnqebbs+pM0Y59vcK7drkVOtWvXVz3k\\\\\\\/xomQLnnT8X23Y3Czqht3DcCqRmjgHnas+uVERhi9EAfdsP8WqARYWN6uAIhSY+1eyrvHfOSybQ9JCLykxnpzNUrD5ZdFOgAY0eMx5998XlU+Edo4nRBDvnW+jS60RzmTy3zs5uwNzT4mQtD+LAnUsC4bbAnO8dKLLjnCXu0jcWOeFjVMA8kkYgy3joZHBkDQ0EMKOOLMN5MPMUlOgA4+MKyWLqQ4z\\\\\\\/bZKSr5fvM+py3TPp5KtdlMwv43e\\\\\\\/VdvbornOdtPsBuIfAv0y2SSN7P\\\\\\\/R7SieiE\\\\\\\/e3qGUKVHEnggLUx7NdkVJBToNEi0rmyGtqJGwb+UOPTpWjZwSRFMiz3G04a8bNVaF1mjXibsdxpJI0LuoDHC0B\\\\\\\/111cOy5nU+mESjMGfbwVG+RXAYvJJu+uvyP7fTtK1IHwal9\\\\\\\/k+rrYsgvAG4jYOW3mVc5rWOnQMbB1H4h8Q4yPumLoacLx8nPgCkZuDNONphfVPbTPKOXpcJOUQKtyXbVbae7Q\\\\\\\/9WB0AdhSplih9reNlR2KDXMdkm\\\\\\\/IuQzkdwkIrxe+xdxio7h2aPcdqIKi5pheU+96r5JVUpaNuGbBWtzoVi7GBjVdB1yApZBMB1TNdLN36qOB6z12iokgAMFhz5CeRVhMa\\\\\\\/8Y90KeMbtSkQOWpUnwobKcX+KorxxE\\\\\\\/TS94hAsaGQVJdJmX59At7j1FOFcTiM3z00xL2J01Dvo1H\\\\\\\/RSE+Cu+Sck5JJlAJESgqayoy2O+MnNbVEFLJWHIQNtj0M42h52bDqzJHvjYpTmxPygWiwiQncHOe93ht5tF9IKPX2x2VD9CtYJTmuKTM\\\\\\\/1ps0onAq0cboeqv4\\\\\\\/NJQ4Kept4fN\\\\\\\/bgSTpv07eVkc1R9g+ZwLYFD6TmodVAo8qeOgZ7xfe7\\\\\\\/80CD\\\\\\\/3RqG0Yh10SVO2q2EdULqvbD0lXY7PVzb1z9tEnFIE9OAJ4gx3cgNo0ZLeB\\\\\\\/VUqrc3O8QfkmNoS7L43KMneSBoNnPawmxuMgMMRGtstqZEy\\\\\\\/YlEi4LF85YyQ+HEwWkYxhuFrUzFmm+qG5DjANlI2f2gGV9fOvy+l8\\\\\\\/IIMtaCizNCKj6lX+Sh2j+ocs2KbkNxS685Va380C+RdV1JklPIPgQ9Rf+ow829sQL5qd5jCqamra8yIJPOX1njmTujEHYDHOpxyhQt5VOJ4iyHxGloJR085SK7ixnOmCVPGgauFu6anUENea\\\\\\\/lfaVhyzMC1nelE4mQeWmxoHBSOrPAPpT42YXVYRyxPaVd3fM6OsUTIM+CzxswHdOo8KuqGJXkWVSYqsmniKEmJWC1CWg6beXIaBfTSNHVao64A9Bmsj7mgSdZh691xCK4hqiA8jtJjZhfJmQqaZF6v8vJLY5Tv2gO+t3ty5NMpQrdiHevzxUOiNeTzGwvAxvTXKIqoJSBjc0uuqlUyxoQR+DFibrlxQ00V7adtaFvT9c1VLTMbuTUbBaPhttcdrrDcNZ5tPbQSD6bScyDC4zkokzy1\\\\\\\/tG4ExEuX7q2GGBHpaI6GutE\\\\\\\/lcKJwLZFz\\\\\\\/aolwwIUEqx4Cu+dA066jxivXH\\\\\\\/ERX6NsWk8LoKcpc3TCCBc907p+8InFSqQcui3p5J6vBUC5iFyRDpL3NO\\\\\\\/6ojIJG9jK52STfFgd2VVjRO6\\\\\\\/NkQWGcNM6OW25akVmIY2WTBmXs7jBsqSS+xfX5T\\\\\\\/SmLKQwQIkvDR758xDSdnbt6Z9ZHS06LwMpnXDj9g0XP1quNMJh+NOjuVGUBNv2NP4xDlIKrfBbQqeBta9I43wlYYCssWKXMdOifoNFtFWZqy3gta1J3ulRnzuvdixkVCR6wYrMVAGWmSm2oyDehrjSw0q+i9qSRfQrC35GGdE+HCKX3Xm2RlTz\\\\\\\/awx286iuWk4ccoaJekbpQsdCfcgQ8ueY\\\\\\\/UX1UCuRpDFNvLLXgEJmC3HFcs+k+C38H\\\\\\\/oNtV\\\\\\\/DtLmpGSQIPx4P0ewk9188yIiaeGK0XteC2PnfjoAVnn4\\\\\\\/J1OnL1Wlj9QAIlhoc9jaI9e\\\\\\\/ME94hS\\\\\\\/F6mY4LitO8Yyqu8X4mbrrDtBSUP6LW8b7sbxw4Ue7sPzyCNzTxWmM60Zu9rX97oHdjyOyupeV1Tuaye2DDWqmyg7MbOPkY4fgoL9UPwtRWu7oRc2QirdYFD8gn5O7oz05tpRzTR0Na16H7C8tmh85oGh5KdPgTT0TQ6fWUaHgnqateDWPwzVCdJMdlCOsWb527Yohq+EXb7VItnLQOvBsQr3YecNBb4HO5BKzPAO\\\\\\\/73fUXU6FImLeOQruL9iViInyWIm1vv\\\\\\\/VW1PUAWreuvooyA1zCRQZ4+ZjES3hbIUs6f3R+lUX4VdMdi3FthbuborO+X8wmPqzMHp5mzoQZz0t7zUd2WRuWCTPPuOvMZNhHpcsF3yH7Z0CZeHVvIhjgRO7YJ9nVM787+dJaXAz8Zvzobu65BWcBRhpBCtNr5slkfCCIbvamLOQXFbIUvvV2h87CLXDv3EU6F2gS2pYvZZlATSc89CGMtEQGmetEb1R6NZE5LLLQUtAa\\\\\\\/x7T5bBzeALd8h3GU3ydXaYCbWEh8rIE1jzgcG7fDNLteUDSNGNuvet8\\\\\\\/KoO3zRjTz2gKqVmFubls6Ocya4GGhoTfZAO4IL4JSFYEz86VrIknSqmFE8tIMuzi\\\\\\\/3P01L\\\\\\\/Y9oT4hxYARxjnx2Yq+FB\\\\\\\/fJNuwtMan+XWV0S1SCUPWUitYO57WPPjYJrZ9JbalM\\\\\\\/JAYc7XUm\\\\\\\/KaQkrEBGNCtMndlDIwSs5Ie164iYUOf1xBfqX0pEGhnAyb3fWVIfndHd7acJU5TSsdbgg6AQIwmx5WuVOLPRXqlR18F9bojP5HPyATF3+slyL6qvHJI6WL1Zpx1VT5hd8VScnwMW8FUKYclwCb0qn9kfbytaj0yLGke8Au0BAgQ3nqyHHw4AsaLKn2zxnps0gp5O4LfS3e+5IkEQuzpf5VxxeKDkAYGDuOU+8vcA1HmKCS09rV22Bl70R1pYrp2LxomhCkVaebtVobatSTyOKIhjNMXlV\\\\\\\/P8FYLtoBjpO7s7heEYev198VFhPwtXSfd1cLO+l5R0WM7GR0s2HwuLiRHnFix8BYFjhgj44SOr0wt9\\\\\\\/tqD2UwvRnO1McI2McaQe5l\\\\\\\/2dt6e\\\\\\\/DJVj5eJIJ6GAjL+ZyXu++crGQGawTd8ML3eapYJuOuu9eUT4ah74vcIWjNCoIPG0zcXSOmT2EywwJRfE4P0uxV9QSbTWi5OwgHAIkSxUIHVG7nKXdXbckX18bmdrGKwQdMXq1Unnx8g0ey8rocDgeBgd3Qky9YJU2lGiA\\\\\\\/eFjZTSWyFzsdWsgyFK5jt5kkJXvD+EaU4qC+yxAydmnAS15ufkKvW1vJAG\\\\\\\/xLJVOjOmHs+xRtzdHjDE3hMJZ23LsEu9LwJzJC4\\\\\\\/xOxQ8if1qCOuIgUC3fKJ3yAjyNcDMKW2hKySr8TiJJYEw0ouWC4Cncr1WQlepAm06a99Gw\\\\\\\/hVBjB2MN939bq7boESo2I4NUHP1fL\\\\\\\/V+hrZzv3YwIutURAtsXn8qaouHOkZmHDvZZkWLpsjeF+R5eeaPRWMIeTyZZ880Se+7X2B4Qdw4cNvWuMWgkN\\\\\\\/94m4Sc0LZaj84cv3H94LZC3nhbC4ryC\\\\\\\/LfppYU9AgtlttLtnhRxZN8A7DOTtLatejDacvXUH9NOtRZ6v3SfejiAjy6tWGuglS10B9IrBl5tg3JrXUnG2j+Bw3L8\\\\\\\/miYRs6RaepDxcCSfncBGRqnVzxtujtpA551pqRa6+MAgOPulGTg1r2eremFqgp0s96BLpHw3Q4rN1xF\\\\\\\/Lsq582A1PSpNO39SN\\\\\\\/rXHS+1ybvPMkEuI1wK89WJsyJU6Dwwm0hmN5iXJsXxoTrQF2b9gd7LG3B7ZhdoQmtqRGHS7pCZ+PeqlwbZzFnQ0OrQ9zeQfEZ8qXiqvd4iZO1qxsPi5wqQQd2ZekxgucFm8JoNd2hO2n8J2ZnUD313AUNS55rs+X+BRjfY6i8RINKD+WGUx6ssWm5PJaEgA3hSpkTcatiI5y5FBk0im7qr0gZCy8\\\\\\\/xMbFNBFSdUEzV7GfUARv0Yq7BACyQeCDpAnoN4wyfjaMArUrLtupNTeiKFyz5Cc6VYTGj16ZlN1FFzH8+nUPDXfICz+96h7hYeFVNrL+Vj+scfgOkC3z8ncqV61zi6p+q0punm18oxn2ObYGX4yRY8Ny0Z5HSzfvKfvvzz7MppU9x4TwSs91lKpQHN9rPskrcDFY8kjb0xOjNcx7ecfRSZ09XAkRc3agvHKBzzEg71KjHM3qtJGAw18K7JVt+etANxmU0umd\\\\\\\/caRSlsX4hwU5b9Ldg4uFCJ\\\\\\\/iGDvq9v2xk\\\\\\\/hVc20EUCIACRJ6wT2viGOD8hsYuaLe7T0VNdq\\\\\\\/VBu5SP1iXr8kB6LQNNbnqXreYwTH\\\\\\\/n2ZUjme6kO8II72\\\\\\\/+55q4TCS5EPlWkuWTMsoHhGDOjc4AUVHRKQD\\\\\\\/MR9YOxeJxnW+dtijhqUP3JxiBL5qfmDw7YhWeX9z\\\\\\\/UVhLwQ00B1aErVJgSvQy64fjYzqDb3ouhnWvK3GkibRhs+A+FiSks5w1M09nZAnuWNWepYiQg5dlwr0AsiUibXpVsBaQ7UYu+m85CpmIT0UapPHaA4QxSlIh7wtYJqmWaAfbsSVMAxvgBVx\\\\\\\/pJQ61+tUyLFZPXLkTRW7tUtweOOXhuZiI83bQEGyX0iVW0mRmPn+P1DH+MjaSE1k1qEVija20Qu1ZE4lMjyn+rKeAuLZO+dBnesxRvu0c4Eb0DZdVnoIX6YIyl+qIjqxTyDXNjx7zITxYQTiBadRlL6eQWEcw9\\\\\\\/yw1RlNrxJiBYCLcjHm\\\\\\\/0u7EJqAk8Ub8mMYl5gi1rakdOcQVNwfoM\\\\\\\/mYxGUJFxdQeyXEmXesw1E2FwwwJbJy8DoIc4sPgI6Y8m6t8pIHshrr7V+ErTrS4r9cuRV2SWB1CFR4xGolh3PQ7HeDzJwNCWT3If8B67Vl6MnnYcGig6vcJuFqdjPqJiwcBxCQgj1JH+GTo\\\\\\\/4yMKWu\\\\\\\/U2QVuSjcoISZM0JRJsjoNk2juu1k1rPsG\\\\\\\/Et9rSsG4J5G9jGD88MFYaOZHfec+ZAmUW9kesYDZV1UtZCjeLi9pemwpgaOrFurjVR0hbALHtEjLXWcTTtSqGlmxheylv61Wmsr7oMm9\\\\\\\/4PLH8Jhw9CkvKb3sL7DlKFKNkk4lusk9+bOzMqJRI6eyX1DqzGKn8Qk1+K0lLVnPxwQdUuM1VVw3OuKatsbCXk+1XgpJLHcraTN8yJRzkV11q8ALhWfvZVh5dvRqQBo0cI+st4yoWLOZvwl5mPZYqPq8rdm4amdzF+6lYWoVv8rLes5DKXpf2WjJgbvnrPUV+kmamRj6KEOn4F\\\\\\\/2fDT5zICyNqdHN7eKrKQU6uvk4Bj5olYK65vMU\\\\\\\/MDIPXu6B7DbrB6sfpSuXRn3mWFLFmkJlZ7zWVcxM6k+5b7XSNlO2VW6fpyiVgMmu97bbWmMeR5Gi87p4lc\\\\\\\/e6zIjWtBcc3mI8oEKWYat9l82h99kdaIiJqjh17oLp7eMEIcuso04DQ7tC8OtaXnVYfnjo\\\\\\\/K8rUrRq0R2Tpinz2ESBq5mOMsA9r14jI7O2WDj9b8iHKzKBacwSGQm8OYqoLEGoHMolPOeeEl9lGTsoTY0AETTuCk6EfcuFbD1flqeX3q7ALqUJol7\\\\\\\/qxkT1TeOXx7\\\\\\\/jdhaxLFpGoQMKjm+gIchFc+PQFkSbYhJDw0V8bfQkFDdBxW2Rbu\\\\\\\/Zs3JCEptfXCf7WJEwnha5Dz3vKmg0o7ekemQvqGLAJIvJe95oSuQBNYMAxjltb+vuHeijvF7MYy7LMcjoA24jx2UzZmIEpmYnCkEIXG4vWLhJIGA1evg9Iahc2lxw6V7hfZP+3fJZI4sOJWumRbymFwrOd6soRGorAdj3IuoMYeaIAzKNYfX2\\\\\\\/3bBTLD8VaJuY08qVF93QocPc6v\\\\\\\/JBRXATb7dAt1RfoakK6SbUslaXYkt0P+NCBOk1P1zMh44IyC+APH2XK13XOoA93wg02jaNojqhovoC8KLeDERweFHb\\\\\\\/kU2itiZaomk6eBss3ti2B3F70SRh2xK+0k5BMg276GrdejOiNpKOO75JB3QBQdsrxHkLbzB+vSbSFLdne\\\\\\\/9APYZPlDAV\\\\\\\/LmcdCPfq2pJjY3RrAMMp+jVZIIMqUJ0H043QKQ+b8QZxSPGcw0q6qXAiAt15ktCpMoelksVt2tU5kvH+hoppu7PDeHe7BwGGpLOVj\\\\\\\/\\\\\\\/vUkxNuozKBcQD1RMyENCvUvdhJoVl9iIJKmOFZtMSilrDbayCCkwRT8qvynHsHfXHa5csiH2NdkKHi+zQ+hjjPfZzdcnuDcu7FtPZzHXtUKMTdRIvLXyVkqbG6CG8rOlRNMwHVoVUG1duvhsxqJ9BmBNEjxvJk5aWQ09F2LMeULRWOYsKLu1oMJxpXFJWii0EU\\\\\\\/ycV4i5LPOVG1y7260pBblg+V+YZTmrxfPz51GJdOLTCFsVYJzry1xqej3M3FSnvjmhmcfOC8FaRf4P5CODWm\\\\\\\/7np7g2FsvnpK1WFIbRsAB178EaTmF426xCBgjWVh\\\\\\\/s5FgAmYw6o7pQN4scDgKXAXXS1\\\\\\\/JC5ElE7ds06Dln+rB8UaF+qrt3wwHTKf6n2WXu3TGXsuR9jWkUaOgZ\\\\\\\/qjoJ8Hwwd1Kpk3IcPmsUOnCvejfcZoYHzf+PotVaXQEgAQdDLi9Xx9mafrp3dtGbH\\\\\\\/lYIg4YDAHeyS198XFg1chE4UM3eQ\\\\\\\/UjTaAXQkWy9\\\\\\\/FP18+ze3XMaEK124WsZQntyR9e72RZrUJtqx+fKgqoIl+p1oGYuGulIJQaI0eCpIhqanNIPD2ruwtuZyoE873J6kEqpx96Yz8G\\\\\\\/jvTXrlSh4SDKTQR0vPLPou+YWHZXmxKKlfe8TUHAUlc4OZpqRKD8gDu5P2sgl00gDnOoZgC4sXya29tvKoghqUHrbfaCDdyaCDCSSLDDLZuILMJhsNZguvqXAxRbPYFw6e2zQW21zBYmNCarUWCjSnSg9ByJdlg6ZVkRsrKSW3Hosy1G51Lg30batlHdGttw4rZdPdYUJi0NXkroPbrQncwR2DBHahEf6T9BzyIfl619CGCXp9+jga+hs8N2yddLOHIHE8JIpWoKmqvlvKXDA3qeLRNjUvWQED23poj6gmwcFMxjKXbUNivQocSfmipqbG9rQzLj+dlf+kkZUdfYsBYtFhwdpjqU5ntR4gwgN\\\\\\\/Pcz1yEjW2Pwu0bv3k17KamzU0QhQHCoo6VrxAu6H\\\\\\\/zRR+oHGRmthq\\\\\\\/qh5tcPB\\\\\\\/KyRzJwi\\\\\\\/jdCUTapMo5KI9tSTq+A\\\\\\\/mQaCeXvXgwW1aDBkqDq98BjBCJ43MhM8xDOpWGztYOZ4tdnb6aoXI80zFzUcd+Hhpyev8qRDkRBhwP6nVBoSTmFVVdVVvWKC2yP2MxgC82Z8m7Tba0pq9UuXqEyviWV1yZ+4tuHSOC3W2dY4O06z7W\\\\\\\/DuaW9in7LXuc12hg0P6rmjSZGvIl9FiAvQ6sNS4lPkQ4AQykzBBGm0bMEgIVYqCrBbLZYW8zaglFi0207FP9Ae+82I68ewXFQ8i3MPI2gcw4uqaNYipMME+k3WUHNkP6qvQwy9FjZho11DqUupjVwMeFT9qau2wOtlFSCR8dhHh1f0nZBNeU+qT0Dk7TI0csavVPEMHeTBilMSh61goPCnIjNuNzfIAxzcC2ffUfWQsG2jUmHBxzuKTgF6Gt\\\\\\\/WBXDGqOFpj7jAI2OpAFZeTcd5fjTJadhn+sgwlSPGa10oTF770MRf7obL3HkgGif3mXmgqR3\\\\\\\/r7pHyj6rCrxN6pvzjiILUdp5vEoIgKQj6VdAtOZZEM8vYNtoHC4Gw3SrpvoF8v\\\u0022,\\\u0022iv\\\u0022:\\\u002286XHYVPVNdXJCnwxpN1V8g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022d934530e3837ec05a93089c910fdffe05d35b252226b923ca17acdc0aa6237aa\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022vFeIv3oa8FFm92cOHrnOPVwQFZw2g\\\\\\\/inF90t7HrvFZvN\\\\\\\/YPueQsvF7ZWJrzJ\\\\\\\/HTrm2+GnxGlzX1Lavh3hvJ6JJJwT1mI1vw9ITC7XDzN2yZE3B5qz4ryMJydFqzkJYKpjwN4nAvCIzG7qW6jlFMd3yG+56G6RYr6vi1yYjU7jfqixdMEoovwpSzDjndAvmLx9UlfJJfxkqbBFf3ZmWJxUwmF6J8zefNL3MTrlzc9R0oFkKGsjYugtaTkSkkfypPUPcBrHKJv2jbTycDxe5P3fEvXyizO+XY0Kqot2z8ZaTdZF0JZlZVVOzZfN7K9zCHDwlN9srctZdqaocTjGN0aILNgY5YDbFsFzSv4uJPpcgjP2\\\\\\\/A8GsTQmXm\\\\\\\/VcoZu544dUG+yc7DYN394Zx5D+DDdcVbYt8EIyYU2RCCDxVZODkS47OLAzXn7z1NxG9M5Eto0bVhNobs6fdu3+yIuLnMvXr4WMPCwqeMT5fjV0gYsN05\\\\\\\/RlUVQjNhhJM\\\\\\\/q0xwiH4nndK4kFUHQST3ZOiqXyrXvhj6Oq3enI3I0xy9dYtKbdj8J\\\\\\\/5xFD1U+gzt4qW94ACGF8FrAShg60atYgTTryR3Y5jUhSj2AcyvuBs5wIgb2r\\\\\\\/hP5NLSlEaRLZ3t1prQCkwDHMy8A826d\\\\\\\/8R6Bdn\\\\\\\/lrjIYYhIjhs17bx\\\\\\\/pFgJduJCaIvV\\\\\\\/R7zih0iZDNCqQ5EYYu4NiJKAt3PltJtc8bJyc33lj1FOCY5fbQ\\\\\\\/j6cE\\\\\\\/wSsywXRlM5HxfeJuI\\\\\\\/PhHDWTSCl5pTbWWw1CLT0hSj2tKUPnygRbtOrGc+oIDuhUilrPPCgeS2TCwTZ5zN6wxgFgnf\\\\\\\/mcaO0D26H3Vg6fg93kfScQsmbJ1pU8BcL6khyUc2ECJmFLhb+gHGZim79owYLn9MBMuhkISJwhrcbY5Eo6RAI3E6+gzqmyEp4T36B0fWyNrtf8a1cXjpjRsG4jJohfIUThOdSS0yjDR6tvfQ01TW86JsdP9rqSKYyFbspFfruonB3hyekayRs5kFjhOTAVY2VM2wQ7PA5GQ2AnqvFkdy5a2\\\\\\\/ZDe1oFRNtJoPuTPeiiZrG\\\\\\\/07V9+t71pjya28G3xtTNmLzpKe8Ph9adYHZdHfgXl3UVwJ8U8mzQwDSDXJJ+O+vP1bQOKyltNl3DpJ9wJnxHM3OoQDBYqy4E6Gxf8Z+ZCZUaJZT6f7IQsEkK4TkISH995m8r\\\\\\\/npCoyCvj6AEx42KOK3w0BwlqXe3z7tRKAJbIn8ep7r5V3ZSccZFxSBKvcc0ja2J4jCV1ZSNutOMeoxpfcbjCee5UuDpJmiuWu8Dx6qyBqKRE33lJez3Kp\\\\\\\/ZtIEMgUEi\\\\\\\/+nBordoT9IFWl9IVeG7Z7inTiDTuYkYGp5kWwVB6sGD2F+13DrlOIQcg5q8LeZ5NwUMn1YdVOMTweAozhAqa4u5ljUO43H5AUAI8qYj3fn1Cu\\\\\\\/Mmaq+Z6v0vtqkBHn76tmfmq9ADYsanEh\\\\\\\/udFFyJ6RLA6a7YsIgs8SbqewVVAMEKBAHNZZBZZJ1sup0ee+UCMVoj7zXUFkWhfHWDthNxXdTLr5XVZOAaR1VV84mTWr33g03uooi8CyoAG9j09xepPyiXZDE4e5hvxvmchKmn6ihb9EXZJDwGlHwpMoGIukaxRCJaOqd0AgeHtZYqmFrvJhjLkxiCXpnQf0kYsf+cmicSFbrqZD9PXe+LOQshJXraKX\\\\\\\/GL1HA\\\\\\\/D4T9pCxFMoW+2KFzEoPb0S8IgOtH3ctfi9ESX30vZKRcGO5Bho41nyamK2LM+AjAiVepHursMi1Z\\\\\\\/JicBkLQn3TPUJyDTtiPJMvPvWQ+UyLdtHHFNsU9klSIgDGJBn86Cmu2RO84yTK2bNuLd4aSDPo8mCCG1FVS2dFACqxrfoitBTihFUrH38pRROEgNbTRtTUNYEvyQL6HmKK\\\\\\\/SQdK\\\\\\\/qIk7rtiYADNgOLGxD3RbAGWiL5PDr9wfbMjKKcaV3AOHFGJL8UHotHApRlMJi3kl4sKJH2+hnfceHTZ+4S6QkuKUGMFTG0BhiqooVDcxfk=\\\u0022,\\\u0022iv\\\u0022:\\\u0022jug21hYvfZfsWR1Yj873Yg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022bcf637d446bae6aefc7982f8bd73bb260724610d4aa8f373231ddc9b205a20e5\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460155 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genomic-selection">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/10/1991" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-U1GigecfKycfmzg665-puImejXclwr7cgC882r4KHCs" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/10/1991" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-KeUHgG13mNMqOyfRAhODAzImlfXTvXJzt2HwcaZOScQ" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/10/1983" title="Development of a Tightly Controlled Off Switch for Saccharomyces cerevisiae Regulated by Camphor, a Low-Cost Natural Product" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/10/1999" title="Fingerprinting Soybean Germplasm and Its Utility in Genomic Research" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460155" id="node460155" data-pisa="ggg;5/10/1991" data-pisa-master="ggg;g3.115.019745" data-apath="/ggg/5/10/1991.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Comparison of Genomic Selection Models to Predict Flowering Time and Spike Grain Number in Two Hexaploid Wheat Doubled Haploid Populations</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Saravanan Thavamanikumar</span>, <span class="highwire-citation-author" data-delta="1">Rudy Dolferus</span> and <span class="highwire-citation-author" data-delta="2">Bala R. Thumma</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1991-1998;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.019745 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460155" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Saravanan Thavamanikumar </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>CSIRO Agriculture, Canberra ACT 2601, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Saravanan%2BThavamanikumar%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Thavamanikumar%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASaravanan%2BThavamanikumar%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:saro@gondwanagenomics.com.au" class="" data-icon-position="" data-hide-link-title="0">saro@gondwanagenomics.com.au</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Rudy Dolferus </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>CSIRO Agriculture, Canberra ACT 2601, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rudy%2BDolferus%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dolferus%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARudy%2BDolferus%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Bala R. Thumma </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>CSIRO Agriculture, Canberra ACT 2601, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bala%2BR.%2BThumma%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Thumma%20BR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABala%2BR.%2BThumma%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/5/10/1991" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460155" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460155/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/10/1991.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460155" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460155/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/10/1991.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460155" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460155/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/10/1991.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460155" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460155/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/5/10/1991.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460155" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460155/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Genomic selection (GS) is becoming an important selection tool in crop breeding. In this study, we compared the ability of different GS models to predict time to young microspore (TYM), a flowering time-related trait, spike grain number under control conditions (SGNC) and spike grain number under osmotic stress conditions (SGNO) in two wheat biparental doubled haploid populations with unrelated parents. Prediction accuracies were compared using BayesB, Bayesian least absolute shrinkage and selection operator (Bayesian LASSO / BL), ridge regression best linear unbiased prediction (RR-BLUP), partial least square regression (PLS), and sparse partial least square regression (SPLS) models. Prediction accuracy was tested with 10-fold cross-validation within a population and with independent validation in which marker effects from one population were used to predict traits in the other population. High prediction accuracies were obtained for TYM (0.51–0.84), whereas moderate to low accuracies were observed for SGNC (0.10–0.42) and SGNO (0.27–0.46) using cross-validation. Prediction accuracies based on independent validation are generally lower than those based on cross-validation. BayesB and SPLS outperformed all other models in predicting TYM with both cross-validation and independent validation. Although the accuracies of all models are similar in predicting SGNC and SGNO with cross-validation, BayesB and SPLS had the highest accuracy in predicting SGNC with independent validation. In independent validation, accuracies of all the models increased by using only the QTL-linked markers. Results from this study indicate that BayesB and SPLS capture the linkage disequilibrium between markers and traits effectively leading to higher accuracies. Excluding markers from QTL studies reduces prediction accuracies.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/genomic-prediction" class="hw-term hw-article-keyword hw-article-keyword-genomic-prediction" rel="nofollow">genomic prediction</a></li><li class="kwd"><a href="/keyword/snp" class="hw-term hw-article-keyword hw-article-keyword-snp" rel="nofollow">SNP</a></li><li class="kwd"><a href="/keyword/biparental-populations" class="hw-term hw-article-keyword hw-article-keyword-biparental-populations" rel="nofollow">biparental populations</a></li><li class="kwd"><a href="/keyword/genomic-selection" class="hw-term hw-article-keyword hw-article-keyword-genomic-selection" rel="nofollow">genomic selection</a></li><li class="kwd"><a href="/keyword/wheat" class="hw-term hw-article-keyword hw-article-keyword-wheat" rel="nofollow">wheat</a></li><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">GenPred</a></li><li class="kwd"><a href="/keyword/shared-data-resource" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resource" rel="nofollow">shared data resource</a></li></ul><p id="p-4">Identification of molecular markers that can be used for predicting phenotypes is a major research area in plant and animal breeding. Traditionally, QTL mapping has been used to identify markers linked to traits. Another approach used to identify markers linked to traits is association mapping in which populations with broad diversity are used. Although these methods are useful in identifying markers linked to traits, their application in breeding programs is limited (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Bernardo 2008</a>). This is mainly because the individual marker effects are often small for predicting traits, especially quantitative traits, which are influenced by many genes. In a landmark article, <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Meuwissen <em>et al.</em> (2001)</a> proposed a new method termed genomic selection (GS), which uses information from genome-wide markers to predict phenotypes. GS has been widely used by animal breeders (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Hayes <em>et al.</em> 2009b</a>). More recently, GS studies have been conducted in crop breeding as well as in forest tree populations (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Crossa <em>et al.</em> 2010</a>, <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">2013</a> <span class="xref-sep">,</span><a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Heffner <em>et al.</em> 2011</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Burgueno <em>et al.</em> 2012</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Ornella <em>et al.</em> 2012</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Resende <em>et al.</em> 2012a</a>,<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">b</a>,<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">c</a>). GS is currently operational in animal breeding programs in several countries (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Schefers and Weigel 2012</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Pryce and Daetwyler 2012</a>). However, GS is still new to plant and tree breeding programs.</p><p id="p-5">In GS, large numbers of markers randomly distributed across the genome are genotyped in small effective populations with high levels of LD. A high density of markers is required to maximize the chances of at least one marker being in LD with the QTL. Effects of all the markers are used simultaneously to develop prediction models in “training” populations (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Heffner <em>et al.</em> 2009</a>). Because the number of predictors (markers) is generally much higher than the sample size (<em>P</em> &gt; &gt; <em>n</em>), fixed regression methods using ordinary least squares cannot be used for developing prediction models. Statistical methods that treat marker effects as random, such as ridge regression best linear unbiased prediction (RR-BLUP) and various Bayesian models, are used for developing prediction models. The estimated model is then used to predict phenotypes in a “testing” population using only the marker genotype information. The predictions generated using marker genotypes [genomic estimated breeding values (GEBVs)] are used to select individuals without phenotypic data. The accuracy of GS is assessed by correlating GEBVs with the breeding values predicted using traditional methods that utilize phenotypic data (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Heffner <em>et al.</em> 2009</a>).</p><p id="p-6">A major advantage of using GS in crop breeding is the acceleration of genetic improvement per unit-time through the reduction in time required to complete breeding cycles (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Hickey <em>et al.</em> 2014</a>). Several GS models have been developed for predicting phenotypes using large numbers of markers. These models mainly differ in the assumptions of marker effects contributing to total variance. Methods such as RR-BLUP assume that marker effects are homogeneously distributed across the loci, whereas Bayes methods allow for heterogeneity among markers, with some markers having higher effects than others (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Goddard and Hayes 2007</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Pérez <em>et al.</em> 2010</a>). In methods such as BayesB, priors are used to select the number of markers with large effects; in LASSO (least absolute shrinkage and selection operator) models, penalties are used to select for the markers with major effects (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Daetwyler <em>et al.</em> 2010</a>). BayesB and Bayesian LASSO (BL) methods can identify a subset of markers with large effect (variable selection) and use them for making predictions. Reduced dimension regression methods such as principal components regression (PCR) and partial least squares (PLS) extract the latent variables for making predictions on the response (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Jannink <em>et al.</em> 2010</a>). Although PLS uses response variable of the regression for selecting latent components (supervised method), PCR does not take into account the dependent variable when selecting latent components (unsupervised method); therefore, it is not optimal for making predictions (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Boulesteix and Strimmer 2007</a>). PLS is mainly developed to deal with many correlated predictors and relatively few samples (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Mevik and Wehrens 2007</a>). Although dimension reduction with PLS is effective for dealing with the problems of small samples compared to predictors (<em>P</em> &gt; &gt; <em>n</em>) and multi-collinearity among the predictors, all predictors are used in the final model. <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Chung and Keles (2010)</a> have proposed sparse partial least square regression (SPLS), a variant of PLS that can simultaneously reduce the dimensions and subselect the variables in prediction models.</p><p id="p-7">We tested five models (BayesB, BL, RR-BLUP, PLS, and SPLS) to test prediction accuracies for time to young microspore (TYM) stage and spike grain number (SGN) in two hexaploid wheat biparental doubled haploid (DH) populations. The data used in this study were obtained from two QTL mapping experiments, where a wheat DH population from a biparental Cranbrook × Halberd (C×H) cross was phenotyped under controlled environment conditions using a hydroponics infrastructure (R. Dolferus, X. Ji, S. Thavamanikumar, E. Tanaka, J. Edlington, K. Forrest, G. Rebetzke, M. Hayden, and B. Cullis, unpublished data) and where QTL mapping for a second DH population from a Sundor × AUS30604 (S×A) cross, which was phenotyped for the same traits, is currently in progress (Dolferus <em>et al.</em>, unpublished data). The aim of the experiment was to identify QTL for maintenance of SGN under osmotic stress conditions. The young microspore stage of pollen development is the stage of reproductive development that is most sensitive to abiotic stresses (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Ji <em>et al.</em> 2010</a>). Because the lines of the DH population segregate for flowering time, plants were treated at the same young microspore stage independent of flowering time. The traits that were phenotyped for the individual lines of the DH populations were SGN under control and osmotic stress treatments (SGNC and SGNO), as well as the time for the individual plants to reach the young microspore stage of pollen development (TYM).</p><p id="p-8">The main objectives of this study are to compare the accuracies of different GS prediction models and to compare the accuracies of two traits with contrasting genetic architecture using several hundreds of random markers and a few QTL-linked markers.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Plant materials and trait measurements</h3><p id="p-9">Two wheat doubled haploid (DH) populations were used in this study. This includes a population of 165 DH lines from a cross between Cranbrook and Halberd (C×H) (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Chalmers <em>et al.</em> 2001</a>) and 159 DH lines from a cross between Sundor and AUS30604 (S×A) (Dolferus, unpublished data). These two populations were subjected to osmotic stress experiments using a hydroponics facility and NaCl as osmoticum in a glasshouse to identify QTL for osmotic stress and drought resistance in two separate studies (R. Dolferus, X. Ji, S.Thavamanikumar, E. Tanaka, J. Edlington, K. Forrest, G. Rebetzke, M. Hayden, and B. Cullis, unpublished data). The date when the plants had reached a stage where the auricle distance was between −1 and +4 was recorded as TYM. From this day, the plants were stressed for 5 d with NaCl as an osmoticum using a hydroponics system (R. Dolferus, X. Ji, S. Thavamanikumar, E. Tanaka, J. Edlington, K. Forrest, G. Rebetzke, M. Hayden and B. Cullis, unpublished data). Unstressed control plants were kept in the master tank, and plants for stress treatments were treated with salt in a smaller tank in the same glasshouse. At maturity, stressed and control spikes were harvested individually and SGN (SGNO and SGNC) were determined. Two biological repeat experiments were performed between October and December in 2009 and 2010 to get independent estimates. A mixed model was fitted in ASReml 3.0 (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">VSN International 2009</a>) by fitting the response variable (TYM or SGNC or SGNO) as a fixed effect and by fitting the line effects and effects due to repeated measurements as random effects. Using this model, best linear unbiased predictions (BLUP) were estimated for the three traits. BLUP for TYM, SGNC, and SGNO were used in genomic prediction models.</p></div><div id="sec-3" class="subsection"><h3>SNP genotyping</h3><p id="p-10">DH lines from both crosses were genotyped using a 90K SNP chip developed using gene-specific SNPs, which provides a dense coverage of the wheat genome (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Wang <em>et al.</em> 2014</a>). After removing monomorphic sites, totals of 17,328 and 17,293 SNPs were obtained in C×H and S×A populations, respectively. After removing SNPs that are in complete LD (redundant markers), only 1975 and 1483 were left in C×H and S×A populations, respectively. Of these nonredundant SNPs, 808 SNPs are common to both the populations. Both sets of SNPs (population-specific and common SNPs) were used in the genomic prediction analyses. The main reason for removing these redundant markers is that they provide redundant information and will not add any power to genomic prediction analyses. Moreover, including these redundant SNPs may increase the computing time of genomic prediction analyses. Missing genotypes were imputed with mean imputation with the RR-BLUP package as well as with “rfImpute,” which uses a proximity matrix to fill-in missing values using the R package “randomForest” (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Liaw and Wiener 2002</a>).</p><p id="p-11">Raw genotype data and trait (BLUPs) data are provided in supporting information, <a href="http://www.g3journal.org/content/suppl/2015/07/22/g3.115.019745.DC1/FileS1.xlsx">File S1</a>, <a href="http://www.g3journal.org/content/suppl/2015/07/22/g3.115.019745.DC1/FileS2.xlsx">File S2</a>, <a href="http://www.g3journal.org/content/suppl/2015/07/22/g3.115.019745.DC1/FileS3.xlsx">File S3</a>, <a href="http://www.g3journal.org/content/suppl/2015/07/22/g3.115.019745.DC1/FileS4.xlsx">File S4</a>, and <a href="http://www.g3journal.org/content/suppl/2015/07/22/g3.115.019745.DC1/FileS5.xlsx">File S5</a>.</p></div><div id="sec-4" class="subsection"><h3>Genomic prediction models</h3><div id="sec-5" class="subsection"><h4>Prediction models:</h4><p id="p-12">We used different statistical models that treat marker effects as random. Five prediction models were used: RR-BLUP, BL, BayesB, PLS, and SPLS. All prediction models were tested using different packages in the R statistical environment (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">R Development Core Team 2011</a>). The basic prediction model is represented as<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/1991/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/1991/embed/mml-math-1.gif"/></noscript></span></span>where <em>y<sub>i</sub></em> is the phenotype of individual <em>i</em>, <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/1991/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/1991/embed/mml-math-2.gif"/></noscript></span></span> is the 1 × <em>p</em> vector of SNP genotypes of individual <em>i</em> at locus <em>k</em> of <em>p</em> loci, and <em>β<sub>k</sub></em> is the effect of SNP <em>k</em>, and <em>e<sub>i</sub></em> is the residual term.</p><p id="p-13">The RR-BLUP model assumes homogenous variance of all markers and shrinks all marker effects equally to zero. RR-BLUP is equivalent to BLUP and uses the realized relationship matrix estimated from the markers. RR-BLUP was implemented using the “rrBLUP” package (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Endelman 2011</a>). The BL model assumes marker-specific shrinkage related to the absolute value of the estimated regression coefficient. BL models shrink markers with zero effect more than those with large effects, leading to variable selection when making predictions. BL models were implemented using “BLR” (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Pérez <em>et al.</em> 2010</a>) and “BGLR” packages (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Pérez and de los campos 2014</a>).</p><p id="p-14">The BayesB model, which was proposed by <a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Meuwissen <em>et al.</em> (2001)</a>, assumes unique variance for each marker and a proportion (π) of markers to have large effects while most of the markers have zero effect. Marker effects are estimated with Monte Carlo Markov Chain (MCMC) simulations. We used a value of 0.95 for π, and the model was run for 5000 iterations with a burn-in period of 1000 iterations. We used “BGLR” and “GenSel” packages to implement BayesB model. PLS is a dimension reduction regression that identifies the latent components that explain most of the variation in the response variable to make predictions. The optimum number of components that minimized the prediction error was selected by 10-fold cross-validation using the training samples. The optimum number of components selected from the previous step was used to predict traits in the testing population. PLS model was implemented using the “PLSR” package (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Mevik and Wehrens 2007</a>). In SPLS, in addition to the number of components, the optimum number of variables was selected based on mean squared prediction error by 10-fold cross-validation (CV) of the training samples. The selected optimum components and variables were used to predict traits in testing population. SPLS models were implemented using the “SPLS” package (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Chung <em>et al.</em> 2013</a>).</p></div></div><div id="sec-6" class="subsection"><h3>Estimating accuracy of genomic predictions</h3><div id="sec-7" class="subsection"><h4>Cross-validation:</h4><p id="p-15">Prediction accuracies of the markers within each of the two DH populations were evaluated using a 10-fold CV process (<a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">Pérez <em>et al.</em> 2010</a>). Each population (C×H or S×A) was divided into 10 folds and individuals from the population were assigned to each fold randomly. Marker and phenotype data (BLUPs) from the nine folds were collectively used to predict the phenotypes (GEBVs) of individuals assigned to the tenth fold using only the marker data. This process was repeated 10 times. At each step, the predictive accuracy of the markers was assessed by Pearson’s correlation between the predicted values and the phenotypes. Average of the 10 left out folds was reported as accuracy of the prediction. SEs were estimated from 10 estimates of accuracies.</p></div><div id="sec-8" class="subsection"><h4>Independent validation:</h4><p id="p-16">In independent validation, the prediction model developed using the marker and phenotype data in one population (C×H, training population) was used to predict the phenotypes in the other population using only the marker information (S×A, testing population). Prediction accuracy of the markers was estimated by Pearson’s correlation between the GEBVs and the phenotypic data in the testing population.</p></div></div><div id="sec-9" class="subsection"><h3>Data availability</h3><p id="p-17"><a href="http://www.g3journal.org/content/suppl/2015/07/22/g3.115.019745.DC1/FileS1.xlsx">File S1</a> contains genotype data for 1975 SNPs and trait data for CxH population. <a href="http://www.g3journal.org/content/suppl/2015/07/22/g3.115.019745.DC1/FileS2.xlsx">File S2</a> contains genotype data for 1483 SNPs and trait data for SxA population. <a href="http://www.g3journal.org/content/suppl/2015/07/22/g3.115.019745.DC1/FileS3.xlsx">File S3</a> contains genotype data for common 808 SNPs and trait data for both CxH and SxA population. <a href="http://www.g3journal.org/content/suppl/2015/07/22/g3.115.019745.DC1/FileS4.xlsx">File S4</a> contains genotype data for 42 SNPs from chromosome 5A and trait data for both CxH and SxA population. <a href="http://www.g3journal.org/content/suppl/2015/07/22/g3.115.019745.DC1/FileS4.xlsx">File S4</a> contains genotype data for 766 SNPs excluding 42 SNPs from chromosome 5A and trait data for both CxH and SxA population.</p></div></div><div class="section results" id="sec-10"><h2 class="">Results</h2><p id="p-18">Estimates of broad sense heritability were generally higher for TYM (0.74 in S×A and 0.96 in C×H) and lower for SGNC (0.39 in C×H and 0.29 in S×A) and SGNO (0.40 in C×H and 0.24 in S×A). CV and independent validation methods were used to test the accuracy of genomic predictions. CV was used to test accuracies within a population and independent validation was used to test accuracies across the populations. Independent validation was performed using the marker effects from the C×H training population to estimate GEBVs of the S×A testing population.</p><div id="sec-11" class="subsection"><h3>Prediction accuracy of TYM</h3><div id="sec-12" class="subsection"><h4>Cross-validation:</h4><p id="p-19">We used genotype data of 1975 SNPs in the C×H population and 1483 SNPs in the S×A population for testing within-population prediction accuracies. Missing genotypes were estimated with the mean imputation method implemented in the “rrBLUP” package and the proximity matrix method implemented in the “randomForests” package. We observed similar accuracies with both methods. To be able to compare within-population accuracies (CV) with across-population accuracies (independent validation), we used 808 markers that were common to both populations for testing accuracies in each population. Therefore, in each population CV accuracies were tested with full as well as reduced set of 808 markers.</p><p id="p-20">For TYM, prediction accuracies ranging from 0.61 to 0.82 were observed in the C×H population using the 1975 SNPs in CV. Slightly lower accuracies (ranging between 0.55 and 0.78) were observed with the common 808 markers compared to a full set of markers (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). In the S×A population, prediction accuracies ranged between 0.51 and 0.84 with all 1483 SNPs. Accuracies with the common 808 markers are similar to those with full set of markers ranging between 0.52 and 0.84 (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). In all these analyses, BayesB and SPLS models yielded the highest accuracies, followed by BL, whereas RR-BLUP and PLS models yielded the lowest accuracies.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/10/1991/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Prediction accuracies obtained for TYM in C&#xD7;H and S&#xD7;A populations using 10-fold CV. Analyses were conducted with population-specific SNPs (1975 SNPs for C&#xD7;H and 1483 SNPs for S&#xD7;A) as well as 808 SNPs that are common to both the populations. Error bars are standard errors of mean from 10 repeats." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1862924428" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Prediction accuracies obtained for TYM in C&#xD7;H and S&#xD7;A populations using 10-fold CV. Analyses were conducted with population-specific SNPs (1975 SNPs for C&#xD7;H and 1483 SNPs for S&#xD7;A) as well as 808 SNPs that are common to both the populations. Error bars are standard errors of mean from 10 repeats.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/10/1991/F1.medium.gif" width="440" height="256"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/5/10/1991/F1.medium.gif" width="440" height="256"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/10/1991/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/10/1991/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466829" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-21" class="first-child">Prediction accuracies obtained for TYM in C×H and S×A populations using 10-fold CV. Analyses were conducted with population-specific SNPs (1975 SNPs for C×H and 1483 SNPs for S×A) as well as 808 SNPs that are common to both the populations. Error bars are standard errors of mean from 10 repeats.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h4>Independent validation:</h4><p id="p-22">In independent validation, marker effects of TYM estimated in the C×H population were used to develop the prediction model that is then used to predict the GEBVs in the S×A population. We used C×H population as a training population because this population was used in our earlier QTL studies to identify the QTL linked to the two traits used in this study. The S×A population was used as a testing population because it is an independent population not used in our previous QTL studies and the parents of this population are different from C×H population. In independent validation, prediction accuracies were tested with the 808 common markers.</p><p id="p-23">Prediction accuracies of TYM in the S×A population were generally lower than CV and ranged between 0.32 and 0.70 (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). Similar to the results from CV, higher accuracies were observed with BayesB (0.70) and SPLS (0.66), followed by BL (0.59), whereas RR-BLUP (0.36) and PLS (0.32) produced the lowest accuracies. Next, we tested the accuracies of prediction models using only the 42 markers from chromosome 5A, which harbored the main TYM QTL observed in our previous study. Higher accuracies were observed with 42 QTL-linked SNPs compared to the 808 common markers (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). Accuracies among all the five models were similar (ranging between 0.68 and 0.71). However, the increase in accuracy using the QTL-linked markers compared to 808 common markers was large for PLS (0.69 <em>vs.</em> 0.32) and RR-BLUP (0.68 <em>vs.</em> 0.36) compared to BL (0.70 <em>vs.</em> 0.59), SPLS (0.70 <em>vs.</em> 0.66), and BayesB (0.71 <em>vs.</em> 0.70). BayesB and SPLS had the least improvement in accuracy, indicating the ability of these models to accurately select the large effect markers from 808 markers when making prediction. To test the importance of the QTL-linked SNPs in predictions, we analyzed the accuracies by excluding the 42 QTL-linked markers from the 808 common markers. Prediction accuracies were very low and close to zero for all models when the QTL-linked markers were excluded from the models (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466793/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466793/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Prediction accuracies of TYM and SGN in the S×A population using the models developed in the C×H population</span><div class="sb-div caption-clear"></div></div></div></div></div><div id="sec-14" class="subsection"><h3>Prediction accuracy of SGN</h3><div id="sec-15" class="subsection"><h4>Cross-validation—SGNC:</h4><p id="p-28">Ten-fold CV was used to test the prediction accuracies for SGN under control (SGNC) and osmotic (SGNO) conditions in each population separately.</p><p id="p-29">Prediction accuracies were lower with high error rates in the S×A population compared to the C×H population under control conditions (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). In the C×H population, accuracies ranged between 0.40 and 0.42, whereas in the S×A population they are between 0.10 and 0.14 using the full set of markers. When only the 808 common SNPs were used in prediction models, accuracies were either similar or slightly lower than those with all markers in the C×H population (ranging between 0.32 and 0.41). However, in the S×A population accuracies with the 808 common markers are higher than those using all markers (ranging between 0.11 and 0.21) for all models except for SPLS. Prediction accuracies with either a full set or a reduced set of markers are similar among all the five models in both populations except for SPLS. Prediction accuracy for SPLS is lower compared to other models with a reduced set of markers in the S×A population.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/10/1991/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Prediction accuracies for SGNC in C&#xD7;H and S&#xD7;A populations using 10-fold CV. Analyses were conducted with both population-specific SNPs (1975 SNPs in C&#xD7;H and 1483 SNPs in S&#xD7;A) and 808 SNPs that are common to both the populations. Error bars are standard errors of mean from 10 repeats." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1862924428" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Prediction accuracies for SGNC in C&#xD7;H and S&#xD7;A populations using 10-fold CV. Analyses were conducted with both population-specific SNPs (1975 SNPs in C&#xD7;H and 1483 SNPs in S&#xD7;A) and 808 SNPs that are common to both the populations. Error bars are standard errors of mean from 10 repeats.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/10/1991/F2.medium.gif" width="440" height="275"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/5/10/1991/F2.medium.gif" width="440" height="275"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/10/1991/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/10/1991/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466706" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-30" class="first-child">Prediction accuracies for SGNC in C×H and S×A populations using 10-fold CV. Analyses were conducted with both population-specific SNPs (1975 SNPs in C×H and 1483 SNPs in S×A) and 808 SNPs that are common to both the populations. Error bars are standard errors of mean from 10 repeats.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-16" class="subsection"><h4>Cross-validation—SGNO:</h4><p id="p-31">In stress treatment, the prediction accuracies in contrast to the control treatment are lower in the C×H population than those in the S×A population with both a full set of markers and common markers (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). In the C×H population accuracies ranged between 0.27 and 0.30 with a full set of markers (1975 SNPs), whereas in the S×A population they ranged between 0.43 and 0.45 with a full set of markers (1483 SNPs). Accuracies with the common 808 markers are similar to a full set of markers in both populations. In the C×H population, accuracies ranged between 0.30 and 0.32, whereas in the S×A population the accuracies ranged between 0.41 and 0.46. Similar to the control treatment, the accuracies among all five models are very similar in stress treatment (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/10/1991/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Prediction accuracies for SGNO in C&#xD7;H and S&#xD7;A populations using 10-fold CV. Analyses were conducted with both population-specific SNPs (1975 SNPs in C&#xD7;H and 1483 SNPs in S&#xD7;A) and 808 SNPs that are common to both populations. Error bars are standard errors of mean from 10 repeats." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1862924428" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Prediction accuracies for SGNO in C&#xD7;H and S&#xD7;A populations using 10-fold CV. Analyses were conducted with both population-specific SNPs (1975 SNPs in C&#xD7;H and 1483 SNPs in S&#xD7;A) and 808 SNPs that are common to both populations. Error bars are standard errors of mean from 10 repeats.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/10/1991/F3.medium.gif" width="440" height="256"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/5/10/1991/F3.medium.gif" width="440" height="256"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/10/1991/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/10/1991/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466713" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-32" class="first-child">Prediction accuracies for SGNO in C×H and S×A populations using 10-fold CV. Analyses were conducted with both population-specific SNPs (1975 SNPs in C×H and 1483 SNPs in S×A) and 808 SNPs that are common to both populations. Error bars are standard errors of mean from 10 repeats.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-17" class="subsection"><h4>Independent validation—SGNC:</h4><p id="p-33">In independent validation, marker effects of SGN in the C×H population were used to develop the prediction model for control (SGNC) and osmotic (SGNO) treatments separately with the 808 common markers. This model was then used to predict GEBVs in the S×A population. Under control conditions, SPLS and BayesB had the highest accuracies (0.24 and 0.22, respectively), followed by the BL (0.16), whereas PLS and RR-BLUP had the lowest accuracies (0.12 and 0.13, respectively; <a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>). We then compared the accuracies of independent validation (across population) with CV (within population) obtained with the 808 common markers. Accuracies with independent validation were lower than those with CV in the S×A population for all models except for SPLS and BayesB (<a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a> and <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>). For SPLS, the accuracy has increased from 0.11 in CV to 0.24 in independent validation, and for BayesB there is a slight increase in the accuracy with independent validation compared to CV (<a id="xref-table-wrap-1-6" class="xref-table" href="#T1">Table 1</a> and <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>) in the S×A population. Similar to TYM, we observed a QTL associated with SGNO on chromosome 5A in our previous QTL studies. To test if the 42 markers from chromosome 5A that are associated with SGN under osmotic conditions can predict SGN under control conditions, we used only these 42 markers in the prediction models. Prediction accuracies have slightly increased for BL, RR-BLUP, and PLS but decreased for SPLS and BayesB with QTL-linked markers compared to all 808 common markers (<a id="xref-table-wrap-1-7" class="xref-table" href="#T1">Table 1</a>). However, when the QTL-linked markers were excluded from the 808 common marker set, accuracies were close to zero for all five models (<a id="xref-table-wrap-1-8" class="xref-table" href="#T1">Table 1</a>).</p></div><div id="sec-18" class="subsection"><h4>Independent validation—SGNO:</h4><p id="p-34">In contrast to the control treatment, accuracies in osmotic treatment were very low for all five models using the 808 common markers (<a id="xref-table-wrap-1-9" class="xref-table" href="#T1">Table 1</a>). The accuracies ranged from 0.11 (PLS) to 0.05 (BL). However, using only the QTL-linked markers the accuracies have increased for all five models (<a id="xref-table-wrap-1-10" class="xref-table" href="#T1">Table 1</a>). Accuracies with QTL-linked markers ranged from 0.23 (SPLS) to 0.31 (RR-BLUP). These accuracies are higher than those observed under control conditions (<a id="xref-table-wrap-1-11" class="xref-table" href="#T1">Table 1</a>). Similar to control conditions, accuracies obtained after excluding the QTL-linked markers were close to zero for all models.</p></div></div></div><div class="section discussion" id="sec-19"><h2 class="">Discussion</h2><div id="sec-20" class="subsection"><h3>Prediction accuracies among different models</h3><p id="p-35">In this study, we used different prediction models to compare the accuracies of flowering time and grain number traits in wheat. BayesB and SPLS outperformed all other models, followed by BL in predicting TYM (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>). The performance of all five models is similar in predicting SGNC and SGNO (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a> and <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>) with CV. The differences in prediction performance of different models reflect the underlying genetic architecture of the traits. TYM is a trait influenced by a few loci with major effects. In a recent QTL study, we identified a major QTL on chromosome 5A explaining 72% of phenotypic variation in TYM. SPLS, BayesB, and BL are regarded as variable selection models (<a id="xref-ref-32-4" class="xref-bibr" href="#ref-32">Pérez <em>et al.</em> 2010</a>). Because of the different underlying assumptions of these models compared to others, BayesB and BL identify a subset of markers with large effects to make predictions that increase the prediction accuracy of the traits, especially those controlled by a few large QTL. SPLS combines variable selection and modeling in one step (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Le Cao <em>et al.</em> 2008</a>). For traits that are influenced by several loci, such as SGN, the performance of all models was similar with CV (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2</a> and <a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3</a>). Several studies have shown better performance of BayesB in predicting traits influenced by a few genes of large effect (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">VanRaden <em>et al.</em> 2009</a>; <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Daetwyler <em>et al.</em> 2010</a>; <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Jannink <em>et al.</em> 2010</a>). Studies have also shown that accuracies from GBLUP, which is equivalent to RR-BLUP and BayesB, are similar in predicting quantitative traits (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Clark <em>et al.</em> 2011</a>). <a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Daetwyler <em>et al.</em> (2010)</a> have shown through simulations that BayesB has an advantage over GBLUP when the number of QTL underlying a trait are small, and this advantage is diminished when the number of QTL increased similar to the results observed in this study. However, <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Riedelsheimer <em>et al.</em> (2012)</a> did not find any major differences between BayesB, RR-BLUP, and other models in predicting several traits, including traits with large QTL effects. They attributed this to the high level of LD among the diverse maize lines used in their study. Results from our study, however, show that even when LD is high within a population the performance of BayesB and SPLS is higher than that of other models, especially for traits with large QTL effects.</p></div><div id="sec-21" class="subsection"><h3>Prediction accuracies of TYM</h3><p id="p-36">Accuracies from CV are higher than those from independent validation, where the prediction model developed in C×H was used to predict flowering time in S×A. In independent validation, prediction accuracies using the 42 chromosome 5A SNPs from QTL region are higher than those from the 808 common SNPs for all models except BayesB and to some extent for SPLS. For BayesB and SPLS, however, there is no big change in the accuracy using either 42 SNPs from the QTL region or all the 808 common SNPs (<a id="xref-table-wrap-1-12" class="xref-table" href="#T1">Table 1</a>). This indicates that BayesB and SPLS are able to correctly select the subset of markers with a large effect from all the SNPs when predicting the trait. This trend is also seen to a certain extent with BL, which is another variable selection model. With RR-BLUP and PLS, there is more than 80% improvement in accuracies using the linked markers compared to the random markers (<a id="xref-table-wrap-1-13" class="xref-table" href="#T1">Table 1</a>). RR-BLUP assumes equal variance and shrinks all the marker effects to zero, leading to lower accuracy, especially when there are some large-effect QTL present among the markers. Although PLS reduces dimensions, it does not automatically select variables and uses all markers in the model (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Chung and Keles 2010</a>).</p></div><div id="sec-22" class="subsection"><h3>Prediction accuracies for SGN</h3><p id="p-37">Accuracies for SGN are generally lower compared to TYM, reflecting the lower heritability estimates observed for SGN compared to TYM. This may also reflect the differences in genetic architecture of these traits. Our earlier QTL studies in the C×H population have shown that TYM is influenced by a major QTL with large effects, whereas SGNO is influenced by several QTL with smaller effects (R. Dolferus, X. Ji, S. Thavamanikumar, E. Tanaka, J. Edlington, K. Forrest, G. Rebetzke, M. Hayden and B. Cullis, unpublished data). Of these QTL, one QTL from chromosome 5A has accounted for a significant portion of variation in TYM (71%) and SGN (30%). Accuracies for SGNC are lower in independent validation than CV using the 808 common markers in the S×A population for all models except for SPLS and BayesB (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2</a>; <a id="xref-table-wrap-1-14" class="xref-table" href="#T1">Table 1</a>). For SPLS and BayesB, there was an increase in accuracy with independent validation compared to CV in the S×A population. However, when only the 42 QTL-linked markers from chromosome 5A are used in independent validation, prediction accuracies were higher than those with 808 common SNPs for all models (<a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2</a>). These results indicate that SPLS, BayesB, and to some extent BL are able to select a subset of markers with large effects when predicting SGNC similar to TYM. For cross-population predictions, prediction models that capture LD between markers and QTL are more important than those capturing genetic relationships.</p><p id="p-38">For SGNO, accuracies were substantially lower with independent validation compared to CV with 808 markers in the S×A population (<a id="xref-table-wrap-1-15" class="xref-table" href="#T1">Table 1</a> and <a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3</a>). This could be due to these two populations responding to osmotic stress differently because of the differences in the genetic background, leading to differences in marker effects between the two populations. Similar to control treatment, accuracies using 42 SNPs from chromosome 5A are higher than those from 808 markers with independent validation (<a id="xref-table-wrap-1-16" class="xref-table" href="#T1">Table 1</a>). However, none of the models were able to identify the large-effect markers correctly from 808 markers, leading to lower accuracies with 808 markers compared to 42 QTL-linked markers. The improvement in accuracies when using QTL-linked markers is higher for osmotic treatment compared to control conditions (<a id="xref-table-wrap-1-17" class="xref-table" href="#T1">Table 1</a>). This may reflect the method used to detect the QTL in our previous study. We used contrast BLUPs (difference between control and stress treatment) estimated with a mixed model analysis for detecting QTL for SGNO. Therefore, these QTL maintain grain number under osmotic conditions.</p></div><div id="sec-23" class="subsection"><h3>Within population <em>vs.</em> across population accuracies</h3><p id="p-39">There are several studies in wheat where GS was studied using biparental populations (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Heffner <em>et al.</em> 2011</a>), elite breeding lines (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Poland <em>et al.</em> 2012</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Storlie and Charmet 2013</a>; <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Crossa <em>et al.</em> 2010</a>), and diverse landraces (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Daetwyler <em>et al.</em> 2014</a>). In all these studies, the prediction accuracy of GS was assessed by using CV methods in which the same population was used for training as well as testing the accuracies. Although CV methods are useful to assess the accuracies in populations of the same genetic background, these accuracies may not be generalized to other populations with different genetic backgrounds. CV methods in general overestimate the potential of genomic prediction (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Hofheinz <em>et al.</em> 2012</a>). This is reflected by the high accuracies observed in general with CV compared to independent validation in this study.</p><p id="p-40">In addition to CV, we used independent validation to assess the prediction accuracies. In independent validation, prediction models developed in one population (C×H, training population) are applied in a different population (S×A, testing population) to estimate the GEBVs. The training population (C×H) was used in our previous QTL studies to identify QTL linked to the traits studied in this study, whereas the testing population (S×A) is an independent population. The two DH populations used in this study do not have the same parents. Yet, the accuracies observed with independent validation are as good as CV in the S×A population for TYM and SGNC (<a id="xref-table-wrap-1-18" class="xref-table" href="#T1">Table 1</a>, <a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a>, and <a id="xref-fig-2-8" class="xref-fig" href="#F2">Figure 2</a>). This is in contrast to the findings of <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Riedelsheimer <em>et al.</em> (2013)</a>, who observed zero or negative accuracies when training and validation populations were unrelated. Several other studies have also shown low accuracy when training and testing populations are unrelated (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Hickey <em>et al.</em> 2014</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Hayes <em>et al.</em> 2009a</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Asoro <em>et al.</em> 2011</a>). Differences between our study and other studies may reflect the difference in the factors that contribute to prediction accuracy of GS models. Within a biparental population, the accuracy of GS is determined by the relationships captured by the markers and markers that are linked to QTL (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Riedelsheimer <em>et al.</em> 2013</a>). Different models use these two types of information differently. RR-BLUP mainly captures the genetic relationships, whereas BayesB uses LD between markers and QTL in making predictions (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Habier <em>et al.</em> 2007</a>). Accuracies of the models using genetic relationships decay over generations. However, accuracies of the models using LD between markers and QTL persist for several generations (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Habier <em>et al.</em> 2007</a>). In the populations used in this study, it seems that the accuracy is influenced more by LD between the markers and QTL than the genetic relationships. In CV within a population, the performance of all the models is similar, especially for SGN (<a id="xref-fig-2-9" class="xref-fig" href="#F2">Figure 2</a> and <a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3</a>), suggesting accuracies of the models that predominantly use the genetic relationships (RR-BLUP) and those that use marker–trait associations (BayesB and SPLS) are similar. In independent validation, with unrelated training and test populations, accuracy of BayesB and SPLS is higher than all other models. BayesB and SPLS capture the LD between markers and QTL, leading to higher accuracy, whereas RR-BLUP, which captures genetic relationships, performed poorly when the training and testing populations are unrelated. This is further evidenced by the zero or negative accuracies (<a id="xref-table-wrap-1-19" class="xref-table" href="#T1">Table 1</a>) observed for both TYM and SGN when the markers from the 5A chromosome linked to the QTL are excluded from the prediction model in independent validation.</p><p id="p-41">Our earlier QTL study indicated that chromosome 5A harbors the strongest QTL for both TYM and SGN. Results from this study validate these results in an independent population (S×A population). Higher accuracies observed in the S×A population with the 42 QTL-linked SNPs compared to the random SNPs indicate that the QTL observed in our earlier study are mainly due to LD between markers and traits. In this study, we have observed higher accuracies with QTL-linked markers compared to the random markers, especially in independent validation. Improvement in prediction accuracies by using “prior information” from association and QTL studies has been suggested previously (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Thavamanikumar <em>et al.</em> 2013</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Zhang <em>et al.</em> 2014</a>). Several studies have shown higher accuracies with QTL-linked markers compared to random markers (<a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Zhang <em>et al.</em> 2014</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Westbrook <em>et al.</em> 2013</a>). In a recent study in rice, it was suggested to use markers detected with genome-wide association studies in GS rather than random markers to increase the efficiency and accuracy of GS (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Spindel <em>et al.</em> 2015</a>). Similarly, in a recent study of wheat, <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Zhao <em>et al.</em> (2014)</a> observed higher accuracies for heading time with marker-assisted selection using functional markers compared to genomic selection with random markers. They developed a method in which functional markers were given more weightage compared to random markers, which increased the prediction accuracy. The importance of including markers identified from QTL and association studies in prediction models is demonstrated in this study by the low accuracy observed when the QTL-linked markers were excluded from the models.</p><p id="p-42">Although a few studies have shown higher accuracies with QTL-linked markers, especially for traits influenced by a few loci with large effects (<a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Zhao <em>et al.</em> 2014</a> and <a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Spindel <em>et al.</em> 2015</a>), several studies have shown that the prediction accuracies with marker-assisted selection or marker-assisted recurrent selection in which only a few QTL-linked markers are used for prediction (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Bernardo 2008</a>) were lower than those with GS (<a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">Heffner <em>et al.</em> 2011</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Guo <em>et al.</em> 2012</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Lorenzana and Bernardo 2009</a>). These findings are in contrast to the results from this study. We observed higher accuracies with a few QTL-linked markers compared to several random markers for both TYM and SGN in this study (<a id="xref-table-wrap-1-20" class="xref-table" href="#T1">Table 1</a>). This may be due to the differences in relatedness between training and testing populations in our study compared to other studies. In most of the previous studies, prediction accuracies of marker-assisted selection and GS were compared using CV with related training and testing populations. In this study, however, accuracies were estimated with unrelated training and testing populations. When training and testing populations are related, capturing relationships with GS may lead to higher accuracy than using only the significant markers in marker-assisted selection. However, when training and testing populations are unrelated, accuracy of marker-assisted selection is higher because there would be fewer relationships to be captured with GS and the nonsignificant markers may add to the noise, thus decreasing the accuracy of the GS. Including QTL-linked markers in the panel of GS markers therefore may increase the prediction accuracies, especially when the training and testing populations are unrelated.</p></div></div><div class="section conclusions" id="sec-24"><h2 class="">Conclusions</h2><p id="p-43">In this study, we compared the accuracies of flowering time and grain number traits in wheat using different GS models. BayesB and SPLS outperformed other models in predicting time to young microspore, whereas their performance is similar to other models in predicting grain number per spike when training and testing populations are related. However, when the training and testing populations are unrelated, BayesB and SPLS are effective in capturing LD between markers and QTL, leading to higher accuracy for both simpler and complex traits compared to other models. Results from this study indicate that the accuracy of the GS models can be increased by using markers identified from linkage and association studies.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-44">This research was supported by the Australian Grains Research and Development Corporation (GRDC grants CSP00130, CSP00175, and DAV00127).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-supplementary-material" id="fn-6"><p id="p-45">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019745/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019745/-/DC1</a></p></li><li class="fn" id="fn-7"><p id="p-46"><em>Communicating editor: E. Huang</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2015-06-14"><span class="received-label">Received </span>June 14, 2015.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2015-07-20"><span class="accepted-label">Accepted </span>July 20, 2015.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2015 Thavamanikumar <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.1991.1" data-doi="10.3835/plantgenome2011.02.0007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Asoro</span>  <span class="cit-name-given-names">F. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Newell</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beavis</span>  <span class="cit-name-given-names">W. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scott</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Accuracy and training population design for genomic selection on quantitative traits in elite North American oats.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">132</span>–<span class="cit-lpage">144</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D4%26rft.spage%253D132%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2011.02.0007%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2011.02.0007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.1991.2" data-doi="10.2135/cropsci2008.03.0131"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Molecular markers and selection for complex traits in plants: Learning from the last 20 years.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">1649</span>–<span class="cit-lpage">1664</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DBernardo%26rft.auinit1%253DR.%26rft.volume%253D48%26rft.issue%253D5%26rft.spage%253D1649%26rft.epage%253D1664%26rft.atitle%253DMolecular%2BMarkers%2Band%2BSelection%2Bfor%2BComplex%2BTraits%2Bin%2BPlants%253A%2BLearning%2Bfrom%2Bthe%2BLast%2B20%2BYears%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2008.03.0131%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2008.03.0131&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000259792100001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.1991.3" data-doi="10.1093/bib/bbl016"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boulesteix</span>  <span class="cit-name-given-names">A.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Strimmer</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Partial least squares: a versatile tool for the analysis of high-dimensional genomic data.</span> <abbr class="cit-jnl-abbrev">Brief. Bioinform.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">32</span>–<span class="cit-lpage">44</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBrief.%2BBioinform.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbib%252Fbbl016%26rft_id%253Dinfo%253Apmid%252F16772269%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYmliIjtzOjU6InJlc2lkIjtzOjY6IjgvMS8zMiI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTAvMTk5MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.1991.4" data-doi="10.2135/cropsci2011.06.0299"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burgueno</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weigel</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic prediction of breeding values when modeling genotype x environment interaction using pedigree and dense molecular markers.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">707</span>–<span class="cit-lpage">719</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D52%26rft.spage%253D707%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2011.06.0299%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2011.06.0299&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.1991.5" data-doi="10.1071/AR01081"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chalmers</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Campbell</span>  <span class="cit-name-given-names">A. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kretschmer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Karakousis</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henschke</span>  <span class="cit-name-given-names">P. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Construction of three linkage maps in bread wheat, <em>Triticum aestivum</em>.</span> <abbr class="cit-jnl-abbrev">Aust. J. Agric. Res.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">1089</span>–<span class="cit-lpage">1119</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAust.%2BJ.%2BAgric.%2BRes.%26rft.volume%253D52%26rft.spage%253D1089%26rft_id%253Dinfo%253Adoi%252F10.1071%252FAR01081%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1071/AR01081&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.1991.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chung</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keles</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Sparse partial least squares classification for high dimensional data.</span> <abbr class="cit-jnl-abbrev">Stat. Appl. Genet. Mol. Biol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">17</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStat.%2BAppl.%2BGenet.%2BMol.%2BBiol.%26rft.volume%253D9%26rft.spage%253D17%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-web" id="cit-5.10.1991.7"><div class="cit-metadata unstructured">Chung, D., H. Chun, and S. Keles, 2013 SPLS: Sparse Partial Least Squares (SPLS) Regression and Classification. R package version 2.2–1. <a href="http://CRAN.R-project.org/package=spls">http://CRAN.R-project.org/package=spls</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.1991.8" data-doi="10.1186/1297-9686-43-18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van der Werf</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Different models of genetic variation and their effect on genomic evaluation.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">18</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DClark%26rft.auinit1%253DS.%2BA.%26rft.volume%253D43%26rft.spage%253D18%26rft.epage%253D18%26rft.atitle%253DDifferent%2Bmodels%2Bof%2Bgenetic%2Bvariation%2Band%2Btheir%2Beffect%2Bon%2Bgenomic%2Bevaluation.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-43-18%26rft_id%253Dinfo%253Apmid%252F21575265%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-43-18&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21575265&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F1991.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.1991.9" data-doi="10.1534/genetics.110.118521"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgueno</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Prediction of genetic values of quantitative traits in plant breeding using pedigree and molecular markers.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">186</span>: <span class="cit-fpage">713</span>–<span class="cit-lpage">724</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.118521%26rft_id%253Dinfo%253Apmid%252F20813882%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg2LzIvNzEzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMC8xOTkxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-other" id="cit-5.10.1991.10"><div class="cit-metadata unstructured">Crossa, J., Y. Beyene, S. Kassa, P. Perez, J.M. Hickey <em>et al.</em>, 2013 Genomic prediction in maize breeding populations with genotyping-by-sequencing. <em>G3 (Bethesda)</em> 3:1903–1926.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.1991.11" data-doi="10.1534/genetics.110.116855"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pong-Wong</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Villanueva</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Woolliams</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The impact of genetic architecture on genome-wide evaluation methods.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">185</span>: <span class="cit-fpage">1021</span>–<span class="cit-lpage">1031</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.116855%26rft_id%253Dinfo%253Apmid%252F20407128%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4NS8zLzEwMjEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzEwLzE5OTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.1991.12" data-doi="10.1007/s00122-014-2341-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bansal</span>  <span class="cit-name-given-names">U. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bariana</span>  <span class="cit-name-given-names">H. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayden</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genomic prediction for rust resistance in diverse wheat landraces.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">127</span>: <span class="cit-fpage">1795</span>–<span class="cit-lpage">1803</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D127%26rft.spage%253D1795%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-014-2341-8%26rft_id%253Dinfo%253Apmid%252F24965887%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-014-2341-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24965887&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F1991.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.1991.13" data-doi="10.3835/plantgenome2011.08.0024"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Endelman</span>  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Ridge Regression and Other Kernels for Genomic Selection with R Package rrBLUP.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">250</span>–<span class="cit-lpage">255</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D4%26rft.spage%253D250%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2011.08.0024%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2011.08.0024&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.1991.14" data-doi="10.1111/j.1439-0388.2007.00702.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genomic selection.</span> <abbr class="cit-jnl-abbrev">J. Anim. Breed. Genet.</abbr> <span class="cit-vol">124</span>: <span class="cit-fpage">323</span>–<span class="cit-lpage">330</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BAnim%2BBreed%2BGenet%26rft.aulast%253DGoddard%26rft.auinit1%253DM.%2BE.%26rft.volume%253D124%26rft.issue%253D6%26rft.spage%253D323%26rft.epage%253D330%26rft.atitle%253DGenomic%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1439-0388.2007.00702.x%26rft_id%253Dinfo%253Apmid%252F18076469%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1439-0388.2007.00702.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18076469&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F1991.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251432800002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.1991.15" data-doi="10.1007/s00122-011-1702-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tucker</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kishore</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gay</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Evaluation of genome-wide selection efficiency in maize nested association mapping populations.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">124</span>: <span class="cit-fpage">261</span>–<span class="cit-lpage">275</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DGuo%26rft.auinit1%253DZ.%26rft.volume%253D124%26rft.issue%253D2%26rft.spage%253D261%26rft.epage%253D275%26rft.atitle%253DEvaluation%2Bof%2Bgenome-wide%2Bselection%2Befficiency%2Bin%2Bmaize%2Bnested%2Bassociation%2Bmapping%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-011-1702-9%26rft_id%253Dinfo%253Apmid%252F21938474%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-011-1702-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21938474&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F1991.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.1991.16" data-doi="10.1534/genetics.107.081190"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Habier</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dekkers</span>  <span class="cit-name-given-names">J. C. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The impact of genetic relationship information on genome-assisted breeding values.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">177</span>: <span class="cit-fpage">2389</span>–<span class="cit-lpage">2397</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHabier%26rft.auinit1%253DD.%26rft.volume%253D177%26rft.issue%253D4%26rft.spage%253D2389%26rft.epage%253D2397%26rft.atitle%253DThe%2BImpact%2Bof%2BGenetic%2BRelationship%2BInformation%2Bon%2BGenome-Assisted%2BBreeding%2BValues%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.081190%26rft_id%253Dinfo%253Apmid%252F18073436%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Ny80LzIzODkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzEwLzE5OTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.1991.17" data-doi="10.1186/1297-9686-41-51"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowman</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chamberlain</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Verbyla</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span><span class="cit-comment">a</span> <span class="cit-article-title">Accuracy of genomic breeding values in multi-breed dairy cattle populations.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">51</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DHayes%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D41%26rft.spage%253D51%26rft.epage%253D51%26rft.atitle%253DAccuracy%2Bof%2Bgenomic%2Bbreeding%2Bvalues%2Bin%2Bmulti-breed%2Bdairy%2Bcattle%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-41-51%26rft_id%253Dinfo%253Apmid%252F19930712%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-41-51&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19930712&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F1991.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.1991.18" data-doi="10.3168/jds.2008-1646"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowman</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chamberlain</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span><span class="cit-comment">b</span> <span class="cit-article-title">Genomic selection in dairy cattle: Progress and challenges.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">433</span>–<span class="cit-lpage">443</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DHayes%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D92%26rft.issue%253D2%26rft.spage%253D433%26rft.epage%253D443%26rft.atitle%253DInvited%2Breview%253A%2BGenomic%2Bselection%2Bin%2Bdairy%2Bcattle%253A%2BProgress%2Band%2Bchallenges%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2008-1646%26rft_id%253Dinfo%253Apmid%252F19164653%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2008-1646&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19164653&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F1991.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000262654900001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.1991.19" data-doi="10.2135/cropsci2008.08.0512"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heffner</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, Jannink <em>et al</em>., <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genomic selection for crop improvement.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">12</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DHeffner%26rft.auinit1%253DE.%2BL.%26rft.volume%253D49%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D12%26rft.atitle%253DGenomic%2BSelection%2Bfor%2BCrop%2BImprovement%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2008.08.0512%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2008.08.0512&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000263344300001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.1991.20" data-doi="10.2135/cropsci2011.05.0253"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heffner</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iwata</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Souza</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genomic selection accuracy for grain quality traits in biparental wheat populations.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">51</span>: <span class="cit-fpage">2597</span>–<span class="cit-lpage">2606</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D51%26rft.spage%253D2597%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2011.05.0253%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2011.05.0253&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000295839200031&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.1991.21" data-doi="10.2135/cropsci2013.03.0195"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dreisigacker</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hearne</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Babu</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Evaluation of genomic selection training population designs and genotyping strategies in plant breeding programs using simulation.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">1476</span>–<span class="cit-lpage">1488</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D54%26rft.spage%253D1476%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2013.03.0195%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2013.03.0195&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.1991.22" data-doi="10.1007/s00122-012-1940-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hofheinz</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borchardt</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weissleder</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frisch</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genome-based prediction of test cross performance in two subsequent breeding cycles.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">125</span>: <span class="cit-fpage">1639</span>–<span class="cit-lpage">1645</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DHofheinz%26rft.auinit1%253DN.%26rft.volume%253D125%26rft.issue%253D8%26rft.spage%253D1639%26rft.epage%253D1645%26rft.atitle%253DGenome-based%2Bprediction%2Bof%2Btest%2Bcross%2Bperformance%2Bin%2Btwo%2Bsubsequent%2Bbreeding%2Bcycles.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-012-1940-5%26rft_id%253Dinfo%253Apmid%252F22814724%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-012-1940-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22814724&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F1991.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.1991.23" data-doi="10.1093/bfgp/elq001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iwata</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genomic selection in plant breeding: From theory to practice.</span> <abbr class="cit-jnl-abbrev">Brief. Funct. Genomics</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">166</span>–<span class="cit-lpage">177</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBrief.%2BFunct.%2BGenomics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbfgp%252Felq001%26rft_id%253Dinfo%253Apmid%252F20156985%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoiYmZncCI7czo1OiJyZXNpZCI7czo3OiI5LzIvMTY2IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMC8xOTkxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.1991.24" data-doi="10.1111/j.1365-3040.2010.02130.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ji</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shiran</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lewis</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jenkins</span>  <span class="cit-name-given-names">C. L. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Importance of pre-anthesis anther sink strength for maintenance of grain number during reproductive stage water stress in wheat.</span> <abbr class="cit-jnl-abbrev">Plant Cell Environ.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">926</span>–<span class="cit-lpage">942</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%252C%2BCell%252C%2Band%2BEnvironment%2B%2528Print%2529%26rft.stitle%253DPlant%252C%2BCell%252C%2Band%2BEnvironment%2B%2528Print%2529%26rft.aulast%253DJi%26rft.auinit1%253DX.%26rft.volume%253D33%26rft.issue%253D6%26rft.spage%253D926%26rft.epage%253D942%26rft.atitle%253DImportance%2Bof%2Bpre-anthesis%2Banther%2Bsink%2Bstrength%2Bfor%2Bmaintenance%2Bof%2Bgrain%2Bnumber%2Bduring%2Breproductive%2Bstage%2Bwater%2Bstress%2Bin%2Bwheat.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-3040.2010.02130.x%26rft_id%253Dinfo%253Apmid%252F20199626%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-3040.2010.02130.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20199626&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F1991.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000277712800005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.1991.25" data-doi="10.2202/1544-6115.1390"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Le Cao</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rossouw</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robert-Granie</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Besse</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A sparse PLS for variable selection when integrating omics data.</span> <abbr class="cit-jnl-abbrev">Stat. Appl. Genet. Mol. Biol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">35</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStat.%2BAppl.%2BGenet.%2BMol.%2BBiol.%26rft.volume%253D7%26rft.spage%253D35%26rft_id%253Dinfo%253Adoi%252F10.2202%252F1544-6115.1390%26rft_id%253Dinfo%253Apmid%252F19049491%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2202/1544-6115.1390&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19049491&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F1991.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-other" id="cit-5.10.1991.26"><div class="cit-metadata unstructured">Liaw, A., and M. Wiener, 2002 Classification and Regression by randomForest. <em>R News: The Newsletter of the R Project</em> 2:18–22.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.1991.27" data-doi="10.1007/s00122-009-1166-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lorenzana</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Accuracy of genotypic value predictions for marker-based selection in biparental plant populations.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">120</span>: <span class="cit-fpage">151</span>–<span class="cit-lpage">161</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DLorenzana%26rft.auinit1%253DR.%2BE.%26rft.volume%253D120%26rft.issue%253D1%26rft.spage%253D151%26rft.epage%253D161%26rft.atitle%253DAccuracy%2Bof%2Bgenotypic%2Bvalue%2Bpredictions%2Bfor%2Bmarker-based%2Bselection%2Bin%2Bbiparental%2Bplant%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-009-1166-3%26rft_id%253Dinfo%253Apmid%252F19841887%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-009-1166-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19841887&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F1991.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000271939900013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.1991.28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Prediction of total genetic value using genome-wide dense marker maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">1819</span>–<span class="cit-lpage">1829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMeuwissen%26rft.auinit1%253DT.%2BH.%2BE.%26rft.volume%253D157%26rft.issue%253D4%26rft.spage%253D1819%26rft.epage%253D1829%26rft.atitle%253DPrediction%2Bof%2BTotal%2BGenetic%2BValue%2BUsing%2BGenome-Wide%2BDense%2BMarker%2BMaps%26rft_id%253Dinfo%253Apmid%252F11290733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ny80LzE4MTkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzEwLzE5OTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.1991.29" data-doi="10.1360/jos180001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mevik</span>  <span class="cit-name-given-names">B. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wehrens</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The pls Package: Principal component and partial least squares regression in R.</span> <abbr class="cit-jnl-abbrev">J. Stat. Softw.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">24</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BStat.%2BSoftw.%26rft.volume%253D18%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1360%252Fjos180001%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1360/jos180001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.1991.30" data-doi="10.3835/plantgenome2012.07.0017"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ornella</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sukhwinder</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgueno</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic prediction of genetic values for resistance to wheat rusts.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">136</span>–<span class="cit-lpage">148</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D5%26rft.spage%253D136%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2012.07.0017%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2012.07.0017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.1991.31" data-doi="10.1534/genetics.114.164442"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pérez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genome-wide regression and prediction with the BGLR statistical package.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">483</span>–<span class="cit-lpage">495</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.164442%26rft_id%253Dinfo%253Apmid%252F25009151%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk4LzIvNDgzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMC8xOTkxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.1991.32" data-doi="10.3835/plantgenome2010.04.0005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pérez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genomic-enabled prediction based on molecular markers and pedigree using the Bayesian linear regression package in R.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">106</span>–<span class="cit-lpage">116</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D3%26rft.spage%253D106%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2010.04.0005%26rft_id%253Dinfo%253Apmid%252F21566722%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2010.04.0005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21566722&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F1991.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.1991.33" data-doi="10.3835/plantgenome2012.06.0006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Endelman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dawson</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rutkoski</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic selection in wheat breeding using genotyping-by-sequencing.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">103</span>–<span class="cit-lpage">113</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D5%26rft.spage%253D103%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2012.06.0006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2012.06.0006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.1991.34" data-doi="10.1071/AN11098"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pryce</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>  <span class="cit-name-given-names">H. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Designing dairy cattle breeding schemes under genomic selection: A review of international research.</span> <abbr class="cit-jnl-abbrev">Anim. Prod. Sci.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">107</span>–<span class="cit-lpage">114</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnim.%2BProd.%2BSci.%26rft.volume%253D52%26rft.spage%253D107%26rft_id%253Dinfo%253Adoi%252F10.1071%252FAN11098%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1071/AN11098&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000301064800006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-book" id="cit-5.10.1991.35"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">R Development Core Team</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-source">R: A Language and Environment for Statistical Computing</span>, <span class="cit-publ-name">R Foundation for Statistical Computing</span>, <span class="cit-publ-loc">Vienna, Austria</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.1991.36" data-doi="10.1111/j.1469-8137.2011.04038.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Resende</span>  <span class="cit-name-given-names">M. D. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Resende</span>  <span class="cit-name-given-names">M. F. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sansaloni</span>  <span class="cit-name-given-names">C. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petroli</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Missiaggia</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span><span class="cit-comment">a</span> <span class="cit-article-title">Genomic selection for growth and wood quality in eucalyptus: Capturing the missing heritability and accelerating breeding for complex traits in forest trees.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">194</span>: <span class="cit-fpage">116</span>–<span class="cit-lpage">128</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytologist%26rft.stitle%253DNew%2BPhytologist%26rft.aulast%253DResende%26rft.auinit1%253DM.%2BD.%26rft.volume%253D194%26rft.issue%253D1%26rft.spage%253D116%26rft.epage%253D128%26rft.atitle%253DGenomic%2Bselection%2Bfor%2Bgrowth%2Band%2Bwood%2Bquality%2Bin%2BEucalyptus%253A%2Bcapturing%2Bthe%2Bmissing%2Bheritability%2Band%2Baccelerating%2Bbreeding%2Bfor%2Bcomplex%2Btraits%2Bin%2Bforest%2Btrees.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-8137.2011.04038.x%26rft_id%253Dinfo%253Apmid%252F22309312%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-8137.2011.04038.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22309312&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F1991.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000300719400014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.1991.37" data-doi="10.1111/j.1469-8137.2011.03895.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Resende</span>  <span class="cit-name-given-names">M. F. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Munoz</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Acosta</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peter</span>  <span class="cit-name-given-names">G. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span><span class="cit-comment">b</span> <span class="cit-article-title">Accelerating the domestication of trees using genomic selection: Accuracy of prediction models across ages and environments.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">193</span>: <span class="cit-fpage">617</span>–<span class="cit-lpage">624</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytologist%26rft.stitle%253DNew%2BPhytologist%26rft.aulast%253DResende%26rft.auinit1%253DM.%2BF.%26rft.volume%253D193%26rft.issue%253D3%26rft.spage%253D617%26rft.epage%253D624%26rft.atitle%253DAccelerating%2Bthe%2Bdomestication%2Bof%2Btrees%2Busing%2Bgenomic%2Bselection%253A%2Baccuracy%2Bof%2Bprediction%2Bmodels%2Bacross%2Bages%2Band%2Benvironments.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-8137.2011.03895.x%26rft_id%253Dinfo%253Apmid%252F21973055%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-8137.2011.03895.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21973055&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F1991.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000298984900008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.1991.38" data-doi="10.1534/genetics.111.137026"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Resende</span>  <span class="cit-name-given-names">M. F. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muñoz</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Resende</span>  <span class="cit-name-given-names">M. D. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garrick</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span><span class="cit-comment">c</span> <span class="cit-article-title">Accuracy of genomic selection methods in a standard dataset of loblolly pine (<em>pinus taeda</em> l.).</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">1503</span>–<span class="cit-lpage">1510</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.137026%26rft_id%253Dinfo%253Apmid%252F22271763%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5MC80LzE1MDMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzEwLzE5OTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.1991.39" data-doi="10.1073/pnas.1120813109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Riedelsheimer</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lisec</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Czedik-Eysenberg</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sulpice</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flis</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genome-wide association mapping of leaf metabolic profiles for dissecting complex traits in maize.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">8872</span>–<span class="cit-lpage">8877</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1120813109%26rft_id%253Dinfo%253Apmid%252F22615396%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA5LzIzLzg4NzIiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzEwLzE5OTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.1991.40" data-doi="10.1534/genetics.113.150227"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Riedelsheimer</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Endelman</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stange</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomic predictability of interconnected biparental maize populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">194</span>: <span class="cit-fpage">493</span>–<span class="cit-lpage">503</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.150227%26rft_id%253Dinfo%253Apmid%252F23535384%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk0LzIvNDkzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMC8xOTkxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.1991.41"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schefers</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weigel</span>  <span class="cit-name-given-names">K. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic selection in dairy cattle: Integration of DNA testing into breeding programs.</span> <abbr class="cit-jnl-abbrev">Anim. Front.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">4</span>–<span class="cit-lpage">9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnim.%2BFront.%26rft.volume%253D2%26rft.spage%253D4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.1991.42" data-doi="10.1371/journal.pgen.1004982"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Spindel</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Begum</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Akdemir</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Virk</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Collard</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genomic selection and association mapping in rice (Oryza sativa): effect of trait genetic architecture, training population composition, marker number and statistical model on accuracy of rice genomic selection in elite, tropical rice breeding lines.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">e1004982</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D11%26rft.spage%253De1004982%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1004982%26rft_id%253Dinfo%253Apmid%252F25689273%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1004982&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25689273&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F1991.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.1991.43"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Storlie</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charmet</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomic selection accuracy using historical data generated in a wheat breeding program.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D6%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.1991.44" data-doi="10.1007/s11295-013-0594-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thavamanikumar</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Southerton</span>  <span class="cit-name-given-names">S. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bossinger</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thumma</span>  <span class="cit-name-given-names">B. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Dissection of complex traits in forest trees - opportunities for marker-assisted selection.</span> <abbr class="cit-jnl-abbrev">Tree Genet. Genomes</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">627</span>–<span class="cit-lpage">639</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTree%2BGenet.%2BGenomes%26rft.volume%253D9%26rft.spage%253D627%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11295-013-0594-z%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11295-013-0594-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.1991.45" data-doi="10.3168/jds.2008-1514"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Tassell</span>  <span class="cit-name-given-names">C. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wiggans</span>  <span class="cit-name-given-names">G. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sonstegard</span>  <span class="cit-name-given-names">T. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schnabel</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Invited review: reliability of genomic predictions for North American Holstein bulls.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">16</span>–<span class="cit-lpage">24</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVanRaden%26rft.auinit1%253DP.%2BM.%26rft.volume%253D92%26rft.issue%253D1%26rft.spage%253D16%26rft.epage%253D24%26rft.atitle%253DInvited%2BReview%253A%2BReliability%2Bof%2Bgenomic%2Bpredictions%2Bfor%2BNorth%2BAmerican%2BHolstein%2Bbulls%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2008-1514%26rft_id%253Dinfo%253Apmid%252F19109259%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2008-1514&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19109259&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F1991.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261948600002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-book" id="cit-5.10.1991.46"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">VSN International</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-source">Asreml 3</span>, <span class="cit-publ-name">VSN Intl</span>, <span class="cit-publ-loc">Hemel Hempstead, UK</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.1991.47" data-doi="10.1111/pbi.12183"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forrest</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Allen</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chao</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Characterization of polyploid wheat genomic diversity using a high-density 90 000 single nucleotide polymorphism array.</span> <abbr class="cit-jnl-abbrev">Plant Biotechnol. J.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">787</span>–<span class="cit-lpage">796</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBiotechnol.%2BJ.%26rft.volume%253D12%26rft.spage%253D787%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fpbi.12183%26rft_id%253Dinfo%253Apmid%252F24646323%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/pbi.12183&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24646323&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F1991.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000339488200015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.1991.48" data-doi="10.1111/nph.12240"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Westbrook</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Resende</span>  <span class="cit-name-given-names">M. F. R.</span>  <span class="cit-name-suffix">Jr</span>.</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Munoz</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walker</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wegrzyn</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Association genetics of oleoresin flow in loblolly pine: Discovering genes and predicting phenotype for improved resistance to bark beetles and bioenergy potential.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">199</span>: <span class="cit-fpage">89</span>–<span class="cit-lpage">100</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytol.%26rft.volume%253D199%26rft.spage%253D89%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fnph.12240%26rft_id%253Dinfo%253Apmid%252F23534834%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/nph.12240&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23534834&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F1991.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.1991.49" data-doi="10.1371/journal.pone.0093017"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ober</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Erbe</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Improving the accuracy of whole genome prediction for complex traits using the results of genome wide association studies.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e93017</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De93017%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0093017%26rft_id%253Dinfo%253Apmid%252F24663104%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0093017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24663104&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F1991.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.1991.50" data-doi="10.1038/hdy.2014.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mette</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gowda</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Longin</span>  <span class="cit-name-given-names">C. F. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reif</span>  <span class="cit-name-given-names">J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Bridging the gap between marker-assisted and genomic selection of heading time and plant height in hybrid wheat.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">112</span>: <span class="cit-fpage">638</span>–<span class="cit-lpage">645</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D112%26rft.spage%253D638%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2014.1%26rft_id%253Dinfo%253Apmid%252F24518889%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2014.1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24518889&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F1991.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/5/10/1991.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/10/1983" title="Development of a Tightly Controlled Off Switch for Saccharomyces cerevisiae Regulated by Camphor, a Low-Cost Natural Product" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/10/1999" title="Fingerprinting Soybean Germplasm and Its Utility in Genomic Research" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/5/10"><span>Volume 5 Issue 10, October 2015</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/5/10.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/5/10.cover-source.jpg?itok=JwgNhVLw" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 5 (10)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/10/1991" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-k6UIEyRSnvHlWwKxkeCcXD2bGGQofBX0IUUD64ePAnw" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genomic-selection" class="" data-icon-position="" data-hide-link-title="0">Genomic Selection</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/5/10/1991" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/5/10/1991" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/5/10/1991" class="active" data-icon-position="" data-hide-link-title="0">Comparison of Genomic Selection Models to Predict Flowering Time and Spike Grain Number in Two Hexaploid Wheat Doubled Haploid Populations</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460155" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-8hAmWcGBakiOgede01oasN3l75M-qDSCQ10Qh9fLvBg" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/10/1991.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/5/10/1991" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-vsZHtcYTQz06Dzh7NZcBlXbWXl3UsfunzJbWzB7I4R4" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/5/10/1991" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/10/1991.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460155" id="node460155--2" data-pisa="ggg;5/10/1991" data-pisa-master="ggg;g3.115.019745" data-apath="/ggg/5/10/1991.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Comparison of Genomic Selection Models to Predict Flowering Time and Spike Grain Number in Two Hexaploid Wheat Doubled Haploid Populations</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Saravanan Thavamanikumar</span>, <span class="highwire-citation-author" data-delta="1">Rudy Dolferus</span> and <span class="highwire-citation-author" data-delta="2">Bala R. Thumma</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1991-1998;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.019745 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460155--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Saravanan Thavamanikumar </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>CSIRO Agriculture, Canberra ACT 2601, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Saravanan%2BThavamanikumar%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Thavamanikumar%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASaravanan%2BThavamanikumar%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:saro@gondwanagenomics.com.au" class="" data-icon-position="" data-hide-link-title="0">saro@gondwanagenomics.com.au</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Rudy Dolferus </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>CSIRO Agriculture, Canberra ACT 2601, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rudy%2BDolferus%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dolferus%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARudy%2BDolferus%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Bala R. Thumma </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>CSIRO Agriculture, Canberra ACT 2601, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bala%2BR.%2BThumma%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Thumma%20BR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABala%2BR.%2BThumma%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/5/10/1991" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/5/10/1991&amp;title=Comparison%20of%20Genomic%20Selection%20Models%20to%20Predict%20Flowering%20Time%20and%20Spike%20Grain%20Number%20in%20Two%20Hexaploid%20Wheat%20Doubled%20Haploid%20Populations" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/5/10/1991&amp;title=Comparison%20of%20Genomic%20Selection%20Models%20to%20Predict%20Flowering%20Time%20and%20Spike%20Grain%20Number%20in%20Two%20Hexaploid%20Wheat%20Doubled%20Haploid%20Populations" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/5/10/1991&amp;title=Comparison%20of%20Genomic%20Selection%20Models%20to%20Predict%20Flowering%20Time%20and%20Spike%20Grain%20Number%20in%20Two%20Hexaploid%20Wheat%20Doubled%20Haploid%20Populations" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/5/10/1991&amp;text=Comparison%20of%20Genomic%20Selection%20Models%20to%20Predict%20Flowering%20Time%20and%20Spike%20Grain%20Number%20in%20Two%20Hexaploid%20Wheat%20Doubled%20Haploid%20Populations" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/5/10/1991&amp;title=Comparison%20of%20Genomic%20Selection%20Models%20to%20Predict%20Flowering%20Time%20and%20Spike%20Grain%20Number%20in%20Two%20Hexaploid%20Wheat%20Doubled%20Haploid%20Populations" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/5/10/1991&amp;t=Comparison%20of%20Genomic%20Selection%20Models%20to%20Predict%20Flowering%20Time%20and%20Spike%20Grain%20Number%20in%20Two%20Hexaploid%20Wheat%20Doubled%20Haploid%20Populations" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/5/10/1991&amp;title=Comparison%20of%20Genomic%20Selection%20Models%20to%20Predict%20Flowering%20Time%20and%20Spike%20Grain%20Number%20in%20Two%20Hexaploid%20Wheat%20Doubled%20Haploid%20Populations" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/5/10/1991&amp;title=Comparison%20of%20Genomic%20Selection%20Models%20to%20Predict%20Flowering%20Time%20and%20Spike%20Grain%20Number%20in%20Two%20Hexaploid%20Wheat%20Doubled%20Haploid%20Populations" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460155/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460155" id="node460155--3" data-pisa="ggg;5/10/1991" data-pisa-master="ggg;g3.115.019745" data-apath="/ggg/5/10/1991.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Comparison of Genomic Selection Models to Predict Flowering Time and Spike Grain Number in Two Hexaploid Wheat Doubled Haploid Populations</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Saravanan Thavamanikumar</span>, <span class="highwire-citation-author" data-delta="1">Rudy Dolferus</span> and <span class="highwire-citation-author" data-delta="2">Bala R. Thumma</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1991-1998;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.019745 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460155--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Saravanan Thavamanikumar </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>CSIRO Agriculture, Canberra ACT 2601, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Saravanan%2BThavamanikumar%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Thavamanikumar%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASaravanan%2BThavamanikumar%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:saro@gondwanagenomics.com.au" class="" data-icon-position="" data-hide-link-title="0">saro@gondwanagenomics.com.au</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Rudy Dolferus </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>CSIRO Agriculture, Canberra ACT 2601, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rudy%2BDolferus%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dolferus%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARudy%2BDolferus%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Bala R. Thumma </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>CSIRO Agriculture, Canberra ACT 2601, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bala%2BR.%2BThumma%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Thumma%20BR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABala%2BR.%2BThumma%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1991&amp;rft.epage=1998&amp;rft.atitle=Comparison%20of%20Genomic%20Selection%20Models%20to%20Predict%20Flowering%20Time%20and%20Spike%20Grain%20Number%20in%20Two%20Hexaploid%20Wheat%20Doubled%20Haploid%20Populations&amp;rft.volume=5&amp;rft.issue=10&amp;rft.date=2015-10-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Thavamanikumar%2C+Saravanan&amp;rft.au=Dolferus%2C+Rudy&amp;rft.au=Thumma%2C+Bala+R."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460155/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460155/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460155/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460155/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460155/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460155/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460155/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460155/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460155/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460155/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460155/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460155/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490720" id="node490720" data-pisa="ggg;8/9/3019" data-pisa-master="ggg;g3.118.200430" data-apath="/ggg/8/9/3019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">A Bayesian Decision Theory Approach for Genomic Selection</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490724" id="node490724" data-pisa="ggg;8/9/2889" data-pisa-master="ggg;g3.118.200311" data-apath="/ggg/8/9/2889.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/2889" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Optimising Genomic Selection in Wheat: Effect of Marker Density, Population Size and Population Structure on Prediction Accuracy</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490727" id="node490727" data-pisa="ggg;8/9/3039" data-pisa-master="ggg;g3.118.200435" data-apath="/ggg/8/9/3039.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3039" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">BGGE: A New Package for Genomic-Enabled Prediction Incorporating Genotype × Environment Interaction Models</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genomic%20Selection" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genomic Selection</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/5/10/1991" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-19" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#sec-24" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Conclusions</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/5/10/1991.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/5/10/1991.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/5/10/1991.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/5/10/1991.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
