<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/5/10/2021" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/5/10/2021.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/5/10/2021.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/5/10/2021.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/5/10/2021/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/5/10/2021.atom" />
<meta name="HW.pisa" content="ggg;5/10/2021" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="High-Density Genotypes of Inbred Mouse Strains: Improved Power and Precision of Association Mapping" />
<meta name="DC.Identifier" content="10.1534/g3.115.020784" />
<meta name="DC.Date" content="2015-10-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2015 Rau et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Human genome-wide association studies have identified thousands of loci associated with disease phenotypes. Genome-wide association studies also have become feasible using rodent models and these have some important advantages over human studies, including controlled environment, access to tissues for molecular profiling, reproducible genotypes, and a wide array of techniques for experimental validation. Association mapping with common mouse inbred strains generally requires 100 or more strains to achieve sufficient power and mapping resolution; in contrast, sample sizes for human studies typically are one or more orders of magnitude greater than this. To enable well-powered studies in mice, we have generated high-density genotypes for ∼175 inbred strains of mice using the Mouse Diversity Array. These new data increase marker density by 1.9-fold, have reduced missing data rates, and provide more accurate identification of heterozygous regions compared with previous genotype data. We report the discovery of new loci from previously reported association mapping studies using the new genotype data. The data are freely available for download, and Web-based tools provide easy access for association mapping and viewing of the underlying intensity data for individual loci." />
<meta name="DC.Contributor" content="Christoph D. Rau" />
<meta name="DC.Contributor" content="Brian Parks" />
<meta name="DC.Contributor" content="Yibin Wang" />
<meta name="DC.Contributor" content="Eleazar Eskin" />
<meta name="DC.Contributor" content="Petr Simecek" />
<meta name="DC.Contributor" content="Gary A. Churchill" />
<meta name="DC.Contributor" content="Aldons J. Lusis" />
<meta name="article:published_time" content="2015-10-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="High-Density Genotypes of Inbred Mouse Strains: Improved Power and Precision of Association Mapping" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Human genome-wide association studies have identified thousands of loci associated with disease phenotypes. Genome-wide association studies also have become feasible using rodent models and these have some important advantages over human studies, including controlled environment, access to tissues for molecular profiling, reproducible genotypes, and a wide array of techniques for experimental validation. Association mapping with common mouse inbred strains generally requires 100 or more strains to achieve sufficient power and mapping resolution; in contrast, sample sizes for human studies typically are one or more orders of magnitude greater than this. To enable well-powered studies in mice, we have generated high-density genotypes for ∼175 inbred strains of mice using the Mouse Diversity Array. These new data increase marker density by 1.9-fold, have reduced missing data rates, and provide more accurate identification of heterozygous regions compared with previous genotype data. We report the discovery of new loci from previously reported association mapping studies using the new genotype data. The data are freely available for download, and Web-based tools provide easy access for association mapping and viewing of the underlying intensity data for individual loci.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2015/10/01" />
<meta name="citation_mjid" content="ggg;5/10/2021" />
<meta name="citation_id" content="5/10/2021" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/5/10/2021" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/5/10/2021.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/5/10/2021.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/5/10/2021.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.115.020784" />
<meta name="citation_pmid" content="26224782" />
<meta name="citation_volume" content="5" />
<meta name="citation_issue" content="10" />
<meta name="citation_num_pages" content="6" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2021" />
<meta name="citation_lastpage" content="2026" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Christoph D. Rau" />
<meta name="citation_author_institution" content="Departments of Human Genetics, Medicine, Microbiology, Immunology and Molecular Genetics, UCLA, Los Angeles, California 90095-1679" />
<meta name="citation_author" content="Brian Parks" />
<meta name="citation_author_institution" content="Departments of Human Genetics, Medicine, Microbiology, Immunology and Molecular Genetics, UCLA, Los Angeles, California 90095-1679" />
<meta name="citation_author" content="Yibin Wang" />
<meta name="citation_author_institution" content="Department of Anesthesiology, UCLA, Los Angeles, California 90095-1679" />
<meta name="citation_author" content="Eleazar Eskin" />
<meta name="citation_author_institution" content="Department of Computer Sciences, UCLA, Los Angeles, California 90095-1679" />
<meta name="citation_author" content="Petr Simecek" />
<meta name="citation_author_institution" content="The Jackson Laboratory, Bar Harbor, Maine 04609" />
<meta name="citation_author" content="Gary A. Churchill" />
<meta name="citation_author_institution" content="The Jackson Laboratory, Bar Harbor, Maine 04609" />
<meta name="citation_author" content="Aldons J. Lusis" />
<meta name="citation_author_institution" content="Departments of Human Genetics, Medicine, Microbiology, Immunology and Molecular Genetics, UCLA, Los Angeles, California 90095-1679" />
<meta name="citation_author_email" content="jlusis@mednet.ucla.edu" />
<meta name="citation_reference" content="citation_journal_title=Developmental dynamics :  an official publication of the American Association of Anatomists;citation_journal_abbrev=Dev Dyn;citation_author=M. Azhar;citation_author=K. Brown;citation_author=C. Gard;citation_author=H. Chen;citation_author=S. Rajan;citation_title=Transforming growth factor Beta2 is required for valve remodeling during heart development.;citation_pages=2127-2141;citation_volume=240;citation_year=2011;citation_issue=9;citation_pmid=21780244;citation_doi=10.1002/dvdy.22702" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=B. J. Bennett;citation_author=C. R. Farber;citation_author=L. Orozco;citation_author=H. M. Kang;citation_author=A. Ghazalpour;citation_title=A high-resolution association mapping panel for the dissection of complex traits in mice.;citation_volume=20;citation_year=2010;citation_pmid=20054062;citation_doi=10.1101/gr.099234.109" />
<meta name="citation_reference" content="citation_title=Journal of molecular and cellular cardiology;citation_journal_abbrev=J Mol Cell Cardiol;citation_author=A. G. Brittsan;citation_author=E. Kiss;citation_author=I. Edes;citation_author=I. L. Grupp;citation_author=G. Grupp;citation_title=The effect of isoproterenol on phospholamban-deficient mouse hearts with altered thyroid conditions.;citation_pages=1725-1737;citation_volume=31;citation_year=1999;citation_issue=9;citation_pmid=10471356;citation_doi=10.1006/jmcc.1999.1010" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_title=The Genome Architecture of the Collaborative Cross Mouse Genetic Reference Population;citation_pages=389-401;citation_volume=190;citation_year=2012;citation_issue=2;citation_pmid=22345608;citation_doi=10.1534/genetics.111.132639" />
<meta name="citation_reference" content="citation_journal_title=Mamm. Genome;citation_author=R. C. Davis;citation_author=A. Van Nas;citation_author=B. Bennett;citation_author=L. Orozco;citation_author=C. Pan;citation_title=Genome-wide association mapping of blood cell traits in mice.;citation_pages=105-118;citation_volume=24;citation_year=2013;citation_pmid=23417284;citation_doi=10.1007/s00335-013-9448-0" />
<meta name="citation_reference" content="citation_journal_title=Mamm. Genome;citation_author=J. P. Didion;citation_author=F. P.-M. de Villena;citation_title=Deconstructing Mus gemischus: advances in understanding ancestry, structure, and variation in the genome of the laboratory mouse.;citation_pages=1-20;citation_volume=24;citation_year=2013;citation_pmid=23223940;citation_doi=10.1007/s00335-012-9441-z" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=J. P. Didion;citation_author=H. Yang;citation_author=K. Sheppard;citation_author=C.-P. Fu;citation_author=L. McMillan;citation_title=Discovery of novel variants in genotyping arrays improves genotype retention and reduces ascertainment bias.;citation_pages=34-34;citation_volume=13;citation_year=2012;citation_pmid=22260749;citation_doi=10.1186/1471-2164-13-34" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=C. R. Farber;citation_author=B. J. Bennett;citation_author=L. Orozco;citation_author=W. Zou;citation_author=A. Lira;citation_title=Mouse genome-wide association and systems genetics identify Asxl2 as a regulator of bone mineral density and osteoclastogenesis.;citation_pages=e1002038-e1002038;citation_volume=7;citation_year=2011;citation_issue=4;citation_pmid=21490954;citation_doi=10.1371/journal.pgen.1002038" />
<meta name="citation_reference" content="citation_journal_title=Mammalian genome :  official journal of the International Mammalian Genome Society;citation_journal_abbrev=Mamm Genome;citation_author=A. Ghazalpour;citation_author=C. D. Rau;citation_author=C. R. Farber;citation_author=B. J. Bennett;citation_author=L. D. Orozco;citation_title=Hybrid mouse diversity panel: a panel of inbred mouse strains suitable for analysis of complex genetic traits.;citation_pages=680-692;citation_volume=23;citation_year=2012;citation_issue=9-10;citation_pmid=22892838;citation_doi=10.1007/s00335-012-9411-5" />
<meta name="citation_reference" content="citation_journal_abbrev=Genet Test Mol Biomarkers;citation_author=I. Gorący;citation_author=K. Safranow;citation_author=G. Dawid;citation_author=K. Skonieczna-Żydecka;citation_author=M. Kaczmarczyk;citation_title=Common Genetic Variants of the BMP4, BMPR1A, BMPR1B, and ACVR1 Genes, Left Ventricular Mass, and Other Parameters of the Heart in Newborns.;citation_pages=1309-1316;citation_volume=16;citation_year=2012;citation_issue=11;citation_pmid=22971142;citation_doi=10.1089/gtmb.2012.0164" />
<meta name="citation_reference" content="citation_journal_title=Genes, brain, and behavior;citation_journal_abbrev=Genes Brain Behav;citation_author=E. Huertas;citation_author=K.-M. Bühler;citation_author=V. Echeverry-Alzate;citation_author=T. Giménez;citation_author=J. A. López-Moreno;citation_title=C957T polymorphism of the dopamine D2 receptor gene is associated with motor learning and heart rate.;citation_pages=677-683;citation_volume=11;citation_year=2012;citation_issue=6;citation_pmid=22487365;citation_doi=10.1111/j.1601-183X.2012.00793.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=H. M. Kang;citation_author=N. A. Zaitlen;citation_author=C. M. Wade;citation_author=A. Kirby;citation_author=D. Heckerman;citation_title=Efficient Control of Population Structure in Model Organism Association Mapping;citation_pages=1709-1723;citation_volume=178;citation_year=2008;citation_issue=3;citation_pmid=18385116;citation_doi=10.1534/genetics.107.080101" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Kirby;citation_author=H. M. Kang;citation_author=C. M. Wade;citation_author=C. Cotsapas;citation_author=E. Kostem;citation_title=Fine mapping in 94 inbred mouse strains using a high-density haplotype resource.;citation_volume=185;citation_year=2010;citation_pmid=20439770;citation_doi=10.1534/genetics.110.115014" />
<meta name="citation_reference" content="citation_journal_title=Cellular and molecular life sciences :  CMLS;citation_journal_abbrev=Cell Mol Life Sci;citation_author=H. Lim;citation_author=Y. Z. Zhu;citation_title=Role of transforming growth factor-beta in the progression of heart failure.;citation_pages=2584-2596;citation_volume=63;citation_year=2006;citation_issue=22;citation_pmid=17013566;citation_doi=10.1007/s00018-006-6085-8" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=C. Lippert;citation_author=J. Listgarten;citation_author=Y. Liu;citation_author=C. M. Kadie;citation_author=R. I. Davidson;citation_title=FaST linear mixed models for genome-wide association studies.;citation_pages=833-835;citation_volume=8;citation_year=2011;citation_issue=10;citation_pmid=21892150;citation_doi=10.1038/nmeth.1681" />
<meta name="citation_reference" content="citation_journal_title=J. Assoc. Res. Otolaryngol.;citation_author=J. Ohmen;citation_author=E. Y. Kang;citation_author=X. Li;citation_author=J. W. Joo;citation_author=F. Hormozdiari;citation_title=Genome-wide association study for age-related hearing loss (AHL) in the mouse: a meta-analysis.;citation_pages=335-352;citation_volume=15;citation_year=2014;citation_pmid=24570207;citation_doi=10.1007/s10162-014-0443-2" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=L. D. Orozco;citation_author=B. J. Bennett;citation_author=C. R. Farber;citation_author=A. Ghazalpour;citation_author=C. Pan;citation_title=Unraveling Inflammatory Responses using Systems Genetics and Gene-Environment Interactions in Macrophages.;citation_pages=658-670;citation_volume=151;citation_year=2012;citation_issue=3;citation_pmid=23101632;citation_doi=10.1016/j.cell.2012.08.043" />
<meta name="citation_reference" content="citation_journal_abbrev=BMC Syst Biol;citation_author=C. C. Park;citation_author=G. D. Gale;citation_author=S. de Jong;citation_author=A. Ghazalpour;citation_author=B. J. Bennett;citation_title=Gene networks associated with conditional fear in mice identified using a systems genetics approach.;citation_pages=43-43;citation_volume=5;citation_year=2011;citation_pmid=21410935;citation_doi=10.1186/1752-0509-5-43" />
<meta name="citation_reference" content="citation_journal_title=Cell Metab.;citation_author=B. W. Parks;citation_author=E. Nam;citation_author=E. Org;citation_author=E. Kostem;citation_author=F. Norheim;citation_title=Genetic control of obesity and gut microbiota composition in response to high-fat, high-sucrose diet in mice.;citation_pages=141-152;citation_volume=17;citation_year=2013;citation_pmid=23312289;citation_doi=10.1016/j.cmet.2012.12.007" />
<meta name="citation_reference" content="citation_journal_title=Cell Metab.;citation_author=B. W. Parks;citation_author=T. Sallam;citation_author=M. Mehrabian;citation_author=N. Psychogios;citation_author=S. T. Hui;citation_title=Genetic architecture of insulin resistance in the mouse.;citation_pages=334-346;citation_volume=21;citation_year=2015;citation_pmid=25651185;citation_doi=10.1016/j.cmet.2015.01.002" />
<meta name="citation_reference" content="citation_journal_abbrev=Clin Transl Sci;citation_author=A. Parsa;citation_author=Y.-P. C. Chang;citation_author=R. J. Kelly;citation_author=M. C. Corretti;citation_author=K. A. Ryan;citation_title=Hypertrophy-associated polymorphisms ascertained in a founder cohort applied to heart failure risk and mortality.;citation_pages=17-23;citation_volume=4;citation_year=2011;citation_issue=1;citation_pmid=21348951;citation_doi=10.1111/j.1752-8062.2010.00251.x" />
<meta name="citation_reference" content="citation_journal_title=BMC genetics [electronic resource];citation_journal_abbrev=BMC Genet;citation_author=J. L. Peirce;citation_author=L. Lu;citation_author=J. Gu;citation_author=L. M. Silver;citation_author=R. W. Williams;citation_title=A new set of BXD recombinant inbred lines from advanced intercross populations in mice.;citation_pages=7-7;citation_volume=5;citation_year=2004;citation_pmid=15117419;citation_doi=10.1186/1471-2156-5-7" />
<meta name="citation_reference" content="citation_journal_title=Circ Cardiovasc Genet;citation_author=C. D. Rau;citation_author=J. Wang;citation_author=R. Avetisyan;citation_author=M. C. Romay;citation_author=L. Martin;citation_title=Mapping genetic contributions to cardiac pathology induced by beta-adrenergic stimulation in mice.;citation_volume=8;citation_year=2015;citation_pmid=25480693;citation_doi=10.1161/CIRCGENETICS.113.000732" />
<meta name="citation_reference" content="citation_journal_title=Journal of human hypertension;citation_journal_abbrev=J Hum Hypertens;citation_author=R. Rosmond;citation_author=T. Rankinen;citation_author=M. Chagnon;citation_author=L. Pérusse;citation_author=Y. C. Chagnon;citation_title=Polymorphism in exon 6 of the dopamine D(2) receptor gene (DRD2) is associated with elevated blood pressure and personality disorders in men.;citation_pages=553-558;citation_volume=15;citation_year=2001;citation_issue=8;citation_pmid=11494094;citation_doi=10.1038/sj.jhh.1001231" />
<meta name="citation_reference" content="citation_journal_title=Circulation;citation_author=K. A. Smith;citation_author=I. C. Joziasse;citation_author=S. Chocron;citation_author=M. van Dinther;citation_author=V. Guryev;citation_title=Dominant-negative ALK2 allele associates with congenital heart defects.;citation_volume=119;citation_year=2009;citation_pmid=19506109;citation_doi=10.1161/CIRCULATIONAHA.108.843714" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=K. L. Svenson;citation_author=D. M. Gatti;citation_author=W. Valdar;citation_author=C. E. Welsh;citation_author=R. Cheng;citation_title=High-Resolution Genetic Mapping Using the Mouse Diversity Outbred Population;citation_pages=437-447;citation_volume=190;citation_year=2012;citation_issue=2;citation_pmid=22345611;citation_doi=10.1534/genetics.111.132597" />
<meta name="citation_reference" content="citation_journal_title=Mammalian genome :  official journal of the International Mammalian Genome Society;citation_journal_abbrev=Mamm Genome;citation_author=D. W. Threadgill;citation_author=K. W. Hunter;citation_author=R. W. Williams;citation_title=Genetic dissection of complex and quantitative traits: from fantasy to reality via a community effort.;citation_pages=175-178;citation_volume=13;citation_year=2002;citation_issue=4;citation_pmid=11956758;citation_doi=10.1007/s00335-001-4001-y" />
<meta name="citation_reference" content="citation_journal_title=JAMA;citation_journal_abbrev=JAMA;citation_author=R. S. Vasan;citation_author=N. L. Glazer;citation_author=J. F. Felix;citation_author=W. Lieb;citation_author=P. S. Wild;citation_title=Genetic variants associated with cardiac structure and function: a meta-analysis and replication of genome-wide association data.;citation_pages=168-178;citation_volume=302;citation_year=2009;citation_issue=2;citation_pmid=19584346;citation_doi=10.1001/jama.2009.978-a" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=B. Yalcin;citation_author=J. Nicod;citation_author=A. Bhomra;citation_author=S. Davidson;citation_author=J. Cleak;citation_title=Commercially available outbred mice for genome-wide association studies.;citation_pages=e1001085;citation_volume=6;citation_year=2010;citation_pmid=20838427;citation_doi=10.1371/journal.pgen.1001085" />
<meta name="citation_reference" content="citation_journal_title=Nat. Methods;citation_author=H. Yang;citation_author=Y. Ding;citation_author=L. N. Hutchins;citation_author=J. Szatkiewicz;citation_author=T. A. Bell;citation_title=A customized and versatile high-density genotyping array for the mouse.;citation_pages=663-666;citation_volume=6;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=J. Yang;citation_author=T. A. Manolio;citation_author=L. R. Pasquale;citation_author=E. Boerwinkle;citation_author=N. Caporaso;citation_title=Genome partitioning of genetic variation for complex traits using common SNPs.;citation_pages=519-525;citation_volume=43;citation_year=2011a;citation_issue=6;citation_pmid=21552263;citation_doi=10.1038/ng.823" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=H. Yang;citation_author=J. R. Wang;citation_author=J. P. Didion;citation_author=R. J. Buus;citation_author=T. A. Bell;citation_title=Subspecific origin and haplotype diversity in the laboratory mouse.;citation_pages=648-655;citation_volume=43;citation_year=2011b;citation_issue=7;citation_pmid=21623374;citation_doi=10.1038/ng.847" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="High-Density Genotypes of Inbred Mouse Strains: Improved Power and Precision of Association Mapping" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10.cover-source.jpg" />
<meta name="twitter:description" content="Human genome-wide association studies have identified thousands of loci associated with disease phenotypes. Genome-wide association studies also have become feasible using rodent models and these have some important advantages over human studies, including controlled environment, access to tissues for molecular profiling, reproducible genotypes, and a wide array of techniques for experimental validation. Association mapping with common mouse inbred strains generally requires 100 or more strains to achieve sufficient power and mapping resolution; in contrast, sample sizes for human studies typically are one or more orders of magnitude greater than this. To enable well-powered studies in mice, we have generated high-density genotypes for ∼175 inbred strains of mice using the Mouse Diversity Array. These new data increase marker density by 1.9-fold, have reduced missing data rates, and provide more accurate identification of heterozygous regions compared with previous genotype data. We report the discovery of new loci from previously reported association mapping studies using the new genotype data. The data are freely available for download, and Web-based tools provide easy access for association mapping and viewing of the underlying intensity data for individual loci." />
<meta name="og-title" property="og:title" content="High-Density Genotypes of Inbred Mouse Strains: Improved Power and Precision of Association Mapping" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/5/10/2021" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Human genome-wide association studies have identified thousands of loci associated with disease phenotypes. Genome-wide association studies also have become feasible using rodent models and these have some important advantages over human studies, including controlled environment, access to tissues for molecular profiling, reproducible genotypes, and a wide array of techniques for experimental validation. Association mapping with common mouse inbred strains generally requires 100 or more strains to achieve sufficient power and mapping resolution; in contrast, sample sizes for human studies typically are one or more orders of magnitude greater than this. To enable well-powered studies in mice, we have generated high-density genotypes for ∼175 inbred strains of mice using the Mouse Diversity Array. These new data increase marker density by 1.9-fold, have reduced missing data rates, and provide more accurate identification of heterozygous regions compared with previous genotype data. We report the discovery of new loci from previously reported association mapping studies using the new genotype data. The data are freely available for download, and Web-based tools provide easy access for association mapping and viewing of the underlying intensity data for individual loci." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10.cover-source.jpg" />
    <title>High-Density Genotypes of Inbred Mouse Strains: Improved Power and Precision of Association Mapping | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__34Arr8QetTzezDWW4Etyv7gbgH-x-ZTNo3yO15Q-gHU__Mk-lyyG5WoK4SRy56eUq5eomxN7OGXjpDv-Ob1a1Q1I__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"o0zkZ7bQ1C3fmIF_jBwk08Ptak2Xx55QaOpheiB7ZvM","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460038","apath":"\/ggg\/5\/10\/2021.atom","pisa":"ggg;5\/10\/2021","ac":{"\/ggg\/5\/10\/2021.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/5\/10\/2021.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;5\/10\/2021"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;5\/10\/2021"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/5\/10\/2021":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022jZBcwhyomp9LTIK3t4y6Yw2lzq2FDRgP4TevaJdxBu\\\\\\\/Dsmh40DeZMJpazapcnj1Fim64VlqR8zwhLPKJWyOhUkZrcU\\\\\\\/yIfhn7uG02nZqDbMsZK2UGkos8GY5zqPZ6iy9RHEdV3RYxoItf1njQaHikhoZe6SfoJgGhpn5X8O9T59lvP61lK+9x\\\\\\\/jfBTrY+AdMckjw5ycgoy0Cr6gXnIyGsD96vb+jjxGM80S1AO3r92csTW9VyW73aCwO8O2jhkrSV7U8XSXBjCJjdW5bTmym\\\\\\\/XxpG7j8PsQ\\\\\\\/RLPtANhIOQm+NALScF5Q6an+7UcMY\\\\\\\/6WPPsPwId5VHk07jKvf66EZ+BE6z+mb91DG5h89vQwB4bEgD9WpD6t3EISSbfz2EWul34Kjf0XUOHzVSOs\\\\\\\/Wb3QljCNntWmrNe\\\\\\\/sjIhEmCICne\\\\\\\/lqTdtuJijum0Si3f\\\\\\\/LVFKo9f4HNai6WSkiAJGRXIbsD7Z\\\\\\\/5+RyOpWaDgLCK1EyRZuujMAA5502k6HKx9aeYvcL0HcNlpBLrfzE9n3lKSvLsLmrDZ25rqdywuMwRt23pQGbiDXdPGuUNqo6djypRI5Ro9LZ5ON0nT8Rv15yIJbgUfH5kYAudoMacZOG5mvEnnw7jgO1OtWSSzCEKVFbgVUWFpTNbSMaPiJKEffLMtygMvcc4ZobiHOu00RcRA5fMqa2+tQiUTmdcqqeSF3WNDgrixe6cwtptQ6jao4u5rgTHaY0PbBZoBC7AFx00Gocm6Ve60ep048QHIa04gXS6iffu1ZWsfJg7Wuwy1ee1TQIX2leUb2jwNMdvRpsLOVNPj4bYOcLJoTEzX9d4DoqzplzKqFAFAyTBABDcrX7zQ\\\\\\\/TwOtKaYwpfaZ\\\\\\\/aNoFMEA+XwPp5CDI\\\\\\\/f3HyDbioINhPDU5\\\\\\\/vs+7PK08U93HuivX9S6HWRjRI8Euaz8eZVEqztYQ2bIATGm5fy7UBCVt5RI8sDiK\\\\\\\/BKssAGayk\\\\\\\/bQ5CAPRNWWJnQHorLzNYB99ggORhMmunLlYTSPY3fkvh\\\\\\\/wG2CoGEMEWrf17V5bqPAfrAZTqLj4QUe\\\\\\\/U\\\\\\\/n4VY+WpJwVzimT2b3LfJp8\\\\\\\/P5eOhBfb6EfIij3WAFaU1qZv1uqrV5djZnWhY\\\\\\\/2FkhGqGrmz8XpuRsvBDThjZAN6d3sKNYl57TpSyCDuIJmPnD6kZug8NtQALcP3mTHWdfE4VDYhff4gkvzhC03HQ1NdreUl8\\\\\\\/fiAnU9eXcAU8SOBrPQpL9RaSxR9QzbI7NUUhoRUi6BnX0M4jLHC9T\\\\\\\/Q+UbnqVpKnTtJb\\\u0022,\\\u0022iv\\\u0022:\\\u00228JnUtSyd0BH8RRvhV+N3cg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002203d9193d2c7f925104874262ecb5bc0e7af74b4bea1991d00300b47d73962b23\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022oelmDqxLTpYsFTPPhnLR6yKBdrdKE2IruVhGcqJLXsHQX1MfXi+fjcCl7vCFx1nqj3c6XXkYsSS5f6DquwHqHytHF3zTQNlKk0wa77UYi3Tc\\\\\\\/3sKkX6PJ7cXrK36eFnJXnELrYAuLxBOpbNNEAgCEIKpcknYaC+CaFRjGWulCBU4KhXojaIXuKvUdsJ967pOk\\\\\\\/RQ4d\\\\\\\/QtWGjtOhIS2iiBb9zvJx076clxGtqbVznWGG30ZS7OH8M3HpZbM+MLzgbL2AlsF8S5GI9MkKq4\\\\\\\/lVj+OdaqhZsOD3LWS0FH0WEuxyoXXpjc+hfiP2canZYUUdYAodq9VRDDY95lG\\\\\\\/pYBqWnjwbeT\\\\\\\/UknbRPc5UkR\\\\\\\/oilLhwkI3ACtDkqxClFmpp6OnwlbyeagXZ29\\\\\\\/A2ahTzc0HHtBUxRGW5OLAWpL7\\\\\\\/rVMJnteH5BFWzeKuJRCDmLQWv2SnMmnNk9F3FeSktoSzYi\\\\\\\/eLJacJojzV0LanKrIyUTHoduJI7cGHrDSd08r5hiL+KWybuPVGRuiIPn3grwOOBe9WviUDkJrKm5RyJicwtjbDc43ebdDP8L1zRaRgySFJ9YR\\\\\\\/ehoG\\\\\\\/T3qR8gG3+3SYRpP5tueSZhYwc44r3XfxwkCWBz63hQq0pbq91LR+avGi4HihAOGTS8ZWZ1D4soZHI\\\\\\\/i2Pa9YtCMsb6sWGlpLm7R4ZCUWcenwm1fM4c4xHh8FcYJ8x1f6VeItyWCxCB\\\\\\\/1fhC2KMhCke1xsszJWixbexxqd4mxj5D+TvcbSiyXNPhVc7CePnmMYFgocw7Rczbsc9tYGNYzMVh2dYCGCykdknmDKI7gUQy6IC11OgAeQrPud8pMC5N5MFteCPu26YMnEqMAGIiHjezdavpXJWGbt7jhkcLeY0RfICvlRYWJUdUF9IawZO9+Sm7Cnfy+6Yh+ZXPlCwjzJDXXrPHDoYHnTjfx2VSZ+kl4INYCgwK02tcRVeyCg2HghfFXcuuYjXgVWYX\\\\\\\/RU3IAD1D4kpPbATArDDEnKUb6+bG1Xgqk7zHI6k3Zzq2vZTGihbJyA6tTqRu6RjLOUlZz3Id6MMBsF4fRMZlzkDAp3YuGl3TemL\\\\\\\/ggDqNOglacCOxkXz42qW1gFn+94HsqjnS2+Qj3BxQ\\\\\\\/GUxQe+FybfxKEJL1XNzDGIggcH6rs+X5j+JVKvsrRnnPZIbaOFO63XkEIQVSAEi2SAedxPzT2n8USndF23\\\\\\\/CgUco3DZqpwTy1gkmbOZNRTgp6\\\\\\\/IzCkaG2CQWeHAbrW+N\\\\\\\/+VDDUwfAOYH\\\\\\\/u6KMZdayRcidx06xLvO4j0ycWN9GmsTcrTzYVu4ze8S1I1exhfXt7EjLY7r8dIimyFedPvuCaPSsFdjOAnLOZev\\\\\\\/H4Ua9K3xasReCjqnfqVRvj0VtswUFiVNotTuUTDxdlWBFrlpI7ObaUS4OwwIYt+K5kIllEZXxxyZwHJTS4\\\\\\\/ixH8ygKcX8Eru680UGyr0FCwfxjVzNJ2tJEXeXle9p8BvQ5dCw\\\\\\\/PnZqyK5IoAsoPloUwvaiEs4yEbjfTDmud9O22QtHaib+IKEZ8uvxtXS0kA9Syit1P4kA3gEaMscVkox8Bweu9SSIDbLh+jKLYJIjoLtZKPavsuWR9tc9W7r2dv0ulrI1TLyBhMrpHl\\\\\\\/VHOzZsre\\\\\\\/wPzep9\\\\\\\/SeDZFdwPtHIGkQTWT+gQnV+cqFucZA+G2xtHDU7I3pt9dkcBXQzAV9GnSn4kLi\\\\\\\/NjXva1HbSJOcuL7TMF+LgpkLWgcmqamgliCAjF6BRxOAscNdBC2j9xp7tw5ZESc7Yh6K6CtHm5vRoSP\\\\\\\/m8YwJKw88aRBu3lvVGpJsTIZCSsszqt60nTKRp6b0vvIo0KIFmrPl+YoqygQNhFo67Qe3DkENoVrxOfH95rzaCRxoQqGVOaprXDWNLdHHsyaOIhUEbuNLowEVIZMYcCFk7RgGB+EOQWgUw1\\\\\\\/TF+bNNUhtS0FOXlKYXzMBxcMikD0fLKhzqqVjz00IZzeFFHW1zVxtJUYFPxqItnyUbe84H7VHlPx45jJZiDvM6Kgwb+FTUYfdMiCSkjmn9KvR9AngkHALEVADyNQSgFig6hC+4cYPQnQi8lsmF0Ou8fl4L0CFcL1WEChJ\\\\\\\/oBiiZVZMylCFT2RVnQqQ6S63GC97px68cSKgCOH9YKWKFHjbRDFutoeNvGbm251QBu1D0AGaUDH043z6UJnQGi\\\\\\\/CEtOyHkGDuDitrePic5i4wGzOvLmgKOCDuuqbWkEt91qFbf8TAjUEREHQzMkU2\\\\\\\/1yHv+zEiDPVkCl1DFPmyIBpAB2fRU0jmMYDVULuqpdaTpJ8ZZHVuWBWrUBj2aCj4LJxZCXwqa50gNxBDsn9wK0YKAj62lLYO3bOGk2tJACAa9TwaeLV22zXeVRTHIZh7aDO8lP4CMNUmdomakQuddPv9elF+GWKJHsdRpDUljbRZs4LmIqnOnYHwVF55m4xrExuCxgFaQI6tge86q78q\\\\\\\/6lU6n0wHzxGldQ61PoOpjPylJZs6Vs8Evm8ywuVfjxBI7eOnCZzK6fvcUgX5noJgZd6VdJ+I77UZR3zNLAfuRk0t57fR95Noa7zWSeBAjBUTCXVh699ctkpKVfsL1IqbQzvIUXsNaUySm6RICVR7lUccpMubY9P5MSfPfZENkVzENko3OUeDA0DQSB5jGEU90QPjGigZPaXo5qkBhyTyHAWdw\\\\\\\/gc67s+LlhkDQl0mBw43XEzUvjuAUph7BT40F87fqp9fIR3UJKOxO2ELWCmMWOzNzdS5daNdXvgskvjh8dW5BlgOKlI22TWmfsTj2wHHd+\\\\\\\/GJoRvIcmBG2vLU3B\\\\\\\/dKXjhfB1uRp3B3Z721Q35K9RNc2Td2Vgn\\\\\\\/wYmoM9QbIV3+yG5aj9VTSNcZFFPNUSlhL8jB5LJEBl5L5ZdH6Rw20yof6WIXObWJXNyKyRnz1vcQfqgJOEV2WsPAQWdF2sX31u5WfOaNxI7yrnRcFyhH+T1srebUDQ4Qecq72Iifz3QJzOmeA0nC01glyszCDLBOrhdOBrqlsNy8\\\\\\\/AL8J\\\\\\\/i6vPR+ARyg4LmGJnbOSE8FeHYyPjLBvrJ\\\\\\\/QQQkVfMxStkJz6PT2hbHQyndvJj7VpLisfMXmGCgR9377TjKVXoiQ\\\\\\\/4sFZ3x41zmAeclSr0v9ZlVdzUE7GM56QiPiVLfWYJHgj7JhUscePmvGeqV4lx2NfuRgpoXiKnXRoeipcgz4uyLx2Gbgn\\\\\\\/awCaM90eagpusEmO2BYgMSQucQ7FT9I7ShwWMDeQMmQCseje4o3hrT9RTJoUQ\\\\\\\/HGlQ2ybPIzsKytXMF4Ne+k3WMpcejUNsA\\\\\\\/AUfxkuSFhH9CRRLYRGGHe0IrPRXARgzN78IZTDPwO7\\\\\\\/gAMQEHezmvhHsfZtC0NWcW4CsmRKYgMPsqoNAw9dCCGVgX2DZ0yD1psvjOMVksfTliXyzlYABcpqpGFA9j9\\\\\\\/2+26hg+Wk7wdiyN1XOSARarL2hAV+CQ9BP2l+qc\\\\\\\/fQxerhFJIhDgtCme\\\\\\\/NyJ\\\\\\\/O\\\\\\\/De6T6rVgeYx2sEdUqmA1JTEk8up1JxgLb3Xz352rMQOih09SkF2cufcagF8\\\\\\\/vgL+MSgIbC8sBbg3jqRn9lC2HYxGvhdQopKRnbn2LX5YrJcuAhlpJ\\\\\\\/tZtRcl3vd4gaszHyK8qUiznEthSbzm30N+\\\\\\\/mgcvXvbz70mU0LW6dG\\\\\\\/+NS9seYTrMS2gBBEkLua0LYMvjpjgjviCZWQBfF5Fr\\\\\\\/Er3\\\\\\\/WuGIXjiDJTVbEuvsLdVt5lFlQmQVSbU1Lgd2MNMJexX3Kpmhy5x3W8tuuHdKHs2j7FvE1rCDKZBwyPoQBtkGKxfxhPw2XnxarFux35czclbnxmqpVNWB3NplMgp2+gyMaaerAkbfIvkN5BVFbd+Rvhzd06r3pgBl4y0HsxxVacOrJTmSYvVGaa5g1csco+rdbkblPIdrxjy\\\\\\\/Atf5Ttn5DDSAFW56rs1nnbOrbOH4D+ktStVnmY4881ZKfBiB4POPaxaFaWenwA+Bji23vJ8kps7p8\\\\\\\/7gvLbEqJTMGe0o5HSBFVts7FMYax\\\\\\\/iXKCQ2Mg7GRipFta7faQJ0Fzrbv2xnLQF7crq6CTbQeZpASkZogACBVDwQ3NdajXQuxE10lNZwv3Xw97Iv9Sy6OwAwICxh9m4YLjxzv5AbrbFpJs\\\\\\\/har51nAFuprFM+2TBEta36Hn3\\\\\\\/+pS3KGwCplIC4UgBACaJmj3p+jFOIelDi5\\\\\\\/q8DY9HkpRu15oGf3qekHVK354PSgTNzUktBme4e95zWBipkWId7Qb90GObYiKhDDC+8TFxzQKCm4sPIlUfzrHRiiyVaxRA5Z\\\\\\\/8RcY1Gl0LbNEMC19s+T138BJoNnl0XvOuXi5m3bysixrzqzFZ7uWB4uH7YJ9vj\\\\\\\/tHzZGh7rM0EZsSk4XKKe3Nbz7KFFeUkcNVV3rNVmM+Qcavv\\\\\\\/LHBcWDB4rh5dEzN8k\\\\\\\/VlTLHbS2lrt9s\\\\\\\/8SSqfNqaaZkaCDFnoZEyUrAbe5\\\\\\\/LEpXBGPOyEPgZDe1+KioRcQ8P8ota5fWiZPitap7Y\\\\\\\/UdJgtkJOird7W\\\\\\\/TXdwGbqkTayv5WSruXfQ6MxHTV2ai5or8jw2wpKsc1jx1z0EK6z5I4yUryCw\\\\\\\/x349JbERaraUfrejfavsmVfDn4Uw6CssS9c1UJYm9z+2aKCj9XX\\\\\\\/k+1VI7o9EKmpzGBCq13MDZFxcLOk19ew6qOvZv\\\\\\\/zJPEvDXQ75Ckk\\\\\\\/LkRR63VVdFZIUBQEydlzlz8CodYFwwVN03nPeHMDvl4iXdChioK7wHxAkmsd7FciGjKLeeTQ7CRuzO\\\\\\\/K0TXXplHP\\\\\\\/pMwRAX9ZM0snfTVmgvVwwYxsFe6xg+AiuRrEOmewBhJwnEEa28io+ircEcAHOLRQyDqGO1COPEuhuItM3FDJhSBRKvxKPj9U2\\\\\\\/wDLeDhaIaXExjLLYzegCdUvS79mYusVrI3g+UenPTsWbrOCcx5kO1srHtlDP1H5MsD0Odvi7\\\\\\\/prB\\\\\\\/16gCBzebllyeE8HLfqvOdaftPhhJk\\\\\\\/IThWCgx71\\\\\\\/3erXFrdN\\\\\\\/n0r80pIcEynZFOGHWawD6IsxkL0KpoDGPYVdaB0cDUZlmTtsznN+WgFU1nKd9iA2rWgzavr9\\\\\\\/VQRHgt6LW\\\\\\\/eZuGw3V\\\\\\\/tuoWnMt\\\\\\\/WQ122\\\\\\\/HD657EtXUQoqw\\\\\\\/vHcl8f4UMjRvsG6ITgqSCqe3itRSX3xxwi2nk6mmZzx+e97pMiJQgv54AxdU+AVC2wbPHiZuP\\\\\\\/x65WMbAISvn0qygQvfP2Y0hCZikSUap4fEGPsh+CuTtaondSjkpVldaxeP8GkYgs8tuz5z9mAMvUUweXLhVsf8zRWeKVNMVivNNPqbZIxqk4J6REgKjXoh\\\\\\\/D1I1oxOe3OspxvsV+Crtp2VJzGibsaIcONWYS+CiZhVHaDUBQvHtHb4pJhdcxO1hRBtfTrHxcq522XOcpoawOh7yQRrWw3m+gi8y4b7TyQ+EUHs+G5J\\\\\\\/BikliV9FhF8\\\\\\\/g1esVvEe3IKUDqcuzsv6PBPt37qmpbz+uwPagJNyx\\\\\\\/6hKkuySjH3c3sZfIlF8sibvY4phq+J4p6dmxY0kjtLhwfw3eZ8+Vs8Qc6\\\\\\\/Hik+ApDiZNJkJtONft27J8VCw+9YtxbEjwrb2iFcDMqsvGNafA4kIdxY2nKqi5R5H+oL99sZHxyZv2Fw5cqUURDgjUkJEloQ9Elo8I3iXDy3s2NCXUFEIJrX5YMQUJ9DKbIE81E92FR9Sj0NPe\\\\\\\/dAJVjZmYLHqx4WEVBGxeDIbZ6y+WUDkhlSlTAP7rIaajzqWrs7SkkYXB7UCzsbtiHBAwINQi0xlNhS1EpSEoJivrkTSl93jULS6qNiP1h9LyGvmpQIV3B5rvcYLzDSsXjf1svFfu+xZDIR\\\\\\\/JIGk3m5plrlM4eRZQLy5S3XCQP4OW+oOwo\\\\\\\/4npd8cD1N4gXDV4KjFFMFBSEUQWcOVlqbRiRhhgXbP9yU8lr945mnTDCN9L5N0Ik\\\\\\\/GVwokpjIBAul\\\\\\\/hyhLFMu8ihizAAnipI\\\\\\\/f0CeyjoZ2AvpjDtnUzPbqfrPHkbvC3nMCalxN0GEVxzWyoyD\\\\\\\/RtffcUTR5J1VnXvkxUGOWbEHo96rU5ZSrcIPg7lx0yCDB6UFcE8eovnvaCi0G+CESkiiVeu8m+mZ4BXx2+BQGB\\\\\\\/2Q0DmPJ0PVkQzK+xpUWu222e5iA\\\\\\\/\\\\\\\/yDMoFjrL15llO+Gri3dLWOxPljMbAShgWtyl245ngeOt9MYbwov8WJE0Uz2ZMeK3lMLv0SdCQS3MflSFSmqtwaNJx6s3C22e\\\\\\\/BE1pjDq7wzY9NDTyDzXXQXVOgJT5l5+8pO4DEoAJACfc6YwxsTGvfuJe9dytcUjcacZa0EE0xHlUFtGJOkcV+nodxa0Ke9jzZhY\\\\\\\/OOuLngbyWHSejY4A3uDBCEJ1wx5\\\\\\\/TvkIRoW3IdsR87PIHNl2K+5JpXzoZFD43aluLH\\\\\\\/4zPwc0fDVSQakvExjVjSbSFTxJslI7XmfoR447+gVq4MaBmalWOZsSpVLL9tHPy0J7eU21MEWoqlpOr8PQ0rIBAsxTyDtKU4fe+sJ\\\\\\\/AuIUB3u490vRdKU+adqqtHKA3\\\\\\\/Wau5CqKX1e+QfosWorS0J+6Aa515+qfXEf+NnpkyHtAYnjf42svEN6OphoV+\\\\\\\/U+INgEo3XxSUqSR6HC3H8GlpmaFy9A6HdPe5pPG8jONZrCNmWsOedyQiIDtezss0r+umgYjbJuRERukB7DM5zLErIAeCOWcmperSt1eC+SDcov7MxpKCtdka\\\\\\\/s2rYse7f\\\\\\\/1f5t7GNg5\\\\\\\/sppv3g5+xFiDD3WM701Q2DxkEwcZJG0HN6h3K8Ne\\\\\\\/MwqeUbq+TQRsP6\\\\\\\/Y2xpW4gBJJHWQPQ3nkvvXicoXmfsyilZ+NmYyAvclP3UDoCL\\\\\\\/RFEsSs9jPr4aEJU1nSE3RiHbZhQ3vrb\\\\\\\/nVQEwdf+pXrtwF6Qz5F8co53ZOqb\\\\\\\/KcbGBdrUxISlsXUIT0f1BBOUTn0CX6QBWs1pO7FlXQLu3UF\\\\\\\/MLTkKKLYKTO1XbHxIakz7A\\\\\\\/ArEEz1A7ciGeR3wRRYdrGzPymRpIG7VW09ZFwmXidGOpI\\\\\\\/HRJ+N5Q3985ry3Vc\\\\\\\/57KcHcPbMRrx3Zoz6zjBQ1dxzzPAG6CXWJNwSd8rngAV38KTnf0OwhhV5A8sXbzLX9q5x95fra72TvGW1CIpKXKClqTrh9uzE1jZH7tkMZcTbv33K\\\\\\\/7hStxfC1OFqqfpnN\\\\\\\/Tlss1GuAbzAUrtXvgyf27VRLNHWNfAureOtmWhCVXgeSqm0NeJcd2QPpq+FheoDHiRsBNiWDaOPxyjePvH8fjwOX6n82wODkJEGFIE79HWku1abg00eiDs9g3WQp1WF32qfhUl2HosBtDIR6NIX3XKfWaqj+XYVZmFRHd\\\\\\\/wUCcSHIoedfKjKZ7Nj4rSeL0ecQWmkN2nIeb\\\\\\\/R4tdRl\\\\\\\/KiD7a1SibmnkGd99W8komQK45+Wn56zcfsMcw7+qxHXo5rfEjKCeiB31ZsaWz77Z83Qe4Ej7sTEavLoXaWcEIgGvxxuG8vTyxGjaHNe1Y+inCaWugWrPRIZYZR9hmRRiVLQTNJu32c0\\\\\\\/F\\\\\\\/oGlQxQtMDGife\\\\\\\/opL0I8ZxGGTy9dzk4uag220YygWg8eFlEXR\\\\\\\/vGEYegsZp\\\\\\\/IOtNsyRWFN6\\\\\\\/vjstcmIcjL43YofETeir9vmZFmsKFhe7Z84FinbBkGpp5YxZfGgi4qqUX1V9aHXbwK5PmW7r11L0WjC9iyuDDAP0Nj4b\\\\\\\/hWUxm4TANDkjN+dw5PgCKNH0cMZuSQjfX7T\\\\\\\/oVAH6ZYaduaoG1fhw2RZFr\\\\\\\/Dzk0I0esLg0vJE2vVfxtHrmhmfVBLkzD1zbKU67iITaWwgWJkoiQtOC9xrd1OmIlLNw8cYEDq2TsugCf0lAnBykAaMWGWoGDWnDdPmgnh4rMCpNd5\\\\\\\/OloAs0AtV8HgXdqrAGBDrBpKN8M3vsRpmrxcieVeWnDnOs8MAAR0Fn0qHDi59gO3hDxNJgbq8+3juYwYQjESGycP90y+B2kJE7RzutApmN5GJi6tXbWY02ZUU4Ar0rlXJypHqMQNBa\\\\\\\/RB+MgvPeiGY6rONanz8OUirO4jB3HUVX18jB9Cd2wH5RMPArVXqSrPKgaq+L4raAgaXQlYG4e6MspbiYnV1TDEXz\\\\\\\/cj7b3w0qFgjvD0KzIJ8CENeZykgbCZYOYAVLDQEti5aMpICYpyPRxbOEI6fS6z\\\\\\\/17nKr9dYfLtMEQaI\\\\\\\/YA4VqCxXLaF7CaN2idHGz2Ey5RZcS3g33yCKf0ncXGaO6i+CPPOLly3LGm4uWEoiWdUJYS5XVL\\\\\\\/mLEN357HeqYHSGW4ZJFXTIVeqpK4VBRbOFx3wfHi+6zkM9LjJ4KtKmgBB8Jnh01FYoTLJzcofaT7G9AgNB4YzW7ZzeOZUhDuyTqdbL7t\\\\\\\/yn101ok7vsvRpTRtZC2n6JTAcp+pxiJVv0q5tdmgEtzyjqSF9nFk1ti3SpOaQ7OT6zhXWwRfQ\\\\\\\/vm4pUvM2jarSKk0ldKkcy\\\\\\\/9YOZ9TnxlnMJz3chh9zzpu4ALxfnrs0XDX1\\\\\\\/HmxW0\\\\\\\/WIemVuwe5DhZhq3Nsx3UOpgBjKqO7Pzp0NIwNRmfb+iBnqSy\\\\\\\/8UExRE4O130U9jw97U+17FnGzarY62hzzkPCDhaMYqcDr5iEqJw3jzJRBKvN1WRmxS0RYYarkSKZrleotoyYcO313jPv4uP8moZ8O7Jq\\\\\\\/UpUSJm3fKtVkCzxraKOUJXI8WE\\\\\\\/TWY1T9Wiw49\\\\\\\/eI4aSgedlxMh\\\\\\\/9HDrM43rOHt1jEkJyHxS4Tmsb5Mnp46qMYTHDnWBS0b+IEDaj1BVqaCeRnVrTeYJFNk10iPGjyId+VWGhzv1EQOCSKHK1Borfg0E6ENQyu6qsyEMqEnHX8Q+CWop22YzoL0kaaMP1D5IzdzI72cFqYosMjAOgtIath+x+QJ7bNknxthESbxeAPBhKk3MKVukrB8Q229UGm1sHTZ7ZmV\\\\\\\/a6xAXAQoToW3gXWsbvleO+odmIqbxFvqQW6jvzpPsL3p8GIW47W5+7N+RqDHzw06tfPYWWOSUjuRQHfO4edy3JkSSzM9r9Hi5GtgztIoA1pZje7NtkJHQAEQW\\\\\\\/7YTgkWqtcRgJRl\\\\\\\/Gll4t30PownbToGmxcSJkTBHNZsLv8qvWJvZwIDxhacBn95ROSju+UCWHMGrnHfmU7J42MS5DMfozCZPl5VVyvx1kpcuCr4+xWEPguOia+Xia6l6NTyQwt12zmxWFssO9ufCKwnZ8dBHGQI\\\\\\\/n4wH4whTxNFNNL4DvbPdEN8w1BSZ3hOulFg3E0STHeXJxQPAfVK19qNc3i2sXxDoDZhARPS2E34mzlpn1ZZFPO1Zi1WsU6sEfviwyFcSkS3\\\\\\\/fPzsST0eWu4ppORbTN7TzpvWI2Ed+u5ui70zlrz5k56usVc7Dnwn5oFBwECfJ6vNluPKmp8Jw2ndko5epi\\\\\\\/1J9OCjB3DQAq94e0OmdWjg0KiDGkTfeuPDQa2fh7n3\\\\\\\/aJFZZIrWTfXTDBdoeRp4LYWYurdE1f7hThc+v\\\\\\\/+1IRAuZvjmfjRuuKKHjdIrdG9utEye1i1h9Kcf6o2OVj1jesSfvmRi2GumxB7OEdqxSj8D1O0TRokHhkEuElEPTsF3H9c6z1MsRqUorGyq0A\\\\\\\/fXeiL3VNOtGcKARggEI5bdEcpqhgINiUHH1RkoUhl\\\\\\\/\\\\\\\/NdCMxl9IS36GwefUrIIj5XPVLW7OImDN3W0W+Gqo2UpautpXUqFs54U8d6jCcYy3dyqKXyeIkZAabrxpSiI2BIWWVzTVYIRz3ANRKeHSnkFuQ\\\\\\\/dtyBpTsRxvnmpfmu1H9CjDgId+dfPlcL9wQs85fQaLxyR5W1cVZ2wT+Q2iIa\\\\\\\/Q+nXf6bT0fNXVXtGBsZVxiJ5aTU83w\\\\\\\/EfYQsxZngeP4kU8UaR83GvmHAQmMctIH8K+U93mZDhQZC9pmNSaQyCPuvcRdLrFJ0mt3RRP\\\\\\\/YNG4vSLrjiAl9Osio00Y1O3X4kcPc5Yq9WnGmtCODqGA9E4adC+N\\\\\\\/6LD8cg76DaUh4QaBOMIaf9aTAdofbD3\\\\\\\/hRDOOVwExiErjgEtVfgi8xUUYbpruUYkQlo1V8jwoveK2oBiqTwUHadNnzffmrIJmApNXqQkwPZ74CblP2V27D29bQy7\\\\\\\/2G\\\\\\\/s51wN3fN77K\\\\\\\/A2vTJwUvQy9kZwRdsoFDX4yuH2\\\\\\\/l3sPZJjj2gDT0cCLPUkjHST2NwIkZa5yBd6W0eMHIKKgBLIgnb7Trq9\\\\\\\/sg8MPUfufcEkQG\\\\\\\/y4MTfMbIEUGdbHjYnULodG+yCH5S9mhOMKTpTLQpwe\\\\\\\/jFfX7ekrNltgJGYFVUz1Wx8RsfR9ukKyUfLNU6csaWhzRCtiLKS14s1L5LlucJifQwk6FQIGORzGsJYZP9AWLkhp9m5PIWnOYI+Rwxk0EguiVHdXMWGr75XcQEnL4fqtflZAXuIpDjT2e6forkTL0IPi4vMsx5vRdzmNlocqJJP7Af+TEps1y\\\\\\\/QdwqNj7uyGd0MqT6YzWINHX05ANQifGiQ7g0aPLyfdSe+yGK+qQ+SQInRL267VzjROlQoFFN8d\\\\\\\/pQhH3cKh+Ku6mzUNdPiGwp4PfsR+EJG1wNg8GoMdHtnhik4JC04lI03wzqWv00gv1uQ\\\\\\\/N020Qx0lOp432aI3KFLU\\\\\\\/vkV2HOVZPkXpSNfrbjVSXLwecVghbWwGJzf1OWhpuz1wHKgcbMQvjLk4ChPhsCHeikoK+NiE5XHesMWdi37pgkOASV\\\\\\\/73sobn3BSLVIO1wrq98lfIr7m0BX4ntivR5nwYtRIGWdQfckW9Dowy\\\\\\\/BPdZSu4Qoc9IztbJHLOlarKsLGpfmdg9ngbyQtW31U62lUZTqYlnOKmFPKs\\\\\\\/fkahTlRw8A\\\\\\\/XrZbIbO\\\\\\\/VsrEyVMN8UXkgqjTY+t8bTF29Bt+ag9bqEyW2idrf8J4M7IQh9Z7bshiuPBHutj5MCpkARg6O1\\\\\\\/n4ug23NMiSU8iVHmzhWqYgF+beRMnqhRJcztOZ\\\\\\\/36p3ETfoevkXktwAbtNxqHjoGxcSzEtPMdbsXgQrxGYpPovDJ7PLXPPYwC4eCk4+qcdynO7O9cj9i2\\\\\\\/9\\\\\\\/ROuJoFuGDE31Ib6uzNwnNB8ia+v17rwMXxfVCbkzk5IGSvZ0Iem+TO\\\\\\\/W8Un1r3ll\\\\\\\/gEMAYMlA3Rx1\\\\\\\/vnd3W+0p3TqZDH6OfhD8vvUxH5+VX6CoNE2VXfmjF7C6ZUBMmKIZCEm\\\\\\\/1+3rMYkWzvjcaL\\\\\\\/rcaGXJlX0giX5qbtRgNS\\\\\\\/MzFUXGzvKogE0N56RikifpYDsNSJHdDRs2HcADZw67Go4bdzOcXixX\\\\\\\/a7zSK8RJ9heR7WlWTdnNzSrrOyHxQrVw+VxhImSxmegZiGoNSqkn2widPEqaKXbKZKtw4WikzQuX6Nhu65+0iUo9oJPWSfJ6L3eV8YIUf6uexMgAK9Kj7G9PQlLpEdIW7\\\\\\\/RnJVKwWYAnAiWJRez3+ni1auc+PkirBx6pI5xZ0nfr9UMCTNzkMmxnVOh6IZQwfHsZhOlG7trvat4Bl7mGsPC0NF9EdHi66Lh8OSYDU3\\\\\\\/gWC7i3Z6v08YNrH3KQNBAkNtmfpfwxXumhWh9d\\\\\\\/w15yBP1mBhfkI18Re5wVNRsYLQBJ4P3SQ26zrWrNFZ1pSa9JjSrnAFYV9s9TBZ\\\\\\\/C4LxAWnujJrSQX5V9f8mjPzKo45cHzhy5UuLGblkxhs3s7GGTD9Swnanjeauo5WTE+cd3WJpzjz4xfOOAmwatezzKO9ALfya25ZmfWUILJL7qTX44UCIrw+Wb+m8LoWbglxhWsYWVo7UO2zWOa4TAH0j9\\\\\\\/iFjS7s59hCATzAjGiENM0mWur6kWSgKwiUgRLX3DklPVeYN\\\\\\\/2qXorEy27yV0yvcaEoi8L\\\\\\\/bImU3ED6lZTlwywc7w9x5JMdaJeS9jbhlOHJINDfEbCy9IruLdyFUlrG3LrW4MuGU+PiXOVxCXr4uQSP3WyXO7qn6d\\\\\\\/r0D\\\\\\\/3sz8P\\\\\\\/I+GrbiT7is4Zeu777k8oa3Fij9hXRAf4UwgcgOMLqztxz8dnrTbSMkFXfeSuQ7rp7xn5vAM6cRzGh2WxATT7J6icTLgoODmN8QUE9vZe\\\\\\\/oeMGCBQiPuTQBlixwjbgcbe5pooJwevd3llblJ08VpeLS6S9mqhSkO0Q\\\\\\\/yozrcbvbc\\\\\\\/mNsgEqJKpFpUrr6iPTlgL5HlZtgyiqeMTaWXqtLY27Fp\\\\\\\/\\\\\\\/1a56Ebhl+43AaUqhdgVMFpAC+1NFcPn8L1MiXLIoqNX\\\\\\\/8bfB6evKu5O6zNw2C6gSMGm4H5Z\\\\\\\/toUKPbLLiGecYq\\\\\\\/x7xcfR8wjjajR0RwlAZVTO9cAyMySTQjorHOZ9IFzldrOSl9J2iBLdtRa43xlo+hHjRdq1Zakmte\\\\\\\/MY7+TiWm5+JXmq8NBv5mVIRpJgwYncmywu1UvEPm3FHI1FVVLci4DeWuRob0vC61P8Vtt0T\\\\\\\/MNwhJ\\\\\\\/jXi9B3FOKG1ooiLinLdGQMH4U1BeHq6k781RkdH9tkJDpe4ANndi5kLEQRbKDOwZ0sXJAyzAQRGXxMTaPj0lSpIiwa7xA2y1vIXXr2xqRDlT6rSgA8R70ujpOkxvXzr2ytr8se8RV7BT7\\\\\\\/VFE1My+qvbb\\\\\\\/a05eOiSm2oo\\\\\\\/+rPAozS3fxWUYpazKCm85wS736Nzr+JKNq768PtN2bmN\\\\\\\/4iTpbUB2UzzplkXns+a+E9ekpq+HMfJYlGjfoCFCxccOa4ziecBKRMJjr4Zdom9YwGswAsf\\\\\\\/ldVbVVSv3s5qmOOK7VpG9vORAgHzfsTYSDXoUpAw8fwJiX0vhQwl8AN8W4cxABPT8vclfddm+6Gi9NvjXdHoQt2BmfXg0S8VFv6ZdAHYfdFio9PsnNTyh91OGt\\\\\\\/AmzvMFaU0k9RFeyE1TaJIj\\\\\\\/0OLOkhAt9OCKRJbsn1iWDHWwOTmUsS5GShITS2108G\\\\\\\/gfAlkb8xm\\\\\\\/3HixJh75hhAiBBBMmsnVKnl9jDrV4IgfUhsDeeiumLR8gWYm9M+jfV6myd1ajaIAP80LYbqCY6CxdhbyuFfmgQek4OoJwv+4PfN7ykaqK3Tki0de\\\\\\\/hlkGasYKzT27zfg0PtGUN2eGisbp2gLCUUuU\\\\\\\/yH4u2iNmoZ7MQ5qMipLCHb8jtApa0Gd8kzl5YxP4f0mx+QJTY2kPyep8xBFJtWxjGLxNU7im0a6KUAZ7oBUBTeJnqwZqCCGtsmdcWITUS4azFeNWRdPiYxRCy2TJ0SoqNXOsToNbhadmhQGcmwBa5vYrp4kF+9nt5u+T4w6ec+KrwKtF\\\\\\\/N9Wc\\\\\\\/D65IJOucA\\\\\\\/xEPcdV2vy2YW\\\\\\\/YSv8OSz9RLV7imkum807aLPREcD4XFdwSpDWQOwlh+1DkHh2zj+qdhkW8CueZAgJTmmhZ5tv+V1VRuCtyB65yJKq1\\\\\\\/AH2EyiECb2h1C8q6qgzzvb42hc6nyCRCuU70UV44sR+0fy8NH2jgWgAiAA6MzDhSv6R1PEpkc4qDaVTJW3SQyP18+zjw\\\\\\\/rNg2mQY5lJHleQKHlIQ4Cl2ViseLk68akqQf8E3Msh9KfLZCb5imUvr5Y8s9HgqStOt3SNkr1Ipbc8fw\\\\\\\/GiJL9mfKNYnCUE+kTwmZmCl+oJLA5ee4UJAOEy2MITeHP0OIdKrLUP3sQwKbAn1dh7WQCBlgPn\\\\\\\/SgdprxU0elYr1RT2XfMQ8EIsogmYJjHtAAV4dU6gQDgYYXx3mDKo4eR4UPPKHUWhtuP0HXl5qNXeFnmaM+jQMS3bSnqqWN\\\\\\\/65uWPusTrSq6K5GVbZAHPnxCgrtcjZ3Oozy9DKZbyt73egWI2Xtfl8+dlBwTlBdBVriAO1QiTYBqQRuxl45TualiYSWS+vkb9RHXGnqY3xS5psQpp7LFS9tktkdAA2HpH0q9HIuQ1LIQiaaxuAV7sKsTJq9SmoscxFa0EED6C0EVU0Ea5WGOqS+7RhAH3TVuYmt16dfhDBPDzclCJABkuanoLWDj9ojFfh8JAe6+HC9IZnvysvoVs3i5jXw5MHlGvu2eLQP0ZDnKE09MdXt+FpjaDVrUJ\\\\\\\/tQ5XYLeRGTjBcs1tXanpjsanH0cXj5\\\\\\\/suc9QXNztZ2ecHXNLDNYEoAKer31IfDOpVxdKFW0ZIJ+jGJamnJsgXs6bvEmCrym0aLh9MQC4s2n69ZbVBYNHjvOThPj0ziwZkFFuJDykzQc2RV0DgF3213TLR\\\\\\\/vbCojmuy\\\\\\\/guIalNtkfJgiWk9ch3piuAtwjctg2FdAGoD\\\\\\\/l37ytjsI964XKjmRpb9O+FKcY4TNNhiQooShupzY4zgfTLbFot8QEHasG5e51yYkamM4UlDMdm\\\\\\\/gBIS\\\\\\\/OeTZqH\\\\\\\/bbSNlV6GZYxDlVgCX1DV+fkfadgiS70n4sC14vSE4BTUlA2P5bSQZmuix8MssFMfQS+bDN+Jkmsd0Ode6AZEIFYCnUPj0SVbexROrAnEYpl6kubapGsxr1T4hrhCeN6uyvIDszQtg2ml4B5qiBxbS1ZvH1zJViw7Aa3H9hG56apr+\\\\\\\/WZCgPHeaVhJP9pOCzVPs4NKM2TUkm1rzudRVYiaJRAdzANodBOMs4cqxZu+ohk6QvvKd3bDYgiKOPbfaZcHkQbZR+Rszepd7H3aTA3NNsxxgxtz7+vRMWe3PiwFmcvraaMQmUxOcwxk2Hucj0O1De\\\\\\\/aa9ay8SwSSpnC+GNmO6nlVHQxUj9F5jHggYy2qAgKjurmaUni7CADLptXjAO\\\\\\\/uyS9XGtdXb9JyE4NR6LVCkxGoUdKrR3sZOKHy17UWYGn6TpKNHNluu0uL\\\\\\\/W83\\\\\\\/oqOpECPDsdmYhwnYeaA1eW4aBQ8VHOCj9Pw0TRYJRxYBINdx+CsALzN4zcq0\\\\\\\/uMLiji09hPPDCvVCmLv2gYirI12\\\\\\\/gchJXFbez\\\\\\\/eukxE5zt8oAiLxYlrUNqxNsyy4ouo\\\\\\\/DhfDjJ0Y9kR7Y7Y+OJPwUcOt7pb0uVOhY6wHvi9mrct\\\\\\\/CXAPpz3ReOFwhoLB1NPfoIcpt9RgpJ8\\\\\\\/y19R2NuFKFDif+4M8Bpglqyk0voJT1y7TKOpTNDgUPX889RHjBCooIMgraUo3zBfCN0WlhTSjyRNkUTNpA0E+4ynHFMny+ZN7ChwHaWlImBN52BYg1V2K9xoZA+\\\\\\\/qaQQNBR1S4JRcD7mgGgS4D0DcyHn8wolmOSTu3uy29LHk\\\\\\\/uyOeKwt7GMfYdhifo04HeKYyBvKXQq9eTbYyl\\\\\\\/VFtLkW1CfY8UYwS17Rj+X4ZvWbhYwM8885zZdpfJaQPDI5GR8B5i7mHwLDQ9aiB5HevJKjnpHPzE1TVN8bMwi9TMDpcXtlP4a38ASk8v7ycmwjINKTHn+LjJ6ETjTCNDT1O\\\\\\\/h+eadVnsZrWgI2oLQnBMAfwDAHMFUaOo5+YD0ELPxQyksaSdjy\\\\\\\/JsyZihakYh3XYS4MQt8EzBYUnsEdbImfrHldIqD8rRhgL41zkg2kTa3n60Dr+V32wyz1WGXyjdDvt8BgcnWPTzFCcUl6maTSJuXkGdSfDDRtOK3wH2cam8QtcQMyTF8VHfEJAx8Iy6TugLhSINIZp4vnODpiTZwiaqseGqe+LMZmJiQ3YDZhRn+8YyeZ7NVra6MJsBHBV\\\\\\\/yVdHe6VQjWQ6cIWKlIUN7OT4unJRl5ssrAzqk7BsNhHN7NKfWu8GfDYUNzsLv\\\\\\\/7+m1DIGgYNr8IlnrKQxAzbGDcy+hc9aC2hWPn3yes00PwF4Ohe7qwL7pD8o\\\\\\\/fSukTiN+faFer+eVhp0KOT7SCE36qiyyu5YVLtG3b9wi+1UtDBET3qhXLSuwElPOiiAO+Jqyc7bRoLMTi+io59Hux7iLtIV6b+nQaK+tybjjg0Tg7aI4913xyj6Jv9uhhULTeR5TXxIRSysUA7k+H4VlyX+QxRJFau10lMxBHDEMS1IQpTZ6YEkjSMfS5eFgw0Wnti0twp85ozBlX6cSrBA2UKm18KeQYBlRnGNYDxZ82n\\\\\\\/0vhTEQOeR71NY7yC+pge5I5JtSUaZTTNfl5Nq9Adf02F5qynazz\\\\\\\/QlRTg4WnkVExOArYuKY\\\\\\\/0\\\\\\\/l9CA6X6rgHWoalDFKYSSvJ25iRAGA1oIl1UreD0K+3FBBPWhPWeQ4ze2SQNdYt8jNNaKk7Vsbrvj6eH3n\\\\\\\/JSCGIeJF3WPNt4cRsM43kmfuSYD4L\\\\\\\/8Si\\\\\\\/ek+uk3ubb8VXtk\\\\\\\/sysBJVoYyOTOE1DalJwCGr7aC8Z87vCOqy+gHymW78Ld5DPDaKGiDsZGYNsy\\\\\\\/tdR8B7X70D4wzlPAjuK8+g60u15HXn8Mhe9YJ9g6SFf2ukQHQ6B5XJppRPsX0v1fAJHIzUn7e0PjA5eH07ZppKf1HkmVqNX0QyJaE47PVTqx0C+iKVp4d3pnO5\\\\\\\/x4UJo81zUk9e\\\\\\\/BjeI3KCWk9X1Pih3GtFNc7k5gZ4yW5q5K5wZ\\\\\\\/+5mfKbGsUkTNcsP1RLe6mlazkJlvGToG5mKGll6FkBuYQLfZn5gIbxmpTAL7UVYfNQ+tL64edtXCTPABiPV43qV\\\\\\\/TNGcYqp8UZSjbQRQqKNYlM8xY0y5nCMIy7EUR8RgSMbKGpF3xnmWTtA445pEpZDidiB0ClN7DNXsYK94B9HF81DXuwgPt9kiIHkLyZfNhNzYdNY6lkhGkIR2ims2ITZJw5+N2AS899cjXlmlwzsv8fBAcPnX6R+M5lEJHr+ysBIgHzH7V5WdwYfC8HS7LynJPazplfES4DMr6xgvGjw3tKb0zKYi9qMkgSBMSjwu1AKZeKv8rWWZFCrTrdEfAE2TqAOjd5Rr5iERg1xf8vqlE6ouODBAZW3Ee+wGRE8JWgUCwPpanb6rXWXGnkkQzp2llkpNMcGNzov4L2hnf8Jj4cIDpA7xPCLlDlEOlwGSalouDoh4TPH8PVKP6ymgJwYtHYfj9yZbH\\\\\\\/vlenDdVScCJi6OptY4TG+z2XeIoQUMrmrx3wgoYpbe0vVdzEFqV2z1JluD+zB4f4TT\\\\\\\/oOGkxe18Hbvfrhb6Mek7dgfR+OkG4p\\\\\\\/lJqUJWTAHQHSAEGuYoXqh3KSNTfbAwd75imaIBlMDMNn1CeMyHYrxNwmDPIOv\\\\\\\/z4SEiDLPtn789JjFRvdPrKusx37IsI1onCbMC4vaxkf9mu6OBRP85KgAEYZ7fkpi7pi4d4q8R3Y16ypm+QL7uYDfsmjbKpQl+XTkdz1QW0LEzxXOF5EkVTbR5WIYKkJ6Up33mZoQ\\\\\\\/Qb\\\\\\\/UApwVYlUf9Fnnb91j4bq2auWp9eZgFDE6nYDnFz78L5wmPTSc4nGI7NaAa5KUI2+E1odQZfE2xD635ot8jAYiM7fN4pNHTJDsk4BSGHX96+G1BNAOqh\\\\\\\/Oz1WvknTzTw\\\\\\\/bLhwxyle3Eber9GagqN8lfZ2OKyFjder4Lim6Sp89WRiAWFiZqsPAawhMCd61yoA80TBBLROqP5fKwSyC4FmxikaZGqPQU+4lnjKs+mfLCihVQH+wYiGxFeJWbYqz\\\\\\\/z5MvLu4NO1RNatmk1J\\\\\\\/tltjcMCnWy7IjK3qqOZWSinOI3vHc4Bjm+243kaJ5WZC0iNJUGv6C\\\\\\\/xS2u4gP5SBtInl1J0QW+uJwhI3ZHUvnTYy3\\\\\\\/Pt+ZW0okK+LYV8YBV8MP+6BsKDqXqAVHCHUzXe+W2\\\\\\\/9KcvzwnB9T29XO4yKD14ywpji5GsfTS4N1wAjE46+SLt2lUp\\\\\\\/V5bLXrjC8s9yBnRUMYEpjGsvHcIJL4TVN5N4Oi378Y6fe4qsgYe9sJz\\\\\\\/SNF55yiNQKdw3w0F1TUpVVaYKWMfhyUgtwzhw2p1XEqZZs9mIl6TGbE8YUyDpoIEEVqZyj3g1GJyo9XcQbcSbcUuG5sMUq2kQ7nUQUWKxN5jAvQrTIk6zoRHVdD36A+nBFivyl5IgmfaWMjToIvOx\\\\\\\/8pU5OMmwDVwBaaUbqV+DzRi+xQv7dJCIgay0r1bmqfS5TJ1vYe7LFd6fvvhlZ4pcMcSsIayJ0Y3IG+lnTrWok8\\\\\\\/\\\\\\\/XqLymSpGMoo47PyaxCr32QH7G78NAQLIM9mrysheBQD7rM7UO93MbchNl4CpA7gQb\\\\\\\/XL\\\\\\\/Q3Z\\\\\\\/xM\\\\\\\/WvbwLwhrq198vFsdUvxZH23gjErsOXADz+k3QKohSyJoLh5H91Re0H3\\\\\\\/xjb8b5JAsDEG7fD3wPGDJUS7HgzgyDyp7hTksJuNLksddFDIlz9wqHdYEWEJVz9kb6nk4UBkSu4z+Jhj+oruJrtB8b4dL5ULBvgQyj8QvqMX9iz05y8OR5KadoMUP1kxjS86faYBTgjjFsDNLJNdQXuSiQ6yvRbaE6CffluJuaMMwBBejiSHgHE2+ZNRHgLiZLIpDPwPYmCekYypl0bRwo9QbT7nBLYrQF6eYGCf+pZVbAllCUmb9x44WmDBAt7lo+FByFfc3hjGwTOMBiSfmT0qKHrwvOGWgo3iq3k7f6Ywuj2wdiPdWsBOKIHW4VAEDjp3JGwhlpLwsWvQEVLitXjofxYBK+Ul0rp1eRFZ+RR\\\\\\\/RorHOomUfQDn0HNVhk3yCpZnAng0CA4cz95hGrD26txE8\\\\\\\/rfBFCKu53ZfvW5bV1\\\\\\\/tcyqpf7bD1XRUauj\\\\\\\/clZyC\\\\\\\/nAU8h18VHPxsZWVQaEpKWc5vNXVqY41DwZVP494fCSS2CRpy8vjs6EQegFgxf4CdWbiEmHWqnz6T4pkQn1cTqp1pAR7VBNBpbjooV\\\\\\\/1fgxPfmjlZMvHhwzI9eHRDJbipyPCK4h53H0u2quO1+xjPy9gvCY7RrTqIhIs\\\\\\\/4uAkzvYodUINVZKpWWX7h8+P6DzZ9Vwwh1ZR7Fo8qUzAlmjxzblBgxRQY8M7TLvVwoZbhDHXlwVIxI2dvH6lHGwpJ4jAQ\\\\\\\/laIOnkjkFxKceeb\\\\\\\/BI7fvXHvVX9q1BLdlekt\\\\\\\/904cQTQXAs2qc32OeMv3A1Ma7JTdHERvodeVTAXoxCvkb+uri5uECpUJChyzpYEtqi7ctJbbzrDPFsFOAdYRTVzQ+PC8YSFI8AwYczobqhnpkJ27sUA6MfF7NENI2qCTn7EqjtYoRaDJp8VQ\\\\\\\/GpqNl8BALWjkC3pwXbVOrMrgPkq6bP\\\\\\\/kURM2S6KiqlA1JRNhpRJffk1GpMGrHbd\\\\\\\/5P\\\\\\\/jsadapHgZwJf8zUQ0Mnr0Yykz3olGf1pMRK4A+TpBg3GSXkl5iFO\\\\\\\/DzVs4h5W0eUr23I3lVQH+HVJETZBcGHNTsW3ejvTvG7f0knHXnNdWKQTYp\\\\\\\/sSy5ELkRyjXXHqJYxfJKhNG4glsW9aZc6wpHvzuz3leZbfbW0NRN2CzpSeA5RGlY+EUgG1k2jkPnkgLR8sdAiwG4kIi\\\\\\\/65iyyAYnY0AAkk9WllZdDpRg+YCsZL\\\\\\\/pe0REsSuQJDfDVFLIZ0vJKGzRWYBORyQVeOby4Yo+HjCJm7Y3UhN1jk8SqiAhGXwHxBS2xx27MmAvEDc1C6mC+9iVhRByk7ZIpYViNtuHDe1aUD0TH2qiF0K3o2QX6JGwXVWx2ghYT6eDztJxBkgX0\\\\\\\/5y6GGGy4bYLl9geYFqVmXqye4CQb6s9xg+eAhmGgelLHX2AKqSAL8w7ezPa6R4uMau2yx2HmNj6u2VLCFCW5KhmDdUKS4Jiy1MC4hMBe6pSuBopA+ZTzmHtnF1mXcOri7o1zg54Zxvz8+Ji1T0TgHxCTuU0S5CjwNiBIA1NGf64XWMgulD84lqmbbeM5BbDhJ+qW\\\\\\\/ETjVWG5yG0VkREGCpWAPDOTkVGI23Wyw8BMzb6TyOM1YWKEKa\\\\\\\/CyMSyAhhPAIhxRZmVBwICb8HFX2lkqSvqrO2tfLFv274Gn3f3K\\\\\\\/PEe3Xrpmt88BBTmOj+sNsXGAD9tIBhac+UMnXg+KfxV3D6KUL8Lv\\\\\\\/zGnXeZLWDq\\\\\\\/bxvAgI+29BGfigqFM40UFQEn7se0vqQwIfKjpLZI8f4pNSwXdyQqM2ibqVU7eolqtsE\\\\\\\/Z35OUKe48O9+Id2\\\\\\\/XSDduyJqyF\\\\\\\/wW4nAMW1JrNBPL32ErwRGJSEjk2WBwdI9Rl9ad8ks82U0XkeAjzi5wcaU1E+G8i3ljpTkWZDDdeIwcgCL2ih0SGlE9Vv0MgbUbZPtAmD3oi5iBoTMBAivT2HLIK6PlZPPbws+UEuB4ASGorK+QJOQXVA4g12R5\\\\\\\/iqt3zZwBgz\\\\\\\/dm+ZY\\\\\\\/aw2a0rTIc4P0nYJ0o+68nGLEQ00PoK2ntZY6FDUM\\\\\\\/gaZ6VyIIOJFf9hEd7gsb7VROv3ZpcOdhgCFmCJo0fvQzDTCv1Nwa2K1bWSv+GspayjkKdbn2W5s7qOdXGYJr9Ubta7k6KApXpLkjquEIO2c49Bje\\\\\\\/oJkbG7nsuFSjUKhYzT\\\\\\\/0HjvS9zyS8SyA7F5c1F9jh7cvPo9zi72YlC+XHPqJUQjvZyIlaIfuLzAHw\\\\\\\/LIe85sfpOvmnfGHs+AXRpuh9EXEBEofRo07sk5Z5kPJVR+0pJUvIFC1KzslGu8XRC0mCazmBfpdOYhOcg2qzC52515suplsGaNETH7cnj\\\\\\\/0dBkudo3ZdcPuBWUzFGV18DLiESIfpdnpko7noQS0jijR0cYEOtiKuRYqj5RbGd6cCl368XnsS2dR9tta06tbn5EmMfEc5tBZ3W9gP43BWbb9lEzQaYfn82SAIqT3jkW1cC2\\\\\\\/V1\\\\\\\/rxYC9xKhRo4o0riIoxkEOVE2p8wcbIDYMweq2HzUefrpvFc8cAz0cP1w3Xm7ZFBwUsYkOafVMLzziSVAr6vU0JXBV+2oB0\\\\\\\/sLnuqkB60I7eWIMLlurBe+fwI\\\\\\\/PuHwTVIp9Q9ZvQ8z6ymfyyFxY2sDdXn9dUQmGj+Nlvt6dX0vg7s6UCIFoWpNiTe5r8lMGDBfyZFm70RVjCLZNLNS6KbMPsQDUCB++UBchbIYg7\\\\\\\/AZIYeGPr9VYFaldBOocElfoTXjFvwn3BU1eeoCHEZzdJoszEDNrSKvzFEMJky7DCkEzroLejwkLiezYqyU8shenpPHfJEkJAoE\\\\\\\/eHiYHJDR0NVhmZGykFuKD6lfgB3IJnrT9dOfqtwtfb9MFtIjc7g\\\\\\\/JD0A0Q002Jcg4CI181rD+qhtNsO0hC9e5fOsM4Ky1Ky6XrgG+zA2tJKC45LssrY6ZCNFCr0LIJk54Tmcw0MIArtoLjPtRB3bLdBLYTbU32LtZpg+zA6JrwHce697NmPDeFV8RBrIoUbFWGuqg5j3koRdo2H1gfmF+kNDisDBYMPiCtyQG+RQh4eyfNegxQ9BME5FQ8rqnDIWywR3s6aSF13vpuk26ZsFU7OkSRvhspJo6xghzlH1y511I746EueNY8IF24msXlEOw0fxD3V7fY51UjpncHRiOrOK5AijnMaQmbJ8lTk94Vvtqhsfj7mWxk3Xmp9knjrwu8OKKejkOtnnJsVXTPO+T0hXEdXuhV8uOx8nwUC6YGD9tBk0nZAZ7oOUaB9KpOTk6+Jzt69hCDPpkmS2VnCe+AdnEyDQ+mGupHf2gDdHrPRbqe0iWTJ7dNUkI+FP8BxzSScKqvkEMqLDD+7HAMPF\\\\\\\/\\\\\\\/k9GLUlOX1D2aeOsLOIpxs3kafbZUYbOZVEbt3VXo98PWC9XRIg2v7uVdc1za4Qb8clsjOWlbkPTegmxBIJDnCIwSer1gXx0R2fMicYj676gygTwcECb870uBitedCCkkK65lFtqk7yI21fo0t75EgA8jzGT10H\\\\\\\/oHFCYjLXv8dU+4yb2zbRSlsfmPp5k8jKMPvAzBliSW0tuKLGe8WDeP8NjwjWthGTEVor3XGYaz40ojWAINvHshU8\\\\\\\/DWJ5fIYxltakDFKfvDNl4kVDeyjNHWYcSVFAlvF1\\\\\\\/76A9TxCIajGnvaDzpRAsaJgCD8eS8cW1yj1eATqmupfHg3cJUBUeywmgh8Q9kSrCMKSFOE3q5QPfr5071AsHvdJoUXZerLr2gdZl42wBym\\\\\\\/u5v\\\\\\\/MOYVAPNLYGqTb4R1rYa+kaQ4Gdje3Xq9YYNU8mTP4xgFfjr1n24LW5kODOw4XVTQx59t1QvioEEFT4x\\\\\\\/9wt35aUd4qSD+IEa8DKCABobBoZOk6rO+bb3mdUsruStNs4QwEen3Ek1Ikr77VNphavNOoyttZZgM+wTdNqF+20IIl\\\\\\\/O\\\\\\\/SWWYmIdLGvSiHAQqxnjuY7\\\\\\\/\\\\\\\/+rMa1nWHwA6i1YRIMbHvTk7MOKCnixnWfFPC\\\\\\\/f7r4ABiArWu27ONS\\\\\\\/oEZSARD5RZdCvJhZ5Rci2UQwiZ6QBTG9\\\\\\\/ZL25JSAsOTJ3PTZ5jp6tlzwbuzBHsXIjcHMujpoqO1JeJim\\\\\\\/FyzLIlOOLjvw+YKCPMGH1+fN9Ib5B1vKNiH9KKvCbC6TqjVLwE6B5TVnrlnkdZhMftwLD6t0i\\\\\\\/YHA\\\\\\\/PPWZflf1lttV1m5OEk0xTAU7oRaNryqEoNAOFsINpm\\\\\\\/t\\\\\\\/wXZfBEyFN7MMhNp5BHPsdk3ZPCtWIpPQ7KStT98rvNULIrsHFIZdrNI5Am0XOfGHKHd4tRNQotmYYCLgYenVuW2b5bya8ZxAp3hsrj0vXFwrWqdEmZlEDiU7RY8A06fi9JQ2wUrR2YRChLE+job8JFUx0C9XXvaJtosPEjYYtgXJRpvXflRDApeDc1tZFXlFGY3uqsio1q1CcvbdrY0IZwQ5lQUYQCJ5TpsuCZPf+dUkHajBq1UQp3M1i3\\\\\\\/L1XlO\\\\\\\/sww80I6jt8PncsX5hhdD5AHCfDSXKp6TBEZGKJbICt6OAcxVymd2G8yxOLIlj+b0XoUYaxWNkGYghCHe9TZgZHzrUXU\\\\\\\/B6ADp85d+\\\\\\\/QbIDtweIc8FmeVtAUmah+JN5fvCNtqpeuJOiSzHWD+2+jPNofFYOGmTgT36RukWECWn6S8ti2lK+xXzGcYQYy6aSYH2WfvALWkNiEdyAMfhEb4gJqB\\\\\\\/gfTkjkOHhSqyIxdkX5+7vbk06E5qc6n1oNX\\\\\\\/N9Rq7jJtyDZPcUQY25mmjlQ\\\\\\\/hBeB8ofEvmf3DP7zBxNvUMn8YBeDnXqiZrRgDWggxJmO8sZYXUnln5GeuSk5M639NW3uHwWWzoLz1s8z50OH33blKyzd\\\\\\\/zfB7NeCx9olUmaWRgvcRRHyH6k8hUF1+sbjiCSiGc4JSYys7EdE0+Asu1wRuGBCxl3kXazq58CqOgBOyKbT6byVjLRmnQiAi7dniOy+NUyDYioySbqIiDGqRDVdPBk\\\\\\\/R5aP3+eikXGTIWmiZmcaUYiokYEaCXmLw2M\\\\\\\/CATcVMmxiFE02Z+qryaUa+CQkIBPmN4B\\\\\\\/uYoFDBccuqMqW4mh1pPcOMy0Vq8KjZldLuEPmMcZCRzR6xLKBNVE0fYTrsb6nwCv4FrasClUC6uhLCdGd47dyHzN7FBbMemKF\\\\\\\/Lqau6XAuQT9cMxAY5eDZYTSFq3bUdsLKU2FR7lsH8A\\\\\\\/kZQlGPJYFTGHt82dalPW8fgiOuMMNmij0u2CK7S1bz4PiVZsDGZQFFZvuFtfpDWiYr+klKApf4yeVIPHs4jMb++opx414zFsOD7BP5tT5dmqff4f2N6vbb\\\\\\\/Y1argTa1uymDTQ2+yP\\\\\\\/EZezv0C8pwckthH+K0ioZacASFYs8\\\\\\\/OZi3CQUBgyE91iKjpIwb9Dz1n21kPsgMQ8seO+8njeQqiCK7wyvorfnN16oGxTCx65keDAP+4m9SjJI4CHBmyarpNJE1lJrYqc+sy4Z0YfHGMwMqVZ+LNRPusRXux+Vu8V7td\\\\\\\/t434WNwqUTienDL4Qok+s3W+973buspJPOOwH9rEmwjo28x8RGH1z5FefJmoEuhvJJmZ5TjFCBSpBsFTIBn3eSvf4UsYeFQbyKC9udx\\\\\\\/KTNp9xY2dqAiJP7MIuHu\\\\\\\/g5cWyumCs4La9TQ\\\\\\\/t6XD6VpKlWLO5vXD6JP5JTvaXx2IJ0I2HlJhPCa7227S2+v6Fy953\\\\\\\/zz5TH2nR\\\\\\\/EwwGwQf5DsFxNPWJuz3ukzuuOsN20IlWIGm6HdhVjxxVk5fkLn8gU3j2dkp8ZjF1jTV\\\\\\\/q+yagfJel6OECDTKLKD+yfvryIijtNq3EfE\\\\\\\/cRoa5uTVEgdeOzSo7pmz5l7CwlAk92rveemhEzHXxx6fZpkckkRtb8Rwz+NvGIlgsxuSk7bwphrKnd2vlr9TiQZ\\\\\\\/vDB3\\\\\\\/PZmEgJ2+7F7k6AIXZOFAE\\\\\\\/Sxi8Zo59lBBTWtBMckMHg6XrJHSetW0IATcXa7tmNiFYeM2OnKCeOP6Jm3IVKO3gdE5RWAwAGR9qs7Ev7iSLoHHpii6PWWl8jObvg5Ypu7wCHI3fo+gTozkNkjENTVKf4u+QRX0swK1SD3huzKLIZO9Jzak4FYVvENPcru+a8boHAPpMh6YjceP9TE+RiKtKMq\\\\\\\/ErQ96tOIxVJsuIxtFCHrbYeM0D9oNQ5MEfhJI9hES4xAWVb+MGOyoVofNDqA8XDIV+OBxHok+1DfoBi1XrP5dMn0lx6psulK5OrPrSTHNniGx3JRU7a3Bu5A2Rrs+FTPNQ+6oBADLVF4FvwTYmC5hTMn5bLYToOi1KQ5d2YZIBFgzp4KEd1CbX14AbYVNl0FKnK42ibBJd\\\\\\\/LsPseD\\\\\\\/\\\\\\\/JkRkQF+SqyQr5UilzpaMXXXqSF6O+iIQX0Fb2SDPlrLVQKdhiQWLOTLDm\\\\\\\/IuyupEHgVv\\\\\\\/0\\\\\\\/vflI3lK9jPQt30TiYoCGzX+BFqnr2KE0I+hoDx0sECLBUFGRUH5UDOgAqeUx3SDmu0KMTI9P1mmSOxxt1FJeQGLpNx\\\\\\\/mKgkC9tHNCWZB7j1w1kkznvi+aYuTHt526cnu38zB9GxSBZdusPwXNvuGZIUVaA5712KJK+I68Ic\\\\\\\/8pxvZGWWngpVYy7\\\\\\\/ffvGK0YZVt3j1LrhgpNRZKWFxyfQ7reWYEAsrWJToUlIAPGbvRH\\\\\\\/4SXT9KQkUFm6ECPwPpGuDO90pX7dWFZq7Q9+fLL5g4oPdodfM3jQ+Mqm2i+4k\\\\\\\/JIH6Rz6vNQ5kXv\\\\\\\/Wv76s8Bd5X6mksfR61i9OyoqXtf24oziDEUuTIiAQDdoPlzKG+zVCvM6ke4Rs7Q0jk91cLtc95q11g5MfH+oSUuQdg5EK\\\\\\\/zPYLlEVnRgbs\\\\\\\/DahPj0Z0ElWf8j+dIcKvm2htpX4oN6R998bRv6ykomWhmO4+3bxRafiryGn5mzKbCfJVFTkWGYayatRJom6fTcB0GA4+WzbGlKgZC3cMaRBDCLN3UmyMaj3dJgn2d0sXiXm5TbeDSri0BRcMl1pdEhpLgyjYaKPbAnbP0AcOnjhvs4Rg0wG\\\\\\\/Kt3fhomVfoIZzhoBa0OljJqMBqnqFB0+DxvY0hq7yZSswonvHq86paQSzfsi86Oa7Ub1meBn8JQ0w+Q7lrogw\\\\\\\/p+vPMMckGD7BD\\\\\\\/GZjHMf3azZ8FU58ehMswmU4Z+WCPpCEANH0TEh6Ro4FQ7dYZL8rFD58V0pln1RTaDAgQjYdfMMA3gNJll4cIIkQFnHPVedQgMgQjRQjitcBJ3P1GzN+6O6QSzVRdNsbpqHBxqWbtoKqUrnLEmIkEifd9ng6pIYTf8i1wUwhIBjVUmYMWR\\\\\\\/tox2fKWVdir0gCXcHNRVBRtWeBztdFQWrmbZQr35u6mdAkYDU5OKSyd2DtI661+eWsWeS+yTb5jWTP6RF8l4GLzr9M98FculeM1aUZ3zPgaJVDEK3J5vDEyFugKqR0nQWUZDEgDcsdB2bMVmCzzRJVgV0SYi\\\\\\\/MQ6sErtP2sy6kq1ayTOeS7VjG\\\\\\\/MsqRMj\\\\\\\/zoNLRhgjaALaB5tpHJ57HgUn+tkpFYQPt3hnV9NVfovSPlj7PaPMXF3Hn8bkEHAypQcIJgKuZaPYHFoh6Bsw\\\\\\\/Ci8KI04pC9ExftstAY2fxb1T3vKKbdPrMuwUVP\\\\\\\/KHz7CBRn3dQdt40JC1VT\\\\\\\/athRepNJ20PDt9UTjpbAGaPrVh2V6NXotz8bN4+aH6WCWbNqTs\\\\\\\/k1MgdV\\\\\\\/pPLlIxXf6uYLOiYaAnOiRpXn+NNxAbZMmWcHtO7AkNKmly7o+meDAvmpajcChkFY8Ycs2z6BlEfnSnoVAas\\\\\\\/en7C0kn2NPK4r4IUiehnZeWzrYfilqVO\\\\\\\/f6bbWtAarnoeSDW68JNt03iqMNcm\\\\\\\/FAHXp4ZN7T2PkNcu\\\\\\\/4B4xkTNi6fg3eC2YuYLKZjo7IKDP713w1euliNL4Z+cApnFpPfd\\\\\\\/VNCNnE+sMTZ2iyig9igztlYNz0RM6dl9NJ+V8w2krytBf7FcxT5E64NPwBKz5Eu17iL7bp\\\\\\\/zCj7ADqGDDnh7Uzv+hc1DFa2W17ggbd+QVsEo3+9\\\\\\\/9gIpGbhVYNXZcw3PRO9sW0VEzHSu\\\\\\\/xVMurgNT8jKn3kjmmKuD9tfE4\\\\\\\/uY68W\\\\\\\/eipBV+YqAxMWW+UmVLnnIonEIFjcR7NjDJ\\\\\\\/N9ggBJfBiWr6ogV+JgVGqE9JXMjp\\\\\\\/L8RvUUEn+7k2T1\\\\\\\/o4\\\\\\\/XmKE9dQ7D6k8ITZYzf2sYEj51fBpewtETU675At1N+tTGB7BQXxYyQyIKoJHZF82T7jfQ2O+Djal55aWn707fSa0iWbYPkWooiuOixfIiRi4fFDrDctQTyp\\\\\\\/yfAH2J+2OfPp5r2+9i4GhEE9+Jwr32kOg2tdeiYZNArfQXEn\\\\\\\/RV7rNuRg8MRuiNrj0C0+ESWJunksQb\\\\\\\/8xObLRqP5x9WGmntk55QuLyuWjLoTpyKPT3DI1Jaq3eubkUNJiVhwA4PVhQkW+3NEOpzR7V6qu1\\\\\\\/x8nSggbDp3w4v8gqRoB5npQXImU2RTjd21Ask4EBcmZo+BNPJoK+Bopjwf0F7vu2bM9TJH+GfchPnvKyoJQUH3CDAcf1Jw2fjTKshBhXKnHUoSZVTr0Zk2r3bF3gDL3QbUUY4eVotNKmGOdZ2qrk+lv7tHgNze7xUuNSnR62U4BzlqeJpVsqtlhwNIwa\\\\\\\/WoQney+1AGXXy4b5LQxuC0d1ZyG9SmfhpbOFdl8ve7q8EGxNf1d7XGYLsyDTX6AyuGFgsjW0riOYB7DOraVBgsWXSOtDMakYSZ1d5zvfTpDWaZOx++Lti1ciqjknf5M6uxwTg3NvsdRFBxOzfOt8f5Rb07NOVAM3tk8LZgJjk2\\\\\\\/v0\\\\\\\/\\\\\\\/JxJx4cS72ckcsonkG93sDIAMvPmF1JrazNh4pjfTSeH0jNbeK+kFaBzQ2fn2T43p2uJDAjvTw5CMh8XyMCvsjEimMh0LVtjVnzR0QJ\\\\\\\/Wkwmel8b4bX8Ctq6ljcYDCEXFgp9rZVR7lenyzJtJH4K5mx8zowdE1oDNdhwOCepjE5roUIdp+1vj\\\\\\\/IgKwpXZRimWVqwCxwCEPo5p44YrO9xfT4bVHq6haL+ASTgIui0nvGmYke8AhCHMgFNao57rt1P6LRnHm7sl2blbPTft3Eku9xte89ztBweNF999LBjf+KHHDGR+Ao8IdI9aYLDPaHlqaQYxc7ohvmAHhK\\\\\\\/ErxZFPshSKDXjEPDDZvro8tm4WFfELl3CrVk\\\\\\\/6F+0nSy+eQXtDzemJIY+AYEVoVLXPcVsfiOXTCB2IAQhf39aB2zwmQGD5yO0w\\\\\\\/3+DjwapBAdKMQeKUouUjMSGzGAgqPxjpOc5XhZ\\\\\\\/glz6mp+6L5OxMy\\\\\\\/Wa5Z4qhXOuyYiclXd6g2Oq4tEsZQ0XJvLitD3Ob7YXCRMApv3YxQMqocHwm6CCqP7B3S+Wunf6mCe4pE25kHgEz1wJTw2tAyZureAZEz39VfpGzm6OJgxNxnrMbjVNT4qZbxkff24wRVxUqYZoZ8r0awBQ8glR6hMwG7OEIf4xLIUwTJoJNCM3SFEmWQKlDqAwSS0eHX89h+XiD65iofK6vR4UI9XcUIpZ2hK1R7GEJl\\\\\\\/m4wL23Ncpr5f+O0MKBMEQ\\\\\\\/vG2jGbsT\\\\\\\/A651JpXckD8rn+fsU4mlaBKp1TfgXKPeTromNrheTNy6XjTxGpfslBA91w9gJIWIx0KTnhoIuHmBV\\\\\\\/c1FbVNks3TPqjS6rORZR6kEacU9J27AixiE3JflVQT4PgJ6Fd55RgLDttLz7FalcGR3cwcYtyOmzCpJh4sxc6ApFPA3pX3hBfbO2IGOhY6eGyj83JuVnd3BDEW88NymTzp9GRxZNo6j12iMs9jUZiPVaQgiM3LWqA4NPNRb45leYzUe2YXzRKOKISq6uc\\\\\\\/\\\\\\\/FGhzqvgTYQkSdKeHeR4VcUgggqn+LInS5LbmtlgDP5AK4C\\\\\\\/IVxjQRF48wcp4kSocjl186p0\\\\\\\/1alPDj8GtLa0BVcNdyq89BUJq4ERT\\\\\\\/THWOXqYcupTg9CVHOesjFyrOWIMLyITAsjsV9ZQvo78dD\\\\\\\/x4nKn93UZkgVeHLakuh\\\\\\\/SqaAoHZ1MPDBOg8MbiGHtzeKF3AdEtBJxCF4rbP0LtdIdNmVQPGWn8TOE7jxD7DPcTpVCPKZpVP6AJrWxAiSv\\\\\\\/AHbrFKyh4tSZ9pM11RB0w==\\\u0022,\\\u0022iv\\\u0022:\\\u00228pHe\\\\\\\/mKKj6JJxQxSDll4KA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022ce3bd571a3eca172937e066e96b5d7a8f10af6b62e0d38fc106af3f386a3c598\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022nKJ37pqNek6zKSPnWQrOPIcOqaTWRUGraSsck+NbCMPG5SSmwI1BhGQg5kZEhix\\\\\\\/Soj1JwvN2rnScR3QNr+GLzSQCqfpfSnaQfZfc2VqYmLLK2O+uVP8Aayj3haHZRsj9Tu5A07VO98ogImglAo0E37wQop9AmO7Gu5jK5gBBFRBPlkkPbJWVdUxk5vuJBcjMBB0cu8PyBW+3VZrgopgQzLQ7InYmryMfB3hJdw7vdPd3d6s5d0wmrLnDgRn7HqXwOTtb\\\\\\\/jNdXnZXc7iKHVyz4OVNVZxMvD7MOfVa6ZKufHOzDj8AryNKelPcp1SIFkSYjtzPHKkXcziTscJzogC9X4Abw4YNjg5VtMIE9PsfljoOEZ2lDQFgC5T7dIdjoVOK7bjckkaiHJVKDq5z\\\\\\\/kbEgpcGOaMisfQunSnr1TScGW0wyO66hAOvibnlm+GBAx6lnovS7P6qLTiPIjlJ5z7Yez0mSQC650VYiZDmk1UmN+DTU2FrwcsK\\\\\\\/5bXG3t1TUTtF7I5aNPwn5golPJWSr3x0lxw+Il8WqnMeIqXdsNOFSghIYm4F6Jmo5ZMVDbAEIV5ItqbbJ5fYV3g1kVHRg6TesNXvZgguXFsBWXlKexYIeKGlJ61uPzLNczYYJVzJ4B1f\\\\\\\/9rM7x0umABP7w6S02bM3kgqPH\\\\\\\/gvgwWlpOajhHrTmW2+BITj\\\\\\\/SJk5WOPwXN06+4GBkVpEtVAOJMzz3vwh6djXzlb7YOLBh2dAZmQWe3auBQWBENCXqFKwVyZnIlGMfiE26YxDuv28zsMk4rHqS1+aysQqRiloINlKH\\\\\\\/HvyTxULtfJlDnDD4ZuwyulZkeWVhPPZ\\\\\\\/lCw9kdjnD2V5Q+iiVn+HPa8XBtEOH1SuRIQMogX1v7lkXye3sJXDas\\\\\\\/dsf8bvoU\\\\\\\/rBt0iuEBE0RMqrG9Yy+4GvHR+xx\\\\\\\/0g\\\\\\\/UdWxuSkO4tUSCHaQ8+bP8QOloT9Pa\\\\\\\/vpYRwrWsWsOLzH0O+7ksZosoP2qe\\\\\\\/mDJ6q2uLcKK\\\\\\\/KJMbnrCzfbzNwXt0TZhYSCzJP08Hyjg6MiM3MpbkgbJY5HRvVwGnOdimBmBfHOwvAnC5acTQhLwaMQL\\\\\\\/Nlj+SIGr3TN5naVmShkbVE75X+uc3aYUxfCrt3MZbcIfkf1PlMhmcdOo95QNNQBf4GVsnYgBaarNjoC27ys+foK0Y3XVSkbMLphrASfiWOnJ2MqCvp3MioZ0pNjF5f0UjJfsn6zr9pAi\\\\\\\/EGFGpSUqu1lndom207b8aSGuKvNkUKtxg2OxFQKZxDeHgFbdkny9DwtBhF0NhmuLdMLLQGKWuh2tmOwNm95Er6L3inhXg+HUaK1BdbGQRAjACD0BvWYQVFhb\\\\\\\/3gbKIUhayJ\\\\\\\/6YVJzyuxiZ2kCdEOh1KaIvsdZbpnJqlLKuxC2Wd8NivJ1S4povmnPmX3NMumCDnKKHcwkZstXTIrhcSUXjkmIXcs1TmFcf5xpMlj9SLwxU0dHUXvprx8IY7YEmFVUrP\\\\\\\/ivJOi23GFnhQd+Zij\\\\\\\/nvvmfG05zjvy7Hpo+IFp2fsFxKKZMCllOtFoHSE\\\\\\\/9j4HtGUnXXWny6zUtq7pApN\\\\\\\/3cSxg20jcxuuCVtlRwjfTG1KlGoeWiZ0u6\\\\\\\/tqL1VidJrMxpPMLUQpzt\\\\\\\/CXxwasoxyM2J4GfYkXeO6nM2WTyYHoF62jb8tccBbaujpUcR5f8MM2XhFjXExKG4v8Rm7Nl5I6YGo\\\\\\\/0eB6Pc7\\\\\\\/0Ix4q5VXu6j8KuroJS64LfpyWG4ku2Dfop+Qi8kpPYOR1nluYn7SqThtJH4BjDMCBeVjwb+a0d4tmUqNm0lIc6hcSmvlApHGX\\\\\\\/RoQVSIcMF\\\\\\\/qp+AhKpcMWk2JCwoH6wDYhroAQjAdMBgr+oNrsoyQ9HissKOBvUbvp+DhtmHwW94zjSC\\\\\\\/52HplnaW\\\\\\\/A782FLgd51FYw4u0ZaEM9rS7kGPqYuZ+HLgiOqSZUV\\\\\\\/Wn2Sx7cs\\\\\\\/FWY+r+2Uo8N2\\\\\\\/V4AGXLbIxbQy0myHBJpHkBsQXppGOkaCEa+9w8UbMNeszA==\\\u0022,\\\u0022iv\\\u0022:\\\u00229elrPwZZava16EYOUcc9Mg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022cbf2b3f27b2ea36b53c7f0256066e2fc9e08fe45820250a560bd2ecc001314fe\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460038 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/10/2021" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-OM0LMNGkf0QzAg3MRx2c8RSZTzBwLuuV0DQ86gifGZo" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/10/2021" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-8g_hYLvZUZuNNQ7Sf9bGwq0Ln_Pbw53Na3oqourXtN4" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/10/2007" title="Second-Generation Linkage Maps for the Pacific Oyster Crassostrea gigas Reveal Errors in Assembly of Genome Scaffolds" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/10/2027" title="GC-Content of Synonymous Codons Profoundly Influences Amino Acid Usage" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460038" id="node460038" data-pisa="ggg;5/10/2021" data-pisa-master="ggg;g3.115.020784" data-apath="/ggg/5/10/2021.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">High-Density Genotypes of Inbred Mouse Strains: Improved Power and Precision of Association Mapping</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Christoph D. Rau</span>, <span class="highwire-citation-author" data-delta="1">Brian Parks</span>, <span class="highwire-citation-author" data-delta="2">Yibin Wang</span>, <span class="highwire-citation-author" data-delta="3">Eleazar Eskin</span>, <span class="highwire-citation-author" data-delta="4">Petr Simecek</span>, <span class="highwire-citation-author" data-delta="5">Gary A. Churchill</span> and <span class="highwire-citation-author" data-delta="6">Aldons J. Lusis</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2021-2026;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.020784 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460038" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Christoph D. Rau </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Departments of Human Genetics, Medicine, Microbiology, Immunology and Molecular Genetics, UCLA, Los Angeles, California 90095-1679</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christoph%2BD.%2BRau%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rau%20CD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChristoph%2BD.%2BRau%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Brian Parks </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Departments of Human Genetics, Medicine, Microbiology, Immunology and Molecular Genetics, UCLA, Los Angeles, California 90095-1679</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Brian%2BParks%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Parks%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABrian%2BParks%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Yibin Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Anesthesiology, UCLA, Los Angeles, California 90095-1679</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yibin%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYibin%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Eleazar Eskin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Computer Sciences, UCLA, Los Angeles, California 90095-1679</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Eleazar%2BEskin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Eskin%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEleazar%2BEskin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Petr Simecek </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>The Jackson Laboratory, Bar Harbor, Maine 04609 </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Petr%2BSimecek%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Simecek%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APetr%2BSimecek%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Gary A. Churchill </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>The Jackson Laboratory, Bar Harbor, Maine 04609 </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gary%2BA.%2BChurchill%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Churchill%20GA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGary%2BA.%2BChurchill%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Aldons J. Lusis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Departments of Human Genetics, Medicine, Microbiology, Immunology and Molecular Genetics, UCLA, Los Angeles, California 90095-1679</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aldons%2BJ.%2BLusis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lusis%20AJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAldons%2BJ.%2BLusis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jlusis@mednet.ucla.edu" class="" data-icon-position="" data-hide-link-title="0">jlusis@mednet.ucla.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/5/10/2021" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460038" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460038/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/10/2021.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460038" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460038/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/10/2021.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460038" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460038/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/10/2021.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460038" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460038/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/5/10/2021.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460038" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460038/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Human genome-wide association studies have identified thousands of loci associated with disease phenotypes. Genome-wide association studies also have become feasible using rodent models and these have some important advantages over human studies, including controlled environment, access to tissues for molecular profiling, reproducible genotypes, and a wide array of techniques for experimental validation. Association mapping with common mouse inbred strains generally requires 100 or more strains to achieve sufficient power and mapping resolution; in contrast, sample sizes for human studies typically are one or more orders of magnitude greater than this. To enable well-powered studies in mice, we have generated high-density genotypes for ∼175 inbred strains of mice using the Mouse Diversity Array. These new data increase marker density by 1.9-fold, have reduced missing data rates, and provide more accurate identification of heterozygous regions compared with previous genotype data. We report the discovery of new loci from previously reported association mapping studies using the new genotype data. The data are freely available for download, and Web-based tools provide easy access for association mapping and viewing of the underlying intensity data for individual loci.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/mouse" class="hw-term hw-article-keyword hw-article-keyword-mouse" rel="nofollow">mouse</a></li><li class="kwd"><a href="/keyword/genotyping" class="hw-term hw-article-keyword hw-article-keyword-genotyping" rel="nofollow">genotyping</a></li><li class="kwd"><a href="/keyword/mouse-diversity-array" class="hw-term hw-article-keyword hw-article-keyword-mouse-diversity-array" rel="nofollow">mouse diversity array</a></li><li class="kwd"><a href="/keyword/hmdp" class="hw-term hw-article-keyword hw-article-keyword-hmdp" rel="nofollow">HMDP</a></li></ul><p id="p-3">The advent of high-density DNA genotyping has revolutionized the ability of geneticists to identify genes associated with polymorphisms that contribute to common diseases and complex traits. Using genotyping technologies, researchers can now assay hundreds of thousands of single nucleotide polymorphisms (SNPs) in human cohorts in tens of thousands of subjects. To date, more than 6900 loci have been associated with phenotypes ranging from cancer to neurological, cardiovascular, and metabolic disorders in human populations (<a href="http://genome.gov/gwas">genome.gov/gwas</a>). Genome-wide association studies (GWAS) often have the precision to identify single candidate genes but in many cases the biological mechanisms that underlie these associations remain uncertain. The process of moving from a locus to a gene to a mechanism is challenging and often requires follow-up studies in model organisms, especially rodents.</p><p id="p-4">Genetic mapping also can be carried out directly in rodent models, and when similar phenotypes are ascertained, it is highly likely that the biological processes that lead to a disease will be shared between humans and mice. Associations of disease phenotypes to polymorphisms in the mouse provide a direct means to identify disease models to support mechanistic studies. Classical approaches to genetic analysis in rodents use low-resolution mapping crosses and correspondingly low-density genotyping is sufficient to achieve the (limited) maximal resolution available from these studies. Recognition that the potential for high-resolution mapping in rodents was not being realized lead to the development of new strategies and resources (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Threadgill <em>et al.</em> 2002</a>). New resources that have been developed include collections of existing inbred strains of mice (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Bennett <em>et al.</em> 2010</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Ghazalpour <em>et al.</em> 2012</a>) as well as the construction of new panels of genetically diverse strains (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Collaborative Cross Consortium 2012</a>). In addition, there has been increased interest in the use of outbred rodent populations for genetic mapping studies (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Yalcin <em>et al.</em> 2010</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Svenson <em>et al.</em> 2012</a>). Here, we focus on the Hybrid Mouse Diversity Panel (HMDP), which consists of a collection of approximately 175 strains, of which approximately 30 are “classic” inbred strains and 145 are recombinant inbred strains derived from pairs of inbred strains. Generally, about 100 strains are required for sufficient power to map typical complex traits (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Bennett <em>et al.</em> 2010</a>). The HMDP has been used to examine the genetics of a wide array of phenotypes, including plasma lipids (<a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">Bennett <em>et al.</em> 2010</a>), bone density (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Farber <em>et al.</em> 2011</a>), blood cell traits (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Davis <em>et al.</em> 2013</a>), conditioned fear responses (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Park <em>et al.</em> 2011</a>), gene-by-diet interactions in obesity (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Parks <em>et al.</em> 2013</a>), inflammatory responses (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Orozco <em>et al.</em> 2012</a>), hearing loss (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Ohmen <em>et al.</em> 2014</a>), diabetes (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Parks <em>et al.</em> 2015</a>), and heart failure (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Rau <em>et al.</em> 2015</a>). In many of these studies, genes at the identified loci were validated as causal using engineered mouse models and a number of the loci or genes corresponded to human GWAS results.</p><p id="p-5">Until recently, mapping studies using the HMDP have relied on a set of ∼140,000 SNP loci that were ascertained from multiple sources and merged, including data from the Broad Institute (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Kirby <em>et al.</em> 2010</a>) and the Welcome Trust Center for Human Genetics (<a href="http://mus.well.ox.ac.uk/mouse/INBREDS/">http://mus.well.ox.ac.uk/mouse/INBREDS/</a>). Gaps were filled using imputation to create a uniform set of SNPs for each strain (<a href="http://mouse.cs.ucla.edu/mousehapmap/">http://mouse.cs.ucla.edu/mousehapmap/</a>). Here we describe genotyping of ∼650,000 SNP loci for the 175 strains in the HMDP using the Mouse Diversity Array (MDA) (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Yang <em>et al.</em> 2010</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Didion and de Villena 2013</a>). These results complement a previous effort to examine 198 inbred mouse lines using the MDA (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Yang <em>et al.</em> 2011b</a>); however, all data has been independently generated, and roughly 80% (138) of the strains are novel compared with <a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Yang <em>et al.</em> (2011b)</a>. The data have been curated to remove poorly performing SNP probes and to correct a handful of errors in strain identification and the sex of genotyped animals. All probes have been remapped by alignment to the most recent release of the reference mouse genome (GRCm38). Updated probe annotations, genotype calls and raw probe intensity are available for download from the Jackson Laboratory (<a href="http://churchill.jax.org/mda">http://churchill.jax.org/mda</a>). In addition, the MDA genotypes now support the online mapping tool (<a href="http://mouse.cs.ucla.edu/emmaserver/">http://mouse.cs.ucla.edu/emmaserver/</a>). We describe the new genotype data and demonstrate that it improves the performance of GWAS using the HMDP.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Genotyping</h3><p id="p-6">Prior Genotypes: genotypes were obtained as previously described (<a id="xref-ref-2-4" class="xref-bibr" href="#ref-2">Bennett <em>et al.</em> 2010</a>) through the combination of genotypes from the Broad Institute (<a href="http://www.broadinstitute.org/mouse/hapmap">http://www.broadinstitute.org/mouse/hapmap</a>) and genotypes from the Wellcome Trust Center for Human Genetics. Genotypes of RI strains were imputed from Wellcome Trust genotypes by interpolating alleles at polymorphic SNPs among parental strains.</p><p id="p-7">MDA DNA isolation and hybridization was performed at the Jackson Laboratories as previously described (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Yang <em>et al.</em> 2010</a>). Genotype calls were obtained using the MouseDivGeno R package (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Didion <em>et al.</em> 2012</a>)</p><p id="p-8">All analysis of genotypes was performed using the R programming language.</p></div><div id="sec-3" class="subsection"><h3>GWAS</h3><p id="p-9">Association mapping was performed as described in <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Rau <em>et al.</em> 2015</a>. We performed the association testing of each SNP using the Efficient Mixed Model Algorithm (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Kang <em>et al.</em> 2008</a>). This algorithm corrects for population structure among the HMDP using the following model:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2021/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2021/embed/mml-math-1.gif"/></noscript></span></span>where m is the mean, b is the allele effect of the SNP, x is the (n × 1) vector of observed genotypes of the SNPs (using additive coding of 0,0.5,1), u is the random effects due to genetic relatedness with <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2021/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2021/embed/mml-math-2.gif"/></noscript></span></span> and e is the random noise with <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2021/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2021/embed/mml-math-3.gif"/></noscript></span></span>. K denotes the identity-by-state kinship matrix estimated from all of the SNPs, I denotes the (n × n) identity matrix and 1n is the (n × 1) vector of ones. Both u and e follow normal distributions. <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2021/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2021/embed/mml-math-4.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2021/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2021/embed/mml-math-5.gif"/></noscript></span></span> are estimated using restricted maximum likelihood and computed p values using the standard F test to test the null hypothesis b = 0. Thresholds reported in <a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">Rau <em>et al.</em> 2015</a> (P &lt; 4.1E-6 suggestive, P &lt; 4.1E-7 significant) were used in this study as well.</p></div><div id="sec-4" class="subsection"><h3>Data availability</h3><p id="p-10">Both raw data (CEL files) and genotypes (SQLite database) are available for download and visualization at <a href="http://churchill.jax.org/mda">http://churchill.jax.org/mda</a>. We offer an online MDA browser to explore raw intensity data for SNPs in a region of interest (<a href="http://www.g3journal.org/content/suppl/2015/07/28/g3.115.020784.DC1/020784SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/content/suppl/2015/07/28/g3.115.020784.DC1/FigureS1.pdf">Figure S1</a>). This is useful as a diagnostic tool and to help identify other strains (not in the HMDP) that are likely to share causal variants. GWAS results for data reanalyzed from <a id="xref-ref-24-4" class="xref-bibr" href="#ref-24">Rau <em>et al.</em> 2015</a> can be found at <a href="http://systems.genetics.ucla.edu/">http://systems.genetics.ucla.edu/</a> as well as tools for visualization and analysis of these data.</p></div></div><div class="section results" id="sec-5"><h2 class="">Results</h2><div id="sec-6" class="subsection"><h3>The Mouse Diversity Array</h3><p id="p-11">The MDA consists of 623,124 SNP probe sets that uniformly cover the nonrepetitive portions of the mouse genome and 916,269 invariant genomic probes that target regions with segmental duplications (<a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">Yang <em>et al.</em> 2010</a>). SNPs were selected to represent the genetic diversity of the classical inbred strains, which derive primarily from <em>Mus musculus domesticus</em> ancestry, as well as sampling the genetic diversity of other mouse species and subspecies including <em>M.m. musculus</em>, <em>M.m. castaneus and M. spretus</em>. This selection strategy maximizes the discrimination of strains and as such it does not necessarily reflect phylogenic divergence, especially for wild-derived inbred strains.</p><p id="p-12">In total, DNA samples from more than 1900 inbred strains, hybrids, or wild-caught mice have been hybridized at The Jackson Laboratory using the protocol previously described (<a id="xref-ref-31-4" class="xref-bibr" href="#ref-31">Yang <em>et al.</em> 2010</a>). In this paper we focus attention on SNP genotype calls obtained from the 175 strains that have been used to comprise the HMDP.</p></div><div id="sec-7" class="subsection"><h3>MDA genotypes improve coverage and identify residual heterozygosity</h3><p id="p-13">The previous set of mouse genotypes (Prior) contains ∼140,000 SNPs (<a id="xref-ref-2-5" class="xref-bibr" href="#ref-2">Bennett <em>et al.</em> 2010</a>), with an average spacing of 20 kb between SNPs. By comparison, the 623,000 SNPs on the MDA have an average spacing of 4.3 kb (<a id="xref-ref-31-5" class="xref-bibr" href="#ref-31">Yang <em>et al.</em> 2010</a>). We identified ∼550,000 high-quality MDA SNPs and tabulated these by functional classes defined by their location relative to known genomic features (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). As further indications of the quality of the MDA genotypes, we examined the frequency of missing data both within strains and within SNP loci (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1, A and B</a>). Overall, the rate of missing genotype calls was ∼0.1% on the MDA compared with 2.4% on the Prior SNPs (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1A</a>). Only six strains have <em>more</em> than 1% missing values in the MDA genotypes, whereas only three strains have <em>less</em> than 1% missing values in the Prior genotypes. In the MDA genotypes, we observed 187 SNPs (0.03%) with a missing call rate greater than 10%; in contrast, the Prior genotypes include ∼9800 SNPs (7%) with more than 10% missing values across the 175 HMDP strains (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1B</a>). The increased density of genotyped loci and reduced levels of missing data are important improvements for the identification of GWAS loci as we illustrate below.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466876/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466876/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">All SNPs present in the Mouse Diversity Array</span><div class="sb-div caption-clear"></div></div></div><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/10/2021/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparisons of Prior genotypes with Mouse Diversity Array (MDA) genotypes. (A) Fraction of single-nucleotide polymorphisms (SNPs) with missing calls in each strain for Prior (left) and MDA (right) genotypes. The red line indicates the average value. (B) Histogram showing the proportion of missing strains for each SNP for the prior (left) and MDA (right) genotypes. Highlighted in yellow and displayed as a percentage are the numbers of SNPs with more than 10% missing values (7% for prior, 0.03% for MDA). (C) Fraction of heterozygous SNPs within each strain for prior (left) and MDA (right) genotypes. The red line indicates the average value. (D) Histogram of concordance between SNPs found in both genotyping sets." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1718299918" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Comparisons of Prior genotypes with Mouse Diversity Array (MDA) genotypes. (A) Fraction of single-nucleotide polymorphisms (SNPs) with missing calls in each strain for Prior (left) and MDA (right) genotypes. The red line indicates the average value. (B) Histogram showing the proportion of missing strains for each SNP for the prior (left) and MDA (right) genotypes. Highlighted in yellow and displayed as a percentage are the numbers of SNPs with more than 10% missing values (7% for prior, 0.03% for MDA). (C) Fraction of heterozygous SNPs within each strain for prior (left) and MDA (right) genotypes. The red line indicates the average value. (D) Histogram of concordance between SNPs found in both genotyping sets.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/10/2021/F1.medium.gif" width="440" height="181"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/5/10/2021/F1.medium.gif" width="440" height="181"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/10/2021/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/10/2021/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466874" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-15" class="first-child">Comparisons of Prior genotypes with Mouse Diversity Array (MDA) genotypes. (A) Fraction of single-nucleotide polymorphisms (SNPs) with missing calls in each strain for Prior (left) and MDA (right) genotypes. The red line indicates the average value. (B) Histogram showing the proportion of missing strains for each SNP for the prior (left) and MDA (right) genotypes. Highlighted in yellow and displayed as a percentage are the numbers of SNPs with more than 10% missing values (7% for prior, 0.03% for MDA). (C) Fraction of heterozygous SNPs within each strain for prior (left) and MDA (right) genotypes. The red line indicates the average value. (D) Histogram of concordance between SNPs found in both genotyping sets.</p><div class="sb-div caption-clear"></div></div></div><p id="p-16">To be useful for mapping, a SNP must be polymorphic in the study population. Furthermore, the minor allele frequency (MAF) should not be too low if used for GWAS to avoid the potential for spurious findings (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). In our studies we restrict attention to the 42% of high-quality SNPs that have a MAF greater than 5% in the HMDP strains (see <a href="http://www.g3journal.org/content/suppl/2015/07/28/g3.115.020784.DC1/TableS1.pdf">Table S1</a> for a list of strains). In contrast, 82% of SNPs in the Prior genotyping panel have MAF greater than 5%. This difference reflects the selection of SNPs on the MDA, which was designed in part to work with the Collaborative Cross and thus incorporates probes that specifically discriminate among wild-derived strains and are comparatively rare among the common inbred mouse strains (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Yang <em>et al.</em> 2011a</a>). After eliminating these SNPs as well as SNPs with over 10% missing values, our new genotypes contain 202,473 SNPs (1 SNP per 13.4 kb) that are suitable to genomic analyses. This represents a 1.ninefold increase over the Prior genotypes, which contain 108,565 SNPs (1 SNP per 25 kb) after filtering to remove MAF &lt; 5% and missing values over 10% (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/10/2021/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Allele frequencies in genotyping datasets. Histograms of the allele frequency of single-nucleotide polymorphisms (SNPs) in the Prior (left) and Mouse Diversity Array (right) genotypes. Highlighted in yellow and displayed as a percentage are the SNPs whose allele frequencies are too low for genome-wide association studies." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1718299918" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Allele frequencies in genotyping datasets. Histograms of the allele frequency of single-nucleotide polymorphisms (SNPs) in the Prior (left) and Mouse Diversity Array (right) genotypes. Highlighted in yellow and displayed as a percentage are the SNPs whose allele frequencies are too low for genome-wide association studies.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/10/2021/F2.medium.gif" width="440" height="164"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/5/10/2021/F2.medium.gif" width="440" height="164"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/10/2021/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/10/2021/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466798" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-17" class="first-child">Allele frequencies in genotyping datasets. Histograms of the allele frequency of single-nucleotide polymorphisms (SNPs) in the Prior (left) and Mouse Diversity Array (right) genotypes. Highlighted in yellow and displayed as a percentage are the SNPs whose allele frequencies are too low for genome-wide association studies.</p><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466898/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466898/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Informative SNPs for performing GWAS in the Hybrid Mouse Diversity panel</span><div class="sb-div caption-clear"></div></div></div><p id="p-19">The HMDP is composed of inbred strains, many of which have been maintained by brother-sister mating for hundreds of generations and are expected to be homozygous throughout their entire genome. However, some of the strains are more recently derived and these may retain regions of residual heterozygosity. The MDA genotypes allow us to gain a better understanding of the heterozygosity remaining in the inbred mouse strains (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1C</a>). The MDA genotype calls are heterozygous at 1.2% of SNPs on average across the HMDP strains. Although the majority of these “H” calls are known to be errors (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Didion and de Villena 2013</a>), genomic regions with multiple “H” genotypes in a strain are likely to reflect residual heterozygosity. We note that only a few strains have heterozygous call rates greater than 2%, and all of these are from the more recently derived BXD43-103 panel (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Peirce <em>et al.</em> 2004</a>).</p><p id="p-20">A common set of ∼71,000 SNPs are represented in both the MDA and Prior genotyping data. Discordant genotypes (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1D</a>) were observed to exceed 10% at only 335 (0.5%) of the common SNP loci. We observed 10 SNP loci with discordance rates greater than 50% between the two data sets. For association analyses, we assumed that the MDA genotypes are correct.</p></div><div id="sec-8" class="subsection"><h3>Improved GWA results</h3><p id="p-21">To illustrate the improved performance of GWAS using the MDA genotypes, we present a new analysis of previously reported data on the role of catecholamine stimulation on heart weight (<a id="xref-ref-24-5" class="xref-bibr" href="#ref-24">Rau <em>et al.</em> 2015</a>). We performed GWA analyses of heart weight after catecholamine stimulation using the efficient mixed model algorithm (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Kang <em>et al.</em> 2008</a>) with both the Prior genotypes (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>) and the new MDA genotypes (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a>). Using the Prior genotypes, we identified a single significant locus, while using the MDA genotypes, we identified four additional loci at genome-wide significance (<em>P</em> &lt; 4.1E-6), as determined previously for the HMDP (<a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">Kang <em>et al.</em> 2008</a>; <a id="xref-ref-2-6" class="xref-bibr" href="#ref-2">Bennett <em>et al.</em> 2010</a>). Each of the new loci had achieved a suggestive (<em>P</em> &lt; 0.05) level of significance using the Prior genotypes, which provides an indication of the consistency of these findings; however, as only 71,000 SNPs are shared between the two datasets, the specific SNPs making up the peaks in both genotype sets were not entirely identical. Like other mixed-model algorithms (<em>e.g.</em>, <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Lippert <em>et al.</em> 2011</a>), EMMA uses a kinship matrix to correct for substructure in the study population. We examined whether changes to the kinship matrix might lead to this result by using the Prior kinship matrix while performing EMMA on the MDA genotypes (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3C</a>, <a href="http://www.g3journal.org/content/suppl/2015/07/28/g3.115.020784.DC1/FigureS2.pdf">Figure S2</a>, <a href="http://www.g3journal.org/content/suppl/2015/07/28/g3.115.020784.DC1/FigureS3.pdf">Figure S3</a>, <a href="http://www.g3journal.org/content/suppl/2015/07/28/g3.115.020784.DC1/FigureS4.pdf">Figure S4</a>, <a href="http://www.g3journal.org/content/suppl/2015/07/28/g3.115.020784.DC1/FigureS5.pdf">Figure S5</a>, <a href="http://www.g3journal.org/content/suppl/2015/07/28/g3.115.020784.DC1/FigureS6.pdf">Figure S6</a>, <a href="http://www.g3journal.org/content/suppl/2015/07/28/g3.115.020784.DC1/FigureS7.pdf">Figure S7</a>, and <a href="http://www.g3journal.org/content/suppl/2015/07/28/g3.115.020784.DC1/FigureS8.pdf">Figure S8</a>). Although there were some differences in association strengths (<a href="http://www.g3journal.org/content/suppl/2015/07/28/g3.115.020784.DC1/FigureS9.pdf">Figure S9</a>), the peak SNPs were not affected. When the peak SNP was shared, the SNP had nearly identical genotypes in both sets, which suggests that even small changes to the genotypes can have large effects on the results.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/10/2021/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Effects of new single-nucleotide polymorphisms (SNPs) on genome-wide association study results. In both cases, the phenotype being used is total heart weight after isoproterenol treatment. Red line indicates genome-wide significance threshold (4.1E-6). (A) Results using EMMA on the Prior genotypes reveals a single locus on chromosome 1. (B) Results using EMMA on Mouse Diversity Array (MDA) genotypes reveals four additional loci. (C) Results using EMMA on the MDA genotypes using a kinship matrix generated from the Prior genotypes does not demonstrably change the results from B)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1718299918" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Effects of new single-nucleotide polymorphisms (SNPs) on genome-wide association study results. In both cases, the phenotype being used is total heart weight after isoproterenol treatment. Red line indicates genome-wide significance threshold (4.1E-6). (A) Results using EMMA on the Prior genotypes reveals a single locus on chromosome 1. (B) Results using EMMA on Mouse Diversity Array (MDA) genotypes reveals four additional loci. (C) Results using EMMA on the MDA genotypes using a kinship matrix generated from the Prior genotypes does not demonstrably change the results from B).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/10/2021/F3.medium.gif" width="440" height="192"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/5/10/2021/F3.medium.gif" width="440" height="192"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/10/2021/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/10/2021/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466854" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-22" class="first-child">Effects of new single-nucleotide polymorphisms (SNPs) on genome-wide association study results. In both cases, the phenotype being used is total heart weight after isoproterenol treatment. Red line indicates genome-wide significance threshold (4.1E-6). (A) Results using EMMA on the Prior genotypes reveals a single locus on chromosome 1. (B) Results using EMMA on Mouse Diversity Array (MDA) genotypes reveals four additional loci. (C) Results using EMMA on the MDA genotypes using a kinship matrix generated from the Prior genotypes does not demonstrably change the results from B).</p><div class="sb-div caption-clear"></div></div></div><p id="p-23">The single significant SNP obtained using the Prior genotypes is located on chromosome 1 ∼25 kb upstream of <em>Tgfb2</em>, which has been implicated previously in cardiac morphogenesis and hypertrophy (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Lim and Zhu 2006</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Azhar <em>et al.</em> 2011</a>). The four new loci obtained using MDA genotypes (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>) include: <em>Acvr1</em>, which has previously been implicated in the regulation of left ventricular heart mass in newborns and congenital defects (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Smith <em>et al.</em> 2009</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Gorący <em>et al.</em> 2012</a>); <em>Drd2</em>, a gene previously linked to changes in heart rate (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Huertas <em>et al.</em> 2012</a>) and elevated blood pressure (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Rosmond <em>et al.</em> 2001</a>); <em>Pln</em>, a well-studied gene involved in heart failure (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Brittsan <em>et al.</em> 1999</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Chu and Kranias 2006</a>) and associated by GWAS in human populations with variation in left ventricular internal dimension (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Vasan <em>et al.</em> 2009</a>); and <em>Grik2</em>, a gene that has been associated with heart failure in a human GWAS study (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Parsa <em>et al.</em> 2011</a>). Although the causal role of these genes remains to be established, their known biology supports a role in determining heart weight after catecholamine stimulation.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466872/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466872/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Improved GWAS results due to MDA</span><div class="sb-div caption-clear"></div></div></div><p id="p-25">Examination of additional phenotypes reported in <a id="xref-ref-24-6" class="xref-bibr" href="#ref-24">Rau <em>et al.</em> (2015)</a> (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>) shows that the use of the denser MDA arrays led to more significant or suggestive results in each phenotype except for lung weight. Two significant loci reported in <a id="xref-ref-24-7" class="xref-bibr" href="#ref-24">Rau <em>et al.</em> (2015)</a> were lost in the MDA GWAS: one for liver weight on chromosome 7 over the <em>Calm3</em> gene, which is lost entirely, and another for lung weight on chromosome 6 over the <em>Aqp1</em> gene, which becomes a suggestive locus. In both cases, the relevant SNPs (rs31334298 for liver, rs30022082 for lung) are present in both genotype sets. In both cases the only difference in genotypes at these SNPs occurred in the C57BL/6J strain, which might explain the large change in association based on the importance of this strain in the HMDP. Ultimately, the MDA genotypes resulted in 24 new suggestive loci and 11 new significant loci when compared to the original results reported in <a id="xref-ref-24-8" class="xref-bibr" href="#ref-24">Rau <em>et al.</em> (2015)</a>.</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466862/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466862/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">MDA leads to many new significant loci compared with results from <a id="xref-ref-24-9" class="xref-bibr" href="#ref-24">Rau <em>et al.</em> 2015</a></span><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-9"><h2 class="">Discussion</h2><p id="p-27">Systems-level analyses of complex phenotypes rely on accurate information regarding the underlying genetic variants. Genotypes should be dense enough to ensure that markers are in linkage disequilibrium with most of the potential causal mutations. Equally important in populations with significant levels of population structure, such as admixed human populations or inbred lines of mice, genotypes should be selected to reflect the intrinsic genetic relatedness of the study population. In this study, we examined the effects of obtaining a denser and more accurate set of genotypes in a population (the HMDP), which had previously been analyzed using GWAS (<a id="xref-ref-24-11" class="xref-bibr" href="#ref-24">Rau <em>et al.</em> 2015</a>). Our new genotypes, obtained using the MDA, increased the number of informative SNPs typed by 87% and improved the genotype quality since large portions of the previous genotypes were imputed.</p><p id="p-28">A previous study (<a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">Yang <em>et al.</em> 2011b</a>) reported the genotyping of a set of 198 strains using the MDA. Our study complements this study, with 37 strains overlapping with <a id="xref-ref-33-4" class="xref-bibr" href="#ref-33">Yang <em>et al.</em> 2011b</a> and 138 previously unreported strains. For the 37 strains in common between <a id="xref-ref-33-5" class="xref-bibr" href="#ref-33">Yang <em>et al.</em> 2011b</a> and our data, we compared the genotypes and observed that, on average, 99.5% of informative SNPs in our data had the same call in <a id="xref-ref-33-6" class="xref-bibr" href="#ref-33">Yang <em>et al.</em> 2011b</a>, 0.49% were homozygous in either Yang or the present study but heterozygous in the other and 0.01% had a SNP in one data set but not the other (<a href="http://www.g3journal.org/content/suppl/2015/07/28/g3.115.020784.DC1/TableS2.csv">Table S2</a>). These differences are likely the result of either technical error or genetic drift in the inbred lines.</p><p id="p-29">We observed significant improvements over the previously reported GWAS data, returning over double (33 <em>vs.</em> 14) the number of suggestive loci in the GWAS study examined here. We explored whether changes to the kinship matrix played a role in this improvement but saw very few changes by switching out one kinship matrix for another. Rather, the new loci appear to be the result of a combination of the addition of new SNPs which, perhaps, are in better linkage disequilibrium with causal mutations as well as corrections to the SNP genotypes, especially in the four core strains (A/J, C3H/HeJ, DBA/2J, and C57BL/6J), which contribute the majority of the power of the panel. Our results suggest that the re-examination of previously analyzed results with a more accurate and denser genotype set may lead to the discovery of novel loci and genes of interest, both in mice as well as in human studies.</p></div><div class="section" id="sec-10"></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-30">We thank Matthew Vincent for the development of visualization software. Supported by National Institutes of Health grants HL28481 (A.J.L.), D094311 (A.J.L.), HL126753 (A.J.L.), HL30568 (A.J.L.), HL123295 (Y.W. and A.J.L.), and GM076468 (G.A.C.). </p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-supplementary-material" id="fn-5"><p id="p-31">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.020784/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.020784/-/DC1</a></p></li><li class="fn" id="fn-6"><p id="p-32"><em>Communicating editor: D. J. de Koning</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2015-04-21"><span class="received-label">Received </span>April 21, 2015.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2015-07-27"><span class="accepted-label">Accepted </span>July 27, 2015.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2015 Rau <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2021.1" data-doi="10.1002/dvdy.22702"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Azhar</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gard</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rajan</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Transforming growth factor Beta2 is required for valve remodeling during heart development.</span> <abbr class="cit-jnl-abbrev">Dev. Dyn.</abbr> <span class="cit-vol">240</span>: <span class="cit-fpage">2127</span>–<span class="cit-lpage">2141</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bdynamics%2B%253A%2B%2Ban%2Bofficial%2Bpublication%2Bof%2Bthe%2BAmerican%2BAssociation%2Bof%2BAnatomists%26rft.stitle%253DDev%2BDyn%26rft.aulast%253DAzhar%26rft.auinit1%253DM.%26rft.volume%253D240%26rft.issue%253D9%26rft.spage%253D2127%26rft.epage%253D2141%26rft.atitle%253DTransforming%2Bgrowth%2Bfactor%2BBeta2%2Bis%2Brequired%2Bfor%2Bvalve%2Bremodeling%2Bduring%2Bheart%2Bdevelopment.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fdvdy.22702%26rft_id%253Dinfo%253Apmid%252F21780244%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/dvdy.22702&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21780244&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2021.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2021.2" data-doi="10.1101/gr.099234.109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bennett</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Farber</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Orozco</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ghazalpour</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A high-resolution association mapping panel for the dissection of complex traits in mice.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">281</span>–<span class="cit-lpage">290</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.099234.109%26rft_id%253Dinfo%253Apmid%252F20054062%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIwLzIvMjgxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMC8yMDIxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-other" id="cit-5.10.2021.3" data-doi="10.1006/jmcc.1999.1010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brittsan</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kiss</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Edes</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grupp</span>  <span class="cit-name-given-names">I. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grupp</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">The effect of isoproterenol on phospholamban-deficient mouse hearts with altered thyroid conditions.</span> J<span class="cit-source">. Mol. Cell. Cardiol.</span> <span class="cit-vol">31</span>: <span class="cit-fpage">1725</span>–<span class="cit-lpage">1737</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Band%2Bcellular%2Bcardiology%26rft.stitle%253DJ%2BMol%2BCell%2BCardiol%26rft.aulast%253DBrittsan%26rft.auinit1%253DA.%2BG.%26rft.volume%253D31%26rft.issue%253D9%26rft.spage%253D1725%26rft.epage%253D1737%26rft.atitle%253DThe%2Beffect%2Bof%2Bisoproterenol%2Bon%2Bphospholamban-deficient%2Bmouse%2Bhearts%2Bwith%2Baltered%2Bthyroid%2Bconditions.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmcc.1999.1010%26rft_id%253Dinfo%253Apmid%252F10471356%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmcc.1999.1010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10471356&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2021.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000082336200014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-other" id="cit-5.10.2021.4"><div class="cit-metadata unstructured">Chu, G., and E. G. Kranias, 2006 Phospholamban as a therapeutic modality in heart failure. Novartis Found. Symp. 274: 156–171; discussion 172–5, 272–6.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2021.5" data-doi="10.1534/genetics.111.132639"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">Collaborative Cross Consortium</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The genome architecture of the Collaborative Cross mouse genetic reference population.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">389</span>–<span class="cit-lpage">401</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DCollaborative%2BCross%2BConsortium%26rft.auinit1%253D%2B%26rft.volume%253D190%26rft.issue%253D2%26rft.spage%253D389%26rft.epage%253D401%26rft.atitle%253DThe%2BGenome%2BArchitecture%2Bof%2Bthe%2BCollaborative%2BCross%2BMouse%2BGenetic%2BReference%2BPopulation%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.132639%26rft_id%253Dinfo%253Apmid%252F22345608%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkwLzIvMzg5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMC8yMDIxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2021.6" data-doi="10.1007/s00335-013-9448-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Nas</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bennett</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Orozco</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pan</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genome-wide association mapping of blood cell traits in mice.</span> <abbr class="cit-jnl-abbrev">Mamm. Genome</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">105</span>–<span class="cit-lpage">118</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMamm.%2BGenome%26rft.volume%253D24%26rft.spage%253D105%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00335-013-9448-0%26rft_id%253Dinfo%253Apmid%252F23417284%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00335-013-9448-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23417284&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2021.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2021.7" data-doi="10.1007/s00335-012-9441-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Didion</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Villena</span>  <span class="cit-name-given-names">F. P.-M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Deconstructing <em>Mus gemischus</em>: advances in understanding ancestry, structure, and variation in the genome of the laboratory mouse.</span> <abbr class="cit-jnl-abbrev">Mamm. Genome</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">20</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMamm.%2BGenome%26rft.volume%253D24%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00335-012-9441-z%26rft_id%253Dinfo%253Apmid%252F23223940%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00335-012-9441-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23223940&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2021.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2021.8" data-doi="10.1186/1471-2164-13-34"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Didion</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sheppard</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">C.-P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McMillan</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Discovery of novel variants in genotyping arrays improves genotype retention and reduces ascertainment bias.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">34</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DDidion%26rft.auinit1%253DJ.%2BP.%26rft.volume%253D13%26rft.spage%253D34%26rft.epage%253D34%26rft.atitle%253DDiscovery%2Bof%2Bnovel%2Bvariants%2Bin%2Bgenotyping%2Barrays%2Bimproves%2Bgenotype%2Bretention%2Band%2Breduces%2Bascertainment%2Bbias.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-13-34%26rft_id%253Dinfo%253Apmid%252F22260749%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-13-34&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22260749&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2021.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2021.9" data-doi="10.1371/journal.pgen.1002038"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Farber</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bennett</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Orozco</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zou</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lira</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Mouse genome-wide association and systems genetics identify Asxl2 as a regulator of bone mineral density and osteoclastogenesis.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e1002038</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DFarber%26rft.auinit1%253DC.%2BR.%26rft.volume%253D7%26rft.issue%253D4%26rft.spage%253De1002038%26rft.epage%253De1002038%26rft.atitle%253DMouse%2Bgenome-wide%2Bassociation%2Band%2Bsystems%2Bgenetics%2Bidentify%2BAsxl2%2Bas%2Ba%2Bregulator%2Bof%2Bbone%2Bmineral%2Bdensity%2Band%2Bosteoclastogenesis.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002038%26rft_id%253Dinfo%253Apmid%252F21490954%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002038&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21490954&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2021.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2021.10" data-doi="10.1007/s00335-012-9411-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ghazalpour</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rau</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Farber</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bennett</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Orozco</span>  <span class="cit-name-given-names">L. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Hybrid mouse diversity panel: a panel of inbred mouse strains suitable for analysis of complex genetic traits.</span> <abbr class="cit-jnl-abbrev">Mamm. Genome</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">680</span>–<span class="cit-lpage">692</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMammalian%2Bgenome%2B%253A%2B%2Bofficial%2Bjournal%2Bof%2Bthe%2BInternational%2BMammalian%2BGenome%2BSociety%26rft.stitle%253DMamm%2BGenome%26rft.aulast%253DGhazalpour%26rft.auinit1%253DA.%26rft.volume%253D23%26rft.issue%253D9-10%26rft.spage%253D680%26rft.epage%253D692%26rft.atitle%253DHybrid%2Bmouse%2Bdiversity%2Bpanel%253A%2Ba%2Bpanel%2Bof%2Binbred%2Bmouse%2Bstrains%2Bsuitable%2Bfor%2Banalysis%2Bof%2Bcomplex%2Bgenetic%2Btraits.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00335-012-9411-5%26rft_id%253Dinfo%253Apmid%252F22892838%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00335-012-9411-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22892838&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2021.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000309543200019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2021.11" data-doi="10.1089/gtmb.2012.0164"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gorący</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Safranow</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dawid</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Skonieczna-Żydecka</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaczmarczyk</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Common genetic variants of the BMP4, BMPR1A, BMPR1B, and ACVR1 genes, left ventricular mass, and other parameters of the heart in newborns.</span> <abbr class="cit-jnl-abbrev">Genet. Test. Mol. Biomarkers</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">1309</span>–<span class="cit-lpage">1316</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DGenet%2BTest%2BMol%2BBiomarkers%26rft.aulast%253DGoracy%26rft.auinit1%253DI.%26rft.volume%253D16%26rft.issue%253D11%26rft.spage%253D1309%26rft.epage%253D1316%26rft.atitle%253DCommon%2BGenetic%2BVariants%2Bof%2Bthe%2BBMP4%252C%2BBMPR1A%252C%2BBMPR1B%252C%2Band%2BACVR1%2BGenes%252C%2BLeft%2BVentricular%2BMass%252C%2Band%2BOther%2BParameters%2Bof%2Bthe%2BHeart%2Bin%2BNewborns.%26rft_id%253Dinfo%253Adoi%252F10.1089%252Fgtmb.2012.0164%26rft_id%253Dinfo%253Apmid%252F22971142%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1089/gtmb.2012.0164&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22971142&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2021.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2021.12" data-doi="10.1111/j.1601-183X.2012.00793.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huertas</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bühler</span>  <span class="cit-name-given-names">K.-M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Echeverry-Alzate</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Giménez</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">López-Moreno</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">C957T polymorphism of the dopamine D2 receptor gene is associated with motor learning and heart rate.</span> <abbr class="cit-jnl-abbrev">Genes Brain Behav.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">677</span>–<span class="cit-lpage">683</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%252C%2Bbrain%252C%2Band%2Bbehavior%26rft.stitle%253DGenes%2BBrain%2BBehav%26rft.aulast%253DHuertas%26rft.auinit1%253DE.%26rft.volume%253D11%26rft.issue%253D6%26rft.spage%253D677%26rft.epage%253D683%26rft.atitle%253DC957T%2Bpolymorphism%2Bof%2Bthe%2Bdopamine%2BD2%2Breceptor%2Bgene%2Bis%2Bassociated%2Bwith%2Bmotor%2Blearning%2Band%2Bheart%2Brate.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1601-183X.2012.00793.x%26rft_id%253Dinfo%253Apmid%252F22487365%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1601-183X.2012.00793.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22487365&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2021.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2021.13" data-doi="10.1534/genetics.107.080101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zaitlen</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wade</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kirby</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heckerman</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient control of population structure in model organism association mapping.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">178</span>: <span class="cit-fpage">1709</span>–<span class="cit-lpage">1723</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKang%26rft.auinit1%253DH.%2BM.%26rft.volume%253D178%26rft.issue%253D3%26rft.spage%253D1709%26rft.epage%253D1723%26rft.atitle%253DEfficient%2BControl%2Bof%2BPopulation%2BStructure%2Bin%2BModel%2BOrganism%2BAssociation%2BMapping%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.080101%26rft_id%253Dinfo%253Apmid%252F18385116%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3OC8zLzE3MDkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzEwLzIwMjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2021.14" data-doi="10.1534/genetics.110.115014"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kirby</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wade</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cotsapas</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kostem</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Fine mapping in 94 inbred mouse strains using a high-density haplotype resource.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">185</span>: <span class="cit-fpage">1081</span>–<span class="cit-lpage">1095</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.115014%26rft_id%253Dinfo%253Apmid%252F20439770%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4NS8zLzEwODEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzEwLzIwMjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2021.15" data-doi="10.1007/s00018-006-6085-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lim</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">Y. Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Role of transforming growth factor-beta in the progression of heart failure.</span> <abbr class="cit-jnl-abbrev">Cell. Mol. Life Sci.</abbr> <span class="cit-vol">63</span>: <span class="cit-fpage">2584</span>–<span class="cit-lpage">2596</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCellular%2Band%2Bmolecular%2Blife%2Bsciences%2B%253A%2B%2BCMLS%26rft.stitle%253DCell%2BMol%2BLife%2BSci%26rft.aulast%253DBarman%26rft.auinit1%253DT.%2BE.%26rft.volume%253D63%26rft.issue%253D22%26rft.spage%253D2584%26rft.epage%253D2596%26rft.atitle%253DRole%2Bof%2Btransforming%2Bgrowth%2Bfactor-beta%2Bin%2Bthe%2Bprogression%2Bof%2Bheart%2Bfailure.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00018-006-6085-8%26rft_id%253Dinfo%253Apmid%252F17013566%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00018-006-6085-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17013566&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2021.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000242846100004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2021.16" data-doi="10.1038/nmeth.1681"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lippert</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Listgarten</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kadie</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davidson</span>  <span class="cit-name-given-names">R. I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">FaST linear mixed models for genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">833</span>–<span class="cit-lpage">835</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DLippert%26rft.auinit1%253DC.%26rft.volume%253D8%26rft.issue%253D10%26rft.spage%253D833%26rft.epage%253D835%26rft.atitle%253DFaST%2Blinear%2Bmixed%2Bmodels%2Bfor%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1681%26rft_id%253Dinfo%253Apmid%252F21892150%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1681&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21892150&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2021.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000295358000017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2021.17" data-doi="10.1007/s10162-014-0443-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ohmen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">E. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Joo</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hormozdiari</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genome-wide association study for age-related hearing loss (AHL) in the mouse: a meta-analysis.</span> <abbr class="cit-jnl-abbrev">J. Assoc. Res. Otolaryngol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">335</span>–<span class="cit-lpage">352</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAssoc.%2BRes.%2BOtolaryngol.%26rft.volume%253D15%26rft.spage%253D335%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10162-014-0443-2%26rft_id%253Dinfo%253Apmid%252F24570207%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10162-014-0443-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24570207&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2021.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2021.18" data-doi="10.1016/j.cell.2012.08.043"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Orozco</span>  <span class="cit-name-given-names">L. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bennett</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Farber</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ghazalpour</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pan</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Unraveling inflammatory responses using systems genetics and gene-environment interactions in macrophages.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">151</span>: <span class="cit-fpage">658</span>–<span class="cit-lpage">670</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DOrozco%26rft.auinit1%253DL.%2BD.%26rft.volume%253D151%26rft.issue%253D3%26rft.spage%253D658%26rft.epage%253D670%26rft.atitle%253DUnraveling%2BInflammatory%2BResponses%2Busing%2BSystems%2BGenetics%2Band%2BGene-Environment%2BInteractions%2Bin%2BMacrophages.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2012.08.043%26rft_id%253Dinfo%253Apmid%252F23101632%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2012.08.043&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23101632&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2021.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000310529300020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2021.19" data-doi="10.1186/1752-0509-5-43"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gale</span>  <span class="cit-name-given-names">G. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Jong</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ghazalpour</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bennett</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Gene networks associated with conditional fear in mice identified using a systems genetics approach.</span> <abbr class="cit-jnl-abbrev">BMC Syst. Biol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">43</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DBMC%2BSyst%2BBiol%26rft.aulast%253DPark%26rft.auinit1%253DC.%2BC.%26rft.volume%253D5%26rft.spage%253D43%26rft.epage%253D43%26rft.atitle%253DGene%2Bnetworks%2Bassociated%2Bwith%2Bconditional%2Bfear%2Bin%2Bmice%2Bidentified%2Busing%2Ba%2Bsystems%2Bgenetics%2Bapproach.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1752-0509-5-43%26rft_id%253Dinfo%253Apmid%252F21410935%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1752-0509-5-43&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21410935&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2021.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2021.20" data-doi="10.1016/j.cmet.2012.12.007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Parks</span>  <span class="cit-name-given-names">B. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nam</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Org</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kostem</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Norheim</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genetic control of obesity and gut microbiota composition in response to high-fat, high-sucrose diet in mice.</span> <abbr class="cit-jnl-abbrev">Cell Metab.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">141</span>–<span class="cit-lpage">152</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BMetab.%26rft.volume%253D17%26rft.spage%253D141%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cmet.2012.12.007%26rft_id%253Dinfo%253Apmid%252F23312289%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cmet.2012.12.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23312289&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2021.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2021.21" data-doi="10.1016/j.cmet.2015.01.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Parks</span>  <span class="cit-name-given-names">B. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sallam</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mehrabian</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Psychogios</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hui</span>  <span class="cit-name-given-names">S. T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genetic architecture of insulin resistance in the mouse.</span> <abbr class="cit-jnl-abbrev">Cell Metab.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">334</span>–<span class="cit-lpage">346</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BMetab.%26rft.volume%253D21%26rft.spage%253D334%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cmet.2015.01.002%26rft_id%253Dinfo%253Apmid%252F25651185%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cmet.2015.01.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25651185&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2021.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2021.22" data-doi="10.1111/j.1752-8062.2010.00251.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Parsa</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">Y.-P. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kelly</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Corretti</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ryan</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Hypertrophy-associated polymorphisms ascertained in a founder cohort applied to heart failure risk and mortality.</span> <abbr class="cit-jnl-abbrev">Clin. Transl. Sci.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">17</span>–<span class="cit-lpage">23</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DClin%2BTransl%2BSci%26rft.aulast%253DParsa%26rft.auinit1%253DA.%26rft.volume%253D4%26rft.issue%253D1%26rft.spage%253D17%26rft.epage%253D23%26rft.atitle%253DHypertrophy-associated%2Bpolymorphisms%2Bascertained%2Bin%2Ba%2Bfounder%2Bcohort%2Bapplied%2Bto%2Bheart%2Bfailure%2Brisk%2Band%2Bmortality.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1752-8062.2010.00251.x%26rft_id%253Dinfo%253Apmid%252F21348951%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1752-8062.2010.00251.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21348951&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2021.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2021.23" data-doi="10.1186/1471-2156-5-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peirce</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Silver</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">R. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">A new set of BXD recombinant inbred lines from advanced intercross populations in mice.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">7</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenetics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenet%26rft.aulast%253DPeirce%26rft.auinit1%253DJ.%2BL.%26rft.volume%253D5%26rft.spage%253D7%26rft.epage%253D7%26rft.atitle%253DA%2Bnew%2Bset%2Bof%2BBXD%2Brecombinant%2Binbred%2Blines%2Bfrom%2Badvanced%2Bintercross%2Bpopulations%2Bin%2Bmice.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2156-5-7%26rft_id%253Dinfo%253Apmid%252F15117419%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2156-5-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15117419&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2021.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2021.24" data-doi="10.1161/CIRCGENETICS.113.000732"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rau</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Avetisyan</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Romay</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Mapping genetic contributions to cardiac pathology induced by beta-adrenergic stimulation in mice.</span> <abbr class="cit-jnl-abbrev">Circ Cardiovasc Genet</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">40</span>–<span class="cit-lpage">49</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCirc%2BCardiovasc%2BGenet%26rft_id%253Dinfo%253Adoi%252F10.1161%252FCIRCGENETICS.113.000732%26rft_id%253Dinfo%253Apmid%252F25480693%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiY2lyY2N2ZyI7czo1OiJyZXNpZCI7czo2OiI4LzEvNDAiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzEwLzIwMjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2021.25" data-doi="10.1038/sj.jhh.1001231"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rosmond</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rankinen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chagnon</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérusse</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chagnon</span>  <span class="cit-name-given-names">Y. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Polymorphism in exon 6 of the dopamine D(2) receptor gene (DRD2) is associated with elevated blood pressure and personality disorders in men.</span> <abbr class="cit-jnl-abbrev">J. Hum. Hypertens.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">553</span>–<span class="cit-lpage">558</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bhuman%2Bhypertension%26rft.stitle%253DJ%2BHum%2BHypertens%26rft.aulast%253DRosmond%26rft.auinit1%253DR.%26rft.volume%253D15%26rft.issue%253D8%26rft.spage%253D553%26rft.epage%253D558%26rft.atitle%253DPolymorphism%2Bin%2Bexon%2B6%2Bof%2Bthe%2Bdopamine%2BD%25282%2529%2Breceptor%2Bgene%2B%2528DRD2%2529%2Bis%2Bassociated%2Bwith%2Belevated%2Bblood%2Bpressure%2Band%2Bpersonality%2Bdisorders%2Bin%2Bmen.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.jhh.1001231%26rft_id%253Dinfo%253Apmid%252F11494094%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/sj.jhh.1001231&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11494094&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2021.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2021.26" data-doi="10.1161/CIRCULATIONAHA.108.843714"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Joziasse</span>  <span class="cit-name-given-names">I. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chocron</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Dinther</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guryev</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Dominant-negative ALK2 allele associates with congenital heart defects.</span> <abbr class="cit-jnl-abbrev">Circulation</abbr> <span class="cit-vol">119</span>: <span class="cit-fpage">3062</span>–<span class="cit-lpage">3069</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCirculation%26rft_id%253Dinfo%253Adoi%252F10.1161%252FCIRCULATIONAHA.108.843714%26rft_id%253Dinfo%253Apmid%252F19506109%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTQ6ImNpcmN1bGF0aW9uYWhhIjtzOjU6InJlc2lkIjtzOjExOiIxMTkvMjQvMzA2MiI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTAvMjAyMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2021.27" data-doi="10.1534/genetics.111.132597"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Svenson</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gatti</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valdar</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Welsh</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">High-resolution genetic mapping using the Mouse Diversity outbred population.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">437</span>–<span class="cit-lpage">447</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DSvenson%26rft.auinit1%253DK.%2BL.%26rft.volume%253D190%26rft.issue%253D2%26rft.spage%253D437%26rft.epage%253D447%26rft.atitle%253DHigh-Resolution%2BGenetic%2BMapping%2BUsing%2Bthe%2BMouse%2BDiversity%2BOutbred%2BPopulation%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.132597%26rft_id%253Dinfo%253Apmid%252F22345611%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkwLzIvNDM3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMC8yMDIxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2021.28" data-doi="10.1007/s00335-001-4001-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Threadgill</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hunter</span>  <span class="cit-name-given-names">K. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">R. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Genetic dissection of complex and quantitative traits: from fantasy to reality via a community effort.</span> <abbr class="cit-jnl-abbrev">Mamm. Genome</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">175</span>–<span class="cit-lpage">178</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMammalian%2Bgenome%2B%253A%2B%2Bofficial%2Bjournal%2Bof%2Bthe%2BInternational%2BMammalian%2BGenome%2BSociety%26rft.stitle%253DMamm%2BGenome%26rft.aulast%253DThreadgill%26rft.auinit1%253DD.%2BW.%26rft.volume%253D13%26rft.issue%253D4%26rft.spage%253D175%26rft.epage%253D178%26rft.atitle%253DGenetic%2Bdissection%2Bof%2Bcomplex%2Band%2Bquantitative%2Btraits%253A%2Bfrom%2Bfantasy%2Bto%2Breality%2Bvia%2Ba%2Bcommunity%2Beffort.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00335-001-4001-y%26rft_id%253Dinfo%253Apmid%252F11956758%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00335-001-4001-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11956758&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2021.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000175046300001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2021.29" data-doi="10.1001/jama.2009.978-a"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vasan</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glazer</span>  <span class="cit-name-given-names">N. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Felix</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lieb</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wild</span>  <span class="cit-name-given-names">P. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genetic variants associated with cardiac structure and function: a meta-analysis and replication of genome-wide association data.</span> <abbr class="cit-jnl-abbrev">JAMA</abbr> <span class="cit-vol">302</span>: <span class="cit-fpage">168</span>–<span class="cit-lpage">178</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJAMA%26rft.stitle%253DJAMA%26rft.aulast%253DVasan%26rft.auinit1%253DR.%2BS.%26rft.volume%253D302%26rft.issue%253D2%26rft.spage%253D168%26rft.epage%253D178%26rft.atitle%253DGenetic%2Bvariants%2Bassociated%2Bwith%2Bcardiac%2Bstructure%2Band%2Bfunction%253A%2Ba%2Bmeta-analysis%2Band%2Breplication%2Bof%2Bgenome-wide%2Bassociation%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1001%252Fjama.2009.978-a%26rft_id%253Dinfo%253Apmid%252F19584346%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1001/jama.2009.978-a&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19584346&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2021.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000267696500026&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2021.30" data-doi="10.1371/journal.pgen.1001085"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yalcin</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nicod</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bhomra</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davidson</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cleak</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Commercially available outbred mice for genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e1001085</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D6%26rft.spage%253De1001085%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1001085%26rft_id%253Dinfo%253Apmid%252F20838427%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1001085&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20838427&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2021.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2021.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ding</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hutchins</span>  <span class="cit-name-given-names">L. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Szatkiewicz</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bell</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A customized and versatile high-density genotyping array for the mouse.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">663</span>–<span class="cit-lpage">666</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D6%26rft.spage%253D663%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2021.32" data-doi="10.1038/ng.823"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manolio</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pasquale</span>  <span class="cit-name-given-names">L. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boerwinkle</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Caporaso</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011a</span> <span class="cit-article-title">Genome partitioning of genetic variation for complex traits using common SNPs.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">519</span>–<span class="cit-lpage">525</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DYang%26rft.auinit1%253DJ.%26rft.volume%253D43%26rft.issue%253D6%26rft.spage%253D519%26rft.epage%253D525%26rft.atitle%253DGenome%2Bpartitioning%2Bof%2Bgenetic%2Bvariation%2Bfor%2Bcomplex%2Btraits%2Busing%2Bcommon%2BSNPs.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.823%26rft_id%253Dinfo%253Apmid%252F21552263%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.823&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21552263&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2021.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2021.33" data-doi="10.1038/ng.847"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Didion</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buus</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bell</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011b</span> <span class="cit-article-title">Subspecific origin and haplotype diversity in the laboratory mouse.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">648</span>–<span class="cit-lpage">655</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DYang%26rft.auinit1%253DH.%26rft.volume%253D43%26rft.issue%253D7%26rft.spage%253D648%26rft.epage%253D655%26rft.atitle%253DSubspecific%2Borigin%2Band%2Bhaplotype%2Bdiversity%2Bin%2Bthe%2Blaboratory%2Bmouse.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.847%26rft_id%253Dinfo%253Apmid%252F21623374%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.847&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21623374&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2021.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/5/10/2021.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/10/2007" title="Second-Generation Linkage Maps for the Pacific Oyster Crassostrea gigas Reveal Errors in Assembly of Genome Scaffolds" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/10/2027" title="GC-Content of Synonymous Codons Profoundly Influences Amino Acid Usage" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/5/10"><span>Volume 5 Issue 10, October 2015</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/5/10.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/5/10.cover-source.jpg?itok=JwgNhVLw" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 5 (10)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/10/2021" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-btC353q79Ae5sKLBjy8AIqx1rRxK7UfRYuo388A0qV4" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div><div class="field-item odd"><a href="/category/multiparental-populations" class="" data-icon-position="" data-hide-link-title="0">Multiparental Populations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/5/10/2021" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/5/10/2021" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/5/10/2021" class="active" data-icon-position="" data-hide-link-title="0">High-Density Genotypes of Inbred Mouse Strains: Improved Power and Precision of Association Mapping</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460038" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-6KSP4SmyHTqN3DNjsjJ6YYdMw7caPWxuJZYuHw0qGTU" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/10/2021.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/5/10/2021" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-zXeLlSt6Q3grljaflSaioqarqmCe4gmwF8zMv0pJr2E" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/5/10/2021" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/10/2021.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460038" id="node460038--2" data-pisa="ggg;5/10/2021" data-pisa-master="ggg;g3.115.020784" data-apath="/ggg/5/10/2021.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">High-Density Genotypes of Inbred Mouse Strains: Improved Power and Precision of Association Mapping</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Christoph D. Rau</span>, <span class="highwire-citation-author" data-delta="1">Brian Parks</span>, <span class="highwire-citation-author" data-delta="2">Yibin Wang</span>, <span class="highwire-citation-author" data-delta="3">Eleazar Eskin</span>, <span class="highwire-citation-author" data-delta="4">Petr Simecek</span>, <span class="highwire-citation-author" data-delta="5">Gary A. Churchill</span> and <span class="highwire-citation-author" data-delta="6">Aldons J. Lusis</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2021-2026;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.020784 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460038--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Christoph D. Rau </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Departments of Human Genetics, Medicine, Microbiology, Immunology and Molecular Genetics, UCLA, Los Angeles, California 90095-1679</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christoph%2BD.%2BRau%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rau%20CD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChristoph%2BD.%2BRau%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Brian Parks </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Departments of Human Genetics, Medicine, Microbiology, Immunology and Molecular Genetics, UCLA, Los Angeles, California 90095-1679</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Brian%2BParks%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Parks%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABrian%2BParks%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Yibin Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Anesthesiology, UCLA, Los Angeles, California 90095-1679</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yibin%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYibin%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Eleazar Eskin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Computer Sciences, UCLA, Los Angeles, California 90095-1679</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Eleazar%2BEskin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Eskin%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEleazar%2BEskin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Petr Simecek </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>The Jackson Laboratory, Bar Harbor, Maine 04609 </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Petr%2BSimecek%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Simecek%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APetr%2BSimecek%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Gary A. Churchill </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>The Jackson Laboratory, Bar Harbor, Maine 04609 </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gary%2BA.%2BChurchill%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Churchill%20GA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGary%2BA.%2BChurchill%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Aldons J. Lusis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Departments of Human Genetics, Medicine, Microbiology, Immunology and Molecular Genetics, UCLA, Los Angeles, California 90095-1679</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aldons%2BJ.%2BLusis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lusis%20AJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAldons%2BJ.%2BLusis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jlusis@mednet.ucla.edu" class="" data-icon-position="" data-hide-link-title="0">jlusis@mednet.ucla.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/5/10/2021" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/5/10/2021&amp;title=High-Density%20Genotypes%20of%20Inbred%20Mouse%20Strains%3A%20Improved%20Power%20and%20Precision%20of%20Association%20Mapping" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/5/10/2021&amp;title=High-Density%20Genotypes%20of%20Inbred%20Mouse%20Strains%3A%20Improved%20Power%20and%20Precision%20of%20Association%20Mapping" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/5/10/2021&amp;title=High-Density%20Genotypes%20of%20Inbred%20Mouse%20Strains%3A%20Improved%20Power%20and%20Precision%20of%20Association%20Mapping" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/5/10/2021&amp;text=High-Density%20Genotypes%20of%20Inbred%20Mouse%20Strains%3A%20Improved%20Power%20and%20Precision%20of%20Association%20Mapping" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/5/10/2021&amp;title=High-Density%20Genotypes%20of%20Inbred%20Mouse%20Strains%3A%20Improved%20Power%20and%20Precision%20of%20Association%20Mapping" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/5/10/2021&amp;t=High-Density%20Genotypes%20of%20Inbred%20Mouse%20Strains%3A%20Improved%20Power%20and%20Precision%20of%20Association%20Mapping" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/5/10/2021&amp;title=High-Density%20Genotypes%20of%20Inbred%20Mouse%20Strains%3A%20Improved%20Power%20and%20Precision%20of%20Association%20Mapping" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/5/10/2021&amp;title=High-Density%20Genotypes%20of%20Inbred%20Mouse%20Strains%3A%20Improved%20Power%20and%20Precision%20of%20Association%20Mapping" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460038/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460038" id="node460038--3" data-pisa="ggg;5/10/2021" data-pisa-master="ggg;g3.115.020784" data-apath="/ggg/5/10/2021.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">High-Density Genotypes of Inbred Mouse Strains: Improved Power and Precision of Association Mapping</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Christoph D. Rau</span>, <span class="highwire-citation-author" data-delta="1">Brian Parks</span>, <span class="highwire-citation-author" data-delta="2">Yibin Wang</span>, <span class="highwire-citation-author" data-delta="3">Eleazar Eskin</span>, <span class="highwire-citation-author" data-delta="4">Petr Simecek</span>, <span class="highwire-citation-author" data-delta="5">Gary A. Churchill</span> and <span class="highwire-citation-author" data-delta="6">Aldons J. Lusis</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2021-2026;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.020784 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460038--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Christoph D. Rau </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Departments of Human Genetics, Medicine, Microbiology, Immunology and Molecular Genetics, UCLA, Los Angeles, California 90095-1679</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christoph%2BD.%2BRau%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rau%20CD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChristoph%2BD.%2BRau%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Brian Parks </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Departments of Human Genetics, Medicine, Microbiology, Immunology and Molecular Genetics, UCLA, Los Angeles, California 90095-1679</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Brian%2BParks%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Parks%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABrian%2BParks%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Yibin Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Anesthesiology, UCLA, Los Angeles, California 90095-1679</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yibin%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYibin%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Eleazar Eskin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Computer Sciences, UCLA, Los Angeles, California 90095-1679</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Eleazar%2BEskin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Eskin%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEleazar%2BEskin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Petr Simecek </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>The Jackson Laboratory, Bar Harbor, Maine 04609 </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Petr%2BSimecek%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Simecek%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APetr%2BSimecek%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Gary A. Churchill </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>The Jackson Laboratory, Bar Harbor, Maine 04609 </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gary%2BA.%2BChurchill%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Churchill%20GA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGary%2BA.%2BChurchill%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Aldons J. Lusis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Departments of Human Genetics, Medicine, Microbiology, Immunology and Molecular Genetics, UCLA, Los Angeles, California 90095-1679</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aldons%2BJ.%2BLusis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lusis%20AJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAldons%2BJ.%2BLusis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jlusis@mednet.ucla.edu" class="" data-icon-position="" data-hide-link-title="0">jlusis@mednet.ucla.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2021&amp;rft.epage=2026&amp;rft.atitle=High-Density%20Genotypes%20of%20Inbred%20Mouse%20Strains%3A%20Improved%20Power%20and%20Precision%20of%20Association%20Mapping&amp;rft.volume=5&amp;rft.issue=10&amp;rft.date=2015-10-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Rau%2C+Christoph+D.&amp;rft.au=Parks%2C+Brian&amp;rft.au=Wang%2C+Yibin&amp;rft.au=Eskin%2C+Eleazar&amp;rft.au=Simecek%2C+Petr&amp;rft.au=Churchill%2C+Gary+A.&amp;rft.au=Lusis%2C+Aldons+J."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460038/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460038/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460038/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460038/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460038/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460038/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460038/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460038/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460038/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460038/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460038/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460038/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list"><div class="highwire-articles-in-toc investigations"><h3 class="highwire-toc-heading">Investigations</h3>
<div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span>
</a></div></div><div class="highwire-articles-in-toc multiparental-populations"><h3 class="highwire-toc-heading">Multiparental Populations</h3>
<div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495847" id="node495847" data-pisa="ggg;10/3/951" data-pisa-master="ggg;g3.119.400763" data-apath="/ggg/10/3/951.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/951" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Genome-Wide Association Study in Two Cohorts from a Multi-generational Mouse Advanced Intercross Line Highlights the Difficulty of Replication Due to Study-Specific Heterogeneity</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494983" id="node494983" data-pisa="ggg;9/12/4045" data-pisa-master="ggg;g3.119.400353" data-apath="/ggg/9/12/4045.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/9/12/4045" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Genetic Architecture of Chilling Tolerance in Sorghum Dissected with a Nested Association Mapping Population</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493460" id="node493460" data-pisa="ggg;9/7/2317" data-pisa-master="ggg;g3.119.400098" data-apath="/ggg/9/7/2317.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/9/7/2317" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Testing Pleiotropy <em>vs.</em> Separate QTL in Multiparental Populations</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Multiparental%20Populations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Multiparental Populations</span>
</a></div></div></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/5/10/2021" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-5" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-9" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/5/10/2021.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/5/10/2021.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/5/10/2021.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/5/10/2021.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
