<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/5/10/2027" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/5/10/2027.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/5/10/2027.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/5/10/2027.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/5/10/2027/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/5/10/2027.atom" />
<meta name="HW.pisa" content="ggg;5/10/2027" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="GC-Content of Synonymous Codons Profoundly Influences Amino Acid Usage" />
<meta name="DC.Identifier" content="10.1534/g3.115.019877" />
<meta name="DC.Date" content="2015-10-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2015 Li et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Amino acids typically are encoded by multiple synonymous codons that are not used with the same frequency. Codon usage bias has drawn considerable attention, and several explanations have been offered, including variation in GC-content between species. Focusing on a simple parameter—combined GC proportion of all the synonymous codons for a particular amino acid, termed GCsyn—we try to deepen our understanding of the relationship between GC-content and amino acid/codon usage in more details. We analyzed 65 widely distributed representative species and found a close association between GCsyn, GC-content, and amino acids usage. The overall usages of the four amino acids with the greatest GCsyn and the five amino acids with the lowest GCsyn both vary with the regional GC-content, whereas the usage of the remaining 11 amino acids with intermediate GCsyn is less variable. More interesting, we discovered that codon usage frequencies are nearly constant in regions with similar GC-content. We further quantified the effects of regional GC-content variation (low to high) on amino acid usage and found that GC-content determines the usage variation of amino acids, especially those with extremely high GCsyn, which accounts for 76.7% of the changed GC-content for those regions. Our results suggest that GCsyn correlates with GC-content and has impact on codon/amino acid usage. These findings suggest a novel approach to understanding the role of codon and amino acid usage in shaping genomic architecture and evolutionary patterns of organisms." />
<meta name="DC.Contributor" content="Jing Li" />
<meta name="DC.Contributor" content="Jun Zhou" />
<meta name="DC.Contributor" content="Ying Wu" />
<meta name="DC.Contributor" content="Sihai Yang" />
<meta name="DC.Contributor" content="Dacheng Tian" />
<meta name="article:published_time" content="2015-10-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="GC-Content of Synonymous Codons Profoundly Influences Amino Acid Usage" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Amino acids typically are encoded by multiple synonymous codons that are not used with the same frequency. Codon usage bias has drawn considerable attention, and several explanations have been offered, including variation in GC-content between species. Focusing on a simple parameter—combined GC proportion of all the synonymous codons for a particular amino acid, termed GC&lt;sub&gt;syn&lt;/sub&gt;—we try to deepen our understanding of the relationship between GC-content and amino acid/codon usage in more details. We analyzed 65 widely distributed representative species and found a close association between GC&lt;sub&gt;syn&lt;/sub&gt;, GC-content, and amino acids usage. The overall usages of the four amino acids with the greatest GC&lt;sub&gt;syn&lt;/sub&gt; and the five amino acids with the lowest GC&lt;sub&gt;syn&lt;/sub&gt; both vary with the regional GC-content, whereas the usage of the remaining 11 amino acids with intermediate GC&lt;sub&gt;syn&lt;/sub&gt; is less variable. More interesting, we discovered that codon usage frequencies are nearly constant in regions with similar GC-content. We further quantified the effects of regional GC-content variation (low to high) on amino acid usage and found that GC-content determines the usage variation of amino acids, especially those with extremely high GC&lt;sub&gt;syn&lt;/sub&gt;, which accounts for 76.7% of the changed GC-content for those regions. Our results suggest that GC&lt;sub&gt;syn&lt;/sub&gt; correlates with GC-content and has impact on codon/amino acid usage. These findings suggest a novel approach to understanding the role of codon and amino acid usage in shaping genomic architecture and evolutionary patterns of organisms.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2015/10/01" />
<meta name="citation_mjid" content="ggg;5/10/2027" />
<meta name="citation_id" content="5/10/2027" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/5/10/2027" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/5/10/2027.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/5/10/2027.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/5/10/2027.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.115.019877" />
<meta name="citation_pmid" content="26248983" />
<meta name="citation_volume" content="5" />
<meta name="citation_issue" content="10" />
<meta name="citation_num_pages" content="10" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2027" />
<meta name="citation_lastpage" content="2036" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Jing Li" />
<meta name="citation_author_institution" content="State Key Laboratory of Pharmaceutical Biotechnology, School of Life Sciences, Nanjing University, Nanjing 210093, China" />
<meta name="citation_author_institution" content="Institute for Research on Cancer and Aging, Nice (IRCAN), Nice, France" />
<meta name="citation_author" content="Jun Zhou" />
<meta name="citation_author_institution" content="Department of Organismic and Evolutionary Biology, Harvard University, Cambridge, Massachusetts" />
<meta name="citation_author_email" content="dtian@nju.edu.cn" />
<meta name="citation_author_email" content="jzhou@oeb.harvard.edu" />
<meta name="citation_author" content="Ying Wu" />
<meta name="citation_author_institution" content="State Key Laboratory of Pharmaceutical Biotechnology, School of Life Sciences, Nanjing University, Nanjing 210093, China" />
<meta name="citation_author_institution" content="Institute for Bioscience and Biotechnology Research, University of Maryland, Rockville, Maryland" />
<meta name="citation_author" content="Sihai Yang" />
<meta name="citation_author_institution" content="State Key Laboratory of Pharmaceutical Biotechnology, School of Life Sciences, Nanjing University, Nanjing 210093, China" />
<meta name="citation_author" content="Dacheng Tian" />
<meta name="citation_author_institution" content="State Key Laboratory of Pharmaceutical Biotechnology, School of Life Sciences, Nanjing University, Nanjing 210093, China" />
<meta name="citation_author_email" content="dtian@nju.edu.cn" />
<meta name="citation_author_email" content="jzhou@oeb.harvard.edu" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=G. E. Andersson;citation_author=C. G. Kurland;citation_title=An extreme codon preference strategy: codon reassignment;citation_pages=530-544;citation_volume=8;citation_year=1991;citation_issue=4;citation_pmid=1921708" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=B. G. Barrell;citation_author=A. T. Bankier;citation_author=J. Drouin;citation_title=A different genetic code in human mitochondria.;citation_pages=189-194;citation_volume=282;citation_year=1979;citation_issue=5735;citation_pmid=226894;citation_doi=10.1038/282189a0" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=B. Berkhout;citation_author=F. J. van Hemert;citation_title=The unusual nucleotide content of the HIV RNA genome results in a biased amino acid composition of HIV proteins;citation_pages=1705-1711;citation_volume=22;citation_year=1994;citation_issue=9;citation_pmid=8202375;citation_doi=10.1093/nar/22.9.1705" />
<meta name="citation_reference" content="citation_journal_title=Gene;citation_journal_abbrev=Gene;citation_author=G. Bernardi;citation_title=The compositional evolution of vertebrate genomes.;citation_pages=31-43;citation_volume=259;citation_year=2000;citation_issue=1-2;citation_pmid=11163959;citation_doi=10.1016/S0378-1119(00)00441-8" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=G. Bernardi;citation_author=G. Bernardi;citation_title=Compositional constraints and genome evolution.;citation_pages=1-11;citation_volume=24;citation_year=1986;citation_issue=1-2;citation_pmid=3104608;citation_doi=10.1007/BF02099946" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=J. Bohlin;citation_author=O. Brynildsrud;citation_author=T. Vesth;citation_author=E. Skjerve;citation_author=D. W. Ussery;citation_title=Amino acid usage is asymmetrically biased in AT- and GC-rich microbial genomes.;citation_pages=e69878;citation_volume=8;citation_year=2013;citation_pmid=23922837;citation_doi=10.1371/journal.pone.0069878" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. Bulmer;citation_title=The selection-mutation-drift theory of synonymous codon usage.;citation_pages=897-907;citation_volume=129;citation_year=1991;citation_issue=3;citation_pmid=1752426" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=J. V. Chamary;citation_author=J. L. Parmley;citation_author=L. D. Hurst;citation_title=Hearing silence: non-neutral evolution at synonymous sites in mammals.;citation_pages=98-108;citation_volume=7;citation_year=2006;citation_issue=2;citation_pmid=16418745;citation_doi=10.1038/nrg1770" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=S. L. Chen;citation_author=W. Lee;citation_author=A. K. Hottes;citation_author=L. Shapiro;citation_author=H. H. McAdams;citation_title=Codon usage between genomes is constrained by genome-wide mutational processes.;citation_volume=101;citation_year=2004;citation_pmid=14990797;citation_doi=10.1073/pnas.0307827100" />
<meta name="citation_reference" content="citation_journal_title=Gene;citation_journal_abbrev=Gene;citation_author=H. Chiapello;citation_author=F. Lisacek;citation_author=M. Caboche;citation_author=A. Hénaut;citation_title=Codon usage and gene function are related in sequences of Arabidopsis thaliana.;citation_pages=GC1-GC38;citation_volume=209;citation_year=1998;citation_issue=1-2;citation_pmid=9583944;citation_doi=10.1016/S0378-1119(97)00671-9" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=D. W. Collins;citation_author=T. H. Jukes;citation_title=Relationship between G + C in silent sites of codons and amino acid composition of human proteins.;citation_pages=201-213;citation_volume=36;citation_year=1993;citation_issue=3;citation_pmid=8483158;citation_doi=10.1007/BF00160475" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=F. H. Crick;citation_title=The origin of the genetic code.;citation_pages=367-379;citation_volume=38;citation_year=1968;citation_issue=3;citation_pmid=4887876;citation_doi=10.1016/0022-2836(68)90392-6" />
<meta name="citation_reference" content="citation_journal_title=Mamm. Genome Off. J. Int. Mamm. Genome Soc.;citation_author=S. H. Cross;citation_author=V. H. Clark;citation_author=M. W. Simmen;citation_author=W. A. Bickmore;citation_author=H. Maroon;citation_title=CpG island libraries from human chromosomes 18 and 22: landmarks for novel genes.;citation_pages=373-383;citation_volume=11;citation_year=2000;citation_doi=10.1007/s003350010071" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=G. D’Onofrio;citation_author=D. Mouchiroud;citation_author=B. Aïssani;citation_author=C. Gautier;citation_author=G. Bernardi;citation_title=Correlations between the compositional properties of human genes, codon usage, and amino acid composition of proteins.;citation_pages=504-510;citation_volume=32;citation_year=1991;citation_issue=6;citation_pmid=1908021;citation_doi=10.1007/BF02102652" />
<meta name="citation_reference" content="citation_journal_title=Gene;citation_journal_abbrev=Gene;citation_author=G. D’Onofrio;citation_author=K. Jabbari;citation_author=H. Musto;citation_author=G. Bernardi;citation_title=The correlation of protein hydropathy with the base composition of coding sequences.;citation_pages=3-14;citation_volume=238;citation_year=1999;citation_issue=1;citation_pmid=10570978;citation_doi=10.1016/S0378-1119(99)00257-7" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=M. Dos Reis;citation_author=L. Wernisch;citation_author=R. Savva;citation_title=Unexpected correlations between gene expression and codon usage bias from microarray data for the whole Escherichia coli K-12 genome;citation_pages=6976-6985;citation_volume=31;citation_year=2003;citation_issue=23;citation_pmid=14627830;citation_doi=10.1093/nar/gkg897" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in genetics &amp; development;citation_journal_abbrev=Curr Opin Genet Dev;citation_author=L. Duret;citation_title=Evolution of synonymous codon usage in metazoans.;citation_pages=640-649;citation_volume=12;citation_year=2002;citation_issue=6;citation_pmid=12433576;citation_doi=10.1016/S0959-437X(02)00353-2" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=L. Duret;citation_author=D. Mouchiroud;citation_title=Expression pattern and, surprisingly, gene length shape codon usage in Caenorhabditis, Drosophila, and Arabidopsis;citation_pages=4482-4487;citation_volume=96;citation_year=1999;citation_issue=8;citation_pmid=10200288;citation_doi=10.1073/pnas.96.8.4482" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=A. Eyre-Walker;citation_title=Synonymous codon bias is related to gene length in Escherichia coli: selection for translational accuracy?;citation_pages=864-872;citation_volume=13;citation_year=1996;citation_issue=6;citation_pmid=8754221;citation_doi=10.1093/oxfordjournals.molbev.a025646" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=A. Eyre-Walker;citation_author=L. D. Hurst;citation_title=The evolution of isochores.;citation_pages=549-555;citation_volume=2;citation_year=2001;citation_issue=7;citation_pmid=11433361;citation_doi=10.1038/35080577" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=M. P. Francino;citation_author=H. Ochman;citation_title=Isochores result from mutation not selection.;citation_pages=30-31;citation_volume=400;citation_year=1999;citation_issue=6739;citation_pmid=10403245;citation_doi=10.1038/21804" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=M. Gouy;citation_author=C. Gautier;citation_title=Codon usage in bacteria: correlation with gene expressivity;citation_pages=7055-7074;citation_volume=10;citation_year=1982;citation_issue=22;citation_pmid=6760125;citation_doi=10.1093/nar/10.22.7055" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=R. Grantham;citation_author=C. Gautier;citation_author=M. Gouy;citation_author=R. Mercier;citation_author=A. Pavé;citation_title=Codon catalog usage and the genome hypothesis;citation_pages=r49-62;citation_volume=8;citation_year=1980;citation_issue=1;citation_pmid=6986610;citation_doi=10.1093/nar/8.1.197-c" />
<meta name="citation_reference" content="citation_journal_title=Genetica;citation_author=X. Gu;citation_author=D. Hewett-Emmett;citation_author=W. H. Li;citation_title=Directional mutational pressure affects the amino acid composition and hydrophobicity of proteins in bacteria.;citation_pages=383-391;citation_volume=102–103;citation_year=1998;citation_doi=10.1023/A:1017028102013" />
<meta name="citation_reference" content="citation_journal_title=Trends in biotechnology;citation_journal_abbrev=Trends Biotechnol;citation_author=C. Gustafsson;citation_author=S. Govindarajan;citation_author=J. Minshull;citation_title=Codon bias and heterologous protein expression.;citation_pages=346-353;citation_volume=22;citation_year=2004;citation_issue=7;citation_pmid=15245907;citation_doi=10.1016/j.tibtech.2004.04.006" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genetics;citation_journal_abbrev=Annu Rev Genet;citation_author=R. Hershberg;citation_author=D. A. Petrov;citation_title=Selection on codon bias.;citation_pages=287-299;citation_volume=42;citation_year=2008;citation_pmid=18983258;citation_doi=10.1146/annurev.genet.42.110807.091442" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=F. Hildebrand;citation_author=A. Meyer;citation_author=A. Eyre-Walker;citation_title=Evidence of selection upon genomic GC-content in bacteria.;citation_pages=e1001107;citation_volume=6;citation_year=2010;citation_issue=9;citation_pmid=20838593;citation_doi=10.1371/journal.pgen.1001107" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=T. Ikemura;citation_title=Correlation between the abundance of Escherichia coli transfer RNAs and the occurrence of the respective codons in its protein genes: a proposal for a synonymous codon choice that is optimal for the E. coli translational system.;citation_pages=389-409;citation_volume=151;citation_year=1981;citation_issue=3;citation_pmid=6175758;citation_doi=10.1016/0022-2836(81)90003-6" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=S. Karlin;citation_author=J. Mrázek;citation_title=What drives codon choices in human genes?;citation_pages=459-472;citation_volume=262;citation_year=1996;citation_issue=4;citation_pmid=8893856;citation_doi=10.1006/jmbi.1996.0528" />
<meta name="citation_reference" content="citation_journal_title=Molecular microbiology;citation_journal_abbrev=Mol Microbiol;citation_author=S. Karlin;citation_author=J. Mrázek;citation_author=A. M. Campbell;citation_title=Codon usages in different gene classes of the Escherichia coli genome.;citation_pages=1341-1355;citation_volume=29;citation_year=1998;citation_issue=6;citation_pmid=9781873;citation_doi=10.1046/j.1365-2958.1998.01008.x" />
<meta name="citation_reference" content="citation_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=R. D. Knight;citation_author=S. J. Freeland;citation_author=L. F. Landweber;citation_title=A simple model based on mutation and selection explains trends in codon and amino-acid usage and GC composition within and across genomes.;citation_pages=RESEARCH0010-RESEARCH0010;citation_volume=2;citation_year=2001;citation_issue=4;citation_pmid=11305938" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=G. Kudla;citation_author=L. Lipinski;citation_author=F. Caffin;citation_author=A. Helwak;citation_author=M. Zylicz;citation_title=High guanine and cytosine content increases mRNA levels in mammalian cells.;citation_pages=e180-e180;citation_volume=4;citation_year=2006;citation_issue=6;citation_pmid=16700628;citation_doi=10.1371/journal.pbio.0040180" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=B. Lafay;citation_author=A. T. Lloyd;citation_author=M. J. McLean;citation_author=K. M. Devine;citation_author=P. M. Sharp;citation_title=Proteome composition and codon usage in spirochaetes: species-specific and DNA strand-specific mutational biases.;citation_volume=27;citation_year=1999;citation_pmid=10075995;citation_doi=10.1093/nar/27.7.1642" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=E. S. Lander;citation_author=L. M. Linton;citation_author=B. Birren;citation_author=C. Nusbaum;citation_author=M. C. Zody;citation_title=Initial sequencing and analysis of the human genome.;citation_pages=860-921;citation_volume=409;citation_year=2001;citation_issue=6822;citation_pmid=11237011;citation_doi=10.1038/35057062" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=M. Li;citation_author=A. Tzagoloff;citation_title=Assembly of the mitochondrial membrane system: sequences of yeast mitochondrial valine and an unusual threonine tRNA gene.;citation_pages=47-53;citation_volume=18;citation_year=1979;citation_issue=1;citation_pmid=389433;citation_doi=10.1016/0092-8674(79)90352-0" />
<meta name="citation_reference" content="citation_journal_title=Biosystems;citation_journal_abbrev=Biosystems;citation_author=Q. Liu;citation_author=S. Dou;citation_author=Z. Ji;citation_author=Q. Xue;citation_title=Synonymous codon usage and gene function are strongly related in Oryza sativa.;citation_pages=123-131;citation_volume=80;citation_year=2005;citation_issue=2;citation_pmid=15823411;citation_doi=10.1016/j.biosystems.2004.10.008" />
<meta name="citation_reference" content="citation_journal_title=Gene;citation_journal_abbrev=Gene;citation_author=J. R. Lobry;citation_title=Influence of genomic G+C content on average amino-acid composition of proteins from 59 bacterial species.;citation_pages=309-316;citation_volume=205;citation_year=1997;citation_issue=1-2;citation_pmid=9461405;citation_doi=10.1016/S0378-1119(97)00403-4" />
<meta name="citation_reference" content="citation_journal_title=Current biology :  CB;citation_journal_abbrev=Curr Biol;citation_author=X. Maside;citation_author=A. W. Lee;citation_author=B. Charlesworth;citation_title=Selection on codon usage in Drosophila americana.;citation_pages=150-154;citation_volume=14;citation_year=2004;citation_issue=2;citation_pmid=14738738;citation_doi=10.1016/j.cub.2003.12.055" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=T. Miyata;citation_author=H. Hayashida;citation_author=T. Yasunaga;citation_author=M. Hasegawa;citation_title=The preferential codon usages in variable and constant regions of immunoglobulin genes are quite distinct from each other;citation_pages=2431-2438;citation_volume=7;citation_year=1979;citation_issue=8;citation_pmid=523321;citation_doi=10.1093/nar/7.8.2431" />
<meta name="citation_reference" content="citation_journal_title=Gene;citation_author=M. Nabiyouni;citation_author=A. Prakash;citation_author=A. Fedorov;citation_title=Vertebrate codon bias indicates a highly GC-rich ancestral genome.;citation_pages=113-119;citation_volume=519;citation_year=2013;citation_pmid=23376453;citation_doi=10.1016/j.gene.2013.01.033" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. B. Plotkin;citation_author=H. Robins;citation_author=A. J. Levine;citation_title=Tissue-specific codon usage and the expression of human genes.;citation_volume=101;citation_year=2004;citation_pmid=15314228;citation_doi=10.1073/pnas.0404957101" />
<meta name="citation_reference" content="citation_journal_title=Biochimica et Biophysica Acta. Protein Structure and Molecular Enzymology;citation_journal_abbrev=Biochimica et Biophysica Acta. Protein Structure and Molecular Enzymology;citation_author=T. D. Porter;citation_title=Correlation between codon usage, regional genomic nucleotide composition, and amino acid composition in the cytochrome P-450 gene superfamily.;citation_pages=394-400;citation_volume=1261;citation_year=1995;citation_issue=3;citation_pmid=7742368" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=R. Raghavan;citation_author=Y. D. Kelkar;citation_author=H. Ochman;citation_title=A selective force favoring increased G+C content in bacterial genes.;citation_volume=109;citation_year=2012;citation_pmid=22908296;citation_doi=10.1073/pnas.1205683109" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=P. M. Sharp;citation_author=T. M. Tuohy;citation_author=K. R. Mosurski;citation_title=Codon usage in yeast: cluster analysis clearly differentiates highly and lowly expressed genes;citation_pages=5125-5143;citation_volume=14;citation_year=1986;citation_issue=13;citation_pmid=3526280;citation_doi=10.1093/nar/14.13.5125" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=G. A. C. Singer;citation_author=D. A. Hickey;citation_title=Nucleotide Bias Causes a Genomewide Bias in the Amino Acid Composition of Proteins;citation_pages=1581-1588;citation_volume=17;citation_year=2000;citation_issue=11;citation_pmid=11070046;citation_doi=10.1093/oxfordjournals.molbev.a026257" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=N. Sueoka;citation_title=CORRELATION BETWEEN BASE COMPOSITION OF DEOXYRIBONUCLEIC ACID AND AMINO ACID COMPOSITION OF PROTEIN;citation_pages=1141-1149;citation_volume=47;citation_year=1961;citation_issue=8;citation_pmid=16590864;citation_doi=10.1073/pnas.47.8.1141" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=N. Sueoka;citation_title=ON THE GENETIC BASIS OF VARIATION AND HETEROGENEITY OF DNA BASE COMPOSITION;citation_pages=582-592;citation_volume=48;citation_year=1962;citation_issue=4;citation_pmid=13918161;citation_doi=10.1073/pnas.48.4.582" />
<meta name="citation_reference" content="citation_journal_title=Gene;citation_journal_abbrev=Gene;citation_author=N. Sueoka;citation_author=Y. Kawanishi;citation_title=DNA G+C content of the third codon position and codon usage biases of human genes.;citation_pages=53-62;citation_volume=261;citation_year=2000;citation_issue=1;citation_pmid=11164037;citation_doi=10.1016/S0378-1119(00)00480-7" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=T. V. Tatarinova;citation_author=N. N. Alexandrov;citation_author=J. B. Bouck;citation_author=K. A. Feldmann;citation_title=GC3 biology in corn, rice, sorghum and other grasses.;citation_pages=308-308;citation_volume=11;citation_year=2010;citation_pmid=20470436;citation_doi=10.1186/1471-2164-11-308" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Notes;citation_author=P. Villesen;citation_title=FaBox: an online toolbox for fasta sequences.;citation_pages=965-968;citation_volume=7;citation_year=2007;citation_doi=10.1111/j.1471-8286.2007.01821.x" />
<meta name="citation_reference" content="citation_journal_title=Research in Microbiology (Paris);citation_journal_abbrev=Research in Microbiology (Paris);citation_author=V. Wilquet;citation_author=M. Van de Casteele;citation_title=The role of the codon first letter in the relationship between genomic GC content and protein amino acid composition.;citation_pages=21-32;citation_volume=150;citation_year=1999;citation_issue=1;citation_pmid=10096131;citation_doi=10.1016/S0923-2508(99)80043-6" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=Z. Yang;citation_author=R. Nielsen;citation_title=Mutation-selection models of codon substitution and their use to estimate selective strengths on codon usage.;citation_volume=25;citation_year=2008;citation_pmid=18178545;citation_doi=10.1093/molbev/msm284" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=L. Zhu;citation_author=Y. Zhang;citation_author=W. Zhang;citation_author=S. Yang;citation_author=J.-Q. Chen;citation_title=Patterns of exon-intron architecture variation of genes in eukaryotic genomes.;citation_pages=47-47;citation_volume=10;citation_year=2009;citation_pmid=19166620;citation_doi=10.1186/1471-2164-10-47" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="GC-Content of Synonymous Codons Profoundly Influences Amino Acid Usage" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10.cover-source.jpg" />
<meta name="twitter:description" content="Amino acids typically are encoded by multiple synonymous codons that are not used with the same frequency. Codon usage bias has drawn considerable attention, and several explanations have been offered, including variation in GC-content between species. Focusing on a simple parameter—combined GC proportion of all the synonymous codons for a particular amino acid, termed GCsyn—we try to deepen our understanding of the relationship between GC-content and amino acid/codon usage in more details. We analyzed 65 widely distributed representative species and found a close association between GCsyn, GC-content, and amino acids usage. The overall usages of the four amino acids with the greatest GCsyn and the five amino acids with the lowest GCsyn both vary with the regional GC-content, whereas the usage of the remaining 11 amino acids with intermediate GCsyn is less variable. More interesting, we discovered that codon usage frequencies are nearly constant in regions with similar GC-content. We further quantified the effects of regional GC-content variation (low to high) on amino acid usage and found that GC-content determines the usage variation of amino acids, especially those with extremely high GCsyn, which accounts for 76.7% of the changed GC-content for those regions. Our results suggest that GCsyn correlates with GC-content and has impact on codon/amino acid usage. These findings suggest a novel approach to understanding the role of codon and amino acid usage in shaping genomic architecture and evolutionary patterns of organisms." />
<meta name="og-title" property="og:title" content="GC-Content of Synonymous Codons Profoundly Influences Amino Acid Usage" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/5/10/2027" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Amino acids typically are encoded by multiple synonymous codons that are not used with the same frequency. Codon usage bias has drawn considerable attention, and several explanations have been offered, including variation in GC-content between species. Focusing on a simple parameter—combined GC proportion of all the synonymous codons for a particular amino acid, termed GCsyn—we try to deepen our understanding of the relationship between GC-content and amino acid/codon usage in more details. We analyzed 65 widely distributed representative species and found a close association between GCsyn, GC-content, and amino acids usage. The overall usages of the four amino acids with the greatest GCsyn and the five amino acids with the lowest GCsyn both vary with the regional GC-content, whereas the usage of the remaining 11 amino acids with intermediate GCsyn is less variable. More interesting, we discovered that codon usage frequencies are nearly constant in regions with similar GC-content. We further quantified the effects of regional GC-content variation (low to high) on amino acid usage and found that GC-content determines the usage variation of amino acids, especially those with extremely high GCsyn, which accounts for 76.7% of the changed GC-content for those regions. Our results suggest that GCsyn correlates with GC-content and has impact on codon/amino acid usage. These findings suggest a novel approach to understanding the role of codon and amino acid usage in shaping genomic architecture and evolutionary patterns of organisms." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10.cover-source.jpg" />
    <title>GC-Content of Synonymous Codons Profoundly Influences Amino Acid Usage | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"KgxRE7dLcNvoojJ8Wms6BGQriUFE-O133rwHkfhzaFw","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460151","apath":"\/ggg\/5\/10\/2027.atom","pisa":"ggg;5\/10\/2027","ac":{"\/ggg\/5\/10\/2027.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/5\/10\/2027.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;5\/10\/2027"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;5\/10\/2027"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/5\/10\/2027":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022+9xNBO6ycNZK2GnZglfufbPp182bwPwtPO\\\\\\\/jPMtwbK\\\\\\\/3bAAaosSkXka0Lac3vAt\\\\\\\/0Z8TYTLSM8+vct5IsK8MJyTnCJ8cR9TVXnyoc2DM0REO5gCuiH782a1z1VX4QE9dO47AI7Hq2BHcDoyWXwKP0TXYSUNMyAyS9Zzv5VWdzNLr2JhBptTn+jP1iQLbuG4fD8uE8H5r7a+D3urKNR4q9bHIHjGs4qArqjutNSh4qCef4FoEfclOCnWU9k6XHqSk1cMUr8wGc6P2O68O0L+Tm8QO4RXh+aFw8XpNkpMj0sroRjVo0iq0C0fUBZJy8TU8Y1HEHo8F5D1IjAVe8FI1CLdlNpfSXCrbptbSO8yBPKKRhOLt2fy3tn5DjRkB30n35fRZxyd8W9lRsEyiRo2KDp++lA6uT1lBDdKxiLtc4RwgGYHDeQwva2y9OAb0CVOHplVQ5IgFxO1Op3PcrYKVmcghz40SUa2cqTloogsMu+T\\\\\\\/f+gX+dHw4UsKdtmH8Y7VP6CRlJx0902HJiBDJf3BgQ2Sdbs+4DWikCocwwMzGMCk\\\\\\\/\\\\\\\/tbXP7nCFamP5CJu6jdsb0KduGGz24QhL8z6NjdNzlbTL7cVZB9AVyRImRTci+r2p3aqdSVNwb4S0NMSzYUvV8IepO72gBw86\\\\\\\/KTly2CbZdetl\\\\\\\/NSi01ohcX5iC+XE1svU7PiAMkoxVWuW7KGagDakE8jnCv9bHsqvTHZ+BsiaGdC3OidVaXChcsYQ3VI6w40nmZHDCk1BaRC15msfCif1U+TODZiQplsZ4\\\\\\\/tjaonzIMkSa80qERoZ5ehiXmBjyhfJboXsGaFSP\\\\\\\/f4mgwRqNAJJad2nJX4mh6SYneUCEG7w3O1IqcABwrae0jBdmuEkOw7JziYyGDJfGP0+3cr3Dzx4QMz4FtqO5Bzugxkxr5Idn4HC8WUopQ+vpYKK73Mcvqtf4v3HZiwl68txsI4PbJ0aVF3B8QAd+J34H+OROz7SLN4zXIu7EhDdH1fVKoU8Cq7LRK3Wvh1C\\\\\\\/EW4\\\\\\\/VV42+5ISVl5VMCK3jaOQhdCHRiGqA9rdeglRRC32tC\\\\\\\/TU3137Zoqgc9GKG\\\\\\\/DJ5utimF16L9kvB1CT39Phs5KAMqNgmUFFRbE6qCU1t6Eiso5+DlQHMpAbLa5P8lfZ4OXPP5FhW3ohbY\\\\\\\/bQ02x1ZEf4lLKFEtDpWCSMYCuK3qIvzB\\\\\\\/TADB3vH9pkEz49P2bKeuQd5X9VEks\\\\\\\/Up0HyhVMNFUT3Bnqh2jvxvl0c4H1+k7fXtHeKjRoTu9Z8UIOhPdR6SLm\\\u0022,\\\u0022iv\\\u0022:\\\u0022R7Dkls0Ix2rgdpEbZkgGuw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00224230eca0accfe4a62d2cdb191e731a8df33870ae7faf82880f285eb4a59323f0\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022DG6mrUWOd9QeSlW2HE1hUQDybHHaB6oYayhhqRPuwu7q\\\\\\\/wzW7sU0HjmRQSwwKyR5ESB\\\\\\\/y3wbk4cZzn1gKt19lVxYq+ko6VhHSgDi4pOFURPjC45c6WaXTyzqKplxapXdqwLNaPjOTj8D+JPH\\\\\\\/BzZhIXXpf5T4uByPzhiGqLbGuxwvSDjoGdvpCdZ0ElYs3sRo26YrjrZ8P0tfKLLnxXOO3WHI4ncqbKyrQM5Yq5hUTwScTuMHsZ0V3yOCOR5rRc385oxhI43KRf7n7Nd\\\\\\\/k2STCj8P0AM8mSLG8f6yKq6EpNqT5hqyN43s7rWjCapGFrelptQ8zOvfDxNcNc1pgN4RrUaQ1qw8wIQ1FrsFq7QgCVpfep55baM+z0ta+hWgatOO0WwBAuAQeUYmIJUGY3FcGeE1kBwdJtQoJYahtAYAbJXDYX3\\\\\\\/vtFQTgnYFEj9m+ImfLBq7hBNIPGdmhYFBV+AZXJshKiwBvtRH5qgYRAvTrwIkyVcTtskAEmQJYRwr8LMGShvC1QW3oXy2cM\\\\\\\/x8d6KRuq3FtUplmzSGbjU13awVgaPbL1KYAkT6iJuM2xTh6+OoJgLOVkLB1muiSc8vfcOB7HXNTZOCdoCMnXqmglLi781akTNkxsdBVQniy4xpFZSw3J6Ji3YP3WP66TJ2Bx1gkvkyPf5X+VMoeDLBSklmLNYzYTejZr6xguC1zuMEG0bsbY+3yP2L74F1w75mi7iGW68WQ+7vGKU+D\\\\\\\/VzPYivPjqzxKk3AtEWquSOBSxPvjXXb0sa8rHs6xqqm\\\\\\\/ybjh7TvEKVucGcBn8617Vg9\\\\\\\/BhyGP4ati2PBQp9SWz32TB+RnTp51qN3y6q9Z5IHeqSum5vyrlVNdxkZ0hsaPBG5QB+1rlsyOv2ceSUYm0tNwQQXl3srlcpbKEBN+FYPxn\\\\\\\/6VVMBnxM9cuSS\\\\\\\/BGJ9pComo3xZp\\\\\\\/jp7g3r\\\\\\\/19ASbFHXYQnLpiVWkGWKOAkh7VCxZeuIJxa25wgIFFpETUypcaysjxsPC224xJRygRnU9hhn4+V1tUA56TgBeozoShzvFZnvmH7tESdijbQ+EbOwpGkl2urzNdsXoirc9JfzZMjsz1E4\\\\\\\/frzpwabRdDdeq1ZkJ2oGXRSTR6MYWL8kEH7lHWCiERQjP+6cScwRoxVVhWLF0tA2drrNp9zSdU19uAjWaL+bPBJLamEIOPVlotXymDD0ChBQimWMxtbBOAm78HdxrGo\\\\\\\/oJQ1UeIhc23wFHet5hg91+WNrcKFR3nOeOs\\\\\\\/5Nv7UMzLQdko2G6lOXa6XR7oNkSMLZzgCWTormpsJoqqa7aqB8GyG+cpjzYAd9nsKnBYKP77+hd7UlSeSpdA3zySJc0T9OEax9C1ukUTHOJxz0cIw55qjiZaIVCGgApgANJhaTMaWdKiIip2PnSSybAeafVQne+sliT9Spjg\\\\\\\/Xlzwq4\\\\\\\/8dcK1X\\\\\\\/w7LLGGeHzabt7uZQe4WgKrChfqAEARZtKxmYlZwrXGSPSfRCPPZDbwvNqw4pNPFJD\\\\\\\/nLDfOuCgCcaQobqyHARkqzM8jZXG1v1UY4vFTaPFF5TIbUHHdCaIQmYdEIERcoKaagCFFOce2BnlcHVkq9waEAVWkiCWfkAfnPmnjh7p+DKU+jkWTyfJyfHsgLYV2EdlVoUaSgFElgqRa1pGcKFrAKza9sG5sSyuD5\\\\\\\/+0fGycoXwNZaiRnbcf5dITKINFGxWHLSBQBIR8YhL\\\\\\\/0ZiTnzP2wYilWNRy02rkFYzACBI5LZTUhRmimkY34S69yEhy3DRfJ7sJNQ26mgFKKXIOTY0Z\\\\\\\/zeYZ43I2iqsc0VJTi7Q1C1aVqheSi\\\\\\\/m+7+lmfcayenRE8c1+gDGR\\\\\\\/v9EIgeCa7CQI92Lcc4cJFhSB3xGLziQvX+8XcdM2Qjk0Gi9PKFQdcVy\\\\\\\/J\\\\\\\/AjLpwE9+54Obzpb\\\\\\\/HnkdYJyCa+DBdGRlGzgJ\\\\\\\/b13BS63a1GPOubPJ82kuI+bvDa1QrLv4DuadEnVIpnVeEyCh\\\\\\\/XA2t1w6qzwvAZwFXfHmb4ScNVJerUn+I5n8RC0+hzhL+KUP1udU0voKVowP+m7zpHsiw8kHaNjGLMAu69Zd2j3AAY18ArajfxV+yGhV3dwQBWcRkvoHsw1ZP49PC8UpI0M6P6276gZcoolcT7dsSj1Hhxqn59r0KqJSnE3op\\\\\\\/xtGYYSUZePWtRXbq1unpc+dQDspXiqjaio96BSPNuOD0uPwjhVQCJsfg5Xb2f0TDrytnz8VdscRW18RVi5zdaV6688ou5Psb1qIx+7FsQ5CeO8iCEa3vtmrrs7v8dW\\\\\\\/2ADEYRni6Pw6xTQzZ0rjXmS5YjHQjCt1py\\\\\\\/15exM7NXLhy3BBS41AHEKr1hkelTNiohvSk3\\\\\\\/UBKvGNxHUlG6DvgrpHIjmRKck7LkS8Bl2newpYzPB2OEcbZNNXPlWRbm7wqbQzTpBLkANKM+\\\\\\\/u9sC4JwWLWQX8mMtr6ee6c33iQsuMhtfmYlpEJLChDxM4g3suOKDOirKratfspXa1FGoXHeaVoqffcljKPM3I7fB2Wt3i\\\\\\\/Jr50tBYM3jv2VJgAmSiu2yob0XsfndVgFLd5tVS4g1gBwuRsj8RB9DA2QVC\\\\\\\/oduguP7odlaLhnOnZuoiaE8icFq4F6l45DHl\\\\\\\/ljahs\\\\\\\/o+9T3ME\\\\\\\/1P7bKp4zvyW9owCRTr6e4k54wmA+AkKjwONlslFHwIYVhI\\\\\\\/NRb+g56sIxdVhmA2rhxamHoAmbMHE1knUjvdPJjYan8H3TTxGgmM2pIyuWaM8pgGkzw1EhAcFddFk5+QihdQBTh95EMEhfAvDWpzO2\\\\\\\/5YUBCGxER+oeRoMpf44endQqpovCvgg2tuaX6LAtwenq8qGlCfMdHFUeu09NlJcv2TN0bXiEk8vnXX\\\\\\\/vFx\\\\\\\/aJpz+2brfIvH\\\\\\\/k1IvSWhUSNLMPYWalzICUrhztSN3Sq4Fx2jNI3JD5F15WIk5qQHAULUVtSCNOtEdllFlUwhWsIkP5FOWfSPn53\\\\\\\/jNA1eedZMCs3lxhiMfDcXAdaAnJUk5hGrVwAX9DUrkUwnb6cGjtsP4Hw9IXGx7cRJBPdCZX9whMrWUN7pG8EfgM\\\\\\\/NOP\\\\\\\/TNk7VQdMxB51xEJyt79XLvKA7vKwvp0UIk9ID0XI\\\\\\\/qWs+EUzy5rc7m3egeuII8k3s8+ItCapmkBcMgP5J9QNE+PJAksKDhnCJaKVAgjBf0b9+yEzgHdnLWfPRn+J5ecgCdxVdZuFnnphFhfhfwVNws0EWUr1sUUOoOhNJlkXGlSD9d9h\\\\\\\/pXGPq7LqhXNegzAXwPb0gbD8bPvhtkL4h18UqKNmsUgDDNshrYFnfYjyqvpDTOac43bAeWIamCyRrTQC6KHBZeKugNCuAINaheJXNjo89vXeMvaTnjpg+XeKqJMg9hsBLpvHVIXZO+nAEzbs\\\\\\\/FeqFCb6717U3ppoC9rc3Nm5KsJiQIj161Rh+GQTanqKFF1B4lyfQJOB1aG1kA\\\\\\\/fNk1TVB8lzQ46mYqau6dQmPdkn\\\\\\\/9gDavUWyuNATR3yWFrXkB7fH2yLTwTaraULVpRs8b\\\\\\\/0bruy1fwLEFqUODm37zMXerPtrOuKgMSqnOnFmlLwl5aXfyxE52lJ7E3ef8nzngS\\\\\\\/O2I\\\\\\\/2XX3Hae\\\\\\\/IOINltzPoz9j3YuR1xD\\\\\\\/tgOEJohd8teAW8KRLtuD\\\\\\\/NtFqNkj0Duz2CV2LmmczPVRFkttYkQnQs7Q5rPnuJ7mlxvlJCKQn4ATUvUJQRiXq+ScBMHDBWF7SfpgojJYoFOHqVCDUjQOgBRCxogKbZdgJz5pjSvmN2RPYv8GctdRof2gq438zohpANEtXYKIGaEP9Sb\\\\\\\/qFOAVtF8OZGD+SLpYlL2sdb4BmKcwes+hhRrGABUHjfsT6CpwpB\\\\\\\/nEy22J7MtoznqbovqVwoTpqgyaEqa2XZQUmARRDbtfPidQ4EKEWnv7dUHEgCakv5Jvp5C5odtMit8KrIL1R3+E1l2fVfxb3usu4Ym0f\\\\\\\/eH9+5WOvBdBFVxg5FpBoNZ0KBzesN6TgfPJfwmNJnI5sevsNUQdkxhxQX8C2cCIwMJHqUAAKHhL3+a1DtKH9e8b\\\\\\\/eD3XYxLdaUOCqSy9PS9FbBBqn+tXtq+yqJ41Zz+J51BVtC10lcMImQS9T244R8ytMO8jxE9+IzKsnMhQQLhrtuhtBsIPEkcwcIjFAY4ND\\\\\\\/mfQHADSqlpexQf0lrhngZ\\\\\\\/LNrLKHr719XTI8sn5PWgxj8dQK1j0ecE2WR8eJWYPxJs8GyyZm5vuoTCTzwsI2Tn+Fiqwj1ZQSzet0QWrG7xKWtKBQTMdrDUUOb94tsW0PyniZowAoim1JgX4zi69LXuwdVXSFWkXbPk9n\\\\\\\/KR0tdG9QAtPwWT95wNoUgh0E6OX6DisffXPJ2ZtC339o6CN3QclbtpCithvwoQLr+ydSEim5EpWb6rlto2XbfiZpDyl7qgLl8TuYIvQ1Qcbpd2F9Vbyv\\\\\\\/oo9rLQMzmB99YWpFi5EpWgVC3Kpkx3aipzBXNx8nvTg6Qr\\\\\\\/BdeyzXLfDO5NlykImqWIY3fflF0evKLSewaazxevfHY+3\\\\\\\/eg3Bo1ZL58PCoqDSco82VwitR3r0P4uyPubfWKe85mDVnfpoXPFms4DFOGvJc8PUHoztgdv3ig0KzSryhHxmGmRYK0YtI3qYP5QxGrBexmgHGHnsx3ga89d7gW5yAARCS4mT4UjTcR+\\\\\\\/9mHIWLNC9APFj+K7NodQnDlwK2Wb6M\\\\\\\/kR44U2Vw6EoCmTW+5hUAR0S2aj15BwLB2+3v\\\\\\\/2xBV3StOomTsCwdmAP2LIWhkT89Qo8lJKNuh6g0JWgn5vEavVDATvVilHfCnq37fWWgdlpvk5CSh1q5\\\\\\\/KjYfumNH0ttwRic8UTSXJLIQ36gU49U5aMsCScDwN0d0VTFGT8Dxb64uHzuyyLdH59hiopDINq07at\\\\\\\/R8UDakqY5rgYaKBFNytl0Ydq2wgyXJPT0TCUAEULfg0xaeehMlpsdgjX7yz0\\\\\\\/FLeH07xGQdF2BAUBdJ8NFlDXqTre++Jq64SpQZ3DIgvxm3PVop7RhUDETNfslkRUds3MCrB1PQ5nmv2A6a9D\\\\\\\/huHSdYMGskjb60gHzcqU6xSu8Q6dUivJujcJ8j4XRpbj6XPEvtzg0lzSNi9O9EQ6l9mWYvJ4C6EEKRy\\\\\\\/+R2WmKtmEl31QINxbQ8nvs1I5VOrRf1GZLBuATOax7DPUW4ki92OyeHIf\\\\\\\/t+3ryd8PDs4mQCpWQtxhVowqC9S7D0\\\\\\\/kzFovMu2iEuwjrqPl7UEXPIinpFMGW8Mdnep2lE+fPmoBl4l1myfgIX5JIe0PcgrebdElAAIAom67beRmN\\\\\\\/k92jR8PfDuEXU3dbq3SI5vTcCNUf6T86jrYcH6p8vHaQWdYNKeaNTGImeWQrjiG4dg4sbxadeIvv9wSAAJlCOI91OAIA8xOkb1bSrusUmAWf9b2Vvd4NKGyWO8L4BLoMNelKf2dzTl7Snp9MFif1Gog2AEjNRu4pwQacgGpOa2xtP7RcTMDupmNHhnmeuo0SSxcs08zFYrnFGk3blRIwuyef1gEm3d\\\\\\\/z5Bdy\\\\\\\/cGYgoN6kZnVbr52sbDbHYJuN9yEeOtiSUvKPdZWl7\\\\\\\/2ME6+M3REDGUHe2OWwM7vbAACKUA0ViWhbZg0GwS4tOPBtA8rTXrv5z15Yh03SmVFOIgmyhJUwDalncj564twSLv0vGbu7NzUjjdlEkAf5rzoA03+E00DgLktreOgTfEiA5\\\\\\\/QspJ1OPfVYZgxh3CMz5nw63PAG3e9T8OBJsd+0jSuuIlYCloDKKOTiuHoEnMKylyi6X9q5q25q1TNnV4BmTLjQER7pk8EARuXF0mrbkJuM8AQKkAiYFyf0d1fbbEdUgUcdGVR0iahjt1ixdRpIaE3ZnXJuRvKZIJkJxKPi1Db\\\\\\\/BFZeUy00th++WqTzexb8+88kPiC2XOlZ7U2bNMdUXTdz0WiIwFgGaYeSPRtIm2mFWyCkZn9wcjai1YTg6iYhmpiYlyWD4dJCrHWf47YGZY8kcT79HuDXHUeiFw+fk\\\\\\\/t91IW1nRB+08zIISSQ8Sjyv3uE8mQHOcFXGyrHmXqnJ7xDbLSNXZmtIPx9Sx4qz\\\\\\\/5zB1vlAm0hV4czZzwZXosrQyNTCc5SonTGKhl0JK92kv1gIMC0NpPBL\\\\\\\/j0Joj\\\\\\\/2nVs+Lfe\\\\\\\/5IYAQYPVheczb4GNadqmUwudvSGSRzyZ8BGjrpO1h0rBTh1go0pjaeweAmItQ+9HNnwx1GXzX2q1BBP\\\\\\\/llFYI61fDObq9BgpVCFIvYACO3FAKnhc\\\\\\\/As6\\\\\\\/VoG8QUv6MUwtexVVcsNDs+TzZJjXjwyXI4qTfWk72RGm3SVaVjrPZMs+nPpyPjWUj3TrJINW02333GIGfWpJhh0gdQzSYijHuxy10hbz6lOIQWTF0Nlw4u6w0gv7Snlw8XYUW6oCQsHN6oakI5GNnzOmt8RplIpFs3Pfou3JLIcL0CRtReAZag35k9Yjv6wc9EWshI1zZfzRNOadULpDJR4KjU54S7CZt6V766VI+z55wzuvEhSQnp4K0K52LVU2N3njCeUtpZYvyJEMVsDf0q6M3a+PJPpaoXrWivLcEbcLYMeMGWqIdY2\\\\\\\/7M+tioqUvPW5EAxY+6KnptaGYs16JzdxB0ny5INebmTjBKEc1qBddkBB\\\\\\\/uHa\\\\\\\/87iestJyaO7f05\\\\\\\/2YDhOyrc0+ifj5QF\\\\\\\/LVtyVXfVVm4qToh+WXe6OSlrhlVh5kJi0GRTbxADfjYVhQhB\\\\\\\/0kl7UYH3Xw2V\\\\\\\/MCRY1O36am6Dpq9R6zsKMI\\\\\\\/L6iU\\\\\\\/63vF4BUwqGjcQ8QxpZ\\\\\\\/7qwaxeFDqnN3Ht2ahhCmArYS\\\\\\\/x2PxVJLiGFuhJ8j2UnI8du\\\\\\\/G6m0RRsgTi4fr2hVVcMkuRWy\\\\\\\/mkTjL7AhhuMnQhxGNLW0Y\\\\\\\/6Vmx2SoW9tI9T5irz8cF9qC5of61cpABY85AE91qbPM\\\\\\\/wKX3ctWNm+rD6WEntI4gI8kat6ZFWO0piAK2KvXbxmPp0i9eAczbCs3afGaX2mgJMmZXcAHF590bSrIOR1E4SRMFh+1uVsGWQZbEZkslGVVdZZS0xTNLGbUpf3YBEQXp1PN52RNjFrq1u3K+af8p0NM0QyV0WafCSJPlpkW9VNVKKjtrpZbmdP7DH5DyPaF4PB0Lj64ELhY01bsGP5kP+x0yvsmDIiWPSur0ueWhze4gM31vCtkNVGGQGPTtmFjoAkLsk0tI2etUnsk1clrxx1wpkNXPONfHLYn8iHo3bonN1zoBwwxev4tvhty9CvLUTquy9u+tR6emQFQN3jO0hUa5cdyfsINna8bknz83nLkKorPdbot8+z3EfgHjrpyIw3RJQmt5tWLXE751hANgmLvgHttRlmeCe3ejdZAMlLRnNUxeb0ys8DYsl3LGxhhPZVXhOAGRuwInFZiP2\\\\\\\/fzaPvWRiENacpD4DNt6LCOv5qAO2MRLGKq+xL4qFxh+6M5CpdvoCOENpG07nY2C6S0EykdpaJo1GYWvHbxJzG4Mkg662skJbw5p9nEEemv60zzA6uhvEwcHJ8l1UtZyfQMzX6O7B0n8Zkq5dc3JlgIzJvoiFwJRHK1qP3znv4e4\\\\\\\/eaxlHu4G5kRQ1BQKSoMTrS\\\\\\\/tUOa4bH2DR59PAczZnMmExq8Q\\\\\\\/ONhTDTIHadpxBe0HV0LCqMVPW5IM5MCq7PyLcs9L6gVedJU+uQ1eluZL\\\\\\\/RH5zRMIC6NWdzOVOfTmjOa5gp5bAM48eZkZyLjTlnzASiI+GE9s\\\\\\\/5g73TlhyGNwsdjqN\\\\\\\/tpp81sYJQZRJ5wJY+YgM72KwmAkybfAWVuZHzqnO0NsBR6IVLoxZI52muXw1Idr+QtisdHTjfx+pwEmJLil\\\\\\\/1kZWG112UKgrDQCcWAKuzmz8w9akkbv3Swn8JxUbGmTIWhKMGX1Vb3G0x1eeU7R3U42TwbdujPG5dnPjm+7UMhF6cxdh4yErLLqQ7+m+hTezojCOdUmOi8xkEjXPghGdRDmWXcabRDvmXOgjmC5ClF9TS0vpW7uonjDnXk+rduhILAvA8R\\\\\\\/Y51VUJ9Mzf21LsC2k8EqyG6c0wWJTQG6zqYOfVe8mylwHqttCQJFN\\\\\\\/Ah99tutlvyDgABSrMJmbMP7zgxeivnfvxRglcRq3j4fYIbyECepRUIYdasjG+qvgwb33ReqUEdPoO6zMFkMDfN0q5zQLBbFfZQ3N\\\\\\\/phNdQRJd9hy+wWyAbvNV8\\\\\\\/x6pk+SwwvaSObEyga7PfsgT3848GfBg3TtJZE3wkJCpUe66takyOrpyZSJ6RbOuuIw\\\\\\\/pbwERbmhwlP6zjUjCGUnkyqQUw6AbcefRdXTgTYDaBgFIgPR2MY1BwzBCkGEHfxu5vVDxw5u8r\\\\\\\/EzYXad8ZKE9s\\\\\\\/rMm1KbyiAPYcLf7ULuOQkzWCugxP\\\\\\\/szAtsbqXMwCicm9WUsPLZtitLn8fL7uIwRbtYFQXdLK2cKMuRhEGPENec5Nbk3kXgPJvXQ7X2DYdbnMluBSIJM4j\\\\\\\/urx+SMPZ8LKCKcdsqSTTqLFvS3\\\\\\\/HIp7d7qGmJeyGybkGIsl8ea3oIW3V1qax4JdrgtSOwbgPhSC2Zt4rs0z1ljO7a6KGkIP1+mWc4K29E0m35a9aPI4rCicrJ\\\\\\\/rb1PD6Qxj4hxLZaDSaSToIawIULHoEbjJqTQG5NELDUaXp\\\\\\\/ZEHvjDG6EXzJx3VdZDxaqqXDtfatobZcyvLsbLL976Wls+D+4XYTSJdt+9jrsyQbKMZVO6DcsxblueUu3gb96DIqyYRPt5sklYA\\\\\\\/6+1X2bNTuBcYFuSfY0jHbTN8tX3lDLPLFLvIPk9TnKjlpYR7yBkzUls1Pll8KomjGT4rulrr4kOItjo2d52YQTZTSfRwU\\\\\\\/I5r0I6UQoUmDp8NMnvdwEtgGxPp\\\\\\\/5GhrizltAjKf+1oQ\\\\\\\/XtlCmetNfP07VjxTZ9SRbIDArCHBsWlxIMmu\\\\\\\/vXxLSatgNY6Ebx8D3vG4QF\\\\\\\/t7IicDZ9LcwI+ScSi9+tFnjSLAm0eEmTV0b3lcoJbUIJBiTFbtuizwIXHXwSlgPjvuL\\\\\\\/1Rv7QhvhCBgYoQ0p7VFbu1df1dlnYz2mO4GgD1qJwI+9lmq7VKUckmrVm75ziclqOtC+gEk18cfb5FyBQthrp+u+310wJ3JVHbxSf64Ek2NCHWMoZwE93uU3R+DDgYiIFIgwnSOM8evI2gOBhHTtqQSxgpyuQNApX9Kshqz2XvkCQgwE4Cs8LV+dToBd3BiZ+Drbyrpn0q7Ac0bMmuOKfrxVxCvRKBiwbJqYJTw+7eCylvK8lH2gS\\\\\\\/qvJ36xF7hoOzQPYMbDjvYZgslq7GZ86XZCdqS9Tm2FVVdGF0jxknYf\\\\\\\/jCawbcvQs0MxqQzA5D\\\\\\\/7kqk3BQTSydiuNram8jVwZ\\\\\\\/3LzthrVxTvg4oPOTC1P7AZ9vyNowyc7S1mlvXsxWtu4G294QdfTRKg9zZ+X\\\\\\\/9wlcz9w8AoNR9laMNs59S4OYEV9d7gHmKjW79gD7+Z1lRaoEhwBWW6pLuj06rbOGFg2u03Y1w6qIcpvgCS+Zlt3gjZ+skSaAcHWrHWAzmf9w20n8slEiZMy2rwmPogKA1ogmvGGdKHbywJ9a7xwaND750JjTnig6KfjN4uWaZAVThFKZfafEe2EeffB3ZGqyjyV8EwW0yW\\\\\\\/XreTgXlxYX+2DVfIU7B6P+bd\\\\\\\/bST+\\\\\\\/Z7o5K\\\\\\\/4DfEUEICSPnwc+ZUcezB0J8swZasLF\\\\\\\/OsMF+UIWa3xRiurXs4QYiPc6Q4USdsTeWVQH+ev1C0f9\\\\\\\/yfMqLYaEt0pRqMRatJKSSZpmLy2BJPPtYu7Tw4XElNOznFbMZlvwJU8DElnsu6GAk3uIEpNZ36qwtHMy4MRng2TQfQ9N\\\\\\\/HZvvpAb2jVLwLcLzSgszrMaLBIYrSFCesaFbvP2LOVBpNPIJwH3AxjkKWyhprjDtJ5p+JrEBxvCH8PnInQle6RiwAf+c8KyLXj1hiM+C72CeMo529MMjzT1aLUkyj\\\\\\\/wJqYFT8aMkc2X6We5ldWioZxNeBY\\\\\\\/+et3ONaY4zzB0eDnqsraoZnazsj7npObAhau71a9Ox+vevHLsyANjViw1Wss3nllC7SYb9MmdQz7PsIYfJndpZFVbOlTuVe4ykii9bFBBGmZ6IAN3bRS6bGtSVwKDRjLhDHNQYB7SbHOuXRb4LIgzAsjVbHwtKIM6GoEHrkvtVkR0\\\\\\\/QqeOTq43sVxz8l79bmjlK4iLl2zZQqOs86GC\\\\\\\/f4LRH02Ms13sU5naImDh8ifAeLxDUjjeV7hDqYhsa2Hawl2vsE4km4G7qkcGuo0T5iPUuzxYzhpqHYR8J1OkbcrjSj1ZE6MyuUYnmSsG7nSQEwRy+1vJEdVTzsoCcZ2rP1P+ymZBcd5JKk1I2hOK2fpyfhPECdJNlzLTsHRxIXUE03DS5fNRqgHnVoZHibNFAzze5INy0QgehT0Wklg4nIaUOnf0VnhNrOiHif3wU9vw1Ivn4CDm5RAYSCTW6NC38UF8N2bm\\\\\\\/9TsD8pLlVx6oWfaWEpwJqO3uSZzFEKEZNvsjw7DYujifEqTmm9kNAv9OHwI9fhBkJU3wM4wXzKg2BPMh6d7KeLOId78Am99s2B7wqHuCF0T\\\\\\\/MHhOitLbDZsIHhdLv3wN52IF8PyBZIBZLxshPgLkyesxYCT6yLLsXuvVEa4iNieRPQkUpFDUXp1XB762HCMRo5Wn633aqLMXZsQovvgPQMVSSNPBmu4d\\\\\\\/XXHqP3y2OSTwijJ8QsnwLiXw44Qkh3rszblYDr+lwAWEbyLIhli0C0o11rC1qP5PhM8iqXLJ54kYfoo\\\\\\\/v\\\\\\\/+pH994bT6jQB5\\\\\\\/z4xYC5e3l4ShfmbaV0php6q1gAUD4KV4SlZ9iH6P5BsMYb3t3TK10uFf+TbDPOKt+qF9qdaRNr1rptM3CC3Ahqo5ImuLsMC8q7jI1KBa0u2hGdTzLvwJl9Y1aOoOKeGYRA59Wnf7TsTDtdE9bvcX2vcC21cEMLotMgfCCsawovV6yyMDcxhGO3gkhDKqeBwZTKHgY4aKp+BiTa9oBwjg9zoGGsizL6QNBJhyj1tRGeMDK5KBfeanmlZicwW3bADwWorz79CHB+3nt4ZPdfPkNyqvPXvX8cIYRL9bqt1JvP+gQpPAKHfbPuN8+Z0OOwkvToDLg9h91gs3xhbwjmbNgHdIfpTUmk30wVkqGqEjkvAYvEH1kstpf4yhLezkB59679xKvdyRNmcAAA5NmvXitA16LVC7FzmQHO5Hd07K0VSPuNCU4YjmQz6VXIzIoET6Uk7aG5e51pXpCLYa2DKARBe\\\\\\\/FEdlUfGHnEmACeRVqLbT8JK7cXrxWcYgulvYF\\\\\\\/m2\\\\\\\/gdg8hMMQEXT6KJz5kwzjTDIsT9Jfz1gWbVajUSJDv4nhbGcYFnAWMmNqCBUxY7tbrK8uiGMJEdox7m9q+\\\\\\\/U8vm7Ulg2zQvN0tv7Ph4FTbFMwWCwKGMCemmiu67YplYvAgn4vFl7xQgCxddUQkV3RTS0czmTKK8gm0HgxnALpac+A0xL544r8K8weSbm1BlBkGNFAjPN2zAiV1b8dgDqH57eJ+84p1HsJ+tSi6JQczO1MpJvb9X\\\\\\\/193gvO7tAhD5QcThmmavBIxV1KdPIGffCYM5MJWx5gDmoYO4NqxFqQ\\\\\\\/VCS79QkF9HMq0tNSoovmNBTKiWaDgsqLwPx02xW9SoNndkowN\\\\\\\/1pMo5AqGifcc5cT0oc9rtlVQHyjl+9UwcLBlrZ6epy77XwDElvIO\\\\\\\/yBJZcnYP1xJ0HwpoaZNlGWkcZmuLdaGkkgtTY7pxIGgGbOihdSYLt92GKkcVNlfDn5Jai9YpGgJKIu5smi3Zcmbbjn0eh5Up2EiWCaEfgHPuqkxmaO+\\\\\\\/pEJEaqBpEEwdn+pk9KYhkwNkASTdg6zLv32evGu6x+7OmQYE+JNJLFNf\\\\\\\/1pIyFEaeZ8zZDvahIDbOzmyapZARsy4vYW18tu2SjbFqD3uhiXYHN7GQJn3SVSeh8EC36QaCI6EKL+vUtk9p3\\\\\\\/EeAAm4i\\\\\\\/HgiC+b3kRcAcNdOxEt59I3PbvdBuiHb8Cp4M2HETA2ZnCXrKrqen9I24I001UcA74u7XCbr7zKi6abvbkJmKklN13jNyZEha8wWLmEnRMnRLkET7\\\\\\\/EyrkmpYelZztnCLeBMMf2fCY9W2dbw9QFn1RGUINEFtHfiELIMmKm1CuqEZ4Yd8sHvVrd7j1fvoIfHW8c4fzTQPKMeZbpHCNbJIFZqGc7+HeI95ggTqYXx1RNnyK0KFYUekDQmSb0Y\\\\\\\/ZqOgMPD\\\\\\\/C9Yqj9vC6u8Y3uYhvni69GbI3kzwjiDEyeSnjMdPvUzMVhPmGHRY1xqmcbjE3GTlz4c7uKGNDYnGzhsi03XgJ8omGiSU3t32FntUQIO2F9VbNv3lzBx\\\\\\\/6fqdE3X\\\\\\\/womlUZGxwhb9\\\\\\\/RK\\\\\\\/Hncyax5hG2bjyrSmwkH9WWnCBNotY7ZzkepmLIRifCh\\\\\\\/+Z7cqTYX1ItKJy+\\\\\\\/gY5A4LKahGzAwz6dcuzoISuYpoJY4exwqnT\\\\\\\/FHbMIga5vLCprTtY6Ov4cxfOs\\\\\\\/ZK0Wayyawrsso4jnfG\\\\\\\/PCOrPiJlJNp1oPToAKv2f8oGCKF4CpDwQL0JOv42\\\\\\\/lsHc8RVrZmUQQwRAzqIDmB+LeyGO6I6+ke\\\\\\\/xDRcVfFfpjxrqgZSf1s+ePvBmktUjDh+ToFMAoaBt0fd4tgUyVo1yrEGxwBWaLeNnjvJ8t0VlCWo+9amLTjr1yCVZLaUij0TNDW3Rdyx9R6xexq805\\\\\\\/j8PJtjBplklChSvpFBtVhEvwwCkQw0EIL5cuYBwOjQHYCXq20SzErH9KQtY\\\\\\\/ZLvFnQegsZrjaRmuB1roej9VtJ+D2u3AD\\\\\\\/MZT+4+vj620FK4pNp2xcVSwtJzbeIkTBEwLHp+LV8oNio5nrcf7b7H0WI8ghR8Tkn6lIzU8LZz+Rg1ozziHBUhCniNq2qiXaDiHQ8Rz1zXkkb17xXim6vD1N5msaXOoPOYQ05kb5grN6xSitH9YVexHTWTq+9JbMk+z9cIVmF1qDVK5aAcfynTtrLuey6vKZVCjXg92pz1kQ2reMtzMJeGmhK2Fld2P\\\\\\\/j\\\\\\\/70UkHB+7Wh2JA87paXMrh+7iC\\\\\\\/jtIME8w\\\\\\\/iX8RQwnciYqUolxCB6Qk0woAdyVJ5+mNjs657ePnFyj+MQfq\\\\\\\/+boajR1oqLgAvrhdj6B1XXNuRFgO9GnYZlhSGE0qFnbFQ+RREaW+o8R57Tfk2cNPy5lEZqfY85jRw14atFw781+4xo0F91atLLuOvMO+KsnkHh3PoqJtJmQE0fQB1741ztrkHDjhdkL3sdyUqbqKH\\\\\\\/FIYyoiczgCoQUQ7k4mv4YlQYKek0KyM4Jrof8H2MSh5Nilv2HK14rdhTTzUfRMGis08fZeJRRZ5i1f69P8KGhpnE1x+WMsh\\\\\\\/oKH7DSHqWAftouE3Oje6w0yL5I7d53PZ7gYRuEHLfs1Dgn+XNfmZJcEx3qSNzMflxS3iyUUVUwuk7AvfS\\\\\\\/l8X6aIivuwV+SY6vg4swwL4RrQdGZ52Omvis4p6uGqvRj84Fe6ze3WKgKWq8zaySWeBhKl9owDpAEMrz+CUnOnL6WJXqtp\\\\\\\/txwPoBqS3G4lWxR77lq0qMBQpav8Qj9dRyfgLsCXsJAXV2Sr4hUQAulqUxb2j9avLYgEj3hIbYSxyGalL9A6NPr7i82b3ukrqsnlphC2YJsXPxviHlBzCROXFQqfgnOiir0l\\\\\\\/adgvOur7cp\\\\\\\/miTBzbDl09A2QYojPyWOIvD4y0YpdPI\\\\\\\/1i+kcJqmDAy+EHiLSQ3ROkv1CItEQD9bwUCOptkVvRL0wWSEjjKEfI0loKNwLyjToD6NAEG\\\\\\\/x4gbN1QD6s18a4MnixN8Ne\\\\\\\/u0FZBdGVLkn0zpTfteMMI4X2ENtPlnnWUJvGCCaQ2iZ6VnnIYKSO10UXVm0uPJt8IsC88W3C5oxVMEVLBdSzqi1HHivCZc5BffMqJr0sNmzHyCJIFHoZ3qx\\\\\\\/n79Gv3lEUkltD3hiT+dSuXfvTCKaWpIA8krPZA2TRN3nhwu1nwnSXr4DHCbpi3DPp2nqJXReOPkgxDEwNvCd1XBq2iO1ZS\\\\\\\/x6xfMnWGJSJYnZuFmz0tEraEr+hTBjoTghKzc0sU6vsvcECbJNA62y3sZZZvnO9q1t0xAnPikfXHRWazS3z+aIkT0GV6on2vAKhJmd9TlUTs5vDbfw73BwdjyPhWAT851D5Ajr9ulHzJTDKOYWM4tHjtBgzkfPgfVwb8DF+b6cEVy\\\\\\\/CLJW9Kxbk47lsdJeTBDXSO0QrJXr57OQ+lQKXTfHZWvuWfNKuq\\\\\\\/8dtlZsiU+hkQekHKmxXXkuMGz6VOHOUrLmaykhieyuS1BWzzUNqkMsjkW8abqX47ygPoNcPuSBwKuSgdemduH3GW755pkt7oeJsU24n+RHel9t5\\\\\\\/MTRYykjSyPLE\\\\\\\/aIeRrJrlcqBG7hfeEG8lz1Kvhr345WrobUQzRTfCeaKrhtjXxEPq1hJY+XyT9SW3ghQ+geWZVUhKxe1HhvQIxGq3fhingbjB3PzF5j90WtLsG0JZhLJ6En5m2wdine8crniLygxPw6Ww1L86pGMpv36HRTOXsMZiDO9awFKYtAe2A0yXbDPMMeCBy7c3l+uiUuNFEn3873ReR9G0OMsLrz3qUdN7xaKdH4zvHlBaVuD0f+0SyzKm24sSF0LkuClkKquWVk6BZ5zQ95uIZZ3lgRTIezOhTH7jz1obn8XbOv2hZAS+CZQwvNd6ha2UyKVZlXYXeHkTzc4zfPTIK+7c0RHZTNuVrWF5ph59HVpydd5CIl6Yo3NMovYRsTCs74jnsQlOA8FzFhzlOP7ugaOeFj12ZrniZTTkC9AHfN4R\\\\\\\/B6EcvvJS2gkjunzg81cFulDqqY0elB+1\\\\\\\/p3\\\\\\\/Le4qKnRr1oHdIMeI+ZhVjtv2FyNbp76SO1IdWoDIpTDNQKfwDzwyBdDiUOEx2NEikZ3DLmaASIuWmegpl+br3jY7wO3i1LTzB263kXIBLILIeVzJ3k65oJWyZiEga+ZpxTDoVSrE\\\\\\\/XT9H2CmdhXvmeNTLK1XEmX5kUhgD7aQgzUfRvqERZTM\\\\\\\/S5AIF\\\\\\\/rIW5TzHai3cWdq4VTK8z2VHm9\\\\\\\/FN6\\\\\\\/PAq+J6ghs6ljR6mYpSm5hbA30dq897CJ8QD5MPyOTA4TGFFvfXYEU5QxAUrewPj9H7+EqUywUp4gUIaJX0HeC9EGiGGB\\\\\\\/2lzkxJN5o860BfZP3YQ3yQOdJ0KmrVul84kNzuxt42vaFj0lWf2N\\\\\\\/+NE0y+zWS2f0qR\\\\\\\/yVM2H4C8XgxLvCcYeefhOPDU58on3rJKLfmhl+kM7QNgZEP71RckH6MUSqBz3J+hVrO7SOgEhUd9PPyit5dzxgBu\\\\\\\/5pkaPeMNv1XLkTYp+PpLuR5IDpvYYsQHrf2sI6wGQvV+jVzC4c1xZAf2haWM5BOT3d3JG8sBon05vYKR8u2Epre8qWKQH9uL3pZ0Zf6wyrTZwxe4Iu6+1Lgtfyln4rccU6qA\\\\\\\/BcnjSwaxYb\\\\\\\/Usl0jXdf1E+iqgjdhwE\\\\\\\/Cf+yFHtv4oyqKaqlq5I1Z3hNvshFUjxpJ9+lIecx616OduO4ttJLSTe2rdBZaZd+ouEq1cp\\\\\\\/LHznwscsMiQrDmFvwUGfQHP2DW\\\\\\\/PYmFLcQvxiyl6NPZtzUeQj2InDFJZ8kbkHesJnfMcoxa6QGuB2tosyYIdoqAxHKkDHUC4bItRkqjY2ZMcQeo3qXAjMAZ7h4xVg7YUZ0nIiIqdl\\\\\\\/XBL+NFlQA2UaAfG5dF\\\\\\\/L1i3osnvXWtY6fVdeL7meTeK+rR2BOp6RjSOIBCSGQ8aE9UZB\\\\\\\/i4uWMf3TV5AGCzqUB+8pP1mflmgiB0BDCfXLO3HB\\\\\\\/fEjSbdzz63fXWmLzNXGHciQp2ljXitlclk6ULlqFJrcJUvODtPrwLI7btLh0\\\\\\\/4jrQfc92PTrt\\\\\\\/XNT5Trv\\\\\\\/oIlb329Ta8Z1nuRawUzbkR4jt7Z7Q8P++5F2\\\\\\\/simsaz5yJHepasKFQgX9HOG9tbdRxNPuvPYNlZ2ZvHUerLIyv1Na5ndHk7u1ZG2PdPEx+ja5qL6xXM1unCLC5zxkml65t4v0PsAc2Rr9oQ43mt6SiSYm4ufa2ZGy64tBTBJswXk1996\\\\\\\/SKQKh+ZUQLvc8u3NVh\\\\\\\/\\\\\\\/SuALhCXiD95SLrUi5EqDUikIlzhfVR+aGKdbZ3TnftY1AERtghMMg7e+nIKLhBufvf8N3MldRX7\\\\\\\/WGGOFelHIZyGQJAZ75oAEFnR3BP25nSF8fysu91pCENPI5jKQA7NDYy6ZkqvTJSr3aAVQ2VMONhoU3n3lCDG8nnbnIeGDtcqvQTo3mGt0KrY8JAwYXuWSI9zKtGni0jCmQOWrKjMcBIwf\\\\\\\/TdP733n1a18QlsLnuXHss0iK\\\\\\\/Ee4DooF7B1GtM4ax8mDYfxNbxP173eO\\\\\\\/+b2Z4gYtYOirV8bq\\\\\\\/JbRlFYyGTOCiOWobUfyQSfkhRX3nXZFkJHDJGAG2Gw+68BF1Y0x\\\\\\\/JcnEaPGJN\\\\\\\/crZBDPtJtSzHRfy58zMDMTGX0QQ0K\\\\\\\/Nu9fwcpTAp2uFqIpN\\\\\\\/mgIm4mAYepb8Dx0mcqg1\\\\\\\/Fgz6SsWpmNIRTmMzFtDePmgT70FZ+3g9ZkMVP\\\\\\\/Vbk+BEWVEGtGun+qxL1dP5Tj0Yb\\\\\\\/gYI7ZsRaMQQUcmaGBP4UsBleL3rUtke07IFWHX+6\\\\\\\/Fh9loSfcDB82vFyYjBuT+VAemosGDN3IBt6UEgW802B+RxRXvbxxDLWGrRdJ06kMkat6pQkkcmmfrFB0ACgjpWdfjXptj4p6NHq0OWb354+P2fR0N9BqG+TEuuXQchhroDfaINhA+WawH07ojIkLrynWDY2JZQUzA1TRFmVwldx71mluXmwKF0\\\\\\\/R1sb2fqBVdV3GkMsoucmB6HI2NXOMqk+GkLFys8mu6Y7eXd+In\\\\\\\/Er9zwrKrFlxMJFuEB7I6ZA28yLFDKm74zTspqCrE5MUWyiAqASWyN+a6XeSSOqLSM0fSGYuJrX2KYTZuyXJgjWxW7vRHBBWO7gvpqYiKQU+AwXtC6\\\\\\\/t\\\\\\\/3YPb4ra2Ubua9jt9T4iuiuFvG5jTEkplBvcGqVX2\\\\\\\/R1ywjtPYa8GrVN7Q+NBK4GtFU+EDfEB+nBpLFyndXBeM3LRm+3XzGctrDrj62vbcJuZA1wtgX5+3WzlVKsMIV233wWaMlUZLsAA04kZvbjSdupq2oyNDp0tkWFuYTuaBpNe1th9f3cOVGVW3E8TvLnjUGj26vsAVbS37oMy7UtF9Ok5vyou30+8Bw37tB2S0vGgmDRUkFo5nImHze0hgdsZdkxKlBAFUfsmyOLaqHXDkSB3EwWDv7nAIBRNw7MU3ZevonbTa4OuSzRonB11my5fq3\\\\\\\/nCl0l3UIfd4O13tbpmr5hiStp6v7zlyxqp4+VGzK+ibCZi4snA19BMbEKXprG209xa8TGnC+BEDUqUU7v1gnkB\\\\\\\/DzyXbN5YRjeUd+PQJ8\\\\\\\/JK3i8vdRAVr5bNPUG41v00rNJo1SX6iHnvGfTGqNvu9QvObIKWvwx8vnvz7kAxB794JmFvUV9T3c0vO8k5kChJk4tKXRTvQKCCLqBNyftwolL6c+QQ0mf+exnbH1Xrfe\\\\\\\/7gE+hTiHb+nDCHFprfryzm8DBowRzvYrG5LgI0qys4c39jwFxnOlSI\\\\\\\/1mIPrIO7PE\\\\\\\/YWffOM3WGA7kqlprpnzZ\\\\\\\/6VJe8oAcIWd1fyWO\\\\\\\/M+HKJ\\\\\\\/F2n28v5cFxes5qeQI7htWWop5usUCH67fk\\\\\\\/jXuBazTx8awCmTg6Tx57cvqFoRQBSt+vtl9YkMdxZ9w65\\\\\\\/0akGcFeQUNPI5x5F9X2pDeHCNnFivlihucvbpBOHyWIv1HJQgRBGwXN5+IfefhKtWfCMbY3ZbVyrVrSO9J33Q\\\\\\\/kVqvssbh+oW96tUv36OyTyWbJGkWLxKPXVkCR\\\\\\\/120QLN2mqUjvjpw77H\\\\\\\/xwRnb9YNQv20XiIYnfrv\\\\\\\/0qxavlMkCKdCgD1MDr1ZXghwTib3bhC77kagZL3\\\\\\\/3NqsXuu5bhbMcYjwRxIvi\\\\\\\/LTJimr6Hgc1OktH4+k3e1p29l3W3uC04t6dnYYK8ugBeZzf3TuGTC7h2oTnrZaJMBi4HmogvbcEJvGkVbDDPPy0MXg8eXtegmD32YPZk9TP4ieKRKkdLOoklRW72G9WklJSmf1w237pajOePQfHuM7llW4RnylzyIike3rAu2UMG25oPkVIT2Yu6ukiSP9wMUVesFUARgfq0LTW6lWEal0IVBPG9utNyZAENHUid\\\\\\\/flJm5hyfDnH6M2LItlR69P75o8AJg6svVGsFtgzEFcfheNMOositxK0AQjnIyZiBgVOYe4tDQcinc8LEt\\\\\\\/ZAC8F3LYk3OpNVYI6OBTgnJZdyoBQ+Ghq6I4InN14RmwlCrAhedFcR3ThNArVvoSFI8ZEesBAAOz6reJjVSiXWzQXyx34Tus0s8kN3XzgxI0HOaVHco91vRfYXkuoOUuBKT3aQ9Z0iiwoGAHIBpYRT72euAY7wOgNTr+pt10GKo9nIl9YsbYNsGlckQPu6YwvbiJ2BXhwZQ8BJSYbCsxJZUf144M\\\\\\\/E2\\\\\\\/zvR90E2ONSpiAPuAGFoWGeILQ2hfwdRSJMlLFCERVAeOSyBstBfJGtnVhz4hFidquXBXZQJEvwBgi+JJQ5IbR7SC33ztJdQlxyOnR2yxBrfcnjQGEbQNGF5iyC2Lg4ngijk69py8k7dg66zF2Y+pxKe+R+BRNQRlfSdYLw1yCrazPSjOMmw57BndvSNzu734v+Jz0inE\\\\\\\/soBKwan7daD+VGZZahhLWqEuA7E0jGoonmvZOtKBkliILabFym65WNDAeBfhwrkVLzYDIOXZFzpM3jz\\\\\\\/GZ\\\\\\\/LMQ8j+XBRCSJ\\\\\\\/8XGJBauRZscyUD2CH2p5rYwI2Pz9lR6MyGzCSEk0T8EbH3+vZfA+OizLzmfzbXgEEG1M2DM\\\\\\\/W5Ni1nbflIPZnEcEPZUBOkB+5qvyE0LYX9z3oCQxcJqWWdVd9Qb4394UgNGek\\\\\\\/vdy8MMsCG8pSkw3hko9mrlutzf1iFIYJw6Y5G7gdHmSxiqjk2lZ18r\\\\\\\/S04rkfSD0g2x14cakYZtuNRDGMZ3zDZpdj+m3SFX4zdiFUxlxCC7kBcNt4u9lvTAE+sF5hQrM0n6i67z35YzGsQdUBvnenleupYeUt8WR3v9YjR31q+LGzNyAJL9AWKIIdaoyt25gvKKHXbsQyow1PF58iG5uNuwtj3DBzHLivVFk1w+mwFruXuAFUOnzzhoCbcr+nLTpN4HtUHyNIYONHUgWUH7yQOfZyL4xWZQmZ3MleXe3otgNB1SgqmgvdTs+xszIu0cMZzZ0QyJN9LEOX9fHQcO9PKOPnd9OxzGmgMaryG1BFlY79OEnVyXWm802GPKzh6aiveAp7suplNUbRNG+YGV9KvqpLdAt8DrqX4DzH4aOHumwN9jWqbT1f9pwlHeyn8NYhUbcKRTlaS7prdDH32ikjgBozn5OeLqnrUQmUBsbwVeJFTemu3s2nCxfSCcTRL\\\\\\\/v3PUmFn\\\\\\\/GmnMOjoh1WR3E312Mej6aftgEtZaFqThjsFHn\\\\\\\/3ndQtpIf0UFTrVN5oGeTTtihmmzC\\\\\\\/AAYW9tTCy8mkBIK+HW4EAI9sJK8BpC7R10K6T9XfPeBvOoXsGikKhXnfTyGZ7CWIAuKBGFMSYO1y5bZHzMoQNHh73ppYyhEmesjCTZ4v6mULlsuniw2J2+V17Y5y+0o0Ocwn57+EFAnj6IHu6lkU5lAfJPRWBPGkz6enuq8fSHZF+AEQxY2w4SMMkj2sP5xK7UfVWCZk7EAJtGQLM3XxTpma9OWiEo6TIdO1o88HbPRcEFW9YatLFXvtvFgiPtNklivaJdD2QHrEANK4VlRXcgUvuTL1Rc4jOCw6HzDjlXKgK1hhFRrs6ujwS66fxNL37xF4iNibJy3DeNajhOnzn\\\\\\\/XanWT0Y1IvvK2\\\\\\\/9fDHCoR1HVcFVda7pZfnMEGRK0W4QWlz3emDIKNeSEjvXW2Olpm7CU5QwB09+yZ4xRhkhSZmZKEtM8aUeV3WE1n85iBVgYhr\\\\\\\/Mg6dqy9bnKpUSWx2XhiTwsa2nQlGD0DU2nnQ7DSaGLpZ+3Ooz7WilzIAVTRzMsV+8fuuSq8f0hUOHv6hX\\\\\\\/UngGxcn0DAqbiVfCK2IfbZBl7tucJr6b8ZmrpwGSOhhyCYIox0VW2Pm7mrVHgqqiHbB8xO9LEtZoByibTEA6gn1LwDBDO+zn3DT7otz8uPRtZKvdXAf2trSoARE4OTL8NRP9pavZ1IWWTvocKF9IcCH36k65Ff1xyOxGxIScdprHqKq8ncOx80hq4kHIcWlj\\\\\\\/LYY7bGW\\\\\\\/33AH0uZ1SojlDaH6sca7HTRL2+mMVGP6r8Dxo3dbDoAoZ0QMTXl1Rf9vR8+kZ6r3CyGUNfmedqJviVZPmbZjQenEKMWnkORUQSIaCn8ubg4Dxqfx8iOuVOoTXVLsQZKSbKGL3SKX09XPgvtDvaDDdqc5LlAKNUe6Yvw6JBSrIusYdr+h4n5ZZBvLYaCjN89IJfefkqGGmy9+mSntCwj+M89R16I7T0ToYn6B6bg\\\\\\\/HP3ghqUgHYQ3hQd3mGAP\\\\\\\/P7zCF4iKCbqsmhj\\\\\\\/WLOg6ROo\\\\\\\/aT6508hk9+Ei8E4Ty69QnAj1dGLVXanHcNnImioeUMOhvU1cJPlYA3NL6WHO83SyhKavKsP15jP6wJybkFiryMupoymhWbhjQgGoxbiSayDJGf0PtS6USKkZx0HTwaXnZ0YX6hSsIqmcBbq9wKP39Vx+pWv8VdxfmAWtxwjYHLxwz47wIQMRakrqZOJAWdqbyG2mRiRqTG0kQQf8D5WsVLpdLOM7BG06+jYvfzSOoT8a5MgThEtiVVZWj15+xr+389ZWHd61L6TTGHsbd+cV5mFkZ0C2l4ufNpy+gxNYg9b0Fy2\\\\\\\/6JVjQK1HUdyefhYwO+qJACYvtkQS8ucGer\\\\\\\/IgkhJ7vti3h0HuI4xLSyPlXnUgKo1+bgBsxZ3X2ObDwv0pa\\\\\\\/4oMCSEFjRV9yBJSxH5F4+r+b7DsTnf60WHSRL5zTYPNQMDjkI6kZ3arvYp61izDujRf9JNdLOgsH4wCJAz7PS98Iocdhy7hFsBg1Zi+OJnT1zYDmrxMkwBlo0jqtDuPcoi+8eal8yz5WkpHm\\\\\\\/loYUEkB9Ldtt4qrfmsMI6F\\\\\\\/WDIBT1WLzpOlKIgP0TkhfZVwVIfFhAi2tKAdD3Nj8Q3uMJRz642OYmZzdsaovPpRp447\\\\\\\/mU8lz849IJSCEtL5Ak1kcF\\\\\\\/8QeaYlj\\\\\\\/Suj8LKBMUCuVpQI1WyzdQgyv6Zk+U9xGjkKRc7UhKU14BpwD5t7f6VVmj7DcF3DhLilLuFUjp8oLczYILXDToEfK\\\\\\\/nF6mfe8g+XcLlbO\\\\\\\/VlaS0KTJ6ewZg5Knab5sgRm7PwOxMC5Lx6JEW320Pfk+8W6YeHcvdpXwRubx6UG+UXfuweXoPdb+QPJyy0K9mGZQqI9jZffRB9OxewJEkgxl5Kl46OFAp0NC02GnkC9tnfEvbZ3AdxVTA2hXu1qNN8kwdhPrRqztcowy4H2eQNCdVGUpxKBfDUYJhwsnld9++Dy6dAiNNfoFLv8o1S4TtqCzem1WLVjR\\\\\\\/vdOh9DLB4BRXldkSsY1vhrOg0qtRwnwACx\\\\\\\/ptumpM4CGEV63IGaC9qjgg9K02RLWvlpU4WeFFTNscmVJJj5J1mI6zwJ\\\\\\\/\\\\\\\/cY3NBPrUEmx5t1q8idGIJJrDEy+th4NH2dY+WoZgx9DpxUBQxsQ8useDoDYbj3MIpkQKrPpTLNemmhsbNdHfjeyKpdiMI9OU7z1C\\\\\\\/NSlVc74N3bMZ7X+AaNQVcBz8vlGK4CwTIQNg2yve\\\\\\\/k2Z7wQZFNIvSYdq4eMgEWzJsyKMJGK2fy2AAkqH3zoUQT7GhfYCQy8JGmXXO2Wq06oJq5KVSB1Rb\\\\\\\/hUXl9mac38qQgC9BEXeujzCME\\\\\\\/6ziGhFjt9PEqNfcmKo+UcC8YIGVpRWwt8G6uwFxsIAatJcDno9H5j+sn+3KF01LwuOYLSwjvkG0Fr99dTtm4YY6FR+7T6Hyj8UgYGWx1kL3lbGXO8ZG1tXxgrg6LMRCsD1OZRgnmaYAoMG7TIchjWuzATIi5zyaf6tXt+2QXjF06Pyk76Eq3omuCA1QbcSE2PJyzvOnBxVhzFpUWsrrMu+BesPe0IWEOwOgt5sJP7HwIXWCEhXXsMS10jyTRz0y3vf7uNrXjGcWiSks9NSR1afcOv3q755GNqdfVpMlOUQj0XhvH9YimwlNFeacAmGtqUToO6OWLb4FZO+SIbMWC+\\\\\\\/EBma4llGVnfVcIo6cvZ5yD2RFbPgnet7oH3Lz9Bbg0CVxoET4hip77LxWjHVNZ\\\\\\\/8abhDJADVNEyGkldtemmmSk\\\\\\\/zrgVWkAy2+gcXqHMus8MqrnqOzhJeVBaFPlgGvzYeam0QmaIHNUJ2SDmNol2OoZ\\\\\\\/MpoHGnFxiHvDyWr18tEfJHfroGGIq9\\\\\\\/w+8f1Ye3FN7VcsQSHmVeGcgdPHeB4OObPODg17U+wJGgg7YATAdIr9E1LqvyJoQc6BKAg3eSdi9cEeLuTTUMBWjGiMSY2ApiNJ0XSJ5y9q1N5\\\\\\\/5GSfrSohH8RGlc4giB7mVfdNGODr7TOAd59yyEFoIg3wx8Sg5L4ZfebDHVtYJribUY\\\\\\\/bBIlXRFtOG+Ss5FGj8Jt9ObqKWM30m8UPcwDB6xqX700+KHrhoDPi1ndpGvqGLnZhuzOKqEdCMNsKdsTAza9423LI2FoJ7b9dpDLMfcWGW2hJ4UiJW\\\\\\\/AX\\\\\\\/terSgaqy+VwSC1fq1sG0vSzFj3z81JXuS6r4OPAFdUEZc3LASPSvd2Ez3eJlGRMf4AMWyxJkcxHZ+tRMGsNegPSNmOhg6qcnaTf\\\\\\\/uvMGCo4GaaaLt6pMeVW4jFkO73B7ESR3d2Tz6GroFFQN3nRLnp0M+P2bAydqr4LNifxD0X\\\\\\\/YcnNQPOuYHKNzSZ9kJ0yyPHRABatpzQJLt1rAOmoFPVcnMaXNLjPHunWUjUxvX4SQOsjV1baivGA6PRurfGE2yV0n22tYcMAwIu9bbBy7KnCeYCPiyod1+6UObkNDYtaPrgsvb23SuiyDzjrjSInBm+uJnRu66sHYSckKB1GYTK0PLrnldMu9Q+ma8iQj0ZdGE9oRb7oaTM53EPLNrZSJzPxq2W+APZyKQJqlwItHVP3qsfX8BfNJ+Gb59Ux1y\\\\\\\/7IDBzgOX9DrTfZuB5i43OJTQg98Gh69RHV8h9lMW7a7Kvdc0iD7Y\\\\\\\/chAi2hMt6sYPvOUCYGBl8wXwIylPdZHzHiwPUwowKYH\\\\\\\/qyjSizrk1V92uW0uz2XY2daVYnhJ8Ke2MQ+XNR7sI6c6HuedxdQeH9pXIcfsHepAo0DxEWJcR\\\\\\\/DUZxbssMxXTuYQCsoiZK8yeX7Ir\\\\\\\/s8k2vLVgydQdCvZyaw+63nUSvTXIIvfO8BEv4cvMi5+cVEyJn1\\\\\\\/5b+Rsf3rwHM\\\\\\\/rd08DHn3yPzIhPeJilwj2O3fdCphBKHGxwaJFAApPnlRUA3JDP2hpRcX5cs0ggYMXejf9jNqlidErFffC9YnolG3Ea54GAGd71CvB2Rwsfecn+iZ5M2VlYP53\\\\\\\/6SlszbcyE7ETn5jhESiQMBCn0iiUtVsBIyMujhyr\\\\\\\/2dAIwGx0VmsJlwM3cL06fbc8qolABgurFYdtL3t9chYQnU6umtqtpUI3F+McnmJGUr31iCf9jlJvOkH2mVn3W93T0Kson+u4W7qwaAX5GYbdmngKwRt+Kv6A9OyYPcEpnqD7nfWfu76rN6nsltm1u2DUuegwubyAp8lIk3cRsTJdhyntpgkaIWpfKDfF1+qMKsHZs48pjRAsN19SiCzzLUhprBnEPQ6PxXaJgkVfqEuJTRlHSlL06ljz+zzPHqqP4rUF90J40C2wPJgRjespY+sJ9\\\\\\\/+CZPL6e04PDvOkl23ldKWQwt8ui8ZAp71nT0+MnzlnBBTcY7yAaChkZw83nUDQwOP2y5wibhMJxrVnn7qL5L58qfKybvTVJ1hEF8JfELNmqsSpwkPJr0rgv0F3ef3MqyhcHrcDkeXosEAXfpI0WC7ofnFDczDPA4ohYbLHxAW2MayVK7wsC3Yehx66QFvsWpCLqsoG22bw0sYHS9+usdZEeQx7OAhDdNgItTBh1E75E5SNU\\\\\\\/90Oq6KhKIy2pKCoPZbWwMJniauSJOH9gvmmwAwZWADi1mSk0FFc9GVVgpJw7Yp2scCbEuwhe1vKa8Aln3HEKOIgh80+2DtjBXK\\\\\\\/f2b+WpjvaIP\\\\\\\/0vc2uwFZDQc\\\\\\\/gJ0wUF9ZN7PzBwdObfBubpiaDglLJ1Dih2YdhZHq3kSsd3GqSbXMn0q+Jh9pu4sZo8CEo+bXiSQYM4BkHims1x50WvpSU2AMsm0ea9euFig1tkSU1Wiml+QjP+hu01AxKeQnlNERj\\\\\\\/jm6LMkcLKBC28qISdL\\\\\\\/aCruNrClSBX8k+AAvstFU1gBKLRIk8xICDf5lk1b5TAj59nnpiH4bWy9YwL4JPRaW8rkxC5iDMbAuQGUP0K03\\\\\\\/R95l\\\\\\\/o\\\\\\\/bEsOToIKOSuLa0imXCAqNt3gIPZ+E24IhhjphREbUyAVUoSeGil4geK2fVDC+JzwUn\\\\\\\/q1kUdCEyv16l33ZKIZ8B1AjB5nxOt88CP2TkMUKtkUrM2YBhxCSXmHklPowCW70Yt9vJQuV+ASexlS33r3cwB7SFWqw8cvHxpveTVbG6AbQqlXN4VyIuWet1MgT65CBpDLRnulhCm7FcTwJEhoUdW6EgGGY5HkjO+pV4Ula8KFkj7wUhMDIi+lkL2lc2L3eeVEieoIrBi5eYIK8SYdksBva9CZ4Jef4i0u6awdMU9xQfMptFHsBrUdUeFTYHIdVAdhf\\\\\\\/CpOyftTA\\\\\\\/zcLzSzHCuGJ4fzOzbns79DGcqPit5bZqAp0o9Nl6i\\\\\\\/rQc35qkQ04v5qio562jwcn6YW6d+x\\\\\\\/Ik79wRaHmDXC0Zxwhw5jjTH8jVxTIJFG22rTreT8n3athRlCgYiVKOUYcVzWpYEfynYMKb7R3sZ7SDn5UAwrHSi0mnadqtgmyVPFA\\\\\\\/sWqjwyXb33l5+07j4NxRYwotPsI1Xhe\\\\\\\/mSXC\\\\\\\/Vg7Jyt2KDafExz2z5mRC6hR\\\\\\\/vW6hQHSEggByu4EDkNxhzghccjevmN8DvZNl601jPSvuy4JoFpWVQORIC5TCBpbUrRgwgXoQ\\\\\\\/4TIki4Al7DGAUCTNCoFdvBvemquFQmNiSSE8qk4RvU87IkG1ZVlKrJpAQNRNjEc83CrLvYzocFFc4nC2dS5941ShFpQOXnIC9ihiK8cOgommOPLG8xMSalhCdAk89tpHfzHfd7vyEjaBFA0wyvbgzT09pusCUzHx+lkCAjypvgGpekTlMO+RGMhnTIJdDF1yyXWSRxWZ5ZUhFwDIHQ\\\\\\\/8OgFr13fXbHTdIqR7uL3ZmUI3iwqgxEHK9hqm1HgbBn0gKhTCNRZVZNEIOCrVxKSTiCBJfToWHFMdFPr7J093bdbaJw7Z6wB460ofShaAaaB6MIYz9Pq1WgyvGxuTK8lNQo6DfXr4lFDe9HzlYKx8wP4WMnxQ2L3fPByY9oVGwdAlYUUyi2XcBfGp5HTP\\\\\\\/yIGSgDV0RtrSx+KJi+SkivbJ1KZHSkivoC\\\\\\\/wjsUJFK0DhvL\\\\\\\/kt8bCZp18DRlrCV5VxToNSt1GetmPppFoFXnNc8CdIfYuX+pdoT92hGj7b6QmivpYYxNgnz4w1x8IF\\\\\\\/r7it8Oh5X+YY3WQXEhF3lOc3DgVvGkrAk1NARZT6EB61ZB3UlcN7HS9VdIwVuPJdiI59eQDuAnw6Lmk2+gyrs4ZwsS\\\\\\\/cF\\\\\\\/DmOTVQTJ\\\\\\\/JirObEYo1fP5rFbXajWdDwa\\\\\\\/QPw4oPxX2uNjhpBPNoCymGFd9mfhYZeIUIg76yW4GHBjoegUFeMZIzTwv0xiyst4H1yWnJyV3ZRfdBC2VeIJT4Sy0uu2tlb09GFsnpHK2LjTJbvstwfDNsyyL\\\\\\\/vAVt32YvloIQ4yzR5j2g1BRVkc7SeTM8q8P82\\\\\\\/ojjQRt\\\\\\\/B9Rvhx7k2srJyJUxZ\\\\\\\/nWD2k8c4o0tUgm2pWwdypUz51MRw2OaPr8Ms8wP7T+HHhbObV++1M7HD+XYBfgvWzBOxj1WdZ22yLfENI6j\\\\\\\/+DFDU6Z2j8neDE\\\\\\\/TVURZMBPMaIElUclu5V7LmC+mQz\\\\\\\/7cG7MX2TOW\\\\\\\/uSt3m78mzJ1hdtStVmQWYk4SsTYwZ3oBL0AhDOEa+m+HbOC3DdTrJbU06emlJcR3\\\\\\\/2dKuSqiYZjY5YjHZT7fscYUtaIsxYDUFgWV2f1SJT15P9UBGEOM3D\\\\\\\/IGQIolHuJ9Sq8WpffeiP7uYrkus91zYQNnolT8aK1vFgYwkw2ore6OsA\\\\\\\/t7xg+\\\\\\\/yq1cRuHQVplLLOLeMJetfO+No4YI5hten6\\\\\\\/DVHMvIu+vBlIEVXOlPvPRXTdb3uX8vqb1kWF\\\\\\\/evgphqbIfePPNjcWDSVHRKACDc0Qc1yGPWpkjLNARREbhzhEPRc6Q8KsaDxf0XhY8xQVeV9B3ap3mLCg1AV7ByrqSwYBlgJGhzV8S6tuy5VvUQOqeolKSmyOz5QeFu9uJZ41KCgpO2BowPyMx9Nnw9a5GjQCUxWXLHUPIyJH2hOxHM1ZbtSrcl9UHsg8yrRcBTNJq\\\\\\\/MPX7Ei7Fohj8DrFRHJg8ajE4e5GTiNfTJz6DDaLu9ONVBBblGNEnzy55J9fRXFzkT\\\\\\\/u61+\\\\\\\/WU0rD4JnM3T+xq5glrSqEeuNf4lR8\\\\\\\/BFEIRSbpgfmV0AU+FlafNnnkyXyyTjFm48g5GAgfQNkdsqxmCxg7AtLiIYsAtxIf0xXR6yYWorp5JkH335XmXY6h+Llj6w2dgq5qDdy7IV6buYBXSJluwcf0GFR0JBEub2Hj+qTgiYSmWuvdYjr\\\\\\\/b1aHhZCrTq\\\\\\\/+gz\\\\\\\/2JGqj\\\\\\\/VB8ATFjt50QJMXibsSA2e7a7PFLB4wvVgnG\\\\\\\/dyj32E0AqH2rOx\\\\\\\/eg6JUsubeq8WV9h4d8YCc+HS7pbAXWKGlQJYHKX7XIQOjYHuNfA21MGkTGb8lCN1aUrscFDm+bvlkqz1R3RzUC59lPey9in8q3EOiMF3ppQVkUHPskHEOwf7D3zspP+FTlnz2\\\\\\\/UdN3b2O2Gi+\\\\\\\/5zq\\\\\\\/jDbo+AHICt\\\\\\\/SiK4hzMyl3gC5jsyReJyHALPmMwX\\\\\\\/GfboYg8JHpCWdxiEm4Xrr7JEgdig9dszpWW323i7BvHjgSCYsq0yc9W7kds0u+Z4WdnwevKulIcjXF9zfG6FT4hebf\\\\\\\/HSudE5oU0oOfoFA9MHhMVF2zDqokeOQVeW9bZ+E9NgPP58khVuF2cgZ1Orftey9lczmG9kyhgKgY1LLJevfsusbUpuB5LkEE6ktpZ7bBRkLYqRbVhbLZezmJjxwk+6HL1lGg2kH43jktK54mNdhq1B8qtDwjlFHW7Sq27autCEIKdtq6Rxi4xqahlbfpK5xQdtZVJ106+vdtXKG\\\\\\\/Xrk77q9WnvsDabv4ekvFT3ZLJyO\\\\\\\/ENo855qZlCR0Xly4O44bff4YgrX7lRhBxEFBRHJHsrpO9xBMOfjYn7cDnVXfp98xNakszyRrQ06XLENje2fTRvfKcRZahEtWksMmwVWEOWC9qEJf4sRQ5ULKiKsyYkpUx8rkgPNjjCW4fX0Y2Ijde4AqhB83I6vJA4OZQofq8N1iCDlM8ZzkTYs1nzKBvwt7MtGffusyPq59QZuLLaSh\\\\\\\/JqThlYTXXhpjVkFSH6l6r86oeqN9x\\\\\\\/7YIvFeCTxmmpAaVUN7DTxxZoOArJg6DgOWJ0+btoybdotOnSS2x0lvj5DhmKYO3dcKeG1b\\\\\\\/SUR6fEzRy9Cjv+ZjX8WdmqtyBc6ABilmttFukChxNrvUdCLOwrDv+V2yKBqKT1ZZxaFagKARJPULkl1rzwtGRwIagXL6TqTSauAuqbgxqxD9cbUVLhPliHu\\\\\\\/m0Xo204UU7tXuaJ5aCqba+NYe\\\\\\\/QLXY\\\\\\\/VA+DiWNu9X8rNPK9xhBH74nebb2\\\\\\\/NlRCkfRWxZBrTuZF440mzaBWEn3aYGcqfgioB6JSlMOcsyZXDpr\\\\\\\/Xgbs27hvhI74vKPx3koZUk9QZkhMGHizzX7Dx5TXlmFfqSNOeYGB0LhIB\\\\\\\/mLCSbENX7WmNz4j3j00Hn03lSBjDXnXGb7UbsQMysELZfCGgL+DQdwnsAXOcU96x2h6hAteg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022wm\\\\\\\/U00ONh1TSZz6rsgFLpg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022c8cc593a312f5b64b3455445a4e806fb5a25bf31759f2e5ef38cce2e369d8ad2\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022egHfrhAjcZlIH7jEshVNaXLEitnY8jMcxQ0PCXDYF8cwiEhrfO8n\\\\\\\/d6ssUbEU3Jy6Hsm6CZs\\\\\\\/dX4hgRpdgrIeHBILfeVWxK2Ptx8+y1OBPTV\\\\\\\/n310j1rlUVa3oCqiiS8w05Yw4Z8M0zgFlOHmM3y5koZaRNwYkXkZiraa2IlHyMWSKrsC4+V3ApldcWmExovHfuXIcc7uD4\\\\\\\/OU9VRqTtqSbgzcKUobsQdFv5kJWQYQk3ZGpvODuNAsmpyWRRcA33MfMZ9895MXOX9ywe7psMx7DFyXk\\\\\\\/9lFgb2\\\\\\\/PRJlcHcSfX3vnsGcwBKE9qeBR5z77vWxpFFZ++gAk2dggjs4O\\\\\\\/pAGd+8uSnUffPVuPtDZMhYJIil8kPGzqfN84Ff5oYVvBSFTORN8So2PdHu6Nfb8UUbkyCAmk1IMAApFJbM+Zd6gd61hDwp7tu9CZzdafPrpCdfsv7TaXQw0Wj+6G66blXapJ+EuhJ4JA5pDHUJPn\\\\\\\/jyvf92GzXVtiu2uggvzSTMgpKi8ESuwRiOB5BcCuyBW8RRvcCWXcYl5onbdR5PEgYDVj3MjfxJHcbl169EXE96qR+VWotxfgmGBF196OIeL4xXnx5rqLLFI0W+H5Uf7IcXmSkQ5xGlmBlREHyLc5sP9V+j2qqYX0esL6IM42eKqZNEtDj6XqR\\\\\\\/1NrKhfUOqDIZ7Aw\\\\\\\/X2fKpBKss26KywC3GSCqKlSxlIDZbZTdTafrwpjaYJnnWs+cVEkFx28BplPSlAjFOmSrBYcGTdVHykjwux+YvqiRjxMuoGyZHN6fvnUjI6ECJqKTv+pUjFQoKvXa+vnu9UK\\\\\\\/HGCu\\\\\\\/c5MOAWZhONyGa9Rs6Nnx9nuaIb6\\\\\\\/4ZGPF8FLpyMHqsaA+nD6MC0sSEWrjdHdI9rIRjqugwT2PNq54pbNGMHJYMlW1kkRpe3srCdyfq96N\\\\\\\/6ggCnQUVtU5SV\\\\\\\/iHVoLqER3gcYOjeSr80+7mDSA+h\\\\\\\/D4HYEkwzz3lsBkHrMD280AVk90e+JjqqtxipqRf+PC79nuUktc14VQ6MmksG8sHnVLanTaHLxlYWP41u9k8gNZ4wiM\\\\\\\/A1sXm8X5zsiEptGKIlufTKIPFaq0PcTnDf1VyfeWfYCN+8AHFIqtJGcVhYROpcoI7uFjtknhFDgjV\\\\\\\/22HPjnOFz2smHXl6ng3OTY0Oga+ybi\\\\\\\/GYsQwbADyHW0aItHlOBnqjd8R4H3C0Ar97pMw2ZQEXNZhmwAh4UIwHNpphK6twXtLNRhjMlQWarxTiFXashkHFRiGjO0d+8VZqqR7tr2JD7ml8oOU\\\\\\\/O8h4Vxr8TgYkNRcroOSXyaAwfECZUmrG1w\\\\\\\/y3OhYNkXMBgCI54yrDAKcwskgyiEqdZLhoez0ehnkSUDXanv5wiDyZsP\\\\\\\/q5\\\\\\\/\\\\\\\/Jj3s1M8bfW6Yt9gx+i6HLwz3Evp0QbKU78elagRF9sjRpZIVuUbKkFlqyj3GNISpo4bAchvqS\\\\\\\/QhrY\\\\\\\/+pEyVpOsi1\\\\\\\/6SkIzDDLifXRed3BbpBKAKVstwBwsnnZYiV+BGVAFx+YfDEZF8wHUWU2gqdGUAw4PtP7\\\\\\\/leeaYBoRL+ZraypRdJsVXut5\\\\\\\/PTnGBiINW1TlhmsNJsJch5PDPqUiwhfWgmxVkt9Gj6v7PZcGkB+rTAArXETVlgmSwI9wg9ukE0kYf42ru8sAVTT4EMues8+2Qt6yoQdyZXqTcpM46o7AZL\\\\\\\/hiBBlpceVcaiIv412g\\\\\\\/V9Y6L4+EmBiU8B3oulwXcSau\\\\\\\/aLKI+NRKwrIub\\\\\\\/Eb4ZV3xvxQK1fTxLDQfSC1qr9E01MvBVZqz2+G0ltP\\\\\\\/2tY5MjvVKFTC4mBdUNK+a4taRw7oNlv5cTGgHKxQPATIlKmcIeUNUvEQCW\\\\\\\/15OBOg2vHJr7E0eaHMNZcKWlZ3v+f\\\\\\\/P2rOCNZ\\\\\\\/u6f+Z4+j0owGH7qEGexr3ZlkaV8niSbqS6fNcTXwgiTXXgjDkz540TQ3NxGe6+JkRDw6kaFg+lh94LY\\\\\\\/DhtFUae8se\\\\\\\/4djIcNDlkN17Z7hgJcbD\\\\\\\/JkqCiQlBEw==\\\u0022,\\\u0022iv\\\u0022:\\\u00227zYwmFFHopuCQRILuQu04w==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002277c70fb5b5a1d58e4b5aef48bdda659d31afb57ec2bbe81175293c3ec25ffc9d\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460151 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/10/2027" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-5pyY3nORyn7F1NHwcmY9QwvYjpLPAOPYA_SFp0W-YFE" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/10/2027" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-MM9qX-xaPfPlB5SQ6rtsh-2yBUAFW5y8amkghMP2m1I" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/10/2021" title="High-Density Genotypes of Inbred Mouse Strains: Improved Power and Precision of Association Mapping" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/10/2037" title="Genome-Wide Linkage Analysis Identifies Loci for Physical Appearance Traits in Chickens" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460151" id="node460151" data-pisa="ggg;5/10/2027" data-pisa-master="ggg;g3.115.019877" data-apath="/ggg/5/10/2027.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">GC-Content of Synonymous Codons Profoundly Influences Amino Acid Usage</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Jing Li</span>, <span class="highwire-citation-author" data-delta="1">Jun Zhou</span>, <span class="highwire-citation-author" data-delta="2">Ying Wu</span>, <span class="highwire-citation-author" data-delta="3">Sihai Yang</span> and <span class="highwire-citation-author" data-delta="4">Dacheng Tian</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2027-2036;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.019877 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460151" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jing Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>State Key Laboratory of Pharmaceutical Biotechnology, School of Life Sciences, Nanjing University, Nanjing 210093, China </div><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute for Research on Cancer and Aging, Nice (IRCAN), Nice, France </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jing%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJing%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jun Zhou </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Organismic and Evolutionary Biology, Harvard University, Cambridge, Massachusetts </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jun%2BZhou%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhou%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJun%2BZhou%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:dtian@nju.edu.cn" class="" data-icon-position="" data-hide-link-title="0">dtian@nju.edu.cn</a>
<a href="mailto:jzhou@oeb.harvard.edu" class="" data-icon-position="" data-hide-link-title="0">jzhou@oeb.harvard.edu</a></span></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Ying Wu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>State Key Laboratory of Pharmaceutical Biotechnology, School of Life Sciences, Nanjing University, Nanjing 210093, China </div><div class='author-affiliation'><span class='nlm-sup'>§</span>Institute for Bioscience and Biotechnology Research, University of Maryland, Rockville, Maryland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ying%2BWu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wu%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYing%2BWu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Sihai Yang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>State Key Laboratory of Pharmaceutical Biotechnology, School of Life Sciences, Nanjing University, Nanjing 210093, China </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sihai%2BYang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yang%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASihai%2BYang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Dacheng Tian </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>State Key Laboratory of Pharmaceutical Biotechnology, School of Life Sciences, Nanjing University, Nanjing 210093, China </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dacheng%2BTian%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tian%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADacheng%2BTian%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:dtian@nju.edu.cn" class="" data-icon-position="" data-hide-link-title="0">dtian@nju.edu.cn</a>
<a href="mailto:jzhou@oeb.harvard.edu" class="" data-icon-position="" data-hide-link-title="0">jzhou@oeb.harvard.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/5/10/2027" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460151" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460151/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/10/2027.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460151" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460151/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/10/2027.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460151" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460151/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/10/2027.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460151" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460151/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/5/10/2027.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460151" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460151/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Amino acids typically are encoded by multiple synonymous codons that are not used with the same frequency. Codon usage bias has drawn considerable attention, and several explanations have been offered, including variation in GC-content between species. Focusing on a simple parameter—combined GC proportion of all the synonymous codons for a particular amino acid, termed GC<sub>syn</sub>—we try to deepen our understanding of the relationship between GC-content and amino acid/codon usage in more details. We analyzed 65 widely distributed representative species and found a close association between GC<sub>syn</sub>, GC-content, and amino acids usage. The overall usages of the four amino acids with the greatest GC<sub>syn</sub> and the five amino acids with the lowest GC<sub>syn</sub> both vary with the regional GC-content, whereas the usage of the remaining 11 amino acids with intermediate GC<sub>syn</sub> is less variable. More interesting, we discovered that codon usage frequencies are nearly constant in regions with similar GC-content. We further quantified the effects of regional GC-content variation (low to high) on amino acid usage and found that GC-content determines the usage variation of amino acids, especially those with extremely high GC<sub>syn</sub>, which accounts for 76.7% of the changed GC-content for those regions. Our results suggest that GC<sub>syn</sub> correlates with GC-content and has impact on codon/amino acid usage. These findings suggest a novel approach to understanding the role of codon and amino acid usage in shaping genomic architecture and evolutionary patterns of organisms.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/gc-content-0" class="hw-term hw-article-keyword hw-article-keyword-gc-content" rel="nofollow">GC-content</a></li><li class="kwd"><a href="/keyword/synonymous-codon" class="hw-term hw-article-keyword hw-article-keyword-synonymous-codon" rel="nofollow">synonymous codon</a></li><li class="kwd"><a href="/keyword/amino-acid-usage" class="hw-term hw-article-keyword hw-article-keyword-amino-acid-usage" rel="nofollow">amino acid usage</a></li><li class="kwd"><a href="/keyword/codon-usage" class="hw-term hw-article-keyword hw-article-keyword-codon-usage" rel="nofollow">codon usage</a></li></ul><p id="p-4">The genetic code describes how the 64-nucleotide triplets specify 20 amino acids. Most amino acids have at least two synonymous codons that are, however, not used at the same frequencies in different genomes. <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Grantham <em>et al.</em> (1980)</a> proposed the “genome hypothesis” in 1980 that assumed a species-specific pattern of codon usage. Interestingly, even in the same genome, the codon usage varies significantly among genes with different expression levels (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Dos Reis <em>et al.</em> 2003</a>), functions (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Chiapello <em>et al.</em> 1998</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Karlin <em>et al.</em> 1998</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Liu <em>et al.</em> 2005</a>), and tissue-specific patterns (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Plotkin <em>et al.</em> 2004</a>). Various factors have been suggested to affect codon usage bias, such as relative abundance of iso-accepting transfer RNAs, gene expression level, gene length, gene conversion, messenger RNA structure, and DNA base composition (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Miyata <em>et al.</em> 1979</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Ikemura 1981</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Gouy and Gautier 1982</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Sharp <em>et al.</em> 1986</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Eyre-Walker 1996</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Duret and Mouchiroud 1999</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Sueoka and Kawanishi 2000</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Maside <em>et al.</em> 2004</a>). The most significant factor linked to the phenomenon of codon bias between different organisms is perhaps GC-content.</p><p id="p-5">The great influence of GC-content on codon bias was first predicted by Sueoka in the 1960s (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Sueoka 1961</a>, <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">1962</a>). With limited available nucleotide sequences during the 1980s and 1990s, intragenomic comparisons of heterologous DNA and protein sequences (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bernardi and Bernardi 1986</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">D’Onofrio <em>et al.</em> 1991</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Collins and Jukes 1993</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Berkhout and Van Hemert 1994</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Porter 1995</a>) and intergenomic comparisons of homologous gene sequences (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Lobry 1997</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Gu <em>et al.</em> 1998</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Wilquet and Van De Casteele 1999</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Lafay <em>et al.</em> 1999</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">D’Onofrio <em>et al.</em> 1999</a>) were performed to confirm the correlations between the nucleotide composition of DNA and the amino acid content of the encoded proteins. Later, when more sequenced genomes became available, <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Knight <em>et al.</em> (2001)</a> found similar results and suggested that GC-content is the drive for codon usage and that the correlation between GC-content and amino acid or codon usage is modulated by both mutation and selection. Another study showed that the genome-wide codon bias in eubacteria and archaea could be predicted from intergenic sequences that are not translated, suggesting that genome-wide codon bias is determined primarily by mutational processes throughout the genome (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Chen <em>et al.</em> 2004</a>). On the basis of the complete genome sequences, <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Singer and Hickey (2000)</a> partitioned the universal codon table into GC-rich, AT-rich, and neutral codons. They further confirmed a prediction that GC-rich coding sequences (CDS) would encode amino acids with GC-rich codons, showing that biased DNA encodes biased proteins on a genome-wide scale. The finding that a positive correlation between the degree of amino acid bias and the magnitude of protein sequence divergence further support that mutational bias can have a major effect on the molecular evolution of proteins (<a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Singer and Hickey 2000</a>). The influence of GC-content to codon bias also was demonstrated by other studies (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Karlin and Mrázek 1996</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Kudla <em>et al.</em> 2006</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Hildebrand <em>et al.</em> 2010</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Nabiyouni <em>et al.</em> 2013</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bohlin <em>et al.</em> 2013</a>).</p><p id="p-6">Mutation and natural selection are suggested to be the two main forces shaping the genomic codon and amino acid usage patterns within and between species (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Duret 2002</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Chamary <em>et al.</em> 2006</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Hershberg and Petrov 2008</a>). The mutational explanation posits that codon bias arises from biases in nucleotides composition that are produced by point mutations, contextual biases in the point mutation rates or biases in repair. It is neutral without any fitness advantages. In contrast, the natural selection explanation suggests that synonymous mutations would influence the fitness of organisms and therefore be promoted or repressed during evolution. Despite the fact that both remain elusive, these two mechanisms are not mutually exclusive but can both play important roles in patterning the codon and amino acid usage in genomes (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Bulmer 1991</a>; <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Duret 2002</a>; <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Hershberg and Petrov 2008</a>).</p><p id="p-7">Even though codon usage bias has been documented extensively (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Karlin <em>et al.</em> 1998</a>; <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Chen <em>et al.</em> 2004</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Gustafsson <em>et al.</em> 2004</a>; <a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Liu <em>et al.</em> 2005</a>; <a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">Hershberg and Petrov 2008</a>), any further understanding of codon usage would still have important implications for molecular and genomic evolution. Here, we intend to investigate the impacts of GC-content on codon/amino acid usage quantitatively in more detail, starting from and focusing on a simple parameter, GC proportion of all the synonymous codons for a particular amino acid. We term this proportion the GC<sub>syn</sub>. In this way, the GC-content of codons is defined quantitatively rather than qualitatively regarded as GC-rich or AT-rich. On the basis of GC<sub>syn</sub>, the 20 amino acids are distinctly classified into three groups (four high-, 11 intermediate-, and five low-GC<sub>syn</sub> amino acids), and their usage characteristics could be analyzed independently. Using 65 representative genomes from diverged species including bacteria, plants, and animals, we identified different associations between usage pattern and synonymous codon numbers for an amino acid from different GC<sub>syn</sub> groups, indicative of adaptive evolution. More important, according to distinct usage patterns of the three groups of amino acids, we audaciously predict that identical codon/amino acid usage patterns exist when GC-content is similar, regardless of species or lineage. Comprehensive investigation of codon usage using three different units—50 consecutive codons, CDS, and genomes—in diverse prokaryotic and eukaryotic organisms indicates that our predictions are true, indicating that GC-content is a pronounced determinant of amino acid usage. In addition, we created an equation to estimate the degree of GC-content determinant to amino acid composition and GC<sub>syn</sub> variation in different regions of the genomes. Overall, our results provide a novel view to understand codon bias and its role in the formation of genome architecture.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Selection of genome sequences</h3><p id="p-8">Fifteen eukaryotic and 50 prokaryotic genomes were selected on the basis of the following criteria: widely represented eukaryotic genomes according to phylogeny, and all the prokaryotic genomes greater than 4 Mb in the National Center for Biotechnology Information (<a href="ftp://ftp.ncbi.nlm.nih.gov">ftp://ftp.ncbi.nlm.nih.gov</a>). The genomes and annotations of animals were retrieved from the Ensembl database (<a href="http://www.ensembl.org">www.ensembl.org</a>), and those of <em>Arabidopsis thaliana</em> and <em>Oryza sativa</em> were downloaded from TIGR (<a href="http://www.tigr.org">http://www.tigr.org</a>) and GRAMENE (<a href="http://www.gramene.org">http://www.gramene.org</a>), respectively. The genomic information of other plants was obtained from JGI (<a href="ftp://ftp.jgi-psf.org/pub/JGI_data/">ftp://ftp.jgi-psf.org/pub/JGI_data/</a>). The CDS of each species were extracted with the annotations from corresponding, aforementioned Web sites by the use of Perl programs. Protein coding genes with ambiguous bases were eliminated. For the genes with multitranscripts, we chose the longest CDS.</p></div><div id="sec-3" class="subsection"><h3>Classification of the analysis units</h3><p id="p-9">To investigate the correlation between the frequency of amino acid (or codon) usage and GC-content, we used the whole-genome sequences of different species, CDS, or 50 consecutive codons as separate units. Of note, we also tested 30 consecutive codons and 100 consecutive codons as analysis units in comparison with 50 consecutive codons analyses. The results were highly similar so that we focused on the 50 consecutive codons analysis in the following work. For the “whole-genome sequence” analyze unit, the organisms were ranked according to their genomic GC-content from low to high. For the “CDS” analyze unit, we combined CDSs from 65 organisms and sorted them based on their GC-content, which ranged from 0.30 to 0.80, into 50 groups with an equal interval of 0.01 (second column of <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). Therefore, every group of CDS contains sequences from multiple species instead of from only one genome, which eliminates potential genome biases since any genome may have a particular coding strategy as suggested in Grantham’s genome hypothesis (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Grantham <em>et al.</em> 1980</a>). For the “50 consecutive codon” units, each CDS in any genome was dissected sequentially into sets of 50 consecutive codons, whereas the remaining regions shorter than 50 codons were discarded. This largely diminishes the influence of special or functional motifs in a gene. These sequences also were sorted into 50 groups in the same way as the CDS with a GC interval of 0.01. Since the fractions of sequences whose GC-content is &lt;0.3 or &gt;0.8 were small (total &lt; 1.30%), they were excluded from further analyses.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/10/2027/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The schematic diagram of codon usage analysis. As is shown in the figure, the table represents some of the codon usage results of 50-consecutive codon group and the table was sorted by the GC-content range of each group. The first column is the number of each dataset line; the second column lists the GC-content range of each 50 consecutive codon group; the third column is the average GC-content of each group; and the remaining columns show the usage frequencies of each synonymous codon. GC-nearest pair consists of two datasets with the nearest GC-content (i.e., the nearest dataset line number, such as datasets 1 and 2, 3 and 4, &#x2026;, 49 and 50 in the table); GC-N pair consists of two datasets with larger GC-content divergence and N is the discrepancy between the number of any two datasets (e.g., GC-2 pairs contain the pairs of datasets 1 and 3, 2 and 4, 3 and 5&#x2026;48 and 50; GC-3 pairs contain the pairs of datasets 1 and 4, 2 and 5, 3 and 6 &#x2026; 47 and 50)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-905079360" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The schematic diagram of codon usage analysis. As is shown in the figure, the table represents some of the codon usage results of 50-consecutive codon group and the table was sorted by the GC-content range of each group. The first column is the number of each dataset line; the second column lists the GC-content range of each 50 consecutive codon group; the third column is the average GC-content of each group; and the remaining columns show the usage frequencies of each synonymous codon. GC-nearest pair consists of two datasets with the nearest GC-content (&lt;em&gt;i.e.&lt;/em&gt;, the nearest dataset line number, such as datasets 1 and 2, 3 and 4, &#x2026;, 49 and 50 in the table); GC-N pair consists of two datasets with larger GC-content divergence and N is the discrepancy between the number of any two datasets (&lt;em&gt;e.g.&lt;/em&gt;, GC-2 pairs contain the pairs of datasets 1 and 3, 2 and 4, 3 and 5&#x2026;48 and 50; GC-3 pairs contain the pairs of datasets 1 and 4, 2 and 5, 3 and 6 &#x2026; 47 and 50).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/10/2027/F1.medium.gif" width="440" height="199"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/5/10/2027/F1.medium.gif" width="440" height="199"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/10/2027/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/10/2027/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466750" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-10" class="first-child">The schematic diagram of codon usage analysis. As is shown in the figure, the table represents some of the codon usage results of 50-consecutive codon group and the table was sorted by the GC-content range of each group. The first column is the number of each dataset line; the second column lists the GC-content range of each 50 consecutive codon group; the third column is the average GC-content of each group; and the remaining columns show the usage frequencies of each synonymous codon. GC-nearest pair consists of two datasets with the nearest GC-content (<em>i.e.</em>, the nearest dataset line number, such as datasets 1 and 2, 3 and 4, …, 49 and 50 in the table); GC-N pair consists of two datasets with larger GC-content divergence and N is the discrepancy between the number of any two datasets (<em>e.g.</em>, GC-2 pairs contain the pairs of datasets 1 and 3, 2 and 4, 3 and 5…48 and 50; GC-3 pairs contain the pairs of datasets 1 and 4, 2 and 5, 3 and 6 … 47 and 50).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-4" class="subsection"><h3>Linear regression analysis of codon usage</h3><p id="p-11">We calculated the usage of all the 61 codons (excluding the start and stop codons; of note, we excluded the first Met encoded by start codon in CDS, which would lead to biased analysis of amino acid usage) in all the 65 genomes that were first orderly ranked based on genomic GC-content. Similarly, for “CDS” and “50 consecutive codon” unit, we acquired codon usage data for each of the 50 groups (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a> depicts the organization of the data). Then, we compared the codon usage in any possible pairs, such as two organisms, two groups of CDSs, and two groups of 50 consecutive codons segments. The codon usage data of two groups (<em>i.e.</em>, to compare the linear regression correlation between two sets of the individual codon usage data as demonstrated in the last column of <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>) were plotted on x- and y-axes, respectively. More specifically, to analyze the group pairs with the nearest GC-content, we used the sorted codon usage tables and performed the linear regression analysis on the adjacent group pairs to get the slope and correlation coefficient (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a> “GC-nearest pair”). GC-nearest pair consists of two datasets with the nearest GC-content (<em>i.e.</em>, the nearest dataset line number, such as datasets 1 and 2, 3 and 4, …, 49 and 50 in the table). In addition, we also conducted regression analysis between groups with greater GC-content divergence and calculated slope and R<sup>2</sup>, which were defined as GC-2 pair, GC-3 pair... GC-N pair (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1</a>). GC-N pair consists of two datasets with larger GC-content divergence and N is the discrepancy between the number of any two datasets (<em>e.g.</em>, GC-2 pairs contain the pairs of datasets 1 and 3, 2 and 4, 3 and 5…48 and 50; GC-3 pairs contain the pairs of datasets 1 and 4, 2 and 5, 3 and 6 … 47 and 50). The average values of the slope and R<sup>2</sup> between GC-nearest pairs, GC-2 pairs, GC-3 pairs…GC-N pairs were calculated and plotted against the difference of GC-content.</p><p id="p-12">The linear regression analysis was conducted in the software package R (version 2.14.1) to derive the correlation coefficient (R<sup>2</sup>) and the slope of the linear correlation between pairs. For every paired comparison, regression analyses were conducted for the usages of both 20 amino acids and 61 sense codons.</p></div><div id="sec-5" class="subsection"><h3>Estimation of the effects of regional GC-content</h3><p id="p-13">To estimate the effects of increased regional GC-content (<em>i.e.</em>, ΔGC, the difference of GC-content between the regions with 0.79−0.80 and 0.30−0.31 GC-content), we split it into two components, which are (1) the GC fraction change due to the use of different amino acids and (2) the GC change due to the use of different synonymous codons for the same amino acid. The following formula was used:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2027/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2027/embed/mml-math-1.gif"/></noscript></span></span>where ΔGC indicates the increased GC-content of an amino acid from low (0.30−0.31) to high-GC (0.79−0.80) regions; GC<sub>syn0.8</sub> and GC<sub>syn0.3</sub> are the average GC-content of synonymous codons of an amino acid appearing at the regions with 0.79−0.80 and 0.30−0.31 GC-content, respectively; A<sub>0.8</sub> and A<sub>0.3</sub> are the usages of this amino acid in these two regions, respectively. (A<sub>0.8</sub> − A<sub>0.3</sub>) × GC<sub>syn0.8</sub> explains the proportion of GC-content changes influencing the changes of amino acid usage. (GC<sub>syn0.8</sub> − GC<sub>syn0.3</sub>) × A<sub>0.3</sub> represents the proportion of GC changes influencing the GC changes in the use of high GC-content synonymous codons for that amino acid. It is supposed that the regional GC-content increase would affect the increased usage of an amino acid and its increased GC<sub>syn</sub>. To give a conserved estimate of the contribution, only four high-GC<sub>syn</sub> amino acids (Ala, Gly, Pro, and Arg) are included in our analysis. We can likewise infer that of an amino acid to the change of AT-content from high (0.79−0.80) to low GC (0.30−0.31) regions.</p></div><div id="sec-6" class="subsection"><h3>Simulation with random sequences</h3><p id="p-14">In addition using the real genome sequences, we repeated our analysis on simulated genomes with different level of GC-content as a control. These simulated genomes are composed of random DNA sequences. We used online software to generate random DNA sequences with different GC-content (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Villesen 2007</a>), which varies from 0.30 to 0.80 with an equal interval of 0.01, just the same as we described above. In each of these GC-content groups, we generated 10,000 random DNA sequences with different lengths ranging from 1200 to 1500 bp, which is almost the same as the average CDS length of our real sequence data.</p></div><div id="sec-7" class="subsection"><h3>Data availability</h3><p id="p-15">All the sequence data are available in public databases as described in <em>Materials and Methods</em>.</p></div></div><div class="section results" id="sec-8"><h2 class="">Results</h2><div id="sec-9" class="subsection"><h3>Three types of amino acids based on GC-content of synonymous codons</h3><p id="p-16">We calculated the average GC-content among the synonymous codons for each amino acid (GC<sub>syn</sub>) by using the standard DNA codon table for nuclear genes. GC<sub>syn</sub> was determined to range from 0.11 to 0.83 (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>). With respect to the GC<sub>syn</sub>, there are three largely distinct groups of amino acids: four high-, 11 intermediate-, and five low-GC<sub>syn</sub> amino acids. Within each group, GC<sub>syn</sub> is the same for most amino acids but lower for one amino acid. Trp and Met are unusual in that they are midway between the high/intermediate and intermediate/low groups, respectively (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2A</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/10/2027/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Patterns of amino acid usage and its GC-content of synonymous codons in 65 nuclear genomes in this study. (A) Average GC-content of codons for an amino acid based on the standard DNA genetic code table; the numbers on the bars are the number of synonymous codons for each amino acid; (B&#x2212;D) Usage frequency variation of amino acids in three groups along with the regional GC-content (in 50 consecutive codons analyze unit), (E) is the slope value in a linear regression analysis for the lines in B&#x2212;D, and (F) is for the absolute increased GC-content for each amino acid between regions with 0.3&#x2212;0.31 and 0.79&#x2212;0.80 GC-content. The black and nonblack bars represent the variably and the less-variably used amino acids, respectively." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-905079360" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Patterns of amino acid usage and its GC-content of synonymous codons in 65 nuclear genomes in this study. (A) Average GC-content of codons for an amino acid based on the standard DNA genetic code table; the numbers on the bars are the number of synonymous codons for each amino acid; (B&#x2212;D) Usage frequency variation of amino acids in three groups along with the regional GC-content (in 50 consecutive codons analyze unit), (E) is the slope value in a linear regression analysis for the lines in B&#x2212;D, and (F) is for the absolute increased GC-content for each amino acid between regions with 0.3&#x2212;0.31 and 0.79&#x2212;0.80 GC-content. The black and nonblack bars represent the variably and the less-variably used amino acids, respectively.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/10/2027/F2.medium.gif" width="357" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/5/10/2027/F2.medium.gif" width="357" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/10/2027/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/10/2027/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466844" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-17" class="first-child">Patterns of amino acid usage and its GC-content of synonymous codons in 65 nuclear genomes in this study. (A) Average GC-content of codons for an amino acid based on the standard DNA genetic code table; the numbers on the bars are the number of synonymous codons for each amino acid; (B−D) Usage frequency variation of amino acids in three groups along with the regional GC-content (in 50 consecutive codons analyze unit), (E) is the slope value in a linear regression analysis for the lines in B−D, and (F) is for the absolute increased GC-content for each amino acid between regions with 0.3−0.31 and 0.79−0.80 GC-content. The black and nonblack bars represent the variably and the less-variably used amino acids, respectively.</p><div class="sb-div caption-clear"></div></div></div><p id="p-18">We used 65 representative genomes from bacteria, plants, and animals to investigate the amino acid usage and codon usage based on GC<sub>syn</sub> (<a href="http://www.g3journal.org/content/suppl/2015/08/05/g3.115.019877.DC1/019877SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/content/suppl/2015/08/05/g3.115.019877.DC1/TableS1.pdf">Table S1</a>). Our analyses show evidences for a connection between the usage and the number of synonymous codons for each amino acid. First, within each group of amino acid described previously, there is one amino acid (<em>i.e.</em>, Arg, Leu, or Ile) for which the GC<sub>syn</sub> is lower than other members (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2A</a>). Each of these three amino acids has the greatest number of synonymous codons in its group: 6, 6, and 3, respectively. Among these 15 codons are six of the eight least used sense codons (4 for Arg, 1 for Leu, and 1 for Ile) (<a href="http://www.g3journal.org/content/suppl/2015/08/05/g3.115.019877.DC1/TableS2.pdf">Table S2</a>). When these less frequently used codons are ignored, each of the three amino acids’ GC<sub>syn</sub> becomes greater. Second, the single codon for both Trp and Met does not have a GC<sub>syn</sub> that can fall into one of the three defined groups (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2A</a>), and they represent the least (0.0128, Trp) and the third least-used amino acids (0.0190, Met) in all 65 species (<a href="http://www.g3journal.org/content/suppl/2015/08/05/g3.115.019877.DC1/TableS2.pdf">Table S2</a>). Third, there is a positive correlation between amino acid usage and the number of codons for the 11 amino acids in the intermediate-GC<sub>syn</sub> group, irrespective of the regional GC-content (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>; r = 0.787−0.864, <em>P</em> &lt; 0.01). Interestingly, Leu, which is encoded by six synonymous codons, always has the greatest usage (0.100) although the usage frequencies for each of the six individual codons are different. For all 20 amino acids, a significantly positive correlation (<em>P</em> &lt; 0.05) is still present in the regions in which GC-content is greater than 0.35 (<a href="http://www.g3journal.org/content/suppl/2015/08/05/g3.115.019877.DC1/TableS3.pdf">Table S3</a>). These close connections between GC<sub>syn</sub>, codon usage and the number of synonymous codons suggest possible mechanisms of adaptive evolution.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/10/2027/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The correlation between amino acid usage and the number of synonymous codons for the 11 less variably used amino acids with intermediate-GCsyn at high- (0.79&#x2212;0.80), intermediate- (0.50&#x2212;0.51), and low-GC (0.3&#x2212;0.31) regions." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-905079360" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The correlation between amino acid usage and the number of synonymous codons for the 11 less variably used amino acids with intermediate-GC&lt;sub&gt;syn&lt;/sub&gt; at high- (0.79&#x2212;0.80), intermediate- (0.50&#x2212;0.51), and low-GC (0.3&#x2212;0.31) regions.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/10/2027/F3.medium.gif" width="440" height="218"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/5/10/2027/F3.medium.gif" width="440" height="218"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/10/2027/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/10/2027/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466716" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-19" class="first-child">The correlation between amino acid usage and the number of synonymous codons for the 11 less variably used amino acids with intermediate-GC<sub>syn</sub> at high- (0.79−0.80), intermediate- (0.50−0.51), and low-GC (0.3−0.31) regions.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h3>Three distinct usage patterns of amino acids</h3><p id="p-20">With three obvious types of amino acids based on GC<sub>syn</sub>, one may ask whether this could be related to the different amino acid usage patterns in organisms. To address this question, we focused on the relationship among GC<sub>syn,</sub> regional GC-content, and amino acid usage in different genomes, hypothesizing that the amino acids with high GC<sub>syn</sub> could be used more frequently in high-GC regions and contrarily, amino acids with low GC<sub>syn</sub> could be used more frequently in low-GC regions. In light of this hypothesis, amino acids with intermediate GC<sub>syn</sub> are expected to be less sensitive to regional GC-content than those with extreme GC<sub>syn</sub>. With the 65 genomes, genome sequences, CDS, and 50-consecutive codons were analyzed separately as a unit from each of these species. We calculated the GC-content and amino acid usage for each unit and classified the results from CDS and 50 consecutive codon analysis units with an interval of 0.01 GC-content from “0.30−0.31” to “0.79−0.80” (see <em>Materials and Methods</em> for details; whole-genome comparisons do not result in continuous GC range fractions from one another). Three distinct amino acid usage patterns, consistent with the three types grouped by GC<sub>syn</sub>, were observed regardless of which unit was used (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2</a>, B−D for the 50 consecutive codons unit and <a href="http://www.g3journal.org/content/suppl/2015/08/05/g3.115.019877.DC1/FigureS1.pdf">Figure S1</a> for the other two units). The overall usages of the four amino acids with the highest GC<sub>syn</sub> and the five amino acids with the lowest GC<sub>syn</sub> both vary with the regional GC-content, whereas the usage of the remaining 11 amino acids with intermediate GC<sub>syn</sub> is less variable. Of note, this analysis suggests that both Trp and Met fall in with the intermediate group although their GC<sub>syn</sub> appear midway between adjacent groups (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2A</a>).</p><p id="p-21">To further characterize the three types of amino acids, the slopes of linear regression analyses for the lines in <a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2</a>, B−D were then calculated to reveal the degree of inclination in response to the increase of GC-content. Interestingly, a steeper incline was always observed in the amino acids with extreme GC<sub>syn</sub> (<a id="xref-fig-2-8" class="xref-fig" href="#F2">Figure 2E</a>). The largest positive slopes (from 0.116 to 0.286; 0.170 on average) were observed in the four amino acids with the greatest GC<sub>syn</sub> (Ala, Gly, Pro, and Arg; <em>P</em> &lt; 0.001 for each regression), whereas the most negative ones (from −0.17 to −0.06; −0.110 on average) appeared in five amino acids with the lowest GC<sub>syn</sub> (Tyr, Phe, Asn, Ile, and Lys; <em>P</em> &lt; 0.001 for each). The other 11 amino acids with intermediate GC<sub>syn</sub> exhibited relatively flat patterns (<a id="xref-fig-2-9" class="xref-fig" href="#F2">Figure 2C</a>) (slope = 0.012 on average; <em>P</em> &lt; 0.01). The minor usage changes across regions with different GC-content indicate that these 11 amino acids are less sensitive to the regional GC-content, consistent with our expectation. These results therefore suggest that there is a strong association between GC<sub>syn</sub> for a given amino acid and its usage variation among regions with different GC-content.</p><p id="p-22">Besides, given that the 65 species we investigated here have very different biology, life history traits, effective populations size, etc., which will greatly affect the selective pressures acting on a genome in general, and on synonymous codon and amino acid usage in particular, we performed the same analysis by separating prokaryotes from eukaryotes. Within eukaryotes, we also separated mammals and plants from other species. Indeed, we still observed the three distinct amino acid usage patterns in these different datasets (<a href="http://www.g3journal.org/content/suppl/2015/08/05/g3.115.019877.DC1/FigureS2.pdf">Figure S2</a>). Such findings suggest that the correlation is independent of selective pressures and any specific species.</p><p id="p-23">In addition to calculate regional GC-content by considering all the positions in CDS, we also computed GC-content from the third codon positions (GC3-content) because inside a codon, positions 1, 2, and 3 are not under similar evolutionary forces. We computed GC3-content and sorted the sequences based on their GC3-content, which ranged from 0.06 to 0.32, into 26 groups with an equal interval of 0.01. Then we checked the correlation between amino acid usage and GC3-content. We still observed the same three distinct usage patterns of amino acids (<a href="http://www.g3journal.org/content/suppl/2015/08/05/g3.115.019877.DC1/FigureS4.pdf">Figure S4, A and B</a>). The GC3-content analysis shows that the pattern of amino acid usage and GC3-content is consistent with the results derived from whole GC-content.</p></div><div id="sec-11" class="subsection"><h3>The usages of amino acids and codons are nearly identical for similar GC-content regions</h3><p id="p-24">The regular usage patterns of high-, low- and intermediate- GC<sub>syn</sub> amino acids (<a id="xref-fig-2-10" class="xref-fig" href="#F2">Figure 2, B−D</a>) and the corresponding groups (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>) suggest their usage is near equilibrium for the given GC-content, regardless of their eukaryotic or prokaryotic origins. When regional GC-content is low, the usage ratio of high- to low-GC<sub>syn</sub> amino acids is much less than 1, indicating the more frequent uses of low GC<sub>syn</sub> amino acids. As the GC-content increases, there is a point where high- and low-GC<sub>syn</sub> amino acids are used equally; after this point the usage ratio of high- to low-GC<sub>syn</sub> continues to increase gradually (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4B</a>). The dramatic ratio variation reflects the dynamics of amino acid usage and may indicate that it could be associated with functional variations in these motifs. However, in contrast to the high- and low-GC<sub>syn</sub> groups, the amino acid usage in the intermediate GC<sub>syn</sub> group shows little variation when GC-content changes from 0.3 to 0.7 (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4A</a>). On the basis of these observations, the usage frequencies would be expected to be the same in different genes or motifs with similar GC-content. We then conducted another linear regression analysis in which we calculated 49 slopes and 49 R<sup>2</sup>s from the 20 pairs of amino acid usage frequency between every two adjacent GC-groups (0.010 GC-difference intervals means every GC nearest pair in <a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1</a>). Strikingly, both slopes and R<sup>2</sup>s are close to 1 (1.0123 ± 0.025 and 0.9984 ± 0.0006, respectively) when the regional GC-content is similar, strongly suggesting that the usage frequency of a certain amino acid is likely constant in regions with same GC-content.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/10/2027/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The total usage patterns of three amino acids types. (A) The usage frequencies of high-, low-, and intermediate-GCsyn amino acids change when GC-content increase from 0.30 to 0.80. (B) The amino acid usage ratios of the high-GCsyn to low-GCsyn types are shown." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-905079360" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The total usage patterns of three amino acids types. (A) The usage frequencies of high-, low-, and intermediate-GC&lt;sub&gt;syn&lt;/sub&gt; amino acids change when GC-content increase from 0.30 to 0.80. (B) The amino acid usage ratios of the high-GC&lt;sub&gt;syn&lt;/sub&gt; to low-GC&lt;sub&gt;syn&lt;/sub&gt; types are shown.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/10/2027/F4.medium.gif" width="440" height="152"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/5/10/2027/F4.medium.gif" width="440" height="152"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/10/2027/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/10/2027/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466781" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-25" class="first-child">The total usage patterns of three amino acids types. (A) The usage frequencies of high-, low-, and intermediate-GC<sub>syn</sub> amino acids change when GC-content increase from 0.30 to 0.80. (B) The amino acid usage ratios of the high-GC<sub>syn</sub> to low-GC<sub>syn</sub> types are shown.</p><div class="sb-div caption-clear"></div></div></div><p id="p-26">The comparable usage of amino acids at similar GC-content may essentially reflect a constant usage of codons. On the basis of the usage of each of the 61 sense codons in the same set of data as previously used for the amino acid usage analysis, regression analysis (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>) reveals that both slopes and R<sup>2</sup>s are still close to 1 (1.0045 ± 0.0737 and 0.9946 ± 0.0038, respectively) when GC-content is similar. As the GC-content difference increases, the observed slope or R<sup>2</sup> is expected to deviate from 1. To test this expectation, we calculated slope or R<sup>2</sup> from all possible comparisons between two groups (see <em>Materials and Methods</em> for details; <a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1</a>). Shown in <a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>, the average slope and R<sup>2</sup> are reduced with the increase of GC difference between compared groups. For 50 consecutive codons and CDS analysis units, both the slope and R<sup>2</sup> approach 1 when GC-difference is 0 (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5</a>). In comparison, we used the “genome” analysis unit as a control. The slope and R<sup>2</sup> are 0.805 and 0.660 respectively in the “genome” unit when comparing two genomes with the smallest GC-content difference (0.0068). The lower values of slope and R<sup>2</sup> for “genome” than for “50 consecutive codon” and “CDS” units are mainly contributed by the noncoding sequences. Therefore, these results further suggest that compared with the noncoding regions, the usage frequency of each codon (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5</a>) or amino acid (<a href="http://www.g3journal.org/content/suppl/2015/08/05/g3.115.019877.DC1/FigureS5.pdf">Figure S5</a>) in different genes is more likely to be the same in regions of similar GC-content in the coding regions. In other words, there is a constant codon and amino acid usage when GC-content is similar. We also performed this regression analysis by separating prokaryotes, mammals, plants and other eukaryotes (<a href="http://www.g3journal.org/content/suppl/2015/08/05/g3.115.019877.DC1/FigureS3.pdf">Figure S3</a>). The results also are consistent, indicating that our results are more generally distributed in living organisms rather than specific for one group. It is also true that when the GC3-content is the same, the R square and slope are both approaching 1 (<a href="http://www.g3journal.org/content/suppl/2015/08/05/g3.115.019877.DC1/FigureS4.pdf">Figure S4C</a>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/10/2027/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The patterns of the correlation slope (A, C, and E) and the coefficient of determination R2 values (B, D, and F) for codon usages between different GC groups. Fifty-codon (A and B), CDS (C and D), and whole genome (E and F) were used respectively as an analysis unit, with a GC-content difference less than 0.25 (the values are nearly 0 when &#x394;GC &gt; 0.25). As shown in 50 consecutive codons (A and B) and CDS (C and D) analysis units, when the difference of GC-content from two compared units is 0 (x-axis), both the linear regression slope and the regression coefficient R2 are infinitely approaching 1 (P = 2.32e-54), which strongly suggests that codon usage frequencies are nearly constant in regions with similar GC-content." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-905079360" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The patterns of the correlation slope (A, C, and E) and the coefficient of determination R&lt;sup&gt;2&lt;/sup&gt; values (B, D, and F) for codon usages between different GC groups. Fifty-codon (A and B), CDS (C and D), and whole genome (E and F) were used respectively as an analysis unit, with a GC-content difference less than 0.25 (the values are nearly 0 when &#x394;GC &gt; 0.25). As shown in 50 consecutive codons (A and B) and CDS (C and D) analysis units, when the difference of GC-content from two compared units is 0 (x-axis), both the linear regression slope and the regression coefficient R&lt;sup&gt;2&lt;/sup&gt; are infinitely approaching 1 (&lt;em&gt;P&lt;/em&gt; = 2.32e-54), which strongly suggests that codon usage frequencies are nearly constant in regions with similar GC-content.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/10/2027/F5.medium.gif" width="290" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/5/10/2027/F5.medium.gif" width="290" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/10/2027/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/10/2027/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466754" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-27" class="first-child">The patterns of the correlation slope (A, C, and E) and the coefficient of determination R<sup>2</sup> values (B, D, and F) for codon usages between different GC groups. Fifty-codon (A and B), CDS (C and D), and whole genome (E and F) were used respectively as an analysis unit, with a GC-content difference less than 0.25 (the values are nearly 0 when ΔGC &gt; 0.25). As shown in 50 consecutive codons (A and B) and CDS (C and D) analysis units, when the difference of GC-content from two compared units is 0 (x-axis), both the linear regression slope and the regression coefficient R<sup>2</sup> are infinitely approaching 1 (<em>P</em> = 2.32e-54), which strongly suggests that codon usage frequencies are nearly constant in regions with similar GC-content.</p><div class="sb-div caption-clear"></div></div></div><p id="p-28">To make our result more convincing, we performed an experiment as a control by repeating the analysis on simulated genomes with different level of GC-content and these simulated genomes are made up of random DNA sequences. We used online software to generate random DNA sequences with different GC-content (<a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Villesen 2007</a>). GC-content varies from 0.30 to 0.80 with an equal interval of 0.01. In each GC-content group, we generated 10,000 random DNA sequences with different lengths ranging from 1200 to 1500 bp. Using these random sequences, we repeated our analysis. Different from what we identified using real sequences, the regression result of random sequences (<a href="http://www.g3journal.org/content/suppl/2015/08/05/g3.115.019877.DC1/FigureS6.pdf">Figure S6</a>) shows that when GC-content difference is 0.01, the R square is not equal to 1.0 (mean = 0.980, <em>P</em> = 0.1316, <em>t</em>-test). In comparison, with the real CDS, when GC-content difference is 0.01, the R square is almost 1.0 (mean = 0.994, <em>P</em> = 2.008e-12, <em>t</em>-test). We also performed a paired <em>t</em>-test between the real data and the simulated data, which showed significant differences between each other (<em>P</em> &lt; 2.2e-16). This result indicates that when the GC-content is the same, the amino acid usage or codon usage is not constant in random sequences. In addition, we observed a correlation between amino acid usage and GC-content based on different GC<sub>syn</sub> groups in random sequences as well (<a href="http://www.g3journal.org/content/suppl/2015/08/05/g3.115.019877.DC1/FigureS6.pdf">Figure S6C</a>). Although the pattern of amino acid usage and GC-content appeared similar between the genomic data and simulated data, we found significant differences between them for the majority of the amino acids (<a href="http://www.g3journal.org/content/suppl/2015/08/05/g3.115.019877.DC1/TableS7.pdf">Table S7</a>). This simulation serves as a good control to our analysis of the coding regions and certifies our results: codon usage frequencies are nearly constant in regions with similar GC-content; and that intrinsically harbors some biological significance.</p></div><div id="sec-12" class="subsection"><h3>The influence of GC-content on amino acid composition</h3><p id="p-29">Although GC-content is considered widely to determine amino acid usage, it is interesting to quantitatively dissect how that changes the amino acid composition from one region to another. Besides, GC-content also would influence which synonymous codon for a certain amino acid would be used. Either GC-rich or AT-rich synonymous codons would be chosen on the basis of the variation of GC-content. To examine the effects of GC-content, we divided the contributions of GC-content’s effects into two parts: on the use of different amino acids or their synonymous codons that bears different GC fractions. Here we created an equation to quantify these two parameters due to the change of regional GC-content.<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2027/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2027/embed/mml-math-2.gif"/></noscript></span></span>ΔGC indicates the increased GC-content of an amino acid from low to high GC-content regions (<em>e.g.</em>, GC-content from 0.30−0.31 to 0.79−0.80), which could be split into two components: the amino acid’s increased usage (ΔA) and its increased GC<sub>syn</sub> (ΔGC<sub>syn</sub>) due to the usage of different synonymous codons. The first component reflects the influence of GC-content variation on amino acid usage. The second component indicates the influence on GC<sub>syn</sub> change (see details in <em>Materials and Methods</em>). Here only the four high-GC<sub>syn</sub> amino acids are taken into account because these four amino acids account for most of the increased GC-bases from low- to high-GC regions (<a id="xref-fig-2-11" class="xref-fig" href="#F2">Figure 2E</a>), and the usage of each increases constantly as the regional GC-content increases (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4A</a>). The total increased GC-content is 0.490 for all the 20 amino acids between the regions with 0.30−0.31 and 0.79−0.80 GC-content. We show that the influence on the use of four high-GC<sub>syn</sub> amino acids is 0.376, which accounts for 76.7% of the total increased GC-content whereas the influence on synonymous codons accounts for only 8.2% (<a id="xref-fig-2-12" class="xref-fig" href="#F2">Figure 2F</a>; <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). Similarly, it is also possible to calculate the influence of low-GC<sub>syn</sub> amino acids to the increase of AT-bases between the regions with 0.79−0.80 and 0.30−0.31 GC-content (<a href="http://www.g3journal.org/content/suppl/2015/08/05/g3.115.019877.DC1/TableS4.pdf">Table S4</a>). All these results suggest that high GC-content CDS produces genes with high GC<sub>syn</sub> amino acids whereas low GC-content sequence encodes genes with low GC<sub>syn</sub> amino acids. Therefore, GC-content not only determines the codon/amino acid usage but also has great influence on amino acid composition, especially the extreme-GC<sub>syn</sub> amino acids in a region.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466780/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466780/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">The effects of GC-content by amino acids from low to high GC regions</span><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-13"><h2 class="">Discussion</h2><p id="p-30">All of our findings initiated from a simple parameter, the GC-content of all the synonymous codons for an amino acid (GC<sub>syn</sub>). On the basis of GC<sub>syn</sub>, our results show that the 20 amino acids fall into three distinct groups. The usage of amino acids from different groups is distinct from each other. This result further led us to find constant amino acid usage when GC-content is determined. Therefore, GC-content reveals the profound influence on genomic architecture and divergence. In this point of view, it will shed light on our understanding of fundamental biological questions.</p><p id="p-31">Genetic codons have long been considered to bear no inherent advantages (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Francino and Ochman 1999</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Yang and Nielsen 2008</a>) and resulted from a “frozen accident” (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Crick 1968</a>). Codon reassignment could change the amino acid sequences of most proteins, resulting in a destructive impact on the organism. However, these ideas are inconsistent with the patterns observed in this report. The association between usage and synonymous codon numbers of amino acids with different GC<sub>syn</sub> suggests that a specific codon reassignment could be associated with the usage variation of both the codon and amino acid. In this light, the least used codons presumably have a greater potential to be reassigned because this type of coding changes would have the least impact on the genome. Indeed, several lines of evidence support this point. First, the majority of nonuniversal codons in mitochondria genomes, ATA, AGA, AGG, and CTA (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Li and Tzagoloff 1979</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Barrell <em>et al.</em> 1979</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Andersson and Kurland 1991</a>; <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Knight <em>et al.</em> 2001</a>), include four of the eight least used codons (<a href="http://www.g3journal.org/content/suppl/2015/08/05/g3.115.019877.DC1/TableS2.pdf">Table S2</a>) in our analyzed dataset. These four codons encode Ile, Arg, and Leu, which are exactly the same amino acids with the lowest GC<sub>syn</sub> in each distinct GC<sub>syn</sub> group. Second, Trp and Met that are each encoded by a single codon have an unusual GC<sub>syn</sub>, and are used two times more frequently in mitochondrial genomes than in nucleic genomes (<a href="http://www.g3journal.org/content/suppl/2015/08/05/g3.115.019877.DC1/TableS5.pdf">Table S5</a>). Third, the bacterial <em>Mycoplasma</em> species translate UGA as Trp, which is a property also observed in animal mitochondria. Trp is among the least used amino acids and possesses a GC<sub>syn</sub> different from other amino acids. These evidences confirm that GC<sub>syn</sub>-based association has a great influence on the reassignment of specific codons and, subsequently, the encoded amino acids.</p><p id="p-32">Besides the influence on reassignment, the association between usage pattern and synonymous codon numbers of amino acids with different GC<sub>syn</sub> also may suggest an adaptive pattern in the course of evolution. For example, amino acids (Arg, Leu, or Ile) with a relatively low GC<sub>syn</sub> in each group also have the most synonymous codons within that group, some of which are the least used codons. If these atypically used codon(s) are ignored, each of the three GC<sub>syn</sub> becomes greater and is more similar to others in the corresponding group. Trp and Met both have unique GC<sub>syn</sub> and also are used infrequently, suggesting that the single codon may only fit to the least-used amino acids because they do not have synonymous codons and consequently their GC<sub>syn</sub> are fixed across genomic regions. In addition to these five amino acids, the usage of the eight less-variable amino acids with GC<sub>syn</sub> of 0.5 is positively correlated with their codon numbers, which also provides evidence for adaptive evolution.</p><p id="p-33">The most striking finding in our study is constant codon and amino acid usage in the regions with similar GC-content even though the organisms selected for genome comparisons in our study are widely represented. In other words, this conclusion is beyond the phylogenetic constraints and demonstrates a unique spectrum of amino acid usage that applies to all species, which is extremely critical to understanding amino acid usage and nucleotide sequences in poorly characterized taxa. In addition to this implication, this observation also can lead to an assumption that the fitness of the organism would be greater if the usage of codons and amino acids is in concordance with underlying regulations at the level of GC-content. A recent study in <em>Escherichia coli</em> using <em>GFP</em> genes varying in GC-content directly supports our assumption. In that study, <em>E coli</em> strains harboring GC-rich versions of <em>GFP</em> genes display a greater growth rate (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Raghavan <em>et al.</em> 2012</a>). We further analyzed their experimental data and were interested to discover that the strains showing a greater growth rate are the ones harboring <em>GFP</em> genes with a similar GC-content to that in the <em>E. coli</em>’s overall CDS (<a href="http://www.g3journal.org/content/suppl/2015/08/05/g3.115.019877.DC1/TableS6.pdf">Table S6</a>). The more GC-content in <em>GFP</em> deviated from that in intrinsic CDSs (0.519), the lower the growth rate displayed. The gene harboring the same GC-content as the genomic CDS might give rise to more efficient/accurate transcription and translation, resulting in greater fitness. The authors explain their result as a selective force driving genes toward greater GC-content, which is contrary to the pervasive mutational bias toward AT (<a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Raghavan <em>et al.</em> 2012</a>). We suggest an alternative hypothesis in which selection is strong when codon bias levels deviate from the equilibrium level but weak when it reaches equilibrium. Additional experiments are required to test our equilibrium model.</p><p id="p-34">Another critical question we want to address is whether the GC-content variation across the genome is due to intrinsic mechanisms or external factors, and whether it is a result of a neutral process or selection (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Eyre-Walker and Hurst 2001</a>). As is well known, the genomic organization of GC-rich regions differs significantly from that of GC-poor regions, including gene densities (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Lander <em>et al.</em> 2001</a>), CpG island densities (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Cross <em>et al.</em> 2000</a>), and repetitive DNA elements densities (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bernardi 2000</a>). Genes in GC-rich regions also have been argued to yield different proteins from those in GC-poor regions (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">D’Onofrio <em>et al.</em> 1991</a>), and genes in GC-rich regions generally were found to have increased expression levels especially in mammalian cells. We provide a different approach to explain those findings. Using a formula to measure the influence of regional GC-content change on amino acid usage and GC<sub>syn</sub> separately and quantitatively, we show in our results that GC-content produces the organization of high-, low-, and intermediate- GC<sub>syn</sub> amino acids, particularly those with extreme GC<sub>syn</sub> in any region. This may be related with gene functions. <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Tatarinova <em>et al.</em> (2010)</a> demonstrated that GC-content is greater for genes from electron transport or energy pathways, response to abiotic or biotic stimuli, response to stress, transcription and signal transduction. Accompanying the requirement for GC-content variation, the synonymous codons have to be changed toward a higher (or lower) GC state. The usage changes in the amino acids by the gene’s GC-content also can facilitate and perhaps modulate the formation of genomic structures. Our study further suggests why a particular synonymous codon may be preferentially used. Due to functional requirements, the usage of amino acids in genes could be predetermined. Given that, the different GC-content of synonymous codons makes it possible to adjust more efficiently to the regional GC changes. A close correlation was previously revealed between exon GC-content and its surrounding intron GC-composition (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Zhu <em>et al.</em> 2009</a>). Our observation of functionally associated exon GC-composition provides good evidence for the influence of GC-content from exons to introns, and it will greatly further our understanding of the evolution of gene architecture.</p></div><div class="section" id="sec-14"></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-35">We thank Hitoshi Araki for stimulating discussions during the study. We also thank Daniel Hartl, Danna Eickbush, Hitoshi Araki, Marjorie Lienard, the editor, and two un-known reviewers for helpful comments on the manuscript. This study was supported by National Natural Science Foundation of China (91331205 and 31571267) and PCSIRT (IRT1020) to D.T.</p><p id="p-36">D.T designed the analysis. J.L. and J.Z. performed the research and analysis and wrote the manuscript. Y.W. helped to analyze the data. D.T. and S.Y. helped with the discussion and worked on the manuscript.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-supplementary-material" id="fn-2"><p id="p-37">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019877/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.019877/-/DC1</a></p></li><li class="fn" id="fn-3"><p id="p-38"><em>Communicating editor: E. Akhunov</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2015-06-19"><span class="received-label">Received </span>June 19, 2015.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2015-08-02"><span class="accepted-label">Accepted </span>August 2, 2015.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2015 Li <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2027.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Andersson</span>  <span class="cit-name-given-names">G. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kurland</span>  <span class="cit-name-given-names">C. G.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">An extreme codon preference strategy: codon reassignment.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">530</span>–<span class="cit-lpage">544</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DAndersson%26rft.auinit1%253DG.%26rft.volume%253D8%26rft.issue%253D4%26rft.spage%253D530%26rft.epage%253D544%26rft.atitle%253DAn%2Bextreme%2Bcodon%2Bpreference%2Bstrategy%253A%2Bcodon%2Breassignment%26rft_id%253Dinfo%253Apmid%252F1921708%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjc6IjgvNC81MzAiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzEwLzIwMjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2027.2" data-doi="10.1038/282189a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barrell</span>  <span class="cit-name-given-names">B. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bankier</span>  <span class="cit-name-given-names">A. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Drouin</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1979</span> <span class="cit-article-title">A different genetic code in human mitochondria.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">282</span>: <span class="cit-fpage">189</span>–<span class="cit-lpage">194</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DBarrell%26rft.auinit1%253DB.%2BG.%26rft.volume%253D282%26rft.issue%253D5735%26rft.spage%253D189%26rft.epage%253D194%26rft.atitle%253DA%2Bdifferent%2Bgenetic%2Bcode%2Bin%2Bhuman%2Bmitochondria.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F282189a0%26rft_id%253Dinfo%253Apmid%252F226894%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/282189a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=226894&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1979HT17500033&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2027.3" data-doi="10.1093/nar/22.9.1705"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Berkhout</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Hemert</span>  <span class="cit-name-given-names">F. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">The unusual nucleotide content of the HIV RNA genome results in a biased amino acid composition of HIV proteins.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">1705</span>–<span class="cit-lpage">1711</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DBerkhout%26rft.auinit1%253DB.%26rft.volume%253D22%26rft.issue%253D9%26rft.spage%253D1705%26rft.epage%253D1711%26rft.atitle%253DThe%2Bunusual%2Bnucleotide%2Bcontent%2Bof%2Bthe%2BHIV%2BRNA%2Bgenome%2Bresults%2Bin%2Ba%2Bbiased%2Bamino%2Bacid%2Bcomposition%2Bof%2BHIV%2Bproteins%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F22.9.1705%26rft_id%253Dinfo%253Apmid%252F8202375%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjk6IjIyLzkvMTcwNSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTAvMjAyNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2027.4" data-doi="10.1016/S0378-1119(00)00441-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bernardi</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">The compositional evolution of vertebrate genomes.</span> <abbr class="cit-jnl-abbrev">Gene</abbr> <span class="cit-vol">259</span>: <span class="cit-fpage">31</span>–<span class="cit-lpage">43</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.stitle%253DGene%26rft.aulast%253DBernardi%26rft.auinit1%253DG.%26rft.volume%253D259%26rft.issue%253D1-2%26rft.spage%253D31%26rft.epage%253D43%26rft.atitle%253DThe%2Bcompositional%2Bevolution%2Bof%2Bvertebrate%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0378-1119%252800%252900441-8%26rft_id%253Dinfo%253Apmid%252F11163959%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0378-1119(00)00441-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11163959&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000166338300006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2027.5" data-doi="10.1007/BF02099946"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bernardi</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernardi</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span> <span class="cit-article-title">Compositional constraints and genome evolution.</span> <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">11</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DBernardi%26rft.auinit1%253DG.%26rft.volume%253D24%26rft.issue%253D1-2%26rft.spage%253D1%26rft.epage%253D11%26rft.atitle%253DCompositional%2Bconstraints%2Band%2Bgenome%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF02099946%26rft_id%253Dinfo%253Apmid%252F3104608%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF02099946&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3104608&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1986F784400001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2027.6" data-doi="10.1371/journal.pone.0069878"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bohlin</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brynildsrud</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vesth</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Skjerve</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ussery</span>  <span class="cit-name-given-names">D. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Amino acid usage is asymmetrically biased in AT- and GC-rich microbial genomes.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e69878</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De69878%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0069878%26rft_id%253Dinfo%253Apmid%252F23922837%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0069878&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23922837&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2027.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bulmer</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">The selection-mutation-drift theory of synonymous codon usage.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">129</span>: <span class="cit-fpage">897</span>–<span class="cit-lpage">907</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DBulmer%26rft.auinit1%253DM.%26rft.volume%253D129%26rft.issue%253D3%26rft.spage%253D897%26rft.epage%253D907%26rft.atitle%253DThe%2Bselection-mutation-drift%2Btheory%2Bof%2Bsynonymous%2Bcodon%2Busage.%26rft_id%253Dinfo%253Apmid%252F1752426%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTI5LzMvODk3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMC8yMDI3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2027.8" data-doi="10.1038/nrg1770"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chamary</span>  <span class="cit-name-given-names">J. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parmley</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hurst</span>  <span class="cit-name-given-names">L. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Hearing silence: non-neutral evolution at synonymous sites in mammals.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">98</span>–<span class="cit-lpage">108</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DChamary%26rft.auinit1%253DJ.%2BV.%26rft.volume%253D7%26rft.issue%253D2%26rft.spage%253D98%26rft.epage%253D108%26rft.atitle%253DHearing%2Bsilence%253A%2Bnon-neutral%2Bevolution%2Bat%2Bsynonymous%2Bsites%2Bin%2Bmammals.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg1770%26rft_id%253Dinfo%253Apmid%252F16418745%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg1770&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16418745&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000234714000012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2027.9" data-doi="10.1073/pnas.0307827100"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hottes</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shapiro</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McAdams</span>  <span class="cit-name-given-names">H. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Codon usage between genomes is constrained by genome-wide mutational processes.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">3480</span>–<span class="cit-lpage">3485</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0307827100%26rft_id%253Dinfo%253Apmid%252F14990797%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTAxLzEwLzM0ODAiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzEwLzIwMjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2027.10" data-doi="10.1016/S0378-1119(97)00671-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chiapello</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lisacek</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Caboche</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hénaut</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Codon usage and gene function are related in sequences of <em>Arabidopsis thaliana</em>.</span> <abbr class="cit-jnl-abbrev">Gene</abbr> <span class="cit-vol">209</span>: <span class="cit-fpage">GC1</span>–<span class="cit-lpage">GC38</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.stitle%253DGene%26rft.aulast%253DChiapello%26rft.auinit1%253DH.%26rft.volume%253D209%26rft.issue%253D1-2%26rft.spage%253DGC1%26rft.epage%253DGC38%26rft.atitle%253DCodon%2Busage%2Band%2Bgene%2Bfunction%2Bare%2Brelated%2Bin%2Bsequences%2Bof%2BArabidopsis%2Bthaliana.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0378-1119%252897%252900671-9%26rft_id%253Dinfo%253Apmid%252F9583944%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0378-1119(97)00671-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9583944&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000072941100031&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2027.11" data-doi="10.1007/BF00160475"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Collins</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jukes</span>  <span class="cit-name-given-names">T. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Relationship between G + C in silent sites of codons and amino acid composition of human proteins.</span> <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">201</span>–<span class="cit-lpage">213</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DCollins%26rft.auinit1%253DD.%2BW.%26rft.volume%253D36%26rft.issue%253D3%26rft.spage%253D201%26rft.epage%253D213%26rft.atitle%253DRelationship%2Bbetween%2BG%2B%252B%2BC%2Bin%2Bsilent%2Bsites%2Bof%2Bcodons%2Band%2Bamino%2Bacid%2Bcomposition%2Bof%2Bhuman%2Bproteins.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00160475%26rft_id%253Dinfo%253Apmid%252F8483158%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00160475&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8483158&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1993KN94000001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2027.12" data-doi="10.1016/0022-2836(68)90392-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crick</span>  <span class="cit-name-given-names">F. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1968</span> <span class="cit-article-title">The origin of the genetic code.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">367</span>–<span class="cit-lpage">379</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DCrick%26rft.auinit1%253DF.%2BH.%26rft.volume%253D38%26rft.issue%253D3%26rft.spage%253D367%26rft.epage%253D379%26rft.atitle%253DThe%2Borigin%2Bof%2Bthe%2Bgenetic%2Bcode.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0022-2836%252868%252990392-6%26rft_id%253Dinfo%253Apmid%252F4887876%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0022-2836(68)90392-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=4887876&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1968C697500009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2027.13" data-doi="10.1007/s003350010071"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cross</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">V. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simmen</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bickmore</span>  <span class="cit-name-given-names">W. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maroon</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">CpG island libraries from human chromosomes 18 and 22: landmarks for novel genes.</span> <abbr class="cit-jnl-abbrev">Mamm. Genome Off. J. Int. Mamm. Genome Soc.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">373</span>–<span class="cit-lpage">383</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMamm.%2BGenome%2BOff.%2BJ.%2BInt.%2BMamm.%2BGenome%2BSoc.%26rft.volume%253D11%26rft.spage%253D373%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs003350010071%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s003350010071&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2027.14" data-doi="10.1007/BF02102652"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">D’Onofrio</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mouchiroud</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aïssani</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gautier</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernardi</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Correlations between the compositional properties of human genes, codon usage, and amino acid composition of proteins.</span> <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">504</span>–<span class="cit-lpage">510</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DD%2527Onofrio%26rft.auinit1%253DG.%26rft.volume%253D32%26rft.issue%253D6%26rft.spage%253D504%26rft.epage%253D510%26rft.atitle%253DCorrelations%2Bbetween%2Bthe%2Bcompositional%2Bproperties%2Bof%2Bhuman%2Bgenes%252C%2Bcodon%2Busage%252C%2Band%2Bamino%2Bacid%2Bcomposition%2Bof%2Bproteins.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF02102652%26rft_id%253Dinfo%253Apmid%252F1908021%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF02102652&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1908021&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1991FN31800007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2027.15" data-doi="10.1016/S0378-1119(99)00257-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">D’Onofrio</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jabbari</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Musto</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernardi</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">The correlation of protein hydropathy with the base composition of coding sequences.</span> <abbr class="cit-jnl-abbrev">Gene</abbr> <span class="cit-vol">238</span>: <span class="cit-fpage">3</span>–<span class="cit-lpage">14</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.stitle%253DGene%26rft.aulast%253DD%2527Onofrio%26rft.auinit1%253DG.%26rft.volume%253D238%26rft.issue%253D1%26rft.spage%253D3%26rft.epage%253D14%26rft.atitle%253DThe%2Bcorrelation%2Bof%2Bprotein%2Bhydropathy%2Bwith%2Bthe%2Bbase%2Bcomposition%2Bof%2Bcoding%2Bsequences.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0378-1119%252899%252900257-7%26rft_id%253Dinfo%253Apmid%252F10570978%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0378-1119(99)00257-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10570978&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000083005400001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2027.16" data-doi="10.1093/nar/gkg897"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dos Reis</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wernisch</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Savva</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Unexpected correlations between gene expression and codon usage bias from microarray data for the whole <em>Escherichia coli</em> K-12 genome.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">6976</span>–<span class="cit-lpage">6985</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253Ddos%2BReis%26rft.auinit1%253DM.%26rft.volume%253D31%26rft.issue%253D23%26rft.spage%253D6976%26rft.epage%253D6985%26rft.atitle%253DUnexpected%2Bcorrelations%2Bbetween%2Bgene%2Bexpression%2Band%2Bcodon%2Busage%2Bbias%2Bfrom%2Bmicroarray%2Bdata%2Bfor%2Bthe%2Bwhole%2BEscherichia%2Bcoli%2BK-12%2Bgenome%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkg897%26rft_id%253Dinfo%253Apmid%252F14627830%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIzMS8yMy82OTc2IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMC8yMDI3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2027.17" data-doi="10.1016/S0959-437X(02)00353-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Duret</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Evolution of synonymous codon usage in metazoans.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Genet. Dev.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">640</span>–<span class="cit-lpage">649</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bgenetics%2B%2526%2Bdevelopment%26rft.stitle%253DCurr%2BOpin%2BGenet%2BDev%26rft.aulast%253DDuret%26rft.auinit1%253DL.%26rft.volume%253D12%26rft.issue%253D6%26rft.spage%253D640%26rft.epage%253D649%26rft.atitle%253DEvolution%2Bof%2Bsynonymous%2Bcodon%2Busage%2Bin%2Bmetazoans.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0959-437X%252802%252900353-2%26rft_id%253Dinfo%253Apmid%252F12433576%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0959-437X(02)00353-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12433576&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000179003900003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2027.18" data-doi="10.1073/pnas.96.8.4482"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Duret</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mouchiroud</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Expression pattern and, surprisingly, gene length shape codon usage in <em>Caenorhabditis</em>, <em>Drosophila</em>, and <em>Arabidopsis</em>.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">96</span>: <span class="cit-fpage">4482</span>–<span class="cit-lpage">4487</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DDuret%26rft.auinit1%253DL.%26rft.volume%253D96%26rft.issue%253D8%26rft.spage%253D4482%26rft.epage%253D4487%26rft.atitle%253DExpression%2Bpattern%2Band%252C%2Bsurprisingly%252C%2Bgene%2Blength%2Bshape%2Bcodon%2Busage%2Bin%2BCaenorhabditis%252C%2BDrosophila%252C%2Band%2BArabidopsis%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.96.8.4482%26rft_id%253Dinfo%253Apmid%252F10200288%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI5Ni84LzQ0ODIiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzEwLzIwMjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2027.19" data-doi="10.1093/oxfordjournals.molbev.a025646"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eyre-Walker</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Synonymous codon bias is related to gene length in <em>Escherichia coli</em>: selection for translational accuracy?</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">864</span>–<span class="cit-lpage">872</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DEyre-Walker%26rft.auinit1%253DA%26rft.volume%253D13%26rft.issue%253D6%26rft.spage%253D864%26rft.epage%253D872%26rft.atitle%253DSynonymous%2Bcodon%2Bbias%2Bis%2Brelated%2Bto%2Bgene%2Blength%2Bin%2BEscherichia%2Bcoli%253A%2Bselection%2Bfor%2Btranslational%2Baccuracy%253F%26rft_id%253Dinfo%253Adoi%252F10.1093%252Foxfordjournals.molbev.a025646%26rft_id%253Dinfo%253Apmid%252F8754221%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjEzLzYvODY0IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMC8yMDI3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2027.20" data-doi="10.1038/35080577"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eyre-Walker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hurst</span>  <span class="cit-name-given-names">L. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">The evolution of isochores.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">549</span>–<span class="cit-lpage">555</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DEyre-Walker%26rft.auinit1%253DA.%26rft.volume%253D2%26rft.issue%253D7%26rft.spage%253D549%26rft.epage%253D555%26rft.atitle%253DThe%2Bevolution%2Bof%2Bisochores.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F35080577%26rft_id%253Dinfo%253Apmid%252F11433361%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/35080577&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11433361&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000169681600015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2027.21" data-doi="10.1038/21804"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Francino</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ochman</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Isochores result from mutation not selection.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">400</span>: <span class="cit-fpage">30</span>–<span class="cit-lpage">31</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DFrancino%26rft.auinit1%253DM.%2BP.%26rft.volume%253D400%26rft.issue%253D6739%26rft.spage%253D30%26rft.epage%253D31%26rft.atitle%253DIsochores%2Bresult%2Bfrom%2Bmutation%2Bnot%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F21804%26rft_id%253Dinfo%253Apmid%252F10403245%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/21804&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10403245&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000081255700034&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2027.22" data-doi="10.1093/nar/10.22.7055"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gouy</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gautier</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">1982</span> <span class="cit-article-title">Codon usage in bacteria: correlation with gene expressivity.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">7055</span>–<span class="cit-lpage">7074</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DGouy%26rft.auinit1%253DM.%26rft.volume%253D10%26rft.issue%253D22%26rft.spage%253D7055%26rft.epage%253D7074%26rft.atitle%253DCodon%2Busage%2Bin%2Bbacteria%253A%2Bcorrelation%2Bwith%2Bgene%2Bexpressivity%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F10.22.7055%26rft_id%253Dinfo%253Apmid%252F6760125%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIxMC8yMi83MDU1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMC8yMDI3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2027.23" data-doi="10.1093/nar/8.1.197-c"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grantham</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gautier</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gouy</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mercier</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pavé</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1980</span> <span class="cit-article-title">Codon catalog usage and the genome hypothesis.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">r49</span>–<span class="cit-lpage">r62</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.volume%253D8%26rft.issue%253D1%26rft.spage%253Dr49%26rft.epage%253D62%26rft.atitle%253DCodon%2Bcatalog%2Busage%2Band%2Bthe%2Bgenome%2Bhypothesis%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F8.1.197-c%26rft_id%253Dinfo%253Apmid%252F6986610%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/8.1.197-c&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=6986610&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1980JC46600018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2027.24" data-doi="10.1023/A:1017028102013"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hewett-Emmett</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Directional mutational pressure affects the amino acid composition and hydrophobicity of proteins in bacteria.</span> <abbr class="cit-jnl-abbrev">Genetica</abbr> <span class="cit-vol">102–103</span>: <span class="cit-fpage">383</span>–<span class="cit-lpage">391</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetica%26rft.volume%253D102103%26rft.spage%253D383%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1017028102013%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/A:1017028102013&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2027.25" data-doi="10.1016/j.tibtech.2004.04.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gustafsson</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Govindarajan</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Minshull</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Codon bias and heterologous protein expression.</span> <abbr class="cit-jnl-abbrev">Trends Biotechnol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">346</span>–<span class="cit-lpage">353</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bbiotechnology%26rft.stitle%253DTrends%2BBiotechnol%26rft.aulast%253DGustafsson%26rft.auinit1%253DC.%26rft.volume%253D22%26rft.issue%253D7%26rft.spage%253D346%26rft.epage%253D353%26rft.atitle%253DCodon%2Bbias%2Band%2Bheterologous%2Bprotein%2Bexpression.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tibtech.2004.04.006%26rft_id%253Dinfo%253Apmid%252F15245907%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tibtech.2004.04.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15245907&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000222712000008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2027.26" data-doi="10.1146/annurev.genet.42.110807.091442"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hershberg</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petrov</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Selection on codon bias.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genet.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">287</span>–<span class="cit-lpage">299</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenet%26rft.aulast%253DHershberg%26rft.auinit1%253DR.%26rft.volume%253D42%26rft.spage%253D287%26rft.epage%253D299%26rft.atitle%253DSelection%2Bon%2Bcodon%2Bbias.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.genet.42.110807.091442%26rft_id%253Dinfo%253Apmid%252F18983258%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.genet.42.110807.091442&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18983258&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261767000014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2027.27" data-doi="10.1371/journal.pgen.1001107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hildebrand</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meyer</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eyre-Walker</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Evidence of selection upon genomic GC-content in bacteria.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e1001107</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DHildebrand%26rft.volume%253D6%26rft.issue%253D9%26rft.spage%253De1001107%26rft.atitle%253DEvidence%2Bof%2Bselection%2Bupon%2Bgenomic%2BGC-content%2Bin%2Bbacteria.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1001107%26rft_id%253Dinfo%253Apmid%252F20838593%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1001107&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20838593&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2027.28" data-doi="10.1016/0022-2836(81)90003-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ikemura</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">1981</span> <span class="cit-article-title">Correlation between the abundance of <em>Escherichia coli</em> transfer RNAs and the occurrence of the respective codons in its protein genes: a proposal for a synonymous codon choice that is optimal for the <em>E. coli</em> translational system.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">151</span>: <span class="cit-fpage">389</span>–<span class="cit-lpage">409</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DIkemura%26rft.auinit1%253DT.%26rft.volume%253D151%26rft.issue%253D3%26rft.spage%253D389%26rft.epage%253D409%26rft.atitle%253DCorrelation%2Bbetween%2Bthe%2Babundance%2Bof%2BEscherichia%2Bcoli%2Btransfer%2BRNAs%2Band%2Bthe%2Boccurrence%2Bof%2Bthe%2Brespective%2Bcodons%2Bin%2Bits%2Bprotein%2Bgenes%253A%2Ba%2Bproposal%2Bfor%2Ba%2Bsynonymous%2Bcodon%2Bchoice%2Bthat%2Bis%2Boptimal%2Bfor%2Bthe%2BE.%2Bcoli%2Btranslational%2Bsystem.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0022-2836%252881%252990003-6%26rft_id%253Dinfo%253Apmid%252F6175758%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0022-2836(81)90003-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=6175758&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1981MJ92600003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2027.29" data-doi="10.1006/jmbi.1996.0528"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Karlin</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mrázek</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">What drives codon choices in human genes?</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">262</span>: <span class="cit-fpage">459</span>–<span class="cit-lpage">472</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DKarlin%26rft.auinit1%253DS.%26rft.volume%253D262%26rft.issue%253D4%26rft.spage%253D459%26rft.epage%253D472%26rft.atitle%253DWhat%2Bdrives%2Bcodon%2Bchoices%2Bin%2Bhuman%2Bgenes%253F%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.1996.0528%26rft_id%253Dinfo%253Apmid%252F8893856%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.1996.0528&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8893856&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996VK74900007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2027.30" data-doi="10.1046/j.1365-2958.1998.01008.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Karlin</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mrázek</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Campbell</span>  <span class="cit-name-given-names">A. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Codon usages in different gene classes of the <em>Escherichia coli</em> genome.</span> <abbr class="cit-jnl-abbrev">Mol. Microbiol.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">1341</span>–<span class="cit-lpage">1355</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bmicrobiology%26rft.stitle%253DMol%2BMicrobiol%26rft.aulast%253DKarlin%26rft.auinit1%253DS.%26rft.volume%253D29%26rft.issue%253D6%26rft.spage%253D1341%26rft.epage%253D1355%26rft.atitle%253DCodon%2Busages%2Bin%2Bdifferent%2Bgene%2Bclasses%2Bof%2Bthe%2BEscherichia%2Bcoli%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-2958.1998.01008.x%26rft_id%253Dinfo%253Apmid%252F9781873%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-2958.1998.01008.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9781873&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000076116100003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-other" id="cit-5.10.2027.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Knight</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Freeland</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Landweber</span>  <span class="cit-name-given-names">L. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">A simple model based on mutation and selection explains trends in codon and amino-acid usage and GC composition within and across genomes.</span> <span class="cit-source">Genome Biol.</span> <span class="cit-vol">2</span>: <span class="cit-fpage">RESEARCH0010</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DKnight%26rft.auinit1%253DR.%2BD.%26rft.volume%253D2%26rft.issue%253D4%26rft.spage%253DRESEARCH0010%26rft.epage%253DRESEARCH0010%26rft.atitle%253DA%2Bsimple%2Bmodel%2Bbased%2Bon%2Bmutation%2Band%2Bselection%2Bexplains%2Btrends%2Bin%2Bcodon%2Band%2Bamino-acid%2Busage%2Band%2BGC%2Bcomposition%2Bwithin%2Band%2Bacross%2Bgenomes.%26rft_id%253Dinfo%253Apmid%252F11305938%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=11305938&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2027.32" data-doi="10.1371/journal.pbio.0040180"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kudla</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lipinski</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Caffin</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Helwak</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zylicz</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">High guanine and cytosine content increases mRNA levels in mammalian cells.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e180</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DKudla%26rft.auinit1%253DG.%26rft.volume%253D4%26rft.issue%253D6%26rft.spage%253De180%26rft.epage%253De180%26rft.atitle%253DHigh%2Bguanine%2Band%2Bcytosine%2Bcontent%2Bincreases%2BmRNA%2Blevels%2Bin%2Bmammalian%2Bcells.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0040180%26rft_id%253Dinfo%253Apmid%252F16700628%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0040180&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16700628&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2027.33" data-doi="10.1093/nar/27.7.1642"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lafay</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lloyd</span>  <span class="cit-name-given-names">A. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McLean</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Devine</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sharp</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Proteome composition and codon usage in spirochaetes: species-specific and DNA strand-specific mutational biases.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">1642</span>–<span class="cit-lpage">1649</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F27.7.1642%26rft_id%253Dinfo%253Apmid%252F10075995%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjk6IjI3LzcvMTY0MiI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTAvMjAyNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2027.34" data-doi="10.1038/35057062"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lander</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Linton</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Birren</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nusbaum</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zody</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Initial sequencing and analysis of the human genome.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">409</span>: <span class="cit-fpage">860</span>–<span class="cit-lpage">921</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLander%26rft.auinit1%253DE.%2BS.%26rft.volume%253D409%26rft.issue%253D6822%26rft.spage%253D860%26rft.epage%253D921%26rft.atitle%253DInitial%2Bsequencing%2Band%2Banalysis%2Bof%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F35057062%26rft_id%253Dinfo%253Apmid%252F11237011%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/35057062&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11237011&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000166938800058&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2027.35" data-doi="10.1016/0092-8674(79)90352-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tzagoloff</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1979</span> <span class="cit-article-title">Assembly of the mitochondrial membrane system: sequences of yeast mitochondrial valine and an unusual threonine tRNA gene.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">47</span>–<span class="cit-lpage">53</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DLi%26rft.auinit1%253DM.%26rft.volume%253D18%26rft.issue%253D1%26rft.spage%253D47%26rft.epage%253D53%26rft.atitle%253DAssembly%2Bof%2Bthe%2Bmitochondrial%2Bmembrane%2Bsystem%253A%2Bsequences%2Bof%2Byeast%2Bmitochondrial%2Bvaline%2Band%2Ban%2Bunusual%2Bthreonine%2BtRNA%2Bgene.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252879%252990352-0%26rft_id%253Dinfo%253Apmid%252F389433%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(79)90352-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=389433&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1979HL27000005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2027.36" data-doi="10.1016/j.biosystems.2004.10.008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dou</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ji</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xue</span>  <span class="cit-name-given-names">Q.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Synonymous codon usage and gene function are strongly related in <em>Oryza sativa</em>.</span> <abbr class="cit-jnl-abbrev">Biosystems</abbr> <span class="cit-vol">80</span>: <span class="cit-fpage">123</span>–<span class="cit-lpage">131</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiosystems%26rft.stitle%253DBiosystems%26rft.aulast%253DLiu%26rft.auinit1%253DQ.%26rft.volume%253D80%26rft.issue%253D2%26rft.spage%253D123%26rft.epage%253D131%26rft.atitle%253DSynonymous%2Bcodon%2Busage%2Band%2Bgene%2Bfunction%2Bare%2Bstrongly%2Brelated%2Bin%2BOryza%2Bsativa.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.biosystems.2004.10.008%26rft_id%253Dinfo%253Apmid%252F15823411%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.biosystems.2004.10.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15823411&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2027.37" data-doi="10.1016/S0378-1119(97)00403-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lobry</span>  <span class="cit-name-given-names">J. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Influence of genomic G+C content on average amino-acid composition of proteins from 59 bacterial species.</span> <abbr class="cit-jnl-abbrev">Gene</abbr> <span class="cit-vol">205</span>: <span class="cit-fpage">309</span>–<span class="cit-lpage">316</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.stitle%253DGene%26rft.aulast%253DLobry%26rft.auinit1%253DJ.%2BR.%26rft.volume%253D205%26rft.issue%253D1-2%26rft.spage%253D309%26rft.epage%253D316%26rft.atitle%253DInfluence%2Bof%2Bgenomic%2BG%252BC%2Bcontent%2Bon%2Baverage%2Bamino-acid%2Bcomposition%2Bof%2Bproteins%2Bfrom%2B59%2Bbacterial%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0378-1119%252897%252900403-4%26rft_id%253Dinfo%253Apmid%252F9461405%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0378-1119(97)00403-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9461405&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000071411800033&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2027.38" data-doi="10.1016/j.cub.2003.12.055"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Maside</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">A. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Selection on codon usage in <em>Drosophila americana</em>.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">150</span>–<span class="cit-lpage">154</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DMaside%26rft.auinit1%253DX.%26rft.volume%253D14%26rft.issue%253D2%26rft.spage%253D150%26rft.epage%253D154%26rft.atitle%253DSelection%2Bon%2Bcodon%2Busage%2Bin%2BDrosophila%2Bamericana.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2003.12.055%26rft_id%253Dinfo%253Apmid%252F14738738%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2003.12.055&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14738738&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000188504400027&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2027.39" data-doi="10.1093/nar/7.8.2431"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Miyata</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayashida</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yasunaga</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hasegawa</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1979</span> <span class="cit-article-title">The preferential codon usages in variable and constant regions of immunoglobulin genes are quite distinct from each other.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">2431</span>–<span class="cit-lpage">2438</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DMiyata%26rft.auinit1%253DT.%26rft.volume%253D7%26rft.issue%253D8%26rft.spage%253D2431%26rft.epage%253D2438%26rft.atitle%253DThe%2Bpreferential%2Bcodon%2Busages%2Bin%2Bvariable%2Band%2Bconstant%2Bregions%2Bof%2Bimmunoglobulin%2Bgenes%2Bare%2Bquite%2Bdistinct%2Bfrom%2Beach%2Bother%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F7.8.2431%26rft_id%253Dinfo%253Apmid%252F523321%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjg6IjcvOC8yNDMxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMC8yMDI3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2027.40" data-doi="10.1016/j.gene.2013.01.033"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nabiyouni</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prakash</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fedorov</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Vertebrate codon bias indicates a highly GC-rich ancestral genome.</span> <abbr class="cit-jnl-abbrev">Gene</abbr> <span class="cit-vol">519</span>: <span class="cit-fpage">113</span>–<span class="cit-lpage">119</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.volume%253D519%26rft.spage%253D113%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.gene.2013.01.033%26rft_id%253Dinfo%253Apmid%252F23376453%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.gene.2013.01.033&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23376453&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2027.41" data-doi="10.1073/pnas.0404957101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Plotkin</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robins</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Levine</span>  <span class="cit-name-given-names">A. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Tissue-specific codon usage and the expression of human genes.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">12588</span>–<span class="cit-lpage">12591</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0404957101%26rft_id%253Dinfo%253Apmid%252F15314228%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAxLzM0LzEyNTg4IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMC8yMDI3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2027.42"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Porter</span>  <span class="cit-name-given-names">T. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Correlation between codon usage, regional genomic nucleotide composition, and amino acid composition in the cytochrome P-450 gene superfamily.</span> <abbr class="cit-jnl-abbrev">Biochim. Biophys. Acta</abbr> <span class="cit-vol">1261</span>: <span class="cit-fpage">394</span>–<span class="cit-lpage">400</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochimica%2Bet%2BBiophysica%2BActa.%2BProtein%2BStructure%2Band%2BMolecular%2BEnzymology%26rft.stitle%253DBiochimica%2Bet%2BBiophysica%2BActa.%2BProtein%2BStructure%2Band%2BMolecular%2BEnzymology%26rft.aulast%253DPorter%26rft.auinit1%253DT.%2BD.%26rft.volume%253D1261%26rft.issue%253D3%26rft.spage%253D394%26rft.epage%253D400%26rft.atitle%253DCorrelation%2Bbetween%2Bcodon%2Busage%252C%2Bregional%2Bgenomic%2Bnucleotide%2Bcomposition%252C%2Band%2Bamino%2Bacid%2Bcomposition%2Bin%2Bthe%2Bcytochrome%2BP-450%2Bgene%2Bsuperfamily.%26rft_id%253Dinfo%253Apmid%252F7742368%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=7742368&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2027.43" data-doi="10.1073/pnas.1205683109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Raghavan</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kelkar</span>  <span class="cit-name-given-names">Y. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ochman</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A selective force favoring increased G+C content in bacterial genes.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">14504</span>–<span class="cit-lpage">14507</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1205683109%26rft_id%253Dinfo%253Apmid%252F22908296%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA5LzM2LzE0NTA0IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMC8yMDI3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2027.44" data-doi="10.1093/nar/14.13.5125"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sharp</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tuohy</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mosurski</span>  <span class="cit-name-given-names">K. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span> <span class="cit-article-title">Codon usage in yeast: cluster analysis clearly differentiates highly and lowly expressed genes.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">5125</span>–<span class="cit-lpage">5143</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DSharp%26rft.auinit1%253DP.%2BM.%26rft.volume%253D14%26rft.issue%253D13%26rft.spage%253D5125%26rft.epage%253D5143%26rft.atitle%253DCodon%2Busage%2Bin%2Byeast%253A%2Bcluster%2Banalysis%2Bclearly%2Bdifferentiates%2Bhighly%2Band%2Blowly%2Bexpressed%2Bgenes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F14.13.5125%26rft_id%253Dinfo%253Apmid%252F3526280%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIxNC8xMy81MTI1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMC8yMDI3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2027.45" data-doi="10.1093/oxfordjournals.molbev.a026257"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Singer</span>  <span class="cit-name-given-names">G. A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Nucleotide bias causes a genomewide bias in the amino acid composition of proteins.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">1581</span>–<span class="cit-lpage">1588</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DSinger%26rft.auinit1%253DG.%2BA.%2BC.%26rft.volume%253D17%26rft.issue%253D11%26rft.spage%253D1581%26rft.epage%253D1588%26rft.atitle%253DNucleotide%2BBias%2BCauses%2Ba%2BGenomewide%2BBias%2Bin%2Bthe%2BAmino%2BAcid%2BComposition%2Bof%2BProteins%26rft_id%253Dinfo%253Adoi%252F10.1093%252Foxfordjournals.molbev.a026257%26rft_id%253Dinfo%253Apmid%252F11070046%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjEwOiIxNy8xMS8xNTgxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMC8yMDI3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2027.46" data-doi="10.1073/pnas.47.8.1141"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sueoka</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">1961</span> <span class="cit-article-title">Correlation between base composition of deoxyribonucleic acid and amino acid composition of protein.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">1141</span>–<span class="cit-lpage">1149</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DSueoka%26rft.auinit1%253DN.%26rft.volume%253D47%26rft.issue%253D8%26rft.spage%253D1141%26rft.epage%253D1149%26rft.atitle%253DCORRELATION%2BBETWEEN%2BBASE%2BCOMPOSITION%2BOF%2BDEOXYRIBONUCLEIC%2BACID%2BAND%2BAMINO%2BACID%2BCOMPOSITION%2BOF%2BPROTEIN%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.47.8.1141%26rft_id%253Dinfo%253Apmid%252F16590864%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI0Ny84LzExNDEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzEwLzIwMjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2027.47" data-doi="10.1073/pnas.48.4.582"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sueoka</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">1962</span> <span class="cit-article-title">On the genetic basis of variation and heterogeneity of DNA base composition.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">582</span>–<span class="cit-lpage">592</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DSueoka%26rft.auinit1%253DN.%26rft.volume%253D48%26rft.issue%253D4%26rft.spage%253D582%26rft.epage%253D592%26rft.atitle%253DON%2BTHE%2BGENETIC%2BBASIS%2BOF%2BVARIATION%2BAND%2BHETEROGENEITY%2BOF%2BDNA%2BBASE%2BCOMPOSITION%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.48.4.582%26rft_id%253Dinfo%253Apmid%252F13918161%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo4OiI0OC80LzU4MiI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTAvMjAyNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2027.48" data-doi="10.1016/S0378-1119(00)00480-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sueoka</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawanishi</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">DNA G+C content of the third codon position and codon usage biases of human genes.</span> <abbr class="cit-jnl-abbrev">Gene</abbr> <span class="cit-vol">261</span>: <span class="cit-fpage">53</span>–<span class="cit-lpage">62</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.stitle%253DGene%26rft.aulast%253DSueoka%26rft.auinit1%253DN.%26rft.volume%253D261%26rft.issue%253D1%26rft.spage%253D53%26rft.epage%253D62%26rft.atitle%253DDNA%2BG%252BC%2Bcontent%2Bof%2Bthe%2Bthird%2Bcodon%2Bposition%2Band%2Bcodon%2Busage%2Bbiases%2Bof%2Bhuman%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0378-1119%252800%252900480-7%26rft_id%253Dinfo%253Apmid%252F11164037%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0378-1119(00)00480-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11164037&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000166759700008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2027.49" data-doi="10.1186/1471-2164-11-308"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tatarinova</span>  <span class="cit-name-given-names">T. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alexandrov</span>  <span class="cit-name-given-names">N. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bouck</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feldmann</span>  <span class="cit-name-given-names">K. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">GC3 biology in corn, rice, sorghum and other grasses.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">308</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DTatarinova%26rft.auinit1%253DT.%2BV.%26rft.volume%253D11%26rft.spage%253D308%26rft.epage%253D308%26rft.atitle%253DGC3%2Bbiology%2Bin%2Bcorn%252C%2Brice%252C%2Bsorghum%2Band%2Bother%2Bgrasses.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-11-308%26rft_id%253Dinfo%253Apmid%252F20470436%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-11-308&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20470436&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2027.50" data-doi="10.1111/j.1471-8286.2007.01821.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Villesen</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">FaBox: an online toolbox for fasta sequences.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Notes</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">965</span>–<span class="cit-lpage">968</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BNotes%26rft.volume%253D7%26rft.spage%253D965%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1471-8286.2007.01821.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1471-8286.2007.01821.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000250647700009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2027.51" data-doi="10.1016/S0923-2508(99)80043-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wilquet</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van de Casteele</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">The role of the codon first letter in the relationship between genomic GC content and protein amino acid composition.</span> <abbr class="cit-jnl-abbrev">Res. Microbiol.</abbr> <span class="cit-vol">150</span>: <span class="cit-fpage">21</span>–<span class="cit-lpage">32</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DResearch%2Bin%2BMicrobiology%2B%2528Paris%2529%26rft.stitle%253DResearch%2Bin%2BMicrobiology%2B%2528Paris%2529%26rft.aulast%253DWilquet%26rft.auinit1%253DV.%26rft.volume%253D150%26rft.issue%253D1%26rft.spage%253D21%26rft.epage%253D32%26rft.atitle%253DThe%2Brole%2Bof%2Bthe%2Bcodon%2Bfirst%2Bletter%2Bin%2Bthe%2Brelationship%2Bbetween%2Bgenomic%2BGC%2Bcontent%2Band%2Bprotein%2Bamino%2Bacid%2Bcomposition.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0923-2508%252899%252980043-6%26rft_id%253Dinfo%253Apmid%252F10096131%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0923-2508(99)80043-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10096131&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000080351500003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2027.52" data-doi="10.1093/molbev/msm284"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Mutation-selection models of codon substitution and their use to estimate selective strengths on codon usage.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">568</span>–<span class="cit-lpage">579</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsm284%26rft_id%253Dinfo%253Apmid%252F18178545%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjI1LzMvNTY4IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMC8yMDI3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2027.53" data-doi="10.1186/1471-2164-10-47"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">J.-Q.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Patterns of exon-intron architecture variation of genes in eukaryotic genomes.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">47</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DZhu%26rft.auinit1%253DL.%26rft.volume%253D10%26rft.spage%253D47%26rft.epage%253D47%26rft.atitle%253DPatterns%2Bof%2Bexon-intron%2Barchitecture%2Bvariation%2Bof%2Bgenes%2Bin%2Beukaryotic%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-10-47%26rft_id%253Dinfo%253Apmid%252F19166620%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-10-47&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19166620&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/5/10/2027.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/10/2021" title="High-Density Genotypes of Inbred Mouse Strains: Improved Power and Precision of Association Mapping" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/10/2037" title="Genome-Wide Linkage Analysis Identifies Loci for Physical Appearance Traits in Chickens" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/5/10"><span>Volume 5 Issue 10, October 2015</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/5/10.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/5/10.cover-source.jpg?itok=JwgNhVLw" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 5 (10)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/10/2027" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-ckOY10YhCd3j8CHPXvmgsRqG7m5k6NZ96WmdJ8aVAEg" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/5/10/2027" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/5/10/2027" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/5/10/2027" class="active" data-icon-position="" data-hide-link-title="0">GC-Content of Synonymous Codons Profoundly Influences Amino Acid Usage</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460151" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-XPwYEeRsQbokx1UeOxYQwVCdztY0iNfhfganwVTlLO4" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/10/2027.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/5/10/2027" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-C4DHjpcohllYqSJqozjN6mP7UUnpzPyVBsbcbopXdQ4" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/5/10/2027" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/10/2027.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460151" id="node460151--2" data-pisa="ggg;5/10/2027" data-pisa-master="ggg;g3.115.019877" data-apath="/ggg/5/10/2027.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">GC-Content of Synonymous Codons Profoundly Influences Amino Acid Usage</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Jing Li</span>, <span class="highwire-citation-author" data-delta="1">Jun Zhou</span>, <span class="highwire-citation-author" data-delta="2">Ying Wu</span>, <span class="highwire-citation-author" data-delta="3">Sihai Yang</span> and <span class="highwire-citation-author" data-delta="4">Dacheng Tian</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2027-2036;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.019877 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460151--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jing Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>State Key Laboratory of Pharmaceutical Biotechnology, School of Life Sciences, Nanjing University, Nanjing 210093, China </div><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute for Research on Cancer and Aging, Nice (IRCAN), Nice, France </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jing%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJing%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jun Zhou </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Organismic and Evolutionary Biology, Harvard University, Cambridge, Massachusetts </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jun%2BZhou%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhou%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJun%2BZhou%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:dtian@nju.edu.cn" class="" data-icon-position="" data-hide-link-title="0">dtian@nju.edu.cn</a>
<a href="mailto:jzhou@oeb.harvard.edu" class="" data-icon-position="" data-hide-link-title="0">jzhou@oeb.harvard.edu</a></span></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Ying Wu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>State Key Laboratory of Pharmaceutical Biotechnology, School of Life Sciences, Nanjing University, Nanjing 210093, China </div><div class='author-affiliation'><span class='nlm-sup'>§</span>Institute for Bioscience and Biotechnology Research, University of Maryland, Rockville, Maryland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ying%2BWu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wu%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYing%2BWu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Sihai Yang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>State Key Laboratory of Pharmaceutical Biotechnology, School of Life Sciences, Nanjing University, Nanjing 210093, China </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sihai%2BYang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yang%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASihai%2BYang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Dacheng Tian </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>State Key Laboratory of Pharmaceutical Biotechnology, School of Life Sciences, Nanjing University, Nanjing 210093, China </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dacheng%2BTian%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tian%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADacheng%2BTian%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:dtian@nju.edu.cn" class="" data-icon-position="" data-hide-link-title="0">dtian@nju.edu.cn</a>
<a href="mailto:jzhou@oeb.harvard.edu" class="" data-icon-position="" data-hide-link-title="0">jzhou@oeb.harvard.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/5/10/2027" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/5/10/2027&amp;title=GC-Content%20of%20Synonymous%20Codons%20Profoundly%20Influences%20Amino%20Acid%20Usage" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/5/10/2027&amp;title=GC-Content%20of%20Synonymous%20Codons%20Profoundly%20Influences%20Amino%20Acid%20Usage" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/5/10/2027&amp;title=GC-Content%20of%20Synonymous%20Codons%20Profoundly%20Influences%20Amino%20Acid%20Usage" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/5/10/2027&amp;text=GC-Content%20of%20Synonymous%20Codons%20Profoundly%20Influences%20Amino%20Acid%20Usage" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/5/10/2027&amp;title=GC-Content%20of%20Synonymous%20Codons%20Profoundly%20Influences%20Amino%20Acid%20Usage" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/5/10/2027&amp;t=GC-Content%20of%20Synonymous%20Codons%20Profoundly%20Influences%20Amino%20Acid%20Usage" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/5/10/2027&amp;title=GC-Content%20of%20Synonymous%20Codons%20Profoundly%20Influences%20Amino%20Acid%20Usage" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/5/10/2027&amp;title=GC-Content%20of%20Synonymous%20Codons%20Profoundly%20Influences%20Amino%20Acid%20Usage" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460151/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460151" id="node460151--3" data-pisa="ggg;5/10/2027" data-pisa-master="ggg;g3.115.019877" data-apath="/ggg/5/10/2027.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">GC-Content of Synonymous Codons Profoundly Influences Amino Acid Usage</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Jing Li</span>, <span class="highwire-citation-author" data-delta="1">Jun Zhou</span>, <span class="highwire-citation-author" data-delta="2">Ying Wu</span>, <span class="highwire-citation-author" data-delta="3">Sihai Yang</span> and <span class="highwire-citation-author" data-delta="4">Dacheng Tian</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2027-2036;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.019877 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460151--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jing Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>State Key Laboratory of Pharmaceutical Biotechnology, School of Life Sciences, Nanjing University, Nanjing 210093, China </div><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute for Research on Cancer and Aging, Nice (IRCAN), Nice, France </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jing%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJing%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jun Zhou </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Organismic and Evolutionary Biology, Harvard University, Cambridge, Massachusetts </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jun%2BZhou%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhou%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJun%2BZhou%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:dtian@nju.edu.cn" class="" data-icon-position="" data-hide-link-title="0">dtian@nju.edu.cn</a>
<a href="mailto:jzhou@oeb.harvard.edu" class="" data-icon-position="" data-hide-link-title="0">jzhou@oeb.harvard.edu</a></span></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Ying Wu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>State Key Laboratory of Pharmaceutical Biotechnology, School of Life Sciences, Nanjing University, Nanjing 210093, China </div><div class='author-affiliation'><span class='nlm-sup'>§</span>Institute for Bioscience and Biotechnology Research, University of Maryland, Rockville, Maryland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ying%2BWu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wu%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYing%2BWu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Sihai Yang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>State Key Laboratory of Pharmaceutical Biotechnology, School of Life Sciences, Nanjing University, Nanjing 210093, China </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sihai%2BYang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yang%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASihai%2BYang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Dacheng Tian </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>State Key Laboratory of Pharmaceutical Biotechnology, School of Life Sciences, Nanjing University, Nanjing 210093, China </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dacheng%2BTian%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tian%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADacheng%2BTian%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:dtian@nju.edu.cn" class="" data-icon-position="" data-hide-link-title="0">dtian@nju.edu.cn</a>
<a href="mailto:jzhou@oeb.harvard.edu" class="" data-icon-position="" data-hide-link-title="0">jzhou@oeb.harvard.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2027&amp;rft.epage=2036&amp;rft.atitle=GC-Content%20of%20Synonymous%20Codons%20Profoundly%20Influences%20Amino%20Acid%20Usage&amp;rft.volume=5&amp;rft.issue=10&amp;rft.date=2015-10-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Li%2C+Jing&amp;rft.au=Zhou%2C+Jun&amp;rft.au=Wu%2C+Ying&amp;rft.au=Yang%2C+Sihai&amp;rft.au=Tian%2C+Dacheng"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460151/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460151/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460151/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460151/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460151/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460151/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460151/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460151/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460151/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460151/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460151/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460151/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/5/10/2027" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-13" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/5/10/2027.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/5/10/2027.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/5/10/2027.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/5/10/2027.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
