<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/5/10/2165" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/5/10/2165.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/5/10/2165.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/5/10/2165.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/5/10/2165/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/5/10/2165.atom" />
<meta name="HW.pisa" content="ggg;5/10/2165" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Similar Efficacies of Selection Shape Mitochondrial and Nuclear Genes in Both Drosophila melanogaster and Homo sapiens" />
<meta name="DC.Identifier" content="10.1534/g3.114.016493" />
<meta name="DC.Date" content="2015-10-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2015 Cooper et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Deleterious mutations contribute to polymorphism even when selection effectively prevents their fixation. The efficacy of selection in removing deleterious mitochondrial mutations from populations depends on the effective population size ( Ne ) of the mitochondrial DNA and the degree to which a lack of recombination magnifies the effects of linked selection. Using complete mitochondrial genomes from Drosophila melanogaster and nuclear data available from the same samples, we reexamine the hypothesis that nonrecombining animal mitochondrial DNA harbor an excess of deleterious polymorphisms relative to the nuclear genome. We find no evidence of recombination in the mitochondrial genome, and the much-reduced level of mitochondrial synonymous polymorphism relative to nuclear genes is consistent with a reduction in Ne . Nevertheless, we find that the neutrality index, a measure of the excess of nonsynonymous polymorphism relative to the neutral expectation, is only weakly significantly different between mitochondrial and nuclear loci. This difference is likely the result of the larger proportion of beneficial mutations in X-linked relative to autosomal loci, and we find little to no difference between mitochondrial and autosomal neutrality indices. Reanalysis of published data from Homo sapiens reveals a similar lack of a difference between the two genomes, although previous studies have suggested a strong difference in both species. Thus, despite a smaller Ne , mitochondrial loci of both flies and humans appear to experience similar efficacies of purifying selection as do loci in the recombining nuclear genome." />
<meta name="DC.Contributor" content="Brandon S. Cooper" />
<meta name="DC.Contributor" content="Chad R. Burrus" />
<meta name="DC.Contributor" content="Chao Ji" />
<meta name="DC.Contributor" content="Matthew W. Hahn" />
<meta name="DC.Contributor" content="Kristi L. Montooth" />
<meta name="article:published_time" content="2015-10-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Similar Efficacies of Selection Shape Mitochondrial and Nuclear Genes in Both Drosophila melanogaster and Homo sapiens" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Deleterious mutations contribute to polymorphism even when selection effectively prevents their fixation. The efficacy of selection in removing deleterious mitochondrial mutations from populations depends on the effective population size (&lt;i&gt;N&lt;sub&gt;e&lt;/sub&gt;&lt;/i&gt;) of the mitochondrial DNA and the degree to which a lack of recombination magnifies the effects of linked selection. Using complete mitochondrial genomes from &lt;i&gt;Drosophila melanogaster&lt;/i&gt; and nuclear data available from the same samples, we reexamine the hypothesis that nonrecombining animal mitochondrial DNA harbor an excess of deleterious polymorphisms relative to the nuclear genome. We find no evidence of recombination in the mitochondrial genome, and the much-reduced level of mitochondrial synonymous polymorphism relative to nuclear genes is consistent with a reduction in &lt;i&gt;N&lt;sub&gt;e&lt;/sub&gt;&lt;/i&gt;. Nevertheless, we find that the neutrality index, a measure of the excess of nonsynonymous polymorphism relative to the neutral expectation, is only weakly significantly different between mitochondrial and nuclear loci. This difference is likely the result of the larger proportion of beneficial mutations in X-linked relative to autosomal loci, and we find little to no difference between mitochondrial and autosomal neutrality indices. Reanalysis of published data from &lt;i&gt;Homo sapiens&lt;/i&gt; reveals a similar lack of a difference between the two genomes, although previous studies have suggested a strong difference in both species. Thus, despite a smaller &lt;i&gt;N&lt;sub&gt;e&lt;/sub&gt;&lt;/i&gt;, mitochondrial loci of both flies and humans appear to experience similar efficacies of purifying selection as do loci in the recombining nuclear genome.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2015/10/01" />
<meta name="citation_mjid" content="ggg;5/10/2165" />
<meta name="citation_id" content="5/10/2165" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/5/10/2165" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/5/10/2165.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/5/10/2165.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/5/10/2165.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.114.016493" />
<meta name="citation_pmid" content="26297726" />
<meta name="citation_volume" content="5" />
<meta name="citation_issue" content="10" />
<meta name="citation_num_pages" content="12" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2165" />
<meta name="citation_lastpage" content="2176" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Brandon S. Cooper" />
<meta name="citation_author_institution" content="Department of Biology, Indiana University, Bloomington, Indiana 47405" />
<meta name="citation_author" content="Chad R. Burrus" />
<meta name="citation_author_institution" content="School of Informatics and Computing, Indiana University, Bloomington, Indiana 47405" />
<meta name="citation_author" content="Chao Ji" />
<meta name="citation_author_institution" content="School of Informatics and Computing, Indiana University, Bloomington, Indiana 47405" />
<meta name="citation_author" content="Matthew W. Hahn" />
<meta name="citation_author_institution" content="Department of Biology, Indiana University, Bloomington, Indiana 47405" />
<meta name="citation_author_institution" content="School of Informatics and Computing, Indiana University, Bloomington, Indiana 47405" />
<meta name="citation_author" content="Kristi L. Montooth" />
<meta name="citation_author_institution" content="Department of Biology, Indiana University, Bloomington, Indiana 47405" />
<meta name="citation_author_email" content="kmontooth2@unl.edu" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=H. Akashi;citation_author=N. Osada;citation_author=T. Ohta;citation_title=Weak Selection and Protein Evolution;citation_pages=15-31;citation_volume=192;citation_year=2012;citation_issue=1;citation_pmid=22964835;citation_doi=10.1534/genetics.112.140178" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=P. Awadalla;citation_author=A. Eyre-Walker;citation_author=J. M. Smith;citation_title=Linkage Disequilibrium and Recombination in Hominid Mitochondrial DNA;citation_pages=2524-2525;citation_volume=286;citation_year=1999;citation_issue=5449;citation_pmid=10617471;citation_doi=10.1126/science.286.5449.2524" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=D. Bachtrog;citation_title=Y-chromosome evolution: emerging insights into processes of Y-chromosome degeneration.;citation_pages=113-124;citation_volume=14;citation_year=2013;citation_pmid=23329112;citation_doi=10.1038/nrg3366" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=J. W. Ballard;citation_title=Comparative genomics of mitochondrial DNA in members of the Drosophila melanogaster subgroup.;citation_pages=48-63;citation_volume=51;citation_year=2000;citation_issue=1;citation_pmid=10903372" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. W. O. Ballard;citation_author=M. Kreitman;citation_title=Unraveling selection in the mitochondrial genome of Drosophila.;citation_pages=757-772;citation_volume=138;citation_year=1994;citation_issue=3;citation_pmid=7851772" />
<meta name="citation_reference" content="citation_journal_title=Annu. Rev. Ecol. Evol. Syst.;citation_author=J. W. O. Ballard;citation_author=D. M. Rand;citation_title=The population biology of mitochondrial DNA and its phylogenetic implications.;citation_pages=621-642;citation_volume=36;citation_year=2005;citation_doi=10.1146/annurev.ecolsys.36.091704.175513" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=J. W. O. Ballard;citation_author=M. C. Whitlock;citation_title=The incomplete natural history of mitochondria.;citation_pages=729-744;citation_volume=13;citation_year=2004;citation_issue=4;citation_pmid=15012752;citation_doi=10.1046/j.1365-294X.2003.02063.x" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=E. Bazin;citation_author=S. Glemin;citation_author=N. Galtier;citation_title=Population size does not influence mitochondrial genetic diversity in animals.;citation_pages=570-572;citation_volume=312;citation_year=2006;citation_issue=5773;citation_pmid=16645093;citation_doi=10.1126/science.1122033" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=D. W. Bellott;citation_author=J. F. Hughes;citation_author=H. Skaletsky;citation_author=L. G. Brown;citation_author=T. Pyntikova;citation_title=Mammalian Y chromosomes retain widely expressed dosage-sensitive regulators.;citation_pages=494-499;citation_volume=508;citation_year=2014;citation_pmid=24759411;citation_doi=10.1038/nature13206" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=C. T. Bergstrom;citation_author=J. Pritchard;citation_title=Germline Bottlenecks and the Evolutionary Maintenance of Mitochondrial Genomes;citation_pages=2135-2146;citation_volume=149;citation_year=1998;citation_issue=4;citation_pmid=9691064" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_journal_abbrev=Evolution;citation_author=A. J. Betancourt;citation_author=B. Blanco-Martin;citation_author=B. Charlesworth;citation_title=The relation between the neutrality index for mitochondrial genes and the distribution of mutational effects on fitness.;citation_pages=2427-2438;citation_volume=66;citation_year=2012;citation_issue=8;citation_pmid=22834742;citation_doi=10.1111/j.1558-5646.2012.01628.x" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=J. L. Boore;citation_title=Animal mitochondrial genomes.;citation_volume=27;citation_year=1999;citation_pmid=10101183;citation_doi=10.1093/nar/27.8.1767" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=T. C. Bruen;citation_author=H. Philippe;citation_author=D. Bryant;citation_title=A simple and robust statistical test for detecting the presence of recombination.;citation_volume=172;citation_year=2006;citation_pmid=16489234;citation_doi=10.1534/genetics.105.048975" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=C. D. Bustamante;citation_author=A. Fledel-Alon;citation_author=S. Williamson;citation_author=R. Nielsen;citation_author=M. T. Hubisz;citation_title=Natural selection on protein-coding genes in the human genome.;citation_pages=1153-1157;citation_volume=437;citation_year=2005;citation_issue=7062;citation_pmid=16237444;citation_doi=10.1038/nature04240" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=A. B. Carvalho;citation_author=A. G. Clark;citation_title=Efficient identification of Y chromosome sequences in the human and Drosophila genomes.;citation_volume=23;citation_year=2013;citation_pmid=23921660;citation_doi=10.1101/gr.156034.113" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=A. B. Carvalho;citation_author=L. B. Koerich;citation_author=A. G. Clark;citation_title=Origin and evolution of Y chromosomes: Drosophila tales.;citation_pages=270-277;citation_volume=25;citation_year=2009;citation_issue=6;citation_pmid=19443075;citation_doi=10.1016/j.tig.2009.04.002" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=B. Charlesworth;citation_title=The Effects of Deleterious Mutations on Evolution at Linked Sites;citation_pages=5-22;citation_volume=190;citation_year=2012;citation_issue=1;citation_pmid=22219506;citation_doi=10.1534/genetics.111.134288" />
<meta name="citation_reference" content="citation_journal_title=Philosophical Transactions of the Royal Society B: Biological Sciences;citation_journal_abbrev=Phil Trans R Soc B;citation_author=B. Charlesworth;citation_author=D. Charlesworth;citation_title=The degeneration of Y chromosomes;citation_pages=1563-1572;citation_volume=355;citation_year=2000;citation_issue=1403;citation_pmid=11127901;citation_doi=10.1098/rstb.2000.0717" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=D. O. Clary;citation_author=D. R. Wolstenholme;citation_title=The mitochondrial DNA molecular of Drosophila yakuba: nucleotide sequence, gene organization, and genetic code.;citation_pages=252-271;citation_volume=22;citation_year=1985;citation_issue=3;citation_pmid=3001325;citation_doi=10.1007/BF02099755" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=M. Clement;citation_author=D. Posada;citation_author=K. A. Crandall;citation_title=TCS: a computer program to estimate gene genealogies.;citation_pages=1657-1659;citation_volume=9;citation_year=2000;citation_issue=10;citation_pmid=11050560;citation_doi=10.1046/j.1365-294x.2000.01020.x" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=A. Eyre-Walker;citation_author=P. D. Keightley;citation_title=The distribution of fitness effects of new mutations.;citation_pages=610-618;citation_volume=8;citation_year=2007;citation_issue=8;citation_pmid=17637733;citation_doi=10.1038/nrg2146" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. C. Fay;citation_author=C. I. Wu;citation_title=Hitchhiking Under Positive Darwinian Selection;citation_pages=1405-1413;citation_volume=155;citation_year=2000;citation_issue=3;citation_pmid=10880498" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=Y. X. Fu;citation_author=W. H. Li;citation_title=Statistical tests of neutrality of mutations.;citation_pages=693-709;citation_volume=133;citation_year=1993;citation_issue=3;citation_pmid=8454210" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=D. Garrigan;citation_author=S. B. Kingan;citation_author=A. J. Geneva;citation_author=J. P. Vedanayagam;citation_author=D. C. Presgraves;citation_title=Genome diversity and divergence in Drosophila mauritiana: Multiple signatures of faster X evolution.;citation_volume=6;citation_year=2014;citation_pmid=25193308;citation_doi=10.1093/gbe/evu198" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. H. Gillespie;citation_title=Genetic Drift in an Infinite Population: The Pseudohitchhiking Model;citation_pages=909-919;citation_volume=155;citation_year=2000;citation_issue=2;citation_pmid=10835409" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=T. I. Gossmann;citation_author=B.-H. Song;citation_author=A. J. Windsor;citation_author=T. Mitchell-Olds;citation_author=C. J. Dixon;citation_title=Genome wide analyses reveal little evidence for adaptive evolution in many plant species.;citation_volume=27;citation_year=2010;citation_pmid=20299543;citation_doi=10.1093/molbev/msq079" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=R. E. Green;citation_author=A.-S. Malaspinas;citation_author=J. Krause;citation_author=A. W. Briggs;citation_author=P. L. F. Johnson;citation_title=A complete Neandertal mitochondrial genome sequence determined by high-throughput sequencing.;citation_pages=416-426;citation_volume=134;citation_year=2008;citation_issue=3;citation_pmid=18692465;citation_doi=10.1016/j.cell.2008.06.021" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=C. Haag-Liautard;citation_author=M. Dorris;citation_author=X. Maside;citation_author=S. Macaskill;citation_author=D. L. Halligan;citation_title=Direct estimation of per nucleotide and genomic deleterious mutation rates in Drosophila.;citation_pages=82-85;citation_volume=445;citation_year=2007;citation_issue=7123;citation_pmid=17203060;citation_doi=10.1038/nature05388" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=C. Haag-Liautard;citation_author=N. Coffey;citation_author=D. Houle;citation_author=M. Lynch;citation_author=B. Charlesworth;citation_title=Direct estimation of the mitochondrial DNA mutation rate in Drosophila melanogaster.;citation_pages=e204-e204;citation_volume=6;citation_year=2008;citation_issue=8;citation_pmid=18715119;citation_doi=10.1371/journal.pbio.0060204" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_journal_abbrev=Evolution;citation_author=M. W. Hahn;citation_title=Toward a selection theory of molecular evolution.;citation_pages=255-265;citation_volume=62;citation_year=2008;citation_issue=2;citation_pmid=18302709;citation_doi=10.1111/j.1558-5646.2007.00308.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. W. Hahn;citation_author=M. D. Rausher;citation_author=C. W. Cunningham;citation_title=Distinguishing Between Selection and Population Expansion in an Experimental Lineage of Bacteriophage T7;citation_pages=11-20;citation_volume=161;citation_year=2002;citation_issue=1;citation_pmid=12019219" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=W. G. Hill;citation_author=A. Robertson;citation_title=The effect of linkage on limits to artificial selection.;citation_pages=269-294;citation_volume=8;citation_year=1966;citation_issue=3;citation_pmid=5980116;citation_doi=10.1017/S0016672300010156" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=S. Horai;citation_author=K. Hayasaka;citation_author=R. Kondo;citation_author=K. Tsugane;citation_author=N. Takahata;citation_title=Recent African origin of modern humans revealed by complete sequences of hominoid mitochondrial DNAs.;citation_pages=532-536;citation_volume=92;citation_year=1995;citation_issue=2;citation_pmid=7530363;citation_doi=10.1073/pnas.92.2.532" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=R. R. Hudson;citation_author=N. L. Kaplan;citation_title=STATISTICAL PROPERTIES OF THE NUMBER OF RECOMBINATION EVENTS IN THE HISTORY OF A SAMPLE OF DNA SEQUENCES;citation_pages=147-164;citation_volume=111;citation_year=1985;citation_issue=1;citation_pmid=4029609" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=H. Innan;citation_author=M. Nordborg;citation_title=Recombination or Mutational Hot Spots in Human mtDNA?;citation_pages=1122-1127;citation_volume=19;citation_year=2002;citation_issue=7;citation_pmid=12082131;citation_doi=10.1093/oxfordjournals.molbev.a004170" />
<meta name="citation_reference" content="citation_journal_abbrev=Forensic Sci Int Genet;citation_author=R. S. Just;citation_author=T. M. Diegoli;citation_author=J. L. Saunier;citation_author=J. A. Irwin;citation_author=T. J. Parsons;citation_title=Complete mitochondrial genome sequences for 265 African American and U.S. &quot;Hispanic&quot; individuals.;citation_pages=e45-e48;citation_volume=2;citation_year=2008;citation_issue=3;citation_pmid=19083815;citation_doi=10.1016/j.fsigen.2007.12.001" />
<meta name="citation_reference" content="citation_title=The Neutral Theory of Molecular Evolution;citation_author=M. Kimura;citation_year=1983" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=C. H. Langley;citation_author=K. Stevens;citation_author=C. Cardeno;citation_author=Y. C. G. Lee;citation_author=D. R. Schrider;citation_title=Genomic variation in natural populations of Drosophila melanogaster.;citation_volume=192;citation_year=2012;citation_pmid=22673804;citation_doi=10.1534/genetics.112.142018" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=M. A. Larkin;citation_author=G. Blackshields;citation_author=N. P. Brown;citation_author=R. Chenna;citation_author=P. A. McGettigan;citation_title=Clustal W and clustal X version 2.0.;citation_volume=23;citation_year=2007;citation_pmid=17846036;citation_doi=10.1093/bioinformatics/btm404" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=R. C. Lewontin;citation_title=THE INTERACTION OF SELECTION AND LINKAGE. I. GENERAL CONSIDERATIONS; HETEROTIC MODELS;citation_pages=49-67;citation_volume=49;citation_year=1964;citation_issue=1;citation_pmid=17248194" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=R. Durbin;citation_title=Fast and accurate short read alignment with Burrows-Wheeler transform.;citation_volume=25;citation_year=2009;citation_pmid=19451168;citation_doi=10.1093/bioinformatics/btp324" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=B. Handsaker;citation_author=A. Wysoker;citation_author=T. Fennell;citation_author=J. Ruan;citation_title=The Sequence Alignment/Map format and SAMtools.;citation_volume=25;citation_year=2009;citation_pmid=19505943;citation_doi=10.1093/bioinformatics/btp352" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_journal_abbrev=Evolution;citation_author=Y. F. Li;citation_author=J. C. Costello;citation_author=A. K. Holloway;citation_author=M. W. Hahn;citation_title=&quot;Reverse ecology&quot; and the power of population genomics.;citation_pages=2984-2994;citation_volume=62;citation_year=2008;citation_issue=12;citation_pmid=18752601;citation_doi=10.1111/j.1558-5646.2008.00486.x" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=M. Lynch;citation_title=Mutation accumulation in transfer RNAs: molecular evidence for Muller's ratchet in mitochondrial genomes;citation_pages=209-220;citation_volume=13;citation_year=1996;citation_issue=1;citation_pmid=8583893;citation_doi=10.1093/oxfordjournals.molbev.a025557" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=M. Lynch;citation_title=Mutation accumulation in nuclear, organelle, and prokaryotic transfer RNA genes.;citation_volume=14;citation_year=1997;citation_pmid=9287424;citation_doi=10.1093/oxfordjournals.molbev.a025834" />
<meta name="citation_reference" content="citation_title=Origins of Genome Architecture;citation_author=M. Lynch;citation_year=2007" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=M. Lynch;citation_author=W. Sung;citation_author=K. Morris;citation_author=N. Coffey;citation_author=C. R. Landry;citation_title=A genome-wide view of the spectrum of spontaneous mutations in yeast.;citation_volume=105;citation_year=2008;citation_pmid=18583475;citation_doi=10.1073/pnas.0803466105" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=T. F. C. Mackay;citation_author=S. Richards;citation_author=E. A. Stone;citation_author=A. Barbadilla;citation_author=J. F. Ayroles;citation_title=The Drosophila melanogaster Genetic Reference Panel.;citation_pages=173-178;citation_volume=482;citation_year=2012;citation_issue=7384;citation_pmid=22318601;citation_doi=10.1038/nature10811" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=J. Maynard Smith;citation_author=J. Haigh;citation_title=The hitch-hiking effect of a favourable gene.;citation_pages=23-35;citation_volume=23;citation_year=1974;citation_issue=1;citation_pmid=4407212;citation_doi=10.1017/S0016672300014634" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=J. H. McDonald;citation_author=M. Kreitman;citation_title=Adaptive protein evolution at the Adh locus in Drosophila.;citation_pages=652-654;citation_volume=351;citation_year=1991;citation_issue=6328;citation_pmid=1904993;citation_doi=10.1038/351652a0" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=C. D. Meiklejohn;citation_author=K. L. Montooth;citation_author=D. M. Rand;citation_title=Positive and negative selection on the mitochondrial genome.;citation_pages=259-263;citation_volume=23;citation_year=2007;citation_issue=6;citation_pmid=17418445;citation_doi=10.1016/j.tig.2007.03.008" />
<meta name="citation_reference" content="citation_journal_title=Trends Genet.;citation_author=R. P. Meisel;citation_author=T. Connallon;citation_title=The faster-X effect: integrating theory and data.;citation_pages=537-544;citation_volume=29;citation_year=2013;citation_pmid=23790324;citation_doi=10.1016/j.tig.2013.05.009" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=P. W. Messer;citation_author=D. A. Petrov;citation_title=Frequent adaptation and the McDonald–Kreitman test.;citation_volume=110;citation_year=2013;citation_pmid=23650353;citation_doi=10.1073/pnas.1220835110" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=J. Meunier;citation_author=A. Eyre-Walker;citation_title=The Correlation Between Linkage Disequilibrium and Distance: Implications for Recombination in Hominid Mitochondria;citation_pages=2132-2135;citation_volume=18;citation_year=2001;citation_issue=11;citation_pmid=11606711;citation_doi=10.1093/oxfordjournals.molbev.a003756" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=K. L. Montooth;citation_author=D. M. Rand;citation_title=The spectrum of mitochondrial mutation differs across species.;citation_pages=e213-e213;citation_volume=6;citation_year=2008;citation_issue=8;citation_pmid=18752353;citation_doi=10.1371/journal.pbio.0060213" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=K. L. Montooth;citation_author=D. N. Abt;citation_author=J. Hofmann;citation_author=D. M. Rand;citation_title=Comparative genomics of Drosophila mtDNA: Novel features of conservation and change across functional domains and lineages.;citation_pages=94-114;citation_volume=69;citation_year=2009;citation_issue=1;citation_pmid=19533212;citation_doi=10.1007/s00239-009-9255-0" />
<meta name="citation_reference" content="citation_journal_title=Genetica;citation_author=M. W. Nachman;citation_title=Deleterious mutations in animal mitochondrial DNA.;citation_pages=61-69;citation_volume=102–103;citation_year=1998;citation_doi=10.1023/A:1017030708374" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. W. Nachman;citation_author=W. M. Brown;citation_author=M. Stoneking;citation_author=C. F. Aquadro;citation_title=Nonneutral Mitochondrial DNA Variation in Humans and Chimpanzees;citation_pages=953-963;citation_volume=142;citation_year=1996;citation_issue=3;citation_pmid=8849901" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=M. Nei;citation_author=T. Gojobori;citation_title=Simple methods for estimating the numbers of synonymous and nonsynonymous nucleotide substitutions;citation_pages=418-426;citation_volume=3;citation_year=1986;citation_issue=5;citation_pmid=3444411" />
<meta name="citation_reference" content="citation_journal_title=Proc. Biol. Sci.;citation_author=M. Neiman;citation_author=D. R. Taylor;citation_title=The causes of mutation accumulation in mitochondrial genomes.;citation_volume=276;citation_year=2009;citation_pmid=19203921;citation_doi=10.1098/rspb.2008.1758" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=N. Osada;citation_author=H. Akashi;citation_title=Mitochondrial-nuclear interactions and accelerated compensatory evolution: evidence from the primate cytochrome C oxidase complex.;citation_volume=29;citation_year=2012;citation_pmid=21890478;citation_doi=10.1093/molbev/msr211" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=K. Y. Popadin;citation_author=S. I. Nikolaev;citation_author=T. Junier;citation_author=M. Baranova;citation_author=S. E. Antonarakis;citation_title=Purifying selection in mammalian mitochondrial protein-coding genes is highly effective and congruent with evolution of nuclear genes.;citation_volume=30;citation_year=2013;citation_pmid=22983951;citation_doi=10.1093/molbev/mss219" />
<meta name="citation_reference" content="citation_journal_title=Current biology :  CB;citation_journal_abbrev=Curr Biol;citation_author=D. C. Presgraves;citation_title=Recombination enhances protein adaptation in Drosophila melanogaster.;citation_pages=1651-1656;citation_volume=15;citation_year=2005;citation_issue=18;citation_pmid=16169487;citation_doi=10.1016/j.cub.2005.07.065" />
<meta name="citation_reference" content="citation_journal_title=Genetica;citation_author=D. M. Rand;citation_title=Population genetics of the cytoplasm and the units of selection on mitochondrial DNA in Drosophila melanogaster.;citation_pages=685-697;citation_volume=139;citation_year=2011;citation_pmid=21538136;citation_doi=10.1007/s10709-011-9576-y" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=D. M. Rand;citation_author=L. M. Kann;citation_title=Excess amino acid polymorphism in mitochondrial DNA: contrasts among genes from Drosophila, mice, and humans;citation_pages=735-748;citation_volume=13;citation_year=1996;citation_issue=6;citation_pmid=8754210;citation_doi=10.1093/oxfordjournals.molbev.a025634" />
<meta name="citation_reference" content="citation_journal_title=Genetica;citation_author=D. M. Rand;citation_author=L. M. Kann;citation_title=Mutation and selection at silent and replacement sites in the evolution of animal mitochondrial DNA.;citation_pages=393-407;citation_volume=102–103;citation_year=1998;citation_doi=10.1023/A:1017006118852" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=M. F. Richardson;citation_author=L. A. Weinert;citation_author=J. J. Welch;citation_author=R. S. Linheiro;citation_author=M. M. Magwire;citation_title=Population genomics of the Wolbachia endosymbiont in Drosophila melanogaster.;citation_pages=e1003129;citation_volume=8;citation_year=2012;citation_pmid=23284297;citation_doi=10.1371/journal.pgen.1003129" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=J. Rozas;citation_author=J. C. Sanchez-Delbarrio;citation_author=X. Messeguer;citation_author=R. Rozas;citation_title=DnaSP, DNA polymorphism analyses by the coalescent and other methods;citation_pages=2496-2497;citation_volume=19;citation_year=2003;citation_issue=18;citation_pmid=14668244;citation_doi=10.1093/bioinformatics/btg359" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=S. Rozen;citation_author=J. D. Marszalek;citation_author=R. K. Alagappan;citation_author=H. Skaletsky;citation_author=D. C. Page;citation_title=Remarkably little variation in proteins encoded by the Y chromosome's single-copy genes, implying effective purifying selection.;citation_pages=923-928;citation_volume=85;citation_year=2009;citation_issue=6;citation_pmid=20004767;citation_doi=10.1016/j.ajhg.2009.11.011" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=F. Rubino;citation_author=R. Piredda;citation_author=F. M. Calabrese;citation_author=D. Simone;citation_author=M. Lang;citation_title=HmtDB, a genomic resource for mitochondrion-based human variability studies.;citation_volume=40;citation_year=2012;citation_pmid=22139932;citation_doi=10.1093/nar/gkr1086" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=L. A. Sheldahl;citation_author=D. M. Weinreich;citation_author=D. M. Rand;citation_title=Recombination, Dominance and Selection on Amino Acid Polymorphism in the Drosophila Genome: Contrasting Patterns on the X and Fourth Chromosomes;citation_pages=1195-1208;citation_volume=165;citation_year=2003;citation_issue=3;citation_pmid=14668375" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. D. Shoemaker;citation_author=K. A. Dyer;citation_author=M. Ahrens;citation_author=K. Mcabee;citation_author=J. Jaenike;citation_title=Decreased Diversity but Increased Substitution Rate in Host mtDNA as a Consequence of Wolbachia Endosymbiont Infection;citation_pages=2049-2058;citation_volume=168;citation_year=2004;citation_issue=4;citation_pmid=15611174;citation_doi=10.1534/genetics.104.030890" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=N. D. Singh;citation_author=L. B. Koerich;citation_author=A. B. Carvalho;citation_author=A. G. Clark;citation_title=Positive and purifying selection on the Drosophila Y chromosome.;citation_volume=31;citation_year=2014;citation_pmid=24974375;citation_doi=10.1093/molbev/msu203" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=N. Stoletzki;citation_author=A. Eyre-Walker;citation_title=Estimation of the neutrality index.;citation_volume=28;citation_year=2011;citation_pmid=20837603;citation_doi=10.1093/molbev/msq249" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=F. Tajima;citation_title=Statistical method for testing the neutral mutation hypothesis by DNA polymorphism.;citation_pages=585-595;citation_volume=123;citation_year=1989;citation_issue=3;citation_pmid=2513255" />
<meta name="citation_reference" content="citation_journal_title=Theoretical population biology;citation_journal_abbrev=Theor Popul Biol;citation_author=G. A. Watterson;citation_title=On the number of segregating sites in genetical models without recombination.;citation_pages=256-276;citation_volume=7;citation_year=1975;citation_issue=2;citation_pmid=1145509;citation_doi=10.1016/0040-5809(75)90020-9" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. M. Weinreich;citation_author=D. M. Rand;citation_title=Contrasting Patterns of Nonneutral Evolution in Proteins Encoded in Nuclear and Mitochondrial Genomes;citation_pages=385-399;citation_volume=156;citation_year=2000;citation_issue=1;citation_pmid=10978302" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=D. J. White;citation_author=J. N. Wolff;citation_author=M. Pierson;citation_author=N. J. Gemmell;citation_title=Revealing the hidden complexities of mtDNA inheritance.;citation_pages=4925-4942;citation_volume=17;citation_year=2008;citation_issue=23;citation_pmid=19120984;citation_doi=10.1111/j.1365-294X.2008.03982.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. R. Wolstenholme;citation_author=D. O. Clary;citation_title=SEQUENCE EVOLUTION OF DROSOPHILA MITOCHONDRIAL DNA;citation_pages=725-744;citation_volume=109;citation_year=1985;citation_issue=4;citation_pmid=3921427" />
<meta name="citation_reference" content="citation_journal_title=Annals of human genetics;citation_journal_abbrev=Ann Hum Genet;citation_author=B. Woolf;citation_title=On estimating the relation between blood group and disease.;citation_pages=251-253;citation_volume=19;citation_year=1955;citation_issue=4;citation_pmid=14388528;citation_doi=10.1111/j.1469-1809.1955.tb01348.x" />
<meta name="citation_reference" content="citation_journal_title=Annu. Rev. Ecol. Evol. Syst.;citation_author=S. I. Wright;citation_author=P. Andolfatto;citation_title=The impact of natural selection on the genome: emerging patterns in Drosophila and Arabidopsis.;citation_pages=193-213;citation_volume=39;citation_year=2008;citation_doi=10.1146/annurev.ecolsys.39.110707.173342" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Similar Efficacies of Selection Shape Mitochondrial and Nuclear Genes in Both Drosophila melanogaster and Homo sapiens" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10.cover-source.jpg" />
<meta name="twitter:description" content="Deleterious mutations contribute to polymorphism even when selection effectively prevents their fixation. The efficacy of selection in removing deleterious mitochondrial mutations from populations depends on the effective population size ( Ne ) of the mitochondrial DNA and the degree to which a lack of recombination magnifies the effects of linked selection. Using complete mitochondrial genomes from Drosophila melanogaster and nuclear data available from the same samples, we reexamine the hypothesis that nonrecombining animal mitochondrial DNA harbor an excess of deleterious polymorphisms relative to the nuclear genome. We find no evidence of recombination in the mitochondrial genome, and the much-reduced level of mitochondrial synonymous polymorphism relative to nuclear genes is consistent with a reduction in Ne . Nevertheless, we find that the neutrality index, a measure of the excess of nonsynonymous polymorphism relative to the neutral expectation, is only weakly significantly different between mitochondrial and nuclear loci. This difference is likely the result of the larger proportion of beneficial mutations in X-linked relative to autosomal loci, and we find little to no difference between mitochondrial and autosomal neutrality indices. Reanalysis of published data from Homo sapiens reveals a similar lack of a difference between the two genomes, although previous studies have suggested a strong difference in both species. Thus, despite a smaller Ne , mitochondrial loci of both flies and humans appear to experience similar efficacies of purifying selection as do loci in the recombining nuclear genome." />
<meta name="og-title" property="og:title" content="Similar Efficacies of Selection Shape Mitochondrial and Nuclear Genes in Both Drosophila melanogaster and Homo sapiens" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/5/10/2165" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Deleterious mutations contribute to polymorphism even when selection effectively prevents their fixation. The efficacy of selection in removing deleterious mitochondrial mutations from populations depends on the effective population size ( Ne ) of the mitochondrial DNA and the degree to which a lack of recombination magnifies the effects of linked selection. Using complete mitochondrial genomes from Drosophila melanogaster and nuclear data available from the same samples, we reexamine the hypothesis that nonrecombining animal mitochondrial DNA harbor an excess of deleterious polymorphisms relative to the nuclear genome. We find no evidence of recombination in the mitochondrial genome, and the much-reduced level of mitochondrial synonymous polymorphism relative to nuclear genes is consistent with a reduction in Ne . Nevertheless, we find that the neutrality index, a measure of the excess of nonsynonymous polymorphism relative to the neutral expectation, is only weakly significantly different between mitochondrial and nuclear loci. This difference is likely the result of the larger proportion of beneficial mutations in X-linked relative to autosomal loci, and we find little to no difference between mitochondrial and autosomal neutrality indices. Reanalysis of published data from Homo sapiens reveals a similar lack of a difference between the two genomes, although previous studies have suggested a strong difference in both species. Thus, despite a smaller Ne , mitochondrial loci of both flies and humans appear to experience similar efficacies of purifying selection as do loci in the recombining nuclear genome." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10.cover-source.jpg" />
    <title>Similar Efficacies of Selection Shape Mitochondrial and Nuclear Genes in Both Drosophila melanogaster and Homo sapiens | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"yBk1TYcLcCT_m4OfShqF2D8XiYXB-xcTp3VPU-28KaI","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460046","apath":"\/ggg\/5\/10\/2165.atom","pisa":"ggg;5\/10\/2165","ac":{"\/ggg\/5\/10\/2165.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/5\/10\/2165.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;5\/10\/2165"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;5\/10\/2165"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/5\/10\/2165":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022FkXD9ks51hvHqB5nUsetSdyYWuCapBLtmUKCS70LpT4wnZybuWellrVNCfdAN3Sa16vt7vq\\\\\\\/5hFHbZw0bWbhOQCFE\\\\\\\/KyoJC\\\\\\\/347izldr2ZW1lXPmfmlSpX37g0q9BItalL3JdFxfuXPwqURjAOEzaZFkhvixMUAiugkEIH1Pld\\\\\\\/Y3o8DJWWhKQ\\\\\\\/7hrrmf3zMEJnrOFmy8IYPV9ztcjqt\\\\\\\/s49uXAWpYadGYJxhFrnVkkXhcSQe02EBXM7+AtQuT+cAFXjXlSL3HwcKpyt2xyVpL+YQ+JhUktCVkSe7rEW2Jk88G\\\\\\\/nndkc5NTPVEZESh8II\\\\\\\/en8muUNN2LW5BlIfUV9vOUZpdTtbi5bmNXRkSbR0+oGJMNvDIxWeq731jx8nd+NCmK8E3wmeWglUMKgYscozVpHpIpje75sdMHgFM9V0bk63NzYNKALY2lKBH1A+usfTWGNxoiSkKHJxKoUCSGSY7RYkIopyztGZNBTceNjOsHf8fDmjm1IrvcIFXBLHYw1Sj2vgL2zjwEJPQjnd97UhnpMqxyc+6jeY5P6q4e\\\\\\\/stYPZanAORC7L2CZiNnW4zaXgnewznoGTD9hlaiQxNqH+BXAxdKqMv2TIGGA78w4d9Vfn9cGEq+\\\\\\\/wZJWQ\\\\\\\/Jf5RDvOhRKc4Qr9rzP\\\\\\\/Q+7rrq05c4wis4dbRLmmu1etLiT17K38dSO0j90ancmeXjUrZq4Rk+BncYpvVk502NBCZqPc5J1AEHdwmWtvPqtFt6QujAQjjiD7\\\\\\\/UpJPbBAB3dQV5StjXhyquqsrSq2tfy381qK85OF\\\\\\\/ZG+ROEDSH8FFZwpDXXIkzFDVPSMBYkMYaE4e4N45uxOaUmp4eTDxSuL028N2mHkp1tMGYgU8Oq1WOP+oyUarAHwfz1nU8J+w7i8lSedRwlDV36j\\\\\\\/fkBHuaYL4nYXFdYBRD1lf4UxlYtE5WG\\\\\\\/LW+oISdVAmAFwfnoj7uG6CfW3qhMTLoRazwynneBwyFUlWbmbcSRzSvwqsLykDYGeT+fkM11R17Q1rF4w428hYkiilJIGa6IRihW70bTt36lLQT0n4sP\\\\\\\/A007GEsGgnvxfm5QAko8hi9h\\\\\\\/zWT5hnbz\\\\\\\/iSIiyTDlPvFFNChV3Xz3RhRUnOadvTtMvc7ZMrKf2W06cXtttqkyFH2S42TRlgCm+JDKp9SfGtcphPsGiX3OWHGI53adIr0jUKl5f7z1+wIinw2Any5Y4rRLF\\\\\\\/6muIUnuki7PKEW9ijBXDWkCgLrbfE8Ets8BaiKegZgX2hcc3y27J+IgHesCnUYbsq27W\\\u0022,\\\u0022iv\\\u0022:\\\u0022XqS1mX+WhkdkEpO8SvwsyA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00225b7da210b058dbc3510b3dadb4c685c36f1777da6572b956e10cb651beb83076\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022kcILL8KAh\\\\\\\/orgbECPBB2\\\\\\\/jrrur2NnD7r5M43DC0QsdfWA1aOLaKryEtPPZEa3tfEsAFg1W38Zs57Bj4pOckrbgjiHmuY0uMN+K7Yp6ugjef1D7TjVYnvqZCuJTIQAzjyDX+2CGyYy10\\\\\\\/cLIHSDARNJPHcjXPvwB5ok3Y67MH+OaFysJ\\\\\\\/IfsDUmH8QNW6SI9kyctg5oHGioCw6VY3dZfHWZCCR5Qlbu79WVo2M9oVNxrxAUhvYZhTbo4xdXnGt5BDvWAHag8vhARPFvig+8t5y3Brk1Vhuu9spi\\\\\\\/\\\\\\\/OKd+i+BTXRZ+QncoPhmnLNj4iqH3pRTWWwo6wlm3GJEmDPzMXiuaaKVCFApZLULK\\\\\\\/mNxcH\\\\\\\/zDeUV1mEi5U+Q0tJSBOi2Nb3Yo29kRrqHQnnF4YQckIr5IDYfr93G5AbG9ywE9SNvPCtjoDvlzOBYgrz\\\\\\\/zdvZk57IA+MVFW414XCHrcokiuJDq3NlGlaAwx0G94cvjSQOyleTFTP1BrAQPn7hUiHPoekwkfVIFWyXIkrt\\\\\\\/xTGGv+uWHpqNqW+KgyqqoImZ9sVI\\\\\\\/wEYSwVH0MKE8PoB5stoTry+BWwVm2DhIj6Uq2ulsYKYRHfnRM\\\\\\\/\\\\\\\/VA29hY6z5jmtcgbwr5v4S9hNI93fBcqYaT81yEYRHRCljKioxZDnNEA1pi9xIfxLxncTVNwvR1NEMNan2EPN4Uh8oSGxi5FxUNpoX1X9whv\\\\\\\/QSSLyhkhbgUT+PPRLBjee6O88mloN1ajz5irYIcxP77BZPoQ4pSLfDe2FgiwzcCgf3omUV1JponXnCioURuLBaOl9h9EdhZ4Ha4ONxlGMn+G+mueiEHYNw0sBYZe56psuyFuzTq9gCxcsKDsYPGusffDBDRzgMn0NZiNe4IUYl1ZwfAyP8zr0nB8GgYQrDnZnCsLYtEQA+QmMcOWpzVfkmY9xIYjGJbvXz9g6ft6lw+JHzZdqR1nP39plGB2A6OhigRy1L8Hx2GukBcohA7iRiPHP1duZC3dFEU0zMoPY0Cm8hWj+TCjjdk76pdKpvmgtWLr5XKZ5wmOykRZlERyAuB7qL+L+ugmdzn2f5e5blPaWDFmaDu4Ib1N5ks0Q1tPL3g\\\\\\\/pTPyr7iE08BoI8EvLvf0tBom5WUhuTXn6cV2KSTM\\\\\\\/OEs3HUrZnKkLhTVx4eWBkEba6ORTJb5WzE55d6YsbDuqLl2niMf6ObKKSTM9ECUuyIWPpUUzhvfWQpIbfEDguxtpGneRTFncylj+rFVqEg5+AuSOdaL9kqYm4d0AXnZaM\\\\\\\/pL6MqOlXPOf0jq0ziP1vh9pszhXFTX18ghgrwV5WmKapXHO\\\\\\\/euNXIPfVuhfth7SWo0b\\\\\\\/a\\\\\\\/5UwbcMVjZtaV7xgtODVymM5tyb8glGP9r7DW2obeM1tAYQW6xu8mGfom9I+EAAOO21G60HvK6KgAq5OKmIztAMrzebKC\\\\\\\/FEzCX\\\\\\\/89FwwYTY2Cl6wTv\\\\\\\/d4WEBsab08Lf9NX1k6i6iWiFWY2BsjkEEn\\\\\\\/ukW6Pn9DjmFSvJ2qE5RJIPjJ\\\\\\\/zDkVU\\\\\\\/J19mBMuYwnGw3CiLM2mxqiL4tVwdVtgzAPcrQlTubMaBfVVpPf89F7XqYMkLQx9XgNJkmphE3DWLFbEdOuTGFsZoQHKktiNn82PyBDp0IvFaanwhIXV9BDjIOr0TF8re+IH+j+1BsjoIHaHuJsr6LQcCr9\\\\\\\/tlvK6BtOaUhGz\\\\\\\/PAN1NJUbX4cPAY2s1uWBWBiqhbS9fnCZTIyn3CCqwXU+tpWLl4qYRqDwmSzP4UH7qYwQxFV0qsgQ8+lLihN3b1udnNVjNFCr1llPDI8HIlHp2Y+aqzGCVR9lfqGgr5ICPoGf\\\\\\\/kj+dvpi7gs8LkbwtmOFsnkq06gxAMQ516qW0\\\\\\\/PEWXXygkNXYD+XGv3UhXNuEbZb1b2PiLi0dkRZP4lnzSgg3QAbPNpj12PGWRg28GdTRl4u7lVhEunWstSCRz1xiLmnQrR\\\\\\\/gdLCDaH62iXzFrXAtw2iEN1Suc2yt0ih3jzVutQtqjYrUTsZuzn1uwMsZMMgj1Eyy+nrDLUk5vaIiOyZAWye363W\\\\\\\/tBFQTot2Mpa8eOYUT878jKkAD35hoWkqdHxBBUcJyKvvtOakpjcxq4+ViJxydUNTGZUTolu6bKpBqbSoo5ktM\\\\\\\/lCkVheuhyHQdwzIk2+2OEm5gR\\\\\\\/Z9EKl\\\\\\\/9ZMx8UmZ6Zr34y9j6iIYOPr8d3GCGn6xG1XosOopJogW2z0lHs4pbIGgBss409Gbm41orDJh1WfeBAdh4L4Sd7COqQnmGwyZLxAaBwzxVl+XnU2oFQpVqcnhZrXkvquAMd2TfZzaqxIe\\\\\\\/Gldmmj6spQRVSuUf+dPGLnyfmQ9J2TqxxHl8aF8pHD\\\\\\\/uu9cDmIR4WouKMbF9\\\\\\\/Iydcp3anoCQEyn7trAyR8WcyF1CupC7qomhOuuwQQTs6aBrdE+nrTiq3XC4WTT0IWNtKmJbUASGAYWPEmOLAf1g2L5lWr+lLgskQpgY\\\\\\\/nWyL1qq5wUE7bGAS0P8AYvrarVQOcKrYaKrJLdvmPE\\\\\\\/v6yt1ow2TpOCVmdME35mMbpu5uXrom9bFU4WY7ccpIE+14AHoIvbyqrnkMCqLLWKkFkJP6A5KEkC2oUZUkOiJwvmd7lX3TUqc9ao6RB6bqh4p8WiY+Rh50kTB9PyXxkbqOrKh+YXESoLitk+xkGicTErCTofgqMxzjW+\\\\\\\/Icz1h6K9lCbPUWT7VVaY01RPERykFcGJk7HEwAmdklb8v9Xlim\\\\\\\/BJtkj0sSsWLTgSWiorcP+DyQmwW+jOqnwOTPhKb4G8JPou+iGkKbe9LPzRY0ZMFr\\\\\\\/\\\\\\\/5eCiBgsmVOgEsI\\\\\\\/2n3lvbF585eTXGKOc3aF5Ou+O\\\\\\\/tw2VsQ10ORVxFlBpDPk1ttkeudwWvlV1N6L6H\\\\\\\/Z2RdV4P7S+Vd+S3cjhZ30pigYQC86ODdEATjBbpU8zEvu6\\\\\\\/36oFahN98WoKUfU9UD0Nmx7wYs5vqoRd5ioPtTlJdMgqvWUgu6sprJSdLT7JvKIohb3SK1IHxb9Qk7zgOWOpnTVt6mMBQTKM7L28nya1bgdYrY5Zw5jjnFoEeG2\\\\\\\/7GpI6LwMj5NKMgJPIrCCXsVHHXED7DDuiwKeMdLhVELqAd7daBQF+dgGLMeIz6qvj\\\\\\\/4zocGvK0ykyJTZdqM34pqGCsCBpNAfSfFWsq+gej\\\\\\\/umNWf0R1nhulNdd9IjNLfnx6CZ1CrBmvdt2ZHBRdzXDktXDiJX0uZPS\\\\\\\/xZzBmwXLv1iHUuc+lNsC07mOljX72hVle2ePe3TA4n9mYF+mAC2qvTxOu60ECRd6qvEDaeHSCNEj0iwFZsFjfGFU7WlafufqZfAVWz8i8olRmDofHphBqYxDdPWR4zZ4y0bHNNm83oo46aNThvehyyawTyWPn\\\\\\\/yhBaPbdgwpIRvITi8\\\\\\\/FkwXNY02nQ248x\\\\\\\/zs8Bc8vW\\\\\\\/MextEa7+ZFdxqSCtQewGJ4n4lG5aEvFhHhlDHMsTN7jWQCaerMflA5rshgH3SpRTYJgCI3oBDtie2hN4bY8N0SdbpeRJzRo8akNsglWq+drZah4C0u8SBRQ3b\\\\\\\/nNWBbb\\\\\\\/9iBwul7QhCyCYRm8kOqluZ1rRlcgw24+CWm89Vw0E2tQMpKp5A0lVs87Wo4sL3b77wy1xtb6gLl0rt86EvOywRWYjZVpSy\\\\\\\/Ixqbmc\\\\\\\/hHNxPXZFknSKBWQqbiNqPhi3auFqpIDr7dkPX\\\\\\\/X4qEdODU1RwhLLftgujdId\\\\\\\/cmxnoraxDAnkbcWiHCKrtQGudvtBkojwcVQef6uxD\\\\\\\/4VqcjkloZxbbgpKKNnOpiQzFQZ1OOYy9w6AxqJuJXW+mJinpBGMK7qo4ysUgafOceUCC2uM0haK30xtJnJsBfMs8ecro3bwcM4c5QD7W8188ElfHLDlfd1MzT5hGcvDWLXppjbDLPrtn271eIz6SPq1gHqFngycA0FIneeT8dy1qGnQyCshMRfItKXE7GtjOehB+Ueq5iRA4hZwN\\\\\\\/K2rwk9DQF4cuchsqYizjfFohA1iVDFGD39gnoY6sOVuHZ2eY+5jyPIfmv6yfFwX0K5cn7CXYwzrZDFp3rCPJ9MU0BtKqhuo23cV3Whs76K4nH4OK7hiy75XvCG03h7H7jXYN0tJpkxeFb1Lx\\\\\\\/3pb5MmSt\\\\\\\/xPbMJFFwuj3464r8veb8tkdPrY30Zad6bd\\\\\\\/IBODRRBq1lWIkLQ2+KqKXy3iUrfUna+mXhuphqLwvaZFM3a+ODUO8IMviGKdMrfYodZxRnGbTzsb6D1fmBi9Dpoh4ERz8jEPiIz2AhAETKkQDQqgjBLsoNYhxOV4BgFmuxi5VSlf3j456+ue\\\\\\\/w+G\\\\\\\/e9bDHUnltkKMOxOXnIOuNa01oIAGS76Rzm3G\\\\\\\/80d5PLfB\\\\\\\/33zsvcuC18GxSE6fgJBMbdodYAliZAPg9+jbT56pOhgannaOXx+KNrgEa6z3hFBhRhHtAPn3tcFki3qE0EOTdkeu77ovFVqFKB739gvEjZ4lbkPHRpOiHuIzegJ9\\\\\\\/jPKREQrc5C+1AsE6bfJH\\\\\\\/GzbETjWKPZA4+BlTgRr+1RzAY+1i3YRlyiRgzOowYw6FLxiLuzbQq8X5nzOq1DI8zOR7uT9FO7QYi+C\\\\\\\/AALy98RaiIGesdciRpL7bNhREF10dgq+EGQMU1PFNxec7BiRHlWasPEy8lu4zHkE9RTZyE+DalfH3X8ZSuHNG00gDvYxkL+2eSo5BbYQuEXlFAYvSM3DJQwhQaqsCJE85+yU3lrKV6tBIXbS5QuAh0nEf+4lfOj9Gnmm7iRzfrwLGci4sLqUp8dwZvJBDFLsQDcJdcdNID6GaM30p5l58sOpZZ+YGwMe\\\\\\\/+jc+cYpKl1O0snYsZR7Bk9fBPfbgFU0okLCDn8juPWGJnT11jfSIoA9J5t1WrORywZuuZkzvdgzUY7unR54D2vgQpBQlGuiHy96xYQJg8Pvtn\\\\\\\/sLBJSV4mn2pyhTAdIwX1mrl+Y5pld2u\\\\\\\/6nbpzrDaOla2z3VoMNJNIduJti3Z7v0b08TnEaQFVp5XymMbMnGixsHqoMZqBK96c3x4PIaKFZiRV2buF1afs1rqTYVyCTL\\\\\\\/m+DmsTKgEzAuX\\\\\\\/qxAoAD+RCvKp5XK6ecjyJncu20iofjGn7rp0uN6B9erYlA2Jb6CEb5J5TDBDfPSHwIFr1uGTC1MZCZSYyADI8e7Ur0lJ2k5rqyWzTZQn+Wtr1ZjuI7uHqIMwAON5RRvxftsEpks2vqnutn9a3ZIv0Lh0jWk\\\\\\\/PYqJgt4BR8B1KABkBhhaEukxfPJZ8BoJ+jK3Hp8XMxtdUAs9HER\\\\\\\/rIPdK5XPnWuFjqIq\\\\\\\/g0PR9jj9Djw5WPpRYZRutVBgMZ6uTQ1gS3LjwJoJLB1Am+LL12QVa2IuokGwWSivADPU0qZZaSWqVXhce2DrdhLCmJXk\\\\\\\/l2Ep9ljLEhGK+fr+sJyyv8ex702MMlJedAEbwjC3cPYBvGD\\\\\\\/WOxEoo52sBsiS0kt1troag0PHr5zNyKbpN33kIqo2hCna1hT90YEFAoQ14VS\\\\\\\/pkuYf6fhjkXzbUFci2LC0XSdLqD+nYncdhWdGyADq4+BuToHP4d6OyatEfErMEKmu80WJwZRt0f1eVW0mSNSRvH4vdPyJwdTgI4cfqzlfGXvy5U4NO7VHGIe2ARJzSYRgS0FU6b8gq8atXmDGc99XIZKPhuqUtY6wmhnR42T6Z5Cuh2q0RmG\\\\\\\/cJX93I61N5u8xCvZ\\\\\\\/2GARk8vDovTUc9r1IrCZQaO0adQBO4gv0yyh6yEY0P0cJD6WqhHtayPUDp4Y5ACWnvz5EgBnKIH84jeMz7m94gaxuXqILXAjkiO+AGyGGvfCBOgh1hA4PJ5GWbKXXFXKGw8W2WD26Re3vmIQDf\\\\\\\/8Qz5An3lo9KYPuPxv7FaBm1i4\\\\\\\/h8XddsvHDKXYAW9TfteRoSkjlFh5cS6A6245qLU77UbPft3RsfMxmoGBRE5knr6URZ+lnT\\\\\\\/uP5CcP7xBEe\\\\\\\/5iNXuypdDNPlCxRk1C92NgzySEiW\\\\\\\/RTW8+s0yLAQnjxGMDpi\\\\\\\/SKXAsi1\\\\\\\/eWG8UWT91I1JamO3jnfK1Pzzl0U4dnpcWbyA2T1izJM3I+CQoKjsvc+JEt\\\\\\\/w\\\\\\\/6Y6cxEICWKvwVLltQreJOinoRy39\\\\\\\/878UrN31tfSIJrnv4wzyZRwYBfY+\\\\\\\/i83Cid8c7lvUsx2wDYYacxUTZeUi2DKM1rqZ4+PJLb5hL5NmQeUMUOZJ3+4hlU8brNwOJjH6jVbIsxbhRV6uSm0gFSeSxZIiWdJ0GE7NucvrLNYFRKwZ8wjxCGiCfc5Sc8RENFvz3npY7uz+YVw8z7YqrCmqPl1JEqYiZVNSPrhlXBEv\\\\\\\/9dcNJJHIFH\\\\\\\/VLA1xKxXrypBuH3vV3+sGenTtBifNjzyBSP7LCHzovv1+cUV63dZfTU2qSoPzmnnUr5Y80qFIEoPyXgyP+GKvjM\\\\\\\/9gRcKi4fnzf1ratrZnhbR9WZW\\\\\\\/Zq51mBjUuUg3AGw9qpkqyPLH9wMHnvWf6O7xd21glLV6lNt3Dquv3MWidoNASNmA\\\\\\\/okkwLFdtWXlarPWbJiMJcz48vSzXK3mYuemXPIqIdiZ\\\\\\\/dwVzB9jM4PHQIyXpwAk+d30zHeHuKg5uk1kJH\\\\\\\/EnTPQW+X6qzHiAlI+dNGTSb2GWbQXFZa2N02bVz9a23uLL4OB46XlE2Fam1SglMtGD3Z0DRdM9UkoliO53g5a7R8hhqUY2nxWj0CCq1tgWOKFd2Ew9TTPI\\\\\\\/a1JvxAr+Jmu+V+WKhIb4rsz1+qgbcSVzM\\\\\\\/cuxT0T9I80YQSxckbfCef0oFk35QmvzrwzP3hNF6KKTtq+V9rLGPsmvy5ixnRwHMEk6L7Jz3ruGSjHoUa1ZSg+0LtfPAUi8ddVPYGvwOadlKEUQ63SPluPwKNow6+cAiZNmmAtfqKRc7elaHtlJHm+iM5HCG98XkwxdtQJuKASS3u\\\\\\\/aBTLulrocYP\\\\\\\/0lHCmdJo2mJrK4As67OvIjKH\\\\\\\/kRU\\\\\\\/F41cLwuZOABbzOFKv2\\\\\\\/6n9wLh8yrluOuokDDjHfVfUTuS5q+RJuAEH90nohg5sDoU71sgiDdVdQtJYUwq0EFhDlYZVrF5c5Edfi+6JE7qkU2R5yqkaX\\\\\\\/ZnXqM6Z+FeJ9mL+r6G1WO3rZ1OSjRf3MIyL32kx7RjtigHXLF2QGelm4x6hZfPDXpNerOpYwQuerLGKgkNEVUas933dQzAWQl96u8hMOmXNwnkLfMycwIrH69wGxMck6DrQQrfFoF\\\\\\\/f52UuWgPnhNuKBbfCX5KTOBhHbljq1h3cBjK1cnUCvK778aeF4MAmA+yRRyqxE6S5OGBxQmsbIgswzux7yXzFQ1ZePENunZwSFJ0PhxpZny3+TJPcwonnl+mhYTOR1CKtGstJpeB4iWgddiwIgulJwY4jiOXTgqHMB\\\\\\\/jbvbEPWSGih6n6UcF9nBLKC3RMA4l86IbFQMwViwLbv1rrx61FDTjO5GNV24FTPoJPJ7av4jVuGUQppIvAceCgjhzRwqOGFaEMKOx768bTdfZ\\\\\\\/OOXYN7a5ye4B7gNoqVVgwnrh\\\\\\\/3WcxDuo0cl0w6T7K0EukvBCrMU\\\\\\\/c3hqcw1ftnTyIg0AANFLD0Pv\\\\\\\/0LtoKgCQROPFMF2oEBVIUOOFOvK\\\\\\\/BvLtosZHPix7I09W5T\\\\\\\/OgpP4M\\\\\\\/LnfcCd1b\\\\\\\/Gv63F52xqTfKjy295+HcFwxk2ZwHLGyEUFcldmFMCIj4tjSO7FebUXCtIa8L+aIx1NPRczry8G9yIDxO2nKVp3sT9YQkQSdPbJShOzalTZ9e7ldxeHX4tIrIY5wWmvQ9FiodQSIl5J752aL6sJlKUt55RB6bF4PJeB+ysOGIxTEZ29J0LBYQnX8FwlC5Fi0dAKdHcAd\\\\\\\/9NwKySkDwvgUuXCohW4MTQXYmAtisrmpaajcNRPB0xVCx\\\\\\\/1IcLcf74oqy0k7kRYF5tzYrKJIMlRfwz8ZjG1GGfW8yjuZaR2B3yKRwno3xkZhSMCINQIQg7jGwMWVz2EhsndBIWsERuNEJVbD2Y7RRSdhD1GTIAEJEUD+YuYYquFFkFCShXsF10sZGrljE3ZJxcKLiVw7oH9CvrLfrXLcguIQ4wQfyEp6EtsfEvQ3\\\\\\\/s+qgIPGs+zzLW\\\\\\\/gvTki4MoD\\\\\\\/hMxrSWf9zGlTGYK3xiSRyBOqiXDVk9OmMl56yoanjwjxBYOWEh\\\\\\\/4ZQ7er\\\\\\\/paobavr9GWnKI6aZ7aIjgWChUayxBSISg68nWupWoJqJsF9\\\\\\\/oRQoup76sZ4qHcewINWXZmOS71WB4\\\\\\\/NoiKso8kijMATSWVyEUFj6SsgSNsZF5ew2EO4vP\\\\\\\/RAFsUFoDgmwiX4RLKPk6sMhP0hceZcGIKcfIqxgkiphWdXFv2XWfkVz6u0YrxVdnpzdfBA8ER0aaLm+W2Q+\\\\\\\/9BRJt5FLw6hWYpMra6EyeZpyu4HElPnig9tYXQ+F7CWjWQTVmXjmBf23csLO5tVYSUjHp5FnkJIy1i8VBAlYqbSfXBIp7Vs0AAA\\\\\\\/cs\\\\\\\/dXRuGOWKowzKJCrud7yRHRU2hk2Iafckojw5L+HkyxMvVuSKrsQDy\\\\\\\/WCtpziwJRlXAby+gqaDy4VsCywpoV1I2BTZeW1XGAJx3asOB\\\\\\\/6iYmLCDWmY7JWA6UGxLpWhJO3x91zCMPtQWqjMGysexSbLKyWzvd6FAEq+6qQHDjp22zHD7Q+qlkqzZOhmRlbup\\\\\\\/Ue5J2N8Psv3QDtR17Mhd0\\\\\\\/IW62deI5SyIQm3v7G5zulU3oxHT4EuyYes7vr0fgyxz8VAT7LJLO+wr2icY43dzk98JBQCArTQ76tZXHmIjcAPUI1ZeYwgi74EUBIICJo2eStr4htZpGnvCGyAzsTG\\\\\\\/h8LQgu55YhvXhs+L0tqLHLXlU9ruX36Ovf0d\\\\\\\/coAZoXBc9\\\\\\\/qW08OCwdQ+l\\\\\\\/cHiBt\\\\\\\/ieWcD3CbRkd+cZ+syy1zgv0baC41qYmkgmn6LditulyK0nmNufZjAtymw8StfyHrCEo\\\\\\\/2u6y6kNddB0V1GyRQA1AalX3uzfWJ2BbghwzVVUXeXNQcBmLK46AkZ7PQEz49gksJ3Y089MhA0ax9YtfA3gDltFEkvsLVbvMQ\\\\\\\/dc2sun0nEA2IVUqXToke0XmdV0AchCiJZgOobgVlVfxwc+BX51AdXtMRUv8RkKxNWHMesl9RilD+iomPncOYkigGVpk8JIkjIeZqdeHQ7+ox4LDntZSgcvqKk6Qtnja+bmEAJyfH7W97AyHlrIonKcGMQT92mEdFt3GeP6S5aFiVokg+foUm07GAeJy8zi53zfrMnRkkDTaVez5aqQ0fNAquKgleBq8GVbOhtcIq4bqQCkpt+95VvvON07ouOQJ4iXnJ6HgpgDZJFR1qyekDW7xfuVfIgGeTufbQX4zDhro0CuNNKyS1r8VfzyrX3Unw6mrXZOSBgBIgAnuNvDWXulTT2QCeAgb4BM3\\\\\\\/AO1nHXZvP1hZaLmGc8gDsHagDzRtpK1LMMjb4aeqqc0TydHb\\\\\\\/+u1NSlOop1ManhNuUN4h4TtWB0hg4dF4KdIv8nUBLhbFeHfviwW5+sYEX7N1v6i9yXOwB4VHRXesFpfleNBmS\\\\\\\/VTxWNZUmbmnr3I6+E4TaFhjfWoxJi\\\\\\\/dum17eLMBMk\\\\\\\/5i\\\\\\\/OkBXqwo4INeS1l9gNL7t3ABoURLpSOiz1HmG1iWuWuJHH2WohHdOX7hO1BUfsa21dJYy9TS48uzagID0tqh3UHow3CZ3FEtVZ62hoi8g5FLAHz4+oAqqIaDks1jzUp97kv2iULIeaLKNEKNSyEInMqHNvKot7ifjEtCMJTBt3qV8H69frbbM4uq24U5ZfS0g5wOekbE7Rvexe\\\\\\\/HUgRAOJ9w\\\\\\\/7Lc0\\\\\\\/q7F8CcBUQKU7hh12lIxof3HZw2h7cdpjV+48MBjkobrjytFhyE1FWJ8m0E9NnwAIe0WLCPtK5iXjtxLzDf2j7KAUCZyhXXKSSb0KKlHA9l6fMW5WFR0357Mja9GR4o63bHWSMy2WGMwkbkRwc4mC1gBGwTl819Y+vN2Do\\\\\\\/Rnu3kZffcZreQa9YdKRVmdHNQiUN6XvE9qFY4oR0Mosa1c7K5zWj29ISqidAfIPGZ7jZXgmHocoG9zMcU8d2El81ufrd6lIMRyHWpm2QhD94YvV0mCTO9H0EvpOqr+65mPVsfpURIGpBPocpsydt0XJyUrbkd7We9CLPiGRq+0X3OhVvMRK\\\\\\\/HEcEgfqUjIc\\\\\\\/NaqBAEBQSSO8LlXOtmmslZLiUDzvNZhdUvPl4w\\\\\\\/VBWYZVdQIpnLayjUD9CA2UzMUsBMASZcvThcqzCWEuCponjDYM8sKk25oDXcHYf8CKUX1uVUUA40i5vw29BTRMVfulJtz4twsvGAelSUzbDfX6yUpGBGH9fNOhYsPk2bAv0\\\\\\\/tkHVW4QfgkGYO1KsJ88H62piC0KqodJ8jkhTg8mYuwJYdQuopcou5a112ScRJSI0Hk2M1idrjejlmxZ5Ozl6Oodg2d\\\\\\\/RgE5HIjXtyiyPkjuQoptyHh5btxZw3iVKJvnyYawZeBG0rWLzRSs4CzSuU2lHJYliiK+dSxnCpB+Y6hrpc+G+DCJt0iWMmLnVZzA5mDeN3x\\\\\\\/HhcBxGVPxMeLfNxnYFrnGBkTcrYJEreWgomzkuwSwZ7kKeyjp072miAOP7ufaJL\\\\\\\/bZm1cH1hG6+YX0v81763GtGV2vzD4s4wz9V0clRg8PLvgoXupnOU+lTRrt56cJgjtBsRKDCl\\\\\\\/9\\\\\\\/eB5jqR3B\\\\\\\/SGVtpmEDLKcHwjaEnzgGNN4t0a1LTQN4zLFoRYAIyf0eOkXARTc4aX0JEcuKcMtUY72s0ZQVkfUeMGFq8IVEKEzllDSCOAGw2JgHFSE1hE60GgsU5jTckvrOMXej4c3Ww5eA3dJkI4a\\\\\\\/8a8naZxA3cq5PKG1ZvMWssTwyOivtBSUaJiFFGtFkUd4pEVUDRo5baOVWv1WNyTCK9An7hrsXD6mrvdAyBVe7opWVefhkTLRKNM8FnxaF2GRJJNo73+TcUX0XvB\\\\\\\/7tZ+Fcxs+dDYY3YSa8XDMdfN87apnVaiD5gnI5V0F8AqRihD3EyMPYHaMB32YnMBZJLGMhVqBs5xM4S2FIHJMZIBhvV\\\\\\\/9J21qxw2GJ7HZ6H+YS6ixcRn39hV4Y5+4vUvd8HBi7mSy54bDnBNdjHy5iVChx87nRsb1NycpGTU42WM8B5yVKV+9cBH4Y27B4b4nHbCU\\\\\\\/LAgc8I0Ls1gjaX0zA41C7J6Y4wj3UOjog9xBtIKEbFVAKd3z2E0OF+ApPEOBEGPdhiEMOJJBrKWY7u2pGBwmb+oRN09pggJVzVXylSDjWOMqsaSgrOYXfY0cbqzg7xH3jF3GbiRhIU1lYam8vnFO4HP0FqrxIOK9wk\\\\\\\/nSjNfiiOtv+4YxU0tQPUjQwvjPMzcIo7jsxbXgB3nhgVEVFWhnrVSZ9eQP1G5X4BNMjk+MY8+x0Ev5TNMZlwWHikvEwTJMdNAzYvUnL1qR1l234x\\\\\\\/JZEMgwIxtlS6tPah6NaBN2SaFDjoV3t1cJ8u\\\\\\\/86p1EHARIIZmuL9KTY1DOmqi8aeQ7l42yB6kCidWktO2cvtw\\\\\\\/Zp0yPupjEe7IWLJ43pp6kj186hzSXFnP1IDWv\\\\\\\/E583eXAbF1lsRpVAHyh6jTmmz7czr9XL945nnfVHHjbi\\\\\\\/hvwM4Z9fBDqLmqXFfort0Iac7kihxUzbiGaQ3cwXJP8f93xwaobI\\\\\\\/ZyXBAOBXnhddy2NJ2r2zbb1JKHgeBj6p1jjBz41JVJiOUieElktxEIyPv3wkuvoDwGEKSWaXXKsIrsgLg5sfK8UMBjK+HeGSfcKwexckPwMGCI6HaoCRgNa3\\\\\\\/dc+xiNysFV9UWTZSxvh6Vj7F84DRJdXu6DMPlVkfTBUGe3L94MMr\\\\\\\/HymxAKmboYD\\\\\\\/REr9vqu6ZwyC6WZYQcKdJaH7FR62A6Bl4lw5ZV6l1fr+xnS0EZkfGND\\\\\\\/5HGh+S7wmAZ6mRPYm6fga4HESgagdFFLbFIXOMwumUcowiDgTgZvkXTQ8Nm7MGn6Zytc5cKXTQcE6T\\\\\\\/YUxnspsCzbC8Tk123vWIv31A4EMj+Fm2ClgJGwnq+P1kkOh6VV7E\\\\\\\/i2n+PAbn8\\\\\\\/Fnxyx+D7fmq9Bd2iyvFmAPdvm3BGjrLjXCIuE2GxKVyJWe1Ju7jMKzJGzs9A0PXBs5QFrz3JM6tTiuiA0a9ZkpJ6VMHXjbmkJk8R\\\\\\\/R\\\\\\\/N1klgYLvjJcf0ymlEeZf\\\\\\\/RPsxl5zJkPdJetWLakynDQfOycxiePXcqhKlsUSPxHvsi0xFfmFBS+6I5xrvEI6xcGeVWoKXkmSKaJeyrY2TuoxTsjsjcaO9AdrQ5SP0rgImgQx57fJvpyQtWasGic7+2UvNEpOEwrjiX5ReP4d67LCFKYMaQ4hqMUDcUMMbY4bnAwfLTUNWJUlzjS7wfzMbshBKwOhs0r14tSmPoonPtH+dGGM3tyniPqftoXD1pHfAPKgPcV7ulPEtWsjh21C9NvE7SVlTShR9RiHZgtkeNiTtmvQQ9VnqP\\\\\\\/ARc0aHCBxGEJJiZ5apNMRMwipGLsFI3mFYZeHhdy9ocjL21XSzu715BZ\\\\\\\/jmhJUT\\\\\\\/Yb935Z2iixuQr+qdyAHrzOGPpRxUXttOYakaBbwPeWPAaTD\\\\\\\/TP96fL7e\\\\\\\/gKLDYqYP2mJUh+gqKST+d\\\\\\\/LhAFIyAjYb\\\\\\\/6RFB8bdcnze0xpt2GcI24N9vUXPZ6EC0BliPfK9etdb1zQlWPvBHUqqgpD8741i8yn2RxZNpkW7s5u3sallbrEMPivMzXkHecEsKn+LF9B8FquybEaLrbPJqAanbknz59nGEoQuOLcGlfa2eN0awLzf8CoP1vn2W+Ez8HmY6pFMIWddNDqB0wg4uywDP\\\\\\\/wob3HZc7eH+u\\\\\\\/neZBS9N0VF5yjWiwn3z6KzEvnpVlb7mJ+oqVs12ZjbWIkZvCGQBxcHPgiir6+AUgPZgT5L9RS4N9CcnnINE741e15fMfqqfxe0nSVxikbZy8ZgduG08CrxJHGPhE1R6PVdZ3qAnmDxAyR3JVxegBXlgdgo+CctwydftuITb06\\\\\\\/sQsNVfRw0Li9Dibsfz6tfhiJT7sMcldYlUynzCzoKsHnL7d4h0IWydB0AqXJmF+FTpH7KFuSnhIH9SAV+7ocqGbSosuHPDA6MNnwndQ7ebfFcHPfMqu1qwHpbAVjCfewHZzn+JU1O+XVBV4qHSIAautM\\\\\\\/H6fzZnNKrFmPYN8WnAc2\\\\\\\/zQOr4dJOYOQr6S15ACfi60ICmOTpE2WM70T0GQV+XR6R+j6Ia27mW8xuORmZtuXUO0wZf1zA6d6i1DVpA4vzhAGaIVdxzrfM2KAqSoIOrRaImCoRsdx9FLBCu+Ax3rWYlCPWNRGHmKQnm+IMtWwCcgg5RUMd+jYjHzNzjbdi1Rwm\\\\\\\/2ZPLce0BVSOEgilVgdhhKeW3zQG238Q4bhAH5qnr6FaKqMVpijxKno46Jh6rzvSU\\\\\\\/v+xuKGviWjz3mw62JHl7FZeOrGLQVLCEQBkU8DwX7GH9vpjHLrdh5my09pt7A+\\\\\\\/0XhqMC9xkGMvMTV0eYQsfFutyngMKI1WqHXR+\\\\\\\/hsrh\\\\\\\/P\\\\\\\/93EP2T\\\\\\\/A1wzY3wr\\\\\\\/At\\\\\\\/QhkW4iCN+\\\\\\\/Ml2ZAnNS91eCLc2xSgRCTqtYlexg3I8fggMUA4XqTkAqFONRxTCFZcLpLgwLvEIbpZymFN9YtCjeNYKqfFHFzysDlhcNw1djWh26Fl0OZFl7kIFOks6DKmBlw7TEUn3lgYdqctePGBGWCC9ycRFcTGCdeh0Miv+QgnvJxk4c9yecr0qGXzjLBGLMIckxYdd3IoHV7kCxhVpCLyJYmWS8bQlDFZZyHJ5jJZU5lgiN1MshNYQ5JWOryIgCh1dyb1REe25htF4wT3femIwxkEqSDnsGyDB9MarMmoMxJchzAykGY+AH\\\\\\\/TT58N9tgeWs2OzPfArw8Cgw4788MriB9\\\\\\\/m0U5R4a\\\\\\\/unnuxIe33uYphyxE6jJFXPNY\\\\\\\/lVxHZf6ZpmIq0iqJFgE9MlQvY9zX+Cvj93KmKt4F8OtLoDM7JPLaRxSKXIIY22g2JzK3icEL00ysM7YCE3zxgT0hTU4nGP8isVtPqKoj1bUlKvw88gNH62WzhxH6\\\\\\\/udNLm7jwR2wIyiot1CQg\\\\\\\/DIffSOPG4JZRT7OajbTjgtTDJ1cHoqLrud\\\\\\\/fo0vlJG\\\\\\\/FS5J\\\\\\\/fwGNN62IICw47U91zDbRdr7KC8tznoOhxN\\\\\\\/8yxoH5uoqjHEnLIdnmlKcuphc6JUtTwq\\\\\\\/JhHD8K8LYmveZdG9zIaOSJAQUV9AhhK\\\\\\\/6dlIn\\\\\\\/rhoraIYRgZwkCEW74HE3LadoxTkLEGZD+zQdqhfhEwkGUq43io9T2Avuzk\\\\\\\/7esvtfEwiHu\\\\\\\/LnMb\\\\\\\/+Uz4v40HfcYWDgRcLfKYMrrrEoqxgZrZpL+lB3X1uiRbksHDwcV5\\\\\\\/GCAfVX6u6AFDWSvtaUkAPNes2wSzgEInG6OAYhmaY+CLoZc\\\\\\\/xWk0S\\\\\\\/1AXdrtHdPHaRmERkDe8uOhQUWTCAfMVedb+7FQCKCgIgdu3xVfta7YTV4w8dnqs77kzB3iunAyEQAhpyVNo61hOyrO4d9ZaxjgmoriQF2gPthOeG2WnYoqDtsUuh2E2n5JsusNwsUQtIx2OHnBSYpytUgRSowD4\\\\\\\/BRStKmEq5oIWVguwtjLnge19K\\\\\\\/Vh9PgJVVvxPJq8wZKSPtnz0AQjeNyZkG2F1zQbpUraykCb9z34jExknV+4W4pjxWUGYpHUUh6kWFtl0GLaM\\\\\\\/rkRI1LvtE9XSHoUyCV8rkJjxU15wy0lTXjHiwwbnVFECVp37aIdJZrN\\\\\\\/9mml2kMF9bC4cxzUMSz5ccmRP+8AkJ5yMqgBi8C07fGHoEYhHd6zL+qqZnXMpqeGWxDKw5ovcfhAV5uFXwaSgnOme7Ls9wjtiNQFDE9TexmfHviLwOlOqXcSxNmKsy46FTuIMlK+2dx8OI+kOLzxYf2K4dHnz4egTy17eqelZD0yw\\\\\\\/wjHm1VrmTy5bEC8iRuXxmyP06XgYTaTbF7Z3\\\\\\\/7\\\\\\\/dfL\\\\\\\/OhUIb11WN4yMfxActqys675g1cuV6PQB4YOvHV9DShXoO3ixwzZqRbxxVKx287vDK5+vCi5MEm7UTA7gXwTRLN34x2WePTW11SHfH82sx2nOxcNZc2LWQXKLrnE4wbzR4wc+tKYFclp\\\\\\\/wJk2BxB\\\\\\\/6J7ZHQBG3+tOsdYzxA5t8UIOh7Sj+LMCtbJA7uBCds2Y2y9BSIaBMnfmdLDSllkR1oamftG+mUn2DIyfb9wzgzFBikm6WCyQEtlVNOK+HBWEZpoVVbwRP39Uihi1cqJBGBC7QseQtTk+R7qbo1g43Kyt3PcTLVmziFufZ2VS88+bW\\\\\\\/bTm15l8azY8hvBJMirUgvBf2\\\\\\\/lgKgOeZ6sAyV6B6zPY+EF8FsCDpkJFE8CfE5H6qhtUwRDa0hG8iDL4LhAQ0lWSDXn8gZTsHWqWKFOrpyPyblc9cnLlQESRV\\\\\\\/EDedEjaf0IOONjxts+2rftDdWiqGR6ZIYc7TKj4kUNcSkHqT\\\\\\\/1N7+pvR3bERluX3\\\\\\\/RdjplFIxXctphArKVqDwqsmY894\\\\\\\/K831DcNRAsyO+kg42Zb5s7GNYio\\\\\\\/ScPvGAKFLgmqvWURYnaCMGUNW+\\\\\\\/ux1uvhNMrNpGfEabBRGIWszcmzMmfFp5CNzTczSVK4oE8GTwOOIwQ3EVYHha4HZWNDUgl9ZuzsoBZZ9thmR0dhCQAc3iAXB8r1z12axuu5aKC9o2t\\\\\\\/ZMvZZNUFKDDpraMDrlx2WElde3vdjhZZK0wyG4EQomsgS\\\\\\\/78+yCSSCKiB3qmhsc4P7nBg0N3ntLFhs4OINdcM91nYfuw0F1RHIqFSX\\\\\\\/VluXBr24+CH9tZl6z1mABbtq4GCFfm8Gi0gDqos+lxdCwlSQdpT4gTeHxpLZe\\\\\\\/4UlPXFEPypQoyQe+cDK8cMjpjE2KSgZo4SrQ4WIAFh7PPH2GtoMGFS1EqguVk6YqQyvWW3EMF5Cy3FjIxXTcDOlaMxSEixM44AtDC3uHzgvqnV90veiXoVNFzfmAHxilFMwWzBZxHThnZn2GasekjqD92lsYGF1ORjQCbm7W6zkSqysCRmr9VA65HJp1GRWmxh6MvRLBhnOCGf9fRYzVJJSlUZ+MANf6tWMPNqV42JBV1Wrip9Sp2vHMuHVQFjcYLHv\\\\\\\/dl\\\\\\\/xjgR13HyI+RYQViQtwvoX90tJ0lIHH0Ed9mivkLmuEUUUon60DHyTezEAUb0FOL4UBjbaJzJxRU0nlIHAptJFYAP8kfd3FB5QLW3O65KYNJ9ZnyzIm6q50s8i\\\\\\\/bGsD5IiDRWao9BYBFpxYg8QBqeUbuwQFq6uE3hoeGfEp7wH4L1Woy4qjSmo03kK6vMiQLeT6VpIUmocdG8lhIBb2pNAmnuojNTaquMVcY83BVkEByi\\\\\\\/Zs8N+n9O5kHHrJEPKxMo84b\\\\\\\/oExTnbvrOs12nSeNfSapVq8eY9d4pTqryr6rr0pWD5a9fQKqgUm68I3rYzYrjYVR5qb1VK3vjRtRXpYJmjlRzjhRhnKdFiJAoKjrYvg58gmDwW4YGd07Bl8Lz49irO+26OKtfjtpZlCP6Z7tW11IuuQ+adeeZE1UZvy7ElL2GbUH4FIBwigYx9Pb5hDGOd\\\\\\\/\\\\\\\/3rPA9u67FVjMTqCXRR8g3eASQJDyg+tCZO7LrmHIn3q05nR9Ps2CVuugVat8sfJb5YyvMxScfXEh2jypHN6aelWdnCl1suke5a0jctOv4Mop6UqiYn0XglqiMq0R2GhY9VT0fLZQ9iYJBZLkoBuWVvrRemr\\\\\\\/N\\\\\\\/futvyl\\\\\\\/rkZYmtVr537rKU0+22F4PG0m3M1MLsPjdxRiUDfNThfZpm\\\\\\\/oF0IBwac\\\\\\\/xm+BYY08b936lJ7fqVcyGLxnTerZEOj7bCvIB5bNQd4Vs0w427zI+15yzwDV3QIM9rhxMkH2jjUA7uXq+9Cr+hGxNGZEAxPhPfwd+eneuPSsfNB2mrS44vVgeVIW6kYfHjcR9aFdL3CBwP\\\\\\\/41ZrzWTE83diSrRr4vWLEgSczKE8rAWQ3U2wQY6HYUrcvV5xdxExYUwp1JN1ihDrb1ZaCyl8\\\\\\\/fYT6XcYHp+YNxlZ1CXRUcEMr829G2SYn+D18wRW75hMHPgS94zGgzXMSccNh7v3eWIGoCEoMBOmi3AvtpUKaPcchCBCVHBSpqaDiHz4MaMfBmOCBdHS0Vsed8XZ4lbCApQCyloTBBloHssuuj1bXwA\\\\\\\/St2yAMrsiYQ+ZBugTXROe9C1YVR7BOrkc7+7lbJnHUfHmLgf+W0LfDSccy28m5UCHvrolhYR4pwZtY5cTDluNjLiLbb0zHQj3P3cr9yDuWdi60rle5HE5KRxQxR8AIvRSPeomwMJ6ucFbQfslAvGG6rT5n9pNIR8Xwet15JbbP6XSG+TMYrywRzU9muyZvvFtul4K1kcIURkrHYf5x8zbQvs+HPnWGol7A8VzmtVDfxTr7RpwLTAEM4\\\\\\\/oN8\\\\\\\/KYbFlMNqLEFRkaO7Z3SC0aPGhgQRfV\\\\\\\/\\\\\\\/W3+qHFnIV8C7sdJsTTDKWYa9TF3n9BmmpuwZOjoxsSocYUCe7fl4FNkjB\\\\\\\/ATjmA\\\\\\\/FR+5WaZ2VMqPPr66Lwl1OZVdo3gT9MuF\\\\\\\/HjzVWLbNydk5HEdkjITTFP0uILCca8rmOSQQSLvSdXy3ivN2IrlDCIFN3RWQPh5dLcgHxFAHOJtU6iL5yy1xxr1LqtQCzplJCMkP9GBu2p6Wq\\\\\\\/4DXqXccFL0lf99wUGr84Hsjt7OtC15otMGpx12acqAesw5kcbhPj+\\\\\\\/uFDCtO90zJz0fHoPus0EfLRmmH6OvYnVwv57Kapnjk5H6Hf3zlzmwvUwceta2ZEH8lE19vOIYamvkvcEVnv4iZLrg4Kh+B5fpoMCfBReJPgJko8knfkp1xwf8vUTK8p\\\\\\\/QTMi7jpVsp3n\\\\\\\/zjQhdyP0S107pDjN7y5g6VB5fJRHderjtiwQbTMcsRaOA74DffUryMKHEi89V+wQtxQhDZR1zB8VL8KcoNLR8mx06wrER\\\\\\\/lqNef2AmHms3X5doDqf\\\\\\\/VeerZrwckX2Fp81rUyhtzWTCLA3N2gX7F1AgmmLjm3d21rIksp5fuIR2ol5aPyWBbqnVdgTYfu5nchEEyUkhYVLtzV6XosZTnZUCfRdd8cNZEm4Pb0kPx99Ib3Zmqhzj8T98rer5+BQIAwLGBAEK7yKuRMvRyGoaQGdIGcAopi0E7XcmvPn6Tg\\\\\\\/hG+BUyU9zE2xfragXMdbpP6p5McRcuw4qKONcq+ZrdnlTBwh9O4PR8tGloIcBosjDAWCkUBl8irYsrYFD1v0QmKa4njTBTeujh5+SYRJBdhRXfsqjPLURXnRTKv6lBzOmz4uvCKd5LKIuxnIQxqV60wHwN4dgT1IUYXhEoIDA2Be8kRKTIn+V0TP4BKwCdb1E3J7JcKL5uv+jxcQOJ+EDMQU4FBDpzKDB9CCI4iHXm1Oo6ZZMTvJCVjzH\\\\\\\/4RUg607ilyg4jBpRPrMNzbQgDvFu+LEP5okXM+i8WiGi3Ggqs4nVuMLfmkRBkt5nedgr9pGuiv8+XUWdqbTJWLewIb8wKYT\\\\\\\/UGGt\\\\\\\/t9jNbLpsnXm479SxQYqIwbYt0nke2U\\\\\\\/uL4\\\\\\\/n2NOJZID\\\\\\\/9nZQVykNDsN0WplA99ORYxHVB+MDIGgDIBXSeS6E4KUjwxii8KArRCkx8MV3CsSyxnmEOmVSOpzynt79yHBFOFMq4yrEB2AvwJmT+HVE39EZ2pQSgqjfdoqSjsPvuGwlIh1ZAUar7ESMOZXR9aKWUHtZv70fhW2IHRgjR2pxyvr9u0ShEBUAFE5dXXDH+AqDr+3bRZAaAbUjZXnQhLXlUrqPuEE\\\\\\\/aCnEkMTgvNJ0wJ\\\\\\\/EYiRohg5c98TFAH9Zhsj62vO9k1wPop1SD0iK04mEnguApqVzdUDo2J8pz3NF\\\\\\\/0eVupP9Px6TCr5ehac1SaMJyaEEP0\\\\\\\/eOTw56s0RSDozHBxJqFRGuNCkMdxYuDJPwguIHeZQ9Lm53mBHEvgeULUbvyOof0ndnidIWQNpuianBOSWEpIafMNduMDz4yuAbgfID3jschoYBwt8bj9UwaOOluthJdb\\\\\\\/\\\\\\\/zX3E2xeyKhqnxNUn1M4rtK6Mkw1jCfL9Ei5etoi7+r8qfuROnXsllzKbpeU3yeuaqFvgZNv9GQTIf+pOAV2Ri\\\\\\\/HDfXPmwoQWVDBDztXEDYjd1gDvFxTgbz6vcuaGzYf+cipPoz\\\\\\\/2hd02\\\\\\\/xAwqNcfvvvT8jjdj5WLhyMcPDguGjBWw3Qa\\\\\\\/WapH5fDr8vIXIyOV2\\\\\\\/KnAgdEKDroleN9M\\\\\\\/Sd183ElfVDQh+76e5TMrcr1NHUxRlPRO4AAHBVYxEnJUP5ioQLi2zLZzLAwH+siTM7P716Jr6GenUU4J6jzgSnz0bkTUP3xKYLM41V6ux2Xgl2SWtyNoi8MZIs0Nj4iL3iQ3iGb9Ek4oFG+GcbUAdhP2L5EASBUD0UFWWxvy1mOf7Bb97hq7I1iLVKrJsPtW8YuTRfrDWbKfe+kp8D9YlzwXKjSj9S+4pp1bbxM3avsid+YWj+B2LELrEnz+u0LPUqpXFhAI2uHUr4eaaibctQ64Kf2V1IsQ7Eob2TQ4fofKZTT6f9c+yurf7pqGFu61K\\\\\\\/IN1Ccorhq4BPVHaHLCxTWbl8R6A+iPRxaGqFrOsO\\\\\\\/WN6FGlOPtIF\\\\\\\/x+gV1TfeDOOXw4K2yDPxdRC2O2occWCVh5qetnAyN9vW3tNX7wajTMdzAoXJnurHt0faRus4lPEHuOIjLY+bOSuO41MgXPnfCFQNoWXEDKuGrc9Z7aVCs5eSlGvtp8uXeR7SXRsK75mAc7VhTe+855PoSxcidTyl+nnFkT+5VLCXpUFkFVypFPoSKC0kSe2Q9\\\\\\\/f1QuR8mMIrU7DqJF6g8wS7Olj7t9CYUS0D+B\\\\\\\/OmZtFaVm5uPVDNFu+scwpDUKXgb7DlDG6w6sAU+euay+wQ6Ds95VmwEOQbUkKACFQstetH\\\\\\\/9FMQYT2Xk6lReHPqV\\\\\\\/HmyU5aqqYI8YCcArOWiuQmJCelk2rnUrw\\\\\\\/7ok0DGWpM+DvmdOa+WB8iuqPg3gNrc4pv2g0z5Md6C5URVx0bi4qPmg9+DWRL4HNf++2wlkfBiV3YRgvTtTm08jWO1Uj0sA3eOlziKEplW2eH6kT9UHcY7Axo8cVlh4GPpc01Et1dlojpKPgPjgSPmTAnIzC6dsr70CB+NCS0RJNQVc2XH4EaJC4IXVvmUEmN4nmkPj2QBF6z+QUb\\\\\\\/k14C3u3cd+po83CmZ967vr3b2rU7QRELa9m8aBk3jtQNeg52WpqP47bSpqQ45gkE2g21hoCy1OpRLdYi98WY+IuBBLpfGILBYrm4eDYbZblKgQJWrej5Rk52sM+DiYHIpFfq68Xp5+ZlRkqDIZE2cjXnURhoBHJi9vZs1VzalWS\\\\\\\/0oWZC0W+gvFLUZR6KUrrrz9\\\\\\\/8FEzgaz3z3yCm8HGokjXQkzdA2LTdwh41swbD4s0Xc+0c\\\\\\\/MrYY4Dysn5w5GRdnVKGmMDDd1wcogHMUwpgq\\\\\\\/q36We6SflopLLqjVxbh0nWa2usjQnubVm5vajKEjgadk18wHJ09Fhc\\\\\\\/AwhpPx5symPSoJy+89xdMKw0OZZ9ZDXXk4+K+e1a4IZuyJ251IdLdVc2\\\\\\\/CTugGCnPqhsm4VFl5a\\\\\\\/6Zt8Dg9L31fbuMEFpi6g6PUrGXTfcssgmaU5sCkRA5W7MtndYx\\\\\\\/aoHcRtXu58+vjW6rckwHH4IkPQ9J+u+DgOWkgxXnvnHSgb\\\\\\\/AV61E59\\\\\\\/3JG0fT1ro3iK\\\\\\\/5csMYHhPZoWI54t2+NpEWYLiGtsweV10fMgFRYPNyZHnY9r9nbIgdCURYYgwxhNyknR5VGzqMydAPyyd2\\\\\\\/0RBlL1htkGcuAhdC\\\\\\\/Ddg+E4rvJRzYM5X0LyLcE5sGSjCKtlGLKOmVDV0JSnOma5A5UdKAFrNUgr1Xm1FZHn7RrzsbqbOAJZwb5VHPSvsIfT3H4MsuX3s9ncwxKGmS2rS7wOVcOLdY23jY7HhG0R9NAUvKZw6g7vlsaYpzRAy18UH8\\\\\\\/p6RABYVS5vLocd9YcTijSaCjo3vukInUo6E9frMLpMHw11jYCG2wl0ySQn0n07BPaNOOPaq8G7LiYb3TXuwDbH3rg1qSDbPhOlrPPgvujkXIClA2flEgJAqqoocgHTWfY1AoqeYdZsYWeyMKbDj5S2iR0JLXNLX4cZgDLuN\\\\\\\/EoTHkT4QgzH\\\\\\\/sfRa2nr\\\\\\\/c7TaNY4kjhOILLQrdHACEeqCwCq\\\\\\\/PVezkbx7itEj9Kh\\\\\\\/gMuLlixtLPDQdW050v+ZHEJE4zyEsYzvno\\\\\\\/3YQkvKsjYP10D9QM8Kykj3qw9bkt13WLRSFqP\\\\\\\/TGkIPJt4BaCf4sKU7SNKilFM8wbNwXzEyvtCkLV3OloChPN0FrQrqm6GM+aa7HiX1LJw4PXng3pDfdeQm0+tGeiHNcrBuVIKhBc1E3gI2L1z2qD7V\\\\\\\/VDjSDC6OUCv4yOQ7PWqzgigVKfQ2UYvQf9tQQz5TTzQeLiU92WWgYDxCaeXsTplHEmMMVrpxh18ylX1DF7J9l151U3kSsd+0u3k5x5ktjtndkHoevZOdPTrrmWeYIy7q42gxBVtXb6N0+lWWrJLNqSvCJ71\\\\\\\/w4K\\\\\\\/HDHIjJ+W1lacNYyw17+FFogQ9lsHIoa17xxVRQYRz3pvEYwA0rAobHL9LeVTvgoK7sBDUTz6T1l8N2p8QZ3qt7jwuH\\\\\\\/rdXx4qj1lrsOwij3q5Ah3FSlGdlD5+GZ0xeoPiax6ZTXJfIvOe4c3Rb2H0HA0+VVtp2rTH2jFU5+OKa1fyMkoNrw9qF2Eoan9zarBrh4gxMk3Kpme1SlMjnSLZU+GZsVgNG9KNqgbN2t04UfsQVJnKCr0HXJ3Wv5ysUUx1CaAWdX0BLEmgYszQzDR0YtDui3oEDfGVzA\\\\\\\/5LyGFY4SteX\\\\\\\/XGrnpVR1ioahP7jVQ+YrEsIjCHixhryQC40nTP0qwkIHjoG6KXlawbZZY1esjesiMLgvVaHA+n63o0aKCNpkN3GsHX7vxRCcZ+HpdsK5swDbeD6hqO5qllH\\\\\\\/L1x3bCv9ZE7jQNNxVSwhdNyEPQnLaJ7IJr8rEHL5UYfOkrpS366o+qLAu\\\\\\\/GJFvGlaIcLs3290yAhSTODmxsXKJu9D7SdbSdidzHvRG\\\\\\\/zENdmGSxLNQZfKof9RniUyp7GDFQ7NOWp79WSxSL9663giiq1YyY5upmu2qG\\\\\\\/3\\\\\\\/Y00xyLifMPIErKZN7A9Ls1rjGL8OGImesaaKTfaCi++srcGiuEu7PHPbVkPRddfv7XTgFgx8gcFVDFc5XVdbZkxrkrejnV1\\\\\\\/YqYfuWqT09CbLndng0Rn\\\\\\\/qq7Vc5ABHmVrz1w0Q3YdW9UmhGkE3S\\\\\\\/4U4FEMRJpjkh9SsQuC+sr1CZPHgchh1SQSaehfZQ8eWiVbGJbW0mYWLbIqcv3wgGY3yNaSdklZv++YFe2PSQVrMnW05hkNIhS2Rdu9z8mmqBueE415G9EbKc7DAnj19D4F43L\\\\\\\/HTROA\\\\\\\/0CFZ73Ssdyn9HmB8JhUIAlfsO0g+ku0VIb78RkUGfIehdHRF1C3EezqIkRk2VXCXIiIgioR\\\\\\\/ehj8RJwuPUXpt2X4YG9heBz8OYnCi19IKWBUe5vH+Yb938FnLrdj+ng038WCUgG5ZNSNXgCzKW1xcvr+\\\\\\\/jmCYsAPKtzDGNsIcxOr+2RRXevEQM\\\\\\\/eLhpn\\\\\\\/t7iz7fb9SCCiV5rRVwBRE3e2NHSTgQKPtQxuntK8bcrc+nh+CRRdTRoja10ps5Hr56ZW+YDvH8sIMQoLiFE1Xp896OE22JQNur7X+x4rCzee1XoaSREv1nfXH1FR\\\\\\\/GdHB3C4thA3KDQvAyNCtZUIXfRQ8eZPEeJgZMPHNHX\\\\\\\/D96DqUm2e+A1bZr80U6zqd3+VfeHoSEks2p4siGHurbfpdGxIiyMjFU9RQ13ZAPaupByI7nlQ1x2FPY+2ZTEIvKBKrvbEKijYB5PlbeuHuCWeoZZSLRTRVookAQwZMijZFOk0GNrGXmjRPImFdGrgKQV1X6+blYZYzFXG3CKX47J5O8wGUx2PPvtVnrgCPJK\\\\\\\/qnu7K0RCi9dTDmE8VF5aAZ3epNkq\\\\\\\/9RqdvlZYVLvAb6NfHEANAJnS+q5V6320RQk9d7v2ziORti7ahpmhcs2+avnl5NMzBvznwSv30rjHvi0X0DAsR5ID7h2y6wPiOe0YnZCSkXY2o5B3WlPxBEHILpoiXrfLrlLpJcByVf3kTx1MOJs1ZYNPrDUqHkGwmKqDU8jq8R6hFVzPY1Dc6voqJriKOBBYWJIKScUCT8Y9Znp73+wsFo5Ib21vyeVPjisuVA3thr1gSDoMZo4PKwntAc5aOTi88d+hweZbzG0ZB498QJvLquofCa919Be8dWichk4nqtoGWtCbOtcq979YssLuGlFEvkMg8xCbImmt9AlRLXaXozMR+k8LXe\\\\\\\/aknVn\\\\\\\/UA6sJO54P4GkMHvNReAL1ZniwTobDpob2nZ9b7RIjM+HJXpE3nwdbJevT\\\\\\\/R54alpScU5cwwxaDx0EbCXXwNmMr1CXto0J9UbfLYarSfTJPSEwaySqe0mY3L20+KAqPB+N8A6Ch1HOKlTBDb6w59rXlRzLUDE\\\\\\\/nk4oV3ieIAOJsbC43xDHiO8GKJbwFvt988xutgYoyf6TQYph2YJLv89uDncbNPlzOxXVrBplEoc7sdUiTSUDulXq22hsEVrD1HbGJ8Z487Wje3p1jyWvQiK\\\\\\\/6OCCoZHa8A+C8tKvGjLLeT2X5LujmG533AMawquWw8RhBwCvgj4sMvRKM6eniZXutHacMpNIhizEJHQ0LlRRRqcBf\\\\\\\/l0st7wkp\\\\\\\/IhrpJqA2IhzaOW6SjNW4wuGw15YC7uyidSuLuClpLxvdlAW31Zv6MZ6SNDweXw7wmFpg\\\\\\\/1rzt2LLs9+QILEalI3SsFinPkF+V09\\\\\\\/F83KzmbVVGiK814WEGGWd67dH+Veddr6o0+dZ7OV4QbGD3uD6e77ByreS6VO\\\\\\\/sOwuZ7kck\\\\\\\/0Uk7Pk0ptyAoP+ZQsOT7GiyqRY8CMELNO10YKfzjnHv29kfk01RPp7vjeXu2jpUUvkEpSCHaEtwRQ0URhDjhngxHmF6x7vrg9lEvLf8sLI9lQLL4dqfLmtMMGnpfUY2hMm49f1vuJCKNrXY19YCEer5xyQ4wZSXyOf2vbMygUFDyx\\\\\\\/W7+uw3gnRKG18ifF1EgZwlwqCfxr0Z0V7iClSER3WAz20b1aNmcDj\\\\\\\/gEKwUNiKF5HARN\\\\\\\/aiNTOURMdAyICjh6+sG4qJ78ss3S7oLdRroV1tKe9YMW3cSKvA43of0ro\\\\\\\/Ir7u\\\\\\\/qGyGAMsR8pyxbIMnoQH9cSyderpaI+D+aDM\\\\\\\/NiMfhIPrYsa1tC+ARMYu1s6veEZQWB3iZFQOnw8xiePIbT88X16G6pY8raCuoSz9k21LhYMCGwbKIyQsN7FjcWCL3ANh9KnXcrZ0jRkCDk6yNiPRyr\\\\\\\/H60J2uMABk70hW8OX7hIAeThaRHfuZnxPbusSBE1bi3LWTPVBw7clPF4Kf6yYqWR80A\\\\\\\/IRAFkZO4pqdmTJf8VvGIaBt9W\\\\\\\/7UPuS05LFhCOyO1zdtUiegQcei\\\\\\\/hgCFzlsoMUJfW4FD679reobo\\\\\\\/EsiEE5VDp3BBO3zbF5pemUhsNeQA0wfNxXJmvsrZIgMf2iptt1KliWozy+QIgKUHbw5Gujf8+KdTVVhBw0Lhus2dbI+eTt7oyZxAFp7Tv7HQnOjgSk29AVTEV7P4GpuCZqE+YDiDg1WkVDN0guuoIYYLn2r01qZzgHseuj3r998qMsM59tsSVBXYJgNl6SwUwKTCQTfT56LkT2XvDZappe0tnkwWlEwq0iZEHvbVq99yWgpiSuDR\\\\\\\/SG4ciNRh2Z80S5C\\\\\\\/lwTCIckNeZ5EC8BCcJZ6eQ8vV5ffH+UrMPJelrxMtBpZe3UgF3jPdss7UFy16jdUhDgCXsvo14iC1VWAGW7j+W1VWkwqErhKSUTRCDAFkCU7IQEPAFAobLjksif+A6zOLQWfVDOmODh6kwwmSDgE7nXa1wxgMcXo+DbIEJG+SJIRbVL5vh6azu\\\\\\\/Ecs5lysAgpCmI3XuO9uKUeLZ5oz54fc\\\\\\\/69B05eY+2J+X3C98Jb4asGdI9p8f41oYGSkaMMyFGd3PpcSdjXpdSpFAZuAQxkQaXB5fXfdFanXjSM9Y1cHC6tOTVKQV\\\\\\\/1IhppsTMuKE8D4UACRbmxCzVY\\\\\\\/lQZSzBa3\\\\\\\/Cf2S9\\\\\\\/x+fbVqnK+lm4QCdIipGTfDq2QnNexl8H7QfM6GSAc8f1DpJPjEEajT3EPjmaV1iTu8KxWALVn1RLw4TV73kBJt7C7nXi79fu6SmIpY0Ccd2OgFiNoF7ldNydRXVjyBRRGdeTQvBktJxqN4ogCI2PeMXlP99Iw92qQfwk9M6Vv1MReIxGna57tMYIrzj7thS2BMI4yl3jRdJfmD69jokJUcBNg2wo2UXk18D0sj1c8OIMJmarfMbIpL4DQQA8m832OulK4V3QNMnqwh0gKzso\\\\\\\/8jFrGIok5Z06UiTQdQ6nY95ewLrY63kD2AFc5m0wnjbXrve1dGVQwmYy5K+I5YEmhge3vy9Dl7CWyK8qkKySaPy5GHRbZGnzFogoLnecNSzJftrsffKjAdvUhJ1i61FjVIwX28issTW2vCMgNxhUw\\\\\\\/DV3yLqtkHgGKNPclvqnVIXBmSp0HbaAbtHctXxZSCzGLA9FKLcgTxuDz5rH1w85I1utzhvhR2x82kpykZbccZ1iOGcEcVP5blsvAgESU6kw50tIPPXNaaDE3DtYI5q4wh+46S4SS7AQSUB8v2exIkmouEg4GV6r82D8K88HZFgWCmIsg5Yhyl97Tkgo8fSDYfWI3orxcuGyC9jAIyVktmPc9\\\\\\\/gveFLaK\\\\\\\/hhBuTIS9zwJQYskF5\\\\\\\/Vr0zuRJEk56kzfjAYT6iTbxCzfbB0XcVjlmX2AjbtLap4unjkV0hsJkDQ7Sbmz5g+ie7\\\\\\\/ZcvsSZ8o\\\\\\\/2IzAPv3iM9ggR5v19vQYYc4tAzIekc7bgSoIbdPoxCb3zoYuPXRLwZ8R6ZqAH7bRVisaKxhCmiEzyhxzN2tBxBVvEH3Xvc\\\\\\\/XVOrboEq02XCj+J+q5MLFsc\\\\\\\/pnbOL3dVHOsAdvsMx9K2vHiUDsD80ZDgBntyG3lJxNO\\\\\\\/RhXjCSk8YpTr+R50KaQ\\\\\\\/7FlWH0Tfmz997YdTGwg32rN7hmY7CR+RP0pbkN4ppylcFzTECfcolNZGhxBwPZcysbouRNFUcIenGrri0q\\\\\\\/8pdmMmZM7ioXKDacyhH5FTnTrXxBGuKSJXlqsYxuPeuTAl+AfBsRGvji2tpvf2zQ3mqwC6XHf7WCDWaROu6HFM+tzAj9DEk8U0ELm3m4kkiV\\\\\\\/d3Kxa7TNvPR3LnIcIk4ZU6WSwtYON4n\\\\\\\/+NcTIOu2dlsJKTThnz2n\\\\\\\/JPLIMwV6BlVyTYsEMRexA4lMAYx+WJfTQwrFEQ3H44eQrNECUdL0rS7pP5Iex2ozQvp5\\\\\\\/kSyFObL+\\\\\\\/q2eUtuxYLBNnX9ui99NhOiiSKg04dmPbEF0\\\\\\\/\\\\\\\/Xw86Sbn4rEtDOem5C9KkSX191kYzPYhhL2jtoowM669cvw1ILniIufwV+RY2NyhiVApyuqLxGISZ49WAV9ZiSr8x\\\\\\\/N\\\\\\\/4xQsBHm1+mM3T6xzb83b9PaGIZJV7eVAZK82FKMLHQW9uG49YgXGzPRG2IpOaEZo9QR82nunw51b8hODi7+FD2wqL1TPegFeaXGd9sw3hRRA7Zzsy2V0exwFJ8jjhQLdI1hAocrB0BrtEEiwB1FXLSPw3b4cLa7WrVRO3fIi3DnF5dsW99GWys2UojcynfwcbJgadTn3\\\\\\\/TYnRizTH\\\\\\\/1lAn64KPzq9MxmTFvUsHvbf1sy45+jwrkVAoEYOxlpu\\\\\\\/vG\\\\\\\/PbM7m\\\\\\\/IHuMPo3gw3b+TjiGie59OaTSQbZzDlYp0wnWAi7vFENf03\\\\\\\/rsAUFwNH4NzRKzWR7NfYRCkHDlrNQf6wlJakv6LkO4XmG9eZnbgWV5XHENbpTfhat8wFZhab+bxm7U+2DAxqMmGpwRRh3GNFdPX2DTloYZJVTiIUOyeumnSGns7kVkfqOU154L3yfrxzE3qct8ErdG052BOGb04yrRFkxCy9r1hp82WNueC4Vqh\\\\\\\/lba3yJbUkcdfBCobqkc5NbiFyoglipxgtGWMa0I4PCjYILgYNRhQ2JRaAZZGaGOuQBZUSzQIDywo6vgGn1pLjDs0YCPU3M\\\\\\\/E0hjNVqsDxoHBOc687iSWnRJfDxTG12gLMZLyKfOjeCoaIvh0\\\\\\\/vu2mFC7Y9Go8HftfnULZFQOsz4DA2hPCHacGL0Tog+nAFQLKht0YMWiWIeSpLoYJFIAHMLqX617IrmBLKmM+62lQoATQRjFivyky20k2oTuw0NwrKq+vOZO99+3aEPhiQZuZT6J1jvgYVFQ1z1taQ7LDiFZO33PTJvR9wn+n8ZeiS1AfJdZ6VU8fKWmUVMHoC87umlz7y0UHxkratCx3mzyjG3xE9Dm10KIFI2r84Hl+Ppipy0XwGs6eL4HgaoFj9vtAh4hC0L8xZ1pMFwhloGSDHnXilwGwZ1Ks\\\\\\\/HwrQtEXQevHIujl5xmY0A+U6jAn4VlAQhtVT6thur9cptpb4Ftk9MZB+s6JxprV5PMMaTHVJ8NMUCLNEk6NAjaebxciIj1LMug56cmD78DkztEdlJBrL2gDRaezm20Vp0X4uuqzm8f3ZGpfgk2XDh\\\\\\\/J8gRuhL7LGbfxEoXbOgHl8WWryWU4mDMfzH294PvK9JeOo8N2KQ+Yueko7PzaxYQtTAPakx+ShP4tvznTxTQAiRGex4NHOxJ4sMKDNb+SOKJZw3fc4s+jpwEeWPJ7bwq\\\\\\\/ISJVLRI9wduO1Z0ofH1mpPY7FCL0HypLYhKW0DF4Gk0M2qkJCG44YFebqGO3WKP41hMEczxbLuIJ3frJ1zrOPPNLucjqqKLvrdCRGHZuAZflXDfKNhDlNi0HmOgPoZS0NWtbq1OJ5rnOUpnQLlYw1sDiYqHUnpF7xBVp9GHwNlJ\\\\\\\/2jnW6QKk371ie22qqpO+XEdf+\\\\\\\/xSA04PQD8rHFzlbXd+sP6pLq0OAqAipDiMzGMxgErWqJ8G9r4NS66UOqTNhPR8JQqekzbASY9buXiVDAbJYCfBUWLagdlK4AxNq32JbuDCuoFnhxojMJw7IE5FRZA7vDMMNg1T0JOQ==\\\u0022,\\\u0022iv\\\u0022:\\\u00227uTFHyYM0F5Q86LokoP5Ow==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002285513e505000750374f04ceed1ece2416e856a7dc5ad0d07df2f3f435a1dd20c\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022TBOxcJxjrQBpM7CMoP6y9f08veqsH7DJiZGbV7gIie48RRRF+25xnsAv6MfP0pRJyHAeDWvUmFzlFMUZHKx33tmNvykq79Lj6Z1ZVfuR28aInYOoeXDAXZ8uYJMCvCz0\\\\\\\/9IbTEZL5Ljt+m\\\\\\\/efYWronTckQ2NvH4qm8nQIDt\\\\\\\/ZBXbFGByM052rBv\\\\\\\/tjHTM0B4iizCHDsX0LLdcAY\\\\\\\/H4VKQye\\\\\\\/7APDBAzlWRCaPoR\\\\\\\/kogVbNmJIPDuutW7v\\\\\\\/+EZX0V9\\\\\\\/lTv+0dt6Os4spLADo+H1ciU\\\\\\\/qPEAJpvi5uFq5\\\\\\\/r3mdeQ9U1Q8GB7mcqZA4OWYkjqYuHux+mz3TPyEAMxT75rsgVdDvLmyPvtcv+ITSdqV+6GYhp5pJRFs9QjthFcAO0pZarfMtT5CUx1F8N6WkNwSVB8I0NpoTXpQFNUbC6Bo76bWpJ8oFX8fPjac2ceec2fFH3ZmyXEHgvtKvvYOVyQT9U5lsFCGDnUxoJnmyrzpSwj2vSaQNJ9wgBRgTWB9Qd9NyxIJx2w1VJUf7VJds+Q6UWt0Jzd9p03C892nDR8bf4ytb6gQbydRUgNzAzRZGoxYJNwfCCyAfLcuVcTmw2LlfWfzp40i78CDHjDbXoJq5vDcsbPM9bIg5IT1qh00ntxfPYt8WXnm4JWnJG+wZe75oSB6tcuSEPDrbgaf3dPmO79RKBkVuqBqMK2DIGpF8hY89vp\\\\\\\/SmVdDzP+7z0S9TEZgj3Hw879T6YgJ3+NHIQrf2QmYJTgywhqcFyOcJGCk6mbJSSt+XcHuJgKKOzaPAPL\\\\\\\/qp2CxfjPXWmTfXUxVbJlEPnWR6LWdCmxUUtZtoFuRM6OoTfZhNXIP2vupFccpDaSFUcUS7m2FRX+vnco4oaNP\\\\\\\/hdUiRr\\\\\\\/gWa1UxNoyaSkiIrUsWlRDQhMOuz8Txc5FbkjjqWtLDfZNs+1eWPCMAWltJPCmM9inQWu8+r+hUK8PeuLcVAYAfNFefcp6Tp0JcUCrmZJuVVJyzFgY78sfj0ago+NF7lzzG\\\\\\\/pVfQ7bN+PBxMGV7Jer7+QcqOIFh7zp\\\\\\\/iQgoNcerFYk0iFDj5wK95+IZYrkngaeMk+7n450Cs41Us8o\\\\\\\/aLCm+lLFxbp9e2YN9x1HPWFb\\\\\\\/FdfJsm+S6YwYF02p5aUmFJXJDxm1Z2LA8Q0nStV8gYWkooMYY+CTi85etC9wWJsRKJlYrWZt0uVJWEgN5Z+cvMqrIBJkfoTu2kDPeRxdhD76\\\\\\\/brj7ha52IDgY2Tz2P5PT3L\\\\\\\/Mg6A30hzslEEM7wFIr8dOSI3WFzP6kbB6CJ113wkp+5kEHVeCGp+h7nN982eW+6+Zc5VNVeyQ7sQT2uFIRR4bOTPoOpS5Hpc7jaLjrcGPi9CK31+2dt4\\\\\\\/tHGeuMJD0W1CLRM6XE2VkswinbhEm3iJIYZicLPxD0DzoAIr7GVFIRltgIWQInCKNJVLZwYh3DlllBAfNaZR59v8Z9atY2vGFlxqfDhuD3HNpXed2Ctkcl0zCkYUX56tvTszS5V36Z1tWtE14+2zJ5MjDvON30cQkWREenORmXM\\\\\\\/X09iSNI1MlLGHcuaM3KBZZIL5pPqsYoO0IBeSEfctF\\\\\\\/Be99GFs\\\\\\\/K\\\\\\\/SB77pvcY5r9feNqXCiCx25rtAuEN+mh27+ItWSw23vdZobZoPSZ2bcw4ByiGwFvWDpK9MLgHvKvXPgviRDuiYEEaym0\\\\\\\/8W4yY6jBNB02Ee9r8Iln2NceaZtIqlESNQ9\\\\\\\/dNz6ptvkoDHjTkcw9WrgeW8P+rEOzv4GlNNq25VSr43+ACrGwq1G4gTim\\\\\\\/TIkvnWyZ2DvgHMxGnazbP0LCrRNA0HYyozm+UYhNEA+fgOP+CFQBgyiIRh7i0VAPClhtOzXlvLDLK8VDbwM7ME6qF\\\\\\\/gmw869xx5ihbztAucd7p8eX5fqY3CceZ7klKcIzVR7IxUgs0sEJ3dwQPom0XcD+2c+BDZgtdLHCs7iilQmVE9Ku9Uz72wTE\\\\\\\/5wCRhnp5IHRcMa4F6zfFrQFAGVPlqM4b4fwFDz9A==\\\u0022,\\\u0022iv\\\u0022:\\\u00227XxekARN2J+0mQ3C14SUHQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002293851eb0f6e7f3b87b441f5c2d4a766bbd5448c194efd091f3ae0b056baecbe5\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460046 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/10/2165" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-i7WG_TNM7LEtCz6DZ7toc6GL1V6U5qCvtUHKr4QpbKw" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/10/2165" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-bkdIYWzeV1U6m8yyxeuxElld2R2653C-u3_kRYy4O9g" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/10/2155" title="A Genomic Selection Index Applied to Simulated and Real Data" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/10/2177" title="Identification and Correction of Sample Mix-Ups in Expression Genetic Data: A Case Study" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460046" id="node460046" data-pisa="ggg;5/10/2165" data-pisa-master="ggg;g3.114.016493" data-apath="/ggg/5/10/2165.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Similar Efficacies of Selection Shape Mitochondrial and Nuclear Genes in Both <em>Drosophila melanogaster</em> and <em>Homo sapiens</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Brandon S. Cooper</span>, <span class="highwire-citation-author" data-delta="1">Chad R. Burrus</span>, <span class="highwire-citation-author" data-delta="2">Chao Ji</span>, <span class="highwire-citation-author" data-delta="3">Matthew W. Hahn</span> and <span class="highwire-citation-author" data-delta="4">Kristi L. Montooth</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2165-2176;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.016493 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460046" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Brandon S. Cooper </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Brandon%2BS.%2BCooper%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cooper%20BS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABrandon%2BS.%2BCooper%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Chad R. Burrus </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>School of Informatics and Computing, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chad%2BR.%2BBurrus%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Burrus%20CR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChad%2BR.%2BBurrus%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Chao Ji </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>School of Informatics and Computing, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chao%2BJi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ji%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChao%2BJi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Matthew W. Hahn </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Indiana University, Bloomington, Indiana 47405</div><div class='author-affiliation'><span class='nlm-sup'>†</span>School of Informatics and Computing, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthew%2BW.%2BHahn%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hahn%20MW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMatthew%2BW.%2BHahn%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Kristi L. Montooth </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kristi%2BL.%2BMontooth%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montooth%20KL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKristi%2BL.%2BMontooth%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:kmontooth2@unl.edu" class="" data-icon-position="" data-hide-link-title="0">kmontooth2@unl.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/5/10/2165" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460046" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460046/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/10/2165.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460046" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460046/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/10/2165.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460046" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460046/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/10/2165.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460046" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460046/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/5/10/2165.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460046" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460046/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-4">Deleterious mutations contribute to polymorphism even when selection effectively prevents their fixation. The efficacy of selection in removing deleterious mitochondrial mutations from populations depends on the effective population size (<em>N<sub>e</sub></em>) of the mitochondrial DNA and the degree to which a lack of recombination magnifies the effects of linked selection. Using complete mitochondrial genomes from <em>Drosophila melanogaster</em> and nuclear data available from the same samples, we reexamine the hypothesis that nonrecombining animal mitochondrial DNA harbor an excess of deleterious polymorphisms relative to the nuclear genome. We find no evidence of recombination in the mitochondrial genome, and the much-reduced level of mitochondrial synonymous polymorphism relative to nuclear genes is consistent with a reduction in <em>N<sub>e</sub></em>. Nevertheless, we find that the neutrality index, a measure of the excess of nonsynonymous polymorphism relative to the neutral expectation, is only weakly significantly different between mitochondrial and nuclear loci. This difference is likely the result of the larger proportion of beneficial mutations in X-linked relative to autosomal loci, and we find little to no difference between mitochondrial and autosomal neutrality indices. Reanalysis of published data from <em>Homo sapiens</em> reveals a similar lack of a difference between the two genomes, although previous studies have suggested a strong difference in both species. Thus, despite a smaller <em>N<sub>e</sub></em>, mitochondrial loci of both flies and humans appear to experience similar efficacies of purifying selection as do loci in the recombining nuclear genome.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/cytoplasmic-sweep" class="hw-term hw-article-keyword hw-article-keyword-cytoplasmic-sweep" rel="nofollow">cytoplasmic sweep</a></li><li class="kwd"><a href="/keyword/mtdna" class="hw-term hw-article-keyword hw-article-keyword-mtdna" rel="nofollow">mtDNA</a></li><li class="kwd"><a href="/keyword/neutrality-index" class="hw-term hw-article-keyword hw-article-keyword-neutrality-index" rel="nofollow">neutrality index</a></li><li class="kwd"><a href="/keyword/tests-selection" class="hw-term hw-article-keyword hw-article-keyword-tests-of-selection" rel="nofollow">tests of selection</a></li></ul><p id="p-5">The effective size of a population (<em>N<sub>e</sub></em>) impacts how effectively selection removes deleterious mutations and fixes advantageous mutations. The unique genetics of the mitochondrial genome (mitochondrial DNA; mtDNA) are thought to reduce its <em>N<sub>e</sub></em> relative to the nuclear genome, via haploid, uniparental inheritance, the mitochondrial bottleneck in the maternal germline, and a lack of recombination that decreases <em>N<sub>e</sub></em> via selection on linked sites (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Hill and Robertson 1966</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Maynard Smith and Haigh 1974</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Gillespie 2000</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Meiklejohn <em>et al.</em> 2007</a>; <a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">White <em>et al.</em> 2008</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Charlesworth 2012</a>). In addition, cytoplasmic transmission can link the mtDNA to selfish cytoplasmic elements (<em>e.g.</em>, <em>Wolbachia</em> in insects) that may sweep through populations, further decreasing mitochondrial <em>N<sub>e</sub></em> and possibly increasing mitochondrial substitution rates via the fixation of slightly deleterious mutations (<a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Shoemaker <em>et al.</em> 2004</a>). For these reasons it has been widely hypothesized that selection is less effective in mitochondrial genomes than in their nuclear counterparts and that mitochondrial genomes may accumulate greater numbers of deleterious substitutions (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Lynch 1996</a>, <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">1997</a>). Analyses of sequence data in <em>Drosophila</em> and mammals have largely supported the conclusion that mtDNA harbors significant levels of slightly deleterious polymorphism (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Ballard and Kreitman 1994</a>; <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Rand and Kann 1996</a>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Nachman 1998</a>; <a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Rand and Kann 1998</a>; <a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">Weinreich and Rand 2000</a>).</p><p id="p-6"><em>N<sub>e</sub></em> is not the only evolutionary parameter that distinguishes mitochondrial and nuclear genomes. The distinct functional landscape of the mitochondrial genome likely affects the distribution of selective effects (<em>s</em>) of mutations that arise in this genome. Animal mitochondrial genomes typically encode regulatory information for replication and transcription nested within a hypervariable region (also known as the D-loop, control, or A+T-rich region), 22 transfer RNAs (tRNAs), two ribosomal components, and 13 protein-coding genes—all core components of oxidative phosphorylation (OXPHOS). Outside of the hypervariable region, there is little noncoding DNA in animal mtDNAs. In Drosophilids, 99% of the genome outside of the hypervariable region encodes DNA and RNA genes with highly conserved sequences that function in mitochondrial protein synthesis and aerobic respiration (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Clary and Wolstenholme 1985</a>; <a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">Wolstenholme and Clary 1985</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Ballard 2000</a>; <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Montooth <em>et al.</em> 2009</a>), which suggests that the distribution of selective effects in the mtDNA may be shifted toward larger negative effects on fitness.</p><p id="p-7">The mutational landscape of the mtDNA also differs from the nuclear genome. In most animal taxa, the mitochondrial mutation rate greatly exceeds that of the nuclear genome (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Lynch <em>et al.</em> 2008</a>), and the mitochondrial mutational process is also highly biased (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Montooth and Rand 2008</a>). For example, nearly all mitochondrial mutations in <em>D. melanogaster</em> change a G:C base pair to an A:T (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Haag-Liautard <em>et al.</em> 2008</a>). When combined with the strong A+T-bias in this mitochondrial genome, where 95% of third codon positions are an A or a T (<a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">Montooth <em>et al.</em> 2009</a>), this indicates that the most commonly occurring mutations in protein-coding loci of the <em>Drosophila</em> mtDNA will change an amino acid. Relative to the nuclear genome, animal mitochondrial genomes thus experience a greater mutational pressure that can also be biased in some taxa toward nonsynonymous mutations; these are likely to have deleterious effects in a molecule that encodes such highly conserved functions.</p><p id="p-8">Some of the strongest population genetic patterns in support of distinct selective pressures acting on mitochondrial and nuclear genomes come from analyses of the neutrality index (<em>NI</em>) (<a id="xref-ref-66-2" class="xref-bibr" href="#ref-66">Rand and Kann 1996</a>; <a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">Nachman 1998</a>; <a id="xref-ref-67-2" class="xref-bibr" href="#ref-67">Rand and Kann 1998</a>; <a id="xref-ref-78-2" class="xref-bibr" href="#ref-78">Weinreich and Rand 2000</a>). <em>NI</em> is a summary statistic of the deviation from the neutral expectation in the McDonald-Kreitman (MK) test (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">McDonald and Kreitman 1991</a>; <a id="xref-ref-66-3" class="xref-bibr" href="#ref-66">Rand and Kann 1996</a>) and is calculated from counts of synonymous and nonsynonymous polymorphic and divergent sites within and between related species. Weakly deleterious nonsynonymous mutations that segregate in the population, but that will not contribute to divergence, lead to a value of <em>NI</em> greater than 1. When the efficacy of selection is decreased, the expectation is that the number of segregating weakly deleterious polymorphisms will increase; this is the pattern that has been observed in mtDNA. Meta-analyses of MK tables and their associated <em>NI</em> values for mitochondrial and nuclear loci in animals have concluded that <em>NI</em> is predominantly greater than the neutral expectation of 1 for the mtDNA (<a id="xref-ref-66-4" class="xref-bibr" href="#ref-66">Rand and Kann 1996</a>; <a id="xref-ref-57-3" class="xref-bibr" href="#ref-57">Nachman 1998</a>; <a id="xref-ref-67-3" class="xref-bibr" href="#ref-67">Rand and Kann 1998</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Betancourt <em>et al.</em> 2012</a>) and exceeds the average <em>NI</em> of the nuclear genome (<a id="xref-ref-78-3" class="xref-bibr" href="#ref-78">Weinreich and Rand 2000</a>). Although the relative sparseness of the data was recognized early on (<a id="xref-ref-57-4" class="xref-bibr" href="#ref-57">Nachman 1998</a>), and the conclusions were largely limited to how selection shapes animal mtDNA, these patterns are often taken as evidence that selection is largely ineffective in the mtDNA because of its reduced <em>N<sub>e</sub></em> and that mitochondrial genomes are expected to harbor more deleterious polymorphisms than do their nuclear counterparts.</p><p id="p-9">Here we revisit this pattern using new, complete mitochondrial genomes from <em>D. melanogaster</em> that we compare with published nuclear data from the same samples (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Langley <em>et al.</em> 2012</a>) and with available human data from both mitochondrial and nuclear genomes (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Bustamante <em>et al.</em> 2005</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Just <em>et al.</em> 2008</a>; <a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Rubino <em>et al.</em> 2012</a>). We find little evidence that the effects of purifying selection differ on average between mitochondrial and nuclear genomes within flies or within humans, despite evidence that there is much reduced <em>N<sub>e</sub></em> due to a lack of recombination and linkage with the cytoplasm. We discuss reasons why <em>NI</em> is, on average, similar between mitochondrial and nuclear loci, despite the distinct population genetic properties of these two genomes.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3><em>D. melanogaster</em> mtDNA assembly, annotation, and estimates of sequence diversity</h3><p id="p-10">Raw sequence read files from 38 genetic lines of <em>D. melanogaster</em> from Raleigh, North Carolina (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Mackay <em>et al.</em> 2012</a>), sequenced by the 50 Genomes subproject of the <em>Drosophila</em> Population Genomics Project (<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Langley <em>et al.</em> 2012</a>) were downloaded from the National Center for Biotechnology Information Sequence Read Archive. We used the Burrows-Wheeler Aligner, and specifically the fast and accurate short read alignment with Burrows-Wheeler Transform (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Li and Durbin 2009</a>), to map sequence reads to the <em>D. melanogaster</em> mitochondrial reference genome (NC_001709). We allowed up to five gaps, five gap extensions, and five mismatches per aligned read, but few reads needed such flexibility and most were filtered out in later steps. Using SAMtools, we postprocessed the alignments to filter out low-quality alignments and to detect single-nucleotide polymorphisms (SNPs) (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Li <em>et al.</em> 2009</a>). SNPs with a quality score greater than 20 and indels with a quality score greater than 50 were kept for further analyses. We then generated a consensus sequence for each of the <em>D. melanogaster</em> mtDNAs listed in <a href="http://www.g3journal.org/content/suppl/2015/08/21/g3.114.016493.DC1/016493SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/content/suppl/2015/08/21/g3.114.016493.DC1/TableS1.pdf">Table S1</a>. Because of the high variance in coverage across the hypervariable region, we did not include this region in our final assemblies or analyses.</p><p id="p-11">We annotated the consensus sequence for each mtDNA using the GenBank annotation of the <em>D. melanogaster</em> reference sequence (NC_001709). Using ClustalW (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Larkin <em>et al.</em> 2007</a>), we performed a whole-genome alignment, as well as gene-specific alignments, of each consensus sequence to the reference sequence and to the outgroup species <em>Drosophila yakuba</em> (NC_001322). There are very few indels in the protein-coding regions of <em>Drosophila</em> mtDNA (<a id="xref-ref-56-3" class="xref-bibr" href="#ref-56">Montooth <em>et al.</em> 2009</a>), making alignment straightforward. From these alignments we calculated expected heterozygosity (<em>π</em>), the number of segregating sites (<em>S</em>), and Watterson’s <em>θ<sub>W</sub></em> (<a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">Watterson 1975</a>) as measures of sequence diversity. The mitochondrial haplotype network was inferred from 80 segregating sites in the coding region of the mtDNA for which there were no missing or ambiguous data using TCS version 1.21 (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Clement <em>et al.</em> 2000</a>).</p></div><div id="sec-3" class="subsection"><h3>Tests for recombination in the <em>D. melanogaster</em> mtDNA</h3><p id="p-12">We estimated linkage disequilibrium (LD) between all pairs of mitochondrial SNPs using the statistic <em>D</em><em>′</em> (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Lewontin 1964</a>), where <em>D′</em> = 0 indicates no LD and |<em>D</em>′| = 1 indicates perfect LD. Because recombination erodes LD as a function of distance, a negative correlation between |<em>D</em>′| and genetic distance between pairs of SNPs has been used as evidence for recombination in mtDNA (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Awadalla <em>et al.</em> 1999</a>). To test this prediction, we looked for significant negative correlations between |<em>D</em>′| and genetic distance. We also conducted these same tests using another statistical measure of association, <em>r<sup>2</sup></em> (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Hill and Robertson 1966</a>), which is more robust to variation in mutation rates (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Awadalla <em>et al.</em> 1999</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Meunier and Eyre-Walker 2001</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Innan and Nordborg 2002</a>). We calculated these correlations by using a variety of minor allele frequency cutoffs. We also tested for the presence of all four genotypes at pairs of SNPs (the “four-gamete test”; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Hudson and Kaplan 1985</a>) using DNAsp version 5 (<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Rozas <em>et al.</em> 2003</a>).</p></div><div id="sec-4" class="subsection"><h3>Neutrality tests</h3><p id="p-13">Using <em>π</em> and <em>θ<sub>W</sub></em>, we calculated Tajima’s <em>D</em> (<a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">Tajima 1989</a>), which is expected to be 0 for a neutrally evolving locus. Demographic effects will skew the site-frequency spectrum of both synonymous and nonsynonymous polymorphisms at a locus. Contrasting Tajima’s <em>D</em> between nonsynonymous and synonymous polymorphisms therefore tests whether nonsynonymous alleles experience a greater skew in frequency relative to putatively neutral synonymous alleles, indicative of selection (<a id="xref-ref-66-5" class="xref-bibr" href="#ref-66">Rand and Kann 1996</a>). We implemented this analysis using the heterogeneity test (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Hahn <em>et al.</em> 2002</a>), which simulates 10,000 genealogies with no recombination by using the values of synonymous and nonsynonymous <em>S</em> calculated from the data and compares the estimated difference in Tajima’s <em>D</em> to the random distribution of differences between synonymous and nonsynonymous polymorphisms. We calculated several other summaries of the site-frequency spectrum, including Fu and Li’s <em>D</em>, which characterizes the proportion of mutations on external and internal branches of a genealogy (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Fu and Li 1993</a>) and Fay and Wu’s <em>H</em>, which tests for an excess of high-frequency, derived alleles in a sample relative to the neutral expectation (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Fay and Wu 2000</a>). These latter statistics were calculated using a set of 80 segregating sites in the coding region of the mtDNA for which there were no missing or ambiguous data. Significance was determined using 10,000 coalescent simulations as implemented in DNAsp version 5 (<a id="xref-ref-69-2" class="xref-bibr" href="#ref-69">Rozas <em>et al.</em> 2003</a>).</p><p id="p-14">We constructed MK (<a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">McDonald and Kreitman 1991</a>) two-by-two contingency tables of counts of nonsynonymous and synonymous polymorphisms (<em>P<sub>N</sub></em> and <em>P<sub>S</sub></em>) within <em>D. melanogaster</em> and nonsynonymous and synonymous fixed differences (<em>D<sub>N</sub></em> and <em>D<sub>S</sub></em>) between <em>D. melanogaster</em> and either <em>D. yakuba</em> or <em>Drosophila simulans</em>. Polymorphic sites within <em>D. melanogaster</em> only contributed to fixed differences if the allele in the outgroup sequence was not present in <em>D. melanogaster</em>. We tested for significant deviations from neutrality by using the Fisher’s exact tests of the MK table in R version 2.15.1 (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">R Core Team 2012</a>). We calculated <em>NI</em>—the ratio of <em>P<sub>N</sub>/P<sub>S</sub></em> to <em>D<sub>N</sub>/D<sub>S</sub></em>—as a summary statistic of the MK table (<a id="xref-ref-66-6" class="xref-bibr" href="#ref-66">Rand and Kann 1996</a>). Assuming that selection is constant, the neutral expectation is that <em>D<sub>N</sub>/D<sub>s</sub></em> will equal <em>P<sub>N</sub>/P<sub>S</sub></em> (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Kimura 1983</a>; <a id="xref-ref-50-3" class="xref-bibr" href="#ref-50">McDonald and Kreitman 1991</a>), and <em>NI</em> is expected to be 1. When calculating <em>NI</em> for any gene with a count of 0 in any cell of the MK table, we added a count of 1 to all cells (<a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Sheldahl <em>et al.</em> 2003</a>; <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Presgraves 2005</a>). Twenty-three percent of 13 <em>D. melanogaster</em> mitochondrial genes, 9.5% of 6113 <em>D. melanogaster</em> nuclear genes, 0% of 13 <em>H. sapiens</em> mitochondrial genes, and 73% of 11,624 <em>H. sapiens</em> nuclear genes required these additional counts. If the MK test is significant, an <em>NI</em> value less than 1 indicates a significant excess of nonsynonymous fixed differences, whereas an <em>NI</em> value greater than 1 indicates a significant excess of nonsynonymous polymorphisms. We also calculated <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2165/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2165/embed/mml-math-1.gif"/></noscript></span></span>, as in <a id="xref-ref-63-2" class="xref-bibr" href="#ref-63">Presgraves (2005)</a>, the sign of which is more intuitive; negative values are consistent with an excess of weakly deleterious (negatively selected) polymorphisms and positive values are consistent with an excess of advantageous (positively selected) substitutions.</p><p id="p-15">The short length and low <em>D<sub>N</sub></em> values for mitochondrial genes upwardly biases <em>NI</em> (<a id="xref-ref-78-4" class="xref-bibr" href="#ref-78">Weinreich and Rand 2000</a>), and we initially used <em>D. yakuba</em> as the outgroup to increase the amount of divergence. However, using <em>D. yakuba</em> to calculate divergence also increases the potential for multiple substitutions at silent sites. Because of this, we constructed MK tables for the 13 mitochondrial protein-coding genes using a range of taxa and methods that capture different amounts of sequence divergence (<a href="http://www.g3journal.org/content/suppl/2015/08/21/g3.114.016493.DC1/TableS2.pdf">Table S2</a> and <a href="http://www.g3journal.org/content/suppl/2015/08/21/g3.114.016493.DC1/TableS3.pdf">Table S3</a>). We used both <em>D. simulans</em> and <em>D. yakuba</em> to polarize changes on the branch leading to <em>D. melanogaster</em>, which resulted in very few nonsynonymous substitutions and highly variable <em>NI</em> values (<a href="http://www.g3journal.org/content/suppl/2015/08/21/g3.114.016493.DC1/TableS2.pdf">Table S2</a>). We also used either <em>D. simulans</em> or <em>D. yakuba</em> singly to calculate divergence with <em>D. melanogaster</em> in two ways. The “more-inclusive” approach included codons that were missing data in some lines, and averaged across all possible mutational pathways between codons with multiple substitutions to estimate <em>D<sub>N</sub></em> and <em>D<sub>S</sub></em> (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Nei and Gojobori 1986</a>). The “less-inclusive” method omitted codons with any missing data, omitted mtDNA SRX022291 (which contained more missing data than any other mtDNA) and calculated divergence using the mutational path that minimized <em>D<sub>N</sub></em> between codons with multiple substitutions. Unless otherwise noted, the more-inclusive method using <em>D. yakuba</em> as outgroup is presented and discussed.</p><p id="p-16">In addition to using counts for single genes, we also analyzed MK tables of the summed counts of polymorphic and divergent sites for each of the mitochondrial-encoded OXPHOS complexes: Complex I (NADH dehydrogenase, <em>ND</em>), Complex IV (cytochrome <em>c</em> oxidase), and Complex V (ATP synthase). <em>Cytochrome B</em> is the only Complex III gene encoded by the mtDNA. <a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Stoletzki and Eyre-Walker (2011)</a> emphasize that contingency data generally should not be summed, particularly when there is heterogeneity among contingency tables, and they provide an unbiased estimator of overall <em>NI</em> for combining counts across genes, <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2165/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2165/embed/mml-math-2.gif"/></noscript></span></span>. We calculated this statistic and used the DoFE software package (<a id="xref-ref-75-2" class="xref-bibr" href="#ref-75">Stoletzki and Eyre-Walker 2011</a>) to calculate bootstrap confidence intervals and to conduct Woolf’s tests of homogeneity (<a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">Woolf 1955</a>). The only data set with significant heterogeneity was the <em>D. melanogaster</em> nuclear gene set (<em>P &lt;</em> 0.0001). Similar statistics were used to analyze polymorphism and divergence in the human data sets, as well as in a subset of the mitochondrial haplotypes reported in our study that were independently sequenced and assembled by <a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Richardson <em>et al.</em> (2012)</a> (<a href="http://www.g3journal.org/content/suppl/2015/08/21/g3.114.016493.DC1/TableS4.pdf">Table S4</a>).</p></div><div id="sec-5" class="subsection"><h3>Comparisons of mitochondrial and nuclear <em>NI</em> in flies and humans</h3><p id="p-17">To compare patterns of polymorphism and divergence between mitochondrial and nuclear genomes, we obtained existing data for nuclear genes in <em>D. melanogaster</em> and for nuclear and mitochondrial genes in <em>Homo sapiens</em>. Counts of polymorphism and divergence for <em>D. melanogaster</em> nuclear genes were obtained from the <em>Drosophila</em> Population Genomics Project analysis of the same 38 genomes from Raleigh, North Carolina, with divergence polarized along the <em>D. melanogaster</em> lineage and using the mutational path that minimized <em>D<sub>N</sub></em> between codons with multiple substitutions (<a id="xref-ref-38-3" class="xref-bibr" href="#ref-38">Langley <em>et al.</em> 2012</a>). The human nuclear data from <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Bustamante <em>et al.</em> (2005)</a> included counts of polymorphic and divergent sites from 19 African Americans and 20 European Americans, using the chimpanzee <em>Pan troglodytes</em> as an outgroup. We calculated the number of polymorphic and divergent sites for human mitochondrial genes by using mtDNA sequences from 19 African Americans (<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Just <em>et al.</em> 2008</a>), 20 European Americans (<a id="xref-ref-71-2" class="xref-bibr" href="#ref-71">Rubino <em>et al.</em> 2012</a>), and the chimpanzee mitochondrial reference genome D38113.1 (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Horai <em>et al.</em> 1995</a>) using the “more-inclusive” method described previously. We also analyzed subsets of the human mitochondrial data to illustrate the sensitivity of <em>NI</em> to sampling (<a href="http://www.g3journal.org/content/suppl/2015/08/21/g3.114.016493.DC1/TableS5.pdf">Table S5</a> and <a href="http://www.g3journal.org/content/suppl/2015/08/21/g3.114.016493.DC1/TableS6.pdf">Table S6</a>). Comparisons of the distributions of <em>NI</em> and <em>Z*</em> between gene sets were performed using Mann−Whitney <em>U</em>-tests in R version 2.15.1 (<a id="xref-ref-64-2" class="xref-bibr" href="#ref-64">R Core Team 2012</a>).</p></div><div id="sec-6" class="subsection"><h3>Data availability</h3><p id="p-18"><a href="http://www.g3journal.org/content/suppl/2015/08/21/g3.114.016493.DC1/FileS1.fasta">File S1</a> contains the 38 D. <em>melanogaster</em> assembled mtDNA genomes used in this study aligned with <em>D. yakuba</em> (NC_001322). Human mtDNA sequence data are available in GenBank, and the accession numbers are listed in <a href="http://www.g3journal.org/content/suppl/2015/08/21/g3.114.016493.DC1/TableS6.pdf">Table S6</a>.</p></div></div><div class="section results" id="sec-7"><h2 class="">Results</h2><div id="sec-8" class="subsection"><h3>An excess of low- and high-frequency, derived mitochondrial polymorphisms</h3><p id="p-19">We assembled 14,916 bp of sequence containing the transcribed regions of the mtDNA with a median coverage of 32x for 38 genetic lines sampled from a single population of <em>D. melanogaster</em> in Raleigh, North Carolina (<a id="xref-ref-38-4" class="xref-bibr" href="#ref-38">Langley <em>et al.</em> 2012</a>; <a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Mackay <em>et al.</em> 2012</a>) (<a href="http://www.g3journal.org/content/suppl/2015/08/21/g3.114.016493.DC1/TableS1.pdf">Table S1</a>). More than 98% of these nucleotides encode the 13 protein-coding genes, 22 tRNAs, and two ribosomal RNAs. The per-site expected heterozygosity in this region (<em>π</em>) of the mtDNA was 0.0008. We identified 137 segregating sites in this population sample, 103 of which were in protein-coding genes. Median heterozygosity in protein-coding genes was 0.0023 per synonymous site and 0.0002 per nonsynonymous site. Silent site heterozygosity was significantly lower in mitochondrial genes relative to nuclear genes (Mann−Whitney <em>U</em>, mtDNA <em>vs.</em> X chromosome, <em>P<sub>MWU</sub></em> = 0.00002; mtDNA <em>vs.</em> autosomes, <em>P<sub>MWU</sub></em> &lt; 0.00001) and was only 0.16 times that of the autosomes (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>), lower than what is expected if the mtDNA has an effective population size that is one-quarter that of the autosomes.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/10/2165/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Effect of genomic location on silent-site heterozygosity and on summary statistics of polymorphism and divergence in D. melanogaster. (A) Genomic location has a significant effect on per-site silent-site heterozygosity (PMWU &lt; 0.001 for all pairwise contrasts), consistent with predicted differences in the effective population size (Ne). The ratio of median mitochondrial to autosomal silent site heterozygosity was 0.157, less than predicted for neutral sites if mitochondrial Ne is one quarter that of the autosomes. mitochondrial DNA (mtDNA), X-chromosome, and autosome data sets contained 12, 1255, and 8073 genes, respectively. (B, C) Distributions of neutrality index (NI) and Z* are similar between mitochondrial and autosomal (abbreviated as A) genes, with moderately significant differences between mitochondrial and X-linked (abbreviated as X) genes. The four mtDNA boxes represent estimates from the corresponding MK tables in Table S2, B&#x2212;E that used either D. simulans (Table S2, B&#x2212;C) or D. yakuba (Table S2, D&#x2212;E) as the outgroup. Dashed lines represent the neutral expectations for these statistics. Three nuclear loci for which NI exceeded 50 were excluded from (B) to improve visualization. Statistical results are presented in Table S2. mtDNA, X-chromosome, and autosome data sets contained 13, 712, and 5401 genes, respectively." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1804711170" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Effect of genomic location on silent-site heterozygosity and on summary statistics of polymorphism and divergence in &lt;em&gt;D. melanogaster&lt;/em&gt;. (A) Genomic location has a significant effect on per-site silent-site heterozygosity (&lt;em&gt;P&lt;sub&gt;MWU&lt;/sub&gt;&lt;/em&gt; &lt; 0.001 for all pairwise contrasts), consistent with predicted differences in the effective population size (&lt;em&gt;N&lt;sub&gt;e&lt;/sub&gt;&lt;/em&gt;). The ratio of median mitochondrial to autosomal silent site heterozygosity was 0.157, less than predicted for neutral sites if mitochondrial &lt;em&gt;N&lt;sub&gt;e&lt;/sub&gt;&lt;/em&gt; is one quarter that of the autosomes. mitochondrial DNA (mtDNA), X-chromosome, and autosome data sets contained 12, 1255, and 8073 genes, respectively. (B, C) Distributions of neutrality index (&lt;em&gt;NI&lt;/em&gt;) and &lt;em&gt;Z*&lt;/em&gt; are similar between mitochondrial and autosomal (abbreviated as A) genes, with moderately significant differences between mitochondrial and X-linked (abbreviated as X) genes. The four mtDNA boxes represent estimates from the corresponding MK tables in Table S2, B&#x2212;E that used either &lt;em&gt;D. simulans&lt;/em&gt; (Table S2, B&#x2212;C) or &lt;em&gt;D. yakuba&lt;/em&gt; (Table S2, D&#x2212;E) as the outgroup. Dashed lines represent the neutral expectations for these statistics. Three nuclear loci for which &lt;em&gt;NI&lt;/em&gt; exceeded 50 were excluded from (B) to improve visualization. Statistical results are presented in Table S2. mtDNA, X-chromosome, and autosome data sets contained 13, 712, and 5401 genes, respectively.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/10/2165/F1.medium.gif" width="440" height="269"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/5/10/2165/F1.medium.gif" width="440" height="269"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/10/2165/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/10/2165/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466818" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-20" class="first-child">Effect of genomic location on silent-site heterozygosity and on summary statistics of polymorphism and divergence in <em>D. melanogaster</em>. (A) Genomic location has a significant effect on per-site silent-site heterozygosity (<em>P<sub>MWU</sub></em> &lt; 0.001 for all pairwise contrasts), consistent with predicted differences in the effective population size (<em>N<sub>e</sub></em>). The ratio of median mitochondrial to autosomal silent site heterozygosity was 0.157, less than predicted for neutral sites if mitochondrial <em>N<sub>e</sub></em> is one quarter that of the autosomes. mitochondrial DNA (mtDNA), X-chromosome, and autosome data sets contained 12, 1255, and 8073 genes, respectively. (B, C) Distributions of neutrality index (<em>NI</em>) and <em>Z*</em> are similar between mitochondrial and autosomal (abbreviated as A) genes, with moderately significant differences between mitochondrial and X-linked (abbreviated as X) genes. The four mtDNA boxes represent estimates from the corresponding MK tables in <a href="http://www.g3journal.org/content/suppl/2015/08/21/g3.114.016493.DC1/TableS2.pdf">Table S2</a>, B−E that used either <em>D. simulans</em> (<a href="http://www.g3journal.org/content/suppl/2015/08/21/g3.114.016493.DC1/TableS2.pdf">Table S2, B−C</a>) or <em>D. yakuba</em> (<a href="http://www.g3journal.org/content/suppl/2015/08/21/g3.114.016493.DC1/TableS2.pdf">Table S2, D−E</a>) as the outgroup. Dashed lines represent the neutral expectations for these statistics. Three nuclear loci for which <em>NI</em> exceeded 50 were excluded from (B) to improve visualization. Statistical results are presented in <a href="http://www.g3journal.org/content/suppl/2015/08/21/g3.114.016493.DC1/TableS2.pdf">Table S2</a>. mtDNA, X-chromosome, and autosome data sets contained 13, 712, and 5401 genes, respectively.</p><div class="sb-div caption-clear"></div></div></div><p id="p-21">In addition to the scarcity of segregating sites in the <em>D. melanogaster</em> mtDNA, polymorphisms at these sites were skewed toward low frequencies (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2, A and B</a>), as evidenced by consistently negative values of Tajima’s <em>D</em> (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). Tajima’s <em>D</em> across the mtDNA was −2.607 and differed significantly from the neutral expectation of 0 (<em>S</em> = 80 for coding sites with no missing data, <em>P &lt;</em> 0.0001), as did Fu and Li’s <em>D</em> (<em>D</em> = −2.67, <em>P &lt;</em> 0.05). The minor allele frequency for unpolarized synonymous polymorphisms was always less than 11%, and all but one of the nonsynonymous polymorphisms were singletons (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2, A and B</a>). Using <em>D. yakuba</em> as an outgroup revealed that the derived allele was nearly fixed for 44% of segregating synonymous sites, whereas there was only a single derived nonsynonymous polymorphism at high frequency (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2, C and D</a>). Using both <em>D. simulans</em> and <em>D. yakuba</em> to polarize mutations did not qualitatively change this result (<a href="http://www.g3journal.org/content/suppl/2015/08/21/g3.114.016493.DC1/FigureS1.pdf">Figure S1</a>). Thus, the mitochondrial genome was essentially devoid of intermediate-frequency polymorphisms, with derived synonymous mutations at either very high (greater than 89%) or very low (less than 11%) frequencies and nearly all derived nonsynonymous polymorphisms at frequencies less than 5%. This skew toward high-frequency, derived alleles resulted in a significant negative value of Fay and Wu’s <em>H</em> statistic (<em>H</em> = −41.2, <em>P</em> = 0.005).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/10/2165/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Site-frequency spectra of synonymous and nonsynonymous polymorphisms in the D. melanogaster mitochondrial DNA. (A, B) Folded site-frequency spectra for synonymous and nonsynonymous segregating sites across the mitochondrial protein-coding region reveal that mitochondrial polymorphisms are skewed to low frequencies. (C, D) Unfolded site-frequency spectra reveal that derived, synonymous polymorphisms are almost equally likely to be at low frequency (56% of 59 sites at frequencies less than 0.11) or nearly fixed (44% of 59 sites at frequencies greater than 0.89), while derived, nonsynonymous polymorphisms are nearly always present as singletons (94% of 32 sites). There are essentially no mitochondrial polymorphisms at intermediate frequencies. Sites were omitted from the unfolded site frequency spectra if neither allelic state was shared with D. yakuba. The number of sites included in each distribution is 67 (A), 35 (B), 59 (C), and 32 (D)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1804711170" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Site-frequency spectra of synonymous and nonsynonymous polymorphisms in the &lt;em&gt;D. melanogaster&lt;/em&gt; mitochondrial DNA. (A, B) Folded site-frequency spectra for synonymous and nonsynonymous segregating sites across the mitochondrial protein-coding region reveal that mitochondrial polymorphisms are skewed to low frequencies. (C, D) Unfolded site-frequency spectra reveal that derived, synonymous polymorphisms are almost equally likely to be at low frequency (56% of 59 sites at frequencies less than 0.11) or nearly fixed (44% of 59 sites at frequencies greater than 0.89), while derived, nonsynonymous polymorphisms are nearly always present as singletons (94% of 32 sites). There are essentially no mitochondrial polymorphisms at intermediate frequencies. Sites were omitted from the unfolded site frequency spectra if neither allelic state was shared with &lt;em&gt;D. yakuba&lt;/em&gt;. The number of sites included in each distribution is 67 (A), 35 (B), 59 (C), and 32 (D).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/10/2165/F2.medium.gif" width="437" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/5/10/2165/F2.medium.gif" width="437" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/10/2165/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/10/2165/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466882" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-22" class="first-child">Site-frequency spectra of synonymous and nonsynonymous polymorphisms in the <em>D. melanogaster</em> mitochondrial DNA. (A, B) Folded site-frequency spectra for synonymous and nonsynonymous segregating sites across the mitochondrial protein-coding region reveal that mitochondrial polymorphisms are skewed to low frequencies. (C, D) Unfolded site-frequency spectra reveal that derived, synonymous polymorphisms are almost equally likely to be at low frequency (56% of 59 sites at frequencies less than 0.11) or nearly fixed (44% of 59 sites at frequencies greater than 0.89), while derived, nonsynonymous polymorphisms are nearly always present as singletons (94% of 32 sites). There are essentially no mitochondrial polymorphisms at intermediate frequencies. Sites were omitted from the unfolded site frequency spectra if neither allelic state was shared with <em>D. yakuba</em>. The number of sites included in each distribution is 67 (A), 35 (B), 59 (C), and 32 (D).</p><div class="sb-div caption-clear"></div></div></div><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466751/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466751/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Synonymous and nonsynonymous variation in <em>D. melanogaster</em> mitochondrial genes and OXPHOS complexes</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-9" class="subsection"><h3>A partial sweep in the <em>D. melanogaster</em> mtDNA</h3><p id="p-26">The large fraction of derived alleles at high frequencies is a consequence of the haplotype structure of this sample (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). Nearly 30% of individuals in this population shared an identical mitochondrial haplotype, and an additional 66% of individuals differed from this haplotype by only one to five mutations. The two remaining haplotypes (RAL-639 and RAL-335) were highly divergent from this common haplotype group, contributing nearly half of the segregating sites to the population sample. These two haplotypes shared the ancestral state with <em>D. yakuba</em> at 17 of the 23 derived high-frequency synonymous polymorphisms (<em>i.e.</em>, they have the low-frequency ancestral allele). When these two haplotypes were removed from the analysis, there remained a strong skew toward rare alleles (Tajima’s <em>D</em> = −2.31, <em>P</em> &lt; 0.01; Fu and Li’s <em>D</em> = −3.14, <em>P</em> &lt; 0.02), but Fay and Wu’s <em>H</em>, which is sensitive to the number of high-frequency derived alleles, was only weakly significant (<em>H</em> = −10.34, <em>P</em> = 0.043). The remaining six derived, high-frequency synonymous polymorphisms, as well as the single derived, high-frequency nonsynonymous polymorphism, were the result of single mtDNAs within the common haplotype group having the same allelic state as <em>D. yakuba</em>. Given the lack of recombination in the mtDNA, these are likely new, rather than ancestral, mutations. Six of these seven mutations would have changed a C or G to a T or A, consistent with the mutation bias in the <em>D. melanogaster</em> mtDNA (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Haag-Liautard <em>et al.</em> 2008</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/10/2165/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Haplotype network for 38 D. melanogaster mitochondrial DNAs (mtDNAs) sampled from Raleigh, North Carolina. The network, inferred from 80 coding region single-nucleotide polymorphisms (SNPs) with no missing information, reveals that nearly 30% of individuals sampled (11/38) share the same common haplotype (red) and an additional 65% of individuals carry a haplotype only a few mutations away from this haplotype. This common set of mitochondrial haplotypes is highly diverged from the two other mtDNAs sampled in the population; lines RAL-639 and RAL-335 differ from the common haplotype at 14 and 34 SNPs, respectively. At least one of these two haplotypes carries the ancestral state (shared with D. yakuba) at 38% of these SNPs. Numbers represent the Raleigh line carrying the haplotype. Red, yellow, blue, and white nodes were present in 11, 3, 2, and 1 lines, respectively." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1804711170" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Haplotype network for 38 &lt;em&gt;D. melanogaster&lt;/em&gt; mitochondrial DNAs (mtDNAs) sampled from Raleigh, North Carolina. The network, inferred from 80 coding region single-nucleotide polymorphisms (SNPs) with no missing information, reveals that nearly 30% of individuals sampled (11/38) share the same common haplotype (red) and an additional 65% of individuals carry a haplotype only a few mutations away from this haplotype. This common set of mitochondrial haplotypes is highly diverged from the two other mtDNAs sampled in the population; lines RAL-639 and RAL-335 differ from the common haplotype at 14 and 34 SNPs, respectively. At least one of these two haplotypes carries the ancestral state (shared with &lt;em&gt;D. yakuba&lt;/em&gt;) at 38% of these SNPs. Numbers represent the Raleigh line carrying the haplotype. Red, yellow, blue, and white nodes were present in 11, 3, 2, and 1 lines, respectively.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/10/2165/F3.medium.gif" width="440" height="265"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/5/10/2165/F3.medium.gif" width="440" height="265"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/10/2165/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/10/2165/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466837" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-27" class="first-child">Haplotype network for 38 <em>D. melanogaster</em> mitochondrial DNAs (mtDNAs) sampled from Raleigh, North Carolina. The network, inferred from 80 coding region single-nucleotide polymorphisms (SNPs) with no missing information, reveals that nearly 30% of individuals sampled (11/38) share the same common haplotype (red) and an additional 65% of individuals carry a haplotype only a few mutations away from this haplotype. This common set of mitochondrial haplotypes is highly diverged from the two other mtDNAs sampled in the population; lines RAL-639 and RAL-335 differ from the common haplotype at 14 and 34 SNPs, respectively. At least one of these two haplotypes carries the ancestral state (shared with <em>D. yakuba</em>) at 38% of these SNPs. Numbers represent the Raleigh line carrying the haplotype. Red, yellow, blue, and white nodes were present in 11, 3, 2, and 1 lines, respectively.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h3>No evidence for recombination in the <em>D. melanogaster</em> mtDNA</h3><p id="p-28">We tested for a negative correlation between LD and the distance between each pair of polymorphic sites in the <em>D. melanogaster</em> mitochondrial genome, as a signature of the decay of LD over distance via recombination (<a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">Awadalla <em>et al.</em> 1999</a>). There was no evidence for a decrease in LD with increasing distance between sites, regardless of the measure of LD or the minor allele cutoff used (<a href="http://www.g3journal.org/content/suppl/2015/08/21/g3.114.016493.DC1/TableS7.pdf">Table S7</a>). There were no pairs of polymorphic sites for which all four gametes were present (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Hudson and Kaplan 1985</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Bruen <em>et al.</em> 2006</a>), further supporting an absence of effective recombination.</p></div><div id="sec-11" class="subsection"><h3>Weakly deleterious polymorphism in the <em>D. melanogaster</em> mtDNA</h3><p id="p-29">The skew in the site-frequency spectrum toward rare alleles (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a>) resulted in negative values of Tajima’s <em>D</em> across the entire mtDNA (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). However, there was no evidence that the skew toward rare alleles differed between synonymous and nonsynonymous polymorphisms (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2, A and B</a>)—heterogeneity tests (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Hahn <em>et al.</em> 2002</a>) of Tajima’s <em>D</em> between synonymous and nonsynonymous sites were never significant (<em>P</em> &gt; 0.35 for all genes and complexes). However, unfolding the site frequency spectra revealed that the large number of high-frequency, derived sites were nearly all synonymous (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2, C and D</a>), suggesting that the haplotype that has increased in frequency carried many more synonymous than nonsynonymous polymorphisms. Given that the mutation bias in the <em>D. melanogaster</em> mtDNA greatly favors nonsynonymous mutations (<a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">Haag-Liautard <em>et al.</em> 2008</a>), this pattern suggests a history of effective purifying selection removing mitochondrial haplotypes that contain nonsynonymous polymorphisms. Furthermore, all nonsynonymous polymorphisms that have arisen on the common mitochondrial haplotype are present at very low frequencies.</p><p id="p-30">The current distribution of polymorphisms relative to divergence in <em>D. melanogaster</em> showed little evidence for a large and significant excess of segregating deleterious polymorphisms. Across MK tables, no single gene departed significantly from the neutral expectation after Bonferroni correction (<em>P</em> &lt; 0.05/13) (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a> and <a href="http://www.g3journal.org/content/suppl/2015/08/21/g3.114.016493.DC1/TableS2.pdf">Table S2</a>). For the entire set of protein-coding mitochondrial genes, there was a slight excess of nonsynonymous polymorphism relative to the neutral expectation, as indicated by moderately significant MK tests [Fisher’s exact test, <em>P<sub>FET</sub></em> ranged from 0.0004 to 0.041 across methods (<a href="http://www.g3journal.org/content/suppl/2015/08/21/g3.114.016493.DC1/TableS2.pdf">Table S2</a>)] and values of <em>NI<sub>TG</sub></em> that ranged from 1.67 to 2.57 across methods, with confidence intervals that did not include the neutral expectation of 1 (<a href="http://www.g3journal.org/content/suppl/2015/08/21/g3.114.016493.DC1/TableS2.pdf">Table S2</a>). There was some OXPHOS-complex specificity to this result—Complexes I (<em>ND</em>) and V (<em>ATPase</em>) tended to deviate significantly from neutrality with <em>NI</em> values greater than 1, whereas Complex IV (<em>CO</em>) was consistent with the neutral expectation (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a> and <a href="http://www.g3journal.org/content/suppl/2015/08/21/g3.114.016493.DC1/TableS3.pdf">Table S3</a>).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466763/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466763/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Counts of polymorphic (<em>P</em>) and divergent (<em>D</em>) nonsynonymous (<em><sub>N</sub></em>) and synonymous (<em><sub>S</sub></em>) sites along with summary statistics of the MK table for <em>D. melanogaster</em> mitochondrial genes</span><div class="sb-div caption-clear"></div></div></div><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466758/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466758/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Summary statistics of the MK table for mitochondrially encoded OXPHOS complexes and nuclear genes</span><div class="sb-div caption-clear"></div></div></div><p id="p-42">Analysis of 36 of the 38 mitochondrial haplotypes in our sample that were independently sequenced and assembled by <a id="xref-ref-68-2" class="xref-bibr" href="#ref-68">Richardson <em>et al.</em> (2012)</a> confirmed these patterns (<a href="http://www.g3journal.org/content/suppl/2015/08/21/g3.114.016493.DC1/TableS4.pdf">Table S4</a>). When counts of polymorphism and divergence differed between datasets, they typically differed by a single count. The exception was in several Complex I (<em>ND</em>) genes, for which our assembled mtDNAs had a small number of additional nonsynonymous polymorphisms relative to the <a id="xref-ref-68-3" class="xref-bibr" href="#ref-68">Richardson <em>et al.</em> (2012)</a> data set (<em>ND</em> genes, <em>df</em> = 6, <em>P<sub>MWU</sub></em><sub>,</sub> <em><sub>paired</sub></em> = 0.021; all other genes, <em>df</em> = 5, <em>P<sub>MWU</sub></em><sub>,</sub> <em><sub>paired</sub></em> = 1) that resulted in slightly greater values of <em>NI</em> (<em>ND</em> included, <em>df</em> = 12, <em>P<sub>MWU</sub></em><sub>,</sub> <em><sub>paired</sub></em> = 0.016; all other genes, <em>df</em> = 5, <em>P<sub>MWU</sub></em><sub>,</sub> <em><sub>paired</sub></em> = 1). This was not due to absence of two mitochondrial haplotypes in the <a id="xref-ref-68-4" class="xref-bibr" href="#ref-68">Richardson <em>et al.</em> (2012)</a> data set, and the additional polymorphisms in our data were not clustered on any single mitochondrial haplotype. The reduced number of nonsynonymous polymorphisms in the <a id="xref-ref-68-5" class="xref-bibr" href="#ref-68">Richardson <em>et al.</em> (2012)</a> data provided even less support for an excess of nonsynonymous segregating variation in the mitochondrial genome. Summed counts of polymorphism and divergence for the entire set of mitochondrial-encoded proteins in this dataset did not deviate from the neutral expectation (<em>P<sub>FET</sub></em> = 0.423), and the confidence intervals on <em>NI<sub>TG</sub></em> for mitochondrial-encoded proteins contained the neutral expectation of 1 (<em>NI<sub>TG</sub></em> = 0.821, 95% confidence interval = 0.386 to 1.90).</p></div><div id="sec-12" class="subsection"><h3>On average, <em>NI</em> is similar for mitochondrial and nuclear genes in flies</h3><p id="p-43">Although there is a weak signature of an excess of nonsynonymous segregating variation in the <em>D. melanogaster</em> mitochondrial genome, both mitochondrial and nuclear gene sets have median <em>NI</em>, <em>Z*</em>, or <em>NI<sub>TG</sub></em> values that deviate in the same manner from the neutral expectation, indicative of both genomes harboring weakly deleterious polymorphisms. Furthermore, the distribution of <em>D. melanogaster</em> mitochondrial gene <em>NI</em> values was contained within that of the nuclear genes, with many nuclear genes having both more positive and more negative values of <em>NI</em> and <em>Z*</em> (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1, B and C</a>). Weakly significant differences between mitochondrial and nuclear gene <em>NI</em> were affected by the genomic location of nuclear genes (<a href="http://www.g3journal.org/content/suppl/2015/08/21/g3.114.016493.DC1/TableS2.pdf">Table S2</a>), as X-linked genes had significantly lower values of <em>NI</em> and more positive <em>Z*</em> values relative to autosomal genes (<em>P<sub>MWU</sub></em> &lt; 1e-6 for both statistics) (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1, B and C</a>). Because of this, mitochondrial gene <em>NI</em> differed significantly from X-linked genes (<em>P<sub>MWU</sub></em> ranged from 0.007 to 0.065) but not from autosomal genes (<em>P<sub>MWU</sub></em> ranged from 0.047 to 0.325), with similar patterns for <em>Z*</em> (mtDNA <em>vs.</em> X, <em>P<sub>MWU</sub></em> ranged from 0.002 to 0.019; mtDNA <em>vs.</em> autosomes, <em>P<sub>MWU</sub></em> ranged from 0.012 to 0.104). The lower counts of polymorphic sites in the assembled mtDNAs from <a id="xref-ref-68-6" class="xref-bibr" href="#ref-68">Richardson <em>et al.</em> (2012)</a> provided less support for genomic differences in MK statistics. Neither <em>NI</em> nor <em>Z*</em> differed significantly between the mitochondria and either the X or the autosomes (<em>NI</em>, <em>P<sub>MWU</sub></em> &gt; 0.441 for both comparisons; <em>Z*</em>, <em>P<sub>MWU</sub></em> &gt; 0.471 for both comparisons).</p><p id="p-44">The moderate levels of significance associated with some of these contrasts, and the sensitivity of these contrasts to small differences in MK counts and methods, suggest that although there is a trend for mitochondrial genes to have larger <em>NI</em> (and more negative <em>Z*</em>) values relative to nuclear genes, the differences between genomes are not large. Contrasts with mitochondrial genomes may have low power due to the smaller number of genes and low levels of nonsynonymous polymorphism and divergence, relative to nuclear genomes. However, the mitochondrial data are not a sample of genes, as they represent the complete protein-coding complement of this genome. Nevertheless, a traditional power analysis suggests that we would require an 18-fold increase in the number of mitochondrial genes for the smallest effect size (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>) to reach statistical significance. To provide biological context for the small differences in MK summary statistics that we observed between genomes, we calculated and contrasted effect sizes as the difference in means between genomes divided by the root mean square of the SD for <em>NI</em> and <em>Z*</em>. Across MK tables, the differences in <em>NI</em> between mitochondrial and autosomal genes yielded effect sizes that range from 0.18 to 0.71, smaller than those reported in the meta-analyses of <a id="xref-ref-78-5" class="xref-bibr" href="#ref-78">Weinreich and Rand (2000)</a>, where the difference in mean <em>NI</em> between mitochondrial and nuclear genes was 3.2, with an effect size of 0.96. In an analysis of 98 nuclear loci in <em>D. melanogaster</em>, <a id="xref-ref-63-4" class="xref-bibr" href="#ref-63">Presgraves (2005)</a> reported significant differences in <em>Z*</em> for genes located in regions of high and low recombination for which the effect size was 0.96, over twice that which we observed between mitochondrial and autosomal gene <em>Z*</em> (<a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>).</p></div><div id="sec-13" class="subsection"><h3><em>NI</em> does not differ between mitochondrial and nuclear genes in humans</h3><p id="p-45">Summary statistics of the MK table also did not differ between mitochondrial and nuclear genes in <em>H. sapiens</em> (<em>NI</em>, <em>P<sub>MWU</sub></em> = 0.657; <em>Z*</em>, <em>P<sub>MWU</sub></em> = 0.243), nor did the site-frequency spectrum differ between nonsynonymous and synonymous mitochondrial polymorphisms in humans (heterogeneity test, <em>P</em> &gt; 0.36 for all genes). Values of <em>NI</em> and <em>Z*</em> for mitochondrial genes in humans were well within the distribution of these statistics for nuclear genes (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>), and the confidence intervals around <em>NI<sub>TG</sub></em> for the mitochondrial and nuclear genomes were overlapping (<a id="xref-table-wrap-3-4" class="xref-table" href="#T3">Table 3</a>). Similar to the fly mtDNA and nuclear genome, the median values of <em>NI</em> and <em>Z*</em> for both the human mtDNA and nuclear genome were consistent with a slight excess of nonsynonymous polymorphism (<a id="xref-table-wrap-3-5" class="xref-table" href="#T3">Table 3</a>). The distributions of <em>NI</em> and <em>Z*</em> were also largely overlapping and did not differ significantly between <em>D. melanogaster</em> and <em>H. sapiens</em> mitochondrial genes (<em>NI</em>, <em>P<sub>MWU</sub></em> = 0.545; <em>Z*</em>, <em>P<sub>MWU</sub></em> = 0.441) (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>), despite differing nuclear <em>N<sub>e</sub></em> between these species. This further supports the idea that the efficacy of purifying selection in these mitochondrial genomes is largely independent of <em>N<sub>e</sub></em>.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/10/2165/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distributions of (A) neutrality index (NI) and (B) Z* for mitochondrial and nuclear genes in D. melanogaster and H. sapiens. Three nuclear genes in flies and two nuclear genes in humans that had NI values greater than 50 were removed to improve visualization. Dashed lines represent the neutral expectation for each statistic. The D. melanogaster mitochondrial DNA (mtDNA) and nuclear sets contained 13 and 6113 genes, respectively. The H. sapiens mtDNA and nuclear sets contained 13 and 11,624 genes, respectively. See Table S2 and Table 3 and main text for statistical results." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1804711170" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Distributions of (A) neutrality index (&lt;em&gt;NI&lt;/em&gt;) and (B) &lt;em&gt;Z*&lt;/em&gt; for mitochondrial and nuclear genes in &lt;em&gt;D. melanogaster&lt;/em&gt; and &lt;em&gt;H. sapiens&lt;/em&gt;. Three nuclear genes in flies and two nuclear genes in humans that had &lt;em&gt;NI&lt;/em&gt; values greater than 50 were removed to improve visualization. Dashed lines represent the neutral expectation for each statistic. The &lt;em&gt;D. melanogaster&lt;/em&gt; mitochondrial DNA (mtDNA) and nuclear sets contained 13 and 6113 genes, respectively. The &lt;em&gt;H. sapiens&lt;/em&gt; mtDNA and nuclear sets contained 13 and 11,624 genes, respectively. See Table S2 and Table 3 and main text for statistical results.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/10/2165/F4.medium.gif" width="440" height="292"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/5/10/2165/F4.medium.gif" width="440" height="292"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/10/2165/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/10/2165/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466803" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-46" class="first-child">Distributions of (A) neutrality index (<em>NI</em>) and (B) <em>Z*</em> for mitochondrial and nuclear genes in <em>D. melanogaster</em> and <em>H. sapiens</em>. Three nuclear genes in flies and two nuclear genes in humans that had <em>NI</em> values greater than 50 were removed to improve visualization. Dashed lines represent the neutral expectation for each statistic. The <em>D. melanogaster</em> mitochondrial DNA (mtDNA) and nuclear sets contained 13 and 6113 genes, respectively. The <em>H. sapiens</em> mtDNA and nuclear sets contained 13 and 11,624 genes, respectively. See <a href="http://www.g3journal.org/content/suppl/2015/08/21/g3.114.016493.DC1/TableS2.pdf">Table S2</a> and <a id="xref-table-wrap-3-6" class="xref-table" href="#T3">Table 3</a> and main text for statistical results.</p><div class="sb-div caption-clear"></div></div></div><p id="p-47">Using data from flies and humans, we tested whether contrasts between nuclear and mitochondrial genes with similar function in OXPHOS and putatively similar selective effects of mutations (<em>s</em>) would reveal greater differences in <em>NI</em> between mitochondrial and nuclear genomes as a function of differing <em>N<sub>e</sub></em>. For humans, there was no difference in <em>NI</em> or <em>Z*</em> between OXPHOS genes encoded in the mitochondrial and nuclear genomes (<em>P<sub>MWU</sub></em> &gt; 0.46 for both statistics), whereas for flies there was a weakly significant difference that was driven by the fact that the nuclear OXPHOS genes in our sample had values of <em>NI</em> and <em>Z*</em> that were more consistent with an excess of nonsynonymous substitutions (<em>NI</em>, <em>P<sub>MWU</sub></em> = 0.026; <em>Z*</em>, <em>P<sub>MWU</sub></em> = 0.022) (<a href="http://www.g3journal.org/content/suppl/2015/08/21/g3.114.016493.DC1/FigureS2.pdf">Figure S2</a>). However, these data should be treated with some caution, as there were only 11 genes in our nuclear data set annotated to have OXPHOS function, and nine of these genes are part of Complex I (<em>ND</em>). Mitochondrial <em>ND</em> genes accumulate more amino acid substitutions than do other OXPHOS-complex genes in <em>Drosophila</em> (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Ballard 2000</a>; <a id="xref-ref-56-4" class="xref-bibr" href="#ref-56">Montooth <em>et al.</em> 2009</a>), potentially reflecting differences in functional constraint among complexes that are consistent with the OXPHOS-complex differences in <em>NI<sub>TG</sub></em> that we observed in this study (<a id="xref-table-wrap-3-7" class="xref-table" href="#T3">Table 3</a> and <a href="http://www.g3journal.org/content/suppl/2015/08/21/g3.114.016493.DC1/TableS3.pdf">Table S3</a>).</p><p id="p-48">Finally, we used the human data to illustrate the sensitivity of <em>NI</em> to sampling. When only a few individuals are sampled, the choice of genomes can lead to high variability and extreme values in <em>NI</em>, potentially as a result of single haplotypes that may carry multiple polymorphisms, as appears to be the case for human <em>ND6</em> (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a> and <a href="http://www.g3journal.org/content/suppl/2015/08/21/g3.114.016493.DC1/TableS5.pdf">Table S5</a>). For example, depending on which Japanese individuals we included in our analyses, <em>NI</em> for <em>ND6</em> takes on values of 30.71 (MK test, <em>P<sub>FET</sub></em> = 0.001), 5.50 (<em>P<sub>FET</sub></em> = 0.308), or 1.79 (<em>P<sub>FET</sub></em> = 0.522) when sampling only three mtDNAs. As more mtDNAs are sampled, <em>NI</em> and <em>Z*</em> for each mitochondrial gene become more similar to the neutral expectation (<a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a> and <a href="http://www.g3journal.org/content/suppl/2015/08/21/g3.114.016493.DC1/TableS5.pdf">Table S5</a>). Overall, these analyses using <em>D. melanogaster</em> and <em>H. sapiens</em> mitochondrial genomes highlight the sensitivity of these MK statistics to the number of genomes sampled, the amount of divergence between species, and the low levels of polymorphism in these genes.</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466836/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466836/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">The sensitivity of <em>NI</em> to sampling</span><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-14"><h2 class="">Discussion</h2><p id="p-55">Using a large sample of whole-genome sequence data, we have tested a number of hypotheses about mtDNA evolution, and about differences in the efficacy of selection on mitochondrial <em>vs.</em> nuclear genes. Our data confirm that mtDNA do not have a signature of recombination and have lower silent-site diversity than do nuclear genes in <em>D. melanogaster</em>, which supports the prediction that the mitochondrial genome has a lower <em>N<sub>e</sub></em> than the nuclear genome. We also show a skew in the site-frequency spectrum toward rare alleles in <em>D. melanogaster</em> that likely has two sources: 1) the accumulation of new mutations on what appears to be a mtDNA haplotype that has swept to high frequency in the recent past, and 2) the ancestral polymorphisms contained on migrant or remnant haplotypes that are now rare in this population. Despite the apparent reduction in <em>N<sub>e</sub></em> for mtDNA, our findings indicate that selection is similarly effective at purging deleterious polymorphisms from the mitochondrial and nuclear genomes of <em>D. melanogaster</em>, and that the same is true in <em>H. sapiens</em>. Although all genomes that we analyzed showed some evidence of an excess of nonsynonymous polymorphism relative to the neutral expectation, the only significant differences in <em>NI</em> and <em>Z*</em> were between <em>D. melanogaster</em> mitochondrial genes and X-linked genes. X-linked genes in <em>Drosophila</em> have a greater proportion of beneficial substitutions than do autosomes (<a id="xref-ref-38-5" class="xref-bibr" href="#ref-38">Langley <em>et al.</em> 2012</a>; <a id="xref-ref-48-3" class="xref-bibr" href="#ref-48">Mackay <em>et al.</em> 2012</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Meisel and Connallon 2013</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Garrigan <em>et al.</em> 2014</a>), suggesting that what differs between mitochondrial genes and nuclear genes is likely the fraction of beneficial substitutions rather than the efficacy of purifying selection, which appears to be largely independent of <em>N<sub>e</sub></em> in the <em>D. melanogaster</em> and <em>H. sapiens</em> mitochondrial genomes that we have analyzed.</p><p id="p-56">Given its uniparental and haploid transmission, the expectation under neutrality is that the mtDNA has one-quarter the population size of the autosomes. This reduced value of <em>N</em> (and subsequently <em>N<sub>e</sub></em>) matches that expected for the Y (or W) chromosome, and, like the Y chromosome, the mtDNA has little to no recombination. However, very much unlike the Y chromosomes that have been sequenced (<em>e.g.</em>, <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Charlesworth and Charlesworth 2000</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Carvalho <em>et al.</em> 2009</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Carvalho and Clark 2013</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Bellott <em>et al.</em> 2014</a>), animal mtDNA genomes do not show an accumulation of transposable elements, and the gene content of the animal mitochondrial genome is remarkably stable, with few gene losses and even fewer pseudogenes (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Boore 1999</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Ballard and Rand 2005</a>). Furthermore, <em>d<sub>N</sub></em>/<em>d<sub>S</sub></em> is two to 15 times lower for mitochondrial genes than for nuclear genes in mammals (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Popadin <em>et al.</em> 2013</a>), and average values of <em>d<sub>N</sub></em>/<em>d<sub>S</sub></em> for mitochondrial genes are well under 0.1 and are, on average, only 13% that of nuclear genes in <em>Drosophila</em> (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Bazin <em>et al.</em> 2006</a>; <a id="xref-ref-56-5" class="xref-bibr" href="#ref-56">Montooth <em>et al.</em> 2009</a>). This pattern of amino acid conservation is particularly striking, given that the mutation rate in the <em>D. melanogaster</em> mtDNA is an order of magnitude greater than the per-site mutation rate in the nuclear genome, with an extreme bias toward nonsynonymous mutations in the mitochondrial genome (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Haag-Liautard <em>et al.</em> 2007</a>; <a id="xref-ref-29-4" class="xref-bibr" href="#ref-29">Haag-Liautard <em>et al.</em> 2008</a>). Although heteromorphic Y chromosomes do show signatures of less effective purifying selection, such as proliferation of satellite repeats and reduced codon bias (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bachtrog 2013</a>; <a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Singh <em>et al.</em> 2014</a>), the single copy, X-degenerate genes that have remained on the human Y chromosome experience effective purifying selection (<a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Rozen <em>et al.</em> 2009</a>; <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Bellott <em>et al.</em> 2014</a>), as do the protein sequences of <em>Drosophila</em> Y-linked genes (<a id="xref-ref-74-2" class="xref-bibr" href="#ref-74">Singh <em>et al.</em> 2014</a>). Thus, despite early loss of many genes when heteromorphic Y chromosomes and mtDNA formed, both these nonrecombining chromosomes contain genes maintained by effective purifying selection in the presence of reduced <em>N<sub>e</sub></em>.</p><p id="p-57">Many researchers have cited the early work on <em>NI</em> in <em>Drosophila</em> and mammals in support of the idea that mtDNA accumulate deleterious mutations (<em>e.g.</em>, <a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Meiklejohn <em>et al.</em> 2007</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Green <em>et al.</em> 2008</a>; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Neiman and Taylor 2009</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Akashi <em>et al.</em> 2012</a>). In fact, this idea has become so engrained that it is regularly cited in reviews of mitochondrial gene evolution (<em>e.g.</em>, <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Ballard and Whitlock 2004</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Lynch 2007</a>). What is perhaps surprising about this conversion of a small set of intriguing initial studies into dogma is that the early studies themselves were quite circumspect about the implications of their results. For instance, <a id="xref-ref-57-5" class="xref-bibr" href="#ref-57">Nachman (1998)</a>, in noting that very few nuclear loci were available for comparison, stated “It is also unclear whether the patterns reported here are unique to mitochondrial DNA.” Data from the few nuclear genes that had been sequenced raised “the possibility that the patterns reported here for mtDNA may also be found at some nuclear loci” (<a id="xref-ref-57-6" class="xref-bibr" href="#ref-57">Nachman 1998</a>). Even studies that did have access to additional nuclear datasets were only able to calculate <em>NI</em> for 36 nuclear loci (<a id="xref-ref-78-6" class="xref-bibr" href="#ref-78">Weinreich and Rand 2000</a>), and the <em>NI</em> values that were available often did not deviate significantly from neutrality (<a id="xref-ref-57-7" class="xref-bibr" href="#ref-57">Nachman 1998</a>; <a id="xref-ref-78-7" class="xref-bibr" href="#ref-78">Weinreich and Rand 2000</a>). Those that did reject neutrality tended to do so weakly, perhaps due to the small number of polymorphisms in mitochondrial samples even when the number of individuals sampled is high (<em>e.g.</em>, <em>ND3</em>, <a id="xref-ref-58-2" class="xref-bibr" href="#ref-58">Nachman <em>et al.</em> 1996</a>). Nevertheless, there are mitochondrial genes that do strongly reject neutrality, and some of these had <em>NI</em> values that greatly exceeded <em>NI</em> for the sampled nuclear loci. On the basis of these and similar comparisons, many authors have reached the conclusion that mtDNA evolves in a manner distinct from the nuclear genome. Our results using the whole genomes of flies and humans, combined with observations of low mitochondrial <em>d<sub>N</sub></em>/<em>d<sub>S</sub></em>, suggest that the mitochondrial genomes of flies and humans are not suffering less effective purifying selection relative to the nuclear genome, and that differences in selection between these genomes may lie in differing rates of adaptive evolution.</p><p id="p-58">Reductions in <em>N<sub>e</sub></em>—due either to reductions in census population size or to the increased effect of linked, selected variants in regions of low recombination—are expected to result in a reduction in the efficacy of purifying selection. Indeed, comparisons of MK test results across a range of species with different values of <em>N<sub>e</sub></em> have revealed this expected relationship (<em>e.g.</em>, <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Li <em>et al.</em> 2008</a>; <a id="xref-ref-82-1" class="xref-bibr" href="#ref-82">Wright and Andolfatto 2008</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Gossmann <em>et al.</em> 2010</a>), as have comparisons of <em>NI</em> across regions of the <em>D. melanogaster</em> nuclear genome with different recombination rates (<a id="xref-ref-63-5" class="xref-bibr" href="#ref-63">Presgraves 2005</a>; <a id="xref-ref-38-6" class="xref-bibr" href="#ref-38">Langley <em>et al.</em> 2012</a>). Therefore, all things being equal, mitochondrial loci would be expected to harbor an excess of nonsynonymous polymorphisms relative to nuclear loci due to reduced <em>N<sub>e</sub></em>. Our results suggest that all things are not equal between these two cellular compartments, and that there may be features of the mitochondrion that make it less likely to accumulate deleterious mutations. One such feature is the “bottleneck” that occurs in the number of mtDNAs that are passed from mother to offspring—this event makes it possible for selection to act within hosts, possibly increasing the power of selection to remove deleterious mutations (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Bergstrom and Pritchard 1998</a>; <a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Rand 2011</a>) and reducing variability in mitochondrial <em>N<sub>e</sub></em> among taxa, relative to nuclear genomes. The additional layers of selection imposed by mitochondrial inheritance, combined with stronger negative selective effects of amino acid changing mutations in mitochondrial genes (<em>e.g.</em>, <a id="xref-ref-62-2" class="xref-bibr" href="#ref-62">Popadin <em>et al.</em> 2013</a>), may allow the mtDNA to escape the accumulation of deleterious mutation, resulting in relatively similar values of <em>NI</em> between nucleus and mitochondria. If the selective effects of mutations in mitochondrial genes are beyond the “horizon” where all mutations will behave similarly regardless of <em>N<sub>e</sub></em> (<a id="xref-ref-57-8" class="xref-bibr" href="#ref-57">Nachman 1998</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Eyre-Walker and Keightley 2007</a>), then we expect patterns of mitochondrial polymorphism and divergence to be largely independent of <em>N<sub>e</sub></em>.</p><p id="p-59">Our results come with several caveats. First, we have only studied two organisms—it may be that a more comprehensive review of <em>NI</em> in mtDNA and nuclear loci across many species will reveal a difference in the average efficacy of purifying selection or highlight lineage-specific patterns. The early meta-analyses of <em>NI</em> contained loci from a wide range of animals (<a id="xref-ref-57-9" class="xref-bibr" href="#ref-57">Nachman 1998</a>; <a id="xref-ref-78-8" class="xref-bibr" href="#ref-78">Weinreich and Rand 2000</a>), and using data from only <em>Drosophila</em> and humans may provide a limited perspective. Nevertheless, these are two model organisms for evolutionary biology that span a large range of mtDNA:nuclear substitution rates, and studies of these species have led the way for much of modern population genetics. Second, it is clear from our analysis of the <em>D. melanogaster</em> mtDNA that it is not at equilibrium, and may be recovering from a partial cytoplasmic sweep that may be associated with <em>Wolbachia</em> (<a id="xref-ref-68-7" class="xref-bibr" href="#ref-68">Richardson <em>et al.</em> 2012</a>). Much of the theory used to predict <em>NI</em> values from <em>N<sub>e</sub></em> and <em>s</em> assumes mutation-selection-drift balance (see, <em>e.g.</em>, <a id="xref-ref-57-10" class="xref-bibr" href="#ref-57">Nachman 1998</a>), and deviations from this equilibrium can result in more complex relationships between <em>N<sub>e</sub></em>, <em>s</em>, and <em>NI</em> (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Messer and Petrov 2013</a>). Although nonequilibrium histories may mean that mtDNA <em>NI</em> values are not at equilibrium, it is equally likely that nuclear genes from <em>D. melanogaster</em> are not at mutation-selection-drift equilibrium (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Hahn 2008</a>; <a id="xref-ref-38-7" class="xref-bibr" href="#ref-38">Langley <em>et al.</em> 2012</a>). Whether or not the mtDNA is at equilibrium, and whether or not the <em>NI</em> values calculated from this snapshot of two species represent equilibrium values, our results still imply that there is little difference between nuclear and mitochondrial measures of the efficacy of purifying selection.</p><p id="p-60">Despite the mitochondrial genome experiencing a distinct population genetic environment relative to the nuclear genome, our whole-genome analyses uncovered little evidence for an excess accumulation of slightly deleterious mutations in mitochondrial genomes, relative to nuclear genomes. In fact, the only strong evidence for a reduced efficacy of selection in animal mtDNA, relative to nuclear genomes, comes from comparative studies of nuclear and mitochondrial tRNAs (<a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Lynch 1996</a>; <a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Lynch 1997</a>). As discussed previously in this article, in the absence of a pattern in <em>NI</em>, there are few patterns of molecular evolution in animal mtDNA indicative of deleterious mutation accumulation [but see <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Osada and Akashi (2012)</a>]. This pattern is in stark contrast to the patterns found in analogous nuclear regions with reduced <em>N<sub>e</sub></em> and low recombination, like the Y chromosome. Determining whether mtDNA accumulate deleterious polymorphisms and substitutions more readily than nuclear DNA in a larger sample of species (and what type of loci may be affected) will be a particularly fruitful avenue for future studies.</p></div><div class="section" id="sec-15"></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-61">We thank Colin Meiklejohn and members of the Montooth and Hahn labs for constructive feedback. B.S.C. was supported on the Indiana University Genetics, Molecular and Cellular Sciences Training Grant T32-GM007757 funded by the National Institutes of Health and a Doctoral Dissertation Improvement Grant funded by the National Science Foundation. This research was supported by funding from Indiana University, award DBI-0845494, from the National Science Foundation to M.W.H. and a National Science Foundation CAREER award IOS-1149178 to K.L.M.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-supplementary-material" id="fn-23"><p id="p-62">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.016493/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.016493/-/DC1</a></p></li><li class="fn" id="fn-24"><p id="p-63"><em>Communicating editor: S. I. Wright</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2015-06-05"><span class="received-label">Received </span>June 5, 2015.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2015-08-17"><span class="accepted-label">Accepted </span>August 17, 2015.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2015 Cooper <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-3">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2165.1" data-doi="10.1534/genetics.112.140178"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Akashi</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Osada</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ohta</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Weak selection and protein evolution.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">192</span>: <span class="cit-fpage">15</span>–<span class="cit-lpage">31</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DAkashi%26rft.auinit1%253DH.%26rft.volume%253D192%26rft.issue%253D1%26rft.spage%253D15%26rft.epage%253D31%26rft.atitle%253DWeak%2BSelection%2Band%2BProtein%2BEvolution%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.140178%26rft_id%253Dinfo%253Apmid%252F22964835%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMTkyLzEvMTUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzEwLzIxNjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2165.2" data-doi="10.1126/science.286.5449.2524"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Awadalla</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eyre-Walker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Linkage disequilibrium and recombination in hominid mitochondrial DNA.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">286</span>: <span class="cit-fpage">2524</span>–<span class="cit-lpage">2525</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DAwadalla%26rft.auinit1%253DP.%26rft.volume%253D286%26rft.issue%253D5449%26rft.spage%253D2524%26rft.epage%253D2525%26rft.atitle%253DLinkage%2BDisequilibrium%2Band%2BRecombination%2Bin%2BHominid%2BMitochondrial%2BDNA%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.286.5449.2524%26rft_id%253Dinfo%253Apmid%252F10617471%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyODYvNTQ0OS8yNTI0IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMC8yMTY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2165.3" data-doi="10.1038/nrg3366"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bachtrog</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Y-chromosome evolution: emerging insights into processes of Y-chromosome degeneration.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">113</span>–<span class="cit-lpage">124</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D14%26rft.spage%253D113%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3366%26rft_id%253Dinfo%253Apmid%252F23329112%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3366&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23329112&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2165.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2165.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ballard</span>  <span class="cit-name-given-names">J. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Comparative genomics of mitochondrial DNA in members of the <em>Drosophila melanogaster</em> subgroup.</span> <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">51</span>: <span class="cit-fpage">48</span>–<span class="cit-lpage">63</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DBallard%26rft.auinit1%253DJ.%2BW.%26rft.volume%253D51%26rft.issue%253D1%26rft.spage%253D48%26rft.epage%253D63%26rft.atitle%253DComparative%2Bgenomics%2Bof%2Bmitochondrial%2BDNA%2Bin%2Bmembers%2Bof%2Bthe%2BDrosophila%2Bmelanogaster%2Bsubgroup.%26rft_id%253Dinfo%253Apmid%252F10903372%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10903372&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2165.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000088151700006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2165.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ballard</span>  <span class="cit-name-given-names">J. W. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kreitman</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Unraveling selection in the mitochondrial genome of <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">138</span>: <span class="cit-fpage">757</span>–<span class="cit-lpage">772</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DBallard%26rft.auinit1%253DJ.%2BW.%26rft.volume%253D138%26rft.issue%253D3%26rft.spage%253D757%26rft.epage%253D772%26rft.atitle%253DUnraveling%2Bselection%2Bin%2Bthe%2Bmitochondrial%2Bgenome%2Bof%2BDrosophila.%26rft_id%253Dinfo%253Apmid%252F7851772%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTM4LzMvNzU3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMC8yMTY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2165.6" data-doi="10.1146/annurev.ecolsys.36.091704.175513"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ballard</span>  <span class="cit-name-given-names">J. W. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rand</span>  <span class="cit-name-given-names">D. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">The population biology of mitochondrial DNA and its phylogenetic implications.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Ecol. Evol. Syst.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">621</span>–<span class="cit-lpage">642</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRev.%2BEcol.%2BEvol.%2BSyst.%26rft.volume%253D36%26rft.spage%253D621%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.ecolsys.36.091704.175513%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.ecolsys.36.091704.175513&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2165.7" data-doi="10.1046/j.1365-294X.2003.02063.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ballard</span>  <span class="cit-name-given-names">J. W. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whitlock</span>  <span class="cit-name-given-names">M. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">The incomplete natural history of mitochondria.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">729</span>–<span class="cit-lpage">744</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DBallard%26rft.auinit1%253DJ.%2BW.%26rft.volume%253D13%26rft.issue%253D4%26rft.spage%253D729%26rft.epage%253D744%26rft.atitle%253DThe%2Bincomplete%2Bnatural%2Bhistory%2Bof%2Bmitochondria.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-294X.2003.02063.x%26rft_id%253Dinfo%253Apmid%252F15012752%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-294X.2003.02063.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15012752&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2165.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000220153000002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2165.8" data-doi="10.1126/science.1122033"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bazin</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glemin</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Galtier</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Population size does not influence mitochondrial genetic diversity in animals.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">312</span>: <span class="cit-fpage">570</span>–<span class="cit-lpage">572</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DBazin%26rft.auinit1%253DE.%26rft.volume%253D312%26rft.issue%253D5773%26rft.spage%253D570%26rft.epage%253D572%26rft.atitle%253DPopulation%2Bsize%2Bdoes%2Bnot%2Binfluence%2Bmitochondrial%2Bgenetic%2Bdiversity%2Bin%2Banimals.%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1122033%26rft_id%253Dinfo%253Apmid%252F16645093%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMTIvNTc3My81NzAiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzEwLzIxNjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2165.9" data-doi="10.1038/nature13206"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bellott</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hughes</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Skaletsky</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">L. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pyntikova</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Mammalian Y chromosomes retain widely expressed dosage-sensitive regulators.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">508</span>: <span class="cit-fpage">494</span>–<span class="cit-lpage">499</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D508%26rft.spage%253D494%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature13206%26rft_id%253Dinfo%253Apmid%252F24759411%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature13206&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24759411&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2165.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000334741600030&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2165.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bergstrom</span>  <span class="cit-name-given-names">C. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Germline bottlenecks and the evolutionary maintenance of mitochondrial genomes.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">149</span>: <span class="cit-fpage">2135</span>–<span class="cit-lpage">2146</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DBergstrom%26rft.auinit1%253DC.%2BT.%26rft.volume%253D149%26rft.issue%253D4%26rft.spage%253D2135%26rft.epage%253D2146%26rft.atitle%253DGermline%2BBottlenecks%2Band%2Bthe%2BEvolutionary%2BMaintenance%2Bof%2BMitochondrial%2BGenomes%26rft_id%253Dinfo%253Apmid%252F9691064%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE0OS80LzIxMzUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzEwLzIxNjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2165.11" data-doi="10.1111/j.1558-5646.2012.01628.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Betancourt</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blanco-Martin</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The relation between the neutrality index for mitochondrial genes and the distribution of mutational effects on fitness.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">66</span>: <span class="cit-fpage">2427</span>–<span class="cit-lpage">2438</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DBetancourt%26rft.auinit1%253DA.%2BJ.%26rft.volume%253D66%26rft.issue%253D8%26rft.spage%253D2427%26rft.epage%253D2438%26rft.atitle%253DThe%2Brelation%2Bbetween%2Bthe%2Bneutrality%2Bindex%2Bfor%2Bmitochondrial%2Bgenes%2Band%2Bthe%2Bdistribution%2Bof%2Bmutational%2Beffects%2Bon%2Bfitness.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1558-5646.2012.01628.x%26rft_id%253Dinfo%253Apmid%252F22834742%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1558-5646.2012.01628.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22834742&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2165.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000306804100007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2165.12" data-doi="10.1093/nar/27.8.1767"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boore</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Animal mitochondrial genomes.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">1767</span>–<span class="cit-lpage">1780</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F27.8.1767%26rft_id%253Dinfo%253Apmid%252F10101183%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjk6IjI3LzgvMTc2NyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTAvMjE2NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2165.13" data-doi="10.1534/genetics.105.048975"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bruen</span>  <span class="cit-name-given-names">T. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Philippe</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bryant</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A simple and robust statistical test for detecting the presence of recombination.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">172</span>: <span class="cit-fpage">2665</span>–<span class="cit-lpage">2681</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.048975%26rft_id%253Dinfo%253Apmid%252F16489234%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Mi80LzI2NjUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzEwLzIxNjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2165.14" data-doi="10.1038/nature04240"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bustamante</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fledel-Alon</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williamson</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hubisz</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Natural selection on protein-coding genes in the human genome.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">437</span>: <span class="cit-fpage">1153</span>–<span class="cit-lpage">1157</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DBustamante%26rft.auinit1%253DC.%2BD.%26rft.volume%253D437%26rft.issue%253D7062%26rft.spage%253D1153%26rft.epage%253D1157%26rft.atitle%253DNatural%2Bselection%2Bon%2Bprotein-coding%2Bgenes%2Bin%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature04240%26rft_id%253Dinfo%253Apmid%252F16237444%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature04240&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16237444&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2165.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000232660500044&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2165.15" data-doi="10.1101/gr.156034.113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carvalho</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">A. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Efficient identification of Y chromosome sequences in the human and <em>Drosophila</em> genomes.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">1894</span>–<span class="cit-lpage">1907</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.156034.113%26rft_id%253Dinfo%253Apmid%252F23921660%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIyMy8xMS8xODk0IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMC8yMTY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2165.16" data-doi="10.1016/j.tig.2009.04.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carvalho</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koerich</span>  <span class="cit-name-given-names">L. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">A. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Origin and evolution of Y chromosomes: <em>Drosophila</em> tales.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">270</span>–<span class="cit-lpage">277</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DBernardo%2BCarvalho%26rft.auinit1%253DA.%26rft.volume%253D25%26rft.issue%253D6%26rft.spage%253D270%26rft.epage%253D277%26rft.atitle%253DOrigin%2Band%2Bevolution%2Bof%2BY%2Bchromosomes%253A%2BDrosophila%2Btales.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2009.04.002%26rft_id%253Dinfo%253Apmid%252F19443075%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2009.04.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19443075&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2165.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000267587000005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2165.17" data-doi="10.1534/genetics.111.134288"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The effects of deleterious mutations on evolution at linked sites.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">5</span>–<span class="cit-lpage">22</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DCharlesworth%26rft.auinit1%253DB.%26rft.volume%253D190%26rft.issue%253D1%26rft.spage%253D5%26rft.epage%253D22%26rft.atitle%253DThe%2BEffects%2Bof%2BDeleterious%2BMutations%2Bon%2BEvolution%2Bat%2BLinked%2BSites%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.134288%26rft_id%253Dinfo%253Apmid%252F22219506%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6NzoiMTkwLzEvNSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTAvMjE2NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2165.18" data-doi="10.1098/rstb.2000.0717"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">The degeneration of Y chromosomes.</span> <abbr class="cit-jnl-abbrev">Philos. Trans. R. Soc. Lond. B Biol. Sci.</abbr> <span class="cit-vol">355</span>: <span class="cit-fpage">1563</span>–<span class="cit-lpage">1572</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhilosophical%2BTransactions%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DPhil%2BTrans%2BR%2BSoc%2BB%26rft.aulast%253DCharlesworth%26rft.auinit1%253DB.%26rft.volume%253D355%26rft.issue%253D1403%26rft.spage%253D1563%26rft.epage%253D1572%26rft.atitle%253DThe%2Bdegeneration%2Bof%2BY%2Bchromosomes%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.2000.0717%26rft_id%253Dinfo%253Apmid%252F11127901%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoicm95cHRiIjtzOjU6InJlc2lkIjtzOjEzOiIzNTUvMTQwMy8xNTYzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMC8yMTY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2165.19" data-doi="10.1007/BF02099755"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Clary</span>  <span class="cit-name-given-names">D. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wolstenholme</span>  <span class="cit-name-given-names">D. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1985</span> <span class="cit-article-title">The mitochondrial DNA molecule of <em>Drosophila yakuba</em>: nucleotide sequence, gene organization, and genetic code.</span> <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">252</span>–<span class="cit-lpage">271</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DClary%26rft.auinit1%253DD.%2BO.%26rft.volume%253D22%26rft.issue%253D3%26rft.spage%253D252%26rft.epage%253D271%26rft.atitle%253DThe%2Bmitochondrial%2BDNA%2Bmolecular%2Bof%2BDrosophila%2Byakuba%253A%2Bnucleotide%2Bsequence%252C%2Bgene%2Borganization%252C%2Band%2Bgenetic%2Bcode.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF02099755%26rft_id%253Dinfo%253Apmid%252F3001325%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF02099755&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3001325&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2165.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1985AUU2900007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2165.20" data-doi="10.1046/j.1365-294x.2000.01020.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Clement</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Posada</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crandall</span>  <span class="cit-name-given-names">K. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">TCS: a computer program to estimate gene genealogies.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">1657</span>–<span class="cit-lpage">1659</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DClement%26rft.auinit1%253DM.%26rft.volume%253D9%26rft.issue%253D10%26rft.spage%253D1657%26rft.epage%253D1659%26rft.atitle%253DTCS%253A%2Ba%2Bcomputer%2Bprogram%2Bto%2Bestimate%2Bgene%2Bgenealogies.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-294x.2000.01020.x%26rft_id%253Dinfo%253Apmid%252F11050560%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-294x.2000.01020.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11050560&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2165.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000089998600020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2165.21" data-doi="10.1038/nrg2146"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eyre-Walker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keightley</span>  <span class="cit-name-given-names">P. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The distribution of fitness effects of new mutations.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">610</span>–<span class="cit-lpage">618</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DEyre-Walker%26rft.auinit1%253DA.%26rft.volume%253D8%26rft.issue%253D8%26rft.spage%253D610%26rft.epage%253D618%26rft.atitle%253DThe%2Bdistribution%2Bof%2Bfitness%2Beffects%2Bof%2Bnew%2Bmutations.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2146%26rft_id%253Dinfo%253Apmid%252F17637733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2146&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17637733&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2165.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000248170700015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2165.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fay</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">C. I.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Hitchhiking under positive Darwinian selection.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">155</span>: <span class="cit-fpage">1405</span>–<span class="cit-lpage">1413</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DFay%26rft.auinit1%253DJ.%2BC.%26rft.volume%253D155%26rft.issue%253D3%26rft.spage%253D1405%26rft.epage%253D1413%26rft.atitle%253DHitchhiking%2BUnder%2BPositive%2BDarwinian%2BSelection%26rft_id%253Dinfo%253Apmid%252F10880498%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1NS8zLzE0MDUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzEwLzIxNjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2165.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">Y. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Statistical tests of neutrality of mutations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">133</span>: <span class="cit-fpage">693</span>–<span class="cit-lpage">709</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DFu%26rft.auinit1%253DY.%2BX.%26rft.volume%253D133%26rft.issue%253D3%26rft.spage%253D693%26rft.epage%253D709%26rft.atitle%253DStatistical%2Btests%2Bof%2Bneutrality%2Bof%2Bmutations.%26rft_id%253Dinfo%253Apmid%252F8454210%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTMzLzMvNjkzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMC8yMTY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2165.24" data-doi="10.1093/gbe/evu198"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Garrigan</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kingan</span>  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Geneva</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vedanayagam</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Presgraves</span>  <span class="cit-name-given-names">D. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genome diversity and divergence in <em>Drosophila mauritiana</em>: Multiple signatures of faster X evolution.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">2444</span>–<span class="cit-lpage">2458</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevu198%26rft_id%253Dinfo%253Apmid%252F25193308%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2JlIjtzOjU6InJlc2lkIjtzOjg6IjYvOS8yNDQ0IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMC8yMTY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2165.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gillespie</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Genetic drift in an infinite population: the pseudohitchhiking model.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">155</span>: <span class="cit-fpage">909</span>–<span class="cit-lpage">919</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DGillespie%26rft.auinit1%253DJ.%2BH.%26rft.volume%253D155%26rft.issue%253D2%26rft.spage%253D909%26rft.epage%253D919%26rft.atitle%253DGenetic%2BDrift%2Bin%2Ban%2BInfinite%2BPopulation%253A%2BThe%2BPseudohitchhiking%2BModel%26rft_id%253Dinfo%253Apmid%252F10835409%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU1LzIvOTA5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMC8yMTY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2165.26" data-doi="10.1093/molbev/msq079"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gossmann</span>  <span class="cit-name-given-names">T. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">B.-H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Windsor</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mitchell-Olds</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dixon</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genome wide analyses reveal little evidence for adaptive evolution in many plant species.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">1822</span>–<span class="cit-lpage">1832</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsq079%26rft_id%253Dinfo%253Apmid%252F20299543%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjI3LzgvMTgyMiI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTAvMjE2NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2165.27" data-doi="10.1016/j.cell.2008.06.021"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Green</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Malaspinas</span>  <span class="cit-name-given-names">A.-S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krause</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Briggs</span>  <span class="cit-name-given-names">A. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">P. L. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A complete neandertal mitochondrial genome sequence determined by high-throughput sequencing.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">134</span>: <span class="cit-fpage">416</span>–<span class="cit-lpage">426</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DGreen%26rft.auinit1%253DR.%2BE.%26rft.volume%253D134%26rft.issue%253D3%26rft.spage%253D416%26rft.epage%253D426%26rft.atitle%253DA%2Bcomplete%2BNeandertal%2Bmitochondrial%2Bgenome%2Bsequence%2Bdetermined%2Bby%2Bhigh-throughput%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2008.06.021%26rft_id%253Dinfo%253Apmid%252F18692465%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2008.06.021&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18692465&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2165.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000258665500015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2165.28" data-doi="10.1038/nature05388"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haag-Liautard</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dorris</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maside</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Macaskill</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Halligan</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Direct estimation of per nucleotide and genomic deleterious mutation rates in <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">445</span>: <span class="cit-fpage">82</span>–<span class="cit-lpage">85</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DHaag-Liautard%26rft.auinit1%253DC.%26rft.volume%253D445%26rft.issue%253D7123%26rft.spage%253D82%26rft.epage%253D85%26rft.atitle%253DDirect%2Bestimation%2Bof%2Bper%2Bnucleotide%2Band%2Bgenomic%2Bdeleterious%2Bmutation%2Brates%2Bin%2BDrosophila.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature05388%26rft_id%253Dinfo%253Apmid%252F17203060%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature05388&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17203060&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2165.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000243225100040&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2165.29" data-doi="10.1371/journal.pbio.0060204"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haag-Liautard</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coffey</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Houle</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Direct estimation of the mitochondrial DNA mutation rate in <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e204</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DHaag-Liautard%26rft.auinit1%253DC.%26rft.volume%253D6%26rft.issue%253D8%26rft.spage%253De204%26rft.epage%253De204%26rft.atitle%253DDirect%2Bestimation%2Bof%2Bthe%2Bmitochondrial%2BDNA%2Bmutation%2Brate%2Bin%2BDrosophila%2Bmelanogaster.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0060204%26rft_id%253Dinfo%253Apmid%252F18715119%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0060204&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18715119&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2165.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2165.30" data-doi="10.1111/j.1558-5646.2007.00308.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hahn</span>  <span class="cit-name-given-names">M. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Toward a selection theory of molecular evolution.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">62</span>: <span class="cit-fpage">255</span>–<span class="cit-lpage">265</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DHahn%26rft.auinit1%253DM.%2BW.%26rft.volume%253D62%26rft.issue%253D2%26rft.spage%253D255%26rft.epage%253D265%26rft.atitle%253DToward%2Ba%2Bselection%2Btheory%2Bof%2Bmolecular%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1558-5646.2007.00308.x%26rft_id%253Dinfo%253Apmid%252F18302709%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1558-5646.2007.00308.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18302709&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2165.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000253373900001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2165.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hahn</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rausher</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cunningham</span>  <span class="cit-name-given-names">C. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Distinguishing between selection and population expansion in an experimental lineage of bacteriophage t7.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">161</span>: <span class="cit-fpage">11</span>–<span class="cit-lpage">20</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHahn%26rft.auinit1%253DM.%2BW.%26rft.volume%253D161%26rft.issue%253D1%26rft.spage%253D11%26rft.epage%253D20%26rft.atitle%253DDistinguishing%2BBetween%2BSelection%2Band%2BPopulation%2BExpansion%2Bin%2Ban%2BExperimental%2BLineage%2Bof%2BBacteriophage%2BT7%26rft_id%253Dinfo%253Apmid%252F12019219%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMTYxLzEvMTEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzEwLzIxNjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2165.32" data-doi="10.1017/S0016672300010156"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">W. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robertson</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1966</span> <span class="cit-article-title">The effect of linkage on limits to artificial selection.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">269</span>–<span class="cit-lpage">294</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DHill%26rft.auinit1%253DW.%2BG.%26rft.volume%253D8%26rft.issue%253D3%26rft.spage%253D269%26rft.epage%253D294%26rft.atitle%253DThe%2Beffect%2Bof%2Blinkage%2Bon%2Blimits%2Bto%2Bartificial%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672300010156%26rft_id%253Dinfo%253Apmid%252F5980116%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672300010156&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=5980116&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2165.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A19668750300002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2165.33" data-doi="10.1073/pnas.92.2.532"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Horai</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayasaka</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kondo</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsugane</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Takahata</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Recent African origin of modern humans revealed by complete sequences of hominoid mitochondrial DNAs.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">532</span>–<span class="cit-lpage">536</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DHorai%26rft.auinit1%253DS.%26rft.volume%253D92%26rft.issue%253D2%26rft.spage%253D532%26rft.epage%253D536%26rft.atitle%253DRecent%2BAfrican%2Borigin%2Bof%2Bmodern%2Bhumans%2Brevealed%2Bby%2Bcomplete%2Bsequences%2Bof%2Bhominoid%2Bmitochondrial%2BDNAs.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.92.2.532%26rft_id%253Dinfo%253Apmid%252F7530363%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo4OiI5Mi8yLzUzMiI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTAvMjE2NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2165.34"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hudson</span>  <span class="cit-name-given-names">R. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaplan</span>  <span class="cit-name-given-names">N. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1985</span> <span class="cit-article-title">Statistical properties of the number of recombination events in the history of a sample of DNA sequences.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">111</span>: <span class="cit-fpage">147</span>–<span class="cit-lpage">164</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHudson%26rft.auinit1%253DR.%2BR.%26rft.volume%253D111%26rft.issue%253D1%26rft.spage%253D147%26rft.epage%253D164%26rft.atitle%253DSTATISTICAL%2BPROPERTIES%2BOF%2BTHE%2BNUMBER%2BOF%2BRECOMBINATION%2BEVENTS%2BIN%2BTHE%2BHISTORY%2BOF%2BA%2BSAMPLE%2BOF%2BDNA%2BSEQUENCES%26rft_id%253Dinfo%253Apmid%252F4029609%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTExLzEvMTQ3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMC8yMTY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2165.35" data-doi="10.1093/oxfordjournals.molbev.a004170"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Innan</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nordborg</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Recombination or mutational hot spots in human mtDNA?</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1122</span>–<span class="cit-lpage">1127</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DInnan%26rft.auinit1%253DH.%26rft.volume%253D19%26rft.issue%253D7%26rft.spage%253D1122%26rft.epage%253D1127%26rft.atitle%253DRecombination%2Bor%2BMutational%2BHot%2BSpots%2Bin%2BHuman%2BmtDNA%253F%26rft_id%253Dinfo%253Adoi%252F10.1093%252Foxfordjournals.molbev.a004170%26rft_id%253Dinfo%253Apmid%252F12082131%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjE5LzcvMTEyMiI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTAvMjE2NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2165.36" data-doi="10.1016/j.fsigen.2007.12.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Just</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Diegoli</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saunier</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Irwin</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parsons</span>  <span class="cit-name-given-names">T. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Complete mitochondrial genome sequences for 265 African American and U.S. “Hispanic” individuals.</span> <abbr class="cit-jnl-abbrev">Forensic Sci. Int. Genet.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">e45</span>–<span class="cit-lpage">e48</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DForensic%2BSci%2BInt%2BGenet%26rft.aulast%253DJust%26rft.auinit1%253DR.%2BS.%26rft.volume%253D2%26rft.issue%253D3%26rft.spage%253De45%26rft.epage%253De48%26rft.atitle%253DComplete%2Bmitochondrial%2Bgenome%2Bsequences%2Bfor%2B265%2BAfrican%2BAmerican%2Band%2BU.S.%2B%2522Hispanic%2522%2Bindividuals.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.fsigen.2007.12.001%26rft_id%253Dinfo%253Apmid%252F19083815%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.fsigen.2007.12.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19083815&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2165.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-book" id="cit-5.10.2165.37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kimura</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-source">The Neutral Theory of Molecular Evolution</span>. <span class="cit-publ-name">Cambridge University Press</span>, <span class="cit-publ-loc">Cambridge, United Kingdom</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2165.38" data-doi="10.1534/genetics.112.142018"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langley</span>  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stevens</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cardeno</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">Y. C. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schrider</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic variation in natural populations of <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">192</span>: <span class="cit-fpage">533</span>–<span class="cit-lpage">598</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.142018%26rft_id%253Dinfo%253Apmid%252F22673804%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkyLzIvNTMzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMC8yMTY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2165.39" data-doi="10.1093/bioinformatics/btm404"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Larkin</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blackshields</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">N. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chenna</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McGettigan</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Clustal W and clustal X version 2.0.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">2947</span>–<span class="cit-lpage">2948</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtm404%26rft_id%253Dinfo%253Apmid%252F17846036%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjMvMjEvMjk0NyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTAvMjE2NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2165.40"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lewontin</span>  <span class="cit-name-given-names">R. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1964</span> <span class="cit-article-title">The interaction of selection and linkage. I. General considerations; heterotic models.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">49</span>–<span class="cit-lpage">67</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLewontin%26rft.auinit1%253DR.%2BC.%26rft.volume%253D49%26rft.issue%253D1%26rft.spage%253D49%26rft.epage%253D67%26rft.atitle%253DTHE%2BINTERACTION%2BOF%2BSELECTION%2BAND%2BLINKAGE.%2BI.%2BGENERAL%2BCONSIDERATIONS%253B%2BHETEROTIC%2BMODELS%26rft_id%253Dinfo%253Apmid%252F17248194%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo3OiI0OS8xLzQ5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMC8yMTY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2165.41" data-doi="10.1093/bioinformatics/btp324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fast and accurate short read alignment with Burrows-Wheeler transform.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1754</span>–<span class="cit-lpage">1760</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp324%26rft_id%253Dinfo%253Apmid%252F19451168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTQvMTc1NCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTAvMjE2NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2165.42" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Handsaker</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wysoker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fennell</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The Sequence Alignment/Map format and SAMtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2078</span>–<span class="cit-lpage">2079</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTYvMjA3OCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTAvMjE2NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2165.43" data-doi="10.1111/j.1558-5646.2008.00486.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Costello</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holloway</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hahn</span>  <span class="cit-name-given-names">M. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">“Reverse ecology” and the power of population genomics.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">62</span>: <span class="cit-fpage">2984</span>–<span class="cit-lpage">2994</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DLi%26rft.auinit1%253DY.%2BF.%26rft.volume%253D62%26rft.issue%253D12%26rft.spage%253D2984%26rft.epage%253D2994%26rft.atitle%253D%2522Reverse%2Becology%2522%2Band%2Bthe%2Bpower%2Bof%2Bpopulation%2Bgenomics.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1558-5646.2008.00486.x%26rft_id%253Dinfo%253Apmid%252F18752601%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1558-5646.2008.00486.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18752601&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2165.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261442900004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2165.44" data-doi="10.1093/oxfordjournals.molbev.a025557"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Mutation accumulation in transfer RNAs: molecular evidence for Muller’s ratchet in mitochondrial genomes.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">209</span>–<span class="cit-lpage">220</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DLynch%26rft.auinit1%253DM%26rft.volume%253D13%26rft.issue%253D1%26rft.spage%253D209%26rft.epage%253D220%26rft.atitle%253DMutation%2Baccumulation%2Bin%2Btransfer%2BRNAs%253A%2Bmolecular%2Bevidence%2Bfor%2BMuller%2527s%2Bratchet%2Bin%2Bmitochondrial%2Bgenomes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Foxfordjournals.molbev.a025557%26rft_id%253Dinfo%253Apmid%252F8583893%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjEzLzEvMjA5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMC8yMTY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2165.45" data-doi="10.1093/oxfordjournals.molbev.a025834"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Mutation accumulation in nuclear, organelle, and prokaryotic transfer RNA genes.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">914</span>–<span class="cit-lpage">925</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Foxfordjournals.molbev.a025834%26rft_id%253Dinfo%253Apmid%252F9287424%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjE0LzkvOTE0IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMC8yMTY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-book" id="cit-5.10.2165.46"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-source">Origins of Genome Architecture</span>. <span class="cit-publ-name">Sinauer Associates, Inc.</span>, <span class="cit-publ-loc">Sunderland, Massachusetts</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2165.47" data-doi="10.1073/pnas.0803466105"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sung</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morris</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coffey</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Landry</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A genome-wide view of the spectrum of spontaneous mutations in yeast.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">9272</span>–<span class="cit-lpage">9277</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0803466105%26rft_id%253Dinfo%253Apmid%252F18583475%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA1LzI3LzkyNzIiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzEwLzIxNjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2165.48" data-doi="10.1038/nature10811"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>  <span class="cit-name-given-names">T. F. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Richards</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stone</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barbadilla</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ayroles</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The <em>Drosophila melanogaster</em> genetic reference panel.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">482</span>: <span class="cit-fpage">173</span>–<span class="cit-lpage">178</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DMackay%26rft.auinit1%253DT.%2BF.%26rft.volume%253D482%26rft.issue%253D7384%26rft.spage%253D173%26rft.epage%253D178%26rft.atitle%253DThe%2BDrosophila%2Bmelanogaster%2BGenetic%2BReference%2BPanel.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10811%26rft_id%253Dinfo%253Apmid%252F22318601%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10811&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22318601&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2165.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000299994100029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2165.49" data-doi="10.1017/S0016672300014634"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Maynard Smith</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haigh</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1974</span> <span class="cit-article-title">The hitch-hiking effect of a favourable gene.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">23</span>–<span class="cit-lpage">35</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DSmith%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D23%26rft.issue%253D1%26rft.spage%253D23%26rft.epage%253D35%26rft.atitle%253DThe%2Bhitch-hiking%2Beffect%2Bof%2Ba%2Bfavourable%2Bgene.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672300014634%26rft_id%253Dinfo%253Apmid%252F4407212%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672300014634&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=4407212&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2165.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1974T216900003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2165.50" data-doi="10.1038/351652a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McDonald</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kreitman</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Adaptive protein evolution at the <em>Adh</em> locus in <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">351</span>: <span class="cit-fpage">652</span>–<span class="cit-lpage">654</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DMcDonald%26rft.auinit1%253DJ.%2BH.%26rft.volume%253D351%26rft.issue%253D6328%26rft.spage%253D652%26rft.epage%253D654%26rft.atitle%253DAdaptive%2Bprotein%2Bevolution%2Bat%2Bthe%2BAdh%2Blocus%2Bin%2BDrosophila.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F351652a0%26rft_id%253Dinfo%253Apmid%252F1904993%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/351652a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1904993&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2165.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1991FT11200066&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2165.51" data-doi="10.1016/j.tig.2007.03.008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meiklejohn</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Montooth</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rand</span>  <span class="cit-name-given-names">D. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Positive and negative selection on the mitochondrial genome.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">259</span>–<span class="cit-lpage">263</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DMeiklejohn%26rft.auinit1%253DC.%2BD.%26rft.volume%253D23%26rft.issue%253D6%26rft.spage%253D259%26rft.epage%253D263%26rft.atitle%253DPositive%2Band%2Bnegative%2Bselection%2Bon%2Bthe%2Bmitochondrial%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2007.03.008%26rft_id%253Dinfo%253Apmid%252F17418445%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2007.03.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17418445&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2165.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000247231300001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2165.52" data-doi="10.1016/j.tig.2013.05.009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meisel</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Connallon</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The faster-X effect: integrating theory and data.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">537</span>–<span class="cit-lpage">544</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BGenet.%26rft.volume%253D29%26rft.spage%253D537%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2013.05.009%26rft_id%253Dinfo%253Apmid%252F23790324%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2013.05.009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23790324&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2165.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000324284000008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2165.53" data-doi="10.1073/pnas.1220835110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Messer</span>  <span class="cit-name-given-names">P. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petrov</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Frequent adaptation and the McDonald–Kreitman test.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">8615</span>–<span class="cit-lpage">8620</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1220835110%26rft_id%253Dinfo%253Apmid%252F23650353%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTEwLzIxLzg2MTUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzEwLzIxNjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2165.54" data-doi="10.1093/oxfordjournals.molbev.a003756"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meunier</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eyre-Walker</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">The correlation between linkage disequilibrium and distance: implications for recombination in hominid mitochondria.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">2132</span>–<span class="cit-lpage">2135</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DMeunier%26rft.auinit1%253DJ.%26rft.volume%253D18%26rft.issue%253D11%26rft.spage%253D2132%26rft.epage%253D2135%26rft.atitle%253DThe%2BCorrelation%2BBetween%2BLinkage%2BDisequilibrium%2Band%2BDistance%253A%2BImplications%2Bfor%2BRecombination%2Bin%2BHominid%2BMitochondria%26rft_id%253Dinfo%253Adoi%252F10.1093%252Foxfordjournals.molbev.a003756%26rft_id%253Dinfo%253Apmid%252F11606711%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjEwOiIxOC8xMS8yMTMyIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMC8yMTY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2165.55" data-doi="10.1371/journal.pbio.0060213"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Montooth</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rand</span>  <span class="cit-name-given-names">D. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The spectrum of mitochondrial mutation differs across species.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e213</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DMontooth%26rft.auinit1%253DK.%2BL.%26rft.volume%253D6%26rft.issue%253D8%26rft.spage%253De213%26rft.epage%253De213%26rft.atitle%253DThe%2Bspectrum%2Bof%2Bmitochondrial%2Bmutation%2Bdiffers%2Bacross%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0060213%26rft_id%253Dinfo%253Apmid%252F18752353%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0060213&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18752353&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2165.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2165.56" data-doi="10.1007/s00239-009-9255-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Montooth</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abt</span>  <span class="cit-name-given-names">D. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hofmann</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rand</span>  <span class="cit-name-given-names">D. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Comparative genomics of <em>Drosophila</em> mtDNA: Novel features of conservation and change across functional domains and lineages.</span> <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">69</span>: <span class="cit-fpage">94</span>–<span class="cit-lpage">114</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DMontooth%26rft.auinit1%253DK.%2BL.%26rft.volume%253D69%26rft.issue%253D1%26rft.spage%253D94%26rft.epage%253D114%26rft.atitle%253DComparative%2Bgenomics%2Bof%2BDrosophila%2BmtDNA%253A%2BNovel%2Bfeatures%2Bof%2Bconservation%2Band%2Bchange%2Bacross%2Bfunctional%2Bdomains%2Band%2Blineages.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00239-009-9255-0%26rft_id%253Dinfo%253Apmid%252F19533212%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00239-009-9255-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19533212&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2165.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268577800009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2165.57" data-doi="10.1023/A:1017030708374"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nachman</span>  <span class="cit-name-given-names">M. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Deleterious mutations in animal mitochondrial DNA.</span> <abbr class="cit-jnl-abbrev">Genetica</abbr> <span class="cit-vol">102–103</span>: <span class="cit-fpage">61</span>–<span class="cit-lpage">69</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetica%26rft.volume%253D102103%26rft.spage%253D61%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1017030708374%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/A:1017030708374&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-2" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2165.58"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nachman</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">W. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stoneking</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aquadro</span>  <span class="cit-name-given-names">C. F.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Nonneutral mitochondrial DNA variation in humans and chimpanzees.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">142</span>: <span class="cit-fpage">953</span>–<span class="cit-lpage">963</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DNachman%26rft.auinit1%253DM.%2BW.%26rft.volume%253D142%26rft.issue%253D3%26rft.spage%253D953%26rft.epage%253D963%26rft.atitle%253DNonneutral%2BMitochondrial%2BDNA%2BVariation%2Bin%2BHumans%2Band%2BChimpanzees%26rft_id%253Dinfo%253Apmid%252F8849901%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTQyLzMvOTUzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMC8yMTY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2165.59"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nei</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gojobori</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span> <span class="cit-article-title">Simple methods for estimating the numbers of synonymous and nonsynonymous nucleotide substitutions.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">418</span>–<span class="cit-lpage">426</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DNei%26rft.auinit1%253DM%26rft.volume%253D3%26rft.issue%253D5%26rft.spage%253D418%26rft.epage%253D426%26rft.atitle%253DSimple%2Bmethods%2Bfor%2Bestimating%2Bthe%2Bnumbers%2Bof%2Bsynonymous%2Band%2Bnonsynonymous%2Bnucleotide%2Bsubstitutions%26rft_id%253Dinfo%253Apmid%252F3444411%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjc6IjMvNS80MTgiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzEwLzIxNjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2165.60" data-doi="10.1098/rspb.2008.1758"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Neiman</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">D. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The causes of mutation accumulation in mitochondrial genomes.</span> <abbr class="cit-jnl-abbrev">Proc. Biol. Sci.</abbr> <span class="cit-vol">276</span>: <span class="cit-fpage">1201</span>–<span class="cit-lpage">1209</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BBiol.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.2008.1758%26rft_id%253Dinfo%253Apmid%252F19203921%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Nzoicm95cHJzYiI7czo1OiJyZXNpZCI7czoxMzoiMjc2LzE2NjAvMTIwMSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTAvMjE2NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2165.61" data-doi="10.1093/molbev/msr211"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Osada</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Akashi</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Mitochondrial-nuclear interactions and accelerated compensatory evolution: evidence from the primate cytochrome C oxidase complex.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">337</span>–<span class="cit-lpage">346</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsr211%26rft_id%253Dinfo%253Apmid%252F21890478%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjI5LzEvMzM3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMC8yMTY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2165.62" data-doi="10.1093/molbev/mss219"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Popadin</span>  <span class="cit-name-given-names">K. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nikolaev</span>  <span class="cit-name-given-names">S. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Junier</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baranova</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Antonarakis</span>  <span class="cit-name-given-names">S. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Purifying selection in mammalian mitochondrial protein-coding genes is highly effective and congruent with evolution of nuclear genes.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">347</span>–<span class="cit-lpage">355</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmss219%26rft_id%253Dinfo%253Apmid%252F22983951%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjMwLzIvMzQ3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMC8yMTY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2165.63" data-doi="10.1016/j.cub.2005.07.065"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Presgraves</span>  <span class="cit-name-given-names">D. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Recombination enhances protein adaptation in <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1651</span>–<span class="cit-lpage">1656</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DPresgraves%26rft.auinit1%253DD.%2BC.%26rft.volume%253D15%26rft.issue%253D18%26rft.spage%253D1651%26rft.epage%253D1656%26rft.atitle%253DRecombination%2Benhances%2Bprotein%2Badaptation%2Bin%2BDrosophila%2Bmelanogaster.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2005.07.065%26rft_id%253Dinfo%253Apmid%252F16169487%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2005.07.065&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16169487&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2165.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000232235500024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-web" id="cit-5.10.2165.64"><div class="cit-metadata unstructured">R Core Team, 2012 R: A language and environment for statistical computing. R Foundation for Statistical Computing, Vienna, Austria ISBN 3–900051–07–0: URL, Available from: <a href="http://www.R-project.org/">http://www.R-project.org/</a>. Accessed August 31, 2015.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2165.65" data-doi="10.1007/s10709-011-9576-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rand</span>  <span class="cit-name-given-names">D. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Population genetics of the cytoplasm and the units of selection on mitochondrial DNA in <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Genetica</abbr> <span class="cit-vol">139</span>: <span class="cit-fpage">685</span>–<span class="cit-lpage">697</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetica%26rft.volume%253D139%26rft.spage%253D685%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10709-011-9576-y%26rft_id%253Dinfo%253Apmid%252F21538136%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10709-011-9576-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21538136&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2165.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2165.66" data-doi="10.1093/oxfordjournals.molbev.a025634"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rand</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kann</span>  <span class="cit-name-given-names">L. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Excess amino acid polymorphism in mitochondrial DNA: contrasts among genes from <em>Drosophila</em>, mice, and humans.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">735</span>–<span class="cit-lpage">748</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DRand%26rft.auinit1%253DD.%26rft.volume%253D13%26rft.issue%253D6%26rft.spage%253D735%26rft.epage%253D748%26rft.atitle%253DExcess%2Bamino%2Bacid%2Bpolymorphism%2Bin%2Bmitochondrial%2BDNA%253A%2Bcontrasts%2Bamong%2Bgenes%2Bfrom%2BDrosophila%252C%2Bmice%252C%2Band%2Bhumans%26rft_id%253Dinfo%253Adoi%252F10.1093%252Foxfordjournals.molbev.a025634%26rft_id%253Dinfo%253Apmid%252F8754210%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjEzLzYvNzM1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMC8yMTY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2165.67" data-doi="10.1023/A:1017006118852"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rand</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kann</span>  <span class="cit-name-given-names">L. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Mutation and selection at silent and replacement sites in the evolution of animal mitochondrial DNA.</span> <abbr class="cit-jnl-abbrev">Genetica</abbr> <span class="cit-vol">102–103</span>: <span class="cit-fpage">393</span>–<span class="cit-lpage">407</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetica%26rft.volume%253D102103%26rft.spage%253D393%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1017006118852%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/A:1017006118852&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2165.68" data-doi="10.1371/journal.pgen.1003129"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Richardson</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weinert</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Welch</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Linheiro</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Magwire</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Population genomics of the <em>Wolbachia</em> endosymbiont in <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1003129</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D8%26rft.spage%253De1003129%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003129%26rft_id%253Dinfo%253Apmid%252F23284297%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003129&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23284297&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2165.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2165.69" data-doi="10.1093/bioinformatics/btg359"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rozas</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sanchez-Delbarrio</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Messeguer</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rozas</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">DnaSP, DNA polymorphism analyses by the coalescent and other methods.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">2496</span>–<span class="cit-lpage">2497</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DRozas%26rft.auinit1%253DJ.%26rft.volume%253D19%26rft.issue%253D18%26rft.spage%253D2496%26rft.epage%253D2497%26rft.atitle%253DDnaSP%252C%2BDNA%2Bpolymorphism%2Banalyses%2Bby%2Bthe%2Bcoalescent%2Band%2Bother%2Bmethods%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtg359%26rft_id%253Dinfo%253Apmid%252F14668244%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMTkvMTgvMjQ5NiI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTAvMjE2NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2165.70" data-doi="10.1016/j.ajhg.2009.11.011"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rozen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marszalek</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alagappan</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Skaletsky</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Page</span>  <span class="cit-name-given-names">D. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Remarkably little variation in proteins encoded by the Y chromosome’s single-copy genes, implying effective purifying selection.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">85</span>: <span class="cit-fpage">923</span>–<span class="cit-lpage">928</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DRozen%26rft.auinit1%253DS.%26rft.volume%253D85%26rft.issue%253D6%26rft.spage%253D923%26rft.epage%253D928%26rft.atitle%253DRemarkably%2Blittle%2Bvariation%2Bin%2Bproteins%2Bencoded%2Bby%2Bthe%2BY%2Bchromosome%2527s%2Bsingle-copy%2Bgenes%252C%2Bimplying%2Beffective%2Bpurifying%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2009.11.011%26rft_id%253Dinfo%253Apmid%252F20004767%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2009.11.011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20004767&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2165.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000272797100017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2165.71" data-doi="10.1093/nar/gkr1086"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rubino</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Piredda</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calabrese</span>  <span class="cit-name-given-names">F. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simone</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lang</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">HmtDB, a genomic resource for mitochondrion-based human variability studies.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">D1150</span>–<span class="cit-lpage">D1159</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkr1086%26rft_id%253Dinfo%253Apmid%252F22139932%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjExOiI0MC9EMS9EMTE1MCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzUvMTAvMjE2NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2165.72"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sheldahl</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weinreich</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rand</span>  <span class="cit-name-given-names">D. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Recombination, dominance and selection on amino acid polymorphism in the <em>Drosophila</em> genome: contrasting patterns on the X and fourth chromosomes.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">165</span>: <span class="cit-fpage">1195</span>–<span class="cit-lpage">1208</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DSheldahl%26rft.auinit1%253DL.%2BA.%26rft.volume%253D165%26rft.issue%253D3%26rft.spage%253D1195%26rft.epage%253D1208%26rft.atitle%253DRecombination%252C%2BDominance%2Band%2BSelection%2Bon%2BAmino%2BAcid%2BPolymorphism%2Bin%2Bthe%2BDrosophila%2BGenome%253A%2BContrasting%2BPatterns%2Bon%2Bthe%2BX%2Band%2BFourth%2BChromosomes%26rft_id%253Dinfo%253Apmid%252F14668375%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2NS8zLzExOTUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzEwLzIxNjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2165.73" data-doi="10.1534/genetics.104.030890"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shoemaker</span>  <span class="cit-name-given-names">D. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dyer</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ahrens</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mcabee</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jaenike</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Decreased diversity but increased substitution rate in host mtDNA as a consequence of <em>Wolbachia</em> endosymbiont infection.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">168</span>: <span class="cit-fpage">2049</span>–<span class="cit-lpage">2058</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DShoemaker%26rft.auinit1%253DD.%2BD.%26rft.volume%253D168%26rft.issue%253D4%26rft.spage%253D2049%26rft.epage%253D2058%26rft.atitle%253DDecreased%2BDiversity%2Bbut%2BIncreased%2BSubstitution%2BRate%2Bin%2BHost%2BmtDNA%2Bas%2Ba%2BConsequence%2Bof%2BWolbachia%2BEndosymbiont%2BInfection%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.030890%26rft_id%253Dinfo%253Apmid%252F15611174%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2OC80LzIwNDkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzEwLzIxNjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2165.74" data-doi="10.1093/molbev/msu203"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">N. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koerich</span>  <span class="cit-name-given-names">L. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carvalho</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">A. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Positive and purifying selection on the <em>Drosophila</em> Y chromosome.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">2612</span>–<span class="cit-lpage">2623</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsu203%26rft_id%253Dinfo%253Apmid%252F24974375%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjEwOiIzMS8xMC8yNjEyIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMC8yMTY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2165.75" data-doi="10.1093/molbev/msq249"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stoletzki</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eyre-Walker</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Estimation of the neutrality index.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">63</span>–<span class="cit-lpage">70</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsq249%26rft_id%253Dinfo%253Apmid%252F20837603%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjc6IjI4LzEvNjMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzEwLzIxNjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2165.76"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tajima</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Statistical method for testing the neutral mutation hypothesis by DNA polymorphism.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">123</span>: <span class="cit-fpage">585</span>–<span class="cit-lpage">595</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DTajima%26rft.auinit1%253DF.%26rft.volume%253D123%26rft.issue%253D3%26rft.spage%253D585%26rft.epage%253D595%26rft.atitle%253DStatistical%2Bmethod%2Bfor%2Btesting%2Bthe%2Bneutral%2Bmutation%2Bhypothesis%2Bby%2BDNA%2Bpolymorphism.%26rft_id%253Dinfo%253Apmid%252F2513255%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTIzLzMvNTg1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMC8yMTY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2165.77" data-doi="10.1016/0040-5809(75)90020-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Watterson</span>  <span class="cit-name-given-names">G. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1975</span> <span class="cit-article-title">On the number of segregating sites in genetical models without recombination.</span> <abbr class="cit-jnl-abbrev">Theor. Popul. Biol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">256</span>–<span class="cit-lpage">276</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheoretical%2Bpopulation%2Bbiology%26rft.stitle%253DTheor%2BPopul%2BBiol%26rft.aulast%253DWatterson%26rft.auinit1%253DG.%2BA.%26rft.volume%253D7%26rft.issue%253D2%26rft.spage%253D256%26rft.epage%253D276%26rft.atitle%253DOn%2Bthe%2Bnumber%2Bof%2Bsegregating%2Bsites%2Bin%2Bgenetical%2Bmodels%2Bwithout%2Brecombination.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0040-5809%252875%252990020-9%26rft_id%253Dinfo%253Apmid%252F1145509%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0040-5809(75)90020-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1145509&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2165.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1975W295600011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference  in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2165.78"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weinreich</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rand</span>  <span class="cit-name-given-names">D. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Contrasting patterns of nonneutral evolution in proteins encoded in nuclear and mitochondrial genomes.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">156</span>: <span class="cit-fpage">385</span>–<span class="cit-lpage">399</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DWeinreich%26rft.auinit1%253DD.%2BM.%26rft.volume%253D156%26rft.issue%253D1%26rft.spage%253D385%26rft.epage%253D399%26rft.atitle%253DContrasting%2BPatterns%2Bof%2BNonneutral%2BEvolution%2Bin%2BProteins%2BEncoded%2Bin%2BNuclear%2Band%2BMitochondrial%2BGenomes%26rft_id%253Dinfo%253Apmid%252F10978302%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU2LzEvMzg1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMC8yMTY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference  in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2165.79" data-doi="10.1111/j.1365-294X.2008.03982.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">White</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wolff</span>  <span class="cit-name-given-names">J. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pierson</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gemmell</span>  <span class="cit-name-given-names">N. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Revealing the hidden complexities of mtDNA inheritance.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">4925</span>–<span class="cit-lpage">4942</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DWhite%26rft.auinit1%253DD.%2BJ.%26rft.volume%253D17%26rft.issue%253D23%26rft.spage%253D4925%26rft.epage%253D4942%26rft.atitle%253DRevealing%2Bthe%2Bhidden%2Bcomplexities%2Bof%2BmtDNA%2Binheritance.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2008.03982.x%26rft_id%253Dinfo%253Apmid%252F19120984%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2008.03982.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19120984&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2165.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261104000001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference  in text" id="ref-80">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2165.80"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wolstenholme</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clary</span>  <span class="cit-name-given-names">D. O.</span></span></li></ol><cite>, <span class="cit-pub-date">1985</span> <span class="cit-article-title">Sequence evolution of <em>Drosophila</em> mitochondrial DNA.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">725</span>–<span class="cit-lpage">744</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DWolstenholme%26rft.auinit1%253DD.%2BR.%26rft.volume%253D109%26rft.issue%253D4%26rft.spage%253D725%26rft.epage%253D744%26rft.atitle%253DSEQUENCE%2BEVOLUTION%2BOF%2BDROSOPHILA%2BMITOCHONDRIAL%2BDNA%26rft_id%253Dinfo%253Apmid%252F3921427%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTA5LzQvNzI1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMC8yMTY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference  in text" id="ref-81">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2165.81" data-doi="10.1111/j.1469-1809.1955.tb01348.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Woolf</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1955</span> <span class="cit-article-title">On estimating the relation between blood group and disease.</span> <abbr class="cit-jnl-abbrev">Ann. Hum. Genet.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">251</span>–<span class="cit-lpage">253</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnals%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAnn%2BHum%2BGenet%26rft.aulast%253DWoolf%26rft.auinit1%253DB.%26rft.volume%253D19%26rft.issue%253D4%26rft.spage%253D251%26rft.epage%253D253%26rft.atitle%253DOn%2Bestimating%2Bthe%2Brelation%2Bbetween%2Bblood%2Bgroup%2Band%2Bdisease.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-1809.1955.tb01348.x%26rft_id%253Dinfo%253Apmid%252F14388528%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-1809.1955.tb01348.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14388528&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2165.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-82-1" title="View reference  in text" id="ref-82">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2165.82" data-doi="10.1146/annurev.ecolsys.39.110707.173342"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">S. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andolfatto</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The impact of natural selection on the genome: emerging patterns in <em>Drosophila</em> and <em>Arabidopsis</em>.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Ecol. Evol. Syst.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">193</span>–<span class="cit-lpage">213</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRev.%2BEcol.%2BEvol.%2BSyst.%26rft.volume%253D39%26rft.spage%253D193%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.ecolsys.39.110707.173342%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.ecolsys.39.110707.173342&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/5/10/2165.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/10/2155" title="A Genomic Selection Index Applied to Simulated and Real Data" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/10/2177" title="Identification and Correction of Sample Mix-Ups in Expression Genetic Data: A Case Study" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/5/10"><span>Volume 5 Issue 10, October 2015</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/5/10.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/5/10.cover-source.jpg?itok=JwgNhVLw" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 5 (10)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/10/2165" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-E-AdT2Ix7XTNxfOlEnkDmEIciaJ67r8xipFqHbcmvGw" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/5/10/2165" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/5/10/2165" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/5/10/2165" class="active" data-icon-position="" data-hide-link-title="0">Similar Efficacies of Selection Shape Mitochondrial and Nuclear Genes in Both Drosophila melanogaster and Homo sapiens</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460046" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form--WRCwMocf-u8PLIgMZYc-34H2dxIU9axHHrpMTNtVVI" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/10/2165.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/5/10/2165" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-wlJgRHW95g_7ewKmiYSmT3_E2fOrkp_QvDfutpvGJJ0" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/5/10/2165" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/10/2165.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460046" id="node460046--2" data-pisa="ggg;5/10/2165" data-pisa-master="ggg;g3.114.016493" data-apath="/ggg/5/10/2165.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Similar Efficacies of Selection Shape Mitochondrial and Nuclear Genes in Both <em>Drosophila melanogaster</em> and <em>Homo sapiens</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Brandon S. Cooper</span>, <span class="highwire-citation-author" data-delta="1">Chad R. Burrus</span>, <span class="highwire-citation-author" data-delta="2">Chao Ji</span>, <span class="highwire-citation-author" data-delta="3">Matthew W. Hahn</span> and <span class="highwire-citation-author" data-delta="4">Kristi L. Montooth</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2165-2176;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.016493 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460046--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Brandon S. Cooper </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Brandon%2BS.%2BCooper%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cooper%20BS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABrandon%2BS.%2BCooper%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Chad R. Burrus </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>School of Informatics and Computing, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chad%2BR.%2BBurrus%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Burrus%20CR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChad%2BR.%2BBurrus%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Chao Ji </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>School of Informatics and Computing, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chao%2BJi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ji%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChao%2BJi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Matthew W. Hahn </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Indiana University, Bloomington, Indiana 47405</div><div class='author-affiliation'><span class='nlm-sup'>†</span>School of Informatics and Computing, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthew%2BW.%2BHahn%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hahn%20MW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMatthew%2BW.%2BHahn%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Kristi L. Montooth </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kristi%2BL.%2BMontooth%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montooth%20KL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKristi%2BL.%2BMontooth%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:kmontooth2@unl.edu" class="" data-icon-position="" data-hide-link-title="0">kmontooth2@unl.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/5/10/2165" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/5/10/2165&amp;title=Similar%20Efficacies%20of%20Selection%20Shape%20Mitochondrial%20and%20Nuclear%20Genes%20in%20Both%20Drosophila%20melanogaster%20and%20Homo%20sapiens" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/5/10/2165&amp;title=Similar%20Efficacies%20of%20Selection%20Shape%20Mitochondrial%20and%20Nuclear%20Genes%20in%20Both%20Drosophila%20melanogaster%20and%20Homo%20sapiens" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/5/10/2165&amp;title=Similar%20Efficacies%20of%20Selection%20Shape%20Mitochondrial%20and%20Nuclear%20Genes%20in%20Both%20Drosophila%20melanogaster%20and%20Homo%20sapiens" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/5/10/2165&amp;text=Similar%20Efficacies%20of%20Selection%20Shape%20Mitochondrial%20and%20Nuclear%20Genes%20in%20Both%20Drosophila%20melanogaster%20and%20Homo%20sapiens" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/5/10/2165&amp;title=Similar%20Efficacies%20of%20Selection%20Shape%20Mitochondrial%20and%20Nuclear%20Genes%20in%20Both%20Drosophila%20melanogaster%20and%20Homo%20sapiens" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/5/10/2165&amp;t=Similar%20Efficacies%20of%20Selection%20Shape%20Mitochondrial%20and%20Nuclear%20Genes%20in%20Both%20Drosophila%20melanogaster%20and%20Homo%20sapiens" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/5/10/2165&amp;title=Similar%20Efficacies%20of%20Selection%20Shape%20Mitochondrial%20and%20Nuclear%20Genes%20in%20Both%20Drosophila%20melanogaster%20and%20Homo%20sapiens" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/5/10/2165&amp;title=Similar%20Efficacies%20of%20Selection%20Shape%20Mitochondrial%20and%20Nuclear%20Genes%20in%20Both%20Drosophila%20melanogaster%20and%20Homo%20sapiens" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460046/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460046" id="node460046--3" data-pisa="ggg;5/10/2165" data-pisa-master="ggg;g3.114.016493" data-apath="/ggg/5/10/2165.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Similar Efficacies of Selection Shape Mitochondrial and Nuclear Genes in Both <em>Drosophila melanogaster</em> and <em>Homo sapiens</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Brandon S. Cooper</span>, <span class="highwire-citation-author" data-delta="1">Chad R. Burrus</span>, <span class="highwire-citation-author" data-delta="2">Chao Ji</span>, <span class="highwire-citation-author" data-delta="3">Matthew W. Hahn</span> and <span class="highwire-citation-author" data-delta="4">Kristi L. Montooth</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2165-2176;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.016493 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460046--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Brandon S. Cooper </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Brandon%2BS.%2BCooper%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cooper%20BS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABrandon%2BS.%2BCooper%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Chad R. Burrus </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>School of Informatics and Computing, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chad%2BR.%2BBurrus%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Burrus%20CR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChad%2BR.%2BBurrus%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Chao Ji </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>School of Informatics and Computing, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chao%2BJi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ji%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChao%2BJi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Matthew W. Hahn </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Indiana University, Bloomington, Indiana 47405</div><div class='author-affiliation'><span class='nlm-sup'>†</span>School of Informatics and Computing, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthew%2BW.%2BHahn%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hahn%20MW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMatthew%2BW.%2BHahn%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Kristi L. Montooth </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kristi%2BL.%2BMontooth%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montooth%20KL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKristi%2BL.%2BMontooth%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:kmontooth2@unl.edu" class="" data-icon-position="" data-hide-link-title="0">kmontooth2@unl.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2165&amp;rft.epage=2176&amp;rft.atitle=Similar%20Efficacies%20of%20Selection%20Shape%20Mitochondrial%20and%20Nuclear%20Genes%20in%20Both%20Drosophila%20melanogaster%20and%20Homo%20sapiens&amp;rft.volume=5&amp;rft.issue=10&amp;rft.date=2015-10-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Cooper%2C+Brandon+S.&amp;rft.au=Burrus%2C+Chad+R.&amp;rft.au=Ji%2C+Chao&amp;rft.au=Hahn%2C+Matthew+W.&amp;rft.au=Montooth%2C+Kristi+L."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460046/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460046/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460046/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460046/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460046/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460046/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460046/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460046/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460046/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460046/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460046/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460046/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/5/10/2165" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-7" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-14" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/5/10/2165.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/5/10/2165.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/5/10/2165.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/5/10/2165.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
