<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/5/10/2113.full.txt" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/5/10/2113.full.pdf" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/5/10/2113/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/5/10/2113.atom" />
<meta name="HW.pisa" content="ggg;5/10/2113" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Genomic-Enabled Prediction of Ordinal Data with Bayesian Logistic Ordinal Regression" />
<meta name="DC.Identifier" content="10.1534/g3.115.021154" />
<meta name="DC.Date" content="2015-10-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2015 Montesinos-López et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Most genomic-enabled prediction models developed so far assume that the response variable is continuous and normally distributed. The exception is the probit model, developed for ordered categorical phenotypes. In statistical applications, because of the easy implementation of the Bayesian probit ordinal regression (BPOR) model, Bayesian logistic ordinal regression (BLOR) is implemented rarely in the context of genomic-enabled prediction [sample size ( n ) is much smaller than the number of parameters ( p )]. For this reason, in this paper we propose a BLOR model using the Pólya-Gamma data augmentation approach that produces a Gibbs sampler with similar full conditional distributions of the BPOR model and with the advantage that the BPOR model is a particular case of the BLOR model. We evaluated the proposed model by using simulation and two real data sets. Results indicate that our BLOR model is a good alternative for analyzing ordinal data in the context of genomic-enabled prediction with the probit or logit link." />
<meta name="DC.Contributor" content="Osval A. Montesinos-López" />
<meta name="DC.Contributor" content="Abelardo Montesinos-López" />
<meta name="DC.Contributor" content="José Crossa" />
<meta name="DC.Contributor" content="Juan Burgueño" />
<meta name="DC.Contributor" content="Kent Eskridge" />
<meta name="article:published_time" content="2015-10-01" />
<meta name="article:section" content="Genomic Selection" />
<meta name="citation_title" content="Genomic-Enabled Prediction of Ordinal Data with Bayesian Logistic Ordinal Regression" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Most genomic-enabled prediction models developed so far assume that the response variable is continuous and normally distributed. The exception is the probit model, developed for ordered categorical phenotypes. In statistical applications, because of the easy implementation of the Bayesian probit ordinal regression (BPOR) model, Bayesian logistic ordinal regression (BLOR) is implemented rarely in the context of genomic-enabled prediction [sample size (&lt;i&gt;n&lt;/i&gt;) is much smaller than the number of parameters (&lt;i&gt;p&lt;/i&gt;)]. For this reason, in this paper we propose a BLOR model using the Pólya-Gamma data augmentation approach that produces a Gibbs sampler with similar full conditional distributions of the BPOR model and with the advantage that the BPOR model is a particular case of the BLOR model. We evaluated the proposed model by using simulation and two real data sets. Results indicate that our BLOR model is a good alternative for analyzing ordinal data in the context of genomic-enabled prediction with the probit or logit link.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2015/10/01" />
<meta name="citation_mjid" content="ggg;5/10/2113" />
<meta name="citation_id" content="5/10/2113" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/5/10/2113" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/5/10/2113.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/5/10/2113.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/5/10/2113.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.115.021154" />
<meta name="citation_pmid" content="26290569" />
<meta name="citation_volume" content="5" />
<meta name="citation_issue" content="10" />
<meta name="citation_num_pages" content="14" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genomic Selection" />
<meta name="citation_firstpage" content="2113" />
<meta name="citation_lastpage" content="2126" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Osval A. Montesinos-López" />
<meta name="citation_author_institution" content="Facultad de Telemática, Universidad de Colima, C.P. 28040 Colima, Colima, México" />
<meta name="citation_author" content="Abelardo Montesinos-López" />
<meta name="citation_author_institution" content="Departamento de Estadística, Centro de Investigación en Matemáticas (CIMAT), 36240 Guanajuato, México" />
<meta name="citation_author" content="José Crossa" />
<meta name="citation_author_institution" content="Biometrics and Statistics Unit, International Maize and Wheat Improvement Center (CIMMYT), 06600 México, Distrito Federal, México" />
<meta name="citation_author_email" content="j.crossa@cgiar.org" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-9429-5855" />
<meta name="citation_author" content="Juan Burgueño" />
<meta name="citation_author_institution" content="Biometrics and Statistics Unit, International Maize and Wheat Improvement Center (CIMMYT), 06600 México, Distrito Federal, México" />
<meta name="citation_author" content="Kent Eskridge" />
<meta name="citation_author_institution" content="University of Nebraska, Statistics Department, Lincoln, Nebraska 68583-0963" />
<meta name="citation_reference" content="citation_journal_title=J. Am. Stat. Assoc.;citation_author=J. H. Albert;citation_author=S. Chib;citation_title=Bayesian analysis of binary and polychotomous response data.;citation_pages=669-679;citation_volume=88;citation_year=1993;citation_doi=10.2307/2290350" />
<meta name="citation_reference" content="citation_journal_title=J. Econ. Lit.;citation_author=T. Amemiya;citation_title=Qualitative response models: a survey.;citation_pages=1483-1536;citation_volume=19;citation_year=1981" />
<meta name="citation_reference" content="citation_title=Latent Variable Models and Factor Analysis;citation_author=D. J. Bartholomew;citation_author=M. Knott;citation_year=1999" />
<meta name="citation_reference" content="citation_journal_title=Biometrics;citation_journal_abbrev=Biometrics;citation_author=M. S. Bartlett;citation_title=The use of transformations.;citation_pages=39-52;citation_volume=3;citation_year=1947;citation_issue=1;citation_pmid=20240416;citation_doi=10.2307/3001536" />
<meta name="citation_reference" content="citation_journal_title=Mon. Weather Rev.;citation_author=G. W. Brier;citation_title=Verification of forecasts expressed in terms of probability.;citation_pages=1-3;citation_volume=78;citation_year=1950;citation_doi=10.1175/1520-0493(1950)078&lt;0001:VOFEIT&gt;2.0.CO;2" />
<meta name="citation_reference" content="citation_journal_title=J. Educ. Behav. Stat.;citation_author=G. Camilli;citation_title=Teacher’s corner: origin of the scaling constant d= 1.7 in item response theory.;citation_pages=293-295;citation_volume=19;citation_year=1994" />
<meta name="citation_reference" content="citation_journal_title=J. Crop Improv.;citation_author=J. Crossa;citation_author=P. Pérez-Rodríguez;citation_author=G. de los Campos;citation_author=G. Mahuku;citation_author=S. Dreisigacker;citation_title=Genomic selection and prediction in plant breeding.;citation_pages=239-261;citation_volume=25;citation_year=2011;citation_doi=10.1080/15427528.2011.558767" />
<meta name="citation_reference" content="citation_journal_title=J. Am. Stat. Assoc.;citation_author=A. E. Gelfand;citation_author=A. F. Smith;citation_title=Sampling-based approaches to calculating marginal densities.;citation_pages=398-409;citation_volume=85;citation_year=1990;citation_doi=10.2307/2289776" />
<meta name="citation_reference" content="citation_title=Bayesian Data Analysis;citation_author=A. Gelman;citation_author=J. B. Carlin;citation_author=H. S. Stern;citation_author=D. B. Rubin;citation_year=2003" />
<meta name="citation_reference" content="citation_journal_title=Stat. Sci.;citation_author=C. J. Geyer;citation_title=Practical Markov Chain Monte Carlo.;citation_pages=473-483;citation_volume=7;citation_year=1992;citation_doi=10.1214/ss/1177011137" />
<meta name="citation_reference" content="citation_journal_title=Journal of Animal Science;citation_journal_abbrev=J ANIM SCI;citation_author=D. Gianola;citation_title=A Method of Sire Evaluation for Dichotomies;citation_pages=1266-1271;citation_volume=51;citation_year=1980;citation_issue=6;citation_pmid=7204270" />
<meta name="citation_reference" content="citation_journal_title=Journal of Animal Science;citation_journal_abbrev=J ANIM SCI;citation_author=D. Gianola;citation_title=Theory and Analysis of Threshold Characters;citation_pages=1079-1096;citation_volume=54;citation_year=1982;citation_issue=5;citation_doi=10.2527/jas1982.5451079x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. Gianola;citation_title=Priors in whole-genome regression: the Bayesian alphabet returns.;citation_volume=194;citation_year=2013;citation_pmid=23636739;citation_doi=10.1534/genetics.113.151753" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=D. Gianola;citation_author=J. L. Foulley;citation_title=Sire evaluation for ordered categorical data with a threshold model.;citation_pages=1-23;citation_volume=15;citation_year=1983;citation_pmid=22879102;citation_doi=10.1186/1297-9686-15-1-1" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=J. M. González-Camacho;citation_author=G. de los Campos;citation_author=P. Pérez-Rodríguez;citation_author=D. Gianola;citation_author=J. E. Cairns;citation_title=Genome-enabled prediction of genetic values using radial basis function neural networks.;citation_pages=759-771;citation_volume=125;citation_year=2012;citation_issue=4;citation_pmid=22566067;citation_doi=10.1007/s00122-012-1868-9" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=O. González-Recio;citation_author=S. Forni;citation_title=Genome-wide prediction of discrete traits using Bayesian regressions and machine learning.;citation_pages=7-7;citation_volume=43;citation_year=2011;citation_pmid=21329522;citation_doi=10.1186/1297-9686-43-7" />
<meta name="citation_reference" content="citation_journal_title=Technometrics;citation_author=A. E. Hoerl;citation_author=R. W. Kennard;citation_title=Ridge regression: biased estimation for nonorthogonal problems.;citation_pages=55-67;citation_volume=12;citation_year=1970;citation_doi=10.2307/1267351" />
<meta name="citation_reference" content="citation_journal_title=Methods Ecol. Evol.;citation_author=W. A. Link;citation_author=M. J. Eaton;citation_title=On thinning of chains in MCMC.;citation_pages=112-115;citation_volume=3;citation_year=2012;citation_doi=10.1111/j.2041-210X.2011.00131.x" />
<meta name="citation_reference" content="citation_journal_title=Am. Stat.;citation_author=S. N. MacEachern;citation_author=L. M. Berliner;citation_title=Subsampling the Gibbs sampler.;citation_pages=188-190;citation_volume=48;citation_year=1994" />
<meta name="citation_reference" content="citation_title=G3 (Bethesda);citation_author=O. A. Montesinos-López;citation_author=A. Montesinos-López;citation_author=P. Pérez-Rodríguez;citation_author=G. de los Campos;citation_author=K. M. Eskridge;citation_author=J. Crossa;citation_title=Threshold models for genome-enabled prediction of ordinal categorical traits in plant breeding.;citation_volume=5;citation_year=2015;citation_pmid=25538102;citation_doi=10.1534/g3.114.016188" />
<meta name="citation_reference" content="citation_journal_title=J. R. Stat. Soc. A;citation_author=J. A. Nelder;citation_author=R. W. M. Wedderburn;citation_title=Generalized linear models.;citation_pages=370-384;citation_volume=135;citation_year=1972;citation_doi=10.2307/2344614" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=P. Perez-Rodriguez;citation_author=G. de los Campos;citation_title=Genome-wide regression &amp; prediction with the BGLR statistical package.;citation_volume=198;citation_year=2014;citation_pmid=25009151;citation_doi=10.1534/genetics.114.164442" />
<meta name="citation_reference" content="citation_title=Plant Genome;citation_author=J. Poland;citation_author=J. Endelman;citation_author=J. Dawson;citation_author=J. Rutkoski;citation_author=S. Wu;citation_title=Genomic selection in wheat breeding using genotyping-by-sequencing.;citation_pages=103-113;citation_volume=5;citation_year=2012;citation_doi=10.3835/plantgenome2012.06.0006" />
<meta name="citation_reference" content="citation_title=J. Am. Stat. Assoc.;citation_author=N. G. Polson;citation_author=J. G. Scott;citation_author=J. Windle;citation_title=Bayesian inference for logistic models using Pólya-Gamma latent variables.;citation_pages=1339-1349;citation_volume=108;citation_year=2013;citation_issue=504;citation_doi=10.1080/01621459.2013.829001" />
<meta name="citation_reference" content="citation_journal_title=Psychometrika;citation_author=V. Savalei;citation_title=Logistic approximation to the normal: the KL rationale.;citation_pages=763-767;citation_volume=71;citation_year=2006;citation_doi=10.1007/s11336-004-1237-y" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=D. A. Sorensen;citation_author=S. Andersen;citation_author=D. Gianola;citation_author=I. Korsgaard;citation_title=Bayesian inference in threshold models using Gibbs sampling.;citation_pages=229-249;citation_volume=27;citation_year=1995;citation_doi=10.1051/gse:19950303" />
<meta name="citation_reference" content="citation_title=Generalized Linear Mixed Models: Modern Concepts, Methods and Applications;citation_author=W. W. Stroup;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=Agron. J.;citation_author=W. W. Stroup;citation_title=Rethinking the analysis of non-normal data in plant and soil science.;citation_pages=811-827;citation_volume=107;citation_year=2015;citation_doi=10.2134/agronj2013.0342" />
<meta name="citation_reference" content="citation_journal_title=J. Anim. Sci.;citation_author=B. Villanueva;citation_author=J. Fernandez;citation_author=L. A. Garcia-Cortes;citation_author=L. Varona;citation_author=H. D. Daetwyler;citation_title=Accuracy of genome-wide evaluation for disease resistance in aquaculture breeding programs.;citation_volume=89;citation_year=2011;citation_pmid=21742941;citation_doi=10.2527/jas.2010-3814" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=C. L. Wang;citation_author=X. D. Ding;citation_author=J. Y. Wang;citation_author=J. F. Liu;citation_author=W. X. Fu;citation_title=Bayesian methods for estimating GEBVs of threshold traits.;citation_pages=213-219;citation_volume=110;citation_year=2012;citation_pmid=23149458" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=W. Yang;citation_author=R. J. Tempelman;citation_title=A Bayesian antedependence model for whole genome prediction.;citation_volume=190;citation_year=2012;citation_pmid=22135352;citation_doi=10.1534/genetics.111.131540" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Genomic-Enabled Prediction of Ordinal Data with Bayesian Logistic Ordinal Regression" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10.cover-source.jpg" />
<meta name="twitter:description" content="Most genomic-enabled prediction models developed so far assume that the response variable is continuous and normally distributed. The exception is the probit model, developed for ordered categorical phenotypes. In statistical applications, because of the easy implementation of the Bayesian probit ordinal regression (BPOR) model, Bayesian logistic ordinal regression (BLOR) is implemented rarely in the context of genomic-enabled prediction [sample size ( n ) is much smaller than the number of parameters ( p )]. For this reason, in this paper we propose a BLOR model using the Pólya-Gamma data augmentation approach that produces a Gibbs sampler with similar full conditional distributions of the BPOR model and with the advantage that the BPOR model is a particular case of the BLOR model. We evaluated the proposed model by using simulation and two real data sets. Results indicate that our BLOR model is a good alternative for analyzing ordinal data in the context of genomic-enabled prediction with the probit or logit link." />
<meta name="og-title" property="og:title" content="Genomic-Enabled Prediction of Ordinal Data with Bayesian Logistic Ordinal Regression" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/5/10/2113" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Most genomic-enabled prediction models developed so far assume that the response variable is continuous and normally distributed. The exception is the probit model, developed for ordered categorical phenotypes. In statistical applications, because of the easy implementation of the Bayesian probit ordinal regression (BPOR) model, Bayesian logistic ordinal regression (BLOR) is implemented rarely in the context of genomic-enabled prediction [sample size ( n ) is much smaller than the number of parameters ( p )]. For this reason, in this paper we propose a BLOR model using the Pólya-Gamma data augmentation approach that produces a Gibbs sampler with similar full conditional distributions of the BPOR model and with the advantage that the BPOR model is a particular case of the BLOR model. We evaluated the proposed model by using simulation and two real data sets. Results indicate that our BLOR model is a good alternative for analyzing ordinal data in the context of genomic-enabled prediction with the probit or logit link." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10.cover-source.jpg" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.g3journal.org/content/5/10/2113" />
    <title>Genomic-Enabled Prediction of Ordinal Data with Bayesian Logistic Ordinal Regression | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"A793NLUQDjf-LwOFiOftB2NZhcPERahUFTFGEeqwr4Y","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460039","apath":"\/ggg\/5\/10\/2113.atom","pisa":"ggg;5\/10\/2113","ac":{"\/ggg\/5\/10\/2113.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/5\/10\/2113.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;5\/10\/2113"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;5\/10\/2113"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/5\/10\/2113":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022n+NlaTCqazZU3crs4zPUCPmx+20JVrxSe8WU03t3dRBplF\\\\\\\/LLoiNyE5dMCfoeAXkuZhNrwz5TNvUxM\\\\\\\/CNb9H4TjDX8y8mygS0J7YZWBc8UaXa91OEZPHwj+bJV1jVcyiTP\\\\\\\/2U9ulRR3h8mlSmdy9obPqIvGdWrN0G0tjQAhFCSlDaTs7Z1skemhI9ARIc4y1qYsC878KZdqLns0x5HNmhz1PDtiGXxOv6LGykmfYE4zxfkbOl8nfGt4egQBQyFLyqlVHl\\\\\\\/hwMXdNV+EudodxRj1My1R9bOt+L9ztKTI2x1xcMyCyAGk\\\\\\\/NSTB6mzrS3EEKNlHitwAfavJ7S9KuAE5OSatQEPeufya9ZcnoRTSbu1OfVjzQ6nJyuWjq3ogFkDompGb0JYC\\\\\\\/r1R+mi3m4+LnDk4ido28KtW9x44zlI9iNVnny6y2NqAsxixXZ6EVxilB9Xjbk9bvGUu5Ab0KB8n0ePU+sW50ZrfkK\\\\\\\/AXE9VpWkj0E3ia7TN69O78gFva8JbcZ2ZtChNvseFjEvDi0frsEeIvfIC2u77TTZC1kApHgWKbAZMam0DImSpKA7hnzPWGGcE9tO2bALb7mbAYA\\\\\\\/3VTrrwKU+aZ\\\\\\\/tyHJGb3EhpBh7A+1mRtHubp3iM2xH7wwhpATt5yvhTrivVSNfOcxiu+J7oZRmyvykF\\\\\\\/55AQOvkqY8eX37bY9jE9ryZ+SqSemo2CLQLTzJ20QfLkhjCXSMOiU8vawcNCvg3XaQNobltVl\\\\\\\/8g6R5AAMYPxREIPyHm1eHKiNwXco5Q1XpTQg3OwN4I3r\\\\\\\/o\\\\\\\/enuweGgHvvNOZ5vpfaxUhiKDY3LxZVG7BB\\\\\\\/0UZVdEHhHkiL\\\\\\\/0i+bbLr1mm4DG9LsoCpyvLZzKXG05\\\\\\\/9vNLfgKN1qzEDSb9oGLPvpEejOJWvggD4Z9szVAOtCvXL8jjJCj76NbrR\\\\\\\/jzVf2cEk260aVpAdSCF\\\\\\\/Q04RGtU2tcUFHfUWfrTVzTTeIXX4eUmvvDxmckvtdxELLUoD9l6SctNJGXmTan4nuXwfBgjrrm92DDskewV43ZxngtsDegHdfCx2\\\\\\\/JW1cT5tGeLX2aiNkuOY9HkVsDKooVQwcJRy5OujYjFG3j4YBq48fRp0IbUakFdB\\\\\\\/lCjr\\\\\\\/PETylTHZGPlFLxHIY4RRhEGvywVOKcY+YEuts5DkkvgIxXzQigPZ4DVKUhKdTtsPEWyMDRvPS1L\\\\\\\/FkKu6ENsTDqeClM8qCxWwyD8vyMmOmaSiIM\\\\\\\/dwc5Adf5D7RdE7kYWBTOTMeNrD1ZchfQ2Zu2lJt\\\\\\\/cN84iU8\\\u0022,\\\u0022iv\\\u0022:\\\u0022vfR2TFhKR7lLrHG2PBGgWQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002260cf1443178eb2c6f678c906bfe54707d24b3679f7e4049018bcf7176ac2aa8b\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022eQQbVPK57o8npgfy9AcdDBEjiL+pXqRe+TopxpopzfyZxo\\\\\\\/TgGhxrq4fgXnt1sDi8Ewg6VSnqHTimM0i2jciBMglynLoYslFeIEjGmyPOAmKzqTeuoxCtIGfkAqXDudVhdm6sMwH5f0VidouUBn1MyIQ1V5lZ\\\\\\\/MWtuTpURWbpDY23OQSjws7lyzorFm49il+BVEERBn8KG8LnHZpcqI1wT3fh9qkT8VeFzDTlBpVEoCmNRuoIkjSKvFWA+1sffwaDSDB4bF8A7wFRqRLHr1qzyUowFQwBnzHVq2Yvp1ZLMXm1kLg+4tMNLN483YLpP6uFisRquSJMmYGm7SXmc75sBa\\\\\\\/ovsRivER7D0CrMT284c00eUp4zjEMxjI9OwIUZyGmLzPGCMO4BrEaCZYmdu\\\\\\\/QJBvCeEFSHddnq5smkggCt1DRQJEeBcnyUMy\\\\\\\/xhcrKGkh22HLMMOPs9EHpDObJwp8P6tYaFo5ilY2HxzJ1QTZa2l5wdu4+BiBSyCCoFpWEoWQgi8p4tyaIQterpKiK+y9F+ElK7BPZZQp69jWclaSQVTkyGCNQWs6JwntalHZ4SQL7flU\\\\\\\/zvcE0pZaX\\\\\\\/N5aqikaPhaccUie+le7u7EEvfz5MrCDUAYcg\\\\\\\/cU5enVWX04YfiJJco\\\\\\\/7KxEfGkikDqGnooBAr6OJSLFReOySz7cGQm19Ptz4ubUc0BEGYE4qV+ctEtGFw4\\\\\\\/yP\\\\\\\/8wMkd9SgKvK8AjgEHWW31THHvwLfcG66y9spVgSCuBIoI+sCbroZXFDfsbAH+4UzJ8oAvZf3fyTjU1AznAcT6EeVnTlCkqIT2bAebyj9a+DURXb1Iwx3cmaNL84LSdGx1xw\\\\\\\/at2jf0Ubod1HviCa71sOd\\\\\\\/mlCmOktg93\\\\\\\/XCY8o5t05ZG+F6FPGjWmm00kLTKxMNx9NuYrXEz7kKFzEUEEnCOL3XnqKnDtsqObkD4L4VJEgyA5hCToHzLSQJ7WKidNQOGU9MyOBwI+av\\\\\\\/bFRKvpk2Cv\\\\\\\/tQMSI3hxd1VACQUfnAIJRD6Z9FlOqcc1Krw4eKrykODuG+H8lgV4oHBhQnod\\\\\\\/C9eGzEP\\\\\\\/4UVxCjQnKqx838ZPkLDQfzLXXLiGjaLb+CQHMNzqXjsHe166uwbAIvpiMQWlSzC\\\\\\\/CaiGUG3WvOi2ay72EfZIEG49lsXpRwci\\\\\\\/zXtzqRZbdDuhwbhgRT2hh\\\\\\\/WHmsArLngslDjG6vfKELTz\\\\\\\/e1AFbr88S3SjWtqUZa5+jjqhKZLlIw8fziMkmRxN9tF+xwCmOkgl3Hsc3SSniyLDKpJE02mgzMeuHUrx7YTGKUzLaWY\\\\\\\/fAjhK+Uyqn0dS3\\\\\\\/78J0B0FPZmQDZBAKfpy9PWn00HalULK8iAbSfynQfEbKwGERcT5kKyHe7C3eIF87td0Qw4gwtq5X6B36SfUkyaqUGiaO\\\\\\\/OsIwlkydbQQYXEaGxhWCpDqlPG4M0CfG3OyyKnVOM9o41KqoF\\\\\\\/\\\\\\\/ph3EVe7l1aSX0P7x\\\\\\\/jzI0rgznYSbZcZ5jJ1oesTALdemlPTQIjdSYTGGm+xHdsdSxNtSxY+LbrYnu8Rr4I9J41BlAkoJBosw5jEsjS9z6HjIcxpgq+DHYIw04C2SWvXXD6jvJiLOohraDsvRKx\\\\\\\/nydt4bGDnncy6iyuuvAQWV9gRa0ESVKcIABq7+ux9xjEjuhZpz9qP35\\\\\\\/z+OxTlRblBvfgMAMLVxOCQVeNiikUlJ+IvsPcmQvGbBm+FKpm4pzNhiN3AlPkeovobdpNAZBWYdM4DWw8SOpwf6qUpkgDygtF7Mz3c5XW+rtOnnQnKUl5g3W+ADjix4kM3+npTLKzEPoYgWkGJ9wBrOniYXWeeVHW63+Cd6CJU1\\\\\\\/gr4+S8jCWAB7IGnd4v3\\\\\\\/8dY\\\\\\\/Q9aNQHL2bh028VOqrbLDz0gcOVehnUzrbINtN8hIZ8Ek1HMb3Y\\\\\\\/wsd6P3LO3u8U2nocL3H3WimFu5PCypIq8Hfh7sW0EF3CCzmRExJC923w40J1e2NNDzmMr91haAM1RjyT85Sz4lVCgLwc9Xcmqnrz2Q4AriEHXkGMrwMOEjkQnc2hQ8oYqqDdV+RTynQpQ+JUYHh0Sok9SN0ZxN\\\\\\\/VS5G0ihmgxAiz9A1Z\\\\\\\/kRlhtt4z0nb2jlSMZD\\\\\\\/H4eGo0+LJntbtn4vFyylOhl8p8Eed1pvCU0oBopqizyOVdL8NZTStOH\\\\\\\/y8OgE+Kz4Vo0X1HpJ8CHwK9CdNpkL2D5BAs5qtwVERY52rsNKla\\\\\\\/lRK3qLET6+2gm+oflKRElZyaDFFWCAtyQQ29vzUNrUZ3lYmfMPOrZqBACywXcPipfdV5LUW10i19hvuht9eZzV81h2Vvt76WWopDUxVqlWZ\\\\\\\/cM\\\\\\\/7hyhBYM5pQ3FiMkMi2Kn9wx5\\\\\\\/l7\\\\\\\/0I5QQxz+931gkxnRzIWKpfYyuw1pFJf6sC3NXAmr6z8TjqFVQfHkpqCyn\\\\\\\/AqMkWuuRkpgksZTygceVC65UT4tBP0wydCIaIBoJXOoCBAfd46TFJCyQ\\\\\\\/4P5l9uB\\\\\\\/4CD1r+qWhZ2kAw55UjTLOhkbosWpSxdVGNpJxxkD2K6exCW5pBUVw8NDBGlNWrm+2aacq3qsfkt6DYG36Fc8CAe141GuHwg13VBSt5MdfvRiuw7hCO5t\\\\\\\/8mEEcFycafbOx5AxiY4vU70H76IFhe4GYdSIYiO4c3YFucDFWtPUkYZKGpR0p2NqR9fs9fWchflWw3n8ffOMvXJwulvcOsFMN4EjkthBSspWltrijScFK4c8gkVubrPg7ciEOs90H+Wm2Wz6jDicC6FqazSZ5tIhEUVW2EmCt4k8VYzhGGAK\\\\\\\/xqN8m5a85PO6DE9H3H3NlcijiXUlxgQbe5XOrP5T5FGZiFT2MS2itfau+ooFfbKYHO82PMQwpSsl00RXsrOmNhsaOVQaH1CndSZdsnHoZJct9W3eCvq76M3mEhxZ7I1gA3rSRY5rzqo3gBrHWFRlsNtQ23+RHIXpXqLKRV8PSIqGZYMn2XJWk+c\\\\\\\/ELfGFPMgyeiEguulGg0TDB5ro4v63xqwtbWZQ3Jm1n8gNpCmq83OlovAo2xERjrihkkvjhVEyqAie+DjT7+u3d4NTpQcP\\\\\\\/nukG8ePBDbZgW56yUnMJNg\\\\\\\/7y6P6EmHtMbsNLyKXyu0nSdoMTRwbNdFjKEIVwJ18eQNO2DfA8In\\\\\\\/vMtOKY0kDJk2GXY\\\\\\\/fmtFiTLOi2sTvfJCLMr8zExLG1t2DoE7T8oavUufA1pPHVIWh8K2wLNsnCait5T2KffDw+N2xxYm6SSsi4GmAY7ViYRROMbnLMCUWWALi\\\\\\\/kEvvK2W4PlTHVRCgMgJK9bmxX5rhIKi60wM0V3jpnpKHo9UXpIKpfHjwy+6gFyMr4FZ+7TMUMyIFVfyw6UhoV72A1ITGbgWVp9E9O+XggAaOcMriWV1qPDPafTfnkxuoyt8VKBIIDHQCgO\\\\\\\/b7d7cNdwqVGgDqgjX9R9UJsGpnmtmx9F7wxX0TPSF25zjXA6ELgFgun\\\\\\\/fon1E9T+gRCDa7Niidv08NQ00lvjPu684p62qogW4gCwLv9lwNZNYMk\\\\\\\/FGPlI6R2ebwe95FSOEyiBygLxHjKs2cx5xG\\\\\\\/oO1EdSzIrQzhilWyxOZ3RPW50zKLdWhWbQ+rbzGOKk\\\\\\\/7wQf4VbtoUeh95aBXiQYN5zrRWmizSPIro\\\\\\\/OVhR7SbWZ3SEGLrPPNk9fjs2\\\\\\\/O\\\\\\\/rueiR1ox5m2ecMe5twJAKFubBOuFDDqmVzxbaz1SlDgF19PiGvb3hNwnjuGLGSCyHxNLASw3lNHiIHh39UckTJvw4MMVaty+zdAShNQHPFJHKpQTIaD9UbcogzODUmRmcNZgE\\\\\\\/jG0xGOb8jCVo0o4p9lgkdXQK42Y1S6c1j28Pb0l+Q2+LAA+P\\\\\\\/aOG2bga0j2Uqwc2au6UsW57n\\\\\\\/yoo\\\\\\\/1aCrPPo0WsapaNV+lQKQdZdzkycuNay\\\\\\\/Fpb\\\\\\\/Yo1nrDS1DpDHwktBI\\\\\\\/IPJjFWwRMwo8SJ3FJ5TyKkvzhH18YZwWlVqz1rcf4gRSQJCtFpQTy1GC8Sa+deVvWdu6HPSqj61AQbpEBE8zkiVN85gdQWE\\\\\\\/oYOaxVAsGzyrXZ54vedF7bJIRHv1JSyY54AEBD47fQbm7xS6apVVNh3xZx9FqsKHiwGeKD7QJnHdkLBLpHcmggBt2gapSbU7PEbESZhBUk8t0zrjUpzJ3Po9qMrBovstCKq0l3gdjxn9TPS8KKskapyTXsPEappWb0NbCu2vM1ib0qlJFxOxgO8BFqtKa13ua99w4kmum4BqNflRINc4e08pJOsKkf3G3EPOVZQVw4IwCj8l7De9qFNXfq3yTjNildvVR2M5y3XvmQOM9f32oyTInLAFD7q9SzLrMtcI8uT8YLwd+vzpyq736X\\\\\\\/dBvHNXrxs68KjTl+oAv4MPshE2R8d9dXaKfOsLlvvOTVQWOvQ6JU0XtalL2OMjYGi\\\\\\\/byIRaF7+wv9KAdF02HLopWPdhD7iHfiY8oNg2xFjALIQDh+5NTiKCQ7Kmkn9SMOJAFkl3AWhZHh5XKQlNx+DM3h4TdGl43OXPquzeItfCX61Z2mWPNT5tWBCd9xSjq94xMkEqvO5KPBFnU+42MhM2eJJ\\\\\\\/JL66W+cNafUQWO10HhipiW8L4REmnxHiXPAPPMZvzpOGocNCtYGkZ8pEPbtEED5vXqu+qmkrAD7ycNMcZM0TjEysYgEiBHoukzjOyTL0i4RGKm9UK2lgaxYFfb3XJXhve+2pEvwhWu6fJeKH42i7PBhGC3fmcReKb5tHgk4D5p2t0NFl9xqANJ2nyJvyzRcE1Ro\\\\\\\/0nBOghxl9P3j1U0N\\\\\\\/dJH9VMEn9J8wE7\\\\\\\/tdh1H2q1W0e7VcIgKeRPBMgozK4RfY+N8\\\\\\\/hoTTpF0QGKRHoEEy3G8O6yyKygheJQRnCxLai9hKWLCRCDx74bK1vHBfZKenN0Ssyh5HeDleXyrEZveeMxS\\\\\\\/g6Vux+sMS8qTIj3c+ROoukF7zYYSJrLKIiAWKrSkshXE679lMPSSPvUml2fW\\\\\\\/e+Arl3e\\\\\\\/EUVDhisHdbE3iAGljQuc1RXBg4CzSZbtPbQDSCVuMuy3PEOtk6hOCc0Sa2i\\\\\\\/A3wClKhQ4mstt6N1+dinxxW2NR8uZa62xl2aVHyAvD5jvn5O3UNFtrUmpv7guKzNAJGt+pGqmkkveClH7N0EGdgjX8E350HQYp2B\\\\\\\/1z045qZlvvbNLHxzC0qNzxxQciqhjicrtz3e\\\\\\\/EIExXAvSfs8yRPd7e7hfftGizKLSqH40g+LWw24NxyYO13Xj4Zqo0y8mrgqWvTvcIEQqLRkzeQ2wSQaBxIzFLEldVXzBlWy\\\\\\\/D0VNa3n0EdeqRr9Orz6kZBHD\\\\\\\/oIaN\\\\\\\/S5u6DNeV4nMDz5bpo8J2CPT+V3bdNba2hksXMSzWCKoMNHmdYwjfId42wrexVIm+H0Wfk7mqeSVWD84sPhcwhiHZBFex+t9q76exJja6qzo+qFWBrIvJE0oB5Nt3HyN6ulQ46VMIba4\\\\\\\/ga70zxbmbJmSvjcPl1lW7LcmqXAjg\\\\\\\/K4eAeSuLe5wiY4uMRXqd6lUXDLwirzinsTloWnbDHvwUWRHoCfwZdBqgVs5qEA2ny\\\\\\\/8X8KbJaLtyJeoGX\\\\\\\/wXliNjAruKqRcZYlBXCSfcBFd4qWj98jlZdLk0DVmUPYj6FyQsH84zdKP88mrEhWy6HpRAt99nZ1uedoioiZozK\\\\\\\/qLuLeus4CZVtbY\\\\\\\/IT3Lh6YnzRir9YcgWIhhQqxAQR+SI+doz5CleOxvM1SZBNDhjg0wY8GVnQMSjnBQROm3496qBMUd0U\\\\\\\/0UHvym5RNJtEc5Q\\\\\\\/YpGMYeFE7EnboQQs\\\\\\\/oSwYEBEtWNgZmXjwjih3lDOniwbeK5f\\\\\\\/zuUkRGeoLJtJdGvDMyLXt0boCNb3XufozCLya0Umx3Nhb8S0SouVWYz28BjTUJd3tW20JX17VTRXvqZHOU9cRExQ829pDymEE\\\\\\\/VMjN1KZtlAQuRE2y+EiIAQB0ZQl+cohjXQTQRbG8S8tyXwvQjQbYn5mYmFjKjdlZ+GbvIYViPFkFN9JUg7lrHuxR4LXhrdDjJb4fU+QPIvntLyClWIzpFNuZwWnu+yXDCbiZa6sXB1fhrph7SMGob8eM1yIqQPw4GjQfGIoO4dz7HZ2Cr3I0Dz5iuhKx0BD4Epv2O\\\\\\\/3NcqPAl2yu2ySOlldDvBAPLhlXQzO3NpHfRhEh4J3rJ+oNDEGHO5L3PboQnIEPuEBLOQov6SbhylGeohSU045t9CZam+qchxGNEx293NXxhN0yvmsIDU3FY9GuKqBHXL6rT8HzuQ2PbhNcD9aRQxh4zE2z4JiYEr3DQZh28NfAa0emDwm2KSBqypeFicrZkbTNkIqz0TGNSceUKaON3YiVSXKLGZm6STBUYktVLhXlxuo57+\\\\\\\/Yt1QcHu71a\\\\\\\/HqQPmQ6LqAvwQe1Hvybsw7lW4hvypFeycJhidQz0oJxhRDICGGfvULqX8P0ywCre8ImEtcnGQ\\\\\\\/KRezCQuB1kuKzW3sqKaeYVA5fRQCWc5SIqew8SQ693FHvwcwiWV4TFztFzX56fOrRL+efnc9j6Fj5neTp8ZCzHUbBZPyW21H1c26mnlmW9axIqYsHtUTCvHavqsNvtcylD4Ry3j4BSrOqI8Au5Wz6Aqx8t6E9F8jS5Goz5KZB6phLMiyyzaAmZyWcUf8FslPvkTFXIJk8xcusaxlbfCnvcm8Pv+xo\\\\\\\/xZFPWCuxyaVF69jZvJpIssXyr3MyI3ewG\\\\\\\/ljz8mFHMv0DrLyS4qZhz\\\\\\\/PdJ51r2futdAF05k8Dz0hkYssRcOm7af0g5NUq4P9meZjD0U9Ycy9T6IrOCIykzRlbLj6elnU7752Q8CiuSrTUD\\\\\\\/ZRMtJr8HwGBPmduQiUYa5NhHKVpeZwMAsEcmo9D9uop4TeW1nrCW0uPAPEU2k2N4oAHyL1OtwIsbA1y+BTq9Ev0ZQGX8j5iKCPpNqe2xveH\\\\\\\/Nw8EIIGzvC1VTEbJmMkAE14yiP7sFe36gZjfKI0fY3k0ffZcY4TIcCqhc65P19DwwwojI5vX6HQ2VNLzAwOXhHkjwGoKzP3Xk\\\\\\\/BRqMpg+jtjjUAusXZRNfzuAlowI36NScSRkBNI9PTWbtDIyWDFF7Lm1kPgOik9AnNtsBte6+eXPhHxrD2BycYEPEsxdKemDRCSduN4X45SqScg\\\\\\\/x0ClfhX+BuAQMOgyMuS7VniJ0g3fUiC8fmdUb7sxKPBBauSWdmt3\\\\\\\/rQHjZzD4gZLIzTgqlTguRsdtJP9sCz7gKVxUkZERDyRs5vxJgWCxSen\\\\\\\/h\\\\\\\/XqV7MsetYhHhbeKpjLvFUQ2ErV3L3JmiDK3s\\\\\\\/drtW2LE\\\\\\\/15a9NlB7u\\\\\\\/u7sXK3ZI22FGg73gWqHoz8IDZLhQrXoJxNgLdINhBDN19Jq\\\\\\\/JxdHTdNgOdaBXGqbtNBkfnj1Pa\\\\\\\/r05GIvNAbeaBaIW6tDJd2v3+gBjkfPQqWZqXWQDYiuJHo1cN6hxgtGiL8+yQ0hioICAoDWmwA\\\\\\\/OEHrZSSYljQseoqqVh\\\\\\\/cyW+rNwNiJpLhmeCA+axRTZPbf9foiIZ3AAFRDWurE+HhWgKFFxy\\\\\\\/BNzxXZAM\\\\\\\/aFZW30D4bj7VWYTETApbDuylTdp0LG5nTSeCUJFs0LHNBUej8mA0N0pD7ulQVhAewtXm809fMziTU2olDYThY6QDDAPUEC+FhyCx4+XV0Czkp5T0dAM2rlW0BQ7AWYNQH9grKQ6JR\\\\\\\/Y0YcP02soEKvqJ5tWkmagaxdajQmtHPV2jBAz79MNXYQcUbZYGGt3M5LLJjhvDhfpc9BbE5XR9dOuR0lYfR7MF\\\\\\\/W\\\\\\\/eMqI8BF1CRLljcBVY2zSUhQiy1OY\\\\\\\/LXSXloCJ7BXK1X\\\\\\\/qD6TnBC6fBx0fWitEXT4Z8TN31chlEQcrarpJC0URTuHYivp80Uh6uG24nQJ2ONzezZLkFBqEOzDiTD+RvRYFPnZEi3xmIfTOWuyLOr+zpHbeyvxkXLEkglead5fgO8SrZhODIJTBZOv61L67ANnj4j05r800onM88OQXXpH1oktNuCvMLT0okBKBWXlXoCgTPCf1l0PgKeV0ZDvy+GQPOh9hXls8P\\\\\\\/P+Q6ZWxCbft3Tix8RgiDNFDvl23jkc\\\\\\\/fhGV3LgVfVJ67m02jvJBdUm6UzvWsuM\\\\\\\/W7n0qR6R2FgmHG5b8Fss1myAqy\\\\\\\/G09GKNJY3Rr1P9BL5M2AHksa3JzHXf3Y2sngb88pWmQ6pe+UC+1voepHHqVogrZss5qqKIOggkWrJhMu3EjIMtntFnjNVBbKVEIvU8KysQ6b2keEi09hMYQksY2ALL1qtSforCo4V+GaxADLdFonT4D4VO9nmRTeJXIKM9bh\\\\\\\/jAJLqedFCklq\\\\\\\/Nse2HnvAiX5CFjxtKZ4IuVBQvkvFJqaGS72fSGipLpXAQBea118Ro5I+2qUBPYIdCvwI0ewXaktgXzKvrBl7u3ksQc8ofCYNJIlPLoEPxc6SiFpbTi\\\\\\\/z+4EbI8Z2h0mhJ7LpC7GiG8tnXku8sRWBTcKMEcPKIvayA6XlN6miOlyOR2zn\\\\\\\/WVre\\\\\\\/fFvHuK4k65RhYM2QZThWMtwPKLoHkydZon4edY101KK5W2gK9t+i053WkMPq9Zszax2hAqjwuRr5B64kRPGtAifF0O2pT+W938vG\\\\\\\/l8+H0UapdiR5UxTGRCF6KH6HMfLVq27MKHBwQZ\\\\\\\/67AzYG+aGYGyDF99q9Dy04HDNQxRhWj60eoxqhPL3sgtixbyBbzdkQWyaQA6OsqaGGXTe03OsyJedennkBdZmHzOtNsR1mXK4c8Gu0McVbVlZfI+Yl5\\\\\\\/3Q2J6f6Y8uUS16Quj2TvHc46Hi\\\\\\\/DEnV5ECD8qlKxEn7VH9JEBsnsIvKlF\\\\\\\/7+PVLesVV5vP\\\\\\\/zbkW09dbBYeEXc9pzDpvZcRZff1By964vzYFaj+0MkQzOQb3axM+8AEdU3xqmkI6C7zSVKkJZ9+BlcE4nD5Zc\\\\\\\/3gC3\\\\\\\/6Z0xi\\\\\\\/c08nYJ7LStjiQ9ExiMzummsb87rQqF36p8FfPws6GcjHj1z7Quv3dFJiXEAqVsixubJ2Pbq3UX4bRrjT3bGTs0NxrpKnbzBs\\\\\\\/9ghZdJ0x3MT4djONGqPEzgt+MqAlEeVkeCiPZeFC+YnTLJ8eYvZiwpECRnVc+04dKCjaDGR9uVLOYcUDFUXWFxAfmT1436NExhpkT62eWgnfaP2lMyws8sSWduxn530+fC8tCA2aQbFMvV\\\\\\\/viXpYzDC3p2VoPfO88lpZPrY31Gb1TsA2k\\\\\\\/JKUAQGeXgA2+XWKiOO0QqeU3ro7aaaIM0MBkTLEp\\\\\\\/w5tR4pbbRMty1C7ebZSTORw6zFRe\\\\\\\/0iafdPVZyP0UWUz27lwa+5Iv8yf+1Rx\\\\\\\/ri2YoI1VOM9dVRB8rZ8xrXbEddphq+WEQTSL6j+stnomOVUupwWXu74BXWq+\\\\\\\/etUpG4IE23StagP52gET07jRCgoxTuO8MlXI7nz87d0k50PL2AlPs5u6xG1kDfPE+dvx7iiMME7dpEo1CSm0di3IBaGhGz2olmcA2k0OKewnu55YezIuF+v90Yqir1RfBOffXm4uoMWxFSOubbakd9hi2Pv\\\\\\\/Pu4FzZLXKRd6KrhlejeEqECO7fgsn+lH\\\\\\\/LiDbJrLNBQjcN1cX+U9+\\\\\\\/LtvCcb9sXEmapHZbPNMIdsuBN5b49AizbdPWewyev8dkeBCgmAWjC0x4EC2v8h8eYPL4REeo1KqEMwoy\\\\\\\/4rBT0VnP3DhWEw5IQ4dmDa2jSzlsIdqRv7+ca6dIKkZ8nxOqEqHQl2jZcyS4LOXpIp77C+WmVcN2ONu4wCfZEucMnPVACF0Ihth5JrLM0dNedVJ9Fsr3HFoqnteyr7Lg6SPTCxRWRUuqF9cI4mJww75u925QXJtzwmNckjTFCbkZlWjzAUSnZKZXrTULsDBP1HM0JAkIr1ROf\\\\\\\/qMHmiCSnCSWt2ZZOvxkrp5Sy20ZXUMPyOrCF5yy42xhXaQ+L5kJtg1Ad84W+FOGaD5bATckHI5aU2smsfA1hsEg21J6civdccwVBTWHuyRZbNWVSAAg31OvgOqCTdFF04rWtCelb2ujw2Uj1OBwh+FQ91qz3clerDH6HyplZOXyp6dDFzp9g9V+J8aSpAUumyu1u8tRg6GtLmBuwH6AgetUwPQeCNq2jBrH+2rvDHESJK6Wjs1fruAx5\\\\\\\/kSjbXorXnGUS15JGJ\\\\\\\/rqGqjwlGv25MLQbdPv15FRCnRx5r+fPPz1lSHnamYhXVyybJfBskvi3UEX5xu\\\\\\\/OXOox3vwAtRkvA\\\\\\\/3ZEYcIfs7TlwHzW8QHiWcMS1DE2hCWmdeHusdvYQuei5r72FgXqpJIFdDB8cws0zZaJSj598+GsLtj+A6VUFe4bZVWfmJcs2t3+kSE+sVR9e+QwWCjpN963HYR55lBHB5NTyhw3yO62bKYjJhmuzmrhuDtdZBvQVUkCeEh3vOBeCIMLtaCvtHeL3SJUkmQQNkTcLE98r\\\\\\\/iAglAwTDz4pL8rScIcNED+fFOfuAh9XQPpR+r1Arb8Nc2RpGAGFEQUY+NivpwZCx11hoHudbDt9pAxPSy67ReMlzecWFUdZZ9Zl+xw4VR7frJOucxJMvziSwWzbOp8w9H\\\\\\\/kyYyLqzdIO5K5AwcpGOolBf0Ld72yMQQbbltLv6SYNGxWlJLRlI0bTwuZuMqBHFEoWqYgtv2hrGxclw5t5DggY43RcxTMXOGn+f\\\\\\\/NZ9x78XgGw7lHYTh\\\\\\\/PdC9QFNIGVD3JTFsK0QBanwn\\\\\\\/eM54z9s2WDPO+wAq\\\\\\\/2y1XMFvrX6p0cI91DqQ4p1IbHvxpYCY\\\\\\\/FlvofZ\\\\\\\/ylVAsX6JplYFbCdNfmINpIJHOD8\\\\\\\/wYszbliQXwtOMPXKPvk436Y91ZU3tnJ7aHHZagg0r41nEn3QvyUMf1wB3ewhHxhaun2Wow+nqa6RQOmqrA\\\\\\\/EtJPtDjTfIGTs5tdlXULiC8N3qSws6c2CFQT0BHZQ2TlV+vx6lS7LDIoX5rlZZ2WRXAPcEl\\\\\\\/klDAZUpBL+yfBpGNC0aTcP\\\\\\\/HCJozfI0S+ZRzhD6Y2OIPhAsq79N5VSORPnqMFD9g8PTAUKJG0RCbZk9A0PPnEeW8b+90q+h3YQmM+G3lGTRtekYsP45SUXhdh9hqqc9rQd\\\\\\\/++yMogeu2e1RDfbQTuoNPBrV+KrJlUFG6eF8EQjtC+ShEQTik1NGBQuveuxVryWkNhSd7hY73S7XY7TvCfDiCKhAIMQo30iL53Iz7SUc2d7X4\\\\\\\/lHs4SvB7+fEG+8Mr\\\\\\\/MUJzD6jA9aMGIL\\\\\\\/S\\\\\\\/CYmjNrJaR9WcJO4BdXSDDxEJCG4p2bU0eBE3yaaBb8S\\\\\\\/zYYr6XS20G7spU5puEfdOYlGUJJdGgSuK9h3DM9zT8f\\\\\\\/wgWr8R1UvvAz\\\\\\\/+czBWcgvpSdtIc\\\\\\\/abCmAxsH4UQltQlV8C2KKWcbFhuIvNu99lsCSsW4hlXLbUobPdzwhxtGM0daKx\\\\\\\/ZDBFbUVGH4cx7NKce\\\\\\\/BujKhfGWms2LDq8oXfoI3ohvBVxTzOLYPsGA9WavPJSOLI\\\\\\\/79xnNouI6jIUx3kGw9ogZwTL14RnDTbuTB+WtjjhzHU4nBBEX8kibLlcb3hVWNTqxFZwgk5xBcCUe5EIc\\\\\\\/pnv9Dxj9NrirQLAJEY4B2KXQMq29PT4QIfUGc0pxfqe+IBod2jH7R\\\\\\\/LbTNvSivtah15eJ5WdBUyunjxHK5CjyhkFQpaEixLOpBgGB5uUc31b3WY0\\\\\\\/SEMqTFnE923\\\\\\\/Kycu22mDJGhbLyqnR1Opv+nNf2PSvjJIHj71FlTpaG3hDB2jK89zqlkdB6enguRldw1jUYXMn924NqyZ2RQRBkhbVf5Us3poX1S\\\\\\\/RiUwub8XY4sHuJdIGpozg0WFsnG+6iVP9Q8yAfMKYtNTVpI6n46S4uL+v5homceGeI9sJfP+z6f1Fe7rnqBXNsFiWnu6vaanKpSI1POKfhq59dFEYeDByr76yL\\\\\\\/a\\\\\\\/0fGA0aXZSBhZbL3SfHuYPqWnsDX\\\\\\\/5CSIOYM6sR34GBAROqBrTrQzy6lWLcnPyl4a5kdMQXmibng\\\\\\\/Ql5DGtrTaIzu+0QcV4aJOwiVMcZu7VeAlkuhiSesXTrXSkIo8Y13P2\\\\\\\/Vo3CCcWGhjN3LASuQ4DV6S7Ryo8hSORwTEryNW4E8StMsFpbXhKuFqO+NhZID4bQaNj\\\\\\\/3m2KU0QBAHUSCLVDsF0rCUQfewM1\\\\\\\/Nf\\\\\\\/e5OmSSV+zfbkpNWnP6EHwS+2Uq3rIKFz\\\\\\\/2MwH7yZpOPS6lMAhbhuiByfJDkTqyuL5cd17k5nmKnaQr3aL7OAM5VqFQujZI5zPMu8xpo5KW7prNtdgl+i9nlfmKzh5Vzqzj61QJ+KxSz1ncJWPzCNLmutC58\\\\\\\/qFeUu5imgZnROALnEfpSMI5BPqpinicc7siL1ZbruyKwIwBXoFpaWEPnmPpi4K7XY\\\\\\\/5ziCdS7yhwZLJy8AnsS9dcridltBy6nl\\\\\\\/jsPgsrijtwcTqv2vcTA\\\\\\\/73razHszSyurUht6RFJPaY+wOO+qBaLo8OxEPmCLeFg3ipEJGy+hmPwmitcYl\\\\\\\/FWH1348ouxiTofEVL7JSZ8bWqHBTuEfrXVEVxvpOwdM4cUYrTGCypYNuO0sNb\\\\\\\/Urn\\\\\\\/SMwBCsAaHlUDdNr4\\\\\\\/kPfCNxYIvAgic5+k5Lq42b\\\\\\\/WxtqxZt7G\\\\\\\/itljSDMj5R0Fj6O+Bk6XxLFQvFn5vN\\\\\\\/JkOtbq46vrSo5MW8XszsalwQTbFiQO2ZByv4qpWbR1dFVvmsD0v0S6fO5AVQUanFAnF1ob6Y0Y\\\\\\\/VIopGMSeWBnVv56tDjrGclnnKkq3OufyCVCzZbgHaG7IuglY6bnhScixEm0PfxaCYuHvreIxYNeFvK7GenPVpd9Zj2HiBtz4D5Fnt4HMTD\\\\\\\/KjwkGdvsLtJR\\\\\\\/PFc+cmpXqh6WEoLXyXoTYRAFbpRRGWamn7N5sGHCgRBLdcANfzhqoI5CGRZcMhKXsBiGFCbsF7FTX2Y67Aqya0lHJqiRonP7KfVDthdn3GjZQjGbJjdxiJf7qPXFbsCH6dgefhs0tblkZszdUE11U8JJFWg5uxMYrmqC8hgA5bEl7EGhJwpGZWbqnLPBphvwAxlw3n8kflvQzZknmd5YflKXh6LNWeIAeqSNn03744SQFyORJc+k1PEmU3pMmy4AFl2xh5Su0o6+c3jaQbohL4LJ\\\\\\\/vUgLoGYt5lr0Kb+DznjR1y5sFcGfQIuBnnHdSGQ4s0ARpYR2qnbL79\\\\\\\/+IjSVZyO06FybVraPxWHsEZL18ZPJiLvmoa90ipwwF0hVfFC5kaRdzRT37qpoxH65W5erxf8StV\\\\\\\/1vhzCCBd1PCOb7BfOcu4NtofegLyVczZ\\\\\\\/2AyyvlM83ESEoQ6tE75kWHbf5kq4536M+IgZuSbYIf37JevCc69cnovaiFqkqkf2JTg9W+Woub688gV+imcfIyRg99tBX9gWClntL9cAEQoRmXg\\\\\\\/kpbi3t8ZlJLnzRQ7tjpu5CNVMeLQbiXoQYCGkyuHd4jwxjo8Mzkq1g0AvKIR4OOOPg8LnNaxIPFZdsLxh\\\\\\\/046QDsSEsopSRqmLyZq90qqK37s1G+KjmISuDpFBLivV0+kJ5S5K36WjmNZTnSpmu4Ckn0yqVOVD5JUV8t2Mc+m4ivMYYXhm0txxHti1OPn36KlqMqU7t\\\\\\\/HfUUUqud13lw97DxZidzas74lZVuTkpHrxnA94wKKaYox5EKtKXBwtxA+TpITGYH9r5Z9z1OlN1UjteEme7UGDtbPOyIEoXMMdYT7Bx2wGnlXv7xqFGw8s\\\\\\\/nBTv8NyerYBmXXzcqQj82Ov8utDkgGSrREAmUs03zr3zCEpoqGpng4NNbuYSpHCm5xuCSf8KDsr1dimDYk0a9ykiEG1Ch8QiRR1CWe\\\\\\\/i5uCO7kA73lFFuU\\\\\\\/QVRPlc1T\\\\\\\/SiRTSEHyi10Qkd+nR4efsIOwGaDncBaD6aoNNXZk\\\\\\\/j8zfbrh6Mt4S64lSe7BpnTaVQxw5WJpkZxWgYRQO6KEhPLpnafwmkL8i77CwEoANE4Uc6Krw4zvOGsSKNcZAlSRaxzFSzdnYKA543OScFNyOs48WVZ03J5i3NZqxDbvLCxvaCAkh4vhrElFBctyWKleWwqWbTaCQk1ne06SkiLwwxDP\\\\\\\/EF2tJhcCvdGoJdl\\\\\\\/8j3C+MUBlWjaQA5UlILOaRvthPVyT3hGULTR+O5Dc9QXC3vmd7QLN3Vz69gCA\\\\\\\/Hkr9LAI9F+P3lt774UyvW6dpkp+TXrnkcM1NyIVlA+qc\\\\\\\/dctLpbH3hr3M0iWB0A+98rQBxqvSpwdy8cyKMZ2gCfRG5ICJ\\\\\\\/gIOCsTSG8CKGE6+3p2mYxYpMiPEUskRuIKSxZ5gl+wx9QCiLQRJ9xFs2iSIHZvayuMlH9K7pWjS++lcVFgLJipxe83VaZe2km269KBNqW0HfgGrDmrcLpLjR3ALz+rPL3DOeR8kMgxW1Sa9KBEo\\\\\\\/pDcDpmfxvODhm4BIa2RimuCS9tChz6TkG1EM9PNyc8gccFp0kzApOAd7UXPZJI8fV+JdGX\\\\\\\/Eqi1Mq4MxMUXqDUyFvJNyhYKnExiOoxLZSkb6ZJ77W4KymLpVg3x1hf1ciIUxyPaxPLwnDrURZM61AqXfgfLfMRCxi\\\\\\\/4MgTAhcroGYSV+Bf\\\\\\\/b9bmLZyIcMD\\\\\\\/ZbHlp3f7td9iBEv0b3evenxlqpZnKUqcytP2e+oT4\\\\\\\/TlHsTqpa++Qq5Kcq9dVgP2rRMoDVRxZ1n4+IDv5k2gyq8Iwd1x2VzE2qSRAU0J1y4g0u7Ujewr+LK8LPviobWhaNjFVtVa8BTg5PPTLeMRw3\\\\\\\/02qVaTgd0YNiNjudtO0UH9bIMG1T0yJEtMUSfyt2tReArTUzwaLkmqzUgv4NJwa8mLiuwLWArCpHw5LD2bIFpnch261h\\\\\\\/IK9IBHFHga1BrfvENAp6kEQRxNWmrUF9hvDHuQ8XehAwAirEx9pQBi+TXT8CqpaxYI4aB4AWDanf3auMW8AwphhvdwMy1J8ytHsbNdPAQ5uHQoZLcuzTlfh4avgRwKCtwuWxRckMpfrB62xJlteSQcEmT0YeUjd2xSovAZSNbf8bQfxRlWEn7o3ZcXSVnXRF\\\\\\\/0rw805cXJ5GDrmymoli5SFL3\\\\\\\/A+hnN9Tx76goXi0l95zCWc8hQovD87dHbAN7W0ig8ksFuSevD8zRwssRR02QwGREfdG6BhpBcVYbbStLt9IqmNkLdtlVBl3RP\\\\\\\/Zh7HNQURckcz\\\\\\\/UvrSSetbv3JXtEzbNuCwFXbTIsdqVh+GFFyeK87Q6Q5BGAh9c84Nw0\\\\\\\/mghlRRfGkODUyBFDjKivo5AjrxT1IN6d5UeWtv4FPbd7K5CGSrhwMAL739R0GwHKGrUgTCqB1zn1z9IOHQYXPbXMBqszTiRc1tHwvJfaSPHnSMUNJ5fv+u9qmy+qviuCicEKy1WX7aFk8c1vK9Y6x6wvJ1KRN8TvjtH9kjP8cqEFgwOVDZ4tdkH3D38XrbZ7qP5RXOFE8r1z1rYhGsBWzrKZSvlOjUeepZG9IMMoMnXqjmuXWLBWB+w3HZVhErz0W+eojDCDvPu85kplP37Qk0mql570nWmVXucGoTUS7Ga\\\\\\\/I1ooh20dZBcCVCBLrki0dbA8RJCNWL9MzD2o7CYpbAx7cx91K0T9STNPtTP\\\\\\\/5QnrJX3k5qtnc6Ngnd0gqAPD7zg9fTkyuQQFxJYCGkF5CIFrCqJN6SVhE0Gm16tp4S5+6NLdx3JodH21arDG+Zl1ovpxQBP0MEQakYhyzkp1JamCC5UGJQ4bB8GdcLzufX637ymEw3fcqyPwYXZT6D1nqZvxm1wlVf575KfkgWJpxeYCkg0UDElz0cJHQv\\\\\\\/IqawnBLF\\\\\\\/Oc0RxlwxGEq+ZiOnXNSzyrcPXiQdf41Umr2FfGxA2xs4NQTr1\\\\\\\/LjI1h7RFjk+x12zk1\\\\\\\/9dQ6s9GAwaKeY\\\\\\\/Ap2zY0uGCV47dEfe8mMu8lnGe3T9Yo76zU8zxvTM1ROXadCGTUGE+wbbiLbzld+UrfKH\\\\\\\/\\\\\\\/v9JwmQV3LBiAz45+w93ACjq2esSyuOlrkXM15G\\\\\\\/yeIKRHhHXTW0vcQdZ429EMZIFMrGIfzn0pBZ04Op7wHUV8sqSwG4GjTs7ynJY4xyvgcnaV6Ucf3U+V\\\\\\\/CDK6VYTjq4JPoggcSXLbGxv5mZhRV7en0YukU8\\\\\\\/SOh3A1C29ibE\\\\\\\/q\\\\\\\/yF+p4EdGet30kRuxBku2P1FRybXFvwTa2pX6\\\\\\\/EUUS1rDC3dfFyfCxzNga0EfJe8uSsFEgnDQnY+nmcHfLN+svSoBrWqJS5JKnWkGBz8\\\\\\\/29iG6kXutwA4xYwejdD1lbyANn1yoQycKgzRg7W7FGtkKt7kGUOQsftOkefesNOThae\\\\\\\/fu1Lzbzo2oA1Nd0nukl93g8R3Xe2gAhSlhscnknej1uYOoNdIMZGLuNRCpPxafavbiZdOXcOMSe9KAh3xqlr9MVr39HWomdaBe2EDqssAETBTsh\\\\\\\/MfmxXxKrZLduSLbiXQ1AhkmR0jE337dzofFMKFrNRiOi7WzcDcxnYKSoi\\\\\\\/+\\\\\\\/0OEAIwZDwMVmfUBUP4AylzPYUCooqzFAj\\\\\\\/Ji6sPV2Y1oYUMWuRd7Y+n429obGf5RKq0t4nzykO8WXeyfkRRnvMrS9nBAMUTMpb7rC64CnCqNd4Pn6zjraCWlEqdk\\\\\\\/DFaFtxjZ+\\\\\\\/ETREY0iNNMnsESUoa+gdkYxzVI+FTrZdFpPnwdCDcW39WtB4fTnM\\\\\\\/cwlAG8\\\\\\\/78zAnJ19x1Ap0TKEk30ctNrwvGjXC1X\\\\\\\/fkKai9UuFLAOJgZO7htbWmGNJnot4oWx+lCOyB4BRQu2A66NPyxvpyszI58cJapH8TXLjclERGRLdqEziwQJS5f3t9FX5x7NZGhumIvwUzXVywXuZyPFDKwTs0WvQL3A85IaYiL1qy5qoBJIvkgXTLoE+FiH\\\\\\\/tiDoOJL6fbb4fDqFo8CtWvkk920AjxPj+xh3l4QTzGw7fGFK+Nbe8wyxGXUlRGsdbKFUn1VkxqGqza4xwV3FKqsAFFRE7nW4T+CZTfyyLNKW23zhKsd9NVeZKVhej7KoCmlKDiK8JXZn9geljyo7c63Zkt7r+6g9cbO2TiYssQ4U2GmzlDaiQW8mBhxApLeEOiqRmb1wCFyery9dv2\\\\\\\/14c4iz\\\\\\\/PXdPiC7vBkIyftgqOMZbyv+zWDXUGDY4wMCYfOgmA5mDB4TravIBzasxSJYG2yzF89Ge7pnHJBs+J7pABXBT7A0vCkX6XsdYNS4CZprZ0Nlomn9SicZo11uO1vABngmaNYYRVdRmMGUu42iKAASPeUvGpxSgnDL8SlbsoavOSES1FW8iqVtenPgGgn1yjIpA2mmsiVbjSMg7ml\\\\\\\/rQmy7fnO7DTlkTZOzJWOCoa0qWlALePrx5xQHiAJIPLTU7XtDDEQoeyE+osCmXr9homt+DBGFTkhyg0ZH04G59hqGZRj9ge0jbWIrPqKuNnYRylk5nyWBWzS2YAKXKdXxWQoLcfhmo5RaESn6BI5B5rXDN52jFlReKPp8joU9bHPR6J\\\\\\\/JUUwNED1Zn6xvWYGWu5D7dAiVsZTJKQmb0DADsgfbOGsx9lIRuC8lBkT0fYMt1nNUW\\\\\\\/PpXps4kZrvmw1Hw4HR8EpXbZ2kHYER2PuEIbojtRwyuA+zT0yUBx1OGOZBg2zeQcs1H2zp16moRTDqnBFJi\\\\\\\/78hqtA6UkgSsA7E3I7wfhhwN6KKJ5gM0RRBPvKq51nUiwvjxbg4ADhD\\\\\\\/1cGrHlgeIZTYIj8A+Xl7zPUcZqEIzmMYy6BtoTj\\\\\\\/bZWLls1ACyLyjU0QRYG\\\\\\\/\\\\\\\/QY8pUmoIXfMyB1QiB9G0vmUFTyrV9\\\\\\\/LlLvBhgIoMOD2XM9DjFc0Pgx68iRhzEMDTb2sGc+y5vRUOeWgd7\\\\\\\/BAmIEyO201UCfyg2nZdkEi6aH8Zuzy1MPndYpnCHyhOyZj22QhfIFLU6ftoynCpLay2H7L8fGl\\\\\\\/3KhSZABkUBTTD2z52wbYcDzW0lUgpT5E1c6x3tpCYQcBeNS3YSwu9ZTdQ0QPfxLKE9YGwKElbj\\\\\\\/T7GH4FTwB5UD50+mNpi9J+kchlqec8ZOBtpRgpkQcO+dNpV98FHSWyQlXEBA54y11jOtOGjT\\\\\\\/EMTppdbQXlmFm4sohjcHvvHFygKbQuyrhKEQm2sWoswqOOcUZY7KMr9gO2eJ8TyMCU9Nj\\\\\\\/fcyaoKkWdw5j5IILV1tzhdP\\\\\\\/LMiLVllvNeCtQvATvCZtj6Y+PbMwL6Q4v9UKRxLc5Fa\\\\\\\/FmGufj11QB9TO4DQiYmLEgtKL+u4FTyWbMIFG086Kgi\\\\\\\/4nc+okOK27weXzeKvihKElXgh1v3I60qcI8WAabYVVySCQl7fZ6Q\\\\\\\/CJRgGbaP5NmfGRN2Gz72I3BqlCXRpIFmXREoXeuu7j9GAi9tDVFNehikRu+GRUOf9\\\\\\\/\\\\\\\/WVbsVtfbe81rtKrr1FJZ9trBB1ragZJoQVuvEzfBnexVP4+SxUSLyYTc5QKmV8ocWUYoG\\\\\\\/Y5a5ycmBukIRoVESo5i\\\\\\\/ubM4y7AzJ+hrVirwXbd\\\\\\\/6Uxseed7MJxeG1SgfLT+hSX0WDaYMKOC+6EDGoMBJ4QEYAQX8\\\\\\\/dmYOiUOEs3Si5CDkn+ky25b7qfye1ku\\\\\\\/73JyU4cwrr79Yb9xegzeziBIGOvKPPjdggapip29Q6Sfp0HVgimavaQWy3eV3Hn7OyujSWl05AdSJ1mzO2amBA3\\\\\\\/nTevxS0hwm1U8aA1gr3hS9+LX6nRqfRePwYCaAn+zLjeNX+wVXRnYo3cHQPZCP7OoL7KPfAIz9qQfqV0JChJYrQtk3APZyQaOWxdJ4rBNQYozEzPQfsNFbHFEmD4ly59EEYdLX+IlM54O3MoGpULCvpwH8\\\\\\\/Ah\\\\\\\/Obxbt5lIeR1seJqJ9tGk3FE226JER6nGEFeYjhXOhxWxZGm3bnYlydnzZOeX5LPLq5p0EE8Q\\\\\\\/HsSPpml4n\\\\\\\/2EDB3yrdpgu\\\\\\\/X2eyAPkUCwyeN1Trqzr2u59SSH3XVvSln9ojZwHNDa5fGZ6YqAmFCi7wwdXdRNeIrIdTfCXv2gvQCMtJjVCIOC8yg1s3CdDxNXd\\\\\\\/wqfEB4ggeBh+tseyw5czNgTmgce7zvgm8viwqL2z\\\\\\\/mKCF1OKNDgcfmHudzO0MNa0NHpPnQJ3KYZsphJ3qOCCUgd1EWY2aNtBLLM3zHOgEpraESYsZfcWppBxBNO5z8gwCTW7P4Fq9u7fwAgmRsBGmc8whIp0OWnLP8m7mKLNBbcwJ+I5yGHHrF8n4zrcs+0aV1rVW6s0m410aA7F1Jtld951kRvkF1glAIXjFl41j6f4KUvs\\\\\\\/vrxuLkm99MpMfG\\\\\\\/tjnIFOYcgyxiWkqJ5SnVHz7NWNJYVC512hMBSDnl9wkZvATnmSec9R9fVT2KKmTvyWAludE4e9lFeHyN6BLUsyaIXieX+wv7pnaUOOFsH\\\\\\\/VJ276J5V5G75DugMnd8K5zeeTxWaCQBAkb5IPRcyrs3efRvWSgTQR6vdXslKwgGLHQAyKvriREZ7hfrNXuhJydz\\\\\\\/6UP0AwLZg4FPIGji++d+aPsKCrny60hRixapSF8UqzKH9BUd9uFufQjTf2p+5MZjLAS+Z8+FijFQLldbQafAc0DOt6fTYlkq4VMZdxfU9JgiFZbLUkRzWOysDDWvwAI411Lt1hVEbZ7s\\\\\\\/okevflaK23vzrZvM5sQ\\\\\\\/Z1Hh2zYtOHyv\\\\\\\/uZNrKOcyowDkclXfdiMlAEy1F1c7Tj0TXI4rnIhlYmI6cw8IJU8soV0oxAXTHE9kQZzjEmOq5ETrPv49LhOAkKkb8LGBoZOI6avj8VqINDTDUvMN+t5b8X0DzOYx+3QT9TvxKGfPbZY1kyBFkyYPjUC4uGGR+eddp\\\\\\\/Y5lF18OuuuPUofwq9F5F5dNlX77c3srvdfAdgEdkH\\\\\\\/tI5zTInDVGAN446s0Bdt1QJfnjjy\\\\\\\/Kbyg1+p1xf295hm7Is7aKBVkfOn3pRUg7s7WjPEBovRWpcqR3VQD0OwLgA4J9aWpaayOVRVXOQu09Wcx6sEDd5P+c269AfytmbuzZngVY8QOWtODwycoGzLa4NRmb6e8DepYNC\\\\\\\/wW3dqz8GqlTtFhRFBdnGaSLsvekO0z0Gh87UL0cIaJhZigA5OOudIrVv2Ze9jRlg\\\\\\\/Bc3rPD8CQ5RE4D9652KI67nzP9tYXWLMCFfdMT7K4iCMKK+zwEnSYmY3ZG4S2pGtstaJhPsB7AqMpGj299Ueb9j2ClR3EBiGlAcqBidgM5zWuNJ\\\\\\\/+5qMF6\\\\\\\/K76pUs\\\\\\\/IlcI3TP8welID+SibYc803hp5w3LmHUnY5Tli3Y6JOe1OgfRPe497h7u3DX\\\\\\\/2ydZqpio\\\\\\\/MpyLeG\\\\\\\/fHDORSYq554J9w6RHpb3gvMC9ZfYvg0O5puBauS60sJThEcjz1OURgcZHp2KuqHm97g2Zq8O9X0\\\\\\\/8JdcnRDOoov9FW7J9sFMzTMGMDSW\\\\\\\/EyU92gcrhqkp73mVjbm3rN2LqippzdrDep3S+aUquLaRWjQCj6Y1bk08EGQ1XYgUF9UPe6tLJbomcw3F41Wsg37lLOEgZUZrKbNz8og0XgQ9mVHGk\\\\\\\/cYpAencKRFfW\\\\\\\/wSSkWvtOtdskTJ3\\\\\\\/aJgIakLBqCBAm2RzWbxCFMtN5+6obxtSxlU+DktWAbDQ02lDmJ+EyOl50m4Z5ugtVJIBc9bIYMP9L+4PY+iFmXiVNf9f\\\\\\\/UZ4tOwCGOx9BzYiY75yHkaqVkdWSqhUlmqatiRAYSxT\\\\\\\/5xWOPcsRSAzLTOj3fLrs993a61koPohwHiCemIJZRVqewrzSfu0uflZHKxTpWEboTeQSuwl7GuGxsTfobl4NKhXK+OJVLgJYak9Ehk4xz207oTnKiCLS6CrQAUSBLgJ2XhUvtJ4S2G\\\\\\\/HMqhMy3NTsv24L1JjPuWaXG12wwJIgRmytKrYz1HnQG0YFPVD8agxD0oSJHGjojidcdRrhlDEBETEil9XZw5vmX9Z50bQk7JTBO4o\\\\\\\/MJPqesgaiAox50jDVcct82YHZAN+I7elo2ndGpvF4q8ThwEwdWm1i4gBfMjnzs6uYO\\\\\\\/93cfEw9mUxFMlhtFDgQE2mY8XxT7fRWJpRbTrN7ddURkpNuRzMN1RqrI967W5uwLsC6bfisxxz35NsWcX8P228X4oJr\\\\\\\/D3h0G1Jnb8ZnsrPXIca2pQO0EBmE7n781K6+V3vK9pIVlpRyjprqrfu3JxM+LndrUKDHx9bM+HrrkL5aexqaobOpB\\\\\\\/DsaJQ9bwRU8beYIGmOX1OPRinyVU8014q\\\\\\\/jx2KJY5K5rPlRfoRhSLrWYp2PHV6UXOOsEiTrHk5RIM6GJXOdw5s+o9G\\\\\\\/Mh+gX8XCvnyswL2eJpavyRVBQSftvVMqd6z9DTkyFmJyi71ExU4RI7AsOVtGfi9FvmxaGPUqVFiqBkwVEmnSM3UYsJ3Zn62ue1vi59P1XKqZMPdYLRskOD4zQCaEJRZy4SY\\\\\\\/4VwoenusTTy7qCrPCP6wreYRXJM6VsbsRo9L821oXHiA54reKa\\\\\\\/B1T\\\\\\\/hoqQk8MXWUHatlzNcs\\\\\\\/YPzit7K1Qewf4K64yHfyzk6Qv+W\\\\\\\/\\\\\\\/aVrN1GG1bWqHYaWN6eZaNlzvZFVHJ+fLUA+ClbzMuHpMSLeY0l0+SLCbAsQJBuaL2N6AeVTuTF1wBoXa4iDdP\\\\\\\/i1k6YhW74sTfmCv38F\\\\\\\/udqoZBnhMqwPrIAGAm2QE5ZE1wk2Eblf+wntUD8lB+G7y2E0B02Z6Lgz6SGrAub53Vs6wfNB9+p51YoHhEPIaZIox+7AWdTv852bN8cVX\\\\\\\/n1axj\\\\\\\/N5ogacbwWd35AfQwHG6UEn7J01CCuJYYf2OUawFYPz9jRFTxgcUeZCZdQGYY\\\\\\\/XfUSg7\\\\\\\/gKqLZlHhHB8kWpu1eey5x95KjntiIgyvj+GSEPSYOnFYOwp\\\\\\\/ydd0sfc2PH3K1ro2AUHSy50Ajxdb09mB1EhZv2r70uAHzi7Qp1N8vjEv4jdoxAfDxfSZPec4u69h4ipkXC0x5TDonIpJqh8lKVd87UadJUB6r6x\\\\\\\/vsQtRB\\\\\\\/3XcBAls57NWP\\\\\\\/wp3TIiUSTCqrdG9glp6qZXh\\\\\\\/Q43Uari2K99x+Q5LIOuxTY0Id7Ue2vjkf9rOUR1rTH9PVwJ8xeacNIJ7UHZQE54bHbpxL5hIjLQVE5Q+JvxXAtikVU657e0NLJFMCTprR4uMKkXZfl8eeO+nY+0COaKzzmccrjbYDon5qGpgKstktXKFOBfNnSH8BBg8m9fThgQnRdYA\\\\\\\/NZIOTmRHxDfzy\\\\\\\/qZFWiAx+gY5k0DCh5M9vDr3Vm1Gue5J9lr6DlXpRTNQG6UAM5k3At16btVWjQY5ZTXKNeP98RyE9LQGwDO1y6qnNhubB7JBLQtO9eA9VmcCsRkqm8ke+z3mcTbTWLfrwoDIwtwqzjQGvTqoL5bAL0S2CIbjLVgmP7G6eso+xfqCIw+U2ijs3kuiDeJjYloJhf4AG6jTCis6x\\\\\\\/xu4u7p6B6i70yhppRW+TBcT8S4AfbOiTeqrC6hbR3DZ8YJxWSFqevoAQowI7vAZQgakmM\\\\\\\/5rx4JH+d0BamnCqs3SSSErDFtLsqMPdr3CNRSnGLaVtlwMMmAXc4\\\\\\\/8JLM6rnEya8Nm8FIkH6k3v0V2Yq0Z4futdluUVBaQieGZ04xCorQpVl4x+m0nA7+AjoO5ZB\\\\\\\/Amxa0sIQ\\\\\\\/ryMNA2lgphAR76iuBaHkSnlQ9S8RQa16K9OK15V7TEfmGbUia52CnoCpNgug7zTSB89NuWM+ZGqqZi01sptgpRktDpOzjmCLKgVdHYUYnywLTLP6v0K5Cq0y7rpAtLf3K5+qZG4dlR5X8f5MGEqRBtqY3RbM+SF6O2k0YltI\\\\\\\/mc5yvpvgdrQBo\\\\\\\/YQhPl91Lfma9sGGKNwwZGuRuc2eTQFHo8KXOWKUormtGnj9p5qYR25RN+DdIuXkh9zmieHZbe\\\\\\\/ssxpf1HC\\\\\\\/wIbE1KVkmINx+3PznMxhUdwrl\\\\\\\/Wz3t8\\\\\\\/y5DzOfDHF1eDzaBbZQE268HfGUzr\\\\\\\/ZOilCj\\\\\\\/oUh8RTfwHCEEX8pvuKK8CszFPc43CBXRglffBS2DUFbFfZYv1Fb0EVuSoYLNB4RSH6sYAXbZUIntdn3euFfAAJwNsS8bs6hgSySvvaA4ZOABl\\\\\\\/syjTWSx2Qt8\\\\\\\/9lsAicHpvp\\\\\\\/uCk8Eze+BTdWLaOA8qWKpQ5ncKG70r1vX+fIyVzUEfjJYaJpsn1lbFaoc5cgMu3nLpr+xW1pbAuUCmpwr9Z1TDoUd+4kUeuriWT9UJirOa\\\\\\\/B9ODa2Z7nYwO3xUq0M24jP7jh1cgebuU1Unj06Eot7g5qsxAGMSQsLmIiDMVePJ2dnN72h\\\\\\\/oD2I+ZW10bYmBQzHH3gb4ve5VGQ8KOrE5bmcSn5anw2YrKazxsAD13QBSoDVxuisQ6deZZNTAvNIQitO3MuejUuuRkuVqVd5wugp27schoC3hks\\\\\\\/8t8o0r9+KGZgNnzH0ybIQcuXjb2MTIZ\\\\\\\/6H8UIve4nBUlBAtxXFz4vEoRTT\\\\\\\/W\\\\\\\/SB7ANpL9E9OKJlAbPga509VB3jbafMRu2y6R9v7XuR1MWLmRCObmTXaNFrmVbHDNxfZ5YIhN1mgFxOM9MzJKjyiB\\\\\\\/vY6+B8RlW+A9XRkxlXvtADuBI5YoWqXpB\\\\\\\/RIgG5MwbjrkxUNOQXQ+47l+h01gtoBrLPUpn+Q5FU89wpN+jUQJaJ7\\\\\\\/JbZWE9CU7gyyfcsOjmYesAzl2J2fF8UeQCi14lMmzzFBlRQVvjyO4XVHCyOi6uvFdQOJyBUb4YNQob7yaD0VtjW2iFs8SChS6gQo+2pDoVahPK0S5q0L86ZNpn0MuPGYrc1StlYU8mWCM7iXnrCW8CHDWNJQ8LrKEue\\\\\\\/Il7JR6OLuOlQWkojQzcOHgKenbBS0tgcHNuPjc4\\\\\\\/\\\\\\\/JSzMu4rc+6Zw+hLSbpZqarMZ\\\\\\\/8rVAyYASdChcWo5ZasybY5hxdkw3+\\\\\\\/ZS44e\\\\\\\/3tq8pZCYVSzSh+qb4metuZds1OCi6CJCybOc60PfhXKyIBFKUKEbdgYjWyf+mmVDpdF8yMdU1sg0yi0zYfbFN9FRqT95\\\\\\\/\\\\\\\/7eNN\\\\\\\/oj7X5HRH0jR4of1p2w3jEcrBWbYvYDQYoMByCKXvjZoTALHPtZVSEROI\\\\\\\/UTj10p87gPmvWncb63nLADK2D4kun6nW1EbHeEfXf6Rn3nsb5bT689pTLj0cKJ0Y44DCHL3m30SIowiLlxHbOl1Xeq7\\\\\\\/jDGDOqR0PvUl1Cz+HQusAiVxgbL1Qa5YHHAdjkC5bm2CDPObHGOIANkB10w\\\\\\\/DtmoDjxObLdr3BFBjMQH+sJfBPnJORY4VJQYqeP5KDQIAzukeCvMWaWgywaOjdH7eymTYebk9MeSqdjpgJvXbTm8gKsLNJVUch8CdHw3o16\\\\\\\/EIEqjo6pfsjEsIUAlp7uQzG69vDuieHBsYxKL42eh2zpSMHnXZG5fJK6lw+4P7v3qrCf4MF\\\\\\\/kloOAmSlvySgYcKVI0h1utm9PW16soTxgUiN\\\\\\\/H8JgMvHmp2lRUqbEvAIQZZOwNmfbU+x0WYrndgAV0FXUA9UeWWZJqVMO1j8aLxMWt4VVL27oGIgQTHBoXuhj4d7xwkW4IUZ6cQXZ88XDX4nYy+cQWVRqKMpJhQrT2vHVzA2pNQDFGg5L394gCXn6FskGDAMzxZx8U+qwdJZxwPtLAJUOA6pp0WmZdQKR8mT0GcLL4Tp48tyhfWdMh4ct+ONH9wU1fGDdbwZ6FXERELf6Crb2n7ZprfyCRjltIlDsuvkuG4Xh3BSvDR2Xp8sJJ99pbLMZWKnJW2X5AaS3IbAtbuUdqkwYa4I92XEC5iPe+zAaMLHv+RA5sAH7qeHQ\\\\\\\/lu14Brf9iSHSmqALSxjZwK92vq5iZ8SaNuU8el0fAFWJnryCVVIQ\\\\\\\/yWBYz\\\\\\\/UUhumT+SylbLYJd1VxSnKYEAZCUTa8wLzrMPz4XkElqaDrqskJmRjS6VQmZenekcUu4SVNXZpirnr6qzD7Y54HlVvqmF\\\\\\\/NlJ2vIgiHwROybPFSZvUz+YOk1cIFQrxymSdRDDwOpAAzuNt55C8ssCzeQLRk280O1iQIZIe0V19px1Oqiw\\\\\\\/Gg3tArVRqiKLKEjXTfwykV7RoC0d4TP7OLHwXcUiyWlSsl+hyd5PnvSfPSY6ZxjBJE4raiUYsa04QXLDDgD\\\\\\\/5cwZKijFx7YLr7UYTsyebtz5TyBloby\\\\\\\/OxW\\\\\\\/LVrpB6gJYDtf3KWKOIhf+9azdSVQpqyrBib0isQJC\\\\\\\/iLG9QkHXw+mo0cBAL7aJyq2QocB5LqcWcfYDw+R5EyYM1DAYTMZ5ndHfs3yNHOBeQUpoyZNvyqMyiRp+vU4OfUDlH5TN7MD9Drf6lmFOO1aJIR4WE1a5t7yN\\\\\\\/6rISXbe0yAHR8VjVkHsnPruE0hnOIDQC4IVM2UB2ixpCHpg\\\\\\\/2XJKGlYbeqgXzAxraph2sY8gouWzX1bpF4hMt86d+R7id8SbdHhxyfEeh\\\\\\\/ExKXjGM9u3fYcBm0WW8nkKNKr5sqiwNTrlEmEO+fxEPAIhNVMekdP84o\\\\\\\/ukhu67pm7sFVwHHse3DSdhl8tGCNU1HAey5PD0CPjZx9us6\\\\\\\/w1tGc3M4pzc3C0cmdkONq6hj3rSSQ1Q7tmZjxMQMxumCYzm2unPm8cnEvl2fmPXhQ08tA1h+gZjuoaWp4DiVMHpWU3GdWGlW\\\\\\\/G4\\\\\\\/TDMKaSckLWPhyNyI6tuV11QPid\\\\\\\/3wRflcv5xwJlwHT\\\\\\\/9oIVggJ76N3713Jq3PQpnb1cchGzzpaBKOFl2S6DvL7rTebNMmKbYpkBxa8mfBBErltcyB8srF5BPftN4VN8uMNx+l0LlU3NOx0vGfYlAh8P95uEDCECd73RNNvJvi3izIn6we343VhNlVkY1840zlSQ75wpDwWdzuGGbI4mIIa9DziUyOl6uAK42BXqgNwbE7jJQXiV2PmIL3wLlyoRRld5Uk\\\\\\\/uG4rutWIrNFM2SLHbMWm2A+gf9ihwslH2LP52a\\\\\\\/U2ctJyDcjVpip8ZVMCdk0Rvkl9G+\\\\\\\/DFXVX0TCOPRte+m\\\\\\\/qdjuVFJ4\\\\\\\/07DGmlyUuT2fddQyGU0fMc0W6L2DjdGobEpOmrubpI5wGOrtXkqCMZJdXGwJ8CzpXq02JZp76yRZ7cbLV+JConWi47samqdviVm2MhlohlePtFjgpteDEC0Wi0YEquNKHMEAtu\\\\\\\/v0d53FE\\\\\\\/DWAUuGqxN2KE9PZ8bfsriMPxlZxvbRnhuRtH57rdyksWvpwXV40tjXRScvSlwbRQ\\\\\\\/FmtyenLC7f3DoOJoc1LBRqNZeBTEYWoAt3F0qpPmeha4P6qJi5xzH7QPWxDZaI2gMziAWegDauWBBILzoX1azNY9rZXe8W\\\\\\\/E7LBmT4LbewiNMc3tUAiMaJMw91ZmHhBgJvvoghWblRsiQ97vjW6WC4C+f3RQPj+5Yg\\\\\\\/qVOqCiCla1Wfud8IKBBj26bvKch1k28wjz6PyvRqDX0ur8iB4eQ8N0+HwZfmQHaHFdNfR8X9fvq+lTQR5+GUjwYSO+U72XIRSxzlpxxRvimBxA4gS3329gU+zQUnsF+OMoHe7EheKeBSS4awfuQ6NIsFGsk\\\\\\\/dzSfpwRRhV5vvEwjzUoQGcoyqjOM8jHs1n\\\\\\\/9vXc1m9bd7o3Xxt2RUf+2G1D6FLEJ4KRDEFwRuJkshwwJFQGAuDEJHhbm8jghwOuMM1jjMX3+Ozd5CVsdMLyiBqajrVS0ptTJtzOdtQcPxAcrcW87P8lb+GB+G3fMGwXkJxK5mK2\\\\\\\/XFFS+ylsfVP+23xS02SWKqbylDEYDP53ZIn2JBe2IGG2FYRd7y2sdu+Un9pa0FSh54UKjGTrdze8MaIAqiaPxDGfqpdlVzn2NUf7xTdfbJRUBu2TE1FFwXVvJi35rII9SMEKDxdVQAaTDaefJ3PZpR6qxz9WCv1ZkzQsZW4Nb1sb2ywoXvzNyaBrJAMk1TEbh+4vE\\\\\\\/WUb+5EY0WC+Zib0kfbNucwX8NWoPcfV7mmeKtPdetQOCG67\\\\\\\/OxgcjDzOzRg8b+EDd698NGqNkYl\\\\\\\/sysA+6w1SGBUHwo3iZz5A6+UtWlgTK8E8J3Mi\\\\\\\/08loj9I63YLdAo0WEiF5RPfgcU8\\\\\\\/zHA+yQlN7QjA8XlNM9T5b9G9rNvrYbd6Nsl07tA86b9tGwsm87T5apZ\\\\\\\/G5uWNQJqzSUMDF2IMXmRHZgu5xcYqNsrxQEgynUFp34wgwb5FLMPaju+lo6rGwzJUKVsRZe4ANZr1wmn4F8DvSz9+QrD1HFYkWjdsLSElF6197Lls62uKb2ASD0hbrI2VeVSzKbPFmXu\\\\\\\/i3NBl7SPSvtJ1rTzKbsHiJw9d\\\\\\\/3JsHAiE5xwDiLr1f1UmFRVpgZ1GcvoKJ6kLCI9d9dZ\\\\\\\/z5pfED5sye89MHOpbiaXjVIRiW0KHnoq7fO0QFvgNew8DVUpUFHDqa\\\\\\\/DVHmb1MY6qwGkjk7vdivE4HgD8lJVy0Ra33gwQUbIBa8\\\\\\\/Rhn71wZWOuQEPsGXkS9xqhQ8VTUh6KGfNstxY+Zj6q3D1d2Rf42UGecctKLPFiilS1Q5SgSp\\\\\\\/ZfeTtvlcaf9ORw3ZHsVgWehXCZzlAfNsLdHjU6cxs\\\\\\\/nM6yG5B9twdt3sigmVEQmfkjQN5xNEa5KR80qdAO9Yw6zmlSFFVd36UiOBZ1bP2eMgkHmxp+v+5A7QNVBRdjEMfBpMCsEayx8HZUht+nWq7XMHn4qJ\\\\\\\/yo5BQ\\\\\\\/Rbx1N8ABn+cW6Ht4CDh\\\\\\\/pQakH26U9Q+1fsu6lOf703gbEcUYrKHn3iKBscTu4kCtnJSgEcKLFxSxXc0dpWqPHVjhr6IhmQFvg2HznQ79DMHSQV+pI1DKbT6o6k9DTr1\\\\\\\/hSCRb\\\\\\\/8WZgAfeBM7lR1Xk\\\\\\\/VCmCebM6KshaPEVAlJdVtUGiLBMjr1JwOarWDFq1xgDnfkuBU6eSpEpuBDvt3kPtW\\\\\\\/xVg2sL3KlTDUm+a5URmFspI4POg\\\\\\\/vfcfjBUmFmc8m2k38DzXFX+\\\\\\\/lI0sqZ3i95pA3jAiXCa58YzslNIeQ0Gl\\\\\\\/HsFSH3yng9HSW1X+SmBz301mSfApJdcj\\\\\\\/J1kFQaq9UPqkbFmN2qcq6p98NWh5rFHjZILv+GwagYN9MHqGkpa0dH8Yz0lwqjLEtssDd1pLo48i5N5Kv18m8yjjtx0rvc4iHxwBre8Qpf126T2anMYlb6l\\\\\\\/BgH1h7YPARZIBVSEqeS0T\\\\\\\/E0Z63up+40SdHdz1uf1XT0MfA1fD1HAdSSvo\\\\\\\/o9qaVtG8lPalKjec28MBy8b1bp1BEJP+uUwfhHHMGt0uKe03SNtEiL3bAE341cRgZdVAy9Sy6bCD3i05eWPFVCHrfcKR6Rj33ZCFfkNWaTV4eTMi9qqQczWStt+CWQ==\\\u0022,\\\u0022iv\\\u0022:\\\u002292V1xkmrgPmbwLhUaFdR5Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e10d96e4127cc7867d119bbe3e8569a34b7243a33a6f4a5d5c924a167e7ef5b4\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022bL4J388nLdHwmlZrZWFpHAMij5bdCvJ6IXnbVVouCCDuo1n7BL39WSv6xBBWLKDfH0qA65uXzDeVK3D2\\\\\\\/G+Ju9EupRZBz+Ei\\\\\\\/5UVlQKPsqRyoWodnCbTi6hk\\\\\\\/OG+08R4un1i8Q+FpxP6uO0iog6\\\\\\\/MTw\\\\\\\/wShl3ia1Cv\\\\\\\/mrPB3oRAYZGllb0XidJ1BNzeTGOeQfhTNlZZFwb7qg1dNqzAtHAU56UF\\\\\\\/9ko6RsJDj9Ho0vaHOCOB64L1YxBcRAbqxkVKMy50uD6ziucwCzJQWiJosC6Hs2+TTIn12RsWoVlfM+B5moPNk7qSdrdTziLJj7\\\\\\\/r75ltyrXY+my+VkGm5lxmVtw+xKZUjRTiSfXHXkPX0jM4ihVbJObtmtCGbj1GJLPj9IaWCkF0EgOaA87ODXnHWukTtS9yY3CLXpwy\\\\\\\/AnDppB0aEpEjUq\\\\\\\/r24HMRIsZaAodIGBsXhjrl\\\\\\\/74pGpWzlry6rvMua1jTjPdYAMwINTrJdFjpuGw\\\\\\\/sWhqjSJMnnWpiCw7FLl0\\\\\\\/CcTkf8oojnbit+gf0bcdlvLzUfJm2KuVFjjTNT1Q2WLAWDIiSHrNyRUUX7m+Itfy4Gfg6137uoQSiHDPii9E41LVYFWCjTjQNi7pCJ4MrA9wf4rLjRZ1Bfb46HR0xQv9wnnvMrmWBmwoCrSWjRNJ8tK0p9x92sl0nCXKuGGiZ88sK0uXv2Ftmu0MIAwawvanp6G9gEM8kJeb2BTQ9mM\\\\\\\/pdfNww+cWXS3ZQLunwdciK0GRYAp7WNDVQQF1\\\\\\\/LqgcXRNCO4vE2Z5BTOf94Lrrg98S3YxHUMuipmXw3AoOtx84KCFg8Kn960fAh3RBDsKVFE\\\\\\\/b3wfbnxepCEuhtvcfrukENOEddZ78\\\\\\\/QYrX2IvGRMBgbSxsD5k1qIfVYjFdWe2G+GWbo0OmcCefM7ZgA\\\\\\\/duUcXSRqw9wHn4n9josPphWSZ6BGluazwS0MsKT3nJoKXIZIUo2zK2btztt\\\\\\\/tbaDeu6+6NCKV6tPqEwyeOV4voG1yedvilAtYDxNhQgIMT8DYHmn1Nc+ypYvMG2RTx9Np0jl9ALP3JJhC\\\\\\\/pNVnZgAsZRJVtpKMaoVxCIf7bmWhYiN\\\\\\\/RgcxYaUxWXOS0WCxdSYgoCrllGhAKZwog4vWode2QQCig8Wr4ZQDDco7uAohqHQ8qwQihHQa2Ig5qVwprL4UN+bAupsFAZxbCx9yPWLslXm8DaQz5YNemEEz0O2N5T6fyLNu8icZyKLhc6DN\\\\\\\/4TFPbz0ED3JTg6H5sruEhw5\\\\\\\/QQGbJ1ECOqJhUxmks563s7OOftOhflqvGTwZHzeY\\\\\\\/76vVejNOfpVuhkZA\\\\\\\/HDp5xqMbZL7sZDr9U+gQmiyttLWF2uaJUqcGELGggGIqtldqO3CX0uODolyGSRjkdAmyWAneoRMBSwF6PYGXM0M+2389SOJhjb\\\\\\\/97W6bijAMEAnZHUe4OTJEf\\\\\\\/X+RW09XyXlmrhsMt1lGvM8XBb\\\\\\\/8lq0t6Z\\\\\\\/fLvJs8LX\\\\\\\/w7W7DSJIdj++b8qOLKSnnSPiH\\\\\\\/wffjOZl1iX5PYP78I6zOoNxcZdr9OCIt\\\\\\\/qtCUOEJ+lKLjq\\\\\\\/zXdWOy7uxOz1skE7yo6GnM9jJ0e2Crvzy2X2coCU7s5MUJ7ttbhnPnQ0erMChffvumc4sHE+HQ24bxJmCzRWDZ9r8X\\\\\\\/woC5J6ywUH2pGPgNWYMunqk7iD+Foa5NDjRdq8drsOT9QSx5qosrFPD51hd2zbbwLGhBQU8LAtuj1xD3gYiNpH7qj4lzRHXDz71a35lha+GAygOxKasozoK0Jd06L7yTAbtFZClt7+W4L+kjgWUukzrK+yQsWrZ7NkdbbBVTtDWcgn+6ZEfaohBTRMv+Sh7IeyOKulkOlv+GraNUDg2lErSCOKB7m0rwZhoyCSEOOIRlUCb6MnT1LKYaWKJMdE46Zeug\\\\\\\/F3DxZJIEp2kj6k9u6d+eORArY7fwH+uamJAFmfAHlS9CopL1tUEHXEmc+vh5o9PYXr4LEq29vLs58BXRoimpBHrYKh1XY6HhokXxx7630HQnL1iPZUHiXTHitQSRMw8BcNGlhr5je\\\u0022,\\\u0022iv\\\u0022:\\\u0022bULikpipxMtsjA6zeOmC\\\\\\\/g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002283baf93a6ab5fa263c39f7beed4196964e1898575266c8bfeb5c7b929c3c0cf3\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460039 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genomic-selection">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/10/2113" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-pzTHkD1xtO1UqbL-4tVg9mGO9zGQ-uMY5LHqc2l5OjA" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/10/2113" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-tuchxAQELl2_VtBqOoJGgGAaivNzZbToZZMEV4yX4Ns" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/10/2105" title="A Flippase-Mediated GAL80/GAL4 Intersectional Resource for Dissecting Appendage Development in Drosophila" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/10/2127" title="Developmental Transcriptome for a Facultatively Eusocial Bee, Megalopta genalis" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460039" id="node460039" data-pisa="ggg;5/10/2113" data-pisa-master="ggg;g3.115.021154" data-apath="/ggg/5/10/2113.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic-Enabled Prediction of Ordinal Data with Bayesian Logistic Ordinal Regression</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Osval A. Montesinos-López</span>, <span class="highwire-citation-author" data-delta="1">Abelardo Montesinos-López</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>José Crossa</span>, <span class="highwire-citation-author" data-delta="3">Juan Burgueño</span> and <span class="highwire-citation-author" data-delta="4">Kent Eskridge</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2113-2126;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.021154 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460039" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Osval A. Montesinos-López </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Facultad de Telemática, Universidad de Colima, C.P. 28040 Colima, Colima, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Osval%2BA.%2BMontesinos-L%C3%B3pez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montesinos-L%C3%B3pez%20OA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AOsval%2BA.%2BMontesinos-L%25C3%25B3pez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Abelardo Montesinos-López </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Departamento de Estadística, Centro de Investigación en Matemáticas (CIMAT), 36240 Guanajuato, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Abelardo%2BMontesinos-L%C3%B3pez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montesinos-L%C3%B3pez%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAbelardo%2BMontesinos-L%25C3%25B3pez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">José Crossa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Biometrics and Statistics Unit, International Maize and Wheat Improvement Center (CIMMYT), 06600 México, Distrito Federal, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jos%C3%A9%2BCrossa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crossa%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJos%25C3%25A9%2BCrossa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for José Crossa</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a></span></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Juan Burgueño </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Biometrics and Statistics Unit, International Maize and Wheat Improvement Center (CIMMYT), 06600 México, Distrito Federal, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Juan%2BBurgue%C3%B1o%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Burgue%C3%B1o%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJuan%2BBurgue%25C3%25B1o%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Kent Eskridge </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>University of Nebraska, Statistics Department, Lincoln, Nebraska 68583-0963</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kent%2BEskridge%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Eskridge%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKent%2BEskridge%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/5/10/2113" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460039" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460039/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/10/2113.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460039" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460039/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/10/2113.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460039" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460039/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/5/10/2113.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460039" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460039/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Most genomic-enabled prediction models developed so far assume that the response variable is continuous and normally distributed. The exception is the probit model, developed for ordered categorical phenotypes. In statistical applications, because of the easy implementation of the Bayesian probit ordinal regression (BPOR) model, Bayesian logistic ordinal regression (BLOR) is implemented rarely in the context of genomic-enabled prediction [sample size (<em>n</em>) is much smaller than the number of parameters (<em>p</em>)]. For this reason, in this paper we propose a BLOR model using the Pólya-Gamma data augmentation approach that produces a Gibbs sampler with similar full conditional distributions of the BPOR model and with the advantage that the BPOR model is a particular case of the BLOR model. We evaluated the proposed model by using simulation and two real data sets. Results indicate that our BLOR model is a good alternative for analyzing ordinal data in the context of genomic-enabled prediction with the probit or logit link.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/bayesian-ordinal-regression" class="hw-term hw-article-keyword hw-article-keyword-bayesian-ordinal-regression" rel="nofollow">Bayesian ordinal regression</a></li><li class="kwd"><a href="/keyword/genomic-selection" class="hw-term hw-article-keyword hw-article-keyword-genomic-selection" rel="nofollow">genomic selection</a></li><li class="kwd"><a href="/keyword/probit" class="hw-term hw-article-keyword hw-article-keyword-probit" rel="nofollow">probit</a></li><li class="kwd"><a href="/keyword/logit" class="hw-term hw-article-keyword hw-article-keyword-logit" rel="nofollow">logit</a></li><li class="kwd"><a href="/keyword/gibbs-sampler" class="hw-term hw-article-keyword hw-article-keyword-gibbs-sampler" rel="nofollow">Gibbs sampler</a></li><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">GenPred</a></li><li class="kwd"><a href="/keyword/shared-data-resource" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resource" rel="nofollow">shared data resource</a></li></ul><p id="p-3">Genomic-enabled prediction models are revolutionizing animal and plant breeding. There is some evidence that they are powerful for predicting the genomic merit of animals and plants based on high-density single-nucleotide polymorphism (SNP) marker panels and are being recommended increasingly for genomic prediction in human health (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Yang and Tempelman 2012</a>). However, most genomic-enabled prediction models assume a continuous and normally distributed phenotype. Because often this assumption is not fulfilled, researchers normally approach phenotypes in three ways: (a) they ignore the lack of normality in the phenotypes; (b) they transform the non-normal phenotype to approximate it to normality; or (c) they use generalized linear mixed models (GLMMs) to model the appropriate distribution of the phenotype (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Stroup 2015</a>).</p><p id="p-4">The use of the first approach is justified for large sample sizes with the central limit theorem, which states that treatment means have an approximate normal distribution if the sample size is large enough. However, there is a lot of evidence indicating that the first approach produces highly biased results for small- and moderate-sample sizes (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Stroup 2012</a>, <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">2015</a>). Transformations introduced by <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bartlett (1947)</a> for non-normal data were proposed for variance-stabilization to fulfill the assumption of homogeneous variance; they are still considered standard procedures in many agricultural disciplines. Implementation with transformations is equal to that for phenotypes normally distributed (based on the linear model). However, there is mounting evidence that transformations do more harm than good for the models required by most agricultural research, because the use of the linear model with or without transformed data produces a great loss of accuracy and power (<a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">Stroup 2015</a>), mostly in small sample sizes.</p><p id="p-5"><a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Nelder and Wedderburn (1972)</a> introduced generalized linear models, a major departure from the usual approach to non-normal data. GLMMs extend the linear model theory to accommodate non-normal data with heterogeneous variance and even correlated observations. Viewed through the GLMM lens, the pre-1990s understanding of non-normal data––still pervasive in the agricultural research community––is antiquated at best, obsolete at worst. Today, small sample investigations are providing an increasing body of evidence that GLMMs work as well in practice as they do in theory because they produce more accuracy and power than approaches (a) and (b). Also, now there are textbooks and software available for the implementation of GLMMs, although the implementation of approaches (a) and (b) is still dominant in agricultural research (<a id="xref-ref-31-4" class="xref-bibr" href="#ref-31">Stroup 2015</a>). In genomic-enabled prediction, the use of GLMMs is still in its early stages because their implementation is not straightforward given that the number of observations (<em>n</em>) is usually smaller than the number of covariates (<em>p</em>) included in the model. In addition, a complex dependence structure is observed among covariates (markers) and observations (lines) due to the joint involvement of biological processes and pathways.</p><p id="p-6">To overcome this situation in the pregenomic era, <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Gianola (1980</a>, <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">1982</a>) and <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Gianola and Foulley (1983)</a> proposed a probit (threshold) model for dealing with ordinal categorical traits in animal breeding. This probit model was extended to deal with <em>p &gt;&gt; n</em> in the genomic era by <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">González-Recio and Forni (2011)</a> and <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Villanueva <em>et al.</em> (2011)</a> for binary trials, and by <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Wang <em>et al.</em> (2012)</a> and <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Montesinos-López <em>et al.</em> (2015)</a> for more than two ordinal categories. Also, BGLR (Bayesian generalized linear regression), software developed for genomic-enabled prediction that is able to deal with normal, binary, ordinal, and censored data (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">de los Campos and Perez-Rodriguez 2013</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Perez-Rodriguez and de los Campos 2014</a>), is now available. However, no GLMMs are available for genomic-enabled prediction for counts and percentage phenotypes.</p><p id="p-7">For modeling ordinal categorical phenotypes, the ordinal logistic regression model is often preferred over the ordinal probit model in statistical applications, because it provides regression coefficients that are more interpretable due to their connection to odds ratios (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Zucknick and Richardson 2014</a>). However, in genomic-enabled prediction (when <em>p &gt;&gt; n</em>), only the Bayesian probit model is frequently implemented, given that Bayesian methods that introduce sparseness through additional priors on the model size are very well-suited to this problem. Therefore, because of the lack of a Bayesian logistic ordinal model analogous to the Bayesian probit ordinal model that uses a data augmentation approach, the logistic model is not practical for genomic selection.</p><p id="p-8">Both logistic and normal distributions are symmetric with a basic, unimodal “bell curve” shape. The only difference is that the logistic distribution has a somewhat heavier tails, which means that it is less sensitive to outlying data (and hence somewhat more robust for modeling mis-specifications or erroneous data). This is another advantage of logistic regression over probit regression. Because of its easy implementation, the use of BPOR is extremely common, even though it is less robust for modeling mis-specifications and its coefficients are less interpretable. Because of the aforementioned properties of the logistic model, some researchers have proposed approximations to logit regression. For example, <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bartholomew and Knott (1999)</a> proposed <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-1.gif"/></noscript></span></span>, where Φ is the cumulative density function for the standard normal distribution and <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-2.gif"/></noscript></span></span>, whereas <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Camilli (1994)</a> proposed using <em>k</em> = 1.702, obtained by minimizing the maximum distance between two cumulative distribution functions. Although <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Amemiya (1981)</a> proposed a value of <em>k</em> = 1.6 and computed tables for representative values of the density function for different values of <em>k</em>, he did not explain why he used <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-3.gif"/></noscript></span></span>. More recently, <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Savalei (2006)</a> obtained a value of <em>k</em> = 1.75 based on minimizing the Kullback-Leibler information. However, although some of these approximations do a reasonable job of approximating the logistic distribution, they are only approximations, and it goes without saying that an exact solution is preferred.</p><p id="p-9">In this paper, we propose a Bayesian logistic ordinal regression (BLOR) model for genomic-enabled prediction by using a data augmentation approach. We illustrate our proposed method with simulation and real data. We compare the BLOR with the Bayesian probit ordinal regression (BPOR) model with and without approximation.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Gray leaf spot (GLS) and Septoria data sets</h3><p id="p-10">GLS is one of the most important foliar diseases of maize worldwide. The GLS data set is composed of 278 maize lines; the ordinal trait measured in each line was GLS [1 (no disease), 2 (low infection), 3 (moderate infection), 4 (high infection), 5 (complete infection)] caused by the fungus <em>Cercospora zeae-maydis</em>, evaluated in three environments (México, Zimbabwe, and Colombia). These data are part of a data set previously analyzed by <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Crossa <em>et al.</em> (2011)</a>, <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">González-Camacho <em>et al.</em> (2012)</a>, and <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Montesinos-López <em>et al.</em> (2015)</a>. Genotypes of all 278 lines were obtained using the 55k SNP Illumina platform. SNPs with &gt;10% missing values or a minor allele frequency of ≤ 0.05 were excluded from the data. After line-specific quality control (applying the same quality control to each line separately), the maize data still contained 46,347 SNPs.</p><p id="p-11">On the other hand, the Septoria data set contains 268 wheat lines planted in Toluca, México, in 2010, and the trait (Septoria scores) was measured using an ordinal four-point scale. Genotypes of these lines were obtained with 45,000 genotype by sequencing (GBS), following the protocol of <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Poland <em>et al.</em> (2012)</a>. We kept only 13,913 GBS that had &lt;50% missing data; after filtering for minor allele frequency, we ended up with 6787 GBS that were used in the analysis.</p><p id="p-12">For the implementation of the proposed model, we formed five data sets from these two real data sets (GLS and Septoria), four from the GLS data set and one from the Septoria data set. The first three data sets formed from GLS correspond to each environment in which they were evaluated for GLS; the last one was formed by pooling the data from the three environments (information from the three environments without taking into account the environments as covariates).</p></div><div id="sec-3" class="subsection"><h3>Bayesian logistic ordinal regression</h3><p id="p-13">Let <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-4.gif"/></noscript></span></span> (<span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-5.gif"/></noscript></span></span>where <em>i</em> represents the genotype and <em>j</em> denotes the number of replicates or experimental units of each genotype. The total number of observations is <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-6.gif"/></noscript></span></span>In other words, the observed vector <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-7.gif"/></noscript></span></span> contains <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-8.gif"/></noscript></span></span> elements, and the <em>n</em>-dimensional vector <strong>y</strong> of all responses can be written as <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-9.gif"/></noscript></span></span>. The response variable <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-10.gif"/></noscript></span></span> represents an assignment into one of <em>C</em> mutually exclusive and exhaustive categories that follow an order. Therefore, the ordinal logistic regression model can be written in terms of a latent response variable <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-11.gif"/></noscript></span></span> as follows: <span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-12.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>where <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-13.gif"/></noscript></span></span> are called “liabilities”, <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-14.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-15.gif"/></noscript></span></span> denotes the logistic distribution, and the vectors <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-16.gif"/></noscript></span></span> (<em>p</em> × 1) are explanatory variables associated with the fixed effects <strong><em>β</em></strong>. The random effect <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-17.gif"/></noscript></span></span>. In genomic-enabled prediction, <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-18.gif"/></noscript></span></span> Since <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-19.gif"/></noscript></span></span> are unobservable and can be measured indirectly by an observable ordinal variable <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-20.gif"/></noscript></span></span>, then <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-21.gif"/></noscript></span></span> can be defined by:<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-22.gif"/></noscript></span> </span>This means that <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-23.gif"/></noscript></span></span> is divided by thresholds into <em>C</em> intervals, corresponding to <em>C</em> ordered categories. The first threshold, <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-24.gif"/></noscript></span></span>, defines the upper bound of the interval corresponding to observed outcome 1. Similarly, threshold <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-25.gif"/></noscript></span></span> defines the lower bound of the interval corresponding to observed outcome <em>C</em>. Threshold <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-26.gif"/></noscript></span></span> defines the boundary between the interval corresponding to observed outcomes <em>c</em> − 1 and <em>c</em> for (<em>c</em> = 1,2,…, <em>C</em> − 1). Threshold parameters are <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-27.gif"/></noscript></span></span> with <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-28.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-29.gif"/></noscript></span></span></p><p id="p-14">Assuming that the error term <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-30.gif"/></noscript></span></span> of the latent response <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-31.gif"/></noscript></span></span> is distributed as <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-32.gif"/></noscript></span></span>, the cumulative response probability for the <em>c</em> category of the ordinal outcome <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-33.gif"/></noscript></span></span> is:<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-34.gif"/></noscript></span> </span><span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-35.gif"/></noscript></span></span>for <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-36.gif"/></noscript></span></span>.<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-37.gif"/></noscript></span> <span class="disp-formula-label">(2)</span></span>Similarly, model (2) can be written as a cumulative logit model:<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-38.gif"/></noscript></span></span>This GLMM model is described by: (1) two distributions, one for observations in the response variable <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-39.gif"/></noscript></span></span>Multinomial <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-40.gif"/></noscript></span></span>), where <strong><em>β</em></strong> is the p × 1 vector of fixed effects, and another for the random effects, <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-41.gif"/></noscript></span></span> or <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-42.gif"/></noscript></span></span>, where <em>b<sub>i</sub></em> is the effect of line <em>i</em>; (2) linear predictor <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-43.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-44.gif"/></noscript></span></span> denotes the <em>c</em><sup>th</sup> link (<em>c</em> = 1,2,…, <em>C</em> − 1) for the fixed and random effects combination, <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-45.gif"/></noscript></span></span> is the intercept (threshold) for the <em>c</em><sup>th</sup> link, and <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-46.gif"/></noscript></span></span> are known row incidence vectors corresponding to fixed effects in <strong><em>β</em></strong>. Because there are <em>C</em> categories, a total of <em>C</em> − 1 link functions are required to fully specify the model; and (3) link function: cumulative logit {<span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-47.gif"/></noscript></span></span>, (<em>c</em> = 1,2,…, <em>C</em> − 1)}.</p><p id="p-15">Using the inverse link for this model, we can calculate <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-48.gif"/></noscript></span></span>as follows:<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-49.gif"/></noscript></span>.</span>Since we have latent variables <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-50.gif"/></noscript></span></span> distributed as <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-51.gif"/></noscript></span></span> and we observe <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-52.gif"/></noscript></span></span> if, and only if, <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-53.gif"/></noscript></span></span>, then the joint posterior density of the parameter vector and latent variable becomes<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-54.gif"/></noscript></span> </span>Let’s assume a scaled independent inverse chi-square <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-55.gif"/></noscript></span></span> prior for <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-56.gif"/></noscript></span></span>, a normal prior distribution for <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-57.gif"/></noscript></span></span>, a normal prior distribution for <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-58.gif"/></noscript></span></span>, and also a <span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-59.gif"/></noscript></span></span> prior for <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-60.gif"/></noscript></span></span> (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Gianola 2013</a>). Following <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Sorensen <em>et al.</em> (1995)</a>, the prior for the <em>C</em> − 1 unknown thresholds has been given as order statistics from U(<span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-61.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-62.gif"/></noscript></span></span> distribution,<span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-63.gif"/></noscript></span> </span>where <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-64.gif"/></noscript></span></span>.</p><p id="p-16">The fully conditional posterior distributions are provided below and details of all derivations are given in <a id="xref-app-1-1" class="xref-app" href="#app-1">Appendix A</a>.</p></div><div id="sec-4" class="subsection"><h3>Liabilities and Pólya-Gamma values</h3><p id="p-17">The fully conditional posterior distribution of liability <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-65.gif"/></noscript></span></span> is a truncated normal distribution and its density is <span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-66.gif"/></noscript></span> <span class="disp-formula-label">(3)</span></span>For simplicity, <em>ELSE</em> is the data and the parameters, except the one in question. <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-67.gif"/></noscript></span></span> is a normal density with parameters as indicated in the argument, Φ is the cumulative distribution function of a normal density with mean <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-68.gif"/></noscript></span></span>and variance <strong><span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-69.gif"/></noscript></span></span></strong>, and the fully conditional posterior distribution of <em>ω<sub>ij</sub></em> is</p><span class="disp-formula" id="disp-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-70.gif"/></noscript></span>  <span class="disp-formula-label">(4)</span></span></div><div id="sec-5" class="subsection"><h3>Regression coefficients (<em>β</em>)</h3><p id="p-18">The fully conditional posterior of <strong><em>β</em></strong> is as follows:<span class="disp-formula" id="disp-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-71.gif"/></noscript></span>  <span class="disp-formula-label">(5)</span></span>where <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-72.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-73.gif"/></noscript></span></span>. With <span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-74.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-75.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-75.gif"/></noscript></span></span>, <strong><span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-76.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-76.gif"/></noscript></span></span></strong>, <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-77.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-77.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-78.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-78.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-79.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-79.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-80.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-80.gif"/></noscript></span></span>. It is important to point out that if we use a prior for <span class="inline-formula" id="inline-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-81.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-81.gif"/></noscript></span></span> (improper uniform distribution), then in <span class="inline-formula" id="inline-formula-70"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-82.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-82.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-71"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-83.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-83.gif"/></noscript></span></span> we need to make <strong>0</strong> the term <span class="inline-formula" id="inline-formula-72"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-84.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-84.gif"/></noscript></span></span>.</p></div><div id="sec-6" class="subsection"><h3>Polygenic effects (<em>b</em>)</h3><p id="p-19">Now the fully conditional posterior of <strong><em>b</em></strong> is given as</p><span class="disp-formula" id="disp-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-85.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-85.gif"/></noscript></span><span class="disp-formula-label">(6)</span></span></div><div id="sec-7" class="subsection"><h3>Variance of polygenic effects</h3><p id="p-20">Next, the fully conditional posterior of <span class="inline-formula" id="inline-formula-73"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-86.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-86.gif"/></noscript></span></span> is</p><span class="disp-formula" id="disp-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-87.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-87.gif"/></noscript></span>  <span class="disp-formula-label">(7)</span></span></div><div id="sec-8" class="subsection"><h3>Threshold effects (<strong><em>γ</em></strong>)</h3><p id="p-21">The density of the fully conditional posterior distribution of the <em>c</em><sup>th</sup> threshold, <span class="inline-formula" id="inline-formula-74"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-88.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-88.gif"/></noscript></span></span>, is<span class="disp-formula" id="disp-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-89.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-89.gif"/></noscript></span><span class="disp-formula-label">(8)</span></span></p></div><div id="sec-9" class="subsection"><h3>Variance of regression coefficients</h3><p id="p-22">The fully conditional posterior of <span class="inline-formula" id="inline-formula-75"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-90.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-90.gif"/></noscript></span></span> is</p><span class="disp-formula" id="disp-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-91.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-91.gif"/></noscript></span> <span class="disp-formula-label">(9)</span></span></div><div id="sec-10" class="subsection"><h3>The Gibbs sampler</h3><p id="p-23">The Gibbs sampler is implemented by sampling repeatedly from the following loop:</p><ol class="list-ord " id="list-1"><li id="list-item-1"><p id="p-24">Sample liabilities from the truncated normal distribution in (3).</p></li><li id="list-item-2"><p id="p-25">Sample <span class="inline-formula" id="inline-formula-76"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-92.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-92.gif"/></noscript></span></span> values from the Pólya-Gamma distribution in (4).</p></li><li id="list-item-3"><p id="p-26">Sample the regression coefficients from the normal distribution in (5).</p></li><li id="list-item-4"><p id="p-27">Sample the polygenic effects from the normal distribution in (6).</p></li><li id="list-item-5"><p id="p-28">Sample the variance effect (<span class="inline-formula" id="inline-formula-77"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-93.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-93.gif"/></noscript></span></span> from the scaled inverted χ<sup>2</sup> distribution in (7).</p></li><li id="list-item-6"><p id="p-29">Sample the thresholds from the uniform distribution in (8).</p></li><li id="list-item-7"><p id="p-30">Sample the variance of regression coefficients (<span class="inline-formula" id="inline-formula-78"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-94.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-94.gif"/></noscript></span></span> from the scaled inverted χ<sup>2</sup> distribution in (9).</p></li><li id="list-item-8"><p id="p-31">Return to step 1 or terminate if chain length is adequate to meet convergence diagnostics.</p></li></ol><p id="p-32">In the absence of polygenic effects (<strong>b</strong>), the aforementioned Gibbs sampler can be used only by ignoring steps 4 and 5. If all marker effects are taken into account in the design matrix, <strong><em>X</em></strong>, with a prior <span class="inline-formula" id="inline-formula-79"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-95.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-95.gif"/></noscript></span></span> for the beta regression coefficients, we end up with a threshold Bayesian ridge regression. This is a version for ordinal categorical data of the ridge estimator of <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Hoerl and Kennard (1970)</a>, since the posterior expectation of <strong><em>β</em></strong> is equal to <span class="inline-formula" id="inline-formula-80"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-96.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-96.gif"/></noscript></span></span> with pseudo-response <strong><em>l</em></strong>. Another important point is that by setting each <span class="inline-formula" id="inline-formula-81"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-97.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-97.gif"/></noscript></span></span>, the aforementioned Gibbs sampler for the BLOR with the logistic link is reduced to the Gibbs sampler for the BPOR with the probit link proposed by <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Albert and Chib (1993)</a>. This implies that the proposed BLOR model is more general and includes the Gibbs sampler for the BPOR model as a particular case.</p></div><div id="sec-11" class="subsection"><h3>Simulation study</h3><p id="p-33">The purpose of this simulation study is twofold: (1) to compare the performance of the proposed BLOR with: (a) the approximation resulting from using the estimates of the BPOR with <span class="inline-formula" id="inline-formula-82"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-98.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-98.gif"/></noscript></span></span>, denoted as BLOR*, (b) with the results of using maximum likelihood estimators (MLEs) with logit link for ordinal data (MLLOR), and (c) the approximation resulting from multiplying the MLEs with probit link for ordinal data by 1.75, denoted as MLLOR*; (2) to evaluate the performance of the BLOR in the presence of outliers. We used the value 1.75 because, according to the literature review, it is the most reasonable value (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Savalei 2006</a>).</p><p id="p-34">To reach these two goals, two data sets were simulated. Both simulation studies were carried out with the following liability:<span class="disp-formula" id="disp-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-99.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-99.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-83"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-100.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-100.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-84"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-101.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-101.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-85"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-102.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-102.gif"/></noscript></span></span> and the vectors <span class="inline-formula" id="inline-formula-86"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-103.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-103.gif"/></noscript></span></span> have been drawn independently, with components following a uniform distribution within the interval [−0.1, 0.1]. The threshold parameters used were γ<sub>1</sub>= −0.8416, γ<sub>2</sub>= −0.2533, γ<sub>3</sub>= −0.2533, and γ<sub>4</sub>= −0.8416. Then the response variable was generated as follows: <span class="disp-formula" id="disp-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-104.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-104.gif"/></noscript></span></span>For simulated data set 1, we used four values of sample size <em>n<sub>i</sub></em> = 5, 10, 20, and 40 and all the <span class="inline-formula" id="inline-formula-87"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-105.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-105.gif"/></noscript></span></span> were drawn independently from a <em>L</em>(0,1), whereas for simulated data set 2, we used only one sample size (<em>n<sub>i</sub></em> = 40) and the error terms (<span class="inline-formula" id="inline-formula-88"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-106.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-106.gif"/></noscript></span></span>) were obtained from two distributions [<em>L</em>(0,1) and a student's <em>t</em> distribution with four degrees of freedom, denoted as t4]. We studied four scenarios: Scenario 1, the percentage of outliers (PO) from the t4 was 5% and the remaining percentage was obtained from the <em>L</em>(0,1) distribution; Scenario 2: the PO from the t4 was 10%, and 90% from the <em>L</em>(0,1); Scenario 3: the PO from the t4 was 20%, and 80% from the <em>L</em>(0,1); and Scenario 4: the PO from the t4 was 30%, and 70% from the <em>L</em>(0,1).</p><p id="p-35">The MLE estimates for the ordinal regression were obtained using the polr function of the MASS package in R (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">R Core Team 2015</a>). It is important to point out that the priors used for the Bayesian methods were not informative for <span class="inline-formula" id="inline-formula-89"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-107.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-107.gif"/></noscript></span></span>, and for the hyperparameters for thresholds, we used <span class="inline-formula" id="inline-formula-90"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-108.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-108.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-91"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-109.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-109.gif"/></noscript></span></span>. We computed 20,000 Markov chain Monte Carlo (MCMC) samples. Bayes estimates were computed using 10,000 samples because the first 10,000 were discarded as burn-in.</p></div><div id="sec-12" class="subsection"><h3>Model implementation</h3><p id="p-36">The Gibbs sampler described previously for the BLOR model was implemented with the R-software (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">R Core Team 2015</a>). Implementation was done with a Bayesian approach and MCMC through the Gibbs sampler algorithm, which samples sequentially from the full conditional distributions until it reaches a stationary process, converging with the joint posterior distribution (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Gelfand and Smith 1990</a>). In the real data, to reduce the potential impact of MCMC errors on prediction accuracy, we performed a total of 60,000 iterations with a burn-in of 20,000, so that 40,000 samples were used for inference. We did not apply thinning of the chains, following the suggestions of <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Geyer (1992)</a>, <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Maceachern and Berliner (1994)</a>, and <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Link and Eaton (2012)</a>, who provide justification for the ban on subsampling MCMC output for approximating simple features of the target distribution (<em>e.g.</em>, means, variances and percentiles), since thinning is neither necessary nor desirable, and unthinned chains are more precise. It is important to point out that implementation of the BLOR model for the real data sets was done using the hyperparameters <span class="inline-formula" id="inline-formula-92"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-110.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-110.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-93"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-111.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-111.gif"/></noscript></span></span>, <em>γ</em><sub>min</sub> = −1000 and <em>γ</em><sub>max</sub>= −1000 for thresholds parameters, all of which were chosen to lead weakly informative priors.</p></div><div id="sec-13" class="subsection"><h3>Assessing prediction accuracy</h3><p id="p-37">We used cross-validation to estimate the prediction accuracy of the proposed models. The data set was divided into training and validation sets 10 times, with 90% of the data set used for training and 10% for testing; this was done only for the pooled data. The training set was used to fit the model and the validation set was used to evaluate the prediction accuracy of the proposed models. Since the phenotype response is ordinal categorical, we used the Brier score (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Brier 1950</a>) to measure prediction ability, which is equal to<span class="disp-formula" id="disp-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-112.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-112.gif"/></noscript></span> <span class="disp-formula-label">(10)</span></span>where BS denotes the Brier Score, and <em>d<sub>ic</sub></em> takes a value of 1 if the ordinal categorical response observed for individual <em>i</em> falls into category <em>c</em>; otherwise, <em>d<sub>ic</sub></em> = 0. This scoring rule uses all the information contained in the predictive distribution, not just a small portion like the hit rate or the log-likelihood score. Therefore, it is a reasonable choice for comparing categorical regression models, although there are other scoring rules that also have good properties. The range of BS in <a id="xref-disp-formula-19-1" class="xref-disp-formula" href="#disp-formula-19">equation (10)</a> is between 0 and 2. For this reason, we divided Brier scores (BS)/2 to get the BS bounded between 0 and 1; lower scores imply better predictions. It is important to point out that we also used the BS when analyzing the full data sets. We also used the deviance information criterion (DIC) to compare Bayesian models, as suggested by <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Gelman <em>et al.</em> (2003)</a>; here, the lower the DIC, the better the model.</p></div><div id="sec-14" class="subsection"><h3>Data availability</h3><p id="p-38">The two real data sets and two simulated data sets together with R codes are deposited in the link <a href="http://hdl.handle.net/11529/10254">http://hdl.handle.net/11529/10254</a>. The phenotypic data for GLS in three environments (México, Zimbabwe, and Colombia) for the 278 maize lines, the 46,347 SNPs, and the R scripts developed to fit the predictive models used in this study are given in the files PhenoGLS.RData, and MarkersGLSFinal.RDat. The repository also contains the Septoria genotypic and phenotypic data sets, in files SeptoriaGenotypic.RDat, and SeptoriaPhenotypic. RDat, respectively. The R codes to generate the simulated data sets 1 and 2, and for analyzing the real data set from Colombia are directly given in <a id="xref-app-2-1" class="xref-app" href="#app-2">Appendices B</a>, <a id="xref-app-3-1" class="xref-app" href="#app-3">C</a>, and <a id="xref-app-4-1" class="xref-app" href="#app-4">D</a>, respectively. </p></div></div><div class="section results" id="sec-15"><h2 class="">Results</h2><p id="p-39">In the following sections, we investigate the performance of the proposed BLOR estimator through a simulation study and with real data.</p><div id="sec-16" class="subsection"><h3>Simulated data set</h3><p id="p-40">In <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>, we report average estimates obtained by all methods, along with SDs; all the results are based on 50 replications. From <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>, it is clear that as the sample size increases, the average biases and SD decrease in all cases. This confirmed the consistency properties of all the estimates. <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a> also shows that, in general, the point estimates of the Bayesian estimates (BLOR and BLOR*) are similar to the MLEs (MLLOR, and MLLOR*, which was approximated with the MLE with probit ordinal regression); however, the approximations (BLOR* and MLLOR*) have greater bias and SD. BLOR has less bias and SD in most of the studied parameters, producing better parameter estimates than the MLLOR (which is the correct method that use maximum likelihood with the logit link function). For this reason, the proposed BLOR is an excellent alternative. However, a more in-depth simulation study is required to ensure that these findings are valid for all possible scenarios.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466746/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466746/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Simulated data set 1: Average values (Mean) and SD of MLEs and the Bayesian estimators, with four sample sizes (<em>n<sub>i</sub></em>)</span><div class="sb-div caption-clear"></div></div></div><p id="p-42"><a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a> shows that the smaller the PO, the less bias there is in the parameter estimates under the four methods (two Bayesians and two under Maximum Likelihood). However, under the two Bayesian methods, the proposed BLOR showed less bias than BLOR*, and this behavior is observed for all parameters and PO under study, except when the PO is 30% and for the parameter <em>γ</em><sub>2</sub>. Under the two maximum likelihood methods, the approximate method (MLLOR*) showed greater bias in all scenarios and parameters. Finally, although BLOR and MLLOR produced better results in term of bias, in most cases, MLLOR was better than BLOR.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466732/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466732/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Simulated data set 2: average values (Mean) and SD of MLEs and Bayesian estimators, with four POs</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-17" class="subsection"><h3>Real data sets</h3><p id="p-44">Next we compared BLOR and BPOR with the real data set (GLS and Septoria data) described in the <em>Materials and Methods</em>. However, because with this data set the number of parameters (betas and thresholds) to be estimated is high, we compared both models with point and interval estimates of the probabilities estimated for each category on the four- and five-point ordinal scale for each data set studied. In <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>, we compare BLOR, BPOR, and BLOR* (this is the approximate method since the estimates resulting from BPOR were used to approximate BLOR with <span class="inline-formula" id="inline-formula-152"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-171.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-171.gif"/></noscript></span></span> with five data sets (three locations, the resulting pooling of the three locations and the Septoria data set) made up from the data sets described in the section <em>Materials and Methods</em>. First, when comparing BLOR with BLOR*, we see that there are differences in the point probability estimates, and the lower the probabilities, the greater the differences. However, in general, the widths of the credible sets are similar. Second, when comparing BLOR with BPOR, we see that the point and credible sets for each location and Septoria produced similar results; however, for the pooled data, BLOR produced estimates with narrower confidence sets than BPOR (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>). We observed that the estimates produced using BPOR are less accurate because wider confidence sets are produced when the data are pooled; this could be because the assumption of errors normally distributed with mean zero and variance one when the data are pooled is not fulfilled. Also, there were no differences in the BS of the three models (BLOR, BLOR*, and BPOR).</p><p id="p-45">Comparing the models with DIC, we see that in Zimbabwe, BPOR produced the lowest DIC (DIC = 4150.18), whereas BLOR* produced the highest (DIC = 4156.86). However, the DIC of BLOR (DIC = 4150.29) was very close to that of BPOR. In México, the lowest DIC was for BLOR (DIC = 1313.82), whereas the highest was for BLOR* (DIC = 1315.21). In Colombia, BLOR had the lowest DIC (2577.92) and BLOR* had the highest (2581.66). In the Septoria data set, BPOR had the lowest DIC (651.79) and BLOR had the highest (654.33). Finally, in the pooled data, BLOR (8327.36) also had the lowest DIC, whereas BLOR* had the highest (8339.539). All these results shows that even approximations that did a reasonable job (BLOR*) were sometimes very far from the exact methods (BLOR and BPOR). For this reason, whenever possible, an exact model (BLOR or BPOR) should be chosen. The fact that BPOR is sometimes the best (lower DIC) implies that for these data sets, the assumption <span class="inline-formula" id="inline-formula-153"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-172.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-172.gif"/></noscript></span></span> in <a id="xref-disp-formula-1-1" class="xref-disp-formula" href="#disp-formula-1">Equation (1)</a> is enough.</p><p id="p-46">Finally, in <a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a> we present the BS for the testing sets of the pooled real GLS data with 10 cross-validations and 90% of data used for the training set and 10% for the testing set. For models BLOR and BPOR, the BS are almost identical, which means that, with regard to prediction, both models had a similar performance. However, although the approximation method (BLOR*) produced a higher BS, its prediction accuracy was very close to that of BLOR and BPOR. Although we found that the three models had a similar performance regarding prediction accuracy with this data set, more in-depth research is required to validate this observation.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466852/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466852/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Real data sets: GLS and Septoria data sets</span><div class="sb-div caption-clear"></div></div></div><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466822/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466822/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">GLS data set</span><div class="sb-div caption-clear"></div></div></div><p id="p-49">This paper proposes a method for BLOR using the Pólya-Gamma data augmentation approach of <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Scott and Pillow (2013)</a>, which produces a Gibbs sampler with full conditional distributions similar to that of the BPOR model of <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Albert and Chib (1993)</a>. The proposed method is reduced to the BPOR model when the sampled values, <span class="inline-formula" id="inline-formula-156"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-175.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-175.gif"/></noscript></span></span>, from the Pólya-Gamma distribution in <a id="xref-disp-formula-11-1" class="xref-disp-formula" href="#disp-formula-11">Equation (4)</a> are set to 1. This is an advantage because with the proposed model, researchers can perform an exact logistic or probit ordinal regression without having to do approximations to perform a logistic ordinal regression. The performance of the proposed method was compared with the approximation using the probit ordinal regression model in a small simulation study and real data sets (GLS and Septoria data sets) using a four- and five-point ordinal scale. On the basis of the simulation study, it is clear that the estimation of parameters using the approximation <span class="inline-formula" id="inline-formula-157"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-176.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-176.gif"/></noscript></span></span> produces a considerable amount of bias and can give rise to wrong conclusions in association studies. However, we observed with the real data that, in terms of prediction ability, both models (BLOR and BPOR) have a similar performance even though we observed BLOR had lower DIC values in México, Colombia, and the pooled data. This means that when violation of the assumption <span class="inline-formula" id="inline-formula-158"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-177.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-177.gif"/></noscript></span></span> in <a id="xref-disp-formula-1-2" class="xref-disp-formula" href="#disp-formula-1">Equation (1)</a> is not strong, any model can be used. For this reason, we observed greater accuracy (narrow confidence sets) for the BLOR model compared with the exact BPOR model (BPOR) only with the pooled data set without a covariate for location.</p><p id="p-50">Although with the real data we did not observe an advantage in prediction accuracy with the proposed BLOR model, it is very well documented in statistical literature that logistic ordinal regression is more robust for dealing with outlying data, because logistic distribution has heavier tails which was corroborated in terms of parameter estimates with the simulation done using simulated data set 2. For this reason, the proposed BLOR should be preferred because it is usually not practical to test if the error term in <a id="xref-disp-formula-1-3" class="xref-disp-formula" href="#disp-formula-1">Equation (1)</a> is <span class="inline-formula" id="inline-formula-159"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-178.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-178.gif"/></noscript></span></span> or <span class="inline-formula" id="inline-formula-160"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-179.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-179.gif"/></noscript></span></span>. In addition to being more robust, the proposed method also provides regression coefficients that are more interpretable because of their connection to odds ratios (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Zucknick and Richardson 2014</a>). However, this advantage does not make sense when <em>p &gt;&gt; n</em>, because the main driving force in Bayesian models in the case of <em>p &gt;&gt; n</em> is the prior and not the data (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Gianola 2013</a>). Even with this restriction, this paper unifies logistic and probit ordinal regression under a Bayesian framework and is a useful alternative for genomic-enabled prediction of ordinal categorical trials where available data sets have a larger number of parameters to estimate than observations. Also, the proposed method should be preferred over BPOR when outliers are present and not easily detected. This is especially true for multidimensional data, since many times a few outliers in a data set can have strong influence on parameter estimation and inference.</p><p id="p-51">Finally, it is important to point out that, to devise the method proposed in this paper, we generalized the work of <a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Scott and Pillow (2013)</a> for ordered categorical responses. Our method is elegant, easy to implement, and produces a unified Gibbs sampler framework useful for both the logit and the probit link. For this reason, we believe it is an appealing alternative for plant and animal researchers. Also, the proposed BLOR model can be easily extended to take into account genotype × environment interactions, which play an extremely important role in plant breeding, especially when selecting candidate genotypes.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-177">We thank all International Maize and Wheat Improvement Center (CIMMYT) researchers in the Global Maize Breeding Program (GMP) and in the Global Wheat Breeding Program (GWP), as well as the researchers in the National Programs, that generate the data we use in this study.</p></div><div class="section app" id="app-1"><h2 class="">Appendix A</h2><div class="section" id="sec-18"><h3>Derivation of full conditional distributions</h3><div id="sec-19" class="subsection"><h4>Liabilities and Pólya-Gamma values</h4><p id="p-52">The fully conditional posterior distribution of liability <span class="inline-formula" id="inline-formula-161"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-180.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-180.gif"/></noscript></span></span> is<span class="disp-formula" id="disp-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-181.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-181.gif"/></noscript></span> </span><span class="disp-formula" id="disp-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-182.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-182.gif"/></noscript></span> </span><span class="disp-formula" id="disp-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-183.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-183.gif"/></noscript></span> </span><span class="disp-formula" id="disp-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-184.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-184.gif"/></noscript></span> </span>The last inequality was obtained using a technique called the Pólya-Gamma method (<a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">Scott and Pillow 2013</a>), which is useful when working with logistic likelihoods, and has the form<span class="disp-formula" id="disp-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-185.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-185.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-162"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-186.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-186.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-163"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-187.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-187.gif"/></noscript></span></span> denotes the density of the random variable <span class="inline-formula" id="inline-formula-164"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-188.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-188.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-165"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-189.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-189.gif"/></noscript></span></span> denotes a Pólya-Gamma distribution with parameters <em>b</em> and <em>d</em> and density<span class="disp-formula" id="disp-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-190.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-190.gif"/></noscript></span> , </span>where <span class="inline-formula" id="inline-formula-166"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-191.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-191.gif"/></noscript></span></span> denotes the hyperbolic cosine.</p><p id="p-53">Then the joint distribution of <span class="inline-formula" id="inline-formula-167"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-192.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-192.gif"/></noscript></span></span>and <span class="inline-formula" id="inline-formula-168"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-193.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-193.gif"/></noscript></span></span> is equal to<span class="disp-formula" id="disp-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-194.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-194.gif"/></noscript></span> </span>Therefore, the fully conditional posterior distribution of liability <span class="inline-formula" id="inline-formula-169"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-195.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-195.gif"/></noscript></span></span> is a truncated normal distribution and its density is <span class="disp-formula" id="disp-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-196.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-196.gif"/></noscript></span> </span>where <span class="inline-formula" id="inline-formula-170"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-197.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-197.gif"/></noscript></span></span> is a normal density with parameters as indicated in the argument, Φ is the cumulative distribution function of a normal density with mean <span class="inline-formula" id="inline-formula-171"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-198.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-198.gif"/></noscript></span></span>and variance <strong><span class="inline-formula" id="inline-formula-172"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-199.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-199.gif"/></noscript></span></span></strong>, and the fully conditional posterior distribution of <span class="inline-formula" id="inline-formula-173"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-200.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-200.gif"/></noscript></span></span> is<span class="disp-formula" id="disp-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-201.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-201.gif"/></noscript></span></span>and from here and <a id="xref-disp-formula-12-1" class="xref-disp-formula" href="#disp-formula-12">equation (5)</a> of <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Polson <em>et al.</em> (2013)</a> we get that</p><span class="disp-formula" id="disp-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-202.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-202.gif"/></noscript></span></span></div><div id="sec-20" class="subsection"><h4>Regression coefficients (<em>β</em>)</h4><p id="p-54">First note that the fully conditional posterior of <span class="inline-formula" id="inline-formula-174"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-203.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-203.gif"/></noscript></span></span> is<span class="disp-formula" id="disp-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-204.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-204.gif"/></noscript></span> </span><span class="disp-formula" id="disp-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-205.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-205.gif"/></noscript></span> </span>where <span class="inline-formula" id="inline-formula-175"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-206.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-206.gif"/></noscript></span></span>. Then, the fully conditional posterior distribution of <strong><em>β</em></strong> is<span class="disp-formula" id="disp-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-207.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-207.gif"/></noscript></span> </span><span class="disp-formula" id="disp-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-208.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-208.gif"/></noscript></span> </span><span class="disp-formula" id="disp-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-209.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-209.gif"/></noscript></span> </span>where <span class="inline-formula" id="inline-formula-176"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-210.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-210.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-177"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-211.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-211.gif"/></noscript></span></span>. It is important to point out that if we use a prior for <span class="inline-formula" id="inline-formula-178"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-212.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-212.gif"/></noscript></span></span> (improper uniform distribution), then in <span class="inline-formula" id="inline-formula-179"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-213.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-213.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-180"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-214.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-214.gif"/></noscript></span></span> we need to make <strong>0</strong> the term <span class="inline-formula" id="inline-formula-181"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-215.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-215.gif"/></noscript></span></span>. Finally, the fully conditional posterior of <strong><em>β</em></strong> is</p><span class="disp-formula" id="disp-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-216.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-216.gif"/></noscript></span></span></div><div id="sec-21" class="subsection"><h4>Polygenic effects (b)</h4><p id="p-55">Now the fully conditional posterior of <strong><em>b</em></strong> is given as<span class="disp-formula" id="disp-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-217.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-217.gif"/></noscript></span> </span><span class="disp-formula" id="disp-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-218.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-218.gif"/></noscript></span> </span><span class="disp-formula" id="disp-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-219.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-219.gif"/></noscript></span></span>This implies that the fully conditional posterior of <strong><em>b</em></strong> is </p><span class="disp-formula" id="disp-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-220.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-220.gif"/></noscript></span></span></div><div id="sec-22" class="subsection"><h4>Variance of polygenic effects</h4><p id="p-56">Next, the conditional distribution of <span class="inline-formula" id="inline-formula-182"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-221.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-221.gif"/></noscript></span></span> is obtained. If <span class="inline-formula" id="inline-formula-183"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-222.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-222.gif"/></noscript></span></span>, then<span class="disp-formula" id="disp-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-223.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-223.gif"/></noscript></span> </span>This is the kernel of the scaled inverted <span class="inline-formula" id="inline-formula-184"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-224.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-224.gif"/></noscript></span></span> distribution; therefore, the fully conditional posterior is</p><span class="disp-formula" id="disp-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-225.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-225.gif"/></noscript></span></span></div><div id="sec-23" class="subsection"><h4>Threshold effects (<em>γ</em>)</h4><p id="p-57">The density of the fully conditional posterior distribution of the <em>c</em><sup>th</sup> threshold, <em>γ<sub>c</sub></em>, is <span class="disp-formula" id="disp-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-226.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-226.gif"/></noscript></span> </span><span class="disp-formula" id="disp-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-227.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-227.gif"/></noscript></span><span class="disp-formula-label">(A.1)</span></span>If <a id="xref-disp-formula-43-1" class="xref-disp-formula" href="#disp-formula-43">Equation (A.1)</a> is seen as a function of <em>γ<sub>c</sub></em>, it is evident that the value of <em>γ<sub>c</sub></em> must be larger than all the <span class="inline-formula" id="inline-formula-185"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-228.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-228.gif"/></noscript></span></span> and smaller than all the <span class="inline-formula" id="inline-formula-186"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-229.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-229.gif"/></noscript></span></span>. Hence, as a function of <em>γ<sub>c</sub></em>, <a id="xref-disp-formula-43-2" class="xref-disp-formula" href="#disp-formula-43">Equation (A.1)</a> leads to the uniform density<span class="disp-formula" id="disp-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-230.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-230.gif"/></noscript></span> <span class="disp-formula-label">(A.2)</span></span><a id="xref-disp-formula-44-1" class="xref-disp-formula" href="#disp-formula-44">Equation (A.2)</a> corresponds to a uniform distribution on the interval <span class="inline-formula" id="inline-formula-187"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-231.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-231.gif"/></noscript></span></span> (<a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">Albert and Chib 1993</a>; <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Sorensen <em>et al.</em> 1995</a>).</p></div><div id="sec-24" class="subsection"><h4>Variance of location effects</h4><p id="p-58">If we give <em><span class="inline-formula" id="inline-formula-188"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-232.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-232.gif"/></noscript></span></span></em>, then<span class="disp-formula" id="disp-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-233.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-233.gif"/></noscript></span> </span><span class="disp-formula" id="disp-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-234.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-234.gif"/></noscript></span> </span>This is the kernel of the scaled inverted <span class="inline-formula" id="inline-formula-189"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-235.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-235.gif"/></noscript></span></span> distribution; therefore, the fully conditional posterior is</p><span class="disp-formula" id="disp-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-236.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/10/2113/embed/mml-math-236.gif"/></noscript></span> </span></div></div></div><div class="section app" id="app-2"><h2 class="">Appendix B</h2><div class="section" id="sec-25"><h3>R code for simulating data set 1</h3><p id="p-59">We used this code for each sample size studied, and thus we only need to change ni, that denotes <em>n</em><sub><em>i</em></sub>, for the other values (10, 20, 40). For each sample size, we used this code 50 times for estimating the results in <a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>.</p><p id="p-60">thetav = c(-5,-10,15)</p><p id="p-61">Datos&lt;-numeric(0)</p><p id="p-62">nC=40;ni=5 ####Change this ni for other sample sizes</p><p id="p-63">for(i in 1:nC)</p><p id="p-64">{xi1&lt;-runif(length(thetav),-0.1,0.1)</p><p id="p-65">Eta=t(xi1)%*%thetav</p><p id="p-66">for(j in 1:ni)</p><p id="p-67">{L=Eta + rlogis(1, location = 0, scale = 1)</p><p id="p-68">y=ifelse(L&lt;=-0.8416,1,ifelse(L&lt;=-0.2533,2,ifelse(L&lt;0.2533,3,ifelse(L&lt;0.8416,4,5))))</p><p id="p-69"> Datos&lt;-rbind(Datos,c(j,y,t(xi1)))</p><p id="p-70">}}</p><p id="p-71">colnames(Datos) = c(’j’,’y’,’x1’,’x2’,’x3′)</p><p id="p-72">Datos</p></div></div><div class="section app" id="app-3"><h2 class="">Appendix C</h2><div class="section" id="sec-26"><h3>R codes for simulating data set 2</h3><p id="p-73">We used this code for each percentage of outliers (PO) studied; for this reason, we only need to change sam.out for values: 4, 8, and 12, which represent 10, 20, and 30% of outliers. For each sample size, we used this code 50 times for estimating the results in <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>.</p><p id="p-74">thetav = c(-5,-10,15)</p><p id="p-75">Datos&lt;-numeric(0)</p><p id="p-76">nC=40; ni=40, sam.out =2 ###(this 2 represents 5% of outliers)</p><p id="p-77">for(i in 1:nC)</p><p id="p-78">{xi1&lt;-runif(length(thetav),-0.1,0.1)</p><p id="p-79">Eta=t(xi1)%*%thetav</p><p id="p-80">for(j in 1:ni)</p><p id="p-81">{ L=Eta + ifelse(j&gt;sam.out,rlogis(1, location = 0, scale = 1), rt(1, df=4, ncp=0))</p><p id="p-82">y=ifelse(L&lt;=-0.8416,1,ifelse(L&lt;=-0.2533,2,ifelse(L&lt;0.2533,3,ifelse(L&lt;0.8416,4,5))))</p><p id="p-83"> Datos&lt;-rbind(Datos,c(j,y,t(xi1)))</p><p id="p-84">}}</p><p id="p-85">colnames(Datos) = c(’j’,’y’,’x1’,’x2’,’x3′)</p><p id="p-86">Datos</p></div></div><div class="section app" id="app-4"><h2 class="">Appendix D</h2><div class="section" id="sec-27"><h3>R code for fitting the proposed BLOR for Colombia</h3><p id="p-87">This code was used for all the analysis given in <a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>.</p><p id="p-88">#####Code for the Bayesian Ordinal REgression ##################################</p><p id="p-89">rm(list=ls()) # remove everything from memory in the working environment. </p><p id="p-90">library(matrixcalc)</p><p id="p-91">library(BayesLogit)</p><p id="p-92">library(mvtnorm)</p><p id="p-93">##### We load the matrix of markers ############################################ </p><p id="p-94">load(’MarkersFinal.RData’)</p><p id="p-95">M=as.matrix(X)</p><p id="p-96">DataOrd1=data.frame(read.table(’GLSdataOsval1.csv’,sep=’,’,h=T));</p><p id="p-97">DataOrd=na.omit(DataOrd1)</p><p id="p-98">DataOrd=DataOrd[order(DataOrd$Stock), ]</p><p id="p-99">DataOrd=subset(DataOrd, Loc==’Colombia’)</p><p id="p-100">y=DataOrd[,4]</p><p id="p-101">XD=DataOrd[,1:4]</p><p id="p-102">##### Calculating the marker-derived genomic relationship matrix (GRM) #########</p><p id="p-103">M&lt;-scale(M,center=TRUE,scale=TRUE)</p><p id="p-104">G&lt;-tcrossprod(M)/ncol(M)</p><p id="p-105">LL=t(chol(G))</p><p id="p-106">##### Incidence matrix and covariance for main eff. of lines ###################</p><p id="p-107">XD$Stock&lt;-factor(x=XD$Stock, levels=rownames(M), ordered=TRUE)</p><p id="p-108">ZL&lt;-model.matrix(∼XD$Stock-1)</p><p id="p-109">Z=(ZL%*%LL)</p><p id="p-110">X=Z</p><p id="p-111">tX=t(X)</p><p id="p-112">##### Starting values of Beta0, gammas(thresholds) ############################# </p><p id="p-113">gammas&lt;-c(-Inf,-3.11,-2.54,-1.89,-1.27,Inf)</p><p id="p-114">nt=length(gammas)-2</p><p id="p-115">m = dim(X)[1]</p><p id="p-116">nB=dim(X)[2]</p><p id="p-117">In =diag(nB)</p><p id="p-118">p=nB</p><p id="p-119">Betas=rep(0,,dim(X)[2])</p><p id="p-120">SigmaB=1</p><p id="p-121">L&lt;-rep(0,m)</p><p id="p-122">##### Priors for the parameters ################################################</p><p id="p-123">vB=3</p><p id="p-124">SB=0.001</p><p id="p-125">gam.Min=-1000</p><p id="p-126">gam.Max=1000</p><p id="p-127">##### Number of iterations requiered for the Gibbs sampler #####################</p><p id="p-128">Niter&lt;-60000</p><p id="p-129">#### Matrices for saving the output ############################################</p><p id="p-130">MBetas&lt;-matrix(nrow=Niter,ncol=nB)</p><p id="p-131">Mgammas&lt;-matrix(nrow=Niter,ncol=nt)</p><p id="p-132">Mvar_Beta&lt;-matrix(nrow=Niter,ncol=1)</p><p id="p-133">#### Function for sampling normal truncated values############################### </p><p id="p-134">ntruncada&lt;-function(lo,hi,media,std)</p><p id="p-135">{</p><p id="p-136">U = runif(m)</p><p id="p-137">F0 = pnorm(lo,media,std)</p><p id="p-138">c = pnorm(hi,media,std)-F0</p><p id="p-139">muestra = qnorm(c*U+F0)*std+media</p><p id="p-140"> }</p><p id="p-141">##### Gibbs sampler ############################################################</p><p id="p-142">for(i in 1:Niter)</p><p id="p-143">{</p><p id="p-144">##### Linear predictor #########################################################</p><p id="p-145">eta=as.numeric(Betas)</p><p id="p-146">##### Samples from polya gamma distribution ####################################</p><p id="p-147">##### Replacing w=rpg(num=m,2,-L+eta) with w=rep(1,m) we get the BPOR #######</p><p id="p-148">w=rpg(num=m,2,-L+eta)</p><p id="p-149">D=diag(w)</p><p id="p-150">##### Sample liabilities (L) from a truncated normal distribution ##############</p><p id="p-151">L = ntruncada(gammas[y],gammas[y+1],media=eta,1/sqrt(w)) </p><p id="p-152">##### Sample of thresholds #####################################################</p><p id="p-153">newgammas&lt;-numeric(0)</p><p id="p-154">for(k in 2:(nt+1))</p><p id="p-155">{</p><p id="p-156"> lo&lt;-max(c(max(L[y==k-1]),gammas[k-1],gam.Min))</p><p id="p-157"> hi&lt;-min(c(min(L[y==k]),gammas[k+1],gam.Max))</p><p id="p-158"> newgammas&lt;-append(newgammas,runif(1,lo,hi))</p><p id="p-159"> }</p><p id="p-160"> gammas&lt;-c(-Inf,newgammas,Inf)</p><p id="p-161">##### Sample of Betas ##########################################################</p><p id="p-162">C = 1/SigmaB*In</p><p id="p-163">X_tD = tX%*%D</p><p id="p-164">MM=X_tD%*%X</p><p id="p-165">diag(MM)=diag(MM)+diag(C)</p><p id="p-166">S_a=solve(MM,tol=1e-19)</p><p id="p-167">mu_a = S_a%*%(t(X)%*%D%*%L)</p><p id="p-168">Betas=t(rmvnorm(1, mean =mu_a, sigma =S_a))</p><p id="p-169">##### Sample of sigma_Beta##########################################################</p><p id="p-170"> SigmaB = 1/rgamma(1,shape=(vB+p)/2,(SB+tcrossprod(Betas))/2)</p><p id="p-171">##### Saving output ############################################################</p><p id="p-172">MBetas[i,]=t(Betas)</p><p id="p-173">Mgammas[i,]=t(newgammas)</p><p id="p-174">Mvar_Beta[i]=SigmaB</p><p id="p-175">##### Printing some posterior values estimated #################################</p><p id="p-176">cat(’Niter’,i,’SigmaB =’,SigmaB,’Beta1=’,Betas[1],’\n’)}</p></div></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-5"><p id="p-178"><em>Communicating editor: E. Huang</em> </p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2015-05-13"><span class="received-label">Received </span>May 13, 2015.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2015-08-11"><span class="accepted-label">Accepted </span>August 11, 2015.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2015 Montesinos-López <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2113.1" data-doi="10.2307/2290350"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Albert</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chib</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Bayesian analysis of binary and polychotomous response data.</span> <abbr class="cit-jnl-abbrev">J. Am. Stat. Assoc.</abbr> <span class="cit-vol">88</span>: <span class="cit-fpage">669</span>–<span class="cit-lpage">679</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAm.%2BStat.%2BAssoc.%26rft.volume%253D88%26rft.spage%253D669%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2290350%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2290350&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1993LD68000034&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2113.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Amemiya</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">1981</span> <span class="cit-article-title">Qualitative response models: a survey.</span> <abbr class="cit-jnl-abbrev">J. Econ. Lit.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1483</span>–<span class="cit-lpage">1536</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BEcon.%2BLit.%26rft.volume%253D19%26rft.spage%253D1483%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=A1981MV38700001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-book" id="cit-5.10.2113.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bartholomew</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knott</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-source">Latent Variable Models and Factor Analysis</span>. <span class="cit-publ-name">Oxford University Press</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2113.4" data-doi="10.2307/3001536"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bartlett</span>  <span class="cit-name-given-names">M. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1947</span> <span class="cit-article-title">The use of transformations.</span> <abbr class="cit-jnl-abbrev">Biometrics</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">39</span>–<span class="cit-lpage">52</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrics%26rft.stitle%253DBiometrics%26rft.aulast%253DBartlett%26rft.auinit1%253DM.%2BS.%26rft.volume%253D3%26rft.issue%253D1%26rft.spage%253D39%26rft.epage%253D52%26rft.atitle%253DThe%2Buse%2Bof%2Btransformations.%26rft_id%253Dinfo%253Adoi%252F10.2307%252F3001536%26rft_id%253Dinfo%253Apmid%252F20240416%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/3001536&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20240416&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2113.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1947UG24000003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2113.5" data-doi="10.1175/1520-0493(1950)078&lt;0001:VOFEIT&gt;2.0.CO;2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brier</span>  <span class="cit-name-given-names">G. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1950</span> <span class="cit-article-title">Verification of forecasts expressed in terms of probability.</span> <abbr class="cit-jnl-abbrev">Mon. Weather Rev.</abbr> <span class="cit-vol">78</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">3</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMon.%2BWeather%2BRev.%26rft.volume%253D78%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1175%252F1520-0493%25281950%2529078%253C0001%253AVOFEIT%253E2.0.CO%253B2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1175/1520-0493(1950)078&lt;0001:VOFEIT&gt;2.0.CO;2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2113.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Camilli</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Teacher’s corner: origin of the scaling constant d= 1.7 in item response theory.</span> <abbr class="cit-jnl-abbrev">J. Educ. Behav. Stat.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">293</span>–<span class="cit-lpage">295</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BEduc.%2BBehav.%2BStat.%26rft.volume%253D19%26rft.spage%253D293%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2113.7" data-doi="10.1080/15427528.2011.558767"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Rodríguez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mahuku</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dreisigacker</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genomic selection and prediction in plant breeding.</span> <abbr class="cit-jnl-abbrev">J. Crop Improv.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">239</span>–<span class="cit-lpage">261</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BCrop%2BImprov.%26rft.volume%253D25%26rft.spage%253D239%26rft_id%253Dinfo%253Adoi%252F10.1080%252F15427528.2011.558767%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/15427528.2011.558767&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-web" id="cit-5.10.2113.8"><div class="cit-metadata unstructured">de los Campos, G., and P. Perez-Rodriguez, 2013 BGLR: Bayesian generalized linear regression. R package version. Available at: <a href="http://R-Forge.R-project.org/projects/bglr/">http://R-Forge.R-project.org/projects/bglr/</a>. Accessed August 31, 2015.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2113.9" data-doi="10.2307/2289776"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gelfand</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">A. F.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Sampling-based approaches to calculating marginal densities.</span> <abbr class="cit-jnl-abbrev">J. Am. Stat. Assoc.</abbr> <span class="cit-vol">85</span>: <span class="cit-fpage">398</span>–<span class="cit-lpage">409</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAm.%2BStat.%2BAssoc.%26rft.volume%253D85%26rft.spage%253D398%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2289776%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2289776&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1990DJ33200015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-book" id="cit-5.10.2113.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gelman</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carlin</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stern</span>  <span class="cit-name-given-names">H. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rubin</span>  <span class="cit-name-given-names">D. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-source">Bayesian Data Analysis</span>. <span class="cit-edition">Ed. 2</span>. <span class="cit-publ-name">Chapman &amp; Hall/CRC</span>, <span class="cit-publ-loc">Boca Raton, Florida</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2113.11" data-doi="10.1214/ss/1177011137"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Geyer</span>  <span class="cit-name-given-names">C. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Practical Markov Chain Monte Carlo.</span> <abbr class="cit-jnl-abbrev">Stat. Sci.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">473</span>–<span class="cit-lpage">483</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStat.%2BSci.%26rft.volume%253D7%26rft.spage%253D473%26rft_id%253Dinfo%253Adoi%252F10.1214%252Fss%252F1177011137%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1214/ss/1177011137&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2113.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1980</span> <span class="cit-article-title">A method of sire evaluation for dichotomies.</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">51</span>: <span class="cit-fpage">1266</span>–<span class="cit-lpage">1271</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BAnimal%2BScience%26rft.stitle%253DJ%2BANIM%2BSCI%26rft.aulast%253DGianola%26rft.auinit1%253DD.%26rft.volume%253D51%26rft.issue%253D6%26rft.spage%253D1266%26rft.epage%253D1271%26rft.atitle%253DA%2BMethod%2Bof%2BSire%2BEvaluation%2Bfor%2BDichotomies%26rft_id%253Dinfo%253Apmid%252F7204270%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=7204270&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2113.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2113.13" data-doi="10.2527/jas1982.5451079x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1982</span> <span class="cit-article-title">Theory and analysis of threshold characters.</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">1079</span>–<span class="cit-lpage">1096</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BAnimal%2BScience%26rft.stitle%253DJ%2BANIM%2BSCI%26rft.aulast%253DGianola%26rft.auinit1%253DD.%26rft.volume%253D54%26rft.issue%253D5%26rft.spage%253D1079%26rft.epage%253D1096%26rft.atitle%253DTheory%2Band%2BAnalysis%2Bof%2BThreshold%2BCharacters%26rft_id%253Dinfo%253Adoi%252F10.2527%252Fjas1982.5451079x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2527/jas1982.5451079x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1982NP93300025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2113.14" data-doi="10.1534/genetics.113.151753"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Priors in whole-genome regression: the Bayesian alphabet returns.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">194</span>: <span class="cit-fpage">573</span>–<span class="cit-lpage">596</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.151753%26rft_id%253Dinfo%253Apmid%252F23636739%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk0LzMvNTczIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMC8yMTEzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2113.15" data-doi="10.1186/1297-9686-15-1-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foulley</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-article-title">Sire evaluation for ordered categorical data with a threshold model.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">23</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D15%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-15-1-1%26rft_id%253Dinfo%253Apmid%252F22879102%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-15-1-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22879102&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2113.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2113.16" data-doi="10.1007/s00122-012-1868-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">González-Camacho</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Rodríguez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cairns</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genome-enabled prediction of genetic values using radial basis function neural networks.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">125</span>: <span class="cit-fpage">759</span>–<span class="cit-lpage">771</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DGonzalez-Camacho%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D125%26rft.issue%253D4%26rft.spage%253D759%26rft.epage%253D771%26rft.atitle%253DGenome-enabled%2Bprediction%2Bof%2Bgenetic%2Bvalues%2Busing%2Bradial%2Bbasis%2Bfunction%2Bneural%2Bnetworks.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-012-1868-9%26rft_id%253Dinfo%253Apmid%252F22566067%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-012-1868-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22566067&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2113.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2113.17" data-doi="10.1186/1297-9686-43-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">González-Recio</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forni</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome-wide prediction of discrete traits using Bayesian regressions and machine learning.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">7</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DGonzalez-Recio%26rft.auinit1%253DO.%26rft.volume%253D43%26rft.spage%253D7%26rft.epage%253D7%26rft.atitle%253DGenome-wide%2Bprediction%2Bof%2Bdiscrete%2Btraits%2Busing%2BBayesian%2Bregressions%2Band%2Bmachine%2Blearning.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-43-7%26rft_id%253Dinfo%253Apmid%252F21329522%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-43-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21329522&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2113.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2113.18" data-doi="10.2307/1267351"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hoerl</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kennard</span>  <span class="cit-name-given-names">R. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1970</span> <span class="cit-article-title">Ridge regression: biased estimation for nonorthogonal problems.</span> <abbr class="cit-jnl-abbrev">Technometrics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">55</span>–<span class="cit-lpage">67</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTechnometrics%26rft.volume%253D12%26rft.spage%253D55%26rft_id%253Dinfo%253Adoi%252F10.2307%252F1267351%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/1267351&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1970F898700005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2113.19" data-doi="10.1111/j.2041-210X.2011.00131.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Link</span>  <span class="cit-name-given-names">W. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eaton</span>  <span class="cit-name-given-names">M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">On thinning of chains in MCMC.</span> <abbr class="cit-jnl-abbrev">Methods Ecol. Evol.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">112</span>–<span class="cit-lpage">115</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2BEcol.%2BEvol.%26rft.volume%253D3%26rft.spage%253D112%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.2041-210X.2011.00131.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.2041-210X.2011.00131.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2113.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">MacEachern</span>  <span class="cit-name-given-names">S. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berliner</span>  <span class="cit-name-given-names">L. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Subsampling the Gibbs sampler.</span> <abbr class="cit-jnl-abbrev">Am. Stat.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">188</span>–<span class="cit-lpage">190</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BStat.%26rft.volume%253D48%26rft.spage%253D188%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=A1994PQ61600002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-other" id="cit-5.10.2113.21" data-doi="10.1534/g3.114.016188"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Montesinos-López</span>  <span class="cit-name-given-names">O. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Montesinos-López</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Rodríguez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eskridge</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Threshold models for genome-enabled prediction of ordinal categorical traits in plant breeding.</span> <span class="cit-source">G3 (Bethesda)</span> <span class="cit-vol">5</span>: <span class="cit-fpage">291</span>–<span class="cit-lpage">300</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.114.016188%26rft_id%253Dinfo%253Apmid%252F25538102%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjUvMi8yOTEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzEwLzIxMTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2113.22" data-doi="10.2307/2344614"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nelder</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wedderburn</span>  <span class="cit-name-given-names">R. W. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1972</span> <span class="cit-article-title">Generalized linear models.</span> <abbr class="cit-jnl-abbrev">J. R. Stat. Soc. A</abbr> <span class="cit-vol">135</span>: <span class="cit-fpage">370</span>–<span class="cit-lpage">384</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BR.%2BStat.%2BSoc.%2BA%26rft.volume%253D135%26rft.spage%253D370%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2344614%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2344614&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1972N770800027&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2113.23" data-doi="10.1534/genetics.114.164442"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Perez-Rodriguez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genome-wide regression &amp; prediction with the BGLR statistical package.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">483</span>–<span class="cit-lpage">495</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.164442%26rft_id%253Dinfo%253Apmid%252F25009151%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk4LzIvNDgzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNS8xMC8yMTEzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-other" id="cit-5.10.2113.24" data-doi="10.3835/plantgenome2012.06.0006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Endelman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dawson</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rutkoski</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic selection in wheat breeding using genotyping-by-sequencing.</span> <span class="cit-source">Plant Genome</span> <span class="cit-vol">5</span>: <span class="cit-fpage">103</span>–<span class="cit-lpage">113</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D5%26rft.spage%253D103%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2012.06.0006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2012.06.0006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-other" id="cit-5.10.2113.25" data-doi="10.1080/01621459.2013.829001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Polson</span>  <span class="cit-name-given-names">N. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scott</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Windle</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Bayesian inference for logistic models using Pólya-Gamma latent variables.</span> <span class="cit-source">J. Am. Stat. Assoc.</span> <span class="cit-vol">108</span> (<span class="cit-issue">504</span>): <span class="cit-fpage">1339</span>–<span class="cit-lpage">1349</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAm.%2BStat.%2BAssoc.%26rft.volume%253D108%26rft.spage%253D1339%26rft_id%253Dinfo%253Adoi%252F10.1080%252F01621459.2013.829001%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/01621459.2013.829001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-web" id="cit-5.10.2113.26"><div class="cit-metadata unstructured">R Core Team, 2015 R: A language and environment for statistical computing. R Foundation for Statistical Computing, Vienna, Austria. Available at: <a href="http://www.R-project.org/">http://www.R-project.org/</a>. Accessed August 31, 2015.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2113.27" data-doi="10.1007/s11336-004-1237-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Savalei</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Logistic approximation to the normal: the KL rationale.</span> <abbr class="cit-jnl-abbrev">Psychometrika</abbr> <span class="cit-vol">71</span>: <span class="cit-fpage">763</span>–<span class="cit-lpage">767</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPsychometrika%26rft.volume%253D71%26rft.spage%253D763%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11336-004-1237-y%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11336-004-1237-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000245010700007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-book" id="cit-5.10.2113.28"><div class="cit-metadata unstructured">Scott, J., and J. W. Pillow, 2013 Fully Bayesian inference for neural models with negative-binomial spiking, pp. 1898–1906 in Advances in Neural Information Processing Systems 25, edited by F. Pereira, C. J. C. Burges, L. Bottou, and K. Q. Weinberger. Available at: <a href="http://papers.nips.cc/book/advances-in-neural-information-processing-systems-25-2012">http://papers.nips.cc/book/advances-in-neural-information-processing-systems-25-2012</a>. Accessed August 31, 2015.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2113.29" data-doi="10.1051/gse:19950303"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sorensen</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andersen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Korsgaard</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Bayesian inference in threshold models using Gibbs sampling.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">229</span>–<span class="cit-lpage">249</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D27%26rft.spage%253D229%26rft_id%253Dinfo%253Adoi%252F10.1051%252Fgse%253A19950303%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1051/gse:19950303&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1995RE69800003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-book" id="cit-5.10.2113.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stroup</span>  <span class="cit-name-given-names">W. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-source"><em>Generalized Linear Mixed Models: Modern Concepts</em>, <em>Methods and Applications</em></span>. <span class="cit-publ-name">CRC Press</span>, <span class="cit-publ-loc">Boca Raton, Florida</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2113.31" data-doi="10.2134/agronj2013.0342"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stroup</span>  <span class="cit-name-given-names">W. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Rethinking the analysis of non-normal data in plant and soil science.</span> <abbr class="cit-jnl-abbrev">Agron. J.</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">811</span>–<span class="cit-lpage">827</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAgron.%2BJ.%26rft.volume%253D107%26rft.spage%253D811%26rft_id%253Dinfo%253Adoi%252F10.2134%252Fagronj2013.0342%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2134/agronj2013.0342&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2113.32" data-doi="10.2527/jas.2010-3814"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Villanueva</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernandez</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garcia-Cortes</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Varona</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Accuracy of genome-wide evaluation for disease resistance in aquaculture breeding programs.</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">89</span>: <span class="cit-fpage">3433</span>–<span class="cit-lpage">3442</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAnim.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.2527%252Fjas.2010-3814%26rft_id%253Dinfo%253Apmid%252F21742941%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2527/jas.2010-3814&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21742941&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2113.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2113.33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ding</span>  <span class="cit-name-given-names">X. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">W. X.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Bayesian methods for estimating GEBVs of threshold traits.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">213</span>–<span class="cit-lpage">219</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D110%26rft.spage%253D213%26rft_id%253Dinfo%253Apmid%252F23149458%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=23149458&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F10%2F2113.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-5.10.2113.34" data-doi="10.1534/genetics.111.131540"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tempelman</span>  <span class="cit-name-given-names">R. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A Bayesian antedependence model for whole genome prediction.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">1491</span>–<span class="cit-lpage">1501</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.131540%26rft_id%253Dinfo%253Apmid%252F22135352%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5MC80LzE0OTEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy81LzEwLzIxMTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-web" id="cit-5.10.2113.35"><div class="cit-metadata unstructured">Zucknick, M., and S. Richardson, 2014 MCMC algorithms for Bayesian variable selection in the logistic regression model for large-scale genomic applications. Technical Report. Available at: <a href="http://arxiv.org/abs/1402.2713">http://arxiv.org/abs/1402.2713</a>. Accessed August 31, 2015.</div><div class="cit-extra"></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/5/10/2113.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/10/2105" title="A Flippase-Mediated GAL80/GAL4 Intersectional Resource for Dissecting Appendage Development in Drosophila" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/10/2127" title="Developmental Transcriptome for a Facultatively Eusocial Bee, Megalopta genalis" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/5/10"><span>Volume 5 Issue 10, October 2015</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/5/10.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/5/10.cover-source.jpg?itok=JwgNhVLw" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 5 (10)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/10/2113" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-1pETPBjRoPobDgfJEzC9Hkw6bu6zVjY3OlJ8wEVMS7k" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genomic-selection" class="" data-icon-position="" data-hide-link-title="0">Genomic Selection</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/5/10/2113" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/5/10/2113" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/5/10/2113" class="active" data-icon-position="" data-hide-link-title="0">Genomic-Enabled Prediction of Ordinal Data with Bayesian Logistic Ordinal Regression</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460039" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-iudiQYwqoFexWNReb9PWBPe3kgyjT0xuHHS9uzKxoIc" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/10/2113.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/5/10/2113" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-ZTUrJ7Cqv5OeCvnmRnzuMslRiB2V_NxO-R5QHPlBXMg" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/5/10/2113" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/10/2113.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460039" id="node460039--2" data-pisa="ggg;5/10/2113" data-pisa-master="ggg;g3.115.021154" data-apath="/ggg/5/10/2113.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic-Enabled Prediction of Ordinal Data with Bayesian Logistic Ordinal Regression</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Osval A. Montesinos-López</span>, <span class="highwire-citation-author" data-delta="1">Abelardo Montesinos-López</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>José Crossa</span>, <span class="highwire-citation-author" data-delta="3">Juan Burgueño</span> and <span class="highwire-citation-author" data-delta="4">Kent Eskridge</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2113-2126;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.021154 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460039--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Osval A. Montesinos-López </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Facultad de Telemática, Universidad de Colima, C.P. 28040 Colima, Colima, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Osval%2BA.%2BMontesinos-L%C3%B3pez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montesinos-L%C3%B3pez%20OA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AOsval%2BA.%2BMontesinos-L%25C3%25B3pez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Abelardo Montesinos-López </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Departamento de Estadística, Centro de Investigación en Matemáticas (CIMAT), 36240 Guanajuato, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Abelardo%2BMontesinos-L%C3%B3pez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montesinos-L%C3%B3pez%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAbelardo%2BMontesinos-L%25C3%25B3pez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">José Crossa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Biometrics and Statistics Unit, International Maize and Wheat Improvement Center (CIMMYT), 06600 México, Distrito Federal, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jos%C3%A9%2BCrossa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crossa%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJos%25C3%25A9%2BCrossa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for José Crossa</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a></span></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Juan Burgueño </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Biometrics and Statistics Unit, International Maize and Wheat Improvement Center (CIMMYT), 06600 México, Distrito Federal, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Juan%2BBurgue%C3%B1o%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Burgue%C3%B1o%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJuan%2BBurgue%25C3%25B1o%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Kent Eskridge </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>University of Nebraska, Statistics Department, Lincoln, Nebraska 68583-0963</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kent%2BEskridge%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Eskridge%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKent%2BEskridge%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/5/10/2113" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/5/10/2113&amp;title=Genomic-Enabled%20Prediction%20of%20Ordinal%20Data%20with%20Bayesian%20Logistic%20Ordinal%20Regression" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/5/10/2113&amp;title=Genomic-Enabled%20Prediction%20of%20Ordinal%20Data%20with%20Bayesian%20Logistic%20Ordinal%20Regression" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/5/10/2113&amp;title=Genomic-Enabled%20Prediction%20of%20Ordinal%20Data%20with%20Bayesian%20Logistic%20Ordinal%20Regression" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/5/10/2113&amp;text=Genomic-Enabled%20Prediction%20of%20Ordinal%20Data%20with%20Bayesian%20Logistic%20Ordinal%20Regression" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/5/10/2113&amp;title=Genomic-Enabled%20Prediction%20of%20Ordinal%20Data%20with%20Bayesian%20Logistic%20Ordinal%20Regression" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/5/10/2113&amp;t=Genomic-Enabled%20Prediction%20of%20Ordinal%20Data%20with%20Bayesian%20Logistic%20Ordinal%20Regression" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/5/10/2113&amp;title=Genomic-Enabled%20Prediction%20of%20Ordinal%20Data%20with%20Bayesian%20Logistic%20Ordinal%20Regression" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/5/10/2113&amp;title=Genomic-Enabled%20Prediction%20of%20Ordinal%20Data%20with%20Bayesian%20Logistic%20Ordinal%20Regression" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460039/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460039" id="node460039--3" data-pisa="ggg;5/10/2113" data-pisa-master="ggg;g3.115.021154" data-apath="/ggg/5/10/2113.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic-Enabled Prediction of Ordinal Data with Bayesian Logistic Ordinal Regression</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Osval A. Montesinos-López</span>, <span class="highwire-citation-author" data-delta="1">Abelardo Montesinos-López</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>José Crossa</span>, <span class="highwire-citation-author" data-delta="3">Juan Burgueño</span> and <span class="highwire-citation-author" data-delta="4">Kent Eskridge</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2113-2126;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.021154 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460039--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Osval A. Montesinos-López </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Facultad de Telemática, Universidad de Colima, C.P. 28040 Colima, Colima, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Osval%2BA.%2BMontesinos-L%C3%B3pez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montesinos-L%C3%B3pez%20OA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AOsval%2BA.%2BMontesinos-L%25C3%25B3pez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Abelardo Montesinos-López </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Departamento de Estadística, Centro de Investigación en Matemáticas (CIMAT), 36240 Guanajuato, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Abelardo%2BMontesinos-L%C3%B3pez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montesinos-L%C3%B3pez%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAbelardo%2BMontesinos-L%25C3%25B3pez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">José Crossa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Biometrics and Statistics Unit, International Maize and Wheat Improvement Center (CIMMYT), 06600 México, Distrito Federal, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jos%C3%A9%2BCrossa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crossa%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJos%25C3%25A9%2BCrossa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for José Crossa</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a></span></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Juan Burgueño </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Biometrics and Statistics Unit, International Maize and Wheat Improvement Center (CIMMYT), 06600 México, Distrito Federal, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Juan%2BBurgue%C3%B1o%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Burgue%C3%B1o%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJuan%2BBurgue%25C3%25B1o%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Kent Eskridge </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>University of Nebraska, Statistics Department, Lincoln, Nebraska 68583-0963</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kent%2BEskridge%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Eskridge%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKent%2BEskridge%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2113&amp;rft.epage=2126&amp;rft.atitle=Genomic-Enabled%20Prediction%20of%20Ordinal%20Data%20with%20Bayesian%20Logistic%20Ordinal%20Regression&amp;rft.volume=5&amp;rft.issue=10&amp;rft.date=2015-10-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Montesinos-L%C3%B3pez%2C+Osval+A.&amp;rft.au=Montesinos-L%C3%B3pez%2C+Abelardo&amp;rft.au=Crossa%2C+Jos%C3%A9&amp;rft.au=Burgue%C3%B1o%2C+Juan&amp;rft.au=Eskridge%2C+Kent"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460039/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460039/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460039/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460039/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460039/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460039/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460039/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460039/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460039/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460039/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460039/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460039/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490720" id="node490720" data-pisa="ggg;8/9/3019" data-pisa-master="ggg;g3.118.200430" data-apath="/ggg/8/9/3019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">A Bayesian Decision Theory Approach for Genomic Selection</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490724" id="node490724" data-pisa="ggg;8/9/2889" data-pisa-master="ggg;g3.118.200311" data-apath="/ggg/8/9/2889.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/2889" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Optimising Genomic Selection in Wheat: Effect of Marker Density, Population Size and Population Structure on Prediction Accuracy</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490727" id="node490727" data-pisa="ggg;8/9/3039" data-pisa-master="ggg;g3.118.200435" data-apath="/ggg/8/9/3039.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3039" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">BGGE: A New Package for Genomic-Enabled Prediction Incorporating Genotype × Environment Interaction Models</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genomic%20Selection" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genomic Selection</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/5/10/2113" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-15" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#app-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Appendix A</a></li><li class="even"><a href="#app-2" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Appendix B</a></li><li class="odd"><a href="#app-3" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Appendix C</a></li><li class="even"><a href="#app-4" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Appendix D</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/5/10/2113.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/5/10/2113.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/5/10/2113.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__CK8qNg7zHnCPfipcSU0qjCyIiw2pe-WKGx2V0TgizGw__wprDSLLH9CuacxJkxX2DuWt0EaauZA-f_LGjovfY3aw__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
