<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/8/6/1959" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/8/6/1959.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/8/6/1959.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/8/6/1959.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/8/6/1959/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/8/6/1959.atom" />
<meta name="HW.pisa" content="ggg;8/6/1959" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="diploS/HIC: An Updated Approach to Classifying Selective Sweeps" />
<meta name="DC.Identifier" content="10.1534/g3.118.200262" />
<meta name="DC.Date" content="2018-06-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2018 Kern, Schrider. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Identifying selective sweeps in populations that have complex demographic histories remains a difficult problem in population genetics. We previously introduced a supervised machine learning approach, S/HIC, for finding both hard and soft selective sweeps in genomes on the basis of patterns of genetic variation surrounding a window of the genome. While S/HIC was shown to be both powerful and precise, the utility of S/HIC was limited by the use of phased genomic data as input. In this report we describe a deep learning variant of our method, diploS/HIC, that uses unphased genotypes to accurately classify genomic windows. diploS/HIC is shown to be quite powerful even at moderate to small sample sizes." />
<meta name="DC.Contributor" content="Andrew D. Kern" />
<meta name="DC.Contributor" content="Daniel R. Schrider" />
<meta name="article:published_time" content="2018-06-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="diploS/HIC: An Updated Approach to Classifying Selective Sweeps" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Identifying selective sweeps in populations that have complex demographic histories remains a difficult problem in population genetics. We previously introduced a supervised machine learning approach, S/HIC, for finding both hard and soft selective sweeps in genomes on the basis of patterns of genetic variation surrounding a window of the genome. While S/HIC was shown to be both powerful and precise, the utility of S/HIC was limited by the use of phased genomic data as input. In this report we describe a deep learning variant of our method, diploS/HIC, that uses unphased genotypes to accurately classify genomic windows. diploS/HIC is shown to be quite powerful even at moderate to small sample sizes.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2018/06/01" />
<meta name="citation_mjid" content="ggg;8/6/1959" />
<meta name="citation_id" content="8/6/1959" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/8/6/1959" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/8/6/1959.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/8/6/1959.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/8/6/1959.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.118.200262" />
<meta name="citation_pmid" content="29626082" />
<meta name="citation_volume" content="8" />
<meta name="citation_issue" content="6" />
<meta name="citation_num_pages" content="12" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1959" />
<meta name="citation_lastpage" content="1970" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Andrew D. Kern" />
<meta name="citation_author_institution" content="Department of Genetics, Rutgers University, Piscataway, NJ 08854" />
<meta name="citation_author_email" content="kern@biology.rutgers.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-4381-4680" />
<meta name="citation_author" content="Daniel R. Schrider" />
<meta name="citation_author_institution" content="Department of Genetics, Rutgers University, Piscataway, NJ 08854" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-5249-4151" />
<meta name="citation_reference" content="citation_author=M. Abadi;citation_author=A. Agarwal;citation_author=P. Barham;citation_author=E. Brevdo;citation_author=Z. Chen;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_title=Genetic diversity of the African malaria vector Anopheles gambiae.;citation_pages=96-100;citation_volume=552;citation_year=2017" />
<meta name="citation_reference" content="citation_author=J. Chan;citation_author=V. Perrone;citation_author=J. P. Spence;citation_author=P. A. Jenkins;citation_author=S. Mathieson;citation_year=2018" />
<meta name="citation_reference" content="citation_author=F. Chollet;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=M. DeGiorgio;citation_author=C. D. Huber;citation_author=M. J. Hubisz;citation_author=I. Hellmann;citation_author=R. Nielsen;citation_title=Sweepfinder2: increased sensitivity, robustness and flexibility.;citation_volume=32;citation_year=2016;citation_issue=12;citation_pmid=27153702;citation_doi=10.1093/bioinformatics/btw051" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. C. Fay;citation_author=C. I. Wu;citation_title=Hitchhiking Under Positive Darwinian Selection;citation_pages=1405-1413;citation_volume=155;citation_year=2000;citation_issue=3;citation_pmid=10880498" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=N. R. Garud;citation_author=P. W. Messer;citation_author=E. O. Buzbas;citation_author=D. A. Petrov;citation_title=Recent selective sweeps in north american drosophila melanogaster show signatures of soft sweeps.;citation_pages=e1005004;citation_volume=11;citation_year=2015;citation_issue=2;citation_pmid=25706129;citation_doi=10.1371/journal.pgen.1005004" />
<meta name="citation_reference" content="citation_journal_title=Mach. Learn.;citation_author=P. Geurts;citation_author=D. Ernst;citation_author=L. Wehenkel;citation_title=Extremely randomized trees.;citation_pages=3-42;citation_volume=63;citation_year=2006;citation_issue=1;citation_doi=10.1007/s10994-006-6226-1" />
<meta name="citation_reference" content="citation_author=A. Graves;citation_author=A.-r. Mohamed;citation_author=G. Hinton;citation_pages=6645-6649;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Lancet;citation_author=J. Hemingway;citation_author=H. Ranson;citation_author=A. Magill;citation_author=J. Kolaczinski;citation_author=C. Fornadel;citation_title=Averting a malaria disaster: will insecticide resistance derail malaria control?;citation_pages=1785-1788;citation_volume=387;citation_year=2016;citation_issue=10029;citation_pmid=26880124;citation_doi=10.1016/S0140-6736(15)00417-1" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. Hermisson;citation_author=P. S. Pennings;citation_title=Soft sweeps: molecular population genetics of adaptation from standing genetic variation.;citation_volume=169;citation_year=2005;citation_issue=4;citation_pmid=15716498;citation_doi=10.1534/genetics.104.036947" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=J. D. Jensen;citation_title=On the unfounded enthusiasm for soft selective sweeps.;citation_pages=5281;citation_volume=5;citation_year=2014;citation_pmid=25345443;citation_doi=10.1038/ncomms6281" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. D. Jensen;citation_author=Y. Kim;citation_author=V. B. DuMont;citation_author=C. F. Aquadro;citation_author=C. D. Bustamante;citation_title=Distinguishing between selective sweeps and demography using dna polymorphism data.;citation_volume=170;citation_year=2005;citation_issue=3;citation_pmid=15911584;citation_doi=10.1534/genetics.104.038224" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=N. L. Kaplan;citation_author=R. R. Hudson;citation_author=C. H. Langley;citation_title=The &quot;hitchhiking effect&quot; revisited.;citation_pages=887-899;citation_volume=123;citation_year=1989;citation_issue=4;citation_pmid=2612899" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. K. Kelly;citation_title=A Test of Neutrality Based on Interlocus Associations;citation_pages=1197-1206;citation_volume=146;citation_year=1997;citation_issue=3;citation_pmid=9215920" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. D. Kern;citation_author=D. R. Schrider;citation_title=Discoal: flexible coalescent simulations with selection.;citation_volume=32;citation_year=2016;citation_issue=24;citation_pmid=27559153;citation_doi=10.1093/bioinformatics/btw556" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=Y. Kim;citation_author=R. Nielsen;citation_title=Linkage Disequilibrium as a Signature of Selective Sweeps;citation_pages=1513-1524;citation_volume=167;citation_year=2004;citation_issue=3;citation_pmid=15280259;citation_doi=10.1534/genetics.103.025387" />
<meta name="citation_reference" content="citation_author=D. Kingma;citation_author=J. Ba;citation_year=2014" />
<meta name="citation_reference" content="citation_author=A. Krizhevsky;citation_author=G. Hinton;citation_year=2010" />
<meta name="citation_reference" content="citation_title=Advances in Neural Information Processing Systems;citation_title=Imagenet classification with deep convolutional neural networks;citation_author=A. Krizhevsky;citation_author=I. Sutskever;citation_author=G. E. Hinton;citation_pages=1097-1105;citation_year=2012" />
<meta name="citation_reference" content="citation_author=Y. LeCun;citation_author=F. J. Huang;citation_author=L. Bottou;citation_year=2004" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=K. Lin;citation_author=H. Li;citation_author=C. Schlötterer;citation_author=A. Futschik;citation_title=Distinguishing positive selection from neutral evolution: boosting the performance of summary statistics.;citation_volume=187;citation_year=2011;citation_issue=1;citation_pmid=21041556;citation_doi=10.1534/genetics.110.122614" />
<meta name="citation_reference" content="citation_journal_title=Trends Ecol. Evol.;citation_author=P. W. Messer;citation_author=D. A. Petrov;citation_title=Population genomics of rapid adaptation by soft selective sweeps.;citation_pages=659-669;citation_volume=28;citation_year=2013;citation_issue=11;citation_pmid=24075201;citation_doi=10.1016/j.tree.2013.08.003" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=S. N. Mitchell;citation_author=D. J. Rigden;citation_author=A. J. Dowd;citation_author=F. Lu;citation_author=C. S. Wilding;citation_title=Metabolic and target-site mechanisms combine to confer strong ddt resistance in anopheles gambiae.;citation_pages=e92662;citation_volume=9;citation_year=2014;citation_issue=3;citation_pmid=24675797;citation_doi=10.1371/journal.pone.0092662" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=R. Nielsen;citation_author=S. Williamson;citation_author=Y. Kim;citation_author=M. J. Hubisz;citation_author=A. G. Clark;citation_title=Genomic scans for selective sweeps using SNP data;citation_pages=1566-1575;citation_volume=15;citation_year=2005;citation_issue=11;citation_pmid=16251466;citation_doi=10.1101/gr.4252305" />
<meta name="citation_reference" content="citation_author=K. O’Shea;citation_author=R. Nash;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=P. Pavlidis;citation_author=J. D. Jensen;citation_author=W. Stephan;citation_title=Searching for footprints of positive selection in whole-genome SNP data from nonequilibrium populations.;citation_volume=185;citation_year=2010;citation_issue=3;citation_pmid=20407129;citation_doi=10.1534/genetics.110.116459" />
<meta name="citation_reference" content="citation_title=bioRxiv;citation_title=Creating a universal snp and small indel variant caller with deep neural networks.;citation_author=R. Poplin;citation_author=D. Newburger;citation_author=J. Dijamco;citation_author=N. Nguyen;citation_author=D. Loy;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=M. Pybus;citation_author=P. Luisi;citation_author=G. M. Dall’Olio;citation_author=M. Uzkudun;citation_author=H. Laayouni;citation_title=Hierarchical boosting: a machine-learning framework to detect and classify hard selective sweeps in human populations.;citation_volume=31;citation_year=2015;citation_pmid=26315912;citation_doi=10.1093/bioinformatics/btv493" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. R. Rogers;citation_author=C. Huff;citation_title=Linkage disequilibrium between loci with unknown phase.;citation_volume=182;citation_year=2009;citation_issue=3;citation_pmid=19433632;citation_doi=10.1534/genetics.108.093153" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=R. Ronen;citation_author=N. Udpa;citation_author=E. Halperin;citation_author=V. Bafna;citation_title=Learning natural selection from the site frequency spectrum.;citation_volume=195;citation_year=2013;citation_issue=1;citation_pmid=23770700;citation_doi=10.1534/genetics.113.152587" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=D. R. Schrider;citation_author=A. D. Kern;citation_title=S/hic: Robust identification of soft and hard sweeps using machine learning.;citation_pages=e1005928;citation_volume=12;citation_year=2016;citation_issue=3;citation_pmid=26977894;citation_doi=10.1371/journal.pgen.1005928" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=D. R. Schrider;citation_author=A. D. Kern;citation_title=Soft sweeps are the dominant mode of adaptation in the human genome.;citation_pages=1863-1877;citation_volume=34;citation_year=2017;citation_issue=8;citation_doi=10.1093/molbev/msx154" />
<meta name="citation_reference" content="citation_journal_title=Trends Genet.;citation_author=D. R. Schrider;citation_author=A. D. Kern;citation_title=Supervised machine learning for population genetics: a new paradigm.;citation_pages=301-312;citation_volume=34;citation_year=2018;citation_issue=4;citation_doi=10.1016/j.tig.2017.12.005" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. R. Schrider;citation_author=F. K. Mendes;citation_author=M. W. Hahn;citation_author=A. D. Kern;citation_title=Soft shoulders ahead: spurious signatures of soft and partial selective sweeps result from linked hard sweeps.;citation_volume=200;citation_year=2015;citation_issue=1;citation_pmid=25716978;citation_doi=10.1534/genetics.115.174912" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. R. Schrider;citation_author=A. G. Shanku;citation_author=A. D. Kern;citation_title=Effects of linked selective sweeps on demographic inference and model selection.;citation_volume=204;citation_year=2016;citation_issue=3;citation_pmid=27605051;citation_doi=10.1534/genetics.116.190223" />
<meta name="citation_reference" content="citation_journal_title=PLOS Comput. Biol.;citation_author=S. Sheehan;citation_author=Y. S. Song;citation_title=Deep learning for population genetic inference.;citation_pages=e1004845;citation_volume=12;citation_year=2016;citation_issue=3;citation_doi=10.1371/journal.pcbi.1004845" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=K. L. Simonsen;citation_author=G. A. Churchill;citation_author=C. F. Aquadro;citation_title=Properties of statistical tests of neutrality for DNA polymorphism data.;citation_pages=413-429;citation_volume=141;citation_year=1995;citation_issue=1;citation_pmid=8536987" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=J. M. Smith;citation_author=J. Haigh;citation_title=The hitch-hiking effect of a favourable gene.;citation_pages=23-35;citation_volume=23;citation_year=1974;citation_issue=01;citation_pmid=4407212;citation_doi=10.1017/S0016672300014634" />
<meta name="citation_reference" content="citation_title=Advances in Neural Information Processing Systems;citation_title=Sequence to sequence learning with neural networks;citation_author=I. Sutskever;citation_author=O. Vinyals;citation_author=Q. V. Le;citation_pages=3104-3112;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=F. Tajima;citation_title=EVOLUTIONARY RELATIONSHIP OF DNA SEQUENCES IN FINITE POPULATIONS;citation_pages=437-460;citation_volume=105;citation_year=1983;citation_issue=2;citation_pmid=6628982" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=F. Tajima;citation_title=Statistical method for testing the neutral mutation hypothesis by DNA polymorphism.;citation_pages=585-595;citation_volume=123;citation_year=1989;citation_issue=3;citation_pmid=2513255" />
<meta name="citation_reference" content="citation_journal_title=Theoretical population biology;citation_journal_abbrev=Theor Popul Biol;citation_author=G. A. Watterson;citation_title=On the number of segregating sites in genetical models without recombination.;citation_pages=256-276;citation_volume=7;citation_year=1975;citation_issue=2;citation_pmid=1145509;citation_doi=10.1016/0040-5809(75)90020-9" />
<meta name="citation_reference" content="citation_author=F. Yu;citation_author=V. Koltun;citation_year=2015" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="diploS/HIC: An Updated Approach to Classifying Selective Sweeps" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6.cover-source.jpg" />
<meta name="twitter:description" content="Identifying selective sweeps in populations that have complex demographic histories remains a difficult problem in population genetics. We previously introduced a supervised machine learning approach, S/HIC, for finding both hard and soft selective sweeps in genomes on the basis of patterns of genetic variation surrounding a window of the genome. While S/HIC was shown to be both powerful and precise, the utility of S/HIC was limited by the use of phased genomic data as input. In this report we describe a deep learning variant of our method, diploS/HIC, that uses unphased genotypes to accurately classify genomic windows. diploS/HIC is shown to be quite powerful even at moderate to small sample sizes." />
<meta name="og-title" property="og:title" content="diploS/HIC: An Updated Approach to Classifying Selective Sweeps" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/8/6/1959" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Identifying selective sweeps in populations that have complex demographic histories remains a difficult problem in population genetics. We previously introduced a supervised machine learning approach, S/HIC, for finding both hard and soft selective sweeps in genomes on the basis of patterns of genetic variation surrounding a window of the genome. While S/HIC was shown to be both powerful and precise, the utility of S/HIC was limited by the use of phased genomic data as input. In this report we describe a deep learning variant of our method, diploS/HIC, that uses unphased genotypes to accurately classify genomic windows. diploS/HIC is shown to be quite powerful even at moderate to small sample sizes." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6.cover-source.jpg" />
    <title>diploS/HIC: An Updated Approach to Classifying Selective Sweeps | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"r2_R-9Do4TIv0Gqgg-4P23ZCuy0UkOcQOqQomNRU6_I","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"489846","apath":"\/ggg\/8\/6\/1959.atom","pisa":"ggg;8\/6\/1959","ac":{"\/ggg\/8\/6\/1959.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/8\/6\/1959.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;8\/6\/1959"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;8\/6\/1959"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/8\/6\/1959":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022J6HVFVtQk3vG6atUOd1kIbVZkGIatDfUZDjsP8ukDSZKytHwdPq0DFPG0U124esE852K9\\\\\\\/Utu8mebAK4PJhQWNowtqlqVJ6Ml4WxlU\\\\\\\/zphITSUVwHO0\\\\\\\/5PRRvCyAJKrkNNkgD3g5YRi421+zZ1RvKt5Lm8Dm0huFYEdiDQjTXmHnJM\\\\\\\/69PML5mDqJMI4rOIMP\\\\\\\/D\\\\\\\/a8o47CBdpwYQoTtDexDQL1Ah4xYRlE44IqjbJlmb2gs66twI3Ovmh2djLjU2Ub6WNEXScjr8E\\\\\\\/3icEzGygeGC\\\\\\\/sJehwNv22GJzHzVHx641b1aov34RFsyBKLLkfOaGkLU8\\\\\\\/\\\\\\\/HQdGm4Xjep3MGwx+0mkyGkYq3s7c02SccvgLnBHJSU21efFCeVwAZDtFCaxgHGn6\\\\\\\/aGKtLnmzev1lR2rZigvMF95BDXpAx1QNKLSm\\\\\\\/u4BLFJtoWe1Sdl\\\\\\\/Aotl+g7akudOhdYGgOMFTNBeg75KA5d+qSvtiDm9LropCq8mwJAiVgnYrwuiCofgO5AU9MKc1MzdEWOgo078dGPu0OC74yi+QbDH0mJ\\\\\\\/sAlsoUXN7IxUvRiQTvk6HCa8\\\\\\\/rADoX0p9Cx8E+WYQbt1hAADzmZGlMV52uqx+yy732NSXDOD6B3frXs+S0gZ9vEJlOFvq5W1ek7QiSBaRZI9u+VAVh\\\\\\\/Wx2ckeHmNbrplByu\\\\\\\/GY0Lg++yzvT\\\\\\\/EBvHL30hrLpYz28HxVGlaOzdljqTS+SFRph23lEvWxTzKdvu13QYFYBLUZxcvxZehtycN+ThntO5SPb8DC\\\\\\\/YK6e\\\\\\\/8kwlI9JX1v5fswRsjTTS9v2eJzekz9xWP3I53A+C3eRXipTWkWJhoUY0mpgSV1OhaHZ6tFT+kLAZWGc4mVKOdJM3Gd90LlxFDvhGtG4Sg9A5UlPJeUaj7u7y9K1896NZlU6LxcPx5hBRWPAvcp\\\\\\\/54YZ\\\\\\\/F6Ftts8HLN2u15f4aGXqFq0nQ+0dDVAapG+fE+hbpzVjPhS21j9Rl2SGqX8LUwCLFkzs+BSi8IVsz5v6qk3zAIswPMEvRUGBmvDOlkHMqKEBhHRNfXZSb1t0vXDK6NwwyC+bfoY6ADzm9LNM\\\\\\\/\\\\\\\/Wxi1dYCdb7DFmSI4MdeAnqSh5k8OACgBR5BSF0mRF4VfDv4CVlN8SJ3uxVr62al04yddl7jPac1oCYiabPSPUUTl2dJeyhdbaADttC8OEqUuYGyb305kScMExf2iDIF9KGOZRL2uJRWthTN7I+k7YudTDBQNgdfGRKFm8hFnbakQhXMC4N2GaqeHDSubSM3tWNbmDhlW8\\\u0022,\\\u0022iv\\\u0022:\\\u0022ivvegUZpDS4pFslqxOzlSA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e0c389d35afe1df2e447969204c153c7dbb99fd4c3c9a65b2b88843990dd49b0\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022jwGCJkuURtryzFTwwbsaFRE3HAddrxWjupXI8bvMUxQFqgO7W\\\\\\\/\\\\\\\/FfN8R\\\\\\\/8dZW0jorJYE5kDSgBfopA7lj2482pCpXO36KXZcKCA4urGNF1\\\\\\\/a31qiBRoIOAMgCRNu\\\\\\\/+AUkh6gp8JJcTemMgIh00u3\\\\\\\/Yd+HNhawsue+LGW9HOnSrLN5i0MXTfZZEN\\\\\\\/XMx6PwGaGScS0QaF\\\\\\\/oKaRPU9tE7TYBrRcaj1QkRXsXEMmDejU1H6oA4wJCxl3dP0XttAw+myYanDzk8bp6S448Yo0kYtjnYFuopjt3383rrohG1gB+K1Gi6XSmChCUWiC7QsZ+PHID1mBZz2weugATMuNmDsirXt7kLDxQ8CrsRUByMxbyy2GF3ohJnB3HYGfxKLr2dYxsjzoeqtN\\\\\\\/Ky6\\\\\\\/Co0BDDkEpgApCpoxzDCDCkt88fj80n0zNhrN3KV8x7bKQum+7aTqXw9DvtoiTsjfHy3zbUFeCfsH\\\\\\\/g83kIkY1JvLzD2OB2RvHiZXaIwAnkXdFJvukP56b\\\\\\\/mOVSD+WZMvU44ROAuwdLeMy1PvyvYNlUwYixUGWeQnDLkU3r8eItYuk8gPk4owcQcQ6JxqgREzAm\\\\\\\/+LMd3LEdLiZc\\\\\\\/A\\\\\\\/AZwsIQ++nQ+4N8wSSImigk1fuEHLB1nvPeKuWbVNNOtmkDQt5WLXorrA5iN0CkhOJmdjV3vFKh++twaIdGfv56SJ7Jl+D4OG+o8OqDK6D\\\\\\\/Tjn2Dzt0EX2ip57edoXjAR6AS9MbSHAjiuI4885x4HguGd5c42iRxuuSBKq\\\\\\\/yR08br3sdAlPn94fup3pNhbvdIxesXZxYuBVOn0dnSn7577LwweS1jr3brkWOPHpo3eijGWY2Y3EQ9zQSVU2228ezz9MYGTQkpVbWfp55Z3OCAbapMw1Brflm9McyAJlKbraAGujYCdXUNz13uQ0hWm3OMqZ93xM2X\\\\\\\/18yJqzWsC8KXBroNncC+meXo0OVfTHFCbO++QyeHJ1DKN9oS0O4x35u4iyJgAf3rw3Uid5P8w8WHDoSgNTZW5XZ8fULDkGxxj3tGK1dCHMR5KVOMSpSqPEPHgphW6Q9wZbZPusM6L63oZ73lKEGPTPstjZJGFq\\\\\\\/qjmdYNtTBnv+2gGmgmm+BxoVH2CoSSu3MI96n6teL1DnC\\\\\\\/kG34iKDx\\\\\\\/sXZ4UD1d9D84D4DIYzdfKTpMN8fOOD5wbHFyip8d3gpZWI085YSQSfS29fEeRzfYoMwrVtApyOeJKWCa9EJFdkTfByfQV2W+DK3yscKvWl1NdKLPFQRL0AERHFbEn5fVt1Pvmp1Qr0eKFANTswsiPpPn6oc92Vx0MdTFN7X4p3KussbAtdP4KpnSxQlO2bfsaXEovl6QPFB+L1td8SjCPyxd3+GlWaqC3LxNBJ23+aUfX39SMk7W6rLQRyiR5HdRD\\\\\\\/7oeJbdLVbv2u6zrBZPFjk72MO\\\\\\\/v1AWBWUdrtZqK5f4bQwnaJIl1HGxcB8io9pynxlo+xSgmu9CyaPPOZekG68jX2zdLlprXtyZudUi+nvkKkEeXH\\\\\\\/sWmGdscSLnSDBVMdwT2zMCFiblRofpWka0Abp1oFdjFem2SdB+3qa12TgqgsTsR5SKqe10hsuZuXJ1eND0fxFag8TdoF8MebsNGPoie9evxPWPUsU2vKc\\\\\\\/+T7zvh3vHkMYs0VXYOC\\\\\\\/cIg5MP44oH6qs8D8jwJT6NKRAah3sQ2kkOwQ1GBge+3fufu+GiN3yrL2DUtftrKkvxoe1mgxUpjpYNHMIJYHtfwJoC5NHD8P0i9L25jF0yItLafqL\\\\\\\/AnPG2znDZjfk3XBjgk9VhF0w4eaQ43bmTaFNpzX1uZhq1pgBEXlbwvL1BeGb\\\\\\\/s0hjKosxTiOiXV7PJywzhxCL0RuJ0UWC+g9nShfht+dFOfdfu0V8Z7501rqimLFPt4vejM0zGxKNW+K70yBh4CfIZIpgFO+bTJMyHwtRlyc\\\\\\\/q\\\\\\\/N2aI7zDt345qxgDvmWwIlc6NFjVV6lede46\\\\\\\/Vd7TX34pxFyWQH3vPDCpFc0xqQ7acJH9o3ARM\\\\\\\/UYpNLEwZi696R7yQgvdn1Ykl+BWqZ8ja2A\\\\\\\/TZ0W7hENMh9ONXyHaNqpEjv7SRceIqVpDCbPsENgWwBH7lfkL9BuZWwg1RLAhbgeZJ3gacwR0rgi+UpWHG+BrJ8SESNaqK923FtuyXaK2cNT3n24b54RyfKbr0ZkcGjyQFdGMd08QCnLHgedfoIXJZ17494MnsvGFoe70bw+ruhpWX9k+xc7TYXU9ElCQKSPtwNFcn8kobDqjEogXGhPLtnakVUjq1t22i\\\\\\\/9Um7CrCzXVhfaK61GmIEuhLfB\\\\\\\/whck7ThXYMcDJiBDpoekFL1i+nPmvclWOGVZvwpd6\\\\\\\/beBzYaWu57UV8tVXICDCYgRhZtLWVsTPdtHTDFSBVG4kYO26nTSBibov3afTpBny9oahcb+rxnmmdd64X1\\\\\\\/FMnEQImCabZiZfwwe6SC9ahsrIMuo0YYh4luBDPM3z++v2JoYxwRtlfALefkIRJ0lg6rlotrW+Ze8Uy1m5AIcl87JgpFElN19vcjbEdCq35FdoUTHzM\\\\\\\/WHwAx5f\\\\\\\/UvOTDnoX73gwONtUJJI\\\\\\\/jqolF2rWQvE8qoIXzS+aWXDyT8UZ4DCAA6qytEnyEwwxU6m+hoXrF8+YBvdX6I2FxQhFD50VtI2GrE6lcNQU4C\\\\\\\/Xl6tuZIBHzz\\\\\\\/4tEGQXx55AjRyUSIIW0Wah4BHFvQEbMmgNw2Hscqi8JWSHJG+qqQaF3Ftg87AQUPRp3TmdrqP4wwhNp12K+EzlxfDYYDWpfrtxE4vOxzcgc\\\\\\\/54UjXLfeQusTT9W6wHquX1ilkhY+an5uWaqjM9wxSRrc4Zs\\\\\\\/ASKIzUItcCaRJFqNlgmMlEeVBs2Xj5QDILThK90z7wy9MLBMjbrzZfZIvgK7tpwuNoqi5\\\\\\\/SeD6VB66bKTAIDBY\\\\\\\/4KHYrZSoC+6Ria1bdkKrnqgonGFGcO2gjzFgdM9eheTO7+JqKOd2lsISpxP9ghY0oVxWuLD6+2\\\\\\\/toss3jTpN6FyOj6DTxIgpaNnConmS51JsaVS74hMpDzCLPQhqvVkSiMHjVw\\\\\\\/ljMIlaH1VGVCUX0QTXE8dKXxq5hmVvL3r2rXJNHDI89JmwkBBDoBRFzfXP60d1zARRfzJGsqgTNGKOI4hU77BSYAUlOpKKgS5HwEKuO+Sw\\\\\\\/oY5UNfQky2qekVnPPP7SQMYjofH3iJCUnzUq4Xv\\\\\\\/UEmu2ODe4Y1tLqeC6oVTH0to9gQ35Wwh6nVFp+1gZHlB\\\\\\\/+aTALpjgYngSN1NLrOHfU3jjLbTfawXfXfRgdS0kA71PwI1llpzA0aoW2rS\\\\\\\/2C+kygtuzNMrmL6XrJMNBDhstIH4DyaJZZx71jDwPcR3skcNr5qqTe1sIme6yQ0KCHNfl7tlAGMRIGQUor\\\\\\\/fV90d6ndgNBKberr3KobUWQGUASnZD+\\\\\\\/CgjgVYmjVCLDmrxLZ5RodmPW+J3\\\\\\\/Lx2pEReS1B3upYaJTcepjCPVH5wEtg04PhpWuI0dB0OLpAJx3bnXzLmyJ0QKZ8NWYIZ33DhzYhTHNbWzXqduQZAks5L22gKsVSHtBs7DlPX4kklc0NT323U2Ww1am7LTDQPzQW31tKR1gmyRHFHS5eWPqioVm9Knwz53+7MC4sxpwEtuvolgLp+8qzcJ3Xmm23LaQTcwquPlt\\\\\\\/HSJa6ZJ4US1Cp3QTHd\\\\\\\/u3MakUgw\\\\\\\/A++MApiNW\\\\\\\/UMm4bWwohqHBf1dyn0KMOvEgeBkh5ILoO86oXsw5nxqzfgLAY4PIO\\\\\\\/HSrWbEu+p3SkDX9fI12qShg5MdHNZ1ZjMaIM9bhdY9PYoMn\\\\\\\/QF1ZOk2wu8hLTBvjgVTAfZu7RugZ59FEV2WaMjk5u5HetHYcSvHELHLpIWPsGCkGOiSsLUZX3KLU3Q0HKjUL7HFJlm8X51PbiwcZ5iFy4VnKeGWAg4jMirZa19r5nuWJ++v0\\\\\\\/1iATEXvC1iI+aaxMdKzhp4YMTYCUVXGSAdgbjjZqIQkU\\\\\\\/IP5LDhn39v1DJXa8wOArN7yPrFDCwHY7ks84kTd0n65Fp3GthF599go1fFpLoFxutgWk8fyDj2Hhc+9gE8f\\\\\\\/9tpy+\\\\\\\/FyB1WCZQYWPu+4RrjHfM\\\\\\\/RuhyHgGU69BmTt2s7xEwvwxft3c7H2+jrOriRvDbVpFx5t4I0AQGCaqKzBFvq9jUvQ7cMB5OUyepNhkmE\\\\\\\/5dI72eANwwxdmce2hEkNgSnS5mrKZrolra6n2iRukprjf7+rJaEz0ipgbPZ56l15BwQaq\\\\\\\/wCEESKJqKFWZSl59TcLR6DLMkgczbYjb+EY68juf7aPWQJHg8g349mN1HUKv+kfqpzbToDfa+\\\\\\\/6CkLLP\\\\\\\/OX+4G1oJ4+bWjg2hqoonSQ7spbrkpXJH25PPiHIBOYg\\\\\\\/SSmkQB\\\\\\\/zr4e8CyRdG7QlMaF7WtjlHauV\\\\\\\/efaAYBI39w1JvPgu2NuBF9XajoPdIxP1G26EhNW0Ex9mSY+5rQzMikuFuS1mBovcRPGkuGbqzto\\\\\\\/VBFRRoWXpLdsCYh\\\\\\\/Tk3O48DyEaYxzXUr4e2LnGT8xIJceHheZCUTpAIAR2MBe5ZBCLOTdS6hGfu\\\\\\\/gwwQg+OVPTexb1N5+i8XURaPahp+tpS5cvfmFG+hfr9dcyISrZIO\\\\\\\/Aia6Ofe7BeT5EkNIES+G3G8Cp4HADi9XtANM7oOiLDfBuvJk71ABj8rk082xRJ2QmmHk382d1ab1IGFo6iJSL7MUSPwYiSr9pMG+R4ySNHO3ohIc+VwGFlmU23hQs12czu+CM3mEIkYIaB5Ns3XMcEJsGteyhQPi6slQ+UMBrpkISpkc9jXYHFeH8Lanl\\\\\\\/XG5BVg7lY\\\\\\\/9GiKmkuoqWi4BFVUymoU\\\\\\\/eM2nXj74dvFVbEazJaWkSq6hIkVx+NtuHebR6Ac6nmJEE9MIl3iBGKC95JQSKCpm9Xt+Sgbuz3dHx9rLWD6tSfTgWTbSJS4RONwAJf4mJNVTd1bJRXQQNzpPQ9Y\\\\\\\/bJ6FZzOWDIYSy4nxbNOcft4UZ5mYWuEJfVHWV7GYTNN6Z5QHTxHaJgidYgOIfQExreWDIuZSEmcsp\\\\\\\/EbEl\\\\\\\/0+OTcRnuj818kJq2rndgIhzUZkgH4oR1GWWQQbD9KTUtsk7aq9uRJOjkZURqgAwoFTundS6RkOLwWtorPfsYVgo0X+VqYWJijg2PkPmXgqrIhp4f7ZaCx+0dyePTHar8eo+hGCVE1AKyTA04dVUEHpndB7QiL6\\\\\\\/Tpu4HO8XpWrmOJsUqT\\\\\\\/pTVszuBlCBpUc8byJA4E13et15l9\\\\\\\/cDuCI\\\\\\\/rEyScRkYf03rktEYj9YEdtLXt\\\\\\\/UgWPpDBNSqdpC0+Hq6cDCdt1yxjX4ZYvcZh6jySlm3Oenka7MmO1l9jpiyuIgXogHTWhzwBrK0UzU8DMDjBH\\\\\\\/QGvB9e0n7CH\\\\\\\/pSd4j7oEXapuN1bTjF223WkNGc80XkiVexdOHXUP3l+tTOz1vpeyiw5+BW8R\\\\\\\/BazREelwcEOE6+Q+o+apyz5fJEUwSV6XbPC99dqBuQXII0ZIS78LrDg+osiCgWTKcKRKo37tSKTxkfM+8iKJX8Xlut77Mxfx9KOeVqTm2oD6cMT75EWus+3iweX9cbNaJ\\\\\\\/fnqGWcHlrcVPfN4rwiir4T3GIh1S5+A1yzHm8FxdEsER9sjf1BYW99crXdxbuSlCiuqt1k0FFZzM2jAfZk9PzeKqTHhFd8gzXiuEF+UWysyVmdpG8haEe84s8rGN4qPj9elCFvzkdocZGs\\\\\\\/6Id1fje1LMofTpdtx+7UQvcfVpCf\\\\\\\/HXptI5qG05R+vM\\\\\\\/v+MGnLiwLPdE1lM8rvbSwJtAqkbmL0lOWVtGN3fzADogV7w3v2YYnGGg9SisE\\\\\\\/PUBMlndfzYLaSi13wv5Wr0WnbrpwvhFIwOwaC7VdLRO8dxuLx4m4wFThgAbPbx2VokiLRGCU9c9\\\\\\\/\\\\\\\/PptJL19SDUlXDwlRpi0dCs+sT8sz6dSvLi5EOko5o0yEEeeb\\\\\\\/MgZAAS6MaMx4SSjhazGWN10KyD+7Xatdyn5r1n49zwHzR901E0oByzr9\\\\\\\/7JqqNDRYc4O+nSy8086sEnzAENDdSlTXAk\\\\\\\/WYpgRMB\\\\\\\/Ql9GKoYurH58aBsTayjXEqRK2wt5RtxK+AeEqex7AK+vkjO+4HHTuD+6EjUT2CsyfD+Pm0PCqCtcDLSc8Q8fUq3WuBWXy\\\\\\\/5T1++\\\\\\\/j564d6oo7xfhBLgdqprj0NwwiCK1xRFIaoxdi04+dA3mp2lN+xwIsQM+hjnVsGy7YeZIGzktXalukUPaz8jZLpy\\\\\\\/haIgtGxAaIALta4tj6FLMvy1L2A+8UwMAMyaTsG8k6GmxrhUXbNoptt7I+R1UVxso2\\\\\\\/hEWnKtiXpuTgoIvA1DdTXY2IOAXumGZrBIyZa6l8eBpKZpBFYcL6hmfGiP+UdgcOTBG05SboChmgx\\\\\\\/1wuX9noHbuvWxovrhAZTYvkh4zEBnFVbsQLRyTmGXmQxK3FA3\\\\\\\/w0YBg5vWJlBrS6eKwaEVtg3TKvyNBFEngndshnI6sfeaSxHgLZDNWdaAMoGe6WYGZdv0uN+m7+ZTSehNgKBBkFCI3QOMw1XWo13Kp4pPVqNwicVocKbhf1G+zkYG\\\\\\\/Sj7ZWXbCU3B00cHTO83MNnMykS+mKIEdw0\\\\\\\/T3QB3vtAOdcmmZ64uhQA1wGyMqsajzt0IiQEyLpb\\\\\\\/VwnBGA5W\\\\\\\/HqRILJ\\\\\\\/aJEZm+IVBNhyIUte9RAPuv4MarJxcTb+REBKM3vuT38oah4Kbfwp19ZXkJ3+y0Qt89kRe3oEXjaMvCRCu5iJYt9mhiLYX7iS9wByGe17WXlSHh+dV1mnLJpdt+rCyvcoUm0bIHswshW6\\\\\\\/b1CmYk8V3BSBMc9DWvXqIrQxQu2Nx++v+YgEK03uS3PTmlG2OoCoAm6md8vxlD9\\\\\\\/oWEn2TNEg6M1y6M9Qvp6zo2OHVAZ4NiQ2XyGJ02G\\\\\\\/gWArfaNjIZ6Jea0qrbKLRDHPOj6u8JVllip8dmOXR1yQms4fFavq4fJH1LdEzp7agh3rMK9Ll8jaRUXoUChyDeVadF0nXtJDDNFINVdHnCVESf8pHvAOunHUTD46Sw4lCHZl+Qf8Eqpa4HhLILh8jqOmJs7hjLd+dLSWn8jWPdNe9WBbG\\\\\\\/iIDf0jqGQvgy\\\\\\\/QpKvcbAbO7oNhJKf0Y1yhr0BQbNQTfCJDU4+dVmb10izhm10vKECvZUXT5XjC0JZ5rP1jt+YmeoNTS0ocYhNGts+wRY9TIbZ9GY58Nh3j+RX3Q+UyIkj9788+\\\\\\\/np+w90OxVGJnbnHdu+k6Pp2hYJzdxx6H5rPc7SFwsRW9mx9VB4utjMyOXPn9LTtzKlQrp+B\\\\\\\/L13Bm+MP7jwZUgvs4JMYr+Znxbm7\\\\\\\/LztCOp2Wl0w1NB7zgEAG8cMkNxk1sHq0vMeCDXJUT3EkPbVwy1k8XSAAEthVTLHlU2\\\\\\\/G8NN8cM0ue5JFZk64wSChBKN5VMO\\\\\\\/ako\\\\\\\/8JRD0MT4sa+Z9dqdmz9rZzdsaLG4T5DzQvqPvNdShEC5fgV2loJWyY\\\\\\\/kpa9o2x6ywncb5+\\\\\\\/WQazqjlFtO9Hxo0LGviPtWsJI2sFwyXr65fsIItgfy3I\\\\\\\/8eojjRKn154nPsW0v3alBBaqViKW+1FjfZvb6ZnrFGhlt4rDlpNVFXVzS0MW6S6ISI+QBn\\\\\\\/V\\\\\\\/4GCPMPOvmO4FqF5SW5zL+4l46Nit4lxlY4YbNj4+GamcKCMgDNkuh4bhUXKzbk6cfhqQWreXU\\\\\\\/KAW8bn01CqfsSbS63WD0iu\\\\\\\/0\\\\\\\/LEILDY8tUrcLf6juiSjWDGEMRl8BDyzTCu0HvCOsDqiFVfOA+HDnMVlPLl5fV9yDMmWTKy+faHtwJJ+w3Xcl1I7DlUgxf3GNkVx+xpAkJuEh\\\\\\\/nwOFUVXoZTGu5iwO6f8+TyYyodMWhV7cfF0Skl+YxXcNCBYxV59VF\\\\\\\/vhQ\\\\\\\/XN9uUbgLYwoU2pKhhByICPHemflSWecAS6NClF33l2veT5KYsAhtL\\\\\\\/J23xw5dU0E99MUkE6fJsh7a0A9p\\\\\\\/ip28m\\\\\\\/JxYEnzK1c\\\\\\\/X6+RN5m8SQIioL9lrsgjdmAZFd+fKi5dBa1G3ZmipCc0Q0MZqZZR0FEevvGbI\\\\\\\/m0n+O56OQOg\\\\\\\/dpM2jpAhF87DB\\\\\\\/2cJBIlgbXyomdyHyJ7TJogatINepg2ZRY0dzxm1G3pi92H4RGtuLRZnteiefAtH4rmAokmWbzPSoLMHudWQpJvf790F2JLKqAl3BiG6fa0RW0raX+SCj\\\\\\\/+FPcuKdl2pLFolTNG0QQHOxJi3Db4huKY\\\\\\\/wTHa6VZqPlN9Ee3uNvSSqdVy1RFEP1Fy3cGEDEEU9sAkKMDEC36uXglzjeXUEbYOvFoQ+nslBFR0H\\\\\\\/kyA06scB03CzkMO33bO3Ng6WMUgrSjIRMgZlV6qSidpgEO83x8xBvICb5Fzpk6HdFZglAwjdNWEIqRIxEeURUuOrT+aMDJkMJ+dX1DMxS2OYREHf1pn55PmXcgUs+wboadgcitFTzplim95J29f27FdKRP\\\\\\\/NfHHL15heR7Jv85UTZEd1bSobruBvsWcP+xWXZSTSISCCn8fJ5C4haXuwxeUj3tfS4ligBPry3cbThTKlXoT3wGxjsnYapgqfbOMsiYDjKbW4CTQVdspjfajSQljHEakYzzQmL3vpNCkfbzLMfq81y4+MqO\\\\\\\/THuAKgXm\\\\\\\/fHa5RJmmbJ882DneTwVaCu68dTfhQItYWutOIFej05X+L4vfWUjzJlv1GGLnn8hrxE\\\\\\\/\\\\\\\/rp3682yOxArcjpENw0LmIR5SxBeOQnVjd2SwIbuUApiwh8TkkpeKLt\\\\\\\/hgqeTy2UbThTRoseCKUky9qQ9E1E6DRwE8nUuoKeIHJ7kPC873fvS\\\\\\\/B9e4tXvls7k0VUDc6YEa0Ipv0NhkyEeoMoLbBMYk+mAWuJzifJVFfuotJvyFTlxT2+FNb7VItjwyu\\\\\\\/PaSIUv6AfgQ0tjl6v9jDQp13\\\\\\\/r+QCjcb1Y6Ce47lQeU98xO4OttpQnw6N8XGkjq2SYjzfZd17tQNnOCn5Rx1L9iSIX5L7vLfF8j0BlU2TjF8X\\\\\\\/nhDT14fOAEKg07OeuQCbesDsEyD5unttO95cr1cx+crD+PWbZ6EqRBhoDf7RmNTT5xNv39MNbPtC6G7bmGY7JRK1O8SzTy2uFR+\\\\\\\/lBFYay62oaUXEI6eOjcVnzUogRJdtwlnXNTIuU2bjL16kMF1uAlSSSYldPqszPL4ngKQv9g3zU1RKkTmDXlZqL2xGBM5Q7GexxJLVLK5A2y\\\\\\\/j1VEEJLB6ur0lMNTbbwKlIEdmbAX9Y\\\\\\\/03acj\\\\\\\/GNvk77BpCWxpTvwlFT2gFjSPp4rVf+vYW9GTJmXvIvem5\\\\\\\/l7TDq9niczgygRmo6zwf53VH4ChLOPvAxt0w42zU0JupwoOawIyhiy7K76Y+zwesDcTzjGS5o6GDYH2cidCH6LHyXKBN3L6HI6End34BL8WT6Ta5Xm3nEJDhZOoTuwJxhhKRu9dKfhEgUygjzbj74HIlVuvg9\\\\\\\/ElUbEN1BXlwu2VPTMHF64atQtkn3HWMBo736k\\\\\\\/BFs5VoGiGbZxSJvOCjvCk8HDQyHwGfafj+gj4h9FMBCGCWTMS6irAuYGUBPpdXF0TWRNTifm+MBTWX3QJQtoa5sFwhQzt16HnGzs1qGa7Ip8nbFz6GTr2ebVExzXeVw8RksOPM5WZig0+d1poXzUTMk1BFE1hoQux\\\\\\\/dRtFA63BEjfFyJPdFjaB\\\\\\\/HenWbnNfBgGvKui5ItXaBmFMtPi7gGlw4hpcbS03hJkCugRAaeTGSM7\\\\\\\/FntO+0y7gpx7iQNqjT+0pZDAzLTjoUL04zMUqgOQzmQDHcG214j23XlgDKJHkQrb9BNXQVyCffkD5xLkC3bm0u8By5EEAC0bZvWnZ76Sul95fzUBZPlF0saD5DCdXNuonCHtWB4yPEKFYBUnfJL3HaFcSrZa3ebC5YFpfhoDyDBtzyGzXHTkDRgReVARenMCzJrw07DdbrpSGZzvaqlxrMwR0QuLNCxiEppjCHIEviZBk0gmhS\\\\\\\/1ElFTrQXEvotg2AEhTDy2Xr\\\\\\\/9T9f+hWg\\\\\\\/hLIwJJy6inxzKnN3m\\\\\\\/xHZE54UL6CEMg7x8iXgW6aPwt54R7L4+Vy+PEtjFfcyzMFbLt\\\\\\\/x6E4lgfvdXRcV98nA1\\\\\\\/\\\\\\\/kgJFJTT1pdSMoAejMouV22SxOml+Va+5maPJVM5wgBKwB\\\\\\\/Kbed21wlcpeE2uUfVUu8FfvDE9z4svYmbjovLyFyIyqCQ6CPUMH7MjjR3\\\\\\\/azdxXhLaREcfN9bSpPyO2KNNdjsPQcAzbsR7PeBqWRfXSR030vsqDH+YST++O9qurGeDGvRtemkZbAaxp2XFik2hCoftuawTBU3VBj70gbIoJXwC6ZNg4FlqhvS25\\\\\\\/hYUmXyKdqJrsoTwNboEXECBiOn3BSkWHr7Lx76Hs5wXN+7BU0mQYHHTlLhuJDDgHzgRSRPn9yp2IBWjfcfgJRXsjSFEeJeCSEko5U0mhrTIgOf7nbkgKisfFN+gMnq8G+y\\\\\\\/8EB9hE+VMSSL3TIgo8z8THg\\\\\\\/r\\\\\\\/M2lRbQtaiDb5CFOgRwfiESZJmMNVwD+YrQWlbtDlbT7xOg\\\\\\\/cxqqFm5rKJKNe1Qn1t8lNo1cQjGir3WttzcEbbuh1MNmDbxouuy9FhrSt80p+LFc9K3QEkMvRbdmHY6Jzh0FiMNgHca5M1BMHf2Pbd3WQ1fEYYp8EiYYAoRowoAef\\\\\\\/WRpml8Y73Zy9+UbmTjNbFfb\\\\\\\/jU6FuN5+C0YNNmRsnVRrMmt+3u1n8vtzjyelIuQRxAMzzw\\\\\\\/cdqgGcWJqfMq+MwNUNGf3mCz+wCtO0UUolAdFbDb2gpfIi+YzoCbrwQXOr44leqDMeWQdi9jJH53O2MhRvbL9HSmlf0rfZpz0IUqtkBOgm+oSkZx6p5fHcRK0cQcQPH3MquR2XDdHtgCOGlvFu6odrXsuE4v7e0er5sA3ucXJ4brxOxS3YEvkFsLmiecFI7ug5tve7T8qckZipEW\\\\\\\/POBZeGsLx2\\\\\\\/Gf2XZHWBn9+rWYOGvmnZzBnFaxWfT9wKwBMK1VOM5QlUTccPeezRQXozLqJukwEDpxudDScHksbw4w\\\\\\\/alCf10b\\\\\\\/ozc9SHtxtp2Wriddk+89Cti9u0Gud+ZMSf423w5eDw8vBRDQH51ExHOvdOzq6JqSKnMASyX\\\\\\\/SoaBY9kW0wmj2ZcrmMUnQvmEpG1J6B2cA2bxgGITrWV2x3i5aQIy7MzsMgV7IhdimIv61uYXy7+q9\\\\\\\/0FlJ4PgZOUHhVHw68i8ypdRjctowVuJ0Jjs\\\\\\\/I5\\\\\\\/x7C4FIkEhT6fi5YsGzD2m1LJLIXSwmqk1GsQpT1+WkZFGmWypyNKhNu\\\\\\\/XLOXp9nBKyNffpuQiLoVPQJDGxeOosbehjlOYE7IHXLUInx+JkAZpGOZTcPojLVmGxwiMe7hkQuRiKCusGS+mn\\\\\\\/diUq5D6MKCD27GU2O7UQc5E5VN3k2PkdEKDKhVqqPCp2\\\\\\\/WJgo2GtlhqENWzrT+cus\\\\\\\/ElwV6GTTy6+KWc+pZyLf6pqSMIzkxfufwUIef8TSwJ+\\\\\\\/OapUzW3kaC3pee3\\\\\\\/gTwqMKIKCnzFxaSDkx9MoYjeZN0iz2HSVZDU\\\\\\\/U4Se1o+YSCBAzyeHcRytadfXSastjL95AIopRmLuh2PHM5xhPwUfvNU95XURsMZO99tQQDTboivMm+ra17YOzAwZ09VGg5bVwgu+ZWOW3AxMIVJRIhEy55ddRpkaLezWdZtL+YkxDRnNPEwm3Ahqsa1ZnbGmnFUWoxbABBwe41aQy6BNa5ugzsFZDkWaqGDAerVIrHcaLXMb\\\\\\\/9kJsDeNE3+VYCICAam+\\\\\\\/lJPdTwlnCsQAYC9hgItVGsLkGOyJMCa6XHeC8TPe9QGhIM4XKgokmu0Px5QTaqT1sgUy6EMXGa6HRtb4yAOTqQYpewGg4hjK\\\\\\\/p4In7EyYQumWwyc5\\\\\\\/tuahcbztRBil79U2sEgb6K1KafEJqCDHtP0cBfh9xYzle\\\\\\\/YMqhZuJk0RNUvD2B\\\\\\\/WITWyeFgDX1JcHpCu3KzGXusNuOVGdHKaqT0JZSuE0ePeBPWLPFgQhGJy4yJQyR7fGt7c1Kny5D4wPNo258vUs6MWM+oIOG1OyZHAfH2Z67PxYSiyzMwpn6myHwNde6r6wNE4+vTYj1No7BbaWvB7UtAYE2Bu06\\\\\\\/MVZqnG2xpUpUEDbUwCqIvbgf0+7zJqBZF98vYrhkJCdBwQMIXLEAXkijynz9lM8BMCn4JoQXnwhW\\\\\\\/NfbSU7SfoeXPqwDrELKCCg4KL0gOByXZySZfx0nFAuaiCc+VxElTociNxPr5njuIwGODk+6GLDkVtW3XoaBA\\\\\\\/+ms59Bcoz6PX8mioASFTLrNsn7kKa9ELULiC4VzeAxwoub+G4ig4r\\\\\\\/+AzUCMAAWiuwOIcWuv7Fk1HIeZr1biO0GsnvJ6yMUcLS7mdbXYS2TPV08kOFdSRUp+yJ52dlhlf4m7jVegZsjkz2PrfABbSCFtB1MJJBXA9mu6BHoUeBIZMyb6RisTnN2gPCt4p4WmRUCClsbPaStV1SieDO\\\\\\\/F\\\\\\\/PiOcD3TWzBDUrM5Iq\\\\\\\/CSu6jT+zQ8xMFm5LEdwkNX41yKNfPUGriWBEzlX8SZhAm2pt29kzxtm9\\\\\\\/ae4tKU6WLLzfEc7Y6qbTXuRYzTmcuraLeq3mKnYDQM8ij6UOEvU312kdLCuhrdXBtsD53hLjd30mKTjOZD7ua3vbZ0OAggtbXLZYhijR5WQmGn7IWiv0stbaziT0HW8EoIFtv4rcq9RIC6+lo7Mye+QzsnKq+sLTiODGbXUCWBHcGLFpEhIpTwBDJvibi6H\\\\\\\/QFfousLOsMmPKqOPyWTvV6DhmjvmvVh+hyQj6OYfbYv766z9TD9r7q5Z1R09hRF4lZL1FEzF2UorL8SLPKCLcGa\\\\\\\/AO6vzcFZkzL3vM+uM0mrdYVcJe5jVqM4SUH51Lc9gtU8xiDdCTBuc8sdx3GYXvmcMk+sTwA16z\\\\\\\/RAjuDp2cbHWYW0w6V\\\\\\\/m+CnnN1cq2gY83bD3uQysxZCz3\\\\\\\/MeOTUbE0owc5CIuxn7iX4KsLitminC4r\\\\\\\/ugR1nKBBrRoUm9Zlylf3aMZwdgklxSAXNYrLldD62q8UuWyOuwsnsBfzJe1BLNrGPqRrah8lViFXrp+BkfwNWtmJa5ZcZFrApP+2cuSF36AZv6dIVuB1Gfn33lfWLpQfP5ERxtM5MbiJTRc4PC0dMMV6TEzOxhjOiVYcyNFVfs6tDD6LMb3i\\\\\\\/IYt0o9hakoEi6Uhx3bCjSsy6bDRNE02Zqy9MhYyXKFTvUJ0Ft7xRMqlpVq1ZmZDnL8h4xpPFDZJeormddZeRatSD7a2VCyMao6\\\\\\\/5\\\\\\\/ha+un6Oy4GrfmK3v00CgXQOtVX6wG5XkT0l2rbRpvA4iGllmGSk1kolwhRdsxJS8sOZwc1vgSKUBXTsP16sSCd+u4pZEAsvXiWRY52C6a5h2CJVCsfVxxW83YxO+eFvwOsuNIJzd1BV7R1dqxpoAdB3\\\\\\\/SHB1znrNhxQkOn4WBrmz5x4GTfQTImdj9GiwzUl7o+aZEo16XPHWf1ErYGm2Mup8tpgNdw4bnbM9p5KefUX3rz1jPKWvrLrCiK5AbrxDeaBsFV\\\\\\\/WE9fWLp9EB7S6pCE20gFpjFtuhgz37OL\\\\\\\/M2TaPYIequSQVXHbNROqGgejE+x9x1GsvFjNS8xmy2uuwW1GP8721O4x+IqVx3kBK2\\\\\\\/JSa3GtqBto+luBnMf7Y2v4PDpfnvBhqLHFC64QuphyUk1htyQxM4kpxvn0z0IIBigcY4yHwZQ1LN24zoPbXN8MyhowhTPEX0wNEb8lIm5wRMI3yYKfqfyX\\\\\\\/yOA888gZDwHxcE0LiH+QhsalUKR6hi\\\\\\\/fERqXIQkQrQikoj\\\\\\\/OS5+Z3YzQJX2pBFEozb6S7FFZyr84aaQvWke3AUinnvVR+i2GZWteI1TsWM68de\\\\\\\/H5vPHfqKviEDR38mwN\\\\\\\/CCfGKLhMo46EoNKtw0BwaY6GviKoUZRrGIKLoll43XfoFNktOEgYy074RtalyIxPxoc16\\\\\\\/shNiH1nL\\\\\\\/7jrIE34NCi5v1vCg5W6I6lmqq3uhXL1KIQZvvPay5h4N2f7pb5B4yXRMBCDgjwxOkblRwozCqZ36xFBM0MCqE1RltslLETKCkM6nNm4jA3\\\\\\\/Ip3L1Ynh\\\\\\\/bESTGyCBw8vkENbM2\\\\\\\/d8ZAslNKeNvdepHMaIpRJ4h+1y5Iz\\\\\\\/JqNFqa\\\\\\\/jYtfQARty+R4Lm\\\\\\\/AdU\\\\\\\/Y59VgukvjJdeS5+v5HNUMLjPWd608CowdEz4tfZcS\\\\\\\/qFK\\\\\\\/\\\\\\\/dexUAqEH2OPldQuIs1c1TSaMeXk9QvG6pJoBFcSyC97cIx7m+ldIucZA1Jv8ZBEJZM6JXhGVyiWwxUUEbZoBpfsPYAWf7dJu0DerF7HADg+F\\\\\\\/hLlEDOqb3mRSO2S\\\\\\\/3fVbwAktO+SApkIskQYIdpBxQNd2Nni85Am3WPGHlYAYf0OzR5+sMCWl4Cu\\\\\\\/u3Gd3VKY2Ti0FRvuIEowprvUlzTR+Exv2CrR+pPRMIIPfFZkyPnDAdzmVve\\\\\\\/6Ykfu\\\\\\\/S7DgOV3pzXp\\\\\\\/pg71lz1N7OAdisPWpldPDKCj1GnYJc7KtmTKuzh1gdUgTqQ61w8HZ1sVU+BFIYtDIo1md9S44SJTVe11CTgPTwfLidc6gE7cjziYfmQFI7UDT4ag9Q4ooKN8MItO95fN\\\\\\\/kflO0Mh8VGvoB5R+NXGj3BtV9Y+sxaQD9HNhzplsn1FC8kh\\\\\\\/yEb3P7VBwCKPkwkMqZ+SNXbr71vLBDLiEp10A7anzvl0xsmArl7g\\\\\\\/WfOinPdN5xQH8klIPpDQu65WchpAgchNR55RVkFciiFIkQidxbRpgPrbf9QKksxtiSsDJyKGH76ccdveWXu8PSEqgDe\\\\\\\/3D8uumRK48wlgTyhPObIJJDSWGrtTQaiqCinejYaRLCykxyqTfrhhwu93TL+a0Lem\\\\\\\/T8WiWx4z\\\\\\\/85O6LD\\\\\\\/DO8IZBTAKReEaewQswpFxe0Iu8U124XoiGrq0ZxLUsJab\\\\\\\/fWZB6PKReivYXLZFEcKm8PxqvUaDUlH17PlBEITI1y\\\\\\\/4Wx2krnsz1a6Y5nNIgR7XU3Z4PsaErMSe3gtjS\\\\\\\/pz27smKT2bJrIJkqIgmvM3etU0+2TtthOawtkltpnsmiFQekwc30+6jyukg2QlunSwJk+S7COV05FKJdYols5bOzOTtc3ILYNLT3EFipUK7DOMJWcFDJLK+kT22pevrFfXx\\\\\\\/EmDXXEHP6juseENKfELAqL6OEu1cIlJvVtBO6wbt8bOnnup7Loy3EiITVlNqmcnH7xuoiV7DvFn2rwfeKgb++h9ipvS+coA4jQukvd7609U8VMpacpae2n7X\\\\\\\/MfhZMXTHa5XTARBvneXlm5uh0RB0TtglqSyeAsXGgealHe3mje0iNbowFUU7z+h+2+x9njKaKvM3ZnZWCxwA976GUbpC7Zrr7fR\\\\\\\/o2v7EAuO+060yzEkk2ai4ywU75GYfzqQ7CGqHdUNhWeMhkxJlm02\\\\\\\/Zqa2sQ0GsuvL5dZUzqN\\\\\\\/7UgfYLiIlz2PH1Xd1UWJYZskezkwHs3NLNotkrPmr0zvPFUT9yAsS2TYKHnuLWnCVxk\\\\\\\/3JHrKY+bjXEPoVPImU3Qup3GWXN74P+suAgr\\\\\\\/vDvUQlSrVqWvnDvG0P8160AT7f5ftn6tiymD1vdgK7e2Qru3rOy3qTTSicXCFPjIxOV5snlid3p0LZSeaFbWcv8PnHt90ZMO0Btids+WEAh75L9PgD2m6k1Cdv\\\\\\\/Y4jQvznVgc\\\\\\\/k+2VgBIMR63hJ6i9zWdF06TKCAgYhHgts6rPT+6NZkpxkC9DD1RfnDyWpbTaomIjaUZ0qRtPzNG+wqKHQeOouwrdS000RvMyATIq2PUbmP+rV5LAreDzznqhgcyZRk411VxYSvYNuYZ2OHGsoZL9cuQDHsKS7VIrw0o1PbWsd04jVOr3PcNQLPSxHZon5gCGZtUcf6AxFxB1lmUtlCdUVjhwxkBMN2zaUmBT7Qn2AI52zryR+Ql4fcNX+T5ijY6XzQHx7H8iuDgLTm6o6GjRCxAnplrMIfcKDHzUIAQQDmqVveha+Vs2ZRIUwzu13NqHxj18IVhxqICmcsy1NQJSawHqpzNSoRb8ttBzt5G7h98vfSNk8jo4d0+9FM+AkEtj0yJC0HpSKC56zfm5URGwZuKAUkGm7Dw9lY\\\\\\\/34qXHLSVlP9NbYNEaRlokoN3DbdiZdMwCF9QbFzLS0+g2Zmr+kDAxklqBaGb0lnZ4mtWVOnjPcrA75Z8aNl1ux5ElI2g3FT3fY+Ce\\\\\\\/jDu3fjlBek9p\\\\\\\/Mv\\\\\\\/gsZlqdgdMx3M8rG7RW1qnh10WMxAtyqReBlWFHW\\\\\\\/gU0VbzH2JvLdHhLiFLKUAY2zGN2Jz\\\\\\\/226QChVt89UZ4GXlpabpYL5VOIbrzb5wn53XxGg5h7yi81Th3uggh1+f4naltSDNV7nNXU+jrMdnip5K\\\\\\\/zZjzOLCJvn8svy7KDw0iBmwEEXG3GCUSzkMlyfEy\\\\\\\/5Y2EukQWKA2UN\\\\\\\/PGWcP0ME\\\\\\\/MEk8uH9ozqYVw4VjznOzDYwIqX4XJMvQD8fY4QaJMdCGExVIkaB6DgzTEE40YwDvrSmL8Q4dyaMnwlmrubqnPmoE9rUCURGILF8fsib+uzywKxTZWpvzFHJcV8C1aZlj7+uW5rMLB5AZLFMVa098v9YC8HSVkWQyhNeQ2XsZO2ifbfzreHNLX7c5dw8wwzReDmdrrKX7cP\\\\\\\/8hHXUOIJG2l6E2J9HW6DMWRstCz3+v\\\\\\\/4uoCCB6SLge\\\\\\\/P3agVDrHmyr4T9YvatLj6Euyw2oPs+EurHjfp4D04vYSZd61iqRtbCZX9o6nSk1WLmN0kjMODoypzGWswd0FHdSxSOwTpW0EAGwT3c0SwbSCvVhTDhBmXxqcp4TJmXvZf9BhP+HoajTuToU8EQkAX3KIOAYGWQjMcDN7A63lurubly9HRyn8m9jp6eXpiaT2qJI07NF7Jy33\\\\\\\/tj1Zjgo9E05Y3LECvAOO4NfI4EPSzJZ362RlBwHV\\\\\\\/5uUFBsuPZJIcOzWTjfYn0UqqrbQNZwUYyBoY2USW4hSRoyPggKqFvn69yhPyZBQuNEL8yaMl0T7LhN8ZgZxf2bRDwNSt2Z7mvnmZ0kGlj\\\\\\\/X3RgWjllNcDQBmNcBll8POnkPk32kn0agucb0Yc3Iise\\\\\\\/h3uKpqTtia9S7Naio5vWCVozZu41bFxkHfUbptykuSkrHHuAFaB0VI5DAuKz4oSowbLxoUTIVchhnRNjXn\\\\\\\/4GcxzyLOVnvCcQ3cNOV9XdesvJFk\\\\\\\/dnxXlDUAlyuN8eXneZnwU1qmkCQQguZrwj6u5lhhCUQZAeJc+EpIJApLsmkhzryP2J5K+77JxAeath50ypk8cq+8Ll+5m4x8n5jqSEQMuCHSYugOGlDnxZupUVGoRo8Hky9qZWFPhy6CfjttoqWxckYPoQbA+AtbO1hWQCMPrLDEkNBLCl7L2Bs1CMP+P4trz47g8lTjwzOL3CIk2eKQ+xG\\\\\\\/mdbkh7YG5o+v6cgtT9CE2ksOgrjvW5RLlEj7W3YBhVnbAUpMO4xepJX895aP0vrDLxdsMdudPYZSd98dUj8YADIEZuuNUnibKkp1MDqtV1U9zjDjaK56k\\\\\\\/0ZTD2PBqIrv626qzqkzSF2\\\\\\\/gxVZtYPFXw3QdokRc2JqkJlpppYheosOfs0f1klVHoP5wSmp\\\\\\\/4wyITA7YIQSQyLCYFAx+E\\\\\\\/eMNiKkpkuxD76kIxDPTc8UVu+bDbFA5kbG+ZQa4i51h9aktZEuNdiiTa0\\\\\\\/OgHlkySLkUtV3wvBExPOzycuztWcVEeGOAHgVqaN9mmSNsR3\\\\\\\/9I58mwSiYo2saS\\\\\\\/tTS9QIQpUZ++koIbvbOeHTPVPzGL\\\\\\\/q3OqcHCbxKaht5Pfyaww2AVVFUDfEDTwxvu6rfs3EgE+d\\\\\\\/ZDPbuBsXE5rDBqhSV3xRDdpRP5AW8TVW4WdOw0mP4iIYgzlNPmR1OgXDlD1TUezTdABvanstQtrsrtqqam1B0uc2lGI4t0FVTfQM7KE0PuYQpwuKmnylpBD4IeYGDd890K3tCK5VuPC131OB347b2w+YcdUiEf+K9ynk4pVCfQ5A3S4etxPg9e7RZ3eMYSx3sirHbQl5IgWA+C5Ke7xL7u8\\\\\\\/kVIHrSsXTWdE7OWxx3nGp12RMwpIRoXHfnWF4Dr7w2Hw4WubhOSvk+rwl4u4XvbSuHgt9ucJZhsmP3G2u8LK9MZOGPNWS9pPV8Ig\\\\\\\/J+cXs3gXG8Qw+ACiDhfoxZteRqfCCpE5uVJlpGgFvnbNvPpQwFccWJkVQ1KJ7ct6t9TL7Nwo3eHx1elFsnLOAkzBT0rtp\\\\\\\/WmNxZo0iRsg4UmvytS9pZK178N0zM+\\\\\\\/3HCPpOV4wAt2+P4QWZCYVTnJnvO9C0D\\\\\\\/72s2NlwLEVkuJvqFIu+NywJu7kkasZpYKpiQDaWTeN\\\\\\\/Tt1P6K63Cldf28KsHi2JpEtIhAkCV2dd37L0Nu7b0KNyVzuXtkTd3QvvDiGoUfkmAYJbA2bGrjoRrFlcnvWC6JwP9aXy+wWnJdlxaxOLM\\\\\\\/\\\\\\\/ebb57Sfoaw8n\\\\\\\/XQNfQJaEr9EAWVmkoPOjVkm5FTEfz0ZDeGH+dBJB7WQ8lNzKjQwfRLhwtc73flDeBVehyOpxZ7hehz4Q8o2Y4p9l2U\\\\\\\/BVRCUcgShOUm+2o4mOHCOztRAwKsT2rakqzBAfsyUC7FAzBom7hqnhXJmKwe7w1uqUa5ehUtkmKkvSHkOug2SoUjVpGxpWczF67BHaJ8iBC64raOKmCJaDzcl7sMhIVsZQ2m3uXoLPM0oOjDPbxVqIqeLbFivySg3EgA1J9sfLK8F07VWHafBVQ0myywBS7tv\\\\\\\/ge1uBj8PBZaFLqBdUtvAv8XqAyx9khUYf4gIFnkYITR86qLqwKfpNE8FoitKSCDrX901q1Lzta2OhRkNW5AnTeuuSB\\\\\\\/pxckGbFsOj6ppiGyAFxY32nafLqEzvNkd8iCpC5PNjUpZ+3DrPgokxBxrB7bZPG+\\\\\\\/26XIG9QXdMa5eoNTA\\\\\\\/Frd0kFYFgmU6gPh59HTN26y2bBVDEHmBonMsqzRRBbnoYohJvEcH2O28qeSBmfk9K2TLTfqVrQzeOXG1CCRP5NCl46P0lFoiPOM+qpQtQThim3Rhvlp2WD9irfZ14LoTMHYq3zzFEqin9rOr8MmtudsM883nvRj4cJeWVZ4glsyEmxi6Qtu4ZQ3uFocMW55WB6KX2ZNSfjJ\\\\\\\/X5EM3vXo14okd0jcb1c6+wQtL\\\\\\\/0pAgO8WUrW\\\\\\\/2EYeKoGw8C7vDG6WinSJdJA4ujj9lzYlF3J4XsPqADZgJk3t7CF4X6ZgnhIza\\\\\\\/GRY9d+227GPezNWSyUGMf1DC5u3aF\\\\\\\/hY7Jlv+c48BYqvxz35zxkDbFM+N5y2ItAjh5iyK3e7vLwGD9Ar0To+KkKV8mDFxPLpIhdCyA3HnEDyVBj2jw7OrNnNzH2SzMqv20NF0a2rOPsQ7KKCiKhYq0mT7YYxo2K+rkeSOkESqHay0td0RirdzCGjw29sdxSU31BQI2Wl67Gll679KOLGPeRqA7uU572uc+e0AqA3eRTPZ8yEFDnVdXNzKP20F5T9PnZYHWElpd2QWzPigfdJtpYBwQ24aa9\\\\\\\/X0qOxKx2FlyfYifrreG2sQBMSBr2PH\\\\\\\/K2T3GDXoA2nyl7\\\\\\\/ABbYZRnhP500syHzBVBMSWj5tstfRn09JObohOmEdOaUlzcOdfOqF4AWhWuxfDFAOb3VcrRa14mv8rDIyCYKAuVlHKuZn7P7wukFKT+B\\\\\\\/P4JNDFzI5tcBDnUn8zdTVd7Z5bMyt9DbFcp1WZps4rOXHZjtTgl6BffPbs0QvGl0UHIhQK97ooeQxRp64YzKovOpK6b+hvPS8N7Xs1JFhqDlR46YGtyBZKtnhXHBFxH\\\\\\\/tdoKAoEXIo8eB8vsx1+jY0HS7esmdrLREMaZfWlrip0dzGhmFO09PrnyM\\\\\\\/QyDbQpaxTt440t1BydKrR9iuDBpVDY+DRKm455XXZHEVLPDJC4+6KUr6tNXetF5s0Q3NeNtCGdojkApa2RdL757PcBTf8dtCOmoFukN5fEakP1fw68B86UfP6T+H3uNk0XfRGPuJLf8tEeKXsDHREiqCRrZcGLL6QZYlkCBEcXQjntw\\\\\\\/Z99JABYwag\\\\\\\/eE6+UuPk\\\\\\\/VtG0Sd8OX8j735F3cX6sOOSOieavb0QVWZnQb+6FQFkqpMKylnhhcHHmL2lKD6Yn+x0M8wqQhiFZjiGU09LGYq6RkgbfNDYKILqtlo8n5nPohC1rk\\\\\\\/vt15kNdDyMlVsr6IcwI9t9J5Lt0N9dXyIAMgz0jBoRC36PHbuVxM8kV\\\\\\\/iv1XibmY0EMWNVSvLjMvELwJ2jEvG6aNE6uVNk5m8ZNZjG99gtl5aGWF8XEEXszUNXlscocXJOQHE2cgu9uGQseTD6Jh9jP13JlCCc07KKOhcH5yqj+1EN1sPYVERgkQECmINH16ziemoDH0KY8krazMr8N0BpPPvVH0X4Axb8RLqLknOA9A\\\\\\\/3wHutvN+nFad9CWO6z6p0EfWh7tIqwkSgMQjQu\\\\\\\/iZQnuUlMbI5nyC1FYwGyBa2+6FDGgODya1kAUYL5z79b7W2kpLIeiYAbeX6618Q3R7EJhAkry8b5fEuQo3y8MXNOkBo7DuqYxO5vHA84DmA\\\\\\\/swP0IDUCLSAldKxGAAVEVwWpATMWCmltPfUHRCr84eoalfQG35VI\\\\\\\/Yb4EOavYyPp5VigN7DeTICzbM0cO7icg4jnWXRONaycCKuLOl0asJLB39h\\\\\\\/1qr75KxCSxKJ21Q+NvOLGVJ0MxZwUGQ9DVOaCk7WDwtX4SRCS11OUKs9et9GpYr8V1ic5GnMg+glKq2APT18wHDHDkpLcfzXYUle+sBoTfBAcbLKJampIawT\\\\\\\/UwxLPlQzpU1Ca33lt89tu5345+dBOLikal8jI6BixAUFGDK+4PqLEZDFTVNaYCnLH3Iazj3ZQtk7upKDZLMWfkDBhxhroiRTMieM3pNksGC8pIOFv3io+RQSohcHJLRtu0AM9gk4RZKK3+FMXjwkaRC1e31tfzoyPwrLQE\\\\\\\/L8M0AEmXX8tO6Ie48datUQES1j4n9gHWetUutve3j5cYTQYSX1fZiM7Dvgz5lpvMU8er+1YIH63vFNOyL5JVyUfo0jHSE3jk7fEfMlVlRikzyUVGykJYELg8bBMuZRDk1ARXVkn+6MWxShbRyvkekJsc8OhKPsxWYXToe0m\\\\\\\/\\\\\\\/DB5nVi\\\\\\\/ozv6F3\\\\\\\/UIcyYRlSLyYIi\\\\\\\/UU45gO1KNb4+nAUxVhbxGfkfPUVrP5X+5uKJUjjqTRyos7qYlb5Li+MMmVxj3VPS1idVtsezUwTfMo4G4HLiyWg6nlKD\\\\\\\/OOOm7BoyFWUIZjZgSgE7FRdfUflJsHBYJNf+nV\\\\\\\/X3QVRWt3CNKWsLx9W2RYccE\\\\\\\/8P3eC8ae4kihwggMPxN\\\\\\\/1+Y8v63Z1aaLBw1u9YjbBRcie8dAHOfmJuOPMz47W329b73PqMgdkS8aG50ktdNF8YgdM+qxRN2IYdlftCAjMMBn0HSQsGtmdg3dkyMAb7LS4vL4Jdn38z6AGBXpw2OTGFAa7NpW4Mb1WlOyPVkB2ffTpfwYuSf9Dnr8zJV16JdDImFpRZY+5q+BOnhLMry+9hxgi9poHkxCdjNFBVkBF+ZlS2mb9zOlUhMG3oRb3XsKUV3f3qfs6ApotQrB87ahECmC4oAGvO0KXzGFVXqVk87u\\\\\\\/iPG\\\\\\\/5z73rhuxUFYTs01FfPwwRAT4NEKfpf1HbWAR3pzR4yQMsZsVHVroVrP41F6hQRhvRzZdJdB+Zd1SRnU7vzcUVGbcyZTUSXxNSFxk0k9Cgux+N+diktCiTL+uLc6zcdTz9InwezPPx5J0l\\\\\\\/xn9N0HyU\\\\\\\/VOPMWJtNgbCtXFGMUxzPqBWmO87LJRt2uXFM2UzV89UMRrJD0E2kvml9k81QuVBrqbTNNt4UXL6US9kxkjuiclUeBkZsC\\\\\\\/5wxUaNx7DsPjJCXMsdSiUlJd\\\\\\\/t98HUke1208o3OayXHQUcYTFq5kJyQbQavWj76qtDxPuh\\\\\\\/n7HaPYZ0D1T8eThz3TtmKU4hqBaC+ZOUro9uKAzqYMdqpZM0RoVz4PbMYmRdZcVKzP8wi3ZifP4rrDzVCbfkGCKXjNVX8e8v94aotrGVDXnXLsfWm5PIj+y7WQBtFSW5fV4N6H3lE6FgGV7RaF5HESw17lqoZN9vMWCYnQu\\\\\\\/Of+GCNvBRs88Cx4UUbv41hYA7MfCJiJ9VB9bhJn+TVbbQobb9n7Vy7D8dud1OpKa\\\\\\\/XK7eIUgPOnopqb1Bf17bLJUmAnEFMVl6IUZo2z4qQVC\\\\\\\/dwb+B7kjQORtNJosYW0jB0irz0uDlK0LjR+ZgTjkpaDmOcKm9YV0\\\\\\\/IwDzxdo9OeeFvEq7bFcqEsWF3bAJHlNoL1ce8HZI98wWV7cDP1OrjEA8Evi6glxokRkPCkI+iEhEMIUCKpW3VJTuxuRzQ51zq7ZCYFCzgBg9qGEVFjNUHfK2ElttGcSmDxMGjDreOEX5DPaakWoSd3FsdXt+v\\\\\\\/ejzoumtGvzh300IUpTtMiiBqYrkzOqjIsCrksdyxAutQgZNZzyBAL7JHbZR1TrEk2f5gfcrL525fB9f14d753SVSIqOKC8Klh95gQQWhLehbz6fHr4vN5c2OsQ0zB+rWFfHfvtm7Vx4LBNMrAZoAF1t9j\\\\\\\/H\\\\\\\/5h4TkFHPuCTyQptuthB0wvO3ca+nz3pcym5jFy6CEvUKJEg1\\\\\\\/NuGPGjPFGzDTtbGB0oU3oscoYEbc9UXqz3Y73e9+KuJtTUjs+oPci1shI4oTW06VHgmgvxEel9aMyTSjOfCTI0joKWcqIAb\\\\\\\/O\\\\\\\/rw6WC1gBmLdtXIaeJMxLeY7NB1g0kqqwAP5O0kpe\\\\\\\/I3cm8c7FSnImMakddMzKByysI9kqGk9yKpkri8Hj2QFDlr6twtV41MAZqek9UCJiNrs8b9I0YBprDcXb7SPys+Vh5h051gUWmgFya1bo97eerh4CjCpCielTGTf4cub179Q5LCq7pugZxjFizIyUOyGsXiu2PiaT2YkVdku2ygXK23\\\\\\\/Slmbk76uMNX2DoxsY9qiUPEHTuosA7zw4yhiImMOHmBalU4UGSRSSo8SSw6ftJitVlNRUV3fMCCOF02ca3Jwge4+iMt8KsWD646iNUx90HLulXfXdb+ybfIB\\\\\\\/t4EWp+fOG4dUUhYEqIGOGzJo3tuzqHccCF27igTFA6j4V\\\\\\\/sSok\\\\\\\/fI6t0qW9cebYwsw8gLogApffdlESvjV7PkjMm81bFMbepMM2ax38R9xtOZWE1ruVXtV9Q4ovV2Y2\\\\\\\/rO7\\\\\\\/VDyTMzP+zuFwDuytS5H9Yhmg\\\\\\\/JcczPEl81l93uzJ4+GzGZ+pMH176KXRKTF5TMlVaSJytDjMs3RTeaEBlbsg2GD4mfGXgz+aFPvypMQPjCHU8ebX2gabrFU\\\\\\\/GaY7lR0fDyF0SvdjPtEIhpLZMzQBgTHAiQxgq3EfM9ftSMppnZl90\\\\\\\/Oh7UWKh3D\\\\\\\/zRR7qZgan1qU8gY4pRhQYToqemfFXlnqp7fvk2+\\\\\\\/ZCmScpmCVTcRLeJGER00muViYPb+wE0toYqeFSfgV5Z+ChBc2YTTXcDSNAI+bu7WE6e0ybGPn5L0KHwOG5zn1LEasUny2iyYZBe6CuZHsMsbCm\\\\\\\/V33oYwScYdgaWQNGsM2G91P0LDfLpulgXMfat+x6xOjUaF+DJel9486lKgM1EAhrYUu0sx+oN+Nx5YWMNRE8ogSoZOt87RqnQ+8LN2vx2fX0aPMyajzII76tST2Dqn7NHJXfLEiFOCz0ISR\\\\\\\/cDEL5256XNOwJowZhWFYExcNG3ESxOj6YEDGG\\\\\\\/vSCyUNYVlwf\\\\\\\/y7ceIbz0cJoMA3NejKo3BNBhTzmZ9ETZBjBlLEMV6HibO9VYnczlX+FRx+C\\\\\\\/EghM9+OQUM470DAPMe9M06urvFyoTHwp9+1UXJHVSU+dG7Vm7pXQiOFeAhsccQ45jcCrnr7lY1GoGexI9GOVYOE+I87U+x9YSBSfGU8U7K7YfoW3WjgAWymD7Hb7\\\\\\\/kpuyjgZFNlWMeM+c9OQTvXTFoZQA1KAXRns7ISdVFoGuGYZ0PD2990ibQHeGozGORRVbZgsz+9DSGZ8FWDR5OMe9\\\\\\\/jtKyAevRQrjU8qVoOh+YiMkEg2Hwi4UFEZEvY8DAoiiPA4dynJLnzl2h1noHws1C4x9o5KozT7kfi73kH\\\\\\\/e1n4odFT070d+MH\\\\\\\/0diG8L79LpNADyglG3WkcWpyd1MsuPx2k57lXcBVWAj2uYS9mH0apEO+mSDARKROTD\\\\\\\/H19YzkYHA\\\\\\\/ofPQ0GR0cotn5dQIJdiWelVopXVKh5nAqXlDhgokIeFodz0c8Y4DAbzQXPFvZ3H1vzhoL6hfjJ17thXa4bDjDG9CfFWBmdgiKuBXkP0RAB4kYWYpYCOXnhZmniHjuX\\\\\\\/7DO7IRH7ZSZBS5AAkLRyfS\\\\\\\/3uzXHvplwCA7AY+9\\\\\\\/t\\\\\\\/7vYnndgP\\\\\\\/f5iOPO2zHwkoqiYSZcAhriSPqM5UoO6iQcKbojXFeRCzGP4sS5HNMrNDQWstzU4BFn8NYbcANIyKYfq\\\\\\\/0Pk3cFius\\\\\\\/9\\\\\\\/4\\\\\\\/PfFotOExf56YtVRIkBYIJGBYI28gNokSPWINL3+Ff\\\\\\\/CA\\\\\\\/fOHbn\\\\\\\/xfEEFG6c3umplWpcNoxZvGCv6AVW+rEMMmo7mfVBkUI30PZ7wrYlcwiB4Xq4+8Ku5KUFtNy\\\\\\\/6lxGGMO6UAd\\\\\\\/PlWFO70kb2WIHwq6PkiyO6GaJHMGdLeP7D98Y7MByZMDPeIZo6SCrdfJeo31oTG\\\\\\\/a99xDAB4fcx8V54GJ4fgxQb4GfqSN4WYaAXTUuZb6TLFfQTTMisA\\\\\\\/H\\\\\\\/KxLNesPV6p4ErxMqyztonWVwCFNYRKy0v\\\\\\\/LPibFglO6Pm7cnFBezSGTIX2qnyKOZQLrn8Bk8mrcKix2w61hQJwJMXqVuc\\\\\\\/o5soUaaX70v2gNdYIbxul09eOqybUnEoiOEsD5x2Mp5EoLR\\\\\\\/zVT7w0FV48m45bHinbt2B8aORI+7YsmVgFzeMpxCg3eDB8uL\\\\\\\/xmnXveDKK3F6NKl7DZMpKCzFyGbghrZRkh6JorbJa54u8fV\\\\\\\/aICjrEZmYu5r2FDPreGYfrU9RcXU7uYSncbjy92KIavnS5J+dvxm7AUTP7dfW1LSb9LccdUU616YJDXAWt2E5bzjugU9kQLhUHFtgz1WewaEIHzddy00QPWMbPPeHhKCAW969Dsan4LEg0fQe2UiJmPtRHA+RgjMrMoRVxntGmihwtr6i0utBkJynVDEurh4Zb4IPWm33ze5b23IuSrHAWhLDcBN+2M7XeNjk6Vp5bk4N8hNHIPN9kUHFReX3ethtCeXJfzjmbl2VbLverRuAJVdAQGTWjxUf4faGZ91oOsNqWMr46c7uDUCYWSFYFliVnXYy7r2N5EuTsfgUO19Pi9LXlXS4a23goJ0Wsmrl6HEcTl3zahjtwPNTrRUOLdHqR\\\\\\\/fYfOOyB5v7Kt3sJ2fy0K15IocRfcygIGhls0RDhdD0eb9k3Hea4d2cwwWqOKFJcFpm0IYDTVkY1PvKw\\\\\\\/terPW9CIvwSoTDsw47\\\\\\\/M1a3nngp2O4HGjARRo0tYrBd5NKXRBjOteByyS8lTImhtxvULt9xXcvDYudDYX7wdZr3M1TrQKqQXqzjxKcfCnD1gsPPn63qdhjmMTxESxnLr39SVQ+gzg93o5SlmGbmPWrTVwjrMvhffipy2c8YzyjltjZCXU1tyATkQgyYAB9bAxdA3SJlAeNUTfpv4vYX1e9nlV1lR0ESAj8NVPDF79Xz6vY4ldYRN\\\\\\\/X8CsVGolE3ivtNF1AdY3FD210tTyizbDJL1nLSRc8cxOGPzixhaGd8mXWm4Su2NzcE6FLd1W1PbueNhq0lgFlXuQOlz4iwE\\\\\\\/flvWl\\\\\\\/bfeWqa1BUgA51mxpNJkSJpe\\\\\\\/rI\\\\\\\/kqDMdqLygNAarasb\\\\\\\/sBXhb9bGQQRpmzYQ+XCOwGTtNaWzGrpdJGcBLLnjxy53oUkm7D6d+A5rKxKM8FTsYI47WsEToFE5xeVCoH6wsBzwo7s3y1Mh8EaLzGDLFldgIRFO9VyPH+am5mmhFW6nobmB3rxESk7c6ZFLswNWYrU9IoR0wstMNuGmw74RlnBBQWnPl9zbizynYY5SnNfkqhGElB0xeMPPDIjayY9apS7RRNX07mbfoNfRv7wzr3V\\\\\\\/dSsyVeW2yYI3WVxxroAxcUJjl6z1yD0c98RjAjzEhCR3bXyWy46csCVhxa7+VnSs3bO7ZUAm0PtXhJybGnwrxAPxTrKG4ZzQN5uDViPYyy4sBoG3A4HPt92u4\\\\\\\/DHVLjP9ZsD87+tfRzGEJm2d1QXP7GWCOp31e9GtE+RJ7ajon1W9HVeh+A+mhou+r5iHyezs9S1dkcDT7R\\\\\\\/DiPpPTj9DNC8XFQcByr0NuIbcUyIxO9PwRNQQvf1Urb8mcjy6yViY2PUa81Dl1ILTUhsgjyhRUivcc7GisUNmOmi4fkFEnEGZof1cG90De9U0MDxOe95Po+6zg4A4SM3HnQnsNkS+cVNrw5VEw5ANXbo3MX5xaOJjLVOaMKiCCxJiXfh1UbDaH8tMkNv0fDoz60kBZxM5LLQP8of\\\\\\\/jK1ldfX8MkqYupGFkthEhLte8AcG1sVp+8+RZ+XzDE9FSFsiKKS+y\\\\\\\/OqGP30T+OPIvy8DIhLcbAe76uhlcmFsoP9yRF2uZtjoI\\\\\\\/GmD0Aa+or\\\\\\\/qQYNifT9wL0VklbN9a5YEQOf6qW9+w520VCcMdcPXNmtkhqRiqJDaeSJbTgIhgPzs5knRyHm3GZGSnXW966E2IZzDk3LoFcntm9ow274DUyOsDLaVOYaQcyhoh4rctaPBm2m9F90n6kQdJ+WdsS+By\\\\\\\/ANbPxLOm5fLK7zTAyq+hz3RpNq1kyyp0\\\\\\\/XrOO4p82OqhPTNwFaFauNll3jg391FNXdpHyPMjOnAQrCxGpQZLPM9Eu6snsojOWR24RQ3PBttJvpsNW8vYLst7BJj48HjkHfVA3ulXzgeq0CXWwvbrOH8lOFv+AsxOqEifKs26pSMO3UsuNmIh3Ds4H+zeG1cQArTO0\\\\\\\/cQ\\\\\\\/nMxrppRHnG+OdWLnf3DIMVx4EtVu5vCyflRbAoYEsmm0aggVrrEJQo6ZVVzi2isxAIoK\\\\\\\/j44CFJnSKdRDFwnRytcUJpQkb63oD+6p0BQ\\\\\\\/8tjyXPDxq4nDRtG4CUightckp\\\\\\\/baCMDXdnOrF8TP5RzmHMHYw8MHRBQX3zZJS+RPua+tjetBATZg4NOio1Uqcpb\\\\\\\/FNrR7\\\\\\\/z+RiLOTs8ReFewpcRiBkGsilW2+7bf2aMP6qE5WQNN\\\\\\\/I5JftX7T1to0tBS9XhfKOd5qRVNV+JkX7mPMIASsBD9RAkdCLUCAgsDR8Pl4usUW9tZu7iq\\\\\\\/v7QIv2hOKhl7rnBLcbtyiRE8VFf3TOAF7GWMlrepgM1TExBlEkRbeYI4QOckvN\\\\\\\/4y4Dt0yA\\\\\\\/\\\\\\\/QtKZgjr4jE\\\\\\\/th0vgr3iSSTRl1YMoeNwoumZWXP1C6xGqCjdTL2fnVoiWEerOFON7GPgOLiHFcPE3X4Mj5COQK6iU7RvSWAU+FDgYU3siOcyBR1eRAaRL45s0bjamBfeK+RijGVPqu1XMcgCkyLIeB563eCxmr+Zw61tfyD1chPuYpRO7a7fLS9SoD\\\\\\\/u6rGof7qko1O5wR15MoObyzXZqYmipbiLH5Ib3bOH6FATIYJUV6zGXQ+Xo2k\\\\\\\/naSHsLAcKf6Q5SpZIGgW3tmdIEvJwT6YRUGqmEz40pIuW64K2fiSk\\\\\\\/T4WvSyNvFoNPAk4dvtANMJnOCpkCE60EYYsOE7JL8EaL85Hgvf1qL4urUsSiGiF7Lf0YidvsQurNvV2hY\\\\\\\/DbryvvYd8vT14EXcXqxrcYvoDLnZtjZ5pGaCk1\\\\\\\/Q7\\\\\\\/ZEqAj6VyxoQV0FIWNW7g1VDxIGXo5G\\\\\\\/sTsoyZ5MwGN9w0LiYmjL79zKobz\\\\\\\/I+oingb0G00vICqigtgsTxJnRQIl8lcfkGZfqVnA79WxdH4dV\\\\\\\/dU\\\\\\\/bHJ2COA+GG5keGNXzJLvMD+UjXEoneFu4Ea1tsgasRHmpuBpAcYb3q5PEoWzOVdRewi7rNih59XBOEBF0fF+oqu+cWrmWyGW7N5y9TSLNTj1yL395dXQy\\\\\\\/g\\\\\\\/IQGPMnqeIZziwhT6aPxiJskiG7ognH8tX4NihPdpCg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022JXDXtGPVY0dUNNRvueDwTQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022c35c9ff6c6a3d6e16bae22648a3d90967160de37a77fa9b464d134f98f8d47db\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00229t5azFQZ+lkRZPX4pU6Mym5i5XvKxUugs4XJoV+ecxZXnFBiZ+h2lyFCnH\\\\\\\/xP1f82Pht7aYyDgGuEcMejlgjO7Hlq4m3ox7qNrVK5MjbLoCZX3itShmCdEH8DXtAnCMI6D1imlb1p6PCkRPjTfsKL8O+EbTQfRcZkG4v0kWGwUuEa\\\\\\\/4gCx1Z3GAEQBjwqTv87qOyfdZf8WGOd5P\\\\\\\/DLrdMSeGUfypsqLJe67egtOIYmQJ3ul68avG20r+QZkt80+JOYbV7vNkrDXfUtdM4pH5\\\\\\\/tefuoSk5SxHp18NNhieB\\\\\\\/Lw6ooNeNYVG\\\\\\\/zwSTOADPb+9bMpgNgAarHQm2i7Q8AFlTBIkV2LrL8fJxupqdHfcFIBpQKG7Ua\\\\\\\/bvcNmnXgs3c\\\\\\\/b1Q914QJ4KC5dSUM+MXUyYuGFneJfnTVK5Mv4abYs0Bg7Sf+xvt0xKXVdleXIzs411OF8LIffNRwphxlSBN5Mi+wdSh\\\\\\\/NMNklMhkdWfY4fgAdgkWSQyk1LV6axrXDumVbMWVsvgK96n628kbPpa1N5+sBeIUu8yyL1Q7xS4yB0utN7pbXhHDNOFk9Tj1AhRFr0hL0GC1kGsX7xyNPd+A4XPp9AJsEqQZPCizdAPKGXR4KM0pctuaq504wBuWXK+d5PCn2qP1hst5yZfndw4\\\\\\\/IUhN1pc9MpSp6koKtdkzKX4DI5Q6Uts6X\\\\\\\/UHoyX0bCz8FOamv96nttAtieZm+Y\\\\\\\/lnawi7SVKzdw96nWQAgmR8tuVT+bsZcVAfAejgnXMP4iq2hzKA1ldEAyv3xmVLWGq\\\\\\\/qmQjqWpA5UFZoo9mVIcoabBVpq8T4G12DhzJ2s8KuJvMs\\\\\\\/acjp8jmkis69T8Axh7rK+96LzdwqBq\\\\\\\/C9nqNRrOAOQXy0\\\\\\\/ghcmszhxKeX1kppnUsyeLS9ZayvX\\\\\\\/qjQTl7sPv0Wuy\\\\\\\/stXGo80J0qudi5SlhV8gEfB2Tj1o2XZWqlMcobr8OVdm4kB8iONb6qgOuoqb8oGiqdaDTFeqZs8SR2jl9dYpIqQkeOYGl9oOM5zTsPIK0lAUf+0veyPehRatzo5oxqzAC+u40sWtrCKe9oKCcS1nLE961FADclfYQCHFw2xrYqFduN\\\\\\\/vFHmiDECSh1nvYmsF6LMBHjW0zWWFpm2ekLBxXyHxKr+9a1fKQZYwDNNiC7jlDLWx6scjLAGYj+9xlEkuNllgwNaZjGS28WPPhnJe5sjd8JR1\\\\\\\/KcmrmIOkbDSJZOs9OmU9\\\\\\\/guJ9mtfUEEnK\\\\\\\/ZWabtJ6Z1Lst5iae7K6XYvDoSNQpRaxyB++1ZcOH5Am7TjKDlWwvIO6U5L\\\\\\\/Zwli1wUPdN7LXN8IXtn9HJhpTtVJusrAXEth0Ms1MZ+E0wGoPZzo1eARP7vGbU1+0G1avqarWuBTfa4p\\\\\\\/idhPNQeklmiR7nAZimt7\\\\\\\/oHbaBr2spdEAZi4e68zAzPMsPIdAPC03OiBf08ibkblC68i6oX2KrztzKzXAGd9TcH1roaD31p7fJ3+FrvGxdyToaA6\\\\\\\/Kaqp7jS\\\\\\\/mfAT2NZrDMbicY2TqLz42FgO\\\\\\\/tsslrNMfcL2xcr3SZaVFgHMqMUDYKdGUMsAVITwNpV+YjssKwj0VNRNbow7STf+OFzXpC8oCHzkIYx1R0SDiRA4LUUfAL68tog4qqhNbw5YAf6C11yXyHpAJqB1oSKAhHptbEDmgd682mbVeMsii+VXoN8gHZ4N\\\\\\\/\\\\\\\/HEUqlr\\\\\\\/vCJphWPIt1n5kysvkDf4Aoo+Ieh7jOH7mAX6DpcJkwhh3O8ms0xdN8+O+6dxr91c6oB06FDbtV6ngs1fVr0Dt1ugApXfryJpZqWZcLvIY9gXC7Pk8Da6hl30FVLuN1z8U+IjGHEJOEUtSwRxQJNjFwD1Kbv+IBmlq7O48ESVj+yhl6fj7kCv+Cr03meMLhTiNnkXW6XmWsr2r3xix7LsEGw8RJH0+Y97ROhudEyUPdyW4bEgDP4TnezCeQYjnRMgX8T3qlx\\\\\\\/K0ifLqZsE5VadydlMxEr+BPB6BIIFNyFH0f\\\\\\\/m564BhpeQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022AXTQ4LKdlmAdAdZD36slyw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022186df41225119e552d2e661945c61cfca46b92a9c546b6933720997e61f0d160\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-489846 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/6/1959" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-NnDZnDTD0jPW6NRU48g3T6JNbCBs7l0z7fqH-K-Uu5g" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/6/1959" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-lBXMHT2vZAuUPlVXY62K3rOgka1NGyW5_NOzpgWtks0" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/6/1943" title="The Pseudokinase Domain of Saccharomyces cerevisiae Tra1 Is Required for Nuclear Localization and Incorporation into the SAGA and NuA4 Complexes" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/6/1971" title="Sex Differences in Recombination in Sticklebacks" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="489846" id="node489846" data-pisa="ggg;8/6/1959" data-pisa-master="ggg;g3.118.200262" data-apath="/ggg/8/6/1959.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">diploS/HIC: An Updated Approach to Classifying Selective Sweeps</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-4381-4680" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Andrew D. Kern</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-5249-4151" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Daniel R. Schrider</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1959-1970;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200262 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node489846" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Andrew D. Kern </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genetics, Rutgers University, Piscataway, NJ 08854</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrew%2BD.%2BKern%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kern%20AD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAndrew%2BD.%2BKern%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-4381-4680" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Andrew D. Kern</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:kern@biology.rutgers.edu" class="" data-icon-position="" data-hide-link-title="0">kern@biology.rutgers.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Daniel R. Schrider </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genetics, Rutgers University, Piscataway, NJ 08854</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BR.%2BSchrider%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schrider%20DR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADaniel%2BR.%2BSchrider%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5249-4151" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Daniel R. Schrider</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/8/6/1959" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:489846" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:489846/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/6/1959.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:489846" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:489846/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/6/1959.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:489846" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:489846/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/6/1959.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:489846" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:489846/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/8/6/1959.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:489846" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:489846/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Identifying selective sweeps in populations that have complex demographic histories remains a difficult problem in population genetics. We previously introduced a supervised machine learning approach, S/HIC, for finding both hard and soft selective sweeps in genomes on the basis of patterns of genetic variation surrounding a window of the genome. While S/HIC was shown to be both powerful and precise, the utility of S/HIC was limited by the use of phased genomic data as input. In this report we describe a deep learning variant of our method, diploS/HIC, that uses unphased genotypes to accurately classify genomic windows. diploS/HIC is shown to be quite powerful even at moderate to small sample sizes.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/machine-learning" class="hw-term hw-article-keyword hw-article-keyword-machine-learning" rel="nofollow">Machine Learning</a></li><li class="kwd"><a href="/keyword/deep-learning" class="hw-term hw-article-keyword hw-article-keyword-deep-learning" rel="nofollow">Deep learning</a></li><li class="kwd"><a href="/keyword/selective-sweeps" class="hw-term hw-article-keyword hw-article-keyword-selective-sweeps" rel="nofollow">Selective Sweeps</a></li><li class="kwd"><a href="/keyword/adaptation" class="hw-term hw-article-keyword hw-article-keyword-adaptation" rel="nofollow">Adaptation</a></li><li class="kwd"><a href="/keyword/and-population-genetics" class="hw-term hw-article-keyword hw-article-keyword-and-population-genetics" rel="nofollow">and Population genetics</a></li></ul><p id="p-3">An important goal of population genetics is to accurately identify loci in the genome which have undergone recent selective sweeps. In natural populations with complex demographic histories this is a difficult proposition, and thus much attention has been devoted to improving sweep-finding methods (<em>e.g.</em>, <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Nielsen <em>et al.</em> (2005)</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Lin <em>et al.</em> (2011)</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">DeGiorgio <em>et al.</em> (2016)</a>). The vast majority of this work has centered upon a “hard” sweep model of adaptation, wherein a <em>de-novo</em> beneficial mutation sweeps to high frequency carrying with it its linked genetic background (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Smith and Haigh 1974</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Kaplan <em>et al.</em> 1989</a>). While such hard sweeps may be important sources of adaptive evolution, there is good reason to believe that selection from standing variation, a.k.a. “soft” sweeps, might be equally if not more important. Indeed, both theoretical and empirical evidence have accumulated sufficiently as to suggest that soft sweeps might be the more frequent mode of adaptation in many natural populations (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Hermisson and Pennings (2005)</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Messer and Petrov (2013)</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Garud <em>et al.</em> (2015)</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Sheehan and Song (2016)</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Schrider and Kern (2017)</a> but see <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Jensen (2014)</a>). Thus if we wish to generate a more complete catalog of the genomic targets of selection, we need to be able to reliably identify both hard and soft selective sweeps.</p><p id="p-4">While this sounds straightforward at first blush, a lingering issue is that hard sweeps can generate spurious signatures of soft and partial sweeps in regions that are at intermediate genetic distances from the true target of selection (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Schrider <em>et al.</em> (2015)</a>). This “soft-shoulder” effect thus requires any method developed for finding soft or partial sweeps to be aware of the genomic spatial context. We recently developed a method known as S/HIC that meets this criterion and is able to find both hard and soft sweeps in the genome with unparalleled accuracy and robustness (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Schrider and Kern (2016)</a>). S/HIC, like several contemporary tools for sweep finding (<em>e.g.</em>, <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Pavlidis <em>et al.</em> (2010)</a>; <a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Lin <em>et al.</em> (2011)</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Ronen <em>et al.</em> (2013)</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Pybus <em>et al.</em> (2015)</a>; <a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Sheehan and Song (2016)</a>), utilizes supervised machine learning to classify individual windows of the genome as sweeps. In particular S/HIC summarizes a genomic window on the basis of a large vector of transformed summary statistics and then uses those “features” as input to an Extra-Trees classifier (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Geurts <em>et al.</em> (2006)</a>). The feature vector of S/HIC captures spatial patterns of variation across a region of the genome by recording the relative values of a given statistic in subwindows as a vector. While S/HIC performs well both on simulated and empirical data, its use is currently limited to population genomic samples with phased genotype data as it relies on haplotype-based statistics.</p><p id="p-5">Here we introduce a new version of S/HIC that alleviates this issue that we call diploS/HIC (pronounced “deep-lo-shick”). We outline a series of summary statistics which we calculate on unphased genotypes across a genomic window which allow for accurate classification of genomic windows in the context of supervised machine learning (c.f. <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Schrider and Kern (2018)</a>). diploS/HIC uses a deep convolutional neural network (CNN) approach to classification (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">LeCun <em>et al.</em> (2004)</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Krizhevsky and Hinton (2010)</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Krizhevsky <em>et al.</em> (2012)</a>) whereby we represent as an image the multidimensional vector of statistics calculated from the window to be classified. Using simulation we show that diploS/HIC retains nearly all of the power and accuracy of S/HIC. Moreover we show that convolutional neural networks outperform our original Extra-Trees classifier both for our original S/HIC formulation and diploS/HIC.</p><div class="section methods" id="sec-1"><h2 class="">Methods and Model</h2><div id="sec-2" class="subsection"><h3>Coalescent simulation</h3><p id="p-6">diploS/HIC is a supervised machine learning classifier that uses coalescent simulation to generate training data. All of our simulations in this paper are performed using discoal (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Kern and Schrider (2016)</a>), though other simulators could be used provided they are flexible enough to simulate the evolutionary scenarios described below. We simulated under a number of demographic scenarios and selective scenarios both with constant population size and with the very strong population growth we observed in <em>Anopheles gambiae</em> from Burkina Faso (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Anopheles gambiae 1000 Genomes Consortium (2017)</a>). These simulations were performed under five sample sizes, <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-1.gif"/></noscript></span></span> haploid chromosomes. Command lines for each scenario are given in Table S1. For simulations with selective sweeps we sampled from a uniform prior of population scaled selection coefficients (<span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-2.gif"/></noscript></span></span>). For each combination of sample size, demographic history, and rage of <em>α</em>, we simulated large chromosomal regions that we later split into 11 subwindows (as in <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Schrider and Kern (2016)</a>). We simulate hard sweep training examples by generating <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-3.gif"/></noscript></span></span> simulations where we conditioned upon a fixation in the exact center of our region (<em>i.e.</em>, the middle of the 6<sup>th</sup> subwindow). Linked-hard training examples are generated by simulating <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-4.gif"/></noscript></span></span> sweeps where the selected site is at the center of each of the remaining 10 subwindows flanking the central window. For soft sweep and linked-soft training examples we follow this same procedure but add a uniform prior on the frequency, <em>f</em>, at which a mutation is segregating at the time it becomes beneficial such that <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-5.gif"/></noscript></span></span> Most simulations under constant population sized varied the time of fixation of the beneficial allele, <em>τ</em>, over a uniform range <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-6.gif"/></noscript></span></span> in units of <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-7.gif"/></noscript></span></span> generations, although where noted we fix <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-8.gif"/></noscript></span></span> Our simulations using <em>Anopheles gambiae</em> demography were done with a more recent distribution of fixations times, <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-9.gif"/></noscript></span></span> Finally we generate <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-10.gif"/></noscript></span></span> simulations without sweeps but with the same demographic history as selected training examples.</p><p id="p-7">From these simulations a balanced set with equal representation of all five classes (neutral loci, hard sweeps, soft sweeps, loci linked to hard sweeps, and loci linked to soft sweeps) to be used for machine learning is created through sampling without replacement. This constitutes the training set for our classifier. The simulation procedure is then repeated to generate an independent test set of <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-11.gif"/></noscript></span></span> examples per class on which to benchmark trained classifiers.</p></div><div id="sec-3" class="subsection"><h3>The diploS/HIC feature vector and associated summary statistics</h3><p id="p-8">Our original version of S/HIC used as a feature vector the transformed values of 11 summary statistics calculated in 11 sub-windows centered upon the window to be classified (<a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">Schrider and Kern (2016)</a>). These statistics were <em>π</em> (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Tajima (1983)</a>), <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-12.gif"/></noscript></span></span> (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Watterson (1975)</a>), Tajima’s <em>D</em> (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Tajima (1989)</a>), <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-13.gif"/></noscript></span></span> and Fay and Wu’s <em>H</em> (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Fay and Wu (2000)</a>), the number of distinct haplotypes, average haplotype homozygosity, <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-14.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-15.gif"/></noscript></span></span> (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Garud <em>et al.</em> (2015)</a>), <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-16.gif"/></noscript></span></span> (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Kelly (1997)</a>), and the maximum value of ω (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Kim and Nielsen (2004)</a>). These statistics are then normalized across subwindows to capture the relative shape of a given statistic across the larger region such that the value of some statistic <em>x</em> in the <em>i</em>th subwindow is <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-17.gif"/></noscript></span></span> If the minimum value of <em>x</em> is negative, the elements of the vector are rescaled such that <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-18.gif"/></noscript></span></span> Classification of genomic windows then proceeds using an Extra-Trees classifier (ETC), and windows were identified as belonging to one of five classes: a hard sweep, a soft sweep, a window linked to a hard sweep, a window linked to a soft sweep, or a “neutral” window unlinked to a sweep. Given our choice of summaries, S/HIC relies upon phased haplotype data (<em>e.g.</em>, <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-19.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-20.gif"/></noscript></span></span>) and a polarized site frequency spectrum (SFS) (Fay and Wu’s <em>H</em> and <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-21.gif"/></noscript></span></span>), though the latter can be removed with little loss of power (data not shown).</p><p id="p-9">To relax these constraints we set out to design a feature vector that would allow for genomic windows to be classified into our five classes on the basis of a set of statistics that could be calculated from unphased genotype data. In particular we aim to capture quantities that describe three axes of the data: the SFS, haplotype structure in the region (we will only have partial information about this), and linkage disequilibrium (LD).</p><p id="p-10">Unphased data do not lead to difficulties in summarizing the SFS, so we use three summary statistics: <em>π</em> (<a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Tajima (1983)</a>), <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-22.gif"/></noscript></span></span> (<a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Watterson (1975)</a>), and Tajima’s <em>D</em> (<a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Tajima (1989)</a>). Describing the haplotype structure and LD from unphased data are more difficult. Let the genotype vector of the <em>k</em>th individual be <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-23.gif"/></noscript></span></span> where individual <em>k</em> has been genotyped at <em>m</em> loci and <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-24.gif"/></noscript></span></span> according to the number of non-reference (minor) alleles individual <em>k</em> has at locus <em>i</em>. We define the genotype distance, <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-25.gif"/></noscript></span></span> between two individuals <em>k</em> and <em>l</em> such that <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-26.gif"/></noscript></span></span> We note that <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-27.gif"/></noscript></span></span> is an underestimate of the true sequence divergence, and thus only provides a lower bound on haplotype distance. While that is so, moments of the distribution of pairwise <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-28.gif"/></noscript></span></span> comparisons for a sample prove informative for differentiating among our sweep and neutral classes of loci. In particular we use the variance, the skewness, and the kurtosis of the <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-29.gif"/></noscript></span></span> distribution as summary statistics.</p><p id="p-11"><a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> shows mean values of the normalized subwindow <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-30.gif"/></noscript></span></span> distribution moments from constant size population simulations of hard and soft sweeps (see figure legend for details). While each of these moments contains information about the location of a sweep, the kurtosis in particular adds information as to whether a given sweep is soft <em>vs.</em> hard. Here we are using this for diploid genotype data, however we note that a similar summary of haploid data (<em>i.e.</em>, the higher order moments of haplotype distances) might add excellent information for sweep classification from phased data.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/6/1959/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genotype distance distribution summaries for soft and hard sweeps. We simulated  sweeps under constant population size with moderately strong selection (), in which the fixation of the beneficial allele happened immediately prior to sampling (i.e., ). The relative magnitude of the variance, skew, and kurtosis of the distribution of pairwise genotype distances,  across sub-windows of a larger simulated region is shown for soft and hard sweeps respectively." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-34569528" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Genotype distance distribution summaries for soft and hard sweeps. We simulated &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-31&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; sweeps under constant population size with moderately strong selection (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-32&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;), in which the fixation of the beneficial allele happened immediately prior to sampling (&lt;em&gt;i.e.&lt;/em&gt;, &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-33&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;). The relative magnitude of the variance, skew, and kurtosis of the distribution of pairwise genotype distances, &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-34&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; across sub-windows of a larger simulated region is shown for soft and hard sweeps respectively.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/6/1959/F1.medium.gif" width="440" height="215"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/8/6/1959/F1.medium.gif" width="440" height="215"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/6/1959/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/6/1959/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/489909" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-12" class="first-child">Genotype distance distribution summaries for soft and hard sweeps. We simulated <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/F1/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/F1/embed/mml-math-31.gif"/></noscript></span></span> sweeps under constant population size with moderately strong selection (<span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/F1/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/F1/embed/mml-math-32.gif"/></noscript></span></span>), in which the fixation of the beneficial allele happened immediately prior to sampling (<em>i.e.</em>, <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/F1/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/F1/embed/mml-math-33.gif"/></noscript></span></span>). The relative magnitude of the variance, skew, and kurtosis of the distribution of pairwise genotype distances, <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/F1/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/F1/embed/mml-math-34.gif"/></noscript></span></span> across sub-windows of a larger simulated region is shown for soft and hard sweeps respectively.</p><div class="sb-div caption-clear"></div></div></div><p id="p-13">We also summarized the frequency distribution of multilocus genotypes in a manner analogous to the haplotype frequency summary statistics of <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Messer and Petrov (2013)</a> and <a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">Garud <em>et al.</em> (2015)</a>. The simple idea here is to treat the multilocus genotype as a population genetic entity, sometimes referred to as the “diplotype”. We recorded four summaries: the number of distinct genotype vectors, and then the multilocus genotype equivalents of <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-35.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-36.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-37.gif"/></noscript></span></span> call them <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-38.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-39.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-40.gif"/></noscript></span></span> The normalized values of <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-41.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-42.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-43.gif"/></noscript></span></span> across simulated genomic regions that contain either a soft or hard selective sweep are shown in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>. Each of these statistics adds useful information for detecting hard sweeps, however they are less sensitive to soft sweeps than the <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-44.gif"/></noscript></span></span> distribution introduced above. Thus, these statistics may be useful for discriminating between hard and soft sweeps. Finally, to summarize patterns of linkage disequilibrium we computed the equivalents of <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-45.gif"/></noscript></span></span> (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Kelly (1997)</a>) and <em>ω</em> (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Kim and Nielsen (2004)</a>) using genotypic LD rather than gametic LD calculated according to <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Rogers and Huff (2009)</a>.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/6/1959/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Multilocus genotype frequency spectrum statistics for soft and hard sweeps. We simulated  sweeps under constant population size with moderately strong selection () and in which the fixation of the beneficial allele happened immediately prior to sampling (i.e., ). The relative magnitude of the number of diploid genotypes,   and  across sub-windows of a larger simulated region is shown for soft and hard sweeps, respectively." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-34569528" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Multilocus genotype frequency spectrum statistics for soft and hard sweeps. We simulated &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-46&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; sweeps under constant population size with moderately strong selection (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-47&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;) and in which the fixation of the beneficial allele happened immediately prior to sampling (&lt;em&gt;i.e.&lt;/em&gt;, &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-48&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;). The relative magnitude of the number of diploid genotypes, &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-49&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-50&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; and &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-51&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; across sub-windows of a larger simulated region is shown for soft and hard sweeps, respectively.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/6/1959/F2.medium.gif" width="440" height="215"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/8/6/1959/F2.medium.gif" width="440" height="215"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/6/1959/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/6/1959/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/489933" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-14" class="first-child">Multilocus genotype frequency spectrum statistics for soft and hard sweeps. We simulated <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/F2/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/F2/embed/mml-math-46.gif"/></noscript></span></span> sweeps under constant population size with moderately strong selection (<span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/F2/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/F2/embed/mml-math-47.gif"/></noscript></span></span>) and in which the fixation of the beneficial allele happened immediately prior to sampling (<em>i.e.</em>, <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/F2/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/F2/embed/mml-math-48.gif"/></noscript></span></span>). The relative magnitude of the number of diploid genotypes, <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/F2/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/F2/embed/mml-math-49.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/F2/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/F2/embed/mml-math-50.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/F2/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/F2/embed/mml-math-51.gif"/></noscript></span></span> across sub-windows of a larger simulated region is shown for soft and hard sweeps, respectively.</p><div class="sb-div caption-clear"></div></div></div><p id="p-15">Thus our complete feature vector for diploS/HIC consists of 12 summary statistics calculated in each of our 11 sub-windows (132 in total): <em>π</em>, <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-52.gif"/></noscript></span></span> and Tajima’s <em>D</em>, the variance, the skew, and the kurtosis of <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-53.gif"/></noscript></span></span> the number of multilocus genotypes, <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-54.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-55.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-56.gif"/></noscript></span></span> unphased <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-57.gif"/></noscript></span></span> and the maximum value of unphased <em>ω</em>.</p></div><div id="sec-4" class="subsection"><h3>Sweep finding as image recognition</h3><p id="p-16">In our original implementation of S/HIC we used an Extra-Trees classifier to classify genomic windows on the basis of a large vector of summary statistics that had been transformed to capture spatial information. While the classifier was thus able to use spatial information, the actual implementation was completely naive to any notion of spatial structuring among subwindows. In an effort to utilize spatial relationships of summary statistics among windows more effectively, we reasoned that we might be able to represent our transformed summary statistics as an image that we could then train a convolutional neural network (CNN) to recognize. As before, we record our vector of 12 summary statistics across 11 subwindows in which our central window is the one to be classified. Putting summary statistics on rows and subwindows on columns we then arrive at an image representation of a locus on the basis of our collection of summary statistics (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). In <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a> we present averages across each class and in Supplemental Figure S1 we show examples of several individual simulation images; note that any given individual realization shown in Supplemental Figure S1 can differ substantially from the mean. These are the raw material that are used for training our CNN classifier.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/6/1959/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Average-case images for neutral regions, soft sweeps, and hard sweeps. Here we show the average images for three classes of training data, neutral regions, soft sweeps, and hard sweeps, that are used as input to our convolutional neural network for classification. Individual subwindows of the region to be classified are shown in columns. Rows represent each of the 12 summary statistics (see text). The upper end of the color scale has been truncated at 0.25 for visualization. These two images represent the average image from a set of  simulations with parameter values as described in Figure 1." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-34569528" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Average-case images for neutral regions, soft sweeps, and hard sweeps. Here we show the average images for three classes of training data, neutral regions, soft sweeps, and hard sweeps, that are used as input to our convolutional neural network for classification. Individual subwindows of the region to be classified are shown in columns. Rows represent each of the 12 summary statistics (see text). The upper end of the color scale has been truncated at 0.25 for visualization. These two images represent the average image from a set of &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-58&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; simulations with parameter values as described in Figure 1.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/6/1959/F3.medium.gif" width="440" height="133"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/8/6/1959/F3.medium.gif" width="440" height="133"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/6/1959/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/6/1959/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/489966" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-17" class="first-child">Average-case images for neutral regions, soft sweeps, and hard sweeps. Here we show the average images for three classes of training data, neutral regions, soft sweeps, and hard sweeps, that are used as input to our convolutional neural network for classification. Individual subwindows of the region to be classified are shown in columns. Rows represent each of the 12 summary statistics (see text). The upper end of the color scale has been truncated at 0.25 for visualization. These two images represent the average image from a set of <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/F3/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/F3/embed/mml-math-58.gif"/></noscript></span></span> simulations with parameter values as described in <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-5" class="subsection"><h3>Convolutional Neural Network</h3><p id="p-18">To classify genomic regions into one of our five classes (a hard sweep, a soft sweep, a window linked to a hard sweep, a window linked to a soft sweep, or a “neutral” window unlinked to a sweep) diploS/HIC uses a deep convolutional neural network (CNN) trained from simulation data. CNNs are characterized by having one or more “convolutional” layers in their networks whose purpose is to capture essential features of an image by sliding a receptive field of specified size across the image an then computing dot products between the original image values and that of the convolutional filter. CNNs are incredibly powerful tools for image recognition because they reduce the number of parameters necessary for deep learning on images while maintaining enough of the complexity of the original data for accurate classification or regression (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">LeCun <em>et al.</em> (2004)</a>; <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Krizhevsky and Hinton (2010)</a>; <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Krizhevsky <em>et al.</em> (2012)</a>).</p><p id="p-19">diploS/HIC uses a CNN architecture that attempts to capture relationships between windows at multiple physical scales. To do this our input image is used as input to three different branches of a CNN, each of which has two dimensional convolution layers with different sized filters (see <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). Each of these convolutional layers slides a “receptive filter,” basically a focal window, across the input image, taking dot-products between filter weights (<em>i.e.</em>, the fitted parameters of the network layer) and the input itself. Doing this at multiple scales simultaneously allows us to capture how summary stats are responding across the physical chromosome more effectively. The top-most “convolutional unit” has two <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-59.gif"/></noscript></span></span> convolutional layers, where as the other two convolutional units have <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-60.gif"/></noscript></span></span> filters that are dilated such that the receptive filter is rectangular–the middle unit uses a <span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-61.gif"/></noscript></span></span> dilation and the bottom unit uses a <span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-62.gif"/></noscript></span></span> dilation (<em>c.f</em>. <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Yu and Koltun (2015)</a>). In experiments that we do not show here, this structure was found to outperform simpler CNNs that did not attempt to capture multiscale information. Moreover, while the two dimensional convolution structure implies that the summary statistic row ordering might matter, we experimented with different permutations and observed no discernible effect on accuracy. Indeed removing any dependence of row order by changing the convolutional filters to sizes of <span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-63.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-64.gif"/></noscript></span></span> respectively yields no difference in performance (Table S2). Each of the convolutional units used consists of two convolutional layers with ReLu activations, followed by a max pooling layer (to reduce the dimensionality of the representation), which itself is followed by a dropout layer to control overfitting by regularizing the network through setting a subset of weights to zero stochastically. This combination of convolutional layers followed by pooling and dropout layers is quite standard in image recognition (see <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">O’Shea and Nash (2015)</a> for an approachable introduction). At this point in the network the outputs from all three convolutional units are flattened and concatenated and then fed to a series of two fully connected dense layers, between which we again utilize dropout layers, before finally arriving at a softmax activation layer that outputs categorical class membership probabilities (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/6/1959/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Convolutional neural network structure of diploS/HIC. The CNN takes a single image as input and then passes that image to three three convolution layer units each with different filter sizes to capture variation in the image at different physical scales. Each convolution unit consists of two convolution layers followed by a max pooling and a dropout layer. The outputs from the convolutional units are then concatenated and fed to two fully connected dense layers, each themselves followed by dropout. Finally a softmax activation layer is applied to get a categorical classification." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-34569528" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Convolutional neural network structure of diploS/HIC. The CNN takes a single image as input and then passes that image to three three convolution layer units each with different filter sizes to capture variation in the image at different physical scales. Each convolution unit consists of two convolution layers followed by a max pooling and a dropout layer. The outputs from the convolutional units are then concatenated and fed to two fully connected dense layers, each themselves followed by dropout. Finally a softmax activation layer is applied to get a categorical classification.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/6/1959/F4.medium.gif" width="440" height="334"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/8/6/1959/F4.medium.gif" width="440" height="334"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/6/1959/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/6/1959/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/489960" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-20" class="first-child">Convolutional neural network structure of diploS/HIC. The CNN takes a single image as input and then passes that image to three three convolution layer units each with different filter sizes to capture variation in the image at different physical scales. Each convolution unit consists of two convolution layers followed by a max pooling and a dropout layer. The outputs from the convolutional units are then concatenated and fed to two fully connected dense layers, each themselves followed by dropout. Finally a softmax activation layer is applied to get a categorical classification.</p><div class="sb-div caption-clear"></div></div></div><p id="p-21">We trained the diploS/HIC CNN using simulations generated as described above. In general both training and independent test sets consisted of <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-65.gif"/></noscript></span></span> simulations per class unless otherwise specified. Optimization of the CNN parameters was performed using the Adam algorithm (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Kingma and Ba (2014)</a>) tracking as our objective function the accuracy of the classifier on a validation set. Optimization epochs were run until the validation accuracy stopped improving beyond a difference of 0.001 between epochs. In practice this training was quite rapid, with generally less than 20 epochs needed until stopping.</p><p id="p-22">We implement our CNN using the open source package TensorFlow (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Abadi <em>et al.</em> (2016)</a>) and its associated higher level python API Keras (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Chollet <em>et al.</em> (2015)</a>).</p></div><div id="sec-6" class="subsection"><h3>Data Availability</h3><p id="p-23">Our software for performing the diploS/HIC feature vector calculation and deep learning is available from <a href="https://github.com/kern-lab/">https://github.com/kern-lab/</a>. Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.6097337">https://doi.org/10.25387/g3.6097337</a>.</p></div></div><div class="section results" id="sec-7"><h2 class="">Results</h2><p id="p-24">We were first interested in examining how using a CNN with diploS/HIC affected classification accuracy in comparison to the Extra-Trees classifier we had used with S/HIC. For this we generated constant population size simulations with moderately strong selection, <span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-66.gif"/></noscript></span></span> where the beneficial mutation had fixed immediately prior to sampling (<em>i.e.</em>, <span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-67.gif"/></noscript></span></span>) across five different sample sizes of <span class="inline-formula" id="inline-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-68.gif"/></noscript></span></span> haploid chromosomes. The complete parameterization of these simulations is given in Table S1. We were also interested in characterizing the difference between using unphased and phased data in both S/HIC implementations. Our expectation was that phased data should add significant information to the classification task we are interested in. <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a> shows accuracy of four classifiers as a function of sample size. Those classifiers are the original S/HIC classifier (labeled “ETC_haploid”), a CNN classifier which uses the original S/HIC feature vector (“CNN_haploid”), and our new, unphased data feature vectors trained with either a CNN (labeled “CNN”) or an ETC (“ETC”). A few trends are notable. First, it is clear that phased data add useful information for classification– both of our phased (haploid) classifiers are more accurate than our our unphased classifiers. Second, our unphased classifiers themselves are performing quite well even at small sample size (for instance <span class="inline-formula" id="inline-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-69.gif"/></noscript></span></span> accuracy at <span class="inline-formula" id="inline-formula-70"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-70.gif"/></noscript></span></span> diploid individuals). Third, the convolutional neural network approach adds a bit of accuracy, although the gains aren’t dramatic. While that is so, the size of the training set used here, 2000 examples per class, is quite small for deep learning, thus we might expect to see a greater difference between the CNN and the ETC approaches if we were to increase the size of our training set. These trends generalize to stronger and weaker selection although accuracy varies across these cases (see Figures S2 and S3).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/6/1959/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Multiclass classification accuracy for diploS/HIC and related classifiers. Here we compare the multiclass classification accuracy of four related classifiers across a variety of sample sizes: the original S/HIC classifier (&#x201C;ETC_haploid&#x201D;), a S/HIC classifier which uses a CNN rather than an ETC (&#x201C;CNN_haploid&#x201D;), our new classifier for unphased data trained with an ETC (&#x201C;ETC&#x201D;), and finally our new classifier training with a CNN (&#x201C;CNN&#x201D;). Note the y-axis has been truncated to better display the separation between methods." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-34569528" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Multiclass classification accuracy for diploS/HIC and related classifiers. Here we compare the multiclass classification accuracy of four related classifiers across a variety of sample sizes: the original S/HIC classifier (&#x201C;ETC_haploid&#x201D;), a S/HIC classifier which uses a CNN rather than an ETC (&#x201C;CNN_haploid&#x201D;), our new classifier for unphased data trained with an ETC (&#x201C;ETC&#x201D;), and finally our new classifier training with a CNN (&#x201C;CNN&#x201D;). Note the y-axis has been truncated to better display the separation between methods.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/6/1959/F5.medium.gif" width="440" height="328"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/8/6/1959/F5.medium.gif" width="440" height="328"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/6/1959/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/6/1959/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/489939" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-25" class="first-child">Multiclass classification accuracy for diploS/HIC and related classifiers. Here we compare the multiclass classification accuracy of four related classifiers across a variety of sample sizes: the original S/HIC classifier (“ETC_haploid”), a S/HIC classifier which uses a CNN rather than an ETC (“CNN_haploid”), our new classifier for unphased data trained with an ETC (“ETC”), and finally our new classifier training with a CNN (“CNN”). Note the y-axis has been truncated to better display the separation between methods.</p><div class="sb-div caption-clear"></div></div></div><p id="p-26">We next turn our attention to a slightly more realistic scenario in which the beneficial mutation of interest has not fixed immediately before sampling the population, but instead fixed within some uniform range of time before sampling. We generated simulations as before, under constant population size and moderately strong selection, <span class="inline-formula" id="inline-formula-71"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-71.gif"/></noscript></span></span> but allowing the time of fixation, τ, to vary such that <span class="inline-formula" id="inline-formula-72"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-72.gif"/></noscript></span></span> To characterize performance of the classifier under this regime, in <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a> we show ROC curves (<em>i.e.</em>, true positive <em>vs.</em> false positive rates) for our classifiers’ abilities to distinguish between sweeps, both hard and soft, <em>vs.</em> unselected regions, in this case both neutral and linked regions. At all sample sizes we have excellent discriminatory power for this binary classification task. This is also the case for stronger and weaker selection (Figures S4 and S5). Across each of these scenarios considered diploS/HIC is surprisingly competitive with our original, phased data implementation. This suggests that for sweep finding diploS/HIC will be a very useful tool in datasets without information on phase.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/6/1959/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="ROC curves showing the true and false positive rates for various classifiers. Here we show ROC curves for the binary classification task of Hard and Soft vs. linked and neutral regions. ROC curves are shown for both unphased and phased data classifiers as well as for the two machine learning algorithms considered. Each panel represents a different sample size of haploid chromosomes from . Note the scale of the x and y axes does not run from (0,1) but instead is zoomed in to show what little separation among methods there is." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-34569528" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;ROC curves showing the true and false positive rates for various classifiers. Here we show ROC curves for the binary classification task of Hard and Soft &lt;em&gt;vs.&lt;/em&gt; linked and neutral regions. ROC curves are shown for both unphased and phased data classifiers as well as for the two machine learning algorithms considered. Each panel represents a different sample size of haploid chromosomes from &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-73&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;. Note the scale of the x and y axes does not run from (0,1) but instead is zoomed in to show what little separation among methods there is.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/6/1959/F6.medium.gif" width="317" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/8/6/1959/F6.medium.gif" width="317" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/6/1959/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/6/1959/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/489864" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-27" class="first-child">ROC curves showing the true and false positive rates for various classifiers. Here we show ROC curves for the binary classification task of Hard and Soft <em>vs.</em> linked and neutral regions. ROC curves are shown for both unphased and phased data classifiers as well as for the two machine learning algorithms considered. Each panel represents a different sample size of haploid chromosomes from <span class="inline-formula" id="inline-formula-73"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/F6/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/F6/embed/mml-math-73.gif"/></noscript></span></span>. Note the scale of the x and y axes does not run from (0,1) but instead is zoomed in to show what little separation among methods there is.</p><div class="sb-div caption-clear"></div></div></div><p id="p-28">ROC curves show one aspect of our diploS/HIC classifier, but perhaps more illuminating is the examination of classification in a spatial context (<em>i.e.</em>, along a recombining chromosome). To do this we visualize the confusion matrix for 10 subwindows surrounding a central subwindow that has undergone either a hard or soft sweep. Simulated subwindow examples are then classified as belonging to one of five classes (hard sweep, soft sweep, linked to a hard sweep, linked to a soft sweep, or neutral) and the fraction of each such classification is recorded. <a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a> shows such a confusion matrix for a constant size population with moderately strong selective sweeps, <span class="inline-formula" id="inline-formula-74"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-74.gif"/></noscript></span></span> and for an intermediate sample size of <span class="inline-formula" id="inline-formula-75"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-75.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-75.gif"/></noscript></span></span> haploid chromosomes. For this sample size and parameterization diploS/HIC performs quite similarly to our original haploid S/HIC implementation with greater than <span class="inline-formula" id="inline-formula-76"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-76.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-76.gif"/></noscript></span></span> of sweep windows identified correctly and with a false positive rate of <span class="inline-formula" id="inline-formula-77"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-77.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-77.gif"/></noscript></span></span> The original S/HIC had a both a slightly higher accuracy on sweeps (<span class="inline-formula" id="inline-formula-78"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-78.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-78.gif"/></noscript></span></span>) and false positive rate (<span class="inline-formula" id="inline-formula-79"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-79.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-79.gif"/></noscript></span></span>) on the same test scenario (but with a slightly smaller sample size of <span class="inline-formula" id="inline-formula-80"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-80.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-80.gif"/></noscript></span></span> see <a id="xref-ref-32-4" class="xref-bibr" href="#ref-32">Schrider and Kern (2016)</a> <a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a>).</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/6/1959/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Confusion matrices of subwindows across a recombining chromosome. On the y-axis the location of the classified subwindow relative to the sweep is shown while the x-axis shows the predicted class for each subwindow from diploS/HIC. These results are from simulations using the same parameter set as in Figure 6, using a sample size of  haploid chromosomes." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-34569528" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Confusion matrices of subwindows across a recombining chromosome. On the y-axis the location of the classified subwindow relative to the sweep is shown while the x-axis shows the predicted class for each subwindow from diploS/HIC. These results are from simulations using the same parameter set as in Figure 6, using a sample size of &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-81&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; haploid chromosomes.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/6/1959/F7.medium.gif" width="440" height="296"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/8/6/1959/F7.medium.gif" width="440" height="296"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/6/1959/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/6/1959/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/489962" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-29" class="first-child">Confusion matrices of subwindows across a recombining chromosome. On the y-axis the location of the classified subwindow relative to the sweep is shown while the x-axis shows the predicted class for each subwindow from diploS/HIC. These results are from simulations using the same parameter set as in <a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a>, using a sample size of <span class="inline-formula" id="inline-formula-81"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/F7/embed/mml-math-81.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/F7/embed/mml-math-81.gif"/></noscript></span></span> haploid chromosomes.</p><div class="sb-div caption-clear"></div></div></div><p id="p-30">For stronger sweeps we see qualitatively similar patterns, with an excellent ability to localize both hard and soft selective sweeps (<a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure 8</a>). Simulating sweeps with <span class="inline-formula" id="inline-formula-82"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-82.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-82.gif"/></noscript></span></span> but holding constant the rate of recombination means that a greater portion of the chromosome is affected by the sweep. We can see this in our increased misclassification rate of subwindows closest to hard sweeps in <a id="xref-fig-8-2" class="xref-fig" href="#F8">Figure 8</a> relative to <a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7</a>. Nevertheless our multiclass accuracy (the prediction accuracy across all data classes) on stronger sweeps is still high (<span class="inline-formula" id="inline-formula-83"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-83.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-83.gif"/></noscript></span></span> Figure S2).</p><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/6/1959/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Confusion matrices of subwindows across a recombining chromosome with stronger selection. On the y-axis the location of the classified subwindow relative to the sweep is shown while the x-axis shows the predicted class for each subwindow from diploS/HIC. These results are from simulations using the same parameter set as in Figure 6 with the exception of selection which is an order of magnitude stronger,  and again using a sample size of  haploid chromosomes." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-34569528" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Confusion matrices of subwindows across a recombining chromosome with stronger selection. On the y-axis the location of the classified subwindow relative to the sweep is shown while the x-axis shows the predicted class for each subwindow from diploS/HIC. These results are from simulations using the same parameter set as in Figure 6 with the exception of selection which is an order of magnitude stronger, &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-84&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; and again using a sample size of &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-85&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; haploid chromosomes.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 8" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/6/1959/F8.medium.gif" width="440" height="296"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 8" src="https://www.g3journal.org/content/ggg/8/6/1959/F8.medium.gif" width="440" height="296"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/6/1959/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/6/1959/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/489902" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 8</span> <p id="p-31" class="first-child">Confusion matrices of subwindows across a recombining chromosome with stronger selection. On the y-axis the location of the classified subwindow relative to the sweep is shown while the x-axis shows the predicted class for each subwindow from diploS/HIC. These results are from simulations using the same parameter set as in <a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6</a> with the exception of selection which is an order of magnitude stronger, <span class="inline-formula" id="inline-formula-84"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/F8/embed/mml-math-84.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/F8/embed/mml-math-84.gif"/></noscript></span></span> and again using a sample size of <span class="inline-formula" id="inline-formula-85"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/F8/embed/mml-math-85.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/F8/embed/mml-math-85.gif"/></noscript></span></span> haploid chromosomes.</p><div class="sb-div caption-clear"></div></div></div><p id="p-32">On the other hand, when selection is comparatively weak, a relatively smaller portion of the chromosome is perturbed by a selective sweep and thus sweep signatures can be harder to detect, but somewhat easier to localize if detected. In Figure S6 we show confusion matrices associated with an order of magnitude weaker selection, using <span class="inline-formula" id="inline-formula-86"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-86.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-86.gif"/></noscript></span></span> In this scenario we have reduced sensitivity to sweeps, a slightly higher false positive rate, and struggle to determine the mode of selection. With weak selection diploS/HIC is less likely to identify flanking regions as sweeps, though we struggle with classification of hard-linked and soft-linked regions, misclassifying each type both as the other, or often as neutral in the case of soft-linked sites; such errors are of minor concern if one’s primary goal is the discrimination between sweeps and unselected regions. Supporting this point is the fact that our binary classifier (sweep <em>vs.</em> linked+neutral) has a high area under the curve (AUC; <span class="inline-formula" id="inline-formula-87"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-87.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-87.gif"/></noscript></span></span>) even though our multiclass accuracy is lower (<span class="inline-formula" id="inline-formula-88"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-88.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-88.gif"/></noscript></span></span> see Figure S3); thus even with very weak selection our classifier can find sweeps.</p><p id="p-33">Each of the above scenarios simulated populations with constant population size, which is a rarity in the natural world. We thus turn attention to the more realistic case in which population size has changed in recent history. In particular we sought to characterize the performance of diploS/HIC for samples from populations with strong growth, such as we recently observed in a Burkino Faso population of <em>Anopheles gambiae</em> (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Anopheles gambiae 1000 Genomes Consortium (2017)</a>). We inferred using information from the site frequency spectrum that this sample, shorthanded BFS, had experienced greater than 30-fold growth in the past 100,000 years (<a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">Anopheles gambiae 1000 Genomes Consortium (2017)</a>). We simulated training sets under the BFS sample’s inferred demographic parameters (see Table S1) for a variety of sample sizes and selection scenarios. <a id="xref-fig-9-1" class="xref-fig" href="#F9">Figure 9</a> shows the multiclass accuracy of diploS/HIC (labeled CNN) and related classifiers when testing against an independent set of simulations. As in the case of constant population size, the general trend is that CNN-based classifiers consistently outperform those based on Extra-Trees. Moreover in this case diploS/HIC even tends to outperform our original implementation of S/HIC which uses phased haplotypes. Supplementary Figures S7 and S8 show the same comparisons for when selection is an order of magnitude stronger and weaker respectively. Under this demographic scenario in our stronger selection case, <span class="inline-formula" id="inline-formula-89"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-89.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-89.gif"/></noscript></span></span> selection is strong relative to recombination and we lose the ability to localize hard sweeps, resulting in a loss of accuracy. On the contrary our weak selection scenario for <em>Anopheles</em> yields excellent multiclass accuracies. As before, we prefer to visualize performance using confusion matrices, and in <a id="xref-fig-10-1" class="xref-fig" href="#F10">Figure 10</a> we show representative classifications as a function of distance from each sweep type for simulations of BFS demography and moderately strong sweeps. With <span class="inline-formula" id="inline-formula-90"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-90.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-90.gif"/></noscript></span></span> haploid chromosomes we have <span class="inline-formula" id="inline-formula-91"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-91.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-91.gif"/></noscript></span></span> accuracy and our ability to localize sweeps is quite good, although selection in this case is sufficently strong relative to recombination such that we have a significant misclassification rate of windows neighboring hard sweeps into the hard sweep category. If selection is an order of magnitude weaker for BFS demography then our localization for those windows neighboring sweeps improves considerably (Figure S10). However, if selection is an order of magnitude stronger diploS/HIC and related classifiers all struggle to accurately find hard sweeps, as for this parameterization selection is too strong relative to recombination (Figure S9), therefore larger window sizes would be required.</p><div id="F9" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/6/1959/F9.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Multiclass classification accuracy for diploS/HIC and related classifiers for Anopheles demographic history with moderate selection. Here we compare the multiclass classification accuracy of four related classifiers across a variety of sample sizes: the original S/HIC classifier (&#x201C;ETC_haploid&#x201D;), a S/HIC classifier which uses a CNN rather than an ETC (&#x201C;CNN_haploid&#x201D;), our new classifier for unphased data trained with an ETC (&#x201C;ETC&#x201D;), and finally our new classifier training with a CNN (&#x201C;CNN&#x201D;)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-34569528" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Multiclass classification accuracy for diploS/HIC and related classifiers for &lt;em&gt;Anopheles&lt;/em&gt; demographic history with moderate selection. Here we compare the multiclass classification accuracy of four related classifiers across a variety of sample sizes: the original S/HIC classifier (&#x201C;ETC_haploid&#x201D;), a S/HIC classifier which uses a CNN rather than an ETC (&#x201C;CNN_haploid&#x201D;), our new classifier for unphased data trained with an ETC (&#x201C;ETC&#x201D;), and finally our new classifier training with a CNN (&#x201C;CNN&#x201D;).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 9" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/6/1959/F9.medium.gif" width="440" height="324"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 9" src="https://www.g3journal.org/content/ggg/8/6/1959/F9.medium.gif" width="440" height="324"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/6/1959/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 9" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/6/1959/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/489923" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 9</span> <p id="p-34" class="first-child">Multiclass classification accuracy for diploS/HIC and related classifiers for <em>Anopheles</em> demographic history with moderate selection. Here we compare the multiclass classification accuracy of four related classifiers across a variety of sample sizes: the original S/HIC classifier (“ETC_haploid”), a S/HIC classifier which uses a CNN rather than an ETC (“CNN_haploid”), our new classifier for unphased data trained with an ETC (“ETC”), and finally our new classifier training with a CNN (“CNN”).</p><div class="sb-div caption-clear"></div></div></div><div id="F10" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/6/1959/F10.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Confusion matrices of diploS/HIC for an Anopheles gabiae demographic history. On the y-axis the location of the classified subwindow relative to the sweep is shown while the x-axis shows the predicted class for each subwindow from diploS/HIC. These results are from simulations using the parameters reflecting the population history of a BFS sample of Anopheles gambiae, using a sample size of  haploid chromosomes. Here we have simulated sweeps with moderate selection coefficients, ." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-34569528" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Confusion matrices of diploS/HIC for an &lt;em&gt;Anopheles gabiae&lt;/em&gt; demographic history. On the y-axis the location of the classified subwindow relative to the sweep is shown while the x-axis shows the predicted class for each subwindow from diploS/HIC. These results are from simulations using the parameters reflecting the population history of a BFS sample of &lt;em&gt;Anopheles gambiae&lt;/em&gt;, using a sample size of &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-92&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; haploid chromosomes. Here we have simulated sweeps with moderate selection coefficients, &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-93&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 10" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/6/1959/F10.medium.gif" width="440" height="300"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 10" src="https://www.g3journal.org/content/ggg/8/6/1959/F10.medium.gif" width="440" height="300"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/6/1959/F10.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 10" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/6/1959/F10.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/489936" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 10</span> <p id="p-35" class="first-child">Confusion matrices of diploS/HIC for an <em>Anopheles gabiae</em> demographic history. On the y-axis the location of the classified subwindow relative to the sweep is shown while the x-axis shows the predicted class for each subwindow from diploS/HIC. These results are from simulations using the parameters reflecting the population history of a BFS sample of <em>Anopheles gambiae</em>, using a sample size of <span class="inline-formula" id="inline-formula-92"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/F10/embed/mml-math-92.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/F10/embed/mml-math-92.gif"/></noscript></span></span> haploid chromosomes. Here we have simulated sweeps with moderate selection coefficients, <span class="inline-formula" id="inline-formula-93"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/F10/embed/mml-math-93.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/F10/embed/mml-math-93.gif"/></noscript></span></span>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-36">We previously showed our S/HIC classifier to be quite robust to demographic model misspecification (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Schrider <em>et al.</em> (2016)</a>), thus we were curious to see if our new implementation should share similar properties. <em>A priori</em> we expect diploS/HIC to be robust, as such robustness stems from the summary statistic transform that we apply to our input statistics, rather than some higher level property of the ML algorithm. To characterize accuracy in the face of model misspecification we used the classifiers that we had trained on constant size population simulations (<em>e.g.</em>, <a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>) to make predictions from data drawn from the BFS sample demographic model (<em>i.e.</em>, strong population growth). In Figure S11 we show ROC curves for both our original S/HIC and diploS/HIC with and without model misspecification for each of a set of five sample sizes. Both S/HIC and diploS/HIC are indeed robust to model misspecification and while S/HIC is more accurate, owing to its use of haploid information, diploS/HIC is more robust to such misspecification.</p><p id="p-37">Practicing population geneticists often are faced with data can only be phased computationally, rather than through transmission or inbreeding. We were thus interested in characterizing the performance of our classifiers in the face of phasing switch errors. To explore this, we trained haploid and diploid diploS/HIC classifiers, along with S/HIC classifiers on data that contained no switch errors, but then tested those trained classifiers on data which had switch errors inserted at various probabilities (<em>i.e.</em>, genotypes at heterozygous sites were flipped at a given rate). In Figure S12 we show the accuracy of three classifiers as a function of switch error rate. Those classifiers are the original S/HIC classifier (labeled “ETC_haploid”), a CNN classifier which uses the original S/HIC feature vector (“CNN_haploid”), and our unphased diploS/HIC classifier (labeled “CNN”). As expected the accuracy of the unphased classifier does not change with increasing switch error rate, however both of the classifiers that make use of phase information do lose accuracy. Thus if there is reason to believe that computational phasing might lead to appreciable error rates we recommend that the user should use the default, unphased version of diploS/HIC.</p><p id="p-38">In general we find that under a more complex history of population growth, such as that from the BFS sample, diploS/HIC performs quite well at finding both hard and soft sweeps and differentiating among them. Moreover we find that diploS/HIC is quite robust to model misspecification during training. This is the case even for reasonably small population sizes, although our smallest sample size tested did lag in accuracy considerably. Thus a fair degree of caution should be taken when analyzing smaller samples (<em>i.e.</em>, <span class="inline-formula" id="inline-formula-94"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-94.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-94.gif"/></noscript></span></span>) using our method.</p><div id="sec-8" class="subsection"><h3>An application to Anopheles gabiae genome sequence data</h3><p id="p-39">As an example of how diploS/HIC might be used to elucidate the evolutionary history of an organism we here provide a targeted look at the <em>Gste</em> region of the <em>A. gambiae</em> genome. Over the last 20 years as part of the Roll Back Malaria initiative, mosquito populations have been subject to large-scale application of insecticides and as a consequence have begun to adapt to this new environment through the evolution of resistance (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Hemingway <em>et al.</em> (2016)</a>). The <em>Gste</em> locus contains a cluster of glutathione S-transferase genes including <em>Gste2</em>, which has previously been shown to be involved in detoxification of pyrethroids and DDT (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Mitchell <em>et al.</em> (2014)</a>). In <a id="xref-fig-11-1" class="xref-fig" href="#F11">Figure 11</a> we show diploS/HIC classifications of windows throughout the region at three physical scales (10kb, 20kb, and 50kb). For computational convenience, we trained a classifier using simulations of genetic distances that correspond to ∼5kb, but predict on feature vectors from a larger region. Though ideally simulation with a matching window size should be used, this is acceptable for our purposes here because our feature vector itself is scale free, and the ratio of the recombination rate to the selection coefficient is what determines the spatial patterns of variation across a region. At each scale a soft sweep is identified at the <em>Gste</em> cluster, and one can see from patterns of Tajimas’s <em>D</em>, <span class="inline-formula" id="inline-formula-95"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-95.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-95.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-96"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-96.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6/1959/embed/mml-math-96.gif"/></noscript></span></span> that the signal of selection extends for a long distance in each direction away from the sweep. This squares nicely with the haplotype based analysis done by <a id="xref-ref-2-4" class="xref-bibr" href="#ref-2">Anopheles gambiae 1000 Genomes Consortium (2017)</a> which showed that numerous haplotypes were under selection at <em>Gste</em>. We have included the workflow of this entire empirical application as an example use case of diploS/HIC here: <a href="https://github.com/kern-lab/diploSHIC/wiki/A-soup-to-nuts-example">https://github.com/kern-lab/diploSHIC/wiki/A-soup-to-nuts-example</a>.</p><div id="F11" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/6/1959/F11.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A large soft sweep at Gste in Anopheles gambiae. Here we show diploS/HIC classifications at three physical scales along with associated summary statistics for a megabase region surrounding the Gste cluster on chromosome 3R for the BFS sample. diploS/HIC classifications are colored here such that windows classified as soft sweeps are dark blue, linkedSoft as light blue, and linkedHard as light red. Above the diploS/HIC classifications we show a subset of the summary statistics used as part of the feature vector (see Methods for details)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-34569528" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;A large soft sweep at &lt;em&gt;Gste&lt;/em&gt; in &lt;em&gt;Anopheles gambiae&lt;/em&gt;. Here we show diploS/HIC classifications at three physical scales along with associated summary statistics for a megabase region surrounding the &lt;em&gt;Gste&lt;/em&gt; cluster on chromosome 3R for the BFS sample. diploS/HIC classifications are colored here such that windows classified as soft sweeps are dark blue, linkedSoft as light blue, and linkedHard as light red. Above the diploS/HIC classifications we show a subset of the summary statistics used as part of the feature vector (see Methods for details).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 11" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/6/1959/F11.medium.gif" width="440" height="240"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 11" src="https://www.g3journal.org/content/ggg/8/6/1959/F11.medium.gif" width="440" height="240"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/6/1959/F11.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 11" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/6/1959/F11.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/489881" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 11</span> <p id="p-40" class="first-child">A large soft sweep at <em>Gste</em> in <em>Anopheles gambiae</em>. Here we show diploS/HIC classifications at three physical scales along with associated summary statistics for a megabase region surrounding the <em>Gste</em> cluster on chromosome 3R for the BFS sample. diploS/HIC classifications are colored here such that windows classified as soft sweeps are dark blue, linkedSoft as light blue, and linkedHard as light red. Above the diploS/HIC classifications we show a subset of the summary statistics used as part of the feature vector (see Methods for details).</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-9"><h2 class="">Discussion</h2><p id="p-41">Creating a complete catalog of selective sweeps in the genome of any organism remains an elusive goal. Population genetics, as a field, requires knowledge of the targets of selection for at least two separate purposes. At a broad scale, we would like to know about genome-wide parameters such as the rate of selective sweeps per generation and their associated distribution of selective effects. At a finer scale, we would like to know the precise location of individual beneficial fixations, so that we can learn about the functional ramifications of adaptive evolution. Thus through the creation of a catalog of selective sweeps we hope to learn both about the process of adaptation as well as the nature of the adaptations themselves.</p><p id="p-42">While methods development for sweep finding tools has a rich history in the literature, in the past few years substantial performance gains have come through leveraging supervised machine learning approaches (reviewed in <a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Schrider and Kern (2018)</a>). In particular, the use of machine learning has led to more powerful methods for detecting sweeps that are robust to complicated demographic histories which had confounded earlier approaches. While this is so, nearly all machine learning methods for finding selection require the use of phased haplotype information for input (the sole exception being SFSselect (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Ronen <em>et al.</em> (2013)</a>), which is based strictly on the site frequency spectrum). In this report we present a new method that we call diploS/HIC that has exceptional power for finding both hard and soft selective sweeps and differentiating among them without requiring phased data. diploS/HIC builds on the strategy of our earlier method, S/HIC (<a id="xref-ref-32-5" class="xref-bibr" href="#ref-32">Schrider and Kern (2016)</a>), in utilizing a large vector of summary statistics that have been transformed to capture spatial information around a focal region to be classified. For diploS/HIC we have introduced a number of new population genetic summary statistics that are compatible with unphased genotypes and which are useful for detecting and discriminating between hard and soft selective sweeps. Note that, like the original S/HIC, diploS/HIC was designed with completed sweeps in mind, but could be extended to handle incomplete sweeps in a straightforward manner by simulating appropriate training data and including additional summary statistics.</p><p id="p-43">Algorithmically diploS/HIC utilizes a deep convolutional neural network (CNN) as the basis of its classification (<a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">LeCun <em>et al.</em> (2004)</a>; <a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">Krizhevsky and Hinton (2010)</a>; <a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">Krizhevsky <em>et al.</em> (2012)</a>). Thus, we cast the sweep finding problem as one of image recognition. Our goal in doing so is to more explicitly utilize the spatial covariance structure of summary statistics in a given region, beyond the simple spatial transformation we had been using in <a id="xref-ref-32-6" class="xref-bibr" href="#ref-32">Schrider and Kern (2016)</a>. While it seems this has succeeded to some degree (<em>i.e.</em>, we see that a CNN classifier using the S/HIC feature vector outperforms our original implementation – <a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5</a>) we suspect that there is still considerable room for improvement. First, the use of recurrent neural network approaches that are specifically formulated for capturing signals in sequential data should provide a principled way of gleaning further spatial signal using summary statistics (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Graves <em>et al.</em> (2013)</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Sutskever <em>et al.</em> (2014)</a>). Second, we imagine that improvements in using the full information in the spatial arrangement of polymorphism surrounding a sweep could come from abandoning summary statistic representations of loci, and instead using image recognition (<em>i.e.</em>, CNN) approaches on images created directly from sequence alignments of a genomic region (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Chan <em>et al.</em> (2018)</a>; L. Flagel, personal communication). A similar approach using images of read pileups was recently introduced for variant calling, yielding solid performance improvements over competing methods (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Poplin <em>et al.</em> (2017)</a>).</p><p id="p-44">In summary we have shown that diploS/HIC performs quite well when compared to our previous S/HIC under both simple and more complex population size histories. Identifying selective sweeps in populations with non-equilibrium demographic histories remains an important and difficult problem, particularly in cases where the underlying demographic model is unknown or poorly estimated. For instance it is well known that certain demographic models, such as population growth, can mimic selective perturbations (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Simonsen <em>et al.</em> (1995)</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Jensen <em>et al.</em> (2005)</a>). Here we have shown as with S/HIC before it, diploS/HIC is accurate in the face of non-equilibrium demography, even when misspecified. In conclusion, we believe that diploS/HIC provides yet another powerful tool for population geneticists to use when phased information is unavailable.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-45">We thank Jeff Adrion, Jeff Ross-Ibarra, Alexander Xue, and two anonymous reviewers for comments on this manuscript. ADK was supported by National Institutes of Health award no. R01GM117241. DRS was supported by NIH award no. K99HG008696.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-46">Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.6097337">https://doi.org/10.25387/g3.6097337</a>.</p></li><li class="fn" id="fn-2"><p id="p-47"><em>Communicating editor: J. Ross-Ibarra</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2018-02-26"><span class="received-label">Received </span>February 26, 2018.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2018-04-04"><span class="accepted-label">Accepted </span>April 4, 2018.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2018 Kern, Schrider</li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-web" id="cit-8.6.1959.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Abadi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Agarwal</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barham</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brevdo</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-comment">Tensorflow: Large-scale machine learning on heterogeneous distributed systems. arXiv preprint arXiv:1603.04467</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-8.6.1959.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">Anopheles gambiae 1000 Genomes Consortium</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genetic diversity of the African malaria vector Anopheles gambiae.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">552</span>: <span class="cit-fpage">96</span>–<span class="cit-lpage">100</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D552%26rft.spage%253D96%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-web" id="cit-8.6.1959.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perrone</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spence</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jenkins</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mathieson</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-comment">A likelihood-free inference framework for population genetic data using exchangeable neural networks. arXiv preprint arXiv:1802.06153</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-web" id="cit-8.6.1959.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chollet</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-comment">Keras. <a href="https://github.com/fchollet/keras">https://github.com/fchollet/keras</a></span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-8.6.1959.5" data-doi="10.1093/bioinformatics/btw051"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">DeGiorgio</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huber</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hubisz</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hellmann</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Sweepfinder2: increased sensitivity, robustness and flexibility.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">32</span>(<span class="cit-issue">12</span>): <span class="cit-fpage">1895</span>–<span class="cit-lpage">1897</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btw051</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtw051%26rft_id%253Dinfo%253Apmid%252F27153702%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btw051&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27153702&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F6%2F1959.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-8.6.1959.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fay</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">C. I.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Hitchhiking under positive darwinian selection.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">155</span>: <span class="cit-fpage">1405</span>–<span class="cit-lpage">1413</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DFay%26rft.auinit1%253DJ.%2BC.%26rft.volume%253D155%26rft.issue%253D3%26rft.spage%253D1405%26rft.epage%253D1413%26rft.atitle%253DHitchhiking%2BUnder%2BPositive%2BDarwinian%2BSelection%26rft_id%253Dinfo%253Apmid%252F10880498%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1NS8zLzE0MDUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzYvMTk1OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-8.6.1959.7" data-doi="10.1371/journal.pgen.1005004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Garud</span>  <span class="cit-name-given-names">N. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Messer</span>  <span class="cit-name-given-names">P. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buzbas</span>  <span class="cit-name-given-names">E. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petrov</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Recent selective sweeps in north american drosophila melanogaster show signatures of soft sweeps.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">11</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">e1005004</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1005004</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D11%26rft.spage%253De1005004%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1005004%26rft_id%253Dinfo%253Apmid%252F25706129%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1005004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25706129&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F6%2F1959.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-8.6.1959.8" data-doi="10.1007/s10994-006-6226-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Geurts</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ernst</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wehenkel</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Extremely randomized trees.</span> <abbr class="cit-jnl-abbrev">Mach. Learn.</abbr> <span class="cit-vol">63</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">3</span>–<span class="cit-lpage">42</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s10994-006-6226-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMach.%2BLearn.%26rft.volume%253D63%26rft.spage%253D3%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10994-006-6226-1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10994-006-6226-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-book" id="cit-8.6.1959.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Graves</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mohamed</span>  <span class="cit-name-given-names">A.-r.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hinton</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> Speech recognition with deep recurrent neural networks, pp. <span class="cit-fpage">6645</span>–<span class="cit-lpage">6649</span> in <em>IEEE</em> International Conference on Acoustics, Speech and Signal Processing (ICASSP). <span class="cit-publ-name">IEEE</span>, Vancouver, Canada.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-8.6.1959.10" data-doi="10.1016/S0140-6736(15)00417-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hemingway</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ranson</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Magill</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kolaczinski</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fornadel</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Averting a malaria disaster: will insecticide resistance derail malaria control?</span> <abbr class="cit-jnl-abbrev">Lancet</abbr> <span class="cit-vol">387</span>(<span class="cit-issue">10029</span>): <span class="cit-fpage">1785</span>–<span class="cit-lpage">1788</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0140-6736(15)00417-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLancet%26rft.volume%253D387%26rft.spage%253D1785%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0140-6736%252815%252900417-1%26rft_id%253Dinfo%253Apmid%252F26880124%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0140-6736(15)00417-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26880124&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F6%2F1959.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-8.6.1959.11" data-doi="10.1534/genetics.104.036947"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hermisson</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pennings</span>  <span class="cit-name-given-names">P. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Soft sweeps: molecular population genetics of adaptation from standing genetic variation.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">169</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">2335</span>–<span class="cit-lpage">2352</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.104.036947</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.036947%26rft_id%253Dinfo%253Apmid%252F15716498%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2OS80LzIzMzUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzYvMTk1OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-8.6.1959.12" data-doi="10.1038/ncomms6281"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jensen</span>  <span class="cit-name-given-names">J. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">On the unfounded enthusiasm for soft selective sweeps.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">5281</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ncomms6281</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D5%26rft.spage%253D5281%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms6281%26rft_id%253Dinfo%253Apmid%252F25345443%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms6281&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25345443&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F6%2F1959.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-8.6.1959.13" data-doi="10.1534/genetics.104.038224"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jensen</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">DuMont</span>  <span class="cit-name-given-names">V. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aquadro</span>  <span class="cit-name-given-names">C. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bustamante</span>  <span class="cit-name-given-names">C. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Distinguishing between selective sweeps and demography using dna polymorphism data.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">170</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">1401</span>–<span class="cit-lpage">1410</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.104.038224</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.038224%26rft_id%253Dinfo%253Apmid%252F15911584%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3MC8zLzE0MDEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzYvMTk1OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-8.6.1959.14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kaplan</span>  <span class="cit-name-given-names">N. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hudson</span>  <span class="cit-name-given-names">R. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Langley</span>  <span class="cit-name-given-names">C. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">The hitchhiking effect revisited.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">123</span>: <span class="cit-fpage">887</span>–<span class="cit-lpage">899</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKaplan%26rft.auinit1%253DN.%2BL.%26rft.volume%253D123%26rft.issue%253D4%26rft.spage%253D887%26rft.epage%253D899%26rft.atitle%253DThe%2B%2522hitchhiking%2Beffect%2522%2Brevisited.%26rft_id%253Dinfo%253Apmid%252F2612899%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTIzLzQvODg3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC82LzE5NTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-8.6.1959.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kelly</span>  <span class="cit-name-given-names">J. K.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">A test of neutrality based on interlocus associations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">146</span>: <span class="cit-fpage">1197</span>–<span class="cit-lpage">1206</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKelly%26rft.auinit1%253DJ.%2BK.%26rft.volume%253D146%26rft.issue%253D3%26rft.spage%253D1197%26rft.epage%253D1206%26rft.atitle%253DA%2BTest%2Bof%2BNeutrality%2BBased%2Bon%2BInterlocus%2BAssociations%26rft_id%253Dinfo%253Apmid%252F9215920%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE0Ni8zLzExOTciO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzYvMTk1OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-8.6.1959.16" data-doi="10.1093/bioinformatics/btw556"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kern</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schrider</span>  <span class="cit-name-given-names">D. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Discoal: flexible coalescent simulations with selection.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">32</span>(<span class="cit-issue">24</span>): <span class="cit-fpage">3839</span>–<span class="cit-lpage">3841</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btw556</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtw556%26rft_id%253Dinfo%253Apmid%252F27559153%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btw556&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27559153&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F6%2F1959.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-8.6.1959.17" data-doi="10.1534/genetics.103.025387"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Linkage disequilibrium as a signature of selective sweeps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">167</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">1513</span>–<span class="cit-lpage">1524</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.103.025387</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKim%26rft.auinit1%253DY.%26rft.volume%253D167%26rft.issue%253D3%26rft.spage%253D1513%26rft.epage%253D1524%26rft.atitle%253DLinkage%2BDisequilibrium%2Bas%2Ba%2BSignature%2Bof%2BSelective%2BSweeps%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.103.025387%26rft_id%253Dinfo%253Apmid%252F15280259%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2Ny8zLzE1MTMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzYvMTk1OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-web" id="cit-8.6.1959.18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kingma</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ba</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> Adam: A method for stochastic optimization. arXiv preprint arXiv:1412.6980.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-other" id="cit-8.6.1959.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Krizhevsky</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hinton</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> Convolutional deep belief networks on cifar-10. Unpublished manuscript 40.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-book" id="cit-8.6.1959.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Krizhevsky</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sutskever</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hinton</span>  <span class="cit-name-given-names">G. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Imagenet classification with deep convolutional neural networks</span>, pp. <span class="cit-fpage">1097</span>–<span class="cit-lpage">1105</span>, in <span class="cit-source"><em>Advances in Neural Information Processing Systems</em></span>, edited by <span class="cit-ed"><span class="cit-name-surname">Pereira</span>  <span class="cit-name-given-names">F.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Burges</span>  <span class="cit-name-given-names">C.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Bottou</span>  <span class="cit-name-given-names">L.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Weinberger</span>  <span class="cit-name-given-names">K.</span></span>. <span class="cit-publ-name">Curran Associates</span>, <span class="cit-publ-loc">Red Hook, NY</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-confproc" id="cit-8.6.1959.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">LeCun</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">F. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bottou</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> Learning methods for generic object recognition with invariance to pose and lighting. In <em>Computer Vision and Pattern Recognition</em>, <em>2004</em>. <em>CVPR 2004</em>. <em>Proceedings of the 2004 IEEE Computer Society Conference on</em>, volume 2, pp. II–104, IEEE.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-8.6.1959.22" data-doi="10.1534/genetics.110.122614"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schlötterer</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Futschik</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Distinguishing positive selection from neutral evolution: boosting the performance of summary statistics.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">187</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">229</span>–<span class="cit-lpage">244</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.110.122614</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.122614%26rft_id%253Dinfo%253Apmid%252F21041556%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg3LzEvMjI5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC82LzE5NTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-8.6.1959.23" data-doi="10.1016/j.tree.2013.08.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Messer</span>  <span class="cit-name-given-names">P. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petrov</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Population genomics of rapid adaptation by soft selective sweeps.</span> <abbr class="cit-jnl-abbrev">Trends Ecol. Evol.</abbr> <span class="cit-vol">28</span>(<span class="cit-issue">11</span>): <span class="cit-fpage">659</span>–<span class="cit-lpage">669</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tree.2013.08.003</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BEcol.%2BEvol.%26rft.volume%253D28%26rft.spage%253D659%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tree.2013.08.003%26rft_id%253Dinfo%253Apmid%252F24075201%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tree.2013.08.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24075201&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F6%2F1959.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000326666200007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-8.6.1959.24" data-doi="10.1371/journal.pone.0092662"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mitchell</span>  <span class="cit-name-given-names">S. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rigden</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dowd</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilding</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Metabolic and target-site mechanisms combine to confer strong ddt resistance in anopheles gambiae.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">e92662</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0092662</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De92662%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0092662%26rft_id%253Dinfo%253Apmid%252F24675797%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0092662&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24675797&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F6%2F1959.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-8.6.1959.25" data-doi="10.1101/gr.4252305"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williamson</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hubisz</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Genomic scans for selective sweeps using snp data.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">15</span>(<span class="cit-issue">11</span>): <span class="cit-fpage">1566</span>–<span class="cit-lpage">1575</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.4252305</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DNielsen%26rft.auinit1%253DR.%26rft.volume%253D15%26rft.issue%253D11%26rft.spage%253D1566%26rft.epage%253D1575%26rft.atitle%253DGenomic%2Bscans%2Bfor%2Bselective%2Bsweeps%2Busing%2BSNP%2Bdata%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.4252305%26rft_id%253Dinfo%253Apmid%252F16251466%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxNS8xMS8xNTY2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC82LzE5NTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-web" id="cit-8.6.1959.26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">O’Shea</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nash</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> An introduction to convolutional neural networks. arXiv preprint arXiv:1511.08458.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-8.6.1959.27" data-doi="10.1534/genetics.110.116459"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pavlidis</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jensen</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephan</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Searching for footprints of positive selection in whole-genome SNP data from nonequilibrium populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">185</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">907</span>–<span class="cit-lpage">922</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.110.116459</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.116459%26rft_id%253Dinfo%253Apmid%252F20407129%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg1LzMvOTA3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC82LzE5NTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-web" id="cit-8.6.1959.28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Poplin</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Newburger</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dijamco</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nguyen</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Loy</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Creating a universal snp and small indel variant caller with deep neural networks.</span> <span class="cit-source">bioRxiv</span> <a href="https://doi.org/10.1101/092890">https://doi.org/10.1101/092890</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-8.6.1959.29" data-doi="10.1093/bioinformatics/btv493"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pybus</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luisi</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dall’Olio</span>  <span class="cit-name-given-names">G. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Uzkudun</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laayouni</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Hierarchical boosting: a machine-learning framework to detect and classify hard selective sweeps in human populations.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">3946</span>–<span class="cit-lpage">3952</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtv493%26rft_id%253Dinfo%253Apmid%252F26315912%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btv493&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26315912&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F6%2F1959.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-8.6.1959.30" data-doi="10.1534/genetics.108.093153"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rogers</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huff</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Linkage disequilibrium between loci with unknown phase.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">182</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">839</span>–<span class="cit-lpage">844</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.108.093153</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.093153%26rft_id%253Dinfo%253Apmid%252F19433632%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgyLzMvODM5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC82LzE5NTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-8.6.1959.31" data-doi="10.1534/genetics.113.152587"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ronen</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Udpa</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Halperin</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bafna</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Learning natural selection from the site frequency spectrum.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">195</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">181</span>–<span class="cit-lpage">193</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.113.152587</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.152587%26rft_id%253Dinfo%253Apmid%252F23770700%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk1LzEvMTgxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC82LzE5NTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-8.6.1959.32" data-doi="10.1371/journal.pgen.1005928"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schrider</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kern</span>  <span class="cit-name-given-names">A. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">S/hic: Robust identification of soft and hard sweeps using machine learning.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">12</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">e1005928</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1005928</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D12%26rft.spage%253De1005928%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1005928%26rft_id%253Dinfo%253Apmid%252F26977894%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1005928&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26977894&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F6%2F1959.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-8.6.1959.33" data-doi="10.1093/molbev/msx154"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schrider</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kern</span>  <span class="cit-name-given-names">A. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Soft sweeps are the dominant mode of adaptation in the human genome.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">34</span>(<span class="cit-issue">8</span>): <span class="cit-fpage">1863</span>–<span class="cit-lpage">1877</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msx154</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft.volume%253D34%26rft.spage%253D1863%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsx154%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msx154&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-8.6.1959.34" data-doi="10.1016/j.tig.2017.12.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schrider</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kern</span>  <span class="cit-name-given-names">A. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Supervised machine learning for population genetics: a new paradigm.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">34</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">301</span>–<span class="cit-lpage">312</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tig.2017.12.005</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BGenet.%26rft.volume%253D34%26rft.spage%253D301%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2017.12.005%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2017.12.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-8.6.1959.35" data-doi="10.1534/genetics.115.174912"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schrider</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mendes</span>  <span class="cit-name-given-names">F. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hahn</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kern</span>  <span class="cit-name-given-names">A. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Soft shoulders ahead: spurious signatures of soft and partial selective sweeps result from linked hard sweeps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">200</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">267</span>–<span class="cit-lpage">284</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.115.174912</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.174912%26rft_id%253Dinfo%253Apmid%252F25716978%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjAwLzEvMjY3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC82LzE5NTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-8.6.1959.36" data-doi="10.1534/genetics.116.190223"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schrider</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shanku</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kern</span>  <span class="cit-name-given-names">A. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Effects of linked selective sweeps on demographic inference and model selection.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">204</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">1207</span>–<span class="cit-lpage">1223</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.116.190223</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.116.190223%26rft_id%253Dinfo%253Apmid%252F27605051%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIwNC8zLzEyMDciO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzYvMTk1OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-8.6.1959.37" data-doi="10.1371/journal.pcbi.1004845"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sheehan</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">Y. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Deep learning for population genetic inference.</span> <abbr class="cit-jnl-abbrev">PLOS Comput. Biol.</abbr> <span class="cit-vol">12</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">e1004845</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pcbi.1004845</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BComput.%2BBiol.%26rft.volume%253D12%26rft.spage%253De1004845%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1004845%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1004845&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-8.6.1959.38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Simonsen</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aquadro</span>  <span class="cit-name-given-names">C. F.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Properties of statistical tests of neutrality for dna polymorphism data.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">141</span>: <span class="cit-fpage">413</span>–<span class="cit-lpage">429</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DSimonsen%26rft.auinit1%253DK.%2BL.%26rft.volume%253D141%26rft.issue%253D1%26rft.spage%253D413%26rft.epage%253D429%26rft.atitle%253DProperties%2Bof%2Bstatistical%2Btests%2Bof%2Bneutrality%2Bfor%2BDNA%2Bpolymorphism%2Bdata.%26rft_id%253Dinfo%253Apmid%252F8536987%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTQxLzEvNDEzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC82LzE5NTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-8.6.1959.39" data-doi="10.1017/S0016672300014634"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haigh</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1974</span> <span class="cit-article-title">The hitch-hiking effect of a favourable gene.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">23</span>(<span class="cit-issue">01</span>): <span class="cit-fpage">23</span>–<span class="cit-lpage">35</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S0016672300014634</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DSmith%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D23%26rft.issue%253D1%26rft.spage%253D23%26rft.epage%253D35%26rft.atitle%253DThe%2Bhitch-hiking%2Beffect%2Bof%2Ba%2Bfavourable%2Bgene.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672300014634%26rft_id%253Dinfo%253Apmid%252F4407212%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672300014634&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=4407212&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F6%2F1959.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1974T216900003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-book" id="cit-8.6.1959.40"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sutskever</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vinyals</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Le</span>  <span class="cit-name-given-names">Q. V.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Sequence to sequence learning with neural networks</span>, pp. <span class="cit-fpage">3104</span>–<span class="cit-lpage">3112</span> in <span class="cit-source"><em>Advances in Neural Information Processing Systems</em></span>. <span class="cit-publ-name">Curran Associates</span>, <span class="cit-publ-loc">Red Hook, NY</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-8.6.1959.41"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tajima</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-article-title">Evolutionary relationship of dna sequences in finite populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">437</span>–<span class="cit-lpage">460</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DTajima%26rft.auinit1%253DF.%26rft.volume%253D105%26rft.issue%253D2%26rft.spage%253D437%26rft.epage%253D460%26rft.atitle%253DEVOLUTIONARY%2BRELATIONSHIP%2BOF%2BDNA%2BSEQUENCES%2BIN%2BFINITE%2BPOPULATIONS%26rft_id%253Dinfo%253Apmid%252F6628982%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTA1LzIvNDM3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC82LzE5NTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-8.6.1959.42"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tajima</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Statistical method for testing the neutral mutation hypothesis by dna polymorphism.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">123</span>: <span class="cit-fpage">585</span>–<span class="cit-lpage">595</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DTajima%26rft.auinit1%253DF.%26rft.volume%253D123%26rft.issue%253D3%26rft.spage%253D585%26rft.epage%253D595%26rft.atitle%253DStatistical%2Bmethod%2Bfor%2Btesting%2Bthe%2Bneutral%2Bmutation%2Bhypothesis%2Bby%2BDNA%2Bpolymorphism.%26rft_id%253Dinfo%253Apmid%252F2513255%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTIzLzMvNTg1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC82LzE5NTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-8.6.1959.43" data-doi="10.1016/0040-5809(75)90020-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Watterson</span>  <span class="cit-name-given-names">G. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1975</span> <span class="cit-article-title">On the number of segregating sites in genetical models without recombination.</span> <abbr class="cit-jnl-abbrev">Theor. Popul. Biol.</abbr> <span class="cit-vol">7</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">256</span>–<span class="cit-lpage">276</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/0040-5809(75)90020-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheoretical%2Bpopulation%2Bbiology%26rft.stitle%253DTheor%2BPopul%2BBiol%26rft.aulast%253DWatterson%26rft.auinit1%253DG.%2BA.%26rft.volume%253D7%26rft.issue%253D2%26rft.spage%253D256%26rft.epage%253D276%26rft.atitle%253DOn%2Bthe%2Bnumber%2Bof%2Bsegregating%2Bsites%2Bin%2Bgenetical%2Bmodels%2Bwithout%2Brecombination.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0040-5809%252875%252990020-9%26rft_id%253Dinfo%253Apmid%252F1145509%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0040-5809(75)90020-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1145509&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F6%2F1959.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1975W295600011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-web" id="cit-8.6.1959.44"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koltun</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> Multi-scale context aggregation by dilated convolutions. arXiv preprint arXiv:1511.07122.</cite></div><div class="cit-extra"></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/8/6/1959.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/6/1943" title="The Pseudokinase Domain of Saccharomyces cerevisiae Tra1 Is Required for Nuclear Localization and Incorporation into the SAGA and NuA4 Complexes" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/6/1971" title="Sex Differences in Recombination in Sticklebacks" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/8/6"><span>Volume 8 Issue 6, June 2018</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/8/6.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/8/6.cover-source.jpg?itok=4pU6R124" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 8 (6)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/6/1959" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-nPJXO4wX9tJdK2Wy1dojGnF66W-BaxyHc0b2GS8Soo0" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/8/6/1959" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/8/6/1959" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/8/6/1959" class="active" data-icon-position="" data-hide-link-title="0">diploS/HIC: An Updated Approach to Classifying Selective Sweeps</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/489846" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-w4wjs_J5MCaiTvSW9h682iOAe9cFUT-OtNW5LQJtrkY" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/6/1959.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/8/6/1959" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-THg2slpDq7Nc2iTMCtmkinf-jyEi2bKLt13_siLYDRs" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/8/6/1959" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/6/1959.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="489846" id="node489846--2" data-pisa="ggg;8/6/1959" data-pisa-master="ggg;g3.118.200262" data-apath="/ggg/8/6/1959.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">diploS/HIC: An Updated Approach to Classifying Selective Sweeps</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-4381-4680" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Andrew D. Kern</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-5249-4151" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Daniel R. Schrider</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1959-1970;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200262 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node489846--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Andrew D. Kern </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genetics, Rutgers University, Piscataway, NJ 08854</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrew%2BD.%2BKern%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kern%20AD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAndrew%2BD.%2BKern%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-4381-4680" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Andrew D. Kern</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:kern@biology.rutgers.edu" class="" data-icon-position="" data-hide-link-title="0">kern@biology.rutgers.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Daniel R. Schrider </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genetics, Rutgers University, Piscataway, NJ 08854</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BR.%2BSchrider%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schrider%20DR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADaniel%2BR.%2BSchrider%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5249-4151" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Daniel R. Schrider</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/8/6/1959" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/8/6/1959&amp;title=diploS/HIC%3A%20An%20Updated%20Approach%20to%20Classifying%20Selective%20Sweeps" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/8/6/1959&amp;title=diploS/HIC%3A%20An%20Updated%20Approach%20to%20Classifying%20Selective%20Sweeps" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/8/6/1959&amp;title=diploS/HIC%3A%20An%20Updated%20Approach%20to%20Classifying%20Selective%20Sweeps" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/8/6/1959&amp;text=diploS/HIC%3A%20An%20Updated%20Approach%20to%20Classifying%20Selective%20Sweeps" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/8/6/1959&amp;title=diploS/HIC%3A%20An%20Updated%20Approach%20to%20Classifying%20Selective%20Sweeps" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/8/6/1959&amp;t=diploS/HIC%3A%20An%20Updated%20Approach%20to%20Classifying%20Selective%20Sweeps" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/8/6/1959&amp;title=diploS/HIC%3A%20An%20Updated%20Approach%20to%20Classifying%20Selective%20Sweeps" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/8/6/1959&amp;title=diploS/HIC%3A%20An%20Updated%20Approach%20to%20Classifying%20Selective%20Sweeps" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/489846/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="489846" id="node489846--3" data-pisa="ggg;8/6/1959" data-pisa-master="ggg;g3.118.200262" data-apath="/ggg/8/6/1959.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">diploS/HIC: An Updated Approach to Classifying Selective Sweeps</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-4381-4680" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Andrew D. Kern</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-5249-4151" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Daniel R. Schrider</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1959-1970;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200262 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node489846--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Andrew D. Kern </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genetics, Rutgers University, Piscataway, NJ 08854</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrew%2BD.%2BKern%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kern%20AD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAndrew%2BD.%2BKern%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-4381-4680" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Andrew D. Kern</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:kern@biology.rutgers.edu" class="" data-icon-position="" data-hide-link-title="0">kern@biology.rutgers.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Daniel R. Schrider </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genetics, Rutgers University, Piscataway, NJ 08854</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BR.%2BSchrider%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schrider%20DR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADaniel%2BR.%2BSchrider%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5249-4151" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Daniel R. Schrider</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1959&amp;rft.epage=1970&amp;rft.atitle=diploS/HIC%3A%20An%20Updated%20Approach%20to%20Classifying%20Selective%20Sweeps&amp;rft.volume=8&amp;rft.issue=6&amp;rft.date=2018-06-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Kern%2C+Andrew+D.&amp;rft.au=Schrider%2C+Daniel+R."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/489846/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/489846/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/489846/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/489846/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/489846/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/489846/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/489846/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/489846/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/489846/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/489846/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/489846/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/489846/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/8/6/1959" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Methods and Model</a></li><li class="odd"><a href="#sec-7" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-9" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/8/6/1959.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/8/6/1959.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/8/6/1959.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/8/6/1959.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
