<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/8/6/1921" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/8/6/1921.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/8/6/1921.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/8/6/1921.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/8/6/1921.atom" />
<meta name="HW.pisa" content="ggg;8/6/1921" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Subtle Perturbations of the Maize Methylome Reveal Genes and Transposons Silenced by Chromomethylase or RNA-Directed DNA Methylation Pathways" />
<meta name="DC.Identifier" content="10.1534/g3.118.200284" />
<meta name="DC.Date" content="2018-06-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2018 Anderson et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="DNA methylation is a chromatin modification that can provide epigenetic regulation of gene and transposon expression. Plants utilize several pathways to establish and maintain DNA methylation in specific sequence contexts. The chromomethylase (CMT) genes maintain CHG (where H = A, C or T) methylation. The RNA-directed DNA methylation (RdDM) pathway is important for CHH methylation. Transcriptome analysis was performed in a collection of Zea mays lines carrying mutant alleles for CMT or RdDM-associated genes. While the majority of the transcriptome was not affected, we identified sets of genes and transposon families sensitive to context-specific decreases in DNA methylation in mutant lines. Many of the genes that are up-regulated in CMT mutant lines have high levels of CHG methylation, while genes that are differentially expressed in RdDM mutants are enriched for having nearby mCHH islands, implicating context-specific DNA methylation in the regulation of expression for a small number of genes. Many genes regulated by CMTs exhibit natural variation for DNA methylation and transcript abundance in a panel of diverse inbred lines. Transposon families with differential expression in the mutant genotypes show few defining features, though several families up-regulated in RdDM mutants show enriched expression in endosperm tissue, highlighting the potential importance for this pathway during reproduction. Taken together, our findings suggest that while the number of genes and transposon families whose expression is reproducibly affected by mild perturbations in context-specific methylation is small, there are distinct patterns for loci impacted by RdDM and CMT mutants." />
<meta name="DC.Contributor" content="Sarah N. Anderson" />
<meta name="DC.Contributor" content="Gregory J. Zynda" />
<meta name="DC.Contributor" content="Jawon Song" />
<meta name="DC.Contributor" content="Zhaoxue Han" />
<meta name="DC.Contributor" content="Matthew W. Vaughn" />
<meta name="DC.Contributor" content="Qing Li" />
<meta name="DC.Contributor" content="Nathan M. Springer" />
<meta name="article:published_time" content="2018-06-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Subtle Perturbations of the Maize Methylome Reveal Genes and Transposons Silenced by Chromomethylase or RNA-Directed DNA Methylation Pathways" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;DNA methylation is a chromatin modification that can provide epigenetic regulation of gene and transposon expression. Plants utilize several pathways to establish and maintain DNA methylation in specific sequence contexts. The chromomethylase (CMT) genes maintain CHG (where H = A, C or T) methylation. The RNA-directed DNA methylation (RdDM) pathway is important for CHH methylation. Transcriptome analysis was performed in a collection of &lt;i&gt;Zea mays&lt;/i&gt; lines carrying mutant alleles for CMT or RdDM-associated genes. While the majority of the transcriptome was not affected, we identified sets of genes and transposon families sensitive to context-specific decreases in DNA methylation in mutant lines. Many of the genes that are up-regulated in CMT mutant lines have high levels of CHG methylation, while genes that are differentially expressed in RdDM mutants are enriched for having nearby mCHH islands, implicating context-specific DNA methylation in the regulation of expression for a small number of genes. Many genes regulated by CMTs exhibit natural variation for DNA methylation and transcript abundance in a panel of diverse inbred lines. Transposon families with differential expression in the mutant genotypes show few defining features, though several families up-regulated in RdDM mutants show enriched expression in endosperm tissue, highlighting the potential importance for this pathway during reproduction. Taken together, our findings suggest that while the number of genes and transposon families whose expression is reproducibly affected by mild perturbations in context-specific methylation is small, there are distinct patterns for loci impacted by RdDM and CMT mutants.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2018/06/01" />
<meta name="citation_mjid" content="ggg;8/6/1921" />
<meta name="citation_id" content="8/6/1921" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/8/6/1921" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/8/6/1921.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/8/6/1921.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/8/6/1921.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.118.200284" />
<meta name="citation_pmid" content="29618467" />
<meta name="citation_volume" content="8" />
<meta name="citation_issue" content="6" />
<meta name="citation_num_pages" content="12" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1921" />
<meta name="citation_lastpage" content="1932" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Sarah N. Anderson" />
<meta name="citation_author_institution" content="Department of Plant and Microbial Biology, University of Minnesota, St. Paul, MN 55108" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-1671-2286" />
<meta name="citation_author" content="Gregory J. Zynda" />
<meta name="citation_author_institution" content="Texas Advanced Computing Center, University of Texas, Austin, TX 78758" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-4992-9614" />
<meta name="citation_author" content="Jawon Song" />
<meta name="citation_author_institution" content="Texas Advanced Computing Center, University of Texas, Austin, TX 78758" />
<meta name="citation_author" content="Zhaoxue Han" />
<meta name="citation_author_institution" content="State Key Laboratory of Crop Stress Biology for Arid Areas, Northwest A&amp;F University, Yangling 712100, Shaanxi, China" />
<meta name="citation_author" content="Matthew W. Vaughn" />
<meta name="citation_author_institution" content="Texas Advanced Computing Center, University of Texas, Austin, TX 78758" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-1384-4283" />
<meta name="citation_author" content="Qing Li" />
<meta name="citation_author_institution" content="National Key Laboratory of Crop Genetic Improvement, Huazhong Agricultural University, Wuhan, Hubei, 430070, China" />
<meta name="citation_author_email" content="springer@umn.edu" />
<meta name="citation_author_email" content="qingli@mail.hzau.edu.cn" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-3232-9479" />
<meta name="citation_author" content="Nathan M. Springer" />
<meta name="citation_author_institution" content="Department of Plant and Microbial Biology, University of Minnesota, St. Paul, MN 55108" />
<meta name="citation_author_email" content="springer@umn.edu" />
<meta name="citation_author_email" content="qingli@mail.hzau.edu.cn" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-7301-4759" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=M. Alleman;citation_author=L. Sidorenko;citation_author=K. McGinnis;citation_author=V. Seshadri;citation_author=J. E. Dorweiler;citation_title=An RNA-dependent RNA polymerase is required for paramutation in maize.;citation_pages=295-298;citation_volume=442;citation_year=2006;citation_issue=7100;citation_pmid=16855589;citation_doi=10.1038/nature04884" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=S. Anders;citation_author=P. T. Pyl;citation_author=W. Huber;citation_title=HTSeq–a Python framework to work with high-throughput sequencing data.;citation_volume=31;citation_year=2015;citation_issue=2;citation_pmid=25260700;citation_doi=10.1093/bioinformatics/btu638" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=W. T. Barber;citation_author=W. Zhang;citation_author=H. Win;citation_author=K. K. Varala;citation_author=J. E. Dorweiler;citation_title=Repeat associated small RNAs vary among parents and following hybridization in maize.;citation_volume=109;citation_year=2012;citation_issue=26;citation_pmid=22689990;citation_doi=10.1073/pnas.1202073109" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=A. J. Bewick;citation_author=C. E. Niederhuth;citation_author=L. Ji;citation_author=N. A. Rohr;citation_author=P. T. Griffin;citation_title=The evolution of CHROMOMETHYLASES and gene body DNA methylation in plants.;citation_pages=65;citation_volume=18;citation_year=2017;citation_issue=1;citation_doi=10.1186/s13059-017-1195-1" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=D. Bouyer;citation_author=A. Kramdi;citation_author=M. Kassam;citation_author=M. Heese;citation_author=A. Schnittger;citation_title=DNA methylation dynamics during early plant life.;citation_pages=179;citation_volume=18;citation_year=2017;citation_issue=1;citation_doi=10.1186/s13059-017-1313-0" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=R. A. Brink;citation_title=A GENETIC CHANGE ASSOCIATED WITH THE R LOCUS IN MAIZE WHICH IS DIRECTED AND POTENTIALLY REVERSIBLE;citation_pages=872-889;citation_volume=41;citation_year=1956;citation_issue=6;citation_pmid=17247669" />
<meta name="citation_reference" content="citation_journal_title=Plant J.;citation_author=A. B. Brohammer;citation_author=T. J. Y. Kono;citation_author=N. M. Springer;citation_author=S. E. McGaugh;citation_author=C. N. Hirsch;citation_title=The limited role of differential fractionation in genome content variation and function in maize (Zea mays L.) inbred lines.;citation_pages=131-141;citation_volume=93;citation_year=2018" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=V. V. Cavrak;citation_author=N. Lettner;citation_author=S. Jamge;citation_author=A. Kosarewicz;citation_author=L. M. Bayer;citation_title=How a retrotransposon exploits the plant’s heat stress response for its activation.;citation_pages=e1004115;citation_volume=10;citation_year=2014;citation_issue=1;citation_pmid=24497839;citation_doi=10.1371/journal.pgen.1004115" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=V. L. Chandler;citation_title=Paramutation: from maize to mice.;citation_pages=641-645;citation_volume=128;citation_year=2007;citation_issue=4;citation_pmid=17320501;citation_doi=10.1016/j.cell.2007.02.007" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=V. L. Chandler;citation_author=V. Walbot;citation_title=DNA modification of a maize transposable element correlates with loss of activity.;citation_pages=1767-1771;citation_volume=83;citation_year=1986;citation_issue=6;citation_pmid=3006070;citation_doi=10.1073/pnas.83.6.1767" />
<meta name="citation_reference" content="citation_journal_title=EMBO J.;citation_author=P. S. Chomet;citation_author=S. Wessler;citation_author=S. L. Dellaporta;citation_title=Inactivation of the maize transposable element Activator (Ac) is associated with its DNA modification.;citation_pages=295-302;citation_volume=6;citation_year=1987;citation_pmid=3034583" />
<meta name="citation_reference" content="citation_journal_title=Mol. Plant;citation_author=X. Dong;citation_author=M. Zhang;citation_author=J. Chen;citation_author=L. Peng;citation_author=N. Zhang;citation_title=Dynamic and Antagonistic Allele-Specific Epigenetic Modifications Controlling the Expression of Imprinted Genes in Maize Endosperm.;citation_pages=442-455;citation_volume=10;citation_year=2017;citation_issue=3;citation_doi=10.1016/j.molp.2016.10.007" />
<meta name="citation_reference" content="citation_journal_title=The Plant Cell Online;citation_journal_abbrev=Plant Cell;citation_author=J. E. Dorweiler;citation_author=C. C. Carey;citation_author=K. M. Kubo;citation_author=J. B. Hollick;citation_author=J. L. Kermicle;citation_title=mediator of paramutation1 Is Required for Establishment and Maintenance of Paramutation at Multiple Maize Loci;citation_pages=2101-2118;citation_volume=12;citation_year=2000;citation_issue=11;citation_pmid=11090212;citation_doi=10.1105/tpc.12.11.2101" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=S. R. Eichten;citation_author=R. Briskine;citation_author=J. Song;citation_author=Q. Li;citation_author=R. Swanson-Wagner;citation_title=Epigenetic and genetic influences on DNA methylation variation in maize populations.;citation_volume=25;citation_year=2013;citation_issue=8;citation_pmid=23922207;citation_doi=10.1105/tpc.113.114793" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=S. R. Eichten;citation_author=R. A. Swanson-Wagner;citation_author=J. C. Schnable;citation_author=A. J. Waters;citation_author=P. J. Hermanson;citation_title=Heritable epigenetic variation among maize inbreds.;citation_pages=e1002372-e1002372;citation_volume=7;citation_year=2011;citation_issue=11;citation_pmid=22125494;citation_doi=10.1371/journal.pgen.1002372" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=K. F. Erhard;citation_author=J. L. Stonaker;citation_author=S. E. Parkinson;citation_author=J. P. Lim;citation_author=C. J. Hale;citation_title=RNA Polymerase IV Functions in Paramutation in Zea mays;citation_pages=1201-1205;citation_volume=323;citation_year=2009;citation_issue=5918;citation_pmid=19251626;citation_doi=10.1126/science.1164508" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=K. F. Erhard;citation_author=J.-E. R. B. Talbot;citation_author=N. C. Deans;citation_author=A. E. McClish;citation_author=J. B. Hollick;citation_title=Nascent transcription affected by RNA polymerase IV in Zea mays.;citation_volume=199;citation_year=2015;citation_issue=4;citation_pmid=25653306;citation_doi=10.1534/genetics.115.174714" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=C. Forestan;citation_author=R. Aiese Cigliano;citation_author=S. Farinati;citation_author=A. Lunardon;citation_author=W. Sanseverino;citation_title=Stress-induced and epigenetic-mediated maize transcriptome regulation study by means of transcriptome reannotation and differential expression analysis.;citation_pages=30446;citation_volume=6;citation_year=2016;citation_issue=1;citation_doi=10.1038/srep30446" />
<meta name="citation_reference" content="citation_journal_title=BMC Plant Biol.;citation_author=C. Forestan;citation_author=S. Farinati;citation_author=R. Aiese Cigliano;citation_author=A. Lunardon;citation_author=W. Sanseverino;citation_title=Maize RNA PolIV affects the expression of genes with nearby TE insertions and has a genome-wide repressive impact on transcription.;citation_pages=161;citation_volume=17;citation_year=2017;citation_issue=1;citation_doi=10.1186/s12870-017-1108-1" />
<meta name="citation_reference" content="citation_journal_title=Annu. Rev. Genet.;citation_author=M. Gehring;citation_title=Genomic imprinting: insights from plants.;citation_pages=187-208;citation_volume=47;citation_year=2013;citation_issue=1;citation_pmid=24016190;citation_doi=10.1146/annurev-genet-110711-155527" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=J. I. Gent;citation_author=N. A. Ellis;citation_author=L. Guo;citation_author=A. E. Harkess;citation_author=Y. Yao;citation_title=CHH islands: de novo DNA methylation in near-gene chromatin regulation in maize.;citation_volume=23;citation_year=2013;citation_issue=4;citation_pmid=23269663;citation_doi=10.1101/gr.146985.112" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=J. I. Gent;citation_author=T. F. Madzima;citation_author=R. Bader;citation_author=M. R. Kent;citation_author=X. Zhang;citation_title=Accessible DNA and relative depletion of H3K9me2 at maize loci undergoing RNA-directed DNA methylation.;citation_volume=26;citation_year=2014;citation_issue=12;citation_pmid=25465407;citation_doi=10.1105/tpc.114.130427" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=Q. Gouil;citation_author=D. C. Baulcombe;citation_title=DNA Methylation Signatures of the Plant Chromomethyltransferases.;citation_pages=e1006526;citation_volume=12;citation_year=2016;citation_issue=12;citation_doi=10.1371/journal.pgen.1006526" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=C. J. Hale;citation_author=J. L. Stonaker;citation_author=S. M. Gross;citation_author=J. B. Hollick;citation_title=A novel Snf2 protein maintains trans-generational regulatory states established by paramutation in maize.;citation_pages=e275-e275;citation_volume=5;citation_year=2007;citation_issue=10;citation_pmid=17941719;citation_doi=10.1371/journal.pbio.0050275" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=B. T. Hofmeister;citation_author=K. Lee;citation_author=N. A. Rohr;citation_author=D. W. Hall;citation_author=R. J. Schmitz;citation_title=Stable inheritance of DNA methylation allows creation of epigenotype maps and the study of epiallele inheritance patterns in the absence of genetic variation.;citation_pages=155;citation_volume=18;citation_year=2017;citation_issue=1;citation_doi=10.1186/s13059-017-1288-x" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=J. B. Hollick;citation_title=Paramutation and related phenomena in diverse species.;citation_pages=5-23;citation_volume=18;citation_year=2017;citation_issue=1;citation_doi=10.1038/nrg.2016.115" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. B. Hollick;citation_author=J. L. Kermicle;citation_author=S. E. Parkinson;citation_title=Rmr6 maintains meiotic inheritance of paramutant states in Zea mays.;citation_volume=171;citation_year=2005;citation_issue=2;citation_pmid=16020780;citation_doi=10.1534/genetics.105.045260" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=L. Hu;citation_author=N. Li;citation_author=C. Xu;citation_author=S. Zhong;citation_author=X. Lin;citation_title=Mutation of a major CG methylase in rice causes genome-wide hypomethylation, dysregulated genome expression, and seedling lethality.;citation_volume=111;citation_year=2014;citation_issue=29;citation_pmid=25002488;citation_doi=10.1073/pnas.1410761111" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=Y. Jia;citation_author=D. R. Lisch;citation_author=K. Ohtsu;citation_author=M. J. Scanlon;citation_author=D. Nettleton;citation_title=Loss of RNA-dependent RNA polymerase 2 (RDR2) function causes widespread and unexpected changes in the expression of transposons, genes, and 24-nt small RNAs.;citation_pages=e1000737-e1000737;citation_volume=5;citation_year=2009;citation_issue=11;citation_pmid=19936292;citation_doi=10.1371/journal.pgen.1000737" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=Y. Jiao;citation_author=P. Peluso;citation_author=J. Shi;citation_author=T. Liang;citation_author=M. C. Stitzer;citation_title=Improved maize reference genome with single-molecule technologies.;citation_pages=524-527;citation_volume=546;citation_year=2017;citation_doi=10.1038/nature22971" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=T. Kawakatsu;citation_author=J. R. Nery;citation_author=R. Castanon;citation_author=J. R. Ecker;citation_title=Dynamic DNA methylation reconfiguration during seed development and germination.;citation_pages=171;citation_volume=18;citation_year=2017;citation_issue=1;citation_doi=10.1186/s13059-017-1251-x" />
<meta name="citation_reference" content="citation_journal_title=Nat. Plants;citation_author=T. Kawakatsu;citation_author=T. Stuart;citation_author=M. Valdes;citation_author=N. Breakfield;citation_author=R. J. Schmitz;citation_title=Unique cell-type-specific patterns of DNA methylation in the root meristem.;citation_pages=16058;citation_volume=2;citation_year=2016;citation_issue=5;citation_doi=10.1038/nplants.2016.58" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. L. Kermicle;citation_title=DEPENDENCE OF THE R-MOTTLED ALEURONE PHENOTYPE IN MAIZE ON MODE OF SEXUAL TRANSMISSION;citation_pages=69-85;citation_volume=66;citation_year=1970;citation_issue=1;citation_pmid=17248508" />
<meta name="citation_reference" content="citation_author=J. L. Kermicle;citation_author=M. Alleman;citation_year=1990" />
<meta name="citation_reference" content="citation_author=D. Kim;citation_author=G. Pertea;citation_author=C. Trapnell;citation_author=H. Pimentel;citation_author=R. Kelley;citation_pages=R36;citation_volume=14;citation_year=2013;citation_pmid=23618408;citation_doi=10.1186/gb-2013-14-4-r36" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=J. A. Law;citation_author=S. E. Jacobsen;citation_title=Establishing, maintaining and modifying DNA methylation patterns in plants and animals.;citation_pages=204-220;citation_volume=11;citation_year=2010;citation_issue=3;citation_pmid=20142834;citation_doi=10.1038/nrg2719" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=Q. Li;citation_author=S. R. Eichten;citation_author=P. J. Hermanson;citation_author=V. M. Zaunbrecher;citation_author=J. Song;citation_title=Genetic perturbation of the maize methylome.;citation_volume=26;citation_year=2014;citation_issue=12;citation_pmid=25527708;citation_doi=10.1105/tpc.114.133140" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=Q. Li;citation_author=J. I. Gent;citation_author=G. Zynda;citation_author=J. Song;citation_author=I. Makarevitch;citation_title=RNA-directed DNA methylation enforces boundaries between heterochromatin and euchromatin in the maize genome.;citation_volume=112;citation_year=2015;citation_issue=47;citation_pmid=26553984;citation_doi=10.1073/pnas.1514680112" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=D. Lisch;citation_author=C. C. Carey;citation_author=J. E. Dorweiler;citation_author=V. L. Chandler;citation_title=A mutation that prevents paramutation in maize also reverses Mutator transposon methylation and silencing.;citation_volume=99;citation_year=2002;citation_issue=9;citation_pmid=11959901;citation_doi=10.1073/pnas.052152199" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=Q. Li;citation_author=J. Song;citation_author=P. T. West;citation_author=G. Zynda;citation_author=S. R. Eichten;citation_title=Examining the causes and consequences of context-specific differential DNA methylation in maize.;citation_volume=168;citation_year=2015;citation_issue=4;citation_pmid=25869653;citation_doi=10.1104/pp.15.00052" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=M. I. Love;citation_author=W. Huber;citation_author=S. Anders;citation_title=Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2.;citation_pages=550;citation_volume=15;citation_year=2014;citation_issue=12;citation_pmid=25516281;citation_doi=10.1186/s13059-014-0550-8" />
<meta name="citation_reference" content="citation_journal_title=Epigenetics;citation_author=T. F. Madzima;citation_author=J. Huang;citation_author=K. M. McGinnis;citation_title=Chromatin structure and gene expression changes associated with loss of MOP1 activity in Zea mays.;citation_pages=1047-1059;citation_volume=9;citation_year=2014;citation_issue=7;citation_pmid=24786611;citation_doi=10.4161/epi.29022" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=I. Makarevitch;citation_author=R. M. Stupar;citation_author=A. L. Iniguez;citation_author=W. J. Haun;citation_author=W. B. Barbazuk;citation_title=Natural Variation for Alleles Under Epigenetic Control by the Maize Chromomethylase Zmet2.;citation_volume=177;citation_year=2007;citation_issue=2;citation_pmid=17660570;citation_doi=10.1534/genetics.107.072702" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in plant biology;citation_journal_abbrev=Curr Opin Plant Biol;citation_author=G. Martínez;citation_author=R. K. Slotkin;citation_title=Developmental relaxation of transposable element silencing in plants: functional or byproduct?;citation_pages=496-502;citation_volume=15;citation_year=2012;citation_issue=5;citation_pmid=23022393;citation_doi=10.1016/j.pbi.2012.09.001" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=M. A. Matzke;citation_author=R. A. Mosher;citation_title=RNA-directed DNA methylation: an epigenetic pathway of increasing complexity.;citation_pages=394-408;citation_volume=15;citation_year=2014;citation_issue=6;citation_pmid=24805120;citation_doi=10.1038/nrg3683" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=K. M. McGinnis;citation_author=C. Springer;citation_author=Y. Lin;citation_author=C. C. Carey;citation_author=V. Chandler;citation_title=Transcriptionally silenced transgenes in maize are activated by three mutations defective in paramutation.;citation_volume=173;citation_year=2006;citation_issue=3;citation_pmid=16702420;citation_doi=10.1534/genetics.106.058669" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=M. Mirouze;citation_author=J. Reinders;citation_author=E. Bucher;citation_author=T. Nishimura;citation_author=K. Schneeberger;citation_title=Selective epigenetic control of retrotransposition in Arabidopsis.;citation_pages=427-430;citation_volume=461;citation_year=2009;citation_issue=7262;citation_pmid=19734882;citation_doi=10.1038/nature08328" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=A. Miura;citation_author=S. Yonebayashi;citation_author=K. Watanabe;citation_author=T. Toyama;citation_author=H. Shimada;citation_title=Mobilization of transposons by a mutation abolishing full DNA methylation in Arabidopsis.;citation_pages=212-214;citation_volume=411;citation_year=2001;citation_issue=6834;citation_pmid=11346800;citation_doi=10.1038/35075612" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=R. Narsai;citation_author=Q. Gouil;citation_author=D. Secco;citation_author=A. Srivastava;citation_author=Y. V. Karpievitch;citation_title=Extensive transcriptomic and epigenomic remodelling occurs during Arabidopsis thaliana germination.;citation_pages=172;citation_volume=18;citation_year=2017;citation_issue=1;citation_doi=10.1186/s13059-017-1302-3" />
<meta name="citation_reference" content="citation_journal_title=The Plant Cell Online;citation_journal_abbrev=Plant Cell;citation_author=C. M. Papa;citation_author=N. M. Springer;citation_author=M. G. Muszynski;citation_author=R. Meeley;citation_author=S. M. Kaeppler;citation_title=Maize Chromomethylase Zea methyltransferase2 Is Required for CpNpG Methylation;citation_pages=1919-1928;citation_volume=13;citation_year=2001;citation_issue=8;citation_pmid=11487702;citation_doi=10.1105/tpc.13.8.1919" />
<meta name="citation_reference" content="citation_journal_title=Developmental biology;citation_journal_abbrev=Dev Biol;citation_author=S. E. Parkinson;citation_author=S. M. Gross;citation_author=J. B. Hollick;citation_title=Maize sex determination and abaxial leaf fates are canalized by a factor that maintains repressed epigenetic states.;citation_pages=462-473;citation_volume=308;citation_year=2007;citation_issue=2;citation_pmid=17612519;citation_doi=10.1016/j.ydbio.2007.06.004" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. R. Quinlan;citation_author=I. M. Hall;citation_title=BEDTools: a flexible suite of utilities for comparing genomic features.;citation_volume=26;citation_year=2010;citation_issue=6;citation_pmid=20110278;citation_doi=10.1093/bioinformatics/btq033" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=M. Regulski;citation_author=Z. Lu;citation_author=J. Kendall;citation_author=M. T. Donoghue;citation_author=J. Reinders;citation_title=The maize methylome influences mRNA splice sites and reveals widespread paramutation-like switches guided by small RNA.;citation_volume=23;citation_year=2013;citation_issue=10;citation_pmid=23739895;citation_doi=10.1101/gr.153510.112" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=M. J. Rowley;citation_author=M. H. Rothi;citation_author=G. Böhmdorfer;citation_author=J. Kuciński;citation_author=A. T. Wierzbicki;citation_title=Long-range control of gene expression via RNA-directed DNA methylation.;citation_pages=e1006749;citation_volume=13;citation_year=2017;citation_issue=5;citation_doi=10.1371/journal.pgen.1006749" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. E. Sloan;citation_author=L. Sidorenko;citation_author=K. M. McGinnis;citation_title=Diverse gene-silencing mechanisms with distinct requirements for RNA polymerase subunits in Zea mays.;citation_volume=198;citation_year=2014;citation_issue=3;citation_pmid=25164883;citation_doi=10.1534/genetics.114.168518" />
<meta name="citation_reference" content="citation_title=Current Protocols in Plant Biology;citation_author=J. Song;citation_author=G. Zynda;citation_author=S. Beck;citation_author=N. M. Springer;citation_author=M. W. Vaughn;citation_title=Bisulfite Sequence Analyses Using CyVerse Discovery Environment: From Mapping to DMRs;citation_year=2016;citation_doi=10.1002/cppb.20034" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=P. Wang;citation_author=H. Xia;citation_author=Y. Zhang;citation_author=S. Zhao;citation_author=C. Zhao;citation_title=Genome-wide high-resolution mapping of DNA methylation identifies epigenetic variation across embryo and endosperm in Maize (Zea may).;citation_pages=21;citation_volume=16;citation_year=2015;citation_issue=1;citation_pmid=25612809;citation_doi=10.1186/s12864-014-1204-7" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=T. Wicker;citation_author=F. Sabot;citation_author=A. Hua-Van;citation_author=J. L. Bennetzen;citation_author=P. Capy;citation_title=A unified classification system for eukaryotic transposable elements.;citation_pages=973-982;citation_volume=8;citation_year=2007;citation_issue=12;citation_pmid=17984973;citation_doi=10.1038/nrg2165" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=B. P. Williams;citation_author=D. Pignatta;citation_author=S. Henikoff;citation_author=M. Gehring;citation_title=Methylation-sensitive expression of a DNA demethylase gene serves as an epigenetic rheostat.;citation_pages=e1005142;citation_volume=11;citation_year=2015;citation_issue=3;citation_pmid=25826366;citation_doi=10.1371/journal.pgen.1005142" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. R. Woodhouse;citation_author=M. Freeling;citation_author=D. Lisch;citation_title=The mop1 (mediator of paramutation1) mutant progressively reactivates one of the two genes encoded by the MuDR transposon in maize.;citation_volume=172;citation_year=2006;citation_issue=1;citation_pmid=16219782;citation_doi=10.1534/genetics.105.051383" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=Y. Xi;citation_author=W. Li;citation_title=BSMAP: whole genome bisulfite sequence MAPping program.;citation_pages=232-232;citation_volume=10;citation_year=2009;citation_issue=1;citation_pmid=19635165;citation_doi=10.1186/1471-2105-10-232" />
<meta name="citation_reference" content="citation_journal_title=Plant Mol. Biol.;citation_author=T. Yamauchi;citation_author=Y. Johzuka-Hisatomi;citation_author=R. Terada;citation_author=I. Nakamura;citation_author=S. Iida;citation_title=The MET1b gene encoding a maintenance DNA methyltransferase is indispensable for normal development in rice.;citation_pages=219-232;citation_volume=85;citation_year=2014;citation_issue=3;citation_pmid=24535433;citation_doi=10.1007/s11103-014-0178-9" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Subtle Perturbations of the Maize Methylome Reveal Genes and Transposons Silenced by Chromomethylase or RNA-Directed DNA Methylation Pathways" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6.cover-source.jpg" />
<meta name="twitter:description" content="DNA methylation is a chromatin modification that can provide epigenetic regulation of gene and transposon expression. Plants utilize several pathways to establish and maintain DNA methylation in specific sequence contexts. The chromomethylase (CMT) genes maintain CHG (where H = A, C or T) methylation. The RNA-directed DNA methylation (RdDM) pathway is important for CHH methylation. Transcriptome analysis was performed in a collection of Zea mays lines carrying mutant alleles for CMT or RdDM-associated genes. While the majority of the transcriptome was not affected, we identified sets of genes and transposon families sensitive to context-specific decreases in DNA methylation in mutant lines. Many of the genes that are up-regulated in CMT mutant lines have high levels of CHG methylation, while genes that are differentially expressed in RdDM mutants are enriched for having nearby mCHH islands, implicating context-specific DNA methylation in the regulation of expression for a small number of genes. Many genes regulated by CMTs exhibit natural variation for DNA methylation and transcript abundance in a panel of diverse inbred lines. Transposon families with differential expression in the mutant genotypes show few defining features, though several families up-regulated in RdDM mutants show enriched expression in endosperm tissue, highlighting the potential importance for this pathway during reproduction. Taken together, our findings suggest that while the number of genes and transposon families whose expression is reproducibly affected by mild perturbations in context-specific methylation is small, there are distinct patterns for loci impacted by RdDM and CMT mutants." />
<meta name="og-title" property="og:title" content="Subtle Perturbations of the Maize Methylome Reveal Genes and Transposons Silenced by Chromomethylase or RNA-Directed DNA Methylation Pathways" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/8/6/1921" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="DNA methylation is a chromatin modification that can provide epigenetic regulation of gene and transposon expression. Plants utilize several pathways to establish and maintain DNA methylation in specific sequence contexts. The chromomethylase (CMT) genes maintain CHG (where H = A, C or T) methylation. The RNA-directed DNA methylation (RdDM) pathway is important for CHH methylation. Transcriptome analysis was performed in a collection of Zea mays lines carrying mutant alleles for CMT or RdDM-associated genes. While the majority of the transcriptome was not affected, we identified sets of genes and transposon families sensitive to context-specific decreases in DNA methylation in mutant lines. Many of the genes that are up-regulated in CMT mutant lines have high levels of CHG methylation, while genes that are differentially expressed in RdDM mutants are enriched for having nearby mCHH islands, implicating context-specific DNA methylation in the regulation of expression for a small number of genes. Many genes regulated by CMTs exhibit natural variation for DNA methylation and transcript abundance in a panel of diverse inbred lines. Transposon families with differential expression in the mutant genotypes show few defining features, though several families up-regulated in RdDM mutants show enriched expression in endosperm tissue, highlighting the potential importance for this pathway during reproduction. Taken together, our findings suggest that while the number of genes and transposon families whose expression is reproducibly affected by mild perturbations in context-specific methylation is small, there are distinct patterns for loci impacted by RdDM and CMT mutants." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/6.cover-source.jpg" />
    <title>Subtle Perturbations of the Maize Methylome Reveal Genes and Transposons Silenced by Chromomethylase or RNA-Directed DNA Methylation Pathways | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"om4LxK1lwKX4GfBZ6VTuCOZ5dUYXDN140ko91FkGT2Q","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"489827","apath":"\/ggg\/8\/6\/1921.atom","pisa":"ggg;8\/6\/1921","ac":{"\/ggg\/8\/6\/1921.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/8\/6\/1921.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;8\/6\/1921"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;8\/6\/1921"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/8\/6\/1921":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00220pdE+RgIZZCSrNbrdWXQwQMGA9s4w5WCmXGHQvBkWEQn7i40meijLA8wQMnEibbzqsKFHdfmPl7xEDFTV808JbbtS05Ad+76m2NkYI1E3yxC715J\\\\\\\/7gA2Sw0ABUYJX8G8wui020cTdjZTRkRE9B+mqpGMwwUPnaq9tDBNHx6P820CMTZc1SRi3rbCTcBGifOCjqA+DWf7KfyNed6X7M+NG4RlYRdvBKSIP7b7L18\\\\\\\/B7ATKjb6r5rg5ww5tTrvikU2EpX6sMCTe5szs++h5YRX+D505rNbq1OPya09razmY8rDbmBt6FfQYliP\\\\\\\/jSZw1J1arQOExjBeAQjQJALFBuM9XJFUXgGCuHDxxhDNFLQj64rpunzb1l0YMDusRqIpUu7Id2PTMU2YqZ07bSa8evOsRHTIKBGjqPqgvb1CHzDoTV0ssf21H5vtPKhAkTLbeNwKMOlsRDKDv2uJve5Yvn0KzBqpso6kFlx3UwlVDOQfFpUOyWvjwOZyNC2AfDgkJKLbE4Ttt9rlhxEQ5iwabwUFVzPTSlPL+rtBm938x5K6yAcVyvtLDQqVelZ115ARb8X6mhkeaqOU5m\\\\\\\/\\\\\\\/NT7Pvkx3A3kd3y+3Q7PzfEJv8T4UpC4N6wSkoup5awxVfm\\\\\\\/5ALFPLw0cVX4MfGBFavEevjgHPTnHVvCB608y2nzVce2A5iw9\\\\\\\/IG2M71cksGKKcQYlOgCZ3J1wVXEH8D8\\\\\\\/ssvTUFiHHm1p0y9yqg64+ZE3CWwwuNUxlq9RCxJ1AMpBsj1grWeVT6CcsF5ev8K0OrheiBh2Dw16l4ZyKCRfjTONO\\\\\\\/GCkytuJzGyljRp3sAI205B\\\\\\\/fvaSahDh5NmrGQgcYqBLX1Yvg8Pqh06lgkJz2WTfjlN7xW0q5KN6KKa32R7o5mblFgyGlZVOXNzwrX1RxpkXfcXkikW1F6T07rWkF8JPrYlsoMtT23xh8m5N36BTRbdCA8no3JtYn9K8YwIjxw3EZuB6KiLNwkI7bNfPKOEbz\\\\\\\/nUvfS2vA1NGJo8\\\\\\\/LBPryLTLRPrLpOzIXrFweF9uj5wQfeIDaCJ5OPJpJvgq0RR5WJQeKcfy+cfvk+SoryDt\\\\\\\/ROpjBauDFbWL39I5E9MiGlo49mgVvsbhpO+u\\\\\\\/ITI\\\\\\\/egrCB05fFjV7K78eTYIIozrO6CApxihP\\\\\\\/\\\\\\\/o5dcqwQWj+lq8aLQmvUMR6uBMl02TpCxP4VNqha7bUYNqiQWk7Ns7OCggYp8jDlwz32KqlzgkPHj0qMCGY2v1QNe1axTrFDTGFAxxrseH7MZMqdbSae1Fru\\\u0022,\\\u0022iv\\\u0022:\\\u0022lrxRcjlR0a924wehiZ+gpQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022d9f6b0700fd5e4980b9d6af7a9d12ad3acfe6de08f76235e44d2c97ad4f7bdee\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022CteKBAQojev4GJxP8VuQWh156EqO3h3N0xc7oGlSMEdKEs\\\\\\\/\\\\\\\/Nbo560zQ0PbkfUj5VlzUYRtVzcmw2lA8GeC4Hzz2SY1M4c5Xmxt3EaA8isudCxgIvEGqKoocCuRjnKWRRW6sNFEtIC1HtGl3HR7xMLic06mi0mNohXE7LE70Fy\\\\\\\/hlBjQRG7OlQdsaTIQb9HwHWbGlcPNZysgKNfgjzJmpAG594v96mhM1aJBl5iRddrYNf9zRCwkcO2gqS79GIYcW8kmkcsazZBAPlsDUSCpBZEAgQPI1\\\\\\\/L\\\\\\\/7UmKMVLy2TDGAUr1EICnzNvKH\\\\\\\/UtG\\\\\\\/PEOlRuuvCJ1\\\\\\\/uHPkNWAgfyUHvTlBuKVj\\\\\\\/gjrcsWS2qB921Or8NJmaUhiqZJRkfts6UTtUp3YYwjEVAGVzM1bNW16kAX2k98HevaEI785eDG03YvTCGLpp9ywc+o9PYejkP\\\\\\\/QAGhcPj6qaSoflIndVNPbLL08K6SDSmu+WsoPPXbFJb9uvo+UDpHfpLgJ6XpeHdrCUoDh1yy8Gouc2DdWcchUGSNpHZ1+MHwvr+\\\\\\\/zdtDpsUnN4MEmd1P6skOIIxGtTO3k5CcxTp4lJva0qTEzBpQDzDzQXpgallcjElZsF9p6ko3NeLhMhUEvgQJZTcdTeS1TQaoM+I+JnF08Q3YDsMifzPncZZ07BhBL81orVUw9rPiEh3H7ywc4PEQG\\\\\\\/2aW+nbNrsna1C4AKvL0k0S8BQMHGUOx\\\\\\\/U03zHeayMSjFPgGGWnlsGhsCrwScCb\\\\\\\/oQNTi68JyjvNGJTuhahMkLClv2BvnTmcs0Wr5j1nzB1KOxx2GYDdhKq9ZtDZqiQ32mtYYNY5Wbk5bDnOOIsvonfsGxIyypMkbTwvLDn1bVYf7T1NeZR9OCII\\\\\\\/OI4iiUMO7xI7nBC1epWhCT\\\\\\\/riAmaVhkDMBvsZ+peTAxHPqBYmLBngdJnBZYS6ikN95s3d5JENfKreFCQ5bhbm6YmB0Pe5itU9VcdB54KitNFufz\\\\\\\/SceEpxA7f+oDfo6rQd3sHw1LLqpDaVtf+wlZi9vd18D8M6\\\\\\\/TKr0xUqZgG+k6LlxCWZs7oUpia56nGyK2rjo3my0DG49dp2Ef+d3XXggMMnIpirKwSMTCSq1rT5TAi9L1ITqHt15FQ8QnY8FPijdO5jq79QRzLty6x9zrCrTdqDoVCIZ12\\\\\\\/zZesdE4Iv8oey3zMMbug2\\\\\\\/X5ZBqAKVfFJFmxEFnfJ89gzZQENitR975e8eCII7uBQt8TxYsIzMbjl13FV7I5amirg1VMcQDIgCNq7E9lkdUNkPe\\\\\\\/1+AnMIoKayUxrBqVvaKcFzqeQtRjuNalUxLi0l0aY7Y8LTZAPlTTTqQqxHVScrwVfg8jJCqoidSoZ73PC\\\\\\\/R12V9P2Vv1mxVuNRTfSZ7qpFDYj\\\\\\\/U8FS2tRsnAMV4+0JIViJdiLiSiKXwlTIvVLxkTrqaP84anMljZDQHXhhrCsgOdNrasRfJCSZlAq35P9Je5f\\\\\\\/SDzWf+AzFN3v7RnKOKyg3hdjIa1LwpfD85FZNGfaR8Cb30kIkPhN7Xi9eyiaRn+NYjskM2fOMyBF9RtU7CrBde3M5FokEc+384UCUjSoawTcgyKH1D6YfITnbJ3lzZi2NrW2Z6zhA3RWHUY18PS7AbNzWsFR0CaYXsr03MLSYnCdFzq4XPFzVJ+HWmrJOtD6MOtM0iZqCVDM0L+\\\\\\\/uBAdkUXtBqoWNP9Lemf9vj9Xw0nEuNWwiz\\\\\\\/Ey0LWC85+8UEughgALg8HdD0VFQcWJ\\\\\\\/35tdZjvzPFfUXn7HhzMnGD0bebS0Ad4iVI2NgGqkycv6z2qkQrY8sMJb8SRf0KlNx2qBPwvF0MFhlcjO+r7rvrmvu4BFuaksgQOLjbfEKVT1tQN4G70QkNqhbh8GUHVevDz4BIoaKGar\\\\\\\/isD3Dgg9mIYi7JwuT1bWbYsCU5cRAfiVpq+1P2XAiIJOWRyYhTCeEA4Gi+7g5B+YSzKGFbjo\\\\\\\/vuJtsfvZQGl4pG7fm\\\\\\\/LkH15Sk1MR60JD2bTDqCzVoOtIZhKQopfiBphcz7Xm1z2b1iTTX1SivTpmdW4lNJJI84VsjP4hBIT6s7euAb6FwyEIJ6lGz9O7KhdWT3msl1KPI6yHbg+LA77P2C+7i87xOgFi\\\\\\\/qEU5vKxALRohMSiwRp03LvNSlBQ4BVHrRtIsi1ztCzEmmVMmAWp\\\\\\\/jh5fWwyE7rHHa4yg0fuvSzsoIywxIZzdae39tOz6phZjwMOx\\\\\\\/WqcdOhlYNKR4Pkg4lgrA24O74vMj+U6PiTimLDUO2mc+t8LnSklcBk9m039of2MvWF0e+Q+xA1cZXQWQq\\\\\\\/Zrb4aEsS3kA+oQNZkPq7XOQdscsufnrnmRfGawKvIlHAKBcTx4RJ\\\\\\\/C7tzjLGUNuaR\\\\\\\/pIFMhzMjKZv676f6vPAVvsgtnas25UfonjrbfTb6WMgs3Kon227sThnsMHcEkCMUTJWSrtDndZYKT7Xbcgw6LzyvihN8hULaOYdRfSIMxVGJBnDz9us+vp6HD2FcIHivb8doNuKzdK6NvemYH1lUySC4k8NkyLc35Xb6lsxM7u+eczueSP1V618YVsz7rI244b97LKYZEjdUxEdWJpinaLne+O7TKda9bGNNYmHLCtQHaPBk22enu6ytMtw4goAvyNrnu6uHBrTmZ5pkgQB\\\\\\\/TXGoQhUnC7fXpFIhh2fy0L5TQ2XtCmMiQhVvhNEv54yXdaXRiVQ0eyyjmLRc0QH4FimuBWK9mm4PzRzTJqAr5jXKqI6QwyQJTmnpA6+8ZRbiwmvcuyaPJJLOvPC6g9uZR0aVC5TDZ3GJunhdWHcW1Iqkxb9YAnXzOhEay6ly\\\\\\\/qiYjZNmh0zrAP6++hfmkSagFSOrdJJFdUubJjc6yrU3T0JC05dmK2mIP8jjiNuPLJE5VZ35BDkmLVcTa2MP4P1a5Tdxl0HK10XJZ50NM6wP234FwoFZX4o+6utml1\\\\\\\/wOO7xUmaAAqoPLT5\\\\\\\/mGYkG6K\\\\\\\/+gn76xy7mbktMIZ8kFv7u6zCrT\\\\\\\/Qd1wT7oLZy6GX2j19J0UmE5Y\\\\\\\/tfIt2kejtJ2L2Glpy5r1gGKweN+bP98HuCyvUEjrEGZB6ftLBudMzPGFYu76XHGXlBl2CWhWOEOIoevuuMtb4BoJQyRrtTUKPe0d4aNdb2+AT2a3Jk7y9mZ5WjK4rpvoKC2YqH6UMp2kmdH+5vCVkl15boL0Hk3CaoZNI+UyXtjDpcGOxeD4JDdLau4XXzTaxGGV9Tw1Moxq19GcrHpwRBJBT39gosrIngHjqsUippCQ84NTlPCPOVRcBBnUTkzyFp\\\\\\\/s7gZp7iZHgN4+VbnvXvXoJrVWezuveNnqtnn8qvsbx9IfkXXjihWgG7O0mVVSrla5hWniaGeEqRmJcT06jzkINLW6wFYRfC45iAUIeMyVAi0OQ8Seys0Z7+sHY8Pqemrf66udDf4qOpYjy7FT85FnMHv48yYRpUWp\\\\\\\/7bUcHagRpoY2uvjH4GMwIUwvXw+X+0PM0QhPKm1YkVDGjysxjx6+wN04cY1MeYF\\\\\\\/HFH82Oef3R8iqmPw5jbuU545u+I6wqJqIQ72plXS4OyMrjJmcoeL+bnJrqqX1juGGfZToyVWu5MamG5Nyfe3XwtbBAX\\\\\\\/3o2zt37NYRC9sUN3FAC3LWmDmq5Dk0WI2f7OEaXHmfl6HJr1JUdHjr+6fJoEkpmHVMisFHt2RnDrfnrUFX7dDaJR9lfqf75BwWKkkFCpiDQEiTnrrjE+JJsTyfWgiY+EkuR65jTF4wow2cv5u3hiqat5nza9AEVWW8iQAvroZUIGL+dC6QTlM8FqP40l6Neivz2GkbFiFKYw6KOVqleWfgLS6aUQ8q1OuqYKL2TYRQzfJlUlfTT\\\\\\\/zAJIDQaJ8vI70losrlY0hgvJY3SsSjqhjz3UNMzsclO5yKtUaJUmeGO4R7bQY8wRWOWvJXdCo0zG0CwlhvEDZ7R70VN5XdO\\\\\\\/GINzVs9LKjyR9+jvIqYbvwt8B2oIT1Yldm845sBvppSOjbhzP7cwqAemGCp4InUylAlllV\\\\\\\/gYpA8XaBL\\\\\\\/b58M8hKAtQFgCYDlvTozIk9sRnYSdJ0NdEO\\\\\\\/ksy4wiJZI74Vxq5f3e04WJ2IUrq77v9Hj+eD1z2NMEL4Huu7cHCWyQlKUJuluuHTVuEUf52xUqn7qETPNBNHkTOc5vzcYRNIbg9tc+ypHXNCiBF1SGkarZ7pLI+jU\\\\\\\/rB3IVyuE57ljfvMd+Sgar9X8FxOyQ5es7oYbXt6GVZBKI6R2wlvIFLBqT9zwHj0+m\\\\\\\/I7Flh48g9nURw7MV4dCcvY2YTqK+2UddhTirqaW9aYMv781aL0ogfYUntu2sGZr7yNB\\\\\\\/6QjQ7MVe2oGCihLhErIQqMbW0U8ymWEGzT83k+yfXul0DpfbtTP+mBWNeqedHrAz3+MfbUJZktQjfKiPiyBVKqieORSwTbNHU8KgmWYajb6k3AL2GcK3V8eTypvW6k3QImHr7Z\\\\\\\/THwd9YgEJkGgRK5I6S2nz72iiWm9MWLREP+Ppqsi8Iwv+2VAGPaNyGhT9mohqylz9WbNwh2xk\\\\\\\/brvCBrLqryhWdKCCzDoYQ5B6ofdO\\\\\\\/Ec+y1pYY4EfLWs0JOwv1VfI8V\\\\\\\/NDnyg1yDJRM+K9gWjDlFPZYM71OHZa6y98dq5PEQkE0S5nUfKuTHDcEVIVBYVHqf4OZ73PJy81N\\\\\\\/nuEr\\\\\\\/nsaGdDkkpd5qLHIduOgSSbcm+BQVTlH196xa6pGNbB8u617DNHTCOEDR\\\\\\\/8dmgki9oMorGGYvwYazCwdjZysxpY2akwlxirGVE6PI4gSUptGA1i4A6kqG4kF\\\\\\\/MQrxAbqxZbkVV7RErlyZSiQNbM9S1MaC43XJqwYcH1ZJe0mRjP21ZV\\\\\\\/e\\\\\\\/s0Hjs7GP7b0JRzV2Et81F6XJdzRWEfani9O4LS5Mdx3YWwyWAU1b2r3qwz3huw3v19ZRElOZX6\\\\\\\/SPR+WnXtc8\\\\\\\/rBSNmt+pdcSApzxBdlaWOaofwtfvjjgQ2UoZAMvAAUUsoMMJ7vUIRJ5fMQyO2pM1Grz3Yo5BTJFNUs+ksmhN\\\\\\\/G4LrAb6E7kB5lXxEVdqmuRklSs5frIhJeyRsKmNUAI1CwK+amMBKYUknrkQ2ZDmUC7jV5P5Zpa9DQGlQ3pmiaqSPfwRr7LqG5UrDsKOPuPuzDQCNPHvyk7s2K+LMPtbnaxE\\\\\\\/qDrkTM5IQ8GkPmGtwPHjz4lh6iRjEq+HjHAgZAsn581z9ibgFZAopiySeCDLJ0bfXIcTPa\\\\\\\/nPseHdaAiERLEjz\\\\\\\/nnYn7BpwKWRUJUW6HOskFlLRzxik2Iztj64z8BewyG4V4cfDJXgNuCabtGJYXa80gYyBUSZspKQyHT50knknRMAoB+dmybornPu\\\\\\\/\\\\\\\/DzTSBsIAou1PNbb5VhPVNAzdQCNRIQ3bWM0dJywqDwZTXCIkU4lEQ2mb2f7fDG+4ced5NY1CxdcYqnMy\\\\\\\/k\\\\\\\/7SDCZafGvu1TK6sR0ybIEVrffQgUorJtkjmk5rj\\\\\\\/\\\\\\\/nI2AL9UnB+\\\\\\\/1aSU\\\\\\\/ziEi7wxbl3HGagdLOwod0Ae9SMwtdwXvDmMC5UHKWAlmB60kYHP+uSkTUsH45BVH782cd5zSJf822dwrDQbaZ9AmC\\\\\\\/924y6JVd6xma3xicoIMGpFBhxEupQzmGw4G6PX5Y\\\\\\\/1bG9Gu39fWeXy6cHx6piop4btEcvLpL3tpJCfqTHMxD\\\\\\\/0vZPc8JLJpJDlff4ADqUWVZSOLp+9ilfzl\\\\\\\/kXgwIQLOIVk\\\\\\\/iJ\\\\\\\/nTEO29rvxP5b1xinikxUbchDTrjkJujIWMhd29KHiERBSt8XE7PF3\\\\\\\/c4MUhBHzBxygy+tALTeaLqzqRjZlODm9fMVLK2ty\\\\\\\/XyXQmB9\\\\\\\/GC4mz09C7LzLQ4K0L9jqxzWsWG6KclvGNDv74RB9ykFl\\\\\\\/FtFkd2yjEE2NmaDb+6wyLTVbrBRMlS30X\\\\\\\/\\\\\\\/K2i6KN+798bmqRXHBhVeF3tsv34seMtstsFg4Ai\\\\\\\/71isfU476b1Yz3m6\\\\\\\/01qpIWwGskjliOOztuzhJuJLLR6geEphxLGk7el5QeHfkyiqpjuurwc32gl6S1kiRvIIis1TOw8deaN2mEFDoN4U+icmKXgZSGa4lTFTff7mfcL7xchGWmEQdr4IhT+PTT5Z9TRBv5jYtIkrgA3BeP5eDGX4invSmsg6yFwgx+UEhuhyK0i\\\\\\\/gW8n7PvRMWjzgIbNf2Cuso4F\\\\\\\/ri1eNbqQ1bjuLg11eRYCeJCVpkESyQokBWsQwk4QsFXwNjMDGIYRnVf4\\\\\\\/v+j0ZVYWfZTq4emHUTQpZWvR6vqHKD1dxXya4xMNSfsF8lHQkp+Mfn43RHOsSlW7nvaq39sMYimlbSaL6GOZ+A22vCsMXxryCwRvaEaRPS12SHAKcZBfRKTNv8XhpbAstqhA8\\\\\\\/HFPU3Ozn917L2Kv9UikyQH8A6svSWSQbesTwS2UMzvdUQ3XV9GiPu4lkxGTE3QNSjN9hgtLGzZTKQ83uHTc2hnpiE3eAtt1fZOdyXrZNPQrMKOLK67tD2wLzwGif9V6dtkrGmJSQpWkLNvS8EE1DnjZZ2syWwAyV5S1XQ9eBDQHgY2fYs1f4qfPO0xP1woK3qfKbBxtj8sw\\\\\\\/BokEpUH2TJ3MR5jp9JgzYg1MdNKhSD17PHHVbXLQyKshUH+nuETr0dJ5PY23LM3\\\\\\\/iIHjfY5FXV4jHIZwI\\\\\\\/v3XMpiUaVeMqjG+mFSa7N2475ZAoxwTGGCgqf2YdRLNArZ95W5b+1TgZ5BbXKgdiLvuRN4ztvWCECYZoG0D7K+l2P1RRFP522Cs5bERtd5ZLqGQ+\\\\\\\/AAUlfDxmvDpVto\\\\\\\/TZnQf70Q1\\\\\\\/ITgBwYFUo+n7mmDBRqAq5iYrqK6O4dcPav2KEJVT4wXrP0dgP1ZFjj4yueqe2y+wKhwDaHLxtwWMHrKu3ykmGanVNrKkdkkGxlc7HpaHaJxuqE7CIbA0UPAzToexQ47Fx7psaMdnJlkRVLT9J6BBwiaXH0SZ6N3dJvn3h6pq05RP99bZ4RWcN93XXMO8iKGSlwR15FUt2kE+OXMyN4UrSV6dJA4M9d8WPphIgnJsgjTMOOVV\\\\\\\/py1LBdVgykmVw2lQcfz+9KbUga0vWMUAy2+NRjhenJORCpMCh+yJs8ot2Fhlx0F2tSQ0fXYgxTYU9+zaf+eq7+oaoyjjZS0pOzvQ3My05iiT3HRpYTqBwLiHNFVp640TBfg3w7Yvb5hluBuPImp7iUUZ73xLLvNS+gw0IzMhGAcMDZWSX+pRKN0aK2Ys6yqtXdg0pIf7cEA6dlpo3kLNn2+xnAbLLVRczmSNVR5zAfOAyVe0CUldveIxWe2WJRB1C3y\\\\\\\/x9Hhhxt8PTegLx19yzGfhat2z5Tcb9TjRZK8UVlK1eJOqDvuSySoHkqIOGoW39MhBFuR89xEVuNdO2s6MOslBx0fJYjDiYPnCCaX5FGTQirceB5q\\\\\\\/IvXBVpI6DAvCEh8wODQu0hzLMSeB0NVhHL1jmC8lPlykky1Ig4yEgU3qNj7yvohIDCRkGmbQ5o9iHk2w8IUp1Vl0gKw1zhNmiuU8\\\\\\\/0+ZKSfGdgjeBWICe4OWINl8+0KxxW381NtcTuOuqjwQQ2uPsmllpydioE6oK7p+FhLEp\\\\\\\/e\\\\\\\/+My\\\\\\\/dQvK6Jm0pXZ6GOldqmz2PnhwSNEhQwCeSBWgdIb4sKTMcKjKqYEC\\\\\\\/ZruczhH740eFjXuiJNIx3UBd3\\\\\\\/x96ZXhKyIToTtP1hG1pE452pkfXIOCAAHC7cI2kxazzjkuy1Y1gY3KPqhEiwL9HNGuqvIJFpUtLVt05KETV9R7wFBAdG9brlhvjqYk+9LqWcRAKUOBmOqS4kgOQHy2S6SfVMfykrCEHoLi0m\\\\\\\/Snvx3J+wIH9yLP6N\\\\\\\/ReQWAnJI+tFjCDFKCAyUE4sMYJ4P4BfYNQZyPDVltjBiPRkhgBJwhL3cRcgXobDLrfVF4rvsKPLH0H8MOYZ8Uz1RvBbw6shnZvV2o5GoMVNSLBrXTDEyYaY3WMQ1dB1FtscoKX06+Td0tmIbASBDM9r1GVwXewgqCiVFK5N76J6ntbA0vWDFIxdn\\\\\\\/d5SHBvsWfxgPJaKSmipU0gR8ffm\\\\\\\/FNljzyZfpBu+auu1m4SwwxV12lYS9TG4UgU73gezHxqiWQ2O5fwFDcXdPBlp4qf2HKmALhkYzWb\\\\\\\/tPdZW3GYL6lAuguobcHEmCZOsrZLX5LdFUJ9SVH+sobJtHVxhxewSjB\\\\\\\/N0bSN+LOOGR5YP1I83LwaGymiNTatCR9P+Mj\\\\\\\/mF2mIm693KHGtsXLfxEss8uHCqS6Y49nI2Qe+0q0inWfTA1p\\\\\\\/U8Obfn+yhpLxP8AEeKOgqaNB3kbsp4V1G1K0TH5WQUdNGzAhNx7hETaADUSJczaKVdZ21eIWHEmPK4hoXevfHMkhQHXzVxEgqzGPH\\\\\\\/5V7YbQYGn\\\\\\\/Iy6VBlXlgm\\\\\\\/sBTK6FoM54LR+rXbBch4RGtmwcuZhADloCIGKmB6DZYFGvSRQXZLzAdmqZ17DvQO+QUJBOsSR9QQoSG21uEwuy+dd9mkZncGVOqm6x28TgdccpxocGrSEMs9ngu\\\\\\\/2T5BB+wIaPkMpOBI2Cyurc6DXzlByWpkmQko11YVByiA8yGUWo8Qv0iZEwKeUwS0\\\\\\\/qkdPLAZIh1Ocs9kw4ghlz0t+x7qwhZ0aytDmLyZR59XIQTi\\\\\\\/4GRTsfFbzONHRr1SmLqO65MnxL945F7Tk\\\\\\\/Ec4j0xp8G8G5MlL5jaYx3MSYffnLqV49URBuihAaGY+HyPGWbJdGTdqFtYDTgPbl71LyXAkk7PcRBSA0F\\\\\\\/2IZ0G2jW6B+V1ruOz033kTW5eX8G4rj9pUdcNA2tiqDFxJgi3ZMzYWTxYK5lSqJQIBUpS\\\\\\\/3wpj7iMjX8FM\\\\\\\/FweO4PsuJ6YCcF20LkkTZhAUm+AVBf\\\\\\\/uM3Xm2+2F+iU6Tt\\\\\\\/VaJ8dwoIpAwAcxHZ00Kk2AdcvYUllSa5DSVDw96do\\\\\\\/Xe5UcEo\\\\\\\/HeBYrarwJuyWIwgtlmFLWJ69pusZxH0DCNtPrUjmeuzrZFF1P1RYDyJUeEGchLTvVM+BS4iQfDmv8a3UN+S+ciOeOLs\\\\\\\/9ud4UTCUdCW3HPudez5Q47tXYds5fP1dQFADrcXDii0fBThHkftPQ9YYp70SmdEQDoyfjO3cHeyaQ3gigF1LWNMrkRVBtQvfL6U6TKzU\\\\\\\/oAs\\\\\\\/VwQCHwTRaGvVv2OR\\\\\\\/2\\\\\\\/V4Uj9RVZixVL8FMz6Uys+wJlyCqe0k+CwS5SUp+oMJvwePETstkXyxp40MmPbBU44FNxwBh2Nj1oxQHzXFI2ZJez7V+7LRw9edO4MbZWPLZxthqHAWUv88QL9vxPImasRa6l\\\\\\\/0Yy1vBpcDHRAohe00eATT7WH1khEydUpys3\\\\\\\/x1cZmYqDC3Tkwiq5U4sw23r13GVFLb16OISiC\\\\\\\/vw1JjkQZt90IjwmAKyKaQWYpjIdjtUPBStnmAhi83amARQCJvesdaH2M+0P3yojGv5fGI0A+7C9fFLtyvg4nAfR9yqPrf9PZS4hKX4j+LtV7uGLN5fbeR8U4qH7CBI2RSXycBvvFW1vbfzvyeHWC9A3FGGCLtpDpGcmVcXMGfwEIGUOQ5ivIvhQIfW3jT5hRCtuoQaBBjgkjcgfQIZx3S65erPZydbOKdNwzoTKZK+jyGV\\\\\\\/mZtmbex15AuiN9NZI4rS7V5LvdTuk4lTST8wiBRwMhXeQM00rilgQ9F56TAJutB1suobA1b9U51OjoKaEK91j8owPGdOp8z910NdNmz4F76thhtSK+sDBCDFGNDMyHJ\\\\\\\/4XW+uZ30Ur\\\\\\\/E4HdUut97bWPvsKHwzmT5fbpicoVvJhOBR55YY7CazXJyqdsRgBOLMnfTzo686M1Xq9GOdKNCNW9rzSSrqF7fF8D9TZM\\\\\\\/4ga2HDcZOr+\\\\\\\/aoDvlHZ\\\\\\\/lk2LcQ4mCQri8L+QOq6v+zIaaqL5LsUNO03DkoKgzSSPUpN7yZIcidm2Am4AxRbHo27QFzcVCRMlz6Kpg8VIvj288ws6pkv9PbC4xdcau938rkcDz7GUt1Qo1s+1wMqPPO9dllPXPtNSR2oqPDISlN+bO52rCFU6LnWuTO67O0eaGw4W+WSMiYXrUytM10M6Ubd+0sXiSXCPyx9xMuGNyn1COv+ryoiILba3SXoe6BkVZGMln4Vd0NxG\\\\\\\/Au8H9TaiNOL8elG6h5FSrc768xxFh46MRy315ip+3cphxA+JIx3jiCgKG6OlgfDMK0FXR9jp8M1i7EwDJUu+Bv3yBC4nZmUN0yArHqt1D5LzIsEC6kfvpzLYoxBF3fl70RnzUblMiOlrg2pPS+mdiQMvTDyAVftHQApZ9bONLNoaS1gHKyPIsvQPcotdOb4rlobvNG7DGXs9jMm0t8+MMvq4zIRkrGEISVQ\\\\\\\/kgcxSpoJ6jt0eeWVetXqRzl6ImQKcU1QketedHD09UR\\\\\\\/C9OdGd0VBx2oX93WJlFbzhAMMTvXfEex9eRsBqPLUa98ZEvCekHWhTMoby9iiTEOb7mtmee\\\\\\\/ZhrQuS+OquXAjwvPGytEguU4E979u8Axu7NUSOgCOvkEY3BYAU5Dli0LXoUMwBhGdEMZSv28ySmBQkBXDgrRoZQ5thWjFN\\\\\\\/rL29+RV\\\\\\\/lMWeyLBxb0GPcheOVtrRfwe5t4DDy7FqlZ\\\\\\\/J\\\\\\\/VXhVJMZf4JN8MXnEy52agNOoLqlKOf\\\\\\\/cnebm+wQhvZOWEsiSFSFpj85ghDTNRoy28u0wSmw+FhGCnxtMCvSBtpysfBukg9EhFHKAXB\\\\\\\/tfhpH7nBQmuvNKbAIMFMrN2xHT1DuvJ\\\\\\\/xXjJ3fzUW3nwGSlzLBuPqybw4Pxb6N2Q0g0hTybeGSGOc9mLiNUHvlA\\\\\\\/HF0l\\\\\\\/8oRio6CNiOyKzS8kFXsEpW+bQn6gwEDXhONZ92WsEc8iRKOpho6MiEZ\\\\\\\/W\\\\\\\/Qtdxevdby7VyiM5FZSv3TRJbIDZ6V1rVEqWIXUY\\\\\\\/HroWYzrVWjLnxa\\\\\\\/SdAi+GJZ5WgGoxYzum5f11Co6tixEesvE3bwYPIexoPwPSU+UAtSL3sIy39daYkoq+XUuGJrNM+utCpS2H0Wnsh\\\\\\\/me0CglyasWcJ4SD2qreuRFOtc\\\\\\\/yM3IC2fZsJy1mFWBcQM4dCCLj5JZH3nZ8AKTakcceswo\\\\\\\/ZmgLjn+Goucpom0j+bncNhzCDBvEfkpZ9BlvUHSKuCJM2vJNlTJtLXyX2sO3KBP7KTPVqhG8nI0sKpZQeo1OddG3O2N0oWRlW7E\\\\\\\/AP1m83y2H0iau2Y1w8osXdb7k0vX0xTb9PgeWW1iMs5ueCsIvy6VhQKxW58u6YfFzAj5SyZsdENBAK0Nl+oXfZNwAjdhcoOsdeUwPbDBEjuHf3TphVNpafTUg6PFI8qThD3ibY2K5rKTnTeCzkdbeWwAfDOmki0b\\\\\\\/8rnuMBSrh227pQhZ48sTQLR2tEZOrZurGKS6rHxtaPG8cOiCcL7eGlazFb+XoK+l2PoEqCU0dPfAbj6R99JfRbEFDEfZPbHLBT1lCSF3g0uqpZdkg39NXCRYOQB978GlJ00Fr31t2+wwyi3PEw0uHN4zWsD5DkHXH3rOeSt7X9867WWJo2u2wrFIlx20Lx\\\\\\\/KH3aBGfa0GXmrFW\\\\\\\/qtwCKuMYVis+blI+umJ6HXvNJ4FGDWpPgK1uUdfHhNjj+5S11QHxtVgAcrewC4k8J1Fx6DKxHLf8bkealJZOIf+Cyw1erQRWz3E6Csa5mjSNur9eTcGXq1vXkqG72wduGKk8rzHGH2ESgvRhqG3px0Yn7UvcGrzqYV4AkY1K+1C8Q2O80+IymdxzsGqgtFAgCjeXOIz8MG3RHglcqqPmL4UenFrmnm6LgQNho7gi75kfklRoPzJnEsbjxKbHU1\\\\\\\/vO7RI6ZMACAH4tBmtujwPeG8\\\\\\\/uzsg9wXvVVRjYJ6aXQ58LXgOk0s+GZNn7dLQo40wJUeLxkW0hv9QBEJk031mraFlNrd9+w95YOy9wRz7lkgkLzS9TveSs0Sy36n6JigMfTTW+pE57n+SyaNS3zpmqHukRnJhx+25IqM+c2n9Rw45KjV1fMblPVbmIDlow3sceJB8m8JqBSBZqt08HteOJi\\\\\\\/+jp5rc9Q\\\\\\\/5yAOj4LTyDiAvKn3BFzXQvFrjSZ4iWDEPXaBziHr3LFUK+Pz5xS\\\\\\\/ACMnqGp30OU4WDDcceipGCUE5taz3yC0OV2jlrpZ+B27q\\\\\\\/YEs1LyC5\\\\\\\/cJ1+STdqxQ7jp9nqJ5xPBuYDFxHc5Ff5wrPO+54TBtl7jV93cnQ3sSFbQO586R\\\\\\\/P3naCNUJ\\\\\\\/SkXayKYP4wU+eXr2cppmHtF8I491\\\\\\\/XvLX1OmezPO4P4XyT0hs3FEavNBFOaT9RNGhBmmRS7Z95Zg8uyzr0r9XJbJUUw17t7c2F4jnabpqkQ0m0giMFRR\\\\\\\/6hrsYkiC\\\\\\\/dwkT5o0+1yUIw8aZVlGvkCT5UPbrPJ0pM6OreTFpjNf9PjKEydtUlClB4szQQV\\\\\\\/IBxf0qJ\\\\\\\/o+8SjloFj\\\\\\\/MwKQp5Ru6CVh5cRTBd\\\\\\\/7zOlM5IZDKD7OpACaKZ1WyqfpGEyd9GvmY8pbMWJ87J+IOH+o46R4IiEDKuZfdaV6b7tlQWmgda9FKnZAIib6CRZdAGtwI08AkUuV+t9AtHcG639lc4\\\\\\\/fLso2ffHEyWGgzL+8+YEjpcSC8YbmrdMssY3zIfgUJjxIZkY6x6dWKFaAV4EDjXoao\\\\\\\/Eyo4lSxphEbWwkRfgKEwnDo992THm1z1qscsmD5VzTZi7vxAuI1zdz\\\\\\\/PEH0NR54AcPkLIPsHl7bZP3luWDBikGxQuKLO9N3\\\\\\\/r0ksAUOuCbOM1bTL\\\\\\\/nJKT6+YEdtTBlcY6XQldEBGxw3j0cCz3qvLrq6Zqx5+mu\\\\\\\/c\\\\\\\/2NTZcyBMkA+seVaO1EcRCrQx7X4pJFJPgxnrt23KM\\\\\\\/JsKAz65gKLqCxPuCDKkMSldK\\\\\\\/FhU\\\\\\\/yPMzyr2oYWM\\\\\\\/hUO0mhOWYNobHUgy6cIrS7T4VoB9XK2Ifrb2EkMBLt13a8OAdgOPryVObKloY3FLEAH9hasJf\\\\\\\/SrdIdF\\\\\\\/qVjP57UtQQYCKHktLiIJMs0zcJiQHbvnk1V2u+GpZyNl2ZkHriNndS15pmC+rOdESUUU5VITcHodkHmDvjfVAZSuiWQykOqIQyOb9KotQSGdMid1vISvOaw6l\\\\\\\/VxY4VU99D4r2t+UZvq3hoXndMMIoJWSfu\\\\\\\/4XwxpGaNLDoF83jlAFDGsjYErPZrXxEqTcI8ZXipwfeji4JDMMKtuC0ctEqIg27bcel1xDXbxXXJmzyb5YneABdHnLrTrWEwqt09Q\\\\\\\/OtTSCYQaD80KnudVic1H+X9BH6ZUV3ef1xcmps32dAY2jW+rxn6J+ImxP585Aqk8fBPizXYYwTfMZOz2jkDqdTQtnUIAbhQ759Ci0E8JPvxR17YC9p\\\\\\\/ldLB974bwqHn2rlnfMZ2mG06T6gZmA3tv1\\\\\\\/pS4IM1T+NkDe00oqgrN4Qsa0t+ZR+hqp9mkoqumN1E6kTl3lrMlDUxdy73ArBwlY8Kl0vAgl6cJZLaD1v5wIXRfvZ\\\\\\\/SwnVsVAYPec6gxsUsft7IwKkzcoRS6OBlTG2U5nc+BTnqFmxbEgDTnSDbBIgI7H1x8kD\\\\\\\/jdZEPTOmvqYoPnWupmeM6CkrM14i72AUFsRu3HYqSGeKkHFCP22OnhguYTmjvzG+vR2OY0fS2fOr3bkRH99+NMv\\\\\\\/BIQ5wYHQyhvyI7ipB9SQUvQEdNkIpfhWFPn58rzI5LO1nkB3W2Vaf6BY\\\\\\\/1FCi8FYaowXAemdc0eTaVY\\\\\\\/eR9Mv9yTJRkAH42ODWPJWnfvjJGwhpsZRucB4LJoJCMlqdIyV8bcRQpaxCw2yP0lmd8WhbkQQojcrDGeHWrEtLapMapwfR01hB+hJI3MYxQ09YnkQvfxHx\\\\\\\/Yx0zlHldWbrsbJLe9K\\\\\\\/RzBzOu9P3\\\\\\\/N8jj1HZFoYp\\\\\\\/0zAM4rRu1XrH3aiiHmSHkYSDUDefz5dKSneNa5JMxi2tfAo7v\\\\\\\/lqzVNcH+jYFcYEEu43uqmJtDV8WkGU4okR3\\\\\\\/H\\\\\\\/hFt9qcKUuIaprWrhyNk094N7G+k43pQYF6qXRfQt6qqI2eRdpIVp86E1peRX4gcPbS6mNBZYLodcZXZxX3MwWGS56OBWK48akKzauu6Ehxl8QuEreCZAIGtag5KAj5ZQIAV8Pg\\\\\\\/n990ys6bUUkjkK7HXTCucuELSCrKl2r2y7caSk2PjbZyWzK\\\\\\\/YMZIzwIv05oiGLMsx4WbhE+2eSwWlJW6uu0dqkUASTjXMvLKYfNuV9b1GvAARBc0aEtU6S7eZ3QbLoBvOnMhKkmdFqBJwwr9dHGs\\\\\\\/tw8XkV3h71nkUef+kOa01jLC8vmHC+IpqY1Xssqr8jkjGOU1uXYiwnSDPiIUnilE3PaPtSvRFm4VFCadWI97Xj7T32hsmWZ4tojubwz6wTgBwAcc5O2JroFmnQpV4jVtMOizrlBQbvs9xNEJeuiTpiexDsdzXFKKEwf5bvR0ph+kzYCK\\\\\\\/aEEs+jpAIhFrE\\\\\\\/7Atl3V91ZaLTZFIFwMptxpKlXAxoZlGMqKUVyrhBIKQuIe5JoUACJoxUFrZqnd91U559dP7KFt\\\\\\\/iBlVvh6vwCHQvZI\\\\\\\/VxGEX7ceRUR67f1OXNpnJHSphQeJTOolwLEQxTZzeo1tpQX\\\\\\\/qODzUgwZ35eNN41RUiFi4JeUEfx4ITS5DWr47RnqsFuH9zq0DkOufRNWI1f4C98LdXxqMEkgJGW5FQXZ0tPWyGaybhu247mXYJzq\\\\\\\/18nasxpgKSiU2ZpMhZFF6MEd43sjyCiV6Bi1R0JrU5tEehRTp2qCuiL19YP\\\\\\\/DjuYIGarEFr8G\\\\\\\/Yg1XWB930BNMtB6efyoh02sTxygbRFb+uKdBaayMdksF1Hpl\\\\\\\/XkZE2QUWSvH2z7HTv6ulo5bBSv04WS59yibhDd96Vm+cbdY2OmtIj6MUYP6g3rvfNcafDSSbIPUq97VoAyeo5yRiBW2VFr8L\\\\\\\/eRE0tUfl2pKC9FCSHXmHO1Ua8QoxKsPb8Cup2oigyJhDcOQSs30LyVPNbxvRGG1Vq\\\\\\\/+Dy+G6w5GwRujvPvM+\\\\\\\/00F2c6qGaa0mulqY2iqpTYSPuyfGmCtcGuV9gHbLg\\\\\\\/iZfsYO\\\\\\\/2qjhPVR\\\\\\\/9Q3Gsi8WC3zxCPpqp3mLknJiZWYFUjcBUNaY5X5N1oU\\\\\\\/DRpAxO0dc1fsyzYn0QdvX2EDIaR7f6ku6\\\\\\\/Lwmo4fBIr55aJmkBN55+t9Cc3sYWDpL2ytDys3fHwfZDp6Qtf8PUx8N8Jy0urRLuU+hKipVHTCmHPPDImsI5nUybVhxMJM7hxjjCS1kE7hgk9cV7j6EYLJmNYq31QlYqM4TCnZC4jZwkzjSXSn9v1oT3\\\\\\\/RGx61U4fY3949Y6DTT7lHD1cl1IfRsJH9PdCQ1xKGTgO2BXFcGuwvmNa7tISMca9u8aXYOT+ib7CIEi6g9axPkW2+Ac4oIe6pkrl2bJbvP62RMIdXcIrF\\\\\\\/wx4OlUo4FLHt32ct0B36JXO5BFTmzvJ\\\\\\\/oxj7iI\\\\\\\/22oVl8aV9EIv2RaFwFrlnd8qgu+0hglyVC5ge0GItThOBdacS1O\\\\\\\/+oixkkDTGX8mrPmjEYqMVTmT5ywky6s7u2KASlCc9JYWJeIWSm\\\\\\\/fMOF9Klx4g97kbNG5bUllGbgw27mRzyGGuNg3KYeGn+OsyBHFH52upDIigW34iykln4lNQ+ew3OAFISRJ2TG+BKnJwbSaPUcRL2JyheDjLC2xac0S0J0fC8crpGwQxpZqA8v3FMqOJXjWFwKQcwb3RP2fgrMI1Dfz5jpqLlCFFN0bdwT0Y1MMddfCub77OnQXbr3ASljczUnCSLkF8\\\\\\\/8I1CWE9Bx2pCLm2vDHbANiHrj5QInxfRc3bF4n\\\\\\\/+HpfZCjhxgieIO4RcCI+OCI+2D\\\\\\\/XmJejMaZtejHbvvGE4UuczHfbr3Yrd5rWv15q0M+la4FQUAPmMOoTbFO+qNpHJQberaqyS1ZkWGBb79QOhrSsDc0ZESvB6r1RjWfePA7CSC0HDRO9RVnxs1Z+8XCGk5x+x6IoIBXVLUwNkTvcL7EJndcHUmZivD6rw8iHiyMZUo8TI\\\\\\\/GSsr4kAwZFatvu1kh3dCQPyzoyy\\\\\\\/g7NTtwgO+niqSLRt18h8POrmyiZNFgkZTVS0vc8dc0KyEPE5KOQtbAGTwJnP8TmDzU0iUf7TM1KcsFgXQnWO2+WspV7IZNhXL2N9NQZX5WK8ttRcVUcPfoeVgm2xO+YqQA0SR83FmVMlpYXPmT82rknZQwDwbTuhvorAX2MFfeyd1i8\\\\\\\/s7l4y+MeoK1sUDMpLuk+dNMB\\\\\\\/A\\\\\\\/rkec5DmfuowHVh5pFay34rxxgW46E\\\\\\\/G57vMM\\\\\\\/NW\\\\\\\/XdxSIZeOoy\\\\\\\/WrVdwKazM8JNVqNpAmHAKGthW5P5ev+6Fe0DsfNXi8jgbl20DGHehFWCLmk6YnBedK1LCcO5L3Zi5WBQhAGR6K4\\\\\\\/WkneUDNuQZsG1L20sb2QU9c3TpyPRrMeB2Uxqugsutt2zzMZm7bKb3\\\\\\\/oBUbYHIh9cDIfupiKMIQovfm5+12v66pQ3CjZzi29xQggOB2LxLKEmjGcPKTsjuDIgyaJsQGTZIqqYkNTd+d3P0v4sgwj74zZT\\\\\\\/5gHs5QSFHWYVFMPN38\\\\\\\/pg9MlkBqXiCI6Icx4JRnLuOBGkRgmobgr79X16uiFgQuEF9+\\\\\\\/b4IBPP1FNv\\\\\\\/bxMEjFMUaG7QmxazM5wWudERWQSuYdZp4GG7bWrWYpjWfD7sOGGU7L1lJr5KSzmIB+mLGZdnGQwUThsqvbaaNGQ9ZiVjxuP6OI0\\\\\\\/\\\\\\\/twpCOcNz5IZmFwvKnyKFRo0wS\\\\\\\/Jm7A1MxlQkmMt145nI6PBMsru7nUWLJQSLmrSSyn2xuvMgrBRpGt0JnIbq4XDvFVoCkLVeIPnD0qKGLj8buPyAZto6sLj2KC+bbV56HZjnJFeabe\\\\\\\/umtik\\\\\\\/JHhDKZjN9aeQHKd6LilPhgPCS9UpvrPwsbGq8WN6pzvEoNO4rRVW6Z\\\\\\\/C6XW\\\\\\\/G1FFfHEb9oWxr5bni9DoKJm88zZMIsRYFyjB+nu06HEByO34E8y7\\\\\\\/0PpyOMGRppnZrlnJ1SJAT1zTBFRUK6BunU2+2IqG1Xu53ozMO+GvrUTJiGDC9u1M4Bt8Rrh6XseMMCRG2Sp80MD\\\\\\\/mNqNVpgR0Ag54tAwEldkWBNkBHscd8BNwttXo\\\\\\\/8Ty47dcxqj+dE0R1PF35+Hnkr42q\\\\\\\/muckIdbrwv98LO4+vRup1WhGL7oEEdiEKHNw5Wk3aMgWjTyD2ELWpajSWTll4\\\\\\\/taoZh1UsSJGATLIGLvPJUmEPgXkCpW+6ku+NbowCtsz1Z6XtOtzbZ7A5mn1JRtn72lFpHNL9LLDqOFASwgHsjq101IsY3+dShuK1hPSxOBa6xYLZnECrLzuLxNjIxJzQE555NJ87GLXEpL4L34YLPUu51zgpP+jP\\\\\\\/FnXjndQjo4yCWt3wK6BvWbThKF8sxTospv1OqXmGFZ2XZ7tEEC+vpR1100YG6TW\\\\\\\/Z4pOxAR+xY8hRJoGRQUsdHpuVLZ6UMFNjog7AvPrx3uifpf0lNe8RxpXAJUA+yv2ITkgn3YtyJyYttIc2pk6XMW5aPO8mq1GMLAlsiKAf7ATzrUREuYxRhwu7u1LiqImc8MJd4B3+ZGSuBg0TbXFYZzZXHMDTQdPbOe1VeacBlX2YUthS8HvpQ5b24l\\\\\\\/K44Be0sPTaMe9BHbP9O\\\\\\\/5T28stKG5jjVoGyvk\\\\\\\/b5Ih2NlnjGTmTXg5oxKMMlqgyqJSwEp8yxsDicZdphLx1clzqc6SP7sGfyJCUcBwcVwwcDTkAlB9Z0b\\\\\\\/YbbhExIOA1Fszd1Htzzdwqb6\\\\\\\/osrOkOXudaJJtl8\\\\\\\/jOSmof3lx3NHM20HwCRztTYDXoFXc2KbZxy4UsXt14pi3BGLIZflDkTT7z0Fs8GWpHVO2L37GBd+DVFpYP6vXGDhxgwr+lrIw9XrOfCZJggtgMWbEIb5D9EK0rP1kuwYv1TtLb0EBqnBbYpUMsWvk9gM7+WfYH3ywv0ix5rPNQzBWo1qVeEi0p4k4wpqFVd\\\\\\\/pZ2fxgFA0js1FjOrXbpB1UHo5k+qeEJrdL73RH2tJU68OQTsiQyaurkUfKuOS34\\\\\\\/q0kziHSJ1aVFrof4KQXY780hoENdARg3vwQqT\\\\\\\/0ClG4+9v0GPJWhAH0a8KPQMZgZIaPEu55zGo\\\\\\\/9SauL9TKbZ0lZGAX4MaGL4QXHNe8Kh4pATNaO6H0plrzpW0JZxy94gH3Nof2JVsXCqUGNcvmoe9ovlzl5vBRBItKtOxOFVg3NAoxJlOI+Tpv5BEHVCe5mhAtOJfLdUO0Y6em6OmhDGANR7zxwK6MUqkagGUBHGg8gZs\\\\\\\/NsAkksWppEL5ugT0+yNf019H\\\\\\\/Ld1RjLVwn6wQ0HT04bvJ4AijqXBLF0Atc8sP95HGEHEJoPmj2XMNFIqh+ZC4cRSgTXFvey45S0ZGFWVUAg+YlUo4f1WMEjzVXdWdYF26poTod8geamVUX+wFyjXgi2pcYwzr2\\\\\\\/iFMArkC6EpZmZCxdQ8yV\\\\\\\/049gqm82UQ9ILH92EWVfnL8iPWaRY\\\\\\\/j1BrgfxtzGzpI6oqtTJpRVXvTcNulOln8Qzq4l\\\\\\\/5kSQ+bg3cwQD8tDYwmk+XeIfz2zJf5Sp8py6zbm\\\\\\\/AGm5cZQkAaCZX1Em0jebBa0OpkSy1P5jqS4p0sumhJQyReXwzi7l9Qa8dwkYVF87aDJZAH9WsH2C6z9phX0vzpLNBeTR7jz5j2oFeS9pUmHBsyj8sjxQDyC1jB+5wrdeix5JShPXvu7+25anZeNFRpVqXgmhmKGBcZanpvVHVA\\\\\\\/wsY\\\\\\\/yIdqtI8U8+4AmOvxBdsjPqiOBWUdtYzzSUjOa7dKLVY+UO4kCdlEp8iZ+etnSGNkQhQ\\\\\\\/FPuWvJBmvCy0TfSb7LizbQJRjmLyWVIvAoh9lTqLBPVCCBDQqou7KdvF+QcGNXPBtlsDV\\\\\\\/wnkdct\\\\\\\/CpNFyF7o8YF0BSFzP1pCN8kK8CMcyYYKfi3i\\\\\\\/jdFmXnYFydG\\\\\\\/aAZt9w5rmEKYMU0hlfEN5HN4Te7qjgYlRbg0nPjqGAEjbHwEgQcM9vVMuRn73gWfU56YzbDqe4oXH7I+J+yk42tFk2TUtf6gdMP7up58+BG7JOzKTaTJwHW7BAlopyTQkhrgskD\\\\\\\/auoYdRw86J+TPg75B+DuQQETg\\\\\\\/FOIvB0LgRsnkMUL7qoRTdfBXruXsKq6iv8s2+lqvziE7QKw+SCtA668o4YR4037cUzkPrl9KfA\\\\\\\/o94VzI6KIL+7mg3HMgI0ZKQn64nsh2X3+a5TGs8+7umo74zztXZOiJQgmZKIlLtX64+n9r0G1GoR5ZC7tQ34IpVGt1pNFmND\\\\\\\/vXE3Gv\\\\\\\/yezhp2mApPXPBd4EPhje0uIyCzmDm3oXF9OR06+KLF05f2BGQ7OpvRYJ5QWkzMwfJwluIqi7YREBuckkZaP8mzEYEt4dy\\\\\\\/OhutuPAO8E3rFg+vrMn5SkK8ISmPXaxWbG4Dcq6YQ5ms5l1NGrtH0lUTwIrkJgdfR\\\\\\\/3csjvUk3zqDRMQyP4Fj1EVjSrhujrs2dcy+NkjAV6VLGU4eCdfImJA8RuOB+eQRVpD8qPRja3\\\\\\\/H3t3Ervm+12R8O5dNSufJE03gZoio0\\\\\\\/d8Yj6ZpUn086HGHEbqOgna4I8Q8+U20KsN6XbF8HixDsh4Exf5AmgmSAWPk\\\\\\\/l3UxZYW\\\\\\\/JxR1g0wSvzF4AUmhWjV1Mm3Rj9fvSEgYP78e5rZO8wnZN\\\\\\\/rjL\\\\\\\/xLDWdtOiGAi9qKNxRXIxFYOpDgnd7fH1rYlS+mPVnNuEiW2T5bcue4Xw0tTd9HHuy03TxwFNyaUz20ckmL8fZjD3a1jKIfDDMpRIVyTVBtN6FCMctf0IZl+jC8Rzs4cIh7HOLT2NWid\\\\\\\/XLQoqQvRZ0e5gP4Gj97S9Lv2fPl1DJSJ0sNhADK0c1VW5ZK9GphaOZnZ3vD2HsIqMoRZJ4CVioqfw0SFxW\\\\\\\/SYfMcIQoctJoCMoBRVsvd4i65tY7p\\\\\\\/QrBFBBl0bXoZRbBaOwXuHYtCfOyBg5Sx1yHDJH7ujg\\\\\\\/eKJzdylnGpJCnyfRmCdoSdbTP3BezyBCqQLm\\\\\\\/5iX9mA0A9XBwTiurZWtEnnvg8lB17d+hgMfHIQqf8DtJagHKVeusiUb6yfYL6KwLB8vsTzvzI2nJZYS8gMlGVtuyzqc\\\\\\\/yq5ssXxv+18Bmt5llitlpI7ZyVly4UHvnrZcq\\\\\\\/CU8DmhIVqLQ4kIWQo0NdpiX7hK0SXYB73A74hsIWkDA8IIODsIYxHqX0xsNVnw11ZFwBkPeGeJdWwjvN8p9WHzyVtRUnNLfLujDRKdSE\\\\\\\/xkApT9G0\\\\\\\/e03Mkxx0x9LfsIwVyGDk83sYzdzrvRhZh355f1adfO5\\\\\\\/pkncUerwW2Kak0w2z20tE1SGIr2S2XqlYEdccQvhdpVLfj2+w1\\\\\\\/uDFOFka9ieAZbdUn74bKCp+f2otyT3YCDqbpTvnZYW\\\\\\\/hueSWrXpwbUFd\\\\\\\/tp0t+fDChu4MfgZI9TUugYT6UFbEAFFjXeTSdzCYpASoTFlpPDRvaLtNKe7FXevEUlw0gjW6cyASAY5uSubb4CZh1BUiUAKcqIjBEKLsHdt6mIbmfgT10pgpaGOVeih2E8281NHavTe6\\\\\\\/UMCZyUeAPia1lN+kWa40NExR7fefVIDh3YNv3ocDsIq4sGv7zxdOq4fhGSxZ+W+3aJmFleS4M8zhUJfc\\\\\\\/Ol4m\\\\\\\/Eq9RSSgbtCoCRl3ARNEzlMyFt6qIh8xqFt2MmBrtqME1gJFsv81lT2HhzqQmcv4gJ52jQW7ucYGXwy555S21QQQzIo8ghtUrSz94lGihEqtiv30cnpIzkP7EOGA7CHTPET792\\\\\\\/YcXle04YK3kk\\\\\\\/lyP28uIzXNB9HM1KWAatvIcuEwmC7go1uM6S9V0ZCEXGEYkjpXNEddJB+bFPELV3hzXofIj\\\\\\\/icVDqwavUtRzuS49Vs\\\\\\\/W7miDoqrKKzlJYdKV1FlwQokCeJNkx8wJSOqGUqSrFIo2Y2hVQ7D+gbcP1LGJz5Ya6gD5orjjYWH5lnFxoXk9UUlK0OFDgMtnG+EX\\\\\\\/6CMyGssn5UES1pcB+6quhqCaQy3qu18+LCd\\\\\\\/C9Ga8hYCgGQ7280heG2BIOY3eRFrh7D2ZSgvrA9di\\\\\\\/22giqDsij7sNS5VEZhZPvbO5xgRBorcOKavwsOO7YwgQDqOpdQZrxQqPbTRf52KGYl+HsSVL1fo9u5j+Bqfei3z6WUOvwKVfTAMKTwDfVYTsr51zv4mDxBAI6v1DMigEdPg9lUWxDNjIy4jXlDysyRtPr3i+t6aBaQWdgafsd5JVuqdzC0uGMPwZ\\\\\\\/QkP0IvvzhAKn6vebcjGEMn\\\\\\\/x7kVO6zZmu1LpBc0hZ8o27bzo4ZxivqShPd3+eeyk8+w8Ho9wMhyHwVV2biJP1zqBe5VMMxSzrsn0M5XhXW1zRaJLCTy0OvVENI0rZkvjjbxNr9DycugwGa7mipLAjaZAEYFNjU17SQ+XPHwnSMsvDgGys1SlZtyfCWpsICTlhhojPREUDn\\\\\\\/hBFkYrNOd2bjUp65KMXnq91htjX0\\\\\\\/28badIHCfhOrUMtXDmjR9BxmGuqKMIIhcH6afZ7lVsIHEqQ4ZzDrUvKY\\\\\\\/Xl9UAgpTfy5AbeiVMhOYdpJ9GoB0bgWGQC\\\\\\\/tmLZuUtmpTFIT8ivTdORDXEWVPZa2a8eO6kBD6yA1njM2a+Ojk7NzHunEM7z4RgGt54vxDKsyti8ZGBM65CwjnTh8sA9DJ+hLiTlDd\\\\\\\/f+YfkPbcZ6MgIQczOnBitpEYuBoJ0W4q+SZh3ebJUMLjS7Vd754GBer0Mlfyfn6C9qnWlwZSJZ7CmLFzV9DoDlXwH7t0T02N2uJH7s+DNQZ9SOkcH5frpZ4eFTliw\\\\\\\/\\\\\\\/qUzf6Bnz9gaS63iCfpgvBRaXD6KjyHroZX+QFISfUzCUE3\\\\\\\/HCXLMyprOBgAfbMfgvqO8G39bcZhbSAy8Mbhn46ofWwAGb\\\\\\\/R3dIPmtrRCvKSQIhilxk50Ee6O0CWPgV34K+Osyodj9yvQ2mWIN9cWPRHfxM6UP3L3qqUK93z\\\\\\\/V3A3oXBvp42Fo57uynzAZEk8JxZcbATDh8ikCI+GAZ2bRGWQtms9Y0QrHrOYpm6LOt8qFb\\\\\\\/7qAeOGOB14N1AkL7jLFg4PI0RIJIpAXknWn9Zit+jzEkV+LWBpOD4IVfgX5ExwnHcTvAaCpx+L1JuSwC3JZjvStHFosaWI8vaSTjxZ+yjGjoiZ6C+dEySANg9u1qmSFabDkNIU0bPBnHEyJ649qP9fR4+f3v0cxjc7VTCoJ+Kd\\\\\\\/bx16H1hwGySScJ\\\\\\\/3wPNytUlaweW5VbQX8Q4h2xCDK6XBWMBQ5BI0bn1tdkb2\\\\\\\/q79dVD4tjq+uHnttB36peBG0bWAGMCb3RMvaGO6fWx4rGZZgbk1Q6\\\\\\\/4W6F7fu4ZxckEsrBF8qYq\\\\\\\/WFw9sFq7+0eO0ZWo7Y03KLjOS2aF7cMMey3h10GdK0U3Zq5oqMA6jABu6BeY8dRdld6\\\\\\\/P\\\\\\\/Ef7Q8PY8ZjXA2\\\\\\\/Pb6FOugHW7doH4TmJINQsILRwy6GoDlqqNr5jqu2IgPiObUpU5e5GqdceCAuqBJJGXRlHVieCW7HPPG\\\\\\\/ltEI9Ieoa95v64\\\\\\\/WaoPfG6ZR8ss\\\\\\\/ofNYVyCk1CJXznCc7\\\\\\\/NUjRIMcVh60wD0LuX04FDbXU3T1AVbSjneVyrjqO+96WW0dE8LvgMLFyjnTQP1iPuxpXzlTsg2V\\\\\\\/SdaFcPZfMi3tLuNh\\\\\\\/DFexzmjJoc1dkYow1kuh3yyXebBHQYIyJS8mF0CwClHRHAMuFfRSX1G5QOrBdNtpWCsDvPhTgGJKeFuWPmS3mut0BHA8y1Z7dGoS+G3GRgsJ8S7XFx5ssFPbCpaJulQbIAXZtTttBHCI0vLbcsx3uuEdQ4EhnFlkXhTXap328EO2S3SVuzif5LfH\\\\\\\/aXI+1wINogznuwoI9AvkF+LJPBm5Tm8oH2OT20vdXHMWaDjlkHjyxnmfdBEvlvc+4\\\\\\\/ZbVvTUG5Xsq3eKZwpwqsqe5T03+eXcJJdN6j2FyLaEmPA0JBSNp2nUAuh5rQ4KHKX7EPdKExsqJRFo0yZvcG9Qq7KTUDR0E46lt+egaqDIjK6LrUyBQsbirFr7snp0lfRMF3NWqSjB\\\\\\\/AYkmMxzEoA2ocQYdkFysZBT0HNhYnBliol4MWN7Gj302ebfXNjJ63mTQYOlZJSwnJmMKC1htBVCZeIigUNGq2Ynjftz6QBmDW3hMuA9qse++N187OOdQblwELXkbyCWrENdFg6q42kPxCUhZI8Sp85a3yIYwFIjwpjMNP0bDhtbs6io67Q0kHqV7K6W2ed1R\\\\\\\/nV2gypY9j3QuG3Yssy8faF8JTtukvzQ4ECby\\\\\\\/ibgCSgKgVJxKSLWute9tA1SqAzurk2BsXhD4dIEv2I8tYaLGGY\\\\\\\/r0h3VxILdskzcBWeHbyDWQH+sSugEC8kutWxVcRSsLphT3MpyCnA4Rp37t\\\\\\\/ZLDUsVlh8qadQWhfngovKQ4u1ce\\\\\\\/rmfoXsIh+zNPRaxLM7URdjvAybLRNvoBUMBr7Mx77DCqyvUZOgkJPN\\\\\\\/KvO9pdroMvCmUtU+HHQE2j6CWAwwbVMXO7KoHFEAPDNPM+IQomRRW3apOtY3mb5h6ilDvRdCNv6980hsrfsVhzlCy2WiRIXdxZP4U2IL8wV71HN\\\\\\\/gBcy9f0FU31jlm\\\\\\\/Tf66aeG1KrZ9cRjW1HTb4kF2AcxxQzy1NsRq7FLM4RBJS1hBCKaHSuJqByO5dbfAlGdiaqTcP9uHR6+F4KcTq35fAmB\\\\\\\/UFRQIJVboBnlzrukXQ7\\\\\\\/i9PGPv1K063HOFiTKI\\\\\\\/WT1U7exR\\\\\\\/y3CtwNxC9JpUDDZufokYkEUIMt+yz3yaHfis8mmcM6rrKg8BYJORlvrn+b4pUPdhN\\\\\\\/3jn6X3hFNQtM1fPn1LidbbYtp8B+KySaJAQIjJ6UCb7\\\\\\\/xgAuuH2nzsQbCLPtrQHHADTJgXE1Htb3\\\\\\\/4uz5PGkDsFP9F9EJMiks22FG8nAswCaY7p58szuV\\\\\\\/\\\\\\\/n2oTVzFdu5VSUzkoMDxdJbZ7iybpYbPLYdrDZBoUjnfKIof1a3Tr3JWpMrwzxK2AIQoQcFPhWEXVwXxOrFp9zgU9y\\\\\\\/1fhT9FxbMD2E6kCAh2IO8KpaGyKx3u4w\\\\\\\/4ZSAO7eSgjXBRb1pmYZjsT9B64dYMTEYK6Af2absEgm7LdOq7EzLOgf95bzCtaCPM1Lf6TEI4aKwBgFPD7oO9EI32ELIuSxCcgk3lDg4I35iJj1IcJp58XHExvUtBaUJqEa3f10qndMb6YtQXiQnj+aQfPL65TsygDtZ5+qQ\\\\\\\/DDUCkIJk\\\\\\\/M+D\\\\\\\/7LdFRheqPIfDpS8Xza6xon61lE0fVQCwyfV1\\\\\\\/nTVkQkDmpSMkulciI+9WU9QCp6\\\\\\\/+K059TeLi7wHP4JHNd9OuyGtHBWrKxUcF8u\\\\\\\/BRkBfSfk9orYqvF5RqXSWB+Yxoh\\\\\\\/4uQRhH0IqcfphvJfcd3WiC4EDDGINBBirZXvpYK1qCkBzSrj7PgM9Jb\\\\\\\/9Cz8IJ3qBC7ap+mvpHh+L\\\\\\\/2GkdE5KefDCwD7opY0fAzVSokvLVQxJWM2pmB7nte0oEz04aAQYO0zvM0K2AEmuKSgTqm\\\\\\\/OM09PUbtkaRBJCJJ3mhGU4\\\\\\\/cqgzjpQqGCza0gsEpu2mp7YKNEyuP9G+VJn1LS3TGduEc+L7CnWS0efJU\\\\\\\/8lmdU4bYXw1tIES7BEibwcUFltw+kl08PmTa5MiAenGKyoqzIQFjsd6Y7xR5OJlOSF6D02KvYKjn5cO1mML8Aq4L8L3sct14NY\\\\\\\/CA\\\\\\\/AGbHN4PfqxUMcD7sFoP394\\\\\\\/x6oyt8QY9RepzWHL+B\\\\\\\/Unwe99Cezb69wS3L4sSMc8NdxPRLA3Ls3ZLxeU+XJAiCFDfuxOgjHE8K4x0W1+qKqEnke7aVlQJKKkXKtkZlqnALsGIEgASPiL+hvouC04eOzZkGG3ehCZg61jan\\\\\\\/GIDwG+Hd64G0LxYstA1zNmer+Ebeq\\\\\\\/RAtC41WgaTHceu6PXMpMqan6kLSQgZwPGyy7aUJMpCZkl+qtaVWyYdd4Odgpe9IYH+XrMv46zQM8iEvThuZhFsv5JthE2PXqimxLJg+oauFeqEvHDpx3o5Nlz8bdphUz88RBLr2ljUQZqTv+T2sgYT+GJH4uX5eSVKx7ctVmkQnYvZLqb+xMecUOXOo4FkY+Ab1tWLQCzZ4oNfe2ayYW73D4wfi0pMQGt8On+juNvumDb2WA1pMouBCBDMMd9XVnEFeHRsIkJE5MvRKENGanczTPElGv65yjoEsx\\\\\\\/BwOtcAjKoZ2s5vxTo8rcZfsIiVXjA5NJbL\\\\\\\/rNcX4H\\\\\\\/W91WL1\\\\\\\/KRWzElETyEmuTciM8Ocap7n9fl7jZqeLWMFC2fba4VCqfY161sBzwhvL8Iyg5EAzGHurS24xymXm36i0lW6ezIm6FVUnU2NrSOLgMWxYxeTLFgdU0Dd4lSu9\\\\\\\/JUwaKeoYdO9XgXzJePX69bVN8qwsddfF+lzYTgTsL4QL+mHgT\\\\\\\/8AqWmnu6HSYcdNfY0Ye09x0o80Hq9jt2MjGClJHIIsMkgDKoKApI6mXr4+cw4olOIT0P833zHRgWTblHcxh+FC8ap\\\\\\\/xYAPC5oXbDGiCiz4K\\\\\\\/aUpUgTOmarGJ2cyrXw41w9SGiaQKaTqF42ChydrVSkSQjRTfY5PzhN78DVZu0FRz1Kl6NKpOYBHM6qZ\\\\\\\/2tvQMRM9RPw2gXIyfiYAQFrx1QXbRg4FBsculqaW0OiTRI3XNDnAXZD6Zr6diKoX5x5X2YBTEHwwyx\\\\\\\/yYFUTN5oKyHLt\\\\\\\/brnCjXcZi2\\\\\\\/G+ALnkd3\\\\\\\/ObQ+cohOP37YjcuhfXWgvfqjcJBJeDopSHSGFtyPYIH+tG4zTml6Fwi1bZDtPmudhThc6HAWun0jZxr+9aICr2lMTDYe9mNbPvxo3BLhwIojNbED2DAwSbW7EhUlXrl7sSZbYJ6iowKd551CBx6jxi2jo1UbDMch0xTlEdJHmGEZV7cUXAgrAHLQbzB436oTAytk0hQ7fk85PxIBjqqypga57TkXag5BQsQP3ltCUE259Ou3dUQu82ILao3Pu8HeEy8HecDRqwC\\\\\\\/rfezbtg\\\\\\\/8xIwEiEpgg95LDuu1+SceDCC8wI5S8+Eb8IXxzfnyoQQfP5Nw5jd+3JrbNjD6sHUmB5MJlUfU4kcMQcOAXRYvYme6Ol88Oc3unh8nviEkXEnvk8Tqw2FaRxar8a3Luq5qp5OM5fYOYh7IZL5bcLjPpDcSjCp51rVyFR0iG\\\\\\\/JoOzsNEqytyroaxuldPiv8H7U7u2LZlJBgYR1GCDVNbvxR+GsnPKpd2aLMHPdOVJmDUTrvVOvNWfJaiiS8GoQnMFbiXoxufulkpNANB4QIBfmrUwIn43LB+lGhIJEHIZ0SAck73DaALpxnMqYXSorhkm3nUMk7XusTzDro5X01C3xW6hW4RsISreLdBYz8KKpbRiNau9vsogNwUAq+t80DLgsGK6hpb1cIdXrzu7bNcJ4YD7qvxz4KfsMncIqlWtvTZ5TC\\\\\\\/qp6xH\\\\\\\/c\\\\\\\/DPuZMhteZwQgVJcaXOcfoWPp8n\\\\\\\/zLcd6ygOok91dOV9VXxIh6IrKMdUin1fUD4KoNoUKlDEKrZLZpN2WwPEjQGPw891lBI2zHxOOxgKf1nailhaXTyRvA93SXdvahUI1n5fsgcPaKM8JcpfIrgq0z7EzsdLynoeORYbUYq67nwEcY8JZ9bReEJYBsglaRwTwVOCSVDSAyWKXA\\\\\\\/mfs+XTRSO73SjO1gucguLNjZ5fS\\\\\\\/JgcZG12PieyUGsnk952oPbIK6DvmBmJZYmHgk6xCAQGp7ONwcSSrZKfrHD60h8f5gT0JYAxIHTkihIlX0Aty698toqvxDKUY1F\\\\\\\/YKOY9gdzJ7QosqPPh48h8AOjTL6RMHMj40YEg7AvNIVfkCBiZlin+MpFl8qKAf+us2z+ANCi6au6cr2dtNzTwvr8QDutNy8TeAySfmb0LTyX4PgRHEqeKa\\\\\\\/qbcfOqPdiE6w1p0DSwTq+EScsdkEJnWXAVxx79gz69gU+gpjVIUcJgrq+B1aaDkz35AoMGRCClXoNYQ0OxXMbnwGGwt1fza66YFjKPDObT\\\\\\\/M\\\\\\\/sUfaWsk2v+78rewKQoPnpBYO8mpwa\\\\\\\/obYvygO0kQJtbGXzgdNVRgL5ptNle3RHNohLI33ZAIRASOgG3tUdF5+C5DyIkg4CZcVL4VwVHEO6w3AVZYR9ZFx9dTihE4pf+pescVI6nbLJ8pOf9BypdVubi+0NetUT61HlSlFvbUqHs7uM5BBeUQOnqfc5v2kOdgp+1oQ6f2tFGkd8flnzFXNyD85VXyeRTReiDf4yyo1WFWiEcNhsyCyjP13we+qUh1Yt3DOWyDe12+zt0\\\\\\\/AWjblqhuiBls0DKN1L5MocEp+Qvxog\\\\\\\/PjtO\\\\\\\/ZFRIu\\\\\\\/b+bwDhsggNCoyF7oTVrL9jJRPZbR5FebCt+fgR+L7o+yl1zGTrjz7vwRX0p9tlf\\\\\\\/eAyIXvEMhUd+oqUzhzDMY4EOqpS67oSMwhggODnU2z2oEE7opmZp88jx9X79Bb3yo1kqtMoxzgwkSGT1O2V3nk6CQ2Y6CyTobZ3qclldlw5s9RsIs4g\\\\\\\/Gql8iFHJcc3VdA9TM8s83ygf+VZZ\\\\\\\/X3KetgoXcB+HRQMhxJz3kW3vIVGVw5SUIKIbEuS0iyypceUB5coy+gdRHP0ljAg5\\\\\\\/Z6jchGBtBpYp7UBE+t5I2F\\\\\\\/CMHFcd9byaA1rmAbVqsDUgYT2iJzuN8q59YfB\\\\\\\/xCjFQFbyDcc8CYrTKc3+BKtDzgcoJZt\\\\\\\/Eg1QbNGCtjbWCISSEnqQYVINyGqBR3rikbr\\\\\\\/BE30qTYR+NaX85DkWNYIN4ORoVxXzgyqR2SGGSX3Ad0JOpEndHoNR6wPDbU4xIWV6WR7pFLgB3F1jYW1Qe4a3quiK7uUXHFAYeVYXAK1hYodIwwbfcQlmSeVmqGPdE\\\\\\\/t2KDrnrgYn0x9cpOl5TAB4o8c7xvOBdqkhRprkUAD3P77zESCBLXaPMEYkn9Xweq0MRq4pD1R18Qtd6l5JpSdfpC8ucQjGjVClLKvZCRrsqkKV17AFXTalM+HaP4P8whYKe30gYOcjYDJQURttivecNsbKa8jAUIbbqydslL1yFhGjA58ZTcQ6h9xbZIMXMYcFHOJeDuzN6fr\\\\\\\/36XUb9Gh2GGF\\\\\\\/bjCuSQ1SXW6UZ3z3uL5VIoqYAi11r4RdamrdFml+sOw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022Xl7p\\\\\\\/TUkfKhiIxW+6v3iMw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002290c56d79b955c5085369d946840ade8443ebafe925abc7a9b7e768338268ebfb\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022XPCtOwX88gsW8lV+Aq79IOy5PMuCnJHqKvpqVyra3q15sZmZa++wclWIygr4LB4vSsWhhtXqum1dHeoe9zSdzdqYIx3ifa4cF8vNw59bo\\\\\\\/\\\\\\\/t2yBgJbYB9vhcZ1PfsidtyKT4w85mC27hWX\\\\\\\/pXtYGxloxdOXxaqF4dn8YBvl37o8yLKF3mPPSf9\\\\\\\/Fzy8T\\\\\\\/jtMDRHwphiiqDfDxFKNQy5PsCSr4JIjeD6gJgW9iYpFqON77hzEDmzxtr\\\\\\\/VvmxzYCABnCYLf80eE2jcTWrpJkEeQFNF8m0P+IDkqog9EuY8SfJyd17voO+kYO\\\\\\\/VjBp\\\\\\\/eG6S0je\\\\\\\/yAtPy+R3M+z5hkNMHKnUgzw5QL7rOHqYbzi6FUu5vAIzRG5ro+LIiO0MA4ecmUjQl4pq\\\\\\\/y5gdEWbPYbQa138VQyYT8HKIo6z0tXbJ3aTRWgznfjJCv4tIez\\\\\\\/6Br0F5FrVONywLVbvr7DUetIwAgilXsCcNk6IuZ24NeeZtPFKTqnylDqsPhVgJErApRUcbjFyVhuXHN+CLmfPxTdgtmE2d5VbFexZ4cvz9IuRDDkpuxjVYFUnt9bmxmc5nl+qvOH8um\\\\\\\/21lQB169cSGTSkMqCfJV5ujR7eqw1g1pX2H6oI\\\\\\\/q\\\\\\\/zg5Lf\\\\\\\/4hCAhoyd4AfKKemZLiT6iYQnAohYg83ORuuI\\\\\\\/ZjznL7Qgihc53usuVjkUb0+hHw8Jlsd4TKylP1T8Zj8kTpLsCgUX6CnSBFJvzmNEUmB39OaxdG8vzydPNUtRbiZl3q\\\\\\\/1b4TTrYjc\\\\\\\/wVL2FWTGss\\\\\\\/TvNgmKR9lWjkk9rirpBKiDqLnUssgEINwu66\\\\\\\/d11dgkdk2FX7mc8b2q12s6clG2XUFti6KR0vR4xqBE4Y9DWENd7BkdY841gneOi\\\\\\\/w2m7Uzb3xkI8eAqvg268mxQoEie+tgPlKc2hYoIfCy01HXA2+WfSBJx4GouIi7FES3JHP4S\\\\\\\/oqThEq+gkhz4z2\\\\\\\/8rLEMyBcpL3Exclz8+95eN7\\\\\\\/Z3MmVP1wtPcc\\\\\\\/5BtVpf6ZhY8QBYDkCkd3NTnmQVsxp\\\\\\\/jBIIToAvxlx9R4yFhtKzYQaTRNRT\\\\\\\/PyvtD8Hjp9a\\\\\\\/rzh7Wmwnu1W6Sslz\\\\\\\/\\\\\\\/aWmVf+alQlJrGFkEkVJBLz+XLPmTOjPix6JQB6\\\\\\\/pIwzoFQlyeanduhVYuiyFs7fRGJs3eIaB+7PqubvIfHAgR6epjBjGzIrvMfgO+I43W2Fz3KONCBRwSCcEJfF\\\\\\\/hLOCz9Ylpto+S50MZpy9VEdQFWfMeVY3ukSmxAe7l00CLthIEIsDiO5n7StRBCzHE\\\\\\\/el9bjjhkhbIxr\\\\\\\/\\\\\\\/XW8zRH8LsalQuOrdEpX7qEeQq3rAXgVxvCbyPN53SvtCcYIW0p6bbUTRO4zGBiNjuiFbgUGD8UhoTMy4FnADeJNenTvH+gZJ2Jl7SBqTCCdKsraDvVruC8DBv5PZPn+f+KPie+19l4ZMQHanaHFx8ZvzihpxYjYIjQbclGqo\\\\\\\/FMqK9G1UNhXw+MJqcuhspp64UOHDbqH1o+U3hE2Ch5Dpb\\\\\\\/6QDoQksBI4OryY29a4F1AyZSWjGraehSke6Wt1SF\\\\\\\/Tm\\\\\\\/P0lL3fk9YxHX3LXQqwRtQnz7LNtzHJYKZ0+CGJOdXDJyrxMqPhvtJzzJTcePGgkqiDnSszwxzu25SyXu0AwDFtyPLxfq4FQ+XPGEXXn4HYecNG0UUwIvWmSL8uS+8wB2Na2eb\\\\\\\/4m1G0d5xN3NFLZZtDsk7EVpdz+CjfAgCs0SBtpBrS8wpDAEO2Quv6de2cmhkIX2Yvk8Mc9eQwOphSgq7KxjeWcAZUNDGgM\\\\\\\/iLxFuuNk4X+ou5X9kJZ0taw99vfj6DOKY2FjDrh+enWuqAX2XhgJsE609X\\\\\\\/HPKqS72EncjEX\\\\\\\/A5bN4Mv9R8xOyHe95T+wlhuNe5fjq\\\\\\\/vVqIjRhz1YTh4pUv68kIgxMEgzd255t5eU9NdECvUTcT21iEXrN8zMclS\\\\\\\/kfdtvHk7Rtu7a0H3gN6KOTyMD+fjG43q0jbVo8zBV+MH81vkLjvn\\\\\\\/0IgxA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022nRN1gXCANdFZuVs8W7V7Aw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022637f94e8bb180658369b4a101a0300f91b33ec70b20260345c528abf48f56fd7\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-489827 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/6/1921" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-4K6_tQ5K04jdDgBw6J5ieeV233jvlcD5JmVb0yNsb3g" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/6/1921" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-cnMPOJw2SASSKdK2HMAwRijDXvMFAiuWb_0Q1saRz-U" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/6/1909" title="Accounting for Genotype-by-Environment Interactions and Residual Genetic Variation in Genomic Selection for Water-Soluble Carbohydrate Concentration in Wheat" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/6/1933" title="Effect of Larval Nutrition on Maternal mRNA Contribution to the Drosophila Egg" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="489827" id="node489827" data-pisa="ggg;8/6/1921" data-pisa-master="ggg;g3.118.200284" data-apath="/ggg/8/6/1921.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Subtle Perturbations of the Maize Methylome Reveal Genes and Transposons Silenced by Chromomethylase or RNA-Directed DNA Methylation Pathways</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-1671-2286" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Sarah N. Anderson</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-4992-9614" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Gregory J. Zynda</span>, <span class="highwire-citation-author" data-delta="2">Jawon Song</span>, <span class="highwire-citation-author" data-delta="3">Zhaoxue Han</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-1384-4283" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Matthew W. Vaughn</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0003-3232-9479" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Qing Li</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0002-7301-4759" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Nathan M. Springer</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1921-1932;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200284 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node489827" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Sarah N. Anderson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant and Microbial Biology, University of Minnesota, St. Paul, MN 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sarah%2BN.%2BAnderson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Anderson%20SN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASarah%2BN.%2BAnderson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-1671-2286" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Sarah N. Anderson</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Gregory J. Zynda </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Texas Advanced Computing Center, University of Texas, Austin, TX 78758</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gregory%2BJ.%2BZynda%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zynda%20GJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGregory%2BJ.%2BZynda%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4992-9614" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Gregory J. Zynda</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jawon Song </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Texas Advanced Computing Center, University of Texas, Austin, TX 78758</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jawon%2BSong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Song%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJawon%2BSong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Zhaoxue Han </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>State Key Laboratory of Crop Stress Biology for Arid Areas, Northwest A&F University, Yangling 712100, Shaanxi, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhaoxue%2BHan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Han%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZhaoxue%2BHan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Matthew W. Vaughn </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Texas Advanced Computing Center, University of Texas, Austin, TX 78758</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthew%2BW.%2BVaughn%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Vaughn%20MW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMatthew%2BW.%2BVaughn%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-1384-4283" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Matthew W. Vaughn</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Qing Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>National Key Laboratory of Crop Genetic Improvement, Huazhong Agricultural University, Wuhan, Hubei, 430070, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Qing%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20Q&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AQing%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-3232-9479" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Qing Li</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:springer@umn.edu" class="" data-icon-position="" data-hide-link-title="0">springer@umn.edu</a>
<a href="mailto:qingli@mail.hzau.edu.cn" class="" data-icon-position="" data-hide-link-title="0">qingli@mail.hzau.edu.cn</a></span></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Nathan M. Springer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant and Microbial Biology, University of Minnesota, St. Paul, MN 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nathan%2BM.%2BSpringer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Springer%20NM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANathan%2BM.%2BSpringer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-7301-4759" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Nathan M. Springer</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:springer@umn.edu" class="" data-icon-position="" data-hide-link-title="0">springer@umn.edu</a>
<a href="mailto:qingli@mail.hzau.edu.cn" class="" data-icon-position="" data-hide-link-title="0">qingli@mail.hzau.edu.cn</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/8/6/1921" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:489827" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:489827/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/6/1921.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:489827" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:489827/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/6/1921.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:489827" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:489827/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/6/1921.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:489827" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:489827/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/8/6/1921.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:489827" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:489827/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">DNA methylation is a chromatin modification that can provide epigenetic regulation of gene and transposon expression. Plants utilize several pathways to establish and maintain DNA methylation in specific sequence contexts. The chromomethylase (CMT) genes maintain CHG (where H = A, C or T) methylation. The RNA-directed DNA methylation (RdDM) pathway is important for CHH methylation. Transcriptome analysis was performed in a collection of <em>Zea mays</em> lines carrying mutant alleles for CMT or RdDM-associated genes. While the majority of the transcriptome was not affected, we identified sets of genes and transposon families sensitive to context-specific decreases in DNA methylation in mutant lines. Many of the genes that are up-regulated in CMT mutant lines have high levels of CHG methylation, while genes that are differentially expressed in RdDM mutants are enriched for having nearby mCHH islands, implicating context-specific DNA methylation in the regulation of expression for a small number of genes. Many genes regulated by CMTs exhibit natural variation for DNA methylation and transcript abundance in a panel of diverse inbred lines. Transposon families with differential expression in the mutant genotypes show few defining features, though several families up-regulated in RdDM mutants show enriched expression in endosperm tissue, highlighting the potential importance for this pathway during reproduction. Taken together, our findings suggest that while the number of genes and transposon families whose expression is reproducibly affected by mild perturbations in context-specific methylation is small, there are distinct patterns for loci impacted by RdDM and CMT mutants.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/dna-methylation" class="hw-term hw-article-keyword hw-article-keyword-dna-methylation" rel="nofollow">DNA methylation</a></li><li class="kwd"><a href="/keyword/chromomethylase" class="hw-term hw-article-keyword hw-article-keyword-chromomethylase" rel="nofollow">chromomethylase</a></li><li class="kwd"><a href="/keyword/transposable-elements" class="hw-term hw-article-keyword hw-article-keyword-transposable-elements" rel="nofollow">transposable elements</a></li><li class="kwd"><a href="/keyword/rna-directed-dna-methylation" class="hw-term hw-article-keyword hw-article-keyword-rna-directed-dna-methylation" rel="nofollow">RNA directed DNA methylation</a></li><li class="kwd"><a href="/keyword/maize" class="hw-term hw-article-keyword hw-article-keyword-maize" rel="nofollow">maize</a></li></ul><p id="p-3">The epigenome describes the potential for additional heritable information that can be passed on through mitosis or meiosis (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Hofmeister <em>et al.</em> 2017</a>). DNA methylation is one molecular mechanism that can provide epigenetic information. There is interest in the potential for cryptic information in genomes which is normally silenced by epigenetic mechanisms but could be activated through epigenetic changes without requiring any genetic change. This cryptic information may represent an untapped source of variation that could be used for crop improvement.</p><p id="p-4">Much is known about the mechanisms that control DNA methylation and the functional roles of DNA methylation in regulating transposon and gene expression in the model plant <em>Arabidopsis thaliana</em> (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Law and Jacobsen 2010</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Matzke and Mosher 2014</a>). However, our knowledge of the regulating mechanisms and function of DNA methylation is much more limited in crop plants. Evidence in rice and maize suggests that major perturbations of DNA methylation disrupt development and the seeds/plants are not viable (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Yamauchi <em>et al.</em> 2014</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Hu <em>et al.</em> 2014</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Li <em>et al.</em> 2014</a>). Forward genetic screens for factors involved in epigenetic phenomena such as paramutation (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Dorweiler <em>et al.</em> 2000</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Hollick <em>et al.</em> 2005</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Alleman <em>et al.</em> 2006</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Hale <em>et al.</em> 2007</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Erhard <em>et al.</em> 2009</a>) or transgene silencing (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">McGinnis <em>et al.</em> 2006</a>) have identified several genes that are associated with DNA methylation or chromatin in maize (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Hollick 2017</a>). In addition, reverse genetic approaches have been utilized in attempts to document the function of putative methyltransferase genes or other genes associated with DNA methylation (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Papa <em>et al.</em> 2001</a>; <a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Li <em>et al.</em> 2014</a>). To date, these mutants have provided partial reductions in DNA methylation in specific sequence contexts but no mutants with drastic reductions in genomic DNA methylation have been recovered in maize.</p><p id="p-5">Surveys of natural variation for DNA methylation among diverse lines of maize have revealed many examples of differentially methylated regions (DMRs) (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Eichten <em>et al.</em> 2011</a>, <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">2013</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Regulski <em>et al.</em> 2013</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Li <em>et al.</em> 2015b</a>). A subset of the genes located near DMRs exhibit a negative correlation between DNA methylation and gene expression (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Eichten <em>et al.</em> 2013</a>; <a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Li <em>et al.</em> 2015b</a>). This is primarily found at genes that have CG or CHG methylation in regions surrounding the transcriptional start site (TSS) and show qualitative (on/off) expression variation among genotypes (<a id="xref-ref-40-3" class="xref-bibr" href="#ref-40">Li <em>et al.</em> 2015b</a>). This suggests the potential for cryptic information in the maize genome that is epigenetically silenced in some lines but can be active due to epigenetic changes in other genotypes.</p><p id="p-6">Several maize mutant lines with context-specific or locus-specific perturbations of DNA methylation have been previously identified (<a id="xref-ref-37-3" class="xref-bibr" href="#ref-37">Li <em>et al.</em> 2014</a>). The mutants include <em>mop1-1</em> and <em>mop3-1</em>, two mutants recovered in screens for factors required for maintenance of the paramutated state at the B’ locus (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Dorweiler <em>et al.</em> 2000</a>). The <em>mop1</em> locus encodes an RNA-dependent RNA polymerase related to RDR2 in <em>Arabidopsis</em> (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Alleman <em>et al.</em> 2006</a>) while the <em>mop3</em> locus encodes the largest subunit of RNA Pol IV (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Sloan <em>et al.</em> 2014</a>) and this same locus has also been defined by the <em>rmr6-1</em> mutant allele and has been named as the <em>rmr6</em> (<em>required to maintain repression 6</em>) / <em>rpd1</em> (<em>rna polymerase d1</em>) locus (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Erhard <em>et al.</em> 2009</a>). Mutant alleles for the two chromomethylase genes present in the maize genome, <em>Zmet2</em> and <em>Zmet5</em> also influence context-specific DNA methylation patterns (<a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Papa <em>et al.</em> 2001</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Makarevitch <em>et al.</em> 2007</a>; <a id="xref-ref-37-4" class="xref-bibr" href="#ref-37">Li <em>et al.</em> 2014</a>). These genes are likely paralogs resulting from a whole genome duplication event and are orthologous to <em>CMT3</em> from <em>Arabidopsis thaliana</em> (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bewick <em>et al.</em> 2017</a>). Previous research has found that <em>mop1-1</em> and <em>mop3-1</em> genotypes have lost CHH methylation at many genomic regions with elevated CHH, and there are changes in CG and CHG at these sites as well (<a id="xref-ref-37-5" class="xref-bibr" href="#ref-37">Li <em>et al.</em> 2014</a>). However, as these types of regions are quite rare in the maize genome these mutants have minimal effects on genome-wide levels of CG and CHG methylation. The <em>zmet2-m1</em> mutant and, to a lesser extent, the <em>zmet5-m1</em> mutant, result in a reduction of CHG methylation. These mutants also cause reductions of CWA methylation (where W is A or T) in genomic regions with low, but detectable, CWA methylation (<a id="xref-ref-37-6" class="xref-bibr" href="#ref-37">Li <em>et al.</em> 2014</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Gouil and Baulcombe 2016</a>). Attempts to recover double mutants for <em>zmet2-m1/zmet5-m1</em> were unsuccessful, suggesting at least partially redundant function for these paralogous genes (<a id="xref-ref-37-7" class="xref-bibr" href="#ref-37">Li <em>et al.</em> 2014</a>).</p><p id="p-7">Plants with loss-of-function alleles for <em>mop1</em>, <em>mop3/rmr6/rpd1</em>, <em>zmet2</em> and <em>zmet5</em> are viable but can exhibit some morphological abnormalities (<a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">Dorweiler <em>et al.</em> 2000</a>; <a id="xref-ref-50-3" class="xref-bibr" href="#ref-50">Papa <em>et al.</em> 2001</a>; <a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Makarevitch <em>et al.</em> 2007</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Parkinson <em>et al.</em> 2007</a>; <a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">Erhard <em>et al.</em> 2009</a>). The <em>mop1-1</em> and/or <em>mop3-1</em> mutations have been shown to play important roles in the regulation of specific maize loci (<a id="xref-ref-13-4" class="xref-bibr" href="#ref-13">Dorweiler <em>et al.</em> 2000</a>; <a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">Alleman <em>et al.</em> 2006</a>; <a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">Sloan <em>et al.</em> 2014</a>), transgenes (<a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">McGinnis <em>et al.</em> 2006</a>) or transposable elements (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Lisch <em>et al.</em> 2002</a>; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Woodhouse <em>et al.</em> 2006</a>). Microarray profiling of gene expression has revealed evidence for altered expression of small sets of genes in studies of <em>mop1-1</em> (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Madzima <em>et al.</em> 2014</a>) and <em>zmet2-m1</em> (<a id="xref-ref-43-3" class="xref-bibr" href="#ref-43">Makarevitch <em>et al.</em> 2007</a>). There is also evidence from a RNA-seq experiment for altered regulation of transposable element expression in apical meristem tissue (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Jia <em>et al.</em> 2009</a>). Transcriptome analysis of <em>rmr6-1</em>, representing a distinct mutant allele at the <em>mop3/rmr6/rpd1</em> locus, provided evidence for a potential role in stress response (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Forestan <em>et al.</em> 2016</a>). The <em>rmr6-1</em> mutation appears to increase the proportion of the genome that is transcribed but has subtle effects at most loci with relatively few genes with significant changes in expression level (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Forestan <em>et al.</em> 2017</a>). However, there have not been comprehensive studies on the overlap of genes or transposons that are sensitive to mutations in different CMT or RdDM genes in maize.</p><p id="p-8">Each of the mutant genotypes used for this study has subtle effects on genomic methylation levels and can produce viable plants. There are several phenotypic abnormalities observed in <em>mop1-1</em> and <em>mop3-1</em> stocks (<a id="xref-ref-13-5" class="xref-bibr" href="#ref-13">Dorweiler <em>et al.</em> 2000</a>; <a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Parkinson <em>et al.</em> 2007</a>; <a id="xref-ref-16-4" class="xref-bibr" href="#ref-16">Erhard <em>et al.</em> 2009</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Barber <em>et al.</em> 2012</a>; <a id="xref-ref-55-3" class="xref-bibr" href="#ref-55">Sloan <em>et al.</em> 2014</a>) although the penetrance in multiple backgrounds has not been well characterized. We sought to determine if the subtle changes in DNA methylation in these mutants would reveal genes or transposons that are sensitive to these shifts in DNA methylation or chromatin. A limited number of genes and transposon families have altered transcript abundance in these genotypes, with changes representing both direct effects of the mutated genes and indirect effects of direct targets. A subset of differentially expressed genes have high levels of DNA methylation in wild-type that are reduced in the mutant genotypes. Many of these genes exhibit natural variation for DNA methylation and transcript abundance. This provides evidence that the natural variation at these genes is due to epigenetic rather than genetic variation and highlights cryptic information present in the maize genome that could be accessible through alterations to the epigenome.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Biological materials</h3><p id="p-9">All the mutant and control samples used in this study are listed in Table S1 and SRA accession numbers are listed for each dataset. The <em>zmet2-m1</em>, <em>zmet2-m2</em>, and <em>zmet5-m1</em> mutant alleles were backcrossed into B73 for at least 7 generations. The <em>mop1-1</em> and <em>mop3-1</em> plant materials have been previously described (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Gent <em>et al.</em> 2014</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Li <em>et al.</em> 2015a</a>); the <em>mop1-1</em> tissues were from plants that have been backcrossed into a B73 genetic background while <em>mop3-1</em> plants represent an uncharacterized genetic background. In all cases, “mutant” samples are homozygous for the mutant allele and “control” samples are homozygous for the wild-type allele. Tissue for RNA and DNA isolations was collected from three biological replicates. Plants were grown in standard greenhouse conditions for 20 days to reach the V3 stage. The 2<sup>nd</sup> and 3<sup>rd</sup> leaves were collected individually for each seedling. The 2<sup>nd</sup> leaf was used to isolate DNA for genotyping, and the 3<sup>rd</sup> leaf was used for RNA isolation and sequencing. For each biological replicate, 4-6 seedlings were pooled.</p></div><div id="sec-3" class="subsection"><h3>Library preparation and sequencing</h3><p id="p-10">Total RNA was isolated using the TRIZol reagent following the manufacturer’s protocol. RNA was quantified using RiboGreen and 3 μg total RNA was used to construct libraries using TruSeq strand-specific kit (Illumina) following manufacturer’s suggestions. The final library was quantified using PicoGreen and twelve libraries were pooled per Illumina lane. Library quality was checked using Agilent Bioanalyzer. Sequencing was performed on HiSeq2500 using 2 × 50 bp mode.</p></div><div id="sec-4" class="subsection"><h3>Gene expression analysis</h3><p id="p-11">Trim_glore was used to trim low-quality base from the 3′ end of the reads, as well as to remove adapters. Reads that passed quality control were mapped to B73 version 4 genome (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Jiao <em>et al.</em> 2017</a>) using Tophat2 (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Kim <em>et al.</em> 2013</a>), allowing at most 1 mismatch (-N 1) and the expected inner distance between mate pairs of 200 bp (-r 200). Reads that are properly paired and uniquely mapped were filtered out using samtools (-f 0x0002 –q 50). HTSeq (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Anders <em>et al.</em> 2015</a>) was used to summarize the number of reads mapped to each V4 gene model with the union mode, generating a matrix of count values for each gene in each genotype.</p><p id="p-12">Raw read counts were input into DEseq2 (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Love <em>et al.</em> 2014</a>) to perform differential expression analysis. Pair-wise comparisons were made between each mutant and the appropriate wild type. Genes with a FDR value of &lt; 0.05 and log<sub>2</sub>(FoldChange) &gt; 1 were called differentially expressed (DE) genes. Detailed analysis was restricted to genes with consistent DE calls in at least two mutant contrasts in the same pathway (RdDM or CMT). Genes were considered expressed if at least 3 replicates in the libraries described had an RPM (reads per million) value &gt;1.</p></div><div id="sec-5" class="subsection"><h3>TE expression analysis</h3><p id="p-13">B73v4 (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Jiao <em>et al.</em> 2017</a>) TE annotation was modified to remove helitrons and the file was resolved using RTrackLayer in R so that each base of the genome was assigned to only a single TE. Exon regions were masked from the TE file using Bedtools (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Quinlan and Hall 2010</a>) subtract. Gene annotations were added to this modified TE annotation file, and mapped reads were assigned to features using HTSeq (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Anders <em>et al.</em> 2015</a>). A custom script was used to read through the HTSeq sam output, assigning unique-mapping reads to individual TE elements and multi-mapped reads to TE families if mapped positions hit only a single TE family. Unique and multi-mapped reads were combined for per-family expression counts, and RPM values were calculated by normalizing to the number of gene reads plus TE family reads in each library. All reads mapped to gene annotations plus TE annotations were excluded from the TE expression analysis. Differentially expressed TE families were determined using DESeq2 (<a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Love <em>et al.</em> 2014</a>) using a log<sub>2</sub>(FoldChange) cutoff of 1 and FDR adjusted p-value cutoff of 0.05. Detailed analysis was restricted to TE families with consistent DE calls in at least two mutant contrasts in the same pathway (RdDM or CMT).</p></div><div id="sec-6" class="subsection"><h3>WGBS data analysis</h3><p id="p-14">The WGBS datasets used in this study are detailed in Table S2 and SRA accession numbers for each sample are provided. One μg DNA was used to prepare libraries for whole-genome bisulfite sequencing using the KAPA library preparation kit. DNA was sheared to a peak between 200-250 bp. End repair was performed to make blunt-ended fragments, followed by adding base A to the 3′ end, and adapter ligation. Size selection was performed to enrich library with a size between 250-450 bp. Bisulfite conversion was then carried out using Zymo DNA methylation lightning kit according to user’s manual. Finally, the libraries were enriched using PCR amplification. Library quality was checked using the Agilent Bioanalyzer. Library quantification was performed with qPCR before sequencing. Sequencing was performed on HiSeq2000 with paired end 100 cycles.</p><p id="p-15">Analysis was performed as previously described (<a id="xref-ref-40-4" class="xref-bibr" href="#ref-40">Li <em>et al.</em> 2015b</a>; <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Song <em>et al.</em> 2016</a>). Read quality was checked with FASTQC, adapters and low-quality bases at the 3′ end of each read were trimmed using Trim_glore. The high quality reads were mapped to B73 V4 genome (<a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">Jiao <em>et al.</em> 2017</a>) using BSMAP (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Xi and Li 2009</a>) allowing at most 5 mismatches. Only properly paired reads with unique mappings were kept and used for calling DNA methylation. Methylation calls were performed using the methratio.py script from BSMAP. Finally, DNA methylation in each context (CG, CHG, CHH) was summarized for each 100-bp non-overlapping tile of the 10 maize chromosomes.</p></div><div id="sec-7" class="subsection"><h3>DMR calling</h3><p id="p-16">DMRs were called using previously described criteria (<a id="xref-ref-40-5" class="xref-bibr" href="#ref-40">Li <em>et al.</em> 2015b</a>). Briefly, each 100-bp tile with &gt; 6 CG/CHG sites, &gt; 2X coverage and &gt; 60% difference for CG/CHG were called as CG and/or CHG DMRs. CHH DMRs were called using the same coverage and site number criteria, but with a requirement for &lt;5% CHH in one genotype and &gt;25% CHH in another genotype, reflecting the low level of CHH methylation in the maize genome.</p></div><div id="sec-8" class="subsection"><h3>mCHH Islands</h3><p id="p-17">High CHH tiles were called genome-wide by requiring CHH methylation over 25% with at least 10 informative read counts per tile. Genes and TEs were considered to be associated with a mCHH island if when at least one high methylation tile was identified within the gene/TE or in the 2 kb region surrounding the gene/TE.</p></div><div id="sec-9" class="subsection"><h3>Data and reagent availability</h3><p id="p-18">Mutant lines are available upon request. All data used in this study are deposited at the NCBI Sequence Read Archive (SRA), and accession numbers for all libraries are listed in Table S1. Expression values, differential expression calls, list assignments, and DMR calls for all genes and TE families are listed in Tables S2 and S3, respectively. Unique mapping read counts and descriptors for each transposable element are listed in Table S4. Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.6071000">https://doi.org/10.25387/g3.6071000</a>.</p></div></div><div class="section results" id="sec-10"><h2 class="">Results</h2><div id="sec-11" class="subsection"><h3>Alteration of gene and transposon expression in maize mutants with perturbed methylomes</h3><p id="p-19">RNA-seq was used to perform transcriptome profiling for maize lines carrying mutations in genes encoding CMT (this study) or RdDM components (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Gent <em>et al.</em> 2014</a>; <a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Li <em>et al.</em> 2015a</a>). Together these factors are expected to be responsible for the majority of CHG and CHH methylation in the maize genome. For CMT genes, three biological replicates of seedling leaf tissue were profiled for mutations in two different genes, with multiple alleles utilized for one of the genes (Table S1). In addition, for the RdDM genes we analyzed seedling leaf tissue for <em>mop1-1</em> and <em>mop3-1</em> (<a id="xref-ref-38-3" class="xref-bibr" href="#ref-38">Li <em>et al.</em> 2015a</a>), along with immature ear tissue for <em>mop1-1</em> (<a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">Gent <em>et al.</em> 2014</a>). Homozygous wild-type siblings were used as controls for RdDM mutants. The genetic background, read number and accession information for each sample is provided in Table S1.</p><p id="p-20">The transcript abundance of individual genes was estimated from the RNA-seq data for each sample. Differentially expressed (DE) genes in each mutant line (relative to the appropriate control) were identified using DESeq2 followed by a requirement for a minimum of 2 fold-change and an FDR value of less than 0.05 (Table S2). The observed differences in gene expression in the mutant lines could be direct effects of the mutation on transcript abundance, indirect effects caused by direct targets, or could be the result of introgressions of linked loci that contain <em>cis</em>-regulatory variation. The number of genes in each 2 Mb bin with differential expression was assessed throughout the genome (Figure S1). For mutations that were identified in one background and then backcrossed into B73 (<em>zmet2-m1</em>, <em>zmet2-m2</em>, <em>zmet5-m1</em>, <em>mop1-1</em>), there were often a cluster of DE genes surrounding the locus of the mutation itself. For the other mutation (<em>mop3-1</em>) that was not backcrossed into another genetic background, there is less evidence for expression changes at linked genes (Figure S1). Based on these results we omitted DE genes located within 40 Mb of the mutations in subsequent analyses.</p><p id="p-21">A principal component analysis was performed using all DE genes to cluster samples used in this study (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). When combining all samples, PC1 separates ear transcriptomes from seedling leaf transcriptomes, and PC2 separates <em>mop3-1</em> samples from mutants and controls in the B73 background, suggesting that tissue and genotype represent the strongest sources of differences in our transcriptomes (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1A</a>). When comparing only leaf libraries in the B73 background, few genotypes were substantially different from the wild-type controls, suggesting limited changes to transcript levels induced by each mutation (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1B</a>). The number of differentially expressed genes in each mutant genotype relative to tissue and genotype-matched controls was highly variable (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1C</a>). In most cases the homozygous mutant individuals exhibit more up-regulated genes than down-regulated genes, which is compatible with the concept that the CMT and RdDM genes normally provide silencing activities.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/6/1921/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Summary of differentially expressed genes in mild methylation mutants. A-B: A principal component analysis (PCA) was performed using log2(RPM+1) expression values for genes that are DE in at least one mutant line relative to the appropriate control. (A) The full set of samples used for this study was assessed and we found that samples in other genetic backgrounds (mop3-1 and WT Mop3) or tissues (mop1-1 ear and Mop1-B73 ear) have the highest level of variation. All genotypes are homozygous (mutant or wild-type) for the gene of interest. (B) A second PCA was performed using only samples in the B73 genetic background assessed in leaf tissue. WT samples are denoted with triangles and mutants with circles. C-D: The number of up- (red) and down-regulated (blue) genes (C) and TEs (D) is shown for each mutant relative to the appropriate wild-type control. The percent of DE genes or TE families that are up-regulated is marked above each bar, and the number of genes or TEs with consistent changes in two or more mutants from the same pathway is labeled. E-F: Each of the genes (E) or TE families (F) that are DE in at least two CMT or RdDM mutants was used to perform hierarchical clustering using the Euclidean method and the log2 of the fold-change relative to wild-type is visualized with a heat map." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-395745604" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Summary of differentially expressed genes in mild methylation mutants. A-B: A principal component analysis (PCA) was performed using log&lt;sub&gt;2&lt;/sub&gt;(RPM+1) expression values for genes that are DE in at least one mutant line relative to the appropriate control. (A) The full set of samples used for this study was assessed and we found that samples in other genetic backgrounds (&lt;em&gt;mop3-1&lt;/em&gt; and WT &lt;em&gt;Mop3&lt;/em&gt;) or tissues (&lt;em&gt;mop1-1&lt;/em&gt; ear and &lt;em&gt;Mop1-B73&lt;/em&gt; ear) have the highest level of variation. All genotypes are homozygous (mutant or wild-type) for the gene of interest. (B) A second PCA was performed using only samples in the B73 genetic background assessed in leaf tissue. WT samples are denoted with triangles and mutants with circles. C-D: The number of up- (red) and down-regulated (blue) genes (C) and TEs (D) is shown for each mutant relative to the appropriate wild-type control. The percent of DE genes or TE families that are up-regulated is marked above each bar, and the number of genes or TEs with consistent changes in two or more mutants from the same pathway is labeled. E-F: Each of the genes (E) or TE families (F) that are DE in at least two CMT or RdDM mutants was used to perform hierarchical clustering using the Euclidean method and the log&lt;sub&gt;2&lt;/sub&gt; of the fold-change relative to wild-type is visualized with a heat map.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/6/1921/F1.medium.gif" width="440" height="302"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/8/6/1921/F1.medium.gif" width="440" height="302"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/6/1921/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/6/1921/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/489986" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-22" class="first-child">Summary of differentially expressed genes in mild methylation mutants. A-B: A principal component analysis (PCA) was performed using log<sub>2</sub>(RPM+1) expression values for genes that are DE in at least one mutant line relative to the appropriate control. (A) The full set of samples used for this study was assessed and we found that samples in other genetic backgrounds (<em>mop3-1</em> and WT <em>Mop3</em>) or tissues (<em>mop1-1</em> ear and <em>Mop1-B73</em> ear) have the highest level of variation. All genotypes are homozygous (mutant or wild-type) for the gene of interest. (B) A second PCA was performed using only samples in the B73 genetic background assessed in leaf tissue. WT samples are denoted with triangles and mutants with circles. C-D: The number of up- (red) and down-regulated (blue) genes (C) and TEs (D) is shown for each mutant relative to the appropriate wild-type control. The percent of DE genes or TE families that are up-regulated is marked above each bar, and the number of genes or TEs with consistent changes in two or more mutants from the same pathway is labeled. E-F: Each of the genes (E) or TE families (F) that are DE in at least two CMT or RdDM mutants was used to perform hierarchical clustering using the Euclidean method and the log<sub>2</sub> of the fold-change relative to wild-type is visualized with a heat map.</p><div class="sb-div caption-clear"></div></div></div><p id="p-23">Transposable elements (TEs) comprise a large portion of the maize genome, and typically have high levels of CG and CHG methylation, with CHH methylation peaks at the edges of some TE families. There are two classes of TEs, Class I (retrotransposons) and Class II (DNA transposons), which transpose either through a copy-and-paste mechanism requiring an RNA intermediate (retrotransposons) or through a cut-and-paste mechanism (DNA transposons) (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Wicker <em>et al.</em> 2007</a>). Within each class are several orders divided into superfamilies, distinguished by structural and protein-coding features. Families within each superfamily are defined by sequence identity, and each family can contain many individual TE elements (<a id="xref-ref-30-4" class="xref-bibr" href="#ref-30">Jiao <em>et al.</em> 2017</a>). Individual TE elements are defined at a single location within a genome and are associated with a family, superfamily and class. We sought to document how perturbations to the methylome impacted expression of TEs. Due to the highly repetitive nature of TE sequences, we assessed per-family levels of expression by mapping RNA-seq reads to the genome, reporting up to 20 best hits for each read using Tophat2. Per-family read counts were determined by summing unique mapping reads (to a specific element) and multi-mapping reads that align to only a single TE family. Overall, the total portion of RNA-seq reads that map to TE families is not significantly higher in the mutants than in wild-type plants suggesting a lack of genome-wide activation of TEs in these mutants (Figure S2). In order to assess expression of individual TE families in each genotype, per-family expression was normalized by dividing the family counts by the total number of reads in the library assigned to either TE families or genes, generating a reads per million (RPM) estimate. Using this approach, we were able to detect expression of 1,694 TE families in at least one of the genotypes used for this study. A relatively small number of DE TE families (log<sub>2</sub>FC &gt; 1, FDR &lt; 0.05) were identified in each mutant (Table S3; <a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1D</a>). Consistent with the role of DNA methylation in silencing TEs, more families were identified as up-regulated rather than down-regulated in mutants compared with WT controls. However, the majority of the TE families expressed in these libraries do not exhibit significant changes in expression level in CMT or RdDM mutants in maize.</p><p id="p-24">There is a significant overlap in the number of genes and TEs that exhibit consistent changes in gene expression in at least two samples of CMT mutants (<em>zmet2-m1/zmet2-m2/zmet5-m1</em>) or RdDM mutants (<em>mop1-1/mop3-1</em>) (Figure S3). In order to understand the reproducible effects of these pathways on expression, we focused our analyses on the set of 237 genes and 104 TE families that exhibit consistent up- or down-regulation in multiple CMT or RdDM mutants. Hierarchical clustering of the fold-change of these genes or TE families relative to WT in all samples reveals a relatively consistent magnitude of expression changes between similar genotypes (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1E-F</a>). Within CMT mutants, 112 genes show shared expression changes between <em>zmet2-m1</em> and <em>zmet2-m2</em>, with a smaller subset of genes primarily shared between <em>zmet2-m2</em> and <em>zmet5-m1</em> (35 genes, <a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1E</a>). Although both <em>zmet2-m1</em> and <em>zmet2-m2</em> are predicted to encode loss of function alleles, there are genetic differences in the behavior of these alleles (<a id="xref-ref-50-4" class="xref-bibr" href="#ref-50">Papa <em>et al.</em> 2001</a>; <a id="xref-ref-43-4" class="xref-bibr" href="#ref-43">Makarevitch <em>et al.</em> 2007</a>; <a id="xref-ref-37-8" class="xref-bibr" href="#ref-37">Li <em>et al.</em> 2014</a>). The <em>zmet2-m1</em> mutation exhibits co-dominant effects on CHG methylation levels that may reflect dominant negative action of the protein that could be produced from this allele (<a id="xref-ref-50-5" class="xref-bibr" href="#ref-50">Papa <em>et al.</em> 2001</a>). Plants that are homozygous for <em>zmet2-m1</em> have the greatest loss of CHG methylation and this could result from influence of the ZMET2-M1 protein product of the <em>zmet2-m1</em> allele on functional ZMET5 protein.</p><p id="p-25">The <em>mop1-1</em> and <em>mop3-1</em> seedling leaf samples have a number of examples of consistent up-regulation (26 genes) but fewer examples of consistent down-regulation (7 genes), consistent with the greater number of up-regulated than down-regulated genes in RdDM mutants in general. An analysis of the RNA-seq data for <em>rmr6-1</em> (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Forestan <em>et al.</em> 2017</a>), which represents a distinct mutant allele at the <em>mop3</em> locus, identified very similar numbers of DE genes and similar overlaps with other RdDM mutants (Figure S3). A portion of the “genes” that are affected by CMT or RdDM mutations may represent mis-annotated transposons or pseudogenes. We assessed the proportion of genes with altered transcript abundance that are classified as syntenic based on a comparison with sorghum and rice genomes (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Brohammer <em>et al.</em> 2018</a>). The genes that are mis-regulated in RdDM or CMT mutants are often non-syntenic, but in all cases there are also a number of examples of syntenic genes (Figure S4). This suggests that are least a portion of the targets of CMT and RdDM represent potentially functional genes.</p></div><div id="sec-12" class="subsection"><h3>Some genes that are up-regulated in CMT mutants exhibit high CHG methylation levels</h3><p id="p-26">The differential expression observed in each mutant background could result from direct changes in DNA methylation or chromatin at these loci or could result from indirect effects due to secondary effects from genes that are direct targets. Genes that are direct targets for silencing by DNA methylation would be expected to have high levels of DNA methylation in wild-type plants. The context-specific DNA methylation profiles were assessed in wild-type B73 for genes that were up- or down- regulated compared with all expressed genes (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>). The genes that are differentially expressed in RdDM mutants do not show unusual patterns of DNA methylation. The methylation profile for genes that are down-regulated in CMT mutants are similar to all expressed genes. In contrast, genes that are up-regulated in the CMT mutants exhibit distinct patterns of CG and CHG methylation within gene bodies relative to other expressed genes (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2A</a>). Among the 112 genes up-regulated in multiple CMT mutants, approximately half have high (&gt;50%) and half have low (&lt;20%) methylation in the CG and CHG contexts in the region surrounding the transcription start site (TSS) (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2B-C</a>). In contrast, only ∼4% of all expressed genes have high CG and CHG methylation in the same region. While a small number of genes with high methylation overlap annotated TEs, most of the genes in this subset do not, suggesting that this genic methylation is not solely due to nearby TEs. In wild-type samples, genes with high CHG methylation in the gene body also have high CG methylation (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2D</a>). However, in <em>zmet2-m1</em> mutants, CHG methylation for these genes is reduced, with few examples having any corresponding reduction in CG methylation (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2D-E</a>). As in the examples (<em>Zm00001d045627</em> and <em>Zm00001d021982</em>) shown in <a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2F</a>, these genes have high levels of CG and CHG methylation in wild-type B73, and the reduction in CHG methylation in <em>zmet2-m1</em> mutants is associated with increased expression, suggesting that CMT-dependent silencing of these genes depends on CHG but not CG methylation.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/6/1921/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Methylation profiles of DE genes. A. The meta-profile of DNA methylation levels in wild-type B73 seedling leaf tissue was assessed for different sets of DE genes. The DNA methylation levels surrounding the TSS and TTS were plotted for all expressed genes and genes DE in CMT (blue) and RdDM (orange) mutants. The three panels show the levels of CG, CHG, and CHH methylation, with the y-axis showing DNA methylation levels. Error bars represent standard error. B-C Histogram of the average methylation level in the first 400 bp downstream of the TSS for genes up-regulated in CMT mutants in the CG (B) and CHG (C) contexts, showing a bimodal distribution of methylation values. D. Wild-type methylation levels in the CG and CHG contexts are correlated. zmet2-m1 mutant methylation data are shown for those genes with high (&gt;50%) CHG methylation in wild-type (orange dots), showing a loss of CHG but not CG methylation in the mutant. E. Histogram of the difference between mutant and WT methylation in the CG (red) and CHG (blue) contexts for those genes up-regulated in CMT mutants that have WT CHG methylation &gt;50%. F. IGV view of two up-regulated genes in CMT mutants: Zm00001d045627 and Zm00001d021982, which have high CG and CHG methylation in WT and reduced CHG methylation near the TSS (yellow box) in zmet2-m1 mutants. G. The proportion of genes within 2 kb of mCHH islands (mCHH) is shown for different sets of genes. The black bar shows the proportion of all genes with a mCHH island while the other bars show the proportion of genes with altered expression in specific mutant backgrounds that have mCHH islands, and * denotes significantly higher proportion than expected relative to all genes (p-value &lt; 0.01, chi squared test)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-395745604" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Methylation profiles of DE genes. A. The meta-profile of DNA methylation levels in wild-type B73 seedling leaf tissue was assessed for different sets of DE genes. The DNA methylation levels surrounding the TSS and TTS were plotted for all expressed genes and genes DE in CMT (blue) and RdDM (orange) mutants. The three panels show the levels of CG, CHG, and CHH methylation, with the y-axis showing DNA methylation levels. Error bars represent standard error. B-C Histogram of the average methylation level in the first 400 bp downstream of the TSS for genes up-regulated in CMT mutants in the CG (B) and CHG (C) contexts, showing a bimodal distribution of methylation values. D. Wild-type methylation levels in the CG and CHG contexts are correlated. &lt;em&gt;zmet2-m1&lt;/em&gt; mutant methylation data are shown for those genes with high (&gt;50%) CHG methylation in wild-type (orange dots), showing a loss of CHG but not CG methylation in the mutant. E. Histogram of the difference between mutant and WT methylation in the CG (red) and CHG (blue) contexts for those genes up-regulated in CMT mutants that have WT CHG methylation &gt;50%. F. IGV view of two up-regulated genes in CMT mutants: &lt;em&gt;Zm00001d045627&lt;/em&gt; and &lt;em&gt;Zm00001d021982&lt;/em&gt;, which have high CG and CHG methylation in WT and reduced CHG methylation near the TSS (yellow box) in &lt;em&gt;zmet2-m1&lt;/em&gt; mutants. G. The proportion of genes within 2 kb of mCHH islands (mCHH) is shown for different sets of genes. The black bar shows the proportion of all genes with a mCHH island while the other bars show the proportion of genes with altered expression in specific mutant backgrounds that have mCHH islands, and * denotes significantly higher proportion than expected relative to all genes (p-value &lt; 0.01, chi squared test).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/6/1921/F2.medium.gif" width="440" height="366"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/8/6/1921/F2.medium.gif" width="440" height="366"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/6/1921/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/6/1921/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/489871" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-27" class="first-child">Methylation profiles of DE genes. A. The meta-profile of DNA methylation levels in wild-type B73 seedling leaf tissue was assessed for different sets of DE genes. The DNA methylation levels surrounding the TSS and TTS were plotted for all expressed genes and genes DE in CMT (blue) and RdDM (orange) mutants. The three panels show the levels of CG, CHG, and CHH methylation, with the y-axis showing DNA methylation levels. Error bars represent standard error. B-C Histogram of the average methylation level in the first 400 bp downstream of the TSS for genes up-regulated in CMT mutants in the CG (B) and CHG (C) contexts, showing a bimodal distribution of methylation values. D. Wild-type methylation levels in the CG and CHG contexts are correlated. <em>zmet2-m1</em> mutant methylation data are shown for those genes with high (&gt;50%) CHG methylation in wild-type (orange dots), showing a loss of CHG but not CG methylation in the mutant. E. Histogram of the difference between mutant and WT methylation in the CG (red) and CHG (blue) contexts for those genes up-regulated in CMT mutants that have WT CHG methylation &gt;50%. F. IGV view of two up-regulated genes in CMT mutants: <em>Zm00001d045627</em> and <em>Zm00001d021982</em>, which have high CG and CHG methylation in WT and reduced CHG methylation near the TSS (yellow box) in <em>zmet2-m1</em> mutants. G. The proportion of genes within 2 kb of mCHH islands (mCHH) is shown for different sets of genes. The black bar shows the proportion of all genes with a mCHH island while the other bars show the proportion of genes with altered expression in specific mutant backgrounds that have mCHH islands, and * denotes significantly higher proportion than expected relative to all genes (p-value &lt; 0.01, chi squared test).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>Genes that are up-regulated in RdDM mutants are enriched for being near mCHH islands</h3><p id="p-28">A large number of maize genes (∼60%) have been associated with the presence of a region of elevated CHH in the promoter region, termed a mCHH island (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Gent <em>et al.</em> 2013</a>). Genes with mCHH islands are enriched for high expression and the mCHH island often occurs at the edge of the TE nearest these genes (<a id="xref-ref-38-4" class="xref-bibr" href="#ref-38">Li <em>et al.</em> 2015a</a>). These mCHH islands may form important boundaries that could protect TE heterochromatin from the influence of genes (<a id="xref-ref-38-5" class="xref-bibr" href="#ref-38">Li <em>et al.</em> 2015a</a>) and may also be important for long-distance chromatin interactions (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Rowley <em>et al.</em> 2017</a>). The methylation within these mCHH islands requires <em>Mop1</em> and <em>Mop3</em> (<a id="xref-ref-37-9" class="xref-bibr" href="#ref-37">Li <em>et al.</em> 2014</a>, <a id="xref-ref-38-6" class="xref-bibr" href="#ref-38">2015a</a>). Genes that are up- or down-regulated in the RdDM mutants are enriched for the presence of mCHH islands, but this is only significant for the up-regulated genes with 87.8% having a mCHH island within 2kb of the gene, compared to 64.6% of all expressed genes (<a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2G</a>, p-value &lt; 0.01, chi squared test). The observation that both RdDM up- and down-regulated genes are often near mCHH islands could be due to the fact that the mCHH island may provide long-range interactions (<a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">Rowley <em>et al.</em> 2017</a>) that could have either positive or negative influences on gene expression.</p><p id="p-29">In some cases, the mCHH island itself may result in transcriptional regulation. Work in <em>Arabidopsis</em> has noted a positive feedback loop involving DNA methylation levels and expression of the demethylase <em>ROS1</em> such that reduced levels of DNA methylation result in lower <em>ROS1</em> expression but increased methylation is associated with elevated <em>ROS1</em> expression (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Williams <em>et al.</em> 2015</a>). Reduced transcript abundance of maize DNA glycosylases has also been observed in several transcriptome datasets of maize RdDM mutants (<a id="xref-ref-59-2" class="xref-bibr" href="#ref-59">Williams <em>et al.</em> 2015</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Erhard <em>et al.</em> 2015</a>). We find that one maize gene with sequence homology to <em>ROS1</em>, <em>Zm00001d038302</em>, showed significantly reduced transcript abundance in the <em>mop1-1</em> and <em>mop3-1</em> mutants and has a strong mCHH island in several inbred lines (Figure S5). This provides evidence to support a requirement for RdDM and CHH methylation in the proper transcriptional control of this gene in maize.</p></div><div id="sec-14" class="subsection"><h3>Genes regulated by CMT are enriched for natural DMRs</h3><p id="p-30">There is a substantial amount of natural variation for methylation levels among maize inbreds. We sought to determine the extent to which genes that are sensitive to CMT or RdDM mutations also display variable methylation levels in these lines. These would reflect examples of potential cryptic information in one inbred line that is expressed in other genotypes, reflecting natural epigenetic variation. We used WGBS data from B73 and 17 other diverse maize inbreds to document natural variation for DNA methylation among maize inbreds. Differentially Methylated Regions (DMRs) were identified in all three contexts (CG, CHG, and CHH) between B73 and the other inbreds. More than 200,000 DMRs were called in the CG and CHG contexts, with over 50,000 DMRs in the CHH context. Each maize gene was classified based on whether there was a DMR within 200 bp up or downstream of the transcription start site for each of the three sequence contexts. A relatively small portion (∼3–7%) of maize genes has CG, CHG or CHH DMRs near the promoter (Table S2). We proceeded to assess whether naturally occurring DMRs were more prevalent near genes that exhibit altered expression in CMT or RdDM mutants (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>). Genes that are up- or down-regulated in CMT mutants exhibit a significant enrichment for CG and CHG DMRs in their promoter regions. RdDM up-regulated genes are significantly enriched for having CHH and CHG DMRs and also show an enrichment (though not significant) for CG DMRs near the promoter (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3A</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/6/1921/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Natural variation for methylation level. A. DMRs in diverse genotypes located within 200 bp of the TSS for all genes and for genes with differential expression in either CMT or RdDM mutants. Significant enrichment compared with expressed set is denoted with * (p-value &lt; 0.01, chi squared test). B-C. Examples of genes up-regulated in CMT that have negatively correlated expression and CHG methylation at the bin overlapping the TSS. Data points show values for B73 (red), zmet2-m1 mutants (blue), and 9 diverse genotypes (B97, CML322, HP301, IL14, Mo17, Oh43, P39, Tx303, and W22, black)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-395745604" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Natural variation for methylation level. A. DMRs in diverse genotypes located within 200 bp of the TSS for all genes and for genes with differential expression in either CMT or RdDM mutants. Significant enrichment compared with expressed set is denoted with * (p-value &lt; 0.01, chi squared test). B-C. Examples of genes up-regulated in CMT that have negatively correlated expression and CHG methylation at the bin overlapping the TSS. Data points show values for B73 (red), &lt;em&gt;zmet2-m1&lt;/em&gt; mutants (blue), and 9 diverse genotypes (B97, CML322, HP301, IL14, Mo17, Oh43, P39, Tx303, and W22, black).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/6/1921/F3.medium.gif" width="440" height="427"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/8/6/1921/F3.medium.gif" width="440" height="427"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/6/1921/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/6/1921/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/489950" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-31" class="first-child">Natural variation for methylation level. A. DMRs in diverse genotypes located within 200 bp of the TSS for all genes and for genes with differential expression in either CMT or RdDM mutants. Significant enrichment compared with expressed set is denoted with * (p-value &lt; 0.01, chi squared test). B-C. Examples of genes up-regulated in CMT that have negatively correlated expression and CHG methylation at the bin overlapping the TSS. Data points show values for B73 (red), <em>zmet2-m1</em> mutants (blue), and 9 diverse genotypes (B97, CML322, HP301, IL14, Mo17, Oh43, P39, Tx303, and W22, black).</p><div class="sb-div caption-clear"></div></div></div><p id="p-32">This suggests that many of the genes with altered transcript abundance in CMT or RdDM mutations may have pre-existing natural variation for DNA methylation that would affect expression levels in maize populations. RNA-seq data from leaf tissue for ten of the inbred lines with WGBS data were utilized to determine whether there was a significant association (p-value &lt; 0.05, Pearson correlation) between context-specific methylation level at the DMR and transcript abundance levels. We found that nearly 50% of the genes that exhibit altered expression in CMT or RdDM mutants that are located near DMRs had natural variation for gene expression levels that was significantly associated with DNA methylation levels. Two examples of CMT up-regulated genes that exhibit significantly correlated transcript abundance and CHG methylation at the bin overlapping the TSS among diverse lines are shown in <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3B-C</a>. In wild-type maize inbred lines we see two classes with respect to transcript abundance and CHG methylation levels at the DMR near the TSS. In one group of lines, including B73 (red dots), the DMR is highly methylated and the gene is transcriptionally silent. In the other group of genotypes (and in B73 <em>zmet2-m1</em> mutant lines - blue dots) the DMR has low methylation and transcripts are observed.</p></div><div id="sec-15" class="subsection"><h3>Properties of TEs with altered expression</h3><p id="p-33">There are 104 TE families with altered expression in mutants that perturb RdDM or CMT components in maize. These included 32 families up- and 7 families down-regulated in at least two contrasts of RdDM mutants, and 48 families up- and 24 families down-regulated in at least two contrasts of CMT mutants (<a id="xref-fig-1-8" class="xref-fig" href="#F1">Figure 1D</a>). There are examples of both class I (specifically Long Terminal Repeat or LTR) and class II (specifically Terminal Inverted Repeat or TIR) TE families that exhibit altered expression in both RdDM and CMT mutants (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>). Most families with varied expression were small (&lt; 10 members), consistent with the genome-wide distribution (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4B</a>). The analysis of the relative age of LTR families, approximated by comparing the sequence similarity of the two LTR sequences of an element, that have altered expression reveals that LTR elements up-regulated in RdDM mutants are enriched for younger LTR elements when compared with the distribution of ages present genome-wide (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4C</a>). The TEs that exhibit increased transcript abundance in RdDM mutants tend to be located further from genes than TEs that are expressed in CMT mutants or all expressed TEs (Figure S6; p-value &lt; 0.01, <em>t</em>-test). We also tested the mean GC content of TEs within families to test whether families depleted in cytosines are more susceptible to subtle perturbations in methylation, as is the case for the <em>ONSEN</em> family in <em>Arabidopsis</em> (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Cavrak <em>et al.</em> 2014</a>). TE families up-regulated in RdDM mutants do have a slightly lower GC content on average, though it is not clear if this change alone is sufficient to cause the expression changes (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4D</a>, p-value &lt; 0.01, <em>t</em>-test).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/6/1921/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Attributes of TE families with altered expression in methylation mutants. A. TE super family membership for TE families genome-wide and with varied expression in mutants, where RST = SINE, RIT = LINE, RIL = LINE-L1-like, DTX = TIR-unclassified, DTT = TIR-Tc1/Mariner, DTH = TIR-PIF/Harbinger, DTC = CACTA, RLX = LTR-unclassified, RLG = LTR-Gypsy, RLC = LTR-Copia, DTM = TIR-Mu, and DTA = TIR-hAT. B. Size distribution for TE families genome-wide and with varied expression in mutants, where small: 2-9 members, medium: 10-99 members, and large: &gt;= 100 members. C. Boxplot of the average LTR similarity per-family for LTR TE families genome-wide, along with those LTR families with differential expression in methylation mutants. D. Boxplot of the average GC content per-family for TE families genome-wide and with expression changes in the mutant. * denotes significant deviation from the mean for all TE families (p-value &lt; 0.01, t-test)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-395745604" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Attributes of TE families with altered expression in methylation mutants. A. TE super family membership for TE families genome-wide and with varied expression in mutants, where RST = SINE, RIT = LINE, RIL = LINE-L1-like, DTX = TIR-unclassified, DTT = TIR-Tc1/Mariner, DTH = TIR-PIF/Harbinger, DTC = CACTA, RLX = LTR-unclassified, RLG = LTR-Gypsy, RLC = LTR-Copia, DTM = TIR-Mu, and DTA = TIR-hAT. B. Size distribution for TE families genome-wide and with varied expression in mutants, where small: 2-9 members, medium: 10-99 members, and large: &gt;= 100 members. C. Boxplot of the average LTR similarity per-family for LTR TE families genome-wide, along with those LTR families with differential expression in methylation mutants. D. Boxplot of the average GC content per-family for TE families genome-wide and with expression changes in the mutant. * denotes significant deviation from the mean for all TE families (p-value &lt; 0.01, &lt;em&gt;t&lt;/em&gt;-test).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/6/1921/F4.medium.gif" width="440" height="207"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/8/6/1921/F4.medium.gif" width="440" height="207"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/6/1921/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/6/1921/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490011" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-34" class="first-child">Attributes of TE families with altered expression in methylation mutants. A. TE super family membership for TE families genome-wide and with varied expression in mutants, where RST = SINE, RIT = LINE, RIL = LINE-L1-like, DTX = TIR-unclassified, DTT = TIR-Tc1/Mariner, DTH = TIR-PIF/Harbinger, DTC = CACTA, RLX = LTR-unclassified, RLG = LTR-Gypsy, RLC = LTR-Copia, DTM = TIR-Mu, and DTA = TIR-hAT. B. Size distribution for TE families genome-wide and with varied expression in mutants, where small: 2-9 members, medium: 10-99 members, and large: &gt;= 100 members. C. Boxplot of the average LTR similarity per-family for LTR TE families genome-wide, along with those LTR families with differential expression in methylation mutants. D. Boxplot of the average GC content per-family for TE families genome-wide and with expression changes in the mutant. * denotes significant deviation from the mean for all TE families (p-value &lt; 0.01, <em>t</em>-test).</p><div class="sb-div caption-clear"></div></div></div><p id="p-35">We sought to further document the properties of these TE families through analysis of their transcript abundance in nearly 100 developmental tissues or stages of B73. During typical development, approximately 3,400 TE families are expressed in at least one tissue or stage. There are 5 TE families up-regulated in RdDM mutants and 18 TE families up-regulated in CMT mutants that are not expressed in any tissue or developmental stage assessed. The other families of TEs that are up-regulated in RdDM mutants (25 families) or CMT mutants (27 families) were assessed to determine if they exhibit distinct patterns of expression. Interestingly, approximately one third of the TE families that are up-regulated in RdDM mutants show higher expression in the endosperm than other tissues (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5A</a>). In contrast, the TE families up-regulated in CMT mutants do not show any evidence for higher expression in a particular tissue. The enrichment for endosperm expression in TE families up-regulated in RdDM mutants does not extend to genes up-regulated in the mutants and cannot be simply attributed to lower expression of the <em>Mop1</em> and <em>Mop3</em> genes in these tissues (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>). This result highlights the potential for some TEs to escape RdDM-based silencing in endosperm, where dynamic changes to DNA methylation may reinforce TE silencing in the embryo (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Martínez and Slotkin 2012</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Gehring 2013</a>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Wang <em>et al.</em> 2015</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Dong <em>et al.</em> 2017</a>). Meanwhile, both genes and TE families susceptible to mis-regulation in CMT mutants are less often expressed across development, consistent with the greater developmental stability of CHG methylation over CHH methylation (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Kawakatsu <em>et al.</em> 2016</a>, <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">2017</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Narsai <em>et al.</em> 2017</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bouyer <em>et al.</em> 2017</a>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/6/1921/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Developmental expression of TE families (A) and genes (B) up-regulated in RdDM and CMT mutants, along with the typical expression of genes mutated in this study (C), where rows show TE families or genes and columns show RNA-seq libraries. Developmental samples are grouped by tissue type, with seed samples split into two clusters based on relative contribution of endosperm: endosperm &amp; late seed (12+ days after pollination) and embryo &amp; early seed (up to 10 days after pollination). For full list of tissue assignments and RNA-seq library accession numbers, see Table S1. Approximately one third of TE families up-regulated in RdDM mutants have higher expression in endosperm than other tissues across development, a pattern not observed for genes. In contrast, many more TE families and genes up-regulated in CMT mutants are never or lowly expressed during typical development." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-395745604" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Developmental expression of TE families (A) and genes (B) up-regulated in RdDM and CMT mutants, along with the typical expression of genes mutated in this study (C), where rows show TE families or genes and columns show RNA-seq libraries. Developmental samples are grouped by tissue type, with seed samples split into two clusters based on relative contribution of endosperm: endosperm &amp; late seed (12+ days after pollination) and embryo &amp; early seed (up to 10 days after pollination). For full list of tissue assignments and RNA-seq library accession numbers, see Table S1. Approximately one third of TE families up-regulated in RdDM mutants have higher expression in endosperm than other tissues across development, a pattern not observed for genes. In contrast, many more TE families and genes up-regulated in CMT mutants are never or lowly expressed during typical development.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/6/1921/F5.medium.gif" width="372" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/8/6/1921/F5.medium.gif" width="372" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/6/1921/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/6/1921/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/489985" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-36" class="first-child">Developmental expression of TE families (A) and genes (B) up-regulated in RdDM and CMT mutants, along with the typical expression of genes mutated in this study (C), where rows show TE families or genes and columns show RNA-seq libraries. Developmental samples are grouped by tissue type, with seed samples split into two clusters based on relative contribution of endosperm: endosperm &amp; late seed (12+ days after pollination) and embryo &amp; early seed (up to 10 days after pollination). For full list of tissue assignments and RNA-seq library accession numbers, see Table S1. Approximately one third of TE families up-regulated in RdDM mutants have higher expression in endosperm than other tissues across development, a pattern not observed for genes. In contrast, many more TE families and genes up-regulated in CMT mutants are never or lowly expressed during typical development.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-16" class="subsection"><h3>Evidence for locus-specific and coordinated changes in expression of TEs</h3><p id="p-37">While per-family analysis of TE expression is useful in capturing expression dynamics of repetitive transposable elements, the expression of individual elements can be influenced by a variety of location-specific attributes such as methylation levels and proximity to genes as well as family-level attributes such as binding motifs and nucleotide content. We were interested in documenting the relative behavior of different elements within the same family to understand whether the changes in transcript abundance of TEs were occurring in an element-specific or family-wide manner. Coordinate changes in transcript abundance could indicate the importance of RdDM or CMT for family-wide regulation while element-specific changes could reflect influences at particular loci. A set of TE families with &lt;10 elements that had altered transcript abundance and for which at least 50% of the reads could be uniquely assigned to specific element were identified and used for analysis of coordinate <em>vs.</em> locus-specific expression (Tables S3, S4). The unique mapping reads for these families were used to evaluate element-specific transcript abundance. Half of the testable TE families had expression of a single member of the family indicating locus-specific changes (examples in <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6A, C</a>). In the other half of the TE families there was evidence for expression changes for multiple elements of the same family (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6B, D</a>). This suggests at least some level of coordinate regulation of multiple members of the family by CMT or RdDM pathways. However, even in examples of coordinate expression a single element accounted for the vast majority of unique reads mapping to the family. Examples of both locus-specific and coordinate changes in expression for both CMT and RdDM mutants were found but we were not able to assess enough families to determine if there was any enrichment for the type of regulation for these two silencing pathways.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/6/1921/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="TE families up-regulated in methylation mutants can have expression of a single element or multiple elements. A-B show two LTR families up-regulated in RdDM mutants, and C-D show two LTR families up-regulated in CMT mutants. All families have both unique (red) and multi-mapped (blue) reads. The proportion of unique-mapping reads assigned to each element is shown for a representative library. Unique-mapping reads showed confident expression of only a single member of a family (A and C) or coordinated expression of more than one member of a family (B and D)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-395745604" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;TE families up-regulated in methylation mutants can have expression of a single element or multiple elements. A-B show two LTR families up-regulated in RdDM mutants, and C-D show two LTR families up-regulated in CMT mutants. All families have both unique (red) and multi-mapped (blue) reads. The proportion of unique-mapping reads assigned to each element is shown for a representative library. Unique-mapping reads showed confident expression of only a single member of a family (A and C) or coordinated expression of more than one member of a family (B and D).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/6/1921/F6.medium.gif" width="440" height="296"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/8/6/1921/F6.medium.gif" width="440" height="296"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/6/1921/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/6/1921/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/489908" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-38" class="first-child">TE families up-regulated in methylation mutants can have expression of a single element or multiple elements. A-B show two LTR families up-regulated in RdDM mutants, and C-D show two LTR families up-regulated in CMT mutants. All families have both unique (red) and multi-mapped (blue) reads. The proportion of unique-mapping reads assigned to each element is shown for a representative library. Unique-mapping reads showed confident expression of only a single member of a family (A and C) or coordinated expression of more than one member of a family (B and D).</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-17"><h2 class="">Discussion</h2><p id="p-39">Maize has been a model system for the discovery of several epigenetic phenomena such as imprinting (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Kermicle 1970</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Kermicle and Alleman 1990</a>), paramutation (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Brink 1956</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Chandler 2007</a>; <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Hollick 2017</a>) and transposon silencing (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Chandler and Walbot 1986</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Chomet <em>et al.</em> 1987</a>). An unresolved question is whether epigenetic regulation plays important roles in quantitative trait variation beyond handful of well characterized loci. Our ability to document the full role for epigenetic regulation and DNA methylation has been limited by our inability to recover plants with major reductions in the level of DNA methylation (<a id="xref-ref-37-10" class="xref-bibr" href="#ref-37">Li <em>et al.</em> 2014</a>). Forward genetic screens have uncovered a number of components of the RNA-directed DNA methylation (RdDM) machinery as playing critical roles in maintenance of silenced paramutant states (<a id="xref-ref-13-6" class="xref-bibr" href="#ref-13">Dorweiler <em>et al.</em> 2000</a>; <a id="xref-ref-16-5" class="xref-bibr" href="#ref-16">Erhard <em>et al.</em> 2009</a>) or transgene silencing (<a id="xref-ref-46-3" class="xref-bibr" href="#ref-46">McGinnis <em>et al.</em> 2006</a>). These mutants have substantial effects on CHH methylation in maize but have minimal effect on genome-wide levels of CG or CHG methylation (<a id="xref-ref-37-11" class="xref-bibr" href="#ref-37">Li <em>et al.</em> 2014</a>). Reverse-genetic analyses have identified loss-of-function alleles for a number of other genes predicted to play important roles in DNA methylation but the only single mutants with significant effects on genome-wide DNA methylation are the CMT genes of maize, <em>zmet2</em> and <em>zmet5</em> (<a id="xref-ref-37-12" class="xref-bibr" href="#ref-37">Li <em>et al.</em> 2014</a>). In this study we have documented how these subtle perturbations of the maize methylome affect the transcriptome in order to find genes subject to epigenetic regulation.</p><p id="p-40">The effects of mutations in RdDM or CMT genes in maize are quite limited. Our evidence suggests that there is little effect on the overall transcriptome of these plants. This might be expected given the limited effect on overall plant phenotype for each of these mutations. A recent study found that <em>rmr6-1</em> (allelic to <em>mop3-1</em>) mutants exhibited transcription changes from a larger portion of the genome but much of this was associated with increased transcriptional ‘noise’ at lowly expressed regions (<a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">Forestan <em>et al.</em> 2017</a>). Reanalysis of gene and TE family expression using our methods in <em>rmr6-1</em> mutant and wild-type libraries revealed that the number of differentially expressed genes and TEs is quite similar to the mutants used in our study (Figure S3). Many of the genes or TEs with altered expression in <em>rmr6-1</em> also exhibit altered expression in <em>mop3-1</em> and/or <em>mop1-1</em> (Figure S3). While relatively few genes exhibit major changes in expression there are a significant number of genes that exhibit similar expression changes in multiple RdDM or CMT mutants. These findings are compatible with the concept that there are a small number of genes in the maize genome that have epigenetic regulation that is solely dependent upon RdDM or CMT mediated regulation. While some of these “genes” lack synteny with other related species others do show syntenic genomic positions and likely represent potentially functional genes (Figure S4). It is likely that a much larger number of genes are redundantly regulated by the RdDM and CMT pathway along with MET1 mediated CG methylation.</p><p id="p-41">Many of the genes that are up-regulated in CMT mutants exhibit high levels of CHG methylation. The CMT mutants reduce this methylation and may result in increased transcript abundance. Previous studies noted that the genes sensitive to <em>zmet2-m1</em> mutations varied in different maize inbreds (<a id="xref-ref-43-5" class="xref-bibr" href="#ref-43">Makarevitch <em>et al.</em> 2007</a>). This prompted us to investigate whether the genes that are up-regulated in CMT mutants might exhibit natural variation for DNA methylation levels. Many of the genes that are up-regulated in CMT mutants have CHG DMRs nearby and many of these exhibit variable levels of expression among maize genotypes that is negatively correlated with CHG methylation levels. This suggests epiallelic diversity for targets of CMT-mediated gene silencing. If these changes in expression lead to phenotypic variation, plant breeders are likely able to select for preferred epigenetic states. However, it would also be possible to introduce novel epigenetic variation through reductions of CHG methylation.</p><p id="p-42">DNA methylation is often considered to play a primary role in maintaining genome integrity by silencing transposable elements. Indeed, there are clear examples of release of transposon silencing in mutants affecting DNA methylation in <em>Arabidopsis</em> (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Miura <em>et al.</em> 2001</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Mirouze <em>et al.</em> 2009</a>) and maize (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Lisch <em>et al.</em> 2002</a>; <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Jia <em>et al.</em> 2009</a>). However, generating a complete understanding of transposon expression is complicated by the highly repetitive nature of transposable elements. In order to survey expression using RNA-seq most researchers focus on unique mapping reads to ensure that expression is accurately attributed to the proper genomic locus. In this study we elected to primarily focus on TE families rather than individual elements and we utilized an approach that allowed for the combined use of unique and multiple-mapping reads to assess TE family expression. We did find evidence that a number of TE families require RdDM and/or CMT for silencing. There were few distinguishing features about these TEs relative to others making it unclear why the silencing of these families was easily released in these mutants. For several families we were able to document evidence for specific release of silencing of a single member of the family while in other cases we found that multiple members of the same family were reactivated.</p><p id="p-43">This study defines a set of genes and TE families that are regulated by DNA methylation. The silencing of these genes and TEs relies solely upon RdDM or CMT based epigenetic regulation. These loci provide important insights into the mechanisms that allow for epigenetic regulation and the natural variation for epigenetic regulation in maize.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-44">We thank Peter Hermanson and Jonathan Giesler for technical assistance and Jonathan Gent for helpful discussions and feedback. Michelle Stitzer and Jeffrey Ross-Ibarra provided helpful discussions and transposon resources. Funding for this work was grants from USDA-NIFA2016-67013-24747 to N.M.S. and the National Science Foundation IOS-1237931 to N.M.S. and M.W.V. Q.L. was supported by The National Key Research and Development Program of China (2016YFD0101003).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-45">Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.6071000">https://doi.org/10.25387/g3.6071000</a>.</p></li><li class="fn" id="fn-2"><p id="p-46"><em>Communicating editor: J. Birchler</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2018-01-26"><span class="received-label">Received </span>January 26, 2018.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2018-04-03"><span class="accepted-label">Accepted </span>April 3, 2018.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2018 Anderson <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-8.6.1921.1" data-doi="10.1038/nature04884"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alleman</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sidorenko</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McGinnis</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seshadri</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dorweiler</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">An RNA-dependent RNA polymerase is required for paramutation in maize.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">442</span>(<span class="cit-issue">7100</span>): <span class="cit-fpage">295</span>–<span class="cit-lpage">298</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature04884</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DAlleman%26rft.auinit1%253DM.%26rft.volume%253D442%26rft.issue%253D7100%26rft.spage%253D295%26rft.epage%253D298%26rft.atitle%253DAn%2BRNA-dependent%2BRNA%2Bpolymerase%2Bis%2Brequired%2Bfor%2Bparamutation%2Bin%2Bmaize.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature04884%26rft_id%253Dinfo%253Apmid%252F16855589%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature04884&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16855589&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F6%2F1921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000239122100037&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-8.6.1921.2" data-doi="10.1093/bioinformatics/btu638"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Anders</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pyl</span>  <span class="cit-name-given-names">P. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huber</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">HTSeq–a Python framework to work with high-throughput sequencing data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">31</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">166</span>–<span class="cit-lpage">169</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btu638</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu638%26rft_id%253Dinfo%253Apmid%252F25260700%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu638&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25260700&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F6%2F1921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000347832300003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-8.6.1921.3" data-doi="10.1073/pnas.1202073109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barber</span>  <span class="cit-name-given-names">W. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Win</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Varala</span>  <span class="cit-name-given-names">K. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dorweiler</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Repeat associated small RNAs vary among parents and following hybridization in maize.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">109</span>(<span class="cit-issue">26</span>): <span class="cit-fpage">10444</span>–<span class="cit-lpage">10449</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1202073109</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1202073109%26rft_id%253Dinfo%253Apmid%252F22689990%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA5LzI2LzEwNDQ0IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC82LzE5MjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-8.6.1921.4" data-doi="10.1186/s13059-017-1195-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bewick</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Niederhuth</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ji</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rohr</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Griffin</span>  <span class="cit-name-given-names">P. T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">The evolution of CHROMOMETHYLASES and gene body DNA methylation in plants.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">18</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">65</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s13059-017-1195-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D18%26rft.spage%253D65%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-017-1195-1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-017-1195-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-8.6.1921.5" data-doi="10.1186/s13059-017-1313-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bouyer</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kramdi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kassam</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heese</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schnittger</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">DNA methylation dynamics during early plant life.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">18</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">179</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s13059-017-1313-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D18%26rft.spage%253D179%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-017-1313-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-017-1313-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-8.6.1921.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brink</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1956</span> <span class="cit-article-title">A Genetic Change Associated with the R Locus in Maize Which Is Directed and Potentially Reversible.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">872</span>–<span class="cit-lpage">889</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DBrink%26rft.auinit1%253DR.%2BA.%26rft.volume%253D41%26rft.issue%253D6%26rft.spage%253D872%26rft.epage%253D889%26rft.atitle%253DA%2BGENETIC%2BCHANGE%2BASSOCIATED%2BWITH%2BTHE%2BR%2BLOCUS%2BIN%2BMAIZE%2BWHICH%2BIS%2BDIRECTED%2BAND%2BPOTENTIALLY%2BREVERSIBLE%26rft_id%253Dinfo%253Apmid%252F17247669%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo4OiI0MS82Lzg3MiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzgvNi8xOTIxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-8.6.1921.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brohammer</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kono</span>  <span class="cit-name-given-names">T. J. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Springer</span>  <span class="cit-name-given-names">N. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McGaugh</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hirsch</span>  <span class="cit-name-given-names">C. N.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">The limited role of differential fractionation in genome content variation and function in maize (<em>Zea mays L</em>.) inbred lines.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">131</span>–<span class="cit-lpage">141</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BJ.%26rft.volume%253D93%26rft.spage%253D131%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-8.6.1921.8" data-doi="10.1371/journal.pgen.1004115"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cavrak</span>  <span class="cit-name-given-names">V. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lettner</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jamge</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kosarewicz</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bayer</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">How a retrotransposon exploits the plant’s heat stress response for its activation.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">10</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">e1004115</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1004115</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D10%26rft.spage%253De1004115%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1004115%26rft_id%253Dinfo%253Apmid%252F24497839%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1004115&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24497839&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F6%2F1921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-8.6.1921.9" data-doi="10.1016/j.cell.2007.02.007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chandler</span>  <span class="cit-name-given-names">V. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Paramutation: from maize to mice.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">128</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">641</span>–<span class="cit-lpage">645</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.cell.2007.02.007</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DChandler%26rft.auinit1%253DV.%2BL.%26rft.volume%253D128%26rft.issue%253D4%26rft.spage%253D641%26rft.epage%253D645%26rft.atitle%253DParamutation%253A%2Bfrom%2Bmaize%2Bto%2Bmice.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2007.02.007%26rft_id%253Dinfo%253Apmid%252F17320501%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2007.02.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17320501&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F6%2F1921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000245098600006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-8.6.1921.10" data-doi="10.1073/pnas.83.6.1767"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chandler</span>  <span class="cit-name-given-names">V. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walbot</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span> <span class="cit-article-title">DNA modification of a maize transposable element correlates with loss of activity.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">83</span>(<span class="cit-issue">6</span>): <span class="cit-fpage">1767</span>–<span class="cit-lpage">1771</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.83.6.1767</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DChandler%26rft.auinit1%253DV.%2BL.%26rft.volume%253D83%26rft.issue%253D6%26rft.spage%253D1767%26rft.epage%253D1771%26rft.atitle%253DDNA%2Bmodification%2Bof%2Ba%2Bmaize%2Btransposable%2Belement%2Bcorrelates%2Bwith%2Bloss%2Bof%2Bactivity.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.83.6.1767%26rft_id%253Dinfo%253Apmid%252F3006070%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI4My82LzE3NjciO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzYvMTkyMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-8.6.1921.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chomet</span>  <span class="cit-name-given-names">P. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wessler</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dellaporta</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">Inactivation of the maize transposable element Activator (Ac) is associated with its DNA modification.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">295</span>–<span class="cit-lpage">302</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEMBO%2BJ.%26rft.volume%253D6%26rft.spage%253D295%26rft_id%253Dinfo%253Apmid%252F3034583%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=3034583&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F6%2F1921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-8.6.1921.12" data-doi="10.1016/j.molp.2016.10.007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dong</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peng</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Dynamic and Antagonistic Allele-Specific Epigenetic Modifications Controlling the Expression of Imprinted Genes in Maize Endosperm.</span> <abbr class="cit-jnl-abbrev">Mol. Plant</abbr> <span class="cit-vol">10</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">442</span>–<span class="cit-lpage">455</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.molp.2016.10.007</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BPlant%26rft.volume%253D10%26rft.spage%253D442%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molp.2016.10.007%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molp.2016.10.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-8.6.1921.13" data-doi="10.1105/tpc.12.11.2101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dorweiler</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carey</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kubo</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hollick</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kermicle</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Mediator of Paramutation1 is Required for Establishment and Maintenance of Paramutation at Multiple Maize Loci.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">12</span>(<span class="cit-issue">11</span>): <span class="cit-fpage">2101</span>–<span class="cit-lpage">2118</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1105/tpc.12.11.2101</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2BCell%2BOnline%26rft.stitle%253DPlant%2BCell%26rft.aulast%253DDorweiler%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D12%26rft.issue%253D11%26rft.spage%253D2101%26rft.epage%253D2118%26rft.atitle%253Dmediator%2Bof%2Bparamutation1%2BIs%2BRequired%2Bfor%2BEstablishment%2Band%2BMaintenance%2Bof%2BParamutation%2Bat%2BMultiple%2BMaize%2BLoci%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.12.11.2101%26rft_id%253Dinfo%253Apmid%252F11090212%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjEwOiIxMi8xMS8yMTAxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC82LzE5MjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-8.6.1921.14" data-doi="10.1105/tpc.113.114793"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eichten</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Briskine</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Swanson-Wagner</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Epigenetic and genetic influences on DNA methylation variation in maize populations.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">25</span>(<span class="cit-issue">8</span>): <span class="cit-fpage">2783</span>–<span class="cit-lpage">2797</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1105/tpc.113.114793</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.113.114793%26rft_id%253Dinfo%253Apmid%252F23922207%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjI1LzgvMjc4MyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzgvNi8xOTIxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-8.6.1921.15" data-doi="10.1371/journal.pgen.1002372"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eichten</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Swanson-Wagner</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schnable</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waters</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hermanson</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Heritable epigenetic variation among maize inbreds.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">7</span>(<span class="cit-issue">11</span>): <span class="cit-fpage">e1002372</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1002372</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DEichten%26rft.auinit1%253DS.%2BR.%26rft.volume%253D7%26rft.issue%253D11%26rft.spage%253De1002372%26rft.epage%253De1002372%26rft.atitle%253DHeritable%2Bepigenetic%2Bvariation%2Bamong%2Bmaize%2Binbreds.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002372%26rft_id%253Dinfo%253Apmid%252F22125494%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002372&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22125494&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F6%2F1921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-8.6.1921.16" data-doi="10.1126/science.1164508"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Erhard</span>  <span class="cit-name-given-names">K. F.</span>  <span class="cit-name-suffix">Jr</span>.</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stonaker</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parkinson</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lim</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hale</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">RNA polymerase IV functions in paramutation in Zea mays.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">323</span>(<span class="cit-issue">5918</span>): <span class="cit-fpage">1201</span>–<span class="cit-lpage">1205</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1164508</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DErhard%26rft.auinit1%253DK.%2BF.%26rft.volume%253D323%26rft.issue%253D5918%26rft.spage%253D1201%26rft.epage%253D1205%26rft.atitle%253DRNA%2BPolymerase%2BIV%2BFunctions%2Bin%2BParamutation%2Bin%2BZea%2Bmays%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1164508%26rft_id%253Dinfo%253Apmid%252F19251626%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMjMvNTkxOC8xMjAxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC82LzE5MjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-8.6.1921.17" data-doi="10.1534/genetics.115.174714"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Erhard</span>  <span class="cit-name-given-names">K. F.</span>  <span class="cit-name-suffix">Jr</span>.</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Talbot</span>  <span class="cit-name-given-names">J.-E. R. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deans</span>  <span class="cit-name-given-names">N. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McClish</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hollick</span>  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Nascent transcription affected by RNA polymerase IV in Zea mays.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">199</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">1107</span>–<span class="cit-lpage">1125</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.115.174714</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.174714%26rft_id%253Dinfo%253Apmid%252F25653306%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5OS80LzExMDciO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzYvMTkyMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-8.6.1921.18" data-doi="10.1038/srep30446"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Forestan</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aiese Cigliano</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Farinati</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lunardon</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sanseverino</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Stress-induced and epigenetic-mediated maize transcriptome regulation study by means of transcriptome reannotation and differential expression analysis.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">6</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">30446</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/srep30446</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D6%26rft.spage%253D30446%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep30446%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/srep30446&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-8.6.1921.19" data-doi="10.1186/s12870-017-1108-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Forestan</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Farinati</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aiese Cigliano</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lunardon</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sanseverino</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Maize RNA PolIV affects the expression of genes with nearby TE insertions and has a genome-wide repressive impact on transcription.</span> <abbr class="cit-jnl-abbrev">BMC Plant Biol.</abbr> <span class="cit-vol">17</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">161</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12870-017-1108-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BPlant%2BBiol.%26rft.volume%253D17%26rft.spage%253D161%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12870-017-1108-1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12870-017-1108-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-8.6.1921.20" data-doi="10.1146/annurev-genet-110711-155527"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gehring</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomic imprinting: insights from plants.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genet.</abbr> <span class="cit-vol">47</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">187</span>–<span class="cit-lpage">208</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1146/annurev-genet-110711-155527</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRev.%2BGenet.%26rft.volume%253D47%26rft.spage%253D187%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-genet-110711-155527%26rft_id%253Dinfo%253Apmid%252F24016190%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-genet-110711-155527&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24016190&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F6%2F1921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000329816000009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-8.6.1921.21" data-doi="10.1101/gr.146985.112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gent</span>  <span class="cit-name-given-names">J. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ellis</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harkess</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">CHH islands: de novo DNA methylation in near-gene chromatin regulation in maize.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">23</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">628</span>–<span class="cit-lpage">637</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.146985.112</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.146985.112%26rft_id%253Dinfo%253Apmid%252F23269663%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIzLzQvNjI4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC82LzE5MjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-8.6.1921.22" data-doi="10.1105/tpc.114.130427"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gent</span>  <span class="cit-name-given-names">J. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Madzima</span>  <span class="cit-name-given-names">T. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bader</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kent</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Accessible DNA and relative depletion of H3K9me2 at maize loci undergoing RNA-directed DNA methylation.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">26</span>(<span class="cit-issue">12</span>): <span class="cit-fpage">4903</span>–<span class="cit-lpage">4917</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1105/tpc.114.130427</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.114.130427%26rft_id%253Dinfo%253Apmid%252F25465407%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjEwOiIyNi8xMi80OTAzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC82LzE5MjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-8.6.1921.23" data-doi="10.1371/journal.pgen.1006526"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gouil</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baulcombe</span>  <span class="cit-name-given-names">D. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">DNA Methylation Signatures of the Plant Chromomethyltransferases.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">12</span>(<span class="cit-issue">12</span>): <span class="cit-fpage">e1006526</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1006526</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D12%26rft.spage%253De1006526%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1006526%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1006526&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-8.6.1921.24" data-doi="10.1371/journal.pbio.0050275"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hale</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stonaker</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gross</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hollick</span>  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A novel Snf2 protein maintains trans-generational regulatory states established by paramutation in maize.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">5</span>(<span class="cit-issue">10</span>): <span class="cit-fpage">e275</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pbio.0050275</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DHale%26rft.auinit1%253DC.%2BJ.%26rft.volume%253D5%26rft.issue%253D10%26rft.spage%253De275%26rft.epage%253De275%26rft.atitle%253DA%2Bnovel%2BSnf2%2Bprotein%2Bmaintains%2Btrans-generational%2Bregulatory%2Bstates%2Bestablished%2Bby%2Bparamutation%2Bin%2Bmaize.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0050275%26rft_id%253Dinfo%253Apmid%252F17941719%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0050275&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17941719&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F6%2F1921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251072700007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-8.6.1921.25" data-doi="10.1186/s13059-017-1288-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hofmeister</span>  <span class="cit-name-given-names">B. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rohr</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schmitz</span>  <span class="cit-name-given-names">R. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Stable inheritance of DNA methylation allows creation of epigenotype maps and the study of epiallele inheritance patterns in the absence of genetic variation.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">18</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">155</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s13059-017-1288-x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D18%26rft.spage%253D155%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-017-1288-x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-017-1288-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-8.6.1921.26" data-doi="10.1038/nrg.2016.115"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hollick</span>  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Paramutation and related phenomena in diverse species.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">18</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">5</span>–<span class="cit-lpage">23</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg.2016.115</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D18%26rft.spage%253D5%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg.2016.115%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg.2016.115&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-8.6.1921.27" data-doi="10.1534/genetics.105.045260"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hollick</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kermicle</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parkinson</span>  <span class="cit-name-given-names">S. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Rmr6 maintains meiotic inheritance of paramutant states in Zea mays.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">171</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">725</span>–<span class="cit-lpage">740</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.105.045260</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.045260%26rft_id%253Dinfo%253Apmid%252F16020780%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTcxLzIvNzI1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC82LzE5MjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-8.6.1921.28" data-doi="10.1073/pnas.1410761111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhong</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Mutation of a major CG methylase in rice causes genome-wide hypomethylation, dysregulated genome expression, and seedling lethality.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">111</span>(<span class="cit-issue">29</span>): <span class="cit-fpage">10642</span>–<span class="cit-lpage">10647</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1410761111</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1410761111%26rft_id%253Dinfo%253Apmid%252F25002488%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTExLzI5LzEwNjQyIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC82LzE5MjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-8.6.1921.29" data-doi="10.1371/journal.pgen.1000737"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jia</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lisch</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ohtsu</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scanlon</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nettleton</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Loss of RNA-dependent RNA polymerase 2 (RDR2) function causes widespread and unexpected changes in the expression of transposons, genes, and 24-nt small RNAs.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>(<span class="cit-issue">11</span>): <span class="cit-fpage">e1000737</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1000737</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DJia%26rft.auinit1%253DY.%26rft.volume%253D5%26rft.issue%253D11%26rft.spage%253De1000737%26rft.epage%253De1000737%26rft.atitle%253DLoss%2Bof%2BRNA-dependent%2BRNA%2Bpolymerase%2B2%2B%2528RDR2%2529%2Bfunction%2Bcauses%2Bwidespread%2Band%2Bunexpected%2Bchanges%2Bin%2Bthe%2Bexpression%2Bof%2Btransposons%252C%2Bgenes%252C%2Band%2B24-nt%2Bsmall%2BRNAs.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000737%26rft_id%253Dinfo%253Apmid%252F19936292%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000737&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19936292&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F6%2F1921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-8.6.1921.30" data-doi="10.1038/nature22971"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jiao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peluso</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liang</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stitzer</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Improved maize reference genome with single-molecule technologies.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">546</span>: <span class="cit-fpage">524</span>–<span class="cit-lpage">527</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature22971</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D546%26rft.spage%253D524%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature22971%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature22971&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-8.6.1921.31" data-doi="10.1186/s13059-017-1251-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kawakatsu</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nery</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Castanon</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ecker</span>  <span class="cit-name-given-names">J. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Dynamic DNA methylation reconfiguration during seed development and germination.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">18</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">171</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s13059-017-1251-x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D18%26rft.spage%253D171%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-017-1251-x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-017-1251-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-8.6.1921.32" data-doi="10.1038/nplants.2016.58"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kawakatsu</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stuart</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valdes</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Breakfield</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schmitz</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Unique cell-type-specific patterns of DNA methylation in the root meristem.</span> <abbr class="cit-jnl-abbrev">Nat. Plants</abbr> <span class="cit-vol">2</span>(<span class="cit-issue">5</span>): <span class="cit-fpage">16058</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nplants.2016.58</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BPlants%26rft.volume%253D2%26rft.spage%253D16058%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnplants.2016.58%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nplants.2016.58&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-8.6.1921.33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kermicle</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1970</span> <span class="cit-article-title">Dependence of the R-Mottled Aleurone Phenotype in Maize on Mode of Sexual Transmission.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">66</span>: <span class="cit-fpage">69</span>–<span class="cit-lpage">85</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKermicle%26rft.auinit1%253DJ.%2BL.%26rft.volume%253D66%26rft.issue%253D1%26rft.spage%253D69%26rft.epage%253D85%26rft.atitle%253DDEPENDENCE%2BOF%2BTHE%2BR-MOTTLED%2BALEURONE%2BPHENOTYPE%2BIN%2BMAIZE%2BON%2BMODE%2BOF%2BSEXUAL%2BTRANSMISSION%26rft_id%253Dinfo%253Apmid%252F17248508%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo3OiI2Ni8xLzY5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC82LzE5MjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-other" id="cit-8.6.1921.34"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kermicle</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alleman</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> Gametic imprinting in maize in relation to the angiosperm life cycle. Development (Cambridge, England). Supplement 9–14.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-8.6.1921.35" data-doi="10.1186/gb-2013-14-4-r36"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pertea</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trapnell</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pimentel</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kelley</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span><span class="cit-comment"> TopHat2: accurate alignment of transcriptomes in the presence of insertions, deletions and gene fusions. Genome Biol. 14: R36–2013–14–4–r36</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/gb-2013-14-4-r36</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.volume%253D14%26rft.spage%253DR36%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2013-14-4-r36%26rft_id%253Dinfo%253Apmid%252F23618408%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2013-14-4-r36&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23618408&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F6%2F1921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-8.6.1921.36" data-doi="10.1038/nrg2719"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Law</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jacobsen</span>  <span class="cit-name-given-names">S. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Establishing, maintaining and modifying DNA methylation patterns in plants and animals.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">11</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">204</span>–<span class="cit-lpage">220</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg2719</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DLaw%26rft.volume%253D11%26rft.issue%253D3%26rft.spage%253D204%26rft.epage%253D220%26rft.atitle%253DEstablishing%252C%2Bmaintaining%2Band%2Bmodifying%2BDNA%2Bmethylation%2Bpatterns%2Bin%2Bplants%2Band%2Banimals.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2719%26rft_id%253Dinfo%253Apmid%252F20142834%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2719&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20142834&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F6%2F1921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000274649900011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-8.6.1921.37" data-doi="10.1105/tpc.114.133140"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eichten</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hermanson</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zaunbrecher</span>  <span class="cit-name-given-names">V. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genetic perturbation of the maize methylome.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">26</span>(<span class="cit-issue">12</span>): <span class="cit-fpage">4602</span>–<span class="cit-lpage">4616</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1105/tpc.114.133140</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.114.133140%26rft_id%253Dinfo%253Apmid%252F25527708%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjEwOiIyNi8xMi80NjAyIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC82LzE5MjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-8.6.1921.38" data-doi="10.1073/pnas.1514680112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gent</span>  <span class="cit-name-given-names">J. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zynda</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Makarevitch</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span><span class="cit-comment">a</span> <span class="cit-article-title">RNA-directed DNA methylation enforces boundaries between heterochromatin and euchromatin in the maize genome.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">112</span>(<span class="cit-issue">47</span>): <span class="cit-fpage">14728</span>–<span class="cit-lpage">14733</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1514680112</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1514680112%26rft_id%253Dinfo%253Apmid%252F26553984%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTEyLzQ3LzE0NzI4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC82LzE5MjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-8.6.1921.39" data-doi="10.1073/pnas.052152199"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lisch</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carey</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dorweiler</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chandler</span>  <span class="cit-name-given-names">V. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">A mutation that prevents paramutation in maize also reverses Mutator transposon methylation and silencing.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">99</span>(<span class="cit-issue">9</span>): <span class="cit-fpage">6130</span>–<span class="cit-lpage">6135</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.052152199</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.052152199%26rft_id%253Dinfo%253Apmid%252F11959901%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI5OS85LzYxMzAiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzYvMTkyMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-8.6.1921.40" data-doi="10.1104/pp.15.00052"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">West</span>  <span class="cit-name-given-names">P. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zynda</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eichten</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span><span class="cit-comment">b</span> <span class="cit-article-title">Examining the causes and consequences of context-specific differential DNA methylation in maize.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">168</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">1262</span>–<span class="cit-lpage">1274</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1104/pp.15.00052</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.15.00052%26rft_id%253Dinfo%253Apmid%252F25869653%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czoxMDoiMTY4LzQvMTI2MiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzgvNi8xOTIxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-8.6.1921.41" data-doi="10.1186/s13059-014-0550-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Love</span>  <span class="cit-name-given-names">M. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huber</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anders</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">15</span>(<span class="cit-issue">12</span>): <span class="cit-fpage">550</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s13059-014-0550-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D15%26rft.spage%253D550%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-014-0550-8%26rft_id%253Dinfo%253Apmid%252F25516281%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-014-0550-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25516281&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F6%2F1921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-8.6.1921.42" data-doi="10.4161/epi.29022"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Madzima</span>  <span class="cit-name-given-names">T. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McGinnis</span>  <span class="cit-name-given-names">K. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Chromatin structure and gene expression changes associated with loss of MOP1 activity in Zea mays.</span> <abbr class="cit-jnl-abbrev">Epigenetics</abbr> <span class="cit-vol">9</span>(<span class="cit-issue">7</span>): <span class="cit-fpage">1047</span>–<span class="cit-lpage">1059</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.4161/epi.29022</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEpigenetics%26rft.volume%253D9%26rft.spage%253D1047%26rft_id%253Dinfo%253Adoi%252F10.4161%252Fepi.29022%26rft_id%253Dinfo%253Apmid%252F24786611%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4161/epi.29022&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24786611&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F6%2F1921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000338429500013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-8.6.1921.43" data-doi="10.1534/genetics.107.072702"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Makarevitch</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stupar</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iniguez</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haun</span>  <span class="cit-name-given-names">W. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barbazuk</span>  <span class="cit-name-given-names">W. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Natural Variation for Alleles Under Epigenetic Control by the Maize Chromomethylase Zmet2.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">177</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">749</span>–<span class="cit-lpage">760</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.107.072702</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.072702%26rft_id%253Dinfo%253Apmid%252F17660570%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTc3LzIvNzQ5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC82LzE5MjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-8.6.1921.44" data-doi="10.1016/j.pbi.2012.09.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Martínez</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Slotkin</span>  <span class="cit-name-given-names">R. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Developmental relaxation of transposable element silencing in plants: functional or byproduct?</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Plant Biol.</abbr> <span class="cit-vol">15</span>(<span class="cit-issue">5</span>): <span class="cit-fpage">496</span>–<span class="cit-lpage">502</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.pbi.2012.09.001</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bplant%2Bbiology%26rft.stitle%253DCurr%2BOpin%2BPlant%2BBiol%26rft.aulast%253DMartinez%26rft.auinit1%253DG.%26rft.volume%253D15%26rft.issue%253D5%26rft.spage%253D496%26rft.epage%253D502%26rft.atitle%253DDevelopmental%2Brelaxation%2Bof%2Btransposable%2Belement%2Bsilencing%2Bin%2Bplants%253A%2Bfunctional%2Bor%2Bbyproduct%253F%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.pbi.2012.09.001%26rft_id%253Dinfo%253Apmid%252F23022393%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.pbi.2012.09.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23022393&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F6%2F1921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-8.6.1921.45" data-doi="10.1038/nrg3683"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Matzke</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mosher</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">RNA-directed DNA methylation: an epigenetic pathway of increasing complexity.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">15</span>(<span class="cit-issue">6</span>): <span class="cit-fpage">394</span>–<span class="cit-lpage">408</span> <span class="cit-comment">(erratum: Nat. Rev. Genet. 15: 570)</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg3683</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D15%26rft.spage%253D394%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3683%26rft_id%253Dinfo%253Apmid%252F24805120%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3683&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24805120&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F6%2F1921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-8.6.1921.46" data-doi="10.1534/genetics.106.058669"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McGinnis</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Springer</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carey</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chandler</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Transcriptionally silenced transgenes in maize are activated by three mutations defective in paramutation.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">173</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">1637</span>–<span class="cit-lpage">1647</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.106.058669</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.058669%26rft_id%253Dinfo%253Apmid%252F16702420%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3My8zLzE2MzciO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzYvMTkyMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-8.6.1921.47" data-doi="10.1038/nature08328"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mirouze</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reinders</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bucher</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nishimura</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schneeberger</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Selective epigenetic control of retrotransposition in Arabidopsis.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">461</span>(<span class="cit-issue">7262</span>): <span class="cit-fpage">427</span>–<span class="cit-lpage">430</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature08328</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DMirouze%26rft.auinit1%253DM.%26rft.volume%253D461%26rft.issue%253D7262%26rft.spage%253D427%26rft.epage%253D430%26rft.atitle%253DSelective%2Bepigenetic%2Bcontrol%2Bof%2Bretrotransposition%2Bin%2BArabidopsis.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08328%26rft_id%253Dinfo%253Apmid%252F19734882%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature08328&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19734882&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F6%2F1921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000269828100047&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-8.6.1921.48" data-doi="10.1038/35075612"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Miura</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yonebayashi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Watanabe</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Toyama</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shimada</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Mobilization of transposons by a mutation abolishing full DNA methylation in Arabidopsis.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">411</span>(<span class="cit-issue">6834</span>): <span class="cit-fpage">212</span>–<span class="cit-lpage">214</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/35075612</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DMiura%26rft.auinit1%253DA.%26rft.volume%253D411%26rft.issue%253D6834%26rft.spage%253D212%26rft.epage%253D214%26rft.atitle%253DMobilization%2Bof%2Btransposons%2Bby%2Ba%2Bmutation%2Babolishing%2Bfull%2BDNA%2Bmethylation%2Bin%2BArabidopsis.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F35075612%26rft_id%253Dinfo%253Apmid%252F11346800%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/35075612&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11346800&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F6%2F1921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000168563000055&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-8.6.1921.49" data-doi="10.1186/s13059-017-1302-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Narsai</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gouil</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Secco</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Srivastava</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Karpievitch</span>  <span class="cit-name-given-names">Y. V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Extensive transcriptomic and epigenomic remodelling occurs during Arabidopsis thaliana germination.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">18</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">172</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s13059-017-1302-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D18%26rft.spage%253D172%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-017-1302-3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-017-1302-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-8.6.1921.50" data-doi="10.1105/tpc.13.8.1919"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Papa</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Springer</span>  <span class="cit-name-given-names">N. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muszynski</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meeley</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaeppler</span>  <span class="cit-name-given-names">S. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Maize chromomethylase Zea methyltransferase2 is required for CpNpG methylation.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">13</span>(<span class="cit-issue">8</span>): <span class="cit-fpage">1919</span>–<span class="cit-lpage">1928</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1105/tpc.13.8.1919</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2BCell%2BOnline%26rft.stitle%253DPlant%2BCell%26rft.aulast%253DPapa%26rft.auinit1%253DC.%2BM.%26rft.volume%253D13%26rft.issue%253D8%26rft.spage%253D1919%26rft.epage%253D1928%26rft.atitle%253DMaize%2BChromomethylase%2BZea%2Bmethyltransferase2%2BIs%2BRequired%2Bfor%2BCpNpG%2BMethylation%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.13.8.1919%26rft_id%253Dinfo%253Apmid%252F11487702%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjEzLzgvMTkxOSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzgvNi8xOTIxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-8.6.1921.51" data-doi="10.1016/j.ydbio.2007.06.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Parkinson</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gross</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hollick</span>  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Maize sex determination and abaxial leaf fates are canalized by a factor that maintains repressed epigenetic states.</span> <abbr class="cit-jnl-abbrev">Dev. Biol.</abbr> <span class="cit-vol">308</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">462</span>–<span class="cit-lpage">473</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ydbio.2007.06.004</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bbiology%26rft.stitle%253DDev%2BBiol%26rft.aulast%253DParkinson%26rft.auinit1%253DS.%2BE.%26rft.volume%253D308%26rft.issue%253D2%26rft.spage%253D462%26rft.epage%253D473%26rft.atitle%253DMaize%2Bsex%2Bdetermination%2Band%2Babaxial%2Bleaf%2Bfates%2Bare%2Bcanalized%2Bby%2Ba%2Bfactor%2Bthat%2Bmaintains%2Brepressed%2Bepigenetic%2Bstates.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ydbio.2007.06.004%26rft_id%253Dinfo%253Apmid%252F17612519%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ydbio.2007.06.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17612519&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F6%2F1921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000248890700018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-8.6.1921.52" data-doi="10.1093/bioinformatics/btq033"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Quinlan</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">I. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">BEDTools: a flexible suite of utilities for comparing genomic features.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>(<span class="cit-issue">6</span>): <span class="cit-fpage">841</span>–<span class="cit-lpage">842</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btq033</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq033%26rft_id%253Dinfo%253Apmid%252F20110278%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btq033&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20110278&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F6%2F1921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000275243500019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-8.6.1921.53" data-doi="10.1101/gr.153510.112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Regulski</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kendall</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donoghue</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reinders</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The maize methylome influences mRNA splice sites and reveals widespread paramutation-like switches guided by small RNA.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">23</span>(<span class="cit-issue">10</span>): <span class="cit-fpage">1651</span>–<span class="cit-lpage">1662</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.153510.112</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.153510.112%26rft_id%253Dinfo%253Apmid%252F23739895%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIyMy8xMC8xNjUxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC82LzE5MjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-8.6.1921.54" data-doi="10.1371/journal.pgen.1006749"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rowley</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rothi</span>  <span class="cit-name-given-names">M. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Böhmdorfer</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuciński</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wierzbicki</span>  <span class="cit-name-given-names">A. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Long-range control of gene expression via RNA-directed DNA methylation.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">13</span>(<span class="cit-issue">5</span>): <span class="cit-fpage">e1006749</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1006749</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D13%26rft.spage%253De1006749%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1006749%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1006749&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-8.6.1921.55" data-doi="10.1534/genetics.114.168518"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sloan</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sidorenko</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McGinnis</span>  <span class="cit-name-given-names">K. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Diverse gene-silencing mechanisms with distinct requirements for RNA polymerase subunits in Zea mays.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">1031</span>–<span class="cit-lpage">1042</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.114.168518</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.168518%26rft_id%253Dinfo%253Apmid%252F25164883%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5OC8zLzEwMzEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzYvMTkyMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-book" id="cit-8.6.1921.56" data-doi="10.1002/cppb.20034"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zynda</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beck</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Springer</span>  <span class="cit-name-given-names">N. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vaughn</span>  <span class="cit-name-given-names">M. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Bisulfite Sequence Analyses Using CyVerse Discovery Environment: From Mapping to DMRs</span>, <span class="cit-source">Current Protocols in Plant Biology</span>, <span class="cit-publ-name">John Wiley &amp; Sons, Inc.</span>, <span class="cit-publ-loc">Hoboken, NJ</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/cppb.20034</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2BProtocols%2Bin%2BPlant%2BBiology%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fcppb.20034%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/cppb.20034&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-8.6.1921.57" data-doi="10.1186/s12864-014-1204-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xia</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genome-wide high-resolution mapping of DNA methylation identifies epigenetic variation across embryo and endosperm in Maize (Zea may).</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">16</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">21</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12864-014-1204-7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D16%26rft.spage%253D21%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-014-1204-7%26rft_id%253Dinfo%253Apmid%252F25612809%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-014-1204-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25612809&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F6%2F1921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-8.6.1921.58" data-doi="10.1038/nrg2165"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wicker</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sabot</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hua-Van</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bennetzen</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Capy</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A unified classification system for eukaryotic transposable elements.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">8</span>(<span class="cit-issue">12</span>): <span class="cit-fpage">973</span>–<span class="cit-lpage">982</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg2165</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DMathew%26rft.auinit1%253DC.%2BG.%26rft.volume%253D8%26rft.issue%253D12%26rft.spage%253D973%26rft.epage%253D982%26rft.atitle%253DA%2Bunified%2Bclassification%2Bsystem%2Bfor%2Beukaryotic%2Btransposable%2Belements.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2165%26rft_id%253Dinfo%253Apmid%252F17984973%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2165&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17984973&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F6%2F1921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-8.6.1921.59" data-doi="10.1371/journal.pgen.1005142"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">B. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pignatta</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henikoff</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gehring</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Methylation-sensitive expression of a DNA demethylase gene serves as an epigenetic rheostat.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">11</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">e1005142</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1005142</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D11%26rft.spage%253De1005142%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1005142%26rft_id%253Dinfo%253Apmid%252F25826366%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1005142&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25826366&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F6%2F1921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-8.6.1921.60" data-doi="10.1534/genetics.105.051383"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Woodhouse</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Freeling</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lisch</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">The mop1 (mediator of paramutation1) mutant progressively reactivates one of the two genes encoded by the MuDR transposon in maize.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">172</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">579</span>–<span class="cit-lpage">592</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.105.051383</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.051383%26rft_id%253Dinfo%253Apmid%252F16219782%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTcyLzEvNTc5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC82LzE5MjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-8.6.1921.61" data-doi="10.1186/1471-2105-10-232"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xi</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">BSMAP: whole genome bisulfite sequence MAPping program.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">10</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">232</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-10-232</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DXi%26rft.auinit1%253DY.%26rft.volume%253D10%26rft.spage%253D232%26rft.epage%253D232%26rft.atitle%253DBSMAP%253A%2Bwhole%2Bgenome%2Bbisulfite%2Bsequence%2BMAPping%2Bprogram.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-10-232%26rft_id%253Dinfo%253Apmid%252F19635165%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-10-232&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19635165&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F6%2F1921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-8.6.1921.62" data-doi="10.1007/s11103-014-0178-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yamauchi</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johzuka-Hisatomi</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Terada</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nakamura</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iida</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The MET1b gene encoding a maintenance DNA methyltransferase is indispensable for normal development in rice.</span> <abbr class="cit-jnl-abbrev">Plant Mol. Biol.</abbr> <span class="cit-vol">85</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">219</span>–<span class="cit-lpage">232</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s11103-014-0178-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BMol.%2BBiol.%26rft.volume%253D85%26rft.spage%253D219%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11103-014-0178-9%26rft_id%253Dinfo%253Apmid%252F24535433%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11103-014-0178-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24535433&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F6%2F1921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000336030800002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/8/6/1921.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/6/1909" title="Accounting for Genotype-by-Environment Interactions and Residual Genetic Variation in Genomic Selection for Water-Soluble Carbohydrate Concentration in Wheat" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/6/1933" title="Effect of Larval Nutrition on Maternal mRNA Contribution to the Drosophila Egg" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/8/6"><span>Volume 8 Issue 6, June 2018</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/8/6.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/8/6.cover-source.jpg?itok=4pU6R124" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 8 (6)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/6/1921" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-6TgPOLtU9xDzgvagXS6K824duH_VVT-7AItuzSSrp6U" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/8/6/1921" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/8/6/1921" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/8/6/1921" class="active" data-icon-position="" data-hide-link-title="0">Subtle Perturbations of the Maize Methylome Reveal Genes and Transposons Silenced by Chromomethylase or RNA-Directed DNA Methylation Pathways</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/489827" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-ISMvTseVB4U1ulp7ouV9eRa2lsL9aCxv-TL2y8qbBUQ" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/6/1921.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/8/6/1921" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-uW3-yQv2j7byQbfGAbgnWd5mAEERhYFL0omVdlny4NQ" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/8/6/1921" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/6/1921.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="489827" id="node489827--2" data-pisa="ggg;8/6/1921" data-pisa-master="ggg;g3.118.200284" data-apath="/ggg/8/6/1921.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Subtle Perturbations of the Maize Methylome Reveal Genes and Transposons Silenced by Chromomethylase or RNA-Directed DNA Methylation Pathways</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-1671-2286" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Sarah N. Anderson</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-4992-9614" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Gregory J. Zynda</span>, <span class="highwire-citation-author" data-delta="2">Jawon Song</span>, <span class="highwire-citation-author" data-delta="3">Zhaoxue Han</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-1384-4283" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Matthew W. Vaughn</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0003-3232-9479" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Qing Li</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0002-7301-4759" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Nathan M. Springer</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1921-1932;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200284 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node489827--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Sarah N. Anderson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant and Microbial Biology, University of Minnesota, St. Paul, MN 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sarah%2BN.%2BAnderson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Anderson%20SN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASarah%2BN.%2BAnderson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-1671-2286" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Sarah N. Anderson</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Gregory J. Zynda </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Texas Advanced Computing Center, University of Texas, Austin, TX 78758</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gregory%2BJ.%2BZynda%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zynda%20GJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGregory%2BJ.%2BZynda%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4992-9614" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Gregory J. Zynda</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jawon Song </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Texas Advanced Computing Center, University of Texas, Austin, TX 78758</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jawon%2BSong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Song%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJawon%2BSong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Zhaoxue Han </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>State Key Laboratory of Crop Stress Biology for Arid Areas, Northwest A&F University, Yangling 712100, Shaanxi, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhaoxue%2BHan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Han%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZhaoxue%2BHan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Matthew W. Vaughn </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Texas Advanced Computing Center, University of Texas, Austin, TX 78758</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthew%2BW.%2BVaughn%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Vaughn%20MW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMatthew%2BW.%2BVaughn%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-1384-4283" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Matthew W. Vaughn</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Qing Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>National Key Laboratory of Crop Genetic Improvement, Huazhong Agricultural University, Wuhan, Hubei, 430070, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Qing%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20Q&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AQing%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-3232-9479" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Qing Li</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:springer@umn.edu" class="" data-icon-position="" data-hide-link-title="0">springer@umn.edu</a>
<a href="mailto:qingli@mail.hzau.edu.cn" class="" data-icon-position="" data-hide-link-title="0">qingli@mail.hzau.edu.cn</a></span></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Nathan M. Springer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant and Microbial Biology, University of Minnesota, St. Paul, MN 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nathan%2BM.%2BSpringer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Springer%20NM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANathan%2BM.%2BSpringer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-7301-4759" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Nathan M. Springer</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:springer@umn.edu" class="" data-icon-position="" data-hide-link-title="0">springer@umn.edu</a>
<a href="mailto:qingli@mail.hzau.edu.cn" class="" data-icon-position="" data-hide-link-title="0">qingli@mail.hzau.edu.cn</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/8/6/1921" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/8/6/1921&amp;title=Subtle%20Perturbations%20of%20the%20Maize%20Methylome%20Reveal%20Genes%20and%20Transposons%20Silenced%20by%20Chromomethylase%20or%20RNA-Directed%20DNA%20Methylation%20Pathways" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/8/6/1921&amp;title=Subtle%20Perturbations%20of%20the%20Maize%20Methylome%20Reveal%20Genes%20and%20Transposons%20Silenced%20by%20Chromomethylase%20or%20RNA-Directed%20DNA%20Methylation%20Pathways" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/8/6/1921&amp;title=Subtle%20Perturbations%20of%20the%20Maize%20Methylome%20Reveal%20Genes%20and%20Transposons%20Silenced%20by%20Chromomethylase%20or%20RNA-Directed%20DNA%20Methylation%20Pathways" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/8/6/1921&amp;text=Subtle%20Perturbations%20of%20the%20Maize%20Methylome%20Reveal%20Genes%20and%20Transposons%20Silenced%20by%20Chromomethylase%20or%20RNA-Directed%20DNA%20Methylation%20Pathways" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/8/6/1921&amp;title=Subtle%20Perturbations%20of%20the%20Maize%20Methylome%20Reveal%20Genes%20and%20Transposons%20Silenced%20by%20Chromomethylase%20or%20RNA-Directed%20DNA%20Methylation%20Pathways" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/8/6/1921&amp;t=Subtle%20Perturbations%20of%20the%20Maize%20Methylome%20Reveal%20Genes%20and%20Transposons%20Silenced%20by%20Chromomethylase%20or%20RNA-Directed%20DNA%20Methylation%20Pathways" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/8/6/1921&amp;title=Subtle%20Perturbations%20of%20the%20Maize%20Methylome%20Reveal%20Genes%20and%20Transposons%20Silenced%20by%20Chromomethylase%20or%20RNA-Directed%20DNA%20Methylation%20Pathways" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/8/6/1921&amp;title=Subtle%20Perturbations%20of%20the%20Maize%20Methylome%20Reveal%20Genes%20and%20Transposons%20Silenced%20by%20Chromomethylase%20or%20RNA-Directed%20DNA%20Methylation%20Pathways" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/489827/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="489827" id="node489827--3" data-pisa="ggg;8/6/1921" data-pisa-master="ggg;g3.118.200284" data-apath="/ggg/8/6/1921.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Subtle Perturbations of the Maize Methylome Reveal Genes and Transposons Silenced by Chromomethylase or RNA-Directed DNA Methylation Pathways</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-1671-2286" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Sarah N. Anderson</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-4992-9614" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Gregory J. Zynda</span>, <span class="highwire-citation-author" data-delta="2">Jawon Song</span>, <span class="highwire-citation-author" data-delta="3">Zhaoxue Han</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-1384-4283" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Matthew W. Vaughn</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0003-3232-9479" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Qing Li</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0002-7301-4759" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Nathan M. Springer</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1921-1932;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200284 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node489827--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Sarah N. Anderson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant and Microbial Biology, University of Minnesota, St. Paul, MN 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sarah%2BN.%2BAnderson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Anderson%20SN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASarah%2BN.%2BAnderson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-1671-2286" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Sarah N. Anderson</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Gregory J. Zynda </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Texas Advanced Computing Center, University of Texas, Austin, TX 78758</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gregory%2BJ.%2BZynda%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zynda%20GJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGregory%2BJ.%2BZynda%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4992-9614" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Gregory J. Zynda</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jawon Song </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Texas Advanced Computing Center, University of Texas, Austin, TX 78758</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jawon%2BSong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Song%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJawon%2BSong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Zhaoxue Han </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>State Key Laboratory of Crop Stress Biology for Arid Areas, Northwest A&F University, Yangling 712100, Shaanxi, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhaoxue%2BHan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Han%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZhaoxue%2BHan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Matthew W. Vaughn </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Texas Advanced Computing Center, University of Texas, Austin, TX 78758</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthew%2BW.%2BVaughn%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Vaughn%20MW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMatthew%2BW.%2BVaughn%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-1384-4283" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Matthew W. Vaughn</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Qing Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>National Key Laboratory of Crop Genetic Improvement, Huazhong Agricultural University, Wuhan, Hubei, 430070, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Qing%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20Q&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AQing%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-3232-9479" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Qing Li</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:springer@umn.edu" class="" data-icon-position="" data-hide-link-title="0">springer@umn.edu</a>
<a href="mailto:qingli@mail.hzau.edu.cn" class="" data-icon-position="" data-hide-link-title="0">qingli@mail.hzau.edu.cn</a></span></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Nathan M. Springer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant and Microbial Biology, University of Minnesota, St. Paul, MN 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nathan%2BM.%2BSpringer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Springer%20NM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANathan%2BM.%2BSpringer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-7301-4759" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Nathan M. Springer</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:springer@umn.edu" class="" data-icon-position="" data-hide-link-title="0">springer@umn.edu</a>
<a href="mailto:qingli@mail.hzau.edu.cn" class="" data-icon-position="" data-hide-link-title="0">qingli@mail.hzau.edu.cn</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1921&amp;rft.epage=1932&amp;rft.atitle=Subtle%20Perturbations%20of%20the%20Maize%20Methylome%20Reveal%20Genes%20and%20Transposons%20Silenced%20by%20Chromomethylase%20or%20RNA-Directed%20DNA%20Methylation%20Pathways&amp;rft.volume=8&amp;rft.issue=6&amp;rft.date=2018-06-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Anderson%2C+Sarah+N.&amp;rft.au=Zynda%2C+Gregory+J.&amp;rft.au=Song%2C+Jawon&amp;rft.au=Han%2C+Zhaoxue&amp;rft.au=Vaughn%2C+Matthew+W.&amp;rft.au=Li%2C+Qing&amp;rft.au=Springer%2C+Nathan+M."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/489827/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/489827/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/489827/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/489827/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/489827/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/489827/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/489827/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/489827/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/489827/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/489827/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/489827/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/489827/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/8/6/1921" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-17" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/8/6/1921.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/8/6/1921.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/8/6/1921.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/8/6/1921.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
