<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/3/12/2151" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/3/12/2151.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/3/12/2151.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/3/12/2151.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/3/12/2151/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/3/12/2151.atom" />
<meta name="HW.pisa" content="ggg;3/12/2151" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Detecting Selection Using Time-Series Data of Allele Frequencies with Multiple Independent Reference Loci" />
<meta name="DC.Identifier" content="10.1534/g3.113.008276" />
<meta name="DC.Date" content="2013-12-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2013 Nishino. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Relation" content="10.1534/g3.114.010983" />
<meta name="DC.Description" content="Recently, in 2013 Feder et al. proposed the frequency increment test (FIT), which evaluates natural selection at a single diallelic locus by the use of time-series data of allele frequencies. This test is unbiased under conditions of constant population size and no sampling noise. Here, we expand upon the FIT by introducing a test that explicitly allows for changes in population size by using information from independent reference loci. Various demographic models suggest that our proposed test is unbiased irrespective of fluctuations in population size when sampling noise can be ignored and that it has greater power to detect selection than the FIT if sufficient reference loci are used." />
<meta name="DC.Contributor" content="Jo Nishino" />
<meta name="article:published_time" content="2013-12-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Detecting Selection Using Time-Series Data of Allele Frequencies with Multiple Independent Reference Loci" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Recently, in 2013 Feder &lt;i&gt;et al.&lt;/i&gt; proposed the frequency increment test (FIT), which evaluates natural selection at a single diallelic locus by the use of time-series data of allele frequencies. This test is unbiased under conditions of constant population size and no sampling noise. Here, we expand upon the FIT by introducing a test that explicitly allows for changes in population size by using information from independent reference loci. Various demographic models suggest that our proposed test is unbiased irrespective of fluctuations in population size when sampling noise can be ignored and that it has greater power to detect selection than the FIT if sufficient reference loci are used.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2013/12/01" />
<meta name="citation_mjid" content="ggg;3/12/2151" />
<meta name="citation_id" content="3/12/2151" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/3/12/2151" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/3/12/2151.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/3/12/2151.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/3/12/2151.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.113.008276" />
<meta name="citation_pmid" content="24082032" />
<meta name="citation_volume" content="3" />
<meta name="citation_issue" content="12" />
<meta name="citation_num_pages" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2151" />
<meta name="citation_lastpage" content="2161" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Jo Nishino" />
<meta name="citation_author_institution" content="Center for Information Biology and DNA Data Bank of Japan, National Institute of Genetics, Research Organization of Information and Systems, Mishima, Shizuoka 411-8540, Japan" />
<meta name="citation_author_email" content="jnishino@nig.ac.jp" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=J. P. Bollback;citation_author=J. P. Huelsenbeck;citation_title=Clonal interference is alleviated by high mutation rates in large populations.;citation_volume=24;citation_year=2007;citation_pmid=17379621;citation_doi=10.1093/molbev/msm056" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. P. Bollback;citation_author=T. L. York;citation_author=R. Nielsen;citation_title=Estimation of 2Nes From Temporal Allele Frequency Data;citation_pages=497-502;citation_volume=179;citation_year=2008;citation_issue=1;citation_pmid=18493066;citation_doi=10.1534/genetics.107.085019" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Feder;citation_author=S. Kryazhimskiy;citation_author=J. B. Plotkin;citation_title=Identifying signatures of selection in genetic time series.;citation_volume=196;citation_year=2014;citation_pmid=24318534;citation_doi=10.1534/genetics.113.158220" />
<meta name="citation_reference" content="citation_journal_title=Metron;citation_author=R. A. Fisher;citation_title=Applications of “Student’s” distribution.;citation_pages=90-104;citation_volume=5;citation_year=1925" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=R. Gallet;citation_author=T. F. Cooper;citation_author=S. F. Elena;citation_author=T. Lenormand;citation_title=Measuring selection coefficients below 10−3: method, questions, and prospects.;citation_volume=190;citation_year=2012;citation_pmid=22042578;citation_doi=10.1534/genetics.111.133454" />
<meta name="citation_reference" content="citation_journal_title=Genes and immunity;citation_journal_abbrev=Genes Immun;citation_author=S. Hummel;citation_author=D. Schmidt;citation_author=B. Kremeyer;citation_author=B. Herrmann;citation_author=M. Oppermann;citation_title=Detection of the CCR5-Delta32 HIV resistance gene in Bronze Age skeletons.;citation_pages=371-374;citation_volume=6;citation_year=2005;citation_issue=4;citation_pmid=15815693;citation_doi=10.1038/sj.gene.6364172" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=C. J. R. Illingworth;citation_author=V. Mustonen;citation_title=Distinguishing driver and passenger mutations in an evolutionary history categorized by interference.;citation_volume=189;citation_year=2011;citation_pmid=21900272;citation_doi=10.1534/genetics.111.133975" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=C. J. Illingworth;citation_author=L. Parts;citation_author=S. Schiffels;citation_author=G. Liti;citation_author=V. Mustonen;citation_title=Quantifying selection acting on a complex trait using allele frequency time series data.;citation_volume=29;citation_year=2012;citation_pmid=22114362;citation_doi=10.1093/molbev/msr289" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=M. Kimura;citation_title=Average time until fixation of a mutant allele in a finite population under continued mutation pressure: Studies by analytical, numerical, and pseudo-sampling methods;citation_pages=522-526;citation_volume=77;citation_year=1980;citation_issue=1;citation_pmid=16592764;citation_doi=10.1073/pnas.77.1.522" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=M. Kimura;citation_author=N. Takahata;citation_title=Selective constraint in protein polymorphism: Study of the effectively neutral mutation model by using an improved pseudosampling method;citation_pages=1048-1052;citation_volume=80;citation_year=1983;citation_issue=4;citation_pmid=6573657;citation_doi=10.1073/pnas.80.4.1048" />
<meta name="citation_reference" content="citation_title=Testing Statistical Hypotheses;citation_author=E. L. Lehmann;citation_author=J. P. Romano;citation_year=2005" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. S. Malaspinas;citation_author=O. Malaspinas;citation_author=S. N. Evans;citation_author=M. Slatkin;citation_title=Estimating allele age and selection coefficient from time-serial data.;citation_volume=192;citation_year=2012;citation_pmid=22851647;citation_doi=10.1534/genetics.112.140939" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=I. Mathieson;citation_author=G. McVean;citation_title=Estimating selection coefficients in spatially structured populations from time series data of allele frequencies.;citation_volume=193;citation_year=2013;citation_pmid=23307902;citation_doi=10.1534/genetics.112.147611" />
<meta name="citation_reference" content="citation_title=In All Likelihood: Statistical Modeling and Inference Using Likelihood;citation_author=Y. Pawitan;citation_year=2001" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. Saunders;citation_author=M. Slatkin;citation_author=C. Garner;citation_author=M. Hammer;citation_author=M. Nachman;citation_title=The extent of linkage disequilibrium caused by selection on G6PD in humans.;citation_volume=171;citation_year=2005;citation_pmid=16020776;citation_doi=10.1534/genetics.105.048140" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=P. C. Sabeti;citation_author=D. E. Reich;citation_author=J. M. Higgins;citation_author=H. Z. P. Levine;citation_author=D. J. Richter;citation_title=Detecting recent positive selection in the human genome from haplotype structure.;citation_pages=832-837;citation_volume=419;citation_year=2002;citation_issue=6909;citation_pmid=12397357;citation_doi=10.1038/nature01140" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Detecting Selection Using Time-Series Data of Allele Frequencies with Multiple Independent Reference Loci" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12.cover-source.jpg" />
<meta name="twitter:description" content="Recently, in 2013 Feder et al. proposed the frequency increment test (FIT), which evaluates natural selection at a single diallelic locus by the use of time-series data of allele frequencies. This test is unbiased under conditions of constant population size and no sampling noise. Here, we expand upon the FIT by introducing a test that explicitly allows for changes in population size by using information from independent reference loci. Various demographic models suggest that our proposed test is unbiased irrespective of fluctuations in population size when sampling noise can be ignored and that it has greater power to detect selection than the FIT if sufficient reference loci are used." />
<meta name="og-title" property="og:title" content="Detecting Selection Using Time-Series Data of Allele Frequencies with Multiple Independent Reference Loci" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/3/12/2151" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Recently, in 2013 Feder et al. proposed the frequency increment test (FIT), which evaluates natural selection at a single diallelic locus by the use of time-series data of allele frequencies. This test is unbiased under conditions of constant population size and no sampling noise. Here, we expand upon the FIT by introducing a test that explicitly allows for changes in population size by using information from independent reference loci. Various demographic models suggest that our proposed test is unbiased irrespective of fluctuations in population size when sampling noise can be ignored and that it has greater power to detect selection than the FIT if sufficient reference loci are used." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12.cover-source.jpg" />
    <title>Detecting Selection Using Time-Series Data of Allele Frequencies with Multiple Independent Reference Loci | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"aItDo5WcpyVhsX-HwCwdxolXeWkO5yanKSvDn31QAgk","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459599","apath":"\/ggg\/3\/12\/2151.atom","pisa":"ggg;3\/12\/2151","ac":{"\/ggg\/3\/12\/2151.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/3\/12\/2151.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;3\/12\/2151"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;3\/12\/2151"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/3\/12\/2151":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022zG91Obom2VHv8fxcO3ZpPkClE400GihQp6f\\\\\\\/meZaSOipd\\\\\\\/PUuxU53hijg3fStCmAiCDhl33PqYogzp2Z1UAYkdAD+6Yje4+rLYIy7tAiyBG6s7E\\\\\\\/+AikifR83QNrQ\\\\\\\/X5E3zvUGD4w1v9ypoLSa0nynIdAxg\\\\\\\/Zz2NanEHl4Bdk\\\\\\\/1PAhZsInRsmswzyQ5X0OJLFf75YI9lg2ZF95iEAjZfngWMf9htinCZqQB51blpus0REZ\\\\\\\/OHVtZTmaSVSBoNTBDZ0j9Fgc+7kTUTqD8OC3ztTr53Fp\\\\\\\/hX4X+mM3aJ9udvVmiwE++9HHBtQDHBP\\\\\\\/97+CBqA88bPVYZvofErX0euLshReY6GKLeJBLjwrBSPVWSBMWfCU5QS4oq8g+YY\\\\\\\/VXvLGleUZtiVogI6lLxOieD4bRgVnnH8DPtDCKZQvU0tikTUxloVcRTpuKC6KbVv6XuXHz4i0D8xF1gbsICvyv+cyiJVOMZCJ9f9uyXNNnEWexSbJUWWIAQ1eI6142KWiWbxctt6F5mJXIpwGGIVRob2IO69dlMh80e1jJ\\\\\\\/45OK4IrhWuFO7VgjqYdRJAFG6oXnbencmbrnndlDnOLRaeYmfcMX+y\\\\\\\/XeFk0YDNSPPrQtvSZ8Wk3swy7f1h8wCJnGMrWiF5OM0QdvGB2ogqvo5YV0oS5rhWc\\\\\\\/PoZN5xIz14f6z0Lr9jC9TU+nD22vNjiqjtZz\\\\\\\/MrCne5E\\\\\\\/HMzseYaLlVM3EnXvaCUOJJLzOZc+8CPO1Dp6j\\\\\\\/VoRfugr0CyZHW69hYmJ4Luev6YjdhEOMh\\\\\\\/Ef1dmGZ291RexB61sEDJ3suuWE1FakGs6RVBkqnC5ETDov9CWtVYjPUA5HV2fDD2Z5EoHgXyCy2DbX\\\\\\\/lOtjEGaVuTfV5mAHiPGxvB8SJkxb8o6fOC1CY7YOMEkRzZnRiUqqnW\\\\\\\/LwDvCi5SlOccn3bc36pDJ5SYaQAP9+EQlPdbubzATyz5Ibx+cTZm2iMZOA5VFngaImpEHYDToiNNwY8SEOaObQTBlEhJvqz\\\\\\\/TKfPfNvK7vf1WCCYf8gWpp03wHPzHojPv1Qq4t2g\\\\\\\/kn+E\\\\\\\/w5Nh8dSKFvhVsFSJRZKInZM8qZHf5dRKka8pWO2QbQ63N4Ib\\\\\\\/LkWaTKFo\\\\\\\/d03UfoiWJij4+EWhjYvHV+pg0WdBJdr9ZDrWtobdGwJutIPjIBmsAp4z\\\\\\\/dcN1AS8\\\\\\\/XfXh0JLKvD1k4ovmQcqnebAGaRkJ23A12vc4PVCE92Pxo150yZYBo7rrQEdCY92caOTPLBXwqbWCBBAPZb8ZEuTIanlaRocj\\\u0022,\\\u0022iv\\\u0022:\\\u0022+rK32zWiwaOdjH68oaK85g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022fb53cfcb031be581c5db4171e738609823ac292f7a0a46b607772e914c9c47ac\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022R0TkiTLUqmTE7LrlEYNx5vtiPa3pwNVvkIwUQEL1SG\\\\\\\/bhQuEEbLtZoEuefgSD0lLomEv\\\\\\\/NMn0WHKBfrPrFGTf2F9C+hFDj4+vqeIl23vN\\\\\\\/cbrzNi\\\\\\\/bxclueyrDcgjZXYdRxNfxOCuIKEwOKZXok57OtCkv5PShxW5pm0YYa3mifzx4Q9vCgpws+cICaSG+7VhX685q3bK1WBFH8W7Fpla5hWktDFwsKXilhooV6bP6AoRl0NoPS+JSZgeoKRqaIhttmBt9HPvm8sv9q\\\\\\\/83L73Xb7ZDk1xzLE\\\\\\\/L9WCv+jUlrrJsaOs11rYoU2ilFcSd5BQxTy\\\\\\\/7H0MyfpZzhm34W60FAhGSRnzUtWAPhQ0Y+8WGN29FvhMuQefLdQriuZAvUFq+RVM3FKllA+6\\\\\\\/c32Pd2Kyc58IsKY8S5w62ue38A+V8Sps\\\\\\\/UyAoA+E0PYqs5vMY91brkD7i4eJQNlb2Z9SIgRp\\\\\\\/F1J3yIK8+xSugeE92KiCBKSQ+XMeWYf4dTHQRwavPQM1+UgyqYnsvybXb8QmdUnAfxEd8DQR66KCcuknZ2Ho\\\\\\\/I6beI6G0QE7HG3dxuplWl5EhzJQMZ5YavTuXLVnGpFlRAxOBdKF11H0CUD2nL+sZOH2MyTF+k38VEhJczb7I+Imyb+rwznoS92c3NhXGnWwf7XHkb6x8i1tqjVR1ikevDf5L5yIxQt34vlvUYUc4CUYpZu2KjSrVj\\\\\\\/ZtJI2TV9NU+TTvrELPEiVV+S3pnpP0epOiriAFEPJEVn4qLNV0qpFEU0gk39yccorp9vo7TbPF4ja1O8OtN+uFE0yH\\\\\\\/RzbTZpxhhKTnU83lmjXKMUYwIaFJBduMZg6RbPdaM2MvmkC2lCqiUzLklM\\\\\\\/7CHx1NHRFiJcGAyTAMYlFP1tJGgWD0CxbKJpaKGh3GcIvE6Za1iVHAN6V\\\\\\\/xg\\\\\\\/9y46hgLrkG2sBQjCpJDKVPmJcEJP1Mu2KJMvE7EwwdyMBgogeCZTIlWmA41Ko2HywdeeIkAQtExsjRsi3D9XzFUFbwoW1tfWp0fenFR\\\\\\\/CFuHztKWGVT4S7RBMiQXO1g41pO\\\\\\\/8T8pXqSpCzZ3a95VAWInkuPGUK3HaKrXStzDKJ2MoNdZaWnmqLyv6jSzMAA\\\\\\\/QjMz7zhG0cYl0OJWYkEgAVc7X27qj2YSeLs6FQw92Uyk22G5cmyuq+196lC0vFkYRNlU35onfxh1FqAkrJsw4MTFxPc7JS5pB4E1rBZimpglwn9fcH2NVTWm6fW9h6LY5FnHV8dVtvuehWDoUh8H6OKInsf4pytaHtl\\\\\\\/HBzx1UEZVtSpZ0uIkiwZyC2GGLQISQicjcSyAxYtJEx7k2DKvVgHVV\\\\\\\/tko2AHsDTYUEnvRKK9CAStEyKGJH1pNQXeWasOD6Efdnu6FOBGcEGTV9TRfuDLFBdq1RSkZY87ngbNAIgrnsVkoSTGH+BiGYlklF0RGJMIPg8O3mJ4ZDG8UGzAduigC99DsZIQGcVm\\\\\\\/4K6niUWY717S4DbBZHZzWCotWvjRWNMBxDxz62+fdjjt0CV8V+RXdSHDzVkEUKb9WJFXi1etQBOwRpU8erNjHQI8QMfBsKct3Ajnd5X5MqUf5zlnyMyhM+ldrXi4E0UEu68gu5+iUuxms25wqTk8VTxWRXkIaCGVCW0dEOFGE2m47P\\\\\\\/\\\\\\\/wUlFc9ZHgLRmryxDE9z04qi2FwlIE5HuciBPKxfy7ARpQ00cLE+AAo4UFyCwAlLOifZajxUfGSHmTu0LFq17V1YcG5CGRgT3z0FofzlmR0AjGEyf+ugl0Ik9jOMl31QJinPkn7PaeaWisUDM8G8gKGfRlaM3poRhmo7HvfhYHnzv2kPS3bsv2lkGPucOVBazd7\\\\\\\/fzfa3+y8p9W2kb1it6A6ncL3jLq48WD501fazIdGELbgJEpdNVo+YC982xVtexiF6Hi9p5MurG8IWZ4pnbsj3IJY8XJBvYIrnaF2RLF2ahnWG0ygdrlgGdy+rDs0yrimf219Yzyv1yAUnvt0uWASp+WF\\\\\\\/7HR14AO1FIt+yB96afNAqDtz+MJHzNjzdeIBVpPCvlBgKdxmfqTCqzU2MUpmBUYvTnH5eCunTknbWlNa4LS+a8Hr9w+IlkMQyl\\\\\\\/SZxmoqM5mTX9TS1xH\\\\\\\/Um4nFTNbMltmr1YrkYWaO7Nzavvl783AN0zG+j7cWsjHPD\\\\\\\/U5TurQoxJFxAEDrg8cBcDu720ED27OamZLIjWqHzhhLYpvdsddhg9B6eGhV0tzxdSULWRB1oGBCpuehhU+bTFYgLf2QBoutODQhT\\\\\\\/XNw72573yM2jy96qJ5SBvtzegVGEty+D5IvwMDrW9sAJvrO2asKfEE4WkVKq9wbiEl5VvTBqS3kaIEWje92nmlqGy7vC9optKspF42BVy9MHPXZvXmH8hP6ifdUqnc\\\\\\\/FlMUc2JmScTRNjK\\\\\\\/aV71Pz\\\\\\\/URSmVzIM1iTN0Eq13P2kEMnCvr862So72BGyPdBY3ZNeNl2VlyHp342ofKJUcdGWJ7dMK\\\\\\\/YrDyLq48Zyj0FFDaPReKXflWzsfGoE7ginv\\\\\\\/FGmxFUfhyJCQoV5X41OTEkEELl71lGOrViYGuRwcLz0EPvl3c3Mw\\\\\\\/HndNIlDwZLnlTizuksWkfhikT6YsBdVZOM8N2kaWNOzMOnpjq29BFE\\\\\\\/z5Ne4oUXU1BUlTad7ef2aXaMx7cOw22AJtpc9VIzI+OrsnAwXaiRgwskEExBfu3+qWJncCdwFMbyBs8zxShG5sQbq8pKHKLkWSgSDAq6OVx94w1KtmESrn3gJQyiZEyreY4+D1+dsG95gsuPwv8X6OmGY7KDpAeYtMiDDPi3py+xy\\\\\\\/jrHNvlSby5AMalI\\\\\\\/j2VUHz9bLiQ\\\\\\\/rF4CPOhv\\\\\\\/x84VmLxLx12ocBRMuaHb8JMrzXjglCsYZeoBYnex8hJo9GsFMJiAhWxaFZqFgfu9dl9Xbt7BQThSfR+O6mVnEwMWVrzo0i7ydOCPOEMXiE8FYocu2CCbVofyWSh9ETO6ztVA\\\\\\\/BBoF3NqV\\\\\\\/xwEpZiAEuV1Qkx3KEaviaXMuXZqFRBXwYCUE+OYk3d7IWD2qeE4gdUvzg6E8hLJ7qBKi0W0npSKO2Rd29UxcYWK\\\\\\\/tRsdjQTUahPgbotzB9qfGOf\\\\\\\/csUkyylUSHmbBvWAcHbVPngph0gXYUlQB8jckCT2zfazck8RpXj8iizPZFu\\\\\\\/0qEIgkE\\\\\\\/MLmxFxmMAGbuPSm4rWqJdcvKdCNt3o\\\\\\\/YYb2Y2hUd7tpvHOuvfVve30g+jswhbminV6jW8Mh7QAv12KNmlMIAd+Kfg8ygnSzXSDVB26Q0IsFKvpzO82f0TDhaWEXOjx7oVOErxmtSnYR8IfYHmdyfL2pveBuW9Xp+WfBDXuoeZd0I4+yTdaLw23a\\\\\\\/iULkwWf03szq+CpHfE42Ap28Bf5+B4l2mzIuKZBIOwuDeBTqC1N6ZUNQHQczr\\\\\\\/KjQ8GqbgW4mDgfRVPm4Cy7Q+Y+S\\\\\\\/mzkiVTHDg\\\\\\\/nww58bqz7dpqfZAA5UaZF9CSYRp3gr8LDGqYLj7UCsoKnxxUPMGt2GzykRLCU5q7YD4\\\\\\\/0fImdLv2T2xaxFfqKWOzpfM9wdpzAZhJnCtSqay4N\\\\\\\/ViE+qHu\\\\\\\/hZSBgTkI6+wE5xNTsS3rvNnS+7yS+g5zTZhMW6SVdXMO7HKW2qwnwrFrrm03xCmGyIqZjb7E134M9rkypIQtBaIp8BHhlCgka4m84KjVn790IGItGN7xXQyAP8ZE\\\\\\\/zkRPDSjee1LEyZjJCni2Ge4PaEdcoIYD4CqSTcVynjLFT6ZNcvkpU5Y\\\\\\\/FwasMc9Drwoi6SGaVYZrvzN4mWQMawv2etHaXBxC1Ox7sib3aSwL\\\\\\\/rIJNZTBeTlc9L5n7K2toMDlhu+8FWEo5rGIJUfg4i9v0yXd2qWwpCeL3PLJoyTuTy9OvO51Y4CZWpAoNtXwUheV0qexR59IOGREsb26mYy0yj+f7Hctk5z87txS+tHw\\\\\\\/8nPI2ti1l0hBS8wSohREUAQ0YIZv9ai8+VGsDwR2wIKxn5jz69deTcp9+bGqq8A2WFT2wfdWbCX9te90VggPKn3fcnQHfQ8wcKBOxiKzxFw5dMU\\\\\\\/duRdFjry3D3RZHJRIOT+EQmcO9EvudwwcRR6pvJ2KE2s7B0jdZ8Ilxd5qwR6Eyu1XTYvKyUxMBOPs6e4nM5ASU9J9Uty7WTl\\\\\\\/7vCfq7T+ucWna2ojkG5yqkyOVBNU\\\\\\\/KmEG6qx0kkAy0U\\\\\\\/wbt779em82UQo04Pmhu11rNmX89ow3DYEb3OTky2mN6iTPQiguXatkZgRzM5yBAxdL1ulHHWKiWOmn0X1LFNd2k5lE1hwF6vQSs4xu0cLxpT2\\\\\\\/IKOgmtu8cLz9gjtDA3OWrvi5dw7LVvR10+PGoL6Nbj+b9cdbg6wXb+i9enpdZ0pI22i7lbdfA0oZbmt4CVs6rXtcEywdrXAV1k2FF3xM8NX9Yt2B4V7Nrk7+2RQo8XpWVXLUgMb8P60D2aTZwA1picSaK\\\\\\\/Gija\\\\\\\/ZL8caV+XXTlrF8K\\\\\\\/ojsZEJzcjUreJCvReZWZh5kmp6r9vA5Ubg3QIjDkK+hNhULyiLL1O\\\\\\\/YSDF4QhTrKUOnJJO7itIPQ0jcpMMjwV27b1dwdx8ZiaIhzl+uswSU\\\\\\\/rwZPmHppHH\\\\\\\/zvbGIknMHv3BEg4kyBPqazU4iEhnOG5C8UO+ljajdqLFAj+G9MPc+Uyc9MQ3ol9Bfpntl+9OZN\\\\\\\/rVsSZSLxNpNSVgsy8i17QH9iZQu0t0iOXu1S17+F0EJz3WLa8pNaaYov7g1w\\\\\\\/LygpxPk9GZSNtk+xS1Ta8rwjRXlcrKbmsfiQA1XNDxsS6upJBJWNMy8YAX8Mb\\\\\\\/ii0WiZntMNNrw\\\\\\\/oq7pkLHygHQqlh3\\\\\\\/7+dsfmLxmzsrcrVHC6y5f5rihrpCqfHdhYdMIkrbns\\\\\\\/bC1nmoK6HvP7afltkUp5kML2BDOYplia3yS8UfIZeq8XYcExLA4PzVAuPQU8almNEc24SRevyIFGY8rzyR5heroj\\\\\\\/2scQyZRj55gyRRng28DDEJ7U1PtgUxkJu7tVUBMI44WG8Q3hWS8khtyZF5oAw\\\\\\\/hRYvoQiYPfWd0NkskosNahln27zWdNUmquX4FK1PtZ3VleGVyuksrpqL9etXCEGGeGXmkTVO98srh1IaEhh6mrUyzLG3GxBCkOqpeOQ02uMk6j0p4RfogefFTLZ1DY5L7AWqPEooVAQLtspWU8Q+I5+pc7ahVYqyndY1AY5aiArm5pUKEV0PyCEJuZdxZTbnvdHVbR8HoGMboswpfK2wXCkr9uhbkGH9nCfy4OoBFt1ycYs3LxpqyDGpl667YowuWu68TJKrPdwziNK6sk3BzDQ7TcfRhj4aIq\\\\\\\/U9xrTxxUXbvEtDJVn\\\\\\\/qQSQ7eyb82CM7\\\\\\\/DXy3T8XgYzMzE1+q7n\\\\\\\/k7g+hjX\\\\\\\/iAdYCkdQlCaLEWRmKIUCtSEdFI\\\\\\\/oHS89JbI6Ebt3XqxgDnkFomutq3GjpOC8EjC8MEP8Yan0Ewe3BK2HSzWgQRxnfRkd4TgrjIjf4x0\\\\\\\/p6l3TDwuoUHLi3Jczke5R\\\\\\\/Da6Zm3JhSoCjTlILleG2Zht2LimHzrXMzlcOfbZb8hpWVzTeQGgdm9w7FvVWD\\\\\\\/s3VPfa9k9p\\\\\\\/L2c9zlWP96UkRbebwB48dSN1pWrY5WsX4lnywdkdBEynp5k3999riqhLfJ2vBlYrxUN46CqMdU4szrB5MNQ8OP6lNsk9FofMqMY4DD4fUjEyLmPlRsWB6bm0in6H6hB36OShrd9+XcQDWJPUpPwit45znAGp8uMY0s7NH6StSzhicfiIxuX8v\\\\\\\/G5E69ZWZ+tfbZeNnc5QOYH0b17NHTye\\\\\\\/7OkJ4f1YvPsu03iS6RDy1hnqrWKboe+BwqR6bMDPW3FpK48yZ+UjXgux0NqJxHhiMfU4ohlApPOOtLmXf6ZasEpqK5x3vJsdgTTJrabvcJfctSrKPipCZ+9\\\\\\\/qLCf5ZbWT+jVq2vNqAPywiY\\\\\\\/x0lXzXfyAcaWg6qCEWIc64yIZWr5KALcIYdUe928QmPN2\\\\\\\/2MSY\\\\\\\/PjK7RiRpswXoarLkI726nMBRbbSw2ht+6TFv4fdReRhGPw5tzjFPUvYV20c46YrzyGO0U+\\\\\\\/DhqjIN47WMvLhICSWu53hqLEvTf9l\\\\\\\/8Tv4bnM5Qqi7Mo2Bv3FCi5NLBQjgVK\\\\\\\/xBDndmTTnAzPqa7YMN1K9ltcMmWUPPbZNFEfgCsQ7G+JrpOp9qtb3zHXiQnDbTlODA9kA3GNPb8qe6N\\\\\\\/X\\\\\\\/iDrVpkWzc98WQROSgSIMU051Hz84jNIWbx1K0JnugtEsVcNDRFGtoNtrSNzry6AYQr8tv8p9vnQ+cD+v8RXh6NkRKCrNwY9mvWpoii4py3RNhaQ0Z15dXTx4tUgVVviO5HSZ+mcutNEAuZ2L2g94Mo3j6ZrCBuh5xCK\\\\\\\/Jl+ABQVWR5+SB58Ng1XvGLMobZINhNd4\\\\\\\/fU1ljjms98HkkyrVe4d3wHCs0kmmsI5I0TeZHlPKHjsc9003vrhK\\\\\\\/K\\\\\\\/nESVrKG187PZqgdHfRQS7jwiUd\\\\\\\/olyegQeyAU\\\\\\\/8+ZF8bWoRmm+H6pxaETUpF4mBol0iKELwVKZlB6MWTYTwcW5Y4o0rgXhidXMku49Xxkt6ElVm5ptprXaqPoTPaFRsmXiU6FMTDEdswV3\\\\\\\/hxRFRQlpjccYsVtRfUQ4nbjfCVuRQ2qaRr\\\\\\\/g3Dn5YguTCuFlP6qUs4hSwUJNWnXuvZuRJK2PBRB+4Zh7QV18bhcAxWWxzfQy7CFSxINVmBm0N8tejQ63EB2srPDt\\\\\\\/yF1oUsEQflCrP1csw0gdK7Rhh75AxsFB3bvkxaWiJ2w1aF9tz9KEkcrLPLa43qepXK04gG1Yrim+xT+qk0EFL4SYrl3uHS8uIhC\\\\\\\/QXimWm4l36MZu8mFROPuPsOzqnhW5n2574Uw57Db1MfYWlw9kt4VpBpiRbYu7SyWZO44swRTOiI2Dpt8BB+r+NvN4qV+3XKERaKbWxQ9B\\\\\\\/XYuMHImwgRb9fzfdUcxhsWXbC8GtN6HWvHcww7vnPhesCLFYpCt+\\\\\\\/UpLxOcpfEXfkv+HiDCtxcObMquL01HIS46dBOrLbym7Ql9HGlFNfXhifOEf\\\\\\\/Mknae9NRi1n0+UU+WhNNuKhjnmBfaHfJIExPZqHq\\\\\\\/mKPgXlrDBfZsof6ldVHV9EwgU+ZDpVBdBBtRtV+T\\\\\\\/faXRve\\\\\\\/DEsfp\\\\\\\/Ex57si3z6uKGkYwxJjUuxH3ro92MgATAaTlg9ANPEGsdW8R4rvhEcdGp+YjAHsDlCiwTWypjpZh5UV8wz0s1chbkI6uWSuvjw0nHt66CvtrOdbYM5s478UlKbH1IIqi196QnXMIPf686zDfYzN0myU8XUCex76RfkOYPk5SQUo9q0KXttKuBr1jWNTWup1E1mpHIBw0mTQ8seYwvwp4P+dKpEw4fCKEdd\\\\\\\/r7vAV\\\\\\\/+47KuExXUKHCgcrTJhGhYyu2jw10vxnYaY4oXN3aN\\\\\\\/aFKy2ysn\\\\\\\/FTz5s8To4W35xvp1YfA5nExomIoM3llbN79km3qnBeS8ipE+gnpiT6kiGNLnf8E8J3qdOHLWPaIu7XxINnKnQ8oqgQhCeM31wxyLSJEHhO6v3JZb5CpFziBKg2dZ7FjQKRbII76cHelsVSX9D0XjI2xoaXx6fbAKm8+ujooyA2I307SLGdOMN4mMqMeob9wE19yU2985LwnSHM3GnboGMGfFxVSuFI6LfkZvbgNTlpz\\\\\\\/IgdEyXEBvWDtyONFNqbSBkNr2UhKJCrepv1gwqtk2llhqRJMA597BRDtzSSS6yQiDHfuY4CiwoJY21p3EE3vhqntf57YnDeF8XSY2SjaHEBCSaCi9aBZKgXEbJN7QJxw\\\\\\\/yqPvOihYicmY6\\\\\\\/Qg4qu\\\\\\\/FJz2dmgrFVtWdYlUnwbhNf+wfo5VUBGhkD\\\\\\\/bvvwthj+iYxQlrzJuNtbr2kKXmbabx7lRCAEiUwm4N5BKjLy1P\\\\\\\/K4oPCIcT9PBYygSrCy9T4bOUx7cT5eZ1DliHsSK3+gLV7KC\\\\\\\/ovLcB9tDZKUZNqaKitxOXKhPrvEFgq7DM5G8GhHiEu0ZUYXltITb5zU0HP+q0mE6uwLiLRuzlPmYvQ6gKdOmw8xi\\\\\\\/KDWC4VVe5F5H7F4hjhu4gnOh8WLzWSgZQ\\\\\\\/Ta+JTOdw7CRCFuS4DPbkuLT+zimtexStipnNgB+gbbNXP03v2YhwMtZtYNiXTyAC4ztPGdDo5wCZaPMUpJu1ybg6PYGQSWflaPwBOfIszH65gRmnAIf09C3uRJs1bnb75kPlb7PRnNqLb5qJV2HG+Olpje5Gwzm6mYUIb1N36fxoL2RXHBLAF5AtCruT7qd6NWDNBIpVi6ad32O7RmGGd3JKOjT96lSfJbrxxrTPtw4Q7YqDVKudKZrs8hb7uq3Ana1W7Thx7pQ9tcN1BtyV4tGXDTy0xknKXRrn5JZWiVNO9CAVcyuB\\\\\\\/FhgmZNFJdpD9fkXLQfmgsUFt5vsX5\\\\\\\/L1ZdRpzSQ4obeDkqKRmunFLuPgFES7dO04fAoTznsoy4Asjl61kgoQUVwhtNazjrwpLCyIHMBRx89AyO1J8KivUWKAYoQNy60X+5iFpfT3m2HuuO2BEcqOV5+BNQK4fceUjCgdUrdHNK1XfsJxGQ\\\\\\\/trHUsSlrexZyggua\\\\\\\/rjs5i3iek17Uxx2CAK+gtlqkhM7DpGPk6OaYWiqSTHKGFXCYNYFbHnbAuSDiyu63SvjoYTqadhXu+d9EPMhDVjP9VtBBOtuNh\\\\\\\/jhRrCd6wVuoRoDUDpo76nrNnYGVPSmmae5i1SphFVE1+\\\\\\\/juJCG8RPUArLJR1gLugfOaQLM4AJQCsoyI87khISHBpHU+q7x5ubwXw06PZYm3dx4D5vB+cxAGfGSWIYfhaQV2mmna\\\\\\\/vOrM\\\\\\\/sTHerFbXMWyhJ2XDNIYt+glG8lJbzwGF1CABP9cMoz8HIURUFKPbB0DXrfXjbDfDcC0Z7vBPul0Gy0eOWqQwB7ZbglK2m22hgA+EXlc5VYcKxvj+AXGYHiSt9zKVzyqd0lV2K4llxutShUceoWp9eA5\\\\\\\/5+sQSiZ7rFwE1GdbIgElHXwoYct5O8KDmnuNI++6tEDjsAohiTq36KoFZs7fRN3rtToQpWD34P0HTABT8xHV0jfT45Zva3sag9F0cUuRrUc+Obtb1YVQYGYJe2ZKDp\\\\\\\/EbIZs\\\\\\\/o4YCorjVxsAQJ0ZYRBGIRdN6QRfr408WWDVSRtC4vOAk1nv78QE5cYEi\\\\\\\/dK0YMKGJiiGmM7ExJswb37I\\\\\\\/kz\\\\\\\/Cxh13GpoPxztKjGT5NXXQeoTZ52SN4Ss6IBMsCyWtt3x21834+nh9aY\\\\\\\/BsYpznZtog5XKXrgKyJo2FOCTfXbCd7YBRpGIyIGJFZ3nnetg3o94EmN1o0WFHltGgTbVrz4I3qT6M0v+g7PfEW4WwPTqE8D5qnBVh7CZNFL4RfO88h3xaYscu9InCAAr1urQpwy+siZsM9hADaX7MPOXpxenk8qWjvuyENmW3xZ0rNx+tu9fMtT6RDlpz6vCzqSp2sjWBLzZnZkD8ogE6ZPQ5e8g5BlMQSjyExwk5AZPPhBxmoR7bjVfR7e0NMHZj4IzZEqF8PBKoAhq6gf\\\\\\\/Rn4jdCxWfQeoaUF5oRsTX2BtvxvdLD627BXlFEcfV88QW8mqyYLpcewAGmEv5eNfC7roOEic1pCGAs\\\\\\\/7gjLLdtOtKeYOCXzI5NiV+hF3Udi3FUKHM1+h7jl0Usg8qCg5wU9djM1Tao31mcGCh\\\\\\\/IV7TRQ5wqLQBtgyLkRZXfMOrIDEJ06LlG1Kv8jxlHv4XiqHVVHhgY2agVJPdyBXh4qIFPCOUaVkGK1yulJ2MtgDUssP933uOL2kgCe\\\\\\\/jwKR3tYnzCOoRru1ciIo+IRPeHMKoVaaE79DaepYx9dwAKxfNeGlAsPvT0T0VKdYq+jT9QttQXdkf69prMtwFwm2f2EYDwROADoXQC18keR9jKkTFnTZGDeFfX7gMybcjq4J0n\\\\\\\/mBfGKi0okNnhD8erZeZyNsYrb80pe1ul0E+q1PuYIFQWNFSAnUJiqtjGISrwM4kmMuS3r5INpi0R4We6IksEYf5q6AI88A8F+EA\\\\\\\/a75unGoVsys81TKArzdj7EdQntPHH1Ua+4t0TKwYbhNrKv9vT2gST7i1guSCq6sBTXDt1A0MWvjKnkFHK\\\\\\\/hb2YfUy7A53OcM+bvOp4thtWku0VMBmBjU+kJl3mHmL\\\\\\\/W5QbvwFNAlHGMFuR9VylbO7WBxY13A+TKysVDXDSpA\\\\\\\/oJy7So\\\\\\\/lQhPCsDhCoaRsJAUpK6zkUX7zu7YCXI57mE1zRwFynO4dPDrbcEVybV8dcAX30FOjMjITPrIaQjGQIMXMSNn7PXkKDVsV\\\\\\\/wUcC6qEzQY2uax1i3\\\\\\\/q8mna3I\\\\\\\/v\\\\\\\/UvtGeTRid+yj+p6V0kCfyMHlAByErrsO7+70GojNrl7dgn\\\\\\\/X59F2ki6hmrfSKuIPgSKdAVMiFZ03r1I39e42t\\\\\\\/mBvB\\\\\\\/scqx\\\\\\\/1+aAWk32Zw\\\\\\\/7XG+7YIDRS\\\\\\\/WYdW9YYUJPdayiGpzR5I6wuu9Rs0gMt+QCjWHCbM6aCTotZjQuFkUrZzaIJJHAu7vKYLiminTjrDNfs\\\\\\\/NrYoUMMWnTsUNunzcrud9PXoBwh26OiLO0mF6s3hDlS0eVF1\\\\\\\/IuCetgx8cjBn8A26UvTOBayQlIxi60feTo6MerQFDmXLfVg4h+s0Wy2bmy9+iHqfc\\\\\\\/pa9bog8mdCQJkoxiCoMzLPafqGpmACJ6Cd7Xcbn7vwBCO59jrrcGLqVNPQEX0i7lKYVRZ6qDLwJkXUu6EvrIYXx38oznkgy+Utz\\\\\\\/+vglzU5Arzna+YPy8tBhEK+jmhuxP6Y+Gkx+W8tLCMqUpVFEXDiPRItRNn7TmSZ2FPnyv7BZyt+CQLO0DAzy44JUTClFxWZdGUlKkSfNkANjPwuDXH4xEKbpfSyZ0WidWxlBynDP4U2CCmoAqbruO3XLmA12Vpep5Xb\\\\\\\/wAHZx9E9gFefWPPs+\\\\\\\/yr0dPGBQ6hK9Ppd7dIwR5ADrqODfNXOiPJeLhqc0jPf4yTqUEsHchCgZqCsTAiutYP+HC40O4kbFaL6csVqtfICOrscQH\\\\\\\/\\\\\\\/pBcoV77hINvErH3rC3U2VxaEFrlFvQOh7q3wcBJauaY2+RqGBPn5d8FUp1F1s7I3q+wC61NUdtutPVAmP36IYeSZiCkqO6cUVZucQsJu3YvPz5TOSjwqr2d8LVUU8LL\\\\\\\/iGBw\\\\\\\/dPTrwyCjyJa7qQiRqhEkMwYt9AUPPqiWgYiAVaT9Xg7ib5SnNuebIrynYW\\\\\\\/7Ydk3DwOUPAF9ZN5RZ8qedpciMESnYyO8SYny79n4kEI+JY\\\\\\\/CoojuA9SAm0TlHc+m+QCkeAp8ptfaSe436cKLjzp3BCdFewwSr6ybxYPBLlFVqFqwQj5lup2imXSAjtES9bJlCb+3GT35ts5IFu8eOc+k+uLNAdmAikWMH353Eh+f3WIlp\\\\\\\/BRFS3sXMkM+b56fNWbA9O7JRR0upJPJPoQ2H+ulr4aqlMSLSfUxLRGQdm2ThPd0KGJKnLFZBmFRlSaZJ8oGNFAvHJvMWMdYFvbt49rhf9aATQIziy0gTsTAazkKRp2SpvK2pDa8tZYSlFNTq1CcxyKMQWXLZCI9metSHWdvK\\\\\\\/1WRUgHPE1X34MgPwzBzT7nn9E37Bw+vrx3UOpwjkYMiWwVAB\\\\\\\/xi6+LRXdEkxRCGjDfLBodnod5o5e\\\\\\\/vMSnQGowDzIY9\\\\\\\/\\\\\\\/MW\\\\\\\/++eBvwek7OacNEk5IdtG25\\\\\\\/KDu2glLdBZtKPb5E9\\\\\\\/TvOdsfY7wK2DnG6p\\\\\\\/tM+Ouc2AgGYf1dSAbIwqsdd3AlfBqO3f4\\\\\\\/Kzp\\\\\\\/zU9VQjVC27uKgDRJ01TKhlwih6eejTYpaPWXPO2PGOzMGpXR9tiLTmOlcA+iZw+wjDCVjiOAffwp41RN7YQHPvPu0fZyXCzosJ8SD1vR2M2wtygER1WMz0QlQr+UG0Dt+xhUBDa6ltWDTgFGZrKVBt49akHEYOzNKkJMkt\\\\\\\/vMeP\\\\\\\/M8LoqscNQ1DJnlE97UKNhaceJa+uPO5bMjFo3FaFh57X2PJfPLrMZMFwc9N8uqNsbg2Zef7VmGeDQ8\\\\\\\/i6J+HZ\\\\\\\/3xD8dxzDKcVFR\\\\\\\/cwihI0y1ksQ1Da24knZJ7HLxD75ElVUDK9G53zOIxh0kI\\\\\\\/\\\\\\\/pKDf30JIj10vOloiQ8ZQf3e1fpNxPQ9gyytAWnPWRvuenzzSLbABqQ8kd0PYXdLxwm\\\\\\\/3tg1HAM1rldYoUXMnIwsQkhzON\\\\\\\/pSsedjUSwRzgTchl6\\\\\\\/EScBOE17AsHzTEwzLp0VP7MSc1r9UpZ2CCm94dfeRgLKWzskhyPx1huFo6Dtfh16VCc5QeaAoUyTJ+9yaiwYhru3SUpA6yQVBF60hopyYV2A1D18VQ5jgcs8kFw\\\\\\\/KP6xcudOFTpDvCrlk9iAByk5w5QunW9mC3qJBmmSqjLTH5tuVgsD6TDWJCoBWAXCtsnr1\\\\\\\/\\\\\\\/jIFy5Lh1hnIb8x+IJPoK2C4GNNANn3HI5EVEHaCjArF1ccDr+yDf2VTIdks6m6P5XnXg4xMbe4iHOqXbrO+aDKhVWtmBfwWFLTjCNP+132IChNC1dk3FrhvQV5x6MOFpBJFwwoUOdyI6kBMcaLCLr9vX0\\\\\\\/IGUWhbfdExj20AXgJQUGJ69qttY8YYhQCTLYY95ai4nItWLAXmUEuY1YGK6Gg\\\\\\\/Qxepg7nTNqa4D33BSQd86IYkSzds5Gtfcrpy7K8F8suvDPEKfZY61PpQKzdxTXTz8C6vwUhfiGmBbJhlpQv3\\\\\\\/cthtw4HOhW5E3Nqg1LBP3HchQAFHuO3HKnyNQfDtt+Sztj4l6MjaumCelj78EkkP\\\\\\\/aY4SSeeR4Y+zuVS80i22gztbmsqOdPaICbC5xmpnYCCSPQ2UzD0EYKZ7XdJrscoO8ZKCDZzlESrar8zbhqJQfbZutMZHfYIKmiW7nRqnDjoFDE2CA9jBK3J2VOQk\\\\\\\/cZBKyyOi88AGfCPK8pKGPVBCZ2ICcEqqtK7GWv0oOEUPI\\\\\\\/Kc1MCVKkzCu\\\\\\\/xjz2UrC1ZyTcbC5BmAe9rhEPq8VPjb+EhKfISK5hmFlT+B9crhEU5dP\\\\\\\/R1UJIprUSgS5VVFGMd0r9y79b3x94XlJxhVciz1yzJq+qIOvexX5m4NaMXkKFdpdPjVSq2ZNSS9AOEmAnbORERw07rRyxRsMsAXkAHd1f4MsY2sZP844vKkX0BwFYTBLBtlu6qq8rWSvJhY7+M5zoh1Y\\\\\\\/fKEp68IGWmUaV1YSC7UffSyc4LlCGJ3SPapeJW5C\\\\\\\/8JyPlXbQFWJzxGsQ7Grg08HBjiTrOdHyAxKz2wgh7wXdACelx7pbDozVnqTmsEo2LclLmh9FTQB0c+48O30x\\\\\\\/Sb\\\\\\\/HQz15jIXqwt3nuCciGhwWeZHEma6bkCSIRCXhizso+48N8rZos8w4ln+SEdK\\\\\\\/6d2dlhotaOURdIokAn\\\\\\\/JIudOuR2AMBdep7hAHQlhHGeq4\\\\\\\/a8iJ5J2ikFxvGCsfg5ZH7+WKopJPJEPPlvo91gnF1iGlRHuye6rjjkJC0ghZRflKFNlLb8XuP79ebO1F\\\\\\\/VsNOVZO3jDSupXXG3RXeAMFi3\\\\\\\/MiaJQWqRAH9lgrl9bS8GktfiSZ0ZnV+sEOzUmfnjNHt7tdjeSO37Q4Nhs8bNNwDbPsVQxlfriVowxdiyFcsK+I200i0U3195hhpvTjoNN\\\\\\\/gcGuysu8Li5FP7BaLpNS5WGro4YqmlDHecZOo5vpmd\\\\\\\/ca21KylWFb1xrfxmF7ncnJ9NQqflHV98I3hpRTguHiHnLyck3Zq0hYRBrijB84Z\\\\\\\/oPMeGXXKEoO9GqY0ft5MiPsIsBgtlMdWeICeEoFb+Q1o0JMNKRIA+lOOVX9Uw3+zzhqSfCCe4GceyWuM23OQNOZCEsxhE94Kgnpq1ApG+EOH1XR8AQG67MylJGTl5fOhhI3xsifUyJ79yGP3\\\\\\\/CGKnTV\\\\\\\/AWtzJz0SkmeMbGk39uJIfX7SigTClpg343PfIkrxTBHYgzvMzodjHNW+29Ezz5eqBqgzEOTrODs1fxnCNx9K1UC\\\\\\\/hkpW5WSsGrf53Z7WEmzwWrU0F7P+OMSHwD96n\\\\\\\/WUTClZsR6OqKAV6eYJs0eGckQO4B6ibpnnqE9Ro8AwTXQiwrN\\\\\\\/7OlnJYfmS3oQbt\\\\\\\/mdaAJS19h6gotYQogfMTXnoGil9WRfBHIfqQwolChcCmRiixdem4Zq5vigueDHmzQTy6u24P8bKRCf8xC8HNgxdoSIvpzfDRneELtKN8Dj14gQIACBt65X9UT7YE7MBZTHLjRGcYjmtMTpnCVK69001pUrEI0nUVBSAj5Ix7PNCvA6uCiBfaUN\\\\\\\/GoBG0OfvHEo7XdTldCc9S9jwQ2UvXeDTpz9hKN9XeOZK0EVBB+eatEZtkbK3uOo9NTlU658OYzaVsdPubE2pnZa9KaNs7ITQhTznLFoqec\\\\\\\/xOm182ZxvnqL6hnGtA6gBGtuGkyaiWNad6b5ETgqw57b3qyc16++hVhKT0LcpJJ\\\\\\\/jNoV\\\\\\\/aznVLnnmbwrshlwgkiuSfCGqH5M1sqfFRNO0AizzF41FUthFuwCFvtTum4HgwmzaReUdNSX410QomJKpuCii6Tb9P7GHUNeMAMkYofW6ZadEh1K4MRna0Jcv7zSnTQxFuQhU9E7L88\\\\\\\/QYKy8vcfKD1DtGJkTJbVS7kwz0q9e1wkh0VkPoMVXmvyRQWfcE6JKshJdWAaq\\\\\\\/CSL\\\\\\\/njjm46D8uu1RjnqvDENFzDQ5CsCnZBkT8tS8BnrwHVYMA0vPN6ehYMVIX9KNEFg1PE039UsGKZsx6pqiZ+FPhJOwtMkR9o4MavBgNRr\\\\\\\/uq8ejdUCbGik3D\\\\\\\/7kgTboV36bavTnKxQkj8QngdIMblYpJyzvEHR4W+uq6eYQEHRzrpuvhzY8dNA\\\\\\\/acQiIHEorTV2+KyAw+VlAJ6fpIujuJQGS\\\\\\\/m+UtbLxKOfwbhjFKbiDNezh3VweIhfSg4Bu\\\\\\\/n5w71VCmZniBnyihGcMWmtvo\\\\\\\/VxV3Ey\\\\\\\/oM952VHKXdGyzooGnUG7vXCegYw6ck1yByi4QuGKLxRAX0AcZRrze2YC7laSSTceuuWStj3oJaH3GwTWIjl3opoZzu3qIOTlzz395fHztBKgkQ2LklvpGvg983b\\\\\\\/4Bul3BsLS2E010hLbvnO0FoCUgeSL3QgIHMBQe370SNEIz0Jw83ze\\\\\\\/zDEqxUK2e3jg0OGsDw7aMoQbarJH3wVKBqADL\\\\\\\/FcVYohwCUKIQz9LvbJkDsnt0FdOmHr61j5Nmnl42K0LHjLXsbKjOpvKNXZp0EiM1IOOywxL8XLkEl5xU3iizyIwHnjioKTpRxjCOvhuR7Owr9HsJnrSSxGUmp4POf5wwA4MOm8dgQ6Nw8M0\\\\\\\/Pr3zRul1twZIuPudCwDaMvfdcUu6w+xE81emMbaZ5BLRjv8s9cFy6gmJ3R3E9M87nP62oxFdWNh5LdNZSd7sIa2GGuqq438wFBFqtU3zGzoeDkOkc4WpiPsmBefI1dvjZcsBL\\\\\\\/KVhAqIkiM+pEGBa\\\\\\\/vk3bTOLDAQQIxOrzlGeT6vsNzYN2bAfNCvEuywoMivd3TnhzC4spqlHkL5ocOw+e3z8Ivi5tiqQC5MW3A5SMT05cCIvKtQnzbRtjNXdzN0lr+f+lckRc463lRl1GD5Pu7zZl8Hh6Iu\\\\\\\/dcXK6ayhI3B\\\\\\\/A6P63HW9krmJPNEpRtehZRdpGlH7nAVZCSxyg4zPgGgtglLJWlD7nj1xWhBm3fDxqRkuuq285pxwF6MuVBnVokUEg\\\\\\\/glGZij7Z\\\\\\\/lEY6Hng2qyNZt3\\\\\\\/KfiPqUm6v44TsBDsRdLy0mOSpKeS87IxUZnLE46uLPYZMIrpdbWY\\\\\\\/IOiUKsmae\\\\\\\/1VknyQ5sVRhGFgR5fXYGpF3+qVvomMmuNUJGQ3RZSisM4UwIETECJ71PQTAARJMET8J65QUUECGrAY+R5sdvD\\\\\\\/qzwvhKA1WAyzm1iF+mNMvjtW5xcqYGswqsDEBJuovkzhxDKIp64xfQn37bmQSI0dq2P9DOEO5\\\\\\\/XRssANFazMpVqW0OHv\\\\\\\/DwdcBpT2EVA7phfZ5+c8OCH08QDAKKIxCSOEKudm7dluWh0Zins0QNajv92+LIXzZZC6hcAivIfmZXn6kMerCWyl2Gdl0oVGcyMVzu7ERSo86dNT5g6ETyNc5E7IBO3R3VVCJNI7Jvg4T1fBFnTpOw8NFT5J0M\\\\\\\/K1A6Foxif0a44Ie80jeaogeMzwQN0FO8ffDUcRU6BJwjKG+4LwriwuoJbqbO+B2CdYHFtk5yDCqsEDcUn4eKJ1BWJzRVKyqywYZjHZ2wnBgOWPAbzDa3fE\\\\\\\/jAlpOY0\\\\\\\/8t3M7HgL3cK61wSre2tqDHQCYSHaYc8HNLdYfIyYAL21NLhsTJo0Oa4GbgnL79szCj6721uInSCrpny0\\\\\\\/Q9HNLlplYUU5AxeXivKwRbnb1JUV35k3XvjdA0kldFZtiLHinoNYvgsLrqagdNiESWTbd\\\\\\\/DAvMXqBszYXHjAJDz\\\\\\\/lh2Cy5AM630fTq3XW5W5+QRV5TC6tWL8EfhBRUC3j\\\\\\\/KS3Np6lE4UoZ+fF76gIiEWOJLBPcwYtMTCwgwPCwRv0+XxShX6Ez257E9viVwK8Yo2Hr7YEmZKnq7j5rMKyqUrzS4U2ht6XWRMHSMUa8nUi1LHGiErqxCjJ+z1QDzRQ5Ouv68duMGmsaSnz0t6Aqyg4ew++GsZ8R7m1ih5JnFINzjLTwo9V+3yNrDrFdkegalLbvUWFMOvz6OOco2sqQuTSRHOKvWX5OEZ22fj+Rz07ebe186zLBwbPaC+9heEnQPDGpnAnVp8qAXbSzTsc5n2ZkZPpe+8GvFSDI4sKfI0R1f7bdSfdWurA7r227d2jXhX2KxYoo2GWSLJ\\\\\\\/ET6s2D5exDwkegB8gKWaU7auol22zlV8adkBj5IHN63+ZZdPEwpwvxPw788xjqdSGTHvqOmQPv0o23GVeeCR804x+LvxAq6Ty7a3V5RobqcobTwOqSdYpEToSJrYwNlnuD7VMimE4n\\\\\\\/al4kTzMtOa2r6kpjN6dIt6Q\\\\\\\/dwmeaiyGpYw8TnqC9nctxgkKgKivS4rbiu1eh3s9PBZ1o1ew0PQt2QEjUznVgnkslf7UkZeoTtGRukK6vqPxhZuyZktsEE5POgqP0H9Qco9NB1f4j8T\\\\\\\/3kEf0EjFwnVpdKNFzZUdetUJk321r5pV3Q4dVfip+eHLnJpnti3XpXhNQDPj8uLidkgpb7ubFDE9fylE6k6cvp7fO\\\\\\\/+DOQ3G7W1\\\\\\\/kpqAgapkytOlKv3aZW79aqkFrZKfQD2dPZ\\\\\\\/lw8XqZUvyYjEseBTz4qdcd72vp9TNGQYKtUJ4Ax2jjk1x41Dfj55NzkVWBUEygwcrLr8LcXBb7R+v2VLNx1stTNzxnBOkM1+O9aDqQ6rB0M5dL04UkbKsyy9rqtSiL34053WkM2gHLEhJX8sk2Y2SN\\\\\\\/aGLayT\\\\\\\/XcA+8\\\\\\\/01oaRNeS8lo7W7hbGZPmrHAEk8PE6DLBnzoLi8gqlPYHJDsuCwWIVgXs+1Msd2aMBjyZkjL+qUmTE2oKJpXoclr9TrgH2tPmVJ20UZI6GwthtdraB1JzCsgOdT6mKjoCiS67vETzt9QtsiQUgO5u+KgnJKkf4IetRivoWXB5N7FE6CISm1sE8XJM6af1zJDTsT2u5QjEs7devNVPeFwX2nDEMrS3INwFOFy4WxgXuwyE15mRClvoDJXIe44EL7B1GBFFqfPLiXYnOqWObBWAyYOfuFcaJUzEAC2Bb5aD5Gl4gj\\\\\\\/dZ7r+aQDDTlJbE7Sl7czGAbpAas0CHncV9vy5awAtd3iOZsnDyYBDioxcxPQ4K\\\\\\\/O+ts3IEdT2nB2CZYqqkqDM+5GhMeqLaEv2SWoM3Lvq837Rrs+wNm5GTci0P45+ZFwsg7kLHeBtfSskg8hl9+8h+OEvZKI3ZgDVm4PJNINFElO9pgWJnYn3P\\\\\\\/Pw3bsCe0ozjIdQtbwmjfM9+elYaaRqyY5nAnzliOfJNAjVABmvEdgIoPMx6+oUVFE7db3ftsILw05yqmH0xD6+ZHZkd05IL7aljL\\\\\\\/ueM745kaVlH7GigFL9ED7KAaVwDYugAjYgJLeFSAPeHe4KR28vy0N+N\\\\\\\/awBFzlAKNTPBmnUUSj6VrzcNabIJUJEE+vl\\\\\\\/Dy4b\\\\\\\/E5TBvdAbFL4s+rjlkeHeK++TOAskM1dxdDtMzPIT\\\\\\\/XVEhF\\\\\\\/wSF2uKcDRmgnOkWsRGHRUhr0eWUwO2ZDlZakqevoFB9K0\\\\\\\/XqD+fgVO7t+WRI40BPvwYlIaCE6cxxvEbgRUqLOh5bHrApTOHrvCaVWvcGjZwWnqXtigt6jgIU+ohVqsDv3GIvKrSJC0KKRoDol9umasvZduzo4WsGilxSgUkC8UfBveF1NsHgZKBKrs2dfFP0yV6EJgaqj6rrUaARnpyKgk\\\\\\\/qqHR3etB\\\\\\\/m5H4AA41vZKjrmiWbpZ\\\\\\\/JS7a9+GtC2dFwDrG3Q1VFYWnrKKlTe48yBz\\\\\\\/EGAg9om7X5zyCMtw6MMCI+x5r1RAVJhkpdhy69EvYdM5csk2R3sIBxoWrPu4Yvz6pknlrs05pZIXXud5IzQcqC7Gmo1JRX\\\\\\\/fYuuR7fvNDx299GKNewspqnZ1o5BVdURfyjlYXgr8SCfQ9yauWBFZpRb01L4iQxoqhGPTrTIY6TaGHBn\\\\\\\/FzkqHFJBCr7GcLLohScBzKAuhRG+gQ1vEnwbcpSZtH2J7E+IqlLlqFyrX499Tf1wc3BByTqXnrb8JY3ORw21hSPwD1sxkTe8uB6wotmQDTYuWXhUlGejT\\\\\\\/4gXT4VoT92uhxU7B1xcKa+2zBFJSeip+LXr3YqIO8G0yYSNqEm2e3xawPd8EZLpOHQ2E789kTd\\\\\\\/MMBA4tYgKA3ARrT+LmLRLnfVldseliPTiJUjR+Phwb2cdVqjU3PZ5kFei6avGCuenY2bcfLxRRbdo8XfQ7saTo25iu3c53yZzJXnxHJ+TC\\\\\\\/gU91Cg7LyoPI5hzd\\\\\\\/7qP3ePFLmO4D8+pRrBgxsNMEb5Pjsy0F0IX74IYZeoHEphz74LK2pmcSd+0gd93r8jn2+wJLq\\\\\\\/HRjKT00jdcqXiYSU325xlzjOK4b7g3vfF\\\\\\\/R45EaxlqdOYvAE+LYgtf7aFJfhRLiZv9F2kDuWRhzX7PDyUktX78GMeEzIHaoB4qRBT9Hez8MvXeIbzoLuTyEJamrxnjFYeY1tlT26D3luHgcG0H4N+1nQKM7cuEz07V9XSOiMnt\\\\\\\/8Bh1me6J8HZBsWIFYRupaUziywmp\\\\\\\/CtvJzWkN22NRq5LcG5m39cHL9Ep2cglIE+MCqBoYrxNlMXKHXqSdgqGhnb70yaZKKZ92wiCqS8RANiEZEkFH971y9Gd1i6Ltj29+wmrZKZqLhcl0+YwE\\\\\\\/Mb5rYhz7AxDNxCy3JiMBIU3vl3SJPCbco69iJQQp2hHNAVHFvALJk44kWIrSuOxOKFAfxE1nC1Lo9FbBK5nbVAGFcihxqFnJl\\\\\\\/vOQfUXfgF+2ztuZrt5An\\\\\\\/b+tgja30nj601UYkuMsoVXWeKouRcPr5unXuLnWu0Fma3gqo+jLAzsq6+T7Z5ltfUkr\\\\\\\/AUIXOTJFHjo9k+6CD4CpVtfaVT590UXfFYxvOb0tdoADekC1tQ\\\\\\\/loj5csYJ5AsV\\\\\\\/KRaaMt7sjtGSa7wNlqb+YzS4TIU3Vf6w\\\\\\\/iHq0YFZW+YK6yfKIMN6wrMzcLaLHLvRgzKPpUZMykk8c3m4vIWMxqtwvK7jqYwefopoObL+Jhb2PqHjnNkmCBjaK96s8N1TLytih7eXJF1qshcUQ05OQaZisxckK2mK\\\\\\\/fMvxoOnANVnWRDWMXT7wPS5M3uMjPoHsTXWbgVPB3JcgAKecL5J6Dquj8AdhhnNVqPjMyUF5TLemC3Dmy4d1561ydFZLdt+Fm\\\\\\\/wxlPPN6aAOyh6afyU3LfW\\\\\\\/GTRTMyJ5ZZ\\\\\\\/8GXZEkMcg38NEXe1F2hfmNA\\\\\\\/ERH5NRHeTjbtvaFjRXw4TVa537RGTn7QbcZieYew0NHX+DHNDndqCQNlmbBWiVXqTFHjkH2yuCxakb8XR+fqMojICuj1Wu8pgZGFzw1j78rVb9bMhYA+FZAJsHiFpA07R7KrIUyyCZgyVWBV3IIwS1A7cahMeyJJdYOVvJ2aKsCPq1+1Xwzf8Ea5I6pIYlYDiMt10JrW7XvwydQUIm6V2mTnu8IrELw7GARjsxay1WYNrRy+dvAtTyAOk6v3YDvuz9X4\\\\\\\/Ye+R60a11Io\\\\\\\/STzO1g2iMgAStOwWPgZRAqj3q\\\\\\\/STL1EEdBBsDABzKu0OUDl\\\\\\\/9eXha38LIVH4Vq\\\\\\\/lYaNP3HtyEp55zE2qxZGzbh2cazOlno7BgMsYyhlGRJt63fjoUZQPCucmZPrc\\\\\\\/WH9gtCAoUFPmJjcRr8uEkiHxSnrkQfKUrZfnneI372ACU1XRFp14dXW88ELXVhmd3hivvEZiON2\\\\\\\/QvJxt\\\\\\\/GhJL33j\\\\\\\/sVl4kRaW4TbVdrYpXZqhBOii0DuZT7T92b8L9MExNrzDXr2itqBNXn1IldKR2uZmbQyjo\\\\\\\/PItuYvaK9MRI59S7LdWUtVcg9OoGoTjOTvJ124QR0bFyftSCTpHq713gmJqsjbjkjCy+jF7sdfj0BXmh4Cn197V5JcTUMVOL6lJGicR2SeCY32fI+ibTJ7pUvFdQTEkL0tsw7k2DLxkhqoJ+3tuN+ZeMI4tYOzZuwhOJcFZKzOc8uDznZTH8ERVyZ8AOn3UyngSE5NacQ0xNpn5RzVpZcDVMY54MgHw0z9f2Dz8FZnkAi0UkGlGDNrFLPRs6o3vXpgO48RFGZGM2QEHK6sUeNzE55mwdVkpvgl5\\\\\\\/fHKqmmtJqtERmmMMTbWa5KxRycfDp9eNaYKLpu\\\\\\\/ZP3h7RfPD6vZb8+qQCEBierry9tyIFqWMvPledUXzpjvzn+F6gp8hYKSo9TSnGKTJLF1hA+1fEJcy61xOR8iWfUYaOqQZF8e6kYUlZLYlZ5UHJ2i8CcvEoxZGgqPKJ1MIn4EpfyA\\\\\\\/yxhWueml+MO3V9fJb4xwpvbyAqDtmqdKeaNaGSJO8of8ViF4dHPwoZ1t9ORFbPlPZSuPkYzkcl1rolhpg4t4Y20c+SAyImIyIIYFp7V7\\\\\\\/i6s4VITBh6cpNGzCBAzId5fF9c3t2+QxppelwN7ZpwfQoMn3h9LQwwoa\\\\\\\/362VHqMR8XnoJe8Qys2GESM00aApYU7q7NtqsIH8794iMgvb0SEbV6vm6rfRnZOx+xbvFBkmRCzXVhyI8kWPn4MzJf57O87GprM96kqcHZksqJ6EO0LEfFL4i+URbcbDwSR6nHwbf3h7Upb5+rA+LlvrAP6SZTncdr9OyNC4jrQ\\\\\\\/KjWmjwKrbMVFi8V9KkJL46d1mVcKF8ETAMUuE1CaNeTynNSBRmc4yhyZAKCvqSSmnLUiDbRXcm3G0DSd+f6HdqCMm\\\\\\\/M\\\\\\\/YUOVqiHpmFdWPfrBk+Tegos1RyP1l4mbpUehP9pbNzsKQQZQ9KkxbALcIIrmDF8D4GPVH2riNj6EHEL7GZQn\\\\\\\/cQjkTZ8oSf3BdH91CqUaupvSNN6whpl0hjQECc\\\\\\\/DKNEe9U5FN9KRMqOi9Zev6yKLH\\\\\\\/KKACpRgcZzQaWeWDh7CT+wiIjKk0LV2Z+KuicaUT7imDjynS7ovrnFPs4Uj4wYEAy3j8dza3PXek82lzficx3zzFKKqlZHu92ZHJ4SSbSNBgsUBzzfdhNLEvuQstqVtLaKQeALkah9I00HNMHAgM4LBYi9SlbtOnu0sLsiz9GNqNIyP4YtYJAEet3oSEmx6248HQ9oDVK+OxLi4rzLCwavlcQEowixXfbl0Gz6DIZviC7Il3UcCYsUHGbGZ64f0ZTyPSjBgnkyqr\\\\\\\/WjFjPIWYCtcaDzle0WQukv6AQa6PzanwVx4rxyTsvUz8\\\\\\\/wSNrz9souIlZLF133A\\\\\\\/t+CCOsfuDDH170DGSXPcsG5P5OJPMOwSNKYLqbnkHw51Hcz3g8Nejm+HXCKlrbNkZn+b+FRRL+UT0TVAPql5mAV9r00CvHwthoWqAaxZHHJ2IwSQofgvyHR1wDvUdXnh\\\\\\\/bmeejGUfrBJO03rl7BFoh8PcvTlHNf3KRDygFjA1ABnjeHh2Lt3TWnKJ85IoglMN0ku6HOoQf\\\\\\\/5FZZtUYAE8BgggTS97BhxZEgUIgn\\\\\\\/rhfyatSz12fTldjFLykqvkngmVYXqGovBlmU0v500iYvMX8+UAzNPiyOFy\\\\\\\/MVyIjuP3GAFK+4IdB5znfVqgKGU2hwn7fCw2CSbdf0EbkNpzLUMhRc8k1PWPLij3lgM3IA9MwClnHgByvyzCQNzoJg6mYe299sDu3OvpF6kscmYw2dINzAyMqJLa5NfF0Bqu8wngUY52RgDc7B1lofezR6cfR7bAJmjWraWV\\\\\\\/O6h3vd\\\\\\\/kwmD4TREkgas3WyBUj0iPE\\\\\\\/gWGb2iKmqHO2EdJx3URkhmLu5ojXqUMq3p7yU5JcTF+FPGODsTrvu\\\\\\\/1KQpoDpqpJ8YZNMczmj9cIVh4q+FWwotOlai0+t1uhF4oPYn2wB6u699pkInlPUQL2R0wuYFb22fvYFe3Dyj9g6H8ztJyGXSp6GsWAGMFugkcIViZ1eumfsuNthdXT2Z77F81dsw+0bCDhsCaZ6WAeyTIJHx\\\\\\\/AlPJ\\\\\\\/kbo25tnKiFEpdC3SFj4p8U96VafFJ\\\\\\\/Kb72ytS3w+vhMAF\\\\\\\/I0KOLg7z9zdCToi7u9REsuStB960Y2sT8z5W8naI6QA0lmuXpCsjWDrMzzGWnqJU4A47nWDVa+vy4S2E9tqSCRxpj01i1wDdvI9Szq4rFqLzTqfS\\\\\\\/cMOYCftVlhuLPyjIDuCdwifr\\\\\\\/ciA5u+T5M3P6YipUJrh\\\\\\\/kb4+NutJ73xOr90lvQW+PoG3NS9X6TAOEIAwCepQaTFPGYv2gf8WbxaZZyHOh9mbPjC3BQFs7oSBvDLt6yRgYV2py6Cm3oWGlZ0aDe+5CPm8aC4erwjl+YwkBWlZG\\\\\\\/+jNUQIKvw+KP6VQOThxOd0AyCUmwXgxqSNRs\\\\\\\/X\\\\\\\/8UN4nAndu\\\\\\\/Ameip+1p\\\\\\\/eahNJ5C9BWW87wcELoxp4SpL2vFTbTyrtRHNC8Mjz+GcY4OaR5Lkxx2e91WmH2QRn8XnjyC8vYh4wNJEZkcS4ibhMv0X8Q8pBbBw+oDYWYLHZWRa9nCBlhCZ\\\\\\\/AGs02qmtibTEQW7IgDU1TwvrBA1ke8CvndOWzbvxhRsOdK\\\\\\\/Bk7\\\\\\\/w7pXOCcz2SsgEl8pF5txKs26bCZ8h6ewyrdK4lLBnuXTmGYfajbFDLZuSlz\\\\\\\/itWne81WpF1vluXN6LgQ\\\\\\\/ZeVE+wrv+dh8QH6BaxfU35KFL4NjlscazyzPpgpFi6CURPeMJ0yLZgx7btB57ENu846RtxQClURfu4hLU0m+w+ya87EQJrHh1hbO510MltZlynWwO\\\\\\\/A1X9LEqh00QFHqqE1yxf5SULZ0iKbYMeMaUPbhESdg8f1ZWReLRV83Gq6D1gQmtAO5Hx\\\\\\\/WJ+aSVn6gV21+moREv1IgKPaR\\\\\\\/iaYx1lr3JLgIXIu65vth\\\\\\\/iNAiGfH7Fk+OD8Ak+Ri3a8lcACTT7XfrS\\\\\\\/JdkwWk1Fo6HMYLq9F830BJzMZAGeR6\\\\\\\/BJgJXAF3Bxv0YaB35B+\\\\\\\/MXXkOK+Py13BtcTZzUoVwGDXDt9WAmm2sJUjlpBD\\\\\\\/eJH3Bo1XmX+ztN5l7c6EMbtmb+RV7p2yN6pQM+\\\\\\\/Q7vFXiditHcHAw+TXHaNVnrIF6yge4voG\\\\\\\/t+5ut8GIgrYvVvlRmbdRpPb35uwoFwaBaymF4nNep2W42JEvrHduw3O9fbHgn7WCpv9DXYfl8Ph\\\\\\\/PDi+3sHIO7WVD3zoU8sBQmBQo8R817DrfS17GGS5fsOzbXui80FWfGfFsKkRMsc8\\\\\\\/sW+vcTYOBxtUTeN0mCObuGhyMgN6hjTgQFe1xETQOsgeJyIzYzzbZx3U53VIQuXdtgP9Dw64JvTa3SHYiOYLqZXFqFBl\\\\\\\/deWe5IqrHyb2xKL+KModrwMhk1OosWmkqC\\\\\\\/1I9MaIKRkQ6szJocP5PeSDNZ7pAwZV6ZE0hQCWqd8pjstjnWogHB2jPamWnpDfSFCpRbEBlqohxsJ+TmKpW84NHJiVMCVygCRkV\\\\\\\/F+hoaTJVyK2IwQ9sz8CtQyE\\\\\\\/7k5xl+aOtNGSndMiUbqPE6vd8qWoKJK4ypf7mUk87fCbFJ+MFS3TU4HMdVN0WpwUdv7jT3wuSaMtkHlkXASW7gpAffxpobbRJo2I96nz2mKLBl+uB7v7WSxoxlv1mGcuZPhA+GvQCul0Oz5ao5qwgQmbzIHqwzKY\\\\\\\/V+Gs\\\\\\\/WJsSSeRkmEso6gwxpbe\\\\\\\/VozinTeFET8rtxVDYXM2aSIOioUQHf9NzfHjPe5E7U8iIAq2Y2mswaIPGeVCVlBBvFIKFMXWSr5Zch\\\\\\\/zVPi98uzu\\\\\\\/C7U4IB8nstMZnOvdpdBwVC70HjMGf06cMHd+y4fWk62f79BHtz6bmBsAsOs3mmRE59AAZob5JyQJ8ny+1yyKYBH87CggqRcyGul\\\\\\\/fa63srQ4DdWdZ96qaZoZSFiZnZjjcu0Zh70QtxV48miF\\\\\\\/1VJ+u0hueAa24cO5lJXUHymCayIN3Li7Sw0yZ05MNMOZwyQ7b07xfvGhS539574TV5\\\\\\\/pG59bUNQdFqiVmhYDfzpfpyrSPbz1X48nXW3+x+dgZ6gKGA+ae\\\\\\\/QUKo4fG1eSZCAV1HlOEW9vQjh3QEZJbw5AnPTHnx\\\\\\\/0QegsmEVpQTpiqWoPBixstyEp5vvUWmmU68OKbSwSI\\\\\\\/VUFRglo+wG2kE6Unf1pWFcTl6fuZ7t5ydyXGEYeXsjH1ndJs6CFYrW9Yb3bj1yMi47kV28USXREvYBMx02aiTE0eROMSDwgJmhkvp\\\\\\\/+u81b7sZoGU3h2bSe9YMChTZM2xmSDMUg9VXwuuQnWzfa0sShjNLTu5m20Xda2p4DTgxEOgyyr3ChiN2+I+ZOdAKw0ry8fZc3M6DhxCucn4ufmEFohtma5i65wIPK9wcefc0WgJP5IRcG0c2xVx1SlHBg5koXBlWbkVEeMoX0vRwcGT0paYRpoLzFxh\\\\\\\/fO92fndubxt\\\\\\\/W3w8xBef\\\\\\\/Awm9TPvssrlG3HWrAjDfu2Me5G+wmAPELVsUWKskx\\\\\\\/+JbnZgoPkETpundUenyKS7wfkMqf20sg1WAf+nX5JCM3mHt8O4fKXFVcADkgWkhhYxLku3beEI1wh8sozdeRpU5AYYxI59Nmhmgc9pUd89JeVw24K9Tc4\\\\\\\/ZsmD5Cp4JNMbO5PzHUI97kzA1cGCfdW5x1CIn3yU2tB+lqivDwCfE1Tv5fFEvIakyYz+ITqXwmoo450XIhDej9JdieDsUrCr2MwtcrUFSFq2C1ic4YmKr7gF8chpomfxTwfn3EchMBNBj3OhnDWG9YUDfnsZGnelz+zb+D69wUD9oID80bbOvc5raA9iJ0CCE\\\\\\\/XychiVYn+9OtY9bBQEy85S8MqnYTz9QcBAH6WNxoB+GdzjvbVyqX6aov1BYWsNmrbpS+tkNF7nesDClUQajZBM+xXZooE1SQt8ZYeABHXU9ZsN+Jf51Ju8fgsJpHCy82xBZ9cG8qWj8p0soxO2tIS+XzhL0gkozbVft93k5Sxf0qox0CM5LDtZDYtHObs2iv\\\\\\\/gCuyoscbF2VRL\\\\\\\/1R0mgrjr+ks+ZZUgxJiQexLPkxpmUheFDM19L1S\\\\\\\/l6Aewd3Vfz8jIKsh2O0hGr6ClEBYmzxEGNea5MTQcVqtilO7jw8Xiw9sk6o0Y1AQ3WfHKezXboIJn5GO37L2GDU7tCCFEsb53OGrEvQ6GGTQZvaVkyfQrw\\\\\\\/N2vlSudRqhuE27vDaWFMEHC0in2+gIUqfNg4rJsjXiLbl1cG6n5N2VxmBYeC0rXJIa\\\\\\\/exPaBjZ3BwniJgbg5\\\\\\\/TihmCqjalANv2LkNNODPo2rff4YQwHI85YZGqVOwARz0ZRGh2fVEioQMKcVp\\\\\\\/U\\\\\\\/HcjtDZn44IDgd8KgyYae7fng4uQrd3bdTm3HQrJDBzJBa84DMhte66v43Md2w\\\\\\\/Rq+RMEWDjC+eX6sOWvFQQuRJ62L8utrw80Xnx1GbGOmy+lKUYXLJZDOhALy\\\\\\\/4Ry2U7YSk4A88cG532z+VHfNDc6M2RMfskcZSZtsx9mWCDXdZeJaRgPs4lhq80xHJcseSFPFpNqgzFdV3l2entI0GnDcFC23sQ1FmjsnEiMjuTf+fl205ivIEYQbZN9j3kDHvZ+d3u8soYeDj8tkwyYMplaFk+Iju2QI62xfjYI2CXYvOOJIk3nTLjQk3ZHZW9sjfPnmdqefRI\\\\\\\/1EmI2vhLKWmrFEEErdGd8gpGJZ5YdpRSRya5GVZ9\\\\\\\/UCsVhs7L+QUc9OQ4qkBbcZ5OHNvjQFQfD5O3UPxeDBBh8f06XPgiVpfxmeKD5t5U\\\\\\\/IrShnMCwCG2I279F6CC\\\\\\\/D9AIt1BzJCUCswdvaffXgZX9tsG2LSVqxqqn+wEuoy+iyC++gcynIzi6vJEzbJz25wVE+7tUc4qdAWFzhXvLiZrXGNKBRpgY+RtbkXJvYpNEzTvbTYC1lQ0aVyqAah1VpTnSTwzL67DR2ZgIuP++8VYiyRp7fBEiJuw3sOlKW3iPiYZEdr5nS2fU8CxUt\\\\\\\/iQXKI9\\\\\\\/R5ghyypjPjSI4el21kewL3leE2Cl6\\\\\\\/H\\\\\\\/gAbs1gdePo7+f\\\\\\\/oP7k2LiwLqC9UQpUUej1XicpZ7JPoqRKGnYSqfyS\\\\\\\/0WwFUzujSPPkG3r56ulOqJszYnl7oS1VcEus9J6fRJ\\\\\\\/ov1U4wWZU8LsfI2gtPMvC78bnQTS5ZKP8LIYSrowZEmdVh3rMW5M\\\\\\\/eAbfWL4rR0Q8SOo53T0eXctSLGD\\\\\\\/FSIDCFzC1kx2ZYS1Afy8HfODYiiPwZ+h\\\\\\\/ZQR5gDlEhpON+pC1cgeAvf8sR46gHkS\\\\\\\/CxCTOpQnK2sZHMpSvbDk2vAEhJqcc4SWFIV7L7lHVwLSVeE73ktj1n663STNtDJT6rLnx4HfHKoGnaUYJidhiW6cnNIzCBz9SWAL8+fxwXRTrxpEEr12PwAl3qKV92xoEvuQKKNiwEMBvABVdv0ywD8hLJ2Y0VASLtMkqFjEZCnIIe2VdXhWAn0P9qzZ9zUn38AFW1ipAfXYux3FBMaUXVgQuelnVkUXtQXHwy8\\\\\\\/T9wdo7qp3mIe96flaKHSVnABJCOowHKblDXgkxAcaoQYDuHVT7otOpllILFM+d6UNZamjvm8bBb4Nzptrk4k3kcY4JQ5mgMjaJQJ35gn7p1th1Pmt1ZZGtMM1GeJ8vvcGg5kr7+PWFroTdOYtuFgJTJrbAZsN1o3qADC2zxoiDzW+fSaNz6bUTNYUtBDdeVy21fFfPcnIKL1p6IcltCncjd1eZOjIfi+JxOx8JCqHq0IqT84PRmsUMeQFRjNAg4u963ApjYTrbpWuwcHlX7\\\\\\\/dEwN4EZVnfBiO1Bc8lIm31r\\\\\\\/u0qy0qRSOO0G8ZTg73u+5O+mEpDGM+E\\\\\\\/KGP8StTeMBYaVCNPHydI1NaervU\\\\\\\/zRVCvredZuBlp2brvwH7KhPiCypLOkDHV9Ws4vt7Fw8yZcBkNCDya6Uab8Y4hf1lTDj4WeVngcCr3pRNPtjAe\\\\\\\/HkBDnjNYulfn8RxK1QccPG3AhM8XfI8najR\\\\\\\/JcK41+rDXwdoFVQlwHe6nJ4AOJxfxDSzfwXUJ1LYx33woFhEnRtSUUhrMRhvwRp3e\\\\\\\/tqWaaCp1+n3SABaaE9xxjkdsdxY3d89JvFxQrPN3QvjF86IN\\\\\\\/mYo38tHsJjp9YABxoStN4l\\\\\\\/8DdaEQc2GX3dfkWRLjt6fNKgsoUcj3y+p58P6RViWJfl3cu1VnMZj26IYhzS3wJ6IRm+jjnjAnISZLDc1m+BzTU+78WskFD2HIpr2Fj36O8J9wvYXuG3q9Or0W7AyfQl13K3vjsIP7Lb5fJ2WOk7xedjUtAkknTQSh8HmAq4HNdwHf75cOKON8Xqu3mFiYUbaP8DErhJE09lklXu74t55y+vNqd5DG+ldWnjw1kIdonj\\\\\\\/irizBrS7+DrMSHxjCwT67T\\\\\\\/GwhgVNI7I\\\\\\\/7sADlXVWHjxle\\\\\\\/nY4ekQ2blNTArxh0LuuiQLZo6jShbq\\\\\\\/2+UAyxaRCAnVZlVLRaf\\\\\\\/5DOhTmAlAvuDaYRJbL4E1BkKKkNeiY2XTHCJmJgg9eDF9oY++VyY5Rb5nMWH8ygB3yh7vhflekb01LBhnul6eY127ZtnLyf+aWPIGHFCTfEZD02atwq+U+uY+n4IcZlMKqkfQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022bf8zMLCjw2JRPl3+zHmStw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022d15077600cc5a452f294aacee265b70c2dd9a81597fe71c79053ede397164e2e\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022NgTOeUJZX4aJtKVkQ0j8RxtV8hDZWgDT2TPUDiwtdh1sqliTQzY1uNFfEZVJu4pXFStu9APN7PWMV\\\\\\\/J1nsdcwuom\\\\\\\/fK3k2Bt\\\\\\\/ugFybDSyLLEvgZsRNB6kklqRKhhpo5uiFKzuQgsLiFfgqvKrxK\\\\\\\/WUuliULd\\\\\\\/XDdw5q6DatoJEx30ikY\\\\\\\/0jYxFpBuWRVtjGD1sM9\\\\\\\/9nt53jyKN3woLJ3G9+o4gEzbZYBUvSCGBPSyCaUKKr75P\\\\\\\/\\\\\\\/ey+\\\\\\\/V41nP2taTA42nO0qmtU7n3q4SKrRAkv72I9csEDkLrooLV\\\\\\\/xNeJwsrezCI7yJuYm6H+MY43hWNJTtIBbSPVBHIW701H8MgyPWMbe1kQefnzQVr4JP3YE3OSyVvPjeD1UlisrErah22nSGgK\\\\\\\/Q6Db8iE9Z8\\\\\\\/Zh6MMnx5CeLH9Z2bXyffa67iuTv7WfOa3MFGs\\\\\\\/d4PqbtllwW5gWxKz10QdzK2ZNwdDdSn93qUPe4HO2aEt2534+BWKWm13E5+XfA7foAOU\\\\\\\/WQmnWzcDb3ufR6HAjrYHxdr\\\\\\\/2QMPJgjDqImycWDmFuW6zJTNQRhyTJdv\\\\\\\/VHMW1FPYX0VR2pC8XyLrBIJccW9KY+SuCpII9J5Sog+9qMRLZz9ANoe+v0Y59CGgInXrVEQ2bCZvIX6UHe48CjrBsthyIL9iCyUJxtG3qRBD2byVD4Iv2bKFJJCJxOuo6oyuZtxtjBZXK39yKdYfWKEg2UzP\\\\\\\/QjnmGifcHYE1\\\\\\\/stbKnIJEy9UzEDJ7QWRQ\\\\\\\/m9NoNkQAliB2nz\\\\\\\/61OWCReEm9d\\\\\\\/lBQEuUQseLr3IlAue25CCDim9yD\\\\\\\/SYE0QRownnJ+Ca5jdLAqL0zzbtxUzzCubPcYrsAd\\\\\\\/7yWGzgu88MjzEq3fXH16kWkgk+BKDlNyWdzh+Hf1C5CtGbV90Sjb6QBKxJw1cvH+dh5AuwKZjguTr9iNLPT7KqW9nNP21n078OfinI1FeUY1LvRx6\\\\\\\/MRYeaSAqzBjUK0mi9goQ7GOINI27Qarevp5xDP4\\\\\\\/agIXwhTMrpNO4yI7QXYXx4SwvufM2CUTK8Wl8jWCgZV886jN3kYz1Wr3mP+jfgdjY8qJTe65vgnm21j7KJJB\\\\\\\/dr5Ajgcd1VLdHNmM61KAu9u\\\\\\\/URh4ejcZkVjLNfpfXncFkyi6fReQOWx1eD8AU2JaPrTkQCMLsH9Oc88X3KiK9F1jwezbQs1IsEkqFExLiGMLWIiv0vw7CfmlYgcztX1+vw86ILr7f\\\\\\\/6QT1NZH9TLgR2NEyVaHldImHQxOo+AVNLa3C7lqUPWOXOKi++pnvwABOVqeRgmwhz2XnJUObwhRqCjOKOrs0vzpv6ca7X4rcly7XwB+7VWW5eV8kTzrJK8fi2aHHkRhIiclQs3R1oVgLOJPARB70Oo9Z4ASldR1ylPmP\\\\\\\/o81tosiuOEcDFzj1OYk7Gy+SYYETRtUoJ2pp78z3uzNLnAdEgNcedutsmx4j08QjUJMYCzKnPVjXdjA0RfgB0DoR5ZincBNm3c3JfL93+55DWwubsmSKjbNNywzjqz4IrDuccwTugYThO63yY9u2uuIMDrriyO73c+fob4g3DfDNWUNgg3PXXBgeHoB9+197bnYZ\\\\\\\/4DVcf3vtc+CKz+SFAJRJORRRFKQw9hT8t409rycFuhpWC7yev8u6RsgAaouYRjOTJaCLQ7+AbyWc+hJyTgGDi6O4Cll4JKJtswavVv4LWcohawMlyReM+I0vyja9zci1vcEiExh0GczHtFNYMew8boOJ9NtyQZLT+73CUd7CiyA9Eh8zmTIPvxA25F0s6KHVVSJITjuvSnXV\\\\\\\/32LPG1kdg\\\\\\\/cNt+0pcvoNtb9HSr54D4hxFqtMu+gO9JEAvlEN1ZbshEAyoedIY+S6VDDQAlarY4+IdrEspXXw+SJDjiMxlN9cmOmWBL1NRaaVUxqu5serc+A6wXsbFjB2gFuG6Da2LfR46kHCAEe1KTf2pver7OPIltzgSsb\\\\\\\/PO8\\\\\\\/5RXXMHA5umweCBWE5iNEtdA5yo9MqVIyv89g==\\\u0022,\\\u0022iv\\\u0022:\\\u0022KtBTN3YAJFzSJTA0\\\\\\\/aoNVw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022b149a43a3b894d656dc5eddf1b1030f1f4089cf6b8467560103f1c19e0ccfca7\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459599 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/12/2151" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form--RtLoJZRW7qfexpzj2pETTxxYHP5XckTGGa9W1QBQtk" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/12/2151" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-3th0npmomL9j_V-PBVj_AmVjHe5R70npts_dTK4VygY" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/12/2147" title="MAPfastR: Quantitative Trait Loci Mapping in Outbred Line Crosses" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/12/2163" title="Genomic Sequence Diversity and Population Structure of Saccharomyces cerevisiae Assessed by RAD-seq" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459599" id="node459599" data-pisa="ggg;3/12/2151" data-pisa-master="ggg;g3.113.008276" data-apath="/ggg/3/12/2151.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Detecting Selection Using Time-Series Data of Allele Frequencies with Multiple Independent Reference Loci</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Jo Nishino</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2151-2161;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.008276 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459599" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jo Nishino </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Center for Information Biology and DNA Data Bank of Japan, National Institute of Genetics, Research Organization of Information and Systems, Mishima, Shizuoka 411-8540, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jo%2BNishino%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nishino%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJo%2BNishino%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jnishino@nig.ac.jp" class="" data-icon-position="" data-hide-link-title="0">jnishino@nig.ac.jp</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/3/12/2151" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459599" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459599/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/12/2151.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459599" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459599/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/12/2151.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459599" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459599/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/12/2151.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459599" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459599/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/3/12/2151.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459599" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459599/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-corrections" >
  
      
  
  <div class="pane-content">
    <div class="messages highwire-corrections highwire-correction-type-correction"><h3 class="highwire-corrections-title">This article has a correction. Please see:</h3><ul class="highwire-corrections-list"><li class="relation-67639 first last"><a href="/content/4/5/941" class="" data-icon-position="" data-hide-link-title="0">Corrigendum - May 01, 2014</a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Recently, in 2013 Feder <em>et al.</em> proposed the frequency increment test (FIT), which evaluates natural selection at a single diallelic locus by the use of time-series data of allele frequencies. This test is unbiased under conditions of constant population size and no sampling noise. Here, we expand upon the FIT by introducing a test that explicitly allows for changes in population size by using information from independent reference loci. Various demographic models suggest that our proposed test is unbiased irrespective of fluctuations in population size when sampling noise can be ignored and that it has greater power to detect selection than the FIT if sufficient reference loci are used.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/selection" class="hw-term hw-article-keyword hw-article-keyword-selection" rel="nofollow">selection</a></li><li class="kwd"><a href="/keyword/neutrality" class="hw-term hw-article-keyword hw-article-keyword-neutrality" rel="nofollow">neutrality</a></li><li class="kwd"><a href="/keyword/time-series-data-allele-frequencies" class="hw-term hw-article-keyword hw-article-keyword-time-series-data-of-allele-frequencies" rel="nofollow">time-series data of allele frequencies</a></li><li class="kwd"><a href="/keyword/reference-loci" class="hw-term hw-article-keyword hw-article-keyword-reference-loci" rel="nofollow">reference loci</a></li><li class="kwd"><a href="/keyword/population-size-fluctuations" class="hw-term hw-article-keyword hw-article-keyword-population-size-fluctuations" rel="nofollow">population size fluctuations</a></li></ul><p id="p-3">In population genetics, most data are obtained from a single point in time. When genetic time-series data are available, the use of such data to detect and estimate natural selection is an attractive concept. Time-series data may have direct information about natural selection because they affect allele frequencies in time. <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Bollback <em>et al.</em> (2008)</a> introduced a statistical framework for estimating and testing natural selection by using time-series data of allele frequencies at a single diallelic locus. These authors applied their framework to an ancient human DNA sequence (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Hummel <em>et al.</em> 2005</a>) and a sample from an experimental evolution study of the bacteriophage, MS2 (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Bollback and Huelsenbeck 2007</a>).</p><p id="p-4">Recent advances in high-throughput sequencing technology, including pooled DNA sequencing, have facilitated the acquisition of time-series data, and <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Bollback <em>et al.</em>’s (2008)</a> method has been extended to more complicated situations. <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Mathieson and McVean (2013)</a> applied a lattice model of population subdivision that enabled joint estimation of migration rate and spatially varying selection coefficients. The allele age also is an important parameter in selection inferences because it can provide information regarding the origin of a particular phenotype associated with the allele. <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Malaspinas <em>et al.</em> (2012)</a> developed a method to estimate the selection coefficient and the allele age simultaneously. In addition, there has been increasing numbers of studies in which researchers focus on specific settings in each evolutionary experiment (<em>e.g.</em>, <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Illingworth and Mustonen 2011</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Gallet <em>et al.</em> 2012</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Illingworth <em>et al.</em> 2012</a>).</p><p id="p-5"><a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Feder <em>et al.</em> (2013)</a> reported recently that <a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">Bollback <em>et al.</em>’s (2008)</a> standard χ<sup>2</sup>-based test for selection is biased for realistic data with few sampled time points. When the number of sampled time points is sufficiently large, the likelihood ratio statistic (LRS) follows a χ<sup>2</sup> distribution. However, the actual number of sampled time points rarely exceeds a few dozen. Particularly, when the null hypothesis is composite and the profile likelihood is used, the estimation of nuisance parameters can substantially bias inferences of the parameters of interest (<em>e.g.</em>, see Chapter 10 of <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Pawitan 2001</a>). For the problem described in this report, the nuisance parameter is the population size.</p><p id="p-6">To avoid bias, <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Feder <em>et al.</em> (2013)</a> proposed two methods that both were modeled under conditions of constant population size and no sampling noise. In the empirical likelihood ratio test (ELRT), the population size is preliminarily estimated under neutrality as a first approximation. The estimated population size then is used to generate the empirical distribution of the LRS by computer simulation. Neutrality then can be evaluated by comparing the observed LRS with the empirical distribution. Although the ELRT was shown to be unbiased, this approach can be computationally intensive. To reduce the computational load, <a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">Feder <em>et al.</em> (2013)</a> proposed the frequency increment test (FIT). The statistic used for FIT is defined as the following: Let <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-1.gif"/></noscript></span></span> be the population frequencies of one allele at a diallelic locus of interest at the sampled time, <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-2.gif"/></noscript></span></span>. The sampling time scales are short compared to the population size. Then the standardized allele frequency increment,<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-3.gif"/></noscript></span> <span class="disp-formula-label">(1)</span></span>is approximately normally distributed with mean 0 under the null model, that is, neutral evolution. The variance of <em>Y<sub>i</sub></em> is equal to 1/(2<em>N</em>) in the Wright−Fisher model with <em>N</em> diploids. However, the variance is unknown because <em>N</em> is unknown. In such a situation, natural selection can be evaluated by letting<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-4.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-5.gif"/></noscript></span></span> and <em>S</em> are the sample mean and variance of <em>Y<sub>i</sub></em>, respectively. We then perform a <em>t</em>-test using the fact that <em>t<sub>FIT</sub></em> follows the Student’s <em>t</em> distribution with <em>L</em> − 1 degrees of freedom under the null model. This is the FIT.</p><p id="p-7">The FIT treats the nuisance parameter, <em>N</em>, as an unknown parameter instead of estimating it. When <em>Y<sub>i</sub></em> for any <em>i</em> follows the normal distribution with the same variance under the null model, the FIT is an exact and unbiased test. <a id="xref-ref-3-4" class="xref-bibr" href="#ref-3">Feder <em>et al.</em> (2013)</a> verified that actual type I error rates approach the nominal significance level for various parameter settings. These investigators also demonstrated that the power of the FIT is equal to or greater than that of the ELRT. Although the FIT does not account for the sampling process from a population explicitly, the test was shown to work well even when the sampling process exists if the sample size is not small.</p><p id="p-8">The FIT is a simple and bias-controlled method to detect selection. However, it is not clear whether the FIT works well when the population size fluctuates. Theoretically, under the null model with fluctuating population size <em>Y<sub>i</sub></em> does not follow the same normal distribution for all <em>i</em>, and therefore, <em>t<sub>FIT</sub></em> (Data) does not follow the Student’s <em>t</em> distribution.</p><p id="p-9">This study is an extension of <a id="xref-ref-3-5" class="xref-bibr" href="#ref-3">Feder <em>et al.</em>’s (2013)</a> FIT that allows for fluctuations in population size by using reference loci. First, the FIT’s actual type I error rates in a fluctuating population is investigated. Then, a new test is introduced, the frequency increment test with reference loci (FITR). Given a fluctuating population size, the FITR’s actual type I error rates are almost the same as the nominal significance level. Then, the powers of the FITR and the FIT to detect natural selection were evaluated. Finally, the simulation method used in this study was validated and the properties of the FITR in practical situations were investigated. Model descriptions are presented just below and added before introducing the FITR.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Model and simulation methods</h3><p id="p-10">Let us consider a population evolving according to the Wright–Fisher model with fluctuating population size. The population size fluctuates as a function of generation time, <em>t</em>, and is denoted by <em>N</em>(<em>t</em>). To investigate the actual type I errors and the powers of <a id="xref-ref-3-6" class="xref-bibr" href="#ref-3">Feder <em>et al.</em>’s (2013)</a> FIT and the FITR introduced in this study, we conducted computer simulations under the five demographic models shown in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>. The two alleles at a diallelic locus of interest are denoted by <em>A</em><sub>0</sub> and <em>a</em><sub>0</sub>, respectively. At generation times <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-6.gif"/></noscript></span></span>, the frequencies of <em>a</em><sub>0</sub> are denoted by <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-7.gif"/></noscript></span></span>. Here, <em>t</em><sub>0</sub> = 0 and <em>t<sub>L</sub></em> = <em>T</em> are the first and the last sampling times, respectively, and the number of sampled times is <em>L</em> + 1. The fitnesses of genotypes <em>A</em><sub>0</sub><em>A</em><sub>0</sub>, <em>A</em><sub>0</sub><em>a</em><sub>0</sub>, and <em>a</em><sub>0</sub><em>a</em><sub>0</sub> are assumed to be 1, <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-8.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-9.gif"/></noscript></span></span>, respectively (<em>i.e.</em>, no dominance is assumed). The population size, <em>N</em>(<em>t</em>), is independent of the frequency of <em>a</em><sub>0</sub>. As described in the next section, the FITR also uses neutral reference loci.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/12/2151/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Demographic models used in this study. Model 1: constant-size model (N(t) = 104); Model 2: slow-growth (grows exponentially from N(0) = 104 to N(T) = 105); Model 3: moderate-bottleneck model (); Model 4: rapid-growth model (); Model 5: severe-bottleneck model ()." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1806088008" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Demographic models used in this study. Model 1: constant-size model (&lt;em&gt;N&lt;/em&gt;(&lt;em&gt;t&lt;/em&gt;) = 10&lt;sup&gt;4&lt;/sup&gt;); Model 2: slow-growth (grows exponentially from &lt;em&gt;N&lt;/em&gt;(0) = 10&lt;sup&gt;4&lt;/sup&gt; to &lt;em&gt;N&lt;/em&gt;(&lt;em&gt;T&lt;/em&gt;) = 10&lt;sup&gt;5&lt;/sup&gt;); Model 3: moderate-bottleneck model (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-10&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;); Model 4: rapid-growth model (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-11&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;); Model 5: severe-bottleneck model (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-12&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/12/2151/F1.medium.gif" width="440" height="357"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/3/12/2151/F1.medium.gif" width="440" height="357"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/12/2151/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/12/2151/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463733" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-11" class="first-child">Demographic models used in this study. Model 1: constant-size model (<em>N</em>(<em>t</em>) = 10<sup>4</sup>); Model 2: slow-growth (grows exponentially from <em>N</em>(0) = 10<sup>4</sup> to <em>N</em>(<em>T</em>) = 10<sup>5</sup>); Model 3: moderate-bottleneck model (<span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/F1/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/F1/embed/mml-math-10.gif"/></noscript></span></span>); Model 4: rapid-growth model (<span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/F1/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/F1/embed/mml-math-11.gif"/></noscript></span></span>); Model 5: severe-bottleneck model (<span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/F1/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/F1/embed/mml-math-12.gif"/></noscript></span></span>).</p><div class="sb-div caption-clear"></div></div></div><p id="p-12">In the Wright–Fisher model, the allele frequency can be obtained exactly every generation as a binomial distribution. However, the generation of these data poses an extreme computational burden that is impractical for large populations (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). To avoid this burden and simulate changes in allele frequencies, we applied the pseudo-sampling method (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Kimura and Takahata 1983</a>), which is an improved version of the methods of <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Kimura (1980)</a>. In this method, to determine the allele frequency every generation, a uniform random number with the same mean and variance as those of the exact binomial distribution is used when the allele frequency is moderate. When the allele frequency is high or low, a Poisson random number with the same mean as that of the exact binomial random number is used. In this study, a frequency of ≤5 minor alleles in the population was used as the criteria for high or low allele frequencies. In addition, the normal distribution was used instead of the uniform distribution because the normal distribution better approximates the binomial distribution, which next-generation allele frequencies follow under the Wright–Fisher model.</p></div></div><div class="section results-discussion" id="sec-3"><h2 class="">Results and Discussion</h2><div id="sec-4" class="subsection"><h3>Type I error rate of FIT</h3><p id="p-13"><a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a> summarizes the actual type I error rates for <a id="xref-ref-3-7" class="xref-bibr" href="#ref-3">Feder <em>et al.</em>’s (2013)</a> FIT. In demographic model 1 (constant-size model), as shown by <a id="xref-ref-3-8" class="xref-bibr" href="#ref-3">Feder <em>et al.</em> (2013)</a>, the actual type I error rates approach the nominal level. For model 2 (slow-growth model) and model 3 (moderate-bottleneck model), the test becomes somewhat conservative. For the purposes of controlling type I error, this is a desirable property. However, in model 4 (rapid-growth model) and model 5 (severe-bottleneck model), the test tends to be too conservative, causing it to have less power to detect selection when the population size is fluctuating.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/463750/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/463750/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Actual type I error rates (%) of FIT</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-5" class="subsection"><h3>Frequency increment test with reference loci (FITR)</h3><p id="p-16">Here we propose a new test, the FITR. Consider <em>R</em> reference loci in addition to the focal locus. It is assumed that these <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-14.gif"/></noscript></span></span> loci are evolving independently and that <em>R</em> reference loci are evolving under neutrality. We denote by <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-15.gif"/></noscript></span></span> the population frequencies of one allele at the <em>h</em>-th reference diallelic locus at times <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-16.gif"/></noscript></span></span>. Recall that <em>x</em><sub>0<em>,i</em></sub> is the allele frequency of the focal locus at <em>t<sub>i</sub></em>. Suppose that <em>N</em>(<em>t</em>) is a step function and let <em>N<sub>i</sub></em> be the population size from <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-17.gif"/></noscript></span></span> to <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-18.gif"/></noscript></span></span> such that <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-19.gif"/></noscript></span></span>. Note that although the following FITR discussion assumes <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-20.gif"/></noscript></span></span> is a step function, the same discussion can apply even the case that <em>N</em>(<em>t</em>) is a continuous function (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>) because <em>N<sub>i</sub></em> can be interpreted as the variance effective size over the period from <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-21.gif"/></noscript></span></span>. For this reason, the FITR is unbiased irrespective of fluctuations in population size.</p><p id="p-17">When <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-22.gif"/></noscript></span></span> is not close to 0 or 1 and <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-23.gif"/></noscript></span></span>(<span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-24.gif"/></noscript></span></span>) is small compared with <em>N<sub>i</sub></em>,<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-25.gif"/></noscript></span> <span class="disp-formula-label">(2)</span></span>where<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-26.gif"/></noscript></span></span>follows the standard normal distribution for <em>h</em> = 0 under the null model and for <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-27.gif"/></noscript></span></span> under the null or alternative models. We then consider whether the allele frequency change from <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-28.gif"/></noscript></span></span> for the focal locus, <em>Y</em><sub>0<em>,i</em></sub>, is significant. If <em>N<sub>i</sub></em> is known, we can test for neutrality using the fact that <em>Y</em><sub>0<em>,i</em></sub> follows the standard normal distribution. In this case, however, <em>N<sub>i</sub></em> is unknown. Let us then define a statistic,<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-29.gif"/></noscript></span> <span class="disp-formula-label">(3)</span></span><span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-30.gif"/></noscript></span> <span class="disp-formula-label">(4)</span></span>The statistic <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-31.gif"/></noscript></span></span> is independent of <em>N<sub>i</sub></em>, as seen in (4), because <em>N<sub>i</sub></em> in (3) is canceled out. In addition, <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-32.gif"/></noscript></span></span> is independent of Δ<em>t<sub>i</sub></em>. In (3), the numerator follows the standard normal distribution, and the denominator is equal to the square root of the χ<sup>2</sup> random variable divided by its degrees of freedom, <em>R</em>. Because the numerator and denominator are independent, <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-33.gif"/></noscript></span></span> follows a Student’s <em>t</em> distribution with <em>R</em> degrees of freedom (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Fisher 1925</a>). Although we determined the form of the test statistic, <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-34.gif"/></noscript></span></span>, intuitively, <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-35.gif"/></noscript></span></span> can be derived as the exact LRS using data, <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-36.gif"/></noscript></span></span>, in a plausible setting (see Appendix). In other words, the aforementioned <em>t</em>-test is equivalent to the likelihood ratio test under conditions of normality, as observed in several statistical situations (see, <em>e.g.</em>, <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Lehmann and Romano 2005</a>).</p><p id="p-18">Next, let us define a statistic using all the data from <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-37.gif"/></noscript></span></span> to <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-38.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-39.gif"/></noscript></span></span>,<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-40.gif"/></noscript></span> <span class="disp-formula-label">(5)</span></span><span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-41.gif"/></noscript></span> <span class="disp-formula-label">(6)</span></span><span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-42.gif"/></noscript></span></span> is the standardized sum of <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-43.gif"/></noscript></span></span> overall <em>i</em>. The standardization factor <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-44.gif"/></noscript></span></span> allows for a straightforward interpretation of the statistic because <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-45.gif"/></noscript></span></span> asymptotically follows the standard normal distribution as <em>R</em> becomes large. The exact distribution of <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-46.gif"/></noscript></span></span> with infinite <em>R</em> is difficult to express explicitly, but the distribution of <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-47.gif"/></noscript></span></span> can be obtained empirically by generating <em>L</em> Student’s <em>t</em> random variables with <em>R</em> degrees of freedom and summing them. This approach is valid because each <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-48.gif"/></noscript></span></span> follows a Student’s <em>t</em> distribution with <em>R</em> degrees of freedom. We obtained <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-49.gif"/></noscript></span></span> using the statistical package R (<a href="http://www.R-project.org">http://www.R-project.org</a>) with 100,000 simulations of <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-50.gif"/></noscript></span></span> for each combination of <em>R</em> and <em>L</em>. The test using <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-51.gif"/></noscript></span></span> is the FITR, an exact significance test assuming <em>Y<sub>h,i</sub></em> follows the standard normal distribution. That is, the actual type I error rate of the test is expected to be close to the nominal significance level regardless of fluctuations in population size. Unlike <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-52.gif"/></noscript></span></span>, the <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-53.gif"/></noscript></span></span> statistic is not the exact LRS, which is very difficult to express explicitly. An ad hoc interpretation of the test statistic, <em>t<sub>FITR</sub></em>, is presented in the Appendix.</p></div><div id="sec-6" class="subsection"><h3>Type I error rate of FITR and powers of FITR and FIT</h3><p id="p-19"><a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a> shows the actual type I error rates of the FITR. As expected, for all demographic models, the actual type I error rates are close to the nominal level. <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> shows the powers of the FITR and the FIT as a function of the strength of selection. In all demographic models, including the constant-size model, the FITR had more power than the FIT if five or more reference loci were used. For model 2 (slow-growth model) and model 3 (moderate-bottleneck model), the power of the FIT was acceptable. However, for model 4 (rapid-growth model) and model 5 (severe-bottleneck model), the power of the FIT was relatively small, and the FITR demonstrated much larger power than the FIT.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/463778/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/463778/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Actual type I error rates (%) of FITR</span><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/12/2151/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Powers of the FITR and the FIT in various demographic models. Powers of the FIT (black line) and the FITR with R reference loci (colored lines) are shown as functions of the selection coefficients in the five demographic models. Each point corresponds to the power obtained by 100,000 simulations at the 5% significance level. The duration of sampling time and the number of sampled points were T = 1000 and (L + 1) = 11, respectively. The intervals between any two adjacent sampled points were the same at . The initial frequency for all R+ 1 loci, , was assumed to be 0.5." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1806088008" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Powers of the FITR and the FIT in various demographic models. Powers of the FIT (black line) and the FITR with &lt;em&gt;R&lt;/em&gt; reference loci (colored lines) are shown as functions of the selection coefficients in the five demographic models. Each point corresponds to the power obtained by 100,000 simulations at the 5% significance level. The duration of sampling time and the number of sampled points were &lt;em&gt;T&lt;/em&gt; = 1000 and (&lt;em&gt;L&lt;/em&gt; + 1) = 11, respectively. The intervals between any two adjacent sampled points were the same at &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-55&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;. The initial frequency for all &lt;em&gt;R&lt;/em&gt;+ 1 loci, &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-56&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, was assumed to be 0.5.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/12/2151/F2.medium.gif" width="440" height="358"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/3/12/2151/F2.medium.gif" width="440" height="358"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/12/2151/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/12/2151/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463686" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-23" class="first-child">Powers of the FITR and the FIT in various demographic models. Powers of the FIT (black line) and the FITR with <em>R</em> reference loci (colored lines) are shown as functions of the selection coefficients in the five demographic models. Each point corresponds to the power obtained by 100,000 simulations at the 5% significance level. The duration of sampling time and the number of sampled points were <em>T</em> = 1000 and (<em>L</em> + 1) = 11, respectively. The intervals between any two adjacent sampled points were the same at <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/F2/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/F2/embed/mml-math-55.gif"/></noscript></span></span>. The initial frequency for all <em>R</em>+ 1 loci, <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/F2/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/F2/embed/mml-math-56.gif"/></noscript></span></span>, was assumed to be 0.5.</p><div class="sb-div caption-clear"></div></div></div><p id="p-24"><a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a> displays the powers of the FITR and the FIT as functions of the number of sampling points, <em>L</em>. In many cases, the powers approached certain asymptotic values with increasing <em>L</em>. In this case, the values of <em>L</em> for which the powers approached their asymptotes were relatively small (<em>e.g.</em>, <em>L</em> = 10 or 20). The powers of the FITR for <em>R</em> = 1 or 2 in Model 1 decreased somewhat as the sampling points increased. This trend also was observed for the FITR with <em>R</em> = 1 and for the FIT in Model 5. The powers of the FITR and the FIT as functions of the duration of sampling time, <em>T</em>, are given in <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a>. For all cases, the powers increased with increasing <em>T</em>, as expected. For <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3, A and B</a>, the FITR had more power than the FIT if 10 or more reference loci were used even in the constant-size model. This difference in power was more obvious for model 5 (severe-bottleneck model).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/12/2151/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Powers of the FITR and the FIT as functions of (A) the number of sampling points and (B) the duration of sampling. Powers of the FIT (black line) and the FITR with R reference loci (colored lines) are shown for the demographic models 1 and 5. Each point corresponds to the power obtained by 100,000 simulations at the 5% significance level. The selection coefficients were s = 0.002 for model 1 and s = 0.0005 for model 5. The intervals between any two adjacent sampled points were the same, . The initial frequency for all R+1 loci, , was assumed to be 0.5. (A) The duration of sampling time was fixed at T = 1000. (B) The number of sampled points was fixed at L = 10. (Note: The FIT curve nearly overlapped with the FITR curve for L = 5 in Model 1.)" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1806088008" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Powers of the FITR and the FIT as functions of (A) the number of sampling points and (B) the duration of sampling. Powers of the FIT (black line) and the FITR with &lt;em&gt;R&lt;/em&gt; reference loci (colored lines) are shown for the demographic models 1 and 5. Each point corresponds to the power obtained by 100,000 simulations at the 5% significance level. The selection coefficients were &lt;em&gt;s&lt;/em&gt; = 0.002 for model 1 and &lt;em&gt;s&lt;/em&gt; = 0.0005 for model 5. The intervals between any two adjacent sampled points were the same, &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-57&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;. The initial frequency for all &lt;em&gt;R&lt;/em&gt;+1 loci, &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-58&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, was assumed to be 0.5. (A) The duration of sampling time was fixed at &lt;em&gt;T&lt;/em&gt; = 1000. (B) The number of sampled points was fixed at &lt;em&gt;L&lt;/em&gt; = 10. (Note: The FIT curve nearly overlapped with the FITR curve for &lt;em&gt;L&lt;/em&gt; = 5 in Model 1.)&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/12/2151/F3.medium.gif" width="346" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/3/12/2151/F3.medium.gif" width="346" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/12/2151/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/12/2151/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463662" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-25" class="first-child">Powers of the FITR and the FIT as functions of (A) the number of sampling points and (B) the duration of sampling. Powers of the FIT (black line) and the FITR with <em>R</em> reference loci (colored lines) are shown for the demographic models 1 and 5. Each point corresponds to the power obtained by 100,000 simulations at the 5% significance level. The selection coefficients were <em>s</em> = 0.002 for model 1 and <em>s</em> = 0.0005 for model 5. The intervals between any two adjacent sampled points were the same, <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/F3/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/F3/embed/mml-math-57.gif"/></noscript></span></span>. The initial frequency for all <em>R</em>+1 loci, <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/F3/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/F3/embed/mml-math-58.gif"/></noscript></span></span>, was assumed to be 0.5. (A) The duration of sampling time was fixed at <em>T</em> = 1000. (B) The number of sampled points was fixed at <em>L</em> = 10. (Note: The FIT curve nearly overlapped with the FITR curve for <em>L</em> = 5 in Model 1.)</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-7" class="subsection"><h3>Applying the simulation method and FITR to practical situations</h3><p id="p-26">The FITR was developed and evaluated for type I error rate and power under ideal conditions of “selectively neutral” reference loci evolving independently of the focal locus and of each other, allele frequencies at reference loci ≠ 0 or 1, definable FITR statistics, and exactly known population allele frequencies. In practice, these ideal conditions may be violated. Before this section, the computer simulation method used in this study had not been validated. Here, we discuss the applicability of the simulation and describe cases that violate the aforementioned assumptions.</p><p id="p-27">For the determination of allele frequencies in successive generations at <em>R</em> + 1 loci, the exact binomial sampling is computationally intensive and impractical for realistic population sizes (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a>). Therefore, we used a pseudosampling method to simulate the binomial sampling process (Wright−Fisher model). Even for a small, constant-size (<em>n</em> = 20) Wright−Fisher population, the fixation times for a mutant obtained by the pseudosampling were consistent with those obtained by binomial sampling (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Kimura 1980</a>). However, we considered only relatively short time scales, and the performance of the pseudo-sampling method was not obvious.</p><p id="p-28">In <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>, the rows denoted by <em>r</em> = “free” correspond with an assumption of free recombination (<em>i.e.</em>, evolving independently) among <em>R</em> + 1 loci. Rejection rates simulated by the exact binomial sampling and by the pseudosampling are given. Demographic Models 1 and 5 with reduced population sizes were used (see <a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>, legend). We did not observe any differences in the results generated by the binomial sampling <em>vs.</em> the pseudosampling under neutral or selective cases. These findings support the applicability of pseudo-sampling to our problem of concern.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/463737/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/463737/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Results obtained by binomial sampling with various recombination fractions</span><div class="sb-div caption-clear"></div></div></div><p id="p-37">Next, we considered the case in which the reference loci and focal loci were not independent. We limited our analysis to the case in which <em>R</em> + 1 loci were in linkage equilibrium (LE) at t=0. For closely linked loci, linkage disequilibrium (LD) is a distinct possibility. In addition, selection at the focal locus can drastically promote LD (<em>e.g.</em>, <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Sabeti <em>et al.</em> 2002</a>). However, for example, empirical studies of human genomes suggest that LD can be extended, at most, to several megabase pairs from the selective locus (<em>e.g.</em>, <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Saunders <em>et al.</em> 2005</a>). Because the genome is large compared with the megabase pairs scale, we can select <em>R</em> reference loci such that <em>R</em> + 1 loci are in LE. For this reason, our discussion is limited to the case in which <em>R</em> + 1 loci are in LE.</p><p id="p-38">In <a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>, the rows indicated by <em>r</em> = 0.1, 0.01, and 0 describe results corresponding to a case in which the per-generation recombination fraction between any two adjacent loci are 0.1, 0.01, and 0, respectively. At <em>t</em> = 0, <em>R</em> + 1 loci are assumed to be in LE. That is, the alleles at <em>R</em> + 1 loci are randomly combined to form haplotypes. The simulations were conducted by the exact binomial sampling. For the neutral or selective cases, we observed no obvious differences between free recombination and limited recombination (<em>r</em> = 0.1, 0.01, and 0; <a id="xref-table-wrap-3-4" class="xref-table" href="#T3">Table 3</a>). That is, the type I error rates and powers were maintained regardless of recombination fractions.</p><p id="p-39">A case in which the reference loci are under selection is evaluated in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>. The selection model is the same because the focal loci and <em>R</em> loci are under the same degree of selection. That is, for all <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-60.gif"/></noscript></span></span> loci, the fitnesses of genotypes <em>A<sub>h</sub>A<sub>h</sub></em>, <em>A<sub>h</sub>a<sub>h</sub></em>, and <em>a<sub>h</sub>a<sub>h</sub></em> are assumed to be 1, 1 + 0.5<em>s<sub>h</sub></em>, and 1 + <em>s<sub>h</sub></em> (<em>s</em><sub>1</sub> = <em>s</em><sub>2</sub> = ··· = <em>s<sub>R</sub></em>), respectively. The effects of selection at the reference loci are conservative for type I error rates (see the case of <em>s</em><sub>0</sub> = 0 in <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>). The results of Model 1 suggest that if <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-61.gif"/></noscript></span></span>, there is little difference in rejection rates compared to the neutral case. Including Model 5, if the condition <em>s<sub>h</sub></em> ≤ 1/2s<sub>0</sub> is met, the power is not decreased. That is, the power is not highly sensitive to selection at the reference loci. Nevertheless, we recommend using synonymous sites or noncoding regions as references.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/12/2151/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The effects of selection at reference loci on the power of the FITR. The powers of the FITR under various selection strengths, s0, at focal loci are shown as functions of the selection coefficient, , at reference loci for demographic models 1 and 5. Each point corresponds to the power obtained by 100,000 simulations at the 5% significance level. The number of reference loci, the duration of sampling time, and the number of sampled points were R = 10, T = 1000, and (L + 1) = 11, respectively. The intervals between any two adjacent sampled points were the same at . The initial frequency for all  loci, , was assumed to be 0.5." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1806088008" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;The effects of selection at reference loci on the power of the FITR. The powers of the FITR under various selection strengths, &lt;em&gt;s&lt;/em&gt;&lt;sub&gt;0&lt;/sub&gt;, at focal loci are shown as functions of the selection coefficient, &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-62&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, at reference loci for demographic models 1 and 5. Each point corresponds to the power obtained by 100,000 simulations at the 5% significance level. The number of reference loci, the duration of sampling time, and the number of sampled points were &lt;em&gt;R&lt;/em&gt; = 10, &lt;em&gt;T&lt;/em&gt; = 1000, and (&lt;em&gt;L&lt;/em&gt; + 1) = 11, respectively. The intervals between any two adjacent sampled points were the same at &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-63&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;. The initial frequency for all &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-64&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; loci, &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-65&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, was assumed to be 0.5.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/12/2151/F4.medium.gif" width="440" height="272"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/3/12/2151/F4.medium.gif" width="440" height="272"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/12/2151/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/12/2151/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463754" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-40" class="first-child">The effects of selection at reference loci on the power of the FITR. The powers of the FITR under various selection strengths, <em>s</em><sub>0</sub>, at focal loci are shown as functions of the selection coefficient, <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/F4/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/F4/embed/mml-math-62.gif"/></noscript></span></span>, at reference loci for demographic models 1 and 5. Each point corresponds to the power obtained by 100,000 simulations at the 5% significance level. The number of reference loci, the duration of sampling time, and the number of sampled points were <em>R</em> = 10, <em>T</em> = 1000, and (<em>L</em> + 1) = 11, respectively. The intervals between any two adjacent sampled points were the same at <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/F4/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/F4/embed/mml-math-63.gif"/></noscript></span></span>. The initial frequency for all <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/F4/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/F4/embed/mml-math-64.gif"/></noscript></span></span> loci, <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/F4/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/F4/embed/mml-math-65.gif"/></noscript></span></span>, was assumed to be 0.5.</p><div class="sb-div caption-clear"></div></div></div><p id="p-41">We next considered a situation in which allele frequencies at the reference loci were low at <em>t</em> = 0 and some allele frequencies could become 0 or 1 by <em>t</em> = <em>T</em>. When allele frequencies became 0 or 1, the statistic, <em>t</em><sub>FITR</sub>, in (6) could not be defined. Therefore, it was practical to remove these loci from the calculation of <em>t</em><sub>FITR</sub>. <a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a> indicates how rejection rates are changed by the data handling. Values in parentheses indicate the average number of reference loci used to calculate the FITR statistics. For <em>L</em> = 2 the type I error rate was inflated by a few percent (<em>e.g.</em>, 6.69% at most, Model 1) possibly because changes in allele frequencies at reference loci are biased toward smaller values when loci are removed for which the frequencies of alleles become 0 or 1. These apparently reduced changes in allele frequencies could bring about overestimates of change at the selective locus. For <em>L</em> = 10 the inflation of the type I error rate becomes small. In general, to prevent inflation of the type I error rate, loci having moderate frequencies of alleles (<em>e.g.</em>, ≥10%) should be used in this test.</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/463673/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/463673/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Effects of allele frequencies at the reference loci</span><div class="sb-div caption-clear"></div></div></div><p id="p-45">The effects of sampling error on the type I error rate and power of the FITR are shown in <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>. In general, the effects of sampling error on the type I error rate were conservative. As expected, the power decreased as the number of sampled individuals increased. The degree of power reduction differed for different demographic models or values of <em>L</em>. This finding reflects that the power is influenced by the relative magnitudes of changes in allele frequencies at <em>R</em> + 1 loci and sampling errors. As <em>L</em> increased, the relative changes in allele frequencies to the sampling errors decreased. Thus, power was more reduced for larger <em>L</em>. Regarding the demographic models, the population size of Model 5 was larger than that of Model 1. Therefore, the relative changes in allele frequencies to the sampling errors were larger in Model 5, and the degree of power is large in Model 5.</p><p id="p-46">In this study, we proposed a neutrality test, the FITR, to accommodate fluctuations in population size using reference loci. Our test is an extension of <a id="xref-ref-3-9" class="xref-bibr" href="#ref-3">Feder <em>et al.</em>’s (2013)</a> FIT. By computer simulation, the actual type I error rate of the FITR was nearly equal to the nominal significant level regardless of fluctuations in population size when sampling noise could be ignored. The FITR detected selection with remarkable power under conditions of rapid growth (model 4) and severe bottleneck (model 5). Even under a model of constant population size, the FITR using 10 or more reference loci had more power than the FIT.</p><p id="p-47">We also discussed the performance of the FITR in practical situations. The effects of selection at the reference loci were small unless selection was strong. Our findings indicated that when <em>R</em> + 1 were in LE, those loci should be considered independent of each other. In addition, loci with moderate frequencies of alleles should be used as references. Our findings may facilitate the development of more sophisticated methods using independent reference loci, including a method that can quantify (estimate) the strength of selection. These methods will enable appropriate inferences about natural selection in real and dynamic populations. <a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/12/2151/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The effects of sampling error on the power of the FITR. The powers of the FITR under various sampling regimes are shown as functions of the selection coefficients for demographic models 3 and 5. All, or 5000, 1000, 500, or 100 individuals in a population were assumed to be sampled. Sampling was assumed to be conducted by binomial sampling at each (R + 1) locus and at each (L + 1) time point. Each point corresponds to the power obtained by 100,000 simulations at the 5% significance level. The number of reference loci, the duration of sampling time, and the number of sampled points were R = 10, T = 1000, and (L + 1) = 3 (upper graphs) or 11 (lower graphs), respectively. The intervals between any two adjacent sampled points were the same at &#x394;t = ti&#x2013;ti-1 = 500 (top graphs) or 100 (bottom graphs) (i = 1,2,&#x2026;,L). The initial frequency for all  loci, , were assumed to be 0.5." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1806088008" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;The effects of sampling error on the power of the FITR. The powers of the FITR under various sampling regimes are shown as functions of the selection coefficients for demographic models 3 and 5. All, or 5000, 1000, 500, or 100 individuals in a population were assumed to be sampled. Sampling was assumed to be conducted by binomial sampling at each (&lt;em&gt;R&lt;/em&gt; + 1) locus and at each (&lt;em&gt;L&lt;/em&gt; + 1) time point. Each point corresponds to the power obtained by 100,000 simulations at the 5% significance level. The number of reference loci, the duration of sampling time, and the number of sampled points were &lt;em&gt;R&lt;/em&gt; = 10, &lt;em&gt;T&lt;/em&gt; = 1000, and (&lt;em&gt;L&lt;/em&gt; + 1) = 3 (upper graphs) or 11 (lower graphs), respectively. The intervals between any two adjacent sampled points were the same at &#x394;&lt;em&gt;&lt;sub&gt;t&lt;/sub&gt;&lt;/em&gt; = &lt;em&gt;t&lt;sub&gt;i&lt;/sub&gt;&#x2013;t&lt;sub&gt;i-1&lt;/sub&gt;&lt;/em&gt; = 500 (top graphs) or 100 (bottom graphs) (&lt;em&gt;i&lt;/em&gt; = 1,2,&#x2026;,L). The initial frequency for all &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-67&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; loci, &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-68&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, were assumed to be 0.5.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/12/2151/F5.medium.gif" width="363" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/3/12/2151/F5.medium.gif" width="363" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/12/2151/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/12/2151/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463641" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-48" class="first-child">The effects of sampling error on the power of the FITR. The powers of the FITR under various sampling regimes are shown as functions of the selection coefficients for demographic models 3 and 5. All, or 5000, 1000, 500, or 100 individuals in a population were assumed to be sampled. Sampling was assumed to be conducted by binomial sampling at each (<em>R</em> + 1) locus and at each (<em>L</em> + 1) time point. Each point corresponds to the power obtained by 100,000 simulations at the 5% significance level. The number of reference loci, the duration of sampling time, and the number of sampled points were <em>R</em> = 10, <em>T</em> = 1000, and (<em>L</em> + 1) = 3 (upper graphs) or 11 (lower graphs), respectively. The intervals between any two adjacent sampled points were the same at Δ<em><sub>t</sub></em> = <em>t<sub>i</sub>–t<sub>i-1</sub></em> = 500 (top graphs) or 100 (bottom graphs) (<em>i</em> = 1,2,…,L). The initial frequency for all <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/F5/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/F5/embed/mml-math-67.gif"/></noscript></span></span> loci, <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/F5/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/F5/embed/mml-math-68.gif"/></noscript></span></span>, were assumed to be 0.5.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-58">I thank K. Ikeo and S. Mano for useful suggestions on the study. I also thank the associate editor and the two anonymous reviewers for their helpful comments on an earlier version of this manuscript, which improved this work substantially. This work was supported by health labor sciences research grant from The Ministry of Health Labor and Welfare (H23-jituyouka(nanbyou)-006).</p></div><div class="section app" id="app-1"><h2 class="">Appendix</h2><p id="p-49">Suppose that there are two segregating alleles, denoted by <em>A<sub>h</sub></em> and <em>a<sub>h</sub></em>, at the <em>h</em>-th locus <span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-69.gif"/></noscript></span></span>. The fitnesses of genotypes <em>A<sub>h</sub>A<sub>h</sub></em>, <em>A<sub>h</sub>a<sub>h</sub></em>, and <em>a<sub>h</sub>a<sub>h</sub></em> are assumed to be <span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-70.gif"/></noscript></span></span>, respectively. As described in the main text, the case of <em>h</em> = 0 corresponds to the focal locus. When <span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-71.gif"/></noscript></span></span> is small compared with <em>N</em><sub>i</sub>, the change in the allele frequency of <span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-72.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-73.gif"/></noscript></span></span>, from <span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-74.gif"/></noscript></span></span> approximately follows a normal distribution,<span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-75.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-75.gif"/></noscript></span></span>The normalized change, <span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-76.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-76.gif"/></noscript></span></span>, is approximately<span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-77.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-77.gif"/></noscript></span></span>Then the probability density of <span class="inline-formula" id="inline-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-78.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-78.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-79.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-79.gif"/></noscript></span></span>, under <span class="inline-formula" id="inline-formula-70"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-80.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-80.gif"/></noscript></span></span> is</p><span class="disp-formula" id="disp-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-81.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-81.gif"/></noscript></span></span><p id="p-50">Considering <span class="inline-formula" id="inline-formula-71"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-82.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-82.gif"/></noscript></span></span> for all <span class="inline-formula" id="inline-formula-72"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-83.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-83.gif"/></noscript></span></span>, the joint probability density of <span class="inline-formula" id="inline-formula-73"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-84.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-84.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-74"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-85.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-85.gif"/></noscript></span></span>, under <span class="inline-formula" id="inline-formula-75"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-86.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-86.gif"/></noscript></span></span>.</p><span class="disp-formula" id="disp-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-87.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-87.gif"/></noscript></span> <span class="disp-formula-label">(A1)</span></span><ul class="list-simple " id="list-1"><li id="list-item-1"><p id="p-51"><span class="list-label">i) </span><span class="inline-formula" id="inline-formula-76"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-88.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-88.gif"/></noscript></span></span> <strong>is the exact LRS using data</strong> <span class="inline-formula" id="inline-formula-77"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-89.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-89.gif"/></noscript></span></span></p><p id="p-52">We will show that <span class="inline-formula" id="inline-formula-78"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-90.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-90.gif"/></noscript></span></span> given by (3) or (4) is the exact statistic using data <span class="inline-formula" id="inline-formula-79"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-91.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-91.gif"/></noscript></span></span> to test <span class="inline-formula" id="inline-formula-80"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-92.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-92.gif"/></noscript></span></span> (for all <em>h</em>) against <span class="inline-formula" id="inline-formula-81"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-93.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-93.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-82"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-94.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-94.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-83"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-95.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-95.gif"/></noscript></span></span> in the likelihood ratio test framework.</p><p id="p-53">In our model, the likelihood ratio test is a test for which we reject <span class="inline-formula" id="inline-formula-84"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-96.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-96.gif"/></noscript></span></span> if<span class="disp-formula" id="disp-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-97.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-97.gif"/></noscript></span> <span class="disp-formula-label">(A2)</span></span>Otherwise, we accept <em>H</em><sub>0</sub>. Here, <span class="inline-formula" id="inline-formula-85"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-98.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-98.gif"/></noscript></span></span> is the maximum likelihood estimator (MLE) of <em>N<sub>i</sub></em> under <em>H</em><sub>0</sub>, and <span class="inline-formula" id="inline-formula-86"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-99.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-99.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-87"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-100.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-100.gif"/></noscript></span></span> are the MLEs of <span class="inline-formula" id="inline-formula-88"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-101.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-101.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-89"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-102.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-102.gif"/></noscript></span></span> under <em>H</em><sub>1</sub>.</p><p id="p-54">Under <em>H</em><sub>0</sub>, the probability density of <span class="inline-formula" id="inline-formula-90"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-103.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-103.gif"/></noscript></span></span> is given by<span class="disp-formula" id="disp-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-104.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-104.gif"/></noscript></span></span>from (A1). Solving</p><p id="p-55"><span class="disp-formula" id="disp-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-105.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-105.gif"/></noscript></span></span>we get<span class="disp-formula" id="disp-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-106.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-106.gif"/></noscript></span></span>Then<span class="disp-formula" id="disp-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-107.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-107.gif"/></noscript></span></span>Under <em>H</em><sub>1</sub>, the probability density of <span class="inline-formula" id="inline-formula-91"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-108.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-108.gif"/></noscript></span></span> is given by<span class="disp-formula" id="disp-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-109.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-109.gif"/></noscript></span></span>from (A1). Solving<span class="disp-formula" id="disp-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-110.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-110.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-111.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-111.gif"/></noscript></span></span>we get<span class="disp-formula" id="disp-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-112.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-112.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-113.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-113.gif"/></noscript></span></span>Then<span class="disp-formula" id="disp-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-114.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-114.gif"/></noscript></span></span>Thus, <span class="inline-formula" id="inline-formula-92"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-115.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-115.gif"/></noscript></span></span> is equivalent to<span class="disp-formula" id="disp-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-116.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-116.gif"/></noscript></span></span>With some algebra, we let a new constant <span class="inline-formula" id="inline-formula-93"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-117.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-117.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-118.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-118.gif"/></noscript></span></span>Finally, we get<span class="disp-formula" id="disp-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-119.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-119.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-94"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-120.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-120.gif"/></noscript></span></span>. Therefore, <em>t<sub>FITR(i)</sub></em> as given by (3) or (4) is the exact LRS using data <span class="inline-formula" id="inline-formula-95"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-121.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-121.gif"/></noscript></span></span>.</p></li><li id="list-item-2"><p id="p-56"><span class="list-label">ii) </span><strong>Ad hoc interpretation for <em>t<sub>FITR</sub></em> as a test statistic using the data Δ<em>x</em></strong></p><p id="p-57">As <em>R</em> grows, the <em>t</em> distribution for <em>t<sub>FITR(i)</sub></em> approaches the normal distribution with mean <span class="inline-formula" id="inline-formula-96"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-122.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-122.gif"/></noscript></span></span> and variance 1,<span class="disp-formula" id="disp-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-123.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-123.gif"/></noscript></span></span>Unfortunately, <span class="inline-formula" id="inline-formula-97"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-124.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-124.gif"/></noscript></span></span> varies with <em>N</em><sub>i</sub>, <span class="inline-formula" id="inline-formula-98"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-125.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-125.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-99"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-126.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-126.gif"/></noscript></span></span>. When <span class="inline-formula" id="inline-formula-100"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-127.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-127.gif"/></noscript></span></span>; <span class="inline-formula" id="inline-formula-101"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-128.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-128.gif"/></noscript></span></span> 0.50, 0.490, 0.458, 0.400, 0.300 and 0.218, respectively. Nevertheless, the values of <span class="inline-formula" id="inline-formula-102"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-129.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-129.gif"/></noscript></span></span> are roughly the same if <span class="inline-formula" id="inline-formula-103"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-130.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-130.gif"/></noscript></span></span> are far from 0 or 1. In addition, the values of <span class="inline-formula" id="inline-formula-104"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-131.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-131.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-105"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-132.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-132.gif"/></noscript></span></span> vary with <em>i</em>. However, we try to consider <span class="inline-formula" id="inline-formula-106"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-133.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-133.gif"/></noscript></span></span> as a constant <em>μ</em>, such that<span class="disp-formula" id="disp-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-134.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-134.gif"/></noscript></span></span>Consider a test <em>H</em><sub>0</sub>:μ = 0 against <em>H</em><sub>1</sub>:μ ≠ 0 using <em>L</em> i.i.d samples from the distribution. The LRS is given by the sum of <span class="inline-formula" id="inline-formula-107"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-135.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-135.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-108"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-136.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-136.gif"/></noscript></span></span>. That is, when <em>R</em> is large and the variation of the value for <span class="inline-formula" id="inline-formula-109"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-137.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12/2151/embed/mml-math-137.gif"/></noscript></span></span> is not large, <em>t<sub>FITR</sub></em> given by (5) or (6) is expected to be close to the LRS.</p></li></ul></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-17"><p id="p-59"><em>Communicating editor: P. Pfaffelhuber</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2013-08-19"><span class="received-label">Received </span>August 19, 2013.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2013-09-26"><span class="accepted-label">Accepted </span>September 26, 2013.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2013 Nishino</li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-3.12.2151.1" data-doi="10.1093/molbev/msm056"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bollback</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huelsenbeck</span>  <span class="cit-name-given-names">J. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Clonal interference is alleviated by high mutation rates in large populations.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">1397</span>–<span class="cit-lpage">1406</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsm056%26rft_id%253Dinfo%253Apmid%252F17379621%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjI0LzYvMTM5NyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzMvMTIvMjE1MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-3.12.2151.2" data-doi="10.1534/genetics.107.085019"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bollback</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">York</span>  <span class="cit-name-given-names">T. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Estimation of 2Nes from temporal allele frequency data.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">179</span>: <span class="cit-fpage">497</span>–<span class="cit-lpage">502</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DBollback%26rft.auinit1%253DJ.%2BP.%26rft.volume%253D179%26rft.issue%253D1%26rft.spage%253D497%26rft.epage%253D502%26rft.atitle%253DEstimation%2Bof%2B2Nes%2BFrom%2BTemporal%2BAllele%2BFrequency%2BData%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.085019%26rft_id%253Dinfo%253Apmid%252F18493066%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTc5LzEvNDk3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMy8xMi8yMTUxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-3.12.2151.3" data-doi="10.1534/genetics.113.158220"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Feder</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kryazhimskiy</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plotkin</span>  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Identifying signatures of selection in genetic time series.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">196</span>: <span class="cit-fpage">509</span>–<span class="cit-lpage">522</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.158220%26rft_id%253Dinfo%253Apmid%252F24318534%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk2LzIvNTA5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMy8xMi8yMTUxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-3.12.2151.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fisher</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1925</span> <span class="cit-article-title">Applications of “Student’s” distribution.</span> <abbr class="cit-jnl-abbrev">Metron</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">90</span>–<span class="cit-lpage">104</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMetron%26rft.volume%253D5%26rft.spage%253D90%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-3.12.2151.5" data-doi="10.1534/genetics.111.133454"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gallet</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cooper</span>  <span class="cit-name-given-names">T. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elena</span>  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lenormand</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Measuring selection coefficients below 10<sup>−3</sup>: method, questions, and prospects.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">175</span>–<span class="cit-lpage">186</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.133454%26rft_id%253Dinfo%253Apmid%252F22042578%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkwLzEvMTc1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMy8xMi8yMTUxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-3.12.2151.6" data-doi="10.1038/sj.gene.6364172"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hummel</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schmidt</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kremeyer</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Herrmann</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oppermann</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Detection of the CCR5–D32 HIV resistance gene in Bronze Age skeletons.</span> <abbr class="cit-jnl-abbrev">Genes Immun.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">371</span>–<span class="cit-lpage">374</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2Band%2Bimmunity%26rft.stitle%253DGenes%2BImmun%26rft.aulast%253DHummel%26rft.auinit1%253DS.%26rft.volume%253D6%26rft.issue%253D4%26rft.spage%253D371%26rft.epage%253D374%26rft.atitle%253DDetection%2Bof%2Bthe%2BCCR5-Delta32%2BHIV%2Bresistance%2Bgene%2Bin%2BBronze%2BAge%2Bskeletons.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.gene.6364172%26rft_id%253Dinfo%253Apmid%252F15815693%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/sj.gene.6364172&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15815693&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F12%2F2151.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000229465600014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-3.12.2151.7" data-doi="10.1534/genetics.111.133975"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Illingworth</span>  <span class="cit-name-given-names">C. J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mustonen</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Distinguishing driver and passenger mutations in an evolutionary history categorized by interference.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">189</span>: <span class="cit-fpage">989</span>–<span class="cit-lpage">1000</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.133975%26rft_id%253Dinfo%253Apmid%252F21900272%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg5LzMvOTg5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMy8xMi8yMTUxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-3.12.2151.8" data-doi="10.1093/molbev/msr289"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Illingworth</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parts</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schiffels</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liti</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mustonen</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Quantifying selection acting on a complex trait using allele frequency time series data.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">1187</span>–<span class="cit-lpage">1197</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsr289%26rft_id%253Dinfo%253Apmid%252F22114362%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjI5LzQvMTE4NyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzMvMTIvMjE1MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-3.12.2151.9" data-doi="10.1073/pnas.77.1.522"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kimura</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1980</span> <span class="cit-article-title">Average time to fixation of a mutant allele in a finite population under continued mutation pressure: studies by analytical, numerical and pseudosampling methods.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">77</span>: <span class="cit-fpage">522</span>–<span class="cit-lpage">526</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DKimura%26rft.auinit1%253DM.%26rft.volume%253D77%26rft.issue%253D1%26rft.spage%253D522%26rft.epage%253D526%26rft.atitle%253DAverage%2Btime%2Buntil%2Bfixation%2Bof%2Ba%2Bmutant%2Ballele%2Bin%2Ba%2Bfinite%2Bpopulation%2Bunder%2Bcontinued%2Bmutation%2Bpressure%253A%2BStudies%2Bby%2Banalytical%252C%2Bnumerical%252C%2Band%2Bpseudo-sampling%2Bmethods%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.77.1.522%26rft_id%253Dinfo%253Apmid%252F16592764%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo4OiI3Ny8xLzUyMiI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzMvMTIvMjE1MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-3.12.2151.10" data-doi="10.1073/pnas.80.4.1048"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kimura</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Takahata</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-article-title">Selective constraint in protein polymorphism: study of the effectively neutral mutation model by using an improved pseudosampling method.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">80</span>: <span class="cit-fpage">1048</span>–<span class="cit-lpage">1052</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DKimura%26rft.auinit1%253DM.%26rft.volume%253D80%26rft.issue%253D4%26rft.spage%253D1048%26rft.epage%253D1052%26rft.atitle%253DSelective%2Bconstraint%2Bin%2Bprotein%2Bpolymorphism%253A%2BStudy%2Bof%2Bthe%2Beffectively%2Bneutral%2Bmutation%2Bmodel%2Bby%2Busing%2Ban%2Bimproved%2Bpseudosampling%2Bmethod%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.80.4.1048%26rft_id%253Dinfo%253Apmid%252F6573657%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI4MC80LzEwNDgiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8zLzEyLzIxNTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-book" id="cit-3.12.2151.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lehmann</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Romano</span>  <span class="cit-name-given-names">J. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-source">Testing Statistical Hypotheses</span>, <span class="cit-edition">Ed. 3</span>. <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-3.12.2151.12" data-doi="10.1534/genetics.112.140939"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Malaspinas</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Malaspinas</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Evans</span>  <span class="cit-name-given-names">S. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Slatkin</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Estimating allele age and selection coefficient from time-serial data.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">192</span>: <span class="cit-fpage">599</span>–<span class="cit-lpage">607</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.140939%26rft_id%253Dinfo%253Apmid%252F22851647%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkyLzIvNTk5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMy8xMi8yMTUxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-3.12.2151.13" data-doi="10.1534/genetics.112.147611"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mathieson</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McVean</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Estimating selection coefficients in spatially structured populations from time series data of allele frequencies.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">193</span>: <span class="cit-fpage">973</span>–<span class="cit-lpage">984</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.147611%26rft_id%253Dinfo%253Apmid%252F23307902%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkzLzMvOTczIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMy8xMi8yMTUxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-book" id="cit-3.12.2151.14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pawitan</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-source">In All Likelihood: Statistical Modeling and Inference Using Likelihood</span>, <span class="cit-publ-name">Oxford University Press</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-3.12.2151.15" data-doi="10.1534/genetics.105.048140"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Saunders</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Slatkin</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garner</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hammer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nachman</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">The extent of linkage disequilibrium caused by selection on G6PD in humans.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">171</span>: <span class="cit-fpage">1219</span>–<span class="cit-lpage">1229</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.048140%26rft_id%253Dinfo%253Apmid%252F16020776%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3MS8zLzEyMTkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8zLzEyLzIxNTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-3.12.2151.16" data-doi="10.1038/nature01140"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sabeti</span>  <span class="cit-name-given-names">P. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reich</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Higgins</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Levine</span>  <span class="cit-name-given-names">H. Z. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Richter</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Detecting recent positive selection in the human genome from haplotype structure.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">419</span>: <span class="cit-fpage">832</span>–<span class="cit-lpage">837</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DSabeti%26rft.auinit1%253DP.%2BC.%26rft.volume%253D419%26rft.issue%253D6909%26rft.spage%253D832%26rft.epage%253D837%26rft.atitle%253DDetecting%2Brecent%2Bpositive%2Bselection%2Bin%2Bthe%2Bhuman%2Bgenome%2Bfrom%2Bhaplotype%2Bstructure.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature01140%26rft_id%253Dinfo%253Apmid%252F12397357%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature01140&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12397357&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F12%2F2151.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000178769800045&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/3/12/2151.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/12/2147" title="MAPfastR: Quantitative Trait Loci Mapping in Outbred Line Crosses" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/12/2163" title="Genomic Sequence Diversity and Population Structure of Saccharomyces cerevisiae Assessed by RAD-seq" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/3/12"><span>Volume 3 Issue 12, December 2013</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/3/12.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/3/12.cover-source.jpg?itok=Gq_aE1-I" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 3 (12)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/12/2151" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-AAWutzpc527_ybxBhnojZzd7EEh7YaQNhfJwNeKU2w0" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/3/12/2151" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/3/12/2151" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/3/12/2151" class="active" data-icon-position="" data-hide-link-title="0">Detecting Selection Using Time-Series Data of Allele Frequencies with Multiple Independent Reference Loci</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459599" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-p936JV2vZxVHvvpIzpQKFyIuSjBDZ6jTxku4yKalISM" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/12/2151.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/3/12/2151" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-vaszwOqJNE5oWCYjnNPudcoTekyjfORLg4OuhTdnxBc" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/3/12/2151" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/12/2151.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459599" id="node459599--2" data-pisa="ggg;3/12/2151" data-pisa-master="ggg;g3.113.008276" data-apath="/ggg/3/12/2151.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Detecting Selection Using Time-Series Data of Allele Frequencies with Multiple Independent Reference Loci</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Jo Nishino</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2151-2161;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.008276 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459599--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jo Nishino </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Center for Information Biology and DNA Data Bank of Japan, National Institute of Genetics, Research Organization of Information and Systems, Mishima, Shizuoka 411-8540, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jo%2BNishino%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nishino%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJo%2BNishino%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jnishino@nig.ac.jp" class="" data-icon-position="" data-hide-link-title="0">jnishino@nig.ac.jp</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/3/12/2151" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/3/12/2151&amp;title=Detecting%20Selection%20Using%20Time-Series%20Data%20of%20Allele%20Frequencies%20with%20Multiple%20Independent%20Reference%20Loci" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/3/12/2151&amp;title=Detecting%20Selection%20Using%20Time-Series%20Data%20of%20Allele%20Frequencies%20with%20Multiple%20Independent%20Reference%20Loci" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/3/12/2151&amp;title=Detecting%20Selection%20Using%20Time-Series%20Data%20of%20Allele%20Frequencies%20with%20Multiple%20Independent%20Reference%20Loci" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/3/12/2151&amp;text=Detecting%20Selection%20Using%20Time-Series%20Data%20of%20Allele%20Frequencies%20with%20Multiple%20Independent%20Reference%20Loci" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/3/12/2151&amp;title=Detecting%20Selection%20Using%20Time-Series%20Data%20of%20Allele%20Frequencies%20with%20Multiple%20Independent%20Reference%20Loci" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/3/12/2151&amp;t=Detecting%20Selection%20Using%20Time-Series%20Data%20of%20Allele%20Frequencies%20with%20Multiple%20Independent%20Reference%20Loci" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/3/12/2151&amp;title=Detecting%20Selection%20Using%20Time-Series%20Data%20of%20Allele%20Frequencies%20with%20Multiple%20Independent%20Reference%20Loci" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/3/12/2151&amp;title=Detecting%20Selection%20Using%20Time-Series%20Data%20of%20Allele%20Frequencies%20with%20Multiple%20Independent%20Reference%20Loci" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459599/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459599" id="node459599--3" data-pisa="ggg;3/12/2151" data-pisa-master="ggg;g3.113.008276" data-apath="/ggg/3/12/2151.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Detecting Selection Using Time-Series Data of Allele Frequencies with Multiple Independent Reference Loci</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Jo Nishino</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2151-2161;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.008276 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459599--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jo Nishino </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Center for Information Biology and DNA Data Bank of Japan, National Institute of Genetics, Research Organization of Information and Systems, Mishima, Shizuoka 411-8540, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jo%2BNishino%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nishino%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJo%2BNishino%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jnishino@nig.ac.jp" class="" data-icon-position="" data-hide-link-title="0">jnishino@nig.ac.jp</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2151&amp;rft.epage=2161&amp;rft.atitle=Detecting%20Selection%20Using%20Time-Series%20Data%20of%20Allele%20Frequencies%20with%20Multiple%20Independent%20Reference%20Loci&amp;rft.volume=3&amp;rft.issue=12&amp;rft.date=2013-12-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Nishino%2C+Jo"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459599/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459599/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459599/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459599/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459599/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459599/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459599/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459599/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459599/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459599/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459599/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459599/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/3/12/2151" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-3" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results and Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#app-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Appendix</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/3/12/2151.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/3/12/2151.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/3/12/2151.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/3/12/2151.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
