<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/3/12/2313" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/3/12/2313.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/3/12/2313.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/3/12/2313.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/3/12/2313.atom" />
<meta name="HW.pisa" content="ggg;3/12/2313" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Rapid Recombination Mapping for High-Throughput Genetic Screens in Drosophila" />
<meta name="DC.Identifier" content="10.1534/g3.113.008615" />
<meta name="DC.Date" content="2013-12-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2013 Sapiro et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Mutagenesis screens are a staple of classical genetics. Chemical-induced mutations, however, are often difficult and time-consuming to identify. Here, we report that recombination analysis with pairs of dominant visible markers provides a rapid and reliable strategy to map mutations in Drosophila melanogaster . This method requires only two generations and a total of six crosses in vials to estimate the genetic map position of the responsible lesion with high accuracy. This genetic map position can then be reliably used to identify the mutated gene through complementation testing with an average of nine deficiencies and Sanger sequencing. We have used this approach to successfully map a collection of mutations from an ethyl methanesulfonate−based mutagenesis screen on the third chromosome. We propose that this method also may be used in conjunction with whole-genome sequencing, particularly when multiple independent alleles of the mutated locus are not available. By facilitating the rapid identification of mutated genes, our mapping strategy removes a primary obstacle to the widespread use of powerful chemical mutagenesis screens to understand fundamental biological phenomena." />
<meta name="DC.Contributor" content="Anne L. Sapiro" />
<meta name="DC.Contributor" content="Robert J. Ihry" />
<meta name="DC.Contributor" content="Derek L. Buhr" />
<meta name="DC.Contributor" content="Kevin M. Konieczko" />
<meta name="DC.Contributor" content="Sarah M. Ives" />
<meta name="DC.Contributor" content="Anna K. Engstrom" />
<meta name="DC.Contributor" content="Nicholas P. Wleklinski" />
<meta name="DC.Contributor" content="Kristin J. Kopish" />
<meta name="DC.Contributor" content="Arash Bashirullah" />
<meta name="article:published_time" content="2013-12-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Rapid Recombination Mapping for High-Throughput Genetic Screens in Drosophila" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Mutagenesis screens are a staple of classical genetics. Chemical-induced mutations, however, are often difficult and time-consuming to identify. Here, we report that recombination analysis with pairs of dominant visible markers provides a rapid and reliable strategy to map mutations in &lt;i&gt;Drosophila melanogaster&lt;/i&gt;. This method requires only two generations and a total of six crosses in vials to estimate the genetic map position of the responsible lesion with high accuracy. This genetic map position can then be reliably used to identify the mutated gene through complementation testing with an average of nine deficiencies and Sanger sequencing. We have used this approach to successfully map a collection of mutations from an ethyl methanesulfonate−based mutagenesis screen on the third chromosome. We propose that this method also may be used in conjunction with whole-genome sequencing, particularly when multiple independent alleles of the mutated locus are not available. By facilitating the rapid identification of mutated genes, our mapping strategy removes a primary obstacle to the widespread use of powerful chemical mutagenesis screens to understand fundamental biological phenomena.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2013/12/01" />
<meta name="citation_mjid" content="ggg;3/12/2313" />
<meta name="citation_id" content="3/12/2313" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/3/12/2313" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/3/12/2313.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/3/12/2313.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/3/12/2313.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.113.008615" />
<meta name="citation_pmid" content="24170736" />
<meta name="citation_volume" content="3" />
<meta name="citation_issue" content="12" />
<meta name="citation_num_pages" content="7" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2313" />
<meta name="citation_lastpage" content="2319" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Anne L. Sapiro" />
<meta name="citation_author_institution" content="Division of Pharmaceutical Sciences, University of Wisconsin-Madison, Madison, Wisconsin 53705-2222" />
<meta name="citation_author_institution" content="College of Letters and Sciences or College of Agriculture and Life Sciences, University of Wisconsin-Madison, Madison, Wisconsin 53705-2222" />
<meta name="citation_author" content="Robert J. Ihry" />
<meta name="citation_author_institution" content="Division of Pharmaceutical Sciences, University of Wisconsin-Madison, Madison, Wisconsin 53705-2222" />
<meta name="citation_author_institution" content="Cellular and Molecular Biology Graduate Program, University of Wisconsin-Madison, Madison, Wisconsin 53705-2222" />
<meta name="citation_author" content="Derek L. Buhr" />
<meta name="citation_author_institution" content="Division of Pharmaceutical Sciences, University of Wisconsin-Madison, Madison, Wisconsin 53705-2222" />
<meta name="citation_author_institution" content="College of Letters and Sciences or College of Agriculture and Life Sciences, University of Wisconsin-Madison, Madison, Wisconsin 53705-2222" />
<meta name="citation_author" content="Kevin M. Konieczko" />
<meta name="citation_author_institution" content="Division of Pharmaceutical Sciences, University of Wisconsin-Madison, Madison, Wisconsin 53705-2222" />
<meta name="citation_author_institution" content="College of Letters and Sciences or College of Agriculture and Life Sciences, University of Wisconsin-Madison, Madison, Wisconsin 53705-2222" />
<meta name="citation_author" content="Sarah M. Ives" />
<meta name="citation_author_institution" content="Division of Pharmaceutical Sciences, University of Wisconsin-Madison, Madison, Wisconsin 53705-2222" />
<meta name="citation_author_institution" content="College of Letters and Sciences or College of Agriculture and Life Sciences, University of Wisconsin-Madison, Madison, Wisconsin 53705-2222" />
<meta name="citation_author" content="Anna K. Engstrom" />
<meta name="citation_author_institution" content="Division of Pharmaceutical Sciences, University of Wisconsin-Madison, Madison, Wisconsin 53705-2222" />
<meta name="citation_author_institution" content="College of Letters and Sciences or College of Agriculture and Life Sciences, University of Wisconsin-Madison, Madison, Wisconsin 53705-2222" />
<meta name="citation_author" content="Nicholas P. Wleklinski" />
<meta name="citation_author_institution" content="Division of Pharmaceutical Sciences, University of Wisconsin-Madison, Madison, Wisconsin 53705-2222" />
<meta name="citation_author_institution" content="College of Letters and Sciences or College of Agriculture and Life Sciences, University of Wisconsin-Madison, Madison, Wisconsin 53705-2222" />
<meta name="citation_author" content="Kristin J. Kopish" />
<meta name="citation_author_institution" content="Division of Pharmaceutical Sciences, University of Wisconsin-Madison, Madison, Wisconsin 53705-2222" />
<meta name="citation_author_institution" content="College of Letters and Sciences or College of Agriculture and Life Sciences, University of Wisconsin-Madison, Madison, Wisconsin 53705-2222" />
<meta name="citation_author" content="Arash Bashirullah" />
<meta name="citation_author_institution" content="Division of Pharmaceutical Sciences, University of Wisconsin-Madison, Madison, Wisconsin 53705-2222" />
<meta name="citation_author_institution" content="Cellular and Molecular Biology Graduate Program, University of Wisconsin-Madison, Madison, Wisconsin 53705-2222" />
<meta name="citation_author_email" content="bashirullah@wisc.edu" />
<meta name="citation_reference" content="citation_title=Drosophila: A Laboratory Manual;citation_author=M. Ashburner;citation_year=1989" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. P. Blumenstiel;citation_author=A. C. Noll;citation_author=J. A. Griffiths;citation_author=A. G. Perera;citation_author=K. N. Walton;citation_title=Identification of EMS-induced mutations in Drosophila melanogaster by whole-genome sequencing.;citation_volume=182;citation_year=2009;citation_pmid=19307605;citation_doi=10.1534/genetics.109.101998" />
<meta name="citation_reference" content="citation_title=The Third-Chromosome Group of Mutant Characters of Drosophila Melanogaster;citation_author=C. B. Bridges;citation_author=T. H. Morgan;citation_year=1923" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=D. Chen;citation_author=A. Ahlford;citation_author=F. Schnorrer;citation_author=I. Kalchhauser;citation_author=M. Fellner;citation_title=High-resolution, high-throughput SNP mapping in Drosophila melanogaster.;citation_pages=323-329;citation_volume=5;citation_year=2008;citation_issue=4;citation_pmid=18327265" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=R. K. Cook;citation_author=S. J. Christensen;citation_author=J. A. Deal;citation_author=R. A. Coburn;citation_author=M. E. Deal;citation_title=The generation of chromosomal deletions to provide extensive coverage and subdivision of the Drosophila melanogaster genome.;citation_pages=R21-R21;citation_volume=13;citation_year=2012;citation_issue=3;citation_pmid=22445104;citation_doi=10.1186/gb-2012-13-3-r21" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=R. J. Ihry;citation_author=A. L. Sapiro;citation_author=A. Bashirullah;citation_title=Translational Control by the DEAD Box RNA Helicase belle Regulates Ecdysone-Triggered Transcriptional Cascades.;citation_pages=e1003085-e1003085;citation_volume=8;citation_year=2012;citation_issue=11;citation_pmid=23209440;citation_doi=10.1371/journal.pgen.1003085" />
<meta name="citation_reference" content="citation_title=The Genome of Drosophila melanogaster;citation_author=D. L. Lindsley;citation_author=G. G. Zimm;citation_year=1992" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=S. J. Marygold;citation_author=P. C. Leyland;citation_author=R. L. Seal;citation_author=J. L. Goodman;citation_author=J. Thurmond;citation_title=FlyBase: improvements to the bibliography.;citation_volume=41;citation_year=2013;citation_pmid=23125371;citation_doi=10.1093/nar/gks1024" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=J. M. Osterloh;citation_author=J. Yang;citation_author=T. M. Rooney;citation_author=A. N. Fox;citation_author=R. Adalbert;citation_title=dSarm/Sarm1 is required for activation of an injury-induced axon death pathway.;citation_volume=337;citation_year=2012;citation_pmid=22678360;citation_doi=10.1126/science.1223899" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=A. L. Parks;citation_author=K. R. Cook;citation_author=M. Belvin;citation_author=N. A. Dompe;citation_author=R. Fawcett;citation_title=Systematic generation of high-resolution deletion coverage of the Drosophila melanogaster genome.;citation_pages=288-292;citation_volume=36;citation_year=2004;citation_issue=3;citation_pmid=14981519;citation_doi=10.1038/ng1312" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=E. Ryder;citation_author=M. Ashburner;citation_author=R. Bautista-Llacer;citation_author=J. Drummond;citation_author=J. Webster;citation_title=The DrosDel deletion collection: a Drosophila genomewide chromosomal deficiency resource.;citation_volume=177;citation_year=2007;citation_pmid=17720900;citation_doi=10.1534/genetics.107.076216" />
<meta name="citation_reference" content="citation_journal_title=J. Exp. Zool.;citation_author=A. H. Sturtevant;citation_title=The linear arrangement of six sex-linked factors in Drosophila, as shown by their mode of association.;citation_pages=43-59;citation_volume=14;citation_year=1913;citation_doi=10.1002/jez.1400140104" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=L. Wang;citation_author=J. Evans;citation_author=H. K. Andrews;citation_author=R. B. Beckstead;citation_author=C. S. Thummel;citation_title=A genetic screen identifies new regulators of steroid-triggered programmed cell death in Drosophila.;citation_volume=180;citation_year=2008;citation_pmid=18757938;citation_doi=10.1534/genetics.108.092478" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=R. G. Zhai;citation_author=P. R. Hiesinger;citation_author=T.-W. Koh;citation_author=P. Verstreken;citation_author=K. L. Schulze;citation_title=Mapping Drosophila mutations with molecularly defined P element insertions.;citation_volume=100;citation_year=2003;citation_pmid=12960394;citation_doi=10.1073/pnas.1832753100" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Rapid Recombination Mapping for High-Throughput Genetic Screens in Drosophila" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12.cover-source.jpg" />
<meta name="twitter:description" content="Mutagenesis screens are a staple of classical genetics. Chemical-induced mutations, however, are often difficult and time-consuming to identify. Here, we report that recombination analysis with pairs of dominant visible markers provides a rapid and reliable strategy to map mutations in Drosophila melanogaster . This method requires only two generations and a total of six crosses in vials to estimate the genetic map position of the responsible lesion with high accuracy. This genetic map position can then be reliably used to identify the mutated gene through complementation testing with an average of nine deficiencies and Sanger sequencing. We have used this approach to successfully map a collection of mutations from an ethyl methanesulfonate−based mutagenesis screen on the third chromosome. We propose that this method also may be used in conjunction with whole-genome sequencing, particularly when multiple independent alleles of the mutated locus are not available. By facilitating the rapid identification of mutated genes, our mapping strategy removes a primary obstacle to the widespread use of powerful chemical mutagenesis screens to understand fundamental biological phenomena." />
<meta name="og-title" property="og:title" content="Rapid Recombination Mapping for High-Throughput Genetic Screens in Drosophila" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/3/12/2313" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Mutagenesis screens are a staple of classical genetics. Chemical-induced mutations, however, are often difficult and time-consuming to identify. Here, we report that recombination analysis with pairs of dominant visible markers provides a rapid and reliable strategy to map mutations in Drosophila melanogaster . This method requires only two generations and a total of six crosses in vials to estimate the genetic map position of the responsible lesion with high accuracy. This genetic map position can then be reliably used to identify the mutated gene through complementation testing with an average of nine deficiencies and Sanger sequencing. We have used this approach to successfully map a collection of mutations from an ethyl methanesulfonate−based mutagenesis screen on the third chromosome. We propose that this method also may be used in conjunction with whole-genome sequencing, particularly when multiple independent alleles of the mutated locus are not available. By facilitating the rapid identification of mutated genes, our mapping strategy removes a primary obstacle to the widespread use of powerful chemical mutagenesis screens to understand fundamental biological phenomena." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12.cover-source.jpg" />
    <title>Rapid Recombination Mapping for High-Throughput Genetic Screens in Drosophila | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"MinBtqKz6czpTXkUmMdGac9tDLmH7AjJ-wSLIA8NOag","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459592","apath":"\/ggg\/3\/12\/2313.atom","pisa":"ggg;3\/12\/2313","ac":{"\/ggg\/3\/12\/2313.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/3\/12\/2313.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;3\/12\/2313"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;3\/12\/2313"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/3\/12\/2313":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00229DFrrPal1mUAolVYp5GD5y6FsDKeUrd73oOhIxpNXzCHScmK3FgjGJV1sboHBVXqmYh99m3PiOJr0VmdRuhlFYiCh7QqoYSFVm2LZ3zLYeYM3KmgCOzi20uAAiem\\\\\\\/qzuyPoFTXazuiqoxXVCjFLal+hUjBeoGz\\\\\\\/9iAZBJIoS2i6oZnLij6VRJfOvL0eEh4olZSjoeOFbT9kX1tso9gFyjgX+0OguTKQW2YsHt5U+TWRnv7n8yEUx0TlJje6lIyftiIkVre436lgCtIIKmKi6mAsunkVR3xeGb0viG3OzY9IBcEEzF0f3uP4Wag4t26ZHehdoHQwgNch+4PQ3bwkqGanJ1KdsrydT26jAHFlx3O447k+Am9y8BAzJDKprenAQvb2S56OX78m3kuIqaIMcHrP\\\\\\\/kLfTVptCwlyk4tEoV99KHcQX99kKGjnAqQECNzcSrLji+prFslH00kjcqPgITJgnHkU0pJ3yNo0k1acgbPFUL9DmlEOPnM2SZzU\\\\\\\/qB+b7LhjZP8ROfMQM4t7dGaTWFWUBAAn4bNQbwX5i3g1hgxYsTXAgekl3VrYntmWgymgmYR4wPghnlu9HVd4sKAL67QwsFsqbcJdTXWNKjSPcaGNdisUPsofe9r5OqLcxA4ku+NFRm0nFcOgZieSohkBNuslPg84dfAYyshUK1O92enAORId\\\\\\\/KjiOQysUHGLnH2clLFcHt0xGJx83+9ExC4t37J8VQkJmFKCSAmBxik9+\\\\\\\/E2gGoVkKPwgq6Q0nfPi01edX4CsC9BHPCg7uPDr\\\\\\\/hcyOfSGn0F1CjgCiscktHeGHSMmypHHMPRaILrcAVjjeCxGuYkRKltM4hdtfm3vWTKT5JZ2DS7FD0ZqX6Fp37uGncwgnOUwZR166VGsB8ABh1bvGzMNiwWS489yd5zkjuqg9u+4shV\\\\\\\/W3M1IkehBNKesCJF8tvVQIrw+MxxtEK+fJyqZ+boMAlu59VpwoqqkhzBhz2MnAJrQEn3BNaWj7qFl7IfN+en\\\\\\\/5rjcVxjoEsR63fAXmxF\\\\\\\/v7trHMWM6hDgTp3tFGmaoYNLczPVMhM8R6Tc7UjMeIClXyX2A\\\\\\\/\\\\\\\/PYrAaolS3XMgGVdIlEuMuNB1ao5VcH9laBkz2tJpkMajPPZd+0Qb\\\\\\\/PMjO2IhhWyg6oktDWw9LmQelu4VAFyuNwYs\\\\\\\/Ugt2FcYQXB9KAcXnYzzp522Zfql4wT1QH8vzmwu+4Gz35mlRY09DdBgGtC7HUTG4KV5UguBR2VZP5qRq\\\\\\\/QfZnaw3j9hTUwByWGcFRZOZer6y1T\\\u0022,\\\u0022iv\\\u0022:\\\u0022UqA2o7UytPeydyOMExLqbw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00226a7d8f1e442cb5cf4626f51e1449ea6e948775c48c4e55b211f0e94c3d79c63a\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022z06HX6GEHLB28PDI82Fat2cOLC7tpkOaCyEcsHJtocQYz+6AoDcs\\\\\\\/TG6IuBgxdbJYxc+8ssHgYUa6P2werIRGTWdrMYHc1jSsNYNKjHVNnUpeOpFUkw2DUrsPWxQ4mOG4rtC7P7l6N1ebc4PAr5so0kmOpavW1mWGmeENizR4t8ugf0amu+HYrgZlAdQUqIC18N9UG+D0RAvgyQtvej0I3aAwb0yKDlgPX6bImzKXoRJI0H3BoHoFbwYRxrrj\\\\\\\/kzAzS8p9Te5Mdo3732\\\\\\\/qCqHJYjGDuAU5xldHnARwQKuHVzqhM0ZWARI6hYsPJ7qDIN6OUyB436nsseg1CK0ZU2e0uYmpYITvk6ZVjZRRcjEJCTA\\\\\\\/pLFZi9e9ME4drm2Ali7GZfunaWlNr7ti+wdUuIFGT4fUFm7rLN51eLEkdyKWiWXs5VHsDbfIRIbfnsmcN\\\\\\\/AiTj\\\\\\\/C+Z91\\\\\\\/3oAAOOr2VmsfZsoYxjE\\\\\\\/5dRmqG9cf0Lpfhp6SzxczQ3roWGfmBZQlxvoI3P\\\\\\\/\\\\\\\/n4eY4sVlAiSplAm6MTnhWtolq4OUcJh3XxFa2lbTwCqACNq+v\\\\\\\/GMKZrCpCPnNOdQtPb2oD96gr0OO3FaI8Xi8mj7oGj+sFIcvD5bk8DpQytCd5ZLpKiEarYOJl7iE0wtDDCSa+KIHMfQ7RLxxtzkZT9VvSDcU8aJmNpzupvF1Dd4c9w1tz3O40DVtrcoOZu\\\\\\\/HR3CgraphxikGMcel9XKWQcwfKf0NSfZMK9KNKQDkXxd4YWUwarztPifAWk44ldsjVClX2dMMVKva3qwMj6jA8TlwvjBN+DnQVIP0Ul7sSVyU5E\\\\\\\/iGGz8j4gC\\\\\\\/Ow4FW2XdF71KPsUjlbywzi871zTXXqaf3gkihfqharTcJi+cKz0Y3VhlC3gy1g7PJDeqb\\\\\\\/NrZmvKyudvTT+9V3f\\\\\\\/Z5d0J73zX5AfeYixhm50I3aL6yCxI0thYF6esabBens\\\\\\\/HzyjHbqmqBqvd\\\\\\\/JZiii\\\\\\\/V9thG55a7BNtY2k9E9xkn77Hxqjxk91HIi1q1mf4Pl4NExjFxzvFfUZ5eUKLdwEkT1dKehrPmhKftUF\\\\\\\/uzMC0JShKPfJ+Jz7B20FpG6meIVYqDuDEiibNGTHzVCo808YYymcUoS7slDs7MjrTm5HdcMDdkptZBFe1tJcqwJqo\\\\\\\/xRsIXEqVYnfxI0pchFga+IZVqp4LQJxN677O8bS+IWgPGvCsi+k5ZHm4eBBO22mMwCu0nizWYkjFAWyx1HWDmJF1ySZQ9DZwP9M\\\\\\\/H2wJcuEvaqiO5Yxvthk0SnDJ7rwZ1814Y8lr2Zg6lTnwRFPQmu5WXUdmciKWFa78qor+PS6JmZWuC\\\\\\\/8+p732\\\\\\\/tLIdfO00NwLYt\\\\\\\/dtLE4daanI\\\\\\\/MIuMY\\\\\\\/XKKXYHTy1e55hguBo0Iz6sqRNTvegrZeZZf5BAxPJfIxx5l536NXY9lFeuDy5oRs5zlNWu3gdTtbKndG21m34TyvYAOdKpFpTXx06+7ggoSkkfgiHN0gOD+6fFXvcjzANJXsRZd4fXRQ7qM7oz4qFpFj+onrbjpOYTcJzf764Ss3X6XqClXYdBMNJWNHwlpOasBbjN2hB1eMII3pOqOGygBiTts7QbINRxsat0j\\\\\\\/1GGInPJevCFtO4SRbu\\\\\\\/87SkL\\\\\\\/qu\\\\\\\/bOlDii9RCWC7K3t0iO3BKScCxHHwi0+9uZMD9k\\\\\\\/RYR0tRjZFvBMkjq7LDsFdQmngrg5TPw1vcB5aL3pbioeeBZOHh5nO1sXOpujFPSOpSD4Od1pwEG62xv7XKp9zHDz4OKyG2H0roO2Jbt8iermreDWbtnXVIxe9tBHfXkzh28RQ62RH2d3BHvy1sktH6VcEeN+sdTCLyhUvSm65AxHk+61Or7+ru8MwgcLI1U6UQGBJeilcijFS58C5nBkTEtf5UIg9TUxW7rVkoPoAnlZ40WUhabbWh9A8s2kecxmRv\\\\\\\/i7JPkJmNNpotPLSu0DA6tOogbEzBMF5WaEbFn39EzMpwrWL24AL8JG2kSaPKURedCJE1yHrobbVUJvUsmkFVi7gIEfxgfz4xQpa81RXmw56v69ogul6UZ4TKntWC9Bz1zUK2Kp2e0kRfhOABgBKSt8erxKylAjUvABHlE1mkiY4ae7HINYjyyIM8JV+7gVretxN6Afvytc4NygamzEXE17UcehL6aVBo54xcSsImORPK6qaimYONUiPmpD\\\\\\\/4XUIXwZQ9zHYDIMV0eYrGEkoUq\\\\\\\/BnELL13DL\\\\\\\/sWshP06TwZQzXyNSFaQAZLK2GuvNRQ7WcYrxJ6snmHTkEZubJ+stvmQ4FVrbwHtpokjgZ0axtUjjTyf5Z2vKH1DH0Hre8BIEZh+8M7umHZL8x9+no22BbA9r5YbcsmX7voQH1yVL5qbwMgYeqFKvFw6VqUZVDtl7qaOck+vl2DpWmderi9EzEbAWZPc0xUQnMdAsZBsmaWBb\\\\\\\/PET+ISmeft7JsK3FFC7G0Pgy+y+jgU8yS21HK5hqYrQAGes6a97Glpoh7llfYIsYabVE8BogIyPgxXZGZGcmkx7GCeeGzeLYtTDQi7aLw5NDB8b1IT3X6r\\\\\\\/vyEMVNToVw44G8yh0p+rWp98nIsahF5NCXlnxUw+b4uxUq\\\\\\\/2FKU5ekIrMweWghU8Cd5OBZoexdXF+6bDf8W02zMhWR7YGaKa+Pgb6RE6sQmi7YXeJjH4SaaeYk3PDaZMFL0S2h9AqmMeIc533dqV75fhsLIsww3hvyq5v2QVwl0D5rr9o+0Wt4rndsoVz6VDxcNGc2JwhlACElWv7KaguobD0CQbYKsPS1Bz89pYQUUuzgJ7Fkg2Vto77TC8P72eEP1lBRCvMP57AF\\\\\\\/O6mhFFMvDmuLzkY09XFOv630Csah5NpyD9b7PZmRtjlZt8kXzkvbvz0MD2gqbFB2pB7g5j3OKSbQzFBImbBQxJH7L6lypaGoDK564aWvIFk9Vi8c7cjCI99nn7VfyT47qmaXOE6dtdfYdzIHBSTpsjrPauoHXJlUW0FgYFxAmt+OUzWpsgF2yxyqDf6O\\\\\\\/RSAL6Vo6UFNcK2pL7EgkX2xMvnqESyo0i6sZIath7XhCK7G8FVHauUvZlyn81iCHkwlGvq3qMBEysJyO1dWwKeYX2dgVDY\\\\\\\/sT7JfHUV2nbWNHtIb9q\\\\\\\/yFPVpMUe4sp9lO\\\\\\\/ItAewBTrg8LOO+bozPaBxKjPVJMIqNjBxTCSM5jlUT824282tZ6UVZucIyan2kOLQexaIhUuTyGHBrpsz+x0jjESQutNzku+mrGGW7zNacLF5FZyylUwVZxi9sSnFyD9jNvXZpkM8UJztjVb0f0SWPvK5f3Q\\\\\\\/cipPNP4TSzMtBDO+YPlPLY8eSCROX7cUltlhweBUMr89xHWK2e5inC7Hz0d5Y1aWgAHq\\\\\\\/vjIXUIDDHOGTCMCald1puHhPVBsPrijczqNQzEpHgtAEjiHyKq6hkaOGsRZZKGERtABqmV27luGLO1CFaz1ENJtPJIPAfEhBPi70sn2W1NVGFashd+VhyadWvoGpdDarvDMTrK807Kr1yp\\\\\\\/66Neaw6zoj2fhYDVHYI9d2\\\\\\\/gtyp2eokE7WZOepi4acPWB10GBZ1oUFml58E3pl\\\\\\\/hNejpeQdnXVnyyIiR298iVxYbdqJuygtWoUNhyLPIQYCQ9dBjUjS6TIIdAy2vPhR3nAhUTdnq43o4pD3aoDdeJ9Ovq0uuyaXvYUaET0eu3yvVOiEjp9xkWMQ7ijh+MgS8uboZjplvNz9ygZDRWAfEPg2rRiEuz3w+GqtrcADY8sfHFMbDeUVgvsrnQTaSvFvpodDqV5Y157GwgpCCYpY\\\\\\\/5jTY5h64fBGsoFdhLAfkph1fZoGm0WL5sro24subvJ5tlFSSvv1wxvTTkZpIutNqY51hg5aWAWp9U0ZHcnUrZTXsRlkCBFnqISIPPt3lRDtsbogBniaxT4Q5svG94Rg3pZ2a1dDLTUWXiNBtq3giMEiLqDjUao33VTUcSOMwmzyyxzmn3LeNQBiJ5NDrgZcT2jngEwQKDwMekzyA+0Vef4Bv8CEwj1P8hLfeufB+\\\\\\\/yOLNn752tIlEnNmrEDFW6ecrrRSGVv4Fr97Bp9YC4exjWiIaF0g+t6cZQ0HVJbAY5uNxwSlnDvdYjLXyw\\\\\\\/gyQafJ5oGOJV\\\\\\\/FwRT\\\\\\\/GNnFPQl5q8Lxmu6u4fAqHhwbTABt6wBZJ4obE6kS0cA526khyOAu3NCJaswZDCIcYJCKlYuSkrh+TUmtS4kFZZReV6mKghNQaYrkd+wc1nMVz4owF+lEDVdLUAUi0koDnerTA6Bj\\\\\\\/IsFcs1QtzeBPS1eRw8nmFoB5YQNSG\\\\\\\/l6CFe39opuj1fBdIpPhl6Eg4Mb9W7Is6xBWuiwrzql0KYyU6bgjEwbUL9+SytelLUm9KZwY6QTg7ePoisGD\\\\\\\/\\\\\\\/52KePmIHuSPzeegqPsv1P3HuWiDAgz3T8RG2EhSKuTiMtsb\\\\\\\/LDZbybMhUB3LyIF0Cz\\\\\\\/Cbii26pbxWwmhju1fF6\\\\\\\/QHPDQZ+OLuwuQkxAa2YvWXqHLDm6QjaF3+fxHOMaXq+9si9lwO72pIENHwf5dgf3HEQ\\\\\\\/Jb4w0ltpuIfzC8Vx7C4sD4AfT82Pu\\\\\\\/dQlWeoRVK9e6s2AepOppwV9PC2FpZNq2dbA1NXLdvtdBdlzSVC2NaTRgMtM3KO0+SdpWl1Miju\\\\\\\/oHR8gQd1w22+eNsXNRAcysFUR7eoa71MfyY1v8YwuAVoRoyMted07PPX7jsNlC2OIPD2KvH5r3pvIgEvo03FYqLjnIOMbTohaVh7ElC100zUrzE2TQW2HjdgDRssa1hoB1DJHq8CC\\\\\\\/q37gEONFCPb4LBSzBME50qoaVK2q43wrOVSkggmtcigdf2hriAKZPofhnmj32yPz26w943iNzLJoGOIszIt5MV7DpjLAzT+m3y4QWJsC5zaVUthwX5XxyKc3SAr6oDLmG5RSo1CAYh6eiylXChJeNY5RwYNJEg3fTnV2rozQN9zba9iD3guN4KoT0Y9jrFMEB0v8jDcidTD3ShADBT257dDMmILiIcgRa6LnIoVInpzDeOm5ihMG5xIJewa7RNZgQ9kFVwGyQeJhmhUoU973F\\\\\\\/zKBccQw0tDMo4OtpyQtp+5GnaCUdhzbOcgvyAeH4bJ8ng82U1JjsKefYHK6+tyLArDac0KPftRrn9KnM8LR2fVCHLbzTAsxzhSYB+f\\\\\\\/ywnQufe5QZJcKaiOTbLN\\\\\\\/dMzPCKcYPtc3cu9feCrR1KqcAh6\\\\\\\/nFmtYb\\\\\\\/G6LTShuuSsVyUtmxvy+S2wO3LtvZyRhkk7SxbdA1ZZ4V0da4hRJRsZnySk4Hoxd+t5BcxynLco9IjusT04oP53wWCV9ly6rCf\\\\\\\/h84rvPLZguATkT2K6GxRJNj0Lv3Stpup9wtOKTEgWGLQI1Z40tPVw\\\\\\\/aYAIdHVN8cbPFTDbbCx\\\\\\\/RuLVQntinKdieFuqKjOD8CQL\\\\\\\/RnXXkU2809UhFb54od\\\\\\\/J+tmbtQbdtHxnnED47p1nkoPqtbZxsNgU8DxqRtYM7GXUE7LCGaemy6O1ddLMABqIn9w8Lb9o7MM3NEuZ5XUSPonaIuAzw8aZjga3qJQcSKKl+iHt2mppxrQBHL0qztXei24TkqfxUW8\\\\\\\/97SwEasqy7qafsPB8\\\\\\\/HFqvt3hzZu+4UgI6CEym9DyJhwZiiLUBMziqgX1G+Hg7lRf01gR3CNv2paUj9NRbhZEN+3yidfPD8wHIAxVjqDk60NQvtlHjarejr05PJ+7zohtyPMfLhHR0b0PpVaXvRh\\\\\\\/DiqADBs32p8gfGL8U+z0PtTzEi4iSXT10vHaGAWUK1tbtWkrT1hB7uwJkaNAfX2zVSObN6EjsNqNT6r4\\\\\\\/eCd9F9B03Gf9XzKEvm6RAtW8BYIMqAM7SyWFpJuv0ckpfgd6N4FYjvbH0pvryh0NLp4SuPbB+VARQ4X+osM6Q7cYwaCG5u33tXSDqxjq9J+xY8ha2EeVS02rzd41uhLPVLeOqJeUyO7M0On33M+cEWnY8C4YrXoqvJkU\\\\\\\/mlfAhZU83JyCstxSO2aUdeC4QAoXx+xZ0uHs6FcXAiJBwXRmChnY\\\\\\\/DQNxNj11PLiA7YuhaurgWwcRqADLjr3hB2rknqgRoh7yRRCZaukANtTwrCVzaNsNC5h7NZ2SyKU\\\\\\\/AC2ER\\\\\\\/UarHnZq6wZ5X0y0p2D\\\\\\\/p2roATDRg2Co8rLWOsG57303dsbFYv98odmHNJsT0c93O8YRFbKjSVBLpcy0btbSpG8+cRalBz4\\\\\\\/Nrk83rlMUYPgLkZ2thARA5PS3sRx3ClBi5rvY2QdnF6iHxsUMS9WLOlLoJQDodJafgPAxDeWm9q28MMutgQtz1DOuq3PdAWZRck0ZOWXVM2GwL4kYEku8fyFl5\\\\\\\/La1sbkSQMSc7UuYkNm2BoRErUzw0eVpEM76CCJPKER9Sj1PYq97nmGXSY1VdYEuRTEFHq+qePMS9QNTD2zOYBMOA+XFSDUwx9mAkpkAZiWwdz+kiVuh4Z85zutEeZv1i8waoPage6LEK8pTAF2d8RRge\\\\\\\/0qwFYRh\\\\\\\/5\\\\\\\/R1raKK4ao6t0KT6p2ZCRBD4lim9XSCOA7xmugECwSO5MN\\\\\\\/XpJRVXuZxTkfOllCVUjNpTs8CFc1DJhHTI69xFMKs7y0efRXj6ExHPi76d3QbEtFbuQ5kbr\\\\\\\/3LGT9n9U4qKenhUXIux2ASuVRnsEIVoF9Z2AusDN7NTiEymQQinXbCkd5twsx6douG+mPhstp+Ka9lkfqoUBsXFNkCtWmd8PrIxAbFSzcSKR5357q\\\\\\\/Y6MvFpZdIrcfgOxWFWdYKQdG+umY4\\\\\\\/5WENxkheKvd7t9phDz7Ugkdp1Z9Xo7yBPWRutyvf1qWOqZ9mnw2l5L4eBO6qVN+mqCjfUl33E+g1GHOmZvuW9A3146Nc6n3TRUZNCCwTFK+FfB\\\\\\\/nVAdnP3MFjdwSMlessLToQSB4vsHUoIqMi5yIj95jNcrwr+Fi8cJ4rhO1N+0usq2Uy8MrmR72m+ejZ5Y3U8X\\\\\\\/aQX95Ek7d\\\\\\\/VUMr\\\\\\\/nqm2nLMwxBDBxxhK5iakP077JEsqpqTbwbEsr+9q1n4fZ3zmYDNrY\\\\\\\/jccUp+lKmgA4BXcH\\\\\\\/CkJfo2rw11knKdmjaNrDJIlREO6r\\\\\\\/qdVTRgqTvcM7983L\\\\\\\/6Vl1L1Z7auzqkSMNECPqjpsfzWeMGr7neYE5YGFFX3p\\\\\\\/kZbGQdx2Xtz48sq9cKqXQt0lyTAnDbTWluPdKjsB5H\\\\\\\/uJpl\\\\\\\/BMZ5EsqvwIw0uxE6Vp+K+BbB3jlPkiKEp7hzW5D1r40DlTUgnPJt3bs\\\\\\\/0ulNGZOO2k9Rk6UUrzxLcFAWM9\\\\\\\/uu3Ix97BqyRfU36yKgmzp+GvoztSg+oz9gS9LXEMBAlmmShXfU3e\\\\\\\/f8nHyglzsgoWAaOYwIAHglZZ\\\\\\\/x9qqMUui5DNbRydCpjr7a9bk\\\\\\\/DHzkcjC\\\\\\\/RxbzSK1blPVsUPLhhhNldhBhG\\\\\\\/Pb37idN9LANqwJ7cX8TKBbY\\\\\\\/jdhZccML0j6qYYy99QQB1hkhqCYQG8bOs5JW9OB8IDeb6HyXbXJYeIxyC8M9bJWYpDv9Fa3TuQXE2MrX5FLWpThEJh0TALxEjV0rclhXCvqZit4CPVWHfu87nHGTddx5zV\\\\\\\/owj4\\\\\\\/PT+T1Si55QppPN1JaV7RzRZN1eAHSs0nOVL4kBl1R7T5zsjUcP0DwKwN1kwlM\\\\\\\/I1rdWRbqp+Uv3nle1SCZznBvjbfKr9NlWeFi71vC1Vyfk+jgu90wk9xcYcB9Erob8AC9UwuGOgYBS6nExw67DOJyXv38g4gAzG1LHdhMc4oYteDd6Um9eAFRx0C3ochTalQyLy2v61j27W7mmLhiSzvgiwlxNTSuLcnIfvuBRo3gFvFwNVYoDOUSS25y+uYuPBbAT2GZ\\\\\\\/oX+83d9u8BLyWUtnqJqcXHoyLkxu6F8CepD1rbIHphndD8IqHLOw7I\\\\\\\/BV8muFZD5xD0dWiuiJZPyA3imchRK\\\\\\\/cvqm9KZuUmVtlxsFcg6Cq7dYRSEhao3cdqjhEpaFODpHrNbQCHzaUa596GrgbWNcLDQSoHrRVOxza00kbBAEGYrkwi1ta7bp6NCFpIqFv97UFfCKwo+yzs9ueW5oPw5Mw3ett6mWPXf7JOMuDx57jYMHCVEKa43U7Frm8q8z+fpuIo+teSXInfAiIKSkhEvZ\\\\\\\/Wm2RSWoibbVDqLwo9Gx8UcWOKvxU24C1INDGAx0QoYiDfi2GLZFJeCGhODiC7JGOJD2UOUzWYG9YltRwC0Jox6xELKS1v9BvntkQBlax45DAzrT5iQD+mOOxvxD5d6Ka92YtRPKuYzfBqcuz2IYzErUFR9hQf\\\\\\\/UV8t04aSVEp5L\\\\\\\/OYjFZOovowmMy\\\\\\\/e\\\\\\\/IxJE1K9wJZtrdb6zFAHYSgraeEwpfY3wrEkXQ8vsffWrXTJlX6105tyY5Lk+G2aOkeLsbD+4eozW2383rjW6tkL2kcZqE45gQk3aHGN+5Zw3OY2Sc+y+DN2DnjtZLKBtBi9LF3x7s3O3g2Ty+O58U1YceCJcWSKbXEQ9LkPCKq1h9yMbFi6lGsAbEGlnAYzX8SbIfUHTZCauMQ8B9FpVM\\\\\\\/v2TJpzGgIbC1t10agaUCtkHhSk9gTX5qIkC5KCF\\\\\\\/hlsCV8PYGaq2YWkdx+M7ByXiKTS3PaBjjDGRo8osq+2tOZrjVDu6koZMSovJluEvvOs2WI7AxCbXRp7YOJsn7ZCRhrxq6Bp0NaZId7FTS+5t6XyaiskXBIg8iE+hcJjikHjdrFRuX0OrrBbAfrG15bFlljf16wcgcC6eWCf+ZatR8cR574BR9GFgnJw0YGJZY9BuLrMYGFQSe6p+j+yknchRD0kjSaNmnM12XfksR\\\\\\\/ZsfqMiH59MgJd4atU\\\\\\\/cMz2GqqaDtvLysa+WgJYa8qcZwIS9nBWHKmR74daFm8oT1tpo53YGEcfw2+dr6pJyZOc2vTSIjWmzXe9m\\\\\\\/qm8ZurtoplGTzAumF0DI\\\\\\\/vdyI7BLeGu9ZFqxFVhkq0zpQfVFwZ\\\\\\\/lGGqm1p4A4x7D0GdVwik47OwupQn30\\\\\\\/BF56aBoqGOrQpz2Z7y6kgz\\\\\\\/7QFzw49iSBPIj+ek7eYu2fe4p\\\\\\\/W7SrBotiQITLAHIgkFJ4pwRfAmp7rgr\\\\\\\/lSYzkSUx8aAnMEZT45kJg\\\\\\\/czIwzhP7ksVyA21tl8l59w0TPrjf9AAhjl\\\\\\\/jK97JQYe8JoFGH3ClWkTsbOov85c149WwXZ+RpxyWjWy4bf2SChYVgqrrBMnlxhPg+tzWWlJU6G+J4O\\\\\\\/fX\\\\\\\/SJGHLJVYjF\\\\\\\/y7jhKodrKqEZczx3emrGQffv6I\\\\\\\/yUdkGM4K6nUMl5623nP6Jnq035CBsoOmyB5djMFVdkRYzYDIW+zMkZ3jRNafhfO69UZKFY2Tp42FW2skD5puLj\\\\\\\/2iXgEPmdtDZLUwebMc2VxI4MpD8Md9kFgHtJs\\\\\\\/tjYIzKavConAglszJRrqFwZDaYH5NVdMnoz7Rd8UnsBmRs6PRr5UdPReLJcfp10PjrOJ+GIDnlgxH5SVlsSPkPBWPdQT2T8P3wafc09rOFYyrpOi0LLbQWWtiAOLvmC4+rWXpK81M7bFLgbkeCm0omJwsVcGHzcLspL41fxy61ysX\\\\\\\/S6azEVnba1GZPn2fhQpfvc3qDoi01bPFzmSmAc88sEui0NS135L6b7gKHROtIJP0I0r55BFqWbdOzaz7AUJDR7\\\\\\\/wM5czLja0u7FaNcZrpthKOCyZCIRvLDvrKXpLOEzdvRwfOT0zLxAbQD1HAx4qNJ9L3p8n55+CvupZrMPNF7ExivMwuMKbdEjE1T9mk1Arwv+AoBrIfIUeZbbU\\\\\\\/7JJmQYvV3K9Lhb\\\\\\\/Qs0p7InP96RduVRJO8Yng\\\\\\\/UM9zR\\\\\\\/8jI5SBUwjBvikn+a\\\\\\\/8xQBiciBcCK\\\\\\\/NTu3yf+IVTL3ZoKoiXMN6png1ELUj6HxIv89kQVVOaDX9S1qnhwfjq\\\\\\\/FnHRLBj643DoxUF4ZiV2tQb7bP38UxAdCZE19PjyqR+KBSwWxrs207xiFSFcy3hwZdCLVzdVNWZHWTDLQa\\\\\\\/F4MjVPcUtm\\\\\\\/s9UOocq\\\\\\\/43JifQC9\\\\\\\/HWIg425LFcltvPRTriVace9YUUI+tL3Cz\\\\\\\/PYIys6q3tYjFe2HFq9v1u2gvbPWdlVULcsfzd8x6VRquKfv+MLWMrVkePkJbcJ9SRvhlz49t6y4k8TD6Dz8HGFlHBgk5Uc6v\\\\\\\/IOZfBG\\\\\\\/YBdkFor5zLp5+0D1QjBLNsyCVwg88UughB7rqH6+Ajgb6BcPNmA5WZvU11KlTPkiBSEL+CHBinkazrs1TjE9ZtvNyvm+ynhZQbMibiI4cWXRwTfRJnGMXH+QFwCCdh0ExhROzssqp7KyIphozd4EjHwfFCsXG+s4nh63HuYox0yj5z\\\\\\\/s7Rp7kaqtB6Z+0P8Q\\\\\\\/onT4s0o6BpR\\\\\\\/yvzY4cdxd3hJZttazS1BTdwa4zmJs0ypdYU3gvH93XpF2C5KaPUAY7d0PreLoK9RHICzYQD8imJyy6yc4X8lp1+meizQkET5z25A+bHgUy9\\\\\\\/GCIJzLUD5aU\\\\\\\/S0UVXeR3eI45x3lcAqMXcixnfmDY5L9Fr2G9mQE0HOcIUQV7hRDjsNj2F1z9t4HPipVmwDlGRE7+3ryYD4ng2V4ePt61i5tVliFrbSN13qqFtGkFBBGpAp9Pp5yyKdE35kcl98gAc5oLxow\\\\\\\/WR9EcnmhgRca3nEfdZdHxLs2sho95Wnb6rjlcXbigi\\\\\\\/o3NXh4PfYvkS5ATvp4\\\\\\\/GaV+i1tIOUtvfiF2s0LvFWbLpVKIRFWM2mD+jVaRb9XQ3TQhUwLaIBI6FQtLeNO2NAOMaYh380JbAstuzNHcFizIxiDIDxhyqrQ8LRrCsMoE5NvgJcJF4DDw7koAAOj24ms4uAsFDPzeMha0qKi9zqous8Z4HLaheZKC4o\\\\\\\/S0tjSq8hotEM8Ui4Xn0XpimFxty+QiVvELkLHB5rLfcwM8SxPXnSK1ySxd7b4xs8q+5R6ZNh0OJlWHsVD3BcxnzhHAuZnBYNCaizyDemGZOFU15bLX\\\\\\\/ibHF2o8DWhcAUikC2YPM3yaCFRRFLm6FFhh5+4Z9\\\\\\\/eWbTq3J5VyslgO+hZppmawDwfGDy5r8tBu0wJJ+jok49tq\\\\\\\/DeamtowSPKC7MBhUxy7sMZEMyrxU3fL9OPLqh95WyszuD9Rg\\\\\\\/cDiN3gwrGQVRTsDun9EIkRMf3hjYfgZ5sBzca06rz7It89GUKsyt4HhX4shXJnMy7P\\\\\\\/nm6uWZpxSPC8AHJ10EBTV8YeqXeJstlA2ACfpXARZHfCYzRXjAB3mVJIo+s8CoM1jyRlW7XYsw99eIumKKr0\\\\\\\/ilP1E6NBG4pfGA84PbzBAa6aBK96SOx\\\\\\\/o8QszVcaFwoHu8PI6ZT6JfCQkHEuCLSAOXheknKswzi2ItoUf+wTdyGZ5mCRUnMLuxi4aG5NQdPbx\\\\\\\/UvPay3j2tmY5o\\\\\\\/0+bcYh+RpvTMqtu+1tspAcmvNfBBRaRviM2HDJkcEDL3rALAABLmEBB8+uqqOOI7SryugykA+wtTsCxJDgG9feA5ivYyP3dYHtcNihC+iZurVIvtXUgWN+ZQFnWmeaAfwU+8B+ThMBiAIfSn\\\\\\\/gDDUFWQ1C3ayDHMEJP+Zp+b\\\\\\\/+01Vycj4d2qrQBlouXLCcZkhgnQcJBACvPs+vhAUPy6Jch3HXo5DncYtdh3Q1zAH3H2qu756iupvfhj0yiIIUpNJmH8BjLLF5z5YFIpKNL4v9jGHvQF\\\\\\\/hcRzW7DqsHdMO47CYiLC43Hdk6ODXCmJ8KdR7SISIYZoSQVOPX2gThpDEr+VdijvcKxgXdv8yUB+qxxELmX1KERhR8sTvErxs6JpzdFUebzDk\\\\\\\/a57yYI4b\\\\\\\/+FmtZI2sDy39u5HagcYKjUF\\\\\\\/OqoLAMzeNGXm7WDhMNOwlJxyd9vOmxz5YFKJ+Uko7EizmmhrClHm88zDUtvaZphTOelcM6evweM1oBILsE7wx6i9qCCWxsIkfG2ASfLz+\\\\\\\/B6ujWp98AzlKiiauBd8PIp+oSsQfF70Q103YuFfAMkRaISP3wK3Lkh8kXYZS5CZk0FF4PLSOGGMjN3S35+UjGfjTQMmBeM8TggbHP6i0qTpo4GdEqhxJmWc6K7C0EdqrMWTNvMwlFFXku5ySKx06o8t1Tf3tpy\\\\\\\/qW9YF++ulvaRO2tOfac2IiTdCw\\\\\\\/D9SZcKmMVodgGjV4Q4LcvGRr0Wz\\\\\\\/5pRGHq6XDvio69nSMVs3FBwxzkvYs8+ZRoMtwESRPICApnwiefa3tCJL6uFngpxCZ3Z8jFN7U\\\\\\\/XuUwJABiNBp+qNtwY83p8blYNAAJFbID7ZInRuqMRUaFtFaqdVVvxXps5eSzkP64ADNKzomcukAa3JyvYPXBqkyuS6EGnX5sM7PrOgdgTdDzIMU07Aq5GsnUr45SbFDFKvSonsa3A5DSmjj9Jbx4Lh8499BHDCxKk+cOv2AKUfR+1nHhMMp3j7w\\\\\\\/Fxx6agA5ISGBKl71EpLStyGOaTP4e\\\\\\\/QJV+msphgfxCHKKHGiRA6oYr\\\\\\\/AIN8hb55OZBqKgc+iHyitjvR3hwSigz2LpCN+4n+GuF4AjAi+SEOx50IuRNxbjiHOJjAOMKyCn2kcWiOGu0X6RRUnQer4YweJKwy2K+TZoz+wUGXwKuBsMqmi35nVqbBpnj6coNBkmFaLPE3EY+3SW+YRTm5MU2A1lYstJPsyoYSvXwbn\\\\\\\/anqq0QsxwR\\\\\\\/esQUnZKBjMtIw5s\\\\\\\/O2Iv73JfVi36ng7c+b4C2yj5tTy\\\\\\\/wiuWZ002f4q4\\\\\\\/LJZeyVpZz1iZOM3QDZsGcr+oshZbXy86DjkMJNRvZuzosUdsUCJvP35Y69eqr7YR6dRudYSFFwKy9AXxS0cz6acda+qrDzGVKOLCs9TDMxEJcuQJ\\\\\\\/oqOMSKgIdy1X6\\\\\\\/6DLM3WT0URI5l22nSyGj8veXhqAMDFZdlFDfBE01z3JG43v+9oVOD+NTZ2qHi2usiMQUwwOljjjYeNZ5du3bU87u4+BZZtlbbpRiE3zKba2JOTP\\\\\\\/FUSdXckjs7BXThwpiaKv2Qpw3KOQOx4MG5cqCyOq7KzQgWZqSFv63rYjDJvWdcsvSL4W3LMERh53Lje2+ix0iJfnpuI4L2u3IB4b5qzDLeEAL9uFAuHt+3PkdEwdl+4lWlaLvr3gTrMK+s1cPJ\\\\\\\/+hWv3qTHttV5c74gSmAwf4oz7Q0Z0n7jUyUqeYt2DAQ6mhyogcDlyxilNucsFBmjT0F5B6elXlcyMiVcUsp4qTki0\\\\\\\/xzc+UpdjCItEo48jMZqYQqYQwcdAFMIGsJgTv+PxFVQrM1etGBTaVdt3vGw3PyQ8wZ1RDmKXXRxg8\\\\\\\/v7V4nbsfTTpgbBCZJ3CYeJlQDnDFxUBt4BkBOwQ9g\\\\\\\/OZrhZs56qX8Iy+k\\\\\\\/tk\\\\\\\/xFpfqO7KFUzGJ\\\\\\\/GXHPZ2iL3m\\\\\\\/riYA70mfa0WeOtiEVzGVpLTNJQMKEwYg1vhxwL6GeQRooYvCA4OBc5\\\\\\\/NX6ve4b0ogl7wCAFGN92S36ozFt1MzjFJLEFX5SIYpc\\\\\\\/tRa38yBPZBEzmsq8JjthOqP83DmKafjCNNmH244p972h3u7otwfuqQBCcLmL6UAiVAsyb+EOWDEJbxKeiUlVEvJsTPwO\\\\\\\/7TcRb2URCfBjqJGGs5ygD8ADSaz0RWjb5pN7B6+p4i9p437HZ4dbhTUieMK5gFVixKMjG0CTcGWljfZSJ3WHgCvFhYkqrmnX+jsx844cwkdpmsTCUWrq+60U7QHT3AE\\\\\\\/15nNl4O7\\\\\\\/TCgdK0AVPCIOCV0i5jXYFJvWl\\\\\\\/aXGBqOcsLx+LkkqnZZTsvvEli5jKNj+FVVF2\\\\\\\/hTZRJNiGBl+0SaLj\\\\\\\/golSIlMN\\\\\\\/sZRNfIt8TBVc4uHwCXNrb10sfZDgpD9W5LOkkJez66n+\\\\\\\/OwlvQngcBSX04S8WRkx6RCufaItPhnVbljnnDovtT9InerbV6wwCmPEZApkimR4t0EQAxQ9\\\\\\\/ohid9bn1hcyeg22ecK0Z89a2XFQP5QIRHy1bNJqWVFVMpKW5njnGdoq+xFDNlhyDb0LfuxJXXw0y5mlXhyWJE4Ry\\\\\\\/WYQv0Inw7PPIE3WGZuDJl4eeZuIJEgZS+x9pU0QQT4ezb9AWRvVwclq8DKsdQc8GDPgC6ZTxsLcx\\\\\\\/c10GCTnBAggVelhHSvAKIt4V0iHP+XGF7\\\\\\\/vxhCyGB6qcJcA3osSECG9C0J9kTxnjbRmgkXAWh\\\\\\\/ypLOIQqY6mFeksQ603ZwYv6B4QKLxP2oVLSioBfRyIVl8PRhEnE1mHPFl6tC7O+yUyclu4qIx744sjvh\\\\\\\/4CsZVpXrYE+mgIHjQketvRDQZkUd4iBv\\\\\\\/YGKH\\\\\\\/S08EMFNNBQSy7TQzH4mhdzI1vEn00le6yEf+IRCsj\\\\\\\/58AD5Fk8r\\\\\\\/tf\\\\\\\/tIx\\\\\\\/oH5Cdxe4RjLG37JewYyHHzezmKRdtm3q1rdyJGrbR0DJOxA07Lw5hUGsekxb2AaReyDToLu4TnOfHamrEiszMYB5UWxu022YprF486iOFQI99iUVQSUYfSxwmtCPTya41lLrWg5dzibQnXIU6MXWpyiHKz8BoFV2j9iubnHfJxT5ajUDiqR+FE4\\\\\\\/j7PsZ\\\\\\\/oiNv5CmwTz7B5uhtWh8gKTTOOSq1p23iScRUmgQiCOqdw5fSEf53EPKGBPq7\\\\\\\/QHRrWW0RiRkxxsYpXrYCLk\\\\\\\/ju+tc3QL7ngBuPh128Y3MgZOI\\\\\\\/j9RMpY5KTwMA5JdNLTik0FCg6njPrV2Br4XR3GGX58p5xfiXlTNEDwlYtq7mywHZxuoxryj7JlQ4n2NzqpvbybaBkyJCAHY6HXJme0Y2EgRph2VXtM\\\\\\\/\\\\\\\/pJUub6w1at8kQraWxVXya\\\\\\\/Ry\\\\\\\/jTWtONRa77XKjF+ObcG4G7UD6Adr1gYfN\\\\\\\/vL4yhrcIwhPITp7MFL9VVyAvAKHQE44zCdgGpJoj\\\\\\\/G7R6BDmR59mPqR2ZYsefWsebmtLiQXZLvZhv1xD77wZkjdr6Ijj7x6i20QWsrJPZazXHnw2AQHje+SG1VAc5+SSZoAVYEjtpHLBmPfso4ghz+ovyt1L1tSB+lP1zHyl92zt739U2G76srEDl\\\\\\\/Z174rYUxipcG9477Fcer9jgePKdNHGhjdkk0MlusCFRbVaDnam6yUFQ6zi64BcyXXBydBKHeSlKNQlevgmFIRfoucgBJGi7mc2gwiNSwdOSe7X67u8YV\\\\\\\/F6JqqXe83QpXrlUe5u2rxFGnissjXtpwc0\\\\\\\/HR88MFk74PA8iqLOEHxn8opAcJagow3TDPGdO4r6Ww\\\\\\\/GdvLm5l77i7yFfEV2yMKxBvGgiMwYHCpnpjRVqTToLYFuJ3UGfgSlJ04PDl6NswW9Bj+sNeFNWmicUqTKIQ9dwxdaDKmLz4GgBkRetrsZ3CAlh1pkV\\\\\\\/hSUrcAqXsbwl9Lj7p10YmqKZ7WHUp0woxcPQg3+8yHzpi4EbEX2JrmtNEN9lHO1TdS4XZ\\\\\\\/j7WiMFPD9Id44yBrMn1uOrmW6vLFfQMjMV5hWmuAsE2WyX1BjyX2IhpIH6BdDKyIkUfNAviE0\\\\\\\/n+hodNP1sSyha34KIMZlRyzrbWR1TWKznJfdjVygxhc0plSvFPyPMMcsQgAYksqAuFWyNDQlnOOLiJJYHjRaibGliuFzsn1vgjwLvpjV4jQoA9zLt\\\\\\\/oGg+mFTkcaZ1cVXD6rJizJAFhTXSckrOCqKHiNQlKE0p1x2Vzd6X0sjGqw9NUFLZtQvB1bQ0TIk7DXmGC2xKSZ0lQUu8j2yd0ISPM9rQMf6aRUzLEoebX0yNh\\\\\\\/f6YLqoq+UKhC27HPwLfSvXuqrJ+FlEL3fD0AYeSosLxONmN8xlWXePHkJZBvNzrjzYNHOsle\\\\\\\/529vQAFVZQ\\\\\\\/eH5pMnHZHzjrircnta4NLq0igEVbaZWvXQha25OBo7IFxlc\\\\\\\/L9K5Fr+9OcERUl71CFNZTDlMHZXaGuGMKYKuu\\\\\\\/KIMe+MeHjVnOQ5LRH\\\\\\\/OdSGINUP4\\\\\\\/TRKMB5PgovJyIaTDH97jnqsk\\\\\\\/R769E4Sn4HsdOLziZzkaotslZBu2OpeTo\\\\\\\/re9M0Ix+ZK1B0DNoG0rjedNCGvpra10cAXfH1fbczv\\\\\\\/8Ke7RaVuFkPMCrd0NeK3JFJT4O9rXZ7JaoC\\\\\\\/f8RG5uGubl2y9HHlb8YqySJFC+YnakM728eyzZzNrHgLq0J8F8JDz+CNGb7PjdnsrEE6P2evBERxKp01Sr6Ali1xlg4AuH2ecwNGedZG4XOYWqWpLp472R9QvcRojK4xiAHR70e4MrG4EKqOGgIi0+7YP0E3BonxZifD\\\\\\\/M3ozkpQwkCVIJX\\\\\\\/vQi7jr4VkRRioyeiA3IEiztswEMdZl0zKi2e22ldCejoe6RAA\\\\\\\/gBO908mJ7n09fVWBjigmxeSI24EdL00jDrAkQAtmIWjo9p4yrT4Yla7iLyVr4uMdLicudH\\\\\\\/gEZgU9VHHw144iE\\\\\\\/uX1wUwK\\\\\\\/dmmZ8idtGkygWFB2gs7Kv40RdYiso7h92ixCfa+oaUzCqNQvflnPEpx26M2WrYPCr9cWuDv6ucsehdom+mfIxwe4behDAt8jGX8EsBM25y+r+WxDxtOa5va5XMV2CyT8vkFdzz9luwwPG88K9bI3YNo95HoI4AJlIiHbf2040EYAFT92M0pMWr1UHegpzokIP8xWFwFhCX\\\\\\\/DcxqWulzQWpBrO4szZViBD4+REXv\\\\\\\/eyaUdjKixlyoSRySLsmP+NQ3afo\\\\\\\/G8zHdyWMUdbNBSn9n6zg2rPXD6DGDc7cA6QaSzmxnRtY1OPpb4EwEaqUCTncaxFu9iZebh8GE1XiLlXfYjFww1Bw1nEshuqHuLyWAAsBDsXwTCwS0WbxPxNrB24S4nRxzYEt9I9A2H4p2TP0k9E8SuoNSoVGYw7qwGuZxCNb\\\\\\\/axaUh4ZY5XPZMnxACh1MjaWrpUbU70s9ipli9F7IdwLd0VT6R3a80\\\\\\\/npEuFTcA+MrfsZCFdQv7pVP2pGZC4Idwii1\\\\\\\/db68eRxoD56oAGXMP9LnV734mw0lsJPcZm9LuJcIdm1+tEwtuI0KccoYgNpQiJ\\\\\\\/VyZKvC5MgBlNEZVEcdFPjD0sgG3aunL0ho5goDsFzsWfjLBL4mGVVFtcR2vwitYNmaNxb6yEhHfaMa6dh9Xr5ww7CdnSnngDuTw\\\\\\\/Nu\\\\\\\/Gx1NkprYCeQr3oALwDkylKawgEf\\\\\\\/vqHhJggAyHpY8bwF0ucTcoR7PXdnz7HkArdZ+pNtOtYvx8SOvU25iurM3ig9QUwBmlxZhEZAWQMlY3UMRkWOACklVpiqIEA5gDjTKAenEspZdOVm1PCPJq8a3RdT3xm8prKGVDlg+BpQT19lupPWLimLUQa8+0IG+CQ6FKr6DkYPvLHen9nipGOtoMe0HWgYf4nza0jidMNSnYpTNHe8KPoLtk4I2GJFJUlYlQfzWQV30GragPUlXQbf++wTykZF5gkNaC+9DmugsSH6fs1ohbmMH5D7NNlBuH3j7jvVwqz+McURy5re1hzTLSp1Il3CuHhxa8rB+72etPgaLFfW9bzYncjnTGXqx1V2Tjns7UZOCpd4QOKBsi3R4mfOTBEkvF+H0yrMNJmfkUspExSslj9PNr7CrLRjqjamhCaabWIVbOYkM+wkJmfpRaUc4PSHb4UKoEJWHqEtFKPT3hdBkEsJ50wAt+xNNYUOtHyiyr3e1DvTzCuMWgqsCGLgE7Ts4NGMkdTOJ2AY6KPUN\\\\\\\/aFQkOYu8JwKmL7fkcPumkFEZ+G1cWPp7qX8pLxRzBGnq8h9kLbRpIxx9sFiXFW7OsfoVvTiU7qmlvW8phrd8LdjyFFN4jutJh14O1ymk0jtp4PhwntekGgzq3HoMmOODGjmGgY7YvZVpjoaoxjKGD\\\\\\\/IkrZUkvP9JR4jAZXPA8EG4T+OQdSPLOwPmXGP72T1bP1IRVwm+p199cjLZsJ47uVCziEKhPs\\\\\\\/4G1RtH58uL1DJi9ylEILzZ\\\\\\\/H63ugPwkWj8Maqf\\\\\\\/ncPvPR\\\\\\\/YHI+b2CNFwLcdpoApxxcX62xFwOJlXJj9CGUQ\\\\\\\/Jw0+GluKVjUgy1RDY3xqBY9q28ARpnd6hdEWoPYwDZCJGcm6T+EtzI1peHLWNNIZlfLtENChOFfazdiDMuB9FYGqbqtPmuNud38JhHLQPwRjxS34g4OcagKKPER53Ip3zUP6r7PglmZGdf+seyanFsHvj+bANlOJzwQ6QMX47gC76A2Dr43M11ItB5n8qq2KeorTmmeLurYvc7fIMkWm1DLARTMep3BJDwKF\\\\\\\/1EhsccLxmFjwr\\\\\\\/iMbkB7H\\\\\\\/RSP5VR2\\\\\\\/qp5EPh56SHi8Pbfnr4WqMWAdWnxwAMfDBx7vbbhCBj\\\\\\\/vgsey57fbN\\\\\\\/K3acDI82pqVtFd\\\\\\\/3lIhM\\\\\\\/yB5MLmQDEQ0xnWBGfKIhhlaeEKODt\\\\\\\/KzMMB7r7I4D9inFrH45PE+gVkeaG8HTyaPENh1KdaP+9BzQehR+A\\\\\\\/lx4II0ieXSNiOKNv5ErePJ2Hnju\\\\\\\/fZNel3v1hjlIWgUF2GmEMOJoy5v81wMD89NQW89DXPZzovv+sz9qQ71qROrlZwJ9pqH6WlBTd5Xx2OWYE7\\\\\\\/CmZteGPlcW3h0yBkfMcX0bXxTpd5MZFi8qZpDyuTSvfyG\\\\\\\/ygfbtMmcqN6ebl1DmtnTtITDtkHQgkC4PFyOy8FXCb\\\\\\\/GuetczkEjjV5NlP6VzZwz5f4L20wrnJCKQA0pYRi67dcYf7yi0UzLaq8I0iyHCWo8qO77Ajhky+NqiPzDAugR1p6beUUr95NWUrbd1S0SazgG1vj+iqJFTFPv73S+0XW8WazdOKS6Gz44aOIe+WmYdauqo2XyT5DPWoFV6WBcCRgua6c\\\\\\\/c+nXsWWvbFNymAtSZaY1qtUlLqzQGaHZSCeiqcqpQD8Ayn6ZpeNz5uY4i20x+A14K2bbaiLnEoikygwjQNvFDXQLcYYjOclCPYY0NtnsFw3vJKq4EX4KIUIRO+JusnVEB00TNyEK8rbW86hpr0qWEwIziA9IaiJ5g8\\\\\\\/r37bE8YdlcFh1SAIR2v5ean6LBNBl9FztNdqSa816QYhGZdulVyyTJ+\\\\\\\/p2cqac\\\\\\\/xw+zbAmV0GVKHfJg73AEUmexJUZGlYnfHX2YgCOU1H1MgSXFllQAtMd3FAO0J96lIJ6wWZ0D+YyKb3PhErfVZ6WlSr38MiiWHt\\\\\\\/IMgDQU4abfBkEdFwDMdTyfPNaLVgvheqYonN6ySBgagqO6X9z5gbtp\\\\\\\/zVpTuH\\\\\\\/LqiMt+LCOhfTvxwRLRc6Oolh9jZ2\\\\\\\/\\\\\\\/hox+xt4dOxD7KE+OrSnxemGpYDpvkgerKBRXNa6Y3w2s2pXX1BegilUFhwU2Lq0p24Ge7EMpop9PfW2\\\\\\\/6DtQnEZUnx\\\\\\\/NNbshm7rQWsVBqOU7b7ocnMwJzJtLmzLr0PDuAe5aMPLGMhyVIYq3MEw2xOzHKIH3O9l\\\\\\\/ofJAnwfNBex9mPRAPUTZ7Uha7ftwV8HAZmJmVj\\\\\\\/pUrNuSUkvSk3VGSB5vzviXVlPpbeV+itEocvgTWDuQ\\\\\\\/RVn5VvVgBhC3mEdbGr4t2epnDtd28le5AL\\\\\\\/lyyVvLXGqRt3rLv5NQlA59Z6e8XiXztDEek1j+goETvw0HFvpv9nDWnaR8X7rgHQ0wrX8q\\\\\\\/K5gdtLLwvvyjBSBFD0lDbeuYavrR63H2lWPaGjGIYZYWfs5vft3nBHl\\\\\\\/EScm\\\\\\\/cHHfr7qCL8HtXk0w\\\\\\\/QVyb1bziW+BZczz68vPHFB04cACK5JUHdNZVn8\\\\\\\/fLQit+IQSqoQGN7R94uY6WU+f2QPHVEQOXyoAuDfh66S0ag3YsckZGmTnrnLg31w7\\\\\\\/dvJ26YBdKgUG83sl79gnUpQBiO5yHT+jYeaxEcx19vb8WCTnTahgOVTJTVspEuhEff2OfCPNhbSfHl38uByXJbjyzlLnq\\\\\\\/b83RuQa9gRszvvuuVnQpyQEN4xSHmO7hobbalCIix\\\\\\\/6JDmM5ZF6a5zaU9LN\\\\\\\/GHTniKAK6RuKCcN5x7JnFTqfbmNmItGDqMr7U2VUIFBdpXDRqjP2F\\\\\\\/ve7TWp+p\\\\\\\/J7TUqiwSgFJctw3OS1pTVJLfirsdRIli0LRrMwi\\\\\\\/g7RZeOiKH9bWEJTntpWKntVFyS2Z+xxnhKeg4qAuqJhiBekCD0aoA3Zd\\\\\\\/l87pb+v\\\\\\\/4jdXgyphrNDyhlLrU3EGUp4B2nZNV15cvzFHSAEEEfuiGty6gWcsxcC2P6PoIhPurV1JiWwENKYv981OxynSka8noMcJoUOnFS+AspaAtIjz5Ra+wAV2g5RHjhK9+A4x9LYwfu2eo4lDvLnWHA6oHxxIA38uJQRGlog6czFgr\\\\\\\/LBtyJ4t\\\\\\\/wFoGy2DfjWHQdtBvhR08BGNE7KCIonvnUn8QAcCF6ry6M9OrdnHUbww1fXaXYyELCvn\\\\\\\/CsdkaVQojsF7YjAaxqIIr2qrJhBJEktE3A89EhjXKjZWuP1A0vcq7pb5CDi2fgIDwdL3GCtTdHVx4Au8bU\\\\\\\/8spYCrE27LiJQysLk4za0noHsHmjDcGP941VwOGSkC4eI+2kkdbKi5ne8p\\\\\\\/56kE\\\\\\\/fGokndo1B+Fy7\\\\\\\/qzKK9zVAiP+5DoQ5bbRkyieEazVNvL8d3+kx5gPjDhAVVLRm28bM7kgcxJSngzftqieJvJ0P\\\\\\\/st5MTbOARYOfqRoOSaViK+xJfwg2HNCtumhxRku998M\\\\\\\/TduZ124lPuJao1KU\\\\\\\/aFJu1r5VqWj+b1913feWT9a4\\\\\\\/MVXp\\\\\\\/5qZ+zXZQaDKrXfCKrWAj++eOYBB5ZdDJp\\\\\\\/UpeuojLXE6GtDr7Ouv73HPvMAFrw7eCU\\\\\\\/6mxpKOtoDD8rdJ6B8yTf2TPzzQz8jW\\\\\\\/9KTgjUeVZatjqG6v+xR\\\\\\\/f9cAHQ3ucOgLqqArCLQGJROUDkOHBfUy2es10zjSJeKGvQnkd4mpbff1L98w2N0DjnHeiEVfcwA\\\\\\\/iAWyKxUZfcEdKgIOhH2uHQOJds8yHKl7v2Bcjkl0e03RvlLdOjjJQ9PldDTu6GykrcHOee5bg4DYDSxNgWDgzgpMZJ0z0SIW9xw0nWbXx6MBsAQVVJBP3u7i8QBOIXZLfi6iwvJp2HZ6\\\\\\\/3y5E1h0e1mm1eswVaePHeKxzwaamEGA6yhgwAn6jXsEuMQUalB4GAAyUEJDToQnM\\\\\\\/B521aK4kBBfQXEIBJndqpsgpA9V8qwfNss5+slq+07egpEuhbwifFmb89bj6Yl+6\\\\\\\/EoLBcxYTaFRdGvTkU3k\\\\\\\/OdecSFWbSZkY3iGrw7foGoUTWzKKcumLpOBrVjkNy54\\\\\\\/e6L8wD0ULwwHIAmulc67PC80i8TVIWPNqMro3EQnhxoE\\\\\\\/fuWsEFqM7KR2H1F4QkNHEkrsgiXRqqjrwwAA4D3Z3FPkAui4LlwDXbOWIYuuHj6Cg6w9PnRqYFXhOCCFrTpXYlz87mqCfgAXDNMykkNlfCagefNfHMMA1ImmQwwZtJ9B8eyiW6+FUxolHL\\\\\\\/wOGK4w1hNNvQPSf3YXK3ZipZwLTD1oBSp0431OZbNr9JRllrbiInw9mibNQM9p13IF9T\\\\\\\/OACgNjDQtQiG9gB2SUt+KXb8ToY+bp3Gg256ig\\\\\\\/kaCXjvtGATnfz306CQqSOh7ipjVkng5q\\\\\\\/gi4+JuZiWwla4\\\\\\\/6tx9yi5b69z8\\\\\\\/D3fXAjXYKCztM6TcyrEY29odYnq9ZMPFloga7lLRxolZ3UM0eAUdW2GP2ChXNwouuIT+PUSkvMukhLLQIzM9ok86dZ1iFCFvrQEycIbQL\\\\\\\/6WQSxXAA2mT0TW534u0d56SpGlIqovoKwbaDnhMfPIRQAyaB0DWg1I5VistcZnq8RtgCyNKC58G8Jf5AQhm5UZ+Jt9PDwKicLmi5oYO9\\\\\\\/2tWZLQZ\\\\\\\/IOrH8TDSaJhnUTMVBAVUAbj\\\\\\\/P88zbdJ0vPqyg65Qn9L3IfAzhY4Xqp41BK2c2RjGU\\\\\\\/x1YxAp7Vz0r7o+T3HKUGKioXD7kiIMpsEHDknJsH7lQ7IYdeSJ4mSxjbSh\\\\\\\/swjPpypSrpQ+D9GNjmyLTDSSMTvtoUH82\\\\\\\/FOcPJiCmRQk65pE5eAMH9xvHtC\\\\\\\/Tiw9ghS8L2lomiUQhFDUWvFn1ezsG5ztm3B\\\\\\\/flPebBuk2mpjxPHKi2tNl3R0vUInP9pgcDrda8H7JmgG7tB8NWWPphjgcBnAMWFRPDnWHcRQ6N2ogToA8F1nJXAEr5mvpFgfh\\\\\\\/TzC2cH0P+vNLSU5gZtEqTnx0F9W+SEj7W+xhcAmT3vxl+vNFOwgnTkzQnyanAI\\\\\\\/XItOfH3qX\\\\\\\/9gg2p5i3eDhVOhW8OT0w1TcQIb4AOhXoWl\\\\\\\/mn6pB5Ls\\\\\\\/kc5n2WuoHd9AH1EDHfXwGM4A5rua145Ya+X3wmcpB6kG9vINKQjwDqJEJiC14iPb8LCPAhD37UI41LyBHOxJcL\\\\\\\/0q4x4LCiNEAVlU2Jbf8gsRqaHgF3kd\\\\\\\/pD1ifwqqbQBxxTFqVMIP3\\\\\\\/l42gNXS\\\\\\\/fXJ7f032jJ6QhOSy0AuB4\\\\\\\/FIiZR2XcOaWN8lsyRoaGx9ifNdynPs4E0XMZt85uabN5vFezEXzjm9JuXNVkBpoqA\\\\\\\/YedwDJREDARvnvJoRl+lNE2MGu7qpjo+gmA9MTOnsHlOQK\\\\\\\/k0sHy9P7p5f\\\\\\\/YvE4u5mXBbRRyd1S6EMw7wV8QK8ocekPSMDyq+3xQ\\\\\\\/eQRe515lB0YWzpdxMgQyJiudnSVqYpenm0SLQaO9Om6ZnipPJq7P0dFbcdDye4AboYQIbOiB85Cv+56QelR\\\\\\\/kaGgsvUxxRnQ\\\\\\\/\\\\\\\/euTR67C6JzYcSRqEfygFlME75QK7vDAOm9d4Bs+Ue2+wA20LLJbE56Emajhs6qjDxN28\\\\\\\/0FzGtiYyofYPhIq8HOa\\\\\\\/zWl3aK+TprTgobWc3asuh8Lfe9t4gRwcZoCzhcllWw6QtUgrTI2W7ZJWzOnFsJYaYOP3BKjz+U9L0\\\\\\\/SQl88T0TUfsRgQHL+U69P9dO8obAn265mnB2NozxkFofZQJwEGQTjvMX4Tg5RoChVcz+LMHrFJfuM2mSs+t7jevMZcNgTNePsSZc7UJumCvt5bC46PJRpA2GyFvk9MW0OE+BnM8uoY3RaxMJo2Mkvas8AjTXzP7U77hlczna7FWeab+XJRivfnnhnFirUmNKEAD5gUWqf3bKim7ZZYbzZyM9\\\\\\\/4rPHa0oJpiTB6icY8cU68IMZoLAwv5azqwOF92Ga2niTGzBUXK6pNV0uTAgCQK3E3dGCRnlAk2tq8E11xbpteMxuyvPRnwP7Qz26JxWBreUj7acWPksXNPRkYG2O6AN1E1FdzFlqR0zRV1ziGghkprHjQKDZxL5w06dB\\\\\\\/KXPQbJT+SiV\\\\\\\/jw1oHP2kP7L5vMHI8NELy7S6ZB7g3F\\\\\\\/C0KhtzoXFJJZx7B7tsgEets9qZguMgDYAl8uTSJFehN9p4e2mwe5NECy4PrdIXqSFyh5cbxbV8A7YkA96n3kjkPLcZezcUMQsXh6dJ368sA1QBtZTNDRTm8AG01b\\\\\\\/9oK0uQSfdx3yGDnpP8Ykz9iIqzjhDR1AbUD0t+1S21oyRvH+gAvdSn0Vz4OlXXLnmASr1vc3adjst0Zi+Gp\\\\\\\/3WlTneFTLdaJyDARvFqqlAO7fxDFpgBEewYWVb+15mLyOLWkoKB9O5H3kaQvMA9teO7V4cxwTA5a63mXsD2Vi0eiPIuJj5u\\\\\\\/2rpSufv4xWHjmD907xVXHWyTDIvQO1tWtoqWljv8OnNPDPaUGVA7zjUngDWjO2+KUuRwdJXGB8T\\\\\\\/Ane1\\\\\\\/NnvXZorBThaA5EaoqY7NyI+NoHIGyz3DXXgToMi5AeYEb0yd2J+udqLcL3Y5F2orJS5BtUIgl8fhMe0vY3DHZpJARZ59QRV6wfBjKyVgyiAy8gU8cIZrDeL7sDVYagQzHjqrxqkDFECTlNcGp99EZ\\\\\\\/m7AMEypMR\\\\\\\/k5s\\\\\\\/Of4kZAnnrCOwVpCd7ro4I0dmT7dQoRS1tP9\\\\\\\/11lD1w48BTeB0H0L67b7xVDSx6N5oWmTfQnkgnzARRyaN3NJgyDVuio0F1olkpmwEJSqTF1Cc5uFP4Nda5uO0XPIh4nMsDNqY1iuS4vxb94vq1PYjz\\\\\\\/QF1VI0BotXsrLn5v8GMy\\\\\\\/1VlPfac7jp6UtI7fPWt6DTCzq6cqUfWj1NIvwD1TIMflCYDahDh\\\\\\\/3q8HeqKpj1O4vwMccYkp7PVvyTqou04Qopg69EFnlnkYqKbSqWLSocsvWG8QsY2AlBL8G\\\\\\\/19x1Mfeg40BDHsAGT2vzfZCah29woMoTId\\\\\\\/vRsifzdmBzUZOD9MH2FQSh5E8hI46jcxRIURh5ywnGj18FPhSiIlC9xic77Vr63PxxFn8P74BonfzbC3kSJXghbTN+PisdT104DZomoaeZ8hGhCzsWla+YKExuDF5w+AEgO0FH+VgTfuFVKA8ISvpmbVJzuRtCXNhHNYFyFdjfZxwgBrCPZgxpqY8fKmGH9gWb4H3hbA4bjMYYRgF\\\\\\\/aTuGtDNGMZwrb9fZSxjoxNl1DGoXaT3FVsWDvuvJsSLzgCxur0YJDH1VdOBrg1lyi+FTeydUxi12YvfV4fpATJvTltGoC25aRFFnPL9Fdt5DhetQ78iuu4cjcO3FlE1OdVZyaMVEWTIeUHBICAZ2TBl94ssETiCtWXu\\\\\\\/0GyO\\\\\\\/P\\\\\\\/\\\\\\\/W+Pa7TGbBwA+jXKew+HKiBAWGDCC5JSSYTk5Ten8b23tLzpFHA9vJOFmT87iFU1dhx\\\\\\\/bRd6VC5tFWdmxjMr9vKuatXyCsVheOyQcRQlR9sYZiaXdnQsO7pJffmV5PGfjHnmiLTeWDN3c6oTCrFpNEpXPq50bGEFH\\\\\\\/YELg01TilQqDaQAsXF2ADD1GjM33ymTL3yqRtsTA2+eMtF5XaTcwUpIuodiK5RBWcSGIo98xyLpU3EZ8\\\\\\\/VzqTSSFzq\\\\\\\/mdTbC6+OxGQC287OYOOTJeDabwIP6JDoiRk9pn+DeO2VF2taH7NnxKffNDCha8P8vHwGaSQzbFc2Qpoq9edCmxZ7CPpxmyv4dJKFCvc2+z4zCi1Q+v4tFnyUUQvlI\\\\\\\/60zLqNWbu4y3B\\\\\\\/gOgSx59XbWhoqvYUJwgCfI7fjCh+TA2Xcso1bhKFj5jwNiBEyNSF6vN2Y9wn89NUe+5l8fkKOUA5P7nTdgJfP+CsmVWcyMJHnit2AD+HWGoRFAeiEiTEdXs56nmutyh2hTcF9Sd\\\\\\\/poXPKdhJOk43VL6\\\\\\\/JS6\\\\\\\/d9KFBqrltfV0xqfQ+nPgYPuHiq0Vz61rqPdtuRuWtsFUFyBk9JTHa1nnOO51Okf8EppcISkGAnCcwREsi9iWdQ3EVtHitJD+eUeAHszMaQhBWZHeDp6FFcSD\\\\\\\/fhXr\\\\\\\/pyTK\\\\\\\/X9\\\\\\\/yv2WboIdFxjb5Qq8w7dtJyBdajqu5V0j9j8eUCawGdpfG6yQFovNeXe2EX8gB1y1KywBMKbKZDwd93CEzZOKy3ONidBaDrHtvPe36eqmbNI8jcHv5ZUO0JAg0NxIjEmiHZcVkS4wINjuFTgid9HAfrynwVcH6tuEwg1Pe6Xzrw9PU6Wf60H1zt2cXyJIXikFcU+re92dEHobsZ\\\\\\\/j2lxbz4OF1BkB\\\\\\\/XonOC5Yu1UK4PXE3yxS0vmwb+dEKbzVmEWEfFeo4W8hi1s0HZRkL7OZsPNiX9V3eTljszAvzV+q4YAd3du71RSJN0q9z+0kGYTksoGQ5zs3ZckeolFmBZdSiAi0LvPq0r4EL\\\\\\\/McbAvdtg8njELCJKPG+biFovSn8gk6k\\\\\\\/OtwbQV6dL6w9gyIOtvNamH+GYnbU\\\\\\\/ijdPNvkqmFwAp8dqiyUygH2XrRcvD+tVaK0Jv\\\\\\\/YJd2cPdeyZOSEcA9FikATZIwfh6UNHTP5W7jdj89eJXUkGeHa4US8eOecUtJxJh07WC0sLTf9S4oNjz48OmI8DXW\\\\\\\/hbv4LlbfvoXLjcsyCqXh9S++aUkPnMMJRMlrvjH\\\\\\\/hGF1bgQ4wrrEIS2OHvx1X2BgooIuXTWmwJWlnnB33U44M4qWTSknzKRVmo0z5CHLrRftUjHRxeMpQOYZ4VaffRbD2mPUD\\\\\\\/B7ihTGqYAmmqLjQ3lAtHekgFMXTvhAoA7bLQgWNs\\\\\\\/LPj+5IKlDaWzFhrubo3X6zuJ2G2SQ4TP\\\\\\\/c9hwOKFyQcS4jhSQu09N\\\\\\\/mzshVQ\\\\\\\/cYIKJye4zrU2AW\\\\\\\/4flLcl8j73I5KnwipkY9n9hPUX8tmfRhGY7plvjQkvtmNzKzA3QRCTgVdswQ4QtQUwbFx3FbKfYIYQry8ck4HyCVwUwFstsQ1F76gcSNJ6eQWGD+6vHmyGOWTugD+q0hBGqYXUe2Zk3DonZlnoC1vWJHogZdKMl1RtymllVL6sBnKROhKbqt8RTK0LLez5NrUWNKSkQNCACR5uxKiLe0+vwuQfDiqZ0cY0r5IJ9PKlBFUIXecmShTcEgkNTzmqtAvL4nVLUuCzBRdKZXtBiAUvUNkICLCZR89CBVHDGqxdkECJbzeefTG+wsI1WFwVkSS6NwToCHy3qlJvcx1f939YFSxnGcZaib9sMy+BIw4UytnzeI3EzayoOnHYnzOkX1cZmSP4bWe12XbF+XKCLq0DTv4AczN8SzhE51i1X88blfpBFP7LXtO9Gw0C2w45Augb6TMVH3Z1Wc3FvVa3D7WNSF7e2RBYvTk\\\\\\\/MkFZpegEVvs6NQzusFEVrGt9V0MoS+OuwExw+oItCuVCiauH+\\\\\\\/uXLoamEB\\\\\\\/i4VCR59SKbQgLAXrjsaqwkelcWDyr3ZJuVqLIYmUUSc08+RZ+LTkQ+CinRX6jhJ8mBlCOCprBQHcDh\\\\\\\/QA8k\\\\\\\/bspL+tZZvBucC0IcLWBC85urPG7Fe25If+f7bPbO3oRstTb9vc3\\\\\\\/jljI2rvSicUIbf1w1JL6j+UjzVHPmYl27CoD2aw7PZxSuepDAnoJIjEqc6yr5WcATx0FR5VIiAtdeeiorkIGCMNNlv5pg02N2ci+eLS\\\\\\\/vmaW+kahy6F1i6ZweFndB\\\\\\\/g\\\\\\\/\\\\\\\/38IkjiJJ58S6w7zGazMUES9s+N8+F6kef9NX6y9\\\\\\\/dDM\\\\\\\/kscL8J\\\\\\\/V7wLN3\\\\\\\/lTDlbkZvrjwOvlKJ2HvcSikyH\\\\\\\/JJvPu8193p1pNpj\\\\\\\/F7YMC2WLZXei9n26q30LU2R3eldw+nM6PDRb9wkibaGMUjalR1ZeIaXQFH6ZgkSOuvnCwcn47kiRjBRLsP1Y94yBO2JVv9epe+Wpx4SCHSSaxzF\\\\\\\/HbloouVGR1wBp2TXwpdTYm3+zmfoztSpglXitA7P7xSgRzn0X34BYZl7hEL8UnXeZV01nuCdxZjK3XzwJTWjvoJsbXBQKznkUHhK9+kqG8ktcuSr0s8Fk+uSWUgUE\\\\\\\/zz1JKSfmwBKIC050L+SIvcGeRTW94BsmGtCJ0ar8BIkFvYdG7hq0iX45qs71r9X+5teqxL48fKXvv0p8TdQ1QPR0y9g6WpQu0E+OQK5kWA9o1sRVijunZxNorF\\\\\\\/KcQW1O1IzOFbp4sA5yR8\\\\\\\/GZfLLcridrRMNJojTcsjCGETLk0Ps4VvXukbCrvgoWkdAq+\\\\\\\/mJWTr3ha\\\\\\\/7abvH1bq1Zu\\\\\\\/5Po9RqXTm6D0WuGu7KQb5qPzno0l+1roRpqpBwlOaV\\\\\\\/fI7SfHlhNJVskHKr7fzrVEEFZR+eFsTIvKS0zyjg2wbIacWZb9hH2OZD0JcxhuV4qSD6D0sNDNJvF0Tfj+rYFXbRw68apDj9biQQ7Lez5jFGS85U7DRX+BZ+ZmkKf9a5DJYvX38wrvdoCwUigUVDkQLKKqmMZiNQ9I39lpOjNdmwu1utGS3vH21T\\\\\\\/XCgTdQfHihEpK0U6to5mPMBN60\\\\\\\/XJDFATGYIhgRcO3pHqqF8JzP3jdqIk8yyatbOshjsCFLvtFeimhLRg7VO1g3qrejCuLGBEnQ2qpxBFeY1GLVzZ7JEYVNXZ4YkGHoakVokZcAcVOJaezr8qSQjYepe4y9467cjbHuXqMUnQdFIWG8WrIo46lk1nAyTdzZKgezNkjQYrwqYynBI8vfYGQqgFOTYdR4B7eH9awU2HOaUDkKRgSuiCe+\\\\\\\/gUZiTcG7HM1o3Rsg2c9hxXRoXXAZoJjScmvmlADe7+43fUauu3iVh3rlmZ5SupVHP3Bp01VPSsHkNvhQEToianL7bYpR57+UeQ+rAXvXrGJeKPKe6Dubyakvl7dHUWeQSiD5P2eGhAP78HtSe4nLVfDXOc3qAkkTofsgKyYqjWgpXSRAnqMZDq8H182f5ocWgwSPI4e9FhTV025HU6CZ0GXi9ytxkda069Ise3nHksy9RxIXZAMC90B27AhEfCCcwJG60YC03FWp2HQC9D4zX5\\\\\\\/cDyhvCRFbtCA2u+QOVG+YXw4bPVlCQQb5bficYtzcz+QO35wFCjX8Q==\\\u0022,\\\u0022iv\\\u0022:\\\u0022EdC1FLAPIDKPOAyY5nK1ng==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002211451dac152556a141b48830103f19ab2cb185353e89ee611260a02e176cf21a\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022pCCeziLGIGCJM6m2NuLw80lUPm9ZTzFgGIKuq9AJQ4KxbJ8jrVKiE73SKneFDMsK4tUwCfnvDu2zN4+CqUhb+VbmCW6BI\\\\\\\/6o8BgY4czGV4fbLjZiS\\\\\\\/3QDx9e\\\\\\\/NosSHX\\\\\\\/JV6zk+Sol3\\\\\\\/M5ERUqWX+6fgW\\\\\\\/Poix30YXiB7cZMnFUX90HHJ+6JI9efj99J0Q5505\\\\\\\/cA0XU9vnpd8OIxv7mKzgHlUF9SRKNQM15axOTmdpO+kNcXLYGDHuqg6KCuYqGxo8t5D\\\\\\\/p1g5aieE1L9BStcTddMpl0k6pUb7s5Tbf9\\\\\\\/WeGtMHN8NSlMfCEfPmnp8Y+E8qOqNQM\\\\\\\/8XObpgzUXC1rajvgFcVbf3zUfLLm3JwiEuIDNQXfhHYgOhej7IHoxIHDKMDuCFE\\\\\\\/KtdLCC22RS0ZCBxJeiehxZ4JtOcp9F95GA1apTcFMUUR85NQXzPOP6dfqFHIe0yAyfqpU7wxRa\\\\\\\/RpA0In47FLfKskp96+IBybbIYT9+VFOGflRZ2qFduShRSSSw58qrsN9fM4fbZPYfuOCpwgt\\\\\\\/v4YN4M7Ee6XUt93IBCvVlgS+mqbAVGmR0t8TCF5SFpXnyeSefyIXnhY5nt82Wbx64wlzpruGrEq57aOvvK5WTq9DBIy6OA7WkIxcPPYU6cOIT+Q+MSI1o6Ln\\\\\\\/lG4l1UIDdz\\\\\\\/7e5JNrh0XAJJ7kM81tKfO7yXW3BGKmh5M7wDAbGdaLBMsWeIhqZNTujhAt5pUZUNcg2z+9nez1FQhQAAGMSt3Umq4rP9jheTqfatX0l0UqRh2Z6YPlI7LP78NcOIphZLufTX1vlJOlHqT5\\\\\\\/TBFaCeNo6JcA\\\\\\\/cBi8dXEZZgFFONm6Z3N7JS6fLRaJ8Fcvn2vwYuxHWaBCIu180hhmYxcOGMhzgHKKC7liZTda4B2iHa38vWAJ4vpxkhsf8qYjjol95DNEOGnP8JPxGhjvb7EXpL7vMAu6p2XS4laMVg9vuyqZ34HCE9scOfguK67tTa25Nlm+gFCT1+ihsrf948quc4TBHe6ywd2+xB+w80Gym3gaGHwaj80GCNDmWXZ5nL9dkAfKlXRMLSdCYDSuRqLUjDFm5vz0\\\\\\\/uyk2BklPobCt58QHxR+lEIxk8JfTgjHuLuvi9n6e4nf8r8aI0k79q3gx2DztJ6WW4hGCOlVv8I+81xIQta8W4yFZzGhdGRn7yCL+faa3w7ZrT7AWLXYhkI468wSSucxLWfPTQF+MqkCJwnLLEreA2wfb88c2kKqLDzAyphTfbxwfR\\\\\\\/ldkAmUqGcvbVe\\\\\\\/R5h5Uhj+\\\\\\\/OZDzFDGW1dMVSrTsWpiAZNv45eKSmGeIEsXiGmEDCYbE3escp0MM7fSLWMxgJ5hW8rr7xd273+NWVg5fii02U5ArzmhFcWvqSL7r+uiY3JArR4Dq2CRa40+BM4nUT4amyItGw6g+41UJPJIA6Gf4WF\\\\\\\/9ltPj8w9DaMbVqiaxH8oCydBd4l\\\\\\\/G4bkqyfGPllHjYc04iCU7DRux5z4+iBjQ102iwo5MB29qY58qk8jWIzuImm\\\\\\\/5WgoJtQHDMAJCde\\\\\\\/s0yr5FBMyZnjE2bdJdZhbrlHwsu9mzgSDAx+jzMWU4Geos13e6E8B0Q\\\\\\\/4+2Pg08tTT6nO2Pgh167UPRRqQ0eKuH4GVvMPiWZpM+yxkMUwFQZcBgB1ScX2SGH5H2wn3rswE3Y6N3QHwWYf1ZbDrjMjV+Gbxjbr9uJpAFgSm2JJDGzE\\\\\\\/FWdqBWKeE5LcweLms3O1OzlP96HruMni3YiLxDasbXVEX2N+2qpjWLUqwxh1d39L\\\\\\\/wJH2G81i2aZuRQb6p\\\\\\\/wx1u3hf5irN4ABW\\\\\\\/1FyjqMfMlEJSmsA4a7FLOOV2a5D\\\\\\\/F3eTElfHgmz0Nkzj5mHtqDAtmYjbt0GhfxjY\\\\\\\/gcbktVNxtZXTcqGIGT07iGRWkrR6xniT8cATymZczoXhuWcfdPMbtKz8gU0pZN7a59oJFWP9Z55kG1JXTZmz6QpbBeNgz3RIpYwfnIdwPKOp6qHX1vu2qj35U0mO4o1vu5g==\\\u0022,\\\u0022iv\\\u0022:\\\u00226\\\\\\\/LkUwcuaXXpjE18By1nuA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00227ce0844ae025f69434e77b729b24e4a6c1ad8e5b2e6e747634e33904d5a19c35\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459592 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/12/2313" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-CrYDOTe_nF9fhwVs86Db5T5WkAtdhYXnB5TFiROIE6s" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/12/2313" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-pQj1Eo_aT9h69VMSoTzgksX3um0xluWfQuW4XUWIaF0" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/12/2305" title="Dynamics of Adaptive Alleles in Divergently Selected Body Weight Lines of Chickens" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/12/2321" title="Saccharomyces cerevisiae Genes Involved in Survival of Heat Shock" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459592" id="node459592" data-pisa="ggg;3/12/2313" data-pisa-master="ggg;g3.113.008615" data-apath="/ggg/3/12/2313.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Rapid Recombination Mapping for High-Throughput Genetic Screens in <em>Drosophila</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Anne L. Sapiro</span>, <span class="highwire-citation-author" data-delta="1">Robert J. Ihry</span>, <span class="highwire-citation-author" data-delta="2">Derek L. Buhr</span>, <span class="highwire-citation-author" data-delta="3">Kevin M. Konieczko</span>, <span class="highwire-citation-author" data-delta="4">Sarah M. Ives</span>, <span class="highwire-citation-author" data-delta="5">Anna K. Engstrom</span>, <span class="highwire-citation-author" data-delta="6">Nicholas P. Wleklinski</span>, <span class="highwire-citation-author" data-delta="7">Kristin J. Kopish</span> and <span class="highwire-citation-author" data-delta="8">Arash Bashirullah</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2313-2319;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.008615 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459592" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Anne L. Sapiro </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Division of Pharmaceutical Sciences, University of Wisconsin-Madison, Madison, Wisconsin 53705-2222</div><div class='author-affiliation'><span class='nlm-sup'>†</span>College of Letters and Sciences or College of Agriculture and Life Sciences, University of Wisconsin-Madison, Madison, Wisconsin 53705-2222</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anne%2BL.%2BSapiro%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sapiro%20AL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnne%2BL.%2BSapiro%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Robert J. Ihry </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Division of Pharmaceutical Sciences, University of Wisconsin-Madison, Madison, Wisconsin 53705-2222</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Cellular and Molecular Biology Graduate Program, University of Wisconsin-Madison, Madison, Wisconsin 53705-2222</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BJ.%2BIhry%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ihry%20RJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARobert%2BJ.%2BIhry%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Derek L. Buhr </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Division of Pharmaceutical Sciences, University of Wisconsin-Madison, Madison, Wisconsin 53705-2222</div><div class='author-affiliation'><span class='nlm-sup'>†</span>College of Letters and Sciences or College of Agriculture and Life Sciences, University of Wisconsin-Madison, Madison, Wisconsin 53705-2222</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Derek%2BL.%2BBuhr%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Buhr%20DL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADerek%2BL.%2BBuhr%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Kevin M. Konieczko </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Division of Pharmaceutical Sciences, University of Wisconsin-Madison, Madison, Wisconsin 53705-2222</div><div class='author-affiliation'><span class='nlm-sup'>†</span>College of Letters and Sciences or College of Agriculture and Life Sciences, University of Wisconsin-Madison, Madison, Wisconsin 53705-2222</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kevin%2BM.%2BKonieczko%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Konieczko%20KM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKevin%2BM.%2BKonieczko%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Sarah M. Ives </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Division of Pharmaceutical Sciences, University of Wisconsin-Madison, Madison, Wisconsin 53705-2222</div><div class='author-affiliation'><span class='nlm-sup'>†</span>College of Letters and Sciences or College of Agriculture and Life Sciences, University of Wisconsin-Madison, Madison, Wisconsin 53705-2222</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sarah%2BM.%2BIves%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ives%20SM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASarah%2BM.%2BIves%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Anna K. Engstrom </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Division of Pharmaceutical Sciences, University of Wisconsin-Madison, Madison, Wisconsin 53705-2222</div><div class='author-affiliation'><span class='nlm-sup'>†</span>College of Letters and Sciences or College of Agriculture and Life Sciences, University of Wisconsin-Madison, Madison, Wisconsin 53705-2222</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anna%2BK.%2BEngstrom%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Engstrom%20AK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnna%2BK.%2BEngstrom%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Nicholas P. Wleklinski </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Division of Pharmaceutical Sciences, University of Wisconsin-Madison, Madison, Wisconsin 53705-2222</div><div class='author-affiliation'><span class='nlm-sup'>†</span>College of Letters and Sciences or College of Agriculture and Life Sciences, University of Wisconsin-Madison, Madison, Wisconsin 53705-2222</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nicholas%2BP.%2BWleklinski%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wleklinski%20NP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANicholas%2BP.%2BWleklinski%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Kristin J. Kopish </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Division of Pharmaceutical Sciences, University of Wisconsin-Madison, Madison, Wisconsin 53705-2222</div><div class='author-affiliation'><span class='nlm-sup'>†</span>College of Letters and Sciences or College of Agriculture and Life Sciences, University of Wisconsin-Madison, Madison, Wisconsin 53705-2222</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kristin%2BJ.%2BKopish%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kopish%20KJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKristin%2BJ.%2BKopish%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Arash Bashirullah </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Division of Pharmaceutical Sciences, University of Wisconsin-Madison, Madison, Wisconsin 53705-2222</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Cellular and Molecular Biology Graduate Program, University of Wisconsin-Madison, Madison, Wisconsin 53705-2222</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Arash%2BBashirullah%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bashirullah%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AArash%2BBashirullah%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:bashirullah@wisc.edu" class="" data-icon-position="" data-hide-link-title="0">bashirullah@wisc.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/3/12/2313" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459592" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459592/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/12/2313.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459592" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459592/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/12/2313.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459592" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459592/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/12/2313.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459592" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459592/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/3/12/2313.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459592" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459592/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-10">Mutagenesis screens are a staple of classical genetics. Chemical-induced mutations, however, are often difficult and time-consuming to identify. Here, we report that recombination analysis with pairs of dominant visible markers provides a rapid and reliable strategy to map mutations in <em>Drosophila melanogaster</em>. This method requires only two generations and a total of six crosses in vials to estimate the genetic map position of the responsible lesion with high accuracy. This genetic map position can then be reliably used to identify the mutated gene through complementation testing with an average of nine deficiencies and Sanger sequencing. We have used this approach to successfully map a collection of mutations from an ethyl methanesulfonate−based mutagenesis screen on the third chromosome. We propose that this method also may be used in conjunction with whole-genome sequencing, particularly when multiple independent alleles of the mutated locus are not available. By facilitating the rapid identification of mutated genes, our mapping strategy removes a primary obstacle to the widespread use of powerful chemical mutagenesis screens to understand fundamental biological phenomena.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/dominant-markers" class="hw-term hw-article-keyword hw-article-keyword-dominant-markers" rel="nofollow">dominant markers</a></li><li class="kwd"><a href="/keyword/ems-mutagenesis" class="hw-term hw-article-keyword hw-article-keyword-ems-mutagenesis" rel="nofollow">EMS mutagenesis</a></li><li class="kwd"><a href="/keyword/persistent-salivary-glands-psg" class="hw-term hw-article-keyword hw-article-keyword-persistent-salivary-glands-psg" rel="nofollow">persistent salivary glands (PSG)</a></li></ul><p id="p-11">Open-ended forward genetic screens remain one of the most powerful genetic approaches to identify critical, and often unexpected, regulators of biological processes. These mutagenesis screens, however, have fallen out of favor because identifying the mutated genes can be a slow and tedious process. The available methods for identifying chemical- and radiation-induced mutations fall into two broad categories, those that rely on recombination analysis and those that do not. Direct mapping methods that do not use recombination mapping are often labor-intensive and/or expensive, making these approaches suitable only for mapping a small number of mutations of high interest. For example, crossing a mutation to all deficiencies that span a chromosome often works, but with the large number of deficiencies required to cover the chromosome, simultaneously mapping many mutations becomes impractical. In theory, whole-genome sequencing (WGS) provides an ideal method for mapping mutations; in practice, however, given that a stock identified in an ethyl methanesulfonate (EMS) mutagenesis screen may reveal hundreds of coding-changing lesions, WGS approaches may require multiple alleles to unambiguously identify the lesion of interest (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Blumenstiel <em>et al.</em> 2009</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Osterloh <em>et al.</em> 2012</a>).</p><p id="p-12">Mapping methods that rely on recombination analysis offer a reliable way to map mutations and have been routinely used, essentially unchanged, since Sturtevant proposed the method 100 years ago (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Sturtevant 1913</a>). In these methods, the frequency of chromosomal exchange with respect to a reference locus provides an estimated distance along the chromosome. The resolution of recombination-based mapping approaches, however, is determined by the density of markers used and the number of recombinants generated and examined. High density can be achieved with molecular markers such as single-nucleotide polymorphisms (SNPs). The use of SNPs for recombination mapping can identify locations within 50 kb but require hundreds of SNP-detection reactions involving polymerase chain reaction and sequencing on a large number of recombinants, making it expensive and labor-intensive (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Martin <em>et al.</em> 2001</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Chen <em>et al.</em> 2008</a>). Similarly, the use of molecularly defined <em>P</em>-elements for recombination mapping also can provide high resolution, but this degree of resolution requires scoring approximately 10,000 F2 progeny (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Zhai <em>et al.</em> 2003</a>). Thus, neither of these recombination-based methods is easily scalable for mapping large collections of mutations generated in forward genetic screens. Moreover, standard mapping strategies using recessive markers, although scalable, cannot achieve comparable resolution and often are equally labor-intensive.</p><p id="p-13">Here, we describe the use of pairs of dominant visible markers as a way to rapidly perform recombination analysis. Compared with mapping with recessive markers, the use of dominant markers offers the ability to reduce the number of generations and crosses required to estimate genetic map positions. Although the use of dominant markers for recombination analysis dates back to the days of the first “fly room” (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bridges and Morgan 1923</a>), the use of these markers for mapping was likely abandoned because good dominant markers are rare and thus cannot provide sufficient density for accurate mapping. However, in our strategy, the goal of simultaneously using pairs of dominant markers is not to identify the true genetic map position of a mutation, but instead to rapidly estimate its position relative to the two markers and use this information to guide subsequent steps with either WGS or molecularly-defined deficiencies. We demonstrate the usefulness of using pairs of dominant markers through the mapping of a collection of mutations identified in an EMS-based mutagenesis screen. We had previously identified 38 complementation groups on the third chromosome that selectively disrupt the ecdysone-triggered destruction of larval salivary glands, resulting in a persistent salivary gland (PSG) phenotype (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Wang <em>et al.</em> 2008</a>). Thirty of these complementation groups were represented by single alleles. Employing pairs of dominant markers, we estimated the genetic map positions of recessive lethal mutations in two generations and six crosses. The positional information obtained is of sufficiently high resolution to map the mutated gene through complementation tests with less than 10 deficiencies. Thus, the use of pairs of dominant markers overcomes existing barriers to the identification of genes through forward genetic approaches by providing a fast, simple, and affordable alternative for recombination mapping.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Fly stocks and husbandry</h3><p id="p-14">The following stocks with dominant markers were obtained from the Bloomington Drosophila Stock Center (BDSC): R,D (FBst0001689), Gl,Sb (FBst0000510), Sb,H (FBst0000586), and H,Pr (FBst0000516). Also from BDSC but not listed are the BSC (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Cook <em>et al.</em> 2012</a>), DrosDel (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Ryder <em>et al.</em> 2007</a>), and Exilixis (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Parks <em>et al.</em> 2004</a>) stocks carrying molecularly defined deletions, included in the deficiency kit on the third chromosome (DK3), that were used for complementation tests. The mutations with a PSG phenotype were identified in a large-scale EMS-mutagenesis screen on the third chromosome (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Wang <em>et al.</em> 2008</a>). All flies were raised on cornmeal molasses media with yeast, and crosses were performed at room temperature with three to six virgins and at least two males per vial.</p></div><div id="sec-3" class="subsection"><h3>Scoring dominant markers</h3><p id="p-15">Recombinant F2 progeny were scored for presence of the mapping dominant markers and for absence of the balancer dominant marker (see genetic scheme in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>). Phenotypes of dominant markers used for mapping are as follows: Roughened (R), scored for rough eye; Dichaete (D), scored for extended wings and missing alulae; Glued (Gl), scored for smaller rough eyes; Stubble (Sb), scored for short and thick bristles on the notum; Hairless (H), scored for loss of the postvertical bristles in the head; and Prickly (Pr), scored for short and thin-tipped bristles on the notum (of note, H and Pr together cause a loss of bristles). For the purpose of this mapping method, the relevant F2 progeny are those that have lost one or both markers; for example, when the Sb,H pair for mapping is used, the relevant progeny to be scored are Sb,+, +,H and +,+ (the remaining progeny are not needed to calculate relative position; see <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2B</a>). The balancer used was marked by Drop (TM6B,Dr), which shows a near-complete ablation of the eyes. The use of the Dr-containing balancer facilitates sorting of F2 progeny (the Dr eye phenotype is considerably stronger than those of R and Gl and thus easily identified even in their presence). Although we describe this recombination mapping method for the third chromosome, the principle of using pairs of dominant markers can be applied for mapping mutations on other chromosomes. The second chromosome has a number of good dominant markers that can be used for recombination mapping (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008615/-/DC1/008615SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008615/-/DC1/TableS1.pdf">Table S1</a> for examples). Moreover, even though dominant visible mutations are rare on the X chromosome, the principle of pairs of dominant markers can be applied by scoring recessive markers in male recombinant progeny.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/12/2313/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Using pairs of dominant markers for recombination analysis. The figure illustrates the concept of recombination analysis with pairs of dominant markers, using the Sb,H pair as an example. (A) In the parental cross (P), mutant animals are crossed to a stock that contains two dominant markers on the same chromosome, and the resulting female progeny are selected for presence of these markers and the chromosome carrying the mutation of interest (i.e., no balancer; in this case, no Dr progeny). In the F1 cross, these females are then backcrossed to males from the mutant stock, and the viable nonbalancer F2 progeny are scored for the loss of Sb and/or H (i.e., only score Sb,+, +,H and +,+ progeny). (B-C) The panels illustrate the possible recombination events between the marked and mutant chromosomes depending on whether the mutation (indicated with a purple asterisk) is located inside (B) or outside (C) of the two pairs of markers. (B) If a recombination event occurs in &#x201C;a&#x201D; or &#x201C;d&#x201D; (outside the pair), the recombinant chromosome will be viable, resulting in Sb,H progeny. If recombination occurs in &#x201C;b,&#x201D; the recombinant chromosome containing only Sb will be lethal while the recombinant progeny containing only H will be viable, resulting in +,H progeny. Conversely, if the recombination occurs in &#x201C;c,&#x201D; the recombinant chromosome containing Sb will be viable whereas the recombinant progeny containing H will now be lethal, resulting in Sb,+ progeny. The ratio of these recombination &#x201C;splits&#x201D; between markers reflects the relative position of the mutation. In addition, an important consequence of the mutation being inside is that viable progeny that have lost both markers (+,+ progeny) will only appear if a rare double recombination event occurs in &#x201C;b&#x201D; and &#x201C;c.&#x201D; (C) As with the preceding example, recombination events in &#x201C;e&#x201D; and &#x201C;h&#x201D; will result in viable Sb,H progeny. However, recombination events in &#x201C;f&#x201D; will result in only +,H viable progeny (Sb,+ progeny would only be viable in the case of a double recombination event in &#x201C;f&#x201D; and &#x201C;g&#x201D;). Importantly, the distinguishing characteristic of mutations located outside the pair of markers, unlike those located inside, is that unmarked progeny (+,+) can be generated by a single recombination event (in &#x201C;g&#x201D;). Thus, the number of viable recombinant progeny that have lost one or both markers is used in the following manner: (1) if the unmarked progeny is absent or rare, then the mutation is inside the markers and the ratio of the &#x201C;splits&#x201D; estimates the relative position of the mutation within the pair; and (2) if the unmarked progeny is common, then the mutation is outside the markers and the ratio of the &#x201C;splits&#x201D; indicates direction of the mutation. In the example shown, Sb,+ would be less frequent than +,H indicating that the mutation is to the right of the pair. Given that viable Sb,H progeny can also result in the absence of recombination, we do not score this class for mapping." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-763482392" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Using pairs of dominant markers for recombination analysis. The figure illustrates the concept of recombination analysis with pairs of dominant markers, using the Sb,H pair as an example. (A) In the parental cross (P), mutant animals are crossed to a stock that contains two dominant markers on the same chromosome, and the resulting female progeny are selected for presence of these markers and the chromosome carrying the mutation of interest (&lt;em&gt;i.e.&lt;/em&gt;, no balancer; in this case, no Dr progeny). In the F1 cross, these females are then backcrossed to males from the mutant stock, and the viable nonbalancer F2 progeny are scored for the loss of Sb and/or H (&lt;em&gt;i.e.&lt;/em&gt;, only score Sb,+, +,H and +,+ progeny). (B-C) The panels illustrate the possible recombination events between the marked and mutant chromosomes depending on whether the mutation (indicated with a purple asterisk) is located inside (B) or outside (C) of the two pairs of markers. (B) If a recombination event occurs in &#x201C;a&#x201D; or &#x201C;d&#x201D; (outside the pair), the recombinant chromosome will be viable, resulting in Sb,H progeny. If recombination occurs in &#x201C;b,&#x201D; the recombinant chromosome containing only Sb will be lethal while the recombinant progeny containing only H will be viable, resulting in +,H progeny. Conversely, if the recombination occurs in &#x201C;c,&#x201D; the recombinant chromosome containing Sb will be viable whereas the recombinant progeny containing H will now be lethal, resulting in Sb,+ progeny. The ratio of these recombination &#x201C;splits&#x201D; between markers reflects the relative position of the mutation. In addition, an important consequence of the mutation being inside is that viable progeny that have lost both markers (+,+ progeny) will only appear if a rare double recombination event occurs in &#x201C;b&#x201D; and &#x201C;c.&#x201D; (C) As with the preceding example, recombination events in &#x201C;e&#x201D; and &#x201C;h&#x201D; will result in viable Sb,H progeny. However, recombination events in &#x201C;f&#x201D; will result in only +,H viable progeny (Sb,+ progeny would only be viable in the case of a double recombination event in &#x201C;f&#x201D; and &#x201C;g&#x201D;). Importantly, the distinguishing characteristic of mutations located outside the pair of markers, unlike those located inside, is that unmarked progeny (+,+) can be generated by a single recombination event (in &#x201C;g&#x201D;). Thus, the number of viable recombinant progeny that have lost one or both markers is used in the following manner: (1) if the unmarked progeny is absent or rare, then the mutation is inside the markers and the ratio of the &#x201C;splits&#x201D; estimates the relative position of the mutation within the pair; and (2) if the unmarked progeny is common, then the mutation is outside the markers and the ratio of the &#x201C;splits&#x201D; indicates direction of the mutation. In the example shown, Sb,+ would be less frequent than +,H indicating that the mutation is to the right of the pair. Given that viable Sb,H progeny can also result in the absence of recombination, we do not score this class for mapping.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/12/2313/F1.medium.gif" width="399" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/3/12/2313/F1.medium.gif" width="399" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/12/2313/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/12/2313/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463644" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-16" class="first-child">Using pairs of dominant markers for recombination analysis. The figure illustrates the concept of recombination analysis with pairs of dominant markers, using the Sb,H pair as an example. (A) In the parental cross (P), mutant animals are crossed to a stock that contains two dominant markers on the same chromosome, and the resulting female progeny are selected for presence of these markers and the chromosome carrying the mutation of interest (<em>i.e.</em>, no balancer; in this case, no Dr progeny). In the F1 cross, these females are then backcrossed to males from the mutant stock, and the viable nonbalancer F2 progeny are scored for the loss of Sb and/or H (<em>i.e.</em>, only score Sb,+, +,H and +,+ progeny). (B-C) The panels illustrate the possible recombination events between the marked and mutant chromosomes depending on whether the mutation (indicated with a purple asterisk) is located inside (B) or outside (C) of the two pairs of markers. (B) If a recombination event occurs in “a” or “d” (outside the pair), the recombinant chromosome will be viable, resulting in Sb,H progeny. If recombination occurs in “b,” the recombinant chromosome containing only Sb will be lethal while the recombinant progeny containing only H will be viable, resulting in +,H progeny. Conversely, if the recombination occurs in “c,” the recombinant chromosome containing Sb will be viable whereas the recombinant progeny containing H will now be lethal, resulting in Sb,+ progeny. The ratio of these recombination “splits” between markers reflects the relative position of the mutation. In addition, an important consequence of the mutation being inside is that viable progeny that have lost both markers (+,+ progeny) will only appear if a rare double recombination event occurs in “b” and “c.” (C) As with the preceding example, recombination events in “e” and “h” will result in viable Sb,H progeny. However, recombination events in “f” will result in only +,H viable progeny (Sb,+ progeny would only be viable in the case of a double recombination event in “f” and “g”). Importantly, the distinguishing characteristic of mutations located outside the pair of markers, unlike those located inside, is that unmarked progeny (+,+) can be generated by a single recombination event (in “g”). Thus, the number of viable recombinant progeny that have lost one or both markers is used in the following manner: (1) if the unmarked progeny is absent or rare, then the mutation is inside the markers and the ratio of the “splits” estimates the relative position of the mutation within the pair; and (2) if the unmarked progeny is common, then the mutation is outside the markers and the ratio of the “splits” indicates direction of the mutation. In the example shown, Sb,+ would be less frequent than +,H indicating that the mutation is to the right of the pair. Given that viable Sb,H progeny can also result in the absence of recombination, we do not score this class for mapping.</p><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/12/2313/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Use of pairs of dominant markers to map a lethal mutation on the third chromosome. An example of the mapping process illustrating the effectiveness of using pairs of dominant markers. (A) The analysis described in Figure 1 is conducted with four pairs of dominant markers (R,D, Gl,Sb, Sb,H, and H,Pr) to map a lethal mutation on the third chromosome, psg24 (indicated with a purple asterisk). Scoring the viable F2 progeny indicates that only one pair has no unmarked progeny, thus the mutation is located inside the H,Pr pair. Consistent with this interpretation, the ratio of &#x201C;splits&#x201D; in the R,D, Gl,Sb and Sb,H crosses point to the H,Pr region. (B) The recombinant &#x201C;splits&#x201D; in H,Pr are used to calculate the approximate location of psg24. The formula provides the relative distance of the mutation from the left marker. This distance is indicated by the frequency of loss of the left marker among the viable F2 progeny &#x201C;splits.&#x201D; In this case, the estimated genetic map position for psg24 is approximately 80 cM. This genetic location is then used to estimate a cytological location with positional information of known genes (see File S1), estimating the physical location of psg24 to around 95A. (C) We used complementation tests with deficiencies near 95A to identify the actual location of psg24. The mutation was crossed to 10 deficiencies from the DK3 collection spanning the region from 94A to 96C, and it failed to complement Df(3R)BSC619 in 94E, which is about 1 cM or approximately one deficiency away from the initial estimated physical map position. The arrow reflects the approximate reliability of the recombination analysis, where the base of the arrow (dot) represents the estimated genetic map position and the arrowhead represents the actual physical location." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-763482392" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Use of pairs of dominant markers to map a lethal mutation on the third chromosome. An example of the mapping process illustrating the effectiveness of using pairs of dominant markers. (A) The analysis described in Figure 1 is conducted with four pairs of dominant markers (R,D, Gl,Sb, Sb,H, and H,Pr) to map a lethal mutation on the third chromosome, &lt;em&gt;psg24&lt;/em&gt; (indicated with a purple asterisk). Scoring the viable F2 progeny indicates that only one pair has no unmarked progeny, thus the mutation is located inside the H,Pr pair. Consistent with this interpretation, the ratio of &#x201C;splits&#x201D; in the R,D, Gl,Sb and Sb,H crosses point to the H,Pr region. (B) The recombinant &#x201C;splits&#x201D; in H,Pr are used to calculate the approximate location of &lt;em&gt;psg24&lt;/em&gt;. The formula provides the relative distance of the mutation from the left marker. This distance is indicated by the frequency of loss of the left marker among the viable F2 progeny &#x201C;splits.&#x201D; In this case, the estimated genetic map position for &lt;em&gt;psg24&lt;/em&gt; is approximately 80 cM. This genetic location is then used to estimate a cytological location with positional information of known genes (see File S1), estimating the physical location of &lt;em&gt;psg24&lt;/em&gt; to around 95A. (C) We used complementation tests with deficiencies near 95A to identify the actual location of &lt;em&gt;psg24&lt;/em&gt;. The mutation was crossed to 10 deficiencies from the DK3 collection spanning the region from 94A to 96C, and it failed to complement Df(3R)BSC619 in 94E, which is about 1 cM or approximately one deficiency away from the initial estimated physical map position. The arrow reflects the approximate reliability of the recombination analysis, where the base of the arrow (dot) represents the estimated genetic map position and the arrowhead represents the actual physical location.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/12/2313/F2.medium.gif" width="440" height="378"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/3/12/2313/F2.medium.gif" width="440" height="378"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/12/2313/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/12/2313/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463664" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-17" class="first-child">Use of pairs of dominant markers to map a lethal mutation on the third chromosome. An example of the mapping process illustrating the effectiveness of using pairs of dominant markers. (A) The analysis described in <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a> is conducted with four pairs of dominant markers (R,D, Gl,Sb, Sb,H, and H,Pr) to map a lethal mutation on the third chromosome, <em>psg24</em> (indicated with a purple asterisk). Scoring the viable F2 progeny indicates that only one pair has no unmarked progeny, thus the mutation is located inside the H,Pr pair. Consistent with this interpretation, the ratio of “splits” in the R,D, Gl,Sb and Sb,H crosses point to the H,Pr region. (B) The recombinant “splits” in H,Pr are used to calculate the approximate location of <em>psg24</em>. The formula provides the relative distance of the mutation from the left marker. This distance is indicated by the frequency of loss of the left marker among the viable F2 progeny “splits.” In this case, the estimated genetic map position for <em>psg24</em> is approximately 80 cM. This genetic location is then used to estimate a cytological location with positional information of known genes (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008615/-/DC1/FileS1.pdf">File S1</a>), estimating the physical location of <em>psg24</em> to around 95A. (C) We used complementation tests with deficiencies near 95A to identify the actual location of <em>psg24</em>. The mutation was crossed to 10 deficiencies from the DK3 collection spanning the region from 94A to 96C, and it failed to complement Df(3R)BSC619 in 94E, which is about 1 cM or approximately one deficiency away from the initial estimated physical map position. The arrow reflects the approximate reliability of the recombination analysis, where the base of the arrow (dot) represents the estimated genetic map position and the arrowhead represents the actual physical location.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-4" class="subsection"><h3>Converting genetic map positions into physical map positions</h3><p id="p-18">Conversions between genetic and physical map positions, given that they do not have a linear relationship along the chromosome, are performed individually by comparing the positional information of nearby genes. The available genome sequence of <em>Drosophila melanogaster</em> is not cross-referenced with genetic map positions. In fact, only a limited percentage of genes have experimentally derived genetic map positions. An extensive list of genes with genetic map positions and their corresponding physical positions (described by chromosomal cytology) is available in the cytogenetic map generated by Michael Ashburner for Lindsley &amp; Zimm’s now classic “Redbook” (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Lindsley and Zimm 1992</a>). This information is also included, when available, on FlyBase Gene Reports (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Marygold <em>et al.</em> 2013</a>). To facilitate the task of conversion, we added 42 reference loci with experimentally derived genetic map positions to a physical map of the third chromosome that includes deficiencies available in BDSC (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008615/-/DC1/FileS1.pdf">File S1</a>). Once a genetic map position is obtained by recombination analysis, we use <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008615/-/DC1/FileS1.pdf">File S1</a> to estimate the corresponding physical location. For this study, the deficiencies near this initial estimated physical location were then tested for complementation to validate the recombination mapping analysis.</p></div></div><div class="section results-discussion" id="sec-5"><h2 class="">Results and Discussion</h2><p id="p-19">To test whether recombination analysis with pairs of dominant visible markers provides a fast and efficient method for obtaining genetic map positions, we used this strategy to map newly generated EMS-induced lethal mutations on the third chromosome. The use of dominant markers allows scoring recombinant progeny in the F2 generation, a generation earlier than mapping schemes using recessive markers, eliminating the need to set up crosses with each recombinant progeny. In theory, the use of two broadly spaced markers increases the frequency of recombination events which, in turn, potentially generates positional information with a smaller number of progeny. Although this strategy may reduce the resolution and accuracy of recombination analysis, it allows for rapidly estimating the relative position of a mutation between the two markers. The method is best illustrated by following one class of viable F2 recombinant progeny from a mapping cross: those that have lost both dominant markers present on the mapping chromosome (follow the “+,+” gametes in <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1, B and C</a>). When crosses are performed on a small scale (<em>i.e.</em>, in vials), the absence or presence of very few unmarked recombinant progeny (carrying the “+,+” gamete) indicates that the mutation is located inside that pair of markers. Given that three to four pairs of dominant markers can span the entire chromosome, only one of these pairs will show that the mutation is inside its boundaries. In this latter cross, the ratio of recombinant progeny that have lost only one dominant marker (see “splits” in <a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1B</a>) provides the relative position of the mutation within these markers. In the remaining crosses, the ratio of these “splits” indicates direction, whether the mutation is to the left or the right of its boundaries (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1C</a>).</p><p id="p-20">We used four pairs of dominant markers to span the third chromosome: R,D on the left arm, Gl,Sb across the centromere, and Sb,H and H,Pr on the right arm. These pairs of markers are approximately 20 cM apart, except for R and D, which are approximately 40 cM apart. Heterozygous females undergoing recombination—between a pair of markers and the mutation of interest—were backcrossed to males carrying the mutation and a TM6B,Dr balancer (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1A</a>). The progeny carrying recombinant female gametes that have lost the mutation (viable nonbalancer progeny) were then scored for the loss of one or both dominant markers used. In practice, scoring is very fast: Dr progeny are easily identified and moved aside, and then the progeny that have lost one or both of the two markers are scored (scoring progeny with both markers is not informative; see <a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1</a>). <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a> illustrates the mapping results for a lethal mutation generated in our EMS screen, <em>psg24</em>. Only the H,Pr cross lacks unmarked recombinant progeny (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2A</a>), indicating that the mutation is located inside its boundaries. Consistently, the ratio of “splits” in the other crosses points to the mutation being in H,Pr (<em>e.g.</em>, more +,D than R,+ progeny indicates the mutation is to the right of D; <a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2A</a>). Then, the ratio of “splits” between H and Pr was used to calculate the relative position of the mutation within the pair of markers (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2B</a>). Thus, we estimate that the mutation lies halfway between H and Pr (a 13:14 ratio) at approximately 80 cM (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2B</a>).</p><p id="p-21">To demonstrate that the estimated genetic map position could be used for gene identification, we used deletion mapping with the BDSC deficiency kit for the third chromosome (DK3). Given that these deficiencies are not associated with genetic locations, we converted the estimated genetic map position of our mutation (in cM) to a physical map position (in cytology) (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008615/-/DC1/FileS1.pdf">File S1</a>). This conversion estimated the <em>psg24</em> mutation to roughly 95A (<a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2B</a>). We then tested 10 nearby deficiencies from the deficiency kit for complementation—the mutation failed to complement one of these, Df(3R)BSC619 in cytological band 94E (<a id="xref-fig-2-8" class="xref-fig" href="#F2">Figure 2C</a>). Thus, the estimated genetic location for <em>psg24</em> was approximately 1 cM away from the deficiency that uncovered it (genetic map position of the deficiency was estimated using available genetic map positions of genes contained within it). Another way to measure the distance between the estimated genetic map position and the actual physical map position is to use the number of DK3 deficiencies that separate them; in this case, the estimated and actual positions are one deficiency apart. The latter is a more practical measure of the work required to locate a mutation after recombination mapping. Importantly, these results demonstrate that, even when scoring a relatively small number of F2 recombinant progeny, recombination mapping with pairs of dominant markers works.</p><p id="p-22">To determine whether this method could be used to facilitate mapping after large-scale forward genetic screens, we used pairs of dominant markers to simultaneously map a collection of mutations identified in an EMS-mutagenesis screen on the third chromosome. The mapping process described above for <em>psg24</em> was simultaneously applied to 38 complementation groups of recessive lethal mutations that had a PSG phenotype (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008615/-/DC1/TableS2.pdf">Table S2</a>). As a result, we successfully mapped 30 of the 38 complementation groups to deficiencies within the DK3 (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). All mutations within the 30 complementation groups failed to complement at least one other deficiency within their respective region, further validating the location of the lethal mutations. Moreover, subsequent analysis showed that the PSG phenotypes also map to the same locations, validating the ability to map the loci of interest. In two separate cases, complementing mutations mapped to the same gene, reducing the number of mapped loci to 28 (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). Among the mutations we failed to map, one maps to a region of the third chromosome without available deficiencies and was not pursued. Given that approximately 98% of the third chromosome is covered by deficiencies (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Cook <em>et al.</em> 2012</a>), this is expected to be a rare occurrence. The remaining seven unmapped mutations had inconsistent mapping results that were likely caused by multiple independent and/or interacting lesions on the same chromosome—a common occurrence with alleles generated in chemical mutagenesis screens. Thus, our results demonstrate that we were able to map the great majority (79%) of the mutations identified in the EMS screen; these mapped mutations likely include most, if not all, of the loci with a tractable PSG phenotype. Taken together, these results demonstrate that using pairs of dominant markers is effective and can be easily scaled to map a large collection of chemically‐induced lethal mutations.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/463634/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/463634/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Mapping the collection of PSG mutations using pairs of dominant markers</span><div class="sb-div caption-clear"></div></div></div><p id="p-26">To evaluate the overall reliability of using pairs of dominant markers for recombination analysis, we determined the distance between the estimated and actual map positions for all mapped mutations. Each mutation is represented by an arrow where the base (marked by a dot) indicates the estimated genetic map positions, and the arrowhead indicates the actual physical map positions (<a id="xref-fig-2-9" class="xref-fig" href="#F2">Figure 2C</a> and <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). Thus, the length of the arrow reflects the relative reliability of the recombination data generated by this method. The “arrows” for all 28 mapped complementation groups are displayed within a graph that illustrates the average relationship between genetic and physical map positions on the third chromosome (gray line, modified from <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Ashburner 1989</a>). The most striking feature of this relationship is the nonlinearity along the chromosome caused by the inhibitory effects of the centromere on recombination rates. Not surprisingly, the reliability of our recombination mapping parallels this relationship and reflects the inefficiency of recombination-based methods to map mutations located between Gl and Sb due to the centromere effect (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>). This region represents 43% of the third chromosome based on nucleotides and contains the same fraction of the mutations generated in our screen (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>). We found that mutations located between <em>Gl</em> and <em>Sb</em> are best mapped by crossing them to all DK3 deficiencies that cover this region. On the other hand, mutations located within the remaining ~60% of the chromosome were easily and reliably mapped to deficiencies. This is illustrated by the relatively short “arrows” separating the estimated genetic and actual physical map positions (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3</a>). Moreover, by measuring this distance in terms of DK3 deficiencies, mutations located on the left arm are mapped by crossing them to, on average, 10 DK3 deficiencies (five on either side of the estimated genetic location); mutations located on the right arm require, on average, 8 DK3 deficiencies (four on either side of the estimated genetic location) (calculated from data in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008615/-/DC1/TableS2.pdf">Table S2</a>). The difference in resolution between the arms likely reflects the larger distance between the dominant markers on the left arm.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/12/2313/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Reliability of using pairs of dominant markers for mapping a collection of EMS-induced lethal mutations. The graph illustrates the relationship between genetic and cytological map positions across the third chromosome (thick gray line). The estimated genetic locations of the 28 mapped mutations are shown by the dots at the base of arrows on the gray line, with arrowheads pointing to actual cytological location as determined by complementation tests with deficiencies. Thus, the length of the resulting arrows reflects the reliability of the mapping process. Not surprisingly, the regions of greater reliability (reflected by shorter arrows) correlate with the regions that have a linear relationship between the genetic and physical map positions (i.e., left of D and right of Sb). The number of mutations that map to each region is proportional to the physical size of these regions, indicating the ability to map mutations was not affected by their chromosomal location." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-763482392" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Reliability of using pairs of dominant markers for mapping a collection of EMS-induced lethal mutations. The graph illustrates the relationship between genetic and cytological map positions across the third chromosome (thick gray line). The estimated genetic locations of the 28 mapped mutations are shown by the dots at the base of arrows on the gray line, with arrowheads pointing to actual cytological location as determined by complementation tests with deficiencies. Thus, the length of the resulting arrows reflects the reliability of the mapping process. Not surprisingly, the regions of greater reliability (reflected by shorter arrows) correlate with the regions that have a linear relationship between the genetic and physical map positions (&lt;em&gt;i.e.&lt;/em&gt;, left of &lt;em&gt;D&lt;/em&gt; and right of &lt;em&gt;Sb&lt;/em&gt;). The number of mutations that map to each region is proportional to the physical size of these regions, indicating the ability to map mutations was not affected by their chromosomal location.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/12/2313/F3.medium.gif" width="440" height="348"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/3/12/2313/F3.medium.gif" width="440" height="348"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/12/2313/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/12/2313/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463809" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-27" class="first-child">Reliability of using pairs of dominant markers for mapping a collection of EMS-induced lethal mutations. The graph illustrates the relationship between genetic and cytological map positions across the third chromosome (thick gray line). The estimated genetic locations of the 28 mapped mutations are shown by the dots at the base of arrows on the gray line, with arrowheads pointing to actual cytological location as determined by complementation tests with deficiencies. Thus, the length of the resulting arrows reflects the reliability of the mapping process. Not surprisingly, the regions of greater reliability (reflected by shorter arrows) correlate with the regions that have a linear relationship between the genetic and physical map positions (<em>i.e.</em>, left of <em>D</em> and right of <em>Sb</em>). The number of mutations that map to each region is proportional to the physical size of these regions, indicating the ability to map mutations was not affected by their chromosomal location.</p><div class="sb-div caption-clear"></div></div></div><p id="p-28">Accurate measurements of genetic map positions is a persnickety endeavor, requiring rigorous control of diverse parameters like temperature and crowding of cultures, age of females, and genetic backgrounds. Moreover, a genetic map position reflects the total frequency of exchange between a locus of interest and a reference gene less than 10 cM away (to minimize interference due to double crossover events) and thus requires scoring all viable progeny. In contrast, our results demonstrate that, by using pairs of dominant markers, we can simplify the process of recombination analysis for the purpose of mapping. Although we attempted to maintain uniform culture conditions for our crosses, we were not fastidious about it. Moreover, we only scored a small subset of the viable recombinant progeny and used markers that were 20 and even 40 cM apart. Despite these simplifications, the empirical evidence presented here indicates that the process is still effective and reliable. In fact, our mapping data indicates that we can simplify the process even further: we can skip the Gl,Sb pair for mapping. Instead of using the Gl,Sb pair, the direction of “splits” in the other three pairs reliably locate mutations that map between Gl and Sb (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008615/-/DC1/TableS3.pdf">Table S3</a>). Thus, recombination analysis can be achieved with three pairs of dominant markers in six crosses total. The overall effectiveness of recombination analysis using pairs of markers is likely twofold. On the one hand, currently available reagents, like the collections of molecularly-defined deficiencies that span the genome (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Parks <em>et al.</em> 2004</a>; <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Ryder <em>et al.</em> 2007</a>; <a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">Cook <em>et al.</em> 2012</a>), help reduce the resolution required to translate a genetic map position into a physical location. On the other hand, by using pairs of markers, we are estimating a relative position between the two markers (<em>e.g.</em>, 30% of the distance between <em>H</em> and <em>Pr</em> from <em>H</em>), not measuring a true genetic map position (<em>e.g.</em>, 6.2 cM to the right of <em>H</em>), likely minimizing the effects of environmental and genetic conditions on the ability to estimate genetic map positions.</p><p id="p-29">Taken together, our results demonstrate that using pairs of dominant markers provides a rapid and effective method to identify the genetic map positions of chemically-induced mutations. Requiring only two generations and six crosses in vials, this method is faster and less labor-intensive than other available methods for recombination analysis. We have shown that this method is not only effective in mapping one mutation at a time, it is also easily scalable to simultaneously tackle a large collection of mutations identified from a forward genetic screen. The resulting genetic map positions are reliable and offer high accuracy to guide complementation tests with deficiencies. This method can also be used iteratively to map even larger collections of mutations, effectively replacing the need for complementation tests as the first step after a mutagenesis screen. For example, we successfully mapped the majority of loci in an independent collection of 134 EMS-induced lethal mutations, by first using the R,D pair and then the H,Pr pair. In addition, although we describe this method for mapping lethal mutations, the strategy can be easily adapted to map visible phenotypes. In fact, the concept of using broadly-spaced pairs of markers can be used with molecular markers like SNPs or with any visible markers like GFP, white, or yellow on <em>P</em>-elements. Finally, in addition to its practical value in research, we have found this method to be a good didactic tool in our laboratory to introduce new graduate and undergraduate students to genetics—the excitement of mapping your first mutation is undeniable and provides a reliable way to hook, especially undergraduates, to the joys of science.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-30">We are greatly indebted to the dedicated staff at the BDSC and the FlyBase Project for generating, maintaining, distributing, and cataloging all the stocks and associated genetic and molecular information that make mapping mutations so much easier today. We also thank Kate O’Connor-Giles and Bruce Reed for critical comments on the manuscript. This work was supported by the University of Wisconsin School of Pharmacy and by an NIGMS/NIH grant (R01 GM095944) to A.B.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008615/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008615/-/DC1</a></p></li><li class="fn" id="fn-12"><p id="p-31"><em>Communicating editor: S. R. Hawley</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2013-09-24"><span class="received-label">Received </span>September 24, 2013.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2013-10-18"><span class="accepted-label">Accepted </span>October 18, 2013.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2013 Sapiro <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-9">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-book" id="cit-3.12.2313.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ashburner</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-source">Drosophila: A Laboratory Manual</span>. <span class="cit-publ-name">Cold Spring Harbor Laboratory Press</span>, <span class="cit-publ-loc">Cold Spring Harbor, NY</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-3.12.2313.2" data-doi="10.1534/genetics.109.101998"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Blumenstiel</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Noll</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Griffiths</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perera</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walton</span>  <span class="cit-name-given-names">K. N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Identification of EMS-induced mutations in <em>Drosophila melanogaster</em> by whole-genome sequencing.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">182</span>: <span class="cit-fpage">25</span>–<span class="cit-lpage">32</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.101998%26rft_id%253Dinfo%253Apmid%252F19307605%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMTgyLzEvMjUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8zLzEyLzIzMTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-book" id="cit-3.12.2313.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bridges</span>  <span class="cit-name-given-names">C. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morgan</span>  <span class="cit-name-given-names">T. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1923</span> <span class="cit-source">The Third-Chromosome Group of Mutant Characters of Drosophila Melanogaster</span>. <span class="cit-publ-name">Carnegie Institution of Washington</span>, <span class="cit-publ-loc">Baltimore</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-3.12.2313.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ahlford</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schnorrer</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kalchhauser</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fellner</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">High-resolution, high-throughput SNP mapping in <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">323</span>–<span class="cit-lpage">329</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DChen%26rft.auinit1%253DD.%26rft.volume%253D5%26rft.issue%253D4%26rft.spage%253D323%26rft.epage%253D329%26rft.atitle%253DHigh-resolution%252C%2Bhigh-throughput%2BSNP%2Bmapping%2Bin%2BDrosophila%2Bmelanogaster.%26rft_id%253Dinfo%253Apmid%252F18327265%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=18327265&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F12%2F2313.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-3.12.2313.5" data-doi="10.1186/gb-2012-13-3-r21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cook</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Christensen</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deal</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coburn</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deal</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The generation of chromosomal deletions to provide extensive coverage and subdivision of the Drosophila melanogaster genome.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">R21</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DCook%26rft.auinit1%253DK.%26rft.volume%253D13%26rft.issue%253D3%26rft.spage%253DR21%26rft.epage%253DR21%26rft.atitle%253DThe%2Bgeneration%2Bof%2Bchromosomal%2Bdeletions%2Bto%2Bprovide%2Bextensive%2Bcoverage%2Band%2Bsubdivision%2Bof%2Bthe%2BDrosophila%2Bmelanogaster%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2012-13-3-r21%26rft_id%253Dinfo%253Apmid%252F22445104%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2012-13-3-r21&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22445104&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F12%2F2313.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-3.12.2313.6" data-doi="10.1371/journal.pgen.1003085"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ihry</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sapiro</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bashirullah</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Translational control by the DEAD Box RNA helicase <em>belle</em> regulates ecdysone-triggered transcriptional cascades.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1003085</span>. <span class="cit-comment">doi:10.1371/journal.pgen.1003085.</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DIhry%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D8%26rft.issue%253D11%26rft.spage%253De1003085%26rft.epage%253De1003085%26rft.atitle%253DTranslational%2BControl%2Bby%2Bthe%2BDEAD%2BBox%2BRNA%2BHelicase%2Bbelle%2BRegulates%2BEcdysone-Triggered%2BTranscriptional%2BCascades.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003085%26rft_id%253Dinfo%253Apmid%252F23209440%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003085&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23209440&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F12%2F2313.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-book" id="cit-3.12.2313.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lindsley</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zimm</span>  <span class="cit-name-given-names">G. G.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-source">The Genome of Drosophila melanogaster</span>. <span class="cit-publ-name">Academic Press</span>, <span class="cit-publ-loc">San Diego</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-other" id="cit-3.12.2313.8"><div class="cit-metadata unstructured">Martin S. G., Dobi K. C., St Johnston D., 2001 A rapid method to map mutations in Drosophila. Genome Biol <strong>2:</strong> research0036.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-3.12.2313.9" data-doi="10.1093/nar/gks1024"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marygold</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leyland</span>  <span class="cit-name-given-names">P. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seal</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goodman</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thurmond</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">FlyBase: improvements to the bibliography.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">D751</span>–<span class="cit-lpage">D757</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgks1024%26rft_id%253Dinfo%253Apmid%252F23125371%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiI0MS9EMS9ENzUxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMy8xMi8yMzEzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-3.12.2313.10" data-doi="10.1126/science.1223899"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Osterloh</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rooney</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fox</span>  <span class="cit-name-given-names">A. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Adalbert</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">dSarm/Sarm1 is required for activation of an injury-induced axon death pathway.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">337</span>: <span class="cit-fpage">481</span>–<span class="cit-lpage">484</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1223899%26rft_id%253Dinfo%253Apmid%252F22678360%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMzcvNjA5My80ODEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8zLzEyLzIzMTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-3.12.2313.11" data-doi="10.1038/ng1312"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Parks</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cook</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Belvin</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dompe</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fawcett</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Systematic generation of high-resolution deletion coverage of the <em>Drosophila melanogaster</em> genome.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">288</span>–<span class="cit-lpage">292</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DParks%26rft.auinit1%253DA.%2BL.%26rft.volume%253D36%26rft.issue%253D3%26rft.spage%253D288%26rft.epage%253D292%26rft.atitle%253DSystematic%2Bgeneration%2Bof%2Bhigh-resolution%2Bdeletion%2Bcoverage%2Bof%2Bthe%2BDrosophila%2Bmelanogaster%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1312%26rft_id%253Dinfo%253Apmid%252F14981519%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1312&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14981519&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F12%2F2313.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000189250400019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-3.12.2313.12" data-doi="10.1534/genetics.107.076216"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ryder</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ashburner</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bautista-Llacer</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Drummond</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Webster</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The DrosDel deletion collection: a Drosophila genomewide chromosomal deficiency resource.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">177</span>: <span class="cit-fpage">615</span>–<span class="cit-lpage">629</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.076216%26rft_id%253Dinfo%253Apmid%252F17720900%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTc3LzEvNjE1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMy8xMi8yMzEzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-3.12.2313.13" data-doi="10.1002/jez.1400140104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sturtevant</span>  <span class="cit-name-given-names">A. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1913</span> <span class="cit-article-title">The linear arrangement of six sex-linked factors in Drosophila, as shown by their mode of association.</span> <abbr class="cit-jnl-abbrev">J. Exp. Zool.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">43</span>–<span class="cit-lpage">59</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BExp.%2BZool.%26rft.volume%253D14%26rft.spage%253D43%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fjez.1400140104%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/jez.1400140104&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000187376000003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-3.12.2313.14" data-doi="10.1534/genetics.108.092478"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Evans</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andrews</span>  <span class="cit-name-given-names">H. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beckstead</span>  <span class="cit-name-given-names">R. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thummel</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A genetic screen identifies new regulators of steroid-triggered programmed cell death in Drosophila.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">180</span>: <span class="cit-fpage">269</span>–<span class="cit-lpage">281</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.092478%26rft_id%253Dinfo%253Apmid%252F18757938%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgwLzEvMjY5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMy8xMi8yMzEzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-3.12.2313.15" data-doi="10.1073/pnas.1832753100"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhai</span>  <span class="cit-name-given-names">R. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hiesinger</span>  <span class="cit-name-given-names">P. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koh</span>  <span class="cit-name-given-names">T.-W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Verstreken</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schulze</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Mapping Drosophila mutations with molecularly defined P element insertions.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">100</span>: <span class="cit-fpage">10860</span>–<span class="cit-lpage">10865</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1832753100%26rft_id%253Dinfo%253Apmid%252F12960394%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAwLzE5LzEwODYwIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMy8xMi8yMzEzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/3/12/2313.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/12/2305" title="Dynamics of Adaptive Alleles in Divergently Selected Body Weight Lines of Chickens" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/12/2321" title="Saccharomyces cerevisiae Genes Involved in Survival of Heat Shock" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/3/12"><span>Volume 3 Issue 12, December 2013</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/3/12.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/3/12.cover-source.jpg?itok=Gq_aE1-I" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 3 (12)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/12/2313" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-nJLJ8Kq4d-u7CjVK-Ud1o09vAoK9rz8yYBjyCqdi5aY" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/3/12/2313" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/3/12/2313" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/3/12/2313" class="active" data-icon-position="" data-hide-link-title="0">Rapid Recombination Mapping for High-Throughput Genetic Screens in Drosophila</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459592" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-_jveDOnw0rDPfgzp1mfQ3pFQKjhgc7-lRsU27C0ulDg" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/12/2313.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/3/12/2313" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-DsIhATLt8tpfCBDYFkhsWxMm_5S24L2LYuIX7PaA3O0" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/3/12/2313" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/12/2313.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459592" id="node459592--2" data-pisa="ggg;3/12/2313" data-pisa-master="ggg;g3.113.008615" data-apath="/ggg/3/12/2313.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Rapid Recombination Mapping for High-Throughput Genetic Screens in <em>Drosophila</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Anne L. Sapiro</span>, <span class="highwire-citation-author" data-delta="1">Robert J. Ihry</span>, <span class="highwire-citation-author" data-delta="2">Derek L. Buhr</span>, <span class="highwire-citation-author" data-delta="3">Kevin M. Konieczko</span>, <span class="highwire-citation-author" data-delta="4">Sarah M. Ives</span>, <span class="highwire-citation-author" data-delta="5">Anna K. Engstrom</span>, <span class="highwire-citation-author" data-delta="6">Nicholas P. Wleklinski</span>, <span class="highwire-citation-author" data-delta="7">Kristin J. Kopish</span> and <span class="highwire-citation-author" data-delta="8">Arash Bashirullah</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2313-2319;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.008615 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459592--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Anne L. Sapiro </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Division of Pharmaceutical Sciences, University of Wisconsin-Madison, Madison, Wisconsin 53705-2222</div><div class='author-affiliation'><span class='nlm-sup'>†</span>College of Letters and Sciences or College of Agriculture and Life Sciences, University of Wisconsin-Madison, Madison, Wisconsin 53705-2222</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anne%2BL.%2BSapiro%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sapiro%20AL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnne%2BL.%2BSapiro%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Robert J. Ihry </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Division of Pharmaceutical Sciences, University of Wisconsin-Madison, Madison, Wisconsin 53705-2222</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Cellular and Molecular Biology Graduate Program, University of Wisconsin-Madison, Madison, Wisconsin 53705-2222</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BJ.%2BIhry%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ihry%20RJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARobert%2BJ.%2BIhry%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Derek L. Buhr </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Division of Pharmaceutical Sciences, University of Wisconsin-Madison, Madison, Wisconsin 53705-2222</div><div class='author-affiliation'><span class='nlm-sup'>†</span>College of Letters and Sciences or College of Agriculture and Life Sciences, University of Wisconsin-Madison, Madison, Wisconsin 53705-2222</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Derek%2BL.%2BBuhr%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Buhr%20DL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADerek%2BL.%2BBuhr%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Kevin M. Konieczko </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Division of Pharmaceutical Sciences, University of Wisconsin-Madison, Madison, Wisconsin 53705-2222</div><div class='author-affiliation'><span class='nlm-sup'>†</span>College of Letters and Sciences or College of Agriculture and Life Sciences, University of Wisconsin-Madison, Madison, Wisconsin 53705-2222</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kevin%2BM.%2BKonieczko%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Konieczko%20KM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKevin%2BM.%2BKonieczko%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Sarah M. Ives </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Division of Pharmaceutical Sciences, University of Wisconsin-Madison, Madison, Wisconsin 53705-2222</div><div class='author-affiliation'><span class='nlm-sup'>†</span>College of Letters and Sciences or College of Agriculture and Life Sciences, University of Wisconsin-Madison, Madison, Wisconsin 53705-2222</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sarah%2BM.%2BIves%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ives%20SM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASarah%2BM.%2BIves%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Anna K. Engstrom </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Division of Pharmaceutical Sciences, University of Wisconsin-Madison, Madison, Wisconsin 53705-2222</div><div class='author-affiliation'><span class='nlm-sup'>†</span>College of Letters and Sciences or College of Agriculture and Life Sciences, University of Wisconsin-Madison, Madison, Wisconsin 53705-2222</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anna%2BK.%2BEngstrom%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Engstrom%20AK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnna%2BK.%2BEngstrom%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Nicholas P. Wleklinski </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Division of Pharmaceutical Sciences, University of Wisconsin-Madison, Madison, Wisconsin 53705-2222</div><div class='author-affiliation'><span class='nlm-sup'>†</span>College of Letters and Sciences or College of Agriculture and Life Sciences, University of Wisconsin-Madison, Madison, Wisconsin 53705-2222</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nicholas%2BP.%2BWleklinski%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wleklinski%20NP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANicholas%2BP.%2BWleklinski%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Kristin J. Kopish </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Division of Pharmaceutical Sciences, University of Wisconsin-Madison, Madison, Wisconsin 53705-2222</div><div class='author-affiliation'><span class='nlm-sup'>†</span>College of Letters and Sciences or College of Agriculture and Life Sciences, University of Wisconsin-Madison, Madison, Wisconsin 53705-2222</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kristin%2BJ.%2BKopish%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kopish%20KJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKristin%2BJ.%2BKopish%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Arash Bashirullah </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Division of Pharmaceutical Sciences, University of Wisconsin-Madison, Madison, Wisconsin 53705-2222</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Cellular and Molecular Biology Graduate Program, University of Wisconsin-Madison, Madison, Wisconsin 53705-2222</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Arash%2BBashirullah%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bashirullah%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AArash%2BBashirullah%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:bashirullah@wisc.edu" class="" data-icon-position="" data-hide-link-title="0">bashirullah@wisc.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/3/12/2313" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/3/12/2313&amp;title=Rapid%20Recombination%20Mapping%20for%20High-Throughput%20Genetic%20Screens%20in%20Drosophila" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/3/12/2313&amp;title=Rapid%20Recombination%20Mapping%20for%20High-Throughput%20Genetic%20Screens%20in%20Drosophila" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/3/12/2313&amp;title=Rapid%20Recombination%20Mapping%20for%20High-Throughput%20Genetic%20Screens%20in%20Drosophila" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/3/12/2313&amp;text=Rapid%20Recombination%20Mapping%20for%20High-Throughput%20Genetic%20Screens%20in%20Drosophila" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/3/12/2313&amp;title=Rapid%20Recombination%20Mapping%20for%20High-Throughput%20Genetic%20Screens%20in%20Drosophila" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/3/12/2313&amp;t=Rapid%20Recombination%20Mapping%20for%20High-Throughput%20Genetic%20Screens%20in%20Drosophila" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/3/12/2313&amp;title=Rapid%20Recombination%20Mapping%20for%20High-Throughput%20Genetic%20Screens%20in%20Drosophila" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/3/12/2313&amp;title=Rapid%20Recombination%20Mapping%20for%20High-Throughput%20Genetic%20Screens%20in%20Drosophila" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459592/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459592" id="node459592--3" data-pisa="ggg;3/12/2313" data-pisa-master="ggg;g3.113.008615" data-apath="/ggg/3/12/2313.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Rapid Recombination Mapping for High-Throughput Genetic Screens in <em>Drosophila</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Anne L. Sapiro</span>, <span class="highwire-citation-author" data-delta="1">Robert J. Ihry</span>, <span class="highwire-citation-author" data-delta="2">Derek L. Buhr</span>, <span class="highwire-citation-author" data-delta="3">Kevin M. Konieczko</span>, <span class="highwire-citation-author" data-delta="4">Sarah M. Ives</span>, <span class="highwire-citation-author" data-delta="5">Anna K. Engstrom</span>, <span class="highwire-citation-author" data-delta="6">Nicholas P. Wleklinski</span>, <span class="highwire-citation-author" data-delta="7">Kristin J. Kopish</span> and <span class="highwire-citation-author" data-delta="8">Arash Bashirullah</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2313-2319;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.008615 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459592--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Anne L. Sapiro </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Division of Pharmaceutical Sciences, University of Wisconsin-Madison, Madison, Wisconsin 53705-2222</div><div class='author-affiliation'><span class='nlm-sup'>†</span>College of Letters and Sciences or College of Agriculture and Life Sciences, University of Wisconsin-Madison, Madison, Wisconsin 53705-2222</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anne%2BL.%2BSapiro%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sapiro%20AL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnne%2BL.%2BSapiro%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Robert J. Ihry </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Division of Pharmaceutical Sciences, University of Wisconsin-Madison, Madison, Wisconsin 53705-2222</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Cellular and Molecular Biology Graduate Program, University of Wisconsin-Madison, Madison, Wisconsin 53705-2222</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BJ.%2BIhry%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ihry%20RJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARobert%2BJ.%2BIhry%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Derek L. Buhr </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Division of Pharmaceutical Sciences, University of Wisconsin-Madison, Madison, Wisconsin 53705-2222</div><div class='author-affiliation'><span class='nlm-sup'>†</span>College of Letters and Sciences or College of Agriculture and Life Sciences, University of Wisconsin-Madison, Madison, Wisconsin 53705-2222</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Derek%2BL.%2BBuhr%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Buhr%20DL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADerek%2BL.%2BBuhr%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Kevin M. Konieczko </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Division of Pharmaceutical Sciences, University of Wisconsin-Madison, Madison, Wisconsin 53705-2222</div><div class='author-affiliation'><span class='nlm-sup'>†</span>College of Letters and Sciences or College of Agriculture and Life Sciences, University of Wisconsin-Madison, Madison, Wisconsin 53705-2222</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kevin%2BM.%2BKonieczko%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Konieczko%20KM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKevin%2BM.%2BKonieczko%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Sarah M. Ives </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Division of Pharmaceutical Sciences, University of Wisconsin-Madison, Madison, Wisconsin 53705-2222</div><div class='author-affiliation'><span class='nlm-sup'>†</span>College of Letters and Sciences or College of Agriculture and Life Sciences, University of Wisconsin-Madison, Madison, Wisconsin 53705-2222</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sarah%2BM.%2BIves%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ives%20SM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASarah%2BM.%2BIves%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Anna K. Engstrom </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Division of Pharmaceutical Sciences, University of Wisconsin-Madison, Madison, Wisconsin 53705-2222</div><div class='author-affiliation'><span class='nlm-sup'>†</span>College of Letters and Sciences or College of Agriculture and Life Sciences, University of Wisconsin-Madison, Madison, Wisconsin 53705-2222</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anna%2BK.%2BEngstrom%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Engstrom%20AK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnna%2BK.%2BEngstrom%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Nicholas P. Wleklinski </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Division of Pharmaceutical Sciences, University of Wisconsin-Madison, Madison, Wisconsin 53705-2222</div><div class='author-affiliation'><span class='nlm-sup'>†</span>College of Letters and Sciences or College of Agriculture and Life Sciences, University of Wisconsin-Madison, Madison, Wisconsin 53705-2222</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nicholas%2BP.%2BWleklinski%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wleklinski%20NP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANicholas%2BP.%2BWleklinski%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Kristin J. Kopish </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Division of Pharmaceutical Sciences, University of Wisconsin-Madison, Madison, Wisconsin 53705-2222</div><div class='author-affiliation'><span class='nlm-sup'>†</span>College of Letters and Sciences or College of Agriculture and Life Sciences, University of Wisconsin-Madison, Madison, Wisconsin 53705-2222</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kristin%2BJ.%2BKopish%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kopish%20KJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKristin%2BJ.%2BKopish%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Arash Bashirullah </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Division of Pharmaceutical Sciences, University of Wisconsin-Madison, Madison, Wisconsin 53705-2222</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Cellular and Molecular Biology Graduate Program, University of Wisconsin-Madison, Madison, Wisconsin 53705-2222</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Arash%2BBashirullah%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bashirullah%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AArash%2BBashirullah%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:bashirullah@wisc.edu" class="" data-icon-position="" data-hide-link-title="0">bashirullah@wisc.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2313&amp;rft.epage=2319&amp;rft.atitle=Rapid%20Recombination%20Mapping%20for%20High-Throughput%20Genetic%20Screens%20in%20Drosophila&amp;rft.volume=3&amp;rft.issue=12&amp;rft.date=2013-12-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Sapiro%2C+Anne+L.&amp;rft.au=Ihry%2C+Robert+J.&amp;rft.au=Buhr%2C+Derek+L.&amp;rft.au=Konieczko%2C+Kevin+M.&amp;rft.au=Ives%2C+Sarah+M.&amp;rft.au=Engstrom%2C+Anna+K.&amp;rft.au=Wleklinski%2C+Nicholas+P.&amp;rft.au=Kopish%2C+Kristin+J.&amp;rft.au=Bashirullah%2C+Arash"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459592/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459592/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459592/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459592/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459592/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459592/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459592/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459592/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459592/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459592/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459592/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459592/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/3/12/2313" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-5" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results and Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/3/12/2313.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/3/12/2313.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/3/12/2313.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/3/12/2313.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
