<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/3/12/2163" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/3/12/2163.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/3/12/2163.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/3/12/2163.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/3/12/2163.atom" />
<meta name="HW.pisa" content="ggg;3/12/2163" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Genomic Sequence Diversity and Population Structure of Saccharomyces cerevisiae Assessed by RAD-seq" />
<meta name="DC.Identifier" content="10.1534/g3.113.007492" />
<meta name="DC.Date" content="2013-12-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2013 Cromie et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="The budding yeast Saccharomyces cerevisiae is important for human food production and as a model organism for biological research. The genetic diversity contained in the global population of yeast strains represents a valuable resource for a number of fields, including genetics, bioengineering, and studies of evolution and population structure. Here, we apply a multiplexed, reduced genome sequencing strategy (restriction site−associated sequencing or RAD-seq) to genotype a large collection of S. cerevisiae strains isolated from a wide range of geographical locations and environmental niches. The method permits the sequencing of the same 1% of all genomes, producing a multiple sequence alignment of 116,880 bases across 262 strains. We find diversity among these strains is principally organized by geography, with European, North American, Asian, and African/S. E. Asian populations defining the major axes of genetic variation. At a finer scale, small groups of strains from cacao, olives, and sake are defined by unique variants not present in other strains. One population, containing strains from a variety of fermentations, exhibits high levels of heterozygosity and a mixture of alleles from European and Asian populations, indicating an admixed origin for this group. We propose a model of geographic differentiation followed by human-associated admixture, primarily between European and Asian populations and more recently between European and North American populations. The large collection of genotyped yeast strains characterized here will provide a useful resource for the broad community of yeast researchers." />
<meta name="DC.Contributor" content="Gareth A. Cromie" />
<meta name="DC.Contributor" content="Katie E. Hyma" />
<meta name="DC.Contributor" content="Catherine L. Ludlow" />
<meta name="DC.Contributor" content="Cecilia Garmendia-Torres" />
<meta name="DC.Contributor" content="Teresa L. Gilbert" />
<meta name="DC.Contributor" content="Patrick May" />
<meta name="DC.Contributor" content="Angela A. Huang" />
<meta name="DC.Contributor" content="Aimée M. Dudley" />
<meta name="DC.Contributor" content="Justin C. Fay" />
<meta name="article:published_time" content="2013-12-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Genomic Sequence Diversity and Population Structure of Saccharomyces cerevisiae Assessed by RAD-seq" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The budding yeast &lt;i&gt;Saccharomyces cerevisiae&lt;/i&gt; is important for human food production and as a model organism for biological research. The genetic diversity contained in the global population of yeast strains represents a valuable resource for a number of fields, including genetics, bioengineering, and studies of evolution and population structure. Here, we apply a multiplexed, reduced genome sequencing strategy (restriction site−associated sequencing or RAD-seq) to genotype a large collection of &lt;i&gt;S. cerevisiae&lt;/i&gt; strains isolated from a wide range of geographical locations and environmental niches. The method permits the sequencing of the same 1% of all genomes, producing a multiple sequence alignment of 116,880 bases across 262 strains. We find diversity among these strains is principally organized by geography, with European, North American, Asian, and African/S. E. Asian populations defining the major axes of genetic variation. At a finer scale, small groups of strains from cacao, olives, and sake are defined by unique variants not present in other strains. One population, containing strains from a variety of fermentations, exhibits high levels of heterozygosity and a mixture of alleles from European and Asian populations, indicating an admixed origin for this group. We propose a model of geographic differentiation followed by human-associated admixture, primarily between European and Asian populations and more recently between European and North American populations. The large collection of genotyped yeast strains characterized here will provide a useful resource for the broad community of yeast researchers.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2013/12/01" />
<meta name="citation_mjid" content="ggg;3/12/2163" />
<meta name="citation_id" content="3/12/2163" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/3/12/2163" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/3/12/2163.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/3/12/2163.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/3/12/2163.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.113.007492" />
<meta name="citation_pmid" content="24122055" />
<meta name="citation_volume" content="3" />
<meta name="citation_issue" content="12" />
<meta name="citation_num_pages" content="9" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2163" />
<meta name="citation_lastpage" content="2171" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Gareth A. Cromie" />
<meta name="citation_author_institution" content="Institute for Systems Biology, Seattle, Washington 98109" />
<meta name="citation_author" content="Katie E. Hyma" />
<meta name="citation_author_institution" content="Bioinformatics Facility (CBSU), Institute for Biotechnology, Cornell University, Ithaca, New York 14850" />
<meta name="citation_author" content="Catherine L. Ludlow" />
<meta name="citation_author_institution" content="Institute for Systems Biology, Seattle, Washington 98109" />
<meta name="citation_author" content="Cecilia Garmendia-Torres" />
<meta name="citation_author_institution" content="Institute for Systems Biology, Seattle, Washington 98109" />
<meta name="citation_author" content="Teresa L. Gilbert" />
<meta name="citation_author" content="Patrick May" />
<meta name="citation_author_institution" content="Institute for Systems Biology, Seattle, Washington 98109" />
<meta name="citation_author_institution" content="Luxembourg Centre for Systems Biomedicine, University of Luxembourg, L-4362 Esch-sur-Alzette, Luxembourg" />
<meta name="citation_author" content="Angela A. Huang" />
<meta name="citation_author_institution" content="University of Pennsylvania, Philadelphia, Pennsylvania 19104" />
<meta name="citation_author" content="Aimée M. Dudley" />
<meta name="citation_author_institution" content="Institute for Systems Biology, Seattle, Washington 98109" />
<meta name="citation_author_email" content="aimee.dudley@gmail.com" />
<meta name="citation_author_email" content="jfay@genetics.wustl.edu" />
<meta name="citation_author" content="Justin C. Fay" />
<meta name="citation_author_institution" content="Department of Genetics, Washington University, St. Louis, Missouri 63108" />
<meta name="citation_author_email" content="aimee.dudley@gmail.com" />
<meta name="citation_author_email" content="jfay@genetics.wustl.edu" />
<meta name="citation_reference" content="citation_journal_title=FEMS Yeast Res.;citation_author=E. Aa;citation_author=J. P. Townsend;citation_author=R. I. Adams;citation_author=K. M. Nielsen;citation_author=J. W. Taylor;citation_title=Population structure and gene evolution in Saccharomyces cerevisiae.;citation_volume=6;citation_year=2006;citation_pmid=16879422;citation_doi=10.1111/j.1567-1364.2006.00059.x" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=N. A. Baird;citation_author=P. D. Etter;citation_author=T. S. Atwood;citation_author=M. C. Currey;citation_author=A. L. Shiver;citation_title=Rapid SNP discovery and genetic mapping using sequenced RAD markers.;citation_pages=e3376-e3376;citation_volume=3;citation_year=2008;citation_issue=10;citation_pmid=18852878;citation_doi=10.1371/journal.pone.0003376" />
<meta name="citation_reference" content="citation_title=Yeasts in Food;citation_author=T. Boekhout;citation_author=V. Robert;citation_year=2003" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. Botstein;citation_author=G. R. Fink;citation_title=Yeast: An Experimental Organism for 21st Century Biology;citation_pages=695-704;citation_volume=189;citation_year=2011;citation_issue=3;citation_pmid=22084421;citation_doi=10.1534/genetics.111.130765" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=S. W. Doniger;citation_author=H. S. Kim;citation_author=D. Swain;citation_author=D. Corcuera;citation_author=M. Williams;citation_title=A catalog of neutral and deleterious polymorphism in yeast.;citation_pages=e1000183-e1000183;citation_volume=4;citation_year=2008;citation_issue=8;citation_pmid=18769710;citation_doi=10.1371/journal.pgen.1000183" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=T. K. Ezov;citation_author=E. Boger-Nadjar;citation_author=Z. Frenkel;citation_author=I. Katsperovski;citation_author=S. Kemeny;citation_title=Molecular-genetic biodiversity in a natural population of the yeast Saccharomyces cerevisiae from “Evolution Canyon”: microsatellite polymorphism, ploidy and controversial sexual status.;citation_volume=174;citation_year=2006;citation_pmid=16980391;citation_doi=10.1534/genetics.106.062745" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=J. C. Fay;citation_author=J. A. Benavides;citation_title=Evidence for domesticated and wild populations of Saccharomyces cerevisiae.;citation_pages=66-71;citation_volume=1;citation_year=2005;citation_issue=1;citation_pmid=16103919;citation_doi=10.1371/journal.pgen.0010066" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=H. Gao;citation_author=S. Williamson;citation_author=C. D. Bustamante;citation_title=A Markov chain Monte Carlo approach for joint inference of population structure and inbreeding rates from multilocus genotype data.;citation_volume=176;citation_year=2007;citation_pmid=17483417;citation_doi=10.1534/genetics.107.072371" />
<meta name="citation_reference" content="citation_journal_title=Environmental Microbiology (Print);citation_journal_abbrev=Environmental Microbiology (Print);citation_author=M. R. Goddard;citation_author=N. Anfang;citation_author=R. Tang;citation_author=R. C. Gardner;citation_author=C. Jun;citation_title=A distinct population of Saccharomyces cerevisiae in New Zealand: evidence for local dispersal by insects and human-aided global dispersal in oak barrels.;citation_pages=63-73;citation_volume=12;citation_year=2010;citation_issue=1;citation_pmid=19691498;citation_doi=10.1111/j.1462-2920.2009.02035.x" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=A. Goffeau;citation_author=B. G. Barrell;citation_author=H. Bussey;citation_author=R. W. Davis;citation_author=B. Dujon;citation_title=Life with 6000 Genes;citation_pages=546-567;citation_volume=274;citation_year=1996;citation_issue=5287;citation_pmid=8849441;citation_doi=10.1126/science.274.5287.546" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=K. E. Hyma;citation_author=J. C. Fay;citation_title=Mixing of vineyard and oak-tree ecotypes of Saccharomyces cerevisiae in North American vineyards.;citation_pages=2917-2930;citation_volume=22;citation_year=2013;citation_doi=10.1111/mec.12155" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=M. Jakobsson;citation_author=N. A. Rosenberg;citation_title=CLUMPP: a cluster matching and permutation program for dealing with label switching and multimodality in analysis of population structure.;citation_volume=23;citation_year=2007;citation_pmid=17485429;citation_doi=10.1093/bioinformatics/btm233" />
<meta name="citation_reference" content="citation_journal_title=International journal of food microbiology;citation_journal_abbrev=Int J Food Microbiol;citation_author=J. L. Legras;citation_author=O. Ruh;citation_author=D. Merdinoglu;citation_author=F. Karst;citation_title=Selection of hypervariable microsatellite loci for the characterization of Saccharomyces cerevisiae strains.;citation_pages=73-83;citation_volume=102;citation_year=2005;citation_issue=1;citation_pmid=15925004;citation_doi=10.1016/j.ijfoodmicro.2004.12.007" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=J. L. Legras;citation_author=D. Merdinoglu;citation_author=J. M. Cornuet;citation_author=F. Karst;citation_title=Bread, beer and wine: Saccharomyces cerevisiae diversity reflects human history.;citation_pages=2091-2102;citation_volume=16;citation_year=2007;citation_issue=10;citation_pmid=17498234;citation_doi=10.1111/j.1365-294X.2007.03266.x" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=R. Durbin;citation_title=Fast and accurate short read alignment with Burrows-Wheeler transform.;citation_volume=25;citation_year=2009;citation_pmid=19451168;citation_doi=10.1093/bioinformatics/btp324" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=B. Handsaker;citation_author=A. Wysoker;citation_author=T. Fennell;citation_author=J. Ruan;citation_title=The Sequence Alignment/Map format and SAMtools.;citation_volume=25;citation_year=2009;citation_pmid=19505943;citation_doi=10.1093/bioinformatics/btp352" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=G. Liti;citation_author=D. M. Carter;citation_author=A. M. Moses;citation_author=J. Warringer;citation_author=L. Parts;citation_title=Population genomics of domestic and wild yeasts.;citation_pages=337-341;citation_volume=458;citation_year=2009;citation_issue=7236;citation_pmid=19212322;citation_doi=10.1038/nature07743" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=K. Lorenz;citation_author=B. A. Cohen;citation_title=Small- and large-effect quantitative trait locus interactions underlie variation in yeast sporulation efficiency.;citation_volume=192;citation_year=2012;citation_pmid=22942125;citation_doi=10.1534/genetics.112.143107" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=M. R. Miller;citation_author=J. P. Dunham;citation_author=A. Amores;citation_author=W. A. Cresko;citation_author=E. A. Johnson;citation_title=Rapid and cost-effective polymorphism identification and genotyping using restriction site associated DNA (RAD) markers.;citation_volume=17;citation_year=2007;citation_pmid=17189378;citation_doi=10.1101/gr.5681207" />
<meta name="citation_reference" content="citation_journal_title=Biochimica et Biophysica Acta. Protein Structure and Molecular Enzymology;citation_journal_abbrev=Biochimica et Biophysica Acta. Protein Structure and Molecular Enzymology;citation_author=C. A. Nieduszynski;citation_author=G. Liti;citation_title=From sequence to function: Insights from natural variation in budding yeasts.;citation_pages=959-966;citation_volume=1810;citation_year=2011;citation_issue=10;citation_pmid=21320572;citation_doi=10.1016/j.bbagen.2011.02.004" />
<meta name="citation_reference" content="citation_journal_title=Journal of Clinical Microbiology;citation_journal_abbrev=J. Clin. Microbiol.;citation_author=F. C. Odds;citation_author=R. Bernaerts;citation_title=CHROMagar Candida, a new differential isolation medium for presumptive identification of clinically important Candida species.;citation_pages=1923-1929;citation_volume=32;citation_year=1994;citation_issue=8;citation_pmid=7989544" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=M. Ramazzotti;citation_author=L. Berna;citation_author=I. Stefanini;citation_author=D. Cavalieri;citation_title=A computational pipeline to discover highly phylogenetically informative genes in sequenced genomes: application to Saccharomyces cerevisiae natural strains.;citation_volume=40;citation_year=2012;citation_pmid=22266652;citation_doi=10.1093/nar/gks005" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=J. Schacherer;citation_author=J. A. Shapiro;citation_author=D. M. Ruderfer;citation_author=L. Kruglyak;citation_title=Comprehensive polymorphism survey elucidates population structure of Saccharomyces cerevisiae.;citation_pages=342-345;citation_volume=458;citation_year=2009;citation_issue=7236;citation_pmid=19212320;citation_doi=10.1038/nature07670" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=D. Schuller;citation_author=F. Cardoso;citation_author=S. Sousa;citation_author=P. Gomes;citation_author=A. C. Gomes;citation_title=Genetic diversity and population structure of Saccharomyces cerevisiae strains isolated from different grape varieties and winemaking regions.;citation_pages=e32507-e32507;citation_volume=7;citation_year=2012;citation_issue=2;citation_pmid=22393409;citation_doi=10.1371/journal.pone.0032507" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=I. Stefanini;citation_author=L. Dapporto;citation_author=J. L. Legras;citation_author=A. Calabretta;citation_author=M. Di Paola;citation_title=Role of social wasps in Saccharomyces cerevisiae ecology and evolution.;citation_volume=109;citation_year=2012;citation_pmid=22847440;citation_doi=10.1073/pnas.1208362109" />
<meta name="citation_reference" content="citation_journal_title=FEMS Yeast Res.;citation_author=S. Swinnen;citation_author=J. M. Thevelein;citation_author=E. Nevoigt;citation_title=Genetic mapping of quantitative phenotypic traits in Saccharomyces cerevisiae.;citation_volume=12;citation_year=2012;citation_pmid=22150948;citation_doi=10.1111/j.1567-1364.2011.00777.x" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=K. Tamura;citation_author=D. Peterson;citation_author=N. Peterson;citation_author=G. Stecher;citation_author=M. Nei;citation_title=MEGA5: molecular evolutionary genetics analysis using maximum likelihood, evolutionary distance, and maximum parsimony methods.;citation_volume=28;citation_year=2011;citation_pmid=21546353;citation_doi=10.1093/molbev/msr121" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=Q. M. Wang;citation_author=W. Q. Liu;citation_author=G. Liti;citation_author=S. A. Wang;citation_author=F. Y. Bai;citation_title=Surprisingly diverged populations of Saccharomyces cerevisiae in natural environments remote from human activity.;citation_pages=5404-5417;citation_volume=21;citation_year=2012;citation_issue=22;citation_pmid=22913817;citation_doi=10.1111/j.1365-294X.2012.05732.x" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=J. Warringer;citation_author=E. Zorgo;citation_author=F. A. Cubillos;citation_author=A. Zia;citation_author=A. Gjuvsland;citation_title=Trait variation in yeast is defined by population history.;citation_pages=e1002111-e1002111;citation_volume=7;citation_year=2011;citation_issue=6;citation_pmid=21698134;citation_doi=10.1371/journal.pgen.1002111" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=J. W. Wenger;citation_author=K. Schwartz;citation_author=G. Sherlock;citation_title=Bulk segregant analysis by high-throughput sequencing reveals a novel xylose utilization gene from Saccharomyces cerevisiae.;citation_pages=e1000942-e1000942;citation_volume=6;citation_year=2010;citation_issue=5;citation_pmid=20485559;citation_doi=10.1371/journal.pgen.1000942" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=E. A. Winzeler;citation_author=C. I. Castillo-Davis;citation_author=G. Oshiro;citation_author=D. Liang;citation_author=D. R. Richards;citation_title=Genetic Diversity in Yeast Assessed With Whole-Genome Oligonucleotide Arrays;citation_pages=79-89;citation_volume=163;citation_year=2003;citation_issue=1;citation_pmid=12586698" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Genomic Sequence Diversity and Population Structure of Saccharomyces cerevisiae Assessed by RAD-seq" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12.cover-source.jpg" />
<meta name="twitter:description" content="The budding yeast Saccharomyces cerevisiae is important for human food production and as a model organism for biological research. The genetic diversity contained in the global population of yeast strains represents a valuable resource for a number of fields, including genetics, bioengineering, and studies of evolution and population structure. Here, we apply a multiplexed, reduced genome sequencing strategy (restriction site−associated sequencing or RAD-seq) to genotype a large collection of S. cerevisiae strains isolated from a wide range of geographical locations and environmental niches. The method permits the sequencing of the same 1% of all genomes, producing a multiple sequence alignment of 116,880 bases across 262 strains. We find diversity among these strains is principally organized by geography, with European, North American, Asian, and African/S. E. Asian populations defining the major axes of genetic variation. At a finer scale, small groups of strains from cacao, olives, and sake are defined by unique variants not present in other strains. One population, containing strains from a variety of fermentations, exhibits high levels of heterozygosity and a mixture of alleles from European and Asian populations, indicating an admixed origin for this group. We propose a model of geographic differentiation followed by human-associated admixture, primarily between European and Asian populations and more recently between European and North American populations. The large collection of genotyped yeast strains characterized here will provide a useful resource for the broad community of yeast researchers." />
<meta name="og-title" property="og:title" content="Genomic Sequence Diversity and Population Structure of Saccharomyces cerevisiae Assessed by RAD-seq" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/3/12/2163" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="The budding yeast Saccharomyces cerevisiae is important for human food production and as a model organism for biological research. The genetic diversity contained in the global population of yeast strains represents a valuable resource for a number of fields, including genetics, bioengineering, and studies of evolution and population structure. Here, we apply a multiplexed, reduced genome sequencing strategy (restriction site−associated sequencing or RAD-seq) to genotype a large collection of S. cerevisiae strains isolated from a wide range of geographical locations and environmental niches. The method permits the sequencing of the same 1% of all genomes, producing a multiple sequence alignment of 116,880 bases across 262 strains. We find diversity among these strains is principally organized by geography, with European, North American, Asian, and African/S. E. Asian populations defining the major axes of genetic variation. At a finer scale, small groups of strains from cacao, olives, and sake are defined by unique variants not present in other strains. One population, containing strains from a variety of fermentations, exhibits high levels of heterozygosity and a mixture of alleles from European and Asian populations, indicating an admixed origin for this group. We propose a model of geographic differentiation followed by human-associated admixture, primarily between European and Asian populations and more recently between European and North American populations. The large collection of genotyped yeast strains characterized here will provide a useful resource for the broad community of yeast researchers." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/12.cover-source.jpg" />
    <title>Genomic Sequence Diversity and Population Structure of Saccharomyces cerevisiae Assessed by RAD-seq | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"1oY3wR-PoXXrPCUu61dP_9j2pViubpsSNqKswEVSv8I","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459604","apath":"\/ggg\/3\/12\/2163.atom","pisa":"ggg;3\/12\/2163","ac":{"\/ggg\/3\/12\/2163.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/3\/12\/2163.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;3\/12\/2163"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;3\/12\/2163"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/3\/12\/2163.abstract":true,"\/content\/3\/12\/2163":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022\\\\\\\/TlIQqjd7muNd10nai6VRvRmsuO9FnT36F9WWYdSrfmfCogVykekrOXzXZAOHfr2+8z5BaBKOCcM\\\\\\\/MrH0Ox5PuIu+95I87neLb6ixq5vNtC\\\\\\\/uXsKOo\\\\\\\/6R1iRChq0y7wncvV+t4tcHFe0LmT7b3+s8F6KG30RlGgmpxnK\\\\\\\/MvLx5yAbb0WF5Ne80QzP8TQitA\\\\\\\/83jfEaesWXi47u0iaQNl9ldXWkddip0E5Rh\\\\\\\/rEht0OPcWmVfWibX2vSjOI+z3r7PU1pmErEp9uA6rYou3kSnG42ROSQ2fKG1a54jeDlri6fxSsVODNazJrl0k2Jw8dFGxUONzUrYEeUs6ljCAClzRZLX0RcRCPYZtr7WZ8hPXGlRnW8gLNxOyN6FUHT4Ws4\\\\\\\/dIw9d61bgGAOslrWAJTUPuBtzrrWCeTF7QA4BafLhOs8QbeNTr8XNLXIGoI+UL0+SPUHT6Zr8KetfnyRVuqVBVC6KIcMJ790pcL3Q31Ju\\\\\\\/H2+5qPuH7jguq64D1AfzQ7ZcIG8PPQ09524QeFlDiLdzgTNng0PsJEIqd1ZqRrrfZBmTSDyyimM+XuTfFUgA6TayZts8MvXkWVvJBsXLy4Oc+IVhRA3MUgOWMmAC1c0ckCXYV1OMVASoczc9XUOi28qJkYsMmyIdGrEEXcWhakG4qcR6zBDzfw16TLDX7Ljskgc\\\\\\\/vG7xdL74PoS8PFgRnEeLt1V3vj\\\\\\\/fod5EKzmCM0ywAEIovn1MT5\\\\\\\/GR3ug\\\\\\\/btrG7RnEf4Jcn28HgLQmC3vlJdmryN1KJOVVgKss2lCWwT8biJl8OftEX6J\\\\\\\/o8Oyxm\\\\\\\/Tzh3pbYrr2\\\\\\\/t0mEYWQ6esTetrETDl\\\\\\\/1+ajDERablBe7kGeDVEUPhSACGa21GDQaABsj1SczXqX30BCOcUNG02+AG\\\\\\\/qKGEGvtyjsDgc4NQLzGb6vNK8eZiqZ1NiKujLLJKPBw9F1jFgYWecUUzOmQkEX1CSwt0+fy\\\\\\\/WNZIlwLCmI4OaVb1FOlyrLA7oPrPGm90CLsxMuyeuNnIvC+2PsTGgnEC0QFVKGRuM6YBBKvZiwDxlkKd3SXjovGbWNWgYJH5A2oGPMX7OTFThV2igUBNrJtnyc2KlAcbFiWG7sfGhEuyJB\\\\\\\/hW17oHqUK5uaO2JXgr\\\\\\\/bhLAyCc1xJLmDNF2tU0Ezs9PsIeqzDzq3I\\\\\\\/+2zgChiCKbJdAEoWuWc7YHBSSrkpXEVh6GiE3ytGNngD7HDXPQJpZEeyvi8KuYpciuwRaCs7xQleTsxTRT6oQcY8zQL3oXWFTTu2HSVvZtLomXL\\\\\\\/\\\u0022,\\\u0022iv\\\u0022:\\\u0022ThByZZFT53wstbRVf2x+Dg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00226c8aedac8dafa93a256fd65c61152ba2b778e944a28741a4adc9a69193b9e2ac\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022JL6\\\\\\\/AOwQgxdEi3PEY+kW\\\\\\\/aD1SiWbdgKhXVp4HQ4GLlz+X58Pg9XEHDs9rQoMzgYYi9Q1QrqwXRO6qGMINSnhtvgVCwXKF9OTZnWn+IKMfB\\\\\\\/tRM5AyERQl9OLeBaL\\\\\\\/kdfjBDPhxsEOQWFVhpfV11Vg0QU9l2Qw47OdwMj1HWnyR0eILV4R2gtkPos9\\\\\\\/3kyYAzG12PBALsTD1t7JF7Y3CXZm\\\\\\\/L14i6m3QyEHVK5J8YkvbEIQkhUQzQzEksNQa1G88rvJetupM9O\\\\\\\/B3sQuMrpzJGaYauCoZZ1pKIXSmyCYwG00Iu+lYi8iV72whh3rPsbr9W\\\\\\\/cVwV9yBEW6pYK6o9uTYINK2Scg2wmYX95o\\\\\\\/zGkbK1fq8f8Z+Y5ofvjsMIXs96pUGRfrtlXyUSkRjE2Lw3Af+x5fvLnqYDg65op8KktDGNA2edkiJsYYboeJ4BmpEF14nHRNWx2mxcbCWHa7w7+PHbbWpTUKaC3MpFg\\\\\\\/OcjrBKtbSROKBMq4ajpn78Ll\\\\\\\/aVmZZpzUTKw5GGxEZL+BUbzgDfXjQIrgVOoMOX6hj4C0EphRbQg\\\\\\\/f4uMHQAGXD6Vv7T2N2TivKBhV1bu3dTR4UJLKsSdV450k9fK4ja7lp66NLSVi1VfWLJOguqTmZC6OKcZ6wK1uQ6sdnRUPq3KlD28Uff1LCZVw+zpRtbJsuqkHQz+Z7lgk+HEiRJwJwgS4vVyyckdZ6qN\\\\\\\/rZCnBIKhtMY9\\\\\\\/IrKIVreR4zYZcwEtsXhsrAta0gX9AiIm78in708BKHKh\\\\\\\/CCk\\\\\\\/acKHRDwCB6szMPvdoax2Lj8sjk0c8ZDqE+rbeaDocDq8U3IM1i3ra77w5C9k+lJV17bQLypWfG2QmINAzalZjG7hxvAZL8Lx5Kis0qhRgcqYeiU9b+4jy6xDiqsij7RT6mNDpRSxQRtrYDI3RFWzUv9V7OrBRc4FEHWjIW1cG8Rmtd7klCTayGU5mQLsnctkw\\\\\\\/ImjT7OWvLAhxBti0NCG7EQS07arK+oc6yqvdk6NNCaeX7v3CJjcIbUQ38hS6Pu6jX49gM\\\\\\\/fmVKeGpdDVLq8brvyEeMvn\\\\\\\/Hp9liH7RLaoV\\\\\\\/BXidI1Cl8KizgF3wL6PkBxwUqFoFm7vlrPUXApUo98Udn0HvHYU7e7EO77Oe5EdVeg9zfFHPNMA2DwvAuz2nwtscbuGGCzkn2XK6Jny9cOF0ZkE8wEnjPzAXEO4Fj5QKrdrZItmpbgRFwp+kG\\\\\\\/j+lyjP8DwfSbkg4kBXTUVVvvQLLjHvj3\\\\\\\/JltYXADTz\\\\\\\/7hsfXu9KC11nr\\\\\\\/YzMQq\\\\\\\/1OE98aN4Kn0V6WgW54xHeObIXHCLS358XKdiBmEjthDnQbn8ShaN\\\\\\\/pBKHqJR\\\\\\\/na919hbY15dMXppQbBSpA4SY6+PLalWgfl3jj+nFqnzJnD0PfCYGStN0\\\\\\\/GHoPrtiMUisbYeRvKqX2x819vU8\\\\\\\/0QibFu1lMEiY607XwyCMLEokdIWjq13In+vzHnV8R3gpKiwMikHveD0dYDwCs1Dp8hClL367uLMjerWBYQSD7oQUvr9N0OgMv3dJTdurX3GlsFI\\\\\\\/\\\\\\\/vtAX29j\\\\\\\/55VMXOWXCMrgN62ZdUhakI32+XcCoQyChn72gzn9RUKLu0woO6ip3QDiJm1tac8ezakVVS5RM8kuJ40yHkj86toGDiTEVzFXMrdSF6qbC\\\\\\\/U+WdwUdgSCuWy+gn7pt6qPZyoTyXeD5wjnuZC8sQJjhtx387g6FmqZO4PnHEx\\\\\\\/KgQqyLobGd7jxmCteIMI3E8ibiJALNPXQWQqQ\\\\\\\/yuPQiOk0fp4HFBnG29DktNguRsxCkQ7D5zzpA\\\\\\\/L5tA6H1CguOs8Afqluz\\\\\\\/tC9WvL7UgohJbb67xEAytoCvX3nmosJi6jqzYqvdTRWR8yI38HqbG8KiNMJa\\\\\\\/JiMgHO5VX0X\\\\\\\/NrD3GWcLa9TiwvVn6WLvXjxaZvj+kvx2N1Y+chFQ0o6qEWbZXz4L5mzzbU3yTWhfIOgl6rwARlLSKjsGTcp8G0vrPK0iUnsfFZ5VDrKSpqYpB5sWVULvRlewIqaH8iO8oHH1JL07s8l0OKEG5K5qRltpEFtxslTMvdQi12l5mEQ3Y55xV6uinvrP8b7xaDjGfuhOKVyZGhaZYbnuKstp8IIE7JYX3iU7lFjyQpet+JsmLaC+ADFsCALEsnytJOl9O70zFm+AZaN4JJN+U3uBzAmm\\\\\\\/ezEgcgwIJ8PA6KN0dC2Z6fQLvRoziUTW1Z2ti2DXoAVFY8C\\\\\\\/75SmBC78rYYESAIqomAM58+OS6pyxuRlWBraKH9CDMNJbkN2NKDaGJrRtTxopgGd0nO\\\\\\\/QdZTZvC5bW7vBQTteJMQVtOu+D61NYZoiZpPon5041vmZgm+Hv6Wsjpj6xqM1\\\\\\\/iovj\\\\\\\/qigeILgSsTwOvD7pSKdXLg\\\\\\\/4c4KQLxEPuYlrKOBEovlsFuZAZjPSFltn1lz89ZkNe2EI4dSrzDnm6EV1PKRVMKc+dwjr+GRKFS2IikwxRd2594VU3d4Bol+rMu9QSQCVBSTdvUBAAdQ2h4S64em8m1DN\\\\\\\/5qXqrtni3YgEskQTSO8VQytYTcarNnBDQVDE5J5fQiew+FdjgQKaD9NHuHKvhdyrcWdo5R00FBrJOq5YxkKZqAp9ZobWbCY1oplWQAmOLuqqsnzgwMFiGXsOHeZWbJDtrGBqxi7P82Ijdxm3mgtPTRsiKwTwy8JzNHvdZ\\\\\\\/wHnWryKOqBlFmhwQNdA0QZVfOVLAPQwgvpyukY49niYImN7TSbdLveJXTEaLAYSR2NhDSpJ2qvJCirtYNH8TZhPZv82uihyonxT4vtaTTtmIUhQGWNpFD7XMKqizXOON\\\\\\\/PRXOUPBx0zLsQ3OAR35wZoSjMGIEcedu+JvNb9GkWk6KqR77bBqoDhN\\\\\\\/IVbQo1RDEAUrK85RgKAIx2nFTrNvPA3o3+vaxD5PnZOIEIOuSakWyCxW8nUNhxeV05oANKkgUhlzrLQGGm9zE9lpTLLeEWzlW+PbsCuE7656dki9HYIMie3kwSEmaQ+xsoF4KxIfQqr4CoHUIEep86VvTbbNw5MAmYK23hsUAkGZNmIlv1dPdgy4udhWVbFpzNBwuc3imhSmcbYyv1sMgXgRP3t2G90Igz5Nx\\\\\\\/k+BHpHddEgFO0cmYzHMGMMeaKxaZRT2v1h4QEZTlGMQoUNjQcwCoYkrYQJTydURIthb+d52gWMcPowuoR9qOn0H91C5Og+SVoSUia+F\\\\\\\/lUsIG\\\\\\\/eS1ZddvCN4\\\\\\\/EbD65ex2jCiCK+OL5RKwN9yOJZfurRWnJrZa1zP7sq+dENyFz6g4ml01zDhTFx3u1fScgGNQWHteNzfC9ubv\\\\\\\/y6DJ3JY0Wn81vYn5Hrl9QaWeA2jnRRKukLi9TVfoyuZ4y3pU6gmLPZASb12QSWGqolvzWDkKHB8V\\\\\\\/9lA0QbuSVylSzfdH2WgBgPLIqLljnnA9f4k39O7mTvu0PxJOyXrfs7e63UERUG53tB6C6Ucpq4dBYJH7cIoN+n7KpkxJ33BihlcIZzZaLbWnMzgCq6j3He0sx+buKXP10IV5bwJ6jzLjLKAimqcCl1wfSQ8byGjOUl8YL82GLa9R\\\\\\\/8JXRKlOKNpCmANMdiiHAiT5Z010EFDnVv5si54UVuENVz4dJUFlZK\\\\\\\/8G0UFnd0wTx6WBhyL8x\\\\\\\/yOlEm4utZeudgIkDv9+dqXWuPieh3JcmRkClUOGpRAuoBBRv1w0gD9sbI8f6buLmF9uroqewkKxcPf0R\\\\\\\/SDz8gHErODf35stQFl4jU5OizdpSbzcB+gcKkqAxJxNFOnWrkv6Q1m9MTgbU7Q9OJiy+9qUP8vuSSt32v3fx6UzJsCCJPkaErW6BU3UOxmgp8IU4LZ4ela7LTovFGRkCLKLV+UY27LcHkUQG+ve4hQX2xTiHWZbX4pbmOyx6dM7R0gxVCbJf9HN130z5kAj\\\\\\\/zCHLyMJE0keKaiAeQTc1xfStYabkeVc8yyehr8DEq0eTBK++oO4nIGbt5y1hdbij0pyhotexckeEGPcsbIpnz5Iojwy9Ikf6dVavLJccfh2UjGHSA72bBFUQV5dKiX9eX5nSeCujJwcDWScNhOAswJZ3PI8qjpE2Fn+X2VM453SyhRAa7nhzyw3rq7sGRqbXwWh9XwZZKQC87uburwHxGHr\\\\\\\/AEpV07Vu0WP620hW81wv1jbGH4ZRlSc4qNwQGN2F0GSIW06f4h5R9\\\\\\\/iUGlIpy2qWMrT3LA3raUeeifiA5++cYXigiPs+NRtQvwsoGUlh2nrbOYqzqf6EatJLn+wGrWLG0V3MtaFshB0yQH77A6XfFosKcmtLnZXl0mItpnSQepDLvX\\\\\\\/m0qL5zcr7f49djJieitjLkuYjRu5y+QqF2VDk2Kvg0BAybVnskBfVvltek8fO61zIR1LSa\\\\\\\/ZyKbHTj32BYnQgwIGi3\\\\\\\/f6mRKAziGH+5k+iHNLpkyuQM\\\\\\\/6L2X0FYkQAuEhijkmBv7LGgfCstRpOQGrE6Bsg9XR1F5GpsaE4SS3tO4m8QjMudXll7oGQmTC6KIfWvaIiipWJZB19qjfvM\\\\\\\/l5tYK0MWVLoY7ZsRs4zT7DsPkhZzsfDqp3EzT8BH48c5YnE3lyrUBoRFg2V\\\\\\\/5nZgz887A4ricHp+mgoRXhTJLg5Az5HnNHlB2NXkg6FluVwUEs5akM3Zfy3t4S+taqmo0JfmhCqjnXHOvTARyHwff4DcC+lQ+cUKdZDafgkqCWStNp0yGSkRNtkyKfmxqOWSrcBfiF1v0iOcPbcSg1tFsqHRgNYRhs673MmVHhrLmvPf5XYXREE4i+R\\\\\\\/oMPBEps7Ertjj4xZO9TRoatR47T8rdf2MrFTGeOht0XCwPtJtR2COlrEglzpFiMu4LZs6LSYxgJahhcRI\\\\\\\/fz8NbWJUEQFCe+Vh7z4DDN8J1RV4SZm75JpLrJY+gVLcxx2KYMJhRN7WEKlFrOO\\\\\\\/uJCTmNVIoKSRVfRELLR+2KdrxKgX0RV2WNsmxPMaiSqtdaq3a7MTLRQTQILHPsciQFGvNzcgQ+IOMj0zS9T+Salp\\\\\\\/YczxV\\\\\\\/KAa1rwGO6wZ5+bqk+Bgti0eZY0904bKZvd\\\\\\\/uZmYWB6UAIhzhSilzxNHLxM6cAiCl5v9+l0EIVQjCnSLddFFVf8lMDusyZpqdDmSeuEJZjObGxzgb18oIpYQBMB9IaoBg\\\\\\\/fKNxOtIg9IpCOf6iD+ddeK1FAhrud\\\\\\\/UXjoQFeHysi28mP6RSqq5t64C\\\\\\\/dxZZks6Ynvc5mmAJIa3RCFEaHn2hvxFNMDw4Zok2nJNirnkPZlsyf24g6aKSLclqfmBIp48iVIiltgde+zDq8akH0ZzZxrccm18\\\\\\\/NNKxdvWLFBhbV1YZEjNIYnZlCXygy3KXQvsr2Ze0xcUa1Eoy+D+a7hg+7Nrvzplc8yEF\\\\\\\/v9w\\\\\\\/bO\\\\\\\/J6v8Z\\\\\\\/i2lbNirxrtnPLEtyoJ6\\\\\\\/B93yPmnpT9ocwmqgqWWV8dcWRZRxfyyzOLmngGZNzEZQDulHoXvyDNPg5vPraGdjd7rGkS4ev+fTbz8n4QunbQ\\\\\\\/uMU5lrAxuo7ssw2qLnCaYBx1G3ZAGBTrDH52bU+macbMIww8TVhbXnDgxRSIWYaFc70xgysH3td3fHwberEdWlGFX88AdOb4RkKLtEHxOuLGmKdFvgzSFSuwjKX2pTaVg10wQGLXON5pn6xFG\\\\\\\/1SxBJWO36yE7XbMV4rOOTxRn2LqU02IfO3d7ceqcSc39GDjlsuJhJRhpWQckomBX\\\\\\\/tT2P0iuHa22jHam5w+jlz2wl9A7\\\\\\\/12EPYjgN8vGS3NcckHAepiEXu+g3r8nHfgBFxJYwEQ4YUlZ+4dFLBUql5XvXqG+PfE4sJ3iEZpx5q3Xd+0mtgBa2u8P78aAbPrecodQYrffr2SY0ODXy\\\\\\\/4Ky7vcl7BK987CmPZ2qDQP6syVRdS8HrXMt4c\\\\\\\/8LMpyksBXyzQkxT1I+qEzEh5YcbvycMuaq9waXSbCzBaCD8hVa5nsJdcDbnwzcHrd9vSQRaYx2gBK6uFA0Qm8J9qTsM0uwnkUHIA1ULM9m7TsNapgOzaRyPAOChWY1A63TrRkQuHYv2jdpAk\\\\\\\/ZIe1Ap8+pzWXs36XXwIqyj9MZm5mRlY8N6LqbB+g7q037vb+Y9ky5aSXNqCB7sD0tYWIgD3rrb6wLojaqAP9O8lN2NPXpgbdQM9lqZkw\\\\\\\/PZzMIihZPgMXwQfe\\\\\\\/4be82ZPfaVlMprfMj1P4zxs3sIdsuxeaiYEqAr50iHfH4rqfcEL40GySbDE1p9YWuyk5xkBd1wvUEjuvtzhaHafHc29Ryct1+mSeyevAnYk0HkBmihXdBIt5z7NJ\\\\\\\/gXAJiiIRUHykO8zkSmOPu+i0weU8LIv8mPMHwDkkFQBEifNJKNrjvdwjgfBFXNIxQ8MHnL\\\\\\\/3OGOu2S\\\\\\\/iIUB13MbwrTT+pcuDTHfDowIorsEx0f7iQJFu+TfRze+JWjXTl8M5bYmTcadk2gXou93ByiQhsX\\\\\\\/Jcr\\\\\\\/sIhieOWq8YHjXlnQunSWQSEJ0MyRzEjucPkZGr+BYnmP3Lls2azwZbgYBCB5d9eFScdXJDZQnwITmM41vBttcxaVzoOonNtZO3ckfdOS2jyfYZP13P8Hc8HcL1Zp6WUAr3KCcTAouL0N4c1duzsS783oGfs44wU4AK0TOiBKkdgeYr8p\\\\\\\/dByXD98HVMCsy6xB+Cj3rnluB+2T\\\\\\\/QNh9ears2wswBWlyNhjzr7PH4A0xLJGYVBp9D7rCjO9BQBEm\\\\\\\/9DXkCGFoUSvuKq3o3rs25NjkK0qbSTzy9A3ANaqEVRpxRduDrs6il2CHqzjAgv5rMfo4kpN9YBiOlsBpOt6v+6TZ8iBbm\\\\\\\/NjoemFimPbLk9ydyJGmH9Z+5iooJvBrMpcSWrN0M+OJTkTDq8zBHH0JcENRtzAncQUtBIBcn\\\\\\\/xqkMThxJ1HLiQtXKWKuYGxDYDlOP0wv5Vb5ZH3GZ+aNs6UQ4EoHMEQ9Lm4+\\\\\\\/rbVQ1L2n9mP43+GTngmL3HkzmVcp7v0O2mCb\\\\\\\/8MqY2RHlhL88LBxXtccy7uMcAIWs7O9Ri3aqkL2uhtTNsmNp4PfRM25hpS\\\\\\\/UlcBhwqRlq1Mvo0aGUqPJN+3aFhb4ACWrV2n1l4Zc+i6uv3ntYvFi8+qz8vy1l4kS2t\\\\\\\/Ea\\\\\\\/9cF+cgg9EnSfyxcDduhE\\\\\\\/3iUdlCR+4L723yg5mFefYtpeUs8T8uNJ70QW6SQ+eRFBtgWyXZz51X9KvnWeVII7OZCYh8S\\\\\\\/Cj1SHbAghdT3Kg5bGmuvjMn\\\\\\\/20VOabvVO\\\\\\\/gD8ppqJAqfCtyh\\\\\\\/ASiGEZZHTSYr8QpH+RyBsZOununw\\\\\\\/+xQroQ1peAL0Cg7vGM178vjKcNiJd60q4hOOuztg9bQWuewQ4xIZRpfkcn5+fsn5cgFxcuurQ7mTgv6rG4Q7SKfKhDVtC9QpPcf4iGF8qGFOSTf9M2MdUar3bni9qpaQRhmW\\\\\\\/yVj2bQnUrXcXwvVA3IESRzzKhIrc87kwUmdEFznI4nkEK86Zl7iSOl+1VpLoanS4SJwV8Ehr+h3DVBz9FO6zZpHyk411m9x3gJnD1jdYmofjbZdm3TygN\\\\\\\/sgcMI67hXcORfR7RkZd2kGZkpQtACePFFUYMjzNWJKUmdmGTQ9Q7\\\\\\\/p\\\\\\\/DYcfIQYu3WTWeAwwY4JmRo3Vrb+hBk4R1srMs0a06h2kyFE3ULw526TMjo4szWkeq99rudKJqimKy1DF+UiDz\\\\\\\/6fySnHIgS3u3xJmVGcE\\\\\\\/gIXwNKVKJ7cQ3u2W8B7qQETjpFIXeh6qvh7nz83NgR2\\\\\\\/LtNcbVKX8AWFpzFssXWSzXYXchA2US3uoxdBgbxUlbA50IorX+wwle4aBA93FGum1QR701Q1Ch0IFAnsTMVH+6Gxrp\\\\\\\/FyFlDHgxPGmf40gmE9zvdR1Y9\\\\\\\/diL5W5EeShyQzsC+pEH9XopouucMDOAqWSbAhcnA76K3PU1KHYDoPmIk\\\\\\\/+9rbxTFm3N+LDYab\\\\\\\/TiMGkf4wQZzt9XjPN1fFVU\\\\\\\/+tIEE2GQYiSbhUM84w9mm3yV40TNKYU\\\\\\\/jkrQcqtS\\\\\\\/CENyeDC6BLIzchZp2Kdbh2I7JCz2+6QiRAsgKmeN5RQbPCj2ost4mqEQPERneyn8ivuyns7CQ1vdJW5z5Aab03F5qxew\\\\\\\/B51BW2N9da7uDsogOCxT13vumK1ZPd6cjMmSXie4\\\\\\\/ie3t0WDRvg+SQlhMUiEEw\\\\\\\/q6sx2Ni9AXJQV7HUCdaedU4EDCvaq3ehvak6dl7mEdKcUy4Y2Y7pS2kBWwfpdGr2NLE2QBiduUnhTOWdIGk9wwvlrRlH\\\\\\\/aETR5YC8imI7twpvN8+DidLf5CZ8PkeIWMZORaBKixpuvkVhXUGQGohBkiG5IjgSVQVGEeRkbvugPA\\\\\\\/s6vIOr01VbyppmvsoIi5wZxW+67G4qk1j+tKweZbwiJHk0sSeXvfLoZwrZqB61onTKXwpgdvg0Thc6uplDfum9leQQebEeOb7P1p9Z9+ELw6qIUwxR0+otk9Lw14AB3CbmmptL3D7mNtoUDlhN3mH7X44c4l2GhXHVFzLGudJnp+z5rQ6a1AxUez5LctDyw7ilHC\\\\\\\/LlfAyXPBvxI0CneNEpsjKf\\\\\\\/GXf6EV\\\\\\\/FeegYC4lZvIn5A7qQa85RubudlYySrxvSlKw37m1ygORD3NZSVENmz30fSCNJ1ceeHsuWeOKQbuFvqxEKxbcKOVYrM2vwxL8xUz\\\\\\\/oHEfMqmdH8vOUbWIjH4uHb6KCYeINxf\\\\\\\/9kPGoaSV9jgVXsO5prG30OOCHo80wJQIoa3Dq6\\\\\\\/HtLLn1c\\\\\\\/BlskJA6YFFg0FkzfIaCd9DrnV7tRdMmI1urtybIAiiHV6kVT8K\\\\\\\/OslNZgM3KyFhllHUSAjJUzh46f\\\\\\\/SxsrPz7GMWpt\\\\\\\/+45QATH+3T7EyE70dZpkhA1WfUFp2K8tELhT7fOUbM\\\\\\\/zG4nxzMoAbwCn9\\\\\\\/QofGBfviPTzXoh1ECiFdH3LoRVE3dTpk4kQT5ddI9TIF4JEr3iVqAYhW9KdhtBk2+uxKPd97NyZuDcTacJl9cxkLHS97iNV+sLDJ6Zsoi1WS6KzcSxSH1MlH7Gw3JYRjq21MOetnnqTgK222gLnwn3NVauoRnPhbj5KqnTgiVF+7KEzisIJgPAgmlTj+EKApaK2gW6nEtV8xvn1kOeJdK0Xg28yB8WXszVyTTzYqA9VOXoFXCEpphDgBB0QymLZ8kIvEzOwjO3lXrqUCU3DfkrcSShUwQe5B+n1AHlmaK98oAC+Jb6PdatIel4W6TxM8DCQl+qubhpwS4tINwqqV+sIAZGpxoVUAbxNboTn9LVHEtyiXu9RwiXZ68qubi33M3VJV+70WddilKZlCgXD6sw6dbFwMOUZ0K8XlojPtgCsglxMh1rPu37MvzTXnD7APiakXWs1q1b50O1Jy2CBUElx2jGGTsfyhxin8WZMiXs1Ym8wRl5AHOe7x44\\\\\\\/Ltny5Nc2Bt8C245gMUhxCKyn87pulXMK4xxydSMULNBWjPiHd+zcsr+P4vVw761lTluEBPLDfAZlEHaTTNWgJKh5gX0zMzsrybOlH8NSdXJivdm8nEi2UtQ5uAcm7LCg1ScmgLwUDBScMq9c1iU4LoXH6PBsUn7iiBKbOxpVlhAzfwgqAecgeSVQ3rlNWaDc6d4jZYf73f0RAOYPsd8GScOJWMH1pJAnG26InSLzFdRBAQMfqb0BF0n47tlL+z6b+r5JLJJy+fz7WLbEzPX3+oAgN+kOFiIN5+xDI9DumNoC1Wty2Yjc68ahMtoXHyXU6VhDvnvWVE0jAHvegSSyZGvecq0\\\\\\\/bN5tYr4+rdH84sHtCLBGFHZYRojcTtvdh7wXna9fKdgeyPqqQ+7rzVZKrO5mp4UnDW3AQRaCRjB5wzS+FyqCP82ldWfAwXGkb1iuO5mFfNLQke908psOdHejQK8Z2XSQeSSVctaZYAHFsoTiPdOnfXRh\\\\\\\/tbwn1cSqJpCMe1dpLYcdA8DSb7S3xor1tzbiFohHHVXkfdMFsrHaXR0Gd9\\\\\\\/s\\\\\\\/UUZ3L8tRhHMPJMA4HZrCpcMiWQM0KHTTj9KN9Tt2zu2kDWlOpHqqhlKtQIfSOyG973apMAoK6u1sp+QJlJa4Rv5b6FT3ifryMkIKuw9Sazup+0S00LmnXoLFeelB8V0RYX3pbNWCXENZgvxwcPTxzFUVqtw3QD4iLzcrz\\\\\\\/jATbArhNwxT3BVnXG++dM6iDfYUhHeYyQuYcspELUaDPvaBnQNGeM2k\\\\\\\/EB046PH25suWoKoYJRtj\\\\\\\/cbHShOwU9ZeJx1+RjI3Zj\\\\\\\/pBhAezFBt8S9vEPt8sE9+cpoI4O4ekllxtja90BxthqIRnbosMMR+cfAHQd2gQ8iDEx86Pf32R2Te5Bpj7b9NgEAO33H9dOlkNb2erTyul3Phkrs8rP4\\\\\\\/D5HL3LQPc29vVmI01bEopo62qhCr0ljzz8sksB+nKUjC1GAEnsz2GUDbpE90NZImApx\\\\\\\/OV0N4hd19hcUJJYq703OZL+pBItuuT+NCqu1Rdzpe5hx5j1LU5Zz9U0e2jErMfLGVhRv2mQDABHpPLO2lD6gXEVPZkar9Kdq0NbyHZZOjY4SMtWD0+gX7O2NOSHhDQtTuF5qL9zn1D+dxunRcffh8kB5VQzBunTIZrygV+UtXKsN6nEN148SVaH4wJQvQ\\\\\\\/Dk8kD0ENFRZWWjJ1CPfQuRr5\\\\\\\/BiLwQykLYHoiXSXefhgG8cYZZ0hbMBNYlcnF6QFd8BIViL8KgJ6EBAsgd0ns+VP8RnpbdE2y5oReZVjMHmWR9VWGBW0lksIQICBAcylc+4rvL8yW4MGODrK+n7I0qWH1faIN5NHcLtdIztHVRJ896y3kPeWmkUCdW3EIOkwW9ZUwR\\\\\\\/x9I5Y2ZZk+TvnCXPm6+xpqZ\\\\\\\/CsqCsOiVWJyLIvJcaFqElVMe7FQ4yfzFm6FcaWdjGr413ZnDBo0X+tbOt\\\\\\\/t2KDlplv\\\\\\\/d2R3C7Vcx7OdJIxLHBwzQxdRIXPJTHl3N6rnuHe5xUwG0ojkwXqJWZjjgV8RUwt+Mm5wM7UvWjfX4RZdDEGa8jmlveE0sFg\\\\\\\/OwaLgwFGqbLHzmAyu4grFmuICfmShf14qTkyRwfVvMcvy99V5o92yeY8kTRm4KjqbQqFsNI\\\\\\\/wsGweM\\\\\\\/BfHgodvsr8pMB2C75JW0g\\\\\\\/aTRhVtkxlAEnCqzoF2\\\\\\\/Tu8ynrXPdRNgpa5R0ZKDP40lSdsoKRa8T\\\\\\\/R1T9ZikQN9eKBzWGD+dKzBkP5+gErlVF\\\\\\\/0k0TMfCHUFH5noMMzh12k5vOWigkqLHYnZM9wS1WfgKh8\\\\\\\/JNewuy\\\\\\\/9rO3JOeWzlgW014Co8CptvK2ZR1\\\\\\\/in7uCOwDfR0B5ddz7jVUgBhROGu9\\\\\\\/IX7KinyN9LwUAcCvLlHnuDX95EFt90Cb28cyc7JHMYDUrVDFm00O6LvQ6D5iZOq2a32PQejo5jdZkYLlURwc31f0YeScJmJKHn0\\\\\\\/8zLkP7TUeb8v19t0w4mDqpJ9nG4kqXMGrncgekb8IDQ3iYhCf6Pp6mbJsjvcSPwY1euxAh7EtjBWpTDQLtyfUAj7O0RSRDLg7v9Mjw04wTn8h9xG44Oh3yxccN+INoBIQeWd0oUaL2kRuRuBmJ1G\\\\\\\/XLBvcBLF0\\\\\\\/ipcGY2PujbekvCj\\\\\\\/24NAWk5kZvWmHx6P+bhUDZdlfGd8pamD3q3BKw1\\\\\\\/wPp\\\\\\\/47vFuUexErjclpIIT0gd2+NapNhWTFtUbNz6PPntTS076B2O2G4PPm946cLZFaa8mbvg\\\\\\\/5ZKQuCFPSxxfqlaQA9ZSARX2hpMaEh4tOlLzWjUlaOfmOUGIUgq3VtdmqcAhe0PbwqvAkjYqFwNQHEzyioR5ff5E4GkHn0L\\\\\\\/iL4\\\\\\\/UB7GAgrak+CzH0xme4hMXzLE3wJFWGJ7+rY+qoavUUZ2+epXt1w47QMCy5kMa3MKJVFah3kMCSU7YJbncVQlhF2AgsnlegtJERJrrk8atdSheTpxQJcEPZ+1fSpKmESOZi+7LIWno2am7u8EiXnM9vNgIqakyXl6P4iU9wt9lDHvMx17gEbZvqsQrKEIt7jeSSAVbUouRvCttOpwbM4MJ1N0wTsz\\\\\\\/gUbB1roBKympZJooVzovd7mpd1emIeMdCirb4Pb7xSRVGXmO\\\\\\\/SeOkhmO76Hw3VepKlMxzDF5o\\\\\\\/f+7RnA18upMQ68FvpPbndjnzC9E1zWRyTJd4k1o90Uamdrfcf6XjdbIKWzmAS73Eu\\\\\\\/dfCho4H5Gel32rmMRwF3KHp4e2Vg\\\\\\\/rAGFPS3NuzQI7cohbFZE6Qeoh4WkJ8+Y4QmohnBqdDUMrhF8v+UUZBvcasVbMqZakCB2GPrLkfEGTNESzwWZVMVOy+Otded2c1XlXjkv9\\\\\\\/Atm+0Z3sp\\\\\\\/t5rSLihR0zbsYk58DqnecjPFTrd1SNS1tWjc+L7KmctXsTddHtHK9Ch39DA02iPCWTuEivZBE2OKoHxWTNUIpINqjoK4J+p64QhiHWBrew65NPxq6yYhj3go5DADUdQxBGIRmm9hcvNTWRw1W6o8Y+JoMl+Zk23jjQ\\\\\\\/QryfhH+z4k6mVhznsNuzzJiF\\\\\\\/RQMs6sDbwrYWa1cOOAvEhBHI1ZpdYI8x4ureeOhHVY4Q0bUZ7rpcjts4hHEQBjYGutgyhEH4YhNgE5xuJB8qouLAin+ZazAHGvep3DrEwhOICTVrC6yqkarqHPNL5p9iQEaf6aU1QV9\\\\\\\/mxUIuAwc\\\\\\\/q1jbYSK3k7gFuVyrgOPxNZF7G87ZOnRMHT2bA5a\\\\\\\/b3R99GngXwJruAIK\\\\\\\/jqgJXzF\\\\\\\/dVh9R0CGNXsDeTGkKwSxTJkYhB9t0U5Q2Nq8quz4+egGrYSWsPMEXAKgDR5DagNiGA\\\\\\\/YBg1lCHn4g\\\\\\\/pSdysKWVqqCzkguay2IU0Bc+Hd58UqwrjyGalKHPV0wFTkT8RMIFEFOmmKxMmABkmkOcEtjuEIH2aSmzJzMWMo+OHEXCMTVvkDiGe2A8GcYQP0yYzz9VIJeexsQaVvQPHWycekVqhYUMQjeCP4LgyV89Cwh+xCdiS3tVF8hKzCuAzAK5rdDqKli2imYCiZtP+smmQFWxNTf2XCkixDlZexuAfOJc3oTFk0ZgHNvYCWZ0t+e10OihnYZjJI0WvswPSSB0FWqOLg+zOmDkIyxx1S03Ips4GGpRI+C6oxaH2UnUtLWgPs9s9pLPsP+gx\\\\\\\/3Cfv1SM22WkUIrxOI4CmPKOfZUmixsTR1xBousNK24x5LArGG\\\\\\\/E0YQ4Gu56oMXdYS1fAJofMV+JgKiO\\\\\\\/yCoh4C\\\\\\\/jhURJ3EBHj+KtAug94PcYn4gr0lbPN128QJwvs7K57TNAyRYbjfotjJB6TuyTxsHDiMuFe7xjRnBhjTyAKZk5jIg732yJal\\\\\\\/AVZBn0MvEgzn76szj5+jGuZvWKMIFyvYYKEw8i9IC9oxthLlHhxz1+gEoyAVRfcEwM9uxo36D9nB5RNWG\\\\\\\/KOQZQ0AgZtjerkBZQhqItyHKOHSXMgpiaiSzN75xEJDmqZftRdg2DO6TgevgEvayowna3Sa0PaQVXzAUa4GSC3lzZkzUjtnxgPscolpxRGERpABo5r6iHOWOQBIBx8rTSex9MQd8ONzSqaNzxNxFDXUqjy+8KjB1FUSUd++LMuiwiBmC\\\\\\\/pnRPWSCAD6IM0Cafkk74pJicC2AGdNG76zLkE7w1LJBfsuvJ8UjA9PmqR0YQvSuoILpF\\\\\\\/MjpfB6uL0yJZQmXsfilE4pjEeQtbKQpRKOENM3GNyK0\\\\\\\/7zv0QvhfB1G0cYSF48CaKaHwS1LFUeRnKWadTQIC0vy4vw\\\\\\\/F\\\\\\\/K0t6mEdGiv\\\\\\\/py3JdAMBgxARiIQRuf0FpCE85IBQzOloGp6xmwEETSTGnZlToUJ4v3hdh9Jbi+YPOdSOubQ3eBeyCCAaVinKwiRKFIAJDV0OsiC\\\\\\\/GV2nLEc7\\\\\\\/VrIutGx0zBE2DkVxMQ6oIxL9uyvdWq+L0HuNgePf\\\\\\\/AJ2ONiK8hQtQPBudZUsYu2g1zphx8yPbKMyMc3D\\\\\\\/Gf\\\\\\\/8e8X3Xz8hmvld1WXZFkwHcr078dYppPDjCed9GNBYrztdcqrNuYSg+msvleJWtUAsgzhbmruzZk9lOnLk4IF+CLoPmxYYHOKUbIGVuK\\\\\\\/YhRStkaw4X5Q28NU+sHWjPwI7zps4A7znuXW0uubtSBu59oQqMa+TANtMISBLaH6gkCQH+uiZRbk2fZOBbqlVvi2oMh\\\\\\\/2GhBrpXH0mq4xoWRR5TAbu4vLNNjeiYbrQkepjf0KO4uIN7eX6lR4y\\\\\\\/SjaUUY7075taKKzqyF7XrLvbZL9HihHXGahzmomm2FAENRnA73OOy2j2xCfM+\\\\\\\/q\\\\\\\/W93CpJ3EOTGAJuQg94qDEvMAIehvTGas9Rhc7+n3Z\\\\\\\/c6TVtRjck1H0vdMywTybFGhMfKJraagKyIiLslMdtpiGaM\\\\\\\/pBlG\\\\\\\/l7MGSah+Xmtn1RYS\\\\\\\/plfVLQ0QL7oGRmdg3BBT6Li5usGB5qcusPOqe3KhABH60VnVg5z7UoaTxUfFlQ3qjsfRs2U7jgIZAvT7pf\\\\\\\/atUoOrhZZfBOUw0RS1kRKp+TBJ7bCW+wUGKGV+7jtPUiPUvoH3G\\\\\\\/3pBgx5q30JpmWgnK5\\\\\\\/zi7c4o7xrw0wq2we7KhyOsgHfi6kiBrjSZDZnP9sYRjA8GSZoCLzAQnf85khBO7z018BYV71QG0wcmuMXb22TtwMQYz\\\\\\\/\\\\\\\/H+c2GXKY4NDPhuwQ7iJFd7eeajkTr+rigAptsCSejEZg1JpIUi7jI+OVqPjR8TzQDgeFDoNAw1qCfVie0NpnW86+guj\\\\\\\/tg1+DwpiMM3i3KTyj+JOFdLvoYXlEosviUKIfLTfG+i12DJJLM9kal1P0mwlMBbFNegfEknAq6pGLIHQGGy3ettQg+0vNBBitO5QSL8RMWQ\\\\\\\/qdLk8zFSvJtSQ7Who6dihWUefCosZ1q9u1d38Or\\\\\\\/AjHy8NCkn1IKwumjcU8moIqF9r8kpgpTHCFStPFol42BzZbiTu1F2U\\\\\\\/65qPt6AJms78qI51oGzkFOKx\\\\\\\/IDfAIep8gJszcxA+7nfW\\\\\\\/KrzRGxK+\\\\\\\/gGNOViCgErYt3h5WIZj1d0yWBQHOfjFa4QFIpwnRRVfkHbDwmATfTQa6voqYT8fbiaGzI7u7uKGVM4tLImOiZuru6CBmcOzCMDhFFq3WlsxUX4oTHfj9UOyZ1MY1ZJkYaQGhJj574ob66xGD3DIyMXYFzHx\\\\\\\/D4m+pYJNQV1bWt5RvTMqz+8JiTbx6OVZmbwhYQFLSIBbAMAvptwIRVvtaJcOLy9YBvIscOr4XmgCZh7E3FcuVGeuRqXFVff1BCWrfCH3GUCYn2OX1cPgxeeeMyIOoaU4spGWMZZ9HNKCYmHl\\\\\\\/uO+mc97J82csZ4ESXvFdJiSutewn6Lvd7laNHjycWeAVrtzOPvfRakW5PDMyoDKFkShokiTurupOktnf2LR+pfdob7W2BBr3H9RkUiNucDolmkuj7b\\\\\\\/3AShdIV7yLfss8o61dKgz7ydaX\\\\\\\/++9z3kkow5gG09PAtc3pswKBKKv72UBcmkND2JkuCwcTdPPGfr5MwoqQkH8nyHPK4RrILXpg1GL5mHlm1mLVZblPhIUa\\\\\\\/Evsrsrg9tbtQ7HjGJF92KFDkbKzHJ2Rq4hF2Ai9lFjmSB66eBUW8Q4NAWH6lAqSwESMnNezx9ZwGi3rAyC3ZhNh1qclmHFXYvN3Sj6L25h67hq0sVvOwaFaFbFqejMRE4zmD0csLC96V53hGza4JQX2kl0J5FbALglVQXghI5VzO3e2rnxsM\\\\\\\/96rF\\\\\\\/0aIIpJuo20fzVznBqWWvL4axIUBsfxn6UHEH2ZjB9oXulmdb1xCj2oaCR6P3Q6jaEfDmf+Uhnm73JxSXOZMkXOVbgQfCHaEg1ANJjpbUP+dPC6\\\\\\\/gzLheybsD7YbmADTV9FVDh1OORF1YRLGGchuCURoY2nZAHkNzFFE7UVKZvdCE+C\\\\\\\/Xjx\\\\\\\/xZABM8Cyvpe\\\\\\\/RH\\\\\\\/EM63f8N+h5m8RD65ckwTIQJUf2h9sO1Ze1pUCb63WITQrJFpspfLbo00DWfxQ+0Jj9W1FSWyRN4RawrJI8xIEY0wXDRTUcnmKVNr3rgL3m3oQXFtNQcpE28ABWEz\\\\\\\/YpzKrGMZx5DLFGCY0BQXEKpGrvPdiapWC78URhoV7i6fJNzBlapJscXxJaqBQ03h2GoXj1gHK\\\\\\\/3W+V3\\\\\\\/RFAgfCSEbKWOK3Z3zKMUjttIVLd9ZAkoQ2sxJxLw4dLQ51vPLO5TZbij41xKrdNt96e10jw9C2YO+SRcEQtbEbgwLga\\\\\\\/iID9WjR3ItrJTjKvydFdxOyFzOqJHaGNRRYkRfkW3uuSJPFefc7+L04zvVLohQhhju7H5O5rMHkfPQuLDzYWPpWrFyAOtDOhNz1VayTdPguuJvrWwUhSgH5dUlqJ5YYeVWR0SCUO0G\\\\\\\/LK1xwQ7p5pgtj8YXLUcHm+uvAE1HFH6fPRi0UkYqexLT+TBcb8\\\\\\\/JRQHO9\\\\\\\/f1ktoPGK\\\\\\\/EyMkytyYk8n+qu5IdZILaBI88WojCKJmyBnrq5KDH3Ovy\\\\\\\/Hwk954hSaLOoQounTsJEjYg5yWuZJKBzGPWy\\\\\\\/KLj1mQ5TnIpQykP0n1wHGMlSww3vwzK3SENDIYcNI3B7xrlRbTE4Cu5SbTEXkb8\\\\\\\/MZ8j0xgZ2JqZGUb8MproI3hY\\\\\\\/MKG53VPeKTzo6I96rx5JMCt0eFlf3vOpsyaOj0DxRTtP3SbDTjZ1iKfHPSfSdapPCsILxdAGs67jp5AZDjA7kX4qupPUrq9zCxd6nmlr4vh3893Pa5mkakGvhsoBLvI+gOEXaUZ0W8ku3i6aj0kO49fkheS1xex3qpOIqXcoUMA\\\\\\\/sdyhYI8B+LbNelPmXDJy2+osoic\\\\\\\/nu\\\\\\\/4WvadYfTDsrClIZWmZBWAPcnUbHcTOl5RFkUFKvaZQcNKTtkG7I\\\\\\\/dK4oirzcl3LhZd1jzSUEay9f+zfdfNidXnLuNAjGtQoMrV\\\\\\\/ehkVRMzbuzFtYKk88bNYz2B7I82ezOb4nG9UG8qQsqgvfYqTRRYu8267sbB\\\\\\\/CUt36LmisUu\\\\\\\/9eC1efINyJUs0j2BywOR7eG6ucclBBMpXReyVZWYnSeOJ1+j22QGR1jeGU74+tt6DTOyHUnIVldkNjUUD3AA7R5bWfnZ\\\\\\\/tnuoGuwEBhXoXK4li0yTJnbclqFK0Ua20VTjoUGpJ6sQoWtj6L24nuxhT\\\\\\\/c0fwylTUGatdH6K4LubQYapiBjPuUXisFmbOrfFAEznTzPq+CBVHewwYf+eXou\\\\\\\/q4pHgazoNbjKHszoAZB2BGJiqTxwkBodjG40DvobLGJugJZO7bz9YIqUNjTXYZcxl003DcxgOPi8phjyaXVRrKblrkabfznKt\\\\\\\/qmd+64iewebhHMJyLK1IkC8MCpf32NokuwjMWSIEKpJoLRd8SGqwbpcqhLBd\\\\\\\/+CKw5fu9nLNFQIuFUT+aEzr4wVVC2vJis4aPUqDhQ6IyjMWdu757fnedN3WQ0i3rse\\\\\\\/CCa4TrW5sfATaF23zYOQtjzdL8C7WJILXjjDbAw5OJlelBg5Y3JqtJdOLkhacYmBMxcmNX4PKGj6zBFJjYRdVNlQkvQWahLgdABngXvb\\\\\\\/PQJmAN5RJXU+oCPLQx19nxzia4UDWB2P1769Wie32JHMCvqHqG2E1qMCM5aMHUiKYoXgt1UQV\\\\\\\/0rTT1il\\\\\\\/sLQG6uGUNw+fBiRiiVTMlJqyFpU0PbGnkJJJf3f3VI7HbUu1Rw4eqwPFiHFbM8Uy3NuTJ6ItzAxcUyW\\\\\\\/vzJ9K3CRY6N8ezDlpSvYA4DrNWGWn1iIqpv0Lajhpb1ovbH8ekvnIXjM2IGTrsV3wV0AzlCRAenjX0l5Cka1OJ5DLNXlLEqdr52rpvCMAltJaPtjdmSNRQ\\\\\\\/4W2PJ4HhVPH22sOSXdE2WhpsCntwPgM10vaeElkoaTuR0GvtLW\\\\\\\/RlxmZw2SFnQm4lW\\\\\\\/5Tk3dhFLTbhwwMSCxXNGQsZiWzuq7\\\\\\\/CoGA6yam8X+UHxv2hAzmACkN8kEL5T60oCSyLZ8APzxeXk4c45K5u\\\\\\\/6bBB8Pwv4mTINu3qjjpzlRtHyldU8d+KhN3FMayGctYeF3khIaC5fteYKruSOuuP9JEW8cMpNsmuLADdrrFdDwRQFvKr0BXBW+qZrHNF8ZTDyMLS5lcMUjFYwK1Vci0XdpCfcN3HwTy7zkfN+0oxyTpCkBjEk40bxF209+sqJx6TcWbQs13gQdcdB2EGcJDdNT0pZ9gLIc6wf7m3eIPKFcWi0tib4WOgXefHSNfdD\\\\\\\/qu4XX3OaQiqMPwj1WpTpN\\\\\\\/02qzxJfFa0ihqKcq8icyilXHg2Gy173Bc3C9sZ4bNURjRTU\\\\\\\/5HKjYm79JzjYIeUlBi8\\\\\\\/AFZDAXDQ+pf7iFwBIAThtoj\\\\\\\/+Tsad3\\\\\\\/yK0tbwe73Z\\\\\\\/DTFcOlRJiO8SV\\\\\\\/wJ2ioILpwAhk5uFr0DIGuWcDPD23LkUG0BtXsz1U4b9T6txJn1P\\\\\\\/QTkJ3gFXJr13PRqTq4x5jJhC4nK0MVMsMehWBmeYW07wzmh+SEOsQGQNJ18iICPeAqLHoFnBuAPqW3ScnmwFTEMNHIri2C3HmqdFiG35FhTzc3dyv10GArQ7SI\\\\\\\/TmZ6Jx9ZagWUFDpBVxYrW\\\\\\\/PQq4cHTncPfjcd5q9of\\\\\\\/yeshyIOtOwLXpKKLwbX4958wOmcSYi6wYwu8oC1rfQJj\\\\\\\/4gSsmRRSjLpqTze5JEA3ptWQgZK1viSRPly4Ik2HS\\\\\\\/0tTuHyYlOJZw274EpcNFOZKnTOuJ7htHYOVaBYfrfsjHDmgJDp5EfkF3r+oSKe0QR1L2hAWe0obLAmMUs\\\\\\\/nRUJU3k4AdeICkOBuSOoyzhre7xNUtzU8Z+yh7RQRk+XMloMc+dWNhL56hHzaHicbXjCsYpH7te7jFkjiI1\\\\\\\/ipld12hxLfsSODfhEMFsmLH+SqfM73Sp8xAJHB3Bz+5HzuUFxNq8b2qUo1+sVX4TnKf+OUm\\\\\\\/MW1UE9qseEvxRdwNF1ntwUq+AjMlC8SBB96KzJWFOKMV1A10E5WnjCfUH9S97LfL8ySS12ZY4SxP+H6GWlVAPN2XJ54DjVfixOeaMqcJC1tYtmMMFSBCZyU6UpWEqOJnGlT1lHOBYe7llPTTUeCr8Dc+eB5AxQ4UxKT3uNTUSoI1lyuvd3dOQYOZ+GNjAqKZ1ZTZ0Ndaff+qCHZWBFBDOEyAVkLBAPEli2Tk0r7NzS08OMMWqJPWFG5\\\\\\\/8A5\\\\\\\/ZKW+gj2iTwU6jtLc\\\\\\\/GSU17K65E3htGgPSdIbsuY7apmxA2rHsQOC8LZolLLpxv4EIDMUtd1cm6BkeZgdY1FSRrX34wxxFgVn68jVIO4WtkQirQ5ap639vsKqyOikVryhQ+OJLk+1MvJIS1zY9RZcrwvjclmQshiT7IMt6G0SLAGJQGhOamga02KBi8XmNVq3nj3+v2KhPwPx2QFkbWuXpr6Tjl\\\\\\\/\\\\\\\/351WUH6NDX8IiBXq1ly8q+6gJ9gwjdMX2OyCblreiosHTYr7SbYILwteM4K6eC8wp7UFA2BZ7dGZb0F1P7gMWtdvcKA7e395UyBkz\\\\\\\/+Tk4pVGTxZQgBZF\\\\\\\/1Hu+Dwr\\\\\\\/Pdwucrmao39Mky1DpSnJKrkQ+CCdeszp+NOmg3AVj\\\\\\\/ps+FCaxaf11wE105vL1NnnP3pcXoT91gpVVYUuuZwjLzm6l+urxP\\\\\\\/5D7E3P2N4D9nzKRRiDv6HNtlFm+cdwGLqxzsH+gtFO+W4tdpQiPwqFXSGzmfJS5iqRd9sL6FmUTccQKY+aoczn4BxCWZh\\\\\\\/9MpNuG\\\\\\\/WM2B3I1zN08L6NjrNHyW3RRqjAdblFh6G0eHFYDPPcquxb+HJfkruPTWWxtWeIxnPOF79Wpe5qeSF2qMQFmUGjNTayabIBP77zVFG3hZbEKJcy8sEOtRzDHDLM6A8GqMLktQXNxb+6rByE7e62A6+myFIJpmDNurWXe+O2mLJP8a1JOUcvrN7mI1wQEG4rdbFHh78HAewy73jzErJoPKvWspDtF4rJ8mWWvjnk2w8\\\\\\\/OZL\\\\\\\/bJPVmfra8PqAVkI2Y5oWGMdseWZiaMT6TiduZr95loyA\\\\\\\/E28BMp817XJ4BmBVmy3pEPjL2NpPs10ZXWq5sMuXKDaxjbGCP9Uz8pTob46E\\\\\\\/F5e6pVV9Ati7Lm1MJdZn89hY\\\\\\\/IFe5rfY0sCkHr921mBZ22WiVuwwi9+92evK4O8MYKF5UJVXQFCxggLkkGnUgqOUc4ZiTZjNOIg8Y0T\\\\\\\/6275ZNAzrw4pZw0K2V5odosiyRweoknOyzR7C44kC6+WTnPmXRZkQyAR\\\\\\\/R4YpYYFIiJ7sPwpMFGrkniS955IGDQlNptnUobMCTXOzWBdidtJOlSPtDvTeR20OskyR6I6zBEUn2gY3nY2ktYq9UgVSp7fkySkA76QVFYsLwRl1peGns\\\\\\\/JZTWCJaiObw+BB7xkkMTFAKg48XGLAw8cOb8BYSNgHgwEDTmtLjbmruJtk5R0ReAPlloBFtW\\\\\\\/PeaRqU4cOvaA1g5mem4kRQo4\\\\\\\/n3u4kX\\\\\\\/KYHL5zPSGdeVSm+TYF5Gn1WwBtJ5Qhwks57SxGyhOPNi1wF+t\\\\\\\/2lIKRpo3wRttRomkOXLniXiZbGuQPvigdFlmi\\\\\\\/TCnQ5oCWjo0CsTY8WHug5iVyoWqXt6wU5TS2UZzShAjgQmb8Dnx2czdQJkeFHoiEXUUwDFrDToOZae3KmdlQcyzdG27NVTB0p2ENSxUiF2mxVIMMH\\\\\\\/AoiR5T6fkkh8TSFZDMxfKsCCFqhL9k0eFHSJokGIYk2VFI7q7JJTYc3\\\\\\\/uv48b8LehCNCevd01h4m7i4UpY\\\\\\\/VO9bWNZqWuNqobJ+rb7uZjqIytvRLsnCuuYcyBFKEc7RalgXOpLLLWkPQ6lR4yZYaAZLY\\\\\\\/ssyYu56dr\\\\\\\/wzGI7D\\\\\\\/r2PXqvtxYqYnNxd5cEmr8EyDR7ImcrFcsjojppWFf5vccXn7HDiSTXV4LkO0LtXjl\\\\\\\/qp5ZSOY6C\\\\\\\/yORvEL\\\\\\\/Spba8po6XMk7B7bMM\\\\\\\/Y4rUPv6kwUm9G72mTfmR9wWrHEhsqOb9XtnkdtKc7NtKzPCRezfHadO0F60uHqpqoO0cu\\\\\\\/R4QadcXBD\\\\\\\/LhC8MLQ7TSJRbaj40tDTPskUBMMCXUNWf8vEHCjUjeVXJfPW7UJR88UB7vF8jg+a0rFzA59JkGldoYcz2cizXnF\\\\\\\/QmxYI\\\\\\\/eUF7j5UBgiAs7itqr\\\\\\\/AameygIm3oExyRcPTHLTTeedjWRBoz1aQJrPG1zP6gi8qnPN\\\\\\\/bX6sA\\\\\\\/QccA2H4nsYnx55Gvt00EALKaATbFU1M0cE9Y2HnT\\\\\\\/0Q1eSEBsC6xEbDeRGL1\\\\\\\/EzJ65t8nBPki1U9bGTpyrJXz2aSenvetSKcNKZ7gZSrwPxpIJgAHaiSLcZ3jOjYnW3LpAKNbi3eYzsepAm+VNyV8PHb4zyi65Mbu58CHfsLBQS9qSC0ts0hVVF2pc7\\\\\\\/DAH5LGUiJTcUT2WrOK0uocBWXlX4a2ljNCbcp26Bx8+T7NTbP1vnDa8MsZaruRt+JbpWdN1d8XMUh+dwRZDe23BSUZaBkMy0ZkviNrzsEaEcSnkhFKkFlDqdqcaZA5rFEE6dN\\\\\\\/ga2DZSiap9b8oP6KFMqr+CkC0eho+3W+cGrHMu1KP72z8VkIAOywtFuR1MDigtnQDkZt\\\\\\\/YCamH3E3eRXL+YfskuuoRbIaFDGvgYJmtmZ\\\\\\\/32VerZHcQztpr3\\\\\\\/O0Xf+2HomogJGECE7GVv6YJPOyf\\\\\\\/FVKS6+vF0jQ6kS3rEJBVU4lwNdGjftl\\\\\\\/ayr+vSSTKTHlinc1aT68460APMHUAaBDn72qBnxNw14d\\\\\\\/1zYkST2GYbbMR54yOdok4hYP1+8hIGGRBM\\\\\\\/VG5RGgpZkeS0TqToKN7EKkhjXTn0sRlyZ4Cz5gICgcBkWisG7R8XxqRekF25LFse6T5Oth9bhAQfqH1cSvPtI2f\\\\\\\/4PVK2qxwyaaupfRmKk3Cmz4LffB0XtYEQ9wbimFsQV6QxI\\\\\\\/sTUHc5D5lBszgDwcw7YqdxsuCa2BnVwrKuo8MZUEJbiq590X9QZH84tmsagWas49FWEo90QDtK8Th\\\\\\\/+jFeis5olNMHV1CdOWPu3htglKx+gV\\\\\\\/+YMSCpQnkj38umXFQo4Rl+YHHZKv3lj\\\\\\\/TOQs5GMiQARPJp0FUro\\\\\\\/mBGeL+3v2gxPM7Ev8FwUXYueBbWD\\\\\\\/JWbUNgq8qHolcLL0s4MB1pJQjWavm5ScxFaz4D5YgOU\\\\\\\/HlxguDF11SB5nKsCJ2\\\\\\\/GlcZ8vf6LObzU4yPlJN\\\\\\\/uNNCmYWFDeKOWp8i890zMgBpgETM\\\\\\\/V7EWzLA\\\\\\\/iTOD3Qq6kPvtki93X+QJwVJs1vKs5I97iyTzX7+Zg5iEKk4+O6RszFO8ZFYoGYh1BWgDkXtVSUr4K6OWmPv\\\\\\\/XbIXSbxQ55TPE0oEOu7ILwjY\\\\\\\/G4t066jBX3R81cN1J2UxFqDr0o7BfXVZ8EML4gWjhwekXwXyWz\\\\\\\/nPJxmbrDBKwzBpt7P\\\\\\\/dAU9TQOOXirCVqek+NuNBjqw0sd98ba\\\\\\\/1FEO9FifnFSwpGUy576MIQshqSnPMslLyh9EQUf4Jh9IGwpH6cExl3FPu\\\\\\\/IJsJWDUy1qRsL1F0lVKCH3QB5Dslr8Z2UCcEo0dHDPuZ\\\\\\\/usBz9Oe7UzgM+V8wDOWpNLIGbfA1Dc+A\\\\\\\/p9MIVKplWWBgiFmNDVAU8nzHFFIqkrmoIHIrNQ5G3wsdxJxec3MRjOUHVisrHMQ72RzdvzaHPqNMqQHSpZky1S7WDciI836z1aa2ar5lGuUF7Gb3mjVNQPUbFdhLNQVP2Hhy5ENvMv\\\\\\\/4W3+cT\\\\\\\/M8NyaeXpssMgnLkoBWvs\\\\\\\/C49swriXWBPw3llPwE+iQCVE7+1IQce0df+tGO9ZFmVRQDKhQzCIoDM7DixglwIU37Hb3BZgQVuYrsykEMN4Uy5ZXTman36Rksyqb5r4psaWQUqILe2OPFHx9M5f9\\\\\\\/mhHEuObulLxwtwS7c6bgLEPXX3JEmoQWSj9WLvkQLB7gnp3ov+\\\\\\\/7lE9tDsBtsXhZqm\\\\\\\/4fAHjsTbi+LNNY68wI4cxpBtb7UOGn5YsXyRYHHUAHTzpPMvtdKHmTn5g+0\\\\\\\/mLqDV\\\\\\\/CCbu2gh9U9wMTJY3pqIyTvIAFfZgeZ\\\\\\\/ub9oj8f7l6EV6GvGJ09PNwkKPTTHZnTQjl7QP4\\\\\\\/\\\\\\\/8uz22VMTkcKNn2HDvVtNKeTZ4XGmT4764CpvSlDmKlvS3uS8M7Ke4741XbbE35YHthuPjZhSNuTnZtU+WovdbWcvy\\\\\\\/m7nMDHPbiDZuIC0xW\\\\\\\/gNXzQQbP27v1VYQ5KeWCVlBKGqo37BaBk0yAozmoYKDj53kkH0HD9wjRhReprzkxl4eGkvIZwcAECp9BiSgXySdLsBENtzZBJrfLx+\\\\\\\/g0yAPKUxZw1GQQDjjsb1y+J8rkNqQZ+dkUbXM+Ow4MbJJTuL3AJ0X6Ua4Du5P4YP0Ls5fqTfo4LjirsoLHe0b8YHCYKakvElmjO2nn4T\\\\\\\/V8Di0Uz5PQNRoYTOkCP4W8YQzo9fCgvNKVc0SkA1\\\\\\\/k3lHle8P13uj86Qym\\\\\\\/XDYA6LrAFrtcnOuGpQwJwpSUAcDQfLy0iG7NDmI8pd3J6sTvUVUVLzG6maCTIreOP3isHtJbPEOopx2AGNXCHpP3cmOr4ON25vA1gFEao7h8s9\\\\\\\/KGVnb1c6WXXmwMIO\\\\\\\/VWa9pjFQx5BFXkK5wWtvjYLXdDhQEChxJw8YDcZbe9Xk\\\\\\\/JJ81JC9R8w1YaMUWSG62ILzUrRB+2qFKrHcA94RDHvMyfnHdwPqljM4jpYgDYE3HAw2OlUdgwojVnSIMZxQsQ1RoxijDHqpsITD608k0N2TBOqeeIgbQsLMYKsBjqKp3kzhDJ1eEVZOXlvCoN7o4262pWciCsrMO7urAXyYVT1C4+F19+IzNhLnuCu31UIkQ0\\\\\\\/21zda6kYKxOfh6FcshEe7Ohs6rMFeEpkVKL\\\\\\\/6G\\\\\\\/mGmEzau0q8CtHoQeFe2rKQBqGL+icgMBju\\\\\\\/PRW1Jx6vO3CNUeZyQXJGd06D33qWtViv5v8i2TTx+8iILlvxf+i0vdCGc7mmBfQDZYnqhcCBvhT9J7Mpj4F5uBCXbg+x8qH0kNEFQde0nWa\\\\\\\/VZFDYNizSLt\\\\\\\/EMG2LPl8KT7h0l0CxTqwBFGspJSac5qkaTTZ7hqgpGxsLT0s0QvurL7OmlHNkX7dnXMtiaZLAHiV6Ri08nR3kunkkEM7VPwFjckqONL3Rx8PX\\\\\\\/vzhCCHHmOAPjWqGyIIKqSCowbdW0hJFKwwLWVE\\\\\\\/YVCmI5mrknNGmX6vC3xSHd9zkB2hbr45C5XH3gc0xL4Xoh4bHNEAdDfewrPwRhToNVRXvdy2uh7LV2SCP3fclhDorGXrvSq01uLU2ABvtiYjzFH0PJsj\\\\\\\/xI\\\\\\\/PWumuE3I\\\\\\\/LcuF4QQyKawHRlOVqv3hDlLBWPLxXHJ1dV+Hnl4+z6CbmGbce9Au3hD5HBrYlC94z+qR9FNi4REnexaOnX3gySKSQt+\\\\\\\/8z0aBRD4Fy0Un+1murrUod5jSfc0JRau4tL6omGyIgzX+JtgbAJLmNCm1tog4qWApiTm8MKKD27OSpSMPKEWX\\\\\\\/RgJ8VpvSuVgWJarXLQHwVIvr40sLnxK1KD27+KWYao1Bdde3UVJZVfdN50wLiaMyzKAvAOTa2qt0K8jSU4jdm1KOnWq2ggC\\\\\\\/29yrHMNZnaUXNbc7amML6J4hXJMdyRTiI23bVPLQF0N7Iqt31NYiNPUITNR7Hd\\\\\\\/ZhUmqvJmIIoGzpe9AKp4tfK1FPqDXe\\\\\\\/O85oba0sg2P4+ZwbTLqdMHJqA\\\\\\\/+PrdFtcCxdXOzk7FBzVBN4Poara1TnWAJfEjxNjm5PJ1mPFreyoWQhdLmD+0c+1ZwfrSDAhovbYdrwUS89yqAGNCtsmWfwAh8lDd5HfkUMVuATL9S\\\\\\\/SZnQvB9LQBXIpo3JhbkTKkkNrnvQLMA\\\\\\\/sYcdTo+ARt51IIPBUtDtdCCWhhjQYX9KRoz0quM7V18DK3UU\\\\\\\/1u6ZFzG4mOnF1u99lhNliR7KROga8+PzS6IZjeEBlbbPXZlSFsgWP5d2zNtxuxjQCmtmv8twy2oKeg8jEmTkvw1rP+QB9q\\\\\\\/g9+PAXjkJazyPqSDWufNvp7Fu3Bo4x3gYabkZbBVK\\\\\\\/LvgRwNYjTu9LNT7G5q1hb5s8EXONvgF1eHenNcZhYNycmlJ9Cbo8IhyWfNdCUMwrd3zMqmbG04CrnJD6jGhiqxzCRtBhgA61DD5lVrizi8iB3FxzRHAWW64W7oEGSd6ZhVGgq8ClBDoZu1hThYttWhUgGZ5ivGSf5ORKYKbC6XEwe5ID+NPuXT8q3OsJvppD96Mm2MJx913yoViZXS3pLfkn4+6TZjmbEqVwSFI1eciSeixNNKbh1t3wbmY+JpPBSoFuhaeIuYX7ez2V5a3UT6Zxg9RmJjm6iGWlXWWRpXqt\\\\\\\/9fpqxkaSCuCuL\\\\\\\/JjRMCossTxhl9qGeGGN9uUZsUaKFiQeiAPLmiD9Giy73SJMpSDhPu5jXft8BLcnB+RNUyT93xm2VwbUYxem2UEPgdQpN8rie8V\\\\\\\/ld\\\\\\\/5kKZWdGBRgpcgJvKJ\\\\\\\/hw5VDq5NpfnLTuxzjsfZml64Q1Xuspa8QSDmVggKKpr+UN5C8YkPU\\\\\\\/6J0RDYXKE5RqXphcyKMgGd2S+SqgTy2GttMn\\\\\\\/Cv\\\\\\\/5i4hqphF0sjXlFQ2ymJD3Jvb7AmLwBW2MaCgiaMdk4Ez9wHHZ4QH0P6BtTpV7NgYAW7yw9JASMnSkTe8pd2sm5q5MoGVd4hHjoDy\\\\\\\/nzTepQ\\\\\\\/4FMordYa7cBB\\\\\\\/paCCLSmMgo1wmLoBxGeDIvglNoEV8\\\\\\\/9R0Ms6yzlxv43hL6nif5RD8HcUAHk84z7JPdjoL6StHmyUlrQ2qr\\\\\\\/l2m78Yg+fUwolZinBWU\\\\\\\/RhoEWTsytBAVQGTUcPgGP6RLX0TbQQPFVSUvyVqs\\\\\\\/YPXHyuxiOj4s63CHprGIl26JtyGQcpUOk4CQrqvzct1SCQ\\\\\\\/UcezOaqokff3aAPUq16gkVKEz1G8PI4ksPfSjTeAH6Kl6r+5JzAnk0dz+ZSMwtv8+NbeSCYkSKyPj1bTGuWajrzlX+vWu52Z1sfn10WkQRHRoNbAqgQeiVRlZTankF+gAP2+nOnOOH0VGKbeqH2qmHmQr6YLGFDZjB+ghthNwref+1zE4tj2OgdQhHSmQseqNicpxvWOF644aqRa0afdMqI51Yoz+u2c\\\\\\\/LOYN6FnHbNtcsqx+kRYL39TSZQTiIwgnSCy5MZ5QpWiYzzwxH0wO7G\\\\\\\/NBYihrwFcirse7mdswzYv1KBGf5lCeANvw7FYcC5GIPmMNaXIDS4Qpcj\\\\\\\/fg4sJrexOyaLnVGEKWim76XYpP0zQQxjUC6qG\\\\\\\/71gtX3BZBDJCUXx9yZLBUDbbM9x3e\\\\\\\/y4xe94YmXzo1W2AV5Ll2\\\\\\\/E8AwMarHprKp4RLR5efw2CRv4zP7ZuiQOka1I31VX33nD6tuTENryTx+TUN6U9VKT3egxy6FZKmA5pHReepTMhF+Kr2XVHPYL3aMcBXExYvVOnxHDqJMfADjrsNLBpqYQYnV5eIpXkiUlpo+33O1QY4L9fPsrBVKr+70PlFHxFvK04EPZ6WOgZPFrsaAEPilzjYaVtj191aDVDzqbbg0WSmlRETi1SWPh\\\\\\\/wQPTwW+Y9BV4TBJOgB9JJNqQQehyEE+J0tx1sbbq4ZmlwzTZlbLzRrTCNo337mBWK7toA+ovNi6wl\\\\\\\/ssRfqYQ2Vj0kiHkWBvCEAhscU4JDlbpiXs1FgOzPOtmeJyhrn4gBiqn\\\\\\\/h9xksiIZWWsKHWfcNszGGGfGiqq\\\\\\\/7fDt03B0w9oLBeuYQ165BweKKPjXHl2P7EVdHuP8xopmKhH6emDeRlhYKm7A5p53HqU17m0+Jc+oUaZqQeseM6Dp4kHnNOQjKNkLsCCRHZDW1aIb5kQ7YXmz+O+SjE+H\\\\\\\/wRm2BwqeosCLyXV8QwS4RLNzaPTPIsod7RwG7U5C8\\\\\\\/rnRUWFVlI0tAgIgZeXvZ9yAXzHiJd2F9RbE0QDp6MRXGwUR00w7ihzySVB5aWDt3xUljvZlDc530l5nO9Q6W9ds+G1l6lyLrYt55OY4l2R8V1I+dybruOB8KVJS8mW2mm7w96Q4Pb+HG1X7p2bthDJ8cPCwAZ\\\\\\\/0xJkEdgGsB1jiHMyBECTsDp8W3xLvigSHYczQqgwpbVn2oJ8PBkE+dVU7jkjghfEFi4d1NlK7Ql2gS3fF6DAIVnCo5gKvYxCYgd3eCsVmSdspYabMJWGwA0H\\\\\\\/Y+0Ht3fHQLIuFlYv7u+SOXfAlkqDEf7Ss+cMHl9ErYLpz8ES\\\\\\\/70U3+CYRum0UNuemnheyywZVfXkQ5BnNpdSW4uDJ8i4d978ATVT0LExFuBBJK34kf0uuf6BCS4FW2FosFbIYjR6mfk06km\\\\\\\/RwS6QPzNVdIBzVm77FCyBrMIkuvWnHaDzLw1+KAnJB9IGXT39pEdqbkKJI3mKO7QN275+3jByyDNouV32krJR\\\\\\\/63c1xFpnJ1fJDrFjGaM+bUMIK4O8NPCZw4CEAPaErhGdpoDLiJE3GhWQqZHKGUUTvm2h7IVZNY2axnaWafIvPTVy26weE13F\\\\\\\/KYY30USHzchZ4\\\\\\\/K7fxpOEqx2MbKtbX8IFo4khVvpC+77HQmoqa7FDtO1g3kN+03hCcnnEHjQBoYHA4P3S6o502GNlKk4VgYxdsER+5qeDoK1tKcu7YCx86V027WY2SvxJOMX9r6vFU4idSlyvyCjdzbS0BfWCrSXBg9azPDT7adqqSEs+LxN0UCotqOPEIjuBlRbDOxXbnVIpccpkqoPMY6iI1AvsWp2VAu8+CwZloy3CrvqNIVPzNoNTVLIQowKfQOxl4Oxm8wQ4H\\\\\\\/tfbJmwrKUe6\\\\\\\/K6SiA+DQmz7K6s4UJnAei9K8G7GSowvZQUrO1BfEUXkQ5R+xUS7OyaRAKJmhEnlOKeR67omiRkkBHbUxuUq0sZzIIrhr9wjQGeX43ms1iWBxbCOiwwHcwSWsjHjKa5c1djUjhySQetWfb5l\\\\\\\/jk1a2cJOwH7oJXL4SGX8G578UX20z\\\\\\\/t5FNW8Gulax\\\\\\\/tr\\\\\\\/rRjfpoTWhCEm+vFhBbH9ncGgobm\\\\\\\/AClMa6+5cfG2143u8LS4Vo+DiYvOXf32x5em+TtPof+OxTMybUAB35nswYm6DJUFptuty3LZbYD9OPOC+14sS6JcSB5s\\\\\\\/c1jEoGb1qfLxwwyZ+z5M7rxPSXzV+jvYg0QxMJN9zUVnxVh+JjOiM5gKnR4hMLo5MhsJLWFtv9WKuKmz+owfZKQK8PXgKmyL+SB3uikZx50KtKVhnz+m1yuLapMq0bjBTFg2\\\\\\\/\\\u0022,\\\u0022iv\\\u0022:\\\u0022\\\\\\\/ioAACPrQmTi40tecu9vKw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00225a9130317653e99c41f5387c650c0cca3d0ab0c922a1eef125d513a40d69043d\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022kl1\\\\\\\/plPKR7z3kUaHl4nVhuOFNj2TpNDyhJYvuHkr8spmbxyz1You\\\\\\\/67bGa7bpgZo8Dcy9rvtROY6HGzWQUpA8ow9ikvQJVUznEAAUcGHHFbw7nlGmhUE7vdgN84M6KQc\\\\\\\/TjiNiF15Zgm\\\\\\\/S7y8FkrhHZqrFkfqWrbSvgloy3cEiTHCa9dlTLDOlXqcWa2+GZ8cJZVSmXGKLVmFesaW\\\\\\\/ocI68k4MsACvaGumc1fAIq5UA+6D0H71j4Wz0F2IBdAHICsE+vHbaVVh1UwK5eygvevGTQGgmSWoh+MXaKdE6qFgR2vT8jLB1JYxn3SoUT8bQbogNFJX3nMOWT7eJ84YnxJCg0QGMuWyeonLKV0XzLdVngH2EQClj8kpsPpcMOFHp4DS9pjrEx91qQSGjfcKPHiXAy0tRhdayIUNSOBTXNkn9S\\\\\\\/dmK1P3iZAs66hVK72zUGy3T0ggF9YdZnPnZz0dnywKE1batdaYmhu7BkivUfRwtiMie3MXzoL1pwRvhyXZRhRHidzjcAy0reyWO1blS\\\\\\\/U+\\\\\\\/q\\\\\\\/hYA\\\\\\\/YK0ssgjGWdlpyPpoCVfJA8Z3LnAXrOK+uE1XNTd09I2uP8YTS5nq4H2mA2OCw3tg13EOLoEfDoU4cXGFoXWXfJ8L6uUs9oUYz6zmi2EkJKH+F4LTLPcnRUTe4LNDMoQd8KsB9u8c4SbLKYoJrPNPu2C3iDnMeaQ6ptRGq5VuUq1lMpvIAqJz9i4FJn0UDqogoD5nIFdQ4vwloRAEfcMDLnF2UxY+57VzETY+lk3GDk8pT1aR0KGboz36blCJurkQbz+g2SllcmfQ6wQDgXQfF0Ns0qRwWNYSXuDPIDlFzyTkDVHxIZidp1s693U2nLTt7qM9haFY+fv2PbvIlK+OfScFZXr5QQhDAYtv9MIKW6oEqzSXWiTlaQVJhrfVwQqAQqZ8P5vWlfCgkRffOK7WwFclhcq1s30XvYlempQdmTqE2xE8CuR32ZjQbfGfUDva4S\\\\\\\/mzmb8eF8ZKCI7ct1R+1w42R2IUhPSyHHBFbLyEFfVfPz90x8kzOn9ieS6Nl0i+4WqM3PSoRojI4X3mflVtqoYYOQYbtudFyWUGVKt79MTnZ18jpiEvQcVWl2vvXP2PZ\\\\\\\/NFhttRZqHV80cPZ84BI45ViwuogBZPcy7SPF6GBytBOaEO\\\\\\\/pxnxucWwAlsFVcyAcrdawU\\\\\\\/OAiouIK1hwYjOnO6rUfZ87dnlienHHdLhEfg9GpeMidka8wasJ5zHAMHauJtpJvrFTjFNJe2dWQqzJ7UCLEXKUf+El0T3LrgsrE9EdvofoYgFYuX9HlvD2CpH0VwgZO5TUD9NQII44O\\\\\\\/UMzlsDGl6bslfd2M+Y9WtavIIzSwPCMWMCpVShlWzI6fyKCsPBAloQTMRTVQAtWDMtaq9NaPlQmzKaf8FEZ9naTFBOHfxT57AlWJXO\\\\\\\/wMtpfxRRn5dQVgNnYWWpouMDO3geQyALBOg3Cxc4sBXw6SebFDYdP8hU3VCXR4+fK48F3nR3MEYAPmevh6xoYCUQyZqhZwokPfVQn+NGiMNtk5v5G84wgQBhPfPW\\\\\\\/cFLqk+JKkfwXqCH7raoP2XktgEb+K7mH2KzEx\\\\\\\/F+kopbCiBTZtRALnXIFgHwV\\\\\\\/CZjZgfnYH8z25v2aETrkcjO2bLJ8TOxBSoWqnmylpVbXi15\\\\\\\/GoBBqLAiSWnKYE2wnVkvpBt3XC\\\\\\\/APHB4qVYB8mQODL22bsPPuvYueiMh68Ho9CzmbBfv7qAqFjrwAc9xxUIcwPbUbmYMN5RWd7AYXW8dHjscS\\\\\\\/3t0AmaPxYEQ2gjYK4FRIfNzeI0W0LXdZK5IkbYtjarl9nOnzwTKXejCLupC\\\\\\\/5pFoAtHAiP0+I15oWGh2AIAPJZUSkarZs9QCFv8zIevL+rGs3B90\\\\\\\/PzYCHxAv+B79HlfhpPZzXv3cVt5O973bXm44EyXaVfhKH6oZvS+9O0yRftfw6gW6ksiQQeVZ5LVx1QbgpqUbW\\\\\\\/Q2LsNvPaA\\\\\\\/SatRaUF32U7wTNo\\\\\\\/FPybB2hF01SybxvId11rc9U=\\\u0022,\\\u0022iv\\\u0022:\\\u0022ildf8qA0m9UDlF4rnkxVTg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00221f05d86d481104ae3ae827ef3ad6861e746b6b5043389da2db2640e021bc0d29\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459604 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/12/2163" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-ef9hlSDyk03jWQzlJszw9DbtRNNY9D0azvaROkAgar0" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/12/2163" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-YjuFRs1rFXiLWWBQuk7BZHaBQJDXTDadONIuKLSFNQA" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/12/2151" title="Detecting Selection Using Time-Series Data of Allele Frequencies with Multiple Independent Reference Loci" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/12/2173" title="STAT3 Targets Suggest Mechanisms of Aggressive Tumorigenesis in Diffuse Large B-Cell Lymphoma" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459604" id="node459604" data-pisa="ggg;3/12/2163" data-pisa-master="ggg;g3.113.007492" data-apath="/ggg/3/12/2163.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic Sequence Diversity and Population Structure of <em>Saccharomyces cerevisiae</em> Assessed by RAD-seq</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Gareth A. Cromie</span>, <span class="highwire-citation-author" data-delta="1">Katie E. Hyma</span>, <span class="highwire-citation-author" data-delta="2">Catherine L. Ludlow</span>, <span class="highwire-citation-author" data-delta="3">Cecilia Garmendia-Torres</span>, <span class="highwire-citation-author" data-delta="4">Teresa L. Gilbert</span>, <span class="highwire-citation-author" data-delta="5">Patrick May</span>, <span class="highwire-citation-author" data-delta="6">Angela A. Huang</span>, <span class="highwire-citation-author" data-delta="7">Aimée M. Dudley</span> and <span class="highwire-citation-author" data-delta="8">Justin C. Fay</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2163-2171;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.007492 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459604" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Gareth A. Cromie </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute for Systems Biology, Seattle, Washington 98109</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gareth%2BA.%2BCromie%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cromie%20GA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGareth%2BA.%2BCromie%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Katie E. Hyma </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Bioinformatics Facility (CBSU), Institute for Biotechnology, Cornell University, Ithaca, New York 14850</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Katie%2BE.%2BHyma%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hyma%20KE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKatie%2BE.%2BHyma%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Catherine L. Ludlow </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute for Systems Biology, Seattle, Washington 98109</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Catherine%2BL.%2BLudlow%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ludlow%20CL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACatherine%2BL.%2BLudlow%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Cecilia Garmendia-Torres </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute for Systems Biology, Seattle, Washington 98109</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Cecilia%2BGarmendia-Torres%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Garmendia-Torres%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACecilia%2BGarmendia-Torres%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Teresa L. Gilbert </div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Teresa%2BL.%2BGilbert%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gilbert%20TL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATeresa%2BL.%2BGilbert%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Patrick May </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute for Systems Biology, Seattle, Washington 98109</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Luxembourg Centre for Systems Biomedicine, University of Luxembourg, L-4362 Esch-sur-Alzette, Luxembourg</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Patrick%2BMay%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=May%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APatrick%2BMay%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Angela A. Huang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>University of Pennsylvania, Philadelphia, Pennsylvania 19104</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Angela%2BA.%2BHuang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Huang%20AA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAngela%2BA.%2BHuang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Aimée M. Dudley </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute for Systems Biology, Seattle, Washington 98109</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aim%C3%A9e%2BM.%2BDudley%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dudley%20AM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAim%25C3%25A9e%2BM.%2BDudley%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:aimee.dudley@gmail.com" class="" data-icon-position="" data-hide-link-title="0">aimee.dudley@gmail.com</a>
<a href="mailto:jfay@genetics.wustl.edu" class="" data-icon-position="" data-hide-link-title="0">jfay@genetics.wustl.edu</a></span></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Justin C. Fay </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Genetics, Washington University, St. Louis, Missouri 63108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Justin%2BC.%2BFay%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fay%20JC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJustin%2BC.%2BFay%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:aimee.dudley@gmail.com" class="" data-icon-position="" data-hide-link-title="0">aimee.dudley@gmail.com</a>
<a href="mailto:jfay@genetics.wustl.edu" class="" data-icon-position="" data-hide-link-title="0">jfay@genetics.wustl.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/3/12/2163" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459604" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459604/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/12/2163.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459604" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459604/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/12/2163.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459604" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459604/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/12/2163.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459604" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459604/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/3/12/2163.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459604" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459604/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-5">The budding yeast <em>Saccharomyces cerevisiae</em> is important for human food production and as a model organism for biological research. The genetic diversity contained in the global population of yeast strains represents a valuable resource for a number of fields, including genetics, bioengineering, and studies of evolution and population structure. Here, we apply a multiplexed, reduced genome sequencing strategy (restriction site−associated sequencing or RAD-seq) to genotype a large collection of <em>S. cerevisiae</em> strains isolated from a wide range of geographical locations and environmental niches. The method permits the sequencing of the same 1% of all genomes, producing a multiple sequence alignment of 116,880 bases across 262 strains. We find diversity among these strains is principally organized by geography, with European, North American, Asian, and African/S. E. Asian populations defining the major axes of genetic variation. At a finer scale, small groups of strains from cacao, olives, and sake are defined by unique variants not present in other strains. One population, containing strains from a variety of fermentations, exhibits high levels of heterozygosity and a mixture of alleles from European and Asian populations, indicating an admixed origin for this group. We propose a model of geographic differentiation followed by human-associated admixture, primarily between European and Asian populations and more recently between European and North American populations. The large collection of genotyped yeast strains characterized here will provide a useful resource for the broad community of yeast researchers.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/rad-seq" class="hw-term hw-article-keyword hw-article-keyword-rad-seq" rel="nofollow">RAD-seq</a></li><li class="kwd"><a href="/keyword/yeast" class="hw-term hw-article-keyword hw-article-keyword-yeast" rel="nofollow">yeast</a></li><li class="kwd"><a href="/keyword/phylogenetics" class="hw-term hw-article-keyword hw-article-keyword-phylogenetics" rel="nofollow">phylogenetics</a></li><li class="kwd"><a href="/keyword/population-structure" class="hw-term hw-article-keyword hw-article-keyword-population-structure" rel="nofollow">population structure</a></li><li class="kwd"><a href="/keyword/genetic-diversity" class="hw-term hw-article-keyword hw-article-keyword-genetic-diversity" rel="nofollow">genetic diversity</a></li></ul><p id="p-6">The budding yeast <em>Saccharomyces cerevisiae</em> has been used by humans for thousands of years to produce food and drink products that rely on fermentation, such as bread, beer, sake, and wine. In recent decades, <em>S. cerevisiae</em> has also proved to be a powerful model organism for the study of eukaryotic biology. Many cellular processes are highly conserved from yeast to higher eukaryotes, and the ease of propagating and manipulating this simple single-celled organism has led to its use in a wide variety of research areas. Yeast is a particularly powerful model system for genetics. <em>S. cerevisiae</em> was the first eukaryote to have its genome fully sequenced (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Goffeau <em>et al.</em> 1996</a>), and nearly 85% of yeast genes have functions that are characterized to some extent, a much greater fraction than any other eukaryotic organism. <em>S. cerevisiae</em> also has been used to develop many of the modern high-throughput tools to study the genome, transcriptome and proteome (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Botstein and Fink 2011</a>). Together, these advantages have positioned <em>S. cerevisiae</em> as the eukaryotic organism that we have come closest to understanding at a global or systems level.</p><p id="p-7">Research using <em>S. cerevisiae</em> traditionally has focused on a small number of well-studied laboratory strains. However, in recent years, interest in natural isolates has increased as it has become clear that many nonlaboratory strains (including those adapted to various food/industrial processes) have properties absent from the laboratory strains, such as the ability of several wine strains to ferment xylose (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Wenger <em>et al.</em> 2010</a>). The wider population of yeast strains represents a deep pool of naturally occurring sequence variation that has been leveraged to investigate the genetic architecture of polygenic traits (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Swinnen <em>et al.</em> 2012</a>). In addition, the polymorphisms that are observed in the global yeast population have been acted upon by evolution, making this set of sequences a powerful tool for investigating protein and regulatory sequence function as well as evolution (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Nieduszynski and Liti 2011</a>). Understanding the genetic diversity of yeast is therefore relevant to both the food/industrial roles of yeast and its role as a model organism in scientific research.</p><p id="p-8">The question of the global population structure of <em>S. cerevisiae</em> is itself an ongoing topic of research. In several publications in the past few years, investigators have explored the genetic diversity and population structure of yeast by using techniques such as multigene sequencing (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Fay and Benavides 2005</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Aa <em>et al.</em> 2006</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Ramazzotti <em>et al.</em> 2012</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Stefanini <em>et al.</em> 2012</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Wang <em>et al.</em> 2012</a>), whole-genome sequencing (WGS; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Liti <em>et al.</em> 2009</a>), tiling array hybridization (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Schacherer <em>et al.</em> 2009</a>), and microsatellite comparisons (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Legras <em>et al.</em> 2005</a>, <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">2007</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Ezov <em>et al.</em> 2006</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Goddard <em>et al.</em> 2010</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Schuller <em>et al.</em> 2012</a>). These studies demonstrated that <em>S. cerevisiae</em> is not a purely domesticated organism but can be isolated from a variety of natural environments around the globe. Although there appears to be some clustering of yeast genotypes by geography (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Liti <em>et al.</em> 2009</a>), it also appears that yeast involved in particular human food-industrial processes often are genetically similar to one another. For example, wine strains isolated from around the world display a very high degree of sequence similarity (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Fay and Benavides 2005</a>; <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Legras <em>et al.</em> 2007</a>; <a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">Liti <em>et al.</em> 2009</a>; <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Schacherer <em>et al.</em> 2009</a>). Unfortunately, several of the most diverged groups identified in these studies were represented by relatively small numbers of strains, suggesting that analysis of additional strains might help clarify the structure of global yeast diversity.</p><p id="p-9">WGS of a large, diverse set of individuals is the most comprehensive approach to exploring the population structure and genetic diversity of an organism. However, despite the decreasing costs of DNA sequencing, complete genome sequencing of several hundred yeast strains is still a significant expense. In contrast, methods that compare strains by genotyping relatively small numbers of loci, such as microsatellites or a small number of genes (<a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">Fay and Benavides 2005</a>), are less expensive, but the results may not reflect the relationships between strains genome-wide. A genome reduction strategy referred to as restriction site−associated sequencing (RAD-seq; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Miller <em>et al.</em> 2007</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Baird <em>et al.</em> 2008</a>) directs sequence reads to genomic locations adjacent to particular restriction sites. However, because most restriction sites are common across strains of the same species, nearly the same subset of every genome is sequenced. Thus, RAD-seq permits the genotyping of a set of strains across a large number of positions scattered across the genome at modest cost.</p><p id="p-10">In this work, we apply a multiplexed RAD-seq reduced genome sequencing strategy to explore genetic diversity and population structure in <em>S. cerevisiae</em>. Using this approach we sequenced more than 200 strains over ~1% of the yeast genome. The strains include multiple representatives from six continents, 38 different countries, and were isolated from disparate sources, including fruits, insects, plants, soil, and a variety of human fermentations, such as ragi, togwa, cacao, and olives. From analysis of the resulting multiple alignment, we observed a clear geographical stratification of strains along with evidence of admixture between populations and human-associated strain dispersal.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><p id="p-11">The <em>S. cerevisiae</em> strains analyzed in this study were obtained from a variety of sources, including the Phaff Yeast Culture Collection (<a href="http://phaffcollection.ucdavis.edu">http://phaffcollection.ucdavis.edu</a>), the Agricultural Research Service (NRRL) Culture Collection (<a href="http://nrrl.ncaur.usda.gov/">http://nrrl.ncaur.usda.gov/</a>), published strains from individual laboratories or our own isolates from wild or domesticated sources. Details, including references and information about strain requests, are included in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007492/-/DC1/007492SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007492/-/DC1/TableS1.xlsx">Table S1</a>. While analyzing the data, we came across a small number of anomalies, such as two dissimilar genome sequences for strain 322134S. These are likely to represent errors in strain labeling.</p><div id="sec-2" class="subsection"><h3>Yeast isolation</h3><p id="p-12">Soil, bark and leaves, or food samples were bathed in medium consisting of 2 g/L Yeast Nitrogen Base without Amino Acids (Difco, BD), 5 g/L ammonium sulfate, and 80 g/L glucose. Chloramphenicol (30 µg/mL) and carbenicillin (50 µg/mL) were added to the medium to suppress bacterial growth, and cultures were incubated at 30°. When necessary to suppress mold overgrowth, cultures were subcultured to liquid medium containing 1–5% ethanol. Cultures were examined by microscopy at 3 and 10 d, and those harboring budding yeast were plated onto CHROMagar Candida (DRG International, Inc.) and incubated at 30° for 3−5 d. CHROMagar Candida is a culture medium containing proprietary chromogenic substrates that can aid the identification of clinically important yeast (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Odds and Bernaerts 1994</a>). On CHROMagar Candida, <em>S. cerevisiae</em> colonies are known to range in hue from white to lavender to deep purple with most exhibiting the “purple” phenotype (C. L. Ludlow and A. M. Dudley, unpublished data; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Boekhout and Robert 2003</a>). Colonies exhibiting these color phenotypes were picked and saved for further study.</p></div><div id="sec-3" class="subsection"><h3>RAD-sequencing and alignment</h3><p id="p-13">A subset of strains were RAD-sequenced previously (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Hyma and Fay 2013</a>). For the rest, RAD-sequencing was performed as previously described (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Lorenz and Cohen 2012</a>; <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Hyma and Fay 2013</a>). In summary, yeast genomic DNA was extracted in 96-well format and fragmented by restriction enzyme digestion with <em>Mfe</em>I and <em>Mbo</em>I. P1 and P2 Adaptors were then ligated onto the fragments. The P1 adaptor contains the Illumina PCR Forward sequencing primer sequence followed by one of 48 unique 4-nucleotide barcodes and finally the <em>Mfe</em>I overhang sequence. The P2 adaptor contains the Illumina PCR Reverse primer sequence followed by the <em>Mbo</em>I overhang sequence. After ligation, the barcoded ligation products were pooled, concentrated, and size selected on agarose gels, with fragments from 150 to 500 bp extracted from the gel. Gel-extracted DNA was further pooled to multiplex 48 uniquely barcoded samples in one sequencing library. The multiplexed DNA library was then enriched with a polymerase chain reaction using Illumina PCR Forward and Reverse primers. Sequencing runs were performed on the Genome Analyzer IIx (Illumina) for 40 bp single-end reads, with one library of 48 multiplexed samples per flow cell lane, yielding 20−40 million reads. The read sequences generated for this study are available at the Sequence Read Archive under accession ERP003504, and for the subset of strains that were RAD-sequenced previously, DRYAD entry doi:10.5061/dryad.g5jj6.</p><p id="p-14">Multiple sequence alignments were generated by mapping reads to the S288c reference genome (chromosome accessions: <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=NC_001133.8&amp;atom=%2Fggg%2F3%2F12%2F2163.atom">NC_001133.8</a>, NC_001134.7, NC_001135.4, NC_001136.8, NC_001137.2, NC_001138.4, NC_001139.8, NC_001140.5, NC_001141.1, NC_001142.7, NC_001143.7, NC_001144.4, NC_001145.2, NC_001146.6, NC_001147.5, NC_001148.3) and generating consensus reduced-genome sequences for each strain. The tagged reads were split into strain-pools by their 4 base prefix barcodes. Reads with Ns or with Phred quality scores less than 20 in the barcode sequence were removed. Any reads with more than 2 Ns outside the barcode also were removed. Reads were aligned to the S288c reference using BWA (version 0.5.8; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Li and Durbin 2009</a>), with six or fewer mismatches tolerated. Samtools (version 0.1.8; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Li <em>et al.</em> 2009</a>) was then used to generate a pileup from the aligned reads using the “pileup” command and the “-c” parameter. Base calls were retained if they had a consensus quality greater than 20. Positions with root mean squared mapping qualities less than 15 and insertion/deletion polymorphisms were ignored. After filtering there was an average of 209,765 bp for each strain. Sequences from each strain were combined into a multiple sequence alignment via their common alignment to the S288c genome. Sites with more than 10% missing data were removed, resulting in a multiple sequence alignment of 116,880 bp.</p></div><div id="sec-4" class="subsection"><h3>WGS alignment</h3><p id="p-15">Previously generated WGS were incorporated into the RAD-seq dataset for population genetic analysis. For genomes with an S288c NCBI coordinate system, sequences were extracted directly based on S288c reference coordinates. For genomes using an alternative coordinate system (Saccharomyces Genome Resequencing Project, SGRP), blat was used to convert from the S288c NCBI reference coordinates to the alternative coordinate system prior to extracting sequences. For assembled genomes without S288c alignments, coordinates were obtained by blast. A fasta file of the S288c reference sequence was generated for each contiguous segment in the multiple sequence alignment. The resulting files were used to query each genome assembly using blast. When quality scores were available, sites with sequence quality less than 20 were converted to “N” before blasting or after sequence retrieval.</p></div><div id="sec-5" class="subsection"><h3>Duplicated strains</h3><p id="p-16">Some strains were sequenced by both WGS and RAD-seq. For duplicate strains with pairwise divergence less than 0.0005 substitutions per site, excluding singleton alleles (<em>i.e.</em>, found in only 1 strain), only the RAD-seq data were retained for analysis. For duplicate strains that exceeded the threshold, both RAD-seq and WGS data were retained and strain names were labeled with an “r” and “g,” respectively. Differences between the WGS and RAD-seq data could be a result of: (1) sequencing/alignment errors, (2) different monosporic clones from an originally heterozygous isolate, or (3) mislabeled strains. However, we were not able to confidently distinguish between these possibilities.</p></div><div id="sec-6" class="subsection"><h3>Population analysis</h3><p id="p-17">Neighbor-joining phylogenetic tree construction was carried out using MEGA (version 5.0; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Tamura <em>et al.</em> 2011</a>), based on P-distance with pairwise deletion. Population structure was inferred using InStruct (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Gao <em>et al.</em> 2007</a>). Because InStruct failed to converge using all sites, it was instead run on 759 sites with allele frequency greater than or equal to 10%. Polymorphic sites were made biallelic by treating third alleles as missing data. InStruct was run with the parameters “-u 40000 -b 20000 -t 10 -c 10 -sl 0.95 -a 0 -g 1 -r 1000 -p 2 -v 2” with K (number of populations) ranging from 3 to 15. Although the lowest deviance information criterion (DIC) was obtained from a chain with K = 15, there was substantial variation among independent chains. We chose K = 9 as the optimal model to work with based on the average DIC for K = 10 being nearly identical to that of K = 9 and subsequent decreases in DIC for larger values of K being small compared with the standard deviation in DIC among chains (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007492/-/DC1/TableS3.xlsx">Table S3</a>). Consensus population assignments for K = 8, 9, and 10 were obtained for the five chains with the highest likelihood using CLUMPP (version 1.1.2; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Jakobsson and Rosenberg 2007</a>) with parameters “-m 3 -w 0 -s 2” and with greedy option = 2 and repeats = 10,000. The similarity among the five chains (H’) was 0.995 for K = 9, very close to the maximum similarity of 1.0. Compared with K = 9, populations 6 (African, S. E. Asia/Palm, Cocoa, Fruit) and 7 (Israel/Soil) were merged for K = 8, and a new population was inferred within populations 3 (Asian/Food, Drink) and 6 (African, S. E. Asia/Palm, Cacao, Fruit) for K = 10 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007492/-/DC1/FigureS1.pdf">Figure S1</a>). A second InStruct analysis was performed by the use of a pruned dataset to better conform to InStruct's assumption of independence among markers. We eliminated SNPs within 5 kb of another SNP based on the decline in r<sup>2</sup> as a function of distance between SNPs (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007492/-/DC1/FigureS2.pdf">Figure S2</a>). The pruned dataset contains 495 SNPs and an average distance between SNPs of 22.4 kb compared with 14.9 kb in the full set of 759 SNPs. In comparison with the full dataset, the pruned data also had an optimum of 9 populations but with more variance among runs as indicated by H’ (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007492/-/DC1/TableS3.xlsx">Table S3</a>). The similarity (H’) between the CLUMPP consensus of the full and pruned dataset was 0.90. Seven strains showed population admixture proportions that changed by more than 0.25 for any population. The seven strains are all Israeli strains in the Israeli population (#7) and showed an increase in admixture with European (#8) and Human (#4) populations in the pruned analysis. Most of the strains (211) showed no changes in admixture proportions greater than 0.125.</p><p id="p-18">Multidimensional scaling was performed on all 5868 sites and 262 strains using the identity by state distance between each pair of strains and the “cmdscale” function in R with three dimensions. Hierarchical clustering of either sites or strains was performed using the “hclust” function in R with complete linkage and the Euclidean distance of identity by state.</p></div></div><div class="section results" id="sec-7"><h2 class="">Results</h2><p id="p-19">In an effort to expand the number and diversity of characterized <em>S. cerevisiae</em> strains available to the yeast community, we assembled and characterized a collection of &gt;200 strains (<em>Materials and Methods</em> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007492/-/DC1/TableS1.xlsx">Table S1</a>). This strain set covers a diverse range of ecological niches and geographical locations, including strains used in previous studies of yeast global and local population structure (<a id="xref-ref-7-4" class="xref-bibr" href="#ref-7">Fay and Benavides 2005</a>; <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Ezov <em>et al.</em> 2006</a>; <a id="xref-ref-17-4" class="xref-bibr" href="#ref-17">Liti <em>et al.</em> 2009</a>; <a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">Schacherer <em>et al.</em> 2009</a>; <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Goddard <em>et al.</em> 2010</a>) and strains with published WGS data. We sequenced each of these strains using a RAD-seq strategy to produce an initial multiple alignment (<em>Materials and Methods</em>). Strains with published WGS data were then added to the alignment to facilitate comparison between the results generated using WGS and RAD-seq data (<em>Materials and Methods</em>). The final dataset contained 262 strains genotyped across 116,880 base positions, of which 5868 sites were polymorphic (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007492/-/DC1/FileS1.txt">File S1</a>).</p><div id="sec-8" class="subsection"><h3>Genetic relationship among strains</h3><p id="p-20">To visualize the phylogenetic relationships between the strains, we generated a neighbor-joining tree from the reduced genome multiple alignment (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> and <a href="http://www.g3journal.org/content/suppl/2013/10/10/g3.113.007492.DC1/FileS2.nwk.zip">File S2</a>). The tree agrees well with the geographic origins of the strains and, for the subset of strains in common, is also consistent with a previous study that used WGS (<a id="xref-ref-17-5" class="xref-bibr" href="#ref-17">Liti <em>et al.</em> 2009</a>). To more directly compare our results to those obtained using WGS, we constructed a phylogenetic tree for only the subset of strains (38) analyzed in both our study and the previous whole-genome analysis (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007492/-/DC1/FigureS3.pdf">Figure S3</a>). The structure of the resulting tree is very similar to that produced in the previous study (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1C</a> in <a id="xref-ref-17-6" class="xref-bibr" href="#ref-17">Liti <em>et al.</em> 2009</a>) and shows the same clustering of “Wine,” “West African,” “Malaysian,” “Sake,” and “North American” strains. Similarly, using our full dataset, these groups are found in clear and well-separated clusters on our tree (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>). We also identified a small isolated cluster of strains from Ghana involved in cacao fermentation and another discrete cluster of strains from the Philippines.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/12/2163/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Neighbor-joining tree of the 262 S. cerevisiae strains based on multiple alignments of 116,880 bases. Branch lengths are proportional to sequence divergence measured as P-distance. Scale bar indicates 5 polymorphisms/10 kb of sequence. Geographical and environmental clusters of strains are named and are indicated by black-outlined/gray-filled ovals. Colored ovals with numbering refer to strain populations identified in Figure 2. Seven strains widely used in the laboratory are labeled." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1570416845" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Neighbor-joining tree of the 262 &lt;em&gt;S. cerevisiae&lt;/em&gt; strains based on multiple alignments of 116,880 bases. Branch lengths are proportional to sequence divergence measured as P-distance. Scale bar indicates 5 polymorphisms/10 kb of sequence. Geographical and environmental clusters of strains are named and are indicated by black-outlined/gray-filled ovals. Colored ovals with numbering refer to strain populations identified in Figure 2. Seven strains widely used in the laboratory are labeled.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/12/2163/F1.medium.gif" width="432" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/3/12/2163/F1.medium.gif" width="432" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/12/2163/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/12/2163/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463679" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-21" class="first-child">Neighbor-joining tree of the 262 <em>S. cerevisiae</em> strains based on multiple alignments of 116,880 bases. Branch lengths are proportional to sequence divergence measured as P-distance. Scale bar indicates 5 polymorphisms/10 kb of sequence. Geographical and environmental clusters of strains are named and are indicated by black-outlined/gray-filled ovals. Colored ovals with numbering refer to strain populations identified in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>. Seven strains widely used in the laboratory are labeled.</p><div class="sb-div caption-clear"></div></div></div><p id="p-22">A clear exception to this geographical stratification is the dispersal of European/wine strains around the globe, a result that is also consistent with the previous study (<a id="xref-ref-17-7" class="xref-bibr" href="#ref-17">Liti <em>et al.</em> 2009</a>). We identified two clusters of strains that appear closely related to the European/wine cluster, one isolated from European olives and another consisting primarily of a collection of environmental isolates from New Zealand (<a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">Goddard <em>et al.</em> 2010</a>). Results for this second group are consistent with the hypothesis that the strains largely reflect a population brought to New Zealand as a consequence of European settlement. Together with the main “European/Wine” cluster, these two groups of strains appear to identify a “greater-European” region of the tree.</p><p id="p-23">Strains isolated from North America fell into two highly diverged regions of the tree. One set of strains (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a>, “North America Wild”) defines a cluster of strains almost universally isolated from North America (largely environmental samples from soil and vegetation). The second set is genetically similar to the European/wine strains, with strains scattered within the main European/wine cluster and related groups (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007492/-/DC1/TableS1.xlsx">Table S1</a>). There are also a small number of strains isolated from North American environments in the “New Zealand” cluster. As previously observed (<a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Hyma and Fay 2013</a>), North American strains isolated from even the same locale (<em>e.g.</em>, a single vineyard) split into subsets from both the North American Wild cluster and greater-European regions of the tree. These results are consistent with the assertion that in many locations across North America (particularly vineyards), a native population of yeast strains coexists sympatrically with a population introduced by European settlement (<a id="xref-ref-11-4" class="xref-bibr" href="#ref-11">Hyma and Fay 2013</a>).</p><p id="p-24">Another instance in which highly diverged strains were isolated from a single small geographical location is provided by the set of strains isolated from “Evolution Canyon”, a well-studied location in Mount Carmel National Park of Israel (<a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Ezov <em>et al.</em> 2006</a>). These strains fell into one large and two smaller clusters on the tree (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1</a>; Israel 1, Israel 2, and a third cluster within a diverse set of strains labeled “Mixed”). The genomic diversity of these strains is remarkable, given that they were collected within a few hundred meters of each other.</p></div><div id="sec-9" class="subsection"><h3>Strains widely used in the laboratory</h3><p id="p-25">Included in the multiple alignment and phylogenetic tree is a group of seven strains widely used in the laboratory (S288c, W303, RM11-1a, FL100, Sigma 1278b, SK1, Y55), several of which are known to be closely related (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Winzeler <em>et al.</em> 2003</a>). The strains SK1 and Y55 are closely related to the West African cluster while S288c, FL100 and W303 are related and close to the European/Wine cluster. The position of these strains on the tree agrees with two previous studies (<a id="xref-ref-17-8" class="xref-bibr" href="#ref-17">Liti <em>et al.</em> 2009</a>; <a id="xref-ref-23-4" class="xref-bibr" href="#ref-23">Schacherer <em>et al.</em> 2009</a>), both of which described the limited sequence diversity of the lab strains. For example, none of the commonly used lab strains are derived from certain major populations, including the Asian group and the North America Wild group (<a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1</a>). Together, these results suggest that the total sequence diversity of the yeast global population is poorly sampled by this set of strains in common laboratory use.</p><p id="p-26">To compare the total sequence diversity captured by the full set of 262 strains relative to that present in the subset of laboratory strains, we analyzed all alleles (defined as single base pair polymorphisms) that occurred in more than one strain. Alleles found in only one strain (singletons) were ignored to reduce the effect of sequencing errors, as were heterozygous calls. The results show a total of 3321 polymorphic loci with 6680 total alleles (3283 biallelic, 38 triallelic, and 0 tetraallelic positions). Only 1703 of these 6680 alleles were observed in the set of lab strains, and thus the set of strains assembled in our panel represents a significant increase (~4-fold) in sequence diversity over the set of laboratory strains.</p></div><div id="sec-10" class="subsection"><h3>Population structure</h3><p id="p-27">The infrequent sexual cycle of <em>S. cerevisiae</em>, combined with its high rate of self-mating, promotes the establishment of strong population structure and enables clonal expansion of admixed populations. To infer population structure and admixture between populations while accounting for selfing, we applied a Monte Carlo Markov chain algorithm, InStruct (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Gao <em>et al.</em> 2007</a>), to the 759 sites with an allele frequency of 10% or more. On the basis of the deviance information criterion, we inferred the most likely number of populations to be nine (<em>Materials and Methods</em>) and labeled each population by the most common geographic location and/or substrate from which the strains were originally isolated (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007492/-/DC1/TableS2.xlsx">Table S2</a>). The relevant genotypes of each strain along with their inferred population ancestry are shown in <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007492/-/DC1/TableS1.xlsx">Table S1</a>. The nine populations consist of two North American oak populations, an Asian food and drink population, a European wine and olive population, an African/S. E. Asian population, a New Zealand population, an Israeli population, and two populations associated with industrial/food processes. These populations match well with the major groupings seen on the phylogenetic tree, with the two North American populations identified by InStruct corresponding to the “North America Wild” grouping (<a id="xref-fig-1-8" class="xref-fig" href="#F1">Figure 1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007492/-/DC1/FigureS1.pdf">Figure S1</a>). It is notable that these two subdivisions do not reflect a clear geographic pattern within North America (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007492/-/DC1/TableS1.xlsx">Table S1</a>). The New Zealand population clearly shares many alleles with the European strains, but harbors a small number of sites that make it unique. One of the two human-associated groups contains the majority of laboratory strains, emphasizing the uneven sampling of yeast populations represented by the set of laboratory strains.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/12/2163/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Clustered genotypes with inferred population structure and membership. Sites were clustered by complete hierarchical clustering by use of the Euclidean distance of allele sharing (identity by state). Strains were grouped by population structure and memberships inferred using InStruct. Minor alleles are shown in red, heterozygous sites in yellow, common alleles in black, and missing data are gray. Populations are labeled by the most common source and/or geographic location from which they were originally isolated." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1570416845" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Clustered genotypes with inferred population structure and membership. Sites were clustered by complete hierarchical clustering by use of the Euclidean distance of allele sharing (identity by state). Strains were grouped by population structure and memberships inferred using InStruct. Minor alleles are shown in red, heterozygous sites in yellow, common alleles in black, and missing data are gray. Populations are labeled by the most common source and/or geographic location from which they were originally isolated.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/12/2163/F2.medium.gif" width="440" height="305"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/3/12/2163/F2.medium.gif" width="440" height="305"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/12/2163/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/12/2163/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463630" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-28" class="first-child">Clustered genotypes with inferred population structure and membership. Sites were clustered by complete hierarchical clustering by use of the Euclidean distance of allele sharing (identity by state). Strains were grouped by population structure and memberships inferred using InStruct. Minor alleles are shown in red, heterozygous sites in yellow, common alleles in black, and missing data are gray. Populations are labeled by the most common source and/or geographic location from which they were originally isolated.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>Admixture</h3><p id="p-29">For each population, strains were observed with high levels of ancestry to that population. However, 38% of strains showed appreciable levels of admixture, defined as less than 80% ancestry from a single population. To assess the overall coincidence of mixture between pairs of populations we tabulated the number of strains with at least 20% ancestry from each pair of populations (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). Most admixed strains involved the European, Asian or African populations. However, not all pairs of populations were equally likely to admix. Admixture was detected between the European population and the first North American (InStruct #1), but not the second North American (InStruct #2) populations. More generally, admixture with the two North American populations was largely restricted to the African and European populations or to admixture between the two populations themselves. Like the European population, the Asian population showed admixture with most other groups. The two human-associated populations were largely admixed with either the Asian or European populations. Finally, the New Zealand population only admixed with the European population, and the Israeli population was largely admixed with the Asian and one of the human-associated populations.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/12/2163/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Coincidence of admixture between pairs of populations. Each bar shows the number of strains with at least 20% ancestry from a reference population (bar labels) and 20% ancestry with another population (indicated by color in the legend). For comparison, gray filled circles show the number of strains with more than 80% ancestry from each population." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1570416845" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Coincidence of admixture between pairs of populations. Each bar shows the number of strains with at least 20% ancestry from a reference population (bar labels) and 20% ancestry with another population (indicated by color in the legend). For comparison, gray filled circles show the number of strains with more than 80% ancestry from each population.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/12/2163/F3.medium.gif" width="440" height="271"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/3/12/2163/F3.medium.gif" width="440" height="271"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/12/2163/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/12/2163/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463724" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-30" class="first-child">Coincidence of admixture between pairs of populations. Each bar shows the number of strains with at least 20% ancestry from a reference population (bar labels) and 20% ancestry with another population (indicated by color in the legend). For comparison, gray filled circles show the number of strains with more than 80% ancestry from each population.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h3>Heterozygosity</h3><p id="p-31">Matings within or between populations can result in strains with a large proportion of heterozygous sites. Most strains in this study had zero or a relatively small number of such sites. These strains could be naturally occurring homozygotes, haploids, or converted to homozygous diploids, a standard practice in some laboratories. However, we did identify 65 strains with more than 20 heterozygous sites (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007492/-/DC1/TableS1.xlsx">Table S1</a>). The two strains with the greatest number of heterozygous sites DCM6 (n = 305) and DCM21 (n = 288) were isolated from cherry trees in North America and appear to be hybrids between the European and North American populations (<a id="xref-ref-11-5" class="xref-bibr" href="#ref-11">Hyma and Fay 2013</a>). Other strains with a large number of heterozygous sites (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007492/-/DC1/TableS1.xlsx">Table S1</a>) also were isolated from fruit-related sources, including three from cacao fermentations, one from banana fruit, one from fruit juice, and one from a spontaneous grape juice fermentation. Across these 65 strains, 42 also exhibit notable admixture, defined by less than 80% ancestry from a single population. The proportion of heterozygous strains exhibiting appreciable admixture (65%) is significantly greater (Fisher’s exact test, <em>P</em> = 1.5 × 10<sup>−4</sup>) than strains with little or no heterozygosity (38%), suggesting that heterozygosity was derived in part by admixture between populations. The proportion of admixture in heterozygous strains (71%) compared with strains with little or no heterozygosity (27%) is even more significant if strains with WGS are removed. Among the heterozygous strains, the greatest proportion of ancestry comes from one of the human-associated populations (#4, 31%), followed by the European (20%), Asian (17%) and African (14%) populations. To examine rates of heterozygosity across populations, we compared expected to observed heterozygosity within each population (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007492/-/DC1/TableS1.xlsx">Table S1</a>). Although most populations exhibit a deficit of observed, compared to expected heterozygosity, the two human-associated populations show noticeably more heterozygosity than the other populations.</p></div><div id="sec-13" class="subsection"><h3>Relatedness between populations</h3><p id="p-32">Whereas heterozygosity and admixture can provide information about strain ancestry, relatedness between populations can provide information about the history of entire populations, some of which may themselves be derived from historical admixture events. To examine relatedness between populations, we applied multidimensional scaling (<em>Materials and Methods</em>) to the entire dataset (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). The first principal coordinate differentiates the European population from the other populations; the second principal coordinate distinguishes the two North American populations from the Asian population; and the third principal coordinate differentiates the African/S. E. Asian population from the others. The remaining populations and most of the admixed strains lie between these four major groups (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>). Consistent with their positions on the neighbor-joining tree (<a id="xref-fig-1-9" class="xref-fig" href="#F1">Figure 1</a>) and their genotypes (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a>), the New Zealand and Israeli populations are most closely related to the European population, and the two human-associated populations lie between the European and Asian populations. The results, combined with its high rates of heterozygosity, also suggest that the first human-associated population (population #4) appears to be a recently derived population originating from hybrids between the European and Asian populations.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/12/2163/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Relatedness among strains and the inferred populations to which they belong. The first and second principal coordinates (A) and the first and third principal coordinates (B) obtained from multidimensional scaling. Each circle shows a strain with color indicating the population contributing the largest proportion of ancestry and size indicating the proportion of ancestry from that population (see key). Circles ringed in black show strains with more than 20 heterozygous sites. The first, second, and third coordinates explain 29%, 9.3%, and 3.9% of variation among strains, respectively." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1570416845" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Relatedness among strains and the inferred populations to which they belong. The first and second principal coordinates (A) and the first and third principal coordinates (B) obtained from multidimensional scaling. Each circle shows a strain with color indicating the population contributing the largest proportion of ancestry and size indicating the proportion of ancestry from that population (see key). Circles ringed in black show strains with more than 20 heterozygous sites. The first, second, and third coordinates explain 29%, 9.3%, and 3.9% of variation among strains, respectively.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/12/2163/F4.medium.gif" width="440" height="206"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/3/12/2163/F4.medium.gif" width="440" height="206"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/12/2163/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/12/2163/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463734" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-33" class="first-child">Relatedness among strains and the inferred populations to which they belong. The first and second principal coordinates (A) and the first and third principal coordinates (B) obtained from multidimensional scaling. Each circle shows a strain with color indicating the population contributing the largest proportion of ancestry and size indicating the proportion of ancestry from that population (see key). Circles ringed in black show strains with more than 20 heterozygous sites. The first, second, and third coordinates explain 29%, 9.3%, and 3.9% of variation among strains, respectively.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>Subpopulations</h3><p id="p-34">Low frequency alleles (&lt;10%) can sometimes define subpopulations not captured by inference of population structure based on common alleles. Two-dimensional hierarchical clustering of the low frequency alleles identified a number of such subgroups (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007492/-/DC1/TableS1.xlsx">Table S1</a>). Whereas the average number of derived low frequency alleles shared between any two strains is 3.5, there are 86 strains that share at least 100 derived low frequency alleles with another strain. These subgroups include a previously described Malaysian/Bertram Palm population (<a id="xref-ref-17-9" class="xref-bibr" href="#ref-17">Liti <em>et al.</em> 2009</a>), but also groups of strains from Philippines/Nipa palm, togwa, olives, sake, and cacao. Although the number of strains in each group is small, the number of sites defining the groups is not. In support of these groups representing populations that have been isolated from other populations for some time, many of the rare variants that define these groups are not present in other strains but, in at least some cases, are variable within the subgroup. Interestingly, the subpopulations defined by the largest numbers of alleles are strains with primary membership to the African/S. E. Asian population, suggesting that there may be undiscovered subpopulations and diversity among strains of African or S.E. Asian origin.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/12/2163/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Subpopulations defined by clustering of low frequency alleles. Two-dimensional hierarchical clustering of low frequency sites and strains. InStruct assignments, from Figure 2, are shown on the left, clustered genotypes are shown in the middle, with minor alleles in red, heterozygous sites in yellow, common alleles in black, and missing data in gray. Selected subpopulations are labeled on the right." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1570416845" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Subpopulations defined by clustering of low frequency alleles. Two-dimensional hierarchical clustering of low frequency sites and strains. InStruct assignments, from Figure 2, are shown on the left, clustered genotypes are shown in the middle, with minor alleles in red, heterozygous sites in yellow, common alleles in black, and missing data in gray. Selected subpopulations are labeled on the right.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/12/2163/F5.medium.gif" width="440" height="235"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/3/12/2163/F5.medium.gif" width="440" height="235"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/12/2163/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/12/2163/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463656" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-35" class="first-child">Subpopulations defined by clustering of low frequency alleles. Two-dimensional hierarchical clustering of low frequency sites and strains. InStruct assignments, from <a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2</a>, are shown on the left, clustered genotypes are shown in the middle, with minor alleles in red, heterozygous sites in yellow, common alleles in black, and missing data in gray. Selected subpopulations are labeled on the right.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-15"><h2 class="">Discussion</h2><p id="p-36">Genetic variation within <em>S. cerevisiae</em> has been shaped by a complex history, influenced by human-associated dispersal and admixture. Understanding this history and the resulting patterns of diversity is important for capturing and harnessing its fermentative capabilities as well as for quantitative and population genetics research. In this study, we used a reduced genome sequencing strategy to characterize the genetic diversity among a global sample of 262 strains isolated from a wide range of ecological habitats and environmental substrates. Our findings indicate that the major axes of differentiation correspond to broad geographic regions. In addition to previously described populations and patterns of differentiation (<a id="xref-ref-7-5" class="xref-bibr" href="#ref-7">Fay and Benavides 2005</a>; <a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Legras <em>et al.</em> 2007</a>; <a id="xref-ref-17-10" class="xref-bibr" href="#ref-17">Liti <em>et al.</em> 2009</a>; <a id="xref-ref-23-5" class="xref-bibr" href="#ref-23">Schacherer <em>et al.</em> 2009</a>; <a id="xref-ref-9-4" class="xref-bibr" href="#ref-9">Goddard <em>et al.</em> 2010</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Warringer <em>et al.</em> 2011</a>), two new patterns indicative of human influence also emerge. First, we find a population represented by multiple human-associated strains that contains a mixture of European and Asian alleles. Second, we find human-associated subpopulations from togwa, olive, cacao, and sake fermentations that are defined by a unique set of variants not present elsewhere in the global population. While inferences of population structure can depend on sampling, indeed our analysis points to areas of uncertainty, the structure of <em>S. cerevisiae</em> described here is based on the largest collection of strains typed across the genome. This work also provides a foundation for studying the genetic underpinnings of complex traits, the origin and evolution of strains used by humans, and the relationships between such traits and population history.</p><div id="sec-16" class="subsection"><h3>Geographic differentiation</h3><p id="p-37">A major unanswered question in the study of yeast population structure has been the relative importance of geography <em>vs.</em> ecological niche. Although the strains in this study were isolated from many different ecological habitats, a number of lines of evidence suggest that the groups they form are defined better by geographic differentiation than by ecological niche. The two North American populations contain predominantly oak-associated strains, but they also contain strains from plants and insects. Similarly, the European population contains primarily vineyard-associated strains, but also contains a number of European soil and clinical isolates (<a id="xref-ref-17-11" class="xref-bibr" href="#ref-17">Liti <em>et al.</em> 2009</a>). The Asian population also includes strains isolated from multiple countries and several different habitats, including strains used in Sake fermentation and several strains isolated from food. The Asian population shares many alleles with the North American populations, but is genetically distinct and includes only a handful of strains from outside of Asia. What is less clear is how this Asian population is related to a number of diverged lineages represented by strains from primeval and secondary forests in China (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Wang <em>et al.</em> 2012</a>).</p><p id="p-38">In comparison with the European, North American and Asian populations, the African/S. E. Asian population is not as well defined. Most of the strains are inferred to have mixed ancestry, and the strains that are most representative of the population (&gt;80% ancestry) combine previously separated populations (<a id="xref-ref-17-12" class="xref-bibr" href="#ref-17">Liti <em>et al.</em> 2009</a>) of West Africa and Malaysia, two populations that are also separated on our tree (<a id="xref-fig-1-10" class="xref-fig" href="#F1">Figure 1</a>). Because the trees are consistent, the different results of the two population analyses could be a result of differences in the methods of analysis (<em>e.g.</em>, Structure <em>vs.</em> InStruct), the larger number of strains used in this study, or the larger number of sites used by Liti <em>et al.</em> (<a id="xref-ref-17-13" class="xref-bibr" href="#ref-17">Liti <em>et al.</em> 2009</a>).</p></div><div id="sec-17" class="subsection"><h3>Admixture</h3><p id="p-39">Evidence of admixture was seen in a large fraction of strains and in every population. Although admixture is most common among the European, African, and Asian populations (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>), the smaller number of admixed strains from the North America and New Zealand populations may represent the more recent establishment of European strains in these locations or may be related to the frequency of mating in the oak tree or soil environment. Some of the admixed strains also exhibit high rates of heterozygosity, indicating a relatively recent mating between strains with different ancestries. Interestingly, many of the heterozygous strains were isolated from fruits or orchards, an observation that is consistent with the isolation of admixed (mosaic) strains from fruits and orchards in China (<a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">Wang <em>et al.</em> 2012</a>).</p><p id="p-40">Because yeast can grow asexually, entire populations can arise as a consequence of even rare admixture events. The two human-associated populations bear a strong signature of an admixed origin as they carry alleles from both European and Asian populations and lie between these two groups in the principal coordinate analysis (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2</a> and <a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a>). Human-associated population #4 bears the additional signature of high rates of heterozygosity, implying relatively recent mating events in the origin of this group. In contrast, human-associated population #5 harbors fewer heterozygous strains, but also contains multiple laboratory strains (Sigma 1278b, FL100, W303, S288C, and FY4), some of which show mosaic patterns across their genome indicative of an admixed origin (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Winzeler <em>et al.</em> 2003</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Doniger <em>et al.</em> 2008</a>; <a id="xref-ref-17-14" class="xref-bibr" href="#ref-17">Liti <em>et al.</em> 2009</a>).</p><p id="p-41">The New Zealand and Israeli populations may also have an admixed origin. These two populations carry a large subset of the European alleles, similar to many of the admixed European strains, but also carry a small number of alleles present at high frequency in the North American or Asian populations. This pattern is consistent with New Zealand and Israeli populations being derived from an admixture event between the European and these other populations followed by clonal (or nearly clonal) expansion. However, the New Zealand and Israeli populations also carry a small number of alleles that are not present in either the North American or Asian populations (<a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2</a>). This raises the possibility that the New Zealand and Israeli populations were derived from admixture between the European and as yet undiscovered populations, or instead, rather than derived from an admixture event, that they represent lineages with roots in an ancestral European population (similar to the “Olive” grouping). The diversity of strains sampled from Evolution Canyon in Israeli is particularly notable. Of the 15 Israeli strains, seven define the nearly clonal Israeli population, three are assigned with 100% ancestry to the human-associated population #4, and four show comparable percentages of ancestry from the Asian, Israeli and human-associated (#5) populations.</p></div><div id="sec-18" class="subsection"><h3>Derived subpopulations</h3><p id="p-42">The use of common sites to infer population structure eliminates the detection of small populations defined by rare variants. With clustering based solely on rare variants, we identified a number of such subpopulations (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>). Although many of these groups were isolated from human-associated fermentations, the number of strains is too small to clearly indicate whether they are related by geographic or environmental origin. For example, the olive strain group contains isolates from Spanish olives imported to Seattle and one from olives in Spain. Yet, this group does not contain two strains isolated from the brine of olives from Mexico and one from an olive tree in California. The two North American groups contain strains from different states, and the togwa and cacao strains were each sampled from the same country. Although some of these subpopulations may be the result of recently expanded clones, several of them are defined by sites that are variable within the subpopulation. This latter observation points to the establishment of small groups that have remained isolated due to either geographic or ecological barriers to gene flow.</p></div><div id="sec-19" class="subsection"><h3>Prospects for future studies</h3><p id="p-43">As our understanding of <em>S. cerevisiae</em> population history increases, so does the need to incorporate such information into quantitative and population genetic studies. Our results highlight the complex relationships between strains and populations, but also characterize a set of strains and sequences that can be used by the community. Using WGS or a reduced genome sequencing strategy, such as the RAD-seq method used here, new strains can be readily placed in the context of global population structure. We anticipate that new genetic diversity will be discovered, particularly in Africa for which we found less certain relationships and a number of derived subpopulations. Our results may also prove useful to studies of existing strains, either by controlling for population history in genome-wide association studies or by aiding the selection of strains for linkage analysis. In both cases, strain choice is an important consideration as the results can depend on what variation is captured and the structure of this variation across strains. Although many quantitative genetic studies have been based on crosses with laboratory strains, our results underscore the presence of additional variation that is available beyond those strains. Finally, the global diversity and increased variation uncovered by our study highlight the potential for identifying novel properties which could prove valuable to the improvement of existing strains or the engineering of new strains for use in industrial fermentations.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-44">We thank Meridith Blackwell, Andreas Hellström, Eviatar Nevo, Mat Goddard, and Lene Jesperson for providing strains. We thank Eric Jeffery for help with the manuscript, Adrian Scott for help with the figures, and Scott Bloom for assistance with Illumina sequencing. A.M.D. was supported by a National Institutes of Health Genome Scholar/Faculty Transition Award (K22 HG002908) and a strategic partnership between ISB and the University of Luxembourg. J.C.F. is supported by a National Institutes of Health grant GM080669. A.A.H. was supported by the National Institutes of Health P50 GM076547/Center for Systems Biology.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007492/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007492/-/DC1</a></p></li><li class="fn" id="fn-4"><p id="p-45"><em>Communicating editor: F. P. Roth</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2013-06-27"><span class="received-label">Received </span>June 27, 2013.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2013-10-02"><span class="accepted-label">Accepted </span>October 2, 2013.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2013 Cromie <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-4">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-3.12.2163.1" data-doi="10.1111/j.1567-1364.2006.00059.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aa</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Townsend</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Adams</span>  <span class="cit-name-given-names">R. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">J. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Population structure and gene evolution in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">FEMS Yeast Res.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">702</span>–<span class="cit-lpage">715</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BYeast%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1567-1364.2006.00059.x%26rft_id%253Dinfo%253Apmid%252F16879422%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZmVtc3lyIjtzOjU6InJlc2lkIjtzOjc6IjYvNS83MDIiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8zLzEyLzIxNjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-3.12.2163.2" data-doi="10.1371/journal.pone.0003376"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baird</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Etter</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Atwood</span>  <span class="cit-name-given-names">T. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Currey</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shiver</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Rapid SNP discovery and genetic mapping using sequenced RAD markers.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">e3376</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DBaird%26rft.auinit1%253DN.%2BA.%26rft.volume%253D3%26rft.issue%253D10%26rft.spage%253De3376%26rft.epage%253De3376%26rft.atitle%253DRapid%2BSNP%2Bdiscovery%2Band%2Bgenetic%2Bmapping%2Busing%2Bsequenced%2BRAD%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0003376%26rft_id%253Dinfo%253Apmid%252F18852878%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0003376&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18852878&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F12%2F2163.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-book" id="cit-3.12.2163.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boekhout</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robert</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-source">Yeasts in Food</span>. <span class="cit-publ-name">Woodhead Publishing Ltd.</span>, <span class="cit-publ-loc">Cambridge, England</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-3.12.2163.4" data-doi="10.1534/genetics.111.130765"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Botstein</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fink</span>  <span class="cit-name-given-names">G. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Yeast: an experimental organism for 21st Century biology.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">189</span>: <span class="cit-fpage">695</span>–<span class="cit-lpage">704</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DBotstein%26rft.auinit1%253DD.%26rft.volume%253D189%26rft.issue%253D3%26rft.spage%253D695%26rft.epage%253D704%26rft.atitle%253DYeast%253A%2BAn%2BExperimental%2BOrganism%2Bfor%2B21st%2BCentury%2BBiology%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.130765%26rft_id%253Dinfo%253Apmid%252F22084421%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg5LzMvNjk1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMy8xMi8yMTYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-3.12.2163.5" data-doi="10.1371/journal.pgen.1000183"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Doniger</span>  <span class="cit-name-given-names">S. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">H. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Swain</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Corcuera</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A catalog of neutral and deleterious polymorphism in yeast.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e1000183</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DDoniger%26rft.auinit1%253DS.%2BW.%26rft.volume%253D4%26rft.issue%253D8%26rft.spage%253De1000183%26rft.epage%253De1000183%26rft.atitle%253DA%2Bcatalog%2Bof%2Bneutral%2Band%2Bdeleterious%2Bpolymorphism%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000183%26rft_id%253Dinfo%253Apmid%252F18769710%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000183&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18769710&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F12%2F2163.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-3.12.2163.6" data-doi="10.1534/genetics.106.062745"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ezov</span>  <span class="cit-name-given-names">T. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boger-Nadjar</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frenkel</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Katsperovski</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kemeny</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Molecular-genetic biodiversity in a natural population of the yeast <em>Saccharomyces cerevisiae</em> from “Evolution Canyon”: microsatellite polymorphism, ploidy and controversial sexual status.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">174</span>: <span class="cit-fpage">1455</span>–<span class="cit-lpage">1468</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.062745%26rft_id%253Dinfo%253Apmid%252F16980391%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3NC8zLzE0NTUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8zLzEyLzIxNjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-3.12.2163.7" data-doi="10.1371/journal.pgen.0010066"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fay</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Benavides</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Evidence for domesticated and wild populations of <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">66</span>–<span class="cit-lpage">71</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DFay%26rft.auinit1%253DJ.%2BC.%26rft.volume%253D1%26rft.issue%253D1%26rft.spage%253D66%26rft.epage%253D71%26rft.atitle%253DEvidence%2Bfor%2Bdomesticated%2Band%2Bwild%2Bpopulations%2Bof%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.0010066%26rft_id%253Dinfo%253Apmid%252F16103919%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.0010066&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16103919&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F12%2F2163.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000234295900008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-3.12.2163.8" data-doi="10.1534/genetics.107.072371"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williamson</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bustamante</span>  <span class="cit-name-given-names">C. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A Markov chain Monte Carlo approach for joint inference of population structure and inbreeding rates from multilocus genotype data.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">176</span>: <span class="cit-fpage">1635</span>–<span class="cit-lpage">1651</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.072371%26rft_id%253Dinfo%253Apmid%252F17483417%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Ni8zLzE2MzUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8zLzEyLzIxNjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-3.12.2163.9" data-doi="10.1111/j.1462-2920.2009.02035.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anfang</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gardner</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jun</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A distinct population of <em>Saccharomyces cerevisiae</em> in New Zealand: evidence for local dispersal by insects and human-aided global dispersal in oak barrels.</span> <abbr class="cit-jnl-abbrev">Environ. Microbiol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">63</span>–<span class="cit-lpage">73</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEnvironmental%2BMicrobiology%2B%2528Print%2529%26rft.stitle%253DEnvironmental%2BMicrobiology%2B%2528Print%2529%26rft.aulast%253DGoddard%26rft.auinit1%253DM.%2BR.%26rft.volume%253D12%26rft.issue%253D1%26rft.spage%253D63%26rft.epage%253D73%26rft.atitle%253DA%2Bdistinct%2Bpopulation%2Bof%2BSaccharomyces%2Bcerevisiae%2Bin%2BNew%2BZealand%253A%2Bevidence%2Bfor%2Blocal%2Bdispersal%2Bby%2Binsects%2Band%2Bhuman-aided%2Bglobal%2Bdispersal%2Bin%2Boak%2Bbarrels.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1462-2920.2009.02035.x%26rft_id%253Dinfo%253Apmid%252F19691498%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1462-2920.2009.02035.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19691498&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F12%2F2163.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000274234200006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-3.12.2163.10" data-doi="10.1126/science.274.5287.546"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goffeau</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barrell</span>  <span class="cit-name-given-names">B. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bussey</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dujon</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Life with 6000 genes.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">274</span>: <span class="cit-fpage">546</span>, <span class="cit-fpage">563</span>–<span class="cit-lpage">567</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DGoffeau%26rft.auinit1%253DA.%26rft.volume%253D274%26rft.issue%253D5287%26rft.spage%253D546%26rft.epage%253D567%26rft.atitle%253DLife%2Bwith%2B6000%2BGenes%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.274.5287.546%26rft_id%253Dinfo%253Apmid%252F8849441%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyNzQvNTI4Ny81NDYiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8zLzEyLzIxNjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-3.12.2163.11" data-doi="10.1111/mec.12155"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hyma</span>  <span class="cit-name-given-names">K. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fay</span>  <span class="cit-name-given-names">J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Mixing of vineyard and oak-tree ecotypes of <em>Saccharomyces cerevisiae</em> in North American vineyards.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">2917</span>–<span class="cit-lpage">2930</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D22%26rft.spage%253D2917%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12155%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12155&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000319675300006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-3.12.2163.12" data-doi="10.1093/bioinformatics/btm233"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jakobsson</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosenberg</span>  <span class="cit-name-given-names">N. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">CLUMPP: a cluster matching and permutation program for dealing with label switching and multimodality in analysis of population structure.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">1801</span>–<span class="cit-lpage">1806</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtm233%26rft_id%253Dinfo%253Apmid%252F17485429%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjMvMTQvMTgwMSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzMvMTIvMjE2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-3.12.2163.13" data-doi="10.1016/j.ijfoodmicro.2004.12.007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Legras</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruh</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Merdinoglu</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Karst</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Selection of hypervariable microsatellite loci for the characterization of <em>Saccharomyces cerevisiae</em> strains.</span> <abbr class="cit-jnl-abbrev">Int. J. Food Microbiol.</abbr> <span class="cit-vol">102</span>: <span class="cit-fpage">73</span>–<span class="cit-lpage">83</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInternational%2Bjournal%2Bof%2Bfood%2Bmicrobiology%26rft.stitle%253DInt%2BJ%2BFood%2BMicrobiol%26rft.aulast%253DLegras%26rft.auinit1%253DJ.%2BL.%26rft.volume%253D102%26rft.issue%253D1%26rft.spage%253D73%26rft.epage%253D83%26rft.atitle%253DSelection%2Bof%2Bhypervariable%2Bmicrosatellite%2Bloci%2Bfor%2Bthe%2Bcharacterization%2Bof%2BSaccharomyces%2Bcerevisiae%2Bstrains.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ijfoodmicro.2004.12.007%26rft_id%253Dinfo%253Apmid%252F15925004%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ijfoodmicro.2004.12.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15925004&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F12%2F2163.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-3.12.2163.14" data-doi="10.1111/j.1365-294X.2007.03266.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Legras</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Merdinoglu</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cornuet</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Karst</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Bread, beer and wine: <em>Saccharomyces cerevisiae</em> diversity reflects human history.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">2091</span>–<span class="cit-lpage">2102</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DLegras%26rft.auinit1%253DJ.%2BL.%26rft.volume%253D16%26rft.issue%253D10%26rft.spage%253D2091%26rft.epage%253D2102%26rft.atitle%253DBread%252C%2Bbeer%2Band%2Bwine%253A%2BSaccharomyces%2Bcerevisiae%2Bdiversity%2Breflects%2Bhuman%2Bhistory.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2007.03266.x%26rft_id%253Dinfo%253Apmid%252F17498234%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2007.03266.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17498234&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F12%2F2163.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000246366800009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-3.12.2163.15" data-doi="10.1093/bioinformatics/btp324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fast and accurate short read alignment with Burrows-Wheeler transform.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1754</span>–<span class="cit-lpage">1760</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp324%26rft_id%253Dinfo%253Apmid%252F19451168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTQvMTc1NCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzMvMTIvMjE2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-3.12.2163.16" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Handsaker</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wysoker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fennell</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The Sequence Alignment/Map format and SAMtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2078</span>–<span class="cit-lpage">2079</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTYvMjA3OCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzMvMTIvMjE2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-3.12.2163.17" data-doi="10.1038/nature07743"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liti</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carter</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moses</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Warringer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parts</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Population genomics of domestic and wild yeasts.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">458</span>: <span class="cit-fpage">337</span>–<span class="cit-lpage">341</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLiti%26rft.auinit1%253DG.%26rft.volume%253D458%26rft.issue%253D7236%26rft.spage%253D337%26rft.epage%253D341%26rft.atitle%253DPopulation%2Bgenomics%2Bof%2Bdomestic%2Band%2Bwild%2Byeasts.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07743%26rft_id%253Dinfo%253Apmid%252F19212322%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature07743&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19212322&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F12%2F2163.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000264285600041&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-3.12.2163.18" data-doi="10.1534/genetics.112.143107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cohen</span>  <span class="cit-name-given-names">B. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Small- and large-effect quantitative trait locus interactions underlie variation in yeast sporulation efficiency.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">192</span>: <span class="cit-fpage">1123</span>–<span class="cit-lpage">1132</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.143107%26rft_id%253Dinfo%253Apmid%252F22942125%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5Mi8zLzExMjMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8zLzEyLzIxNjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-3.12.2163.19" data-doi="10.1101/gr.5681207"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dunham</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amores</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cresko</span>  <span class="cit-name-given-names">W. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">E. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Rapid and cost-effective polymorphism identification and genotyping using restriction site associated DNA (RAD) markers.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">240</span>–<span class="cit-lpage">248</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.5681207%26rft_id%253Dinfo%253Apmid%252F17189378%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE3LzIvMjQwIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMy8xMi8yMTYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-3.12.2163.20" data-doi="10.1016/j.bbagen.2011.02.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nieduszynski</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liti</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">From sequence to function: insights from natural variation in budding yeasts.</span> <abbr class="cit-jnl-abbrev">Biochim. Biophys. Acta</abbr> <span class="cit-vol">1810</span>: <span class="cit-fpage">959</span>–<span class="cit-lpage">966</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochimica%2Bet%2BBiophysica%2BActa.%2BProtein%2BStructure%2Band%2BMolecular%2BEnzymology%26rft.stitle%253DBiochimica%2Bet%2BBiophysica%2BActa.%2BProtein%2BStructure%2Band%2BMolecular%2BEnzymology%26rft.aulast%253DNieduszynski%26rft.auinit1%253DC.%2BA.%26rft.volume%253D1810%26rft.issue%253D10%26rft.spage%253D959%26rft.epage%253D966%26rft.atitle%253DFrom%2Bsequence%2Bto%2Bfunction%253A%2BInsights%2Bfrom%2Bnatural%2Bvariation%2Bin%2Bbudding%2Byeasts.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.bbagen.2011.02.004%26rft_id%253Dinfo%253Apmid%252F21320572%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.bbagen.2011.02.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21320572&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F12%2F2163.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-3.12.2163.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Odds</span>  <span class="cit-name-given-names">F. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernaerts</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">CHROMagar Candida, a new differential isolation medium for presumptive identification of clinically important Candida species.</span> <abbr class="cit-jnl-abbrev">J. Clin. Microbiol.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">1923</span>–<span class="cit-lpage">1929</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BClinical%2BMicrobiology%26rft.stitle%253DJ.%2BClin.%2BMicrobiol.%26rft.aulast%253DOdds%26rft.auinit1%253DF%2BC%26rft.volume%253D32%26rft.issue%253D8%26rft.spage%253D1923%26rft.epage%253D1929%26rft.atitle%253DCHROMagar%2BCandida%252C%2Ba%2Bnew%2Bdifferential%2Bisolation%2Bmedium%2Bfor%2Bpresumptive%2Bidentification%2Bof%2Bclinically%2Bimportant%2BCandida%2Bspecies.%26rft_id%253Dinfo%253Apmid%252F7989544%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNtIjtzOjU6InJlc2lkIjtzOjk6IjMyLzgvMTkyMyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzMvMTIvMjE2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-3.12.2163.22" data-doi="10.1093/nar/gks005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ramazzotti</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berna</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stefanini</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cavalieri</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A computational pipeline to discover highly phylogenetically informative genes in sequenced genomes: application to <em>Saccharomyces cerevisiae</em> natural strains.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">3834</span>–<span class="cit-lpage">3848</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgks005%26rft_id%253Dinfo%253Apmid%252F22266652%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjk6IjQwLzkvMzgzNCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzMvMTIvMjE2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-3.12.2163.23" data-doi="10.1038/nature07670"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schacherer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shapiro</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruderfer</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kruglyak</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Comprehensive polymorphism survey elucidates population structure of <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">458</span>: <span class="cit-fpage">342</span>–<span class="cit-lpage">345</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DSchacherer%26rft.auinit1%253DJ.%26rft.volume%253D458%26rft.issue%253D7236%26rft.spage%253D342%26rft.epage%253D345%26rft.atitle%253DComprehensive%2Bpolymorphism%2Bsurvey%2Belucidates%2Bpopulation%2Bstructure%2Bof%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07670%26rft_id%253Dinfo%253Apmid%252F19212320%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature07670&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19212320&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F12%2F2163.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000264285600042&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-3.12.2163.24" data-doi="10.1371/journal.pone.0032507"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schuller</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cardoso</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sousa</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gomes</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gomes</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genetic diversity and population structure of <em>Saccharomyces cerevisiae</em> strains isolated from different grape varieties and winemaking regions.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e32507</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DSchuller%26rft.auinit1%253DD.%26rft.volume%253D7%26rft.issue%253D2%26rft.spage%253De32507%26rft.epage%253De32507%26rft.atitle%253DGenetic%2Bdiversity%2Band%2Bpopulation%2Bstructure%2Bof%2BSaccharomyces%2Bcerevisiae%2Bstrains%2Bisolated%2Bfrom%2Bdifferent%2Bgrape%2Bvarieties%2Band%2Bwinemaking%2Bregions.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0032507%26rft_id%253Dinfo%253Apmid%252F22393409%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0032507&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22393409&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F12%2F2163.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-3.12.2163.25" data-doi="10.1073/pnas.1208362109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stefanini</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dapporto</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Legras</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calabretta</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Di Paola</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Role of social wasps in <em>Saccharomyces cerevisiae</em> ecology and evolution.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">13398</span>–<span class="cit-lpage">13403</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1208362109%26rft_id%253Dinfo%253Apmid%252F22847440%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA5LzMzLzEzMzk4IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMy8xMi8yMTYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-3.12.2163.26" data-doi="10.1111/j.1567-1364.2011.00777.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Swinnen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thevelein</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nevoigt</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genetic mapping of quantitative phenotypic traits in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">FEMS Yeast Res.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">215</span>–<span class="cit-lpage">227</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BYeast%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1567-1364.2011.00777.x%26rft_id%253Dinfo%253Apmid%252F22150948%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZmVtc3lyIjtzOjU6InJlc2lkIjtzOjg6IjEyLzIvMjE1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMy8xMi8yMTYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-3.12.2163.27" data-doi="10.1093/molbev/msr121"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tamura</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peterson</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peterson</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stecher</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nei</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">MEGA5: molecular evolutionary genetics analysis using maximum likelihood, evolutionary distance, and maximum parsimony methods.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">2731</span>–<span class="cit-lpage">2739</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsr121%26rft_id%253Dinfo%253Apmid%252F21546353%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjEwOiIyOC8xMC8yNzMxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMy8xMi8yMTYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-3.12.2163.28" data-doi="10.1111/j.1365-294X.2012.05732.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Q. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">W. Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liti</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bai</span>  <span class="cit-name-given-names">F. Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Surprisingly diverged populations of <em>Saccharomyces cerevisiae</em> in natural environments remote from human activity.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">5404</span>–<span class="cit-lpage">5417</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DWang%26rft.auinit1%253DQ.%2BM.%26rft.volume%253D21%26rft.issue%253D22%26rft.spage%253D5404%26rft.epage%253D5417%26rft.atitle%253DSurprisingly%2Bdiverged%2Bpopulations%2Bof%2BSaccharomyces%2Bcerevisiae%2Bin%2Bnatural%2Benvironments%2Bremote%2Bfrom%2Bhuman%2Bactivity.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2012.05732.x%26rft_id%253Dinfo%253Apmid%252F22913817%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2012.05732.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22913817&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F12%2F2163.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000310583800005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-3.12.2163.29" data-doi="10.1371/journal.pgen.1002111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Warringer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zorgo</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cubillos</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zia</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gjuvsland</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Trait variation in yeast is defined by population history.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e1002111</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DWarringer%26rft.auinit1%253DJ.%26rft.volume%253D7%26rft.issue%253D6%26rft.spage%253De1002111%26rft.epage%253De1002111%26rft.atitle%253DTrait%2Bvariation%2Bin%2Byeast%2Bis%2Bdefined%2Bby%2Bpopulation%2Bhistory.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002111%26rft_id%253Dinfo%253Apmid%252F21698134%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002111&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21698134&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F12%2F2163.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-3.12.2163.30" data-doi="10.1371/journal.pgen.1000942"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wenger</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schwartz</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sherlock</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Bulk segregant analysis by high-throughput sequencing reveals a novel xylose utilization gene from <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e1000942</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DWenger%26rft.auinit1%253DJ.%2BW.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253De1000942%26rft.epage%253De1000942%26rft.atitle%253DBulk%2Bsegregant%2Banalysis%2Bby%2Bhigh-throughput%2Bsequencing%2Breveals%2Ba%2Bnovel%2Bxylose%2Butilization%2Bgene%2Bfrom%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000942%26rft_id%253Dinfo%253Apmid%252F20485559%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000942&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20485559&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F12%2F2163.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-3.12.2163.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Winzeler</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Castillo-Davis</span>  <span class="cit-name-given-names">C. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oshiro</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liang</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Richards</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Genetic diversity in yeast assessed with whole-genome oligonucleotide arrays.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">163</span>: <span class="cit-fpage">79</span>–<span class="cit-lpage">89</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DWinzeler%26rft.auinit1%253DE.%2BA.%26rft.volume%253D163%26rft.issue%253D1%26rft.spage%253D79%26rft.epage%253D89%26rft.atitle%253DGenetic%2BDiversity%2Bin%2BYeast%2BAssessed%2BWith%2BWhole-Genome%2BOligonucleotide%2BArrays%26rft_id%253Dinfo%253Apmid%252F12586698%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMTYzLzEvNzkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8zLzEyLzIxNjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/3/12/2163.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/12/2151" title="Detecting Selection Using Time-Series Data of Allele Frequencies with Multiple Independent Reference Loci" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/12/2173" title="STAT3 Targets Suggest Mechanisms of Aggressive Tumorigenesis in Diffuse Large B-Cell Lymphoma" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/3/12"><span>Volume 3 Issue 12, December 2013</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/3/12.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/3/12.cover-source.jpg?itok=Gq_aE1-I" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 3 (12)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/12/2163.abstract" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-3rg28IRnTOppO6yXgtN11fP3LZIl_u3zFRtrW2jpkrE" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/3/12/2163" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/3/12/2163" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/3/12/2163" class="active" data-icon-position="" data-hide-link-title="0">Genomic Sequence Diversity and Population Structure of Saccharomyces cerevisiae Assessed by RAD-seq</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459604" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-lOXpK3ZO-sOi72cSkpyjjK3j0osBpwGtCRcudjAjslU" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/12/2163.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/3/12/2163" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-sYv4ODvgN7IxW4G6_T5jh-fcz5rhRGNGcjSz1LFJgME" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/3/12/2163" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/12/2163.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459604" id="node459604--2" data-pisa="ggg;3/12/2163" data-pisa-master="ggg;g3.113.007492" data-apath="/ggg/3/12/2163.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic Sequence Diversity and Population Structure of <em>Saccharomyces cerevisiae</em> Assessed by RAD-seq</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Gareth A. Cromie</span>, <span class="highwire-citation-author" data-delta="1">Katie E. Hyma</span>, <span class="highwire-citation-author" data-delta="2">Catherine L. Ludlow</span>, <span class="highwire-citation-author" data-delta="3">Cecilia Garmendia-Torres</span>, <span class="highwire-citation-author" data-delta="4">Teresa L. Gilbert</span>, <span class="highwire-citation-author" data-delta="5">Patrick May</span>, <span class="highwire-citation-author" data-delta="6">Angela A. Huang</span>, <span class="highwire-citation-author" data-delta="7">Aimée M. Dudley</span> and <span class="highwire-citation-author" data-delta="8">Justin C. Fay</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2163-2171;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.007492 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459604--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Gareth A. Cromie </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute for Systems Biology, Seattle, Washington 98109</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gareth%2BA.%2BCromie%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cromie%20GA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGareth%2BA.%2BCromie%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Katie E. Hyma </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Bioinformatics Facility (CBSU), Institute for Biotechnology, Cornell University, Ithaca, New York 14850</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Katie%2BE.%2BHyma%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hyma%20KE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKatie%2BE.%2BHyma%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Catherine L. Ludlow </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute for Systems Biology, Seattle, Washington 98109</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Catherine%2BL.%2BLudlow%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ludlow%20CL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACatherine%2BL.%2BLudlow%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Cecilia Garmendia-Torres </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute for Systems Biology, Seattle, Washington 98109</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Cecilia%2BGarmendia-Torres%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Garmendia-Torres%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACecilia%2BGarmendia-Torres%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Teresa L. Gilbert </div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Teresa%2BL.%2BGilbert%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gilbert%20TL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATeresa%2BL.%2BGilbert%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Patrick May </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute for Systems Biology, Seattle, Washington 98109</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Luxembourg Centre for Systems Biomedicine, University of Luxembourg, L-4362 Esch-sur-Alzette, Luxembourg</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Patrick%2BMay%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=May%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APatrick%2BMay%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Angela A. Huang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>University of Pennsylvania, Philadelphia, Pennsylvania 19104</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Angela%2BA.%2BHuang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Huang%20AA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAngela%2BA.%2BHuang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Aimée M. Dudley </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute for Systems Biology, Seattle, Washington 98109</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aim%C3%A9e%2BM.%2BDudley%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dudley%20AM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAim%25C3%25A9e%2BM.%2BDudley%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:aimee.dudley@gmail.com" class="" data-icon-position="" data-hide-link-title="0">aimee.dudley@gmail.com</a>
<a href="mailto:jfay@genetics.wustl.edu" class="" data-icon-position="" data-hide-link-title="0">jfay@genetics.wustl.edu</a></span></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Justin C. Fay </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Genetics, Washington University, St. Louis, Missouri 63108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Justin%2BC.%2BFay%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fay%20JC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJustin%2BC.%2BFay%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:aimee.dudley@gmail.com" class="" data-icon-position="" data-hide-link-title="0">aimee.dudley@gmail.com</a>
<a href="mailto:jfay@genetics.wustl.edu" class="" data-icon-position="" data-hide-link-title="0">jfay@genetics.wustl.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/3/12/2163" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/3/12/2163&amp;title=Genomic%20Sequence%20Diversity%20and%20Population%20Structure%20of%20Saccharomyces%20cerevisiae%20Assessed%20by%20RAD-seq" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/3/12/2163&amp;title=Genomic%20Sequence%20Diversity%20and%20Population%20Structure%20of%20Saccharomyces%20cerevisiae%20Assessed%20by%20RAD-seq" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/3/12/2163&amp;title=Genomic%20Sequence%20Diversity%20and%20Population%20Structure%20of%20Saccharomyces%20cerevisiae%20Assessed%20by%20RAD-seq" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/3/12/2163&amp;text=Genomic%20Sequence%20Diversity%20and%20Population%20Structure%20of%20Saccharomyces%20cerevisiae%20Assessed%20by%20RAD-seq" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/3/12/2163&amp;title=Genomic%20Sequence%20Diversity%20and%20Population%20Structure%20of%20Saccharomyces%20cerevisiae%20Assessed%20by%20RAD-seq" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/3/12/2163&amp;t=Genomic%20Sequence%20Diversity%20and%20Population%20Structure%20of%20Saccharomyces%20cerevisiae%20Assessed%20by%20RAD-seq" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/3/12/2163&amp;title=Genomic%20Sequence%20Diversity%20and%20Population%20Structure%20of%20Saccharomyces%20cerevisiae%20Assessed%20by%20RAD-seq" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/3/12/2163&amp;title=Genomic%20Sequence%20Diversity%20and%20Population%20Structure%20of%20Saccharomyces%20cerevisiae%20Assessed%20by%20RAD-seq" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459604/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459604" id="node459604--3" data-pisa="ggg;3/12/2163" data-pisa-master="ggg;g3.113.007492" data-apath="/ggg/3/12/2163.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic Sequence Diversity and Population Structure of <em>Saccharomyces cerevisiae</em> Assessed by RAD-seq</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Gareth A. Cromie</span>, <span class="highwire-citation-author" data-delta="1">Katie E. Hyma</span>, <span class="highwire-citation-author" data-delta="2">Catherine L. Ludlow</span>, <span class="highwire-citation-author" data-delta="3">Cecilia Garmendia-Torres</span>, <span class="highwire-citation-author" data-delta="4">Teresa L. Gilbert</span>, <span class="highwire-citation-author" data-delta="5">Patrick May</span>, <span class="highwire-citation-author" data-delta="6">Angela A. Huang</span>, <span class="highwire-citation-author" data-delta="7">Aimée M. Dudley</span> and <span class="highwire-citation-author" data-delta="8">Justin C. Fay</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2163-2171;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.007492 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459604--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Gareth A. Cromie </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute for Systems Biology, Seattle, Washington 98109</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gareth%2BA.%2BCromie%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cromie%20GA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGareth%2BA.%2BCromie%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Katie E. Hyma </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Bioinformatics Facility (CBSU), Institute for Biotechnology, Cornell University, Ithaca, New York 14850</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Katie%2BE.%2BHyma%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hyma%20KE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKatie%2BE.%2BHyma%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Catherine L. Ludlow </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute for Systems Biology, Seattle, Washington 98109</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Catherine%2BL.%2BLudlow%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ludlow%20CL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACatherine%2BL.%2BLudlow%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Cecilia Garmendia-Torres </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute for Systems Biology, Seattle, Washington 98109</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Cecilia%2BGarmendia-Torres%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Garmendia-Torres%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACecilia%2BGarmendia-Torres%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Teresa L. Gilbert </div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Teresa%2BL.%2BGilbert%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gilbert%20TL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATeresa%2BL.%2BGilbert%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Patrick May </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute for Systems Biology, Seattle, Washington 98109</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Luxembourg Centre for Systems Biomedicine, University of Luxembourg, L-4362 Esch-sur-Alzette, Luxembourg</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Patrick%2BMay%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=May%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APatrick%2BMay%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Angela A. Huang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>University of Pennsylvania, Philadelphia, Pennsylvania 19104</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Angela%2BA.%2BHuang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Huang%20AA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAngela%2BA.%2BHuang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Aimée M. Dudley </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute for Systems Biology, Seattle, Washington 98109</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aim%C3%A9e%2BM.%2BDudley%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dudley%20AM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAim%25C3%25A9e%2BM.%2BDudley%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:aimee.dudley@gmail.com" class="" data-icon-position="" data-hide-link-title="0">aimee.dudley@gmail.com</a>
<a href="mailto:jfay@genetics.wustl.edu" class="" data-icon-position="" data-hide-link-title="0">jfay@genetics.wustl.edu</a></span></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Justin C. Fay </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Genetics, Washington University, St. Louis, Missouri 63108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Justin%2BC.%2BFay%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fay%20JC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJustin%2BC.%2BFay%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:aimee.dudley@gmail.com" class="" data-icon-position="" data-hide-link-title="0">aimee.dudley@gmail.com</a>
<a href="mailto:jfay@genetics.wustl.edu" class="" data-icon-position="" data-hide-link-title="0">jfay@genetics.wustl.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2163&amp;rft.epage=2171&amp;rft.atitle=Genomic%20Sequence%20Diversity%20and%20Population%20Structure%20of%20Saccharomyces%20cerevisiae%20Assessed%20by%20RAD-seq&amp;rft.volume=3&amp;rft.issue=12&amp;rft.date=2013-12-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Cromie%2C+Gareth+A.&amp;rft.au=Hyma%2C+Katie+E.&amp;rft.au=Ludlow%2C+Catherine+L.&amp;rft.au=Garmendia-Torres%2C+Cecilia&amp;rft.au=Gilbert%2C+Teresa+L.&amp;rft.au=May%2C+Patrick&amp;rft.au=Huang%2C+Angela+A.&amp;rft.au=Dudley%2C+Aim%C3%A9e+M.&amp;rft.au=Fay%2C+Justin+C."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459604/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459604/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459604/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459604/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459604/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459604/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459604/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459604/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459604/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459604/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459604/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459604/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/3/12/2163" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-7" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-15" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/3/12/2163.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/3/12/2163.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/3/12/2163.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/3/12/2163.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
