<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/2/12/1595" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/2/12/1595.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/2/12/1595.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/2/12/1595.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/2/12/1595/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/2/12/1595.atom" />
<meta name="HW.pisa" content="ggg;2/12/1595" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Comparison Between Linear and Non-parametric Regression Models for Genome-Enabled Prediction in Wheat" />
<meta name="DC.Identifier" content="10.1534/g3.112.003665" />
<meta name="DC.Date" content="2012-12-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2012 Pérez-Rodríguez et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="In genome-enabled prediction, parametric, semi-parametric, and non-parametric regression models have been used. This study assessed the predictive ability of linear and non-linear models using dense molecular markers. The linear models were linear on marker effects and included the Bayesian LASSO, Bayesian ridge regression, Bayes A, and Bayes B. The non-linear models (this refers to non-linearity on markers) were reproducing kernel Hilbert space (RKHS) regression, Bayesian regularized neural networks (BRNN), and radial basis function neural networks (RBFNN). These statistical models were compared using 306 elite wheat lines from CIMMYT genotyped with 1717 diversity array technology (DArT) markers and two traits, days to heading (DTH) and grain yield (GY), measured in each of 12 environments. It was found that the three non-linear models had better overall prediction accuracy than the linear regression specification. Results showed a consistent superiority of RKHS and RBFNN over the Bayesian LASSO, Bayesian ridge regression, Bayes A, and Bayes B models." />
<meta name="DC.Contributor" content="Paulino Pérez-Rodríguez" />
<meta name="DC.Contributor" content="Daniel Gianola" />
<meta name="DC.Contributor" content="Juan Manuel González-Camacho" />
<meta name="DC.Contributor" content="José Crossa" />
<meta name="DC.Contributor" content="Yann Manès" />
<meta name="DC.Contributor" content="Susanne Dreisigacker" />
<meta name="article:published_time" content="2012-12-01" />
<meta name="article:section" content="Genomic Selection" />
<meta name="citation_title" content="Comparison Between Linear and Non-parametric Regression Models for Genome-Enabled Prediction in Wheat" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;In genome-enabled prediction, parametric, semi-parametric, and non-parametric regression models have been used. This study assessed the predictive ability of linear and non-linear models using dense molecular markers. The linear models were linear on marker effects and included the Bayesian LASSO, Bayesian ridge regression, Bayes A, and Bayes B. The non-linear models (this refers to non-linearity on markers) were reproducing kernel Hilbert space (RKHS) regression, Bayesian regularized neural networks (BRNN), and radial basis function neural networks (RBFNN). These statistical models were compared using 306 elite wheat lines from CIMMYT genotyped with 1717 diversity array technology (DArT) markers and two traits, days to heading (DTH) and grain yield (GY), measured in each of 12 environments. It was found that the three non-linear models had better overall prediction accuracy than the linear regression specification. Results showed a consistent superiority of RKHS and RBFNN over the Bayesian LASSO, Bayesian ridge regression, Bayes A, and Bayes B models.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2012/12/01" />
<meta name="citation_mjid" content="ggg;2/12/1595" />
<meta name="citation_id" content="2/12/1595" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/2/12/1595" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/2/12/1595.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/2/12/1595.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/2/12/1595.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.112.003665" />
<meta name="citation_pmid" content="23275882" />
<meta name="citation_volume" content="2" />
<meta name="citation_issue" content="12" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genomic Selection" />
<meta name="citation_firstpage" content="1595" />
<meta name="citation_lastpage" content="1605" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Paulino Pérez-Rodríguez" />
<meta name="citation_author_institution" content="Colegio de Postgraduados, Montecillo, Texcoco 56230, México" />
<meta name="citation_author_email" content="perpdgo@gmail.com" />
<meta name="citation_author" content="Daniel Gianola" />
<meta name="citation_author_institution" content="Departments of Animal Sciences, Dairy Science, and Biostatistics and Medical Informatics, University of Wisconsin-Madison, Madison, Wisconsin 53706" />
<meta name="citation_author" content="Juan Manuel González-Camacho" />
<meta name="citation_author_institution" content="Colegio de Postgraduados, Montecillo, Texcoco 56230, México" />
<meta name="citation_author" content="José Crossa" />
<meta name="citation_author_institution" content="Biometrics and Statistics Unit and Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), 06600 Mexico, D.F., México" />
<meta name="citation_author" content="Yann Manès" />
<meta name="citation_author_institution" content="Biometrics and Statistics Unit and Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), 06600 Mexico, D.F., México" />
<meta name="citation_author" content="Susanne Dreisigacker" />
<meta name="citation_author_institution" content="Biometrics and Statistics Unit and Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), 06600 Mexico, D.F., México" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=R. Bernardo;citation_author=J. M. Yu;citation_title=Prospects for Genomewide Selection for Quantitative Traits in Maize;citation_pages=1082-1090;citation_volume=47;citation_year=2007;citation_issue=3;citation_doi=10.2135/cropsci2006.11.0690" />
<meta name="citation_reference" content="citation_journal_title=Complex Systems;citation_author=D. S. Broomhead;citation_author=D. Lowe;citation_title=Multivariable functional interpolation and adaptive networks;citation_pages=321-355;citation_volume=2;citation_year=1988" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=J. Burgueño;citation_author=J. Crossa;citation_author=P. L. Cornelius;citation_author=R. Trethowan;citation_author=G. McLaren;citation_title=Modeling Additive x Environment and Additive x Additive x Environment Using Genetic Covariances of Relatives of Wheat Genotypes;citation_pages=311-320;citation_volume=47;citation_year=2007;citation_issue=1;citation_doi=10.2135/cropsci2006.09.0564" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=J. Burgueño;citation_author=J. Crossa;citation_author=J. M. Cotes;citation_author=F. San Vicente;citation_author=B. Das;citation_title=Prediction assessment of linear mixed models for multienvironment trials;citation_pages=944-954;citation_volume=51;citation_year=2011;citation_issue=3;citation_doi=10.2135/cropsci2010.07.0403" />
<meta name="citation_reference" content="citation_journal_title=Neural Networks, IEEE Transactions on;citation_author=S. Chen;citation_author=C. F. N. Cowan;citation_author=P. M. Grant;citation_title=Orthogonal least squares learning algorithm for radial basis function networks;citation_pages=302-309;citation_volume=2;citation_year=1991;citation_issue=2;citation_doi=10.1109/72.80341" />
<meta name="citation_reference" content="citation_journal_title=J. Exp. Bot.;citation_author=J. Cockram;citation_author=H. Jones;citation_author=F. J. Leigh;citation_author=D. O’Sullivan;citation_author=W. Powell;citation_title=Control of flowering time in temperate cereals: genes, domestication, and sustainable productivity;citation_volume=58;citation_year=2007;citation_issue=6;citation_pmid=17420173;citation_doi=10.1093/jxb/erm042" />
<meta name="citation_reference" content="citation_journal_title=Journal of applied genetics;citation_journal_abbrev=J Appl Genet;citation_author=V. Conti;citation_author=P. F. Roncallo;citation_author=V. Beaufort;citation_author=G. L. Cervigni;citation_author=R. Miranda;citation_title=Mapping of main and epistatic effect QTLs associated to grain protein and gluten strength using a RIL population of durum wheat.;citation_pages=287-298;citation_volume=52;citation_year=2011;citation_issue=3;citation_pmid=21523429;citation_doi=10.1007/s13353-011-0045-1" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=J. Crossa;citation_author=J. Burgueño;citation_author=P. L. Cornelius;citation_author=G. McLaren;citation_author=R. Trethowan;citation_title=Modeling Genotype x Environment Interaction Using Additive Genetic Covariances of Relatives for Predicting Breeding Values of Wheat Genotypes;citation_pages=1722-1733;citation_volume=46;citation_year=2006;citation_issue=4;citation_doi=10.2135/cropsci2005.11-0427" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. Crossa;citation_author=G. de los Campos;citation_author=P. Perez;citation_author=D. Gianola;citation_author=J. Burgueño;citation_title=Prediction of genetic values of quantitative traits in plant breeding using pedigree and molecular markers;citation_pages=713-724;citation_volume=186;citation_year=2010;citation_issue=2;citation_pmid=20813882;citation_doi=10.1534/genetics.110.118521" />
<meta name="citation_reference" content="citation_journal_title=J. Crop Improv.;citation_author=J. Crossa;citation_author=P. Perez;citation_author=G. de los Campos;citation_author=G. Mahuku;citation_author=S. Dreisigacker;citation_title=Genomic selection and prediction in plant breeding;citation_pages=239-261;citation_volume=25;citation_year=2011;citation_issue=3;citation_doi=10.1080/15427528.2011.558767" />
<meta name="citation_reference" content="citation_author=G. de los Campos;citation_author=P. Perez;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=G. de los Campos;citation_author=H. Naya;citation_author=D. Gianola;citation_author=J. Crossa;citation_author=A. Legarra;citation_title=Predicting quantitative traits with regression models for dense molecular markers and pedigree;citation_pages=375-385;citation_volume=182;citation_year=2009;citation_issue=1;citation_pmid=19293140;citation_doi=10.1534/genetics.109.101501" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=G. de los Campos;citation_author=D. Gianola;citation_author=G. J. M. Rosa;citation_author=K. A. Weigel;citation_author=J. Crossa;citation_title=Semi-parametric genomic-enabled prediction of genetic values using reproducing kernel Hilbert spaces methods.;citation_pages=295-308;citation_volume=92;citation_year=2010;citation_issue=4;citation_pmid=20943010;citation_doi=10.1017/S0016672310000285" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_title=Whole genome regression and prediction methods applied to plant and animal breeding;citation_author=G. de los Campos;citation_author=J. M. Hickey;citation_author=R. Pong-Wong;citation_author=H. D. Daetwyler;citation_author=M. P. L. Calus;citation_year=2012" />
<meta name="citation_reference" content="citation_author=D. Foresee;citation_author=M. T. Hagan;citation_year=1997" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. Gianola;citation_author=J. B. C. H. M. van Kaam;citation_title=Reproducing kernel Hilbert spaces regression methods for genomic assisted prediction of quantitative traits;citation_pages=2289-2303;citation_volume=178;citation_year=2008;citation_issue=4;citation_pmid=18430950;citation_doi=10.1534/genetics.107.084285" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. Gianola;citation_author=R. L. Fernando;citation_author=A. Stella;citation_title=Genomic-assisted prediction of genetic value with semiparametric procedures;citation_pages=1761-1776;citation_volume=173;citation_year=2006;citation_issue=3;citation_pmid=16648593;citation_doi=10.1534/genetics.105.049510" />
<meta name="citation_reference" content="citation_journal_title=BMC genetics [electronic resource];citation_journal_abbrev=BMC Genet;citation_author=D. Gianola;citation_author=H. Okut;citation_author=K. A. Weigel;citation_author=G. J. M. Rosa;citation_title=Predicting complex quantitative traits with Bayesian neural networks: a case study with Jersey cows and wheat.;citation_pages=87-87;citation_volume=12;citation_year=2011;citation_pmid=21981731;citation_doi=10.1186/1471-2156-12-87" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=I. Goldringer;citation_author=P. Brabant;citation_author=A. Gallais;citation_title=Estimation of additive and epistatic genetic variances for agronomic traits in a population of doubled-haploid lines of wheat;citation_pages=60-71;citation_volume=79;citation_year=1997;citation_doi=10.1038/hdy.1997.123" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=J. M. González-Camacho;citation_author=G. de los Campos;citation_author=P. Perez;citation_author=D. Gianola;citation_author=J. Cairns;citation_title=Genome-enabled prediction of genetic values using radial basis function neural networks.;citation_pages=759-771;citation_volume=125;citation_year=2012;citation_issue=4;citation_pmid=22566067;citation_doi=10.1007/s00122-012-1868-9" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=D. Habier;citation_author=R. L. Fernando;citation_author=K. Kizilkaya;citation_author=D. J. Garrik;citation_title=Extension of the bayesian alphabet for genomic selection.;citation_pages=186-186;citation_volume=12;citation_year=2011;citation_pmid=21605355;citation_doi=10.1186/1471-2105-12-186" />
<meta name="citation_reference" content="citation_title=The Elements of Statistical Learning: Data Mining, Inference and Prediction;citation_author=T. Hastie;citation_author=R. Tibshirani;citation_author=J. Friedman;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=E. L. Heffner;citation_author=M. E. Sorrells;citation_author=J. L. Jannink;citation_title=Genomic Selection for Crop Improvement;citation_pages=1-12;citation_volume=49;citation_year=2009;citation_issue=1;citation_doi=10.2135/cropsci2008.08.0512" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=N. Heslot;citation_author=H. P. Yang;citation_author=M. E. Sorrells;citation_author=J. L. Jannink;citation_title=Genomic selection in plant breeding: a comparison of models;citation_pages=146-160;citation_volume=52;citation_year=2012;citation_issue=1;citation_doi=10.2135/cropsci2011.06.0297" />
<meta name="citation_reference" content="citation_title=AlphaBayes (Beta): Software for Polygenic and Whole Genome Analysis. User Manual;citation_author=J. M. Hickey;citation_author=B. Tier;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=Technometrics;citation_author=A. E. Hoerl;citation_author=R. W. Kennard;citation_title=Ridge regression: biased estimation for nonorthogonal problems;citation_pages=55-67;citation_volume=12;citation_year=1970;citation_issue=1;citation_doi=10.2307/1267351" />
<meta name="citation_reference" content="citation_journal_title=Plant Breeding Reviews;citation_author=J. B. Holland;citation_title=Epistasis and plant breeding;citation_pages=27-92;citation_volume=21;citation_year=2001" />
<meta name="citation_reference" content="citation_author=J. B. Holland;citation_year=2008" />
<meta name="citation_reference" content="citation_journal_title=Neural networks :  the official journal of the International Neural Network Society;citation_journal_abbrev=Neural Netw;citation_author=J. Lampinen;citation_author=A. Vehtari;citation_title=Bayesian approach for neural networks--review and case studies.;citation_pages=257-274;citation_volume=14;citation_year=2001;citation_issue=3;citation_pmid=11341565;citation_doi=10.1016/S0893-6080(00)00098-8" />
<meta name="citation_reference" content="citation_journal_title=Genome (Ottawa. Print);citation_journal_abbrev=Genome (Ottawa. Print);citation_author=D. A. Laurie;citation_author=N. Pratchett;citation_author=J. W. Snape;citation_author=J. H. Bezant;citation_title=RFLP mapping of five major genes and eight quantitative trait loci controlling flowering time in a winter x spring barley (Hordeum vulgare L.) cross.;citation_pages=575-585;citation_volume=38;citation_year=1995;citation_issue=3;citation_pmid=18470191" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=N. Y. Long;citation_author=D. Gianola;citation_author=G. J. M. Rosa;citation_author=K. A. Weigel;citation_author=A. Kranis;citation_title=Radial basis function regression methods for predicting quantitative traits using SNP markers.;citation_pages=209-225;citation_volume=92;citation_year=2010;citation_issue=3;citation_pmid=20667165;citation_doi=10.1017/S0016672310000157" />
<meta name="citation_reference" content="citation_journal_title=Neural Comput.;citation_author=D. J. C. MacKay;citation_title=A practical Bayesian framework for backpropagation networks;citation_pages=448-472;citation_volume=4;citation_year=1992;citation_issue=3;citation_doi=10.1162/neco.1992.4.3.448" />
<meta name="citation_reference" content="citation_journal_title=ASHRAE Transactions;citation_author=D. J. C. MacKay;citation_title=Bayesian non-linear modelling for the prediction competition;citation_pages=1053-1062;citation_volume=100;citation_year=1994" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=R. Makowsky;citation_author=N. M. Pajewski;citation_author=Y. C. Klimentidis;citation_author=A. I. Vazquez;citation_author=C. W. Duarte;citation_title=Beyond missing heritability: prediction of complex traits.;citation_pages=e1002051-e1002051;citation_volume=7;citation_year=2011;citation_issue=4;citation_pmid=21552331;citation_doi=10.1371/journal.pgen.1002051" />
<meta name="citation_reference" content="citation_journal_title=Front. Genet.;citation_author=B. A. McKinney;citation_author=N. M. Pajewski;citation_title=Six degrees of epistasis: statistical network models for GWAS;citation_pages=109;citation_volume=2;citation_year=2012;citation_pmid=22303403" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. H. E. Meuwissen;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_title=Prediction of Total Genetic Value Using Genome-Wide Dense Marker Maps;citation_pages=1819-1829;citation_volume=157;citation_year=2001;citation_issue=4;citation_pmid=11290733" />
<meta name="citation_reference" content="citation_author=R. M. Neal;citation_year=1996" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=U. Ober;citation_author=J. F. Ayroles;citation_author=E. A. Stone;citation_author=S. Richards;citation_author=D. Zhu;citation_title=Using whole-genome sequence data to predict quantitative trait phenotypes in Drosophila melanogaster.;citation_pages=e1002685-e1002685;citation_volume=8;citation_year=2012;citation_issue=5;citation_pmid=22570636;citation_doi=10.1371/journal.pgen.1002685" />
<meta name="citation_reference" content="citation_journal_title=Genet. Res. Camb.;citation_author=H. Okut;citation_author=D. Gianola;citation_author=G. J. Rosa;citation_author=K. A. Weigel;citation_title=Prediction of body mass index in mice using dense molecular markers and a regularized neural network;citation_pages=189-201;citation_volume=93;citation_year=2011;citation_doi=10.1017/S0016672310000662" />
<meta name="citation_reference" content="citation_journal_title=J. Am. Stat. Assoc.;citation_author=T. Park;citation_author=G. Casella;citation_title=The Bayesian LASSO;citation_pages=681-686;citation_volume=103;citation_year=2008;citation_doi=10.1198/016214508000000337" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=P. Perez;citation_author=G. de los Campos;citation_author=J. Crossa;citation_author=D. Gianola;citation_title=Genomic-enabled prediction based on molecular markers and pedigree using the Bayesian linear regression package in R;citation_pages=106-116;citation_volume=3;citation_year=2010;citation_issue=2;citation_pmid=21566722;citation_doi=10.3835/plantgenome2010.04.0005" />
<meta name="citation_reference" content="citation_journal_title=Proc. IEEE;citation_author=T. Poggio;citation_author=F. Girosi;citation_title=Networks for approximation and learning;citation_pages=1481-1497;citation_volume=78;citation_year=1990;citation_issue=9;citation_doi=10.1109/5.58326" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. F. R. Resende;citation_author=P. Muñoz;citation_author=M. D. V. Resende;citation_author=D. J. Garrick;citation_author=R. L. Fernando;citation_title=Accuracy of genomic selection methods in a standard data set of loblolly pine (Pinus taeda L.);citation_pages=1503-1510;citation_volume=4;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=The Plant journal :  for cell and molecular biology;citation_journal_abbrev=Plant J;citation_author=S. Shimada;citation_author=T. Ogawa;citation_author=S. Kitagawa;citation_title=A genetic network of flowering-time genes in wheat leaves, in which an APETALA1/FRUITFULL-like gene, VRN1, is upstream of FLOWERING LOCUS T.;citation_pages=668-681;citation_volume=58;citation_year=2009;citation_issue=4;citation_pmid=19175767;citation_doi=10.1111/j.1365-313X.2009.03806.x" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=C. S. Wang;citation_author=J. J. Rutledge;citation_author=D. Gianola;citation_title=Bayesian analysis of mixed linear models via Gibbs sampling with an application to litter size in Iberian pigs;citation_pages=91-115;citation_volume=26;citation_year=1994;citation_doi=10.1051/gse:19940202" />
<meta name="citation_reference" content="citation_journal_abbrev=J Genet Genomics;citation_author=K. Zhang;citation_author=J. Tian;citation_author=L. Zhao;citation_author=S. Wang;citation_title=Mapping QTLs with epistatic effects and QTL x environment interactions for plant height using a doubled haploid population in cultivated wheat.;citation_pages=119-127;citation_volume=35;citation_year=2008;citation_issue=2;citation_pmid=18407059;citation_doi=10.1016/S1673-8527(08)60017-X" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Comparison Between Linear and Non-parametric Regression Models for Genome-Enabled Prediction in Wheat" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12.cover-source.jpg" />
<meta name="twitter:description" content="In genome-enabled prediction, parametric, semi-parametric, and non-parametric regression models have been used. This study assessed the predictive ability of linear and non-linear models using dense molecular markers. The linear models were linear on marker effects and included the Bayesian LASSO, Bayesian ridge regression, Bayes A, and Bayes B. The non-linear models (this refers to non-linearity on markers) were reproducing kernel Hilbert space (RKHS) regression, Bayesian regularized neural networks (BRNN), and radial basis function neural networks (RBFNN). These statistical models were compared using 306 elite wheat lines from CIMMYT genotyped with 1717 diversity array technology (DArT) markers and two traits, days to heading (DTH) and grain yield (GY), measured in each of 12 environments. It was found that the three non-linear models had better overall prediction accuracy than the linear regression specification. Results showed a consistent superiority of RKHS and RBFNN over the Bayesian LASSO, Bayesian ridge regression, Bayes A, and Bayes B models." />
<meta name="og-title" property="og:title" content="Comparison Between Linear and Non-parametric Regression Models for Genome-Enabled Prediction in Wheat" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/2/12/1595" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="In genome-enabled prediction, parametric, semi-parametric, and non-parametric regression models have been used. This study assessed the predictive ability of linear and non-linear models using dense molecular markers. The linear models were linear on marker effects and included the Bayesian LASSO, Bayesian ridge regression, Bayes A, and Bayes B. The non-linear models (this refers to non-linearity on markers) were reproducing kernel Hilbert space (RKHS) regression, Bayesian regularized neural networks (BRNN), and radial basis function neural networks (RBFNN). These statistical models were compared using 306 elite wheat lines from CIMMYT genotyped with 1717 diversity array technology (DArT) markers and two traits, days to heading (DTH) and grain yield (GY), measured in each of 12 environments. It was found that the three non-linear models had better overall prediction accuracy than the linear regression specification. Results showed a consistent superiority of RKHS and RBFNN over the Bayesian LASSO, Bayesian ridge regression, Bayes A, and Bayes B models." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12.cover-source.jpg" />
    <title>Comparison Between Linear and Non-parametric Regression Models for Genome-Enabled Prediction in Wheat | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"vq-UoAR19Jr4yrV_vvx1VBBd5rNeaLDzCGVN1EJdXaA","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459370","apath":"\/ggg\/2\/12\/1595.atom","pisa":"ggg;2\/12\/1595","ac":{"\/ggg\/2\/12\/1595.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/2\/12\/1595.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;2\/12\/1595"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;2\/12\/1595"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/2\/12\/1595":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022EfuNmWpnhbQnxWDeQRA1Fiy8tuPLXrPUeDu91C7ZD3Lhl88EirEgu+HH7gy0O\\\\\\\/rRz7e1p9lsCqi2XzNHVmleApE3NctHmcgHFj1UW2DXmlgXdZ\\\\\\\/gwonWfWUfoot6UBQfpI2nWtYIpoagHJdbjM089wha1SC+\\\\\\\/j5Ftlj4ydm7\\\\\\\/EOuSwMYW08TebrdrnWRO8PGi\\\\\\\/UaS+wbrzQjEhUoZrcY4Fa8mj3QUPj9rhwLG\\\\\\\/Me1ZdyYIZn5AJuDFWsgQx6qyhLTisuOO\\\\\\\/WhXBXCC5VRfS8Xt6sBj5ViJeUk9vHaHB+nTyQsOOPMkJabicQ9lwTdDwT\\\\\\\/x09ouRoU0P6wX\\\\\\\/I5mmG9FWDR9GEL+om+3OYbvuAs8ys0iqTIjxJREy\\\\\\\/qIMPz9+WnlcdvNCKVpvUvOJz8Txi+7CUoEew\\\\\\\/qJfNvVFU1Sk7qQIDVm5nw0lRK7xF3XFWKPJDcKE4NtnTztD\\\\\\\/IV7FDIzsHZVsz+7t5cJIbmkJjQ1iCOlIOwHXm\\\\\\\/bs0KZ1776RmcLg8ccgm4YTGT58IZu0OiBFiGsYmTf4x84fnOWhLlkSI42zCmRogsNokLCNASK+ZmW6vaYfyvA0JGwieL1Y6DtbZeiZfX+LZdwESkgpSewMFUJz7z\\\\\\\/GNCmNtoahsbwn5F5z4avJWB64b9sjdOwyEdWTfFOwJGqIR68DgoE2rT16h2\\\\\\\/HEnEMgC1QzT8F8pSWoP35BUt0v37cKTl6QbLnKEOa6cKRQ3UnP3ssAhkFnI80SBBVLMANEP+Y6c8p6qPKl2QhK2hEU7WFdxSR2ZwV+2841EF\\\\\\\/W0WVvG4bGD6FtgBvGfDNRGNWrBKEz08D0oaNz66bp+GpBYG3u7tfHeN4DsckGoTUBr1W5SD3t4HadOUoAUFoN2xWfJEXLWy4wj0RlJ+85cptkuQMLrtBsd0qEI2+HCVwvvkXlil2cDL6RRYkV+MjygwTE5FIpTIW3iGxXFTdxUW9sPyZFNHkMQZH\\\\\\\/y7O\\\\\\\/etjo65qyIpwVz7kFaCf9IcAttPxR3cWr0Fm7qUc5Nnydo\\\\\\\/UfbtF\\\\\\\/hdBpY9RoO7vnMGShQ8bBxOdJ1F+MtnPUJ\\\\\\\/F3dIV2AdeJu5mHY5nycIkYYFwaJfnsTGtYipZnXxdGIGc7TQr1CtgRRQ9ulOjV7qNXZBN8eXfXjvPtx+lWZuDkhARFadpAvsbqeo9GtJHB29fyLR+rDq16rwxk4vVD4rMdeB6RnKwrhCX8ZVWYp9cQw5oJuRJdN1j0FC0AIzjohlUCX925JnaysoO08Dib\\\\\\\/OOD69QxGsGNsUmrp\\\\\\\/opJW\\\u0022,\\\u0022iv\\\u0022:\\\u0022zBjYZhuRcR+JHzbbhHoAUg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022687b3e4167f979dda920076bcbf22158601a0f071d90da75787fe94e051e601b\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022nd+wlzV5q+JFJMPc24aHoPF6d33LcQn0bKKGpEvHOOj2UXt04mykQTB4owQvLwXA4D8oOuhPLH2RVBn3wfbd+0TPId7DaDmiG+WT7rboqPQzXECKxLiIcQLwM+k644yK\\\\\\\/u8XkqZItNFfnnTWsuC7vyK4Axxm1qImXmq2020htSIhHU\\\\\\\/FFOWvcpDu3CxPUVLyM47ygW\\\\\\\/TYtFOw8Q\\\\\\\/xR\\\\\\\/wDhtp0Fsv2wPvbGuTtnsPLahGx46Ld6QiIYQFbXTdpnUYafzIPf8IJK991yNoDkY0IzFNizN+IGl34hnUv+2mwkAYmI9Yk2SP6kTqGULhkXezbkNaizNc8WAg4tvxhwYjlDGQjxuX37b9rFcwQElXOgKsbpBywD18yXIThQ5xqs4VQEN0o7yw1Vnj\\\\\\\/fhtJe2q2ZfCkWUHsK\\\\\\\/NtbTWJZyaLMOzpG23AWmEfZ5qjvj7KMXM9A8tmPAg115WZzgSNQBTb2yWBmjN6qCN7uvVI9Fl3FLKp7u2g907fk8Hu8NMp5B7QMw8Jw8MuDkZa3AoBOvFkK4L9Q+1lck1gIExrPPpjdREU9IrzcT7dJHotwNeuNXg7RFGdiXmRq4gQf5vIXKoDu6rQsbXBV5MIUZjnCce6zC6yYbERaOygfkNjwKQZ7FNt3ThxVE4ynuwqTLU6ze3wropfqdOmWLRQbSCYd69R3eMWaVsh15d78ivSi+i4a\\\\\\\/4GTIP7\\\\\\\/77lCFwS2oK7vv3yye82EYXLhatQVgwQA5ol9K7b\\\\\\\/oKAkeX93OlaNRiZngko33RZ6KJA0F8DbSjab+6nn0rsF4m7g6gTs53pibfdGFdhi\\\\\\\/yrSYSgvPWWsDIGF\\\\\\\/4Q2Hc55RfxoMK2xDRSNZJhFIWeDI8UrKHc7VQGKVGdRUnq0ecrdZ0Z1Pws+SJzMhM0U+xZHx4WHR+q2akMkQruWTre1xUK5zWqquqP4AnS8qvxaAO0+0EVqH47wokNQB4UrMojpSqxLXiectydVPN\\\\\\\/eJtUqFuyygfcOYoq\\\\\\\/r+ic0YJQg46V2Vlh3fvaKh0ZWu\\\\\\\/JSfOHo\\\\\\\/FRQGSc1WoEOVHMHGRWcUnIqcmruSEfpm\\\\\\\/xTsnnsl2iI8oXT7IJPQ1rUCHXc3pZ3UJoWYGW2WKjCmx3L2EdvbVNxJ7DAVjlGWhfwxIQnsighkCheWLrQo8ns9DYkmjamEAPAnmSjHO8x3uog5ju0g+FFEtMsOm7w9XcXzcGt9OMniztnql\\\\\\\/8bi4fzlNwwQ\\\\\\\/DkrFr2XroSOYXTz5wOl3RkiCiIo9iTNHV+h2Bg4kD0iYDonhdk2r330x5xe4gWRDc+a5lkDMH89xnfdYNgQHG585CtClXouQfovPBLDR2e0xeVe60\\\\\\\/5BeWLcpbssn5i0zmAU95HE6QveZ3FIfDQZKeGenebiDP7CL+kT8BS7vCrIBVneA+DfmKZMtXY5\\\\\\\/VSPpw38aH\\\\\\\/puWMd8KSZqIvFbUyxKrqiAj6KdvvDQZnwrlZ3YeF+THGETf+eKmm5mvFUoyUnAEBHJXx90ivdYjgQHh9gwLNFHOJBsygfE\\\\\\\/8\\\\\\\/a9300nt0ydgIX+F7FE1tEGs2W+CYO2fhe+xA+SKfUZi5Igisy556eosz2APqs6RoasaMzXtp5gwu+GloMgiQeP\\\\\\\/Rd0M3SlYW3hHTSMP\\\\\\\/bsD+qYQlMckr3buE3DQukKpJrZkudgAqMq6H\\\\\\\/wzaIIZL2Pp9ygZzOPi9ugLyI5nYo4g5u6l8oKvDmSX3miXjujgwnPjAPqaY40LQQZ\\\\\\\/g2ODT26jQk5jV1jnODkLmxqC3Cm3SEezL3mNEf2nffmFNiZEhQnNK5l3rw8ddewZVZHC3ofjyk+E72lOdc1b2dghn0P8T1M\\\\\\\/GZ4I7KoV7Sng\\\\\\\/OGIA31gj5QBF06xSHzsAl41eTHlyUQL1TziZyrBIIYsYxGE2\\\\\\\/1Ki5IKegrB8A1i3GpbeR6VqsEaY7qtIZqLUDlFD788JizZY2s1cz8UujCWw98ukWasoyH2XEdeHH60S9Wy2gZChAqx9REnHKLfOV8mEavk8qinM4oXby0gEVG1rf8g9pqyQOAC7TEkZCAMGMJ\\\\\\\/mtJGFRKDWGGyK5A2hjjDH3zZvyotZWB0Qj2CVxPN\\\\\\\/Na8WHIq7sA24hObGJSIZhcPkHN+Kx9H+DhZ0kH9swYS0EbPUa40IrWt9E\\\\\\\/zNcIPTPRaHL29pJfIcm8gkuLIO+NArk0G3W6Lc6yTJ++m7zTp\\\\\\\/kbiolt7kofe6eyuqFWL14gx\\\\\\\/+CnB+7JZ7NA7ZLWOI8WhhQUBfkkfcg9N9NEB4sahsZ4dx5zG0U+a7Ax0lBPpwoVsxPLGum9L1LQ2wJmlE2\\\\\\\/zF6+TPAj0aR7dvvbb3ioCJqdDCnXJIiaVvLgfkTkKZxLvCV2\\\\\\\/PuMvu+NuNj+5Xiyan1w2MfGyigfJGqHc32yWJlxCWakYLH9BDWAja3hm5FqYqzfcTzm8x0YFMjWb\\\\\\\/7og0Bx5rE2W9V6+wr355BruUtbJBeSjn+LflY3nlvHyrk8SinOvyFelVD96ZHgnsiOPvndvtaoU5HH582AyVxCqOuHsYNH5ZfDhWcLtY89JAz8+GUn\\\\\\\/zgSpr+tLyQwGTJiQ2fqgWyg7G3UVcId35iu5HeKD7IWtJhWCoc1NGMZZqamAU9sSv82fo+YfAOGZmc5l3Cqu13UqKgf\\\\\\\/CH9LNi9wfJip+nqy5ROBp0GXaPqzoDI4wELOw0kbcqfZUUAMd\\\\\\\/2Lh8T4ji\\\\\\\/uNgsalmd+4O\\\\\\\/wvRhQrRO5H1iNrHxiRaHHNvclbFP2LTmB\\\\\\\/XUGwEgWYM2cjdEUavoUNidtHonfEkq8N7rhW+G4OyCEHkJ1\\\\\\\/BHkgL7yW6fQeDrz9UgtEL9YgwdkRSYzBEvqnUPtWxKJQlV949MGhm+bia3M9DzjsEVVASEAXE7KU\\\\\\\/GZCIj\\\\\\\/lT4flPZJoN12Tx4R3uDoPp5+rrYExLt25Kj5vldeDfRflXeUnM01v3b86FrUt5q5hzUh38xOVz5XeQqLil0WcYr0lA2SGA5w8aDOp3VPE9dJxboURf\\\\\\\/dCLVBf1\\\\\\\/osd33c64Wy\\\\\\\/ARe12FPKcn4+w4DPaHzCCOw173f9YOvkeDs8S8hG6YG\\\\\\\/Jv6hSohw45hvE0yM2Xy8SgaJgn0hvSOCZ2Xvr4A9pNEpnTih6lZjmhPF+Rf2JHGBCiRD3wAPxNkAKXD41SvDYKf1WTF1Hfk8d0YwAzRHfJE7uEc4a5GMeWOyGUQzWjBJNaBu5oa6nJt+Yhy5i7fghWGSSYuJgOk71aPe8PGX86GU79WXFmCuVsewh+enRQNz6wFK2ToohXO0r8Ee8fVAsDmX+OYLtqIs5trX5Kx8UkHQRZzD+\\\\\\\/cfhqAQ057v5gojYfpmEDrBM+n3vGvo6hbomfa+hkF1+kaIEWeohI2FWUVj7KVjr7rln3d8WPWJfhk8C0Ncs0z3Zr1b6VsGobz0vrmVMrLJUmxH\\\\\\\/vq4iHpZ+Ddlqg3crPGraK9gUjjoDiCWloy0TcUDStk6ngtTPYtFH0TvDGlqxYz22LkLLQuX0gpkVPqVWqDjF2ZbAHyZj93jgLFDGEjAZOc+35azATZreCnXD+KX2YPGdZZ53NvKQPuoLeIyMk0u48X7hqPqvIVr7dl3aUViD2Fr4SYXZCuWN+R9c2hxp8381ZQxeBE63o3ilSzwrPMOyEoq6rETEIaAon+U9CgXZvELQt+xrrA91wqXjVJRcnpVqfZ\\\\\\\/RKPpiSUJybdRijVfKqOobxx7z53sREL7kY23J5fq6KrR6l9uetSo1M\\\\\\\/LEipbianljTMoXWByHKnvshAR1ghCxDQt4jV8DPsNLSgYtZ9\\\\\\\/Pkp6JwXNc4eaQd7cVBIlIyZhpFmQ1ZQbI29jQ0W+u4rOaFSr7Od5iN5VJxafVotwyFOjAeEGiPEVhcYRCDyZZYuTZOjwc\\\\\\\/0tz7J0d0QPHgeiOtn9KTJZarhq5UIoqxgaoAoTzJqGUxhk4Uyl7yH6+p1AhOR6g4EUoyG3GqoOkAgBKuxKuQ2fgTPZGU1Qqzb8YyDjejd+h3j1ckzqYYTHD2lkc2X4iQJKTme9X9wOBfO3AjHH2bRYAKcLOFWJgIxe\\\\\\\/Z7uJlKpwhRGzNpuryhu+WcgPp5o6c95Bq4FMNL\\\\\\\/mdbL+jXKdSCZ+ro+aKKaijVz1PBeSzLg+9G1rA+Hcl4xeuwbiwHxbfWLEC8qAOPza1iigrJiJR5ifL01gLVmx8gKY9nAKlLuP8wPs40h25EifEPPW30rYchSpaxIpIzojc06ZkJ97iNVWRsVJHLe7pKNuOnR18n6MKiOX43nlMSb6PQSw+d+nkBOj8TUJSnHlXt3V8zW+StqmiPu7A\\\\\\\/d2tLMR2pieAbij4K9upbRyvgkb6VmYXd4APFTZAdY+5ZXlK5MuBcJ+MuTJtyEgj10e\\\\\\\/GJlGFsP5CrWY20jkpDcJ+aUfnk8ooXxKfYhyeaQ4\\\\\\\/vHs1\\\\\\\/4UWIkBfhgVoIiZVJXNBx9HJgtrtYHVD3NBFx2YYoQJOa4KnfoolYn1HZXkoLH9QdLM7caS3\\\\\\\/wcDe17rIFrIAus8dVaOmGnUjAjnTeOW\\\\\\\/BppAkooed5TUce++AQ10unz9YZ3UBC4y2Wsv+t\\\\\\\/uw1hPNU\\\\\\\/D0X92B90CdG1vwpWOXCiPUG\\\\\\\/+dz4ESuGDgyGzOdweJpqwWdH4Cmyqzv2M3R4O\\\\\\\/QMpVNHIDDwx9tbGdXhUsC7tZhZh5K97fVMdCB9OX7RU9eml1inzXZ2WvZnE0ibkXmsWMJVSoG6YLHQGVOYdbSbhnpBdVta1sQfik1FD8FERWZfBryv5ibsXphxxCtS7zm0pnAFTgVtlizcYr6gIr7fb1m1cYfQ8ZCTIfETALe13i5coqTb9QVpMDOgX2cMqQsM5L4piRuYlqxuB2RYU8nNoayohEW6GkfmHZxR2EaD5QI9rOgTBf0\\\\\\\/ZQk6qsyqOPRoBk7tAW32DEl1\\\\\\\/sGIJRHq1WAFZzrHx32JkqG+eJEU7vB4R+Dck0QGNq0sBo5nIpcyjIr\\\\\\\/T2awa8oKSvDM5j8aXFsnyFJFy7j8niRb1FqMLY4XIpwPljMeA43W7LPJ1zEGWe0ao2jwEEx4FNOapalYkooK3JR4u445BIuL2eE9\\\\\\\/B1\\\\\\\/R6wKjJYeG7BML9J7PGXlkK4PDnw++0\\\\\\\/9nGEDd4tVd4walaSAbFeoM7kufMwl+hnpr7M2CPMV6Q0kcJ0RwgIlnOIleDnj1aZUV8SzO4U9MbjtIhRWw2iAwi+WmarpdXIRJ9gSYNJDraDLqp+unnAe9EqZYZnM4nb1\\\\\\\/Orm7ZQOW\\\\\\\/VFivIOyPbMcSnku\\\\\\\/zmDGGeYSZhRRfY1ElBfMWr709sXmJ8R0q6cYEgQOWvm12oWbesaju0T+OyBGTe7xmsYY\\\\\\\/rL6QujUv\\\\\\\/mSzZ2gWxaczhwaf\\\\\\\/HhVysdKBotbscScwKcCG4QnILEKotp6ZdxF7gx1o1rwU3Wx4X3096QvnJB2tLbdPvui54SIp2KdvBFouSvOjPKJLFS+MbYc5+tAgUyC33yBhC67JwHmhtTXexT\\\\\\\/GxZ8vNfWKu4FlMvZEi2sR7iUDVGU\\\\\\\/7hy91dICHStqfGRYLCsUfVVtvO9WNnR6zTbOxjF6Miy4Y8NyqMhCbXO1Hb+1hgONUta41zdtNBdAuxH4pOnDOus\\\\\\\/LpGb1p3GU8UNazQSP8f\\\\\\\/s7UbodJPE3+9LZyQ+A660OusbesKoob9SYJ\\\\\\\/9A65xw2w1VGsjT7glbF7WbYii81mCkreLfrtcHKiYM1sjj8jR50vk2aiekf7djl9NpVsfVnY\\\\\\\/eNRuh1voUBbYiRRightGFjzr48YD4JjNjfoiBeJAJlThtpS5WMic+hbuxouBuWQz5Mw6zltr3sy1B7m80yOMJGWBT2pyx4JR8X\\\\\\\/TcAImhMAg2qW1agJfmfQRnvgCL5RAW\\\\\\\/8yqX+tJNKDWQMXMejyRZ+YjavwWqY30k\\\\\\\/popSoMP86awU1AWf1vhXqoWGAmC8I84CbNdfKZuw613ohI\\\\\\\/NKUodjrVHwRVBvICK3QrY6bofyRZoCMSsWRXzKf8E93HacWT6w\\\\\\\/VuBJxzpIzz4U04jddwLNipFQ3y+BoX\\\\\\\/CxmmQhnV0TuXvEULyL9SKKo5LVfz2Wb8CnVqPQiBF+A7a69jSedbwa\\\\\\\/FfbWXS6i+Nx4q7Z0m8k18i4S4u1aEE6ZRAN1JjjfFy7jbdsloDfOg8RURTtOR2XgbGTUzPcL5TqC288aYF3LeJ6XDc7G0F\\\\\\\/QuPxivvYdVOW18eMeWmN6SL+rtE4cmV5Mn4UcdZeb+wksEArutJvOdbMkldBCWrG+zHiGRW6U+SmtzJ3ucKs1dxpvNVJQQ0dCIYAoQLnmmPWV67mt6MjlSMGguooWO9ZsKQ2SP5c36KlRFUuC\\\\\\\/eJ\\\\\\\/r0xP4p8okaoGOqceYv+5ApJ16Slzdrq+xbcopobdBTokZ5b\\\\\\\/5MJrTd2t9SPL4HC59PwIU1jCauXxlb0EYzVpd5LgzngElr\\\\\\\/4aqw6sm92KgimztUqcZsq8xjpMDWtC84o40RQgk\\\\\\\/udHa\\\\\\\/bIc8OIVhjsiCUO6WH5Zv2wXDtOdyurdoFgTGNfwn4Dy6fI3h\\\\\\\/lrArP8hc0zb5mTLdCGaMOxDrFNUZTnEZAn4UUEdmcO9UU0IkSy3Grb7BCzQhPmGgUvUmLqbBSqJ9C4+o96KH1taf0oTOvlq4GdWpxHLja4UwYt10JmL5zHxw3+LUaJdwz9yHbaVDDEgV4P8Gq+LMO2+Rwh6pKXn+opITYogWwLYfWq0wl7gFlVteFJYd1NkSGPkDxAwO76HZNKI1LcAEVgHluqHwxDUi2ARU66p3C7RAf3QIGK64g5g3ISWB9ti+zJLU2uxL03AUF2QiZpVT3YkRzweGnMEi+s8BnDASFGXNzB677S7RWKUeDnNKOxBVQPes\\\\\\\/IcDyScbfelk00zIyplhCcfRxRusNqiQCiP\\\\\\\/n91Xnvn4pLuk5n7NSDZAVkkz0oFZLN0vWWdwdC\\\\\\\/FAajvQsC17iTrRShExgCLIvOwSqC+SnhdEwUaFhviMvCe2Nv2uEFEwtdpUJyKOVOTkA7MwG5xxJv8flrwOC\\\\\\\/zas6eQ5QBw6375MDpTk\\\\\\\/1Qt4nUiU0frhGCs1fLoT9lQ3mnyVSmDlnxYVdokG5xt+97fV0dtcORsbD4agYllJ4gksFp81Gze\\\\\\\/N\\\\\\\/PtV76joKYjk62BsJPWlexJbACodZe8VkYKjyEotHUAnXEy3QBmY0SJKkvIyABurdaeNgBw\\\\\\\/D524j5MP81abhs00cju\\\\\\\/KJMJIafTMJisuLQO5JjR3GUTQ2olqjRBGAmYtsBoW5UlDMLcTH0RseC6dD4IkKgWmP89qh5LtApnlfvao5IH65gvfL2Q1Qw9okb6rKr3P8UERC0syjdPVyNMpuMjYwcG0hX7uE7Y+f+jfKSeSh9ET\\\\\\\/6tqoxTGhV+0I7N9oj7pQAJ3uhp8naqLJ+rqeU17jY8okrZ9lNV8iyXEzuiv+x0NxPdfjliBYnd3fUc75J9YyZhLx\\\\\\\/kuq9nLXkT0PN3TtMk8mj31BIWyWZkcPaRoh2STHIyXsf7wDO3Nr0xWOwZQG9eGEqXCDsx39NmnxxxGWfKnPfVCWZO4\\\\\\\/Du243GywkSixtuJXkU2mPDn7JF6I7Ydax82zX8nC0O1Lf3oPnEKc5UIRSkt+U8m0mTRzlmIq6aQVC23QpA20EY+fPglQAR2nmTUuYHkhq8fvMjVf9B01MQBF55cfdv5Mg+noiq2hY2CIlbO+dQXKRdwLXB9cq7DtZFzPe5nGYqrS9rhN8XRmxANiHLeZV2vHI8a4RufeE+w3lIFkc1ht+hNHr1EIeatuV69t7Bbl5moErDrQFd\\\\\\\/ZcIhKelErh4gceEV8gNW4cLELJH6QGWTcgmrB5KSX9z0vOjki7Qq2TcMDbpOcYDh2YLcU7g0I1TFDOfe3hFZB1Y6oFTvsjrqMhrKeMysmWWxdlRchyvZa4Pc2uhI1EqOX+zse5AOt2Yt3b3ZNaLgGVidcUydzsy3SMCX7iRcuwr2BwQmNc4ru9J9g3HhOeykLanGCobRDGkUy+0Cmn0c3t1kcWitVP\\\\\\\/4J9KsoKl1nI5\\\\\\\/STp29cdH8XHxPcMPz5TpbQYY6nOtGd9FWVj9MeZutEb7ExRHmhQJsCr\\\\\\\/qz4d4WExPW6HHqf29R8K6i+3nWUFuKjKXYCXZQ6nCZkq74gGmF2zpRZWl4uhjB6QsSwZZ5OCzEdGojm0kg8biAhKAYrkIsgcPKGVd3DWr1hM84UFpSxfdzRZKiPv8ReRuyTA09PlfNlaTLscwigqPyI1ddrbYia5hLpznKwJ8\\\\\\\/1qhwnh1ivnFCapAgxdrDXGMqFGAz6WujJkGC7KFkJ4MV6QVCJUs\\\\\\\/D2Lx0OK+OYIagCDyjvsDnQHl6LqD3xJ6P4EIhcc2ddUW756VniB+6owjZ94xro3WRaMtnqiJIYNROpJib1NiVGn+MCr3knEBCsnE+8AWFX8aPSvwdMYgKPvu6iop2Ksv8gwOmVs7XWnK6Ri2+4YDi3376kR3u90V\\\\\\\/fAdzFyD4U5foSNplG5wPFWfxaNtRN4kbiBqpNpHfkhIFMyh\\\\\\\/fgP78oy0WJ+2\\\\\\\/ke1eXrxjxYKblahqldG09FT9U8daAmKRkeLoJwyB3o0kCi0bN\\\\\\\/GxsFEBAxQoG+tvqMP6Th6lBSYvqVjYK8Dp3CM\\\\\\\/\\\\\\\/B\\\\\\\/rKWP253tiY0y+DAvew\\\\\\\/qHe+rOUnlThpjYl5HOG1wb2g0ZxA\\\\\\\/ZZRNpykVxQtCp300Dhi7CtGmosrkqztrLIiEqy8HXJPRlIyTzONt0ARgrogZUev0Olb1dbFcmM3Unk6lgus3K4pz58yD\\\\\\\/+JW4nXCAz0voddVBZ14XS1fI1eKjez3CRv1kSGwFGIG+pkFSlb910styKJfnUlq58u7\\\\\\\/WToaKcWd8O7Tx4T7GB+rHkXViy8dLyn9d9RbfAeo+xUZ2rNd3Jy3tYZyPivZ0ygvhFznKBu3Uv7ofMKshWPYzMxbOGdAH3Z98114m6VEry2iOO\\\\\\\/5cb2ZqpWJdbJpELG3QSUsD90Z3Nju4WWD3fm7Z\\\\\\\/4+Ic1MbXI7vNbFcRhB2N6KgCVAbBbfxWkUv\\\\\\\/\\\\\\\/hCxhIQMWEJLvRU4sheSoUwozMMotkVR6XhinNuQAiZ3uZwpnUFNXowimNvY4SIW0ROAge1NzMWPv3xNtmtlQPuVjSinZhPA9oDPzcYexojtl84CSvKxbkRMiEiWe2Y1pnRTpOO5au79TahChxVOOIbCLnWjmlAhreTw36h9VISYjOaFQoogV8\\\\\\\/T9G\\\\\\\/6\\\\\\\/WVsYRQHeK7cc58orr4JNzBRU4j5rhDCkoSBrPTA8wQVzSnvko4y1XZiR9xVVIRYbhMVRYegkFHF+bDp\\\\\\\/1bSXld8Dn\\\\\\\/aV21pwsESxU5qD8mSWcaMG\\\\\\\/B5mjVmVfyrXAv7tLZifw3\\\\\\\/O9IZPz1XgpVz7fC55\\\\\\\/r26ie86hvLdFPPHa6UnDfBxc\\\\\\\/39QIGjJSoGq1A40eJxMX60uX7ZGqkY5QyrjvJ\\\\\\\/FY7UGqsXdJnMoClofj8d4ILH2CXBNIIwYHvpRWACS+AyZQcv\\\\\\\/oKJiObUpZCTHtuNrFe8fgkNcpAaavhLiTNIr9lNN5ArWl8GVcgzLAosRzjvzbv\\\\\\\/ZcD7FLt\\\\\\\/pvqZ4+TQk4gGpy6HQYgP4jgEn3kPwuKL0GtGP5hcwnijP\\\\\\\/RjQ06M9LASx5GpSkgXzB+BtIXmYjrNjXe8ttgD6hfZGql\\\\\\\/yt+q7CC8NR2eFatEMH9POmEJOGpAWxUe7a5KYnDOzXKprbFdAMD49LNKjdSaa+luYEZXDHy9e4I2N8hXO9qeMpPJOWacTpKktxr2Tv+mRJIHqys0b4mHH67uGN5lS4ZBiVgrVljDTUT89+LMGmd6NZMTSTM6gtENJV1U5qB8QQ7+VTGzNvcghNrxRBFs2RiJAhaonMAwaKIynCwXE\\\\\\\/TYlQanXR0lfHuhQKL58ruSH45Kr1DYdgCGzZ6SI4JVMMRuGt5R2ujYTmWBD5oeGyuwp62JgNHXa1LrzBMZFO8dA0XIZKUx0Vm09yIorPZK6EorUwL4z6VzJbLUE67HhsxJOjhUN6DfWKdukJVCZ13jPS4DMRucCcONVzBi2NsRwaJv3oPrp+BTLSKleEUq26nXk1pKDpGlPObhfrMIAwotPeYspvckEBmOQOlq+qfSuy00RXzUM8oyeYMEQdR8Bt5LMW2iT+BFW6c1Cz44ME7spl6HCH+gFMkmPw0ctypVVFsPRyO6qWbw2Z2eOGGpMHUC4Rq6kNyxUDhO\\\\\\\/qmZKalheGIqt4SnOqIHbZaeQci1sbm2lQov9ijx4M\\\\\\\/QesM7zpb17CiY2dixVm7tKXkDO84r3zDX7qGnQXozh4Ua+1KHcT91KvBq+53sUT\\\\\\\/rS3QGR15kFFVRj9gnyVu+wtXmo\\\\\\\/PwPQxtX6fgsKVYU9152UlGqfJYME3\\\\\\\/0rhRRgOEm7\\\\\\\/Yla5rvT4XlohOgYGpA9sTVbYq7pKgSAHy7zuhjpwjeFpsXg7wJNUHbihdyfU5ot+2IJq2MUS8AdfvhlNG1rYovROePwiQaSi84sbfY0lUmZMqZ7VQj6JFHPI0\\\\\\\/NoJN6Z7\\\\\\\/dGxed9qulNyfn86IdC\\\\\\\/vHRyW8bqDuqBh6lVkYxH9F0oXIGkr5FojVH55sslX2NkSsFjRRMwxjnN6YXi3cZhSjGT7HAzFUn7s6b6AlD46VCOtbY62A5hL\\\\\\\/\\\\\\\/2loC39W4rXomGs3MIPgEz3LZjiYPV6VzWih4ybZHfwhHNqHEm0pJ5HyotJ\\\\\\\/A9GBBD1RuYwyKc9Jqe37pqRlDW0W+dxW1M3F+f94FfuEXQqFXZb2NaowjvwIcASCeK8MewPK1iH\\\\\\\/QpWj9\\\\\\\/lvocrtdnddgIm55n7y21DTVUflAQpSAOWKBNAZVZbocOxT80LgQP2NfmFPC1wVYs3NPADs0FuLLiv21RPQ2fvqJZNdg0tc5Up62X3VnVq7QzsRqmkkuURXEC\\\\\\\/0t5Yfc5Gpe7Mjq3QIrGi3hpF0o4uPC0xqqI7ZWrfCqO7uv9J\\\\\\\/w+FMVCURmReaOveTePWkw6Ruq90MGby4cj70ZNDLimMAlqlBq0ZkA220zx4jv6zk48kstqJw0nuwYXLfuG+Z\\\\\\\/\\\\\\\/ccpAsRm71\\\\\\\/qFw9WeFbbrbgb2rgbvpJuvCZE1rD4W3SKxPqRMd2dvmyRlELRrYo1KpViqH59kRkj3cN9GaZ9J056K\\\\\\\/\\\\\\\/K\\\\\\\/V6BAcsUwBZQDgOXeE43iAURQoGcJt0S96s\\\\\\\/y9JKYWVeJBlDdFntwc6fvXsodHhyEfa+g1X\\\\\\\/XcC+mY2TkocsLDkpJqH7jrK8gy716shugkohwmqi9EGwQSHoe1w\\\\\\\/gJa7uOQcSeiB8fIJl0UOeyNyrutlPAe8fwS4YykIPDuuf9br8fuN+WjxNgnQb8xiFvaMA3k8VfgR8DAJxHPEe\\\\\\\/DME44vRDtUHcPAREkPEMdONBEp1Rpr2s3L2W5abgnYaf\\\\\\\/zHttmBVMhNUAM+CMpoBgWARs8DXnfCF1y\\\\\\\/+v8ArTCnhg52mK6asgzUL+MQbkhekov8S\\\\\\\/\\\\\\\/Y4RYfIrqwh4832eDLLh37+miMye373mriAV4pOj6QbvZd03n8r48w8qmknkhHDUWsVKhRcpv7uX+22Yx04X6lbktD3nP5mr\\\\\\\/1QipZlAkXX+I6c29KQwOzmsMOkTCdbJ9uIX2jE86mUE+jEOYtWZsq2l4AP0Fzf2hKXaShpzn3+S5psnyKQHnRKNl2NNIEsjxHClYh1nnySEwgU41Kx0ksGCcToDzaQP4IDbVmVzcdC1G0uFrViO4dqKfdGPtOfVmPD5m+IO468QI6LDoe\\\\\\\/aEdGUDOiPCoqRKWK1PD1GiZGLRWEbNtL8WIANUqku4tUW2PscnOIjERMUdjFdZBsnjbp+4li8tV7NkeGRyJEB7lW4tG8zfsx3Gw1HwNj+IEXZ+qvbxWJCZUro8RIaOoqib5aLrknepvz4O5IvgAUMhZPuHUo6i6DjJwKS2k6HiJ13Uh9vXWO1GJ1snlRICzX8NNikejWI0zI3ostWdIugrtzsxj2fXG31V12mAO+2TDWNVldOz4p6KBSnkeMRFko7MFUxMVM\\\\\\\/cWUXvbzv6IGwYs9zoIJGAtx\\\\\\\/awXmgSjUDhCP5LKqooX5cB1W8idLKbXaGipzK1XLRC3G6yhyH8dk13sTwcmZUWw7KaF+5aALktAxjoJYDvOu\\\\\\\/v4wEQP1RQN\\\\\\\/ptD1M2oJeOTQQFq0YlTSr7PbSkXMMcR0TEasEYPRfo9DftdbBqGpniYen4e7P6bP6j1PKAMeLsOSXljNHIpSBVmmJIbd9Kj3O6xTzR+jMpkVCIluTlZHILFfTjRs4I90ZtkfmWQ1iXdHduYZUx5k7ToM9w+owaJuzGbZHuXLae+RLSTkxNDTdhGA2zSTPK6s+mipL80V5YikF+qNVYhhMxEiR4Q6i\\\\\\\/NhO8dmaGX5ulZgS9UPdGWDSLXMNaTW32U37+avqFnjklZfk313v6ea2crWJtabeqGlo5qdmFGCS1V9jb\\\\\\\/OA7fAkmXGTYQuVh1pdGsl9ordW2OywofKql3fGq5R1Bkht1KXhZuNvb+nb6+Kl10yqUdOTc2caTm5Z4fGntlqpeXk3mILYl96ylC86ILlFCDM8heWEUid7da4BDF3rMxPZ9af5H0o1syVyTLNN\\\\\\\/6gBEeDSowqvYqQJcSch0j9I8RMKkz4V6bzzZq7RZRgpoR85LWslRl4VLvuRvM\\\\\\\/IlZNPOPtJABIGPo+FzQZFnexfXJnimDEOPV7cpQODCCsYyFk85rVsSTyhHz7oUHbtc6Qc\\\\\\\/vEteu6Wie3ritqNj\\\\\\\/CHsBLI+MjukJnhLzTGzKqQlg\\\\\\\/pN3CwSGF2QgIxlTWjRMhkIG77R7P3R3uMhY5Hua4rFFIUJ202hgGAHxjbaobiBGXyzcmH+Kyw+O1MYE3GP76x9+l7PeY7oypt4h9+ovsbplznAnVF2rimiHBio9jPGn7PvwnurQZQQ3\\\\\\\/a5qBWra4MHoe0bZ\\\\\\\/EYkkpXiJsFSoeUTJw1nBGJyQ5NurXSavEg9PxeobTYNI\\\\\\\/XozKfFkeejA1e\\\\\\\/wKae3BKaJq8YeinRMVR9cMvvBuKx8KK334irPXQ4MVWyVbgkTGtKR\\\\\\\/MMgkMPHmKDt3BbUzygyc+UyhspR9ntBaz4W03GDxKtM\\\\\\\/wVcd7t0k1uKXcBv61wvROy3+huGT1GSvwYZbgWdMn3SWD4oAhxzUSfOGb4XJKjG\\\\\\\/mexe47OeKO81JKgpw8fBHDyxuWCBWwX770fxAUWeUX3qRUGYPS2JEA\\\\\\\/d5D\\\\\\\/Re6Tg+ptCaOG5J7k81CBNjEnxQep\\\\\\\/KKJb1DmXv2a6+md\\\\\\\/V9GqwOj8prfjhditY4MUCQGOLaN0FFF8pavVYHbBvJJDcTdZ6wmUdIO6hj9sdDbSg8C\\\\\\\/PYKYWMX2JUGHi0UPId2ziXDctNHbKS3fJyPeghJ8Kdz8+6j6laRxeG6oDaOXg9Kz4NTvDFceyRGyDkk+k21Q4+z5ZhyN14NXh16NYQ\\\\\\\/1TGmN5S7k\\\\\\\/VW3oD7rWf27d4dBlxfVt1jElpzWJqyJpAHjdkV414XM4ttnS5ugrMuPPR1Vb\\\\\\\/UgihEcJXCdS3sC5qgdRm5XLvDtLKkGIa8UUt12UeBrLCuF1UDzhEHxiJHsBjZz\\\\\\\/DgB6o5ZhH6S7QcRIal5T6VpFCcYsGwpacmjk2dHw7crrcHTGSs0nJOo6G9e\\\\\\\/4OeMqk8uGFaSFheqYsV1YYrSycrHm3TtiDBUzsYmBMi78w9YJkeeWv75D1Pla+fqiKJM1+m\\\\\\\/BDChe5vtERerk0pzDhQgtcUspJHJwUSzNcassV5ZdwV\\\\\\\/I6xP2r6zWN880uNwqiKjiwqts0CLVtrFjCzqhCHrWsdz785RI\\\\\\\/iiHG3wAQgw+Mbc7zkMaqA0R7ZYtUFyrRh5xwsTRN+GrRxmZNaS1VO6gxPEvd6udNGB7DVc6QY+dV3tY0jSPZ3NhX32wqUlTw+TC73KdV2HQtPUo+v86MECuJM\\\\\\\/CKFGeha+6H8PuTZcTfFcxzto4nacg8bMzXefVSqw4x4G1va8DDZ4mKXogOEhv4RrwRg\\\\\\\/rWOHBZtFkU95rzjWAUqvwgvCv4J6GXhRPBewiyj704s\\\\\\\/U3nr69gkllI7kL64VvtpiIjLBibQFaUXAjUS8QznaPsKkRz7mCVUpgfhGAzT4Sv\\\\\\\/u64fq+NRhqI6Cda90oFUWeHSS\\\\\\\/nu+hIjVhGw3eLKLelDLNDK6V3+fj7VQfAcjzfBQ9wmryPtZGDkA6NxouCRoxONPQ2IjyQ61\\\\\\\/JDlxLgNl8VjEQczGUJL0LtzYm+LNw8Q2Evt4K+3GpBC1LE8mIHJf6mHgUuFhBeXlVp8axrBD1jGshElv\\\\\\\/FIomBAeXeeKBem0Zuu8FsASONwxIxEN+6PnVe9oTAoWhEMGAFzjfCNgi3x4eqsxOI2D1x95HtwkTs2+tGQfw7kGuQcsQsrKxcQ5b\\\\\\\/GdKdSLG9ibJZvS86U\\\\\\\/w6dZJCdDSiponE2eIaKYsmw86d6SLiMeWJvYpgYuNwXw\\\\\\\/d6N6A6yZNc9a7stonxhOv4yn63I\\\\\\\/2RNL7jHRdNcnvueLrViO0QjtWLh7k2K9s1txNndcEqL\\\\\\\/30Po5o+++61XmN+ECwOhMZ80gBxa+V0PR+sbTQYnFqXwF7Gbo\\\\\\\/Esjsj6UA27zZ2IS+roqYLCuCyoaVOa70BDzEKRoIWIwyg4tt6wlTDBIhxidG4LX1RQEN9jWIGMdjLUOOj9FBcvhl18QLlS3CCq7aqXJMHhjT6oR3jDr0eQZk1Oj68BR3k5b+VL+e\\\\\\\/8dJf6i\\\\\\\/kTQMs1wN8CbbOQT1Y96DMXrR7pOPIA7y\\\\\\\/lLUpUB54p2d0RGu7B086rRrAp7hOnqvcjVTgqhNylwRjwoDaWgqnCf5eqdgr7cJC9v\\\\\\\/0rTDSET+Blo22YjTYYMFc4vFxiB6va57yj1+moMDKvgYNNv3mL9QkYhgm6ex1k4IiSt\\\\\\\/lBM0WmZaZArOx58HlnHd9VT3Fe6IC7FCM1\\\\\\\/+R00WyfgaSWUgOijrYOISFu9eJSuT\\\\\\\/Ct6JpKy2ZhMcBMwdZ1hYwCvzTgUD+ScSlV6+rLI5jqHxZvkreiOXvNk9aEfEyMIZL1RHHLoAS8ObNS0D3VpM4cc+PZtcQZONZiyrAX81DqwiAuWt5oNwZUdJR\\\\\\\/eXAKwUK0+qs3lO8E0exUQqNP+n+Amn\\\\\\\/lm2dAbjc3Kmf8BOvrI6sA3FPTWyxfebsgbPKPNDblS6jz+RNRPVm2RUcmX7bkWNcVFL\\\\\\\/it7UBwj9yUaWRSifP0B6Lru4CLsbPCP7BEu4IM4Vwub8RPFNGOTC0PUA+VGKMKFqMx+alc53qj7HBKP7NxKLK\\\\\\\/V\\\\\\\/ymRBBhgGEjQAMSpJachvaaY7MnUUs5Qy1DQ4xqLaRy5Rc\\\\\\\/ApgOIZsr4a6c\\\\\\\/JkWTLTG720E\\\\\\\/XoTYvL\\\\\\\/4KsOflmWxamwhDWMsCvkTF0WCn4LE1d0Nw7YflfVp5rmsbEHkrTmtcZfq\\\\\\\/GPzbO3lyBzOMHrVhf1M43hIwOsyvvzs84LjCxPUiLn5JTwZXqWIRb7RNcyXinn58GfoWu3i0+yWFQI7X43EWY9XCbCaNuVPSROIg4evSlYNUJ21sUya3LjSWutvYvfTXOL5BBAtE8rRUqRBhxAmnHZdE0mACKHoCJx8x5jQyGu4LXE\\\\\\\/rGzF68gTcyhC3ueGhfdrzzGphX5\\\\\\\/Nq5L2S9cgWSqJ5NUPhDftQ\\\\\\\/ILNRHZliDoEEQetHxQFbjkBXcnV1zOav1TZzQ47\\\\\\\/s3KfcJw4rQzl7jiyEFOfj2duq9Am5IJA2f8fspJ5x3tPhb7s6Tax1+5MctmK8H0xcT+\\\\\\\/GK2eo6PHqtODKkY2YgkodeB5dHjXCUJwj92NtRJKR3UPUN+6lzJSXUrqCo\\\\\\\/u+NmUMabtjEIEpPwI7XoraIhVE95vBIw7URNk49xvC2vkr4yhyNQEvdoF5tWgV2w0MMbGqzcklfQqtmEzH3aIY1Yob9IwrIaAvxbpxc7sqwCanw0wlKtMZAZ\\\\\\\/a1SHZ\\\\\\\/AOFuvgH91iIPOUllxmsugj79BIUiWpShRlUCA1vX\\\\\\\/tzJKLKQx5ORf0fCJGXwHjZ1LBiDrkMxA6x5nI3GCgnXZqFkRrgPLyZe6rL9O7sd0CbKyfuCOpUEqcb3sjV1PAfBwkm\\\\\\\/cM+TmlxZD\\\\\\\/bTeIUbMGMN+WOWgzlavBapLYKc9Aj+PDnIGjslc\\\\\\\/CLEceZC3OFdjsiHbTd3qcA1XerNwEna5SUzVzFtF7yOS8LYxfCThTGxR8mkR4q5Cf+Z1hp0yzlr6xe+c18wPxVGfPElLhXHzyAUfOFpRIx5HQGfh0QDZzKDSpD1ohW9ntG7OriXaBxJ+y\\\\\\\/DpSnfmrZDIalxDcS5rgDrEqxdPJDNL797nYMFAgQEcwE3EFM8S5IRoek\\\\\\\/74BDoIedZSEa\\\\\\\/EfoMGTmc6UTvlrOPyQtJjZczYnX62TEyUKVCgBETY99+EZ3QeihonLAwwvkfUlYES+zNnsCg3NdpEn0PcTTL6Clg3T1SGM+Zz+sRxKmRFItgMcK4oSyEQk7Hwr\\\\\\\/VBPPL1LltkrI+bbVMJzG3uamXiq02fXFkpLU5P0X9D0TJGA5H8w+ovTkywq4ERL\\\\\\\/nmqlp0uOpt365B+FR7RF+b7RWAq5OpC6BmXIizBbpPa10v4LdIGMxxGZIsC9VZbjR4Ozn6lmBkBrfGxnlH26B759KA11219jyf1IvhGVMUxlYOzRI3MVaBPZxXR5IPk+t4KSiE\\\\\\\/PaprGKb1rSJ4waN4bSzdY4E++6DwF2Ljarz5tN8YgHsfWtHs4fbX9oyYs8OyBOJbVXBQodWmIZU+WBx9P7XTKso+622msIz5bePssb4GA9BNLEoQpK7o8SbAPGW4WLXWhqRlIXsMX8lhSREgWjHbbLgIY97iP75E3\\\\\\\/VFbL2ZqtKvDWWfruw9xkJx8Mdp3Dyr2OyFsYXE9v\\\\\\\/otcEoY4R3ioOHqubXDPHvzLTPxA8B21hycqDAqeSo0EyStAD3lVQCNGxkA+ma2mE\\\\\\\/lG1MtmFRVItcOjwmP\\\\\\\/Kpa9l3PMtLINJ+tCPSNTutVm2\\\\\\\/qtJxREupCTwJinAeRE5JwRQaPnk0Bc\\\\\\\/yAYrZGEX8657lRN8XLBgwW44rmNidATRKAPxu7DCMLiDbARvrcTfjoB2pAAdnIscUxz3nJmHBd2qAAd2Uwnmp3snHG4h4RiFThdmjaDqijV7d9rwwDboDDVKT7G8le+BOgjS7\\\\\\\/f7YP1aC8+O7U1pN3L6AlxmaSKTbExnYkqkUkppPPGf7snhgksXcCEVm1V7sqxlt81sbShdryE6qByKT32GILPb3f9qxKfzsAZ9DhV37i1ojX9HVN9FLDUo8gSL3sb1vOL5veFmRi7wM2r485Vkcc9vsMWun84g91PV2\\\\\\\/XyIB7OPmJe6ZR0wCbSIWwjXjaM91FIH57KuEeQ5GQ7UBIgRlaZRpDv8nbPU5TOlZyYW8XYnrE6UBAJ7dXQmvF5M1bfnUzxFnbaCP5wohwkGlQ7+uACKrSyCHyEzYGgW6AG8HIec6bFfwZJU5rp+GT322ivqguhyPnpvkWE7gygKz8S4zel4rFg5NThJw\\\\\\\/HSQ2auPTOWYeN4y0Te5\\\\\\\/6ShGdqCyNak3wXt4zOARa\\\\\\\/RJJkNf3zT1Am4Bp9Ldii8ojo65HKyrDUOAeWcfLKdYZyELJAalANCiCq0t05eQb\\\\\\\/eTVZ9xc1+vTJVoACeYJh7mxuKjxKnnw+Ci3ZJR02a+crC8GqHpvOjrrIfAIDNGwpzOHZR5CRXbbKKzbOFGj4YlokqqABxCz8Pf8\\\\\\\/Kh4p+GIT+72KHXvvgk3KsIa3Lp+3+OLpg3QWEc6ZnsJ\\\\\\\/1AJqj33Hyzjj8Q5w2jQixS2Ygf8SkUj+i2\\\\\\\/GmzxFUDdxPqyLFeHYsFLNuuZAZdXbOh0SEKPPIWMBWqO4\\\\\\\/sXZL2dJgjk5IyZoktYicvE9KeQXKK5375c2Pm0br5iLVOI0zobrS3uxQlvZEyIq3lelrw2bQB2t+8AwOjjZN5QaepYxea1jg9zLvZWGjULwJ42eI1ZNLuXE3a9QmVjUiwr7L6ite2wTGR8l+3RNcBNtbWD7M58yOCafH9EWu6W5mmH5WiTa2hWQ0WqJF6Nl1y\\\\\\\/8Vy0wUFv1u\\\\\\\/PO+S9d245l2uqjrsXEL9uHCtwqWXcVOed3Y8N4jRgcJw9eGf6oXGgp+DmRS7Sr\\\\\\\/TCD5FRl4Iii1vGeZLERdSfhhX1Sf\\\\\\\/W\\\\\\\/rAVHLZtnBI49sBcLGWV5yZh8pmos\\\\\\\/VAsT7CpO2K3bjczILSVYS8H5NFLAxh7qx6S59vJjMaFGiV9+URTb\\\\\\\/CIbc+lyD4vhUR0aQNv4W6TsxQliUC7UlqsOJkaXBXTpYVYZBIBC8YndlW6SLBiOchPCa8NZmWnvnK28E4hUg0stLM9ejPpxVrVvk5YpatIHE6CGj214KW\\\\\\\/35\\\\\\\/YT9q7lO7bul8ABbLWQya6iYFZLNMoZa2c4NrwplVF8hlyf2yFOvcmM+2PYxoonnONv0aaHxU5ewzGhu\\\\\\\/myEbjCHN22Y0R9bD\\\\\\\/ypvZlg6itAQPWNZ1vl8mtnnA5u\\\\\\\/HT+iXpLxd4bUE2fq5\\\\\\\/xqUUPEHBM\\\\\\\/bxEGIPwESW0pOLVfw6rMoM+8iHrnvJ1CWnk\\\\\\\/D5NTkD6I79PrCpFAoFuSw8xhMjB0zkxsN4WVeO+lLHLWVZiQUUxsOex2jt3Ri3kOnjeTANNtfjWOUzh4pTVEgAKS4FPF4wb7tY3v9\\\\\\\/aF70sKnC\\\\\\\/nyz61vbfSoNA5f+zjarbmzTJU7S9juCxMeRuHcjVzvvV8Hy7t3OXTeI3wnnE1BiRCQ+tt8A4\\\\\\\/o6dNu1eoN9ttrrhvp7fttc2LxONvv7Xyva1z1qexYARX\\\\\\\/znKpnz4IAUzB\\\\\\\/UXS4obkE+kNNY2Bs9IYHmvEXXGcq632Sut0nJtPlVTnn\\\\\\\/seirxouxvdOVtkHSiVcazIjYGMPzyWZdNGMVPxm8okqV7Is\\\\\\\/5DJ7Y5TuDH1\\\\\\\/zJlgnmJY0+IqxrwVr4kTtmZAD0xz\\\\\\\/1Vh\\\\\\\/DufZ2LESBsfv7ArdXxiiMjdzwwcPrU8z4gNWq2aHxlzCQGg7rM36ytrrRiRiCPFv3JANh5ANIblCtcHVBJw1r4qZnQDHjAeM\\\\\\\/Ixrq5gwPg41mBmcho3oEy5Ct561VoRrUbnQlnel7aoJYcpJW1tvMgZx+FcWbbQvpEBYOUrQVl4QmJ3OUiemNtFAwxUfy6NzmjOzsmeOR\\\\\\\/uedjJ+sn0DL3dxQ1RWkcWKk3vb+uFjIT6\\\\\\\/q6z3NPaU1gVL3i1CrvYjLFbP02\\\\\\\/TYA\\\\\\\/bDWzpl7QeV6nHPeqakR8WMb6bCwMYa\\\\\\\/ElaUtXNTmbAxL13V6KtkiJUIQU7m89hXOsafr70\\\\\\\/kVwg2OgUbJqRY2jHJS7yyI6UlaG1uB1lAFhn6eMporrx7CkiJfctTG8vEbEn+6dwZOfBi\\\\\\\/ByOhVLpGe+Hs2KpIGgsFvZNJufZ+5O6nwcmQWfbXcVF5Z3DZkQOQnokd9vmwIgHZvkBMMqFvLOolKIta+S5XR+0Xb3uZYlZ2AZQ0nk69\\\\\\\/aYtr5KVdGlFWxjo+kEshJGx8FrqPmbFlvP92XULZksJye\\\\\\\/S4\\\\\\\/UQPClWSCbjOC2Tq2GkD2ZklNV65A1w+Ws9iwKCY7pApqTSJG6Cend6XrEIRAQtg6B3iNwc87sTgWs862o4FtrbTyQwFx5Jr6NQAMiUBvqNnj3IacrspcGPek1kRkZUfAzhjcY3mUDaSiAnZkiFDAeERccO+XJgDCG6BSNJlTConBsOuQdlCmfNmdvGQMrIcBXc5CnIGtGwqfVlFh1aapswVtheqxU303UXnIcOaBjHnkl8zRZ8NC7GMvfi5HIfF+alZPrN1cyiK4O99naxIIdeJ78sa6fIU7mFJ+mdKUlJbhgaWiEjrstCbNJMUNFf7GEJ7BXVPCX4cNM2mYcXOHBcceKMdPVSjnl975wa4dTHhRJP4xvH78TK+z1ubfjQrQulVx2B3qvvIoPMrbXyzc2Bym8+1ue0Th2NFC53Jrc+OPXcBdw2Oke1GxU0fnFKIOXtiQTwB9OOMfQG0EdB8WKpT304ilxtDuuMMpi\\\\\\\/x\\\\\\\/fmiAS3F\\\\\\\/SuJfuo1U\\\\\\\/nZuVra8ZcAKiv0WMOc056IdXXZ1e7cNf7l1wyVnGLIHvWGg8NlZH7uDYGRvnx7YKcOutOyTsbSvERIz3lv7wXsUkcD8VRpEEHVBhl0n6ZNaFf51IZjuIJdOGAulH9onKcfDxxFeQc\\\\\\\/creV2wyF7d3+evUAilkTi49v00Kolhl\\\\\\\/VogMtMqPrXfNwiKo5hkssIPB5fZUyjgbCiNNqEusFyBqcwuXuDY2oIBV+TkCFyXyzWsuJTSbAErGd09EVYSAPA5yRfbr0iRrNk3a3v0zMp+dWjySMb1ZxvwaPU0lUo0qYs2I9HZXDvQrT00QHymIy9EeiFyDGJ\\\\\\\/z4ODRMYlGjaAUraTKcX1senEiIakS9xQZPKrB6ySHNk6Zk9RlNY\\\\\\\/6kdV\\\\\\\/B8e2hqBxLy4FPLnHqhEg5YfxlBhiGyAMNG8S+lcI4XcsFzXSF58mNvFqd5taQV69xic5IXlOasnG6oGclRAp1kVXMzYQxxHGeAwMiFaKI\\\\\\\/uvy1DD3aeU+GfIWmtXFZYHuTnX6y7My6DP0VAnhjgFWFc+zuXajjfKmCuXKYQ4WQty\\\\\\\/9xtnHIL028aU10E\\\\\\\/54j5ZtZlfMBqrnHwOzhU\\\\\\\/voIDloJG15dzrXwDIcyqatMOxl\\\\\\\/YLPx2BPSbEcSYP\\\\\\\/4Dc4Q3PvwJbjlbYhAK3bVfun2bRymBp1rH+XEQqy19Xw7JfWPbZ4feWkzAC2Y4qPYF4BGHKHyaWcYQkxky9Ej6\\\\\\\/rHdQOSo5fPE5kauwcPnD5kpKNFNtvNTqvcJnWa4oqO0kX2g+xMJVMlGp9goO14G46ZPP1f+VQXopD3mr+3PDvFqvi9iDqYTiLBbToyc9t6K44CRMB6jGmt\\\\\\\/j1JK+HZr8H7FFDrx5gTpCQDNAKSJFIVXwAC19fK2BDUWSkajRxs8fo1WFNJMSVFaZZlcYNa4gwO1E8vWfcAlCZQIhcTrGR0L9pkPSmvMTxEPNvARyBS5+jPwuOw7ipe4CrM44G1b4FnFwrFUWVuA4nhh+W8V+xG1IE1\\\\\\\/I9CIn81orf5G6MKqiZNNoGg8gnZQE44hgao4MA2E1WlUjjMrJHZSKB1Czf\\\\\\\/muBKkPAPl6B4N1ZDzH7EriV2+K+0mZ3EXyVSrvi4Nz5yWNFPb\\\\\\\/uN9oCG4BJIorU7tajJ55Sz0zcYQqrN3N9vapKtu+7Ski45RjDry9fNWm+eSognILmq9tQ4BkHbY2zIK1UwBTfLE+pR2e9lceQLkg5trO6TgDqaYtUTrm+8msW6jdHhoybld2o+ghTvnwS\\\\\\\/8jykeF\\\\\\\/szEmQevOPA43aDrmlKrjx9HjkUtGJuz0K6eRuls3mRw2uhZKMUy\\\\\\\/Lu5PGBw2y3lTBtgVPAxO4xHvUAN1j1s7WivnFwaHjKEWIAAGdSPho+Y3NQqrciJs3jzlkqPEEJyok7PpKVe2waXpOhE7q2EwR0bRxPw2vV5+hzJwpSCknEYuorRHfEcjMNuJzY4KIpA8G1ZI3MWeXKw8FxjuLSMmpUz+DpXTdUEnl90mieF7vqhPDXBoXfYCVerb1Fn7J0VlUjKCPr\\\\\\\/vfosJyo8V+Mtk4Xn96UEbknqFoegklD4FbJudKRKlr6C4JI\\\\\\\/v73fkIhRYYb+nP+iKiPMOyi+3S+VOSI3FHVzF24wM4B\\\\\\\/PDu4Y2PZdAxksyiaKp7q2HbwaIJud1oBmAN\\\\\\\/ugfrPMQTdAUixiaAGVKdWiyi3qSTLmm4rKt5c+rtDkYwopt3RxdPrqprgypFXOC6zvkCLE+j40N3se6QK1SXciwpT\\\\\\\/5ydS0kK6ycPwVkxc\\\\\\\/BaUyvpPmp2eQSnwAnhkeiPNoJXMPmpgJxXciIShVzwVtb006+m1E9\\\\\\\/6Hfie1wdEbGTWNsJEAvSwZLx1GOkkdh7CgIri\\\\\\\/YVy5sb0r1OMTGUgcMhvwheJosK3H63CH7euGiqJg8awAiwnaI6uINgYvtlVYtsRIuD5cZV26Rc3H47xdGbII7j96+KHdTH+HHaD8ufKKicvOcKlAO+YtvY2+cMFXJs6BXvfgvyrAGQ2QFB7EwG4Of8qWNhV+YMKDTa8KBw7p\\\\\\\/JT0H3HASzzCHnPLsis4Dq8d1K8kbvU+m8NKgvocX9IwAf8SyJvL3Q+Gmm46pDJCwdk5waJ6FAYW5XL5fhZXHVfXgMPTgVctd4HaTXthhM5G4guvvbzyvkrQpZunkLKKyiVoWvcwMr8Qn5MLq9uQhW6eXKMwVcL4TOqIc2M6zSY9dbM+nDICYYmSU7EiQVlFw5tZRHsk5icxbvyBWv0etCxvtDvKAVG3Yg93iALG\\\\\\\/945ScvYSKdGRfMRD46TEBaFc2AWNe9ob9IP6NdPTZpsT4oU6NKBtg7w92NwfiitDI4nBB2YB1395jfL1avPBn747FBPx5pQu8MZnP5ZcPaXHXeV6QqIeBozZrcr1ITiRDFVn9uUXnGHCas9I+ciJjPy2jkVILvS5orV1WVlIZr3XW3wh9qqgBQz+jK36Ps\\\\\\\/WW0Me+jjB64XjqOs3hsrnCyYx3Dnl+1OXI0KB8hBJH7+YuxhjV5Hp71WYsZQuwnHCcC4hLBsoZpkJsfAnn4yLWrq3qCQyiDMMUmy1Ia1y\\\\\\\/TiR\\\\\\\/sySKsqFS+rmjdmyVYci2tJLeqtbjPgVwdmnsLrIBWFFAk6Gcq\\\\\\\/XbU9lMw1DmvCRqv4fA+Ih8E7atkxQMeQNTk8y\\\\\\\/4mzaXz\\\\\\\/hc90ak9B8QsjAU4QyFraCdAYHmXq75K3AmGVcqHWyflfT7h2y1sUMc5uBuiNMxpJnmtrb1k3J\\\\\\\/R8N4S1aeEZv0phWj0v09Ak\\\\\\\/83jCV6zSnZE6Ff\\\\\\\/uDZOTc\\\\\\\/vam0FepxM7DmzPVjJMTL7ysSfzx8sdbRgUT3H8AiTXy4GJuUg1s7qFdG1H7HsGyBYFGI0q1JlJx4iohsWc16FAQ5qyG25mJV32bIck6Sc+jKxbSSk6ZpBr3Y6pMY9pHw0WStSgYOYgz0J9Ae0g+5hph06T+g2b6YsGTljt1KLTHuIvD0qpAlBv56EreHjYfMv+HUDhpEUnl5Qlqc9\\\\\\\/JgayldPJgd53enTj85zKd+AWYGExSGdSY\\\\\\\/2Ck4lz4Fex8zCRB08B9jm\\\\\\\/Z\\\\\\\/IPHBWpteJIC1qS1SehqYSuqrK+MoY1hd6WoHsbHthE8l+fhhXH6XxLzWRtByFw14hiENeNbaD4TqHWrqiJjccU6nFa89c+hGc7oKLijNWL9gcGSck7hDeydn3ARmkXPRviUSL7TSNpzJAZOZw0t6DhXMLjlNjut9umm7ZMDeexpCMR3gA7wQJbJGkXycM5qrV+znOuaXb75RW\\\\\\\/IpCpAJxucdlJdO5FfmU2JJzoQzuN+1QwpkafFH0kkC+6uf1TLo6TLMcJiEooYDMqK10dRZjDB9BocbO9A7wrIvHc35BdaDrQmh4m6TqLGB6nMLuDVLoh+2ccxxDwRGQj58BX6KeKAHrvFzupWFbR8yN9cjA0QOdaCCYkGNzqBjWfAkZVHyPr7i2DkdkiDHnHBYPdU8qDnqcZLX4VACtVJlbZMNPYEv+U7tzwi+Rk29xgQKtTogdm1jnMdWRgexJoRKBRy196b96ZyGOX5mqpRIN6QF5IRJ5ahUkeoFKCsJQNyOJFXfCQaY53aFLR+MvuQX0TS2DvB2PrH3ALYWMFEHllPHX1AiQUmPAvWZEkNp6digGnDqL\\\\\\\/xjPHPDOrRXws9GVshSFOKw2CPHGI9TAwBpJoR36o9c+NLFFxfXQXe3OUOmsVXmsAQd4dA+XkyXbd0zPDu+HQnSKNUzLiHjNbqrwdarVh4dxi4ovT\\\\\\\/DKwXzR4cDMuAO1aeBriotBzBGg9F6OjKgmXZDaLrtiUcUkWL7AsOF9sfbKTVWbUw1thZ9vfGZYimm90jn5nBOL\\\\\\\/W\\\\\\\/\\\\\\\/NRxGFqHgMO8Wxn\\\\\\\/wCO6xZVwP0UnFADApELwxBmWgpUr1lZ+xHdWgbcbsKVdRVcB111tkMUCYi3D7dY4Lx8RA1kbrawV2zgaolES3TkB9w3bKCqRYs19NHTs\\\\\\\/n7eBfPBgBfMI6V+v391+ny\\\\\\\/t0PV1XddQrUDMJ1Jp5w9jvoNV6Xi6VaUKrd1KNn\\\\\\\/CW+BqrPVqSXiDCW9bhwoW9zJJdnit4yZE7TOK9XBo4bKA6y1wtptY1vN2r6ioLolwgUWBEMvZK2rjRMYESuu+WNpEMXy7paEqdg4fg6En4se2x9C2yE16xJJw8GFKKQc+snYEWaereJBjr2uLilrAs4QeJPc5om9OU55IdrzSBQNujk+GMFKbHadF2WtU+7opEg7OFfXDasvUDJcQxxTrSlyrtEz0NhqPm0cEgTWRnoS0BtcXPL8ZuAjuwJECn2K9YWHdlDmenBOeeVmuH23h7urjigDcqk2RdgFyyqjpWu5ktp4QeMvoANKBUhGxMVnUx\\\\\\\/ESmh724TeC2ttm0s5A\\\\\\\/lG42RyW0wsCXShh2m09Zg5MamCdacU\\\\\\\/4GaA2LaIh1c3wFnPSGjHJYxPEWPNhGrGBVTSOAz1rkrjcJS5a4lDd4\\\\\\\/7K2iACwopsGo9YX179M9UvUz8EEb0r0ZVxW6p7Fyb6\\\\\\\/Fglzrwcs\\\\\\\/mask1TEZBNWYRVUKR5rSQ4VvdYu+PnhXZ7lX7O1gCES0edg7meMsIN3MM8E\\\\\\\/bHAIvhZCxVacUnhTkW2IsXUlmDjY\\\\\\\/EhGQL6Rd5T9bpfFroVotqvwH8I41Pyp3GdgJYuw8La7lLtRiowm4qhKVBwM\\\\\\\/OfUOIsZaLIEJDH6cDIygUXQ+48GnlUbCUXUHDrBfaMPe2PJJiE3bddkhNd57C2TYbVqPZQFmNiznRIprgjRr1zIVMmq2bZ2axpsPTMUrsVHzHCb8EFrMfALFKFOuFm8X24VvTWsOTEXo6ef019Cc+7\\\\\\\/kt05e6\\\\\\\/ZvE0vX+2rSydFhbe80tCOW4sAKv\\\\\\\/OupdI9aalgAsTozRs+U8yB+JlxsSvtys+cp9Y7kL25+BVHv8k+UQm6w4NhnWj7y9xAIgDl8AnoS\\\\\\\/bicYhPJiDsryoV+b4jHQ7HgVcVhZR78QzuAfvbcL3t5p+82RwfpTJxmzJIFLi5oji2VtYBJ+8rrXj4GtRLyfxm2CqQ1B8QUjmTGdEo3OTBkfGErkZ4y2AC3jP5qTtzHB\\\\\\\/k3ekKVDFWnGqpUHOWMquNwBFuffBxTGk+nmeUxTcKfq7wdjP7HahN1YwWSBxaWIiEYIApbwQE+eTitSwh3dt4uEoqFBQnQo8WbOS8UBlBLljVBhGVYG76wE1eAUVOusV7rdE3ob8fBvBJPVYY+gkMgzPWG5\\\\\\\/84DZmxZz1DsAyPxS2odI0x8u7qz3ne8aZmF3D4JR01D\\\\\\\/rvFJm+eyAjcwG3bpQXmoJ0upIl1WkTc3SDcHU1Ui+MtNB9erPSRhC1BWSkV9EfFvnd9JI0WZxdYOowhw4pq84yyePFIuyPXPF7gIXl6sUXt3c580VYRn62Idf53ZCr9ukrY8doKrimgkE2b3xuHgWa3rk+uC6xxi8nq0Zz1OTbwTLbZ1rjWm68yDUfEIbj\\\\\\\/Elt+p985BUI5TJSLghk+24uHt6gc1w0Nlvbfb5Du0cj7toOIO1RGRT0HWnFbKRa2jUcRZ6us6C2JYYZODabj\\\\\\\/U4\\\\\\\/IqgbYkOj\\\\\\\/DgWxiNyVMCJbfo\\\\\\\/nSOW9BJy8zazvVEm4hPirjnXpznNEA8KafCSUDzuYoi5ktrZTMo9fN9opvwwV7FUAksh9SJvCI86RB+yaWgisk7brbo\\\\\\\/HrMXgQWHApiezRXc+5Q4s+hwznAZ4qeVFuGZc0I3Q5yq1QetoU4U+yTYfaWbFUH+lxwoPVZb6t6h9gj6snIVi99L+kYF8Bim8BU40L0EDQcSa8asT24uy1xWN0wP3uSlZbbZsoPMalN5fJ7C8j35kMlL6El5YPNMY5+MH+LErFIbu5ORixtx+oC4ZD8bY9pdmSf\\\\\\\/L1njacQnElvNMp82O5gfTATtwVlXjzFuyT6MS\\\\\\\/eGJroVs51oOAqanBtekyuVCyTp5YiCTw9xSQGvp0+urC9wrq\\\\\\\/zNLJVMilCt34Kk8pUBaLv3sEdS+CTSWkwmJP0zf7F\\\\\\\/12Xcc\\\\\\\/0OAN26YRevy4hx3XN7hEE3UwH66Ds2lgYPdDc5eIgBw+XVVAVTOEN\\\\\\\/lh76zvv4GjCD7mfI7JnW0it9CCzTeEhJaJaGolfE0QkIZT8\\\\\\\/VzNDyXZhCGCZF\\\\\\\/YLgsSin\\\\\\\/gxQ3astmYqzLgU4LmEg7UTpxGH09pRAG9AipY7Vo2G+PgcSLgMx0KK35gBjgbw5k553iRWXWX8Vh+oXT3P1UohW56D4F9La+wA78dlIt1hmxEsA\\\\\\\/3bEnJ1z53iaYThOJEmVm6m6f4xtEzbKxUJYGz51d1\\\\\\\/EEFBO4qt59Ytw3\\\\\\\/+H7YvRhNStf8UG6srYlhq7wZg01f4MOw92RkeQsB5dVearq3x6mYnD2xnW7nYYSMT0F01oj1DkjK086wGcYeigIk3fUfVLCy4pa8RXWiFjexk2NCaRB2iKe\\\\\\\/hm1KQes7Wi0zghUeBzF3zuAdK7DnlRwJwbiC9UlyFgno+Oc7NQlAN5qXgqRrvejgEb\\\\\\\/QV23otpsz8GoSFByohwQ2X8Ql3MGl04fyl+3kKY0CFCAeu3gfLPwfqfH0Y91Hsq9q2DTSEoz7eqN9ortG2cneIrHo3gxvNDt6O4j4jB2njAfbacoLNdK+ExpYlrBo4rs88GwuEYNtZqIe6b+bh44VZqHyLXbgBHn21Exa5swgznH7N3YozvimhUdcAMLzydU0c5oHFXsf5tRzkHx+8jePhOc6c3Lj5iIHvXJDV9OondtbdiYNQUUwABfhBWT+l2Jr3rykJzvk8FrKFyHwK\\\\\\\/JvEwBLsMRhwgw8x2b++\\\\\\\/4OvRr939OXugiV2j2x9DST6aB7J5irUrcwj1o6+YIGI3yNg5uGSOLNRWh0BefKREbpD83hCN9RJGvEW+wJgv2JJRaK+\\\\\\\/3HCCyNX4z\\\\\\\/t2sjQ1RztfgXp12g1VoY1W+2CdaaBTq5yQlr4eqHLtziCtfTSUnZk\\\\\\\/XmkVSc\\\\\\\/UUCIrGSgQqujXQ\\\\\\\/ThSxbxCO89SF9cILPsmFDEx4vm43BuyId1Z9tsY7xY\\\\\\\/RTMRAsG2AZpPaXZZ9m9rCQ21A0olgfnqrfKRC0WJRYObn2NfvjdF\\\\\\\/aw7L1Csjuu+SDcISMWbJIixYZNw9ChfkeT0bdUfhdxGNea8PZd76NtRu3GZsFMpAkb2TACWmXbowNsQ+6rcpr2tNDhjWBWfeKLJ41x8r\\\\\\\/GebCOo352\\\\\\\/vrq3YtsywshgvFCkqzk4iFDw3BYUudX\\\\\\\/3iUUKfHvF732f5XVLSFU5BGg8E0fQanWX4Zwji5JtyGUAqpVgzUYOLup\\\\\\\/noJeRL9rpyG41DQKv3ua9nfDnOljf4y4rvyVR+uvgrMb1kPoSyOFR3ZbzvULFc7Hz41nm7uli\\\\\\\/5WB4uFcXSiBYWpIi3MLItOv1mBQqARJybMiJEKW2aLYp+RgSC8wneSev\\\\\\\/xpdSHiCWbYKBaf20LhGkaYyWB\\\\\\\/dCxRA3BbUR7ug49OE5ebufmajYy9nVPoro1yHzy0THIw3eJ2R+yWrFeQxP4dAABRlqdcyIyiNwY60eJEt6P2qmYpknJTM2q6OxFs2Mk2HJBUEhszY\\\\\\\/LQ9\\\\\\\/qT4lQEEhynhunUfqyvJx6s5898UqD\\\\\\\/KO3sktvZ5BvfAjKy8cxJBwG89bZ1I1\\\\\\\/KYwbWpqDY\\\\\\\/BUi0WXhpTm+RGSvrl26bWIYBg+s\\\\\\\/MKawiIsdjDJn6GzWASd1ki4VDmhwcEkr2jUoJwW1BxmjDWrwqZoSvc\\\\\\\/CWRJFtdfkd4fzwy2cFYEbnfAM8VwvfLaQSkZMPpJTy+ADYvQD6y12HybuVRC0t9ui9v59m2yzwCbDfx9Ma73f9\\\\\\\/34RmflPoFlK5hnyuI5FkdW5D0HHgBWOyXqgeY\\\\\\\/Y6Gl\\\\\\\/uL17xwPMgUt2H6haGCHC\\\\\\\/sYrF9GZazenpk9Deru428iXm5vMaASgoIyqhW9L+nXjePX66bZVt0bOfmaQwM3BieDYuO9o3YCCsK4PcvuFnmaq\\\\\\\/ObUekZnIij4sqEzfEZDFxCNObBa7TNJ8a5Iig3VUrMlW8adGLkmiu\\\\\\\/tdhWI909Gl1DiCb57dfb5YVJi4E5utFZWyaHv03pb+GiBeFoOS9ZyCd1QrgbGnb7tJKB\\\\\\\/ZPUsoLvAfe8u28Tr0BPwYnVEBpZwe+tTmqDw4ELoqSGHI88I45qw7dFS9vw8pl8g1Wdm7GousrzkHxhOajjjK9BeLgoCO\\\\\\\/F6GgJGxSzYVjh7mhrhLNMB0J4OcrvTZCloP7eZWuS3mto4995lfrOPm6A==\\\u0022,\\\u0022iv\\\u0022:\\\u0022EV7d1VRFf3evGxZeH1Pytw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022465854253973d1f43e0c9ef96301970fd545dd344601bc24120383be27e0daa3\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022lT\\\\\\\/oksrUb0U+cgQ6IxQSYAFOD0CPg+Z3WgolcPqmiHTYvWfys+P6hXSgcwdlve2vkOTPAch5lrQB+20bfkUO1KwqtJ3kAKurOCBV7tATjpEEGxz0qJgRgjxI8Cw4U0uaPFX9rnL7f8QRaebFOYhAUZQdgXwaGFH1G0yazSHDNhCTrsCFTYs3rCgLawHaI\\\\\\\/e587NDwVwBLUcBuTqMlc8tNx2gr+b4+UIcvR\\\\\\\/SEi6Sn8cPufXiTKKjZeFnjRqfZOov7dz1poUGV0jBZlQ8JM8cO3DQtM1H2eI775hVFmlrXAEsTzARvZQKzvakHwEM7y9StNJ2oWRY6sDnbALA+k1UjwL0cyaxxmtBaEv3nKb8TGNj7EW\\\\\\\/eoVk7f1BwWdQDdULsgfIWyl6SqqUQ8rqx9ZIkuCe+1lvyvWe+OlkcPMQqwoOYqVieYuuIJBLXeFAoePGkRvBGJeBGsmcbPG8oYp05rAHTp4Z+kYWUUsa1dhunNDc+rYvySTmFsbpihVTVQvLY1ZXGwkqWmMd3CXulUMHlukLemJMLaQWjUKir1CS31AaWBzeWTmHZkTi9h8TY\\\\\\\/Py0vmb1z\\\\\\\/lWIHpRvJ0FV8Ty9qegPxdO45LntIXDwBLUhrHMTEinxKmROt4QBZabkpCibPyVeVEUKuMXo7fEH+kfq4HY6irIBp5Vnw61cAtw4U01fQk6B+nOtU8DGcprpGYV5XoC\\\\\\\/lv0BVpFCV2nlAYWwGrBmSN2gmJPI4j66jHzJXdnJxvN0hafqJISzAUd\\\\\\\/BkEztTAtmC10PPauyj6CDTXaFc26aED1yI+HSt+\\\\\\\/656SuOA7r9OSHfhMdn4DfD82GQh7h75fW3rB5f3GZWsUfBLYtEbjSqtXK9gwpCQqLV76eOE7hb0Hg9Co952BlO8EpThW1rGeJmgC\\\\\\\/a4XnFt2lnrR+ZNnZEThaQBLY32i8WXTnGVzg+mNKRkMa3\\\\\\\/1T8904L57rvu0b2riMQIve3i1Ytpga3EqfrajHTVk3a63jQ6gAcXFxH36KIGyyGhPd5\\\\\\\/Xp\\\\\\\/64Czxqs+Z+L+btN8I6P+uMNzEOKmjbE3uCK5DIHbjhGPNlfOdM8rFb5gE4JD38iavNB13Iv72omB5\\\\\\\/6KVNPkzbg8CGTJc2CKPP6BXC3jpaJ0QY9VcrIe7T9dWuiXCSOCrApBzVZ6J1KvUU1+9dDURib7azZFTYlvBPctmIZUkfSP0DUMfKlosSwFE+E9+8I41pZULZB7T2UAqS1e3a2VV5viTbZj\\\\\\\/hLQdOYfME+zrqG+92QTWhvYPtxZgLFzcrGY6J7kgK153jahYg9QYvD612oIDLQJMFAMk15MX7cPgLnnX8dULaUvQCf7nZ9kjTZeoPzXkI0Ck7jHmaBdDkfFtt6sm2MbHmyoyIulnaPkhZmNTjgQSylI3JF1mkGEgQD351O1kjcuKDyUi6CNgjjvU3uWlGANIb5Q3VFhxrJHDVOcSV1Gev8eQthT8dEBRxOi7ZDYF6J727FHZznBf2TljeT2EI+HBzWiM\\\\\\\/LF3JvWCavGxpKlca1l8Ar6mVFLVmzXAXbAKLhJKmiRhO8652gpXFw32GM9Z57W1nCj8KKbMarbQeRIFEr3l4I8cE39DNplr25fp1Ormc9m7+Dz65wQLkv13kPSubjG2OS1Sw9eIHB3kH1FQ7oIpWYauWk4bIy2eFLbHphIhIXSJgJ9uPOpfwP66tr\\\\\\\/XgVY6tFBkWVLGk0xGODq6QADzgqmyWPiavzBv7WDc4QTJy5SNrdqmQYPP5a\\\\\\\/WpEF+01wt5L+WRLkUe\\\\\\\/B+PrUrYwOgyoUhReIZO51PMwfXShD3G7Cv9eedzIDIi0UTAwYVsflPcJU11J8jVGIcFiemE7UpwcNWvX1qH3NoCzIUI3\\\\\\\/5EHbv2ML8AD8+62KjCPFHsQ7xjCbmy9valA4EOVQrtXEWlaev1jXZr5XHCsWLrpy\\\\\\\/oLsEpf95jxaW92\\\\\\\/lTHAesJXEIig7WeoOiKJlHfY74G4Rhf7lxD0pOOyRZL7iSNB+6cO++Pls5HPuw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022lWZ15jCSyLvtvrb069sLjg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00225037664d25248737aeac9756c69752cd97e4b7e58cb0af3478676dcc0b5af41c\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459370 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genomic-selection">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/12/1595" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-TCLUxq8Etq8D54YEH-U-DOlHyiTeHLGZeIpvrOLKkuc" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/12/1595" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-WXfpyXWn3eYgOkaD2ftGc3tYQr-hofcB4akQLgR9e0I" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/2/12/1585" title="Genetic Interactions Between Brassinosteroid-Inactivating P450s and Photomorphogenic Photoreceptors in Arabidopsis thaliana" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/2/12/1607" title="Natural Variation in the Yeast Glucose-Signaling Network Reveals a New Role for the Mig3p Transcription Factor" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459370" id="node459370" data-pisa="ggg;2/12/1595" data-pisa-master="ggg;2/12/1595" data-apath="/ggg/2/12/1595.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Comparison Between Linear and Non-parametric Regression Models for Genome-Enabled Prediction in Wheat</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Paulino Pérez-Rodríguez</span>, <span class="highwire-citation-author" data-delta="1">Daniel Gianola</span>, <span class="highwire-citation-author" data-delta="2">Juan Manuel González-Camacho</span>, <span class="highwire-citation-author" data-delta="3">José Crossa</span>, <span class="highwire-citation-author" data-delta="4">Yann Manès</span> and <span class="highwire-citation-author" data-delta="5">Susanne Dreisigacker</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1595-1605;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.003665 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459370" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Paulino Pérez-Rodríguez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Colegio de Postgraduados, Montecillo, Texcoco 56230, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Paulino%2BP%C3%A9rez-Rodr%C3%ADguez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=P%C3%A9rez-Rodr%C3%ADguez%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APaulino%2BP%25C3%25A9rez-Rodr%25C3%25ADguez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:perpdgo@gmail.com" class="" data-icon-position="" data-hide-link-title="0">perpdgo@gmail.com</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Daniel Gianola </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Departments of Animal Sciences, Dairy Science, and Biostatistics and Medical Informatics, University of Wisconsin-Madison, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BGianola%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gianola%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADaniel%2BGianola%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Juan Manuel González-Camacho </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Colegio de Postgraduados, Montecillo, Texcoco 56230, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Juan%2BManuel%2BGonz%C3%A1lez-Camacho%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gonz%C3%A1lez-Camacho%20JM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJuan%2BManuel%2BGonz%25C3%25A1lez-Camacho%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">José Crossa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Biometrics and Statistics Unit and Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), 06600 Mexico, D.F., México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jos%C3%A9%2BCrossa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crossa%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJos%25C3%25A9%2BCrossa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Yann Manès </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Biometrics and Statistics Unit and Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), 06600 Mexico, D.F., México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yann%2BMan%C3%A8s%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Man%C3%A8s%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYann%2BMan%25C3%25A8s%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Susanne Dreisigacker </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Biometrics and Statistics Unit and Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), 06600 Mexico, D.F., México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Susanne%2BDreisigacker%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dreisigacker%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASusanne%2BDreisigacker%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/2/12/1595" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459370" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459370/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/12/1595.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459370" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459370/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/12/1595.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459370" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459370/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/12/1595.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459370" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459370/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/2/12/1595.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459370" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459370/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">In genome-enabled prediction, parametric, semi-parametric, and non-parametric regression models have been used. This study assessed the predictive ability of linear and non-linear models using dense molecular markers. The linear models were linear on marker effects and included the Bayesian LASSO, Bayesian ridge regression, Bayes A, and Bayes B. The non-linear models (this refers to non-linearity on markers) were reproducing kernel Hilbert space (RKHS) regression, Bayesian regularized neural networks (BRNN), and radial basis function neural networks (RBFNN). These statistical models were compared using 306 elite wheat lines from CIMMYT genotyped with 1717 diversity array technology (DArT) markers and two traits, days to heading (DTH) and grain yield (GY), measured in each of 12 environments. It was found that the three non-linear models had better overall prediction accuracy than the linear regression specification. Results showed a consistent superiority of RKHS and RBFNN over the Bayesian LASSO, Bayesian ridge regression, Bayes A, and Bayes B models.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">GenPred</a></li><li class="kwd"><a href="/keyword/shared-data-resources" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resources" rel="nofollow">Shared data resources</a></li></ul><p id="p-4">Genome-enabled prediction of complex traits based on marker data are becoming important in plant and animal breeding, personalized medicine, and evolutionary biology (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Meuwissen <em>et al.</em> 2001</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Bernardo and Yu 2007</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">de los Campos <em>et al.</em> 2009</a>, <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">2010</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Crossa <em>et al.</em> 2010</a>, <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">2011</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Ober <em>et al.</em> 2012</a>). In the standard, infinitesimal, pedigree-based model of quantitative genetics, the family structure of a population is reflected in some expected resemblance between relatives. The latter is measured as an expected covariance matrix among individuals and is used to predict genetic values (<em>e.g.</em> <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Crossa <em>et al.</em> 2006</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Burgueño <em>et al.</em> 2007</a>, <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">2011</a>). Whereas pedigree-based models do not account for Mendelian segregation and the expected covariance matrix is constructed using assumptions that do not hold (<em>e.g.</em> absence of selection and mutation and random mating), the marker-based models allow tracing Mendelian segregation at several positions of the genome and observing realized (as opposed to expected) covariances. This enhances the potential for improving the accuracy of estimates of genetic values, thus increasing the genetic progress attainable when these predictions are used for selection purposes in lieu of pedigree-based predictions. Recently, <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">de los Campos <em>et al.</em> (2009</a>, <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">2010</a>) and <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Crossa <em>et al.</em> (2010</a>, <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">2011</a>) used Bayesian estimates from genomic parametric and semi-parametric regressions, and they found that models that incorporate pedigree and markers simultaneously had better prediction accuracy for several traits in wheat and maize than models based only on pedigree or only on markers.</p><p id="p-5">The standard linear genetic model represents the phenotypic response of the <em>i<sup>th</sup></em> individual (<span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-1.gif"/></noscript></span></span>) as the sum of a genetic value, <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-2.gif"/></noscript></span></span>, and of a model residual, <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-3.gif"/></noscript></span></span>, such that the linear model for <em>n</em> individuals <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-4.gif"/></noscript></span></span> is represented as <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-5.gif"/></noscript></span></span>. However, building predictive models for complex traits using a large number of molecular markers (<em>p</em>) with a set of lines comprising individuals (<em>n</em>) with <em>p ≫n</em> is challenging because individual marker effects are not likelihood-identified. In this case, marker effects can be estimated via penalized parametric or semi-parametric methods or their Bayesian counterparts, rather than via ordinary least squares. This reduces the mean-squared error of estimates; it also increases prediction accuracy of out-of-sample cases and prevents over-fitting (<a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">de los Campos <em>et al.</em> 2010</a>). In addition to the well-known Bayes A and B linear regression models originally proposed by <a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Meuwissen <em>et al.</em> (2001)</a> for incorporating marker effects into <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-6.gif"/></noscript></span></span>, there are several penalized parametric regression methods for estimating marker effects, such as ridge regression, the least absolute shrinkage and selection operator (LASSO), and the elastic net (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Hastie <em>et al.</em> 2009</a>). The Bayesian counterparts of these models have proved to be useful because appropriate priors can be assigned to the regularization parameter(s), and uncertainty in the estimations and predictions can be measured directly by applying the Bayesian paradigm.</p><p id="p-6">Regression methods assume a linear relationship between phenotype and genotype, and they typically account for additive allelic effects only; however, evidence of epistatic effects on plant traits is vast and well documented (<em>e.g.</em> <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Holland 2001</a>, <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">2008</a>). In wheat, for instance, detailed analyses have revealed a complex circuitry of epistatic interactions in the regulation of heading time involving different vernalization genes, day-length sensitivity genes, and earliness <em>per se</em> genes, as well as the environment (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Laurie <em>et al.</em> 1995</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Cockram <em>et al.</em> 2007</a>). Epistatic effects have also been found to be an important component of the genetic basis of plant height and bread-making quality traits (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Zhang <em>et al.</em> 2008</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Conti <em>et al.</em> 2011</a>). It is becoming common to study gene × gene interactions by using a paradigm of networks that includes aggregating gene × gene interaction that exists even in the absence of main effects (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">McKinney and Pajewski 2012</a>). Interactions between alleles at two or more loci could theoretically be represented in a linear model via use of appropriate contrasts. However, this does not scale when the number of markers (<em>p</em>) is large, as the number of 2-locus, 3-locus, <em>etc</em>., interactions is mind boggling.</p><p id="p-7">An alternative approach to the standard parametric modeling of complex interactions is provided by non-linear, semi-parametric methods, such as kernel-based models (<em>e.g.</em> <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Gianola <em>et al.</em> 2006</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Gianola and van Kaam 2008</a>) or artificial neural networks (NN) (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Okut <em>et al.</em> 2011</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Gianola <em>et al.</em> 2011</a>), under the assumption that such procedures can capture signals from high-order interactions. The potential of these methods, however, depends on the kernel chosen and on the neural network architecture. In a recent study, <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Heslot <em>et al.</em> (2012)</a> compared the predictive accuracy of several genome-enabled prediction models, including reproducing kernel Hilbert space (RKHS) and NN, using barley and wheat data; the authors found that the non-linear models gave a modest but consistent predictive superiority (as measured by correlations between predictions and realizations) over the linear models. In particular, the RKHS model had a better predictive ability than that obtained using the parametric regressions.</p><p id="p-8">The use of RKHS for predicting complex traits was first proposed by <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Gianola <em>et al.</em> (2006)</a> and <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Gianola and van Kaam (2008)</a>. <a id="xref-ref-13-4" class="xref-bibr" href="#ref-13">de los Campos <em>et al.</em> (2010)</a> further developed the theoretical basis of RHKS with “kernel averaging” (simultaneous use of various kernels in the model) and showed its good prediction accuracy. Other empirical studies in plants have corroborated the increase in prediction accuracy of kernel methods (<em>e.g.</em> <a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">Crossa <em>et al.</em> 2010</a>, <a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">2011</a>; <a id="xref-ref-13-5" class="xref-bibr" href="#ref-13">de los Campos <em>et al.</em> 2010</a>; <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Heslot <em>et al.</em> 2012</a>). Recently, <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Long <em>et al.</em> (2010)</a>, using chicken data, and <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">González-Camacho <em>et al.</em> (2012)</a>, using maize data, showed that NN methods provided prediction accuracy comparable to that obtained using the RKHS method. In NN, the bases functions (adaptive “covariates”) are inferred from the data, which gives the NN great potential and flexibility for capturing complex interactions between input variables (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Hastie <em>et al.</em> 2009</a>). In particular, Bayesian regularized neural networks (BRNN) and radial basis function neural networks (RBFNN) have features that make them attractive for use in genomic selection (GS).</p><p id="p-9">In this study, we examined the predictive ability of various linear and non-linear models, including the Bayes A and B linear regression models of <a id="xref-ref-36-3" class="xref-bibr" href="#ref-36">Meuwissen <em>et al.</em> (2001)</a>; the Bayesian LASSO, as in <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Park and Casella (2008)</a> and <a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">de los Campos <em>et al.</em> (2009)</a>; RKHS, using the “kernel averaging” strategy proposed by <a id="xref-ref-13-6" class="xref-bibr" href="#ref-13">de los Campos <em>et al.</em> (2010)</a>; the RBFNN, proposed and used by <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">González-Camacho <em>et al.</em> (2012)</a>; and the BRNN, as described by <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Neal (1996)</a> and used in the context of GS by <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Gianola <em>et al.</em> (2011)</a>. The predictive ability of these models was compared using a cross-validation scheme applied to a wheat data set from CIMMYT’s Global Wheat Program.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Experimental data</h3><p id="p-10">The data set included 306 elite wheat lines, 263 lines that are candidates for the 29<sup>th</sup> Semi-Arid Wheat Screening Nursery (SAWSN), and 43 lines from the 18<sup>th</sup> Semi-Arid Wheat Yield Trial (SAWYT) from CIMMYT’s Global Wheat Program. These lines were genotyped with 1717 diversity array technology (DArT) markers generated by Triticarte Pty. Ltd. (Canberra, Australia; <a href="http://www.triticarte.com.au">http://www.triticarte.com.au</a>). Two traits were analyzed: grain yield (GY) and days to heading (DTH) (see Supporting Information, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003665/-/DC1/FileS1.zip">File S1</a>).</p><p id="p-11">The traits were measured in a total of 12 different environments (1–12) (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>): GY in environments 1–7 and DTH in environments 1–5 and 8–12 (10 in all). Different agronomic practices were used. Yield trials were planted in 2009 and 2010 using prepared beds and flat plots under controlled drought or irrigated conditions. Yield data from experiments in 2010 were replicated, whereas data from trials in 2009 were adjusted means from an alpha lattice incomplete block design with adjustment for spatial variability in the direction of rows and columns using the autoregressive model fitted in both directions.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/462169/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/462169/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Twelve environments representing combinations of diverse agronomic management (drought or full irrigation, sowing in standard, bed, or flat systems), sites in Mexico, and years for two traits, grain yield (GY) and days to heading (DTH), with their broad-sense heritability (<em>h</em><sup>2</sup>) measured in 2010</span><div class="sb-div caption-clear"></div></div></div><p id="p-12">Data used to train the models for GY and DTH in 2009 were the best linear unbiased estimator (BLUE) after spatial analysis, whereas the BLUE data for 2010 were obtained after performing analyses in each of the 12 environments and combined. The experimental designs in each location consisted of alpha lattice incomplete block designs of different sizes, with two replicates each.</p><p id="p-13">Broad-sense heritability at individual environments was calculated as <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-7.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-8.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-9.gif"/></noscript></span></span> are the genotype and error variance components, respectively, and <em>nreps</em> is the number of replicates. For the combined analyses across environments, broad-sense heritability was calculated as <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-10.gif"/></noscript></span></span>, where the term <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-11.gif"/></noscript></span></span> is the genotype × environment interaction variance component, and <em>nenv</em> is the number of environments included in the analysis.</p></div><div id="sec-3" class="subsection"><h3>Statistical models</h3><p id="p-14">One method for incorporating markers is to define <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-12.gif"/></noscript></span></span> as a parametric linear regression on marker covariates <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-13.gif"/></noscript></span></span> with form <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-14.gif"/></noscript></span></span>, such that <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-15.gif"/></noscript></span></span> ( <em>j</em> = 1,2,…,<em>p</em> markers); here, <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-16.gif"/></noscript></span></span> is the partial regression of <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-17.gif"/></noscript></span></span> on the <em>j<sup>th</sup></em> marker covariate (<a id="xref-ref-36-4" class="xref-bibr" href="#ref-36">Meuwissen <em>et al.</em> 2001</a>). Extending the model to allow for an intercept<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-18.gif"/></noscript></span> <span class="disp-formula-label">(1)</span></span></p><p id="p-15">We adopted Gaussian assumptions for model residuals; specifically, the joint distribution of model residuals in <a id="xref-disp-formula-1-1" class="xref-disp-formula" href="#disp-formula-1">Equation 1</a> was assumed normal with mean zero and variance <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-19.gif"/></noscript></span></span>. The likelihood function is<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-20.gif"/></noscript></span> <span class="disp-formula-label">(2)</span></span>where <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-21.gif"/></noscript></span></span> is a normal density for random variable <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-22.gif"/></noscript></span></span> centered at <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-23.gif"/></noscript></span></span> and with variance <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-24.gif"/></noscript></span></span>. Depending on how priors on the marker effects are assigned, different Bayesian linear regression models result.</p></div><div id="sec-4" class="subsection"><h3>Linear models: Bayesian ridge regression, Bayesian LASSO, Bayes A, and Bayes B</h3><p id="p-16">A standard penalized regression method is ridge regression (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Hoerl and Kennard 1970</a>); its Bayesian counterpart, Bayesian ridge regression (BRR), uses a prior density of marker effects, <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-25.gif"/></noscript></span></span>, that is, Gaussian, centered at zero and with variance common to all the markers, that is, <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-26.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-27.gif"/></noscript></span></span> is a prior-variance of marker effects. Marker effects are assumed independent and identically distributed <em>a priori</em>. We assigned scaled inverse chi distributions <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-28.gif"/></noscript></span></span> to the variance parameters <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-29.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-30.gif"/></noscript></span></span>. The prior degrees of freedom parameters were set to <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-31.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-32.gif"/></noscript></span></span>. It can be shown that the posterior mean of marker effects is the best linear unbiased predictor (BLUP) of marker effects, so Bayesian ridge regression is often referred to as RR-BLUP (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">de los Campos <em>et al.</em> 2012</a>).</p><p id="p-17">The Bayesian LASSO, Bayes A, and Bayes B relax the assumption of common prior variance to all marker effects. The relationship among these three models is as follows: Bayes B can be considered as the most general of the three, in the sense that Bayes A and Bayesian ridge regression can be viewed as special cases of Bayes B. This is because Bayes A is obtained from Bayes B by setting <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-33.gif"/></noscript></span></span> (the proportion of markers with null effects), and Bayesian ridge regression is obtained from Bayes B by setting <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-34.gif"/></noscript></span></span> and assuming that all the markers have the same variance.</p><p id="p-18">Bayes B uses a mixture distribution with a mass at zero, such that the (conditional) prior distribution of marker effects is given by</p><span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-35.gif"/></noscript></span> <span class="disp-formula-label">(3)</span></span><p id="p-19">The prior assigned to <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-36.gif"/></noscript></span></span> is the same for all markers, <em>i.e.</em> a scaled inverted chi squared distribution <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-37.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-38.gif"/></noscript></span></span> are the degrees of freedom and <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-39.gif"/></noscript></span></span> is a scaling parameter. Bayes B becomes Bayes A by setting <em>π</em> = 0.</p><p id="p-20">In the case of Bayes B, we took <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-40.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-41.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-42.gif"/></noscript></span></span> with <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-43.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-44.gif"/></noscript></span></span> is the allele frequency for marker <em>j</em> and <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-45.gif"/></noscript></span></span> is the additive genetic variance explained by markers [see <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Habier <em>et al.</em> (2011)</a> and <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Resende <em>et al.</em> (2012)</a> for more details]. In the case of <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-46.gif"/></noscript></span></span>, we assigned a flat prior as in <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Wang <em>et al.</em> (1994)</a>.</p><p id="p-21">The Bayesian LASSO assigns a double exponential (DE) distribution to all marker effects (conditionally on a regularization parameter <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-47.gif"/></noscript></span></span>), centered at zero and with marker-specific variance, that is, <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-48.gif"/></noscript></span></span>. The DE distribution does not conjugate with the Gaussian likelihood, but it can be represented as a mixture of scaled normal densities, which allows easy implementation of the model (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Park and Casella 2008</a>; <a id="xref-ref-12-4" class="xref-bibr" href="#ref-12">de los Campos <em>et al.</em> 2009</a>). The priors used were exactly the same as those used in <a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">González-Camacho <em>et al.</em> (2012)</a>.</p><p id="p-22">The models used in this study, the Bayesian ridge regression, Bayesian LASSO (BL), Bayes A, and Bayes B, are explained in detail in several articles; for example, Bayes A and Bayes B are described in <a id="xref-ref-36-5" class="xref-bibr" href="#ref-36">Meuwissen <em>et al.</em> (2001)</a>, <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Habier <em>et al.</em> (2011)</a>, and <a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Resende <em>et al.</em> (2012)</a>, and an account of BL is given in <a id="xref-ref-12-5" class="xref-bibr" href="#ref-12">de los Campos <em>et al.</em> (2009</a>, <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">2012</a>), <a id="xref-ref-9-4" class="xref-bibr" href="#ref-9">Crossa <em>et al.</em> (2010</a>, <a id="xref-ref-10-4" class="xref-bibr" href="#ref-10">2011</a>), <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Perez <em>et al.</em> (2010)</a>, and <a id="xref-ref-20-4" class="xref-bibr" href="#ref-20">González-Camacho <em>et al.</em> (2012)</a>.</p></div><div id="sec-5" class="subsection"><h3>Non-linear models: RBFNN, BRNN, and RKHS</h3><p id="p-23">In this section, we describe the basic structure of the non-linear single hidden layer feed-forward neural network (SLNN) with two of its variants, the radial basis function neural network and the Bayesian regularized neural network. We also give a brief explanation of RKHS with the averaging kernel method at the end of this section.</p><div id="sec-6" class="subsection"><h4>Single hidden layer feed-forward neural network:</h4><p id="p-24">In a single-layer feed-forward (SLNN), the non-linear activation functions in the hidden layer enable a NN to have universal approximation ability, giving it great potential and flexibility in terms of capturing complex patterns. The structure of the SLNN is depicted in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>, which illustrates the structure of the method for a phenotypic continuous response. This NN can be thought of as a two-step regression (<em>e.g.</em> <a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">Hastie <em>et al.</em> 2009</a>). In the first step, in the non-linear hidden layer, <em>S</em> data-derived basis functions (<em>k</em> = 1, 2,…, <em>S</em> neurons), <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-49.gif"/></noscript></span></span>, are inferred, and in the second step, in the linear output layer, the response is regressed on the basis functions (inferred in the hidden layer). The inner product between the input vector and the weight vector (<span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-50.gif"/></noscript></span></span>) of each neuron of the hidden layer, plus a bias (intercept <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-51.gif"/></noscript></span></span>), is performed, that is, <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-52.gif"/></noscript></span></span> (<em>j</em> = 1,…,<em>p</em> markers); this is then transformed using a non-linear activation function <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-53.gif"/></noscript></span></span>. One obtains <span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-54.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-55.gif"/></noscript></span></span> is an intercept and (<em>β</em><sub>1</sub><sup>[</sup><em><sup>1</sup></em><sup>]</sup>, …, <em>β<sub>p</sub></em><sup>[</sup><em><sup>1</sup></em><sup>]</sup> ;…, <em>β</em><sub>1</sub><sup>[</sup><em><sup>S</sup></em><sup>]</sup>, …, <em>β<sub>p</sub></em><sup>[</sup><em><sup>S</sup></em><sup>]</sup>)′ is a vector of regression coefficients or “weights” of each neuron <em>k</em> in the hidden layer. The <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-56.gif"/></noscript></span></span> is the activation function, which maps the inputs into the real line in the closed interval [−1,1]; for example, <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-57.gif"/></noscript></span></span> is known as the tangent hyperbolic function. Finally, in the linear output layer, phenotypes are regressed on the data-derived features, <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-58.gif"/></noscript></span></span>, according to</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/12/1595/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Structure of a single-layer feed-forward neural network (SLNN) adapted from Gonz&#xE1;lez-Camacho et al. (2012). In the hidden layer, input variables  =  (j = 1,&#x2026;,p markers) are combined for each neuron (k=1,&#x2026;,S neurons) using a linear function, , and subsequently transformed using a non-linear activation function, yielding a set of inferred scores, . These scores are used in the output layer as basis functions to regress the response using the linear activation function on the data-derived predictors ." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-833070398" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Structure of a single-layer feed-forward neural network (SLNN) adapted from Gonz&#xE1;lez-Camacho &lt;em&gt;et al.&lt;/em&gt; (2012). In the hidden layer, input variables &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-59&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; = &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-60&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (&lt;em&gt;j = 1,&#x2026;,p&lt;/em&gt; markers) are combined for each neuron (&lt;em&gt;k=&lt;/em&gt;1,&#x2026;,&lt;em&gt;S&lt;/em&gt; neurons) using a linear function, &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-61&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, and subsequently transformed using a non-linear activation function, yielding a set of inferred scores, &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-62&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;. These scores are used in the output layer as basis functions to regress the response using the linear activation function on the data-derived predictors &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-63&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/12/1595/F1.medium.gif" width="440" height="363"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1&#x2002;" src="https://www.g3journal.org/content/ggg/2/12/1595/F1.medium.gif" width="440" height="363"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/12/1595/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/12/1595/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462201" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1 </span> <p id="p-25" class="first-child">Structure of a single-layer feed-forward neural network (SLNN) adapted from <a id="xref-ref-20-5" class="xref-bibr" href="#ref-20">González-Camacho <em>et al.</em> (2012)</a>. In the hidden layer, input variables <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/F1/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/F1/embed/mml-math-59.gif"/></noscript></span></span> = <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/F1/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/F1/embed/mml-math-60.gif"/></noscript></span></span> (<em>j = 1,…,p</em> markers) are combined for each neuron (<em>k=</em>1,…,<em>S</em> neurons) using a linear function, <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/F1/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/F1/embed/mml-math-61.gif"/></noscript></span></span>, and subsequently transformed using a non-linear activation function, yielding a set of inferred scores, <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/F1/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/F1/embed/mml-math-62.gif"/></noscript></span></span>. These scores are used in the output layer as basis functions to regress the response using the linear activation function on the data-derived predictors <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/F1/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/F1/embed/mml-math-63.gif"/></noscript></span></span>.</p><div class="sb-div caption-clear"></div></div></div><span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-64.gif"/></noscript></span><span class="disp-formula-label">(4)</span></span></div><div id="sec-7" class="subsection"><h4>Radial basis function neural network:</h4><p id="p-26">The RBFNN was first proposed by <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Broomhead and Lowe (1988)</a> and <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Poggio and Girosi (1990)</a>. <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> shows the architecture of a single hidden layer RBFNN with <em>S</em> non-linear neurons. Each non-linear neuron in the hidden layer has a Gaussian radial basis function (RBF) defined as <span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-65.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-66.gif"/></noscript></span></span> is the Euclidean norm between the input vector <span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-67.gif"/></noscript></span></span> and the center vector <span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-68.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-69.gif"/></noscript></span></span> is the bandwidth of the Gaussian RBF. Subsequently, in the linear output layer, phenotypes are regressed on the data-derived features, <span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-70.gif"/></noscript></span></span>, according to <span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-71.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-72.gif"/></noscript></span></span> is a model residual.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/12/1595/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Structure of a radial basis function neural network adapted from Gonz&#xE1;lez- Camacho et al. (2012). In the hidden layer, information from input variables  (j = 1,&#x2026;,p markers) is first summarized by means of the Euclidean distance between each of the input vectors  with respect to S (data-inferred) (k=1,&#x2026;,S neurons) centers , that is, . These distances are then transformed using the Gaussian function . These scores are used in the output layer as basis functions for the linear regression ." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-833070398" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Structure of a radial basis function neural network adapted from Gonz&#xE1;lez- Camacho &lt;em&gt;et al.&lt;/em&gt; (2012). In the hidden layer, information from input variables &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-73&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (&lt;em&gt;j = 1,&#x2026;,p&lt;/em&gt; markers) is first summarized by means of the Euclidean distance between each of the input vectors &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-74&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; with respect to &lt;em&gt;S&lt;/em&gt; (data-inferred) (&lt;em&gt;k=&lt;/em&gt;1,&#x2026;,&lt;em&gt;S&lt;/em&gt; neurons) centers &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-75&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, that is, &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-76&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;. These distances are then transformed using the Gaussian function &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-77&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;. These scores are used in the output layer as basis functions for the linear regression &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-78&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/12/1595/F2.medium.gif" width="440" height="344"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2&#x2002;" src="https://www.g3journal.org/content/ggg/2/12/1595/F2.medium.gif" width="440" height="344"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/12/1595/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/12/1595/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462111" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2 </span> <p id="p-27" class="first-child">Structure of a radial basis function neural network adapted from <a id="xref-ref-20-6" class="xref-bibr" href="#ref-20">González- Camacho <em>et al.</em> (2012)</a>. In the hidden layer, information from input variables <span class="inline-formula" id="inline-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/F2/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/F2/embed/mml-math-73.gif"/></noscript></span></span> (<em>j = 1,…,p</em> markers) is first summarized by means of the Euclidean distance between each of the input vectors <span class="inline-formula" id="inline-formula-70"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/F2/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/F2/embed/mml-math-74.gif"/></noscript></span></span> with respect to <em>S</em> (data-inferred) (<em>k=</em>1,…,<em>S</em> neurons) centers <span class="inline-formula" id="inline-formula-71"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/F2/embed/mml-math-75.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/F2/embed/mml-math-75.gif"/></noscript></span></span>, that is, <span class="inline-formula" id="inline-formula-72"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/F2/embed/mml-math-76.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/F2/embed/mml-math-76.gif"/></noscript></span></span>. These distances are then transformed using the Gaussian function <span class="inline-formula" id="inline-formula-73"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/F2/embed/mml-math-77.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/F2/embed/mml-math-77.gif"/></noscript></span></span>. These scores are used in the output layer as basis functions for the linear regression <span class="inline-formula" id="inline-formula-74"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/F2/embed/mml-math-78.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/F2/embed/mml-math-78.gif"/></noscript></span></span>.</p><div class="sb-div caption-clear"></div></div></div><div id="sec-8" class="subsection"><h5>Estimating the parameters of the RBFNN:</h5><p id="p-28">The vector of weights <span class="inline-formula" id="inline-formula-75"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-79.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-79.gif"/></noscript></span></span> of the linear output layer is obtained using the ordinary least-squares fit that minimizes the mean squared differences between the <span class="inline-formula" id="inline-formula-76"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-80.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-80.gif"/></noscript></span></span> (from RBFNN) and the observed responses <span class="inline-formula" id="inline-formula-77"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-81.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-81.gif"/></noscript></span></span>in the training set, provided that the Gaussian RBFs for centers <span class="inline-formula" id="inline-formula-78"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-82.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-82.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-79"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-83.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-83.gif"/></noscript></span></span> of the hidden layer are defined. The centers are selected using an orthogonalization least-squares learning algorithm, as described by <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Chen <em>et al.</em> (1991)</a> and implemented in Matlab 2010b. The centers are added iteratively such that each new selected center is orthogonal to the others. The selected centers maximize the decrease in the mean-squared error of the RBFNN, and the algorithm stops when the number of centers (neurons) added to the RBFNN attains a desired precision (<em>goal</em> error) or when the number of centers is equal to the number of input vectors, that is, when <em>S = n</em>. The bandwidth <span class="inline-formula" id="inline-formula-80"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-84.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-84.gif"/></noscript></span></span> of the Gaussian RBF is defined in terms of a design parameter of the net <em>spread</em>, that is, <span class="inline-formula" id="inline-formula-81"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-85.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-85.gif"/></noscript></span></span> for each Gaussian RBF of the hidden layer. To select the best RBFNN, a grid for training the net was generated, containing different values of <em>spread</em> and different precision values (<em>goal</em> error). The initial value of the <em>spread</em> was the median of the Euclidean distances between each pair of input vectors (<span class="inline-formula" id="inline-formula-82"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-86.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-86.gif"/></noscript></span></span>), and an initial value of 0.02 for the <em>goal</em> error was considered. The parameter <em>spread</em> allows adjusting the form of the Gaussian RBF such that it is sufficiently large to respond to overlapping regions of the input space but not so big that it might induce the Gaussian RBF to have a similar response.</p></div></div><div id="sec-9" class="subsection"><h4>Bayesian regularized neural networks:</h4><p id="p-29">The difference between SLNN and BRNN is in the function to be minimized (see the penalized function below); therefore, the basic structure of a BRNN can be represented in <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a> as well. The SLNN described above is flexible enough to approximate any non-linear function; this great flexibility allows NN to capture complex interactions among predictor variables (<a id="xref-ref-22-4" class="xref-bibr" href="#ref-22">Hastie <em>et al.</em> 2009</a>). However, this flexibility also leads to two important issues: (1) as the number of neurons increases, the number of parameters to be estimated also increases; and (2) as the number of parameters rises, the risk of over-fitting also increases. It is common practice to use penalized methods via Bayesian methods to prevent or palliate over-fitting.</p><p id="p-30"><a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">MacKay (1992</a>, <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">1994</a>) developed a framework for obtaining estimates of all the parameters in a feed-forward single neural network by using an empirical Bayes approach. Let <span class="inline-formula" id="inline-formula-83"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-87.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-87.gif"/></noscript></span></span>(<em>w</em><sub>1</sub>,…,<em>w<sub>S</sub></em>; <em>b</em><sub>1</sub>,…,<em>b<sub>S</sub></em> ; <em>β</em><sub>1</sub><sup>[1]</sup>, …, <em>β<sub>p</sub></em><sup>[1]</sup> ;…, <em>β</em><sub>1</sub><sup>[</sup><em><sup>S</sup></em><sup>]</sup>, …, <em>β<sub>p</sub></em><sup>[</sup><em><sup>S</sup></em><sup>]</sup>, <em>µ</em>)′ be the vector containing all the weights, biases, and connection strengths. The author showed that the estimation problem can be solved in two steps, followed by iteration:</p><ul class="list-simple " id="list-1"><li id="list-item-1"><p id="p-31"><span class="list-label">(1) </span>Obtain the conditional posterior modes of the elements in <span class="inline-formula" id="inline-formula-84"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-88.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-88.gif"/></noscript></span></span> assuming that the variance components <span class="inline-formula" id="inline-formula-85"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-89.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-89.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-86"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-90.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-90.gif"/></noscript></span></span> are known and that the prior distribution for the all the elements in <span class="inline-formula" id="inline-formula-87"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-91.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-91.gif"/></noscript></span></span> is given by <span class="inline-formula" id="inline-formula-88"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-92.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-92.gif"/></noscript></span></span>. It is important to note that this approach assigns the same prior to all elements of <span class="inline-formula" id="inline-formula-89"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-93.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-93.gif"/></noscript></span></span>, even though this may not always be the best thing to do. The density of the conditional (given the variance parameters) posterior distribution of the elements of <span class="inline-formula" id="inline-formula-90"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-94.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-94.gif"/></noscript></span></span>, according to Bayes’ theorem, is given by<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-95.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-95.gif"/></noscript></span> <span class="disp-formula-label">(5)</span></span></p></li></ul><p id="p-32">The conditional modes can be obtained by maximizing <a id="xref-disp-formula-5-1" class="xref-disp-formula" href="#disp-formula-5">Equation 5</a> over <span class="inline-formula" id="inline-formula-91"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-96.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-96.gif"/></noscript></span></span>. However, the problem is equivalent to minimizing the following penalized sum of squares [see <a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">Gianola <em>et al.</em> (2011)</a> for more details]<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-97.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-97.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-92"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-98.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-98.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-93"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-99.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-99.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-94"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-100.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-100.gif"/></noscript></span></span> is the difference between observed and predicted phenotypes for the fitted model, and <span class="inline-formula" id="inline-formula-95"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-101.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-101.gif"/></noscript></span></span> (<span class="inline-formula" id="inline-formula-96"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-102.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-102.gif"/></noscript></span></span>) is the <em>j<sup>th</sup></em> element of vector <span class="inline-formula" id="inline-formula-97"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-103.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-103.gif"/></noscript></span></span>.</p><ul class="list-simple " id="list-2"><li id="list-item-2"><p id="p-33"><span class="list-label">(2) </span>Update <span class="inline-formula" id="inline-formula-98"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-104.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-104.gif"/></noscript></span></span> and<span class="inline-formula" id="inline-formula-99"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-105.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-105.gif"/></noscript></span></span>. The updating formulas are obtained by maximizing an approximation to the marginal likelihood of the data <span class="inline-formula" id="inline-formula-100"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-106.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-106.gif"/></noscript></span></span> (the “evidence”) given by the denominator of <a id="xref-disp-formula-5-2" class="xref-disp-formula" href="#disp-formula-5">Equation 5</a>.</p></li><li id="list-item-3"><p id="p-34"><span class="list-label">(3) </span>Iterate between (1) and (2) until convergence.</p></li></ul><p id="p-35">The original algorithm developed by MacKay was further improved by <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Foresee and Hagan (1997)</a> and adopted by <a id="xref-ref-18-4" class="xref-bibr" href="#ref-18">Gianola <em>et al.</em> (2011)</a> in the context of genome and pedigree-enabled prediction. The algorithm is equivalent to estimation via maximum penalized likelihood estimation when “weight decay” is used, but it has the advantage of providing a way of setting the extent of “weight decay” through the variance component<span class="inline-formula" id="inline-formula-101"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-107.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-107.gif"/></noscript></span></span>. <a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Neal (1996)</a> pointed out that the procedure of <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">MacKay (1992</a>, <a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">1994</a>) can be further generalized. For example, there is no need to approximate probabilities via Gaussian assumptions; furthermore, it is possible to estimate the entire posterior distributions of all the elements in <span class="inline-formula" id="inline-formula-102"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-108.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-108.gif"/></noscript></span></span>, not only their (conditional) posterior modes. Next, we briefly review Neal’s approach to solving the problem; a comprehensive revision can be found in <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Lampinen and Vehtari (2001)</a>.</p><div id="sec-10" class="subsection"><h5>Prior distributions:</h5></div><div id="sec-11" class="subsection"><h5>a) Variance component of the residuals:</h5><p id="p-36"><a id="xref-ref-37-3" class="xref-bibr" href="#ref-37">Neal (1996)</a> used a conjugate inverse Gamma distribution as a prior for the variance associated with the residual, <span class="inline-formula" id="inline-formula-103"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-109.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-109.gif"/></noscript></span></span>, given in <a id="xref-disp-formula-4-1" class="xref-disp-formula" href="#disp-formula-4">Equation 4</a>, that is, <span class="inline-formula" id="inline-formula-104"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-110.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-110.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-105"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-111.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-111.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-106"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-112.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-112.gif"/></noscript></span></span> are the scale and degrees of freedom parameters, respectively. These parameters can be set to the default values given by <a id="xref-ref-37-4" class="xref-bibr" href="#ref-37">Neal (1996)</a>, <span class="inline-formula" id="inline-formula-107"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-113.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-113.gif"/></noscript></span></span>=0.05, <span class="inline-formula" id="inline-formula-108"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-114.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-114.gif"/></noscript></span></span>=0.5. These values were also used by <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Lampinen and Vehtari (2001)</a>.</p></div><div id="sec-12" class="subsection"><h5>b) Connection strengths, weights, and biases:</h5><p id="p-37"><a id="xref-ref-37-5" class="xref-bibr" href="#ref-37">Neal (1996)</a> suggested dividing the network parameters in <span class="inline-formula" id="inline-formula-109"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-115.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-115.gif"/></noscript></span></span> into groups and then using hierarchical models for each group of parameters; for example, connection strengths (<em>β</em><sub>1</sub><sup>[1]</sup>, …, <em>β<sub>p</sub></em><sup>[</sup><em><sup>1</sup></em><sup>]</sup> ;…; <em>β<sub>1</sub></em><sup>[</sup><em><sup>S</sup></em><sup>]</sup>, …, <em>β<sub>p</sub></em><sup>[</sup><em><sup>S</sup></em><sup>]</sup>), biases (<em>b</em><sub>1</sub>,…,<em>b<sub>S</sub></em>) of the hidden layer, and output weights (<em>w<sub>1</sub></em>,…,<em>w<sub>S</sub></em>), and general mean or bias (<em>µ</em>) of the linear output layer. Suppose that <em>u</em><sub>1</sub>,…,<em>u<sub>k</sub></em> are parameters of a given group; then assume</p><span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-116.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-116.gif"/></noscript></span></span><p id="p-38">And, at the last stage of the model, assign the prior <span class="inline-formula" id="inline-formula-110"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-117.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-117.gif"/></noscript></span></span>. The scale parameter of the distribution associated with the group of parameters containing the connection strengths (<em>β</em><sub>1</sub><em><sup>[</sup></em><sup>1]</sup>, …, <em>β<sub>p</sub></em><sup>[1]</sup> ;…; <em>β</em><sub>1</sub><sup>[</sup><em><sup>S</sup></em><sup>]</sup>, …, <em>β<sub>p</sub></em><sup>[</sup><em><sup>S</sup></em><sup>]</sup>) changes according to the number of inputs, in this case, <span class="inline-formula" id="inline-formula-111"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-118.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-118.gif"/></noscript></span></span> with <span class="inline-formula" id="inline-formula-112"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-119.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-119.gif"/></noscript></span></span> and <em>p</em> is the number of markers in the data set.</p><p id="p-39">By using Markov chain Monte Carlo (MCMC) techniques through an algorithm called hybrid Monte Carlo, <a id="xref-ref-37-6" class="xref-bibr" href="#ref-37">Neal (1996)</a> developed a software termed flexible Bayesian modeling (FBM) capable of obtaining samples from the posterior distributions of all unknowns in a neural network (as in <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>).</p></div></div><div id="sec-13" class="subsection"><h4>Reproducing kernel Hilbert spaces regression:</h4><p id="p-40">RKHS models have been suggested as an alternative to multiple linear regression for capturing complex interaction patterns that may be difficult to account for in a linear model framework (<a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">Gianola <em>et al.</em> 2006</a>). In RKHS model, the regression function takes the form<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-120.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-120.gif"/></noscript></span> <span class="disp-formula-label">(6)</span></span>where <span class="inline-formula" id="inline-formula-113"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-121.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-121.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-114"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-122.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-122.gif"/></noscript></span></span> are input vectors of marker genotypes in individuals <em>i</em> and <em>i′</em>; <span class="inline-formula" id="inline-formula-115"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-123.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-123.gif"/></noscript></span></span> are regression coefficients; and <span class="inline-formula" id="inline-formula-116"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-124.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-124.gif"/></noscript></span></span> is the reproducing kernel defined (here) with a Gaussian RBF, where <span class="inline-formula" id="inline-formula-117"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-125.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-125.gif"/></noscript></span></span> is a bandwidth parameter and <span class="inline-formula" id="inline-formula-118"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-126.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-126.gif"/></noscript></span></span> is the Euclidean norm between each pair of input vectors. The strategy termed “kernel averaging” for selecting optimal values of <span class="inline-formula" id="inline-formula-119"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-127.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-127.gif"/></noscript></span></span> within a set of candidate values was implemented using the Bayesian approach described in <a id="xref-ref-13-7" class="xref-bibr" href="#ref-13">de los Campos <em>et al.</em> (2010)</a>. Similarities and connections between the RKHS and the RBFNN are given in <a id="xref-ref-20-7" class="xref-bibr" href="#ref-20">González-Camacho <em>et al.</em> (2012)</a>.</p></div></div><div id="sec-14" class="subsection"><h3>Assessment of the models’ predictive ability</h3><p id="p-41">The predictive ability of the models given above was compared using Pearson’s correlation and predictive mean-squared error (PMSE) using predicted and realized values. A total of 50 random partitions were generated for each of the data sets, and each partition randomly assigned 90% of the lines to the training set and the remaining 10% to the validation set. The partition scheme used was similar to that in <a id="xref-ref-18-5" class="xref-bibr" href="#ref-18">Gianola <em>et al.</em> (2011)</a> and <a id="xref-ref-20-8" class="xref-bibr" href="#ref-20">González-Camacho <em>et al.</em> (2012)</a>.</p><p id="p-42">All scripts were run in a Linux work station; for Bayesian ridge regression and Bayesian LASSO, we used the R package BLR (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">de los Campos and Perez 2010</a>), whereas for RKHS, we used the R implementation described in <a id="xref-ref-13-8" class="xref-bibr" href="#ref-13">de los Campos <em>et al.</em> (2010)</a>, which was kindly provided by the authors. In the case of Bayes A and Bayes B, we used a program described by <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Hickey and Tier (2009)</a>, which is freely available at <a href="http://sites.google.com/site/hickeyjohn/alphabayes">http://sites.google.com/site/hickeyjohn/alphabayes</a>. For the BRNN, we used the FMB software available at <a href="http://www.cs.toronto.edu/~radford/fbm.software.html">http://www.cs.toronto.edu/∼radford/fbm.software.html</a>. Because the computational time required to evaluate the predictive ability of the BRNN network was great, we used the Condor high throughput computing system at the University of Wisconsin-Madison (<a href="http://research.cs.wisc.edu/condor">http://research.cs.wisc.edu/condor</a>). The RBFNN model was run using Matlab 2010b for Linux. The differences in computing times between the models were great. The computing times for evaluating the prediction ability of the 50 partitions for each trait were as follows, 10 min for RBFNN, 1.5 hr for RKHS, 3 hr for BRR, 3.5 hr for BL, 4.5 hr for Bayes B, 5.5 hr for Bayes A, and 30 days for BRNN. In the case of RKHS, BRR, BL, Bayes A, and Bayes B, inferences were based on 35,000 MCMC samples, and on 10,000 samples for BRNN. The estimated computing times were obtained using, as reference, a single Intel Xeon CPU 5330 2.4 GHz and 8 Gb of RAM memory. Significant reduction in computing time was achieved by parallelizing the tasks.</p></div></div><div class="section results" id="sec-15"><h2 class="">RESULTS</h2><p id="p-43">Data from replicated experiments in 2010 were used to calculate the broad-sense heritability for each trait in each environment (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). Broad-sense heritability across locations for 2010 data were 0.67 for GY and 0.92 for DTH. These high estimates can be explained, at least in part, by the strict environmental control of trials conducted at CIMMYT’s experiment station at Ciudad Obregon. The heritability of the two traits for 2009 was not estimated because the only available phenotypic data were adjusted means for each environment.</p><div id="sec-16" class="subsection"><h3>Predictive assessment of the models</h3><p id="p-44">The predictive ability of the different models for GY and DTH varied among the 12 environments. The model deemed best using correlations (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>) tended to be the one with the smallest average PMSE (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). The three non-parametric models had higher predictive correlations and smaller PMSE than the linear models for both GY and DTH. Within the linear models, the results are mixed, and all models gave similar predictions. Within the non-parametric models, RBFNN and RKHS always gave higher correlations between predicted values and realized phenotypes, and a smaller average PMSE than the BRNN. The mean of the correlations and the associated standard errors can be used to test for statistically significant improvements in the predictability of the non-linear models <em>vs.</em> the linear models. The <em>t</em>-test (with <span class="inline-formula" id="inline-formula-120"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-128.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/12/1595/embed/mml-math-128.gif"/></noscript></span></span>) showed that RKHS gave significant improvements in prediction in 13/19 cases (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>) compared with the BL, whereas RBFNN was significantly better than the BL in 10/19 cases. Similar results were obtained when comparing RKHS and RBFNN with Bayes A and Bayes B.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/462096/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/462096/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Average correlation (SE in parentheses) between observed and predicted values for grain yield (GY) and days to heading (DTH) in 12 environments for seven models</span><div class="sb-div caption-clear"></div></div></div><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/462088/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/462088/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Predictive mean- squared error (PMSE) between observed and predicted values for grain yield (GY) and days to heading (DTH) in 12 environments for seven models</span><div class="sb-div caption-clear"></div></div></div><p id="p-47">Correlations between observed and predicted values for DTH were lowest overall in environments 4 and 8, in Cd. Obregon, 2009, and in Toluca, 2009. Average PMSE was in agreement with the findings based on correlations. Although accuracies in environment 4 were much lower than in other environments, the higher accuracy of the non-parametric models (RKHS, RBFNN, and BRNN) over that of the linear models (BL, BRR, Bayes A, and Bayes B) was consistent with what was observed in the other environments. <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figures 3</a> and <a id="xref-fig-4-1" class="xref-fig" href="#F4">4</a> give scatter plots of the correlations obtained with the three non-parametric models <em>vs.</em> the BL for DTH and GY, respectively; each circle represents the estimated correlations for each of the two models included in the plot. In <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3, A–C</a>, DTH had a total of 500 points (10 environments and 50 random training-testing partitions). In <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4, A–C</a>, GY had a total of 350 points (7 environments and 50 random partitions in each environment). A point above the 45-degree line represents an analysis where the method whose predictive correlation is given on the vertical axis (RKHS, RBFNN, BRNN) outperformed the one whose correlation is given on the horizontal axis (BL). Both figures show that although there is a great deal of variability due to partition, for both DTH and GY, the overall superiority of RKHS and RBFNN over the linear model BL is clear. For both traits, BL had slightly better prediction accuracy than the BRNN in terms of the number of individual correlation points. It is interesting to note that some cross-validation partitions picked subsets of training data that had negative, zero, or very low correlations with the observed values in the validation set. These results indicate that lines in the training set are not necessarily related to those in the validation set.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/12/1595/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Plots of the predictive correlation for each of 50 cross-validation partitions and 10 environments for days to heading (DTH) in different combinations of models. (A) When the best non-parametric model is RKHS, this is represented by an open circle; when the best linear model is BL, this is represented by a filled circle. (B) When the best non-parametric model is RBFNN, this is represented by an open circle; when the best linear model is BL, this is represented by a filled circle. (C) When the best non-parametric model is BRNN, this is represented by an open circle; when the best linear model is BL, this is represented by a filled circle. The histograms depict the distribution of the correlations in the testing set obtained from the 50 partitions for different models. The horizontal (vertical) dashed line represents the average of the correlations for the testing set in the 50 partitions for the model shown on the Y (X) axis. The solid line represents Y = X; i.e. both models have the same prediction ability." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-833070398" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Plots of the predictive correlation for each of 50 cross-validation partitions and 10 environments for days to heading (DTH) in different combinations of models. (A) When the best non-parametric model is RKHS, this is represented by an open circle; when the best linear model is BL, this is represented by a filled circle. (B) When the best non-parametric model is RBFNN, this is represented by an open circle; when the best linear model is BL, this is represented by a filled circle. (C) When the best non-parametric model is BRNN, this is represented by an open circle; when the best linear model is BL, this is represented by a filled circle. The histograms depict the distribution of the correlations in the testing set obtained from the 50 partitions for different models. The horizontal (vertical) dashed line represents the average of the correlations for the testing set in the 50 partitions for the model shown on the Y (X) axis. The solid line represents Y = X; &lt;em&gt;i.e.&lt;/em&gt; both models have the same prediction ability.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/12/1595/F3.medium.gif" width="440" height="439"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3&#x2002;" src="https://www.g3journal.org/content/ggg/2/12/1595/F3.medium.gif" width="440" height="439"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/12/1595/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/12/1595/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462147" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3 </span> <p id="p-48" class="first-child">Plots of the predictive correlation for each of 50 cross-validation partitions and 10 environments for days to heading (DTH) in different combinations of models. (A) When the best non-parametric model is RKHS, this is represented by an open circle; when the best linear model is BL, this is represented by a filled circle. (B) When the best non-parametric model is RBFNN, this is represented by an open circle; when the best linear model is BL, this is represented by a filled circle. (C) When the best non-parametric model is BRNN, this is represented by an open circle; when the best linear model is BL, this is represented by a filled circle. The histograms depict the distribution of the correlations in the testing set obtained from the 50 partitions for different models. The horizontal (vertical) dashed line represents the average of the correlations for the testing set in the 50 partitions for the model shown on the Y (X) axis. The solid line represents Y = X; <em>i.e.</em> both models have the same prediction ability.</p><div class="sb-div caption-clear"></div></div></div><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/12/1595/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Plot of the correlation for each of 50 cross-validation partitions and seven environments for grain yield (GY) in different combinations of models. (A) When the best model is RKHS, this is represented by an open circle; when the best model is BL, this is represented by a filled circle. (B) When best model is RBFNN, this is represented by an open circle; when the best model is BL, this is represented by a filled circle. (C) When the best model is BRNN, this is represented by an open circle; when the best model is BL, this is represented by a filled circle. The histograms depict the distribution of the correlations in the testing set obtained from the 50 partitions for different models. The horizontal (vertical) dashed line represents the average of the correlations for the testing set in the 50 partitions for the model shown on the Y (X) axis. The solid line represents Y = X; i.e. both models have the same prediction ability." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-833070398" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Plot of the correlation for each of 50 cross-validation partitions and seven environments for grain yield (GY) in different combinations of models. (A) When the best model is RKHS, this is represented by an open circle; when the best model is BL, this is represented by a filled circle. (B) When best model is RBFNN, this is represented by an open circle; when the best model is BL, this is represented by a filled circle. (C) When the best model is BRNN, this is represented by an open circle; when the best model is BL, this is represented by a filled circle. The histograms depict the distribution of the correlations in the testing set obtained from the 50 partitions for different models. The horizontal (vertical) dashed line represents the average of the correlations for the testing set in the 50 partitions for the model shown on the Y (X) axis. The solid line represents Y = X; &lt;em&gt;i.e.&lt;/em&gt; both models have the same prediction ability.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/12/1595/F4.medium.gif" width="440" height="429"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4&#x2002;" src="https://www.g3journal.org/content/ggg/2/12/1595/F4.medium.gif" width="440" height="429"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/12/1595/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/12/1595/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462142" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4 </span> <p id="p-49" class="first-child">Plot of the correlation for each of 50 cross-validation partitions and seven environments for grain yield (GY) in different combinations of models. (A) When the best model is RKHS, this is represented by an open circle; when the best model is BL, this is represented by a filled circle. (B) When best model is RBFNN, this is represented by an open circle; when the best model is BL, this is represented by a filled circle. (C) When the best model is BRNN, this is represented by an open circle; when the best model is BL, this is represented by a filled circle. The histograms depict the distribution of the correlations in the testing set obtained from the 50 partitions for different models. The horizontal (vertical) dashed line represents the average of the correlations for the testing set in the 50 partitions for the model shown on the Y (X) axis. The solid line represents Y = X; <em>i.e.</em> both models have the same prediction ability.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion-conclusions" id="sec-17"><h2 class="">DISCUSSION AND CONCLUSIONS</h2><p id="p-50">Understanding the impact of epistasis on quantitative traits remains a major challenge. In wheat, several studies have reported significant epistasis for grain yield and heading or flowering time (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Goldringer <em>et al.</em> 1997</a>). Detailed analyses have shown that vernalization, day-length sensitivity, and earliness <em>per se</em> genes are mainly responsible for regulating heading time. The vernalization requirement relates to the sensitivity of the plant to cold temperatures, which causes it to accelerate spike primordial formation. Transgenic and mutant analyses, for example, have suggested a pathway involving epistatic interactions that combines environment-induced suppression and upregulation of several genes, leading to final floral transition (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Shimada <em>et al.</em> 2009</a>).</p><p id="p-51">There is evidence that the aggregation of multiple gene × gene interactions (epistasis) with small effects into small epistatic networks is important for explaining the heritability of complex traits in genome-wide association studies (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">McKinney and Pajewski 2012</a>). Epistatic networks and gene × gene interactions can also be exploited for GS via suitable statistical-genetic models that incorporate network complexities. Evidence from this study, as well as from other research involving other plant and animal species, suggests that models that are non-linear in input variables (<em>e.g.</em> SNPs) predict outcomes in testing sets better than standard linear regression models for genome-enabled prediction. However, it should be pointed out that better predictive ability can have several causes, one of them the ability of some non-linear models to capture epistatic effects. Furthermore, the random cross-validation scheme used in this study was not designed to specifically assess epistasis but rather to compare the models’ predictive ability.</p><p id="p-52">It is interesting to compare results from different predictive machineries when applied to either maize or wheat. Differences in the prediction accuracy of non-parametric and linear models (at least for the data sets included in this and other studies) seem to be more pronounced in wheat than in maize. Although differences depend, among other factors, on the trait-environment combination and the number of markers, it is clear from <a id="xref-ref-20-9" class="xref-bibr" href="#ref-20">González-Camacho <em>et al.</em> (2012)</a> that for flowering traits (highly additive) and traits such as grain yield (additive and epistatic) in maize, the BL model performed very similarly to the RKHS and RBFNN. On the other hand, in the present study, which involves wheat, the RKHS, RBFNN, and BRNN models clearly had a markedly better predictive accuracy than BL, BRR, Bayes A, or Bayes B. This may be due to the fact that, in wheat, additive × additive epistasis plays an important role in grain yield, as found by <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Crossa <em>et al.</em> (2006)</a> and <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Burgueño <em>et al.</em> (2007</a>, <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">2011</a>) when assessing additive, additive × additive, additive × environment, and additive × additive × environment interactions using a pedigree-based model with the relationship matrix <strong>A</strong>.</p><p id="p-53">As pointed out first by <a id="xref-ref-17-4" class="xref-bibr" href="#ref-17">Gianola <em>et al.</em> (2006)</a> and subsequently by <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Long <em>et al.</em> (2010)</a>, non-parametric models do not impose strong assumptions on the phenotype-genotype relationship, and they have the potential of capturing interactions among loci. Our results with real wheat data sets agreed with previous findings in animal and plant breeding and with simulated experiments, in that a non-parametric treatment of markers may account for epistatic effects that are not captured by linear additive regression models. Using extensive maize data sets, <a id="xref-ref-20-10" class="xref-bibr" href="#ref-20">González-Camacho <em>et al.</em> (2012)</a> found that RBFNN and RKHS had some similarities and seemed to be useful for predicting quantitative traits with different complex underlying gene action under varying types of interaction in different environmental conditions. These authors suggested that it is possible to make further improvements in the accuracy of the RKHS and RBFNN models by introducing differential weights in SNPs, as shown by <a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">Long <em>et al.</em> (2010)</a> for RBFs.</p><p id="p-54">The training population used here was not developed specifically for this study; it was made up of a set of elite lines from the CIMMYT rain-fed spring wheat breeding program. Our results show that it is possible to achieve good predictions of line performance by combining phenotypic and genotypic data generated on elite lines. As genotyping costs decrease, breeding programs could make use of genome-enabled prediction models to predict the values of new breeding lines generated from crosses between elite lines in the training set before they reach the yield testing stage. Lines with the highest estimated breeding values could be intercrossed before being phenotyped. Such a “rapid cycling” scheme would accelerate the fixation rate of favorable alleles in elite materials and should increase the genetic gain per unit of time, as described by <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Heffner <em>et al.</em> (2009)</a>.</p><p id="p-55">It is important to point out that proof-of-concept experiments are required before genome-enabled selection can be implemented successfully in plant breeding programs. It is necessary to test genomic predictions on breeding materials derived from crosses between lines of the training population. If predictions are reliable enough, an experiment using the same set of parental materials could be carried out to compare the field performance of lines coming from a genomic-assisted recurrent selection program scheme <em>vs.</em> lines coming from a conventional breeding scheme. The accuracies reported in this study represent prediction of wheat lines using a training set comprising lines with some degree of relatedness to lines in the validation set. When the validation and the training sets are not genetically related (unrelated families) or represent populations with different genetic structures and different linkage disequilibrium patterns, then negligible accuracies are to be expected. It seems that successful application of genomic selection in plant breeding requires some genetic relatedness between individuals in the training and validation sets, and that linkage disequilibrium information <em>per se</em> does not suffice (<em>e.g.</em> <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Makowsky <em>et al.</em> 2011</a>).</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-56">Financial support by the Wisconsin Agriculture Experiment Station and the AVIAGEN, Ltd. (Newbridge, Scotland) to Paulino Pérez and Daniel Gianola is acknowledged. We thank the Centro Internacional de Mejoramiento de Maíz y Trigo (CIMMYT) researchers who carried out the wheat trials and provided the phenotypic data analyzed in this article.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003665/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003665/-/DC1</a></p></li><li class="fn" id="fn-4"><p id="p-57"><em>Communicating editor: J. B. Holland</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2012-07-09"><span class="received-label">Received </span>July 9, 2012.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2012-10-05"><span class="accepted-label">Accepted </span>October 5, 2012.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2012 Pérez-Rodríguez <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-2.12.1595.1" data-doi="10.2135/cropsci2006.11.0690"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Prospects for genome-wide selection for quantitative traits in maize</span>. <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">47</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">1082</span>–<span class="cit-lpage">1090</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DBernardo%26rft.auinit1%253DR.%26rft.volume%253D47%26rft.issue%253D3%26rft.spage%253D1082%26rft.epage%253D1090%26rft.atitle%253DProspects%2Bfor%2BGenomewide%2BSelection%2Bfor%2BQuantitative%2BTraits%2Bin%2BMaize%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2006.11.0690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2006.11.0690&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000247424200020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-2.12.1595.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Broomhead</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lowe</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">Multivariable functional interpolation and adaptive networks</span>. <abbr class="cit-jnl-abbrev">Complex Systems</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">321</span>–<span class="cit-lpage">355</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DComplex%2BSystems%26rft.volume%253D2%26rft.spage%253D321%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-2.12.1595.3" data-doi="10.2135/cropsci2006.09.0564"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burgueño</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cornelius</span>  <span class="cit-name-given-names">P. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trethowan</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McLaren</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Modeling additive × environment and additive × additive × environment using genetic covariances of relatives of wheat genotypes</span>. <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">47</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">311</span>–<span class="cit-lpage">320</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DBurgueno%26rft.auinit1%253DJ.%26rft.volume%253D47%26rft.issue%253D1%26rft.spage%253D311%26rft.epage%253D320%26rft.atitle%253DModeling%2BAdditive%2Bx%2BEnvironment%2Band%2BAdditive%2Bx%2BAdditive%2Bx%2BEnvironment%2BUsing%2BGenetic%2BCovariances%2Bof%2BRelatives%2Bof%2BWheat%2BGenotypes%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2006.09.0564%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2006.09.0564&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000244430100039&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-2.12.1595.4" data-doi="10.2135/cropsci2010.07.0403"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burgueño</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cotes</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">San Vicente</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Das</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Prediction assessment of linear mixed models for multienvironment trials</span>. <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">51</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">944</span>–<span class="cit-lpage">954</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D51%26rft.spage%253D944%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2010.07.0403%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2010.07.0403&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-2.12.1595.5" data-doi="10.1109/72.80341"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cowan</span>  <span class="cit-name-given-names">C. F. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grant</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Orthogonal least squares learning algorithm for radial basis function networks</span>. <abbr class="cit-jnl-abbrev">Neural Networks, IEEE Transactions on</abbr> <span class="cit-vol">2</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">302</span>–<span class="cit-lpage">309</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNeural%2BNetworks%252C%2BIEEE%2BTransactions%2Bon%26rft.volume%253D2%26rft.spage%253D302%26rft_id%253Dinfo%253Adoi%252F10.1109%252F72.80341%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1109/72.80341&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-2.12.1595.6" data-doi="10.1093/jxb/erm042"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cockram</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leigh</span>  <span class="cit-name-given-names">F. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Sullivan</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Powell</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Control of flowering time in temperate cereals: genes, domestication, and sustainable productivity</span>. <abbr class="cit-jnl-abbrev">J. Exp. Bot.</abbr> <span class="cit-vol">58</span>(<span class="cit-issue">6</span>): <span class="cit-fpage">1231</span>–<span class="cit-lpage">1244</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BExp.%2BBot.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjxb%252Ferm042%26rft_id%253Dinfo%253Apmid%252F17420173%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamV4Ym90IjtzOjU6InJlc2lkIjtzOjk6IjU4LzYvMTIzMSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzIvMTIvMTU5NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-2.12.1595.7" data-doi="10.1007/s13353-011-0045-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Conti</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roncallo</span>  <span class="cit-name-given-names">P. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beaufort</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cervigni</span>  <span class="cit-name-given-names">G. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miranda</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Mapping of main and epistatic effect QTLs associated to grain protein and gluten strength using a RIL population of durum wheat</span>. <abbr class="cit-jnl-abbrev">J. Appl. Genet.</abbr> <span class="cit-vol">52</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">287</span>–<span class="cit-lpage">298</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bapplied%2Bgenetics%26rft.stitle%253DJ%2BAppl%2BGenet%26rft.aulast%253DConti%26rft.auinit1%253DV.%26rft.volume%253D52%26rft.issue%253D3%26rft.spage%253D287%26rft.epage%253D298%26rft.atitle%253DMapping%2Bof%2Bmain%2Band%2Bepistatic%2Beffect%2BQTLs%2Bassociated%2Bto%2Bgrain%2Bprotein%2Band%2Bgluten%2Bstrength%2Busing%2Ba%2BRIL%2Bpopulation%2Bof%2Bdurum%2Bwheat.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs13353-011-0045-1%26rft_id%253Dinfo%253Apmid%252F21523429%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s13353-011-0045-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21523429&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F12%2F1595.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-2.12.1595.8" data-doi="10.2135/cropsci2005.11-0427"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgueño</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cornelius</span>  <span class="cit-name-given-names">P. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McLaren</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trethowan</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Modeling genotype × environment interaction using additive genetic covariances of relatives for predicting breeding values of wheat genotypes</span>. <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">46</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">1722</span>–<span class="cit-lpage">1733</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DCrossa%26rft.auinit1%253DJ.%26rft.volume%253D46%26rft.issue%253D4%26rft.spage%253D1722%26rft.epage%253D1733%26rft.atitle%253DModeling%2BGenotype%2Bx%2BEnvironment%2BInteraction%2BUsing%2BAdditive%2BGenetic%2BCovariances%2Bof%2BRelatives%2Bfor%2BPredicting%2BBreeding%2BValues%2Bof%2BWheat%2BGenotypes%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2005.11-0427%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2005.11-0427&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000239054100036&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-2.12.1595.9" data-doi="10.1534/genetics.110.118521"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgueño</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Prediction of genetic values of quantitative traits in plant breeding using pedigree and molecular markers</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">186</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">713</span>–<span class="cit-lpage">724</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D186%26rft.spage%253D713%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.118521%26rft_id%253Dinfo%253Apmid%252F20813882%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/genetics.110.118521&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20813882&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F12%2F1595.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000282807400022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-2.12.1595.10" data-doi="10.1080/15427528.2011.558767"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mahuku</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dreisigacker</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genomic selection and prediction in plant breeding</span>. <abbr class="cit-jnl-abbrev">J. Crop Improv.</abbr> <span class="cit-vol">25</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">239</span>–<span class="cit-lpage">261</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BCrop%2BImprov.%26rft.volume%253D25%26rft.spage%253D239%26rft_id%253Dinfo%253Adoi%252F10.1080%252F15427528.2011.558767%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/15427528.2011.558767&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-other" id="cit-2.12.1595.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perez</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span>. <span class="cit-comment">BLR: Bayesian Linear Regression R package, version 1.2</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-2.12.1595.12" data-doi="10.1534/genetics.109.101501"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Naya</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Predicting quantitative traits with regression models for dense molecular markers and pedigree</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">182</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">375</span>–<span class="cit-lpage">385</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D182%26rft.spage%253D375%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.101501%26rft_id%253Dinfo%253Apmid%252F19293140%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/genetics.109.101501&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19293140&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F12%2F1595.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-2.12.1595.13" data-doi="10.1017/S0016672310000285"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosa</span>  <span class="cit-name-given-names">G. J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weigel</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Semi-parametric genomic-enabled prediction of genetic values using reproducing kernel Hilbert spaces methods</span>. <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">92</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">295</span>–<span class="cit-lpage">308</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DDE%2BLos%2BCampos%26rft.auinit1%253DG.%26rft.volume%253D92%26rft.issue%253D4%26rft.spage%253D295%26rft.epage%253D308%26rft.atitle%253DSemi-parametric%2Bgenomic-enabled%2Bprediction%2Bof%2Bgenetic%2Bvalues%2Busing%2Breproducing%2Bkernel%2BHilbert%2Bspaces%2Bmethods.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672310000285%26rft_id%253Dinfo%253Apmid%252F20943010%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672310000285&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20943010&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F12%2F1595.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-2.12.1595.14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pong-Wong</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calus</span>  <span class="cit-name-given-names">M. P. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Whole genome regression and prediction methods applied to plant and animal breeding</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> DOI: 10.1534/genetics.112.14331.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-other" id="cit-2.12.1595.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Foresee</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hagan</span>  <span class="cit-name-given-names">M. T.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span>. <span class="cit-comment">Gauss-Newton approximation to Bayesian learning. International Conference on Neural Networks, June 9–12, Houston, TX</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2.12.1595.16" data-doi="10.1534/genetics.107.084285"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Kaam</span>  <span class="cit-name-given-names">J. B. C. H. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Reproducing kernel Hilbert spaces regression methods for genomic assisted prediction of quantitative traits</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">178</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">2289</span>–<span class="cit-lpage">2303</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D178%26rft.spage%253D2289%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.084285%26rft_id%253Dinfo%253Apmid%252F18430950%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/genetics.107.084285&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18430950&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F12%2F1595.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000255239600037&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2.12.1595.17" data-doi="10.1534/genetics.105.049510"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stella</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Genomic-assisted prediction of genetic value with semiparametric procedures</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">173</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">1761</span>–<span class="cit-lpage">1776</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D173%26rft.spage%253D1761%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.049510%26rft_id%253Dinfo%253Apmid%252F16648593%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/genetics.105.049510&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16648593&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F12%2F1595.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-2.12.1595.18" data-doi="10.1186/1471-2156-12-87"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Okut</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weigel</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosa</span>  <span class="cit-name-given-names">G. J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Predicting complex quantitative traits with Bayesian neural networks: a case study with Jersey cows and wheat</span>. <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">87</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenetics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenet%26rft.aulast%253DGianola%26rft.auinit1%253DD.%26rft.volume%253D12%26rft.spage%253D87%26rft.epage%253D87%26rft.atitle%253DPredicting%2Bcomplex%2Bquantitative%2Btraits%2Bwith%2BBayesian%2Bneural%2Bnetworks%253A%2Ba%2Bcase%2Bstudy%2Bwith%2BJersey%2Bcows%2Band%2Bwheat.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2156-12-87%26rft_id%253Dinfo%253Apmid%252F21981731%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2156-12-87&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21981731&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F12%2F1595.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-2.12.1595.19" data-doi="10.1038/hdy.1997.123"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goldringer</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brabant</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gallais</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Estimation of additive and epistatic genetic variances for agronomic traits in a population of doubled-haploid lines of wheat</span>. <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">79</span>: <span class="cit-fpage">60</span>–<span class="cit-lpage">71</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D79%26rft.spage%253D60%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.1997.123%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.1997.123&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-2.12.1595.20" data-doi="10.1007/s00122-012-1868-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">González-Camacho</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cairns</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genome-enabled prediction of genetic values using radial basis function</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">125</span>: <span class="cit-fpage">759</span>–<span class="cit-lpage">771</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DGonzalez-Camacho%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D125%26rft.issue%253D4%26rft.spage%253D759%26rft.epage%253D771%26rft.atitle%253DGenome-enabled%2Bprediction%2Bof%2Bgenetic%2Bvalues%2Busing%2Bradial%2Bbasis%2Bfunction%2Bneural%2Bnetworks.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-012-1868-9%26rft_id%253Dinfo%253Apmid%252F22566067%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-012-1868-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22566067&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F12%2F1595.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-2.12.1595.21" data-doi="10.1186/1471-2105-12-186"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Habier</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kizilkaya</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garrik</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Extension of the Bayesian alphabet for genomic selection</span>. <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">186</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DHabier%26rft.auinit1%253DD.%26rft.volume%253D12%26rft.spage%253D186%26rft.epage%253D186%26rft.atitle%253DExtension%2Bof%2Bthe%2Bbayesian%2Balphabet%2Bfor%2Bgenomic%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-12-186%26rft_id%253Dinfo%253Apmid%252F21605355%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-12-186&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21605355&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F12%2F1595.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-book" id="cit-2.12.1595.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hastie</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tibshirani</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Friedman</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-source">The Elements of Statistical Learning: Data Mining, Inference and Prediction</span>, <span class="cit-edition">Ed. 2</span>. <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-2.12.1595.23" data-doi="10.2135/cropsci2008.08.0512"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heffner</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genomic selection for crop improvement</span>. <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">49</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">1</span>–<span class="cit-lpage">12</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DHeffner%26rft.auinit1%253DE.%2BL.%26rft.volume%253D49%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D12%26rft.atitle%253DGenomic%2BSelection%2Bfor%2BCrop%2BImprovement%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2008.08.0512%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2008.08.0512&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000263344300001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-2.12.1595.24" data-doi="10.2135/cropsci2011.06.0297"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heslot</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">H. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic selection in plant breeding: a comparison of models</span>. <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">52</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">146</span>–<span class="cit-lpage">160</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D52%26rft.spage%253D146%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2011.06.0297%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2011.06.0297&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-book" id="cit-2.12.1595.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tier</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-source">AlphaBayes (Beta): Software for Polygenic and Whole Genome Analysis. User Manual</span>. <span class="cit-publ-name">University of New England</span>, <span class="cit-publ-loc">Armidale, Australia</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-2.12.1595.26" data-doi="10.2307/1267351"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hoerl</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kennard</span>  <span class="cit-name-given-names">R. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1970</span> <span class="cit-article-title">Ridge regression: biased estimation for nonorthogonal problems</span>. <abbr class="cit-jnl-abbrev">Technometrics</abbr> <span class="cit-vol">12</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">55</span>–<span class="cit-lpage">67</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTechnometrics%26rft.volume%253D12%26rft.spage%253D55%26rft_id%253Dinfo%253Adoi%252F10.2307%252F1267351%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/1267351&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1970F898700005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-2.12.1595.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Holland</span>  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Epistasis and plant breeding</span>. <abbr class="cit-jnl-abbrev">Plant Breeding Reviews</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">27</span>–<span class="cit-lpage">92</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBreeding%2BReviews%26rft.volume%253D21%26rft.spage%253D27%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-other" id="cit-2.12.1595.28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Holland</span>  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-comment">Theoretical and biological foundations of plant breeding, pp. 127–140 in <em>Plant Breeding: The Arnel R. Hallauer International Symposium</em>, edited by K. R. Lamkey and M. Lee. Blackwell Publishing, Ames, IA</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-2.12.1595.29" data-doi="10.1016/S0893-6080(00)00098-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lampinen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vehtari</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Bayesian approach for neural networks - review and case studies</span>. <abbr class="cit-jnl-abbrev">Neural Netw.</abbr> <span class="cit-vol">14</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">257</span>–<span class="cit-lpage">274</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNeural%2Bnetworks%2B%253A%2B%2Bthe%2Bofficial%2Bjournal%2Bof%2Bthe%2BInternational%2BNeural%2BNetwork%2BSociety%26rft.stitle%253DNeural%2BNetw%26rft.aulast%253DLampinen%26rft.auinit1%253DJ.%26rft.volume%253D14%26rft.issue%253D3%26rft.spage%253D257%26rft.epage%253D274%26rft.atitle%253DBayesian%2Bapproach%2Bfor%2Bneural%2Bnetworks--review%2Band%2Bcase%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0893-6080%252800%252900098-8%26rft_id%253Dinfo%253Apmid%252F11341565%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0893-6080(00)00098-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11341565&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F12%2F1595.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000168334700002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-2.12.1595.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Laurie</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pratchett</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Snape</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bezant</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">RFLP mapping of five major genes and eight quantitative trait loci controlling flowering time in a winter × spring barley (<em>Hordeum vulgare</em> L.) cross</span>. <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">38</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">575</span>–<span class="cit-lpage">585</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.stitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.aulast%253DLaurie%26rft.auinit1%253DD.%2BA.%26rft.volume%253D38%26rft.issue%253D3%26rft.spage%253D575%26rft.epage%253D585%26rft.atitle%253DRFLP%2Bmapping%2Bof%2Bfive%2Bmajor%2Bgenes%2Band%2Beight%2Bquantitative%2Btrait%2Bloci%2Bcontrolling%2Bflowering%2Btime%2Bin%2Ba%2Bwinter%2Bx%2Bspring%2Bbarley%2B%2528Hordeum%2Bvulgare%2BL.%2529%2Bcross.%26rft_id%253Dinfo%253Apmid%252F18470191%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=18470191&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F12%2F1595.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-2.12.1595.31" data-doi="10.1017/S0016672310000157"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Long</span>  <span class="cit-name-given-names">N. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosa</span>  <span class="cit-name-given-names">G. J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weigel</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kranis</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Radial basis function regression methods for predicting quantitative traits using SNP markers</span>. <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">92</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">209</span>–<span class="cit-lpage">225</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DLong%26rft.auinit1%253DN.%26rft.volume%253D92%26rft.issue%253D3%26rft.spage%253D209%26rft.epage%253D225%26rft.atitle%253DRadial%2Bbasis%2Bfunction%2Bregression%2Bmethods%2Bfor%2Bpredicting%2Bquantitative%2Btraits%2Busing%2BSNP%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672310000157%26rft_id%253Dinfo%253Apmid%252F20667165%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672310000157&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20667165&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F12%2F1595.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-2.12.1595.32" data-doi="10.1162/neco.1992.4.3.448"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">MacKay</span>  <span class="cit-name-given-names">D. J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">A practical Bayesian framework for backpropagation networks</span>. <abbr class="cit-jnl-abbrev">Neural Comput.</abbr> <span class="cit-vol">4</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">448</span>–<span class="cit-lpage">472</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNeural%2BComput.%26rft.volume%253D4%26rft.spage%253D448%26rft_id%253Dinfo%253Adoi%252F10.1162%252Fneco.1992.4.3.448%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1162/neco.1992.4.3.448&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1992HZ47600011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-2.12.1595.33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">MacKay</span>  <span class="cit-name-given-names">D. J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Bayesian non-linear modelling for the prediction competition</span>. <abbr class="cit-jnl-abbrev">ASHRAE Transactions</abbr> <span class="cit-vol">100</span>(<span class="cit-comment">Pt. 2</span>): <span class="cit-fpage">1053</span>–<span class="cit-lpage">1062</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DASHRAE%2BTransactions%26rft.volume%253D100%26rft.spage%253D1053%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-2.12.1595.34" data-doi="10.1371/journal.pgen.1002051"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Makowsky</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pajewski</span>  <span class="cit-name-given-names">N. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klimentidis</span>  <span class="cit-name-given-names">Y. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vazquez</span>  <span class="cit-name-given-names">A. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Duarte</span>  <span class="cit-name-given-names">C. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Beyond missing heritability: prediction of complex traits</span>. <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">7</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">e1002051</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DMakowsky%26rft.auinit1%253DR.%26rft.volume%253D7%26rft.issue%253D4%26rft.spage%253De1002051%26rft.epage%253De1002051%26rft.atitle%253DBeyond%2Bmissing%2Bheritability%253A%2Bprediction%2Bof%2Bcomplex%2Btraits.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002051%26rft_id%253Dinfo%253Apmid%252F21552331%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002051&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21552331&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F12%2F1595.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-2.12.1595.35"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McKinney</span>  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pajewski</span>  <span class="cit-name-given-names">N. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span>. <span class="cit-article-title">Six degrees of epistasis: statistical network models for GWAS</span>. <abbr class="cit-jnl-abbrev">Front. Genet.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">109</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BGenet.%26rft.volume%253D2%26rft.spage%253D109%26rft_id%253Dinfo%253Apmid%252F22303403%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=22303403&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F12%2F1595.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-2.12.1595.36"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Prediction of total genetic value using genome-wide dense marker maps</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">1819</span>–<span class="cit-lpage">1829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMeuwissen%26rft.auinit1%253DT.%2BH.%2BE.%26rft.volume%253D157%26rft.issue%253D4%26rft.spage%253D1819%26rft.epage%253D1829%26rft.atitle%253DPrediction%2Bof%2BTotal%2BGenetic%2BValue%2BUsing%2BGenome-Wide%2BDense%2BMarker%2BMaps%26rft_id%253Dinfo%253Apmid%252F11290733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ny80LzE4MTkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8yLzEyLzE1OTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-other" id="cit-2.12.1595.37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Neal</span>  <span class="cit-name-given-names">R. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span>. <span class="cit-comment"><em>Bayesian Learning for Neural Networks (Lecture Notes in Statistics)</em>, Vol. 118. Springer-Verlag, NY</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-2.12.1595.38" data-doi="10.1371/journal.pgen.1002685"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ober</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ayroles</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stone</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Richards</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Using whole-genome sequence data to predict quantitative trait phenotypes in <em>Drosophila melanogaster</em></span>. <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">8</span>(<span class="cit-issue">5</span>): <span class="cit-fpage">e1002685</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DOber%26rft.auinit1%253DU.%26rft.volume%253D8%26rft.issue%253D5%26rft.spage%253De1002685%26rft.epage%253De1002685%26rft.atitle%253DUsing%2Bwhole-genome%2Bsequence%2Bdata%2Bto%2Bpredict%2Bquantitative%2Btrait%2Bphenotypes%2Bin%2BDrosophila%2Bmelanogaster.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002685%26rft_id%253Dinfo%253Apmid%252F22570636%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002685&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22570636&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F12%2F1595.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-2.12.1595.39" data-doi="10.1017/S0016672310000662"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Okut</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosa</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weigel</span>  <span class="cit-name-given-names">K. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Prediction of body mass index in mice using dense molecular markers and a regularized neural network</span>. <abbr class="cit-jnl-abbrev">Genet. Res. Camb.</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">189</span>–<span class="cit-lpage">201</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BRes.%2BCamb.%26rft.volume%253D93%26rft.spage%253D189%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672310000662%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672310000662&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-2.12.1595.40" data-doi="10.1198/016214508000000337"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casella</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The Bayesian LASSO</span>. <abbr class="cit-jnl-abbrev">J. Am. Stat. Assoc.</abbr> <span class="cit-vol">103</span>: <span class="cit-fpage">681</span>–<span class="cit-lpage">686</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAm.%2BStat.%2BAssoc.%26rft.volume%253D103%26rft.spage%253D681%26rft_id%253Dinfo%253Adoi%252F10.1198%252F016214508000000337%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1198/016214508000000337&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000257897500025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-2.12.1595.41" data-doi="10.3835/plantgenome2010.04.0005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Perez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genomic-enabled prediction based on molecular markers and pedigree using the Bayesian linear regression package in R</span>. <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">3</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">106</span>–<span class="cit-lpage">116</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D3%26rft.spage%253D106%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2010.04.0005%26rft_id%253Dinfo%253Apmid%252F21566722%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2010.04.0005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21566722&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F12%2F1595.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-2.12.1595.42" data-doi="10.1109/5.58326"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Poggio</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Girosi</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Networks for approximation and learning</span>. <abbr class="cit-jnl-abbrev">Proc. IEEE</abbr> <span class="cit-vol">78</span>(<span class="cit-issue">9</span>): <span class="cit-fpage">1481</span>–<span class="cit-lpage">1497</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BIEEE%26rft.volume%253D78%26rft.spage%253D1481%26rft_id%253Dinfo%253Adoi%252F10.1109%252F5.58326%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1109/5.58326&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-2.12.1595.43"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Resende</span>  <span class="cit-name-given-names">M. F. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muñoz</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Resende</span>  <span class="cit-name-given-names">M. D. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garrick</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Accuracy of genomic selection methods in a standard data set of loblolly pine (<em>Pinus taeda</em> L.)</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">1503</span>–<span class="cit-lpage">1510</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D4%26rft.spage%253D1503%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-2.12.1595.44" data-doi="10.1111/j.1365-313X.2009.03806.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shimada</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ogawa</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kitagawa</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A genetic network of flowering-time genes in wheat leaves, in which an <em>APETALA1/FRUITFULL</em>-like gene, VRN-1, is upstream of <em>FLOWERING LOCUS T</em></span>. <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">58</span>: <span class="cit-fpage">668</span>–<span class="cit-lpage">681</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bjournal%2B%253A%2B%2Bfor%2Bcell%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BJ%26rft.aulast%253DShimada%26rft.auinit1%253DS.%26rft.volume%253D58%26rft.issue%253D4%26rft.spage%253D668%26rft.epage%253D681%26rft.atitle%253DA%2Bgenetic%2Bnetwork%2Bof%2Bflowering-time%2Bgenes%2Bin%2Bwheat%2Bleaves%252C%2Bin%2Bwhich%2Ban%2BAPETALA1%252FFRUITFULL-like%2Bgene%252C%2BVRN1%252C%2Bis%2Bupstream%2Bof%2BFLOWERING%2BLOCUS%2BT.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-313X.2009.03806.x%26rft_id%253Dinfo%253Apmid%252F19175767%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-313X.2009.03806.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19175767&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F12%2F1595.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000265984000011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-2.12.1595.45" data-doi="10.1051/gse:19940202"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rutledge</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Bayesian analysis of mixed linear models via Gibbs sampling with an application to litter size in Iberian pigs</span>. <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">91</span>–<span class="cit-lpage">115</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D26%26rft.spage%253D91%26rft_id%253Dinfo%253Adoi%252F10.1051%252Fgse%253A19940202%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1051/gse:19940202&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1994NL04000002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-2.12.1595.46" data-doi="10.1016/S1673-8527(08)60017-X"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tian</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Mapping QTLs with epistatic effects and QTL × environment interactions for plant height using a doubled haploid population in cultivated wheat</span>. <abbr class="cit-jnl-abbrev">J. Genet. Genomics</abbr> <span class="cit-vol">35</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">119</span>–<span class="cit-lpage">127</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BGenet%2BGenomics%26rft.aulast%253DZhang%26rft.auinit1%253DK.%26rft.volume%253D35%26rft.issue%253D2%26rft.spage%253D119%26rft.epage%253D127%26rft.atitle%253DMapping%2BQTLs%2Bwith%2Bepistatic%2Beffects%2Band%2BQTL%2Bx%2Benvironment%2Binteractions%2Bfor%2Bplant%2Bheight%2Busing%2Ba%2Bdoubled%2Bhaploid%2Bpopulation%2Bin%2Bcultivated%2Bwheat.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1673-8527%252808%252960017-X%26rft_id%253Dinfo%253Apmid%252F18407059%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1673-8527(08)60017-X&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18407059&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F12%2F1595.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/2/12/1595.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/2/12/1585" title="Genetic Interactions Between Brassinosteroid-Inactivating P450s and Photomorphogenic Photoreceptors in Arabidopsis thaliana" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/2/12/1607" title="Natural Variation in the Yeast Glucose-Signaling Network Reveals a New Role for the Mig3p Transcription Factor" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/2/12"><span>Volume 2 Issue 12, December 2012</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/2/12.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/2/12.cover-source.jpg?itok=kEAHvMS0" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 2 (12)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/12/1595" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form--UqbZLITkWuyW19ju7-1f5HF6PDt5khgFeSRL84F8Vc" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genomic-selection" class="" data-icon-position="" data-hide-link-title="0">Genomic Selection</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/2/12/1595" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/2/12/1595" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/2/12/1595" class="active" data-icon-position="" data-hide-link-title="0">Comparison Between Linear and Non-parametric Regression Models for Genome-Enabled Prediction in Wheat</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459370" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-HNu-EzeqgYBw6jSKyGUA1UQbzvgOrOGrpKC3c1KqDDY" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/2/12/1595.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/2/12/1595" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-o_6X0gJU_5e72WUDv_AAyrLEPu4jQZb4g-5psymKzC4" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/2/12/1595" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/2/12/1595.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459370" id="node459370--2" data-pisa="ggg;2/12/1595" data-pisa-master="ggg;2/12/1595" data-apath="/ggg/2/12/1595.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Comparison Between Linear and Non-parametric Regression Models for Genome-Enabled Prediction in Wheat</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Paulino Pérez-Rodríguez</span>, <span class="highwire-citation-author" data-delta="1">Daniel Gianola</span>, <span class="highwire-citation-author" data-delta="2">Juan Manuel González-Camacho</span>, <span class="highwire-citation-author" data-delta="3">José Crossa</span>, <span class="highwire-citation-author" data-delta="4">Yann Manès</span> and <span class="highwire-citation-author" data-delta="5">Susanne Dreisigacker</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1595-1605;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.003665 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459370--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Paulino Pérez-Rodríguez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Colegio de Postgraduados, Montecillo, Texcoco 56230, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Paulino%2BP%C3%A9rez-Rodr%C3%ADguez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=P%C3%A9rez-Rodr%C3%ADguez%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APaulino%2BP%25C3%25A9rez-Rodr%25C3%25ADguez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:perpdgo@gmail.com" class="" data-icon-position="" data-hide-link-title="0">perpdgo@gmail.com</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Daniel Gianola </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Departments of Animal Sciences, Dairy Science, and Biostatistics and Medical Informatics, University of Wisconsin-Madison, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BGianola%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gianola%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADaniel%2BGianola%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Juan Manuel González-Camacho </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Colegio de Postgraduados, Montecillo, Texcoco 56230, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Juan%2BManuel%2BGonz%C3%A1lez-Camacho%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gonz%C3%A1lez-Camacho%20JM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJuan%2BManuel%2BGonz%25C3%25A1lez-Camacho%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">José Crossa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Biometrics and Statistics Unit and Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), 06600 Mexico, D.F., México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jos%C3%A9%2BCrossa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crossa%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJos%25C3%25A9%2BCrossa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Yann Manès </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Biometrics and Statistics Unit and Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), 06600 Mexico, D.F., México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yann%2BMan%C3%A8s%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Man%C3%A8s%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYann%2BMan%25C3%25A8s%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Susanne Dreisigacker </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Biometrics and Statistics Unit and Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), 06600 Mexico, D.F., México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Susanne%2BDreisigacker%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dreisigacker%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASusanne%2BDreisigacker%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/2/12/1595" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/2/12/1595&amp;title=Comparison%20Between%20Linear%20and%20Non-parametric%20Regression%20Models%20for%20Genome-Enabled%20Prediction%20in%20Wheat" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/2/12/1595&amp;title=Comparison%20Between%20Linear%20and%20Non-parametric%20Regression%20Models%20for%20Genome-Enabled%20Prediction%20in%20Wheat" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/2/12/1595&amp;title=Comparison%20Between%20Linear%20and%20Non-parametric%20Regression%20Models%20for%20Genome-Enabled%20Prediction%20in%20Wheat" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/2/12/1595&amp;text=Comparison%20Between%20Linear%20and%20Non-parametric%20Regression%20Models%20for%20Genome-Enabled%20Prediction%20in%20Wheat" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/2/12/1595&amp;title=Comparison%20Between%20Linear%20and%20Non-parametric%20Regression%20Models%20for%20Genome-Enabled%20Prediction%20in%20Wheat" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/2/12/1595&amp;t=Comparison%20Between%20Linear%20and%20Non-parametric%20Regression%20Models%20for%20Genome-Enabled%20Prediction%20in%20Wheat" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/2/12/1595&amp;title=Comparison%20Between%20Linear%20and%20Non-parametric%20Regression%20Models%20for%20Genome-Enabled%20Prediction%20in%20Wheat" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/2/12/1595&amp;title=Comparison%20Between%20Linear%20and%20Non-parametric%20Regression%20Models%20for%20Genome-Enabled%20Prediction%20in%20Wheat" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459370/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459370" id="node459370--3" data-pisa="ggg;2/12/1595" data-pisa-master="ggg;2/12/1595" data-apath="/ggg/2/12/1595.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Comparison Between Linear and Non-parametric Regression Models for Genome-Enabled Prediction in Wheat</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Paulino Pérez-Rodríguez</span>, <span class="highwire-citation-author" data-delta="1">Daniel Gianola</span>, <span class="highwire-citation-author" data-delta="2">Juan Manuel González-Camacho</span>, <span class="highwire-citation-author" data-delta="3">José Crossa</span>, <span class="highwire-citation-author" data-delta="4">Yann Manès</span> and <span class="highwire-citation-author" data-delta="5">Susanne Dreisigacker</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1595-1605;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.003665 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459370--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Paulino Pérez-Rodríguez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Colegio de Postgraduados, Montecillo, Texcoco 56230, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Paulino%2BP%C3%A9rez-Rodr%C3%ADguez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=P%C3%A9rez-Rodr%C3%ADguez%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APaulino%2BP%25C3%25A9rez-Rodr%25C3%25ADguez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:perpdgo@gmail.com" class="" data-icon-position="" data-hide-link-title="0">perpdgo@gmail.com</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Daniel Gianola </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Departments of Animal Sciences, Dairy Science, and Biostatistics and Medical Informatics, University of Wisconsin-Madison, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BGianola%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gianola%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADaniel%2BGianola%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Juan Manuel González-Camacho </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Colegio de Postgraduados, Montecillo, Texcoco 56230, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Juan%2BManuel%2BGonz%C3%A1lez-Camacho%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gonz%C3%A1lez-Camacho%20JM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJuan%2BManuel%2BGonz%25C3%25A1lez-Camacho%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">José Crossa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Biometrics and Statistics Unit and Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), 06600 Mexico, D.F., México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jos%C3%A9%2BCrossa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crossa%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJos%25C3%25A9%2BCrossa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Yann Manès </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Biometrics and Statistics Unit and Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), 06600 Mexico, D.F., México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yann%2BMan%C3%A8s%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Man%C3%A8s%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYann%2BMan%25C3%25A8s%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Susanne Dreisigacker </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Biometrics and Statistics Unit and Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), 06600 Mexico, D.F., México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Susanne%2BDreisigacker%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dreisigacker%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASusanne%2BDreisigacker%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1595&amp;rft.epage=1605&amp;rft.atitle=Comparison%20Between%20Linear%20and%20Non-parametric%20Regression%20Models%20for%20Genome-Enabled%20Prediction%20in%20Wheat&amp;rft.volume=2&amp;rft.issue=12&amp;rft.date=2012-12-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=P%C3%A9rez-Rodr%C3%ADguez%2C+Paulino&amp;rft.au=Gianola%2C+Daniel&amp;rft.au=Gonz%C3%A1lez-Camacho%2C+Juan+Manuel&amp;rft.au=Crossa%2C+Jos%C3%A9&amp;rft.au=Man%C3%A8s%2C+Yann&amp;rft.au=Dreisigacker%2C+Susanne"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459370/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459370/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459370/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459370/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459370/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459370/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459370/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459370/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459370/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459370/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459370/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459370/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490720" id="node490720" data-pisa="ggg;8/9/3019" data-pisa-master="ggg;g3.118.200430" data-apath="/ggg/8/9/3019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">A Bayesian Decision Theory Approach for Genomic Selection</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490724" id="node490724" data-pisa="ggg;8/9/2889" data-pisa-master="ggg;g3.118.200311" data-apath="/ggg/8/9/2889.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/2889" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Optimising Genomic Selection in Wheat: Effect of Marker Density, Population Size and Population Structure on Prediction Accuracy</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490727" id="node490727" data-pisa="ggg;8/9/3039" data-pisa-master="ggg;g3.118.200435" data-apath="/ggg/8/9/3039.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3039" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">BGGE: A New Package for Genomic-Enabled Prediction Incorporating Genotype × Environment Interaction Models</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genomic%20Selection" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genomic Selection</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/2/12/1595" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-15" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">RESULTS</a></li><li class="even"><a href="#sec-17" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">DISCUSSION AND CONCLUSIONS</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/2/12/1595.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/2/12/1595.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/2/12/1595.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/2/12/1595.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
