<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/8/11/3567" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/8/11/3567.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/8/11/3567.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/8/11/3567.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/8/11/3567/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/8/11/3567.atom" />
<meta name="HW.pisa" content="ggg;8/11/3567" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Empirical Comparisons of Different Statistical Models To Identify and Validate Kernel Row Number-Associated Variants from Structured Multi-parent Mapping Populations of Maize" />
<meta name="DC.Identifier" content="10.1534/g3.118.200636" />
<meta name="DC.Date" content="2018-11-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2018 Yang et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Advances in next generation sequencing technologies and statistical approaches enable genome-wide dissection of phenotypic traits via genome-wide association studies (GWAS). Although multiple statistical approaches for conducting GWAS are available, the power and cross-validation rates of many approaches have been mostly tested using simulated data. Empirical comparisons of single variant (SV) and multi-variant (MV) GWAS approaches have not been conducted to test if a single approach or a combination of SV and MV is effective, through identification and cross-validation of trait-associated loci. In this study, kernel row number (KRN) data were collected from a set of 6,230 entries derived from the Nested Association Mapping (NAM) population and related populations. Three different types of GWAS analyses were performed: 1) single-variant (SV), 2) stepwise regression (STR) and 3) a Bayesian-based multi-variant (BMV) model. Using SV, STR, and BMV models, 257, 300, and 442 KRN-associated variants (KAVs) were identified in the initial GWAS analyses. Of these, 231 KAVs were subjected to genetic validation using three unrelated populations that were not included in the initial GWAS. Genetic validation results suggest that the three GWAS approaches are complementary. Interestingly, KAVs in low recombination regions were more likely to exhibit associations in independent populations than KAVs in recombinationally active regions, probably as a consequence of linkage disequilibrium. The KAVs identified in this study have the potential to enhance our understanding of the genetic basis of ear development." />
<meta name="DC.Contributor" content="Jinliang Yang" />
<meta name="DC.Contributor" content="Cheng-Ting “Eddy” Yeh" />
<meta name="DC.Contributor" content="Raghuprakash Kastoori Ramamurthy" />
<meta name="DC.Contributor" content="Xinshuai Qi" />
<meta name="DC.Contributor" content="Rohan L. Fernando" />
<meta name="DC.Contributor" content="Jack C. M. Dekkers" />
<meta name="DC.Contributor" content="Dorian J. Garrick" />
<meta name="DC.Contributor" content="Dan Nettleton" />
<meta name="DC.Contributor" content="Patrick S. Schnable" />
<meta name="article:published_time" content="2018-11-01" />
<meta name="article:section" content="Multiparental Populations" />
<meta name="citation_title" content="Empirical Comparisons of Different Statistical Models To Identify and Validate Kernel Row Number-Associated Variants from Structured Multi-parent Mapping Populations of Maize" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Advances in next generation sequencing technologies and statistical approaches enable genome-wide dissection of phenotypic traits via genome-wide association studies (GWAS). Although multiple statistical approaches for conducting GWAS are available, the power and cross-validation rates of many approaches have been mostly tested using simulated data. Empirical comparisons of single variant (SV) and multi-variant (MV) GWAS approaches have not been conducted to test if a single approach or a combination of SV and MV is effective, through identification and cross-validation of trait-associated loci. In this study, kernel row number (KRN) data were collected from a set of 6,230 entries derived from the Nested Association Mapping (NAM) population and related populations. Three different types of GWAS analyses were performed: 1) single-variant (SV), 2) stepwise regression (STR) and 3) a Bayesian-based multi-variant (BMV) model. Using SV, STR, and BMV models, 257, 300, and 442 KRN-associated variants (KAVs) were identified in the initial GWAS analyses. Of these, 231 KAVs were subjected to genetic validation using three unrelated populations that were not included in the initial GWAS. Genetic validation results suggest that the three GWAS approaches are complementary. Interestingly, KAVs in low recombination regions were more likely to exhibit associations in independent populations than KAVs in recombinationally active regions, probably as a consequence of linkage disequilibrium. The KAVs identified in this study have the potential to enhance our understanding of the genetic basis of ear development.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2018/11/01" />
<meta name="citation_mjid" content="ggg;8/11/3567" />
<meta name="citation_id" content="8/11/3567" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/8/11/3567" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/8/11/3567.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/8/11/3567.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/8/11/3567.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.118.200636" />
<meta name="citation_pmid" content="30213868" />
<meta name="citation_volume" content="8" />
<meta name="citation_issue" content="11" />
<meta name="citation_num_pages" content="9" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Multiparental Populations" />
<meta name="citation_firstpage" content="3567" />
<meta name="citation_lastpage" content="3575" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Jinliang Yang" />
<meta name="citation_author_institution" content="Department of Agronomy" />
<meta name="citation_author_institution" content="Department of Agronomy and Horticulture, University of Nebraska-Lincoln, Lincoln, NE 68583" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-0999-3518" />
<meta name="citation_author" content="Cheng-Ting “Eddy” Yeh" />
<meta name="citation_author_institution" content="Department of Agronomy" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-1392-2018" />
<meta name="citation_author" content="Raghuprakash Kastoori Ramamurthy" />
<meta name="citation_author_institution" content="Department of Agronomy and Horticulture, University of Nebraska-Lincoln, Lincoln, NE 68583" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-7804-712X" />
<meta name="citation_author" content="Xinshuai Qi" />
<meta name="citation_author_institution" content="Department of Ecology and Evolutionary Biology, University of Arizona, Tucson, AZ" />
<meta name="citation_author" content="Rohan L. Fernando" />
<meta name="citation_author_institution" content="Department of Animal Science and Center for Integrated Animal Genomics" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-5821-099X" />
<meta name="citation_author" content="Jack C. M. Dekkers" />
<meta name="citation_author_institution" content="Department of Animal Science and Center for Integrated Animal Genomics" />
<meta name="citation_author" content="Dorian J. Garrick" />
<meta name="citation_author_institution" content="Department of Animal Science and Center for Integrated Animal Genomics" />
<meta name="citation_author" content="Dan Nettleton" />
<meta name="citation_author_institution" content="Department of Statistics, Iowa State University, Ames, IA 50011" />
<meta name="citation_author" content="Patrick S. Schnable" />
<meta name="citation_author_institution" content="Department of Agronomy" />
<meta name="citation_author_email" content="schnable@iastate.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-9169-5204" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=S. Atwell;citation_author=Y. S. Huang;citation_author=B. J. Vilhjálmsson;citation_author=G. Willems;citation_author=M. Horton;citation_title=Genome-wide association study of 107 phenotypes in Arabidopsis thaliana inbred lines.;citation_pages=627-631;citation_volume=465;citation_year=2010;citation_issue=7298;citation_pmid=20336072;citation_doi=10.1038/nature08800" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=P. J. Brown;citation_author=N. Upadyayula;citation_author=G. S. Mahone;citation_author=F. Tian;citation_author=P. J. Bradbury;citation_title=Distinct genetic architectures for male and female inflorescence traits of maize.;citation_pages=e1002383-e1002383;citation_volume=7;citation_year=2011;citation_issue=11;citation_pmid=22125498;citation_doi=10.1371/journal.pgen.1002383" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=E. S. Buckler;citation_author=J. B. Holland;citation_author=P. J. Bradbury;citation_author=C. B. Acharya;citation_author=P. J. Brown;citation_title=The Genetic Architecture of Maize Flowering Time;citation_pages=714-718;citation_volume=325;citation_year=2009;citation_issue=5941;citation_pmid=19661422;citation_doi=10.1126/science.1174276" />
<meta name="citation_reference" content="citation_journal_title=Gigascience;citation_author=R. Bukowski;citation_author=X. Guo;citation_author=Y. Lu;citation_author=C. Zou;citation_author=B. He;citation_title=Construction of the third generation zea mays haplotype map.;citation_pages=1-12;citation_volume=7;citation_year=2015;citation_doi=10.1093/gigascience/gix134" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. Cockram;citation_author=J. White;citation_author=D. L. Zuluaga;citation_author=D. Smith;citation_author=J. Comadran;citation_title=Genome-wide association mapping to candidate polymorphism resolution in the unsequenced barley genome.;citation_volume=107;citation_year=2010;citation_pmid=21115826;citation_doi=10.1073/pnas.1010179107" />
<meta name="citation_reference" content="citation_journal_title=Biometrics;citation_journal_abbrev=Biometrics;citation_author=B. Devlin;citation_author=K. Roeder;citation_title=Genomic control for association studies.;citation_pages=997-1004;citation_volume=55;citation_year=1999;citation_issue=4;citation_pmid=11315092;citation_doi=10.1111/j.0006-341X.1999.00997.x" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=R. J. Elshire;citation_author=J. C. Glaubitz;citation_author=Q. Sun;citation_author=J. A. Poland;citation_author=K. Kawamoto;citation_title=A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.;citation_pages=e19379-e19379;citation_volume=6;citation_year=2011;citation_issue=5;citation_pmid=21573248;citation_doi=10.1371/journal.pone.0019379" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=B. Fan;citation_author=S. K. Onteru;citation_author=Z. Q. Du;citation_author=D. J. Garrick;citation_author=K. J. Stalder;citation_title=Genome-wide association study identifies Loci for body composition and structural soundness traits in pigs.;citation_pages=e14726-e14726;citation_volume=6;citation_year=2011;citation_issue=2;citation_pmid=21383979;citation_doi=10.1371/journal.pone.0014726" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=H. Fu;citation_author=Z. Zheng;citation_author=H. K. Dooner;citation_title=Recombination rates between adjacent genic and retrotransposon regions in maize vary by 2 orders of magnitude.;citation_volume=99;citation_year=2002;citation_pmid=11792865;citation_doi=10.1073/pnas.022635499" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=T. E. Galesloot;citation_author=K. Van Steen;citation_author=L. A. Kiemeney;citation_author=L. L. Janss;citation_author=S. H. Vermeulen;citation_title=A comparison of multivariate genome-wide association methods.;citation_pages=e95923;citation_volume=9;citation_year=2014;citation_pmid=24763738;citation_doi=10.1371/journal.pone.0095923" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=D. Habier;citation_author=R. L. Fernando;citation_author=K. Kizilkaya;citation_author=D. J. Garrick;citation_title=Extension of the bayesian alphabet for genomic selection.;citation_pages=186-186;citation_volume=12;citation_year=2011;citation_pmid=21605355;citation_doi=10.1186/1471-2105-12-186" />
<meta name="citation_reference" content="citation_title=Quantitative genetics in maize breeding;citation_author=A. R. Hallauer;citation_author=M. J. Carena;citation_author=J. d. Miranda Filho;citation_volume=Vol. 6;citation_year=2010" />
<meta name="citation_reference" content="Hallauer, A. R., A. J. Ross, and M. Lee, 2004 Long-term Divergent Selection for Ear Length in Maize 24." />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=X. Huang;citation_author=X. Wei;citation_author=T. Sang;citation_author=Q. Zhao;citation_author=Q. Feng;citation_title=Genome-wide association studies of 14 agronomic traits in rice landraces.;citation_pages=961-967;citation_volume=42;citation_year=2010;citation_issue=11;citation_pmid=20972439;citation_doi=10.1038/ng.695" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=H. M. Kang;citation_author=J. H. Sul;citation_author=S. K. Service;citation_author=N. Zaitlen;citation_author=S.-Y. Kong;citation_title=Variance component model to account for sample structure in genome-wide association studies.;citation_pages=348-354;citation_volume=42;citation_year=2010;citation_issue=4;citation_pmid=20208533;citation_doi=10.1038/ng.548" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=S. Kim;citation_author=V. Plagnol;citation_author=T. T. Hu;citation_author=C. Toomajian;citation_author=R. M. Clark;citation_title=Recombination and linkage disequilibrium in Arabidopsis thaliana.;citation_pages=1151-1155;citation_volume=39;citation_year=2007;citation_issue=9;citation_pmid=17676040;citation_doi=10.1038/ng2115" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=R. J. Klein;citation_author=C. Zeiss;citation_author=E. Y. Chew;citation_author=J.-Y. Tsai;citation_author=R. S. Sackler;citation_title=Complement factor H polymorphism in age-related macular degeneration.;citation_volume=308;citation_year=2005;citation_pmid=15761122;citation_doi=10.1126/science.1109557" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=E. S. Lander;citation_author=S. Botstein;citation_title=Mapping mendelian factors underlying quantitative traits using RFLP linkage maps.;citation_pages=185-199;citation_volume=121;citation_year=1989;citation_issue=1;citation_pmid=2563713" />
<meta name="citation_reference" content="citation_journal_title=Plant molecular biology;citation_journal_abbrev=Plant Mol Biol;citation_author=M. Lee;citation_author=N. Sharopova;citation_author=W. D. Beavis;citation_author=D. Grant;citation_author=M. Katt;citation_title=Expanding the genetic map of maize with the intermated B73 x Mo17 (IBM) population.;citation_pages=453-461;citation_volume=48;citation_year=2002;citation_issue=5-6;citation_pmid=11999829;citation_doi=10.1023/A:1014893521186" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=S. Leiboff;citation_author=X. Li;citation_author=H.-C. Hu;citation_author=N. Todt;citation_author=J. Yang;citation_title=Genetic control of morphometric diversity in the maize shoot apical meristem.;citation_pages=8974;citation_volume=6;citation_year=2015;citation_doi=10.1038/ncomms9974" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=X. Li;citation_author=C. Zhu;citation_author=C.-T. Yeh;citation_author=W. Wu;citation_author=E. M. Takacs;citation_title=Genic and nongenic contributions to natural variation of quantitative traits in maize.;citation_volume=22;citation_year=2012;citation_pmid=22701078;citation_doi=10.1101/gr.140277.112" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=H.-y. Lin;citation_author=Q. Liu;citation_author=X. Li;citation_author=J. Yang;citation_author=S. Liu;citation_title=Substantial contribution of genetic variation in the expression of transcription factors to phenotypic variation revealed by erd-gwas.;citation_pages=192;citation_volume=18;citation_year=2017;citation_doi=10.1186/s13059-017-1328-6" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=K. Liu;citation_author=M. Goodman;citation_author=S. Muse;citation_author=J. S. Smith;citation_author=E. Buckler;citation_title=Genetic Structure and Diversity Among Maize Inbred Lines as Inferred From DNA Microsatellites;citation_pages=2117-2128;citation_volume=165;citation_year=2003;citation_issue=4;citation_pmid=14704191" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=L. Liu;citation_author=Y. Du;citation_author=D. Huo;citation_author=M. Wang;citation_author=X. Shen;citation_title=Genetic architecture of maize kernel row number and whole genome prediction.;citation_pages=2243-2254;citation_volume=128;citation_year=2015;citation_doi=10.1007/s00122-015-2581-2" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=L. Liu;citation_author=Y. Du;citation_author=X. Shen;citation_author=M. Li;citation_author=W. Sun;citation_title=Krn4 controls quantitative variation in maize kernel row number.;citation_pages=e1005670;citation_volume=11;citation_year=2015;citation_doi=10.1371/journal.pgen.1005670" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=S. Liu;citation_author=C. T. Yeh;citation_author=T. Ji;citation_author=K. Ying;citation_author=H. Wu;citation_title=Mu transposon insertion sites and meiotic recombination events co-localize with epigenetic marks for open chromatin across the maize genome.;citation_pages=e1000733-e1000733;citation_volume=5;citation_year=2009;citation_issue=11;citation_pmid=19936291;citation_doi=10.1371/journal.pgen.1000733" />
<meta name="citation_reference" content="citation_journal_title=Mol. Breed.;citation_author=M. Lu;citation_author=C. X. Xie;citation_author=X. H. Li;citation_author=Z. F. Hao;citation_author=M. S. Li;citation_title=Mapping of quantitative trait loci for kernel row number in maize across seven environments.;citation_pages=143-152;citation_volume=28;citation_year=2011;citation_doi=10.1007/s11032-010-9468-3" />
<meta name="citation_reference" content="citation_journal_title=New England Journal of Medicine;citation_journal_abbrev=NEJM;citation_author=T. Manolio;citation_title=Genomewide association studies and assessment of the risk of disease.;citation_pages=166-176;citation_volume=363;citation_year=2010;citation_issue=2;citation_pmid=20647212;citation_doi=10.1056/NEJMra0905980" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=J. Marchini;citation_author=B. Howie;citation_title=Genotype imputation for genome-wide association studies.;citation_pages=499-511;citation_volume=11;citation_year=2010;citation_issue=7;citation_pmid=20517342;citation_doi=10.1038/nrg2796" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=M. Meijón;citation_author=S. B. Satbhai;citation_author=T. Tsuchimatsu;citation_author=W. Busch;citation_title=Genome-wide association study using cellular traits identifies a new regulator of root development in Arabidopsis.;citation_pages=77-81;citation_volume=46;citation_year=2014;citation_pmid=24212884;citation_doi=10.1038/ng.2824" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. H. Meuwissen;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_title=Prediction of Total Genetic Value Using Genome-Wide Dense Marker Maps;citation_pages=1819-1829;citation_volume=157;citation_year=2001;citation_issue=4;citation_pmid=11290733" />
<meta name="citation_reference" content="citation_author=G. P. Morris;citation_author=P. Ramu;citation_author=S. P. Deshpande;citation_author=C. T. Hash;citation_author=T. Shah;citation_pages=453-8;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=P. T. Nelson;citation_author=N. D. Coles;citation_author=J. B. Holland;citation_author=D. M. Bubeck;citation_author=S. Smith;citation_title=Molecular Characterization of Maize Inbreds with Expired U.S. Plant Variety Protection;citation_pages=1673-1685;citation_volume=48;citation_year=2008;citation_issue=5;citation_doi=10.2135/cropsci2008.02.0092" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=A. L. Price;citation_author=N. J. Patterson;citation_author=R. M. Plenge;citation_author=M. E. Weinblatt;citation_author=N. Shadick;citation_title=Principal components analysis corrects for stratification in genome-wide association studies.;citation_pages=904-909;citation_volume=38;citation_year=2006;citation_issue=8;citation_pmid=16862161;citation_doi=10.1038/ng1847" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=S. Purcell;citation_author=B. Neale;citation_author=K. Todd-Brown;citation_author=L. Thomas;citation_author=M. R. Ferreira;citation_title=PLINK: a tool set for whole-genome association and population-based linkage analyses.;citation_pages=559-575;citation_volume=81;citation_year=2007;citation_issue=3;citation_pmid=17701901;citation_doi=10.1086/519795" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=V. Segura;citation_author=B. J. Vilhjálmsson;citation_author=A. Platt;citation_author=A. Korte;citation_author=U. Seren;citation_title=An efficient multi-locus mixed-model approach for genome-wide association studies in structured populations.;citation_pages=825-830;citation_volume=44;citation_year=2012;citation_issue=7;citation_pmid=22706313;citation_doi=10.1038/ng.2314" />
<meta name="citation_reference" content="citation_journal_title=Methods in molecular biology (Clifton, N.J.);citation_journal_abbrev=Methods Mol Biol;citation_author=C. Silva Lda;citation_author=S. Wang;citation_author=Z. B. Zeng;citation_title=Composite interval mapping and multiple interval mapping: procedures and guidelines for using Windows QTL Cartographer.;citation_pages=75-119;citation_volume=871;citation_year=2012;citation_pmid=22565834;citation_doi=10.1007/978-1-61779-785-9_6" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=R. Sladek;citation_author=G. Rocheleau;citation_author=J. Rung;citation_author=C. Dina;citation_author=L. Shen;citation_title=A genome-wide association study identifies novel risk loci for type 2 diabetes.;citation_pages=881-885;citation_volume=445;citation_year=2007;citation_issue=7130;citation_pmid=17293876;citation_doi=10.1038/nature05616" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=B. Stich;citation_author=J. Möhring;citation_author=H.-P. Piepho;citation_author=M. Heckenberger;citation_author=E. S. Buckler;citation_title=Comparison of mixed-model approaches for association mapping.;citation_volume=178;citation_year=2008;citation_pmid=18245847;citation_doi=10.1534/genetics.107.079707" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=F. Tian;citation_author=P. J. Bradbury;citation_author=P. J. Brown;citation_author=H. Hung;citation_author=Q. Sun;citation_title=Genome-wide association study of leaf architecture in the maize nested association mapping population.;citation_pages=159-162;citation_volume=43;citation_year=2011;citation_issue=2;citation_pmid=21217756;citation_doi=10.1038/ng.746" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=P. M. Visscher;citation_author=M. A. Brown;citation_author=M. I. McCarthy;citation_author=J. Yang;citation_title=Five years of GWAS discovery.;citation_pages=7-24;citation_volume=90;citation_year=2012;citation_issue=1;citation_pmid=22243964;citation_doi=10.1016/j.ajhg.2011.11.029" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=J. Yang;citation_author=T. Ferreira;citation_author=A. P. Morris;citation_author=S. E. Medland;citation_author=P. F. Madden;citation_title=Conditional and joint multiple-SNP analysis of GWAS summary statistics identifies additional variants influencing complex traits.;citation_pages=369-375;citation_volume=44;citation_year=2012;citation_pmid=22426310;citation_doi=10.1038/ng.2213" />
<meta name="citation_reference" content="citation_journal_title=Plant J.;citation_author=J. Yang;citation_author=H. Jiang;citation_author=C.-T. Yeh;citation_author=J. Yu;citation_author=J. A. Jeddeloh;citation_title=Extreme-phenotype genome-wide association study (xp-gwas): a method for identifying trait-associated variants by sequencing pools of individuals selected from a diversity panel.;citation_pages=587-596;citation_volume=84;citation_year=2015;citation_doi=10.1111/tpj.13029" />
<meta name="citation_reference" content="citation_journal_title=European journal of human genetics :  EJHG;citation_journal_abbrev=Eur J Hum Genet;citation_author=J. Yang;citation_author=M. N. Weedon;citation_author=S. Purcell;citation_author=G. Lettre;citation_author=K. Estrada;citation_title=Genomic inflation factors under polygenic inheritance.;citation_pages=807-812;citation_volume=19;citation_year=2011;citation_issue=7;citation_pmid=21407268;citation_doi=10.1038/ejhg.2011.39" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. Yu;citation_author=J. B. Holland;citation_author=M. D. McMullen;citation_author=E. S. Buckler;citation_title=Genetic design and statistical power of nested association mapping in maize.;citation_volume=178;citation_year=2008;citation_pmid=18202393;citation_doi=10.1534/genetics.107.074245" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=J. Yu;citation_author=G. Pressoir;citation_author=W. H. Briggs;citation_author=I. Vroh Bi;citation_author=M. Yamasaki;citation_title=A unified mixed-model method for association mapping that accounts for multiple levels of relatedness.;citation_pages=203-208;citation_volume=38;citation_year=2006;citation_issue=2;citation_pmid=16380716;citation_doi=10.1038/ng1702" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=Z. B. Zeng;citation_title=Theoretical basis for separation of multiple linked gene effects in mapping quantitative trait loci.;citation_pages=10972-10976;citation_volume=90;citation_year=1993;citation_issue=23;citation_pmid=8248199;citation_doi=10.1073/pnas.90.23.10972" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Empirical Comparisons of Different Statistical Models To Identify and Validate Kernel Row Number-Associated Variants from Structured Multi-parent Mapping Populations of Maize" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11.cover-source.jpg" />
<meta name="twitter:description" content="Advances in next generation sequencing technologies and statistical approaches enable genome-wide dissection of phenotypic traits via genome-wide association studies (GWAS). Although multiple statistical approaches for conducting GWAS are available, the power and cross-validation rates of many approaches have been mostly tested using simulated data. Empirical comparisons of single variant (SV) and multi-variant (MV) GWAS approaches have not been conducted to test if a single approach or a combination of SV and MV is effective, through identification and cross-validation of trait-associated loci. In this study, kernel row number (KRN) data were collected from a set of 6,230 entries derived from the Nested Association Mapping (NAM) population and related populations. Three different types of GWAS analyses were performed: 1) single-variant (SV), 2) stepwise regression (STR) and 3) a Bayesian-based multi-variant (BMV) model. Using SV, STR, and BMV models, 257, 300, and 442 KRN-associated variants (KAVs) were identified in the initial GWAS analyses. Of these, 231 KAVs were subjected to genetic validation using three unrelated populations that were not included in the initial GWAS. Genetic validation results suggest that the three GWAS approaches are complementary. Interestingly, KAVs in low recombination regions were more likely to exhibit associations in independent populations than KAVs in recombinationally active regions, probably as a consequence of linkage disequilibrium. The KAVs identified in this study have the potential to enhance our understanding of the genetic basis of ear development." />
<meta name="og-title" property="og:title" content="Empirical Comparisons of Different Statistical Models To Identify and Validate Kernel Row Number-Associated Variants from Structured Multi-parent Mapping Populations of Maize" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/8/11/3567" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Advances in next generation sequencing technologies and statistical approaches enable genome-wide dissection of phenotypic traits via genome-wide association studies (GWAS). Although multiple statistical approaches for conducting GWAS are available, the power and cross-validation rates of many approaches have been mostly tested using simulated data. Empirical comparisons of single variant (SV) and multi-variant (MV) GWAS approaches have not been conducted to test if a single approach or a combination of SV and MV is effective, through identification and cross-validation of trait-associated loci. In this study, kernel row number (KRN) data were collected from a set of 6,230 entries derived from the Nested Association Mapping (NAM) population and related populations. Three different types of GWAS analyses were performed: 1) single-variant (SV), 2) stepwise regression (STR) and 3) a Bayesian-based multi-variant (BMV) model. Using SV, STR, and BMV models, 257, 300, and 442 KRN-associated variants (KAVs) were identified in the initial GWAS analyses. Of these, 231 KAVs were subjected to genetic validation using three unrelated populations that were not included in the initial GWAS. Genetic validation results suggest that the three GWAS approaches are complementary. Interestingly, KAVs in low recombination regions were more likely to exhibit associations in independent populations than KAVs in recombinationally active regions, probably as a consequence of linkage disequilibrium. The KAVs identified in this study have the potential to enhance our understanding of the genetic basis of ear development." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11.cover-source.jpg" />
    <title>Empirical Comparisons of Different Statistical Models To Identify and Validate Kernel Row Number-Associated Variants from Structured Multi-parent Mapping Populations of Maize | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"TFNPen5cg9_oWm5pwz4xG9lEBJ6g8jV8q9MaGOjjrck","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"491245","apath":"\/ggg\/8\/11\/3567.atom","pisa":"ggg;8\/11\/3567","ac":{"\/ggg\/8\/11\/3567.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/8\/11\/3567.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;8\/11\/3567"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;8\/11\/3567"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/8\/11\/3567.abstract":true,"\/content\/8\/11\/3567":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022MA4LJHLrruIhEoc67FALzBCUUaUaQzHkyT5iMPJRjDsHnyEC8g6tCpZKyWlqCuVTssZf3E+uCLl9SMGoxA4yWz0FfkNHLxcgG656EBEeTpDkgOJsm4VXJxNGIhRaSDEUsLjOwbMCLgtFXkctSOyAnrw8tI0S9LQQ3wycS8HOBFvGOXhyPhBpyD5qAEDk0JphOYF9QZO7bolhlRqK1YWrTLsmcM+AWcEKITuZPhKPXkNoRP4IUMaXpy8GoedI7JZyW6d3GYeSTxyh7qDbBKAzxHzFBKaqdKnPWG1QLVIJtDJ+oXZC1BiZqiYrgWbtpcbFVMo7cw\\\\\\\/t0FCJV37yV9cuWv899O3Mwp+cHzHazYMSwbN4wl1VZkNAjjN6hXXZgZnx9zjnbZnHgdq4bnVHfJX3GEhYWMewerHpnBqjKv5qoMkIM+Zw0ygIHl07BstEMahxq7aAwTjVbY\\\\\\\/6lPmQoVsXYNHmsoA3bXP+lTfa5ZHFf\\\\\\\/+i74UyLd+mDs8E01p40hfOk4qHJs3F5a4ml3SsxLR67Qi2nVX\\\\\\\/NYOxbsxCkfE6zQbfEkgQg7MJuucVdBiUuba3eLSv3KtUmEzz\\\\\\\/XbtLN\\\\\\\/JSbNoI9JBJIQ\\\\\\\/7Ck+xIVhY+WNs51xQ4z7MBexm3oDrOY6Cy8lwDcVdIMfzjnyNAP2M1Uc5shUdiZ2KNrKHxhPfGL9NLXppJNgo7mSW2Wftldkur3XQiWI7OdKXHyo\\\\\\\/eq784UcmeNR19nnewJ9LbuWYn8heixAFs1bNcS16D58mPYigFC0AijvSsaUs51Yh3QWH06L77Hd7XN2AkHIAiu8Wjoci\\\\\\\/x6Qc8uy7HlxLFSUnt\\\\\\\/KVAoaQbFf7kFjLhG2q67Wi5gZC5f5Z7YUKir88kggeaV8vA775Fo++DNztCfh+kQ\\\\\\\/i7EC2jSpz8WpqquflUq8YIKdYhSz\\\\\\\/k\\\\\\\/3YUiUBnVU\\\\\\\/kqgC58DRnLlkGAPam4Gqb1+p16YHI9x0kzNzfWmyjaHroX4QDmu81TUeEUdSfA3xrn2fPh3iWHuJiOGCzsU2nbK90eO2L6BRW7\\\\\\\/ZsF0xuj15Hwc7IoAkNGk+t\\\\\\\/6Pvg6aqYGasPqIp4eYlzYEWrbsqG+aQPFWyfmtSpmJd0\\\\\\\/g1uYSoaxSp4RguuNrWcJ6mTtU3kLRLZ8EPDDr5g5wT8kYWfEOh9LIQKbphfwnggkExpaO4GphodcXbRQNhQ8ZRbs5GlF\\\\\\\/+\\\\\\\/WtG+irjSXV8Garvsj3l\\\\\\\/\\\\\\\/TMiqFJ91DofQS7AMF1Mhs99ScAZAD2hy8ddbckOtnji9aEa\\\\\\\/tVICaVrw5xi\\\u0022,\\\u0022iv\\\u0022:\\\u0022L4yavfK5314hQCCevBg0QA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e211f230320c4a98284aeaba403255ce805416da658f9677542653f412048e4c\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022CwFyGhtmTD996vpF8YQrWXBTRTVN3Oa7n5vvFwpT\\\\\\\/AhnX5o4mdjjtWO1smIdgybLsqK9D\\\\\\\/POmvx2SaAdumwAqz+AONZhuCqR\\\\\\\/nD13EmsmuHXBSekAdMmEqJt7o9wGPHTPYjDXgt4\\\\\\\/kve29P3Y\\\\\\\/xy0LYjCpjs3adaRNnsIqjlBmjjzYu3V4vxO6dcZVYPjip3nBAedU2VzxRdeu9QacVufF2eamoY18VN754f320\\\\\\\/peTyhX29Baf7MzckAAEnwuGYHWGUumrPwoz4Rt2GOCdDCZtXH\\\\\\\/MdypRgoFEYU1ISHnSclxUv9WL+067YSnaP1QGKkA+qssip1i+W\\\\\\\/NN2WPCMsmAZ+qNeXq3Vi55Pd9kfSjvIbMHIPeo01pFyPkTYuxZnCLxWCEcqnPFjAj5\\\\\\\/dcxeBaOUusK\\\\\\\/ZU8\\\\\\\/YRPEHIzXHqqkVOud27QsPrDf2ommvQiD8xk+1qz6lncRjwQbiQ9hCpE1z\\\\\\\/b+uL3mmNCk\\\\\\\/aToSN49OmD8vk4ih9kBST6dwx3FceDRCRSkTlmOHcHHq3Qxh7lYDQccTv5qp4WHRKyWKNurBZk1xlqDy1jmP6dU20Y5jdZucfS7Ap1d8U6DhWdaBWPkglgHmCw3IhVDHrcMi7GsPsSpP2XG4HjGL6kABUMOmIlSQdiBc7Ym94\\\\\\\/XfwvA82aijZ7URx0H0UeNuJ+KQAX1FCCxUi5I02dBJHNH\\\\\\\/LcOYYe+m9BUh9V\\\\\\\/9jGXx+\\\\\\\/5ORj2cUQ5G+dIi+cHJfYZ8hiQmpm1lPBpIb9GfEBJxKVG7pvpqLq5j9jLTTkBMJL9wW590dbH11WbsCrStY81x2aKxIMr6pq7QHKrS5imgEoqQ9TZAMF2dQcD8xhVt7muiepKhj7gJ2RMX6ShtjX8bhd8Sh8\\\\\\\/04HCt0HF4KZ\\\\\\\/HdtGNCtQdch2gnvkyAnbOJLcSXEMxGTZp8gOpfna1GudDFOCgbJooANaY2HI0u75W2xRITAMURGpmZJEEe8LWM0BXpk3qD\\\\\\\/u\\\\\\\/YqUfPQSPhXNj2ZbC1PW4M0FExviHLB5IlFRDB+5jkaLw38oPL2VEa\\\\\\\/PmXbYEUJGC3M7mC0u1Wx5AheJoLQ257e+B8LEqoEbYowHWrrYTf7YKEDzSU7cf+tUYTI\\\\\\\/d5ntq6ZymlWtYCdW6gSBvjUvweorBWWA72Zsh2IFLE2lt3M9KUc7bwtEyYM6LSWm1Td8sezPJYZ6DJNNgyxvW7gsHl\\\\\\\/ZACNgtVqjPM+YwsyDxdgOILU4mCj1OpL8nY7M4sWC+qrkUOcnX+LaYu+wFQcGqwgB\\\\\\\/k9srby2RZBWto4UlLzIpUKYy2X5Hircn2ZDaY5U69raS0DWaMHjhL6jGc9KNMz1PZkGTvuAaKxGurXFDspwVGKNq1elPyK5IPwOQDPclA6Ji3WhVegRlkD\\\\\\\/8LfIH6B92D\\\\\\\/je4HUJNzhWjqKtdUIXdNQM28O6cPYkv\\\\\\\/JW30rrYq79PUpcvWhxD8E2QYwpuR7ZVCMTi4aSjAoFduGvxQEFHgmzmzGHf2OS+03wH7foBQoQ862VdCklO32gp5C8ZE7+UfCusVoFo8na\\\\\\\/lmsy4wZGLuX57A26h5+wXHaEq71XWANxZK6kPOPjpPVKgG7gDEY12plv49u969iQWMOKMnajy6fA+KlpgEYrcXrpG6aXblnHWEgBqSsTmgdbyZr2VGefyICnj76LYtWV50iAVEGJoMiGIOZT+lCNFJkDbDHVtnpucWjN4z4TLRdN8hVAWcyQ7gjRVCI2Yobjia6TZYGF5WjARHXJ3bVa105RxUvLVkxm7uj3jFNTOsUq\\\\\\\/vz78oeyPkCUw0KH+9oj0KU1Cx1vw0ztxWua3\\\\\\\/gebXc6EPcYrJn8pjTpOIGothIL8xy7FhtskNmzIWLosWG03Ewx+uNmpWHac\\\\\\\/+Ajk2A+3w7c3UO3vs2qH6h1HgHO\\\\\\\/fxqAF9gG+PBqikotT75zQ8dhZvlh+7K\\\\\\\/wp3fEwxDFWUhR98Wspl\\\\\\\/RIirrCnd4ijUBf\\\\\\\/TmXGwkaH7vg3N1GTu4uVAwGll4g2OZWQ6E7HOMgqHxenJokNKzIkxYflINaRDKe9\\\\\\\/+EADXem3HtAKpTh\\\\\\\/PrcnIvh1hGZRXcwJvkb1EejCCLRS3lPqtsX7ZTBwgViWTAT1f+bQF+izzZD\\\\\\\/H3SO4HqUR\\\\\\\/n+LtOAF+xAhlLsIcbA1u2hzTxPYZ4Kx\\\\\\\/qBD2JcB\\\\\\\/u3U0rfoidinuLrVjhbwYniYKl3WtqADxNPAp24KpP\\\\\\\/2l6exC9dRTsPc4rIMv7Jrdgmx8YNuqLSHURVkxmckESC9J6EBa32wDE5nwGilY3y\\\\\\\/vJ7jirWulDfFUYQSsEBK5VKS7gcEqzJGeWkTwHXM\\\\\\\/IcQV4U4B76Sm25uq1mLHr\\\\\\\/kO4isgcdAWrjImG+wq+Y\\\\\\\/b7mnnem5eHU7cqf0LfQSSR4K++xnyCG1I7VQ0UthJguDAAaftTJ3037sSJSJn4i9yqy2kubAG0W6nlLXqH8LGESoDqW46IFWtVSlVH1Mtu61DLAbr86KL9cJLetPAqar6+b681Ne30BS2X8nPFpiCqwWskEqq1\\\\\\\/5YBwV4pm8zKzu2+PSuXAjW5+tNFCa\\\\\\\/8WFa6j0nR5+lx54JNLd6GJXezMDEX6A3SeMdmytGtYeGjUkWE2mHNVen4tpbEyln4BzS4XNEVzuuvWtTY9A+KkNbLVZHjIH6BD3bO4gYzqH2o8E1wbouP9BftaosfX3wQ4czEc97vpe5JG6ANpNCYZe2SQ6Zmh\\\\\\\/W5hzTkMLboC7DC0i+llcLdg1scVOh2HCnW0W8Gyfvjsylf63GQSg18fJa0GTH4PQ9A4uK8ifYX+RXIBfnEi\\\\\\\/hHUpLhh2f+4qEd9U1RbMgA87y+2VmCWhEgDtrF0zW0gyKh2yTgZc2CGtEahF+\\\\\\\/kLbx9qDKvYy74lwiFb5riW9J6LJVm1mIz9\\\\\\\/I25MXxKo5SyZ1OeiZeeu3qX8dMVP6wXbOTYezQNClaZ1EPSrkfwPyf6naEuLmlrfpDU1kDOiOlzJHpE4Wl8arIcqtg7GKEyDIkMFwnGlfd6kpSf6Wvx9erlDudZ67z7TL1THs4nhbeBhO0yeJglGYynUZKo+sencKyGSHQbXiNY2ofJS840LBccp4V+idSTm\\\\\\\/jWem2uWpGpBbznt8ujlpVnlis7azyAyqlKRnWdQEkm0308uENdbI5VquwuqZjD2n11mTKwPcAmyCLHzBleUFZeRrBvsNoU7PxrkewrfYlFmoH6+HJwWFVIOWuSrWVBJwz52Rq8Mo9cqt0DlMaNhGT52bcSi6i4CNWFPplwGL\\\\\\\/aZO7hyPI+DchCXRs4Tm++WF+QVgVQBRW\\\\\\\/qGE7SQoj3ILzD2HZrS7MBFiG0CepNCnYCu5QrCMmeG2dAitwPyROu2txQzfQkM4MqU+YMOlmwlFAfTIAsMElL0\\\\\\\/bVvODR7PE2v6h4jX2VK47yLK+v7IOjCsBoXh8EpRWBVC56rRPgOi4f9G0wCcJ9CZ\\\\\\\/xRb7VKoq4XgNIPDXVRkFB7uofKC3Mh7VXQNWQhZUseSvG8vrQ1g9CkRsH5haUVLL1RRdkx2h\\\\\\\/IPv4Btg8LTuQC+KXZAeBRzgoq053HJ7oHlvfEQTqDLDwzUygwBRfbVM7CGw7BWxqv+hqKFgFo0PYgt7HVxqWRqoPPMKHeFukgU1Ca3PGMUmM6IwQHpo3U5ZJUqxBaipI4wLKVeyW61l1kDYlWUl\\\\\\\/+1xolxW4NPRGE63xR2k44aEzx0xm3HB6hcpQkjgp0Zd\\\\\\\/LgkiEud6O2MZDQe3e5kttRPvtTVpGO1fyWhyn30ul2dz5b6ugmUezwE8TRDrgwz6pAMpPnvUSmdJ7nesAzldYv4b6vrOHm2K72w9oqZdH\\\\\\\/F7nPj7vpwPmT+ZF5XYPsoNUftazJd1f4lHNGMTcM\\\\\\\/Lun1iNmh4FlHa4ecwL5F1gFbGSdLTygspD5BgrHwqwI9zIqEOBOcl\\\\\\\/cl1wXKtaKRjhcyZkBKLs0Wj8Ua+uEniVZ3hJqUcSRuH2Doi747rjPnlU1ubsSoAZ4KfN+su+YwMIVDs3RvS1EJsBsxIym05wfLZLxB6Z9ZzkS08EZEDUOqC1BqWpo7KCI3pJ3Ucs6Sfp0Ryno+N96PXa1BIRXm5J0j\\\\\\\/RsCRNZbgIU3yvbTG6yjdgX5VBmyCwKoDB7HnGbxWTTDdp+B\\\\\\\/y2WfAdAxuKatj1BlZNuRi8phYOn3gz3bT+gknxviaF0uydu4dkUd\\\\\\\/TeBoQz3MGIHed\\\\\\\/UnoJW\\\\\\\/tVYuJML\\\\\\\/ONbTlFv1+8ms0pz2qdeAe97l93MMQYQnItOVRqYm1TlLn5aY0++qhz0\\\\\\\/ZIby1KupPT6e\\\\\\\/0q5lpqHA0TbtlS5hFtAOgnGj1XktKw7pnDAT9F7D9J1i3psVvQzs2Hd0yGjtRbdsWVPdr+hXcpYod7BL5R\\\\\\\/7HTggSLSnEe2jkg0ms7gl2u7\\\\\\\/juYI4J990\\\\\\\/kZg\\\\\\\/kVKLRLiLpt7wa4SxXiPJYIN8qTAZnt\\\\\\\/naFtqBCErcDshtYyQPFDbFLeblI0F29XFFMzM\\\\\\\/7c6hbKObmhb\\\\\\\/9C\\\\\\\/JYX7hhj2cwLlwXzjGqsLgo2fSSpgu05CPj9N1Y3W1DxopcBmb\\\\\\\/Z074pwNBIfg1EKX6CPAzGraCSHikhzssh0Wd5AZrlGZoUdTcNlek37mtUkp2wxv7WyE0u8G5Io1s7rsIHfHKWugPV\\\\\\\/VhxYq4BDQAQ8Q6BWMizew4cYI6V5BJYzlhS2sfyFsr7HEh2fnG2WuLjp\\\\\\\/UfXtDBrU3iwoB1vjt4eV5P8KCpGzT1mqMgUKjIlinj7BQyQFf2yDbIXha9w0mCVmU7wm8XvQqsr+Do1ge+olfG3ez0\\\\\\\/SaXbhBDl51iCxuTvY4nWQZkNXyBi4IUdOpYWFxnFzfdnbVB9HYVislgMUc\\\\\\\/Se3aMUBwEVAxbutOj\\\\\\\/YnS+MiSXVE6KK\\\\\\\/LgwC9OUmnw\\\\\\\/E2orxGPHEq504Hd2iJSEaq0skG4ZN6S8rLjDrxIWVMQhNZIkyEGJQG0NMB\\\\\\\/Jn2L4ZK1n\\\\\\\/uUVrL5v1DWIJU\\\\\\\/nFiS5RCxJJSx3+6JM\\\\\\\/sZoxPCY64Lpcl\\\\\\\/q8IkK8yf1V9AtgzEYem7jq0AoH3pECBVsMyJR5QdF3LYQ+dGecYDehmG2Y5mu0iGF3OEtXypO7dqH9NgfsnP7cA56Va1e\\\\\\\/+x1VfTNPEe4no3ARpcE0NkoUQsuiVRCwCbs6bofYaNgRKgL9yNYMJ8BLdend0z4KMuDsiCI658XMHXPfa2lJBTWx9BCJ14A9qC4\\\\\\\/iPqGKfzjsE\\\\\\\/aYO1gSRREbHz3L7weo+ppbHK3q4V4nXTL9yhuyB1WiKAhVM135OOHHRfQRUgVV\\\\\\\/xisTa5aaY3+MPsEE33uKzhGRZH1Rj5MInw0RciPRlrHxQL7v4o+o3epkZSoYtBY9\\\\\\\/COWhNgHpr5T6OP\\\\\\\/WRkfbduy72bRqvA+ialmBDa\\\\\\\/fFOKQb7lnhSd3nhb+JoXyBqPQ+dz+NI3Pbv0BybqKs3Hn94NMB8\\\\\\\/\\\\\\\/DMSnRo2KOf0orKkSxk+TieaKC5YNuAMOs85+QLSMD8nowlVtVlIIC3fDHSafqbgtwSCIA3ambyNpXPLd6ldtqis0VWKtk5zCGcWagF3TrbMM68aSq+\\\\\\\/9qUcUUCFvQe\\\\\\\/NOQ4S\\\\\\\/pUlCCyx3gdQcsOaY4uz4ml7ikc2Mgbb1niBRYGIwxqpKqNf1ThDi+eBe4Yt0IkiUxpVWmEcSke02e1my2WRo2y3nberuiOZdSDsvbVXM2+7FO9pPaPfHPi9IJgsaUXRF6cnXH7t36Fp4DDqjmKJfx9hHQT5co\\\\\\\/Z\\\\\\\/hksdN1nV3qcUgXnuLM6U7WmhQaD7zABJWoJL2Yuy7Tca3K6JuqHKwOGY5SvoPgLfrrnWJ7pavTP+n88RAJHB+y\\\\\\\/fThKiTkkZhWzxaUft1lHaCqC1DhQEpPytQxyUZ2zZVlg8DISArtkSd0bmaJFIKuScjRUW49mUoG9sGPwvSjJpf+wkFydenyq8bdZq8T2aCK4gK9qmt3g+NwnQ\\\\\\\/qJOzfSE8Qjptgnkq7VnWJq5c413b3FpJupS\\\\\\\/gO3k7NF9\\\\\\\/1voDqgx7Tr3mnRW2qPruZing4mKwed\\\\\\\/Le91JvpeN8nNAQ+gNFJDY55AjMoKoh+Epse4p7SBx78KRefNkw2SgdXtyttaNfr5wnW2VniTlUfN6fsYBWfPOMsPKLYLjKKC1lNjUezoJgYewWCwVP5om\\\\\\\/veoAdh1JkHM7l1u0FM1gvs3qXz2H\\\\\\\/zmEuYyBc3Bp+pNaxfIlZamqQJuL4KI0hf1yIZIjPV5VZ25z6hFzzL7cry3LLGzkZM8T6SbqygWMCNW6xylgvlMjSyJmP0N+wyZKjKnjC8tc0rs31AacOWH2tuFK8kI9iHgBAKFSy8C\\\\\\\/CV9loVj72a1Nir1i33v+UIFyG77DJF5t5gSj54yMujCw+H821lkG6EkVIdKPtW5tyNrTqAGfplaDbFTYur7yEYlP7KyGtMg+IlLL2opOqLfy6laijenYlo8UeewyqD01lAkBYiKdCOhnTEmW+2Jcd7ipcdoZFpHrp4t0gPDEBgRKXDJEMKblMoTqaafkn3m2y4A\\\\\\\/gsCXwA8jOeo+PagjKgw\\\\\\\/2eDYjO1zstskWGWjOaypcj873s\\\\\\\/SLHjA5GNSqogDgIwaoLSDNZENFBjnzbB2lsGcE9uc4mX176JzOt0M3jTtI\\\\\\\/MPOFWIseYt61YL21rKK5KOtyKdQjn8uXRL+hnwx9CEWi4R3lAEJTiyXsl3hFPFNbfO7NXGhyAWiWlUA9iEXM2tc4dh+bKuna0fqFCQbxBt28LqZzLFyYUoo5kkwhPXk108S3\\\\\\\/WhyIQ714HhP9fmXKrHvk\\\\\\\/xjVFiTRmLyWXq18PvRw9akJ\\\\\\\/bO34B5i3sNEIze9Sbzf27Qz\\\\\\\/iM2tvhoLA5eaiYGepymi\\\\\\\/8MwjhC3x8Mk2mMAQNOwovo79ypT+fOeNod0YyK0g6rbCR01ptMBtAaaxo4fT+8nsCT8GvV\\\\\\\/qlDnHr9+km+dYTSfBVtziWvuQ48Z1OFQ0lgf3F0jKiAH+G14muVcaRRUyDgfttqboHG7gwY73gO427gURHO7LU\\\\\\\/2Iz7Bs68GuNiF7mb7KJAGP0AXbNK6VDiPvn3Fv57oR0KIjQ3RyHz9fS98NA35edlTMYsqJJ0a9qEHcdDuLhnTaw\\\\\\\/wUsDT2KRpztsV8uQF+Og71XS54LB0XUfb5We0vf12l1Gt9PsxXplnqghP2k1ISd4pRMeIVfx1Skv0dLxNbag2kdR\\\\\\\/qpIkA0N30bPVc3S68Mswz1aUJC84dAsViJejwxoZZVg\\\\\\\/mcPTUjTYlfVCI2mHzUV2ahvVj17D5ovrcxRkNmpgYokTEyiJ6MDRKRo25p8NWsDo71bGxWZUIAKiS\\\\\\\/gxivY42d2DkmXfH6Adg1jiQb56gpG\\\\\\\/IB181kHrOdP1trBGzMPf2gvtowUk\\\\\\\/C8dmINo3KnD\\\\\\\/ozcpgl17EN3RCFOGHo3Y6Oon1pO7lp3wuk\\\\\\\/DUMYVlt3HxtxyH6msP\\\\\\\/J3ZhCpx49wY90hdiWS9l0yPU2s0+2fCaFsKWfmW1zCkoruMsSUf87ZqzqDwjtVVrniV6IB5x+elAk1JIl7lPSyMsXngM6G+1Oif4TdAt2VM+nur+Msswcpj6DRhYapKvxJcXHWKuitviTfZkZMPx\\\\\\\/vs4e\\\\\\\/q+wHwVZbkcpuYxVguJOZErNn0JZ4LzcNQ6CTzW6PRNMiun6lyY6mMpcXHYlfPR0P\\\\\\\/ZEFi\\\\\\\/dnDRGHoKyxqpX9P0kYWKX7XD3NtMR3hmZICOAx+2OtPt1CchQw3dxVX+tPYKACIQbu7v2p7aJfsXNRTKPgvueWAOvpoZUGOKllQx\\\\\\\/JtOzAO6Fy9yrbnmJtNJaiwhYleNw7dnnq1qrAS1JVec8KQRq5G2LH98uTYZZTNg6DRXtMji44VdzZDUktq2z869JJUn9oRdiMQtqUM\\\\\\\/o5DHnnJlEASRzafOs3iX3rO1BJupQtb8Nb4czuZQ7P6hinUuxsT4SclxbH+Otkq3KBDLlHtMAiag+toMDcaCFhXBUj\\\\\\\/y1nVJYJfHtEeH3HDs0rW5QASd+xSyu6\\\\\\\/NYWMXaC4jeI8ROwez+oNjaqZhDoKfb7adjvG5ZIae14cHD6buzPdxCFYNjpC+M1KpxvW2GfAHgo8HQZDUwR2rEUNs1WEOo56+zRu4tNbWQtW0afNSKGgrbvT7tOwvyetPQyTexc2iin63UhNv8tZlMurA87ifpUWmt+TRuwest5bNZbkfPslaLnUcdMBsiY9sF093VhFAcR7XLIABBF+WTcYG7INGK3RAer+nmGMZEMNrZP8Zod8tPY5gSqXwITOA5JzJclalc1Xgqm5GfLBLV6DXhuQ0hVuD7YrLMqSVrRMW4sk6ramHMx0b32vJp\\\\\\\/ZAP\\\\\\\/7m9u7ktbN1iTUDmyDUQdgqtbSmeNRmRBKEydHpKXeMbndtnFBy4JmjTTIJMj+t1xIJ5zPB8JP8FNz5ZvCIddEdMG3sA3lLm1UE2v\\\\\\\/zHiglo1RzC02YyahXzzZSAC9XQzpFrTjsr3Nxj5Nn3k8afAFZdnfnrIvl8Tcgsyfu3LeaC6Q+3JWCU7TlbUIIaynCMchamIHSU4GGn75KJX2SkMacDFeqobFqcyraW\\\\\\\/jq7jdcIOtGzWeL1Ov\\\\\\\/t0JosFCO4J\\\\\\\/K0vkX026R5f8R5GcJM7HltOjGJ0+MoTe14jjTHaJRn6bRahP2G4A+neTD5xvujAdB3nDYGyZDOqHQ3Pba8ce+tKZKKG6tbSLQsy3s8newVbk+49iw4EJvugHu\\\\\\\/mYPFaRv2k1IcYTPBScoah0jyUPCBdxTz+b8VG91l20yYoJyMtvQe9YovmL8dRgafBpAc1+vwSMBwxOVKMJ++G8GIqFNF46eD59BHvW+8Ep+iR1NOH8LwQ1he2S+\\\\\\\/mhCyeIqllGCGb\\\\\\\/am8rxqvGCkvjItvrwkSNO0i2qfWqvYDCgOhoTjqHjH8mWPdCobM6nIJLD8emUCULDMeiBDm1PDGLWnda7a3utqOEW+vsN0\\\\\\\/DBUr8BidnN2rUT0DMPSxZ1P4z66IjE3RJ\\\\\\\/OQuUNcmHcInnCwYJsqHLME1I97xirHIRjOd\\\\\\\/QSKkxkFO99ZwtnCjZWDiJ9wXbdc4YZ0+g4yp2Me3l6fUtT72cmNpqv5cdqU5+2vWzZqCkacDlPJ5B\\\\\\\/RG9JgvjIBtWtwg7vYXpbGCuN9OGRas08Sq6nhTVHzfisDhgregGWqYa94NfdmTKJyu8zKLNkfi5qcbQ6ZJm9fLWvFxA4qQpeeQ9Y01ilwStYC6qKpTjcKHJHxRhGP2mWb7usVnJ6EFvB4L5Mm3kBs0HRcem3PFsORdiXEjXBT2ottkN7igjRVZ6sLS3AHV5Y1ZlivnJe9kJu0TkPUTu9rT7wjcDwj0Bmz8dhFHDrV4+LP+3\\\\\\\/FeGpxE4lXS8r9pnYgrAPQSUrkYf1P3wYh1ZIrIHyFDwHyCCjEYv8goPbAB3VhO33lz+TCXF7jX4lvssuLAk9sN62x+wz+TXn8ZSRJzgW0jQpva6EAXlki1EH2tluBP0zQAi5nx8r5C\\\\\\\/C8FAz1KfEnV3aUVFT6dLLOmnIRaJVOqJ1bmZ8KwRnMbBaZCYfYCVdaqMmmWrw6VXEs8o\\\\\\\/uG+cxKVlzAd+evQyI6MdgpGZu2+u3wqM9M3hCVTFueBYm2jOdETQRhK77wXP7aMIsrqrmLbEw6HRcNUfCCMTyZB2RoutCkFpXkOppYwRgBzVEGW6sTCoUb7RdkQ9wBja1zt4KXz5\\\\\\\/pI\\\\\\\/EIpCronruXnwjRzs\\\\\\\/WsaZiRZYuD\\\\\\\/KWANb+sm7roUAjSWRFimbFMHdDCM8Pzm4GsHiZd+aUJweK9G71+Uo6U7SImrUWi+kMD+6FB5Mt0GqGqQJQ8\\\\\\\/gNqSCYvbnk6m8Bw+\\\\\\\/F+5UQ1LB\\\\\\\/sZ1vGGgi3xsi3wQWCozXf\\\\\\\/5P02N+SPQym15hspuZJNYCpSP5INayq4KKR856cM1AlDy16x9TguyKMFz6JcxPzcl0L\\\\\\\/JlIZsgxVrQfMnmA38Zw9GR9MrFYghQaVNvw\\\\\\\/OxjXwK\\\\\\\/dJQestyR+R\\\\\\\/WiLi40xegl1CZsj06WFqgWz0OFVcW5VA2z1\\\\\\\/juwae9+a4wS+kq9jJARGgPCZISPU0DmTh+58lgznDCMMYDqlY48+SGcMU2O+KhdxvhqmQ1mjHvFNqToIe1RvQLTLIUhSXnLasF\\\\\\\/2koME9BRHryjP9xOLPfcP8uw1dBte9UY8JSdf5e8FuqKZ3JAIqt41kNjph\\\\\\\/O0rO2DyhlEcGYQfx2kbCeuUfVLelfY+YuGUF7Aie7sbEEjXhpwSAzG7OatBa2VDqgqXPgcwammkKwkKfsDJFp1Skm2A03u9opI\\\\\\\/1mKuUzpr4cDAJcrp4tWJ0c19es0SbTcPnxNJFJE6EbpJw0MZ\\\\\\\/wtWIh0QffiiPAROytcQJlTygaxSaP2tpoefaadmx8Jsv1opCLcJWhQ0PhRVXqYTAD0wBnpEaKqui0kfdGfj3rxiUwT7evVYLNYDXSa5yddzg2ZuJsuu1zoU09KE9uu04PYitqpAvqlFon6ItD3slv8vNuR0+3JZF1B4EXost04jG1Oi4i2LJYQ8Crmz2Q30Az8l5D3v9haPQ14GIGSCsHctN\\\\\\\/dz1nev77JeOZJpMrHR867dIzpcV78J+kR\\\\\\\/IMMElwUt\\\\\\\/QiBFvtHZ\\\\\\\/8QDuF2BDiffg8c6OoilgIdxolNxJOfhiMhfE010OJpVmHBXsX+ZZo\\\\\\\/GlIs9TBdjuizPtlFxqxavsWDT06k2wYtGFuFkBnnt7dCyfzTM4kdtfQeNZKAM\\\\\\\/F1KJ\\\\\\\/H2SYheNxvkOI5nX0PCmtmafy2H4YqT6y8os0y4nJ62wm725cst+nCRcwwVgMiVF2y0jW8cW440AHAL2nVuVDwHCtbWuYmF3VwDq29hMvPAfu4a1RVXya8xxGhdvNQVm2qwL0\\\\\\\/wyuEi0k9s9FwjmEXsJXI5dcbWQrn0GCmczR\\\\\\\/UVlPh1QovwSUExZimWBxSRlvQojzTDNuOaVUtKVqckvuGyW2mOYdqUfWYOziJD97obEVyZ7Dj44DgnITv8UbtCQcQECRozVg5PDkjJsaiUuvNFA7Zmn4WAOVfU7CnbsqQxJw37CGLoQJ3rkZt0vacHJYW\\\\\\\/k+Sx7ouiGIHVeWdTzlUF2nwNaoIm+B9DMNliQLFcqhFjkWAeDWpskm16PYprM4J\\\\\\\/DbYPTmZyxASMwFob4aKR9cOGrY6VXkFgnvGlX5ftpB4DOBm8CFZH9aEO1eoqXuUmoz153bLm1k3z5n6lBO2deBLyFNQVGKMT6S3zd2SSC00wKNFi3fg\\\\\\\/lEQ1s11dPNo3uEDhUGsNNA4FHx2TMBWFHfduNVELriPWuDpILWI6znzb4uvV+SngkGcy5tLbNb9xmB47kCR9QEUZ2ClI3Wa+iINmSk3gA4oAMfv+gKJhblntAxzcVd8dbtmTjR3jCvE9qhfhTk5L0+lC5vntyXKULIAh5A2P9ROjkZU7yNfLsq7oNQ5GviE40+iGsfdCU2bZkZB2BRBxnJ6tpnRGwvGxBDNY\\\\\\\/Osk5U53cqR3xYIwCSv9zSCPJVBcWmTmMQ36eSGReECBKWaGvmBjYaeYVGHIwDKZM0Up749kQXj5r6\\\\\\\/UlAhEY8lmlcQ39Pp09pcpQuI544BRzyjqqtgQYc13zJwZQCRi\\\\\\\/bWdFOTaC7EdZKYjfkyDtU3f4aNDIzQ2XsgKvy4odr98OGXUxqrVQl5E4d7kpVVaQ7SVTP8Ke+cxSU8YbW2l8p2vVkLqkHqMJkvGpzxP7UGAN2h0ClP6lCRAYDtLyBPl4n29U7xr1V6gb+eKSxfAe33BZXgdrf0\\\\\\\/Fc2QNR64kf7O18cKb5zQbLgPzt9PRT\\\\\\\/B\\\\\\\/PcerfgOYBIqEZZG5drZ\\\\\\\/BDkHMX5H99sGiDjk7nPOhEUwV3sRdmeyKGwoPvzriRv1YDqbLmyavwm1W1x2ttj2z8e4gdwuYKEvDuvOdAg2eryEIzvdN+WcWgS0bLvOaNa6TLu9TEvenyTXLJO7qu8J\\\\\\\/YGX1ymoln\\\\\\\/y2wGd1oet4NWwSlqkoAPYCS9D4oFaPeI\\\\\\\/dumVcWftka4fpFoY7nxnoCS\\\\\\\/AZ+r6nnqsydstQ\\\\\\\/QBRgjs\\\\\\\/2TYx\\\\\\\/cMa30Y3qKwAlfRKBPisCAH3Uv+LxMXTHTHi+Fe3et3\\\\\\\/3YNfTwz+Uh8zn206SVfujfjZDK9Yc78w9HR\\\\\\\/EbUiSmqkddXqPlcXmtJQi9ekA4fyl5fyn1O3P\\\\\\\/VxcJ5myoXThhCvnw7+Pvi9CSqwLSD1CpS612zQkVEUA63JXv0ZCEYJ+G2BXNEMkPOUxNszAAz83hLS0mm7F4n1zV13KOgXbSEUON+6\\\\\\\/c35SFEoDovZqK71rb7kcH0G7FtTlojmdkWEW0ncZj0sYJXdhYbq7x2kzmG32MIYp6MsAnJUctCZc0wbIRsCkbipZYpTy8fh8NJBy8V+B1XWOKFLtA2GqFFh7O020ADRNaE+xB4vOlt6uHz\\\\\\\/wUKp0fsKt8MkV+Ag4AYvZmpZZNePreaiT3nhi3aRe0rSEEVRVm9rUuBrvfg1\\\\\\\/VROCWBPjE\\\\\\\/fCD6CV1Be6bbQfokgRLy0YybObSGLaHIxYkQRe+P741ed\\\\\\\/U5UmsIXzFKEQBcBTsNYtvavt715lBYD+nMviK9IZ3kCNfgZPKaE9JKLoIXT4mOTfTmXbOWkMkExES+1RTzX+K6DY4TXyjqjPaJYaclSmy58KMf8qWz6WDPy4j236zDO78\\\\\\\/HTBqH2dJErEpBhMXcs3jsaZ9CGudzrVbAtSa\\\\\\\/zVP9xz6xXP6Pv2Ds6vVYJBEYCnAy\\\\\\\/23BcNjaOEAmr7uVOHEjkh\\\\\\\/sx9tZG1wrtyyS5QBT9L2HhOWkiIO2K5AVp5TpCIJ8rkHoSfRq8uDJvWBxykS6c1F8U8EgG5c9Z\\\\\\\/x7fZhs6waaEwe7D2wXNzVJta+BzTQyg\\\\\\\/PAk561YJGehAj69KJK2VHmp9RMb0JN2leTWVYQwOXd7AUp94uFg83rVKTxyxIyRcq6WROe54718BJOTQmpG3AgEiw+EQE8jH8SmAvTtFVcsSwQr6IkbuKvJkMcEFJhZwMSFWkMP5w6wRGVuMMCMxOsm+PHKiSbRZ5Bk0qNwBA8bigDSmEytTajoiDtWBK7e1rn\\\\\\\/\\\\\\\/leRFO5PoN45dsYubIzcY6MsHoEZd60gfhn3xEtu0juZIulmHMsBA7\\\\\\\/+cWmp2NogvnT5viUtmfWGiY9CrDx7soypyuLW4ZvM2BjAmODpWfAqiG4YJnodxt4ryS0HIocr5vc5lunPHSD7DPuytw8\\\\\\\/HAjeNbKKPEH7ITlhYb7cP\\\\\\\/658Oq9jwWMwvrL3YiSY3dxGANpkh9D+4UF+3Yh5QSWyWoKydYjoUKdCDamtMhdb8PJUn7a4MU5sYubpzKOOoRQzicSvSe97PCzjdnpk5z4Osg3iodqeqUFCHwS\\\\\\\/63hD1IFY3BvXnQ0l8abyImTGE8G9sWccLqAmTIpAS952izqcKvBNVjB81N2jHHKL4ah5tWdOGGVyL8XzaMpAsxG82sro2SERS9pEL55ySWUbN8nnhP6YSDM\\\\\\\/KZC3zh8dCRA6JeTTez8PR1P01xsw76nzUIecwta5ASvw3GDu3e9L7cCTHkTmKPwKVe6E1P4gDAnnyOIKQkTA154m7xl7EaYSU+P6z0dct+\\\\\\\/o7yFh0LsosHX6FmBwrSDJJjbb8fd4Nn5Q2wHZrsv8MAhSNgkWhICK0kWb3TNzT01N7idqc72JiGIUQa60q6X9Bu5rRF8Mt3PpCih2lsUsd+JFuQTbkNDtJYfjkETJkoUcZ66eyPhCHEXMNh0\\\\\\\/fgSXONIkdEhc\\\\\\\/4pyMyA4erFjoaQZxPaEt0BcrPF0nxbjaFchVPN4561Axn4FdoaIcmIjFfszHvQXkyGhNRLtIJEyKjlzv26GYvzYCqVyONB\\\\\\\/WJ+vy+Utu4j5bMh4EMFESVR8BrbLoAxi4p4KzZVtpia7fmR\\\\\\\/ouxPCm5fOtqAm2VW6OVMGSeBDcBLv1ogh8vSQ5SYG4Ae68Zad4LTdwLn5OWxUzCkw4vVJl5vTYfdVx05trvXDPhQ\\\\\\\/4EBFNfzdLs4pkoglkDtu0V7S7stGSblFGd4KO7pVmsdvTfaFGZjCzPVp+TqD12z0CG+Gd8luz8Nm1aFcHZwSY6pyi6xW3huKVBXMnJyYRgK9jbyjpdILV95MfnWpMWvd8ja7Vl+vm51oV0nTZ6VqZns\\\\\\\/rwoxvB60EZRDXoUjlBFR8J6IWSfP3EPytRy6Om36h36qRk\\\\\\\/tLCPkVsjnvCsHd89rlY+VIHbMUtcr8kjPEba\\\\\\\/WXElJbvzAIRO1ZMIJ0Y87hgwW8NXh7sfTvb0mFDuN0pBQ\\\\\\\/Zk3xPi+9pP0ODVWynYSzYozHZl7XYKqPY\\\\\\\/f59OckqpqQPCgmlKR8vA\\\\\\\/Wvj4MFnzUrzX28wAxgqiyDKaxBqPrbx7pxtv7YjrENe5cLxmpDBRpNPZln79TO8TbFNlWvYzdb4dxX3EPM1HUghwOp1dEzG78MWQsCtf27yYVkMzjiX8fjlRy0Rpnenu1IZsJQyG7iyt9MnTWWoBFjCAfn2EDYa9DeCb8MDXrzI9gD7uPUAfY\\\\\\\/GXKshxH88\\\\\\\/s8SxE1KT\\\\\\\/szv1M\\\\\\\/iSB4cGUHXDyrBrnjnJ8VnCUWsuzybcJDJknXx7YphFKD4eLiLO\\\\\\\/NEDb4fmcb1E8GJ5jobjEYX604Qnh49RyP4cx8myPZOCIHsNIQIwRyiFCeq7GTbZxNSu2QY1PsnVOxE5XRcZCRqcvpxKuRI1fAEh2vX3t7W5GDihNXnR5YtJC2agTfTiJr6QWI9OS96LbGzkKrbTgF\\\\\\\/J8ckjGBtrJgK8tR3PhOmOSU\\\\\\\/wCJPmhc9yUNaWk1qcxcPbVN8yhGC1aewFs4hsD7Cmvx5rRRZ3uSzO\\\\\\\/GL+KWKOmTR3tGFemIshxuwUuBXegsOxXk6q9MOePDUd3Hc0nOnIAGT6Vd4dvQt7QFoNWIC1ZU0l6dnhwr+VKDJBaDaloyp0JLHSBrQ\\\\\\\/R+IRD06Rdcp9O7knoKsJe\\\\\\\/nzrTlpodXqzKOWUMM\\\\\\\/YJVsxnVZA6Uisfr51MeMduKWZl6vViYVezWaFm91bQW90H8aOJW+4l2EZjJYPMvnGP5iqhxhgBsWcxA4tR8DSjdrg6XjJMUL78zxJPxop3QGIAnxh9kojwdAmzTSXfQZ\\\\\\\/Y7CEH8NWpzbhrht2YTTNhQmd9Cztb4HVWdcV92q69OGTyt7QCOD2IXe6tfaPb9x2OH8JHXul6pIl95nTPSrlPPgggikcDBExRV5YaIytqQT95ZmcSMTVr1h\\\\\\\/CoeEyLoAISOlk6Ae2QajpnU\\\\\\\/SXcI2K+7ZCH\\\\\\\/7w49\\\\\\\/QCMXFzaFmszasYnJgUF6xZ9F0VHNPYjKlWAmeCSO6R30tSZFzYlqKprvcPqUiUvbblV0Rc9EUKTztGJf+PRi+g64Ojx6+mAJtl9Tl+Imh4gRUK6JFwNOtTKCfftH3am6t8jcp1525nx1g9\\\\\\\/O\\\\\\\/1sB5Zoy0rF0c48yPHkwTtqQUqh5UHC\\\\\\\/KMLJ52l3rHUHHJBqaQveOzMoYTqXwuYv\\\\\\\/Bw6e58xcSuHDOv2\\\\\\\/MQLmBow6HO7HtWhqDT2fK0PxD94fRSFgFVPP6CSrTEhLcvekeNFb5ZmMxkxv0HfJA\\\\\\\/lUHFfB7vYF1Us4UFwsMFwUs2p6igkI+A6GKLtkFRzHihea4SgZI9ug45KXWORq5ifsGvKkvGLl3J9\\\\\\\/5dODI5McFRTCqtX1ux75tS1uKKljN3Xu3yoxtyAjXhXICzW9X3KvH2JkjSIY+OHXYhKs+IbDUh3lFnNrn0tKxT59Y7KUNyk5+ta3UZW7alxuy+gRghuxflwXx6ToLbZrpMPPCzNuMd5ThVzC\\\\\\\/37FXRDreXX0Aem1pa8dloCbkbuc3Dd4K8q23Fq1UbEWLI0cRBcVgIZhn61HL0ch2OyuiRp7N12jrljuD7nvOK8enIRIrioGz6jzHoxnd6dHMjDCSA60ZfRY6rbf5MV6HrNE7KY42ZEK8GN7UOQ0UOafETl4xHjLIsBemgQRvrJBfduPhD6n3RTiZCUtORtZc5bAQy8Flnij\\\\\\\/DATeJUyJAtlhVu+l5n2UXYiJcR3M4qd7kV\\\\\\\/ZVs7MIo7W\\\\\\\/VwbAMzr7uE9ItnP1YXhciu6Z8NOtlvhoxGea55fWK5A1KjlLL9w+VBRqR8S50U8UmuXB2Z7kjKPEL4MEIA1HX+tVP3Hlj\\\\\\\/cje+KRmZRq9Jhga9wXbruAI1QbQW3qfg2D7kYVmh3AIkGp5JoNdy+felZ3ooPgOXiUqXjfhnwmNLXBCkKRhCfdH4t9+wJQRR3gWMqTJSlLclhOnTJ6AJgxmM7BE5GyFTBv9TO8r4tIa9kpNqkztckuaf8xvN2Zox+fQ42C7FbzagJkw6+7PLUUqIkNqDe5QdzfHQMk3kw0V4\\\\\\\/x17IRJvp\\\\\\\/tPrAIfdLjBPvzkzxIj7UelzK3XfFcYen0Tx0bZS2xjfN2rCyjIcB+tMAgERheVmZF3Seixx4LYBzECwENATsDMwb93497noGhlfQRyQbTh\\\\\\\/tYUE\\\\\\\/VvXCagiPf0Nxc\\\\\\\/u7f34aYTnumSanor0mZ9KFVDJTu4QE5pHaIvnlNj3Z+4h2mdKkaM97nLuTbtX2rqnMXCysTM\\\\\\\/eDXaWLGlX+X0jvgRP7WrxLq6Do1jy4pQnBVbFNkSJfRV2e8sfmQTZwPyCGdye8NOgx+lXXp1gBXkXIeGU0VgIII0JOrnEvJ7wAMs8xeewqkbdIaXcPICT6OKMdbrWxtdrLzbhJDbCgrJcvfNog48SP+becJ6zr\\\\\\\/S6iXqEbTf6L2WEP\\\\\\\/GUbeNExdFPuB27DKZHOVktZ4WpuFB\\\\\\\/VTLChJX1KMaWnN9igogfupvDjwOGRcRSU47UxUGqYn4oo8x5v\\\\\\\/EDwL875LmoMnfbWwMGokolvJe0W6nARk1m\\\\\\\/S24SUOCUXVOrpSJnPTzCGf64SRNqNbrArimZWPvAwdGBdHzjEX\\\\\\\/Nbphm5jnaSTeWTWOj0fgbWl8dS1gAzIuet6bct7FV1GxQtwR0ZfhxR5xqJUbJC3EY6q27Jjjk+lQYAMZOtMIPtKfL8KXGP39KsRNY9tmH7+ZKNby9XcNoSo0PccP20Gwi24lqKCYGypr5lVJNvV8h57oU7hQxVI2JkZYVKldvBv3aEKT+6dHjtEl8jWICdQIav\\\\\\\/GrZmyGiw23zrArxVUC6rzMQMMGsocm\\\\\\\/sFlROM3nUHj3NsGGm66Ft6cQc4fYRAD\\\\\\\/h\\\\\\\/nuyLYB9ft4hfKM+TtZ4vG\\\\\\\/PN48RZcSrrAyHD2BVwYtsGu88iS0IAgadIA3pxhmiZuyWRGxkXxwRKWAgJuupN+5rT4z06TcCoo0MJTtKe\\\\\\\/Ojrv1wdeKmi72sjJn4aGWXf2Buf9PbE\\\\\\\/igmiVOxRBJo5o9hB2q+oSrQCyBq+l5n9fwuqR7T3oh5uEf35mj+V\\\\\\\/AHBVIVeW7nz0BkYMiAuq9zSmtFvCfcf6pyXhuv4I6C4i0NjJir5K1gP9myp1\\\\\\\/F4XeGURGbj\\\\\\\/2y1QlFyAI9C3j1u5ck5PHfi2UHf\\\\\\\/AImhX+akNl6HyX9JD0pCywBoG8CYdsJYcNTj+Acuwr2jM\\\\\\\/sdZ5YNHjMUbJEJQDPbrPVSGpuzMf0\\\\\\\/wqj4j9LiMMm2RLMahT2\\\\\\\/Co2oDdAaueya+EKXZ9XCt5m4I19592\\\\\\\/BY5Ltqlc+2AqYhKH9VJGeZ8rGDE65MrGV3CN1NbqRfeM6aFfbWw3kOR4RfV+FNDmqE10u3F9oCHBQ69b9a2Y\\\\\\\/xezXdCzHawg607BU69tN0xF7sGvxCJgmcQVWoZvIyjxPAAxzvXx0btxLN3VJdZ98mkau5PpDudaq+yJuAEb2bS+91JAWwWMT7sIA8PcX7O27QcKdhYietU9TP+KPb6GjT4SRrGNX37V\\\\\\\/uvDWiVhrN7w7MNg0sJoEGsHLYI0flifmduTH1Hchv3sdfaWOTyYOZFS9hw94C9sq4TgGYqTA\\\\\\\/Jag4Fpva2GH0OIpcd89uMnXhsCI6GtDcvPfgv9g\\\\\\\/edQP6l6fAJFbC2b39ANjUlnJgbRJxeI4hQYR3QRtgQGtJJo+sXEj5G\\\\\\\/7DilW+iHv8G9ZTluOUnyqNcaLjDn7N9NC4+Gao\\\\\\\/tnVyE79zyxm5ABFR+m1eCLItduGzxJNAA3r3F8F\\\\\\\/9aDm4yHC7S17If7W6G7szUMM0sGma\\\\\\\/9RDKJgYiN1BaP692cV+YEtPgINns73jrPuuj0z7E9zvSdLv\\\\\\\/dMeMuBY\\\\\\\/M7cbKANM6qkdx8NxYm7W7A\\\\\\\/FqAk9iqVP5qqyOv9zh41sky+rvyrtkOuNbhWrQmF1mVJDmNoecw1V\\\\\\\/IWY4SxgqyHzDcRn6dvjeJfhKULT0FEJW9yWPucdX+rfYKg\\\\\\\/tCXhPD1lF16CISc2Sv69fb\\\\\\\/WmE2zCg5NG8cfUn4WzFQDFEUMKIujHGUBp63LlM5c0S6Cp2KNg7n7CE13GNjrr3rfA4fwYN0f40V0aqc+6\\\\\\\/jckK9X0B5RwgxYx5+q8og0MKrNdMNQBQTE150Q6PGMr2yhS6PKIrMAG595pAJNCYnrj\\\\\\\/DYJpf+XposP63XnncZPrUDoaItQZe1FeC0Dp6o708Hwqnss9O4htF83BieYLWqeVQxhxFlSwE+0ct1tKjKT5cSiMhOBgakuOecyOIyIuGkMIdW9\\\\\\\/c8OtiFAnc39Oi3ybnbQR2f+dY3RF8sYMtl6CJaqqhtZcGL3X8DVTLqoQtZkkxrNNPAZnlLnRKFXbFvqtDr0r\\\\\\\/ShasSfsQdqB1mDtsVhhMNlCvShY2ylz2NBhmsZMw7+z8zPsAPzsCWimHr\\\\\\\/df\\\\\\\/F1+I7ANS\\\\\\\/s00YRjeAABpM+ILj\\\\\\\/UP6vbOovsAKdsYCHcf2VOhU1JTl87m7oy3rK7L1F8XDtY8gL5HsOHG\\\\\\\/mIZQkHHZZxh1s+FGhBBl5TsSXlH7aBWBhJhqodTO+3KoRZVO1QGLjNueObwOR84M6sdCK97ztl1hOXHKDFqbqcOFkplHCA2Q9vMiiw7gbhgNl6jdB8VdDLnIEtZZEFveeb9xtdZv5dc5ftA+neUS+uNv4t6cEA+\\\\\\\/3ann+eg7BdjCzoQwulddLSpyHwzNiF98OLsK2qfZApsqRSOmRzgGzpsyrkTA7cyRcbM9HgxBKBVeSD327d5Ip9BkgJau+7gG9l9NUcLCzjSB+1tivWtlC1m15IxyCxWAV6\\\\\\\/U6t3ExK96LYNyxA1TXHrplqeZ8QmLyuuBF331njwgNYU4HlJNJNp3m7o6PEcegX9zQWohBIC1a3xKRtZiPmiAOlNMFCmAVI7p72mLKKcCKfXDT5MiOmrwKl7Ml+VCiTONGc3\\\\\\\/yAJSFO6EBTCnV+ztDyZmfvLJMZrkUHTOSD76+rzeQqHIMz+9wEoOJVYfqbpImTLxYf3d\\\\\\\/OgYEJYrIEP8a5H0Xp3yCYqntk9rLpun8jUliMYTOgAbc1YSCzaTMidvolbIZurPrF\\\\\\\/KJOReMAyCOMFKD1lOs3aQa\\\\\\\/qI7yPkLISozrsHIc3Z2bQS0uGnrhxf943ZwDKD1U4A7+BizAhjXZp+uasWX+PvxfEp7nUTxkMjJD8m0NzLTGcJzijez\\\\\\\/7PR9\\\\\\\/6ExsZgL3n\\\\\\\/EyFBhsct+VksG6jwln6cOcK72rm3e9RpwkIfnqa8hNWfceKUU9DXF4EbSJAZBXfBX9m0RLC6RvYodIBbBeOuzm2UI5iFU8G6ne2m1lf+Tf8XG66jsU5hQMFyllqrjA24l\\\\\\\/jh91\\\\\\\/QKQ+O7ejSAmx2O2OSxPY6Je++yayxt3n3ZNpkhRUUwm3JYlhg8efOjUBAvLmKlXYV3Q16uzssHXCnDILGGWKsDqtR\\\\\\\/V1Iok07zfibTNVgHncks7KIjkAEwi8LiUXcUd99Cj2prinzyPInVAJA2F4vCCqA0LI0w1AEnlhO0n+yRGOLONdjRbS\\\\\\\/U\\\\\\\/cUb0hIL8gseAk+Msw+sLDNJ7KHYfSVYr6F2+aKFkQLtrvM2s+XLh0f0foNmTlodMQsSkJXCSvGloGCvrgh7gOBseGO6xFzyXxd5zBuRYXfHrcZNnYujON5fnSHFf7MJ+vsi6f0eHiX63eQJjlHbf\\\\\\\/mI5yPiZnYB8lzU9Vh8Ec5LS\\\\\\\/\\\\\\\/X6uC+yaCuoSpKQZRnR3rClwMlXAolDbjeUKeD0fuNqLnq6MfMHJp\\\\\\\/8OpRhD94BpsY8AOY0k2UCRI+j7jeTTA151NROlqPUmsuST3H5ggvMFV0nXWiX1Rnel5SrNsEa0QQ1ZGUTXcsQKNshDM+eHHbineEXQagEDI6GQ9NOrW5TlX4gqjS52d+RnMLSxY\\\\\\\/dyD4QB3aiMk4vlNP8tsadeYhYUDRRQHdfGzvWTfWyQdJRbpB85jD6zqX\\\\\\\/h1r5aAyaLhujL0dV2Ucofye2p6UmOR0ywtC5V2wMWToEiMCmSYa4XPx0cRxgaFoJpIhJEYuQcBDoTr95sVJU5zK2DxQp7E07PCajw1cmPGsXxNlVW1b8YhltiwgJAMlcB78vdr2ERdREoz+\\\\\\\/bTSLawFsZUq2Q30Iew88mSlpqMqe\\\\\\\/C3G5+J8GM5jzKOkE3zTEmCIy0l4wdGbeZl3T8qs0h0OFgnMXSp+zurYVgPRpup06EIC7v0lBrPnh1w1FRVkcfhqVX1wPKonuB\\\\\\\/8Acjig9GmY52YihE+23haAwDDC3udorN5m8Mwc8IUssn59nnno44S98Fcel3RJKGP\\\\\\\/I\\\\\\\/yYC7CpD8Mc\\\\\\\/slzYqjXQqADzMb1SrYGEeElJuxmUTAgPQjpGLgvzSZEVGtMtDGdJn8SOkkbPK9uOYxTKZnnEIP2ZDpmtu3zooKRDekjEXGaR\\\\\\\/TwRdAoGkw4mrKKcPKyCMbT0HaU5GaGpCcQsxHDMaMktH+Jao7nuJQzCfn5c\\\\\\\/u3YX2SqAnMOqJYdkSoV9FRHwsyeOHOuYcCSSs9AUiqRUXxpev2u8gecIqSaJdALmMNC9Al6+7ngMb0tO6ocCBYvDkpOa\\\\\\\/oP18a3igYqLB+cXWmryircq7QiUL6EpnE655f6ADo9ZnjVWRa\\\\\\\/XPSDGFkmAQ5XAgCqbzIgDinf7IV7OmvknI19A9zynYxVZl5j6rLCmhHOheM4MRQkfhA4FRPr7zKeH1hVOHgyaR7\\\\\\\/UwlcM81R+CW8jyojjcP0ABtzDY3fSE9COvBixrvgej4wa0bebarSM3Yn7CvpIVsLvoFuPHDcQIjLvBGM537egdKVjqcsTtbW0WqwVSGU9HZEkMbSJCHMU3BqWN06IVCmwfNv6Hop9vSXd2bqrajjy+EWlsZvUeFN1nfv9GpJIGiMZ3NeSseAnwdkqsDHLfHxwCWRCgX0xteQ3U+dAzsMyQ7iMcs2THnNRQh2wTUMOB9YlhEo1nsqtzWYvnChHfDEWXbW9\\\\\\\/\\\\\\\/rpuejjfFndgq1nrCVsWQJCYz7i1ThW6uUo5UHqZ2OGAIBCE9FUzzmRh7EZQ4jkehAwwk1BPhp2WYKj7NdCtMD+R7lgk7IssWPOKUU6YUZ75Ak\\\\\\\/6SGIPa5q1Atoy7xvWAKlF1ldiXsO15UdMDvDlqmb6wqje9JstUYWil1fe1h5Oc\\\\\\\/SZAD5kZKCdn5P0TVxcMftQ\\\\\\\/6CKLDHmwZfV8sgVB68Fp7q44smU1y4QkpNjYZyP1VjppSIQ3v74j+KefGKBwVURrogc9xcclthUNOMJEBx3brfqp0FxT16mWxbTBfywcK4Dg\\\\\\\/tr2PUyhp6J9xzv+yfq3xM83rIX8nUI0XSxY6nY5GUkysuOCbHECDfhWPPa9ln232reRjU95iIZ84xcAFewV9j1digq8gRtVjRkD1bmAB9PuqpyyjHsxSixEoGzttuMszmvoM76KRidYJLhiqyQ2eOSf8SyPVRgecNE7cLLPXLF92mnmB0Do5z3xYHTnHKZ7W8sj7bZBbHJCEnE05BQ2uTwT2dbQYwvnqBdE4Gcv8YNf0bh5N7+9DnoOpsv5Gqrb4YKODMFsUMcMYAd2QeLdU01kOA84kmQI8eVCZIVnsRq2El\\\\\\\/rd2jXacMVgO7PLMDiWMNoHeQbndhoexExhrssEg6q4s0EF8KzStTMEn3dyBrpt9MLrf0o3udR74knSgM6yOsDxc2Gba8SNyH4UEKOgtquG1dODS+RO8AfIrjbvt\\\\\\\/hIh3YUbO2\\\\\\\/XtqflptmAmg3NoO+SiipicEhKPEErpWWMKzQ47prJZ\\\\\\\/x8Gq8VhuVz9SOJBQt5jp6J7BcUKlm\\\\\\\/xnTyN+MKjSo8rN\\\\\\\/7vOqSUX+2wCvwbMNfgkIKGlq8wxy\\\\\\\/iig+aX9zvaiLOuciPHxgSONK0hSDye6M\\\\\\\/BkJW\\\\\\\/x8oJDhnf2DVoNIhMvd4hcGSaoPq4ly28N0Sw\\\\\\\/xcHV23ZEadwlrxQu6FCxBg2W1jeES17hWdiRGsAmU6h7nTAQFy0pfz3K9wWUvsIAsT8tlfQwTRKGasZrPb0eS1QxtYzvIU2v3PDgBys2vEp9tm+AZ+zYwbJf2KWbwothaCkiX07m0L2WTqkATEkUwVOp0BOfDt3DcOG9UiPAVAZp73L1leJD9OGhVQt0V3ska2CBp6H\\\\\\\/nRzlKkiRnp9fybpnzC2i\\\\\\\/xV4GbzCsXjeUa1YsymzJg1c0Iq8nfY4TLV4wA+MjQHyGwuia6K3+8atqsayVPIQbXQbK4KDm6lMTsBVJAFU0bwYoL4MzGozAquoryxiGaLcPOeZ0Y0g8GikcqKbWjYrGUYH7yfJ2AU9rtDMgabuj52bgusSsPt215yfmw58ao0Q+nOtVU2o9yzA0YOyBEPNM3SizgBS\\\\\\\/FbA9tK5HRgJzzxwKku+g8\\\\\\\/Jpn0juHx\\\\\\\/HncRuIO8NqHnRo\\\\\\\/xNkx+wcw\\\\\\\/wuBEnkDp8qGFGmAGgnZ8Efd94OzSs\\\\\\\/VSa+\\\\\\\/FUAsWSSceU5E0hM\\\\\\\/uogYf1a6Wv08sewCvQgaNyqj3V66h0EYSG\\\\\\\/Vo3bVJ99UHeeJZF7qolBKVjWaQikI5wk7hRUXyv9+cQZYyvIgL2nsw5c6TDBglOpQlSvPOAEyPEOOqsK9L4IqKcEWQ7YWVLFOGKzosP2k8FmCHjCMxyYp2MQ0BTV1l29Zzrp1QdkDIb4GSf1lRJDVWe97TmFopCBl6xknHz92TLeQG0wJn0BNc4oEjbyEp0Z\\\\\\\/pCF4nZ5oTJsCmx5QXsvmboO2T\\\\\\\/i6IF0bMpYCDrX\\\\\\\/vOkhgUoCqB\\\\\\\/njBPYR0koxO9p7UQi9S3bc6XlNGksqtYMa9tGNukDwoUzEgo1pudXrWn40+HonKOLy62lJsCMpoP3ldcTwtDMkOPbE\\\\\\\/S2mN+47kFJ31BAfrvpgdMK4yUwNqQ4yFBc9BomzYSpUNYDYYYulA\\\\\\\/Pz+7d3rrnOpoC4bmGRVU1rhAlxfV69yWOIJMNRgpqbnBf0Y39LtZtALqFp56C0V8OEamE9W1FbccEa0CtxgBHWZwOC58VVEw2nrKRSAAFrBmjJN2MyjomWz9GgNNKV01CQEhsJhrFRXSH4KE60baFu7W2\\\\\\\/tWkJcFCCvmii8YHCl\\\\\\\/nnPzVUs+aiKDivm0jSUULOty7EQNRVLZWYhKKHpo\\\\\\\/uuvj2yhclsw50holGY8CMV0ufcHZK2m9gjjeouvED4Kp1fIspkzRejUYF0onfp74fc8hFnzF\\\\\\\/wTFYk9c70APwxL4ITCK839PtLzHuAdL6ZRoeR2n563MgfhhDBqcgCZY\\\\\\\/CtLef5EK2uYWY14JqlMvKxCFJxd2nNRrYXSAar1chRNC37YE+AIquJucSa5VnLeZWLxBwbOkn2MdWuMd1hp+4hU\\\\\\\/Yfg0eFlxG8zypsXV6SQWWE6T3CrSLjlWnkcHDpa\\\\\\\/dOd1MXTV7Sr+\\\\\\\/47v7E77d4epgKiaHJjfTwjZzLUzyNkOVU9Dw75srAU4evMvCP0JwCnssdnHOir5wxood5jc4xcd+i+HHuc5jywrVHQXA0TemZKXvVKJBHmq0imHt3+3lzNrKgT8PEGjGqiShsgwmVZ2iYpCGIVOj1mcjg7tojeJPh5OY94OKBwlDGdtWDT5FjJfijZ3LCByCmnibj0AaVlaR9yD8Bestcoo3HCF3nRmYsMz1lKlM8Zqzt66P+ajz+vca5w12KyLBuDWyEbjJNcULy82z0SvNkmGE0\\\\\\\/9COcc3w4shsytFZdkCMBwlrCEh+pzQl2lN8v65wHM9wubxenXzYefsVoTg9Tj5wF1YVGehAQaLu+3ISW0qCc9AS9iqrvSiiTPOBqCQ8oDZg6Gegy6NMSGaEkt0d7FrJey+jp\\\\\\\/+oR4Awl5uU477LAMqkW0Ho9B73JjHvvDF9\\\\\\\/61TjUDa6gwUUdMiS6snyN+X6kXziBWy1e+sNelMZs0GHYBxFiGI1K8\\\\\\\/h1x1dpYUs9GyBBjWLG2334hi7BqONh7PeG8katsNJan\\\\\\\/yMaiI9Zz4LOf6Fe7Wx932Qhb3cCx+E3ZEbMFqyhTlGtUFDKU2hNAV7g2M96ypNp5yU3ji2smtZmI7IpcaM2StA52Bs6GjYy7naM5cUZ4ukRttfE8h95pn6ya7BJ\\\\\\\/LPCcEDFhXHzwUPNTyYQAJm2I94L8cBp0Na9FT35NdDk0fsSl1cWkYG9oPPJPlJ+clHN\\\\\\\/TZcoZAR7iRiilbuV\\\\\\\/WJBfpomC9DJHpN6BE1isf1No2x1bZ3DumHiB+iU5UGxpco+FoGFWxgfbP0ue9ZBOxSrjSzUYKgZaRfZ4sE0mPYsUxrO8qz84wnDeL8zcUQttMos6kS\\\\\\\/yMbj\\\\\\\/dhX07vGM2dTTfKOrdQdzsnsEthb8UK4MsA6UvOm77XiheVsIaf6Y2hPU+usHmKFFzmHuRqtdeXE\\\\\\\/LXC16RcXi1Dqto4lKflmV4JvLU64ql5uoAEAT\\\\\\\/zZ5xot9HRjHZfR27SbBcogoFlOcXaDpfQocZPKubySigvJYGhcQyqgoLpKUFAYPIQHcbwAWNqywuXYAkx2TjeMk5jNkGogFfZqVXsIgvoOk7zR8ugEm\\\\\\\/d8jg2pOv1ez6Ug3sZA5q26eDG3nlzF\\\\\\\/ShrqyPEXFplztrKPeHE9Q2t+xD7f890WOeKbR1pvt9zx7EEGGOfYFv89dF5AnsIYVxSbZrk3W9BQ8idYxJmhXikPTj80r4nTHECHBHJeHN3hJZpil6fgEzf8txs9lrRXDpF8EXds7Kx9eTSP6jcedJ4fkw+sfp+R6cEsCPoSolxLs7UDeOkIO5CyQmiY4RUkNWEOavytigECg+JBZht8H95zZGCeg\\\\\\\/E6i3ccq4zbUO\\\\\\\/4uAUclKmtuJUUDbXP1m6g3pu6tcq2RfGWFG85Mrn4\\\\\\\/Q3kiu1HzTwIuOL71KqozRrGkVDpJS\\\\\\\/ixLoZ6zIPBn0sMMXb24pLyTI2B2SLHQOF8jEgx+WSiBfWyne9zZGufhR5AUMg92gMBS3rgK5cLXONwCO6OfYHDE1mrqoZTh5qt0pqks74rCnCpEYIF2LQX8lLaHxUACYdVJeZc8AUeic961ILShIF+Jrw1HLDsP2ymGlaXB7YOVGC7SZ\\\\\\\/OFPVT4ZvuGsIFcoBfIFBGi421VSeJS3P8egZ04+CaDNJMwnjHArLTaFR+KvBahWyHt7W5+t2kRfwVtRO8RqCDRYXkyCHeWfCqW327G5gPs713p0sE8pw9oqueEdonB5CkBC+\\\\\\\/1\\\\\\\/4MwS0hThjaMmtiewSIhcu3oxlCZJVh1Ko1aiJSToTIBxU5ztiyDm3gUnWZVRdIosIStrUKvaE77lBJBYijl3sUC\\\\\\\/bKh8chWBLHy6JqfO5JZf\\\\\\\/fZhBUQUpG5bx4VxKQYPvZ8Rs41BAKYsDZZ6oNUMctxCuiwAjXe4HQ9UYZDMx4uAk4uTMbrurHna2\\\\\\\/QLbSfaX8RPE2KDwKwUuUIykM4jA3Wwdv4wzfjcVegClFTRMLVQB2MkPa9nJaOnuDf9QzAAFtqbewCTqrjOXNSLFGfB3WDPdcdITEdefgFQyZlNFgLPjtyofd+li7ZhfJ\\\\\\\/F0REahMcP2wpko9e4jJF+XKbghzp080AV23ZpVK\\\\\\\/E36HnmCQw6xUkd646gX5boBiwGaws+SoQ8z7wN5\\\\\\\/L9SeT3RS\\\\\\\/AKFFR\\\\\\\/sNLRMzz8PyYPKWsR9IupI+nkNVn8h3a9GsSDYui6wH6aNKcLlO7Mcn+dU19CfbdJ\\\\\\\/jYcCpxGJAnHYWou90kZ96D72Fjlc3Xa1hVO66W5uG6JGVHxoEEFwtsULMGDSubMCouWZPnIT7ml\\\\\\\/bvWE6xCwRxeKGOhYANeVBynLUL6zMb7OppklOC3SNzoeLjk+DJRt6dThp7Kr9Wlr6B4HIkeW26brnPDMMMIaLZ\\\\\\\/AalLMkm+38nx+rT8u4j5eyEZURLfgvqPA+15NI915jTEz3vSnI6NRAtgHeYvUpG7Io+r12MOZ5YoexSss2w2bd8wpqkG2hu5zIvNxo9WCE\\\\\\\/uPAtUbpnp1KQlUU2+4o769H+s\\\\\\\/+cWYHczY3aKfbV2Abey8eGF07OhTnujQdThEtpvpwYAWegq7ObhZ+TtrboivmK\\\\\\\/IjaPABXyWtcyojFawV3mao4PoqS\\\\\\\/XLT9Ez1q9pqMcM79oa9Mlztd\\\\\\\/ka1FPfpBQaG+gzbuzBqE28xbWaPEV7zKQJdrcURuIy5WBlGf2KvS4X0bQJ1cKTrwmfWWVY6W2oUthc4Oa5vvZ1YR5hoiAkB9pvAprfyo7J9kCtxEGybXY8NiZvpeQdICQw2su9LgymQ9QCsDqZmYkp8+2\\\\\\\/1EDgClcddSz+4lppUF1S62SrFkFGN9zFlDFbjDsvoRH+kyKRbpSR+Rw9tUZpXWmUzyRsj6ZKTxMiEas9iWj8LpbYLg1RzwbURe\\\\\\\/tvcGPyxS3PfM2WicBGYLPHMIWnH7lju4XgqGB2ziSSwLsnx0n7Mx6pk3uHW6wpKcJc8Q5K7lNL2QVcyXMWBKblj6UBq8+ExzZ5+gf2H4Da7aq701lshZvcARFZtPvgBQ4LRDYclwpDzSOKgYK9TBbT6WjAdo\\\\\\\/AstgP8JuZkKhqnFu6KqeG4\\\\\\\/l5ShQ6Km5hhcNP9FA5j4AnYgtTOtYOypjvmALGQstY85797P6BtLto5Akyx64eV4md7Wcr9r99GiQaBlm8Db1jDJm1QW1JjQIC1KYNu0g86\\\\\\\/1R8Ni\\\\\\\/I959njwgVUG3MTfxvUvOQbcs+\\\\\\\/Z1PQUS+03J8A3GXqQJmsOIphujJRh9ThnjePDqPE1qYBVFIs8tW\\\\\\\/CvhnQcZuU7cCcX+XQHJHbTH\\\\\\\/gJpaWEm2jA1T6Hr7dZkNvjeBs0vKoE7iw9jrgJeucMN5+Mx0xzNRWit\\\\\\\/43qzgT8D3jFmE2OOT2GgykjyYY45E3V61y47fRdczpZrKRDLXnu94Z78T0Idj80neqlDMvxex2cH\\\\\\\/2\\\\\\\/v9qhvvQWUh6CRvaKKIoqeytvOMn+ghU2hUIAS8JMPIu\\\\\\\/1HxeZgBdCcGn2vim4Qjys6Q\\\\\\\/pcO2pNpysUXmZG6dyKjDfdLDDQl0AF4fP+6JTXfgJYVQ0mFjKuLWTnK1tWYSjdqpKM66J02XmfqHj1jfensa\\\\\\\/bp5dvc6jmGD0sfa1hA4BAfmu9UvS7fVuzZfHbFOeZVrDCopzsIeSbfHhyH2d2QudHfe6mxIeZIZPAoO\\\\\\\/B1XEeMPqFK6uTldx\\\\\\\/aETPUUjb2IHFn3J6CrdE7t0AbAAfZVVH8ntdYtULopi025+9oi4VL9bU4fRiALv3OgP\\\\\\\/zWGCtIg4OYOzV2mm5DRMNG+furcTFXCEwMuwNse0PDuIpSTE79WR5ue6jXKcBiS3eYlB6U5wxUoxpK3TfWH9ofs3EEIrxmxb8Y19nx5J+QT1Im+mJJ9\\\\\\\/7TflnMfDCpyEK3+xoUoEXBTtfHewDX6CDG8\\\\\\\/EO\\\\\\\/OHfiY8NrBh\\\\\\\/rRM\\\\\\\/c7\\\\\\\/gABVx9khjXX0+zOs\\\\\\\/NLoZUjGunCjeS4R3qrDpc5C19IdxIIampC8eIaN2BMfc5KpHzCUJoiTypW2YHZeEf8z1pafGKxp+QSXmYgoqmGW0SLe+8to3uJQmy9bciYxWEz112Icupqlo7gog35+Wsa0i\\\\\\\/mt+TGPnI40eJ7Cy6M3CDlyXOWSEhuSjxqP67LTdj9UWNvB6NuYhEMmZyoU2wCqjEwxYSHchIYwjz5p8IQAFzzwoZ6R3eedCgix5E3zBOJw0dX5zBQHW5dfeunhTjwireH10skBnLIDVJzqvnvEeqzyevBDMUO6hn0MkHSEdxXXvalO70psLnRuH7tF51k6bj+54Rcz\\\\\\\/\\\\\\\/UuGCpA3ol2VXm5\\\\\\\/zslW8OCoZIik4f+5E++uF\\\\\\\/Nt3gxLlDLwjYlPCnHjUJXZBJZZ\\\\\\\/oYMucW6w203UH6RZZdXWAP1570DphZrG6boDJcDpBTPyWmgSYbqbMWZP2eyEgdfWEyZ7miXj33T6FChQDbs5TTuL3dVNeh\\\\\\\/knIQ36w\\\\\\\/3NwkXocmQxtV3tkhBGsfBROSYhcCaM\\\\\\\/HWN7sw8hZiYTcA\\\\\\\/HKRvDfDCoM4x14zaNU+fMpk\\\\\\\/b4vM4Jhqp8ToSM1Z3EIznWrc\\\\\\\/LiUq0cZtpZdfbSPbTztslIdvSPkI+ijUXftgcOW\\\u0022,\\\u0022iv\\\u0022:\\\u0022sgh\\\\\\\/cMl\\\\\\\/cmXSmCOhG5CxDA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022ca4e626f17f9fafbb0ce86f212db4351eb414a93fa60c1d8add28ed60d0803dd\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022ba3S7W2Q32YQwdwF1bUsTuCG\\\\\\\/BknA7DnMEVwldYgb98n\\\\\\\/iS2M10DHjsXYkCY7ooUYXLWZNIh0T46bAxJonI5IU\\\\\\\/HyzltA3Dcq5LTdTUHvWo4elzZyxQ8SD0rrZKu7AnjzgYnk11cNWzVv\\\\\\\/4xYfpgylB8EfZSzHBN9L\\\\\\\/QbTsU4Uv8Y\\\\\\\/O4pjgZUjkkSB+Pp+RC\\\\\\\/dQcFFmFavp1naugh0r8\\\\\\\/QYVzdGO2j6Ws6P8slFgrs6gRD4Rk+LaLdTciCRaa9f\\\\\\\/ZrJLAgfIU7EvbNbrATcp4KdSFVi148KohMGrwjhluOYzV7aBv3XfMlKrJyLffvD2AvfzrOCNvpAHk9tCIexiuFLk0EJuHYokguCMMuLJPgWb8LMemoCP7gDQqS92c\\\\\\\/mA0PFg738GvsSebR+mlv3Kk9yCUXqxGgxEE2ioyQ1pmyySSfhSSrQK1TdUtKN9sr3F0jPiOO2bzFKyyEeEfzxwDSmWhBOVMxTES0c714y65fNtunWfYZRC96XFR41Fnxn1Bz5NuXxduYdwF8kS6TzGt7vYPAemaEywL1EUBYv31k8nWYqqdRJJ7E9OZID77oHx74HxiRRg1mpMkrk1y2KkH0RY2fQ43ubOiQuWVMS27EgSUoiPAepkP5AkmVmeLxDoTJxmGYicsgZs9fCt\\\\\\\/Zyr2NfndgHLQh3J\\\\\\\/zCcUlAfiU\\\\\\\/mH2gjQLnNa1PXIz1nrrc5Pa0ZiMZvd6jWd5n2\\\\\\\/yfqIGdPg7O3MNcpCMOjKjp4akm7FaSUoJIRnuCLx+cQpKQv1P0sdDxmIFWCU0zmNUaJjAICnQkIAmVhy7Zrpk6CEQ6drXP43Q\\\\\\\/nptZ00QdctCG5BX9TFqYfP0LGTgYk2xB4yO5wNCnIVEqa74N9KCOImeNv6A98ZsnhBPflJObYtopqFvWPr8CrfVfwcT2XPRIFcXQNXeSmPaNOhW6uIdlnTXz7f5RLyYne0r9iFslq5BKQb0VG5CbBxwvf9zujwyURWRb5ZG2A2ETnpt4IosVJREQj167Xaz5EPLLNCLl28wKSWAL6kamrgyNd\\\\\\\/Aem7X2XaJp6YzpwEe7kSrhSQP6jyeeT9bZXi8YXxM\\\\\\\/KYx+bjJ+0GURTrlIRbXw3Xtqki00T6aQy9QYO7NEwHoYqb0jeiFgo4V+8sef7cL47KniJEH\\\\\\\/do3gG\\\\\\\/fGyL1EUSfVYH8DfdeAPVXUjrjav\\\\\\\/CpH8K4d10uXaSOwlo\\\\\\\/GeDPfR8dWZSnOkOg\\\\\\\/0yu0OQhQfxnRHomz9ToGhL+J95FvpoXyDAkc11mx6EOQkZa\\\\\\\/K79xqdgKGqJBFBSPR6l2SxKGRCV2IqUcINu4QW9ccySd2\\\\\\\/QQo60P29HeFqGH2h89uOo0GSCs+rGlMavB56UVstfrXa0tnvJqAYA27xD0Nz\\\\\\\/FJr3cQs1MGwYZscPMlTzCBhReFVamvXVHQ9ThCx31z9w8VyQ9i1rVH3DMxlMwdydj4SZu1v\\\\\\\/HtvB9ka05jWEpgtvTbnfAIJzhUA9GbMmVV2IjpiI4IDdcC4BILMIzvHIcLY8d+GK14ZioKxGLsVn899tTZvaltv54VF\\\\\\\/wvcWL9MrQh1FedUdjKvBn8DhL7CH8Ku6wCjq2SvIEUWoDvh1L7uKVs+XnNURuwzuYSI\\\\\\\/EiMBkWNPvBHnOZlvl8M5QxueNwcO3TZ+xCJjk64gkPysOKSSQIghXrpLxJ424ha7ubDgIe11fuyyhu1ppqgMsPHjF1SsRLbgVBddcc1J3GcQqA9FkA+AHfbPrjG8Py9K95IxeEcrVLRibwmn2thf7CV887ubZdyqFDT9hdc8Ac0u7GIlKCq\\\\\\\/AjpIpQxEvq7kd0x40KMCJpVCFlh8RRMR67+Aw1piWtdnmQGsnwA1FPxGigGmhZUJztNO+qT+Ntfau6FJZLYblvX8xxc1zK9k2ljmAoJxCQEu0ZiEmXQna65JgTPlByVtpPQMs5cqWiWjN2reRTbMxmv6uklIrbWdvqA64GGRhJFJd3XffpTg5QxJ6U3xQXxNf1BgjB6Iv9yk2\\\\\\\/5qeQnd1OrtXQCgJJ5oC7a\\\\\\\/r8vzGiNbI+Sg=\\\u0022,\\\u0022iv\\\u0022:\\\u0022CRLh\\\\\\\/nYmgp\\\\\\\/Cv7xTNxXw\\\\\\\/A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022a9fb7e45c44a7ee761019b699295caf9a5855cb4ec21116cda61ce007036f83a\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-491245 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-multiparental-populations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/11/3567" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-MXGwJNU2k9FhcskE9NpFnqSRFI9OXeLDIDnTkbwLuZA" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/11/3567" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-JCMqxrXuDeUawXPkuxlovshv4pzVqaOQT3-H88zrkDo" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/11/3559" title="Discovery of QTL Alleles for Grain Shape in the Japan-MAGIC Rice Population Using Haplotype Information" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/11/3577" title="Estimating Asian Contribution to the Brazilian Population: A New Application of a Validated Set of 61 Ancestry Informative Markers" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="491245" id="node491245" data-pisa="ggg;8/11/3567" data-pisa-master="ggg;g3.118.200636" data-apath="/ggg/8/11/3567.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Empirical Comparisons of Different Statistical Models To Identify and Validate Kernel Row Number-Associated Variants from Structured Multi-parent Mapping Populations of Maize</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-0999-3518" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jinliang Yang</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-1392-2018" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Cheng-Ting “Eddy” Yeh</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-7804-712X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Raghuprakash Kastoori Ramamurthy</span>, <span class="highwire-citation-author" data-delta="3">Xinshuai Qi</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0001-5821-099X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Rohan L. Fernando</span>, <span class="highwire-citation-author" data-delta="5">Jack C. M. Dekkers</span>, <span class="highwire-citation-author" data-delta="6">Dorian J. Garrick</span>, <span class="highwire-citation-author" data-delta="7">Dan Nettleton</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="8"><a href="http://orcid.org/0000-0001-9169-5204" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Patrick S. Schnable</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3567-3575;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200636 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node491245" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jinliang Yang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Agronomy and Horticulture, University of Nebraska-Lincoln, Lincoln, NE 68583</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jinliang%2BYang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yang%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJinliang%2BYang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-0999-3518" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jinliang Yang</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Cheng-Ting “Eddy” Yeh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Cheng-Ting%2B%E2%80%9CEddy%E2%80%9D%2BYeh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yeh%20CE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACheng-Ting%2B%25E2%2580%259CEddy%25E2%2580%259D%2BYeh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-1392-2018" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Cheng-Ting “Eddy” Yeh</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Raghuprakash Kastoori Ramamurthy </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Agronomy and Horticulture, University of Nebraska-Lincoln, Lincoln, NE 68583</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Raghuprakash%2BKastoori%2BRamamurthy%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ramamurthy%20RK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARaghuprakash%2BKastoori%2BRamamurthy%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7804-712X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Raghuprakash Kastoori Ramamurthy</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Xinshuai Qi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Ecology and Evolutionary Biology, University of Arizona, Tucson, AZ </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xinshuai%2BQi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Qi%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXinshuai%2BQi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Rohan L. Fernando </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Animal Science and Center for Integrated Animal Genomics</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rohan%2BL.%2BFernando%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fernando%20RL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARohan%2BL.%2BFernando%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5821-099X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Rohan L. Fernando</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Jack C. M. Dekkers </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Animal Science and Center for Integrated Animal Genomics</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jack%2BC.%2BM.%2BDekkers%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dekkers%20JC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJack%2BC.%2BM.%2BDekkers%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Dorian J. Garrick </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Animal Science and Center for Integrated Animal Genomics</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dorian%2BJ.%2BGarrick%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Garrick%20DJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADorian%2BJ.%2BGarrick%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Dan Nettleton </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Statistics, Iowa State University, Ames, IA 50011</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dan%2BNettleton%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nettleton%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADan%2BNettleton%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Patrick S. Schnable </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Patrick%2BS.%2BSchnable%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schnable%20PS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APatrick%2BS.%2BSchnable%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-9169-5204" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Patrick S. Schnable</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:schnable@iastate.edu" class="" data-icon-position="" data-hide-link-title="0">schnable@iastate.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/8/11/3567" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:491245" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:491245/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/11/3567.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:491245" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:491245/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/11/3567.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:491245" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:491245/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/8/11/3567.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:491245" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:491245/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Advances in next generation sequencing technologies and statistical approaches enable genome-wide dissection of phenotypic traits via genome-wide association studies (GWAS). Although multiple statistical approaches for conducting GWAS are available, the power and cross-validation rates of many approaches have been mostly tested using simulated data. Empirical comparisons of single variant (SV) and multi-variant (MV) GWAS approaches have not been conducted to test if a single approach or a combination of SV and MV is effective, through identification and cross-validation of trait-associated loci. In this study, kernel row number (KRN) data were collected from a set of 6,230 entries derived from the Nested Association Mapping (NAM) population and related populations. Three different types of GWAS analyses were performed: 1) single-variant (SV), 2) stepwise regression (STR) and 3) a Bayesian-based multi-variant (BMV) model. Using SV, STR, and BMV models, 257, 300, and 442 KRN-associated variants (KAVs) were identified in the initial GWAS analyses. Of these, 231 KAVs were subjected to genetic validation using three unrelated populations that were not included in the initial GWAS. Genetic validation results suggest that the three GWAS approaches are complementary. Interestingly, KAVs in low recombination regions were more likely to exhibit associations in independent populations than KAVs in recombinationally active regions, probably as a consequence of linkage disequilibrium. The KAVs identified in this study have the potential to enhance our understanding of the genetic basis of ear development.</p></div><span class="kwd-group-title">KEYWORDS:</span><ul class="kwd-group"><li class="kwd"><a href="/keyword/gwas-krn-maize-bayesian" class="hw-term hw-article-keyword hw-article-keyword-gwas-krn-maize-bayesian" rel="nofollow">GWAS KRN maize Bayesian</a></li><li class="kwd"><a href="/keyword/multiparent-advanced-generation-inter-cross-magic" class="hw-term hw-article-keyword hw-article-keyword-multiparent-advanced-generation-inter-cross-magic" rel="nofollow">Multiparent Advanced Generation Inter-Cross (MAGIC)</a></li><li class="kwd"><a href="/keyword/multiparental-populations" class="hw-term hw-article-keyword hw-article-keyword-multiparental-populations" rel="nofollow">multiparental populations</a></li><li class="kwd"><a href="/keyword/mpp" class="hw-term hw-article-keyword hw-article-keyword-mpp" rel="nofollow">MPP</a></li></ul><p id="p-3">Following the adoption of genome wide association studies (GWAS) (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Klein <em>et al.</em> 2005</a>), ∼2,000 loci have been identified as being statistically associated with human disease and other quantitative traits (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Visscher <em>et al.</em> 2012</a>). Using GWAS approaches, hundreds of loci associated with traits have been identified in crops including maize (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Brown <em>et al.</em> 2011</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Tian <em>et al.</em> 2011</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Leiboff <em>et al.</em> 2015</a>), rice (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Huang <em>et al.</em> 2010</a>), sorghum (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Morris <em>et al.</em> 2013</a>) and barley (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Cockram <em>et al.</em> 2010</a>), and in non-crop models such as <em>Arabidopsis</em> (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Atwell <em>et al.</em> 2010</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Meijón <em>et al.</em> 2014</a>).</p><p id="p-4">There are multiple statistical models for conducting GWAS, including both single-variant (SV) and multi-variant (MV) models. SV analysis compares the phenotypic distributions of alternative genotypes at each polymorphic site independently. They can be conducted without correction for population structure or with correction using techniques such as genomic control (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Devlin and Roeder 1999</a>) or principal component analysis (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Price <em>et al.</em> 2006</a>). More recently, genetic relatedness among individuals can be accounted using a kinship matrix in mixed linear models (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Yu <em>et al.</em> 2006</a>). Although SV analysis is often used in published literature, it has a number of inherent limitations, such as not being able to distinguish among the contributions of closely linked loci (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Yang <em>et al.</em> 2012</a>). Sometimes SV analysis overcorrects for the genomic inflation of the test statistics caused by genetic structure because covariates identified via population structure analysis can be associated with causal loci (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Yang <em>et al.</em> 2011</a>). In comparison, MV models have already been demonstrated to be superior in classical linkage analyses, where for example, composite interval mapping outperforms simple interval mapping (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Zeng 1993</a>). MV models can explicitly account for large-effect loci and estimate the effects of multiple loci simultaneously. It has been suggested that the power of GWAS may be improved by conditioning on major-effect QTL (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Kang <em>et al.</em> 2010</a>). One challenge to using MV models, however, is the substantial computational burden associated with analyzing a large number of polymorphic sites. As a partial solution, stepwise regression, which selects markers based on forward inclusion and backward elimination, has been proposed (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Segura <em>et al.</em> 2012</a>). As an alternative to stepwise regression, multi-variant Bayesian models that were initially developed for genomic prediction, by simultaneously fitting all genotyped loci across the genome (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Meuwissen <em>et al.</em> 2001</a>), have been used for GWAS (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Fan <em>et al.</em> 2011</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Habier <em>et al.</em> 2011</a>). These Bayesian-based approaches fit hierarchical models that allow the effects of many loci to shrink toward zero. An empirical comparison of the above models is necessary to investigate their relative advantages in dissecting the genetic architecture of target traits (either individually or in combination).</p><p id="p-5">Many models for conducting GWAS have been compared using simulated data (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Galesloot <em>et al.</em> 2014</a>). Although such studies can provide insight, they suffer from the limitation that simulated data do not necessarily reflect all characteristics of real data because some characteristics of empirical data may be unknown to the simulator. Hence, comparisons based on empirical data are complementary to those based on simulated data. Other than studies that compared different versions of mixed linear model approaches (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Stich <em>et al.</em> 2008</a>), to our knowledge there are no published reports that compare results generated from empirical data analyzed using single-variant (SV), stepwise regression (STR) and Bayesian-based multi-variant (BMV) models for GWAS. In this study, our objective is to compare the effectiveness and/or complementarity of different GWAS (SV, STR, and BMV) models using a single empirical data set. Our second objective is to assess the degree to which our findings from different GWAS models would support other independent empirical data sets.</p><p id="p-6">We perform our initial GWAS analyses on four populations (described in <strong>Material and Methods</strong>) related to the nested association mapping (NAM) population, which combines the strengths of historical recombination events across a broad base of genetic diversity and recombination events that arise following experimental crosses (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Yu <em>et al.</em> 2008</a>). Rather than studying diversity panels often featured in GWAS, we consider multi-parent mapping populations to take advantage of the statistical power of QTL mapping and the gene-level resolution of association mapping.</p><p id="p-7">We study the kernel row number (KRN) phenotype, which is both a component of yield and a model trait for genetic studies in maize (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Hallauer <em>et al.</em> 2010</a>). KRN is highly heritable and a polygenic trait (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Liu <em>et al.</em> 2015a</a>,<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">b</a>) that exhibits little variation in response to environment (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Lu <em>et al.</em> 2011</a>). In addition, KRN is easily scored as an integer, and this scoring can be conducted after completion of the busy pollination season. We collected new KRN data and downloaded previously published KRN data (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Brown <em>et al.</em> 2011</a>).</p><p id="p-8">Collectively, the three GWAS models identified 988 unique KRN-associated variants (KAVs), 231 of which were subjected to genetic validation tests using three unrelated populations that were not included in the initial GWAS. Approximately 60% of the validated KAVs were identified by only one of the three approaches.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>KRN phenotyping for initial GWAS populations</h3><p id="p-9">KRN phenotypes were collected from four related populations: 1) the nested association mapping (NAM, N = 4,699) population which was composed of 25 recombinant inbred line (RIL) subpopulations (<a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Yu <em>et al.</em> 2008</a>), plus RILs of intermated B73 and Mo17 (IBM, N = 325) (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Lee <em>et al.</em> 2002</a>), 2) a subset of the RILs that were backcrossed to the inbred line B73 (B73 x RILs, N = 692 BC1 lines), 3) a subset of the RILs that were backcrossed to the inbred line Mo17 (Mo17 x RILs, N = 289 BC1 lines) and 4) a partial diallel created from the 26 NAM founders and Mo17 (N = 225 F1 hybrids). Because reciprocal crosses were not considered and some of the crosses were not successful, the diallel population was both partial and incomplete (225/351 = 64%).</p><p id="p-10">During the years 2008-2011, NAM RIL populations were sequentially planted in Ames, IA (summer season) and in Molokai, HI (winter season), such that only a fraction of the full set of RILs were grown in a given environment. For each NAM RIL, five plants were grown within each row. KRN data were collected from the mature ears. In addition, we obtained KRN data from eight trials for NAM RILs from a previous study (<a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">Brown <em>et al.</em> 2011</a>). The <a id="xref-ref-2-4" class="xref-bibr" href="#ref-2">Brown <em>et al.</em> (2011)</a> data set is eight times larger than ours. We obtained final phenotypic values by fitting a mixed linear model with fixed effects for entries and random effects for trials. Phenotypic density distributions in this study were estimated and plotted using R with default smoothing parameters.</p><p id="p-11">During the summer of 2011, the B73 x RILs, Mo17 x RILs, and the diallel populations (GWAS populations 2-4) were planted in Ames, IA in 12-plant rows. The B73 x RILs and Mo17x RILs were planted with two replications, and the diallel population was planted with three replications. For each replication, KRN data were collected from three mature ears. The obtained KRN phenotypic data were analyzed using a mixed linear model, where genotype was fitted as a fixed factor and replication was fitted as a random factor.</p></div><div id="sec-3" class="subsection"><h3>KRN phenotyping for subsequent validation populations</h3><p id="p-12">Elite maize inbred lines, extreme KRN USDA accessions, and lines obtained from Iowa long ear synthetic (BSLE) population were used for genetic validation of KAVs identified through GWAS. A total of 220 elite inbred lines, commercial lines that had formerly been subject to IP (Intellectual Property) protection via the plant variety protection act, were obtained from the USDA Plant Introduction (PI) Station in Ames, IA (<a href="http://www.ars.usda.gov/main/site_main.htm?modecode=36-25-12-00">http://www.ars.usda.gov/main/site_main.htm?modecode=36-25-12-00</a>). During year 2011, these accessions were planted in three replications and observed for KRN phenotypes. About 7,000 of the maize accessions have been phenotyped for the KRN trait in the database of USDA PI Station. We selected the 225 accessions with the largest KRN values, the 208 accessions with the smallest KRN values, and 173 random accessions to serve as the second genetic validation population. Empirical KRN phenotypes were obtained from our previously published replicated field experiment (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Yang <em>et al.</em> 2015</a>). Because of the genetic heterogeneity of the accessions, up to 12 random seeds were germinated and pooled together for each accession for DNA isolation. The BSLE population was the product of a long-term selection project conducted to divergently select long and short ears from a single founder population (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Hallauer <em>et al.</em> 2004</a>). Parental lines (N = 10/12) of BSLE and bulked seeds from cycle 0 (C0), cycle 30 short ear (C30 SE) and cycle 30 long ear (C30 LE) were obtained from Arnel Hallauer as our third validation population.</p></div><div id="sec-4" class="subsection"><h3>Genomic variant processing</h3><p id="p-13">A set of 6.2 million genic variants (SNPs and small InDels) was identified via analysis of RNA-seq data from five tissues (shoot apical meristem, ear, tassel, shoot and root) on 26 NAM founder lines and Mo17 (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Lin <em>et al.</em> 2017</a>). Another two sets of genomic variants generated from the maize HapMap project (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bukowski <em>et al.</em> 2015</a>) were extracted from the Panzea database (<a href="https://www.panzea.org">https://www.panzea.org</a>). These three sets of variants were merged using the consensus mode of PLINK (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Purcell <em>et al.</em> 2007</a>). The merged variants were further filtered by discarding variants with a call rate of &lt; 0.4 across entries. We further filtered variants using a minor allele frequency (MAF) cutoff of &lt; 0.1 to exclude minor SNPs only present in fewer than five non-B73 parents. The finalized set consists of 12,966,279 genomic variants on NAM founders, which were used for imputation or projection onto the four related GWAS populations.</p><p id="p-14">Imputation of genotypes was performed as described below. NAM RILs had been directly genotyped using genotyping-by-sequencing (GBS) technology (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Elshire <em>et al.</em> 2011</a>). We obtained the GBS data from the Panzea database (<a href="https://www.panzea.org">https://www.panzea.org</a>). Based on the GBS data and the known pedigrees, the ∼13 million variants discovered in the NAM founders were imputed onto NAM RILs using a python script (<a href="https://github.com/yangjl/zmSNPtools">https://github.com/yangjl/zmSNPtools</a>) as previously described (<a id="xref-ref-45-3" class="xref-bibr" href="#ref-45">Yu <em>et al.</em> 2008</a>). Because the B73xRIL, Mo17xRIL and partial diallel populations were composed of pairs of known haplotypes, their genotypes were directly projected from their parents using the above python script.</p></div><div id="sec-5" class="subsection"><h3>Association variants thinning</h3><p id="p-15">Because progeny of bi-parental crosses comprised much of the initial GWAS population, the strong linkage of genetic variants violated the assumption of independence needed to determine statistical-based thresholds. Therefore, a variant thinning procedure was developed to select the most significant variants, and to avoid concentration of selected variants in certain regions. For variants located in the 28 QTL intervals from the joint QTL analysis and their 1-Mb flanking regions, the top 10 most significant variants were selected. For variants located in other regions, significant variants were determined following the arbitrary thresholds: <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3567/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3567/embed/mml-math-1.gif"/></noscript></span></span> for the SV model, posterior model frequency (MF) &gt; 0.02 for the BMV model and an inclusion <em>P</em> value &lt; 0.05 for the stepwise regression. These significant variants were clustered as groups if none of their pair-wise physical distances exceeded 10-Mb. From these clustered groups, no more than 10 most significant variants were selected.</p></div><div id="sec-6" class="subsection"><h3>Amplicon sequencing for validation of KAVs</h3><p id="p-16">Amplicon sequencing assays were designed for 140/231 (61%) KAVs. A total of 1,102 DNA samples from elite inbred lines (N = 208), extreme KRN lines from the USDA germplasm collection (N = 606) and individuals from the Iowa Long Ear Synthetic (BSLE, N = 288) were individually genotyped, by sequencing all multiplexed amplicons from all 1,102 samples in one HiSeq2000 lane. Informative variants, defined as those which were successfully genotyped, were polymorphic, and had a call rate of &gt; 0.4 and a MAF &gt; 0.05 were used for genetic validation.</p><div id="sec-7" class="subsection"><h4>Statistical Analysis:</h4><div id="sec-8" class="subsection"><h5>Joint QTL analysis</h5><p id="p-17">Joint linkage analysis was performed on NAM and IBM RILs using their corresponding genetic maps. A two-step composite interval mapping (CIM) (<a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Zeng 1993</a>) method was employed using a suite of programs within QTL cartographer (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Silva Lda <em>et al.</em> 2012</a>). First, an automatic forward stepwise regression procedure was used to sequentially test all SNP markers; the most significant marker (inclusion threshold = 0.05) was kept after each iteration. This procedure was repeated until no SNP met the inclusion threshold. In the second step, linkage analyses were conducted at 1-Mb intervals along the chromosome treating previously selected SNPs (other than those within the 1-Mb interval under analysis) as co-variates. A significance threshold was determined by conducting 1,000 permutations and QTL confidence intervals were defined using a 1.5-LOD drop from QTL peak (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Lander and Botstein 1989</a>).</p><p id="p-18">To account for documented stratification effects, the statistical model included fixed effects for population and subpopulation for all three GWAS models described below.</p></div><div id="sec-9" class="subsection"><h5>Single-variant (SV) GWAS model</h5><p id="p-19">Additional fixed effects were fitted in the model to control for effects of QTL on other chromosomes, while all variants on a single chromosome were scanned, resulting in the following model 1 for the <em>k</em>th variant:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3567/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3567/embed/mml-math-2.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>where <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3567/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3567/embed/mml-math-3.gif"/></noscript></span></span> is the adjusted KRN phenotypic value for line <em>l</em> from the mixed linear model analysis; <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3567/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3567/embed/mml-math-4.gif"/></noscript></span></span> is an intercept parameter; <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3567/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3567/embed/mml-math-5.gif"/></noscript></span></span> is 1 if line <em>l</em> is of GWAS population <em>i</em> and is 0 otherwise, and <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3567/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3567/embed/mml-math-6.gif"/></noscript></span></span> is the effect of the <em>i</em>th population in the model for variant <em>k</em>; <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3567/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3567/embed/mml-math-7.gif"/></noscript></span></span> is 1 if line <em>l</em> is from subpopulation <em>j</em> and 0 otherwise, <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3567/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3567/embed/mml-math-8.gif"/></noscript></span></span> is the effect of subpopulation <em>j</em> in the model for variant <em>k</em>; <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3567/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3567/embed/mml-math-9.gif"/></noscript></span></span> indicates the line <em>l</em> genotype of the <em>m</em>th QTL detected by the joint linkage analyses, <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3567/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3567/embed/mml-math-10.gif"/></noscript></span></span> is the effect of the <em>m</em>th QTL in the model for variant <em>k</em>, <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3567/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3567/embed/mml-math-11.gif"/></noscript></span></span> is the set of QTL detected by the joint linkage analysis that are located on chromosomes other than the chromosome of variant <em>k</em>; <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3567/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3567/embed/mml-math-12.gif"/></noscript></span></span> indicates the genotype of the <em>k</em>th variant in line <em>l</em>, <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3567/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3567/embed/mml-math-13.gif"/></noscript></span></span> is the effect of the <em>k</em>th variant; and <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3567/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3567/embed/mml-math-14.gif"/></noscript></span></span> is an error term. This SV model 1 was implemented using SNPTEST v2.3.0 (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Marchini and Howie 2010</a>).</p></div><div id="sec-10" class="subsection"><h5>Stepwise regression (STR) GWAS model</h5><p id="p-20">In the stepwise regression test, population and subpopulation effects were fitted first as fixed effects, and then markers were added in a stepwise manner. For each marker, <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3567/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3567/embed/mml-math-15.gif"/></noscript></span></span> was calculated as the proportion of sums of squares after the fixed effects. We employed the STR method that was implemented in GenSel v4.1 (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Habier <em>et al.</em> 2011</a>) with the option of “StepWise”. We used the following three options to control the STR model, 1) inputMaxRsquared (default 0.8), inputMaxMarkers (300) and alphaValue (default 0.05).</p></div><div id="sec-11" class="subsection"><h5>Bayesian-based multi-variant (BMV) GWAS model</h5><p id="p-21">A Bayesian-based MV model was constructed using the “BayesC” option of GenSel v4.1 (<a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Habier <em>et al.</em> 2011</a>). This model differs from the SV model in that it estimates the effects of all variants simultaneously rather than testing them one-at-a-time. The effects of the variants were fitted as random effects. The following mixed model was used.<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3567/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3567/embed/mml-math-16.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span>where <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3567/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3567/embed/mml-math-17.gif"/></noscript></span></span> indicates the genotype of the <em>k</em>th variant in line <em>l</em> and <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3567/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3567/embed/mml-math-18.gif"/></noscript></span></span> is the effect of the <em>k</em>th variant; other terms in the model are as described in the SV model 1 except that neither the <em>u</em>, <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3567/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3567/embed/mml-math-19.gif"/></noscript></span></span> or <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3567/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3567/embed/mml-math-20.gif"/></noscript></span></span> parameters nor the <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3567/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3567/embed/mml-math-21.gif"/></noscript></span></span> error terms are specific to the <em>k</em>th variant in the BMV model 2.</p><p id="p-22">We trained the BMV model using a two-step procedure. In the first step, we ran 1,000 iterations with 100 burn in of MCMC simulation using default priors, <em>i.e.</em>, genetic variance (genVariance = 1) and residual variance (resVariance = 1). In the 2<sup>nd</sup> step, we replaced the priors using the posteriors obtained from step 1 and ran a longer chain of simulations (chainLength = 41,000 and burnIn = 1,000).</p></div></div></div><div id="sec-12" class="subsection"><h3>Data Availability</h3><p id="p-23">File S1 contains phenotypic data for the GWAS populations. File S2 contains the KAVs identified using the three GWAS models. File S3 contains genotypes of KAVs for the three validation populations. File S4 contains the KRN phenotypic data for the validation populations. In the table, “Internal_id” could be used as the identifier to match genotypic data. File S5 contains the genetic validation results. Stars indicate SNPs that were consistently associated with KRN in initial GWAS, and in at least one of the validation populations. All the supplementary files have been uploaded to Figshare (<a href="/lookup/doi/10.6084/m9.figshare.6902144">10.6084/m9.figshare.6902144</a>). R code for the analyses is available in the public GitHub repository (<a href="https://github.com/yangjl/KRN-GWAS">https://github.com/yangjl/KRN-GWAS</a>). Supplemental material available at Figshare: <a href="/lookup/doi/10.6084/m9.figshare.6902144">10.6084/m9.figshare.6902144</a>.</p></div></div><div class="section results" id="sec-13"><h2 class="">Results</h2><div id="sec-14" class="subsection"><h3>KRN Phenotype in GWAS and Validation Populations</h3><p id="p-24">We collected KRN phenotypic data from 6,230 entries within four related GWAS populations grown at two locations over four years (see Materials and Methods). Best linear unbiased estimators (BLUE) of the KRN phenotype were calculated for each of the 6,230 entries in the four GWAS populations (File S1). In this combined analysis, KRN phenotypic values ranged from 9.1 to 23.6, with a mean of 14.9 rows, whereas the B73 inbred had an above average KRN phenotype of 17.1 rows. Density plots of the four GWAS populations exhibited the expected bell-shaped distributions (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/11/3567/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Phenotypic distributions of the KRN trait in GWAS and validation populations. (A) Density plots of the four GWAS populations. Embedded picture shows the typical KRN counts for B73, B73xMo17 (BxM), Mo17xB73 (MxB) and Mo17 lines. (B) Density plots of two validation populations, elite inbred lines and extreme KRN accessions. Embedded pictures shows examples of an extreme low KRN accession and an extreme high KRN accession. (C) Density plots of cycle 30 long ear (C30 LE) and cycle 30 short ear (C30 SE) in the BSLE population. Embedded pictures indicate the ear length and KRN variation after 30 generations of selection. Blue and red dashed lines indicate the mean KRN values of B73 (KRN = 17.1) and Mo17 (KRN = 10.8)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-845087043" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Phenotypic distributions of the KRN trait in GWAS and validation populations. (A) Density plots of the four GWAS populations. Embedded picture shows the typical KRN counts for B73, B73xMo17 (BxM), Mo17xB73 (MxB) and Mo17 lines. (B) Density plots of two validation populations, elite inbred lines and extreme KRN accessions. Embedded pictures shows examples of an extreme low KRN accession and an extreme high KRN accession. (C) Density plots of cycle 30 long ear (C30 LE) and cycle 30 short ear (C30 SE) in the BSLE population. Embedded pictures indicate the ear length and KRN variation after 30 generations of selection. Blue and red dashed lines indicate the mean KRN values of B73 (KRN = 17.1) and Mo17 (KRN = 10.8).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/11/3567/F1.medium.gif" width="306" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/8/11/3567/F1.medium.gif" width="306" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/11/3567/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/11/3567/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491439" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-25" class="first-child">Phenotypic distributions of the KRN trait in GWAS and validation populations. (A) Density plots of the four GWAS populations. Embedded picture shows the typical KRN counts for B73, B73xMo17 (BxM), Mo17xB73 (MxB) and Mo17 lines. (B) Density plots of two validation populations, elite inbred lines and extreme KRN accessions. Embedded pictures shows examples of an extreme low KRN accession and an extreme high KRN accession. (C) Density plots of cycle 30 long ear (C30 LE) and cycle 30 short ear (C30 SE) in the BSLE population. Embedded pictures indicate the ear length and KRN variation after 30 generations of selection. Blue and red dashed lines indicate the mean KRN values of B73 (KRN = 17.1) and Mo17 (KRN = 10.8).</p><div class="sb-div caption-clear"></div></div></div><p id="p-26">We also collected KRN data from three unrelated populations for subsequent validation purpose. The three populations consist of elite inbred lines that expired from US plant variety protection (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Nelson <em>et al.</em> 2008</a>), extreme KRN accessions from USDA germplasm database, and a long term selection population — Iowa long ear synthetic (BSLE). The elite lines have KRN that are less extreme and less variable than the NAM lines (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a>). This fits with our understanding of breeder practices; they do not select for high KRN phenotypes. The USDA Plant Introduction station maintains a large collection of maize germplasm. The KRN phenotypes in this population are extreme, ranging from an average of 13-30 rows in the high KRN pool to 6-12 rows in the low KRN pool according to data obtained from a replicated field trial (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1B</a>) (<a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Yang <em>et al.</em> 2015</a>). The BSLE population had been subjected to 30 generations of divergent selection for long ears (LE) and short ears (SE) (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Hallauer <em>et al.</em> 2010</a>). During selection, KRN exhibited a negatively correlated response to ear length (<span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3567/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3567/embed/mml-math-22.gif"/></noscript></span></span> Pearson’s correlation test <em>P</em> value <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3567/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3567/embed/mml-math-23.gif"/></noscript></span></span>), <em>i.e.</em>, longer and shorter ears had smaller and larger KRN trait values, respectively (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1C</a>). The parental lines and bulked seeds from cycle 0 (C0), cycle 30 long ear (C30 LE) and cycle 30 short ear (C30 SE) served as our third genetic validation population.</p></div><div id="sec-15" class="subsection"><h3>Identification of KAVs With different statistical models using GWAS populations</h3><p id="p-27">In each GWAS model, population and subpopulation were included as fixed effects to account for inherent structure in the 6,230 entries included in the GWAS. First, a SV model (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Manolio 2010</a>) was used to scan the ∼13M variants one-by-one using QTL detected in the joint analysis as covariates. The linear mixed model with genetic relationship matrix was not used here because the four initial GWAS populations are NAM and NAM related multi-parent mapping populations. Using an arbitrary cutoff of <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3567/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3567/embed/mml-math-24.gif"/></noscript></span></span> this approach identified linked clusters of variants (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2C</a>), most of which were located within the 28 QTL intervals that had been identified by the joint QTL analysis (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2D</a>). To diminish the over-representation of certain regions by significant variants, a binning (bin size = 100-kb) procedure was used that resulted in the identification of 257 KAVs, which in combination accounted for 51% of the KRN variation.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/11/3567/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Stacked plots of GWAS and QTL results. From upper to lower panels are results from the Bayesian-based multi-variant (A) stepwise regression (B) and single variant(C) models for GWAS and the joint QTL mapping result (D). The red dashed line in the QTL plot indicates the 1,000 permutation threshold and black lines show the QTL confidence intervals. Red squares in panel (A), triangles in panel (B) and circles in panel (C) indicate the kernel row number associated variants selected for further genetic validation." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-845087043" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Stacked plots of GWAS and QTL results. From upper to lower panels are results from the Bayesian-based multi-variant (A) stepwise regression (B) and single variant(C) models for GWAS and the joint QTL mapping result (D). The red dashed line in the QTL plot indicates the 1,000 permutation threshold and black lines show the QTL confidence intervals. Red squares in panel (A), triangles in panel (B) and circles in panel (C) indicate the kernel row number associated variants selected for further genetic validation.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/11/3567/F2.medium.gif" width="440" height="201"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/8/11/3567/F2.medium.gif" width="440" height="201"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/11/3567/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/11/3567/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491401" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-28" class="first-child">Stacked plots of GWAS and QTL results. From upper to lower panels are results from the Bayesian-based multi-variant (A) stepwise regression (B) and single variant(C) models for GWAS and the joint QTL mapping result (D). The red dashed line in the QTL plot indicates the 1,000 permutation threshold and black lines show the QTL confidence intervals. Red squares in panel (A), triangles in panel (B) and circles in panel (C) indicate the kernel row number associated variants selected for further genetic validation.</p><div class="sb-div caption-clear"></div></div></div><p id="p-29">Second, in an attempt to improve mapping resolution, a STR approach was used, which resulted in identification of 300 KAVs representing 296 100-kb bins (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2B</a>). In combination, these variants accounted for 78% of phenotypic variation.</p><p id="p-30">Third, a BMV model (<a id="xref-ref-11-4" class="xref-bibr" href="#ref-11">Habier <em>et al.</em> 2011</a>) was used to estimate effects of all ∼13M variants simultaneously. After applying the variant thinning procedure and cutoffs described in Materials and Methods, a set of 442 variants representing 343 100-kb bins, which together accounted for 74% of the phenotypic variation, was identified (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2A</a>). Most promisingly, this model identified smaller chromosomal intervals than the SV model.</p></div><div id="sec-16" class="subsection"><h3>Comparison of KAVs identified by three GWAS Models</h3><p id="p-31">In combination, the three GWAS models identified 764 100-kb bins (File S2), each of which contained one or more significant variants. Encouragingly, among these 764 bins, 66 (containing 169 variants) were detected by at least two models. Only one of these bins was detected by all three GWAS models. That bin (chr4:229.0-Mb) overlaps the most significant QTL peak detected in the joint QTL study. To estimate an upper bound for cross-validation rate for each approach and to determine whether the KAVs that were detected by more than one approach are more reliable, a set of 231 KAVs was selected for genetic validation. This set of KAVs included the 169 variants in the 66 bins detected by at least two approaches and 62 of the most significant one or two variants selected from 20 bins that had only been detected by one approach (approach-specific variants). Hence, in total 231 KAVs from a total of 126 bins (66 + 20 × 3) were selected for genetic validation.</p><p id="p-32">Collectively, the 231 selected KAVs explained 64% of phenotypic variation in the initial GWAS population by fitting these KAVs simultaneously using an additive model (<em>i.e.</em>, narrow-sense heritability <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3567/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3567/embed/mml-math-25.gif"/></noscript></span></span>) . Individually, most of the KAVs (83%, 192/231) explained less than 5% of the phenotypic variation, but 17% (39/231) of the KAVs individually accounted between 5–10% of phenotypic variation.</p><p id="p-33">Note that a causal variant might be represented by multiple selected KAVs. The B73 variant-types were alleles for higher KRN for nearly three-quarters (73%, 168/231) of these KAVs, consistent with the high KRN value of the B73 genotype. Consistent with our previous study (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Li <em>et al.</em> 2012</a>), KAVs are substantially enriched for variants located within genes or within 5-kb upstream of genes (2.0-fold change, Chi-square <em>P</em> value &lt; 0.01) and enriched in variants discovered from the RNA-seq data (1.ninefold change, Chi-square <em>P</em> value &lt; 0.01) relative to the ∼13M variants used for GWAS.</p></div><div id="sec-17" class="subsection"><h3>Genetic validation of KAVs using three unrelated populations</h3><p id="p-34">To distinguish true positive association signals from potentially false positive associations, three previously described genetic validation populations that are unrelated to the GWAS populations and to each other were genotyped at the KAVs (genotype data are provided in File S3 and S4).</p><p id="p-35">To control for population structure in the elite inbred lines, a set of SNPs that had previously been used to genotype a subset (N = 91) of these lines was obtained (<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Nelson <em>et al.</em> 2008</a>). We fitted a mixed linear model to estimate the fixed effects of KAVs. In the model, we included random effects for lines. These random effects were assumed to be correlated according to a kinship matrix calculated from the genome-wide SNPs (<a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">Nelson <em>et al.</em> 2008</a>). Using this approach, 22/70 (31%) of the informative KAVs could be genetically validated in the set of 91 elite inbreds with an FDR &lt; 0.05. Because the elite inbreds are not closely related to the GWAS populations, it is unlikely that uncontrolled population structure could yield false-positive validation assays for KAVs derived from the GWAS populations. Hence, we also conducted a naive analysis using the entire set of elite inbreds (N = 209) without controlling for population structure. In this analysis, 33/70 (47%) of the KAVs, which included all of the 22 KAVs discussed above, could be validated.</p><p id="p-36">Because extreme KRN accessions were maintained via random pollination within accessions, individual accessions are both heterogeneous and heterozygous. We therefore genotyped pools of DNA extracted from up to 12 plants per accession. A model fitted to the estimated allele frequencies was used to test the hypothesis that alleles for higher KRN have higher frequencies in the high KRN pools than in low KRN pools. Among the 56/131 (43%) informative variants, 14/56 (25%) could be validated.</p><p id="p-37">Of the 51 informative KAVs in the BSLE population, 7/51 (14%) showed significant differences in allele frequency between C30 LE and C30 SE populations. To rule out the possibilities of genetic drift or stochastic sampling error, we conducted simulation to mimic the selection procedure. After simulation, one validated KAV did not pass the cutoff (FDR &lt; 0.05) and was removed. Hence, even after accounting for drift and stochastic sampling errors, 6/51 (12%) KAVs were deemed to have been under divergent selection. We applied the simulation to the negative control variants, and they were indeed negative. Collectively, these loci account for 40% of the total between-population variance for KRN trait. Variants that are segregating in BSLE but not in GWAS populations or that were simply not detected as being KAVs in the GWAS populations may explain the remaining phenotypic variation between C30 LE and C30 SE.</p></div><div id="sec-18" class="subsection"><h3>Summary of the genetic validation results</h3><p id="p-38">In summary, 40/77 (52%) of informative KAVs exhibited associations with the KRN trait in at least one of the genetic validation populations (File S5). The genetic validation results from the three GWAS models are illustrated in <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>. Considering all KAVs detected by each approach, the validation rates were 61% (20/33) for the single-variant approach, 43% (6/14) for the stepwise regression approach and 45% (14/31) for the Bayesian-based approach (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a>). Validation rates were 67% (10/15) for KAVs detected only by the SV model, 43% (6/14) for those detected only by STR model, 35% (9/26) for KAVs detected only by the BMV model, 73% (16/22) for KAVs detected by both SV and BMV models, and 11% (1/9) for control variants that are randomly chosen from a set of SNPs that were not associated with KRN in the initial GWAS (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3A</a>). Although both the STR and BMV models had lower validation rates than the SV model, these results demonstrate that each of the three models identified validated KAVs that were not identified by other approaches. Thus, the three GWAS models are complementary.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/11/3567/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genetic validation results of selected kernel row number associated variants (KAVs). (A) Transformed P values using single variant (SV) model and posterior model frequencies using Bayesian-based multi-variant (BMV) model were extracted and plotted for the 77 informative KAVs identified by at least one of the three GWAS models. KAVs detected only by the SV model are plotted in the lower right quadrant, KAVs detected only by the stepwise regression model are plotted as non-gray dots in the lower left quadrant, the KAVs detected only by the BMV model are plotted in the upper left quadrant, KAVs detected by both the SV and BMV models are plotted in the upper right quadrant and control variants are plotted as gray dots in the lower left quadrant. Validated KAVs are marked in red. A control variant is a genomic variant that is randomly chosen from a set of SNPs that were not associated with KRN in the initial GWAS. (B) Venn diagram of the validation results." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-845087043" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Genetic validation results of selected kernel row number associated variants (KAVs). (A) Transformed P values using single variant (SV) model and posterior model frequencies using Bayesian-based multi-variant (BMV) model were extracted and plotted for the 77 informative KAVs identified by at least one of the three GWAS models. KAVs detected only by the SV model are plotted in the lower right quadrant, KAVs detected only by the stepwise regression model are plotted as non-gray dots in the lower left quadrant, the KAVs detected only by the BMV model are plotted in the upper left quadrant, KAVs detected by both the SV and BMV models are plotted in the upper right quadrant and control variants are plotted as gray dots in the lower left quadrant. Validated KAVs are marked in red. A control variant is a genomic variant that is randomly chosen from a set of SNPs that were not associated with KRN in the initial GWAS. (B) Venn diagram of the validation results.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/11/3567/F3.medium.gif" width="440" height="210"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/8/11/3567/F3.medium.gif" width="440" height="210"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/11/3567/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/11/3567/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491307" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-39" class="first-child">Genetic validation results of selected kernel row number associated variants (KAVs). (A) Transformed P values using single variant (SV) model and posterior model frequencies using Bayesian-based multi-variant (BMV) model were extracted and plotted for the 77 informative KAVs identified by at least one of the three GWAS models. KAVs detected only by the SV model are plotted in the lower right quadrant, KAVs detected only by the stepwise regression model are plotted as non-gray dots in the lower left quadrant, the KAVs detected only by the BMV model are plotted in the upper left quadrant, KAVs detected by both the SV and BMV models are plotted in the upper right quadrant and control variants are plotted as gray dots in the lower left quadrant. Validated KAVs are marked in red. A control variant is a genomic variant that is randomly chosen from a set of SNPs that were not associated with KRN in the initial GWAS. (B) Venn diagram of the validation results.</p><div class="sb-div caption-clear"></div></div></div><p id="p-40">We also obtained genotyping data for 34 KAVs reported in an earlier GWAS (<a id="xref-ref-2-5" class="xref-bibr" href="#ref-2">Brown <em>et al.</em> 2011</a>). Using the statistical analyses described above, 26% (9/34) of these KAVs could be validated in at least one of the three unrelated populations.</p><p id="p-41">The amount of genetic recombination per Mb in maize varies substantially across the genome (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Fu <em>et al.</em> 2002</a>). To investigate whether the probability of genetic validation varies based on the amount of recombination per Mb, the 111 tested KAVs (77 from this study and 34 from <a id="xref-ref-2-6" class="xref-bibr" href="#ref-2">Brown <em>et al.</em> (2011)</a>) were projected onto the NAM genetic map (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Buckler <em>et al.</em> 2009</a>) using our previously published method (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Liu <em>et al.</em> 2009</a>). Recombination rates (cM/Mb) were estimated for every 10 cM window. KAVs were classified as being located in regions recombinationally “cold” (&lt; 1 cM/Mb) or “hot” (&gt; 1 cM/Mb) chromosomal regions. KAVs located in recombinational cold zones were 3.5 × more likely to be genetically validated than those in recombinational hot zones (Chi-square <em>P</em> value &lt; 0.03).</p></div></div><div class="section discussion" id="sec-19"><h2 class="">Discussion</h2><p id="p-42">GWAS is typically associated with high rates of false discovery (<a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Visscher <em>et al.</em> 2012</a>). In human studies, a second cohort is often used to genetically validate the most significant SNPs discovered in the first cohort, thereby cost effectively reducing the number of false discoveries (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Sladek <em>et al.</em> 2007</a>). To our knowledge, no large sets of genetic variants identified via GWAS as being associated with a trait of interest has been subjected to this type of genetic validation in maize. Here, we report on such a set of genetic variants that were consistently detected as being associated with KRN in both the initial GWAS and the validation populations, each of which therefore has the potential to enhance our understanding of ear development in maize.</p><p id="p-43">Each GWAS model has its own strengths and weaknesses. Results from our study indicate that the three GWAS models complement each other through initial association studies and cross-validation of KAVs. In this study, genetic validation strategies that exploit the extensive genetic resources of maize were used to estimate maximum rates of cross-validation. This was accomplished by testing whether KAVs identified via the GWAS also exhibit associations with the KRN trait in independent populations. We tested not only KAVs in or near genes that had previously been associated with the KRN trait via functional analyses, but also KAVs that were not located in or near genes with prior evidence of affecting the KRN trait. There are multiple reasons for a KAV that would not be genetically cross-validated. These include biological differences in the genetic control of the KRN trait among populations and Type II errors in the validation analyses.</p><p id="p-44">Overall, at least 52% (40/77) of KAVs were cross-validated in at least one of three unrelated populations. Because KRN is mainly controlled by additive effect loci (<a id="xref-ref-2-7" class="xref-bibr" href="#ref-2">Brown <em>et al.</em> 2011</a>; <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Lu <em>et al.</em> 2010</a>), traits controlled by different modes of inheritance may yield different validation results. Because the four GWAS populations were all derived from 27 founder lines, they differ from a diversity panel with respect to the genetic base. While diversity panels exploit LD from a broader genetic base, multi-parent mapping populations are usually derived from few founders (narrow genetic base), which could also contribute to different validation results. Therefore, it may not be possible to generalize our results from these structured multi-parent mapping populations to other types of populations such as diversity panels. Further, GWAS conducted in plants often have access to immortalized genotypes and replicated observations, which provides the opportunity to better control for stochastic factors, such as environmental effects, that could affect the rate of false discovery as compared to GWAS conducted on humans or some other species.</p><p id="p-45">Although the SV model had a somewhat higher genetic validation rate than the other two models (possibly at least partly because of analytic similarities between the SV model and the genetic validation experiments), each GWAS model identified validated KAVs that were not detected by the others. By definition, this means that the three GWAS models are complementary. Therefore, the use of multiple approaches or the development of a statistical model that combines their advantages, promises to enhance the power of GWAS.</p><p id="p-46">The genetic validation rate of KAVs identified in this experiment (40/77 = 52%) is higher than KAVs identified in an earlier KRN GWAS study (9/34 = 26%) (<a id="xref-ref-2-8" class="xref-bibr" href="#ref-2">Brown <em>et al.</em> 2011</a>). The improved power of our study (which made use of data from <a id="xref-ref-2-9" class="xref-bibr" href="#ref-2">Brown <em>et al.</em> (2011)</a>, as well as additional data generated as part of the current study) could be due to the use of three complementary GWAS models for identifying KAVs, the inclusion of more genotypes, more phenotypic data and/or higher marker density.</p><p id="p-47">KAVs located in chromosomal regions with low rates of recombination (cM/Mb) were 3.5 times more likely to be genetically validated than those in chromosomal regions with high rates of recombination per physical distance. This is probably a consequence of the relationship between recombination and LD (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Kim <em>et al.</em> 2007</a>). Specifically, a KAV that is not causative but that is only linked to the causative variant is more likely to exhibit an association with the KRN trait in an independent population if it is located in a large LD block as compared to a KAV that is in a region with low LD, as a consequence of higher rates of recombination separating a marker from the functional polymorphism across different genetic backgrounds.</p><p id="p-48">In conclusion, this study identified hundreds of KAVs that in combination explained 64% of phenotypic variation for KRN in lines that sample ∼60% of the genetic diversity of maize (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Liu <em>et al.</em> 2003</a>). Over 50% of KAVs that were tested could be genetically validated. In-depth analyses of KAV-linked genes will enable us to better understand the molecular and developmental processes that control variation in the KRN trait and may eventually be useful in breaking the negative correlation between KRN and ear length, thereby increasing grain yields.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-49">We gratefully acknowledge Dr. Arnel Hallauer, Dr. Kendall Lamkey and Mr. Paul White of Iowa State University and Dr. Candice Gardner of the USDA’s North Central Regional Plant Introduction Station (NCRPIS) for sharing genetic stocks, Drs. Sanzhen Liu (currently Kansas State University) and Ed Allen (Monsanto) for useful discussions, Dr. Wei Wu (currently LGC Genomics), Dr. Haiying Jiang (currently Shenyang Agricultural University), Dr. Li Li (currently China Agricultural University), Ms. Uyen Pham and Ms. Talissa Sari for technical support, and Ms. Lisa Coffey for the generation and maintenance of genetic stocks. We also thank anonymous reviewers for their suggestions.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-50">Supplemental material available at Figshare: 10.6084/m9.figshare.6902144.</p></li><li class="fn" id="fn-2"><p id="p-51"><em>Communicating editor: J. Holland</em></p></li><li class="fn" id="fn-3"><p id="p-52">Copyright © 2018 by the Genetics Society of America</p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2018-08-02"><span class="received-label">Received </span>August 2, 2018.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2018-08-31"><span class="accepted-label">Accepted </span>August 31, 2018.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2018 Yang <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3567.1" data-doi="10.1038/nature08800"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Atwell</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">Y. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vilhjálmsson</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Willems</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Horton</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genome-wide association study of 107 phenotypes in Arabidopsis thaliana inbred lines.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">465</span>: <span class="cit-fpage">627</span>–<span class="cit-lpage">631</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature08800</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DAtwell%26rft.auinit1%253DS.%26rft.volume%253D465%26rft.issue%253D7298%26rft.spage%253D627%26rft.epage%253D631%26rft.atitle%253DGenome-wide%2Bassociation%2Bstudy%2Bof%2B107%2Bphenotypes%2Bin%2BArabidopsis%2Bthaliana%2Binbred%2Blines.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08800%26rft_id%253Dinfo%253Apmid%252F20336072%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature08800&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20336072&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3567.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000278249000044&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3567.2" data-doi="10.1371/journal.pgen.1002383"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Upadyayula</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mahone</span>  <span class="cit-name-given-names">G. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tian</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bradbury</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Distinct Genetic Architectures for Male and Female Inflorescence Traits of Maize.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e1002383</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1002383</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DBrown%26rft.auinit1%253DP.%2BJ.%26rft.volume%253D7%26rft.issue%253D11%26rft.spage%253De1002383%26rft.epage%253De1002383%26rft.atitle%253DDistinct%2Bgenetic%2Barchitectures%2Bfor%2Bmale%2Band%2Bfemale%2Binflorescence%2Btraits%2Bof%2Bmaize.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002383%26rft_id%253Dinfo%253Apmid%252F22125498%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002383&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22125498&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3567.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3567.3" data-doi="10.1126/science.1174276"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Buckler</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holland</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bradbury</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Acharya</span>  <span class="cit-name-given-names">C. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The genetic architecture of maize flowering time.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">325</span>: <span class="cit-fpage">714</span>–<span class="cit-lpage">718</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1174276</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DBuckler%26rft.auinit1%253DE.%2BS.%26rft.volume%253D325%26rft.issue%253D5941%26rft.spage%253D714%26rft.epage%253D718%26rft.atitle%253DThe%2BGenetic%2BArchitecture%2Bof%2BMaize%2BFlowering%2BTime%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1174276%26rft_id%253Dinfo%253Apmid%252F19661422%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjUvNTk0MS83MTQiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzExLzM1NjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3567.4" data-doi="10.1093/gigascience/gix134"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bukowski</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zou</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Construction of the third generation zea mays haplotype map.</span> <abbr class="cit-jnl-abbrev">Gigascience</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">12</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/gigascience/gix134</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGigascience%26rft.volume%253D7%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgigascience%252Fgix134%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/gigascience/gix134&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3567.5" data-doi="10.1073/pnas.1010179107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cockram</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">White</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zuluaga</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Comadran</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genome-wide association mapping to candidate polymorphism resolution in the unsequenced barley genome.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">21611</span>–<span class="cit-lpage">21616</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1010179107</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1010179107%26rft_id%253Dinfo%253Apmid%252F21115826%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA3LzUwLzIxNjExIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMS8zNTY3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3567.6" data-doi="10.1111/j.0006-341X.1999.00997.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Devlin</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roeder</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Genomic Control for Association Studies.</span> <abbr class="cit-jnl-abbrev">Biometrics</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">997</span>–<span class="cit-lpage">1004</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.0006-341X.1999.00997.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrics%26rft.stitle%253DBiometrics%26rft.aulast%253DDevlin%26rft.auinit1%253DB.%26rft.volume%253D55%26rft.issue%253D4%26rft.spage%253D997%26rft.epage%253D1004%26rft.atitle%253DGenomic%2Bcontrol%2Bfor%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.0006-341X.1999.00997.x%26rft_id%253Dinfo%253Apmid%252F11315092%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.0006-341X.1999.00997.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11315092&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3567.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000084218000001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3567.7" data-doi="10.1371/journal.pone.0019379"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawamoto</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A robust, simple genotyping-by-sequencing (gbs) approach for high diversity species.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e19379</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0019379</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DElshire%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253De19379%26rft.epage%253De19379%26rft.atitle%253DA%2Brobust%252C%2Bsimple%2Bgenotyping-by-sequencing%2B%2528GBS%2529%2Bapproach%2Bfor%2Bhigh%2Bdiversity%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0019379%26rft_id%253Dinfo%253Apmid%252F21573248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0019379&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21573248&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3567.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3567.8" data-doi="10.1371/journal.pone.0014726"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fan</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Onteru</span>  <span class="cit-name-given-names">S. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Du</span>  <span class="cit-name-given-names">Z. Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garrick</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stalder</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome-wide association study identifies loci for body composition and structural soundness traits in pigs.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e14726</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0014726</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DFan%26rft.auinit1%253DB.%26rft.volume%253D6%26rft.issue%253D2%26rft.spage%253De14726%26rft.epage%253De14726%26rft.atitle%253DGenome-wide%2Bassociation%2Bstudy%2Bidentifies%2BLoci%2Bfor%2Bbody%2Bcomposition%2Band%2Bstructural%2Bsoundness%2Btraits%2Bin%2Bpigs.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0014726%26rft_id%253Dinfo%253Apmid%252F21383979%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0014726&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21383979&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3567.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3567.9" data-doi="10.1073/pnas.022635499"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dooner</span>  <span class="cit-name-given-names">H. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Recombination rates between adjacent genic and retrotransposon regions in maize vary by 2 orders of magnitude.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">99</span>: <span class="cit-fpage">1082</span>–<span class="cit-lpage">1087</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.022635499</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.022635499%26rft_id%253Dinfo%253Apmid%252F11792865%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI5OS8yLzEwODIiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzExLzM1NjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3567.10" data-doi="10.1371/journal.pone.0095923"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Galesloot</span>  <span class="cit-name-given-names">T. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Steen</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kiemeney</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Janss</span>  <span class="cit-name-given-names">L. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vermeulen</span>  <span class="cit-name-given-names">S. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A comparison of multivariate genome-wide association methods.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e95923</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0095923</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De95923%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0095923%26rft_id%253Dinfo%253Apmid%252F24763738%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0095923&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24763738&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3567.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3567.11" data-doi="10.1186/1471-2105-12-186"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Habier</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kizilkaya</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garrick</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Extension of the bayesian alphabet for genomic selection.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">186</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-12-186</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DHabier%26rft.auinit1%253DD.%26rft.volume%253D12%26rft.spage%253D186%26rft.epage%253D186%26rft.atitle%253DExtension%2Bof%2Bthe%2Bbayesian%2Balphabet%2Bfor%2Bgenomic%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-12-186%26rft_id%253Dinfo%253Apmid%252F21605355%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-12-186&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21605355&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3567.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-book" id="cit-8.11.3567.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hallauer</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carena</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miranda Filho</span>  <span class="cit-name-given-names">J. d.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-source">Quantitative genetics in maize breeding</span>, <span class="cit-vol">Vol. 6</span>. <span class="cit-publ-name">Springer Science &amp; Business Media</span>, <span class="cit-publ-loc">Berlin, Germany</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-other" id="cit-8.11.3567.13"><div class="cit-metadata unstructured">Hallauer, A. R., A. J. Ross, and M. Lee, 2004 Long-term Divergent Selection for Ear Length in Maize 24.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3567.14" data-doi="10.1038/ng.695"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sang</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feng</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genome-wide association studies of 14 agronomic traits in rice landraces.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">961</span>–<span class="cit-lpage">967</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.695</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DHuang%26rft.auinit1%253DX.%26rft.volume%253D42%26rft.issue%253D11%26rft.spage%253D961%26rft.epage%253D967%26rft.atitle%253DGenome-wide%2Bassociation%2Bstudies%2Bof%2B14%2Bagronomic%2Btraits%2Bin%2Brice%2Blandraces.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.695%26rft_id%253Dinfo%253Apmid%252F20972439%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.695&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20972439&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3567.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000283540500012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3567.15" data-doi="10.1038/ng.548"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sul</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Service</span>  <span class="cit-name-given-names">S. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zaitlen</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kong</span>  <span class="cit-name-given-names">S.-Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Variance component model to account for sample structure in genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">348</span>–<span class="cit-lpage">354</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.548</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DKang%26rft.auinit1%253DH.%2BM.%26rft.volume%253D42%26rft.issue%253D4%26rft.spage%253D348%26rft.epage%253D354%26rft.atitle%253DVariance%2Bcomponent%2Bmodel%2Bto%2Baccount%2Bfor%2Bsample%2Bstructure%2Bin%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.548%26rft_id%253Dinfo%253Apmid%252F20208533%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.548&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20208533&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3567.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276150500016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3567.16" data-doi="10.1038/ng2115"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plagnol</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">T. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Toomajian</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Recombination and linkage disequilibrium in Arabidopsis thaliana.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">1151</span>–<span class="cit-lpage">1155</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng2115</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DKim%26rft.auinit1%253DS.%26rft.volume%253D39%26rft.issue%253D9%26rft.spage%253D1151%26rft.epage%253D1155%26rft.atitle%253DRecombination%2Band%2Blinkage%2Bdisequilibrium%2Bin%2BArabidopsis%2Bthaliana.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng2115%26rft_id%253Dinfo%253Apmid%252F17676040%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng2115&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17676040&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3567.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3567.17" data-doi="10.1126/science.1109557"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Klein</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zeiss</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chew</span>  <span class="cit-name-given-names">E. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsai</span>  <span class="cit-name-given-names">J.-Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sackler</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Complement factor H polymorphism in age-related macular degeneration.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">308</span>: <span class="cit-fpage">385</span>–<span class="cit-lpage">389</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1109557</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1109557%26rft_id%253Dinfo%253Apmid%252F15761122%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMDgvNTcyMC8zODUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzExLzM1NjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3567.18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lander</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Botstein</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Mapping mendelian factors underlying quantitative traits using RFLP linkage maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">121</span>: <span class="cit-fpage">185</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLander%26rft.auinit1%253DE.%2BS.%26rft.volume%253D121%26rft.issue%253D1%26rft.spage%253D185%26rft.epage%253D199%26rft.atitle%253DMapping%2Bmendelian%2Bfactors%2Bunderlying%2Bquantitative%2Btraits%2Busing%2BRFLP%2Blinkage%2Bmaps.%26rft_id%253Dinfo%253Apmid%252F2563713%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTIxLzEvMTg1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMS8zNTY3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3567.19" data-doi="10.1023/A:1014893521186"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sharopova</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beavis</span>  <span class="cit-name-given-names">W. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grant</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Katt</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Expanding the genetic map of maize with the intermated B73 x Mo17 (IBM) population.</span> <abbr class="cit-jnl-abbrev">Plant Mol. Biol.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">453</span>–<span class="cit-lpage">461</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1023/A:1014893521186</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BMol%2BBiol%26rft.aulast%253DLee%26rft.auinit1%253DM.%26rft.volume%253D48%26rft.issue%253D5-6%26rft.spage%253D453%26rft.epage%253D461%26rft.atitle%253DExpanding%2Bthe%2Bgenetic%2Bmap%2Bof%2Bmaize%2Bwith%2Bthe%2Bintermated%2BB73%2Bx%2BMo17%2B%2528IBM%2529%2Bpopulation.%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1014893521186%26rft_id%253Dinfo%253Apmid%252F11999829%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/A:1014893521186&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11999829&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3567.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000174606400003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3567.20" data-doi="10.1038/ncomms9974"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Leiboff</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">H.-C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Todt</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genetic control of morphometric diversity in the maize shoot apical meristem.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">8974</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ncomms9974</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D6%26rft.spage%253D8974%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms9974%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms9974&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3567.21" data-doi="10.1101/gr.140277.112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yeh</span>  <span class="cit-name-given-names">C.-T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Takacs</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genic and nongenic contributions to natural variation of quantitative traits in maize.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">2436</span>–<span class="cit-lpage">2444</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.140277.112</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.140277.112%26rft_id%253Dinfo%253Apmid%252F22701078%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIyMi8xMi8yNDM2IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMS8zNTY3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3567.22" data-doi="10.1186/s13059-017-1328-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">H.-y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Substantial contribution of genetic variation in the expression of transcription factors to phenotypic variation revealed by erd-gwas.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">192</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s13059-017-1328-6</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D18%26rft.spage%253D192%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-017-1328-6%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-017-1328-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3567.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goodman</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muse</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buckler</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Genetic structure and diversity among maize inbred lines as inferred from DNA microsatellites.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">165</span>: <span class="cit-fpage">2117</span>–<span class="cit-lpage">2128</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLiu%26rft.auinit1%253DK.%26rft.volume%253D165%26rft.issue%253D4%26rft.spage%253D2117%26rft.epage%253D2128%26rft.atitle%253DGenetic%2BStructure%2Band%2BDiversity%2BAmong%2BMaize%2BInbred%2BLines%2Bas%2BInferred%2BFrom%2BDNA%2BMicrosatellites%26rft_id%253Dinfo%253Apmid%252F14704191%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2NS80LzIxMTciO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzExLzM1NjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3567.24" data-doi="10.1007/s00122-015-2581-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Du</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huo</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span><span class="cit-comment">a</span> <span class="cit-article-title">Genetic architecture of maize kernel row number and whole genome prediction.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">128</span>: <span class="cit-fpage">2243</span>–<span class="cit-lpage">2254</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-015-2581-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D128%26rft.spage%253D2243%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-015-2581-2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-015-2581-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3567.25" data-doi="10.1371/journal.pgen.1005670"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Du</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span><span class="cit-comment">b</span> <span class="cit-article-title">Krn4 controls quantitative variation in maize kernel row number.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">e1005670</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1005670</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D11%26rft.spage%253De1005670%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1005670%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1005670&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3567.26" data-doi="10.1371/journal.pgen.1000733"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yeh</span>  <span class="cit-name-given-names">C. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ji</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ying</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Mu transposon insertion sites and meiotic recombination events co-localize with epigenetic marks for open chromatin across the maize genome.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e1000733</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1000733</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DLiu%26rft.auinit1%253DS.%26rft.volume%253D5%26rft.issue%253D11%26rft.spage%253De1000733%26rft.epage%253De1000733%26rft.atitle%253DMu%2Btransposon%2Binsertion%2Bsites%2Band%2Bmeiotic%2Brecombination%2Bevents%2Bco-localize%2Bwith%2Bepigenetic%2Bmarks%2Bfor%2Bopen%2Bchromatin%2Bacross%2Bthe%2Bmaize%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000733%26rft_id%253Dinfo%253Apmid%252F19936291%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000733&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19936291&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3567.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3567.27" data-doi="10.1007/s11032-010-9468-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">C. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hao</span>  <span class="cit-name-given-names">Z. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Mapping of quantitative trait loci for kernel row number in maize across seven environments.</span> <abbr class="cit-jnl-abbrev">Mol. Breed.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">143</span>–<span class="cit-lpage">152</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s11032-010-9468-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBreed.%26rft.volume%253D28%26rft.spage%253D143%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11032-010-9468-3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11032-010-9468-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3567.28" data-doi="10.1056/NEJMra0905980"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Manolio</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genomewide association studies and assessment of the risk of disease.</span> <abbr class="cit-jnl-abbrev">N. Engl. J. Med.</abbr> <span class="cit-vol">363</span>: <span class="cit-fpage">166</span>–<span class="cit-lpage">176</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1056/NEJMra0905980</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BEngland%2BJournal%2Bof%2BMedicine%26rft.stitle%253DNEJM%26rft.aulast%253DManolio%26rft.auinit1%253DT.%2BA.%26rft.volume%253D363%26rft.issue%253D2%26rft.spage%253D166%26rft.epage%253D176%26rft.atitle%253DGenomewide%2Bassociation%2Bstudies%2Band%2Bassessment%2Bof%2Bthe%2Brisk%2Bof%2Bdisease.%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJMra0905980%26rft_id%253Dinfo%253Apmid%252F20647212%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1056/NEJMra0905980&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20647212&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3567.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000279574100009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3567.29" data-doi="10.1038/nrg2796"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marchini</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Howie</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genotype imputation for genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">499</span>–<span class="cit-lpage">511</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg2796</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DMarchini%26rft.auinit1%253DJ.%26rft.volume%253D11%26rft.issue%253D7%26rft.spage%253D499%26rft.epage%253D511%26rft.atitle%253DGenotype%2Bimputation%2Bfor%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2796%26rft_id%253Dinfo%253Apmid%252F20517342%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2796&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20517342&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3567.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000278998500012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3567.30" data-doi="10.1038/ng.2824"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meijón</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Satbhai</span>  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsuchimatsu</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Busch</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genome-wide association study using cellular traits identifies a new regulator of root development in Arabidopsis.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">77</span>–<span class="cit-lpage">81</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.2824</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D46%26rft.spage%253D77%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2824%26rft_id%253Dinfo%253Apmid%252F24212884%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2824&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24212884&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3567.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3567.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Prediction of total genetic value using genome-wide dense marker maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">1819</span>–<span class="cit-lpage">1829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMeuwissen%26rft.auinit1%253DT.%2BH.%2BE.%26rft.volume%253D157%26rft.issue%253D4%26rft.spage%253D1819%26rft.epage%253D1829%26rft.atitle%253DPrediction%2Bof%2BTotal%2BGenetic%2BValue%2BUsing%2BGenome-Wide%2BDense%2BMarker%2BMaps%26rft_id%253Dinfo%253Apmid%252F11290733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ny80LzE4MTkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzExLzM1NjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-confproc" id="cit-8.11.3567.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morris</span>  <span class="cit-name-given-names">G. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramu</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deshpande</span>  <span class="cit-name-given-names">S. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hash</span>  <span class="cit-name-given-names">C. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shah</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <em>et al</em>., <span class="cit-pub-date">2013</span> Population genomic and genome-wide association studies of agroclimatic traits in sorghum. Proceedings of the National Academy of Sciences of the United States of America 110: <span class="cit-fpage">453</span>–<span class="cit-lpage">8</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3567.33" data-doi="10.2135/cropsci2008.02.0092"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">P. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coles</span>  <span class="cit-name-given-names">N. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holland</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bubeck</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Molecular characterization of maize inbreds with expired us plant variety protection.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">1673</span>–<span class="cit-lpage">1685</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2008.02.0092</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DNelson%26rft.auinit1%253DP.%2BT.%26rft.volume%253D48%26rft.issue%253D5%26rft.spage%253D1673%26rft.epage%253D1685%26rft.atitle%253DMolecular%2BCharacterization%2Bof%2BMaize%2BInbreds%2Bwith%2BExpired%2BU.S.%2BPlant%2BVariety%2BProtection%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2008.02.0092%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2008.02.0092&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000259792100003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3567.34" data-doi="10.1038/ng1847"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Price</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patterson</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plenge</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weinblatt</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shadick</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Principal components analysis corrects for stratification in genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">904</span>–<span class="cit-lpage">909</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng1847</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DTo%26rft.auinit1%253DM.%2BD.%26rft.volume%253D38%26rft.issue%253D8%26rft.spage%253D904%26rft.epage%253D909%26rft.atitle%253DPrincipal%2Bcomponents%2Banalysis%2Bcorrects%2Bfor%2Bstratification%2Bin%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1847%26rft_id%253Dinfo%253Apmid%252F16862161%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1847&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16862161&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3567.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000239325700019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3567.35" data-doi="10.1086/519795"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Purcell</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neale</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Todd-Brown</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferreira</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">PLINK: a tool set for whole-genome association and population-based linkage analyses.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">81</span>: <span class="cit-fpage">559</span>–<span class="cit-lpage">575</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1086/519795</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DPurcell%26rft.auinit1%253DS.%26rft.volume%253D81%26rft.issue%253D3%26rft.spage%253D559%26rft.epage%253D575%26rft.atitle%253DPLINK%253A%2Ba%2Btool%2Bset%2Bfor%2Bwhole-genome%2Bassociation%2Band%2Bpopulation-based%2Blinkage%2Banalyses.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F519795%26rft_id%253Dinfo%253Apmid%252F17701901%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/519795&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17701901&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3567.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3567.36" data-doi="10.1038/ng.2314"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Segura</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vilhjálmsson</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Platt</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Korte</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seren</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">An efficient multi-locus mixed-model approach for genome-wide association studies in structured populations.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">825</span>–<span class="cit-lpage">830</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.2314</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DSegura%26rft.auinit1%253DV.%26rft.volume%253D44%26rft.issue%253D7%26rft.spage%253D825%26rft.epage%253D830%26rft.atitle%253DAn%2Befficient%2Bmulti-locus%2Bmixed-model%2Bapproach%2Bfor%2Bgenome-wide%2Bassociation%2Bstudies%2Bin%2Bstructured%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2314%26rft_id%253Dinfo%253Apmid%252F22706313%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2314&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22706313&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3567.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3567.37" data-doi="10.1007/978-1-61779-785-9_6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Silva Lda</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">Z. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Composite interval mapping and multiple interval mapping: Procedures and guidelines for using windows QTL cartographer.</span> <abbr class="cit-jnl-abbrev">Methods Mol. Biol.</abbr> <span class="cit-vol">871</span>: <span class="cit-fpage">75</span>–<span class="cit-lpage">119</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/978-1-61779-785-9_6</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Bmolecular%2Bbiology%2B%2528Clifton%252C%2BN.J.%2529%26rft.stitle%253DMethods%2BMol%2BBiol%26rft.aulast%253DSilva%26rft.auinit1%253DL.%2Bd.%2Ba.%2B%2B.%2BC.%26rft.volume%253D871%26rft.spage%253D75%26rft.epage%253D119%26rft.atitle%253DComposite%2Binterval%2Bmapping%2Band%2Bmultiple%2Binterval%2Bmapping%253A%2Bprocedures%2Band%2Bguidelines%2Bfor%2Busing%2BWindows%2BQTL%2BCartographer.%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-61779-785-9_6%26rft_id%253Dinfo%253Apmid%252F22565834%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-1-61779-785-9_6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22565834&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3567.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3567.38" data-doi="10.1038/nature05616"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sladek</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rocheleau</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rung</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dina</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A genome-wide association study identifies novel risk loci for type 2 diabetes.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">445</span>: <span class="cit-fpage">881</span>–<span class="cit-lpage">885</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature05616</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DSladek%26rft.auinit1%253DR.%26rft.volume%253D445%26rft.issue%253D7130%26rft.spage%253D881%26rft.epage%253D885%26rft.atitle%253DA%2Bgenome-wide%2Bassociation%2Bstudy%2Bidentifies%2Bnovel%2Brisk%2Bloci%2Bfor%2Btype%2B2%2Bdiabetes.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature05616%26rft_id%253Dinfo%253Apmid%252F17293876%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature05616&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17293876&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3567.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000244341200041&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3567.39" data-doi="10.1534/genetics.107.079707"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stich</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Möhring</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Piepho</span>  <span class="cit-name-given-names">H.-P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heckenberger</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buckler</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Comparison of mixed-model approaches for association mapping.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">178</span>: <span class="cit-fpage">1745</span>–<span class="cit-lpage">1754</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.107.079707</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.079707%26rft_id%253Dinfo%253Apmid%252F18245847%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3OC8zLzE3NDUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzExLzM1NjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3567.40" data-doi="10.1038/ng.746"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tian</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bradbury</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hung</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome-wide association study of leaf architecture in the maize nested association mapping population.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">159</span>–<span class="cit-lpage">162</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.746</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DTian%26rft.auinit1%253DF.%26rft.volume%253D43%26rft.issue%253D2%26rft.spage%253D159%26rft.epage%253D162%26rft.atitle%253DGenome-wide%2Bassociation%2Bstudy%2Bof%2Bleaf%2Barchitecture%2Bin%2Bthe%2Bmaize%2Bnested%2Bassociation%2Bmapping%2Bpopulation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.746%26rft_id%253Dinfo%253Apmid%252F21217756%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.746&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21217756&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3567.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000286623800016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3567.41" data-doi="10.1016/j.ajhg.2011.11.029"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Visscher</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCarthy</span>  <span class="cit-name-given-names">M. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Five Years of GWAS Discovery.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">90</span>: <span class="cit-fpage">7</span>–<span class="cit-lpage">24</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ajhg.2011.11.029</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DVisscher%26rft.auinit1%253DP.%2BM.%26rft.volume%253D90%26rft.issue%253D1%26rft.spage%253D7%26rft.epage%253D24%26rft.atitle%253DFive%2Byears%2Bof%2BGWAS%2Bdiscovery.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2011.11.029%26rft_id%253Dinfo%253Apmid%252F22243964%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2011.11.029&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22243964&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3567.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3567.42" data-doi="10.1038/ng.2213"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferreira</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morris</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Medland</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Madden</span>  <span class="cit-name-given-names">P. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Conditional and joint multiple-SNP analysis of GWAS summary statistics identifies additional variants influencing complex traits.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">369</span>–<span class="cit-lpage">375</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.2213</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D44%26rft.spage%253D369%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2213%26rft_id%253Dinfo%253Apmid%252F22426310%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2213&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22426310&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3567.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3567.43" data-doi="10.1111/tpj.13029"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yeh</span>  <span class="cit-name-given-names">C.-T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jeddeloh</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Extreme-phenotype genome-wide association study (xp-gwas): a method for identifying trait-associated variants by sequencing pools of individuals selected from a diversity panel.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">84</span>: <span class="cit-fpage">587</span>–<span class="cit-lpage">596</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/tpj.13029</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BJ.%26rft.volume%253D84%26rft.spage%253D587%26rft_id%253Dinfo%253Adoi%252F10.1111%252Ftpj.13029%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/tpj.13029&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3567.44" data-doi="10.1038/ejhg.2011.39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weedon</span>  <span class="cit-name-given-names">M. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Purcell</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lettre</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Estrada</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genomic inflation factors under polygenic inheritance.</span> <abbr class="cit-jnl-abbrev">Eur. J. Hum. Genet.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">807</span>–<span class="cit-lpage">812</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ejhg.2011.39</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuropean%2Bjournal%2Bof%2Bhuman%2Bgenetics%2B%253A%2B%2BEJHG%26rft.stitle%253DEur%2BJ%2BHum%2BGenet%26rft.aulast%253DYang%26rft.auinit1%253DJ.%26rft.volume%253D19%26rft.issue%253D7%26rft.spage%253D807%26rft.epage%253D812%26rft.atitle%253DGenomic%2Binflation%2Bfactors%2Bunder%2Bpolygenic%2Binheritance.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fejhg.2011.39%26rft_id%253Dinfo%253Apmid%252F21407268%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ejhg.2011.39&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21407268&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3567.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3567.45" data-doi="10.1534/genetics.107.074245"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holland</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McMullen</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buckler</span>  <span class="cit-name-given-names">E. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Genetic design and statistical power of nested association mapping in maize.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">178</span>: <span class="cit-fpage">539</span>–<span class="cit-lpage">551</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.107.074245</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.074245%26rft_id%253Dinfo%253Apmid%252F18202393%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTc4LzEvNTM5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMS8zNTY3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3567.46" data-doi="10.1038/ng1702"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pressoir</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Briggs</span>  <span class="cit-name-given-names">W. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vroh Bi</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yamasaki</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A unified mixed-model method for association mapping that accounts for multiple levels of relatedness.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">203</span>–<span class="cit-lpage">208</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng1702</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DYu%26rft.auinit1%253DJ.%26rft.volume%253D38%26rft.issue%253D2%26rft.spage%253D203%26rft.epage%253D208%26rft.atitle%253DA%2Bunified%2Bmixed-model%2Bmethod%2Bfor%2Bassociation%2Bmapping%2Bthat%2Baccounts%2Bfor%2Bmultiple%2Blevels%2Bof%2Brelatedness.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1702%26rft_id%253Dinfo%253Apmid%252F16380716%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1702&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16380716&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3567.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000234953200015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3567.47" data-doi="10.1073/pnas.90.23.10972"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">Z. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Theoretical basis for separation of multiple linked gene effects in mapping quantitative trait loci.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">90</span>: <span class="cit-fpage">10972</span>–<span class="cit-lpage">10976</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.90.23.10972</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DZeng%26rft.auinit1%253DZ.%2BB.%26rft.volume%253D90%26rft.issue%253D23%26rft.spage%253D10972%26rft.epage%253D10976%26rft.atitle%253DTheoretical%2Bbasis%2Bfor%2Bseparation%2Bof%2Bmultiple%2Blinked%2Bgene%2Beffects%2Bin%2Bmapping%2Bquantitative%2Btrait%2Bloci.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.90.23.10972%26rft_id%253Dinfo%253Apmid%252F8248199%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiOTAvMjMvMTA5NzIiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzExLzM1NjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/8/11/3567.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/11/3559" title="Discovery of QTL Alleles for Grain Shape in the Japan-MAGIC Rice Population Using Haplotype Information" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/11/3577" title="Estimating Asian Contribution to the Brazilian Population: A New Application of a Validated Set of 61 Ancestry Informative Markers" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/8/11"><span>Volume 8 Issue 11, November 2018</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/8/11.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/8/11.cover-source.jpg?itok=lFO8v6bg" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 8 (11)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/11/3567.abstract" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-RtmpUg94mOQoWTKnweoND0PA3HJQPd7u0MZXShsdttQ" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/multiparental-populations" class="" data-icon-position="" data-hide-link-title="0">Multiparental Populations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/8/11/3567" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/8/11/3567" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/8/11/3567" class="active" data-icon-position="" data-hide-link-title="0">Empirical Comparisons of Different Statistical Models To Identify and Validate Kernel Row Number-Associated Variants from Structured Multi-parent Mapping Populations of Maize</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/491245" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-t2EbMIjqW-NvIx2hvxPg3ntumEtC94H_ZzqAza2qXmg" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/11/3567.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/8/11/3567" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-swo4-HsKCJ8ZV2CxoJutJmOzkMRaTkfy9fBNJ7PQwMY" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/8/11/3567" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/11/3567.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="491245" id="node491245--2" data-pisa="ggg;8/11/3567" data-pisa-master="ggg;g3.118.200636" data-apath="/ggg/8/11/3567.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Empirical Comparisons of Different Statistical Models To Identify and Validate Kernel Row Number-Associated Variants from Structured Multi-parent Mapping Populations of Maize</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-0999-3518" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jinliang Yang</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-1392-2018" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Cheng-Ting “Eddy” Yeh</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-7804-712X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Raghuprakash Kastoori Ramamurthy</span>, <span class="highwire-citation-author" data-delta="3">Xinshuai Qi</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0001-5821-099X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Rohan L. Fernando</span>, <span class="highwire-citation-author" data-delta="5">Jack C. M. Dekkers</span>, <span class="highwire-citation-author" data-delta="6">Dorian J. Garrick</span>, <span class="highwire-citation-author" data-delta="7">Dan Nettleton</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="8"><a href="http://orcid.org/0000-0001-9169-5204" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Patrick S. Schnable</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3567-3575;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200636 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node491245--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jinliang Yang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Agronomy and Horticulture, University of Nebraska-Lincoln, Lincoln, NE 68583</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jinliang%2BYang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yang%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJinliang%2BYang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-0999-3518" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jinliang Yang</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Cheng-Ting “Eddy” Yeh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Cheng-Ting%2B%E2%80%9CEddy%E2%80%9D%2BYeh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yeh%20CE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACheng-Ting%2B%25E2%2580%259CEddy%25E2%2580%259D%2BYeh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-1392-2018" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Cheng-Ting “Eddy” Yeh</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Raghuprakash Kastoori Ramamurthy </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Agronomy and Horticulture, University of Nebraska-Lincoln, Lincoln, NE 68583</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Raghuprakash%2BKastoori%2BRamamurthy%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ramamurthy%20RK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARaghuprakash%2BKastoori%2BRamamurthy%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7804-712X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Raghuprakash Kastoori Ramamurthy</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Xinshuai Qi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Ecology and Evolutionary Biology, University of Arizona, Tucson, AZ </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xinshuai%2BQi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Qi%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXinshuai%2BQi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Rohan L. Fernando </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Animal Science and Center for Integrated Animal Genomics</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rohan%2BL.%2BFernando%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fernando%20RL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARohan%2BL.%2BFernando%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5821-099X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Rohan L. Fernando</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Jack C. M. Dekkers </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Animal Science and Center for Integrated Animal Genomics</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jack%2BC.%2BM.%2BDekkers%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dekkers%20JC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJack%2BC.%2BM.%2BDekkers%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Dorian J. Garrick </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Animal Science and Center for Integrated Animal Genomics</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dorian%2BJ.%2BGarrick%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Garrick%20DJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADorian%2BJ.%2BGarrick%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Dan Nettleton </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Statistics, Iowa State University, Ames, IA 50011</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dan%2BNettleton%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nettleton%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADan%2BNettleton%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Patrick S. Schnable </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Patrick%2BS.%2BSchnable%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schnable%20PS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APatrick%2BS.%2BSchnable%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-9169-5204" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Patrick S. Schnable</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:schnable@iastate.edu" class="" data-icon-position="" data-hide-link-title="0">schnable@iastate.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/8/11/3567" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/8/11/3567&amp;title=Empirical%20Comparisons%20of%20Different%20Statistical%20Models%20To%20Identify%20and%20Validate%20Kernel%20Row%20Number-Associated%20Variants%20from%20Structured%20Multi-parent%20Mapping%20Populations%20of%20Maize" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/8/11/3567&amp;title=Empirical%20Comparisons%20of%20Different%20Statistical%20Models%20To%20Identify%20and%20Validate%20Kernel%20Row%20Number-Associated%20Variants%20from%20Structured%20Multi-parent%20Mapping%20Populations%20of%20Maize" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/8/11/3567&amp;title=Empirical%20Comparisons%20of%20Different%20Statistical%20Models%20To%20Identify%20and%20Validate%20Kernel%20Row%20Number-Associated%20Variants%20from%20Structured%20Multi-parent%20Mapping%20Populations%20of%20Maize" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/8/11/3567&amp;text=Empirical%20Comparisons%20of%20Different%20Statistical%20Models%20To%20Identify%20and%20Validate%20Kernel%20Row%20Number-Associated%20Variants%20from%20Structured%20Multi-parent%20Mapping%20Populations%20of%20Maize" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/8/11/3567&amp;title=Empirical%20Comparisons%20of%20Different%20Statistical%20Models%20To%20Identify%20and%20Validate%20Kernel%20Row%20Number-Associated%20Variants%20from%20Structured%20Multi-parent%20Mapping%20Populations%20of%20Maize" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/8/11/3567&amp;t=Empirical%20Comparisons%20of%20Different%20Statistical%20Models%20To%20Identify%20and%20Validate%20Kernel%20Row%20Number-Associated%20Variants%20from%20Structured%20Multi-parent%20Mapping%20Populations%20of%20Maize" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/8/11/3567&amp;title=Empirical%20Comparisons%20of%20Different%20Statistical%20Models%20To%20Identify%20and%20Validate%20Kernel%20Row%20Number-Associated%20Variants%20from%20Structured%20Multi-parent%20Mapping%20Populations%20of%20Maize" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/8/11/3567&amp;title=Empirical%20Comparisons%20of%20Different%20Statistical%20Models%20To%20Identify%20and%20Validate%20Kernel%20Row%20Number-Associated%20Variants%20from%20Structured%20Multi-parent%20Mapping%20Populations%20of%20Maize" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/491245/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="491245" id="node491245--3" data-pisa="ggg;8/11/3567" data-pisa-master="ggg;g3.118.200636" data-apath="/ggg/8/11/3567.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Empirical Comparisons of Different Statistical Models To Identify and Validate Kernel Row Number-Associated Variants from Structured Multi-parent Mapping Populations of Maize</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-0999-3518" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jinliang Yang</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-1392-2018" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Cheng-Ting “Eddy” Yeh</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-7804-712X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Raghuprakash Kastoori Ramamurthy</span>, <span class="highwire-citation-author" data-delta="3">Xinshuai Qi</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0001-5821-099X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Rohan L. Fernando</span>, <span class="highwire-citation-author" data-delta="5">Jack C. M. Dekkers</span>, <span class="highwire-citation-author" data-delta="6">Dorian J. Garrick</span>, <span class="highwire-citation-author" data-delta="7">Dan Nettleton</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="8"><a href="http://orcid.org/0000-0001-9169-5204" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Patrick S. Schnable</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3567-3575;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200636 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node491245--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jinliang Yang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Agronomy and Horticulture, University of Nebraska-Lincoln, Lincoln, NE 68583</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jinliang%2BYang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yang%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJinliang%2BYang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-0999-3518" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jinliang Yang</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Cheng-Ting “Eddy” Yeh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Cheng-Ting%2B%E2%80%9CEddy%E2%80%9D%2BYeh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yeh%20CE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACheng-Ting%2B%25E2%2580%259CEddy%25E2%2580%259D%2BYeh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-1392-2018" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Cheng-Ting “Eddy” Yeh</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Raghuprakash Kastoori Ramamurthy </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Agronomy and Horticulture, University of Nebraska-Lincoln, Lincoln, NE 68583</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Raghuprakash%2BKastoori%2BRamamurthy%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ramamurthy%20RK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARaghuprakash%2BKastoori%2BRamamurthy%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7804-712X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Raghuprakash Kastoori Ramamurthy</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Xinshuai Qi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Ecology and Evolutionary Biology, University of Arizona, Tucson, AZ </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xinshuai%2BQi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Qi%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXinshuai%2BQi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Rohan L. Fernando </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Animal Science and Center for Integrated Animal Genomics</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rohan%2BL.%2BFernando%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fernando%20RL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARohan%2BL.%2BFernando%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5821-099X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Rohan L. Fernando</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Jack C. M. Dekkers </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Animal Science and Center for Integrated Animal Genomics</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jack%2BC.%2BM.%2BDekkers%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dekkers%20JC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJack%2BC.%2BM.%2BDekkers%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Dorian J. Garrick </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Animal Science and Center for Integrated Animal Genomics</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dorian%2BJ.%2BGarrick%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Garrick%20DJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADorian%2BJ.%2BGarrick%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Dan Nettleton </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Statistics, Iowa State University, Ames, IA 50011</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dan%2BNettleton%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nettleton%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADan%2BNettleton%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Patrick S. Schnable </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Patrick%2BS.%2BSchnable%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schnable%20PS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APatrick%2BS.%2BSchnable%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-9169-5204" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Patrick S. Schnable</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:schnable@iastate.edu" class="" data-icon-position="" data-hide-link-title="0">schnable@iastate.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3567&amp;rft.epage=3575&amp;rft.atitle=Empirical%20Comparisons%20of%20Different%20Statistical%20Models%20To%20Identify%20and%20Validate%20Kernel%20Row%20Number-Associated%20Variants%20from%20Structured%20Multi-parent%20Mapping%20Populations%20of%20Maize&amp;rft.volume=8&amp;rft.issue=11&amp;rft.date=2018-11-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Yang%2C+Jinliang&amp;rft.au=Yeh%2C+Cheng-Ting+%E2%80%9CEddy%E2%80%9D&amp;rft.au=Ramamurthy%2C+Raghuprakash+Kastoori&amp;rft.au=Qi%2C+Xinshuai&amp;rft.au=Fernando%2C+Rohan+L.&amp;rft.au=Dekkers%2C+Jack+C.+M.&amp;rft.au=Garrick%2C+Dorian+J.&amp;rft.au=Nettleton%2C+Dan&amp;rft.au=Schnable%2C+Patrick+S."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/491245/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/491245/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/491245/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/491245/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/491245/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/491245/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/491245/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/491245/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/491245/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/491245/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/491245/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/491245/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495847" id="node495847" data-pisa="ggg;10/3/951" data-pisa-master="ggg;g3.119.400763" data-apath="/ggg/10/3/951.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/951" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Genome-Wide Association Study in Two Cohorts from a Multi-generational Mouse Advanced Intercross Line Highlights the Difficulty of Replication Due to Study-Specific Heterogeneity</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494983" id="node494983" data-pisa="ggg;9/12/4045" data-pisa-master="ggg;g3.119.400353" data-apath="/ggg/9/12/4045.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/9/12/4045" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Genetic Architecture of Chilling Tolerance in Sorghum Dissected with a Nested Association Mapping Population</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493460" id="node493460" data-pisa="ggg;9/7/2317" data-pisa-master="ggg;g3.119.400098" data-apath="/ggg/9/7/2317.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/9/7/2317" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Testing Pleiotropy <em>vs.</em> Separate QTL in Multiparental Populations</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Multiparental%20Populations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Multiparental Populations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/8/11/3567" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-13" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-19" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/8/11/3567.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/8/11/3567.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/8/11/3567.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
