<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/8/11/3637" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/8/11/3637.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/8/11/3637.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/8/11/3637.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/8/11/3637.atom" />
<meta name="HW.pisa" content="ggg;8/11/3637" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="HO Endonuclease-Initiated Recombination in Yeast Meiosis Fails To Promote Homologous Centromere Pairing and Is Not Constrained To Utilize the Dmc1 Recombinase" />
<meta name="DC.Identifier" content="10.1534/g3.118.200641" />
<meta name="DC.Date" content="2018-11-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2018 Yisehak, MacQueen. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Crossover recombination during meiosis is accompanied by a dramatic chromosome reorganization. In Saccharomyces cerevisiae , the onset of meiotic recombination by the [Spo11][1] transesterase leads to stable pairwise associations between previously unassociated homologous centromeres followed by the intimate alignment of homologous axes via synaptonemal complex (SC) assembly. However, the molecular relationship between recombination and global meiotic chromosome reorganization remains poorly understood. In budding yeast, one question is why SC assembly initiates earliest at centromere regions while the DNA double strand breaks (DSBs) that initiate recombination occur genome-wide. We targeted the site-specific [HO][2] endonuclease to various positions on S. cerevisiae ’s longest chromosome in order to ask whether a meiotic DSB’s proximity to the centromere influences its capacity to promote homologous centromere pairing and SC assembly. We show that repair of an [HO][2]-mediated DSB does not promote homologous centromere pairing nor any extent of SC assembly in [spo11][1] meiotic nuclei, regardless of its proximity to the centromere. DSBs induced en masse by phleomycin exposure likewise do not promote homologous centromere pairing nor robust SC assembly. Interestingly, in contrast to [Spo11][1], [HO][2]-initiated interhomolog recombination is not affected by loss of the meiotic kinase, [Mek1][3], and is not constrained to use the meiosis-specific [Dmc1][4] recombinase. These results strengthen the previously proposed idea that (at least some) [Spo11][1] DSBs may be specialized in activating mechanisms that both 1) reinforce homologous chromosome alignment via homologous centromere pairing and SC assembly, and 2) establish [Dmc1][4] as the primary strand exchange enzyme.

 [1]: http://www.yeastgenome.org/locus/S000001014/overview
 [2]: http://www.yeastgenome.org/locus/S000002386/overview
 [3]: http://www.yeastgenome.org/locus/S000005878/overview
 [4]: http://www.yeastgenome.org/locus/S000000981/overview" />
<meta name="DC.Contributor" content="Lina Yisehak" />
<meta name="DC.Contributor" content="Amy J. MacQueen" />
<meta name="article:published_time" content="2018-11-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="HO Endonuclease-Initiated Recombination in Yeast Meiosis Fails To Promote Homologous Centromere Pairing and Is Not Constrained To Utilize the Dmc1 Recombinase" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Crossover recombination during meiosis is accompanied by a dramatic chromosome reorganization. In &lt;i&gt;Saccharomyces cerevisiae&lt;/i&gt;, the onset of meiotic recombination by the Spo11 transesterase leads to stable pairwise associations between previously unassociated homologous centromeres followed by the intimate alignment of homologous axes via synaptonemal complex (SC) assembly. However, the molecular relationship between recombination and global meiotic chromosome reorganization remains poorly understood. In budding yeast, one question is why SC assembly initiates earliest at centromere regions while the DNA double strand breaks (DSBs) that initiate recombination occur genome-wide. We targeted the site-specific HO endonuclease to various positions on &lt;i&gt;S. cerevisiae&lt;/i&gt;’s longest chromosome in order to ask whether a meiotic DSB’s proximity to the centromere influences its capacity to promote homologous centromere pairing and SC assembly. We show that repair of an HO-mediated DSB does not promote homologous centromere pairing nor any extent of SC assembly in &lt;i&gt;spo11&lt;/i&gt; meiotic nuclei, regardless of its proximity to the centromere. DSBs induced &lt;i&gt;en masse&lt;/i&gt; by phleomycin exposure likewise do not promote homologous centromere pairing nor robust SC assembly. Interestingly, in contrast to Spo11, HO-initiated interhomolog recombination is not affected by loss of the meiotic kinase, Mek1, and is not constrained to use the meiosis-specific Dmc1 recombinase. These results strengthen the previously proposed idea that (at least some) Spo11 DSBs may be specialized in activating mechanisms that both 1) reinforce homologous chromosome alignment via homologous centromere pairing and SC assembly, and 2) establish Dmc1 as the primary strand exchange enzyme.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2018/11/01" />
<meta name="citation_mjid" content="ggg;8/11/3637" />
<meta name="citation_id" content="8/11/3637" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/8/11/3637" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/8/11/3637.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/8/11/3637.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/8/11/3637.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.118.200641" />
<meta name="citation_pmid" content="30254180" />
<meta name="citation_volume" content="8" />
<meta name="citation_issue" content="11" />
<meta name="citation_num_pages" content="23" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="3637" />
<meta name="citation_lastpage" content="3659" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Lina Yisehak" />
<meta name="citation_author_institution" content="Department of Molecular Biology and Biochemistry, Wesleyan University, Middletown, CT" />
<meta name="citation_author" content="Amy J. MacQueen" />
<meta name="citation_author_institution" content="Department of Molecular Biology and Biochemistry, Wesleyan University, Middletown, CT" />
<meta name="citation_author_email" content="amacqueen@wesleyan.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-0919-7538" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=S. Agarwal;citation_author=G. S. Roeder;citation_title=Zip3 provides a link between recombination enzymes and synaptonemal complex proteins.;citation_pages=245-255;citation_volume=102;citation_year=2000;citation_issue=2;citation_pmid=10943844;citation_doi=10.1016/S0092-8674(00)00029-5" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. Ajimura;citation_author=S. H. Leem;citation_author=H. Ogawa;citation_title=Identification of new genes required for meiotic recombination in Saccharomyces cerevisiae.;citation_pages=51-66;citation_volume=133;citation_year=1993;citation_issue=1;citation_pmid=8417989" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=E. Alani;citation_author=R. Padmore;citation_author=N. Kleckner;citation_title=Analysis of wild-type and rad50 mutants of yeast suggests an intimate relationship between meiotic chromosome synapsis and recombination.;citation_pages=419-436;citation_volume=61;citation_year=1990;citation_issue=3;citation_pmid=2185891;citation_doi=10.1016/0092-8674(90)90524-I" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=T. Allers;citation_author=M. Lichten;citation_title=Differential timing and control of noncrossover and crossover recombination during meiosis.;citation_pages=47-57;citation_volume=106;citation_year=2001;citation_issue=1;citation_pmid=11461701;citation_doi=10.1016/S0092-8674(01)00416-0" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=T. Allers;citation_author=M. Lichten;citation_title=Intermediates of yeast meiotic recombination contain heteroduplex DNA.;citation_pages=225-231;citation_volume=8;citation_year=2001;citation_issue=1;citation_pmid=11511375;citation_doi=10.1016/S1097-2765(01)00280-5" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=J. Bachant;citation_author=A. Alcasabas;citation_author=Y. Blat;citation_author=N. Kleckner;citation_author=S. J. Elledge;citation_title=The SUMO-1 isopeptidase Smt4 is linked to centromeric cohesion through SUMO-1 modification of DNA topoisomerase II.;citation_pages=1169-1182;citation_volume=9;citation_year=2002;citation_issue=6;citation_pmid=12086615;citation_doi=10.1016/S1097-2765(02)00543-9" />
<meta name="citation_reference" content="citation_journal_title=Genes to Cells;citation_journal_abbrev=GENES CELLS;citation_author=D. K. Bishop;citation_author=Y. Nikolski;citation_author=J. Oshiro;citation_author=J. Chon;citation_author=M. Shinohara;citation_title=High copy number suppression of the meiotic arrest caused by a dmc1 mutation: REC114 imposes an early recombination block and RAD54 promotes a DMC1-independent DSB repair pathway;citation_pages=425-444;citation_volume=4;citation_year=1999;citation_issue=8;citation_pmid=10526232;citation_doi=10.1046/j.1365-2443.1999.00273.x" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=D. K. Bishop;citation_author=D. Park;citation_author=L. Xu;citation_author=N. Kleckner;citation_title=DMC1: a meiosis-specific yeast homolog of E. coli recA required for recombination, synaptonemal complex formation, and cell cycle progression.;citation_pages=439-456;citation_volume=69;citation_year=1992;citation_issue=3;citation_pmid=1581960;citation_doi=10.1016/0092-8674(92)90446-J" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=Y. Blat;citation_author=R. U. Protacio;citation_author=N. Hunter;citation_author=N. Kleckner;citation_title=Physical and functional interactions among basic chromosome organizational features govern early steps of meiotic chiasma formation.;citation_pages=791-802;citation_volume=111;citation_year=2002;citation_issue=6;citation_pmid=12526806;citation_doi=10.1016/S0092-8674(02)01167-4" />
<meta name="citation_reference" content="citation_journal_title=Current biology :  CB;citation_journal_abbrev=Curr Biol;citation_author=H. G. Blitzblau;citation_author=G. W. Bell;citation_author=J. Rodriguez;citation_author=S. P. Bell;citation_author=A. Hochwagen;citation_title=Mapping of meiotic single-stranded DNA reveals double-stranded-break hotspots near centromeres and telomeres.;citation_pages=2003-2012;citation_volume=17;citation_year=2007;citation_issue=23;citation_pmid=18060788;citation_doi=10.1016/j.cub.2007.10.066" />
<meta name="citation_reference" content="citation_journal_title=Chromosome research :  an international journal on the molecular, supramolecular and evolutionary aspects of chromosome biology;citation_journal_abbrev=Chromosome Res;citation_author=V. Borde;citation_title=The multiple roles of the Mre11 complex for meiotic recombination.;citation_pages=551-563;citation_volume=15;citation_year=2007;citation_issue=5;citation_pmid=17674145;citation_doi=10.1007/s10577-007-1147-9" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=G. V. Börner;citation_author=N. Kleckner;citation_author=N. Hunter;citation_title=Crossover/noncrossover differentiation, synaptonemal complex formation, and regulatory surveillance at the leptotene/zygotene transition of meiosis.;citation_pages=29-45;citation_volume=117;citation_year=2004;citation_issue=1;citation_pmid=15066280;citation_doi=10.1016/S0092-8674(04)00292-2" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Cell;citation_author=G. A. Brar;citation_author=A. Hochwagen;citation_author=L. S. Ee;citation_author=A. Amon;citation_title=The multiple roles of cohesin in meiotic chromosome morphogenesis and pairing.;citation_volume=20;citation_year=2009;citation_pmid=19073884;citation_doi=10.1091/mbc.E08-06-0637" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=D. A. Bressan;citation_author=B. K. Baxter;citation_author=J. H. J. Petrini;citation_title=The Mre11-Rad50-Xrs2 Protein Complex Facilitates Homologous Recombination-Based Double-Strand Break Repair inSaccharomyces cerevisiae;citation_pages=7681-7687;citation_volume=19;citation_year=1999;citation_issue=11;citation_pmid=10523656;citation_doi=10.1128/MCB.19.11.7681" />
<meta name="citation_reference" content="citation_journal_title=Cold Spring Harb. Perspect. Biol.;citation_author=M. S. Brown;citation_author=D. K. Bishop;citation_title=DNA strand exchange and RecA homologs in meiosis.;citation_volume=7;citation_year=2014;citation_pmid=25475089;citation_doi=10.1101/cshperspect.a016659" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=V. Busygina;citation_author=M. G. Sehorn;citation_author=I. Y. Shi;citation_author=H. Tsubouchi;citation_author=G. S. Roeder;citation_title=Hed1 regulates Rad51-mediated recombination via a novel mechanism;citation_pages=786-795;citation_volume=22;citation_year=2008;citation_issue=6;citation_pmid=18347097;citation_doi=10.1101/gad.1638708" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=M. Bzymek;citation_author=N. H. Thayer;citation_author=S. D. Oh;citation_author=N. Kleckner;citation_author=N. Hunter;citation_title=Double Holliday junctions are intermediates of DNA break repair.;citation_pages=937-941;citation_volume=464;citation_year=2010;citation_issue=7290;citation_pmid=20348905;citation_doi=10.1038/nature08868" />
<meta name="citation_reference" content="citation_journal_title=Nat. Struct. Mol. Biol.;citation_author=C. K. Cahoon;citation_author=R. S. Hawley;citation_title=Regulating the construction and demolition of the synaptonemal complex.;citation_pages=369-377;citation_volume=23;citation_year=2016;citation_doi=10.1038/nsmb.3208" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=T. L. Callender;citation_author=R. Laureau;citation_author=L. Wan;citation_author=X. Chen;citation_author=R. Sandhu;citation_title=Mek1 Down Regulates Rad51 Activity during Yeast Meiosis by Phosphorylation of Hed1.;citation_pages=e1006226;citation_volume=12;citation_year=2016;citation_pmid=27483004;citation_doi=10.1371/journal.pgen.1006226" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=J. A. Carballo;citation_author=A. L. Johnson;citation_author=S. G. Sedgwick;citation_author=R. S. Cha;citation_title=Phosphorylation of the axial element protein Hop1 by Mec1/Tel1 ensures meiotic interhomolog recombination.;citation_pages=758-770;citation_volume=132;citation_year=2008;citation_issue=5;citation_pmid=18329363;citation_doi=10.1016/j.cell.2008.01.035" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell. Biol.;citation_author=H. Cartagena-Lirola;citation_author=I. Guerini;citation_author=N. Manfrini;citation_author=G. Lucchini;citation_author=M. P. Longhese;citation_title=Role of the Saccharomyces cerevisiae Rad53 checkpoint kinase in signaling double-strand breaks during the meiotic cell cycle.;citation_volume=28;citation_year=2008;citation_pmid=18505828;citation_doi=10.1128/MCB.00375-08" />
<meta name="citation_reference" content="citation_journal_title=Cell cycle (Georgetown, Tex.);citation_journal_abbrev=Cell Cycle;citation_author=H. Cartagena-Lirola;citation_author=I. Guerini;citation_author=V. Viscardi;citation_author=G. Lucchini;citation_author=M. P. Longhese;citation_title=Budding Yeast Sae2 is an In Vivo Target of the Mec1 and Tel1 Checkpoint Kinases During Meiosis.;citation_pages=1549-1559;citation_volume=5;citation_year=2006;citation_issue=14;citation_pmid=16861895;citation_doi=10.4161/cc.5.14.2916" />
<meta name="citation_reference" content="citation_journal_title=Developmental cell;citation_journal_abbrev=Dev Cell;citation_author=S. Y. Chen;citation_author=T. Tsubouchi;citation_author=B. Rockmill;citation_author=J. S. Sandler;citation_author=D. R. Richards;citation_title=Global analysis of the meiotic crossover landscape.;citation_pages=401-415;citation_volume=15;citation_year=2008;citation_issue=3;citation_pmid=18691940;citation_doi=10.1016/j.devcel.2008.07.006" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=P. R. Chua;citation_author=G. S. Roeder;citation_title=Zip2, a meiosis-specific protein required for the initiation of chromosome synapsis.;citation_pages=349-359;citation_volume=93;citation_year=1998;citation_issue=3;citation_pmid=9590170;citation_doi=10.1016/S0092-8674(00)81164-2" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=V. Cloud;citation_author=Y.-L. Chan;citation_author=J. Grubb;citation_author=B. Budke;citation_author=D. K. Bishop;citation_title=Rad51 Is an Accessory Factor for Dmc1-Mediated Joint Molecule Formation During Meiosis;citation_pages=1222-1225;citation_volume=337;citation_year=2012;citation_issue=6099;citation_pmid=22955832;citation_doi=10.1126/science.1219379" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. de los Santos;citation_author=N. Hunter;citation_author=C. Lee;citation_author=B. Larkin;citation_author=J. Loidl;citation_title=The Mus81/Mms4 Endonuclease Acts Independently of Double-Holliday Junction Resolution to Promote a Distinct Subset of Crossovers During Meiosis in Budding Yeast;citation_pages=81-94;citation_volume=164;citation_year=2003;citation_issue=1;citation_pmid=12750322" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=A. De Muyt;citation_author=L. Jessop;citation_author=E. Kolar;citation_author=A. Sourirajan;citation_author=J. Chen;citation_title=BLM helicase ortholog Sgs1 is a central regulator of meiotic recombination intermediate metabolism.;citation_pages=43-53;citation_volume=46;citation_year=2012;citation_issue=1;citation_pmid=22500736;citation_doi=10.1016/j.molcel.2012.02.020" />
<meta name="citation_reference" content="citation_journal_title=J. Cell Biol.;citation_author=H. Dong;citation_author=G. S. Roeder;citation_title=Organization of the Yeast Zip1 Protein within the Central Region of the Synaptonemal Complex.;citation_volume=148;citation_year=2000;citation_pmid=10662769;citation_doi=10.1083/jcb.148.3.417" />
<meta name="citation_reference" content="citation_journal_title=Developmental cell;citation_journal_abbrev=Dev Cell;citation_author=J. E. Falk;citation_author=A. C. Chan;citation_author=E. Hoffmann;citation_author=A. Hochwagen;citation_title=A Mec1- and PP4-dependent checkpoint couples centromere pairing to meiotic recombination.;citation_pages=599-611;citation_volume=19;citation_year=2010;citation_issue=4;citation_pmid=20951350;citation_doi=10.1016/j.devcel.2010.09.006" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. A. Farah;citation_author=G. Cromie;citation_author=L. Davis;citation_author=W. W. Steiner;citation_author=G. R. Smith;citation_title=Activation of an Alternative, Rec12 (Spo11)-Independent Pathway of Fission Yeast Meiotic Recombination in the Absence of a DNA Flap Endonuclease.;citation_volume=171;citation_year=2005;citation_pmid=16118186;citation_doi=10.1534/genetics.105.046821" />
<meta name="citation_reference" content="citation_journal_title=Genes Dev.;citation_author=W. M. Fricke;citation_author=S. J. Brill;citation_title=Slx1-Slx4 is a second structure-specific endonuclease functionally redundant with Sgs1-Top3.;citation_volume=17;citation_year=2003;citation_pmid=12832395;citation_doi=10.1101/gad.1105203" />
<meta name="citation_reference" content="citation_journal_title=Mutat. Res. Fundam. Mol. Mech. Mutagen.;citation_author=J. C. Game;citation_author=R. K. Mortimer;citation_title=A genetic study of X-ray sensitive mutants in yeast.;citation_pages=281-292;citation_volume=24;citation_year=1974;citation_doi=10.1016/0027-5107(74)90176-6" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. C. Game;citation_author=T. J. Zamb;citation_author=R. J. Braun;citation_author=M. Resnick;citation_author=R. M. Roth;citation_title=THE ROLE OF RADIATION (rad) GENES IN MEIOTIC RECOMBINATION IN YEAST;citation_pages=51-68;citation_volume=94;citation_year=1980;citation_issue=1;citation_pmid=17248996" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=J. L. Gerton;citation_author=J. DeRisi;citation_author=R. Shroff;citation_author=M. Lichten;citation_author=P. O. Brown;citation_title=Global mapping of meiotic recombination hotspots and coldspots in the yeast Saccharomyces cerevisiae;citation_pages=11383-11390;citation_volume=97;citation_year=2000;citation_issue=21;citation_pmid=11027339;citation_doi=10.1073/pnas.97.21.11383" />
<meta name="citation_reference" content="citation_journal_title=Microb. Cell;citation_author=E. Gobbini;citation_author=C. Cassani;citation_author=M. Villa;citation_author=D. Bonetti;citation_author=M. P. Longhese;citation_title=Functions and regulation of the MRX complex at DNA double-strand breaks.;citation_pages=329-337;citation_volume=3;citation_year=2016;citation_doi=10.15698/mic2016.08.517" />
<meta name="citation_reference" content="citation_journal_title=Yeast (Chichester, England);citation_journal_abbrev=Yeast;citation_author=A. L. Goldstein;citation_author=J. H. McCusker;citation_title=Three new dominant drug resistance cassettes for gene disruption in Saccharomyces cerevisiae.;citation_pages=1541-1553;citation_volume=15;citation_year=1999;citation_issue=14;citation_pmid=10514571;citation_doi=10.1002/(SICI)1097-0061(199910)15:14&lt;1541::AID-YEA476&gt;3.0.CO;2-K" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. E. Haber;citation_title=Mating-Type Genes and MAT Switching in Saccharomyces cerevisiae;citation_pages=33-64;citation_volume=191;citation_year=2012;citation_issue=1;citation_pmid=22555442;citation_doi=10.1534/genetics.111.134577" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=K. A. Henderson;citation_author=S. Keeney;citation_author=N. Kleckner;citation_title=Tying synaptonemal complex initiation to the formation and programmed repair of DNA double-strand breaks.;citation_volume=101;citation_year=2004;citation_pmid=15070750;citation_doi=10.1073/pnas.0400843101" />
<meta name="citation_reference" content="citation_journal_title=Microb. Cell;citation_author=N. M. Hollingsworth;citation_title=Mek1/Mre4 is a master regulator of meiotic recombination in budding yeast.;citation_pages=129-131;citation_volume=3;citation_year=2016;citation_doi=10.15698/mic2016.03.487" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=N. M. Hollingsworth;citation_author=L. Ponte;citation_title=Genetic Interactions Between HOP1, RED1 and MEK1 Suggest That MEK1 Regulates Assembly of Axial Element Components During Meiosis in the Yeast Saccharomyces cerevisiae;citation_pages=33-42;citation_volume=147;citation_year=1997;citation_issue=1;citation_pmid=9286666" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=N. M. Hollingsworth;citation_author=L. Ponte;citation_author=C. Halsey;citation_title=MSH5, a novel MutS homolog, facilitates meiotic reciprocal recombination between homologs in Saccharomyces cerevisiae but not mismatch repair.;citation_pages=1728-1739;citation_volume=9;citation_year=1995;citation_issue=14;citation_pmid=7622037;citation_doi=10.1101/gad.9.14.1728" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell;citation_author=S. Hong;citation_author=Y. Sung;citation_author=M. Yu;citation_author=M. Lee;citation_author=N. Kleckner;citation_title=The logic and mechanism of homologous recombination partner choice.;citation_pages=440-453;citation_volume=51;citation_year=2013;citation_pmid=23973374;citation_doi=10.1016/j.molcel.2013.08.008" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=N. Humphryes;citation_author=W.-K. Leung;citation_author=B. Argunhan;citation_author=Y. Terentyev;citation_author=M. Dvorackova;citation_title=The Ecm11-Gmc2 Complex Promotes Synaptonemal Complex Formation through Assembly of Transverse Filaments in Budding Yeast.;citation_pages=e1003194;citation_volume=9;citation_year=2013;citation_pmid=23326245;citation_doi=10.1371/journal.pgen.1003194" />
<meta name="citation_reference" content="citation_journal_title=Cold Spring Harb. Perspect. Biol.;citation_author=N. Hunter;citation_title=Meiotic Recombination: The Essence of Heredity.;citation_volume=7;citation_year=2015;citation_pmid=26511629;citation_doi=10.1101/cshperspect.a016618" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=N. Hunter;citation_author=R. H. Borts;citation_title=Mlh1 is unique among mismatch repair proteins in its ability to promote crossing-over during meiosis.;citation_pages=1573-1582;citation_volume=11;citation_year=1997;citation_issue=12;citation_pmid=9203583;citation_doi=10.1101/gad.11.12.1573" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=N. Hunter;citation_author=N. Kleckner;citation_title=The single-end invasion: an asymmetric intermediate at the double-strand break to double-holliday junction transition of meiotic recombination.;citation_pages=59-70;citation_volume=106;citation_year=2001;citation_issue=1;citation_pmid=11461702;citation_doi=10.1016/S0092-8674(01)00430-5" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=S. C. Ip;citation_author=U. Rass;citation_author=M. G. Blanco;citation_author=H. R. Flynn;citation_author=J. M. Skehel;citation_title=Identification of Holliday junction resolvases from humans and yeast.;citation_pages=357-361;citation_volume=456;citation_year=2008;citation_issue=7220;citation_pmid=19020614;citation_doi=10.1038/nature07470" />
<meta name="citation_reference" content="citation_journal_title=Cold Spring Harbor Symposia on Quantitative Biology;citation_journal_abbrev=Cold Spring Harb Symp Quant Biol;citation_author=R. E. Jensen;citation_author=I. Herskowitz;citation_title=Directionality and Regulation of Cassette Substitution in Yeast;citation_pages=97-104;citation_volume=49;citation_year=1984;citation_issue=0;citation_pmid=6397325;citation_doi=10.1101/SQB.1984.049.01.013" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=L. Jessop;citation_author=M. Lichten;citation_title=Mus81/Mms4 endonuclease and Sgs1 helicase collaborate to ensure proper recombination intermediate metabolism during meiosis.;citation_pages=313-323;citation_volume=31;citation_year=2008;citation_issue=3;citation_pmid=18691964;citation_doi=10.1016/j.molcel.2008.05.021" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=L. Jessop;citation_author=B. Rockmill;citation_author=G. S. Roeder;citation_author=M. Lichten;citation_title=Meiotic chromosome synapsis-promoting proteins antagonize the anti-crossover activity of sgs1.;citation_pages=e155-e155;citation_volume=2;citation_year=2006;citation_issue=9;citation_pmid=17002499;citation_doi=10.1371/journal.pgen.0020155" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=K. Johzuka;citation_author=H. Ogawa;citation_title=Interaction of Mre11 and Rad50: two proteins required for DNA repair and meiosis-specific double-strand break formation in Saccharomyces cerevisiae.;citation_pages=1521-1532;citation_volume=139;citation_year=1995;citation_issue=4;citation_pmid=7789757" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=L. C. Kadyk;citation_author=L. H. Hartwell;citation_title=Sister chromatids are preferred over homologs as substrates for recombinational repair in Saccharomyces cerevisiae.;citation_pages=387-402;citation_volume=132;citation_year=1992;citation_issue=2;citation_pmid=1427035" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=M. Kellis;citation_author=B. W. Birren;citation_author=E. S. Lander;citation_title=Proof and evolutionary analysis of ancient genome duplication in the yeast Saccharomyces cerevisiae.;citation_pages=617-624;citation_volume=428;citation_year=2004;citation_issue=6983;citation_pmid=15004568;citation_doi=10.1038/nature02424" />
<meta name="citation_reference" content="citation_journal_title=Genes Dev.;citation_author=B. Kemp;citation_author=R. M. Boumil;citation_author=M. N. Stewart;citation_author=D. S. Dawson;citation_title=A role for centromere pairing in meiotic chromosome segregation.;citation_volume=18;citation_year=2004;citation_pmid=15289462;citation_doi=10.1101/gad.1227304" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=K. P. Kim;citation_author=B. M. Weiner;citation_author=L. Zhang;citation_author=A. Jordan;citation_author=J. Dekker;citation_title=Sister cohesion and structural axis components mediate homolog bias of meiotic recombination.;citation_pages=924-937;citation_volume=143;citation_year=2010;citation_issue=6;citation_pmid=21145459;citation_doi=10.1016/j.cell.2010.11.015" />
<meta name="citation_reference" content="citation_author=S. Klapholz;citation_author=R. E. Esposito;citation_year=1980" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=F. Klein;citation_author=P. Mahr;citation_author=M. Galova;citation_author=S. B. Buonomo;citation_author=C. Michaelis;citation_title=A central role for cohesins in sister chromatid cohesion, formation of axial elements, and recombination during yeast meiosis.;citation_pages=91-103;citation_volume=98;citation_year=1999;citation_issue=1;citation_pmid=10412984;citation_doi=10.1016/S0092-8674(00)80609-1" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=E. Kraus;citation_author=W. Y. Leung;citation_author=J. E. Haber;citation_title=Break-induced replication: A review and an example in budding yeast;citation_pages=8255-8262;citation_volume=98;citation_year=2001;citation_issue=15;citation_pmid=11459961;citation_doi=10.1073/pnas.151008198" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=E. L. Kurdzo;citation_author=D. Obeso;citation_author=H. Chuong;citation_author=D. S. Dawson;citation_title=Meiotic Centromere Coupling and Pairing Function by Two Separate Mechanisms in Saccharomyces cerevisiae.;citation_volume=205;citation_year=2017;citation_pmid=27913618;citation_doi=10.1534/genetics.116.190264" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=U. K. Laemmli;citation_title=Cleavage of structural proteins during the assembly of the head of bacteriophage T4.;citation_pages=680-685;citation_volume=227;citation_year=1970;citation_issue=5259;citation_pmid=5432063;citation_doi=10.1038/227680a0" />
<meta name="citation_reference" content="citation_journal_title=Cold Spring Harb. Perspect. Biol.;citation_author=I. Lam;citation_author=S. Keeney;citation_title=Mechanism and regulation of meiotic recombination initiation.;citation_volume=7;citation_year=2014;citation_pmid=25324213;citation_doi=10.1101/cshperspect.a016634" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=E. J. Lambie;citation_author=G. S. Roeder;citation_title=REPRESSION OF MEIOTIC CROSSING OVER BY A CENTROMERE (CEN3 ) IN SACCHAROMYCES CEREVISIAE;citation_pages=769-789;citation_volume=114;citation_year=1986;citation_issue=3;citation_pmid=3539697" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=E. J. Lambie;citation_author=G. S. Roeder;citation_title=A yeast centromere acts in cis to inhibit meiotic gene conversion of adjacent sequences.;citation_pages=863-873;citation_volume=52;citation_year=1988;citation_issue=6;citation_pmid=3280137;citation_doi=10.1016/0092-8674(88)90428-X" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=J. P. Lao;citation_author=V. Cloud;citation_author=C.-C. Huang;citation_author=J. Grubb;citation_author=D. Thacker;citation_title=Meiotic Crossover Control by Concerted Action of Rad51-Dmc1 in Homolog Template Bias and Robust Homeostatic Regulation.;citation_pages=e1003978;citation_volume=9;citation_year=2013;citation_pmid=24367271;citation_doi=10.1371/journal.pgen.1003978" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=P. Lefrançois;citation_author=B. Rockmill;citation_author=P. Xie;citation_author=G. S. Roeder;citation_author=M. Snyder;citation_title=Multiple Pairwise Analysis of Non-homologous Centromere Coupling Reveals Preferential Chromosome Size-Dependent Interactions and a Role for Bouquet Formation in Establishing the Interaction Pattern.;citation_pages=e1006347;citation_volume=12;citation_year=2016;citation_doi=10.1371/journal.pgen.1006347" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=D. Lydall;citation_author=Y. Nikolsky;citation_author=D. K. Bishop;citation_author=T. Weinert;citation_title=A meiotic recombination checkpoint controlled by mitotic checkpoint genes.;citation_pages=840-843;citation_volume=383;citation_year=1996;citation_issue=6603;citation_pmid=8893012;citation_doi=10.1038/383840a0" />
<meta name="citation_reference" content="citation_journal_title=Curr. Opin. Genet. Dev.;citation_author=A. Malkova;citation_author=G. Ira;citation_title=Break-induced replication: functions and molecular mechanism.;citation_pages=271-279;citation_volume=23;citation_year=2013;citation_pmid=23790415;citation_doi=10.1016/j.gde.2013.05.007" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=A. Malkova;citation_author=E. L. Ivanov;citation_author=J. E. Haber;citation_title=Double-strand break repair in the absence of RAD51 in yeast: a possible role for break-induced DNA replication.;citation_pages=7131-7136;citation_volume=93;citation_year=1996;citation_issue=14;citation_pmid=8692957;citation_doi=10.1073/pnas.93.14.7131" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=A. Malkova;citation_author=F. Klein;citation_author=W. Y. Leung;citation_author=J. E. Haber;citation_title=HO endonuclease-induced recombination in yeast meiosis resembles Spo11-induced events;citation_pages=14500-14505;citation_volume=97;citation_year=2000;citation_issue=26;citation_pmid=11121053;citation_doi=10.1073/pnas.97.26.14500" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=A. Malkova;citation_author=L. Ross;citation_author=D. Dawson;citation_author=M. F. Hoekstra;citation_author=J. E. Haber;citation_title=Meiotic Recombination Initiated by a Double-Strand Break in rad50{Delta} Yeast Cells Otherwise Unable to Initiate Meiotic Recombination;citation_pages=741-754;citation_volume=143;citation_year=1996;citation_issue=2;citation_pmid=8725223" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=Y. Mao-Draayer;citation_author=A. M. Galbraith;citation_author=D. L. Pittman;citation_author=M. Cool;citation_author=R. E. Malone;citation_title=Analysis of Meiotic Recombination Pathways in the Yeast Saccharomyces cerevisiae;citation_pages=71-86;citation_volume=144;citation_year=1996;citation_issue=1;citation_pmid=8878674" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=T. E. Markowitz;citation_author=D. Suarez;citation_author=H. G. Blitzblau;citation_author=N. J. Patel;citation_author=A. L. Markhard;citation_title=Reduced dosage of the chromosome axis factor Red1 selectively disrupts the meiotic recombination checkpoint in Saccharomyces cerevisiae.;citation_pages=e1006928;citation_volume=13;citation_year=2017;citation_doi=10.1371/journal.pgen.1006928" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=M. S. McMahill;citation_author=C. W. Sham;citation_author=D. K. Bishop;citation_title=Synthesis-dependent strand annealing in meiosis.;citation_pages=e299-e299;citation_volume=5;citation_year=2007;citation_issue=11;citation_pmid=17988174;citation_doi=10.1371/journal.pbio.0050299" />
<meta name="citation_reference" content="citation_journal_title=eLife;citation_author=D. Medhi;citation_author=A. S. H. Goldman;citation_author=M. Lichten;citation_title=Local chromosome context is a major determinant of crossover pathway biochemistry during budding yeast meiosis.;citation_volume=5;citation_year=2016;citation_doi=10.7554/eLife.19669" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=C. Michaelis;citation_author=R. Ciosk;citation_author=K. Nasmyth;citation_title=Cohesins: chromosomal proteins that prevent premature separation of sister chromatids.;citation_pages=35-45;citation_volume=91;citation_year=1997;citation_issue=1;citation_pmid=9335333;citation_doi=10.1016/S0092-8674(01)80007-6" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. R. Mullen;citation_author=V. Kaliraman;citation_author=S. J. Brill;citation_title=Bipartite Structure of the SGS1 DNA Helicase in Saccharomyces cerevisiae;citation_pages=1101-1114;citation_volume=154;citation_year=2000;citation_issue=3;citation_pmid=10757756" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=K. Nairz;citation_author=F. Klein;citation_title=mre11S---a yeast mutation that blocks double-strand-break processing and permits nonhomologous synapsis in meiosis;citation_pages=2272-2290;citation_volume=11;citation_year=1997;citation_issue=17;citation_pmid=9303542;citation_doi=10.1101/gad.11.17.2272" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=M. J. Neale;citation_author=M. Ramachandran;citation_author=E. Trelles-Sticken;citation_author=H. Scherthan;citation_author=A. S. Goldman;citation_title=Wild-type levels of Spo11-induced DSBs are required for normal single-strand resection during meiosis.;citation_pages=835-846;citation_volume=9;citation_year=2002;citation_issue=4;citation_pmid=11983174;citation_doi=10.1016/S1097-2765(02)00498-7" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=L. Newnham;citation_author=P. Jordan;citation_author=B. Rockmill;citation_author=G. S. Roeder;citation_author=E. Hoffmann;citation_title=The synaptonemal complex protein, Zip1, promotes the segregation of nonexchange chromosomes at meiosis I.;citation_volume=107;citation_year=2010;citation_pmid=20080752;citation_doi=10.1073/pnas.0913435107" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Cell;citation_author=H. Niu;citation_author=L. Wan;citation_author=B. Baumgartner;citation_author=D. Schaefer;citation_author=J. Loidl;citation_title=Partner choice during meiosis is regulated by Hop1-promoted dimerization of Mek1.;citation_volume=16;citation_year=2005;citation_pmid=16221890;citation_doi=10.1091/mbc.E05-05-0465" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=H. Niu;citation_author=L. Wan;citation_author=V. Busygina;citation_author=Y. Kwon;citation_author=J. A. Allen;citation_title=Regulation of meiotic recombination via Mek1-mediated Rad54 phosphorylation.;citation_pages=393-404;citation_volume=36;citation_year=2009;citation_issue=3;citation_pmid=19917248;citation_doi=10.1016/j.molcel.2009.09.029" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. E. Novak;citation_author=P. B. Ross-Macdonald;citation_author=G. S. Roeder;citation_title=The Budding Yeast Msh4 Protein Functions in Chromosome Synapsis and the Regulation of Crossover Distribution;citation_pages=1013-1025;citation_volume=158;citation_year=2001;citation_issue=3;citation_pmid=11454751" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell;citation_author=J. Oh;citation_author=A. Al-Zain;citation_author=E. Cannavo;citation_author=P. Cejka;citation_author=L. S. Symington;citation_title=Xrs2 dependent and independent functions of the Mre11-Rad50 complex.;citation_pages=405-415;citation_volume=64;citation_year=2016;citation_doi=10.1016/j.molcel.2016.09.011" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=S. L. Page;citation_author=R. S. Hawley;citation_title=Chromosome Choreography: The Meiotic Ballet;citation_pages=785-789;citation_volume=301;citation_year=2003;citation_issue=5634;citation_pmid=12907787;citation_doi=10.1126/science.1086605" />
<meta name="citation_reference" content="citation_journal_title=Annual review of cell and developmental biology;citation_journal_abbrev=Annu Rev Cell Dev Biol;citation_author=S. L. Page;citation_author=R. S. Hawley;citation_title=The Genetics and Molecular Biology of the Synaptonemal Complex.;citation_pages=525-558;citation_volume=20;citation_year=2004;citation_pmid=15473851;citation_doi=10.1146/annurev.cellbio.19.111301.155141" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=J. Pan;citation_author=M. Sasaki;citation_author=R. Kniewel;citation_author=H. Murakami;citation_author=H. G. Blitzblau;citation_title=A hierarchical combination of factors shapes the genome-wide topography of yeast meiotic recombination initiation.;citation_pages=719-731;citation_volume=144;citation_year=2011;citation_issue=5;citation_pmid=21376234;citation_doi=10.1016/j.cell.2011.02.009" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=S. Panizza;citation_author=M. A. Mendoza;citation_author=M. Berlinger;citation_author=L. Huang;citation_author=A. Nicolas;citation_title=Spo11-accessory proteins link double-strand break sites to the chromosome axis in early meiotic recombination.;citation_pages=372-383;citation_volume=146;citation_year=2011;citation_issue=3;citation_pmid=21816273;citation_doi=10.1016/j.cell.2011.07.003" />
<meta name="citation_reference" content="citation_journal_title=Trends in cell biology;citation_journal_abbrev=Trends Cell Biol;citation_author=W. P. Pawlowski;citation_author=W. Z. Cande;citation_title=Coordinating the events of the meiotic prophase.;citation_pages=674-681;citation_volume=15;citation_year=2005;citation_issue=12;citation_pmid=16257210;citation_doi=10.1016/j.tcb.2005.10.005" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=S. Prieler;citation_author=A. Penkner;citation_author=V. Borde;citation_author=F. Klein;citation_title=The control of Spo11's interaction with meiotic recombination hotspots;citation_pages=255-269;citation_volume=19;citation_year=2005;citation_issue=2;citation_pmid=15655113;citation_doi=10.1101/gad.321105" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=A. Ray;citation_author=I. Siddiqi;citation_author=A. L. Kolodkin;citation_author=F. W. Stahl;citation_title=Intra-chromosomal gene conversion induced by a DNA double-strand break in Saccharomyces cerevisiae.;citation_pages=247-260;citation_volume=201;citation_year=1988;citation_issue=2;citation_pmid=3047399;citation_doi=10.1016/0022-2836(88)90136-2" />
<meta name="citation_reference" content="citation_journal_title=Current biology :  CB;citation_journal_abbrev=Curr Biol;citation_author=B. Rockmill;citation_author=J. C. Fung;citation_author=S. S. Branda;citation_author=G. S. Roeder;citation_title=The Sgs1 helicase regulates chromosome synapsis and meiotic crossing over.;citation_pages=1954-1962;citation_volume=13;citation_year=2003;citation_issue=22;citation_pmid=14614820;citation_doi=10.1016/j.cub.2003.10.059" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=B. Rockmill;citation_author=P. Lefrançois;citation_author=K. Voelkel-Meiman;citation_author=A. Oke;citation_author=G. S. Roeder;citation_title=High Throughput Sequencing Reveals Alterations in the Recombination Signatures with Diminishing Spo11 Activity.;citation_pages=e1003932;citation_volume=9;citation_year=2013;citation_pmid=24204324;citation_doi=10.1371/journal.pgen.1003932" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=B. Rockmill;citation_author=G. S. Roeder;citation_title=A meiosis-specific protein kinase homolog required for chromosome synapsis and recombination.;citation_pages=2392-2404;citation_volume=5;citation_year=1991;citation_issue=12b;citation_pmid=1752435;citation_doi=10.1101/gad.5.12b.2392" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=B. Rockmill;citation_author=M. Sym;citation_author=H. Scherthan;citation_author=G. S. Roeder;citation_title=Roles for two RecA homologs in promoting meiotic chromosome synapsis.;citation_pages=2684-2695;citation_volume=9;citation_year=1995;citation_issue=21;citation_pmid=7590245;citation_doi=10.1101/gad.9.21.2684" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=P. Ross-Macdonald;citation_author=G. S. Roeder;citation_title=Mutation of a meiosis-specific MutS homolog decreases crossing over but not mismatch correction.;citation_pages=1069-1080;citation_volume=79;citation_year=1994;citation_issue=6;citation_pmid=8001134;citation_doi=10.1016/0092-8674(94)90037-X" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=A. Schwacha;citation_author=N. Kleckner;citation_title=Identification of joint molecules that form frequently between homologs but rarely between sister chromatids during yeast meiosis.;citation_pages=51-63;citation_volume=76;citation_year=1994;citation_issue=1;citation_pmid=8287479;citation_doi=10.1016/0092-8674(94)90172-4" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=A. Schwacha;citation_author=N. Kleckner;citation_title=Identification of double Holliday junctions as intermediates in meiotic recombination.;citation_pages=783-791;citation_volume=83;citation_year=1995;citation_issue=5;citation_pmid=8521495;citation_doi=10.1016/0092-8674(95)90191-4" />
<meta name="citation_reference" content="citation_title=Interhomolog bias during meiotic recombination: meiotic functions promote a highly differentiated interhomolog-only pathway;citation_author=A. Schwacha;citation_author=N. Kleckner;citation_pages=1123-1135;citation_volume=90;citation_year=1997;citation_issue=6" />
<meta name="citation_reference" content="citation_journal_title=Genes to Cells;citation_journal_abbrev=GENES CELLS;citation_author=A. Shinohara;citation_author=S. Gasior;citation_author=T. Ogawa;citation_author=N. Kleckner;citation_author=D. K. Bishop;citation_title=Saccharomyces cerevisiae recA homologues RAD51 and DMC1 have both distinct and overlapping roles in meiotic recombination;citation_pages=615-629;citation_volume=2;citation_year=1997;citation_issue=10;citation_pmid=9427283;citation_doi=10.1046/j.1365-2443.1997.1480347.x" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=A. Shinohara;citation_author=H. Ogawa;citation_author=T. Ogawa;citation_title=Rad51 protein involved in repair and recombination in S. cerevisiae is a RecA-like protein.;citation_pages=457-470;citation_volume=69;citation_year=1992;citation_issue=3;citation_pmid=1581961;citation_doi=10.1016/0092-8674(92)90447-K" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=M. Shinohara;citation_author=S. D. Oh;citation_author=N. Hunter;citation_author=A. Shinohara;citation_title=Crossover assurance and crossover interference are distinctly regulated by the ZMM proteins during yeast meiosis.;citation_pages=299-309;citation_volume=40;citation_year=2008;citation_issue=3;citation_pmid=18297071;citation_doi=10.1038/ng.83" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. Shinohara;citation_author=E. Shita-Yamaguchi;citation_author=J. M. Buerstedde;citation_author=H. Shinagawa;citation_author=H. Ogawa;citation_title=Characterization of the Roles of the Saccharomyces cerevisiae RAD54 Gene and a Homologue of RAD54, RDH54/TID1, in Mitosis and Meiosis;citation_pages=1545-1556;citation_volume=147;citation_year=1997;citation_issue=4;citation_pmid=9409820" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=R. S. Sikorski;citation_author=P. Hieter;citation_title=A system of shuttle vectors and yeast host strains designed for efficient manipulation of DNA in Saccharomyces cerevisiae.;citation_pages=19-27;citation_volume=122;citation_year=1989;citation_issue=1;citation_pmid=2659436" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=M. N. Stewart;citation_author=D. S. Dawson;citation_title=Changing partners: moving from non-homologous to homologous centromere pairing in meiosis.;citation_pages=564-573;citation_volume=24;citation_year=2008;citation_issue=11;citation_pmid=18804891;citation_doi=10.1016/j.tig.2008.08.006" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=M. Sym;citation_author=J. A. Engebrecht;citation_author=G. S. Roeder;citation_title=ZIP1 is a synaptonemal complex protein required for meiotic chromosome synapsis.;citation_pages=365-378;citation_volume=72;citation_year=1993;citation_issue=3;citation_pmid=7916652;citation_doi=10.1016/0092-8674(93)90114-6" />
<meta name="citation_reference" content="citation_journal_title=The Journal of Cell Biology;citation_journal_abbrev=JCB;citation_author=M. Sym;citation_author=G. S. Roeder;citation_title=Zip1-induced changes in synaptonemal complex structure and polycomplex assembly;citation_pages=455-466;citation_volume=128;citation_year=1995;citation_issue=4;citation_pmid=7860625;citation_doi=10.1083/jcb.128.4.455" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=D. Thacker;citation_author=N. Mohibullah;citation_author=X. Zhu;citation_author=S. Keeney;citation_title=Homologue engagement controls meiotic DNA break number and distribution.;citation_pages=241-246;citation_volume=510;citation_year=2014;citation_pmid=24717437;citation_doi=10.1038/nature13120" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=L.W. Thorne;citation_author=B. Byers;citation_title=Stage-specific effects of X-irradiation on yeast meiosis.;citation_pages=29-42;citation_volume=134;citation_year=1993;citation_issue=1;citation_pmid=8514137" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=H. Towbin;citation_author=T. Staehelin;citation_author=J. Gordon;citation_title=Electrophoretic transfer of proteins from polyacrylamide gels to nitrocellulose sheets: procedure and some applications.;citation_pages=4350-4354;citation_volume=76;citation_year=1979;citation_issue=9;citation_pmid=388439;citation_doi=10.1073/pnas.76.9.4350" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=H. Tsubouchi;citation_author=H. Ogawa;citation_title=A Novel mre11 Mutation Impairs Processing of Double-Strand Breaks of DNA during Both Mitosis and Meiosis;citation_pages=260-268;citation_volume=18;citation_year=1998;citation_issue=1;citation_pmid=9418873;citation_doi=10.1128/MCB.18.1.260" />
<meta name="citation_reference" content="citation_journal_title=Developmental cell;citation_journal_abbrev=Dev Cell;citation_author=H. Tsubouchi;citation_author=G. S. Roeder;citation_title=The importance of genetic recombination for fidelity of chromosome pairing in meiosis.;citation_pages=915-925;citation_volume=5;citation_year=2003;citation_issue=6;citation_pmid=14667413;citation_doi=10.1016/S1534-5807(03)00357-5" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=H. Tsubouchi;citation_author=G. S. Roeder;citation_title=Budding yeast Hed1 down-regulates the mitotic recombination machinery when meiotic recombination is impaired.;citation_pages=1766-1775;citation_volume=20;citation_year=2006;citation_issue=13;citation_pmid=16818607;citation_doi=10.1101/gad.1422506" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=T. Tsubouchi;citation_author=A. J. MacQueen;citation_author=G. S. Roeder;citation_title=Initiation of meiotic chromosome synapsis at centromeres in budding yeast;citation_pages=3217-3226;citation_volume=22;citation_year=2008;citation_issue=22;citation_pmid=19056898;citation_doi=10.1101/gad.1709408" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=T. Tsubouchi;citation_author=G. S. Roeder;citation_title=A Synaptonemal Complex Protein Promotes Homology-Independent Centromere Coupling;citation_pages=870-873;citation_volume=308;citation_year=2005;citation_issue=5723;citation_pmid=15879219;citation_doi=10.1126/science.1108283" />
<meta name="citation_reference" content="citation_journal_title=Developmental cell;citation_journal_abbrev=Dev Cell;citation_author=T. Tsubouchi;citation_author=H. Zhao;citation_author=G. S. Roeder;citation_title=The meiosis-specific zip4 protein regulates crossover distribution by promoting synaptonemal complex formation together with zip2.;citation_pages=809-819;citation_volume=10;citation_year=2006;citation_issue=6;citation_pmid=16740482;citation_doi=10.1016/j.devcel.2006.04.003" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=T. Usui;citation_author=H. Ogawa;citation_author=J. H. Petrini;citation_title=A DNA damage response pathway controlled by Tel1 and the Mre11 complex.;citation_pages=1255-1266;citation_volume=7;citation_year=2001;citation_issue=6;citation_pmid=11430828;citation_doi=10.1016/S1097-2765(01)00270-2" />
<meta name="citation_reference" content="citation_journal_title=eLife;citation_author=N. Vincenten;citation_author=L.-M. Kuhl;citation_author=I. Lam;citation_author=A. Oke;citation_author=A. R. W. Kerr;citation_title=The kinetochore prevents centromere-proximal crossover recombination during meiosis.;citation_volume=4;citation_year=2015;citation_pmid=26653857;citation_doi=10.7554/eLife.10850" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=K. Voelkel-Meiman;citation_author=S. Y. Cheng;citation_author=S. J. Morehouse;citation_author=A. J. MacQueen;citation_title=Synaptonemal Complex Proteins of Budding Yeast Define Reciprocal Roles in MutSgamma-Mediated Crossover Formation.;citation_volume=203;citation_year=2016;citation_pmid=27184389;citation_doi=10.1534/genetics.115.182923" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=K. Voelkel-Meiman;citation_author=S. S. Moustafa;citation_author=P. Lefrançois;citation_author=A. M. Villeneuve;citation_author=A. J. MacQueen;citation_title=Full-Length Synaptonemal Complex Grows Continuously during Meiotic Prophase in Budding Yeast.;citation_pages=e1002993-e1002993;citation_volume=8;citation_year=2012;citation_issue=10;citation_pmid=23071451;citation_doi=10.1371/journal.pgen.1002993" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=K. Voelkel-Meiman;citation_author=L. F. Taylor;citation_author=P. Mukherjee;citation_author=N. Humphryes;citation_author=H. Tsubouchi;citation_title=SUMO Localizes to the Central Element of Synaptonemal Complex and Is Required for the Full Synapsis of Meiotic Chromosomes in Budding Yeast.;citation_pages=e1003837;citation_volume=9;citation_year=2013;citation_pmid=24098146;citation_doi=10.1371/journal.pgen.1003837" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Cell;citation_author=L. Wan;citation_author=T. de los Santos;citation_author=C. Zhang;citation_author=K. Shokat;citation_author=N. M. Hollingsworth;citation_title=Mek1 Kinase Activity Functions Downstream of RED1 in the Regulation of Meiotic Double Strand Break Repair in Budding Yeast.;citation_volume=15;citation_year=2004;citation_pmid=14595109;citation_doi=10.1091/mbc.E03-07-0499" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=L. Xu;citation_author=B. M. Weiner;citation_author=N. Kleckner;citation_title=Meiotic cells monitor the status of the interhomolog recombination complex.;citation_pages=106-118;citation_volume=11;citation_year=1997;citation_issue=1;citation_pmid=9000054;citation_doi=10.1101/gad.11.1.106" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=K. Zakharyevich;citation_author=S. Tang;citation_author=Y. Ma;citation_author=N. Hunter;citation_title=Delineation of joint molecule resolution pathways in meiosis identifies a crossover-specific resolvase.;citation_pages=334-347;citation_volume=149;citation_year=2012;citation_issue=2;citation_pmid=22500800;citation_doi=10.1016/j.cell.2012.03.023" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genetics;citation_journal_abbrev=Annu Rev Genet;citation_author=D. Zickler;citation_author=N. Kleckner;citation_title=MEIOTIC CHROMOSOMES: Integrating Structure and Function;citation_pages=603-754;citation_volume=33;citation_year=1999;citation_issue=1;citation_pmid=10690419;citation_doi=10.1146/annurev.genet.33.1.603" />
<meta name="citation_reference" content="citation_journal_title=Cold Spring Harb. Perspect. Biol.;citation_author=D. Zickler;citation_author=N. Kleckner;citation_title=Recombination, Pairing, and Synapsis of Homologs during Meiosis.;citation_volume=7;citation_year=2015;citation_pmid=25986558;citation_doi=10.1101/cshperspect.a016626" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="HO Endonuclease-Initiated Recombination in Yeast Meiosis Fails To Promote Homologous Centromere Pairing and Is Not Constrained To Utilize the Dmc1 Recombinase" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11.cover-source.jpg" />
<meta name="twitter:description" content="Crossover recombination during meiosis is accompanied by a dramatic chromosome reorganization. In Saccharomyces cerevisiae , the onset of meiotic recombination by the [Spo11][1] transesterase leads to stable pairwise associations between previously unassociated homologous centromeres followed by the intimate alignment of homologous axes via synaptonemal complex (SC) assembly. However, the molecular relationship between recombination and global meiotic chromosome reorganization remains poorly understood. In budding yeast, one question is why SC assembly initiates earliest at centromere regions while the DNA double strand breaks (DSBs) that initiate recombination occur genome-wide. We targeted the site-specific [HO][2] endonuclease to various positions on S. cerevisiae ’s longest chromosome in order to ask whether a meiotic DSB’s proximity to the centromere influences its capacity to promote homologous centromere pairing and SC assembly. We show that repair of an [HO][2]-mediated DSB does not promote homologous centromere pairing nor any extent of SC assembly in [spo11][1] meiotic nuclei, regardless of its proximity to the centromere. DSBs induced en masse by phleomycin exposure likewise do not promote homologous centromere pairing nor robust SC assembly. Interestingly, in contrast to [Spo11][1], [HO][2]-initiated interhomolog recombination is not affected by loss of the meiotic kinase, [Mek1][3], and is not constrained to use the meiosis-specific [Dmc1][4] recombinase. These results strengthen the previously proposed idea that (at least some) [Spo11][1] DSBs may be specialized in activating mechanisms that both 1) reinforce homologous chromosome alignment via homologous centromere pairing and SC assembly, and 2) establish [Dmc1][4] as the primary strand exchange enzyme.

 [1]: http://www.yeastgenome.org/locus/S000001014/overview
 [2]: http://www.yeastgenome.org/locus/S000002386/overview
 [3]: http://www.yeastgenome.org/locus/S000005878/overview
 [4]: http://www.yeastgenome.org/locus/S000000981/overview" />
<meta name="og-title" property="og:title" content="HO Endonuclease-Initiated Recombination in Yeast Meiosis Fails To Promote Homologous Centromere Pairing and Is Not Constrained To Utilize the Dmc1 Recombinase" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/8/11/3637" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Crossover recombination during meiosis is accompanied by a dramatic chromosome reorganization. In Saccharomyces cerevisiae , the onset of meiotic recombination by the [Spo11][1] transesterase leads to stable pairwise associations between previously unassociated homologous centromeres followed by the intimate alignment of homologous axes via synaptonemal complex (SC) assembly. However, the molecular relationship between recombination and global meiotic chromosome reorganization remains poorly understood. In budding yeast, one question is why SC assembly initiates earliest at centromere regions while the DNA double strand breaks (DSBs) that initiate recombination occur genome-wide. We targeted the site-specific [HO][2] endonuclease to various positions on S. cerevisiae ’s longest chromosome in order to ask whether a meiotic DSB’s proximity to the centromere influences its capacity to promote homologous centromere pairing and SC assembly. We show that repair of an [HO][2]-mediated DSB does not promote homologous centromere pairing nor any extent of SC assembly in [spo11][1] meiotic nuclei, regardless of its proximity to the centromere. DSBs induced en masse by phleomycin exposure likewise do not promote homologous centromere pairing nor robust SC assembly. Interestingly, in contrast to [Spo11][1], [HO][2]-initiated interhomolog recombination is not affected by loss of the meiotic kinase, [Mek1][3], and is not constrained to use the meiosis-specific [Dmc1][4] recombinase. These results strengthen the previously proposed idea that (at least some) [Spo11][1] DSBs may be specialized in activating mechanisms that both 1) reinforce homologous chromosome alignment via homologous centromere pairing and SC assembly, and 2) establish [Dmc1][4] as the primary strand exchange enzyme.

 [1]: http://www.yeastgenome.org/locus/S000001014/overview
 [2]: http://www.yeastgenome.org/locus/S000002386/overview
 [3]: http://www.yeastgenome.org/locus/S000005878/overview
 [4]: http://www.yeastgenome.org/locus/S000000981/overview" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11.cover-source.jpg" />
    <title>HO Endonuclease-Initiated Recombination in Yeast Meiosis Fails To Promote Homologous Centromere Pairing and Is Not Constrained To Utilize the Dmc1 Recombinase | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"bLINvx5wg0aQphOXLcJElVskj6Ac1mhnowFFCfAI-7k","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"491243","apath":"\/ggg\/8\/11\/3637.atom","pisa":"ggg;8\/11\/3637","ac":{"\/ggg\/8\/11\/3637.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/8\/11\/3637.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;8\/11\/3637"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;8\/11\/3637"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/8\/11\/3637":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00221zonVNDCVgeHGjRlnmR7GsDX6D9b6980ADXtDQtlhcG7Em37SkvZZSJCVRllGXr+WNKfL5EspCsRPPKnAgZ4DbU+OgVrh95Tget+EfJhrcHa\\\\\\\/\\\\\\\/d8BuQaIFoJED+vpu+lbqjuZi17uhZVLMwsOvhM+wzB+1RYnlXFQ\\\\\\\/fWJASJYDiudPD9Of66bMZCulaeC4VWdmF6j8i9Q11tYiSW7OWoA+ivmp3uQLPm6f6m89dhLJ0AeaBX9nkKL\\\\\\\/Oku+tjD+m+fH67oy20I4ReSiN\\\\\\\/KtuoxBNBMcx0DtRQusWG\\\\\\\/Cx4YTUq6\\\\\\\/u2oqCK6oei1Bde\\\\\\\/I5b+GcZLVurVcJ7EpcYAfXdiqQN2rL8Lc0jLoIohc0y5NGuJiKedSCogqiM4WrcrWC7Bz2vxQYvxKvFNKGdWa4J0iUFJdPN9XubPjJnd5V4S8rNbWgBuIy7jgJ74wwNMxNrB6dn7ijW8OT7EZqZRKruffAGhNy6HtAfq84Dz63Sk8oWC2Mil4B\\\\\\\/60Oj0zaP3LvXdOMSMJsVsEH+OVpu6004aGtrpLDx1mw9HZxgd0kwfHv2NWmMMnd22OEaWsN1avdci4H9AsWBe\\\\\\\/i2oFLcB4pax0kvZwzkfME5QAU3j31fzMA4UreGDUZPScxQy3qfozrtJcIlev27C4ad0OhzcDX9j8tLPUbGCZtM1SwEOVZfMlBbtXCXJjUO4Ltt\\\\\\\/mXQOxdO+DuxqmriutaRYR57cDvOdhM\\\\\\\/QhbksIBYjdr+j2QLeeJ68m1OKrIHBE0ReIQpcN6xAxmKClwxMIEcLQEu5ldJIpn5q5B5aeyrMmkavp0h7FhTMFfTeVZ5p24kf7f+BYvEZFU7mkrVnFARZCOgLKPwTWqXmOtFgbFEB6dW6PZBlpL2SMWL\\\\\\\/jnM3SnktatqnEG1W9GS2Gj0qQIq2WH5iPJekJC\\\\\\\/dNYXnxcpsd28zLQtMODtuGsi3cac4Al2M\\\\\\\/52zCFLiJE\\\\\\\/brwz2qfseRdpDtlaGgIatwOMmxKYNaZFPjFqYSMPd8RYiV0FmHIpKb8RywUQA+BqJvSqrcTYUIVbr\\\\\\\/QYqbBYDlUXZZx+aiCMX0lzIg\\\\\\\/b64ohe15i62muLHdQP6n\\\\\\\/t84FXihpSIwYr4B+5bD6IaKI92YNXv25gOTIIaBZx96AYi\\\\\\\/y9sktn8PCocUl9jf1iWh7NVMuEAUGBVxQFO7bJWrbx7ib57WYKRBoiS0SeyNtbKLLjAHv\\\\\\\/5NVv3vBAapODHLy8EGxrcwWkp4dXjsMAyoZSEkxJVuly94d+93oPFN4m1RW1ftQqi\\\\\\\/FjP\\\\\\\/hRVw9\\\u0022,\\\u0022iv\\\u0022:\\\u0022Abu89Tc0KgyYreWiFF6ABg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022334c9174686b0c636aeb90bbedb1c90ade5411583632efc480ce3c559b30bbbd\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022CBADw70wCDPkfGhVKgd1jC+Br+NDIXG\\\\\\\/t4cvWmnVufoNVjDHN6CNSaZ9RRopPKr6Rpbd6Y6v5pMSGK67+tPGZrqjKTdRDoJxkByfF\\\\\\\/G2Vco7dubZDbb6AYkgWgAtNVo60ydSXDn+KAlNWjkIzQT5ewRW7k2P9iX4C9QfsNRBMmxxiI\\\\\\\/1qDm2p0LmzPxwaqkvY5D9SG3+6svRxfwxwtvsUPG44S8fgdg2mHaeoegktPMnwluSr0D+IZHb9GUiX+HnjjM5P0bvo9O\\\\\\\/eUcN4y7mZ\\\\\\\/LxflXsTrkzgtoOikPvSM16aO8rSJ2bJQ5fjDvUU3MTI1x\\\\\\\/isWB7auCDHyCjH\\\\\\\/6yVzoE++wY+ZmMZ5Ox4jsV95LFaOIuxk3VSg7ZU1YEkcUV3RTgNsVtkSShSJUS+wAs41JWe8Q8IfU27kaYPY6pX3V52m9fkh8tGkd\\\\\\\/mtGUk+rsTtPxDuYVdNQQcmo3NzlBVdeScXt6r7YqqGQYH6XQj5p9KOw0oLPL9DeixFgjIjwJCyNpi0uxc07gIAleWAZKqckKSlWq8we0qRgssQ0LhgWzXZKx\\\\\\\/ExK9hc1kkJSNtOfOqWzVEaKvo7rX1HvhsZifm0EkWKDGJhDFk\\\\\\\/yv\\\\\\\/DDQPgPY0uS1H3mSDrdmrUXbZg03lzLWWAO5CflSUMK8uN5fZWWoIqiQGOM6EBG8GGJfka43gEF\\\\\\\/toWyNgQlywrQdZf4+z4Z8jxq0Wvn7JTGuF7zUH7Ghto29GH29\\\\\\\/PdZkcVjW6\\\\\\\/pp+tdBOR\\\\\\\/rh7MENRPL0f508XALX7\\\\\\\/+kho0K\\\\\\\/8fdmwAUlqCkHMJKPB7zISosW1yul0r8huV8wAuOmSU8Vw54mntApGwzHsCZCHYp1eisqmlrQit9\\\\\\\/i1zRDnKVVsLMMc\\\\\\\/V\\\\\\\/ZfdqeIe98++EklRsvmaVYhjnqL4jooorsIQ\\\\\\\/9egldFUv4OQ4C+O8Mlx5c98nJrEvN8rAfYB0hvnOXEcnE6ByKyBYnqb6aj8RxcWJ8pbqH0\\\\\\\/ko6rMi8Fx9fe3\\\\\\\/Zeae42AUQSTqSG8TrUt+zaxRk69laJxlyHTjN8Ori615eQv2cPuvs260yEwiIwuDr0wsHA0M8oL4gYYRIoxQFF0j+lMfuu4loAGJ7xrS5qN\\\\\\\/NLSPvinqNCWT9ZuWBSBa5wzGMyc+86uTLG2Qq\\\\\\\/oiR\\\\\\\/P7jGi5JVByUZIDxqS+KiI9B+V+xw5YI1B0NOrruHYoyS5ikJIOM0dAUOksOLzGEO8YBbEQr17Wh7T9hoGDAhy4elaR1QpLprSpbgGMw62\\\\\\\/RkMrYniWY3knRtp1Zf5bIRhP0wb9tbs1vkqJPuXenfZ8dD4vrA5MNfYJECckyiSTDkiLvUsJGOjLTC54TPJE36S2aBmZ4qsxRRqiFxPXhUDY+J9tEI3mrQ9zW78zpeny+E+HM6nUqBzYhgum4zn0l0vOz13d+H5DQiX0wIVASime\\\\\\\/DN+MVugF50Upny\\\\\\\/NCVEpXaRjsr3Ma9mtb4+6cXFJFq15WBWb0iDLqUzWR4vW3gqlf+ckrsbbvzIOcjkwfdIh1mHdm1uWUYrU88ofcoVHq7JRpg8mHWQwxB0j6RT\\\\\\\/fdgDpEwNQEbCSdGVNwFC85QKO8o3w1K4mWjTDb0apRvREtYKBzdD69idG2BbTbnNacjfrT\\\\\\\/nCeKZFwc4y4SZsrObD35BMLTl5ueQzp5pFJA2BmxdXQ3ruY2hbLr8swJ8H\\\\\\\/zFVgldswIkXWaAkrbmDc\\\\\\\/og50SlSPJHxpJXeCRk6wZ0f2KZaZd5mVze+2vwpltVa7HYgR3w2ZTb4PFfhzJ0ZrycgtA4frofnLUsBM9BnAi\\\\\\\/oIoNvVVE0hPSvVHIIdOqkfHRsQFDoQof0IkKFpQPmBqkEyrZMSHcXbm326hZTSA1N2bQ+rI+6CC17BTMOQHIVewZEq+k4pkpo2fGqAOnVfcI57gnoLYhUvwDUSpAPinobCtqNzdTDxpbZq7NDo3GhEdwPBmfGi0M3HrhTArell6OEHQnvIJsYKAlNyn6OY+ogkfXNLbE6ZK4Fu6fKJM6qwqzkopHFuL0R68d69Ah81v8zU53G\\\\\\\/UeFW6ZiiQeI6U7iCeNz4p7RXomt72ludcCOuuAdgP6eoF9SkgfpLCH70fApjQHvP8FVX5EIXEiZyoS6pc9iaEx6fKmUAsOCB78xBL8Ta27suf1wEsGnWF0PdSKSzknmqc2wOMxhnpD2swNoLPOFnkX9ndTVdV91rZ4XN91dcnL7mJSRlGDIM50rDYLUd1vn7aLgb4L2ZD\\\\\\\/CwhJosmnIl7Z9kFfKEkKO6usnzNB3rYzxogFTZTVQIbOTijzRuSjCKUrYPQkiXxiYhoU\\\\\\\/TWDWIJIA4ofXPi3gmfdpBevQ2UHhZ6Qalfh1bbhVCdvZIJCsi8rs\\\\\\\/vQ+pfidrOmNYl+3PTR23W83FIhu1SUyt+3Th+DQ5BaLpIC9Yq5zeLlm6yqAZKpjUYzGxI7xaF1BB0CR6a+ewK2M1dRidH5ue7RHFf2BrvBgJ3fYan87s4UgzZPYquh7eWi6FKanSNcHjcngTRXw42TBpZh7\\\\\\\/SBhG50YJlvo2TusOQ+QSPXaTCsrkW\\\\\\\/W0b3ScCwjUqk78wUqZqtMBJf\\\\\\\/tEubZx\\\\\\\/Oa3kqUGTmlqRAV1ua8AMf8su6aOjo\\\\\\\/LbJMSpMOvdbRxuJi53kOI65pQvZTlBv\\\\\\\/DCzBoMoJ6vo+eLca6Zv8HNtfHeTBjPyoxrAPXxPp\\\\\\\/7MmoIkMihqtJa6JWhHmDHVej9ddDfMyXA3\\\\\\\/VBOikQf0IBiKRAgeUKhbeWLBvjUFYYOn5t4hihUBWMrMkuKNYXLNBG9Lbue+IJhhonzSoOcGufMDJhq4rJ4ukbteJrxtr2WYtrfm73lMt7nJ0hJTPqz2qq+yybtwHwP+jHV\\\\\\\/31fk7F9n+nSHppfYJxRvorcYv2dhEFQgihWaeje57ns5c76vs6sBiPzUYE8wFzXeVK7+lt6tUB5rwPz+ia8ykS6j2mqa1Atn7QvKOdCyqT8GbTZKHtgvnq0TpyLhdh5+AAb4oDw\\\\\\\/r1QQUfAt6knoCMgVmYDQMe3K+DeaMtDou9QFmkHdhS+JYJGpy+XTVQqQEhn3wYNAxQvsJJbw4eZ9KfUSUZJqYrjeeGntXbQuBKv1mqEJuYJNFrkzpgn5aEPnaOGp1oZpQ408Ss8KZYzeqpHDnJ8Sl9f8kAgMK54WjspITxTebvJqA538Wnjpjow5ngltpZMPhTym2yNtthdIus8va7LdFV68w4F\\\\\\\/lj3n8wilTNpDfTeLyYmDZ4\\\\\\\/b5TbVfysFZENvnEUOKohiUt5APENVev9hR1N6VNcCm1cZDf187mKe5JyUtg6wTwKyTQNAzulRSOrDaY5jdhfm54aieGJr6SuD9suD7xvY\\\\\\\/uoP3MPPoYrUohilZgcjNz+Y+5aBZgwGNw4O5alHyueAhcMojwT95AMXwoFu54Wi+mNtqSC\\\\\\\/g0zGD3YM4lqbMDA8to1p6pjNenMX\\\\\\\/rS+\\\\\\\/59ifP6Y0EwLDuzzgIEsTVx2xuJ7TKKA5NEQXSvLc9YSWCnOPW2c9B0rx+jziTHTXRw58lpkpoYcU0p4W6yfezCDUIuW7iHO3xSfNdSbwrZhQ6jfjkyMtSxRTAv4TZTFnzHwbjA4AaHoSk+hput6ivD3jIctpUiFDlkecmjXnOiazqVU5lr3FO80PM48fALwvmEnPjcUvpUttkaSQKHCNrbAsOuCbfQfNcf9LIZZWxCuLL2pqjJbNaGhHbE1+61Iy9Mkavcfpd0EMREHi55gmSALI2Ul884qcZEihixbB9FStxCkDTMJP0WQFKQBqBJpLTEU1Y71+jH+oKwDOECZLozBRpMge0lRuNozLqCxhB\\\\\\\/ySO38rO8iQyOOafpaUHDKpdx\\\\\\\/Th9kxyS9jr3q5G9R4ibRdlqX\\\\\\\/viP4uRqrfadlaDx0Wqkm2xf7D\\\\\\\/DODLF\\\\\\\/8+C\\\\\\\/ZuyHe34MLqWgS5d0tD77k\\\\\\\/Z0sdRarHcHRUxOLJsvd8ie5ghZ7cX6dbe+Anv5xHWJAkJdcgs6SvlOMnG3c7w\\\\\\\/9+8LetUt1f8FkgZq8Xn+SKG3YWShJ2Rq5g6+XpPgyZbU1klmlZe9M\\\\\\\/yGbGmlmwlKl45lmb9Pqtp4N5mtowN1VR2o6\\\\\\\/Tm0ewYkcsCRdCsgwV5Dz0gDglPf1GeQrbE07UQQIILBb2L8WtPdUPKFZbPwNnhaTHPeG\\\\\\\/aWF3cp0kCZqLNWIJvGakOljzOW5NyO3OzPOZ6aX2xCj1GPRXB193LrZfudUbwtzUKe\\\\\\\/mGxXQFBC4kIryBsFJecuPcq1jIHflmBlFhvwV2wbs7Bexf8hG13Kd0qOxjSghMPIb\\\\\\\/VsdLxggljd8bY+67Wv5uh3KtdJj+V0sVXVFm4HSvTA7F9muDLkhe9opD3KTwI0EXZ6nx9E6RAIV2n5NKtr1tfZuSWACSdziPqyZrHaHgR\\\\\\\/uuVpwpOec4AeMuxut2Vfqldlq4ZjQJf4Ml88d7mKT6v+NoR2\\\\\\\/qxsb+eTI2W6onyBcrgMOoFwlsPyEXfp7DsmvHAGfmtZgmL6ZYPNez8ih1ArQ1yARfi659AYoCci\\\\\\\/0D8J8Lj2vhINMc96uPm6YIiljpGM3MO\\\\\\\/v4RF2xf1WCVwxsGaEN84qQ6zYP5tz8ghgq00wB3bBaUKmrc7gb3enQCd6ChqymynhnCMCr4rLS\\\\\\\/82gyhalGUfLy3wtqoEw3rhjy7S7spuuLe1DCWiz97wF97t0ltACSYIi3lqYe1VqGL4YAqqezIUrkaFUILTMLpJ\\\\\\\/izz9oxb6GlWRvvXPy66Bc5grhCCJGrkUZ1x0Ld70x9aAi2FovM3zigQ1lnkaPt6jEZnIGKxToTYxoF9ZIICvihhkWXsMfanHXti39hX7xnMUUuyNES3U1UQGMF8VVSv4dlvHkgN7zuUixtuBbOuf2YiOPzA84bAOV\\\\\\\/TuDNyjk7HQTf7eznSR8vE7iAXLXVNKzjRCOFdnfUxYty5KpN8HCn67YwXpZTR6Bb9i41GUcvTTT5NfNoP1iGmU5qmjlScwhwbfYqI4Tyo5Qldn1Yz2G\\\\\\\/TuOnoQAferPd4JxfUboSoP2Txcj1DQGmGoEzxwS3bER9eijDEhphXebe4cBMw02mmgwZ31G6UJijtkMRXGzzxv8O0Wmb9rdQunXucZ4AHVlq+QeWcAOWsDLOrmYZVBWVoKoWFYqDnp0emSGU+KJ7f4nWPmmI9UW5th66rZ8QRpzbxsIoR+V6tj4R+IkExkGEcZofCR0JgICNepIYKLn63WeJeekBHojvCtNmHPcyCj4hNpsidQ8FIYqc2j9b2e3IPJUYkiqf6UkWffGI5HxW9LP2sywYb2Z3zciTMEmjC5H+U9OQ5VcRjhnACmBaD\\\\\\\/yRwL+fxuaGwYeFhE7iIt7GWEDHATOuqMjY5ZZICjp9Quw7H\\\\\\\/wSp2SXUEvAgsPWlxK5Cb3KPt4bbyLqDFOwCL+LX\\\\\\\/tO9nJYwzCNvZCdNZb0TILTnrCQRtW6mzGgccYniiJL6izJV4KxElWZ5h\\\\\\\/0ox1VOxXCvunOZAmJx\\\\\\\/61H\\\\\\\/o6Db9nW17I43xn\\\\\\\/5o5kwk7yko3AVgsYb6JUuHgYxBZktFMU8FVguGlpdbUAkcZKY\\\\\\\/awp0ZlZsIN4Gp+LW73orAB4jT3DSi0cp0N3kGof9IOBp025j58dUw+Js3zoaRPEhj3Xwr6cov5cHCka1Ag7LKSXvWGq3LA+8d9scHkOD9l9H83mNxWf8KoIjtN4H3NoMTjhmdUZA9Cimo3zs64ArPv11ma1Up8EXAtAmulfiTWKxcT\\\\\\\/MuuHmqI0rQSg+Elo0HDuqLcN73vsZfQL2VRzQK6t1MThKzRqmCBKSt8Ts7ut+kG5728uBykbIjvQM0YKBR37bDAQv9S\\\\\\\/8gGUpUbe+WFTe0aEEypz5SBIw4SFPrOlmvGtn1dwU06G1vmZDCvbV214tMhighjvO9pc+tLfGp4ubZe2YV3egLFAv7DOwTreq9Nuf2X9xFPtU87QnJTAdROjYYIJo1Q4uT3e1nh57L7p6QX5vs6wd5K78Yk9PTzIh0NTfyLS8wAxvXliftlq3tbs1lzCXN1\\\\\\\/ba2WRdrs0ypylC\\\\\\\/fQcbw1vyJg6KUSekB\\\\\\\/+cEnBRHcVCGiGriE95TuL7PvnNdggbGjIhfq+AyfAiwg3YZmYkIkj+mLdAt6D5kSOHnHPa6uRI\\\\\\\/\\\\\\\/RFKLUSqJo3OgxLnq7s3v+J8mw87HVvdkZOIcRp0fprPrOORaNLN5kMJRULDTQkYov9Nq\\\\\\\/PkNMvRznZY0H6\\\\\\\/9rMAMI8GpBMngUVmfI3f\\\\\\\/2RvOa8w0X9Fyk1OlN4\\\\\\\/uclM5glJGx9UuMxCwnTm8NY7EgeQb1d78+EYIln96VoLYU2d8\\\\\\\/n8TbLikI2vs13iUn5meG+3Ep\\\\\\\/YgZ6yry0L8BTrBHfje6dQ6HR8KcFE21YaZ+qfg\\\\\\\/Qj3sNp6G3j3UHBYMBz2Vv0F1+GRUdJmrSVo\\\\\\\/SgOEOwem7P4cTQLz9+m7G+gBnAbcP7JeXGz3pWKOiTvIzEJIxCqOtW4oaxWYFrOG+stnjTLhhEhTFqXpnSV3NVVJR59IwbnJZ2AW8\\\\\\\/fd9L4SrBNooTXjGwftiZ\\\\\\\/GNDmEyS1KSAY3V1upkHeX9K2teLKYcXiHgK9L42HT5VYcQ0kc67EjE58pyeT+SjCNAQJWSbBhcu15ULBof3F7x5RN0nyIxezFflRvMqT5zymedNM8QuNrcU5nZrnWrSqVfkMWrnzINgwc2YH7NEHnsP1J7vSiENVu5oLk2Gfr6r9ZbZbwFOqBY2EhnqfKJSgOVnQ4m4GOwDp8EizZsJKhvYnGS69iU+4nqP2AuaGQjqQ41TQKF5rNOzoGUeGHSSeuFcwWMuQWWVRX+i0cNFHVoWAuTIcZIf5ZtJW3cbKUks6BVQYZAS+xRmHPn2ijuK9BQtecJRCjG9P+b4q+Cfw05dg9cdKunHaD\\\\\\\/hyxIxj9bgXhSnYfT8m4KvCZOJND\\\\\\\/978Ff7dvEeSsqEkExXd7elTGBugNPBlyPTHKIAcfoGl35SVJeRgKAW7cwAEou+MJzJsCHDujB6Q4OQHx\\\\\\\/\\\\\\\/lcUa9D6S3hag1fJZzsTOV2eXyv\\\\\\\/24Ad\\\\\\\/P88pus2FShU683Fbl4e1Z7IAERIKTGmiS5\\\\\\\/3MrpUQqp5I8ojS\\\\\\\/hoIVMbgzEDrShZEzySWyNZtY8U54hf5jkC1xzMrrWS40PqaeCoG9UauKtCmsGAepfIHzKAW5eKEAoupH0Q4K6xrvUujU4pk+4AU8vHDueBGFcAgAB0hxNs7G7eRiujNk2ctVV0jKxcwUBEEstXl2d5Gk3KimHDKAxuN\\\\\\\/7t5vEs0cCyCiHChWsr8Iat92ugFS6neb9ww9tCH9F39Ww4uYkre3LQtSY0LwvN5kLor6fXXPmBEBi+g+ZO7QUUua2KccCNu17ynY+Zgx8hNkJ1K6azB0MdLQWpvca\\\\\\\/kD4qyiz1NIholOqWNR1jx4cU8D7c7q4o9ZAKEA9SYQ+KcM76nyghsBKQWI0e\\\\\\\/zWL8Ydx9DI3L5UMYBb+Oxjx55cdRJZdMQDFqQBF62H7HSUr+anTDtx5fXOazLXXozTl0AJ\\\\\\\/vHRNQQ\\\\\\\/3IhLFQ\\\\\\\/sMmk+DI\\\\\\\/vgobifJ7wQSOhDnIL0RTzpm85TmH6nLfT8StH0IRKd\\\\\\\/mnLWLGKlyr2eTjnz+RDpDl3vQSREAGnQrcNB9byC+zTh4ksRLi0ynhMpt9DMpNrp+t3beXaLjup3eYt9uSdhr3pwjaIxSohDR0MexOkKrTHLpZXk5FKpWY4Ony2hwql924u2L3He7s6UsLTUCZD8Z6XbZiwzCsyb2FdVZ9ZhMLFd3RTwMZ+pbLu+0gSjRaIR8lpxKKEx\\\\\\\/TLVfbdcz04CXqkOnMhPREv9bg2Q4AI3WxkKiaCuXCAA4tOfIImHYFJt6aPMXkooPAGaJ\\\\\\\/+ga9XLQhsdjpjPgZAx9Qo7RWjj3mD69QIyjnSELuHAIFQwkZfu5ANNej3gHPQ1xW\\\\\\\/MT8dpr1gb2Pmjoz0Eyoot0FlozDsJs\\\\\\\/KqCLtr2sfsdj73QSalGwLiov4BhFagBFKADpVZEvOPrA61Lwdx8vNd9TwVxWhZMAzmyGJVB5Z9fzTAAePaCa7VysOmRF\\\\\\\/LRFyI3I\\\\\\\/eK8fWZIaxOMIcGsTpWFJ2N53xBJ6Ep7rDhL\\\\\\\/rkz\\\\\\\/y9Fnlvohp\\\\\\\/ovvuZ2yj5pADvmd5pjiEwzfz4Gqjd\\\\\\\/SBT6zsouPri5NiuIIBFYpDCXYKBZkTKdrh1QILnlJjcUHXiXKiL2nLMP0iU2Su+iiBy5Vm1jtY\\\\\\\/tkzEr7rIDfY1JcvhMcP3G6mqCGi556cUkUjUaSRuwx8v2pAHPOsx1wISKd0PZDKOtMHaugoGiJa7WSVFqQxyAEzWmxpEvq\\\\\\\/VnosyuAwm3O7+IJeParyU9bk767RC9DBGzvbJNAqu7dq1GK+mpFSxE5HW6uR+NMoRNPAW\\\\\\\/CPn7Lt44Fa5CxnFb94kcOrU4Pvx0GhUp60nT4rqbS4uYXly3P8WxOC9dhGvoepfP+JlervNt3+hglW9IWpm2WTxRW16XMACzHPg45n7q4wP4cWE7RpisLyFMiY4jMrjni4qy8QMF0mxYGZdHZoI8QzN39J8rlbo933CE3+Wd5VIXAknrdjBjCeo43unV3dNIv29+yHQG2QnuyGWKOHItUYEn019EKqY7ntzUoV0MDF\\\\\\\/6xPcXWmL3\\\\\\\/1CG+M8aAH1f7xj28nIB0vwk+9SarS77KjfHS5J\\\\\\\/Ndl7E8iMDKoFqx+3UtXogq\\\\\\\/6FL1ZijN0DWcgJRIDfNAGaEiDLIrONmS5XuFP\\\\\\\/TpMtW87Ynj4WMn9qYd\\\\\\\/zyA7ydwba5aIvsKyJzwrHtnzEsRX1pDQ8zfq9zjjtuaaVp\\\\\\\/lU4dVpiSFVfe+UB8Cqrob4OjFctULqDsupopLGN+OJwErHwZ1oh4UC8RpxcrPt8gv73M\\\\\\\/5FDRFU0gxoDkmxfbhJaxlE69+FPPcTJo15OZCfjgUtBJuSZoh7KKKUK\\\\\\\/ibJR51wHdmwbJsJ1QBhLS3AlbI9bWwbC19zlALQs8pP0ggVeGgc8KW7a+3olOfRHQpm5EB4SD9SG9h5OOL8B+9oJq6mxq\\\\\\\/03luuc10meawzrtlMuKptATod0+yfoZI\\\\\\\/RmgJcOn+vsDzEKAfWfRA1onipgSl0Gl8\\\\\\\/pmiXr2ZNjnkCftO2dShuvOTsgc2xtiVnnTuMuOAsQOTClDgOE9lSrIewxBw3yFj9IP3RLMixbiMJPj5f775uLY\\\\\\\/DN34a8jRN4MO64QUoAgsbI4pC1BZ\\\\\\\/213ZaYaYEWHuuUz6DivFF1WQfo24n\\\\\\\/QdpNEofuphVlXnVj6nmbzhySkQ\\\\\\\/gZLTaHwODUhVvSWhJrfTQXdWTMBtGuaBxXPNMk0nMKAi5hMCaSHgY+HEIgMODhXLxeyXnKq9qNzFuKIx3eVtafVb09u4tZgi1JjHgY\\\\\\\/hwtYlh8KN61SyVc6Z\\\\\\\/\\\\\\\/Azq23zv71CVKXiraT8revavX+Ey9sh8N2i7JmbjDW00QE+6ah80xDaSrM7aqMvFuuhjN19lieo3FpbYbZMOB56Ws1pPkBIDJjEDYnUo8KD9YxcVifdVDX59khW6dBeOZ9bSTmyJ647G\\\\\\\/FPlNtuGusrYIeAb5qGC9fkiV9OXk\\\\\\\/HMZhdd\\\\\\\/DLwg7lbtvxiJRXydQ9+mSxpjs9ZGVl7P0Dn9XZ1tCXk8Uwe\\\\\\\/BG3WyfaAkOfEf+\\\\\\\/yXY\\\\\\\/7TsR6P9r4NsGj8O2qiOZ2T8RTitA3z1EXEwCxUbzI9PHrbYWGL0PBYjq5Ttwd9nnKpQsYSXSd4m\\\\\\\/0qUW7rU8MKXpLceZS6DvVTn5hC3JYA1Vg5TDcBqPQ\\\\\\\/8OL3Sh8Umgpw89pNh\\\\\\\/uuGAUILgIoMBsK7LUPm7TZ\\\\\\\/z9y8dab01QlN4KxS5nK5LYSxskK1iqHAMscaaWR\\\\\\\/AczHpljCH2mZRRCbdY2ZmI57MLlSbJhTS94zgjzR3ca9crPEcdgQA2pSQPC3K943NnzCxSlKAxD+WqRU8RtCg+flgL09Q3y6twSJU5WdsB8pVFrnJLpVLcqdd4YntiOK9gatFxVdamFoyRAhGem6kolpMyVPGj3Jcu8eWuiVntin2fz66HR8XcboPsIdhAHw1YYhr3i4YHJObRdLxbWzwyEaVkxNaKJ6JAbqhrD8jqVY+Ja9t3kve2oKIvIY77wdohOxHGorNlZM+cY0ow0hPVi6wM6HGda2Ehazv3EdFpdybpiM3JYel3ivn5Yatz0lcyatDbuOZsX9lPoB5mztZ1P2oLfLE40NdEizFLWJLtmXctppx0htEe2u7XeoWt50JZameX83p96HFfQBNjA3XzScFzpG+yXILwh0OwB2SEOfZOwSMfC9yI8dhKma50YAi8IKPAxNwftR5y\\\\\\\/SxzdEE5Q1uPD6yMzb9cGDUu+oFNESTDv3KvTAMRlTLY1ZIiSykxyyv8E4cH8jNpng8xG4q8\\\\\\\/Jz9OJjjykLlyjQ8mA0LJyO6srtWBKOg0NJLVi5A0I1rl+W2eVI45KermVzQkQdt86LflhexF5b5VfuYVx\\\\\\\/GdXG956bCs2n8aRSu664BUdV2HipCtHYcA88jF5rR0BpLr6AGkCFyuG7Ar158390NtdnBHlEcvypu0SNcfsnYJ7tRvZvU52b+zC8QkRX7UYdd+tPlOiEm5i94anusFKHvFETjqpvs87OEhu6EfwrNNgP27bJEz4I62gWJkFWTSoJV3+pMYpgNJ0ob0HwvexcjDKunMFmYYokC81z+aODuygDBh\\\\\\\/ceOwR7Ym\\\\\\\/+UpD2Mcb5ZR8hwF4eVAu0G5zWcLAP+0XVPj7zXgiBJsZ3N76qQsMOtEcUVu0ChJZ0oQciKPPH\\\\\\\/qyjr4YGoFslOVWvXDpirkcZ1cPjPk3tVKd8EoJcVSPtDRlJuCK0V9YJgiRCV95KAYlGDaOwKWAfAITyLS0VDBnC8WOS+7bWj51uXxs\\\\\\\/GZXYZIoxsE+DEH4+2m6wOvSrnCe637MPHyZ9t6q1nOWKXstodZfUPn5tFSGQzivr81xIehMzgoNHhghyJnpeghmmUjnsp5tdTJUsyVy07ANo3Tfp7yeB10GTrs8x6QlD3T1l+SbWPbTh5uNXXZxz4cUoodHlXCvEaniUBthdXnGhdSll08HHNIESMYQX\\\\\\\/xpwfO7ha+vqLLWccT8PITSZtsx4GScQ6EoQNI4nUWY5M20ekCUY1X++Se95tLOm4PMsELDbIyNtFijdMrxoqmW9DfP4hohZQD5V8ge7fYwoA2QmiUzIRJ5CfnzfA8Ek9bgeJvqa\\\\\\\/Z4FrpyaJ5jkt1Lf82Z9zMcaE8UtbkTFjTW141SJkQUoOdyzc9l6jgBjq4pAR1gvLNuWkbDSWQetL4nEcTGIRYGQLMLxVYQa5nJzf1KTCocqN07S07NuBO0rMWifOeTmgecA+iWeyEyPCY8ZnnvzLeK9NrIqkr2IrM9EJcMsh7XLq\\\\\\\/i7\\\\\\\/rn5yv8Y2RR9FmRBFInH9FCiEOP5ciSmaOFCYtubGXCokV94ZpTcC2h8WRajebtqdqH9UCtmZUHSWguJgYcAFXJJFAFUUYI7qq0Z0HybNBdKen1RRCXBUMVRZQ1DmZCHt2r3XZiknYUwKp+mdvUArgAOM0onROR1R7QpnTmDZZ1DHI96CjIext8oKj078UBoSTfJUqUNU1p\\\\\\\/hhLZiU3NFdU6SCmy5n3KTNi4JjM7jXNLEZLNnA\\\\\\\/6fO\\\\\\\/HVACRtQYefm3KMGWcZoWTXQSIvHOMR2LP3iZT95cnB8afb4hOzQtlFiFaU+3y73GI7NTHMOS7SdkA3Vtfu0wyBv\\\\\\\/jdU7KLcXincieUpp5mQfgYzrJPizkDBe\\\\\\\/\\\\\\\/eoCGH1g107U5yOiAvYGYfTRYRYjxeudBRNXXU4eaSA+cs3ze6aZIZkrGiYDQK+pMxt8i451Ii3jxCO6JBToL+Vjt2xpX\\\\\\\/AcP4BxHfEmg0VvCFhs2uEuBhGi+3crZkhRfcWPD1GB1kunzkfSUSh8EnLShC5gP2KHsNo7iXcrs9gv0pKz6aDOu\\\\\\\/CHb0LX0TOQQXvDu0dNkMT8fXeztONgBzrG57NCtAdh5zhaXYDUTYlPbZSC5YMYEWrCziUGN\\\\\\\/Vx7NeaqsyRCaGRDYyC\\\\\\\/qeOU18UeoZ9t7v3xO2w+GuuzGvknIbPCHhhzKRjOHaDrei43EGbLLOjZlD5wUSqweABtrdMxY8CiwxkF6znHanOJvvgGB6JcPTTCqPluyNcqc+KoorOdZXKlaz9tc4ubKwOtZprfCa\\\\\\\/HtMSirtQgYpf5RgU11CJWxwLDhMBk95AeBtEYn7RuQY0PT4uCLJs94aN76oxH\\\\\\\/5ztD\\\\\\\/6lWhJtgbBju9UFRPjSYPxVI+9MuBorWWIxEBRdRNEkC4WgudAGqsgEkQeCKW7mhxzYBBpFFgz3nrO+swj2opOexMFG9Zvb\\\\\\\/bA\\\\\\\/GMUGhbrX1mnIMofihLaiVWw3ZGxQXFTYSmUX72m9jZc00ftwmZ55\\\\\\\/kYtwuhXsVyFkJHwSOL0qvJ0Aj5ldBo1N9U\\\\\\\/S9IsMjUyAP70F7SYfZUcZKhE9CK7PncmFrNfBNk2R0OztJaJEXsQSeh2hAvBZ9FF91k2\\\\\\\/chNjEs+OMpIarh9h1\\\\\\\/tP0\\\\\\\/BpK7JzbMZG07VdCdEh2guZJfqd4JhBbAHtmScR+AOiebmUveGGSXMtVauksajXX\\\\\\\/2ORYxyD3Vs9ZlJyymprz0EUz2+bImq96ztFX4MwSmB4yjzWbvnrSLL8pmu5jrYNiMSdtweaUgQoqSM\\\\\\\/vHQZoZ5GK5YU5kcOQvgAbj0DtjdkYLpPaRmEpdYRMJgkmSY8vTL5Nvh\\\\\\\/kQEMBTtc5+LfsdPqZ1ekI+hGzNvb6tChpRRddSwrHcXRsu2dcoQjoK1gknFVcVStRNCInzaMknWxUHugCDleOCrIS0Rs6Z71mY2VolY\\\\\\\/ewmLtg4S+1MjjC4wH8QPHFxJCNcBsUWruwJMZJGkzY8XgpB+JrptI7GthYdIVLmigcwyxZR9OO76a6icsdSRwx32QgJC+zLv7zxcnTNWKHfXe6eP9bcG369HhjPB+lux2cq\\\\\\\/BYxGMsrRaAlR2IypdzLl5GcG3cJ49pyWDELEOndkR3cQVcSfcV70ivVQvY66CJORKZlQ1HwqylsvyKf7QvKgG+2TcDA4yvgjIUzPX\\\\\\\/vcAerabgUj3ZUzY2O9oeMDiYrJGpNsQkoeYCWriblA2jib2Zo+J+GtoPzl9UQjo0baHwhMvE4uiy\\\\\\\/eYbqaNWVbaGmFZ61DA7R0o1O\\\\\\\/\\\\\\\/CvDytL2A8qe71vsJ2Vwh4toguj0xIi6U6PQ7yYrsu9QaucmWO+DY5PTnqNiFZx9\\\\\\\/yel5OAHGr6YiF\\\\\\\/wVoZc7mXQ9FFDJrvBVQe7nkkMBfpj5qH7WxRB5n5mexfUdhBDcskhQNFxkTzMhyrClWO7P1xHOq9xNrCq5ua7W0Xf7r+qTg7qZJMMEFOk\\\\\\\/8sddZWHAO8nRqYM2B9YVOREQTRBAE3IiUKeGUPkaKiIGJLNTRkJ8WheUJ80mQ+Hb38YMs7BMi6nsVLnf5PpRZ3FEe\\\\\\\/HWU83WQ+2rRIoDacSxb32KWrDBB8jKH\\\\\\\/Vg\\\\\\\/yg8xi+M0zeWNrzsqt1aMUy9\\\\\\\/dZifIH2\\\\\\\/PzmDfKMZOTpTFaex+xBPVgE2oKZGOCjGPa38\\\\\\\/vrvIF+zfNSBTyov9L5CgprFLR\\\\\\\/OjXhTls1b1qODEdXMcw+BSmGyY1qxEHtmm3IkgRlGc0qy1crpUemdlgT4o8U1fF8+KAVkuQjGj3JpQGzTCBHLKq0korWGYo07uGwlLTYjx7hh9nk1qfroMy4SPBMP55o\\\\\\\/SHYJZP+CFS4wg5oTtgKG8Whjb1s0whHt\\\\\\\/eK7vjLhhjcd6+AetDBuOglF7EzbEomTqetqqpNYljsNN6cYHgjLRXEalnMF2dy1SX0885epoWcS6AsFASTOnGdsfj+n9OeQhVMvSaKG6jqNR6ckoo6hRudpFJB2t+0sxMwFjnoSZjK4OOfy5vk5rBQ9OJ\\\\\\\/zAsKyVX3MN0Sn89pt2MKPeW6jlMX\\\\\\\/Iflx\\\\\\\/J8H1htuuNFulIU0hJqMt7K49pqA\\\\\\\/6OAvlmzKydIwUzRJhu9LuPeb14G1iUmZK+RghP4dXcoS\\\\\\\/C2VvBIhkhfIErgt8Ey6zs+ViX2\\\\\\\/0Bz7fsc\\\\\\\/KaqEnpIDd5LLERBYey8v3nwPiwyZYIlMghtSMGAF98cLk789vfTegTYcaA76801lB2l1pjte4qXevhrMhSta7u2G\\\\\\\/gneYaAxvehFdMjUyvIC\\\\\\\/7e2UdEvW7fasR2AJAtKLMqUa6a9VV8\\\\\\\/zGx5uU17Jb5VXyYGJ6fPW1gDZ26HVTkIgk0qHExwNTo5HPIxLUxU+vF2HErSy3Ar8DHKfFUpBKiEHocTp2UACFwmUoYX3jPbPDUtEQ76+5WUf1GvutyWzmyiwgnnqrid+6RTuXQ7ra+w\\\\\\\/ExVzpZsbo2U6ZmauTkhyaTZKMRTCDQfUbec0xPDrw3usW0c1J5tVUybcxKFYh1AZ4LbFmuhuDO5lHxloB4OJkjOfaCSgYLhrnRBCbHO1iyh8XaiotVNYGjtEdoYaWAV1cR8qn7NoutkWLejFnYBCWEIK\\\\\\\/EjWDx7BibzLwgPKiYqQelIxKcZe7grMTDF1mpfM+pAEVVXIGGmb1rluy8iizl4LKy05SmAqvYq23AVVPxGzBfbm3vK77UWEq35oyerMaksgCKAkbOGUUZ2wg9DkOtx4KEo53toa5CNdCOuCxxWFHuRjSm\\\\\\\/simHd+\\\\\\\/l4uNClOvBcD13A+u1ErdN6dlt1YubkGo+3pVVArbHF3t62aNjPtXGtPrEQ1r59B3dsMTfOGiggINq1Vo+NTDAozQF0Aqn7wU8uZ6Xm3s\\\\\\\/sDN450SDIt86G2g2lLiZGY3DUPjb2w5zV3W1bRRmGzVYwCDlTK1iltnqFMe\\\\\\\/nHRqiYT98IkhsUO2sp0h4mVm4bIGXfIacgrpSeeAbMuRV5R0\\\\\\\/VvQkh2EJI6LRL8CfuJ5xLhLuaJXEdwptkrXtMf0uh\\\\\\\/cUFAwIzKz4mWJxvVfEvf3yJ1kwICuFYKHBysbMYeaB+4IoWVlar2btRgDzi\\\\\\\/vgSzdtVz+gH0uSHfLCGGthmQZfWKcNJl7OzHxeRAhX3N+uGrumhBCmR4bPzQIwFToniQCTngOnbbNqBZWUUZjiU3nFjiGULpO2E3yWDXRVM1epRgbmmYESn0wKH7KEGiEVaJ3iRY+5TK2+Ka\\\\\\\/RMOOl3rl9O5HBxhUONtTz8Z3FuEZ9t6cjeoyjn4+HKlpcewwjbeMOl+s9SJj\\\\\\\/F3PI7xXxOIkAG\\\\\\\/2qwe48j8oOU+0Ma8LLSuUNZaHRgUBxQeMfKiFiahdps\\\\\\\/qytJMrZFd5LGo+7hjmFrXKpO6gqyHGuHB2FQOPwnxcGA7tyXjhosz8jne96CphPFz9LE15Vnv+JecHsJThHzlbNRQX6tEp6MltIJtHuA35bR7XkZRzL1Rv7Ylxe3fx23\\\\\\\/URlqAYxiGVSLiwUwGBdd6DV7sq8Tv4gihoOnAJhzvwG1dS0N8THpnJWmV6Z5+T8Q9lf0FX8ndlmimyM7Wnvr03rsKZ31U5l7wtdzWYhFwPJvlKX1q1ZjWaoDV9sHJlKWmskNuYxqawZiQT9B5id7tmK2Lmy7NFMHzO8NBdvoTGhTfq4ZKF+Zw+hidXn9oi3pZQscL\\\\\\\/z3pfykzR8tzzRo\\\\\\\/d8h6VXQt6vW+g7mKbAX0L5aZ7Wvm806Go\\\\\\\/F1ER1RPrwF6JuomsPu+SC1TxnrHDKt1d9yIEOKEKbIhF6FbSVe73CCnFdpE4Co8GBNqw116h1RkB+UFxsqvA7AHY0AXsJyOaIZ1ha7S4gWBALrv17OSsOTJ3syph+3vwyWKZBCSGG2cXEPkczHzcU2+GKFahKNX8mrCeNKHilqgwwOQrx99JL+LKsCPxNDtB+j18Ixpo0jRVdoQa\\\\\\\/8h3wVk0buvsKD4EQGGVfgrxI08IbGnmpnJuS8OzBvhnpgMtfaHUnl73hR0+aBhGLkf5rfQkca4EUVuPHt510zOZPdsvh+PGUifMI7O2P\\\\\\\/TnZEsSHec8x085WuduxZLYALjNbjpmmGOkKLcdEkQk+NieOb\\\\\\\/phzfIRXktqE9rGAzO0ok5a6h1SlAMo6eVPys9W8JYMM9yyzts0E6RfOp2IvZmjKIBAhq3Ru3kXLP5ov1B78KGEE0x68rI57uecoOspxGlnssa5P6Mq7IwpctKpGj\\\\\\\/5\\\\\\\/uBBuPTKSDYIATlIInBvNIUfKSosGScHgw5MbUAbUIxdVD6Z4oHYwqpioHXgKQGtJkpy1sxOm9qR8UC9M5LNTvAHF7FDB59Q\\\\\\\/f49OkbxokdRbc7qs2ZfPy2Zf0q93Pe9CSIfIyJa+UpJFN3vbJabL1vXSg9ps\\\\\\\/SK1J2G23I3gyjAdbXp1RBPUorKNAN3+56sir9StfDD+tKCeHzsMIZ31k5+tvqPvCuSt\\\\\\\/Onf0UYqRw0cWr8YGjtlVzI1QfM6PrQyDplu+TuG0dQjDRiXjD3cHOXioVuuGaDsEDlQfUMeDA6IwDGxDv8YFMh4fAWlwzevwpZZZPEc4twnS\\\\\\\/O\\\\\\\/7JRa0+5pbJkAoPSV6Z2YBBIRHuqbNCZoWubBWYHfmPEMr2KN4IG5pHRkE0YnaP2tLV7dBoBqp6TiyN3hlNfgl4YYqKEQ7OjXgah+\\\\\\\/IW2i9ZPiifM34m9K3IDs8u8L1iO7OsCCmaKh2Wd4sLQ\\\\\\\/qcIseECMgrOTJN6Mk+qophqwlCMBID47J6q8SOjpZIGdnDf4mI6xBUckD1vyVXwAeuSYdcu40Aauk0XfMA2nyiHLJkjix8lEdtfFjeMEkOFHOPC2Ym+ETQb1U7O9HJRQZfq5Kfg1e4NS1Qb+7vOc86BM8WeOnPYnYhG8jSXima91gjB6TWimF+n6Eet2b3xWQbg9jWrHYmIs8WOse+hDB83QvlkmmwlJr63APXiiBRrWdjIJalggeZIQcxsYRQOlGp16SyaC3OgB8fYzcKM1qpWpr15lak4RxH2CBSfhjWb82NH5oDYy7CzApdZZyQDN8\\\\\\\/D1pbd41KkWG8uLAjZLv9k\\\\\\\/GRCYSnMySB4vzqL2kbqlqn5CTT7yVLwuq64I11Mw65zjbzIdgaI\\\\\\\/lpWph95+XBxCRM\\\\\\\/nH3bZx2hG7OPP293hLkcPUv2gNAuzTNnKO3WvS3riMKGBrvWit7CJLin0Ohed3HNsfBN\\\\\\\/dhp8WUx7usgRfeMdh209DIOArEdQ\\\\\\\/FKsZY23V7RwCMgZYFwMHEe4c4OrID7DN\\\\\\\/U3BFA+oMtL3gh9sfoEfsO\\\\\\\/k+9zlOUaTmfBKpJCY9uEifG44N\\\\\\\/uS+Mpk71qXLgLHhSvwSw3rU3VSVBveVohJjsthE9vfO+BgxY5ua2tA3wj1X43HlAfq7eq8ZE9p2oy5EIVdjCDDjO4gEnAa0BfjsT9wCoxp4BX\\\\\\\/w8V\\\\\\\/P3wFv4IHTfCtz2NaGzj054r+YCQSW4TbhkouBfIefB82HTxhzY4O3\\\\\\\/xQ6wervOlB0cMqrtNcfMwolk1qQ\\\\\\\/xu7dcNy\\\\\\\/XB11+EHF3IwRtsnqE+KgQNtgN1R0nfryI+M3OfZpcNlJ491NEYCT81GieOuDfOjKPuVJLXIK5svVNABQWRCsyyihvaFAgwAlo8mYQzQcdRnIUHkp3VsjATWvznumnIBWiFGxSW0CXCdqP\\\\\\\/5X4R3yHqDGSnSCLRwvhahW08uLSaRd1obIbSQ5M5mNKwMYTvpxnM4oitMxGLthQJpaAn0\\\\\\\/ryAdjQ+orzmdGd+aYiCp5S\\\\\\\/noAPNoA+ZhcnR7iQOXgL\\\\\\\/XU56MoY08sjkgUuFOlQANTfBLWblHdA+XrlWefzOaMd\\\\\\\/ZKnHfkKSnlVFO3kGAtz5LKaUZfOikrmVPt3nt3ySEuxyC61shlgBe3btNfLeJH2PvqVW+djrvtLyQyo2mszBbXmmO90ju6CCVASZih8XAioLjcR7PlHGoLMA42SikCeUApOnZ2hOgBskzpvEXAFhhrDo\\\\\\\/UJEh+4VPaeCDdyFkzF3T+XUCulv\\\\\\\/Zw4PHxIjf9qnWTxv4dnotAlPgQYepr4IlY+II3YRXZ13QwjTt4xgyklQKt8V0oi8ivsVYCdYYkQYwm5Ki9xdvgI\\\\\\\/L82U\\\\\\\/evQRiWZ55hH+HwycTUFh71kZexrwVIvrDQKnEG4ZBclAOjFw9mOTZ9zd\\\\\\\/hcnKXhHHL84D1r1zgWbAcGdH+pVWf8LwwUNqLRcSl\\\\\\\/1LojPFb7HBBtRnakOdG5Egso68\\\\\\\/2adA9Qvuxuy2PKj+pTKcM+WRlE1QV281u46QN6eiz7lZSc5sEtAb3OU2pbvbult0v1gw8vsCIbqTB9B9908THv5Tc8ckz\\\\\\\/LosaQyL6tI7HBk\\\\\\\/VXwlmrs5Nc7Z6y0M3eDNUVXgNMR6y2\\\\\\\/U3CxFVhK55qQuuzCXqlPXhZ0aO3TR\\\\\\\/9gqdFqjX18DPokvoa4zxKfCW5G8ydBW1QgFI4dS0oI5oaIly8YcRgjpYUdcTAolPdc+rAZYB\\\\\\\/vTCUGaGXd+4QZd5ak0AXFMpZcSyIM4CfcO44u4XUH2mYF6in9qQgxBIbTrfT1GqXDzicFmTMZoEuX\\\\\\\/zIN1qvP\\\\\\\/B4zwOnqvRM152ELZr81sOf2khhKOU71hpd2TBbwsngjuvrIYEWsmDhWUOGl9qs\\\\\\\/colrMN5cn\\\\\\\/HSjMlavsPLo4DkBJ2r\\\\\\\/s6O2aHM\\\\\\\/6p6fBcbgI5Trbloryqn58miRZhaAzK7OVMRCUckyPtUzYy9eyZfRdzVLCpxnmm+86RlUbBTR2aBNlMuMFjE4EttgXMojq1m5itcykIty0qngYH+VWHgDZoMPx3fKTJIHgMt4X8mOlDGw31oNSbGHLGIXAGKIVWP4T26YalcgxeCab3HVseijYDQyDppBR3+SykhcYeRrJcFU9WPc2J9V20p4l3PyijixgA6U0baYvlbULnN2QWarQP+DbiP0QQ5g4zMUCd0syMugzuJ0PYD\\\\\\\/D99B+5Q++C4RowtPCPDqsvAuc71SAh9GxE5jvVoABZBid9BQYtoQi9HBIwIJCgCfCGC0BUlmn2JkBdtTQTKTwvkX9G\\\\\\\/bl8KJaNqR4S+\\\\\\\/vy6oIuUIqLfJ+fyARbGW+mE9vsvJNah6wlI++wc1Vaswj1JW2XNLz9RM+77qrJgs1TrwjiLoFj6B2Orr7NEcMFGjH9Cyom1QbYLUgQ5icWCILq0xDKUUgPpkvKL3cNGxzG85XL0HnC+ZuoZ9bu7ncRjyYNqAj3c7cL5MyxHpByFHKv00LheHYZO+kMrPXZaIs3CY\\\\\\\/cutjREqp9LUfgN3om12kUcPJRc0YrilZm9UTazGU\\\\\\\/JD3ao\\\\\\\/7NmbaBiYJFY1PgDCv4zBRz\\\\\\\/NqE1aA1l3l9SE7FsZHox0oZrmnkal90jnwHaFjrKK8PccYJF8TpTXKvUFVkqMI4hLA4pkO+PG6kC+W2ZPebOjRww1xyMa7Fd1AHdsW3XG0JkSn\\\\\\\/xru0SISiycqJMDUjisX5dq4EVVxGA6LzRRzI6KTdzHoifjNOx6SUS\\\\\\\/YwLGLWZzSL1Lg3vL0zBI86zRzQ8Vk2f57vPxz8KzVoaKxDxHublDbKp87TqIOLBar70YCOUOw\\\\\\\/q0o3TWuvDeUpRfUdLvMcRB8FRF\\\\\\\/MlwYEjOozlr+tBUCj2rcWIiBnv5tyeMoSl9LvYgSNOu4UxU+hlnVKQCImY62Sm3oqO1d43LIOWCrBK7hCzVA5tmMXsrZLpcs+iNUaCYuoIKc1sU78AtVqTRTmFpk4VB4yDgzM5LHucvhdB5jVLZzego2E1y\\\\\\\/cT5HcNmgUGj5WHICh5vAjIx3JbOq\\\\\\\/72RnGph3dOQenUfwnfzpG53gIe\\\\\\\/sc5Y31U10fLnZep0073WhpTIskdPVqFtAShDUMPNvOFTSZsS9GmB22rNvnBVsiNBc3uulH\\\\\\\/RvEvR1ldKFJ+8e3v3KS79TxYQZGyWxfe0u7BHvjrkGAUFBivnk9MHrTolAGsFp2uhUtCoD+BBSwZkhoneRSb4RETO05GzYgM43z6eFQHxEyTxNBN\\\\\\\/L6HiSj+olQ+\\\\\\\/8CqWjAOb0xOT0BF7DFQaEttpUQC+bOsqV\\\\\\\/klRca457uFCPh\\\\\\\/ReuyLsgzLf+apIh2qk\\\\\\\/C3a8qWam73AEHk6AHjLMbr3eSiCN1vJ7OPGP7Pbh1QRHXZ7BIv5QxbTCEJiDZ3KYBmp1svHsyELCn5B1C8zweazNnS1k94+DlIxcMnEIwIMRAVhDvUBFK\\\\\\\/ZLuBrtBfVHP94jvTsD5paBUmO\\\\\\\/ZCMD\\\\\\\/zKtH+KmfS13XafuL0IumtNnDQuPogV2vkaW5MSthqoBIgkb\\\\\\\/7IpkrWKF3QtOLNo+5NywHeb+nH\\\\\\\/5Re6dxPPZiaisMsux\\\\\\\/6KTgN6hMloekPYXkJcFEofmgjJOyUaJExuIq3csJHpxBwKUno1I+dpBJfZuamLms23P5S1aA82N687CY2FXfgRQQLdxdDWzB2Nknxulr1uAfD2OtbZL+B1Wmn0e1wn\\\\\\\/Zlr1hvzFymskN0ZW15I3P1+Spl4d3pJQFdTU8ClgucDRxbUQISJuavXKkhRtHLa0d8p9wVp2SNAN7wEH\\\\\\\/rkeDh6B+70Fa6P++V3om9wGV\\\\\\\/6+uDONKZc9bCq5gsFdSc3LKmgLjEGr0G39ihKGwKl4cnCjKlTCSZ5n7FzO\\\\\\\/cQboHbNqg+kBzEIFYyuBHOJVnkxsK4mYpHaHtt9duQkxob12MECQTUiYXS\\\\\\\/QSzCwNEWLMNFEzWaIf8jbgd\\\\\\\/AleLTyRoYrhBJsBK0V6tgnuYbU346r+bX21BodXyEViuaGbFqAWzLCWJWkD\\\\\\\/xdf1YnXJGfakLwN67QG20CsasJOuUApJ7\\\\\\\/klzX4KiImtzX0tsIxnxbHt3z+QvbPze69xKcgbq\\\\\\\/uPJDQb9zzGZwGp3eh7blNiIjuMA50OuYfk8+UzSe+21X\\\\\\\/O02tW8exdrmFnepwyol+PoJtfse3Idza0WbgbU8QFMpZrctwc3\\\\\\\/bsRaoRcSc0q636jAG8gEat88KXPyCWalfDAyYjHN63uUzOweMoMgYy8lzJm1MbLsix2VpgW678N+7npex3de5q78CKG780HB6dGT4g9+kevXJFyb7JDU7HOP6j4YkSKi9hWOwr5gXBBeH+to5p1z+4kxUorkmrLJ9vak1ljpRaypmfT1IMpSGP4ZmZwM1cv0skPZjEWtsuBtCBrK6CDKuxlUYawrRzA+q\\\\\\\/ql9QPjc5I1h3xTNc+9SSTG1llUeXa2+GIbSkufHIHmuMyhDRmo8RLm0DonwfuxQQSgS4RsRE+AbqAkBH6DLtP19qhZRq+a8Qbrv8B6hlSaSgyY9TeAZV4ELerjRgcltnoHWIGqg91BPJ+CyFgNSvZiTHReusc3WexIDW7d8tQ\\\\\\\/nnnTStyawwoulrQvr3i9HAGfYcX+P49ncx5i2lX9ye0AKpwiSTQd388OxD1IQV2A9lnHUarQ4gM4E3V7thmGhEUlXZ0Q9e10HjJg2VWhpslm64YtTVCXpck8ZZcsARQTIRgtbW\\\\\\\/PtaoKi4uKqFpmnJ7KH1xeQbznS6LDk0RSQOQ6g\\\\\\\/\\\\\\\/Eyz+cCtYwVSNNk3eW5KAqNv73NBLD9msleNreIGsD9zR31UFiE6ALXY8P1PcONvDCdhYj58MycbV+lqew0qNfaFyM17AMvQEJrB5ogFOwLRaXAW5xt8sHiYyPoAOpKKxAj+JR00Ed53DLGMEkdDm+ovOWLGRKNxGLKvRcX\\\\\\\/SnZm\\\\\\\/uAXzibqo8Hf6us2CW5o9djPcSmlb38IQnn3I9TyiGuf6x7amisfb0PB8YIJ79aUI3CQV\\\\\\\/QAwc1PL+zWHRPUj25eW6LClgpfG3kHNhlQVBD+1urQZoZyUweh0\\\\\\\/G\\\\\\\/KHzFq\\\\\\\/JXfUUiDnGF6E1eAG4++jb5DwlTkCZu\\\\\\\/G2XicA\\\\\\\/Mqx7yGp9Bf2liuATyanjyeGxuuJAn5lvZHHVHl0lsKZ+0Gauk891BZ8bfrEGaIE4f\\\\\\\/EJDmXTJ1DQwGS3k9eJ5qnoGu3Kb\\\\\\\/5GwmHtFgsXcjC3Xf1aZKfO8yM21r7PXctmhmsTgSkKhPQdnlDaPav8LHn\\\\\\\/FJatjpeQGWrr\\\\\\\/2zKtcPLDNUA063W2SiRLqRfvW+rCYZBe+6WH1rF6dFVKlhxiWKBqgmpgAbmIA9rILMhwR3GzZch3vjEMjzSzBKH9dZ8tW4mrYbtiDchb3NCAAiWA53+NlBxFvHahMbTKms98Q1Z8gRiCmaXgxiCxbmFHMJO3QokFYM1RYfQ0WEtVB0Rec1Lu\\\\\\\/6EJEyO4YaJkCbyOiU8Jy5ORJ+NwRIQbtI9K83U4K2XVqLie2oni\\\\\\\/BKC7HsRs55AMstEu2Ezz6i839d5qN9MHM5QEn4H\\\\\\\/e4s\\\\\\\/9K+j1X8\\\\\\\/6Xf3B95fz9dAJ7R51rNQemuK\\\\\\\/YuMyMvpBQuhxi1ojckN8V7krFPfDXRnsGepQJpdCYM4wwsNLlRj901tXvo6xeZHZYGMRf4EWsCDenU\\\\\\\/UYp6Y5miL1Ati2q6sqgqMc0zhtITaoy\\\\\\\/D9DYfkvgGDawfPxAf68WE5xOnYXhPPh\\\\\\\/km18zuVbkwbnDbtBnBTxFbr8JQipCM0H9fkyYdAblHo6TMJev01UFbergS8DNqRcfaxmiMQf1hdRveKcPCtmj+m+l7Qrk4xpPDJ1en7dy1eeXfHW8atf1P5ecatMCwMbLIskvu26GfxEG0o7\\\\\\\/M+zE1FQejAuuSy81lAfdz\\\\\\\/lu\\\\\\\/tqayjFMOCRyr3Rpg0Q6bUdXIRsHzwRyzne2xdDU7aSQwiRv15HnpTu3cDoijbuaWPE2ewo82ftDAhOe\\\\\\\/9FXk+PiTURUqa8MXjxSRB3jX3SlGbHuhWrQMpAOQpTfq+bJhyfROvVb9BJU\\\\\\\/w\\\\\\\/lIIrCPJwLOHz9v+x0BfOhgpMAYxOtQIvcdFQhK93z4zayXAgoxEXGTjW8wUa+fj9IhJR7CFuquXdOswZvWkNewpDuqQknsHLlyCfwXzhJ9\\\\\\\/gV9IJlZKzmFahHh4BryHm7RU0VOtx4ZAQU4+ypOIXZB\\\\\\\/hcucz4+gwnN3K4+iNphJdiaKxRcS6mG3YnuoNzb7J96bVlCafUw3LmNx7px8hiZzhrzdv3F9ieexNnXuKk8A3AJKc+zsRdrwPguA40DWbHtMI1GxrgcGaPxlvtBNFPEoMxe1jnfJv9DvQUoW\\\\\\\/u0q\\\\\\\/Y0BKNLvxfpebLOvhVtSdgmtUnWRl+P\\\\\\\/qRYCgxPhJBvswhy2LfBc75n3KyetroS\\\\\\\/U49TOGF0fZ2yAuqO8ajybZZj9IHASAZE\\\\\\\/gRTKz0kZrSsJSphbp06rYEvjtgsJU0S8u9uzjbvnoL4liWzAeKCMbvVa20aoA2EOZ9+TMOtgVLoQ\\\\\\\/8ujOSV1YJ1R1Pwohjfsqb0or+prM5jZ8\\\\\\\/xem8xCy5VP0wo4ieJNHFW4vt1l07D8OSbIrK+WsNVzMDolwYVnl19zX95VXQuYZedx\\\\\\\/20rx0rUCg5R3R9HVKD0beA+uNdSl\\\\\\\/YRbVDRLsi\\\\\\\/4nu+jycNXZey9azaYJQRaIQxDtQElKGgKBBkNu\\\\\\\/UbIim8h35bGURYliNVoZSCdT9NPtc\\\\\\\/JeHntKrgYnbBuWxeFPzQdXvLlD\\\\\\\/ruB1AtWzs+b5pWjldvuMFgrPu4+JJCU0KlbboloVr1NSZsr+mFLvfwJbKBICXk24IRiM2k3Be0GfRNUr4VWdNXNlkffhrP7cGEdkhFbLYexKFR4YIQuELB\\\\\\\/svVXj134pQ9F6Amqs0zQjm7X5BQt7cE3LKA8ESJG9OnhK0beruzNyPwjNziUyQ05ic1j0xnJsotXUUc4goaiFX+29FoVtBG0olVjispbmJ2FLgOSgrfRUslLdIgYS6K+RAvgzN12jk0uEj9YX7LX4O4NS+cnM1A3AuPLI1X8Dvdllv46CywQlYLGV1rWbXQSlSe\\\\\\\/sLyCzjYtDWJa7l01phViaSA2glGxxgEOf4p7c1xVXOcNxCsf9NpstuuoNLuKs5Kw9QJs14eWPD5n\\\\\\\/bZ\\\\\\\/IKailoUzUHjbpf\\\\\\\/\\\\\\\/sIZ7dIlpp+5uI979iyQpUPRR7L\\\\\\\/xjLLflcjhkCsienzRAAXBDkz7wxzEvYJrrg0ot8c+1kW7JHSR7ZcXcKB0N0LHwm6447GoJVOxHMxDshzXHbMZu5Qn2sZ0niFve8SHvPSYRa8EjP9mx8x7MHF9pkdwaZWQLX8in9xr3+l+jHIlwLAOotEaZ4umI+iOOJ0KtYyq9GCwTXel0zNtuE88Qof1lBWn+Q1AlhIjPf+u4YqizsCMvBleVNVzkljg8eOG2R\\\\\\\/bWcktxZWPx8YwEOBWFIMKg0txf\\\\\\\/FCz9UTIeN3M+QvGg8warozZsWTRp8GXyCUQ58ajcBsXNGVaAuYKGRPNZfb8VrJ+h2FoCFf+kxTsls4em6cdiz5PnSTD+HLGM17PQISikhnb65w+vhBFygq\\\\\\\/g+MMXXyZne8hCdzuG\\\\\\\/Box3QDjyX1yTi+cjhzHtLUa\\\\\\\/aFBr6P8ToZ\\\\\\\/\\\\\\\/JsrpApbb0UAT9M9DLmHubHoWLliSzZ\\\\\\\/+qK4jU5y1tXUSonPY65B5yGh7b28dpwEa+3kHT3PJXXhiWVJP3oGcBiyEs8HtAd1\\\\\\\/Mhumzay803\\\\\\\/JkLu3LEg6+AvDj+270KxjB1e7YWk9cqgiChw5\\\\\\\/LuRxQw3C1U5jTLHHEBFpQYKELyAh+oulnA3Oz0U+pmfT7C2vhzj9SYbHcH6vG+W7BFkB69iCkcLRBa9pkUbL8fGUgFeUar2PHIy17CkH0ygrtcbhHaRBOYDvuvmdHZ6Y2oWEF3fiZ+QKi2zNlvbqPupBqUyk7UVrReS5PGTl\\\\\\\/cZfsj98GU23IRqLiMaf3y76AuJ\\\\\\\/3YWMX3Cr8CezRJltIQ0E+zbCBXNyetS4d6AV+2+d7Cf4coKSA8x6C1rBwL6vG\\\\\\\/UDbN3sz+JNeB9gILJ7Lucfbqyhdmrajxc6LJ9p934Ltzyk6GivHhYK9QwBhHLxKOK734QxseO8Fmb4654byl\\\\\\\/e278oS3WVUw6Aihi73Q6kmqRIScqizxFknjRrxoVyhBFdDd2eyh9CJq9ztZSUxri7KfYOT34YAF\\\\\\\/RLKwwvbujV3uzvbxb8GeaULd\\\\\\\/0jxkYczYGlTa99gaMNv7z+YeqKPS7LzRKUD1B6xvLZ0A0zyPVlDQUhTA8fLtXjPBSVZl9Rx2hvcU0CyToHcQvAtjg1tUsCTRXMPvx+QbWXcg88fLFFf6gdq6E+yGf5lLeno4Hs9q4748el+3D1nvArQYtN7INoT3kM9VZz5MMt\\\\\\\/lEOcO2vE+it6Or2Z\\\\\\\/k54ryMP9bhUmIkANe6ydaTc2sjgVNh7rcE78E2ADPuGuuX9cqQewfdun\\\\\\\/8rjKGdtyC7YnGAQH38HLKT2+tTKhS2Yzjim23rXPOTRQi831sbKxBVeOLJw8k522uASYMv0tnL5vRWup5XaIjqVRWKEvJm9Qd5fmkXsdS+by2W44FhOsz6p43UqBaFKO11L1jT8PbYX1XksyWRKpiuMSTPzf\\\\\\\/U4+cEfdVJUnQ0BSkEaJR4GkV5\\\\\\\/DJjhtqRbvoBKundANkVJGosi4XyXbxM2thMerfr01j+G+26OxdGhENyuVefjW0kG2YPXdmsFo\\\\\\\/ONmib+Dsvf93NdHILqkIhDYlCuwTF1FbYGzSr+gP8wx8P71Um\\\\\\\/xIy+cDLj2nHOOiyaQm28von3pBxsvo1Fl+2znkxWYV0VlJWTIlKN4H5u9g8lD\\\\\\\/sQGG0df9gu8OLYAuJAha0ifsQntea4erjlHIUs6iURSnwCTp1XSCaEVNh1NL9CzB955ygqqFiVEp40h9zee9oUp8CsZ5jz2Ys0o9sIerhV2RouN2ICqTxMgZZwLUDMEvczNwK27yQ3HPzwKN8dvv78RNHbAFuGKnTZK\\\\\\\/28dbWzF64j6Xaf76AbAlfG\\\\\\\/raK7ZFYjAYaVq9qfu8znwk+W\\\\\\\/RA8Eyn7SLCPna7RKBVGLbzF44\\\\\\\/1M7FiOrD39iJQzMJr5kaspj1fCKjZIlYtDXj2nQ3sCTmuokM404yoj3Uv7l\\\\\\\/CyEX1EZtkkhTfDir7v8eDJVvY9NyDJfS+0Nf2gMIfNv13jlcaW1FTlXmNt\\\\\\\/DD4S1t4UkRf681zoks25IEyAKz9muGFRJvy3PBHnq8ZCdFevirA\\\\\\\/\\\\\\\/uW\\\\\\\/hAUHtTfeq6C+JVC7MeRwRaVqSPcuXAjU4ZzNS0kDB+GFSSUm8TCcFfxJ6IsMdz3q+\\\\\\\/aAE3qjXbiegHXTSPkjdE6NAQ0bDj4Bx\\\\\\\/omaamSad6\\\\\\\/+7mU2t119b0ArSwKPg7MrjRz8uF8PBEUFzWBVx506OyC4+XgZNK29zR8ZhvDUenw5j3xSd9\\\\\\\/+vrQ5FADbVBfqlzRxTxgAIHczbq4rBNBHLfa4vaoiiHA5oYO1zKlfCFaK\\\\\\\/T8z6Sb7BTBhSakGI3vEkQcnCCUb3x1nRduj7DBwygLIcsS3Wiq8iGUhQl9JfhJn9yMvrLnCfh46yavtScUiaUE3CIyHPWw80XNi7y22Vz\\\\\\\/XF8r01164khdIPvCdoluslHnZh2rq8hR4OMMgSeEZ8Kudqqw1ZWOQ1R\\\\\\\/70QyThEWFPGdr9pBcxiA2XTYQys8Bg2eDXwv4LQfzn+TLv80YC0YAtuPjt89\\\\\\\/ZSyOMyp\\\\\\\/xYndEvYNfNo70Gnj6Nci6u6tYYjsP6rR3HCjycPZf0b\\\\\\\/4MEcDXoz2Y42PTXJGL\\\\\\\/WxoHNNbc1kfoUH88sMg\\\\\\\/riwiTctLjoQTezU5ZE7Zy+ELAwZV16r\\\\\\\/2YB4zzdSb7UMJqxOC2jvUG6IkXBOLdydi3RviWGhSbaAatWcPmmv54RgdSIa2PCgXQsVQjQcLA7lz717H+ooSTNCilr\\\\\\\/ihFoNjmCkphU5JwN+rkxY9Nf8SSQhktEpmPIU\\\\\\\/9wgVvXr6BlF2CamqD9hk5JZnsM8R9ODsdQfjDtUwPQkhEBJW9L9o\\\\\\\/Y3DdvSe4zQiu+SvU1YazrDqemj+OdOCDg3rdm+LYtAFD8wPNCPr\\\\\\\/uHvxHhv+MWaWMILZ0OwOkqphzyfRTb1\\\\\\\/wYqJ5QB0YbAknx1yuqGn8m6\\\\\\\/F3LnPdHxtnJALd9yFjqz5\\\\\\\/JE6NK\\\\\\\/L2bkR+YkQBsDCE6qlK+nGE9QP2swEFIcz2wBpGAPwd3K9tIV5mtaeUqb\\\\\\\/7hICRfBJFinvoPHRTdZ1SmnfM3KQNOy\\\\\\\/zW+ArhdfDCBuGA7QtakayErteJEHvOVQBZfnVyvNEOm97Kod9mDlOtuFEtzJrnDDhjEova8L4d8EBvb2cUewVv2BKc2jVwuG\\\\\\\/r3AEXspdhVYD8TFbTllawSX+8QeAAlpqkL++Pr5CsDSelN70A6jQdkO8oX4PLpYFvnSBQ6KyVhAgOybOf9OC1RXWA3DVnT7HdMC+GPa\\\\\\\/v6hr4c2\\\\\\\/inNJbHNMkdIXVxxmCsMM0kyOmuX8EUjcDq+f8rmcmXtJ6WiEnxX\\\\\\\/helBvzei3hrH1KMMLnj7l+KyK214EfgLOaDKv44UIIKYypdIxr+q2AD1DRi5bpMZKqcttcyqq+N6NE567sYAic\\\\\\\/EKcNqHtik43IGi2Y8D6BRQTKxXSiXLpO\\\\\\\/7CF2c\\\\\\\/iTZMc8evjjA9urm0NUiCH0LmGHpQbctRbCEwk+PTTl2iry0V8pL6UD69g4Z3h5FHxkxSSUCx\\\\\\\/pcdJs7SOLknMLNBGno90mB3OnkQ32QjScFjXW+BhreRnyp0SmQ8ywTI5IGP446GAbOBkfYwJA4Dhfr6qrka6jpTHOb5vjoJgsM33iWLqWbji6xQ4WA45vW6E9JQr5HfhMEFSylOa\\\\\\\/jSnM3bXxv4dAafCeEB6X+81ufkZ3wFTJyrMQFmduL3yj\\\\\\\/xvcdPF7z4dZCvVVJVHOzxPJ7Y+0ORZ8ZP0jE1UJBX5bMR0zQ1yus4W+\\\\\\\/mzF6QYEKUqj+QPdoM6pfUPay6AtiA+JeSTG9qrjQJhiBCaV+gkYUrik+8uesZbAQ12+a5\\\\\\\/i8crvvR7N+vNp8tIxm9KMocfof6V80hvw07Q3FYSNRvUUst2Vvbev8umRIiV4uS\\\\\\\/UC70AQq1qbRHK+MsCUqDrqCx9z\\\\\\\/74bgwYHMWHN9FJDLLwZDGzGfyVCeMHgIR\\\\\\\/jATSmrOye4Na\\\\\\\/wO3t3L5VGRIn+Z3bieAudYuVB9khULiZawaO1d9b743dAPEEmf2GLRr0ikPP1E4i7UCSzWG2UDevgDN9JYvwl8XHGH0\\\\\\\/z1T6Z7hquvE37KPf5+zcwjHEdmphWa7V1B3igvXTZX9gOVW2lLBR6OOhMEYm8Zu3\\\\\\\/9T\\\\\\\/Q+BeqwBQF9\\\\\\\/i1S1u0O7g7Q7ww4+9SKgitoaLereazaHGgGskenvAZucjAdyBnYApTXiRrOL7YbnoqjV8hUBpWhRYEsWGnfKsTpkC4CnncEvKlGDUYB6ooDyA6c6P80vBojDpDqkYSqCqaAeC+hMl+ngTNBWQRJo3aqPqI3Qojq6pyiivsdsx4+lKjbcL282er7yhGHKTB6CmI86dcYYTw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022jmg\\\\\\\/0wJnmdPGT1dh1Z3SRA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022928602b8d1dc26188df42dd857ce65a6a146c6e9258934de6af5c78c3503e356\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022ft7II5urMAXkcWAiM0NUlUFHs0hdMs0p52rfuo3Fls3Jo1jgsv56fhi2JJlz50v0wiolWxP33LcQqe\\\\\\\/pmwSdREsmd046dukGm+4\\\\\\\/dMFTWtVK5y4K13KG6HhKVupK4YrwiXQHGHVVGHE85L\\\\\\\/ccCtcRqL8nsHJIPMyD7I\\\\\\\/kF3STFR57qndXXEIGOt8e365bNVPIkH3n2ZfszwZoMl0GpT7UQ0GrmwYEX51ZmFgRxOO0YwCGV2eCd52d65G5uQqej+xNoBgBLHx9UjCAEHql9tA5fPERb4FNcZXSLnf6yvrKUQxnmEU+flp4nVWoSX1NBtPIWkyB02BCsqfo8nCd2p26tHNXGO6KTKnvf7oOemWVwOxyD7NTb8R\\\\\\\/20xAHPrZYjf3bME0P6133mGsoPBssW9Vcip9FDQyxkC+PQP\\\\\\\/yApBspn4GsS3OqSErB+JZKsaAMac1Fjzruo5SifT7J4BUApsjHHPGphR71JWQpsymofPw\\\\\\\/5oZRmOmFeJ7qNFsbjXKwoAVlwpyk4crOTFgKN1Fmym\\\\\\\/pH0CO5N7okYvPnC2OGCQqO\\\\\\\/+g61zOWV3HfbsVS32TczL5O1CLWoeYgrMJy4BZhx2IcT+rdme6YNwxG5fIqziCZ5Q6+AxjKvm6fMFDtJdHeezQYarFY+DoO\\\\\\\/A7XbJHpRN7fxh9YpS9D8XNGLGs9PiwWbPIkokKKo8S5PjbVPvVaNhxfRQdX9c03nM\\\\\\\/UDQ7GpLB4P5bO6UQdunec\\\\\\\/ntbcBTrhLPdsd6dg42QwSb6y+Z6m2IcVp\\\\\\\/b4ui9wzc8yzUbKMe\\\\\\\/II44jfSE+jsnFPMepzH5JuoJK2gzAZ9ToGSDbpeoH1F6nQ5ENAvVZfaAnD5RkYV0UQPlnBQhA5w4Ob9ctrXYlIrFjXd4yirYVrowYDbDGv60ovqzI8un6Asq9lXZ2b1BpFTBbwLNkvVQBbVBngxYMv3RSAj0Q6k9eTAIepAhl2WUb3ZN7UMzFIY5u1dVkR0mkC8leHDVGrDqVANAOtWT+SrM1sSNpDjOtN03kBIZ+iNB5YkbPUpGy2s76NWgaVWawfoFDSRSNPWCvYFm9\\\\\\\/UaHLvdlwI7TEeXHc5fWJ+lkv4vYCNRWCVebAGk9BCz\\\\\\\/bO3LMHOHIPaRh834I4YrN4Xugh0ndiLZE9hyJoxYnVr\\\\\\\/rPSRPMjsNyFQpqEKgpCuLaqQrF5SgD6i\\\\\\\/+BOgMuEgzRu0KhaWD6F1Rv8a33QDLMJLbRxLG9DtxedmnDv\\\\\\\/QnXALbl\\\\\\\/IiMcquwqe3f\\\\\\\/0mjpRf7xn1qF9LgNrNCdJM5FupUo4bF5wywyAqUwVklt6m75NZKZ0FYOFue3JzMHwP0kjFCyQ6E+Wps57vU5Kd\\\\\\\/IzSOV2QYZECcmWMIqPz6rkXJSOqTi6rgoAzhAQ9vFTz7tGbK9MPeDWig+sE1BAjPw6tj0e++tU5lboDHuOBFUfc5yjGZZn7\\\\\\\/3hhYWHPK7kNHoZzJDO33le0iW8vJUsh1cCNEbU7EAmM9hpo7MKYWVt+jGqqx5tCJ3653rpdZEf8hePKkUnQSZYKRQxaAQkH6UY\\\\\\\/3Xjre\\\\\\\/57c9Ea1RL4u+aKCntX1vGbaLJfcctQqMigV9CbJyXsX978iY1ycVJKdKHvzRN\\\\\\\/wxFeJfw0Zvr6tV1J6++EjZDBud+yrwSOa\\\\\\\/tkNLPDUnCnN9P1EwRt5QOAoDBsS3UN8kS4BSSUo72we9EgLHbSFPd+Mr+wbCyvRFfBTbevqYjSfXYi5DOGGf7pWj2GUoFXNkc5+D8YU0luGZehVA2OThXielPwTssOD2C6f5gXT0h+Q2YA7V9tJ9HsINhV95OJSlWZZX1Wnr7VlqZWDi64kpSLZ5tGCM6gpVUtDv8z\\\\\\\/Fu1fKgs2L6Y3l6UpMNHCEk+Ubu72zX1I3G7I1f5UB0X\\\\\\\/lfzhoYSvqMC9DHGgbhnTYgXKHK8jK7l6hl2qCEuEpFfOP3N\\\\\\\/u9LD7MGqa8aqdC4Jp2SG5TGL2oxwjdkGtc9qOkUn80ztiZ44EDFBAvTBK\\\\\\\/0wByLvpTJoFoV6SPRfA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022sqUoyQ5OOztJXEiFn\\\\\\\/KHRA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022a8d5278dbbd2cdb13b8e39e26e65a9201a7324873c0e1ab6b4e9af7233f29a83\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-491243 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/11/3637" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-GjpBC9bvVa5WOcfFxqLHI0ysy9bpNlcCdwD13V3_37M" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/11/3637" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-UCIUriHlPNYxbi6u_EKdCByPsPTi2R9qJEnA_fMq8Cw" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/11/3627" title="Whole-Genome Multi-omic Study of Survival in Patients with Glioblastoma Multiforme" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/11/3661" title="A GAL80 Collection To Inhibit GAL4 Transgenes in Drosophila Olfactory Sensory Neurons" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="491243" id="node491243" data-pisa="ggg;8/11/3637" data-pisa-master="ggg;g3.118.200641" data-apath="/ggg/8/11/3637.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">HO Endonuclease-Initiated Recombination in Yeast Meiosis Fails To Promote Homologous Centromere Pairing and Is Not Constrained To Utilize the Dmc1 Recombinase</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Lina Yisehak</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-0919-7538" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Amy J. MacQueen</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3637-3659;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200641 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node491243" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Lina Yisehak </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Molecular Biology and Biochemistry, Wesleyan University, Middletown, CT</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lina%2BYisehak%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yisehak%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALina%2BYisehak%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Amy J. MacQueen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Molecular Biology and Biochemistry, Wesleyan University, Middletown, CT</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Amy%2BJ.%2BMacQueen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=MacQueen%20AJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAmy%2BJ.%2BMacQueen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-0919-7538" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Amy J. MacQueen</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:amacqueen@wesleyan.edu" class="" data-icon-position="" data-hide-link-title="0">amacqueen@wesleyan.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/8/11/3637" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:491243" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:491243/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/11/3637.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:491243" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:491243/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/11/3637.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:491243" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:491243/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/11/3637.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:491243" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:491243/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/8/11/3637.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:491243" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:491243/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Crossover recombination during meiosis is accompanied by a dramatic chromosome reorganization. In <em>Saccharomyces cerevisiae</em>, the onset of meiotic recombination by the <a href="http://www.yeastgenome.org/locus/S000001014/overview">Spo11</a> transesterase leads to stable pairwise associations between previously unassociated homologous centromeres followed by the intimate alignment of homologous axes via synaptonemal complex (SC) assembly. However, the molecular relationship between recombination and global meiotic chromosome reorganization remains poorly understood. In budding yeast, one question is why SC assembly initiates earliest at centromere regions while the DNA double strand breaks (DSBs) that initiate recombination occur genome-wide. We targeted the site-specific <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> endonuclease to various positions on <em>S. cerevisiae</em>’s longest chromosome in order to ask whether a meiotic DSB’s proximity to the centromere influences its capacity to promote homologous centromere pairing and SC assembly. We show that repair of an <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated DSB does not promote homologous centromere pairing nor any extent of SC assembly in <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a></em> meiotic nuclei, regardless of its proximity to the centromere. DSBs induced <em>en masse</em> by phleomycin exposure likewise do not promote homologous centromere pairing nor robust SC assembly. Interestingly, in contrast to <a href="http://www.yeastgenome.org/locus/S000001014/overview">Spo11</a>, <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-initiated interhomolog recombination is not affected by loss of the meiotic kinase, <a href="http://www.yeastgenome.org/locus/S000005878/overview">Mek1</a>, and is not constrained to use the meiosis-specific <a href="http://www.yeastgenome.org/locus/S000000981/overview">Dmc1</a> recombinase. These results strengthen the previously proposed idea that (at least some) <a href="http://www.yeastgenome.org/locus/S000001014/overview">Spo11</a> DSBs may be specialized in activating mechanisms that both 1) reinforce homologous chromosome alignment via homologous centromere pairing and SC assembly, and 2) establish <a href="http://www.yeastgenome.org/locus/S000000981/overview">Dmc1</a> as the primary strand exchange enzyme.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/chromosome-pairing" class="hw-term hw-article-keyword hw-article-keyword-chromosome-pairing" rel="nofollow">chromosome pairing</a></li><li class="kwd"><a href="/keyword/ho-endonuclease" class="hw-term hw-article-keyword hw-article-keyword-ho-endonuclease" rel="nofollow">HO endonuclease</a></li><li class="kwd"><a href="/keyword/meiosis" class="hw-term hw-article-keyword hw-article-keyword-meiosis" rel="nofollow">meiosis</a></li><li class="kwd"><a href="/keyword/recombination" class="hw-term hw-article-keyword hw-article-keyword-recombination" rel="nofollow">recombination</a></li><li class="kwd"><a href="/keyword/spo11" class="hw-term hw-article-keyword hw-article-keyword-spo11" rel="nofollow">Spo11</a></li><li class="kwd"><a href="/keyword/synapsis" class="hw-term hw-article-keyword hw-article-keyword-synapsis" rel="nofollow">synapsis</a></li></ul><p id="p-3">Successful ploidy reduction during meiosis requires that chromosomes efficiently search for, recognize, and establish a stable connection with their homologous partners, allowing them to orient and segregate properly to opposite poles of the meiosis I spindle apparatus (<a id="xref-ref-84-1" class="xref-bibr" href="#ref-84">Page and Hawley 2003</a>). Accordingly, a characteristic feature of meiotic prophase nuclei is a large-scale reorganization in which homologous chromosomes transition from an unpaired distribution to an intimate, lengthwise alignment, usually in the context of an elaborate proteinaceous structure, the synaptonemal complex (SC). However, the connection that homologous chromosomes (homologs) ultimately rely on for their proper segregation is more discrete: this crucial link is nearly always provided by a crossover recombination event. Thus, the central task of meiosis is to coordinate a dramatic change in the spatial distribution of chromosomes with the formation and repair of DNA double strand breaks (DSBs) in a manner that promotes crossing over (<a id="xref-ref-88-1" class="xref-bibr" href="#ref-88">Pawlowski and Cande 2005</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Hunter 2015</a>; <a id="xref-ref-125-1" class="xref-bibr" href="#ref-125">Zickler and Kleckner 2015</a>). The molecular basis for how DNA repair and chromosome pairing mechanisms are coordinated during meiosis remains poorly understood.</p><p id="p-4">Meiotic recombination is normally initiated via DSBs created by the conserved transesterase, <a href="http://www.yeastgenome.org/locus/S000001014/overview">Spo11</a>, in conjunction with several accessory proteins (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Lam and Keeney 2014</a>). Each end of a <a href="http://www.yeastgenome.org/locus/S000001014/overview">Spo11</a>-mediated DSB undergoes nucleolytic processing to generate 3′ single stranded DNA; these single-stranded termini associate with homologs of the <em>E. coli</em> RecA recombinase protein in order to assemble nucleoprotein filaments capable of catalyzing strand exchange with homologous duplex DNA (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Brown and Bishop 2014</a>; <a id="xref-ref-61-2" class="xref-bibr" href="#ref-61">Lam and Keeney 2014</a>; <a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Hunter 2015</a>). In budding yeast meiosis, a subset of nascent strand exchange events undergo dissolution after limited DNA synthesis, and DNA repair is completed for these events via a synthesis-dependent-strand-annealing (SDSA) mechanism to form a noncrossover (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Allers and Lichten 2001a</a>; <a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Mcmahill <em>et al.</em> 2007</a>). Another subset of strand exchange events become stable single end invasion intermediates, and then mature into double-Holliday junction (dHJ) containing joint molecule structures, after “capture” of the second end of the DSB (<a id="xref-ref-96-1" class="xref-bibr" href="#ref-96">Schwacha and Kleckner 1994</a>; <a id="xref-ref-97-1" class="xref-bibr" href="#ref-97">Schwacha and Kleckner 1995</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Allers and Lichten 2001b</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Hunter and Kleckner 2001</a>). In budding yeast, the interhomolog crossovers that are critical for proper chromosome segregation form predominantly by the resolution of dHJs (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Allers and Lichten 2001a</a>).</p><p id="p-5">The diploid meiotic cell offers three homologous templates that could be targeted for strand exchange by either end of a DSB: One sister and two non-sister chromatids. In contrast to vegetative cells (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Kadyk and Hartwell 1992</a> ; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Bzymek <em>et al.</em> 2010</a>) homologous recombination in meiotic cells preferentially utilizes a non-sister chromatid (the homolog) as a template for DNA repair (<a id="xref-ref-96-2" class="xref-bibr" href="#ref-96">Schwacha and Kleckner 1994</a>; <a id="xref-ref-97-2" class="xref-bibr" href="#ref-97">Schwacha and Kleckner 1995</a>; <a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Hunter and Kleckner 2001</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Hong <em>et al.</em> 2013</a>). One meiosis-specific mechanism that promotes interhomolog <em>vs.</em> intersister DNA repair involves the specialized, concerted action of two RecA homologs: The strand exchange activity of a meiosis-specific RecA homolog, <a href="http://www.yeastgenome.org/locus/S000000981/overview">Dmc1</a> (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Bishop <em>et al.</em> 1992</a>) in conjunction with a supporting activity of the mitotic RecA protein, <a href="http://www.yeastgenome.org/locus/S000000897/overview">Rad51</a> (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Game and Mortimer 1974</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Game <em>et al.</em> 1980</a>; <a id="xref-ref-100-1" class="xref-bibr" href="#ref-100">Shinohara <em>et al.</em> 1992</a>; <a id="xref-ref-98-1" class="xref-bibr" href="#ref-98">Schwacha and Kleckner 1997</a>; <a id="xref-ref-99-1" class="xref-bibr" href="#ref-99">Shinohara <em>et al.</em> 1997a</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Cloud <em>et al.</em> 2012</a>). In the context of solely <a href="http://www.yeastgenome.org/locus/S000000981/overview">Dmc1</a> or <a href="http://www.yeastgenome.org/locus/S000000897/overview">Rad51</a>, meiotic interhomolog recombination is dramatically diminished and residual DSB repair occurs primarily using the sister chromatid (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Bishop <em>et al.</em> 1992</a>; <a id="xref-ref-100-2" class="xref-bibr" href="#ref-100">Shinohara <em>et al.</em> 1992</a>; <a id="xref-ref-99-2" class="xref-bibr" href="#ref-99">Shinohara <em>et al.</em> 1997a</a>; <a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Hong <em>et al.</em> 2013</a>; <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Lao <em>et al.</em> 2013</a>). During normal meiosis in budding yeast, <a href="http://www.yeastgenome.org/locus/S000000897/overview">Rad51</a>’s strand exchange activity is diminished and the preferential use of a <a href="http://www.yeastgenome.org/locus/S000000981/overview">Dmc1</a> recombinase pathway may be ensured in part by the inhibition of the <a href="http://www.yeastgenome.org/locus/S000003131/overview">Rad54</a> motor protein (<a id="xref-ref-102-1" class="xref-bibr" href="#ref-102">Shinohara <em>et al.</em> 1997b</a>; <a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">Niu <em>et al.</em> 2005</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Busygina <em>et al.</em> 2008</a>; <a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">Niu <em>et al.</em> 2009</a>) and the stabilization of an interaction between <a href="http://www.yeastgenome.org/locus/S000000897/overview">Rad51</a> and its inhibitor, <a href="http://www.yeastgenome.org/locus/S000113613/overview">Hed1</a> (<a id="xref-ref-112-1" class="xref-bibr" href="#ref-112">Tsubouchi and Roeder 2006</a>; <a id="xref-ref-64-2" class="xref-bibr" href="#ref-64">Lao <em>et al.</em> 2013</a>); both of these pathways rely on the activity of the meiosis-specific kinase, <a href="http://www.yeastgenome.org/locus/S000005878/overview">Mek1</a> (<a id="xref-ref-122-1" class="xref-bibr" href="#ref-122">Xu <em>et al.</em> 1997</a>; <a id="xref-ref-42-3" class="xref-bibr" href="#ref-42">Hong <em>et al.</em> 2013</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Callender <em>et al.</em> 2016</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Hollingsworth 2016</a>).</p><p id="p-6">In budding yeast, meiotic recombination not only generates interhomolog crossovers but also promotes homologous chromosome synapsis - the assembly of SC between lengthwise-aligned chromosomes (<a id="xref-ref-85-1" class="xref-bibr" href="#ref-85">Page and Hawley 2004</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Cahoon and Hawley 2016</a>). The SC has a conserved, tripartite structure in which rod-like transverse filament proteins assemble in perpendicular orientation to the long axis of the chromosome. Transverse filament proteins bridge chromosome axes (called lateral elements within assembled SC) and a distinct substructure, the central element, assembles at the midline of the SC. The transverse filament of the budding yeast SC is comprised of the <a href="http://www.yeastgenome.org/locus/S000002693/overview">Zip1</a> protein, which has an extensive central coiled-coil motif that is predicted to fold into a rod-shaped homodimer or tetramer (<a id="xref-ref-105-1" class="xref-bibr" href="#ref-105">Sym <em>et al.</em> 1993</a>; <a id="xref-ref-106-1" class="xref-bibr" href="#ref-106">Sym and Roeder 1995</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Dong and Roeder 2000</a>). The interacting <a href="http://www.yeastgenome.org/locus/S000002854/overview">Ecm11</a> and <a href="http://www.yeastgenome.org/locus/S000004437/overview">Gmc2</a> proteins assemble the central element substructure of budding yeast SC (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Humphryes <em>et al.</em> 2013</a>; <a id="xref-ref-120-1" class="xref-bibr" href="#ref-120">Voelkel-Meiman <em>et al.</em> 2013</a>).</p><p id="p-7">Synapsis initiates at multiple discrete points along the length of chromosomes, many of which are likely sites of interhomolog recombination (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Chua and Roeder 1998</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Agarwal and Roeder 2000</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Henderson <em>et al.</em> 2004</a>). Interestingly, however, the earliest SC assembly events in budding yeast meiotic cells occur predominantly from centromeres (<a id="xref-ref-113-1" class="xref-bibr" href="#ref-113">Tsubouchi <em>et al.</em> 2008</a>). <a href="http://www.yeastgenome.org/locus/S000001014/overview">Spo11</a>-dependent SC assembly from centromeres raises the mechanistic question of how this class of synapsis events is coupled to meiotic recombination, given that centromeres are not thought to correspond to sites that undergo frequent interhomolog recombination in budding yeast meiosis (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Lambie and Roeder 1986</a>; <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Lambie and Roeder 1988</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Blitzblau <em>et al.</em> 2007</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Chen <em>et al.</em> 2008</a>; <a id="xref-ref-86-1" class="xref-bibr" href="#ref-86">Pan <em>et al.</em> 2011</a>; <a id="xref-ref-117-1" class="xref-bibr" href="#ref-117">Vincenten <em>et al.</em> 2015</a>).</p><p id="p-8">One explanation for initial SC assembly from centromeres in budding yeast may relate to the existence of an SC-independent “coupling” mechanism that can reinforce pair-wise interactions between homologous centromeres. The SC transverse filament protein, <a href="http://www.yeastgenome.org/locus/S000002693/overview">Zip1</a>, mediates two-by-two associations between centromeres, regardless of homology and independent of <a href="http://www.yeastgenome.org/locus/S000001014/overview">Spo11</a> activity, at the onset of meiosis in budding yeast (<a id="xref-ref-114-1" class="xref-bibr" href="#ref-114">Tsubouchi and Roeder 2005</a>). <a href="http://www.yeastgenome.org/locus/S000002693/overview">Zip1</a> also mediates pair-wise associations between homologous centromeres in a <a href="http://www.yeastgenome.org/locus/S000001014/overview">Spo11</a>-dependent manner during later meiotic prophase (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Kemp <em>et al.</em> 2004</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Falk <em>et al.</em> 2010</a>; <a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">Newnham <em>et al.</em> 2010</a>; <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Kurdzo <em>et al.</em> 2017</a>). <a href="http://www.yeastgenome.org/locus/S000002693/overview">Zip1</a>’s centromere pairing activity does not involve a conventional SC structure, as the SC central element protein <a href="http://www.yeastgenome.org/locus/S000002854/overview">Ecm11</a> is dispensable for both <a href="http://www.yeastgenome.org/locus/S000001014/overview">Spo11</a>-independent and <a href="http://www.yeastgenome.org/locus/S000001014/overview">Spo11</a>-dependent centromere pairing (<a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Humphryes <em>et al.</em> 2013</a>; <a id="xref-ref-59-2" class="xref-bibr" href="#ref-59">Kurdzo <em>et al.</em> 2017</a>). However, the local abundance of <a href="http://www.yeastgenome.org/locus/S000002693/overview">Zip1</a> at centromeres perhaps bestows these chromosomal regions with an increased capacity to assemble SC in response to <em>trans</em> acting signals from recombination sites.</p><p id="p-9">How are meiotic DNA repair processes connected to specialized chromosome pairing and synapsis outcomes? <a id="xref-ref-108-1" class="xref-bibr" href="#ref-108">Thorne and Byers (1993)</a> showed that the capacity for X-ray induced DSBs to partially rescue the low spore viability of <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a></em> meiotic cells depends on the meiosis-specific chromosomal protein, <a href="http://www.yeastgenome.org/locus/S000001334/overview">Hop1</a>; this result indicates that recombination intermediates might ensure meiotic chromosome pairing and/or segregation outcomes at least in part by interfacing with meiotic factors that function at the chromosome axis. Consistent with this notion, a handful of meiosis-specific proteins that localize within the SC, including <a href="http://www.yeastgenome.org/locus/S000002693/overview">Zip1</a>, <a href="http://www.yeastgenome.org/locus/S000003218/overview">Zip2</a>, <a href="http://www.yeastgenome.org/locus/S000004386/overview">Zip3</a>, <a href="http://www.yeastgenome.org/locus/S000001335/overview">Zip4</a>, <a href="http://www.yeastgenome.org/locus/S000001196/overview">Spo16</a>, and the MutSγ complex <a href="http://www.yeastgenome.org/locus/S000001891/overview">Msh4</a>-<a href="http://www.yeastgenome.org/locus/S000002313/overview">Msh5</a>, are critical for normal levels of dHJs and crossing over during meiosis (<a id="xref-ref-105-2" class="xref-bibr" href="#ref-105">Sym <em>et al.</em> 1993</a>; <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Agarwal and Roeder 2000</a>; <a id="xref-ref-82-1" class="xref-bibr" href="#ref-82">Novak <em>et al.</em> 2001</a>; <a id="xref-ref-115-1" class="xref-bibr" href="#ref-115">Tsubouchi <em>et al.</em> 2006</a>; <a id="xref-ref-101-1" class="xref-bibr" href="#ref-101">Shinohara <em>et al.</em> 2008</a>). Furthermore, meiosis-specific chromosome axis proteins such as <a href="http://www.yeastgenome.org/locus/S000004253/overview">Red1</a>, <a href="http://www.yeastgenome.org/locus/S000001334/overview">Hop1</a> and <a href="http://www.yeastgenome.org/locus/S000006211/overview">Rec8</a> have been associated with promoting interhomolog interactions (<a id="xref-ref-99-3" class="xref-bibr" href="#ref-99">Shinohara <em>et al.</em> 1997a</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Kim <em>et al.</em> 2010</a>; <a id="xref-ref-42-4" class="xref-bibr" href="#ref-42">Hong <em>et al.</em> 2013</a>).</p><p id="p-10">Further insight into how DNA repair machinery is connected to specialized chromosome pairing and synapsis outcomes in budding yeast is provided by elegant studies that compare recombination outcomes in meiotic <em>vs.</em> mitotic cells. <a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Malkova <em>et al.</em> (2000)</a> found that an <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated DSB is four times more likely to repair as an interhomolog crossover in <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a></em> meiotic cells than in mitotic cells; these results, similar to the aforementioned results of Thorne and Byers, suggest the possibility that <a href="http://www.yeastgenome.org/locus/S000001014/overview">Spo11</a>-independent mechanisms ensure that any DSB (no matter the source) is processed in a manner that is “meiotic-like” in budding yeast meiotic cells. However, it was also noted that <a href="http://www.yeastgenome.org/locus/S000001014/overview">Spo11</a> activity has a <em>trans</em> effect on <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> DSB repair: The presence of <a href="http://www.yeastgenome.org/locus/S000001014/overview">Spo11</a> correlated with a shorter interhomolog conversion tract length and an even higher likelihood of crossing over relative to when <a href="http://www.yeastgenome.org/locus/S000001014/overview">Spo11</a> is absent (<a id="xref-ref-69-2" class="xref-bibr" href="#ref-69">Malkova <em>et al.</em> 2000</a>). Independent studies using VDE, a site-specific endonuclease, provide additional evidence that <a href="http://www.yeastgenome.org/locus/S000001014/overview">Spo11</a> can increase the likelihood of interhomolog repair (<a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">Neale <em>et al.</em> 2002</a>), reduce gene conversion tract length <em>in trans</em> (<a id="xref-ref-78-2" class="xref-bibr" href="#ref-78">Neale <em>et al.</em> 2002</a>), as well as influence the repair factors utilized (<a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Medhi <em>et al.</em> 2016</a>). Furthermore, when assessed, prior studies found no evidence that an <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated or VDE-mediated DSB promotes SC assembly in <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a></em> mutant meiocytes (<a id="xref-ref-69-3" class="xref-bibr" href="#ref-69">Malkova <em>et al.</em> 2000</a>; <a id="xref-ref-78-3" class="xref-bibr" href="#ref-78">Neale <em>et al.</em> 2002</a>), although partial SC assembly may have been missed because of the small size of the chromosomes sustaining the DSB. Taken together these findings support the idea that unique properties of meiosis-specific DSB machinery, in conjunction with recombination-independent features of the meiotic nucleus, ensure that recombination is accompanied by robust homolog engagement and crossing over in budding yeast. Our understanding of whether and how meiotic chromosome pairing processes are uniquely regulated by <a href="http://www.yeastgenome.org/locus/S000001014/overview">Spo11</a>, however, remains incomplete.</p><p id="p-11">In this study, we addressed the question of whether a meiotic DSB’s position relative to the centromere affects the homologous centromere pairing or synapsis outcome. We evaluated the genetic and cytological behavior of meiotic cells devoid of <a href="http://www.yeastgenome.org/locus/S000001014/overview">Spo11</a> but capable of <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> endonuclease-mediated DSBs at several distinct positions along budding yeast’s longest chromosome (IV), including positions that <a href="http://www.yeastgenome.org/locus/S000001014/overview">Spo11</a> utilizes frequently. We asked whether <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated DSBs positioned nearby or distal to centromere IV, or alternatively whether <em>en masse</em> phleomycin-induced DSBs, are capable of promoting homologous centromere pairing and/or any extent of SC assembly in <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a></em> mutant meiotic nuclei. Our investigation revealed that <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated and phleomycin-induced DSBs fail to promote homologous centromere pairing and even partial SC assembly, regardless of their proximity to the centromere. Our results reveal additional potential distinctions between the processing of <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>- <em>vs.</em> <a href="http://www.yeastgenome.org/locus/S000001014/overview">Spo11</a>-initiated recombination intermediates in meiotic cells, including a difference in overall reliance on the meiosis-specific <a href="http://www.yeastgenome.org/locus/S000005878/overview">Mek1</a> kinase and <a href="http://www.yeastgenome.org/locus/S000000981/overview">Dmc1</a> recombinase. Our findings add to mounting evidence that one or more specialized properties of at least a subset of <a href="http://www.yeastgenome.org/locus/S000001014/overview">Spo11</a> DSBs serve to couple a recombination-based homology recognition process with mechanisms that reinforce homolog pairing in yeast meiosis, and that one critical and unique feature of pro-synapsis <a href="http://www.yeastgenome.org/locus/S000001014/overview">Spo11</a> DSBs may be their engagement with the mechanism that establishes <a href="http://www.yeastgenome.org/locus/S000000981/overview">Dmc1</a> as the primary recombinase.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Strain Construction</h3><p id="p-12">Yeast strains used in this study are isogenic to BR1919-8B (<a id="xref-ref-94-1" class="xref-bibr" href="#ref-94">Rockmill <em>et al.</em> 1995</a>); Table S4 lists their genotypes. Strains were constructed using standard genetic and transformation methods. Primers are listed in Table S5.</p><p id="p-13">To build a plasmid to integrate <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> endonuclease driven by <em><a href="http://www.yeastgenome.org/locus/S000001056/overview">SPO13</a></em> promoter sequences at the <em><a href="http://www.yeastgenome.org/locus/S000000319/overview">LYS2</a></em> locus, sequences encompassing <em><a href="http://www.yeastgenome.org/locus/S000000319/overview">LYS2</a></em>, including 420 bp upstream and 50 bp downstream, were amplified using primers AJM838 and AJM839, and inserted at the <em>Sma</em>I site of pRS315 to create pAM191. An <em>Xba</em>I site was engineered into the reverse primer AJM839 so that <em>Xba</em>I sites flank the <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> cassette in pAM191. A step-wise PCR was used to generate <em>P<sub><a href="http://www.yeastgenome.org/locus/S000001056/overview">SPO13</a></sub>-<a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a></em>. First, <em><a href="http://www.yeastgenome.org/locus/S000001056/overview">SPO13</a></em> promoter sequence was amplified from genomic DNA using primers AJM763 and AJM764. AJM764 has sequence that overlaps with the beginning of <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>. Second, <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> endonuclease ORF sequence (ATG through 200 bp downstream of the STOP) were amplified from a plasmid carrying <em>P<sub>GAL</sub>- <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a></em> (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Jensen and Herskowitz 1984</a>), using primers AJM765 and AJM766. Finally, the two overlapping DNA fragments were “stitched” in an amplification using primers AJM763 and AJM766. The <em>P<sub><a href="http://www.yeastgenome.org/locus/S000001056/overview">SPO13</a></sub></em> –<em><a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a></em> fragment was inserted at the SnaBI site of pAM191 to create pAM200. Strains in which <em>P<sub><a href="http://www.yeastgenome.org/locus/S000001056/overview">SPO13</a></sub></em> –<em><a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a></em> successfully integrated at the <em><a href="http://www.yeastgenome.org/locus/S000000319/overview">LYS2</a></em> locus were identified first by using counter-selection against <em><a href="http://www.yeastgenome.org/locus/S000000319/overview">LYS2</a></em> on alpha-aminoadipate plates and then by PCR.</p><p id="p-14">To create a template containing the <em><a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> cut site</em> sequence (<em><a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> cs</em>) linked to a genetic marker, a 100 bp <em>Hin</em>dIII fragment containing the <em><a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> cs</em> sequence was excised from pAR134 (<a id="xref-ref-90-1" class="xref-bibr" href="#ref-90">Ray <em>et al.</em> 1988</a>) and inserted at the <em>Hin</em>dIII site of pAG25, which carries <em>natMX</em> (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Goldstein and Mccusker 1999</a>) to create pAM159. <em><a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> cs</em> sequences were amplified from pAM159 using primers with sequence homology to various locations on chromosome IV (see Table S5). <em><a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> cs</em> sequences were integrated at the chromosome IV coordinates indicated in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>; except for <em>cs1</em> and <em>cs2</em>, each of these chromosomal positions have been identified as regions of frequent cleavage by <a href="http://www.yeastgenome.org/locus/S000001014/overview">Spo11</a> (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Blitzblau <em>et al.</em> 2007</a>; <a id="xref-ref-86-2" class="xref-bibr" href="#ref-86">Pan <em>et al.</em> 2011</a>; <a id="xref-ref-107-1" class="xref-bibr" href="#ref-107">Thacker <em>et al.</em> 2014</a>; <a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Markowitz <em>et al.</em> 2017</a>). The chromosomal position of <em>cs2</em>, which is 250 nucleotides from <em>CENIV</em>, has been found to be enriched for the meiosis-specific cohesin subunit <a href="http://www.yeastgenome.org/locus/S000006211/overview">Rec8</a> but not for the meiosis-specific axis proteins <a href="http://www.yeastgenome.org/locus/S000004253/overview">Red1</a> and <a href="http://www.yeastgenome.org/locus/S000001334/overview">Hop1</a> (<a id="xref-ref-87-1" class="xref-bibr" href="#ref-87">Panizza <em>et al.</em> 2011</a>). For technical reasons, two consecutive PCRs were performed for creating DNA unique to each cut site location on chromosome IV. The first PCR was carried out using AJM760, a forward primer common to all integration site cassettes, in conjunction with a reverse primer carrying homology to both pAM159 and the specific integration site on chromosome IV. The second PCR was performed using the same reverse primer in conjunction with a site-specific forward primer carrying homology to the template and the specific chromosome IV integration site. The specific forward and reverse primers used for PCR2 are: <em>cs1</em> (AJM750 and AJM751), <em>cs2</em> (AJM752 and AJM753), <em>cs4</em> (AJM756 and AJM757), <em>cs5</em> (AJM975 and AJM976), <em>cs6</em> (AJM1137 and AJM1138), <em>cs7</em> (AJM1141 and AJM1142), <em>cs8</em> (AJM1145 and AJM1146), <em>cs9</em> (AJM1255 and AJM1256) and <em>cs10</em> (AJM1259 and AJM1260). The position of the <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> <em>cs</em> in each strain was confirmed by sequence analysis.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/11/3637/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Creating strains in which HO endonuclease is the sole source of meiotic DSBs. (A) Illustration indicates the chromosomal positions of a meiosis-specific HO endonuclease gene cassette and various HO cut sites (HO cs). PSPO13 -HO interrupts the LYS2 locus on chromosome II. HO cs sequences together with the natMX4 drug marker were targeted to the indicated chromosome IV coordinates. CENIV (solid circle) corresponds to coordinates 449,711- 449,821bp (Saccharomyces Genome Database). (B) Illustration depicts genotypic and phenotypic MAT locus outcomes of spo11 spo13 meiotic nuclei with or without meiotic expression of HO endonuclease. Meiotic cells undergo a single equational division in spo11 spo13 strains producing a dyad with diploid spores. In the absence of HO-mediated recombination, each dyad spore receives one copy of MATa and one copy of MAT&#x3B1;, resulting in two non-mating spores (nm). In the presence of HO endonuclease, interhomolog or intrachromosomal recombination at the MAT locus on any or all of the chromatids (1, 2, 3 or 4) can produce homozygous MATa or MAT&#x3B1; spores, which will be phenotypically a or &#x3B1; &#x201C;maters&#x201D;." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2138819651" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Creating strains in which HO endonuclease is the sole source of meiotic DSBs. (A) Illustration indicates the chromosomal positions of a meiosis-specific HO endonuclease gene cassette and various &lt;em&gt;HO&lt;/em&gt; cut sites (&lt;em&gt;HO cs)&lt;/em&gt;. &lt;em&gt;P&lt;sub&gt;SPO13&lt;/sub&gt; -HO&lt;/em&gt; interrupts the &lt;em&gt;LYS2&lt;/em&gt; locus on chromosome II. &lt;em&gt;HO cs&lt;/em&gt; sequences together with the &lt;em&gt;natMX4&lt;/em&gt; drug marker were targeted to the indicated chromosome IV coordinates. &lt;em&gt;CENIV&lt;/em&gt; (solid circle) corresponds to coordinates 449,711- 449,821bp (&lt;em&gt;Saccharomyces&lt;/em&gt; Genome Database). (B) Illustration depicts genotypic and phenotypic &lt;em&gt;MAT&lt;/em&gt; locus outcomes of &lt;em&gt;spo11 spo13&lt;/em&gt; meiotic nuclei with or without meiotic expression of HO endonuclease. Meiotic cells undergo a single equational division in &lt;em&gt;spo11 spo13&lt;/em&gt; strains producing a dyad with diploid spores. In the absence of HO-mediated recombination, each dyad spore receives one copy of &lt;em&gt;MAT&lt;/em&gt;&lt;strong&gt;&lt;em&gt;a&lt;/em&gt;&lt;/strong&gt; and one copy of &lt;em&gt;MAT&#x3B1;&lt;/em&gt;, resulting in two non-mating spores (nm). In the presence of HO endonuclease, interhomolog or intrachromosomal recombination at the &lt;em&gt;MAT&lt;/em&gt; locus on any or all of the chromatids (1, 2, 3 or 4) can produce homozygous &lt;em&gt;MAT&lt;/em&gt;&lt;strong&gt;&lt;em&gt;a&lt;/em&gt;&lt;/strong&gt; or &lt;em&gt;MAT&#x3B1;&lt;/em&gt; spores, which will be phenotypically a or &lt;em&gt;&#x3B1;&lt;/em&gt; &#x201C;maters&#x201D;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/11/3637/F1.medium.gif" width="440" height="252"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/8/11/3637/F1.medium.gif" width="440" height="252"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/11/3637/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/11/3637/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491430" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-15" class="first-child">Creating strains in which HO endonuclease is the sole source of meiotic DSBs. (A) Illustration indicates the chromosomal positions of a meiosis-specific HO endonuclease gene cassette and various <em>HO</em> cut sites (<em>HO cs)</em>. <em>P<sub>SPO13</sub> -HO</em> interrupts the <em>LYS2</em> locus on chromosome II. <em>HO cs</em> sequences together with the <em>natMX4</em> drug marker were targeted to the indicated chromosome IV coordinates. <em>CENIV</em> (solid circle) corresponds to coordinates 449,711- 449,821bp (<em>Saccharomyces</em> Genome Database). (B) Illustration depicts genotypic and phenotypic <em>MAT</em> locus outcomes of <em>spo11 spo13</em> meiotic nuclei with or without meiotic expression of HO endonuclease. Meiotic cells undergo a single equational division in <em>spo11 spo13</em> strains producing a dyad with diploid spores. In the absence of HO-mediated recombination, each dyad spore receives one copy of <em>MAT</em><strong><em>a</em></strong> and one copy of <em>MATα</em>, resulting in two non-mating spores (nm). In the presence of HO endonuclease, interhomolog or intrachromosomal recombination at the <em>MAT</em> locus on any or all of the chromatids (1, 2, 3 or 4) can produce homozygous <em>MAT</em><strong><em>a</em></strong> or <em>MATα</em> spores, which will be phenotypically a or <em>α</em> “maters”.</p><div class="sb-div caption-clear"></div></div></div><p id="p-16">To generate strains with genetic markers flanking <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> <em>cs2</em>, the <em><a href="http://www.yeastgenome.org/locus/S000000523/overview">LEU2</a></em> gene cassette was amplified using primers AJM1702 and AJM1698 and inserted at position 447 kb (2.8 kb left of <em>CEN4</em>). <em><a href="http://www.yeastgenome.org/locus/S000001067/overview">THR1</a></em> was PCR amplified with AJM1241 and AJM1242 and inserted at 1,416 kb on chromosome IV. To generate strains with genetic markers flanking all other <em><a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> cs</em> loci, <em><a href="http://www.yeastgenome.org/locus/S000000523/overview">LEU2</a></em> was integrated 705 nucleotides to the right of <em>CEN4</em> (described below). <em><a href="http://www.yeastgenome.org/locus/S000001067/overview">THR1</a></em> was inserted as described for strains carrying <em><a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> cs2</em>.</p><p id="p-17">To build strains for pairing analysis, a <em>lacO</em> array was integrated 705 bp to the right of <em>CEN4</em> by digesting pJBN156 (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bachant <em>et al.</em> 2002</a>) with <em>Nhe</em>I, and a <em>tetO</em> array was integrated at coordinate 1,242 kb of chromosome IV by digesting pAM152 (kindly donated by Karen Voelkel-Meiman) with <em>Eco</em>R1. pAM152 was generated as follows: First, a 920 bp fragment encompassing the sequences at position 1,242 kb on chromosome IV was amplified from genomic DNA using primers AJM650 and AJM651, and cloned into the <em>Hin</em>dIII/<em>Sph</em>I site of p306tetO224 (<a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Michaelis <em>et al.</em> 1997</a>), which contains 224 tandem <em>tetO</em> (11.2 kb) sequence repeats in a pRS306 vector (<a id="xref-ref-103-1" class="xref-bibr" href="#ref-103">Sikorski and Hieter 1989</a>); this created pAM145. Next, the <em><a href="http://www.yeastgenome.org/locus/S000001067/overview">THR1</a></em> gene was amplified using primers AJM666 and AJM667 and cloned between AatII and <em>Sac</em>I sites of pAM145, creating pAM152.</p><p id="p-18"><em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a>-Y135F</em>::<em>kanMX4</em> was integrated to replace the <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">SPO11</a></em> gene by digesting pJY20 (a kind gift of Dr. Scott Keeney, Memorial Sloan Kettering Cancer Center) with <em>Sal</em>I and <em>Sac</em>II. pJY20 contains <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a>-Y135F</em>::<em>kanMX4</em> inserted into the multicloning site of pRS316 (<a id="xref-ref-103-2" class="xref-bibr" href="#ref-103">Sikorski and Hieter 1989</a>). Dr. Douglas Bishop (University of Chicago) kindly provided a <em><a href="http://www.yeastgenome.org/locus/S000000897/overview">rad51</a>-II3A</em> strain, which was used to amplify the <em><a href="http://www.yeastgenome.org/locus/S000000897/overview">rad51</a>-II3A</em> allele sequence that we introduced into our strain background.</p><p id="p-19">The 2μ-<em><a href="http://www.yeastgenome.org/locus/S000004253/overview">RED1</a>-<a href="http://www.yeastgenome.org/locus/S000001334/overview">HOP1</a></em>, pNH219 (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Hollingsworth and Ponte 1997</a>) were a kind gift of Dr. Nancy Hollingsworth. A 2μ-<em><a href="http://www.yeastgenome.org/locus/S000006211/overview">REC8</a>-MYC</em> (pAM356) plasmid was constructed using gap repair to replace the <em><a href="http://www.yeastgenome.org/locus/S000001334/overview">HOP1</a></em> ORF in pDW72 with <em><a href="http://www.yeastgenome.org/locus/S000006211/overview">REC8</a>-MYC</em>, which was amplified from strain LY893 (Table S4) with primers AJM2164 and AJM2165 (Table S5). The viability of spores from a strain homozygous for the <em><a href="http://www.yeastgenome.org/locus/S000006211/overview">REC8</a>-MYC</em> sequence that is carried by LY893 is 97% (n= 52), similar to the wild-type value (95%, n = 52).</p><p id="p-20">Sporulation efficiency and spore viability of strains utilized for analysis of <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated recombination are listed in <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>, Table S2 and Table S3.</p></div><div id="sec-3" class="subsection"><h3>Assay for mitotic HO-mediated MAT recombination</h3><p id="p-21">A single culture of each diploid strain used for the genetic analysis of crossover recombination (LY407, LY208, LY555, LY324, LY322 and LY207), which express <em>P<sub><a href="http://www.yeastgenome.org/locus/S000001056/overview">SPO13</a></sub>-<a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a></em> or no <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> endonuclease (control) was grown overnight in liquid rich medium at 30°. Cells were then diluted and transferred to rich medium plates at a density of 100-200 colonies per plate. After 2 days of incubation, all colonies were assessed for mating type using a complementation-based assay in which only those cells that mate with a tester <em>MAT</em><strong><em>a</em></strong> or <em>MATα</em> strain are capable of growth on minimal media. Zero colonies from strains without <em>P<sub><a href="http://www.yeastgenome.org/locus/S000001056/overview">SPO13</a></sub>-<a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a></em> exhibited any capacity to mate with either the <em>MAT</em><strong><em>a</em></strong> or <em>MATα</em> tester strain, as assessed by growth on minimal media (n = 421). A low frequency of plated colonies (0.7%, 0.7%, 1.5% and 1.2% in four independent tests involving &gt;400 colonies) from strains carrying <em>P<sub><a href="http://www.yeastgenome.org/locus/S000001056/overview">SPO13</a></sub>-<a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a></em> exhibited papilla of growth on minimal media after mixing with either the <em>MAT</em><strong><em>a</em></strong> or <em>MATα</em> tester strain, indicating a low level of <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> activity during vegetative growth.</p></div><div id="sec-4" class="subsection"><h3>Southern Blots</h3><p id="p-22">Genomic DNA was prepared at 0, 12, 18 and 24 hr of sporulation from strains, LY491, LY456, LY492, LY459, LY481, LY457 and LY458, which are homozygous for a <em><a href="http://www.yeastgenome.org/locus/S000000897/overview">rad51</a></em> null mutation and thus severely delayed in completing DNA repair using homologous recombination. After digestion with the appropriate restriction enzyme (<em>Eco</em>RV for strains carrying <em><a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> cs1</em>, <em>cs5</em> or <em>cs7</em>, <em>Xho</em>I for strains carrying <em><a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> cs2</em>, <em>Pvu</em>II and <em>Xho</em>I for strains carrying <em><a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> cs6</em> and <em>Spe</em>I for strains carrying <em><a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> cs4</em>), genomic DNA was separated on a 0.8% TAE-buffered agarose gel and transferred to Hybond-Nylon membrane (<span class="sc">klapholz and esposito</span>). A 500 bp probe homologous to <em>natMX4</em> sequences, prepared using a DIG High Prime DNA Labeling and Detection kit was hybridized to the membranes in order to detect genomic DNA fragments containing <em><a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> cs</em> sequences that were either cut or intact. A Syngene G:Box was used to detect chemiluminescence of the hybridized probe; intensity profiles for each lane were generated using the Syngene Gene Tools program. The percentage of <em><a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> cs</em> DNA that was cut was calculated by dividing the intensity of the cut <em><a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> cs</em>–containing fragment by the sum of the cut and uncut fragments. The average of two experiments is presented for each strain.</p></div><div id="sec-5" class="subsection"><h3>Western blots</h3><p id="p-23">Protein was precipitated from 5 ml meiotic cultures at 24 hr of sporulation by trichloroacetic acid precipitation as previously described (<a id="xref-ref-118-1" class="xref-bibr" href="#ref-118">Voelkel-Meiman <em>et al.</em> 2016</a>). Precipitated protein was dissolved in 2x Laemmli sample buffer (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Laemmli 1970</a>), supplemented with 30 mM DTT. Samples were heated for 10 min at 65°, then centrifuged at top speed before the concentration of soluble protein was determined using a NanoDrop (Thermo Fisher). 25-50 μg of protein was loaded on an 8% polyacrylamide/SDS gel and separated at 200 volts for one hour. Separated proteins were transferred to 0.2 μm nitrocellulose membranes (Amersham) after equilibrating the membrane and gel in Towbin buffer (<a id="xref-ref-109-1" class="xref-bibr" href="#ref-109">Towbin <em>et al.</em> 1979</a>) for 15 min. Protein transfer was performed at 60 volts for one hour at room temperature, using a stir bar and ice pack. Ponceau S (Sigma) was used to confirm protein transfer to the membrane. Mouse anti-MYC (clone 9E10, Invitrogen), mouse anti-<a href="http://www.yeastgenome.org/locus/S000001334/overview">Hop1</a> (gift from S. Roeder, Yale) and rat anti-Tubulin (Santa Cruz) were used at 1:2500, 1:1000 and 1:5000 dilutions, respectively. Primary antibody incubations were performed overnight at 4°. HRP-conjugated goat anti-mouse (Jackson ImmunoResearch) and donkey anti-rat (Santa Cruz) were applied at 1:5000 for 2 hr at room temperature. Amersham ECL Prime Western Blotting Detection Reagent was used to visualize bands. A Syngene “G”:box” was used to detect chemiluminescence and the Syngene Tools program was used to analyze the data. Normalization was achieved by dividing the intensity of anti-<a href="http://www.yeastgenome.org/locus/S000001334/overview">Hop1</a> signal by the intensity of anti-Tubulin signal in each lane. The fold change of <a href="http://www.yeastgenome.org/locus/S000001334/overview">Hop1</a> abundance in strains with <a href="http://www.yeastgenome.org/locus/S000001334/overview">Hop1</a> overexpression was calculated by dividing the normalized <a href="http://www.yeastgenome.org/locus/S000001334/overview">Hop1</a> intensity in the overexpression strains by the normalized <a href="http://www.yeastgenome.org/locus/S000001334/overview">Hop1</a> intensity in the corresponding control strains. The same approach was used to quantify <a href="http://www.yeastgenome.org/locus/S000006211/overview">Rec8</a>-MYC overexpression. The average of 3 experiments is shown.</p></div><div id="sec-6" class="subsection"><h3>Cytology</h3><p id="p-24">Preparations of surface-spread meiotic chromosomes, their immunostaining and imaging were performed as previously described (<a id="xref-ref-118-2" class="xref-bibr" href="#ref-118">Voelkel-Meiman <em>et al.</em> 2016</a>). In a subset of experiments (noted in text), phleomycin (Invivogen) was added to the sporulation medium at a final concentration of 0, 30, 100 and 120 µg/ml. Primary antibodies were applied for 16 hr at 4° in a humid chamber. The following primary antibodies were used at the indicated dilutions: chicken anti-GFP (1:100) (Abcam), rabbit anti-mCherry (1:100) (Abcam), mouse anti-<a href="http://www.yeastgenome.org/locus/S000001334/overview">Hop1</a> (1:100) (a gift from S. Roeder, Yale), rabbit anti-<a href="http://www.yeastgenome.org/locus/S000000897/overview">Rad51</a> (1:100) (Calbiochem), affinity purified rabbit anti-Zip1(1:100) (YenZym Antibodies; raised against a C-terminal fragment of <a href="http://www.yeastgenome.org/locus/S000002693/overview">Zip1</a> as described (<a id="xref-ref-105-3" class="xref-bibr" href="#ref-105">Sym <em>et al.</em> 1993</a>), mouse anti-MYC (1:100) (clone 9E10, Invitrogen) and a polyclonal antibody raised (ProSci Inc.) against a mixed population of partial Emc11 and <a href="http://www.yeastgenome.org/locus/S000004437/overview">Gmc2</a> proteins (partial proteins kindly provided by Dr. Owen Davies, New Castle University). Fluorophore-conjugated secondary antibodies (Jackson ImmunoResearch) were applied at a 1:200 dilution for 2 hr at room temperature. Following antibody staining, samples were mounted in Vectashield medium (Vector Laboratories) supplemented with 1 µg/ml 4,6-diamidino-2-phenylindole (DAPI).</p><p id="p-25">Imaging was carried out using a Deltavision RT imaging system (Applied Precision/GE) adapted to an Olympus (IX71) microscope and processed using Softworx software (GE). The 3-dimensional distances between the centers of GFP and mCherry foci within each surface-spread nucleus were measured using the Softworx Measure Distance Tool. Distances less than or equal to 0.5 µm were considered paired.</p></div><div id="sec-7" class="subsection"><h3>Data Availability</h3><p id="p-26">Strains and plasmids are available upon request. The authors affirm that all data necessary for confirming the conclusions of this article are represented fully within the article, tables, figures, supplemental figures and supplemental tables. Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.7052072">https://doi.org/10.25387/g3.7052072</a>.</p></div></div><div class="section results" id="sec-8"><h2 class="">Results</h2><div id="sec-9" class="subsection"><h3>Creating strains in which HO endonuclease is the sole initiator of meiotic recombination</h3><p id="p-27">We created strains in which <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> endonuclease is the sole source of meiotic DSB activity by integrating a DNA cassette, <em>P<sub><a href="http://www.yeastgenome.org/locus/S000001056/overview">SPO13</a></sub>-<a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a></em>, which contains the <em><a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a></em> ORF downstream of promoter sequences for the meiosis-specific <em><a href="http://www.yeastgenome.org/locus/S000001056/overview">SPO13</a></em> gene, as was done by a prior study to assess <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated meiotic DSBs (<a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Malkova <em>et al.</em> 1996b</a>). We integrated <em>P<sub><a href="http://www.yeastgenome.org/locus/S000001056/overview">SPO13</a></sub>-<a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a></em> at the <em><a href="http://www.yeastgenome.org/locus/S000000319/overview">LYS2</a></em> locus in strains missing both endogenous <em><a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a></em> activity and <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">SPO11</a></em> (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1A</a>).</p><p id="p-28">A <em><a href="http://www.yeastgenome.org/locus/S000001056/overview">spo13</a></em> mutation was used to facilitate analysis of interhomolog recombination in a <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a></em> strain background. The <em><a href="http://www.yeastgenome.org/locus/S000001056/overview">spo13</a></em> mutation allows the isolation of viable spore products from diploid cells that fail to initiate or complete recombination, because <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a> <a href="http://www.yeastgenome.org/locus/S000001056/overview">spo13</a></em> meiotic cells undergo a single equational division to produce two diploid (dyad) spores instead of four haploid spores (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Klapholz and Esposito 1980</a>). Genetic markers can be inspected in the diploid spores to determine if interhomolog recombination has occurred.</p><p id="p-29">Robust <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> endonuclease activity was detectable in <em>P<sub><a href="http://www.yeastgenome.org/locus/S000001056/overview">SPO13</a></sub>-<a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a></em>-containing meiotic cells, as meiotic spore products frequently carried a chromosome III that had undergone interhomolog recombination at the <em>MAT</em> locus, which carries the sequence that <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> endonuclease targets <em>(<a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> cs)</em> (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Haber 2012</a>). In the absence of meiotic recombination at <em>MAT</em>, a <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a> <a href="http://www.yeastgenome.org/locus/S000001056/overview">spo13</a></em> (<em>MAT</em><strong><em>a</em></strong><em>/MATα</em>) meiotic cell will produce two <em>MAT</em><strong><em>a</em></strong><em>/MATα</em> diploid spores (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1B</a>). Indeed, almost all of the dyad spores that result from control <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a> <a href="http://www.yeastgenome.org/locus/S000001056/overview">spo13</a></em> diploids (with no <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> endonuclease expression) fail to mate with either <em>MAT</em><strong><em>a</em></strong> or <em>MATα</em> cells, as predicted for the <em>MAT</em><strong><em>a</em></strong><em>/MATα</em> genotype (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). By contrast, approximately 40% of dyads from strains carrying <em>P<sub><a href="http://www.yeastgenome.org/locus/S000001056/overview">SPO13</a></sub>-<a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a></em> carry at least one diploid spore that is phenotypically either <em>MAT</em><strong><em>a</em></strong> or <em>MATα</em> (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table1</a>), reflecting meiotic interhomolog recombination at <em>MAT</em>. These data are similar to the findings of (<a id="xref-ref-70-2" class="xref-bibr" href="#ref-70">Malkova <em>et al.</em> 1996b</a>), where 29% of otherwise recombination-less (<em><a href="http://www.yeastgenome.org/locus/S000005194/overview">rad50</a></em> mutant) meioses involving <em>P<sub><a href="http://www.yeastgenome.org/locus/S000001056/overview">SPO13</a></sub>-<a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a></em> expression and a single cleavable <em>MAT</em> locus displayed interhomolog conversion at <em>MAT</em>; this prior study also determined that every <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-associated meiotic conversion event arose from interhomolog recombination, <em>vs.</em> intra-chromosomal recombination using one of the two silent mating type loci.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/491409/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/491409/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title"> HO-mediated meiotic recombination at the <em>MAT</em> locus. Chromosome segregation is predominantly equational during <em>spo11 spo13</em> meiosis, resulting in dyads that are diploid and transheterozygous for <em>MAT</em><em><strong>a</strong></em> and <em>MAT</em><em>α</em> (the parental genotype) and are thus non-maters (nm). HO-mediated interhomolog recombination at <em>MAT</em><em></em> locus during <em>spo11 spo13</em> meiosis can result in mating-capable spores, homozygous for <em>MAT</em><em><strong>a</strong></em> or <em>MAT</em><em>α</em> (Figure 1B). Shown is the percentage of two-spore-viable dyads that carry spores of a specific phenotype (bold), and inferred genotype (unbold), from the diploid strains (LY407, LY208, LY555, LY207, LY322 and LY324) listed. Companion spores within a dyad are separated by a vertical line. The total number of two-spore-viable dyads in the analysis is given in the final column. The proportion of dyads carrying two non-mater (nm) spores is similar between strains carrying different <em>HO cs</em> loci: LY555 = 66.3%, LY207 = 58.3%, LY322 = 53.3% and LY324 = 67.3% (n&gt;500 for each of these strains).</span><div class="sb-div caption-clear"></div></div></div><p id="p-30">We observed similar levels of meiotic recombination at <em>MAT</em> in each of four <em>P<sub><a href="http://www.yeastgenome.org/locus/S000001056/overview">SPO13</a></sub>-<a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a></em> strains containing chromosome IV-targeted <em><a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> cs</em> loci (generated for experiments discussed below; <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). Diploid strains carrying <em>P<sub><a href="http://www.yeastgenome.org/locus/S000001056/overview">SPO13</a></sub>-<a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a></em> display very little mating type switching during vegetative growth (see Materials and Methods), indicating that <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> endonuclease activity in our <em>P<sub><a href="http://www.yeastgenome.org/locus/S000001056/overview">SPO13</a></sub>-<a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a></em> strains is nearly completely meiosis-specific.</p></div><div id="sec-10" class="subsection"><h3>HO forms DSBs at HO cs loci on chromosome IV during spo11 meiosis</h3><p id="p-31"><em>P<sub><a href="http://www.yeastgenome.org/locus/S000001056/overview">SPO13</a></sub>-<a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a></em> strains were used to investigate whether the position of a meiotic DSB along the chromosome is a variable in assuring homologous centromere pairing and SC assembly. Toward this end, we integrated an <em><a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> cs</em> at nine distinct locations on chromosome IV, including a site positioned 250 nucleotides to the left of <em>CENIV</em> (<em><a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> cs2</em>; <a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1A</a>; Materials and Methods). Except for <em>cs1</em> and <em>cs2</em>, the chromosomal position of each of our <em><a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> cs</em> integration sites is associated with a high probability of DSB formation by <a href="http://www.yeastgenome.org/locus/S000001014/overview">Spo11</a> during wild-type meiosis (<a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Blitzblau <em>et al.</em> 2007</a>; <a id="xref-ref-86-3" class="xref-bibr" href="#ref-86">Pan <em>et al.</em> 2011</a>). Neither <em>cs1</em> nor <em>cs2</em> are positioned at sites that have been found to be strongly enriched for meiosis-specific axis proteins but the position of <em>cs2</em> has been found to be strongly enriched for the meiosis-specific cohesin subunit, <a href="http://www.yeastgenome.org/locus/S000006211/overview">Rec8</a> (<a id="xref-ref-87-2" class="xref-bibr" href="#ref-87">Panizza <em>et al.</em> 2011</a>).</p><p id="p-32">We directly evaluated <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated DSB formation in six <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a></em> null strains, each homozygous for a different chromosome IV <em><a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> cs</em> (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). These strains were additionally homozygous for a <em><a href="http://www.yeastgenome.org/locus/S000000897/overview">rad51</a></em> null mutation, in order to slow or abolish the completion of DNA repair and thereby allow us to evaluate the maximal level of <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated DSB formation at these chromosomal sites during meiosis. Genomic DNA was isolated from <em><a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> cs</em>-containing strains at 0, 12, 18 and 24 hr of sporulation; prior studies using this strain background indicate that most cells at the 24 hr time point have reached or progressed beyond late meiotic prophase (<a id="xref-ref-119-1" class="xref-bibr" href="#ref-119">Voelkel-Meiman <em>et al.</em> 2012</a>). Genomic DNA was digested using restriction enzymes that create a 2-7 kb DNA fragment encompassing the <em><a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> cs</em>, and probed for <em><a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> cs</em>-associated sequences using a 500 bp probe on a Southern blot.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/11/3637/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="HO-mediated DSBs at HO cs sequences on chromosome IV in spo11 rad51 meiosis. (A) Southern blot analysis shows DNA cleavage by HO endonuclease at HO cs sequences on chromosome IV in spo11 spo13 rad51 diploid strains (LY491, LY456, LY492, LY481, LY459, LY457 and LY458; see Figure 1 for HO cs positions, Table S4 for strain genotypes). Samples were collected and processed at 0, 12, 18 and 24 hr after placement in sporulation medium. Genomic DNA was digested with restriction enzymes that target sites flanking each of the HO cs loci within a 10 kb region; DNA fragment sizes (kb) are displayed next to blots. Fragments were visualized using a probe that hybridizes to the natMX4 sequence adjacent to each HO cs. In the absence of HO and before entry into meiosis in the presence of HO (0 hr of sporulation), the probe detects a single large fragment that corresponds to an intact fragment of DNA containing the HO cs. In the presence of HO endonuclease, a faster migrating (smaller) fragment is also seen for each of the cut sites beginning at 12hr, except for cs6. The absence of HO-mediated DSBs at cs6 was verified using two different restriction enzymes (See Materials and Methods). (B) Bar graph shows the percent of DNA cut by HO endonuclease within PSPO13-HO spo11 spo13 rad51 meiotic nuclei at the indicated time points. Values were calculated by dividing the intensity of the smaller fragment with the sum of the intensities of the smaller and larger fragment. Bars depict the range given by two experiments." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2138819651" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;HO-mediated DSBs at &lt;em&gt;HO cs&lt;/em&gt; sequences on chromosome IV in &lt;em&gt;spo11 rad51&lt;/em&gt; meiosis. (A) Southern blot analysis shows DNA cleavage by HO endonuclease at &lt;em&gt;HO cs&lt;/em&gt; sequences on chromosome IV in &lt;em&gt;spo11 spo13 rad51&lt;/em&gt; diploid strains (LY491, LY456, LY492, LY481, LY459, LY457 and LY458; see Figure 1 for &lt;em&gt;HO cs&lt;/em&gt; positions, Table S4 for strain genotypes). Samples were collected and processed at 0, 12, 18 and 24 hr after placement in sporulation medium. Genomic DNA was digested with restriction enzymes that target sites flanking each of the &lt;em&gt;HO cs&lt;/em&gt; loci within a 10 kb region; DNA fragment sizes (kb) are displayed next to blots. Fragments were visualized using a probe that hybridizes to the &lt;em&gt;natMX4&lt;/em&gt; sequence adjacent to each &lt;em&gt;HO cs&lt;/em&gt;. In the absence of HO and before entry into meiosis in the presence of HO (0 hr of sporulation), the probe detects a single large fragment that corresponds to an intact fragment of DNA containing the &lt;em&gt;HO cs&lt;/em&gt;. In the presence of HO endonuclease, a faster migrating (smaller) fragment is also seen for each of the cut sites beginning at 12hr, except for &lt;em&gt;cs6&lt;/em&gt;. The absence of HO-mediated DSBs at &lt;em&gt;cs6&lt;/em&gt; was verified using two different restriction enzymes (See Materials and Methods). (B) Bar graph shows the percent of DNA cut by HO endonuclease within &lt;em&gt;P&lt;sub&gt;SPO13&lt;/sub&gt;-HO spo11 spo13 rad51&lt;/em&gt; meiotic nuclei at the indicated time points. Values were calculated by dividing the intensity of the smaller fragment with the sum of the intensities of the smaller and larger fragment. Bars depict the range given by two experiments.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/11/3637/F2.medium.gif" width="440" height="293"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/8/11/3637/F2.medium.gif" width="440" height="293"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/11/3637/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/11/3637/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491290" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-33" class="first-child">HO-mediated DSBs at <em>HO cs</em> sequences on chromosome IV in <em>spo11 rad51</em> meiosis. (A) Southern blot analysis shows DNA cleavage by HO endonuclease at <em>HO cs</em> sequences on chromosome IV in <em>spo11 spo13 rad51</em> diploid strains (LY491, LY456, LY492, LY481, LY459, LY457 and LY458; see <a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1</a> for <em>HO cs</em> positions, Table S4 for strain genotypes). Samples were collected and processed at 0, 12, 18 and 24 hr after placement in sporulation medium. Genomic DNA was digested with restriction enzymes that target sites flanking each of the <em>HO cs</em> loci within a 10 kb region; DNA fragment sizes (kb) are displayed next to blots. Fragments were visualized using a probe that hybridizes to the <em>natMX4</em> sequence adjacent to each <em>HO cs</em>. In the absence of HO and before entry into meiosis in the presence of HO (0 hr of sporulation), the probe detects a single large fragment that corresponds to an intact fragment of DNA containing the <em>HO cs</em>. In the presence of HO endonuclease, a faster migrating (smaller) fragment is also seen for each of the cut sites beginning at 12hr, except for <em>cs6</em>. The absence of HO-mediated DSBs at <em>cs6</em> was verified using two different restriction enzymes (See Materials and Methods). (B) Bar graph shows the percent of DNA cut by HO endonuclease within <em>P<sub>SPO13</sub>-HO spo11 spo13 rad51</em> meiotic nuclei at the indicated time points. Values were calculated by dividing the intensity of the smaller fragment with the sum of the intensities of the smaller and larger fragment. Bars depict the range given by two experiments.</p><div class="sb-div caption-clear"></div></div></div><p id="p-34">In strains devoid of <em>P<sub><a href="http://www.yeastgenome.org/locus/S000001056/overview">SPO13</a></sub>-<a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a></em>, or at the zero-hour time point in strains carrying <em>P<sub><a href="http://www.yeastgenome.org/locus/S000001056/overview">SPO13</a></sub>-<a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a></em>, a single DNA fragment of predicted size was detected (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2A</a>). In strains carrying <em>P<sub><a href="http://www.yeastgenome.org/locus/S000001056/overview">SPO13</a></sub>-<a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a></em> and chromosome IV-targeted <em><a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> cs1</em>, <em>cs2</em>, <em>cs4</em>, <em>cs5</em> or <em>cs7</em>, a faster migrating fragment, corresponding to the product of an <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated DSB, was also detected (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2A</a>). In these five strains, the percentage of DNA cut by <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> endonuclease ranged from ∼20–60% (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2B</a>). These numbers suggest that <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> endonuclease maximally cuts one or two out of the four available chromatids in a given meiotic nucleus. We note the possibility that this calculation underestimates the true number of <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated DSBs, as some DSBs may be undetectable due to hyperresection of sequences flanking the DSB site (which has been noted to occur in <em><a href="http://www.yeastgenome.org/locus/S000000897/overview">rad51</a></em> mutants (<a id="xref-ref-100-3" class="xref-bibr" href="#ref-100">Shinohara <em>et al.</em> 1992</a>). However, our probe did not detect faster migrating <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> cleavage products, which would be expected observable intermediates in the case of substantial resection activity.</p><p id="p-35">Unexpectedly, DSB formation at <em><a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> cs6</em> was not detected in Southern blotting experiments with either of two different restriction enzymes (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2A, B</a>; Materials and Methods). <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> endonuclease is active in the <em>cs6</em> strain, based on observed meiotic recombination at the <em>MAT</em> locus that was comparable to strains carrying the other <em><a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> cs</em> loci (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>). The position of <em>cs6</em> on chromosome IV has been previously classified as a frequent target of <a href="http://www.yeastgenome.org/locus/S000001014/overview">Spo11</a> (<a id="xref-ref-10-4" class="xref-bibr" href="#ref-10">Blitzblau <em>et al.</em> 2007</a>; <a id="xref-ref-86-4" class="xref-bibr" href="#ref-86">Pan <em>et al.</em> 2011</a>); genetic background may affect the accessibility of <em>cs6</em> DNA to the <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> endonuclease in our strain.</p></div><div id="sec-11" class="subsection"><h3>HO promotes interhomolog recombination on chromosome IV in spo11 meiotic nuclei</h3><p id="p-36">Meiotic DSBs preferentially engage the homolog over the sister chromatid for repair (<a id="xref-ref-96-3" class="xref-bibr" href="#ref-96">Schwacha and Kleckner 1994</a>; <a id="xref-ref-98-2" class="xref-bibr" href="#ref-98">Schwacha and Kleckner 1997</a>; <a id="xref-ref-42-5" class="xref-bibr" href="#ref-42">Hong <em>et al.</em> 2013</a>). To ask whether <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated meiotic DSBs on chromosome IV engage the homologous chromosome for repair when <a href="http://www.yeastgenome.org/locus/S000001014/overview">Spo11</a> is absent, we assessed interhomolog crossing over between genetic markers that flank an <em><a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> cs</em> (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>). The <em><a href="http://www.yeastgenome.org/locus/S000000523/overview">LEU2</a></em> gene was introduced near to the centromere on one copy of chromosome IV, either 705 bases to the right of <em>CEN4</em> in <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a> <a href="http://www.yeastgenome.org/locus/S000001056/overview">spo13</a></em> strains homozygous for <em><a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> cs5</em>, <em>cs6</em> or <em>cs7</em> or 2.8 kilobases to left of <em>CEN4</em> in analogous strains homozygous for <em><a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> cs</em>2. In all strains, the <em><a href="http://www.yeastgenome.org/locus/S000001067/overview">THR1</a></em> gene was inserted at coordinate 1,416,000 on the right arm of the chromosome IV carrying <em><a href="http://www.yeastgenome.org/locus/S000000523/overview">LEU2</a>+</em>. The <em><a href="http://www.yeastgenome.org/locus/S000000523/overview">LEU2</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000001067/overview">THR1</a></em> genetic markers thus allowed us to measure the frequency of <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated crossing over.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/11/3637/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="HO-mediated interhomolog recombination during meiosis. (A) Assay for interhomolog recombination at HO cs loci on chromosome IV in spo11 spo13 diploids. LEU2 is inserted at 450 kb (705 bp to the right of CEN4) and THR1 is inserted at 1,416 kb on chromosome IV for all strains, except for cs2 where LEU2 is at 447 kb (2.8 kb to the left of CEN4). A single HO cs is integrated between LEU2 and THR1. spo11 spo13 diploid cells undergo a single equational division during meiosis, resulting in spores carrying a chromatid from each parental homolog. In the absence of HO, each spore receives one LEU2 THR1 chromatid and one chromatid with neither marker (&#x201C;No Recombination&#x201D; column). In the presence of HO mediated DSBs, interhomolog reciprocal crossover recombination can result in a spore lacking the THR1 marker on chromosome IV (accompanied by a sister spore with two THR1 markers; upper right dyad). Because half of the crossover events will be invisible by this assay (lower right dyad), the percentage of apparent reciprocal crossovers were calculated by dividing twice the number of observed Thr- spores by the number of total 2-spore viable dyads (Table 2 and Table 4). (B) The upper bar graph plots the percentage of apparent crossing over calculated as described in (A) (n &gt; 100 2-spore viable dyads assayed; Table 2) in various strains that carry no chromosome IV HO cs (LY208), or that carry distinct chromosome IV HO cs locations (left to right: LY208, LY555, LY207, LY324, LY322). The lower bar graph plots the percentage of apparent crossing over in mutant strains carrying HO cs5 (left to right: LY207, LY459, LY290, LY393, LY904, LY939, LY935, LY957 and LY910). Vegetative cultures of control expressing no HO nor HO cs (LY407) as well as LY208, LY555, LY324, LY322 and LY207 were independently evaluated for the uniform presence of the LEU2 and THR1 marker by assessment of growth of &gt;400 single colonies on selective media." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2138819651" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;HO-mediated interhomolog recombination during meiosis. (A) Assay for interhomolog recombination at &lt;em&gt;HO cs&lt;/em&gt; loci on chromosome IV in &lt;em&gt;spo11 spo13&lt;/em&gt; diploids. &lt;em&gt;LEU2&lt;/em&gt; is inserted at 450 kb (705 bp to the right of &lt;em&gt;CEN4&lt;/em&gt;) and &lt;em&gt;THR1&lt;/em&gt; is inserted at 1,416 kb on chromosome IV for all strains, except for &lt;em&gt;cs2&lt;/em&gt; where &lt;em&gt;LEU2&lt;/em&gt; is at 447 kb (2.8 kb to the left of &lt;em&gt;CEN4&lt;/em&gt;). A single &lt;em&gt;HO cs&lt;/em&gt; is integrated between &lt;em&gt;LEU2&lt;/em&gt; and &lt;em&gt;THR1&lt;/em&gt;. &lt;em&gt;spo11 spo13&lt;/em&gt; diploid cells undergo a single equational division during meiosis, resulting in spores carrying a chromatid from each parental homolog. In the absence of HO, each spore receives one &lt;em&gt;LEU2 THR1&lt;/em&gt; chromatid and one chromatid with neither marker (&#x201C;No Recombination&#x201D; column). In the presence of HO mediated DSBs, interhomolog reciprocal crossover recombination can result in a spore lacking the &lt;em&gt;THR1&lt;/em&gt; marker on chromosome IV (accompanied by a sister spore with two &lt;em&gt;THR1&lt;/em&gt; markers; upper right dyad). Because half of the crossover events will be invisible by this assay (lower right dyad), the percentage of apparent reciprocal crossovers were calculated by dividing twice the number of observed Thr- spores by the number of total 2-spore viable dyads (Table 2 and Table 4). (B) The upper bar graph plots the percentage of apparent crossing over calculated as described in (A) (n &gt; 100 2-spore viable dyads assayed; Table 2) in various strains that carry no chromosome IV &lt;em&gt;HO cs&lt;/em&gt; (LY208), or that carry distinct chromosome IV &lt;em&gt;HO cs&lt;/em&gt; locations (left to right: LY208, LY555, LY207, LY324, LY322). The lower bar graph plots the percentage of apparent crossing over in mutant strains carrying &lt;em&gt;HO cs5&lt;/em&gt; (left to right: LY207, LY459, LY290, LY393, LY904, LY939, LY935, LY957 and LY910). Vegetative cultures of control expressing no HO nor &lt;em&gt;HO cs&lt;/em&gt; (LY407) as well as LY208, LY555, LY324, LY322 and LY207 were independently evaluated for the uniform presence of the &lt;em&gt;LEU2&lt;/em&gt; and &lt;em&gt;THR1&lt;/em&gt; marker by assessment of growth of &gt;400 single colonies on selective media.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/11/3637/F3.medium.gif" width="440" height="342"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/8/11/3637/F3.medium.gif" width="440" height="342"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/11/3637/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/11/3637/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491438" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-37" class="first-child">HO-mediated interhomolog recombination during meiosis. (A) Assay for interhomolog recombination at <em>HO cs</em> loci on chromosome IV in <em>spo11 spo13</em> diploids. <em>LEU2</em> is inserted at 450 kb (705 bp to the right of <em>CEN4</em>) and <em>THR1</em> is inserted at 1,416 kb on chromosome IV for all strains, except for <em>cs2</em> where <em>LEU2</em> is at 447 kb (2.8 kb to the left of <em>CEN4</em>). A single <em>HO cs</em> is integrated between <em>LEU2</em> and <em>THR1</em>. <em>spo11 spo13</em> diploid cells undergo a single equational division during meiosis, resulting in spores carrying a chromatid from each parental homolog. In the absence of HO, each spore receives one <em>LEU2 THR1</em> chromatid and one chromatid with neither marker (“No Recombination” column). In the presence of HO mediated DSBs, interhomolog reciprocal crossover recombination can result in a spore lacking the <em>THR1</em> marker on chromosome IV (accompanied by a sister spore with two <em>THR1</em> markers; upper right dyad). Because half of the crossover events will be invisible by this assay (lower right dyad), the percentage of apparent reciprocal crossovers were calculated by dividing twice the number of observed Thr- spores by the number of total 2-spore viable dyads (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a> and <a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>). (B) The upper bar graph plots the percentage of apparent crossing over calculated as described in (A) (n &gt; 100 2-spore viable dyads assayed; <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>) in various strains that carry no chromosome IV <em>HO cs</em> (LY208), or that carry distinct chromosome IV <em>HO cs</em> locations (left to right: LY208, LY555, LY207, LY324, LY322). The lower bar graph plots the percentage of apparent crossing over in mutant strains carrying <em>HO cs5</em> (left to right: LY207, LY459, LY290, LY393, LY904, LY939, LY935, LY957 and LY910). Vegetative cultures of control expressing no HO nor <em>HO cs</em> (LY407) as well as LY208, LY555, LY324, LY322 and LY207 were independently evaluated for the uniform presence of the <em>LEU2</em> and <em>THR1</em> marker by assessment of growth of &gt;400 single colonies on selective media.</p><div class="sb-div caption-clear"></div></div></div><p id="p-38">In the absence of <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>, <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a> <a href="http://www.yeastgenome.org/locus/S000001056/overview">spo13</a></em> dyads heterozygous for <em><a href="http://www.yeastgenome.org/locus/S000000523/overview">LEU2</a></em> near the centromere and <em><a href="http://www.yeastgenome.org/locus/S000001067/overview">THR1</a></em> on the arm of chromosome IV would be expected to undergo equational division to give diploid dyads containing spores that are each heterozygous for the <em><a href="http://www.yeastgenome.org/locus/S000000523/overview">LEU2</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000001067/overview">THR1</a></em> insertions (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3A</a>, “No Recombination” outcome). An <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated, interhomolog crossover recombination event at an <em><a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> cs</em> to the right of the centromere and between the <em><a href="http://www.yeastgenome.org/locus/S000000523/overview">LEU2</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000001067/overview">THR1</a></em> markers, however, can result in a situation where both chromosomes IV carry one chromatid that is devoid of the <em><a href="http://www.yeastgenome.org/locus/S000001067/overview">THR1</a></em> insertion (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3A</a>, “Recombination” outcome). Half of the diploid dyads arising from meioses involving such recombinant chromosomes IV will carry a spore that is phenotypically Leu+ Thr-. <a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3A</a> illustrates the phenotypes of each spore in such recombinant dyads. Note that a reciprocal crossover at <em>cs2</em>, which is positioned to the left of <em>CENIV</em>, will result in “recombinant” dyads that contain a Leu-, Thr+ spore.</p><p id="p-39">We measured percent of meioses with an apparent reciprocal crossover event by dividing twice the number of “recombinant” dyads (dyads containing a Leu+, Thr- spore for non-<em>cs2</em> strains and containing a Leu-, Thr+ spore for the <em>cs2</em> strain) by the total number of dyads analyzed (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3B</a>, <a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>). Based on this calculation, <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated, interhomolog crossover recombination occurred in ∼21% of meioses for <em><a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> cs5</em>, 18% of meioses for <em><a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> cs7</em>, and 19% of meioses for <em><a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> cs2</em> (closest to <em>CENIV</em>) (<a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2</a>, <a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3B</a>). Consistent with the absence of detectable DSBs at <em><a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> cs6</em>, interhomolog crossover recombination could be detected at this cut site in only 1.3% of meioses (<a id="xref-table-wrap-2-5" class="xref-table" href="#T2">Table 2</a>, <a id="xref-fig-3-7" class="xref-fig" href="#F3">Figure 3B</a>).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/491275/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/491275/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title"> HO-stimulated meiotic recombination in strains with <em>HO cs</em> loci on Chromosome IV. The number of dyads with particular spore phenotypes was used to calculate apparent interhomolog crossover recombination in <em>spo11 spo13</em> meiotic cells with HO-mediated DSB formation at a chromosome IV <em>HO cs</em> location (See Figure 3A). <em>LEU2</em> is at chromosome IV coordinate 450,000 bp, to the immediate right of <em>CEN4</em>,<em></em> except for the <em>cs2</em> strain where <em>LEU2</em> is positioned at 447,000 bp (to the left of <em>CENIV)</em>; <em>THR1</em> is positioned at chromosome IV coordinate 1,416,000. The percentage of apparent crossing over in all strains except LY555 (<em>cs2</em>) was calculated by first doubling the number of dyads carrying a single Leu+, Thr− spore along with a Leu+, Thr+ spore (the first phenotype column), in order to account for the fraction of events not observed, and then by dividing the product by the total number of two-spore-viable dyads (n). The percentage of apparent crossing over in LY555 was calculated by doubling the number of dyads carrying a single Leu−, Thr+ spore along with a Leu+, Thr+ spore (the sixth phenotype column) and dividing this product by the total number of two-spore-viable dyads. The minor population of dyads from LY555 that carry one Leu+, Thr+ spore and one Leu+, Thr− spore could be explained by a reductional chromosome segregation event, may be associated with recombination nearby to the centromere (Simchen and Hugerat 1993). The minor population of dyad spores with loss of both <em>THR1</em> markers in some strains is possibly due to two interhomolog recombination events (involving all 4 chromatids). The minor class dyads were not included in the recombination calculation. A Fisher's Exact test was used to determine whether values differ significantly from the LY207 value. 400 or more colonies from vegetative cultures of LY208, LY555, LY324, LY322 and LY207 were independently evaluated for the uniform presence of the <em>LEU2</em> and <em>THR1</em> markers among all cells in the population by plating on selective media. All colonies evaluated were found to be Leu+, Thr+.</span><div class="sb-div caption-clear"></div></div></div><p id="p-40"><a id="xref-table-wrap-2-6" class="xref-table" href="#T2">Table 2</a> and <a id="xref-fig-3-8" class="xref-fig" href="#F3">Figure 3</a> refer to <em><a href="http://www.yeastgenome.org/locus/S000000523/overview">LEU2</a>-<a href="http://www.yeastgenome.org/locus/S000001067/overview">THR1</a></em> recombinants as “apparent” crossovers, because the Leu+, Thr- spores that we have presumed to be due to interhomolog crossover recombination could instead arise from an interhomolog recombination event that is nonreciprocal, in which sequences encompassing <em><a href="http://www.yeastgenome.org/locus/S000001067/overview">THR1</a></em> are converted without an accompanying crossover. In the case of an interhomolog conversion without a reciprocal crossover, the Leu+, Thr+ sister spore in the recombinant dyad is expected to carry one chromatid devoid of the <em><a href="http://www.yeastgenome.org/locus/S000001067/overview">THR1</a></em> insert. Whereas in the case of a reciprocal crossover, both chromatids in the Leu+, Thr+ spore within a recombinant dyad are expected to contain the <em><a href="http://www.yeastgenome.org/locus/S000001067/overview">THR1</a></em> insert. We used PCR to detect the presence or absence of the <em><a href="http://www.yeastgenome.org/locus/S000001067/overview">THR1</a></em> insert in Thr+ spores of 44 recombinant dyads for <em>cs5</em>-carrying strains, and to detect the presence or absence of the <em><a href="http://www.yeastgenome.org/locus/S000000523/overview">LEU2</a></em> insert in Leu+ spores of 54 recombinant dyads from <em>cs2</em>-carrying strains. We found that the 71% (31 out of 44) of Thr+ spores from <em>cs5</em> recombinant dyads carried only <em><a href="http://www.yeastgenome.org/locus/S000001067/overview">THR1</a></em> insert-carrying chromosomes IV, indicating that 71% of recombinant dyads in this strain result from a reciprocal crossover event. ∼30% of Thr+ spores were heterozygous for <em><a href="http://www.yeastgenome.org/locus/S000001067/overview">THR1</a></em> insert-carrying chromosome IV, indicating that the recombinant spore in these dyads likely resulted from a nonreciprocal interhomolog recombination event instead of a reciprocal crossover. In an analogous analysis for <em>cs2</em>, we found that both chromosomes IV carried the <em><a href="http://www.yeastgenome.org/locus/S000000523/overview">LEU2</a></em> insert in only 52% (28 out of 54) of Leu+ spores from <em>cs2</em> recombinant dyads. Thus, about half of the recombinant dyads from the <em>cs2</em> strain result from a reciprocal crossover event, while the remainder apparently derive from a nonreciprocal recombination event.</p><p id="p-41">The higher fraction of nonreciprocal interhomolog recombination events in <em>cs2</em> <em>vs.</em> <em>cs5</em> strains may be due to the proximity of the <em><a href="http://www.yeastgenome.org/locus/S000000523/overview">LEU2</a></em> marker to the DSB site: the <em><a href="http://www.yeastgenome.org/locus/S000000523/overview">LEU2</a></em> marker is approximately 2.5 kilobases from <em>cs2</em>, whereas the <em><a href="http://www.yeastgenome.org/locus/S000001067/overview">THR1</a></em> insert is ∼580 kilobases from <em>cs5</em>. On the other hand, it is remarkable that ∼30% of interhomolog recombination events in <em>cs5</em> strains involve conversion of sequences ∼580 kilobases from the DSB site. Such remarkably long conversion tracts have been previously associated with <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated DSB repair in <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a></em> meiotic nuclei (<a id="xref-ref-70-3" class="xref-bibr" href="#ref-70">Malkova <em>et al.</em> 1996b</a>; <a id="xref-ref-69-4" class="xref-bibr" href="#ref-69">Malkova <em>et al.</em> 2000</a>) and have been proposed to arise through a Break-Induced Replication (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Kellis <em>et al.</em> 2004</a>) process whereby a strand invasion intermediate is extended by replication through the end of the chromosome instead of undergoing second-end capture (<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Malkova <em>et al.</em> 1996a</a>; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Kraus <em>et al.</em> 2001</a>; <a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Malkova and Ira 2013</a>).</p><p id="p-42">In sum, as was reported in (<a id="xref-ref-69-5" class="xref-bibr" href="#ref-69">Malkova <em>et al.</em> 2000</a>) for the endogenous <em><a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a></em> cs on chromosome III, our data suggest that <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated meiotic DSBs on chromosome IV can engage a non-sister chromatid for their repair. Furthermore, our data indicate that a non-sister chromatid is utilized for <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated meiotic DSB repair in at least 20% of meioses, and that a substantial fraction (∼30%) of <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated interhomolog events involve extremely long conversion tracts.</p></div><div id="sec-12" class="subsection"><h3>Dmc1 is dispensable for HO-mediated interhomolog recombination in meiotic cells</h3><p id="p-43">We next asked whether <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated interhomolog recombination events in <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a></em> meiosis rely on meiosis-specific strand exchange machinery. In <em><a href="http://www.yeastgenome.org/locus/S000000897/overview">rad51</a></em> single or <em><a href="http://www.yeastgenome.org/locus/S000000897/overview">rad51</a> <a href="http://www.yeastgenome.org/locus/S000000981/overview">dmc1</a></em> double mutants, the viability of diploid dyad spores is dramatically decreased, from 74% in the <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a> <a href="http://www.yeastgenome.org/locus/S000001056/overview">spo13</a></em> control meiocytes expressing <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>, to 38% in the analogous strain missing <a href="http://www.yeastgenome.org/locus/S000000897/overview">Rad51</a>, and to 34% in the combined absence of <a href="http://www.yeastgenome.org/locus/S000000897/overview">Rad51</a> and <a href="http://www.yeastgenome.org/locus/S000000981/overview">Dmc1</a> (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>). Genetic removal of <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> endonuclease from the <em><a href="http://www.yeastgenome.org/locus/S000000897/overview">rad51</a></em> single or <em><a href="http://www.yeastgenome.org/locus/S000000897/overview">rad51</a> <a href="http://www.yeastgenome.org/locus/S000000981/overview">dmc1</a></em> double mutant strain restored dyad spore viability to 76% and 77%, respectively (<a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>), indicating that the spore lethality observed in our <em><a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a></em>-expressing, <em><a href="http://www.yeastgenome.org/locus/S000000897/overview">rad51</a></em> strains is due to a failure to repair <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated DSBs.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/491292/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/491292/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title"> Meiotic HO-mediated DSBs lead to spore death in the absence of Rad51 activity or Mre11. The percentage of total dyads (n) from <em>spo11 spo13</em> diploids homozygous for various mutant alleles carrying two viable spores (2-sv), one viable spore (1-sv) and 0 viable spores (0-sv) is shown. The percentage of the total number of spores (nx2) that are viable is given for each strain in the final column (% Spore viability). Asterisks indicate a significant difference between spore viabilities for indicated strains, as determined by a Fisher's Exact test (<em>P</em> ≤ 0.01). Note that LY481 and LY500 were created by replacing <em>lys2::P</em><sub><em>SPO13</em></sub><em>-HO</em> in LY459, and LY393, respectively, with <em>LYS2</em> DNA sequence.</span><div class="sb-div caption-clear"></div></div></div><p id="p-44">We also found that <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated interhomolog crossovers at <em>cs5</em> are nearly abolished in the absence of <a href="http://www.yeastgenome.org/locus/S000000897/overview">Rad51</a>. Apparent crossovers between the <em><a href="http://www.yeastgenome.org/locus/S000000523/overview">LEU2</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000001067/overview">THR1</a></em> markers flanking <em><a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> cs5</em> decreased from 20.8% observed among two-spore viable dyads from <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a> <a href="http://www.yeastgenome.org/locus/S000001056/overview">spo13</a></em> control meiocytes expressing <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>, to 0% and 1.4% observed among two-spore viable dyads from <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a> <a href="http://www.yeastgenome.org/locus/S000001056/overview">spo13</a> <a href="http://www.yeastgenome.org/locus/S000000897/overview">rad51</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a> <a href="http://www.yeastgenome.org/locus/S000001056/overview">spo13</a> <a href="http://www.yeastgenome.org/locus/S000000897/overview">rad51</a> <a href="http://www.yeastgenome.org/locus/S000000981/overview">dmc1</a></em> strains expressing <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>, respectively (<a id="xref-fig-3-9" class="xref-fig" href="#F3">Figure 3B</a>, <a id="xref-table-wrap-2-7" class="xref-table" href="#T2">Table 2</a>). <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated meiotic interhomolog conversion events at the <em>MAT</em> locus also disappeared in <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a> <a href="http://www.yeastgenome.org/locus/S000001056/overview">spo13</a> <a href="http://www.yeastgenome.org/locus/S000000897/overview">rad51</a></em> meiotic cells expressing <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>: 99.0% and 96.4% of two-spore viable dyads from <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a> <a href="http://www.yeastgenome.org/locus/S000001056/overview">spo13</a> <a href="http://www.yeastgenome.org/locus/S000000897/overview">rad51</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a> <a href="http://www.yeastgenome.org/locus/S000001056/overview">spo13</a> <a href="http://www.yeastgenome.org/locus/S000000897/overview">rad51</a> <a href="http://www.yeastgenome.org/locus/S000000981/overview">dmc1</a></em> strains, respectively, exhibited two non-mating spores (n = 105; Table S1A), similar to the proportion of non-mating spores from <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a> <a href="http://www.yeastgenome.org/locus/S000001056/overview">spo13</a></em> strains that lack meiotic <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> expression altogether (<a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>). Furthermore, only 2 out of 181 (1.1%; Table S1B) one-spore viable dyads from <em>P<sub><a href="http://www.yeastgenome.org/locus/S000001056/overview">SPO13</a></sub>-<a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> <a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a> <a href="http://www.yeastgenome.org/locus/S000001056/overview">spo13</a> <a href="http://www.yeastgenome.org/locus/S000000897/overview">rad51</a></em> meiotic cells and 5 out of 363 (1.4%; Table S1B) one-spore viable dyads from <em>P<sub><a href="http://www.yeastgenome.org/locus/S000001056/overview">SPO13</a></sub>-<a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> <a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a> <a href="http://www.yeastgenome.org/locus/S000001056/overview">spo13</a> <a href="http://www.yeastgenome.org/locus/S000000897/overview">rad51</a> <a href="http://www.yeastgenome.org/locus/S000000981/overview">dmc1</a></em> meiotic cells exhibited the capacity to mate; these few <em>MAT</em> homozygotes may have arisen as a consequence of meiotic recombination or chromosome loss. Similarly, only 4 out of 181 (2.2%) and 1 out of 363 (0.3%) one spore viable dyads from <em>P<sub><a href="http://www.yeastgenome.org/locus/S000001056/overview">SPO13</a></sub>-<a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> <a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a> <a href="http://www.yeastgenome.org/locus/S000001056/overview">spo13</a> <a href="http://www.yeastgenome.org/locus/S000000897/overview">rad51</a></em> strains and <em>P<sub><a href="http://www.yeastgenome.org/locus/S000001056/overview">SPO13</a></sub>-<a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> <a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a> <a href="http://www.yeastgenome.org/locus/S000001056/overview">spo13</a> <a href="http://www.yeastgenome.org/locus/S000000897/overview">rad51</a> <a href="http://www.yeastgenome.org/locus/S000000981/overview">dmc1</a></em> strains, respectively, exhibited a non-parental configuration of the <em><a href="http://www.yeastgenome.org/locus/S000000523/overview">LEU2</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000001067/overview">THR1</a></em> markers that flank <em>cs5</em> on chromosome IV.</p><p id="p-45">The low spore viability and absence of interhomolog recombination when <a href="http://www.yeastgenome.org/locus/S000000897/overview">Rad51</a> is missing from <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-expressing, <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a> <a href="http://www.yeastgenome.org/locus/S000001056/overview">spo13</a></em> meiocytes indicate that <a href="http://www.yeastgenome.org/locus/S000000897/overview">Rad51</a> is essential for robust repair of <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated DSBs during meiosis, which aligns with the expectation that strand exchange is a requirement for meiotic recombination. Most strikingly, in contrast to <a href="http://www.yeastgenome.org/locus/S000001014/overview">Spo11</a>-mediated meiotic DSBs, which are capable of utilizing a <a href="http://www.yeastgenome.org/locus/S000000981/overview">Dmc1</a>-only pathway for repair by homologous recombination when <a href="http://www.yeastgenome.org/locus/S000000897/overview">Rad51</a> is absent (<a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">Bishop <em>et al.</em> 1992</a>; <a id="xref-ref-99-4" class="xref-bibr" href="#ref-99">Shinohara <em>et al.</em> 1997a</a>), our data reveal that <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated meiotic DSBs are incapable of undergoing homologous recombination using a <a href="http://www.yeastgenome.org/locus/S000000981/overview">Dmc1</a>-only pathway.</p><p id="p-46">In contrast to the <a href="http://www.yeastgenome.org/locus/S000000897/overview">Rad51</a>-deficient context, no decrease in spore viability is associated with <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> DSB activity at <em>cs5</em> in the absence of <a href="http://www.yeastgenome.org/locus/S000000981/overview">Dmc1</a> (75% spore viability in <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a> <a href="http://www.yeastgenome.org/locus/S000001056/overview">spo13</a> <a href="http://www.yeastgenome.org/locus/S000000981/overview">dmc1</a></em> <em>vs.</em> 74% in <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a> <a href="http://www.yeastgenome.org/locus/S000001056/overview">spo13</a> <a href="http://www.yeastgenome.org/locus/S000000981/overview">DMC1</a></em> strains respectively; <a id="xref-table-wrap-3-4" class="xref-table" href="#T3">Table 3</a>) and apparent crossovers at <em><a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> cs5</em> are only slightly decreased in <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a> <a href="http://www.yeastgenome.org/locus/S000001056/overview">spo13</a></em> meiotic cells missing <a href="http://www.yeastgenome.org/locus/S000000981/overview">Dmc1</a> (15.8% in <em><a href="http://www.yeastgenome.org/locus/S000000981/overview">dmc1</a></em> <em>vs.</em> 20.8% in <em><a href="http://www.yeastgenome.org/locus/S000000981/overview">DMC1</a></em> strains; <a id="xref-fig-3-10" class="xref-fig" href="#F3">Figure 3</a>; <a id="xref-table-wrap-2-8" class="xref-table" href="#T2">Table 2</a>). Furthermore, <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated interhomolog recombination in <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a> <a href="http://www.yeastgenome.org/locus/S000001056/overview">spo13</a></em> meiotic cells appears unaffected by the individual removal of two factors that normally promote the use of the <a href="http://www.yeastgenome.org/locus/S000000981/overview">Dmc1</a> recombinase (<a id="xref-ref-98-3" class="xref-bibr" href="#ref-98">Schwacha and Kleckner 1997</a>; <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Callender <em>et al.</em> 2016</a>): <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated apparent interhomolog crossover levels are unchanged in <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a> <a href="http://www.yeastgenome.org/locus/S000001056/overview">spo13</a></em> meiotic cells missing the meiosis-specific kinase, <a href="http://www.yeastgenome.org/locus/S000005878/overview">Mek1</a> (20.8% in both <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a> <a href="http://www.yeastgenome.org/locus/S000001056/overview">spo13</a></em> control and <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a> <a href="http://www.yeastgenome.org/locus/S000001056/overview">spo13</a> <a href="http://www.yeastgenome.org/locus/S000005878/overview">mek1</a></em> meiotic cells; <a id="xref-fig-3-11" class="xref-fig" href="#F3">Figure 3</a>, <a id="xref-table-wrap-2-9" class="xref-table" href="#T2">Table 2</a>), and in the absence of the meiosis-specific chromosomal protein, <a href="http://www.yeastgenome.org/locus/S000004253/overview">Red1</a> (22.4% in <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a> <a href="http://www.yeastgenome.org/locus/S000001056/overview">spo13</a> <a href="http://www.yeastgenome.org/locus/S000004253/overview">red1</a></em> strains; <a id="xref-fig-3-12" class="xref-fig" href="#F3">Figure 3</a>, <a id="xref-table-wrap-2-10" class="xref-table" href="#T2">Table 2</a>).</p><p id="p-47">Thus, in contrast to wild-type meiosis, where removal of factors that promote <a href="http://www.yeastgenome.org/locus/S000000981/overview">Dmc1</a> utilization causes a severe reduction in interhomolog crossovers (<a id="xref-ref-93-1" class="xref-bibr" href="#ref-93">Rockmill and Roeder 1991</a>; <a id="xref-ref-8-4" class="xref-bibr" href="#ref-8">Bishop <em>et al.</em> 1992</a>; <a id="xref-ref-94-2" class="xref-bibr" href="#ref-94">Rockmill <em>et al.</em> 1995</a>; <a id="xref-ref-99-5" class="xref-bibr" href="#ref-99">Shinohara <em>et al.</em> 1997a</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Bishop <em>et al.</em> 1999</a>; <a id="xref-ref-111-1" class="xref-bibr" href="#ref-111">Tsubouchi and Roeder 2003</a>), <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-initiated interhomolog DSB repair in our experimental (<em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a></em>) strain is not dramatically affected by the loss of <a href="http://www.yeastgenome.org/locus/S000005878/overview">Mek1</a>, <a href="http://www.yeastgenome.org/locus/S000004253/overview">Red1</a>, or the <a href="http://www.yeastgenome.org/locus/S000000981/overview">Dmc1</a> recombinase, apparently because these <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated DSBs can easily access a “<a href="http://www.yeastgenome.org/locus/S000000897/overview">Rad51</a>-only” recombinase pathway.</p></div><div id="sec-13" class="subsection"><h3>Can HO-mediated interhomolog recombination in meiotic cells utilize Dmc1 when Rad51 recombinase activity is absent?</h3><p id="p-48">The data presented above establish a requirement for <a href="http://www.yeastgenome.org/locus/S000000897/overview">Rad51</a> and the lack of a requirement for <a href="http://www.yeastgenome.org/locus/S000000981/overview">Dmc1</a> in the interhomolog repair of <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated DSBs in meiosis. We used a <a href="http://www.yeastgenome.org/locus/S000000897/overview">Rad51</a> separation-of-function genetic context to determine if the <a href="http://www.yeastgenome.org/locus/S000000981/overview">Dmc1</a> recombinase functions redundantly with <a href="http://www.yeastgenome.org/locus/S000000897/overview">Rad51</a> recombinase activity, or whether <a href="http://www.yeastgenome.org/locus/S000000981/overview">Dmc1</a> is completely unavailable to the repair of <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated meiotic DSBs in <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a></em> mutants.</p><p id="p-49">We analyzed <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated meiotic DSB outcomes in the context of the <em><a href="http://www.yeastgenome.org/locus/S000000897/overview">rad51</a>-II3A</em> separation-of-function allele (a kind gift of D. Bishop; <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Cloud <em>et al.</em> 2012</a>). <em><a href="http://www.yeastgenome.org/locus/S000000897/overview">rad51</a>-II3A</em> encodes a protein that lacks strand exchange activity and thus supports a <a href="http://www.yeastgenome.org/locus/S000000981/overview">Dmc1</a>-<a href="http://www.yeastgenome.org/locus/S000000897/overview">Rad51</a> joint recombinase pathway but not a <a href="http://www.yeastgenome.org/locus/S000000897/overview">Rad51</a>-only pathway. We observed robust <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated interhomolog recombination in <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a> <a href="http://www.yeastgenome.org/locus/S000001056/overview">spo13</a></em> meiotic cells homozygous for <em><a href="http://www.yeastgenome.org/locus/S000000897/overview">rad51</a>-II3A</em> (15.7%; <a id="xref-fig-3-13" class="xref-fig" href="#F3">Figure 3</a>, <a id="xref-table-wrap-2-11" class="xref-table" href="#T2">Table 2</a>). Furthermore, we observed a strong diminishment in apparent interhomolog crossovers when <a href="http://www.yeastgenome.org/locus/S000000981/overview">Dmc1</a> is removed from the <em><a href="http://www.yeastgenome.org/locus/S000000897/overview">rad51</a>-II3A</em> strain, from 15.7% in <em><a href="http://www.yeastgenome.org/locus/S000000897/overview">rad51</a>-II3A</em> to 2% in <em><a href="http://www.yeastgenome.org/locus/S000000897/overview">rad51</a>-II3A <a href="http://www.yeastgenome.org/locus/S000000981/overview">dmc1</a></em> (<a id="xref-fig-3-14" class="xref-fig" href="#F3">Figure 3</a>, <a id="xref-table-wrap-2-12" class="xref-table" href="#T2">Table 2</a>), indicating that the interhomolog recombination measured in <em><a href="http://www.yeastgenome.org/locus/S000000897/overview">rad51</a>-II3A</em> mutants is primarily due to <a href="http://www.yeastgenome.org/locus/S000000981/overview">Dmc1</a> recombinase activity.</p><p id="p-50">Altogether these data indicate that <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-initiated, interhomolog recombination intermediates may primarily engage the <a href="http://www.yeastgenome.org/locus/S000000897/overview">Rad51</a> recombinase, but are capable of utilizing <a href="http://www.yeastgenome.org/locus/S000000981/overview">Dmc1</a> as a recombinase so long as <a href="http://www.yeastgenome.org/locus/S000000897/overview">Rad51</a> is also present.</p><p id="p-51"><em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a> <a href="http://www.yeastgenome.org/locus/S000001056/overview">spo13</a> <a href="http://www.yeastgenome.org/locus/S000000897/overview">rad51</a>-II3A</em> meiotic cells that express <em><a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a></em> exhibit diminished spore viability relative to control strains (52% <em>vs.</em> 74% in the control; <a id="xref-table-wrap-3-5" class="xref-table" href="#T3">Table 3</a>). This observation indicates that <a href="http://www.yeastgenome.org/locus/S000000981/overview">Dmc1</a> is only partially capable of rescuing genome-wide <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated meiotic DSB repair when <a href="http://www.yeastgenome.org/locus/S000000897/overview">Rad51</a> recombinase activity is diminished. Consistent with this interpretation, we observe evidence of <a href="http://www.yeastgenome.org/locus/S000000981/overview">Dmc1</a>-mediated recombinase activity even in meiocytes that presumably also sustained lethal unrepaired <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> DSBs due to the absence of <a href="http://www.yeastgenome.org/locus/S000000897/overview">Rad51</a> activity: ∼29% (n = 189) of one-spore-viable dyads from <em><a href="http://www.yeastgenome.org/locus/S000000897/overview">rad51</a>-II3A <a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a> <a href="http://www.yeastgenome.org/locus/S000001056/overview">spo13</a></em> strains expressing <em>P<sub><a href="http://www.yeastgenome.org/locus/S000001056/overview">SPO13</a></sub>-<a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a></em> contain a spore that is homozygous at the <em>MAT</em> locus (presumably due to interhomolog recombination at <em>MAT</em>), in contrast to <em><a href="http://www.yeastgenome.org/locus/S000000897/overview">rad51</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000000897/overview">rad51</a> <a href="http://www.yeastgenome.org/locus/S000000981/overview">dmc1</a> <a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a> <a href="http://www.yeastgenome.org/locus/S000001056/overview">spo13</a></em> strains expressing <em>P<sub><a href="http://www.yeastgenome.org/locus/S000001056/overview">SPO13</a></sub>-<a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a></em> where only ∼1% of one-spore viable dyads carry a spore that is homozygous at <em>MAT</em> (Table S1B).</p></div><div id="sec-14" class="subsection"><h3>HO-mediated DSB repair in meiotic cells is influenced by Mre11 and Xrs2</h3><p id="p-52">The <a href="http://www.yeastgenome.org/locus/S000004837/overview">Mre11</a> nuclease, <a href="http://www.yeastgenome.org/locus/S000005194/overview">Rad50</a> ATPase, and FHA-containing <a href="http://www.yeastgenome.org/locus/S000002777/overview">Xrs2</a> protein can form a complex (“MRX”) that functions in DSB repair in mitotic cells, and has been implicated in both DSB formation and repair during meiosis (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Borde 2007</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Gobbini <em>et al.</em> 2016</a>). We investigated the role of this complex in the repair of <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated, meiotic DSBs in <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a></em> mutants by examining interhomolog recombination in <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a> <a href="http://www.yeastgenome.org/locus/S000001056/overview">spo13</a> <a href="http://www.yeastgenome.org/locus/S000004837/overview">mre11</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a> <a href="http://www.yeastgenome.org/locus/S000001056/overview">spo13</a> <a href="http://www.yeastgenome.org/locus/S000002777/overview">xrs2</a></em> meiotic cells that carry <em>P<sub><a href="http://www.yeastgenome.org/locus/S000001056/overview">SPO13</a></sub>-<a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a></em> and the <em><a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> cs5</em>. We found that, even in the absence of <em><a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a></em> expression, the <em><a href="http://www.yeastgenome.org/locus/S000004837/overview">mre11</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000002777/overview">xrs2</a></em> mutations alone confer a substantial decrease in spore viability to <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a> <a href="http://www.yeastgenome.org/locus/S000001056/overview">spo13</a></em> strains (50% and 65%, respectively, <em>vs.</em> 89% for the <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a> <a href="http://www.yeastgenome.org/locus/S000001056/overview">spo13</a></em> control; <a id="xref-table-wrap-3-6" class="xref-table" href="#T3">Table 3</a>). This data indicates that <a href="http://www.yeastgenome.org/locus/S000004837/overview">Mre11</a> and <a href="http://www.yeastgenome.org/locus/S000002777/overview">Xrs2</a> activities are critical for maintaining genomic integrity during a <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a> <a href="http://www.yeastgenome.org/locus/S000001056/overview">spo13</a></em> meiotic cell cycle even in the absence of meiotic DSBs, and is consistent with a prior finding that diminished <a href="http://www.yeastgenome.org/locus/S000004837/overview">Mre11</a> activity reduces the spore viability of a <em><a href="http://www.yeastgenome.org/locus/S000001056/overview">spo13</a></em> strain (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Ajimura <em>et al.</em> 1993</a>). In strains carrying <em>P<sub><a href="http://www.yeastgenome.org/locus/S000001056/overview">SPO13</a></sub>-<a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a></em>, we observed that the <em><a href="http://www.yeastgenome.org/locus/S000004837/overview">mre11</a></em> mutation confers a further reduction in spore viability (from 50% in the absence of <em>P<sub><a href="http://www.yeastgenome.org/locus/S000001056/overview">SPO13</a></sub>-<a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a></em> to 21% in the presence of <em>P<sub><a href="http://www.yeastgenome.org/locus/S000001056/overview">SPO13</a></sub>-<a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a></em>; <a id="xref-table-wrap-3-7" class="xref-table" href="#T3">Table 3</a>), indicating that <a href="http://www.yeastgenome.org/locus/S000004837/overview">Mre11</a> is also important for the repair of <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated meiotic DSBs. Zero out of thirteen two-spore viable dyads and only 3 out of 60 (5%) one-spore-viable dyads produced by the <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a> <a href="http://www.yeastgenome.org/locus/S000001056/overview">spo13</a> <a href="http://www.yeastgenome.org/locus/S000004837/overview">mre11</a></em> mutant exhibited a Leu+ Thr- marker configuration, but this dataset is too small to conclude that <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated interhomolog and/or crossover repair is dependent on <a href="http://www.yeastgenome.org/locus/S000004837/overview">Mre11</a> in <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a> <a href="http://www.yeastgenome.org/locus/S000001056/overview">spo13</a></em> meiotic cells. The strikingly low spore viability might lead one to conclude that <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated meiotic DSBs uniformly fail to repair in <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a> <a href="http://www.yeastgenome.org/locus/S000001056/overview">spo13</a> <a href="http://www.yeastgenome.org/locus/S000004837/overview">mre11</a></em> meiocytes expressing <em>P<sub><a href="http://www.yeastgenome.org/locus/S000001056/overview">SPO13</a></sub>-<a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a></em>. However, this may not be the case: 46% of two spore viable dyads and 45% of one spore viable dyads from <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a> <a href="http://www.yeastgenome.org/locus/S000001056/overview">spo13</a> <a href="http://www.yeastgenome.org/locus/S000004837/overview">mre11</a></em> mutants were found to exhibit at least one mating capable spore (Table S1). While diploid spores homozygous for <em>MAT</em> can arise from a <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a> <a href="http://www.yeastgenome.org/locus/S000001056/overview">spo13</a></em> meiosis as a consequence of chromosome loss, the near absence of such <em>MAT</em> homozygotes among two-spore-viable and one-spore-viable dyads from <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a> <a href="http://www.yeastgenome.org/locus/S000001056/overview">spo13</a> <a href="http://www.yeastgenome.org/locus/S000000897/overview">rad51</a></em> strains suggests that chromosome loss rarely occurs in these <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-expressing meiocytes and thus at least some <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated DSBs in <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a> <a href="http://www.yeastgenome.org/locus/S000001056/overview">spo13</a> <a href="http://www.yeastgenome.org/locus/S000004837/overview">mre11</a></em> meiotic cells are likely repaired via interhomolog recombination. Thus, we conclude that <a href="http://www.yeastgenome.org/locus/S000004837/overview">Mre11</a> is not uniformly required for the interhomolog repair of <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated DSBs during meiosis, but that a subset of <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated meiotic DSBs cause inviable meiotic products when <a href="http://www.yeastgenome.org/locus/S000004837/overview">Mre11</a> is absent.</p><p id="p-53">Similar to <a href="http://www.yeastgenome.org/locus/S000004837/overview">Mre11</a>, <a href="http://www.yeastgenome.org/locus/S000002777/overview">Xrs2</a> is not required for the interhomolog repair of <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated meiotic DSBs, as is evident by the robust <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated interhomolog recombination previously observed at <em>MAT</em> in <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">SPO11</a> <a href="http://www.yeastgenome.org/locus/S000001056/overview">spo13</a> <a href="http://www.yeastgenome.org/locus/S000002777/overview">xrs2</a></em> strains (<a id="xref-ref-70-4" class="xref-bibr" href="#ref-70">Malkova <em>et al.</em> 1996b</a>), Indeed, similar to <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a> <a href="http://www.yeastgenome.org/locus/S000001056/overview">spo13</a></em> control strains that express <em>P<sub><a href="http://www.yeastgenome.org/locus/S000001056/overview">SPO13</a></sub>-<a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a></em> (<a id="xref-table-wrap-1-6" class="xref-table" href="#T1">Table 1</a>), we found that sporulated <em>P<sub><a href="http://www.yeastgenome.org/locus/S000001056/overview">SPO13</a></sub>-<a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> <a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a> <a href="http://www.yeastgenome.org/locus/S000001056/overview">spo13</a> <a href="http://www.yeastgenome.org/locus/S000002777/overview">xrs2</a></em> strains give rise to a large fraction (31.5%; Table S1A) of two spore viable dyads in which one or both spores is mating-capable, indicative of robust meiotic interhomolog recombination at the <em>MAT</em> locus. In contrast to <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a> <a href="http://www.yeastgenome.org/locus/S000001056/overview">spo13</a> <a href="http://www.yeastgenome.org/locus/S000004837/overview">mre11</a></em> strains, however, meiotic expression of <em><a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a></em> in <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a> <a href="http://www.yeastgenome.org/locus/S000001056/overview">spo13</a> <a href="http://www.yeastgenome.org/locus/S000002777/overview">xrs2</a></em> strains is associated with only a slight reduction in spore viability (from 65% in the absence of <em><a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a></em>, to 55% in the presence of <em><a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a></em>; <a id="xref-table-wrap-3-8" class="xref-table" href="#T3">Table 3</a>), indicating that <a href="http://www.yeastgenome.org/locus/S000002777/overview">Xrs2</a> is less critical than <a href="http://www.yeastgenome.org/locus/S000004837/overview">Mre11</a> for the repair of that putative subset of <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated DSBs that, left unrepaired, cause the low spore viability observed for <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a> <a href="http://www.yeastgenome.org/locus/S000001056/overview">spo13</a> <a href="http://www.yeastgenome.org/locus/S000004837/overview">mre11</a></em> strains.</p><p id="p-54">Interestingly, the level of <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated, apparent meiotic crossovers at <em><a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> cs5</em> is also significantly reduced when <a href="http://www.yeastgenome.org/locus/S000002777/overview">Xrs2</a> is absent, from 20.8% in the <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a> <a href="http://www.yeastgenome.org/locus/S000001056/overview">spo13</a></em> control to 5.5% in <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a> <a href="http://www.yeastgenome.org/locus/S000001056/overview">spo13</a> <a href="http://www.yeastgenome.org/locus/S000002777/overview">xrs2</a></em> meiotic cells (<em>P</em> = 0.0001; <a id="xref-fig-3-15" class="xref-fig" href="#F3">Figure 3</a>, <a id="xref-table-wrap-2-13" class="xref-table" href="#T2">Table 2</a>). Thus, while <a href="http://www.yeastgenome.org/locus/S000002777/overview">Xrs2</a> (and <a href="http://www.yeastgenome.org/locus/S000004837/overview">Mre11</a>) are not essential <em>per se</em> for the interhomolog repair of <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated meiotic DSBs, the strong diminishment in apparent meiotic crossover events in <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a> <a href="http://www.yeastgenome.org/locus/S000001056/overview">spo13</a> <a href="http://www.yeastgenome.org/locus/S000002777/overview">xrs2</a></em> strains may reflect a capacity of <a href="http://www.yeastgenome.org/locus/S000002777/overview">Xrs2</a> to drive <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-initiated interhomolog repair intermediates toward a crossover outcome.</p></div><div id="sec-15" class="subsection"><h3>Does HO-mediated, meiotic DSB repair in the absence of Spo11 rely on canonical meiotic recombination proteins that act downstream of strand exchange?</h3><p id="p-55">We asked whether the <a href="http://www.yeastgenome.org/locus/S000001014/overview">Spo11</a>-independent repair of <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated DSBs utilize meiosis-specific recombination proteins that act downstream of strand exchange. Removal of the SC associated crossover-promoting factors <a href="http://www.yeastgenome.org/locus/S000002693/overview">Zip1</a>, <a href="http://www.yeastgenome.org/locus/S000003218/overview">Zip2</a>, <a href="http://www.yeastgenome.org/locus/S000001335/overview">Zip4</a>, <a href="http://www.yeastgenome.org/locus/S000003220/overview">Mer3</a>, <a href="http://www.yeastgenome.org/locus/S000004386/overview">Zip3</a>, <a href="http://www.yeastgenome.org/locus/S000001891/overview">Msh4</a>-<a href="http://www.yeastgenome.org/locus/S000002313/overview">Msh5</a> or <a href="http://www.yeastgenome.org/locus/S000006085/overview">Mlh3</a> during otherwise wild-type meiosis leads to a ∼50–70% reduction in crossing over in otherwise wild-type meiotic cells (<a id="xref-ref-105-4" class="xref-bibr" href="#ref-105">Sym <em>et al.</em> 1993</a>; <a id="xref-ref-95-1" class="xref-bibr" href="#ref-95">Ross-Macdonald and Roeder 1994</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Hollingsworth <em>et al.</em> 1995</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Hunter and Borts 1997</a>; <a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">Agarwal and Roeder 2000</a>; <a id="xref-ref-82-2" class="xref-bibr" href="#ref-82">Novak <em>et al.</em> 2001</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Börner <em>et al.</em> 2004</a>; <a id="xref-ref-112-2" class="xref-bibr" href="#ref-112">Tsubouchi and Roeder 2006</a>; <a id="xref-ref-118-3" class="xref-bibr" href="#ref-118">Voelkel-Meiman <em>et al.</em> 2016</a>). Of these factors, <a href="http://www.yeastgenome.org/locus/S000002693/overview">Zip1</a>, <a href="http://www.yeastgenome.org/locus/S000003218/overview">Zip2</a> and <a href="http://www.yeastgenome.org/locus/S000001335/overview">Zip4</a> are known to be absolutely essential for SC assembly, while at least some assembled SC has been observed in mutants missing <a href="http://www.yeastgenome.org/locus/S000004386/overview">Zip3</a> or <a href="http://www.yeastgenome.org/locus/S000001891/overview">Msh4</a>-5 complexes (<a id="xref-ref-1-4" class="xref-bibr" href="#ref-1">Agarwal and Roeder 2000</a>; <a id="xref-ref-82-3" class="xref-bibr" href="#ref-82">Novak <em>et al.</em> 2001</a>). We found that removal of <a href="http://www.yeastgenome.org/locus/S000002693/overview">Zip1</a> does not reduce the level of <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated interhomolog recombination observed at <em><a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> cs5</em> in <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a> <a href="http://www.yeastgenome.org/locus/S000001056/overview">spo13</a></em> meiotic cells (21% in <em><a href="http://www.yeastgenome.org/locus/S000002693/overview">zip1</a></em> <em>vs.</em> 20.8% in the control), and removal of <a href="http://www.yeastgenome.org/locus/S000003218/overview">Zip2</a> conferred only a mild (∼16%; <em>P</em> = 0.13) reduction in apparent interhomolog crossovers observed at <em><a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> cs5</em> (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>, <a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a>). However, removal of <a href="http://www.yeastgenome.org/locus/S000004386/overview">Zip3</a>, <a href="http://www.yeastgenome.org/locus/S000001891/overview">Msh4</a>, <a href="http://www.yeastgenome.org/locus/S000006085/overview">Mlh3</a>, or <a href="http://www.yeastgenome.org/locus/S000003220/overview">Mer3</a> resulted in a significant reduction of <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated, apparent interhomolog crossovers in <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a> <a href="http://www.yeastgenome.org/locus/S000001056/overview">spo13</a></em> meioses, by approximately 30–43% (<em>P</em> &lt; 0.0001 for <em><a href="http://www.yeastgenome.org/locus/S000004386/overview">zip3</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000001891/overview">msh4</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em>; <em>P</em> = 0.0095 for <em><a href="http://www.yeastgenome.org/locus/S000003220/overview">mer3</a></em>; <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>, <a id="xref-table-wrap-4-3" class="xref-table" href="#T4">Table 4</a>), suggesting that these meiosis-specific factors do influence the repair of a subset of <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated recombination events. Taken together, these results suggest that <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> meiotic recombination intermediates may be accessible to a subset of SC-associated recombination factors but not those that are absolutely critical for SC assembly.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/11/3637/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="HO-mediated recombination in the absence of Spo11 does not rely heavily on canonical meiotic recombination factors. Bar graph shows the frequency of apparent crossovers in spo11 spo13 strains carrying PSPO13-HO, an HO cs on chromosome IV, and mutant alleles of various recombination factors. Calculations were performed as described in Figure 3 (n &gt; 250; precise values and strain names are reported in Table 4). Left part of graph shows data for strains carrying HO cs5, and the right 3 strains carry HO cs7. Significant deviations, relative to the wild-type value, were determined using Fisher&#x2019;s Exact test (*P-value &#x2264; 0.05, **P-value &#x2264; 0.01, ***P-value &#x2264; 0.001)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2138819651" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;HO-mediated recombination in the absence of Spo11 does not rely heavily on canonical meiotic recombination factors. Bar graph shows the frequency of apparent crossovers in &lt;em&gt;spo11 spo13&lt;/em&gt; strains carrying &lt;em&gt;P&lt;sub&gt;SPO13&lt;/sub&gt;-HO&lt;/em&gt;, an &lt;em&gt;HO cs&lt;/em&gt; on chromosome IV, and mutant alleles of various recombination factors. Calculations were performed as described in Figure 3 (n &gt; 250; precise values and strain names are reported in Table 4). Left part of graph shows data for strains carrying &lt;em&gt;HO cs5&lt;/em&gt;, and the right 3 strains carry &lt;em&gt;HO cs7&lt;/em&gt;. Significant deviations, relative to the wild-type value, were determined using Fisher&#x2019;s Exact test (*&lt;em&gt;P&lt;/em&gt;-value &#x2264; 0.05, **&lt;em&gt;P&lt;/em&gt;-value &#x2264; 0.01, ***&lt;em&gt;P&lt;/em&gt;-value &#x2264; 0.001).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/11/3637/F4.medium.gif" width="436" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/8/11/3637/F4.medium.gif" width="436" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/11/3637/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/11/3637/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491267" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-56" class="first-child">HO-mediated recombination in the absence of Spo11 does not rely heavily on canonical meiotic recombination factors. Bar graph shows the frequency of apparent crossovers in <em>spo11 spo13</em> strains carrying <em>P<sub>SPO13</sub>-HO</em>, an <em>HO cs</em> on chromosome IV, and mutant alleles of various recombination factors. Calculations were performed as described in <a id="xref-fig-3-16" class="xref-fig" href="#F3">Figure 3</a> (n &gt; 250; precise values and strain names are reported in <a id="xref-table-wrap-4-4" class="xref-table" href="#T4">Table 4</a>). Left part of graph shows data for strains carrying <em>HO cs5</em>, and the right 3 strains carry <em>HO cs7</em>. Significant deviations, relative to the wild-type value, were determined using Fisher’s Exact test (*<em>P</em>-value ≤ 0.05, **<em>P</em>-value ≤ 0.01, ***<em>P</em>-value ≤ 0.001).</p><div class="sb-div caption-clear"></div></div></div><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/491327/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/491327/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title"> HO-mediated recombination in the absence of canonical meiotic recombination factors. The number of dyads with particular spore phenotypes was used to calculate apparent meiotic interhomolog crossover recombination in <em>spo11 spo13</em> strains carrying mutant alleles of various meiotic genes and HO-mediated DSB formation at a chromosome IV <em>HO cs</em> location. Genetic markers and calculation of apparent crossover frequency among the total number of two-spore-viable dyads (n) are as described in <a id="xref-table-wrap-2-14" class="xref-table" href="#T2">Table 2</a> and <a id="xref-fig-3-17" class="xref-fig" href="#F3">Figure 3</a>. A Fisher's Exact test was used to determine whether values differ significantly from the control for the <em>HO cs5</em> or the <em>HO cs7</em> strains.</span><div class="sb-div caption-clear"></div></div></div><p id="p-57">We note that the observed reduction in crossing over in <em><a href="http://www.yeastgenome.org/locus/S000004386/overview">zip3</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000001891/overview">msh4</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000003220/overview">mer3</a></em> mutants could reflect a selective diminishment of nonreciprocal recombination events that involve extremely long conversion tract lengths, instead of a diminishment in reciprocal crossovers. Under this scenario, the effect of <a href="http://www.yeastgenome.org/locus/S000004386/overview">Zip3</a>, <a href="http://www.yeastgenome.org/locus/S000001891/overview">Msh4</a>, <a href="http://www.yeastgenome.org/locus/S000006085/overview">Mlh3</a> and <a href="http://www.yeastgenome.org/locus/S000003220/overview">Mer3</a> meiotic crossover-associated proteins on <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-initiated meiotic recombination intermediates would be to promote and/or stabilize longer conversion tracts. To address this possibility, we used PCR analysis to assess the fraction of apparent crossovers resulting from a reciprocal <em>vs.</em> a nonreciprocal event in the <em><a href="http://www.yeastgenome.org/locus/S000004386/overview">zip3</a> <a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em> double mutant strain, which displays a ∼37% reduction in apparent interhomolog crossovers in our <em>cs5</em> experimental strain. We found that 20% (4/20) of apparent interhomolog crossovers in <em><a href="http://www.yeastgenome.org/locus/S000004386/overview">zip3</a> <a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em> dyads (compared to 29% of apparent crossovers in wild-type) remain the result of long-tract interhomolog conversion events, which possibly arise through a BIR mechanism. This result indicates that the ∼37% reduction in apparent interhomolog crossovers caused by the combined absence of <a href="http://www.yeastgenome.org/locus/S000004386/overview">Zip3</a> and <a href="http://www.yeastgenome.org/locus/S000006085/overview">Mlh3</a> does not reflect a selective diminishment of nonreciprocal recombination events.</p><p id="p-58">Factors that process joint molecule recombination intermediates independent of an SC-associated pathway include <a href="http://www.yeastgenome.org/locus/S000000302/overview">Mms4</a>, the <a href="http://www.yeastgenome.org/locus/S000000432/overview">Slx1</a>/<a href="http://www.yeastgenome.org/locus/S000004125/overview">Slx4</a> complex, and <a href="http://www.yeastgenome.org/locus/S000000843/overview">Yen1</a> (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">De Los Santos <em>et al.</em> 2003</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Fricke and Brill 2003</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Ip <em>et al.</em> 2008</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Jessop and Lichten 2008</a>). To assess if <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> DSBs on chromosome IV rely on any of these so-called “class II” recombination factors, we examined <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated crossing over at <em><a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> cs5</em> in <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a></em> strains deficient in meiotic <a href="http://www.yeastgenome.org/locus/S000000302/overview">Mms4</a> and <a href="http://www.yeastgenome.org/locus/S000000843/overview">Yen1</a> (<em>P<sub><a href="http://www.yeastgenome.org/locus/S000006323/overview">CLB2</a></sub>-<a href="http://www.yeastgenome.org/locus/S000000302/overview">MMS4</a></em> and <em>P<sub><a href="http://www.yeastgenome.org/locus/S000006323/overview">CLB2</a></sub>-<a href="http://www.yeastgenome.org/locus/S000000302/overview">MMS4</a> <a href="http://www.yeastgenome.org/locus/S000000843/overview">yen1</a></em> double mutants). We found that <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated interhomolog recombination is not significantly diminished in the absence of these proteins (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a>; <a id="xref-table-wrap-4-5" class="xref-table" href="#T4">Table 4</a>). Finally, we created a strain in which both SC-associated and SC-independent classes of recombination factors are missing. Together, <a href="http://www.yeastgenome.org/locus/S000006085/overview">Mlh3</a>, <a href="http://www.yeastgenome.org/locus/S000000843/overview">Yen1</a>, <a href="http://www.yeastgenome.org/locus/S000000302/overview">Mms4</a> and <a href="http://www.yeastgenome.org/locus/S000000432/overview">Slx1</a>/<a href="http://www.yeastgenome.org/locus/S000004125/overview">Slx4</a> proteins have been implicated in ensuring the resolution of most <a href="http://www.yeastgenome.org/locus/S000001014/overview">Spo11</a>- mediated crossovers during wild-type meiosis (<a id="xref-ref-123-1" class="xref-bibr" href="#ref-123">Zakharyevich <em>et al.</em> 2012</a>). Simultaneous removal of these four factors did not, however, significantly reduce <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated, apparent crossovers at <em>cs5</em>, nor <em>cs7</em> in our <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a> <a href="http://www.yeastgenome.org/locus/S000001056/overview">spo13</a></em> meiotic cells (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4</a>, <a id="xref-table-wrap-4-6" class="xref-table" href="#T4">Table 4</a>).</p><p id="p-59">The <a href="http://www.yeastgenome.org/locus/S000004802/overview">Sgs1</a> helicase has both pro- and anti-crossover activity and its absence allows multi-chromatid exchange events during meiosis (<a id="xref-ref-91-1" class="xref-bibr" href="#ref-91">Rockmill <em>et al.</em> 2003</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Jessop <em>et al.</em> 2006</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">De Muyt <em>et al.</em> 2012</a>). Expression of <em><a href="http://www.yeastgenome.org/locus/S000004802/overview">sgs1</a>-C795</em>, a meiotic null allele (<a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">Mullen <em>et al.</em> 2000</a>; <a id="xref-ref-91-2" class="xref-bibr" href="#ref-91">Rockmill <em>et al.</em> 2003</a>), results in a mild increase in <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated apparent crossover recombination at <em><a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> cs5</em>, from 20.8 to 25.9% <em>P</em> = 0.008; <a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4</a>, <a id="xref-table-wrap-4-7" class="xref-table" href="#T4">Table 4</a>). Thus, <a href="http://www.yeastgenome.org/locus/S000004802/overview">Sgs1</a> may limit <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated interhomolog recombination in <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a></em> meiotic cells.</p><p id="p-60">These data, however, need to be considered in light of the unexpected result that <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a> <a href="http://www.yeastgenome.org/locus/S000001891/overview">msh4</a></em> double mutants and our <em><a href="http://www.yeastgenome.org/locus/S000000432/overview">slx1</a> <a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a> <a href="http://www.yeastgenome.org/locus/S000000843/overview">yen1</a> P<sub><a href="http://www.yeastgenome.org/locus/S000006323/overview">CLB2</a></sub>-<a href="http://www.yeastgenome.org/locus/S000000302/overview">MMS4</a></em> quadruple mutant exhibit no significant decrease in <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated crossing over, despite the reduction in crossing over observed in <em><a href="http://www.yeastgenome.org/locus/S000001891/overview">msh4</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000006085/overview">mlh3</a></em> single mutants. While this observation could reflect <em>bona fide</em> genetic interactions between crossover promoting factors, it may instead indicate that non-specific strain background effects modulate the recombination phenotype.</p><p id="p-61">Altogether, our data suggest that in meiotic cells devoid of <a href="http://www.yeastgenome.org/locus/S000001014/overview">Spo11</a>, <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated interhomolog recombination intermediates might engage with a subset, but not the full cohort of SC associated (class I) crossover factors. Furthermore, <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated meiotic DSBs do not absolutely rely on either class I or class II crossover proteins for interhomolog repair.</p></div><div id="sec-16" class="subsection"><h3>An HO-mediated DSB located proximal or distal to the centromere is not sufficient to promote stable pairwise associations between homologous centromeres during meiosis</h3><p id="p-62">We targeted <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> DSBs to budding yeast’s longest chromosome in order ask whether the position of an interhomolog recombination event with respect to the centromere determines homolog pairing outcomes in meiosis. One homolog pairing process is the recombination-dependent transition from homology-independent centromere “coupling” to homologous centromere pairing (<a id="xref-ref-114-2" class="xref-bibr" href="#ref-114">Tsubouchi and Roeder 2005</a>; <a id="xref-ref-104-1" class="xref-bibr" href="#ref-104">Stewart and Dawson 2008</a>). We used <em>lacO</em>-associated GFP-LacI and <em>tetO</em>-associated TetR-mCherry to assess whether <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated recombination, proximal or distal to <em>CENIV</em>, promotes homologous centromere pairing in <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a></em> mutant meiosis.</p><p id="p-63"><em>lacO</em> DNA sequences were integrated 705 nucleotides to the right of <em>CENIV</em>, and <em>tetO</em> DNA sequences were introduced at coordinate 1,242 kb on chromosome IV (See Materials and Methods). We visualized <em>lacO</em> and <em>tetO</em> DNA sequences on surface spread meiotic chromosomes by <em>lacO</em>- or <em>tetO</em>-binding proteins GFP-LacI or TetR-mCherry, expressed in <em>trans</em> within the same strains. These cytological tools were built into diploids carrying <em><a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> cs2</em>, <em>cs4</em>, or <em>cs5</em>, and into a strain carrying seven active chromosome IV <em><a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> cs</em> loci (although this strain is also heterozygous for <em>cs6</em>, which (presumably) is not actively cut; <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5A</a>). Diploid strains were sporulated for ∼15 hr before preparing surface-spread nuclei for immunofluorescence; an antibody that targets the meiosis-specific chromosomal protein <a href="http://www.yeastgenome.org/locus/S000001334/overview">Hop1</a> was utilized to verify that nuclei had progressed into meiosis.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/11/3637/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A centromere-proximal or distal HO DSB is not sufficient to pair homologous chromosomes. (A) Cartoons show the locations of lacO DNA sequences 705 bp to the right of centromere IV (green) and tetO DNA sequences at 1,242 kb on chromosome IV in strains used for cytology experiments. GFP-LacI and TetR-mCherry, expressed in trans, bind lacO and tetO respectively. Strains homozygous for a single HO cs (top) or carrying seven active HO cs loci (bottom) were utilized in cytological experiments. (B) Images show surface spread meiotic nuclei from wild-type strains (LY42; left column), and from spo11-Y135F strains expressing meiosis-specific HO endonuclease and carrying HO cs5 (LY887; center and right columns) at 15 hr. of sporulation. The presence of Hop1 (not shown) was used to select meiotic nuclei for pairing analysis. The distance between foci corresponding to LacI-GFP bound to lacO sequences near CEN IV (green), and TetR-mCherry bound to tetO sequences on the arm of chromosome IV were considered paired if foci center to foci center &lt; 0.5 &#x3BC;m apart. Bar, 1&#x3BC;m. (C) Bar graphs display the average frequency of CEN IV pairing or chromosome IV arm pairing at 15 hr of sporulation in control (LY176) and chromosome IV HO cs &#x2013; carrying strains (left to right: LY176, LY173, LY174, LY175, LY331and LY887). 100 meiotic nuclei per genotype were analyzed in triplicate (n = 300 total per genotype). Bars depict standard error of the mean. (D) Homologous and non-homologous centromere pairing between centromeres indicated on the x axis was assessed in spo11 mutant strains (left to right: LY303, LY176, LY356, LY357 and LY358) at 15 hr of sporulation (n &gt; 100)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2138819651" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;A centromere-proximal or distal HO DSB is not sufficient to pair homologous chromosomes. (A) Cartoons show the locations of &lt;em&gt;lacO&lt;/em&gt; DNA sequences 705 bp to the right of centromere IV (green) and &lt;em&gt;tetO&lt;/em&gt; DNA sequences at 1,242 kb on chromosome IV in strains used for cytology experiments. GFP-LacI and TetR-mCherry, expressed in &lt;em&gt;trans&lt;/em&gt;, bind &lt;em&gt;lacO&lt;/em&gt; and &lt;em&gt;tetO&lt;/em&gt; respectively. Strains homozygous for a single &lt;em&gt;HO cs&lt;/em&gt; (top) or carrying seven active &lt;em&gt;HO cs&lt;/em&gt; loci (bottom) were utilized in cytological experiments. (B) Images show surface spread meiotic nuclei from wild-type strains (LY42; left column), and from &lt;em&gt;spo11-Y135F&lt;/em&gt; strains expressing meiosis-specific HO endonuclease and carrying &lt;em&gt;HO cs5&lt;/em&gt; (LY887; center and right columns) at 15 hr. of sporulation. The presence of Hop1 (not shown) was used to select meiotic nuclei for pairing analysis. The distance between foci corresponding to LacI-GFP bound to &lt;em&gt;lacO&lt;/em&gt; sequences near &lt;em&gt;CEN IV&lt;/em&gt; (green), and TetR-mCherry bound to &lt;em&gt;tetO&lt;/em&gt; sequences on the arm of chromosome IV were considered paired if foci center to foci center &lt; 0.5 &#x3BC;m apart. Bar, 1&#x3BC;m. (C) Bar graphs display the average frequency of &lt;em&gt;CEN IV&lt;/em&gt; pairing or chromosome IV arm pairing at 15 hr of sporulation in control (LY176) and chromosome IV &lt;em&gt;HO cs&lt;/em&gt; &#x2013; carrying strains (left to right: LY176, LY173, LY174, LY175, LY331and LY887). 100 meiotic nuclei per genotype were analyzed in triplicate (n = 300 total per genotype). Bars depict standard error of the mean. (D) Homologous and non-homologous centromere pairing between centromeres indicated on the &lt;em&gt;x&lt;/em&gt; axis was assessed in &lt;em&gt;spo11&lt;/em&gt; mutant strains (left to right: LY303, LY176, LY356, LY357 and LY358) at 15 hr of sporulation (n &gt; 100).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/11/3637/F5.medium.gif" width="440" height="332"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/8/11/3637/F5.medium.gif" width="440" height="332"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/11/3637/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/11/3637/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491373" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-64" class="first-child">A centromere-proximal or distal HO DSB is not sufficient to pair homologous chromosomes. (A) Cartoons show the locations of <em>lacO</em> DNA sequences 705 bp to the right of centromere IV (green) and <em>tetO</em> DNA sequences at 1,242 kb on chromosome IV in strains used for cytology experiments. GFP-LacI and TetR-mCherry, expressed in <em>trans</em>, bind <em>lacO</em> and <em>tetO</em> respectively. Strains homozygous for a single <em>HO cs</em> (top) or carrying seven active <em>HO cs</em> loci (bottom) were utilized in cytological experiments. (B) Images show surface spread meiotic nuclei from wild-type strains (LY42; left column), and from <em>spo11-Y135F</em> strains expressing meiosis-specific HO endonuclease and carrying <em>HO cs5</em> (LY887; center and right columns) at 15 hr. of sporulation. The presence of Hop1 (not shown) was used to select meiotic nuclei for pairing analysis. The distance between foci corresponding to LacI-GFP bound to <em>lacO</em> sequences near <em>CEN IV</em> (green), and TetR-mCherry bound to <em>tetO</em> sequences on the arm of chromosome IV were considered paired if foci center to foci center &lt; 0.5 μm apart. Bar, 1μm. (C) Bar graphs display the average frequency of <em>CEN IV</em> pairing or chromosome IV arm pairing at 15 hr of sporulation in control (LY176) and chromosome IV <em>HO cs</em> – carrying strains (left to right: LY176, LY173, LY174, LY175, LY331and LY887). 100 meiotic nuclei per genotype were analyzed in triplicate (n = 300 total per genotype). Bars depict standard error of the mean. (D) Homologous and non-homologous centromere pairing between centromeres indicated on the <em>x</em> axis was assessed in <em>spo11</em> mutant strains (left to right: LY303, LY176, LY356, LY357 and LY358) at 15 hr of sporulation (n &gt; 100).</p><div class="sb-div caption-clear"></div></div></div><p id="p-65">We found that among meiotic nuclei from control <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a></em> strains lacking <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> endonuclease, 25% exhibited homologous pairing between the centromere regions of chromosome IV (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5B, C</a>). This frequency of association is higher than expected if pairwise centromere associations between 32 individual chromosomes are random and completely independent of homology in <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a></em> mutants, thus we also examined pairing between the centromere regions of chromosomes III and V, and between centromeres III, V, and IV, in <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a></em> mutant meiotic cells. Consistent with previous studies that found a preferential association between centromeres belonging to chromosomes of similar size (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Lefrançois <em>et al.</em> 2016</a>), centromeres III and V were paired in 22% of surface-spread nuclei (n &gt;100), while centromeres III and IV (which have a greater size differential) displayed a 15% pairing frequency (n &gt; 100; <a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5D</a>).</p><p id="p-66">Having established the baseline of centromere IV pairing in <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a></em> meiotic cells, we next asked whether strains with <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated DSBs on chromosome IV display an increase in homologous centromere pairing. We found that <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a></em> mutants carrying a single <em><a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> cs</em> in conjunction with <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> endonuclease exhibited a similar frequency of paired centromeres IV as <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a></em> cells devoid of <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> endonuclease, even when the <em><a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> cs</em> is adjacent to the <em>CENIV</em> locus. Homologous centromeres IV were paired in 22%, 29% and 24% of meiotic cells at 15 hr of sporulation from strains carrying <em>cs2</em>, cs<em>4</em>, or cs<em>5</em>, respectively (n= 300; <a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5B, C</a>). Furthermore, homologous centromeres IV were paired in 21% of meiotic cells from strains carrying seven active chromosome IV <em><a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> cs</em> loci (<em>cs2</em>, <em>cs4</em>, <em>cs5</em>, <em>cs7</em>, <em>cs8</em>, <em>cs9</em> and <em>cs10</em>; n= 300; <a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5B, C</a>).</p><p id="p-67">Similarly, pairing at a chromosome IV arm location, detected in about 8% of <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a></em> control cells devoid of <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> endonuclease, was detected at approximately the same frequency in <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a></em> cells carrying single or multiple <em><a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> cs</em> loci and meiotic <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> endonuclease: Chromosome IV arm pairing was observed in 9%, 15%, and 11% of meiotic cells from <em><a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a></em>-expressing strains carrying <em>cs2</em>, <em>cs4</em> and <em>cs5</em>, respectively, and in 10% of the meiotic cells from a strain carrying seven active chromosome IV <em><a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> cs</em> loci (<a id="xref-fig-5-6" class="xref-fig" href="#F5">Figure 5B, C</a>).</p><p id="p-68">We conclude that a single <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated recombination event, while capable of promoting interhomolog recombination in ∼20% of meioses, is not sufficient to promote stable pairwise associations between homologous centromeres, even when positioned 250 nucleotides from the centromere.</p></div><div id="sec-17" class="subsection"><h3>An HO-mediated meiotic DSB fails to promote SC assembly in the context of a null or catalytically inactive spo11 allele, even when meiotic axis proteins are overexpressed</h3><p id="p-69">Initial steps in meiotic recombination are required for the assembly of SC in budding yeast, which, at early time points, occurs predominantly from centromere regions (<a id="xref-ref-113-2" class="xref-bibr" href="#ref-113">Tsubouchi <em>et al.</em> 2008</a>). An earlier study reported that <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated recombination between homologous chromosomes III was unaccompanied by SC assembly in <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a></em> meiotic cells (<a id="xref-ref-69-6" class="xref-bibr" href="#ref-69">Malkova <em>et al.</em> 2000</a>), but the small size of chromosome III leaves open the possibility that a partial SC assembly event was missed in this prior experiment. We thus asked whether SC structure(s) assemble on the largest yeast chromosome (IV) in response to an <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated DSB, and furthermore whether the distance from the centromere of the <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated recombination event matters.</p><p id="p-70">Surface-spread meiotic nuclei from <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-expressing and control <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a></em> strains at 15 hr of sporulation were labeled with antibodies against the SC transverse filament protein <a href="http://www.yeastgenome.org/locus/S000002693/overview">Zip1</a> to visualize SC, and with antibodies against the meiosis-specific chromosomal protein <a href="http://www.yeastgenome.org/locus/S000001334/overview">Hop1</a> to verify that nuclei had progressed into meiosis (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6A</a>). In <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a></em> meiotic nuclei, <a href="http://www.yeastgenome.org/locus/S000002693/overview">Zip1</a> localizes diffusely on surface-spread chromosomes, with several brighter foci likely corresponding to centromeres (<a id="xref-ref-114-3" class="xref-bibr" href="#ref-114">Tsubouchi and Roeder 2005</a>). A large fraction of <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a></em> control nuclei also display an aggregate of <a href="http://www.yeastgenome.org/locus/S000002693/overview">Zip1</a> called a polycomplex. <a href="http://www.yeastgenome.org/locus/S000002693/overview">Zip1</a>’s distribution pattern on meiotic chromatin from <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a></em> strains expressing <em><a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a></em> and carrying seven active <em><a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> cs</em> loci (including <em>cs2</em>, positioned immediately adjacent to <em>CENIV</em>) appeared indistinguishable from <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a></em> control meiotic nuclei (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6A, B</a>). The absence of linear <a href="http://www.yeastgenome.org/locus/S000002693/overview">Zip1</a> structures on meiotic chromatin indicates that <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated recombination is incapable of interfacing with and/or successfully activating the molecular pathway(s) that facilitate SC assembly in <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a></em> meiotic nuclei.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/11/3637/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Synaptonemal complex does not assemble in response to an HO-mediated meiotic DSB. (A) Representative surface spread meiotic nuclei from SPO11 (top row; YAM424), or spo11 null strains carrying seven active HO cs loci (cs2, cs4, cs5, cs6, cs7, cs8, cs9, cs10; LY371) at 15 hr of sporulation (upper panel, second and third row). Lower panel displays a representative nucleus from the spo11-Y135F strain carrying HO cs5 and overexpressing Rec8 (LY890; lower panel, top row) and two representative nuclei from the spo11 null strain carrying seven active HO cs and overexpressing Rec8 (LY892, lower panel, bottom two rows) at 24 hr of sporulation in ndt80 strains. Zip1 (green) binds diffusely to and also assembles some bright foci on DAPI-stained meiotic chromatin (blue) from these spo11 strains, regardless of HO-induced meiotic DSBs; polycomplex aggregates of Zip1 (white arrowheads) are often observed. The presence of the meiosis-specific Hop1 protein or Rec8-MYC (red) is displayed in the third column. Bar, 1&#x3BC;m. (B) The proportions of nuclei (n = 50) with different Zip1 and Rec8-MYC distribution phenotypes at 24 hr of sporulation are plotted for a control strain missing PSPO13-HO (LY893), the spo11 null strain carrying seven active HO cs loci (LY371), a Rec8-overexpression control spo11-Y135F strain with no chromosome IV HO cs (LY891), the spo11-Y135F strain carrying HO cs5 and overexpressing Rec8 (LY890) and the spo11 null strain carrying seven active HO cs loci and overexpressing Rec8 (LY892). Immunoblot in (C) shows Rec8-MYC levels in meiotic cells at 24 hr of sporulation from a control SPO11 ndt80 strain in which REC8 is untagged and which happens to also be homozygous for PGAL-HOP1 (LY769), a control spo11-Y135F ndt80 strain homozygous for REC8-MYC (LY893), followed by REC8-overexpressing LY890, LY891 and LY892 strains. The same cultures were used to prepare meiotic surface spread nuclei analyzed at the 24 hr time point. Molecular weight indicators are given (kDa) to the left. (D) Graph plots Rec8-MYC protein levels from a strain carrying two endogenous copies of REC8-MYC and strains carrying 2&#xB5;-REC8-MYC. Tubulin levels were used to normalize Rec8-MYC levels across samples. The average of 3 replicates is plotted; bars give standard error of the mean." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2138819651" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Synaptonemal complex does not assemble in response to an HO-mediated meiotic DSB. (A) Representative surface spread meiotic nuclei from &lt;em&gt;SPO11&lt;/em&gt; (top row; YAM424), or &lt;em&gt;spo11&lt;/em&gt; null strains carrying seven active &lt;em&gt;HO cs&lt;/em&gt; loci (&lt;em&gt;cs2&lt;/em&gt;, &lt;em&gt;cs4&lt;/em&gt;, &lt;em&gt;cs5&lt;/em&gt;, &lt;em&gt;cs6&lt;/em&gt;, &lt;em&gt;cs7&lt;/em&gt;, &lt;em&gt;cs8&lt;/em&gt;, &lt;em&gt;cs9&lt;/em&gt;, &lt;em&gt;cs10&lt;/em&gt;; LY371) at 15 hr of sporulation (upper panel, second and third row). Lower panel displays a representative nucleus from the &lt;em&gt;spo11-Y135F&lt;/em&gt; strain carrying &lt;em&gt;HO cs5&lt;/em&gt; and overexpressing Rec8 (LY890; lower panel, top row) and two representative nuclei from the &lt;em&gt;spo11&lt;/em&gt; null strain carrying seven active &lt;em&gt;HO cs&lt;/em&gt; and overexpressing Rec8 (LY892, lower panel, bottom two rows) at 24 hr of sporulation in &lt;em&gt;ndt80&lt;/em&gt; strains. Zip1 (green) binds diffusely to and also assembles some bright foci on DAPI-stained meiotic chromatin (blue) from these &lt;em&gt;spo11&lt;/em&gt; strains, regardless of HO-induced meiotic DSBs; polycomplex aggregates of Zip1 (white arrowheads) are often observed. The presence of the meiosis-specific Hop1 protein or Rec8-MYC (red) is displayed in the third column. Bar, 1&#x3BC;m. (B) The proportions of nuclei (n = 50) with different Zip1 and Rec8-MYC distribution phenotypes at 24 hr of sporulation are plotted for a control strain missing &lt;em&gt;P&lt;sub&gt;SPO13&lt;/sub&gt;-HO&lt;/em&gt; (LY893), the &lt;em&gt;spo11&lt;/em&gt; null strain carrying seven active &lt;em&gt;HO cs&lt;/em&gt; loci (LY371), a Rec8-overexpression control &lt;em&gt;spo11-Y135F&lt;/em&gt; strain with no chromosome IV &lt;em&gt;HO cs&lt;/em&gt; (LY891), the &lt;em&gt;spo11-Y135F&lt;/em&gt; strain carrying &lt;em&gt;HO cs5&lt;/em&gt; and overexpressing Rec8 (LY890) and the &lt;em&gt;spo11&lt;/em&gt; null strain carrying seven active &lt;em&gt;HO cs&lt;/em&gt; loci and overexpressing Rec8 (LY892). Immunoblot in (C) shows Rec8-MYC levels in meiotic cells at 24 hr of sporulation from a control &lt;em&gt;SPO11 ndt80&lt;/em&gt; strain in which &lt;em&gt;REC8&lt;/em&gt; is untagged and which happens to also be homozygous for &lt;em&gt;P&lt;sub&gt;GAL&lt;/sub&gt;-HOP1&lt;/em&gt; (LY769), a control &lt;em&gt;spo11-Y135F ndt80&lt;/em&gt; strain homozygous for &lt;em&gt;REC8-MYC&lt;/em&gt; (LY893), followed by &lt;em&gt;REC8&lt;/em&gt;-overexpressing LY890, LY891 and LY892 strains. The same cultures were used to prepare meiotic surface spread nuclei analyzed at the 24 hr time point. Molecular weight indicators are given (kDa) to the left. (D) Graph plots Rec8-MYC protein levels from a strain carrying two endogenous copies of &lt;em&gt;REC8-MYC&lt;/em&gt; and strains carrying 2&#xB5;-&lt;em&gt;REC8-MYC&lt;/em&gt;. Tubulin levels were used to normalize Rec8-MYC levels across samples. The average of 3 replicates is plotted; bars give standard error of the mean.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/11/3637/F6.medium.gif" width="440" height="306"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/8/11/3637/F6.medium.gif" width="440" height="306"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/11/3637/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/11/3637/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491314" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-71" class="first-child">Synaptonemal complex does not assemble in response to an HO-mediated meiotic DSB. (A) Representative surface spread meiotic nuclei from <em>SPO11</em> (top row; YAM424), or <em>spo11</em> null strains carrying seven active <em>HO cs</em> loci (<em>cs2</em>, <em>cs4</em>, <em>cs5</em>, <em>cs6</em>, <em>cs7</em>, <em>cs8</em>, <em>cs9</em>, <em>cs10</em>; LY371) at 15 hr of sporulation (upper panel, second and third row). Lower panel displays a representative nucleus from the <em>spo11-Y135F</em> strain carrying <em>HO cs5</em> and overexpressing Rec8 (LY890; lower panel, top row) and two representative nuclei from the <em>spo11</em> null strain carrying seven active <em>HO cs</em> and overexpressing Rec8 (LY892, lower panel, bottom two rows) at 24 hr of sporulation in <em>ndt80</em> strains. Zip1 (green) binds diffusely to and also assembles some bright foci on DAPI-stained meiotic chromatin (blue) from these <em>spo11</em> strains, regardless of HO-induced meiotic DSBs; polycomplex aggregates of Zip1 (white arrowheads) are often observed. The presence of the meiosis-specific Hop1 protein or Rec8-MYC (red) is displayed in the third column. Bar, 1μm. (B) The proportions of nuclei (n = 50) with different Zip1 and Rec8-MYC distribution phenotypes at 24 hr of sporulation are plotted for a control strain missing <em>P<sub>SPO13</sub>-HO</em> (LY893), the <em>spo11</em> null strain carrying seven active <em>HO cs</em> loci (LY371), a Rec8-overexpression control <em>spo11-Y135F</em> strain with no chromosome IV <em>HO cs</em> (LY891), the <em>spo11-Y135F</em> strain carrying <em>HO cs5</em> and overexpressing Rec8 (LY890) and the <em>spo11</em> null strain carrying seven active <em>HO cs</em> loci and overexpressing Rec8 (LY892). Immunoblot in (C) shows Rec8-MYC levels in meiotic cells at 24 hr of sporulation from a control <em>SPO11 ndt80</em> strain in which <em>REC8</em> is untagged and which happens to also be homozygous for <em>P<sub>GAL</sub>-HOP1</em> (LY769), a control <em>spo11-Y135F ndt80</em> strain homozygous for <em>REC8-MYC</em> (LY893), followed by <em>REC8</em>-overexpressing LY890, LY891 and LY892 strains. The same cultures were used to prepare meiotic surface spread nuclei analyzed at the 24 hr time point. Molecular weight indicators are given (kDa) to the left. (D) Graph plots Rec8-MYC protein levels from a strain carrying two endogenous copies of <em>REC8-MYC</em> and strains carrying 2µ-<em>REC8-MYC</em>. Tubulin levels were used to normalize Rec8-MYC levels across samples. The average of 3 replicates is plotted; bars give standard error of the mean.</p><div class="sb-div caption-clear"></div></div></div><p id="p-72"><a href="http://www.yeastgenome.org/locus/S000001014/overview">Spo11</a> requires 9 additional accessory proteins to initiate recombination during meiosis (<a id="xref-ref-61-3" class="xref-bibr" href="#ref-61">Lam and Keeney 2014</a>). A catalytically inactive mutant version of <a href="http://www.yeastgenome.org/locus/S000001014/overview">Spo11</a>, <a href="http://www.yeastgenome.org/locus/S000001014/overview">Spo11</a>-Y135F, is capable of localizing to chromatin during meiosis (<a id="xref-ref-89-1" class="xref-bibr" href="#ref-89">Prieler <em>et al.</em> 2005</a>). With the hope of targeting <a href="http://www.yeastgenome.org/locus/S000001014/overview">Spo11</a> partners to an <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated DSB, we initially created a gene fusion between <a href="http://www.yeastgenome.org/locus/S000001014/overview">Spo11</a>-Y135F and the N terminus of <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> endonuclease in our <em><a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> cs</em>-containing <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a></em> cells. However, the chimeric protein did not exhibit <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> endonuclease activity, as indicated by the absence of meiotic interhomolog recombination at the <em>MAT</em> locus in strains carrying this fusion. We next analyzed strains sustaining <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated, meiotic DSBs with <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a>-Y135F</em> expressed <em>in trans</em>. We found no evidence of homologous centromere pairing nor SC assembly in meiotic nuclei carrying meiotic <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated DSBs at <em>cs5</em>, nor in strains carrying seven active <em>cs</em> loci and homozygous for <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a>-Y135F</em> (<a id="xref-fig-5-7" class="xref-fig" href="#F5">Figures 5</a>, <a id="xref-fig-6-3" class="xref-fig" href="#F6">6A</a>), indicating that the <a href="http://www.yeastgenome.org/locus/S000001014/overview">Spo11</a>-Y135F protein cannot confer to <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated DSBs the capacity to promote meiotic chromosome pairing processes.</p><p id="p-73">SC assembly requires the meiosis-specific cohesin, <a href="http://www.yeastgenome.org/locus/S000006211/overview">Rec8</a> (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Klein <em>et al.</em> 1999</a>). It was previously reported that artificial DSBs, applied to <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a></em> mutant meiotic cells that overexpress <a href="http://www.yeastgenome.org/locus/S000006211/overview">Rec8</a>, lead to <a href="http://www.yeastgenome.org/locus/S000002693/overview">Zip1</a> linear assemblies which were interpreted to be tripartite SC (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Brar <em>et al.</em> 2009</a>). This result led us to wonder whether the overexpression of a meiosis-specific cohesin or another chromosome axis protein would enable <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated DSBs to promote SC assembly. To address this question, we introduced a 2-micrometer plasmid carrying <em><a href="http://www.yeastgenome.org/locus/S000006211/overview">REC8</a>-MYC</em> driven by the <em><a href="http://www.yeastgenome.org/locus/S000001334/overview">HOP1</a></em> promoter in order to overexpress <em><a href="http://www.yeastgenome.org/locus/S000006211/overview">REC8</a>-MYC</em> in <em><a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a></em>-expressing <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a></em> meiotic cells carrying no <em><a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> cs</em>, a single <em><a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> cs</em>, or seven active <em><a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> cs</em> loci. A western blot demonstrated that <a href="http://www.yeastgenome.org/locus/S000006211/overview">Rec8</a>-MYC is approximately five times more abundant in our <em><a href="http://www.yeastgenome.org/locus/S000006211/overview">REC8</a>-MYC</em> overexpression strains relative to the level produced by two chromosomal copies of <em><a href="http://www.yeastgenome.org/locus/S000006211/overview">REC8</a>-MYC</em> (<a id="xref-fig-6-4" class="xref-fig" href="#F6">Figure 6C</a>). However, <a href="http://www.yeastgenome.org/locus/S000002693/overview">Zip1</a>’s distribution on surface-spread chromosomes from these meiotic cells appeared dotty-diffuse regardless of whether the strain expressed higher than wild-type levels of <a href="http://www.yeastgenome.org/locus/S000006211/overview">Rec8</a>-MYC (<a id="xref-fig-6-5" class="xref-fig" href="#F6">Figure 6A, B</a>). Thus, the level of <a href="http://www.yeastgenome.org/locus/S000006211/overview">Rec8</a>-MYC overexpression achieved in this experiment fails to bestow <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated DSBs with the capacity to promote SC assembly.</p><p id="p-74">We also asked whether the overexpression of the meiosis-specific chromosome axis proteins <a href="http://www.yeastgenome.org/locus/S000001334/overview">Hop1</a> and <a href="http://www.yeastgenome.org/locus/S000004253/overview">Red1</a> results in <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated SC assembly in <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a></em> meiotic nuclei. A western blot confirmed that <a href="http://www.yeastgenome.org/locus/S000001334/overview">Hop1</a> is overexpressed at least ninefold relative to control cells in <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a></em> cells carrying a <em><a href="http://www.yeastgenome.org/locus/S000001334/overview">HOP1</a>-<a href="http://www.yeastgenome.org/locus/S000004253/overview">RED1</a></em> 2-micrometer plasmid (a kind gift of N. Hollingsworth; (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Hollingsworth and Ponte 1997</a>) (Figure S1). Immunofluorescence on surface-spread nuclei indicated that, with or without <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated DSBs, <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a></em> cells carrying the <em><a href="http://www.yeastgenome.org/locus/S000001334/overview">HOP1</a>-<a href="http://www.yeastgenome.org/locus/S000004253/overview">RED1</a></em> 2-micrometer plasmid exhibited a dotty-diffuse <a href="http://www.yeastgenome.org/locus/S000002693/overview">Zip1</a> distribution on chromatin and/or a <a href="http://www.yeastgenome.org/locus/S000002693/overview">Zip1</a> polycomplex, with no linear SC-like structures (Figure S1).</p></div><div id="sec-18" class="subsection"><h3>Artificial DSBs supplied en masse by exposure to phleomycin influence SC protein distribution but fail to promote robust SC assembly in spo11 meiotic nuclei</h3><p id="p-75">We wondered whether the failure of our meiotic <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated DSBs to support centromere pairing or SC assembly is due to the fact that these processes need a minimal threshold level of DSBs in order to progress. In order to address this question, we examined pairing and SC protein distribution in meiotic nuclei from <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a></em> mutants that had sustained multiple DSBs due to phleomycin exposure. Whereas surface-spread meiotic nuclei from <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a></em> mutants rarely exhibit more than one or two <a href="http://www.yeastgenome.org/locus/S000000897/overview">Rad51</a> foci, a significant increase in <a href="http://www.yeastgenome.org/locus/S000000897/overview">Rad51</a> foci (ranging between 5-55; <em>P</em> ≤ 0.001; Figure S2A) was observed upon exposing <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a></em> meiotic nuclei to an increasing series of phleomycin doses. The dozens of <a href="http://www.yeastgenome.org/locus/S000000897/overview">Rad51</a> foci observed reflect a snapshot of one stage in the repair of phleomycin-dependent DSBs, and are likely an underestimate of DSBs formed in this experiment. We observed a statistically insignificant increase in homologous pairing at centromeres and arm regions of chromosome IV in strains that experienced the highest dose of phleomycin (<em>P</em> = 0.378; Figure S2B). The vast majority of meiotic nuclei exposed to high doses of phleomycin display a “dotty-diffuse” distribution of SC structural proteins <a href="http://www.yeastgenome.org/locus/S000002693/overview">Zip1</a> and <a href="http://www.yeastgenome.org/locus/S000002854/overview">Ecm11</a> (similar to that seen in the negative control), although a small fraction (∼10%) of phleomycin-exposed nuclei display multiple short linear assemblies of coincident <a href="http://www.yeastgenome.org/locus/S000002693/overview">Zip1</a> and <a href="http://www.yeastgenome.org/locus/S000002854/overview">Ecm11</a> (Figure S2C, D). However, nuclei from cells that had been exposed to high doses of phleomycin only rarely display the <a href="http://www.yeastgenome.org/locus/S000002693/overview">Zip1</a> polycomplex structure that is frequently observed in control nuclei. Taken together, these results indicate that exposure of <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a></em> meiotic cells to phleomycin causes a change in the distribution of <a href="http://www.yeastgenome.org/locus/S000002693/overview">Zip1</a> protein such that <a href="http://www.yeastgenome.org/locus/S000002693/overview">Zip1</a> is less likely to form a polycomplex structure, and may weakly facilitate homologous associations between chromosomes and minimal SC-like assemblies, but does not trigger robust homologous synapsis.</p></div></div><div class="section discussion" id="sec-19"><h2 class="">Discussion</h2><p id="p-76">The current study was motivated by an interest in the molecular mechanism that connects meiotic recombination to processes that generate and reinforce homolog pairing in budding yeast. The <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> endonuclease was found to promote interhomolog crossover recombination in <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a></em> yeast meiosis at higher frequency than in mitotic cells (<a id="xref-ref-70-5" class="xref-bibr" href="#ref-70">Malkova <em>et al.</em> 1996b</a>; <a id="xref-ref-69-7" class="xref-bibr" href="#ref-69">Malkova <em>et al.</em> 2000</a>), consistent with the idea that <a href="http://www.yeastgenome.org/locus/S000001014/overview">Spo11</a>-independent features of the meiotic nucleus may allow DSBs to engage with chromosome pairing and/or recombination pathways that promote favorable outcomes for meiosis; an example of such a mechanism might be the interhomolog bias-promoting activity of meiosis-specific chromosome axis proteins like <a href="http://www.yeastgenome.org/locus/S000004253/overview">Red1</a> and <a href="http://www.yeastgenome.org/locus/S000006211/overview">Rec8</a>, and/or meiosis-specific recombinase machinery (<a id="xref-ref-98-4" class="xref-bibr" href="#ref-98">Schwacha and Kleckner 1997</a>; <a id="xref-ref-42-6" class="xref-bibr" href="#ref-42">Hong <em>et al.</em> 2013</a>). However, the presence of <a href="http://www.yeastgenome.org/locus/S000001014/overview">Spo11</a> has also been found to exert a positive effect on the capacity of artificially-supplied meiotic DSBs to repair as interhomolog crossovers (<a id="xref-ref-70-6" class="xref-bibr" href="#ref-70">Malkova <em>et al.</em> 1996b</a>; <a id="xref-ref-69-8" class="xref-bibr" href="#ref-69">Malkova <em>et al.</em> 2000</a>; <a id="xref-ref-78-4" class="xref-bibr" href="#ref-78">Neale <em>et al.</em> 2002</a>; <a id="xref-ref-74-2" class="xref-bibr" href="#ref-74">Medhi <em>et al.</em> 2016</a>). The extent to which <a href="http://www.yeastgenome.org/locus/S000001014/overview">Spo11</a> activity is uniquely engaged with and capable of driving meiotic chromosome pairing processes remains obscure.</p><p id="p-77">Our investigation into how meiotic DSB repair is coordinated with chromosome pairing in meiotic cells speaks to several questions: Does the position of a DSB relative to the centromere affect the success of early homologous pairing events? Does a threshold level of DSBs need to be met in order to ensure homolog alignment pairing or SC assembly? Are programmed meiotic DSBs specialized in their capacity to promote homolog pairing events? In this study, we asked whether <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated DSBs, many positioned at chromosome IV locations that are frequently cut by <a href="http://www.yeastgenome.org/locus/S000001014/overview">Spo11</a>, are capable of facilitating any of the meiotic chromosome pairing processes that normally accompany programmed DSBs. We furthermore asked the same question of DSBs supplied <em>en masse</em> by exposure to phleomycin.</p><div id="sec-20" class="subsection"><h3>HO-mediated meiotic DSBs on chromosome IV may lack robust repair template bias</h3><p id="p-78">Successful homolog segregation in meiosis relies on recombination-based associations between non-sister chromatids; it follows that meiotic recombination is strongly biased toward utilizing the homolog as repair template (<a id="xref-ref-96-4" class="xref-bibr" href="#ref-96">Schwacha and Kleckner 1994</a>; <a id="xref-ref-98-5" class="xref-bibr" href="#ref-98">Schwacha and Kleckner 1997</a>; <a id="xref-ref-46-3" class="xref-bibr" href="#ref-46">Hunter and Kleckner 2001</a>; <a id="xref-ref-42-7" class="xref-bibr" href="#ref-42">Hong <em>et al.</em> 2013</a>). By contrast, homologous recombination mechanisms in mitotic cells almost exclusively use the sister chromatid (<a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">Kadyk and Hartwell 1992</a>). One question that our experiments highlight is to what extent might a <a href="http://www.yeastgenome.org/locus/S000001014/overview">Spo11</a>-associated DSB be specialized in its capacity to preferentially engage the homolog?</p><p id="p-79">One can explore this question by asking whether <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated, meiotic DSBs exhibit a meiotic-like or mitotic-like preference in repair template choice. An earlier study demonstrated that <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> DSBs frequently access the non-sister chromatid for repair (<a id="xref-ref-69-9" class="xref-bibr" href="#ref-69">Malkova <em>et al.</em> 2000</a>), and our observation of interhomolog repair of meiotic <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> DSBs on chromosome IV is consistent with the idea that <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> DSB repair in meiotic cells has a more “meiotic” <em>vs.</em> “mitotic” repair template bias.</p><p id="p-80">However, considering the frequency of <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated DSB formation in our system, we suggest that <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated DSB repair in <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a></em> meiotic cells exhibit a modest to strong inter-sister repair template bias. Southern blot analysis of total <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated DSBs in our system suggests that 20–60% of total DNA is cleaved in strains homozygous for a given <em><a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a></em> cut site; this suggests that, on average, one or two (out of four) chromatids are cut by <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> in every meiosis. Presuming a random resolution process that does not favor a crossover outcome, the detected ∼15–20% <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated interhomolog crossover events per meiosis suggests that ∼30–40% of meioses may have involved an <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> meiotic DSB that repaired off of the homolog (half resulting in a crossover, and half resulting in a non-crossover). In fact, the aforementioned study that analyzed <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated meiotic DSB repair determined that, in the absence of <a href="http://www.yeastgenome.org/locus/S000001014/overview">Spo11</a>, interhomolog repair may be biased toward a non-crossover outcome: In this earlier study which measured both crossover and noncrossover interhomolog DSB repair events, only 27% of total <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>–mediated interhomolog recombination events gave a crossover outcome in the absence of <a href="http://www.yeastgenome.org/locus/S000001014/overview">Spo11</a> (<em>vs.</em> 52% in the presence of <a href="http://www.yeastgenome.org/locus/S000001014/overview">Spo11</a>) (<a id="xref-ref-69-10" class="xref-bibr" href="#ref-69">Malkova <em>et al.</em> 2000</a>). Taking this into consideration, perhaps as high as ∼50% of meioses involved interhomolog repair of an <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> break in our system. As mentioned in the Results, our Southern blots may underestimate the total number of <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated DSBs generated, due to the possibility of hyperresected DSBs in the <em><a href="http://www.yeastgenome.org/locus/S000000897/overview">rad51</a></em> background. However, even if we assume one <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated DNA break per meiotic cell in our <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a></em> populations, these estimations suggest that <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated meiotic DSBs utilize the sister chromatid with a higher probability than would be expected in the absence of bias (no bias corresponds to an interhomolog:intersister repair ratio of 2:1).</p><p id="p-81">Moreover, since <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated meiotic interhomolog recombination remains robust in <em><a href="http://www.yeastgenome.org/locus/S000004253/overview">red1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000000981/overview">dmc1</a></em>, and <em><a href="http://www.yeastgenome.org/locus/S000005878/overview">mek1</a></em> mutants, the interhomolog to intersister repair template bias we infer for <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated DSBs in <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a></em> meiosis is likely to be similar to the repair template bias associated with <a href="http://www.yeastgenome.org/locus/S000001014/overview">Spo11</a>-mediated DSBs in the absence of <a href="http://www.yeastgenome.org/locus/S000004253/overview">Red1</a>, <a href="http://www.yeastgenome.org/locus/S000000981/overview">Dmc1</a> or <a href="http://www.yeastgenome.org/locus/S000005878/overview">Mek1</a> proteins, which has been reported to be strongly biased toward the sister chromatid (<a id="xref-ref-98-6" class="xref-bibr" href="#ref-98">Schwacha and Kleckner 1997</a>; <a id="xref-ref-42-8" class="xref-bibr" href="#ref-42">Hong <em>et al.</em> 2013</a>).</p><p id="p-82">Our results are consistent with the idea that, relative to the <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> DSBs examined in our system, at least a subset of <a href="http://www.yeastgenome.org/locus/S000001014/overview">Spo11</a>-mediated DNA breaks are more successful at preferentially engaging the homolog for repair.</p></div><div id="sec-21" class="subsection"><h3>Distinct roles for MRX complex components in the repair of a non-Spo11 meiotic DSB</h3><p id="p-83">The <a href="http://www.yeastgenome.org/locus/S000004837/overview">Mre11</a>, <a href="http://www.yeastgenome.org/locus/S000005194/overview">Rad50</a>, and <a href="http://www.yeastgenome.org/locus/S000002777/overview">Xrs2</a> proteins have been implicated in assembling a complex (MRX) which facilitates the formation and repair of <a href="http://www.yeastgenome.org/locus/S000001014/overview">Spo11</a>-mediated DSBs (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Alani <em>et al.</em> 1990</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Johzuka and Ogawa 1995</a>; <a id="xref-ref-110-1" class="xref-bibr" href="#ref-110">Tsubouchi and Ogawa 1998</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Bressan <em>et al.</em> 1999</a>). Separation-of-function <em><a href="http://www.yeastgenome.org/locus/S000004837/overview">mre11</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000005194/overview">rad50</a></em> alleles indicate that these factors have evolved distinct activities that are independently critical for <a href="http://www.yeastgenome.org/locus/S000001014/overview">Spo11</a> DSB formation and repair (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Alani <em>et al.</em> 1990</a>; <a id="xref-ref-110-2" class="xref-bibr" href="#ref-110">Tsubouchi and Ogawa 1998</a>); on the other hand, bypass of <em><a href="http://www.yeastgenome.org/locus/S000002777/overview">xrs2</a></em> null meiotic phenotypes by artificial localization of <a href="http://www.yeastgenome.org/locus/S000004837/overview">Mre11</a> to the nucleus suggests that <a href="http://www.yeastgenome.org/locus/S000002777/overview">Xrs2</a> is indirectly involved in <a href="http://www.yeastgenome.org/locus/S000001014/overview">Spo11</a> DSB formation and repair through its singular capacity to recruit <a href="http://www.yeastgenome.org/locus/S000004837/overview">Mre11</a> (<a id="xref-ref-83-1" class="xref-bibr" href="#ref-83">Oh <em>et al.</em> 2016</a>).</p><p id="p-84">One conclusion we draw from our analysis of <em><a href="http://www.yeastgenome.org/locus/S000004837/overview">mre11</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000002777/overview">xrs2</a></em> mutants is that MRX components are differentially relied upon for the repair of at least a subset of <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated DSBs during meiosis. The <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-associated spore inviability phenotype observed in <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a> <a href="http://www.yeastgenome.org/locus/S000001056/overview">spo13</a> <a href="http://www.yeastgenome.org/locus/S000004837/overview">mre11</a></em> is not observed in <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a> <a href="http://www.yeastgenome.org/locus/S000001056/overview">spo13</a> <a href="http://www.yeastgenome.org/locus/S000002777/overview">xrs2</a></em> strains, suggesting that a subset of <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated meiotic DSBs critically depend upon <a href="http://www.yeastgenome.org/locus/S000004837/overview">Mre11</a> but not <a href="http://www.yeastgenome.org/locus/S000002777/overview">Xrs2</a> for their repair. In light of the idea that <a href="http://www.yeastgenome.org/locus/S000002777/overview">Xrs2</a>’s role in MRX function during meiosis is thought to be through its capacity to recruit <a href="http://www.yeastgenome.org/locus/S000004837/overview">Mre11</a> to the nucleus, the successful repair of <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated meiotic DSBs in <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a> <a href="http://www.yeastgenome.org/locus/S000001056/overview">spo13</a> <a href="http://www.yeastgenome.org/locus/S000002777/overview">xrs2</a></em> meiocytes implies that <a href="http://www.yeastgenome.org/locus/S000002777/overview">Xrs2</a> is not required to recruit <a href="http://www.yeastgenome.org/locus/S000004837/overview">Mre11</a> to the <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated DNA damage that relies on <a href="http://www.yeastgenome.org/locus/S000004837/overview">Mre11</a> for repair.</p><p id="p-85">Second, we observed robust meiotic interhomolog recombination at <em>MAT</em> in both <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a> <a href="http://www.yeastgenome.org/locus/S000001056/overview">spo13</a> <a href="http://www.yeastgenome.org/locus/S000004837/overview">mre11</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a> <a href="http://www.yeastgenome.org/locus/S000001056/overview">spo13</a> <a href="http://www.yeastgenome.org/locus/S000002777/overview">xrs2</a></em> strains expressing <em>P<sub><a href="http://www.yeastgenome.org/locus/S000001056/overview">SPO13</a></sub>-<a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a></em>. This result suggests that while a subset of <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated meiotic DSBs critically depends upon <a href="http://www.yeastgenome.org/locus/S000004837/overview">Mre11</a> for their repair, neither <a href="http://www.yeastgenome.org/locus/S000004837/overview">Mre11</a> nor <a href="http://www.yeastgenome.org/locus/S000002777/overview">Xrs2</a> is essential <em>per se</em> for the interhomolog repair of <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated meiotic DSBs. A prior study’s finding that <a href="http://www.yeastgenome.org/locus/S000002777/overview">Xrs2</a> and <a href="http://www.yeastgenome.org/locus/S000005194/overview">Rad50</a> are dispensable for <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated meiotic DSB repair at <em>MAT</em> (<a id="xref-ref-70-7" class="xref-bibr" href="#ref-70">Malkova <em>et al.</em> 1996b</a>) is consistent with our observation and furthermore indicates that each MRX component is dispensable for the interhomolog repair of at least some <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated meiotic DSBs when <a href="http://www.yeastgenome.org/locus/S000001014/overview">Spo11</a> is absent. This is in contrast to <a href="http://www.yeastgenome.org/locus/S000001014/overview">Spo11</a>-mediated meiotic DSBs, which rely on <a href="http://www.yeastgenome.org/locus/S000004837/overview">Mre11</a> and <a href="http://www.yeastgenome.org/locus/S000005194/overview">Rad50</a>, and possibly <a href="http://www.yeastgenome.org/locus/S000002777/overview">Xrs2</a>, for their repair during meiosis (<a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">Alani <em>et al.</em> 1990</a>; <a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">Nairz and Klein 1997</a>; <a id="xref-ref-110-3" class="xref-bibr" href="#ref-110">Tsubouchi and Ogawa 1998</a>; <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Borde 2007</a>; <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Gobbini <em>et al.</em> 2016</a>).</p><p id="p-86">Our data also indicates that while dispensable for repair <em>per se</em>, <a href="http://www.yeastgenome.org/locus/S000002777/overview">Xrs2</a> nevertheless influences the repair outcome of meiotic <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> DSBs, at least those formed at <em><a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> cs5</em>. Apparent crossover levels on chromosome IV diminished by 74% at <em><a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> cs5</em> in <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a> <a href="http://www.yeastgenome.org/locus/S000001056/overview">spo13</a></em> mutants devoid of <a href="http://www.yeastgenome.org/locus/S000002777/overview">Xrs2</a>. The fact that <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated interhomolog conversions at the <em>MAT</em> locus remained unchanged in the <em><a href="http://www.yeastgenome.org/locus/S000002777/overview">xrs2</a></em> mutant relative to the control (<em><a href="http://www.yeastgenome.org/locus/S000002777/overview">XRS2</a></em>) strain leads us to suggest that <a href="http://www.yeastgenome.org/locus/S000002777/overview">Xrs2</a> activity may engage with <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated interhomolog recombination intermediates in a manner that promotes a crossover outcome. However, we note that data from earlier studies, while not completely conclusive, indicates little effect of <a href="http://www.yeastgenome.org/locus/S000005194/overview">Rad50</a> or <a href="http://www.yeastgenome.org/locus/S000002777/overview">Xrs2</a> on the likelihood of a crossover outcome associated with <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated meiotic recombination at the <em>MAT</em> locus (if one compares <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated crossover outcomes at the <em>MAT</em> locus determined for <em><a href="http://www.yeastgenome.org/locus/S000005194/overview">rad50</a></em> mutants and inferred for <em><a href="http://www.yeastgenome.org/locus/S000002777/overview">xrs2</a></em> mutants (<a id="xref-ref-70-8" class="xref-bibr" href="#ref-70">Malkova <em>et al.</em> 1996b</a>), with crossover outcomes at the <em><a href="http://www.yeastgenome.org/locus/S000000523/overview">LEU2</a></em> locus determined for <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a></em> mutants (<a id="xref-ref-69-11" class="xref-bibr" href="#ref-69">Malkova <em>et al.</em> 2000</a>)). Taking these earlier results into consideration, diminished <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated crossing over at <em>cs5</em> among diploid dyads from <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a> <a href="http://www.yeastgenome.org/locus/S000001056/overview">spo13</a> <a href="http://www.yeastgenome.org/locus/S000002777/overview">xrs2</a></em> strains may not be due to <a href="http://www.yeastgenome.org/locus/S000002777/overview">Xrs2</a>’s involvement in the crossover/noncrossover decision, but instead may reflect a role for <a href="http://www.yeastgenome.org/locus/S000002777/overview">Xrs2</a> in promoting interhomolog (over intersister) repair at certain select DSB locations in the genome, for example at <em>cs5</em> but not at <em>MAT</em>.</p></div><div id="sec-22" class="subsection"><h3>HO-mediated meiotic DSBs may be accessed by a subset of class I recombination proteins</h3><p id="p-87"><a id="xref-ref-69-12" class="xref-bibr" href="#ref-69">Malkova <em>et al.</em> (2000)</a> demonstrated that <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated meiotic interhomolog repair events in <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">SPO11</a>+</em> cells rely on the meiosis-specific crossover promoting complex, MutSγ, to a similar extent as <a href="http://www.yeastgenome.org/locus/S000001014/overview">Spo11</a>-mediated meiotic DSBs. Interestingly, this prior study also provided evidence that <a href="http://www.yeastgenome.org/locus/S000001014/overview">Spo11</a> activity can confer on <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> DSBs, <em>in trans</em>, an increased capacity to resolve as a crossover. <a id="xref-ref-74-3" class="xref-bibr" href="#ref-74">Medhi <em>et al.</em> (2016)</a> also reported evidence in support of the idea that <a href="http://www.yeastgenome.org/locus/S000001014/overview">Spo11</a> increases the likelihood that an artificially-supplied meiotic DSB will repair via a “meiotic-like” crossover pathway. However, whether artificial DSBs utilize meiosis-specific crossover factors, such as MutSγ, when <a href="http://www.yeastgenome.org/locus/S000001014/overview">Spo11</a> is absent was not examined in prior studies. We find that <a href="http://www.yeastgenome.org/locus/S000001891/overview">Msh4</a>-<a href="http://www.yeastgenome.org/locus/S000002313/overview">Msh5</a> (MutSγ), as well as <a href="http://www.yeastgenome.org/locus/S000004386/overview">Zip3</a>, <a href="http://www.yeastgenome.org/locus/S000006085/overview">Mlh3</a> and <a href="http://www.yeastgenome.org/locus/S000003220/overview">Mer3</a> may influence a substantial fraction (∼40%) of <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated meiotic interhomolog repair events even when <a href="http://www.yeastgenome.org/locus/S000001014/overview">Spo11</a> is absent, but <a href="http://www.yeastgenome.org/locus/S000002693/overview">Zip1</a> and <a href="http://www.yeastgenome.org/locus/S000003218/overview">Zip2</a> have little or no influence over the interhomolog repair of <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated, meiotic DSBs. Taken together, our results and the results of <a id="xref-ref-69-13" class="xref-bibr" href="#ref-69">Malkova <em>et al.</em> (2000)</a> suggest that additional factors, apart from access of the <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-recombination intermediate to MutSγ <em>per se</em> must be involved in <a href="http://www.yeastgenome.org/locus/S000001014/overview">Spo11</a>’s capacity to promote a crossover outcome to interhomolog <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> DSB repair events in meiotic cells; we speculate that one of these factors could be proper engagement with a class I recombination complex that includes the <a href="http://www.yeastgenome.org/locus/S000002693/overview">Zip1</a> and <a href="http://www.yeastgenome.org/locus/S000003218/overview">Zip2</a> (and associated) proteins.</p></div><div id="sec-23" class="subsection"><h3>Artificial DSBs are a poor substitute for yeast Spo11 in promoting homologous pairing or synapsis from centromeres</h3><p id="p-88">We are particularly interested in the molecular features of a meiotic DSB that facilitate the process of homologous centromere pairing. In budding yeast, centromeres may be the first regions of meiotic chromosomes to undergo stable homologous alignment in response to recombination, since SC assembly initiates earliest from centromere regions (<a id="xref-ref-113-3" class="xref-bibr" href="#ref-113">Tsubouchi <em>et al.</em> 2008</a>). However, centromere pairing is also independent of SC assembly as it remains intact in mutants that are missing certain building block components of the SC, such as <a href="http://www.yeastgenome.org/locus/S000002854/overview">Ecm11</a> and <a href="http://www.yeastgenome.org/locus/S000004437/overview">Gmc2</a> (<a id="xref-ref-59-3" class="xref-bibr" href="#ref-59">Kurdzo <em>et al.</em> 2017</a>). Our data indicate that artificial DSBs, generated close to or far from a centromere, and either delivered as singular events by the <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> endonuclease or <em>en masse</em> by the radiomimetic drug phleomycin, are unable to promote stable homologous centromere pairing in <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a></em> meiotic cells. This data suggests that (at least a subset of) <a href="http://www.yeastgenome.org/locus/S000001014/overview">Spo11</a>-associated recombination events are uniquely specialized to facilitate this chromosome reorganization process.</p><p id="p-89">Synapsis normally initiates from centromere regions as well as from interhomolog recombination events along chromosome arms (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Chua and Roeder 1998</a>; <a id="xref-ref-1-5" class="xref-bibr" href="#ref-1">Agarwal and Roeder 2000</a>; <a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Henderson <em>et al.</em> 2004</a>; <a id="xref-ref-113-4" class="xref-bibr" href="#ref-113">Tsubouchi <em>et al.</em> 2008</a>). A prior study reported that meiotic <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> DSBs on the small chromosome III fail to promote SC assembly (<a id="xref-ref-69-14" class="xref-bibr" href="#ref-69">Malkova <em>et al.</em> 2000</a>). Similarly, despite their capacity to promote interhomolog recombination, we observed that <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> DSBs positioned at <a href="http://www.yeastgenome.org/locus/S000001014/overview">Spo11</a> hotspot locations close to or far from the centromere in <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a></em> meiotic cells are incapable of even partial SC assembly on the long chromosome IV.</p><p id="p-90">Our data thus reveal that <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated DSBs, while capable of promoting interhomolog crossover recombination during meiotic prophase, are nevertheless <em>incapable</em> of promoting homologous centromere pairing or robust SC assembly. These observations indicate that the artificial DSBs examined in this study are processed in a fundamentally different manner from at least a subset of programmed (<a href="http://www.yeastgenome.org/locus/S000001014/overview">Spo11</a>-mediated) DSBs.</p><p id="p-91">It is important to bear in mind that not all <a href="http://www.yeastgenome.org/locus/S000001014/overview">Spo11</a>-mediated DSBs are necessarily capable of promoting homologous centromere pairing and synapsis during wild-type meiosis; some programmed meiotic DSBs may be processed similarly to the artificially-supplied DSBs examined in this study. This consideration does not diminish the importance of understanding the molecular basis that underlies the difference between programmed and artificially-supplied meiotic DSBs with respect to their capacity to drive homologous pairing and synapsis.</p><p id="p-92">An alternative explanation for the failure of artificially-supplied DSBs to drive meiotic chromosomal outcomes, which does not involve a specialized function of <a href="http://www.yeastgenome.org/locus/S000001014/overview">Spo11</a>, is that either the chromosomal position, timing, or overall abundance of DSBs is the critical feature that determines whether one or more DSBs can promote homologous centromere pairing and/or SC assembly. Indeed, the extent of synapsis in meiotic nuclei has been observed to be directly correlated with <a href="http://www.yeastgenome.org/locus/S000001014/overview">Spo11</a> DSB abundance (<a id="xref-ref-38-3" class="xref-bibr" href="#ref-38">Henderson <em>et al.</em> 2004</a>; <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Brar <em>et al.</em> 2009</a>; <a id="xref-ref-92-1" class="xref-bibr" href="#ref-92">Rockmill <em>et al.</em> 2013</a>); this correlation could be explained by a reliance of homologous synapsis mechanisms on the chromosomal position, timing, or background abundance of one or more programmed DSBs.</p><p id="p-93">While it is formally possible that either chromosomal position, timing or overall DSB abundance is the primary reason for the failure of artificially-supplied DSBs to promote centromere pairing and synapsis in <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a></em> meiotic nuclei, this explanation alone is challenging to reconcile with our experimental data. First, in stark contrast to even <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a></em> hypomorphic mutants that have severe reductions in programmed DSB abundance (<a id="xref-ref-38-4" class="xref-bibr" href="#ref-38">Henderson <em>et al.</em> 2004</a>; <a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">Brar <em>et al.</em> 2009</a>; <a id="xref-ref-92-2" class="xref-bibr" href="#ref-92">Rockmill <em>et al.</em> 2013</a>), the <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated DSBs in our experiments uniformly fail to promote even partial SC assembly outcomes. This is true even in the context of prolonged arrest provided by an <em><a href="http://www.yeastgenome.org/locus/S000001166/overview">ndt80</a></em> null mutation; the removal of the <a href="http://www.yeastgenome.org/locus/S000001166/overview">Ndt80</a> transcription factor prevents progression beyond mid-late meiotic prophase and theoretically would allow <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated DSBs to accumulate and give incipient SC assembly events additional time to develop into mature SC stretches, as is reflected by the <em><a href="http://www.yeastgenome.org/locus/S000001166/overview">ndt80</a></em>-mediated rescue of synapsis in <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a></em> severe hypomorphs (<a id="xref-ref-92-3" class="xref-bibr" href="#ref-92">Rockmill <em>et al.</em> 2013</a>). Second, our high-dose phleomycin experiments likely generate dozens to hundreds of DSBs in meiotic nuclei that are at multiple stages of prophase (as our strains progress through meiosis asynchronously), but phleomycin-induced DSBs fail to promote robust homologous centromere pairing or synapsis in any <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a> <a href="http://www.yeastgenome.org/locus/S000001166/overview">ndt80</a></em> meiotic nuclei examined after a prolonged duration (24 hr) in sporulation conditions. Thus, while not unequivocally ruling out alternative models, our experimental data favors the possibility that <a href="http://www.yeastgenome.org/locus/S000001014/overview">Spo11</a>-association may be a critical prerequisite for a meiotic DSB to engage with the molecular mechanisms that drive centromere pairing and SC assembly.</p></div><div id="sec-24" class="subsection"><h3>A possible molecular basis for the capacity of meiotic DSBs to promote homolog pairing processes</h3><p id="p-94">How might pro-centromere pairing/synapsis DSBs differ from meiotic DSBs that fail to promote pairing and synapsis outcomes? Homologous centromere pairing may be regulated by the phosphorylation of <a href="http://www.yeastgenome.org/locus/S000002693/overview">Zip1</a> by the <a href="http://www.yeastgenome.org/locus/S000000340/overview">Mec1</a> kinase. Prior to meiotic recombination initiation in budding yeast meiosis, <a href="http://www.yeastgenome.org/locus/S000002693/overview">Zip1</a> mediates the pairwise association of centromeres without regard to homology (<a id="xref-ref-114-4" class="xref-bibr" href="#ref-114">Tsubouchi and Roeder 2005</a>). <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Falk <em>et al.</em> (2010)</a> demonstrated that <a href="http://www.yeastgenome.org/locus/S000002693/overview">Zip1</a>’s capacity to mediate centromere association is abolished by phosphorylation of serine residue 75; these authors propose that Mec1-mediated phosphorylation of <a href="http://www.yeastgenome.org/locus/S000002693/overview">Zip1</a> might serve to promote the dissolution of centromere coupling events in coordination with recombination initiation, facilitating the subsequent formation of homologous centromere interactions. The failure of <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>- or phleomycin-induced meiotic DSBs to promote robust homologous centromere pairing may not be due to the absence of <a href="http://www.yeastgenome.org/locus/S000000340/overview">Mec1</a> kinase activity <em>per se</em>, however, as artificial DSBs, like <a href="http://www.yeastgenome.org/locus/S000001014/overview">Spo11</a> DSBs, have been reported to activate <a href="http://www.yeastgenome.org/locus/S000000340/overview">Mec1</a> and <a href="http://www.yeastgenome.org/locus/S000000184/overview">Tel1</a> kinases (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Lydall <em>et al.</em> 1996</a>; <a id="xref-ref-116-1" class="xref-bibr" href="#ref-116">Usui <em>et al.</em> 2001</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Cartagena-Lirola <em>et al.</em> 2006</a>) and to phosphorylate the meiosis-specific kinase, <a href="http://www.yeastgenome.org/locus/S000005878/overview">Mek1</a> during meiosis (<a id="xref-ref-121-1" class="xref-bibr" href="#ref-121">Wan <em>et al.</em> 2004</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Cartagena-Lirola <em>et al.</em> 2008</a>). Instead, the presence of activated <a href="http://www.yeastgenome.org/locus/S000000340/overview">Mec1</a> in a local, specialized chromosomal context might serve to influence centromere behavior. Alternatively, or in addition, perhaps successful pro-centromere pairing DSB events are specialized in activating a second regulatory factor required in addition to activated <a href="http://www.yeastgenome.org/locus/S000000340/overview">Mec1</a> for the dissolution of non-homologous centromere associations, or for the re-establishment of centromere pairing between homologs.</p><p id="p-95">While our experiments do not reveal a mechanism, our data do offer some clues that may be mechanistically relevant for understanding how certain meiotic DSBs are uniquely capable of facilitating synapsis. SC assembly normally occurs downstream of a discrete intermediate step in the homologous recombination process (<a id="xref-ref-38-5" class="xref-bibr" href="#ref-38">Henderson <em>et al.</em> 2004</a>; <a id="xref-ref-125-2" class="xref-bibr" href="#ref-125">Zickler and Kleckner 2015</a>). As discussed above, certain meiosis-specific recombination factors (such as <a href="http://www.yeastgenome.org/locus/S000004386/overview">Zip3</a>, <a href="http://www.yeastgenome.org/locus/S000003220/overview">Mer3</a>, <a href="http://www.yeastgenome.org/locus/S000001891/overview">Msh4</a> and <a href="http://www.yeastgenome.org/locus/S000006085/overview">Mlh3</a>) appear to functionally influence <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated, meiotic recombination intermediates in the absence of <a href="http://www.yeastgenome.org/locus/S000001014/overview">Spo11</a>. However, <a href="http://www.yeastgenome.org/locus/S000004386/overview">Zip3</a>, <a href="http://www.yeastgenome.org/locus/S000003220/overview">Mer3</a>, <a href="http://www.yeastgenome.org/locus/S000001891/overview">Msh4</a> and <a href="http://www.yeastgenome.org/locus/S000006085/overview">Mlh3</a> are unlikely to be acting on <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> recombination intermediates in their normal “class I” manner since other factors in the same recombination pathway, such as <a href="http://www.yeastgenome.org/locus/S000002693/overview">Zip1</a> and <a href="http://www.yeastgenome.org/locus/S000003218/overview">Zip2</a>, do not similarly engage with <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated recombination intermediates, at least when <a href="http://www.yeastgenome.org/locus/S000001014/overview">Spo11</a> is absent. Importantly, while <a href="http://www.yeastgenome.org/locus/S000002693/overview">Zip1</a>, <a href="http://www.yeastgenome.org/locus/S000003218/overview">Zip2</a>, <a href="http://www.yeastgenome.org/locus/S000004386/overview">Zip3</a> and <a href="http://www.yeastgenome.org/locus/S000001891/overview">Msh4</a>-<a href="http://www.yeastgenome.org/locus/S000002313/overview">Msh5</a> act in the same (class I) crossover recombination pathway during wild-type meiosis, <a href="http://www.yeastgenome.org/locus/S000002693/overview">Zip1</a> and <a href="http://www.yeastgenome.org/locus/S000003218/overview">Zip2</a> play a more critical role in SC assembly than <a href="http://www.yeastgenome.org/locus/S000004386/overview">Zip3</a> and <a href="http://www.yeastgenome.org/locus/S000001891/overview">Msh4</a> during wild-type meiosis. Thus, we suggest that the success of a meiotic DSB to promote SC assembly depends upon its early engagement with recombination complexes that properly coordinate <a href="http://www.yeastgenome.org/locus/S000004386/overview">Zip3</a> and <a href="http://www.yeastgenome.org/locus/S000001891/overview">Msh4</a> with the pro-crossover proteins that are essential for SC assembly (<a href="http://www.yeastgenome.org/locus/S000002693/overview">Zip1</a>, <a href="http://www.yeastgenome.org/locus/S000003218/overview">Zip2</a>, <a href="http://www.yeastgenome.org/locus/S000001335/overview">Zip4</a>, <a href="http://www.yeastgenome.org/locus/S000001196/overview">Spo16</a>). The engagement of a meiotic DSB with this sort of “complete” class I recombination pathway could dually ensure that 1) the ensuing recombination intermediate is channeled into a stable crossover-associated configuration, and 2) global alignment of homologous chromosomes, via the promotion of processes like centromere pairing and assembly of tripartite SC, is achieved.</p><p id="p-96">Several pieces of evidence also suggest the possibility that pro-synapsis recombination intermediates may uniquely interface with the chromosome axis. While DNA that is most frequently cleaved by <a href="http://www.yeastgenome.org/locus/S000001014/overview">Spo11</a> lies in chromosome “loop” regions, away from axis locations where the meiosis-specific <a href="http://www.yeastgenome.org/locus/S000001334/overview">Hop1</a> and <a href="http://www.yeastgenome.org/locus/S000004253/overview">Red1</a> proteins are enriched (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Gerton <em>et al.</em> 2000</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Blat <em>et al.</em> 2002</a>; <a id="xref-ref-10-5" class="xref-bibr" href="#ref-10">Blitzblau <em>et al.</em> 2007</a>; <a id="xref-ref-86-5" class="xref-bibr" href="#ref-86">Pan <em>et al.</em> 2011</a>), the axis localization of several of <a href="http://www.yeastgenome.org/locus/S000001014/overview">Spo11</a>’s accessory proteins (<a id="xref-ref-87-3" class="xref-bibr" href="#ref-87">Panizza <em>et al.</em> 2011</a>) and the reliance of meiotic DSB formation on <a href="http://www.yeastgenome.org/locus/S000001334/overview">Hop1</a> and <a href="http://www.yeastgenome.org/locus/S000004253/overview">Red1</a> (<a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Mao-Draayer <em>et al.</em> 1996</a>; <a id="xref-ref-98-7" class="xref-bibr" href="#ref-98">Schwacha and Kleckner 1997</a>; <a id="xref-ref-122-2" class="xref-bibr" href="#ref-122">Xu <em>et al.</em> 1997</a>) suggests that many <a href="http://www.yeastgenome.org/locus/S000001014/overview">Spo11</a> DSB sites are physically close to chromosome axes during break formation and subsequent repair steps (<a id="xref-ref-87-4" class="xref-bibr" href="#ref-87">Panizza <em>et al.</em> 2011</a>). The axis-association of DNA sequences undergoing a DSB is furthermore consistent with the localization of “recombination nodules” (structures associated with ongoing DNA repair visualized by electron microscopy) between intimately-aligned homologous chromosome axes (<a id="xref-ref-124-1" class="xref-bibr" href="#ref-124">Zickler and Kleckner 1999</a>). Finally, <em><a href="http://www.yeastgenome.org/locus/S000001334/overview">hop1</a></em> or <em><a href="http://www.yeastgenome.org/locus/S000006211/overview">rec8</a></em> separation-of-function alleles that confer proficiency in meiotic DSB formation are nevertheless defective in homologous synapsis (<a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">Klein <em>et al.</em> 1999</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Carballo <em>et al.</em> 2008</a>; <a id="xref-ref-13-4" class="xref-bibr" href="#ref-13">Brar <em>et al.</em> 2009</a>).</p><p id="p-97">The possibility that axis proximity is a critical prerequisite for a newly-initiated DSB to engage with meiotic-like repair pathways is supported by a recent finding that repair of a VDE-mediated DNA break during meiosis utilizes the meiosis-specific MutLγ complex more frequently when the break is positioned within axis-protein enriched DNA (<a id="xref-ref-74-4" class="xref-bibr" href="#ref-74">Medhi <em>et al.</em> 2016</a>). Importantly, this effect was found to be dependent on the presence of <a href="http://www.yeastgenome.org/locus/S000001014/overview">Spo11</a> in <em>trans</em>. Furthermore, in their classic study, Thorne and Byers noted a role for the meiotic chromosome axis-associated protein, <a href="http://www.yeastgenome.org/locus/S000001334/overview">Hop1</a>, in enabling artificial DSBs to rescue the chromosome segregation functions of <a href="http://www.yeastgenome.org/locus/S000001014/overview">Spo11</a> in meiotic cells (Thorne Lw 1993).</p><p id="p-98">Our finding that interhomolog repair of <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated meiotic DSBs in <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a></em> meiocytes both fails to promote centromere pairing and SC assembly, and occurs independently of the chromosome-axis associated protein <a href="http://www.yeastgenome.org/locus/S000004253/overview">Red1</a> and the <a href="http://www.yeastgenome.org/locus/S000005878/overview">Mek1</a> kinase, further strengthens the correlation between the chromosome axis and a mechanism that coordinates DSB repair with homologous synapsis in budding yeast. A prior study indicated that artificial DSBs can rescue <a href="http://www.yeastgenome.org/locus/S000001014/overview">Spo11</a>’s SC assembly function if the <a href="http://www.yeastgenome.org/locus/S000006211/overview">Rec8</a> cohesin protein is overexpressed (<a id="xref-ref-13-5" class="xref-bibr" href="#ref-13">Brar <em>et al.</em> 2009</a>), but we were unable to observe robust synapsis by <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated DSBs when <a href="http://www.yeastgenome.org/locus/S000006211/overview">Rec8</a>, <a href="http://www.yeastgenome.org/locus/S000001334/overview">Hop1</a> or <a href="http://www.yeastgenome.org/locus/S000004253/overview">Red1</a> proteins were overexpressed in our strain background. While alternative models remain possible, our results together with previously published data support the possibility that <a href="http://www.yeastgenome.org/locus/S000001014/overview">Spo11</a> maintains a specialized capacity to ensure (even <em>in trans</em>) that at least a subset of DSBs are processed in the context of axis protein-regulated structures.</p></div><div id="sec-25" class="subsection"><h3>Programmed meiotic DSBs may trigger global chromosome dynamics through a specialized interface with Dmc1-associated factors</h3><p id="p-99">Our experiments also reveal the possibility that a constraint to repair using the <a href="http://www.yeastgenome.org/locus/S000000981/overview">Dmc1</a> recombinase may be a unique property of (at least a large subset of) <a href="http://www.yeastgenome.org/locus/S000001014/overview">Spo11</a> DSBs. Interestingly, Smith and colleagues also found that the repair of a set of non-canonical DSBs in fission yeast meiosis also occurs independently of <a href="http://www.yeastgenome.org/locus/S000000981/overview">Dmc1</a>, but is dependent on <a href="http://www.yeastgenome.org/locus/S000000897/overview">Rad51</a>, unlike programmed meiotic DSBs in fission yeast meiocytes (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Farah <em>et al.</em> 2005</a>).</p><p id="p-100"><a id="xref-ref-98-8" class="xref-bibr" href="#ref-98">Schwacha and Kleckner (1997)</a> characterized <a href="http://www.yeastgenome.org/locus/S000004253/overview">Red1</a> and <a href="http://www.yeastgenome.org/locus/S000000981/overview">Dmc1</a> as important core features of an “interhomolog only” pathway for DSB repair, in which axis-associated <a href="http://www.yeastgenome.org/locus/S000004253/overview">Red1</a> is responsible for coupling the maturation of recombination intermediates to the <a href="http://www.yeastgenome.org/locus/S000000981/overview">Dmc1</a> recombinase. In the absence of <a href="http://www.yeastgenome.org/locus/S000004253/overview">Red1</a>, remnant meiotic DSBs do not engage with this “interhomolog only” pathway and instead undergo repair without interhomolog template bias. Similarly, we show that the interhomolog repair of an <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a> DSB during meiosis does not rely on <a href="http://www.yeastgenome.org/locus/S000004253/overview">Red1</a> nor <a href="http://www.yeastgenome.org/locus/S000000981/overview">Dmc1</a>, nor <a href="http://www.yeastgenome.org/locus/S000005878/overview">Mek1</a> (which targets multiple molecular pathways that coordinately promote the use of the <a href="http://www.yeastgenome.org/locus/S000000981/overview">Dmc1</a> recombinase by <a href="http://www.yeastgenome.org/locus/S000001014/overview">Spo11</a>-associated meiotic DSBs) and appears to exhibit low interhomolog repair template bias. We therefore speculate that pro-synapsis, <a href="http://www.yeastgenome.org/locus/S000001014/overview">Spo11</a>-associated DSBs may differ from <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-associated DSBs in their capacity to productively interface with the <a href="http://www.yeastgenome.org/locus/S000004253/overview">Red1</a>-<a href="http://www.yeastgenome.org/locus/S000001334/overview">Hop1</a>-<a href="http://www.yeastgenome.org/locus/S000005878/overview">Mek1</a>-<a href="http://www.yeastgenome.org/locus/S000000981/overview">Dmc1</a> “interhomolog only” pathway for DSB repair. Artificially supplied DSBs in <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a></em> meiotic cells have been shown to induce <a href="http://www.yeastgenome.org/locus/S000005878/overview">Mek1</a> phosphorylation (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Cartagena-Lirola <em>et al.</em> 2008</a>), indicating that events apart from <a href="http://www.yeastgenome.org/locus/S000005878/overview">Mek1</a> activation are required for artificially-supplied DSBs to properly engage with the “interhomolog only” mechanism.</p><p id="p-101">While we propose that pro-synapsis DSBs may differ from artificially-supplied DSBs in the manner by which they engage with a <a href="http://www.yeastgenome.org/locus/S000000981/overview">Dmc1</a>-associated pathway for meiotic recombination, the capacity to promote SC assembly is not likely to be due solely to utilization of the <a href="http://www.yeastgenome.org/locus/S000000981/overview">Dmc1</a> recombinase <em>per se</em>. As is particularly evident in strains that progress through meiotic prophase more slowly (such as those of the BR background), the absence of <a href="http://www.yeastgenome.org/locus/S000000981/overview">Dmc1</a> or <a href="http://www.yeastgenome.org/locus/S000005878/overview">Mek1</a> diminishes but does not abolish synapsis, indicating the existence of redundant molecular mechanisms to couple <a href="http://www.yeastgenome.org/locus/S000001014/overview">Spo11</a> DSBs to SC assembly (<a id="xref-ref-93-2" class="xref-bibr" href="#ref-93">Rockmill and Roeder 1991</a>; <a id="xref-ref-94-3" class="xref-bibr" href="#ref-94">Rockmill <em>et al.</em> 1995</a>; <a id="xref-ref-111-2" class="xref-bibr" href="#ref-111">Tsubouchi and Roeder 2003</a>). Our <em><a href="http://www.yeastgenome.org/locus/S000000897/overview">rad51</a></em> separation-of-function experiments furthermore indicate that <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-mediated meiotic DSBs are not prohibited from utilizing the <a href="http://www.yeastgenome.org/locus/S000000981/overview">Dmc1</a> recombinase, and we found that forcing phleomycin-induced or <a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a>-induced meiotic DNA breaks to repair using the <a href="http://www.yeastgenome.org/locus/S000000981/overview">Dmc1</a> recombinase (through use of the <em><a href="http://www.yeastgenome.org/locus/S000000897/overview">rad51</a>-II3A</em> allele) does not confer a robust capacity to promote synapsis (Figure S3). We speculate that pro-synapsis DSBs may possess the unique capacity to drive homologous centromere pairing and synapsis processes because of a specialized engagement with the <a href="http://www.yeastgenome.org/locus/S000004253/overview">Red1</a>/<a href="http://www.yeastgenome.org/locus/S000001334/overview">Hop1</a>/<a href="http://www.yeastgenome.org/locus/S000005878/overview">Mek1</a> pathway, but that this capability is at least partly reliant on additional downstream pathway outcomes that are independent of <a href="http://www.yeastgenome.org/locus/S000000981/overview">Dmc1</a>.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-102">We are extremely grateful to Scott Holmes, Jim Haber, Doug Bishop, Scott Keeney and Nancy Hollingsworth for sharing reagents critical for experiments reported in this study. We are indebted to Karen Voelkel-Meiman for her valuable input on experiments and preparation of the manuscript. This work was supported by National Institutes of Health grants R00 GM084293, and R15 GM104827 (to A.J.M.).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-103">Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.7052072">https://doi.org/10.25387/g3.7052072</a>.</p></li><li class="fn" id="fn-2"><p id="p-104"><em>Communicating editor: D. Bishop</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2018-08-04"><span class="received-label">Received </span>August 4, 2018.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2018-09-16"><span class="accepted-label">Accepted </span>September 16, 2018.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2018 Yisehak, MacQueen</li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.1" data-doi="10.1016/S0092-8674(00)00029-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Agarwal</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roeder</span>  <span class="cit-name-given-names">G. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Zip3 provides a link between recombination enzymes and synaptonemal complex proteins.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">102</span>: <span class="cit-fpage">245</span>–<span class="cit-lpage">255</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0092-8674(00)00029-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DAgarwal%26rft.auinit1%253DS.%26rft.volume%253D102%26rft.issue%253D2%26rft.spage%253D245%26rft.epage%253D255%26rft.atitle%253DZip3%2Bprovides%2Ba%2Blink%2Bbetween%2Brecombination%2Benzymes%2Band%2Bsynaptonemal%2Bcomplex%2Bproteins.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252800%252900029-5%26rft_id%253Dinfo%253Apmid%252F10943844%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0092-8674(00)00029-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10943844&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3637.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000088366800014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ajimura</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leem</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ogawa</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Identification of New Genes Required for Meiotic Recombination in Saccharomyces Cerevisiae.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">133</span>: <span class="cit-fpage">51</span>–<span class="cit-lpage">66</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DAjimura%26rft.auinit1%253DM.%26rft.volume%253D133%26rft.issue%253D1%26rft.spage%253D51%26rft.epage%253D66%26rft.atitle%253DIdentification%2Bof%2Bnew%2Bgenes%2Brequired%2Bfor%2Bmeiotic%2Brecombination%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Apmid%252F8417989%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMTMzLzEvNTEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzExLzM2MzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.3" data-doi="10.1016/0092-8674(90)90524-I"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alani</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Padmore</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kleckner</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Analysis of wild-type and rad50 mutants of yeast suggests an intimate relationship between meiotic chromosome synapsis and recombination.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">61</span>: <span class="cit-fpage">419</span>–<span class="cit-lpage">436</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/0092-8674(90)90524-I</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DAlani%26rft.auinit1%253DE.%26rft.volume%253D61%26rft.issue%253D3%26rft.spage%253D419%26rft.epage%253D436%26rft.atitle%253DAnalysis%2Bof%2Bwild-type%2Band%2Brad50%2Bmutants%2Bof%2Byeast%2Bsuggests%2Ban%2Bintimate%2Brelationship%2Bbetween%2Bmeiotic%2Bchromosome%2Bsynapsis%2Band%2Brecombination.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252890%252990524-I%26rft_id%253Dinfo%253Apmid%252F2185891%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(90)90524-I&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2185891&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3637.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990DC93400006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.4" data-doi="10.1016/S0092-8674(01)00416-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Allers</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lichten</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span><span class="cit-comment">a</span> <span class="cit-article-title">Differential Timing and Control of Noncrossover and Crossover Recombination during Meiosis.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">106</span>: <span class="cit-fpage">47</span>–<span class="cit-lpage">57</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0092-8674(01)00416-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DAllers%26rft.auinit1%253DT.%26rft.volume%253D106%26rft.issue%253D1%26rft.spage%253D47%26rft.epage%253D57%26rft.atitle%253DDifferential%2Btiming%2Band%2Bcontrol%2Bof%2Bnoncrossover%2Band%2Bcrossover%2Brecombination%2Bduring%2Bmeiosis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252801%252900416-0%26rft_id%253Dinfo%253Apmid%252F11461701%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0092-8674(01)00416-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11461701&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3637.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000169870300008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.5" data-doi="10.1016/S1097-2765(01)00280-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Allers</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lichten</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span><span class="cit-comment">b</span> <span class="cit-article-title">Intermediates of Yeast Meiotic Recombination Contain Heteroduplex DNA.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">225</span>–<span class="cit-lpage">231</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S1097-2765(01)00280-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DAllers%26rft.auinit1%253DT.%26rft.volume%253D8%26rft.issue%253D1%26rft.spage%253D225%26rft.epage%253D231%26rft.atitle%253DIntermediates%2Bof%2Byeast%2Bmeiotic%2Brecombination%2Bcontain%2Bheteroduplex%2BDNA.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1097-2765%252801%252900280-5%26rft_id%253Dinfo%253Apmid%252F11511375%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1097-2765(01)00280-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11511375&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3637.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000170081900024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.6" data-doi="10.1016/S1097-2765(02)00543-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bachant</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alcasabas</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blat</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kleckner</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elledge</span>  <span class="cit-name-given-names">S. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">The SUMO-1 Isopeptidase Smt4 Is Linked to Centromeric Cohesion through SUMO-1 Modification of DNA Topoisomerase II.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">1169</span>–<span class="cit-lpage">1182</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S1097-2765(02)00543-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DBachant%26rft.auinit1%253DJ.%26rft.volume%253D9%26rft.issue%253D6%26rft.spage%253D1169%26rft.epage%253D1182%26rft.atitle%253DThe%2BSUMO-1%2Bisopeptidase%2BSmt4%2Bis%2Blinked%2Bto%2Bcentromeric%2Bcohesion%2Bthrough%2BSUMO-1%2Bmodification%2Bof%2BDNA%2Btopoisomerase%2BII.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1097-2765%252802%252900543-9%26rft_id%253Dinfo%253Apmid%252F12086615%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1097-2765(02)00543-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12086615&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3637.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000176390300007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.7" data-doi="10.1046/j.1365-2443.1999.00273.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bishop</span>  <span class="cit-name-given-names">D. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nikolski</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oshiro</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chon</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shinohara</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">High copy number suppression of the meiotic arrest caused by a dmc1 mutation: REC114 imposes an early recombination block and RAD54 promotes a DMC1-independent DSB repair pathway.</span> <abbr class="cit-jnl-abbrev">Genes Cells</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">425</span>–<span class="cit-lpage">444</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1046/j.1365-2443.1999.00273.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2Bto%2BCells%26rft.stitle%253DGENES%2BCELLS%26rft.aulast%253DBishop%26rft.auinit1%253DD.%26rft.volume%253D4%26rft.issue%253D8%26rft.spage%253D425%26rft.epage%253D444%26rft.atitle%253DHigh%2Bcopy%2Bnumber%2Bsuppression%2Bof%2Bthe%2Bmeiotic%2Barrest%2Bcaused%2Bby%2Ba%2Bdmc1%2Bmutation%253A%2BREC114%2Bimposes%2Ban%2Bearly%2Brecombination%2Bblock%2Band%2BRAD54%2Bpromotes%2Ba%2BDMC1-independent%2BDSB%2Brepair%2Bpathway%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-2443.1999.00273.x%26rft_id%253Dinfo%253Apmid%252F10526232%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-2443.1999.00273.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10526232&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3637.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000083840800001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.8" data-doi="10.1016/0092-8674(92)90446-J"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bishop</span>  <span class="cit-name-given-names">D. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kleckner</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">DMC1: a meiosis-specific yeast homolog of E. coli recA required for recombination, synaptonemal complex formation, and cell cycle progression.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">69</span>: <span class="cit-fpage">439</span>–<span class="cit-lpage">456</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/0092-8674(92)90446-J</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DBishop%26rft.auinit1%253DD.%2BK.%26rft.volume%253D69%26rft.issue%253D3%26rft.spage%253D439%26rft.epage%253D456%26rft.atitle%253DDMC1%253A%2Ba%2Bmeiosis-specific%2Byeast%2Bhomolog%2Bof%2BE.%2Bcoli%2BrecA%2Brequired%2Bfor%2Brecombination%252C%2Bsynaptonemal%2Bcomplex%2Bformation%252C%2Band%2Bcell%2Bcycle%2Bprogression.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252892%252990446-J%26rft_id%253Dinfo%253Apmid%252F1581960%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(92)90446-J&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1581960&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3637.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1992HT07800007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.9" data-doi="10.1016/S0092-8674(02)01167-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Blat</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Protacio</span>  <span class="cit-name-given-names">R. U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hunter</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kleckner</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Physical and functional interactions among basic chromosome organizational features govern early steps of meiotic chiasma formation.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">111</span>: <span class="cit-fpage">791</span>–<span class="cit-lpage">802</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0092-8674(02)01167-4</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DBlat%26rft.auinit1%253DY.%26rft.volume%253D111%26rft.issue%253D6%26rft.spage%253D791%26rft.epage%253D802%26rft.atitle%253DPhysical%2Band%2Bfunctional%2Binteractions%2Bamong%2Bbasic%2Bchromosome%2Borganizational%2Bfeatures%2Bgovern%2Bearly%2Bsteps%2Bof%2Bmeiotic%2Bchiasma%2Bformation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252802%252901167-4%26rft_id%253Dinfo%253Apmid%252F12526806%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0092-8674(02)01167-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12526806&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3637.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000179862600006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.10" data-doi="10.1016/j.cub.2007.10.066"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Blitzblau</span>  <span class="cit-name-given-names">H. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bell</span>  <span class="cit-name-given-names">G. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rodriguez</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bell</span>  <span class="cit-name-given-names">S. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hochwagen</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Mapping of meiotic single-stranded DNA reveals double-stranded-break hotspots near centromeres and telomeres.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">2003</span>–<span class="cit-lpage">2012</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.cub.2007.10.066</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DBlitzblau%26rft.auinit1%253DH.%2BG.%26rft.volume%253D17%26rft.issue%253D23%26rft.spage%253D2003%26rft.epage%253D2012%26rft.atitle%253DMapping%2Bof%2Bmeiotic%2Bsingle-stranded%2BDNA%2Breveals%2Bdouble-stranded-break%2Bhotspots%2Bnear%2Bcentromeres%2Band%2Btelomeres.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2007.10.066%26rft_id%253Dinfo%253Apmid%252F18060788%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2007.10.066&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18060788&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3637.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251482100021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.11" data-doi="10.1007/s10577-007-1147-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Borde</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The multiple roles of the Mre11 complex for meiotic recombination.</span> <abbr class="cit-jnl-abbrev">Chromosome Res.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">551</span>–<span class="cit-lpage">563</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s10577-007-1147-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DChromosome%2Bresearch%2B%253A%2B%2Ban%2Binternational%2Bjournal%2Bon%2Bthe%2Bmolecular%252C%2Bsupramolecular%2Band%2Bevolutionary%2Baspects%2Bof%2Bchromosome%2Bbiology%26rft.stitle%253DChromosome%2BRes%26rft.aulast%253DBorde%26rft.auinit1%253DV.%26rft.volume%253D15%26rft.issue%253D5%26rft.spage%253D551%26rft.epage%253D563%26rft.atitle%253DThe%2Bmultiple%2Broles%2Bof%2Bthe%2BMre11%2Bcomplex%2Bfor%2Bmeiotic%2Brecombination.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10577-007-1147-9%26rft_id%253Dinfo%253Apmid%252F17674145%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10577-007-1147-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17674145&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3637.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000249011500004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.12" data-doi="10.1016/S0092-8674(04)00292-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Börner</span>  <span class="cit-name-given-names">G. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kleckner</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hunter</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Crossover/noncrossover differentiation, synaptonemal complex formation, and regulatory surveillance at the leptotene/zygotene transition of meiosis.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">117</span>: <span class="cit-fpage">29</span>–<span class="cit-lpage">45</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0092-8674(04)00292-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DB%25C3%25B6rner%26rft.auinit1%253DG.%2BV.%26rft.volume%253D117%26rft.issue%253D1%26rft.spage%253D29%26rft.epage%253D45%26rft.atitle%253DCrossover%252Fnoncrossover%2Bdifferentiation%252C%2Bsynaptonemal%2Bcomplex%2Bformation%252C%2Band%2Bregulatory%2Bsurveillance%2Bat%2Bthe%2Bleptotene%252Fzygotene%2Btransition%2Bof%2Bmeiosis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252804%252900292-2%26rft_id%253Dinfo%253Apmid%252F15066280%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0092-8674(04)00292-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15066280&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3637.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000221500300006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.13" data-doi="10.1091/mbc.E08-06-0637"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brar</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hochwagen</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ee</span>  <span class="cit-name-given-names">L. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amon</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The multiple roles of cohesin in meiotic chromosome morphogenesis and pairing.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">1030</span>–<span class="cit-lpage">1047</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1091/mbc.e08-06-0637</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.E08-06-0637%26rft_id%253Dinfo%253Apmid%252F19073884%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjIwLzMvMTAzMCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzgvMTEvMzYzNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.14" data-doi="10.1128/MCB.19.11.7681"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bressan</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baxter</span>  <span class="cit-name-given-names">B. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petrini</span>  <span class="cit-name-given-names">J. H. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">The Mre11-Rad50-Xrs2 Protein Complex Facilitates Homologous Recombination-Based Double-Strand Break Repair in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">7681</span>–<span class="cit-lpage">7687</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/MCB.19.11.7681</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DBressan%26rft.auinit1%253DD.%2BA.%26rft.volume%253D19%26rft.issue%253D11%26rft.spage%253D7681%26rft.epage%253D7687%26rft.atitle%253DThe%2BMre11-Rad50-Xrs2%2BProtein%2BComplex%2BFacilitates%2BHomologous%2BRecombination-Based%2BDouble-Strand%2BBreak%2BRepair%2BinSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.19.11.7681%26rft_id%253Dinfo%253Apmid%252F10523656%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIxOS8xMS83NjgxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMS8zNjM3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.15" data-doi="10.1101/cshperspect.a016659"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bishop</span>  <span class="cit-name-given-names">D. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">DNA strand exchange and RecA homologs in meiosis.</span> <abbr class="cit-jnl-abbrev">Cold Spring Harb. Perspect. Biol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">a016659</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/cshperspect.a016659</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCold%2BSpring%2BHarb.%2BPerspect.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fcshperspect.a016659%26rft_id%253Dinfo%253Apmid%252F25475089%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6ImNzaHBlcnNwZWN0IjtzOjU6InJlc2lkIjtzOjExOiI3LzEvYTAxNjY1OSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzgvMTEvMzYzNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.16" data-doi="10.1101/gad.1638708"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Busygina</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sehorn</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">I. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsubouchi</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roeder</span>  <span class="cit-name-given-names">G. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Hed1 regulates Rad51-mediated recombination via a novel mechanism.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">786</span>–<span class="cit-lpage">795</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gad.1638708</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DBusygina%26rft.auinit1%253DV.%26rft.volume%253D22%26rft.issue%253D6%26rft.spage%253D786%26rft.epage%253D795%26rft.atitle%253DHed1%2Bregulates%2BRad51-mediated%2Brecombination%2Bvia%2Ba%2Bnovel%2Bmechanism%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.1638708%26rft_id%253Dinfo%253Apmid%252F18347097%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6ODoiMjIvNi83ODYiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzExLzM2MzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.17" data-doi="10.1038/nature08868"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bzymek</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thayer</span>  <span class="cit-name-given-names">N. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oh</span>  <span class="cit-name-given-names">S. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kleckner</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hunter</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Double Holliday Junctions are Intermediates of DNA Break Repair.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">464</span>: <span class="cit-fpage">937</span>–<span class="cit-lpage">941</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature08868</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DBzymek%26rft.auinit1%253DM.%26rft.volume%253D464%26rft.issue%253D7290%26rft.spage%253D937%26rft.epage%253D941%26rft.atitle%253DDouble%2BHolliday%2Bjunctions%2Bare%2Bintermediates%2Bof%2BDNA%2Bbreak%2Brepair.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08868%26rft_id%253Dinfo%253Apmid%252F20348905%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature08868&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20348905&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3637.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276397300047&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.18" data-doi="10.1038/nsmb.3208"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cahoon</span>  <span class="cit-name-given-names">C. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hawley</span>  <span class="cit-name-given-names">R. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Regulating the construction and demolition of the synaptonemal complex.</span> <abbr class="cit-jnl-abbrev">Nat. Struct. Mol. Biol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">369</span>–<span class="cit-lpage">377</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nsmb.3208</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BStruct.%2BMol.%2BBiol.%26rft.volume%253D23%26rft.spage%253D369%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnsmb.3208%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nsmb.3208&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.19" data-doi="10.1371/journal.pgen.1006226"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Callender</span>  <span class="cit-name-given-names">T. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laureau</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wan</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sandhu</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Mek1 Down Regulates Rad51 Activity during Yeast Meiosis by Phosphorylation of Hed1.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">e1006226</span> <span class="cit-comment">(correction: PLOS Genetics 12: e1006283)</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1006226</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D12%26rft.spage%253De1006226%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1006226%26rft_id%253Dinfo%253Apmid%252F27483004%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1006226&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27483004&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3637.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.20" data-doi="10.1016/j.cell.2008.01.035"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carballo</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sedgwick</span>  <span class="cit-name-given-names">S. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cha</span>  <span class="cit-name-given-names">R. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Phosphorylation of the axial element protein Hop1 by Mec1/Tel1 ensures meiotic interhomolog recombination.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">132</span>: <span class="cit-fpage">758</span>–<span class="cit-lpage">770</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.cell.2008.01.035</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DCarballo%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D132%26rft.issue%253D5%26rft.spage%253D758%26rft.epage%253D770%26rft.atitle%253DPhosphorylation%2Bof%2Bthe%2Baxial%2Belement%2Bprotein%2BHop1%2Bby%2BMec1%252FTel1%2Bensures%2Bmeiotic%2Binterhomolog%2Brecombination.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2008.01.035%26rft_id%253Dinfo%253Apmid%252F18329363%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2008.01.035&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18329363&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3637.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000253818000014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.21" data-doi="10.1128/MCB.00375-08"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cartagena-Lirola</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guerini</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manfrini</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lucchini</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Longhese</span>  <span class="cit-name-given-names">M. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Role of the Saccharomyces cerevisiae Rad53 checkpoint kinase in signaling double-strand breaks during the meiotic cell cycle.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">4480</span>–<span class="cit-lpage">4493</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/MCB.00375-08</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.00375-08%26rft_id%253Dinfo%253Apmid%252F18505828%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIyOC8xNC80NDgwIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMS8zNjM3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.22" data-doi="10.4161/cc.5.14.2916"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cartagena-Lirola</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guerini</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Viscardi</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lucchini</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Longhese</span>  <span class="cit-name-given-names">M. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Budding Yeast Sae2 is an In Vivo Target of the Mec1 and Tel1 Checkpoint Kinases During Meiosis.</span> <abbr class="cit-jnl-abbrev">Cell Cycle</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">1549</span>–<span class="cit-lpage">1559</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.4161/cc.5.14.2916</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2Bcycle%2B%2528Georgetown%252C%2BTex.%2529%26rft.stitle%253DCell%2BCycle%26rft.aulast%253DCartagena-Lirola%26rft.auinit1%253DH.%26rft.volume%253D5%26rft.issue%253D14%26rft.spage%253D1549%26rft.epage%253D1559%26rft.atitle%253DBudding%2BYeast%2BSae2%2Bis%2Ban%2BIn%2BVivo%2BTarget%2Bof%2Bthe%2BMec1%2Band%2BTel1%2BCheckpoint%2BKinases%2BDuring%2BMeiosis.%26rft_id%253Dinfo%253Adoi%252F10.4161%252Fcc.5.14.2916%26rft_id%253Dinfo%253Apmid%252F16861895%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4161/cc.5.14.2916&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16861895&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3637.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000240697800015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.23" data-doi="10.1016/j.devcel.2008.07.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">S. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsubouchi</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rockmill</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sandler</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Richards</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Global Analysis of the Meiotic Crossover Landscape.</span> <abbr class="cit-jnl-abbrev">Dev. Cell</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">401</span>–<span class="cit-lpage">415</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.devcel.2008.07.006</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bcell%26rft.stitle%253DDev%2BCell%26rft.aulast%253DChen%26rft.auinit1%253DS.%2BY.%26rft.volume%253D15%26rft.issue%253D3%26rft.spage%253D401%26rft.epage%253D415%26rft.atitle%253DGlobal%2Banalysis%2Bof%2Bthe%2Bmeiotic%2Bcrossover%2Blandscape.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.devcel.2008.07.006%26rft_id%253Dinfo%253Apmid%252F18691940%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.devcel.2008.07.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18691940&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3637.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000259358100011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.24" data-doi="10.1016/S0092-8674(00)81164-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chua</span>  <span class="cit-name-given-names">P. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roeder</span>  <span class="cit-name-given-names">G. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Zip2, a Meiosis-Specific Protein Required for the Initiation of Chromosome Synapsis.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">349</span>–<span class="cit-lpage">359</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0092-8674(00)81164-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DChua%26rft.auinit1%253DP.%2BR.%26rft.volume%253D93%26rft.issue%253D3%26rft.spage%253D349%26rft.epage%253D359%26rft.atitle%253DZip2%252C%2Ba%2Bmeiosis-specific%2Bprotein%2Brequired%2Bfor%2Bthe%2Binitiation%2Bof%2Bchromosome%2Bsynapsis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252800%252981164-2%26rft_id%253Dinfo%253Apmid%252F9590170%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0092-8674(00)81164-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9590170&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3637.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000073471500011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.25" data-doi="10.1126/science.1219379"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cloud</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chan</span>  <span class="cit-name-given-names">Y.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grubb</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Budke</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bishop</span>  <span class="cit-name-given-names">D. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Rad51 Is an Accessory Factor for Dmc1-Mediated Joint Molecule Formation During Meiosis.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">337</span>: <span class="cit-fpage">1222</span>–<span class="cit-lpage">1225</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1219379</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DCloud%26rft.auinit1%253DV.%26rft.volume%253D337%26rft.issue%253D6099%26rft.spage%253D1222%26rft.epage%253D1225%26rft.atitle%253DRad51%2BIs%2Ban%2BAccessory%2BFactor%2Bfor%2BDmc1-Mediated%2BJoint%2BMolecule%2BFormation%2BDuring%2BMeiosis%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1219379%26rft_id%253Dinfo%253Apmid%252F22955832%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMzcvNjA5OS8xMjIyIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMS8zNjM3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Santos</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hunter</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Larkin</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Loidl</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">The Mus81/Mms4 Endonuclease Acts Independently of Double-Holliday Junction Resolution to Promote a Distinct Subset of Crossovers During Meiosis in Budding Yeast.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">164</span>: <span class="cit-fpage">81</span>–<span class="cit-lpage">94</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253Dde%2Blos%2BSantos%26rft.auinit1%253DT.%26rft.volume%253D164%26rft.issue%253D1%26rft.spage%253D81%26rft.epage%253D94%26rft.atitle%253DThe%2BMus81%252FMms4%2BEndonuclease%2BActs%2BIndependently%2Bof%2BDouble-Holliday%2BJunction%2BResolution%2Bto%2BPromote%2Ba%2BDistinct%2BSubset%2Bof%2BCrossovers%2BDuring%2BMeiosis%2Bin%2BBudding%2BYeast%26rft_id%253Dinfo%253Apmid%252F12750322%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMTY0LzEvODEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzExLzM2MzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.27" data-doi="10.1016/j.molcel.2012.02.020"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">De Muyt</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jessop</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kolar</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sourirajan</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">BLM helicase ortholog Sgs1 is a central regulator of meiotic recombination intermediate metabolism.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">43</span>–<span class="cit-lpage">53</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.molcel.2012.02.020</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DDe%2BMuyt%26rft.auinit1%253DA.%26rft.volume%253D46%26rft.issue%253D1%26rft.spage%253D43%26rft.epage%253D53%26rft.atitle%253DBLM%2Bhelicase%2Bortholog%2BSgs1%2Bis%2Ba%2Bcentral%2Bregulator%2Bof%2Bmeiotic%2Brecombination%2Bintermediate%2Bmetabolism.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2012.02.020%26rft_id%253Dinfo%253Apmid%252F22500736%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2012.02.020&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22500736&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3637.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000302838000007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.28" data-doi="10.1083/jcb.148.3.417"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dong</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roeder</span>  <span class="cit-name-given-names">G. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Organization of the Yeast Zip1 Protein within the Central Region of the Synaptonemal Complex.</span> <abbr class="cit-jnl-abbrev">J. Cell Biol.</abbr> <span class="cit-vol">148</span>: <span class="cit-fpage">417</span>–<span class="cit-lpage">426</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1083/jcb.148.3.417</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BCell%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1083%252Fjcb.148.3.417%26rft_id%253Dinfo%253Apmid%252F10662769%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNiIjtzOjU6InJlc2lkIjtzOjk6IjE0OC8zLzQxNyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzgvMTEvMzYzNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.29" data-doi="10.1016/j.devcel.2010.09.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Falk</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chan</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoffmann</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hochwagen</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A Mec1- and PP4-dependent checkpoint couples centromere pairing to meiotic recombination.</span> <abbr class="cit-jnl-abbrev">Dev. Cell</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">599</span>–<span class="cit-lpage">611</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.devcel.2010.09.006</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bcell%26rft.stitle%253DDev%2BCell%26rft.aulast%253DFalk%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D19%26rft.issue%253D4%26rft.spage%253D599%26rft.epage%253D611%26rft.atitle%253DA%2BMec1-%2Band%2BPP4-dependent%2Bcheckpoint%2Bcouples%2Bcentromere%2Bpairing%2Bto%2Bmeiotic%2Brecombination.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.devcel.2010.09.006%26rft_id%253Dinfo%253Apmid%252F20951350%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.devcel.2010.09.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20951350&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3637.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000283522700012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.30" data-doi="10.1534/genetics.105.046821"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Farah</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cromie</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steiner</span>  <span class="cit-name-given-names">W. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">G. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Activation of an Alternative, Rec12 (Spo11)-Independent Pathway of Fission Yeast Meiotic Recombination in the Absence of a DNA Flap Endonuclease.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">171</span>: <span class="cit-fpage">1499</span>–<span class="cit-lpage">1511</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.105.046821</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.046821%26rft_id%253Dinfo%253Apmid%252F16118186%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3MS80LzE0OTkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzExLzM2MzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.31" data-doi="10.1101/gad.1105203"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fricke</span>  <span class="cit-name-given-names">W. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brill</span>  <span class="cit-name-given-names">S. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Slx1-Slx4 is a second structure-specific endonuclease functionally redundant with Sgs1-Top3.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">1768</span>–<span class="cit-lpage">1778</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gad.1105203</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2BDev.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.1105203%26rft_id%253Dinfo%253Apmid%252F12832395%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjE3LzE0LzE3NjgiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzExLzM2MzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.32" data-doi="10.1016/0027-5107(74)90176-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Game</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mortimer</span>  <span class="cit-name-given-names">R. K.</span></span></li></ol><cite>, <span class="cit-pub-date">1974</span> <span class="cit-article-title">A genetic study of X-ray sensitive mutants in yeast.</span> <abbr class="cit-jnl-abbrev">Mutat. Res. Fundam. Mol. Mech. Mutagen.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">281</span>–<span class="cit-lpage">292</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/0027-5107(74)90176-6</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMutat.%2BRes.%2BFundam.%2BMol.%2BMech.%2BMutagen.%26rft.volume%253D24%26rft.spage%253D281%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0027-5107%252874%252990176-6%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0027-5107(74)90176-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Game</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zamb</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Braun</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Resnick</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roth</span>  <span class="cit-name-given-names">R. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1980</span> <span class="cit-article-title">The Role of Radiation (rad) Genes in Meiotic Recombination in Yeast.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">94</span>: <span class="cit-fpage">51</span>–<span class="cit-lpage">68</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DGame%26rft.auinit1%253DJ.%2BC.%26rft.volume%253D94%26rft.issue%253D1%26rft.spage%253D51%26rft.epage%253D68%26rft.atitle%253DTHE%2BROLE%2BOF%2BRADIATION%2B%2528rad%2529%2BGENES%2BIN%2BMEIOTIC%2BRECOMBINATION%2BIN%2BYEAST%26rft_id%253Dinfo%253Apmid%252F17248996%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6NzoiOTQvMS81MSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzgvMTEvMzYzNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.34" data-doi="10.1073/pnas.97.21.11383"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gerton</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">DeRisi</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shroff</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lichten</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">P. O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Global mapping of meiotic recombination hotspots and coldspots in the yeast <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">97</span>: <span class="cit-fpage">11383</span>–<span class="cit-lpage">11390</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.97.21.11383</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DGerton%26rft.auinit1%253DJ.%2BL.%26rft.volume%253D97%26rft.issue%253D21%26rft.spage%253D11383%26rft.epage%253D11390%26rft.atitle%253DGlobal%2Bmapping%2Bof%2Bmeiotic%2Brecombination%2Bhotspots%2Band%2Bcoldspots%2Bin%2Bthe%2Byeast%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.97.21.11383%26rft_id%253Dinfo%253Apmid%252F11027339%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiOTcvMjEvMTEzODMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzExLzM2MzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.35" data-doi="10.15698/mic2016.08.517"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gobbini</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cassani</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Villa</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bonetti</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Longhese</span>  <span class="cit-name-given-names">M. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Functions and regulation of the MRX complex at DNA double-strand breaks.</span> <abbr class="cit-jnl-abbrev">Microb. Cell</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">329</span>–<span class="cit-lpage">337</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.15698/mic2016.08.517</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrob.%2BCell%26rft.volume%253D3%26rft.spage%253D329%26rft_id%253Dinfo%253Adoi%252F10.15698%252Fmic2016.08.517%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.15698/mic2016.08.517&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.36" data-doi="10.1002/(SICI)1097-0061(199910)15:14&lt;1541::AID-YEA476&gt;3.0.CO;2-K"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goldstein</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCusker</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Three new dominant drug resistance cassettes for gene disruption in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Yeast</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1541</span>–<span class="cit-lpage">1553</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/(SICI)1097-0061(199910)15:14&lt;1541::AID-YEA476&gt;3.0.CO;2-K</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%2B%2528Chichester%252C%2BEngland%2529%26rft.stitle%253DYeast%26rft.aulast%253DGoldstein%26rft.auinit1%253DA.%2BL.%26rft.volume%253D15%26rft.issue%253D14%26rft.spage%253D1541%26rft.epage%253D1553%26rft.atitle%253DThree%2Bnew%2Bdominant%2Bdrug%2Bresistance%2Bcassettes%2Bfor%2Bgene%2Bdisruption%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1002%252F%2528SICI%25291097-0061%2528199910%252915%253A14%253C1541%253A%253AAID-YEA476%253E3.0.CO%253B2-K%26rft_id%253Dinfo%253Apmid%252F10514571%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/(SICI)1097-0061(199910)15:14&lt;1541::AID-YEA476&gt;3.0.CO;2-K&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10514571&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3637.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000083104100010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.37" data-doi="10.1534/genetics.111.134577"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haber</span>  <span class="cit-name-given-names">J. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Mating-type genes and MAT switching in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">191</span>: <span class="cit-fpage">33</span>–<span class="cit-lpage">64</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.111.134577</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHaber%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D191%26rft.issue%253D1%26rft.spage%253D33%26rft.epage%253D64%26rft.atitle%253DMating-Type%2BGenes%2Band%2BMAT%2BSwitching%2Bin%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.134577%26rft_id%253Dinfo%253Apmid%252F22555442%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMTkxLzEvMzMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzExLzM2MzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.38" data-doi="10.1073/pnas.0400843101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Henderson</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keeney</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kleckner</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Tying synaptonemal complex initiation to the formation and programmed repair of DNA double-strand breaks.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">4519</span>–<span class="cit-lpage">4524</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.0400843101</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0400843101%26rft_id%253Dinfo%253Apmid%252F15070750%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTAxLzEzLzQ1MTkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzExLzM2MzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.39" data-doi="10.15698/mic2016.03.487"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hollingsworth</span>  <span class="cit-name-given-names">N. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Mek1/Mre4 is a master regulator of meiotic recombination in budding yeast.</span> <abbr class="cit-jnl-abbrev">Microb. Cell</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">129</span>–<span class="cit-lpage">131</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.15698/mic2016.03.487</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrob.%2BCell%26rft.volume%253D3%26rft.spage%253D129%26rft_id%253Dinfo%253Adoi%252F10.15698%252Fmic2016.03.487%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.15698/mic2016.03.487&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.40"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hollingsworth</span>  <span class="cit-name-given-names">N. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ponte</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Genetic interactions between Hop1, Red1 and Mek1 suggest that Mek1 regulates assembly of axial element components during meiosis in the yeast <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">147</span>: <span class="cit-fpage">33</span>–<span class="cit-lpage">42</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHollingsworth%26rft.auinit1%253DN.%2BM.%26rft.volume%253D147%26rft.issue%253D1%26rft.spage%253D33%26rft.epage%253D42%26rft.atitle%253DGenetic%2BInteractions%2BBetween%2BHOP1%252C%2BRED1%2Band%2BMEK1%2BSuggest%2BThat%2BMEK1%2BRegulates%2BAssembly%2Bof%2BAxial%2BElement%2BComponents%2BDuring%2BMeiosis%2Bin%2Bthe%2BYeast%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Apmid%252F9286666%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMTQ3LzEvMzMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzExLzM2MzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.41" data-doi="10.1101/gad.9.14.1728"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hollingsworth</span>  <span class="cit-name-given-names">N. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ponte</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Halsey</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">MSH5, a novel MutS homolog, facilitates meiotic reciprocal recombination between homologs in Saccharomyces cerevisiae but not mismatch repair.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">1728</span>–<span class="cit-lpage">1739</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gad.9.14.1728</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DHollingsworth%26rft.auinit1%253DN%2BM%26rft.volume%253D9%26rft.issue%253D14%26rft.spage%253D1728%26rft.epage%253D1739%26rft.atitle%253DMSH5%252C%2Ba%2Bnovel%2BMutS%2Bhomolog%252C%2Bfacilitates%2Bmeiotic%2Breciprocal%2Brecombination%2Bbetween%2Bhomologs%2Bin%2BSaccharomyces%2Bcerevisiae%2Bbut%2Bnot%2Bmismatch%2Brepair.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.9.14.1728%26rft_id%253Dinfo%253Apmid%252F7622037%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6OToiOS8xNC8xNzI4IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMS8zNjM3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.42" data-doi="10.1016/j.molcel.2013.08.008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hong</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sung</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kleckner</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The logic and mechanism of homologous recombination partner choice.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">51</span>: <span class="cit-fpage">440</span>–<span class="cit-lpage">453</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.molcel.2013.08.008</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell%26rft.volume%253D51%26rft.spage%253D440%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2013.08.008%26rft_id%253Dinfo%253Apmid%252F23973374%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2013.08.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23973374&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3637.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000323693900005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.43" data-doi="10.1371/journal.pgen.1003194"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Humphryes</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leung</span>  <span class="cit-name-given-names">W.-K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Argunhan</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Terentyev</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dvorackova</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The Ecm11-Gmc2 Complex Promotes Synaptonemal Complex Formation through Assembly of Transverse Filaments in Budding Yeast.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1003194</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1003194</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003194%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003194%26rft_id%253Dinfo%253Apmid%252F23326245%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003194&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23326245&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3637.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.44" data-doi="10.1101/cshperspect.a016618"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hunter</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Meiotic Recombination: The Essence of Heredity.</span> <abbr class="cit-jnl-abbrev">Cold Spring Harb. Perspect. Biol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">a016618</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/cshperspect.a016618</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCold%2BSpring%2BHarb.%2BPerspect.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fcshperspect.a016618%26rft_id%253Dinfo%253Apmid%252F26511629%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6ImNzaHBlcnNwZWN0IjtzOjU6InJlc2lkIjtzOjEyOiI3LzEyL2EwMTY2MTgiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzExLzM2MzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.45" data-doi="10.1101/gad.11.12.1573"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hunter</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borts</span>  <span class="cit-name-given-names">R. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Mlh1 is unique among mismatch repair proteins in its ability to promote crossing-over during meiosis.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">1573</span>–<span class="cit-lpage">1582</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gad.11.12.1573</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DHunter%26rft.auinit1%253DN%26rft.volume%253D11%26rft.issue%253D12%26rft.spage%253D1573%26rft.epage%253D1582%26rft.atitle%253DMlh1%2Bis%2Bunique%2Bamong%2Bmismatch%2Brepair%2Bproteins%2Bin%2Bits%2Bability%2Bto%2Bpromote%2Bcrossing-over%2Bduring%2Bmeiosis.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.11.12.1573%26rft_id%253Dinfo%253Apmid%252F9203583%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjExLzEyLzE1NzMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzExLzM2MzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.46" data-doi="10.1016/S0092-8674(01)00430-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hunter</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kleckner</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">The Single-End Invasion: An Asymmetric Intermediate at the Double-Strand Break to Double-Holliday Junction Transition of Meiotic Recombination.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">106</span>: <span class="cit-fpage">59</span>–<span class="cit-lpage">70</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0092-8674(01)00430-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DHunter%26rft.auinit1%253DN.%26rft.volume%253D106%26rft.issue%253D1%26rft.spage%253D59%26rft.epage%253D70%26rft.atitle%253DThe%2Bsingle-end%2Binvasion%253A%2Ban%2Basymmetric%2Bintermediate%2Bat%2Bthe%2Bdouble-strand%2Bbreak%2Bto%2Bdouble-holliday%2Bjunction%2Btransition%2Bof%2Bmeiotic%2Brecombination.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252801%252900430-5%26rft_id%253Dinfo%253Apmid%252F11461702%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0092-8674(01)00430-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11461702&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3637.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000169870300009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.47" data-doi="10.1038/nature07470"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ip</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rass</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blanco</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flynn</span>  <span class="cit-name-given-names">H. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Skehel</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Identification of Holliday junction resolvases from humans and yeast.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">456</span>: <span class="cit-fpage">357</span>–<span class="cit-lpage">361</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature07470</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DIp%26rft.auinit1%253DS.%2BC.%26rft.volume%253D456%26rft.issue%253D7220%26rft.spage%253D357%26rft.epage%253D361%26rft.atitle%253DIdentification%2Bof%2BHolliday%2Bjunction%2Bresolvases%2Bfrom%2Bhumans%2Band%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07470%26rft_id%253Dinfo%253Apmid%252F19020614%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature07470&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19020614&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3637.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261039600036&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.48" data-doi="10.1101/SQB.1984.049.01.013"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jensen</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Herskowitz</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title">Directionality and regulation of cassette substitution in yeast.</span> <abbr class="cit-jnl-abbrev">Cold Spring Harb. Symp. Quant. Biol.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">97</span>–<span class="cit-lpage">104</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/SQB.1984.049.01.013</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCold%2BSpring%2BHarbor%2BSymposia%2Bon%2BQuantitative%2BBiology%26rft.stitle%253DCold%2BSpring%2BHarb%2BSymp%2BQuant%2BBiol%26rft.aulast%253DJensen%26rft.auinit1%253DR.E.%26rft.volume%253D49%26rft.issue%253D0%26rft.spage%253D97%26rft.epage%253D104%26rft.atitle%253DDirectionality%2Band%2BRegulation%2Bof%2BCassette%2BSubstitution%2Bin%2BYeast%26rft_id%253Dinfo%253Adoi%252F10.1101%252FSQB.1984.049.01.013%26rft_id%253Dinfo%253Apmid%252F6397325%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic3FiIjtzOjU6InJlc2lkIjtzOjc6IjQ5LzAvOTciO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzExLzM2MzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.49" data-doi="10.1016/j.molcel.2008.05.021"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jessop</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lichten</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Mus81/Mms4 endonuclease and Sgs1 helicase collaborate to ensure proper recombination intermediate metabolism during meiosis.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">313</span>–<span class="cit-lpage">323</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.molcel.2008.05.021</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DJessop%26rft.auinit1%253DL.%26rft.volume%253D31%26rft.issue%253D3%26rft.spage%253D313%26rft.epage%253D323%26rft.atitle%253DMus81%252FMms4%2Bendonuclease%2Band%2BSgs1%2Bhelicase%2Bcollaborate%2Bto%2Bensure%2Bproper%2Brecombination%2Bintermediate%2Bmetabolism%2Bduring%2Bmeiosis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2008.05.021%26rft_id%253Dinfo%253Apmid%252F18691964%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2008.05.021&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18691964&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3637.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000258556500005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.50" data-doi="10.1371/journal.pgen.0020155"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jessop</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rockmill</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roeder</span>  <span class="cit-name-given-names">G. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lichten</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Meiotic Chromosome Synapsis-Promoting Proteins Antagonize the Anti-Crossover Activity of Sgs1.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">e155</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.0020155</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DJessop%26rft.auinit1%253DL.%26rft.volume%253D2%26rft.issue%253D9%26rft.spage%253De155%26rft.epage%253De155%26rft.atitle%253DMeiotic%2Bchromosome%2Bsynapsis-promoting%2Bproteins%2Bantagonize%2Bthe%2Banti-crossover%2Bactivity%2Bof%2Bsgs1.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.0020155%26rft_id%253Dinfo%253Apmid%252F17002499%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.0020155&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17002499&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3637.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.51"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Johzuka</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ogawa</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Interaction of Mre11 and Rad50: two proteins required for DNA repair and meiosis-specific double-strand break formation in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">139</span>: <span class="cit-fpage">1521</span>–<span class="cit-lpage">1532</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DJohzuka%26rft.auinit1%253DK.%26rft.volume%253D139%26rft.issue%253D4%26rft.spage%253D1521%26rft.epage%253D1532%26rft.atitle%253DInteraction%2Bof%2BMre11%2Band%2BRad50%253A%2Btwo%2Bproteins%2Brequired%2Bfor%2BDNA%2Brepair%2Band%2Bmeiosis-specific%2Bdouble-strand%2Bbreak%2Bformation%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Apmid%252F7789757%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjEzOS80LzE1MjEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzExLzM2MzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.52"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kadyk</span>  <span class="cit-name-given-names">L. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hartwell</span>  <span class="cit-name-given-names">L. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Sister Chromatids Are Preferred over Homologs as Substrates for Recombinational Repair in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">132</span>: <span class="cit-fpage">387</span>–<span class="cit-lpage">402</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKadyk%26rft.auinit1%253DL.%2BC.%26rft.volume%253D132%26rft.issue%253D2%26rft.spage%253D387%26rft.epage%253D402%26rft.atitle%253DSister%2Bchromatids%2Bare%2Bpreferred%2Bover%2Bhomologs%2Bas%2Bsubstrates%2Bfor%2Brecombinational%2Brepair%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Apmid%252F1427035%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTMyLzIvMzg3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMS8zNjM3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.53" data-doi="10.1038/nature02424"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kellis</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Birren</span>  <span class="cit-name-given-names">B. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lander</span>  <span class="cit-name-given-names">E. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Proof and evolutionary analysis of ancient genome duplication in the yeast Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">428</span>: <span class="cit-fpage">617</span>–<span class="cit-lpage">624</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature02424</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DKellis%26rft.auinit1%253DM.%26rft.volume%253D428%26rft.issue%253D6983%26rft.spage%253D617%26rft.epage%253D624%26rft.atitle%253DProof%2Band%2Bevolutionary%2Banalysis%2Bof%2Bancient%2Bgenome%2Bduplication%2Bin%2Bthe%2Byeast%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature02424%26rft_id%253Dinfo%253Apmid%252F15004568%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature02424&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15004568&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3637.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000220697200034&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.54" data-doi="10.1101/gad.1227304"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kemp</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boumil</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stewart</span>  <span class="cit-name-given-names">M. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dawson</span>  <span class="cit-name-given-names">D. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">A role for centromere pairing in meiotic chromosome segregation.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">1946</span>–<span class="cit-lpage">1951</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gad.1227304</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2BDev.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.1227304%26rft_id%253Dinfo%253Apmid%252F15289462%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjE4LzE2LzE5NDYiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzExLzM2MzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.55" data-doi="10.1016/j.cell.2010.11.015"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">K. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weiner</span>  <span class="cit-name-given-names">B. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jordan</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dekker</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Sister cohesion and structural axis components mediate homolog bias of meiotic recombination.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">143</span>: <span class="cit-fpage">924</span>–<span class="cit-lpage">937</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.cell.2010.11.015</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DKim%26rft.auinit1%253DK.%2BP.%26rft.volume%253D143%26rft.issue%253D6%26rft.spage%253D924%26rft.epage%253D937%26rft.atitle%253DSister%2Bcohesion%2Band%2Bstructural%2Baxis%2Bcomponents%2Bmediate%2Bhomolog%2Bbias%2Bof%2Bmeiotic%2Brecombination.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2010.11.015%26rft_id%253Dinfo%253Apmid%252F21145459%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2010.11.015&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21145459&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3637.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000285182600010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.56"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Klapholz</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Esposito</span>  <span class="cit-name-given-names">R. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1980</span> Recombination and Chromosome Segregation During the Single Division Meiosis in SPO12–1 and SPO13–1 Diploids. Genetics 96: 589.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.57" data-doi="10.1016/S0092-8674(00)80609-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Klein</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mahr</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Galova</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buonomo</span>  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Michaelis</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">A central role for cohesins in sister chromatid cohesion, formation of axial elements, and recombination during yeast meiosis.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">98</span>: <span class="cit-fpage">91</span>–<span class="cit-lpage">103</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0092-8674(00)80609-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DKlein%26rft.auinit1%253DF.%26rft.volume%253D98%26rft.issue%253D1%26rft.spage%253D91%26rft.epage%253D103%26rft.atitle%253DA%2Bcentral%2Brole%2Bfor%2Bcohesins%2Bin%2Bsister%2Bchromatid%2Bcohesion%252C%2Bformation%2Bof%2Baxial%2Belements%252C%2Band%2Brecombination%2Bduring%2Byeast%2Bmeiosis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252800%252980609-1%26rft_id%253Dinfo%253Apmid%252F10412984%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0092-8674(00)80609-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10412984&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3637.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000081451000011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.58" data-doi="10.1073/pnas.151008198"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kraus</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leung</span>  <span class="cit-name-given-names">W. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haber</span>  <span class="cit-name-given-names">J. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Break-induced replication: a review and an example in budding yeast.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">98</span>: <span class="cit-fpage">8255</span>–<span class="cit-lpage">8262</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.151008198</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DKraus%26rft.auinit1%253DE.%26rft.volume%253D98%26rft.issue%253D15%26rft.spage%253D8255%26rft.epage%253D8262%26rft.atitle%253DBreak-induced%2Breplication%253A%2BA%2Breview%2Band%2Ban%2Bexample%2Bin%2Bbudding%2Byeast%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.151008198%26rft_id%253Dinfo%253Apmid%252F11459961%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiOTgvMTUvODI1NSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzgvMTEvMzYzNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.59" data-doi="10.1534/genetics.116.190264"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kurdzo</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Obeso</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chuong</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dawson</span>  <span class="cit-name-given-names">D. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Meiotic Centromere Coupling and Pairing Function by Two Separate Mechanisms in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">205</span>: <span class="cit-fpage">657</span>–<span class="cit-lpage">671</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.116.190264</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.116.190264%26rft_id%253Dinfo%253Apmid%252F27913618%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjA1LzIvNjU3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMS8zNjM3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.60" data-doi="10.1038/227680a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Laemmli</span>  <span class="cit-name-given-names">U. K.</span></span></li></ol><cite>, <span class="cit-pub-date">1970</span> <span class="cit-article-title">Cleavage of structural proteins during the assembly of the head of bacteriophage T4.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">227</span>: <span class="cit-fpage">680</span>–<span class="cit-lpage">685</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/227680a0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLaemmli%26rft.auinit1%253DU.%2BK.%26rft.volume%253D227%26rft.issue%253D5259%26rft.spage%253D680%26rft.epage%253D685%26rft.atitle%253DCleavage%2Bof%2Bstructural%2Bproteins%2Bduring%2Bthe%2Bassembly%2Bof%2Bthe%2Bhead%2Bof%2Bbacteriophage%2BT4.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F227680a0%26rft_id%253Dinfo%253Apmid%252F5432063%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/227680a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=5432063&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3637.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1970H022600020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.61" data-doi="10.1101/cshperspect.a016634"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lam</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keeney</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Mechanism and regulation of meiotic recombination initiation.</span> <abbr class="cit-jnl-abbrev">Cold Spring Harb. Perspect. Biol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">a016634</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/cshperspect.a016634</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCold%2BSpring%2BHarb.%2BPerspect.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fcshperspect.a016634%26rft_id%253Dinfo%253Apmid%252F25324213%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6ImNzaHBlcnNwZWN0IjtzOjU6InJlc2lkIjtzOjExOiI3LzEvYTAxNjYzNCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzgvMTEvMzYzNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.62"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lambie</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roeder</span>  <span class="cit-name-given-names">G. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span> <span class="cit-article-title">Repression of Meiotic Crossing over by a Centromere (<em>CEN3</em>) in <em>Saccharomyces cerevisiae</em></span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">114</span>: <span class="cit-fpage">769</span>–<span class="cit-lpage">789</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLambie%26rft.auinit1%253DE.%2BJ.%26rft.volume%253D114%26rft.issue%253D3%26rft.spage%253D769%26rft.epage%253D789%26rft.atitle%253DREPRESSION%2BOF%2BMEIOTIC%2BCROSSING%2BOVER%2BBY%2BA%2BCENTROMERE%2B%2528CEN3%2B%2529%2BIN%2BSACCHAROMYCES%2BCEREVISIAE%26rft_id%253Dinfo%253Apmid%252F3539697%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTE0LzMvNzY5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMS8zNjM3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.63" data-doi="10.1016/0092-8674(88)90428-X"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lambie</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roeder</span>  <span class="cit-name-given-names">G. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">A yeast acts in (<em>Cis</em>) to inhibit meiotic gene conversion of adjacent sequences.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">863</span>–<span class="cit-lpage">873</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/0092-8674(88)90428-X</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DLambie%26rft.auinit1%253DE.%2BJ.%26rft.volume%253D52%26rft.issue%253D6%26rft.spage%253D863%26rft.epage%253D873%26rft.atitle%253DA%2Byeast%2Bcentromere%2Bacts%2Bin%2Bcis%2Bto%2Binhibit%2Bmeiotic%2Bgene%2Bconversion%2Bof%2Badjacent%2Bsequences.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252888%252990428-X%26rft_id%253Dinfo%253Apmid%252F3280137%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(88)90428-X&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3280137&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3637.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1988M803100010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.64" data-doi="10.1371/journal.pgen.1003978"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lao</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cloud</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">C.-C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grubb</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thacker</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Meiotic Crossover Control by Concerted Action of Rad51-Dmc1 in Homolog Template Bias and Robust Homeostatic Regulation.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1003978</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1003978</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003978%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003978%26rft_id%253Dinfo%253Apmid%252F24367271%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003978&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24367271&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3637.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.65" data-doi="10.1371/journal.pgen.1006347"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lefrançois</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rockmill</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roeder</span>  <span class="cit-name-given-names">G. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Snyder</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Multiple Pairwise Analysis of Non-homologous Centromere Coupling Reveals Preferential Chromosome Size-Dependent Interactions and a Role for Bouquet Formation in Establishing the Interaction Pattern.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">e1006347</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1006347</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D12%26rft.spage%253De1006347%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1006347%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1006347&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.66" data-doi="10.1038/383840a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lydall</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nikolsky</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bishop</span>  <span class="cit-name-given-names">D. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weinert</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">A meiotic recombination checkpoint controlled by mitotic checkpoint genes.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">383</span>: <span class="cit-fpage">840</span>–<span class="cit-lpage">843</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/383840a0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLydall%26rft.auinit1%253DD.%26rft.volume%253D383%26rft.issue%253D6603%26rft.spage%253D840%26rft.epage%253D843%26rft.atitle%253DA%2Bmeiotic%2Brecombination%2Bcheckpoint%2Bcontrolled%2Bby%2Bmitotic%2Bcheckpoint%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F383840a0%26rft_id%253Dinfo%253Apmid%252F8893012%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/383840a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8893012&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3637.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996VQ14400068&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.67" data-doi="10.1016/j.gde.2013.05.007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Malkova</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ira</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Break-induced replication: functions and molecular mechanism.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Genet. Dev.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">271</span>–<span class="cit-lpage">279</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.gde.2013.05.007</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BOpin.%2BGenet.%2BDev.%26rft.volume%253D23%26rft.spage%253D271%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.gde.2013.05.007%26rft_id%253Dinfo%253Apmid%252F23790415%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.gde.2013.05.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23790415&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3637.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.68" data-doi="10.1073/pnas.93.14.7131"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Malkova</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ivanov</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haber</span>  <span class="cit-name-given-names">J. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span><span class="cit-comment">a</span> <span class="cit-article-title">Double-strand break repair in the absence of RAD51 in yeast: a possible role for break-induced DNA replication.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">7131</span>–<span class="cit-lpage">7136</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.93.14.7131</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DMalkova%26rft.auinit1%253DA.%26rft.volume%253D93%26rft.issue%253D14%26rft.spage%253D7131%26rft.epage%253D7136%26rft.atitle%253DDouble-strand%2Bbreak%2Brepair%2Bin%2Bthe%2Babsence%2Bof%2BRAD51%2Bin%2Byeast%253A%2Ba%2Bpossible%2Brole%2Bfor%2Bbreak-induced%2BDNA%2Breplication.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.93.14.7131%26rft_id%253Dinfo%253Apmid%252F8692957%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiOTMvMTQvNzEzMSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzgvMTEvMzYzNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.69" data-doi="10.1073/pnas.97.26.14500"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Malkova</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klein</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leung</span>  <span class="cit-name-given-names">W. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haber</span>  <span class="cit-name-given-names">J. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">HO endonuclease-induced recombination in yeast meiosis resembles Spo11-induced events.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">97</span>: <span class="cit-fpage">14500</span>–<span class="cit-lpage">14505</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.97.26.14500</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DMalkova%26rft.auinit1%253DA.%26rft.volume%253D97%26rft.issue%253D26%26rft.spage%253D14500%26rft.epage%253D14505%26rft.atitle%253DHO%2Bendonuclease-induced%2Brecombination%2Bin%2Byeast%2Bmeiosis%2Bresembles%2BSpo11-induced%2Bevents%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.97.26.14500%26rft_id%253Dinfo%253Apmid%252F11121053%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiOTcvMjYvMTQ1MDAiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzExLzM2MzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.70"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Malkova</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ross</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dawson</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoekstra</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haber</span>  <span class="cit-name-given-names">J. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span><span class="cit-comment">b</span> <span class="cit-article-title">Meiotic Recombination Initiated by a Double-Strand Break in Rad50Δ Yeast Cells Otherwise Unable to Initiate Meiotic Recombination.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">143</span>: <span class="cit-fpage">741</span>–<span class="cit-lpage">754</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMalkova%26rft.auinit1%253DA.%26rft.volume%253D143%26rft.issue%253D2%26rft.spage%253D741%26rft.epage%253D754%26rft.atitle%253DMeiotic%2BRecombination%2BInitiated%2Bby%2Ba%2BDouble-Strand%2BBreak%2Bin%2Brad50%257BDelta%257D%2BYeast%2BCells%2BOtherwise%2BUnable%2Bto%2BInitiate%2BMeiotic%2BRecombination%26rft_id%253Dinfo%253Apmid%252F8725223%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTQzLzIvNzQxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMS8zNjM3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.71"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mao-Draayer</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Galbraith</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pittman</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cool</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Malone</span>  <span class="cit-name-given-names">R. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Analysis of Meiotic Recombination Pathways in the Yeast <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">144</span>: <span class="cit-fpage">71</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMao-Draayer%26rft.auinit1%253DY.%26rft.volume%253D144%26rft.issue%253D1%26rft.spage%253D71%26rft.epage%253D86%26rft.atitle%253DAnalysis%2Bof%2BMeiotic%2BRecombination%2BPathways%2Bin%2Bthe%2BYeast%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Apmid%252F8878674%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMTQ0LzEvNzEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzExLzM2MzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.72" data-doi="10.1371/journal.pgen.1006928"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Markowitz</span>  <span class="cit-name-given-names">T. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Suarez</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blitzblau</span>  <span class="cit-name-given-names">H. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patel</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Markhard</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Reduced dosage of the chromosome axis factor Red1 selectively disrupts the meiotic recombination checkpoint in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">e1006928</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1006928</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D13%26rft.spage%253De1006928%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1006928%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1006928&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.73" data-doi="10.1371/journal.pbio.0050299"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McMahill</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sham</span>  <span class="cit-name-given-names">C. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bishop</span>  <span class="cit-name-given-names">D. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Synthesis-dependent strand annealing in meiosis.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e299</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pbio.0050299</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DMcMahill%26rft.auinit1%253DM.%2BS.%26rft.volume%253D5%26rft.issue%253D11%26rft.spage%253De299%26rft.epage%253De299%26rft.atitle%253DSynthesis-dependent%2Bstrand%2Bannealing%2Bin%2Bmeiosis.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0050299%26rft_id%253Dinfo%253Apmid%252F17988174%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0050299&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17988174&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3637.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.74" data-doi="10.7554/eLife.19669"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Medhi</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goldman</span>  <span class="cit-name-given-names">A. S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lichten</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Local chromosome context is a major determinant of crossover pathway biochemistry during budding yeast meiosis.</span> <abbr class="cit-jnl-abbrev">eLife</abbr> <span class="cit-vol">5</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.7554/eLife.19669</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DeLife%26rft_id%253Dinfo%253Adoi%252F10.7554%252FeLife.19669%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.7554/eLife.19669&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.75" data-doi="10.1016/S0092-8674(01)80007-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Michaelis</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ciosk</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nasmyth</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Cohesins: Chromosomal Proteins that Prevent Premature Separation of Sister Chromatids.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">35</span>–<span class="cit-lpage">45</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0092-8674(01)80007-6</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DMichaelis%26rft.auinit1%253DC.%26rft.volume%253D91%26rft.issue%253D1%26rft.spage%253D35%26rft.epage%253D45%26rft.atitle%253DCohesins%253A%2Bchromosomal%2Bproteins%2Bthat%2Bprevent%2Bpremature%2Bseparation%2Bof%2Bsister%2Bchromatids.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252801%252980007-6%26rft_id%253Dinfo%253Apmid%252F9335333%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0092-8674(01)80007-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9335333&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3637.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997XZ80900007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.76"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mullen</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaliraman</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brill</span>  <span class="cit-name-given-names">S. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Bipartite structure of the SGS1 DNA helicase in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">154</span>: <span class="cit-fpage">1101</span>–<span class="cit-lpage">1114</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMullen%26rft.auinit1%253DJ.%2BR.%26rft.volume%253D154%26rft.issue%253D3%26rft.spage%253D1101%26rft.epage%253D1114%26rft.atitle%253DBipartite%2BStructure%2Bof%2Bthe%2BSGS1%2BDNA%2BHelicase%2Bin%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Apmid%252F10757756%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1NC8zLzExMDEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzExLzM2MzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.77" data-doi="10.1101/gad.11.17.2272"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nairz</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klein</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">mre11S–a yeast mutation that blocks double-strand-break processing and permits nonhomologous synapsis in meiosis.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">2272</span>–<span class="cit-lpage">2290</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gad.11.17.2272</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DNairz%26rft.auinit1%253DK.%26rft.volume%253D11%26rft.issue%253D17%26rft.spage%253D2272%26rft.epage%253D2290%26rft.atitle%253Dmre11S---a%2Byeast%2Bmutation%2Bthat%2Bblocks%2Bdouble-strand-break%2Bprocessing%2Band%2Bpermits%2Bnonhomologous%2Bsynapsis%2Bin%25A0meiosis%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.11.17.2272%26rft_id%253Dinfo%253Apmid%252F9303542%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjExLzE3LzIyNzIiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzExLzM2MzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference  in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.78" data-doi="10.1016/S1097-2765(02)00498-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Neale</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramachandran</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trelles-Sticken</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scherthan</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goldman</span>  <span class="cit-name-given-names">A. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Wild-type levels of Spo11-induced DSBs are required for normal single-strand resection during meiosis.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">835</span>–<span class="cit-lpage">846</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S1097-2765(02)00498-7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DNeale%26rft.auinit1%253DM.%2BJ.%26rft.volume%253D9%26rft.issue%253D4%26rft.spage%253D835%26rft.epage%253D846%26rft.atitle%253DWild-type%2Blevels%2Bof%2BSpo11-induced%2BDSBs%2Bare%2Brequired%2Bfor%2Bnormal%2Bsingle-strand%2Bresection%2Bduring%2Bmeiosis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1097-2765%252802%252900498-7%26rft_id%253Dinfo%253Apmid%252F11983174%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1097-2765(02)00498-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11983174&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3637.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000175244400016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference  in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.79" data-doi="10.1073/pnas.0913435107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Newnham</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jordan</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rockmill</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roeder</span>  <span class="cit-name-given-names">G. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoffmann</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The synaptonemal complex protein, Zip1, promotes the segregation of nonexchange chromosomes at meiosis I.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">781</span>–<span class="cit-lpage">785</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.0913435107</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0913435107%26rft_id%253Dinfo%253Apmid%252F20080752%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiIxMDcvMi83ODEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzExLzM2MzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference  in text" id="ref-80">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.80" data-doi="10.1091/mbc.E05-05-0465"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Niu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wan</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baumgartner</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schaefer</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Loidl</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Partner choice during meiosis is regulated by Hop1-promoted dimerization of Mek1.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">5804</span>–<span class="cit-lpage">5818</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1091/mbc.e05-05-0465</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.E05-05-0465%26rft_id%253Dinfo%253Apmid%252F16221890%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjEwOiIxNi8xMi81ODA0IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMS8zNjM3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference  in text" id="ref-81">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.81" data-doi="10.1016/j.molcel.2009.09.029"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Niu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wan</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Busygina</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kwon</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Allen</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Regulation of meiotic recombination via Mek1-mediated Rad54 phosphorylation.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">393</span>–<span class="cit-lpage">404</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.molcel.2009.09.029</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DNiu%26rft.auinit1%253DH.%26rft.volume%253D36%26rft.issue%253D3%26rft.spage%253D393%26rft.epage%253D404%26rft.atitle%253DRegulation%2Bof%2Bmeiotic%2Brecombination%2Bvia%2BMek1-mediated%2BRad54%2Bphosphorylation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2009.09.029%26rft_id%253Dinfo%253Apmid%252F19917248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2009.09.029&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19917248&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3637.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000272506200007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-82-1" title="View reference  in text" id="ref-82">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.82"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Novak</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ross-Macdonald</span>  <span class="cit-name-given-names">P. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roeder</span>  <span class="cit-name-given-names">G. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">The budding yeast Msh4 protein functions in chromosome synapsis and the regulation of crossover distribution.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">158</span>: <span class="cit-fpage">1013</span>–<span class="cit-lpage">1025</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DNovak%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D158%26rft.issue%253D3%26rft.spage%253D1013%26rft.epage%253D1025%26rft.atitle%253DThe%2BBudding%2BYeast%2BMsh4%2BProtein%2BFunctions%2Bin%2BChromosome%2BSynapsis%2Band%2Bthe%2BRegulation%2Bof%2BCrossover%2BDistribution%26rft_id%253Dinfo%253Apmid%252F11454751%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1OC8zLzEwMTMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzExLzM2MzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-83-1" title="View reference  in text" id="ref-83">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.83" data-doi="10.1016/j.molcel.2016.09.011"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Oh</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Al-Zain</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cannavo</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cejka</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Symington</span>  <span class="cit-name-given-names">L. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Xrs2 dependent and independent functions of the Mre11-Rad50 complex.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">64</span>: <span class="cit-fpage">405</span>–<span class="cit-lpage">415</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.molcel.2016.09.011</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell%26rft.volume%253D64%26rft.spage%253D405%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2016.09.011%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2016.09.011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-84-1" title="View reference  in text" id="ref-84">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.84" data-doi="10.1126/science.1086605"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Page</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hawley</span>  <span class="cit-name-given-names">R. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Chromosome choreography: the meiotic ballet.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">301</span>: <span class="cit-fpage">785</span>–<span class="cit-lpage">789</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1086605</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DPage%26rft.auinit1%253DS.%2BL.%26rft.volume%253D301%26rft.issue%253D5634%26rft.spage%253D785%26rft.epage%253D789%26rft.atitle%253DChromosome%2BChoreography%253A%2BThe%2BMeiotic%2BBallet%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1086605%26rft_id%253Dinfo%253Apmid%252F12907787%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMDEvNTYzNC83ODUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzExLzM2MzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-85-1" title="View reference  in text" id="ref-85">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.85" data-doi="10.1146/annurev.cellbio.19.111301.155141"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Page</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hawley</span>  <span class="cit-name-given-names">R. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">The Genetics and Molecular Biology of the Synaptonemal Complex.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Cell Dev. Biol.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">525</span>–<span class="cit-lpage">558</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1146/annurev.cellbio.19.111301.155141</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bcell%2Band%2Bdevelopmental%2Bbiology%26rft.stitle%253DAnnu%2BRev%2BCell%2BDev%2BBiol%26rft.volume%253D20%26rft.spage%253D525%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.cellbio.19.111301.155141%26rft_id%253Dinfo%253Apmid%252F15473851%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.cellbio.19.111301.155141&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15473851&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3637.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000225318200020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-86-1" title="View reference  in text" id="ref-86">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.86" data-doi="10.1016/j.cell.2011.02.009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sasaki</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kniewel</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murakami</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blitzblau</span>  <span class="cit-name-given-names">H. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A hierarchical combination of factors shapes the genome-wide topography of yeast meiotic recombination initiation.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">144</span>: <span class="cit-fpage">719</span>–<span class="cit-lpage">731</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.cell.2011.02.009</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DPan%26rft.auinit1%253DJ.%26rft.volume%253D144%26rft.issue%253D5%26rft.spage%253D719%26rft.epage%253D731%26rft.atitle%253DA%2Bhierarchical%2Bcombination%2Bof%2Bfactors%2Bshapes%2Bthe%2Bgenome-wide%2Btopography%2Bof%2Byeast%2Bmeiotic%2Brecombination%2Binitiation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2011.02.009%26rft_id%253Dinfo%253Apmid%252F21376234%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2011.02.009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21376234&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3637.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000288007100011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-87-1" title="View reference  in text" id="ref-87">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.87" data-doi="10.1016/j.cell.2011.07.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Panizza</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mendoza</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berlinger</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nicolas</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Spo11-accessory proteins link double-strand break sites to the chromosome axis in early meiotic recombination.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">146</span>: <span class="cit-fpage">372</span>–<span class="cit-lpage">383</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.cell.2011.07.003</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DPanizza%26rft.auinit1%253DS.%26rft.volume%253D146%26rft.issue%253D3%26rft.spage%253D372%26rft.epage%253D383%26rft.atitle%253DSpo11-accessory%2Bproteins%2Blink%2Bdouble-strand%2Bbreak%2Bsites%2Bto%2Bthe%2Bchromosome%2Baxis%2Bin%2Bearly%2Bmeiotic%2Brecombination.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2011.07.003%26rft_id%253Dinfo%253Apmid%252F21816273%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2011.07.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21816273&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3637.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000293570500012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-88-1" title="View reference  in text" id="ref-88">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.88" data-doi="10.1016/j.tcb.2005.10.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pawlowski</span>  <span class="cit-name-given-names">W. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cande</span>  <span class="cit-name-given-names">W. Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Coordinating the events of the meiotic prophase.</span> <abbr class="cit-jnl-abbrev">Trends Cell Biol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">674</span>–<span class="cit-lpage">681</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tcb.2005.10.005</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bcell%2Bbiology%26rft.stitle%253DTrends%2BCell%2BBiol%26rft.aulast%253DPawlowski%26rft.auinit1%253DW.%2BP.%26rft.volume%253D15%26rft.issue%253D12%26rft.spage%253D674%26rft.epage%253D681%26rft.atitle%253DCoordinating%2Bthe%2Bevents%2Bof%2Bthe%2Bmeiotic%2Bprophase.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tcb.2005.10.005%26rft_id%253Dinfo%253Apmid%252F16257210%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tcb.2005.10.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16257210&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3637.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000234316700009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-89-1" title="View reference  in text" id="ref-89">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.89" data-doi="10.1101/gad.321105"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Prieler</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Penkner</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borde</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klein</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">The control of Spo11’s interaction with meiotic recombination hotspots.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">255</span>–<span class="cit-lpage">269</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gad.321105</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DPrieler%26rft.auinit1%253DS.%26rft.volume%253D19%26rft.issue%253D2%26rft.spage%253D255%26rft.epage%253D269%26rft.atitle%253DThe%2Bcontrol%2Bof%2BSpo11%2527s%2Binteraction%2Bwith%2Bmeiotic%2Brecombination%2Bhotspots%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.321105%26rft_id%253Dinfo%253Apmid%252F15655113%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6ODoiMTkvMi8yNTUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzExLzM2MzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-90-1" title="View reference  in text" id="ref-90">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.90" data-doi="10.1016/0022-2836(88)90136-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ray</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Siddiqi</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kolodkin</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stahl</span>  <span class="cit-name-given-names">F. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">Intra-chromosomal gene conversion induced by a DNA double-strand break in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">201</span>: <span class="cit-fpage">247</span>–<span class="cit-lpage">260</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/0022-2836(88)90136-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DRay%26rft.auinit1%253DA.%26rft.volume%253D201%26rft.issue%253D2%26rft.spage%253D247%26rft.epage%253D260%26rft.atitle%253DIntra-chromosomal%2Bgene%2Bconversion%2Binduced%2Bby%2Ba%2BDNA%2Bdouble-strand%2Bbreak%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0022-2836%252888%252990136-2%26rft_id%253Dinfo%253Apmid%252F3047399%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0022-2836(88)90136-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3047399&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3637.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1988N543900002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-91-1" title="View reference  in text" id="ref-91">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.91" data-doi="10.1016/j.cub.2003.10.059"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rockmill</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fung</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Branda</span>  <span class="cit-name-given-names">S. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roeder</span>  <span class="cit-name-given-names">G. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">The Sgs1 Helicase Regulates Chromosome Synapsis and Meiotic Crossing Over.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">1954</span>–<span class="cit-lpage">1962</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.cub.2003.10.059</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DRockmill%26rft.auinit1%253DB.%26rft.volume%253D13%26rft.issue%253D22%26rft.spage%253D1954%26rft.epage%253D1962%26rft.atitle%253DThe%2BSgs1%2Bhelicase%2Bregulates%2Bchromosome%2Bsynapsis%2Band%2Bmeiotic%2Bcrossing%2Bover.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2003.10.059%26rft_id%253Dinfo%253Apmid%252F14614820%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2003.10.059&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14614820&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3637.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000186558100021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-92-1" title="View reference  in text" id="ref-92">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.92" data-doi="10.1371/journal.pgen.1003932"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rockmill</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lefrançois</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Voelkel-Meiman</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oke</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roeder</span>  <span class="cit-name-given-names">G. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">High Throughput Sequencing Reveals Alterations in the Recombination Signatures with Diminishing Spo11 Activity.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1003932</span> <span class="cit-comment">(correction: PLOS Genetics 9: 10.1371/annotation/b3e65d4a-f7ed-4481-9b5b-92d952d5ee02 )</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1003932</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003932%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003932%26rft_id%253Dinfo%253Apmid%252F24204324%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003932&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24204324&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3637.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-93-1" title="View reference  in text" id="ref-93">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.93" data-doi="10.1101/gad.5.12b.2392"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rockmill</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roeder</span>  <span class="cit-name-given-names">G. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">A meiosis-specific protein kinase homolog required for chromosome synapsis and recombination.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">2392</span>–<span class="cit-lpage">2404</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gad.5.12b.2392</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DRockmill%26rft.auinit1%253DB%26rft.volume%253D5%26rft.issue%253D12b%26rft.spage%253D2392%26rft.epage%253D2404%26rft.atitle%253DA%2Bmeiosis-specific%2Bprotein%2Bkinase%2Bhomolog%2Brequired%2Bfor%2Bchromosome%2Bsynapsis%2Band%2Brecombination.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.5.12b.2392%26rft_id%253Dinfo%253Apmid%252F1752435%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjUvMTJiLzIzOTIiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzExLzM2MzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-94-1" title="View reference  in text" id="ref-94">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.94" data-doi="10.1101/gad.9.21.2684"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rockmill</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sym</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scherthan</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roeder</span>  <span class="cit-name-given-names">G. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Roles for two RecA homologs in promoting meiotic chromosome synapsis.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">2684</span>–<span class="cit-lpage">2695</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gad.9.21.2684</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DRockmill%26rft.auinit1%253DB%26rft.volume%253D9%26rft.issue%253D21%26rft.spage%253D2684%26rft.epage%253D2695%26rft.atitle%253DRoles%2Bfor%2Btwo%2BRecA%2Bhomologs%2Bin%2Bpromoting%2Bmeiotic%2Bchromosome%2Bsynapsis.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.9.21.2684%26rft_id%253Dinfo%253Apmid%252F7590245%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6OToiOS8yMS8yNjg0IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMS8zNjM3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-95-1" title="View reference  in text" id="ref-95">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.95" data-doi="10.1016/0092-8674(94)90037-X"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ross-Macdonald</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roeder</span>  <span class="cit-name-given-names">G. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Mutation of a meiosis-specific MutS homolog decreases crossing over but not mismatch correction.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">79</span>: <span class="cit-fpage">1069</span>–<span class="cit-lpage">1080</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/0092-8674(94)90037-X</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DRoss-Macdonald%26rft.auinit1%253DP.%26rft.volume%253D79%26rft.issue%253D6%26rft.spage%253D1069%26rft.epage%253D1080%26rft.atitle%253DMutation%2Bof%2Ba%2Bmeiosis-specific%2BMutS%2Bhomolog%2Bdecreases%2Bcrossing%2Bover%2Bbut%2Bnot%2Bmismatch%2Bcorrection.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252894%252990037-X%26rft_id%253Dinfo%253Apmid%252F8001134%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(94)90037-X&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8001134&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3637.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1994PY08600016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-96-1" title="View reference  in text" id="ref-96">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.96" data-doi="10.1016/0092-8674(94)90172-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schwacha</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kleckner</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Identification of joint molecules that form frequently between homologs but rarely between sister chromatids during yeast meiosis.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">76</span>: <span class="cit-fpage">51</span>–<span class="cit-lpage">63</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/0092-8674(94)90172-4</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DSchwacha%26rft.auinit1%253DA.%26rft.volume%253D76%26rft.issue%253D1%26rft.spage%253D51%26rft.epage%253D63%26rft.atitle%253DIdentification%2Bof%2Bjoint%2Bmolecules%2Bthat%2Bform%2Bfrequently%2Bbetween%2Bhomologs%2Bbut%2Brarely%2Bbetween%2Bsister%2Bchromatids%2Bduring%2Byeast%2Bmeiosis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252894%252990172-4%26rft_id%253Dinfo%253Apmid%252F8287479%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(94)90172-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8287479&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3637.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1994MR49500006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-97-1" title="View reference  in text" id="ref-97">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.97" data-doi="10.1016/0092-8674(95)90191-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schwacha</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kleckner</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Identification of double holliday junctions as intermediates in meiotic recombination.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">83</span>: <span class="cit-fpage">783</span>–<span class="cit-lpage">791</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/0092-8674(95)90191-4</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DSchwacha%26rft.auinit1%253DA.%26rft.volume%253D83%26rft.issue%253D5%26rft.spage%253D783%26rft.epage%253D791%26rft.atitle%253DIdentification%2Bof%2Bdouble%2BHolliday%2Bjunctions%2Bas%2Bintermediates%2Bin%2Bmeiotic%2Brecombination.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252895%252990191-4%26rft_id%253Dinfo%253Apmid%252F8521495%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(95)90191-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8521495&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3637.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995TH94800015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-98-1" title="View reference  in text" id="ref-98">↵</a><div class="cit ref-cit ref-book" id="cit-8.11.3637.98"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schwacha</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kleckner</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span>, pp. <span class="cit-fpage">1123</span>–<span class="cit-lpage">1135</span> in <span class="cit-source">Interhomolog bias during meiotic recombination: meiotic functions promote a highly differentiated interhomolog-only pathway</span>, <span class="cit-publ-name">Cell</span> <span class="cit-vol">90</span>(<span class="cit-issue">6</span>): <span class="cit-fpage">1123</span>–<span class="cit-lpage">35</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInterhomolog%2Bbias%2Bduring%2Bmeiotic%2Brecombination%253A%2Bmeiotic%2Bfunctions%2Bpromote%2Ba%2Bhighly%2Bdifferentiated%2Binterhomolog-only%2Bpathway%26rft.volume%253D90%26rft.spage%253D1123%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-99-1" title="View reference  in text" id="ref-99">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.99" data-doi="10.1046/j.1365-2443.1997.1480347.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shinohara</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gasior</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ogawa</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kleckner</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bishop</span>  <span class="cit-name-given-names">D. K.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span><span class="cit-comment">a</span> <span class="cit-article-title">Saccharomyces cerevisiae recA homologues RAD51 and DMC1 have both distinct and overlapping roles in meiotic recombination.</span> <abbr class="cit-jnl-abbrev">Genes Cells</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">615</span>–<span class="cit-lpage">629</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1046/j.1365-2443.1997.1480347.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2Bto%2BCells%26rft.stitle%253DGENES%2BCELLS%26rft.aulast%253DShinohara%26rft.auinit1%253DA%26rft.volume%253D2%26rft.issue%253D10%26rft.spage%253D615%26rft.epage%253D629%26rft.atitle%253DSaccharomyces%2Bcerevisiae%2BrecA%2Bhomologues%2BRAD51%2Band%2BDMC1%2Bhave%2Bboth%2Bdistinct%2Band%2Boverlapping%2Broles%2Bin%2Bmeiotic%2Brecombination%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-2443.1997.1480347.x%26rft_id%253Dinfo%253Apmid%252F9427283%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-2443.1997.1480347.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9427283&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3637.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997YJ05700003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-100-1" title="View reference  in text" id="ref-100">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.100" data-doi="10.1016/0092-8674(92)90447-K"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shinohara</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ogawa</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ogawa</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Rad51 protein involved in repair and recombination in <em>S. cerevisiae</em> is a RecA-like protein.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">69</span>: <span class="cit-fpage">457</span>–<span class="cit-lpage">470</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/0092-8674(92)90447-K</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DShinohara%26rft.auinit1%253DA.%26rft.volume%253D69%26rft.issue%253D3%26rft.spage%253D457%26rft.epage%253D470%26rft.atitle%253DRad51%2Bprotein%2Binvolved%2Bin%2Brepair%2Band%2Brecombination%2Bin%2BS.%2Bcerevisiae%2Bis%2Ba%2BRecA-like%2Bprotein.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252892%252990447-K%26rft_id%253Dinfo%253Apmid%252F1581961%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(92)90447-K&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1581961&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3637.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1992HT07800008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-101-1" title="View reference  in text" id="ref-101">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.101" data-doi="10.1038/ng.83"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shinohara</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oh</span>  <span class="cit-name-given-names">S. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hunter</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shinohara</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Crossover assurance and crossover interference are distinctly regulated by the ZMM proteins during yeast meiosis.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">299</span>–<span class="cit-lpage">309</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.83</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DShinohara%26rft.auinit1%253DM.%26rft.volume%253D40%26rft.issue%253D3%26rft.spage%253D299%26rft.epage%253D309%26rft.atitle%253DCrossover%2Bassurance%2Band%2Bcrossover%2Binterference%2Bare%2Bdistinctly%2Bregulated%2Bby%2Bthe%2BZMM%2Bproteins%2Bduring%2Byeast%2Bmeiosis.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.83%26rft_id%253Dinfo%253Apmid%252F18297071%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.83&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18297071&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3637.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000253548400020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-102-1" title="View reference  in text" id="ref-102">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.102"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shinohara</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shita-Yamaguchi</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buerstedde</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shinagawa</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ogawa</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1997</span><span class="cit-comment">b</span> <span class="cit-article-title">Characterization of the roles of the Saccharomyces cerevisiae RAD54 gene and a homologue of RAD54, RDH54/TID1, in mitosis and meiosis.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">147</span>: <span class="cit-fpage">1545</span>–<span class="cit-lpage">1556</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DShinohara%26rft.auinit1%253DM.%26rft.volume%253D147%26rft.issue%253D4%26rft.spage%253D1545%26rft.epage%253D1556%26rft.atitle%253DCharacterization%2Bof%2Bthe%2BRoles%2Bof%2Bthe%2BSaccharomyces%2Bcerevisiae%2BRAD54%2BGene%2Band%2Ba%2BHomologue%2Bof%2BRAD54%252C%2BRDH54%252FTID1%252C%2Bin%2BMitosis%2Band%2BMeiosis%26rft_id%253Dinfo%253Apmid%252F9409820%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE0Ny80LzE1NDUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzExLzM2MzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-103-1" title="View reference  in text" id="ref-103">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.103"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sikorski</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hieter</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">A System of Shuttle Vectors and Yeast Host Strains Designed for Efficient Manipulation of DNA in <em>Saccharomyces Cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">122</span>: <span class="cit-fpage">19</span>–<span class="cit-lpage">27</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DSikorski%26rft.auinit1%253DR.%2BS.%26rft.volume%253D122%26rft.issue%253D1%26rft.spage%253D19%26rft.epage%253D27%26rft.atitle%253DA%2Bsystem%2Bof%2Bshuttle%2Bvectors%2Band%2Byeast%2Bhost%2Bstrains%2Bdesigned%2Bfor%2Befficient%2Bmanipulation%2Bof%2BDNA%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Apmid%252F2659436%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMTIyLzEvMTkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzExLzM2MzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-104-1" title="View reference  in text" id="ref-104">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.104" data-doi="10.1016/j.tig.2008.08.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stewart</span>  <span class="cit-name-given-names">M. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dawson</span>  <span class="cit-name-given-names">D. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Changing partners: moving from non-homologous to homologous centromere pairing in meiosis.</span> <abbr class="cit-jnl-abbrev">Trends Genet</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">564</span>–<span class="cit-lpage">573</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tig.2008.08.006</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DStewart%26rft.auinit1%253DM.%2BN.%26rft.volume%253D24%26rft.issue%253D11%26rft.spage%253D564%26rft.epage%253D573%26rft.atitle%253DChanging%2Bpartners%253A%2Bmoving%2Bfrom%2Bnon-homologous%2Bto%2Bhomologous%2Bcentromere%2Bpairing%2Bin%2Bmeiosis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2008.08.006%26rft_id%253Dinfo%253Apmid%252F18804891%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2008.08.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18804891&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3637.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261533700005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-105-1" title="View reference  in text" id="ref-105">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.105" data-doi="10.1016/0092-8674(93)90114-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sym</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Engebrecht</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roeder</span>  <span class="cit-name-given-names">G. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">ZIP1 is a synaptonemal complex protein required for meiotic chromosome synapsis.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">72</span>: <span class="cit-fpage">365</span>–<span class="cit-lpage">378</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/0092-8674(93)90114-6</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DSym%26rft.auinit1%253DM.%26rft.volume%253D72%26rft.issue%253D3%26rft.spage%253D365%26rft.epage%253D378%26rft.atitle%253DZIP1%2Bis%2Ba%2Bsynaptonemal%2Bcomplex%2Bprotein%2Brequired%2Bfor%2Bmeiotic%2Bchromosome%2Bsynapsis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252893%252990114-6%26rft_id%253Dinfo%253Apmid%252F7916652%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(93)90114-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7916652&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3637.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1993KM16200007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-106-1" title="View reference  in text" id="ref-106">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.106" data-doi="10.1083/jcb.128.4.455"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sym</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roeder</span>  <span class="cit-name-given-names">G. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Zip1-induced changes in synaptonemal complex structure and polycomplex assembly.</span> <abbr class="cit-jnl-abbrev">J. Cell Biol.</abbr> <span class="cit-vol">128</span>: <span class="cit-fpage">455</span>–<span class="cit-lpage">466</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1083/jcb.128.4.455</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2BCell%2BBiology%26rft.stitle%253DJCB%26rft.aulast%253DSym%26rft.auinit1%253DM.%26rft.volume%253D128%26rft.issue%253D4%26rft.spage%253D455%26rft.epage%253D466%26rft.atitle%253DZip1-induced%2Bchanges%2Bin%2Bsynaptonemal%2Bcomplex%2Bstructure%2Band%2Bpolycomplex%2Bassembly%26rft_id%253Dinfo%253Adoi%252F10.1083%252Fjcb.128.4.455%26rft_id%253Dinfo%253Apmid%252F7860625%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNiIjtzOjU6InJlc2lkIjtzOjk6IjEyOC80LzQ1NSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzgvMTEvMzYzNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-107-1" title="View reference  in text" id="ref-107">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.107" data-doi="10.1038/nature13120"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thacker</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mohibullah</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keeney</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Homologue engagement controls meiotic DNA break number and distribution.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">510</span>: <span class="cit-fpage">241</span>–<span class="cit-lpage">246</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature13120</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D510%26rft.spage%253D241%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature13120%26rft_id%253Dinfo%253Apmid%252F24717437%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature13120&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24717437&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3637.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000337032400044&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-108-1" title="View reference  in text" id="ref-108">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thorne</span>  <span class="cit-name-given-names">L.W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Byers</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>., <span class="cit-pub-date">1993</span> <span class="cit-article-title">Stage-specific effects of X-irradiation on yeast meiosis.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">134</span>: <span class="cit-fpage">29</span>–<span class="cit-lpage">42</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DThorne%26rft.auinit1%253DL.%2BW.%26rft.volume%253D134%26rft.issue%253D1%26rft.spage%253D29%26rft.epage%253D42%26rft.atitle%253DStage-specific%2Beffects%2Bof%2BX-irradiation%2Bon%2Byeast%2Bmeiosis.%26rft_id%253Dinfo%253Apmid%252F8514137%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMTM0LzEvMjkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzExLzM2MzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-109-1" title="View reference  in text" id="ref-109">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.109" data-doi="10.1073/pnas.76.9.4350"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Towbin</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Staehelin</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gordon</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1979</span> <span class="cit-article-title">Electrophoretic transfer of proteins from polyacrylamide gels to nitrocellulose sheets: procedure and some applications.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">76</span>: <span class="cit-fpage">4350</span>–<span class="cit-lpage">4354</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.76.9.4350</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DTowbin%26rft.auinit1%253DH.%26rft.volume%253D76%26rft.issue%253D9%26rft.spage%253D4350%26rft.epage%253D4354%26rft.atitle%253DElectrophoretic%2Btransfer%2Bof%2Bproteins%2Bfrom%2Bpolyacrylamide%2Bgels%2Bto%2Bnitrocellulose%2Bsheets%253A%2Bprocedure%2Band%2Bsome%2Bapplications.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.76.9.4350%26rft_id%253Dinfo%253Apmid%252F388439%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI3Ni85LzQzNTAiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzExLzM2MzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-110-1" title="View reference  in text" id="ref-110">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.110" data-doi="10.1128/MCB.18.1.260"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tsubouchi</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ogawa</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">A novel mre11 mutation impairs processing of double-strand breaks of DNA during both mitosis and meiosis.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">260</span>–<span class="cit-lpage">268</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/MCB.18.1.260</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DTsubouchi%26rft.auinit1%253DH.%26rft.volume%253D18%26rft.issue%253D1%26rft.spage%253D260%26rft.epage%253D268%26rft.atitle%253DA%2BNovel%2Bmre11%2BMutation%2BImpairs%2BProcessing%2Bof%2BDouble-Strand%2BBreaks%2Bof%2BDNA%2Bduring%2BBoth%2BMitosis%2Band%2BMeiosis%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.18.1.260%26rft_id%253Dinfo%253Apmid%252F9418873%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIxOC8xLzI2MC1hIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMS8zNjM3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-111-1" title="View reference  in text" id="ref-111">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.111" data-doi="10.1016/S1534-5807(03)00357-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tsubouchi</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roeder</span>  <span class="cit-name-given-names">G. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">The importance of genetic recombination for fidelity of chromosome pairing in meiosis.</span> <abbr class="cit-jnl-abbrev">Dev. Cell</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">915</span>–<span class="cit-lpage">925</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S1534-5807(03)00357-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bcell%26rft.stitle%253DDev%2BCell%26rft.aulast%253DTsubouchi%26rft.auinit1%253DH.%26rft.volume%253D5%26rft.issue%253D6%26rft.spage%253D915%26rft.epage%253D925%26rft.atitle%253DThe%2Bimportance%2Bof%2Bgenetic%2Brecombination%2Bfor%2Bfidelity%2Bof%2Bchromosome%2Bpairing%2Bin%2Bmeiosis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1534-5807%252803%252900357-5%26rft_id%253Dinfo%253Apmid%252F14667413%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1534-5807(03)00357-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14667413&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3637.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000187528300012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-112-1" title="View reference  in text" id="ref-112">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.112" data-doi="10.1101/gad.1422506"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tsubouchi</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roeder</span>  <span class="cit-name-given-names">G. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Budding yeast Hed1 down-regulates the mitotic recombination machinery when meiotic recombination is impaired.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">1766</span>–<span class="cit-lpage">1775</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gad.1422506</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DTsubouchi%26rft.auinit1%253DH.%26rft.volume%253D20%26rft.issue%253D13%26rft.spage%253D1766%26rft.epage%253D1775%26rft.atitle%253DBudding%2Byeast%2BHed1%2Bdown-regulates%2Bthe%2Bmitotic%2Brecombination%2Bmachinery%2Bwhen%2Bmeiotic%2Brecombination%2Bis%2Bimpaired.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.1422506%26rft_id%253Dinfo%253Apmid%252F16818607%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjIwLzEzLzE3NjYiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzExLzM2MzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-113-1" title="View reference  in text" id="ref-113">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.113" data-doi="10.1101/gad.1709408"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tsubouchi</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">MacQueen</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roeder</span>  <span class="cit-name-given-names">G. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Initiation of meiotic chromosome synapsis at centromeres in budding yeast.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">3217</span>–<span class="cit-lpage">3226</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gad.1709408</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DTsubouchi%26rft.auinit1%253DT.%26rft.volume%253D22%26rft.issue%253D22%26rft.spage%253D3217%26rft.epage%253D3226%26rft.atitle%253DInitiation%2Bof%2Bmeiotic%2Bchromosome%2Bsynapsis%2Bat%2Bcentromeres%2Bin%2Bbudding%2Byeast%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.1709408%26rft_id%253Dinfo%253Apmid%252F19056898%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjIyLzIyLzMyMTciO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzExLzM2MzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-114-1" title="View reference  in text" id="ref-114">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.114" data-doi="10.1126/science.1108283"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tsubouchi</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roeder</span>  <span class="cit-name-given-names">G. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">A synaptonemal complex protein promotes homology-independent centromere coupling.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">308</span>: <span class="cit-fpage">870</span>–<span class="cit-lpage">873</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1108283</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DTsubouchi%26rft.auinit1%253DT.%26rft.volume%253D308%26rft.issue%253D5723%26rft.spage%253D870%26rft.epage%253D873%26rft.atitle%253DA%2BSynaptonemal%2BComplex%2BProtein%2BPromotes%2BHomology-Independent%2BCentromere%2BCoupling%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1108283%26rft_id%253Dinfo%253Apmid%252F15879219%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMDgvNTcyMy84NzAiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzExLzM2MzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-115-1" title="View reference  in text" id="ref-115">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.115" data-doi="10.1016/j.devcel.2006.04.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tsubouchi</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roeder</span>  <span class="cit-name-given-names">G. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">The meiosis-specific zip4 protein regulates crossover distribution by promoting synaptonemal complex formation together with zip2.</span> <abbr class="cit-jnl-abbrev">Dev. Cell</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">809</span>–<span class="cit-lpage">819</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.devcel.2006.04.003</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bcell%26rft.stitle%253DDev%2BCell%26rft.aulast%253DTsubouchi%26rft.auinit1%253DT.%26rft.volume%253D10%26rft.issue%253D6%26rft.spage%253D809%26rft.epage%253D819%26rft.atitle%253DThe%2Bmeiosis-specific%2Bzip4%2Bprotein%2Bregulates%2Bcrossover%2Bdistribution%2Bby%2Bpromoting%2Bsynaptonemal%2Bcomplex%2Bformation%2Btogether%2Bwith%2Bzip2.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.devcel.2006.04.003%26rft_id%253Dinfo%253Apmid%252F16740482%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.devcel.2006.04.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16740482&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3637.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000238244700014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-116-1" title="View reference  in text" id="ref-116">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.116" data-doi="10.1016/S1097-2765(01)00270-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Usui</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ogawa</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petrini</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">A DNA damage response pathway controlled by Tel1 and the Mre11 complex.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1255</span>–<span class="cit-lpage">1266</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S1097-2765(01)00270-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DUsui%26rft.auinit1%253DT.%26rft.volume%253D7%26rft.issue%253D6%26rft.spage%253D1255%26rft.epage%253D1266%26rft.atitle%253DA%2BDNA%2Bdamage%2Bresponse%2Bpathway%2Bcontrolled%2Bby%2BTel1%2Band%2Bthe%2BMre11%2Bcomplex.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1097-2765%252801%252900270-2%26rft_id%253Dinfo%253Apmid%252F11430828%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1097-2765(01)00270-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11430828&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3637.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000169547400013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-117-1" title="View reference  in text" id="ref-117">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.117" data-doi="10.7554/eLife.10850"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vincenten</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuhl</span>  <span class="cit-name-given-names">L.-M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lam</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oke</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kerr</span>  <span class="cit-name-given-names">A. R. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The kinetochore prevents centromere-proximal crossover recombination during meiosis.</span> <abbr class="cit-jnl-abbrev">eLife</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e10850</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.7554/eLife.10850</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DeLife%26rft_id%253Dinfo%253Adoi%252F10.7554%252FeLife.10850%26rft_id%253Dinfo%253Apmid%252F26653857%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.7554/eLife.10850&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26653857&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3637.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-118-1" title="View reference  in text" id="ref-118">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.118" data-doi="10.1534/genetics.115.182923"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Voelkel-Meiman</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">S. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morehouse</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">MacQueen</span>  <span class="cit-name-given-names">A. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Synaptonemal Complex Proteins of Budding Yeast Define Reciprocal Roles in MutSgamma-Mediated Crossover Formation.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">203</span>: <span class="cit-fpage">1091</span>–<span class="cit-lpage">1103</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.115.182923</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.182923%26rft_id%253Dinfo%253Apmid%252F27184389%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIwMy8zLzEwOTEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzExLzM2MzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-119-1" title="View reference  in text" id="ref-119">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.119" data-doi="10.1371/journal.pgen.1002993"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Voelkel-Meiman</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moustafa</span>  <span class="cit-name-given-names">S. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lefrançois</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Villeneuve</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">MacQueen</span>  <span class="cit-name-given-names">A. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Full-Length Synaptonemal Complex Grows Continuously during Meiotic Prophase in Budding Yeast.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1002993</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1002993</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DVoelkel-Meiman%26rft.auinit1%253DK.%26rft.volume%253D8%26rft.issue%253D10%26rft.spage%253De1002993%26rft.epage%253De1002993%26rft.atitle%253DFull-Length%2BSynaptonemal%2BComplex%2BGrows%2BContinuously%2Bduring%2BMeiotic%2BProphase%2Bin%2BBudding%2BYeast.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002993%26rft_id%253Dinfo%253Apmid%252F23071451%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002993&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23071451&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3637.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-120-1" title="View reference  in text" id="ref-120">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.120" data-doi="10.1371/journal.pgen.1003837"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Voelkel-Meiman</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">L. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mukherjee</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Humphryes</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsubouchi</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">SUMO Localizes to the Central Element of Synaptonemal Complex and Is Required for the Full Synapsis of Meiotic Chromosomes in Budding Yeast.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1003837</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1003837</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003837%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003837%26rft_id%253Dinfo%253Apmid%252F24098146%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003837&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24098146&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3637.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-121-1" title="View reference  in text" id="ref-121">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.121" data-doi="10.1091/mbc.E03-07-0499"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wan</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Santos</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shokat</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hollingsworth</span>  <span class="cit-name-given-names">N. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Mek1 Kinase Activity Functions Downstream of RED1 in the Regulation of Meiotic Double Strand Break Repair in Budding Yeast.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">11</span>–<span class="cit-lpage">23</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1091/mbc.e03-07-0499</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.E03-07-0499%26rft_id%253Dinfo%253Apmid%252F14595109%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjc6IjE1LzEvMTEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzExLzM2MzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-122-1" title="View reference  in text" id="ref-122">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.122" data-doi="10.1101/gad.11.1.106"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weiner</span>  <span class="cit-name-given-names">B. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kleckner</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Meiotic cells monitor the status of the interhomolog recombination complex.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">106</span>–<span class="cit-lpage">118</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gad.11.1.106</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DXu%26rft.auinit1%253DL%26rft.volume%253D11%26rft.issue%253D1%26rft.spage%253D106%26rft.epage%253D118%26rft.atitle%253DMeiotic%2Bcells%2Bmonitor%2Bthe%2Bstatus%2Bof%2Bthe%2Binterhomolog%2Brecombination%2Bcomplex.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.11.1.106%26rft_id%253Dinfo%253Apmid%252F9000054%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6ODoiMTEvMS8xMDYiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzExLzM2MzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-123-1" title="View reference  in text" id="ref-123">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.123" data-doi="10.1016/j.cell.2012.03.023"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zakharyevich</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hunter</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Delineation of joint molecule resolution pathways in meiosis identifies a crossover-specific resolvase.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">149</span>: <span class="cit-fpage">334</span>–<span class="cit-lpage">347</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.cell.2012.03.023</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DZakharyevich%26rft.auinit1%253DK.%26rft.volume%253D149%26rft.issue%253D2%26rft.spage%253D334%26rft.epage%253D347%26rft.atitle%253DDelineation%2Bof%2Bjoint%2Bmolecule%2Bresolution%2Bpathways%2Bin%2Bmeiosis%2Bidentifies%2Ba%2Bcrossover-specific%2Bresolvase.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2012.03.023%26rft_id%253Dinfo%253Apmid%252F22500800%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2012.03.023&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22500800&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3637.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000302805600013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-124-1" title="View reference  in text" id="ref-124">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.124" data-doi="10.1146/annurev.genet.33.1.603"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zickler</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kleckner</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Meiotic Chromosomes: Integrating Structure and Function.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genet.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">603</span>–<span class="cit-lpage">754</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1146/annurev.genet.33.1.603</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenet%26rft.aulast%253DZickler%26rft.auinit1%253DD.%26rft.volume%253D33%26rft.issue%253D1%26rft.spage%253D603%26rft.epage%253D754%26rft.atitle%253DMEIOTIC%2BCHROMOSOMES%253A%2BIntegrating%2BStructure%2Band%2BFunction%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.genet.33.1.603%26rft_id%253Dinfo%253Apmid%252F10690419%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.genet.33.1.603&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10690419&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3637.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000084956600018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-125-1" title="View reference  in text" id="ref-125">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3637.125" data-doi="10.1101/cshperspect.a016626"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zickler</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kleckner</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Recombination, Pairing, and Synapsis of Homologs during Meiosis.</span> <abbr class="cit-jnl-abbrev">Cold Spring Harb. Perspect. Biol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">a016626</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/cshperspect.a016626</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCold%2BSpring%2BHarb.%2BPerspect.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fcshperspect.a016626%26rft_id%253Dinfo%253Apmid%252F25986558%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6ImNzaHBlcnNwZWN0IjtzOjU6InJlc2lkIjtzOjExOiI3LzYvYTAxNjYyNiI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzgvMTEvMzYzNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/8/11/3637.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/11/3627" title="Whole-Genome Multi-omic Study of Survival in Patients with Glioblastoma Multiforme" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/11/3661" title="A GAL80 Collection To Inhibit GAL4 Transgenes in Drosophila Olfactory Sensory Neurons" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/8/11"><span>Volume 8 Issue 11, November 2018</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/8/11.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/8/11.cover-source.jpg?itok=lFO8v6bg" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 8 (11)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/11/3637" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-hlqoiSH6Y6gD75tI54PGVnbinz7lRxP1VQB9Bl7O5bI" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/8/11/3637" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/8/11/3637" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/8/11/3637" class="active" data-icon-position="" data-hide-link-title="0">HO Endonuclease-Initiated Recombination in Yeast Meiosis Fails To Promote Homologous Centromere Pairing and Is Not Constrained To Utilize the Dmc1 Recombinase</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/491243" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-sSXsQLUQPpFd6RgTgbLLkVkJJLRylLtBk3n7WWBpD0Q" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/11/3637.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/8/11/3637" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-LOEJGxICselUycYVdbpKB5Mawi0zw6_-Ir__XxQZsPk" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/8/11/3637" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/11/3637.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="491243" id="node491243--2" data-pisa="ggg;8/11/3637" data-pisa-master="ggg;g3.118.200641" data-apath="/ggg/8/11/3637.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">HO Endonuclease-Initiated Recombination in Yeast Meiosis Fails To Promote Homologous Centromere Pairing and Is Not Constrained To Utilize the Dmc1 Recombinase</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Lina Yisehak</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-0919-7538" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Amy J. MacQueen</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3637-3659;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200641 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node491243--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Lina Yisehak </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Molecular Biology and Biochemistry, Wesleyan University, Middletown, CT</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lina%2BYisehak%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yisehak%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALina%2BYisehak%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Amy J. MacQueen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Molecular Biology and Biochemistry, Wesleyan University, Middletown, CT</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Amy%2BJ.%2BMacQueen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=MacQueen%20AJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAmy%2BJ.%2BMacQueen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-0919-7538" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Amy J. MacQueen</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:amacqueen@wesleyan.edu" class="" data-icon-position="" data-hide-link-title="0">amacqueen@wesleyan.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/8/11/3637" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/8/11/3637&amp;title=HO%20Endonuclease-Initiated%20Recombination%20in%20Yeast%20Meiosis%20Fails%20To%20Promote%20Homologous%20Centromere%20Pairing%20and%20Is%20Not%20Constrained%20To%20Utilize%20the%20Dmc1%20Recombinase" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/8/11/3637&amp;title=HO%20Endonuclease-Initiated%20Recombination%20in%20Yeast%20Meiosis%20Fails%20To%20Promote%20Homologous%20Centromere%20Pairing%20and%20Is%20Not%20Constrained%20To%20Utilize%20the%20Dmc1%20Recombinase" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/8/11/3637&amp;title=HO%20Endonuclease-Initiated%20Recombination%20in%20Yeast%20Meiosis%20Fails%20To%20Promote%20Homologous%20Centromere%20Pairing%20and%20Is%20Not%20Constrained%20To%20Utilize%20the%20Dmc1%20Recombinase" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/8/11/3637&amp;text=HO%20Endonuclease-Initiated%20Recombination%20in%20Yeast%20Meiosis%20Fails%20To%20Promote%20Homologous%20Centromere%20Pairing%20and%20Is%20Not%20Constrained%20To%20Utilize%20the%20Dmc1%20Recombinase" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/8/11/3637&amp;title=HO%20Endonuclease-Initiated%20Recombination%20in%20Yeast%20Meiosis%20Fails%20To%20Promote%20Homologous%20Centromere%20Pairing%20and%20Is%20Not%20Constrained%20To%20Utilize%20the%20Dmc1%20Recombinase" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/8/11/3637&amp;t=HO%20Endonuclease-Initiated%20Recombination%20in%20Yeast%20Meiosis%20Fails%20To%20Promote%20Homologous%20Centromere%20Pairing%20and%20Is%20Not%20Constrained%20To%20Utilize%20the%20Dmc1%20Recombinase" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/8/11/3637&amp;title=HO%20Endonuclease-Initiated%20Recombination%20in%20Yeast%20Meiosis%20Fails%20To%20Promote%20Homologous%20Centromere%20Pairing%20and%20Is%20Not%20Constrained%20To%20Utilize%20the%20Dmc1%20Recombinase" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/8/11/3637&amp;title=HO%20Endonuclease-Initiated%20Recombination%20in%20Yeast%20Meiosis%20Fails%20To%20Promote%20Homologous%20Centromere%20Pairing%20and%20Is%20Not%20Constrained%20To%20Utilize%20the%20Dmc1%20Recombinase" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/491243/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="491243" id="node491243--3" data-pisa="ggg;8/11/3637" data-pisa-master="ggg;g3.118.200641" data-apath="/ggg/8/11/3637.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">HO Endonuclease-Initiated Recombination in Yeast Meiosis Fails To Promote Homologous Centromere Pairing and Is Not Constrained To Utilize the Dmc1 Recombinase</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Lina Yisehak</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-0919-7538" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Amy J. MacQueen</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3637-3659;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200641 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node491243--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Lina Yisehak </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Molecular Biology and Biochemistry, Wesleyan University, Middletown, CT</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lina%2BYisehak%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yisehak%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALina%2BYisehak%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Amy J. MacQueen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Molecular Biology and Biochemistry, Wesleyan University, Middletown, CT</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Amy%2BJ.%2BMacQueen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=MacQueen%20AJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAmy%2BJ.%2BMacQueen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-0919-7538" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Amy J. MacQueen</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:amacqueen@wesleyan.edu" class="" data-icon-position="" data-hide-link-title="0">amacqueen@wesleyan.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3637&amp;rft.epage=3659&amp;rft.atitle=HO%20Endonuclease-Initiated%20Recombination%20in%20Yeast%20Meiosis%20Fails%20To%20Promote%20Homologous%20Centromere%20Pairing%20and%20Is%20Not%20Constrained%20To%20Utilize%20the%20Dmc1%20Recombinase&amp;rft.volume=8&amp;rft.issue=11&amp;rft.date=2018-11-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Yisehak%2C+Lina&amp;rft.au=MacQueen%2C+Amy+J."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/491243/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/491243/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/491243/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/491243/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/491243/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/491243/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/491243/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/491243/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/491243/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/491243/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/491243/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/491243/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/8/11/3637" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-19" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/8/11/3637.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/8/11/3637.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/8/11/3637.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/8/11/3637.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
