<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/8/11/3669" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/8/11/3669.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/8/11/3669.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/8/11/3669.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/8/11/3669/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/8/11/3669.atom" />
<meta name="HW.pisa" content="ggg;8/11/3669" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Discovery of Anthocyanin Acyltransferase1 (AAT1) in Maize Using Genotyping-by-Sequencing (GBS)" />
<meta name="DC.Identifier" content="10.1534/g3.118.200630" />
<meta name="DC.Date" content="2018-11-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2018 Paulsmeyer et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Relation" content="https://doi.org/10.25387/g3.7108748" />
<meta name="DC.Description" content="The reduced acylation phenotype describes the inability of certain accessions of maize ( Zea mays [ L .]) to produce significant amounts of acylated anthocyanins, which are typically the most abundant pigments. Acylated anthocyanins are important for their association with stability and are therefore important for the various industries using anthocyanins as natural colorants to replace synthetic dyes. Many anthocyanin acyltransferases have been characterized in other species; however, no anthocyanin acyltransferases have been characterized in maize. Therefore, a mapping population was developed from a cross between mutant stock 707G and wild-type acylation line B73 to identify the locus associated with the reduced acylation trait. High-performance liquid chromatography was used to assay the pigment content and composition of 129 F2 lines generated in the mapping population. Recessive alleles of Colorless1 , Colored1 , and the reduced acylation mutant all decreased anthocyanin content while Intensifier1 increased anthocyanin content in aleurone tissue. The association of increased proportions of acylation with increased anthocyanin content indicates acylation may be important for increasing the stability of anthocyanins in vivo . Genotyping-by-sequencing was used to create SNP markers to map the reduced acylation locus. In the QTL analysis, a segment of Chromosome 1 containing transferase family protein GRMZM2G387394 was found to be significant. A UniformMu Mu transposon knockout of GRMZM2G387394 demonstrated this gene has anthocyanidin malonyltransferase activity and will therefore be named Anthocyanin Acyltransferase1 ( AAT1 ). AAT1 is the first anthocyanin acyltransferase characterized in a monocot species and will increase our knowledge of all acyltransferase family members." />
<meta name="DC.Contributor" content="Michael N. Paulsmeyer" />
<meta name="DC.Contributor" content="Patrick J. Brown" />
<meta name="DC.Contributor" content="John A. Juvik" />
<meta name="article:published_time" content="2018-11-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Discovery of Anthocyanin Acyltransferase1 (AAT1) in Maize Using Genotyping-by-Sequencing (GBS)" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The reduced acylation phenotype describes the inability of certain accessions of maize (&lt;i&gt;Zea mays&lt;/i&gt; [&lt;i&gt;L&lt;/i&gt;.]) to produce significant amounts of acylated anthocyanins, which are typically the most abundant pigments. Acylated anthocyanins are important for their association with stability and are therefore important for the various industries using anthocyanins as natural colorants to replace synthetic dyes. Many anthocyanin acyltransferases have been characterized in other species; however, no anthocyanin acyltransferases have been characterized in maize. Therefore, a mapping population was developed from a cross between mutant stock 707G and wild-type acylation line B73 to identify the locus associated with the reduced acylation trait. High-performance liquid chromatography was used to assay the pigment content and composition of 129 F&lt;sub&gt;2&lt;/sub&gt; lines generated in the mapping population. Recessive alleles of &lt;i&gt;Colorless1&lt;/i&gt;, &lt;i&gt;Colored1&lt;/i&gt;, and the reduced acylation mutant all decreased anthocyanin content while &lt;i&gt;Intensifier1&lt;/i&gt; increased anthocyanin content in aleurone tissue. The association of increased proportions of acylation with increased anthocyanin content indicates acylation may be important for increasing the stability of anthocyanins &lt;i&gt;in vivo&lt;/i&gt;. Genotyping-by-sequencing was used to create SNP markers to map the reduced acylation locus. In the QTL analysis, a segment of Chromosome 1 containing transferase family protein GRMZM2G387394 was found to be significant. A UniformMu &lt;i&gt;Mu&lt;/i&gt; transposon knockout of GRMZM2G387394 demonstrated this gene has anthocyanidin malonyltransferase activity and will therefore be named &lt;i&gt;Anthocyanin Acyltransferase1&lt;/i&gt; (&lt;i&gt;AAT1&lt;/i&gt;). &lt;i&gt;AAT1&lt;/i&gt; is the first anthocyanin acyltransferase characterized in a monocot species and will increase our knowledge of all acyltransferase family members.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2018/11/01" />
<meta name="citation_mjid" content="ggg;8/11/3669" />
<meta name="citation_id" content="8/11/3669" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/8/11/3669" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/8/11/3669.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/8/11/3669.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/8/11/3669.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.118.200630" />
<meta name="citation_pmid" content="30257861" />
<meta name="citation_volume" content="8" />
<meta name="citation_issue" content="11" />
<meta name="citation_num_pages" content="10" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="3669" />
<meta name="citation_lastpage" content="3678" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Michael N. Paulsmeyer" />
<meta name="citation_author_institution" content="Department of Crop Sciences, University of Illinois at Urbana-Champaign, Urbana, Illinois, 61801" />
<meta name="citation_author" content="Patrick J. Brown" />
<meta name="citation_author_institution" content="Department of Plant Sciences, University of California at Davis, Davis, California, 95616" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-1332-711X" />
<meta name="citation_author" content="John A. Juvik" />
<meta name="citation_author_institution" content="Department of Crop Sciences, University of Illinois at Urbana-Champaign, Urbana, Illinois, 61801" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=T. Agarwal;citation_author=E. Grotewold;citation_author=A. I. Doseff;citation_author=J. Gray;citation_title=MYB31/MYB42 syntelogs exhibit divergent regulation of phenylpropanoid genes in maize, sorghum and rice.;citation_pages=28502;citation_volume=6;citation_year=2016;citation_doi=10.1038/srep28502" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=C. M. Andorf;citation_author=C. J. Lawrence;citation_author=L. C. Harper;citation_author=M. L. Schaeffer;citation_author=D. A. Campbell;citation_title=The Locus Lookup tool at MaizeGDB: identification of genomic regions in maize by integrating sequence information with physical and genetic maps.;citation_volume=26;citation_year=2010;citation_pmid=20124413;citation_doi=10.1093/bioinformatics/btp556" />
<meta name="citation_reference" content="citation_journal_title=Pol. J. Food Nutr. Sci.;citation_author=A. Bakowska-Barczak;citation_title=Acylated anthocyanins as stable, natural food colorants - a review.;citation_pages=107-116;citation_volume=14/15;citation_year=2005" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=P. J. Bradbury;citation_author=Z. Zhang;citation_author=D. E. Kroon;citation_author=T. M. Casstevens;citation_author=Y. Ramdoss;citation_title=TASSEL: software for association mapping of complex traits in diverse samples.;citation_volume=23;citation_year=2007;citation_pmid=17586829;citation_doi=10.1093/bioinformatics/btm308" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Hum. Genet.;citation_author=B. L. Browning;citation_author=S. R. Browning;citation_title=Genotype imputation with millions of reference samples.;citation_pages=116-126;citation_volume=98;citation_year=2016;citation_pmid=26748515;citation_doi=10.1016/j.ajhg.2015.11.020" />
<meta name="citation_reference" content="citation_journal_title=The Plant Cell Online;citation_journal_abbrev=Plant Cell;citation_author=F. A. Burr;citation_author=B. Burr;citation_author=B. E. Scheffler;citation_author=M. Blewitt;citation_author=U. Wienand;citation_title=The Maize Repressor-like Gene intensifier1 Shares Homology with the r1/b1 Multigene Family of Transcription Factors and Exhibits Missplicing;citation_pages=1249-1259;citation_volume=8;citation_year=1996;citation_issue=8;citation_pmid=8776895;citation_doi=10.1105/tpc.8.8.1249" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=Z. Chen;citation_author=B. Wang;citation_author=X. Dong;citation_author=H. Liu;citation_author=L. Ren;citation_title=An ultra-high density bin-map for rapid QTL mapping for tassel and ear architecture in a large F2 maize population.;citation_pages=433;citation_volume=15;citation_year=2014;citation_pmid=24898122;citation_doi=10.1186/1471-2164-15-433" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in plant biology;citation_journal_abbrev=Curr Opin Plant Biol;citation_author=J. C. D’Auria;citation_title=Acyltransferases in plants: a good time to be BAHD.;citation_pages=331-340;citation_volume=9;citation_year=2006;citation_issue=3;citation_pmid=16616872;citation_doi=10.1016/j.pbi.2006.03.016" />
<meta name="citation_reference" content="citation_title=Chromosome Structure and Function;citation_title=Molecular cloning of the maize R-nj allele by transposon tagging with Ac;citation_author=S. L. Dellaporta;citation_author=I. Greenblatt;citation_author=J. L. Kermicle;citation_author=J. B. Hicks;citation_author=S. R. Wessler;citation_pages=263-282;citation_year=1988" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=S. R. Eichten;citation_author=J. M. Foerster;citation_author=N. de Leon;citation_author=Y. Kai;citation_author=C.-T. Yeh;citation_title=B73-Mo17 near-isogenic lines demonstrate dispersed structural variation in maize.;citation_volume=156;citation_year=2011;citation_pmid=21705654;citation_doi=10.1104/pp.111.174748" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=R. J. Elshire;citation_author=J. C. Glaubitz;citation_author=Q. Sun;citation_author=J. A. Poland;citation_author=K. Kawamoto;citation_title=A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.;citation_pages=e19379-e19379;citation_volume=6;citation_year=2011;citation_issue=5;citation_pmid=21573248;citation_doi=10.1371/journal.pone.0019379" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_journal_abbrev=Evolution;citation_author=J. Felsenstein;citation_title=Confidence limits on phylogenies; an approach using the bootstrap;citation_pages=783-791;citation_volume=39;citation_year=1985;citation_issue=4;citation_pmid=7288891;citation_doi=10.2307/2408678;citation_georef=1996015960" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=C. A. Fragoso;citation_author=C. Heffelfinger;citation_author=H. Zhao;citation_author=S. L. Dellaporta;citation_title=Imputing Genotypes in Biallelic Populations from Low-Coverage Sequence Data.;citation_volume=202;citation_year=2016;citation_pmid=26715670;citation_doi=10.1534/genetics.115.182071" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=E. Grotewold;citation_author=B. J. Drummond;citation_author=B. Bowen;citation_author=T. Peterson;citation_title=The myb-homologous P gene controls phlobaphene pigmentation in maize floral organs by directly activating a flavonoid biosynthetic gene subset.;citation_pages=543-553;citation_volume=76;citation_year=1994;citation_issue=3;citation_pmid=8313474;citation_doi=10.1016/0092-8674(94)90117-1" />
<meta name="citation_reference" content="citation_journal_abbrev=Annu Rev Food Sci Technol;citation_author=J. He;citation_author=M. M. Giusti;citation_title=Anthocyanins: natural colorants with health-promoting properties.;citation_pages=163-187;citation_volume=1;citation_year=2010;citation_pmid=22129334;citation_doi=10.1146/annurev.food.080708.100754" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=C. Heffelfinger;citation_author=C. A. Fragoso;citation_author=M. A. Moreno;citation_author=J. D. Overton;citation_author=J. P. Mottinger;citation_title=Flexible and scalable genotyping-by-sequencing strategies for population studies.;citation_pages=1;citation_volume=15;citation_year=2014;citation_pmid=24382143;citation_doi=10.1186/1471-2164-15-349" />
<meta name="citation_reference" content="citation_journal_title=Recent Adv. Phytochem.;citation_author=N. G. Irani;citation_author=J. M. Hernandez;citation_author=E. Grotewold;citation_title=Chapter three: regulation of anthocyanin pigmentation.;citation_pages=59-78;citation_volume=37;citation_year=2003" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. L. Kermicle;citation_title=DEPENDENCE OF THE R-MOTTLED ALEURONE PHENOTYPE IN MAIZE ON MODE OF SEXUAL TRANSMISSION;citation_pages=69-85;citation_volume=66;citation_year=1970;citation_issue=1;citation_pmid=17248508" />
<meta name="citation_reference" content="citation_journal_title=Plant physiology and biochemistry :  PPB / SociÃ©tÃ© franÃ§aise de physiologie vÃ©gÃ©tale;citation_journal_abbrev=Plant Physiol Biochem;citation_author=D. H. Kim;citation_author=S. K. Kim;citation_author=J.-H. Kim;citation_author=B.-G. Kim;citation_author=J.-H. Ahn;citation_title=Molecular characterization of flavonoid malonyltransferase from Oryza sativa.;citation_pages=991-997;citation_volume=47;citation_year=2009;citation_issue=11-12;citation_pmid=19733090;citation_doi=10.1016/j.plaphy.2009.08.004" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=S. Kumar;citation_author=G. Stecher;citation_author=K. Tamura;citation_title=MEGA7: Molecular Evolutionary Genetics Analysis version 7.0 for bigger datasets.;citation_volume=33;citation_year=2016;citation_pmid=27004904;citation_doi=10.1093/molbev/msw054" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=B. Langmead;citation_author=S. L. Salzberg;citation_title=Fast gapped-read alignment with Bowtie 2.;citation_pages=357-359;citation_volume=9;citation_year=2012;citation_issue=4;citation_pmid=22388286;citation_doi=10.1038/nmeth.1923" />
<meta name="citation_reference" content="citation_journal_title=Plant molecular biology;citation_journal_abbrev=Plant Mol Biol;citation_author=M. Lee;citation_author=N. Sharopova;citation_author=W. D. Beavis;citation_author=D. Grant;citation_author=M. Katt;citation_title=Expanding the genetic map of maize with the intermated B73 x Mo17 (IBM) population.;citation_pages=453-461;citation_volume=48;citation_year=2002;citation_issue=5-6;citation_pmid=11999829;citation_doi=10.1023/A:1014893521186" />
<meta name="citation_reference" content="citation_author=X. Li;citation_author=Z. Zhou;citation_author=J. Ding;citation_author=Y. Wu;citation_author=B. Zhou;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Cell. Mol. Biol.;citation_author=D. R. McCarty;citation_author=A. M. Settles;citation_author=M. Suzuki;citation_author=B. C. Tan;citation_author=S. Latshaw;citation_title=Steady-state transposon mutagenesis in inbred maize. Plant J.;citation_pages=52-61;citation_volume=44;citation_year=2005" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=J. S. Papadopoulos;citation_author=R. Agarwala;citation_title=COBALT: constraint-based alignment tool for multiple protein sequences.;citation_volume=23;citation_year=2007;citation_pmid=17332019;citation_doi=10.1093/bioinformatics/btm076" />
<meta name="citation_reference" content="citation_journal_title=J. Agric. Food Chem.;citation_author=M. Paulsmeyer;citation_author=L. Chatham;citation_author=T. Becker;citation_author=M. West;citation_author=L. West;citation_title=Survey of anthocyanin composition and concentration in diverse maize germplasms.;citation_pages=4341-4350;citation_volume=65;citation_year=2017;citation_doi=10.1021/acs.jafc.7b00771" />
<meta name="citation_reference" content="citation_journal_title=The EMBO Journal;citation_journal_abbrev=EMBO Journal;citation_author=J. Paz-Ares;citation_author=D. Ghosal;citation_author=U. Wienand;citation_author=P. A. Peterson;citation_author=H. Saedler;citation_title=The regulatory c1 locus of Zea mays encodes a protein with homology to myb proto-oncogene products and with structural similarities to transcriptional activators.;citation_pages=3553-3558;citation_volume=6;citation_year=1987;citation_issue=12;citation_pmid=3428265" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=J. A. Poland;citation_author=P. J. Brown;citation_author=M. E. Sorrells;citation_author=J.-L. Jannink;citation_title=Development of high-density genetic maps for barley and wheat using a novel two-enzyme genotyping-by-sequencing approach.;citation_pages=e32253-e32253;citation_volume=7;citation_year=2012;citation_issue=2;citation_pmid=22389690;citation_doi=10.1371/journal.pone.0032253" />
<meta name="citation_reference" content="citation_title=R: a language and environment for statistical computing;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=N. Saitou;citation_author=M. Nei;citation_title=The neighbor-joining method: a new method for reconstructing phylogenetic trees;citation_pages=406-425;citation_volume=4;citation_year=1987;citation_issue=4;citation_pmid=3447015;citation_doi=10.1099/ijs.0.01472-0" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. Sharma;citation_author=M. Cortes-Cruz;citation_author=K. R. Ahern;citation_author=M. McMullen;citation_author=T. P. Brutnell;citation_title=Identification of the Pr1 gene product completes the anthocyanin biosynthesis pathway of maize.;citation_volume=188;citation_year=2011;citation_pmid=21385724;citation_doi=10.1534/genetics.110.126136" />
<meta name="citation_reference" content="citation_title=Recent Advances in Phytochemistry;citation_title=Evolution of acyltransferase genes: origin and diversification of the BAHD superfamily of acyltransferases involved in secondary metabolism;citation_author=B. St-Pierre;citation_author=V. De Luca;citation_pages=285-315;citation_year=2000" />
<meta name="citation_reference" content="citation_journal_title=Plant Sci.;citation_author=H. Suzuki;citation_author=T. Nakayama;citation_author=M. Yamaguchi;citation_author=T. Nishino;citation_title=cDNA cloning and characterization of two Dendranthema × morifolium anthocyanin malonyltransferases with different functional activities.;citation_pages=89-96;citation_volume=166;citation_year=2004;citation_doi=10.1016/j.plantsci.2003.08.010" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiology;citation_journal_abbrev=Plant Physiol.;citation_author=H. Suzuki;citation_author=T. Nakayama;citation_author=K. Yonekura-Sakakibara;citation_author=Y. Fukui;citation_author=N. Nakamura;citation_title=cDNA Cloning, Heterologous Expressions, and Functional Characterization of Malonyl-Coenzyme A:Anthocyanidin 3-O-Glucoside-6&quot;-O-Malonyltransferase from Dahlia Flowers;citation_pages=2142-2151;citation_volume=130;citation_year=2002;citation_issue=4;citation_pmid=12481098;citation_doi=10.1104/pp.010447" />
<meta name="citation_reference" content="citation_author=K. Swarts;citation_author=H. Li;citation_author=J. A. Romero Navarro;citation_author=D. An;citation_author=M. C. Romay;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=H. Unno;citation_author=F. Ichimaida;citation_author=H. Suzuki;citation_author=S. Takahashi;citation_author=Y. Tanaka;citation_title=Structural and mutational studies of anthocyanin malonyltransferases establish the features of BAHD enzyme catalysis.;citation_volume=282;citation_year=2007;citation_pmid=17383962;citation_doi=10.1074/jbc.M700638200" />
<meta name="citation_reference" content="citation_journal_title=Plant Foods Hum. Nutr.;citation_author=D. A. Urias-Lugo;citation_author=J. B. Heredia;citation_author=M. D. Muy-Rangel;citation_author=J. B. Valdez-Torres;citation_author=S. O. Serna-Saldívar;citation_title=Anthocyanins and Phenolic Acids of Hybrid and Native Blue Maize (Zea mays L.) Extracts and Their Antiproliferative Activity in Mammary (MCF7), Liver (HepG2), Colon (Caco2 and HT29) and Prostate (PC3) Cancer Cells.;citation_pages=193-199;citation_volume=70;citation_year=2015;citation_doi=10.1007/s11130-015-0479-4" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=K. Youens-Clark;citation_author=E. Buckler;citation_author=T. Casstevens;citation_author=C. Chen;citation_author=G. DeClerck;citation_title=Gramene database in 2010: updates and extensions.;citation_volume=39;citation_year=2011;citation_pmid=21076153;citation_doi=10.1093/nar/gkq1148" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=J. Zhao;citation_author=D. Huhman;citation_author=G. Shadle;citation_author=X.-Z. He;citation_author=L. W. Sumner;citation_title=MATE2 mediates vacuolar sequestration of flavonoid glycosides and glycoside malonates in Medicago truncatula.;citation_volume=23;citation_year=2011;citation_pmid=21467581;citation_doi=10.1105/tpc.110.080804" />
<meta name="citation_reference" content="citation_journal_title=Food Chem.;citation_author=C.-L. Zhao;citation_author=Y.-Q. Yu;citation_author=Z.-J. Chen;citation_author=G.-S. Wen;citation_author=F.-G. Wei;citation_title=Stability-increasing effects of anthocyanin glycosyl acylation.;citation_pages=119-128;citation_volume=214;citation_year=2017;citation_doi=10.1016/j.foodchem.2016.07.073" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=Z. Zhou;citation_author=C. Zhang;citation_author=Y. Zhou;citation_author=Z. Hao;citation_author=Z. Wang;citation_title=Genetic dissection of maize plant architecture with an ultra-high density bin map based on recombinant inbred lines.;citation_pages=1-15;citation_volume=17;citation_year=2016;citation_pmid=26818753;citation_doi=10.1186/s12864-015-2294-6" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Discovery of Anthocyanin Acyltransferase1 (AAT1) in Maize Using Genotyping-by-Sequencing (GBS)" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11.cover-source.jpg" />
<meta name="twitter:description" content="The reduced acylation phenotype describes the inability of certain accessions of maize ( Zea mays [ L .]) to produce significant amounts of acylated anthocyanins, which are typically the most abundant pigments. Acylated anthocyanins are important for their association with stability and are therefore important for the various industries using anthocyanins as natural colorants to replace synthetic dyes. Many anthocyanin acyltransferases have been characterized in other species; however, no anthocyanin acyltransferases have been characterized in maize. Therefore, a mapping population was developed from a cross between mutant stock 707G and wild-type acylation line B73 to identify the locus associated with the reduced acylation trait. High-performance liquid chromatography was used to assay the pigment content and composition of 129 F2 lines generated in the mapping population. Recessive alleles of Colorless1 , Colored1 , and the reduced acylation mutant all decreased anthocyanin content while Intensifier1 increased anthocyanin content in aleurone tissue. The association of increased proportions of acylation with increased anthocyanin content indicates acylation may be important for increasing the stability of anthocyanins in vivo . Genotyping-by-sequencing was used to create SNP markers to map the reduced acylation locus. In the QTL analysis, a segment of Chromosome 1 containing transferase family protein GRMZM2G387394 was found to be significant. A UniformMu Mu transposon knockout of GRMZM2G387394 demonstrated this gene has anthocyanidin malonyltransferase activity and will therefore be named Anthocyanin Acyltransferase1 ( AAT1 ). AAT1 is the first anthocyanin acyltransferase characterized in a monocot species and will increase our knowledge of all acyltransferase family members." />
<meta name="og-title" property="og:title" content="Discovery of Anthocyanin Acyltransferase1 (AAT1) in Maize Using Genotyping-by-Sequencing (GBS)" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/8/11/3669" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="The reduced acylation phenotype describes the inability of certain accessions of maize ( Zea mays [ L .]) to produce significant amounts of acylated anthocyanins, which are typically the most abundant pigments. Acylated anthocyanins are important for their association with stability and are therefore important for the various industries using anthocyanins as natural colorants to replace synthetic dyes. Many anthocyanin acyltransferases have been characterized in other species; however, no anthocyanin acyltransferases have been characterized in maize. Therefore, a mapping population was developed from a cross between mutant stock 707G and wild-type acylation line B73 to identify the locus associated with the reduced acylation trait. High-performance liquid chromatography was used to assay the pigment content and composition of 129 F2 lines generated in the mapping population. Recessive alleles of Colorless1 , Colored1 , and the reduced acylation mutant all decreased anthocyanin content while Intensifier1 increased anthocyanin content in aleurone tissue. The association of increased proportions of acylation with increased anthocyanin content indicates acylation may be important for increasing the stability of anthocyanins in vivo . Genotyping-by-sequencing was used to create SNP markers to map the reduced acylation locus. In the QTL analysis, a segment of Chromosome 1 containing transferase family protein GRMZM2G387394 was found to be significant. A UniformMu Mu transposon knockout of GRMZM2G387394 demonstrated this gene has anthocyanidin malonyltransferase activity and will therefore be named Anthocyanin Acyltransferase1 ( AAT1 ). AAT1 is the first anthocyanin acyltransferase characterized in a monocot species and will increase our knowledge of all acyltransferase family members." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11.cover-source.jpg" />
    <title>Discovery of Anthocyanin Acyltransferase1 (AAT1) in Maize Using Genotyping-by-Sequencing (GBS) | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"Cq4Ajui_ikuS8qEppzjyln4TFLGb95hEeBg-QLZcs6A","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"491225","apath":"\/ggg\/8\/11\/3669.atom","pisa":"ggg;8\/11\/3669","ac":{"\/ggg\/8\/11\/3669.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/8\/11\/3669.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;8\/11\/3669"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;8\/11\/3669"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/8\/11\/3669":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022hJBHnEr++1Dhc4ntn+\\\\\\\/WqDuXPbjHbJixbbrbJF2ncx+pPa5EBlgZ\\\\\\\/Kezh2UD4HJqpjRA992obP\\\\\\\/P3iAfAsbLT53KB\\\\\\\/W2B+lxXfTa0Y072NtzyunH6qBHPAf4aiAZkD+BDZTuS8cmd3oaLgGG3H17AufSAyS4WBLg+UXzZ34QrnZNOhxNkxY+o0E40Q5Tbbp2NappmIRmaDm5x4QF4K9Nckx1BIZcXO7mngzmLBqHR6pGE3WPD2KSWAgnd9DVAEibKHjZURNC8Zo76JdX4cXQAP+RcTvpHxTptrGkLfmHd2\\\\\\\/23zpfImsQrJVlr\\\\\\\/iTvTS+9h+NTNKQ6lx603J\\\\\\\/anbDaaWAP5IMnoUjC5+BWwRx8gHTKdUDJIsMBnTK25lpo1KBqigH1bvekZ\\\\\\\/svzUtE8oNSGcU\\\\\\\/sgjMkucUgljSF+ZP3LRGjGhKWwyImRhkdK+M2qIROmNejlSRunSPGGbyvbZ2xf3XhP4uQ1kNPLEVh3A9aSPGE\\\\\\\/\\\\\\\/ECER6QFTCOZukGhdhHwcm7vZ9yc32t7QT5nURYGpAsL7p3bXtm3POMxZTdcee5C48+cqVSkMzOevDFZAvpwhIgM3ujGZsfhoTQEjmVSWc3k56vtMUY8e\\\\\\\/Z+fCK057wiutB2dn5trmhE7aEwMwf5g0arfj1NgCS+dnORDqtz5tTKHNsj4helRZ59FUnGZf7CgU6k4ZOVH57aPBcXAbwt2xGdDc87mcA80s3VXGVa4KTa85b0ws\\\\\\\/4t3ODVsxBeHIMa1OiR1Kq1np\\\\\\\/dTdT\\\\\\\/qszRx8k5WPP0C5X\\\\\\\/l+uIvcTExavOc4Jmd5Do2otWevwkiQbaoycAutiljaRW\\\\\\\/Tf9okDFP+HoY5U\\\\\\\/2SzqFxuzsz9ZgHt78KOPeQMZ0UbVSVs3oXIrugAMbYv7rneNFusE256+px8t+ws4VWjO+S6ZEUvwBvdyDzXxKPf0AidpwaT5VDkwhK\\\\\\\/lh7xeQpXk8fGo8AToj4JotBjvyGTk9ZV+XjAn2ldCvhRPHcNRVxd4rQNseKSnt70f4UiAf\\\\\\\/WUTwU8Wr1sK6yNEmXUP7+RtPqNevTUDkhF+oyIL3H4rDNx5AZkfYiKJPOT7cBtMG3fis9GWcPbc3hbh\\\\\\\/ij4KIFkHcCmQJc6BkmXvaBrK+5+aEQC8tQFK5mTAsusd60bNNd831ZNS6jbwg0T6142SwhoC2yBF79DJR6\\\\\\\/F+r75pzkGuhusbhhD6Uvx9tYbe2zbOCBajG9GylGD1vXTsuNVEfA0K6l2HMMq\\\\\\\/0VsWnXrk5sJaa0xF9GQ\\\\\\\/vtISBAukzffWUASMD\\\u0022,\\\u0022iv\\\u0022:\\\u0022suNzteUbLHlioQ0FFN6sSw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00229814c0aa4d04f45421ee941cbe3b0d608f1af1f9901b5022c25b9e111b89cb29\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022btBLlhzz6nuI\\\\\\\/LwoZBr0dVhRJLC3GNmfzkzL8uieBOYxLOfF\\\\\\\/CjDmpA1S4HK4UOg6H9eJMOq4hjsQJrB7eJa1EI\\\\\\\/YOqu+SzdyMdsII2NIuQjhw\\\\\\\/0KLDj0UXWxRdd4onAa2nWU6ccOP\\\\\\\/kvrxnO3H35L\\\\\\\/no\\\\\\\/kb3NW68I76bGKonGXaGzmWGoTvCtd62KIkdJPXrDOastQ8ZIiwQniX8L4nis3gKzCmrG8jkxqC+HMIdHE+IllaeCMlLDZIZEK3IhZL+myC7p1iFauEHV7+sjhDNU5m6z4etrPIvh9sonEJHgX3xB0MheB7gDWgPgqGbqfrordRwWiSWVvAL6dkgzgPzWn92DvrYpk\\\\\\\/2j\\\\\\\/Vo2WKUYlXHdFTPRruyqvErqRBUaSh5Czb7oGC1CqPFbz8mWlMxfczMqZVrrt0YK8BV+dY9iEpmGjt7hFRUPVmNO9OF1aXnUaRlKd+ofdmEbMhPQEGRT0CmtwPaN\\\\\\\/nbCXTBJ4ETpO+Y64iGRqM65CohFXiLcM4fka9DgzCw+WsO\\\\\\\/zTA5YnaUuhxuPSaVaeXM5sAPZ08OAIp7kZWTPWTzXYeUXRxhG6V8eu9usHL5z5Pby\\\\\\\/hlHAYYsTatNl6X8Pa53LL9FLxCqkFcZoNkuKUpSEYWnvRGpJvqea4xeGRYAPCim4SeSV9RiLQ50tYdmSJuj5pJOm7y7knNaHMgWe35YF1mQRSQQ+zgRqiVvuoZpxJvNG6bsp4BHqOifk0c4iWRCnHtEGLpVWDjrtLUIVkuO7M+xtCWk59fI+iaDi1onk5P1t9nnz6H\\\\\\\/9yCm7vfeUTPEB\\\\\\\/df9ub1sJ9xplEnWORGgWi9XkOteyyoK1oNg+kTVhqS1JHFUNzZZ\\\\\\\/Y1zIqm0DoI+R5UQfD4MKDMWZ6qjNBJaJWGv760Ceh5bFhhP\\\\\\\/5LFy3FuE88Cr4ooiKKOaGOzKYxgOzKEsunS\\\\\\\/icoZ4N1iyIfzINTznRf8BZDggaIDABdCUMz4QoXycouL9SxElnYbNMsiFFxZGEzzidLQE0fKXAmaRSrdEs35plWktFISiNDFa5HNQicj\\\\\\\/cbEysE5xBP0p8Gem1FkU\\\\\\\/kz4qHUSrSa7SKYPDi7sJ37tBeQJuMmv\\\\\\\/PNKJnQpj\\\\\\\/FHgm35lJmEioQSD4oQxRTgN4F0du1FPfe0hP7qGLIxOe4EABUzZt2mxQwST5+U+zYyZSz18YjNl1oGHq1XVF2XCNc26sQoYvEFbkHtuJcV2aEPx4NAv7HElJcNvWueNMWRCdjVC3bUMO3bkEidsd9fh8GOrf13jSJgwhcNJqfXAIAfxKVRIs0vVPg0+jlpmPQXKCndbGXocMcDNOzrqZXGlU3SYKOn8t076Mq8UD7vafgWbkLoosmhhbvPBl4P3gdoRBIFpWrNYRihpGEeRaP+CLPwPbj6RKmR++Frv89KIA7pRQfQxX9529MGGS+avoSEfQ4AvD8Tujk9rvpeK2BEwPFmJPWsNqvXqoKx91+x9RXkgZVwhHJggSuj25G82BcGy4phczAuljVIjP6zafo23mjuXOI0c2j9l0PAck5TxkIIo\\\\\\\/nhDR+QuMRU4qJCkE2Ai86EFtLlhb3wfXBDfZ4GvIf3SgVZjRfToOdgeEbD3XlOjtoI5YvgWfolNyv7KdK7w6mcHRoqVLiFWUTiHcVNSkfpbgliGbR9\\\\\\\/r1AgLu1yf1MH2RLgEdiX3\\\\\\\/4hEYfRhGvpSxvrqQXx1RjZhxufbNkg8B8ER2kf2LvgOW7KtKpouuL\\\\\\\/9pPvfsZ4CXu\\\\\\\/S8V4pObcht1VA17nDH2uCP8FhacpT8IT2uRu2166\\\\\\\/vyoIXhIAbyEGuKU0geIwm1emFOgnK2TDZO+k2hvaVwd15ztt\\\\\\\/o84mPF4TQ+NX3d6o7V6XgCy6VGofLCiFD6LM3o6j6VItCRtByjfcM7\\\\\\\/KqG2CHoAImv\\\\\\\/QvjtDOtFT7W7QKl0Eblosi99si6M\\\\\\\/J7Gv1l3T2wU7sKsQLYxYSYqqdSHKCYaNHrTd8gMjbUON\\\\\\\/r2GBa57X8JHvjp6vxdZrQOnIoEo5RpnbRnWsVJbyGEpSfuQe+7QZrlWahzSqw\\\\\\\/mWmTXmgVPZH1xAgoEuqhX6NMuyYUFXNdUAvceEmOZgchgrB\\\\\\\/0m233EsMk97TH39MwGCUP+9SwICT6qo8MGhWnCOEHTgzUwZ0dUMSzP+A\\\\\\\/rZ83+aVMJ2whDTsQoD7lG7irdB\\\\\\\/ofyItfPLFiJjPjtuBC45dGPPBHwoVFZEgwfqTuKK+63lZDCHA4mNrD66sT7YOwN21e\\\\\\\/fcgCy707DskAFLO3tZCfZ7z7cuwpoNRxFAyznONHRrOjt\\\\\\\/1urayNZkR6JsD+42ebJZF1RrcRwbmMqsv5R67vLwm8XdJ9cNhlCev81iStD\\\\\\\/5Tfx28jpL9R369jelBwumZ2LkZ96H12YujUgL1JuW\\\\\\\/\\\\\\\/I+Olg77Fhj10U\\\\\\\/jOqWLHJuOjCQbLpZksYuuD\\\\\\\/xGN8P2u95yGnfIAqtUnAPpzZ1Ayre1iJ5e5GGO9eVqNWrEgJ35d3GYlyKd2GrgmqQtjkwSPBarkzs5qrIy1+6eF2\\\\\\\/zu1x8QzjZa2e2tXxOxoqdVhwU87wAc8QZ\\\\\\\/BydbOwJEvPHhcS\\\\\\\/tzVWBSIiKwwyBqVSiv9WZUgVlZQjtb0FYv9HnFMji\\\\\\\/hD\\\\\\\/UqHxO6GjY2RRQYLsz+CGqtmGz90Kgj4kEuBaciQr7IvnXLAOnZtTFYusDT8x7qVjScfqmfHztg5VR7fo6XJQ\\\\\\\/foqNH7I2zXIwq08S66K+ytJEOPjt0sERNT9htSxSMZnaUviIFVptIgDb+JA5DY0pP5SIvewN5P7gtDHb+5DX6UHQ+R2nMfZWXvSsFa0vDjawFQBA+ZEZ3mhkj7G2GNcr1kY0AHxxzL6d1zr4p3aNEVl+SNANJ1qLmUf7AYa2WnC\\\\\\\/tz13EJlIOxg7TqV\\\\\\\/9fd0o8vPnt3pR7v25LBZnNIG0RLEETfjYHjGpMq5ugzsG7qL1IvuWMMukmJeUtZD+omqH6tuazoo0VlPhhxY991Xgk5+Tkj05EKwCkVLAlLBWbjVKD7io5zOH6m0X7vHKv5a+M1HHsejBHv1kSM315yuoJYnSHUqo93dqQ7\\\\\\\/6Tu\\\\\\\/SW78+vX3mCqQ1lE3z00Kv2+2TFeIpRo0hr5IrtvEbqP49ZFSyJ4DIUs00e9QmIaKhcs4r4USfcrbir7g6ztZf4bwq8ZLz6yEutr6sWDskgwtznEo8ZusVi0TyL2vGUOJeQC0qVXdw82cYfwXHpk0ZubjWYbWQJrXux8HQUR2J6XSWEdIJvVktjDCBLabfGnCAFsXaMjsPOHp51XglG+xOB8GgWXpGQjRnojtIckVAJtSRGgiy00yxIb09Tiktq9WpY2F6YqYngdqwXiDES7w60BfGZGmJIT\\\\\\\/8zrY3iX+d6AQkhUSfgpv0tD5PP85M0j2TAgSSWrxm7aFiVHXakhbSaFHopoJZVkKWDjh+NGXwp9xxItyRDRscDTHLQjhb2MIuYwKb0mwyf5GORHGsENc10Z\\\\\\\/Bzaj\\\\\\\/OTzzAfVwivtUqMfIS3twP5I2qJn8WoxJkMLwMAeUK1Pm25cYzLW1drR5LwEwVJFsZvdPgoial2wGgRf+GZVsnPoegEdRHRXrs\\\\\\\/efzcdgNbm1jxraq8OT\\\\\\\/34\\\\\\\/U+j5LkWwFarTRnZXgv9xQDeY11H4ERHkkDPfNJbXwlPqrVndIKquHa9f0o3\\\\\\\/bRNREYNa3kfMeRMpOrblBWJ2QF9\\\\\\\/lXQ+m35dwtQwvaHZ+f0NtKPCKfHWlgeS3WqrYUDfOCK70T8AnE\\\\\\\/iCqSmmkTPXfQQHuQ5rOFsc6eUTbn\\\\\\\/gQGEduzJBjupVg3uJ0+im+amW8G8aWCW\\\\\\\/yyjXgB7Qtm0pJk3OrYWgmSDqPOdTSr2VarVrwdlVGNAO9tw4vCD1ZwnNQ64\\\\\\\/c45QVidQP3bDTeNTdtZpWACRqDq7CtBFgX7wk09tZJfHkfv1bMSOhhH0\\\\\\\/2x8SNnZJ9PXijB7Qrc022khgqKv8\\\\\\\/G52Cuazc0UcJJc6g+XY0qNGsUM8MLeYzqGmigRN9kVw3LBgh5rT\\\\\\\/3Wv8RBsRVgiG\\\\\\\/1MH4ie+IMAfJ0MtNAyKNy0yqN0QGML2uZI2E55DoT2WGMMvQ07FPotBGi5E\\\\\\\/JUaIH8l0b0e\\\\\\\/hsxyLqvKBMiuxITAK1zSpwGuIPKvcAdvJ+6bgEYB3TRJEjgzmOr1zGTCF7UB5myFgU\\\\\\\/bAFpXwORIdyaSdqQ+\\\\\\\/GZnCQYAsXm7JfY6TDoS0Pb+RaH1WMfuUxcBQ717xp\\\\\\\/FLQKTF35Wdl7\\\\\\\/OyfO1d5w6+PbGTdvoKEU3AX9ZIJQm+Gl5lUKNAPfx+BEGqM+bI4AywORP56k0wNNu5SnyQc3xU89Voyf+1ILAdWpr\\\\\\\/u0\\\\\\\/EMMungWMt4LeEJZeRo\\\\\\\/7HOfveKCYn2o3IvklUlZs0U7kgHmEqpps9OhO0QQN\\\\\\\/zmq9ZOBSec7MJ+Z1Mt5HP+n567bzApN9F8rGPJZXTTWlZlRcNjpwsT\\\\\\\/ANiZzx+AM6x1siJ7WNjy3zmRwmZaooAUhZHsOpRjMo4Ha6XDxJDgA7XxIQC2X\\\\\\\/nkkUwj0yia1Wl8KNYy20sRqBWl4Zf7hSs0bijfCJblDZZ++ngVoCJ7MyjAgtn1M4u9KorSKzubzg3qdPH9U\\\\\\\/3O7705GtaeNcz9RHbcUuZIjKOBpt57NeP1evsRWF6PY7mObvT9oxodz3KjfAxxp4MCy1ZMWidyYVDlEexf7v0gGFXoCoAS3qhb9TUDKO16yLVTWaAr8EHWlOIAW3jGi\\\\\\\/3WPh0qN9gD5tfBEevPugiuepPPN0OyNY0Ch0jot9pBWBoxZoDjCrqYIcJuZJqexLO0Zpg2lEqX08Shu+dQBC1Gbun5eKS6Mc\\\\\\\/lbfOmO4fXqSeeNY62bjtjChgm+r+dl5uOC7xABHfz81zkqHu7p8+x5HpHZy9m0a1zHou3ohurFfkFv\\\\\\\/6PseCFEAphRZy8jGYj+Ou06ss1wD5rd1IOeJOLSQ9LH9Xx3Nqqq\\\\\\\/1PUgM+g2CVRfFxCf\\\\\\\/u+zvpEEeBeV\\\\\\\/exQIz4YFEMoZnsAUlYb1y1eDucc1FruB5d06YWUrOPQJuyqinD\\\\\\\/NIsOMM3Crm2v\\\\\\\/gNS43olSgV+svLOrXoBriQTtIESi0fVVsjA6lkDLD2toJQvOEnZ05bLFge0t7Y41GZ+skgia9hXcxILpmutxCVu7Zf9b0JDRDlQzOY9A8sJLfFdQeLo\\\\\\\/QTzBd5V+e1C4+dq7Qkz7XpW5PaubYEAE\\\\\\\/d6zqxr\\\\\\\/fND2LuNRV+DzHU806g\\\\\\\/WPPgjY1h3ABQJnABI9sEoZ6ZPajX\\\\\\\/4ZgRXlRsEgjuk0+IJAeIkKtUaMSCOiAPy2W5I9bdY1a+dnsdSfg6Iv\\\\\\\/bM6iOffMrt5xa+efcBqY2w\\\\\\\/duXp3Bt4BAV27VesePBdCZW9nUa6Ge0Zr+CzxkZ4DVybov54aEBT5m4S7i38fFd+J3xoaT4OnRQjqsQi4mO6HOuztpU4NIlXM9Tg9qnbR\\\\\\\/8KBqF9Rc\\\\\\\/3MJUOxKb\\\\\\\/ubhZeoKSuPCubYCOXWk4cVlnkTdwST1fNOBHaV6hmHZqG5FtkbWYESH89Gibxb3cXy8tmMvo4L9pYX885Ml5LlKA0mO0OXnBOvnWcenngN3772sHPxsHCdbXk33G5rJOUgtXSAE0o0piYqCCcMXEsYYiMscnIUIFf8142DCHC1DjuA+nsNCIN4Q1XO5cCWRu0ZlYMK\\\\\\\/baCqUxFArEs7b9O6yMaLQfdgshTXF0g66M1PBf\\\\\\\/23SoyrreHhJRozcuBlW\\\\\\\/aEqm6EpdbXMuL5bV02noKEsvP2TtcCaco0430M855BR1rmsLGoIZrDaOKCnF3tK+KJhc6Eat17m\\\\\\\/KVkCKdoBReCRYI3YyAoRy3mC00H9jkgwHsi7t3LbQZcAoEkB\\\\\\\/SYdK\\\\\\\/VvrK+7FLepvlJWndNGMALIyHoBP+3vcPfEaKhX\\\\\\\/ypy1u3p7zVE1G3Fk3JLNIB6CQyI0FUoRRk9wrcSCkqtV6bsQr6Mw\\\\\\\/ozu8HAET1mGv1wTL6PzjdBXx\\\\\\\/Aw3qLatYu3EhF7xrePQLev2VBEq\\\\\\\/ql+yCakKXVPD0STW\\\\\\\/c0VpeA1U2Z8YAuiK+A651lhvrtnKaeKORvpI3rcXhWwPsSLD2jFygSnbl+A8MMH9PauivcsoQYNjbxsUofl7c8MUINmanDMuRZBbt1tuu4zm6eC4MfkTsI2dsd+3670CdbW8z6S65TTkt4Rlxlkr21PR5Vl6ZVkv\\\\\\\/JwygtlTpVfZuf55UhBHR3dcsWs73U756eviyr4iU2OzUFi4DGan0d39dZjg21hvQsRZngI6xBRv++XKeds+Lx3+RjEAOQxGXveIdlh5xwR7wIcViLCSjR0i+HTCon14u3BPhHeDISldrCZNwb9zoUVdRYoW4mW4Fcrq9VzjiTLTUAoKTnxH6UfwOeDl8rrijRlAZ76ZJe4J+7F+LAjFhllcDcXOOFKJSKTKHVz\\\\\\\/9CjKY1n5U9PpCp1BwthUUQiCDvYMr0vn0J2pfLMXDcIwnQjM+PVYfCeWPd\\\\\\\/f+Yq3G0\\\\\\\/7GDTKHg8emE7ENnAoinoL6+qMNE\\\\\\\/1XcBFQbHar7OCcIxxzctNYw7fV+iTkW8niFlzUYgqoKSsbGfwiOqhwMiuvzwnSP86\\\\\\\/2Zw\\\\\\\/MAK\\\\\\\/nUAZOGWAwKbh471YDccqELkURsgp5Yo\\\\\\\/ZSCWDLwvzqKgSzpDxlqyH+6GKscDKA9zOYR7XpmqiOUWV7g+3NdCTlkQSNR2WRYjF3A1gd9hbaQifkhN2srhy1EvHoztC15\\\\\\\/z1rjYZ57cOVki92Empr6bde0\\\\\\\/fTYRTH7y\\\\\\\/bnBf4+csUYRDPingbseioETSxDoDNYOcPfuVA\\\\\\\/69qbDc3gf0mcMSwS7v4Pmc4g0cGrN1XuHl7GB6LC94oT9JNuFj9Bymv2mQbdE96NWMXV3tkQvqHClMvNBplt+Fc\\\\\\\/xn81sG5u+3gY3YPojKUskH1P7hLS67qZwY5roFfeg1K1bJoj+LyZCzXAesv+wyFdj3XLUpZnB7M1tUMWWxWhnkpHR4vta6cl+c7obKSQTl8RMxANmxOKq9p3y\\\\\\\/bCRjUI3lO5m2fzcCqvZKjlwA8pAvcolb6mH4WC+Lf3RMs5AASP\\\\\\\/GmAGxH8qb7aNb4tehOKPp5FMiVchYIZx+xbXMN1uaQx8pzsQKwu8XvPKYQqlTqPYXT44eJ\\\\\\\/OZM2+yCdx2OhiykN2iQlEep0hgdEFEO7AK0cc5d96HnFkuC\\\\\\\/BQKRmULJe6wS3WOj6UO6\\\\\\\/BdXZWe6s6EeV0x6Rp\\\\\\\/TVsK7+JVm36f+f+Ih4S4gfW1scqIPwVMhKdOY3YQ+pu3kUSKBchrUkIuNF4xzERxIerdi6CCc8dJ01grNVxBeZ77jNPm\\\\\\\/BjpTbo2o8IKfbEB4B6cxtyPdmR2hFrtMxtgv8rZoDIoDMn122ZKHOLa+7fUgYHSQY2A863ubUmMKwAjyGu2n5FVORX\\\\\\\/FgzZOkrNct60Qq75VcV2IxkAxFVhbQkolKCzM9643JWvUD\\\\\\\/ikr3eYNcm3DOsacw3D8T+mwq3byTcEuzJPARFerjrAR0Nfgi2OS8C31QX1Ml4TTkPXXAwjkNvu1YrSYlmxENNdhRv+xoXFHfwv7erjGS1sO3o2K7lpo5w4pkG+VQSSaq2w4R1yOxJ\\\\\\\/sqCQw5aSNtOlHDYF5vmbgSsBY7vRtPqeRj9oJ9ut9J4B+Q3eHey8085nxIKtd966CuwjHdKq7NZ0gnMFjwU4zZ4NM0rVTDXHRvXR4sbAk0qDfvEmfttw1yp+z7Ym2Zdan+5i0vFuhOGKDG1iZKM+pVJqi9CHL5gpmPwwUG\\\\\\\/xXoVdlw9vilK2dfNjFgXmbpTSqDz4+HBy3nqUfYq+jdiMv0xpjlL68\\\\\\\/ZCecW6CipEltKJYIhqEnGMGOXFO\\\\\\\/ku4bqGcbQeVtBrIe\\\\\\\/p9n1tvIh+ZMyrC0JWHOc\\\\\\\/e+PNZh8kSJIlmci+z6HB\\\\\\\/YPEy7OcbxXxiV3VVpmSPADZcz9pD0a7oXsx+WtoLZn7b8cpuVRrg3bf+6J8JkjSd7\\\\\\\/zVReJ\\\\\\\/fgmXnh0cmbFAqQk1kjqG\\\\\\\/vHluqs2NlvCBSipnipZA+QtxWFBAZNIVWh7U8gzoezhDvjmWTk\\\\\\\/2WzOINkKugzj30JUBuB+l51vkFn0b6pAhO2Yd8gntw5JGJJfD7WUGjEbzGNrrfl6oxl+xZDF2QAVhqGv8SyPqi1wNbbNV21brkkHFm8MDPIpnpARxC7XUCqbz8M0BcqxbvV1NyvvOdp9blEpo6tqcrdkZHgFqwua9pNxowhqgD0NKBxbYR5Ys4Wepli+N714v6O0j1OXQGakXTilGgQ506O2KpywaMY1NWXpwpYBBOXhvSSFFff\\\\\\\/\\\\\\\/yBoSD\\\\\\\/1VvXEeJE0q+jwUlxSqgn6OeVy0KMEKkJXuSFORd1kreTaausPdJq8G5hE87zT3f9nIt\\\\\\\/qLfwVIN1xETCfAT7z3cn299sYE+PIYIyctWmx1jZESPowEfu14guJZEgA2TAvMvnL3H1B7Z5cr3vE0VrZux8uPpStJaPvMrU889ST0v2m7DZEBGkx\\\\\\\/78Jua5L5iPXa3c8N0DBCtv4qdwBzFiDsYRwvK72zIRDQDn4TQwvA4sb+SkO0IJh0qQ95YoeDUKK8CAj4YuEc4t1XjeKug1HbXKrcQScHnELCIXaCrLL5qpGlvypiGuyxFkWS1Jl\\\\\\\/5puuCVJtRKa+HQQfql7oIj5YT7WinJm3y7+vhEeABW4wJ9+Uof+FGbR8zrLmLCiEWlCrqjkaLgBwVvZO77H+vklUAXC7XYYre1X0lhLCj12oebhMGqMBTH7jKHllmKd+ib96rxL3qCi4gWx1aUhY+73s0uaGifDSXW+xMlhDBMEhxEyIJWYAyDm++aPXJCSfdtJhXy49K2Pkmi0t1II6JlQ5I9wAPy6ZCqRcH4k2+K9zUVQII88difWLsbjWBhFQJ4MOse3cCeX9gP0c+PfOyIBzwoaaYJZ0OCDt8EY9JotXbPVvRzMpwr5qpMV+nDLJWQmCL0dmxqgEl28doGqBvI+veYhrv\\\\\\\/t8Q4YvjgoEdNJXbCyxro4b8v7NpFAmoDS\\\\\\\/9Dz9tS0+fhlBvwZ9V1Uw5r4fnLB+ewtyhasnvRVXBOVx9RGRxb5gAXFa4QA9GxHapS8wCUUAP2Q1pnXaDOvPVh2CpV6mCqci97W4\\\\\\\/56ZZ9xyWnSxjp6hdvJGOfiB9F7F3mT2WnHGlNVk1SOb\\\\\\\/5bhKBVFoeQ4n85TRHgMvr1kqE\\\\\\\/xQ9wGYFi2q\\\\\\\/zPjUysmyNwS1HlcmVk2rU73AEo3oVuehwOFr4Us5tXELOjTOfrQBqv3EWJPw1+ZVWVfmtpT00SxSxqd9O5lKQoFQzkMRavWhxhKcLyhmJh\\\\\\\/Tle9bvJ3+mtPmT72zLFC3LBj5zJ+5JV3zzpxvkffY1kXzBKGi371UsrAyXGf953zfGWBURf\\\\\\\/ZrQFHjVroDm4cx+VXT1h8ae1l0erutTQtReztYP+uNWdb1rEv1\\\\\\\/f7PXYt9ndEp2BV\\\\\\\/WQLh604bAEfj1Fiwl61LrMGsUGkMg0T3OSfj64MPyLHDAwaskgJ+ST5EbekKme7EgjoUkxkMm0yXjnsgVeX247bnjThLIp+L+d9Po0wc9+thbROCF0mZV2Y2vqnYiS91suQ2b33ukd4XG8P5FkB6i4N94wqDwc4H+uCWMtAoSCwyx+6VNSghI6\\\\\\\/hCWar8wfWvda126i0FAirXXmQ0Mz6G6nymIlgXF82nK8maEUADmI513LTbjWeT+JsP\\\\\\\/MPGYSELUrzP\\\\\\\/rZ1y7P4hMPCFud2beS38\\\\\\\/XsxLrKCEB\\\\\\\/4EMtUqLRsxHgUVI1vI+nLIsf2YiOk3xcJEq\\\\\\\/Sll1znJ8t4c4uqonLzJcWQQOH9fe4rO6ehvEBOYwH0RcYEYUm9Z4yx5fikkl1Qn7pKTKQhACuUqktYjg06VAWFo5PwN3ip35Aq5sqcVnrX2wIzwjp7kBwX+kKV23VV\\\\\\\/vW5sfnM+ryqLBfRwhFVrNNI29a7WbQzz+5D+2v5uQRuuel0jZzmYc9iISxr7Z+CwQOpfe1GVs8txTG9Ua9Xn92bZNRW6n\\\\\\\/G+CkyNnDaRzNmzyN3P2whnj4NIIF4BBSbCmQhaX7JOjJkggYqcnjt6g9lg+cTPyaZDiUex17JK6tq0CtapwBBdqibFxnycc32VUTzQlv+1YVxcT06z0UxGD5BitrN16KoybgGEUMaGJAd5dZELWNaWROJ477okN\\\\\\\/YtCiY3LHkTNfRdlWBB40j1Ulk2gqtJWqy8nYGNy\\\\\\\/\\\\\\\/AugcEluqsSwsD+zQwR5fXu7RqSNE5Xy7bApMf+kjc1+i9M07JU2YB20OAruQixt5OzNuzxRFzE11b28B8Hh1HLH\\\\\\\/H1jsWoYHyD3UN\\\\\\\/Tpl0aX\\\\\\\/B\\\\\\\/QtkIRKKsOQ0Y3Fp\\\\\\\/rkDs1OSOY9XoRu9g6Vxmbafg12p3lvjsQrSrQHmALlEM\\\\\\\/msREi7HOmrTheYQA9uOd5qBYZF7i8XpA7jmgu0k4WDjEbDSNhsZTJZ9Kb0WQ88eTt+tp7uPvCDrT3iAqrl9qgQc+Ulni\\\\\\\/19Wre0qwNGUuOff2fMfKoD4PIU9JddaiH23TN3wxXQL79cBNGqIBs7GzZD5TBWcFtneC\\\\\\\/WxGCOAYEHabUkxJABqJwxb6xd8gRA6RHhsYfmt6gB56PLzoy2Ok0h1Z9EyBq9qu40VqGzxhiEagpgkD83n7BfCnKM9GUlJTuxcKKTyfy8qfxYMZ\\\\\\\/eOAPspEHa22FBc6feN7Edj6e1MYtnZHveZNccDqqwjCia882v7dAub6DVNKYNVkvyvBsJQImiw4kj01lZg543mgqiMsFTXIun7hV9jo04Jyv9iImNFb7HJRdS3xFzggyGIZDQIzDPq037aeCbzvXDYYvAL5UVkVbZvSFGVXuto75oOX\\\\\\\/B7JwLT42GNWNGwicMibbCXAnpAgz6ylrkEXRR3PFh0KlVtEsgyYd7XuTRgFg5m6Mql5OrvGe9e\\\\\\\/K\\\\\\\/nWSaSCTVZZRMTA18PG9fs+8sxcnMIgpCSXSAhfMqSfgZcacWT9mPazjTQQyDBnVkWvh3JwFTSFoHUOzzIMKV9Tn3MLns6+aNHoIo2uf5Gh2fRd66jKwBleXWcaDq\\\\\\\/NgDuWYWffNzp+lT2TQq9RNB\\\\\\\/hwU6pnsV757DihfbXsGGMYNFzmnGYzGxPzqisRr3WTtUP0QAyPRcmzVDdaDk6YL6uCfkc9FcdwQ7A7tBEFjHSDL4N+h6R0\\\\\\\/TEk\\\\\\\/U8ADwxQ3xoMiRsL0Xzv710XrL+d3S2W7tvb9I3+e6OuqiLXAdHyRVy1Jm83p80Llf3q\\\\\\\/dEiryvBFBZXRsA1veUSpIkbbmQHzNe5hI0FAd4RaiAWqzkUPAr0Wzu9iAjDMcDtdNb7VFFAB6xkbalPHV+emq1wPaj0gVqq0GF\\\\\\\/cn44Kc03ZE8qTBUJH25SPJRPH3cBeDZ17tVv312abxz0mKSkY0MtjzSamFjiL3xH7c4KX0KoBawphEdk8hvTy0SCBeNtRi968S0xj7jgc9DpGKe9mlZ4RPc51o5uUJf\\\\\\\/tDUCG\\\\\\\/srQDoMwEyNTFdHVOX44ja9hyWaOMAX\\\\\\\/oJzyiZh2jJyUMohcCR\\\\\\\/HRVAePrrCB5nPK57203+UCQxCYqJ2sW2E3xG5YkSbkHPBVg3YCPZdNXUg6jKNO1c8IyelM1GR+bc4D2AzFsBNPXggAGQWmnIhLYti2d9DAr3jLbj2Z0r7H0++bPOpBhcbQDgx6iiTnv9aYHpBQtbu9sh5WZpauPnJfxlgQnPFddY8k94u8\\\\\\\/HYjjhzA5L+opH1hZb9QE0vk\\\\\\\/urPbEypiYswfDsVGDUZ2xvlAkdC+Jyefip\\\\\\\/sSq1mfdAkWYJB1UgOKdMqH22wnKW8WOurP9Y8av+OFIO9c+8q0mq2Y2N5L5heYdVGxmXDpsCfkc8BQhhNr0sTNzLStGTLy\\\\\\\/d6ny79XeX72sVyGTnmBw3jRHGTgWZSYn\\\\\\\/5u+BVi\\\\\\\/q8p3557B8IMUGAH+mCAPP0Vd+6NomCIWKRJxe\\\\\\\/T\\\\\\\/3LW+XuT8KZKUTOPkAMMA0QsAHxs+oWarmpK4\\\\\\\/AvqsQuGWskXYZI9SE0wKiRyMqD71xJb5udljnWKP6xtvCkm0JbrvRxmILUxPUrVPVVmXuPpC1VITcUAG+75S3oOPXO6NXdn8vhN\\\\\\\/a1PGrayuBoCWroKx5S8vz2zG8U2\\\\\\\/ncCE0eP7Ya9KvnnLz91M86UFDqEll4ILzH0BanTdqeME2zOp0bJIxJ8i0RfXX57WKQaFR37T2Y8rLP\\\\\\\/Ii65XbbdcOCsNbrnfPNUkEI91z3v3RXyeJGVtmAmTQkgJGemovufu4jgqM3dFoxa4xizyLWHfMmouTquvie62RvQ2u3F+4pntYTsbzrSBT5WvJJWsn6+0HWOpKVXb7bwdWm4yByYE2na8jP\\\\\\\/KmoZPNDNrEJN1m33JO8232yuwXNlkERID+dU5Pl2nbfvwKh84VFDfxpngeQ1ihiGbu1Ka33k6f2RmkM1jsYvYgTmONVvPRNWtBuflo9nENH4qz3ZxfGS1WTDqCczbbmIN0SxTnfNOe4tEMkE\\\\\\\/0Bo1GFRcSz1a+hpf5IA6OXWyziuqji2mUbJszqET4tJ3obyrGwb2dZkA\\\\\\\/O5FJ8HM\\\\\\\/RalKZLX3D1u7g2nY7kRKX9swHvyOuPIxM7soV4LHsveY0kCx\\\\\\\/L2t\\\\\\\/mXPft9JKe3yKOEhUcamhARxVlO0vESgt7G4HlAhCDW60ixz3ZLYQukoDGjAzbULfhdd\\\\\\\/oDe5F+ZPpNVVS6ofxe7sdJCTni8UDx0WYnjACHHg3aq6hw+JcwaiAGFl1mwOu1RtwUp8uGYT6aMFxnt3TrtA6odEOY6FWl6vxDPYVAnEerZrHiH917YyfVaWMcDoC7GUHV2YI9+cRZ19lkqv0b8eZQbtlhsfkKWEhhksK9gzE\\\\\\\/nbRrEO1kPbUBHn0tYCl+ierLDkzw6IaT5f0HIPEPlakVj4wACsWSHAKmv4HfoRJK0uFFgq0NaYHc22HnxISnZVvkjg7nZfhyBNAepxF34uueU+QzMdlYaxJ2QZ5Etn+mmgxAYwtv3jJxoSSc+4IhnIDt0PC7cw2R8sYGm4dmHncoUKak6hoHGZ3OVuV9pQuxZR5fhEY1rZDm4Gt1oMxXTe5N5aQqx9djcDZ4xiptgk1pt3PruUwb3YKJNcSv9baGAveYJkLbMIOB6hPjxqzCjyMzjZfnZg8\\\\\\\/IeLT\\\\\\\/tNXs0Zqh7QGTvbvOiwk3b\\\\\\\/Xw6CV+IaSV2b\\\\\\\/lx0t3iF\\\\\\\/uncQhIK0x\\\\\\\/kK6ZkWZNKMOVHRY9aBdUxuPOId3\\\\\\\/IE1nV7l5KeetbPtSLiXRwbfOpWlImID9U7Np33bTOKGT7OaeXrWkj8M0ykXSMcSJrHK8huzFrSfi4Nkve913A\\\\\\\/IUo\\\\\\\/+dCR1VIuSC9gdzkeCtNDUJYdbLI2tMf4ehj6DRWbq9AjkDStoopkBKXtmjEUwGdpYdhF5vVl0\\\\\\\/M8\\\\\\\/j9Kj7nWkBsyFrH6UpzcLmmapAYKuuy3oywApLYP5zEjctKMTWdch3N\\\\\\\/ZNxRDqZBGmHT02sa+0bM0Ps7uKPlmctxdqTAneD4zZb8cYPrcyGi9bz3AId1qHdfhiNTHqibBXKVw6TN0H8b2+8Nym0g3sK1t9FgfjMu9cmdlBGqwMq8H2dZzqoKAzc0kVXCtOCp865emXdfkBL2yzM6VmpcsAP8CVtei0wSfQZMdvRwkKEJHoyZpaZd3bZ7A\\\\\\\/Q+G1IpecsvIk+hNe4TnBwfPNec0gBm68TFPoUWJA0mbS2rJLcxGiPXDQ3GWsMkvL2xUMOtTh7c79Bfm4t1JvYAdgjcZLlCS3\\\\\\\/fOA8F3Xvs0dc4gOVjwjpnSY+TmXyJiAR2dJ81IMVJC98+2iFgGrPzRwwn+LsqXwmPZGF97H3gq2Pefz3TCZqJGEvB3UXtbxUjy3BzMD0NJ6gii8mB1uqueRrfFoklrqrhTDvYy4IDainGiJOIvEcugxeKkkFUeFOKwX3A0\\\\\\\/9zu1uSxV\\\\\\\/AMeVBlT0gG6kmNefveLICeO9bCsvQOBkK59hRcK7li7JT+SocBJj5UqIgW5Y4xWlWwePyCpwLe3qp9qFgtqbXbAQ90p84zsw\\\\\\\/2W\\\\\\\/7uRy6NUgT+0R5cMKj\\\\\\\/m8Yl2d2rvS2v4Y+TSMM5\\\\\\\/JJdmhR5UZPIF1S53BDlV5m+4PXRwhC977ByWH3b7wjePlBGIP\\\\\\\/MKjEpmO4BnnIhOill+vSgK6h\\\\\\\/O1G3yfTy5noP1KmgEZzqqmVkyvPsovC0weEsfrSK3PkfR8vUHFMhhhe17P\\\\\\\/GwonDoolsjT2VwLsSX3RkTArcthSwEb10CO738I7u7ksiPELv8eQsuwTR3y5cAtWPUpaXHyVffm6QADHkz\\\\\\\/HawK1T4rUQX4IwdcxtC3f3LUGc52ZgLN00+T1xu3pP1xdNlg6v2jvIKr3DCe8Y2xi17H7qYrKL8qKpQRWrP20n7eNTo9U9Wu2WK8SgbJEqGHnKeJEIH6ndZR4pZHPPy1Wlm\\\\\\\/bR7g1HQ0rWyaaxG\\\\\\\/0kiYiCATDE4H33wQNrvBvyZTtMMwJqiwuBrSpX+qlUOWy2Y6DxqQIubC5nie+GnQmjvSXS9GVbYl\\\\\\\/J8H9SgA8YzP4W2CYxcecxVeneGWO72YvMQm5cdb+4OHrebYxFj38H8fs7WUeiTxm15nMOYQXlGpJ9i+bDCfT\\\\\\\/EIbRFzIIxYBn8kQ8GLJNYdNJ9qik6A2yPI2\\\\\\\/UJo6flifuZ9AaqXufgWtKmn2+rrlyRr4DQeRy+z\\\\\\\/EfwcV5ukGZ\\\\\\\/pZ4diKf+YloOT\\\\\\\/glYXEGIQ8YRUrt2CPvn3hpY2Ms20WywMppm5xBGDpssB5r5F5DVq8UHplknPGUb2XcbcMpGtJBmTZsxXBmF1zVjQq0G2ayznQf5MXkuQolR\\\\\\\/qWCobEi0aLFlSttblX4Xvm3BITGNqzAN2ZX7hodSA6bGjGNtydPGgRDB3biAmCL8MpUEU5x8jMnfDyuklYa0wMmb9rlQt1Drc94XEGr4zVcck\\\\\\\/0+QgvExWeMSqJ+1KURdkt6B2FB2Ab2PaomOdcLHOguUJWXnbSr9k4oVdRvuAxfOc54cTr20cOc\\\\\\\/irNPDUDAPBQUQ8ewzqvxZNDNFR9I7Qnvq50ZDwd5H6Hz9XbVV+tyI+trMw97nsrCDI2Cm57IRKQ3xcCU76kgStAayxOIZ36nXBLM6Mk9pU\\\\\\\/74kdtkw1Ym7TenLUKgOEYWNiDU2MnlebjK8s5TiNbslNbrxX8XDmiP2pOtZe7n1\\\\\\\/Qg3fJ89YAlRskldqXzjr7ikOrtvDQtPYYaJLHDLdyVkwC8ICdIG5Lmhl\\\\\\\/MFLf+hhtP\\\\\\\/sOE03YtdpUw+Fbd7yRzGpoeEGS0hE7Givmt+43Zw\\\\\\\/iqrWTUjHTaYp7gsXt1gnkW3bWbM+GFkXKTCpJoCkujy5wMxbYDsHVhAf4l1W9753SCc84+UrlOA6NhFOI8qkujW8ETzpmFKEIJe9c7S\\\\\\\/kyFhFiF7iTci1M8BP1H\\\\\\\/45OSnRYgaowf6IDMCp5GghO14dqZAU2TGNlAZaOPma63ZV3ZPiKwB0STOrI8lFY9wmYDxyPw6TKxQASjomk03MM0yiyDSRCaPpMaAQnZK1o6\\\\\\\/1Rf\\\\\\\/lX45K\\\\\\\/l7oAUMPg8VL6heUSrSZMEQbqtlHnQuBdzVCVCAIobxdktbcCn2Xb2KW\\\\\\\/P6lhKpSUcsZ\\\\\\\/VKyKwIASyzap1gPwrV9x63nYXZ+X58fLlxySgjvAicLeTgjOBKEYClAwlPaYzEdzqO8RQOJnvgPjO3SSIKqljTIR6SswyC2fCYHQ0zm4nDMcttEqEvkS\\\\\\\/bKyV7LKdVlEDYV607ZA102e4vp0wJY40VyErZzJDaUxm+GEPqOQmFqKweEILK\\\\\\\/3DqnwJ8sxafRtjNe2NkqlmjntvUsOIhMFARPPdnnC2e9bXUxG\\\\\\\/gn8wiNTRek+h5EF8nY7hT9S8qqYaBIjyIyZ6Llr5mL5HwdZSN4tyNdhedFL+YbduDgda64Q2\\\\\\\/rgYJi1ob4PSTxiWCPzgMOVp5zKZxTJouYj9uGmmAnYW0Alhd4GEU0Fw4nNLRbiA81AzygW8XJBpOXC1ZUZ1HGkaULtv+EGEQk7N0L2lpUsYKtwASXnLUl5jC0qx8oth1qGNxhkFK39M374mkeA7lBwcleHy0TkBh3INNns0sLVQVf7wn9vBUwNYWkFgRiboCoQfKOgsmA8Y3O71tmI\\\\\\\/M6JdtVTQ9SGVgfXmzkjF8TQvP8WF2\\\\\\\/ywgSmFiQeWKItPMZAAmvfkWDK9lDcxyupCggZgUIhuYYpqVUitgLjFmZks7CGfp1TvC8HsUZeVxQHmJqC2yMKX9crrw6TMvYhAfdV\\\\\\\/8JGaf\\\\\\\/dZ6GIVZ\\\\\\\/9Nqyh4ZElVvf0si+xwuHHqCLtkjYOfjGfCjPq8mPqrUI5Fwz3JZdDPVKaR8p+OLIjPETIqa1CIhVWA60uUjuEVPxTYxBoKYGWRo41rTCZHQKkTsKWo99NHg70vRU1Q3g9Oxt+Jt9RxbDb3L+U4vjMNyycaSFpMupXn5E4aBMnfWVpsuWWXWubF6arbfW\\\\\\\/o7iwrimOSyeZKJ2AfsqO7Ozo82NGQKWiJ2y4b+wbq8WCj0CjGBzj2uDw53T48hz7LD1Wm9voBDaXgf07j6Db0vdm5Byt4y\\\\\\\/UacEOrtNS0gAacK6jQul7aoGsqkPEbmu1DTE3vroonhGn6MjyBJrRLrH9xAybE9vb2fYOCM9zTEJbotOjszlblja1OFWmTV89ztE4NJlyhFDvmNJwVUWE0ba1g2\\\\\\\/MeQBU\\\\\\\/0ipjzj\\\\\\\/RV9BaWdPnay+L7Lh7Yiu92RowRgXn\\\\\\\/Bbq16vXLZuEWtEH+5u+5kYD6HeSKcaVqGunFMr9fnkCCEFfL4U66WsDvyR69MwWuGgOUTsTzj7NnCHKNum+HEILUwjDrDgrA9bjPGd8SCM1QeoSJ89WBfOvk9kDzQ7F37TTgXwo6st6PR0gQn80VJPClXJ66OaGZ4CfF9NM7TghIKOZSyRHZXD64TMIrudABLZ9\\\\\\\/hj\\\\\\\/PDqZNa\\\\\\\/oBs2vORnXfRlepL68NigpPqDqjmL\\\\\\\/YMl1np8wYmqxbVq4Nf2vj\\\\\\\/yM9kNkVmzX7+j0b4ppuTYsXBTU\\\\\\\/cLuRK38qbnlbun7U7oabWPVP2zEtRauv559y1YiENx6E09LHbmkdIthAdDqp81++K8ASH3277oevhigxl9kJNkN8CqdTkyhNLBdmAsr4dkSlWygTTpT2gSJXOI5d4D+qGcGJIF4ti0T9jwY2toz8q6Kdjo9Z7riaG+svCeUSKOpPK1vxKaa0P1Yko+EQ\\\\\\\/qlRwXajA0BnXNGyfEydfPE+EHUI6GLIdWe2D4TB511BoKpa7La\\\\\\\/gQqsvsZXOQLi020Aq9Cvi7wstkO8DLJNIE3fQrg4Nv\\\\\\\/RH179VZwjRTjxvpOPJ6W699TlzkO9lrhvuiqoUpHDz1Q7vHbLhIC\\\\\\\/4VH9qWWAbMyUhs+FTlG3\\\\\\\/ImgYmeV2XdsVXKXp1YHQcmvdGK2Jqum6QHOV+KkJn6KGhVshbHmHvrUr6f6Rgsbfsn70dJXaawDrFY4RwD7truWKAexBsckCIlEAMygqRAZRmFjLrj3FuxTgk3RKE2oia+\\\\\\\/ZlJuZszB0moGDRojbV\\\\\\\/IyFx7lMDWzRvtNEPJ7oYo2llvy92K+s1qmyDo7T3B193htejATTTCd\\\\\\\/E0DZSfuzq3dJR9R7sMkdEQ\\\\\\\/msc6JcQ2VmQKGzu4rNiqcbauQP+UkMh4FRhmpkH5EqHlVZ\\\\\\\/\\\\\\\/3b30zHUhTIouwJM0aQ63bB45vkhayqHhb3r35IYy6Pve4rM3wCS0fBRMUEJyuSW6v5\\\\\\\/L1L02LXCPZEm3SQy8rOgtHG7B1yndgIn5L1phfwSllQyimucU3w6gP8KMr4IYE0wYZtz\\\\\\\/Y+wSleqC6W0v5ttGb1B4cjOQ3SnExrPiVjXOesHc4+T4cCQVOudWUM\\\\\\\/osIoxBMHbvn+K6mbgFpqISYZybCvAKg8+xl+KYumCjpAD9IJR9u6jyVxGHluhKhGRbnUoG71+\\\\\\\/jx17yTSetXJY+GUbQ704SvJ6g3Q0Nt\\\\\\\/lupRQ50HSMpVbrrMQYtdAuif+CcMhnFlIA4asNYLPfhQIPeKuyC4zMJ32kb1oyldCOYiakUz7NUsCTafcw4c9WvMM883VFhQy0kYNEJ+CdKaediFt41vWFtZdPilkwt610FQwjJVeSyArAvazeM+Ly31Cvuu4+EBrzZ2MOs7Yg8vpWW+0TKi782VvWsTWCIkRWkAEhX1VzmaM\\\\\\\/1EZl2rRaspPZLkUw8HQk66QULidAVknNAXrckAi77TXzO+SBmBjhlLsaETyJVEQNxHcFtwp3O28zHx0+hAqpbqqvNawaq9rPlkMWvQhIqpVoC\\\\\\\/pUYYmcCHhB79CXhzLbcetbxoYUBuunMJUHCohin2fOmR0VdC9hFhqIHzE6IlHbv3VI76GbHEgzW5VDwgsjSQelNKWsBg1TavSkXv3Rp\\\\\\\/TencNVVVgg0YCzu1Mt83I\\\\\\\/4DHUBZ3TVLHu8czZQ9By+dCQERqfV3ifc8nm8+cv62QISaBlz\\\\\\\/14jENlwC6ZTbYztX9hMYSOJ+sCZSxdZymWR2iBSru9jMY6wtbqgHpiY4yX3Bm5kkD3SerfT54eGtO8psKfc0TDM0F9ivKTQxViHMDM7g5PN7ab1o+0KiYqRXTiVmMwkJo1KsFgR7hMEaLiqQcrCMkWmFNQG205KfPE6+lZkxoTPf1Aepn1CwRgVPmXo7I8IaGQN+UrGBrb\\\\\\\/ToSvrfeZY46kglzKevx9\\\\\\\/d+Rvwdjgjpb4oe\\\\\\\/WW4csYohC75CgUAaEF3+6bU1iwC3Rtk7Im6fYUrETV1dtKYE4YQnUPDgZXzNHQA\\\\\\\/2KbEX\\\\\\\/aEvljZugtlGzLo9lHLu+0jSDuUblfYCIWOZ+SNbBQL6FuFZqBSQ1pTC2CzUDjuNbwZfbN4+TzCi9\\\\\\\/VPK2wADmaaRKsEJSj30G4lkz0Wfn89yTg112WjiIFpkWzrKSrqdzHVosICWTUg3uSnZPP\\\\\\\/\\\\\\\/AgP\\\\\\\/72YmjBc5veHtWilnIdlK91GUXbbgPsVqApXzp+7UJ0uMG51sT8xVs6wlj5Aq0V5JAGU5c3PM8uwqB7XHk86iaMXpRvhp2HtnK78FjpiZfId0QjCIY\\\\\\\/6q\\\\\\\/9Ve817lN0MfNBiA9AU787P\\\\\\\/9eGahp3S0SxD5erHmgKzehEMt\\\\\\\/Swgb1wdLTSmSI54Y9nYeFpgFQb29S33mTiaLemmuL\\\\\\\/4Kx263JOTNAfm8dNVRgtWhdHOaW0vuZGa7Co26XE+QOP4TX8k4wuvIz0vuRkPLSNCvHbDZ5eFlhI\\\\\\\/ETbCNosBHYB7QO6mV0qKmREKvSLHQkZo5Vi4tY0qjIXCizSNwpl3\\\\\\\/EYc0uPDVE4gD+HSIp2df9dbzIel6tZ9OvE\\\\\\\/ELatJssNv7\\\\\\\/kzi+nf\\\\\\\/2eTBnl3DyPM3qake5CJ9geODKBNxyQYug+Dk0XdGB8Z4fVDbhvLxnBqJaLyGh1rRa3uLhuYlUhTMoPlTZRtvwQoZhD6HpVhDpjdKay1JHKkHt2koruT+leQaGl2NAaKAm190kmP7e\\\\\\\/h3\\\\\\\/cL+Y8sHD0ZMuzzybUgq9yK\\\\\\\/vy0I3wvkCH7V6Ky08EvoPsJACtyFrRHr7qa1UrGEc+t7OMkeCAeFVc3By\\\\\\\/uLsv4lo1JIiH9c5+B618BGsI0m9wfmTymxd6lv7JXNI2ctdoO05nWdbtfaruILVy6NdjpEoI03vDX9cNjK6hd2KYAy2WLijBbEVr0VNQmgF2msutwyy0MaReTMzsC64Wk2+vkluSegGzdZ21V4bgKEmydMRC+jMjMSmKciP8S4L2UCAkQe2RsnwiNaFJ11wv8zZx1gXUeA85VrWyWh1xlYXTIgcToR1VQloliBzYM20l3yxvH+yNXTliV0BhX2isLqGtXyIzQV8TADoGyiRtLCCjCWkHwTJEnHD3srI8iYTNCss1cBz\\\\\\\/vSMEKyLeSnFiQRc\\\\\\\/bP9vcg2ISgZVApzInx8e6dMQSgkfEIJ5Gz5vfrvonieh95N9lYqWn7xppaYay3LmaMN7eEXeLhCw5kPGeALXZ9NT1+kBBiEYZJaYkmjjh6bLQvYAT6j2qxOMBsnZHK0HEFaGT0Kpx7vAAUAH2Exkkc6Z5h7FDFmyynJ7FPmtOkFkRnLxPSju07AgND\\\\\\\/3aIl\\\\\\\/FsZqbVd\\\\\\\/sB1A7MCYbJjFThwZTPmgmqyM4pQs0508Q+cVDXa6HDsOxEM9ViS0CozUmNzQpLC6\\\\\\\/Hv0v1es7BDK55cpF8sZUsz71avpkANwscyHUltzmcgI13WDJ9CNLQSsmLvNNvwo7oPU7GP0\\\\\\\/9XXtm3amV0Y\\\\\\\/Xm4cK4ywubRIQdcYHAH11kSvCPlL1QTg4Ba\\\\\\\/uQldF+SwWqtXHapryQ2+nzyyEJEw\\\\\\\/mCdLtIK1ajtjwxdkOafIUpuwebOalBZ9wXRGX+hk4IFYlZK7Aymr41V2v1tQ5qSEv4HnBPSza0S8ShqQS9RacJzO3S0gzSi2B24BeXbPDZW4be4QIJRgdwr0CELfIPOpjE1WgQOnzJqzWY7\\\\\\\/G8YbFJps32nsXsa1QzmCDabx27nvBJY1S+24EURsWSwN6VEWoO3lMMXs\\\\\\\/MoHi5K0yQhyNdy++WiaxT6G28sM9lGFM4zavDsnZr1EHoMnlA6z4n7exfgynDnGlVCL4iTc2a8E22vAYkALZt1LQTatPUtPmNofx3\\\\\\\/WE+wqBFQNUDkLfjcsoo24qyq5pXoZdraCP94cbowMlDBs0nA9IjlQVJkxj\\\\\\\/nI42JTvWykP3QSC8S9zwr\\\\\\\/9vx8VvEf8euKPSoumCJHhMarxLX1LPnzsVA6cr9NOeDy+dGmxiGU9wKEIcKq+a3KfP1XHxY2IjevsqDhPLOJyrWRUvUU8d4kuEFjdARfQrrIe7f+Y99D4uRCnuzTXFZULD3dArh483fTpfIBYWd4BG06XVfubI9YI6HO08uxUFgFqZ7jS+AJE50M2OrC0D9gT3dYI\\\\\\\/+M1J9VaKkTImB97Iy5igR38Swo4Za6aGqhw2lTm3SDjEMDCYLOU0nlFBXvWYqBneZawyh0NC46fjj20MlB7bkYKu7T77DSz17Q4+s2+WOiKRrffZJ63qiaeU8tXvFRy9xYsNzVyeFnAvA9EDffkyFI+aJo5ym+eVM2U0EhDpvMYJebP7XfW7aizZd5uLTLVRcjYVpslhY7C87BJWOYSQJ+LkyyCqHQ+I3rKZjF64fLAnv2zC8q1FleJ7fa4Naght5RjYYsUmyC\\\\\\\/5xLRNvPS6BJ75UdO4McZML6+MFtSujjSFMN\\\\\\\/1Mp\\\\\\\/p\\\\\\\/pD62dX5F2WNLZthnsHEJJW2jBEpjuAv\\\\\\\/wGcvqiyEHXx9gGjBnudAohUNrt\\\\\\\/b0B0RcVSdcJecQNfgB3yZjfl8hG3XyazhVWE0TBniKMXX\\\\\\\/vegBQ\\\\\\\/jwlhhCZ9CYxDtS5nCW5Z2ER\\\\\\\/25+7ZrPUxMO8OMyhYYqlupacI+qPwz9aIU6ekcBI2yaV3AEHpi7P+4zUFVfsLfcxjQoxk5ImNtNyjE7JR0gaCsYnWf17Q7su943nJVu7NfovVjQVUCk0fUrGEhxpuX+G442BpZ7V+HIYd+8kmiA+aVKJBIT2rkLXLoz7gLSKQDMuHnvUS2fLQGX8bnxkwy3YfOBz6\\\\\\\/7kS\\\\\\\/0D0e5Bzmmk34fWPNhW9V+wrqWRoEzSI4nH98yGgBIM88uqpQgSGegkMTDTxcATg7yj+NTy4P41vTQ9xytI5xnreuTj93nxQ\\\\\\\/8ZNGNbbK1DQI5EXBiQNWw2NR+3miWDy3bH6Xt3CTEUWQ6I+UMVB0VjjAYn9AvEe5slGVMKwHQ1oMVGTUzEuGj8myxif7bnckH+x21QVPZdWwq+x81FGncUyL9myknSSI\\\\\\\/5LRBDGfXwlyQOGAG8J9ZVzfdDa0VeD\\\\\\\/a34PpUxzkirNbdukxQpMNcGRw6hmwYJ87qxYURN\\\\\\\/qUBsANOlL2weM5zWnwBPd8hmOmyXH4S4xoH70O5SVogrtsWdV9E6AMZ\\\\\\\/yEziyLihjTBgRn08DCU3L+58etC52uXfLJZ2Ked1TuF0MLwki++8HH1QNbqwAkFMJChN1\\\\\\\/nWcjY9QHjY7V\\\\\\\/AhEZXnk4foep+YlIyaYrH\\\\\\\/+UvDAhpcgshGgyHItxgI\\\\\\\/nn4QE5bR59pBuT4VmNt5w6LvrfVtZoLoLRTviVQEaAHUnSUOjCa7qHvimwnzoY5l4qOl6vaVTTg2pwNW76IFFM+R9MjfTBA48nJqhH9fiEeCtAnl+JNk9vm2C4d+mArk9n5R4v40TXDb2XBwJc7FX8OxIOpmrerNcLKuY1cPvUxIaSD1rmk337hajLGRsC4n3oQcO5pW2Y6aCbVDi7zfgG8ccmQ7AU5Ju1OxdETsp2y\\\\\\\/IJFU+sT2er41W4K8MWU3bVeZ9GtLn7cTqIhckymC36M\\\\\\\/R05KcsijC6a4Ys3yvIDo3UXGyAa3ZZhx7HZGjVOQhtMHNdtmkjAbAPqZNRbV4FZxaelyu1RnuCdGbWpl8mMB14eWp1b9cn\\\\\\\/mh4xJSRacGO8kAmxhl\\\\\\\/Fis1M0DKcpKKrUJA6jbax0jW8jOcShx1ktUt+aC6aqiKNuJHVLlF0\\\\\\\/xgT3p107gyElIFWxtcvXFsXzuxUTy+Cp35nbnacoOY4Ew7x3+rBQvx5T7GKrnjpk1mwfdQyyUUijMJ2F\\\\\\\/uGZsLfOaoOr5IA\\\\\\\/k5WahX7SXIcdVxkOhQUhaFumA8hiikLsxX3RbCpVZtaZY+MwIKLNEeqWmAEKW1pdAunkddAaZYLfLo9LWYmQYIH4FwF+7RTlbLkMTdWa\\\\\\\/4cFK4yh24Ik9KhFpbcyhzeovlJocM9I815b7foCO+nFfNpRVyn0VLTn1XaENXloFryqYVuqNhf9z2PJvRvIFMRKvGCH6I5R4W1ChLSelmkYQ\\\\\\\/5Y06EZlc1wJV7Gn0rI1SrXMmkk+weK\\\\\\\/evMVSDX6baSt18\\\\\\\/cGd3dMME2AuaQCsJxyQZYDq2OWTFas\\\\\\\/PxaKW4PpdMHObHZLM1pDD1pRV6X3nT4U0DbjSaAYj2+0C60kugMddJk0zj3klZggiMBbVHtgwqp+2yqAofvnikL5u1cKuySsjbTHjrrFY49F9VfpMJFnxILz2S17yLlT5X9waV5FvKpvSdjF6UKBItn6rbZ6hiC+PX2lyuaqb1hcG4StXmFMKNEEoSQsxP35yu5lvZ5PQ1clm6xAE90WeR6v51nKVDQoCDJ3nuY3yEt3HbZQ1Miq42Gdkn5fGv1cYCBDdOFQlB1iwuaWcigTVjKSEvzfplRAX1k1h3iTxnSJxMF026wXQiqfVC0gqzuaIq35Ze6j\\\\\\\/YpnOa654ZAYwe73eOGiEFy5K+QoijtVki0gzxJkBXQXY5Zz+0VKLjLYJOHYu3Q4QIgHTSfNc+VnPOgglBhF81YcsK\\\\\\\/kmXceUEfUbatNTC0i6gnHCojuCW2SAxtu4euLo6bsegpw2f4CLLJV3MtIfabzR7cQa9Uydr73ZWm1LcpcII8YMuv5lnYPv\\\\\\\/WswJxgOtLAC5NB\\\\\\\/oDUeUP\\\\\\\/ZydDtij2Y+V7R21xEGOFkybFmFVHnaIO9fkfEzshiiEcxoD17fbcqBbwErSch5tRMy7PRpNRrwMY9plTQ7rMqJA\\\\\\\/XGGhUlUdzQrKXYnN\\\\\\\/VblAjRwE5\\\\\\\/Dta+U2K4Ivl67thxl0aEHAJUAC+kjLVmf8AiCvSU1KrwkmgA+wIdD+id07M5uB+xnXoecUuuxk3bz+7g8IEaPBZU7RCWWv1sW1Gy3FZC7TPne0\\\\\\\/35zs7q9T2JqnD6uyGDm71rCwO\\\\\\\/QKMsmnoo6afK+oOMsYmTTIM4muzdYU4tmfg1gWZJgloje1UyN28CNPqGwJlfjs7gHk5aGChUAfhDIwr\\\\\\\/BBvf363DYWUzRdZmTseRLn2MLLGvwpZ\\\\\\\/uU7yirtapYPxvSOfjhXOcy8Ilz25mAmEmyNhDVAQUn9bHPJ76vwrUPjOU1iG6wJU4v+GbnvriF6f836g4LOH5jfcqqqOXY9ZaQ631THqmfOvXsk8V52bkFIxUrBJElmmCtyiwG4xmtcC7K+FrRW\\\\\\\/hM1Y01hl8uJGIl+pEEx9ilzJmE9cSzDSWnX4IPSV3qaTIcxPs\\\\\\\/kgPq4ZbDAohCbxlZinSDBQ5dSlVF5rObE5Nn4y2DZR61ExmoQ0ICK33jyNpzbevv2e6gRzjUNvfBYLc9\\\\\\\/PiqEW5c9PvjSKsL8CYUYcONhh+\\\\\\\/di2EnxEKcT19NE6pZuOLaB5vIDImy6RzkHcl0YyPkEXP3kK6iQtc13QeWetGO5A2Wb+smq6QIvhdtbGrWH0jedCGgeLTIuAtPrGuYRtSHm\\\\\\\/ax0csOd6f\\\\\\\/k5427St0yWkOCA6GgHkk4LxfusvaBtBMjVmKY+2MbMO\\\\\\\/BeU7QylImYeXCtuPlUog1NX95T9VGsvIk8BIY3LLww9YADgNE4ax2Lf+rsmmoETHQk5m0eOTEOlaTo5zDmN7lIuLzxl\\\\\\\/fuVf3H6D9aWMz8Wsn28Y2yUyftKiNCbyfDKebXVvu\\\\\\\/\\\\\\\/nKuYDvRD050F0o1Ntra92L+F9hEZpfTfIxgiPoH8R7HoBuRU0cMprTgniIelgmCPnCrb812EvDv\\\\\\\/VXtVWFkELqOLzBmDNYjs64os7eOwhrwPFz9Sss0uZyG5RwslAGEmKizoIKG3wZyTxlVEFbajRwPsF6Ddamc2A72JUosU5eaOTj4xMLNRTz0ihdckFiVNNuQz3msFLqwzk8l8Mod6muj4Jmw5NuXzrV0j6ZD\\\\\\\/f+4graCNhq1FJNPL1Zb46D7BMOdcn8PweS7VJgZWJU1xz0W70anNxgTkthzsUjPyRIQ7nWRxn1pAiAqR\\\\\\\/r8OsQ+CyY7vttUm2iMAm0hywpCsd289W2VpH4nP8xXR7V7lenwwMoqPPWgf9evvNV9CAwcDiWLRSqkpa58roCVSCULtOT0LDjldz51R7B3K\\\\\\\/MYBxvQEW0u34sc1A00u5CwwPEpOP3dbFCanRgxmXmp5PC5dq+dHBCZt5wRewHqm4vz2qymGrcRSsB+DWxeayB37L8E5vG1VRERmtPjldFgl\\\\\\\/mQc\\\\\\\/Oe4+OC8XtXT1QLyasX8toqmX+rpORTneRH6tPFfYEkAzWHElznVYnzN9dp28GTwhSxXqc5MAkgU4fE6dGemIuy9EFQSRc\\\\\\\/RSQBKsVcmYFmSzBlxlZcK2JzWBJNDvW\\\\\\\/cClaimuZWiMf37u0ZkJF5zVO6fZB6B0KpEYe5PG51bDc2xb8D+a8nZXnujKvDGCarLqQAerHUedeEfpd0xVFmnOSgTA2u2jnTilMvI4646wsQaEr\\\\\\\/WT42uHCvEjKm9V8VcVcSnoOhT059s\\\\\\\/VXkSbnPenzqfi68j8V6kZ9Trcx19DAuDwzfWmTN\\\\\\\/c+l0COiJmzuDcixA5XUKB3\\\\\\\/TeLxFCeX\\\\\\\/HKNZbKLJPpzdUK2r1j5ZgIN86rFI3PUvmNK4DS6CQ+Jk8jX67XQvhBkgh6dlaujyY7G9ibYx8TUHeCy332+3IYG0JQ9lChyyRoOx3w51pnpUH3roxfykKHp2vE3uvYZB4h9Ype5fQ8VwqGF0aWj3\\\\\\\/GC0MBvIb3+ru5Z2eFk9dUIxLcuzJwNpZ2kOBwZRzv9ypjUs0En87\\\\\\\/6ebFlTiUiEQdkud9zvz7AxhcovxwZ4v5iI6vUDprI\\\\\\\/xhCxTCjH6sGHFclCP5eB2attTURIH3MtyqfdEuoPXMJe5M4Fd9ul1lcn2rjfRO\\\\\\\/k4XTxlAuZ8ztPjelTt6WI9Jw7iKZl02X0ULO6rYKUHULWssJUh8X7NXY1dRubN7zWsa\\\\\\\/LzhUCDHYj5l+GzmG3oVJ2FSlYyZOsQajyGeWPkcYNGRXupaP3a2A4uT85++AOyMeDHBezq1h76zScR8Og1unzZhy4hdR8v+KfQCt+voKTDfSvm1MbMFkp83YvPMRTh1TZojBOvy9ckkIIQRh0VQ\\\\\\\/P0\\\\\\\/qsMUIQ3xRlpGl2OMzCcbSnCfddGsXSe5ccJG7+MQFAoTi7t9rKHpFb\\\\\\\/xb6dlk\\\\\\\/s8B\\\\\\\/GzQDRZipw97hLnL5H0sSZwEdI\\\\\\\/ywQR\\\\\\\/S6qVZqsaD+eDIxNpCR1tWom+e3c6LQhjQZNVvVCQO6jzroYoIt5I6azi\\\\\\\/UZdPfCSElhvwg\\\\\\\/pdl9rMdJ9NpWQb1sg50VkSFYwE8u6PrI0WjgpS\\\\\\\/mZx8I757STMf3VIdqUuF0lRPuTGORXkkgoY0G\\\\\\\/j6hYRFPCzcy\\\\\\\/J6Pyf\\\\\\\/I9YUagg6P72v2CM0vP6z2rZ7spZxbHYJlkP+pL3X4kPaA9n38J+rHfKyCidhJKXVq\\\\\\\/tjFd0oOiuCG6IzqpjOcrIbPEzTMjN2K+qwJ2ZsPTJbX7ePLRUmNc\\\\\\\/ddjEJPEtkz9DJRju60x3vtj4aSl5p4XARKo9ngsvrDOZ+R7kOIeACngb4X0SsvSZjXu5oAlWCFqm7ofzwhOOVUhfbrtVO4t1dlxhBbtyQUSxEQS1rXmiy0RQSLt67MgJcNtU2zRDF89JpPh1wJbe5r5blAir05OrnMotN9Wk7VQGbPIZgfOF+raJLHOLDDBev0jPPUlaPfn\\\\\\\/J0pmTZGsFIUDoiyoyCz\\\\\\\/MRBwMPHAuRe6JDtHNE5V\\\\\\\/BSonjEPykap2kBvHQwYpyxMCjWhNUNrTLLGnctLyTFZx3VRV5gteBUcIJ4J7PrS0J9gK9JGTKaZDNQKjWeah51yGUsvvJS\\\\\\\/Btx8CahiVSekAjAWdh12TlYq9fJQ7n0N5+QjYtAQ85TC5\\\\\\\/T7N7uCwanUfheqVC\\\\\\\/OZC5RqCfuSwOpezRwQWAxY6+Fha++7PPkvd9Bx+bVaYpijYDkhuy20vhMit6\\\\\\\/q8WzPLgrQhGf9olL8RcSJx2mE3vwNg3ygmE+ngsK\\\\\\\/GVSSWadBpYZIL4sHgXt4JmYbhijW2QiNVNaS2bv2+PqB9ZtZ3kFPImdivpksRLzF5K0aIgvzDPLaEQ1YML3PEGAt7fAzDWDsPNjA\\\\\\\/7JYQuhYvnlRunhZ468ctP08I5rNF+9fs7MJPWTPYJxtgGL8Xo9v9UgxEneZmMBaqZAJ3E05WK43tDutMHaeiikOslsVE8+C183ObhmTB1\\\\\\\/QKF5LM65pPHqVQKMI+htnJpboVKglN22Iu60h\\\\\\\/mqK9rGV5L3uEIYYbkZqwsDL38iOaWhH43gBey7d4jJpkZgrGzQakJHlQh8mg+HOQTsqDpMMLwfD6km3B+PEwwtHJNADIjk0nYnAPWYHrrJkw9otj\\\\\\\/vbjBXgvHhDX3bfsmEYy4S6znI4SyFXuCmh4cT3JMJgAPzg8wYtfGw7LSGarwKMNl7uj3FrDjjZR9AnpnaCuNBXIGh9gXE7n6pTZXGpqm\\\\\\\/5rHz19YZ7OXP4ELesRoifUqQPiffNcruddXQ7oVtPJWf3GHrXLfv+7NKOEEbr0EN64ptovZ83sTZ06GHxNbBwv0zi5YJqtOBj\\\\\\\/CCOHlDo1VY9hhJtBEYrpmmDayWlgMi4G7NAM4y\\\\\\\/\\\\\\\/kgT8uVrWBv1HMLgccD4t1Gl6Qqxy1GaK4xgQ6GSxcBIJhHeMN0TeGqQwKAu5gvDaISjrfytPwSWJfOvKgdE8jJErL2d45HHxyvDKx6\\\\\\\/E9mY2udc6ALRg9xfckFdcAZHQ0o\\\\\\\/m98L3l9jxD41EdS0qLkVAAR20159V93PgM4S4y3oUUWuxJNQEgQh4GN0VHzXNvE+x5etr7\\\\\\\/pst1fitBT6wbtdYosSWiz7XpxKjTqP\\\\\\\/V6pIWe5+QnojVi4hMZiv+cpE0\\\\\\\/DH1sdw5C6d1V3yA7wvG39M+7ljtLf10I7lntDEEnzf2SqV+n049Rjuxs2cgG6GSLviRzYqDgwwY8Au8IYVupT4Jkd0cubrAcGu0cqTJ1ATIkAsgEWftVXXepbrI4d7c\\\\\\\/i\\\\\\\/o8I\\\\\\\/8c5ID69fzYhs58fNMhrr9gsTVt3EBS4hMjqCm82BoxU6Jsvgx\\\\\\\/SfwQKshyY6pstRjeXar1Q3CfuGxMqVm\\\\\\\/F8sbibyhObxoSstf23icJhe7lZdXKffMz3Q+2Q8rc331OHGl+QIiQR0ibEfi6n9+JaTSpg21veP+LvTEU5unM+jzOYCwHEOwhEE8xaVJNydaXDgUfW8pNdupo5k03nEidJXrxSDEfI30ySVzRit\\\\\\\/uwQy0TkOgmf5OdVhVyoZ4x4hhZErm\\\\\\\/cAixszjxMlaqtmOKFiF9Tbco7QnYLtXotx5EKLX1LmZtg0bo\\\\\\\/vmR8edJGVz7bP66mW0d90u4MeCO\\\\\\\/\\\\\\\/HqLJuibl+Y0kDK7k5dmSYYADB7rPaueULMuJvCOHVmWwGfm6m0C3AY6j5bUZy+ha52eNJWs+0k35hNZ7wEk+53k3PmEmnq47Bo7q3i9ebcXvomJITSZeaOhtZnhkq6t0mhsGNsSrLw9XxVhFKaxePMrzPwyzRCMkg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022wtc20ronOLRZuc7oXOxBUw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002277ce022194a7ae6053b19fde06a8502b377e3838aa5994e1bc2b230fd8dac3af\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022WI4Ku8knCaYb7NaXdi1WZcMq\\\\\\\/aOTi5KQD8oiP4Xk6d8YDVwg1Yg7xZCfpxdQQo23VGIFnTTr0t7Hy9YV32YtFaTpXxyfnv8DKO3CmmSCsOyUVv2QhJU0ZW1VrVJQF8bftZZzoQfz2c7KBnCOEk9yJA4wPdmph5hywsmh6DvFuXkDBa+yJjInkD5qneYTt9TAHBZKdlnJ7pnVP0O6rqeGVziRZT+KxfGl4ZN\\\\\\\/F9KvhwwtTBqkY1o6Qpz3P9tuLdEXlBWbhMCeHY7J6aoWs9L0SCbX3FQye4ltQ6OZrdJxsvvtZy9J6jo+o2OcXDlOKiIDqF4uk2GZSRKqywTbOQGmf\\\\\\\/yN01K1lHKQk1f\\\\\\\/EG\\\\\\\/MGht2aZI+5lqLP2\\\\\\\/omzkjULIfk8RQ+A1FGgII1+3Yultcsp1sGsZezo5Q7toBWmwE4xAKMxrgu0KL6xbyDUwOu9B2NolJOJOQweQZrrxtUo3YMjt7wazp8lrlizLTKTnVZXNz7o7O28z8grdjmv88Jvb\\\\\\\/tz07WcM4CrkJ\\\\\\\/6PKi8H8QroSYngR1OVamFR6FebGRSGEDeiXR3epwH68oEWQomA9VY86JUR+ptyRKSWyd1jP\\\\\\\/YIMmZq4NSdvxU1KgIDt8Ny8KHBebkEn271QJzbrcxpIa24Vss1HhSxdYGmNxmZKPGwwqo6bI5CVJqZewzJ4uRBLSud6M6qjxCYZsSxKXVo6+LlMIECPmwQJaQnujxc9KopySjRAIP\\\\\\\/pl8B7ItEmx1izjzLfJEHZN34p8RE+efCCTQ4EAY8wp9rZ9Ly8hk23oevEhCvrYQ7ZWk\\\\\\\/u0BE1jQkLyJugi8YwiimhTJLc0qxhtzC8I+1beGd62xKhlYphV7cU3Z1sxDCYFux6tfpS9RVdkjF\\\\\\\/9pNHCeDiXSPvPtie4+ZsOCCGVp5BgjhpkQfrQKLGdKMZbfBydMbJNjmEkGQDydwhuwjeyYxPe2l2kb\\\\\\\/53TK0k3w7wGufIhwH55Q2dfNptIlaCQWHeUiSWOmDmjnJX9DKSbx\\\\\\\/Qus4tDfWXU\\\\\\\/txhz6d3SfmQtM8y5zP9xAnGznU\\\\\\\/zl6v3B\\\\\\\/RAh12aWd6C9xwpP5kmMKmUDXWO+pCMOZEvHxPo1x1RPH2NNPIOt8uYImu4H8ee2ZqA+vayNGyqm5izeRqFYjUlPsTaq\\\\\\\/VDzoEJIWMoo+Vaq2PixRNWDEhjLCwlOKsQAjFWj5MCbOctDwzvW0pO4rINAV5pdtWpcqmlEAHtxOQsX0jv\\\\\\\/VzHFcLgwsai+GGdzmpaQwSaZXU9JItwWr8SuUtWvhOrELsCIoiYeBkiAF63clv8keNkrL+rpXdTdszu0Ddp4bRabFZLx\\\\\\\/E25ak+thS6ZgUpTqqphNbc6XH0V7Cs60IqGayHRL0drPR+qC+mriQxOyqRA1AH7SFKp5NF9eUofR3NWsTWjZBmnNX9S12\\\\\\\/85XYSl6pN8GFzZfkD7VxfDJst6Yw027OFYwj7JbolDxQnm9DodKTUsKqJP8jHGPJNSymj9MKaZIdk1I+Z8CBYg4+lx991mm3To0VgUPJqmDA17QynicQpOVl7R3ZIyGbFB+fGIiswL9e8NTExUib6yI4HWMGw70v8FIcK3LXelxV5DgR5+MV5bMxXqadPjqZqr4I4syVc5b\\\\\\\/fpbEej4ZFn87WZoM9lZx21rPvSSOsJW+EVxW18BjK9FX1wzKBTgQ6sWteAKNBHrHqXS\\\\\\\/M+\\\\\\\/VM7QPutzSs6NORVOiTzg\\\\\\\/PsQb+K\\\\\\\/VczKkeEIIDd3zNqiSMOjOdU311lnUxnmVWF71oeGqbU+kP7\\\\\\\/b+h7H\\\\\\\/e0i9X4wX5csHc9IT95LhSyFR\\\\\\\/LAELx6klh3u2hq6\\\\\\\/BgYkt41jwTlJT4n1AjAr5LrAFJumXIUjupk4AH4LD+ayVmat2JeqHL+\\\\\\\/Nk0CYG84W6IvlGwIkGtgqrMfY27xKI3mDnzGBzy8abo0M4LBGYQC15GU7Hr1lAaw4hd6iFtUl42V4KbNIEhTnVvWUYRzRL2pze64tFlczXWpvi4Nqae6mLm6u6I\\\\\\\/NSp5mHZtA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022ndqEXkjrcHMzkWsGHmYCEQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022816561f5df7cd577a40b7ccafd9144e66759cefc99193cb485fe9cd5674d6bc0\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-491225 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/11/3669" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-_LtwmoVsrxwJD_UqIMObPYM00HNnnnVF2dx-BrhSejI" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/11/3669" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-j8zF9-w0xugoYvGb45KnjCsSWbmCl5LLl3ibYDxOOO8" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/11/3661" title="A GAL80 Collection To Inhibit GAL4 Transgenes in Drosophila Olfactory Sensory Neurons" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/11/3679" title="The Role of Standing Variation in the Evolution of Weedines Traits in South Asian Weedy Rice (Oryza spp.)" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="491225" id="node491225" data-pisa="ggg;8/11/3669" data-pisa-master="ggg;g3.118.200630" data-apath="/ggg/8/11/3669.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Discovery of Anthocyanin Acyltransferase1 (AAT1) in Maize Using Genotyping-by-Sequencing (GBS)</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Michael N. Paulsmeyer</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-1332-711X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Patrick J. Brown</span> and <span class="highwire-citation-author" data-delta="2">John A. Juvik</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3669-3678;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200630 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node491225" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Michael N. Paulsmeyer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Crop Sciences, University of Illinois at Urbana-Champaign, Urbana, Illinois, 61801</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BN.%2BPaulsmeyer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Paulsmeyer%20MN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BN.%2BPaulsmeyer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Patrick J. Brown </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Sciences, University of California at Davis, Davis, California, 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Patrick%2BJ.%2BBrown%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brown%20PJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APatrick%2BJ.%2BBrown%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-1332-711X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Patrick J. Brown</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">John A. Juvik </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Crop Sciences, University of Illinois at Urbana-Champaign, Urbana, Illinois, 61801</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BA.%2BJuvik%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Juvik%20JA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohn%2BA.%2BJuvik%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/8/11/3669" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:491225" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:491225/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/11/3669.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:491225" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:491225/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/11/3669.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:491225" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:491225/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/11/3669.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:491225" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:491225/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/8/11/3669.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:491225" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:491225/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">The reduced acylation phenotype describes the inability of certain accessions of maize (<em>Zea mays</em> [<em>L</em>.]) to produce significant amounts of acylated anthocyanins, which are typically the most abundant pigments. Acylated anthocyanins are important for their association with stability and are therefore important for the various industries using anthocyanins as natural colorants to replace synthetic dyes. Many anthocyanin acyltransferases have been characterized in other species; however, no anthocyanin acyltransferases have been characterized in maize. Therefore, a mapping population was developed from a cross between mutant stock 707G and wild-type acylation line B73 to identify the locus associated with the reduced acylation trait. High-performance liquid chromatography was used to assay the pigment content and composition of 129 F<sub>2</sub> lines generated in the mapping population. Recessive alleles of <em>Colorless1</em>, <em>Colored1</em>, and the reduced acylation mutant all decreased anthocyanin content while <em>Intensifier1</em> increased anthocyanin content in aleurone tissue. The association of increased proportions of acylation with increased anthocyanin content indicates acylation may be important for increasing the stability of anthocyanins <em>in vivo</em>. Genotyping-by-sequencing was used to create SNP markers to map the reduced acylation locus. In the QTL analysis, a segment of Chromosome 1 containing transferase family protein GRMZM2G387394 was found to be significant. A UniformMu <em>Mu</em> transposon knockout of GRMZM2G387394 demonstrated this gene has anthocyanidin malonyltransferase activity and will therefore be named <em>Anthocyanin Acyltransferase1</em> (<em>AAT1</em>). <em>AAT1</em> is the first anthocyanin acyltransferase characterized in a monocot species and will increase our knowledge of all acyltransferase family members.</p></div><span class="kwd-group-title">Keywords:</span><ul class="kwd-group"><li class="kwd"><a href="/keyword/acyltransferase" class="hw-term hw-article-keyword hw-article-keyword-acyltransferase" rel="nofollow">Acyltransferase</a></li><li class="kwd"><a href="/keyword/intensifier1" class="hw-term hw-article-keyword hw-article-keyword-intensifier1" rel="nofollow">intensifier1</a></li><li class="kwd"><a href="/keyword/acylation-anthocyanins" class="hw-term hw-article-keyword hw-article-keyword-acylation-anthocyanins" rel="nofollow">Acylation anthocyanins</a></li></ul><p id="p-3">Anthocyanins are the colorful molecules responsible for most of the red, blue, pink, and purple colors exhibited in plants. Anthocyanins are a diverse class of secondary metabolites with over 635 unique compounds discovered to date (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">He and Giusti 2010</a>). The anthocyanin biosynthetic pathway is the most thoroughly studied secondary metabolite pathway in the plant kingdom (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Irani <em>et al.</em> 2003</a>). In maize, all the essential genes in the anthocyanin biosynthetic pathway have been cloned and sequenced to date, with <em>Purple aleurone1</em> completing the core pathway (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Sharma <em>et al.</em> 2011</a>). However, the genes involved with increasing the diversity of anthocyanin compounds in maize have not all been discovered.</p><p id="p-4">The most common modification to anthocyanins in maize is acylation. Acylation is the esterification of organic acids, usually coumaroyl or malonic acid, to the glycoside of an anthocyanin molecule (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Zhao <em>et al.</em> 2017</a>). Most interest in acylated pigments is for their role in stability. Acylation has been shown to increase the stability of the anthocyanin molecule under heat stress, intense light, and high pH under processing conditions (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Zhao <em>et al.</em> 2017</a>). <em>In vivo</em>, acylation sterically hinders enzymatic breakdown and nucleophilic attack under cytosolic and vacuolar conditions (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Suzuki <em>et al.</em> 2002</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bakowska-Barczak 2005</a>). Addition of acyl groups to anthocyanins directly increases anthocyanin content (AC) by enhancing anthocyanin solubility and increasing uptake into the vacuoles (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Suzuki <em>et al.</em> 2002</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Zhao <em>et al.</em> 2011</a>). Acylated anthocyanins may also have a beneficial role in health promotion. In a study analyzing the effect of blue maize extracts on various cancer cell lines, certain acylated anthocyanins showed strong correlations with antiproliferation (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Urias-Lugo <em>et al.</em> 2015</a>). Overall, increasing proportions of acylated anthocyanins in plant extracts would benefit human health and benefit the industries seeking to use anthocyanins as natural alternatives to synthetic dyes like FD&amp;C Red 40.</p><p id="p-5">Maize pigment extracts are an abundant source of acylated anthocyanins. The proportions of these compounds is usually greater than half of the total anthocyanins according to a survey of pigmented maize accessions (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Paulsmeyer <em>et al.</em> 2017</a>). Some pigmented accessions in the survey, however, exhibited a unique phenotype referred to as “reduced acylation” where acylated anthocyanins were drastically reduced or missing. It is hypothesized that this phenotype may be involved with the anthocyanin acyltransferase (AAT) synthesizing these compounds (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Paulsmeyer <em>et al.</em> 2017</a>). The diverse family of AATs belong to the BAHD superfamily of acyltransferases that acylate many diverse secondary metabolites (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">D’Auria 2006</a>). Fortunately, several AATs have been characterized to date in other plant species (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Unno <em>et al.</em> 2007</a>). Specificity of AAT function is limited to particular acyl group substrates and sites of acylation. In maize, the acyl group substrate is most often malonic acid. The primary site of acylation in maize is at the 6’’-position of the 3-glucoside. The secondary site of acylation occurs on the 3′’-position to form various anthocyanidin 3-<em>O</em>-(3′’,6’’-dimalonyl) glucosides. Only one enzyme with the capability to produce these compounds has been characterized and sequenced: <em>Dendranthema × morifolium</em> (<em>Chrysanthemun morifolium</em>) anthocyanidin 3-<em>O</em>-3′’,6’’-<em>O</em>-dimalonyltransferase (Dm3MaT2) (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Suzuki <em>et al.</em> 2004</a>). Candidate AATs in maize should have some homology to Dm3MaT, although sequence identity among BAHD members can be as low as 25–34% (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">St-Pierre and De Luca 2000</a>).</p><p id="p-6">To map the locus associated with the reduced acylation phenotype, a mapping population was created by crossing B73, the maize reference line with normal acyltransferase function, to reduced acylation genetic stock 707G from the Maize Genetics Cooperation Stock Center (MGCSC; Urbana, IL, USA). High-performance liquid chromatography (HPLC) was used to phenotype the AC and composition in the 129 F<sub>2</sub> progeny generated from this cross. The same individuals were genotyped using a double restriction enzyme digest modification of genotyping-by-sequencing (GBS) to generate SNPs. Genotyping-by-sequencing is a high-throughput SNP genotyping protocol that reduces the complexity of the genome by only sequencing fragments associated with restriction enzyme cut sites (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Elshire <em>et al.</em> 2011</a>). The protocol allows for the use methylation-sensitive restriction enzymes for enrichment of euchromatic regions and allows for hundreds of samples to be multiplexed in one sequencing run with the ligation of unique barcodes (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Elshire <em>et al.</em> 2011</a>). Presented here is a demonstration of how GBS can be effectively used to discover candidate genes even with small mapping populations and highly multiplexed individuals. The discovery of the AAT in this study, which will be designated <em>Anthocyanin Acyltransferase1</em> (<em>AAT1</em>), helps expand our knowledge of the anthocyanin biosynthetic pathway in maize and also our knowledge of AATs in general.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Plant Materials</h3><p id="p-7">Most reduced acylation mutants used in this study were first discovered in <a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">Paulsmeyer <em>et al.</em> (2017)</a>, but the list was expanded to include newly found mutant accessions M142X and M741I from the MGCSC, and Apache Red Cob from Siskiyou Seeds (Williams, OR, USA). UniformMu stock UfMu-09775 was donated by the MGCSC and was generated in a <em>Mutator</em> (<em>Mu</em>) transposon mutagenesis study (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">McCarty <em>et al.</em> 2005</a>). Genetic stock 707G was also donated by the MGCSC and was chosen as the reduced acylation mutant parent for the F<sub>2</sub> mapping population. B73 was chosen as the normal acylation parent, since the genome has been published for this line (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Youens-Clark <em>et al.</em> 2011</a>). All genetic stocks are available upon request by the corresponding author. Genetic stock 707G contributed many of the alleles for genes that influenced pigment production in the population. Photos of these traits can be seen in Figure S1. <em>Pericarp color1</em> (<em>P1</em>) was responsible for the red/white cob phenotype. <em>P1-ww</em> with white cobs was contributed by 707G, while functional allele <em>P1-wr</em> with red cobs was contributed by B73 (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Grotewold <em>et al.</em> 1994</a>). Genetic stock 707G contains a recessive <em>Intensifier1</em> gene known to increase AC in aleurone tissues when recessive (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Burr <em>et al.</em> 1996</a>). The stock also donated functional copies of two transcription factors that are required for aleurone pigmentation: <em>Colorless1</em> (<em>C1</em>) and <em>Colored1</em> (<em>R1</em>) (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Andorf <em>et al.</em> 2010</a>). 707G was selfed for three generations before crossing to B73, meaning it was not fully inbred (M. Sachs, personal communication, August 1, 2018). The cross between B73 and 707G was made in a winter nursery and pigmented F<sub>2</sub> kernels were grown the next summer with 7.62 m plots spaced 0.76 m apart at the University of Illinois Vegetable Research Farm (40° 04′ 38.89″ N, 88° 14′ 26.18″ W). These plants were selfed to generate F<sub>2:3</sub> kernels for analysis. Harvested ears were dried in a forced air dryer at ∼38° for five days before further analyses.</p></div><div id="sec-3" class="subsection"><h3>HPLC Analyses</h3><p id="p-8">Approximately 50 F<sub>2:3</sub> colored kernels of each F<sub>2</sub> family were ground with a coffee grinder into a fine powder. Samples containing less than 50 kernels had approximately half of their colored kernels sampled. Following the extraction procedure from <a id="xref-ref-26-4" class="xref-bibr" href="#ref-26">Paulsmeyer <em>et al.</em> (2017)</a>, a 2.0 g subsample of corn powder was added to 10 mL ACS reagent grade 2% (<em>v/v</em>) formic acid within a 15 mL centrifuge tube. The mixture was extracted overnight in the dark on an Innova 4000 incubator shaker (New Brunswick Scientific, Edison, NJ, USA) set at room temperature and 200 RPM. Extracts were filtered through a 25 mm 0.45 µm Millex Millipore (EMD Millipore, Merck KGaA, Darmstadt, Germany) LCR PTFE Syringe Filter after centrifugation at approximately 7500 x <em>g</em> on a Beckman J2-21M floor centrifuge with a JA-20.1 rotor (Beckman Coulter, Inc, Brea, CA, USA). For the UniformMu cross analyses, single kernels (∼0.2 g) were ground and extracted overnight in 5 mL 2% (<em>v/v</em>) formic acid as above. A 20 µL aliquot of each extract was injected into a Hitachi L-7200 HPLC (Hitachi High Technologies, Inc. Schaumburg, IL, USA) equipped with a Grace Prevail (W. R. Grace &amp; Co., Columbia, MD, USA) C<sub>18</sub> 5 µm analytical column (250 mm × 4.6 mm) and a Hitachi L-7455 Diode Array Detector quantifying absorbances at 520 nm. The column was heated to a constant 30.0°. The mobile phase consisted of 2% formic acid and 100% HPLC grade acetonitrile at a flow rate of 1 mL/min in the following linear gradient: 15% acetonitrile at 0 min, 30% acetonitrile at 10 min, and 15% acetonitrile at 15 min. After each sample, the column was allowed to equilibrate with 15% acetonitrile for 10 min. Samples were all run twice and averaged.</p></div><div id="sec-4" class="subsection"><h3>Identification of Anthocyanins</h3><p id="p-9">Pigment identification was inferred from the method used in <a id="xref-ref-26-5" class="xref-bibr" href="#ref-26">Paulsmeyer <em>et al.</em> (2017)</a>. A list of identified compounds in this study is in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>. One limitation of this method is that several unknown compounds would co-elute with peonidin 3-glucoside (Pn3G; <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). All peaks found under Pn3G would be summed together to simplify quantification. In addition, an unidentified peak arbitrarily labeled “ID #6” would appear next to pelargonidin 3-(6’’-malonyl) glucoside (Pg3MG; <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>). This peak was considered as a separate acylated anthocyanin compound because it eluted with other known acylated anthocyanins. This peak may be an isomer of an identified acylated anthocyanin.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/11/3669/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Structure of the most common anthocyanin molecules in maize. R1:H, R2:H, R3:H = Pelargonidin 3-Glucoside (Pg3G, ID #2) ; R1:OH, R2:H, R3:H = Cyanidin 3-Glucoside (C3G, ID #1); R1:CH3, R2:H, R3:H = Peonidin 3-Glucoside (Pn3G, ID #3); R1:H, R2:malonyl, R3:H = Pelargonidin 3-(6&#x2033;-malonyl)glucoside (Pg3MG, ID #5); R1:OH, R2:malonyl, R3:H = Cyanidin 3-(6&#x2033;-malonyl)glucoside (C3MG, ID #4); R1:CH3, R2:malonyl, R3:H = Peonidin 3-(6&#x2033;-malonyl)glucoside (Pn3MG, ID #7*); R1:H, R2:malonyl, R3:malonyl = Pelargonidin 3-(3&#x2033;,6&#x2033;-dimalonyl)glucoside (Pg3DMG, ID #8); R1:OH, R2:malonyl, R3:malonyl = Cyanidin 3-(3&#x2033;,6&#x2033;-dimalonyl)glucoside (C3DMG, ID #7*); R1:CH3, R2:malonyl, R3:malonyl = Peonidin 3-(3&#x2033;,6&#x2033;-dimalonyl)glucoside (Pn3DMG, not detected). *Pigments co-elute and cannot be separated." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2078375927" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Structure of the most common anthocyanin molecules in maize. R1:H, R2:H, R3:H = Pelargonidin 3-Glucoside (Pg3G, ID #2) ; R1:OH, R2:H, R3:H = Cyanidin 3-Glucoside (C3G, ID #1); R1:CH&lt;sub&gt;3&lt;/sub&gt;, R2:H, R3:H = Peonidin 3-Glucoside (Pn3G, ID #3); R1:H, R2:malonyl, R3:H = Pelargonidin 3-(6&#x2033;-malonyl)glucoside (Pg3MG, ID #5); R1:OH, R2:malonyl, R3:H = Cyanidin 3-(6&#x2033;-malonyl)glucoside (C3MG, ID #4); R1:CH&lt;sub&gt;3&lt;/sub&gt;, R2:malonyl, R3:H = Peonidin 3-(6&#x2033;-malonyl)glucoside (Pn3MG, ID #7*); R1:H, R2:malonyl, R3:malonyl = Pelargonidin 3-(3&#x2033;,6&#x2033;-dimalonyl)glucoside (Pg3DMG, ID #8); R1:OH, R2:malonyl, R3:malonyl = Cyanidin 3-(3&#x2033;,6&#x2033;-dimalonyl)glucoside (C3DMG, ID #7*); R1:CH&lt;sub&gt;3&lt;/sub&gt;, R2:malonyl, R3:malonyl = Peonidin 3-(3&#x2033;,6&#x2033;-dimalonyl)glucoside (Pn3DMG, not detected). *&lt;em&gt;Pigments co-elute and cannot be separated&lt;/em&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/11/3669/F1.medium.gif" width="440" height="266"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/8/11/3669/F1.medium.gif" width="440" height="266"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/11/3669/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/11/3669/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491285" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-10" class="first-child">Structure of the most common anthocyanin molecules in maize. R1:H, R2:H, R3:H = Pelargonidin 3-Glucoside (Pg3G, ID #2) ; R1:OH, R2:H, R3:H = Cyanidin 3-Glucoside (C3G, ID #1); R1:CH<sub>3</sub>, R2:H, R3:H = Peonidin 3-Glucoside (Pn3G, ID #3); R1:H, R2:malonyl, R3:H = Pelargonidin 3-(6″-malonyl)glucoside (Pg3MG, ID #5); R1:OH, R2:malonyl, R3:H = Cyanidin 3-(6″-malonyl)glucoside (C3MG, ID #4); R1:CH<sub>3</sub>, R2:malonyl, R3:H = Peonidin 3-(6″-malonyl)glucoside (Pn3MG, ID #7*); R1:H, R2:malonyl, R3:malonyl = Pelargonidin 3-(3″,6″-dimalonyl)glucoside (Pg3DMG, ID #8); R1:OH, R2:malonyl, R3:malonyl = Cyanidin 3-(3″,6″-dimalonyl)glucoside (C3DMG, ID #7*); R1:CH<sub>3</sub>, R2:malonyl, R3:malonyl = Peonidin 3-(3″,6″-dimalonyl)glucoside (Pn3DMG, not detected). *<em>Pigments co-elute and cannot be separated</em>.</p><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/11/3669/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="HPLC chromatograms of wild type and reduced acylation samples. A) Wild type profile with the structure of cyanidin 3-(6&#x2019;&#x2019;-malonyl) glucoside inset. B) Profile of a reduced acylation mutant with the structure of cyanidin 3-glucoside inset. Note: Peak labels correspond to the &#x201C;ID #&#x201D; listed in Figure 1." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2078375927" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;HPLC chromatograms of wild type and reduced acylation samples. A) Wild type profile with the structure of cyanidin 3-(6&#x2019;&#x2019;-malonyl) glucoside inset. B) Profile of a reduced acylation mutant with the structure of cyanidin 3-glucoside inset. &lt;em&gt;Note&lt;/em&gt;: Peak labels correspond to the &#x201C;ID #&#x201D; listed in Figure 1.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/11/3669/F2.medium.gif" width="440" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/8/11/3669/F2.medium.gif" width="440" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/11/3669/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/11/3669/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491389" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-11" class="first-child">HPLC chromatograms of wild type and reduced acylation samples. A) Wild type profile with the structure of cyanidin 3-(6’’-malonyl) glucoside inset. B) Profile of a reduced acylation mutant with the structure of cyanidin 3-glucoside inset. <em>Note</em>: Peak labels correspond to the “ID #” listed in <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-5" class="subsection"><h3>Phenotypic Evaluation</h3><p id="p-12">The Hitachi HPLC System Manager 4.0 software was used to integrate chromatogram peak areas. Phenotypes for individual anthocyanin compounds used these peaks areas as a measure. Anthocyanin content (AC) was quantified using the <em>Maíz Morado</em> external standard method in <a id="xref-ref-26-6" class="xref-bibr" href="#ref-26">Paulsmeyer <em>et al.</em> (2017)</a>. The AC of Angelina’s Gourmet Maize Morado (Swanson, CT, USA) was determined to be 1000 mg/kg using cyanidin 3-glucoside (C3G) standard purchased from Phytolab GmbH &amp; Co. (Vestenbergsgreuth, Germany) in a concentration gradient of 1 – 1000 µg/ml. The percentage of acylated anthocyanins of total anthocyanins was chosen as the phenotype to best describe reduced acylation mutants. Acylation percentage was calculated by summing peak areas of all acylated compounds in the sample, then dividing those by the total integrated area of the chromatogram and multiplying by 100. Alleles of <em>R1</em> were scored as 1/0 for the presence or absence of speckled kernels, respectively. The speckling phenotype is due to imprinting and forms when aleurone receives one <em>R1</em> allele from the pollen parent in a female <em>r1r1</em> endosperm (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Kermicle 1970</a>). The presence of recessive <em>C1</em> alleles was scored if the sample had yellow kernels without speckling, or if the proportion of colorless kernels per ear was approximately 50%.</p></div><div id="sec-6" class="subsection"><h3>GBS Library Construction</h3><p id="p-13">Tissue from six to eight germinating F<sub>2:3</sub> seedlings was collected into 1.2 mL microtiter tubes from the developing leaf whorl one week after planting. The six to eight F<sub>2:3</sub> kernels used to represent F<sub>2</sub> plants were chosen to emulate the sample from which they came in terms of pigmentation. A CTAB extraction protocol optimized for 96-well plates was used to isolate DNA. DNA quantification was completed using the PicoGreen assay (Molecular Probes, Eugene, OR). The GBS protocol outlined by <a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Elshire <em>et al.</em> (2011)</a> was used for library construction with a double restriction enzyme modification from <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Poland <em>et al.</em> (2012)</a>. Restriction enzymes chosen for library construction were <em>Hin</em>P1I and <em>Pst</em>I. In addition to the 129 samples from the reduced acylation mapping population, 201 other samples from an independent experiment were multiplexed during sequencing for 330 total samples. The independent population was for a separate mapping population not described here. Libraries were read in one sequencing lane on an Illumina HiSeq 2500 System (San Diego, CA, USA) at the Roy J. Carver Biotechnology Center at the University of Illinois in Urbana, IL, USA.</p></div><div id="sec-7" class="subsection"><h3>SNP Discovery Pipeline</h3><p id="p-14">Genetic markers in the population were generated using the TASSEL 5.0 GBS v2 pipeline (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bradbury <em>et al.</em> 2007</a>). Bowtie 2 (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Langmead and Salzberg 2012</a>) was used to align fragments to <em>Zea mays</em> B73 reference genome version 3 (<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Youens-Clark <em>et al.</em> 2011</a>). Minor allele frequency (MAF) was set to 0.05. This resulted in 4,660,000 total SNPs for all 330 samples. Missing data ranged from 13.2 to 99.7% per sample. Samples with greater than 80% missing data were excluded. This changed the dataset to 328 samples, with one sample being lost from the reduced acylation population. The overall proportion of missing sites in the dataset was 36.32%. Next, insertion-deletions were removed and imported into Beagle 4.1 (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Browning and Browning 2016</a>). The options defined in Beagle 4.1 were a window length of 200 and a step size of 50. After imputation, the reduced acylation population was separated from the independent mapping population within TASSEL 5.0. A MAF of 0.05 was applied once again to the imputed SNPs resulting in a total of 8062 SNPs per sample for the reduced acylation population. Markers per chromosome ranged from 584 for Chromosome 10 to 1212 for Chromosome 1. Alleles were converted in TASSEL 5.0 to numerical genotypes 1 <em>vs.</em> 0 for major <em>vs.</em> minor or 0.5 for heterozygotes based on allele frequency in the population. Since 707G was a genetic stock and not completely inbred, some sites had three to four alleles, which were considered minor alleles. Numerical genotypes were multiplied by two to remove decimals and imported into R (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">R Core Team 2015</a>) to perform QTL analyses.</p></div><div id="sec-8" class="subsection"><h3>Statistical Analyses</h3><p id="p-15">To determine if genes of interest (<em>C1</em>, <em>In1</em>, <em>P1</em>, <em>R1</em>, and the reduced acylation mutant) were significantly altering the concentrations of individual pigments or total AC, ANOVAs were calculated using either Proc Reg or Proc Mixed in SAS 9.4 (SAS institute Inc., Cary, NC, USA) depending on the outputs needed. <em>C1</em>, <em>P1</em>, <em>R1</em>, and reduced acylation mutants were coded as 0 or 1 depending on if the sample contained dominant or homozygous recessive alleles, respectively, based on the phenotypes previously described. Genotypes of <em>In1</em> were inferred from alleles at the most significant site in the QTL analysis for log-transformed AC. All genes of interest were included in a linear model in Proc Reg to determine significance. Genes that were statistically significant were regressed to AC in Proc Reg to estimate the change in AC attributed by the significant genes. Variance components for the statistically significant genes of interest and the proportion of variation attributed by each gene were calculated in Proc Mixed with method equal to Type 3 using each gene as a random independent variable. All QTL analyses were run using the linear model (lm) function in R (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">R Core Team 2015</a>) using phenotypes as the dependent and SNPs as the independent variable. A stepwise QTL analysis was used to find significant markers involved with the phenotype of interest. The first model run would assume a single QTL. The most significant marker from a single-QTL model would then be used as a covariate (<strong>Q</strong>) and the process repeated until no more markers were significant. The general model for stepwise QTL analysis is shown in Equation 1. The term <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3669/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3669/embed/mml-math-1.gif"/></noscript></span></span>represents the summation of the <em>k</em>th significant marker as a covariate for <em>p</em> covariates, while <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3669/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3669/embed/mml-math-2.gif"/></noscript></span></span> represents the numerical genotypes at a SNP site.<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3669/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3669/embed/mml-math-3.gif"/></noscript></span></span>The threshold for significant marker <em>p</em>-values in Equation 1 was determined by a Bonferroni correction of α = 0.05/<em>n</em> where <em>n</em> is the number of SNPs (<em>n</em> = 8062) and 0.05 is 95% confidence. All figures were generated in R (<a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">R Core Team 2015</a>). All supplementary material, custom code, and datasets are available at <a href="https://figshare.com/projects/Discovery_of_Anthocyanin_Acyltransferase1_AAT1_in_maize_using_genotyping-by-sequencing_GBS_/38900">https://figshare.com/projects/Discovery_of_Anthocyanin_Acyltransferase1_AAT1_in_maize_using_genotyping-by-sequencing_GBS_/38900</a>.</p></div><div id="sec-9" class="subsection"><h3>Data availability</h3><p id="p-16">The authors affirm that all data necessary for confirming the conclusions of this article are represented fully within the article and its tables and figures. Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.7108748">https://doi.org/10.25387/g3.7108748</a>.</p></div></div><div class="section results" id="sec-10"><h2 class="">Results</h2><div id="sec-11" class="subsection"><h3>Inheritance of the Trait</h3><p id="p-17">Ten accessions have been found to date that exhibit the reduced acylation trait. Pedigree analysis determined that all the reduced acylation MGCSC stocks are all derived from E. Coe’s <em>in1</em> genetic stocks developed in 1965 (E. Coe and P. Stinard, personal communication, August 1, 2018). The remaining stocks (50%) are all from diverse backgrounds suggesting that the mutation has occurred several times independently. The reduced acylation trait has no visual effect on the kernel phenotype, so selection for this trait is apparently indirect in these accessions. Reciprocal crosses of reduced acylation accessions to wild-type accessions returned normal acylation abundance meaning the trait is recessive to normal acylation. Several reduced acylation accessions were then crossed with each other to see if the trait was determined by the same loci in every accession. Accessions Ames 14276, Jerry Peterson Blue Dent, M142X, M741I, and X19EA failed to return normal acylation abundance when crossed to mapping population parent 707G indicating they are allelic.</p></div><div id="sec-12" class="subsection"><h3>Summary of Mapping Population Phenotypes</h3><p id="p-18">In the mapping population, 129 F<sub>2</sub> individuals generated kernels for phenotyping and sequencing. Prior to sequencing, the reduced acylation mutants in the population were defined visually based on trends in the histogram for percentage of acylation (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). A few samples were intermediate in acylation percentage but the cutoff for a reduced acylation mutant was defined as less than 50% acylation based on the bimodal distribution seen in the histogram. The summary of all visual phenotypes and inferred <em>In1</em> genotypes is in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>. There were 32 progeny in the population displaying the reduced acylation phenotype, which is exactly the number of recessive samples expected for a population segregating at a single, recessive locus. The overall percentage of acylation in the wild-type samples averaged 70.8% while the mutant samples maintained an average of 15.0%. The overall population mean for AC was 96.2 mg/kg. The distribution of AC was skewed right (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>) indicating that <em>in1</em> had a profound affect on AC in this population. Since a new HPLC method was used in this study, the coefficient of variation (CV) was calculated to determine repeatability. Coefficient of variation was calculated by dividing the standard deviation of AC for the replicates by the sample mean. The overall average CV was 3.09% confirming the method is acceptably reproducible. To see which loci had an effect on AC, a linear regression was modeled using all loci as random, independent variables. All variables except <em>P1</em> were significant at <em>P</em> &lt; 0.05. Parameter estimates from the model calculate that <em>in1</em> raises AC an estimated 105.4 mg/kg while reduced acylation, <em>r1</em>, and <em>c1</em> lower AC 23.3 mg/kg, 24.2 mg/kg, and 29.1 mg/kg, respectively. The lowest AC sample was 11.84 mg/kg in a <em>C1c1/R1r1/In1X</em> reduced acylation plant and the highest was 369.54 mg/kg in a <em>C1C1/R1R1/in1in1</em> wild-type plant.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/11/3669/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Histograms of two important phenotypes in the reduced acylation mapping population. (Left) anthocyanin content and (Right) percentage of acylation." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2078375927" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Histograms of two important phenotypes in the reduced acylation mapping population. (Left) anthocyanin content and (Right) percentage of acylation.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/11/3669/F3.medium.gif" width="440" height="110"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/8/11/3669/F3.medium.gif" width="440" height="110"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/11/3669/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/11/3669/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491428" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-19" class="first-child">Histograms of two important phenotypes in the reduced acylation mapping population. (Left) anthocyanin content and (Right) percentage of acylation.</p><div class="sb-div caption-clear"></div></div></div><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/491355/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/491355/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">SUMMARY OF PHENOTYPES IN THE MAPPING POPULATION</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>Identification of Candidate Anthocyanin Acyltransferases</h3><p id="p-20">Since the most likely candidate for an AAT in maize should have anthocyanidin dimalonyltransferase activity, the dimalonyltransferase from chrysanthemum, Dm3MaT2 (<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Suzuki <em>et al.</em> 2004</a>), was used as a query to find candidate AATs in blastp. The blastp search found seventeen candidates in maize with 25% or greater identity and 15% or greater query coverage in the reference proteins database. The seventeen candidates were aligned in COBALT (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Papadopoulos and Agarwala 2007</a>) using defaults. The candidates all showed close homology to the three motifs that are present in acyltransferases: HXXXD (Motif 1), DFGWG (Motif 2), and NYFGNC (Motif 3) (<a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">Suzuki <em>et al.</em> 2002</a>). Motif 3 is an AAT-specific motif. Protein sequences were filtered on whether they contained Motif 3 present in chrysanthemum’s three functional anthocyanin malonyltransferases: XYF/LGNC (<a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">Suzuki <em>et al.</em> 2004</a>). This filter resulted in nine candidate sequences (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). Among these candidates, four were annotated as predicted AATs. One was even annotated as an anthocyanidin 3-<em>O</em>-glucoside 6’’-<em>O</em>-malonyltransferase. The remaining five candidates were uncharacterized proteins or transferase family proteins.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/491369/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/491369/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">CANDIDATE AATs IN MAIZE AFTER FILTERING</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>QTL Analyses of Mapping Population Phenotypes</h3><p id="p-22">The GBS library constructed from the 129 individuals of this mapping population resulted in 128 genotypes with 8062 SNPs per genotype. SNPs were more prevalent in areas near the ends of chromosomes, but generally, the entire genome was represented. The average distance between SNPs in the final dataset was 255 Kb. Distance between SNPs under 1 Mb apart accounted for 95% of the dataset while 17.8% of those were under 1 Kb apart. Several control traits were mapped to assess the accuracy and robustness of the SNP dataset before the reduced acylation trait was mapped. The control loci—<em>C1</em> (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Paz-Ares <em>et al.</em> 1987</a>), <em>In1</em> (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Burr <em>et al.</em> 1996</a>), <em>P1</em> (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Grotewold <em>et al.</em> 1994</a>), and <em>R1</em> (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Dellaporta <em>et al.</em> 1988</a>)—segregating in the population have been previously mapped, sequenced, and characterized. The log-transformation of AC was used as the phenotype to map <em>In1</em>, since the distribution was skewed (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>). <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a> shows the Manhattan plots generated from the four control traits. The most significant SNP correlations always corresponded to the known location of each gene. The most significant marker for <em>In1</em>, with a –log10 <em>p</em>-value of 21.36, was only 134 bp away from coding sequence. The most significant SNPs for loci <em>P1</em>, <em>C1</em>, and <em>R1</em> were 642 Kb, 1.7 Mb, and 637 Kb away, respectively (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Andorf <em>et al.</em> 2010</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/11/3669/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Manhattan plots for known genes segregating in the mapping population. All major QTL correspond to the known location of each gene (Andorf et al. 2010)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2078375927" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Manhattan plots for known genes segregating in the mapping population. All major QTL correspond to the known location of each gene (Andorf &lt;em&gt;et al.&lt;/em&gt; 2010).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/11/3669/F4.medium.gif" width="440" height="219"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/8/11/3669/F4.medium.gif" width="440" height="219"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/11/3669/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/11/3669/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491251" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-23" class="first-child">Manhattan plots for known genes segregating in the mapping population. All major QTL correspond to the known location of each gene (<a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">Andorf <em>et al.</em> 2010</a>).</p><div class="sb-div caption-clear"></div></div></div><p id="p-24">The control genes validated the robustness of the genotype dataset so that the reduced acylation phenotype could be mapped with high confidence. The phenotype chosen to represent the reduced acylation trait was percentage of acylation, since proportions of compounds were not affected by <em>in1</em>. The most significant marker with a –log10 <em>p</em>-value of 18.7 was at position 298,042,277 of Chromosome 1 (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). The Bonferroni correction established a 22.3 Mb interval of significant markers from positions 279,188,074 to 301,476,924 (the end of Chromosome 1). The significant genomic region for the reduced acylation trait spanned only one of the candidate genes from <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>. The candidate transferase gene designated GRMZM2G387394 in B73 reference genome version 3 is at position 300,173,138 of Chromosome 1 (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/11/3669/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Reduced acylation locus Manhattan plot." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2078375927" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Reduced acylation locus Manhattan plot.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/11/3669/F5.medium.gif" width="440" height="218"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/8/11/3669/F5.medium.gif" width="440" height="218"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/11/3669/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/11/3669/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491348" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-25" class="first-child">Reduced acylation locus Manhattan plot.</p><div class="sb-div caption-clear"></div></div></div><p id="p-26">Pigment concentration of the grain is a trait breeders developing high anthocyanin-yielding varieties are seeking to improve. To find genomic regions associated with AC other than the transcription factors known, a model was fit with <em>In1</em> genotypes as a covariate and both log-transformed AC and raw AC as responses using Equation 1. No significant markers could be distinguished from both models at the significance threshold established by the highly conservative Bonferroni correction, so the significance threshold was relaxed to <em>P &lt;</em> 1×10<sup>−4</sup> in accordance with <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Li <em>et al.</em> (2016)</a>. With untransformed AC, a SNP on Chromosome 7 was most significant at position 144,238,875 (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>). Near this marker is <em>MYB152</em>, a transcription factor that has been shown to correlate with <em>PAL</em> expression (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Agarwal <em>et al.</em> 2016</a>). When both <em>In1</em> and the SNP on Chromosome 7 were considered, SNP position 137,169,800 on Chromosome 9 became significant (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a>). No obvious gene candidates surrounding this marker could be found at this time.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/11/3669/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Manhattan plots for step-wise QTL analysis of anthocyanin content. Left:anthocyanin content QTL with in1 as a covariate and a P &lt; 1&#xD7;10-4 significance threshold. Right: anthocyanin content QTL with in1 and S7_144238875 covariates and a P &lt; 1&#xD7;10-4 significance threshold." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2078375927" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Manhattan plots for step-wise QTL analysis of anthocyanin content. &lt;em&gt;Left&lt;/em&gt;:anthocyanin content QTL with &lt;em&gt;in1&lt;/em&gt; as a covariate and a &lt;em&gt;P&lt;/em&gt; &lt; 1&#xD7;10-4 significance threshold. &lt;em&gt;Right&lt;/em&gt;: anthocyanin content QTL with &lt;em&gt;in1&lt;/em&gt; and S7_144238875 covariates and a &lt;em&gt;P&lt;/em&gt; &lt; 1&#xD7;10-4 significance threshold.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/11/3669/F6.medium.gif" width="440" height="106"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/8/11/3669/F6.medium.gif" width="440" height="106"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/11/3669/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/11/3669/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491379" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-27" class="first-child">Manhattan plots for step-wise QTL analysis of anthocyanin content. <em>Left</em>:anthocyanin content QTL with <em>in1</em> as a covariate and a <em>P</em> &lt; 1×10-4 significance threshold. <em>Right</em>: anthocyanin content QTL with <em>in1</em> and S7_144238875 covariates and a <em>P</em> &lt; 1×10-4 significance threshold.</p><div class="sb-div caption-clear"></div></div></div><p id="p-28">In addition to AC, QTL analyses were run on individual compounds distinguished by HPLC. All compounds besides Pn3G and pelargonidin 3-(3′’,6’’-dimalonyl) glucoside (Pg3DMG) had two significant QTL corresponding to <em>AAT1</em> and <em>In1</em> (Figure S2). Pn3G and Pg3DMG had one significant QTL at the site of <em>In1</em>. When Pn3G was modeled with <em>In1</em> as a covariate using Equation 1, SNP markers spanning <em>R1</em> became significant. Only one significant locus could be found with Pg3DMG. The inability to detect more significant regions was most likely because only 26 samples had detectable amounts of Pg3DMG.</p><p id="p-29">Variance components for each AC-altering locus were estimated for all anthocyanin phenotypes to determine the phenotypic variation attributed to each loci (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). The gene that explained the most variance in AC for this population was <em>In1</em> (64.7%), while <em>R1</em>, <em>C1</em>, and the reduced acylation trait accounted for 3.02%, 4.56%, and 2.67% of the total variance, respectively. In general, <em>In1</em> explained the highest proportion of variance for all compounds and ranged from 28.1 to 56.5%. The lowest proportion of variance that could be detected as a significant QTL was 5.55% for <em>R1</em> with Pn3G.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/491411/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/491411/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">PROPORTION OF VARIANCE ATTRIBUTED TO EACH CONTROL LOCI FOR EACH ANTHOCYANIN COMPOUND</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-15" class="subsection"><h3>QTL Simulation</h3><p id="p-31">The dataset was sampled and simulated to determine the lowest number of samples possible to confirm a QTL. The two traits assessed were the presence of speckled kernels (<em>R1</em> phenotype, Figure S1) and percentage of acylation. Speckled kernels were represented in 75 of the 128 genotyped samples, while 32 of the 128 genotyped samples were determined to have the reduced acylation trait. Subsamples of 20, 40, 60, 80, and 100 genotypes were selected at random from the dataset. Sampling was repeated 100 times for each subsample of genotypes. The ability to detect QTL was arbitrarily defined as the most significant SNP above the Bonferroni significance threshold (<em>P</em> &lt; 6.203×10<sup>−6</sup>) being only 10 Mb away from the known locus. The summary of the simulation is presented in <a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>. For the <em>R1</em> locus, forty samples were sufficient to pinpoint <em>R1</em> 100% of the time within 10 Mb. The average distance of the most significant SNP from the <em>R1</em> locus for the forty samples was 1.62 Mb away. Of the 20-genotype simulations, 39% significantly pinpointed the locus within 10 Mb, while 36% were considered false positives. The average <em>p</em>-value for the significant markers ranged from a –log10 <em>p</em>-value of 10.72 to 63.47. The high <em>p</em>-values associated with the 20-genotype subsamples may be due to ascertainment bias. For percentage of acylation, the QTL could be detected 100% of the time with only sixty samples. The average distance from the candidate acyltransferase for sixty samples was 2.27 Mb. In 37% of the simulations, the reduced acylation QTL could be mapped to the correct location with only twenty genotypes. One simulation was able to establish significance with only three of twenty reduced acylation mutants. The average significance for percentage of acylation ranged from 6.57 to 15.15 –log10 <em>p</em>-value. The reduced acylation simulation had more false positives then the <em>R1</em> locus. For the 40-genotype and 20-genotype subset, 6% and 25% of the significant markers were considered false positives, respectively.</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/491298/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/491298/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">QTL SIMULATION STUDY</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-16" class="subsection"><h3>UniformMu Knockout of GRMZM2G387394</h3><p id="p-33">To determine if the candidate acyltransferase on Chromosome 1 was associated with the formation of acylated anthocyanins, a UniformMu stock (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">McCarty <em>et al.</em> 2005</a>), designated UfMu-09775, with a <em>Mutator</em> (<em>Mu</em>) transposon insertion within GRMZM2G387394 was crossed to reduced acylation mutant 707G. UfMu-09775 was heterozygous for the insertion and would theoretically confer the mutant phenotype in 50% of the kernels crossed to 707G. In the cross, 14 of 25 F<sub>1</sub> kernels exhibited the reduced acylation phenotype, which is indicative of the 50% ratio expected (χ<sup>2</sup> test, H<sub>0</sub> = 50%, <em>P</em> = 0.5485). Therefore, it can be concluded that GRMZM2G387394 is at least required for anthocyanidin 3-<em>O</em>-glucoside 6’’-<em>O</em>-malonyltransferase function in maize and is responsible for the reduced acylation phenotype. Given this knowledge, GRMZM2G387394 from now on will be designated <em>Anthocyanin Acyltransferase1</em> or <em>AAT1</em>.</p></div><div id="sec-17" class="subsection"><h3>Phylogenetics of Acyltransferase Members</h3><p id="p-34">At least nineteen flavonoid acyltransferases have been characterized to date (<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Unno <em>et al.</em> 2007</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Kim <em>et al.</em> 2009</a>). All characterized flavonoid acyltransferases are from eudicot species with the exception of one flavonoid malonyltransferase in <em>Oryza sativa</em> (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Kim <em>et al.</em> 2009</a>). The bootstrap consensus neighbor-joining phylogenetic tree constructed in Mega7 of sixteen known flavonoid acyltransferases with the maize candidate acyltransferase is shown in Figure S3 (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Felsenstein 1985</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Kumar <em>et al</em>. 2016</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Saito and Nei 1987</a>). Monocots formed their own clade and the phylogenetic tree otherwise matched previously reported phylogenies (<a id="xref-ref-36-3" class="xref-bibr" href="#ref-36">Unno <em>et al.</em> 2007</a>). The monocot clade is more closely related to the other malonyltransferases than the 5-<em>O</em>-glucoside or aromatic AATs. <a id="xref-ref-36-4" class="xref-bibr" href="#ref-36">Unno <em>et al.</em> (2007)</a> demonstrated that Thr-35, Leu-37, Ala-38, and Gln-51 are all important acyl-acceptor binding sites in Dm3MaT2. In maize, AAT1 Leu-37 and Ala-38 are conserved, while Gln-51 is functionally similar to Ser-69 in AAT1. Thr-35 is not present in AAT1. Three additional amino acids in Dm3MaT2—Tyr-411, Ala-413, and Lys-419—were required for AAT function as well (<a id="xref-ref-36-5" class="xref-bibr" href="#ref-36">Unno <em>et al.</em> 2007</a>). All but Lys-419 had structurally similar counterparts in AAT1. Due to the low identity between AAT1 and Dm3MaT2, functional domains of AATs can be inferred and should be investigated for future studies.</p></div></div><div class="section discussion" id="sec-18"><h2 class="">Discussion</h2><p id="p-35">The reduced acylation phenotype in maize is due to anthocyanin acyltransferase <em>AAT1</em> and had a significant effect on AC in the mapping population. This finding agrees with the hypothesis that aliphatic acylation has a role in protecting the anthocyanin molecule. Therefore, increasing the proportion of acylated anthocyanins could have a positive influence on increasing AC <em>in vivo</em>. In addition to the reduced acylation mutant allele of <em>aat1</em>, <em>In1</em>, <em>C1</em>, and <em>R1</em> all have a significant effect on AC. The quantifiable change in AC associated with alleles of each of these anthocyanin regulators has, to the best of our knowledge, never been studied. Here only pigmented kernels were sampled for <em>C1</em> and <em>R1</em>, which will bias the results toward higher AC. Despite this, significant differences could be found among the combinations of each gene regulator. The effect of <em>In1</em> may be inexact, given the genotypes of <em>In1</em> were inferred from low-coverage SNP sites where heterozygotes may have been miscalled. The high significance (–log10 <em>p</em>-value 21.36) and relative proximity (134 bp away from the transcription start site) of the most significant marker for log-transformed AC along with the approximate 1:3 Mendelian ratio of homozygous recessive <em>in1</em> genotypes to dominant <em>In1</em> genotypes provide evidence that inferred <em>In1</em> genotypes may be accurate.</p><p id="p-36">GBS is a simple and robust method for reduced representation sequencing. The ability to multiplex 384 or more samples per sequencing run greatly reduces costs. In addition, all steps from DNA extraction to library constriction can be completed in 96-well plate format, which helps maximize efficiency. In this study, 330 samples from two independent mapping populations were analyzed in one sequencing lane. A problem with many highly multiplexed GBS datasets is the amount of missing and incomplete data due to low-coverage sequencing (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Swarts <em>et al.</em> 2014</a>). In the initial dataset with 330 samples, two samples were discarded that had missing data over 80%. These samples most likely failed in the PCR or ligation steps of GBS library preparation. The amount of missing data has implications on how imputation fills in SNPs. Despite the uncertainty in variant calling, imputation has been shown to increase power to detect QTL (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Fragoso <em>et al.</em> 2016</a>). Missing data not only has an effect of imputation but also on heterozygote calling. Phenotypes may not correlate with SNP markers even though flanking markers are highly significant. This issue is apparent in Manhattan plots. Log-transformed AC had the largest QTL and spanned 269 SNPs. Of those, only 155 SNPs were significant with the Bonferroni significance threshold. In contrast, ascertainment bias from low-coverage sequences resulted in some false positive associations for some control traits. The conservative Bonferroni threshold should control false positives, but even in <em>P1</em>, a single marker on a different chromosome was significant (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>). Overall, the effect of missing data and incorrectly called bases is seemingly negligible for this study since the most significant SNP markers for each control loci aligned correctly and only a single significant region was found for the reduced acylation trait.</p><p id="p-37">Numerous significant marker associations may be a problem if several candidate genes resided within a single QTL. In this study, the QTL for reduced acylation only spanned a single candidate, so the width was acceptable. In the future, a new mapping approach based on linkage between markers, like bin-mapping, may be necessary to establish thresholds to differentiate candidates and establish a narrower confidence interval (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Chen <em>et al.</em> 2014</a>). Recombination rate in this population is partially to blame for the large number of significant markers. With more recombination events, smaller areas in the genome will remain correlated to the trait due to the breakage of linkage blocks. More recombination events can be introduced with a greater population size. For example, in a study that analyzed F<sub>2</sub> progeny for grain yield traits, the <em>R1</em> gene was also mapped as a control, but used 611 progeny. <em>R1</em> was fine-mapped to an approximately 700 Kb confidence interval, as opposed to the 27.0 Mb interval between significant markers presented here (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Chen <em>et al.</em> 2014</a>). In addition to increasing population size, intermating the F<sub>2</sub> genotypes could have been a way to increase recombination events. This scheme was utilized to develop the intermated B73 x Mo17 (IBM) population in maize that is used extensively for mapping traits (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Lee <em>et al.</em> 2002</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Eichten <em>et al.</em> 2011</a>). Intermating requires additional generations, which is one reason it was not utilized here.</p><p id="p-38">Despite the low SNP coverage relative to other GBS studies, the mapping population was able to map traits with one to three loci. The simulation here showed that QTL could be confirmed with as little as forty to sixty samples depending on the proportion of the recessive allele (<a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a>). The power to correctly detect QTL is best when the dominant/recessive phenotypes are in nearly even proportions. The high power of detection presented here is in direct contrast to a previous study that could only establish significance for a single-locus trait at a chromosome-wide scale with 91 samples (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Heffelfinger <em>et al.</em> 2014</a>).</p><p id="p-39">For quantitative traits, the ability to detect multiple QTL is important. More than three QTL could not be found in this mapping population since <em>in1</em> and <em>aat1</em> explained most of the variability in most traits. The possible number of QTL all depends on the nature of the trait of interest. If the trait is highly heritable and controlled by many genes with smaller effects, more than three significant associations may be possible. For example, <a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">Chen <em>et al.</em> (2014)</a> were able to distinguish seven QTL correlated with tassel branch number in a large population of 692 F<sub>2</sub> plants. The proportion of variance explained by the QTL in this trait ranged from 2.0 to 6.3%, which was the typical proportion of variance of <em>R1</em> and <em>C1</em> in the reduced acylation population. Moreover, another study utilizing GBS for quantitative traits found 35 QTL for plant height, ear height, and internode number with phenotypic variances ranging from 2.6 to 15.68% with 314 recombinant inbred lines (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Zhou <em>et al.</em> 2016</a>). Plant height, ear height, and tassel branch number are highly heritable traits as anthocyanin composition is expected to be (<a id="xref-ref-26-7" class="xref-bibr" href="#ref-26">Paulsmeyer <em>et al.</em> 2017</a>). Future work aimed at increasing AC would have to expand the mapping population to confirm the presence of the smaller effect QTL found in this study (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6</a>).</p><div id="sec-19" class="subsection"><h3>Conclusions</h3><p id="p-40">The anthocyanin acyltransferase in maize, <em>AAT1</em>, synthesizes the majority of acylated anthocyanins, which are the most common anthocyanin compounds in maize. As confirmed in this study, acylated anthocyanins are important for increasing AC <em>in vivo</em> and may be important for use in natural colorants or for human health. Results of this study also demonstrate the effectiveness of utilizing GBS for discovering new genes. GBS is a simple and reliable high-throughput protocol that allows for multiplexing of numerous samples. In this study, GBS was able to discover a new gene in maize involved with the production of acylated anthocyanins, even with a small mapping population size that was highly multiplexed in a single sequencing lane. Knockouts of <em>AAT1</em> generated by a <em>Mu</em> transposon insertion returned the reduced acylation phenotype. With this it could be concluded that <em>AAT1</em> is the gene responsible for the reduced acylation trait. Whether or not this gene has dimalonyltransferase activity is yet to be determined. Current work is being aimed at expressing recombinant protein and assaying enzyme activity. Overall, the discovery of <em>AAT1</em> increases our understanding of the family of AATs and fills in missing steps of the anthocyanin biosynthetic pathway in maize.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-41">This work was supported by a grant from the Kraft Heinz Company of Glenview, Illinois and from DD Williamson, Inc. of Louisville, Kentucky. Fellowship support for MNP was provided by the Illinois Corn Grower’s Association. MNP developed, genotyped, and phenotyped the F<sub>2</sub> population, interpreted the phenotype/genotype data, and drafted the manuscript. PJB assisted in genotyping the population and interpreting genotype data. JAJ assisted in conceiving the study and revised the manuscript. All authors read and approved the manuscript. We would like to thank Brown Lab members Shilpa Manjunatha and Pradeep Hirannaiah for walking us through DNA extractions and GBS library construction. Additionally, we want to thank Pradeep for providing the code used for the TASSEL 5.0 GBS v2 pipeline, Bowtie 2, and Beagle 4.1. Thanks to Laura Chatham for the photos used in Figure S1C and S1D. Finally, thanks to Dr. Tony Studer for assistance in preparing the manuscript.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-4"><p id="p-42">Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.7108748">https://doi.org/10.25387/g3.7108748</a>.</p></li><li class="fn" id="fn-5"><p id="p-43"><em>Communicating editor: M. Scanlon</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2018-08-01"><span class="received-label">Received </span>August 1, 2018.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2018-09-18"><span class="accepted-label">Accepted </span>September 18, 2018.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2018 Paulsmeyer <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3669.1" data-doi="10.1038/srep28502"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Agarwal</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grotewold</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doseff</span>  <span class="cit-name-given-names">A. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gray</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">MYB31/MYB42 syntelogs exhibit divergent regulation of phenylpropanoid genes in maize, sorghum and rice.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">28502</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/srep28502</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D6%26rft.spage%253D28502%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep28502%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/srep28502&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3669.2" data-doi="10.1093/bioinformatics/btp556"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Andorf</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lawrence</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harper</span>  <span class="cit-name-given-names">L. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schaeffer</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Campbell</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The Locus Lookup tool at MaizeGDB: identification of genomic regions in maize by integrating sequence information with physical and genetic maps.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">434</span>–<span class="cit-lpage">436</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btp556</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp556%26rft_id%253Dinfo%253Apmid%252F20124413%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp556&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20124413&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000274342800029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3669.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bakowska-Barczak</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Acylated anthocyanins as stable, natural food colorants - a review.</span> <abbr class="cit-jnl-abbrev">Pol. J. Food Nutr. Sci.</abbr> <span class="cit-vol">14/15</span>: <span class="cit-fpage">107</span>–<span class="cit-lpage">116</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPol.%2BJ.%2BFood%2BNutr.%2BSci.%26rft.volume%253D1415%26rft.spage%253D107%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3669.4" data-doi="10.1093/bioinformatics/btm308"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bradbury</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kroon</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casstevens</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramdoss</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">TASSEL: software for association mapping of complex traits in diverse samples.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">2633</span>–<span class="cit-lpage">2635</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtm308%26rft_id%253Dinfo%253Apmid%252F17586829%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btm308&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17586829&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000250673800021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3669.5" data-doi="10.1016/j.ajhg.2015.11.020"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>  <span class="cit-name-given-names">B. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>  <span class="cit-name-given-names">S. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genotype imputation with millions of reference samples.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">98</span>: <span class="cit-fpage">116</span>–<span class="cit-lpage">126</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ajhg.2015.11.020</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BHum.%2BGenet.%26rft.volume%253D98%26rft.spage%253D116%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2015.11.020%26rft_id%253Dinfo%253Apmid%252F26748515%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2015.11.020&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26748515&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3669.6" data-doi="10.1105/tpc.8.8.1249"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burr</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burr</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scheffler</span>  <span class="cit-name-given-names">B. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blewitt</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wienand</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">The maize repressor-like gene Intensifier1 shares homology with the R1/B1 multigene family of transcription factors and exhibits missplicing.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1249</span>–<span class="cit-lpage">1259</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1105/tpc.8.8.1249</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2BCell%2BOnline%26rft.stitle%253DPlant%2BCell%26rft.aulast%253DBurr%26rft.auinit1%253DF.%2BA.%26rft.volume%253D8%26rft.issue%253D8%26rft.spage%253D1249%26rft.epage%253D1259%26rft.atitle%253DThe%2BMaize%2BRepressor-like%2BGene%2Bintensifier1%2BShares%2BHomology%2Bwith%2Bthe%2Br1%252Fb1%2BMultigene%2BFamily%2Bof%2BTranscription%2BFactors%2Band%2BExhibits%2BMissplicing%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.8.8.1249%26rft_id%253Dinfo%253Apmid%252F8776895%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjg6IjgvOC8xMjQ5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMS8zNjY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3669.7" data-doi="10.1186/1471-2164-15-433"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dong</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ren</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">An ultra-high density bin-map for rapid QTL mapping for tassel and ear architecture in a large F2 maize population.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">433</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2164-15-433</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D433%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-433%26rft_id%253Dinfo%253Apmid%252F24898122%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-433&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24898122&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3669.8" data-doi="10.1016/j.pbi.2006.03.016"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">D’Auria</span>  <span class="cit-name-given-names">J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Acyltransferases in plants: a good time to be BAHD.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Plant Biol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">331</span>–<span class="cit-lpage">340</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bplant%2Bbiology%26rft.stitle%253DCurr%2BOpin%2BPlant%2BBiol%26rft.aulast%253DD%2527Auria%26rft.auinit1%253DJ.%2BC.%26rft.volume%253D9%26rft.issue%253D3%26rft.spage%253D331%26rft.epage%253D340%26rft.atitle%253DAcyltransferases%2Bin%2Bplants%253A%2Ba%2Bgood%2Btime%2Bto%2Bbe%2BBAHD.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.pbi.2006.03.016%26rft_id%253Dinfo%253Apmid%252F16616872%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.pbi.2006.03.016&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16616872&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000237873800016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-book" id="cit-8.11.3669.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dellaporta</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Greenblatt</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kermicle</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hicks</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wessler</span>  <span class="cit-name-given-names">S. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">Molecular cloning of the maize R-nj allele by transposon tagging with Ac</span>, pp. <span class="cit-fpage">263</span>–<span class="cit-lpage">282</span> in <span class="cit-source">Chromosome Structure and Function</span>, <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3669.10" data-doi="10.1104/pp.111.174748"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eichten</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foerster</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Leon</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kai</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yeh</span>  <span class="cit-name-given-names">C.-T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">B73-Mo17 near-isogenic lines demonstrate dispersed structural variation in maize.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">156</span>: <span class="cit-fpage">1679</span>–<span class="cit-lpage">1690</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1104/pp.111.174748</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.111.174748%26rft_id%253Dinfo%253Apmid%252F21705654%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czoxMDoiMTU2LzQvMTY3OSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzgvMTEvMzY2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-book" id="cit-8.11.3669.11" data-doi="10.1371/journal.pone.0019379"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawamoto</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.</span> <span class="cit-source">PLoS ONE</span> <span class="cit-vol">6</span>: <span class="cit-fpage">e19379</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0019379</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DElshire%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253De19379%26rft.epage%253De19379%26rft.atitle%253DA%2Brobust%252C%2Bsimple%2Bgenotyping-by-sequencing%2B%2528GBS%2529%2Bapproach%2Bfor%2Bhigh%2Bdiversity%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0019379%26rft_id%253Dinfo%253Apmid%252F21573248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0019379&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21573248&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3669.12" data-doi="10.2307/2408678"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Felsenstein</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1985</span> <span class="cit-article-title">Confidence Limits on Phylogenies: An Approach Using the Bootstrap.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">783</span>–<span class="cit-lpage">791</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1558-5646.1985.tb00420.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.volume%253D39%26rft.issue%253D4%26rft.spage%253D783%26rft.epage%253D791%26rft.atitle%253DConfidence%2Blimits%2Bon%2Bphylogenies%253B%2Ban%2Bapproach%2Busing%2Bthe%2Bbootstrap%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2408678%26rft_id%253Dinfo%253Apmid%252F7288891%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2408678&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1996015960&amp;link_type=GEOREF" class="cit-ref-sprinkles cit-ref-sprinkles-georef"><span>GeoRef</span></a><a href="/lookup/external-ref?access_num=7288891&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1985APJ8100007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3669.13" data-doi="10.1534/genetics.115.182071"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fragoso</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heffelfinger</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dellaporta</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Imputing Genotypes in Biallelic Populations from Low-Coverage Sequence Data.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">202</span>: <span class="cit-fpage">487</span>–<span class="cit-lpage">495</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.115.182071</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.182071%26rft_id%253Dinfo%253Apmid%252F26715670%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjAyLzIvNDg3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMS8zNjY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3669.14" data-doi="10.1016/0092-8674(94)90117-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grotewold</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Drummond</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowen</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peterson</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">The myb-homologous P gene controls phlobaphene pigmentation in maize floral organs by directly activating a flavonoid biosynthetic gene subset.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">76</span>: <span class="cit-fpage">543</span>–<span class="cit-lpage">553</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DGrotewold%26rft.auinit1%253DE.%26rft.volume%253D76%26rft.issue%253D3%26rft.spage%253D543%26rft.epage%253D553%26rft.atitle%253DThe%2Bmyb-homologous%2BP%2Bgene%2Bcontrols%2Bphlobaphene%2Bpigmentation%2Bin%2Bmaize%2Bfloral%2Borgans%2Bby%2Bdirectly%2Bactivating%2Ba%2Bflavonoid%2Bbiosynthetic%2Bgene%2Bsubset.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252894%252990117-1%26rft_id%253Dinfo%253Apmid%252F8313474%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(94)90117-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8313474&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1994MX20700014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3669.15" data-doi="10.1146/annurev.food.080708.100754"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Giusti</span>  <span class="cit-name-given-names">M. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Anthocyanins: natural colorants with health-promoting properties.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Food Sci. Technol.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">163</span>–<span class="cit-lpage">187</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1146/annurev.food.080708.100754</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DAnnu%2BRev%2BFood%2BSci%2BTechnol%26rft.aulast%253DHe%26rft.auinit1%253DJ.%26rft.volume%253D1%26rft.spage%253D163%26rft.epage%253D187%26rft.atitle%253DAnthocyanins%253A%2Bnatural%2Bcolorants%2Bwith%2Bhealth-promoting%2Bproperties.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.food.080708.100754%26rft_id%253Dinfo%253Apmid%252F22129334%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.food.080708.100754&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22129334&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3669.16" data-doi="10.1186/1471-2164-15-349"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heffelfinger</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fragoso</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moreno</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Overton</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mottinger</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Flexible and scalable genotyping-by-sequencing strategies for population studies.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2164-15-979</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-349%26rft_id%253Dinfo%253Apmid%252F24382143%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-349&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24382143&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3669.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Irani</span>  <span class="cit-name-given-names">N. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hernandez</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grotewold</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Chapter three: regulation of anthocyanin pigmentation.</span> <abbr class="cit-jnl-abbrev">Recent Adv. Phytochem.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">59</span>–<span class="cit-lpage">78</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRecent%2BAdv.%2BPhytochem.%26rft.volume%253D37%26rft.spage%253D59%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3669.18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kermicle</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1970</span> <span class="cit-article-title">Dependence of the R-mottled aleurone phenotype in maize on mode of sexual transmission.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">66</span>: <span class="cit-fpage">69</span>–<span class="cit-lpage">85</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKermicle%26rft.auinit1%253DJ.%2BL.%26rft.volume%253D66%26rft.issue%253D1%26rft.spage%253D69%26rft.epage%253D85%26rft.atitle%253DDEPENDENCE%2BOF%2BTHE%2BR-MOTTLED%2BALEURONE%2BPHENOTYPE%2BIN%2BMAIZE%2BON%2BMODE%2BOF%2BSEXUAL%2BTRANSMISSION%26rft_id%253Dinfo%253Apmid%252F17248508%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo3OiI2Ni8xLzY5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMS8zNjY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3669.19" data-doi="10.1016/j.plaphy.2009.08.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">D. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">S. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">J.-H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">B.-G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ahn</span>  <span class="cit-name-given-names">J.-H.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Molecular characterization of flavonoid malonyltransferase from Oryza sativa.</span> <abbr class="cit-jnl-abbrev">Plant Physiol. Biochem.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">991</span>–<span class="cit-lpage">997</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.plaphy.2009.08.004</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2Bphysiology%2Band%2Bbiochemistry%2B%253A%2B%2BPPB%2B%252F%2BSoci%25C3%25A9t%25C3%25A9%2Bfran%25C3%25A7aise%2Bde%2Bphysiologie%2Bv%25C3%25A9g%25C3%25A9tale%26rft.stitle%253DPlant%2BPhysiol%2BBiochem%26rft.aulast%253DKim%26rft.auinit1%253DD.%2BH.%26rft.volume%253D47%26rft.issue%253D11-12%26rft.spage%253D991%26rft.epage%253D997%26rft.atitle%253DMolecular%2Bcharacterization%2Bof%2Bflavonoid%2Bmalonyltransferase%2Bfrom%2BOryza%2Bsativa.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.plaphy.2009.08.004%26rft_id%253Dinfo%253Apmid%252F19733090%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.plaphy.2009.08.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19733090&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000272767200003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3669.20" data-doi="10.1093/molbev/msw054"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stecher</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tamura</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">MEGA7: Molecular Evolutionary Genetics Analysis version 7.0 for bigger datasets.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">1870</span>–<span class="cit-lpage">1874</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msw054</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsw054%26rft_id%253Dinfo%253Apmid%252F27004904%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msw054&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27004904&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3669.21" data-doi="10.1038/nmeth.1923"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langmead</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Fast gapped-read alignment with Bowtie 2.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">357</span>–<span class="cit-lpage">359</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.1923</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DLangmead%26rft.auinit1%253DB.%26rft.volume%253D9%26rft.issue%253D4%26rft.spage%253D357%26rft.epage%253D359%26rft.atitle%253DFast%2Bgapped-read%2Balignment%2Bwith%2BBowtie%2B2.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1923%26rft_id%253Dinfo%253Apmid%252F22388286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1923&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22388286&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000302218500017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3669.22" data-doi="10.1023/A:1014893521186"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sharopova</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beavis</span>  <span class="cit-name-given-names">W. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grant</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Katt</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Expanding the genetic map of maize with the intermated B73 x Mo17 (IBM) population.</span> <abbr class="cit-jnl-abbrev">Plant Mol. Biol.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">453</span>–<span class="cit-lpage">461</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BMol%2BBiol%26rft.aulast%253DLee%26rft.auinit1%253DM.%26rft.volume%253D48%26rft.issue%253D5-6%26rft.spage%253D453%26rft.epage%253D461%26rft.atitle%253DExpanding%2Bthe%2Bgenetic%2Bmap%2Bof%2Bmaize%2Bwith%2Bthe%2Bintermated%2BB73%2Bx%2BMo17%2B%2528IBM%2529%2Bpopulation.%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1014893521186%26rft_id%253Dinfo%253Apmid%252F11999829%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/A:1014893521186&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11999829&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000174606400003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3669.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ding</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> Combined linkage and association mapping reveals QTL and candidate genes for plant and ear height in maize. Plant Genet. Genomics 833.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3669.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McCarty</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Settles</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Suzuki</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tan</span>  <span class="cit-name-given-names">B. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Latshaw</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Steady-state transposon mutagenesis in inbred maize. Plant J.</span> <abbr class="cit-jnl-abbrev">Cell. Mol. Biol.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">52</span>–<span class="cit-lpage">61</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell.%2BMol.%2BBiol.%26rft.volume%253D44%26rft.spage%253D52%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3669.25" data-doi="10.1093/bioinformatics/btm076"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Papadopoulos</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Agarwala</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">COBALT: constraint-based alignment tool for multiple protein sequences.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">1073</span>–<span class="cit-lpage">1079</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtm076%26rft_id%253Dinfo%253Apmid%252F17332019%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btm076&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17332019&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000246773300005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3669.26" data-doi="10.1021/acs.jafc.7b00771"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Paulsmeyer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chatham</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Becker</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">West</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">West</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Survey of anthocyanin composition and concentration in diverse maize germplasms.</span> <abbr class="cit-jnl-abbrev">J. Agric. Food Chem.</abbr> <span class="cit-vol">65</span>: <span class="cit-fpage">4341</span>–<span class="cit-lpage">4350</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1021/acs.jafc.7b00771</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAgric.%2BFood%2BChem.%26rft.volume%253D65%26rft.spage%253D4341%26rft_id%253Dinfo%253Adoi%252F10.1021%252Facs.jafc.7b00771%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/acs.jafc.7b00771&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3669.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Paz-Ares</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ghosal</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wienand</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peterson</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saedler</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">The regulatory c1 locus of Zea mays encodes a protein with homology to myb proto-oncogene products and with structural similarities to transcriptional activators.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">3553</span>–<span class="cit-lpage">3558</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJournal%26rft.aulast%253DPaz-Ares%26rft.auinit1%253DJ%26rft.volume%253D6%26rft.issue%253D12%26rft.spage%253D3553%26rft.epage%253D3558%26rft.atitle%253DThe%2Bregulatory%2Bc1%2Blocus%2Bof%2BZea%2Bmays%2Bencodes%2Ba%2Bprotein%2Bwith%2Bhomology%2Bto%2Bmyb%2Bproto-oncogene%2Bproducts%2Band%2Bwith%2Bstructural%2Bsimilarities%2Bto%2Btranscriptional%2Bactivators.%26rft_id%253Dinfo%253Apmid%252F3428265%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=3428265&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1987K948600001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-book" id="cit-8.11.3669.28" data-doi="10.1371/journal.pone.0032253"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Development of high-density genetic maps for barley and wheat using a novel two-enzyme genotyping-by-sequencing approach.</span> <span class="cit-source">PLoS ONE</span> <span class="cit-vol">7</span>: <span class="cit-fpage">e32253</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0032253</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DPoland%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D7%26rft.issue%253D2%26rft.spage%253De32253%26rft.epage%253De32253%26rft.atitle%253DDevelopment%2Bof%2Bhigh-density%2Bgenetic%2Bmaps%2Bfor%2Bbarley%2Band%2Bwheat%2Busing%2Ba%2Bnovel%2Btwo-enzyme%2Bgenotyping-by-sequencing%2Bapproach.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0032253%26rft_id%253Dinfo%253Apmid%252F22389690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0032253&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22389690&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-book" id="cit-8.11.3669.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">R Core Team</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-source">R: a language and environment for statistical computing</span>, <span class="cit-publ-name">R Foundation for Statistical Computing</span>, <span class="cit-publ-loc">Vienna, Austria</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3669.30" data-doi="10.1099/ijs.0.01472-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Saitou</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nei</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">The neighbor-joining method: a new method for reconstructing phylogenetic trees.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">406</span>–<span class="cit-lpage">425</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DSaitou%26rft.auinit1%253DN%26rft.volume%253D4%26rft.issue%253D4%26rft.spage%253D406%26rft.epage%253D425%26rft.atitle%253DThe%2Bneighbor-joining%2Bmethod%253A%2Ba%2Bnew%2Bmethod%2Bfor%2Breconstructing%2Bphylogenetic%2Btrees%26rft_id%253Dinfo%253Adoi%252F10.1099%252Fijs.0.01472-0%26rft_id%253Dinfo%253Apmid%252F3447015%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/ijs.0.01472-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3447015&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1987J406700007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3669.31" data-doi="10.1534/genetics.110.126136"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sharma</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cortes-Cruz</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ahern</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McMullen</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brutnell</span>  <span class="cit-name-given-names">T. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Identification of the Pr1 gene product completes the anthocyanin biosynthesis pathway of maize.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">188</span>: <span class="cit-fpage">69</span>–<span class="cit-lpage">79</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.110.126136</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.126136%26rft_id%253Dinfo%253Apmid%252F21385724%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMTg4LzEvNjkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzExLzM2NjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-book" id="cit-8.11.3669.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">St-Pierre</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Luca</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Evolution of acyltransferase genes: origin and diversification of the BAHD superfamily of acyltransferases involved in secondary metabolism</span>, pp. <span class="cit-fpage">285</span>–<span class="cit-lpage">315</span> in <span class="cit-source">Recent Advances in Phytochemistry</span>, <span class="cit-publ-name">Elsevier</span>, <span class="cit-publ-loc">Amsterdam, Netherlands</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3669.33" data-doi="10.1016/j.plantsci.2003.08.010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Suzuki</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nakayama</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yamaguchi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nishino</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">cDNA cloning and characterization of two Dendranthema × morifolium anthocyanin malonyltransferases with different functional activities.</span> <abbr class="cit-jnl-abbrev">Plant Sci.</abbr> <span class="cit-vol">166</span>: <span class="cit-fpage">89</span>–<span class="cit-lpage">96</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BSci.%26rft.volume%253D166%26rft.spage%253D89%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.plantsci.2003.08.010%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.plantsci.2003.08.010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000188786500010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3669.34" data-doi="10.1104/pp.010447"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Suzuki</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nakayama</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yonekura-Sakakibara</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fukui</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nakamura</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">cDNA cloning, heterologous expressions, and functional characterization of malonyl-coenzyme A:anthocyanidin 3-O-glucoside-6”-O-malonyltransferase from Dahlia flowers.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">130</span>: <span class="cit-fpage">2142</span>–<span class="cit-lpage">2151</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiology%26rft.stitle%253DPlant%2BPhysiol.%26rft.aulast%253DSuzuki%26rft.auinit1%253DH.%26rft.volume%253D130%26rft.issue%253D4%26rft.spage%253D2142%26rft.epage%253D2151%26rft.atitle%253DcDNA%2BCloning%252C%2BHeterologous%2BExpressions%252C%2Band%2BFunctional%2BCharacterization%2Bof%2BMalonyl-Coenzyme%2BA%253AAnthocyanidin%2B3-O-Glucoside-6%2522-O-Malonyltransferase%2Bfrom%2BDahlia%2BFlowers%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.010447%26rft_id%253Dinfo%253Apmid%252F12481098%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czoxMDoiMTMwLzQvMjE0MiI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzgvMTEvMzY2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3669.35"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Swarts</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Romero Navarro</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">An</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Romay</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> Novel methods to optimize genotypic imputation for low-coverage, next-generation sequence data in crop plants. Plant Genome 7: 0.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3669.36" data-doi="10.1074/jbc.M700638200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Unno</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ichimaida</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Suzuki</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Takahashi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tanaka</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Structural and mutational studies of anthocyanin malonyltransferases establish the features of BAHD enzyme catalysis.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">282</span>: <span class="cit-fpage">15812</span>–<span class="cit-lpage">15822</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M700638200%26rft_id%253Dinfo%253Apmid%252F17383962%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODIvMjEvMTU4MTIiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzExLzM2NjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3669.37" data-doi="10.1007/s11130-015-0479-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Urias-Lugo</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heredia</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muy-Rangel</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valdez-Torres</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Serna-Saldívar</span>  <span class="cit-name-given-names">S. O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Anthocyanins and Phenolic Acids of Hybrid and Native Blue Maize (Zea mays L.) Extracts and Their Antiproliferative Activity in Mammary (MCF7), Liver (HepG2), Colon (Caco2 and HT29) and Prostate (PC3) Cancer Cells.</span> <abbr class="cit-jnl-abbrev">Plant Foods Hum. Nutr.</abbr> <span class="cit-vol">70</span>: <span class="cit-fpage">193</span>–<span class="cit-lpage">199</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s11130-015-0479-4</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BFoods%2BHum.%2BNutr.%26rft.volume%253D70%26rft.spage%253D193%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11130-015-0479-4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11130-015-0479-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3669.38" data-doi="10.1093/nar/gkq1148"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Youens-Clark</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buckler</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casstevens</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">DeClerck</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Gramene database in 2010: updates and extensions.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">D1085</span>–<span class="cit-lpage">D1094</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkq1148</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkq1148%26rft_id%253Dinfo%253Apmid%252F21076153%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkq1148&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21076153&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000285831700171&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3669.39" data-doi="10.1105/tpc.110.080804"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huhman</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shadle</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">X.-Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sumner</span>  <span class="cit-name-given-names">L. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">MATE2 mediates vacuolar sequestration of flavonoid glycosides and glycoside malonates in Medicago truncatula.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">1536</span>–<span class="cit-lpage">1555</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1105/tpc.110.080804</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.110.080804%26rft_id%253Dinfo%253Apmid%252F21467581%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjIzLzQvMTUzNiI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzgvMTEvMzY2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3669.40" data-doi="10.1016/j.foodchem.2016.07.073"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">C.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">Y.-Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Z.-J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wen</span>  <span class="cit-name-given-names">G.-S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">F.-G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Stability-increasing effects of anthocyanin glycosyl acylation.</span> <abbr class="cit-jnl-abbrev">Food Chem.</abbr> <span class="cit-vol">214</span>: <span class="cit-fpage">119</span>–<span class="cit-lpage">128</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.foodchem.2016.07.073</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFood%2BChem.%26rft.volume%253D214%26rft.spage%253D119%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.foodchem.2016.07.073%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.foodchem.2016.07.073&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3669.41" data-doi="10.1186/s12864-015-2294-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hao</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genetic dissection of maize plant architecture with an ultra-high density bin map based on recombinant inbred lines.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">15</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12864-016-2555-z</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D17%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-015-2294-6%26rft_id%253Dinfo%253Apmid%252F26818753%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-015-2294-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26818753&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/8/11/3669.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/11/3661" title="A GAL80 Collection To Inhibit GAL4 Transgenes in Drosophila Olfactory Sensory Neurons" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/11/3679" title="The Role of Standing Variation in the Evolution of Weedines Traits in South Asian Weedy Rice (Oryza spp.)" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/8/11"><span>Volume 8 Issue 11, November 2018</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/8/11.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/8/11.cover-source.jpg?itok=lFO8v6bg" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 8 (11)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/11/3669" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-R5xiKVJqkYEW8AZIWWU_va2i-ZjR39oaY24wcnsi7rw" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/8/11/3669" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/8/11/3669" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/8/11/3669" class="active" data-icon-position="" data-hide-link-title="0">Discovery of Anthocyanin Acyltransferase1 (AAT1) in Maize Using Genotyping-by-Sequencing (GBS)</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/491225" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-v9VzKNbrjvwxv3l8LDvCWkBUkrepdOONRjx9Z7XYins" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/11/3669.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/8/11/3669" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-CXaNkC56KiX8bpWFYwLHEm93-MOAmjHC9h5a9O-sfP0" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/8/11/3669" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/11/3669.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="491225" id="node491225--2" data-pisa="ggg;8/11/3669" data-pisa-master="ggg;g3.118.200630" data-apath="/ggg/8/11/3669.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Discovery of Anthocyanin Acyltransferase1 (AAT1) in Maize Using Genotyping-by-Sequencing (GBS)</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Michael N. Paulsmeyer</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-1332-711X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Patrick J. Brown</span> and <span class="highwire-citation-author" data-delta="2">John A. Juvik</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3669-3678;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200630 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node491225--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Michael N. Paulsmeyer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Crop Sciences, University of Illinois at Urbana-Champaign, Urbana, Illinois, 61801</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BN.%2BPaulsmeyer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Paulsmeyer%20MN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BN.%2BPaulsmeyer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Patrick J. Brown </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Sciences, University of California at Davis, Davis, California, 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Patrick%2BJ.%2BBrown%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brown%20PJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APatrick%2BJ.%2BBrown%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-1332-711X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Patrick J. Brown</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">John A. Juvik </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Crop Sciences, University of Illinois at Urbana-Champaign, Urbana, Illinois, 61801</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BA.%2BJuvik%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Juvik%20JA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohn%2BA.%2BJuvik%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/8/11/3669" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/8/11/3669&amp;title=Discovery%20of%20Anthocyanin%20Acyltransferase1%20%28AAT1%29%20in%20Maize%20Using%20Genotyping-by-Sequencing%20%28GBS%29" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/8/11/3669&amp;title=Discovery%20of%20Anthocyanin%20Acyltransferase1%20%28AAT1%29%20in%20Maize%20Using%20Genotyping-by-Sequencing%20%28GBS%29" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/8/11/3669&amp;title=Discovery%20of%20Anthocyanin%20Acyltransferase1%20%28AAT1%29%20in%20Maize%20Using%20Genotyping-by-Sequencing%20%28GBS%29" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/8/11/3669&amp;text=Discovery%20of%20Anthocyanin%20Acyltransferase1%20%28AAT1%29%20in%20Maize%20Using%20Genotyping-by-Sequencing%20%28GBS%29" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/8/11/3669&amp;title=Discovery%20of%20Anthocyanin%20Acyltransferase1%20%28AAT1%29%20in%20Maize%20Using%20Genotyping-by-Sequencing%20%28GBS%29" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/8/11/3669&amp;t=Discovery%20of%20Anthocyanin%20Acyltransferase1%20%28AAT1%29%20in%20Maize%20Using%20Genotyping-by-Sequencing%20%28GBS%29" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/8/11/3669&amp;title=Discovery%20of%20Anthocyanin%20Acyltransferase1%20%28AAT1%29%20in%20Maize%20Using%20Genotyping-by-Sequencing%20%28GBS%29" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/8/11/3669&amp;title=Discovery%20of%20Anthocyanin%20Acyltransferase1%20%28AAT1%29%20in%20Maize%20Using%20Genotyping-by-Sequencing%20%28GBS%29" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/491225/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="491225" id="node491225--3" data-pisa="ggg;8/11/3669" data-pisa-master="ggg;g3.118.200630" data-apath="/ggg/8/11/3669.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Discovery of Anthocyanin Acyltransferase1 (AAT1) in Maize Using Genotyping-by-Sequencing (GBS)</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Michael N. Paulsmeyer</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-1332-711X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Patrick J. Brown</span> and <span class="highwire-citation-author" data-delta="2">John A. Juvik</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3669-3678;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200630 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node491225--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Michael N. Paulsmeyer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Crop Sciences, University of Illinois at Urbana-Champaign, Urbana, Illinois, 61801</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BN.%2BPaulsmeyer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Paulsmeyer%20MN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BN.%2BPaulsmeyer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Patrick J. Brown </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Sciences, University of California at Davis, Davis, California, 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Patrick%2BJ.%2BBrown%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brown%20PJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APatrick%2BJ.%2BBrown%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-1332-711X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Patrick J. Brown</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">John A. Juvik </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Crop Sciences, University of Illinois at Urbana-Champaign, Urbana, Illinois, 61801</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BA.%2BJuvik%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Juvik%20JA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohn%2BA.%2BJuvik%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3669&amp;rft.epage=3678&amp;rft.atitle=Discovery%20of%20Anthocyanin%20Acyltransferase1%20%28AAT1%29%20in%20Maize%20Using%20Genotyping-by-Sequencing%20%28GBS%29&amp;rft.volume=8&amp;rft.issue=11&amp;rft.date=2018-11-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Paulsmeyer%2C+Michael+N.&amp;rft.au=Brown%2C+Patrick+J.&amp;rft.au=Juvik%2C+John+A."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/491225/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/491225/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/491225/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/491225/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/491225/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/491225/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/491225/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/491225/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/491225/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/491225/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/491225/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/491225/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/8/11/3669" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-18" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/8/11/3669.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/8/11/3669.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/8/11/3669.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/8/11/3669.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
