<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/8/11/3703" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/8/11/3703.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/8/11/3703.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/8/11/3703.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/8/11/3703.atom" />
<meta name="HW.pisa" content="ggg;8/11/3703" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="A Case Study of Genomic Instability in an Industrial Strain of Saccharomyces cerevisiae" />
<meta name="DC.Identifier" content="10.1534/g3.118.200446" />
<meta name="DC.Date" content="2018-11-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © by the Genetics Society of America. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Relation" content="https://doi.org/10.25387/g3.6406598" />
<meta name="DC.Description" content="The Saccharomyces cerevisiae strain JAY270/PE2 is a highly efficient biocatalyst used in the production of bioethanol from sugarcane feedstock. This strain is heterothallic and diploid, and its genome is characterized by abundant structural and nucleotide polymorphisms between homologous chromosomes. One of the reasons it is favored by many distilleries is that its cells do not normally aggregate, a trait that facilitates cell recycling during batch-fed fermentations. However, long-term propagation makes the yeast population vulnerable to the effects of genomic instability, which may trigger the appearance of undesirable phenotypes such as cellular aggregation. In pure cultures of JAY270, we identified the recurrent appearance of mutants displaying a mother-daughter cell separation defect resulting in rough colonies in agar media and fast sedimentation in liquid culture. We investigated the genetic basis of the colony morphology phenotype and found that JAY270 is heterozygous for a frameshift mutation in the [ACE2][1] gene ( [ACE2][1]/[ace2][1]-A7 ), which encodes a transcriptional regulator of mother-daughter cell separation. All spontaneous rough colony JAY270-derived isolates analyzed carried copy-neutral loss-of-heterozygosity (LOH) at the region of chromosome XII where [ACE2][1] is located ( [ace2][1]-A7/[ace2][1]-A7 ). We specifically measured LOH rates at the [ACE2][1] locus, and at three additional chromosomal regions in JAY270 and in a conventional homozygous diploid laboratory strain. This direct comparison showed that LOH rates at all sites were quite similar between the two strain backgrounds. In this case study of genomic instability in an industrial strain, we showed that the JAY270 genome is dynamic and that structural changes to its chromosomes can lead to new phenotypes. However, our analysis also indicated that the inherent level of genomic instability in this industrial strain is normal relative to a laboratory strain. Our work provides an important frame of reference to contextualize the interpretation of instability processes observed in the complex genomes of industrial yeast strains.

 [1]: http://www.yeastgenome.org/locus/S000004121/overview" />
<meta name="DC.Contributor" content="Aline Rodrigues-Prause" />
<meta name="DC.Contributor" content="Nadia M. V. Sampaio" />
<meta name="DC.Contributor" content="Theodore M. Gurol" />
<meta name="DC.Contributor" content="Guadalupe M. Aguirre" />
<meta name="DC.Contributor" content="Hailey N. C. Sedam" />
<meta name="DC.Contributor" content="Mary J. Chapman" />
<meta name="DC.Contributor" content="Ewa P. Malc" />
<meta name="DC.Contributor" content="V. P. Ajith" />
<meta name="DC.Contributor" content="Parijat Chakraborty" />
<meta name="DC.Contributor" content="Pedro A. Tizei" />
<meta name="DC.Contributor" content="Gonçalo A. G. Pereira" />
<meta name="DC.Contributor" content="Piotr A. Mieczkowski" />
<meta name="DC.Contributor" content="Koodali T. Nishant" />
<meta name="DC.Contributor" content="Juan Lucas Argueso" />
<meta name="article:published_time" content="2018-11-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="A Case Study of Genomic Instability in an Industrial Strain of Saccharomyces cerevisiae" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The &lt;i&gt;Saccharomyces cerevisiae&lt;/i&gt; strain JAY270/PE2 is a highly efficient biocatalyst used in the production of bioethanol from sugarcane feedstock. This strain is heterothallic and diploid, and its genome is characterized by abundant structural and nucleotide polymorphisms between homologous chromosomes. One of the reasons it is favored by many distilleries is that its cells do not normally aggregate, a trait that facilitates cell recycling during batch-fed fermentations. However, long-term propagation makes the yeast population vulnerable to the effects of genomic instability, which may trigger the appearance of undesirable phenotypes such as cellular aggregation. In pure cultures of JAY270, we identified the recurrent appearance of mutants displaying a mother-daughter cell separation defect resulting in rough colonies in agar media and fast sedimentation in liquid culture. We investigated the genetic basis of the colony morphology phenotype and found that JAY270 is heterozygous for a frameshift mutation in the &lt;i&gt;ACE2&lt;/i&gt; gene (&lt;i&gt;ACE2/ace2-A7&lt;/i&gt;), which encodes a transcriptional regulator of mother-daughter cell separation. All spontaneous rough colony JAY270-derived isolates analyzed carried copy-neutral loss-of-heterozygosity (LOH) at the region of chromosome XII where &lt;i&gt;ACE2&lt;/i&gt; is located (&lt;i&gt;ace2-A7/ace2-A7&lt;/i&gt;). We specifically measured LOH rates at the &lt;i&gt;ACE2&lt;/i&gt; locus, and at three additional chromosomal regions in JAY270 and in a conventional homozygous diploid laboratory strain. This direct comparison showed that LOH rates at all sites were quite similar between the two strain backgrounds. In this case study of genomic instability in an industrial strain, we showed that the JAY270 genome is dynamic and that structural changes to its chromosomes can lead to new phenotypes. However, our analysis also indicated that the inherent level of genomic instability in this industrial strain is normal relative to a laboratory strain. Our work provides an important frame of reference to contextualize the interpretation of instability processes observed in the complex genomes of industrial yeast strains.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2018/11/01" />
<meta name="citation_mjid" content="ggg;8/11/3703" />
<meta name="citation_id" content="8/11/3703" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/8/11/3703" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/8/11/3703.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/8/11/3703.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/8/11/3703.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.118.200446" />
<meta name="citation_pmid" content="30254181" />
<meta name="citation_volume" content="8" />
<meta name="citation_issue" content="11" />
<meta name="citation_num_pages" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="3703" />
<meta name="citation_lastpage" content="3713" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Aline Rodrigues-Prause" />
<meta name="citation_author_institution" content="Department of Environmental and Radiological Health Sciences" />
<meta name="citation_author_institution" content="Departamento de Genética, Evolução e Bioagentes, Instituto de Biologia, Universidade Estadual de Campinas, Campinas-SP, Brazil" />
<meta name="citation_author" content="Nadia M. V. Sampaio" />
<meta name="citation_author_institution" content="Department of Environmental and Radiological Health Sciences" />
<meta name="citation_author_institution" content="Cell and Molecular Biology Graduate Program, Colorado State University, Fort Collins-CO 80523" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-6403-0159" />
<meta name="citation_author" content="Theodore M. Gurol" />
<meta name="citation_author_institution" content="Department of Environmental and Radiological Health Sciences" />
<meta name="citation_author_institution" content="Cell and Molecular Biology Graduate Program, Colorado State University, Fort Collins-CO 80523" />
<meta name="citation_author" content="Guadalupe M. Aguirre" />
<meta name="citation_author_institution" content="Department of Environmental and Radiological Health Sciences" />
<meta name="citation_author_institution" content="Cell and Molecular Biology Graduate Program, Colorado State University, Fort Collins-CO 80523" />
<meta name="citation_author" content="Hailey N. C. Sedam" />
<meta name="citation_author_institution" content="Department of Environmental and Radiological Health Sciences" />
<meta name="citation_author_institution" content="Cell and Molecular Biology Graduate Program, Colorado State University, Fort Collins-CO 80523" />
<meta name="citation_author" content="Mary J. Chapman" />
<meta name="citation_author_institution" content="Department of Environmental and Radiological Health Sciences" />
<meta name="citation_author" content="Ewa P. Malc" />
<meta name="citation_author_institution" content="Department of Genetics, University of North Carolina, Chapel Hill-NC 27599" />
<meta name="citation_author" content="V. P. Ajith" />
<meta name="citation_author_institution" content="School of Biology, Indian Institute of Science Education and Research" />
<meta name="citation_author" content="Parijat Chakraborty" />
<meta name="citation_author_institution" content="School of Biology, Indian Institute of Science Education and Research" />
<meta name="citation_author" content="Pedro A. Tizei" />
<meta name="citation_author_institution" content="Departamento de Genética, Evolução e Bioagentes, Instituto de Biologia, Universidade Estadual de Campinas, Campinas-SP, Brazil" />
<meta name="citation_author" content="Gonçalo A. G. Pereira" />
<meta name="citation_author_institution" content="Departamento de Genética, Evolução e Bioagentes, Instituto de Biologia, Universidade Estadual de Campinas, Campinas-SP, Brazil" />
<meta name="citation_author" content="Piotr A. Mieczkowski" />
<meta name="citation_author_institution" content="Department of Genetics, University of North Carolina, Chapel Hill-NC 27599" />
<meta name="citation_author" content="Koodali T. Nishant" />
<meta name="citation_author_institution" content="School of Biology, Indian Institute of Science Education and Research" />
<meta name="citation_author_institution" content="Center for Computation Modeling and Simulation, Indian Institute of Science Education and Research, Thiruvananthapuram, Trivandrum 695016, India" />
<meta name="citation_author" content="Juan Lucas Argueso" />
<meta name="citation_author_institution" content="Department of Environmental and Radiological Health Sciences" />
<meta name="citation_author_institution" content="Cell and Molecular Biology Graduate Program, Colorado State University, Fort Collins-CO 80523" />
<meta name="citation_author_email" content="lucas.argueso@colostate.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-7157-1519" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=J. L. Argueso;citation_author=M. F. Carazzolle;citation_author=P. A. Mieczkowski;citation_author=F. M. Duarte;citation_author=O. V. C. Netto;citation_title=Genome structure of a Saccharomyces cerevisiae strain widely used in bioethanol production.;citation_volume=19;citation_year=2009;citation_pmid=19812109;citation_doi=10.1101/gr.091777.109" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. L. Argueso;citation_author=J. Westmoreland;citation_author=P. A. Mieczkowski;citation_author=M. Gawel;citation_author=T. D. Petes;citation_title=Double-strand breaks associated with repetitive DNA can reshape the genome.;citation_volume=105;citation_year=2008;citation_pmid=18701715;citation_doi=10.1073/pnas.0804529105" />
<meta name="citation_reference" content="citation_title=Current Protocols in Molecular Biology;citation_author=F. M. Ausubel;citation_author=R. Brent;citation_author=R. E. Kingston;citation_author=D. D. Moore;citation_author=J. G. Seidman;citation_year=1998" />
<meta name="citation_reference" content="citation_journal_title=FEMS Yeast Res.;citation_author=L. C. Basso;citation_author=H. V. de Amorim;citation_author=A. J. de Oliveira;citation_author=M. L. Lopes;citation_title=Yeast selection for fuel ethanol production in Brazil.;citation_volume=8;citation_year=2008;citation_pmid=18752628;citation_doi=10.1111/j.1567-1364.2008.00428.x" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=A. R. Borneman;citation_author=B. A. Desany;citation_author=D. Riches;citation_author=J. P. Affourtit;citation_author=A. H. Forgan;citation_title=Whole-genome comparison reveals novel genetic elements that characterize the genome of industrial strains of Saccharomyces cerevisiae.;citation_pages=e1001287-e1001287;citation_volume=7;citation_year=2011;citation_issue=2;citation_pmid=21304888;citation_doi=10.1371/journal.pgen.1001287" />
<meta name="citation_reference" content="citation_journal_title=Yeast (Chichester, England);citation_journal_abbrev=Yeast;citation_author=D. Carro;citation_author=B. Pina;citation_title=Genetic analysis of the karyotype instability in natural wine yeast strains.;citation_pages=1457-1470;citation_volume=18;citation_year=2001;citation_issue=16;citation_pmid=11748723;citation_doi=10.1002/yea.789" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=A. M. Casper;citation_author=P. A. Mieczkowski;citation_author=M. Gawel;citation_author=T. D. Petes;citation_title=Low levels of DNA polymerase alpha induce mitotic and meiotic instability in the ribosomal DNA gene cluster of Saccharomyces cerevisiae.;citation_pages=e1000105-e1000105;citation_volume=4;citation_year=2008;citation_issue=6;citation_pmid=18584028;citation_doi=10.1371/journal.pgen.1000105" />
<meta name="citation_reference" content="citation_journal_title=DNA Repair (Amst.);citation_author=D. A. Cornelio;citation_author=H. N. Sedam;citation_author=J. A. Ferrarezi;citation_author=N. M. Sampaio;citation_author=J. L. Argueso;citation_title=Both R-loop removal and ribonucleotide excision repair activities of RNase H2 contribute substantially to chromosome stability.;citation_pages=110-114;citation_volume=52;citation_year=2017;citation_doi=10.1016/j.dnarep.2017.02.012" />
<meta name="citation_reference" content="citation_title=Diversity and Benefits of Microorganisms from the Tropics;citation_author=H. V. de Amorim;citation_author=H. B. de Amorim Neto;citation_author=M. L. Lopes;citation_author=S. C. de Lima Paulillo;citation_title=Evolution of Yeast Selection for Fuel Ethanol: Breaking Paradigms;citation_year=2017;citation_doi=10.1007/978-3-319-55804-2_17" />
<meta name="citation_reference" content="citation_journal_title=Appl. Microbiol. Biotechnol.;citation_author=B. E. Della-Bianca;citation_author=T. O. Basso;citation_author=B. U. Stambuk;citation_author=L. C. Basso;citation_author=A. K. Gombert;citation_title=What do we know about the yeast strains from the Brazilian fuel ethanol industry?;citation_pages=979-991;citation_volume=97;citation_year=2013;citation_doi=10.1007/s00253-012-4631-x" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=R. A. Donnianni;citation_author=L. S. Symington;citation_title=Break-induced replication occurs by conservative DNA synthesis.;citation_volume=110;citation_year=2013;citation_pmid=23898170;citation_doi=10.1073/pnas.1309800110" />
<meta name="citation_reference" content="citation_journal_title=Molecular microbiology;citation_journal_abbrev=Mol Microbiol;citation_author=M. T. Doolin;citation_author=A. L. Johnson;citation_author=L. H. Johnston;citation_author=G. Butler;citation_title=Overlapping and distinct roles of the duplicated yeast transcription factors Ace2p and Swi5p.;citation_pages=422-432;citation_volume=40;citation_year=2001;citation_issue=2;citation_pmid=11309124;citation_doi=10.1046/j.1365-2958.2001.02388.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=S. Fogel;citation_author=D. D. Hurst;citation_title=COINCIDENCE RELATIONS BETWEEN GENE CONVERSION AND MITOTIC RECOMBINATION IN SACCHAROMYCES;citation_pages=321-328;citation_volume=48;citation_year=1963;citation_issue=3;citation_pmid=13945718" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Forche;citation_author=P. T. Magee;citation_author=A. Selmecki;citation_author=J. Berman;citation_author=G. May;citation_title=Evolution in Candida albicans populations during a single passage through a mouse host.;citation_volume=182;citation_year=2009;citation_pmid=19414562;citation_doi=10.1534/genetics.109.103325" />
<meta name="citation_reference" content="citation_journal_title=Mutation Research;citation_journal_abbrev=Mutation Research;citation_author=K. M. Freeman;citation_author=G. R. Hoffmann;citation_title=Frequencies of mutagen-induced coincident mitotic recombination at unlinked loci in Saccharomyces cerevisiae.;citation_pages=119-132;citation_volume=616;citation_year=2007;citation_issue=1-2;citation_pmid=17156798;citation_doi=10.1016/j.mrfmmm.2006.11.014" />
<meta name="citation_reference" content="citation_journal_title=FEMS Yeast Res.;citation_author=A. R. Ganley;citation_author=T. Kobayashi;citation_title=Ribosomal DNA and cellular senescence: new evidence supporting the connection between rDNA and aging.;citation_volume=14;citation_year=2014;citation_pmid=24373458;citation_doi=10.1111/1567-1364.12133" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. E. Golin;citation_author=M. S. Esposito;citation_title=COINCIDENT GENE CONVERSION DURING MITOSIS IN SACCHAROMYCES;citation_pages=355-365;citation_volume=107;citation_year=1984;citation_issue=3;citation_pmid=6376279" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. E. Golin;citation_author=H. Tampe;citation_title=Coincident recombination during mitosis in saccharomyces: distance-dependent and -independent components.;citation_pages=541-547;citation_volume=119;citation_year=1988;citation_issue=3;citation_pmid=3042509" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=B. M. Hall;citation_author=C. X. Ma;citation_author=P. Liang;citation_author=K. K. Singh;citation_title=Fluctuation analysis calculator: A web tool for the determination of mutation rate using Luria-Delbruck fluctuation analysis.;citation_volume=25;citation_year=2009;citation_pmid=19369502;citation_doi=10.1093/bioinformatics/btp253" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=E. A. Hope;citation_author=C. J. Amorosi;citation_author=A. W. Miller;citation_author=K. Dang;citation_author=C. S. Heil;citation_title=Experimental Evolution Reveals Favored Adaptive Routes to Cell Aggregation in Yeast.;citation_volume=206;citation_year=2017;citation_pmid=28450459;citation_doi=10.1534/genetics.116.198895" />
<meta name="citation_reference" content="citation_journal_title=eLife;citation_journal_abbrev=elife;citation_author=J. H. Koschwanez;citation_author=K. R. Foster;citation_author=A. W. Murray;citation_title=Improved use of a public good selects for the evolution of undifferentiated multicellularity;citation_pages=e00367-e00367;citation_volume=2;citation_year=2013;citation_issue=0;citation_pmid=23577233;citation_doi=10.7554/eLife.00367" />
<meta name="citation_reference" content="citation_journal_title=Trends Genet.;citation_author=J. Kramara;citation_author=B. Osia;citation_author=A. Malkova;citation_title=Break-Induced Replication: The Where, The Why, and The How.;citation_pages=518-531;citation_volume=34;citation_year=2018;citation_doi=10.1016/j.tig.2018.04.002" />
<meta name="citation_reference" content="citation_journal_title=J. Genet.;citation_author=D. E. Lea;citation_author=C. A. Coulson;citation_title=The distribution of the numbers of mutants in bacterial populations.;citation_pages=491-511;citation_volume=28;citation_year=1949" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=P. M. Magwene;citation_author=O. Kayikci;citation_author=J. A. Granek;citation_author=J. M. Reininga;citation_author=Z. Scholl;citation_title=Outcrossing, mitotic recombination, and life-history trade-offs shape genome evolution in Saccharomyces cerevisiae.;citation_volume=108;citation_year=2011;citation_pmid=21245305;citation_doi=10.1073/pnas.1012544108" />
<meta name="citation_reference" content="citation_journal_title=Journal of Biological Chemistry;citation_journal_abbrev=J Biol Chem;citation_author=H. J. McBride;citation_author=Y. Yu;citation_author=D. J. Stillman;citation_title=Distinct Regions of the Swi5 and Ace2 Transcription Factors Are Required for Specific Gene Activation;citation_pages=21029-21036;citation_volume=274;citation_year=1999;citation_issue=30;citation_pmid=10409653;citation_doi=10.1074/jbc.274.30.21029" />
<meta name="citation_reference" content="citation_journal_title=Curr. Genet.;citation_author=M. Minet;citation_author=A. M. Grossenbacher-Grunder;citation_author=P. Thuriaux;citation_title=The origin of a centromere effect on mitotic recombination: A study in the fission yeast Schizosaccharomyces pombe.;citation_pages=53-60;citation_volume=2;citation_year=1980;citation_doi=10.1007/BF00445694" />
<meta name="citation_reference" content="citation_journal_title=Journal of Bacteriology;citation_journal_abbrev=J. Bacteriol.;citation_author=B. A. Montelone;citation_author=S. Prakash;citation_author=L. Prakash;citation_title=Spontaneous mitotic recombination in mms8-1, an allele of the CDC9 gene of Saccharomyces cerevisiae.;citation_pages=517-525;citation_volume=147;citation_year=1981;citation_issue=2;citation_pmid=7021533" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=A. Morrison;citation_author=J. B. Bell;citation_author=T. A. Kunkel;citation_author=A. Sugino;citation_title=Eukaryotic DNA polymerase amino acid sequence required for 3'----5' exonuclease activity.;citation_pages=9473-9477;citation_volume=88;citation_year=1991;citation_issue=21;citation_pmid=1658784;citation_doi=10.1073/pnas.88.21.9473" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=L. A. Muller;citation_author=J. H. McCusker;citation_title=Microsatellite analysis of genetic diversity among clinical and nonclinical Saccharomyces cerevisiae isolates suggests heterozygote advantage in clinical environments.;citation_pages=2779-2786;citation_volume=18;citation_year=2009;citation_issue=13;citation_pmid=19457175;citation_doi=10.1111/j.1365-294X.2009.04234.x" />
<meta name="citation_reference" content="citation_journal_title=Applied and Environmental Microbiology;citation_journal_abbrev=Appl. Environ. Microbiol.;citation_author=D. Nadal;citation_author=D. Carro;citation_author=J. Fernandez-Larrea;citation_author=B. Pina;citation_title=Analysis and Dynamics of the Chromosomal Complements of Wild Sparkling-Wine Yeast Strains;citation_pages=1688-1695;citation_volume=65;citation_year=1999;citation_issue=4;citation_pmid=10103269" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=B. Oud;citation_author=V. Guadalupe-Medina;citation_author=J. F. Nijkamp;citation_author=D. de Ridder;citation_author=J. T. Pronk;citation_title=Genome duplication and mutations in ACE2 cause multicellular, fast-sedimenting phenotypes in evolved Saccharomyces cerevisiae.;citation_volume=110;citation_year=2013;citation_pmid=24145419;citation_doi=10.1073/pnas.1305949110" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=J. Peter;citation_author=M. De Chiara;citation_author=A. Friedrich;citation_author=J. X. Yue;citation_author=D. Pflieger;citation_title=Genome evolution across 1,011 Saccharomyces cerevisiae isolates.;citation_pages=339-344;citation_volume=556;citation_year=2018;citation_doi=10.1038/s41586-018-0030-5" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=M. Plech;citation_author=J. A. de Visser;citation_author=R. Korona;citation_title=Heterosis is prevalent among domesticated but not wild strains of Saccharomyces cerevisiae.;citation_pages=315-323;citation_volume=4;citation_year=2014;citation_doi=10.1534/g3.113.009381" />
<meta name="citation_reference" content="citation_journal_title=FEMS Microbiol. Lett.;citation_author=A. Querol;citation_author=U. Bond;citation_title=The complex and dynamic genomes of industrial yeasts.;citation_volume=293;citation_year=2009;citation_pmid=19175410;citation_doi=10.1111/j.1574-6968.2008.01480.x" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=W. C. Ratcliff;citation_author=J. D. Fankhauser;citation_author=D. W. Rogers;citation_author=D. Greig;citation_author=M. Travisano;citation_title=Origins of multicellular evolvability in snowflake yeast.;citation_pages=6102;citation_volume=6;citation_year=2015;citation_pmid=25600558;citation_doi=10.1038/ncomms7102" />
<meta name="citation_reference" content="citation_journal_title=Braz. J. Microbiol.;citation_author=V. R. Reis;citation_author=A. T. B. F. Antonangelo;citation_author=A. P. G. Bassi;citation_author=D. Colombi;citation_author=S. R. Ceccato-Antonini;citation_title=Bioethanol strains of Saccharomyces cerevisiae characterised by microsatellite and stress resistance.;citation_pages=268-274;citation_volume=48;citation_year=2017;citation_doi=10.1016/j.bjm.2016.09.017" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=D. M. Rosen;citation_author=E. M. Younkin;citation_author=S. D. Miller;citation_author=A. M. Casper;citation_title=Fragile Site Instability in Saccharomyces cerevisiae Causes Loss of Heterozygosity by Mitotic Crossovers and Break-Induced Replication.;citation_pages=e1003817;citation_volume=9;citation_year=2013;citation_pmid=24068975;citation_doi=10.1371/journal.pgen.1003817" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=R. S. Sikorski;citation_author=P. Hieter;citation_title=A system of shuttle vectors and yeast host strains designed for efficient manipulation of DNA in Saccharomyces cerevisiae.;citation_pages=19-27;citation_volume=122;citation_year=1989;citation_issue=1;citation_pmid=2659436" />
<meta name="citation_reference" content="citation_journal_title=Fungal Biol.;citation_author=J. Paulino de Souza;citation_author=C. D. Prado;citation_author=E. C. A. Eleutherio;citation_author=D. Bonatto;citation_author=I. Malavazi;citation_title=Improvement of Brazilian bioethanol production – Challenges and perspectives on the identification and genetic modification of new strains of Saccharomyces cerevisiae yeasts isolated during ethanol process.;citation_pages=583-591;citation_volume=122;citation_year=2018;citation_doi=10.1016/j.funbio.2017.12.006" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=J. St Charles;citation_author=T. D. Petes;citation_title=High-resolution mapping of spontaneous mitotic recombination hotspots on the 1.1 Mb arm of yeast chromosome IV.;citation_pages=e1003434;citation_volume=9;citation_year=2013;citation_pmid=23593029;citation_doi=10.1371/journal.pgen.1003434" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=L. S. Symington;citation_author=R. Rothstein;citation_author=M. Lisby;citation_title=Mechanisms and Regulation of Mitotic Recombination in Saccharomyces cerevisiae;citation_pages=795-835;citation_volume=198;citation_year=2014;citation_issue=3;citation_pmid=25381364;citation_doi=10.1534/genetics.114.166140" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=H. T. Tran;citation_author=J. D. Keen;citation_author=M. Kricker;citation_author=M. A. Resnick;citation_author=D. A. Gordenin;citation_title=Hypermutability of homonucleotide runs in mismatch repair and DNA polymerase proofreading yeast mutants.;citation_pages=2859-2865;citation_volume=17;citation_year=1997;citation_issue=5;citation_pmid=9111358;citation_doi=10.1128/MCB.17.5.2859" />
<meta name="citation_reference" content="citation_journal_title=Eukaryotic Cell;citation_journal_abbrev=Eukaryot Cell;citation_author=W. P. Voth;citation_author=A. E. Olsen;citation_author=M. Sbia;citation_author=K. H. Freedman;citation_author=D. J. Stillman;citation_title=ACE2, CBK1, and BUD4 in Budding and Cell Separation;citation_pages=1018-1028;citation_volume=4;citation_year=2005;citation_issue=6;citation_pmid=15947194;citation_doi=10.1128/EC.4.6.1018-1028.2005" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=E. L. Weiss;citation_title=Mitotic Exit and Separation of Mother and Daughter Cells;citation_pages=1165-1202;citation_volume=192;citation_year=2012;citation_issue=4;citation_pmid=23212898;citation_doi=10.1534/genetics.112.145516" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=H. Zhang;citation_author=A. F. Zeidler;citation_author=W. Song;citation_author=C. M. Puccia;citation_author=E. Malc;citation_title=Gene copy-number variation in haploid and diploid strains of the yeast Saccharomyces cerevisiae.;citation_volume=193;citation_year=2013;citation_pmid=23307895;citation_doi=10.1534/genetics.112.146522" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="A Case Study of Genomic Instability in an Industrial Strain of Saccharomyces cerevisiae" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11.cover-source.jpg" />
<meta name="twitter:description" content="The Saccharomyces cerevisiae strain JAY270/PE2 is a highly efficient biocatalyst used in the production of bioethanol from sugarcane feedstock. This strain is heterothallic and diploid, and its genome is characterized by abundant structural and nucleotide polymorphisms between homologous chromosomes. One of the reasons it is favored by many distilleries is that its cells do not normally aggregate, a trait that facilitates cell recycling during batch-fed fermentations. However, long-term propagation makes the yeast population vulnerable to the effects of genomic instability, which may trigger the appearance of undesirable phenotypes such as cellular aggregation. In pure cultures of JAY270, we identified the recurrent appearance of mutants displaying a mother-daughter cell separation defect resulting in rough colonies in agar media and fast sedimentation in liquid culture. We investigated the genetic basis of the colony morphology phenotype and found that JAY270 is heterozygous for a frameshift mutation in the [ACE2][1] gene ( [ACE2][1]/[ace2][1]-A7 ), which encodes a transcriptional regulator of mother-daughter cell separation. All spontaneous rough colony JAY270-derived isolates analyzed carried copy-neutral loss-of-heterozygosity (LOH) at the region of chromosome XII where [ACE2][1] is located ( [ace2][1]-A7/[ace2][1]-A7 ). We specifically measured LOH rates at the [ACE2][1] locus, and at three additional chromosomal regions in JAY270 and in a conventional homozygous diploid laboratory strain. This direct comparison showed that LOH rates at all sites were quite similar between the two strain backgrounds. In this case study of genomic instability in an industrial strain, we showed that the JAY270 genome is dynamic and that structural changes to its chromosomes can lead to new phenotypes. However, our analysis also indicated that the inherent level of genomic instability in this industrial strain is normal relative to a laboratory strain. Our work provides an important frame of reference to contextualize the interpretation of instability processes observed in the complex genomes of industrial yeast strains.

 [1]: http://www.yeastgenome.org/locus/S000004121/overview" />
<meta name="og-title" property="og:title" content="A Case Study of Genomic Instability in an Industrial Strain of Saccharomyces cerevisiae" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/8/11/3703" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="The Saccharomyces cerevisiae strain JAY270/PE2 is a highly efficient biocatalyst used in the production of bioethanol from sugarcane feedstock. This strain is heterothallic and diploid, and its genome is characterized by abundant structural and nucleotide polymorphisms between homologous chromosomes. One of the reasons it is favored by many distilleries is that its cells do not normally aggregate, a trait that facilitates cell recycling during batch-fed fermentations. However, long-term propagation makes the yeast population vulnerable to the effects of genomic instability, which may trigger the appearance of undesirable phenotypes such as cellular aggregation. In pure cultures of JAY270, we identified the recurrent appearance of mutants displaying a mother-daughter cell separation defect resulting in rough colonies in agar media and fast sedimentation in liquid culture. We investigated the genetic basis of the colony morphology phenotype and found that JAY270 is heterozygous for a frameshift mutation in the [ACE2][1] gene ( [ACE2][1]/[ace2][1]-A7 ), which encodes a transcriptional regulator of mother-daughter cell separation. All spontaneous rough colony JAY270-derived isolates analyzed carried copy-neutral loss-of-heterozygosity (LOH) at the region of chromosome XII where [ACE2][1] is located ( [ace2][1]-A7/[ace2][1]-A7 ). We specifically measured LOH rates at the [ACE2][1] locus, and at three additional chromosomal regions in JAY270 and in a conventional homozygous diploid laboratory strain. This direct comparison showed that LOH rates at all sites were quite similar between the two strain backgrounds. In this case study of genomic instability in an industrial strain, we showed that the JAY270 genome is dynamic and that structural changes to its chromosomes can lead to new phenotypes. However, our analysis also indicated that the inherent level of genomic instability in this industrial strain is normal relative to a laboratory strain. Our work provides an important frame of reference to contextualize the interpretation of instability processes observed in the complex genomes of industrial yeast strains.

 [1]: http://www.yeastgenome.org/locus/S000004121/overview" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11.cover-source.jpg" />
    <title>A Case Study of Genomic Instability in an Industrial Strain of Saccharomyces cerevisiae | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__6uPNahEPJnOkpurvaRxri2qBNHoapqvM8y1Wtdb7u_Q__sE1YcwwFB0K8aMYGFpNWEgt3sH5t9Ld9QAF34gSvqPY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__qoHMxai33VxgVvIHOVQupamFpg1jn-ylrf2gIvUolBs__Fp-C6VyxAewt6D0gYbv6aOnifLuGF1NoN4YO_dqmh1I__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"AqQJ3v0FGfmys7hbIqnfWoqiawqQtSVLbAyk2i6_dr4","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"misc\/ui\/jquery.ui.accordion.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"491247","apath":"\/ggg\/8\/11\/3703.atom","pisa":"ggg;8\/11\/3703","ac":{"\/ggg\/8\/11\/3703.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/8\/11\/3703.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;8\/11\/3703"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;8\/11\/3703"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/8\/11\/3703":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022S7CXqGAGxksSQms\\\\\\\/eKx+JytJLK+C7SF0h25Gisci\\\\\\\/gQzsHTgcI0NW0ZYBZUwr5PAvf4s\\\\\\\/zfIQ0FF+dxJ4FBQM6DO6+QLOt8ye0UIqqhsuZGpFp9plHOJ0QZdBmbdPWurIx6NlzLR+zZhgSBefCgkKG7wcnSDxYaeT39Dy5IfagHsgMtkKQINUjFZm+fqbvB\\\\\\\/HIo1la4v9TNL02UPyk4MfEKbfx\\\\\\\/mJIZVvItbedX7+38mI0xvszF7MykogqmQsSTuBrijhnMR5i\\\\\\\/xu5kvDzu5n\\\\\\\/hffms7PKWbpe5aHNYT4pWKIHruqieyW\\\\\\\/bd63wSD1Xrmya3tlyUKVr2gII\\\\\\\/r\\\\\\\/dtKPd45YavEcnzbNwrG7m8YA3WAk5hnHAl4wMJXiWKjEjjiORehuPDu2fv9A\\\\\\\/sN9nY\\\\\\\/KsAJIp\\\\\\\/INpUhnCYWMu4HjO4D6Pi0yoHsCdUsdIsP8Rq23iaCR\\\\\\\/h++cm\\\\\\\/HvpukwTnN4EImAsb4uoWRkBOKycZvysDL9LesE\\\\\\\/fzAPkHHuUU1vc5xng9DmgzECOJM5gr7t++XCw6RfCcR+FlPhI5Zhmhi4eFzFuaBtH8Rl50savBTZ4+Hyrnmw2tYxCTVGER8SNi\\\\\\\/YSm\\\\\\\/MAOSg6n7a7Ecj9NHH8lwQ5vXCRAqbtsTWxlkFb4U6Hooebr4JCawUMQHH9XibxoHunhPHYZIQoIZaqF2UIcZPZpTtT4wWlK6mdqv1icbpTCVL3hGxA8TtrJRnyNlomEQ7rtQaLlpO9sIJ\\\\\\\/t1eNvzKLsyYV7X9jVj2gjhEYtYVf2SYHR6ece0cUTCttSqfPvCWJSs7cMHCrL68VcJscZq3ZC4koWk5ICEdJ\\\\\\\/cCJRsKLR3pziJYWI9BIS3uCxGmhfKTrbfAnHz4idk1wHmsn9iEAst0Kvk7G5yGnSK0clD55ugBEWdFc9rL147kyEQYphDuTiEBD+0cL5SDZ8lYQrBZOZWu3WtP9EceBNHJ7NQqnUAZW013nGeYhn4AyknPlCQMG7EvgfdgC\\\\\\\/Z1O7BJZh9vUDIR6Y5V3Z2R4FtlRNRjNYy0fCTw6+u7+bKViJ3s0eN5YCJwR9dU5w6i++4hbi5ifa5iiz1scjRxlGUrb7GuwiyMhcmKYbKy0MUzzmYQy+I8Df\\\\\\\/QTy5PvR81XI0k8TI+tkVqaLDojUY+uNrYFQm60zdxlBK7fHaKfjmqVlrKBRNgvF+AmNcFbr\\\\\\\/RSvnqF\\\\\\\/8CrH2035eF0ydKCU6b9ufGDGygI1lj2OzOU9ojbdq4lPgMTwboimQqJ5\\\\\\\/K+FcrnfxTWo0uB6s8jqkVTW\\\\\\\/Y\\\u0022,\\\u0022iv\\\u0022:\\\u0022DKAku8VpqtdSI1uEKbh+Pw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002283fd5c6a0e6bd47760a7cba012ad39bfa2d1f0b4a727f522ca6ac03d27465c51\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00224S+HuwEGJ9cQRrvBbcfqQE+RYmlugsrkl0QKvGustuy2n4aTaVGbQ+CAv\\\\\\\/QcBPRzI8E1opeQT1NF6PUnsPSiyuKW1rMWclBpDZAhEDKjzV6MzCC7ed5YZbcXL1GmpEhFUZPd\\\\\\\/urmbLW2b44cmpeuCus1AKg\\\\\\\/Tme9DakPMCiUgT038d39UdubGeu3\\\\\\\/y39Y+RUhL8P6imJ9oD19Lvvnwjjaq7FijHnTrPcHgX8+G\\\\\\\/kQMZB34BHWFBCIWeeWdX8yOY2n3yCrdlehdkUKSYDYTLmLES2ZYzZBKHHef6wXYZoCkH2tyDrLiGnfu32THl5dqkdP\\\\\\\/y8P7xh6UJ6anvyGpRw\\\\\\\/fdIBKCrYRg6Fyx3MYltUxYbm6ITdNryqqZWJwxHv5a2X7MhSoZlncqpbZajgcZPqUJEFVuMVW1N7jiRD\\\\\\\/D67CnAjNkv7X9BBn3zhtSbJfsAp2smDHg7ZHTHg65xRYga5WRF4vIOgilC1je+1ojgA7oxYDYaocGLOGDydKQgWz\\\\\\\/r\\\\\\\/pQWPafgUClhZiz\\\\\\\/sUVPTCdJ2dbdhkWvWFxjumLOjhwh70g94U9vPNd8cM2VindwZ1GdxDd2s8f3MCfnDA99ASAoGn27\\\\\\\/X6B4t\\\\\\\/HQ87kWmBe7WQxCrU5ht2vouhezqBI0nFggphyUddMUByDazWDxC7AIouM0Y4B5FgbPzJvy2StuGKrWJHDntnCsbHMbmq9mdtlyUveX7xP7Veve1dqhbLVfL7XNBXlv0\\\\\\\/fJjpK3DNrDfTlxlvTgOEjVBzguu4a8+XY1evdkHzzLjkYopPw6q\\\\\\\/xodLmSHgAJSWe\\\\\\\/bTXr8ezf8F0lrzx38E94cvU\\\\\\\/5CPKV9RcSm6AsoirwRg47c\\\\\\\/XWUwEKRCMHcjE76hIWFbJlb7FAvFiHIYmrjFg6O7O1SRlyUNUMEiC\\\\\\\/0L14DgWeaGAV2TiR\\\\\\\/qwvFqWpAX8i5EZTj2le2zkbaK5GyxDTax6EiSWCC0y5loiz1tKCX5YJx\\\\\\\/tDfObeeLun+bFiR21yhOey67o5N6JI8Eh0D8KZRpJI9KUMmi3SKI0PVc21VY6iXtRDqwgEU7R14UPZhzTf6vRKvf3reJJNtleREc53xEe4ywKV6f8yCVAxkfZK046aLaaIBNQOmtfbcK\\\\\\\/LKsQXrJoQgMX5C7AeY4zzSqau3KvpWU9FOVqhNwGVwS\\\\\\\/nElWHKi8gDznbbOjG7HHEUikPYg3ERgtFvyTKqcie4UFN7uGpThCXzZNYrmKL8n1PnNcpaUKhv6RbEjY9j7G1Woh9pYFXHXYZh1ct2ftArmJaV5C04AckqoPEvcNkfR2jPT1dbd7yXHhb0OoHi3m3owDdj8ikDSIlE+uGeOhKN8rzcI02jTlFY2GF21vkwNDmVj0oxchdvM38DO1W9Hlo5GXjUbBFERHogY+0qI7Fj9QGN0cbnqQESj4r2MYI5bqkog4YrC0j4W\\\\\\\/ISMIUfiy3pEtYJhFK\\\\\\\/VneZ9JJUTVMkhWKDEPVCtX+GXKXraw7FAG9qMurGyZI3W3UnGRUuznNsd\\\\\\\/dzOEn\\\\\\\/ZujrRETmmjQoeJhOLCYWh4T\\\\\\\/TABfeQBgAagd6gCD4Fly0XL+\\\\\\\/I7fASqlO\\\\\\\/I60htiNuhCw78kV3AtOGpezzgpskKOkDM1995svD5JwD+ZnQcGFMmKYHoyF3rJJVExlslD\\\\\\\/gYuhV6U4QEKvM+S479Ks5l10fWum04Cax89tkaELwvKsSv1O7sCjRY2\\\\\\\/Zv71RMpbafNjzDI\\\\\\\/lk\\\\\\\/a6OZsRKfQOSwP+vx\\\\\\\/jKAV5Jd\\\\\\\/9iKdAvpm64mcc09c0PfVewRitIXO4IdB2ykRO2tFgxFmdd6vPpQaVBhFTrgGjt2qz3YqNwRshv0JxIF9sW9vuLK0Dx95CPAiPtCDTD5x8Jg5FJU1d0emeGeWCk6oN6QYW8ObwcHkrnQB0PDri8RvumqDH8xF97Zvao+hwCG7NrUPQz6lXtawtQ2R4VfvH\\\\\\\/AVqiCKFLN18RZolSw\\\\\\\/AJq02IFkbUDjX80vsrAx2uU2z5AVb\\\\\\\/ED1nlXT1OiOM+Evkwv0KmgqiIu37mlp9rv3TFnLycL4JMYcJE35Qa7NoQm+dn38H7\\\\\\\/\\\\\\\/adsZXP+eKNjwj+4L8Og5kMDFW0PW56jSKMXiDgNBL5JY6eY5RG\\\\\\\/HD3xe8BjPdRAordg6DCd+IvXMb5KAPeOzGkBq5WjXN7P7eaBhvZX8ko9Ea+7BBVrTi9LbSxv4S4mxljgBtjbfNMi7WsfdyInwMII4A5PhW7V5yGjiJLIzB8Tvsb\\\\\\\/TbVRL9Maa+VKHuoBKfcgig5FQbTPrxzCgZ2BtxJy7GwGk216D772l5TJzeQf7O0fONniwk4XuSAMvPpAB1Ry86mmb4O4QFMi435yTq8dEZKqU2R8daB7mPtINPIG22GXV8jd9zrczOqj5ANipqbJGabChafET\\\\\\\/wzZkI4Pi\\\\\\\/ytZTEgq8i47o5SC2Ej66YptNua+yqW0WQeBGBSGT1ESq\\\\\\\/5NAr55I71Y8KMxZWUNS618FYMbq3Bvd6kB422LBEKBqN5BSdeS3salbFrlkpZRAD0mv+CepHfhF68BmrFyGneIwumHNXY83MlrkK3nm4QxdXol1Sa8BnyP\\\\\\\/NuY9xl4lxKjMhsPBLZ7jn07LXrTAY\\\\\\\/dqFGUiQ74ocgYdCZNGnEWlY1bW6mGTTxCCFezznOlroeJJba7fQs3cpLVtya2hOGik0N0ASpn40LSLCXmSLxfka9BXY8uT+l7VmE1P3PQ8PsXpW7lMBYXMX3ogWd1dq6HDy1Wo6AYIx0FZpMhIOi\\\\\\\/PPNCcGR5nNITFaWeN5eYn2L1MnvukOXMQhdS4EXfB5TMJ72Tb40vPU9jGKzClSVqbPaxg8euOX5zFps\\\\\\\/NcXmQ7IlJRDLPap7ilLwKi6a0rJ6cl1UFsKinweY2W8\\\\\\\/n9DNymEQj5YS9YN\\\\\\\/diqFcyMNXG3jM754\\\\\\\/EgcrSQMA8rYLBO9B8odj7SJVfvWouV9SE3EbY+KJ23T9pdIvI0FQnRuLNuVoBM++gT6w06EuKdKIMijU1Tbu192AIn+RcQVpSxrk3AD9DpApEx5GOVrggP25Wr8PRZA+HCyeptF5Cmqa0Eu29yfPWvOVllHzJWLxW9zX+l+\\\\\\\/\\\\\\\/Y9Cx0cxhN9rDotmIqsmpYdmnbjkv57xhqOl99FIXA1v89LBB4IaiR+w2KwjYrYINL0qrAhEa7N3FTsdsZpGTt9C+Wa4tjZ\\\\\\\/4DD5YHiVk5Q1Tbh5Iq9smab9x4qDwnaqaMngsEDDvxvr0Ad23VgLc5i6E1y6zE7tqztnMYuQrtVQ3gz7Eg\\\\\\\/HlVZ2Tbu622Tln1QiB4qm0wAhaOaTUKjg3qR8ZzQVFta\\\\\\\/eNWTDHXKGMmMGfsGA+WeM\\\\\\\/tQjEGxX6Q6V19GwFCDYdTEGlHehGxe2r5dGNSIzWLbdrrPF2zHuxUA7WjvMdcDsvGAnaxJb2s05zxuN26OPvUQxVnKgSyGf5trCC+GUyZf1HALPoXxgErieGIXQmyHPGz2RY79pQQpJouXiahCXoF6kPSqJcTOSStl5OuyNx9t553S8OXAZzHumxbj+jf5MJ\\\\\\\/upUsqnA68wjXyISM5aUYnVKCnn8CzELuccRbkOBy+DR3WQIybnxBACpIMl16WbURNwLoQRkLCoeSD8lXAYBCC021+RXOA2Ah4uqde7bKSqBDlD8\\\\\\\/aIYnb3DO+Tkm2Al9Kn55fi8Ys\\\\\\\/dCrRUc91VFI8+EcwOEaPT08uVZYc16GJhtZBKi23+D+SPkxijKOWwKwjsPMe1L5gZ+giWCik97j+oq3GSctMWOz82tjGLbd1Rl\\\\\\\/3o3pR7edo9YOrdqIVkT18Na9ymEsFG0p+ZqXwNJx5QeXiuVLLs1j\\\\\\\/\\\\\\\/zjodhzH3GR6IHQnfzeIbDHHRLTMtWLNAEVYr+Zyandp+q4de82zgjaGhliVlJjGRh0Z77m3M\\\\\\\/INFRI+zW7943N7qrAv2phA4h2OCUkKzf3F6FKH7jTyftg2nV9qWSyqXQgAwY2Lqhaivt77ipTFTrSqgiH4tkh5ZA7\\\\\\\/8pggRbcnJzvQ46f3OCb9FgZO8c48XWhbvjRxAJK4BZEmRCeBC56\\\\\\\/nZWvuuTr8FcSJ5289itFg3JVbTPU1lGue0VxhFII8vICzrHMW8lSzLnGRNuI4xExLM4b2iMl6+XlrACTOCN1V0jB9jGovSC+1COLr19itpk2\\\\\\\/BruowOl5G3yke0m1yIJ6vwffrSO3GbtL64mka13KA5Z\\\\\\\/k8nCuE0DQvNWrEYf781vXrWshDFF2fR8lhLTYR2WHMx6v2lfJrpvA40MGkH3xEnzUNKASNCgWVRNqo+aqJLxsTNWIwRn3kyd0\\\\\\\/bsUGAWpbdr2DPZ8tw+AB9Mkwqs4OzlKy5cHBHQy7fxkJftphMjbUDJLgjW07L2k4bItK773yW1F+w8iN2fn2v\\\\\\\/5F9dhdf3XHCtFVOukeSGgxUQ8HDMceHe15LEBpnY6T\\\\\\\/UMdeuMd9bYImlAZeZXF\\\\\\\/VAvaUNaAYFGp8q7KrLwMGu\\\\\\\/JMu4eKWA1pmppSANuUoC\\\\\\\/mh0gwd343CgcUrXiRJ\\\\\\\/9rINHwRdG7ewahyOl9HggVSXOh1ZjqW9qWwJmISx747ZHhn4PG174GAkitPvsLCtoveWMDMxi5O0s0sFAB2radKUKgjWsqTxQRM5iJVfh1qyj9\\\\\\\/Wrhwk8DB+3\\\\\\\/fdWkLrkOIS\\\\\\\/eEfsbRPot3ds9t4kJSZqEq6RvWWrZUPMYo28H3Tv+hGeF5ejvkYnI+cv4KWC77HPIkzjlQrrfq8aO6eVLDaF3NbS0vxH80N7m2IM0xn3BPBXzt+oeq17qVpNVSxPPheO14F4vrd8M9TMro62TJfSkW2GBtfyRwx\\\\\\\/d\\\\\\\/HP4\\\\\\\/e\\\\\\\/4HBZI7mi1T+wZLCiKUvf2n2JohuxqCptZFpj7YDFaS8kHsBPfMAAfLNgpvx95UR42OYe1zRWGUbLVrKUJyruxI0XtGwxkJGsyhFEYPgNJJ3bdZIrXI+4hfUZ9OA09XKwb+F8ApTqMZF2IYzFl6XwKWCNww+iPHl437XxJO4PA95mkfeOzZk625OLbKCXluZR59O7XfjJzwwfojo1BWx7n+jqNahwozbG6yucNbM+sLvz3SzEj2P9sOmTub2jiQ9LV7Dpt0NTil2Ls4LC8veTfuIvU\\\\\\\/hKNgzp2A3wv76WN7EOQaKcNimcI+pFJIh5jkShWcc0DofFg2oeq93\\\\\\\/asJOwHDdZreQ8T4+JR1Z8zUhG9avoPmddk\\\\\\\/kCnPmPPtd2uyMkJXPF+hdqKIn4JSgY+N2HbmI5aDs9JSwOXMwVQ3eXBOGU6p66TTuc40EYliFO\\\\\\\/Tg74J4ImGOxlzjODY3j4ikHgGQIs8agS642hlI4Su7C498OKNmCG15ZRzHRT49CHTlhZYvoJJI8nXGQ\\\\\\\/xJo6eJSc8S1ksVfPnOMp3AcFpxKKS7WzRUxO7wB3kVkRt7mH6aCkB60ND2xZcRRewpsBaFROoPWjjZLX8k\\\\\\\/TC3nW4SCa40U6yulx+36gQAx\\\\\\\/4mGZlnzzXnHJ1rOylZD45eZ1lx\\\\\\\/+J2lNhKdxo1e5pO3d4JYm2y0yXOhxAboeo5wcs5+zusMwboI7zHk3\\\\\\\/c9LZ0mqH7xor9YVyC0U0b\\\\\\\/oLBjHTcpuua379Q\\\\\\\/u6Tn5+ukSeHF08u2U\\\\\\\/kLRkuM6S6cfFFX8D2fgERWg5lv8wHBwfPEpcYe0lt9wzLC2fwKIpyNT0bA9OFJfAN9DgZ5UEUee2M0XP8OcxPprK+inRZQRXcmQhmaCJGsSn2+gVDpBfzuJYa3nb25VgnXPyaJr6icks6R5L\\\\\\\/Wcmm3NBGDZ8llBKGbpi9C9f1rk\\\\\\\/Fw5AANllkSr74vYC4xKJtgfGlOGOc0xQZPe48NFEcJi+uqXmw1Boes6ePJnQYZtY4zjilY1oF9d1rjoxraJdmDsUO+Z\\\\\\\/ETpIhRlWgAZpjt2wdEcrEuPhq+TnZbuLcYtykwJqP2qT1EG4sL0Dmbroqzipqiq2H\\\\\\\/hrSdfam0yoMaN0CMC9bpDKHRUuF0SLlUTpXimbN\\\\\\\/RB4GJdF69B0cHCZrJ9FzZbKS4eJlNkMLaJBFxWhC4xd6cmb7mePgfk4klJOfxOS+bBvj0kzLxI66+DBt+M5voJXqq+HckGSipw+jbS9Qdr7ieyCdX4U1O3041lJc12n57N6hKkqUHOgNQFImW3epC4Ep3Z+ZfGBh5NDDGMDVKyUUQUQqxr24VOV9WgJmAXEylV7DWHpAFlvHxsp7RabasJ3cjIsuHgPg6J+w\\\\\\\/\\\\\\\/Ap9xf8zxRcSzbISbRj7dwYkQ1kzhpuVdaCAtxHG4\\\\\\\/JYiXnN2sOZqOuiEAzxvXXJbN03oH5Ro9ZAlE7w8YCXmmbx3ZL8Ja8rVmIBuOeTXlEAezRuARZq2WT5ODCw0laNKshIffW7zHOzfTK+QSNNxDh6QSvz4pjcBfohYuJ336a8P7NUWoU07atzM80l\\\\\\\/Utb\\\\\\\/Kif8eEagRpapuxegDyf9z6+X8to3BqdeTHq7\\\\\\\/5G3qxVwswmIftNI5TQBAfF9VOiZEL+XarORxaYPI8po+X9o5ZeceqqRRqtyU2g3fbpGKlE2pohMhJtZ\\\\\\\/dQjLnjZdxucFDCR+hjH1KBp1o97HZ9SR6ymjsEG1fBFTdbb760x6xarNhOuXLf9R4LMWcCCxphyjLVtZbbNSGCQA7QBMQurgkYoUGYXR+3n83F\\\\\\\/t0qzYuoUxPeVHn1aZgMbZB+v6MrXVEhEIGoyP9FbPX6YSOuGcdBvxR4PqwwTqDPmhTVsya3iv7rx84svb2xVXJjfFJG7NjAziBjdP\\\\\\\/2SgdNym6NvRdxweAMJnLjZZiSeHpnOUZegA80P\\\\\\\/qpgEW1ilOgL2PWN6K5AzYwJSU\\\\\\\/A4L5gel5lahU92RKIIJ58NwnFJTPxZ\\\\\\\/RIEy2r0aXpO0WZo0HW\\\\\\\/WgC4OxPbrpVlmf23Nr2lAJTAULC2c\\\\\\\/EEoxPvjjo1WZbTL6qM\\\\\\\/lQvdJkRQWxSsnz7hlprqxqd5zAXiTxcB75R2Wbn+9HQtsCvpGNN4CfkK1Y5EOYSUv3i7OxMolOcgvxndbvbQNvmb+vEQy5B8i66O4ahHXTnCu2TlZfSBmiqaOPzanDOU24Ob7m\\\\\\\/l1MwuQLMaQ064xn9NVUyk1BLcYli6tB0\\\\\\\/E8vMRIeTkcy2CwK6FKIXybX156ZZMdVsxuWKp5wkh\\\\\\\/3o59TC++Xb361\\\\\\\/GvSoEQ3djqy0u6cDcI2EjPWyEkwywT46f\\\\\\\/Lr04T6Bq96oUrOq\\\\\\\/0B7pN52BDzEXjnVL1vm6PA863gs892ez56gzOgp5v8ZD7DM5PravvsB0YHO2uNImQRBZ155kmxqsecK3jOyrluojoNa3U4Xe8wIvnyYHcF8tkFSbAT+X\\\\\\\/mgpWf9rww0NoMHOUnE9kS+ampY5tjt3eDAu3tsC3Gvit1gKEzRjQa7pgFZEz7SXci1X3jqrq0q2BZWIWBgtKrJHWiApR3K3ZynVT52cZl5LYR7W6kqBfJSNCOcC+C+UL+n+WNEzR2LEi9wnnfbXFKeWUohg55YEynn+HJ7jgiKZnanD+HLXBG+sxmjg5z6OCe1cwKk\\\\\\\/yV7rpw7MEuIZ9hpusuwA+9MwBRNaeG3vAQov4uEgJ62oceG2OTPHU+wJBTRMzpo4u0HXbaBz4ovG63ahWGXUhl82cvNZwl9qyMMcnEadUYQeX6hHOTKgwhtb8FtIHQr8+gZvMwZx7G6UIU1pQc8TJQPgPj2u3mpMFGCSUBmdXDXuJDj3CqWvZzr54IuOE4Hs8fNd4wTA6n1\\\\\\\/sG1BWjHUYIje5qG5hC8r8uriQV0MIkOHAbEenfbwQYHWESAyHn+08Oe2KkE3mbQxPpkOV\\\\\\\/9Npxw0GEcciR9tHlVE4KZVyAdIghjUvoaIOL3tBOp8I6vBkRzMRnrOVeifepFwv8s5lu8ySdBdUlXE9tdUWKCr8EHX0\\\\\\\/B5Es2fCwRr\\\\\\\/me\\\\\\\/EO+yDmfwF+8f8Kh+THzcliGJHechv7C2LUgFru9H1cF7r\\\\\\\/liQ8pMgK6+0HmGhBTrvZZyKwB3B9QfBOdtfwX4MK5OiLkanc3ANMKqWNL8FwJTI2W4xxiisd6AOPsrVXSGhUvj3qN3PlPi5f\\\\\\\/sjV\\\\\\\/hoBDNmVPMkZ2M2D6YeDNB\\\\\\\/RbhmzOo8tqrsWO42EjFueeAQnFcTqjPx+pr4z7qOTItKQA8zdN623zDo5Wg6dO3zi\\\\\\\/KhbdOBYUGGVUNj+nLHdzE4FWQ3HeAx0xoCSIB3eQCKh7PfbNAUN7S33v8eluE7crBHEpY2FTIskD9MN6XrSEgo6YqcCF04au1tmq5gyCly1hACZq9Gu3hIt2\\\\\\\/AoibHrvipFQtOuCDYCqgoEDXjkjxSFrd0CS9bUdD4JBXR3F5vp5StNGuL0XLBD5Z+tPBsI+M0cLgNWJ4urUGT+Xhr9uAY8Hm12\\\\\\\/LSTrGXRYtudvAdPIdKD3HPPYen8b7QWznNF4d2CRHu4hPE1ZecdZSrXgBI0PriF5dwYXZs0IAEWOlb8aYyoF0jliHIA0inI\\\\\\\/TT6JVn6EYwNY+DvfdTuu59W9toUvRF7d1VDXafMm6HQLN6HmhnCqjpthtIKjMNbm41PuPyr6EM8uHvoM6SdW5cb\\\\\\\/3vNGllm3\\\\\\\/Pf2ZYYV+gG05NfKRu9iY6dbD5hHfCgz7Bi0z1SvxJGP8WB284duf\\\\\\\/u3QgCuJ6+OJC1ClcxjOq77yv5YJqC0NT+vBHKCQAIEX\\\\\\\/AZlxAW0OiQGj+1HNIhovzr+\\\\\\\/fNzZaVY8Cs+PutNEFxuZ9NF05M9\\\\\\\/B37KkRUkhpJy19KpGnHWuV6opXrZkPUQ5WBug4usZhuT1e\\\\\\\/IH+iQxsWOP3l4ca2w8BYhdqTatBOJqlF3yF5l5Sg4BhZ4rNbEETf0bf4wPj3Iha6ZD2S2pgXQE0t\\\\\\\/fG8HaVxIIaQ\\\\\\\/4J3aITIwSThdTaOI+hIlEUCMpzI7M8+M6cTRkfESMiVxY5O6N6lsbi7wcN9oTX+eWlG\\\\\\\/rMdW5eWpIlxXGVw\\\\\\\/errE61FJHwW+tXHpVdRgg58NNhrqF07lxZkW6n+OH0uFMi+WE1NMIIlCGej6p0KAoCbxFmYCdxqvGnLT2He1tymCz5jPOTG3hAwkKfh3C2w2jRT59zgEnfjSbdJNPo3EESq62WYylPxmw+\\\\\\\/j2VdLD4srQC17hCPWaz6GPmkGgLpoZau4P11kKm5wGlAys7u3EmpxCuwMcSJO7QWYiGfmpez8Cl27Nzrfd223HaCWjVQMe5nnNTag03daGY5+g\\\\\\\/i6AWl6W\\\\\\\/6udplQU7O8qG6ohlR25\\\\\\\/cyGeffFKZkgSjivzyKghU9O9tkdCRgeuYza0b7ypruUN0SOQehh\\\\\\\/RBGKt8Nrh4fxTsRIgiyMPW4GZqncT6yK7CqM0o6GqLBNCrfM3gh+YBvuWAb5\\\\\\\/DYls004AQzP2ySWxoXXHWx1m2vR4WRrNOvqbzBrbkHXSZEj6WaxdKsscnARrR1b2Y4MgJSgMZFAAH822TrOccMGTBNTWle7nWPFVQsgdu258Pw9hdIn\\\\\\\/A8nKooYsGZsvwyjTz2BCLYK4xJh19eJppdbrHcRa27FjV0in0D3s2fyFRVLFOVdzQk4S9A4x4Rbo5wsvUMTdcBop274dtteac9o8eaf4R1pEgeDCyf1uDKg9vAoKMyGntjQdhJuOL6BpVO8cZtC6g7vZMGKoImt4VCWJqbMx28wPB8NLbn15K+\\\\\\\/o8Zr0oFpTp3SdxAASbkJ2fsE\\\\\\\/QdqwseI8qi5RkpAbWMkX4pImVi1KK6MEumBp83EzeSMpL8Ep9D6nnq\\\\\\\/wtURCcLRyl06W\\\\\\\/ngz6Z0Sf\\\\\\\/tdA8MBCTYpjT8nZAzDBbfV8R7SQMVotJmmEZUpYyAhuGYYIsf9hF\\\\\\\/GoijgD8tLjPlbldTnHXO8R7HmMAw8KMT5uxKAJUfpyu0frD2PgNMCpWp09\\\\\\\/trOSc9nIFH9lEaRfa6DWqt9oybH0o4stUxxetd3PI9nJqAby10RmXf1+wjjqnCmcP8DLXBvEx+u6fE4k83uACxYpXxU1y6djGRWH3j+YYgi5Y\\\\\\\/ORnCNHkeMLYEGvk1V5zkJxOEWOOUVLjz\\\\\\\/5hNCnB7QuALOHoF7q0FhmIZu8JNNVtFGHd4i6tgWx0ZvHp+2i6boqkgmU9Fzexs6kugCuF7E\\\\\\\/NizCSjBg7BTHoabnFiuNxBhfBSh0XK8Cu28yfRHE55EqO9CkEsMOkM\\\\\\\/KNwDjgHl09TXu\\\\\\\/cVpwUHW7kLYHDs25SHc6q51eaXVptoOI4IYk4cYmE3B\\\\\\\/1+FnME4WAAKAlDp5mVhGwAHQpzAbkK6vYbQEl7bnv5vsqOQWIfd8titfh3O99kMnFylXNB6tW5bXmDrsm0rEXb3IVZ7cY96nGGH+jXW4xACngS3eOX+DK5pSAQvTzZE5cz5PTlikJz3mzx\\\\\\\/uUY3t\\\\\\\/A8DUc24fcYUkQsQa\\\\\\\/SFttf646mHFTMxgSnEF+adH6CJDe7hiOYwkFOr9XxoQylVbDEzDyd3pwLEGcavIOdilKeQcjM7tqiZzbp\\\\\\\/+dE2SSm6HAZcg2bLjQTu\\\\\\\/UHXc\\\\\\\/\\\\\\\/IWUUf\\\\\\\/crKazkEaPVF5TJSzqWRZ0NPjNJuSdMDR8pol9Dk0dDjkqkIVk0qA0XvIqVMRcyLyEeYgADCNnfnhFrEJh68CfOwW\\\\\\\/TfS97oEJqtWwTmdIDjhQun\\\\\\\/mT\\\\\\\/93lPMwOjG44FKU13\\\\\\\/lMnX2sU3WgNFFyiVLqJUac6zi2UEg2encJn7OrtoA3nPlGtXPmkyY8p84aqlmdqj5RTTYeZ31j3CwmQpObKKa+2\\\\\\\/0QyIOSid6VFCJS5BLr9x2oYmVkiNMmLhaXv8GswpmCqIkYWfYG3pSbhjO0C7dd9kmSj+2v7w5W\\\\\\\/dKBaaq8TxkdvLW26NAFG3Ksd2+wqYK\\\\\\\/BMktz1pLM3Ck3HW5gUm8iWPQcvJ2BKevbKe+fmtgS1z6aEZt7SjlRI9zXerPsqszlz27MRG2HtejRW\\\\\\\/8QpM6U\\\\\\\/w6gzlU0AKPRZjWdtF906e\\\\\\\/U7RgovlkXkQliC7pVim4hMxHEDUgI5O\\\\\\\/8pu8P7IP2HZu20KNcJVlM8KkS\\\\\\\/Jru4COUqEjKFvOIwQ531cBsjbe6Y96WpjBQQHdEfVb2Yq3Nj1I5NJLcGKp\\\\\\\/yX7hkvj63VqG7eGm3XYZlQcRodHedpNVw8sG11xpPZwpZQ9IsRicPXa0N9W+R3WpY2qSUv7DDL7DNwekLp4\\\\\\\/9plrEr3ZsFxOYUjZUitmuyVEm5VUpypYyRrap43ucSG2xllXxGkSustk1B17fkp6boSUw+Hwrl8kZtn1mMI3xv7bq+wLjYO2qGLwe0CYrN3KI\\\\\\\/YU22vir1r9g0SAr6tzg\\\\\\\/6WhnhixEXkz9+m0Yob6GG5Mmn41a8OoP5IRDLA9qQ7l6lqOByrAxPnG9wSNq0oRfV+YD104AIopDPqOR+SA1owydp1eiGpbhyiThOe3bQzD4PgXFru2VH5wXOpyJNazi+OFpJaXPI0AKpcZ9eVp8YzzVuiyLvBtS7ByMvm5kyH35cZ+as5GQ8ixt4+0CK7hxyMf\\\\\\\/Rs+abQCGADZgF3R\\\\\\\/5HslBGx+T1q\\\\\\\/uzlFzZf1oKj3Pg07IOCqyIV49AWZjcn44NDJ9tuWTQsAOZ+mQX2jbksLIqy6BMPJTkq\\\\\\\/Knclmzmcky0jCwBm1irJoRLW161lsQm43Sv3Q4dDGr4LJg83SMzf\\\\\\\/HWivWXeJl8ExiMliaGeYvVimQZjijtdaxYjORea2OGKXflFaLb6YxnQc0GMmfjePXvg8RZWA31pR5KISCcgMjJQx2yvGyOUNbc4o\\\\\\\/D8nE4gA4TRMQmjcmcMmm1TJcuxfaeLWR8\\\\\\\/s1tBAYsrarhemVO1Rl11p7OF4JBpwkVJ4tS9ccSOgfNwNXUwr1zgqBKOw1LoG8Rfnvgu3I0jIQjiBavUZ3lvkylvbeNUrIeRkIV6btCfkWYIHMnJoT8yuAi6k9lB6DTlq0rkToE0yL12OdSz+HIm+0wm0TdRma34X9nNMUx6LcqJh63xi7skQ6Z7+ZW+jeF53rlo0Qfv4GK2+LgZI+4e2lR1kb7+MTfsM8F4\\\\\\\/TtGmaeYkNTV\\\\\\\/xqI7Y29zyoPcnzftEgh7m6jLKlstiMdWCJyklJQdIB4zPJX1AfRTi7V1Y19BgfThwPRsdv54oaIhKc6WC+vXGOK86l9GUDXMYpQneXEq4DEsK0NTqXo3l1UGKNWxMnXSckvtAhyp7xtmaqRJfUCTWSrpvGxxR+3yDRSUG+Pq40pdjgFDvqjIovTv83RpVhlIitAPn6kR0+jfQ9ibAu+iLRzcJqPXqJIj5AlS6AMdBhbWvMVy2QGR8qGRPmRgE0RWPzR5UX4FJ7EA1ApKXEO7FrBV7QUT3Rrjk3WdCqQIfUwLJdUG+sUEjmu654DkC50PBF5BFQvq1d2qS+W4vKvaQdQie9S8Qx2DQ09zXUykNMrhanEjch3s16GrN0SzmKXuBRvff7+ILMOPkxwaTEOQ21rAtaEWbD5Q3n1G8u8x7oeQAerbKu1CRlTLx26gkPK\\\\\\\/Gf8M4jvvCxE1BAyeKlo5booV+o3dSWXQlpt\\\\\\\/TGfqOkeXLL2AokpqEVSKBuitwzIz9J5FiGqYW7R6DIxDhaVzWcCbR7oAo8zhv+5nC2V8D5HFaTbYuALXlnOWlRnYxWi8AN4CO8sfz+AKv8w+qXS+9AJrIUgvedhlfT+54wKxkoNMLTnKaO5P+VPbovVq8D5yqaq2TDem1MO9hgoS8kejca2V4m76pwlpz9KhpbHf3gbUfiSRrAEKXvsM9ZqrIK8kvOQZ3AaI\\\\\\\/tyng\\\\\\\/PYHl+1+8nQ8t4jBWfgu44DW1Pzl0Zo5EqPPLweG\\\\\\\/fdTz4bhMKXGGaRSVP218ukDJK8wHdiOFPY6CGFYA9vKB+k61cD+dkk2Ca\\\\\\\/6QwE97KxNXrgBuT1aQPiLD\\\\\\\/bmkdfg\\\\\\\/GBCR9LMHi260\\\\\\\/14BGmUcwDYbsYUaJObyuucjB1y+kIbwcqIs\\\\\\\/MgtwiNMIn1IGOnt3vLbqxCYEFwr6j1CZtReZyP+tGllchdhdFs3BVU8KmwlPzKjrU4Q1zOVNU0CXqpYTiiZ7d9MewZiVAX1ZAwTfXkmNzCcss6qs53ZpDBq+vYS41sNqTGNaR0AbVld6DujDpzVGyfF\\\\\\\/+2NZyefsZNliSSn0DD1oxPrPFMxOejspptmJyvxrmFm9nHhzBsftEkcfNAhJwRRM6oJtP6jBpro9RGQcGqIYcVeAlccJokxLe09VsLkTdyKsleLnjoVQns4y6UZiogiTIVQQeF2ZgxRhVBQvK6ORBbIRTrlQAyKavu5UJkbEKs9ypTWs9ioW0mIcaQ5lZpGwBM+PfX7b7kZV3gOGbl7+58N4BudJHs4aZzgVpV1RYMR0TvDeDklDuKIp5QZAErIncrOC8hewRd2nvwBs3O3Daw0JZpqovop2iAGABZe01bnsqnInlhDUAnRk\\\\\\\/hAGQzElt\\\\\\\/V1Uem7C0ow1Fdt4DOmEGuW2FBOkxmbEBFuvl+55HDlRuGZ4GKvtUR9oXIqt8Q6RMhOy8JTrcDU9F6NlmzEBbiF6gw59+U5qGy6lZwmddC2caukTET4o4MlY2Zw4qG0nDdMFYzvRJYyo5Sx\\\\\\\/LDYbpucoPzoSKthgsgPj\\\\\\\/ZhtP3RsjPK\\\\\\\/gitz6Q4izrqA7KqDQylIfnHESigLNdBJx88+Cg80K5tYty\\\\\\\/e1J7GC2hTKilwJVVCmPqJwO45iqxEWjU1gRinD5alodAnftSdpjBjBaz60k+K8uoZuIJHu++EsJTR0ImMFoER3e1Ky6IzMFZE\\\\\\\/JGoxEWgLfPziRWILFVNACl7EsK1FGQcaNDzROxRwgNq6rE3Nv829IkP5+6l9un8iUXWXqiRL+mSC5jPru6sz0dcNCsNvklK1AQiUZbQwpiQLHj5y7xmqBqqVIgeVRXKyVG0NU3h4JNzfQf4JopKVOVCwQwGy3EHvt1+3UTDYJ3O3MiL4awKfxqwQOqrRuprKFT0mQg3Jmnsm+1JFe015ZZujrUULO1AzPXDNiDW9y\\\\\\\/fXx2NRkpExuo5rg6AqyYIgPuJmpqJ4O1r1RQmRAi5\\\\\\\/exTD2B01ljJ15DSpIa2CQuSu9Bs4eq+YJft2t1OezBi2E5yaPdSJ6bI3Pft7i7m88ceDf1ibIhpBXaJS4W7NlyXLsDnYWQXB86lnFji8JzczUsKFBvO\\\\\\\/0DBTXHz+KroAWmZnOUGt0u\\\\\\\/E2uOAkt4i1r2yg1wIyp9B2mWRHmECt3c6J2GxlmzK3WHIVGqcqLyvSkc77h1XczDlzg6w6jRyatah2Jc6qYVmo1HuAV2DLe0f1hanrJpFMzlA99QjPrKhFOzEXzG2tZgkPHbWIxUf0ClFtmNlvEsYHRdhyvzqVOxLsBtg+8v+GQmoSgKlETXbmsgnyjdK6EHrLKViC1FquWmnI+DH7CF\\\\\\\/B9sXZiANQPc7cHl2xxEg2s97celTcoB4dLdBrIOcBk+a4ecDfJ\\\\\\\/RCpZ49tkY7NDI6yDiLe+VIxsA0xd+BIEqWIT831n4JEhVP8iZswTrpuo3ehgh1xJwNtusZWjf9ssPwX+2p9ny+jOwoe8xS9nasqgFNkKncVZyvKyOl10RQZmQzRu6ePcGqVtLkUjZb6OFEIJcMkVyXcXYmaAeGfj50ni+w8S0irPLP0490sZxlDwEZUvFuosnSsfdDzqSKpqCsFgY1Q+wypRIWtQbSY3uqzrXQpMR1ytGYLRV4YnoJQ1gIRcDgf7O6H9P8uZt7O0fZ0RG7Ag+WQcCuJlbeNluAhBQhnUb9vVl0t30\\\\\\\/BZhBkbI7Xs4ZTJ\\\\\\\/DBvBykujr4wN9Stlt5ptzHXTvXS92ZPHuCi\\\\\\\/CzqaVedA07ntCJ0V5kh1ovAhK5qA7Ye9+Pn2+BkqV6ZY9XiJdANR+SKVoOW1DW9W7n3fsht7jqeBDt3WcsSA8wVK16jSoG\\\\\\\/GZAjgOqYmn62r9vEXzmFBV4SYF1120USAHzu0RVFl4Xnnpi2PMKMirdcdJZGVI4VgHc24LIm8\\\\\\\/lbk1GN9yH0+\\\\\\\/t87CnXO+N6eOCCBvfJCJ0IzPt9w981twvi9V9rld0Td0OuWHYgL5ghq8Ytn4rYvnYdiNoF2cfk\\\\\\\/9Y\\\\\\\/8alEE4R3UvvvO2iIL5EbrK9Po+JXcrKcqjEn3fxXG6kOd7JxtXD2B6\\\\\\\/V0SxwQ+SZK1ragZSFVDgJR\\\\\\\/LD8Ae9n9IYKLwXwksHVIjKYxZRHuUDCnnH7RzqvCuHoxeEizhGpHz0R\\\\\\\/OPKrj5Pfnmf4J3ZcVbaWMqmBCByneRYBo8TNp+pqg0MBRonuEg2NlK8anyk778HkJWHBIUVYq8IAd92g1XYMuL8+yhyWTneNtPPZnp1L2a47pQhoVIUhEtPyyulZVo08E28pJMYr613aYvy3rfPiJPOof73ec22tK4J\\\\\\\/d+kKd6df8nApfLeObM8AbtcI1B3xDQD+qaRbCrfKEq+rGSpDb3lBr2TsOpcyFhXiFH38VbmTz+kBBtRlFsh2yqP1cxlxhsPSclTf9dnqfOi7WD2vU3OJB2jJklD3H+iiTLsX7Rh2sUzdFoLFMVRD99iHn1LVVIPOEYSQC8h3M8HKoKew1gmoBm2xFJUJb5+547ZuQnFD6YDRKEDhUwB3Qz4mB6bti\\\\\\\/BZHBNUZf5c3wh0JzoRc4bZpI3GCoJL3SLEHaYk+PJYkfUebG4ZVbkLvKQL6oejU1yILUh5aayuMVd8B\\\\\\\/9gYoxrY+odVX1iFysBWgWScznvnTxqjZJ8tvgpfs6pEhLkPA9XL7pGf5TQLpj8uve8nKnU\\\\\\\/z+7vytNZYgUuqmrXGSk+Xcjiz0vhh3VOG5VUENR91SzaB0\\\\\\\/+kpzYpRNCkvqQI2wMaeYdCYEsfywIXVFiUQBvzYIw4o\\\\\\\/Xd+3EXRj0BuYzXfsxwRO5wvDqrvMRuNUuoV\\\\\\\/DHkdkjv9FehmEfO7scZxKdAlDwFyxEjP1ClaDzwi9MF8k3FgmLnB6F8992Y1P5pP56stE\\\\\\\/tOQjBRKUmmpObt6IIZiRoxLch9GqxQNT\\\\\\\/Sml6hmOoBu8gSY\\\\\\\/Hs+R0hp8p5YQKMjl6B7B1LUIEqthQGUQFRk0NoY3NWsrsOx3dim3VAtwn0oCs2cSsKkFAiuXB9B7ryq1mA2yn7QOsUlcmmkRQOsktSVel2O4\\\\\\\/2Bmu9Jj8+xQV5+Loq3qyHb4kVS73KgTbfnY4Y2IOmXqIfzDwzJumCS7sfrDXUlgbu2i3LRb5kFIspVFfyOPr\\\\\\\/VEqwyQAtGq5ZirUM6imB9Cp2ReWj\\\\\\\/swxEwJoQbevl0OcoE5OYxe7UV+p2vq\\\\\\\/R3SG7oTOaIJcwzY0Rg6c4RepvzoCRzyaDl5sa2RTfKeJSuJVGmFI6\\\\\\\/8NjN6dRNcPzB4p8JFrJpmUddBciK1hA+XXxqNvHBHLMFdmCgbbVavJLJsisBgqFeD+T+d0enuYeuoq4axEbjJA4U2fbUYmfVx1NlFh9lfKgw4ypWittMyQgIGaiWxGlVhefQerg+2UlekJ1S2qu6VGDyHBKCCjhe3rbt6y6t6g48ecdqcCXbss1mb9kmNirJFXZBunQf429PcUOpQtnhIbZTKnpGhJQ4yWB2sO44VGaV4l0K7\\\\\\\/tRF34hNowZbsCGYOkOH4gZF7YWMIJSqG7cI8juETKKGZgVc+II\\\\\\\/x5qScdA+rgSQKVpsV3qqfK4aHldvlnUKf+bAfNQvZhU09bV\\\\\\\/GAn4EK5WWZZTLCeuliczqPO9h1GH\\\\\\\/EwFwY0FFKykYdAOYD6xX84hMXslD85gXZjHCX9u5k6NaAEl95291nHiAyuhRnx9W\\\\\\\/elQSCH4r98F02SGf1B66RSabekavBqKWh6TGSQtfL6tfB8QHhfbLJWPFMEYXLd4wOZiCVusw1MN\\\\\\\/lQnq0efb94x+40HD2pm9\\\\\\\/sPuM7qTSXfcBJl6xc\\\\\\\/XiziVrMUPmyOdqZN81dp3uxDI0lJeT6a3MWmfFwxV3CUPFswYyAQ2FEQr9BkURkdZV2\\\\\\\/\\\\\\\/gQIs+FxPL1D54Yn2gigkb9JSonK+DWG\\\\\\\/HN+KsKwA5BfLJbc7kXARMj2aNMDg\\\\\\\/Y36GnEsCc2EkYhujvF22JD98EELzY315I9lY8zzV2OD4+pdMq1uaueHCFS6Dv9O9A\\\\\\\/rl4KrmscMEHK2jcs7ZC40W+8HxQZBWTR9pRFNzEXdCueFU+VDvBiRyhE1f2tGWIIIQKTkU3CKUAoV7GBXylO39DRW4oGKTri64Xlb0aYzLKO7hqf7DPjVqGrt2rel2uSJ7MHnQx9tQR34c3F3PwCvlbBcAgm0vdg3t1\\\\\\\/4od6nDpeE3sl4t0g90eqVfNIG2St1dBPCcVDTPa0obBVQIhmd4ygI045ZdIR3A2UOhzhWFz6asU9K+z3862xv9YOUGVMrX2eYPyIhkosfM\\\\\\\/MwsZUqjJC4FUsq0oSE4cu+2bzcX8XuknZUNCRVuOmeApg5rXIjh79sQBbnDLI61iphtdb7nSxXyvrhVgHXImZhYNOdqpz3XxsR6AgjmcFqfJsMHCDxgiA\\\\\\\/\\\\\\\/8y2KY+YuMEGn3kmoKMyV3JdBg5IzfW6DILrs3Hm5WN5GlumEMRxSqZVMsY2TSGEednGvtOlThv9Wp9HgF5HXLk4+IufAp3qdbznlxzSsRMDwg4z15duQmSF\\\\\\\/Kl1EvvEsYcGKMcx40pSjw2NTN+9gDcBoV35I9LqmV1hOnQUJ\\\\\\\/h8nVnuGE8R3EmJPIu\\\\\\\/tciIaZUl3fG+U\\\\\\\/XMuY3yoS+y+6dvrKhFBUVvu+RnHra6XaakpEjz4K8lBWeiiLOLuSAyJpm9uAgey66riAHLFqW2mGk\\\\\\\/XKrD8mi2\\\\\\\/XVkqluIEEtLkjj7JBh5oXXnd176s2G+C+J7mbIvNZ4MOqz07FUMKUSL5uSMTU9hqp1m0JTULB3fZbEXI451mKesN+5Z526IiKivvC9fAEImUbbKUBpe1Rd3Ec+m699Sal50glSg2wm4HRd1o3hO4obTUQGWnkCIVf+iPmB7512BSTdIBKyVG696iInnC0vOlyb8qt6ar01tcHuxLVt9b+EshQCeDRslZQjJtIYJhjKFgobZAWK2SvvICQGJ1NYGbELKxxlkc9rSWTQD\\\\\\\/fxqgL5VyhBS0KWlxyfqs1tAC3P9DJBW3Gb7mrXoupq9H\\\\\\\/CbdWxCvIg9e2FuvMa9ntu3RpIZ3XaBrRLQCIFUU16iGy0pCWlGC7FIgBapkLGfdYhFUUfmaJ2UxDDYuyB4Y4Uzb7IYakVElzt4usfVeYEIeRPYla0xh0OmqZzmnE8ezjI406kTUIsvh2s\\\\\\\/c20\\\\\\\/n+T0QJ+g4yuSeM+7KHVaWxwB+\\\\\\\/HKFhUYZOndJ+1JQuggc7ymD\\\\\\\/SQpQwNTHMhyDHMsN88Kq27oWXER51DSk9e8RbqPNcMkQcj604rIrzkJlCAa5AwJzRcc3acnmQyMc\\\\\\\/Rn287kjyKwnJUppVtifHK0UIZ22SixrAOpT6i4RjcrYPZmWMmioBj7bQMUF2g68VZfARopv9HA\\\\\\\/0WHKt9CKGhzzuj4rlpZk3och+L7XLqmMvNYQ5e+ttKu2d+AUAc5slsGSpFFeJuQfkonQgf1TG2QtT497B61lZBb8f4lmMFlAFcjEE4Mdr6xFLZVeM1tAC6Og3VFvDy491lBEZw1RS8HhWrVvxkiCghIThywVU5bCjVKPCAy7TdK8jMfIT0Sy9Fdi9Bl3BWvYNbyBWeOhg1+umEEDqje8F8cigrNrWkUFmbskajBQVvEKzgLZ0jSt0CY7dMVB7Fy79NDhOt28K9tw\\\\\\\/S9JIw9kbPvg3lgi48OJz8WAYT6WSNxfPnhSp3eInmHiwkf1vXS1tXiMs8t1YcagWHbXDBQDmUapRjFyZMsmlLz4agt3hQCc\\\\\\\/TvEKX59XwjZb+W+sXzGxUG56rE0iLmnjukkIfb22mLZCAcAZu0cSRem\\\\\\\/CwM6urkL\\\\\\\/pGJ7RK6QA+pQiAfb5tqw1EZwPBr1N1Q43QNMf90DCLRrrEEfYM9WabUn32VTsTfXb0jqVlLuYU2hxmROAofWKqltWqRUPXWEgGiz6mQrZHIr9cAxtkrzVz02lO\\\\\\\/6SkpNA+W6b8Abqq7CaCAKOGSWUOLK5jQ+2Vew\\\\\\\/Pa1qEQ2z5FVbk\\\\\\\/kMTiF5aBHuPDWRInPYM0yQHKUn6MnP9mhqFecI0bzVoHpQB6\\\\\\\/DKilBNVx\\\\\\\/i4LevygFCof\\\\\\\/nbydWNTKzhfO7KgirIvVV6YiEhTQ7ALFGAvssFxNgapa6VRPqrxMTz\\\\\\\/Eu0oacchozJGJeB2PF7J0GJEo1ydk\\\\\\\/BexdbGVoqU\\\\\\\/D+BYR\\\\\\\/AMMNGbMil5RHCFbQOKTxt4LLqJ+8WW0zq0Ze0neoyzX2uhqgzwUewXmB1JzsfAeMC8ZHJoH7s\\\\\\\/UeLh5blwStSzaOg9INcHgEZkq5a\\\\\\\/HcmgHtY48UMuz84Yp+y+1BfKn8+IbQYKZXZFMGCY202lLu91Plod5A5aw60JTxkTzlmBlVFZPIZAnHVlzjspmZnwfFfXnydsKQmtnuzexnUYqfvcas2ZYF8zlNrteRSLXV1Jd81Ws6aDw2W1iZvM\\\\\\\/yml54IDVAwqmLlyOT9rX1Dz2JGlyM6fwF\\\\\\\/XFBDwt86lu89iDgd3ffARERGSTxpBboG1PNYGIEeExajuDxlrUOgDks3jmr5DS6mwlCM0L197IrZkScl04mYqRIaFEDMi3cvyo7XVZkxjmdgP8nq+0AgZdVTbHN\\\\\\\/bGTZQikcRnoZwQoirJnh\\\\\\\/qCqssBF8RyJhygUWmxMH5oigUhxTAV9YL4tgwZsw61fKlqyAlRsbbRttj8LsogwNNVH\\\\\\\/uKZFxtrtnPJxCcyFZuZs+P4q+HooCKm2S2ltOuzOYb4wytPq46Y9sDIaMje\\\\\\\/74BfwBn+SJxdr2n19IToSaiI2OV45gSacNoHzg+2iMxEB1GeLPfIMKg8cIherzCdkUG6XKEQLROj6oA04Ca4n9RqP+iP+gubBnr76IbhOil7plvN0t0T\\\\\\\/1tKCpZPIE9wO6uYJWlOI\\\\\\\/RaypIcLkE4\\\\\\\/BT05IS1BZcSOuuanEJOaCdDjLb4Gyedc1LPVmMuDGdJMQ75Tn0AeRSTVWMgyJKSuig7OrvNIzP6ElIPnUxBCxlUHFQ1QEhD099G+Cs+GzoP8zWv\\\\\\\/NtEkAmkCncUDEy2XsNPx7dvyIFBTre8zdCpakvGCD4safhs2TVyVEjui0SO+VeMlII3IZVW9ncYPJp9MvEPP2BwueaJxAAIoch0HtwpVygFYEpvi+Cq7X8F35ZKxqtbXUK11Tu8AsOYzMDo7pkEoVqEt\\\\\\\/q3umjZRS+ZwySbroXzrfPfohFblvkJIL1w7lo9+vRRzJQcBLleo6q7dsqnfuvO5DgdwH7LKz7guQ09Rw\\\\\\\/8m9vvLYvF8tXhfVdZLBWgzBnf8raoxZD1BNGRnHIUVaiq0QAOxJddGoOTr3gwSynQ6Gp2n+fiCHduyEEc5EgxgVZBCFxxTDiDReQkQpfhh2n\\\\\\\/6ByS14Sa2BmWhQD1t0n8Wg0PJn+wYTwbUHHbq\\\\\\\/3HMDXzIjeZJDXHibBFpekWZnQzdpTdmuqTno8rDE09iE\\\\\\\/nkIlcbkOJ+cYVLwsg3yxtccp7cM9hkyqRexKY6JXgMzEGUlEdNT34IO6bgPMTW8NzIE6Sdiq13C0CNWVGRsdEy5HAJRes7OWrVpdWUfBc6BrbhCpeln4lIvU9prKn1J9CpmQAQdgO9++Lbc7m0x+ocUj+oC4xX+lOA3po9e8zr7e4r\\\\\\\/vQS7+EZBu64yrXgqzuOC9sCxNYjGtPRavMpT5kYSQAG0q8FCIuXvWR6YNY8ZFBNpavis+weX65RtI9L\\\\\\\/NJeH6IwrX7D2\\\\\\\/g8M0OsMoVoZebkadMgTnPt\\\\\\\/DcLkcWN41UI3DkmMkMUSAxX5J7z50XPwouJMOA0HuX\\\\\\\/gMNvjG4G4yIF6p4OQ2E6ANEiamDcnk5A\\\\\\\/VuZHjYW7onYTDkAGzoyFZ7tLJb8fTqmzUngFfT7CI2Gh4e4Mpju\\\\\\\/u+KVnFPqJOmNcLID9RO\\\\\\\/j1SHg7AK9cORaMdrJkPDfyclI2xHWs9ZexgUFrs78TzpISXqbKMXXi8qj5kcpqSitt5gmtOqCaf6BOWbebvxGs2CbVR4wHaBK9ZaIorJzlQK4MbHjAD+5DOr\\\\\\\/2PC7UFzJPYAW817KIxtz31r23cDOfZxNIM8kxRR4b7sp2AnIjuC4duY6guzgEWx+w0VCE1Y4rYBO7Omrl\\\\\\\/rsmSrLZBHIX9SVbO87Q5ivh4STkEJyeyx7XeS6fBtf7DyLw7v536EcwL33AVx8UxhTuY9bMD44URTKkxkE7y2CsvfIintGWFDtVr76HVuChXvHIL0v+xhb\\\\\\\/QKVR4k\\\\\\\/rEg0biBfkGaG0nSxnuFvZsIz8bfEWt4Xmh7219YjJN69YF\\\\\\\/16kcvaR4OkgXM\\\\\\\/MUJCRgs1GyH\\\\\\\/\\\\\\\/g4Stt3h4VvJKo+0t39HZUvRWH+hyjUPMTlLth06QQdGHzJw4s204uPancm6ESzIgUjx0H5AUziEI5FJvA7dFTPJGFenRYYAVfVdnlGCCHKsDmHNSRSYuFnOxMIN0AuVWXQmIrrGPKK\\\\\\\/+XSalNyEUi1SAR3n1Wix3uS8AfDo8EIOwzWkVlt8dNFId5C\\\\\\\/1KT\\\\\\\/TBg2EzaEjvyF\\\\\\\/VoRZHKdT3a\\\\\\\/fAHfrSygiX410M+rYuyjfQuSKwoLm9a2vZtPWoKQgpPdnnB6eamULoebn0Y524dYpvwSg5MRdpReu20N56+gpc\\\\\\\/Okn03NvunzL0iPTmTN9rzuWtkB+0zWFVewRZOBXImmlPQLxY8T6QK4vndqGIvvnzS11l6RDPmT7INAmLurtZyIaqwH0NGwb3qCP7g4+FtAb67CoxQxkONJ9oz2AkW9o3lnjNhKmE520NxGZETE\\\\\\\/HtkLAA8R1FAcJPgKCjlwLXlV8WMzRJL9cUOtVxYun1ALv3PcOsrT789yNXErAJZgt0Jw1J2xnnAbTwHfFnRiuV\\\\\\\/Ejav2yA6OsYiN0dUwd+cw8SW7rozYQxnwV\\\\\\\/n1AQ2nmJakMRvuFW+XUO7ynx91V\\\\\\\/I+2qxfA\\\\\\\/E0t1MwCngSRLas9jkKgoNiMq3GNDS50c0vNhsjXr0\\\\\\\/uIpXRA4V3hv5k\\\\\\\/13SG+uSle61B1N1BneMkPQuTp2rSFQKJttjOvnNpIUsZ8YDgOIw7BqkNnFqC\\\\\\\/ux\\\\\\\/iVphjGYEgkO+4CVO6S5UZPujDdFl2HCc6IznULN818DYr\\\\\\\/oQJscmDr6hLB4TTJTuSufOL2hlsMmaflH0PY3\\\\\\\/P2\\\\\\\/wmHMwIoWH0PucPSZuLnnWLXPhNiZbsTj1qfHRuOWiVfsrXadPJHp7cgovSQOEtAhhWjWKT\\\\\\\/oJpTfs\\\\\\\/c6eQIkMFZMBNXtDu1o33B3I2ELz5jR0AMcbmI3rwU\\\\\\\/b\\\\\\\/DJmRcjtdZWiZGq\\\\\\\/cd1HVxJ6ZMXfd1cFiPTfQh91YMBxa+5R\\\\\\\/sX3lVHv4NaGpOTcUx9p42K1mybtJvfC2MTiUPpndJEu1Z6b6mNXy+DyMqMJzfeonAb9wmWSJMxg6wps8qY\\\\\\\/g9xT9fUaVtgvo711j\\\\\\\/KWmPyDkLZnAkNCYl5UpQJES0Fklk90Tu4b8BUwNrOTzG7V0LNnzLdYZHsiFL7P9VKBXXJ1GCO60b5JlpGpUonIAuhQsxfjJWFiXhZ\\\\\\\/9xjs1N1rYsbk0g6Zsx7lFC0O\\\\\\\/9nCfbRcel5GuejIN41M9VJTWZZIUpygxAw9aNfKUiA8dQqe2d\\\\\\\/GMMR6KBTuTTlDJ2c3k2K0Ok4ez40++U8MZqW7ZJV+DisbyShxYRswkdwF2MWxo3q5Zkce5KVRzTzpksJY8smKgbMnA6DBBESonkK\\\\\\\/LDvFFN8LSSRm0ccym13fhvhGApsdlo\\\\\\\/R7f4+jitQyaxGtN5aXw47XN3qjF3MyTvdGb9iYyR4z+pxg0110s1k7S8pbp3txySkGeFXhMxThZZluKpDXKQ8aAgPNRbNO\\\\\\\/fEYNgg4wZgFxyDkWBwG7AfoLkx84QErxM\\\\\\\/mX8X1riwoEd63gBmERIHNDcs7XXod1tETx7j\\\\\\\/\\\\\\\/XN7CXuWVl9lEW0wEIUQj08bGEoi0PaEVuo3b2BGy2qJjMvVv5tTkOVrio7Vauj\\\\\\\/HSkoU8PsCDaPPU3aFhhgzM\\\\\\\/hXNGJs5dZtCdJn6HWXW7f3HDa0mUYpXTYwB6HPN3GddtVoyd3SAOE2aDbPhyHelxTnIj31fbKQ2HohBwNRf4QN6ZMSWqaM5oAycfLZYZng6nUTv4tvk1pNptpM5dCmTdyOZTz4gCdx+jD0PL2BzRDhFUYQgQdyWezSrooPFc4XBQhxLqCS\\\\\\\/uNYO6AvTJAsDBUze5LJE7WTUvamjZIZWSQlyktQi7zd0syyePYmLhT\\\\\\\/Rvx10W0xkZ8MqdSdmMKknesujBDL24tahsRT+RrujHH+YOY9TiFj3J9Zck8mipDp57QyIuNTwdbLaBrqvY2pG80a+D+g+xIXfN23+18rlXe56PF2SizQo4U0lw5aU23jymepWISJotuSpDxsmCNqbbd1qhqvMkqWrq+7XXQvumIN3\\\\\\\/jIvDt2xwJ1Uq8fjjwYdeb8fIZibtul4DId8CPZnuiB\\\\\\\/WeYyDjnKUCrgbqf6IEthTJCQRhG+snO4HUlB5bFAm3C5OMMsLMpefD9mnIUanJ+gD4ku\\\\\\\/\\\\\\\/52UeVAkXc6Neyez4HxmUlbxZ0rLhY\\\\\\\/LZuy8m1DgFAv+fCvmWNkeJ\\\\\\\/VP4PgZ7C8tkoXFOn9+Zyb73MqWCIIXnSZHWzmq5dt\\\\\\\/rbuWR9gEMDmbCALwQsHCovX\\\\\\\/7cTbBISOdniNx5CLfJ\\\\\\\/h3hn1PWz3Jb5KHy5QDx9ZYRNh2UtVLgfZobHYWmRiRJn3FIKeMC3TzHLaKlelNFxF4eVjNkm+3hDlkVQ1dje2kNsewGH+7gqF1WHVfip9S7F5lJB9Yc2FufXISu93s\\\\\\\/YYIDGALqJTu+xuvJVU8VtQ7foduh43zDJzmdINs74CXJNln29VPMgFD9+OX6O36KT1TJNYrlX5v2FMXAXUa1QlFDp8BimDsGfA8cLPByDr+aDI+dfDi6XgvSsmwo9a8f5dmEKltn3l1FuRocwktbYh5lSZvGFTEj9yxx88\\\\\\\/ugpAo7\\\\\\\/ulFOfUPKGaQQUEMOrdD33RFHK7c9imbePPUqJjvMD6jdERF8LSdYISdd4au\\\\\\\/9WQRBNGvLeCJd7wnDdyIDnPfFpEEqoyi\\\\\\\/3ZjJkj2tdOgAetQTeV375\\\\\\\/MZ5Kk5B1KGl5EPM9m71B7yMVY6LT0q6\\\\\\\/tLQTRiI2xsjPYvaW3X4jWLn7g7EElE3tK6GtFyli7t7DEd92PL5wbO9iXJrWtPaXvu3ZqtA7HcykRTvshuJqLEYVwP0QUQGh0s+OsExxjKkiUVwbgb\\\\\\\/d0TyagaW8uxTrNfZCL66nIRXepYigm1uTHswf7HpIjs1qUUOemk4pEA1FkiMVQf376k3pmIJbX5GYnS0hR6Rk+1FH9Qnf\\\\\\\/6r5dm0T4nntDeisSNhW4bmWZFHdKJQIvLJQGgWQzs3auhuzgqUICuEPl5ZehjLxGoEJVZBrwWv61sipPQXLkmGhPz73Xpk3flc1lKQXxZIhJbvScyFuCvAzmOqewBY1C5B1DDvnOr80aDE4MxU70GKfS5m+qPQznJVUHasSWeeLdyk3PC6EndaBEmjkKQzDHFnovyg+cMoKWonDQvXiswyw7Qvy0WUlg13mfBH3CewQWvX\\\\\\\/KHeBHNFOc70AkSAobk2GmUPIzcz78CyjUtYt22I1J6l4s6mp1LsZrmKxvIit7qN19aBbo2uQWGylvmJ2EZ0PDBlorINLkRLvzFWuB+OxOJqj3SF2I5DpK\\\\\\\/EyoQV5TchZ4MM11DfWYN7rAuVuVnSs7a8wV1+UzMAUwh3YGJpW+Jro5ogz9c04R1roHOPQE5zD50Z2K99pz0TtCRbAtz7fp5g10Fnhy9QKKzj+LTU+zW3rW99TRUqE6Yn15QtLJD8WGZXQjy5KX8HhBlaiBOxpDCOJ8AGAJjWr5qmttZqK5fJUVYMYhL0rKVlc+jQhXmJyRGkoqmn4ekDN88yCPHsit5gzNtnSj18oP+drcjIzm8tn9XwWWv0R\\\\\\\/D31Hb1ij5LWmPVGJlaZVzz\\\\\\\/YLprR9do4mVUgQTGBv9kWlEbCMut2zUHqVRT2yFNhwD+CTxfa03GwZjw+zWSPeAG9LdtmOE0XcfPnq621eMc2IF7NdmjnO6pmc8VQkgyvKh2HHQsahmjUyZh4faQHWTVsnLzYIzgl\\\\\\\/mkQvJnCs7qs+3FRGFY8uJyLGmu81tYUbDeFXyBV5PVAQ\\\\\\\/8YZUpaQgT\\\\\\\/GM05vLrUsbYWBxlVyYNpxvlGYEV5SoaCpzFOOgDzbGDOWOug1GXq\\\\\\\/uSUGOVzsYA\\\\\\\/0ohtrmPtAZS34RFBHvuabwDlTFI7Ah9SKqxUiZd9rItmbeWv16PBmRqXM5z7OfVTChXDzvb4QXkzrlaIXknOx8gPUAi9J5hMfIOk5yo04hBjZtxm1YRPK0mxB9\\\\\\\/X96zqnQK0x05kE6YYBlP9ndyA7ghdtOxKhvenSNoRD6qxojeXDAw6mI75sT\\\\\\\/2uOn4VbhNvwUmiSCqSwoaqSfJPtWDlZ26oBNhIFUKLvhujUIUfVE8cFGZDlO4dv0jXtrkCz31QSgTRbJxkZNR9vOBzl955b6wuIzpJnI1oKuABBMaUE9IOXAEGOomAihGLIT92RiUvzW3gXWR2flMY2OivojU5IXduRwCGTb35P\\\\\\\/k1xCPxLE65smFNxNi71u\\\\\\\/XiU4udJ8LP26jX9m1VoHdAYGPD5H5JpB4u4yimSscSO9iNQSk4znlBSDN2VlC+WU3uxiCE7YHEQ2EjrGt3gJd\\\\\\\/qEu4S++wzH13LbAiF018+xdbwLNwKY7zvTKIPxfDA2skLXA43bEJZYm3RxkX2wNHawJ5E6eWZHDO\\\\\\\/lPNNhqCZiBWXSRhFTdafE0+yDwSGjfwVEzZdY453YIlWmS8WOe2\\\\\\\/kTO50HouJVHUtCGbGlt7uDzRJk2wh7uGSpupHVgyF0zBOppnmfQ2rLlnkjLPCBnk3xgMbDWmYOSJJ7LU6ShgdA5SmDoTEDgh50t44W5rG2OAAdw4d3SJvBQNipojd5xDXmMzXpVCuYqhx2u2pEoES8qjO9yFFHj6MNU\\\\\\\/APd9rdjMc2457ZvM35Ji4hcOFQwIT8Hqsc6lE7ZauKxJr5ODGN2ckljsi3T27KjLEiRiw7xjwdZB+hYHvbY+A2AIpWwS6\\\\\\\/qYYGTCCs+ZQbI8+y3+fp9X3gLZ0rJBR2OALNRwvHK2nKXlX6RW5yIyMlGF4e11Th1zwKGIwDCwFQVGuqzdTMb7jf7l93zQnx8XOaMFP66LhQYvJOsy62aqn3C0Kvk0TGWtaV\\\\\\\/GHmZR4Aa96iBozyDPcz8h3eDqW7YJuyjgDjM0DWVPSIHldIJH75jcci8lmy2eiDDMca7ek5LV9lJcAOQa4fndhbRM6dQ8sv+dGLBF5ZPdEMwuPR6M\\\\\\\/Ed1bpe\\\\\\\/V12h+FoMC5bcEcJLZ00QE+mjzXLa5+JmLLXb+GroCMv5RMHlu\\\\\\\/61urF+q3uzzARyE1XMo1DaQN3\\\\\\\/69q1m8i8f+aklm\\\\\\\/A8RXLREme7PkOv6taMZRk89mGHKbuSMzditXow1f5FiSa+z74kkDbK5QtlovtutjO61VrSYMGqr5PcC4VLj3IK1xhjwh9SWc2d0l0LOsIyvblMd+AYZqoBjG+BV+g7eSUK+gQWM4Q2W8MdO4VRLUkzWy57XESvW0n9MJNLsCF\\\\\\\/by6nB5eWdZAPIpGY79aSpClJuUTzNEbyGjxX8DOt0gpNf\\\\\\\/8qz0UrVL95KK8I8swuMqenGuvaNVoPS75uOviXKGWt5R5gOA7dd0R9mrfcRT8\\\\\\\/Z7kIHTc30ooTS7tNbdDtT37+RcTvgps5XRQ8WnyB2aDKGRIuN8WJGru\\\\\\\/h+8X1uGb+dOr4MLSfPhnf3L9LrBRCeNl+KljAu3bwjU+cny+LA5Og2mUgZTrdH3Ds3DjYHhR5CAp\\\\\\\/KNWSPWsEnz+Z18ALZl+6qU7q4L8zF6rG90+HuJ5t3ZEETkmpL0x37DiOBCPBxd8c9iXp4qFSS+CHuxasPAsLlckdfIDZyLgVE8qjwxHWYkrBvJaxKN4uqQHlwxa5UKEXEHLG7gsm3Rdr6dnJ\\\\\\\/OYEGZ4h04iesC4Wx4UL3GVyxqlr82\\\\\\\/GMJJ85XY2zJWzoAsuxBTcTfG2cdAzudlI4F34uCEkZ+kLZgMo9NwDPaaJWGYpBgSElLGkN9FTC1TviosNwnyYo4ghcZrFGPF\\\\\\\/Fw+Sp8g6BDSRKmX\\\\\\\/80ydXdiHTXOtilecKcJJVrUDis3do5lBLpR3hDu4TG2E4AtMJqf7I3OUpF+u+4uIVlK6DfmerifPQdHKpQVmPgZEQVre5IFYlh7ZKGbym+9Dt0acyHTbYe7xOShvZNbtC\\\\\\\/a2keBeSjoTG9OUzak2YohZEDJgDe6sNA78d0RDG1P4k7NNA1NHKqjGZsSbT2JCFikz9jk7s1xKznvO8DmqZX4PWHRB8Besig1+9uVHd2xWvLJCic5dSVV8pKTrK9tdsog8KzRbWXj05CU1JA0YepBA4\\\\\\\/7qgaXjzokKpFJS1Xhln5WDmjuvN9l8SUKwdMQ+\\\\\\\/57RonJyk2G\\\\\\\/ERpt0vfY9LTWyk5V7SlIPpDEECWvTImP+YjqVulv8KrJLux\\\\\\\/dsdCORn4KJzCEoZCL8j4At9I8tkBexgtr47rT\\\\\\\/VC85H0LTesFsFbos2orzPsNzyNMRUWzs0yOBJBsjMwGFDvlYUHGiPwUgfQ393VjcEh\\\\\\\/HrueQh3piurQaJE1ATqTCw4\\\\\\\/R25q9bFjj+0\\\\\\\/F70HJIDv3dNFAS9sMlBMQ3im362CXNHYJ6K+ceYa\\\\\\\/CttMqI1\\\\\\\/i233\\\\\\\/2T73rKaBpne24aU2ZATU84dP6ZC8L7+YZg6tVECGUeAzfZe0iLpvDagVUeRD+2uihltO873qwiyOkfk92koc4xtAHOYinVswn1SiYBT0waoyy2Y8Jtu6VKplcXUfGPSfSOonByg4p+Av3jhMqN\\\\\\\/ilJfyVs1HOeWXYHlvGMDReESkFVz+dKIcOuP9WnoSeBqtFOtOUXyHFB6cysdpBlH4DcuPsbH+35aXWLDTCQVsRvrl8WxXq\\\\\\\/HpoLB\\\\\\\/l8+JLn063\\\\\\\/BffIOD8TGO6IkKZl1DNU9RvdPM+giNvgZVWPnVk7UWGQQQYLTuXU6e9ycmVies0d60D+TQGDdfTvGW6r1O1q54jmvicYiiHY2fhLoiadikjLjk81e\\\\\\\/DS2qpVWwIL3qezW4CaeSe+NMHH8g2A9kGO93TKY5LRaA27qhHH4hdffUJ2IAmB1nrGM7EsZSUiNw6VkUa5WkX8MZv\\\\\\\/8sD712oN5idbyfMn99sxijP8Yw0N7MwAdtAvDf1mpyJVheYQed2fTmV1F5d8uYgkgdQZKaGH\\\\\\\/SNeseusyDntt4qvfAsog+t7FuQ==\\\u0022,\\\u0022iv\\\u0022:\\\u00227jorBI4c89w9Rdsy8f2bBg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022cf15a0d5abb9d9171b5748f0a15358512e3858b69030ba5a76d6f282017bd404\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022vnDO0DJGfsSwlu1C9D9e8sHssUBhqiP5bBfrB\\\\\\\/3MFGcpYEbK5sLBFt7KqZx4wSOtzVVaNukgpBojo51OCsWcPpw\\\\\\\/fAWNQ7kYsZMsbDUYYOpspqJIb2qSshEmZNaHoj1vTkB0\\\\\\\/973TPhrxrKuNpQBie0le2nJAUZ4VTDUDUI0P+HAuQZyWbxuDVkH2wFFLKY9bE17xUumotYShPH0EuOgSLmp2Qd0BztkEw+aasGgqpu0FHxI36E\\\\\\\/6Ne2bLb+QLPZlyH3oEutEuKC6olQ7F7Nsm5pQMWGUVrk367ZDKDFkYn0xqOcmVGCNoPEVIhHBgn3JmqCKCHSLOwGsuARfp5CU\\\\\\\/H8eh9JAP+n2ofLM+5DisRSNEmDgHRr+i8ITX8rvBjD0lz30yGExBXIG+\\\\\\\/tVpjxXhu3cEKTXTbgdBcUflWQGUWj8KQbxjAdjqE4qZPrTawcSJErtmMU2N9NnCKOBhXvI4BPgV365U670mtDTKQBB2c2MgVq3ltlA0\\\\\\\/2jKBcWJz7Pt1xlydpt4RIWoclalXbYKlC2sMXJRLIYPiMB5+4r2rj\\\\\\\/b1j0O8B+Kd3lJrfOH5BFR98kKud\\\\\\\/Oquwz6ETlVw4NhW1+gVGroD1edYlLpyu5EVGA8wwrGOt9CCPAUPom8iajhHGdFqouCyJ7WtP2nXANBBSXcQU0qQikgaSMYAEBOX3+rwHnoSrlGdzom+wNU+BGW8Ey2hDFc0TkLDLgDsAG0bvMBbqLA6ogBW6Po1Lx3yY6UnBC1I6ucNeP7mG+2ODi5M70R1M4s952jAyXV+lYR+sQ4LN5CrHnOC1dUTP21pUBF6IbE6p0dyy8I+WOXap5gHJXxW\\\\\\\/INpjsZlyrwrMCFUZdVoSzxPZf7DXJj5uCWp30VKXiNdUuYNP7o2XKEiTYl9yuOUpqZ5RhRVu0GlN2g0ejDuah7ZlCGf2gOOzDQtyEsL5Rje0na5kfyf38\\\\\\\/EkOhOqfPbz15Qb\\\\\\\/32RUoL+214OIoNZMOlz83g3IusWpU0C+L54e6MmDksvyi8IPFGS3tAOPWKAXaJxUPubG8VFGCtX8Txn\\\\\\\/4y7WNou1F\\\\\\\/C+EAXQ\\\\\\\/yMhYF0nCavo8GpOjzMT\\\\\\\/MvJ\\\\\\\/xjx0WihQAfizTF4Hp+tDXbSR4UQbhoEFIcHfjUuHSEtfSpcP0SY65Xqj+0ushLHdI2BfKDZN0rv96awOXncUpair4kuyZiPB7z9wG7KOMSmoHABJi5oxOwvzJzqtKUHUj\\\\\\\/LUazBBRiWpTd7FWk4lg2EOW9xhY4VJhOdTeJOovMJMCZLS5kpLyrbYYkVjlf6IQCWwxZVogz6znz3sRUjHH8STuOHBqH++mpTBgmzg9lWh8DFEVGtyhAB0zWq1ubOzB+ZXrGj4t4Qr5CODwokaPECUICGgjeFPhsDXuVj+r2+kYQDd6W70Og6OgSiAsDNyVPspRjGOeB3477+d9RgQQklcml3PavNHpMXR9xUzm9BM835+q58irN3ZWshteXBIBdHoqzDnisiHI5V5VzI9d1wD0FSluEdTSX8fE5tgT0YZ5Qx6Zz3l5zVenwOlcib0dBviDkJk5kpohTO79avPbuaMLzLxLDoXs+B00D7Y5smLmA8+Iue8DHfUc9H3J7EZ43S5tfbldbdjh\\\\\\\/E2S2+GBllo7n0u8ST5kRisXAGQoNaLOMw202CiN4FHDPwg8KYKAEJsp4vseuPQsrJrUJT4U+TpLFRutklAK+GGoy6tSxtQ\\\\\\\/8EEZ6lNMTodW0PBOsiakxk\\\\\\\/wlQX1enSx64oLLJLZYkjpv9sVeqSjjPw3U7\\\\\\\/QgI2piQaBYlTqCssJ1aDaXtQs52GUKBEY9O3yPWmJLn0zUIyUQ9j8YuMRfE6imvcOeAfKUmTbPvfFKn9BH8JeU5BYLX+klYuhiyQgDT\\\\\\\/6F7zVGnlxMUl0AnUlWW\\\\\\\/coXy9T+Rj4LzmO4wimTkLNO1Tc08p8BiRUdX93TyrOuVbf6Y+BOm5vPiqthOEdAa4IlBwb8QHf66IdAgtjdeoQm+Ff4G5uu69QdxPzO7dXg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022R3kKUP5WCiAtxZBpMm3djQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00229d4a83f72b3202d6371f3e44d2f4b8fac8cb9e3083da3d6722d8d86b43585fae\u0022}"},"ajax":{"edit-submit--5":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-491247 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/11/3703" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-sDiCvjMZw36xH6PtSxGWrI-Jk3dx6_Rwo3BpFWo9G8k" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/11/3703" method="post" id="highwire-search-quicksearch-form-1" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--3">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--3" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--7" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-sm9DxKl8H-bToIYUREgL6Vv1SBISXyIsnu9kxkY7ZTc" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_1" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-3" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/11/3691" title="Evidence of Zip1 Promoting Sister Kinetochore Mono-orientation During Meiosis in Budding Yeast " rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/11/3715" title="Comparing Genome-Wide Association Study Results from Different Measurements of an Underlying Phenotype" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="491247" id="node491247" data-pisa="ggg;8/11/3703" data-pisa-master="ggg;g3.118.200446" data-apath="/ggg/8/11/3703.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Case Study of Genomic Instability in an Industrial Strain of <em>Saccharomyces cerevisiae</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Aline Rodrigues-Prause</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-6403-0159" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Nadia M. V. Sampaio</span>, <span class="highwire-citation-author" data-delta="2">Theodore M. Gurol</span>, <span class="highwire-citation-author" data-delta="3">Guadalupe M. Aguirre</span>, <span class="highwire-citation-author" data-delta="4">Hailey N. C. Sedam</span>, <span class="highwire-citation-author" data-delta="5">Mary J. Chapman</span>, <span class="highwire-citation-author" data-delta="6">Ewa P. Malc</span>, <span class="highwire-citation-author" data-delta="7">V. P. Ajith</span>, <span class="highwire-citation-author" data-delta="8">Parijat Chakraborty</span>, <span class="highwire-citation-author" data-delta="9">Pedro A. Tizei</span>, <span class="highwire-citation-author" data-delta="10">Gonçalo A. G. Pereira</span>, <span class="highwire-citation-author" data-delta="11">Piotr A. Mieczkowski</span>, <span class="highwire-citation-author" data-delta="12">Koodali T. Nishant</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="13"><a href="http://orcid.org/0000-0002-7157-1519" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Juan Lucas Argueso</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3703-3713;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200446 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node491247" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Aline Rodrigues-Prause </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Environmental and Radiological Health Sciences</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Departamento de Genética, Evolução e Bioagentes, Instituto de Biologia, Universidade Estadual de Campinas, Campinas-SP, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aline%2BRodrigues-Prause%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rodrigues-Prause%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAline%2BRodrigues-Prause%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Nadia M. V. Sampaio </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Environmental and Radiological Health Sciences</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Cell and Molecular Biology Graduate Program, Colorado State University, Fort Collins-CO 80523</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nadia%2BM.%2BV.%2BSampaio%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sampaio%20NM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANadia%2BM.%2BV.%2BSampaio%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-6403-0159" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Nadia M. V. Sampaio</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Theodore M. Gurol </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Environmental and Radiological Health Sciences</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Cell and Molecular Biology Graduate Program, Colorado State University, Fort Collins-CO 80523</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Theodore%2BM.%2BGurol%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gurol%20TM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATheodore%2BM.%2BGurol%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Guadalupe M. Aguirre </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Environmental and Radiological Health Sciences</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Cell and Molecular Biology Graduate Program, Colorado State University, Fort Collins-CO 80523</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Guadalupe%2BM.%2BAguirre%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Aguirre%20GM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGuadalupe%2BM.%2BAguirre%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Hailey N. C. Sedam </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Environmental and Radiological Health Sciences</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Cell and Molecular Biology Graduate Program, Colorado State University, Fort Collins-CO 80523</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hailey%2BN.%2BC.%2BSedam%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sedam%20HN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHailey%2BN.%2BC.%2BSedam%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Mary J. Chapman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Environmental and Radiological Health Sciences</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mary%2BJ.%2BChapman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chapman%20MJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMary%2BJ.%2BChapman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Ewa P. Malc </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Genetics, University of North Carolina, Chapel Hill-NC 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ewa%2BP.%2BMalc%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Malc%20EP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEwa%2BP.%2BMalc%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">V. P. Ajith </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>School of Biology, Indian Institute of Science Education and Research</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=V.%2BP.%2BAjith%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ajith%20VP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AV.%2BP.%2BAjith%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Parijat Chakraborty </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>School of Biology, Indian Institute of Science Education and Research</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Parijat%2BChakraborty%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chakraborty%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AParijat%2BChakraborty%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Pedro A. Tizei </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Departamento de Genética, Evolução e Bioagentes, Instituto de Biologia, Universidade Estadual de Campinas, Campinas-SP, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pedro%2BA.%2BTizei%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tizei%20PA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APedro%2BA.%2BTizei%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Gonçalo A. G. Pereira </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Departamento de Genética, Evolução e Bioagentes, Instituto de Biologia, Universidade Estadual de Campinas, Campinas-SP, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gon%C3%A7alo%2BA.%2BG.%2BPereira%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pereira%20GA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGon%25C3%25A7alo%2BA.%2BG.%2BPereira%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Piotr A. Mieczkowski </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Genetics, University of North Carolina, Chapel Hill-NC 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Piotr%2BA.%2BMieczkowski%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mieczkowski%20PA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APiotr%2BA.%2BMieczkowski%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Koodali T. Nishant </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>School of Biology, Indian Institute of Science Education and Research</div><div class='author-affiliation'><span class='nlm-sup'>††</span>Center for Computation Modeling and Simulation, Indian Institute of Science Education and Research, Thiruvananthapuram, Trivandrum 695016, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Koodali%2BT.%2BNishant%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nishant%20KT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKoodali%2BT.%2BNishant%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-13"><div class="author-tooltip-name">Juan Lucas Argueso </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Environmental and Radiological Health Sciences</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Cell and Molecular Biology Graduate Program, Colorado State University, Fort Collins-CO 80523</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Juan%2BLucas%2BArgueso%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Argueso%20JL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJuan%2BLucas%2BArgueso%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-7157-1519" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Juan Lucas Argueso</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:lucas.argueso@colostate.edu" class="" data-icon-position="" data-hide-link-title="0">lucas.argueso@colostate.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/8/11/3703" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:491247" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:491247/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/11/3703.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:491247" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:491247/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/11/3703.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:491247" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:491247/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/11/3703.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:491247" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:491247/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/8/11/3703.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:491247" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:491247/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">The <em>Saccharomyces cerevisiae</em> strain JAY270/PE2 is a highly efficient biocatalyst used in the production of bioethanol from sugarcane feedstock. This strain is heterothallic and diploid, and its genome is characterized by abundant structural and nucleotide polymorphisms between homologous chromosomes. One of the reasons it is favored by many distilleries is that its cells do not normally aggregate, a trait that facilitates cell recycling during batch-fed fermentations. However, long-term propagation makes the yeast population vulnerable to the effects of genomic instability, which may trigger the appearance of undesirable phenotypes such as cellular aggregation. In pure cultures of JAY270, we identified the recurrent appearance of mutants displaying a mother-daughter cell separation defect resulting in rough colonies in agar media and fast sedimentation in liquid culture. We investigated the genetic basis of the colony morphology phenotype and found that JAY270 is heterozygous for a frameshift mutation in the <em><a href="http://www.yeastgenome.org/locus/S000004121/overview">ACE2</a></em> gene (<em><a href="http://www.yeastgenome.org/locus/S000004121/overview">ACE2</a>/<a href="http://www.yeastgenome.org/locus/S000004121/overview">ace2</a>-A7</em>), which encodes a transcriptional regulator of mother-daughter cell separation. All spontaneous rough colony JAY270-derived isolates analyzed carried copy-neutral loss-of-heterozygosity (LOH) at the region of chromosome XII where <em><a href="http://www.yeastgenome.org/locus/S000004121/overview">ACE2</a></em> is located (<em><a href="http://www.yeastgenome.org/locus/S000004121/overview">ace2</a>-A7/<a href="http://www.yeastgenome.org/locus/S000004121/overview">ace2</a>-A7</em>). We specifically measured LOH rates at the <em><a href="http://www.yeastgenome.org/locus/S000004121/overview">ACE2</a></em> locus, and at three additional chromosomal regions in JAY270 and in a conventional homozygous diploid laboratory strain. This direct comparison showed that LOH rates at all sites were quite similar between the two strain backgrounds. In this case study of genomic instability in an industrial strain, we showed that the JAY270 genome is dynamic and that structural changes to its chromosomes can lead to new phenotypes. However, our analysis also indicated that the inherent level of genomic instability in this industrial strain is normal relative to a laboratory strain. Our work provides an important frame of reference to contextualize the interpretation of instability processes observed in the complex genomes of industrial yeast strains.</p></div><ul class="kwd-group"><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/saccharomyces-cerevisiae" class="hw-term hw-article-keyword hw-article-keyword-saccharomyces-cerevisiae" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Saccharomyces cerevisiae</em></a></li><li class="kwd"><a href="/keyword/loss-heterozygosity-0" class="hw-term hw-article-keyword hw-article-keyword-loss-of-heterozygosity" rel="nofollow">Loss-of-heterozygosity</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/ace2" class="hw-term hw-article-keyword hw-article-keyword-ace2" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">ACE2</em></a></li><li class="kwd"><a href="/keyword/fermentation" class="hw-term hw-article-keyword hw-article-keyword-fermentation" rel="nofollow">Fermentation</a></li><li class="kwd"><a href="/keyword/colony-morphology" class="hw-term hw-article-keyword hw-article-keyword-colony-morphology" rel="nofollow">Colony morphology</a></li><li class="kwd"><a href="/keyword/mitotic-recombination" class="hw-term hw-article-keyword hw-article-keyword-mitotic-recombination" rel="nofollow">Mitotic recombination</a></li></ul><p id="p-4">Over the last decade, the genomic characterization of many diverged diploid isolates of the yeast <em>Saccharomyces cerevisiae</em> has revealed the existence of substantial intra-strain variability in nucleotide sequence and chromosome structure (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Argueso <em>et al.</em> 2009</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Borneman <em>et al.</em> 2011</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Magwene <em>et al.</em> 2011</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Peter <em>et al.</em> 2018</a>). While most conventional laboratory strains, when diploidized, harbor nearly completely homozygous genomes and display stable phenotypes, the genome architecture of wild strains, including clinical and industrial isolates, has often been found to be more complex, with abundant inter-homolog nucleotide heterozygosity and structural variation.</p><p id="p-5">In several species, widespread heterozygosity is often associated with beneficial phenotypes. It is suspected that this genomic configuration may also contribute to the adaptation of wild yeasts to their respective habitats, as well as to the robust growth, stress tolerance, and other desirable traits characteristic of industrial yeasts (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Muller and McCusker 2009</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Plech <em>et al.</em> 2014</a>). Interestingly, the maintenance of heterozygosity by these single-celled organisms is challenging because allelic mitotic recombination events between homologous chromosomes occurs at relatively high rates (∼10<sup>−4</sup>/cell division; (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">St Charles and Petes 2013</a>)), and can lead to immediate fixation of tracts of loss-of-heterozygosity (LOH) in the clonal cell lineage. Accordingly, in JAY270/PE2 and other wild strains, heterozygous regions are interspersed with stretches of homozygosity indicating the occurrence of LOH events in clonal ancestors (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Magwene <em>et al.</em> 2011</a>). In addition, multiple studies have characterized the emergence of chromosomal rearrangements in cells clonally derived from industrial strains (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Nadal <em>et al.</em> 1999</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Carro and Pina 2001</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Basso <em>et al.</em> 2008</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Querol and Bond 2009</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">de Amorim <em>et al.</em> 2017</a>). Those detailed studies, combined with informal field observations occasionally reported by operators of wine, beer and biofuel fermentations, have earned industrial yeast strains a reputation for having unstable genomes. This genomic instability could, and likely does, make fermentations that use such strains vulnerable to process disruption due to the loss of selected traits or appearance of undesirable phenotypes.</p><p id="p-6">One of the first heterozygous <em>S. cerevisiae</em> strains to have its genome characterized was JAY270/PE2, widely used in sugarcane bioethanol production (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Argueso <em>et al.</em> 2009</a>). This heterothallic diploid was originally isolated at a Brazilian distillery as an aggressive, yet highly productive wild contaminant (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Basso <em>et al.</em> 2008</a>). One of its most attractive features is its suitability to continuous cell recycling in batch-fed fermentations. JAY270/PE2 offers distillery operators predictable fermentation conditions during the 8-9 month sugarcane harvest season, because it is often able to outcompete wild yeast contaminants that are steadily introduced through the feedstock into the recycling system.</p><p id="p-7"> Here, we report a case study of two easily discernible and related mutant phenotypes that result from a specific rearrangement of the JAY270/PE2 genome. We characterized the spontaneous appearance of clones displaying a mother-daughter cell separation defect among pure cultures of this strain. This cell separation defect resulted in both rough colony morphology in agar plates and fast sedimentation in liquid culture. We found that these phenotypes were recurrently caused by an LOH event spanning the <em><a href="http://www.yeastgenome.org/locus/S000004121/overview">ACE2</a></em> locus. We measured the quantitative rate and characterized the qualitative spectrum of mitotic recombination at the region of chromosome XII (Chr12) where <em><a href="http://www.yeastgenome.org/locus/S000004121/overview">ACE2</a></em> is located. In order to provide reference for a broader comparison of genomic instability levels, we also measured the rates of LOH at three additional independent chromosomes, using both JAY270/PE2 and a conventional diploid homozygous laboratory strain. We found that the rates of mitotic recombination were quite similar overall between the four genomic regions and between the two strain backgrounds. These results showed that, while chromosomal rearrangements did occur at detectable rates and could lead to new phenotypes in JAY270/PE2, its inherent level of genomic instability was not elevated relative to the reference strain. Instead, we found that the genome of the conventional laboratory strain was just as dynamic as to that of JAY270/PE2.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Yeast genetic backgrounds and culture procedures</h3><p id="p-8"><em>Saccharomyces cerevisiae</em> strains used in this study descended from either the JAY270 or CG379 strain backgrounds (Table S1). JAY270 is a heterothallic diploid single colony isolate derived from the industrial bioethanol strain PE2 (<a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">Argueso <em>et al.</em> 2009</a>). CG379 diploid is related to the S288c laboratory strain background (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Morrison <em>et al.</em> 1991</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Argueso <em>et al.</em> 2008</a>). </p><p id="p-9">Yeast cells were grown in YPD (20 g/L glucose, 20 g/L peptone, 10 g/L yeast extract, 20 g/L bacteriological agar for solid media) and synthetic media (20 g/L glucose, 5 g/L ammonium sulfate, 1.7 g/L yeast nitrogen base without amino acids, 1.4 g/L complete drop-out mix, 20 g/L bacteriological agar). Standard procedures for yeast transformation, crossing and sporulation were followed (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Ausubel <em>et al.</em> 1998</a>). Counter-selections against <em><a href="http://www.yeastgenome.org/locus/S000000747/overview">URA3</a></em> were performed in synthetic media supplemented with 1g/L of 5-Fluoroorotic Acid (5-FOA).</p></div><div id="sec-3" class="subsection"><h3>Allele replacement and complementation tests</h3><p id="p-10">A two-step allele replacement plasmid was constructed by cloning a segment of the wild type <em><a href="http://www.yeastgenome.org/locus/S000004121/overview">ACE2</a></em> allele containing the 8-adenine homopolymer into the <em><a href="http://www.yeastgenome.org/locus/S000000747/overview">URA3</a></em> pRS306 integrative vector (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Sikorski and Hieter 1989</a>). First, pRS306 was digested with <em>Bam</em>HI and <em>Kpn</em>I and ligated to a <em>Bgl</em>II- and <em>Kpn</em>I-digested segment of the wild type <em><a href="http://www.yeastgenome.org/locus/S000004121/overview">ACE2</a></em> allele (PCR amplified from JAY289 genomic DNA using primers JAO904 and JAO905). The resulting plasmid, pAR1, was linearized with <em>Eco</em>RI and transformed into 5-FOA resistant (<a href="http://www.yeastgenome.org/locus/S000000747/overview"><em>ura3</em></a> mutant) derivatives of JAY291 and JAY292 haploids (both mutant <em><a href="http://www.yeastgenome.org/locus/S000004121/overview">ace2</a>-A7</em>). Pop-in Ura+ integration transformants were selected, and pop-out events were then selected for resistance to 5-FOA. 5-FOA resistant candidates were screened by PCR and Sanger sequencing to verify the presence of the wild type <em><a href="http://www.yeastgenome.org/locus/S000004121/overview">ACE2</a></em> allele. This procedure resulted in the <em><a href="http://www.yeastgenome.org/locus/S000004121/overview">ACE2</a></em> strains JAY1051 and JAY1039, isogenic to JAY291 and JAY292 respectively. Complementation tests using these strains were performed as shown in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2C</a>.</p></div><div id="sec-4" class="subsection"><h3>Construction of strains used in LOH assays</h3><p id="p-11">The JAY270 strains used in the LOH assays were constructed from a homozygous <em><a href="http://www.yeastgenome.org/locus/S000000747/overview">ura3</a>Δ/<a href="http://www.yeastgenome.org/locus/S000000747/overview">ura3</a>Δ</em> derivative of JAY270 (JAY585; gift from F. Galzerani). The CORE2 cassette containing the <em>Kluveromyces lactis URA3</em> gene, the <em>S. cerevisiae <a href="http://www.yeastgenome.org/locus/S000000747/overview">URA3</a></em> gene and the <em>Kan</em>MX4 geneticin resistance marker (<em><sub>Kl</sub>URA3-<sub>Sc</sub><a href="http://www.yeastgenome.org/locus/S000000747/overview">URA3</a>-KanMX4</em>) was amplified from pJA40 (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Zhang <em>et al.</em> 2013</a>) with primers targeting integration at four different genomic regions (Table S2). Integration in the maternal or paternal Chr12 homolog was checked by PCR and Sanger genotyping of linked HetSNPs. The same procedures and primers were used integrate CORE2 in the CG379 background. </p></div><div id="sec-5" class="subsection"><h3>Quantitative LOH rate assays</h3><p id="p-12">Diploid yeast cells hemizygous for CORE2 insertions were streaked to single colonies on solid YPD medium and incubated at 30° for two days. Single colonies were inoculated into 5 ml liquid YPD, and incubated for 24 hr at 30° in a rotating drum. The cultures were serially diluted and plated on YPD (permissive) or 5-FOA (selective) media. Colonies were counted after 2 days of growth on permissive and 4 days on selective plates, and colony counts were used to calculate recombination rates and 95% confidence intervals using the Lea &amp; Coulson method of the median within the FALCOR web application (<a href="http://www.keshavsingh.org/protocols/FALCOR.html">http://www.keshavsingh.org/protocols/FALCOR.html</a>) (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Lea and Coulson 1949</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Hall <em>et al.</em> 2009</a>). Statistical analyses of pairwise comparisons between recombination rates were performed using a two-sided nonparametric Mann Whitney test in GraphPad Prism software.</p></div><div id="sec-6" class="subsection"><h3>Genotyping specific HetSNPs Through PCR-RFLP and molecular karyotyping</h3><p id="p-13">Genotyping was performed by PCR-amplification of regions containing the HetSNPs followed by restriction digestion and agarose gel analysis fragment length polymorphisms. In cases of markers where the HetSNPs were not associated with an RFLP, PCR products were Sanger-sequenced. A complete list of the HetSNPs coordinates, primers and restriction enzymes used is provided in Table S3. </p></div><div id="sec-7" class="subsection"><h3>Genome Sequencing Analyses</h3><p id="p-14">The genomes of the JAY270 parent strain and of 56 haploids derived from 14 complete JAY270 tetrads were sequenced using the Illumina short read whole genome sequencing platform. The sequencing reads were used to create a phased draft map of high confidence HetSNPs in JAY270’s genome (detailed information about the pipeline used are provided in the Supplemental Material). Genome sequencing data associated with this study is available in the Sequence Read Archive (SRA) database under study number SRP082524.</p></div><div id="sec-8" class="subsection"><h3>Quantitative digital droplet PCR</h3><p id="p-15"> Genomic DNA was prepared from PFGE agarose plugs as previously described (<a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Zhang <em>et al.</em> 2013</a>). Sheared DNA was then digested with <em>Mfe</em>I-HF, a restriction enzyme that cuts the genome frequently, but does not cut within the ddPCR amplicons. Digested DNA was diluted to 0.05 ng/μl and 2 μl of this dilution was used as a template for each reaction using BioRad’s QX200 EvaGreen Supermix. The internal genomic DNA template normalizer was the average between signal from <em><a href="https://www.yeastgenome.org/locus/S000003924">DNM1</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000003991/overview">YLR001C</a></em> (regions of known two copy number), divided by 2.</p></div><div id="sec-9" class="subsection"><h3>Data and reagent availability</h3><p id="p-16">Strains and plasmids are available upon request. Supplemental files available at FigShare. Sequence data are available at the Sequence Read Archive (SRA) database under study number SRP082524. Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.6406598">https://doi.org/10.25387/g3.6406598</a>.</p></div></div><div class="section results" id="sec-10"><h2 class="">Results</h2><div id="sec-11" class="subsection"><h3>Spontaneous JAY270 derivatives display a recurrent cell separation defect</h3><p id="p-17">Yeast cellular aggregation can be a problem in batch-fed sugarcane bioethanol production because it reduces fermentation kinetics and leads to clogging of the centrifuges and pipes used to recycle the yeast biomass from one tank to the next (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Della-Bianca <em>et al.</em> 2013</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Paulino de Souza <em>et al.</em> 2018</a>). One of the desirable industrial traits of the JAY270/PE2 strain (hereafter referred to simply as JAY270) is that its cells do not usually aggregate (<a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">Basso <em>et al.</em> 2008</a>). Accordingly, JAY270 produces normal hemispherical colonies with smooth surfaces and edges when grown in solid agar medium, and remains in dispersed suspension in liquid culture for prolonged periods (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a> and <a id="xref-fig-1-2" class="xref-fig" href="#F1">1C</a>). While these are the phenotypes typically observed, over the course of our studies using this strain we noticed the sporadic occurrence of colonies clonally derived from JAY270 that displayed altered morphology: relatively flat-growing colonies with rough surfaces and edges (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1B</a>). In addition, when grown in liquid media, these clones sedimented at a faster rate compared to JAY270 (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1D</a>). Under bright field microscopic examination, yeast cells derived from such rough colonies appeared to grow in chains, showing a budding pattern consistent with a defect in the separation of the daughter cells from their mother (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1E-F</a>). We stained these cells with calcofluor white to visualize the chitin-rich ring septa, confirming the persistent attachment of mother and daughter cells at the budding neck site (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1G-J</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/11/3703/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Smooth and rough colony morphologies, liquid sedimentation, mother-daughter cell attachment, and phenotypes of diploids derived from mating specific haploids. A-J show images of the JAY270 smooth parent diploid strain (left panels) and its spontaneous rough derivative JAY663 (right panels). A and B, colony morphologies on YPD agar after 3 days growth at 30C. C and D, cell sedimentation kinetics. 5 ml liquid YPD cultures were grown overnight at 30C in a rotating drum. Test tubes were vortexed vigorously for 10 sec to fully resuspend the cells, then were left to rest and photographed at 15 min intervals (the 0&#x2019; pictures were taken immediately after vortexing). E and F, bright field, and G-J, fluorescence microscopy of cells stained with calcofluor white to highlight chitin septa and the mother-daughter cell attachment. Scale bars are 1mm (A-B), 20&#xB5;m (E-F) and 5&#xB5;m (G-J). K and L, Smooth (S, white circles) and rough (R, black stars) phenotypes of diploids formed by crossing the indicated MATa and MAT&#x3B1; haploids isolated from three tetrads of each JAY270 (K) and JAY663 (L). Thick black lines indicate the four diploids derived from matings of intra-tetrad sibling haploids. The colored backgrounds for each haploid correspond to their inferred genotype (Blue, dominant wild type allele; Red, recessive mutant allele). All 12 haploids from panel K had their whole genomes sequenced. Co-segregation analysis with JAY270 HetSNPs (Fig. S1) was used for identification of the causal mutation at the ACE2 locus (Fig. 2)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-677849688" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Smooth and rough colony morphologies, liquid sedimentation, mother-daughter cell attachment, and phenotypes of diploids derived from mating specific haploids. A-J show images of the JAY270 smooth parent diploid strain (left panels) and its spontaneous rough derivative JAY663 (right panels). A and B, colony morphologies on YPD agar after 3 days growth at 30C. C and D, cell sedimentation kinetics. 5 ml liquid YPD cultures were grown overnight at 30C in a rotating drum. Test tubes were vortexed vigorously for 10 sec to fully resuspend the cells, then were left to rest and photographed at 15 min intervals (the 0&#x2019; pictures were taken immediately after vortexing). E and F, bright field, and G-J, fluorescence microscopy of cells stained with calcofluor white to highlight chitin septa and the mother-daughter cell attachment. Scale bars are 1mm (A-B), 20&#xB5;m (E-F) and 5&#xB5;m (G-J). K and L, Smooth (S, white circles) and rough (R, black stars) phenotypes of diploids formed by crossing the indicated &lt;em&gt;MATa&lt;/em&gt; and &lt;em&gt;MAT&#x3B1;&lt;/em&gt; haploids isolated from three tetrads of each JAY270 (K) and JAY663 (L). Thick black lines indicate the four diploids derived from matings of intra-tetrad sibling haploids. The colored backgrounds for each haploid correspond to their inferred genotype (Blue, dominant wild type allele; Red, recessive mutant allele). All 12 haploids from panel K had their whole genomes sequenced. Co-segregation analysis with JAY270 HetSNPs (Fig. S1) was used for identification of the causal mutation at the &lt;em&gt;ACE2&lt;/em&gt; locus (Fig. 2).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/11/3703/F1.medium.gif" width="434" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/8/11/3703/F1.medium.gif" width="434" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/11/3703/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/11/3703/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491304" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-18" class="first-child">Smooth and rough colony morphologies, liquid sedimentation, mother-daughter cell attachment, and phenotypes of diploids derived from mating specific haploids. A-J show images of the JAY270 smooth parent diploid strain (left panels) and its spontaneous rough derivative JAY663 (right panels). A and B, colony morphologies on YPD agar after 3 days growth at 30C. C and D, cell sedimentation kinetics. 5 ml liquid YPD cultures were grown overnight at 30C in a rotating drum. Test tubes were vortexed vigorously for 10 sec to fully resuspend the cells, then were left to rest and photographed at 15 min intervals (the 0’ pictures were taken immediately after vortexing). E and F, bright field, and G-J, fluorescence microscopy of cells stained with calcofluor white to highlight chitin septa and the mother-daughter cell attachment. Scale bars are 1mm (A-B), 20µm (E-F) and 5µm (G-J). K and L, Smooth (S, white circles) and rough (R, black stars) phenotypes of diploids formed by crossing the indicated <em>MATa</em> and <em>MATα</em> haploids isolated from three tetrads of each JAY270 (K) and JAY663 (L). Thick black lines indicate the four diploids derived from matings of intra-tetrad sibling haploids. The colored backgrounds for each haploid correspond to their inferred genotype (Blue, dominant wild type allele; Red, recessive mutant allele). All 12 haploids from panel K had their whole genomes sequenced. Co-segregation analysis with JAY270 HetSNPs (Fig. S1) was used for identification of the causal mutation at the <em>ACE2</em> locus (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Fig. 2</a>).</p><div class="sb-div caption-clear"></div></div></div><p id="p-19">We initially isolated five independent examples of such rough colonies for genetic characterization (JAY663, JAY664, JAY665, JAY912 and JAY913), all of which were derived either directly from JAY270 or from JAY270-isogenic strains. The rough colony phenotype of these isolates was stably maintained and was not reversible over several clonal generations, suggesting that it was likely encoded genetically and not caused by a transient transcriptional or post-transcriptional state. We estimated that these five isolates appeared spontaneously from a pool of ∼5 × 10<sup>4</sup> smooth colonies. Assuming a genetic origin and based on this high frequency of occurrence in diploid cells, we reasoned that this phenotype was unlikely to be caused by a rare dominant <em>de novo</em> nucleotide point mutation, which might be expected to occur in the 10<sup>−9</sup>-10<sup>−8</sup>/cell division rate range. Instead, a more plausible mutational mechanism was mitotic recombination leading to loss-of-heterozygosity (LOH), because it occurs at a compatible rate (∼10<sup>−4</sup>/cell division; (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">St Charles and Petes 2013</a>)) and would not require a dominant phenotypic effect.</p></div><div id="sec-12" class="subsection"><h3>Investigation of the genetic basis of the rough colony phenotype</h3><p id="p-20">In a parallel project, we observed that crossing two specific haploid descendants of JAY270 (JAY291 <em>MATa</em> and JAY292 <em>MATα</em>; (<a id="xref-ref-1-4" class="xref-bibr" href="#ref-1">Argueso <em>et al.</em> 2009</a>)) resulted in diploid cells that displayed rough colony morphology and fast sedimentation phenotypes indistinguishable from those exhibited by the five isolates described above. Analysis of the JAY291 and JAY292 haploids showed that both displayed a mother-daughter cell separation defect and fast sedimentation, but neither had the rough colony phenotype (data not shown). JAY291, JAY292 and all other haploid derivatives of JAY270 display the normal smooth colony morphology.</p><p id="p-21">The ability to consistently reproduce the mutant colony morphology phenotype in controlled crosses between specific haploids opened an avenue to investigate its genetic basis. We pursued this approach by carrying out crosses between twelve haploid descendants of JAY270 (3 tetrads; <a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1K</a>), all of which had their whole genomes sequenced (see below). All possible <em>MATa</em> x <em>MATα</em> crosses were performed producing 36 different diploids. Among them, we found eight with rough and 28 with smooth colony surfaces, in a pattern that was consistent with recessive inheritance of a trait controlled by a single gene. Even though the rough colony phenotype was not observed in any of the parents, the phenotypes of the respective diploids allowed us to infer which allele was present in the haploids: either the wild type dominant allele or the recessive mutant allele.</p><p id="p-22">In addition, for comparison, we conducted a similar tetrad dissection and mating analysis with one of the spontaneous rough-colony isolates, JAY663. We examined the phenotypes of its haploid derivatives and found that none displayed the rough colony phenotype; they were all smooth (∼100 examined). We then took twelve of these haploids comprising three full tetrads (JAY2176 through JAY2187), determined their mating types, and conducted all possible mating combinations between them (<a id="xref-fig-1-8" class="xref-fig" href="#F1">Figure 1L</a>). In this case, all 36 crosses resulted in rough colony diploids, indicating that all twelve haploid parents carried the causal recessive mutant allele, and therefore, JAY663 must have been homozygous for it. This was consistent with the hypothesis that copy-neutral LOH might be responsible for the sporadic appearance of the mutant morphology phenotype in JAY270 clonal derivatives.</p></div><div id="sec-13" class="subsection"><h3>Rough colony phenotype results from loss of the wild type <em>ACE2</em> allele in diploids</h3><p id="p-23">The results described above suggested that the rough colony phenotype was diploid-specific, and caused by a recessive mutation present in heterozygous state in JAY270. Based on this premise, we divided the twelve sequenced JAY270-derived haploids from <a id="xref-fig-1-9" class="xref-fig" href="#F1">Figure 1K</a> into two groups according to their inferred genotype (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2A</a>). Group 1 included the six haploids inferred to carry the mutant recessive allele of the causal gene, whereas group 2 included the six haploids with the wild type dominant allele.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/11/3703/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Mapping and characterization of the ace2-A7 mutation. A The genetic mapping approach consisted of identifying HetSNP linkage regions in which one allele (M or P) co-segregated in all six haploids inferred to carry the rough mutant allele (red), while the other allele (P or M) co-segregated in all six haploids inferred to contain the smooth wild type allele (blue) (see Fig. 1K). B Genome Browser view of a Chr12 candidate region that satisfied the strict co-segregation criterion. This region spans 13 Kb and contains 9 genes (from YLR125W to CKI1). Review of the functional annotations of the genes in this region identified the ACE2 locus (highlighted in yellow) as a likely candidate, and further sequence review uncovered the ace2-A7 mutation. A second candidate region on Chr11 (not shown) was also identified through co-segregation analysis, but was not pursued further. C Allele replacements and complementation tests. The ace2-A7 allele from two haploid strains JAY291 (MATa) and JAY292 (MAT&#x3B1;) was replaced with the ACE2 allele, generating JAY1051 and JAY1039, respectively. Matings between the ace2-A7 and ACE2 haploids confirmed the restoration of the smooth phenotype in diploid strains derived from at least one haploid parent containing the ACE2 allele. D and E Sanger DNA sequencing analysis of ACE2 locus. D shows segments of PCR-Sanger sequencing chromatograms for the ACE2 locus from haploids with the wild type ACE2 (JAY290) and mutant ace2-A7 (JAY291) alleles. Note the difference in the A-homopolymer runs, with eight consecutive peaks in ACE2 and seven consecutive peaks in ace2-A7. E shows PCR-Sanger chromatograms from diploids (primer extension was from left to right in all cases). The JAY270 chromatogram is a mixture of the two alleles (heterozygous), while the rough colony clone JAY663 only had the ace2-A7 pattern. The inferred DNA sequences in D and E are shown below the chromatograms. The chromatograms from rough clones JAY664, JAY665, JAY912 and JAY913 were indistinguishable from JAY663 (not shown). F Gene copy number analysis of the ACE2 locus. Plots show the calculated copy number for the four genomic regions probed, relative to the average of the signal between the control probes flanking the Chr12 centromere (CEN12-L and CEN12-R), both known to be present in two copies in the JAY270 genome. The ENB1 region was another control probe, known to be present as one copy in JAY270." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-677849688" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Mapping and characterization of the &lt;em&gt;ace2-A7&lt;/em&gt; mutation. A The genetic mapping approach consisted of identifying HetSNP linkage regions in which one allele (M or P) co-segregated in all six haploids inferred to carry the rough mutant allele (red), while the other allele (P or M) co-segregated in all six haploids inferred to contain the smooth wild type allele (blue) (see Fig. 1K). B Genome Browser view of a Chr12 candidate region that satisfied the strict co-segregation criterion. This region spans 13 Kb and contains 9 genes (from &lt;em&gt;YLR125W&lt;/em&gt; to &lt;em&gt;CKI1&lt;/em&gt;). Review of the functional annotations of the genes in this region identified the &lt;em&gt;ACE2&lt;/em&gt; locus (highlighted in yellow) as a likely candidate, and further sequence review uncovered the &lt;em&gt;ace2-A7&lt;/em&gt; mutation. A second candidate region on Chr11 (not shown) was also identified through co-segregation analysis, but was not pursued further. C Allele replacements and complementation tests. The &lt;em&gt;ace2-A7&lt;/em&gt; allele from two haploid strains JAY291 (&lt;em&gt;MATa)&lt;/em&gt; and JAY292 (&lt;em&gt;MAT&#x3B1;&lt;/em&gt;) was replaced with the &lt;em&gt;ACE2&lt;/em&gt; allele, generating JAY1051 and JAY1039, respectively. Matings between the &lt;em&gt;ace2-A7&lt;/em&gt; and &lt;em&gt;ACE2&lt;/em&gt; haploids confirmed the restoration of the smooth phenotype in diploid strains derived from at least one haploid parent containing the &lt;em&gt;ACE2&lt;/em&gt; allele. D and E Sanger DNA sequencing analysis of &lt;em&gt;ACE2&lt;/em&gt; locus. D shows segments of PCR-Sanger sequencing chromatograms for the &lt;em&gt;ACE2&lt;/em&gt; locus from haploids with the wild type &lt;em&gt;ACE2&lt;/em&gt; (JAY290) and mutant &lt;em&gt;ace2-A7&lt;/em&gt; (JAY291) alleles. Note the difference in the A-homopolymer runs, with eight consecutive peaks in &lt;em&gt;ACE2&lt;/em&gt; and seven consecutive peaks in &lt;em&gt;ace2-A7&lt;/em&gt;. E shows PCR-Sanger chromatograms from diploids (primer extension was from left to right in all cases). The JAY270 chromatogram is a mixture of the two alleles (heterozygous), while the rough colony clone JAY663 only had the &lt;em&gt;ace2-A7&lt;/em&gt; pattern. The inferred DNA sequences in D and E are shown below the chromatograms. The chromatograms from rough clones JAY664, JAY665, JAY912 and JAY913 were indistinguishable from JAY663 (not shown). F Gene copy number analysis of the &lt;em&gt;ACE2&lt;/em&gt; locus. Plots show the calculated copy number for the four genomic regions probed, relative to the average of the signal between the control probes flanking the Chr12 centromere (&lt;em&gt;CEN12-L&lt;/em&gt; and &lt;em&gt;CEN12-R&lt;/em&gt;), both known to be present in two copies in the JAY270 genome. The &lt;em&gt;ENB1&lt;/em&gt; region was another control probe, known to be present as one copy in JAY270.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/11/3703/F2.medium.gif" width="347" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/8/11/3703/F2.medium.gif" width="347" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/11/3703/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/11/3703/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491274" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-24" class="first-child">Mapping and characterization of the <em>ace2-A7</em> mutation. A The genetic mapping approach consisted of identifying HetSNP linkage regions in which one allele (M or P) co-segregated in all six haploids inferred to carry the rough mutant allele (red), while the other allele (P or M) co-segregated in all six haploids inferred to contain the smooth wild type allele (blue) (see <a id="xref-fig-1-10" class="xref-fig" href="#F1">Fig. 1K</a>). B Genome Browser view of a Chr12 candidate region that satisfied the strict co-segregation criterion. This region spans 13 Kb and contains 9 genes (from <em>YLR125W</em> to <em>CKI1</em>). Review of the functional annotations of the genes in this region identified the <em>ACE2</em> locus (highlighted in yellow) as a likely candidate, and further sequence review uncovered the <em>ace2-A7</em> mutation. A second candidate region on Chr11 (not shown) was also identified through co-segregation analysis, but was not pursued further. C Allele replacements and complementation tests. The <em>ace2-A7</em> allele from two haploid strains JAY291 (<em>MATa)</em> and JAY292 (<em>MATα</em>) was replaced with the <em>ACE2</em> allele, generating JAY1051 and JAY1039, respectively. Matings between the <em>ace2-A7</em> and <em>ACE2</em> haploids confirmed the restoration of the smooth phenotype in diploid strains derived from at least one haploid parent containing the <em>ACE2</em> allele. D and E Sanger DNA sequencing analysis of <em>ACE2</em> locus. D shows segments of PCR-Sanger sequencing chromatograms for the <em>ACE2</em> locus from haploids with the wild type <em>ACE2</em> (JAY290) and mutant <em>ace2-A7</em> (JAY291) alleles. Note the difference in the A-homopolymer runs, with eight consecutive peaks in <em>ACE2</em> and seven consecutive peaks in <em>ace2-A7</em>. E shows PCR-Sanger chromatograms from diploids (primer extension was from left to right in all cases). The JAY270 chromatogram is a mixture of the two alleles (heterozygous), while the rough colony clone JAY663 only had the <em>ace2-A7</em> pattern. The inferred DNA sequences in D and E are shown below the chromatograms. The chromatograms from rough clones JAY664, JAY665, JAY912 and JAY913 were indistinguishable from JAY663 (not shown). F Gene copy number analysis of the <em>ACE2</em> locus. Plots show the calculated copy number for the four genomic regions probed, relative to the average of the signal between the control probes flanking the Chr12 centromere (<em>CEN12-L</em> and <em>CEN12-R</em>), both known to be present in two copies in the JAY270 genome. The <em>ENB1</em> region was another control probe, known to be present as one copy in JAY270.</p><div class="sb-div caption-clear"></div></div></div><p id="p-25">We previously reported the whole genome sequence of the JAY291 haploid (<a id="xref-ref-1-5" class="xref-bibr" href="#ref-1">Argueso <em>et al.</em> 2009</a>). Since then we have sequenced the genomes of 55 additional JAY270-derived haploids, including the haploids used in the crosses shown in <a id="xref-fig-1-11" class="xref-fig" href="#F1">Figure 1K</a>. This genome sequence dataset, comprising fourteen sets of four-spore tetrads, was generated in a project to characterize the abundance, distribution and phasing of heterozygous loci in the JAY270 genome, the full results of which will be described elsewhere. These haploid genomic sequences were used to create a draft map of phased heterozygous single nucleotide polymorphisms (HetSNPs) containing 12,023 loci unevenly distributed across the genome (Fig. S1).</p><p id="p-26">In order to map the gene underlying the colony morphology trait, we searched for the JAY270 HetSNPs that co-segregated among the haploids from groups 1 or 2. We interrogated each of the genome-wide HetSNPs, and determined which had alleles that co-segregated in all six individuals within group 1, and conversely, had the other allele co-segregating in all six individuals within group 2. This analysis identified only two candidate regions that fit the strict co-segregation criterion. One of the candidate regions corresponded to ∼30 Kb on Chr11, including thirteen genes (not shown); and the other spanned ∼15 Kb containing nine genes on the right arm of Chr12 (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2B</a>), located ∼50 Kb centromere proximal to the ribosomal DNA tandem repeats (rDNA). </p><p id="p-27">We reviewed the annotations of the 22 candidate genes, and identified a gene located in the Chr12 region, <em><a href="http://www.yeastgenome.org/locus/S000004121/overview">ACE2</a></em>, which encodes a transcription factor that controls the expression of genes involved in the mother-daughter cell separation process (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Weiss 2012</a>). In cells lacking <a href="http://www.yeastgenome.org/locus/S000004121/overview">Ace2p</a>, the daughter cell remains attached to the mother cell at the bud neck, resulting in the accumulation of multicellular clusters. These clusters lead to faster sedimentation in liquid culture compared to dispersed cells. Importantly, while both haploid <em><a href="http://www.yeastgenome.org/locus/S000004121/overview">ace2</a></em> and diploid <em><a href="http://www.yeastgenome.org/locus/S000004121/overview">ace2</a></em>/<em><a href="http://www.yeastgenome.org/locus/S000004121/overview">ace2</a></em> mutants form multicellular clusters and display faster sedimentation in liquid culture, only the diploids form rough colonies in agar media (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Voth <em>et al.</em> 2005</a>). Haploid <em><a href="http://www.yeastgenome.org/locus/S000004121/overview">ace2</a></em> mutants form normal smooth colonies. This ploidy-dependent colony morphology phenotype is a result of the interaction between the defect in mother-daughter cell separation and the difference in bud site selection between haploids (axial budding pattern) and diploids (bipolar-pattern) (<a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Voth <em>et al.</em> 2005</a>).</p><p id="p-28">We inspected the genomic sequence of the <em><a href="http://www.yeastgenome.org/locus/S000004121/overview">ACE2</a></em> gene in the haploid derivative JAY291 (<a id="xref-ref-1-6" class="xref-bibr" href="#ref-1">Argueso <em>et al.</em> 2009</a>), and compared it to the sequence in the S288c reference genome. Only one difference was identified: The wild type <em><a href="http://www.yeastgenome.org/locus/S000004121/overview">ACE2</a></em> allele in S288c contains a homopolymer run of 8 adenine nucleotides, while the mutant allele in JAY291 has 7 adenines in this region, resulting in a -1 frameshift mutation and a stop codon shortly downstream. Hence, we named the mutant allele <em><a href="http://www.yeastgenome.org/locus/S000004121/overview">ace2</a>-A7</em>. We then conducted reciprocal complementation tests to formally demonstrate that <em><a href="http://www.yeastgenome.org/locus/S000004121/overview">ace2</a>-A7</em> was the causal mutation responsible for the rough colony phenotype. The mutant alleles in haploids JAY291 and JAY292 were replaced with the wild type allele, resulting respectively in the isogenic <em><a href="http://www.yeastgenome.org/locus/S000004121/overview">ACE2</a></em> strains JAY1051 and JAY1039. When these allele replacement strains were crossed to <em><a href="http://www.yeastgenome.org/locus/S000004121/overview">ace2</a>-A7</em> strains (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2C</a>), the resulting diploids displayed the smooth colony phenotype. This result confirmed that the wild type <em><a href="http://www.yeastgenome.org/locus/S000004121/overview">ACE2</a></em> allele fully complemented the <em><a href="http://www.yeastgenome.org/locus/S000004121/overview">ace2</a>-A7</em> mutation and restored normal colony morphology in heterozygous diploids in this genetic background.</p></div><div id="sec-14" class="subsection"><h3>LOH is the primary mechanism leading to the appearance of spontaneous rough colony isolates</h3><p id="p-29">After identifying the causal relationship between a mutation at the <em><a href="http://www.yeastgenome.org/locus/S000004121/overview">ACE2</a></em> locus and the rough colony phenotype, we determined its DNA sequence in the five spontaneous rough colony derivatives isolated earlier in the study. We PCR-amplified and Sanger-sequenced the region containing the adenine homopolymer run in <em><a href="http://www.yeastgenome.org/locus/S000004121/overview">ACE2</a></em> from JAY270, from the haploid derivatives JAY290 and JAY291, and from the rough colony isolates (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2D-E</a>). This analysis confirmed the presence of a run of 8 adenines in <em><a href="http://www.yeastgenome.org/locus/S000004121/overview">ACE2</a></em> (JAY290) and 7 adenines in <em><a href="http://www.yeastgenome.org/locus/S000004121/overview">ace2</a>-A7</em> (JAY291). The chromatogram in the JAY270 heterozygous diploid was consistent with a mixture of <em><a href="http://www.yeastgenome.org/locus/S000004121/overview">ACE2</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000004121/overview">ace2</a>-A7</em> DNA templates being present in the sequencing reaction: single nucleotide peaks were observed at positions primer-proximal to the homopolymer run, and out-of-register double peaks were seen downstream of the seventh adenine nucleotide. The chromatograms for JAY663 (<a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2E</a>) and all four additional rough colony isolates (data not shown) were consistent with the presence of the <em><a href="http://www.yeastgenome.org/locus/S000004121/overview">ace2</a>-A7</em> frameshift mutation and absence of the <em><a href="http://www.yeastgenome.org/locus/S000004121/overview">ACE2</a></em> allele. The loss of the <em><a href="http://www.yeastgenome.org/locus/S000004121/overview">ACE2</a></em> allele in the diploid rough colony isolates may be explained by either copy-neutral LOH mechanisms, such as inter-homolog mitotic recombination and <em>de novo</em> -1 contraction in the adenine homopolymer run of the <em><a href="http://www.yeastgenome.org/locus/S000004121/overview">ACE2</a></em> allele, or by copy-negative mechanisms such as segmental deletion spanning <em><a href="http://www.yeastgenome.org/locus/S000004121/overview">ACE2</a></em> or Chr12 monosomy. To distinguish between these scenarios, we conducted quantitative digital droplet PCR analysis to determine the copy number status of the <em><a href="http://www.yeastgenome.org/locus/S000004121/overview">ACE2</a></em> locus relative to genes immediately to the left (<em><a href="http://www.yeastgenome.org/locus/S000003924/overview">DNM1</a></em>) and right (<em><a href="http://www.yeastgenome.org/locus/S000003991/overview">YLR001C</a></em>) of the Chr12 centromere (<em>CEN12-L</em> and <em>CEN12-R</em> ddPCR probes, respectively), and relative to the <em><a href="http://www.yeastgenome.org/locus/S000005518/overview">ENB1</a></em> gene, which is known to be present as single hemizygous copy in JAY270’s Chr15 (<a id="xref-ref-1-7" class="xref-bibr" href="#ref-1">Argueso <em>et al.</em> 2009</a>). This analysis showed that all five rough colony isolates contained two copies of <em><a href="http://www.yeastgenome.org/locus/S000004121/overview">ACE2</a></em> locus DNA material (<a id="xref-fig-2-8" class="xref-fig" href="#F2">Figure 2F</a>), thus ruling out the segmental deletion and monosomy mechanisms. </p><p id="p-30">LOH due to interhomolog recombination is typically a regional mutational mechanism. Interstitial tracts of homozygosity can span tens of kilobases, and terminal tracts are even longer, extending all the way to the telomeres (<a id="xref-ref-40-3" class="xref-bibr" href="#ref-40">St Charles and Petes 2013</a>). Therefore, in addition to being homozygous for <em><a href="http://www.yeastgenome.org/locus/S000004121/overview">ace2</a>-A7</em>, the rough colony isolates, if derived from interhomolog recombination, might also be homozygous for flanking HetSNPs. In contrast, copy neutral LOH caused by a <em>de novo</em> -1 contraction should be restricted to the adenine homopolymer run. We distinguished these mechanisms by determining the genotypes at eleven Chr12 HetSNPs flanking the <em><a href="http://www.yeastgenome.org/locus/S000004121/overview">ACE2</a></em> locus using PCR followed by Sanger sequencing or restriction fragment length polymorphism analysis (RFLP; Table S3). The results of this analysis were compiled to produce the LOH tract maps shown in <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>. As expected, JAY270 was heterozygous for all eleven markers tested. Notably, Chr12 in this strain is only heterozygous for positions to the left of the rDNA cluster (Fig. S1). This pattern is similar to that described previously for other heterozygous diploid <em>S. cerevisiae</em> genomes and is suggestive of ancestral LOH events mediated by rDNA instability (<a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">Magwene <em>et al.</em> 2011</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/11/3703/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="LOH tract maps of the ACE2 and Chr12 heterozygous regions. A The genotypes at twelve phased JAY270 Chr12 HetSNP marker loci were determined using PCR and RFLP or Sanger sequencing analyses (Table S3). The approximate coordinates of the markers are shown in Kb. The Chr12 homolog containing the ace2-A7 allele was arbitrarily designated as maternal (Chr12-M, red) and the homolog containing the wild type ACE2 allele as paternal (Chr12-P, blue). JAY270 was heterozygous at all markers, and all rough colony isolates were homozygous for the ace2-A7 allele. White boxes distal to the 450 Kb HetSNP represent &#x223C;1,500 Kb of ribosomal DNA repeats (rDNA). Chr12 right arm regions distal to the rDNA do not contain any heterozygous markers in JAY270. The red or blue shading between homologs corresponds to the directions (M or P, respectively) and approximate endpoint positions of the LOH tracts (midpoint between HetSNP markers). Panels B and C show the patterns of LOH found in independent 5-FOA resistant clones derived from JAY270 with hemizygous insertions of the CORE2 cassette (KlURA3-ScURA3-KanMX4) either adjacent to the ACE2 allele in the paternal Chr12 homolog (B) or adjacent to the ace2-A7 allele in the maternal Chr12 homolog (C). The clones selected in B were ace2-A7/ace2-A7 and formed rough colonies, whereas the clones selected in C were ACE2/ACE2 and formed smooth colonies. The genotypes at the nine HetSNP loci at the indicated positions were determined by PCR-RFLP. The clones showing continuous LOH tracts were grouped according to the endpoint interval between HetSNPs, and the number of clones in each group is indicated to the left. Panel D shows a representation of the endpoint distribution among clones with Chr12 LOH. These clones were arranged in groups according to the PCR-RFLP HetSNPs marker positions used in A, B and C. The table shows the analysis of absolute interval size, relative size compared to the CEN12 to CORE2 distance, and the number and frequency of endpoints (combined from A, B and C) for each Chr12 interval. Note that only clones with uninterrupted tracts are shown in B-C. Two clones with complex tracts (non-contiguous or bidirectional) were omitted from this figure and the endpoint distribution analysis." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-677849688" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;LOH tract maps of the &lt;em&gt;ACE2&lt;/em&gt; and Chr12 heterozygous regions. A The genotypes at twelve phased JAY270 Chr12 HetSNP marker loci were determined using PCR and RFLP or Sanger sequencing analyses (Table S3). The approximate coordinates of the markers are shown in Kb. The Chr12 homolog containing the &lt;em&gt;ace2-A7&lt;/em&gt; allele was arbitrarily designated as maternal (Chr12-M, red) and the homolog containing the wild type &lt;em&gt;ACE2&lt;/em&gt; allele as paternal (Chr12-P, blue). JAY270 was heterozygous at all markers, and all rough colony isolates were homozygous for the &lt;em&gt;ace2-A7&lt;/em&gt; allele. White boxes distal to the 450 Kb HetSNP represent &#x223C;1,500 Kb of ribosomal DNA repeats (rDNA). Chr12 right arm regions distal to the rDNA do not contain any heterozygous markers in JAY270. The red or blue shading between homologs corresponds to the directions (M or P, respectively) and approximate endpoint positions of the LOH tracts (midpoint between HetSNP markers). Panels B and C show the patterns of LOH found in independent 5-FOA resistant clones derived from JAY270 with hemizygous insertions of the CORE2 cassette (&lt;em&gt;&lt;sub&gt;Kl&lt;/sub&gt;URA3-&lt;sub&gt;Sc&lt;/sub&gt;URA3-KanMX4&lt;/em&gt;) either adjacent to the &lt;em&gt;ACE2&lt;/em&gt; allele in the paternal Chr12 homolog (B) or adjacent to the &lt;em&gt;ace2-A7&lt;/em&gt; allele in the maternal Chr12 homolog (C). The clones selected in B were &lt;em&gt;ace2-A7/ace2-A7&lt;/em&gt; and formed rough colonies, whereas the clones selected in C were &lt;em&gt;ACE2/ACE2&lt;/em&gt; and formed smooth colonies. The genotypes at the nine HetSNP loci at the indicated positions were determined by PCR-RFLP. The clones showing continuous LOH tracts were grouped according to the endpoint interval between HetSNPs, and the number of clones in each group is indicated to the left. Panel D shows a representation of the endpoint distribution among clones with Chr12 LOH. These clones were arranged in groups according to the PCR-RFLP HetSNPs marker positions used in A, B and C. The table shows the analysis of absolute interval size, relative size compared to the &lt;em&gt;CEN12&lt;/em&gt; to CORE2 distance, and the number and frequency of endpoints (combined from A, B and C) for each Chr12 interval. Note that only clones with uninterrupted tracts are shown in B-C. Two clones with complex tracts (non-contiguous or bidirectional) were omitted from this figure and the endpoint distribution analysis.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/11/3703/F3.medium.gif" width="332" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/8/11/3703/F3.medium.gif" width="332" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/11/3703/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/11/3703/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491429" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-31" class="first-child">LOH tract maps of the <em>ACE2</em> and Chr12 heterozygous regions. A The genotypes at twelve phased JAY270 Chr12 HetSNP marker loci were determined using PCR and RFLP or Sanger sequencing analyses (Table S3). The approximate coordinates of the markers are shown in Kb. The Chr12 homolog containing the <em>ace2-A7</em> allele was arbitrarily designated as maternal (Chr12-M, red) and the homolog containing the wild type <em>ACE2</em> allele as paternal (Chr12-P, blue). JAY270 was heterozygous at all markers, and all rough colony isolates were homozygous for the <em>ace2-A7</em> allele. White boxes distal to the 450 Kb HetSNP represent ∼1,500 Kb of ribosomal DNA repeats (rDNA). Chr12 right arm regions distal to the rDNA do not contain any heterozygous markers in JAY270. The red or blue shading between homologs corresponds to the directions (M or P, respectively) and approximate endpoint positions of the LOH tracts (midpoint between HetSNP markers). Panels B and C show the patterns of LOH found in independent 5-FOA resistant clones derived from JAY270 with hemizygous insertions of the CORE2 cassette (<em><sub>Kl</sub>URA3-<sub>Sc</sub>URA3-KanMX4</em>) either adjacent to the <em>ACE2</em> allele in the paternal Chr12 homolog (B) or adjacent to the <em>ace2-A7</em> allele in the maternal Chr12 homolog (C). The clones selected in B were <em>ace2-A7/ace2-A7</em> and formed rough colonies, whereas the clones selected in C were <em>ACE2/ACE2</em> and formed smooth colonies. The genotypes at the nine HetSNP loci at the indicated positions were determined by PCR-RFLP. The clones showing continuous LOH tracts were grouped according to the endpoint interval between HetSNPs, and the number of clones in each group is indicated to the left. Panel D shows a representation of the endpoint distribution among clones with Chr12 LOH. These clones were arranged in groups according to the PCR-RFLP HetSNPs marker positions used in A, B and C. The table shows the analysis of absolute interval size, relative size compared to the <em>CEN12</em> to CORE2 distance, and the number and frequency of endpoints (combined from A, B and C) for each Chr12 interval. Note that only clones with uninterrupted tracts are shown in B-C. Two clones with complex tracts (non-contiguous or bidirectional) were omitted from this figure and the endpoint distribution analysis.</p><div class="sb-div caption-clear"></div></div></div><p id="p-32">Analysis of the JAY663 isolate showed that, while it was homozygous for the <em><a href="http://www.yeastgenome.org/locus/S000004121/overview">ace2</a>-A7</em> mutation, it remained heterozygous at all other flanking markers, including those immediately proximal and immediately distal to the <em><a href="http://www.yeastgenome.org/locus/S000004121/overview">ACE2</a></em> locus. Therefore, in this particular case, we were unable to distinguish between a short gene conversion tract limited to the 8.7 Kb region between the immediate flanking HetSNPs <em>vs.</em> a <em>de novo</em> -1 contraction. Despite this limitation, we favor the gene conversion mechanism, since the rate of mitotic recombination (∼10<sup>−4</sup>; (<a id="xref-ref-40-4" class="xref-bibr" href="#ref-40">St Charles and Petes 2013</a>)) is ∼10,000 fold higher than the estimated rate of single nucleotide contractions within A homopolymer runs (∼10<sup>−8</sup>); (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Tran <em>et al.</em> 1997</a>)).</p><p id="p-33">The four remaining isolates (JAY664, JAY665, JAY912, and JAY913) were homozygous for regions well beyond the <em><a href="http://www.yeastgenome.org/locus/S000004121/overview">ACE2</a></em> locus. All four LOH tracts were uninterrupted with homozygosity for nucleotides present in the Chr12 homolog that contained the <em><a href="http://www.yeastgenome.org/locus/S000004121/overview">ace2</a>-A7</em> allele, which we arbitrarily designated the maternal homolog (Chr12-M; red in all figures). The centromere-proximal endpoints of the LOH tracts were roughly mapped to positions ranging from 39 Kb (JAY913) to 184 Kb (JAY664) from the <em><a href="http://www.yeastgenome.org/locus/S000004121/overview">ACE2</a></em> locus. On the distal side, these isolates had an additional 45 Kb of LOH that extended to the HetSNP at position 450 Kb, located 1.4 Kb proximal to the rDNA repeats. From this point, Chr12 contains ∼1,500 Kb of rDNA repeats plus another ∼600 Kb of distal homozygous single copy sequences. Since the 450 Kb HetSNP was the most distal marker in Chr12, we could not distinguish if these LOH tracts were generated as very long interstitial gene conversion events, or if they extended to the right telomere. This analysis also revealed an unexpected unselected LOH event on the left arm of Chr12 in JAY664, but in this case, it was associated with homozygosity for the nucleotide from the paternal homolog (Chr12-P, blue in all figures).</p></div><div id="sec-15" class="subsection"><h3>JAY270 does not show inherently higher rates of mitotic recombination</h3><p id="p-34">Taken together, the results described above showed that the majority of the isolates with altered colony morphology were homozygous not only at the <em><a href="http://www.yeastgenome.org/locus/S000004121/overview">ACE2</a></em> locus, but also at surrounding regions, indicating that interhomolog mitotic recombination was readily detectable in JAY270 and that it likely had a substantial role in shaping the genome of this strain. This was consistent with the uneven genomic distribution of HetSNPs (Fig. S1), with long tracts of homozygosity, suggesting that multiple LOH events accumulated in the ancestral lineage before JAY270 was isolated (<a id="xref-ref-4-4" class="xref-bibr" href="#ref-4">Basso <em>et al.</em> 2008</a>).</p><p id="p-35">To gain a deeper understanding of the degree of genomic instability in JAY270, we conducted experiments to directly measure the rate of LOH in this strain (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A-B</a>). Starting with a homozygous <em><a href="http://www.yeastgenome.org/locus/S000000747/overview">ura3</a>Δ/<a href="http://www.yeastgenome.org/locus/S000000747/overview">ura3</a>Δ</em> derivative of JAY270 (FGY50; gift from F. Galzerani), we introduced one copy of the <em><sub>Kl</sub>URA3-<sub>Sc</sub><a href="http://www.yeastgenome.org/locus/S000000747/overview">URA3</a>-KanMX4</em> CORE2 counter-selectable cassette (<a id="xref-ref-45-3" class="xref-bibr" href="#ref-45">Zhang <em>et al.</em> 2013</a>) at a position immediately proximal to the <em><a href="http://www.yeastgenome.org/locus/S000004121/overview">ACE2</a></em> locus (1.3 Kb from the adenine homopolymer run) in either homolog. We grew cultures of strains carrying this insertion and plated the cells in media containing 5-FOA to select for clones that had lost the cassette. We performed these quantitative LOH rate measurements using the derivative carrying the hemizygous CORE2 insertion at Chr12-M, in which selected LOH produces cells with an <em><a href="http://www.yeastgenome.org/locus/S000004121/overview">ACE2</a></em>/<em><a href="http://www.yeastgenome.org/locus/S000004121/overview">ACE2</a></em> genotype that remain dispersed in liquid culture. The reciprocal strain, with the CORE2 insertion at Chr12-P, could not be used for this quantitative application, since the selected LOH events produces cells with the <em><a href="http://www.yeastgenome.org/locus/S000004121/overview">ace2</a>-A7</em>/<em><a href="http://www.yeastgenome.org/locus/S000004121/overview">ace2</a>-A7</em> genotype that aggregate, and thus the number of colony forming units observed in 5-FOA plates may not precisely correspond to the number of LOH-carrying cells in the population.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/11/3703/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Quantitative analyses of LOH. The median absolute rates of LOH events (A) and normalized LOH rates per kilobase (respective CEN to CORE2 distance) (B) at four different genomic positions were measured in the CG379 laboratory strain background (gray bars) and in the JAY270 bioethanol strain background (black bars). CG379-derived diploids were homozygous at all genomic positions, except at the MAT locus and at the indicated marker positions. JAY270-derived diploids were heterozygous at multiple sites across the genome (Fig. S1), and at the indicated marker positions. In the X axis, Chr4, Chr7, Chr12, and Chr13 indicate diploids with hemizygous insertions of the CORE2 cassette (KlURA3-ScURA3-KanMX4) at each of those chromosomes. The LOH rate shown for Chr12 in JAY270 was derived from the strain carrying the ACE2/ace2-A7::CORE2 insertion (smooth Ura+ to smooth 5-FOAR transition). Error bars indicate 95% confidence intervals (CI) for each rate measurement." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-677849688" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Quantitative analyses of LOH. The median absolute rates of LOH events (A) and normalized LOH rates per kilobase (respective &lt;em&gt;CEN&lt;/em&gt; to CORE2 distance) (B) at four different genomic positions were measured in the CG379 laboratory strain background (gray bars) and in the JAY270 bioethanol strain background (black bars). CG379-derived diploids were homozygous at all genomic positions, except at the &lt;em&gt;MAT&lt;/em&gt; locus and at the indicated marker positions. JAY270-derived diploids were heterozygous at multiple sites across the genome (Fig. S1), and at the indicated marker positions. In the X axis, Chr4, Chr7, Chr12, and Chr13 indicate diploids with hemizygous insertions of the CORE2 cassette (&lt;em&gt;&lt;sub&gt;Kl&lt;/sub&gt;URA3-&lt;sub&gt;Sc&lt;/sub&gt;URA3-Kan&lt;/em&gt;MX4) at each of those chromosomes. The LOH rate shown for Chr12 in JAY270 was derived from the strain carrying the &lt;em&gt;ACE2&lt;/em&gt;/&lt;em&gt;ace2-A7&lt;/em&gt;::&lt;em&gt;CORE2&lt;/em&gt; insertion (smooth Ura&lt;sup&gt;+&lt;/sup&gt; to smooth 5-FOA&lt;sup&gt;R&lt;/sup&gt; transition). Error bars indicate 95% confidence intervals (CI) for each rate measurement.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/11/3703/F4.medium.gif" width="440" height="207"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/8/11/3703/F4.medium.gif" width="440" height="207"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/11/3703/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/11/3703/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491255" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-36" class="first-child">Quantitative analyses of LOH. The median absolute rates of LOH events (A) and normalized LOH rates per kilobase (respective <em>CEN</em> to CORE2 distance) (B) at four different genomic positions were measured in the CG379 laboratory strain background (gray bars) and in the JAY270 bioethanol strain background (black bars). CG379-derived diploids were homozygous at all genomic positions, except at the <em>MAT</em> locus and at the indicated marker positions. JAY270-derived diploids were heterozygous at multiple sites across the genome (Fig. S1), and at the indicated marker positions. In the X axis, Chr4, Chr7, Chr12, and Chr13 indicate diploids with hemizygous insertions of the CORE2 cassette (<em><sub>Kl</sub>URA3-<sub>Sc</sub>URA3-Kan</em>MX4) at each of those chromosomes. The LOH rate shown for Chr12 in JAY270 was derived from the strain carrying the <em>ACE2</em>/<em>ace2-A7</em>::<em>CORE2</em> insertion (smooth Ura<sup>+</sup> to smooth 5-FOA<sup>R</sup> transition). Error bars indicate 95% confidence intervals (CI) for each rate measurement.</p><div class="sb-div caption-clear"></div></div></div><p id="p-37">The median frequency of selected homozygosity for the <em><a href="http://www.yeastgenome.org/locus/S000004121/overview">ACE2</a></em> allele was 1.2 × 10<sup>−4</sup>, and the corresponding LOH rate was calculated to be 1.5 × 10<sup>−5</sup>/cell division (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4A</a>). These measurements were in agreement with the unselected frequency of <em><a href="http://www.yeastgenome.org/locus/S000004121/overview">ace2</a>-A7</em> homozygosity estimated earlier in the study (5 rough isolates from ∼50,000 colonies). We also used hemizygous CORE2 insertions to measure LOH rates at three other positions in the genome (Chr4 near <em><a href="http://www.yeastgenome.org/locus/S000002720/overview">SSF2</a></em>, Chr7 near <em><a href="http://www.yeastgenome.org/locus/S000003520/overview">MAL13</a></em>, and Chr13 near <em><a href="http://www.yeastgenome.org/locus/S000004937/overview">ADH6</a></em>). In order to provide a reference for comparison of LOH rates measured in JAY270, we created these same four CORE2 insertions in a conventional homozygous diploid laboratory strain background routinely used to study genomic instability mechanisms, including LOH (strain CG379; <a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4A-B</a>) (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Morrison <em>et al.</em> 1991</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Rosen <em>et al.</em> 2013</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Cornelio <em>et al.</em> 2017</a>). The measured LOH rates (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4A</a>) were either not significantly different between the two strain backgrounds at Chr12 (<em>P</em> = 0.225) and at Chr7 (<em>P</em> = 0.307), or when different, they were somewhat lower in JAY270 at Chr13 (<em>P</em> = 0.033) and Chr4 (<em>P</em> &lt; 0.001). Overall, these direct quantitative comparisons indicated that the two strain backgrounds have similar LOH rates, and therefore their respective inherent levels of genomic instability appear to be quite comparable (<em>i.e.</em>, JAY270 is as stable as a conventional laboratory strain).</p><p id="p-38">In addition, we asked whether LOH events occur more frequently at the Chr12 interval that spans <em><a href="http://www.yeastgenome.org/locus/S000004121/overview">ACE2</a></em> relative to other regions of the genome. Given that most LOH tracts were suggestive of crossover or break-induced replication (BIR) recombination mechanisms (see below), we calculated the normalized LOH rate per Kb for each interval, by dividing the absolute LOH rate by the distance between the centromeres and the respective CORE2 insertion sites. This analysis (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4B</a>) indicated that all four intervals have relatively similar rates of LOH, and that recombination near the <em><a href="http://www.yeastgenome.org/locus/S000004121/overview">ACE2</a></em> locus does not appear to be elevated by the presence of the rDNA repeats on its distal side.</p><p id="p-39">We also used the 5-FOA<sup>R</sup> selection approach to characterize the qualitative nature of Chr12 LOH tracts in a larger set of independently selected clones. In contrast to the quantitative rate assays above, the difference in cell aggregation phenotype between LOH leading to <em><a href="http://www.yeastgenome.org/locus/S000004121/overview">ACE2</a>/<a href="http://www.yeastgenome.org/locus/S000004121/overview">ACE2</a></em> <em>vs.</em> <em><a href="http://www.yeastgenome.org/locus/S000004121/overview">ace2</a>-A7/<a href="http://www.yeastgenome.org/locus/S000004121/overview">ace2</a>-A7</em> genotypes was not a concern. Thus, we used nine PCR-RFLP markers to map LOH tracts from 5-FOA<sup>R</sup> clones derived from both CORE2 insertions, at Chr12-P (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a>) or at Chr12-M (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3C</a>). The LOH tracts selected from both strains resembled each other’s, and the tracts observed in the five initial spontaneous rough colony isolates (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3A</a>). All 41 selected Chr12 LOH clones were heterozygous for the left arm, and 39 of them had uninterrupted LOH tracts going from M/P to P/P or M/M on the right arm, starting at positions between <em>CEN12</em> and <em><a href="http://www.yeastgenome.org/locus/S000004121/overview">ACE2</a></em>, and extending up to the 450 Kb HetSNP. The two remaining clones had more complex tracts, including interruptions or homozygosity for the homolog opposite to the direction of selection (data not shown). The predominant LOH tract patterns were consistent with interhomolog mitotic crossover or BIR mechanisms.</p><p id="p-40">Finally, we also investigated the distribution of LOH endpoints on Chr12. The region between <em>CEN12</em> and <em><a href="http://www.yeastgenome.org/locus/S000004121/overview">ACE2</a></em> was divided in five intervals delimited by PCR-RFLP HetSNP markers. The distribution of endpoints found at these intervals was not significantly different between the strains carrying the CORE2 insertion at Chr12-P and Chr12-M (χ<sup>2</sup> = 0.855; <em>P</em> = 0.93), suggesting that both homologs shared similar mitotic recombination properties. This similarity allowed us to pool the endpoint distribution data from the 19 5-FOA<sup>R</sup> clones derived from the Chr12-P insertion to the endpoints from 20 5-FOA<sup>R</sup> clones derived from the Chr12-M, and four of the original spontaneous rough clones (JAY664, JAY665, JAY912, JAY913). We compared the total number of endpoints leading to LOH observed within each interval to the expected distribution if endpoints were allocated purely as a function of the sizes of the physical intervals. This analysis indicated that the observed endpoint distribution was significantly different from this simple model (χ<sup>2</sup> = 18.51; <em>P</em> = 0.001) (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3D</a>), suggesting the possible presence of a mild hotspot for initiation of mitotic recombination, particularly between the 227 and 292 Kb HetSNP markers.</p></div></div><div class="section discussion" id="sec-16"><h2 class="">Discussion</h2><div id="sec-17" class="subsection"><h3>Rough colony phenotype and mutation in <em>ACE2</em></h3><p id="p-41">We showed that a frequent colony morphology transition in JAY270 resulted from mitotic recombination events at a region of Chr12 heterozygous for a mutation in the <em><a href="http://www.yeastgenome.org/locus/S000004121/overview">ACE2</a></em> gene. Importantly, <em><a href="http://www.yeastgenome.org/locus/S000004121/overview">ace2</a>-A7/<a href="http://www.yeastgenome.org/locus/S000004121/overview">ace2</a>-A7</em> homozygosity also significantly accelerated the sedimentation of yeast cells in liquid culture, an undesirable trait that is typically associated with decreased fermentation kinetics and clogging of centrifuges and pipes at distilleries that use yeast recycling in fed-batch sugarcane bioethanol production (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Paulino de Souza <em>et al.</em> 2018</a>; <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Della-Bianca <em>et al.</em> 2013</a>).</p><p id="p-42">The <em><a href="http://www.yeastgenome.org/locus/S000004121/overview">ACE2</a></em> gene encodes a transcription factor that controls the expression of at least 20 genes involved in septum destruction and mother-daughter cell separation (<a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Weiss 2012</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Doolin <em>et al.</em> 2001</a>). The rough colony phenotype was easily discernible in <em><a href="http://www.yeastgenome.org/locus/S000004121/overview">ace2</a>-A7/<a href="http://www.yeastgenome.org/locus/S000004121/overview">ace2</a>-A7</em> diploids, but was not observed in <em><a href="http://www.yeastgenome.org/locus/S000004121/overview">ace2</a>-A7</em> haploids. This ploidy-specificity of the colony morphology phenotype caused by loss of <em><a href="http://www.yeastgenome.org/locus/S000004121/overview">ACE2</a></em> function has been previously attributed to the difference in budding patterns between diploid and haploid cells (<a id="xref-ref-43-3" class="xref-bibr" href="#ref-43">Voth <em>et al.</em> 2005</a>). Diploid cells typically display a bipolar pattern of budding, which when combined with cell-separation defects result in textured colony morphology. In contrast, haploid cells follow an axial budding program, during which each new bud emerges in a position adjacent to the previous bud scar. Mutation of the <em><a href="http://www.yeastgenome.org/locus/S000003852/overview">BUD4</a></em> gene gives rise to haploids that display a bipolar budding pattern typical of diploid cells. <a id="xref-ref-43-4" class="xref-bibr" href="#ref-43">Voth <em>et al.</em> 2005</a> showed that <em><a href="http://www.yeastgenome.org/locus/S000004121/overview">ace2</a> <a href="http://www.yeastgenome.org/locus/S000003852/overview">bud4</a></em> double mutant haploids display bipolar budding and a rough colony appearance indistinguishable from <em><a href="http://www.yeastgenome.org/locus/S000004121/overview">ace2</a>/<a href="http://www.yeastgenome.org/locus/S000004121/overview">ace2</a></em> diploids. Additionally, mutations in <em><a href="http://www.yeastgenome.org/locus/S000004121/overview">ACE2</a></em> regulators, such as <em><a href="http://www.yeastgenome.org/locus/S000005105/overview">CBK1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000001672/overview">HYM1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000001144/overview">KIC1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000001859/overview">MOB2</a></em>, cause similar phenotypes including defects in cell separation and altered colony morphology.</p><p id="p-43">The <em><a href="http://www.yeastgenome.org/locus/S000004121/overview">ace2</a>-A7</em> −1 frameshift allele found in JAY270 leads to a premature stop codon resulting in a truncated, likely inactive, protein that lacks three zinc finger domains and a nuclear localization sequence (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">McBride <em>et al.</em> 1999</a>). Another heterozygous diploid industrial strain, FostersB used in brewing (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Borneman <em>et al.</em> 2011</a>), has a +1 frameshift variant in the same adenine homopolymer region of <em><a href="http://www.yeastgenome.org/locus/S000004121/overview">ACE2</a></em> (<em><a href="http://www.yeastgenome.org/locus/S000004121/overview">ace2</a>-A9</em> allele), suggesting that it is also vulnerable to phenotypic changes caused by LOH spanning this locus.</p><p id="p-44">Interestingly, <em>de novo</em> mutations in <em><a href="http://www.yeastgenome.org/locus/S000004121/overview">ACE2</a></em> have been previously identified in multiple bioreactor and chemostat experimental evolution studies using <em>S. cerevisiae</em> haploids (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Oud <em>et al.</em> 2013</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Koschwanez <em>et al.</em> 2013</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Ratcliff <em>et al.</em> 2015</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Hope <em>et al.</em> 2017</a>). The <em><a href="http://www.yeastgenome.org/locus/S000004121/overview">ace2</a>-A7</em>, <em><a href="http://www.yeastgenome.org/locus/S000004121/overview">ace2</a>-A9</em>, and additional loss-of-function alleles were identified in aggregating, fast-sedimenting adapted clones. This acquired multi-cellularity trait is thought to have provided them with enhanced fitness under the specific laboratory conditions where they evolved. In the specific case of JAY270 in the bioethanol distillery environment, it is not known if or how the switch between dispersed to aggregated cell growth states has an effect on fitness. However, it is possible that JAY270’s heterozygous <em><a href="http://www.yeastgenome.org/locus/S000004121/overview">ACE2</a>/ace-A7</em> genotype may offer it the benefit of versatility, by giving individuals in the population the ability to quickly access the aggregated state through LOH. This may provide a short-term adaptive solution as cells encounter various environmental challenges during industrial fermentation (<a id="xref-ref-4-5" class="xref-bibr" href="#ref-4">Basso <em>et al.</em> 2008</a>). Individuals in the population could then later return to the dispersed state through expansion or contraction of the adenine run (<a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Tran <em>et al.</em> 1997</a>), or through sporulation and mating to an <em><a href="http://www.yeastgenome.org/locus/S000004121/overview">ACE2</a></em> haploid. Interestingly, a recent microsatellite-based phylogenetic study of yeast strains isolated from sugarcane bioethanol distilleries reported that within a cluster of isolates that were genetically similar to strain PE2 (the parent of JAY270), some of the isolates formed rough colonies while others had smooth colonies (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Reis <em>et al.</em> 2017</a>). This observation suggests the possibility that smooth-to-rough and perhaps also rough-back-to-smooth, transitions could actually be occurring within PE2 populations in distillery environments.</p></div><div id="sec-18" class="subsection"><h3>Analyses of spontaneous and selected LOH clones</h3><p id="p-45">Mapping of Chr12 heterozygosity in 44 spontaneous and selected clones revealed long tracts of LOH extending for several Kb and spanning hundreds of HetSNPs in addition to the <em><a href="http://www.yeastgenome.org/locus/S000004121/overview">ACE2</a></em> locus (<a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3</a>). Overall, the pattern of recombination observed was simple and consistent with the majority of events being the result of allelic mitotic crossovers or BIR (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Symington <em>et al.</em> 2014</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Kramara <em>et al.</em> 2018</a>). However, we disfavor the BIR mechanism in this case, because the efficiency of BIR is greatly reduced (&lt;40%) when the segment of DNA to be synthesized is longer than ∼100 Kb (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Donnianni and Symington 2013</a>). In the clones characterized here, BIR would require DNA synthesis across extremely long Chr12 DNA segments, including regions proximal to the 450 Kb HetSNP, across ∼1,500 Kb of the highly repetitive rDNA region, and additional ∼600 Kb to reach the right telomere. Thus, completing BIR through this specific region would be particularly challenging, while resolving the recombination intermediates toward the crossover outcome would not.</p><p id="p-46">The <em><a href="http://www.yeastgenome.org/locus/S000004121/overview">ACE2</a></em> gene is positioned fairly close (∼50 Kb proximal) to the most unstable region of the yeast genome, the rDNA repeats (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Ganley and Kobayashi 2014</a>). We asked whether this relative proximity to the rDNA repeats at Chr12 could be leading to an elevated incidence of recombination involving the <em><a href="http://www.yeastgenome.org/locus/S000004121/overview">ACE2</a></em> locus. The normalized rate of mitotic recombination measured at <em><a href="http://www.yeastgenome.org/locus/S000004121/overview">ACE2</a></em> was very similar to or lower than the rates measured at other regions of the genome (<a id="xref-fig-4-6" class="xref-fig" href="#F4">Figure 4B</a>), thus we conclude that the rDNA does not destabilize the centromere-proximal regions of the right arm of Chr12. This result was consistent with previous studies that showed a ∼10 fold higher rate of recombination on the distal side of the rDNA compared to the proximal side (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Casper <em>et al.</em> 2008</a>), and with Chr12 distal tracts of LOH detected in several wild <em>S. cerevisiae</em> isolates (<a id="xref-ref-24-4" class="xref-bibr" href="#ref-24">Magwene <em>et al.</em> 2011</a>).</p><p id="p-47">Surprisingly, our analysis uncovered one rough clone (JAY664) that carried two coincident LOH events in each arm of Chr12 and in opposite directions (the selected [<em><a href="http://www.yeastgenome.org/locus/S000004121/overview">ace2</a>-A7/<a href="http://www.yeastgenome.org/locus/S000004121/overview">ace2</a>-A7</em>] M/M LOH in the right arm, and an unselected P/P LOH in the left arm). The rate of LOH that we measured at the right arm of Chr12 was approximately 1.5 × 10<sup>−5</sup>, thus the rate for two independent events on this chromosome is predicted to be extremely rare (∼10<sup>−10</sup>). Intriguing observations of elevated rates of coincident mitotic recombination in yeast have been reported previously and generally interpreted as linked to a subpopulation of recombination-prone cells (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Fogel and Hurst 1963</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Minet <em>et al.</em> 1980</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Montelone <em>et al.</em> 1981</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Golin and Esposito 1984</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Golin and Tampe 1988</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Freeman and Hoffmann 2007</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Forche <em>et al.</em> 2009</a>). We speculate that JAY664 may have been derived from an analogous genome-wide instability mechanism. Further work offering supporting evidence in this regard is in progress and will be described elsewhere.</p></div><div id="sec-19" class="subsection"><h3>Genomic instability in JAY270 relative to a conventional laboratory strain background</h3><p id="p-48">Finally, the rates of LOH measured at all four genomic regions in JAY270 were comparable to the rates at the same positions in the CG379 laboratory strain background (<a id="xref-fig-4-7" class="xref-fig" href="#F4">Figure 4</a>). The rates of LOH observed in this study in both strains were also compatible with the rates of allelic mitotic recombination reported in the literature (<a id="xref-ref-40-5" class="xref-bibr" href="#ref-40">St Charles and Petes 2013</a>). This result is significant because industrial yeast strains, including PE2, generally have a reputation for having inherently unstable genomes (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">de Amorim <em>et al.</em> 2017</a>). The example characterized in our study suggests that, while karyotype instability is indeed frequently detected in yeast clones isolated from different industrial settings, the genomes of this group of strains may not necessarily be inherently less stable than those of conventional laboratory strains. This misconception may stem from the structural complexity of the genomes of wild and industrial isolates that make recombination between pairs of polymorphic homologs detectable through a simple karyotype analysis. These same recombination events, however, produce no net genetic change when occurring between perfectly identical pairs of homologs in homozygous laboratory diploids. Since allelic interhomolog mitotic recombination is not detectable in most conventional laboratory experimental settings, these strains may appear to be relatively stable, when in reality, recombination is also happening in them at similar rates. We believe that, at least in the case of JAY270, the high genomic instability reputation is not deserved, and the same might also be true for other industrial strains.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-49">We thank Tom Petes and Michael McMurray for valuable insights and comments on the manuscript. We are also grateful to the anonymous reviewers who offered very thoughtful feedback on our manuscript. NMVS received a pre-doctoral fellowship from Brazil’s CAPES (0316/13-0). PC and AVP were supported by a UGC fellowship. KTN was supported by a Wellcome Trust-DBT India Alliance Intermediate fellowship- (IA/I/11/2500268) and IISER-TVM intramural funds. Research reported here was supported by the NIH award number R35GM119788 to JLA.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-2"><p id="p-50">Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.6406598">https://doi.org/10.25387/g3.6406598</a>.</p></li><li class="fn" id="fn-3"><p id="p-51"><em>Communicating editor: J. Hesselberth</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2018-05-20"><span class="received-label">Received </span>May 20, 2018.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2018-09-23"><span class="accepted-label">Accepted </span>September 23, 2018.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © by the Genetics Society of America</li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3703.1" data-doi="10.1101/gr.091777.109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Argueso</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carazzolle</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mieczkowski</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Duarte</span>  <span class="cit-name-given-names">F. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Netto</span>  <span class="cit-name-given-names">O. V. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genome structure of a <em>Saccharomyces cerevisiae</em> strain widely used in bioethanol production.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">2258</span>–<span class="cit-lpage">2270</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.091777.109</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.091777.109%26rft_id%253Dinfo%253Apmid%252F19812109%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxOS8xMi8yMjU4IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMS8zNzAzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3703.2" data-doi="10.1073/pnas.0804529105"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Argueso</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Westmoreland</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mieczkowski</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gawel</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petes</span>  <span class="cit-name-given-names">T. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Double-strand breaks associated with repetitive DNA can reshape the genome.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">11845</span>–<span class="cit-lpage">11850</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.0804529105</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0804529105%26rft_id%253Dinfo%253Apmid%252F18701715%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA1LzMzLzExODQ1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMS8zNzAzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-book" id="cit-8.11.3703.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ausubel</span>  <span class="cit-name-given-names">F. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brent</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kingston</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moore</span>  <span class="cit-name-given-names">D. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seidman</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-source">Current Protocols in Molecular Biology</span>, <span class="cit-publ-name">John Wiley &amp; Sons</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3703.4" data-doi="10.1111/j.1567-1364.2008.00428.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Basso</span>  <span class="cit-name-given-names">L. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Amorim</span>  <span class="cit-name-given-names">H. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Oliveira</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lopes</span>  <span class="cit-name-given-names">M. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Yeast selection for fuel ethanol production in Brazil.</span> <abbr class="cit-jnl-abbrev">FEMS Yeast Res.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1155</span>–<span class="cit-lpage">1163</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1567-1364.2008.00428.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BYeast%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1567-1364.2008.00428.x%26rft_id%253Dinfo%253Apmid%252F18752628%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1567-1364.2008.00428.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18752628&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3703.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260114400018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3703.5" data-doi="10.1371/journal.pgen.1001287"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Borneman</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Desany</span>  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Riches</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Affourtit</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forgan</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Whole-genome comparison reveals novel genetic elements that characterize the genome of industrial strains of <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e1001287</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1001287</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DBorneman%26rft.auinit1%253DA.%2BR.%26rft.volume%253D7%26rft.issue%253D2%26rft.spage%253De1001287%26rft.epage%253De1001287%26rft.atitle%253DWhole-genome%2Bcomparison%2Breveals%2Bnovel%2Bgenetic%2Belements%2Bthat%2Bcharacterize%2Bthe%2Bgenome%2Bof%2Bindustrial%2Bstrains%2Bof%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1001287%26rft_id%253Dinfo%253Apmid%252F21304888%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1001287&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21304888&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3703.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3703.6" data-doi="10.1002/yea.789"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carro</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pina</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Genetic analysis of the karyotype instability in natural wine yeast strains.</span> <abbr class="cit-jnl-abbrev">Yeast</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">1457</span>–<span class="cit-lpage">1470</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/yea.789</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%2B%2528Chichester%252C%2BEngland%2529%26rft.stitle%253DYeast%26rft.aulast%253DCarro%26rft.auinit1%253DD.%26rft.volume%253D18%26rft.issue%253D16%26rft.spage%253D1457%26rft.epage%253D1470%26rft.atitle%253DGenetic%2Banalysis%2Bof%2Bthe%2Bkaryotype%2Binstability%2Bin%2Bnatural%2Bwine%2Byeast%2Bstrains.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fyea.789%26rft_id%253Dinfo%253Apmid%252F11748723%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/yea.789&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11748723&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3703.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000173263000001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3703.7" data-doi="10.1371/journal.pgen.1000105"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Casper</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mieczkowski</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gawel</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petes</span>  <span class="cit-name-given-names">T. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Low levels of DNA polymerase alpha induce mitotic and meiotic instability in the ribosomal DNA gene cluster of Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e1000105</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1000105</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DCasper%26rft.auinit1%253DA.%2BM.%26rft.volume%253D4%26rft.issue%253D6%26rft.spage%253De1000105%26rft.epage%253De1000105%26rft.atitle%253DLow%2Blevels%2Bof%2BDNA%2Bpolymerase%2Balpha%2Binduce%2Bmitotic%2Band%2Bmeiotic%2Binstability%2Bin%2Bthe%2Bribosomal%2BDNA%2Bgene%2Bcluster%2Bof%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000105%26rft_id%253Dinfo%253Apmid%252F18584028%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000105&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18584028&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3703.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3703.8" data-doi="10.1016/j.dnarep.2017.02.012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cornelio</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sedam</span>  <span class="cit-name-given-names">H. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferrarezi</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sampaio</span>  <span class="cit-name-given-names">N. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Argueso</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Both R-loop removal and ribonucleotide excision repair activities of RNase H2 contribute substantially to chromosome stability.</span> <abbr class="cit-jnl-abbrev">DNA Repair (Amst.)</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">110</span>–<span class="cit-lpage">114</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.dnarep.2017.02.012</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDNA%2BRepair%2B%2528Amst.%2529%26rft.volume%253D52%26rft.spage%253D110%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.dnarep.2017.02.012%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.dnarep.2017.02.012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-book" id="cit-8.11.3703.9" data-doi="10.1007/978-3-319-55804-2_17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de Amorim</span>  <span class="cit-name-given-names">H. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Amorim Neto</span>  <span class="cit-name-given-names">H. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lopes</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Lima Paulillo</span>  <span class="cit-name-given-names">S. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Evolution of Yeast Selection for Fuel Ethanol: Breaking Paradigms</span>, <span class="cit-source">Diversity and Benefits of Microorganisms from the Tropics</span>, edited by <span class="cit-ed"><span class="cit-name-surname">de Azevedo</span>  <span class="cit-name-given-names">J.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Quecine</span>  <span class="cit-name-given-names">M.</span></span>. <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">New York</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/978-3-319-55804-2_17</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDiversity%2Band%2BBenefits%2Bof%2BMicroorganisms%2Bfrom%2Bthe%2BTropics%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-3-319-55804-2_17%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-3-319-55804-2_17&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3703.10" data-doi="10.1007/s00253-012-4631-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Della-Bianca</span>  <span class="cit-name-given-names">B. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Basso</span>  <span class="cit-name-given-names">T. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stambuk</span>  <span class="cit-name-given-names">B. U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Basso</span>  <span class="cit-name-given-names">L. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gombert</span>  <span class="cit-name-given-names">A. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">What do we know about the yeast strains from the Brazilian fuel ethanol industry?</span> <abbr class="cit-jnl-abbrev">Appl. Microbiol. Biotechnol.</abbr> <span class="cit-vol">97</span>: <span class="cit-fpage">979</span>–<span class="cit-lpage">991</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00253-012-4631-x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAppl.%2BMicrobiol.%2BBiotechnol.%26rft.volume%253D97%26rft.spage%253D979%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00253-012-4631-x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00253-012-4631-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000314043300005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3703.11" data-doi="10.1073/pnas.1309800110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Donnianni</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Symington</span>  <span class="cit-name-given-names">L. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Break-induced replication occurs by conservative DNA synthesis.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">13475</span>–<span class="cit-lpage">13480</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1309800110</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1309800110%26rft_id%253Dinfo%253Apmid%252F23898170%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTEwLzMzLzEzNDc1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMS8zNzAzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3703.12" data-doi="10.1046/j.1365-2958.2001.02388.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Doolin</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnston</span>  <span class="cit-name-given-names">L. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Butler</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Overlapping and distinct roles of the duplicated yeast transcription factors Ace2p and Swi5p.</span> <abbr class="cit-jnl-abbrev">Mol. Microbiol.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">422</span>–<span class="cit-lpage">432</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1046/j.1365-2958.2001.02388.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bmicrobiology%26rft.stitle%253DMol%2BMicrobiol%26rft.aulast%253DDoolin%26rft.auinit1%253DM.%2BT.%26rft.volume%253D40%26rft.issue%253D2%26rft.spage%253D422%26rft.epage%253D432%26rft.atitle%253DOverlapping%2Band%2Bdistinct%2Broles%2Bof%2Bthe%2Bduplicated%2Byeast%2Btranscription%2Bfactors%2BAce2p%2Band%2BSwi5p.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-2958.2001.02388.x%26rft_id%253Dinfo%253Apmid%252F11309124%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-2958.2001.02388.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11309124&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3703.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000168473700014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3703.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fogel</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hurst</span>  <span class="cit-name-given-names">D. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1963</span> <span class="cit-article-title">Coincidence relations between gene conversion and mitotic recombination in <em>Saccharomyces</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">321</span>–<span class="cit-lpage">328</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DFogel%26rft.auinit1%253DS.%26rft.volume%253D48%26rft.issue%253D3%26rft.spage%253D321%26rft.epage%253D328%26rft.atitle%253DCOINCIDENCE%2BRELATIONS%2BBETWEEN%2BGENE%2BCONVERSION%2BAND%2BMITOTIC%2BRECOMBINATION%2BIN%2BSACCHAROMYCES%26rft_id%253Dinfo%253Apmid%252F13945718%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo4OiI0OC8zLzMyMSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzgvMTEvMzcwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3703.14" data-doi="10.1534/genetics.109.103325"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Forche</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Magee</span>  <span class="cit-name-given-names">P. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Selmecki</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">May</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Evolution in <em>Candida albicans</em> populations during a single passage through a mouse host.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">182</span>: <span class="cit-fpage">799</span>–<span class="cit-lpage">811</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.109.103325</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.103325%26rft_id%253Dinfo%253Apmid%252F19414562%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgyLzMvNzk5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMS8zNzAzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3703.15" data-doi="10.1016/j.mrfmmm.2006.11.014"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Freeman</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoffmann</span>  <span class="cit-name-given-names">G. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Frequencies of mutagen-induced coincident mitotic recombination at unlinked loci in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Mutat. Res.</abbr> <span class="cit-vol">616</span>: <span class="cit-fpage">119</span>–<span class="cit-lpage">132</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.mrfmmm.2006.11.014</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMutation%2BResearch%26rft.stitle%253DMutation%2BResearch%26rft.aulast%253DFreeman%26rft.auinit1%253DK.%2BM.%26rft.volume%253D616%26rft.issue%253D1-2%26rft.spage%253D119%26rft.epage%253D132%26rft.atitle%253DFrequencies%2Bof%2Bmutagen-induced%2Bcoincident%2Bmitotic%2Brecombination%2Bat%2Bunlinked%2Bloci%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.mrfmmm.2006.11.014%26rft_id%253Dinfo%253Apmid%252F17156798%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.mrfmmm.2006.11.014&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17156798&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3703.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000244761300014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3703.16" data-doi="10.1111/1567-1364.12133"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ganley</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kobayashi</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Ribosomal DNA and cellular senescence: new evidence supporting the connection between rDNA and aging.</span> <abbr class="cit-jnl-abbrev">FEMS Yeast Res.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">49</span>–<span class="cit-lpage">59</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/1567-1364.12133</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BYeast%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1567-1364.12133%26rft_id%253Dinfo%253Apmid%252F24373458%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/1567-1364.12133&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24373458&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3703.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3703.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Golin</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Esposito</span>  <span class="cit-name-given-names">M. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title">Coincident gene conversion during mitosis in saccharomyces.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">355</span>–<span class="cit-lpage">365</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DGolin%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D107%26rft.issue%253D3%26rft.spage%253D355%26rft.epage%253D365%26rft.atitle%253DCOINCIDENT%2BGENE%2BCONVERSION%2BDURING%2BMITOSIS%2BIN%2BSACCHAROMYCES%26rft_id%253Dinfo%253Apmid%252F6376279%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTA3LzMvMzU1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMS8zNzAzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3703.18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Golin</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tampe</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">Coincident recombination during mitosis in <em>Saccharomyces</em>: Distance-dependent and -independent components.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">119</span>: <span class="cit-fpage">541</span>–<span class="cit-lpage">547</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DGolin%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D119%26rft.issue%253D3%26rft.spage%253D541%26rft.epage%253D547%26rft.atitle%253DCoincident%2Brecombination%2Bduring%2Bmitosis%2Bin%2Bsaccharomyces%253A%2Bdistance-dependent%2Band%2B-independent%2Bcomponents.%26rft_id%253Dinfo%253Apmid%252F3042509%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTE5LzMvNTQxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMS8zNzAzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3703.19" data-doi="10.1093/bioinformatics/btp253"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">B. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">C. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liang</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">K. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fluctuation analysis calculator: A web tool for the determination of mutation rate using Luria-Delbruck fluctuation analysis.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1564</span>–<span class="cit-lpage">1565</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btp253</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp253%26rft_id%253Dinfo%253Apmid%252F19369502%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp253&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19369502&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3703.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000266498300065&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3703.20" data-doi="10.1534/genetics.116.198895"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hope</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amorosi</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">A. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dang</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heil</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Experimental Evolution Reveals Favored Adaptive Routes to Cell Aggregation in Yeast.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">206</span>: <span class="cit-fpage">1153</span>–<span class="cit-lpage">1167</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.116.198895</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.116.198895%26rft_id%253Dinfo%253Apmid%252F28450459%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIwNi8yLzExNTMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzExLzM3MDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3703.21" data-doi="10.7554/eLife.00367"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koschwanez</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foster</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murray</span>  <span class="cit-name-given-names">A. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Improved use of a public good selects for the evolution of undifferentiated multicellularity.</span> <abbr class="cit-jnl-abbrev">eLife</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">e00367</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.7554/eLife.00367</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DeLife%26rft.stitle%253Delife%26rft.volume%253D2%26rft.issue%253D0%26rft.spage%253De00367%26rft.epage%253De00367%26rft.atitle%253DImproved%2Buse%2Bof%2Ba%2Bpublic%2Bgood%2Bselects%2Bfor%2Bthe%2Bevolution%2Bof%2Bundifferentiated%2Bmulticellularity%26rft_id%253Dinfo%253Adoi%252F10.7554%252FeLife.00367%26rft_id%253Dinfo%253Apmid%252F23577233%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.7554/eLife.00367&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23577233&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3703.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3703.22" data-doi="10.1016/j.tig.2018.04.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kramara</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Osia</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Malkova</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Break-Induced Replication: The Where, The Why, and The How.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">518</span>–<span class="cit-lpage">531</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tig.2018.04.002</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BGenet.%26rft.volume%253D34%26rft.spage%253D518%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2018.04.002%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2018.04.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3703.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lea</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coulson</span>  <span class="cit-name-given-names">C. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1949</span> <span class="cit-article-title">The distribution of the numbers of mutants in bacterial populations.</span> <abbr class="cit-jnl-abbrev">J. Genet.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">491</span>–<span class="cit-lpage">511</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BGenet.%26rft.volume%253D28%26rft.spage%253D491%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3703.24" data-doi="10.1073/pnas.1012544108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Magwene</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kayikci</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Granek</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reininga</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scholl</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Outcrossing, mitotic recombination, and life-history trade-offs shape genome evolution in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">1987</span>–<span class="cit-lpage">1992</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1012544108</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1012544108%26rft_id%253Dinfo%253Apmid%252F21245305%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTA4LzUvMTk4NyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzgvMTEvMzcwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3703.25" data-doi="10.1074/jbc.274.30.21029"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McBride</span>  <span class="cit-name-given-names">H. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stillman</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Distinct regions of the Swi5 and Ace2 transcription factors are required for specific gene activation.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">274</span>: <span class="cit-fpage">21029</span>–<span class="cit-lpage">21036</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1074/jbc.274.30.21029</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiological%2BChemistry%26rft.stitle%253DJ%2BBiol%2BChem%26rft.aulast%253DMcBride%26rft.auinit1%253DH.%2BJ.%26rft.volume%253D274%26rft.issue%253D30%26rft.spage%253D21029%26rft.epage%253D21036%26rft.atitle%253DDistinct%2BRegions%2Bof%2Bthe%2BSwi5%2Band%2BAce2%2BTranscription%2BFactors%2BAre%2BRequired%2Bfor%2BSpecific%2BGene%2BActivation%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.274.30.21029%26rft_id%253Dinfo%253Apmid%252F10409653%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyNzQvMzAvMjEwMjkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzExLzM3MDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3703.26" data-doi="10.1007/BF00445694"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Minet</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grossenbacher-Grunder</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thuriaux</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1980</span> <span class="cit-article-title">The origin of a centromere effect on mitotic recombination: A study in the fission yeast <em>Schizosaccharomyces pombe</em>.</span> <abbr class="cit-jnl-abbrev">Curr. Genet.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">53</span>–<span class="cit-lpage">60</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/BF00445694</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BGenet.%26rft.volume%253D2%26rft.spage%253D53%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00445694%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00445694&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3703.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Montelone</span>  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prakash</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prakash</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">1981</span> <span class="cit-article-title">Spontaneous mitotic recombination in mms8–1, an allele of the CDC9 gene of Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">147</span>: <span class="cit-fpage">517</span>–<span class="cit-lpage">525</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DMontelone%26rft.auinit1%253DB%2BA%26rft.volume%253D147%26rft.issue%253D2%26rft.spage%253D517%26rft.epage%253D525%26rft.atitle%253DSpontaneous%2Bmitotic%2Brecombination%2Bin%2Bmms8-1%252C%2Ban%2Ballele%2Bof%2Bthe%2BCDC9%2Bgene%2Bof%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Apmid%252F7021533%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6OToiMTQ3LzIvNTE3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMS8zNzAzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3703.28" data-doi="10.1073/pnas.88.21.9473"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morrison</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bell</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kunkel</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sugino</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Eukaryotic DNA polymerase amino acid sequence required for 3′-5′ exonuclease activity.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">88</span>: <span class="cit-fpage">9473</span>–<span class="cit-lpage">9477</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.88.21.9473</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DMorrison%26rft.auinit1%253DA.%26rft.volume%253D88%26rft.issue%253D21%26rft.spage%253D9473%26rft.epage%253D9477%26rft.atitle%253DEukaryotic%2BDNA%2Bpolymerase%2Bamino%2Bacid%2Bsequence%2Brequired%2Bfor%2B3%2527----5%2527%2Bexonuclease%2Bactivity.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.88.21.9473%26rft_id%253Dinfo%253Apmid%252F1658784%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiODgvMjEvOTQ3MyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzgvMTEvMzcwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3703.29" data-doi="10.1111/j.1365-294X.2009.04234.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Muller</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCusker</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Microsatellite analysis of genetic diversity among clinical and nonclinical Saccharomyces cerevisiae isolates suggests heterozygote advantage in clinical environments.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">2779</span>–<span class="cit-lpage">2786</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1365-294X.2009.04234.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DMuller%26rft.auinit1%253DL.%2BA.%26rft.volume%253D18%26rft.issue%253D13%26rft.spage%253D2779%26rft.epage%253D2786%26rft.atitle%253DMicrosatellite%2Banalysis%2Bof%2Bgenetic%2Bdiversity%2Bamong%2Bclinical%2Band%2Bnonclinical%2BSaccharomyces%2Bcerevisiae%2Bisolates%2Bsuggests%2Bheterozygote%2Badvantage%2Bin%2Bclinical%2Benvironments.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2009.04234.x%26rft_id%253Dinfo%253Apmid%252F19457175%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2009.04234.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19457175&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3703.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3703.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nadal</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carro</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernandez-Larrea</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pina</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Analysis and dynamics of the chromosomal complements of wild sparkling-wine yeast strains.</span> <abbr class="cit-jnl-abbrev">Appl. Environ. Microbiol.</abbr> <span class="cit-vol">65</span>: <span class="cit-fpage">1688</span>–<span class="cit-lpage">1695</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DApplied%2Band%2BEnvironmental%2BMicrobiology%26rft.stitle%253DAppl.%2BEnviron.%2BMicrobiol.%26rft.aulast%253DNadal%26rft.auinit1%253DD.%26rft.volume%253D65%26rft.issue%253D4%26rft.spage%253D1688%26rft.epage%253D1695%26rft.atitle%253DAnalysis%2Band%2BDynamics%2Bof%2Bthe%2BChromosomal%2BComplements%2Bof%2BWild%2BSparkling-Wine%2BYeast%2BStrains%26rft_id%253Dinfo%253Apmid%252F10103269%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYWVtIjtzOjU6InJlc2lkIjtzOjk6IjY1LzQvMTY4OCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzgvMTEvMzcwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3703.31" data-doi="10.1073/pnas.1305949110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Oud</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guadalupe-Medina</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nijkamp</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Ridder</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pronk</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genome duplication and mutations in ACE2 cause multicellular, fast-sedimenting phenotypes in evolved Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">E4223</span>–<span class="cit-lpage">E4231</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1305949110</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1305949110%26rft_id%253Dinfo%253Apmid%252F24145419%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTEwLzQ1L0U0MjIzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMS8zNzAzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3703.32" data-doi="10.1038/s41586-018-0030-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peter</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Chiara</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Friedrich</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yue</span>  <span class="cit-name-given-names">J. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pflieger</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Genome evolution across 1,011 Saccharomyces cerevisiae isolates.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">556</span>: <span class="cit-fpage">339</span>–<span class="cit-lpage">344</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41586-018-0030-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D556%26rft.spage%253D339%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41586-018-0030-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41586-018-0030-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3703.33" data-doi="10.1534/g3.113.009381"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Plech</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Visser</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Korona</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Heterosis is prevalent among domesticated but not wild strains of Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">315</span>–<span class="cit-lpage">323</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.113.009381</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D4%26rft.spage%253D315%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.113.009381%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/g3.113.009381&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3703.34" data-doi="10.1111/j.1574-6968.2008.01480.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Querol</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bond</span>  <span class="cit-name-given-names">U.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The complex and dynamic genomes of industrial yeasts.</span> <abbr class="cit-jnl-abbrev">FEMS Microbiol. Lett.</abbr> <span class="cit-vol">293</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">10</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1574-6968.2008.01480.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BMicrobiol.%2BLett.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1574-6968.2008.01480.x%26rft_id%253Dinfo%253Apmid%252F19175410%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1574-6968.2008.01480.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19175410&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3703.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3703.35" data-doi="10.1038/ncomms7102"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ratcliff</span>  <span class="cit-name-given-names">W. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fankhauser</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rogers</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Greig</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Travisano</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Origins of multicellular evolvability in snowflake yeast.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">6102</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ncomms7102</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D6%26rft.spage%253D6102%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms7102%26rft_id%253Dinfo%253Apmid%252F25600558%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms7102&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25600558&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3703.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3703.36" data-doi="10.1016/j.bjm.2016.09.017"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Reis</span>  <span class="cit-name-given-names">V. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Antonangelo</span>  <span class="cit-name-given-names">A. T. B. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bassi</span>  <span class="cit-name-given-names">A. P. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Colombi</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ceccato-Antonini</span>  <span class="cit-name-given-names">S. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Bioethanol strains of Saccharomyces cerevisiae characterised by microsatellite and stress resistance.</span> <abbr class="cit-jnl-abbrev">Braz. J. Microbiol.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">268</span>–<span class="cit-lpage">274</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.bjm.2016.09.017</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBraz.%2BJ.%2BMicrobiol.%26rft.volume%253D48%26rft.spage%253D268%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.bjm.2016.09.017%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.bjm.2016.09.017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3703.37" data-doi="10.1371/journal.pgen.1003817"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rosen</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Younkin</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">S. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casper</span>  <span class="cit-name-given-names">A. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Fragile Site Instability in Saccharomyces cerevisiae Causes Loss of Heterozygosity by Mitotic Crossovers and Break-Induced Replication.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1003817</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1003817</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003817%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003817%26rft_id%253Dinfo%253Apmid%252F24068975%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003817&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24068975&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3703.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3703.38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sikorski</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hieter</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">A system of shuttle vectors and yeast host strains designed for efficient manipulation of DNA in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">122</span>: <span class="cit-fpage">19</span>–<span class="cit-lpage">27</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DSikorski%26rft.auinit1%253DR.%2BS.%26rft.volume%253D122%26rft.issue%253D1%26rft.spage%253D19%26rft.epage%253D27%26rft.atitle%253DA%2Bsystem%2Bof%2Bshuttle%2Bvectors%2Band%2Byeast%2Bhost%2Bstrains%2Bdesigned%2Bfor%2Befficient%2Bmanipulation%2Bof%2BDNA%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Apmid%252F2659436%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMTIyLzEvMTkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzExLzM3MDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3703.39" data-doi="10.1016/j.funbio.2017.12.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Paulino de Souza</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prado</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eleutherio</span>  <span class="cit-name-given-names">E. C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bonatto</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Malavazi</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Improvement of Brazilian bioethanol production – Challenges and perspectives on the identification and genetic modification of new strains of Saccharomyces cerevisiae yeasts isolated during ethanol process.</span> <abbr class="cit-jnl-abbrev">Fungal Biol.</abbr> <span class="cit-vol">122</span>: <span class="cit-fpage">583</span>–<span class="cit-lpage">591</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.funbio.2017.12.006</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFungal%2BBiol.%26rft.volume%253D122%26rft.spage%253D583%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.funbio.2017.12.006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.funbio.2017.12.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3703.40" data-doi="10.1371/journal.pgen.1003434"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">St Charles</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petes</span>  <span class="cit-name-given-names">T. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">High-resolution mapping of spontaneous mitotic recombination hotspots on the 1.1 Mb arm of yeast chromosome IV.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1003434</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1003434</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003434%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003434%26rft_id%253Dinfo%253Apmid%252F23593029%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003434&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23593029&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3703.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3703.41" data-doi="10.1534/genetics.114.166140"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Symington</span>  <span class="cit-name-given-names">L. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rothstein</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lisby</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Mechanisms and regulation of mitotic recombination in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">795</span>–<span class="cit-lpage">835</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.114.166140</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DSymington%26rft.auinit1%253DL.%2BS.%26rft.volume%253D198%26rft.issue%253D3%26rft.spage%253D795%26rft.epage%253D835%26rft.atitle%253DMechanisms%2Band%2BRegulation%2Bof%2BMitotic%2BRecombination%2Bin%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.166140%26rft_id%253Dinfo%253Apmid%252F25381364%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk4LzMvNzk1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMS8zNzAzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3703.42" data-doi="10.1128/MCB.17.5.2859"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tran</span>  <span class="cit-name-given-names">H. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keen</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kricker</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Resnick</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gordenin</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Hypermutability of homonucleotide runs in mismatch repair and DNA polymerase proofreading yeast mutants.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">2859</span>–<span class="cit-lpage">2865</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/MCB.17.5.2859</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DTran%26rft.auinit1%253DH.%2BT.%26rft.volume%253D17%26rft.issue%253D5%26rft.spage%253D2859%26rft.epage%253D2865%26rft.atitle%253DHypermutability%2Bof%2Bhomonucleotide%2Bruns%2Bin%2Bmismatch%2Brepair%2Band%2BDNA%2Bpolymerase%2Bproofreading%2Byeast%2Bmutants.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.17.5.2859%26rft_id%253Dinfo%253Apmid%252F9111358%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjk6IjE3LzUvMjg1OSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzgvMTEvMzcwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3703.43" data-doi="10.1128/EC.4.6.1018-1028.2005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Voth</span>  <span class="cit-name-given-names">W. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olsen</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sbia</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Freedman</span>  <span class="cit-name-given-names">K. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stillman</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title"><em>ACE2</em>, <em>CBK1</em>, and <em>BUD4</em> in budding and cell separation.</span> <abbr class="cit-jnl-abbrev">Eukaryot. Cell</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">1018</span>–<span class="cit-lpage">1028</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/EC.4.6.1018-1028.2005</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEukaryotic%2BCell%26rft.stitle%253DEukaryot%2BCell%26rft.aulast%253DVoth%26rft.auinit1%253DW.%2BP.%26rft.volume%253D4%26rft.issue%253D6%26rft.spage%253D1018%26rft.epage%253D1028%26rft.atitle%253DACE2%252C%2BCBK1%252C%2Band%2BBUD4%2Bin%2BBudding%2Band%2BCell%2BSeparation%26rft_id%253Dinfo%253Adoi%252F10.1128%252FEC.4.6.1018-1028.2005%26rft_id%253Dinfo%253Apmid%252F15947194%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZXVrY2VsbCI7czo1OiJyZXNpZCI7czo4OiI0LzYvMTAxOCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzgvMTEvMzcwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3703.44" data-doi="10.1534/genetics.112.145516"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weiss</span>  <span class="cit-name-given-names">E. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Mitotic exit and separation of mother and daughter cells.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">192</span>: <span class="cit-fpage">1165</span>–<span class="cit-lpage">1202</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.112.145516</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DWeiss%26rft.auinit1%253DE.%2BL.%26rft.volume%253D192%26rft.issue%253D4%26rft.spage%253D1165%26rft.epage%253D1202%26rft.atitle%253DMitotic%2BExit%2Band%2BSeparation%2Bof%2BMother%2Band%2BDaughter%2BCells%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.145516%26rft_id%253Dinfo%253Apmid%252F23212898%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5Mi80LzExNjUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzExLzM3MDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3703.45" data-doi="10.1534/genetics.112.146522"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zeidler</span>  <span class="cit-name-given-names">A. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Puccia</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Malc</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Gene copy-number variation in haploid and diploid strains of the yeast <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">193</span>: <span class="cit-fpage">785</span>–<span class="cit-lpage">801</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.112.146522</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.146522%26rft_id%253Dinfo%253Apmid%252F23307895%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkzLzMvNzg1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMS8zNzAzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/8/11/3703.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/11/3691" title="Evidence of Zip1 Promoting Sister Kinetochore Mono-orientation During Meiosis in Budding Yeast " rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/11/3715" title="Comparing Genome-Wide Association Study Results from Different Measurements of an Underlying Phenotype" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/8/11"><span>Volume 8 Issue 11, November 2018</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/8/11.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/8/11.cover-source.jpg?itok=lFO8v6bg" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 8 (11)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/11/3703" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-tgqNu3iXD7c0FoP1SKyE2FXH4CJqtn44-CEU1gZytLg" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/8/11/3703" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/8/11/3703" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/8/11/3703" class="active" data-icon-position="" data-hide-link-title="0">A Case Study of Genomic Instability in an Industrial Strain of Saccharomyces cerevisiae</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/491247" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-YV7yXkaA9sthoCC7gjwqt5wmUaqQRojZCbjzPtVllFU" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit--2" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/11/3703.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/8/11/3703" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--5" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-J7LcnTF0ns6kIJPej_iQPOrS1aceCSaWFwE53sxnGTs" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/8/11/3703" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/11/3703.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="491247" id="node491247--2" data-pisa="ggg;8/11/3703" data-pisa-master="ggg;g3.118.200446" data-apath="/ggg/8/11/3703.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Case Study of Genomic Instability in an Industrial Strain of <em>Saccharomyces cerevisiae</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Aline Rodrigues-Prause</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-6403-0159" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Nadia M. V. Sampaio</span>, <span class="highwire-citation-author" data-delta="2">Theodore M. Gurol</span>, <span class="highwire-citation-author" data-delta="3">Guadalupe M. Aguirre</span>, <span class="highwire-citation-author" data-delta="4">Hailey N. C. Sedam</span>, <span class="highwire-citation-author" data-delta="5">Mary J. Chapman</span>, <span class="highwire-citation-author" data-delta="6">Ewa P. Malc</span>, <span class="highwire-citation-author" data-delta="7">V. P. Ajith</span>, <span class="highwire-citation-author" data-delta="8">Parijat Chakraborty</span>, <span class="highwire-citation-author" data-delta="9">Pedro A. Tizei</span>, <span class="highwire-citation-author" data-delta="10">Gonçalo A. G. Pereira</span>, <span class="highwire-citation-author" data-delta="11">Piotr A. Mieczkowski</span>, <span class="highwire-citation-author" data-delta="12">Koodali T. Nishant</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="13"><a href="http://orcid.org/0000-0002-7157-1519" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Juan Lucas Argueso</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3703-3713;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200446 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node491247--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Aline Rodrigues-Prause </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Environmental and Radiological Health Sciences</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Departamento de Genética, Evolução e Bioagentes, Instituto de Biologia, Universidade Estadual de Campinas, Campinas-SP, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aline%2BRodrigues-Prause%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rodrigues-Prause%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAline%2BRodrigues-Prause%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Nadia M. V. Sampaio </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Environmental and Radiological Health Sciences</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Cell and Molecular Biology Graduate Program, Colorado State University, Fort Collins-CO 80523</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nadia%2BM.%2BV.%2BSampaio%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sampaio%20NM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANadia%2BM.%2BV.%2BSampaio%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-6403-0159" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Nadia M. V. Sampaio</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Theodore M. Gurol </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Environmental and Radiological Health Sciences</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Cell and Molecular Biology Graduate Program, Colorado State University, Fort Collins-CO 80523</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Theodore%2BM.%2BGurol%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gurol%20TM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATheodore%2BM.%2BGurol%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Guadalupe M. Aguirre </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Environmental and Radiological Health Sciences</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Cell and Molecular Biology Graduate Program, Colorado State University, Fort Collins-CO 80523</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Guadalupe%2BM.%2BAguirre%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Aguirre%20GM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGuadalupe%2BM.%2BAguirre%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Hailey N. C. Sedam </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Environmental and Radiological Health Sciences</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Cell and Molecular Biology Graduate Program, Colorado State University, Fort Collins-CO 80523</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hailey%2BN.%2BC.%2BSedam%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sedam%20HN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHailey%2BN.%2BC.%2BSedam%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Mary J. Chapman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Environmental and Radiological Health Sciences</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mary%2BJ.%2BChapman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chapman%20MJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMary%2BJ.%2BChapman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Ewa P. Malc </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Genetics, University of North Carolina, Chapel Hill-NC 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ewa%2BP.%2BMalc%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Malc%20EP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEwa%2BP.%2BMalc%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">V. P. Ajith </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>School of Biology, Indian Institute of Science Education and Research</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=V.%2BP.%2BAjith%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ajith%20VP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AV.%2BP.%2BAjith%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Parijat Chakraborty </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>School of Biology, Indian Institute of Science Education and Research</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Parijat%2BChakraborty%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chakraborty%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AParijat%2BChakraborty%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Pedro A. Tizei </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Departamento de Genética, Evolução e Bioagentes, Instituto de Biologia, Universidade Estadual de Campinas, Campinas-SP, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pedro%2BA.%2BTizei%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tizei%20PA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APedro%2BA.%2BTizei%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Gonçalo A. G. Pereira </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Departamento de Genética, Evolução e Bioagentes, Instituto de Biologia, Universidade Estadual de Campinas, Campinas-SP, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gon%C3%A7alo%2BA.%2BG.%2BPereira%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pereira%20GA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGon%25C3%25A7alo%2BA.%2BG.%2BPereira%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Piotr A. Mieczkowski </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Genetics, University of North Carolina, Chapel Hill-NC 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Piotr%2BA.%2BMieczkowski%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mieczkowski%20PA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APiotr%2BA.%2BMieczkowski%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Koodali T. Nishant </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>School of Biology, Indian Institute of Science Education and Research</div><div class='author-affiliation'><span class='nlm-sup'>††</span>Center for Computation Modeling and Simulation, Indian Institute of Science Education and Research, Thiruvananthapuram, Trivandrum 695016, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Koodali%2BT.%2BNishant%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nishant%20KT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKoodali%2BT.%2BNishant%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-13"><div class="author-tooltip-name">Juan Lucas Argueso </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Environmental and Radiological Health Sciences</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Cell and Molecular Biology Graduate Program, Colorado State University, Fort Collins-CO 80523</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Juan%2BLucas%2BArgueso%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Argueso%20JL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJuan%2BLucas%2BArgueso%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-7157-1519" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Juan Lucas Argueso</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:lucas.argueso@colostate.edu" class="" data-icon-position="" data-hide-link-title="0">lucas.argueso@colostate.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/8/11/3703" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/8/11/3703&amp;title=A%20Case%20Study%20of%20Genomic%20Instability%20in%20an%20Industrial%20Strain%20of%20Saccharomyces%20cerevisiae" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/8/11/3703&amp;title=A%20Case%20Study%20of%20Genomic%20Instability%20in%20an%20Industrial%20Strain%20of%20Saccharomyces%20cerevisiae" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/8/11/3703&amp;title=A%20Case%20Study%20of%20Genomic%20Instability%20in%20an%20Industrial%20Strain%20of%20Saccharomyces%20cerevisiae" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/8/11/3703&amp;text=A%20Case%20Study%20of%20Genomic%20Instability%20in%20an%20Industrial%20Strain%20of%20Saccharomyces%20cerevisiae" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/8/11/3703&amp;title=A%20Case%20Study%20of%20Genomic%20Instability%20in%20an%20Industrial%20Strain%20of%20Saccharomyces%20cerevisiae" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/8/11/3703&amp;t=A%20Case%20Study%20of%20Genomic%20Instability%20in%20an%20Industrial%20Strain%20of%20Saccharomyces%20cerevisiae" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/8/11/3703&amp;title=A%20Case%20Study%20of%20Genomic%20Instability%20in%20an%20Industrial%20Strain%20of%20Saccharomyces%20cerevisiae" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/8/11/3703&amp;title=A%20Case%20Study%20of%20Genomic%20Instability%20in%20an%20Industrial%20Strain%20of%20Saccharomyces%20cerevisiae" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/491247/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="491247" id="node491247--3" data-pisa="ggg;8/11/3703" data-pisa-master="ggg;g3.118.200446" data-apath="/ggg/8/11/3703.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Case Study of Genomic Instability in an Industrial Strain of <em>Saccharomyces cerevisiae</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Aline Rodrigues-Prause</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-6403-0159" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Nadia M. V. Sampaio</span>, <span class="highwire-citation-author" data-delta="2">Theodore M. Gurol</span>, <span class="highwire-citation-author" data-delta="3">Guadalupe M. Aguirre</span>, <span class="highwire-citation-author" data-delta="4">Hailey N. C. Sedam</span>, <span class="highwire-citation-author" data-delta="5">Mary J. Chapman</span>, <span class="highwire-citation-author" data-delta="6">Ewa P. Malc</span>, <span class="highwire-citation-author" data-delta="7">V. P. Ajith</span>, <span class="highwire-citation-author" data-delta="8">Parijat Chakraborty</span>, <span class="highwire-citation-author" data-delta="9">Pedro A. Tizei</span>, <span class="highwire-citation-author" data-delta="10">Gonçalo A. G. Pereira</span>, <span class="highwire-citation-author" data-delta="11">Piotr A. Mieczkowski</span>, <span class="highwire-citation-author" data-delta="12">Koodali T. Nishant</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="13"><a href="http://orcid.org/0000-0002-7157-1519" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Juan Lucas Argueso</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3703-3713;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200446 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node491247--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Aline Rodrigues-Prause </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Environmental and Radiological Health Sciences</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Departamento de Genética, Evolução e Bioagentes, Instituto de Biologia, Universidade Estadual de Campinas, Campinas-SP, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aline%2BRodrigues-Prause%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rodrigues-Prause%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAline%2BRodrigues-Prause%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Nadia M. V. Sampaio </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Environmental and Radiological Health Sciences</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Cell and Molecular Biology Graduate Program, Colorado State University, Fort Collins-CO 80523</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nadia%2BM.%2BV.%2BSampaio%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sampaio%20NM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANadia%2BM.%2BV.%2BSampaio%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-6403-0159" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Nadia M. V. Sampaio</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Theodore M. Gurol </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Environmental and Radiological Health Sciences</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Cell and Molecular Biology Graduate Program, Colorado State University, Fort Collins-CO 80523</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Theodore%2BM.%2BGurol%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gurol%20TM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATheodore%2BM.%2BGurol%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Guadalupe M. Aguirre </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Environmental and Radiological Health Sciences</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Cell and Molecular Biology Graduate Program, Colorado State University, Fort Collins-CO 80523</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Guadalupe%2BM.%2BAguirre%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Aguirre%20GM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGuadalupe%2BM.%2BAguirre%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Hailey N. C. Sedam </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Environmental and Radiological Health Sciences</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Cell and Molecular Biology Graduate Program, Colorado State University, Fort Collins-CO 80523</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hailey%2BN.%2BC.%2BSedam%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sedam%20HN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHailey%2BN.%2BC.%2BSedam%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Mary J. Chapman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Environmental and Radiological Health Sciences</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mary%2BJ.%2BChapman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chapman%20MJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMary%2BJ.%2BChapman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Ewa P. Malc </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Genetics, University of North Carolina, Chapel Hill-NC 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ewa%2BP.%2BMalc%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Malc%20EP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEwa%2BP.%2BMalc%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">V. P. Ajith </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>School of Biology, Indian Institute of Science Education and Research</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=V.%2BP.%2BAjith%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ajith%20VP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AV.%2BP.%2BAjith%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Parijat Chakraborty </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>School of Biology, Indian Institute of Science Education and Research</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Parijat%2BChakraborty%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chakraborty%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AParijat%2BChakraborty%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Pedro A. Tizei </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Departamento de Genética, Evolução e Bioagentes, Instituto de Biologia, Universidade Estadual de Campinas, Campinas-SP, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pedro%2BA.%2BTizei%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tizei%20PA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APedro%2BA.%2BTizei%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Gonçalo A. G. Pereira </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Departamento de Genética, Evolução e Bioagentes, Instituto de Biologia, Universidade Estadual de Campinas, Campinas-SP, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gon%C3%A7alo%2BA.%2BG.%2BPereira%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pereira%20GA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGon%25C3%25A7alo%2BA.%2BG.%2BPereira%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Piotr A. Mieczkowski </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Genetics, University of North Carolina, Chapel Hill-NC 27599</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Piotr%2BA.%2BMieczkowski%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mieczkowski%20PA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APiotr%2BA.%2BMieczkowski%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Koodali T. Nishant </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>School of Biology, Indian Institute of Science Education and Research</div><div class='author-affiliation'><span class='nlm-sup'>††</span>Center for Computation Modeling and Simulation, Indian Institute of Science Education and Research, Thiruvananthapuram, Trivandrum 695016, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Koodali%2BT.%2BNishant%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nishant%20KT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKoodali%2BT.%2BNishant%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-13"><div class="author-tooltip-name">Juan Lucas Argueso </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Environmental and Radiological Health Sciences</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Cell and Molecular Biology Graduate Program, Colorado State University, Fort Collins-CO 80523</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Juan%2BLucas%2BArgueso%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Argueso%20JL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJuan%2BLucas%2BArgueso%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-7157-1519" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Juan Lucas Argueso</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:lucas.argueso@colostate.edu" class="" data-icon-position="" data-hide-link-title="0">lucas.argueso@colostate.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3703&amp;rft.epage=3713&amp;rft.atitle=A%20Case%20Study%20of%20Genomic%20Instability%20in%20an%20Industrial%20Strain%20of%20Saccharomyces%20cerevisiae&amp;rft.volume=8&amp;rft.issue=11&amp;rft.date=2018-11-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Rodrigues-Prause%2C+Aline&amp;rft.au=Sampaio%2C+Nadia+M.+V.&amp;rft.au=Gurol%2C+Theodore+M.&amp;rft.au=Aguirre%2C+Guadalupe+M.&amp;rft.au=Sedam%2C+Hailey+N.+C.&amp;rft.au=Chapman%2C+Mary+J.&amp;rft.au=Malc%2C+Ewa+P.&amp;rft.au=Ajith%2C+V.+P.&amp;rft.au=Chakraborty%2C+Parijat&amp;rft.au=Tizei%2C+Pedro+A.&amp;rft.au=Pereira%2C+Gon%C3%A7alo+A.+G.&amp;rft.au=Mieczkowski%2C+Piotr+A.&amp;rft.au=Nishant%2C+Koodali+T.&amp;rft.au=Argueso%2C+Juan+Lucas"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/491247/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/491247/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/491247/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/491247/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/491247/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/491247/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/491247/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/491247/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/491247/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/491247/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/491247/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/491247/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/8/11/3703" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-16" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/8/11/3703.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/8/11/3703.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/8/11/3703.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/8/11/3703.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
