<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/8/11/3549" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/8/11/3549.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/8/11/3549.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/8/11/3549.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/8/11/3549/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/8/11/3549.atom" />
<meta name="HW.pisa" content="ggg;8/11/3549" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Genomic Prediction Using Multi-trait Weighted GBLUP Accounting for Heterogeneous Variances and Covariances Across the Genome" />
<meta name="DC.Identifier" content="10.1534/g3.118.200673" />
<meta name="DC.Date" content="2018-11-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2018 Karaman et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Implicit assumption of common (co)variance for all loci in multi-trait Genomic Best Linear Unbiased Prediction (GBLUP) results in a genomic relationship matrix ( G ) that is common to all traits. When this assumption is violated, Bayesian whole genome regression methods may be superior to GBLUP by accounting for unequal (co)variance for all loci or genome regions. This study aimed to develop a strategy to improve the accuracy of GBLUP for multi-trait genomic prediction, using (co)variance estimates of SNP effects from Bayesian whole genome regression methods. Five generations (G1-G5, test populations) of genotype data were available by simulations based on data of 2,200 Danish Holstein cows (G0, reference population). Two correlated traits with heritabilities of 0.1 or 0.4, and a genetic correlation of 0.45 were generated. First, SNP effects and breeding values were estimated using BayesAS method, assuming (co)variance was the same for SNPs within a genome region, and different between regions. Region size was set as one SNP, 100 SNPs, a whole chromosome or whole genome. Second, posterior (co)variances of SNP effects were used to weight SNPs in construction of G matrices. In general, region size of 100 SNPs led to highest prediction accuracies using BayesAS, and wGBLUP outperformed GBLUP at this region size. Our results suggest that when genetic architectures of traits favor Bayesian methods, the accuracy of multi-trait GBLUP can be as high as the Bayesian method if SNPs are weighted by the Bayesian posterior (co)variances." />
<meta name="DC.Contributor" content="Emre Karaman" />
<meta name="DC.Contributor" content="Mogens S. Lund" />
<meta name="DC.Contributor" content="Mahlet T. Anche" />
<meta name="DC.Contributor" content="Luc Janss" />
<meta name="DC.Contributor" content="Guosheng Su" />
<meta name="article:published_time" content="2018-11-01" />
<meta name="article:section" content="Genomic Prediction" />
<meta name="citation_title" content="Genomic Prediction Using Multi-trait Weighted GBLUP Accounting for Heterogeneous Variances and Covariances Across the Genome" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Implicit assumption of common (co)variance for all loci in multi-trait Genomic Best Linear Unbiased Prediction (GBLUP) results in a genomic relationship matrix (&lt;b&gt;G&lt;/b&gt;) that is common to all traits. When this assumption is violated, Bayesian whole genome regression methods may be superior to GBLUP by accounting for unequal (co)variance for all loci or genome regions. This study aimed to develop a strategy to improve the accuracy of GBLUP for multi-trait genomic prediction, using (co)variance estimates of SNP effects from Bayesian whole genome regression methods. Five generations (G1-G5, test populations) of genotype data were available by simulations based on data of 2,200 Danish Holstein cows (G0, reference population). Two correlated traits with heritabilities of 0.1 or 0.4, and a genetic correlation of 0.45 were generated. First, SNP effects and breeding values were estimated using BayesAS method, assuming (co)variance was the same for SNPs within a genome region, and different between regions. Region size was set as one SNP, 100 SNPs, a whole chromosome or whole genome. Second, posterior (co)variances of SNP effects were used to weight SNPs in construction of &lt;b&gt;G&lt;/b&gt; matrices. In general, region size of 100 SNPs led to highest prediction accuracies using BayesAS, and wGBLUP outperformed GBLUP at this region size. Our results suggest that when genetic architectures of traits favor Bayesian methods, the accuracy of multi-trait GBLUP can be as high as the Bayesian method if SNPs are weighted by the Bayesian posterior (co)variances.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2018/11/01" />
<meta name="citation_mjid" content="ggg;8/11/3549" />
<meta name="citation_id" content="8/11/3549" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/8/11/3549" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/8/11/3549.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/8/11/3549.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/8/11/3549.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.118.200673" />
<meta name="citation_pmid" content="30194089" />
<meta name="citation_volume" content="8" />
<meta name="citation_issue" content="11" />
<meta name="citation_num_pages" content="10" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genomic Prediction" />
<meta name="citation_firstpage" content="3549" />
<meta name="citation_lastpage" content="3558" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Emre Karaman" />
<meta name="citation_author_institution" content="Center for Quantitative Genetics and Genomics, Aarhus University, 8830 Tjele, Denmark" />
<meta name="citation_author_email" content="emre@mbg.au.dk" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-1010-683X" />
<meta name="citation_author" content="Mogens S. Lund" />
<meta name="citation_author_institution" content="Center for Quantitative Genetics and Genomics, Aarhus University, 8830 Tjele, Denmark" />
<meta name="citation_author" content="Mahlet T. Anche" />
<meta name="citation_author_institution" content="Center for Quantitative Genetics and Genomics, Aarhus University, 8830 Tjele, Denmark" />
<meta name="citation_author" content="Luc Janss" />
<meta name="citation_author_institution" content="Center for Quantitative Genetics and Genomics, Aarhus University, 8830 Tjele, Denmark" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-9804-1529" />
<meta name="citation_author" content="Guosheng Su" />
<meta name="citation_author_institution" content="Center for Quantitative Genetics and Genomics, Aarhus University, 8830 Tjele, Denmark" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=I. Aguilar;citation_author=I. Misztal;citation_author=D. Johnson;citation_author=A. Legarra;citation_author=S. Tsuruta;citation_title=Hot topic: a unified approach to utilize phenotypic, full pedigree, and genomic information for genetic evaluation of Holstein final score.;citation_pages=743-752;citation_volume=93;citation_year=2010;citation_issue=2;citation_pmid=20105546;citation_doi=10.3168/jds.2009-2730" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=R. Brøndum;citation_author=G. Su;citation_author=M. Lund;citation_author=P. Bowman;citation_author=M. Goddard;citation_title=Genome position specific priors for genomic prediction.;citation_pages=543-543;citation_volume=13;citation_year=2012;citation_issue=1;citation_pmid=23050763;citation_doi=10.1186/1471-2164-13-543" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=M. Calus;citation_author=C. Schrooten;citation_author=R. Veerkamp;citation_title=Genomic prediction of breeding values using previously estimated snp variances.;citation_pages=52;citation_volume=46;citation_year=2014;citation_doi=10.1186/s12711-014-0052-x" />
<meta name="citation_reference" content="citation_journal_abbrev=J Anim Breed Genet;citation_author=M. Calus;citation_author=R. Veerkamp;citation_title=Accuracy of breeding values when using and ignoring the polygenic effect in genomic breeding value estimation with a marker density of one SNP per cM.;citation_pages=362-368;citation_volume=124;citation_year=2007;citation_issue=6;citation_pmid=18076473;citation_doi=10.1111/j.1439-0388.2007.00691.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=M. Calus;citation_author=R. Veerkamp;citation_title=Accuracy of multi-trait genomic selection using different methods.;citation_pages=26-26;citation_volume=43;citation_year=2011;citation_pmid=21729282;citation_doi=10.1186/1297-9686-43-26" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=H. Cheng;citation_author=K. Kizilkaya;citation_author=J. Zeng;citation_author=D. Garrick;citation_author=R. Fernando;citation_title=Genomic prediction from multiple-trait bayesian regression methods using mixture priors.;citation_volume=209;citation_year=2018;citation_pmid=29514861;citation_doi=10.1534/genetics.118.300650" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=O. F. Christensen;citation_author=M. S. Lund;citation_title=Genomic prediction when some animals are not genotyped.;citation_pages=2-2;citation_volume=42;citation_year=2010;citation_pmid=20105297;citation_doi=10.1186/1297-9686-42-2" />
<meta name="citation_reference" content="citation_journal_title=Anim. Prod. Sci.;citation_author=H. Daetwyler;citation_author=J. Hickey;citation_author=J. Henshall;citation_author=S. Dominik;citation_author=B. Gredler;citation_title=Accuracy of estimated genomic breeding values for wool and meat traits in a multi-breed sheep population.;citation_pages=1004-1010;citation_volume=50;citation_year=2010;citation_doi=10.1071/AN10096" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=H. Daetwyler;citation_author=R. Pong-Wong;citation_author=B. Villanueva;citation_author=J. Woolliams;citation_title=The impact of genetic architecture on genome-wide evaluation methods.;citation_volume=185;citation_year=2010;citation_pmid=20407128;citation_doi=10.1534/genetics.110.116855" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=G. de los Campos;citation_author=J. Hickey;citation_author=R. Pong-Wong;citation_author=H. Daetwyler;citation_author=M. Calus;citation_title=Whole-genome regression and prediction methods applied to plant and animal breeding.;citation_volume=193;citation_year=2013;citation_pmid=22745228;citation_doi=10.1534/genetics.112.143313" />
<meta name="citation_reference" content="citation_author=D. Dvorkin;citation_year=2012" />
<meta name="citation_reference" content="citation_author=R. Fernando;citation_author=D. Gianola;citation_pages=942;citation_year=2018" />
<meta name="citation_reference" content="citation_journal_title=J. Dairy Sci.;citation_author=H. Gao;citation_author=G. Su;citation_author=L. Janss;citation_author=Y. Zhang;citation_author=M. Lund;citation_title=Model comparison on genomic predictions using high-density markers for different groups of bulls in the nordic holstein population.;citation_pages=4678-4687;citation_volume=96;citation_year=2013;citation_pmid=23660137;citation_doi=10.3168/jds.2012-6406" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=G. Gebreyesus;citation_author=M. Lund;citation_author=B. Buitenhuis;citation_author=H. Bovenhuis;citation_author=N. Poulsen;citation_title=Modeling heterogeneous (co)variances from adjacent-snp groups improves genomic prediction for milk protein composition traits.;citation_pages=89;citation_volume=49;citation_year=2017;citation_doi=10.1186/s12711-017-0364-8" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. Gianola;citation_author=G. de los Campos;citation_author=W. Hill;citation_author=E. Manfredi;citation_author=R. Fernando;citation_title=Additive genetic variability and bayesian alphabet.;citation_volume=183;citation_year=2009;citation_pmid=19620397;citation_doi=10.1534/genetics.109.103952" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=M. Goddard;citation_author=B. Hayes;citation_title=Mapping genes for complex traits in domestic animals and their use in breeding programmes.;citation_pages=381-391;citation_volume=10;citation_year=2009;citation_issue=6;citation_pmid=19448663;citation_doi=10.1038/nrg2575" />
<meta name="citation_reference" content="citation_journal_title=BMC Genet.;citation_author=G. Guo;citation_author=F. Zhao;citation_author=Y. Wang;citation_author=Y. Zhang;citation_author=L. Du;citation_title=Comparison of single-trait and multiple-trait genomic prediction models.;citation_pages=30;citation_volume=15;citation_year=2014;citation_pmid=24593261;citation_doi=10.1186/1471-2156-15-30" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. Habier;citation_author=R. Fernando;citation_author=J. Dekkers;citation_title=The Impact of Genetic Relationship Information on Genome-Assisted Breeding Values;citation_pages=2389-2397;citation_volume=177;citation_year=2007;citation_issue=4;citation_pmid=18073436;citation_doi=10.1534/genetics.107.081190" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=D. Habier;citation_author=J. Tetens;citation_author=F.-R. Seefried;citation_author=P. Lichtner;citation_author=G. Thaller;citation_title=The impact of genetic relationship information on genomic breeding values in German Holstein cattle.;citation_pages=5-5;citation_volume=42;citation_year=2010;citation_pmid=20170500;citation_doi=10.1186/1297-9686-42-5" />
<meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics;citation_author=T. Hayashi;citation_author=H. Iwata;citation_title=A bayesian method and its variational approximation for prediction of genomic breeding values in multiple traits.;citation_pages=34;citation_volume=14;citation_year=2013;citation_doi=10.1186/1471-2105-14-34" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=B. Hayes;citation_author=P. Bowman;citation_author=A. Chamberlain;citation_author=M. Goddard;citation_title=Invited review: Genomic selection in dairy cattle: Progress and challenges;citation_pages=433-443;citation_volume=92;citation_year=2009;citation_issue=2;citation_pmid=19164653;citation_doi=10.3168/jds.2008-1646" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=B. Hayes;citation_author=M. Goddard;citation_title=The distribution of the effects of genes affecting quantitative traits in livestock.;citation_pages=209-229;citation_volume=33;citation_year=2001;citation_issue=3;citation_pmid=11403745;citation_doi=10.1051/gse:2001117" />
<meta name="citation_reference" content="citation_author=L. Janss;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Y. Jia;citation_author=J.-L. Jannink;citation_title=Multiple-trait genomic selection methods increase genetic value prediction accuracy.;citation_volume=192;citation_year=2012;citation_pmid=23086217;citation_doi=10.1534/genetics.112.144246" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=E. Karaman;citation_author=H. Cheng;citation_author=M. Firat;citation_author=D. Garrick;citation_author=R. Fernando;citation_title=An upper bound for accuracy of prediction using gblup.;citation_pages=e0161054;citation_volume=11;citation_year=2016;citation_doi=10.1371/journal.pone.0161054" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Legarra;citation_author=C. Robert-Granié;citation_author=E. Manfredi;citation_author=J.-M. Elsen;citation_title=Performance of genomic selection in mice.;citation_volume=180;citation_year=2008;citation_pmid=18757934;citation_doi=10.1534/genetics.108.088575" />
<meta name="citation_reference" content="citation_journal_title=BMC Genet.;citation_author=X. Li;citation_author=M. Lund;citation_author=L. Janss;citation_author=C. Wang;citation_author=X. Ding;citation_title=The patterns of genomic variances and covariances across genome for milk production traits between chinese and nordic holstein populations.;citation_pages=26;citation_volume=18;citation_year=2017;citation_doi=10.1186/s12863-017-0491-9" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=E. Lipkin;citation_author=A. Bagnato;citation_author=M. Soller;citation_title=Expected Effects on Protein Yield of Marker-Assisted Selection at Quantitative Trait Loci Affecting Milk Yield and Milk Protein Percentage;citation_pages=2857-2863;citation_volume=91;citation_year=2008;citation_issue=7;citation_pmid=18565943;citation_doi=10.3168/jds.2008-1011" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=Z. Liu;citation_author=F. Seefried;citation_author=F. Reinhardt;citation_author=S. Rensing;citation_author=G. Thaller;citation_title=Impacts of both reference population size and inclusion of a residual polygenic effect on the accuracy of genomic prediction.;citation_pages=19-19;citation_volume=43;citation_year=2011;citation_pmid=21586131;citation_doi=10.1186/1297-9686-43-19" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=R. Lorenzana;citation_author=R. Bernardo;citation_title=Accuracy of genotypic value predictions for marker-based selection in biparental plant populations.;citation_pages=151-161;citation_volume=120;citation_year=2009;citation_issue=1;citation_pmid=19841887;citation_doi=10.1007/s00122-009-1166-3" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=T. Meuwissen;citation_author=M. Goddard;citation_title=Accurate prediction of genetic values for complex traits by whole-genome resequencing.;citation_volume=185;citation_year=2010;citation_pmid=20308278;citation_doi=10.1534/genetics.110.116590" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. Meuwissen;citation_author=B. Hayes;citation_author=M. Goddard;citation_title=Prediction of Total Genetic Value Using Genome-Wide Dense Marker Maps;citation_pages=1819-1829;citation_volume=157;citation_year=2001;citation_issue=4;citation_pmid=11290733" />
<meta name="citation_reference" content="citation_journal_title=Front. Genet.;citation_author=G. Morota;citation_author=D. Gianola;citation_title=Kernel-based whole-genome prediction of complex traits: a review.;citation_pages=1-13;citation_volume=5;citation_year=2014;citation_pmid=24567736;citation_doi=10.3389/fgene.2014.00001" />
<meta name="citation_reference" content="citation_journal_title=Journal of Animal Science;citation_journal_abbrev=J ANIM SCI;citation_author=A. Nejati-Javaremi;citation_author=C. Smith;citation_author=J. Gibson;citation_title=Effect of total allelic relationship on accuracy of evaluation and response to selection;citation_pages=1738-1745;citation_volume=75;citation_year=1997;citation_issue=7;citation_pmid=9222829;citation_doi=10.2527/1997.7571738x" />
<meta name="citation_reference" content="R Core Team, 2017 R: A language and environment for statistical computing. R Foundation for Statistical Computing https://www.R-project.org/." />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=L. Sørensen;citation_author=L. Janss;citation_author=P. Madsen;citation_author=T. Mark;citation_author=M. Lund;citation_title=Estimation of (co)variances for genomic regions of flexible sizes: application to complex infectious udder diseases in dairy cattle.;citation_pages=18-18;citation_volume=44;citation_year=2012;citation_pmid=22640006;citation_doi=10.1186/1297-9686-44-18" />
<meta name="citation_reference" content="citation_journal_title=J. Dairy Sci.;citation_author=G. Su;citation_author=O. Christensen;citation_author=L. Janss;citation_author=M. Lund;citation_title=Comparison of genomic predictions using genomic relationship matrices built with different weighting factors to account for locus-specific variances.;citation_pages=6547-6559;citation_volume=97;citation_year=2014;citation_pmid=25129495;citation_doi=10.3168/jds.2014-8210" />
<meta name="citation_reference" content="citation_journal_title=Animal;citation_author=G. Su;citation_author=P. Ma;citation_author=U. Nielsen;citation_author=G. Aamand;citation_author=G. Wiggans;citation_title=Sharing reference data and including cows in the reference population improve genomic predictions in danish jersey.;citation_pages=1067-1075;citation_volume=10;citation_year=2016;citation_doi=10.1017/S1751731115001792" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=F. Tiezzi;citation_author=C. Maltecca;citation_title=Accounting for trait architecture in genomic predictions of us holstein cattle using a weighted realized relationship matrix.;citation_pages=24;citation_volume=47;citation_year=2015;citation_doi=10.1186/s12711-015-0100-1" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=P. VanRaden;citation_title=Efficient Methods to Compute Genomic Predictions;citation_pages=4414-4423;citation_volume=91;citation_year=2008;citation_issue=11;citation_pmid=18946147;citation_doi=10.3168/jds.2007-0980" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=P. VanRaden;citation_author=C. Van Tassell;citation_author=G. Wiggans;citation_author=T. Sonstegard;citation_author=R. Schnabel;citation_title=Invited Review: Reliability of genomic predictions for North American Holstein bulls;citation_pages=16-24;citation_volume=92;citation_year=2009;citation_issue=1;citation_pmid=19109259;citation_doi=10.3168/jds.2008-1514" />
<meta name="citation_reference" content="citation_journal_title=J. Anim. Breed. Genet.;citation_author=R. Veroneze;citation_author=P. Lopes;citation_author=M. Lopes;citation_author=A. Hidalgo;citation_author=S. Guimarães;citation_title=Accounting for genetic architecture in single- and multi-population genomic prediction using weights from genomewide association studies in pigs.;citation_pages=187-196;citation_volume=133;citation_year=2016;citation_doi=10.1111/jbg.12202" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=Z. Weng;citation_author=A. Wolc;citation_author=X. Shen;citation_author=R. Fernando;citation_author=J. Dekkers;citation_title=Effects of number of training generations on genomic prediction for various traits in a layer chicken population.;citation_pages=22;citation_volume=48;citation_year=2016;citation_pmid=26992471;citation_doi=10.1186/s12711-016-0198-9" />
<meta name="citation_reference" content="citation_author=Y. C. J. Wientjes;citation_author=R. F. Veerkamp;citation_author=M. P. L. Calus;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=A. Wolc;citation_author=J. Arango;citation_author=P. Settar;citation_author=J. Fulton;citation_author=N. O’Sullivan;citation_title=Persistence of accuracy of genomic estimated breeding values over generations in layer chickens.;citation_pages=23-23;citation_volume=43;citation_year=2011;citation_pmid=21693035;citation_doi=10.1186/1297-9686-43-23" />
<meta name="citation_reference" content="citation_journal_title=Poult. Sci.;citation_author=A. Wolc;citation_author=J. Arango;citation_author=P. Settar;citation_author=J. Fulton;citation_author=N. O’Sullivan;citation_title=Analysis of egg production in layer chickens using a random regression model with genomic relationships.;citation_volume=92;citation_year=2013;citation_pmid=23687143;citation_doi=10.3382/ps.2012-02882" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=E. Yamamoto;citation_author=H. Matsunaga;citation_author=A. Onogi;citation_author=H. Kajiya-Kanegae;citation_author=M. Minamikawa;citation_title=A simulation-based breeding design that uses whole-genome prediction in tomato.;citation_pages=19454;citation_volume=6;citation_year=2016;citation_doi=10.1038/srep19454" />
<meta name="citation_reference" content="citation_journal_title=Front. Genet.;citation_author=X. Zhang;citation_author=D. Lourenco;citation_author=I. Aguilar;citation_author=A. Legarra;citation_author=I. Misztal;citation_title=Weighting strategies for single-step genomic blup: An iterative approach for accurate calculation of gebv and gwas.;citation_pages=151;citation_volume=7;citation_year=2016;citation_doi=10.3389/fgene.2016.00151" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=Z. Zhang;citation_author=J. Liu;citation_author=X. Ding;citation_author=P. Bijma;citation_author=D.-J. de Koning;citation_title=Best linear unbiased prediction of genomic breeding values using a trait-specific marker-derived relationship matrix.;citation_pages=e12648;citation_volume=5;citation_year=2010;citation_pmid=20844593;citation_doi=10.1371/journal.pone.0012648" />
<meta name="citation_reference" content="citation_journal_title=J. Anim. Breed. Genet.;citation_author=L. Zhou;citation_author=M. Lund;citation_author=Y. Wang;citation_author=G. Su;citation_title=Genomic predictions across nordic holstein and nordic red using the genomic best linear unbiased prediction model with different genomic relationship matrices.;citation_pages=249-257;citation_volume=131;citation_year=2014;citation_pmid=24750283;citation_doi=10.1111/jbg.12089" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Genomic Prediction Using Multi-trait Weighted GBLUP Accounting for Heterogeneous Variances and Covariances Across the Genome" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11.cover-source.jpg" />
<meta name="twitter:description" content="Implicit assumption of common (co)variance for all loci in multi-trait Genomic Best Linear Unbiased Prediction (GBLUP) results in a genomic relationship matrix ( G ) that is common to all traits. When this assumption is violated, Bayesian whole genome regression methods may be superior to GBLUP by accounting for unequal (co)variance for all loci or genome regions. This study aimed to develop a strategy to improve the accuracy of GBLUP for multi-trait genomic prediction, using (co)variance estimates of SNP effects from Bayesian whole genome regression methods. Five generations (G1-G5, test populations) of genotype data were available by simulations based on data of 2,200 Danish Holstein cows (G0, reference population). Two correlated traits with heritabilities of 0.1 or 0.4, and a genetic correlation of 0.45 were generated. First, SNP effects and breeding values were estimated using BayesAS method, assuming (co)variance was the same for SNPs within a genome region, and different between regions. Region size was set as one SNP, 100 SNPs, a whole chromosome or whole genome. Second, posterior (co)variances of SNP effects were used to weight SNPs in construction of G matrices. In general, region size of 100 SNPs led to highest prediction accuracies using BayesAS, and wGBLUP outperformed GBLUP at this region size. Our results suggest that when genetic architectures of traits favor Bayesian methods, the accuracy of multi-trait GBLUP can be as high as the Bayesian method if SNPs are weighted by the Bayesian posterior (co)variances." />
<meta name="og-title" property="og:title" content="Genomic Prediction Using Multi-trait Weighted GBLUP Accounting for Heterogeneous Variances and Covariances Across the Genome" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/8/11/3549" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Implicit assumption of common (co)variance for all loci in multi-trait Genomic Best Linear Unbiased Prediction (GBLUP) results in a genomic relationship matrix ( G ) that is common to all traits. When this assumption is violated, Bayesian whole genome regression methods may be superior to GBLUP by accounting for unequal (co)variance for all loci or genome regions. This study aimed to develop a strategy to improve the accuracy of GBLUP for multi-trait genomic prediction, using (co)variance estimates of SNP effects from Bayesian whole genome regression methods. Five generations (G1-G5, test populations) of genotype data were available by simulations based on data of 2,200 Danish Holstein cows (G0, reference population). Two correlated traits with heritabilities of 0.1 or 0.4, and a genetic correlation of 0.45 were generated. First, SNP effects and breeding values were estimated using BayesAS method, assuming (co)variance was the same for SNPs within a genome region, and different between regions. Region size was set as one SNP, 100 SNPs, a whole chromosome or whole genome. Second, posterior (co)variances of SNP effects were used to weight SNPs in construction of G matrices. In general, region size of 100 SNPs led to highest prediction accuracies using BayesAS, and wGBLUP outperformed GBLUP at this region size. Our results suggest that when genetic architectures of traits favor Bayesian methods, the accuracy of multi-trait GBLUP can be as high as the Bayesian method if SNPs are weighted by the Bayesian posterior (co)variances." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11.cover-source.jpg" />
    <title>Genomic Prediction Using Multi-trait Weighted GBLUP Accounting for Heterogeneous Variances and Covariances Across the Genome | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"RGdz_KUD0IGBhfjKG62KNeZ_ARMkHWMetOh7Z59TUw8","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"491248","apath":"\/ggg\/8\/11\/3549.atom","pisa":"ggg;8\/11\/3549","ac":{"\/ggg\/8\/11\/3549.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/8\/11\/3549.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;8\/11\/3549"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;8\/11\/3549"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/8\/11\/3549":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022bmed2fMWrlEIiXbIr307Bsg9jr8zNd78m5r1SS6dXWPY9n2yr8Dwnxd2pnAEVSx6DbaYqPTdTDS8E77pruWyJyp2Rz8WMJJvcHtrqY296MueKA684+OwzWEPFoEKk61czWqCHpJL1Qyahe0ylkKH8HT81nLil1WdptwABBZoUK\\\\\\\/kasdytjNkw3z8N3diNtOzNRJY+6OQoUM+xyivMaYPYPpEYqld705c9IOQuDAx0N8duZ8lpVR8v5\\\\\\\/uVIR9WdWGxi6WSUbShwJR\\\\\\\/B6ETkEg3fCDUOBtfehadZ8J5MtUA379H2foUsV\\\\\\\/wunxsDDA1VFEdde8rQJjYxkgn9JzE3PHaQiZcCQrRoTjU9zfCJN97bjYv+VuAtqN+9BFAsAMlibBL3Q2zKbBaG68Qyo7TrWGRspX2D9XwLj5ROzYrqCouNSct065y0IgkteRfC7uYKdG2UEbfVXYMO313lGuz7KUgO0hFA3Jj0RmZDC424NV\\\\\\\/i9UWCeFsCWDhsFPH3sfyJE9nl3MzYtxPHS9\\\\\\\/PxzBzcy\\\\\\\/x\\\\\\\/H2NuSMweZIfsJoeStK5JOfGitXZ7tbp+mVZtZvfDYtV\\\\\\\/AAsWLYbiL9L3qYMTir7k7uYi7\\\\\\\/D6nEhjz0OpVNMYkrTgRVqc0iVmZpBt\\\\\\\/mcs9Mx9ZiT+e5VNGtBT5yS6y7qbPPLcwrfdPsuTVtcdLTD9r4WJzgJs94X\\\\\\\/vcI84dpS3FAY5QkT4bcOSrRU8UaZvGqgR0P61BTKtUp\\\\\\\/c0nTlrZNNhVVo\\\\\\\/G9EdXtrs78iH9HKkRPvEueAK8zt82Z49JHpiBnQvWR8R1kHJJc0S+jY1qBUkwFrHNaMeecCPV3F2464tEU0YyeIHQKuUaYtrtd1YE\\\\\\\/lRPv4z1xCvUKPOms3LqH22lCEWT\\\\\\\/YDZS7dex8ndSMn6WEzsFRYQcCr32BX5+pfAaWSNDoPJQxzJFlxn\\\\\\\/oReWMlPrODf\\\\\\\/suDNMNIZRogR2PG4wdicxUARMn4xcNQiSkp\\\\\\\/+rn3H3vSJXUYNn3bB4nHCMcvKyz2\\\\\\\/jX+QGYESdPkoge9yyMWHULygRmnNZqdbv4QDfMFCHYO0o5fVVamyrg83EbTkSBRfJUiMw20maRNZzo5kFklsNoogcXGkw8bslyLV25ipkQLmftI\\\\\\\/wRhpktjiFrbbkdXWRGLBwIKqP4RKaQHJJmq6FdDneDwQvbGgx6dNF61BXxVk1NtTikmXwqEdQnIBMTnKJ0FUoqywxHebYW03OCDnv\\\\\\\/Mg9Qb6YVfDP3XuTVnlK3LhXduQY7MmxCUJCl4K4Z13SggPA3vulwkz\\\u0022,\\\u0022iv\\\u0022:\\\u00221CadNW1vgXaZJv99AZzCsA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022d62de23fb71bd765baab6620b25a9aba03cce280588e208344426b484d979dd5\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Qi6KGinMyT\\\\\\\/i0\\\\\\\/Y7TtIb6n5eI\\\\\\\/EeBEsM68dkEszFu5+cmR9JHbowIo8Zj69lDrg6iirBFBuCXMEpWAnbkc9+AfI3eKGDqGo5FULwgqW+2Fq1G3L\\\\\\\/+6w\\\\\\\/MB3DsZo6Mnl0iCAOBqBTt\\\\\\\/UvL6xCsgxWwRQcMQKrmPIEFj3kpjAo8VdV0PPAtR0ExpMaPQiPQ+I\\\\\\\/NYKgTnU7Qg6b4VZM8QgEe6OPcfSZTNbPyraVD5MjBIzyZhx1jGbSarKia7PuP2XwfZ+IiNyXleQo+THcK0IZOIMMjpkqs9hPxtiQkmc6R1WJ3uo6I65bNfQzx1wfQtfbiax6i7n6Vlruu0kgAP+g9hhwKmLtmPxLkCfR1aBfolCqRaQkTL7F7NF\\\\\\\/3DYQBrR\\\\\\\/++raFTEjVAoc6gh+ywyoSEEHQYMtczc+XLg3UDE72XmgGGDTLEhD3xa3RCHoBMENiqFHoQrLUtPe9+6zHnxhy+9uUVl3HJe0KJBhqW4TlOi0QudZ1rMOvYTRvv271k0yk4WRYHLw+1BtiLeMOTJt9JK7rFnPZmjRA6kOrkFh9B67zicb1b126PxGe0hbgNKaq\\\\\\\/Ge7WfTaBosu\\\\\\\/RR5zQIf3blfk\\\\\\\/aDdkYI2nCLPYcEmq9jmf47hbN\\\\\\\/mFdItwFUgYdWJYAdsTyM\\\\\\\/VjlYscyizbGj2Il391df42VcXMTcMntMQnYD7mQOTGr9EAnnIQwWkGGTVP6Up9odwZHWfgDAGLrBgLoyNeKzn8tkiot8m8ZNazste2Qo6Fo6WMMLmVo+GfOA\\\\\\\/U3XGvAusk++y7PuLrUBFFuTKIID5WBt6senR6XselBmr4hsnu0WkS0vjgJkVBNQ7uGN7SGkMcEhccvMZS3zt22s+rFhaxsZWjoDCgjhi37nhGWpApYyDHgIUWzszVSdfETNl094orEEieOIP0rvpiQN8nqTJerfWYnWEp1XmUYxGBTLKYxbvxypcGwwiYqzvXNWFFYrUiSWXcblxIj6cGaLJNacowHH2crJQVHE6Mm8jVAat4Fd58QX00mHsKr5wJgRcqV8ioZARMlFDAhyyBCP5z38NS\\\\\\\/6xRwB7eeIujlRzw82gvXnLMIIToUcKfTBLqPwuLM7o4SwuoYmbEpBgKNYoVWNpFBxD4HWcqyUu6yYGYvCW2PWcdyr0QA4rmqL5yITw5fpuwoarAUCgxICZ3VXOmekZz7igABkYaQs6Dgi0MBgPU7tEvlIIISa\\\\\\\/buTfaBXgeV72uM9g26ccEfYCbjn9Wn3FgVPZtRlzGzg4uKEyhMmubW5KZipnSV3fhJ4fgELRFPa1\\\\\\\/i4ps7MLNKPOj8+HF1PjPrb4qicyi810ufsZod0aTb0XExspmp9cBMurh6x8\\\\\\\/UaqczKnFYQIKzaB\\\\\\\/YEo1VU+VZ8blDXLYihPiPsD7byW\\\\\\\/UzIo4RofLRTehuzYf\\\\\\\/RbpKMzjMPCbhnRjXiikc6WDmtAw2+4Ec\\\\\\\/VITBq\\\\\\\/uUW2L55AU+WnKqKClFoISepyHvw\\\\\\\/fiO3C4PyDMkMjKUL0OZzMIPrYtpcIX1v\\\\\\\/oLU8A0KATv+\\\\\\\/cTY2ucbNaWHCSgMcXXh5JtuDsK0O4nHGLhms0Gg7vaY92RVqoC108LSL2EuIR6B+tCz8Q2NqeDN1EHoGeLQ4DWH2nnUZzKU695Iq5w\\\\\\\/rrLuLDTbIywrPAsmWt1I4oqJANqAaZ00OYHhStZQWtSoV7gz227c+KGpi4xzBjoiNOm3f5maJ2IidBXMp5IHKhx7sajDzE+LbGHSpj6iKHWuFRmcRRTmDALWdxDt\\\\\\\/\\\\\\\/ZAs7Z5UsY0WaQjx3mrLpEznlEF47ndrJ1cMIdVEVZIqdzr57pwZdrf+MgPxyziCJMKb1ry7+sfrLY\\\\\\\/uG+k0nwfVYUQ6H1GTOy4OexUroNpO5SQlJee56cNV45Zgcv1EdS6h0QFvljpfJ+YQfJWPM5hlgVcfw4uxja6QSWnaJhstE0x2dN1CffjBPK35D2InayTRmdWTk+DNtXstBDJ0bEuZPl9TrnE6U4GtmXDponkjgJOvr\\\\\\\/6lbQF+YFDqjbsR1ZEeKsAq7Ok3KT4JDecB5m23Qxf5RQZxA3jn75OIeFvsNuxxFlhf6yxvXmGpAJIF1xfZFEhlVHptcia5VzU1g60tXGo0AJqJdC9IUIp0KFBIgQs2nYrtmlzN8lItP6CPy9OTsLABuN4c9fV\\\\\\\/PjO4Dw1Z2DAFka9rM3KC4N3wTvM5npMCRz1PWXb8dhbbYXjFmMsTSB1sADPyPcJcWQlmH1iS2dzv9mLsRrSVBLjPTtGvnzndz+WZmxRQJS9Ypm0Up1fDUpvUm3Od1a71bONbBgokA3+NFYT+JarPfeGuSrxAfyWPx1Ef8Ok9m0Bc6SlKdRmNJEOSgt9FttrnEnOoFUTsTk5UZ+fwH10+esBYmV1o23TF5Fe052zKyRZUEfmSbPRGYX6BuZBrnAz\\\\\\\/J1baUuds6Omt83\\\\\\\/wqsNQNwzXcF3cfQovs0XxmQO5MJvZkCab0KkymuQ0rfpY9dyDfPQ9nquokx8NwtReEpyeC+0n8NacPwQuPl\\\\\\\/rCveCID2\\\\\\\/LwG2LBjGcpr5XERpS9RaYoCEq3eq6vtOk+Au7f6B9yMf9Ms0r+P2nLrGvdDgr+c401OcufMCaa5ObEopPbfrhXBLpr48G44+6W9L1V8TEahP2pYfhdlhvAQMIFb9t\\\\\\\/6xl33WC3gZ0yCa+\\\\\\\/wwcXtPAsRAD9EN9RKd2DGj5XeyC\\\\\\\/6OjW+m8Hw8kOFRMcYE+32q4z+1GN6tjqvKlJjbKTzozpCZkV\\\\\\\/pQg+vLe14SeHCfI\\\\\\\/5CEFxGT2\\\\\\\/jbrZjzHw5fEbupCn79Kb\\\\\\\/FGStpVC48VdH7zGZQ66e7VC0G\\\\\\\/8LitEm4XZoMTTo8pHgwStIP7JnKjpSeVWFKetX7vc7nztl80R6XkYAPIBdVrVSFCrw7ZNsmsxzS33IaCGZpytNX4jd0sdVjWzNg7WnW8owBOSrb31IFbYIFqKHAELW7H5SE17PhRFKe+jl6O+8NilvVMZv9j5HJErRm5kDHxC\\\\\\\/s1yFt+EJJOIl35LBv3Tz3jQ46xdzpQdA0i1OQ8WKalM2SfqZFYHCQJO88R\\\\\\\/YcnvvTxfIpE9VHMQD9bqDCj4+7Zsx\\\\\\\/1bCsb7QIHcfiLgLf6lohQ+sirizSJ+jtaocP3621b0j9Uz8eJOrXyg1dJc8tkG9oG7GDtBUcIUbk1ETRW2jgmjLHWBPA2CO76tnD0pzhvK5HMX6uPEJVT5mlud9KqcC0iWaaLJNL7+xBUOct4TZKA8GLTakgc4SPs0FAIB3bCV7E3DZSE96kValaUfUyetslhPQdOzD2df4OIT0FctvG8YBG7szSKSpPTV2BvCM\\\\\\\/jhc5ukYIgv4dJ6TGnKrPHyKYqRIx\\\\\\\/4Oe8lTlqYy2amK7gVuUE4nRwvwCyg\\\\\\\/fkaJJZyNhg3kxwnGiHqfWfMwkicCod7qoDV+rrx2n485vNUg94fhwd9PTHwpb5nE8y6WAUnuMTqY\\\\\\\/38ioiNNrumder4CMaILELz\\\\\\\/MJJBUFDSDbFy6Ns+fQNpqnMDCLbGnM4uETMC\\\\\\\/VOG6wHUsBbtf4sQXIakAJulqixejjnwrU9l27BWJvm8mvFRb+WT93V804pUCx+WX4Z+k\\\\\\\/H6Ipf3dgYwHjmScN74qnS7jE3jFUnj2I\\\\\\\/Pj3ZmMnQMBAbPTcVPK+mtPsWHsyAocUHTrR\\\\\\\/jxpPmQxXfOFZDiCdOyADGJxGdZUP6T6yOXnLpqoh4uhZa\\\\\\\/2OZDKPQpHNk3R+EYq6EZE29tx\\\\\\\/mTbYntQGhdpioadBMRnsQ2FGVesg41gumcZl\\\\\\\/12o5FyxE34aNpGKmKTu26gQ6nGymOXr92QPemb4\\\\\\\/uIssLM7BVx34qy3OUsGVz52Se6iLwHaJS72VkBESBcaTeT0k5InirQbPMX2qZClWqDjo76mb64W8PbO7RMzH2+5Nd83uYo\\\\\\\/psB1Viv\\\\\\\/eBfr2bx0ZyqZBu0WELnTlw\\\\\\\/KP8+BJCgmoajnbsJS5qUKRP3r3hxCn0yFaG18nXg0ix3ZFPnYn5\\\\\\\/5mKmr93DnqjoAWxejXPUW2c32I\\\\\\\/QkCDPvEXtSpTJWe7JwFBUfTwdWG\\\\\\\/fowVZd\\\\\\\/tpEayxcVkHGOmCJAH+r991t3gwrE6hr7ovMcSLvV5heflwq+aGRTW\\\\\\\/5t1PAqpmEUQzvJ+jippYiC4OgHbZ1K9SiNC\\\\\\\/E4YZN2FhfRCQxSBbFVmZKb0PGaE7di1wo8X6cbUdqAV0hLGnMDKCuGjVCcM5CNdqXlLN+4J06Gec\\\\\\\/RPFkYRHSf\\\\\\\/0IlUML84R9cG\\\\\\\/KhFAK2VVxDYbNDqRn3CwxStRO0PvgEuEMilFYOnvZWhpYAzTE8azaFBAZQg\\\\\\\/LecNdqbZRDB94YDW5zd6n4JT9cFD9X1A1K4PjZZFLyPK0\\\\\\\/6wJEjW4Tz25VgDK8NN12o\\\\\\\/cbsZL4WUiiodnIeaDk\\\\\\\/xKZlsnb+CzyqiE0LF6+J9neo6ow3U1lzRlGni1hp50\\\\\\\/UJN0kxpGZUxw1r7rZ3yJs1oyA6xwRBe+Ugdi0DmfuiYluJszcOs2P+7wgyszR46d3nNX4b+yGGOlL7QrgkAb2xFoQJRu4aHD2CGseLWPToPuewC24D0PPDy8E2PZRehaaKlbAb\\\\\\\/qvXXC7mDZXQqRiVYL+t7CrvX6m8G8wcYDZoXr9x3acOQwTQ2NQW6N9g8qPGBEjMPeCBjyzXgjRxh\\\\\\\/GOBdwDxbHcviXmVU22NXSet2aopiKQUw0\\\\\\\/YudYmQs9LyCpKWNBSjE8mzmQwZrwD6Lt9MJqo0qVLqWxZBXE+RLVnsyLQ7jig6SSYXbVaMffWFacEm3\\\\\\\/kY\\\\\\\/qCpeWY1wjwFHCi1JIU8KIT+eZznmuaNZYv329KYOewwQHSkOjvdlbc\\\\\\\/3VLriqW003wrrV0PF4C5lst6LZ4bcnTRkVYw\\\\\\\/ERqAL55bmRbvVDSBJoyF9CR4DxbrYvdz4Vi\\\\\\\/kxo67FOM+BvGQjZaulzRo6T9tTmB0YEDF+9kArVbWePWclR\\\\\\\/dpxS\\\\\\\/Mzw6aeP5gr7vfKO1NfyMsiZwAQCqBAZupm7\\\\\\\/mZFr9WQA2QmTZxtOlSjyjwRwwyKykL3yn6ikQDRkTi73lCSfRM2pmGHnkn9dmSBoRK5lSSGqSm2REF2r\\\\\\\/TKPC37lH50OU3hkKvQxXIl3+oa9cepvVJXFwlLLIGWGBJJn6MyWN62vuN+J4mhNnNzrYgABzwHBm1qPIef4BQgmvCc1X9P5UW9T7g4\\\\\\\/NzNmhvYqhC+k2KpZly8DS9CGg+oT+zalRKOaC9ALK7da9OGxtZ2pXN\\\\\\\/wTVgQa0wPRt3qEeulXITU9UXY+lNqlfddWNPuB2hYFkjxNU0Sf3V5DHMdZoyRmfUdjRN0NmaPL+Pj2pi2VEIi6VfYjs6ff3xm1wpJKCas8i3I9RXI6QpInJHr7fVscPHnliPxXxak1RgYjZD6UFMykUbLCw55luNpAiIzgYUMtBZlP2AObXmaSvo4Gkh5ZbNcXXWbMCSH47wQ4su2iNFTNL9XYzRbqm1029HUy6HS+8hNwh6MHFE7fF5ocwvwU+elBX2orRSYV0A+P4I8VXCtABtF9LIlEXTxBIRA1ExKFKtiLxddt+Nj\\\\\\\/6F6z6QxP7QIUALSh6QJw00ct8jEf9yXWH5HIX8eyOySSorrUaWJGKFCctZt5g6GCvVLUA7cFPBe4bbqN+\\\\\\\/xSFjomKpmCa8SEvAcKhiS5oQRNylHu2bHq8QeYdMbhsmofS5sdFEU4eHraxJBgnl9ksDZ6SLKQMh2FWLNnR1TCjHuDUvvxCloxda\\\\\\\/kqg2l57fWNoXS6CRG80jf3dcGvcU1LXiAaonn\\\\\\\/UybX61CL0w5mdXzJ2CrJZHUSiSz2UNtew0JBU5FZDMk2F+hbIOz3ocf5L3cmzVUkcbWi09pm1EdOwZPvq9xtj9jTG2iVZRTIv97+AJni4UJA6wgEFnMmgw4F0zWHQnHpl+it+py0sCTRkG+m+BJQV1WEZbG\\\\\\\/E3\\\\\\\/ePTEVw8XIsYRYneFqylF6IXHhDckjaTJy8IfuSWaOZV22wgX9UZ05cqF81rQkroFAQWwjs4PkSgiFNr2pxnS+rgGZlgIxGVEoDRbeEFYGBYQbrbP5uw6CpkVRGnbsrZ\\\\\\\/rig8+J+2GfQvrNgYDAHzSrSc297vUIrsmw3JrHt0FOck7m6YSHF\\\\\\\/3YRJStuoz1YmUWUTq+AvjoxcR9qcksdzStxr5Am\\\\\\\/e7bmsSdCnxKy5V\\\\\\\/stYObVNZMU42P0WWSVJrZrISWG2DJTXN+MvsBSwgiccn5S9u+WETEMRbTCDvhNfdjSYfACVmJoqRmmTqNuh1X9YY1V9GPh5jhbxYAvXjCHMbi4yaBg0pp2BDur9TVkVZ4sR+q13GgfaSqayxTNKb6entawmYKbvTi0mWOkIPnsZdZT6D2GdJV1Q7yHwu\\\\\\\/tFRZTx43KSDTLWoYmjRHGO9orPIrWfptnZsqMLgXn4ExojRdEWqMWZGrqLQWeGPpOxPXTZudGAxTy7Gyf0aqdSKM9jdw1n9LJrP8E11BO7PGcJ3ztbcRue\\\\\\\/r7Ua9sQ4DW61vrT8FGHwgGLNyu7nhNn9ZOkB0fAovtqRRzTkYjit6qAi\\\\\\\/+sfGJKMAXnpOYx2GMc6YH\\\\\\\/IrHmm7ae+kGa9GwEbJtAUNLY1YBhDt3WfjArqIDIR74w8Anju3Cls1sxG3Pz19YY8ct+uZi7w88hURLKT7D5y9B616yvP6e8K8Eft0lreyqPPD4b\\\\\\\/lqr4KM9GaD6exZiclL9IdEygZxyb9Z4z6lon4\\\\\\\/bnRDjs5nxUG3hJenHtqk7iqpdf9afPagHAjl3qkbFlp9YegQ3tiKkiHhwnH+umRcXWqjQN7gga8tZ+UblPy15\\\\\\\/pS8Hb6hv6WWCbuTyuYnlYI3j3Zonwv1ZQi4VhBclTVwMVG86BtoGTbn4jceanirkFWOr9EOktdHRwolurB9i8vaNA3z4FKmitukOnozXZ\\\\\\\/KPcj8NW6nK\\\\\\\/SM\\\\\\\/5UTqgt9wKo2lD963iWmC6W8oqNxVEa+kGdMysvcGM0SC4cHkfn518jHQj0H+5Ay2g+bEdEUNThQgpe87cOcquoneWF7KL8ja8o3px+Zs8vOAVA0GruM0+vGisQCRhobqnqIXCXqjQ69i0xF7xRwUYQ81aBw1bGNGFicscAGZ1fi+xutA9s7PqW\\\\\\\/DoRKiAvF5Lepd4YkrvZgylrjrstymZ4D6nK4J3jUc\\\\\\\/6c2UrL9KrZ6gxpEskzs5TjqPhoqlahKRlg+Ejd6Gv3hJZy+c9NZmAeMHqY3AmOFlOPx\\\\\\\/Uln8vP0ZDzwDB9qIRqbIeBVdOq95u9BJuhSqi7RqfOPZHpM\\\\\\\/uO3iM6U5fprzKW58GygH2bzQyPUrkjyZtiHF6hgdgfpdiTF3r05PjP+BnymFjD2S8fYAJeMZuC87sKiLk9lbk8z5cK0LjpIHlmdnp0U2hm+NV8szkULoEw4zxZaxZb98bfGZjhOlRzfEV4U4N\\\\\\\/UcX5EEDYbzFBLEbzLyxJ82Yp+ffViwl9WaP0V7ezpJa5+LPq30tLU8+uR2ZQDzjLReeklPfJoSf+c6N\\\\\\\/FaqNSQykmW9Kq05BJN37kcWdfernjvpml8hwGduwj5\\\\\\\/fxtZm5HAMRPSnhi4RdG9E4hxhfJ+iXaZe2pehVE0HEZcievdEAo5+6i83Ne8ZZ63x2WEke4j8lV8mJk8cX1SOhAhHemwrPHZ\\\\\\\/5r2ky+YsLqu2x281PIbLpIjpnc0qOClDIzvIF3p5Bqp+nmWzuoQD2KeUxk11HutNEKyFPdjtiuZgj2jMSUmNCJCFS3lA2LIcIvApJkt9RUBoPST6P+mKdWNtrN9sQ3hkYJi3G5HQbmWZDLLHvVaImgN8YOBViEIGLszvUKUp85A4Yp2jMwCwSvs2lEk1zQy800vKE\\\\\\\/WJjHF7bckjxtRmeDgiiGbSCbYXZ08E\\\\\\\/27eObwmYazpzFVvDBvhkKVmD5eX+N8r\\\\\\\/AvzSpvR4PXZjslJLkyk0SzN5VJh+Z\\\\\\\/jlsflOCnZCMamJ8Ko1bgKaZXWWlwfSR25xvynZQSNmkDa7kaJOm8SMB3CZyUWKYonEF\\\\\\\/dzvMXfxoMbVp3KkrJE9H7wfn5LL7HsMHbe1qbs2WCKh6v8PkP9ISQ78H1Vh6A9ToWhpkbd+BvSAsJWf1OJ5ubB9R0q+33by7Nu+RgUHvpixzxYjWkevcmoLlTNmbKR0cLbPCVUfR4laCd3\\\\\\\/6wB3sn94kPoRpNyrUH7ebFE+NB+almEQ9yqtcnnA\\\\\\\/vuEzZ7puIpOhJKcHENfOqKeEB3Y3oTRE53IXkizJwXdHuA6lZ3Uh2GW2o4WxIeOCId1jXu0tznR8+E\\\\\\\/cSbcDJ8e0VdkOAcY6+Q9BM5gPap55UD04iPFuXuiqcMT5W87l+4o9YLbdBX3MI51NY8nNNDUL9wpnGxBsohHJE+cSKxMg4rkK\\\\\\\/UENYZLBrroAnygZf+C43XWOXTj\\\\\\\/ShQ3Sh0Ck6QLrDIDW1\\\\\\\/GqSBrn9rla\\\\\\\/AdtnK7tgXgxSji2sA3zYcWRw79M2FUyqTf\\\\\\\/P9Flt9hi\\\\\\\/jltQlujejWJa6F9dxHAaR8OE5nd4Oh86oh7wuhY1TejJ5cVOHoh8QyLTsoiLUjvkkGmHb38YN2VV0HlrnTy7efI\\\\\\\/hyAlEd\\\\\\\/wrnU6OEgbYi46\\\\\\\/YBV02Z6fsYr2p4VWBd8PkUKxQHKCJhH61pFCRP0m8PE667CyufFAiM4EW\\\\\\\/IZy7G5wueN63Vd5NJgATTzjLpDOZxRJIpNVM8kbu57S6ZLjOG84fRrdMghxMwhJ1P4m4BsAtmCbUrncFPMTGxoleJqUEzZo\\\\\\\/EalyoX2lauUCTxdJFLq6mHHt58xEGp4vPyvj85R+7uH7X+JBoDVJFszX8oudHD6jRGD99ccSGXzKcigGV6V64gDAGFFBb6H4e\\\\\\\/jhGxMKjQ048RGn6EIBuDG7Cx5MXpob0xrZJ7cqmri5kX+SiowKHr0W6D6rKDHHuEKKhxceEboo9tFO3nuvNTaFOshH09NNj8ZuSJoarpYvO7HgM5Q7IPiAKlKLKmsq7dmpvGY+B3\\\\\\\/aw+3cCWumdSRdPfFrWoetzVeDQcmXy\\\\\\\/rbnJLTTF5NAwoG4WR\\\\\\\/y8qn+yr4pvC339f2TzIvTQYOwpzgwrgvGrNIXflBNxTG61fqf4BiJp3b4abC2CAAwvWMD03el7cFmh1kXAs+n\\\\\\\/zUWts\\\\\\\/UB5LnEuSdSQ0XrRNNdB0PuGfhISG2HNWU1D10xIIHphPr9\\\\\\\/ZFdRcqBrXzYVfve9ju4Ua5gyvQHECSAV2L0TKicV76oif0NN4eygPpvC6zdvr5bNx45NTJ7WnKvozEvknGMn0vYHLFkvrYaTv4BRfyzeYMX9LGvBYBlEYn5IzJkF9v9Ie1lY1RFEkvncU5cHr+3pSP+jTOl+q5PymGOtOdeU8kLjgVdUYEE7MhBysn\\\\\\\/O3XV6xHxrbJ7CwhOR9mAcOhtnxKNcFU9ipwO1aCd\\\\\\\/oH709jbUyuIADoLmTk\\\\\\\/kDdF2qdV\\\\\\\/qfgCWp3AFHt4H6mFKpe0aAnYUVmRRxKGYCEOees4fhet29GHklmllgGY63a+lKmOgZIu+HUpcfxCHEDHk7OvqufB5NYuYRFpZPIK18tR8MIIk+MWamzOQue0Rn4wrRrdfiayqghmpdpJOkVSZ4nq1olSzl2lI6PTavMt3BDuSMontHBKQ28PEZekDN4ht3\\\\\\\/ckJ9eCNTid\\\\\\\/b4RHtUTiwM80JyvZwL5L+ZDvwuCoHoP9BZJSfGubWZTsLPBxPuslgZBCwozW+TqFzl9QITdxtFy4odoiG1ayVBKMwwRmTv7sPKabI+G97ibuRMCZNSD9O2naDQzaq6c42OefClRlqt3ZI4yqspm527A2TkNuGV9LB4z81+g4P\\\\\\\/JaEPq7StLKg2wIV\\\\\\\/gKXXJ\\\\\\\/jhvTzlI+9OVgvUFpSyDlOQqX7mnWfuAzHcIcmi+RvK+TEtgVHEfMCevLd9gv0OxSs387LXvtnGe8Kg309sldYA3BZloMLkA0safROThSJBN9VX8YJohB+ecoLE8+Qt56d6ZVQ9TJviG5MD+SfiB5MocMAsfr\\\\\\\/b\\\\\\\/TOltzTd1La8DqDqo4QhhGVQCXRcljTZrlrvTNxFBmnIszioL60tCUrKIzd\\\\\\\/c\\\\\\\/rAbkKavXS4Gi5CrutN\\\\\\\/5xw62OBlqN6ufEggXE70V4xhqevyjI32tVs2WjgnI+rJz6\\\\\\\/ElF5NITNy451O\\\\\\\/g6t35lB8QgOtJYP\\\\\\\/TF4ZCrRIo+8vhNZlpy8ZP+xopawvdze\\\\\\\/YQ9XyEIIDDn4fHXmvBQ0Y9QihVL9OeWYBaje1+EXB1QisZlQmJjvsTyPs14tyCaV0HVQcSEEgtBWD062K2el1QVPz40tvt1PnZ4vJbxbcRklcU+evxWhXu3wyOTduL66VEgRanY\\\\\\\/rrvsfChqA9kbGM5eqovFXFWIm12eeXF56MOYHt0K7Az960yNiq9YZIjbBJEf3fQJGit+uxA4pt89x1CKzsKPVADuya4K\\\\\\\/9T4UwvZhPB5Qq0tPQuj+JG7iCLW\\\\\\\/QzOtUa65PPoyfIzMA2eIxelAjIZUqt2u\\\\\\\/lpAWnGXg9gB0sTUSuRR3x2uyB2dNdzcF\\\\\\\/xmAFJyUlHaV0LS+DtbHJI1Mz3uhfRgPV2FPEQdjerHBqN9b5BffT97LUDVYCu8CqxTqRGSWVogf8nDM1fqr\\\\\\\/X4lnMIt3tUYP5QgtlmAaHW1Ml+RVMBkMywDEIMUUT\\\\\\\/Xr6RGojbYtQ2SZCP3OJABG0PcxgrpJOgp9i+Z9NwQIs6R+GzxnJwsAkaaZKHvJ6imH8w4JCMURWRtz93eBISkFZouKDZf1bOsLmU9\\\\\\\/MEvnmqdjQFOfAZ2mtkcsC2pQSFBo48dQwCrCpkgSjfoyYy2NXiFF4asqiMFtH734SNlQHG4h5DuYcgHj9aEbEdv8QQTbSCuR56A8cvXo5Vy4L3QCUQjbDs0I8DX\\\\\\\/9IgUdR5r4Pw2CguC5F266pJKoqZjq0pvz\\\\\\\/rbkOl\\\\\\\/L+SWQ7G0goTMsupyRLdZvPB5+Oibk8Rquithd3Ta5U7KIFXPRtYWSUBatXwfdFegvKMqvW\\\\\\\/FvHVcK8B3iCDhs9q1FAb8z+dCwOsm029ibzawdfa0yZYRWPuqQIBN9Z0s4tygK6Y1anRH8fBYfXtHBHwsplNft9ny4m89eAaLJIlr3uwviTuytTthp9gjLcLAqNcsBtQ+GI1fJ6sxEw7JXOo2TYksZxNOTCUsRTXjqs0ouNQjtamxTBy+vg7BaTz3Tphw57hCRY4uwTJip9r4+MpYIIm9t4d3EeUv\\\\\\\/pEZwaa6YG\\\\\\\/1NwUliAIPnvdCbu58sVm\\\\\\\/4+9eLKE+2+4KdVAuqeC+DGSAH0sW5Cqce9qvk\\\\\\\/LE2\\\\\\\/BhgwubPBTC73wvqmSvactUxv1jvyEOCppT204CSvgl9Za235o0265vCJUmu\\\\\\\/5TDYe\\\\\\\/yKHgChbPfC1Dq6T4H0NtYRTmp6kVQJJfkmrvelQg9OfNYjbBwJRO+8p02g8i3EqVvw3E3idm4FqggpS3rhRNDtbJ2k9t4HVo7QD+NkLUSPlPDs\\\\\\\/1NY53CEsTWFfAx4sE8PpPytrgOKFVhcPrV9f8yGPfK9R2kwrgongwQYEjd9wEqeOTM+P860ypoSW7GT6xmrQKTz+fhEtKojVEPsaZtnyR5Oz8Cc3zaArzIbNHzxwfXQ0O+AeKSTLTDKNYvA3Z3Obot5p0GV5UCmLSScwlFatae5fEQXGtD5u520wETtCva7O47Af\\\\\\\/7aAnWTf4KjTe06SyvGy7LgYCASSpHH8zgJGi9eg3wOGng6P5W8h+geyOPtMzAJTa1jeP54SzvkhLIBy1rFovKsjbNUCHhzw+eA3GMUHqV60SLkvRB61EBOrpouIuCMLL4ZTelkpQsxXPdNeC4k7lupmt4WkkBzsF8mbfX4X9WwSBR3bR\\\\\\\/OJbcvp\\\\\\\/sFDUQ5zKX0w4fJALC16DRnOjp0Y+Vevjn5rX5A5AjocLBSxuD+C6HFUC5mRLUoZ53A0vygMQF1p6WQAjX7uXoKkPTAclQZyCKnSWShr1pIeLHGfaIP7KcCurRrUp2nwPB63H+UVfDVeBMVvKjowQEmhn8LdscNZL4Ai8AWDT9J8YzUVMfsssn8t+ljWotmhgK2PItHr4DUyhQQz\\\\\\\/cDsATNTzdj5H9CNuQEjSht0X7hQ0MH42LoMTuZ6p5NHYJNs72SoeWk1QpfJqVhksqMDmQDs5MSE5NfA9LDBEcbMlcSnNXBfunuhoel0+vlF5LfyS2MC+l6R65mhCkQTAcoyy7kQCt1tE3880dDpL9W9Bq3jQe20gmDCuXxbP0vX9eldH4oM89E8pG2DObyNx+jgDfXqlawCuNMiVd7TDCp3y\\\\\\\/Q9OMFkqGc7iq14ZB5LTLn8gKI5gIcjb\\\\\\\/dK68YS\\\\\\\/B7XELkXP3fTG4gkKkVvOkXQKw9Ww7tQsAv49\\\\\\\/OxKweqqSW8oo\\\\\\\/7LmzYt9uhhn6mYUOOmssZfGk2EFSsDboGp0oiSaCjWFnhd881fFljIepBh6dPfoQ0jB+Yx91QpzijteJtPQXRr94jXmStHzXBZFC+1I7I\\\\\\\/47wz3OwdwGGLWTIqpEx86umUEUDnuhxH69TPF\\\\\\\/c9ul7q\\\\\\\/8pzF3HZR5ogp49Sucl2c9oFTgDldkUfiq0Z30JkOyR22yl6OJT5w0CKWzySgkQ86gA7CvzllK6njqEIOd9\\\\\\\/JyMq+4\\\\\\\/9tnxzUDaNVREhnNvttnr\\\\\\\/v4zQmYJBzWFTrLX6jtE1r5QHxkl+381dvpaK2AmYpuibdmZ1nrjQVuOoADcQ3Jgs7o+Ad8Sw4RNwz2z7jGkdGqBwFplhuXUgAdwh9d7DF0qWxAP7Y\\\\\\\/NaWYoG6\\\\\\\/4bOOP738f4rrY7YeN6p1o2bcrgIwkOFYayuY6IjaOMa3dAzuezmb7O9AMbF32J2QCzBtbB5hlpip6Pikvhra\\\\\\\/dO57BDseGhefDISs8xSNKfogKUsX5F4GfJU8mbhvVwygQ2cMTJMfw9GYiIKxn6QrklzAy\\\\\\\/ZtCvJCZxRAotCfOMWscBssvE1BhUU+uJ2acF\\\\\\\/G03auD3kr1GJEDKeblbVrp\\\\\\\/x9RJL1c1zgAiq8ILwfEVqHHR8NMrd+hr1y7usetp3HXSXzwas1g6PDV2tk+IyYkRITfzePTvvUJiruwH1dpTHVguKx03dFc0yDGmpBWwarrZ5wzApFXvUzfT9jRD2PF3BooLwmbNqjuEu\\\\\\\/mVgc57s\\\\\\\/y6L+e+CSgMFx+MPO2rk8Xf8j7uiecm0VR\\\\\\\/OUSNmii4Rbqr9vYBbAsCqsoOlPKPelaxH7CklMD2NeNnWu81po4Kbosmx0uIqE7fY22EPabcee2ATUfpOGf5H6NHtcke7FJd\\\\\\\/q3SZ8gs9Dc5KiKJOkkp3fMsKx5zX18LqZuLPXB1qHh\\\\\\\/s7w7\\\\\\\/LxxW7hAhJUg4XaIO\\\\\\\/5aYjgKzog6tucg1BG8jPY2+6M3wfKmaeyIyheNI95t2FBHVKvCztzwTdrknEPwlj\\\\\\\/2Fcafjq8tF\\\\\\\/8dIate7ylFnE5DHzpVoGMLnhN4FD9Zsk+gTnZKC0vYQuePKTlDdzb2H6kfA5PigvZPHZIUSE+pSbzPw1V\\\\\\\/NiHBo9JgwEFNS3s+PqgHyosTXzvs9a6qpXO4\\\\\\\/IQ6bTTDgdNiX1PaaiZvd22zrIbtBMBEQOYXhTHPsgrIiatqaVrkYcv\\\\\\\/d3apdC3Vj83dAKh4ANzd\\\\\\\/lsvK2X7QrwBTwzRrfE4slhl7jRrg7Bc6pVpA2sfgg3VjlV4GXpws7c7vCZm10RrnI81+I6MUQhIb7WEnzhkbvICbsPr+QIaFjWFtIgj9+mIrkhQV2YX+zRvXX3YbO\\\\\\\/w5MJtbvX8DBq0MnMQyjGW6KfgHfYcfapPr+ZY9GD\\\\\\\/rJmV5l0YiOUjQcVKCqi0tI55PIVTbaScdbwceoXxj9B6q5ZumLX1++qSb1KZjgNkWc+G46zCfCNENZ7\\\\\\\/D1nwxpTqjw0gP7EWjWZUKwhRSrVNLpcMfT9WAhULDQAYCAhiDJS72XVV0Ut9PVEqWG9fp6jFTnvRdHzsZgi0NJBskyztUO9+2hFXhxNfO6\\\\\\\/42oKPmAk9ZP02IlaCq6eJpiDR+NDwdgfzY\\\\\\\/TOL76JmDmlpnSADAJGNa2\\\\\\\/2IzZxlX3a3haqEApQRUeanGGRN9\\\\\\\/U6ohXw8WLvAPJReT6386TbOsOJ9O2cqh0GG\\\\\\\/DO90m\\\\\\\/DoJ2xbho36G\\\\\\\/vdrGzxc0MpK30eWN\\\\\\\/IhJvMKCpBtTnXnXWCHZ4iXMx3Y+K3Ql3\\\\\\\/aAlZd6kbiYudW2VFLfcbxvuOZMCuATIW\\\\\\\/zSd7uPb8wiW9iBO2\\\\\\\/yT+rJRsIh49tJTqg6UacMY7WC9KYeOM19Tkwwulkg+EQhd5qeardNPqpQoT7RbUO8JgKvosagzYoddIQ+T5swlcJh7BhADe6QmsWcAbczbyLTpW72+UacINNXlnQbiPg1qIu8KW8XPF8BRpxnYSuDUbkK3WaVZY71Bw8YlfF6GbLz+eRcKJH6KSbXwCZDF3VSvne9qsvbmQCysYjrk4S5I+isiBKJuI5KBaPU0eG+B41HIs3LmkZk+hALOgcGKhCMli71kGRWF1R0NWtKxwwu30wpqjhAycPQhGqzuJMnfmFPkUSkswebWT7Xsxipktl\\\\\\\/ppPRB0EEH\\\\\\\/wbLeQdq3M00FsGDRAn89xDH6f2DSqjkhKLdxddAW35C9j+ZkAyVTn38agtQCoQkMnVf8Qcx4PfyJ+uMZP\\\\\\\/0ZLzD\\\\\\\/cZFeS2M3FNurgha4ccHRFjpUPYpgG\\\\\\\/hqF5Qe4M\\\\\\\/y6alHS82S+VOyVyXs2FjZAEKSEYpedLXkicpQXGMaUbAR0FNtLPGFcQPQEJqo73eo+3Q6uYM\\\\\\\/AOjQNUU5073UjUhKmSsbxbdOJDpuJPu\\\\\\\/qkVTY2H6ZIa1JP74FoO7kPVm4Wb4dqLqVmKJX9W0K1Khz+G0MPkyjNXw0lBvP8Oa3MWJTm0CIuilIK5nrY928YlC3dLBqoN3NimEdo\\\\\\\/8rTNNs8V6gaq4JtNoK0\\\\\\\/kGJ9lOZl0i8qhmvXaPwsNAs\\\\\\\/aBcCz+93SXc8\\\\\\\/+IaxKhCwCfB3qIpPuvwi3z7CWDmVA7yK8MnKUFwEd+rJHwT3+Sfb\\\\\\\/SCMWlf8Ea67324v+IPTHJgXTyhwA2uq\\\\\\\/pvSuF2tV6MW6mBnuePdzdMhMS12ev\\\\\\\/laNdfoJfLCc8+m6Dq8Mm7A3juAWDcNN3KKMijYDCdx4r7thdCT5+O1+We2\\\\\\\/2IMQjdASgdXuWrL+GIDGkBZ+ba55oD4tC5BZUEd\\\\\\\/Y4gyhvRsRAhw814LHgw2lQgWChDSKNUEcJ\\\\\\\/HPf851AU0RzJWR4y4R4qm9+bFW2RgmAwuri0BsVVo4XzYF3cg\\\\\\\/o5Wbs5gSWndpwJ9w29ZU+KzM9jvHiegjDbyk9q4hlxER4bfe9iJENeHl7DxGIDGNhJvAPdD33kXX4XF4IahyBWQRB17zlruZ8DPQCHOSslD+MeArE3KJ8Vjl+ICAP4Bsf5kdhLhIfuCQpvc4DROnj\\\\\\\/C8qVjAETCO05ZrKAEOl7gpns6zad8Q2G37iJLo54CqBQsQ8R81junelZSZpfbaREByl7ZKN8ZlWH9LyZRerGPR3d1rh8amPmuM+Zj7x0jyI2GvByZQBBffI3saN7BAm+Xen5Xxi2GDRJCdXsUPHSq66Smbg9t6XzCo+zJl8K\\\\\\\/BhbNeYkGWCJGyNqMFcDf3iqc5THjlntcp0OllJfvBeIaSkVH7Z9WEzRrgPOpHZoQ4OtCWnZ4spdBJ+EwA6wPCHJ2O8TizfmPPxdVM7azN53Nhp4EHngJAqYIPuWS4Di5YGL+nOig9TBi0qTaNPcWv099vZO0eBPyj2+DzK21OakrTvzHRLuFJvu54avb0gqCz9n9tiz5CPE6iQtmuMSW8YakHd2n6YsiV5Gcr4nXeGgQjxJ8zVo4DSawCQ4+SdCgi+AkJQTRjSRzVFGuEiC5+50SxnzUoWc967jBnfdboC5g0WnftmskUZHKxS+LUfJqDUSCq7BmtgIR6o4+Rf3dFpYtlw7wjb3EcaIqcTn2nI06+hE5ZTVVXstezlRcd0tJDm\\\\\\\/YL5Lsepu\\\\\\\/QkLQjKXJoCwbTQppct3JHu1bno2JI+ItRWWZqNp98JgZjhRiphDOwAotBmBh68E2cdSIWdAlp9cdZnREPqDybrPWncJ6nwV7klF6MKhdwc4yBd+LpcBlgzKKdYt6mYd5W10xMLW3CTZw2gA+\\\\\\\/eE8es6AGTxXlEPRcRmK84S6ENIk8Af8uUr1yTsoPeZZMZDiudXcgipKtJ3L9ZPOxZyIjmK5RF5ohw8zGVbDZsXmDIeieckJp1DYUpR3Pol7zb9S9hMjBCFH0wjvTvRmCQNttVCkwda1RpgLPK8jSSx0u0Ndvpk7SufdEQ4TT5xJUly4UVQzg0oycyg+DvMOmyco5um5eqZXT1Oyl4S3pnJ+BKlNJZmc9pdJRb14u+WXya+rmv2k4rlhSBScH0\\\\\\\/pkqWE3mpE489lV95wJHwRmacg6z1mOel6Rj9Dl3fedBhbCGz8mVaJHgUy1says\\\\\\\/Rgu9\\\\\\\/iWrXUGhFH40q7+Gi13P6AktsYzkNz0W\\\\\\\/KRENwbImMo6W87aKqns+kJy+NtRx7OrYlSxtr\\\\\\\/2CSCzy0bFbdLj5wGdQfmZFtb1nGLQ4OOvqsxxKG0S6hKhtqnVV04LL68RepFja9YJXrh3Jm6QTojFQd9jgU5bVKCB3RZ11ZBiDs4BfYf8KjWqo\\\\\\\/NJan4n0OWgQ3UyUR9gqt8LMOsXd25KsffIViyJM0t1WQUeCdxiLnb3uQohXKEX6WPltmvjrjROFsdiQaUt1A4owld8xu4pjk0aIWcd9E\\\\\\\/dgy0DBJQ\\\\\\\/Rg9+migLirpv\\\\\\\/iIwGprFLhyJniVA2TLv9IB3yVkgudffKX+fBbNNw70pz4j8EWDuM\\\\\\\/Wcz9sQO+Yd1I2cvWuNDmYoprrKBQxhYKWGGXGt6qEjh\\\\\\\/5LjV7hsn2eJJiyPa8JQYxSlJohQEkHq04Z8Ipm\\\\\\\/dAh6GhaXxYOF2lQTomBjBb7Zc1PCYgcn+MWktwIMECLJ9mCs8q3dmLo4INgRInRKAWP7M0YkHu3kxBBi3AN1QgO48cMqBUzwL+pQ9ulThkdZYYNULhoc8gBpYvNcdIxADBMVhxdhytLSvUaeNbAsgESXRcr9jyfhJoo5BSwNtlpG9P09\\\\\\\/ah4sQd6meX7TdP89qoTIEoBOxvHNxJdG3iFmClNnn2znqC1wpbtAeGC5UcXx6NnZjZIvrXp2rvIVpVa7BNL4N6q2i4jt3ReH\\\\\\\/cdCS\\\\\\\/jVCIhbqs9pC75K+sTtnZYyTxgRIsevgpArskborOnrS\\\\\\\/3PDUhcnIez4CneQvwKBkVyhf7KwOwuA2+TYfk2HU99VJwp\\\\\\\/n46jCAD6GS4VjYZPFfX3jp5W1iJ3IcAmuhrK\\\\\\\/XSplu7Cs03wmPgU+RWYFW5EA9PUOvQzSccLQ5Jm11wAbQSJhuHCcW2m5PyLL1VuDU8yLjiW4nZQQir3GRpwvYg2XdLLAjQpvomQl8qWTXITNxeZKltMPND4VfKVMk+EuGqkYyvwV8bcGdvDAoGWY6o\\\\\\\/ZA4\\\\\\\/yCJVcpTh1MA23IXTqcWeUWgMIZYq1gOtY6RlMSlPtj+u9+uaVBw+ma45coHs5P8sKOZYdBI030LrduzxWw+FSprXyNVgXTuo9PTUmjQ5vGBBoVDxB9py\\\\\\\/AVP+kRdKqsiTgshs2OFm5Z76v3OKWN2orhVi89GphL82ZEBlqKMEfE0qCeLBIzSAaRVNtw3LYi+TgdpIHMk+qDiSsrjbue4FxezAXotbOVdG52ES5kn+qhZhzdTKhADX\\\\\\\/e7T1SMA+4\\\\\\\/ndZSiNPEsFAC4\\\\\\\/J3HoPEvYP0zEB2aQcvVqluplJLjLUuTA+X+h3pwIXgsMBJUaf1hiPOsZ6gbSu1Sglndz6XJvjB7oZJ5i4q55M\\\\\\\/KjCTEKJK\\\\\\\/BGCZrnuKUiAxX+tWfvT\\\\\\\/x6O4wog2xiTIXwDp3\\\\\\\/k\\\\\\\/I1HODewIKJ2iqXemT3ezkmuhTYY12Vvf7UWbjKshDdxyk3zsCeGYHI95gs0KjgZDwI8NUdV3buIcmHksD+BPuunWLev1uyMJSv6Lv94gaZp2JVT4ZCf7782prHP3+ZhsEnHL8wbgzRB22PAM5ZSbxK0wIOM7KZtRmL\\\\\\\/udls8RkluC9hqmSaAL9A3HCgmq1A70BasCLaHcR39tM6U1ley+VKdI\\\\\\\/GOePbVL7d1bv5MJXrgrnK4P4aZlmVeyi4Z+czTWaMWgPuKz7w5WxmJ9JtxfoZ9sUwulvjqxxy9Z\\\\\\\/VDKjiQkFWVS5V2TL\\\\\\\/m4udlTeWK\\\\\\\/UOvZVEsZ4qb+uY2UduRB+JREyErS2gcSC0NAsIgIkDg8MA9xvmSTEfRl4uwo3y5Po78B6qbDDZcmMY5P4h4ReCFFG09fRKfEqAoDCZ7poYkyqWD05DiXDXx5XqPImeBgcolZR5sT7ZozAiq49gtJvEqvbRfoYVZcv71dBGKXorbSh1VVFB6+O0k+kfNZh0nTzZm33C57ARjn60btV7Q8tBfesg3iWlAlCSdvfP\\\\\\\/dgFynC4f7+\\\\\\\/VSoEcx5NGbQATQPUPFG3ZLZyivQwZuhkx4Ok5NDhGk8q8BUNFhgYPO9F5U55Qw8ThxNS\\\\\\\/tBHQ2OfFUA9YIlmRzPChEd6jTngqMl6wX0hEbT9om7uT3bHQT0+XENvwPaogT9oR2sB7U0toyojBZExh9TYsh96WESB1EZr8tPNb7Vk8MznbKKklfAJSWQZD6MdAOz7ZovSlplcMUJCmDZPjTVm\\\\\\\/\\\\\\\/\\\\\\\/3yTfcQ\\\\\\\/BiajC7O+aaJJOW4DHT\\\\\\\/46x9XHnTulwNiMZenmY9dgVgIoTqsxPUmvIm9DSxZAULHzcPotkQk6uKQLwskKiDfGjtrkHsSrIr6LnHdomRjSpSvnPs8dpJ0N3vfIM0fiWGrSQv8kv3EWtPJmXbSKGNFx6dIGQsLY+\\\\\\\/bHxG8L3SxQF58zZiUDLzOI3NVpECXFhUr2mQ3T8PcTs9V4dk2viDIHKVpUq5\\\\\\\/YND33op70wbpu7WS2gZkTTpMeKxTbb6OGwy+Y9h+Kf8x1joRktAd\\\\\\\/ea7HfWAvlQPVvcpKW7Ri0WFXmTKGA3gbCOzP495fOZb848xEg0brAt7PJUSSDQupNFHnBZ+Zv6Xw8ssm1cv\\\\\\\/KYqtGHDr+W6\\\\\\\/MIK1QaDVOWiXSIX97FXHbCeFoO4oHiWe\\\\\\\/V0mYM3A7zeZ93NenNJIADCss201OxRn8YSLgq1GXaNuXNKir2kYZfDCpjboZPu4p4MjRy12o4vmltNhBdntJwB5rtiUN4g\\\\\\\/eP9Tgm3GkCZgM3EjZNLzY4Fcr68sj6J6yluXHrfyRwJ5gxLAUQc6uMIdgjSkBzEfbfP65tDr4XS6c675SlBZN+qrkwH\\\\\\\/htn13timvdKo9RtlX5EAuXST6nhUP2xfdO5KQ4r1fvs1CElWcbTFFz7yaKN6dBdGzIggaj32Q2\\\\\\\/s2esL+mQHRon2dLZlumVQJXEPXOevdbt\\\\\\\/XY3oj8i0VIFSKr7FiqABAvoys8LnqgDs+3bXV30xHXBzG+PlZRA\\\\\\\/OxauRBTo9uF2WGmpzRkwCrVhPvsUhereVOYUSOjZGi59Y7Z6hRQj8ykhB1uM+Wwmwa6IkatvcK9+JtxOX+44MlaajI+PHJZ0bKB3H9\\\\\\\/6KAwcvMtI9QT75ZNu4\\\\\\\/+KwroJzDqFTfYFXSWz+PFc\\\\\\\/0ky7sonHPADf\\\\\\\/3mCnhrUtxn23tkCO\\\\\\\/lB3OaT0Ithi1a\\\\\\\/xIZRswp2j602qybLs3ra9cz9eZn3\\\\\\\/yRy63e37lTO5o7KZSwtKOG7dE5UpHvYrJbORDCD2Muc2objj0e060vOfl8TZqiui9FfXpcGSxcZxRGmRS4peDf8I6\\\\\\\/G1s0K8OoVRELeMA6aY6SQGwErUnDwyv9lHy6P3sSW4q4XMtVJ63o7FM0KTpMBlvqHgqfpcdzw+I9PbA9X7SWeMntenLnjykiQlTQxIPGHV0MA\\\\\\\/7fHwj+bV50yeJccQxTFq7ySFTfzMJkZskCox7gpDzv8w3DrZxOu7DRbHVb36HJWHAst0oaMXi+eSF8\\\\\\\/1yPPMjxUwh\\\\\\\/3hlNQA6ENe5ny956pDz9WhIN2O1OFviEECEQNzyOrnJ1OUqjE5OfTdHRi006PDBN+EyGevUFrh95rZ39kkBhKDdOuOhnz6XBRw7bYeIrHsQ8ylXyG81K3ZPsyx5Cno5pQ7zTCKegFHaBIFanaf13hgKXQetbmtTvBIyE1LhB\\\\\\\/c+Y+2tiTGftdvv45I66rFN2ETPgBON6OBRRcKET8xykMWIU6qUiijt\\\\\\\/qrFwNZ05KrRuXX9CJPNaEH38zuQLJLPp2sOxhldJwadNDkd20Q5P1ZrJYEk27f5TSIcRuOC2MF9nNxvmVs\\\\\\\/H07gklWeU+DXisz9MFec6ffVoqxzix2rqZ9R5yBfCBgGSWdBmbdfac\\\\\\\/ww7iDPBfhOo3YrwkWFZ9fvv\\\\\\\/TKSzX3w87L0P+zC4Wl2MstrorXiU2cH8knYVM05QCOTmTaqFq6R3WPP0XonFDcH\\\\\\\/02KGzLbJeea\\\\\\\/8ZnNT5uipKE7pv0BqYtuWnIph\\\\\\\/MaRWahX7zLxIPwrIBCTMwyP\\\\\\\/HXEvGG+uaO5HW9uIAPrJd07mY0K8THjLh8bfSlVvwrCSKAfDdil1FxUzQadO2XB5uaOB\\\\\\\/6oUH\\\\\\\/UUjwigcDAG9Px5Uj56+MFuKevVV\\\\\\\/zXcqsKEWBo1GpVB0Gl5gWxQNypTULHE1YieFQQUbm7Mb2CuXabRxbE5K+sjsEUD096lUakqys23W2Zqc32l3nIP8v5ezmCSdyKWIzWUtCM3zD7g6zimDh4TvLWDv\\\\\\\/JLIjLk9mfTueJPOFK9T71ljR7e3nRAii8jg\\\\\\\/sUlPcjQE4u5FeeWfK7qH\\\\\\\/Og3PNWAbvNrg8L9m4jKd6NXkZlAc3VchIucIUS9OECPZ4NCizNPIXCl805KM9BfwEt5t9b\\\\\\\/TBs9CYJteFVjNYffS8xQeprIuMFpKyth8YK0HdkoLUSIIjgSbjq8P83ctaR3bp+2tEvo+KoTG7KTJkR7YnuIIxu5h4ogMN5rIwU0rIJzB1K28v\\\\\\\/EzrI2jRNW6Pc1oSG\\\\\\\/ppNxXxReN7+Hbi\\\\\\\/pjqZZJf+\\\\\\\/rO68giDp+q4BcUtY5i3\\\\\\\/HJWGDsoLAtA4\\\\\\\/HQgyrmsK2gslFeoUjdyDALd7chTEhEeRXJGo\\\\\\\/k6mKRSAGXxzkMaF\\\\\\\/GAqc9To0xtPeVFPUsLDWXMuhKLMTZJQw59EXe\\\\\\\/k7AZAHSo5qZRjPy2xeQ9JO\\\\\\\/sfdyRcxXI1BAOxhLo5ITtOjwqgD86flyVSws2j\\\\\\\/vckEU27ygFnF0UOTkjgO3QktLYWgNBHT5dKaIhhR+8HLZBAsbRG64bHgrhKAIhG0t8VpzZdLt3BHL83WE1OKVytmMt2A0T6tVwXH4DeWdYhDROzG5JpeaeHXBcaFOk0+lJMNvU4CUUkaJIaTtvXil280VOo5TViCA+aqUPWsfAjy5PVpbJr8KuGP0cgscw4B+g5c6uBj1aFquE6iZNYSZAE6+CDiaIhvy8g\\\\\\\/8z0Nm3Ttfk8PCjeOYDezTfvB6h4ic1lL9cnx6oz2X1zoNIeBMskJcL0unUUMQdwerr+zZXwZCyCPONLJkaIu1WGqAUFnvvDI4AI7q5Q5AuPaHkGXx+0YhURzigcjWbZVUNEzUVDu9SSCZWuvevVqmaJFATH+OU67G3aaUuPB2OAQ2DWz+otK6q6KNPlgrv7EhBYhF2jqjlpyTUkSw89DPh9VKgVAntgJyRu1Yfx41YrIdTxVM6D7kEwld2S8BhdjJF2n9TsCm\\\\\\\/Lnw6mzFhNxiHsaa8+eAtm2gFgrnnvrElC6aNj+pivyLJyOomgKlZdz5bR\\\\\\\/iO\\\\\\\/jOodeC4u+CfUohd5V2bCpgURJe7Mea5em3mxgo1w6vH1MykSh7VzS258BOYpGZ\\\\\\\/k3ZigoYF2HEV0dN4rJ0VJgm4JnT3tUaTNf70nRTWA\\\\\\\/4PNOnw7olpdrMRVlmbwboaH7voaTxbrctaPQa6MfL51CM03qkTZruUkhr3mccl+idLaHLru9gFHR4z3l80v\\\\\\\/Engtg49J6MGac4JmATHil0Og1WN\\\\\\\/oVAA9IS1CXBf279aUcngYGv0dwgozWxzcAxBRLRR81sHWX2TkCqAOrVaDcOrL0EC1JbZ2jtEOn89m1za5TB3nVLi+YYIS7SJWhTX\\\\\\\/wUT+mBdtGEk07XXOKoyPS5ZMdqJAXtiEwj4WYFB1plF+Dj9Dq1qJpyo4iHCb7u5bueiIerW3RoAtgCKMw68DWt2\\\\\\\/hCemy3QQsvMgcDqOpouB48jE9GPCLKfvWAaC+PKKAUCbdORRJkNDXr+U1+EpOwCnjeOHgilu2LjnNksoJXAy\\\\\\\/kKHk2dktGfnm96YV9zxeEtNxO\\\\\\\/mf6Adoy75pJ3r6MviHJ86sPjb0Gq4idF8hPhcQzx7IB27WiBs\\\\\\\/syzPFD+MzbeojGpxQQKSRC9aiv9pfsIMHtlokRySiJbLccJcxb9wKAj4fP+rePCCgoXeEGpIk5sF\\\\\\\/nlMPAghV98YxD49w\\\\\\\/SSZ+6381dSJGKFrbugdNuAaTC0TavaCEvAnGxsPqf0SRYssRXrJTryb7K3zh3tpgkYMbM7W3qVVbdudHYA0ENkY9tG79yEO3jmLVMv8yi6jYbDk5i\\\\\\\/QL6Zn\\\\\\\/uJ2RgWlmjivk62f\\\\\\\/zBfYNEcWJ77KihiZwOOL1KndqtD99Gt7SPwx7bGp9LXMwjXhk8ydpcseveeh+KQ\\\\\\\/gieBlngXtltgL5kYCxfz29a3ifoG4YN\\\\\\\/LzzRuaJ7tyzc7Le+pyVJq8Qa9pGvw3\\\\\\\/6H4Yy2w+Y611dEHslvSo5OIDkp51WyQwEQ4pqKUAtDhrUpAPakEKqcGKYDcH3Vc\\\\\\\/P2rxZJPj5\\\\\\\/8hkMfp172BfjWPh7lcpgePX7M\\\\\\\/vTDhDQkExyOUSLgJWu0pCD2VidKWGQcdp9iU8KpiXswEzMzAmPDpOx+x2QvdHsqoURWkuNNv\\\\\\\/l\\\\\\\/PnuWWlasajnlzDfn3xcZ9GtdwzmG0zcYehZ0D93KGEMr7GikJrJaaPUiSy9VMimS742\\\\\\\/xFtqGYroOfHbp9Yt\\\\\\\/qESTW7q7ompa0JnWVfrfPip5VthZY+ExOqBVsoK95pi7\\\\\\\/Rk0O36ayL9tNlh7G20pHVB0h0\\\\\\\/Cg43qma7IgxV1bRzyFvWaIquooKDw4yMm4pBI5Mxk7M5M5JzbjMtJ9U0PZUXr+79o135hmlbkSvuaoNdYsXnikLYIgzHCxJBjmPeqDh+v1qWl8DIx5nSugaAqNx+QKrwQAKGOhJaCRabo7vmufvy07MJiZMnvpqpbYf5gIWPT0DRJIwiD5vVvhapkOjd7dBOY2I\\\\\\\/gpJSSDJqfGhSGI3veekGAyeMXtoCfbJKf53LXAhmJTPtUJGApRrh+JbIBfoyzOlsLp4GSvfd2EFwSV8KxRpnwzEaClcSNe4Q5ve1tnC5sGkMZRWLJm\\\\\\\/o2yzYGJ5VyaIn8FMJs\\\\\\\/QghpcgF4mE+f\\\\\\\/4ifbfStYoDTvBM6m1idK1ZgGvsnO7NeaRa9+oMWMHYNit\\\\\\\/gfvCzCQdbWhHJDD4U1K762HU5VEohfk3S6mGbts+v\\\\\\\/jEgqjA8SXdLtBJqD5B4RoI5VgbdcIQHhuv7r0ytC9v8uYwjC12Jmrin9lSZoBlcNMJ7fhlC3r+jqSEr9l7Ke\\\\\\\/bbDmgFkplFdX6N+UsZP7PYa3TPczXoRSI9S7gPIQWsOXC2MwR9aFTmpuGKBanmH2yPU+WqwriCyHaZ8CeslDhiMO2HWb2sZ\\\\\\\/rdwejsTZ3ziNMUDsRvRfVLIHrCbg+EXZGMxAGCWdvw3RVqwpwvZJEFS+IZ1XZz3uveP7Sk\\\\\\\/wV2N29kdYWIDhq3g90v3I3yxCJUCzrKI7cpfPct1yvttyqqLjCmdG3qggPQGDtL+ZYfF93tSMEUdjwGqzYp\\\\\\\/bInES7Vdhu8+ZvnFGajBh3f22pAMUIKGBAHq+8LJsFWaKEI6\\\\\\\/D72nvZTCouQ118Mi0O+TYmfLpAvXQP5p0oTu22hWMbXCuY0jFx3wxYcfJxJSQDyr8Ll5u3yStwo2lCQcwhcM+qeaXHtcgy\\\\\\\/PDVPpD6A1GAQzZg9wM6qaiJWg0MRCwLJ09Gbh\\\\\\\/r3TapMGwHh91DJ72duicDtAxWnTIKja42lfIr9A2HanEXcYag0NCQTPWAQBy5qIGn\\\\\\\/DMmups2L2HUT2b2\\\\\\\/V0wOcuv34x8wYzd1U5ITrwsJivq5Z+iC0R2SdQcnoQTLL7fDqTumjSudy1\\\\\\\/g38v0fwcJJJoFQusT7PLobLMwFEMK4AEWKBVHGt0lKhB\\\\\\\/rsNu9oG+M4VqMgL0KjR7xavHSQpkyR7zfs1q9Wtrx1NL8ua9NUnC5AjSUD91NRb7+zalG+nJcwG9tKu07OAVxg4MI7QnZMV\\\\\\\/X3F4ovsGQt0lxX8KBJAtMTwc\\\\\\\/IWhItKYplBZa3xKi7PCvzSVveOpyqMDlugPkkzhWF2HB5CAJUzx4nAs\\\\\\\/p\\\\\\\/qlkEb3VKIYjzRGdXqkAOByy9srfDWD40IMKn5M2WLqCfBNZ84vxNsUeANAZZyxZceuH+WGpUYnbAtSnh6ZcXONy3WT1wqYvBr3c1L8bazb2QhaNa9HGaaoEB+qasbK7N\\\\\\\/uP0LCkizimItzCAUzQhfyWIAt1E2pof2kXOkxmZylJySezdizB833Na3SPgPekL8Cw9ya8VqBjybo5cxLrSbgyoj+PPBtReYWg\\\\\\\/fmv4DTIWoR9fWWSWJKUu9dfmSegmBqD43dR3GNMg7l2+c5PRoqwJ9WWkLi8PsH+EEQ5sB9\\\\\\\/sGKcUumqIp1CP2yB9RUO1DHvuT+NeATKEFAwAMjhfw1x9prFqtWmHIu8dnC0AOb81yxcb+FOnM8PF+O+c6VFX2G\\\\\\\/4JM2wfDoH0apb13cCknlsXeEDrr5\\\\\\\/tu3qAjJ56bUURyi6PAiOZK46jyvNo4uH28m04vt1CySf\\\\\\\/V5pUHoMQHmiWqA1qz5DRCjkXPrta1IgscZ5McvpGANyfA\\\\\\\/LZmmzgmA+FrNw8TS9kET5E6IWjSux26uXh9EhLXbRg+sA1gYgoQgylhkTAt25hDZos7HU3z9rR8Ibl5fkKx\\\\\\\/spniBFmBqz9EvbrhaR+kyUn3kD8X\\\\\\\/chFdltxSULz1k0eyl\\\\\\\/6C+1QxmKxCrIlgor0lLWH6lgEe\\\\\\\/1gEp+Or285ufCbpf4xw8MsZhKaPuZatwiWilDTr3nPg2nyjOUDChTHDK5Q1ZFVhyqNBgFB67vrDxaVa5ekBD\\\\\\\/a4sk8\\\\\\\/DcscrbTa4AH\\\\\\\/Vec4HvKeN0h1yJ5Bq9S\\\\\\\/bew4zpx\\\\\\\/MGd3P5WC2dN3MxCy4\\\\\\\/vUpwi4QAylkXf5C5WErHL0SxVRWXr1ztMDKLwdZEozj+XfVbm7U7f3PrPWtjmAGpU6KfoI9oyI5Cz9ZWmdczY4qrQFKQSXDe18EKXibwqo7jSRNmSlt13r0bNWjy\\\\\\\/Yz\\\\\\\/H6PsM7KbdZiCuhd6w4cTvqRJQw9vuhjagJUut8q3eEY0ICRmLCEeI83KYyuTlBjHSAIZxtOukya073L1GXYg1QJtSyu6C7F8n5p8ul7RMo0vJFFPJqwQPEnGFFUNSkZg4HMiopr5BPexZgeZLANMiueXy5ydrRifOlWmLue4nIyZ4om4ZmP9BcSTvAtV6Vvj2i85BcCPpfQQxTLFEP+fFMrFJvjMG6a8tYPy19L7o5zdSzCfW2Lz1lCPSITkhgjPDna9oUOhtBU0eztKezzyvVXO9Yqms6mVaQWMAi1BueygBS3kWOXdvSKJ8SLXd7Za5Os1IYW+MECDMzzSiO\\\\\\\/EaoPDU1wrkmc+Fe618Ba00053hMZckUGj3UlFMX2B1JIdCVeOoLq4h\\\\\\\/0Wnxeu81EOaKfqwut8Oc1OcSIeTm9ej79NnISr\\\\\\\/e9ClYFg5IaCwnWJbOLhsILf7Rj2vq8qV6hl665HXRSGtYcA4Zt3PMAR+Jd2OENp3rHtuTGg0Mp\\\\\\\/w8n6Y7tEL97Wgu8cy8rYdTf4Mihp9pdwQ\\\\\\\/YSIHdsDY5cKspJ\\\\\\\/rM96jbpOJUFP\\\\\\\/UujA1BZeKjOMbI1FPmOLuGch+8GEjVS5Q6BFMM+OwcqllzwWFV\\\\\\\/jW0lLsRHnEbtPC+FRjdCkXqcBIPsOebWSk0M93ol6WQqNJdh4V\\\\\\\/X5gixZdkKgYaFtfZWJoEZhroJ4MwPE+vo3EuM3qcDxGS6p\\\\\\\/qFSYDbwTIIUxaykHlUW4NiJjlUWdHpH\\\\\\\/QM7mzCqAHIbE4VQ3If1pO6AwwnsngJixoYZmOrzz0AfNYgN4rTXIViv307HPuQd37K\\\\\\\/nMb4xKoeYmNJdoTXw9tCVRwayEUOUNHrAYsztHtlq3IujXYRATkyQWoC2ZBgCCtg+FkNtiswybzYPr1\\\\\\\/2VgryaYuhZHgnQESsmdbl\\\\\\\/nlSIwBSoWnKQYy8HQrerQuAv7gBemyNpSTkXIyHIRFdqR0xBnXGYxo\\\\\\\/H8A3pfzLnhsoI+0jKg\\\\\\\/S0+Mol69VjnP4cL+u1QK\\\\\\\/CPoPcJinkO9i4d92h42SUrS\\\\\\\/psKtqosMdvh7XyFUR+GtFHTN\\\\\\\/05cYfO7LowtboPCfsshAeFsGK1TO7pLTTuBMxpu8xMMn0Mc2NXJ1OdteVbooH5VphCqO+U1dMf7ENQ2nOHKoXCrC7\\\\\\\/DZrObWjU1\\\\\\\/\\\\\\\/1Cb\\\\\\\/7KsIVeF\\\\\\\/xuocY2Pc5T+f1eFzYlE8UtWjfZNtWrxwkQIDmXYysHOFTBKIlKYyG5QRZc3nEHITJ1GPJokDxUEJicn6voTwzAs+Zf2S2Ub\\\\\\\/3zITv1R2YUHvU1R03FGAt3l77SNd70tEvYrCTzmADzBPV7xbRbkw1MrbKgMv+j6mzFsvqqIu9Bv7AA\\\\\\\/4VTzzAJT8H3XsKi23+qlsCoAT0mj411aPzi6Wf5POxETPze6sb3utQUoC+ZZPXg6Nu1N8okCc8688JTNDEWJfOpCn7vUCIy+lKNTpS9u1nF8TFqhnXfjjgV87K2oMvJc\\\\\\\/6MzPEoAaZTm4PUmVEdBq81Sasdh27GF+254vwXq9jZ3CPF7omh5yfyNA6eXqWsrRrFOFTZX1ItCu1SpvZF8gNWRCnSaPG4aI2gC7dkYEH2wU66\\\\\\\/xrqLw9zNrMCtVpYf+NHtfFOCXUvAgstPbVMKl4mc+2\\\\\\\/yfVSNPS9Z6+cmM\\\\\\\/pHoCTD22d2oO7rSsUR8+NsNsbAn9VvxLs\\\\\\\/tJgcbr7aBtaiZafvPA6eYnVRNOe0X3t3Pi6ytDVDA0\\\\\\\/tv6cwRJKNW0IzOrxD0wEV\\\\\\\/F\\\\\\\/oi634VSG3F7Rzf4tJ4UGYkFUsApUG6+c05+YB3vloKUNlk4YNtC8dcars6anWk7jh3gfiKSxjeEKO1vvRTaMYkO84dsJwf4cjtmheJL2vM4PhHUkMRJ5TYbWtCNuqhX1TwTR9FAB3nDNqHcr0LCd8U47XtzHrAVQZVknLo1+\\\\\\\/ItlzwJLK+TQhn126pU+rHFLTQ\\\\\\\/qsE1kJiXExLVIZ96mydGAknHdtaG\\\\\\\/l4aD7MK34R3ejyVH56Uj+sW1qheYIlE9rDzKo1MKTkGZh2OeyHWhzrT2nBcXolH359ZtE+qS8HM\\\\\\\/lz0oyDz3jcOujI\\\\\\\/5JmEozau46\\\\\\\/MmnMExJ2B7H7\\\\\\\/Su9bSueNhbnz+4iVLoHNVh8DfBJPqU9e0e0tGj7INEdK3Rv\\\\\\\/p1ws+fbdPYVXrJ7HKvKIXBUxB1ihotQtt1ChMWk9ifyFbDH+LYeavZ8SWC20fjm0rV3vbnTNaufWjGlhuNQrgLj9c\\\\\\\/rVWayIUzWR3a0d6mjfRB6uUV8XwU3Dw+uqE451vcTwlusGGwvtilGhQL72uxK4zBbKTjEQJav1yo7frVP\\\\\\\/C0u+19scvdcLAJVL6+SliqA4UgmxITwHYmVU8P0WsxN6UfXnBJ+GI0pEx3Jzi26UtI08JkVM224Ce9PKzOHFtZRmabN1gFRz+p7DPUhVD13Di7LPiRpv4g04mThpLeCxXYGWjAVSkLID\\\\\\\/2H+87QnSKRC9HYcGFVH6QH3IEAmo7m8qZ\\\\\\\/x20yN5rkOeAD2JS7oar0lLi6S8Tuk+JIYQu3\\\\\\\/3WzZ2mqEu1wE6Kk3pD3X36b2Z1IvXOl1eWBaNOt2t8ZjtEz6g8UwKsO3PHq028OGR3B9JfMm\\\\\\\/scDyjUWcEQoLcZaIXu2tltagoK\\\\\\\/2wjLkm+9efMWS46GHiHT2um5ZCyLCAjMb\\\\\\\/2NoEa726y7iAwxcUErDUjWoXaIwUERqRO7b9PncO88yOAmgiw+nJGZi6\\\\\\\/qSMExLECz977XgiKBjYm\\\\\\\/Fj1gGhZwJbqzyB3+OeV91Ov5Bjl8peXdp8AmA7Qodv5cxEcbTycPCkom9xLZZPORLDJvquB6neOORCMqC4zfP4lH0O8lEdfxkTc65wiNhneojuPIVKhPyX8v2hgQ8SK36cVlbyXxyDeOiqEu8upg6PZH65NGf3yFwm9bA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022\\\\\\\/SBequpzn+pIGlVJe36Svg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00224b7290bbe8ba881adb4771b85447580dedf22fac034b103d33347d4784c47d7e\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022tppFxJT30nY5ked0vxLO8t9mhTK5\\\\\\\/brUOovihWsjNLyBlY1xIoLO+7YLUv\\\\\\\/yB6SqWPxqV0VLFfIAgcRvVVUhVMM2O5WeOwr8JTijuH2angaWQ+EGIiI3KenT6L4SQjQlCjwAwqvEFWI8q1In9m88Q+KC+YMNEnuJA13devSc6eq3lR\\\\\\\/7PoJ\\\\\\\/yLtkiwyjDtQrlLqpDCQOtPFxmUlwJ5gwfIDHUaumo4r\\\\\\\/ZhyhrY9VusiPPrJla9OhrtZFOXZfKuK01F+FKX4GxbiaSf8B9vVY1vBTt5y06BZBUi5nJSSG071Pk0nw0iDqRjehzUKxGX\\\\\\\/iQA4lmI4wsXzg4GeE4eI4TwcziA7h\\\\\\\/XOx3lerVXZ1aVPjAiDdg4OmwDs9i7I4E4pF2uDivC99VEghQsrKg\\\\\\\/og2GDSpzlXgbGDX\\\\\\\/SGNVE77I3u\\\\\\\/cotWyC7Mgix7uBCg+9lz9PGFgMzLlE0cfWfUM\\\\\\\/DZ8ydWTuU+MWoZnySHIdZgb2eyQFSf8\\\\\\\/CYsz+orPp7qPVeBn0+P0VebYx9vxmHXJq8Q6L+mWw0OaMm+y+d9Rkb7hyjbRlDeCKB4slv+dANqFl77PckUOl0HEg5nDZ8XRTC0tYiK6Zlm0XHTbj9B+Qc7Ui8PPs0YJTjKflog\\\\\\\/QzHBdUi7W59HrwkhXpgiWq8U43N0Q2ZCISz62Q2CU52tqb+Wz+J6Txb6cDipGxjrhZeAuI7uSw6Y\\\\\\\/1IHj7fzvfLUjG9TapWMQNQD7U5CCAJj+xifRTg3D5NCSjEdcBuauEYBeCUSd\\\\\\\/dfljgGfSA11BwLRlYF0pDERcGsDtoE\\\\\\\/xIJ\\\\\\\/mj2RrYOLppnFDMnWdpPfAZU6oywS6XKM+MLkPBAygq2ZPtL35Bkfvg7qRkJx1lDQOyEfAQjMEATXhB8fxyEqJPV21IuBUNh39flHOwjpyrGk7KLcvz3OYfgv919IFGvQjCQlp8LfqPZFf9iqNSOXoBGio0CTWIeTtys0\\\\\\\/tsmsKoLIdNvOWMbkaHcSMyVeinKX7H1ypyvamQbTKUn4dI3KGeEO4ToYruDfmVumSTOvMrk96mXPb\\\\\\\/0rES5CtwnEqL8DkjCAHIZt\\\\\\\/FanhYClSUneci6Yx95LgK+ZHh8WcZwY1r\\\\\\\/h5MGXKrVva6Qe5wPjOcZbvvwsL2WJe1AO+YiwpDPxUzk3yy1cGxvgDZLKJRUvRoK\\\\\\\/a9LatIONjAAYacNIFB2YUZM6L7qigPEEePERSCqQXU+WB1n1xghHj\\\\\\\/eMRyTY2SAEgPZLy5Eo48Y4lbTVhqIjJcqt7dUxlN5wk6ApytmlaQxyOfknjX\\\\\\\/JOsi7vDEyh+VNmJ1FmGFHNln85vZckEpC+zVc6PORfkPHQtKz00EILJeSVkkuDew7tchQIb0DagVmM3vBAHOTnWzT6Y7bVzgrAzRdIKykh1dB20NSII3UIdWnW\\\\\\\/kQ5Qo6mrRf6Fq7cvgvEE8HzwKbf1y1q+eZk+AxD35r+Xr9AoBIci49VlXI6CvvPwea4Fji2V7bhJ4O1+e732n5WUeyIT6Dgf1kUhDe343KQQhpi83JsvW7uTH3g9T3T+\\\\\\\/1BEjStIVeA1+FUeySrzsv7IZYFjeOPfuFL7MAC13icQ\\\\\\\/45zlnpwyuxCC6W+eUXn6wH+lILYe8SjbU0SpjyKnRfj4JWzgp55V1LEaOWoZqBBUTJQSleGQsGAFV3ZaUps5j0nu6\\\\\\\/mYWxoppZMtOsv4V\\\\\\\/XaK5OpvAPorfu56sasme7XKLHaXddK7Goj0QsNDa4zRxa0S3ikbL3MiOsZn2sUM8q+EvUfWMJeqKoNiKeC5bMHiHseALEYGYxt7fa6JSXZqqqYaxrA3bCpGg8yLqZzMdeVduf8vWf3+eShRjiefBEMChqmqdEyVxZktFTrRkxaShWLi95RLIT6s4kx+rTiR8QrmAcGI1xgeKb2IlMGrKHmwUtp4GSYyQG1ydjWcVR5NDAG3W8VRyCFOAuQB4DYmZtnfRwTtlccfYxn9WWrQX11IDFSVAuN34hVu256kS3ixluaGhEP8rPFm6fp3Q==\\\u0022,\\\u0022iv\\\u0022:\\\u0022doaMkfjTim0Hhc42\\\\\\\/RMw7g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00220f74029cc7923e75ecdd0533fc51809e60abf0a2ac15ba7e568d399277e8ee4c\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-491248 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genomic-prediction">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/11/3549" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Vxt3rNoFVZ3Vy3g2cGhM2WHM6BOkgGGYnQtCJwdQKYg" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/11/3549" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-zkUcsKehiEEC_wXQ5I6gAx-mKWzMWwqmcYRWRda9izM" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/11/3539" title="Identification and Characterization of Transcripts Regulated by Circadian Alternative Polyadenylation in Mouse Liver" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/11/3559" title="Discovery of QTL Alleles for Grain Shape in the Japan-MAGIC Rice Population Using Haplotype Information" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="491248" id="node491248" data-pisa="ggg;8/11/3549" data-pisa-master="ggg;g3.118.200673" data-apath="/ggg/8/11/3549.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic Prediction Using Multi-trait Weighted GBLUP Accounting for Heterogeneous Variances and Covariances Across the Genome</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-1010-683X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Emre Karaman</span>, <span class="highwire-citation-author" data-delta="1">Mogens S. Lund</span>, <span class="highwire-citation-author" data-delta="2">Mahlet T. Anche</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0001-9804-1529" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Luc Janss</span> and <span class="highwire-citation-author" data-delta="4">Guosheng Su</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3549-3558;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200673 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node491248" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Emre Karaman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Center for Quantitative Genetics and Genomics, Aarhus University, 8830 Tjele, Denmark</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Emre%2BKaraman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Karaman%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AEmre%2BKaraman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-1010-683X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Emre Karaman</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:emre@mbg.au.dk" class="" data-icon-position="" data-hide-link-title="0">emre@mbg.au.dk</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Mogens S. Lund </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Center for Quantitative Genetics and Genomics, Aarhus University, 8830 Tjele, Denmark</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mogens%2BS.%2BLund%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lund%20MS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMogens%2BS.%2BLund%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Mahlet T. Anche </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Center for Quantitative Genetics and Genomics, Aarhus University, 8830 Tjele, Denmark</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mahlet%2BT.%2BAnche%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Anche%20MT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMahlet%2BT.%2BAnche%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Luc Janss </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Center for Quantitative Genetics and Genomics, Aarhus University, 8830 Tjele, Denmark</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Luc%2BJanss%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Janss%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALuc%2BJanss%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-9804-1529" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Luc Janss</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Guosheng Su </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Center for Quantitative Genetics and Genomics, Aarhus University, 8830 Tjele, Denmark</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Guosheng%2BSu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Su%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGuosheng%2BSu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/8/11/3549" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:491248" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:491248/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/11/3549.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:491248" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:491248/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/11/3549.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:491248" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:491248/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/8/11/3549.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:491248" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:491248/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Implicit assumption of common (co)variance for all loci in multi-trait Genomic Best Linear Unbiased Prediction (GBLUP) results in a genomic relationship matrix (<strong>G</strong>) that is common to all traits. When this assumption is violated, Bayesian whole genome regression methods may be superior to GBLUP by accounting for unequal (co)variance for all loci or genome regions. This study aimed to develop a strategy to improve the accuracy of GBLUP for multi-trait genomic prediction, using (co)variance estimates of SNP effects from Bayesian whole genome regression methods. Five generations (G1-G5, test populations) of genotype data were available by simulations based on data of 2,200 Danish Holstein cows (G0, reference population). Two correlated traits with heritabilities of 0.1 or 0.4, and a genetic correlation of 0.45 were generated. First, SNP effects and breeding values were estimated using BayesAS method, assuming (co)variance was the same for SNPs within a genome region, and different between regions. Region size was set as one SNP, 100 SNPs, a whole chromosome or whole genome. Second, posterior (co)variances of SNP effects were used to weight SNPs in construction of <strong>G</strong> matrices. In general, region size of 100 SNPs led to highest prediction accuracies using BayesAS, and wGBLUP outperformed GBLUP at this region size. Our results suggest that when genetic architectures of traits favor Bayesian methods, the accuracy of multi-trait GBLUP can be as high as the Bayesian method if SNPs are weighted by the Bayesian posterior (co)variances.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/genomic-prediction" class="hw-term hw-article-keyword hw-article-keyword-genomic-prediction" rel="nofollow">Genomic prediction</a></li><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">GenPred</a></li><li class="kwd"><a href="/keyword/shared-data-resources" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resources" rel="nofollow">Shared Data Resources</a></li><li class="kwd"><a href="/keyword/genetic-architecture" class="hw-term hw-article-keyword hw-article-keyword-genetic-architecture" rel="nofollow">Genetic architecture</a></li><li class="kwd"><a href="/keyword/genomic-relationship-matrix" class="hw-term hw-article-keyword hw-article-keyword-genomic-relationship-matrix" rel="nofollow">Genomic relationship matrix</a></li><li class="kwd"><a href="/keyword/region-size" class="hw-term hw-article-keyword hw-article-keyword-region-size" rel="nofollow">Region size</a></li><li class="kwd"><a href="/keyword/snp-weight" class="hw-term hw-article-keyword hw-article-keyword-snp-weight" rel="nofollow">SNP weight</a></li></ul><p id="p-3">Genomic prediction methods mainly fall into one of the two categories <em>i.e.</em>, (i) methods assuming a variance specific to each SNP, or to a chromosomal region (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Meuwissen <em>et al.</em> 2001</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Sørensen <em>et al.</em> 2012</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Janss 2014</a>), and (ii) methods assuming a common variance for all SNPs (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Meuwissen <em>et al.</em> 2001</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">de los Campos <em>et al.</em> 2013</a>). For instance, in one extreme BayesA method it is assumed that each SNP follows a normal distribution with null mean and a locus-specific variance <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-1.gif"/></noscript></span></span> while in another extreme Ridge Regression method it is assumed that all SNPs have null means and a common variance <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-2.gif"/></noscript></span></span> It has been shown that this Ridge Regression can be written as a linear mixed model at individual level (taking the sum of SNP effects as additive genetic effect of an individual) with a genomic relationship matrix (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Nejati-Javaremi <em>et al.</em> 1997</a>), <strong>G</strong>, which is known as genomic best linear unbiased prediction (GBLUP) (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Habier <em>et al.</em> 2007</a>). The GBLUP method is popularly used for genomic evaluation in animal breeding programs, mainly due to its straightforward implementation using the existing computer softwares (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Tiezzi and Maltecca 2015</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Karaman <em>et al.</em> 2016</a>). Compared to Bayesian whole genome regression methods, GBLUP has also less computational demand particularly when the number of marker covariates greatly exceeds the number of observations, which is the case in most genome-wide analysis. Moreover, many important traits in animals are complex in nature, and are controlled by a large number of small effect genes distributed across the entire genome, favoring the infinitesimal model (<a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">Meuwissen <em>et al.</em> 2001</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Hayes and Goddard 2001</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Hayes <em>et al.</em> 2009</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Meuwissen and Goddard 2010</a>). For some traits, the distribution of underlying genes violates the infinitesimal model assumption of large number of genes with small effect. Thus, methods assuming SNP specific or region specific (co)variances can lead to higher accuracy of prediction than GBLUP, by accounting for the relative importance of genomic regions (<a id="xref-ref-32-4" class="xref-bibr" href="#ref-32">Meuwissen <em>et al.</em> 2001</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">VanRaden <em>et al.</em> 2009</a>).</p><p id="p-4"><em>A priori</em> assumption of common variance for all loci in GBLUP results in a <strong>G</strong> matrix that is common to all traits. However, any trait deviates from the infinitesimal model to some degree, and different traits can be controlled by different sets of genes and different chromosomal regions (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Lipkin <em>et al.</em> 2008</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Zhang <em>et al.</em> 2010</a>). Hence, it needs a <strong>G</strong> matrix that can reflect the genetic architecture of the traits of interest for improving genomic prediction. <a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Zhang <em>et al.</em> (2010)</a>, for the first time, reported that the estimates from Bayesian regression methods assuming a specific variance for each SNP can be implemented in GBLUP to improve genomic prediction accuracy. Computing a trait-specific genomic relationship matrix by weighting SNPs using posterior variances from BayesB method, the authors reported higher predictive abilities than traditional GBLUP method (<a id="xref-ref-49-3" class="xref-bibr" href="#ref-49">Zhang <em>et al.</em> 2010</a>). Following their work, many others investigated the use of various weights for GBLUP in single-trait evaluation, either weighting SNPs individually or assigning a common weight to adjacent SNPs (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Su <em>et al.</em> 2014</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Calus <em>et al.</em> 2014</a>; <a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Tiezzi and Maltecca 2015</a>).</p><p id="p-5">A great majority of the genomic selection studies (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Calus and Veerkamp 2007</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Legarra <em>et al.</em> 2008</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Lorenzana and Bernardo 2009</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Daetwyler <em>et al.</em> 2010a</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Liu <em>et al.</em> 2011</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Gao <em>et al.</em> 2013</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Wolc <em>et al.</em> 2013</a>; <a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Su <em>et al.</em> 2014</a>) has focused on single-trait breeding value estimation. However, multi-trait methods are expected to yield more accurate predictions than single-trait methods as in traditional BLUP. Simulations have shown that multi-trait genomic prediction can lead to a considerable increase in genomic prediction accuracy (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Calus and Veerkamp 2011</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Jia and Jannink 2012</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Guo <em>et al.</em> 2014</a>). These studies either implemented Bayesian methods that have a higher computational demand especially for large scale of data, or used GBLUP with a <strong>G</strong> matrix that is common to all traits. An alternative estimation procedure can be to combine important features of both methods such that any deviation from the infinitesimal model are accounted for without compromising the simplicity in application. In a bivariate model, for instance, such a procedure should account for heterogeneous covariances between SNP effects on two traits as well as heterogeneous variances of SNP effects on each trait. In the framework of bivariate GBLUP, this immediately requires that genomic relationship matrices are able to account for heterogeneous variances and and covariances accross the genome.</p><p id="p-6">The aim of this study is to (i) extend the weighted single-trait GBLUP methodology to multi-trait case, (ii) compare the performance of proposed methodology with unweighted single and multi-trait GBLUP methods, and (iii) investigate the effect of two weighting strategies (<em>i.e.</em>, single or group SNP weighting).</p><div class="section materials-methods" id="sec-1"><h2 class="">Material and Methods</h2><div id="sec-2" class="subsection"><h3>Data Sets</h3><p id="p-7">Five generations of genotype data were simulated based on 50K haplotype data of 2,200 animals from Danish Holstein population. Only the SNPs (11,154) on first five chromosomes were used. A fixed ratio of male:female, 1:10, was assumed throughout the simulated generations. Thus, at each generation, 200 and 2,000 animals were assumed to be males and females, respectively. Each male was randomly mated with 10 females, and one mating per sire was replicated so as to retain the population size at 2,200 for each generation. The number of recombinations on each chromosome was determined using a random variable drawn from a Poisson distribution, under the assumption that the length of a chromosome in the Morgan’s unit in the linkage map is the lambda parameter (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Yamamoto <em>et al.</em> 2016</a>). Mutation was not considered in the simulations. Animals in the base population (hereinafter referred to as G0) were used to form a reference population, while animals in the simulated generations (hereinafter referred to as G1-G5) were used to form the test populations.</p><p id="p-8">Some SNPs were assigned to QTL in the following way. Each chromosome was divided into windows of 1 million bp, yielding 159, 139, 127, 121 and 125 bins for chromosomes 1-5, respectively. Then, a bin was randomly selected, and a random number, <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-3.gif"/></noscript></span></span> was drawn from a uniform distribution <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-4.gif"/></noscript></span></span> A SNP was randomly selected and designated as QTL among those SNPs, if any, which met the condition of <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-5.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-6.gif"/></noscript></span></span> is the minor allele frequency of the SNP. Each bin was allowed to include one QTL at maximum. The restriction in MAF for QTL was based on the assumption that QTL in general have relatively low MAF. Two traits were considered, and total number of QTL was set at 200 or 500, which were excluded from the final data set of SNP. Averaged over the replicates, MAFs of the QTL and SNPs were 0.15 and 0.26, respectively.</p><p id="p-9">All QTL were assigned into three groups according to their causal relationships with the traits. This was done by assuming a percentage of the total QTL (82%) had pleiotropic effects on two traits, while the rest of the QTL had effects on either of the traits. Two correlated gamma variables (<em>i.e.</em>, QTL effects) with marginal distributions of <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-7.gif"/></noscript></span></span> were simulated for the pleiotropic QTL (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Dvorkin 2012</a>). It was assumed that the pleiotropic QTL had effects either on the same or on the opposite direction for the two traits. The 78% of those QTL were assigned to a correlation between effects on two traits of 0.90, and 22% of -0.90 randomly, to accomplish a genetic correlation of about 0.45. The effects of trait-specific QTL were sampled from a Gamma distribution, <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-8.gif"/></noscript></span></span> and were assigned a positive or negative sign at random.</p><p id="p-10">Phenotypic values of the two traits were simulated to have heritabilities of 0.1 and 0.4, which represents low (<em>L</em>) and high (<em>H</em>) heritability traits, respectively. The phenotypic values of individual <em>i</em> on two traits, <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-9.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-10.gif"/></noscript></span></span> adjusted for fixed effects can be given as:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-11.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-12.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-13.gif"/></noscript></span></span> are the breeding values of individual <em>i</em> for traits <em>L</em> and <em>H</em>, respectively. <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-14.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-15.gif"/></noscript></span></span> are the vectors of the genotypes of QTL that determine either trait <em>L</em> or <em>H</em>, and <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-16.gif"/></noscript></span></span> is the genotype vector of QTL with effect on both traits, for individual <em>i</em>. The <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-17.gif"/></noscript></span></span>s and <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-18.gif"/></noscript></span></span> are vectors of QTL effects for the corresponding three groups of QTL, and <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-19.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-20.gif"/></noscript></span></span> are the random residual effects. Random residual effects were sampled from <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-21.gif"/></noscript></span></span> where the sizes of <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-22.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-23.gif"/></noscript></span></span> were determined according to heritabilities of 0.1 and 0.4, respectively. All animals had genotypes and phenotypes on both traits. In total, 10 replicates were generated. Averaged over the replicates, the heritabilities for traits <em>L</em> and <em>H</em> were 0.1 and 0.4, respectively, and the genetic correlation between the traits was 0.47.</p></div><div id="sec-3" class="subsection"><h3>Models and Methods</h3><div id="sec-4" class="subsection"><h4>Definition of a basic multi-trait model:</h4><p id="p-11">In practice, QTL are unobserved, and breeding value estimates are based on markers instead of QTL. A multi-trait model with marker effects can be written as<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-24.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>where y is the vector of phenotypes corrected for the effects other than genetic effect, X is the matrix of genotypes (centered) for <em>k</em> markers, β is the vector of marker effects, and e is the vector of random residual effects. Equation (1) can be explicitly written as follows.<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-25.gif"/></noscript></span></span>In the case that two traits are measured on the same animals, marker genotype matrices <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-26.gif"/></noscript></span></span> and these matrices will be denoted as <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-27.gif"/></noscript></span></span> hereinafter, to simplify the demonstration. A typical assumption for residuals, <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-28.gif"/></noscript></span></span> is <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-29.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-30.gif"/></noscript></span></span>. Assuming all the genetic variance is captured by markers, vectors of breeding values of the animals for traits <em>L</em> and <em>H</em> are <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-31.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-32.gif"/></noscript></span></span> respectively.</p></div><div id="sec-5" class="subsection"><h4>Multi-trait genomic BLUP:</h4><p id="p-12">We will first start with a multi-trait BLUP model assuming independence of SNP effects on different loci, and homogeneous SNP (co)variances across the genome. We will then make a connection between that model and the traditional BLUP model based on genomic relationship matrix (GBLUP). Finally, we will relax the assumption of homogeneous SNP (co)variances, and introduce a multi-trait GBLUP model where SNP (co)variances across the genome are assumed to be heterogeneous (wGBLUP).</p><p id="p-13">When constant (co)variances are assumed for all the SNPs, <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-33.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-34.gif"/></noscript></span></span>. BLUP of breeding values, <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-35.gif"/></noscript></span></span> are<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-36.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span>where <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-37.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-38.gif"/></noscript></span></span> Assuming Hardy-Weinberg equilibrium, and homogeneous (co)variance for SNPs, total genetic variance and covariances are (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Morota and Gianola 2014</a>) <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-39.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-40.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-41.gif"/></noscript></span></span> respectively, where <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-42.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-43.gif"/></noscript></span></span> are the allele frequencies of 2<sup>nd</sup> and 1<sup>st</sup> alleles at loci <em>j</em>. Using the fact that<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-44.gif"/></noscript></span></span>and in turn replacing respective parameters in equation (2), we get<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-45.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-46.gif"/></noscript></span></span> and <strong>D</strong> is a diagonal matrix with <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-47.gif"/></noscript></span></span> (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">VanRaden 2008</a>). Hence, only a single relationship matrix needs to be computed, and<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-48.gif"/></noscript></span></span>Hereinafter, this multi-trait BLUP model using a single relationship matrix will be referred to as multi-trait GBLUP.</p><p id="p-14">Assume that each SNP has a specific (co)variance, and that <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-49.gif"/></noscript></span></span> with <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-50.gif"/></noscript></span></span>. Here, <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-51.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-52.gif"/></noscript></span></span> are diagonal matrices of SNP variances for traits <em>L</em> and <em>H</em>, respectively, and <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-53.gif"/></noscript></span></span> is a diagonal matrix of the covariances for the SNP effects on the traits. BLUP of breeding values are<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-54.gif"/></noscript></span></span>with<span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-55.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-56.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-57.gif"/></noscript></span></span> are entries for the <em>j</em>’th diagonal of corresponding D matrices, <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-58.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-59.gif"/></noscript></span></span> are the SNP variances for traits <em>L and H</em>, respectively, and <span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-60.gif"/></noscript></span></span> is the covariance of the <em>j</em>th SNP between traits. It is more convenient to write<span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-61.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-62.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-63.gif"/></noscript></span></span> are the genomic relationship matrices that account for the heterogeneous variances, while <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-64.gif"/></noscript></span></span> is the genomic relationship matrix that accounts for heterogeneous covariances across the genome. Hereinafter multi-trait BLUP model based on these genomic relationship matrices will be referred to as multi-trait weighted GBLUP (wGBLUP).</p><p id="p-15">We also used single-trait weighted and unweighted GBLUP to predict breeding values in this study. In the single-trait analysis, the distribution of breeding values were assumed to follow <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-65.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-66.gif"/></noscript></span></span> in weighted GBLUP, and <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-67.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-68.gif"/></noscript></span></span> in unweighted GBLUP.</p></div><div id="sec-6" class="subsection"><h4>Multi-trait BayesAS:</h4><p id="p-16">The multi-trait BayesAS method is an extension of the method proposed by <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Janss (2014)</a>. The assumption in this method is that SNPs in a genomic region have the same genomic (co)variance, but SNPs in different genomic regions have different genomic (co)variances (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Li <em>et al.</em> 2017</a>). The model can be written as<span class="disp-formula" id="disp-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-69.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-70.gif"/></noscript></span></span> is the vector of phenotypes, <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-71.gif"/></noscript></span></span> is the overall mean, <span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-72.gif"/></noscript></span></span> is the matrix of marker genotypes for region <em>s</em>, <span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-73.gif"/></noscript></span></span> is the vector of marker effects of region <em>s</em> for trait <em>t</em>, <span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-74.gif"/></noscript></span></span> is the vector of random residual effects. For the region <em>s</em>, SNP effects across traits are correlated and are formulated by the following hierarchical model<span class="disp-formula" id="disp-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-75.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-75.gif"/></noscript></span><span class="disp-formula-label">(3)</span></span>In the hierarchical model, a SNP effect is decomposed to three components. The vectors <span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-76.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-76.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-77.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-77.gif"/></noscript></span></span> are subsets of latent variables <span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-78.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-78.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-79.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-79.gif"/></noscript></span></span> respectively, and are common to all traits in the analysis. The vectors <span class="inline-formula" id="inline-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-80.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-80.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-81.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-81.gif"/></noscript></span></span> are vectors with SNP effects transformed to an eigen-vector space (<a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">Janss 2014</a>). The purpose of the latent vectors <span class="inline-formula" id="inline-formula-70"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-82.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-82.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-71"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-83.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-83.gif"/></noscript></span></span> is to model covariances between traits. Regression coefficients <span class="inline-formula" id="inline-formula-72"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-84.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-84.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-73"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-85.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-85.gif"/></noscript></span></span> determine the sizes of the covariances. Hence, average covariance between SNP effects are modeled through <span class="inline-formula" id="inline-formula-74"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-86.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-86.gif"/></noscript></span></span> while the deviation of region <em>s</em> from this common covariance is modeled through <span class="inline-formula" id="inline-formula-75"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-87.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-87.gif"/></noscript></span></span> In BayesAS, the elements of the vectors <span class="inline-formula" id="inline-formula-76"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-88.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-88.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-77"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-89.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-89.gif"/></noscript></span></span> are estimated in a hierarchical model specification, where the SNP effects on each trait are taken as a known “response”, the <span class="inline-formula" id="inline-formula-78"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-90.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-90.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-79"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-91.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-91.gif"/></noscript></span></span> parameters are taken as known scaling parameters, and <span class="inline-formula" id="inline-formula-80"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-92.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-92.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-81"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-93.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-93.gif"/></noscript></span></span> are updated by a regular mixed model equation. Then, taking <span class="inline-formula" id="inline-formula-82"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-94.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-94.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-83"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-95.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-95.gif"/></noscript></span></span> as known, <span class="inline-formula" id="inline-formula-84"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-96.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-96.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-85"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-97.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-97.gif"/></noscript></span></span> are updated. Vector of residual SNP effects, <span class="inline-formula" id="inline-formula-86"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-98.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-98.gif"/></noscript></span></span> consists of the SNP effects, which are uncorrelated across traits. For a SNP having effect only on one trait, the sum of the two regression terms <span class="inline-formula" id="inline-formula-87"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-99.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-99.gif"/></noscript></span></span> is expected to be null, and for a SNP having the same effect on both traits, <span class="inline-formula" id="inline-formula-88"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-100.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-100.gif"/></noscript></span></span> is expected to be null. We assumed following priors for the parameters of the models<span class="disp-formula" id="disp-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-101.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-101.gif"/></noscript></span></span>Variance and covariance for each SNP in each region can be computed as <span class="inline-formula" id="inline-formula-89"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-102.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-102.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-90"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-103.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-103.gif"/></noscript></span></span> (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Li <em>et al.</em> 2017</a>). Single-trait BayesAS was also used, where equation (3) reduces to <span class="inline-formula" id="inline-formula-91"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-104.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-104.gif"/></noscript></span></span> and the variances for each SNP in each region becomes <span class="inline-formula" id="inline-formula-92"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-105.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-105.gif"/></noscript></span></span> Priors for single-trait BayesAS model were as defined for multi-trait BayesAS, with <span class="inline-formula" id="inline-formula-93"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-106.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-106.gif"/></noscript></span></span> set at the value from multi-trait analysis.</p></div></div><div id="sec-7" class="subsection"><h3>Statistical Analysis</h3><p id="p-17">We aimed to compare the prediction accuracies of BayesAS, GBLUP and wGBLUP in single and multi-trait genomic prediction. Four different sizes of genomic regions (one SNP, 100 SNPs, one chromosome and whole genome) were considered using BayesAS. Posterior (co)variance components of the traits from BayesAS models with different region sizes were used in GBLUP, and posterior (co)variances of SNPs were used to calculate weights for wGBLUP. It is worth noting that (co)variance components of traits may vary among the BayesAS analyses with different region sizes to some extent (<a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">Li <em>et al.</em> 2017</a>), which in turn may result in accuracies from GBLUP analysis to vary within single and multi-trait analysis.</p><p id="p-18">The analysis of single and multi-trait BayesAS model was carried out using the BAYZ software (<a href="http://www.bayz.biz">www.bayz.biz</a>). Chain length consisted of 20,000 cycles, and the first 5,000 cycles were considered as burn-in. Thinning interval was set to 10, and in total 1,500 samples were saved for the posterior analysis. Mean value of the posterior samples was considered as the estimate of each parameter. GBLUP and wGBLUP results were obtained by solving mixed model equations in R (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">R Core Team 2017</a>).</p><p id="p-19">Accuracy of prediction from different methods was calculated as the correlation between true and estimated breeding values of animals in test populations. Results presented here were the average over 10 replicates.</p><p id="p-20">Single and multi-trait BayesAS models assuming different region sizes were compared for each trait, separately. Prediction accuracies for all methods were compared for each trait and each scenario of region size. Those comparisons were performed by two-sided paired <em>t</em>-tests, for which accuracies were paired across each replicate for the same test population, and based on a <em>p</em>-value of 0.05 with Bonferroni correction.</p></div><div id="sec-8" class="subsection"><h3>Data Availability</h3><p id="p-21">Supplemental material available at Figshare: <a href="https://figshare.com/s/1a1886f38d25b6f389ff">https://figshare.com/s/1a1886f38d25b6f389ff</a>. Files at this URL are, real and simulated genotypes, and File S1, which includes Tables S1 and S2 for accuracies of generations 2-5 under two QTL scenarios. The genotypes and the methodology described previously are sufficient to reproduce the results of this study. The authors affirm that all data necessary for confirming the conclusions of the article are present within the article, figures, and tables.</p></div></div><div class="section results" id="sec-9"><h2 class="">Results</h2><div id="sec-10" class="subsection"><h3>Genomic prediction using BayesAS</h3><p id="p-22">Prediction accuracies from single and multi-trait BayesAS models for different region sizes are presented in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Tables 1</a>-<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">2</a>, for test animals in G1.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/491412/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/491412/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Accuracies of genomic prediction for test animals in generation 1 using different methods with varying region sizes, for 200 QTL scenario</span><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/491402/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/491402/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Accuracies of genomic prediction for test animals in generation 1 using different methods with varying region sizes, for 500 QTL scenario</span><div class="sb-div caption-clear"></div></div></div><p id="p-31">Accuracies from multi-trait analysis were significantly higher than their single-trait counterparts regardless of the region size or the number of QTL, for trait <em>L</em>. For trait <em>H</em>, on the other hand, accuracies from single and multi-trait analyses were similar at all region sizes, except for the region size of whole genome in 500 QTL scenario.</p><p id="p-32">Region size of 100 SNPs resulted in the highest accuracies in single and multi-trait analyses, for both traits. In the scenario of 500 QTL, accuracy from region size of 100 SNPs was not significantly higher than that from region size of 1 SNP, but than from one chromosome or whole genome, for trait <em>L</em> using a single-trait model. For this trait, accuracy for the region size of 100 SNPs was more than 2 percentage points higher than those for other region sizes using a multi-trait model, however, it was not significant.</p></div><div id="sec-11" class="subsection"><h3>Weighted GBLUP using SNP (co)variances From BayesAS</h3><p id="p-33">After fitting BayesAS models with different region sizes, posterior (co)variance components of the traits from these models were used in GBLUP, and posterior (co)variances of SNPs were used to calculate weights for wGBLUP. Accuracies from GBLUP and wGBLUP methods are presented in <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Tables 1</a> and <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">2</a> along with the accuracies from BayesAS methods, for test animals in G1. It is worth noting that what varied among the different region sizes for single or multi-trait GBLUP analyses were the estimates of (co)variance components of traits due to using BayesAS with different region sizes, while the relationship matrices remained unchanged.</p><p id="p-34">Similar to BayesAS model, in general, accuracies from multi-trait wGBLUP and GBLUP analysis were significantly higher than their single-trait counterparts for trait <em>L</em>. For trait <em>H</em>, on the other hand, accuracies from single and multi-trait wGBLUP analyses were generally similar in both QTL scenarios.</p><p id="p-35">In the scenario of 200 QTL, accuracies for trait <em>L</em> from wGBLUP were significantly higher than those from GBLUP, when the weighing factors were the (co)variances from BayesAS with region size of 100 SNPs, for both single and multi-trait analysis. In the scenario of 500 QTL, the accuracy for trait <em>L</em> from wGBLUP was higher than GBLUP (2.5 percentage points) using a multi-trait model with region size of 100 SNPs, however, it was not significant. There was no significant difference between wGBLUP and GBLUP, when the region was one SNP, one chromosome or whole genome for trait <em>L</em>, regardless of the number of QTL.</p><p id="p-36">For trait <em>H</em>, prediction accuracies from wGBLUP models when weights were obtained from BayesAS with region sizes of 100 SNPs, were significantly higher than those from corresponding GBLUP models, regardless of the number of QTL. Weighted GBLUP was similar or superior to traditional GBLUP for region sizes of one SNP, one chromosome or whole genome, for trait <em>H</em>.</p><p id="p-37">Although not always statistically significant, analysis using (co)variance estimates from BayesAS in construction of <strong>G</strong> matrices resulted in higher accuracies than GBLUP, when the accuracies of corresponding BayesAS model was higher than those of GBLUP. Moreover, prediction accuracies of breeding values from wGBLUP were generally as high as those from BayesAS models.</p></div><div id="sec-12" class="subsection"><h3>Change in accuracy Over generations</h3><p id="p-38"><a id="xref-fig-1-1" class="xref-fig" href="#F1">Figures 1</a> and <a id="xref-fig-2-1" class="xref-fig" href="#F2">2</a> show the prediction accuracy of breeding values using wGBLUP method for five consecutive generations, when the total number of QTL were 200 and 500, respectively. It is worth noting that wGBLUP model with region size of whole genome is consistent with GBLUP, where all SNPs are implicitly assumed to have the same (co)variance. For all four region sizes considered, accuracy decreased with generations regardless of the number of QTL. The rate of decay in accuracy for a region size of 100 SNPs was slightly lower than that for other region sizes when the total number of QTL was 200, particularly in multi-trait setting for trait <em>L</em>. When the number of QTL was 200, substantial accuracy was still retained with a region size of 100 SNPs using the multi-trait wGBLUP method, even after five generations. Accuracies from 500 QTL scenario (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>) decayed slower compared to 200 QTL scenario (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>), when the region size was whole genome. The accuracies of (single or multi-trait) BayesAS and wGBLUP were very similar in all five generations (Tables S1 and S2).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/11/3549/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Change in prediction accuracy for weighted GBLUP over generations with varying regions sizes of genome regions, for 200 QTL scenario. L and H: low (0.1) and high (0.4) heritability traits, respectively; different colors represent different region sizes; Chr: chromosome; WG: whole genome (or equivalently, GBLUP)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1259623982" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Change in prediction accuracy for weighted GBLUP over generations with varying regions sizes of genome regions, for 200 QTL scenario. &lt;em&gt;L&lt;/em&gt; and &lt;em&gt;H&lt;/em&gt;: low (0.1) and high (0.4) heritability traits, respectively; different colors represent different region sizes; Chr: chromosome; WG: whole genome (or equivalently, GBLUP).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/11/3549/F1.medium.gif" width="440" height="328"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/8/11/3549/F1.medium.gif" width="440" height="328"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/11/3549/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/11/3549/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491376" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-39" class="first-child">Change in prediction accuracy for weighted GBLUP over generations with varying regions sizes of genome regions, for 200 QTL scenario. <em>L</em> and <em>H</em>: low (0.1) and high (0.4) heritability traits, respectively; different colors represent different region sizes; Chr: chromosome; WG: whole genome (or equivalently, GBLUP).</p><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/11/3549/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Change in prediction accuracy for weighted GBLUP over generations with varying regions sizes of genome regions, for 500 QTL scenario. L and H: low (0.1) and high (0.4) heritability traits, respectively; different colors represent different region sizes; Chr: chromosome; WG: whole genome (or equivalently, GBLUP)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1259623982" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Change in prediction accuracy for weighted GBLUP over generations with varying regions sizes of genome regions, for 500 QTL scenario. &lt;em&gt;L&lt;/em&gt; and &lt;em&gt;H&lt;/em&gt;: low (0.1) and high (0.4) heritability traits, respectively; different colors represent different region sizes; Chr: chromosome; WG: whole genome (or equivalently, GBLUP).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/11/3549/F2.medium.gif" width="440" height="328"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/8/11/3549/F2.medium.gif" width="440" height="328"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/11/3549/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/11/3549/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491436" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-40" class="first-child">Change in prediction accuracy for weighted GBLUP over generations with varying regions sizes of genome regions, for 500 QTL scenario. <em>L</em> and <em>H</em>: low (0.1) and high (0.4) heritability traits, respectively; different colors represent different region sizes; Chr: chromosome; WG: whole genome (or equivalently, GBLUP).</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-13"><h2 class="">Discussion</h2><div id="sec-14" class="subsection"><h3>Region size in BayesAS</h3><p id="p-41">A wide range of statistical methods is available for single-trait genomic prediction, which mainly differ in the assumptions on SNP effects (<a id="xref-ref-32-5" class="xref-bibr" href="#ref-32">Meuwissen <em>et al.</em> 2001</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Gianola <em>et al.</em> 2009</a>; <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">de los Campos <em>et al.</em> 2013</a>). Among those methods, an extreme is the BayesA method, where prior distribution of each SNP effect is assumed to be normal with a SNP specific variance, <span class="inline-formula" id="inline-formula-94"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-107.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-107.gif"/></noscript></span></span> Those variances are further assumed to follow a scaled inverted chi-square distribution with some scale and degree of freedom (<em>df</em>) parameters. This was criticized by <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Gianola <em>et al.</em> (2009)</a> as not following Bayesian learning, because posterior SNP variances have only one additional <em>df</em> compared to their prior. This can be overcome by assigning a common variance for a subset of SNPs, and therefore, posterior distribution of each SNP variance (within a group consisting of <span class="inline-formula" id="inline-formula-95"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-108.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-108.gif"/></noscript></span></span> SNPs) will have <span class="inline-formula" id="inline-formula-96"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-109.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-109.gif"/></noscript></span></span> <em>df</em> in addition to prior <em>df</em> (<a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">Gianola <em>et al.</em> 2009</a>). These subsets can be achieved by grouping SNPs based on fixed length of genomic region, or fixed number of adjacent SNPs in each subset. This is reasonable, because the SNPs in a region likely inherit together, and also likely to be in linkage disequilibrium (LD) with the same QTL (<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Sørensen <em>et al.</em> 2012</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Gebreyesus <em>et al.</em> 2017</a>).</p><p id="p-42">Some of the existing single-trait Bayesian whole genome regression models have been extended to accommodate more than one trait at a time (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Calus and Veerkamp 2011</a>; <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Jia and Jannink 2012</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Hayashi and Iwata 2013</a>). An issue with those models is that when two traits are considered simultaneously, the models are restricted to two situations for a locus <em>i.e.</em>, the locus has effect on both traits, or none of them. <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Cheng <em>et al.</em> (2018)</a> developed a set of models that can accommodate all possible situations, that is, in terms of two traits, their models can handle also with the situation where a locus has effect on one trait, but not on the other. With their new multi-trait BayesCΠ model (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Cheng <em>et al.</em> 2018</a>), they reported the same as or higher accuracies than the model considering two situations only (<a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">Jia and Jannink 2012</a>), using real and simulated data sets.</p><p id="p-43">The Bayesian method used in this study, BayesAS, also assumes that a locus simultaneously affects all the traits, but allows the inclusion of priors for genomic regions of any size. This way, the method exploits the genomic regions that show correlations between traits, which deviate from the genome-wide correlation (<a id="xref-ref-27-4" class="xref-bibr" href="#ref-27">Li <em>et al.</em> 2017</a>). Assuming a common (co)variance for a group of adjacent SNPs is not as flexible as assuming a (co)variance specific to each SNP, but it increases the power in estimation of SNP (co)variances. Taking advantage of grouping adjacent SNPs, therefore, might be critical to achieving a high prediction accuracy, by better exploiting the local deviations from the average genome-wide (co)variance.</p><p id="p-44">Special cases of BayesAS method occur when the region size is one SNP or whole genome, which has similar assumptions to BayesA or GBLUP, respectively. Because our primary aim was to develop a methodology to improve accuracy of multi-trait genomic prediction using GBLUP via weighted <strong>G</strong> matrices, we only investigated a limited number of region sizes (one SNP, 100 SNPs, one chromosome or whole genome). Our results demonstrated the potential of improving genomic prediction accuracy by assigning priors to genomic regions for both single and multi-trait analysis via BayesAS method. Highest accuracies were reached with BayesAS when the region size was 100 SNPs, regardless of the number of QTL, however, some of those for trait <em>L</em> in 500 QTL scenario were not significantly higher than all other region sizes considered here (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Tables 1</a> and <a id="xref-table-wrap-2-3" class="xref-table" href="#T2">2</a>).</p><p id="p-45"><a id="xref-ref-27-5" class="xref-bibr" href="#ref-27">Li <em>et al.</em> (2017)</a> estimated genetic parameters for milk, fat and protein yields of Chinese and Nordic Holstein cattle using BayesAS model, where the same trait in these countries was treated as different traits, and analyzed using a bivariate model. They reported that region size of 100 SNPs resulted in similar estimates of variance components as region sizes of 200 or 400, but more accurate estimates than region size of 50, using a 50K SNP panel. Fitting a Bayesian multi-trait mixed model that uses latent variables to fit (co)variance structures, <a id="xref-ref-36-3" class="xref-bibr" href="#ref-36">Sørensen <em>et al.</em> (2012)</a> estimated genomic (co)variances for six mastitis resistance related traits in Danish cows, for some region sizes. Their results using half-overlapping region size of 100 SNPs on BTA19 indicated the existence of different (co)variance patterns for different traits. <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Brøndum <em>et al.</em> (2012)</a> modified the single-trait BayesR model to allow incorporating prior information on genomic regions, and referred to it as BayesRS in an across-breed genomic prediction study for protein, fat, and milk yield. They reported that, among varying sizes of genome regions they examined, the highest accuracies were obtained with the region size of 100 SNPs. However, the SNP panel used in (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Brøndum <em>et al.</em> 2012</a>) was dense consisting of (before data editing) 777K SNPs, and therefore, the region size of 100 SNPs corresponds to a region size of roughly 5-10 SNPs in the 50K panel.</p><p id="p-46">In a recent study, <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Gebreyesus <em>et al.</em> (2017)</a> used the model for multi-trait genomic prediction, and reported substantial improvements in reliabilities using a region size of 100 SNPs compared to the reliabilities from a bivariate GBLUP model, for most of the milk protein composition traits in Danish Holstein cattle. The extent of LD is highly variable in different populations, and it also varies with respect to SNP density (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Goddard and Hayes 2009</a>). The decision of optimal region size is, therefore, crucial to obtain highest accuracy of genomic prediction (<a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Gebreyesus <em>et al.</em> 2017</a>).</p></div><div id="sec-15" class="subsection"><h3>Weighted GBLUP using SNP (co)variances From BayesAS</h3><p id="p-47">Computation of relationship matrices by weighting SNPs was first proposed by <a id="xref-ref-49-4" class="xref-bibr" href="#ref-49">Zhang <em>et al.</em> (2010)</a>, and higher accuracies were reported for wGBLUP than for GBLUP, using single-trait analysis. Following their work, the use of weights to compute genomic relationships has been studied extensively, and many different weights such as <em>p</em>-values from GWAS studies, genetic variance at a locus etc. were used (<a id="xref-ref-37-3" class="xref-bibr" href="#ref-37">Su <em>et al.</em> 2014</a>; <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Calus <em>et al.</em> 2014</a>; <a id="xref-ref-39-3" class="xref-bibr" href="#ref-39">Tiezzi and Maltecca 2015</a>).</p><p id="p-48">In the applications of weighted GBLUP, SNP specific weights were either used to weight SNPs individually, or averaged (or summed) over adjacent SNPs to assign a common weight to a set of SNPs (<a id="xref-ref-37-4" class="xref-bibr" href="#ref-37">Su <em>et al.</em> 2014</a>; <a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">Calus <em>et al.</em> 2014</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Zhang <em>et al.</em> 2016</a>). <a id="xref-ref-37-5" class="xref-bibr" href="#ref-37">Su <em>et al.</em> (2014)</a> reported that using the mean variance of 30-SNP window as weights in GBLUP improved the reliabilities up to one percent for four production traits and mastitis, compared with single SNP weighting. <a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Zhang <em>et al.</em> (2016)</a> used an iterative weighting procedure, where the SNP variances estimated in the previous iteration were used to weight SNPs in the current iteration when building a <strong>G</strong> matrix for single-step GBLUP (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Christensen and Lund 2010</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Aguilar <em>et al.</em> 2010</a>). Using a common weight for 20-SNP window that sums or averages the individual SNP variances improved the accuracy of prediction.</p><p id="p-49">An alternative, and arguably a more straightforward approach is to assume a variance common to all SNPs in a genomic region using a Bayesian method, and then using those variances as weights to compute the <strong>G</strong> matrix for genomic prediction with GBLUP. The Bayesian whole genome regression method used here, BayesAS, allows assigning prior distributions for genomic regions including any number of SNPs (<a id="xref-ref-27-6" class="xref-bibr" href="#ref-27">Li <em>et al.</em> 2017</a>), and provides (co)variances for those regions which can directly be used to weight SNPs when building the genomic relationship matrices for multi-trait genomic prediction. The gain from a weighted GBLUP analysis depends on whether the Bayesian method used to derive weights is superior to GBLUP for traits under investigation. As discussed previously, this is highly related to the the region size when using BayesAS.</p><p id="p-50">Predictions from single or multi-trait wGBLUP were not found to be superior to their GBLUP counterparts with some exceptions for high heritability trait, when each SNP assumed to have its own (co)variance using BayesAS (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Tables 1</a> and <a id="xref-table-wrap-2-4" class="xref-table" href="#T2">2</a>). Taking full advantage of the Bayesian whole genome regression methods is much more difficult for low heritability traits, because the power to detect the right SNPs associated with the QTL is weak for low heritability trait. When weights were obtained from BayesAS with a region size of 100 SNPs, accuracies for single and multi-trait wGBLUP models were significantly higher than those for the corresponding GBLUP models, except for trait <em>L</em> when the number of QTL was 500. In fact, it is expected that there will be no advantage for Bayesian whole genome regression, and therefore, for wGBLUP over GBLUP if the simulated QTL more closely fit an infinitesimal model, unless the data are large enough (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Hayes <em>et al.</em> 2009</a>; <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Karaman <em>et al.</em> 2016</a>; <a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Cheng <em>et al.</em> 2018</a>). In general, taking one chromosome or whole genome as one region did not yield significantly higher prediction accuracies for wGBLUP than those for GBLUP. This is due to the fact that, as region size is larger than an optimum level, the advantage of Bayesian methods starts to diminish because the assumption on (co)variance approaches to that in GBLUP.</p><p id="p-51">The accuracies for wGBLUP were higher up to 4.3 percentage points for 200 QTL scenario than for 500 QTL scenario, when the region size was 100 SNPs. On the other hand, the accuracies of predictions using GBLUP showed little difference between the QTL scenarios. This is mainly due to the fact that GBLUP rely on average relationships among animals across the genome, and is less sensitive to the genetic architecture of traits (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Daetwyler <em>et al.</em> 2010b</a>; <a id="xref-ref-39-4" class="xref-bibr" href="#ref-39">Tiezzi and Maltecca 2015</a>).</p><p id="p-52">Bayesian whole genome regression methods do not include genomic relationship matrices (<em>i.e.</em>, <span class="inline-formula" id="inline-formula-97"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-110.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-110.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-98"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-111.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-111.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-99"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-112.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3549/embed/mml-math-112.gif"/></noscript></span></span>) explicitly, however, these “weighted” genomic relationship matrices can be implicitly estimated during the MCMC procedure (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Fernando and Gianola 2018</a>). In multi-trait BayesA, all loci contribute to the genomic relationships, while in multi-trait BayesB, only some proportions of the total loci contribute to the genomic relationships in each cycle (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Fernando and Gianola 2018</a>). Generally speaking, under some assumptions, any Bayesian alphabet method can lead to its equivalent weighted GBLUP counterpart. Therefore, one would obtain similar accuracies for BayesB and wGBLUP, if the SNP (co)variances from BayesB were used to weight SNPs when constructing genomic relationship matrices.</p><p id="p-53">In almost all cases, GBLUP was able to reach the accuracy as high as that of BayesAS models, by using posterior SNP (co)variances from the Bayesian models as weights to build <strong>G</strong> matrices (wGBLUP), in both single-and multi-trait analysis. This is in line with published results for single-trait weighted GBLUP (<a id="xref-ref-37-6" class="xref-bibr" href="#ref-37">Su <em>et al.</em> 2014</a>; <a id="xref-ref-3-4" class="xref-bibr" href="#ref-3">Calus <em>et al.</em> 2014</a>). To our knowledge, multi-trait weighted GBLUP has not been reported so far, although some studies constructed <strong>G</strong> matrices in a similar manner for across breed prediction (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Zhou <em>et al.</em> 2014</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Veroneze <em>et al.</em> 2016</a>).</p></div><div id="sec-16" class="subsection"><h3>Implementation of multi-trait weighted GBLUP in genomic prediction</h3><p id="p-54">Most recent applications of genomic prediction are based on a single-trait genomic model. Many important traits in animal breeding, however, have genetic correlations of varying size with one or more traits, and analysis of such genetically correlated traits jointly using a multi-trait model may yield more accurate predictions of breeding values compared to separate single-trait analysis. Further implementation of genomic selection in breeding programs, therefore, will require extension of current single-trait genomic evaluation models to their multi-trait versions (<a id="xref-ref-23-4" class="xref-bibr" href="#ref-23">Janss 2014</a>). This is straightforward for GBLUP as it requires simply replacing the numerator relationship matrix <strong>A</strong> in the traditional multi-trait BLUP models by the genomic relationship matrix <strong>G</strong> (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Nejati-Javaremi <em>et al.</em> 1997</a>; <a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">VanRaden 2008</a>; <a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">Hayes <em>et al.</em> 2009</a>). However, it is implicitly assumed in the commonly used <strong>G</strong> matrices that each loci equally contribute to the genomic (co)variance (<a id="xref-ref-49-5" class="xref-bibr" href="#ref-49">Zhang <em>et al.</em> 2010</a>; <a id="xref-ref-23-5" class="xref-bibr" href="#ref-23">Janss 2014</a>). In the situation of two correlated traits, this assumption of genetic architecture is violated if some genomic regions explain a large proportion of the total genomic variance and/or covariance, while other regions explain only a small amount (<a id="xref-ref-36-4" class="xref-bibr" href="#ref-36">Sørensen <em>et al.</em> 2012</a>; <a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">Brøndum <em>et al.</em> 2012</a>; <a id="xref-ref-27-7" class="xref-bibr" href="#ref-27">Li <em>et al.</em> 2017</a>; <a id="xref-ref-14-4" class="xref-bibr" href="#ref-14">Gebreyesus <em>et al.</em> 2017</a>).</p><p id="p-55">Bayesian whole genome regression methods can be used to relax the assumption of equal contribution of each loci to the genomic (co)variance, however, they have computational disadvantages which make them difficult to be used in routine genomic evaluations. Weighted GBLUP can make it possible to avoid computational costs of Bayesian methods, while maintaining a higher prediction accuracy compared to traditional GBLUP. Such a procedure still requires to perform Bayesian analysis to obtain weights, but less frequently compared to the number of evaluations. <a id="xref-ref-37-7" class="xref-bibr" href="#ref-37">Su <em>et al.</em> (2014)</a> reported that weights obtained from a data set having a lag up to three years did not reduce the reliability of genomic prediction in the evaluation of a dairy cattle population.</p><p id="p-56">In practical genomic evaluations, variance components are generally estimated using a linear mixed model at individual level via REML or Bayesian methods (<a id="xref-ref-21-4" class="xref-bibr" href="#ref-21">Hayes <em>et al.</em> 2009</a>), without accounting for heterogeneous SNP (co)variance structure across the genome. Hence, BayesAS and wGBLUP models in which all SNPs are assumed to have a common (co)variance, is consistent with GBLUP. The decrease in accuracy over the five generations was larger for GBLUP than for wGBLUP with a region size of 100 SNPs (”WG” <em>vs.</em> ”100 SNPs” in <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figures 1</a> and <a id="xref-fig-2-3" class="xref-fig" href="#F2">2</a>). For the region sizes considered, accuracies decreased with generations for all methods (Tables S1 and S2), in agreement with other studies (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Habier <em>et al.</em> 2007</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Wolc <em>et al.</em> 2011</a>). It is worth noting that the decrease of accuracy with generations is not contradictory with the results in <a id="xref-ref-37-8" class="xref-bibr" href="#ref-37">Su <em>et al.</em> (2014)</a>, which used same weights for three years. First, time interval of three years in a dairy cattle breeding scheme corresponds approximately to less than one generation in our simulations. Second, although the same weights were used in <a id="xref-ref-37-9" class="xref-bibr" href="#ref-37">Su <em>et al.</em> (2014)</a> for three years, the genomic predictions were performed using an updated phenotypic data, while not only the same weights, but also the same phenotypes from G0 were used throughout the generations in our study.</p><p id="p-57">SNPs fitted in genomic prediction models not only capture the information on LD between SNPs and QTL, but also capture within-family information (<a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">Habier <em>et al.</em> 2007</a>, <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">2010</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Wientjes <em>et al.</em> 2013</a>). Any change on these factors, therefore, affects the accuracy of genomic prediction. Bayesian whole genome regression methods rely more on population LD, while GBLUP relies more on within-family LD. As test population gets farther from reference population, the genetic relationships between reference and test populations become weak, and therefore, the persistence of population LD across generations becomes more important to maintain the accuracy of prediction (<a id="xref-ref-18-4" class="xref-bibr" href="#ref-18">Habier <em>et al.</em> 2007</a>, <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">2010</a>; <a id="xref-ref-39-5" class="xref-bibr" href="#ref-39">Tiezzi and Maltecca 2015</a>). Due to intensive genomic selection in many countries, such as the Nordic countries, the distance between selection candidates and genomic reference population has increased. It is, therefore, very likely that most genotyped candidates will have no sire with daughter information in the reference population, at the time of they are being selected (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Gao <em>et al.</em> 2013</a>). Thus, it is important for practical applications if wGBLUP performs better than traditional GBLUP when the selection candidates are a few generations apart from the reference population.</p><p id="p-58">The gain of accuracy from a multi-trait over single-trait genomic prediction is more profound for low heritability traits that are genetically correlated with a high heritability trait, and for traits which can not be measured for all individuals (<a id="xref-ref-24-4" class="xref-bibr" href="#ref-24">Jia and Jannink 2012</a>; <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Guo <em>et al.</em> 2014</a>). When the total number of QTL was increased from 200 to 500, the advantage wGBLUP, over traditional GBLUP decreased for trait <em>L</em>. The improvement in accuracy of a low heritability trait could be similar to 500 QTL scenario, which resembles the situation of complex traits, if the methodology presented here is used in practical applications.</p><p id="p-59">Due to the reduction in genotyping costs, and the interest in exchange of data among countries, reference populations now include several thousands of genotyped bulls and cows even for small breeds (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Su <em>et al.</em> 2016</a>). The reference population used here consists of a relatively small number of individuals of 2,200, which may have an impact on comparison of the models, particularly when the simulated QTL more closely fit an infinitesimal model. In order to investigate the potential usefulness of our approach, we run additional analysis combining G0 and G1 to form a reference, and G2 as test population, respectively (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). Although adding more ancestral generations to the reference population is not the same as a reference population consisting of a recent generation with more animals, it can be an alternative strategy (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Weng <em>et al.</em> 2016</a>) to investigate the impact of reference population size on comparison between models. The results of BayesAS models (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>) demonstrate that prediction accuracies from region size of 100 SNPs was superior to those from other region sizes in both single and multi-trait analysis. More importantly, the improvement of wGBLUP over traditional GBLUP at the region size of 100 SNPs was larger than that in a smaller reference population size (<a id="xref-table-wrap-2-5" class="xref-table" href="#T2">Table 2</a>), when trait <em>L</em> was controlled by relatively a large number of QTL. This suggest that weighting SNPs by posterior (co)variance estimates from Bayesian models can be a useful strategy for improving the accuracy of GBLUP, provided that the data used to derive weights is sufficiently large.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/491370/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/491370/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Accuracies of genomic prediction for test animals in generation 2 using different methods with varying region sizes for 500 QTL scenario, when generations 0 and 1 were used as training population</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-17" class="subsection"><h3>CONCLUSIONS</h3><p id="p-64">In this study, we aimed to develop a strategy to improve the accuracy of GBLUP in multi-trait genomic prediction. Our two step strategy involves the estimation of SNP (co)variances via a Bayesian method, and subsequent use of those (co)variances as weights in multi-trait weighted GBLUP analysis. We used BayesAS method to obtain those weights, however, other Bayesian whole genome regression methods could also be used. In general, assuming a common (co)variance for 100 adjacent SNPs led to highest prediction accuracies using BayesAS. When 100 adjacent SNPs were assigned a common weight obtained from posterior (co)variances of BayesAS, wGBLUP outperformed the traditional GBLUP. Our results demonstrate that multi-trait GBLUP can yield accuracies of genomic prediction as high as Bayesian multi-trait genomic prediction method, when SNPs are weighted by posterior SNP (co)variances from the Bayesian method. The gain from a multi-trait weighted GBLUP analysis depends on factors, such as the model used to obtain weights (<em>i.e.</em>, SNP (co)variances), genetic architecture of the traits (<em>e.g.</em>, number of QTL, traits’ heritabilities), and reference population size.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-65">This study was funded by the “Multi-Genomics” project from the Danish Milk Levy Fund (Aarhus, Denmark).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-13"><p id="p-66"><em>Communicating editor: D. J. de Koning</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2018-06-01"><span class="received-label">Received </span>June 1, 2018.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2018-08-31"><span class="accepted-label">Accepted </span>August 31, 2018.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2018 Karaman <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3549.1" data-doi="10.3168/jds.2009-2730"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aguilar</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Misztal</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsuruta</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A unified approach to utilize phenotypic, full pedigree, and genomic information for genetic evaluation of holstein final score.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">743</span>–<span class="cit-lpage">752</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.2009-2730</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DAguilar%26rft.auinit1%253DI.%26rft.volume%253D93%26rft.issue%253D2%26rft.spage%253D743%26rft.epage%253D752%26rft.atitle%253DHot%2Btopic%253A%2Ba%2Bunified%2Bapproach%2Bto%2Butilize%2Bphenotypic%252C%2Bfull%2Bpedigree%252C%2Band%2Bgenomic%2Binformation%2Bfor%2Bgenetic%2Bevaluation%2Bof%2BHolstein%2Bfinal%2Bscore.%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2009-2730%26rft_id%253Dinfo%253Apmid%252F20105546%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2009-2730&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20105546&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3549.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000274102500032&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3549.2" data-doi="10.1186/1471-2164-13-543"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brøndum</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Su</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lund</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowman</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genome position specific priors for genomic prediction.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">543</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2164-13-543</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DBrondum%26rft.auinit1%253DR.%2BF.%26rft.volume%253D13%26rft.issue%253D1%26rft.spage%253D543%26rft.epage%253D543%26rft.atitle%253DGenome%2Bposition%2Bspecific%2Bpriors%2Bfor%2Bgenomic%2Bprediction.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-13-543%26rft_id%253Dinfo%253Apmid%252F23050763%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-13-543&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23050763&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3549.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3549.3" data-doi="10.1186/s12711-014-0052-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Calus</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schrooten</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Veerkamp</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genomic prediction of breeding values using previously estimated snp variances.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">52</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12711-014-0052-x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D46%26rft.spage%253D52%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12711-014-0052-x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12711-014-0052-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3549.4" data-doi="10.1111/j.1439-0388.2007.00691.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Calus</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Veerkamp</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Accuracy of breeding values when using and ignoring the polygenic effect in genomic breeding value estimation with a marker density of one snp per cm.</span> <abbr class="cit-jnl-abbrev">J. Anim. Breed. Genet.</abbr> <span class="cit-vol">124</span>: <span class="cit-fpage">362</span>–<span class="cit-lpage">368</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1439-0388.2007.00691.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BAnim%2BBreed%2BGenet%26rft.aulast%253DCalus%26rft.auinit1%253DM.%2BP.%26rft.volume%253D124%26rft.issue%253D6%26rft.spage%253D362%26rft.epage%253D368%26rft.atitle%253DAccuracy%2Bof%2Bbreeding%2Bvalues%2Bwhen%2Busing%2Band%2Bignoring%2Bthe%2Bpolygenic%2Beffect%2Bin%2Bgenomic%2Bbreeding%2Bvalue%2Bestimation%2Bwith%2Ba%2Bmarker%2Bdensity%2Bof%2Bone%2BSNP%2Bper%2BcM.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1439-0388.2007.00691.x%26rft_id%253Dinfo%253Apmid%252F18076473%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1439-0388.2007.00691.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18076473&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3549.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251432800006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3549.5" data-doi="10.1186/1297-9686-43-26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Calus</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Veerkamp</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Accuracy of multi-trait genomic selection using different methods.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">26</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1297-9686-43-26</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DCalus%26rft.auinit1%253DM.%2BP.%26rft.volume%253D43%26rft.spage%253D26%26rft.epage%253D26%26rft.atitle%253DAccuracy%2Bof%2Bmulti-trait%2Bgenomic%2Bselection%2Busing%2Bdifferent%2Bmethods.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-43-26%26rft_id%253Dinfo%253Apmid%252F21729282%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-43-26&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21729282&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3549.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3549.6" data-doi="10.1534/genetics.118.300650"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kizilkaya</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garrick</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Genomic prediction from multiple-trait bayesian regression methods using mixture priors.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">209</span>: <span class="cit-fpage">89</span>–<span class="cit-lpage">103</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.118.300650</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.118.300650%26rft_id%253Dinfo%253Apmid%252F29514861%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMjA5LzEvODkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzExLzM1NDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3549.7" data-doi="10.1186/1297-9686-42-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Christensen</span>  <span class="cit-name-given-names">O. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lund</span>  <span class="cit-name-given-names">M. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genomic prediction when some animals are not genotyped.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">2</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1297-9686-42-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DChristensen%26rft.auinit1%253DO.%2BF.%26rft.volume%253D42%26rft.spage%253D2%26rft.epage%253D2%26rft.atitle%253DGenomic%2Bprediction%2Bwhen%2Bsome%2Banimals%2Bare%2Bnot%2Bgenotyped.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-42-2%26rft_id%253Dinfo%253Apmid%252F20105297%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-42-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20105297&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3549.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3549.8" data-doi="10.1071/AN10096"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henshall</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dominik</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gredler</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span><span class="cit-comment">a</span> <span class="cit-article-title">Accuracy of estimated genomic breeding values for wool and meat traits in a multi-breed sheep population.</span> <abbr class="cit-jnl-abbrev">Anim. Prod. Sci.</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">1004</span>–<span class="cit-lpage">1010</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1071/AN10096</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnim.%2BProd.%2BSci.%26rft.volume%253D50%26rft.spage%253D1004%26rft_id%253Dinfo%253Adoi%252F10.1071%252FAN10096%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1071/AN10096&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3549.9" data-doi="10.1534/genetics.110.116855"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pong-Wong</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Villanueva</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Woolliams</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span><span class="cit-comment">b</span> <span class="cit-article-title">The impact of genetic architecture on genome-wide evaluation methods.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">185</span>: <span class="cit-fpage">1021</span>–<span class="cit-lpage">1031</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.110.116855</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.116855%26rft_id%253Dinfo%253Apmid%252F20407128%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4NS8zLzEwMjEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzExLzM1NDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3549.10" data-doi="10.1534/genetics.112.143313"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pong-Wong</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calus</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Whole-genome regression and prediction methods applied to plant and animal breeding.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">193</span>: <span class="cit-fpage">327</span>–<span class="cit-lpage">345</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.112.143313</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.143313%26rft_id%253Dinfo%253Apmid%252F22745228%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkzLzIvMzI3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMS8zNTQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-other" id="cit-8.11.3549.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dvorkin</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> lcmix: Layered and chained mixture models. R package version 0.3/r5.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-confproc" id="cit-8.11.3549.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> Bayesian inference of genomic similarity among individuals from markers and phenotypes. In <em>Proceedings of the World Congress on Genetics Applied to Livestock Production</em>, p. <span class="cit-fpage">942</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3549.13" data-doi="10.3168/jds.2012-6406"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Su</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Janss</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lund</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Model comparison on genomic predictions using high-density markers for different groups of bulls in the nordic holstein population.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">96</span>: <span class="cit-fpage">4678</span>–<span class="cit-lpage">4687</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.2012-6406</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BDairy%2BSci.%26rft.volume%253D96%26rft.spage%253D4678%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2012-6406%26rft_id%253Dinfo%253Apmid%252F23660137%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2012-6406&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23660137&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3549.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3549.14" data-doi="10.1186/s12711-017-0364-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gebreyesus</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lund</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buitenhuis</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bovenhuis</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poulsen</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Modeling heterogeneous (co)variances from adjacent-snp groups improves genomic prediction for milk protein composition traits.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">89</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12711-017-0364-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D49%26rft.spage%253D89%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12711-017-0364-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12711-017-0364-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3549.15" data-doi="10.1534/genetics.109.103952"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manfredi</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Additive genetic variability and bayesian alphabet.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">183</span>: <span class="cit-fpage">347</span>–<span class="cit-lpage">363</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.109.103952</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.103952%26rft_id%253Dinfo%253Apmid%252F19620397%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgzLzEvMzQ3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMS8zNTQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3549.16" data-doi="10.1038/nrg2575"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Mapping genes for complex traits in domestic animals and their use in breeding programmes.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">381</span>–<span class="cit-lpage">391</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg2575</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DGoddard%26rft.auinit1%253DM.%2BE.%26rft.volume%253D10%26rft.issue%253D6%26rft.spage%253D381%26rft.epage%253D391%26rft.atitle%253DMapping%2Bgenes%2Bfor%2Bcomplex%2Btraits%2Bin%2Bdomestic%2Banimals%2Band%2Btheir%2Buse%2Bin%2Bbreeding%2Bprogrammes.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2575%26rft_id%253Dinfo%253Apmid%252F19448663%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2575&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19448663&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3549.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000266095200013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3549.17" data-doi="10.1186/1471-2156-15-30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Du</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Comparison of single-trait and multiple-trait genomic prediction models.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">30</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2156-15-30</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenet.%26rft.volume%253D15%26rft.spage%253D30%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2156-15-30%26rft_id%253Dinfo%253Apmid%252F24593261%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2156-15-30&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24593261&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3549.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3549.18" data-doi="10.1534/genetics.107.081190"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Habier</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dekkers</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The impact of genetic relationship information on genome-assisted breeding values.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">177</span>: <span class="cit-fpage">2389</span>–<span class="cit-lpage">2397</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHabier%26rft.auinit1%253DD.%26rft.volume%253D177%26rft.issue%253D4%26rft.spage%253D2389%26rft.epage%253D2397%26rft.atitle%253DThe%2BImpact%2Bof%2BGenetic%2BRelationship%2BInformation%2Bon%2BGenome-Assisted%2BBreeding%2BValues%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.081190%26rft_id%253Dinfo%253Apmid%252F18073436%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Ny80LzIzODkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzExLzM1NDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3549.19" data-doi="10.1186/1297-9686-42-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Habier</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tetens</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seefried</span>  <span class="cit-name-given-names">F.-R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lichtner</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thaller</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The impact of genetic relationship information on genomic breeding values in german holstein cattle.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">5</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1297-9686-42-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DHabier%26rft.auinit1%253DD.%26rft.volume%253D42%26rft.spage%253D5%26rft.epage%253D5%26rft.atitle%253DThe%2Bimpact%2Bof%2Bgenetic%2Brelationship%2Binformation%2Bon%2Bgenomic%2Bbreeding%2Bvalues%2Bin%2BGerman%2BHolstein%2Bcattle.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-42-5%26rft_id%253Dinfo%253Apmid%252F20170500%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-42-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20170500&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3549.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3549.20" data-doi="10.1186/1471-2105-14-34"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hayashi</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iwata</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A bayesian method and its variational approximation for prediction of genomic breeding values in multiple traits.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">34</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-14-34</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D14%26rft.spage%253D34%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-14-34%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-14-34&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3549.21" data-doi="10.3168/jds.2008-1646"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowman</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chamberlain</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Invited review: Genomic selection in dairy cattle: progress and challenges.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">433</span>–<span class="cit-lpage">443</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.2008-1646</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DHayes%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D92%26rft.issue%253D2%26rft.spage%253D433%26rft.epage%253D443%26rft.atitle%253DInvited%2Breview%253A%2BGenomic%2Bselection%2Bin%2Bdairy%2Bcattle%253A%2BProgress%2Band%2Bchallenges%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2008-1646%26rft_id%253Dinfo%253Apmid%252F19164653%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2008-1646&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19164653&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3549.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000262654900001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3549.22" data-doi="10.1051/gse:2001117"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">The distribution of the effects of genes affecting quantitative traits in livestock.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">209</span>–<span class="cit-lpage">229</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1297-9686-33-3-209</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DHayes%26rft.auinit1%253DB.%26rft.volume%253D33%26rft.issue%253D3%26rft.spage%253D209%26rft.epage%253D229%26rft.atitle%253DThe%2Bdistribution%2Bof%2Bthe%2Beffects%2Bof%2Bgenes%2Baffecting%2Bquantitative%2Btraits%2Bin%2Blivestock.%26rft_id%253Dinfo%253Adoi%252F10.1051%252Fgse%253A2001117%26rft_id%253Dinfo%253Apmid%252F11403745%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1051/gse:2001117&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11403745&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3549.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000169502100001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-confproc" id="cit-8.11.3549.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Janss</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> Disentangling pleiotrophy along the genome using sparse latent variable models. In <em>Proceedings</em>, <em>10th World Congress of Genetics Applied to Livestock Production</em>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3549.24" data-doi="10.1534/genetics.112.144246"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jia</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Multiple-trait genomic selection methods increase genetic value prediction accuracy.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">192</span>: <span class="cit-fpage">1513</span>–<span class="cit-lpage">1522</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.112.144246</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.144246%26rft_id%253Dinfo%253Apmid%252F23086217%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5Mi80LzE1MTMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzExLzM1NDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3549.25" data-doi="10.1371/journal.pone.0161054"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Karaman</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Firat</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garrick</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">An upper bound for accuracy of prediction using gblup.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">e0161054</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0161054</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D11%26rft.spage%253De0161054%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0161054%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0161054&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3549.26" data-doi="10.1534/genetics.108.088575"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robert-Granié</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manfredi</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elsen</span>  <span class="cit-name-given-names">J.-M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Performance of genomic selection in mice.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">180</span>: <span class="cit-fpage">611</span>–<span class="cit-lpage">618</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.108.088575</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.088575%26rft_id%253Dinfo%253Apmid%252F18757934%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgwLzEvNjExIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMS8zNTQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3549.27" data-doi="10.1186/s12863-017-0491-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lund</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Janss</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ding</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">The patterns of genomic variances and covariances across genome for milk production traits between chinese and nordic holstein populations.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">26</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12863-017-0491-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenet.%26rft.volume%253D18%26rft.spage%253D26%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12863-017-0491-9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12863-017-0491-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3549.28" data-doi="10.3168/jds.2008-1011"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lipkin</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bagnato</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soller</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Expected effects on protein yield of marker-assisted selection at quantitative trait loci affecting milk yield and milk protein percentage.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">2857</span>–<span class="cit-lpage">2863</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.2008-1011</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DLipkin%26rft.auinit1%253DE.%26rft.volume%253D91%26rft.issue%253D7%26rft.spage%253D2857%26rft.epage%253D2863%26rft.atitle%253DExpected%2BEffects%2Bon%2BProtein%2BYield%2Bof%2BMarker-Assisted%2BSelection%2Bat%2BQuantitative%2BTrait%2BLoci%2BAffecting%2BMilk%2BYield%2Band%2BMilk%2BProtein%2BPercentage%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2008-1011%26rft_id%253Dinfo%253Apmid%252F18565943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2008-1011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18565943&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3549.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3549.29" data-doi="10.1186/1297-9686-43-19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seefried</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reinhardt</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rensing</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thaller</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Impacts of both reference population size and inclusion of a residual polygenic effect on the accuracy of genomic prediction.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">19</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1297-9686-43-19</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DLiu%26rft.auinit1%253DZ.%26rft.volume%253D43%26rft.spage%253D19%26rft.epage%253D19%26rft.atitle%253DImpacts%2Bof%2Bboth%2Breference%2Bpopulation%2Bsize%2Band%2Binclusion%2Bof%2Ba%2Bresidual%2Bpolygenic%2Beffect%2Bon%2Bthe%2Baccuracy%2Bof%2Bgenomic%2Bprediction.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-43-19%26rft_id%253Dinfo%253Apmid%252F21586131%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-43-19&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21586131&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3549.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3549.30" data-doi="10.1007/s00122-009-1166-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lorenzana</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Accuracy fo genotypic value predictions for marker-based selection in biparental plant populations.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">120</span>: <span class="cit-fpage">151</span>–<span class="cit-lpage">161</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-009-1166-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DLorenzana%26rft.auinit1%253DR.%2BE.%26rft.volume%253D120%26rft.issue%253D1%26rft.spage%253D151%26rft.epage%253D161%26rft.atitle%253DAccuracy%2Bof%2Bgenotypic%2Bvalue%2Bpredictions%2Bfor%2Bmarker-based%2Bselection%2Bin%2Bbiparental%2Bplant%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-009-1166-3%26rft_id%253Dinfo%253Apmid%252F19841887%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-009-1166-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19841887&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3549.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000271939900013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3549.31" data-doi="10.1534/genetics.110.116590"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Accurate prediction of genetic values for complex traits by whole-genome resequencing.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">185</span>: <span class="cit-fpage">623</span>–<span class="cit-lpage">631</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.110.116590</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.116590%26rft_id%253Dinfo%253Apmid%252F20308278%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg1LzIvNjIzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMS8zNTQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3549.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Prediction of total genetic value using genome-wide dense marker maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">1819</span>–<span class="cit-lpage">1829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMeuwissen%26rft.auinit1%253DT.%2BH.%2BE.%26rft.volume%253D157%26rft.issue%253D4%26rft.spage%253D1819%26rft.epage%253D1829%26rft.atitle%253DPrediction%2Bof%2BTotal%2BGenetic%2BValue%2BUsing%2BGenome-Wide%2BDense%2BMarker%2BMaps%26rft_id%253Dinfo%253Apmid%252F11290733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ny80LzE4MTkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzExLzM1NDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3549.33" data-doi="10.3389/fgene.2014.00001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morota</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Kernel-based whole-genome prediction of complex traits: a review.</span> <abbr class="cit-jnl-abbrev">Front. Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">13</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fgene.2014.00363</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BGenet.%26rft.volume%253D5%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffgene.2014.00001%26rft_id%253Dinfo%253Apmid%252F24567736%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fgene.2014.00001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24567736&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3549.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3549.34" data-doi="10.2527/1997.7571738x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nejati-Javaremi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gibson</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Effect of total allelic relationship on accuracy of evaluation and response to selection.</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">75</span>: <span class="cit-fpage">1738</span>–<span class="cit-lpage">1745</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2527/1997.7571738x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BAnimal%2BScience%26rft.stitle%253DJ%2BANIM%2BSCI%26rft.aulast%253DNejati-Javaremi%26rft.auinit1%253DA.%26rft.volume%253D75%26rft.issue%253D7%26rft.spage%253D1738%26rft.epage%253D1745%26rft.atitle%253DEffect%2Bof%2Btotal%2Ballelic%2Brelationship%2Bon%2Baccuracy%2Bof%2Bevaluation%2Band%2Bresponse%2Bto%2Bselection%26rft_id%253Dinfo%253Adoi%252F10.2527%252F1997.7571738x%26rft_id%253Dinfo%253Apmid%252F9222829%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2527/1997.7571738x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9222829&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3549.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997XH54200007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-web" id="cit-8.11.3549.35"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">R Core Team</span>, <span class="cit-pub-date">2017</span> R: A language and environment for statistical computing. R Foundation for Statistical Computing <a href="https://www.R-project.org/">https://www.R-project.org/</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3549.36" data-doi="10.1186/1297-9686-44-18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sørensen</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Janss</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Madsen</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mark</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lund</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Estimation of (co)variances for genomic regions of flexible sizes: application to complex infectious udder diseases in dairy cattle.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">18</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1297-9686-44-18</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DSorensen%26rft.auinit1%253DL.%2BP.%26rft.volume%253D44%26rft.spage%253D18%26rft.epage%253D18%26rft.atitle%253DEstimation%2Bof%2B%2528co%2529variances%2Bfor%2Bgenomic%2Bregions%2Bof%2Bflexible%2Bsizes%253A%2Bapplication%2Bto%2Bcomplex%2Binfectious%2Budder%2Bdiseases%2Bin%2Bdairy%2Bcattle.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-44-18%26rft_id%253Dinfo%253Apmid%252F22640006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-44-18&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22640006&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3549.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3549.37" data-doi="10.3168/jds.2014-8210"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Su</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Christensen</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Janss</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lund</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Comparison of genomic predictions using genomic relationship matrices built with different weighting factors to account for locus-specific variances.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">97</span>: <span class="cit-fpage">6547</span>–<span class="cit-lpage">6559</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.2014-8210</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BDairy%2BSci.%26rft.volume%253D97%26rft.spage%253D6547%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2014-8210%26rft_id%253Dinfo%253Apmid%252F25129495%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2014-8210&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25129495&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3549.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3549.38" data-doi="10.1017/S1751731115001792"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Su</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aamand</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wiggans</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Sharing reference data and including cows in the reference population improve genomic predictions in danish jersey.</span> <abbr class="cit-jnl-abbrev">Animal</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">1067</span>–<span class="cit-lpage">1075</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S1751731115001792</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnimal%26rft.volume%253D10%26rft.spage%253D1067%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS1751731115001792%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S1751731115001792&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3549.39" data-doi="10.1186/s12711-015-0100-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tiezzi</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maltecca</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Accounting for trait architecture in genomic predictions of us holstein cattle using a weighted realized relationship matrix.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">24</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12711-015-0100-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D47%26rft.spage%253D24%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12711-015-0100-1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12711-015-0100-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3549.40" data-doi="10.3168/jds.2007-0980"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient methods to compute genomic predictions.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">4414</span>–<span class="cit-lpage">4423</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.2007-0980</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVanRaden%26rft.auinit1%253DP.%2BM.%26rft.volume%253D91%26rft.issue%253D11%26rft.spage%253D4414%26rft.epage%253D4423%26rft.atitle%253DEfficient%2BMethods%2Bto%2BCompute%2BGenomic%2BPredictions%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2007-0980%26rft_id%253Dinfo%253Apmid%252F18946147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2007-0980&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18946147&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3549.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260277200035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3549.41" data-doi="10.3168/jds.2008-1514"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Tassell</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wiggans</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sonstegard</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schnabel</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Invited review: Reliability of genomic predictions for north american holstein bulls.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">16</span>–<span class="cit-lpage">24</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.2008-1514</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVanRaden%26rft.auinit1%253DP.%2BM.%26rft.volume%253D92%26rft.issue%253D1%26rft.spage%253D16%26rft.epage%253D24%26rft.atitle%253DInvited%2BReview%253A%2BReliability%2Bof%2Bgenomic%2Bpredictions%2Bfor%2BNorth%2BAmerican%2BHolstein%2Bbulls%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2008-1514%26rft_id%253Dinfo%253Apmid%252F19109259%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2008-1514&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19109259&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3549.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261948600002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3549.42" data-doi="10.1111/jbg.12202"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Veroneze</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lopes</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lopes</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hidalgo</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guimarães</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Accounting for genetic architecture in single- and multi-population genomic prediction using weights from genomewide association studies in pigs.</span> <abbr class="cit-jnl-abbrev">J. Anim. Breed. Genet.</abbr> <span class="cit-vol">133</span>: <span class="cit-fpage">187</span>–<span class="cit-lpage">196</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/jbg.12202</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAnim.%2BBreed.%2BGenet.%26rft.volume%253D133%26rft.spage%253D187%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fjbg.12202%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/jbg.12202&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3549.43" data-doi="10.1186/s12711-016-0198-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weng</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wolc</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dekkers</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Effects of number of training generations on genomic prediction for various traits in a layer chicken population.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">22</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12711-016-0198-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D48%26rft.spage%253D22%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12711-016-0198-9%26rft_id%253Dinfo%253Apmid%252F26992471%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12711-016-0198-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26992471&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3549.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-other" id="cit-8.11.3549.44"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wientjes</span>  <span class="cit-name-given-names">Y. C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Veerkamp</span>  <span class="cit-name-given-names">R. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calus</span>  <span class="cit-name-given-names">M. P. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> The effect of linkage disequilibrium and family relationships on the reliability of genomic prediction 193: 621–631.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3549.45" data-doi="10.1186/1297-9686-43-23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wolc</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arango</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Settar</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fulton</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Sullivan</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Persistence of accuracy of genomic estimated breeding values over generations in layer chickens.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">23</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1297-9686-43-23</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DWolc%26rft.auinit1%253DA.%26rft.volume%253D43%26rft.spage%253D23%26rft.epage%253D23%26rft.atitle%253DPersistence%2Bof%2Baccuracy%2Bof%2Bgenomic%2Bestimated%2Bbreeding%2Bvalues%2Bover%2Bgenerations%2Bin%2Blayer%2Bchickens.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-43-23%26rft_id%253Dinfo%253Apmid%252F21693035%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-43-23&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21693035&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3549.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3549.46" data-doi="10.3382/ps.2012-02882"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wolc</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arango</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Settar</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fulton</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Sullivan</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Analysis of egg production in layer chickens using a random regression model with genomic relationships.</span> <abbr class="cit-jnl-abbrev">Poult. Sci.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">1486</span>–<span class="cit-lpage">1491</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3382/ps.2012-02882</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPoult.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.3382%252Fps.2012-02882%26rft_id%253Dinfo%253Apmid%252F23687143%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3382/ps.2012-02882&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23687143&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3549.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3549.47" data-doi="10.1038/srep19454"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yamamoto</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matsunaga</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Onogi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kajiya-Kanegae</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Minamikawa</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">A simulation-based breeding design that uses whole-genome prediction in tomato.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">19454</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/srep19454</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D6%26rft.spage%253D19454%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep19454%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/srep19454&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3549.48" data-doi="10.3389/fgene.2016.00151"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lourenco</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aguilar</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Misztal</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Weighting strategies for single-step genomic blup: An iterative approach for accurate calculation of gebv and gwas.</span> <abbr class="cit-jnl-abbrev">Front. Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">151</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fgene.2016.00151</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BGenet.%26rft.volume%253D7%26rft.spage%253D151%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffgene.2016.00151%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fgene.2016.00151&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3549.49" data-doi="10.1371/journal.pone.0012648"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ding</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bijma</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Koning</span>  <span class="cit-name-given-names">D.-J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Best linear unbiased prediction of genomic breeding values using a trait-specific marker-derived relationship matrix.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e12648</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0012648</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D5%26rft.spage%253De12648%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0012648%26rft_id%253Dinfo%253Apmid%252F20844593%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0012648&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20844593&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3549.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3549.50" data-doi="10.1111/jbg.12089"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lund</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Su</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genomic predictions across nordic holstein and nordic red using the genomic best linear unbiased prediction model with different genomic relationship matrices.</span> <abbr class="cit-jnl-abbrev">J. Anim. Breed. Genet.</abbr> <span class="cit-vol">131</span>: <span class="cit-fpage">249</span>–<span class="cit-lpage">257</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/jbg.12089</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAnim.%2BBreed.%2BGenet.%26rft.volume%253D131%26rft.spage%253D249%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fjbg.12089%26rft_id%253Dinfo%253Apmid%252F24750283%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/jbg.12089&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24750283&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3549.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/8/11/3549.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/11/3539" title="Identification and Characterization of Transcripts Regulated by Circadian Alternative Polyadenylation in Mouse Liver" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/11/3559" title="Discovery of QTL Alleles for Grain Shape in the Japan-MAGIC Rice Population Using Haplotype Information" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/8/11"><span>Volume 8 Issue 11, November 2018</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/8/11.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/8/11.cover-source.jpg?itok=lFO8v6bg" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 8 (11)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/11/3549" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-XrwVhNnw8iBXWcbsopnKzCOq3g-2Tpt4g793WsEK_FQ" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genomic-prediction" class="" data-icon-position="" data-hide-link-title="0">Genomic Prediction</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/8/11/3549" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/8/11/3549" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/8/11/3549" class="active" data-icon-position="" data-hide-link-title="0">Genomic Prediction Using Multi-trait Weighted GBLUP Accounting for Heterogeneous Variances and Covariances Across the Genome</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/491248" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-aunMJbj4aS14ip-8aNt7ol5RZ3dKDxPirbuOP9RDFH0" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/11/3549.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/8/11/3549" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-7kPsZpdmzNvHtu2t0EkbLcMZIu1avXZVGmMbBIH9B3E" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/8/11/3549" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/11/3549.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="491248" id="node491248--2" data-pisa="ggg;8/11/3549" data-pisa-master="ggg;g3.118.200673" data-apath="/ggg/8/11/3549.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic Prediction Using Multi-trait Weighted GBLUP Accounting for Heterogeneous Variances and Covariances Across the Genome</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-1010-683X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Emre Karaman</span>, <span class="highwire-citation-author" data-delta="1">Mogens S. Lund</span>, <span class="highwire-citation-author" data-delta="2">Mahlet T. Anche</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0001-9804-1529" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Luc Janss</span> and <span class="highwire-citation-author" data-delta="4">Guosheng Su</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3549-3558;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200673 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node491248--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Emre Karaman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Center for Quantitative Genetics and Genomics, Aarhus University, 8830 Tjele, Denmark</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Emre%2BKaraman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Karaman%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AEmre%2BKaraman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-1010-683X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Emre Karaman</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:emre@mbg.au.dk" class="" data-icon-position="" data-hide-link-title="0">emre@mbg.au.dk</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Mogens S. Lund </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Center for Quantitative Genetics and Genomics, Aarhus University, 8830 Tjele, Denmark</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mogens%2BS.%2BLund%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lund%20MS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMogens%2BS.%2BLund%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Mahlet T. Anche </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Center for Quantitative Genetics and Genomics, Aarhus University, 8830 Tjele, Denmark</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mahlet%2BT.%2BAnche%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Anche%20MT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMahlet%2BT.%2BAnche%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Luc Janss </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Center for Quantitative Genetics and Genomics, Aarhus University, 8830 Tjele, Denmark</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Luc%2BJanss%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Janss%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALuc%2BJanss%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-9804-1529" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Luc Janss</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Guosheng Su </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Center for Quantitative Genetics and Genomics, Aarhus University, 8830 Tjele, Denmark</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Guosheng%2BSu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Su%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGuosheng%2BSu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/8/11/3549" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/8/11/3549&amp;title=Genomic%20Prediction%20Using%20Multi-trait%20Weighted%20GBLUP%20Accounting%20for%20Heterogeneous%20Variances%20and%20Covariances%20Across%20the%20Genome" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/8/11/3549&amp;title=Genomic%20Prediction%20Using%20Multi-trait%20Weighted%20GBLUP%20Accounting%20for%20Heterogeneous%20Variances%20and%20Covariances%20Across%20the%20Genome" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/8/11/3549&amp;title=Genomic%20Prediction%20Using%20Multi-trait%20Weighted%20GBLUP%20Accounting%20for%20Heterogeneous%20Variances%20and%20Covariances%20Across%20the%20Genome" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/8/11/3549&amp;text=Genomic%20Prediction%20Using%20Multi-trait%20Weighted%20GBLUP%20Accounting%20for%20Heterogeneous%20Variances%20and%20Covariances%20Across%20the%20Genome" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/8/11/3549&amp;title=Genomic%20Prediction%20Using%20Multi-trait%20Weighted%20GBLUP%20Accounting%20for%20Heterogeneous%20Variances%20and%20Covariances%20Across%20the%20Genome" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/8/11/3549&amp;t=Genomic%20Prediction%20Using%20Multi-trait%20Weighted%20GBLUP%20Accounting%20for%20Heterogeneous%20Variances%20and%20Covariances%20Across%20the%20Genome" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/8/11/3549&amp;title=Genomic%20Prediction%20Using%20Multi-trait%20Weighted%20GBLUP%20Accounting%20for%20Heterogeneous%20Variances%20and%20Covariances%20Across%20the%20Genome" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/8/11/3549&amp;title=Genomic%20Prediction%20Using%20Multi-trait%20Weighted%20GBLUP%20Accounting%20for%20Heterogeneous%20Variances%20and%20Covariances%20Across%20the%20Genome" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/491248/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="491248" id="node491248--3" data-pisa="ggg;8/11/3549" data-pisa-master="ggg;g3.118.200673" data-apath="/ggg/8/11/3549.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic Prediction Using Multi-trait Weighted GBLUP Accounting for Heterogeneous Variances and Covariances Across the Genome</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-1010-683X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Emre Karaman</span>, <span class="highwire-citation-author" data-delta="1">Mogens S. Lund</span>, <span class="highwire-citation-author" data-delta="2">Mahlet T. Anche</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0001-9804-1529" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Luc Janss</span> and <span class="highwire-citation-author" data-delta="4">Guosheng Su</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3549-3558;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200673 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node491248--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Emre Karaman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Center for Quantitative Genetics and Genomics, Aarhus University, 8830 Tjele, Denmark</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Emre%2BKaraman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Karaman%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AEmre%2BKaraman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-1010-683X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Emre Karaman</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:emre@mbg.au.dk" class="" data-icon-position="" data-hide-link-title="0">emre@mbg.au.dk</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Mogens S. Lund </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Center for Quantitative Genetics and Genomics, Aarhus University, 8830 Tjele, Denmark</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mogens%2BS.%2BLund%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lund%20MS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMogens%2BS.%2BLund%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Mahlet T. Anche </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Center for Quantitative Genetics and Genomics, Aarhus University, 8830 Tjele, Denmark</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mahlet%2BT.%2BAnche%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Anche%20MT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMahlet%2BT.%2BAnche%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Luc Janss </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Center for Quantitative Genetics and Genomics, Aarhus University, 8830 Tjele, Denmark</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Luc%2BJanss%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Janss%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALuc%2BJanss%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-9804-1529" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Luc Janss</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Guosheng Su </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Center for Quantitative Genetics and Genomics, Aarhus University, 8830 Tjele, Denmark</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Guosheng%2BSu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Su%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGuosheng%2BSu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3549&amp;rft.epage=3558&amp;rft.atitle=Genomic%20Prediction%20Using%20Multi-trait%20Weighted%20GBLUP%20Accounting%20for%20Heterogeneous%20Variances%20and%20Covariances%20Across%20the%20Genome&amp;rft.volume=8&amp;rft.issue=11&amp;rft.date=2018-11-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Karaman%2C+Emre&amp;rft.au=Lund%2C+Mogens+S.&amp;rft.au=Anche%2C+Mahlet+T.&amp;rft.au=Janss%2C+Luc&amp;rft.au=Su%2C+Guosheng"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/491248/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/491248/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/491248/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/491248/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/491248/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/491248/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/491248/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/491248/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/491248/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/491248/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/491248/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/491248/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495864" id="node495864" data-pisa="ggg;10/3/1113" data-pisa-master="ggg;g3.119.400968" data-apath="/ggg/10/3/1113.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1113" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Multi-trait Genomic Prediction Model Increased the Predictive Ability for Agronomic and Malting Quality Traits in Barley (<em>Hordeum vulgare</em> L.)</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495543" id="node495543" data-pisa="ggg;10/2/769" data-pisa-master="ggg;g3.119.400759" data-apath="/ggg/10/2/769.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/2/769" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Novel Bayesian Networks for Genomic Prediction of Developmental Traits in Biomass Sorghum</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495557" id="node495557" data-pisa="ggg;10/2/539" data-pisa-master="ggg;g3.119.400866" data-apath="/ggg/10/2/539.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/2/539" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Optimized Genetic Testing for Polledness in Multiple Breeds of Cattle</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genomic%20Prediction" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genomic Prediction</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/8/11/3549" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Material and Methods</a></li><li class="odd"><a href="#sec-9" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-13" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/8/11/3549.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/8/11/3549.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/8/11/3549.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
