<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/8/11/3617" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/8/11/3617.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/8/11/3617.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/8/11/3617.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/8/11/3617.atom" />
<meta name="HW.pisa" content="ggg;8/11/3617" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Genomic Analysis To Identify Signatures of Artificial Selection and Loci Associated with Important Economic Traits in Duroc Pigs" />
<meta name="DC.Identifier" content="10.1534/g3.118.200665" />
<meta name="DC.Date" content="2018-11-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2018 Ma et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Identifying genetic basis of domestication and improvement in livestock contributes to our understanding of the role of artificial selection in shaping the genome. Here we used whole-genome sequencing and the genotyping by sequencing approach to detect artificial selection signatures and identify the associated SNPs of two economic traits in Duroc pigs. A total of 38 candidate selection regions were detected by combining the fixation index and the Composite Likelihood Ratio methods. Further genome-wide association study revealed seven associated SNPs that were related with intramuscular fat content and feed conversion ratio traits, respectively. Enrichment analysis suggested that the artificial selection regions harbored genes, such as MSTN , SOD2 , MC5R and CD83 , which are responsible for economic traits including lean muscle mass, fertility and immunization. Overall, this study found a series of candidate genes putatively associated with the breeding improvement of Duroc pigs and the polygenic basis of adaptive evolution, which can provide important references and fundamental information for future breeding programs." />
<meta name="DC.Contributor" content="Yunlong Ma" />
<meta name="DC.Contributor" content="Saixian Zhang" />
<meta name="DC.Contributor" content="Kaili Zhang" />
<meta name="DC.Contributor" content="Chengchi Fang" />
<meta name="DC.Contributor" content="Shengsong Xie" />
<meta name="DC.Contributor" content="Xiaoyong Du" />
<meta name="DC.Contributor" content="Xinyun Li" />
<meta name="DC.Contributor" content="Debin Ni" />
<meta name="DC.Contributor" content="Shuhong Zhao" />
<meta name="article:published_time" content="2018-11-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Genomic Analysis To Identify Signatures of Artificial Selection and Loci Associated with Important Economic Traits in Duroc Pigs" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Identifying genetic basis of domestication and improvement in livestock contributes to our understanding of the role of artificial selection in shaping the genome. Here we used whole-genome sequencing and the genotyping by sequencing approach to detect artificial selection signatures and identify the associated SNPs of two economic traits in Duroc pigs. A total of 38 candidate selection regions were detected by combining the fixation index and the Composite Likelihood Ratio methods. Further genome-wide association study revealed seven associated SNPs that were related with intramuscular fat content and feed conversion ratio traits, respectively. Enrichment analysis suggested that the artificial selection regions harbored genes, such as &lt;i&gt;MSTN&lt;/i&gt;, &lt;i&gt;SOD2&lt;/i&gt;, &lt;i&gt;MC5R&lt;/i&gt; and &lt;i&gt;CD83&lt;/i&gt;, which are responsible for economic traits including lean muscle mass, fertility and immunization. Overall, this study found a series of candidate genes putatively associated with the breeding improvement of Duroc pigs and the polygenic basis of adaptive evolution, which can provide important references and fundamental information for future breeding programs.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2018/11/01" />
<meta name="citation_mjid" content="ggg;8/11/3617" />
<meta name="citation_id" content="8/11/3617" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/8/11/3617" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/8/11/3617.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/8/11/3617.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/8/11/3617.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.118.200665" />
<meta name="citation_pmid" content="30237295" />
<meta name="citation_volume" content="8" />
<meta name="citation_issue" content="11" />
<meta name="citation_num_pages" content="9" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="3617" />
<meta name="citation_lastpage" content="3625" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Yunlong Ma" />
<meta name="citation_author_institution" content="Key Laboratory of Agricultural Animal Genetics, Breeding, and Reproduction of the Ministry of Education &amp; Key Laboratory of Swine Genetics and Breeding of the Ministry of Agriculture, Huazhong Agricultural University, Wuhan 430070, P. R. China" />
<meta name="citation_author" content="Saixian Zhang" />
<meta name="citation_author_institution" content="Key Laboratory of Agricultural Animal Genetics, Breeding, and Reproduction of the Ministry of Education &amp; Key Laboratory of Swine Genetics and Breeding of the Ministry of Agriculture, Huazhong Agricultural University, Wuhan 430070, P. R. China" />
<meta name="citation_author" content="Kaili Zhang" />
<meta name="citation_author_institution" content="Key Laboratory of Agricultural Animal Genetics, Breeding, and Reproduction of the Ministry of Education &amp; Key Laboratory of Swine Genetics and Breeding of the Ministry of Agriculture, Huazhong Agricultural University, Wuhan 430070, P. R. China" />
<meta name="citation_author" content="Chengchi Fang" />
<meta name="citation_author_institution" content="Key Laboratory of Agricultural Animal Genetics, Breeding, and Reproduction of the Ministry of Education &amp; Key Laboratory of Swine Genetics and Breeding of the Ministry of Agriculture, Huazhong Agricultural University, Wuhan 430070, P. R. China" />
<meta name="citation_author" content="Shengsong Xie" />
<meta name="citation_author_institution" content="Key Laboratory of Agricultural Animal Genetics, Breeding, and Reproduction of the Ministry of Education &amp; Key Laboratory of Swine Genetics and Breeding of the Ministry of Agriculture, Huazhong Agricultural University, Wuhan 430070, P. R. China" />
<meta name="citation_author" content="Xiaoyong Du" />
<meta name="citation_author_institution" content="Key Laboratory of Agricultural Animal Genetics, Breeding, and Reproduction of the Ministry of Education &amp; Key Laboratory of Swine Genetics and Breeding of the Ministry of Agriculture, Huazhong Agricultural University, Wuhan 430070, P. R. China" />
<meta name="citation_author" content="Xinyun Li" />
<meta name="citation_author_institution" content="Key Laboratory of Agricultural Animal Genetics, Breeding, and Reproduction of the Ministry of Education &amp; Key Laboratory of Swine Genetics and Breeding of the Ministry of Agriculture, Huazhong Agricultural University, Wuhan 430070, P. R. China" />
<meta name="citation_author" content="Debin Ni" />
<meta name="citation_author_institution" content="Key Laboratory of Agricultural Animal Genetics, Breeding, and Reproduction of the Ministry of Education &amp; Key Laboratory of Swine Genetics and Breeding of the Ministry of Agriculture, Huazhong Agricultural University, Wuhan 430070, P. R. China" />
<meta name="citation_author_email" content="shzhao@mail.hzau.edu.cn" />
<meta name="citation_author_email" content="nidebin@mail.hzau.edu.cn" />
<meta name="citation_author" content="Shuhong Zhao" />
<meta name="citation_author_institution" content="Key Laboratory of Agricultural Animal Genetics, Breeding, and Reproduction of the Ministry of Education &amp; Key Laboratory of Swine Genetics and Breeding of the Ministry of Agriculture, Huazhong Agricultural University, Wuhan 430070, P. R. China" />
<meta name="citation_author_email" content="shzhao@mail.hzau.edu.cn" />
<meta name="citation_author_email" content="nidebin@mail.hzau.edu.cn" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=A. Abbott;citation_title=Pig geneticists go the whole hog.;citation_pages=315-316;citation_volume=491;citation_year=2012;citation_issue=7424;citation_pmid=23151556;citation_doi=10.1038/491315a" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=H. S. Ai;citation_author=X. D. Fang;citation_author=B. Yang;citation_author=Z. Y. Huang;citation_author=H. Chen;citation_title=Adaptation and possible ancient interspecies introgression in pigs identified by whole-genome sequencing.;citation_pages=217-225;citation_volume=47;citation_year=2015;citation_pmid=25621459;citation_doi=10.1038/ng.3199" />
<meta name="citation_reference" content="citation_journal_title=Reproductive biology and endocrinology [electronic resource] :  RB&amp;E;citation_journal_abbrev=Reprod Biol Endocrinol;citation_author=M. D. Ashworth;citation_author=J. W. Ross;citation_author=D. R. Stein;citation_author=F. J. White;citation_author=U. W. DeSilva;citation_title=Endometrial caspase 1 and interleukin-18 expression during the estrous cycle and peri-implantation period of porcine pregnancy and response to early exogenous estrogen administration.;citation_pages=33-33;citation_volume=8;citation_year=2010;citation_pmid=20380728;citation_doi=10.1186/1477-7827-8-33" />
<meta name="citation_reference" content="citation_journal_title=Biochimica Et Biophysica Acta-Molecular and Cell Biology of Lipids;citation_author=N. Baati;citation_author=C. Feillet-Coudray;citation_author=G. Fouret;citation_author=B. Vernus;citation_author=B. Goustard;citation_title=Myostatin deficiency is associated with lipidomic abnormalities in skeletal muscles.;citation_pages=1044-1055;citation_volume=1862;citation_year=2017;citation_doi=10.1016/j.bbalip.2017.06.017" />
<meta name="citation_reference" content="citation_title=Modern breeds of livestock;citation_author=H. M. Briggs;citation_author=D. M. Briggs;citation_year=1969" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Hum. Genet.;citation_author=B. L. Browning;citation_author=S. R. Browning;citation_title=Genotype Imputation with Millions of Reference Samples.;citation_pages=116-126;citation_volume=98;citation_year=2016;citation_pmid=26748515;citation_doi=10.1016/j.ajhg.2015.11.020" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=J. Chen;citation_author=N. Ingham;citation_author=J. Kelly;citation_author=S. Jadeja;citation_author=D. Goulding;citation_title=Spinster Homolog 2 (Spns2) Deficiency Causes Early Onset Progressive Hearing Loss.;citation_pages=e1004688;citation_volume=10;citation_year=2014;citation_pmid=25356849;citation_doi=10.1371/journal.pgen.1004688" />
<meta name="citation_reference" content="citation_journal_title=J. Anim. Sci.;citation_author=D. L. Clark;citation_author=B. M. Bohrer;citation_author=M. A. Tavarez;citation_author=D. D. Boler;citation_author=J. E. Beever;citation_title=Effects of the porcine IGF2 intron 3–G3072A mutation on carcass cutability, meat quality, and bacon processing.;citation_volume=92;citation_year=2014;citation_pmid=25367517;citation_doi=10.2527/jas.2014-8283" />
<meta name="citation_reference" content="citation_journal_title=BMC Genet.;citation_author=D. N. Do;citation_author=T. Ostersen;citation_author=A. B. Strathe;citation_author=T. Mark;citation_author=J. Jensen;citation_title=Genome-wide association and systems genetic analyses of residual feed intake, daily feed consumption, backfat and weight gain in pigs.;citation_pages=27;citation_volume=15;citation_year=2014;citation_pmid=24533460;citation_doi=10.1186/1471-2156-15-27" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=P. Duggal;citation_author=E. M. Gillanders;citation_author=T. N. Holmes;citation_author=J. E. Bailey-Wilson;citation_title=Establishing an adjusted p-value threshold to control the family-wide type 1 error in genome wide association studies.;citation_pages=516-516;citation_volume=9;citation_year=2008;citation_pmid=18976480;citation_doi=10.1186/1471-2164-9-516" />
<meta name="citation_reference" content="citation_journal_title=Anim. Genet.;citation_author=Z. Edea;citation_author=J. K. Hong;citation_author=J. H. Jung;citation_author=D. W. Kim;citation_author=Y. M. Kim;citation_title=Detecting selection signatures between Duroc and Duroc synthetic pig populations using high-density SNP chip.;citation_pages=473-477;citation_volume=48;citation_year=2017;citation_doi=10.1111/age.12559" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=R. J. Elshire;citation_author=J. C. Glaubitz;citation_author=Q. Sun;citation_author=J. A. Poland;citation_author=K. Kawamoto;citation_title=A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.;citation_pages=e19379-e19379;citation_volume=6;citation_year=2011;citation_issue=5;citation_pmid=21573248;citation_doi=10.1371/journal.pone.0019379" />
<meta name="citation_reference" content="citation_journal_title=Anim. Genet.;citation_author=P. G. Eusebi;citation_author=R. Gonzálezprendes;citation_author=R. Quintanilla;citation_author=J. Tibau;citation_author=T. F. Cardoso;citation_title=A genome-wide association analysis for carcass traits in a commercial Duroc pig population.;citation_pages=466-469;citation_volume=48;citation_year=2017;citation_doi=10.1111/age.12545" />
<meta name="citation_reference" content="citation_journal_title=Genetic epidemiology;citation_journal_abbrev=Genet Epidemiol;citation_author=X. Y. Gao;citation_author=J. Stamier;citation_author=E. R. Martin;citation_title=A multiple testing correction method for genetic association studies using correlated single nucleotide polymorphisms.;citation_pages=361-369;citation_volume=32;citation_year=2008;citation_issue=4;citation_pmid=18271029;citation_doi=10.1002/gepi.20310" />
<meta name="citation_reference" content="citation_journal_title=BMC Genet.;citation_author=X. J. Hao;citation_author=G. Plastow;citation_author=C. Y. Zhang;citation_author=S. T. Xu;citation_author=Z. Q. Hu;citation_title=Genome-wide association study identifies candidate genes for piglet splay leg syndrome in different populations.;citation_pages=64;citation_volume=18;citation_year=2017;citation_doi=10.1186/s12863-017-0532-4" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=B. J. Hayes;citation_author=P. J. Bowman;citation_author=A. J. Chamberlain;citation_author=M. E. Goddard;citation_title=Invited review: Genomic selection in dairy cattle: Progress and challenges;citation_pages=433-443;citation_volume=92;citation_year=2009;citation_issue=2;citation_pmid=19164653;citation_doi=10.3168/jds.2008-1646" />
<meta name="citation_reference" content="citation_journal_title=Molecular biology reports;citation_journal_abbrev=Mol Biol Rep;citation_author=X. P. He;citation_author=X. W. Xu;citation_author=B. Liu;citation_title=Molecular characterization, chromosomal localization and association analysis with back-fat thickness of porcine LPIN2 and LPIN3.;citation_pages=1819-1824;citation_volume=36;citation_year=2009;citation_issue=7;citation_pmid=18989753;citation_doi=10.1007/s11033-008-9385-2" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=Z. L. Hu;citation_author=C. A. Park;citation_author=J. M. Reecy;citation_title=Developmental progress and current status of the Animal QTLdb.;citation_volume=44;citation_year=2016;citation_pmid=26602686;citation_doi=10.1093/nar/gkv1233" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Protoc;citation_author=D. W. Huang;citation_author=B. T. Sherman;citation_author=R. A. Lempicki;citation_title=Systematic and integrative analysis of large gene lists using DAVID bioinformatics resources.;citation_pages=44-57;citation_volume=4;citation_year=2009;citation_issue=1;citation_pmid=19131956;citation_doi=10.1038/nprot.2008.211" />
<meta name="citation_reference" content="citation_journal_title=Anim. Biotechnol.;citation_author=J. H. Hwang;citation_author=S. M. An;citation_author=S. G. Kwon;citation_author=D. H. Park;citation_author=T. W. Kim;citation_title=Associations of the Polymorphisms in DHRS4, SERPING1, and APOR Genes with Postmortem pH in Berkshire Pigs.;citation_pages=288-293;citation_volume=28;citation_year=2017;citation_doi=10.1080/10495398.2017.1279171" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=M. S. Khatkar;citation_author=P. C. Thomson;citation_author=I. Tammen;citation_author=H. W. Raadsma;citation_title=Quantitative trait loci mapping in dairy cattle: review and meta-analysis.;citation_pages=163-190;citation_volume=36;citation_year=2004;citation_issue=2;citation_pmid=15040897;citation_doi=10.1051/gse:2003057" />
<meta name="citation_reference" content="citation_journal_title=Reprod. Biol.;citation_author=J. Kwiatkowska;citation_author=B. Wąsowska;citation_author=P. Gilun;citation_title=Expression of hypoxia inducible factor 1α and antioxidant enzymes: Superoxide dismutases-1 and -2 in ischemic porcine endometrium.;citation_pages=289-293;citation_volume=17;citation_year=2017;citation_doi=10.1016/j.repbio.2017.05.013" />
<meta name="citation_reference" content="citation_journal_title=Yi chuan = Hereditas / Zhongguo yi chuan xue hui bian ji;citation_journal_abbrev=Yi Chuan;citation_author=D. Z. Li;citation_author=J. X. He;citation_author=M. G. Lei;citation_author=D. Q. Xu;citation_author=S. W. Jiang;citation_title=[Polymorphism in exon 2 of pig FIT1 gene and its association with fat-deposition-related traits];citation_pages=375-380;citation_volume=32;citation_year=2010;citation_issue=4;citation_pmid=20423892;citation_doi=10.3724/SP.J.1005.2010.00375" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=R. Durbin;citation_title=Fast and accurate short read alignment with Burrows-Wheeler transform.;citation_volume=25;citation_year=2009;citation_pmid=19451168;citation_doi=10.1093/bioinformatics/btp324" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=B. Handsaker;citation_author=A. Wysoker;citation_author=T. Fennell;citation_author=J. Ruan;citation_title=The Sequence Alignment/Map format and SAMtools.;citation_volume=25;citation_year=2009;citation_pmid=19505943;citation_doi=10.1093/bioinformatics/btp352" />
<meta name="citation_reference" content="citation_journal_title=J. Reprod. Dev.;citation_author=Z. L. Lin;citation_author=X. S. Cui;citation_author=S. Namgoong;citation_author=N. H. Kim;citation_title=Junction-mediating and regulatory protein (JMY) is essential for early porcine embryonic development.;citation_pages=361-367;citation_volume=61;citation_year=2015;citation_doi=10.1262/jrd.2015-021" />
<meta name="citation_reference" content="citation_journal_title=Developmental biology;citation_journal_abbrev=Dev Biol;citation_author=T. Ludwig;citation_author=J. Eggenschwiler;citation_author=P. Fisher;citation_author=A. J. D. Ercole;citation_author=M. L. Davenport;citation_title=Mouse mutants lacking the type 2 IGF receptor (IGF2R) are rescued from perinatal lethality in Igf2 and Igf1r null backgrounds.;citation_pages=517-535;citation_volume=177;citation_year=1996;citation_issue=2;citation_pmid=8806828;citation_doi=10.1006/dbio.1996.0182" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=Y. Ma;citation_author=X. Ding;citation_author=S. Qanbari;citation_author=S. Weigend;citation_author=Q. Zhang;citation_title=Properties of different selection signature statistics and a new strategy for combining them.;citation_pages=426-436;citation_volume=115;citation_year=2015;citation_doi=10.1038/hdy.2015.42" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=Y. L. Ma;citation_author=L. T. Gu;citation_author=L. B. Yang;citation_author=C. H. Sun;citation_author=S. S. Xie;citation_title=Identifying artificial selection signals in the chicken genome.;citation_pages=e0196215;citation_volume=13;citation_year=2018;citation_doi=10.1371/journal.pone.0196215" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=A. McKenna;citation_author=M. Hanna;citation_author=E. Banks;citation_author=A. Sivachenko;citation_author=K. Cibulskis;citation_title=The Genome Analysis Toolkit: A MapReduce framework for analyzing next-generation DNA sequencing data.;citation_volume=20;citation_year=2010;citation_pmid=20644199;citation_doi=10.1101/gr.107524.110" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=R. S. Meyer;citation_author=J. Y. Choi;citation_author=M. Sanches;citation_author=A. Plessis;citation_author=J. M. Flowers;citation_title=Domestication history and geographical adaptation inferred from a SNP map of African rice.;citation_pages=1083-1088;citation_volume=48;citation_year=2016;citation_doi=10.1038/ng.3633" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=R. Nielsen;citation_author=S. Williamson;citation_author=Y. Kim;citation_author=M. J. Hubisz;citation_author=A. G. Clark;citation_title=Genomic scans for selective sweeps using SNP data;citation_pages=1566-1575;citation_volume=15;citation_year=2005;citation_issue=11;citation_pmid=16251466;citation_doi=10.1101/gr.4252305" />
<meta name="citation_reference" content="citation_journal_title=J. Leukoc. Biol.;citation_author=M. P. Pinho;citation_author=I. K. Migliori;citation_author=E. A. Flatow;citation_author=J. A. Barbuto;citation_title=Dendritic cell membrane CD83 enhances immune responses by boosting intracellular calcium release in T lymphocytes.;citation_volume=95;citation_year=2014;citation_pmid=24436459;citation_doi=10.1189/jlb.0413239" />
<meta name="citation_reference" content="citation_journal_title=Current biology :  CB;citation_journal_abbrev=Curr Biol;citation_author=J. K. Pritchard;citation_author=J. K. Pickrell;citation_author=G. Coop;citation_title=The genetics of human adaptation: hard sweeps, soft sweeps, and polygenic adaptation.;citation_pages=R208-R215;citation_volume=20;citation_year=2010;citation_issue=4;citation_pmid=20178769;citation_doi=10.1016/j.cub.2009.11.055" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=S. Purcell;citation_author=B. Neale;citation_author=K. Todd-Brown;citation_author=L. Thomas;citation_author=M. A. R. Ferreira;citation_title=PLINK: a tool set for whole-genome association and population-based linkage analyses.;citation_pages=559-575;citation_volume=81;citation_year=2007;citation_issue=3;citation_pmid=17701901;citation_doi=10.1086/519795" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=S. Qanbari;citation_author=H. Pausch;citation_author=S. Jansen;citation_author=M. Somel;citation_author=T. M. Strom;citation_title=Classic Selective Sweeps Revealed by Massive Sequencing in Cattle.;citation_pages=e1004148;citation_volume=10;citation_year=2014;citation_pmid=24586189;citation_doi=10.1371/journal.pgen.1004148" />
<meta name="citation_reference" content="citation_author=R. Ros-Freixedes;citation_author=S. Gol;citation_author=R. N. Pena;citation_author=M. Tor;citation_author=J. C. M. Dekkers;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=P. C. Sabeti;citation_author=D. E. Reich;citation_author=J. M. Higgins;citation_author=H. Z. P. Levine;citation_author=D. J. Richter;citation_title=Detecting recent positive selection in the human genome from haplotype structure.;citation_pages=832-837;citation_volume=419;citation_year=2002;citation_issue=6909;citation_pmid=12397357;citation_doi=10.1038/nature01140" />
<meta name="citation_reference" content="citation_journal_title=BMC Genet.;citation_author=S. Sato;citation_author=Y. Uemoto;citation_author=T. Kikuchi;citation_author=S. Egawa;citation_author=K. Kohira;citation_title=SNP- and haplotype-based genome-wide association studies for growth, carcass, and meat quality traits in a Duroc multigenerational population.;citation_pages=60;citation_volume=17;citation_year=2016;citation_doi=10.1186/s12863-016-0368-3" />
<meta name="citation_reference" content="citation_author=H. Simianer;citation_author=Y. Ma;citation_author=S. Qanbari;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Development;citation_author=D. Später;citation_author=T. P. Hill;citation_author=R. J. O’Sullivan;citation_author=M. Gruber;citation_author=D. A. Conner;citation_title=Wnt9a signaling is required for joint integrity and regulation of Ihh during chondrogenesis.;citation_volume=133;citation_year=2006;citation_pmid=16818445;citation_doi=10.1242/dev.02471" />
<meta name="citation_reference" content="citation_journal_title=J. Appl. Genet.;citation_author=M. Switonski;citation_author=M. Mankowska;citation_author=S. Salamon;citation_title=Family of melanocortin receptor (MCR) genes in mammals-mutations, polymorphisms and phenotypic effects.;citation_pages=461-472;citation_volume=54;citation_year=2013;citation_doi=10.1007/s13353-013-0163-z" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Hum. Genet.;citation_author=P. M. Visscher;citation_author=N. R. Wray;citation_author=Q. Zhang;citation_author=P. Sklar;citation_author=M. I. McCarthy;citation_title=10 Years of GWAS Discovery: Biology, Function, and Translation.;citation_pages=5-22;citation_volume=101;citation_year=2017;citation_pmid=28686856;citation_doi=10.1016/j.ajhg.2017.06.005" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=K. Wang;citation_author=M. Y. Li;citation_author=H. Hakonarson;citation_title=ANNOVAR: functional annotation of genetic variants from high-throughput sequencing data.;citation_volume=38;citation_year=2010;citation_pmid=20601685;citation_doi=10.1093/nar/gkq603" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Rep.;citation_author=X. M. Wei;citation_author=C. H. Gao;citation_author=J. Luo;citation_author=W. Zhang;citation_author=S. H. Qi;citation_title=Hec1 inhibition alters spindle morphology and chromosome alignment in porcine oocytes.;citation_pages=5089-5095;citation_volume=41;citation_year=2014;citation_doi=10.1007/s11033-014-3374-4" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=B. S. Weir;citation_author=C. C. Cockerham;citation_title=Estimating F-Statistics for the Analysis of Population Structure.;citation_pages=1358;citation_volume=38;citation_year=1984;citation_doi=10.1111/j.1558-5646.1984.tb05657.x" />
<meta name="citation_reference" content="citation_journal_title=Molecular biology reports;citation_journal_abbrev=Mol Biol Rep;citation_author=X. L. Xu;citation_author=X. W. Xu;citation_author=Q. Yin;citation_author=L. Sun;citation_author=B. Liu;citation_title=The molecular characterization and associations of porcine cardiomyopathy asssociated 5 (CMYA5) gene with carcass trait and meat quality.;citation_pages=2085-2090;citation_volume=38;citation_year=2011;citation_issue=3;citation_pmid=20859690;citation_doi=10.1007/s11033-010-0334-5" />
<meta name="citation_reference" content="citation_author=Z. Zhang;citation_author=J. He;citation_author=H. Zhang;citation_author=P. Gao;citation_author=M. Erbe;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=X. Zhou;citation_author=M. Stephens;citation_title=Genome-wide efficient mixed-model analysis for association studies.;citation_pages=821-824;citation_volume=44;citation_year=2012;citation_issue=7;citation_pmid=22706312;citation_doi=10.1038/ng.2310" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Genomic Analysis To Identify Signatures of Artificial Selection and Loci Associated with Important Economic Traits in Duroc Pigs" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11.cover-source.jpg" />
<meta name="twitter:description" content="Identifying genetic basis of domestication and improvement in livestock contributes to our understanding of the role of artificial selection in shaping the genome. Here we used whole-genome sequencing and the genotyping by sequencing approach to detect artificial selection signatures and identify the associated SNPs of two economic traits in Duroc pigs. A total of 38 candidate selection regions were detected by combining the fixation index and the Composite Likelihood Ratio methods. Further genome-wide association study revealed seven associated SNPs that were related with intramuscular fat content and feed conversion ratio traits, respectively. Enrichment analysis suggested that the artificial selection regions harbored genes, such as MSTN , SOD2 , MC5R and CD83 , which are responsible for economic traits including lean muscle mass, fertility and immunization. Overall, this study found a series of candidate genes putatively associated with the breeding improvement of Duroc pigs and the polygenic basis of adaptive evolution, which can provide important references and fundamental information for future breeding programs." />
<meta name="og-title" property="og:title" content="Genomic Analysis To Identify Signatures of Artificial Selection and Loci Associated with Important Economic Traits in Duroc Pigs" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/8/11/3617" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Identifying genetic basis of domestication and improvement in livestock contributes to our understanding of the role of artificial selection in shaping the genome. Here we used whole-genome sequencing and the genotyping by sequencing approach to detect artificial selection signatures and identify the associated SNPs of two economic traits in Duroc pigs. A total of 38 candidate selection regions were detected by combining the fixation index and the Composite Likelihood Ratio methods. Further genome-wide association study revealed seven associated SNPs that were related with intramuscular fat content and feed conversion ratio traits, respectively. Enrichment analysis suggested that the artificial selection regions harbored genes, such as MSTN , SOD2 , MC5R and CD83 , which are responsible for economic traits including lean muscle mass, fertility and immunization. Overall, this study found a series of candidate genes putatively associated with the breeding improvement of Duroc pigs and the polygenic basis of adaptive evolution, which can provide important references and fundamental information for future breeding programs." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11.cover-source.jpg" />
    <title>Genomic Analysis To Identify Signatures of Artificial Selection and Loci Associated with Important Economic Traits in Duroc Pigs | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"AWYIZ0S_acZgHuxBonG4gvq_KcHERWFj0XiKH0wv-Vc","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"491452","apath":"\/ggg\/8\/11\/3617.atom","pisa":"ggg;8\/11\/3617","ac":{"\/ggg\/8\/11\/3617.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/8\/11\/3617.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;8\/11\/3617"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;8\/11\/3617"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/8\/11\/3617.full":true,"\/content\/8\/11\/3617":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022GIpqlyyDAcizfyf6EzUw9T2OvIVk4g8aMT8HRhh07FJRmjVDr7Kr2ggJWs8XxKDDAUOUH7pYO9x4DxSwKtzYEb2c51ScMefbHdz3WV43w27y2spiKVtKpyWCmFp2bKiTzJJ0vEwwpe6erRuBHlkBjghz0FTGtHc\\\\\\\/aMdFAdyvy8WB3RE745in3RL5LpqzpQjJ97sU\\\\\\\/ymLjfWf3its7AF4JoAuXLf\\\\\\\/fqjKXo1kAl2y6STsz0znMwOCPiLue1Yakmow6haLbfBJqJLT+iUabs4ptMUhrEkLqYrm4gIJImqwQ2z5jNGSSXANAtBGgZ914\\\\\\\/ej32xSNK7Nu5Q9yYWvhQy6ZvkUGiPdSio794aPLYIKCs3LJaEhLvAakNvLqVxM0LzMq\\\\\\\/NkbM4SR3FlJzgCPxe3Jas9RQjd3m4JugEfKNrnC78jnmubpD1+rnafUe6r2xSgg6GvGn9vV2GeWJBjcMiZKAQUH3XD1b0isCGHQIKH2ynf2x6lu29PPs7PvEYmT9gv8amQbeUtBOnLekEGXd84GrK1PwP7XwHHT2vK3yKXaSnq1SSARDdVKYBDkSw+mAanExDQXiTVa5Ls3jJLspBC3cGM54Jfuiav9iJzTUX9EgDN6EXbOSUF3qcRSQ0oRz7dfb6bh1b\\\\\\\/na3SQA3kPMR2od1vm5JnoO6FFznHHOQ8jT75aetefj48KsGQ0FmuGCJ8+aBueazyjm9oKGu\\\\\\\/lcQbea60HEPY6\\\\\\\/o6ASHJ\\\\\\\/3zKRTtHM5f7Zlv9JpGDrk9YLgAf7EtPQFk2\\\\\\\/d2LBqK+bAtTr3gHUp+OaoNvE6Xurv+6FR5ZBo8fgkgTBdlXPkDorqPhvJgb1JCRXi5ME6hib3RIl+N8f+SJicE2RtXxML5jHJvL3S+8LyPjOoD6HhtP6NNQKAXcgyK4MXfCW1xvqvC5A0htZ48uKxhfIkrGCijv7soQJEMcAUETlHnoPqEyXN51J2fIhQi1rVfB6XDaFvDT7d6LIpLXXmShAV0FBovLpdLGBRhTEacowkIwo4BRJpAkfA7UYICCwWfdfp1GXUErndH1E\\\\\\\/SjVyLtI4kxOs+\\\\\\\/hwB8KZJYaahj3z\\\\\\\/lVArU8GBcDl\\\\\\\/lfvVPJsiAuFGuZCGO03aYAHwI+tysG4wQH7tZY1FjaQhM3G9aWTVMAwQAB8pPG3nG5S3wupFi+754GRj+GhQkZFntKix75DVgY3yjE7OIqrTjAyUJyTWDQQbARzEF0c7hqGSALagKS7uLK\\\\\\\/VLKiQBiWw8KVLACHcZACQ5nHZbrG52dgIIYnbQiyfPulCA\\\u0022,\\\u0022iv\\\u0022:\\\u0022HZve8DcGF7nH\\\\\\\/4nHXV4BlQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002299d0cd89ce9dc1159ef21447ea07ec8e827203a93a757b304177ee2d1b0607b9\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022bVCrG\\\\\\\/FZu00Dyp3vLw+PnNL4YGsxDSCS3w4rUxbXoGnlvUgWeBsOTsbLkxBJySD3HO+mhhWBuc\\\\\\\/VWRiEiB2V99faD1MJDIxJL\\\\\\\/dwANf3UjllcgbAFQSOWlMLCMgOlCp3E+4\\\\\\\/b4pekS5jm1bHm2gLcd2juptPfzothbggDhrnh56fOAdIURvZnTU1QGt9hA\\\\\\\/WlkFd1pDRN8hM5cKY2wdJkPS\\\\\\\/sQEYIgvclkG+vVNPq1KNNR4n7PUHRRfvC+aZk9k1eXp40fFqPPi5eP1YrUbwWO88+BVC9VFtTg+JkuhA6QeHzaRF6uvoauuamSguGy8XArnZCrf9dK39pz7omRoFUBHtbwLV8gIFlcarjYVHRiHNC6ijdCHg4UlHVrFRXsHjSKBRGZFWYnB9E4NOu1hAhlk0xMD+EwSzCL+7Y7P\\\\\\\/CTeZA63LwxyJT\\\\\\\/qCQuVSTjYrT3Hsntv+Zc+bzIW6GMQEFEyzBzp\\\\\\\/E4r1f9MU9U93KKmubYCK1ENmTtMjiGePOW7kJlbWatcgHhCoA+GbLZIU3Nal6a5A7cwfbgB3kYWas0F\\\\\\\/InQrX7UGOQekzzPQRZpUD89YJ1HxICwMwwB9cHqymdqILp7LWWxlImN12Mo8vR+3XJ2eqT686yL6srSMMwcgjiN\\\\\\\/9iceTTw245gonlOf3JXGFeuAJSSKUHFCGQcSCpLOg1cR2WNyiiNojkvZlkiMf1ZD9ltbKIzw8p7kz5YUPiKhaY8djrRgESCrgkuFPloe4WbYbTpHz\\\\\\\/jJHyEJaVPxX\\\\\\\/K6xbjNbxeMQE69x1CTDh+o6uKRaVgi2bibgIhJGF3HR82D+D6qurqNv8JLhsLBBQ1pP80pPSUkMq0aitIQ9KB+rJBRBrszjezfyRKuQ9PiG9L7VOes+XbTAH7upiEdBLSHMAY2aeJVWyR+vIPEiydCENaPNhGJsb5A5q2z4MBhIQ7num1Uj02idk24d+PrA8ua0HgIXGUbFIjOvygDW4qw7sauAv1\\\\\\\/MpdAvDSSq5icfI1DEf6c7i79p34DJScW3itwr8hfmiWS80NkyZE7v+ssu4rT34veP89cfrMD17eiqazuOiFrd583BulqLSMlmrQdy+JEh0b9Rt5OYFvADdWE\\\\\\\/TQHSj2aSDp3Mut2457X0zn3Suv7GNBSo2GexvfARRYugUy8xnfP\\\\\\\/RnaFwxrplWd4wDwHVtMO6dd4RDllYTtgj9fbtsDX+BQWTtiuBPLp68CEJrBNdxKDAl7+w46qPW3\\\\\\\/x8vL3yWRIfJfyu5G+uA6eol8hw1VIy0YskngPZ3FmRT4Hy\\\\\\\/yj\\\\\\\/rJl7HnTruL7f2AQA2G+mbSXQpRBaYFG6wh\\\\\\\/aQRxxXIihx4GTWSm6fQzIYVnPnqzWakoTvDazeyyPj0YXuTDjzEfSgN+eDR+\\\\\\\/\\\\\\\/lFSFYPa00ufmkL1y709y8eYx6r\\\\\\\/i9DsA3R18mAFRnQ0tLF8Fr+NManqXlXBltbmm8WUh4omU4dOdg1PIQG0K3dxiEQ8Er7LDtdDPkoU+1akedvZNfxjtz7tRh\\\\\\\/ETXCCjeBOTr1Trx2J8cfFEqKQ1VfYzFxQBK\\\\\\\/bcWPRIwB0gVZT\\\\\\\/cmsMZR7p57s9Q+l7JZID3N7FjqqndBhXAQ1J9X3EF2nD\\\\\\\/UUutbgtNoRtRrQIhW2sjFDp3Cc+0ol3bXJI8hUqeu6Xl6b6bNT52rh7yKZP2oNU8s9bWYReydr7THZuMobwlS3BJtXz+iWLKXIxffX+hKXdquR+uLxFlL2HFa8Ij2fBNXuVHdb+wh+R8IND1eOlVXivzCUz7TOfkMYDcOjQ7zfc64051GMbpLaf4quUsGFoIAOizz2oBx34Bvg\\\\\\\/D9nLocbE8x3lADNlMM0ZYwyGWeEX9eJnAGeBsKpdbVnOicdryYeWobS0uZJ+dAKv0UZN7lL2qf\\\\\\\/XCXKS+g4Lyee0jsHcTX5WybtHklNmWpf85V2sy8vtxBebyQLmEWPxDYA76w+lttIGa15D\\\\\\\/dvSlhGtrq73j5\\\\\\\/2nSU\\\\\\\/tIpbzIffabn+mC\\\\\\\/Nu0If15Kqol+vSxtuOijU\\\\\\\/MwSSTUFOZNDXmvacQgzU0xFmIoofcENaSHdwczaDofPXmLrKfYTM1QfnOd\\\\\\\/WZHDyJPou3Rg82KuFexZGw6cahFbpi\\\\\\\/j3RyFas3cuxSMJnQO4Uw54EaDxUhCKNNSxGuLehF0ytH99XGSxBVM45wvPx8ifnIXhZsYd2KDLxh9JDfjae15A1mMPnlUoVrfZlPR4H6rvXfVHCy8yPkvBcHXfRXEzoqzztYSfKU2MCieVPINRch8BWLI0gpm58GA67lHzSIuw5EyBc0wwoLnEVyWaRKqK7FMsKs49LxjqfPx08LR6pJvaiXiEh6QFx89YpYX2q+Yto8lHNlkAXSJajqgZZmDeFdBsT6BkbHcAcdgHhm1pkkst+JudIeLSlgggqg+3GgyEZ6z3xl1xz4axu99B\\\\\\\/5Ul8OLuTLUYNpDNnNMrLgSsko+8Hum4TnSfzg0HKrtYwFcIUhkjMOHcp13AEmSqL\\\\\\\/vlUloASb3ROKjtwd2F5+ML9FD8JyLSrPHLgY97KJOrkQsD3sJHLdQPIaE1T9OIQSyQdr8wij6RkF5D5WBpMMzWKVHIcwv+7PVst943aIq7icHU7FOVzD6veziCygVjC7xh2sBxYzmk4Z\\\\\\\/mIwI\\\\\\\/OKG2Bpkydp1QR9K0Uf4AYkBPBnWJz1ma6RkQpgs2z5ft+j4oq4yLXbNkChrOHs9yIdG03nCOUsGwM13cYdlOiNithtXRdElE4AfBzBm10VFnXOCmREDZY474cQCsa9f7RA1RdASUtQ48sJj+mJsX6gaJsrfFGowNWCkouKk5yIWaoHEjxd8Dy9Tloao5\\\\\\\/D51dHdnCrLQ\\\\\\\/JQpp6iVo1se1xZVmitL0s84kCpcHEl2M5rUlXM77eIt0aa7ABUo5wIea3cBoGAvqPJWgh+Gp\\\\\\\/NE8T\\\\\\\/xDiDXZllx6\\\\\\\/YuxfPJXIMxPLbJy\\\\\\\/dsSeFl2E85SFLsLB+SLzlGDcl73+KnnP1WcA7a4zULl0v17ap6o9A0p4CEXQI16UyxEF7\\\\\\\/df0g0xyd\\\\\\\/ImcTETwnYR8Kc8o7t7Kwrnu\\\\\\\/6bs4hJw5BaHzxUUdfReh6Z7I4SM4rtSX3itQzEJUEBZhnkbxxeGTynX4cZ\\\\\\\/c\\\\\\\/eqr7vhcEQsiWDb9WJGfhUsHkYmMz4Ncaqq3E4QNd3qdHX3iRc7wjZYYDG7EImRwzFfwytLsU3My89FOUQudG0ytPtn+Rdi08maug2L0c+XfezJKerDxmL8wyP76sEng\\\\\\\/SddxLwLxabNYenuucWdQz7L11PXKp\\\\\\\/YYEovmSF7zeIQBwUR5xsxsScj7svrC0b4Ib\\\\\\\/2rJX4itjzk6ANJM63nhAEnS0ltVUZdUr+eT+1PRhABEeD9+vkxS2Oo54wSTQYhLe0qJqXaymL26P2RDyJHmLmC6pFmN2G36k2eRwsqH4x9eykktqtNF9Vol43+W1+rTlJx4yTfhaeUgxj2vyhl9mYwW15nVSzaU9oe+O7kol53BCl\\\\\\\/pjbssGIFY31WEgs1eP5e\\\\\\\/R0Qh11lKhgWE4rkKOkrBZrwj0boZBDcGu2b1Q5PuoJPZmnqmP89IX6anRiSxcF3d3EilOi5WminE5hEvledGLAcac9ZqqSn75BKLBhhuLDljxawznK+ihiaKwcrgD1z\\\\\\\/9Yt51FGwODRq5v5tEE0T58R1+r+kAYFQs1nEC2yWBHsQsk86244OBSBOSKDA7O1JuPze6awzFKP3AhggFzWUsSfcl8lRkNmCLAuq8Sc0maxy63x8XEXPhWy83gadyhkWdljmvB1d5wde6yiPXi5+rCSKQxKewBvgy2P6E\\\\\\\/W4T74amVPZTBmcGBrOQ6z3gkVw5lg7uhzkinlBv\\\\\\\/w7drrS+2\\\\\\\/It1gym3Jy3\\\\\\\/s6kDZMNQoyWE9rrWylPljLE\\\\\\\/HCj\\\\\\\/BW9nRkfRTQZNt8i0JAMsFiDorzjZzSxS8m\\\\\\\/091SMD6I0dO0A6049GKsM61iRtvpbnMeobCqnnpasuNv5DVYWoMJYiCUwwyfswpfiqSmCVZLsMIk7AwOxPJohJfE2Q6EPQcVhEmydYuiXfhKuyDtVM3ICcqlZz1\\\\\\\/0+3wOEIjryMWXQZxu6ZO2qO3MpVdl9PXL\\\\\\\/Y2q83\\\\\\\/0RFTCDnheffwLIoAWq6YNf41LOCvjfTXcFk7dP9NunT51fhP5q+B4WFaqJ5L+o76Zxorgn4pvpz2Z4ZlcHwZO7NWad4FzVPa1u35qf\\\\\\\/NLkXmk9wm5zb5scXoTsIR5FE1vDLccf5rHac17eCbuJJBixbSjdwuwkXL0o40eWgExLx5Jbpxm931Kx7kV1WwlLhf1lhgEv328ryi+iU7wyYtRNJw2lPp7YQgV1qGva35hGTKYhG8HmFgXbJ738LqTujVvffcteLl+FR+QM8woQxxLSs1JRMsjMRBVUKRBkhF4zwpBr3PgVPs7ObSDkQBkmVUfpYLCbmWNeKnqYKANOvvIdW8KOMmenVCFP8fsJ33Jkqm+pyNBRLSCqmrIt+SU1I2zBOy5yPOohlee5hwW+sNUr\\\\\\\/a3F\\\\\\\/aild\\\\\\\/HHToaScyeKL9Wy148dF0xCe6e+sHisySOj1RK+qBwvxEX8fMZW9BTbqjqpTgAdNtMYTQ5LAgOnjFIMZ90LZV\\\\\\\/vy0vSngRd8HaPGjmf0cfpoTRtlAge8WT+j85nnJGUZZR+ERg51\\\\\\\/Mg5ESn2qTbh525tR2MvJJxhRCRPJv1iTVcMQDi8Dxuut2OrCMrFBPdXhccGMO00eeHJlpL7oXa5kEivkNjzYtV4yYXLbU2w4toK1gTVlnK+xErkyljuRfFSxLgcMSNQlUiVlFm8Nw5xh\\\\\\\/XQ6PwSyysy98zZF5+CZTSuUxgdy9A+SIn5HWkVEIacoCfRTLzCdJb0qRAWwkQ2iSm0MJ0qEUKaiJdLxao2dEG9\\\\\\\/to5X3xf3PTjKv6pJQQ222g4BSOSVlbpgIeoXpbt8\\\\\\\/WsfGXy5szi0FA2fywavDxtWu4AuHF5jbmcjsiRtlERyy1GBB0LGwFrmYYm3hZs2UVGfsOmWwe2DFCUT4ml2NjoBajBMz0oxWF4oBG9rEHessx6q9EYIRRrkrGZPzwfx6CxQciCjeQylGXEdXFV8i8SaDMuY1SvBCj1\\\\\\\/L9NEShEfF1U4Y6bIDMDSncbFwPjeq57+U0GcsybQ0HO\\\\\\\/ez5A+qXfPjzeLsaBQD0prKcyxezcj6HUWx9hl5\\\\\\\/fqK5rqbFEHXpvLGRtlEyeH7J2W7Hig9u76\\\\\\\/h+fLKFgEyM2xKkcDwFEiNwOOdYIr840avH8\\\\\\\/uMD1RY68SPyF6s9aYDY0nPHOAYB82VF80x4gm4sB725fLA6feQ36\\\\\\\/6KHIZGkLgp079Ug5HI3PCFjAq5H4PCZv0AoEBadu3QAgnOJYMB5eYAOmHYVbyTYaLFOhtVWdXjSaFA7D32zJfgS5IjL0hXOaIcrZYQd2d7laTkaciJzQjecxum4Zk0iMWMB68HiXpR4K4Kq4tIggz2aee\\\\\\\/fjSVttiuYJKfJVGLyD622UeMilCXaMFEGG5uJM19RyYwGdaSA+PbVoRbzJmaWDQHVbifmNEwgJ7vMLEKdMX2uYSho4K6fSt3\\\\\\\/NYoPh55tttZO0si5R7k63vt84Mg60XKjrlYvRxwh6appHfNnzq0xgsCnA9pih0t7wUAxpvSjsH4ZGC13h0Zpw7uvr9L5uRHfQfLojZx4xTTG3PseisA80ZcJEEqt+Flxn+GUdNybf3irhLeVmjfCUQLSAvnCXvYEO2G1HV6Qh04r5V77oGYD+TQxY7CiuJRmYm7SmXA144HP3bEsadjAtu8raACWM9tBApnY+G6DIamLRu9Y0SrZqgctA0qaujP3QXnmQgs9yoU5bei4+1V3WXLfmC+pmlBafu4JOcveKVFmfCET2LZtKirbKRjYgtOM3kRJos95oy50W\\\\\\\/\\\\\\\/wGMXgr8JN1qplFwHAn9ZGEM05VFx8zUpk\\\\\\\/42+MzoawHrasp3R5M8cM2ERAc1nc4Q\\\\\\\/aw3i9WQy6+TeWuoKdwERfRpguMHjiaBH+CBkp9XDRaZ5c39cj4zWI5AZXRnG8cY7H1ofZhks4qEZ59zV6zOH5DKE97VCcDf5dO8TWSqr\\\\\\\/0UYzjDraV87gEVgJlK7gE\\\\\\\/1o4RHI8DlTfHNh7MN0TGO1ZvmhYcWBiI6hElMBKAckYEs7HXhKlvrHv9lUosFfD8525cM0F7LQpYFwqJfx7\\\\\\\/8XKFnZrlP8hy6rWMRj4hRAUeIAlfT5EvliTBGwkRTe+C82SOt1TH+aK4eCRD7YpTpdD1IQoaW9MH\\\\\\\/5g9p0POXmVKrRNqPG6SExp5TqvTRjx2dYb+UxElcF\\\\\\\/pOY+L7CzHXi2dcHXn56D8+o7xa08jyU+MnXQTAx3l5TD06nseUiKMFZg4diBRHNPaaxoPuqIJ+lm91SJHEPpduyn0+VZmi0x2VD1QMtvUgDcSUlubuMerk+6zlJQYISYCOs3wjQWtsJytZ9SUY0fLg++YDmOEHBUZSKRlvwD0c95no3cp188z88QCxGoKJyVfbadkDa+IgkRlkDrWMPvj96YqrH6QrLUcNPXPNO6uB6WtylBnQR1LCKw6Hhgy8pvRRLvqA9uIFjMxqIPPRAETxdvcmNrLmEQRjgkZc3PcJf72bVt6a8MMeRQ3EHP002atWbimquy903dGIOalmmhzZPZTAORwA+Nvc9HY3Jp5sb8n826VLqu\\\\\\\/zQXqvWD6GGZWZVyl1pfpXBAZS3+Dl\\\\\\\/tjDP2pPg1\\\\\\\/nfFC7V\\\\\\\/dPItncsmTo3Dvw5\\\\\\\/mPQRyfvSGDWAQjAsuiU1HOvXsUkELq2JyGt0o468FLk7M8pxi\\\\\\\/3AeX4zJ7wP2Irg39zbDX0Lt\\\\\\\/dg3oMhkJtPk1XQDhDmtkR4DptyfYJxzBRSd9jrm5A9ZxUSMQbqVIytcGi3nYncoEy5+8BXPdT91Qaq8kVVzeCFZqSklU4ymCIYhc1KOv7\\\\\\\/Ff6xyVfwwOU\\\\\\\/8NV6GyDoc1Bl7ZbXvzzMEv7s3nggiIe2qy7keFEj+0pev3XIr1+plfxu\\\\\\\/1q7gPpdaZu3cauT1NcWfrKFy0xMCYxJJ0Oqb\\\\\\\/Z+oE\\\\\\\/e+qqaCgRaeON0apxPOc7NSNSWaYdz2Xwc0t7de6sDc0MzyANdoxllim0I570oDKR6yiJsJC5vtifCTZJgAnSBOrP3cCTKVsIVGC6Ss3iHvGtbGTUHhvH4NjSOmTVBxyKtDp\\\\\\\/IuglM+rBCtH4KYGASBkTcfa1HJvmdPFO3L27GZcRKKMr8mTOPPUxT1xgVi9Q5XlRHYVVT15tJQKdPAAWdUby2zLKMLBmt+iftlK7B2kdd0Z8gAQespIhdDLQXZqAlm\\\\\\\/LQtFlQBatjkAw0gsTL7RDnWjcJw+J8jAYZncFarU2W71jY\\\\\\\/+UXtoDuZZIKEaqOUIjUfcX7LVjt\\\\\\\/LIHzeawn+6McyHRdbKoGY8XKz1qmhLQyYoyTiVH5PdcCz25cLk6bCg0On24pQgt4dR4hkPXSYyjE\\\\\\\/\\\\\\\/4+cJ6nIgEX\\\\\\\/SwNTy30Q0Y\\\\\\\/ICtze9ykEKf9PNI320qjWTO+cQidL8KhQvYdajY6BZndfBK+YLsSvbOXirHZ9FYg3j9M6RBZEdfi3EqCIQq2wnLfShJ8kb19yY2Nmi2BsODYhoy9OsPVf++Kum9zMHnvTJSEZvLFM6G+40vgXwlGg58s+nsFHWIQSEPCaQlJCOX5AoYFGwpVO5x7I+5eYUMifgqoiZ6eGTCp+oXI\\\\\\\/8xouc\\\\\\\/qdCKjeWAoK+7RazIFPO78X+t\\\\\\\/eNHTK4r4bs2b5O0t+7wjgJ6YFyl9dfQV1wWkFynYvuh9+NakZB+gNfJR05rv+l\\\\\\\/oQ7RK6rjbywkBn1ktJPdLjsnP95mpLSjD6HVk5hIA3QOECsEXHgsX9GDev+5BarrMWpTAbQNiXsUM+Vi9eZNRpA\\\\\\\/7ak3bV2HpYHU0IGQ2Ov8Pa72cMQnIfUef\\\\\\\/M+tUbyz8nmbu\\\\\\\/okl9jPGJdLqs1K6y9qIW2ETyeClJVX6VXphJEHRTfYQPlDtc0Oozz5badDxLo6BEmDMGWJr4N99S9ag6icJdctdvZCZTdH6yYfLmfAxQQjc8qpsj0PlcCZ32Uca+2LzBzI\\\\\\\/hPH9C0fWUJU54NLKTjvyU801t\\\\\\\/zXCbY9vJlzlNHTZtEqlteDq7oK9uFD6XXu1gjhLur894VguFr0vyLQxazV6HgyTfJF1VChHvaIesXQ0gdmGV5AwvSy96HRtVaKQBsHGJVQ3I0164oOG1o0Zb0jwWZ\\\\\\\/NJ4aWCvElLjzkad+4v136dmClu+lknuio6fvvyN\\\\\\\/W\\\\\\\/rXcvtuQLBCg0LmKy42acFLbv8txNYzmZeK3\\\\\\\/yuMM9p4p1ZmEssdHR3WQbxjIAC4vPu6BWDBWbDBrmO8VYytDOP0NgBe3nU1DAixEUyuiO\\\\\\\/NooiK8U9telqIvtDgzLMc9IgXXuoB98D4AL54skTwptVh+g1mlTyzxji9Wrp7tTcL+fjRMNEZcOCRAOOry8l4lr2+Ts+K5\\\\\\\/fMqAoCIEOOA\\\\\\\/KLX2RURho5OHYPT5mUBRV843SarOiklqzKypOtu8w4dC8cewMPUBbO4uM0OLEvoiOkZQJmUYQ7Ach\\\\\\\/05YTBduHspM5fovpuuFP\\\\\\\/UlEpnGpDZWqmIqAkArbeDpe+9MB8XK+G1KbK4LHIhceml4qKxaMUfqSUUqwWTtyj6GohqUmUMH5lj9t6MOH+S5AtiyDHTKxuQcgNf+uaVkZL6sS2NekScaKBNxJjNBMLlQfrWS9K+jS0bsuMB5uu331I0WJ6OGxQtVDdMMhmTlzclkHx2ZRkoom3L4bEO\\\\\\\/PbmFOHgWj5hzysNad4Z2BywjCucoI31vzT0P3pEqQBQljWSTSYkikFkaaRmJ+zNmIIkv78M5GM9Su7GMg4Wgt2VjnMlgIa47rUBSzOYgWs+rpU4TU+GHFVLBZ4eIw2KRLq8vQXMtDx0ndKRWL9LnYSVQoodwfyCZl4gjd42WeOxw4CEGIh7MQ\\\\\\\/qicHOb0bj8B1oJ4VwEGJSx8feCcQKy6OAx623ueIixfNujje\\\\\\\/3xY6HiwFTMkkAz+j1EBv4zoDh0bmZbHfq\\\\\\\/PaST9SyPzTSsVo5xln\\\\\\\/GZ8lFZRDRS6Dfaf9EJm3PxadS2KkNRxIwqglfHbrJRGX\\\\\\\/ciVqnsPsKMFzK0b557lWJJulej2Kj7IhbR1bsGjEKaCPA+r86tNky38avOCBLVaurnA\\\\\\\/MZDrWM3iljuQUXYZ\\\\\\\/bl9W6+neRcoKWD7X+K0MBENcuh9XD0hfux\\\\\\\/xHEfehNuM9QcrqWDomJQlkyy1R1Pp8UUcGHSR+oNLBpMilzPFGtpOsz5BVrF75eQJ4n2jmuATng\\\\\\\/P4ksxThheZR59AcvVQr\\\\\\\/AL\\\\\\\/6YFIlAaULReEvbfKa6bDRt0cqD6Yf83JoY6saiAOSqeE7gPBi6T7mAv3DhyV27QbFiHazTbrGSMDDsYGHqFSKZcFMZd\\\\\\\/IQ7a5epJLRKHx\\\\\\\/toM+y+IldKPF9uha3B+b2eGTCNISXo1IOddDrFcSlVChpphxLVA5VxL652iyGj3thjXh1ye0T2SPs7Vg2Qu+1ENz0hciQoQFiSBzqcX3Xxjq0AKzg\\\\\\\/su5pgwCYbEhPsG2dRnR+zkm8ZbhneLl\\\\\\\/lSwcueumb3BbfJEuExfnKvzb\\\\\\\/\\\\\\\/tTMvpm9bHjzO8PfTseTGHiOFFJpZm6gpqqLElV7G9QVLZ\\\\\\\/lwux7Ft4BSRzUVbbiEd1G9CRII5SxYgB0+iBqvpEjWAq+nnMmLkRSF\\\\\\\/ZXV8PwI4IrAgIek6hTT23HUckydVCEwZsIS7O2o3+o2+pzAsFsPV2LbXHofCXRfDbIBYnkTIvs6TLTBj+5G3zTdjYcJNWHVeIjB4BmnaoUxB+hA0O8o53R8rQ+HNWqeXnFPScJx2fLaS3n5FbczC3p\\\\\\\/MOzPyISS0BkJ0+nmQXwQ3LZnONmHvreEAIadVgAZX3TaRzWGawaGyivxPugzuXJUz08uByPyQEyWVRQF\\\\\\\/oJkyUDkVtAKKFHHLyacvrOG0Y85SeFxpdm4mJ8nSh+nolZkLHqdzikQkIPHs3oo+udlU2jxfL2wefQJquPW3d5XtEaRD66lv0++BPTUUHnv5qcETkOnpmmoiJyo7NqYeM0yExiMDqyu8TWuzdweuE0f6Q1K1C909+bTDGSODE4JFWjTqHZUcjVpehVqUw737SZj78jtVzpzj9hpe5mzqcD3VU8SEkHG4\\\\\\\/sBz1a+GjkVCHBI3mCQhpmCqhtktH62jRXAYeqoI24ny1ivdDJB\\\\\\\/4P5I81xRnPETWdaI+v0TBpbLMuvgSw8X5RcL79nDpsLRhEbccoJR89ywv519NbZKBgc48pWm0B0xuT+yN78RFJe6Jj0Lsx7tGIeUone3LmDDmKkw8HtSnuUOACy124vKDm8BziW0L62qByDeRdDbX7+FKV\\\\\\\/X+px2cANgi1rIUG4UzocdgXBzQG+FIizZXJ7lfEZ7yqceBR7rVSqoajE65VPNHL6DRRmUCkJfTXl5kkgoNbG7WPAtQ6KenOyHx8mbxYomP9oinQAzSIN32\\\\\\\/S9UF077o\\\\\\\/5GL+IAL+gDgFwKV1c02F1ZZY7Bgi+2IFJajf4CGf8hRgeTY\\\\\\\/Utx8HCwEScNW5Ql9xlMCHopnAx+spwNwacFgzMJGxlwdwjHqDImn77Lo6ISmDMb5pfwcuI5S90EOAoNZQ\\\\\\\/vFrBb9Xl9OLFnrI8RIoXdk\\\\\\\/jvBh23L1Z+7XxDpMG\\\\\\\/ZOkHRzhS3dzo21Kk3yIWGQoGAj+HKEP7k427D2zz1+nmQ5wmHkCrE5Vd171La\\\\\\\/+SsrASHq3eUoKO8F4Vy09zcTUu38arROmIq7uHAwIY3GqLzbwa8eK\\\\\\\/jgEmQBf5jZE3Wh1mmsuyszauL7PyU\\\\\\\/AdxkX8lxpIMebE+3MKqDHCfoQWcapZrPA3VQwHi8qn5I8XwUcKDndjBxPY3h7aFdWwnPhG68LmgkWwoZF3J5k39h4XMmnjlW1byfAew26JMOKYozLZDxZckWgn0VwEpE\\\\\\\/6yMecDnD7VOOK5uUYkqP9R30R+9D56kxBbRfXj1cmqgi8stUjjwzORR3w+bxJxpuGzBtLmGu2nftGc4tFuT0Z8DJmZYczD5Pg\\\\\\\/xKDjopx5kJDN1HjBQyUINyXBw9tpIH2BjrJY4WYi5P0tSkewD6\\\\\\\/yJ+GOMm1oeILJCTf\\\\\\\/1qdJ8E5fhkMQQ4VbONI7Ef5KB\\\\\\\/nkE4XIlHOVKgBJAf49XqREjHjhzSD8qoLhUgcXSkGVmoTBXoOhxcFn7wZE+9EF7sBqd\\\\\\\/V2ej\\\\\\\/8NSu+7wnGUUX5+ae7dXXjL4Mt9W3+K2Kb6liICFDulyRE1VbZDcMcscHIllvSGLhPz\\\\\\\/BLOHxShU3jFudFH5PHoaMNJLpH4k7LUka51hwL70\\\\\\\/9d9MnFYLJYA67LJ7XYGJxgBYviKzgMCvvJjOZavcC8bwcyulI8pL0GdCAZJPYyAbQU8l0+SdXdAMBQFkKU\\\\\\\/61MNUdR3gjCiWOeGhCBeYdQ8rmcHfG17wIPbu2nAEfZ3aOrV+U3zhRuFnerV6QezREQjF52Os0j+jp5Jk8MCfOzZ79NoAeEcUVnwOciFjbfPkZKzF9BL733miii39sGAcFxU31RiX2narCHFytu6haoZBxgZ+6c23fKmbNMuaW6sNAge8YtbrKMAYMhGacW5Aai0lTlsaoV7QzU0EzxLdcuDY+pAGTf9BkDWz8ECFmR0vwrZlsVr9yZqDDrUv3z80AkVFXQ+lZ9+\\\\\\\/EByimAlwGJRZX48AKmyQEdghlVcwUBTIujpL7D7p9\\\\\\\/P9qjEvD6zPJLlxOFHsj3dbbrDqbJpDaWbo73meaOINcNEAeeuQD+Xz\\\\\\\/egGMyMA4GpMKMBl9QxyquHEsJhlNH7F\\\\\\\/FcTVb0tj+er8nKTJS+H\\\\\\\/JAX8IfUBF01\\\\\\\/rLCVjYfPsfM3ibJDxfHM\\\\\\\/jlhPzB97R+qK\\\\\\\/TBVUMYyzKUCvSYPSy3YGPZPIGV+K18n6n+HjSakcpr7YhpUSWLieDS8B8Dr7l\\\\\\\/kkJ9mPQZEMgsm7hZWiRnT+sE\\\\\\\/+7JtVqw8l9JmFv\\\\\\\/c8pP6M5bUPfCJ6uiSu7nehPNr+0aa7H2yb4vaRGiQYFdXTuY0SI32tZo7bE3CP\\\\\\\/NYjIWEMPEjmCfxY03\\\\\\\/W8St+uL30C2UhlUw3PfML5wpFfpV+zyjKwGwREWKf+2Bq5bOk8jSZr4F2I1+e4ctukKECUlAZIPK4iofnofqQN8hLRixniv5Gwk\\\\\\\/jHwINQe0ORj6IRUd8BlBcZZk\\\\\\\/0EUxmsqUi25kL+\\\\\\\/J0y\\\\\\\/Ktf69RpU95PGy1rWLtEjO6Ec2jemNufQLcYc4LJcHUNiBDkHbrRJBH00ZZp+uH1XGX8M1XFCmLJtrhqLGypGrgj\\\\\\\/KFQT2VzDFYOwikFU6gu6gjDqCPYH2iE3T6aRrJPlucZm7l5azXaaMFpWHC7Wq8taiHCTYXkOc4Tc+vDJZktl8yUca9bHh6wYy7I2+\\\\\\\/ungx1JXECi+jN\\\\\\\/gSgMkLANnL3nWoYm8FjW9sbi9TWZEYY9H6GWyklzSbtjaa1yivHniyERdEwzXLTnyJv1Nu8PStrJ5+6Y04udKDX29ooGGjKxvJHZU6dbdztyo\\\\\\\/PtKbSuASFJHgYvuirq5YCXajg6zp+zwKbIHL1niIjpgpOp79O1N9kutffJgIOzMqo3ytGsTwiSN+EhoIAbTx+0F\\\\\\\/Yq5zDmX+o8sjYRAH4Rw1ioJB4zMaaWcGbEElkJnNX1HU8V0XQvRuV3xSe2pp9HtID3ZGVaaM3L2N8kxo0n+bUbAPzt3G4PWQxdJblLWnABflfXw7iN2uSiwF6UYAcbnWt5LN3gXfAebt3fJJb2lekm7wXnYY34xUwIeSy+hmp\\\\\\\/rlv9j6YDEJg7+LUvBlAgtRtXs9dQtPqEja58YMqf3GcrVXCyQe4u1WZfQPCp9kE1EgELGoZWNFqWq3fLErKTb5BxIr5U7CWtX5BlpL\\\\\\\/yBPn1w1DfxL\\\\\\\/h+xUTg3wGkQRVSzWME7SWu2U03p0jZ6LkuSY8+RM8fSM5ypRN6zVdr\\\\\\\/EQKweQ8uz6NF2PCz6VmrK\\\\\\\/ud45jMMTMDdtwQV5eYMmetNIE\\\\\\\/DJh1GwNtx225OD9YTovkpKTTVehUyktbaRd9YQ9E+pQjaX35a+pXtKqXQBE86gkOOlnQxwrlxWF5xOj+LYQgWW1TDkSHukLC+2gUXVQeRM0bCmuXVdmIkQGF5JnKBqnx7S7nuklQobeGIltTn47qwTqpMOmqlWAZPbYkQeO3e\\\\\\\/vqZ+\\\\\\\/Q3mb5scdbk6BQc8PW6xGTkRbtxAl716uPR3i\\\\\\\/TXjrQLnMyBhiFWSOYz1tdNoxM+jW4eV91UaYowS8Ubllcf3hn+R\\\\\\\/jyEJla6OC8OJYkbHoZB620KXQ3immuzCZbqPbHZFSu5YwLM3JQ7RDDwas4f9eDUnBNZ\\\\\\\/MEsa6oFrtKHziptuaRBcA5jO53SwMwAd0Ir+t+DSv7U4dg3XzEMpXHA7PFVEUoGjP6joYUKHIiLFsB4nShHdj5itxXV8OyVoCJFS2HytJKtGRsEWA3vt+Cx8MrTHg7HohMnRZiKhoziycOS3IhdrK+N\\\\\\\/isTAJVgbDq0DPZWPBeEbnOFvOploRYQxZUc289Oi8LIi96qAMn4gvYN9iNghOvBm8vQ7ZRQZLj9hwDTgVvmBkU9hxNlo9QrD9jkO7pRByL7J1ld5VvnJjKDtYI7EZ3EmU\\\\\\\/q732EQSg\\\\\\\/Y4umVFWAgL3vD9x5L9zstIAbeCe9TPsX73r+tVQAR4mfvaXt+vetrItpKHvrVaCU+5OvMW7v37D8BZiGsL7aqznrVuCRsoaz1eEuudLGvOK2yhzZln76xzUwmzsCh7nvzVxunCOo2GhbSK\\\\\\\/n8nPhxelxE4QGZa2x14qrntZGiZHLP6Cf2n9K6WHeVT4gLwgaK1EfjQO8+eYRO1EBuKBW2l\\\\\\\/nE+1\\\\\\\/1Uj+gMXuQ+EcG7cv1tjXUZLzLNPZJRY8o5WOmt5Hmk0vv9K8OeoCp+AJLQePRgweMnPFoYKB3QKlJC9Dqnab8R5cw6QObUTKviU2KRiznDnShB82aKlf5re1IiZ2do1bGc9p\\\\\\\/2AugFsAThSkBfnXx9RwRhuZfPdS+2CZsdBoL+Dv5OjQDm\\\\\\\/X6b1SDxqX0HkoLgHVVs1DoNhLNVsDEjepvNYUDCadvCsh6ud+0EcG0HzbQW+nY3fY+qSf3InD7lMV6JazuueRyR0llHBsEycog\\\\\\\/libloYspRm74aac6HHB8u3VysTVdDIsL08V0pHxDdReDvqcGGpOxTe6wgwRLtmQcMRuodchznWSFk2g\\\\\\\/vBXj7IVIEC4Nla65\\\\\\\/8O5A7vb+hl6g81MZbM48\\\\\\\/nvA9HE\\\\\\\/axU9uKDZqKtL2FoCsfNS1p5c65JgWJOQ83R7uDhsi5NEeAEI2FtCeMGXQd7przXiKHSmpA2Lt2d3HURyV9esHi08gYkc64d\\\\\\\/cVfLtl27dgyrAB3BfGkMlHf6WHSZSilaUVvQxxzGHiMQjMpqHfrnVez5P7ELE14d3RDGqCPTr6vOAhykJNVqV38Dn98mIlEEqv7LgZA30tfSJZKZpHVEExqL2IkozfDrCtl3APzYGk1Y0sg3yizrTijcTNErUJzci5wsGJufwWOaF\\\\\\\/IkHi\\\\\\\/rZk8cdDz1epBe4T2UAtojjjcvyLw62ZBpJBkaPO4fMHA4keEuyQJtrMB9wjhSlEJjctMjje\\\\\\\/uNh1f9qIg\\\\\\\/zm\\\\\\\/kKsCXAcES28nkw6sNimSLpI8y8N3kpl\\\\\\\/Em6EHc2clrsXU7t79cTkEMR3qVWkATO5qTio0NUYX1cNh0YNRrJUU206Jwz7iThfJIFtmRFsJf4oCPVlR4vBga8t2J\\\\\\\/whG1Gb0qAOV\\\\\\\/uFUzoe5cYTk3FYhoVmG9gPcuMdwiMTLAMxcuntVkB+9CGnrJiFojzlCMLmS+YgsUqI5psUnDxa+uy+bH\\\\\\\/wEer1jbefL6mCyJAHn17bTx11tM6\\\\\\\/NN\\\\\\\/D3G4y9dRHV+\\\\\\\/vzcdFtLeIkZl4JPWZBXJv1Won+2nQJgha3PrXf+fcsQGwevftNuhXa+\\\\\\\/yGUkoqBR\\\\\\\/YdqgpeSI0sa9s9Sz462FDI5inp4YkzNKUfuBxOSvkTwz1\\\\\\\/f145ixct8hIUp091i0o5CwKMmtbFHgluXEMXgXTNkgt4dg7FGAjkAk55NlMKUGRpnE4XDbrTDnJMLxDqKJh8EW4z19OKF+9X4EwWxSKIYipwEPDNiAw\\\\\\\/erIQBsOY1Rd19h6qJbfcVUyIANRE\\\\\\\/h2ItnZmu3WAtDIRVUTIyNLpm17xORcFpvkQjDBOp20hExUL+CZju4HNaWZ2DWczWC9RlGdQLfEkkC+h0ghLqxfw5ZxXxHKxc0zGniqIGbwVwnEQRzmNMUOf5U5k+ruHn+UUKeRw9ZTbFXzLSDd0VXicFSj4w33w2izRN+YGK1skztouxWsRUW\\\\\\\/UVlB5\\\\\\\/tdzlc59zR9Tqj3S4AK80anE6oj8Jglo+3uMYCq7B4BEy2gguJ2kREKwsvCzeTP9WnDyi\\\\\\\/F58OSNf9ESKLCw5EHlbQstbDpxxmhQYujoUZcVnEVQyWYqjJD9UXAGM+3hqGqHjC\\\\\\\/loqhVIsujGYveUgoiZ+YzOqP9ST2yMpsYXCVOJtSgDhU\\\\\\\/jny2pZTYaevy6nHh\\\\\\\/rHb2ZvktFNflns40miW26Nj0ft34CG+liCnQeexvKcraRUAa2zUgII\\\\\\\/ZrV0J6MKWc5Ei86Pd28RJo3oxX4CANRFOWo4QQHPzctJywpXyksArcqzT0ZcovMZ+K46+FVEIqkXbZ5XxRq6HkmoJ1CsWwguMGmR6bWAFXHauycvbzMrXfrUB9Qe2GzdjHwUhtDaAEvPShJ6N6fpC2G9E27v9\\\\\\\/uoKNXW1NvbPHQCzt7ycZBdXr0mto0ll\\\\\\\/tfnsVTvfwDpl2FvyCm2aqhdKk\\\\\\\/gd9fGcQEYX+dhE13IGdnRqLaAC9XEJ08WZih7lujSni+1DCwn+3tptkPF8AjpYrTlUttkyEE\\\\\\\/cTnTHaIbTChigSDSgn+D9F97rjat4Ud\\\\\\\/X4K9XrBCUqiOGMYL5eYNgJoA9sd+h7d3Bt1Or0P36h1rwTt7mot90N6D6YmBrsrbADB0gF9TPSalAm2ixsbeFZnWanGh1S68lSPQ5ts\\\\\\\/o3LKI5GKl6ASsWd5qnBcbQaNbnpN5hmhwtpgIhJ1crScgrC266X66pfpNqMpextwcyuK2\\\\\\\/ly5iU7z2a6EWTw+qyMsyC850OThHP\\\\\\\/54BC+294+zonfJANsx300vTZYWI4Socjfz3SvDU9GTld0PVPfeaFh\\\\\\\/W3HY\\\\\\\/MD7+VB9Ozi82RdIyF8ipYfxdBh+WCe5+YA1RLzJMo7iWiD0v+0KvQvm+XV7K0cmgSo6bQDCPazpmqEFnQz\\\\\\\/KTn2g9Ih2D8saFUazK+7bkKWqtqdAQ9MEIICieFrrb8YsjM4lIdSybAEC5FYz8InhjT5AMN6D+eo0zukY8yiToleGO2\\\\\\\/NPS3EN26Gcy3noYgZlaJrrOHO5fYLO4xsM4uZb07SRt3Ek6ErRB9Gh2X9WiGlCjVuSf+7xurFCzmrd\\\\\\\/FrMR7nmMO31asofgUF16dHEq+dPWo4czO7G8VNsVgluAPG9pqb9ofegu+EbfFZz2ITBoiX7l9ArNqbvcF\\\\\\\/G4kuUeB4TxArGhcjJcKRC0aLOUpzDVsKA1V8NkqZkfa5fkwInc\\\\\\\/2yLNn3kEP4nn4e4HIR9ERnQePQ2nqoxEo4tYhawL7b+oy10SWbRPlkx8tr89+C+HRBNJg+wA1jSavRN8TtrpliCdXVB9\\\\\\\/3k3+7hfyIkMn2zNL\\\\\\\/HRxWl8O90rur26jVBarXrw7imlJYnB214ul8RmukBjapwKYdO8YPHuLoIDQSZ5ko+\\\\\\\/0bRiutZHiso+ukie7ppFpvVxiBQkGN7iTU\\\\\\\/yokpfigBhLtYFWgO+BgrqRhBSqyEYSlm2eutVwV4XOXEVW0rmgQcD4BR2ODRDetOUkCpTM+h7pHMkynHorVQ6Sk3v1QwXYsjZMT1bXP6QAOl9JXNPjiWGAWkMkEc053KqW+Gb7GWDRXwxeFRFCxrPdfD33ZaiTWFE7otPQg\\\\\\\/PGvVP7sD1Ave5RsC7SLniD63QhMUzMAWTKCrm3A+KqG5FCFyIVRhsR6BpORFi4AkJPTJQ4axLDbtb9M5KDH2mqlbJMXQSow4+rgs+GXGRfqMArINriWs3ipF5f2XCBvrRLeBMK1sdBlEH+7qK663+nIbQV5zhDidVYcpsudN4H4CpUjfjKGyTdtS+Z5ifRcFYkH2VUcvCXoNnIclv84aNlRgopd7htTa4gZhYW\\\\\\\/J4RkAr1z03Uh0hVFgGEbs0u+g0JnHnYNNh03Oc8UG9Swm4hT\\\\\\\/IqcpoUdM7yJDgmRHwmv3ZmGL1swPIZVxAjaTaMPCaMmGzW0gO2PnGjWaAF6lRuEWQASwPq0jdBybBR\\\\\\\/vOU\\\\\\\/7A7T6scCFkoypCj\\\\\\\/5og8s0fAecYINIlZbdLqat76rt19iwbsTugdljaxW97B\\\\\\\/GxJZcsHirx24y\\\\\\\/ljyL0Upx0Yx6Wb0\\\\\\\/yhfQsHpj0WS3jWNK3rn2mbaNcKrQnJk6hS9bVTP\\\\\\\/Aml\\\\\\\/ngdIps6RaTwDJY0q7RGdDzo3+J7OOraFEiEI844tsmvbhYWEEVFm6MkHmBsgD+wqY\\\\\\\/nIa7yABkefMPFngqSz32XFUG6OKkZx5f\\\\\\\/U4Q\\\\\\\/MGKgYglb8vjTtULQiH0BpVbv8l4nKb+EQK0ZR50Mhq74A5ml1zZEQsjZTECDhdZr5BLJTbWpLPQQ3Jxu0sESrS\\\\\\\/3o7xuFbuqpgs5qWqoLKun347CS9HD27ygzMVELWeT3LmwgeTOg2laSqTqFJ6\\\\\\\/Aa1TPx2wbR0DBU8RyzOSWFxORpF7SFWHXikBN7\\\\\\\/KClgu2d7He7tVU4us\\\\\\\/4aPtPXIXKcK7VC4pgZE5UWFODDPLumq+QBQqbonu8CKqnt0XEwMgqWGPpuK6rxMTz+WPaDTKfcyxNZNMyu36a+9NhH3Rw7Xo0\\\\\\\/uamHlC0xpWozoasRPJiXcRYfTPam6k5qzYK1s410OT0YXU7XDMejxT6g0HFNxm2ozhrLdfpNO103fC8QL\\\\\\\/T+P4sOkRzHfwTVPINTOPTqBvrltb9kQq2Kuu0\\\\\\\/K1Fptp2IekM2zvVIWNOn7Yq3w9E\\\\\\\/xpHXV6PD7kLoqoJT6l2MvpbsWR\\\\\\\/cXdQG79WwE9o2Ym4NrlgQJzNR9cgugmvcaxR0hCJ3RdU0Bj1lOvdzfKp+08jVfaStlxc0r3hOQCE1HZsxg\\\\\\\/BQu6q300uGFlwzu\\\\\\\/rEgwLwWcAciCH++JYaxo+f9Q1lWtg9hnbf+mQo3+0ThMjpIo+Xbe+UbU4pKDcku2poRbIweKiZuW9Fuyo6VziPco\\\\\\\/7VRFrY0Jqr\\\\\\\/8+rNjC4maPmFlIeo02vnWdAIpJmtTWP\\\\\\\/seSvOs5mX1PAl\\\\\\\/S3I73ubJDl7ATx59\\\\\\\/P4I8nXTYuN0ttK7PnQP5U0ITJMTS8RE5BWeHhuxzBrzFDLvyIjUI56iZcYbZe+4bRGiNyTWHBThadGdMW246oqbDYBFKy+r8HOx352Oeu7uJKQiyazB+FysKY7a6YnkmXe\\\\\\\/9of4xXjg6mq9NzYm7syEvldLSJ4dQpFmxg0p4XK5zpSy+JNxTSaV42Xqycv4832RDngCkyKIMnNOnr7T2+3mSiKT5r5dUlLCszy8h\\\\\\\/bGDslM6O3hSsgrZisfJYz+FGP28cbaVVrvmeFo1w6QUXPzEKT9UuI8JQhGss0wUV5TLSWNJvOqR76rPcCDVebwua1UHQeHkPvTuxxbr0uisV0zFv0GeSDjVy29yygnupKYi0KOxmO0OyoU7EKwvd4paAs9wkA2fYs87Lt7bKEw\\\\\\\/YWJmBZcv6\\\\\\\/4C0BT532cQgrfrBqL1cwD3Vw6ifjcDyo8iGT03eoE2kLUjgXUcFlYM4FF93h019MopbKijKdXCNuzhhxOBwtmqgFPNf28CIws9dIVU+iIYFIIBEAZFVfZNdgXYn0ZxcWbG+h4PThvQTVoVbUXePhqp8t9ox8qSSLkuhsRWH1Yx9\\\\\\\/bYoCxEwSGIjXjSZy6G0QWmhRZavxN1IQWvEum0HLuTHsLfCclPFCffY8L6sF\\\\\\\/6JlREtg37AHIzNxzAvmjrcUYXKnBKLrNKWBm2vUTpdjoyBxwa7yLYHvnn5M4xp\\\\\\\/DGmc1czF53I639vqi0DktzB2zyHupEAIxprv\\\\\\\/oHMZVabUhIT1J07rHgAEKuxPj7QtprEA80l4fUUAEOtSCZ4gZpW7bTw6FJewS568wsjnWrdr9RlZlFPbNPbu5M0C6TFle8Vh45rBE+\\\\\\\/vChBnVFuY54MRtTzOaDXeA6StEmBnpO0DWtrdSyrnxpOejL3np4q5yhReZFdJ15Wyf53TjVmICzp1ujkhPsiKg2d2NyjhG0SgypjqqpwfGQ3mGJzASKpbee9ZzDg2Fh88aF494Emiv3UEf8Ktqf+3Q2LfZNrU5btxnbwSDosf1fKDsBW\\\\\\\/koV4mkpK9oMiR7+TvTVS79yQZrZ3Jtqcq5XYZyb95KuehfFnQX3fp89vmF2RrI7+atv1PQdO89WuKD\\\\\\\/EdG+LtdZqnh\\\\\\\/3JHaNbNjcHhRxteynegQxYVPvw4uvuJSAW7xolNHCuHjGAPzOxHpTzT2T65Vob\\\\\\\/QGqgrEA9cisrsZn6Ly8b2CtuU7IVr4PyZ60TKZdiFxe+cgj9zXjq7E3DJmuuu\\\\\\\/BIur9V\\\\\\\/ox82yTTlMwBddeGfRPhvH\\\\\\\/Q8Xq3\\\\\\\/Kr5gNKuZCK9JNj6dNAF500zcxzWPDiNMguyhqzo4Qd3AmHqr5JXJFu+sQS3\\\\\\\/nDiypp55zUZU7Ci68+5DlNJLyyLgLwzoZpBg79V8tMnuRvgZqfabUGZg3yNHkGo0MbjETJJbYyAXIl6ul3hwMgQC66GHjx3aykGcksTyrpEgRR6YG2oZrtV7Yhg5D4iT02cvXVEkrUqSz529d9I5NGpR+lRHdcToVkMKESr3Y\\\\\\\/uISjQRTVSd9aLHNhha54a3scjvuqF2Ssno4qgj5kr6vZNS3Kn5y83IqXHPx3tXO9GCJ5n\\\\\\\/NbZ6T7Oo0DPZE7mn8hupsG\\\\\\\/qiMVRf1zCnedAnq59nujmmuXhFykex\\\\\\\/JxNlOrvm6A8MMpmi\\\\\\\/bUJVeJI1sTme1Q1zn02fX6N8EzbaTZ8Oc4FrOTDBnDgtTKi2CCjkrheZ5aoQQIvi7r931PMprFwroBMXr5SXcTtACMctYZ0yWEtnB8n3bR6kmbV1iu1feDa5161IcfnzV\\\\\\\/69GPqZKXFUVhU698W2VR2rjSrLcG6vy4XTydlBdZsCPfnXErBy6VSB\\\\\\\/9FAHoi3bPc7489RIe24+fKOGGKBY7POj3z0MDTGoJMaarJ7u6mvwcR\\\\\\\/fmcHoSQ2EbFkeAcFQzlCSTXqkZvJjYH8qDo79nEbkw6JUzMXah1t4Ca54HDtKl\\\\\\\/bEs0XYXmQPCxH58gYonSYfbWheElZ9NgXy8dFaGDUg1I8tymlX6\\\\\\\/wlKS0MPN5lJzcyjuhl\\\\\\\/vYEhqphCsNSy8DWi7MmC\\\\\\\/s3iVUL21ofiwSQ4h\\\\\\\/OxJhHcdxQIMr1a7TYpi+h\\\\\\\/VhxQNIvcbgkuzQ1nTgf1ePQvawC2OkafGybBNkvaGTYorCHH6F\\\\\\\/UrfMJxMoM\\\\\\\/2oaOzdasXsi\\\\\\\/mGm3X79oRPIDe+PKBBLhAVXReCX3lpiCwvdNfxux65jLLb9EtHJgzto+nUkzh++utGNsBxbRP65i8p7hfCI1HOGZ\\\\\\\/0rvpr+hSdDkE8TUly00+r43yzUwF8eiDvkC+X5Lti1j7l8fthfmXdbTcfQGEhFiOuCk2ztmdAlO6ef7yBRr2GQK7NxQJVLzAK2GVRDMi5hc0NtRZgfD1msQplK31dr97h0FZHfjmPONY+K\\\\\\\/t6+mlMSKbkjJJWuDSgYLU2Z\\\\\\\/BEaZJNAxfwl7S2l1S1DOvb0PJSReRA5gRvrv19JLIwiSU\\\\\\\/Y2ZQq0R1kDH4kBSbGjsWCQrJ9t5SuIGi1pu2LEPkyJiXMn1fU0MzHTpDoJOISEHKfXoiYN16iG8PI31YSe5pVbxW6mrtgwMDFznllIwXDHo97wJU\\\\\\\/BRfk4t4MINhtxaKZV7dXgImcH2+RGMXN1edY1zINr2VLhh3bNdEwL8+OcCG9K\\\\\\\/+EXv381PTvydGNr4lf3+KUjGGKtCCQ\\\\\\\/GM7JfMtXkmn2PEMg2SntbFGIXGOwuCbMraQgyIxgiKTn7Y39rszxKVoh6gezRZwzRhuy5WoxWrlj4um\\\\\\\/x4xuhOIZ\\\\\\\/oXkZik28RfJPC\\\\\\\/Vfu52KYU5gtUXo062xJ4IeGLjXoFO58+rBZIbNcrKYXcbcKTWmqaAI7Wd5hqM+OSB3wf3FnjYiEE1Ga6Zyc+WLxe6SyWVmp9TDXuVqeUx6dDZJEiGyZ7FpdqwD\\\\\\\/WauwlJK5pTSGyF53cJMf16UzEuCrCw6FRe1NMZmC+x+RKgdM2FvU8Lw+b2IQqCWXYjkpLVnCAEi\\\\\\\/d40M+Ua9iY6rEt9e8hEpII2K7ODmxi2scBulivqI0gT6fhdCH255y7FgUHym7Mp+qgG8zSqnKcF7N+KIYwrpmxpJfHS3s\\\\\\\/BjY9JMMfs0sTGZinzJwIs8MR4JX045X1LNo9GpPePySOH0zKLqq\\\\\\\/oOZHdxEDRevJzOjDMHenW849Dc+tLqQn+RWLJXu40EbBPk3uj4NYnhQpBAMX+FEC16EJaiBERiYVY8regbbHnHfDZ2bj8CBMLrMDtY5G8DiANMs3\\\\\\\/xCk\\\\\\\/Gj6NenN+P0R27r1RclLIqKgNW2u4q66eoZOtilyiVVglb\\\\\\\/2dyd1mFJx+MtmIa7ZdkTv6tNLb6AUZmEBzHoiWl7MW7qPb5JHH484GbxEfrPUqQ7znR8J+u\\\\\\\/9H4WbQRbXoWrB6GH7Admg+3QxIjykVkaP60i6W502oSU9dRw2xta0Fc0oi8Q1GC5Kxf3qZ4EXA6yyhFQ\\\\\\\/OUICnn6DM\\\\\\\/+VIlNwlPYn5HN6seOlw94F6qlwp5RqXYHlWPRwV9y48MiOMYhtP1yd8CTNkndsPFkr\\\\\\\/s9lIX7\\\\\\\/xMnzd3Ctm3ZQrCu8caXmM0kfqK+lT8tiU7GHhyMT9Eb87xT7hMFB67EO1AbH1zbn8MZI0YFDcWlFHA2oEeeXDOePZBZ9OKLXBM43CLDbTgx5RyapP92bBydWug1jkbsUuqP1TDVL7D0AbIQIZp6kClZpztFzPfGji+wI4iG5J35y\\\\\\\/8qgAyiobB9zl7va4xEnh+W9rFZQIpSOYmfB1KTH07EORTTliujB81ew+e\\\\\\\/evo88LNhbia+g6HeKk7FHn9jGO38Tv45bS70OApfGttzbndeuu73bt7L6LlfF\\\\\\\/sssuA\\\\\\\/tfWa\\\\\\\/5sjYNVqrlp5\\\\\\\/8QrKltz3oEgPhBB4WP9mHgage+z5pckMQiUAgzvB1vK2pMVPnYT94za8nf5XYKcQl+z7iOoX4k2OoYjIfCskEZdWpoZy2ffCcRmAw65fbq5ZjIxZQQmsZdw2zmIOaBihK96Dcqe9xwMDMZJYyXVR2fAKxUNurWdMOmX7oBNcHUsZjTaCND6r0Qn\\\\\\\/317upxPinR68HAbEb4I2BfCLX8CObxFRTcXy2BrIBc3CYo8+h8RALb3focgOH0ewQk75P+UaEFQr1DaO2\\\\\\\/tcQ1tafeZyGIN1WHzVEmvSsIsPxG8gd4f4G8Flr0PT02qcEKs5FtfdLm2N+G9Xy7kJVK9oC7EtOMrZZ7Mkp1Tr0jNCAEndXBJk38PRSm2eK0O0aXnB1lqNzhW3g+1QdDd9Y1mxw\\\\\\\/iACIqpBuMl9zATmLt8aFHPnUoFWofg6pxdbawrftJY3KJ6s14abv4lUo\\\\\\\/Ett86C\\\\\\\/p27lKW3dUyj4f7bK3pbdsR+Abc5vIeqFOxb+lTCuog9S5ovyKXpXtWNCV1\\\\\\\/I\\\\\\\/dBVE1p90qUBhq9WLIE2xQi2rMEmYIV5wFZf4+zp+CkoY2Z1+Tq+ZV7jRHXBNk9vNGafYbRcGkckpjxicYAxb\\\\\\\/tItFapQkaf+C2BXiAQS6L1mPgRKX9DXDi\\\\\\\/3eTyGJGE1pPL9PSn3cUANTw9YoRAvmGxQR4H9sVGc4TeGQWIzILD6Gjhe9AwboxpYh9KkIwhKSaLuGiJDmI2lOMADov\\\\\\\/xX3daq2Nx44oAPBNZGpulSiWfn8ul\\\\\\\/648B638xyoTbs3YgbF7xxdftZbX9yUTyYurZaZh7tv\\\\\\\/UZ0oMtx7gHmmQ2F+m5sgXA5SBCIdDgFS0ePFw3EUmlbsV9pVIeJBproDTjeGNRNW8csrLz7uGvFDgdMyzfyQWO+5+1ey3a0HA2vjAOrKxubrJTwcRNr7IvA8aiGpjC++UY6bY4MH7JUnUVS6CX13W4uk0NAfnKsZOQdrdAj2YWvouZhOTExEyYCFoGl1B8cCeoA8fh5PYsM1Lf33K63Ya+saCTuRcmi7wN7Pi+AVf9ts8bsUg5Dp5Lu6RuyRVWtBLQprJdxzFSV0bokTUUA2Q3F31FRedqaW8ugPUPLHLx0DHU0cMk0IPuzEzEUDrLw8h\\\\\\\/C1Wt058E334CF6GJgTXembeibVgOqpAAd7k9D2Ar9QD1p\\\\\\\/deUFeS\\\\\\\/MI+XWCccUC9\\\\\\\/xdv\\\\\\\/wIGs7NIIQ5wRR1UVo5\\\\\\\/e6JV6sCAM\\\\\\\/LeqBMKYIw29wui76yz+2IhgNHnsQRoJxSKJRapSr20kJcPypi2F4TcLdd6eIuWI913s1vTK5cBO\\\\\\\/0NuKl2yN3YUmcaatuKwRVyMvxImFVTD+tHzXXdp9ZMC8JmbW\\\\\\\/4HDgYjnmyHto1TMumEkb\\\\\\\/Jt8c\\\\\\\/Dk9y0v5A3US6Vheb7my4uoUI0q1PBwbNOy9mhaeGOmVLDiMHXu3nMNtxTL7yhQ9jca1vb9hNvpgX8Rc0CIg+6Lnv1\\\\\\\/cAVQEpmq1NTbU0X2+DCA8Uqcm1O+rufVM5+BxfRcD0CobC3o+Qc885MGGIHB0BV2UlkmFr0TsabX5XFlMfsi0rsZu+UB+wD4e\\\\\\\/0xVUQANwLXMbGjLZgGbUOfJpFyznWKzsetjthuUKg2SFb3zMQZhk0xRbxKY7X4gaWTSdDT6ci9TQZ7g0yNwxoz93Mej\\\\\\\/3+WFZ+dwAccr71LG71YANgHbpR7JFFCDsLBiDFW5mUzqk\\\\\\\/c6MB1pMuoe2WO2X1v7yig+zRo6gCHaFw\\\\\\\/Wm+WGj2S72oqV+t0VuFYrvDDeYYR7Y0qiXxKGj5E66IJHKx2UugBTy\\\\\\\/2uTM3u7pf2zno3EJ7Am7V7K0t3FPS9qTaJ7ABCLwJngEB3wm3uJMH4XytB0xVnMlMWjQU\\\\\\\/Q\\\\\\\/2o11NI0Yp3yXgv+fmcgrrmh77ATSOHYGkDLX+cdSh5VcLjCHIGtyC0xJFaLemov3vtTPzn7YLILotDPtd47rc1AorFvbO9Xg+kv0T0ZkxdT\\\\\\\/l69m5Zt6VOZsOOfNXpJkfGXYxPFDXJSYY6ugXw4BAzT0gDfQ5GSDqTOzI2Bl1u4llFnHvwmrPskTAxnCLV4nz\\\\\\\/nc1Evx55bKXdJi3aB8jktnwvQLje6g8heYTvqJnIsoNc+ZAz6l\\\\\\\/7xqeuj7LbicyqTUPJtOK5bOnEosBvrtBq8kyICD6QMlskXAh03Vk48XZ3ASbUYc150BTj4zC1IZysqzFt39y0zr5+DoFD2ITbumb\\\\\\\/5ddz9I9wqaPddSjZ61PXU5BKV9YeLqMssmRs2cYU0Zk3Kszsh1lmBFCyVJdA\\\\\\\/lkL7hcOinmNcDrsZaakgsLtKNQhnCO9RqJWr1z15SmbsrPOBo+4JTOMU3\\\\\\\/ri7WOIfyiv58YeYOCdDr6gDpQcmOxKlxaucNEeSr\\\\\\\/wpv6fqmDbk4L\\\\\\\/Ju7oMV1ifqGciuFHkbCvBr9pr2+r2zd0YFcU3KkZp6m3+Jd4esRI\\\\\\\/w1E87tuRVxZDTi0PzRXzo5L5dmMrsnknmKCyrlJm8fyIleN4\\\\\\\/BnhRIdkr30zIdwht4u4zUPEpVXhX2uLreyHTl6egWR0viHACgJsmzt2IazavKNuYwNUTHOJiRoQ2LeQQBgvjw1tiHYPLKlDAzpcEWor3oD5BasRh+y4TlU9JV+s8zunN\\\\\\\/delBbQ8yQgprIqwv8wGDUiClDzkO0Ph\\\\\\\/GQ29nVjSZOj\\\\\\\/OKvkr6CuSJx\\\\\\\/LblZi2DX3Zv4M+7N\\\\\\\/tC9xTdNmEQAEA+Mn5heriDTVU4aTlYYdGbVyqVgNBs1OjiLWeWXXJpK9KLPPpiXMcnPH7Od6LKmLcUgD41BK\\\\\\\/j7yFVwI8v\\\\\\\/oKBl5yWjbOqHvaLCwbzFjLg4U\\\\\\\/PX55VOCllB8b9j\\\\\\\/xjlBJLhQKdnNAcKd2a0AU1qcFkU5PrlMpkQaVgI6hMiw9YoZFxSERAKHfQttiQyfEYVMldu4T7JwYVCmHq89iT7prDf4AchwdbVnPDKmGLmNIuZVKAhEXH5F5\\\\\\\/54oGr8mADzxoUG\\\\\\\/GEo\\\\\\\/iK8g6acZ9Pqbmi6a9JBgDyPs4h8+6pxT217Np8Fc4lfLH65uvP5fOpNPb+GLkIRV3RkcYQjAvk1jz6O4LBU89ujBh8aXqFscT6HJpdK00Gfh\\\\\\\/L17MiM+hbpTAqD4TeVgkw3ezsSn+lkbB4eDFBO61yi29bKMQ\\\\\\\/xSIKKhOpnRI0RWLacE+mRxpVSqPy3IDSgTF8VLwPnpSQhbKIHiNXxPrMyWrB9tSeU4yXFBsfy9eC0I1De8iLjFIULdur5f9h9mJjq9p7kRgBsW3SXGaSTuUnXNRo0dxXb3LHKOrx+mG17z0aQZSpBAPepjdulxX\\\\\\\/5oh1GZlw9\\\\\\\/QPufZrka0PLukEHvv1BmfYdkW1Z6tetZOwCndG7u2Cl\\\\\\\/7g1mr9yo+UgykEB6eCYCgWM9WUeukjNnocb+8cWAgqRYgJ4JhXl\\\\\\\/YRXmSuSOzVavPA5OwKf3Pfbymg7b26orqL9mniV7xklhul9RIam7cnx\\\\\\\/1flCkjBYbjST5M2LGiNVr\\\\\\\/qH5m46e8Qyx2FBUKIYJoGMPlS91J2dWLbHoI8rp2c98P30nE061vrc5ks6DnifDFjJRFId1095pDYkansZWXx3dXlxSpSlkzt+uUbysYapSepi4yhUhSGtvA16DrSNwjncN2WOtIVunEoK3\\\\\\\/8R5L+JuEEVXAaQHX7t0aUI+90cQt9OlWf0me4rRW0YHeOfFZDv8t\\\\\\\/qZniWQudff3oNeoku1hz5hMJag5ctb2WbnMPnrldMgoMxhQ2evzYCM+8HUupZ9ntZcMmgjHzt7196gEGycd5g17tZZdi29GBVEBuY4U9zMj8CiyHof+6VeF5bTHeCApkCRMgn+Z9PlJcrzd9cj2EolCCL4bEDLsLtUyUFradoxLymJ7Q+9lOE3xJRe5l1YypKsBn1\\\\\\\/zjN0B4LLIXGVs85b4JxOBzbbfjOgejMrxjFHzDSKdInP1tT4CErL1GULsbg+aMLlnzGuU5ED0Tcs0FaPPHVpbvplyPNn38HzCusQP2UTOpc3U2\\\\\\\/XQ8XYGi2Az1XvxkH9vM7esL\\\\\\\/WO4r+fFsXFCXHWHUkfgDd8GC6S7ImundWmgdGO6oiIQgnPXLbw8PHvoh+iW9tYhACMhvH\\\\\\\/1ctFurk4M\\\\\\\/S8knXPuQQyDNppWqR0Co2NBzou6FytbRgViohTBOYigUhKHl3ozUz3mDXTxxbaw51P\\\\\\\/vgunGZdLjFfvbAP9AGbPnGHq1ro6zQOPCZIfe3bB4p9vFdwFiJZz9HOUz9qhLe53XsZuzsrQGPEcrnhh8LwRKNEbsA8Nf5gus5tPu0OtymchRElKX5ECFYmH90btOPrCK6p+nQrgqireoFczFJnNKNw9szsjxqlWwKDNKOYHqKUjf0cnT615zL36YT1pk5ntO5Dw94vogMvqXPZwaSMGUQbS\\\\\\\/Y1DLY+cKXQTeLQxMWZwEPy8HZ\\\\\\\/wGcTLWXwCI+0oCf4ICIsWAt7+cDgu2stOCjER59gbfUEXzwAvhVpbiQdjGTnHQuHiWjekV91DbArX1jBYGpij7v2r83XzD4UnHLBeo5awXifcpLowtsAMs5RQLkeeZUeJ6qoejcWIm1ZcTRloXY51ZN9u0o6gh4h6z4vFRY+oD7WY3GXi6hVVQBSB8IrtiL8Vduox4uLxdk5XvpwmXbEVTGGwYgiFKq29\\\\\\\/D6JL2yXYUvf\\\\\\\/UDArDP\\\\\\\/T1\\\\\\\/7BGQIieSCBX\\\\\\\/hQA4IbrdhAEAOBDHLH6XKCnZXqnSoRAXxCgwf3p3xTZAdiQ4Rvz7e1FYMbnqn8h7bEJhn+MRPeDDldSJRreXIO9GsI1TzOUNf208zh1mQ3qXj1YnVfQymPGC36Du3\\\\\\\/IO3LwT2Gby4BR8cmvjhOo+hfmIABep7W+RgfUU+BLhB\\\\\\\/wNv8wXLxw8eSWCXXIGRfd8bWMdhWwU279S\\\\\\\/XWJdHTCUeqj2Rh\\\\\\\/frYNSLTAxwssQaovUNY8aPyeAoOpyWUnO2Vkpj+Kjap5\\\\\\\/Gn+5HdzfOOyh1Jq3IukP4rKp0\\\\\\\/3bdOVigcb8GquoRaRXinxyloMcNWb5xdeiVm4rfu\\\\\\\/1zVdi1Uu1wU+KgIrbq5mnlqrpJE6MSggpis4\\\\\\\/6Z4pNYMEJl2WQRbwO6Fes4K4d+Ggotm9IVTYc\\\\\\\/AqpnTaoJ07mQgU2B9blDeGUFSD6Luvul\\\\\\\/9TBqvZsp2jUdsIhUHVg49mSWCU9ouRM+TMPzOBMZ5Sf4Klf36DzqVXMhMuLiK1o+ckExtY7X\\\\\\\/aDQBh2EnlzO4T6PJplaj9Ae6DBk0KLDBm34PesK1IyCaC0E3vbQ7DQvmGV9soZBvUW4W7eder6rqMg3YArkah\\\\\\\/BKUveSvzLihPT42HV6+A1BCrI+iftbMEJRNQV2uHQWlsAyBdvj1n35Sxt6t3CqIArEkBQEl63QuZx3IfZkXu1mnRF2KYhvisMF0ezPFlZ4B9tM\\\\\\\/g2uI+IGH\\\\\\\/FQvxgruAk14JgODhChDNLLUglIXeH4rq2yMgbo0Asz7OHYHenAjvjzmDlyqCxDpk8aWQH\\\\\\\/ff+poRiumF+jUlu1Uw0u17jNj8ha8e5Zm48b\\\\\\\/D2lE6iWAzVRjG1xIUMhLMWFiQQ5GYgjRAX1lqc2TY3x+ewEp60d0Cg1lfzIWjOvwnIQj79Win9bWBo320MkaUFqZgKzOLcaF\\\\\\\/u67V3TpCkit2u824rWz\\\\\\\/4g5QhoYzyr4bwfuTGSRhWo8C1aKD5CQB\\\\\\\/ExDdgt9\\\\\\\/fJ4a98\\\\\\\/mkU\\\\\\\/qHIxWeo4tSW0ESH7ThkwL0GOPIrpU2SITUYJUbnKyIvugGnVthlgdP8RUjXyUc06tyYceFeziTJQD5Zxj17jS4hUkW8C9y7VePgTtFafjVQE4suNWyL1Fy+w+3\\\\\\\/4BpZAtxT83m+3nI4W5aXNdbaoIUcrGOzr8fkH+WDBClB\\\\\\\/k7jDcfxT9f4u\\\\\\\/FOM6iiZPT7XUKaGdjC7J7uRK6THzgi7XfRVHaSmdhha3p1a7ATuTKOQ1YvPWTUaPpxF1T686CT9pX51+pcNJI5kik2yBJA4fv227jrJbVBtt1xI7FEYqbyongRWt\\\\\\\/FvCtd\\\u0022,\\\u0022iv\\\u0022:\\\u0022l1IA359r1WlxTB89iINuMQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00226be6dabe0496cded2f19def689743b997b5c42cc778c2c6351f01a9fd3c2d7dc\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022\\\\\\\/b9McR8aX\\\\\\\/7WS9e8EmEcffF8VUTAyeUvIzi4SPwZlqRZ3yOYo0GAKQh5ClJsrjj7omLtPH\\\\\\\/6MtEX27c1DRzMRGUEpuGbltN24e+WQRBL8xPDYD4YLaO5l3H9IRMYkaDM9cnJGY7YtOotRSEKMP2CD\\\\\\\/zHJ4lpvLYfqoHFsDEaaCRVO3+XjB0Sf2naMmvOq8fCrjQM2k6VAO+7xrg8twgK4yJZiEjkM11JvSJFcOss4EhpTbB4ir0XYFbvkmyyIXUMZtK7tEuP2041sKnlJlu5GPjHvZHbcZKvjPf+ziuHAzxykuMHOArxSpIfMWfE7k5UgqG02ucKvl9\\\\\\\/EC+6qbzHMa2W+s1QA+qFYt9X7QPF8HkRe0wja7LHOZjOqW2WwkARJ1jhmyQX2BK6bEN+878jVUoJ0JwOJnOiYDX+FY5jTn\\\\\\\/7lf6d4tG3mIDwJ4AK+Px\\\\\\\/hWdThoBF+wVuA3xCHaWHlJXpjx94\\\\\\\/JaJjvg6SMeseBWsZQhfrpAtFf+nxCCROC2gj8PenHrBxYf0V\\\\\\\/kuy7kraqFWbSzlB5IFSl4M8ZwIEiBVr4Lf4if73uMCz+D2FaYoS+54caoWAHNd1jRdAEDo9SImFBPofrfZqGPuMV2oXOwwIcPfsGw+PQ2p\\\\\\\/ZbQDHTX0eByNfSk9hftxONvjgo8d4eROA59d\\\\\\\/XKYRfIt40soV5Y8dSRbv\\\\\\\/HJ2SzO0hjP81b1V860WzmtubRj8eSBkA3Dh9SwUu+SRSXuSnTdvuoXwPVhlhWltYYGLkdS1wwZnrGoXGvtGWyW1mNJi9Zflgbx0wjNJgoCYsTRrjx39Sg9dgQazyChX5o\\\\\\\/0yRVi88I7NIG\\\\\\\/lVgcuJvpDs8xp0xEhNi7ktGoGbUEvzYDDtIkGZTxgOUkVy1Mc4YMVYn3+tvD\\\\\\\/7Yrq2z\\\\\\\/6HLpEUxMoCh6w\\\\\\\/e2kMJ\\\\\\\/KS+IRziEo1us3fCydU\\\\\\\/6orXaNMi4Dzt1hclvgvzUh5TZmnxUsAvDeOXttNKYpfubgifcmAvFqshk\\\\\\\/dFzG8nWebwgaDeE09iNSFV8XNZmqqSFbeKlb3X1o8Lu6LoPdPsepg9mFdwrHzUSylmiE04YVT67r2OPiQ83v8Dxa7ATE5yOZilDEtuU\\\\\\\/Lf76Kyq4hH02FZLfiTZDwpiHp4e1Cbw1JD5fHr2y6BGtBCDZnezfxieFRKpGb+DQdlXIK5zTkbwuZGAfNHDRpOhxoelcMiOy9hXrqDqXl\\\\\\\/Pjfsfn4qp\\\\\\\/wsVOCCCrHGGco5X+Dkqu1PFuV\\\\\\\/PPhkctL762zhO0FmAaWtL01kvQYzcP2vjDgxX37gfoj+3cR6Gw4Ljd0dvLS6bVjBHoKbJeDRPXh23+oiwNOVdJGvG6iYL7TmNkrMtUtHQrBkRRVkch1Y\\\\\\\/lTObt7tai\\\\\\\/+jBGX83D+4OxKXOCXiSsd0HNWzbLLQppKfugrcfQeSUm51+5+WstTf4cW5\\\\\\\/GSCaZqCEvGyZA9KfouG5pJ3FOm0L8RF\\\\\\\/Sk2wYugFBo71Xg1sdU9yaWD6WY\\\\\\\/JXH3T5yDqZMBTEEq+vkvlNlGhaWixATNb3cqMvmJjtqrGUGVXi6sLxTrgogZKLEYyheXxIVAeyJdKFMLVy0aUlBcrp3uVem53r0Mrci2I3XDdoLk7FLo\\\\\\\/rREAioaihabE+z6Fern7E6lQv508RTD2\\\\\\\/xXO\\\\\\\/VCGEPCGA6PmGmJ+7IC3RVqDULx6lzEs\\\\\\\/WdB27tKUZXODJq6bgIeV0Iau5emB9GPOBoNfpojaWwOBPVJKZap94QMWch2VDnVelWktZbTqkjUDfLSMYqaFKHmc9jV9zqyxuKa88DV8zak7HF\\\\\\\/bD0bfIwGbiqDSBOmOXfamIFd2ehxkOue2bH1l0aCrXfcwxemSqxUym6S6kS3l9P94vzyTcIHcjVh5L4kFExydFUEYQpuS3hG2OJTiwxK21zfBjUqjfLBY1plORqcadgpaD15toahBgwy49W9Xcu7KkCG1eUDbI+B060yWgnWJVfF\\\\\\\/agGyTaLGOlt+v\\\\\\\/jwZ6bGfqN9jlFgZdeNhzlOwwXectbvyBt1Rm3xJPbZRAsZjlwrbO4=\\\u0022,\\\u0022iv\\\u0022:\\\u0022nzrhPQ3SRnUmUAycuK\\\\\\\/2lQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022bec98b8aa66156e47c346c2295b741bb6dd3741178dee1a26a11a7f28c15d6c5\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-491452 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/11/3617" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-T3-iP-JpnEha0d4Ikpc4Equ2pjFze7sNpyMNNCq9jdI" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/11/3617" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-dfHsULmhURTuUAicMVM8eTSmC5soZu8DtMKM-K2fTyo" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/11/3607" title="An Efficient Genome Editing Strategy To Generate Putative Null Mutants in Caenorhabditis elegans Using CRISPR/Cas9" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/11/3627" title="Whole-Genome Multi-omic Study of Survival in Patients with Glioblastoma Multiforme" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="491452" id="node491452" data-pisa="ggg;8/11/3617" data-pisa-master="ggg;g3.118.200665" data-apath="/ggg/8/11/3617.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic Analysis To Identify Signatures of Artificial Selection and Loci Associated with Important Economic Traits in Duroc Pigs</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Yunlong Ma</span>, <span class="highwire-citation-author" data-delta="1">Saixian Zhang</span>, <span class="highwire-citation-author" data-delta="2">Kaili Zhang</span>, <span class="highwire-citation-author" data-delta="3">Chengchi Fang</span>, <span class="highwire-citation-author" data-delta="4">Shengsong Xie</span>, <span class="highwire-citation-author" data-delta="5">Xiaoyong Du</span>, <span class="highwire-citation-author" data-delta="6">Xinyun Li</span>, <span class="highwire-citation-author" data-delta="7">Debin Ni</span> and <span class="highwire-citation-author" data-delta="8">Shuhong Zhao</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3617-3625;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200665 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node491452" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Yunlong Ma </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Key Laboratory of Agricultural Animal Genetics, Breeding, and Reproduction of the Ministry of Education & Key Laboratory of Swine Genetics and Breeding of the Ministry of Agriculture, Huazhong Agricultural University, Wuhan 430070, P. R. China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yunlong%2BMa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ma%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYunlong%2BMa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Saixian Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Key Laboratory of Agricultural Animal Genetics, Breeding, and Reproduction of the Ministry of Education & Key Laboratory of Swine Genetics and Breeding of the Ministry of Agriculture, Huazhong Agricultural University, Wuhan 430070, P. R. China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Saixian%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASaixian%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Kaili Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Key Laboratory of Agricultural Animal Genetics, Breeding, and Reproduction of the Ministry of Education & Key Laboratory of Swine Genetics and Breeding of the Ministry of Agriculture, Huazhong Agricultural University, Wuhan 430070, P. R. China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kaili%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKaili%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Chengchi Fang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Key Laboratory of Agricultural Animal Genetics, Breeding, and Reproduction of the Ministry of Education & Key Laboratory of Swine Genetics and Breeding of the Ministry of Agriculture, Huazhong Agricultural University, Wuhan 430070, P. R. China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chengchi%2BFang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fang%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChengchi%2BFang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Shengsong Xie </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Key Laboratory of Agricultural Animal Genetics, Breeding, and Reproduction of the Ministry of Education & Key Laboratory of Swine Genetics and Breeding of the Ministry of Agriculture, Huazhong Agricultural University, Wuhan 430070, P. R. China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shengsong%2BXie%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xie%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AShengsong%2BXie%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Xiaoyong Du </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Key Laboratory of Agricultural Animal Genetics, Breeding, and Reproduction of the Ministry of Education & Key Laboratory of Swine Genetics and Breeding of the Ministry of Agriculture, Huazhong Agricultural University, Wuhan 430070, P. R. China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xiaoyong%2BDu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Du%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXiaoyong%2BDu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Xinyun Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Key Laboratory of Agricultural Animal Genetics, Breeding, and Reproduction of the Ministry of Education & Key Laboratory of Swine Genetics and Breeding of the Ministry of Agriculture, Huazhong Agricultural University, Wuhan 430070, P. R. China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xinyun%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXinyun%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Debin Ni </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Key Laboratory of Agricultural Animal Genetics, Breeding, and Reproduction of the Ministry of Education & Key Laboratory of Swine Genetics and Breeding of the Ministry of Agriculture, Huazhong Agricultural University, Wuhan 430070, P. R. China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Debin%2BNi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ni%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADebin%2BNi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:shzhao@mail.hzau.edu.cn" class="" data-icon-position="" data-hide-link-title="0">shzhao@mail.hzau.edu.cn</a>
<a href="mailto:nidebin@mail.hzau.edu.cn" class="" data-icon-position="" data-hide-link-title="0">nidebin@mail.hzau.edu.cn</a></span></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Shuhong Zhao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Key Laboratory of Agricultural Animal Genetics, Breeding, and Reproduction of the Ministry of Education & Key Laboratory of Swine Genetics and Breeding of the Ministry of Agriculture, Huazhong Agricultural University, Wuhan 430070, P. R. China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shuhong%2BZhao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhao%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AShuhong%2BZhao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:shzhao@mail.hzau.edu.cn" class="" data-icon-position="" data-hide-link-title="0">shzhao@mail.hzau.edu.cn</a>
<a href="mailto:nidebin@mail.hzau.edu.cn" class="" data-icon-position="" data-hide-link-title="0">nidebin@mail.hzau.edu.cn</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/8/11/3617" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:491452" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:491452/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/11/3617.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:491452" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:491452/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/11/3617.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:491452" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:491452/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/11/3617.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:491452" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:491452/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/8/11/3617.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:491452" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:491452/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Identifying genetic basis of domestication and improvement in livestock contributes to our understanding of the role of artificial selection in shaping the genome. Here we used whole-genome sequencing and the genotyping by sequencing approach to detect artificial selection signatures and identify the associated SNPs of two economic traits in Duroc pigs. A total of 38 candidate selection regions were detected by combining the fixation index and the Composite Likelihood Ratio methods. Further genome-wide association study revealed seven associated SNPs that were related with intramuscular fat content and feed conversion ratio traits, respectively. Enrichment analysis suggested that the artificial selection regions harbored genes, such as <em>MSTN</em>, <em>SOD2</em>, <em>MC5R</em> and <em>CD83</em>, which are responsible for economic traits including lean muscle mass, fertility and immunization. Overall, this study found a series of candidate genes putatively associated with the breeding improvement of Duroc pigs and the polygenic basis of adaptive evolution, which can provide important references and fundamental information for future breeding programs.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/duroc-pig" class="hw-term hw-article-keyword hw-article-keyword-duroc-pig" rel="nofollow">Duroc pig</a></li><li class="kwd"><a href="/keyword/artificial-selection-signatures" class="hw-term hw-article-keyword hw-article-keyword-artificial-selection-signatures" rel="nofollow">Artificial selection signatures</a></li><li class="kwd"><a href="/keyword/genome-wide-association-study" class="hw-term hw-article-keyword hw-article-keyword-genome-wide-association-study" rel="nofollow">Genome-wide association study</a></li><li class="kwd"><a href="/keyword/economic-traits" class="hw-term hw-article-keyword hw-article-keyword-economic-traits" rel="nofollow">Economic traits</a></li></ul><div class="section intro" id="sec-1"><h2 class="">Introduction</h2><p id="p-4">Duroc, an older breed of domestic pig, was developed in America and formed the basis for many mixed-breed commercial boars after a long period of artificial selection. It is predominantly used as the terminal sires in pig industry, and is well known for its superior performance in growth, feed conversion efficiency, carcass and meat quality traits (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Briggs and Briggs 1969</a>). The artificial selection affecting these economic traits has left detectable selection signatures within the genome of modern Duroc pigs (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Edea <em>et al.</em> 2017</a>). Although the identification of selection signatures has been studied for decades on the basis of microsatellite and SNP arrays, there is now unprecedented opportunity for progress on fine mapping with the advent of large genome sequencing data sets on pig variation (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Abbott 2012</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Ai <em>et al.</em> 2015</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Sabeti <em>et al.</em> 2002</a>).</p><p id="p-5">In general, most of the economic traits in commercial animals are quantitative and controlled by many genes with small effects (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Khatkar <em>et al.</em> 2004</a>). This quantitative genetics view is supported by most of the recent gene mapping researches, including the identification of selection signatures and genome-wide association studies (GWAS) (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Eusebi <em>et al.</em> 2017</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Ma <em>et al.</em> 2018</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Pritchard <em>et al.</em> 2010</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Ros-Freixedes <em>et al.</em> 2014</a>). Therefore, genomic selection has become the main method in genetic improvement in important economic traits (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Hayes <em>et al.</em> 2009</a>). However, for traits such as meat quality and feed efficiency that are difficult and expensive to measurement, genomic selection is still limited by the sample size of the reference population. So, based on gene mapping, genetic improvement through marker-assisted selection is still an important alternative to improve these traits. Simultaneously, the development of genome selection methods, combined with the information of GWASs, will help further improve the accuracy of genetic evaluation (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Zhang <em>et al.</em> 2014</a>).</p><p id="p-6">To address the growing demand for high quality pork and minimize pig breeding costs, the understanding of genomic architecture underlying pork quality and feed efficiency is valued by animal scientists and breeders. Intramuscular fat (IMF) content is one of the most important factors affecting pork quality. Based on marker-assisted selection, there is increasing interest in mapping some of the causal genes of IMF for its genetic improvement (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Ros-Freixedes <em>et al.</em> 2014</a>). Feed conversion ratio (FCR) is an important indicator for feed efficiency and is usually included in the selection index for genetic improvement in pig breeding (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Do <em>et al.</em> 2014</a>). Although a series of QTL associated with interesting economic traits were reported based on traditional QTL mapping, it is still scarce to investigate the genetic basis of IMF and FCR using genome-wide association study in pigs (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Do <em>et al.</em> 2014</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Hu <em>et al.</em> 2016</a>; <a id="xref-ref-37-3" class="xref-bibr" href="#ref-37">Ros-Freixedes <em>et al.</em> 2014</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Sato <em>et al.</em> 2016</a>).</p><p id="p-7">In this study, we used paired-end Illumina sequencing to resequence the genomes of four purebred Durocs and used the genotyping by sequencing approach to characterize the genomes of 282 individuals from the same population. To identify the signatures of artificial selection, sequences from 23 purebred Durocs, 21 Asian wild boars (AWB), and 25 European wild boars (EWB) were downloaded from the most advanced publically available database. Two different statisticses, the Composite of Likelihood Ratio (CLR) and the fixation index (F<sub>ST</sub>), were applied to detect selection signatures. In addition, we also carried out GWAS to identify the key genes related with IMF and FCR that were changed during the breeding improvement of Durocs. We found strong signatures of selection left in Durocs during breed formation exemplified by several striking selective sweeps overlapping with some major QTLs. The results would provide useful information for those who are interested in further understanding the genetic basis of important commercial traits, and facilitate future breeding of Durocs to improve these traits through genomic selection.</p></div><div class="section materials-methods" id="sec-2"><h2 class="">Material and Methods</h2><div id="sec-3" class="subsection"><h3>Ethics statement</h3><p id="p-8">All research involving animals was conducted under protocols (No. 5 proclaim of the Standing Committee of Hubei People’s Congress) approved by the Standing Committee of Hubei People’s Congress and the ethics committee of Huazhong Agricultural University in P. R. China. In addition, all experiments were performed in accordance with approved relevant guidelines and regulations.</p></div><div id="sec-4" class="subsection"><h3>Animals, genome sequencing, quality checking and filtering</h3><p id="p-9">In this study, genomic DNA was extracted from ear tissues of each of 286 purebred Duroc pigs using a standard phenol-chloroform method. Among them, four individuals were selected to sequence the whole genomes. Paired-end sequencing libraries with an insert size around 350-bp were constructed for each sample. 2×150-bp paired-end sequencing was performed on an Illumina HiSeq X-ten platform at the BGI-Huada Genomics Institute in Shenzhen. The rest of individuals were genotyped by sequencing (GBS) on an Illumina HiSeq 2000 platform in Buckler Lab for Maize Genetics and Diversity at Cornell University (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Elshire <em>et al.</em> 2011</a>). In addition, we also downloaded the whole genome sequence data of 69 individuals from the EMBL-EBI database (<a href="https://www.ebi.ac.uk/">https://www.ebi.ac.uk/</a>), including 23 Duroc pigs, 21 Asian wild boars and 25 European wild boars (Supplementary Table S1). Quality control of sequence data applied the following criteria: reads with (i) &gt; 10 bp aligned to the adapter with up to 10% mismatches, (ii) up to 10% unidentified nucleotides (N), (iii) &gt; 50% bases having a phred quality less than 5 were removed, and (iv) Duplicate reads generated by PCR amplification in the library construction process were also removed.</p></div><div id="sec-5" class="subsection"><h3>Read alignment and SNP calling</h3><p id="p-10">Clean reads passing quality control filter were aligned to the Sus scrofa reference genome (Sscrofa11.1) using the Burrows-Wheeler Aligner (BWA) software (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Li and Durbin 2009</a>). The reference genome sequence was indexed and the command ‘mem –t 8’ was used to find the suffix array (SA) coordinates of good hits for each individual read. SAMtools was then used to convert the SA coordinates into the best alignments in BAM format (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Li <em>et al.</em> 2009</a>). After alignment, ‘CreateSequenceDictionary’, ‘SortSam’ and ‘MarkDuplicates’ of Picard were separately used to indexing, sorting and removing potential PCR duplications (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Li <em>et al.</em> 2009</a>). The BAM files were indexed by Samtools. The ‘HaplotypeCaller’, ‘SelectVariants’ and ‘VariantFiltration’ of GATK with default parameters and SAMtools ‘mpileup’ module with the parameters as ‘-q 1 -ugf’ were used to call SNPs. Finally, the GATK ‘VariantFiltration’ module was used to exclude SNP calling errors in according to the following criteria (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Mckenna <em>et al.</em> 2010</a>). For GBS individuals, joint genotyping was performed by the GATK ‘GenotypeGVCFs’ module.</p><p id="p-11">High quality SNPs with (i) coverage depth ≥ 4, (ii) RMS mapping quality ≥ 20, (iii) the distance of adjacent SNPs ≥ 5 bp, (iv) the missing ratio of samples within each population &lt; 50%, and (v) the minor allele frequency (MAF) ≥ 0.01 were kept for further analysis. The SNPs that were used for GWAS need further apply quality control of Hardy–Weinberg equilibrium (<em>P</em> &lt; 10e–6). To decrease the influence from genotype imputation, only SNPs with the missing ratio of samples within each population &lt; 20% were used to detect selection signatures. Beagle software was used to impute the missing genotypes and infer haplotypes with default settings (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Browning and Browning 2016</a>).</p></div><div id="sec-6" class="subsection"><h3>SNP annotation</h3><p id="p-12">We functionally annotated Single nucleotide variants (SNVs) with the gene-based annotation modules of ANNOVAR (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Wang <em>et al.</em> 2010</a>). Using ENSEMBL genes, we investigated where the SNVs are located in the regions of gene components.</p></div><div id="sec-7" class="subsection"><h3>Linkage disequilibrium</h3><p id="p-13">We calculated the correlation coefficient (<em>r</em><sup>2</sup>) for every pair of SNPs to measure the LD level in Duroc pigs and two wild boar populations using PLINK (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Purcell <em>et al.</em> 2007</a>). To visualize the LD decay in this analysis, the <em>r</em><sup>2</sup> values for 1000-bp distance bins were averaged and the corresponding figure was drawn by R script.</p></div><div id="sec-8" class="subsection"><h3>Identification of artificial selection signatures</h3><p id="p-14">Identification of selection signatures is an important field of population genetics. To detect the selection signals accurately and objectively, a larger sample size is able to better reflect genomic patterns shaped by selection in the particular population. Therefore, all 27 whole-genome sequencing individuals were taken together to detect positive selection signatures referring to the result of principal components analysis (Supplementary Fig. S1). To identify artificial selection signatures, AWB and EWB were defined as the reference population, respectively.</p><p id="p-15">In this analysis, two methods were employed to search for the evidence of artificial selection in two steps. The first step is positive selection detection, which was performed using the composite likelihood ratio (CLR) (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Nielsen <em>et al.</em> 2005</a>). The CLR method calculates the likelihood ratio of selection signals by comparing the spatial distribution of allele frequencies in an observed window to the frequency spectrum of the whole genome. SweepFinder was employed to calculate the CLR with a grid size of 25 kb resulting in a total of 90,588 CLR scores across the genome. In the second step, we identified the underlying artificial selection signals from the above detected positive selection signals by calculating the F<sub>ST</sub> statistic for pairwise sites between the observed populations and the reference population (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Weir and Cockerham 1984</a>). The unbiased F<sub>ST</sub> estimate proposed by Weir and Cockerham was used to measure the population differentiation, with values ranging from 0 (no differentiation) to 1 (complete differentiation). To produce comparable CLR and F<sub>ST</sub> test results, single site scores for F<sub>ST</sub> were averaged in non-overlapping windows of 25 kb resulting in a total of 90,322 and 90,411 F<sub>ST</sub> scores across the genome when AWB and EWB were defined as the reference population, respectively. The empirical P-values were generated by genome wide ranking of F<sub>ST</sub> and CLR values (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Qanbari <em>et al.</em> 2014</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Simianer <em>et al.</em> 2014</a>). Finally, the windows that the scores of the statistics fell into the 98<sup>th</sup> percentile were considered significant in CLR and F<sub>ST</sub> methods, respectively. Note that the artificial selection signatures in this study were defined in the genomic region in which both CLR statistical value and the F<sub>ST</sub> statistical value were greater than the cut-off value at the genome level.</p></div><div id="sec-9" class="subsection"><h3>Genome-wide Association studies</h3><p id="p-16">In this analysis, all 282 castrated Duroc boars were treated similarly. All individuals were in good health and had the same body weight at the beginning of the experiment. The feed conversion ratio was calculated during the growth period from 30Kg to 100Kg, and at the end of the measurement, the intramuscular fat of each pig was determined by B ultrasound (Supplementary Table S2).</p><p id="p-17">After quality control assessment, autosomes of 282 individuals genotyped by sequencing were used in two association studies for IMF and FCR traits, respectively. GEMMA was used to fit the model <strong>y<em>=u+Ss+Xb+Wα+e</em></strong>, where <strong>y</strong> is the phenotypes vector, <strong><em>u</em></strong> is the intercept, <strong><em>S</em></strong> is a design matrix of the fixed effects, <strong><em>s</em></strong> is the batches effect, <strong><em>b</em></strong> is the SNP effect, <strong><em>X</em></strong> is a design matrix for allele dosages for the imputed SNPs, <strong><em>W</em></strong> is an incidence matrix linking α to y, <strong><em>α</em></strong> is the additive genetic effect ∼ N (0, G<em>б</em><sup>2</sup><em><sub>a</sub></em>), where <em>б</em><sup>2</sup><em><sub>a</sub></em> is the additive genetic variance and G is the realized genomic relationship matrix that was estimated using genotype information, and <strong>e</strong> is the random residual term. To avoid double fitting of SNP effects efficiently, the test SNP and the other SNPs in the same chromosome were removed in constructing the G matrix each time based on the standardized relatedness matrix in GEMMA (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Zhou and Stephens 2012</a>).</p><p id="p-18">Since Bonferroni correction is overly conservative especially when genetic data has high linkage disequilibrium, it may cause false negative results (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Duggal <em>et al.</em> 2008</a>). Therefore, a less conservative significance threshold of 1.03 × 10<sup>−5</sup> (0.05/4,853) based on the SimpleM method was used to account for multiple tests in this analysis. A total of 4,853 independent tests were identified here that was in turn inferred by the number of principal components accounting for a 99% of the variance of the SNP matrix (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Gao <em>et al.</em> 2008</a>).</p></div><div id="sec-10" class="subsection"><h3>Enrichment analysis for artificial selection signatures</h3><p id="p-19">Enrichment analysis was carried out for exploring the potential biological functions of genes located in putative artificial selection regions. It is involved all the selected genes in the 200 kb window around the significant signatures, which was determined by the linkage disequilibrium decay (Supplementary Fig. S2). Genes located in putatively selected regions were identified using the BioMart program (<a href="http://www.biomart.org/">http://www.biomart.org/</a>, Kasprzyk 2011), and then an enrichment analysis, which included the terms cellular component (CC), molecular function (MF), biological process (BP), and pathway analysis was performed for the identified genes using DAVID 6.7 (<a href="http://david.abcc.ncifcrf.gov/">http://david.abcc.ncifcrf.gov/</a>) (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Huang <em>et al.</em> 2009</a>).</p></div><div id="sec-11" class="subsection"><h3>Data availability</h3><p id="p-20">The Illumina sequence reads are available in the NCBI Sequence Read Archive under the accession SRP158574. Supplemental files, the GBS data and phenotype data can be downloaded from Figshare (<a href="https://figshare.com/s/e45cc6d717cd498d5013">https://figshare.com/s/e45cc6d717cd498d5013</a>). Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.7077125">https://doi.org/10.25387/g3.7077125</a>.</p></div></div><div class="section results-discussion" id="sec-12"><h2 class="">Results and Discussion</h2><div id="sec-13" class="subsection"><h3>Genome resequencing and genetic variation</h3><p id="p-21">After quality control, a total of 127.98 Gb of sequence data were generated on the basis of our four Duroc pigs, with the average sequencing depth of approximately 13-fold and the average genome coverage of 99.03%. As shown in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>, we identified 9,245,511 SNPs with an average density of 3.76 SNPs/kb. Then, we compared these SNPs that we found with those from the SNPs database that was built using 23 downloaded Duroc pigs. About 88% of the variants (8,205,625 SNPs) in our SNP data set were found in the SNPs database, whereas more than 11% (1,039,886 SNPs) of the variants that we identified were absent from the SNPs database (Supplementary Fig. S3). These novel SNPs substantially expand the database of Duroc genetic variants. Therefore, combining all 27 Duroc individuals together not only increased the sample size, but also significantly increased the SNPs density of the genome. The previous study indicates that a high marker density has positive effects on the identification of selection signatures (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Ma <em>et al.</em> 2015</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/491367/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/491367/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Summary and annotation of SNPs in Duroc pigs</span><div class="sb-div caption-clear"></div></div></div><p id="p-23">Combined, a total of 14,827,549 SNPs with an average density of 6.03 SNPs/kb were detected using 27 sequencing Duroc individuals. Among them, 8,863,149 were located in intergenic regions, 5,496,375 were in intronic regions and 111,759 were in exonic regions. Similarly, 45.49 Gb of sequence data were generated by GBS, with the population sequencing depth of approximately 17-fold and about 0.56% of the bases in the reference genome being covered by at least one reads. A total of 651,425 SNPs were identified in the 282 pig genomes, including 144 intergenic SNPs, 192 intronic SNPs and 16,332 exonic SNPs (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). The SNPs identified by GBS were not evenly distributed across the genome, with an average density of 0.27 SNPs/kb (Supplementary Fig. S4). Similar genome annotation information of EWB and AWB herein is illustrated in Supplementary Table S3.</p></div><div id="sec-14" class="subsection"><h3>Genome-wide artificial selection signatures</h3><p id="p-24">To detect positive selection, the CLR scores were calculated using an identical grid size across the genome. We focused the analyses on windows for which the scores of the statistics fell in the top 2<sup>nd</sup> percentile. As shown in Supplementary Fig. S5, the threshold value is almost greater than the largest CLR scores in the wild populations. In general, we expected a limited amount of artificial selection for commercial traits occurred in the wild populations. Out of 90,588 sliding windows, 1,811 windows were identified as the potential selection signatures in the Duroc population. Then, the windows within a 200kb fragment around the potential selection signatures are merged and the genomic regions were defined as candidate selection regions (CSRs). Correspondingly, a total of 70 fragments, spanning lengths of 81.40 Mb and covering 3.3% of the genome, were identified as CRSs in Duroc pigs (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/11/3617/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Summary of the genomic regions underlying artificial selection in Duroc pigs. (A) Manhattan plot based on CLR tests. (B) Manhattan plot based on FST tests, the points in blue and red (black and orange) represent scores when EWB (AWB) was treated as reference population. (C) The lines illustrate the positions and lengths of candidate artificial selection regions (CASR)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2053718372" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Summary of the genomic regions underlying artificial selection in Duroc pigs. (A) Manhattan plot based on CLR tests. (B) Manhattan plot based on F&lt;sub&gt;ST&lt;/sub&gt; tests, the points in blue and red (black and orange) represent scores when EWB (AWB) was treated as reference population. (C) The lines illustrate the positions and lengths of candidate artificial selection regions (CASR).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/11/3617/F1.medium.gif" width="440" height="244"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/8/11/3617/F1.medium.gif" width="440" height="244"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/11/3617/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/11/3617/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491341" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-25" class="first-child">Summary of the genomic regions underlying artificial selection in Duroc pigs. (A) Manhattan plot based on CLR tests. (B) Manhattan plot based on F<sub>ST</sub> tests, the points in blue and red (black and orange) represent scores when EWB (AWB) was treated as reference population. (C) The lines illustrate the positions and lengths of candidate artificial selection regions (CASR).</p><div class="sb-div caption-clear"></div></div></div><p id="p-26">To further identify the artificial selection signatures, F<sub>ST</sub> statistics were used to detect the differentiation between Duroc and wild pigs. We calculated F<sub>ST</sub> per site and averaged them in non-overlapping 25 kb windows across the genome. The windows that the scores of the statistics fell in the top 2<sup>nd</sup> percentile were considered as significant. Out of 90,322 sliding windows, 1,806 windows were identified as the potential selection signatures and fall into 758 CRSs when AWB was treated as reference population. Similarly, Out of 90,411 sliding windows, 1,804 windows were identified as the potential selection signatures and fall into 684 CRSs when EWB was treated as reference population (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a>).</p><p id="p-27">In this analysis, the overlapping candidate selection regions detected by CLR and F<sub>ST</sub> were defined as candidate artificial selection regions (CASR). Collectively, a total of 38 CASRs, spanning lengths of 40.48 Mb and covering 1.62% of the genome, were identified in Duroc pigs through compared with the wild populations (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1C</a>, Supplementary Table S4). In general, the adaptive evolution of important traits, due to human-driven or natural select, would leave a number of selection signatures in genomic regions, where should harbor the corresponding causal genes. Therefore, the candidate selection regions should be major enriched in genic regions. To test this hypothesis, we identified a total of 246,868 SNPs in the 38 candidate artificial selection regions, of which 143,978 (57.918%) were intergenic, 97,287 (39.136%) were intronic and 1,924 (0.78%) were exonic. Comparing the distribution of SNPs between candidate artificial selection regions and the whole genome, it suggested that there was no predictable pattern in where those selection regions were located (Supplementary Table. S5). This seems to indicate that selection not only plays a role in the gene regions, but also in other regulatory elements of the genome.</p></div><div id="sec-15" class="subsection"><h3>Candidate selection regions harboring loci associated with two economic traits</h3><p id="p-28">Genome-wide association studies have already been proven effective to reveal the underlying genetics of economic traits (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Meyer <em>et al.</em> 2016</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Visscher <em>et al.</em> 2017</a>). If the underlying genetic basis of those domestication traits has been improved by recent artificial selection, the genomic regions that harbored the QTNs revealed through GWAS should overlap with the candidate selection regions. In this analysis, we performed mixed-model association analysis using GBS data of 35,303 high-quality markers. Although the small sample size limited our power, we identified seven associated loci with a less conservative significance threshold (−log10(p-value) ≥4.98), including four SNPs for IMF trait and three SNP for FCR trait (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). Comparing seven associated loci with the candidate selection region, we found that a series of selection signatures were located around the associated loci for both traits (Supplementary Table S6).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/11/3617/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Visualization of the signals revealed by genome-wide association analyses for IMF and FCR traits. (A, C) Manhattan plots present the association of the imputed SNPs with the IMF and FCR traits in 282 Duroc individuals, respectively. (B, D) The corresponding quantile-quantile plots." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2053718372" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Visualization of the signals revealed by genome-wide association analyses for IMF and FCR traits. (A, C) Manhattan plots present the association of the imputed SNPs with the IMF and FCR traits in 282 Duroc individuals, respectively. (B, D) The corresponding quantile-quantile plots.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/11/3617/F2.medium.gif" width="440" height="243"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/8/11/3617/F2.medium.gif" width="440" height="243"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/11/3617/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/11/3617/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491270" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-29" class="first-child">Visualization of the signals revealed by genome-wide association analyses for IMF and FCR traits. (A, C) Manhattan plots present the association of the imputed SNPs with the IMF and FCR traits in 282 Duroc individuals, respectively. (B, D) The corresponding quantile-quantile plots.</p><div class="sb-div caption-clear"></div></div></div><p id="p-30">Because of concerns that the sample size for GWAS is too small to make meaningful scientific conclusion, we investigated the results of GWAS for IMF and FCR traits using a random sampling of 250 individuals, and the process is repeated 10 times. Among them, the similar results of GWAS for IMF and FCR traits have been replicated with smaller sample size, respectively (Supplementary Fig. S6, S7). In addition, we further check genotype-class frequencies and phenotypic means for seven significant SNPs. As shown in Supplementary Table S7, we can clearly observe the trend of phenotypic changes with three genotypes. The results suggested that the SNPs passing the less conservative significance threshold are still promising in this study.</p><p id="p-31">For FCR trait, an associated SNP was close to the 16.375-16.425 Mb selection region on SSC9 and the QTL of ‘time spent feeding’ was reported to be overlapping with this region. For IMF trait, 4 associated SNPs were found close to the 7.1-9.0 Mb selection regions on SSC1. After scanning the pig QTLdb (<a href="https://www.animalgenome.org">https://www.animalgenome.org</a>), we found that these four SNPs were also overlapping with ‘the drip loss’ and ‘the stearic acid content’, respectively (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Hu <em>et al.</em> 2016</a>).</p><p id="p-32">We highlighted the 7.00-10.00 Mb regions around the most significant SNP on SSC1 (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>), where a series of CLR and F<sub>ST</sub> scores exceeded the significance threshold and several harboring genes, such as <em>IGF2R</em>, <em>TMEM181</em>, <em>SOD2</em> and <em>TAGAP</em>, were responsible for sex determination, growth, muscle and bone development (Supplementary Table. S8). Among them, one possible positional candidate gene, insulin like growth factor 2 receptor (<em>IGF2R</em>), serves <em>IGF2</em> turnover in IGF signal mediated process (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Ludwig <em>et al.</em> 1996</a>) and <em>IGF2</em> is known as a major gene that influences the meat quality of pigs (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Clark <em>et al.</em> 2014</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/11/3617/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of GWAS and candidate artificial selection regions (CASR) on SSC1 in Duroc. (A) The top Manhattan plot shows GWAS results for IMF trait. The middle plot shows the distribution of CLR scores calculated in sliding windows. The bottom plot is the distribution of FST scores when AWB (blue) and EWB (green) were treated as reference population, respectively. The red line is the significance threshold. (B) The positions of 31 gene models in this region are indicated: skyblue, minus-strand genes; red, positive-strand genes. The detailed gene list can be found in Supplementary Table S8." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2053718372" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Comparison of GWAS and candidate artificial selection regions (CASR) on SSC1 in Duroc. (A) The top Manhattan plot shows GWAS results for IMF trait. The middle plot shows the distribution of CLR scores calculated in sliding windows. The bottom plot is the distribution of F&lt;sub&gt;ST&lt;/sub&gt; scores when AWB (blue) and EWB (green) were treated as reference population, respectively. The red line is the significance threshold. (B) The positions of 31 gene models in this region are indicated: skyblue, minus-strand genes; red, positive-strand genes. The detailed gene list can be found in Supplementary Table S8.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/11/3617/F3.medium.gif" width="440" height="434"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/8/11/3617/F3.medium.gif" width="440" height="434"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/11/3617/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/11/3617/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491329" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-33" class="first-child">Comparison of GWAS and candidate artificial selection regions (CASR) on SSC1 in Duroc. (A) The top Manhattan plot shows GWAS results for IMF trait. The middle plot shows the distribution of CLR scores calculated in sliding windows. The bottom plot is the distribution of F<sub>ST</sub> scores when AWB (blue) and EWB (green) were treated as reference population, respectively. The red line is the significance threshold. (B) The positions of 31 gene models in this region are indicated: skyblue, minus-strand genes; red, positive-strand genes. The detailed gene list can be found in Supplementary Table S8.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-16" class="subsection"><h3>Go terms, pathways and candidate genes for artificial selection</h3><p id="p-34">To further investigate the genetic basis of improved economic traits, a total of 371 genes overlapping with all 38 candidate artificial selection regions in Duroc pigs were found and the corresponding orthologous genes from human were used to perform an enrichment analysis by DAVID 6.7 (<a href="https://david.ncifcrf.gov/">https://david.ncifcrf.gov/</a>) (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Huang <em>et al.</em> 2009</a>). The results indicated that genes related with a number of terms previously implicated in breed improvement were present within or close to these sweep regions.</p><p id="p-35">Among them, a set of genes overlapping artificial selection regions were enriched for the ability of immunity, which mainly included ‘GO:0042267∼natural killer cell mediated cytotoxicity’ (6 genes), ‘GO:0019882∼antigen processing and presentation’ (6 genes) and ‘hsa04650:Natural killer cell mediated cytotoxicity’ (6 genes) (Supplementary Table S9). Additionally, we also found one significant candidate artificial selection regions that overlapped with <em>CD83</em> gene, which contributes to T lymphocyte proliferation (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Pinho <em>et al.</em> 2014</a>) (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). Due to the high density and selection intensity of rearing animals in the breeding farm, the artificial selection effect is undoubted to play an indispensable role on selecting innate immunity genes indirectly. Six genes with strong selection signatures are significantly enriched in ‘GO:0043651∼linoleic acid metabolic process’ and the <em>LPIN2</em> gene that plays a significant role in fat deposition was found to fall into the 102-104.275 Mb artificial selection regions on SSC6 in Duroc, suggesting that these genes may have been selected during the breeding of lean pigs (Table S2, S9).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/491435/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/491435/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Some candidate genes overlap with the potential regions of artificial selection in Duroc pigs</span><div class="sb-div caption-clear"></div></div></div><p id="p-38">In addition to those significant Go and Pathway terms, a set of artificial selection signatures with extreme <em>P</em>-value coincide with a cluster of genes involved in meat quality, growth, fertility and so on (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). An interesting selection signature was located on SSC15 (104.325-104.8 Mb) and close to the <em>MSTN</em> gene, which was associated with lean muscle mass and played an important role in the process of muscle development (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Baati <em>et al.</em> 2017</a>). As a commercial pig breed of economic importance, leanness has consistently been considered as an objective trait of Durocs breeding. Correspondingly, growth rates and meat quality would also be considered and designed in the breeding program. In this study, <em>WNT9A</em> gene associated with chondrogenesis and <em>CMYA5</em> gene associated with carcass length and meat quality were overlapping with the strong selection signatures. <em>MC5R</em> gene, an established sweep in previous studies, was found close to the 97.625-98.725 Mb regions on SSC6, which was not only associated with back fat thickness but also played an important role in proinflammatory activity (<a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Switonski <em>et al.</em> 2013</a>). The <em>HOMER1</em> gene that is overlapped with the 86.55-92.75 Mb artificial selection regions on SSC2 is associated with muscle differentiation and calcium homeostasis (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Hao <em>et al.</em> 2017</a>). Note that a series of genes, including <em>SOD2</em>, <em>JMY</em>, <em>NDC80</em> and <em>CASP1</em>, overlapping with artificial selection regions in this analysis were associated with the reproductive traits. In comparing with its wild ancestors, litter size, fetal weight and the other fertility traits would change for adapting human-driven selection. These results indicated that the improvement of fertility and other complex traits may due to the polygenic basis, rather than be caused by only a few critical loci.</p></div></div><div class="section conclusions" id="sec-17"><h2 class="">Conclusions</h2><p id="p-39">In summary, this study detected signatures of artificial selection and identified a number of loci associated with some important economic traits, like IMF and FCR, which provided an important resource in future Durocs breeding programs. SNP annotation implied that there was no predictable distribution in where those artificial selection regions were located in genome. Enrichment analyses suggested that the polygenic basis may be a reasonable explanation for the phenomenon, that many different genes associated with the economic traits have often been selected during the improvement and breed formation. The application of combining sweep analysis and genome-wide association analysis are effective in mapping commercial important genes, especially to the data with small sample size caused by the expensive measurement traits.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-40">This work was supported by National Natural Science Foundation of China (No. 31601916), the NSFC-CGIAR project (31361140365), the Fundamental Research Funds for the Central Universities (No. 2662015QD018), the National Swine Industry Technology System (No. CARS-35) and SRF (No. 2017046). Yunlong Ma, Debin Ni and Shuhong Zhao conceived and designed the experiments; Shengsong Xie and Xinyun Li performed the experiments; Yunlong Ma, Saixian Zhang, Kaili Zhang, Xiaoyong Du and Chengchi Fang analyzed the data; Shuhong Zhao contributed reagents/materials/analysis tools; Yunlong Ma wrote the paper. All authors contributed to revision and review of the manuscript.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-5"><p id="p-41">Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.7077125">https://doi.org/10.25387/g3.7077125</a>.</p></li><li class="fn" id="fn-6"><p id="p-42"><em>Communicating editor: D. J. de Koning</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2018-08-12"><span class="received-label">Received </span>August 12, 2018.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2018-09-10"><span class="accepted-label">Accepted </span>September 10, 2018.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2018 Ma <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3617.1" data-doi="10.1038/491315a"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Abbott</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Pig geneticists go the whole hog.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">491</span>: <span class="cit-fpage">315</span>–<span class="cit-lpage">316</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/491315a</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DAbbott%26rft.auinit1%253DA.%26rft.volume%253D491%26rft.issue%253D7424%26rft.spage%253D315%26rft.epage%253D316%26rft.atitle%253DPig%2Bgeneticists%2Bgo%2Bthe%2Bwhole%2Bhog.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F491315a%26rft_id%253Dinfo%253Apmid%252F23151556%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/491315a&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23151556&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3617.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3617.2" data-doi="10.1038/ng.3199"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ai</span>  <span class="cit-name-given-names">H. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fang</span>  <span class="cit-name-given-names">X. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">Z. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Adaptation and possible ancient interspecies introgression in pigs identified by whole-genome sequencing.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">217</span>–<span class="cit-lpage">225</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.3199</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D47%26rft.spage%253D217%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3199%26rft_id%253Dinfo%253Apmid%252F25621459%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.3199&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25621459&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3617.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-8.11.3617.3" data-doi="10.1186/1477-7827-8-33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ashworth</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ross</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stein</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">White</span>  <span class="cit-name-given-names">F. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">DeSilva</span>  <span class="cit-name-given-names">U. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Endometrial caspase 1 and interleukin-18 expression during the estrous cycle and peri-implantation period of porcine pregnancy and response to early exogenous estrogen administration.</span> <abbr class="cit-jnl-abbrev">Reprod. Biol. Endocrinol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">33</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1477-7827-8-33</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DReproductive%2Bbiology%2Band%2Bendocrinology%2B%255Belectronic%2Bresource%255D%2B%253A%2B%2BRB%2526E%26rft.stitle%253DReprod%2BBiol%2BEndocrinol%26rft.aulast%253DAshworth%26rft.auinit1%253DM.%2BD.%26rft.volume%253D8%26rft.spage%253D33%26rft.epage%253D33%26rft.atitle%253DEndometrial%2Bcaspase%2B1%2Band%2Binterleukin-18%2Bexpression%2Bduring%2Bthe%2Bestrous%2Bcycle%2Band%2Bperi-implantation%2Bperiod%2Bof%2Bporcine%2Bpregnancy%2Band%2Bresponse%2Bto%2Bearly%2Bexogenous%2Bestrogen%2Badministration.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1477-7827-8-33%26rft_id%253Dinfo%253Apmid%252F20380728%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1477-7827-8-33&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20380728&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3617.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-2" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3617.4" data-doi="10.1016/j.bbalip.2017.06.017"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baati</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feillet-Coudray</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fouret</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vernus</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goustard</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Myostatin deficiency is associated with lipidomic abnormalities in skeletal muscles.</span> <abbr class="cit-jnl-abbrev">Biochimica Et Biophysica Acta-Molecular and Cell Biology of Lipids</abbr> <span class="cit-vol">1862</span>: <span class="cit-fpage">1044</span>–<span class="cit-lpage">1055</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.bbalip.2017.06.017</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochimica%2BEt%2BBiophysica%2BActa-Molecular%2Band%2BCell%2BBiology%2Bof%2BLipids%26rft.volume%253D1862%26rft.spage%253D1044%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.bbalip.2017.06.017%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.bbalip.2017.06.017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-book" id="cit-8.11.3617.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Briggs</span>  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Briggs</span>  <span class="cit-name-given-names">D. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1969</span> <span class="cit-source">Modern breeds of livestock</span>, <span class="cit-publ-name">Macmillan</span>, <span class="cit-publ-loc">London</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3617.6" data-doi="10.1016/j.ajhg.2015.11.020"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>  <span class="cit-name-given-names">B. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>  <span class="cit-name-given-names">S. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genotype Imputation with Millions of Reference Samples.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">98</span>: <span class="cit-fpage">116</span>–<span class="cit-lpage">126</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ajhg.2015.11.020</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BHum.%2BGenet.%26rft.volume%253D98%26rft.spage%253D116%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2015.11.020%26rft_id%253Dinfo%253Apmid%252F26748515%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2015.11.020&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26748515&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3617.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-8.11.3617.7" data-doi="10.1371/journal.pgen.1004688"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ingham</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kelly</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jadeja</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goulding</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Spinster Homolog 2 (Spns2) Deficiency Causes Early Onset Progressive Hearing Loss.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e1004688</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1004688</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D10%26rft.spage%253De1004688%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1004688%26rft_id%253Dinfo%253Apmid%252F25356849%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1004688&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25356849&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3617.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3617.8" data-doi="10.2527/jas.2014-8283"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bohrer</span>  <span class="cit-name-given-names">B. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tavarez</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boler</span>  <span class="cit-name-given-names">D. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beever</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Effects of the porcine IGF2 intron 3–G3072A mutation on carcass cutability, meat quality, and bacon processing.</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">5778</span>–<span class="cit-lpage">5788</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2527/jas.2014-8283</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAnim.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.2527%252Fjas.2014-8283%26rft_id%253Dinfo%253Apmid%252F25367517%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2527/jas.2014-8283&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25367517&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3617.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3617.9" data-doi="10.1186/1471-2156-15-27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Do</span>  <span class="cit-name-given-names">D. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ostersen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Strathe</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mark</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jensen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genome-wide association and systems genetic analyses of residual feed intake, daily feed consumption, backfat and weight gain in pigs.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">27</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2156-15-27</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenet.%26rft.volume%253D15%26rft.spage%253D27%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2156-15-27%26rft_id%253Dinfo%253Apmid%252F24533460%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2156-15-27&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24533460&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3617.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3617.10" data-doi="10.1186/1471-2164-9-516"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Duggal</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gillanders</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holmes</span>  <span class="cit-name-given-names">T. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bailey-Wilson</span>  <span class="cit-name-given-names">J. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Establishing an adjusted p-value threshold to control the family-wide type 1 error in genome wide association studies.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">516</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2164-9-516</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DDuggal%26rft.auinit1%253DP.%26rft.volume%253D9%26rft.spage%253D516%26rft.epage%253D516%26rft.atitle%253DEstablishing%2Ban%2Badjusted%2Bp-value%2Bthreshold%2Bto%2Bcontrol%2Bthe%2Bfamily-wide%2Btype%2B1%2Berror%2Bin%2Bgenome%2Bwide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-9-516%26rft_id%253Dinfo%253Apmid%252F18976480%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-9-516&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18976480&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3617.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3617.11" data-doi="10.1111/age.12559"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Edea</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hong</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jung</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">Y. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Detecting selection signatures between Duroc and Duroc synthetic pig populations using high-density SNP chip.</span> <abbr class="cit-jnl-abbrev">Anim. Genet.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">473</span>–<span class="cit-lpage">477</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/age.12559</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnim.%2BGenet.%26rft.volume%253D48%26rft.spage%253D473%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fage.12559%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/age.12559&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3617.12" data-doi="10.1371/journal.pone.0019379"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawamoto</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A Robust, Simple Genotyping-by-Sequencing (GBS) Approach for High Diversity Species.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e19379</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0019379</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DElshire%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253De19379%26rft.epage%253De19379%26rft.atitle%253DA%2Brobust%252C%2Bsimple%2Bgenotyping-by-sequencing%2B%2528GBS%2529%2Bapproach%2Bfor%2Bhigh%2Bdiversity%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0019379%26rft_id%253Dinfo%253Apmid%252F21573248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0019379&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21573248&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3617.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3617.13" data-doi="10.1111/age.12545"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eusebi</span>  <span class="cit-name-given-names">P. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gonzálezprendes</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quintanilla</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tibau</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cardoso</span>  <span class="cit-name-given-names">T. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">A genome-wide association analysis for carcass traits in a commercial Duroc pig population.</span> <abbr class="cit-jnl-abbrev">Anim. Genet.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">466</span>–<span class="cit-lpage">469</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/age.12545</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnim.%2BGenet.%26rft.volume%253D48%26rft.spage%253D466%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fage.12545%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/age.12545&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3617.14" data-doi="10.1002/gepi.20310"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">X. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stamier</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">E. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A multiple testing correction method for genetic association studies using correlated single nucleotide polymorphisms.</span> <abbr class="cit-jnl-abbrev">Genet. Epidemiol.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">361</span>–<span class="cit-lpage">369</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/gepi.20310</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetic%2Bepidemiology%26rft.stitle%253DGenet%2BEpidemiol%26rft.aulast%253DGao%26rft.auinit1%253DX.%26rft.volume%253D32%26rft.issue%253D4%26rft.spage%253D361%26rft.epage%253D369%26rft.atitle%253DA%2Bmultiple%2Btesting%2Bcorrection%2Bmethod%2Bfor%2Bgenetic%2Bassociation%2Bstudies%2Busing%2Bcorrelated%2Bsingle%2Bnucleotide%2Bpolymorphisms.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fgepi.20310%26rft_id%253Dinfo%253Apmid%252F18271029%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/gepi.20310&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18271029&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3617.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000255471100007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-2" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3617.15" data-doi="10.1186/s12863-017-0532-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hao</span>  <span class="cit-name-given-names">X. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plastow</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">C. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">S. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">Z. Q.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genome-wide association study identifies candidate genes for piglet splay leg syndrome in different populations.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">64</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12863-017-0532-4</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenet.%26rft.volume%253D18%26rft.spage%253D64%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12863-017-0532-4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12863-017-0532-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3617.16" data-doi="10.3168/jds.2008-1646"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowman</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chamberlain</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Invited review: Genomic selection in dairy cattle: progress and challenges.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">433</span>–<span class="cit-lpage">443</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.2008-1646</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DHayes%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D92%26rft.issue%253D2%26rft.spage%253D433%26rft.epage%253D443%26rft.atitle%253DInvited%2Breview%253A%2BGenomic%2Bselection%2Bin%2Bdairy%2Bcattle%253A%2BProgress%2Band%2Bchallenges%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2008-1646%26rft_id%253Dinfo%253Apmid%252F19164653%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2008-1646&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19164653&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3617.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000262654900001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-8.11.3617.17" data-doi="10.1007/s11033-008-9385-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">X. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">X. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Molecular characterization, chromosomal localization and association analysis with back-fat thickness of porcine LPIN2 and LPIN3.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Rep.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">1819</span>–<span class="cit-lpage">1824</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s11033-008-9385-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bbiology%2Breports%26rft.stitle%253DMol%2BBiol%2BRep%26rft.aulast%253DHe%26rft.auinit1%253DX.%26rft.volume%253D36%26rft.issue%253D7%26rft.spage%253D1819%26rft.epage%253D1824%26rft.atitle%253DMolecular%2Bcharacterization%252C%2Bchromosomal%2Blocalization%2Band%2Bassociation%2Banalysis%2Bwith%2Bback-fat%2Bthickness%2Bof%2Bporcine%2BLPIN2%2Band%2BLPIN3.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11033-008-9385-2%26rft_id%253Dinfo%253Apmid%252F18989753%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11033-008-9385-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18989753&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3617.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3617.18" data-doi="10.1093/nar/gkv1233"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">Z. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reecy</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Developmental progress and current status of the Animal QTLdb.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">D827</span>–<span class="cit-lpage">D833</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkv1233</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv1233%26rft_id%253Dinfo%253Apmid%252F26602686%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkv1233&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26602686&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3617.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3617.19" data-doi="10.1038/nprot.2008.211"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sherman</span>  <span class="cit-name-given-names">B. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lempicki</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Systematic and integrative analysis of large gene lists using DAVID bioinformatics resources.</span> <abbr class="cit-jnl-abbrev">Nat. Protoc.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">44</span>–<span class="cit-lpage">57</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nprot.2008.211</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BProtoc%26rft.aulast%253DHuang%26rft.auinit1%253Dd.%2Ba.%2B%2B.%2BW.%26rft.volume%253D4%26rft.issue%253D1%26rft.spage%253D44%26rft.epage%253D57%26rft.atitle%253DSystematic%2Band%2Bintegrative%2Banalysis%2Bof%2Blarge%2Bgene%2Blists%2Busing%2BDAVID%2Bbioinformatics%2Bresources.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnprot.2008.211%26rft_id%253Dinfo%253Apmid%252F19131956%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nprot.2008.211&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19131956&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3617.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000265781800006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-8.11.3617.20" data-doi="10.1080/10495398.2017.1279171"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hwang</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">An</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kwon</span>  <span class="cit-name-given-names">S. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">D. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">T. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Associations of the Polymorphisms in DHRS4, SERPING1, and APOR Genes with Postmortem pH in Berkshire Pigs.</span> <abbr class="cit-jnl-abbrev">Anim. Biotechnol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">288</span>–<span class="cit-lpage">293</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1080/10495398.2017.1279171</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnim.%2BBiotechnol.%26rft.volume%253D28%26rft.spage%253D288%26rft_id%253Dinfo%253Adoi%252F10.1080%252F10495398.2017.1279171%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/10495398.2017.1279171&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3617.21" data-doi="10.1051/gse:2003057"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Khatkar</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomson</span>  <span class="cit-name-given-names">P. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tammen</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raadsma</span>  <span class="cit-name-given-names">H. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Quantitative trait loci mapping in dairy cattle: review and meta-analysis.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">163</span>–<span class="cit-lpage">190</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1297-9686-36-2-163</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DKhatkar%26rft.auinit1%253DM.%2BS.%26rft.volume%253D36%26rft.issue%253D2%26rft.spage%253D163%26rft.epage%253D190%26rft.atitle%253DQuantitative%2Btrait%2Bloci%2Bmapping%2Bin%2Bdairy%2Bcattle%253A%2Breview%2Band%2Bmeta-analysis.%26rft_id%253Dinfo%253Adoi%252F10.1051%252Fgse%253A2003057%26rft_id%253Dinfo%253Apmid%252F15040897%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1051/gse:2003057&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15040897&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3617.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000188936300002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-8.11.3617.22" data-doi="10.1016/j.repbio.2017.05.013"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kwiatkowska</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wąsowska</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gilun</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Expression of hypoxia inducible factor 1α and antioxidant enzymes: Superoxide dismutases-1 and -2 in ischemic porcine endometrium.</span> <abbr class="cit-jnl-abbrev">Reprod. Biol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">289</span>–<span class="cit-lpage">293</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.repbio.2017.05.013</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DReprod.%2BBiol.%26rft.volume%253D17%26rft.spage%253D289%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.repbio.2017.05.013%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.repbio.2017.05.013&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-8.11.3617.23" data-doi="10.3724/SP.J.1005.2010.00375"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">D. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">J. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lei</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">D. Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">S. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Polymorphism in exon 2 of pig FIT1 gene and its association with fat-deposition-related traits.</span> <abbr class="cit-jnl-abbrev">Yi Chuan</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">375</span>–<span class="cit-lpage">380</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3724/SP.J.1005.2010.00375</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYi%2Bchuan%2B%253D%2BHereditas%2B%252F%2BZhongguo%2Byi%2Bchuan%2Bxue%2Bhui%2Bbian%2Bji%26rft.stitle%253DYi%2BChuan%26rft.aulast%253DLi%26rft.auinit1%253DD.%2BZ.%26rft.volume%253D32%26rft.issue%253D4%26rft.spage%253D375%26rft.epage%253D380%26rft.atitle%253D%255BPolymorphism%2Bin%2Bexon%2B2%2Bof%2Bpig%2BFIT1%2Bgene%2Band%2Bits%2Bassociation%2Bwith%2Bfat-deposition-related%2Btraits%255D%26rft_id%253Dinfo%253Adoi%252F10.3724%252FSP.J.1005.2010.00375%26rft_id%253Dinfo%253Apmid%252F20423892%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3724/SP.J.1005.2010.00375&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20423892&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3617.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3617.24" data-doi="10.1093/bioinformatics/btp324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fast and accurate short read alignment with Burrows-Wheeler transform.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1754</span>–<span class="cit-lpage">1760</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btp324</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp324%26rft_id%253Dinfo%253Apmid%252F19451168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp324&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19451168&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3617.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000267665900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3617.25" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Handsaker</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wysoker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fennell</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The Sequence Alignment/Map format and SAMtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2078</span>–<span class="cit-lpage">2079</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btp352</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp352&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19505943&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3617.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268808600014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-8.11.3617.26" data-doi="10.1262/jrd.2015-021"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">Z. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cui</span>  <span class="cit-name-given-names">X. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Namgoong</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">N. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Junction-mediating and regulatory protein (JMY) is essential for early porcine embryonic development.</span> <abbr class="cit-jnl-abbrev">J. Reprod. Dev.</abbr> <span class="cit-vol">61</span>: <span class="cit-fpage">361</span>–<span class="cit-lpage">367</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1262/jrd.2015-021</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BReprod.%2BDev.%26rft.volume%253D61%26rft.spage%253D361%26rft_id%253Dinfo%253Adoi%252F10.1262%252Fjrd.2015-021%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1262/jrd.2015-021&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3617.27" data-doi="10.1006/dbio.1996.0182"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ludwig</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eggenschwiler</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fisher</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ercole</span>  <span class="cit-name-given-names">A. J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davenport</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Mouse mutants lacking the type 2 IGF receptor (IGF2R) are rescued from perinatal lethality in Igf2 and Igf1r null backgrounds.</span> <abbr class="cit-jnl-abbrev">Dev. Biol.</abbr> <span class="cit-vol">177</span>: <span class="cit-fpage">517</span>–<span class="cit-lpage">535</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1006/dbio.1996.0182</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bbiology%26rft.stitle%253DDev%2BBiol%26rft.aulast%253DLudwig%26rft.auinit1%253DT.%26rft.volume%253D177%26rft.issue%253D2%26rft.spage%253D517%26rft.epage%253D535%26rft.atitle%253DMouse%2Bmutants%2Blacking%2Bthe%2Btype%2B2%2BIGF%2Breceptor%2B%2528IGF2R%2529%2Bare%2Brescued%2Bfrom%2Bperinatal%2Blethality%2Bin%2BIgf2%2Band%2BIgf1r%2Bnull%2Bbackgrounds.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fdbio.1996.0182%26rft_id%253Dinfo%253Apmid%252F8806828%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/dbio.1996.0182&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8806828&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3617.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996VA82900012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3617.28" data-doi="10.1038/hdy.2015.42"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ding</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qanbari</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weigend</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Properties of different selection signature statistics and a new strategy for combining them.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">115</span>: <span class="cit-fpage">426</span>–<span class="cit-lpage">436</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/hdy.2015.42</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D115%26rft.spage%253D426%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2015.42%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2015.42&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3617.29" data-doi="10.1371/journal.pone.0196215"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">Y. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gu</span>  <span class="cit-name-given-names">L. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">L. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">S. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Identifying artificial selection signals in the chicken genome.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">e0196215</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0196215</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D13%26rft.spage%253De0196215%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0196215%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0196215&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3617.30" data-doi="10.1101/gr.107524.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McKenna</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hanna</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sivachenko</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cibulskis</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The Genome Analysis Toolkit: A MapReduce framework for analyzing next-generation DNA sequencing data.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">1297</span>–<span class="cit-lpage">1303</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.107524.110</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.107524.110%26rft_id%253Dinfo%253Apmid%252F20644199%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIwLzkvMTI5NyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzgvMTEvMzYxNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3617.31" data-doi="10.1038/ng.3633"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meyer</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Choi</span>  <span class="cit-name-given-names">J. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sanches</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plessis</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flowers</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Domestication history and geographical adaptation inferred from a SNP map of African rice.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">1083</span>–<span class="cit-lpage">1088</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.3633</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D48%26rft.spage%253D1083%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3633%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.3633&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3617.32" data-doi="10.1101/gr.4252305"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williamson</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hubisz</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Genomic scans for selective sweeps using SNP data.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1566</span>–<span class="cit-lpage">1575</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.4252305</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DNielsen%26rft.auinit1%253DR.%26rft.volume%253D15%26rft.issue%253D11%26rft.spage%253D1566%26rft.epage%253D1575%26rft.atitle%253DGenomic%2Bscans%2Bfor%2Bselective%2Bsweeps%2Busing%2BSNP%2Bdata%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.4252305%26rft_id%253Dinfo%253Apmid%252F16251466%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxNS8xMS8xNTY2IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMS8zNjE3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3617.33" data-doi="10.1189/jlb.0413239"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pinho</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Migliori</span>  <span class="cit-name-given-names">I. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flatow</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barbuto</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Dendritic cell membrane CD83 enhances immune responses by boosting intracellular calcium release in T lymphocytes.</span> <abbr class="cit-jnl-abbrev">J. Leukoc. Biol.</abbr> <span class="cit-vol">95</span>: <span class="cit-fpage">755</span>–<span class="cit-lpage">762</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1189/jlb.0413239</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BLeukoc.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1189%252Fjlb.0413239%26rft_id%253Dinfo%253Apmid%252F24436459%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1189/jlb.0413239&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24436459&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3617.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3617.34" data-doi="10.1016/j.cub.2009.11.055"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pickrell</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coop</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The Genetics of Human Adaptation: Hard Sweeps, Soft Sweeps, and Polygenic Adaptation.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">R208</span>–<span class="cit-lpage">R215</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.cub.2009.11.055</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DPritchard%26rft.auinit1%253DJ.%2BK.%26rft.volume%253D20%26rft.issue%253D4%26rft.spage%253DR208%26rft.epage%253DR215%26rft.atitle%253DThe%2Bgenetics%2Bof%2Bhuman%2Badaptation%253A%2Bhard%2Bsweeps%252C%2Bsoft%2Bsweeps%252C%2Band%2Bpolygenic%2Badaptation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2009.11.055%26rft_id%253Dinfo%253Apmid%252F20178769%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2009.11.055&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20178769&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3617.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000275149200026&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3617.35" data-doi="10.1086/519795"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Purcell</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neale</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Todd-Brown</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferreira</span>  <span class="cit-name-given-names">M. A. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">PLINK: A tool set for whole-genome association and population-based linkage analyses.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">81</span>: <span class="cit-fpage">559</span>–<span class="cit-lpage">575</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1086/519795</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DPurcell%26rft.auinit1%253DS.%26rft.volume%253D81%26rft.issue%253D3%26rft.spage%253D559%26rft.epage%253D575%26rft.atitle%253DPLINK%253A%2Ba%2Btool%2Bset%2Bfor%2Bwhole-genome%2Bassociation%2Band%2Bpopulation-based%2Blinkage%2Banalyses.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F519795%26rft_id%253Dinfo%253Apmid%252F17701901%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/519795&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17701901&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3617.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3617.36" data-doi="10.1371/journal.pgen.1004148"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Qanbari</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pausch</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jansen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Somel</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Strom</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Classic Selective Sweeps Revealed by Massive Sequencing in Cattle.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e1004148</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1004148</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D10%26rft.spage%253De1004148%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1004148%26rft_id%253Dinfo%253Apmid%252F24586189%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1004148&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24586189&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3617.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-confproc" id="cit-8.11.3617.37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ros-Freixedes</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gol</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pena</span>  <span class="cit-name-given-names">R. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tor</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dekkers</span>  <span class="cit-name-given-names">J. C. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> Genome-wide association study for intramuscular fat content and composition in Duroc pigs, pp. in <em>World Congress of Genetics Applied To Livestock Production Wcgalp</em>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3617.38" data-doi="10.1038/nature01140"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sabeti</span>  <span class="cit-name-given-names">P. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reich</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Higgins</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Levine</span>  <span class="cit-name-given-names">H. Z. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Richter</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Detecting recent positive selection in the human genome from haplotype structure.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">419</span>: <span class="cit-fpage">832</span>–<span class="cit-lpage">837</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature01140</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DSabeti%26rft.auinit1%253DP.%2BC.%26rft.volume%253D419%26rft.issue%253D6909%26rft.spage%253D832%26rft.epage%253D837%26rft.atitle%253DDetecting%2Brecent%2Bpositive%2Bselection%2Bin%2Bthe%2Bhuman%2Bgenome%2Bfrom%2Bhaplotype%2Bstructure.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature01140%26rft_id%253Dinfo%253Apmid%252F12397357%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature01140&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12397357&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3617.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000178769800045&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3617.39" data-doi="10.1186/s12863-016-0368-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sato</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Uemoto</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kikuchi</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Egawa</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kohira</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">SNP- and haplotype-based genome-wide association studies for growth, carcass, and meat quality traits in a Duroc multigenerational population.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">60</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12863-016-0368-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenet.%26rft.volume%253D17%26rft.spage%253D60%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12863-016-0368-3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12863-016-0368-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-confproc" id="cit-8.11.3617.40"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Simianer</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qanbari</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> Statistical Problems in Livestock Population Genomics, pp. in <em>World Congress on Genetics Applied To Livestock Production</em>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-8.11.3617.41" data-doi="10.1242/dev.02471"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Später</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">T. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Sullivan</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gruber</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Conner</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Wnt9a signaling is required for joint integrity and regulation of Ihh during chondrogenesis.</span> <abbr class="cit-jnl-abbrev">Development</abbr> <span class="cit-vol">133</span>: <span class="cit-fpage">3039</span>–<span class="cit-lpage">3049</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1242/dev.02471</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopment%26rft_id%253Dinfo%253Adoi%252F10.1242%252Fdev.02471%26rft_id%253Dinfo%253Apmid%252F16818445%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZGV2ZWxvcCI7czo1OiJyZXNpZCI7czoxMToiMTMzLzE1LzMwMzkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzExLzM2MTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-2" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3617.42" data-doi="10.1007/s13353-013-0163-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Switonski</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mankowska</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salamon</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Family of melanocortin receptor (MCR) genes in mammals-mutations, polymorphisms and phenotypic effects.</span> <abbr class="cit-jnl-abbrev">J. Appl. Genet.</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">461</span>–<span class="cit-lpage">472</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s13353-013-0163-z</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAppl.%2BGenet.%26rft.volume%253D54%26rft.spage%253D461%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs13353-013-0163-z%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s13353-013-0163-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3617.43" data-doi="10.1016/j.ajhg.2017.06.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Visscher</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wray</span>  <span class="cit-name-given-names">N. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sklar</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCarthy</span>  <span class="cit-name-given-names">M. I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">10 Years of GWAS Discovery: Biology, Function, and Translation.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">5</span>–<span class="cit-lpage">22</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ajhg.2017.06.005</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BHum.%2BGenet.%26rft.volume%253D101%26rft.spage%253D5%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2017.06.005%26rft_id%253Dinfo%253Apmid%252F28686856%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2017.06.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28686856&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3617.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3617.44" data-doi="10.1093/nar/gkq603"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">M. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hakonarson</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">ANNOVAR: functional annotation of genetic variants from high-throughput sequencing data.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">e164</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkq603</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkq603%26rft_id%253Dinfo%253Apmid%252F20601685%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkq603&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20601685&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3617.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-8.11.3617.45" data-doi="10.1007/s11033-014-3374-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">X. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luo</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qi</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Hec1 inhibition alters spindle morphology and chromosome alignment in porcine oocytes.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Rep.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">5089</span>–<span class="cit-lpage">5095</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s11033-014-3374-4</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BRep.%26rft.volume%253D41%26rft.spage%253D5089%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11033-014-3374-4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11033-014-3374-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3617.46" data-doi="10.1111/j.1558-5646.1984.tb05657.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weir</span>  <span class="cit-name-given-names">B. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cockerham</span>  <span class="cit-name-given-names">C. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title">Estimating F-Statistics for the Analysis of Population Structure.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">1358</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1558-5646.1984.tb05657.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D38%26rft.spage%253D1358%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1558-5646.1984.tb05657.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1558-5646.1984.tb05657.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-8.11.3617.47" data-doi="10.1007/s11033-010-0334-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">X. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">X. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yin</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The molecular characterization and associations of porcine cardiomyopathy asssociated 5 (CMYA5) gene with carcass trait and meat quality.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Rep.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">2085</span>–<span class="cit-lpage">2090</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s11033-010-0334-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bbiology%2Breports%26rft.stitle%253DMol%2BBiol%2BRep%26rft.aulast%253DXu%26rft.auinit1%253DX.%26rft.volume%253D38%26rft.issue%253D3%26rft.spage%253D2085%26rft.epage%253D2090%26rft.atitle%253DThe%2Bmolecular%2Bcharacterization%2Band%2Bassociations%2Bof%2Bporcine%2Bcardiomyopathy%2Basssociated%2B5%2B%2528CMYA5%2529%2Bgene%2Bwith%2Bcarcass%2Btrait%2Band%2Bmeat%2Bquality.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11033-010-0334-5%26rft_id%253Dinfo%253Apmid%252F20859690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11033-010-0334-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20859690&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3617.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-confproc" id="cit-8.11.3617.48"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Erbe</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> Results of Genome Wide Association Studies Improve the Accuracy of Genomic Selection, pp. in <em>World Congress on Genetics Applied To Livestock Production</em>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3617.49" data-doi="10.1038/ng.2310"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genome-wide efficient mixed-model analysis for association studies.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">821</span>–<span class="cit-lpage">824</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.2310</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DZhou%26rft.auinit1%253DX.%26rft.volume%253D44%26rft.issue%253D7%26rft.spage%253D821%26rft.epage%253D824%26rft.atitle%253DGenome-wide%2Befficient%2Bmixed-model%2Banalysis%2Bfor%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2310%26rft_id%253Dinfo%253Apmid%252F22706312%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2310&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22706312&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3617.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/8/11/3617.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/11/3607" title="An Efficient Genome Editing Strategy To Generate Putative Null Mutants in Caenorhabditis elegans Using CRISPR/Cas9" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/11/3627" title="Whole-Genome Multi-omic Study of Survival in Patients with Glioblastoma Multiforme" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/8/11"><span>Volume 8 Issue 11, November 2018</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/8/11.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/8/11.cover-source.jpg?itok=lFO8v6bg" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 8 (11)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/11/3617.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-RU8jvjFETVtW6BGQn7DxI-hj7UE8u97zdEfafBVbx2Q" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/8/11/3617" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/8/11/3617" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/8/11/3617" class="active" data-icon-position="" data-hide-link-title="0">Genomic Analysis To Identify Signatures of Artificial Selection and Loci Associated with Important Economic Traits in Duroc Pigs</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/491452" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-aUzC1iRA90hlK6tL41ZSDtoE4M5sJmD_jAbMjlBPOR4" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/11/3617.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/8/11/3617" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-wlp50z8lzQxhBouBTi8bZkTE4YTs-VU_KSNfT_d_1GA" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/8/11/3617" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/11/3617.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="491452" id="node491452--2" data-pisa="ggg;8/11/3617" data-pisa-master="ggg;g3.118.200665" data-apath="/ggg/8/11/3617.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic Analysis To Identify Signatures of Artificial Selection and Loci Associated with Important Economic Traits in Duroc Pigs</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Yunlong Ma</span>, <span class="highwire-citation-author" data-delta="1">Saixian Zhang</span>, <span class="highwire-citation-author" data-delta="2">Kaili Zhang</span>, <span class="highwire-citation-author" data-delta="3">Chengchi Fang</span>, <span class="highwire-citation-author" data-delta="4">Shengsong Xie</span>, <span class="highwire-citation-author" data-delta="5">Xiaoyong Du</span>, <span class="highwire-citation-author" data-delta="6">Xinyun Li</span>, <span class="highwire-citation-author" data-delta="7">Debin Ni</span> and <span class="highwire-citation-author" data-delta="8">Shuhong Zhao</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3617-3625;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200665 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node491452--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Yunlong Ma </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Key Laboratory of Agricultural Animal Genetics, Breeding, and Reproduction of the Ministry of Education & Key Laboratory of Swine Genetics and Breeding of the Ministry of Agriculture, Huazhong Agricultural University, Wuhan 430070, P. R. China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yunlong%2BMa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ma%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYunlong%2BMa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Saixian Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Key Laboratory of Agricultural Animal Genetics, Breeding, and Reproduction of the Ministry of Education & Key Laboratory of Swine Genetics and Breeding of the Ministry of Agriculture, Huazhong Agricultural University, Wuhan 430070, P. R. China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Saixian%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASaixian%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Kaili Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Key Laboratory of Agricultural Animal Genetics, Breeding, and Reproduction of the Ministry of Education & Key Laboratory of Swine Genetics and Breeding of the Ministry of Agriculture, Huazhong Agricultural University, Wuhan 430070, P. R. China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kaili%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKaili%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Chengchi Fang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Key Laboratory of Agricultural Animal Genetics, Breeding, and Reproduction of the Ministry of Education & Key Laboratory of Swine Genetics and Breeding of the Ministry of Agriculture, Huazhong Agricultural University, Wuhan 430070, P. R. China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chengchi%2BFang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fang%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChengchi%2BFang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Shengsong Xie </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Key Laboratory of Agricultural Animal Genetics, Breeding, and Reproduction of the Ministry of Education & Key Laboratory of Swine Genetics and Breeding of the Ministry of Agriculture, Huazhong Agricultural University, Wuhan 430070, P. R. China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shengsong%2BXie%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xie%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AShengsong%2BXie%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Xiaoyong Du </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Key Laboratory of Agricultural Animal Genetics, Breeding, and Reproduction of the Ministry of Education & Key Laboratory of Swine Genetics and Breeding of the Ministry of Agriculture, Huazhong Agricultural University, Wuhan 430070, P. R. China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xiaoyong%2BDu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Du%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXiaoyong%2BDu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Xinyun Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Key Laboratory of Agricultural Animal Genetics, Breeding, and Reproduction of the Ministry of Education & Key Laboratory of Swine Genetics and Breeding of the Ministry of Agriculture, Huazhong Agricultural University, Wuhan 430070, P. R. China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xinyun%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXinyun%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Debin Ni </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Key Laboratory of Agricultural Animal Genetics, Breeding, and Reproduction of the Ministry of Education & Key Laboratory of Swine Genetics and Breeding of the Ministry of Agriculture, Huazhong Agricultural University, Wuhan 430070, P. R. China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Debin%2BNi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ni%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADebin%2BNi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:shzhao@mail.hzau.edu.cn" class="" data-icon-position="" data-hide-link-title="0">shzhao@mail.hzau.edu.cn</a>
<a href="mailto:nidebin@mail.hzau.edu.cn" class="" data-icon-position="" data-hide-link-title="0">nidebin@mail.hzau.edu.cn</a></span></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Shuhong Zhao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Key Laboratory of Agricultural Animal Genetics, Breeding, and Reproduction of the Ministry of Education & Key Laboratory of Swine Genetics and Breeding of the Ministry of Agriculture, Huazhong Agricultural University, Wuhan 430070, P. R. China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shuhong%2BZhao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhao%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AShuhong%2BZhao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:shzhao@mail.hzau.edu.cn" class="" data-icon-position="" data-hide-link-title="0">shzhao@mail.hzau.edu.cn</a>
<a href="mailto:nidebin@mail.hzau.edu.cn" class="" data-icon-position="" data-hide-link-title="0">nidebin@mail.hzau.edu.cn</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/8/11/3617" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/8/11/3617&amp;title=Genomic%20Analysis%20To%20Identify%20Signatures%20of%20Artificial%20Selection%20and%20Loci%20Associated%20with%20Important%20Economic%20Traits%20in%20Duroc%20Pigs" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/8/11/3617&amp;title=Genomic%20Analysis%20To%20Identify%20Signatures%20of%20Artificial%20Selection%20and%20Loci%20Associated%20with%20Important%20Economic%20Traits%20in%20Duroc%20Pigs" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/8/11/3617&amp;title=Genomic%20Analysis%20To%20Identify%20Signatures%20of%20Artificial%20Selection%20and%20Loci%20Associated%20with%20Important%20Economic%20Traits%20in%20Duroc%20Pigs" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/8/11/3617&amp;text=Genomic%20Analysis%20To%20Identify%20Signatures%20of%20Artificial%20Selection%20and%20Loci%20Associated%20with%20Important%20Economic%20Traits%20in%20Duroc%20Pigs" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/8/11/3617&amp;title=Genomic%20Analysis%20To%20Identify%20Signatures%20of%20Artificial%20Selection%20and%20Loci%20Associated%20with%20Important%20Economic%20Traits%20in%20Duroc%20Pigs" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/8/11/3617&amp;t=Genomic%20Analysis%20To%20Identify%20Signatures%20of%20Artificial%20Selection%20and%20Loci%20Associated%20with%20Important%20Economic%20Traits%20in%20Duroc%20Pigs" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/8/11/3617&amp;title=Genomic%20Analysis%20To%20Identify%20Signatures%20of%20Artificial%20Selection%20and%20Loci%20Associated%20with%20Important%20Economic%20Traits%20in%20Duroc%20Pigs" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/8/11/3617&amp;title=Genomic%20Analysis%20To%20Identify%20Signatures%20of%20Artificial%20Selection%20and%20Loci%20Associated%20with%20Important%20Economic%20Traits%20in%20Duroc%20Pigs" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/491452/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="491452" id="node491452--3" data-pisa="ggg;8/11/3617" data-pisa-master="ggg;g3.118.200665" data-apath="/ggg/8/11/3617.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic Analysis To Identify Signatures of Artificial Selection and Loci Associated with Important Economic Traits in Duroc Pigs</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Yunlong Ma</span>, <span class="highwire-citation-author" data-delta="1">Saixian Zhang</span>, <span class="highwire-citation-author" data-delta="2">Kaili Zhang</span>, <span class="highwire-citation-author" data-delta="3">Chengchi Fang</span>, <span class="highwire-citation-author" data-delta="4">Shengsong Xie</span>, <span class="highwire-citation-author" data-delta="5">Xiaoyong Du</span>, <span class="highwire-citation-author" data-delta="6">Xinyun Li</span>, <span class="highwire-citation-author" data-delta="7">Debin Ni</span> and <span class="highwire-citation-author" data-delta="8">Shuhong Zhao</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3617-3625;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200665 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node491452--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Yunlong Ma </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Key Laboratory of Agricultural Animal Genetics, Breeding, and Reproduction of the Ministry of Education & Key Laboratory of Swine Genetics and Breeding of the Ministry of Agriculture, Huazhong Agricultural University, Wuhan 430070, P. R. China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yunlong%2BMa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ma%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYunlong%2BMa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Saixian Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Key Laboratory of Agricultural Animal Genetics, Breeding, and Reproduction of the Ministry of Education & Key Laboratory of Swine Genetics and Breeding of the Ministry of Agriculture, Huazhong Agricultural University, Wuhan 430070, P. R. China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Saixian%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASaixian%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Kaili Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Key Laboratory of Agricultural Animal Genetics, Breeding, and Reproduction of the Ministry of Education & Key Laboratory of Swine Genetics and Breeding of the Ministry of Agriculture, Huazhong Agricultural University, Wuhan 430070, P. R. China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kaili%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKaili%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Chengchi Fang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Key Laboratory of Agricultural Animal Genetics, Breeding, and Reproduction of the Ministry of Education & Key Laboratory of Swine Genetics and Breeding of the Ministry of Agriculture, Huazhong Agricultural University, Wuhan 430070, P. R. China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chengchi%2BFang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fang%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChengchi%2BFang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Shengsong Xie </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Key Laboratory of Agricultural Animal Genetics, Breeding, and Reproduction of the Ministry of Education & Key Laboratory of Swine Genetics and Breeding of the Ministry of Agriculture, Huazhong Agricultural University, Wuhan 430070, P. R. China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shengsong%2BXie%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xie%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AShengsong%2BXie%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Xiaoyong Du </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Key Laboratory of Agricultural Animal Genetics, Breeding, and Reproduction of the Ministry of Education & Key Laboratory of Swine Genetics and Breeding of the Ministry of Agriculture, Huazhong Agricultural University, Wuhan 430070, P. R. China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xiaoyong%2BDu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Du%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXiaoyong%2BDu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Xinyun Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Key Laboratory of Agricultural Animal Genetics, Breeding, and Reproduction of the Ministry of Education & Key Laboratory of Swine Genetics and Breeding of the Ministry of Agriculture, Huazhong Agricultural University, Wuhan 430070, P. R. China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xinyun%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXinyun%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Debin Ni </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Key Laboratory of Agricultural Animal Genetics, Breeding, and Reproduction of the Ministry of Education & Key Laboratory of Swine Genetics and Breeding of the Ministry of Agriculture, Huazhong Agricultural University, Wuhan 430070, P. R. China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Debin%2BNi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ni%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADebin%2BNi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:shzhao@mail.hzau.edu.cn" class="" data-icon-position="" data-hide-link-title="0">shzhao@mail.hzau.edu.cn</a>
<a href="mailto:nidebin@mail.hzau.edu.cn" class="" data-icon-position="" data-hide-link-title="0">nidebin@mail.hzau.edu.cn</a></span></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Shuhong Zhao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Key Laboratory of Agricultural Animal Genetics, Breeding, and Reproduction of the Ministry of Education & Key Laboratory of Swine Genetics and Breeding of the Ministry of Agriculture, Huazhong Agricultural University, Wuhan 430070, P. R. China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shuhong%2BZhao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhao%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AShuhong%2BZhao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:shzhao@mail.hzau.edu.cn" class="" data-icon-position="" data-hide-link-title="0">shzhao@mail.hzau.edu.cn</a>
<a href="mailto:nidebin@mail.hzau.edu.cn" class="" data-icon-position="" data-hide-link-title="0">nidebin@mail.hzau.edu.cn</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3617&amp;rft.epage=3625&amp;rft.atitle=Genomic%20Analysis%20To%20Identify%20Signatures%20of%20Artificial%20Selection%20and%20Loci%20Associated%20with%20Important%20Economic%20Traits%20in%20Duroc%20Pigs&amp;rft.volume=8&amp;rft.issue=11&amp;rft.date=2018-11-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Ma%2C+Yunlong&amp;rft.au=Zhang%2C+Saixian&amp;rft.au=Zhang%2C+Kaili&amp;rft.au=Fang%2C+Chengchi&amp;rft.au=Xie%2C+Shengsong&amp;rft.au=Du%2C+Xiaoyong&amp;rft.au=Li%2C+Xinyun&amp;rft.au=Ni%2C+Debin&amp;rft.au=Zhao%2C+Shuhong"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/491452/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/491452/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/491452/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/491452/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/491452/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/491452/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/491452/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/491452/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/491452/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/491452/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/491452/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/491452/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/8/11/3617" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Introduction</a></li><li class="odd"><a href="#sec-2" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Material and Methods</a></li><li class="even"><a href="#sec-12" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results and Discussion</a></li><li class="odd"><a href="#sec-17" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Conclusions</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/8/11/3617.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/8/11/3617.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/8/11/3617.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/8/11/3617.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
