<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/8/11/3715" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/8/11/3715.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/8/11/3715.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/8/11/3715.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/8/11/3715/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/8/11/3715.atom" />
<meta name="HW.pisa" content="ggg;8/11/3715" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Comparing Genome-Wide Association Study Results from Different Measurements of an Underlying Phenotype" />
<meta name="DC.Identifier" content="10.1534/g3.118.200700" />
<meta name="DC.Date" content="2018-11-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2018 Gage et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Relation" content="https://doi.org/10.25386/genetics.7097813" />
<meta name="DC.Description" content="Increasing popularity of high-throughput phenotyping technologies, such as image-based phenotyping, offer novel ways for quantifying plant growth and morphology. These new methods can be more or less accurate and precise than traditional, manual measurements. Many large-scale phenotyping efforts are conducted to enable genome-wide association studies (GWAS), but it is unclear exactly how alternative methods of phenotyping will affect GWAS results. In this study we simulate phenotypes that are controlled by the same set of causal loci but have differing heritability, similar to two different measurements of the same morphological character. We then perform GWAS with the simulated traits and create receiver operating characteristic (ROC) curves from the results. The areas under the ROC curves (AUCs) provide a metric that allows direct comparisons of GWAS results from different simulated traits. We use this framework to evaluate the effects of heritability and the number of causative loci on the AUCs of simulated traits; we also test the differences between AUCs of traits with differing heritability. We find that both increasing the number of causative loci and decreasing the heritability reduce a trait’s AUC. We also find that when two traits are controlled by a greater number of causative loci, they are more likely to have significantly different AUCs as the difference between their heritabilities increases. When simulation results are applied to measures of tassel morphology, we find no significant difference between AUCs from GWAS using manual and image-based measurements of typical maize tassel characters. This finding indicates that both measurement methods have similar ability to identify genetic associations. These results provide a framework for deciding between competing phenotyping strategies when the ultimate goal is to generate and use phenotype-genotype associations from GWAS." />
<meta name="DC.Contributor" content="Joseph L. Gage" />
<meta name="DC.Contributor" content="Natalia de Leon" />
<meta name="DC.Contributor" content="Murray K. Clayton" />
<meta name="article:published_time" content="2018-11-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Comparing Genome-Wide Association Study Results from Different Measurements of an Underlying Phenotype" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Increasing popularity of high-throughput phenotyping technologies, such as image-based phenotyping, offer novel ways for quantifying plant growth and morphology. These new methods can be more or less accurate and precise than traditional, manual measurements. Many large-scale phenotyping efforts are conducted to enable genome-wide association studies (GWAS), but it is unclear exactly how alternative methods of phenotyping will affect GWAS results. In this study we simulate phenotypes that are controlled by the same set of causal loci but have differing heritability, similar to two different measurements of the same morphological character. We then perform GWAS with the simulated traits and create receiver operating characteristic (ROC) curves from the results. The areas under the ROC curves (AUCs) provide a metric that allows direct comparisons of GWAS results from different simulated traits. We use this framework to evaluate the effects of heritability and the number of causative loci on the AUCs of simulated traits; we also test the differences between AUCs of traits with differing heritability. We find that both increasing the number of causative loci and decreasing the heritability reduce a trait’s AUC. We also find that when two traits are controlled by a greater number of causative loci, they are more likely to have significantly different AUCs as the difference between their heritabilities increases. When simulation results are applied to measures of tassel morphology, we find no significant difference between AUCs from GWAS using manual and image-based measurements of typical maize tassel characters. This finding indicates that both measurement methods have similar ability to identify genetic associations. These results provide a framework for deciding between competing phenotyping strategies when the ultimate goal is to generate and use phenotype-genotype associations from GWAS.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2018/11/01" />
<meta name="citation_mjid" content="ggg;8/11/3715" />
<meta name="citation_id" content="8/11/3715" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/8/11/3715" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/8/11/3715.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/8/11/3715.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/8/11/3715.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.118.200700" />
<meta name="citation_pmid" content="30262522" />
<meta name="citation_volume" content="8" />
<meta name="citation_issue" content="11" />
<meta name="citation_num_pages" content="8" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="3715" />
<meta name="citation_lastpage" content="3722" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Joseph L. Gage" />
<meta name="citation_author_institution" content="Department of Agronomy" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-5946-4414" />
<meta name="citation_author" content="Natalia de Leon" />
<meta name="citation_author_institution" content="Department of Agronomy" />
<meta name="citation_author_email" content="ndeleongatti@wisc.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-7867-9058" />
<meta name="citation_author" content="Murray K. Clayton" />
<meta name="citation_author_institution" content="Departments of Statistics and Plant Pathology, University of Wisconsin-Madison, Madison, WI 53706" />
<meta name="citation_reference" content="citation_journal_title=J. Math. Psychol.;citation_author=D. Bamber;citation_title=The area above the ordinal dominance graph and the area below the receiver operating characteristic graph.;citation_pages=387-415;citation_volume=12;citation_year=1975;citation_doi=10.1016/0022-2496(75)90001-2" />
<meta name="citation_reference" content="citation_journal_title=Crit. Rev. Plant Sci.;citation_author=C. H. Bock;citation_author=G. H. Poole;citation_author=P. E. Parker;citation_author=T. R. Gottwald;citation_title=PLANT DISEASE SEVERITY ESTIMATED VISUALLY, BY DIGITAL PHOTOGRAPHY AND IMAGE ANALYSIS, AND BY HYPERSPECTRAL IMAGING;citation_pages=59-107;citation_volume=29;citation_year=2010;citation_doi=10.1080/07352681003617285" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=E. A. Boyle;citation_author=Y. I. Li;citation_author=J. K. Pritchard;citation_title=An Expanded View of Complex Traits: From Polygenic to Omnigenic.;citation_pages=1177-1186;citation_volume=169;citation_year=2017;citation_pmid=28622505;citation_doi=10.1016/j.cell.2017.05.038" />
<meta name="citation_reference" content="citation_journal_title=Gigascience;citation_author=R. Bukowski;citation_author=X. Guo;citation_author=Y. Lu;citation_author=C. Zou;citation_author=B. He;citation_title=Construction of the third generation Zea mays haplotype map.;citation_pages=1-12;citation_volume=7;citation_year=2015;citation_doi=10.1093/gigascience/gix134" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=D. H. Chitwood;citation_author=A. Ranjan;citation_author=C. C. Martinez;citation_author=L. R. Headland;citation_author=T. Thiem;citation_title=A Modern Ampelography: A Genetic Basis for Leaf Shape and Venation Patterning in Grape.;citation_volume=164;citation_year=2014;citation_pmid=24285849;citation_doi=10.1104/pp.113.229708" />
<meta name="citation_reference" content="citation_journal_title=Biometrics;citation_journal_abbrev=Biometrics;citation_author=E. R. DeLong;citation_author=D. M. DeLong;citation_author=D. L. Clarke-Pearson;citation_title=Comparing the areas under two or more correlated receiver operating characteristic curves: a nonparametric approach.;citation_pages=837-845;citation_volume=44;citation_year=1988;citation_issue=3;citation_pmid=3203132;citation_doi=10.2307/2531595" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=T. L. Durham Brooks;citation_author=N. D. Miller;citation_author=E. P. Spalding;citation_title=Plasticity of Arabidopsis Root Gravitropism throughout a Multidimensional Condition Space Quantified by Automated Image Analysis.;citation_volume=152;citation_year=2010;citation_pmid=19923240;citation_doi=10.1104/pp.109.145292" />
<meta name="citation_reference" content="citation_journal_title=Trans. R. Soc. Edinb.;citation_author=R. A. Fisher;citation_title=The Correlation between Relatives on the Supposition of Mendelian Inheritance.;citation_pages=399-433;citation_volume=52;citation_year=1919;citation_doi=10.1017/S0080456800012163" />
<meta name="citation_reference" content="citation_journal_title=Plant Methods;citation_author=J. L. Gage;citation_author=N. D. Miller;citation_author=E. P. Spalding;citation_author=S. M. Kaeppler;citation_author=N. de Leon;citation_title=TIPS: a system for automated image-based phenotyping of maize tassels.;citation_pages=21;citation_volume=13;citation_year=2017;citation_doi=10.1186/s13007-017-0172-8" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. L. Gage;citation_author=M. R. White;citation_author=J. Edwards;citation_author=S. Kaeppler;citation_author=N. de Leon;citation_title=Selection signatures underlying dramatic male inflorescence transformation during modern hybrid maize breeding.;citation_volume=210;citation_year=2018;citation_pmid=30257936;citation_doi=10.1534/genetics.118.301487" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=B. Hayes;citation_author=M. E. Goddard;citation_title=The distribution of the effects of genes affecting quantitative traits in livestock.;citation_pages=209-229;citation_volume=33;citation_year=2001;citation_issue=3;citation_pmid=11403745;citation_doi=10.1051/gse:2001117" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=C. N. Hirsch;citation_author=J. M. Foerster;citation_author=J. M. Johnson;citation_author=R. S. Sekhon;citation_author=G. Muttoni;citation_title=Insights into the maize pan-genome and pan-transcriptome.;citation_volume=26;citation_year=2014;citation_pmid=24488960;citation_doi=10.1105/tpc.113.119982" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=H. Lango Allen;citation_author=K. Estrada;citation_author=G. Lettre;citation_author=S. I. Berndt;citation_author=M. N. Weedon;citation_title=Hundreds of variants clustered in genomic loci and biological pathways affect human height.;citation_pages=832-838;citation_volume=467;citation_year=2010;citation_issue=7317;citation_pmid=20881960;citation_doi=10.1038/nature09410" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Bot.;citation_author=M. Li;citation_author=K. Duncan;citation_author=C. N. Topp;citation_author=D. H. Chitwood;citation_title=Persistent homology and the branching topologies of plants.;citation_volume=104;citation_year=2017;citation_pmid=28341629;citation_doi=10.3732/ajb.1700046" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. E. Lipka;citation_author=F. Tian;citation_author=Q. Wang;citation_author=J. Peiffer;citation_author=M. Li;citation_title=GAPIT: Genome association and prediction integrated tool.;citation_volume=28;citation_year=2012;citation_pmid=22796960;citation_doi=10.1093/bioinformatics/bts444" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=X. Liu;citation_author=M. Huang;citation_author=B. Fan;citation_author=E. S. Buckler;citation_author=Z. Zhang;citation_title=Iterative Usage of Fixed and Random Effect Models for Powerful and Efficient Genome- Wide Association Studies.;citation_pages=e1005767;citation_volume=12;citation_year=2016;citation_doi=10.1371/journal.pgen.1005767" />
<meta name="citation_reference" content="citation_journal_title=Integr. Biol.;citation_author=Y. Men;citation_author=Q. Yu;citation_author=Z. Chen;citation_author=J. Wang;citation_author=Y. Huang;citation_title=A high-throughput imaging system to quantitatively analyze the growth dynamics of plant seedlings.;citation_pages=945;citation_volume=4;citation_year=2012;citation_doi=10.1039/c2ib20020a" />
<meta name="citation_reference" content="citation_title=Information processing in medical imaging.;citation_title=A new approach for testing the significance of differences between ROC curves measured from correlated data;citation_author=C. E. Metz;citation_author=P.-L. Wang;citation_author=H. B. Kronman;citation_pages=432-445;citation_year=1984" />
<meta name="citation_reference" content="citation_journal_title=Anim. Feed Sci. Technol.;citation_author=R. Park;citation_author=R. Agnew;citation_author=F. Gordon;citation_author=R. W. Steen;citation_title=The use of near infrared reflectance spectroscopy (NIRS) on undried samples of grass silage to predict chemical composition and digestibility parameters.;citation_pages=155-167;citation_volume=72;citation_year=1998;citation_doi=10.1016/S0377-8401(97)00175-2" />
<meta name="citation_reference" content="citation_year=2016" />
<meta name="citation_reference" content="citation_author=X. Robin;citation_author=N. Turck;citation_author=A. Hainard;citation_author=F. Lisacek;citation_author=J. Sanchez;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=P. Scheet;citation_author=M. Stephens;citation_title=A fast and flexible statistical model for large-scale population genotype data: applications to inferring missing genotypes and haplotypic phase.;citation_pages=629-644;citation_volume=78;citation_year=2006;citation_issue=4;citation_pmid=16532393;citation_doi=10.1086/502802" />
<meta name="citation_reference" content="citation_author=G. Upton;citation_author=I. Cook;citation_year=2008" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=P. M. VanRaden;citation_title=Efficient Methods to Compute Genomic Predictions;citation_pages=4414-4423;citation_volume=91;citation_year=2008;citation_issue=11;citation_pmid=18946147;citation_doi=10.3168/jds.2007-0980" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=P. M. Visscher;citation_title=Sizing up human height variation.;citation_pages=489-490;citation_volume=40;citation_year=2008;citation_issue=5;citation_pmid=18443579;citation_doi=10.1038/ng0508-489" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=Q. Wang;citation_author=F. Tian;citation_author=Y. Pan;citation_author=E. S. Buckler;citation_author=Z. Zhang;citation_title=A SUPER powerful method for genome wide association study.;citation_pages=e107684;citation_volume=9;citation_year=2014;citation_doi=10.1371/journal.pone.0107684" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Comparing Genome-Wide Association Study Results from Different Measurements of an Underlying Phenotype" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11.cover-source.jpg" />
<meta name="twitter:description" content="Increasing popularity of high-throughput phenotyping technologies, such as image-based phenotyping, offer novel ways for quantifying plant growth and morphology. These new methods can be more or less accurate and precise than traditional, manual measurements. Many large-scale phenotyping efforts are conducted to enable genome-wide association studies (GWAS), but it is unclear exactly how alternative methods of phenotyping will affect GWAS results. In this study we simulate phenotypes that are controlled by the same set of causal loci but have differing heritability, similar to two different measurements of the same morphological character. We then perform GWAS with the simulated traits and create receiver operating characteristic (ROC) curves from the results. The areas under the ROC curves (AUCs) provide a metric that allows direct comparisons of GWAS results from different simulated traits. We use this framework to evaluate the effects of heritability and the number of causative loci on the AUCs of simulated traits; we also test the differences between AUCs of traits with differing heritability. We find that both increasing the number of causative loci and decreasing the heritability reduce a trait’s AUC. We also find that when two traits are controlled by a greater number of causative loci, they are more likely to have significantly different AUCs as the difference between their heritabilities increases. When simulation results are applied to measures of tassel morphology, we find no significant difference between AUCs from GWAS using manual and image-based measurements of typical maize tassel characters. This finding indicates that both measurement methods have similar ability to identify genetic associations. These results provide a framework for deciding between competing phenotyping strategies when the ultimate goal is to generate and use phenotype-genotype associations from GWAS." />
<meta name="og-title" property="og:title" content="Comparing Genome-Wide Association Study Results from Different Measurements of an Underlying Phenotype" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/8/11/3715" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Increasing popularity of high-throughput phenotyping technologies, such as image-based phenotyping, offer novel ways for quantifying plant growth and morphology. These new methods can be more or less accurate and precise than traditional, manual measurements. Many large-scale phenotyping efforts are conducted to enable genome-wide association studies (GWAS), but it is unclear exactly how alternative methods of phenotyping will affect GWAS results. In this study we simulate phenotypes that are controlled by the same set of causal loci but have differing heritability, similar to two different measurements of the same morphological character. We then perform GWAS with the simulated traits and create receiver operating characteristic (ROC) curves from the results. The areas under the ROC curves (AUCs) provide a metric that allows direct comparisons of GWAS results from different simulated traits. We use this framework to evaluate the effects of heritability and the number of causative loci on the AUCs of simulated traits; we also test the differences between AUCs of traits with differing heritability. We find that both increasing the number of causative loci and decreasing the heritability reduce a trait’s AUC. We also find that when two traits are controlled by a greater number of causative loci, they are more likely to have significantly different AUCs as the difference between their heritabilities increases. When simulation results are applied to measures of tassel morphology, we find no significant difference between AUCs from GWAS using manual and image-based measurements of typical maize tassel characters. This finding indicates that both measurement methods have similar ability to identify genetic associations. These results provide a framework for deciding between competing phenotyping strategies when the ultimate goal is to generate and use phenotype-genotype associations from GWAS." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11.cover-source.jpg" />
    <title>Comparing Genome-Wide Association Study Results from Different Measurements of an Underlying Phenotype | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bed9eK4HgZgeq1SWIHZ3nUSsO_Lg6SY0VT6zRKv2uw4__AOTOtpTA1sp4jw4ldY-8h8LueAv-V2MjVmPlkIZ--AU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"6OSe8jD0HFzSJNWSfmSG7HuT024h8QIL4e1-yvIK9uk","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"491242","apath":"\/ggg\/8\/11\/3715.atom","pisa":"ggg;8\/11\/3715","ac":{"\/ggg\/8\/11\/3715.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/8\/11\/3715.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;8\/11\/3715"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;8\/11\/3715"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/8\/11\/3715.figures-only":true,"\/content\/8\/11\/3715":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022wKX8HK3FmchHGKcN6xWtZZ40bzWKocJfeMirLpFPaPrR\\\\\\\/Ie9XMqHX+ZJoyB9fElMcslWZ5biwwsXJmd1gQ+Qy8XhZIi5ZifIobV63UMCI8D1K\\\\\\\/voUN794C\\\\\\\/6QPSbWlLU0lS\\\\\\\/0NfiyzkMhqnYtt6ioYDGcKJHia+PumrPsC733SsP\\\\\\\/\\\\\\\/k4RYswMzu1DWTWtCN53Q5BaNmtXgCE5XKtT7VC4K9rUtJUboldeLAUWRKE0k1sOIWwAfAn+0Lp5hTGsZ5ZXOgujNFKWTi191LzUX6KwbAq72Po9cX2mJX09SxawG64ZreKEOGTkWZf\\\\\\\/GkIt+PQUv7JaIAGUpXZ1iSbRAXZiCLmVtLuyG5CLwQ73svykeor5ZqzbDC0pbO72q4dIBrkYqCblDftZAd8tHthn7J0UWEPT7P02odVrJ2t0oKz60\\\\\\\/n0+\\\\\\\/N1O0HyBNfrUCVBGsiCIBkt9vWy2t3ZfMvU4rJvwbVCPrntsP\\\\\\\/3RVTHlj\\\\\\\/9CdqJznlhnF9rS+RgtX0\\\\\\\/861Y2\\\\\\\/emoyr8xRHVhx4DREf12r0cuL+T5MFp6316WCDofPpEV2yfAKcwKc5g9vMiY9vckVOiOf\\\\\\\/vVE5dLgwEfazwB6sFuw+gqt3\\\\\\\/xs0gc0\\\\\\\/Bnknt7ZNzPSCJ4Qtt\\\\\\\/4Yi\\\\\\\/qkz8MmmXnAqIZCPNQPzAY4uksbgvwC8\\\\\\\/9jdlKV8AUYaK07s\\\\\\\/rHP9uNS2+CzbklGvuu4Wl3Pr6ipv8J\\\\\\\/Vi69qLgaXPfSY4i2d8e+IEBi6mSGRO937e2BWB5eR7u8zu\\\\\\\/1O24lL+GUwSk4hfRukeg6uanSMAPi2xeufWtc6vp1DPQXM7mRJlDhkuTqZqSp52gTTEMNpKfa07q8bsmiBsWJZpUCeCSyUxBCF6KHRphNciXNTFIi+xcd8RgaH3yKKbGbyy9faLDnqloGyCd\\\\\\\/lTvA1UXnl\\\\\\\/thiqzCWdMRc43+GMOfPomiKq8PlIWmvdpFsXJbLyk1UiRp6QXdQlystFh7PBOpRJagirOyA7lnPsFU9LhTmbwUHf7dbnRQomIFiGzQNXcjjVL9Nz+2ldt7GRx4v7tssJV5zv7\\\\\\\/h3z71GDmtCoKjiWxbU\\\\\\\/2uxvW9QA5aZHOK91wkg3uQy0hnNtazoJrX7+aarnL8mawxo1lDXvwNmPXSNDbcERrMCV4\\\\\\\/zOZS4Oj9qssUIOsYZ7AuA8cGu7QnIYKYkqJw5YmovUZTmu3u\\\\\\\/\\\\\\\/npgII80helW0a78iygH1q8unYrtb\\\\\\\/yC3L6HSQ7mIW5l8+Q8Q0zQEZqwTrOad6nnDuHQfuMNKABFsOSP6GJXe+Fs2\\\u0022,\\\u0022iv\\\u0022:\\\u00223uTN5EiZzImyMkYU1L9LEg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00226feb6b59805a5549be067d6ea0cd8603e0635d506ca282a5e13a36e4d6e8ec8d\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022G1SRhBoZAjXiqSmkuuxQvx7IwtB7nnm8OT+BsEVrNnqbA3AxmvENKum\\\\\\\/1eju13BwiJNQ7GpUvsv2Y4RI0j9Dl28pWQ4611U830Y045gBJj95T9eU1xPZNOgJm33aqKOMitfIQSrurbHowWTmIim3QXC83jDEiyVr9veiP\\\\\\\/pc9zjJFPQbmygeuMwLpv9RKIeiYBHMR78xJfGPHj0DO7hh2lkpgrjTDkdZgwrZO1EU\\\\\\\/S1EGD0APJW5gcZfnBP4ZtkmO6nHLmwGqT71Su2sDUHd6xHOtcNrb0UUaBSLAXhpL1JtQfWjqBUTU6WeuVwKJ552UAoimv8q4ldgD\\\\\\\/lAL0LUDDUDJn9AibS7BFYfIrLf2GnZqhP7hQ8X5d\\\\\\\/sLWnn8u3WEAevf1snhj1MnADElrZ9d6xodPX5HSB9sTqeVauFX\\\\\\\/mRMApjUGKjGALMvY0YUR4KRP14dTA0KRUT1\\\\\\\/H9hel+EuILR+azNmG2FV9o95S5mGoiG5ITZCyM9yzZuFfrZS3TazqP6E8Li7ahb5Dd390k1F83scYdiD5Uc8RbhdpKG5tBv4b8Wf6uqWxMIIxJIIIhYfLRIxs9DcicKzK7X838YuK8cQWT3RPiK9PC2Fc2niUcavakkqqAPMn6ueOiALzUou8Ph\\\\\\\/54pfCZ+8+ryLfLBfMsgFdSfUIt+Yom7di+OZFbH44qZR+XSleLu2lCgr9DwMTA0tJ3lAd+p\\\\\\\/LnL+\\\\\\\/XXk3wTK9IyxgmknfUwHx9\\\\\\\/tzcSQxyZwyic2cgyMMu3OoGDVT0cSdktHqPIm3IjrJcX5eXPCRNPAeATwUo6iIwkwNWSTv+1L6BYQtkCg704x9qHSmH1jW6hWn4Ga6FQC99gIagi6bM6fNjzXyVYBDHNbRCnMYZbNZbT0Gpl+tdxbcZfjk+7ylaZ4Jr9sLjf7NDLW2HjP32XvhdKOCRuGr0gXdajz60cTs9x++VoJ3CagvaWOY+Zz+zOb9cOwdH7urfKrcGdGGDJ0KI7Hyn2Ip\\\\\\\/wTYs\\\\\\\/+GQCGyNGatSoyaH8WWXbK+FVl5wuygFzYv7ReFNFOSlr4bOLI6e6Z3COO2FzrLPvRqCBfI0gc+043oLwj2qWiiI2yChaOCYooJEDEjMUfEMCrlVK9zanzp6ZvZ8QGOkFTfdEIq5ndQe+08In9fu9vdZGFkBLlmxCrFgxoUPabfA24LqFWvY39Pb6n39lUkCxJoxSPrzf0PAhqn2gDkD4W2HY9zz0jTreT2jfmNkLqUISV\\\\\\\/HkWKFGC1zrjwQT2vbnDd0v9mnCW88CLx6pm0cqhiTJiULISPIimmKgQnn2TTFi2oiGxj9gKUztAsMEHwlZYx+sTJK6EmfX5ATqNHE\\\\\\\/I3TneC4GySyTmp1ER9vlw5YaueBK5pKPOQPDlIzTt5BHJTDy9stuml1HkxQBgRnqJT31svIhbr3YLLG7O4pjOWfrrTM8WrZXrhW7kW8ij7gFfF1gpcT46I0FpPUE05jHNdchgOxm3GfL\\\\\\\/JyGiyBACIttHx5SXtZiHwe+j7ejrM41mTimmZKmaH3cEc44uCJmFpMZtAT0P7am01wav4zv\\\\\\\/zklXaqTtinRmFRozgAo5+QbJ0DDYabBILrS4rC2S2cZuvQ+WBeBMM0m7tsGoK12pjC8OoA+y9i8tv9TBe68isgUX+iKs8kadPyTNaOBPWZQA8Ayx81Dv2O2Y2r+OAMNsQL0veJQxfAXNxxZLJBQK7wPWEKFfwaF353mlsxJMgIsOUQddNxELxF6kcmjDAKH3FVMUzf8LgLgSislu\\\\\\\/+VeR3IsEeNumAQUQD2E\\\\\\\/vMZ6h7BhFRDSd+kDWcBjPhkW8EGiW1Fl\\\\\\\/5ot7njD2Jq9wlh37rKlPWpwnsJ6S9Z8gWrhDnsDkXalep1Vvl+UqI7oNpNYmwwgis1\\\\\\\/ctjRlGmazycMqFhVTicWFvMLL+W\\\\\\\/S2Y2UCgm5I2lXLZOVzmLGUfJRprj55D\\\\\\\/NEqRtq3acuiG7hJzKaxd2IEUBlIfRcGc\\\\\\\/c9rWZmcv\\\\\\\/Swhq1dgJ28+m+MdEcd1b4WqYhOTe\\\\\\\/cNRb9qbvdncHAiRbC90OPAFoLgzrnuqDIiX\\\\\\\/TiXVqP\\\\\\\/bcczlloqlANl86tFjwM+GOgDeNUahSVF+MSXaFRYm5DO9WbKnVzjXqm4yh5O0aj2btZfs2r75fXcIkV+Oa0qjoCcAbWm16h\\\\\\\/h\\\\\\\/J4QrElBkSDVN16AYLakvDFNlewftCFdedNLjWDzOqGTC\\\\\\\/Qg+S1ac9iCXV4fB2TvG2jROnQ9cKxEwqHa9z87Ca5+sBYYim0ftS1wl6PLUUwGLCj5vzVP9U9ueWJ8zW7iZS9Dm5OdvUb2qPRoHkNSxbIDvhzUkALqPGrO9zRFeVlQyHNUYxfrW\\\\\\\/3X3I6a4LRbuZRj1\\\\\\\/l6lavt43Js89zu6LUbe8bhucp22xoO0ZuyyNrRTFlaKKwr\\\\\\\/akfuWZqyGh++mZtBeKS6c2+sajHFJDiXJu+M\\\\\\\/ebxcBj5vi4AmotPqkxI2pJaTGwHN7mpRGrqSuhfD+cA\\\\\\\/7H5zccNOrYZdpgAh\\\\\\\/9RA\\\\\\\/tRzr3LboCmxiz5VESP1WDIEQc0VPzAIvEBgMadv2FJ1osM9hvvFuWaNzfS3CPwbDBq+OxuT1YrV04H0ahtenKRK9UL9I\\\\\\\/JzwxRfyiboDCBxlsbzPAPXY511dADKBjMMNzQ2bP3V3x+eqZTXYpQHiAxt1ICF4\\\\\\\/0eK6vgal4qH2L8rxgvcHkFylujHKmQK33gAJ98IbyhmP\\\\\\\/XvmNdOgsBRisP3K2JgPnZwO9\\\\\\\/JA+q0C0pqpCFAn3hOIks8Ot05tmnLEZCcSyhoe1bz\\\\\\\/ShveJJtHX1DW3RNcKjJF7wpoUci0WIoPOcvfOwXm4Ehg5vItt1j3mYmX431ziU9h94S+GFLjP1119RlZF5uY0KXo7ZncSqzZ8PZ6YIamao7JJCpA6ZT2dxW63kMG3ys6SgTenw9ERI0gxpTNwWuaUXx2GQ5tm6W3F2LBokegKcj45oltvUFFKDOvE+rToEn7FoYYtwYoXxSZu\\\\\\\/6GYKrjQ2Sn+umA9dTdihZn6Ul\\\\\\\/Zaq7YGk+p5j0uduX7hOM8TQP0zYEe2PzEXOBN2nqbzEu9lCy98JWaAI+j+MDNdmYy1YWt5MKQP3ZZF4cJyJAer58WUk4Oz1oKjNQfZg9hRw\\\\\\\/BOtyp310UzsdCaO3BogJNapGky1MV9h21hGuFE3gZrSaz3YgF\\\\\\\/dbAQQScayuh28+izRmj9v5GZSL6ivk\\\\\\\/AdFpthJjDD6P310US1+8na4DMQbjEYO83kfjpcXOmlyJkd1O1ZYHCIVRBW4zag+3+KKZppWYNaNOisElKeJDIJ7Ktg5uR15fwymFaB5xG+UTNtTCiMxx2plKeCNITntd145HVZoq2wrWzPuFUbrMP\\\\\\\/md4QjCUBsTxyoX9pmwx1f5HAjVIrmeDwEnKkBv+0tbLdnb9ejwfPq01R424y7hHY5XXBmoYaSebVSTFobYkqyIDvuWwSLOdDLOM9K8MZdpfNFOpcmH8tlyLWNKaabi+XtaWFc3tTqlbvSLTwfujYEHTU7fl7GPLhJ9KssIXmD5rdm6YVD02MVlOjxtN0FVRggwxswQMI03SQnqcuTvy9A6GoX77O4BnQj\\\\\\\/dEnEaAyNvS9+3u3NWO\\\\\\\/Hlsq28cux\\\\\\\/I7DXtXzV7vqDmgPQEtcZggZhA2IjFj9HOfoPhqnFEUiwaXJDY0lYp4O92NlkwQP5Coc9KhgC+y3vQmAyH1eXGNCTpY7JSm6soehFstbLMx46KqL5WCr56Jv9dU\\\\\\\/Wmfaa6jrs7LStBtm2eywS1OFEXus9Oa6P\\\\\\\/JHyy1x0mpbp1dDPIjSn62E4jOrGnsP3DKu9MFvlOL4BwZbgPumMSSfK3eK0SS8oyiskxGdCPHnyv\\\\\\\/EvwhfB8TYh6InBEqhV7WJt0iws9V\\\\\\\/KADYP1Et4O8VwLI1CrgnoXGZTyQmX1KESzHVb8I5NAvbkQVIRLXUIaEQA3XONuPYb7EyV0wK2B\\\\\\\/FH\\\\\\\/RhAp6vqQnGQoMW\\\\\\\/V\\\\\\\/IphVFfGT5srqaAJD\\\\\\\/nZFV3ZYz0BhaciQDhal5gaLSGfMbxyPb+THf7hjmL+oP1Jxmk1POXuos22iFjLDiCpkTP1xmi0M5Rg7Nqv81FVnuBbr1oW6DjtWd4dlgD4J3qMif2F4MQUkOfYk+g0MWpVcPbx\\\\\\\/Yk7ORU3WkR7WUN10lB9YaW7A7uCJIBgjr57q\\\\\\\/VLGQnEJSF\\\\\\\/kD3r+EztKUj9LD2rptdygUwW0ah\\\\\\\/4ax0D7tefUQPjqkcg6ZGTUaZhGla2PQbQ0NC754zccw1LDyzRHa7mAiGHABW0pXR7bnmAoRCNzSknLV5qsFzBQa4hesXQKeQKYsz874Tg\\\\\\\/OXXIM7GUo7CaeGEsxzBUN+tnG7EZkLM0h+F7lhLzRLll96j38OU2seQwuX0m6IQMFC\\\\\\\/xA9kS6JLbVfSYNLYxsvNh3dAiFi5BVX6667IK+3NHahBJloNBt29uf5wNs\\\\\\\/mDsBDVSJBIwkgjemodI+Pa3nEoIRYzweCp8o7SfmmPwZ5fSl04YGUigw0aXN8osdI8mD5bfnAX3\\\\\\\/H1VO1G8MhRQwSSr+phIqyu\\\\\\\/nK\\\\\\\/1K+TZFddk9GRQb3SsSClV306Ny1QyE5zr3zNACI+4gC\\\\\\\/fLyKWp9pauUlLtELY89pcm7uXvHIBWukwIZbs4gstdMn0tiC0j3qmv8ZRT7gJq\\\\\\\/bzKuvnedOCPnNEm4dUil7wss6t\\\\\\\/YBGGzYOvD31+u3AsWjC4oN0JCPiZFvoyieya+61Ta\\\\\\\/bKtb\\\\\\\/MVqa\\\\\\\/ruRL6sWV\\\\\\\/35L0q\\\\\\\/os9MyoP97zjfjM9SE0y4JOZrMxojmT9Ldb2lo92gfwBWXthMpP22DgIJzVbQIFmW13ehdD+28ecEU+P57+XS+hSJ+D+Rytn4wF59Q57OwcYkVLhziaowvzciMy3RjZuRHsTxbhT80sV3BYB7w1N8gtHYf9+O5MHWOXgf4NNwGjMndweGF9dqtHFacZe0NzJ5n+qRUyt7aZL2K0BPoeCc5bNszgistOTxalXgDa+aPlve+1MLFwKpjwAd497I7izIrCCYqn4SuiosBavi8\\\\\\\/+NVwDMe+bJRYHOe\\\\\\\/ljOzOeu9MmaH4XvmoIwOIIrjRbx\\\\\\\/fl7CnyXE4bdS2PX2TUB7uEj4EOqEkqUKnlDCFZRo9rIYlTecz2Urq69RQ6kfkI0wjOkxNtqcbzzXiZEDiFNcbp036Ri9uQGIGHNEZ060gbsPOYYYJ68JfkHkfwI\\\\\\\/kZy6BZz6ReLVznIWCXQ30\\\\\\\/txF2+LQESq1aDzUUrAvVxtHfK4rFhgLXnYXtLqhqytzr2M5oancLej3Fzsu5PepVPXfVpAZKs2A+HxE8L75Zuskes8unaQBx4v1Mw5crFMSopprweZx\\\\\\\/\\\\\\\/9\\\\\\\/pStH6S0xS2AzcXNM9sdoCyxliUdh5UPKgLWbCun51D8DS7Z916bjvlWx2GPQYRO9q3wSWyXLRDt2hwHpP2OKQpjptKb8hJrwKmQ5xhmaZX8ZU1tmuhToZ8OTKnND54MFQncdR1KpKfeEoElnIxWJ4iPyqgQHS2kvAtjRs27UM2+h7gzvXL7RAp9eKdj9WE6rrkrv7AEB6RTBLHYd+DhjyA55hFXyMQFF0ynbzUkv3DI9qS58Lt94JeTGnFQlKKxvbX8mom\\\\\\\/dySpESRJiOQrnDJ2rr9XQzf5k7rVwVKHo3DM6rMOzXpUWuAUSkX+P7GWvqmBaxzux8nB42z85WyFQCYUgC\\\\\\\/fy1DmlULESkE6G\\\\\\\/a6JZsJTpEVa8zTaavWw5oqBnGtIhcVqFOYDGbLnWWvuJYG+LOx9+\\\\\\\/s0VY26FfG\\\\\\\/ZJsMQINbPHGn4mvtUtcbH62RyU+u9RXRdldg4EBjwN9uIxJDVUuMIIS4lLubObac1DOgvLLn5qGY8iXB3nt6LMMnOjNiVZuNE7xIIuzDkG4c24hKIuibyhxn1oxfkskmK6b51Ey9TfjNRBDyxHMiW1cVVT1LDOsFwouI2rJjJlRxo1UUgcS0niZsheUUylfhxZ2Q18xyAVDO5VI4b96xNixp\\\\\\\/uWfDhgs81Wfnrg1oWgRgalNintDvPqNyMLZiM1Zw31fMjWsxdoFcOWTkTwuPz63zHvVG7SZzqHq68Uv6\\\\\\\/aoD+fLhZTDFhMoXC95I9PEgDRD3PmHGm+tciazZAWDAn5cj64rrhxuMp2v2cNXy8HK90uu8AWOqh5z2fYwCDEKuG20UBkOpFWIRbHYUa2qYWVZwH00Z3ld3Btb0GZ61UjXqrBPa2zYsottOZAkTMC8gKSLqaEjfM1RhK8Kg2P8jopY9uBMFVeq4vGKElXN6IX9opAhoSZLoMrAD7FsjvoWWNnM3eQW8htJNiyl5vwWDEFJLlfkkMeWcPxMFr4TV1lSCttXvHWoxaoS8rkdQ2eJfxu9v3DzOT6H\\\\\\\/BivGXwPgaabS7yLMmfx4tvjnkViAejBkW\\\\\\\/6rFI6XQqzOzfxQEM1l1hiq7chLXD0zHr2wSMgJV7d70eCPHPIfZAoUel8Hx4uKmTYN4xIitDDCliqV+5lQXg9enXvMpl5fZamfdLvsiKeleapSUY7LSCMzWhr\\\\\\\/rOA0fGgO004ODRbks18Yb+Quy0bXsne3AizHZRlrUu5dLb9AjJQ4vFTYIfmFJ438xEUqKSJVV0+oi0430eEuLF75IfRQm4+JA++ZNjosT3sl6k\\\\\\\/u7Vb3c9yWJYf3QnMgnu+q2xq4L3Zi3klUtO\\\\\\\/XDD8wy0VLPMNYLJNpMPJgaVsjabnajnXKyYRmviDnpOPUlHx6kK56NxZpD2Is4b+k0ngPK1PKErBytjzLEmIKFvncuOBz5ulzmpNzwlc7JLM0EMJbJNIT4zgUCLdNlJC1fPSqyXTImNzNWRVazEVYX8h8bMOHHFD0zWFQWZB4jPBXmjxmK9yBOVei+V9XdrCb0xN3RKtVfGW5jt6GcmH1czCXVYJ3uwvXjOPwo4QOhLjMKxnmWHgzvNS1GgyX5pa5JxBC0+dvdQzW6S1b+Ure7IG9Q2f8Q+EYxxhUiIt\\\\\\\/x0ajL7OReQroe3PVEMqklqtOn5kBq6KR0aEXVn4JusIYB9FgothUiqXQBxc6joWz7Cm+rIzUfV11Yu8Ld3OD\\\\\\\/vCdfG7zHP3TqUn1OFedu3xfTkNg43dNNlxi5iF01ZSWXy18H1\\\\\\\/RA4dQ8zQxQnjIOVUMlGTBVdgkWD\\\\\\\/yFqBfl\\\\\\\/VnZY7pt9WsH922iC3XAVEukvI05wxpO8DDM72i0j4Sc7L95\\\\\\\/3VpjetodBj\\\\\\\/ZKCoZ7fW\\\\\\\/Z+G1SOaKEiJff5tmZQNJG+rIkwEfJ3OjGiqYwAzGr8XitLh8YZjBXH3RfIqjmTOheBVuBBBoWAh430VvVQtqbJslN1Met1YmCJKuqFdVDRGfBsNAMRz5LEqNivKeAnB78yMSdxJPtiqwpW5PpYaUl\\\\\\\/WD0cFcVTTOXU1TVxml69BESqmuVnKGqNAgO1OcwoBVwXY+\\\\\\\/5VeLm44CqfqFe\\\\\\\/zsRgtkqp1d8ehbVrG0AmRMMkYfgdPZsrw6K5kIZ434M2SsGCNRWhOmlTpSMQLnEq3PlOkPauTXv0EDYFSYFrCWrEcoSPIXcb410NSpaWgTrtCxfdFlUZVNmQ13ver2F5iI2efML\\\\\\\/iyNiKDoGmamFBb9lTaILJ7iHmUNdGPQf7vMH0aAMaGxuF+65Dso+0eqoD9ZuwCsr81eHWzSitaImpsM0sf9vrl17vc+RXOW+4tVNl\\\\\\\/7daRO5olQOIi3rHCClAYIAkN6oVGtGsnGPDSSD3cWPMwQRJSpH3xi2v3gTvJCzryOGxJ58MZFeU0k6CLQ\\\\\\\/Hhgu9SjxxSZSDaIiVX8RslZqg2ITM3iEpjU0NpQgGTAnoA9Bvw35AliuJouecSJqipEz84CpvQ54EiicOF2EGC7oPt3BOxvtPM9whD+8JHKdTukkrnzImbxql\\\\\\\/9iPblLVOpy6bAcfmrz1EkKYtHucrXLmJloil17gb4BBInQZWge3YIZU0Hf3V4iRzZA9cJi1GGEMHdcpL70PUoqLoM1GlJTrDaGnyztxtPlbtTd5sqCNA7rucIqUuPYKyTRIn62t9C7myZEWYMBUnFe3cbcxHFRaRU1aWOOBnTiIl4DjUSnxHqAokujQudf8EsCTF5fZqW2fCHGpY\\\\\\\/r\\\\\\\/GR2A2R3IjHlDnAGI4VG4Yem6QrfbkG4uYKTnqug794kTNnXKgrw3e5y4RpeeLTIGluOksz8r1PTvdSxr2Geu2F+lUk619ftEtjkNIrKE3HcsxmkOf\\\\\\\/ms9xtXO+1mgX\\\\\\\/Ru01up3YQJdxgqh1ygNHeTBFVntimL7sPG2AzUR9rTztGaUlIliwA7D1d9D8drYEp\\\\\\\/82js+e38YN3DKCJIEPGZe4TPNWTLfRgGoioC2GPXKLx83NU4hJUk9Yo1NifAbd1fAqwO+STlZT1wnrM4\\\\\\\/F1FYbMRP+LnFnelMYJAmhqOm48lKMB5x5SKpz5PidF+QzCAcWLN5swOu7RkOB+sKWHyx0IS24TtS4V7elqFmqb1hPg6mpq8kgWSRwQEY+EO8e05BKJCsC3Wtembs0rFm2HdUGXjb\\\\\\\/P7X4SfNYvOZw1Ex9dfmf+pF7ieHRk1S1IS5xLi4QumvZe6F5e3drDsIu+FpEcibg+1u8G16icMfmvBsHocdPn3xEX0N2NhcfijEliJyXlhM41ObM4N4MZLgIvzvJWnpRXIFtaH1lLgeq80tnwYCDtNGVShc\\\\\\\/92fpuY1Oyk\\\\\\\/O\\\\\\\/yFKIJeFea9g3+WqdoeKWMz8x+sooL66Tqwy0Ca9W416iw0l13toHphQ8tMoVbTvmhE7jghBO8gHuVXkp+hBaQLt1UP+AQYFrht9RQUFsvaaUGMoe+9WubHDG61Z7\\\\\\\/BLpQ7ayH8vPrr2Y7NDqV93S+urlAR3pw6sHrGvE0xbGQwERVPNUMoJlH3EiDuxrg+ldISRQOVKRTn4r6Y\\\\\\\/rs1S5eHChBxGlZvZy5o5UA1iz+U8c8mzV8GtsTLbihjFt7\\\\\\\/PgUELKh89UfhufOq9U8+NS0aqWLA7rkYm2G7pCNxAryaQFRBMKceE16ZvzB1KlF0q6WuqKla96mbn3z3J0kOOgT2nh+oKcCxVC3c4DwnMqMkPhkL9J\\\\\\\/oOZRAYbNM82Y\\\\\\\/2XFTEnhDIsE9x\\\\\\\/R0gl4v8nb6pSx3DqQO9GUvguL7tUGBIFmqq3sEAPYrJSR20uAJvjDuGYs2yGYggup\\\\\\\/+8VB3JZf25BKY8slymsEAV7hxBKZXCN5Obcj99d+lp4FR8\\\\\\\/IX1iFfaVy3zgCab\\\\\\\/cyIfvhmW4wfV4Ja984okOa8\\\\\\\/v2CcqenpCuu9WAczkukBGLp\\\\\\\/6hj\\\\\\\/q4\\\\\\\/5xR3912XDkJm1N40uRsgK3Oq+a3hh+R6HDqRK6TJtlWcK10Ysc2VgJlMDLsU0EbKhEwg6ys+XqqZBdwzt3jYX0PcwEI0NGzk4B2id+XO5+CYO15qiQrZy2w04yFzGx763etTH9Xs18Hht5xoukA9Cszh7UHNDWWCUaOR2MiiGHIajY8g6KpRRWGeID0dSys6m4VPWGZAw89cnX5AJWRLNjp9qyylHtgltDnCiD7D2BaigsYaWCwQ+ttX+XgDH18uDrLcKIOEkNM5iMQ3IHPp+t4R65rJg7pHJ4Vj+oGKdsQoB9vJdswp8m9PjcQ1VqG8ZBiKqx2RFUUJ0DVFTm6+RW9991udfigo5udku4o4Sm5srv2ZJs\\\\\\\/YLe76BU9weuj8r3i20pEdsMQhtVpCrbAE2W1eX5hoTVLYwoafrTwhWS+4Wvl8nIzHrXMbdVfsKchCd34i+GrL2mdenS+M3q2GHwT37wQnhr\\\\\\\/JfrB\\\\\\\/ee5oSgfrK9uysuiw8GmRL3fA3fSyoUZpq1UsamLZnDkl90TfTF6uPv5eKMCM2BAwq5H1Emx+mQZsrizfewCURUznJV78fd3PwzuB61ZtFvg0vVwOJ+cBKYSKZXJaZDvMizUpaLTwX27yiBFImBuV6SJlxTHtVNXUqIFC4+L\\\\\\\/b1UFvtO8BNblGlLl+o4BVyFGuFzxJkku1S0gDDGuOD4RaVaWF29QadI9RYGl\\\\\\\/q7ho3hnz1xXrVH8qBVa6IJN1x0PaXvQDa88ZJKfyZXQ7HAADTMGu4A5Upd6uCcs5b9bNgSgEAvcUKG9YycqHXOo8CavtI3qH49n4+i8MR\\\\\\\/N9SHRedkGJKCpVem9Tnkm8q8IVX1WOzhkULpXpfRcg5FpA0QKxMA5zwhQNTnV9jqVe1GWj+q9QrF94drMCauF9jx8yyQ6hzvZImeiSQLPGH7XC74a7unmqGqdS7T3fofy6muQPznEC\\\\\\\/vyfP4wkXOrlDwQR\\\\\\\/3XSPU5RZ896F+roSxYlbBfaH+WtohNEkKXRXajLjtZMusFHbXgl7MfO4\\\\\\\/gDE\\\\\\\/eL+9ysnCSdfkhDFSDEXV7z5Lgp6H2olYVjkLAAW\\\\\\\/faJ6C15c7sdlWVRLxYS\\\\\\\/Vkun52Lfi5uhXc5zFwHBb5AwGlVvqcRk+Kltb203toUS3zVwuCFEN+jaNW+uGH9yKra7DNQT30rhozYbH6umWhyO7IAIT09rzx2riF0PLVXZyKgXl\\\\\\\/6+6wbUE1wR12uaWOCwA0G2pLrOcwSRF2U9eu9eUtxyme+79Zcp7Wb8WRuAzQ5mRmYiW8K4bQoNLzdMSmgOtJp11LBCo3R2GlaphWaQuXaQesC+8R+LFsITnrVflGoY7qht5f3IZZL4jU+Gjg6GD90MsUgYIK21H5X8u+kz92zbZrPHMnOJ22vjXYsvXpxe2V2dSNsEFMgrWsnkPqtIlO2bAYTeIbs+Ax0KsgLcZNnjrT9fqsMy3AeUn1RN2iYGicU02KIVvhDFMMzCd6KUJJJOVXpA9GWRZjKKRUINWToNN\\\\\\\/O7rTaU4GOjPnx85A\\\\\\\/F4mAZP+XMF1Kjt8\\\\\\\/WKWHshSTIQ7aICntzAbACyRMpnGWTIaWsW3f5ixyxgTssFBBCTeMAZ1Fi2p+zObgiAIv1Mpfsgks9+wFPX4qEX5ZgDEW87syZGBVJ98jTSeEA4lnUxGOVx1\\\\\\\/QgITdWYPo7s1oQ+GnV8wk01B7CDHw0G0ud3QTlefoQ20BfGKdDXW0f62pd4RkCgiLNjQ6jvPFSfUipEoW+ElCzKf6NeLoh51bZFW1JoohBaT6gLZqN3C6GoLg8+noOlibF+v6ss+Emtivre1ttEWqQe+PsTkClqKzK6L8JXJCDP\\\\\\\/sC0lsC2u9LYKoOD8UfoQMFuZ\\\\\\\/fFS2ATR4UjFseISIxvWlxz5kxIjSWytHYFLzW6vPMOolkMyy0bzApdYJprn3yVrCTbqOseJnfFzg128Yh\\\\\\\/2Xc4\\\\\\\/ofxPYXmpwjt3BdjgoZQSrDeD73zrjKZayTGAQdTq1rd98CnDYxfAlMeE02v1mntRxoCACpjk\\\\\\\/zUFzu8o\\\\\\\/ZUEMI+GNoP4XRJNBAWLaOlqVb15rTP9bI0dePAnjhA3e4l3FNw4p4CIXp94zxZaMsd3Ti7vt0fdAfldhYQXQcezBqyUkZPljNFX\\\\\\\/Xl8LKuGV4fgz2us629qJ\\\\\\\/ctc3MfCl4WN57kMaKmO1hdc2dlAofISkvwKdgviV3ndsiqjNBICmaxYDzdnQkrEFNapbCkbllMCPmbmqhgFTlxNy6L9HhiehAhUQeYj9Pef4W8JsaXCcpK8Dd3jpm9GsbHifJf1Gh72+yFIgTldKu5d4BnmTwTIS7nwCVPP80o8c9X9mhQKiLeHI5QLdR+jbNYRh4IkbSCoizIb6lU9RDJGpBxvhu2te\\\\\\\/90j5ETcbWxcODxURdS1EoQ18\\\\\\\/jmp09+bZ3b2PpAS6lXFK0BR3N56i7heuyFu6JPgiQOxicd35iPA4hNAj2zXWD1BYBJYfuAe7oUAbRgK2v1hrlNjgw3Jx78xvysj9sPoexuNuZ6dhlZBjc0HlC2yl6SU8cvkN2oEnEuNGQABWtGrFyhf7oemnHqIRSe9cl0w5SwJK7J4BbZbdi2bX64+pRrTRInMYio\\\\\\\/uvJ5X8BnUDm3HCKOV\\\\\\\/fPy\\\\\\\/B546p+qP0PMnFtdjhXzcZPf+47EdJ5fvP9JdXuFrj9ECuETgrIO6Ao2\\\\\\\/DU3TpMkMIToFCa0CwSdn8b5faMPvIJpRUjiSPhqfuQ5hClsb5E7FkuYIgwy55F6q\\\\\\\/aXdlfFdfPNxv55bBG1mtcBSufvYfWV7BgzqXVgME+sZ39dZAXROXXiWl2aEGkjD6rQa1bPJkTRsZlyUkQQga1xbKUSuly7Ui+4JXRqZt0GxThNKN7E2NX2N1jNPSgIrUeHSi9ouXnmyT8v0NjSSV77LN5qGJIahyditdq4sP8RmkKIfWg4HDb2+RAtJ1UHhTYPrSna0I0Bw4fQiv\\\\\\\/Jyb3dltEnPak9pC7aa\\\\\\\/BKO++dkaPIdiLviMVhQw3lQZlY3yNeGY7b9M\\\\\\\/VvX0qq4RMp5fPOBndcwxnQhgxxYaKcG7ESD9t5eE5lCK9c0vrNug8BxjlObyoxA\\\\\\\/2ZiGqAz5JvSRRcGqkb2TqxyGZUDkz8+MYecufMp3gzZtgHo4LlnBm7o95q0RnsJymlP+dFjzUNcGq+9pBwXJec4AtSz6dSd0FUFMzTuqXO2QXgbGuDD5RUj4\\\\\\\/H5zS3+hGj5gLbW6uyr8wSeVcpKUqJ\\\\\\\/dS5E+tv3eJFtOUiDEceR8cm3nTi2dSLZatgvNJQvHKMtbC\\\\\\\/GYzZoQFDMtQRs4Heuo+IRQd9EnVd6Zef1aHXGsJ9aCIMREyt3TjI7bnergvJ\\\\\\\/GNCS0ISICVQpcqoGnKBeqqgGE7P\\\\\\\/ZQJBWhVveq8CdAqkzRyt8j0BcVKFOsxwPGTG+Wey+Rw2h\\\\\\\/eE0IAZN+3VCgqZyDOBN28Rdbajiiv0ZVQlv0H0POPrkMWB5Ey\\\\\\\/TK3SteXYGN2YTfWJoslYRJFmXomihtAWjf7+85PanIy4FyCpF9jIMNP9X2OGU0LlpZHivjmoC2sLQtu5\\\\\\\/e5Hz0IlLPAMaK8Fz1Aq9tj\\\\\\\/F\\\\\\\/QhyQGpasi0eQWvvlOlAdUy3Gi\\\\\\\/N9u2xK3HVWtFR9jMRzGl8AW1Kz1RdFGBf1\\\\\\\/Bat3OE8RhrtHPV8jJGBaVcpFqleNtwaBQkpRkba2TXjk1B9C+eFhdJ+sfBG+HPd7aDe1ZxWV\\\\\\\/a0iwM4wbmJK6VGW\\\\\\\/edLGCh4gOAqLmTV0OWKN63I5nK+m8Ibr\\\\\\\/uTqrEJudd47EvzH5siKlV8cDKejsG8CiAPeEjI02ljmZMGpbEpsWnHM9qIUmvCteCMaLF+5YF56ppUNMa4s7MjDyobjAN9LSwKBFwjutK19o3UIxxOQA06sF6xYs6CjG6vyKuM1QsUvXRc1yAIstw193X16heanffo+0QwKDWp1YD5HYu9qWPxJ7giz1sPe6O9YKKntmyiTR3v+KBUbigjtO7fQnUsi6qVB2YiPWz1AZ55Z+4pXa9GzhEmHtg6XXDGHOX\\\\\\\/pa1mFfVDFwKN2pQ\\\\\\\/yt+esvFGr0VkU7n424H7DXzMdQzl1PPaQeMHf2gyRX3hM0NV7nFaM7QsZlxTKr8Z8lMf2bgtmq21hBrf+tDnhkaII3PMB5fqz5ZPUxA+PceBF8jfpqpzeAtAA0egTUV5JIQItqCb3bMutghcgwKklZyQ6PyzuJ+Gd6rf1Gef3llcZ3rCsuyorOL9O0XFP\\\\\\\/0vzhX7+1gB3QscN14k2Y9ipltPaZnhDA19chNlb7cmVjdskn0qdxWLoJlCsXb7D12HhHX1sIr6hXYeGk9oeQgv\\\\\\\/5biS9DTOri2H9G5Hr\\\\\\\/nlYlDn97TnilIguXyUFbX\\\\\\\/X\\\\\\\/J9NQn4hdDvTs7NLJm2lwYAeqVPoNNQZseWdqudUZP5nHmpdT8yfYqHPR5L5bXzoXGe+ARdZdOYLuu1h+mTysbvHHYl74eewxh6hfuGClI12jNj5Sn+ozTRVEUGJ2\\\\\\\/S8BweGJEziHkluE4vSVSAFEN1Z35mar8P8FWe\\\\\\\/JnIli+fvYgeZ7nHRoG+1euwaoDBqY7xEPv4W6BJqJ5bgHHZvPK+i6sh5c1V\\\\\\\/mGf6EdX6IbauTWg2mok1oZvXR+tQq3Ef5kl1wxPti4mPOJRYSLmrtB329TgQIp0fTu1fz3ygwZ45YRIYIF5KR0lbYI3OZxM3cBb8IGnb5QfpSaElI7ZIXjqUlV8S9ab2TWVBASMGCxfa4yqU3cHFzgazcZzLstTXiX3zQ55XwaqbRlQ+KyGNaWroz8dGUlXeKNicR00LfEitVg4lIzoA9yfR1odpuIZCQda+9LsANXwx3DLGWxuEKLmMiiGiPOlgws7pdCk6rGvVvEsszdsU4GlCgvw2n6R\\\\\\\/VfcrBc\\\\\\\/1cf4PJfW6tRI6N+8x7oDLXBITybzxeNjizJMUsjy1HzI3ZWX3HwEbfeacGU6BXmwcQbj76XXeghnoCnmnvecWUcLl32l3C+aRQkG8ZsoFar0iRRT9JJlyxWJ\\\\\\\/2iijVTVECrkzpFtUMK+63m9usOmnOHj9zuNJ9zqSROp6OaIcT4YPMMHLwYOX9fBNspH3+8ob3UPubp6S4HiNEtignwcED10AC6j4owYoWSyYUS5qQM6UQ6768iFVKkQavbkqBDtd1XLf6G2vpBsS7BXuPpklU5HXejGX4Qyn7TUXlHv93rJiZfNx7ACuTEfi6ungXI5T4114mwQDDl423pKt+loWxDZTFXxXX65juyV5cMWmBnHzO+2oPpC6ufAnOrxrews5mdJS71ebz8OYFXSSYHeXERaeTrCe\\\\\\\/pxw8O24r2DEPJmfbGLTP57jk1SaR8Nu\\\\\\\/XlgEGvz3160Fwgcre7gzrk7nFNpO24XCGH1YizvsyyCM+PKoaJY71nBOy5KVNOZ4MuNRveiLWSloYuaj6b+1Yc6g\\\\\\\/BGgalJkKr4BUQfABSWTS0\\\\\\\/0pQ2bTc8Ipwd6+3ejgsGCygDnKbLLzfEDUUo0p9FVNhBJMTkfJCK5GU\\\\\\\/h+W3jIEeeMxu3o\\\\\\\/v5Ar83+HOZUkEXtV+rgWlddjXv3UHSvkNQ3dvKnNBVWwekmQRuH06oCG0BhIyHybWmFXLu1Rj1ajFN2jSydohMprletKShcfeAQnDY3mkUc1LgFw5NW4u\\\\\\\/fLcT2UWU0Pf0CzQgE8bpbtCjV8qlSBcHl7AMDKdN5K\\\\\\\/vwCRCp7spcUEvY8xbuYAzS2NKJawiUfXKZaFuFrrl5HsxE9w48wrXBIWTq1VGlDoh+K9h2cXUOg256spncrVaq7OAdxlYirS8AuYyBH6GAfbcVWc6yoLapFjNim6JufUpKg2FAAe2fqyAX0gViPPwdZ\\\\\\\/JZfZuz77o9qI\\\\\\\/m9q1uXbGqkw97tho+NnzUsoKs+3gMFjT16Tna6\\\\\\\/y+3lAutDm05C2NgivJXoS78aTXkbQ3TXSQy2Qx0Gmm4Cgx2bwRb8uXfZ\\\\\\\/6zxIwdUq5g4wF7T\\\\\\\/UYZQNTWCidFSeYT6qOMeMremR+nknxMuF2fON1sWOzjU0YSMXNa1+cFQefta0o0\\\\\\\/MY\\\\\\\/d3Gz48xP8Yn6ZD8Wv3G2mH6S7R34mS7Kv3C5BJs4+J7KFB1IwN+LBV5DD46wctptg8ejAFWXP\\\\\\\/PhbgsAPgTtV2hWFItGnDw\\\\\\\/8lhB9w4eIMp9dnG04+nnxBf1OIX9yrZU289hpyhnt7SKTODL36cfOHa6DQprhDYeQ15FoU\\\\\\\/9U+emJdJZlBt0bdlE4uuDikdhWlMeyPurMT1z3RQg7t13IzDXSeK\\\\\\\/X0mv7SoSavF\\\\\\\/XL+eE+EAQcYiYN+XXvlZoY7p2tXvjjh6ZDbLxwVRkB4kC7\\\\\\\/Ff4yMBiB8VtX68uaifVneIOWrOrtbyeH\\\\\\\/49LTWgRqwHF5qmgvWLmJu2bi6lCRR615808V2mKngfDOaWa6TEeeKEl2NwdxgvblolKRaKtad\\\\\\\/M95mp4z4IKD4\\\\\\\/mS5djzRlBwurshddXbPEdALXRSfwriVtgTLpP2j02BXk9nYdKOtWGT3Ww26TkT7G87maFOKahIK9kAtoitIJVvLOboNT3O9v5GNkEqFPYNOVpV\\\\\\\/szsZrQ4BfJ44Nh865ZQ2LeD5kYAK5xnJFRbqGfMpSEMCN\\\\\\\/heAy2\\\\\\\/9q9JuKHfbG3RYvCJWNfglRoOO5LiwGMyfLasVAZiki5BZ9nsmffJKcCPeomBtxYboXM2oMZO6BEl6+LCwh1DTR4OhsNBXZWXM+1\\\\\\\/5VgaizIPngw7Mx0\\\\\\\/2NA3VvWiOe5l7xnswOLL\\\\\\\/f3MV\\\\\\\/9cOnosTjn1fSI\\\\\\\/1egjACEsR06fZdBp3XHLa4cutWfsBtaoef+MjFGJU1V11IzlOIo4U6zOt\\\\\\\/53oesESRsJQFbmtIQKTUB8tYEasZ+6MrYftYhzHWryLOAKQH++bIdDuKWBJjI57THdiwWxRB2Wc34uqd4kTryyzpbn1iNzI+EAcNhO0iO00h5zOQ4wcn3h6OVXSz4gRdFYZarbGtmUwQkPyZCSRpV3sggVBRILXmLjSt7yM+fDfupjOso9yqpOHdTmTbV+xeikkF8ahyHT6L1noQ8DmfyWhAwGViCPKslDKWYMD0MCKRcmpwbMcbS474oyasV6A5JcdDup6G\\\\\\\/nP7ncps\\\\\\\/peFpYiTMdtVr\\\\\\\/4oN1cixIH5fF\\\\\\\/pvuVyZ9gUDF8v5AzfbzZH4aFfuGOmLFwHGramckk4F+V9duRJQU+KUkDziSnsCQKNnRPtTmrT\\\\\\\/bsh9UldUzWGKK9HdCdWVKavn\\\\\\\/GnRudb+Ybv5yoDqBiAxgv2wTDX6YWVo4lcSzxfkD0rPTtY0UcHLYOTAB5A7CtF3Wc3PwOcCKv\\\\\\\/n8lOHuEWD9sDOBIcAhvGCvr1BpVIx0iu972O\\\\\\\/t83CJMMap6rYgmpJblDjXSpsGB0cUUWlzzZJtXimjKRYhoGCTYxFRfOeSLOGs5iFKTZSr3W3dQcldyphlcoDco3HL1cN1u01FIt4PvBGIjm0234R1tTLvWU4XthF+piwGEbTY4CVBJhdu4Z8dv1K0b9wJvsWDocYjBUKP8GTYYxRriUCUnQQEiaA8GSkyLJZv2P4DgMZ4+NeHZIMXSBUpSWbOR5oK25iuIB9vVTmMDV7iQyXOzgxDT2EqyCxpUALvBOVpwumRUcsTeRCr\\\\\\\/t2bCfgG6n8LKH5vuf\\\\\\\/MApQlOUFPaSkwvUMb71ZEvMtiY2QtnDmEE2GgpM5bsSFjM6\\\\\\\/XHB4akFSl0fWORCk3qMNlpJKnccw4avFtEdUbaIuSUraY8xaDd3zorTZtquM90KrUmuYCIcp2Dnm6fvH1Sx0ebP\\\\\\\/ODxDr2rnJhmX1gDSztWPQxEWgthIxZU3YR5LIjSTNQ0kaBn2C4awFKlpBqW+nBTbRHFRaKzaOWbZQuGxXub68hudd2+j6eMlm3yprhaN\\\\\\\/X7r3E4IZ\\\\\\\/hPS9Zz29LyJxF0YX+\\\\\\\/a8GOszmH2FJC93Gs8hXfDv\\\\\\\/CJZ\\\\\\\/8nFCKv3s5o9vzEYHfhEC+nR4f9SJlfiMPsQxPV\\\\\\\/Iv3d2D3w\\\\\\\/8gW8rw4VC3L9W4NSu4ylpAUbE\\\\\\\/9TtqkW+K8rdKf3pxVsq5sycq38PBUlTfWWComuiR1fLfZENXKwxZwJKhoCsvvIGVm5cOu2kVuOkI3r4eOi+CR5cCEVpC3pY8mJ8BeksdwDValCJcZYKjZr4zrS\\\\\\\/5bm6l4rMgA\\\\\\\/zVgETuClxtq2hniXTuiV7XhnMkBMFqW5rphIiYxsEO0yKPjUd2U2lZUlPmkI\\\\\\\/fK6oBO11ofGj9q9pegI8sfBkiK6iTPn0xNqkKLrL+fJvALMs+RUKI+NkYoQj78GyyFI6wpBoucnTFsRxFg6DIwYb4kG0hUPUu65ZcHDCGsnKvBom2Iu7oyOaL1tTHyXqjTC7k0FdjLXHJ5+DgrUG33Prfy3KBswx0nk3ZLGIhPOBvB4K3x4+graltLxSTIbfs1FnGCV8voTvD1BBfHjYIfgZiK2ZPYR3uKVb0+PWOmj4OBr5eAwBWAKmJVj12Ci8Z0TWsqsa2UJ2fGAYjsFqOn\\\\\\\/dAVg+Xjugaeek9M4QG1LZmqr0N1B5KSm38XbdYMDoP8G0Xc4oHtjYz1XXU4wVib5CUaZTVGCJ9szVwEsU2GuxDnlavZtOwSXNyNZYS8\\\\\\\/IjoVnxYccO2KjXmVC989TGQkSCEZRZDZnqbiu\\\\\\\/7d4FGjnZH+6Os2tIdLM2IlcWw3nIl9ptoipmllxmkrnVALsyTs3X4CPtA7o18RQ7S5lUEEQqUw7caymvlopWSi+KkLcHiXJY2Ud9bmWTBSAWTj+l4+An\\\\\\\/yzCHlESBX5y1Ea+45a5dAITlm0CxlWsg6es9OOsXxlDye+P17wmhhHzQg5punBePywTfr7nI1exIsBgCoMFCIqqwxcVmCjPNDIgWxeRUnTMQXrKApK1VWGwT8uDYkJ58Yf2LyZft5u7JBuCV1Eh7xKHCjjPHmUKk1BRhW7XCOGofUTzN4WSH\\\\\\\/GGoYvm+ZxDq1uv4yFlFrrk\\\\\\\/lu984QZevYEznbnX\\\\\\\/HWIuN0zFHil3PriuuVkFTfJWCaUDWLNbaQjCJ82ox0asSaXdMBSLBh+GbacgdQGRefRbO1lWHb01esZNGlpcKoTN5vHidRtaaBvLJF3FAB5Ys\\\\\\\/A9i65UeV\\\\\\\/L5PK\\\\\\\/hJ6QKIMwf8PUkV1ZJwBh7CnGA861fnz75WTWJ2YIry0GOWDXVTpCPSqeK0aqhIeWTC+aimqiFZB\\\\\\\/NKivdRhhWBdhkIoJsuOmPwJi6B2LisMU9PLNMRQvI6iln9wJmGtuxTTiYVUhAT8hhP8TMSxhhddNdEN\\\\\\\/ji4b9mXn0NitEacGgLU3eLLoY2eY5BPEWbVTAl4dmOxbVYHb7DYFfmMDBU2HxRdDHk2JyNQOm4+LESdYC5sA2sk1Ujf7excr8qMDP1l\\\\\\\/wibQRcqzvd\\\\\\\/kBWfidtsdm+aK6RR9EwtEQjoA73T69makc+7ePe4zKcYXSql0dhmIRJfSwRvu4K8mGxuTB6pumq6XjbNyfHjraIETdy\\\\\\\/xvQxYcCWBcr7dptHwpCEA6+uNpvn10\\\\\\\/+uWykGz2XG90\\\\\\\/zewaix7UcXQZ1vsNs7blYfKuC5EopD35FFq4yASqig\\\\\\\/1s7Ah2JoqGUGWhAsVZuQr28CPqX1aHToLjONCcghN8zsUarj33ZpImT3vnHvZ2x60MRFylyit\\\\\\\/XHmPdEW3SX8R24vynxbWpxpt4Geu9vgv85pLPvHiqLP7tOPif6cfTxqwPtj04i61wnC9azEi0JRZYuOXGVUyG7RmYVpxAi+t1WqP5rmKCOWTHLH2QXKGUmPiGt8C9HCZb7LziCIac8FclNlcLBFL8uejqaU7HZaOoVs5e9CRZAYEoYuQ+HUL91C66UGxtZBuWM8GDOTMdtPrPD5zKeNA48vDJVV7v634dbo8LyRMzC4ts7QDjMdkrYYFNFhQm2UhPfi3CdRena4LoCKRLA09wbTihjDA7qaiuiqqO8d1BIl92UuXWP\\\\\\\/QlQdYE0XToshW0qmMk9oXEwQixtPWbvdsXFhLAnuqnoTvc8JF4vdufi9EDBYfV4RmJjfgAi2n5AtCYXtoya0UvfEFHEKdz3ssjUS\\\\\\\/Ajame4XHPAJbDQqEHctO+C+bdteHGMUaJi8e2nsc+rkGw1fFM9IvjZFqMFSD1zeEpe0SDAlLte5Ocaw6ZXhG3bk4qca6KUyAP2HmlGQkCMo40PlRt4ZdRmVK6Rtm4cuaTylYW+y7O6fyghmLtkgksy3uX6rjvhA4LXBCxJLvVRMaWZGr13\\\\\\\/u4pE4w5BtHpD0SHVQh4\\\\\\\/OiY1YGB4j1gnzthyWUqp1FvRy2E+W8XZSAY27LOfL2gu2CzZY4afonlVDLnCDuWUPdL\\\\\\\/rdYZ6ZgV1h57hM0QolBWCdnj2xCW6MpugHHpR7h59Sd+D5VM+aLWP+O\\\\\\\/BN2hj5tnKdb8ppcl4BvFdDIoOLZqsljsHS1Ct1PAvJq1Zk7GM6a9SbQ4WgZS6\\\\\\\/Pk5h1\\\\\\\/D5LzUvJmNVFo+q5i9kORfwscArDOjSta9CS2c32G4\\\\\\\/mKifpsQOC1wdSr\\\\\\\/boL4Nyt1kjnNbpgxcebvBu4qwLCGrFEZfXcfhWN5nV1ojUuMSdPZmHifUi+KkZUj8LmywX7nJ9ULD8E6O+0GBPXWv1Mqrnieyu2xCE2PfbGSZfeuoKdgIG42xSEbgODcgU3iUcog8svHQ1gjEn8FP+ejt4Ii9xfEXO4nBuNPhJAbVApv4o4pRmuAbkg4u4X\\\\\\\/K+ueB1XGVtNwg\\\\\\\/JMCdJ2qw0UxskQdQZHS2nJkAX4UnkDzRIPosbRIhRvwe+yEWGJTd3LPgJnsBSxcWgzzGY1ZJYvY3Oot8D4ywvA+HcVZQYFh9UJaF3OXxuUzCZMjk5LiQjn9xsAUBUPSey7KwZuFs34452UVDAo+G2suPfFaQZpSydZaTiNlAod0YuSfz6zp7zNd2p1Jz\\\\\\\/H1raGhghAM8xlOVj66WsPsFtWaJE808LP6cSMBzMJfBHYpQizyYQXhujmuqhcQvAXYs3GBDJ1XGSIVgp0j1C\\\\\\\/rHDzddioko8bBaRMiwcKz1ps6obwi6SLEfZ0geZJ9C2tAKbDTcyzYLUGqKGosVJcIk45U6Pl9n78cRby\\\\\\\/7HMdbs+LWqZn6G6+SP9WMTnUN7FDuRpMPfY5eEr3jFrXfi+9ISs7CPwTnVoAkjno5AFHkduaw4Cj88fYcaNRb4QDdsuJZ6Gz96iVpOslyrohPAgrp58hb+Fa+QhFcMbd9udNDOgadRCku0ahhD1NgLVU1t5VkgTPWYin8KPDMWItZvG3fuJHDNVW1vju5Zx+Nnr19vIohDqDQsbQhIDZ1lIkDCQm2gJws3kMiTcXsVvFZILxq2N4Fq9qk2\\\\\\\/4ugqxXu3BMvMqSqG2BMu3MSJ1SsS+\\\\\\\/QUNG+4mxXRnddQI7jskQizNXU76cqY9sJ20W3ecHPnrWIBAkSYQ8VUDTJr7LWkTJdoxJ6iOxKpYhZ9UGrnt6E+YVoVhPI3dw0CLe9V1EU+cMO9\\\\\\\/Pzm0oyWLqhbYgoH3YvvBAlgvbhN5mYfe7QdR061x5mM8aO+ND1W\\\\\\\/AcDtV\\\\\\\/HbzP3bDKjeMKQjUX9\\\\\\\/Icq3syapj7WP1EKOYtvlEMFU3KVprVCO0IKNtZs9BNPsCbIHvj\\\\\\\/ElW8RxBJTRY9f\\\\\\\/6XhCzhYXbD6LYAHoX3rO8hyXKMPXA+7I3nAyaqLk4cgUyndZkyZYufK6r9kuztsnNn8iSZy7AkDqsmh5r7hXk+AAijGUTn91CF0DYIXx8waZEJ4cHXlpTfW7nMom865vph9wzoa44\\\\\\\/WI7pqmt5I+Xo0GTizm27tLLTIxcyVKUyglGeMQNnckSnDg7qZfbIURiYClWPww70tQQWJytAdejHUTG7I584leZaav8hLgWH9g3+WZ5ntQ6qwvxBL2lczzhuC0wWh10ZmrXq\\\\\\\/PQtfNNIPEfWeGJ991GIUxNkYsI+93rlRnOnqv8u2e+TO\\\\\\\/U6NMg8zaeUMeWUhkJmKMNzKnNEaUx\\\\\\\/eim9r8uUKta7VQxpQ3ZTKDT9MnD0R1C318U\\\\\\\/fN0VI4QoOI5tCflF1Ft4ahdxAHgvTLa454BYTsgEwmMHeJIst8hgjGNX2wveYXMpu6rtm155tugOOrK0HQsGafxUqg07wuffHaWKTLrfaDlnxBCHg4TLtEFwVeDSsvLuX7SkJR3q2VkQahZdJ0c8ZRzVPgASerT+lyYsfmVF6wnY5ESu7xI+M7MpWUo5xrJpgz1zXKBwZ6wC3vxVDgIApoRdZzuEawDo9cFgv+EgznmQSk+eIKj06pP3ln+RzJ9BY3jd2WPMFG7cu514uK2amMZZtIiBvdoropqoTfDSPQ76x5gA5NDb\\\\\\\/iUP7TeOdJFIop2xokin4A6fCm+9447iRVQh5vKS52l+G4Qi2psSbvfggq1IeOSHZXwrg2LJE8hYXlM6L1wcNgGY+JXQ3Anqdj77bIHi+dLxwFV5Ux57tZuSqGDQC+ltWoLB00ycPZs+luTv48SCP92FBGDBbNfJIOyN99\\\\\\\/urjjrzETYMREOctyXMA0DvRXj17msR88cwHnTKVVJP2HrMbjqqtmxKtFdxIa4nQesuMylfuo9SaTSlF+QSKmPQ4pMq8wuglPZE2JDFQV8rXv4hR2gahxF74nx9aifaY70sAuHDPa1xBNTZUKTTkZaakzs8crHWeauDOOJxdvgvPqvy85vu6SpoGcu1U3zZz5EF0MU6M5WvSaCgGPXZrYE+Oa7BMewcVtOEYUbK\\\\\\\/Ri+CjsjwNkixV1JBXqJ19TtMMt8ju+0LGGVddQ8VjRuyuMrminwDgvPgPzRtIFQRWBIpb1ifV8qRCcfcxjQWcR3\\\\\\\/8kHvLSxDcuruIhSDlbayNJlRai0PczoGqvBZBjDQCVjpYhTn4MJJjdQspalTvyvQTaEriqIS6UxVmHMnnxC8+sola7zE2lTK2zmBxVLVuv7lP6NN4jk5FYZAL4INPkydQu7tysEHoGhRhd43Z+3noO6lnx82irZZ9kZvo0znsObISmUs4PqZi2GZ44hgZENhHJ188KnkRtQ6i4VRCzAk1P+0MzyH2UQ79yVaJuMLdclunSnjxv63yB8Nzg67+rjqNgnPQWWAEbc+tXWI3Ph1s6LJXCK0TaUgviZXjuHnQfpnXW6AXVQSyqrPki8Ng00K7DZ7LhTbj2LSVNXNRERqvnJ3OVBNq6w47kLXTrYe8KYhYRLu62bfBSDUKlMqNEE85mMfQAiQnsVgk2LHoLAPJux9QeffIIsGKgiQe9GnxV6ThJ3Ym+5jlBeuhNrKRtTEo4Ai8dOO508l7IBph53hvqa3xILJxIHg8IvSMh+QAeehWmUe2vp6ekuai3VzcVGXiuG5Yhjb6vldPdGvVw7RH65RCenscUwkdznGeOZGuHNTfi85XVXN2yl4rmzveOLLIyXVuL225u8j+5dItfKy06bUYDCWeyqAADdJxYZUns+kF0qr2e4EVy59aMMMa\\\\\\\/9xTjjrqARatn21yrHuJzXxRQNKn3HhNIsV5ceAVBkNHGhg23\\\\\\\/xh4nhwfmGRf639P71x54w8aqvCOVSU97IqrlTs9gZcDTA1bbCiiOfc2CSOYjqDmEmL8lmYfjFVUfNgybyRe9iMnZIaSTmvfnlq4xcR\\\\\\\/kKrKY5KTD2xIyXnqA6O7MKEDE4NxhgXbXhYzWoRy4H6deaoSe+5TVme7ZYbHT3n7\\\\\\\/mp9vlkiG+06MgsqcnpWjGSjDWsvH4hfSo73C6VmzVms9osUKqwaNcrKkWD7ROXsq1ZGXilvRah3QlLtjaJNwp+F2KH338PwynuAP+gf8zhxnewMR\\\\\\\/txZzLLRe9R930\\\\\\\/u\\\\\\\/3dE32LyuR7dScxgw8\\\\\\\/4cor+kvTFIIXoPsvqErPpwXX6\\\\\\\/c3DckE+p6ftYfA9e4ZtGobRtGlf6WD0NwnhGgZrriifOgDsqvHizHdhqObbFA8j9aNUJBOBMZEZlzk7fJfgZxNaLa1WTxXIMBcdqQdEwpHFAizEO4jsmdIH7gXLkIsHoevTQoQHMufD5\\\\\\\/zeoEHjqdchzaRK6N1AFkAObBrcmoMjjI3dvZ65jBk1nGLdHrvZ6culOu6SNlQmgTK+s0Z0Gt1o3rpQMPUdHkxcPiTMIxx+OjWmBGuk2ruM5M0L51I76Z+iz+KPlBMAzUqUeKDj+LKOncjDsM5uCD1KsQRq8zHldWUNSQ4rTtVbLZXAdWHoz3pK\\\\\\\/lvaZMKnprfQJIekUNYJE9jIhuLuF2Bd9qBROpbDM3olA59cnol8RpYhOEW0TY8SNbSkYnO2WBsBup11RMyySCEmZakp8CuvBcCUQzb6UZl7\\\\\\\/cccRoUp8SCM1+DThKiBCBzrwFUSibJCTVwyYlK\\\\\\\/5RyFL5yzH8hl5dQyvqx6Rdz8eZPo6yeJPGnmxv7Qgbjps2+k4pzwq6CmvhMxdcXq+GYJMFamr8HauNE934XC91qnySUjK6gRtpsP6eV6IlkItfsd70h5AEJtNYOfoqRlPfwZ6ts\\\\\\\/r8qIQsYzToOgtyuIkeSWrxnlvpRU4Tg+qCFKNUq+FABbqgPxKV9YXqbOWH6SbECfGJQaFEJRmTmQUnMLmtbTWoenGmFlA6SLJgCwJiFqj+o9qc52UpL9qFyG+1gZhjxfr\\\\\\\/K5vmUdmnKB2X5tXF\\\\\\\/38Trz3FpubihsFe+x+eQr1hudRxZbzch\\\\\\\/hH1kgV\\\\\\\/aj0oAFwfJeE+CEUaX23e4wkIhUbLxU71mBFx6k+v8SaCZ09hHTvrNsgd1lyMqNWkvdBeERnKwzZQs6msYz\\\\\\\/ktA0lUfIAO2qKjh8lkVrl8\\\\\\\/TDz6AN691U4VlFSZgClLnwfMKzel69noqdZtBvYbpgjvd1eQevct3aI9PUjG3e\\\\\\\/qVS7dYOReytgwrighmwKYBP6bAqO7iZV8XU199n+ohgxqaX02J8i4L1yjyytf1NgnTtLlGLNNRE25Ehl9ocTmh1VlUS7R4AMZTeeSZxs\\\\\\\/8bJBZaLdG+xmopUzmDNz75z5n2LUOI8V+7xbe6moSMaSQ+YcMvI1eoA2N5Ngv3jaoq8yocdldc2qO5qXdwA0Lna7G0BJYxbKLAfF\\\\\\\/3ESnx30Ex1\\\\\\\/CpXhsy0kboGjR2GJjcLNP1jfxebYRNOw44hcKS351zb5Tqsp+hNSIA0lll0mywvki5J\\\\\\\/4G0x3Qtdz51df56wzOkW\\\\\\\/yCmINTW\\\\\\\/k0lY\\\\\\\/R\\\\\\\/NfR9NW5+dlPdyXwMtj5Agu5dfX8DLvxqk1LRf7qyge3hUZFUufPQckCXvjN0hbtcH1olLvjr4qPOI7YkTEem7K8TQkMsKu78EQxEW5l7NeuKkQjlDA2t3Zh+8bUK8g\\\\\\\/lrEpDwl1wuAB3rDoy77SnAVZm14SeB\\\\\\\/083pc2PMDiCqA3uZLhM1HbK+fNgA769cNX52o2glfHzzbSUunI7Vlul+alu6ZuTv319RFa85h5uXef9U4SWrcCluuSAxTIH4bX4GZc+c58w1NDHVXsCaauPOWI4JpANTBcOIU4yFWIOW86pX3WjbyAsknhXfVr870cZYROls\\\\\\\/kvSKdaHuIbO9QsYM5ICrF9CQMD60zZ8IEp9M8eOD3TXeem6G8JNlS8cLJbKQIKlXFSnxpRUDN3MHdudN2i5d\\\\\\\/8pPuL36GOXwwAxCZGRZH5cMJX+rA76f6rQQwn9Iu2paaMRQMZHHd7tUk+EJaZEFd3MxJpYPfR\\\\\\\/kqUovNKszFxMB8w1x9uvp+2gG9ipBhWoFbBbCFa1+If3WPkYCjY4wrXGumYdb4+rNBsiM4dZNgImHtBvKwAe2SLtES5oXIYFQGC6xSnBLE6hULM8wCZ2figPFYNN4m64Z8mmbB3afgx24TLlUKsGZSVCjLQUXVjgB3fXn1FS0oifF+otcfHghcOkIamGNGcqAyx9pilcA3GCvVgxXfym8n\\\\\\\/EUI+lR6ysxiq1xo+tMoHyH0z7NaLdUUQfjmz7tOkjI\\\\\\\/3a1KiKccCyEdCIgsXPDq6QFofwtpG8Hf2bK9mJ\\\\\\\/Q1RO6uvGrcqBQ3tZg0ojE6aSeOznfY2To6wazLWBXNbFJzY5BiSoVjV6rw5n0NAr0wdFwJSPeSPCzOap\\\\\\\/4wMdFsuec6l5Yc1jIITI50wYo5C+TQPpt\\\\\\\/EwQhcZaFqk86jC5YQfOKlmPqe9XK03ftr4fIzO4N\\\\\\\/gY4n6PwatTjGofOZTdOqkICoe9c00IoNDAl7Neds1NIUMiephtoLVqFMM9IB+FypJvGRICoV1sUliTpb+i2qKWff95UwY\\\\\\\/WWI89LAjkkK+DZWH\\\\\\\/ByDO4N2\\\\\\\/mjaqkgyxUSwynp3uQA1koyU\\\\\\\/523RDfusktfMptXgmZjA030+V2hBlek03bpUGYV\\\\\\\/b5c58qt2OE6bYJRsSNPnrM0Bgi61AOaykzFKubCjz0gVinlyJO0DqK\\\\\\\/e4Hz53S8XC+J1SIGhbRAGjCJ\\\\\\\/sE1DpkgGqDsfxwIfB6VA7lmG+Fwi\\\\\\\/UCiJM8L+RaQt+aeQDEv9X\\\\\\\/+E4iyWNpLQRH5hz5AVbFWDUCOBbaXkmW5umtKtWQZTDnOXyVf1wlfnDxRMANQBB+4a4X0glHbQ3QFUVS\\\\\\\/HuI4U4rLN9ZGNjMNTMz688yTT2UPaaHAwoeiEpUNLYdpGMJoylQjvddaf1VcQ2kj9gQwFMUJzw35OFDxayhlupU\\\\\\\/teEVAaXUqsFwHqEdVs6RTguajqs6EV9TMS76sKQBeos6lc4VyFmmoK4BX7X\\\\\\\/qEMCjTqu8dDnD6Fdb+rg+IS+slFbrJBUg2LgKKHKeTxAv25gNS4Cila+2qa6I4ai9OLUx3czXZf+SYWQN28tk9eVG1APH9mRH4C+7jdGQztHM0DZttezpn\\\\\\\/s\\\\\\\/MrOJAml+4TBLBBC2WlT\\\\\\\/uz+tXQoR7afSjen52sOuP4035b+5bXamuJ7v12QibBim7h3JVhwHCnaEmVhwijBnH0GZjO+Re0ghdSNBz+6aq\\\\\\\/9z12OEehS77NstUAvj4ngBCg8aBS9xoZ9ew3rF2juT1HbT2sgyBZlRm1sEcb\\\\\\\/7yPfpzfwAUkBDj57QKs4+1R+U55LViMn5LJ\\\\\\\/rKSrlRgDD52MMuiNJR6lTslKZnbtJU02HXQTnDPVQifce6DJD8j8v72TTNGsjXr5wZHoZ68G4CKBHf0n92w\\\\\\\/mwEH75fFg3yF2RWizoorvaWBvGy0dudpKeX7PXVQ7aV0x9s7L5Ftw1QMqNV6v3D4mpH55QDld2rJ2jQ3SrtTEUdaTVcGE6xPA5D5zMMKPcg\\\\\\\/al3wZ5PGy1Wa4CbJ9JUrTXgmu1QLqjhTWTHJXyFHSDXd7ZGT6onsXnKU6n6rX2A2LtYI02GoLjsjVK77l+ZZdZ7Nqj5Wc4wLP56TDZAQszr1KEdJphKe1gWpr7CR8CUn2FMCCER5fGIdB5wVgp7zPnPIwx00Y7lwM+bank2BGZCnR3+hsQvfd6NCFE07LNAMme93WApnvPyjIxED0DmO3aWO4\\\\\\\/PVYu2Qv42wTkGZout2fBUQWVH31HSqMPdL6Gqjgi473WYbqrqKSlLTX1\\\\\\\/\\\\\\\/lqsoDNTgLkEqavc6VDW+UwWXgfB6b5bja1QXnTnaJx+OHVNi6PoLaLsFqfeFrHFaNcX9E2CUHzRTZYWgvfSuNcl8U4qhfXd7opOO7kS65+ENJWhI2gtr8mZh2OePIkgkihn0V14JoSexP5DUHg8uo6rOd4a4+0xRn1Gm2Q8mHIO8SfAX3mDm631Sq+qHE2oipkXF2AcwuZYw5pgFQgR+e8yFNZi\\\\\\\/ZSzujVYPfN1DI53fgZkDHvPqM1eS2\\\\\\\/+NE6hR98VxEUC13+2iGc7QGTOpkJx+9+Zi17P6Eind6UZ8OzbWo2k1pF+kbqFGPWbLgzKR2GuWLJhiJJI7UrOptWazI7ujvJBVA0UCzLQc1MHRj4v8oArbJ\\\\\\\/lPqegutHMsahvZx4PT58eY3e0HMgfKgemeKM+ecCQcvKmi80wgsXARi2LYxTTBDzsWbqK0pzTw5uBb4Jz0ajRclKdMgYdVay4QX+rnuxKIRw9Nj+nNcDv6DGdd205CQ7W8jub2ZDBW8sNb2iKY8dHzA32qXBr9WfM33xioMJ0wEA6Xjtd9lHHh0E+69r59iq227Un13kr3qfGSZOfguPSqNFOi1Vnc1Gv2O+NpwbSiCfshaqqB3Xy4G97Hq7ybJfQQkBK6SUg1d5rZz1e0sOx7AIzRTfnI5w1JUJOwX2MXGh1aOkDAnSo3evksWAujmD7X\\\\\\\/tCoCpvrFOSOFUubdMK7H12ZiLbUg70zYxbp\\\\\\\/+XGHTQ1RIxF9JO\\\\\\\/lS8e\\\\\\\/pUtUYh6R6D4E1otZL1zDH6S1j2Y87s+WwcVvJXmKrvBtTvPlAXaLfD+JWYYxETZh7aDsSnUYGxTHLcbrMXgNLN5i+6ckTaKQyLGPd6sozCHUFO6lqcFh+7gyMIuikTNR4uiVN1\\\\\\\/GdrcVpZFDk1QKEoRhgIzPlUr1GvWww1YDWUCe+z7oTHP45Nv1RH3fFde6noEwWJB9VIkvr0j\\\\\\\/ARwa6RdRAxbr\\\\\\\/SCqn\\\\\\\/5G+\\\\\\\/qefcTol43jD7isa8yqRT2u+pbCvuf5vCXwfaRQ2PKNc\\\\\\\/ryWD8ZnYumSF1SI83SdeyB9M9icgYjVo1GxFFjNh9l\\\\\\\/MMfbNSt3PKy5Ps5D++sPThmF7lm5\\\\\\\/IvNS5qxs9D8\\\\\\\/u84\\\\\\\/xlXyVcmcQrZxNkm7zIsXzzdMp4iXUcqlytuM75cJaIPre27iKlvpQTnjLXsEpoLCvCXW0tNli8pdI1BTe9m0umYkoQXDRyxBi+H5+Lmo4UyshpWzX22tL32eiqC+KszBKnIupGy4lZWUaYlrzkcM7iVOFrKNiUyiJV4IdgscxaMVOZmgQe4SjQw\\\\\\\/fAXQyg+8pTL8x7qiq4e2FJu\\\\\\\/vdpUi4KyFI3tCet9Img\\\\\\\/FRxBh7RanXfTB9j3nr1ZJdJOAftKpJkhW\\\\\\\/6WFq0Gm7h4uYs4nktepEJpEdvWMY7owAr5G95LGjHCYK9D0pxMh85NrMnD866A==\\\u0022,\\\u0022iv\\\u0022:\\\u0022vskqGM7oU0UBpP8CdHFaCw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00220e943a722171561ec52025236928c762acca81a8c2f4769580bef33c523306d4\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022NT7Uj2Z20hrAMIrGypBUCUEEz7\\\\\\\/qIHlvZexnbMSsujfko523NbuReixhNcY3hDUyQasnr6cVzq8fovPSU0riPKqEx8ctQN+K1dY5K9524So545J9xmBbyN\\\\\\\/\\\\\\\/hXkIfvKEMMGtHADK2123HViv9qw+3khI81tkoqkkZcQX51bY6DCKmVUMAP+5\\\\\\\/T7UAbqgtAlPp5pwUixOa1elFyO+45vTEalx+28eJ6gacjzHi7QbgNWWBz4LrafaR8Os0xahL5UTobbkJS7gLyw0JewQQ5NfIC0KL9Dq1weNpty\\\\\\\/fCPa\\\\\\\/+6NZAZQG6\\\\\\\/6ytH\\\\\\\/JCNIRFXHm6+WxNFehWeaXBYoR85yJ9ftXIluz7bqtbcEa80DznOrjB5\\\\\\\/7oyfOQtrRpPuZQuH8dIEf3aAB5ng88MPOVl\\\\\\\/gcyNRyXp0ayCLHPOmiu8lfISdiYYBQcfjxf4Aqt2fKzcep1aQ1UJQUsmJ03kKZDr9lJDGYDy7TuQK7p3KKISaP0ZpAlPEADdfgspwUp31I1BAUyaepHBaufcWriuo2HzLJh04ja+D3DoVoAowHxReqMPxu9BL3Sgne3YWsKF3Povl3arnzR0drtba2vXVfq2LqE529s+jd3vDgT1uvW1zm7doThkJqd8uwh6J8R2nKBep84\\\\\\\/4WetWNnVDfDZIPcOfAw0myHHGLJm4mSuxWpVp9apH0Oe9w21YM4sLBpD1ba1K4bUDTO1p\\\\\\\/fGWlRa5WFQYsXzuoFaR4PVEpkakx7znwUI4yIZx6uGHW07kNzeo9wBXhdfcqiJRxjmWYeywEVHYLEj5b\\\\\\\/argwa5z8FJ6EW8lq6JSiQ7yfRRE+pSFxRmEgA5iPWF2x04s1vU6pqKx4eFb5B2U5GdPTTqHPoRNwUdL8lNbgceUOAGqHGtZlBewHJwDLOyeb2Tcvl2aWIZsSGQtc5UrZuj45GkYQTzJBSxgmkoxZYcsnnllvLxPpmpJcXKgCVW+0BLJPrRV2Gyxq7B8Odqss2B+GIxd7rDi1lOY9p2JTpVVC7C+UIPuLULH8o5l5WyrSSdDrji8L3BWQBlI9pt2LmwX6DXyKoazCYEPdpPbom9ues6ODxWu7aU0o923MGsPsSIzNO6I3\\\\\\\/uw0sCwrZ4WCpjCQw7Kd2sMNWfCin65uAdmUeeIMmFdITniTGpJWZ7SHy7IS\\\\\\\/RfpaX5XAXxt0oxDafatkHPU3IL+eamzi2TYOj6UeefWcsXpLCmnLwea\\\\\\\/5fwUsODijSP6XZVO9d64Gi9fiZC7wFVxnCFcVoGS15moJJJmZQDLx5C6L15FtQkPtyk4jkNj2wWpy5W6aApJbk2\\\\\\\/PiXjjReSy+oEZB5rMknjibIKVV6ZauUosUrbpFfLtTd9k98I9OR5L5kmRRb4dm9CyZcam5sKdZazSb\\\\\\\/oGECLApsuP7psbSjwMEmAT1xXVj7+nkYGSWBNc7ccvrpvrnNlHA6P50sE+FBg0S\\\\\\\/1cVDTlImfWz8RzfO8rMlUSmggAr1EAaTu5cptznyYLwMFPdcpn3nvJGd4ShKoCaKJI\\\\\\\/RzlX5GAQwPhZLiPNrNMHLZDB\\\\\\\/QRdHsqFREWVaso8ADu7riazal\\\\\\\/JH4I5m0kMx9RqH7HB+twH2b5SUQjXptFcfFNA4y1fKwC9u5ulvaTSKhOx+odv1A1o142HyWlzBsz5DFG\\\\\\\/gddJm+dKHa7\\\\\\\/C9ZqNyyiHmCfCvil7htIFZhPW1oqTjX+4qldmRfHCEpmUabfD10Dg58d1i9eRkjWm\\\\\\\/xQO9iNEkj88idqDpbTMQdycop06o8p6h2YDkmCO4osYocNUrdWhiIuLizKPtpeFu5YziE2Frc+h2m9jDrQvomedtOAbdM5UMsJ35U9WIY9QqKZUW7Sz\\\\\\\/F8wJqzEGZTReN2B5KUsJW6gfjMKid0cQnbLblcG8Wh2389K1QovGBUyK1VOfVmQI2f7qFCw\\\\\\\/fYDuf2wbtCA5Xvlm7Jkn8xdWe1EMQjiOub+8+eAfkuO10pxxPc6d3tx\\\\\\\/PLCQbmjc+mxq50ytErXclO2U8EGvhN5JoA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022+ZaZJ1M+V+YGoeFDisZaEA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022878d22d941169e71d2f6d68bebfe57772b0411ba7eebed0e25a593228f696b73\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-491242 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/11/3715" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-YpI9clt1XdICb3Y2Y0gNXcJjtbLa7p6kPuP93SnoNIg" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/11/3715" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-rGzo0YpYi7jYy5NbiMnWOY_BuGuBX28wrnF8EXfDJWY" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/11/3703" title="A Case Study of Genomic Instability in an Industrial Strain of Saccharomyces cerevisiae" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/11/3723" title="Characterization of Genetic and Epigenetic Variation in Sperm and Red Blood Cells from Adult Hatchery and Natural-Origin Steelhead, Oncorhynchus mykiss" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="491242" id="node491242" data-pisa="ggg;8/11/3715" data-pisa-master="ggg;g3.118.200700" data-apath="/ggg/8/11/3715.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Comparing Genome-Wide Association Study Results from Different Measurements of an Underlying Phenotype</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-5946-4414" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Joseph L. Gage</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-7867-9058" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Natalia de Leon</span> and <span class="highwire-citation-author" data-delta="2">Murray K. Clayton</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3715-3722;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200700 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node491242" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Joseph L. Gage </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joseph%2BL.%2BGage%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gage%20JL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJoseph%2BL.%2BGage%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5946-4414" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Joseph L. Gage</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Natalia de Leon </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Natalia%2Bde%2BLeon%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=de%20Leon%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANatalia%2Bde%2BLeon%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-7867-9058" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Natalia de Leon</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:ndeleongatti@wisc.edu" class="" data-icon-position="" data-hide-link-title="0">ndeleongatti@wisc.edu</a></span></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Murray K. Clayton </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Departments of Statistics and Plant Pathology, University of Wisconsin-Madison, Madison, WI 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Murray%2BK.%2BClayton%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Clayton%20MK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMurray%2BK.%2BClayton%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/8/11/3715" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:491242" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:491242/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/11/3715.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:491242" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:491242/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/11/3715.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:491242" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:491242/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/11/3715.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:491242" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:491242/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/8/11/3715.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:491242" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:491242/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Increasing popularity of high-throughput phenotyping technologies, such as image-based phenotyping, offer novel ways for quantifying plant growth and morphology. These new methods can be more or less accurate and precise than traditional, manual measurements. Many large-scale phenotyping efforts are conducted to enable genome-wide association studies (GWAS), but it is unclear exactly how alternative methods of phenotyping will affect GWAS results. In this study we simulate phenotypes that are controlled by the same set of causal loci but have differing heritability, similar to two different measurements of the same morphological character. We then perform GWAS with the simulated traits and create receiver operating characteristic (ROC) curves from the results. The areas under the ROC curves (AUCs) provide a metric that allows direct comparisons of GWAS results from different simulated traits. We use this framework to evaluate the effects of heritability and the number of causative loci on the AUCs of simulated traits; we also test the differences between AUCs of traits with differing heritability. We find that both increasing the number of causative loci and decreasing the heritability reduce a trait’s AUC. We also find that when two traits are controlled by a greater number of causative loci, they are more likely to have significantly different AUCs as the difference between their heritabilities increases. When simulation results are applied to measures of tassel morphology, we find no significant difference between AUCs from GWAS using manual and image-based measurements of typical maize tassel characters. This finding indicates that both measurement methods have similar ability to identify genetic associations. These results provide a framework for deciding between competing phenotyping strategies when the ultimate goal is to generate and use phenotype-genotype associations from GWAS.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/auc" class="hw-term hw-article-keyword hw-article-keyword-auc" rel="nofollow">AUC</a></li><li class="kwd"><a href="/keyword/gwas" class="hw-term hw-article-keyword hw-article-keyword-gwas" rel="nofollow">GWAS</a></li><li class="kwd"><a href="/keyword/heritability" class="hw-term hw-article-keyword hw-article-keyword-heritability" rel="nofollow">heritability</a></li><li class="kwd"><a href="/keyword/roc" class="hw-term hw-article-keyword hw-article-keyword-roc" rel="nofollow">ROC</a></li></ul><p id="p-3">As image-based methods for quantifying plant phenotypes grow in popularity, they present the ability to measure phenotypes that previously could not be easily quantified as well as an alternative way to measure phenotypes that previously had to be manually quantified. The types of novel phenotypes enabled by image analysis include fractal dimension (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Gage <em>et al.</em> 2017</a>), principal component analysis of plant organ biomass (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Miller <em>et al.</em> 2016</a>) or shape (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Chitwood <em>et al.</em> 2014</a>), and topological methods for quantifying branching patterns (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Li <em>et al.</em> 2017</a>). Image-based phenotyping enables increases in resolution, such as in <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Miller <em>et al.</em> (2016)</a> scanning maize ears on a flatbed scanner at 1,200 dots per inch; it enables advances in accuracy, such as quantifying disease resistance (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Bock <em>et al.</em> 2010</a>); and it enables scalability and throughput, by building multiple imaging devices (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Durham Brooks <em>et al.</em> 2010</a>) or using a mobile imaging device (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Men <em>et al.</em> 2012</a>). However, image-based phenotype measurements are not always as accurate as high quality manually measured phenotypes, though the decrease in accuracy may be paired with an increase in throughput or decrease in cost if high quality manual phenotypes are time consuming or expensive to measure. Though it is not strictly an image-based method of phenotyping, one example of the tradeoff of accuracy for efficiency is the use of near infrared reflectance spectroscopy (NIRS), which has been used for decades as a way to predict chemical composition of silage feedstock without costly, expensive, and sometimes hazardous wet lab assays (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Park <em>et al.</em> 1998</a>).</p><p id="p-4">Potential tradeoffs between measurement accuracy and throughput need to be carefully considered by scientists preparing for large-scale experiments. In the fields of plant breeding and plant genetics, genetic mapping experiments are one example of the type of study where such considerations are crucial. Genome-wide association studies (GWAS) involve measuring a phenotype, usually in a replicated population of hundreds to thousands of genetically distinct individuals, then scanning for statistical associations between individuals’ phenotype and their genotype at numerous genetic loci. In such studies, the accuracy and precision with which a phenotype is measured will have a direct impact on the ability to detect genetic associations by GWAS.</p><p id="p-5">At its core, GWAS involves testing for a difference in phenotype between individuals with different genotypes at a particular single nucleotide polymorphism (SNP). This process is repeated separately for hundreds of thousands of SNPs across the genome. Ideally, SNPs within or near genes that have some effect on the phenotype of interest will result in strong statistical associations. As such, the precision and accuracy of phenotypic measurements influence the ability to detect statistical differences between groups of individuals with different alleles. The heritability of a phenotype, defined as the ratio of genotypic variance to phenotypic variance, is a useful way to quantify the proportion of phenotypic variability that is attributable to genetic differences between individuals. All other components held equal, heritability will increase as precision of phenotypic measurement increases, due to decreasing apparent phenotypic variability from measurement error. Two methods of measuring the same ‘true’ phenotype with differing precision will have different heritability, and thus different power to detect SNPs that are statistically associated with the ‘true’ phenotype of interest. For the remainder of this study we will refer to the ‘true’ phenotype of an individual as its character, and will refer to different measurements of a character as traits.</p><p id="p-6">In addition to heritability, another parameter that affects power in GWAS is the number of loci that control a character. For two characters measured with the same heritability, one controlled by fewer loci will have on average a larger proportion of variance explained by each locus. Power to detect an association at a particular locus is positively related to the proportion of phenotypic variance explained by that locus (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Visscher 2008</a>). Thus, phenotype-genotype associations for characters controlled by a large number of small-effect loci tend to be more difficult to detect.</p><p id="p-7">Increased throughput of image-based phenotyping methods can make it possible to collect measurements of more traits on more individuals than by manual measurement, which makes image-based phenotyping an attractive way to generate phenotypic data for GWAS. We can consider the manual measurement and the image-based measurement of an individual character to be two traits with differing heritability but the same exact set of causative loci. As in the NIRS example above, researchers may sometimes prefer a less precise method for measuring a character because it is cheaper, faster, or otherwise more desirable. It is unclear just how much loss in precision (decrease in heritability) can occur before GWAS results begin to suffer. Part of the answer to this question lies in the goals and risk tolerance of the researcher: if the goal of the experiment requires identification of few, strong signals then perhaps lower power to detect associations will still be tolerable; if instead the goal is to identify many small-effect loci, then even small reductions in heritability could negatively impact the outcome of the study. In this experiment, we use simulations to investigate the relationship between trait heritability and the ability to detect genetic regions associated with a character. We use receiver operating characteristic (ROC) curves to characterize detection of causative loci. Previous studies have used ROC curves or similar visual aids to evaluate the efficacy of different GWAS methods (<em>e.g.</em>, <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Wang <em>et al.</em> 2014</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Liu <em>et al.</em> 2016</a>). In this study, however, we use ROC curves to evaluate GWAS results for simulated traits, and test for differences between those ROC curves using the area under the curve (AUC). Using AUC to distill GWAS results to a single statistic enables direct comparison of GWAS results from traits with differing simulation parameters. We utilize a method that leverages generalized U-statistic theory for testing the null hypothesis that the AUCs of two correlated ROC curves, representing two distinct measurements of the same character, are the same. This test statistic is used to construct an empirical null distribution from traits simulated with the same parameters, and the distribution is used to predict whether real traits measured manually and by image analysis, which are expected to have different heritabilities, have significantly different AUCs. These results provide a foundational framework for evaluating how differences in heritability between two measures of a character can impact the efficacy of GWAS for identifying loci associated with the character of interest.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Plant populations: phenotyping and genotyping</h3><p id="p-8">The Wisconsin Diversity panel (WiDiv-942) is a set of 942 inbred maize lines that reach grain physiological maturity in the upper Midwest region of the United States (M. Mazaheri, M. Heckwolf, B. Vaillancourt, J. Gage, B. Burdo, S. Heckwolf, K. Barry, A. Lipzen, C. Bastos Riviero, T. Kono, H. Kaeppler, E. Spalding, C. Hirsch, C.R. Buell, N. de Leon, and S. Kaeppler, unpublished data), and represents an expanded version of the 503 line diversity panel described by Hirsch and colleagues (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Hirsch <em>et al.</em> 2014</a>). Phenotypic measurements of tassel morphology in the WiDiv-942 were performed using both manual and image-based measurements. Manual and image-based measurements are described in detail in (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Gage <em>et al.</em> 2018</a>). Briefly, manual measurements included tassel length (TL), the distance (cm) from the lowest tassel branch to the tip of the tassel spike; spike length (SL), the distance (cm) from the uppermost tassel branch to the tip of the tassel spike; branch number (BN), the total number of primary tassel branches; and tassel weight (TW), the weight (g) of the dried tassel biomass above and including the lowest branch. Image-based measurements were made using the output of the tassel phenotyping software TIPS (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Gage <em>et al.</em> 2017</a>). The data output by TIPS are image-based measurements of tassel morphology and were used as explanatory variables in a partial least squares regression model that performs image-based predictions of TL, SL, BN, and TW (referred to as TLp, SLp, BNp, and TWp, respectively) (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Gage <em>et al.</em> 2018</a>). The WiDiv-942 was grown using a replicated complete block design in three different environments: the University of Wisconsin Arlington Agricultural Research Station in the summers of 2013 and 2014, and the University of Wisconsin West Madison Agricultural Research Station in the summer of 2015. TL, SL, and BN were measured in all three environments, while TW, TLp, SLp, BNp, and TWp were measured only in 2015. Best linear unbiased predictors (BLUPs) for each inbred line for all eight traits were calculated using random effects models to account for environment, genotype-by-environment, and replication effects (<a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Gage <em>et al.</em> 2018</a>).</p><p id="p-9">The WiDiv-942 was genotyped at 899,784 SNPs discovered by RNA sequencing (Mazaheri <em>et al.</em>, unpublished data). SNP data contained 30% missing data, which were imputed using fastPHASE (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Scheet and Stephens 2006</a>). After imputation, 0.3% of SNP calls were missing, due to inability of the imputation program to call all missing SNPs. The remaining missing data at any given SNP were imputed to the mean value for that SNP. SNPs with a minor allele frequency of &lt;0.02 were removed from the genotypic data, leaving 529,018 remaining SNPs.</p></div><div id="sec-3" class="subsection"><h3>Simulated phenotypes</h3><p id="p-10">Heritability (<span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3715/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3715/embed/mml-math-1.gif"/></noscript></span></span>) is the ratio of genetic variability <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3715/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3715/embed/mml-math-2.gif"/></noscript></span></span>in a population to overall phenotypic variability <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3715/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3715/embed/mml-math-3.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3715/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3715/embed/mml-math-4.gif"/></noscript></span></span> is all variance not attributed to differences between genotypes. Thus, <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3715/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3715/embed/mml-math-5.gif"/></noscript></span></span> is a measure of the strength of genetic signal in a particular population for a particular phenotype. Two different measures of the same character, for instance, TL and TLp, can be thought of as correlated traits with the same underlying genetic control but differing heritabilities. The true value of the character for a particular individual cannot be measured perfectly, but both the manual and image-based measurements of the character represent a combination of true signal and some (differing) amount of measurement error.</p><p id="p-11">To simulate traits with similar behavior, effect sizes were randomly drawn from a normal distribution for a set of randomly chosen SNPs genotyped in the WiDiv-942, and the ‘true’ phenotypic value was calculated for each of the 942 individuals. In a second step, noise was added to each individual’s true value in order to attain a desired heritability. We varied both the number of causative loci (NCL), to simulate traits controlled by differing numbers of variants, and the heritability of simulated traits. The NCL was set to 10, 100, and 1000, and <em>h</em><sup>2</sup> ranged from 0.1 to 0.9 in increments of 0.1. The causative loci were randomly selected a single time from all SNPs genotyped in the WiDiv-942. The phenotype for an individual <em>i</em> for a trait controlled by NCL = <em>n</em> SNPs and heritability <em>h</em><sup>2</sup> is <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3715/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3715/embed/mml-math-6.gif"/></noscript></span></span>. The standardized genotypic value, <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3715/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3715/embed/mml-math-7.gif"/></noscript></span></span> is the genotype <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3715/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3715/embed/mml-math-8.gif"/></noscript></span></span> of individual <em>i</em> at SNP <em>l</em>, expressed as 0, 1, or 2 copies of the major allele, centered by twice the major allele frequency, <em>p</em>, and divided by the standard deviation of the SNP. The allelic effect <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3715/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3715/embed/mml-math-9.gif"/></noscript></span></span> was drawn from <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3715/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3715/embed/mml-math-10.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3715/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3715/embed/mml-math-11.gif"/></noscript></span></span> from <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3715/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3715/embed/mml-math-12.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3715/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3715/embed/mml-math-13.gif"/></noscript></span></span>. The variance for allelic effects was set arbitrarily, but could be any reasonable number as the error variance is modified to ensure the desired heritability. Genotypic variance <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3715/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3715/embed/mml-math-14.gif"/></noscript></span></span> was calculated simply as the sample variance of the population’s true phenotypic values. For each combination of <em>n</em> and <em>h</em><sup>2</sup>, phenotypes were simulated 10 times. The difference between each simulation with the same set of parameters is simply the <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3715/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3715/embed/mml-math-15.gif"/></noscript></span></span>, representing random draws from the same distribution each time. In total, 270 traits were simulated: pairwise combinations of 3 levels of <em>n</em> and 9 levels of <em>h</em><sup>2</sup>, each replicated 10 times.</p></div><div id="sec-4" class="subsection"><h3>Genome-wide association studies</h3><p id="p-12">Genome-wide association studies (GWAS) were conducted with the software GAPIT (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Lipka <em>et al.</em> 2012</a>), implemented in R (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">R Core Team 2016</a>). The same kinship matrix, calculated from 10,000 randomly selected SNPs by the VanRaden method in GAPIT (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">VanRaden 2008</a>; <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Lipka <em>et al.</em> 2012</a>), was used for all GWAS and no other covariates (<em>e.g.</em>, principal component scores) were included in the model. Compression was turned off by setting the group.to and group.from parameters to 9999. GWAS was run separately for each of the 270 simulated traits.</p></div><div id="sec-5" class="subsection"><h3>Receiver operator characteristic curves</h3><p id="p-13">Receiver operator characteristic (ROC) curves are typically used to assess the ability of a particular method for identifying the true, binary status (case or control) of an individual based on various threshold levels of a continuous predictor variable. The <em>roc()</em> function in the R package pROC (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Robin <em>et al.</em> 2013</a>) was used to construct ROC curves from GWAS results by considering each SNP as an ‘individual’ and coding the randomly selected causative SNPs as cases in the response, while all other SNPs were considered controls. The –log<sub>10</sub>(p-value) of each SNP was used as the predictor variable. A separate empirical ROC curve was fitted to the GWAS results for each of the simulated traits. A representative ROC curve was estimated from all 10 replicates of each parameter combination by calculating the mean of the sensitivities and specificities along the curve.</p><p id="p-14">The ROC curves for different traits can be compared quantitatively by testing for differences in their AUCs. The GWAS ROC curves described in the preceding section are not independent of each other, because for each comparison the causative and non-causative loci are identical. Some of the original methods for testing differences between non-independent ROC curves relied on the assumption that the populations being compared by the ROC curve were both normally distributed (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Metz <em>et al.</em> 1984</a>). However, the assumption of normality is not met by the distribution of –log<sub>10</sub>(p-values) from GWAS, so a nonparametric approach is needed instead.</p><p id="p-15">Fortunately, the empirical AUC of an ROC curve is equivalent to the statistic generated by a Mann-Whitney test on the predictor scores of the case and control groups (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Bamber 1975</a>). The Mann-Whitney test is a nonparametric test of the null hypothesis that two samples have the same distribution, because they were either drawn from the same population or from two populations with the same underlying distribution. The test relies on the assumptions that the samples are random and independent. With the additional assumption that the two sampled populations have the same shaped distribution, the Mann-Whitney test becomes a test of the null hypothesis that both samples have the same median (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Upton and Cook 2008</a>).</p><p id="p-16">The Mann-Whitney statistic belongs to a class of unbiased statistics called U-statistics. Because AUCs are equivalent to Mann-Whitney test statistics, general U-statistic theory can be used to compare AUCs to each other. <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">DeLong <em>et al.</em> (1988)</a> describe methods that leverage U-statistic theory to test contrasts between correlated AUCs. This methodology can be used to test the null hypothesis that the AUCs of two ROC curves are equal. The test statistic described by <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">DeLong <em>et al.</em> (1988)</a>, under the null hypothesis and assumption of independence, is called ‘Z’ and is asymptotically normal with mean zero and variance that is a function of the sample sizes.</p><p id="p-17">In the context of the AUCs from our GWAS of simulated traits, the AUC is theoretically equivalent to a Mann-Whitney statistic where the two samples being compared are the p-values of causative and non-causative loci. This equivalency holds even when the two samples being compared are in violation of the assumption that they follow the same distribution (which is the case for p-values of causative and non-causative loci), as long as the sample sizes are large enough (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Bamber 1975</a>). For AUCs that are not close to zero or one, sample sizes in the high single digits to double digits appear sufficient (<a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">Bamber 1975</a>). One caveat associated with treating the AUCs as Mann-Whitney statistics is that the GWAS p-values of causative and non-causative loci are not independent between or within samples; SNPs in LD with each other are likely to have similar p-values. Because the GWAS results from this study violate the assumptions of the Mann-Whitney test, we still calculate and use the Z statistic to test for differences between AUCs, but do not make the assumption that the distribution of Z is normal. Rather, we assume that ROC curves created from GWAS results on traits with the same parameters (<em>i.e.</em>, NCL and heritability) should not have significantly different AUCs. Therefore, the empirical distribution of Z when the difference in heritability (D) between two traits equals zero is used as the distribution of Z under the null hypothesis that the AUCs of two ROC curves are the same. Thresholds for significance at α=0.05 were calculated empirically from the 2.5<sup>th</sup> and 97.5<sup>th</sup> percentiles of the null distribution of Z.</p><p id="p-18">Z-scores were calculated for tests of the difference between all pairwise combinations of ROC curves with the same NCL using the <em>roc.test()</em> function in pROC with the method argument set to ‘delong’ (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Robin <em>et al.</em> 2013</a>).</p></div><div id="sec-6" class="subsection"><h3>Estimating Z scores and confidence intervals for real phenotypes</h3><p id="p-19">Because the relationship between Z and D for pairs of simulated traits was approximately linear, we fit a simple linear regression of Z against D separately for each value of NCL. Using the differences between estimated heritability for each pair of real traits (TL and TLp; SL and SLp; BN and BNp; TW and TWp), we predicted the value of Z for each character at different NCL using the fitted regressions. Using the thresholds for significance derived from the empirical distribution of Z when D = 0, we tested the null hypothesis that the two traits, measured by different methods, have the same AUC for each character at each NCL.</p></div><div id="sec-7" class="subsection"><h3>Data availability</h3><p id="p-20">Genotypic data are available from doi: 10.5061/dryad.6tg35t6. Phenotypic data for tassel traits are available in Supplemental File S3 from <a id="xref-ref-10-4" class="xref-bibr" href="#ref-10">Gage <em>et al.</em> (2018)</a>. Scripts for analysis can be found at github.com/joegage/GWAS_AUC. Supplemental material available at Figshare: <a href="https://doi.org/10.25386/genetics.7097813">https://doi.org/10.25386/genetics.7097813</a>.</p></div></div><div class="section results" id="sec-8"><h2 class="">Results</h2><div id="sec-9" class="subsection"><h3>Manual and image-based phenotypes are highly correlated</h3><p id="p-21">Empirical phenotypic data used in this study comes from morphological measurements of the male inflorescence of maize, known as the tassel. Four tassel morphological characters were measured manually and by image analysis in the Wisconsin Diversity panel, a population of 942 diverse inbred maize lines (WiDiv-942). The manually measured traits were tassel length (TL), spike length (SL), branch number (BN), and tassel weight (TW), and their image-based counterparts are referred to as TLp, SLp, BNp, and TWp, respectively. TL, SL, and BN were measured in replicated experiments over three years, while the other five traits were measured in a replicated experiment in one year. Best linear unbiased predictors (BLUPs) for manually measured traits are highly correlated with BLUPs for the corresponding image-based traits, with Pearson’s correlation coefficients ranging from 0.81 to 0.9 (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>) (<a id="xref-ref-10-5" class="xref-bibr" href="#ref-10">Gage <em>et al.</em> 2018</a>). Estimated heritability for the traits ranges from 0.95 to 0.97 for manually measured traits and from 0.79 to 0.86 for image-based traits (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>) (<a id="xref-ref-10-6" class="xref-bibr" href="#ref-10">Gage <em>et al.</em> 2018</a>). TL, SL, and BN were measured in three environments, and all image-based traits as well as TW were measured in a single environment. The smaller number of environments could cause lower heritability estimates of the image-based traits, making it reasonable to conclude that the image-based heritability estimates represent a lower bound for their true heritabilities.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/11/3715/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Correlations between manual and image-based phenotypic values. Scatter plots of best linear unbiased predictors (BLUPs) for manual vs. image-based measurements of tassel length (TL; A), spike length (SL; B), branch number (BN; C), and tassel weight (TW; D). Manually measured BLUPs are along the x-axis, while image-based measurements are on the y-axis. Values in the upper left corner of each plot are the Pearson correlation coefficients for each trait." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-70503086" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Correlations between manual and image-based phenotypic values. Scatter plots of best linear unbiased predictors (BLUPs) for manual &lt;em&gt;vs.&lt;/em&gt; image-based measurements of tassel length (TL; A), spike length (SL; B), branch number (BN; C), and tassel weight (TW; D). Manually measured BLUPs are along the x-axis, while image-based measurements are on the y-axis. Values in the upper left corner of each plot are the Pearson correlation coefficients for each trait.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/11/3715/F1.medium.gif" width="440" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/8/11/3715/F1.medium.gif" width="440" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/11/3715/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/11/3715/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491385" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-22" class="first-child">Correlations between manual and image-based phenotypic values. Scatter plots of best linear unbiased predictors (BLUPs) for manual <em>vs.</em> image-based measurements of tassel length (TL; A), spike length (SL; B), branch number (BN; C), and tassel weight (TW; D). Manually measured BLUPs are along the x-axis, while image-based measurements are on the y-axis. Values in the upper left corner of each plot are the Pearson correlation coefficients for each trait.</p><div class="sb-div caption-clear"></div></div></div><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/491405/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/491405/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Comparison of manual and image-based trait heritabilities</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h3>Power in GWAS of simulated traits varies with heritability and number of causal loci</h3><p id="p-24">To create a framework for comparing traits measured manually and by image analysis, we first performed simulations to examine the impact of heritability and number of causative loci (NCL) on GWAS results. We first simulated a number of traits with different heritability and NCL. Phenotypes were simulated as being controlled by single nucleotide polymorphisms (SNPs) from the WiDiv-942, which was genotyped at 529,018 SNPs discovered by RNA sequencing. Simulated phenotypes were controlled by varying NCL: 10, 100, or 1,000 randomly selected SNPs were randomly assigned effect sizes drawn from a normal distribution. For each of the three values for NCL, traits were simulated with heritabilities ranging from 0.1 to 0.9 in increments of 0.1. Each combination of NCL and heritability was simulated ten times. GWAS were performed on all simulated phenotypes, and empirical ROC curves were created with the results from each GWAS. ROC curves plot the proportion of true positives (true positive rate; TPR) against the proportion of false positives (false positive rate; FPR), as the threshold for labeling an observation as positive moves from stringent to more liberal. The ROC curve for a test with very good ability to identify true positives without too many false positives will rise steeply from the origin and approach the point (0, 1), before flattening out and continuing on to the point (1,1), producing an AUC close to 1. A test that is no better than randomly guessing which observations are positives will yield an ROC curve that follows a line with slope equal to one from the origin to (1,1), producing an AUC of 0.5.</p><p id="p-25">ROC curves are typically constructed by classifying a number of individuals as either cases or controls, based on some continuous predictor variable. The TPR and FPR are calculated at different levels of the predictor variable to create the curve. To create ROC curves for GWAS results each SNP is treated as an individual, the true status of which is either causative (case) or non-causative (control). The continuous predictor variable is the –log<sub>10</sub>(p-value) for each SNP from GWAS.</p><p id="p-26">Our empirical results show that for any given NCL, simulated traits with higher heritability generally had better ROC curves, as measured by AUC (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). This was expected, as greater heritability implies greater genetic variance relative to error, which makes it easier to identify associations between phenotypic values and genotypic groups at causal loci. However, higher heritability does not guarantee better ROC curves as there are ROC curves with different heritability that intersect, particularly when the NCL is low (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/11/3715/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Receiver operating characteristic curves for GWAS results of simulated traits controlled by 10 (A), 100 (B), or 1000 (C) causal loci. For each number of causal loci, simulation of traits with heritabilities ranging from 0.1 to 0.9 were replicated 10 times each. Each curve represents the average of the ten replications for each combination of causal loci and heritability. TPR: true positive rate; FPR: false positive rate; h2: heritability." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-70503086" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Receiver operating characteristic curves for GWAS results of simulated traits controlled by 10 (A), 100 (B), or 1000 (C) causal loci. For each number of causal loci, simulation of traits with heritabilities ranging from 0.1 to 0.9 were replicated 10 times each. Each curve represents the average of the ten replications for each combination of causal loci and heritability. TPR: true positive rate; FPR: false positive rate; h&lt;sup&gt;2&lt;/sup&gt;: heritability.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/11/3715/F2.medium.gif" width="440" height="139"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/8/11/3715/F2.medium.gif" width="440" height="139"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/11/3715/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/11/3715/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491356" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-27" class="first-child">Receiver operating characteristic curves for GWAS results of simulated traits controlled by 10 (A), 100 (B), or 1000 (C) causal loci. For each number of causal loci, simulation of traits with heritabilities ranging from 0.1 to 0.9 were replicated 10 times each. Each curve represents the average of the ten replications for each combination of causal loci and heritability. TPR: true positive rate; FPR: false positive rate; h<sup>2</sup>: heritability.</p><div class="sb-div caption-clear"></div></div></div><p id="p-28">For any given heritability, the NCL plays an even larger role in the shape of the ROC curve, with traits controlled by more loci having worse ROC curves than those with fewer loci (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>). This result was also expected. The effect sizes of individual loci become smaller as NCL increases, making detection of true associations more difficult.</p></div><div id="sec-11" class="subsection"><h3>Heritability and number of causal loci influence ability to detect differences between ROC curves</h3><p id="p-29">In order to evaluate whether the difference in heritability (D) between two traits has a significant effect on GWAS results, we used a nonparametric method for testing the null hypothesis that two correlated ROC curves have the same AUC (<a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">DeLong <em>et al.</em> 1988</a>). We hypothesized that as D between two traits increased so would the test statistic, Z, corresponding to a difference between AUCs of the two traits. We tested all 90 AUCs with the same NCL (10 replications times nine levels of heritability) against each other, resulting in <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3715/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11/3715/embed/mml-math-16.gif"/></noscript></span></span> test statistics for each NCL (comparisons between each pair of traits were only made once). The goal of this study is to assess how differing heritability of two measurements of the same underlying character affects GWAS results. Because manual and image-based measurements of a character have equivalent underlying genetic structure, we limited our comparisons to AUCs of simulated traits with the same NCL.</p><p id="p-30">The Z scores for each pairwise test of two traits were regressed against D (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). As expected, the Z values get more extreme as D gets larger – this is a reflection of higher heritability traits generally having larger AUCs than lower heritability traits. Note that the tests were always done in a consistent direction; therefore we mostly observed results with positive Z scores. The relationship between D and Z is more extreme for simulated traits with greater NCL. Practically, this indicates that within the assumptions of these simulations, heritability plays a smaller role in the ability to detect GWAS associations when the trait is controlled by a small NCL. ROC curves for traits with more complex genetic architectures, however, deteriorate more quickly as heritability declines.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/11/3715/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Results of testing area under the curve for simulated phenotypes. The Z score for testing the difference of two AUCs is plotted against the absolute difference in heritability (D) between the two traits. Traits are controlled by 10 (A), 100 (B), or 1000 (C) causal loci. Small gray dots represent the Z score from a single pairwise test between simulated traits, while horizontal gray bars represent the median Z score for a given D. Larger colored dots represent the D estimates for real traits, plotted along the line that best fits the Z scores of the simulated data. Dashed lines represent the thresholds for significance at &#x3B1;=0.05 (i.e., 2.5th and 97.5th percentiles), calculated from the empirical distribution of Z when D = 0. BN: branch number; SL: spike length; TL: tassel length; TW: tassel weight." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-70503086" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Results of testing area under the curve for simulated phenotypes. The Z score for testing the difference of two AUCs is plotted against the absolute difference in heritability (D) between the two traits. Traits are controlled by 10 (A), 100 (B), or 1000 (C) causal loci. Small gray dots represent the Z score from a single pairwise test between simulated traits, while horizontal gray bars represent the median Z score for a given D. Larger colored dots represent the D estimates for real traits, plotted along the line that best fits the Z scores of the simulated data. Dashed lines represent the thresholds for significance at &#x3B1;=0.05 (&lt;em&gt;i.e.&lt;/em&gt;, 2.5&lt;sup&gt;th&lt;/sup&gt; and 97.5&lt;sup&gt;th&lt;/sup&gt; percentiles), calculated from the empirical distribution of Z when D = 0. BN: branch number; SL: spike length; TL: tassel length; TW: tassel weight.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/11/3715/F3.medium.gif" width="440" height="368"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/8/11/3715/F3.medium.gif" width="440" height="368"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/11/3715/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/11/3715/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491291" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-31" class="first-child">Results of testing area under the curve for simulated phenotypes. The Z score for testing the difference of two AUCs is plotted against the absolute difference in heritability (D) between the two traits. Traits are controlled by 10 (A), 100 (B), or 1000 (C) causal loci. Small gray dots represent the Z score from a single pairwise test between simulated traits, while horizontal gray bars represent the median Z score for a given D. Larger colored dots represent the D estimates for real traits, plotted along the line that best fits the Z scores of the simulated data. Dashed lines represent the thresholds for significance at α=0.05 (<em>i.e.</em>, 2.5<sup>th</sup> and 97.5<sup>th</sup> percentiles), calculated from the empirical distribution of Z when D = 0. BN: branch number; SL: spike length; TL: tassel length; TW: tassel weight.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h3>Alternative measurements of real phenotypes are not predicted to have differing AUCs</h3><p id="p-32">Having established an empirical relationship between Z and D for different NCL, we then used the results from our simulations to predict whether there is a significant difference between the AUCs of manual and image-based traits of a real character. We used the estimated heritabilities of manual and image-based measurements of TL, SL, BN, and TW to predict whether AUCs for the two measurement methods will be significantly different. We fit a regression between Z scores and D values for each different NCL and used that regression to obtain a predicted value of Z for each trait pair based on D estimates for manual measurements and image-based measurements. The estimates of D for real trait pairs were 0.1 (TW), 0.14 (BN), 0.16 (TL), and 0.17 (SL), with the manually measured trait always having higher heritability than the image-based trait (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). The predicted values of Z for each trait ranged from 0.15 to 0.26 when NCL = 10, from 0.47 to 0.75 when NCL = 100, and from 0.70 to 1.10 when NCL = 1,000 (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>). For each NCL we used the distribution of Z scores when D = 0 and set the 2.5<sup>th</sup> and 97.5<sup>th</sup> percentiles as thresholds for significance to test the null hypothesis that two traits have the same AUC at α=0.05. The thresholds were (-1.26, 1.09) for NCL = 10, (-1.49, 1.25) for NCL = 100, and (-1.44, 1.60) for NCL = 1,000. Regardless of NCL, predictions of Z for all four tassel traits fall within the thresholds for significance (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>). Therefore, under the assumptions made in these simulations the manual and image-based measurements are expected to have AUCs that are not significantly different from each other.</p></div></div><div class="section discussion" id="sec-13"><h2 class="">Discussion</h2><p id="p-33">In this study, we use AUCs of ROC curves constructed from GWAS results of simulated traits to test for significant differences in the ability to detect common genetic signal underlying traits with differing heritability. Our results show that as D increases, the test for differences between the traits’ AUCs becomes more significant. Though there is a strong relationship between D and Z, there is also substantial variability for Z scores at a given value of D. We predicted Z scores for real tassel morphological traits using the relationship between D and Z of simulated traits. Because each tassel trait was measured by manual and image-based methods, we predicted Z using D from the estimated heritabilities of the two measurement methods. Regardless of NCL, the predicted values of Z for real tassel traits were within the thresholds for significance that were calculated from the null distribution of Z. Based on these results, we conclude that there is unlikely to be a significant difference between AUCs of measurements made by different methods for any of the four tassel morphological phenotypes studied.</p><p id="p-34">This conclusion is highly dependent on the assumptions made when creating simulated traits and performing subsequent GWAS. Here, we assumed independent and randomly positioned causal SNPs, when in fact quantitative traits can be controlled in part by numerous causal variants clustered on the same locus (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Lango Allen <em>et al.</em> 2010</a>). Additionally, by choosing causal SNPs directly from the WiDiv-942 genotypic data, we are assuming that the causal variants are SNPs that are part of our genotypic data. The 529,018 SNPs used in this study are a small sample relative to the 60 million variants identified in the maize HapMap3 (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bukowski <em>et al.</em> 2015</a>), and do not include other variant types that can affect quantitative traits such as insertion/deletion and copy number variants. By selecting causative SNPs from our genotyped SNPs, we make the true associations easier to find by GWAS. In reality, causative variants may not be genotyped and therefore can only be identified by linkage disequilibrium with genotyped SNPs. For simplicity’s sake, we drew the simulated effect sizes at each causal SNP from a normal distribution. Previous work by <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Hayes and Goddard (2001)</a> has shown that quantitative traits in livestock appear to follow a gamma distribution with a large number of very small effect loci; they posit that there may be even more small effect loci than predicted by their distributions. This idea can be seen in its most extreme form in the omnigenic model proposed by Boyle <em>et al.</em> (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Boyle <em>et al.</em> 2017</a>) which is based on Fisher’s infinitesimal model (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Fisher 1919</a>). By drawing our effect sizes from a normal distribution, we may be creating more large-effect variants than is realistic, therefore increasing our ability to detect causal variants by GWAS.</p><p id="p-35">Though our choices for location and effect size of causative SNPs may be increasing the probability of detecting associations, we also assume that only identifying an exact chosen causative SNP counts as a true positive. In reality, identifying associations with SNPs that are within the same gene as, or a small distance away from, the true causative SNP may be close enough. GWAS often serves as an initial sweep to find regions of interest for further study, and associations that lead to fruitful downstream analysis may still be considered a ‘success’. This is reflected in software that calculate power of GWAS by considering associations within a certain distance of the causative variant to be true positives (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Liu <em>et al.</em> 2016</a>). By only considering the exact causative SNPs as true positives we make the true positives more difficult to identify, partially counteracting the assumptions above that make the detection of associations easier.</p><p id="p-36">By trying to predict the Z score for the difference between AUCs of real traits, we also make some assumptions about the tassel morphological traits that we use. Our estimates of heritability are not exact; they are population- and experiment-specific. Because the image-based traits and TW were measured in one environment, whereas TL, SL, and BN were measured in three, their estimates of heritability may have differing accuracy. By predicting Z scores for the real traits for NCL set to 10, 100, and 1,000, we were able to predict how Z scores changed as NCL changed. The true NCL for tassel morphological traits likely numbers in the hundreds or higher, with an upper limit of the total number of expressed genes (tens of thousands), as posited in the omnigenic model (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Boyle <em>et al.</em> 2017</a>). If the true NCL is greater than 1,000, the relationship between Z and D will be even steeper, meaning that the small values of D for the real tassel morphological traits may in fact result in significantly different AUCs.</p><p id="p-37">The use of AUC as a metric to quantify the success of GWAS is also accompanied by assumptions about the goals of GWAS. ROC curves, and thus the AUC, consider both power and type I error, as measured by true and false positive rates. Depending on the goals of the GWAS study, power and type I error may not both be of equal importance. For genomic prediction or marker assisted selection, a high type I error rate is not particularly concerning as long as power is high and trait prediction is accurate. On the other hand, studies using GWAS to choose genes for further molecular characterization have a large financial incentive to minimize type I error.</p><p id="p-38">Using AUC to quantify the effectiveness of GWAS assumes that the entire ROC curve is of interest. When NCL is low this assumption may be true, but as NCL increases, it may be the case that only the beginning of the ROC curve is of practical interest. The simulated ROC curves for NCL = 1,000 (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2C</a>) are close to the 1:1 line that would be achieved by randomly selecting SNPs as putatively causative. It is unlikely that a researcher would want or expect to identify every single causative locus when a trait is controlled by thousands of genes. Instead, the interest is often in large-effect loci that are likely to be identified by a stringent significance threshold. The ability to identify the most significant loci is characterized by the portion of the ROC curve close to the origin. Thus, for highly complex traits, the AUC of a partial ROC may be more informative.</p><p id="p-39">In this study, we use AUC of ROC curves to characterize and quantitatively compare GWAS results from different traits. Overall, our findings show an expected relationship between NCL, heritability, and AUC of ROC curves. Greater NCL and lower heritability both reduce the AUC, while lower NCL and higher heritability can increase AUC. Results suggest that there is no significant difference between AUCs from GWAS using manual and image-based measurements of typical maize tassel characters. Creation of more nuanced simulation models and consideration of partial ROC curves may enable improvement upon the results presented in this study. Additionally, these results are specific to additive traits in diversity panels of inbred lines. Further studies are required to determine whether different population structures and/or incorporation of non-additive effects to the simulated traits will change the relationship between heritability and AUC. The results presented here provide a foundational framework that may facilitate decision-making for researchers weighing the benefits of different phenotyping methods.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-40">Funding: JG is supported by the National Research Initiative for Agriculture and Food Research Initiative Competitive Grants Program grant no. # 2012-67013-19460 from the USDA National Institute of Food and Agriculture, as well as the Irwin Smith Wisconsin Distinguished Graduate Fellowship. This work was also supported by the USDA National Institute of Food and Agriculture, Hatch projects WIS01923 and WIS02020. This research was funded in part by the DOE Great Lakes Bioenergy Research Center (DOE BER Office of Science DE-FC02-07ER64494). Work conducted by the U.S. Department of Energy Joint Genome Institute, a DOE Office of Science User Facility, is supported by the Office of Science of the U.S. Department of Energy under Contract No. DE-AC02-05CH11231.</p><p id="p-41">Authors’ contributions: All authors contributed ideas, and all authors were involved in writing, editing, and approving the final manuscript.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-2"><p id="p-42">Supplemental material available at Figshare: <a href="https://doi.org/10.25386/genetics.7097813">https://doi.org/10.25386/genetics.7097813</a>.</p></li><li class="fn" id="fn-3"><p id="p-43"><em>Communicating editor: R. W. C. Doerge</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2018-08-26"><span class="received-label">Received </span>August 26, 2018.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2018-09-24"><span class="accepted-label">Accepted </span>September 24, 2018.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2018 Gage <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3715.1" data-doi="10.1016/0022-2496(75)90001-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bamber</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1975</span> <span class="cit-article-title">The area above the ordinal dominance graph and the area below the receiver operating characteristic graph.</span> <abbr class="cit-jnl-abbrev">J. Math. Psychol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">387</span>–<span class="cit-lpage">415</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/0022-2496(75)90001-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BMath.%2BPsychol.%26rft.volume%253D12%26rft.spage%253D387%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0022-2496%252875%252990001-2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0022-2496(75)90001-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1975AU71300001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3715.2" data-doi="10.1080/07352681003617285"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bock</span>  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poole</span>  <span class="cit-name-given-names">G. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parker</span>  <span class="cit-name-given-names">P. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gottwald</span>  <span class="cit-name-given-names">T. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Plant disease severity estimated visually, by digital photography and image analysis, and by hyperspectral imaging. CRC.</span> <abbr class="cit-jnl-abbrev">Crit. Rev. Plant Sci.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">59</span>–<span class="cit-lpage">107</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1080/07352681003617285</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrit.%2BRev.%2BPlant%2BSci.%26rft.volume%253D29%26rft.spage%253D59%26rft.atitle%253DPLANT%2BDISEASE%2BSEVERITY%2BESTIMATED%2BVISUALLY%252C%2BBY%2BDIGITAL%2BPHOTOGRAPHY%2BAND%2BIMAGE%2BANALYSIS%252C%2BAND%2BBY%2BHYPERSPECTRAL%2BIMAGING%26rft_id%253Dinfo%253Adoi%252F10.1080%252F07352681003617285%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/07352681003617285&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3715.3" data-doi="10.1016/j.cell.2017.05.038"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boyle</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">J. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">An Expanded View of Complex Traits: From Polygenic to Omnigenic.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">169</span>: <span class="cit-fpage">1177</span>–<span class="cit-lpage">1186</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.cell.2017.05.038</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D169%26rft.spage%253D1177%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2017.05.038%26rft_id%253Dinfo%253Apmid%252F28622505%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2017.05.038&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28622505&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3715.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3715.4" data-doi="10.1093/gigascience/gix134"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bukowski</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zou</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Construction of the third generation Zea mays haplotype map.</span> <abbr class="cit-jnl-abbrev">Gigascience</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">12</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/gigascience/gix134</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGigascience%26rft.volume%253D7%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgigascience%252Fgix134%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/gigascience/gix134&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3715.5" data-doi="10.1104/pp.113.229708"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chitwood</span>  <span class="cit-name-given-names">D. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ranjan</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martinez</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Headland</span>  <span class="cit-name-given-names">L. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thiem</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A Modern Ampelography: A Genetic Basis for Leaf Shape and Venation Patterning in Grape.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">164</span>: <span class="cit-fpage">259</span>–<span class="cit-lpage">272</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1104/pp.113.229708</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.113.229708%26rft_id%253Dinfo%253Apmid%252F24285849%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo5OiIxNjQvMS8yNTkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzExLzM3MTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3715.6" data-doi="10.2307/2531595"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">DeLong</span>  <span class="cit-name-given-names">E. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">DeLong</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clarke-Pearson</span>  <span class="cit-name-given-names">D. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">Comparing the Areas under Two or More Correlated Receiver Operating Characteristic Curves: A Nonparametric Approach.</span> <abbr class="cit-jnl-abbrev">Biometrics</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">837</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2307/2531595</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrics%26rft.stitle%253DBiometrics%26rft.aulast%253DDeLong%26rft.auinit1%253DE.%2BR.%26rft.volume%253D44%26rft.issue%253D3%26rft.spage%253D837%26rft.epage%253D845%26rft.atitle%253DComparing%2Bthe%2Bareas%2Bunder%2Btwo%2Bor%2Bmore%2Bcorrelated%2Breceiver%2Boperating%2Bcharacteristic%2Bcurves%253A%2Ba%2Bnonparametric%2Bapproach.%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2531595%26rft_id%253Dinfo%253Apmid%252F3203132%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2531595&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3203132&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3715.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1988Q069100016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3715.7" data-doi="10.1104/pp.109.145292"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Durham Brooks</span>  <span class="cit-name-given-names">T. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">N. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spalding</span>  <span class="cit-name-given-names">E. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Plasticity of Arabidopsis Root Gravitropism throughout a Multidimensional Condition Space Quantified by Automated Image Analysis.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">152</span>: <span class="cit-fpage">206</span>–<span class="cit-lpage">216</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1104/pp.109.145292</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.109.145292%26rft_id%253Dinfo%253Apmid%252F19923240%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo5OiIxNTIvMS8yMDYiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzExLzM3MTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3715.8" data-doi="10.1017/S0080456800012163"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fisher</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1919</span> <span class="cit-article-title">The Correlation between Relatives on the Supposition of Mendelian Inheritance.</span> <abbr class="cit-jnl-abbrev">Trans. R. Soc. Edinb.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">399</span>–<span class="cit-lpage">433</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S0080456800012163</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrans.%2BR.%2BSoc.%2BEdinb.%26rft.volume%253D52%26rft.spage%253D399%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0080456800012163%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0080456800012163&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3715.9" data-doi="10.1186/s13007-017-0172-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gage</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">N. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spalding</span>  <span class="cit-name-given-names">E. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaeppler</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Leon</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">TIPS: a system for automated image-based phenotyping of maize tassels.</span> <abbr class="cit-jnl-abbrev">Plant Methods</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">21</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s13007-017-0172-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BMethods%26rft.volume%253D13%26rft.spage%253D21%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13007-017-0172-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13007-017-0172-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3715.10" data-doi="10.1534/genetics.118.301487"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gage</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">White</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Edwards</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaeppler</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Leon</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Selection signatures underlying dramatic male inflorescence transformation during modern hybrid maize breeding.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">210</span>: <span class="cit-fpage">1125</span>–<span class="cit-lpage">1138</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.118.301487</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.118.301487%26rft_id%253Dinfo%253Apmid%252F30257936%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIxMC8zLzExMjUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzExLzM3MTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3715.11" data-doi="10.1051/gse:2001117"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">The distribution of the effects of genes affecting quantitative traits in livestock.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">209</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1297-9686-33-3-209</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DHayes%26rft.auinit1%253DB.%26rft.volume%253D33%26rft.issue%253D3%26rft.spage%253D209%26rft.epage%253D229%26rft.atitle%253DThe%2Bdistribution%2Bof%2Bthe%2Beffects%2Bof%2Bgenes%2Baffecting%2Bquantitative%2Btraits%2Bin%2Blivestock.%26rft_id%253Dinfo%253Adoi%252F10.1051%252Fgse%253A2001117%26rft_id%253Dinfo%253Apmid%252F11403745%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1051/gse:2001117&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11403745&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3715.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000169502100001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3715.12" data-doi="10.1105/tpc.113.119982"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hirsch</span>  <span class="cit-name-given-names">C. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foerster</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sekhon</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muttoni</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Insights into the maize pan-genome and pan-transcriptome.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">121</span>–<span class="cit-lpage">135</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1105/tpc.113.119982</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.113.119982%26rft_id%253Dinfo%253Apmid%252F24488960%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjg6IjI2LzEvMTIxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMS8zNzE1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3715.13" data-doi="10.1038/nature09410"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lango Allen</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Estrada</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lettre</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berndt</span>  <span class="cit-name-given-names">S. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weedon</span>  <span class="cit-name-given-names">M. N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Hundreds of variants clustered in genomic loci and biological pathways affect human height.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">467</span>: <span class="cit-fpage">832</span>–<span class="cit-lpage">838</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature09410</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLango%2BAllen%26rft.auinit1%253DH.%26rft.volume%253D467%26rft.issue%253D7317%26rft.spage%253D832%26rft.epage%253D838%26rft.atitle%253DHundreds%2Bof%2Bvariants%2Bclustered%2Bin%2Bgenomic%2Bloci%2Band%2Bbiological%2Bpathways%2Baffect%2Bhuman%2Bheight.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09410%26rft_id%253Dinfo%253Apmid%252F20881960%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature09410&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20881960&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3715.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000282898700065&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3715.14" data-doi="10.3732/ajb.1700046"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Duncan</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Topp</span>  <span class="cit-name-given-names">C. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chitwood</span>  <span class="cit-name-given-names">D. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Persistent homology and the branching topologies of plants.</span> <abbr class="cit-jnl-abbrev">Am. J. Bot.</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">349</span>–<span class="cit-lpage">353</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3732/ajb.1700046</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BBot.%26rft_id%253Dinfo%253Adoi%252F10.3732%252Fajb.1700046%26rft_id%253Dinfo%253Apmid%252F28341629%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiYW1qYm90IjtzOjU6InJlc2lkIjtzOjk6IjEwNC8zLzM0OSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzgvMTEvMzcxNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3715.15" data-doi="10.1093/bioinformatics/bts444"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lipka</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tian</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peiffer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">GAPIT: Genome association and prediction integrated tool.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">2397</span>–<span class="cit-lpage">2399</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/bts444</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts444%26rft_id%253Dinfo%253Apmid%252F22796960%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bts444&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22796960&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3715.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000308532300066&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3715.16" data-doi="10.1371/journal.pgen.1005767"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fan</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buckler</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Iterative Usage of Fixed and Random Effect Models for Powerful and Efficient Genome- Wide Association Studies.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">e1005767</span> <span class="cit-comment">(correction: PLoS Genet. 12: e1005957)</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1005767</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D12%26rft.spage%253De1005767%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1005767%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1005767&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3715.17" data-doi="10.1039/c2ib20020a"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Men</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A high-throughput imaging system to quantitatively analyze the growth dynamics of plant seedlings.</span> <abbr class="cit-jnl-abbrev">Integr. Biol.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">945</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1039/c2ib20020a</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DIntegr.%2BBiol.%26rft.volume%253D4%26rft.spage%253D945%26rft_id%253Dinfo%253Adoi%252F10.1039%252Fc2ib20020a%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1039/c2ib20020a&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-book" id="cit-8.11.3715.18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Metz</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">P.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kronman</span>  <span class="cit-name-given-names">H. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title">A new approach for testing the significance of differences between ROC curves measured from correlated data</span>, pp. <span class="cit-fpage">432</span>–<span class="cit-lpage">445</span> in <span class="cit-source"><em>Information processing in medical imaging</em>.</span> <span class="cit-publ-name">Nijhoff</span>, <span class="cit-publ-loc">The Hague</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3715.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">N. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haase</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaeppler</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Leon</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> A robust, high-throughput method for computing maize ear, cob, and kernel attributes automatically from images. Plant J. <span class="cit-vol">89</span>: <span class="cit-fpage">169</span>–<span class="cit-lpage">178</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/tpj.13320</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3715.20" data-doi="10.1016/S0377-8401(97)00175-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Agnew</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gordon</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steen</span>  <span class="cit-name-given-names">R. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">The use of near infrared reflectance spectroscopy (NIRS) on undried samples of grass silage to predict chemical composition and digestibility parameters.</span> <abbr class="cit-jnl-abbrev">Anim. Feed Sci. Technol.</abbr> <span class="cit-vol">72</span>: <span class="cit-fpage">155</span>–<span class="cit-lpage">167</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0377-8401(97)00175-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnim.%2BFeed%2BSci.%2BTechnol.%26rft.volume%253D72%26rft.spage%253D155%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0377-8401%252897%252900175-2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0377-8401(97)00175-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-other" id="cit-8.11.3715.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">R Core Team</span></li></ol><cite>, <span class="cit-pub-date">2016</span> R: A Language and Environment for Statistical Computing.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-other" id="cit-8.11.3715.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Robin</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Turck</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hainard</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lisacek</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sanchez</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> Package ‘pROC.’ 2012–09–10 09:34:56 1–71.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3715.23" data-doi="10.1086/502802"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Scheet</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A fast and flexible statistical model for large-scale population genotype data: applications to inferring missing genotypes and haplotypic phase.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">78</span>: <span class="cit-fpage">629</span>–<span class="cit-lpage">644</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1086/502802</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DScheet%26rft.auinit1%253DP.%26rft.volume%253D78%26rft.issue%253D4%26rft.spage%253D629%26rft.epage%253D644%26rft.atitle%253DA%2Bfast%2Band%2Bflexible%2Bstatistical%2Bmodel%2Bfor%2Blarge-scale%2Bpopulation%2Bgenotype%2Bdata%253A%2Bapplications%2Bto%2Binferring%2Bmissing%2Bgenotypes%2Band%2Bhaplotypic%2Bphase.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F502802%26rft_id%253Dinfo%253Apmid%252F16532393%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/502802&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16532393&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3715.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000236755900009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-book" id="cit-8.11.3715.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Upton</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, and <span class="cit-auth"><span class="cit-name-surname">Cook</span>  <span class="cit-name-given-names">I.</span></span>, <span class="cit-pub-date">2008</span> <em>A Dictionary of Statistics</em>. 3rd ed., Oxford University Press, Oxford U.K. 10.1093/acref/9780199679188.001.0001</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3715.25" data-doi="10.3168/jds.2007-0980"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient methods to compute genomic predictions.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">4414</span>–<span class="cit-lpage">4423</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.2007-0980</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVanRaden%26rft.auinit1%253DP.%2BM.%26rft.volume%253D91%26rft.issue%253D11%26rft.spage%253D4414%26rft.epage%253D4423%26rft.atitle%253DEfficient%2BMethods%2Bto%2BCompute%2BGenomic%2BPredictions%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2007-0980%26rft_id%253Dinfo%253Apmid%252F18946147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2007-0980&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18946147&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3715.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260277200035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3715.26" data-doi="10.1038/ng0508-489"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Visscher</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Sizing up human height variation.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">489</span>–<span class="cit-lpage">490</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng0508-489</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DVisscher%26rft.auinit1%253DP.%2BM.%26rft.volume%253D40%26rft.issue%253D5%26rft.spage%253D489%26rft.epage%253D490%26rft.atitle%253DSizing%2Bup%2Bhuman%2Bheight%2Bvariation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng0508-489%26rft_id%253Dinfo%253Apmid%252F18443579%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng0508-489&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18443579&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3715.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000255366700003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3715.27" data-doi="10.1371/journal.pone.0107684"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tian</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pan</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buckler</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A SUPER powerful method for genome wide association study.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e107684</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0107684</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De107684%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0107684%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0107684&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/8/11/3715.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/11/3703" title="A Case Study of Genomic Instability in an Industrial Strain of Saccharomyces cerevisiae" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/11/3723" title="Characterization of Genetic and Epigenetic Variation in Sperm and Red Blood Cells from Adult Hatchery and Natural-Origin Steelhead, Oncorhynchus mykiss" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/8/11"><span>Volume 8 Issue 11, November 2018</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/8/11.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/8/11.cover-source.jpg?itok=lFO8v6bg" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 8 (11)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/11/3715.figures-only" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-leaIgBLjP1A7zUmj1aSrb7JyGO6bTMW-IUX2VZQBjYc" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/8/11/3715" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/8/11/3715" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/8/11/3715" class="active" data-icon-position="" data-hide-link-title="0">Comparing Genome-Wide Association Study Results from Different Measurements of an Underlying Phenotype</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/491242" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-2u45O_wTTZDk3f9OnBSL552mg1cJ1Q9Zy8KiExpmxmE" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/11/3715.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/8/11/3715" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-w2y-nzTxJ6sNdbKvrooB6IWaMjeoAm33KMnN85gTvLs" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/8/11/3715" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/11/3715.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="491242" id="node491242--2" data-pisa="ggg;8/11/3715" data-pisa-master="ggg;g3.118.200700" data-apath="/ggg/8/11/3715.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Comparing Genome-Wide Association Study Results from Different Measurements of an Underlying Phenotype</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-5946-4414" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Joseph L. Gage</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-7867-9058" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Natalia de Leon</span> and <span class="highwire-citation-author" data-delta="2">Murray K. Clayton</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3715-3722;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200700 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node491242--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Joseph L. Gage </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joseph%2BL.%2BGage%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gage%20JL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJoseph%2BL.%2BGage%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5946-4414" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Joseph L. Gage</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Natalia de Leon </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Natalia%2Bde%2BLeon%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=de%20Leon%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANatalia%2Bde%2BLeon%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-7867-9058" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Natalia de Leon</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:ndeleongatti@wisc.edu" class="" data-icon-position="" data-hide-link-title="0">ndeleongatti@wisc.edu</a></span></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Murray K. Clayton </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Departments of Statistics and Plant Pathology, University of Wisconsin-Madison, Madison, WI 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Murray%2BK.%2BClayton%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Clayton%20MK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMurray%2BK.%2BClayton%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/8/11/3715" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/8/11/3715&amp;title=Comparing%20Genome-Wide%20Association%20Study%20Results%20from%20Different%20Measurements%20of%20an%20Underlying%20Phenotype" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/8/11/3715&amp;title=Comparing%20Genome-Wide%20Association%20Study%20Results%20from%20Different%20Measurements%20of%20an%20Underlying%20Phenotype" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/8/11/3715&amp;title=Comparing%20Genome-Wide%20Association%20Study%20Results%20from%20Different%20Measurements%20of%20an%20Underlying%20Phenotype" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/8/11/3715&amp;text=Comparing%20Genome-Wide%20Association%20Study%20Results%20from%20Different%20Measurements%20of%20an%20Underlying%20Phenotype" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/8/11/3715&amp;title=Comparing%20Genome-Wide%20Association%20Study%20Results%20from%20Different%20Measurements%20of%20an%20Underlying%20Phenotype" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/8/11/3715&amp;t=Comparing%20Genome-Wide%20Association%20Study%20Results%20from%20Different%20Measurements%20of%20an%20Underlying%20Phenotype" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/8/11/3715&amp;title=Comparing%20Genome-Wide%20Association%20Study%20Results%20from%20Different%20Measurements%20of%20an%20Underlying%20Phenotype" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/8/11/3715&amp;title=Comparing%20Genome-Wide%20Association%20Study%20Results%20from%20Different%20Measurements%20of%20an%20Underlying%20Phenotype" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/491242/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="491242" id="node491242--3" data-pisa="ggg;8/11/3715" data-pisa-master="ggg;g3.118.200700" data-apath="/ggg/8/11/3715.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Comparing Genome-Wide Association Study Results from Different Measurements of an Underlying Phenotype</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-5946-4414" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Joseph L. Gage</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-7867-9058" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Natalia de Leon</span> and <span class="highwire-citation-author" data-delta="2">Murray K. Clayton</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3715-3722;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200700 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node491242--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Joseph L. Gage </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joseph%2BL.%2BGage%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gage%20JL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJoseph%2BL.%2BGage%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5946-4414" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Joseph L. Gage</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Natalia de Leon </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Natalia%2Bde%2BLeon%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=de%20Leon%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANatalia%2Bde%2BLeon%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-7867-9058" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Natalia de Leon</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:ndeleongatti@wisc.edu" class="" data-icon-position="" data-hide-link-title="0">ndeleongatti@wisc.edu</a></span></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Murray K. Clayton </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Departments of Statistics and Plant Pathology, University of Wisconsin-Madison, Madison, WI 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Murray%2BK.%2BClayton%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Clayton%20MK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMurray%2BK.%2BClayton%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3715&amp;rft.epage=3722&amp;rft.atitle=Comparing%20Genome-Wide%20Association%20Study%20Results%20from%20Different%20Measurements%20of%20an%20Underlying%20Phenotype&amp;rft.volume=8&amp;rft.issue=11&amp;rft.date=2018-11-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Gage%2C+Joseph+L.&amp;rft.au=de+Leon%2C+Natalia&amp;rft.au=Clayton%2C+Murray+K."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/491242/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/491242/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/491242/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/491242/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/491242/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/491242/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/491242/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/491242/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/491242/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/491242/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/491242/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/491242/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/8/11/3715" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-13" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/8/11/3715.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/8/11/3715.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/8/11/3715.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/8/11/3715.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
