<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/8/11/3583" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/8/11/3583.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/8/11/3583.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/8/11/3583.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/8/11/3583.atom" />
<meta name="HW.pisa" content="ggg;8/11/3583" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Bulked-Segregant Analysis Coupled to Whole Genome Sequencing (BSA-Seq) for Rapid Gene Cloning in Maize" />
<meta name="DC.Identifier" content="10.1534/g3.118.200499" />
<meta name="DC.Date" content="2018-11-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2018 Klein et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Forward genetics remains a powerful method for revealing the genes underpinning organismal form and function, and for revealing how these genes are tied together in gene networks. In maize, forward genetics has been tremendously successful, but the size and complexity of the maize genome made identifying mutant genes an often arduous process with traditional methods. The next generation sequencing revolution has allowed for the gene cloning process to be significantly accelerated in many organisms, even when genomes are large and complex. Here, we describe a bulked-segregant analysis sequencing (BSA-Seq) protocol for cloning mutant genes in maize. Our simple strategy can be used to quickly identify a mapping interval and candidate single nucleotide polymorphisms (SNPs) from whole genome sequencing of pooled F2 individuals. We employed this strategy to identify narrow odd dwarf as an enhancer of teosinte branched1 , and to identify a new allele of defective kernel1 . Our method provides a quick, simple way to clone genes in maize." />
<meta name="DC.Contributor" content="Harry Klein" />
<meta name="DC.Contributor" content="Yuguo Xiao" />
<meta name="DC.Contributor" content="Phillip A. Conklin" />
<meta name="DC.Contributor" content="Rajanikanth Govindarajulu" />
<meta name="DC.Contributor" content="Jacob A. Kelly" />
<meta name="DC.Contributor" content="Michael J. Scanlon" />
<meta name="DC.Contributor" content="Clinton J. Whipple" />
<meta name="DC.Contributor" content="Madelaine Bartlett" />
<meta name="article:published_time" content="2018-11-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Bulked-Segregant Analysis Coupled to Whole Genome Sequencing (BSA-Seq) for Rapid Gene Cloning in Maize" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Forward genetics remains a powerful method for revealing the genes underpinning organismal form and function, and for revealing how these genes are tied together in gene networks. In maize, forward genetics has been tremendously successful, but the size and complexity of the maize genome made identifying mutant genes an often arduous process with traditional methods. The next generation sequencing revolution has allowed for the gene cloning process to be significantly accelerated in many organisms, even when genomes are large and complex. Here, we describe a bulked-segregant analysis sequencing (BSA-Seq) protocol for cloning mutant genes in maize. Our simple strategy can be used to quickly identify a mapping interval and candidate single nucleotide polymorphisms (SNPs) from whole genome sequencing of pooled F2 individuals. We employed this strategy to identify &lt;i&gt;narrow odd dwarf&lt;/i&gt; as an enhancer of &lt;i&gt;teosinte branched1&lt;/i&gt;, and to identify a new allele of &lt;i&gt;defective kernel1&lt;/i&gt;. Our method provides a quick, simple way to clone genes in maize.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2018/11/01" />
<meta name="citation_mjid" content="ggg;8/11/3583" />
<meta name="citation_id" content="8/11/3583" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/8/11/3583" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/8/11/3583.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/8/11/3583.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/8/11/3583.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.118.200499" />
<meta name="citation_pmid" content="30194092" />
<meta name="citation_volume" content="8" />
<meta name="citation_issue" content="11" />
<meta name="citation_num_pages" content="10" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="3583" />
<meta name="citation_lastpage" content="3592" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Harry Klein" />
<meta name="citation_author_institution" content="Plant Biology Graduate Program and Biology Department, University of Massachusetts Amherst, Amherst, MA 01003" />
<meta name="citation_author" content="Yuguo Xiao" />
<meta name="citation_author_institution" content="Department of Biology, Brigham Young University, 4102 LSB, Provo, UT 84602" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-0747-5856" />
<meta name="citation_author" content="Phillip A. Conklin" />
<meta name="citation_author_institution" content="Department of Plant Biology, Cornell University, Ithaca, NY 14853" />
<meta name="citation_author" content="Rajanikanth Govindarajulu" />
<meta name="citation_author_institution" content="Department of Biology, West Virginia University, Morgantown, WV 26506" />
<meta name="citation_author" content="Jacob A. Kelly" />
<meta name="citation_author_institution" content="Department of Biology, Brigham Young University, 4102 LSB, Provo, UT 84602" />
<meta name="citation_author" content="Michael J. Scanlon" />
<meta name="citation_author_institution" content="Department of Plant Biology, Cornell University, Ithaca, NY 14853" />
<meta name="citation_author" content="Clinton J. Whipple" />
<meta name="citation_author_institution" content="Department of Biology, Brigham Young University, 4102 LSB, Provo, UT 84602" />
<meta name="citation_author" content="Madelaine Bartlett" />
<meta name="citation_author_institution" content="Plant Biology Graduate Program and Biology Department, University of Massachusetts Amherst, Amherst, MA 01003" />
<meta name="citation_author_email" content="mbartlett@bio.umass.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-0369-8606" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=A. Abe;citation_author=S. Kosugi;citation_author=K. Yoshida;citation_author=S. Natsume;citation_author=H. Takagi;citation_title=Genome sequencing reveals agronomically important loci in rice using MutMap.;citation_pages=174-178;citation_volume=30;citation_year=2012;citation_issue=2;citation_pmid=22267009;citation_doi=10.1038/nbt.2095" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=C. Addo-Quaye;citation_author=E. Buescher;citation_author=N. Best;citation_author=V. Chaikam;citation_author=I. Baxter;citation_title=Forward Genetics by Sequencing EMS Variation-Induced Inbred Lines. G3: Genes, Genomes;citation_pages=413-425;citation_volume=7;citation_year=2017;citation_doi=10.1534/g3.116.029660" />
<meta name="citation_reference" content="citation_journal_title=Nuc. acids research;citation_author=E. Afgan;citation_author=D. Baker;citation_author=B. Batut;citation_author=M. Van Den Beek;citation_author=D. Bouvier;citation_author=M. Čech;citation_title=The Galaxy platform for accessible, reproducible and collaborative biomedical analyses: 2018 update.;citation_pages=W537-W544;citation_volume=46;citation_year=2018" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=E. Afgan;citation_author=D. Baker;citation_author=M. van den Beek;citation_author=D. Blankenberg;citation_author=D. Bouvier;citation_title=The Galaxy platform for accessible, reproducible and collaborative biomedical analyses: 2016 update.;citation_volume=44;citation_year=2016;citation_pmid=27137889;citation_doi=10.1093/nar/gkw343" />
<meta name="citation_reference" content="citation_author=S. Andrews;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Development;citation_author=P. W. Becraft;citation_author=K. Li;citation_author=N. Dey;citation_author=Y. Asuncion-Crabb;citation_title=The maize dek1 gene functions in embryonic pattern formation and cell fate specification.;citation_pages=5217-5225;citation_volume=129;citation_year=2002;citation_pmid=12399313" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=D. Blankenberg;citation_author=A. Gordon;citation_author=G. Von Kuster;citation_author=N. Coraor;citation_author=J. Taylor;citation_title=Manipulation of FASTQ data with Galaxy.;citation_volume=26;citation_year=2010;citation_pmid=20562416;citation_doi=10.1093/bioinformatics/btq281" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. M. Bolger;citation_author=M. Lohse;citation_author=B. Usadel;citation_title=Trimmomatic: a flexible trimmer for Illumina sequence data.;citation_volume=30;citation_year=2014;citation_pmid=24695404;citation_doi=10.1093/bioinformatics/btu170" />
<meta name="citation_reference" content="citation_journal_title=Gigascience;citation_author=R. Bukowski;citation_author=X. Guo;citation_author=Y. Lu;citation_author=C. Zou;citation_author=B. He;citation_title=Construction of the third-generation Zea mays haplotype map.;citation_pages=1-12;citation_volume=7;citation_year=2018;citation_doi=10.1093/gigascience/gix120" />
<meta name="citation_reference" content="citation_journal_abbrev=Fly;citation_author=P. Cingolani;citation_author=A. Platts;citation_author=L. L. Wang;citation_author=M. Coon;citation_author=T. Nguyen;citation_title=A program for annotating and predicting the effects of single nucleotide polymorphisms, SnpEff: SNPs in the genome of Drosophila melanogaster strain w1118; iso-2; iso-3.;citation_pages=80-92;citation_volume=6;citation_year=2012;citation_issue=2;citation_pmid=22728672;citation_doi=10.4161/fly.19695" />
<meta name="citation_reference" content="citation_journal_title=New Phytol.;citation_author=B. Ding;citation_author=F. Mou;citation_author=W. Sun;citation_author=S. Chen;citation_author=F. Peng;citation_title=A dominant-negative actin mutation alters corolla tube width and pollinator visitation in Mimulus lewisii.;citation_pages=1936-1944;citation_volume=213;citation_year=2017;citation_doi=10.1111/nph.14281" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=J. Doebley;citation_author=A. Stec;citation_author=L. Hubbard;citation_title=The evolution of apical dominance in maize.;citation_pages=485-488;citation_volume=386;citation_year=1997;citation_issue=6624;citation_pmid=9087405;citation_doi=10.1038/386485a0" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=B. Ewing;citation_author=L. Hillier;citation_author=M. C. Wendl;citation_author=P. Green;citation_title=Base-Calling of Automated Sequencer Traces Using Phred.  I. Accuracy Assessment;citation_pages=175-185;citation_volume=8;citation_year=1998;citation_issue=3;citation_pmid=9521921;citation_doi=10.1101/gr.8.3.175" />
<meta name="citation_reference" content="citation_journal_title=Development;citation_journal_abbrev=Development;citation_author=A. Gallavotti;citation_author=J. A. Long;citation_author=S. Stanfield;citation_author=X. Yang;citation_author=D. Jackson;citation_title=The control of axillary meristem fate in the maize ramosa pathway;citation_pages=2849-2856;citation_volume=137;citation_year=2010;citation_issue=17;citation_pmid=20699296;citation_doi=10.1242/dev.051748" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=C. S. Gillmor;citation_author=A. H. K. Roeder;citation_author=P. Sieber;citation_author=C. Somerville;citation_author=W. Lukowitz;citation_title=A Genetic Screen for Mutations Affecting Cell Division in the Arabidopsis thaliana Embryo Identifies Seven Loci Required for Cytokinesis.;citation_pages=e0146492;citation_volume=11;citation_year=2016;citation_doi=10.1371/journal.pone.0146492" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. A. Gore;citation_author=J.-M. Chia;citation_author=R. J. Elshire;citation_author=Q. Sun;citation_author=E. S. Ersoz;citation_title=A First-Generation Haplotype Map of Maize;citation_pages=1115-1117;citation_volume=326;citation_year=2009;citation_issue=5956;citation_pmid=19965431;citation_doi=10.1126/science.1177837" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=N. J. Haase;citation_author=T. Beissinger;citation_author=C. N. Hirsch;citation_author=B. Vaillancourt;citation_author=S. Deshpande;citation_title=Shared Genomic Regions Between Derivatives of a Large Segregating Population of Maize Identified Using Bulked Segregant Analysis Sequencing and Traditional Linkage Analysis.;citation_volume=5;citation_year=2015;citation_pmid=26038364;citation_doi=10.1534/g3.115.017665" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=C. N. Hirsch;citation_author=J. M. Foerster;citation_author=J. M. Johnson;citation_author=R. S. Sekhon;citation_author=G. Muttoni;citation_title=Insights into the maize pan-genome and pan-transcriptome.;citation_volume=26;citation_year=2014;citation_pmid=24488960;citation_doi=10.1105/tpc.113.119982" />
<meta name="citation_reference" content="citation_journal_title=Front. Plant Sci.;citation_author=Y. Jiao;citation_author=G. Burow;citation_author=N. Gladman;citation_author=V. Acosta-Martinez;citation_author=J. Chen;citation_title=Efficient Identification of Causal Mutations through Sequencing of Bulked F2 from Two Allelic Bloomless Mutants of Sorghum bicolor.;citation_pages=2267;citation_volume=8;citation_year=2018;citation_doi=10.3389/fpls.2017.02267" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=Y. Jiao;citation_author=P. Peluso;citation_author=J. Shi;citation_author=T. Liang;citation_author=M. C. Stitzer;citation_title=Improved maize reference genome with single-molecule technologies.;citation_pages=524-527;citation_volume=546;citation_year=2017;citation_pmid=28605751;citation_doi=10.1038/nature22971" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=D. C. Koboldt;citation_author=Q. Zhang;citation_author=D. E. Larson;citation_author=D. Shen;citation_author=M. D. McLellan;citation_title=VarScan 2: somatic mutation and copy number alteration discovery in cancer by exome sequencing.;citation_volume=22;citation_year=2012;citation_pmid=22300766;citation_doi=10.1101/gr.129684.111" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=B. Langmead;citation_author=S. L. Salzberg;citation_title=Fast gapped-read alignment with Bowtie 2.;citation_pages=357-359;citation_volume=9;citation_year=2012;citation_issue=4;citation_pmid=22388286;citation_doi=10.1038/nmeth.1923" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=Z. Liang;citation_author=J. C. Schnable;citation_title=RNA-Seq Based Analysis of Population Structure within the Maize Inbred B73.;citation_pages=e0157942;citation_volume=11;citation_year=2016;citation_doi=10.1371/journal.pone.0157942" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=S. E. Lid;citation_author=D. Gruis;citation_author=R. Jung;citation_author=J. A. Lorentzen;citation_author=E. Ananiev;citation_title=The defective kernel 1 (dek1) gene required for aleurone cell development in the endosperm of maize grains encodes a membrane protein of the calpain gene superfamily.;citation_volume=99;citation_year=2002;citation_pmid=11929961;citation_doi=10.1073/pnas.042098799" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=B. Handsaker;citation_author=A. Wysoker;citation_author=T. Fennell;citation_author=J. Ruan;citation_title=The Sequence Alignment/Map format and SAMtools.;citation_volume=25;citation_year=2009;citation_pmid=19505943;citation_doi=10.1093/bioinformatics/btp352" />
<meta name="citation_reference" content="citation_author=L. Li;citation_author=D. Li;citation_author=S. Liu;citation_author=X. Ma;citation_author=C. R. Dietrich;citation_pages=e82333;citation_volume=8;citation_year=2013;citation_pmid=24324772;citation_doi=10.1371/journal.pone.0082333" />
<meta name="citation_reference" content="citation_title=The Subread aligner: fast, accurate and scalable read mapping by seed-and-vote. Nuc. Acids Res.,;citation_author=Y. Liao;citation_author=G. K. Smyth;citation_author=W. Shi;citation_pages=e108;citation_volume=41;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=S. Liu;citation_author=C.-T. Yeh;citation_author=H. M. Tang;citation_author=D. Nettleton;citation_author=P. S. Schnable;citation_title=Gene mapping via bulked segregant RNA-Seq (BSR-Seq).;citation_pages=e36406-e36406;citation_volume=7;citation_year=2012;citation_issue=5;citation_pmid=22586469;citation_doi=10.1371/journal.pone.0036406" />
<meta name="citation_reference" content="citation_journal_title=CBE-Life Sciences Education;citation_journal_abbrev=cellbioed;citation_author=D. Lopatto;citation_author=C. Hauser;citation_author=C. J. Jones;citation_author=D. Paetkau;citation_author=V. Chandrasekaran;citation_title=A Central Support System Can Facilitate Implementation and Sustainability of a Classroom-Based Undergraduate Research Experience (CURE) in Genomics;citation_pages=711-723;citation_volume=13;citation_year=2014;citation_issue=4;citation_pmid=25452493;citation_doi=10.1187/cbe.13-10-0200" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=M. Mascher;citation_author=M. Jost;citation_author=J.-E. Kuon;citation_author=A. Himmelbach;citation_author=A. Aßfalg;citation_title=Mapping-by-sequencing accelerates forward genetics in barley.;citation_pages=R78;citation_volume=15;citation_year=2014;citation_pmid=24917130;citation_doi=10.1186/gb-2014-15-6-r78" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=A. McKenna;citation_author=M. Hanna;citation_author=E. Banks;citation_author=A. Sivachenko;citation_author=K. Cibulskis;citation_title=The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data.;citation_volume=20;citation_year=2010;citation_pmid=20644199;citation_doi=10.1101/gr.107524.110" />
<meta name="citation_reference" content="citation_journal_title=PLoS Biol.;citation_author=N. Merchant;citation_author=E. Lyons;citation_author=S. Goff;citation_author=M. Vaughn;citation_author=D. Ware;citation_title=The iPlant Collaborative: Cyberinfrastructure for Enabling Data to Discovery for the Life Sciences.;citation_pages=e1002342;citation_volume=14;citation_year=2016;citation_doi=10.1371/journal.pbio.1002342" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=R. W. Michelmore;citation_author=I. Paran;citation_author=R. V. Kesseli;citation_title=Identification of markers linked to disease-resistance genes by bulked segregant analysis: a rapid method to detect markers in specific genomic regions by using segregating populations.;citation_pages=9828-9832;citation_volume=88;citation_year=1991;citation_issue=21;citation_pmid=1682921;citation_doi=10.1073/pnas.88.21.9828" />
<meta name="citation_reference" content="citation_journal_title=Bot. J. Linn. Soc.;citation_author=R. J. Mueller;citation_title=Ask the plant: investigating and teaching plant structure.;citation_pages=73-78;citation_volume=150;citation_year=2006;citation_doi=10.1111/j.1095-8339.2006.00489.x" />
<meta name="citation_reference" content="citation_journal_title=Development;citation_journal_abbrev=Development;citation_author=J. Nardmann;citation_author=J. Ji;citation_author=W. Werr;citation_author=M. J. Scanlon;citation_title=The maize duplicate genes narrow sheath1 and narrow sheath2 encode a conserved homeobox gene function in a lateral domain of shoot apical meristems;citation_pages=2827-2839;citation_volume=131;citation_year=2004;citation_issue=12;citation_pmid=15169755;citation_doi=10.1242/dev.01164" />
<meta name="citation_reference" content="citation_journal_title=Developmental cell;citation_journal_abbrev=Dev Cell;citation_author=T. Nawy;citation_author=M. Bayer;citation_author=J. Mravec;citation_author=J. Friml;citation_author=K. D. Birnbaum;citation_title=The GATA factor HANABA TARANU is required to position the proembryo boundary in the early Arabidopsis embryo.;citation_pages=103-113;citation_volume=19;citation_year=2010;citation_issue=1;citation_pmid=20643354;citation_doi=10.1016/j.devcel.2010.06.004" />
<meta name="citation_reference" content="citation_journal_title=Plant J.;citation_author=J. Nestler;citation_author=S. Liu;citation_author=T.-J. Wen;citation_author=A. Paschold;citation_author=C. Marcon;citation_title=Roothairless5, which functions in maize (Zea mays L.) root hair initiation and elongation encodes a monocot-specific NADPH oxidase.;citation_pages=729-740;citation_volume=79;citation_year=2014;citation_pmid=24902980;citation_doi=10.1111/tpj.12578" />
<meta name="citation_reference" content="citation_author=M. G. Neuffer;citation_author=E. H. Coe;citation_author=S. R. Wessler;citation_year=1997" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=M. Rosa;citation_author=M. J. Abraham-Juárez;citation_author=M. W. Lewis;citation_author=J. P. Fonseca;citation_author=W. Tian;citation_title=The Maize MID-COMPLEMENTING ACTIVITY Homolog CELL NUMBER REGULATOR13/NARROW ODD DWARF Coordinates Organ Growth and Tissue Patterning.;citation_volume=29;citation_year=2017;citation_pmid=28254777;citation_doi=10.1105/tpc.16.00878" />
<meta name="citation_reference" content="citation_journal_title=Development;citation_journal_abbrev=Development;citation_author=M. J. Scanlon;citation_author=R. G. Schneeberger;citation_author=M. Freeling;citation_title=The maize mutant narrow sheath fails to establish leaf margin identity in a meristematic domain;citation_pages=1683-1691;citation_volume=122;citation_year=1996;citation_issue=6;citation_pmid=8674408" />
<meta name="citation_reference" content="citation_journal_title=Trends in plant science;citation_journal_abbrev=Trends Plant Sci;citation_author=K. Schneeberger;citation_author=D. Weigel;citation_title=Fast-forward genetics enabled by new sequencing technologies.;citation_pages=282-288;citation_volume=16;citation_year=2011;citation_issue=5;citation_pmid=21439889;citation_doi=10.1016/j.tplants.2011.02.006" />
<meta name="citation_reference" content="citation_journal_title=Front. Plant Sci.;citation_author=J. Song;citation_author=Z. Li;citation_author=Z. Liu;citation_author=Y. Guo;citation_author=L.-J. Qiu;citation_title=Next-Generation Sequencing from Bulked-Segregant Analysis Accelerates the Simultaneous Identification of Two Qualitative Genes in Soybean.;citation_pages=919;citation_volume=8;citation_year=2017;citation_doi=10.3389/fpls.2017.00919" />
<meta name="citation_reference" content="citation_journal_title=Plant J.;citation_author=H. M. Tang;citation_author=S. Liu;citation_author=S. Hill-Skinner;citation_author=W. Wu;citation_author=D. Reed;citation_title=The maize brown midrib2 (bm2) gene encodes a methylenetetrahydrofolate reductase that contributes to lignin accumulation.;citation_pages=380-392;citation_volume=77;citation_year=2014;citation_pmid=24286468;citation_doi=10.1111/tpj.12394" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=M. K. Tello-Ruiz;citation_author=S. Naithani;citation_author=J. C. Stein;citation_author=P. Gupta;citation_author=M. Campbell;citation_title=Gramene 2018: unifying comparative genomics and pathway resources for plant research.;citation_pages=D1181-D1189;citation_volume=46;citation_year=2018;citation_doi=10.1093/nar/gkx1111" />
<meta name="citation_reference" content="citation_journal_title=The Plant Cell Online;citation_author=B. E. Thompson;citation_author=L. Bartling;citation_author=C. Whipple;citation_author=D. H. Hall;citation_author=H. Sakai;citation_title=bearded-ear Encodes a MADS Box Transcription Factor Critical for Maize Floral Development.;citation_volume=21;citation_year=2009;citation_pmid=19749152;citation_doi=10.1105/tpc.109.067751" />
<meta name="citation_reference" content="citation_journal_title=BMC plant biology [electronic resource];citation_journal_abbrev=BMC Plant Biol;citation_author=B. J. Till;citation_author=S. H. Reynolds;citation_author=C. Weil;citation_author=N. Springer;citation_author=C. Burtner;citation_title=Discovery of induced point mutations in maize genes by TILLING.;citation_pages=12-12;citation_volume=4;citation_year=2004;citation_pmid=15282033;citation_doi=10.1186/1471-2229-4-12" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=D. Vlad;citation_author=D. Kierzkowski;citation_author=M. I. Rast;citation_author=F. Vuolo;citation_author=R. Dello Ioio;citation_title=Leaf Shape Evolution Through Duplication, Regulatory Diversification, and Loss of a Homeobox Gene;citation_pages=780-783;citation_volume=343;citation_year=2014;citation_issue=6172;citation_pmid=24531971;citation_doi=10.1126/science.1248384" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=E. Vollbrecht;citation_author=B. Veit;citation_author=N. Sinha;citation_author=S. Hake;citation_title=The developmental gene Knotted-1 is a member of a maize homeobox gene family.;citation_pages=241-243;citation_volume=350;citation_year=1991;citation_issue=6315;citation_pmid=1672445;citation_doi=10.1038/350241a0" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=C. J. Whipple;citation_author=T. H. Kebrom;citation_author=A. L. Weber;citation_author=F. Yang;citation_author=D. Hall;citation_title=grassy tillers1 promotes apical dominance in maize and responds to shade signals in the grasses.;citation_volume=108;citation_year=2011;citation_pmid=21808030;citation_doi=10.1073/pnas.1102819108" />
<meta name="citation_reference" content="citation_title=ggplot2: Elegant Graphics for Data Analysis;citation_author=H. Wickham;citation_year=2016" />
<meta name="citation_reference" content="citation_author=H. Wickham;citation_author=R. Francois;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. P. Woods;citation_author=T. S. Ream;citation_author=G. Minevich;citation_author=O. Hobert;citation_author=R. M. Amasino;citation_title=PHYTOCHROME C is an essential light receptor for photoperiodic flowering in the temperate grass, Brachypodium distachyon.;citation_volume=198;citation_year=2014;citation_pmid=25023399;citation_doi=10.1534/genetics.114.166785" />
<meta name="citation_reference" content="citation_title=MCA1 and MCA2 that mediate Ca2+ uptake have distinct and overlapping roles in Arabidopsis. Plant Physio.;citation_author=T. Yamanaka;citation_author=Y. Nakagawa;citation_author=K. Mori;citation_author=M. Nakano;citation_author=T. Imamura;citation_pages=1284-1296;citation_volume=152;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=Plant Biotechnol. J.;citation_author=C. Zou;citation_author=P. Wang;citation_author=Y. Xu;citation_title=Bulked sample analysis in genetics, genomics and crop improvement.;citation_pages=1941-1955;citation_volume=14;citation_year=2016;citation_doi=10.1111/pbi.12559" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Bulked-Segregant Analysis Coupled to Whole Genome Sequencing (BSA-Seq) for Rapid Gene Cloning in Maize" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11.cover-source.jpg" />
<meta name="twitter:description" content="Forward genetics remains a powerful method for revealing the genes underpinning organismal form and function, and for revealing how these genes are tied together in gene networks. In maize, forward genetics has been tremendously successful, but the size and complexity of the maize genome made identifying mutant genes an often arduous process with traditional methods. The next generation sequencing revolution has allowed for the gene cloning process to be significantly accelerated in many organisms, even when genomes are large and complex. Here, we describe a bulked-segregant analysis sequencing (BSA-Seq) protocol for cloning mutant genes in maize. Our simple strategy can be used to quickly identify a mapping interval and candidate single nucleotide polymorphisms (SNPs) from whole genome sequencing of pooled F2 individuals. We employed this strategy to identify narrow odd dwarf as an enhancer of teosinte branched1 , and to identify a new allele of defective kernel1 . Our method provides a quick, simple way to clone genes in maize." />
<meta name="og-title" property="og:title" content="Bulked-Segregant Analysis Coupled to Whole Genome Sequencing (BSA-Seq) for Rapid Gene Cloning in Maize" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/8/11/3583" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Forward genetics remains a powerful method for revealing the genes underpinning organismal form and function, and for revealing how these genes are tied together in gene networks. In maize, forward genetics has been tremendously successful, but the size and complexity of the maize genome made identifying mutant genes an often arduous process with traditional methods. The next generation sequencing revolution has allowed for the gene cloning process to be significantly accelerated in many organisms, even when genomes are large and complex. Here, we describe a bulked-segregant analysis sequencing (BSA-Seq) protocol for cloning mutant genes in maize. Our simple strategy can be used to quickly identify a mapping interval and candidate single nucleotide polymorphisms (SNPs) from whole genome sequencing of pooled F2 individuals. We employed this strategy to identify narrow odd dwarf as an enhancer of teosinte branched1 , and to identify a new allele of defective kernel1 . Our method provides a quick, simple way to clone genes in maize." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/11.cover-source.jpg" />
    <title>Bulked-Segregant Analysis Coupled to Whole Genome Sequencing (BSA-Seq) for Rapid Gene Cloning in Maize | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"H5JNe5cWzIIhYCJkm02BJfyNQv9pL-lMjwFcWAbMB-k","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"491246","apath":"\/ggg\/8\/11\/3583.atom","pisa":"ggg;8\/11\/3583","ac":{"\/ggg\/8\/11\/3583.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/8\/11\/3583.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;8\/11\/3583"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;8\/11\/3583"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/8\/11\/3583":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022usvT+qMy8YD3WrAc5qKhkLNO7gb0VRSUQ1WCbXQ16cOVNK0eZlSbrtxVGCDWE93JlxjsSRbW1yTCmgQD1gohbgPk8Z7\\\\\\\/UWaVLz9A2R4AJdnMjYVLUMttmUCeINWNCAdZfBaoF90gi9gmTrr4KCo\\\\\\\/mob3Pfa69zqH++uvmZ3PKYoYA7jifjaLW6TcggLPY2AgDIOSLCIoELVSvL6LH9hVm3ememxiVep8Urq4jjv4djRIMIH7922ICLmDbqvtW3zvqj4oFwyqjrVLP2gOFPjC0BeuH4o1seOB6N7xr6VN2M0e3ntYKPdlQsMJP0PSPffB+Vp0zGV3ITy4d8DJ19CqsB9JbdQi7iS0WjFeqZqAwSmILVVKNdnTZxy7WjlooUXMJ5vfD5vNwFuN3\\\\\\\/+jym4jcw9OzlOPNP5Y7oE9WdsNw4o06rdjvHSCaA0dygdxwyM+7fgvCbnZrkOWb+sDhuBTI395moav18RschXuHkzmCLJT7FLB3dROs18nft7HLro5O9JKV7gFnRIo\\\\\\\/FWzIKPPqKE2IqmjYAdZQmtDy9ZkakxESNj6CQyhGnAd1FjLIWjGxSacMeJMrifF8AfW0hfThn9XDZwZQqkdhjx07A2qbKiVEpxeiP5z9izUS\\\\\\\/8ZWkb+sErGRGn7GSAzkHVbV8Bc8\\\\\\\/037LQGhktKL0s4iJENowZDyd2y8NwIoBDY2yPj2Pr9UvvhSe2HhSKZ13fZXa0Fj1JiR7moNJOLbA9kIa5oRHqnGzSHxny2e8RCTMKnQtA5egqxkHb\\\\\\\/9+cMMaabVfJnj4TnKOEmoa9HcsDSDCfg2BbYJCmlQGakEN2P4Tlo\\\\\\\/FsUXw\\\\\\\/sQ3NPbky9v9nX7jFk+VZNkce5NbyV+wGnX+UXXOb54gzvHghNzmBaTjUPuIqE05ztikyzXSYVzZ7N\\\\\\\/oWwFJoXZAu36IuZoRpimMnx5QYwnukIqv1drclNu16u2yS3hHO2C2l18sH0bKNvNSwPPnqUlseElpBMywTUaH\\\\\\\/1BC0Dgts4Q4TvQ+tYPEO0tFJakLZeh0cyAE1Q9QekelCckeRo0iG7PConY5xEEBkE+zb8zfQeHI5A1XBFxDu1jsXdWAEY3LN3FOSUap+U28pqciobmUr8DXDVvGDJ3devY7pfysTXcCAhorF0C3vXDTr4H940b3iO1NYU+gU44HC6alQt+Aeh47Pi0P1ZsNlRDUJPCtU0q5rQ2LobvTuQzOSmF34zIA38O0J4LlcQVHWvsK+lvaKtnOgI94aKS4FGyb0dHl8fLlm4z8l7\\\\\\\/lbLJMue\\\u0022,\\\u0022iv\\\u0022:\\\u0022HsHbFfziMhgN+6O18fZlZw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00224a6cd6d21863f05d422a2b1ea4e55dd76bda41e58aeeefe15e22bfa5e30b675b\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022aUW3ulRMl7wWFXU8ytbo1zh\\\\\\\/PQFxHu\\\\\\\/NdTrWnIZ9aAWPQQvfz4D9JkamHQi5obvF\\\\\\\/FSkMFMEQJ8BdRcgvmY2koYG2uI8sJ6ZDh+2zR4EJRPAUGMmuBbhq1zoeuxxLxWTASY9P4yOHbf0jJ1CrnmHtvXVJgTVTLn9APEsdYmwXG3zXu9SR7c\\\\\\\/6ynJAne5ykA5PsRV+d\\\\\\\/EiTQEY51h2E7bCGoC\\\\\\\/xCsr5Ex31+JCuS96IE2EN8WFwucsKvWMenFFpYfg1ZLWkManIyh9l\\\\\\\/qWWe0YiXjYYcJKybLqguIesPQP0nnqpp\\\\\\\/HVc38uqyTiT169fX9n7kKHbHXd+gqL2XLiuFw8q\\\\\\\/uPCm0c9Ne17EOncg3MB24YJir4L+3FwGvG5BjBnMaMbMR\\\\\\\/vA1yIXrMm4wG7RPhBxwTcWBAmfSa+cHoHWhvKbekYnuAkW\\\\\\\/kiJ2m6U\\\\\\\/bYqVCInuHKgYiWbphygKn0Y6f1jknT9l4j3XWTZI9Fo8pm9yI\\\\\\\/bs6GWUtI4U+lLgixGgpUNtaxOsx4M5VLEJEFVeNT\\\\\\\/AtUPvd7X6xTPUL3bRAxL+RO0aRG4qdGsS2fTs6NvdbIVvMZGJ1syUXTbplRH4BayVfWm7vBd8u+blwLbxuRps7tEyDTAnP+5CANtajsMRdeFHBapMlgGjpQSEoBbGSwnrdCfDTSi2UodbfdEmoAg68sDMC4oHQ6tQhqILC3x3bIe1oOEjoy1EDa1A+NDJgEJmTBfGPEEk8Mfw1Pz4um+fn6MfUCyS6w5LMdvtzKzms7ygT3XKCYGuF6+7fpw5I1ml8Z0MTq9SY5D\\\\\\\/N18XJz3wxdvinKqGJCr9yGd7NKt42D+UiosNhtpGbmZGDa2zJnoDw+xx+JO\\\\\\\/kM61FF4k\\\\\\\/+w1PCSud3oZpYWzenOy4ijYtJUKj51wfmkSZmCwPZTA7VsrPZ1OWqUJqp9Dau1UJKrdOva6KIAXV0z7jS29fHLJzUFyagr22vDDEj9VViXVCIao9ElS8uMjvHn4k7pOL20KFxzitIXLFvRJLpHltC7Jmxz55aXxeELMaPf3CBQ1vMlKRxXL+PYE2S75\\\\\\\/LOvqGRXfpdKCekWBX2JT5fUqANwsd+yLpvPVUazQ4XdPasnp6MtKuM3Qy12632YKRBEZso3D0WonY7G4MNZXfaTSPUMJaFWheG\\\\\\\/6mGurfmm\\\\\\\/dCp+glYVod1OjHEZestETlo16UToqBKK1SMnnHec1Acm5y\\\\\\\/St4T5nKWnA1iq7Wi\\\\\\\/deTtDNwDv\\\\\\\/pLm01ra+fsIhnNsxMX8kN+tpnNgm4grJw70oGydPvVhYil77mi0PytU7a2ZEAEUnYQtJ\\\\\\\/5QWv\\\\\\\/sc\\\\\\\/Da8JoJL1omGqpNFLuW85oyNlkVj3mW2nCK7vDuCXKkcO07hqHXOoXC4ditS7klQq41AW93Zh01owv7gC5WlwLNQdoRQbGAloG5fRIGvCShQ6dHVlhpWTOhDci7MWaalk9inqr5D\\\\\\\/\\\\\\\/3hMI1wFONhiVoq1hFRBoaDX4mCYvgdBHpVBV2l5\\\\\\\/U+94V+jrA4DyzL0imxiMM3PdAwboALU4epy9BAi72FGLzUZlY4AlV5rQJxL0lOPTLgGnP2tXYo624GxbwfpqYDX8HUwXtuFWcKTs7CCz4ufd7tscY54WTl+Pf86rjadAUKZ2+jWxz3OoDHJHE26yY\\\\\\\/JStq8I08d\\\\\\\/2ENqjydiSzOxYHXG1QODfk\\\\\\\/+9PpHcxp73hc4oj5tJoHSXtZfESm36hoMI8481gR\\\\\\\/Q09P65ZF2nLtEUF5aHhRlFbY\\\\\\\/2BZn6YzTt\\\\\\\/cJ\\\\\\\/t8N\\\\\\\/dU3FWEsG98j7mYRBXlnRXk7Ym0diSi99lSuoAIcPeli+tMLMQIxc6tVrBxkqEKI4A0C\\\\\\\/6i3z\\\\\\\/dgYYzpf2VZFPNBoJgDHJowq6J3QBijD5bZ2kRDCd+axkSPa4l+MTEenxpUcT4p6mo0rA0aNH39eFTurYgNvLKbGx\\\\\\\/QiNOJNPULDeIzudslMVqf5IInXW6UX+9im8vEsvm+dod1zS+tgU7PVa3Rcwi9vmjhAupMCDw1ZA4oSlYLzTlnxcUWOXld04pKN+fAWziZZ3FNDAcYl1XtXhte94g1tbeYv2y0P7Y1qQ1wrUWITQJL5XgC1fRbs6I\\\\\\\/QIfgcDvEd6sXW7C02\\\\\\\/RUUxmfWyOvQMeiRCYrembAb8bJF5HAYBAyaUEdH\\\\\\\/h3cL\\\\\\\/cZQw3NYBWQDrwqAJdGd\\\\\\\/Tw2hH9CoPPas9GMqZ180MCOXF1Sg8mIlBXEq3m40NkORYcZYJRzKXm6gTAvTWvJ6YqmXZT91p3TicO9dBvoV7BfjFO+Z4YoyTorFGY7FfGqqr52EnxhcWGy7HO1L+jpXoR4\\\\\\\/+29JN\\\\\\\/XLv4MvakEWuJlZ5hRe52j1KOClRBZc2A0D2CLT0E7h4bJ9RFNmS2IsMihDsQ+68vYP1BrdpNx1jrSj+phyujsPy\\\\\\\/62gMQyH50DSSHjcHZn2EcNtg6NSMOyeRcN8y2Mp0q6RKRItPexgax5SRavsm6Y3wUxKCGEStSy9gTuuUsZoK27BBmVdKryb2m9201rNPkCOgR5sQee8mCz++++YXnVt8P6K\\\\\\\/QFyGvPrNtjpKUP1agv5bVYeGngV2PwNbpVPnMd1i8mN\\\\\\\/X\\\\\\\/tDKeuMH99imOvRsxgXywQVQ4WVOJrtu8R1Gy7Ip+Vr5BoxndSn+Qr6oc\\\\\\\/aNkdBWpKAsPiSnrL2yYmah2qgJ20PgAtyM\\\\\\\/yqPY0k9xH9pWCjOJhH6VDj5h2HI7VNRAyzI03KYK4QFHRopXASkKIum0\\\\\\\/LSOv4kBZ+eJjWxjuvNdgQHf\\\\\\\/8fWsMdnlk2Tpxd7uwFo7Zta2nOXjbFaxVfqXqyd6giNNV2\\\\\\\/vynso8kO1ptizWsOMusU2UWrVcr3xd7cHXDzZYm1AdehZ5DCY0tG9Q38tJ2sCNXC2goHrynZuxOKHA59DhGqYTRBqzdYrOm79dFV4Icp+yzM8G+8bbtqp5zRezocR8Upx9KepQKeV91anoEE46hk3JzuRiToDDm3XUEAiVscOh\\\\\\\/+aig6dvCRk0edMfuv9i3jZ3XJ\\\\\\\/uYkb+Cflw\\\\\\\/Q78tGNWDnix0CHNFlOKR2DwXElAG+I6AJpKPPsaHCexx1chiaQ19lz7qL1EoSI7RPA\\\\\\\/7VAAfKbPYVn7bpDA2vUhFMRmcHjRlUxZVEakKGndO\\\\\\\/6IaUnIW8JbR86qqGrS8f0wUSTynTzJOP12o0a1pDad9qd9XwtvolCB9UmBrTOYUi8N7Nwsy7OovFEjuve+WNn1INYIU6fI1F6x94GDp0+6saydIjBTECSEjxuknBdndyQo4tgeNK\\\\\\\/\\\\\\\/j0zxcxVwTdw3ifkqq35M9\\\\\\\/0oFvmBwGg4\\\\\\\/6vD36o2dchNPNe3VY0jz+IxXYf\\\\\\\/qu2+56FV4lo\\\\\\\/IJLlhl9S3y4q2sIhyEUaH3LzmmLrVHZNadkRZ9z1dd1P0DtqWrmoPkB5YaU55sCW7nz75LQrq9PHGAjbjbPrlwnEjLpOscM+DZ5WTjQ8\\\\\\\/1yP55kexg+jgDJh7KfXURTqwIyjVwG\\\\\\\/mg32mjui5K4udt0C\\\\\\\/puQcFwkUmkwGz9eyAgIv2npL927IDYlDMuWRtePTFysJCiuKBo9y0m4OjwO3i7deDlQlVnqVCjgSSihTB8DSQn8\\\\\\\/coWIDlK7TyXXcWI+2yfYqO3\\\\\\\/NDjFtmNyZgbRSo1wpiKzZT\\\\\\\/KWaGbRxfZbb0k6LBvNo5zxIhpn5ee0ihaqLh+b2cjazxjXp80YC5aDz19IkJSjqv2HYV\\\\\\\/0Gg+PVnhekhNHQwr0Um7ePYvbSEXKKV+78xwO2rxx7EAyuGiQq8+aqLmQPyoJXnzYr4TAsOnOAnjJzjhBAviQTdyGk6zXeTBNAGp2WNIOk3gDWyArWtvU90sE\\\\\\\/166SjDRg7IGvcEJZb4Nwq\\\\\\\/XnAPHicD\\\\\\\/CIX4NcD3lTaR\\\\\\\/MRxTz3koQE4eyOubiuq\\\\\\\/GhL8z11wub7rdGIKSB4afr5tsEgitlOo9sYv7Xs+ujt4MJr9pKupyHYx6IkI9yKRxBzb45PsQZDxVCUqjf98dTubiBVzQ7xMON8ozXFGwgVylRG57memORycxJuQP0kywq2ZCZhBipxMDUTlHyTESN6cKbgGeMaLZE+513O+4WhKzqybgPB\\\\\\\/bNv7\\\\\\\/u3AkZl+GwSg8AcVqnKWTLgZ1dwahWBfIUwI0Gz96oyC7S3HUR+NtChhXK8n3Xd6pnpNwcQoBzMpbWzAODvV2hBKoA0XBl9Lu6VpiBViFBoXgdu0jyfXNlfS6Uwrzkh1+D3f+tSjDVKH+\\\\\\\/sXAGLnHnJx4hTnV+mXmuSW3UYvmpVwV7O2qFnC7nbchA4bkUKPviQmYT5goHC2+AEz4MVSn3h6gtxsOp+EhC\\\\\\\/qYvNjaGrJQXMYsa2PaWQNbNoYydrkDOMpo2QM6MgyZI2OBa5BuE2mgITr+fj0n1rofkcOkwxV0B664sLOd19fkQ\\\\\\\/5RYkTAKCPg2FUONeaZsIrIGziCWlIzYhTtF2WWO9ljdEAcP3jZqePpNtZ4+EZzHniYwGmhBYfxw+g563eKZW1HZtt8VgEATx31Z1miOOeHfVz8vmZkhGzWtULczF4WcX01OwOPTCJdXwLeJJ348Nkk6FgMGe3IU0YzcRmAc01F4oRry38r62eX6BJbOUAiHS1OL8b5jFua5CcbwNbwQuhIt4TtJhA5hYYPbcOVq1oty0l8f0PlE\\\\\\\/nmzy7iZNB7sgM4FJpmRtsmx9DR9KaHO\\\\\\\/EpIViZzARUnrGYwPSx6zc6ieGRBHY35hr0KfIah8n9j25HcOeHOm5QdgmasbgXuFO7dLCXwP3vDFBtmTXiiAIr0GS66JHNjsx1jh1wbdNvk0yaEj1C6SNbl4ocgmBWfTyhIhdzkv45nJvhY7lUqsJkO+cA\\\\\\\/R5iL26Rq+83UvRSfmcO\\\\\\\/V0lDrB0wDaf5w5NeDyvqh0mFaS4Q4Xs7JNDj24hHJpqDkzlRDniDvx0G4Bub0LujpBIF5YPr2ujEE1mZyai+9\\\\\\\/Q+hBdPmvybQ8Ixl+6N+nYuwAJv1KxAU7VQVLXv03as4Jcx4g6dM\\\\\\\/4xHYwN\\\\\\\/TlVHcnnx\\\\\\\/jnEsPVPkvrrYl5JMVyr3c5j8Ep0o+QgBwPk\\\\\\\/dUbf+fSP049ZkzSE\\\\\\\/6vKjpMlUSsg7qHpSmF7tUTJciQsicvWufJLRJx1vBTGOWIYC3ajQ\\\\\\\/xwfhkc6WjQz+kbse9XldxztXYusezrhM4cl0fc2ZBSXMBdWAFEFOcisom6NJGIA18Rgf5xbx6ZDGlPirZCnAKOcw7twRcUvRHBFtS5pM9zWo0G5lqgAzx3Aqc4rdqw8GM3qnYzu4E+jriwuXE7aFoP8aVygQVJypvJmuWxhDwvW9z6RT\\\\\\\/JrJ+uhycA932m5V\\\\\\\/xXGxu0ycle9fLOOjOMYULq6AVDIgWyP+y6ocRlg6eenYEoN\\\\\\\/E1GoNnjTLMfgaBYNliKIeeqa53QKgshdogPyUxi2cr+gGBybhyXfxxdppo81PCyjBFBzKpsty7kQwXpUal+BujDUYGaDonaXTAbm+aHkTOtMNwRsdLywrBh6S6B7\\\\\\\/87dD3HOayB9wnTlt6sw3PiBpcyl6C\\\\\\\/NYaia8ZMrbyUlH6Rk6MwRX4jNSHaTbgpyJv9T0n3j66M3Gn6+xIQfvIxUJSskE7o82t14qO29Xcy8IJcrNt4BWb3YdFB4A4DFfYIE3fLUblX+sDD50OVR8xI2L3xFm8uIJwgeKqGv8D23T04v2zd7h2KgIvb5IU0h\\\\\\\/YDVqwep1Prul5SittIfp4Z\\\\\\\/X15VvULFcyuzPrwLxxYRD+Z\\\\\\\/mTJX88OVN\\\\\\\/rSJRlGIC8zYpP4y7LjROr3zWUtvFYz0OaD4LN759BR+DNjR19KBzqVA8XhFNmbp27gL0XvkfAoDGEmJR6p8s\\\\\\\/h4in5d1hx\\\\\\\/p9gYLge\\\\\\\/mdL9vjFnugHOn9e9xtMxYmOmSTlGFpXW0A2Lglh+eDRU3RY40+weBPU1cFoN\\\\\\\/1Yr8ZtmXQW3UvBjBzUGd+j2AiFsixOEiZmBbmbBXX+JIsBqBCRDGJYhqzNTrRd2PD8kXmK26skBrin0oXlj3wCyzQVZWN7OAFhFiI3iP7aTEAOnrNqxRdpWXZkK2NwA3iOYKt7456tBb28D0Nq7ZPW9ygBLxa80gKpgpro1wMvIDYcH0Z+oxCvQUeiQhBN8DBipwWtT2Xm+\\\\\\\/ubK+JZ8jl77pPH4llOASMBCbyNWLZHxKpHxl7Z6anZ1V65SraesPMqeEKaG4MrNyd72PoBOKvfM6AY1mGfrfun95cQwfZ5MuOJBO52pxX+gSwjcUvdUWXAszwxduwztwrrqY4K7YGgxnC8F4qJTyD6HJR8RyEeYDuKo2K5Hdb4ucR56jZB8fhBbQQ8CuL\\\\\\\/RIO8gp1EpKZMekU7R+P\\\\\\\/4MOrjA9nWARiecZHVF0RGgg7RvfHTRTloMQpxKsJHkaQujuMDPtNCzm5Q9z2\\\\\\\/T8VuNvgaWxg3VSeNSEvkMDGn2CopCnx0Txkux0xrNhS78qhWk3HFZsRWTpwmK6VG2MNvs8\\\\\\\/SZvW66PySWxQjujjNmpOB23G0UPwZ2hOzwi0VCs0gsdFmB2DfFlUlHf0NWsyMttFwcMfkJJgd70XKciubcwuOL8gOvcLN4SBxpe31cQ\\\\\\\/UYpsqZBTxV8dxipq9FL1bigZN3tZXEBczXDYnnLoY7oObRNaQ\\\\\\\/HYMmsHYpYBZKflmS0J7XIusLDUYgNklZ4v\\\\\\\/U7n8vUUPXp808FRBuqcs7swm7Rx2ZyVwxcyYulzdhItRnR7Bl\\\\\\\/cxQADno+6yMUVNU9TLteBcpLHm3fn7V0hIiBS1plnQRwv9OCwrj1hsllF763j4DvQA+Q6QNX8cuPzcznPfOHCbxZh2+WGEOkfqBwucb0FpGjRBJboxThapD8GEOu\\\\\\\/bwfEzxxznRZ2dPy+geJjQYM3GCQj6YwemIISzw3Ql5J5KGMhQ3lUl46zp3hcjSilh5LdlHv7f47enqljEcZqZYMFROkBl9x5bLgro5dgn4cfnFOTyNL6yUkpIyiKjg9TRYAm6GDq1vwzy\\\\\\\/tlqax6XhexFGJhnTcCwy9JAWKxH44Ue62LDv5OlpL0CdA7ylITw2APfr4nAmd\\\\\\\/QAfVXRDIeQrgDom6yHHp51espjkQOiRjjDTBNZXiLBC4OdRjrqvhjAQPm3CD6fy9rVkGg\\\\\\\/E4lQeUW4nmdR8TnhZbmaGLcf8Ju5vaYXW9oiOPboa7SMPdSRh+7w2uJpAkBMgBy\\\\\\\/OcjsdC0cc3mCEmJKjHiZ7zHbXEux3t57IhF8SxDrmJh1sbcC\\\\\\\/6EHJVbPw+590NqZal\\\\\\\/RsSXH4PtkSdhM+yCV\\\\\\\/kuH6rgDTiMqs4pJV97usviciaOJY8kXpR7+BizdO+iyREDBgz9gbQ9Nu8TI98kvcaWOV68uNLM+tvsgJYUDZU3EPQDAav0z8GARfWk0Eg7+dhvgjGLg4th2GPY8Y8PDtG+cwib9OBR565If1NTUmWdHld1bcXxXiR0cG1MhCPJly+oJgGcQSq1fIFYwkoTTcI5ByE0kcTC11J\\\\\\\/KR91IVRBGyLo5CzwS2ys8u0pB8SR2SqmsmQnuBNh1LAjOmhhjpIViOGSESTPITjjfRJshAQXsxAdh\\\\\\\/MvXr4DCSqBVD39sLlUyAmAbbA4HNCm\\\\\\\/ViZLXiJW+KbV6ZqO8V8OEmbDAS7coiu7zczuHL+l\\\\\\\/gccvNnw5147p63uHZHIwxqXQNk2dhPQ3QWLVjRHp5Nas3O+i11NAkpV47oC3A5JDtX2lVJYOQM9foxRCrFrkgUos9VJ6PaevF74HdrC7GRwOHQsICO86GNCQJyvam0OSOxDY1TM4Uo6Nmr66AWOlkRWB3UvzwpsylB3AQBF5IgQnTR+DiYmgM9y6HvK9yE1H75vPjw3yoekSB8ilpSTtx8oZvxJTQ62sTEJjdMqEiFbCAKaUF79AiJCRu83mJl1FL05Ztuzu3fwPA\\\\\\\/yGgcujDvuC9NuQYYI+7MW\\\\\\\/jTNWwK1G8vufhuIcBhLgZuBWBNhbm4CfWsqMhO1Z7bXPvME8qr2qpUbZVTq0w5QBNhqW4DVcxw+Ctko8kd+Q5lhd1WpDgHmShZk+hLFndBrny2Nsj7i3CpL0sNHBlgarwByhs+2E5ynkk30lWkze2mEgMFN9boclA4h8AAB+RwUDnMP0wYJiIhqy4PD3Xfpe8jFRUVW0Vdz1\\\\\\\/NyFKMxH4we1vYXw1xUsiahdOb4V245A\\\\\\\/zf33O0GeeIMczUof6MyjoyDeRrruZOrO28scTJ0x3QTIhmSfGBB8BO3iGxfiWAjRHYWOpK9yvhU950mvqkIAci4rfPYby\\\\\\\/2ACGwKogjXsWIyKf5yRCQ7mcNmzH8QggssLo0TXBKvOujYikZVoKeovEADYrQO30p59Dac33V2xPt6NfGWOJnRGEwevKQ6vOnYP5HXQVZeeBO5j4HSmp38m1W376kl+xsid3vO\\\\\\\/nNNMLAU03JHVnASj8I\\\\\\\/h9pTAOBrQRegqaYtB7pLuWnSNKO5XwBFRvN\\\\\\\/4S2Vez6c7llTR9r3VtSa\\\\\\\/g7HKg8cixYm3Bt0B0dGLnSAJYFRmod17nO1OuAp0qPbpVQA4pL8xOzPlrytxpqFo4R4BCJyYJUFspU0Iiy82lTswuJVf4+zFQMoGAVBXoWnjcpo\\\\\\\/wAbRdXkHiXw1y0RBbW\\\\\\\/Z7ikNTwalwoa1fuS0RyyrzfnIRNOr+fk\\\\\\\/dox\\\\\\\/tY6LMhlH8ggKrI\\\\\\\/I3aGjfj8PVzxRjqM9Ybyjra1DMapEKhlqMrHFKFKFd1zT49BYSvLwiZivs8+oOBZTfYRFHv3HBuPGqrGmIVATqMX9onRObOd5MzQyoPAVdK8yLnvia\\\\\\\/dKmbIcOHh9nY7DlhZl1TVG9fST+rmRgsR1zNr2F5pelYQmpMkwLc2CiVeLXgzNxAZXsDEURWCFEx2UFcdD7YC3PY1M2qtWz\\\\\\\/XdLhUBRfNPIHIKMIano3Snt84UaIuDgSlnOVMl1QH68W4Caghz\\\\\\\/n7aIGVZ5T1u2yl21TujH6brsNnMTbDcBT4mDP9YmuP5tPS\\\\\\\/FjhtM9qpTFe5yBVxs8MMueywoApoB3w5BRiABSgD6WBp3gsuzJnKSAo6igsuFGh0kUCuebFWxBdzREjf0u1b9vqaIYxy5hMlTwwHBg801iE7U4+JiCFFZF80QJZJzE+NxGCxmEnlxKgkjL3xgBAlxFfxGOVh8K2+4M7bcnJU9+OtPGFegzFzQTSKYZH1xb7r4zrp9OpdLHj3lUx6mc\\\\\\\/p19KcYWmBglHRfZ+ow4eng\\\\\\\/OyPThtipvcB2VxYBdJ1Rr93Yqr\\\\\\\/AKakiZL6XyUSATm0p7CjUAgYyAWRkrbLDAU1ES+o2EL2cJ4Y1ZnfKkuf27EY3yEILTslDJYlLukMxpH+CmLFkteTJPgqYNPFDgvkrerdygH50EFvCpmeFVwntujS3BT7FL2pKE57ruKhKMB6UGVwjyRhmTdfaC3MOVhgVN1meVJo21oDR9AzlvlyEBjo6Od+0GYEbU\\\\\\\/hWJ+vBIVfaKI6o7xd+YoPHAxkuOKKfJ3GSEZykO+trrXD5tyFCv6L\\\\\\\/bXjiz5dX2Fe2wHIKl5B+tTJFLiQjgc\\\\\\\/7i\\\\\\\/okbnY1bxjv1BivVV+gVrKmul\\\\\\\/2IJtGE0xUVOfrhMeQrRIBAJ1byoGZWYcthwUGI1X5biT2+ieaRJRK+5UJeknkE2uHffmivJGGc8\\\\\\\/8eGns4qCyjKo9ZPGcjHoOSn9LGBNGlc1t5AVa729cXh7+xRDFNRstyaRXQe3XML2agiYrU9M4x0oXrL3d\\\\\\\/U6DFcig878rPlbt\\\\\\\/IZqHna78TkBiZJMB08PtfjsI\\\\\\\/NU7YFFKAVArlBn17Tpxxhyl07ci5Vlt9bOD5jEyA\\\\\\\/RqzOEBDGBhYDotNuUZ\\\\\\\/aRRcKf9R7psGHxueZ6Hvrkr9js6xETzrEe7OKW56gj6Lpp80bIrueSSPAQIamhJdzTrisz+GaH8HYgo9ub3EfJC9LssQaFf5u32925w8nw5I5oOKleMBqV0scJLJoIcG9XoVdLp5634h4dRWt10ZFFVcbr3o5ZyF4bkLDERMLGwt8UBJkgK9RVf3dYNkl8KcQrbfxs3hyc9kUFOR3pdmmybQC9ajyLdmBxL1a169rl1sg+nFRwC0eC4sJG6BD70Jg02MWmLKn3jamTZPwIMcWP6lMCeyTcfTJtNEz73RVqbnUOpNdk9Bp4Y4MccvUepghR6yOWC2hLDDzHwsBIEZmHkB2VnXLstNE7aMF3VEC4ge5tHIfSHSh3B5DCosQag0JmY3OdNoMxZpJF5PAjdgTm24\\\\\\\/gwcGoR3Iz5SZb4qHEGl\\\\\\\/dRaOkJkkbC4mDwo9UXQEWkn2rNSPW9XYGdIsieYpsRnqSeNXNbEt6VxNZPT4gCD1M1a6SFfnQtoUbIzNbTGuiL2WJubhOIF1uryhV94P9gTJfKkJISNWrSUJ9FYi6u5H3+q96GuHkVnsq6JX+VeXF0ocBkPHRFQN8acp0YmS38em8myJZQqHqaye0dlYg89E0N9S6kwtfOe7GyZfFTewbsBl8sMyRJpDv0NYbgran0Ty+nlPetDsIl8\\\\\\\/KAIJdaeSKuN2jbfq7DJBWMoteUiT+vFm8whPHmjnLUDYnWroPW1sMabpRMW1k9G0wCE43fB+2Y58p7nH08TAJUkOKt2kIv2RzFZyety5Tt3tFTMsVjCnMpgs8uYTqJYdSGZlNEPcckSoA+vTp\\\\\\\/32BadtybkMVAn\\\\\\\/yxFl4Gbd2K0ZFthSlEF\\\\\\\/f1e60OF2bDBd8Xozy97voNAnuxuFKCOJX5ClR1lJ8E4y3QXzkONP7FPIR0T\\\\\\\/UTPtJV50GB7DNwB2qjUgIRdt2kiQuQtX0UAohhEdH++J+pohxP09NguQPag3Az7wW+vJ79DIp1ycQaHQIG4Xbb7UrBK6oHUDBaOeGSRRQr5GrNJPChm3sO\\\\\\\/dMduMWLRf3ZsrGEbLmyRxvvZ4w+ioQ26mUASsqPBsKZBY19TSvYZfZ5NI3h676JHplND0kUGJsCtWg2itDfWpVezS\\\\\\\/KBLzDtfa2owLsbvaUa84pfQ\\\\\\\/ICkZSLadr3KTI7pBXmOq75OGzHE0T7M\\\\\\\/zeFKPRMXOCXYj8gx2WpKcs2wiQldKuXMgXzf7PrNH\\\\\\\/WzNLyPcEWdlRDxum\\\\\\\/8GYTRSoGQnSFnTAFZBYjdtgAq28roeiw3hYoxfZD4EvggBZZrHWTeoBFlGXTxHWPXNVXBjHE8RIWUuTmncUkLcdV+tZjhy4wEqLbnDcNc1g55eAOu0VHuNK6VFltXfL8awgITQ7BQGTuLzPGSzvhySYktks3B4agh2W28Ajc4NYXD+8\\\\\\\/1MLeCfcP6E9n7yzgZfZ1DXKigT+bDPYpqz5E\\\\\\\/acw+2v9Z2jDzka4iiTXgvuc3RG1wTirQK678lVd5MOu9zNYU0\\\\\\\/UCycVDKct2pDl13PcYeSbmYjmZLNs8fSUEHcwi30IYA\\\\\\\/kGKtDLa1d4jehYlgufEd2JxcL0NqBz546HhJ\\\\\\\/gYiLQexMYVT02BeHkAlm+eqAyOUZA9ArvKVGFajJEH5gcwaGrUxTr3Tb+KFYtycNlUDSGA+V+GRxUB17Iyg7VKEpB3Zju85IdMwIsT5hr8kh9i6wiW0T9PljNDEHrpKFli3e5ZuUMgXBVoUIGPwu1gCrsLhd2l+tQ8a4AB30D\\\\\\\/Yyp+O47G\\\\\\\/YghpNUh3h9xJlf14YWjoW7z4zEywX838D5s\\\\\\\/+irscXxTSE4sFvwupyofyKcI0Xk9qkmY4XX\\\\\\\/oIgXCtfynYRHLwxB\\\\\\\/2K7snzvHSnqMMDTXuPuOmSCfhDR8rdXlLLIxTybRBp+6V7EchIw5h55ZGHdzaFH9bdIk6oeNaE+3nUAIPYXPwtqgOhR+4S64TlNW6aFhL5KdYq137nfjGJ+lcmfADcfu9O0Y0A0vwrm2wIF5cfDbWmTwwUnQhYKXQ6SSx76GNL9f\\\\\\\/U3OjO0SJk4J046xEjOG3PXnMtJCWSL9EAa4oHIKWSAXD\\\\\\\/u6Wfnvrdqdg6L3nAxozo038j7RUVxl7bG41KYSksZJiDf4G2SrJhGkKyM7NQZMRhdHZlyAea5pU0FPJoW\\\\\\\/CWlw\\\\\\\/vzCTaiqTOVJTk53Www8pmvGpEMaH6PIMZMuboBpipgE47ausN4xZiGhoQP5hyr2\\\\\\\/pODn8kkHs08Cx2hcETtGb3ikGLHLI3aG5mcgrqHFCoJoG04rsi6me03NzXNfX7fqQIjXKVd0TF1IXAulxjp5TucnIv3dzPvExiOVP2ff\\\\\\\/TzNcTtExOnfeJHou3TVCii\\\\\\\/FjFL826FlKc2mhudtTelpbFsfXUe\\\\\\\/n0Jxw75sZATW2uGCfGDj3cHDGCPGTtXALv38ThjqaGBPi\\\\\\\/A4LdeFNzrjPwQeEsC9seVf5VF9EiAyY\\\\\\\/r2we9R4AqDd0CiZz762fpsSGzK87ccLn3tm6BqP0XpVdVm50AllYqFyHTXnoxkZMV2akIYDIR5Gnvlif9dmpUEoby05ZtKBOCZmX8Mot3zIF\\\\\\\/UxyJe5B58u5c9iHopXum1WomIMTbAzqJMOppLNklzt0V0p\\\\\\\/pboHRQlhjJK\\\\\\\/UtW\\\\\\\/wQQ+rSdt6YtdxmbEg632OGVXweRM+U6uDYITXWFYWGZZ2X+vpuslO1oK8Jvct4JS+t4Lx3GwFMBu8qMp0+N5bbaDfCb7kcEAehKyicRYPERFj\\\\\\\/ujx45UdQN5XI3P8hze81GrQjvkD4i8UxSto8NuoqT71oqCgoIE60C8T+citUnUJMUAzhWXa6Cv2GZU2jgt\\\\\\\/iv7\\\\\\\/Xu38RaG2r1iy\\\\\\\/IDo7z3FuwMWudqMf72rGpraXYXQb8ydBOoGG1TUqcPdPyg\\\\\\\/PYktCNBpMJK8hrm2k6KmrmZaxEkUFuxFgibdjKIscHLb7Pelpl2P6gO38efb7RlPlpDntJDQsDh938D0iC\\\\\\\/b8dvu\\\\\\\/2AXZsfH4kWKKQ4hocqaIOakVNLUno+hm3W7tNiK1PG+v78cvKYTXXdRvJapl0vFKSnTaZPGNXkMsPhAPGoZhWy40VZYZDGoAztElcD4Qksf5yg8uy3k+2\\\\\\\/EBgdrJjs4DRqRdrTvOLZ5AtTP6bZvKrSdtVDGYJHx7AdL+631ItS+2Ir334vDgRGIqT+pj36e8NP01QVNoqxQnI3pg7KSOSpHN5JdlAUJ7TSxRPsxVLGQ\\\\\\\/davL5cqNk+nBJvQ8uG0xLnmfz1j7M43Pu2DHnPJXRG7qcFRFhC4FXiGDhoqFZtiOrlQ3laB+nn7M\\\\\\\/VIFQ3XTTu3uaL8o9OTd7flNE7YwDVDzGINK+unFr20yZ9QDZTLBq7JjViganuz+WTghX5rHkRqrDKhLQyVEih5FKiMRcgeGPtLh4FchcJUV\\\\\\\/35gZRQekH5aSYmC9q7IyT40BcPaKS0tTYLhErGMFGqBVwxJVTDrrPfor8bKcfoEgAjCAy4y1UyeIlYH6u\\\\\\\/hd9WuaCWdKBwVV9TFwIhta+hdccDbjt5+29sCKmM6jcriqcnn+9+YJbigYf4BdBMC7nokZMW\\\\\\\/urWFPvseMNLWoL+ahDpCcrEQXnQoCFEfXbkkzXizAlD8gofucLjKh\\\\\\\/uJsf0XD1L4bkbxDzacrZpiY6VZ+CUbL6ja\\\\\\\/PNetIysyW7eI9ZH6HxhNPH6aQgpBeCkIM9lmVQsTD4DH1f4O\\\\\\\/iO6ssbXiMsOeFQsrV9Tx5uK+\\\\\\\/KPtvL06u37vMDl+\\\\\\\/bb\\\\\\\/2hBvmGGsbW1L00y04AJhSR2eEJpwIdPgf\\\\\\\/MohNyhI1RFmhmFBbhqGjMj+epvTaTqKE4y3MezrNvrofLgKrKFROWWQfb4bdXbHq02eXeUCbYEzKO5hh2tWI55fqplDOH2weZVimZoeO9JIUQwHXhuztuYbGqeNr5532h9EYimYwGJ4yBWYqV4HWMEkvW5NV8dz1dvnQDXQsaRzdMctZKpuQbc0mTJMEJrjTDa5orTdGHUJoTHZvcrOdHY3k2CPDEvgv3PPCLvDhmSy\\\\\\\/hagxHT+pcUGjeqTRKOBR5VDh6u3d1n8HTOydcVyGkQ2i1fN37QB18hmeCdMM18v8OvD1X+oxJuhXbR+DkMEqRZFsJIcXJ1w2Y8kIDacdr74NzXMMnRSuvGy9aW7O4g+8+kT7F6mGhZbinehdD\\\\\\\/lIcZsWrt2H2IPId\\\\\\\/+4hbDZxvn1QH+ipPEjM98aLWfXN6f8BtOVdf71PJ+CPf2n2k5Y3jmNCtDYu\\\\\\\/vFonCdSsn72NXGE8U\\\\\\\/5rsTShDSjz4QxY\\\\\\\/BGaVrjvI\\\\\\\/GitbsnZkMLn5ynJpatNma0ULmAT24j2HwDJp2tRoCM7GyWZvHusduAtLgIXkj+Mp3Rn0ficW5JgPTf+t8CKCnE8C8vnDaD4LatQrU0MOtDDetjS7YF2uK8lsBVN9CUXDqAjKpCGx2yUp3PiPV\\\\\\\/XTte0bPYtlsF5vn9fehFI6NgwPGPGbQe3rARDWsexO+xF8VJsPgmMJjj1WnLJjH5J7LCX\\\\\\\/3u6XpogRa8uiVZSboSA3KUcj+233AdUvj9IEAGdGEdCk+Dw0UWeQHDTBdkRgAN0BvRxRNCtkm8QwULFm4d5\\\\\\\/KpQqo8hGECRf3Io67irFOSA7w64rpjtksOfZT5w\\\\\\\/uGDxO\\\\\\\/AzB\\\\\\\/HVLhTgHH901gbfC2Fj7kx2Lb8PJQ0tUHl1rXW2fjSZJubGbEEQPjoj26IxT1Q1SgLWfOiG9PmIpM43sCRlVGTaWJUAmrp3JMw4UjmHyInvXiP3fzwoeA0a8JSo6TIFH1IXe5kVbUnX0HiOeF11FtwVkavLp\\\\\\\/y366gRynkZp0jTs4n1mFDrCkysWd+ZM3xXm\\\\\\\/SckrxEn1pLUX9feyoCrvBYnMOiBBahxhYeWYBAt7RKP3pzuPGXKE9MAeBbRQeCSVMfaQbgeANvV+jz6z1\\\\\\\/A2Ro35MCotryt81d+sFloQxgVXZpkm0A3QFcl8XpKMRsdfl0xVJM0SKVX8MUibTlPdjSPv0omKwx8OLeg3E20IrWifzULHazuHn9DIyUkvWmlY7XJDxe6XsWbDvc1CotH47XoqZASnExnzGdvPkKIJ13TJwJ2qO37o1bMjUi+HbqHNiFM0Ai51kzSwhRmShIuJ\\\\\\\/Soq7sQ53xceLRaPDYppt9tz8AA0aFu9E7BvBK12eTZZisTDgvtKF4pwNBAkKoVjT18dimkBlEz4J4YzYl7Xy7C8f6dl+A+qJygMYjR7kBe9eGKyetKQF65HEl6xVoOYy8fky8Xi55Sd7wzLsFG9Do9o5X8fJOrIOC67m5\\\\\\\/Mz429\\\\\\\/8thycxWesivpJyZasMf\\\\\\\/bZoWaOAEnvtlK7B33DaYT5OcwS66XDoVf3G6EYD4pCO0oqNR8jM\\\\\\\/2pLwVu2Ami8rto0Ajt\\\\\\\/p7SBoIfE1m4KWYHRWsTjU+GPMncpyg66MWNJ1\\\\\\\/m+uTglZUdXSf5npLLs1Wm9ugsTVphIyoFijCOk7FGGuzbjy+SqC0bQothrWA0F6YKrWp6NZUE\\\\\\\/xf71bbvvfq+bXb442MIVITxwJyFXVh46foo2xAQvF8dIE0DE5X3p1tX9McTy6+rZqAjFIrm\\\\\\\/KVPw\\\\\\\/EV4F56VjQt4i9kPd7Tq1KDzO96oKSkishJV0\\\\\\\/iszIA+0H9J8oB1cNDtl+DCv43DFLB+95YpkhE2q2aBg8NSNq1KzhVomAOZsfXmu8zDLYL\\\\\\\/YKNTETrSmLRCXjRHqwOPtubj0+nW4+ECLx6FTerxEJncHhmqHw4Y8vP+IwsC7WlBO3MLRKDQESL8d0vOb2hElJB0xTAuTB29q8zs1O14qVCPsqLJEQ3li\\\\\\\/SV7IDia0hTTpX9fhVtg02XCrlzebrhzMmHL3VR92tBPCht5M8gYOxSOJu9WA0DqIQXQmNBNZYWeJgdWtVMgmKqa+Z4SevM6B\\\\\\\/1s88FDDveGk5GmrD4PFvPHrZXuNCXm0a1yaTn7WfgGR0msQliv\\\\\\\/LsZoU2Y3kig\\\\\\\/PowK\\\\\\\/i\\\\\\\/1CIhWIWMcoxHVKBCzAHfgJ0BbRiFLaceLpq7tI3M911mR+WAHAYvx81v\\\\\\\/iCEfnS7qFtkR29YPdUgUKG86U\\\\\\\/\\\\\\\/xuBe9oS1eiWu2zxTw0bA+as3ieKg2z6gI74p2KpK1FKokZleJVx727ZFCtnD3vINtFj98M4fq3clKdqt0QIjqDCzHNUDGxHTjlK5NrCYypmE3Qw2uO244uhtCpLTCdM7PgFDWZvo+5ZfhSzwdpCSAIA8NlZYjpp95QoCPZkPZ2sJVkgU12N7Jujf\\\\\\\/bZ6mXDcLZBWsgk1y+REkzQ14B37dkZZk6GlcrooeT\\\\\\\/EFw0xnBhMxL\\\\\\\/WLAOGjgF\\\\\\\/JZviA9sGcLGMkcZMvqAOCkZzNHA\\\\\\\/aXxYhNLEOilDvzrLIX6buaqHxIPTiDNoNdQMZlGJE2qI8iIYQ8OAf81s4V3VfdoqaAgE+oIqu7AIPbX396czTZPjeAfPiayoeaZfwlIrx+Llehellg2U6TRyRLrx\\\\\\\/4gf9kZkarNHkb\\\\\\\/kP5EQ+zlRFg0Y+jBmXKDhVkZhHOQKrCx3KKIMsB8FheK1e7ZSU4bqh9tRqBopcBSKtQUODs\\\\\\\/Kk0pMcDhdfL5p8Nck+q\\\\\\\/gtHTaQEXHk7mpwXQinrDDfNcrJ+jaTvNGexusaXUXb94WLsH06oegSeANrB9D4YJWpqqDWaKKzJtKPLT5BZf5WngkB1ko0YyzlPB+VDOOAY09IEEC\\\\\\\/utB2q3XBKYjzXqH9dSsKOna+hU56kH6erN4S8BN+oi4xEJGL\\\\\\\/8Ibkmiohh5UzDqDeRWia+XcuuOhh+57sMFZSczEESVnCPBYrmPPN4I6xaO10pthBZvjPKnpyFmTJfm6hF6Pk\\\\\\\/h+bPW4z6PYF1ZL9YCxJW9tB\\\\\\\/xcx8XI+Q6FemXXG5kVQpjiaNCL9kDTFMC2wvVMe7hmW6AZfn9ZNbJ6BNzYiphqURptGEO1S6IeAEdhHSOZ7QJVNPbUhgh\\\\\\\/ugM63klRlzxtClWURbQC3CTnM1LqXBoAUKOhU0DXH\\\\\\\/8iHf2CT3MrSd1cdy8hO+gjteGcKq9IeGeF5UqkED8FVyAMhnkn+mS19a7BBYLFmrHpHpbiHXkgwWJgXq5ElNwT2yVQHqyecYvDcnszPL27vmhj\\\\\\\/s96fRedMszreEO4cOA+a3FR\\\\\\\/yGjwFqJ+wQKJlVk6TveuG4GR\\\\\\\/pmv9SCtDuXqGPVyY+dALMbcPiSwKDx4lB\\\\\\\/nE4\\\\\\\/xan0kD4Rt3vhFbs0hvkVMWhnEN\\\\\\\/oxhyts9+Y7mjS+jE\\\\\\\/eL2pNmRqhug6oGSkr3WdP5xUmTlnIJUttO5yTvdikZx0ehyg4DD4jw+NhIUCLgfX3jAuKmt8Dts15UO0KyMvCipN5tlbsd3h596gLh8vjofwNHy2w\\\\\\\/LbII5+4lniaoo+V95hlrIHWPq6kDTxM8I9lDNhI4TMCr8KZOkgRJVUD6FE6Nsfs7\\\\\\\/UQtoeuzwU3w+ZwKdFgC4611nh8GguACaMZaot1Z\\\\\\\/E0cHtFC4B\\\\\\\/0S3zUcfJv6UaEkaMCo0VhTuICaDn0fKS4yvlk1hnwvzv8YLDUBHP5t0JNNQWhP2G9ysRHIEdRDASdeKwXFsfeRiXLqyDqFFOy98CjduCwsGCyYp8DGZj8TKpLJlYgxrW+yBB9AWE0zEpa4lU7Eso7WDvuuFHxMusE6BHSk5puH9\\\\\\\/SSDARrH8JN7jjm95Iu4moqtdjCmIPRm6FXs7ZiMJWHL5rdCV+mOkWboTL0oXOq8RgN3g1FanFu1w0J4yfCkZy7+hbyVqTr5bW31X4BHR1gLio3sBN\\\\\\\/IEHxDNNz3dhi54zbzSjGjXbRAjCXff1XdeiyPYvbpCxUm7LnW7JMOM8XUDdaFP7RKIC6GsbnaQmnyBJyMkSsCltiS72KLm3nGhJzX+otI4J7YPRD796uofCyzxcPAqRKLXTBDd7o0CVAI77lIGIGZeH27pfXOgUm0VvYxoLG4WMfVIMJ+gijfBqFbui\\\\\\\/jpSgaAaGppJclDsMkZF2IH35jBdikLfOH1frcskQo7WEYu6SyYFO7c5A23pbuRfY2X4bSGVQUWOGX0Bo\\\\\\\/KefGGQ1iZyQsD0my0NKkqjEuQQG5J9NOKxLU4n2dEoHksTtROIdP0467QfYNsrD6m\\\\\\\/JjXiGuHOzvi1Gd1PaqcJOvkdfeQab0FQKSJVhYTJpmhiSNYhZHapkCOGPR5T2vSTDuZJrUREOdHcBnx+R9eMzI01UcRCUpClHp3rLWPklseEgRzX\\\\\\\/mNTqxiDV8WytA6BSgm\\\\\\\/siMuMqDBEwxLNlngQcjUrcoddhdQBr9d9q1mWDyQ5wvJS8cNA3iBh+CtDLaPRN+sb5FQzvBFHCyXEthdFypJomazFx0iU8hUUP8WociOMYUjFiNQUIgmg31Qqg\\\\\\\/++R2lKyvWESDn6o9ev3lvnphFcb0jkrW5kkpV4xExpKmr3jG9rb83veaOn+8uH0LH1zmLYsSTDxxav\\\\\\\/zXTH5jZxtrgDpzOwkPyCUkk8js6nB3NU\\\\\\\/atUolOWnMmc6vRA\\\\\\\/kWAC4eWB9EdIplMr+0nF2ELFEuA5gigql\\\\\\\/q5FYRgkBM5dRavDTG5H93SWNs27hAmLh+8gGpfiVNbWZVc4WWXq2hmi9I+gbReGNC+atI3bzrxfFGp0xyhE1YIqiN8yzenx1+5ox5mupvL2oNxnLGUlKdC+ul4BqajVgXI7fuBCHKkwDWDymR85yen24FZ58cLt34cmlmgC+haqD3Iuh\\\\\\\/Suf+wy6uTMlBc1ow1EBl9B+uE+Czt7RfIkfSokLx\\\\\\\/o+4ZVSzUKPxvj0RAyGMWWqlpViLaKRR9TPBUIwvpCzenDjHVruYJSOTn0tP+mPZSCDUky4ky7EQv4G1vMluDBoXKmf0TxL6ZG4sUW4pxgT4Wi8IKGQ0J6hwswoIfAPlWX2mLy\\\\\\\/0kgYf7cuIPHxsHs\\\\\\\/rCtcR8ZRPZzZKQqcvjTNsNVfqULVa2STMrQxd\\\\\\\/VY\\\\\\\/G98sMUB6l9rHhE\\\\\\\/Wprgl\\\\\\\/xYI3ujtSmQxxjkJ3\\\\\\\/zyn7wi++VmdTURoNpkpzto7wHAKEnAnU0C9QCvjwVnuk1Y2Kj0tmdhTa+TWwRbLsaeWte1ral1YIcvhygWxh0FkuUkr5YqAjA0EkyAN4MoDdv5a6pK7AcXzDmiefbWMB+sgXB7m27qu7EOBvMkkGUFtdXfx1hmBxC12muwNAz6\\\\\\\/duzFhOFiurbXYb90vDFAvomJyJ8RHEyadVd2iTNAeLLG7oMWFXKSWmSm6UtMeUrKiXqWwC4sqXXMdTuueNLh8BJeFbHNgCWoNNy8RB2oUexQxRVNtc8HEvgc2GUeBnrV3G7tise8zF2eZRIxjqN9pD\\\\\\\/x4nJMq8KSi5b+7X5Ge4pOd7URToR2u\\\\\\\/fv3hoipWaqbjjJUZArxlBzo8Dm1aPp1oqvTKPHZ6HxYBX4cWgssVNf0eu1NVJvICvtMe6lbjcvzsRxd1rGFrYegUsxcrT0dvUaEuI3p9Bo32MGs\\\\\\\/El1pGp\\\\\\\/ovSyf+UPZwRDEkVZZTO\\\\\\\/oMfv4xY3AD\\\\\\\/uU\\\\\\\/QonCzOOcuc8CgAC9wMnMEzRYt1FGU36ElTBGpHkLc5WkGLHZqZWeJKerPICt5bbrJivyNKOFtA3Ysvju8uE7Yjl3j+ma9UzyTgzZYpR\\\\\\\/8joGO705c5J2Kq3ZhxijTg5FybDb6n\\\\\\\/J9wOjbWV7CC0MTQ+t9JdTnRyMTMl+SlvOtTnQMtT5BO4r8krLnafTxAwut6d0YK\\\\\\\/xNNUyrdR2mVGfvh+cOpNuXeCqHhzMU4Xv0Crf4LS512F4Tb7FdmlIjLDom2i\\\\\\\/x9jSHW8rHrlR6ft\\\\\\\/I6Mrvg2eOYKmQAdA5AbWgj8B01OxCUJqTmqT7gVwR+LUxgWcFHPibf709AjN4YARhYZxgyVQqvVvA2\\\\\\\/N4TxeQMvDmIk39\\\\\\\/5H2zRuaKq4XpJQsaapqw6bXZjqlcMD8BvZQ5O8vkmRtSq5SoTJOK8jj+OeWtkLZoJxN\\\\\\\/U1HHV9RNZy4STu7NQe6oKOjqQxibnVIq4AstJakf96r6tOrucECuCdrbuUqyPH5h0r9aJKMu\\\\\\\/A2a\\\\\\\/juz7ePQcgwNq91m7XH7PhHcKV19o2IeUyxzj4+Z107Bs5Dk7qHAbvRF0RJHdKonJIOFolo6u4CkaGcme60mXan9ReIZ+tCoX1Ceb8mZO+QYakXLLKuFUh8TMi0fnQxDsAyz7KEpypY4uR6CyD\\\\\\\/lc1ERsjN2Au3Wn81rnZ8rKZoXI4cYXuN6Kzw3WyMsVbRn9pW6Ek8q9CTRzXIWrwnvuvBcYBCCgwh6dms8lnutW6MIqO4Hr8s\\\\\\\/e9M9BRSytzIn0jVmuIyUCxGZotWze+ABtf1U8XGNiDXjqIRvHMNT+avQNy61ZviJnPqVe1jGxvgAE05fA+lakoqBGH9X\\\\\\\/Kk4QV50vFsuY6v6oFObkAglqheuWGr7HhXCI0NjKpk4GYfjjCm1YCwNYbHJ1PWuPdufzD9\\\\\\\/rOff9V21urW4hhhfRW3DfIinx2MHPgtjc2Wq\\\\\\\/O2k9Qqm3OTyYgDD87e1QndRokwA1GvX6G7AbrTX8lkW47uGQHcQmTBZMGwBjv7DoFNiWEHgLY1a8iB6piGT1qDorfQGkf5wETrJQpHxdHbW4c8FP3gydrxeUtqxInxWqeP8AUjDOgQmmg9cqKskVK7R6ihnj0B4wKuUeQop0dpL8guime9jhTGlbGb1dnHUXzH8RPt9xUkVzNx3Lkd0bkefO5cYa0892j+xeXxqMUzsVckyjDVQ6QTu83Amuj9i8pXJ5XkhgaV\\\\\\\/ZXIP3j7YEdXXbbJHSAUAI+n24lTBdbEVQCwSJm9BWjzLkGiq\\\\\\\/FM5gA+ZU2NDYUOud6sOChoK1UxqYCUSCsOTOtTmBDAQbI1unsRHPng++MTu9APoqgSs+sJvuTyumg\\\\\\\/OV5cMIXT5XkLq\\\\\\\/7OFkOfkWxl2F\\\\\\\/ZT1leJ\\\\\\\/oA3KkH68DSUi9O89TTbe80jk5jMuShU\\\\\\\/e93vvnwspCfqX6wjy6wp0XoX17gPQt3\\\\\\\/Gs6laOZ5YzzIyhI+3Debc+1I3Cwwm3QAgOur41LiAoJSfua5D7DK2bu48lAPGEYuY4kna7IaIyMgJdrBkRufHB3zfwXqG5DJUEkARtRq87Mf+ghL12wA4X9urflnpCd6066CxuynFcmy0pP2j5c7D3BMqBgXMeE11j5IuLxIhm5Mm4Mxp+ZhEjJY3Rhc+F1B\\\\\\\/X34IQP\\\\\\\/zHk+DQnqtxH92i8dqyUokQ9fhDBca6qHc1bdrRcoKXKel\\\\\\\/+BxCQHMdsirsQeNfYTahC+9qFGU9H9wdpzeOIkSc76AOWTJH93L2UBwrRh3LLowyBd3\\\\\\\/tJROhq26WyoG3YfIKXovfFsYlu7jmEoKU1rEKwSFYG8EBXdQxCrqwP9CqcoFR0pyzv+zEe5r\\\\\\\/NFakwd6missIEfIjrlnTJEdqsdiq3Q8+Ff3m58NFC0G+3drCOYnlPtkNopv7xOo30S+wMHaEtkB\\\\\\\/LJ8shgyLshUR9JL68WK0XtuwgiSDMuJDRpZ3kEEEG7md2CGU0VtyvpOlOVkNPYlsB+LIzRF5nH5vGCpYsW3EK91xjth705dyauH4ecYdlnWx7ZEr5LNyIcRNiIrZ9eX0mdkfNsK9nn0cWJV\\\\\\\/tf0O215bDTWgTl+shhpTJE6sapNpqRhBSHpLtK4VrvLKuVCflBEljr46kXHO9wXVTOtY138iMQFHi7bbEoWalZQYVlHiemKjf2vh4qY+i7\\\\\\\/4vsy+GXa7Oh7ZL7cc+XFhzc12jraVHpZHMDrM9CSrTkJeNG8xkF2Gs5t5QpCZeAwvrdFwyQiPtKQHakQLM2JCOVXoBd2jSX0IRV+u2XI3xfPtENSHKT+zE\\\\\\\/ijRHAhC9DsMICGh0lFU+sEMrr4QTovnGQ9R4Mnhm6AX0PQ8D3XrGHLrIw2HgBvnEirdRmwbG2NZZG0FXZ6dO3EhOOJvEV2+slTFAjATlVnyLw0WdrfcRf3ZFCgANwwkOIzu6NhKIaQEid1CbSUSe\\\\\\\/bFyw6ts\\\\\\\/fmOEAabz192eDJI\\\\\\\/EMfnxNNphGSQNZfzgtK077ASI6j86aXdd7i6QnBJfIY5tLUhanIrVUcMuk2b3VBG4ISzbaTFFfdQGrTRhh1gjTtLOwwYw+9VgYSO\\\\\\\/OxSzzqOs8lJpw80YKybv1SCUDVvIZnuS0DwUT257P6Twe9ngyDSbErAw1rYcSjejaI\\\\\\\/QqcGGIhQ3saSpChHlNSZ5cgeF\\\\\\\/K4Q8n\\\\\\\/vxlstySuYXkjEJcx0IGjQwL1dYFo1EN7OshRLdQUxCppG4ZpW90+VIub1iupjcgr3iN\\\\\\\/NyTaPPkZz7foq60\\\\\\\/d0diDZtskzlDkhO53Vv\\\\\\\/L2c58RnpjB7iXx9pyntxP2JwuyX0HI0hyNuWwPCD5njGUWqacIrBcPtyKEb4cashn17mwZCAM44EJSbMB\\\\\\\/BLQlMtU820qo97idjLAiFdbAwYiF3XR\\\\\\\/FnVvbNpHxn0sSjPna\\\\\\\/HdiD861AbiEj5uG52nQvxb85seXk1fVlZNLsh3usb23GcifRaf5IK9aX4qIMV5hR3bPdk4U\\\\\\\/lfHu0AjlRXJ4Indspn4bKe4y1eZESgogSYmCbuUgkVuUTyQI1vJYYQhe\\\\\\\/dv4neZYBuFkeaq5R86OB1ChuXSG4Pi1sjgFeaBs2q6tsoO7OAUPxsNNOuyJ4l7QpXZQi4VmdNpx0u9KTtAfPT4n1Oow+weqWUsYvG\\\\\\\/al9wt741B4EnDkK7MfHV1CDA16cqyab8gd52EJl9oKeH9Kg42eWw\\\\\\\/VuK5izBbvVPpnMbi6jTMaI+4dZrGJRbfmDDyVelYfQdZ4CohLI0zP27GvpbnUTu0gmConiaasuq0glrYmVZwLE3MPBnX6Y5uDAWQTf1zvhdhTPCA5OM1\\\\\\\/pn+Qj9C5WfsFZxiVsljiy5XXHYV+mlSrjRC8fVCxnvEzcohoEyeO3bmoEOAWVbWZlvwBIi7UCsJBPoxB6gLMuny7fDIPIoXCtyLOfBSxr8SBEJMlqEFjAe2RaLb3uVZKSnkOnYWAG3Ad1ZSNxkDRaxNwIg7Y996p9eOcbUydt26\\\\\\\/KpYY0mPJ7Ju7KyTEPGZUS3VtFdXvOTVovUR\\\\\\\/zu9V32DuQXN0c4Okz+FNazz5BIP8Pnd0v8gcoIvyC8E0h9MzuwN96sftnG+4UxOz8Kpck8D8EDbKKrRqC22aXyVlWS2hbgzOhsI65QW0tEkK5M3qb2l2KEzIAQr855pSrI8WbKYLsUCF6z9dxKcBDhvGCQdyNjQPX3UTjs+c4CSMhMTes\\\\\\\/2qj8PMoklRjX9HEHPUbd7GFO+PAzlR8Gf1C8z3T1bp7uIAnr\\\\\\\/iiOgJCzNjynBs+LD2sz9Df5lp6fUMgqtU8FXX6IBIkAD\\\\\\\/fP9jRP45JXtvG7gp35CyLzDOhWgi1mJkrIP4ZFShFBKwSDoC08l7UZtNJMVljuAY7yGPzRTspZODvcgei5ENGPSkjJNF3KzqoLcFJdAYWfQ+06lPCB\\\\\\\/DK\\\\\\\/07p2Jpx0VWCMSgoFCxRNY5JJOLNYzSLGmM9YaEBC4f8QdLuVStc\\\\\\\/UPQmWCv6hRemXJALPVAI\\\\\\\/zBSWgkNJN9amP+W5Amjst4DDa3iHLTlRtaA4SGjMn4d3O5P\\\\\\\/RCKnUq3AlcU3EteRh8RkjSronikBg67MfV1Ha+MBti6Zaq+uKdLwpCTEeHVhvzNlLMJvqx50T1fQrLh6VW8ZJK9A5tip693PBXLGIhB9gnxD0Ag8uZaesNTqPmfGIQDC2d5erqwdlTvk\\\\\\\/u\\\\\\\/laCKuMmdrZNF06T\\\\\\\/w6oOs1\\\\\\\/pB8I0MbRp0bLg0Lh0PagfHJnKsPiYWc7dB+dx2fpslbWm2TYHaKwJ794K+UN0v\\\\\\\/JsfM\\\\\\\/6QcCuTUAgifSued3P6RsKDRekFeGHBkuxVCsxKOviuNt50LwZvHO0oq+u4DX82+YYiDzd6RSoHnx9uVmRcgSsNqp3ZI3Z8IL5a9th9Cl43W8qTktNBDFsamT+96P0xkfCX6v8ITqf9tVRjOIlTG0ZCXyUcxMMgAd9nCZuZz59PdZiXpC\\\\\\\/xRWXxzHXw23J7z2gL5OuVr4R+RZNH47TM\\\\\\\/8JH\\\\\\\/j1pnbLcAX+v6k4zQEJyRzJmWEEszioeFZwYjm6LWnll3H3oz5cdf6sWb1tHIZXpQpaNoAb2kvuZ2D4aSS1G+ihz8By1TAl0JNVc8khTUllNB9VXizueZVkL5P1mcSNFU6729+9tH9xqgGToSqKcnAZDAkg5vu8vVLTbyiznGLsXzOd0FmVom1NsmocSB+uJ983qxZag1mKfi0Ze1F4xQkcMft2GAQT9+hsxO70NPvPEhTF3Jqvt9HDdlvpFHC411VQ29IqImlREtXdzHAOUSZj5Er5lGMO8VXqQipy8qFW5mdzxL+uOFrdKWXqGlrv7uGXBTG9N+VcprbG015rIsheO098zq4sNmK8NlFH0T76wK5t2BPdUoMVEuVSRgattkE6HXCJ25O9MG8NoaLU3eVrU\\\\\\\/OHhMf66xBKl1feJRrxEfvOeVzaXZ+40ySUloq7EIWqqvhr6GcvDCD14gQDzsj0T6Elb6n35KVPBMGInhdGalzT2lvQyrCyDp9+B5yPN1Nzoly+aF7I9hO7DnbF1xqExy9kJEEdG\\\\\\\/zZ1fQJtHItf9cYwrT0OsusX9QCz7ZxVSG8B1k8IQOpYLxIVuLnE6jzLNxn1rFDF65eGiqkfw0f+amOx3bMM3jsjfAbDn5nriD3Y\\\\\\\/OC9Z4CavkGuGGy4oIysCl0NzPgtjKH+I\\\\\\\/l+i2WPFe33reAN941r2DTMwpdvyq+QV377Z+qV+FDHo15qibC4JdfnnbtzF1UJ6v4Grkn5vHne23aAJHTNbptK\\\\\\\/2DwPbsOou\\\\\\\/+yUDpRLoIj76qUWFa8ZQG29Qskct9Ll8dHDvzqWICoQWomFrTLkGIf0GWyp2gSg8tIJ8OxIyQPEvF0KOkfo1aFkCP1fC84YbXTvzqsBdh10Z1GOAOPElDZBYfJxDTqeSQsVmTF2u4zydvlLWs+2CBiSF6luO4u5dTQCMBeW5T\\\\\\\/zwC9C5MSxcR2UeSDBgNL8vlAiBSMDpkO+FUxSbYqYN1lYRQ7qOoc+Vha4ozsZ4BZtwPOwhYZ6mSwtZ6MiOwwpT7Ptf\\\\\\\/eFkeEDNIpvFSlLq\\\\\\\/qzNPV5LsL89MMiC7+HFVOmIfPMnC0bGkrnENxqRZfGffs7lWUNkU7QvgcuFBuMpQP9KQT91YfRHfpfPI1ycbT8wG5qfl6bnB8ZDE629WXaPOrDG5KNfcUjLM0nBdl+xYmkrEBdEjV1JWQ48ZjM7eKXDELbdsIkmeBtyrXTlHGYq7CdTXCIgCWQjaPZ0TitFoIvNJM4fhP\\\\\\\/JBXpokMfh5bK0E00wBeuKoSkZdNR+QoZDs62XFiBGebIdwU6TpMefUgdyTGtwfUhLc++jetjJDW3tp+4JF\\\\\\\/f3Skd+QmMEDufBOwAhvzjGT09U6SnmT0Zb5mO6MkBuGHGCC3EAIeDN+fP\\\\\\\/LAmUUljtwD7+EZY\\\\\\\/Pky1xPO7lGL1Ym\\\\\\\/OhPIK\\\\\\\/HAo8x816DI3y6lXN4X6uF1JUfmiUMbvh85Zva\\\\\\\/D\\\\\\\/RDS+pFu4xr+8CR\\\\\\\/DZ+Lrq\\\\\\\/vEIwZcf58E92P2MBrh8wjOVYE2jtHYbmNYh5SbKl3R1x5EXMOBIcj1HogdwxQJZ0\\\\\\\/XawpuAxa6v1h1OUSOjA70gWxbKEjGPtn5OVhM1Q+3MMVPUpkDz06zpB8Qelt4ZeFX4JbZRUFieyvfIvAYgFWFlL8G9nxfjrN7kJYB+A8Xpg+KVcgESYI4ytOikjpWUzwqzjPFjnlmmRb8QUAJhPlUiUmndgfmh0pj3IgWjrSrEGHoNp8QGEEIkmpDFoeZwS7hFzcnCA9YUvdaBM6SyFo7m40pyoGaRQe3HitA5FyDiLXWudH+HnQDEkA+Q\\\\\\\/+YQqxxXT1GkhXtkmwE1sm9mEUjUQH0xXp72ryutxtKVoraLYI2yqzAzx4bUn8JjVP\\\\\\\/nQl7fDtPUGZiLn7sDtN9qEge+Vxa1t52uHGDnIUGTlmB3xQs\\\\\\\/0Mi2vYWRvFVfCFrAia0cpcSyQcaDz6sgHhfsi0Z5hNMqG161U9WZ3p\\\\\\\/8vRmLG+spZLBKDM58qYBGWlVWCAUEFjTGf+JEzE7L17xI6ta9I+fZu61F0Ly8SrBxKqHNRmgl+vv4xnLOJiehxpZnNF9qYieB11FEd0YBbOduMg\\\\\\\/+ALYkJiFiQnlF7q4hutnLym2EyeuPNwhtF1uz9epcffHD4DI4yPXPxKuxrGtDNajV0Ji8zzxS9EyXG6pe9Unoz3bLW6uVfL0IrOMhKf6i2LTmk3FC9O1Wn2SEosiqurOYZRCrvjCpBNUSLJaGaFKxsu702MqcpFch3br\\\\\\\/u61UlVGKaZPGyZJTCTVjXmj0ytayy0TmwxuRILDDmcYHC+sM3ECsKPmkxSjSvArEnC5ajjl+\\\\\\\/IMTjYMxyBy2tj5bKztivhaYcgWuCsQf6a8HxXGll1HNdkKvM4PqJ64L2UOfmim4eCw3hXDA0lKtD7HPcyRiKvgfe9QuHurPZe+k86BtIlnqU\\\\\\\/Oor0Aq6fia0y9PxAmuc6+DUGWG2afeYw1vOgQC6dSdWSQ1uk0RbNkNOrlNNlMy9ubS\\\\\\\/VM8188I+3Rrs9KabmNc2PJSw6za1l4Ol5GNLQZ2gORTatR1ZfiFqltKwMrHCF3PgagGhSQ0YPFhkTwDkvdVjH3YPwrgc7SiNH6X8olb3K8NgNxwb7lVoVAe9cNTKiVCuKmZM887FsH9B1GGz2YScTDGxckG\\\\\\\/VidMB8kYFYfGP6z1ibzB4NnLr0+HDflO2MyNriaIbhzPJ\\\\\\\/t84jpv96ovLFJ6BDqiKR2l0cYxB6wct9+sQHPC7cbsf1e1CzIBcknOB360DIkjE2GNP59LMrXEUscxYlZgo8Nis7TY+vvhq9neioYrjgJDBIAwKk8wHzYMibWWCBKiKskUgYzH\\\\\\\/bmKgmPTFVdItezOVGHkCiExSlzHnTvdF5DF9RzFq50XSX\\\\\\\/KuSt6g4x4j3NwTwqEYL4q+ASdxQhD1rehbH2edLsXzmjk0wKe3OdR+Zjd52yDLCZyQaob6Eb1kKUB+eH4n9TrqRmT9w4bcdou1xMAKhQbGTnW\\\\\\\/RQ6\\\\\\\/Ct8MH7wWWuWeJ\\\\\\\/PmyW190NdVRtQFx3qvFtzYUdGzUPrhIRSeY3\\\\\\\/uvcItGGN2uQDqiIStHDx9St74TOP0eRgyqL0i1Cqfld\\\\\\\/9IbWMCvP5fp7vpvN2kFQe9\\\\\\\/gnZ0KP8Sn9Pyv2UniA85SM0ERJBM9rWztYYQIbn3Cd14lUOqYBmozpqGJaW4D0tr+7tRbiigFnu8ldywU1DZNCAG8w3JFqbL7tbSZiW+aL\\\\\\\/pqjHgk9qe\\\\\\\/YrW5jasvJhJZ9P8y7D5bas8KQIcqPQaLlDTOtsOjmPl8mJzodAWziIeND8uINdZXUcg11KVTPpeOSYXsODG\\\\\\\/xM82mmSTCeqOPrOgU2FSDHepfvlAQzJ1vsBMXrqA\\\\\\\/k5PpLcs7KFaHKC9f4HN+xGO8hqkuaX8OdbN92VUcB6r6IpoGHldoIponu6Yt7IGWr1KClGpkm7VpcEpjNfwEpl2ax5F5bjnZNMwwAGOqtrDk0dqbvfrrpZWJqIB\\\\\\\/urAWm3IycunrAsRnh\\\\\\\/YZqn\\\\\\\/viSAgJ9xCe7\\\\\\\/i5pqgGOAq85Lcl+\\\\\\\/zTbi\\\\\\\/Xje20ILSNIkd4WDqFbOEfY4dHaITH5cUZI64NFUCSyHkjnVpSKpzND1jfzhuVPPYf0ewesBMyyVg\\\\\\\/lQ0FC2xawCKHGE2rHhDcGWZi8Ui8sWycnjUzsaSadM\\\\\\\/jkVIWfqkT7IhttEWh71FRDJ2JTBcYbTDyZUHLvOjnHDHEasOz24lHa+48u+5dwrpJt2DDHDcBDdeSPMK8Eks3NCguts5NLfStG8aRPWkCAU7rg1kVAHJ2rgNQHIJdejzJSxvPOhQpRLEZLVi+3bHC5Njki5ZDkCftFFfTydlFY937S2Y6SgfTASuv2i4mH6MLz1P3SGyx0IuBBPIwVPff8tiOCiBupTed5YeCwNgiABwqtZFcO+G783Gnm3l2\\\\\\\/NuXZjPpItTheabqmnL\\\\\\\/Lzsl4V9vRliIMrDp5gcPakbsok8YUxL5GRWhdDnA9g+9LDgzCwvjgddiE5UJE1v\\\\\\\/IcrXzfQCwWCK5huyswFbXRb\\\\\\\/+oqlKknpIt4Qim8ZupJc2gBaK0o\\\\\\\/wuuqLaRTR6+6ew8S\\\\\\\/MzdzRJSRs1ICz4KmpAxUzE3ITLgnAM\\\\\\\/fk3TSk1LrS+SmBIQENXzI5kOiFGn4g8D5DyHU01WilIKTcWOu0ZUDIGhRBsKpJk09hf3G53r0kLNtE\\\\\\\/Yg53kl22nOIY3vFoH8nLIBTnE9KrSTHpFCQlCEU9qXCsxae8TBLnxP+hH8fRkGhD8S6HKgRnVz49PN7ZVM6gnJ+EtepVE+YylbEsrtrf5qv\\\\\\\/s23XnWoV4O\\\\\\\/zSOgWkIDfInRQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022bvMLicqI5yyHBhkA5RqmXg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00227c437d5788a05e28da966c4141d56fa0037d43caa4d0e5d7aee097dd5f32cd8d\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022D7arPuNeYgPAcnBqyVRkcy2\\\\\\\/M2k5mjjc22jM85LwCMB+OakPCM+IueDaar99PmzHVE9h\\\\\\\/rBgZCgj1iha+73ldR5v2kxOcIYZtoI7r7CKYihD9iWAZnLqXX2PWqGnzucE4ucRTxn1eTldQczz0C58ec+WLl\\\\\\\/xyDhwu+p4BgxXA4DSrnSf2pI1hpcActuGA4uqmhGp7+JilVJRkCYYcq5+YWvVrU\\\\\\\/x9zdSL2g13Gx7hVuCVdc8bE1Os0\\\\\\\/zFEBH3KeHZyhVxtkFeaOmOtlOmyVjYXS5dQhaXj3UnGZ6lRPZXhb0dSjhnpdQl6X9YukH1sU6li+GCUp11f5SWVAjPdJPXxBS59s+XgFeoWDYz8nanh1jtkYsrtytULL0nEBGXrO6AxuZFBWpiAicG3iNRNvvmuQ3xpqaSVko+XW7JwVu1oFpYmj+75gLHXj6hwcTaZL\\\\\\\/s2TGSNCiSKvmcx60xtNiShAXILe4ToONgQZlEJrJb21fJMc5h\\\\\\\/Bh2TLE+NzT07UN95SVqFDdpnxgiK2RAgTFUbEYZKkmUMpU6CCS7WRsr0X7i0sYzYMeE9AGdBPibBYeBF1jjQvhnGKiQTJ6szr2nE5TTfT30ox6FyZtgZ6Zf5D9ESnU+ABiysqTxoHAVcLMiEbEIy8oolSQO6Z7goWfmfi3uaV4r3DbtlVrehUltWqgYUhkMyjWMYqSUwoI7ZhhvoJMvVGnz3ICZXvchlFSWu3FwNGULCmPYHD82OIkaTw4G9kgtaSyfa\\\\\\\/I+mcK14Xd9cp4OLF0u3IrTMKpYTS43lduo1HRcFQpQDZ7hZLUJoJ\\\\\\\/\\\\\\\/z5iuFDY9ZICMkmwFed3tYFZbiDVyN50RbKUITcSljCokxutxZrqdMie5oDLiRlk7sTCdVqeRr94Ggo0OIgS6S0MbRuFKlDpZTOBijHMg\\\\\\\/EMAUSACDBGsQni5G+fBATPfORu83tDlhIO5Q6HgI0ct2gQFnpHAlT754PHbvPlzSOU8pQDMryZ2ooZuL8G0mFNoH+UOG60+SWZ4WNLqC5hjKUHPnr0N414jM\\\\\\\/EPlN7WtT6m1jNTVevoUlX81qWS\\\\\\\/d0dy9z8Dym2kE6GqTWnMHF1QixQsjjtrSXbrnBCQyORBRmJAXgbK4aPWd8e2S+oYKYoHSLsKwFW4Rh5VI3pMUXMnYIpWbRXItk9mLcqlHRb6ykp8CYB+SitTsXsk8ObWumS3bixCzBerSpmlvBexYvOwGt0byFY539tDeo86SzbQbAvHqFy+IJNo79rsI3fyvJ\\\\\\\/Er1SPJDPD+ro4Vv+6Xb9RDdVOIkzFLiipDVuJWYIOcVyTbJf0aRSsYjkEsw\\\\\\\/rfMBZgZc+O5BZkOHFuVmM9qidLFO9wOH4UPcJZRFeu75mI4Tt00HyU3upvLxQc+0nk4ZvJwm3b3KmrglIG81Yyt6eKvYhHEFoSW3lBoHj20bv0vFzLDpf5r63qgLr1hw28Bh9F\\\\\\\/yTYAbjKQoe3FRyuY\\\\\\\/iTtaUuSO60F2gNYm8Jq2vSJPqaD0wd1UWdcayEWlnTlqLfu0pplPFdBvV8iH7BTEGZUQRh1ffcat3TaHchJNXXfPA1c1ALcfJoOaqcKXVSlKPunowsqgCAcK7xDh3WHkKy29pUCIR2648t6zhbp1oRuu3btVI5RRu\\\\\\\/Iajt8QCNa6aH3irLKqSzMqFcqeaVRpAXvMO0Pd68iRGw5qrGzI6lw+61+xCaoNTNPIdYTt1QnfoVygxY2eB7OUcpuDW+MuZCJx9MdOTCazZxauv24D2KcxaRxpp345U4DvSvusxmcWiFrTDUjnsM3SPQJM4HLr77Rmo8zbkBkiufSLCOIInCRAF7zTCQ0cMZJuRsv7ENwhsMoMgVXhz\\\\\\\/puAZzOpfmuIGGzgXtWFCLf6Wm801YCxAjPaU7hGsRwiF5Jhl1Wkk4YAgGQYUvTvXiWOumbnX7VBfokI7XI\\\\\\\/5YvAuCiYHkear1iKOsc2Wlslzyg8wgHBKFvHf+xzDRa\\\\\\\/itNQrAoDbKfZAzubOWaKYoJm8Grzirkw==\\\u0022,\\\u0022iv\\\u0022:\\\u00229DcSVhvI8zPlarYSyJwSHQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00226c391b78dc15a88d8fb5774af45817a23f7dc80d44331934bf242855fbddbc60\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-491246 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/11/3583" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-v2O9laXwzAtwCo1k1Tt2erbLG5JwFrAz8TJMmo3Jcd8" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/11/3583" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Yhrudj6wvsgojCk21tG1VGlhnK7_kKw48n_QDDWsaY4" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/11/3577" title="Estimating Asian Contribution to the Brazilian Population: A New Application of a Validated Set of 61 Ancestry Informative Markers" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/11/3593" title="A Drosophila CRISPR/Cas9 Toolkit for Conditionally Manipulating Gene Expression in the Prothoracic Gland as a Test Case for Polytene Tissues" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="491246" id="node491246" data-pisa="ggg;8/11/3583" data-pisa-master="ggg;g3.118.200499" data-apath="/ggg/8/11/3583.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Bulked-Segregant Analysis Coupled to Whole Genome Sequencing (BSA-Seq) for Rapid Gene Cloning in Maize</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Harry Klein</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-0747-5856" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Yuguo Xiao</span>, <span class="highwire-citation-author" data-delta="2">Phillip A. Conklin</span>, <span class="highwire-citation-author" data-delta="3">Rajanikanth Govindarajulu</span>, <span class="highwire-citation-author" data-delta="4">Jacob A. Kelly</span>, <span class="highwire-citation-author" data-delta="5">Michael J. Scanlon</span>, <span class="highwire-citation-author" data-delta="6">Clinton J. Whipple</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0002-0369-8606" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Madelaine Bartlett</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3583-3592;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200499 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node491246" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Harry Klein </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Biology Graduate Program and Biology Department, University of Massachusetts Amherst, Amherst, MA 01003</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Harry%2BKlein%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Klein%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHarry%2BKlein%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Yuguo Xiao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biology, Brigham Young University, 4102 LSB, Provo, UT 84602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yuguo%2BXiao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xiao%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYuguo%2BXiao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-0747-5856" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Yuguo Xiao</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Phillip A. Conklin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Plant Biology, Cornell University, Ithaca, NY 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Phillip%2BA.%2BConklin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Conklin%20PA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APhillip%2BA.%2BConklin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Rajanikanth Govindarajulu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Biology, West Virginia University, Morgantown, WV 26506</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rajanikanth%2BGovindarajulu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Govindarajulu%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARajanikanth%2BGovindarajulu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Jacob A. Kelly </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biology, Brigham Young University, 4102 LSB, Provo, UT 84602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jacob%2BA.%2BKelly%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kelly%20JA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJacob%2BA.%2BKelly%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Michael J. Scanlon </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Plant Biology, Cornell University, Ithaca, NY 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BJ.%2BScanlon%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Scanlon%20MJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BJ.%2BScanlon%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Clinton J. Whipple </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biology, Brigham Young University, 4102 LSB, Provo, UT 84602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Clinton%2BJ.%2BWhipple%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Whipple%20CJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AClinton%2BJ.%2BWhipple%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Madelaine Bartlett </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Biology Graduate Program and Biology Department, University of Massachusetts Amherst, Amherst, MA 01003</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Madelaine%2BBartlett%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bartlett%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMadelaine%2BBartlett%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-0369-8606" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Madelaine Bartlett</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mbartlett@bio.umass.edu" class="" data-icon-position="" data-hide-link-title="0">mbartlett@bio.umass.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/8/11/3583" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:491246" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:491246/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/11/3583.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:491246" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:491246/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/11/3583.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:491246" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:491246/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/11/3583.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:491246" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:491246/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/8/11/3583.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:491246" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:491246/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Forward genetics remains a powerful method for revealing the genes underpinning organismal form and function, and for revealing how these genes are tied together in gene networks. In maize, forward genetics has been tremendously successful, but the size and complexity of the maize genome made identifying mutant genes an often arduous process with traditional methods. The next generation sequencing revolution has allowed for the gene cloning process to be significantly accelerated in many organisms, even when genomes are large and complex. Here, we describe a bulked-segregant analysis sequencing (BSA-Seq) protocol for cloning mutant genes in maize. Our simple strategy can be used to quickly identify a mapping interval and candidate single nucleotide polymorphisms (SNPs) from whole genome sequencing of pooled F2 individuals. We employed this strategy to identify <em>narrow odd dwarf</em> as an enhancer of <em>teosinte branched1</em>, and to identify a new allele of <em>defective kernel1</em>. Our method provides a quick, simple way to clone genes in maize.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/forward-genetics" class="hw-term hw-article-keyword hw-article-keyword-forward-genetics" rel="nofollow">forward genetics</a></li><li class="kwd"><a href="/keyword/maize-genetics" class="hw-term hw-article-keyword hw-article-keyword-maize-genetics" rel="nofollow">maize genetics</a></li><li class="kwd"><a href="/keyword/mutant-genes" class="hw-term hw-article-keyword hw-article-keyword-mutant-genes" rel="nofollow">mutant genes</a></li><li class="kwd"><a href="/keyword/bulked-segregant-analysis-0" class="hw-term hw-article-keyword hw-article-keyword-bulked-segregant-analysis" rel="nofollow">bulked-segregant analysis</a></li><li class="kwd"><a href="/keyword/next-generation-sequencing" class="hw-term hw-article-keyword hw-article-keyword-next-generation-sequencing" rel="nofollow">next generation sequencing</a></li></ul><p id="p-4">Forward genetics remains a powerful way to ‘ask the plant’ which genes matter for a particular trait or phenotype (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Mueller 2006</a>). Because forward genetics relies on random mutagenesis, it presents an unbiased method for identifying novel genes that act in particular pathways. While forward genetic screens can reveal new genes, they can also reveal novel functions for known genes; providing a richer, more deeply nuanced view of gene function essential for a true understanding of how genes and gene networks contribute to building an organism (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Nawy <em>et al.</em> 2010</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Gallavotti <em>et al.</em> 2010</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Vlad <em>et al.</em> 2014</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Gillmor <em>et al.</em> 2016</a>). After random mutagenesis, mutant genes are most often identified through linkage mapping.</p><p id="p-5">Linkage mapping, using a combination of bulked-segregant analysis (BSA) and fine mapping, has been very successful for cloning maize genes (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Gallavotti and Whipple 2015</a>). The process starts when a mutant of interest is crossed to a wild-type individual in a contrasting genetic background. The resulting F1 individuals are selfed or backcrossed, and mutants are identified in the F2 or backcross population. These mutants can be used to identify a region of increased homozygosity in a chromosomal region physically linked to the lesion causing the mutant phenotype. This region of increased homozygosity can be rapidly detected using bulked-segregant analysis (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Michelmore <em>et al.</em> 1991</a>).</p><p id="p-6">In bulked-segregant analysis, pools of wild-type and mutant individuals are genotyped at markers spread across the genome. In chromosomal regions not linked to the mutant lesion, markers will be segregating according to typical 1:2:1 (or 1:1 in backcross) segregation ratios. In a pooled BSA sample, these unlinked markers will all be genotypically heterozygous. Linked markers will be homozygous for the mutant parent genotype, unless F1 recombination has happened between the mutant lesion and a particular marker. A chromosomal region enriched for these linked markers represents a likely location for the mutant gene under study, and the coordinates of these linked markers define an initial mapping interval. Once this initial mapping interval is identified, F1 recombination is again used to place recombination breakpoints in individual mutants, and thus narrow the mapping interval using fine mapping (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Gallavotti and Whipple 2015</a>). The fine mapping process is often time consuming, and identifying causative lesions and mutant genes can take years.</p><p id="p-7">The advent of next-generation sequencing (NGS) means that BSA can be used to very quickly identify a very small mapping interval, and even the causative lesion, without fine mapping. In BSA-Seq, whole genome shotgun sequencing using NGS can be used to quickly genotype mutant <em>vs.</em> non mutant BSA pools at many thousands of markers spread across the genome. These BSA-Seq data can reveal the genomic interval that contains the mutant gene of interest, and with enough coverage, the lesion itself (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Zou <em>et al.</em> 2016</a>). Since random mutagenesis produces lesions throughout the genome, and since NGS does not depend on established genotyping assays, BSA-Seq can be used to identify a chromosomal region using just the polymorphisms induced by mutagenesis. In this case, the contrasting genetic background used for the BSA can simply be that of an unmutagenized parent. This modification to BSA-Seq has been called MutMap (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Abe <em>et al.</em> 2012</a>). BSA-Seq (and MutMap) has been used to identify mutant loci in <em>Arabidopsis thaliana</em>, soybean, barley, <em>Mimulus</em>, rice, sorghum, and <em>Brachypodium distachyon</em> (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Schneeberger and Weigel 2011</a>; <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Abe <em>et al.</em> 2012</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Mascher <em>et al.</em> 2014</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Woods <em>et al.</em> 2014</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Addo-Quaye <em>et al.</em> 2017</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Ding <em>et al.</em> 2017</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Song <em>et al.</em> 2017</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Jiao <em>et al.</em> 2018</a>). In maize, BSA coupled to RNA-Seq (BSR-Seq) has been used successfully to clone mutant genes (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Liu <em>et al.</em> 2012</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Li <em>et al.</em> 2013</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Nestler <em>et al.</em> 2014</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Tang <em>et al.</em> 2014</a>). Conventional BSA-Seq has been used to identify genomic regions underlying variation in flowering time and plant height QTL in a maize population (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Haase <em>et al.</em> 2015</a>), but cloning mutant genes using BSA-Seq is not yet routine in maize.</p><p id="p-8">Here, we report a user-friendly protocol for cloning mutant maize genes using BSA-Seq. We used this protocol to clone two genes recovered from EMS mutagenesis screens. We used conventional BSA-Seq in the case of one gene, and MutMap in the other. We identified <em>narrow odd dwarf</em> (<em>nod</em>) as an enhancer of <em>teosinte branched1</em> (<em>tb1</em>) using conventional BSA-Seq, and identified a seedling lethal allele of <em>defective kernel1</em> (<em>dek1</em>) using MutMap (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Doebley <em>et al.</em> 1997</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Lid <em>et al.</em> 2002</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Becraft <em>et al.</em> 2002</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Rosa <em>et al.</em> 2017</a>). In addition, we used our <em>tb1</em> enhancer data to design insertion-deletion (indel) markers for downstream fine mapping. This fine mapping to reduce the recombination interval is critical in cases where only a single allele for a particular mutant exists, and in cases where no clear candidate lesion is identified. Our method provides a quick, easy method for cloning mutant genes in maize.</p><div class="section methods" id="sec-1"><h2 class="">Methods</h2><div id="sec-2" class="subsection"><h3>Plant material and isolation of mutants</h3><p id="p-9">To identify enhancers and suppressors of the <em>tb1</em> (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Doebley <em>et al.</em> 1997</a>) and <em>narrow sheath</em> (<em>ns)</em> (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Scanlon <em>et al.</em> 1996</a>) phenotypes, we performed EMS mutagenesis screens. <em>tb1</em> encodes a TCP transcription factor with a well characterized role in suppressing axillary branching or tillering (<a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Doebley <em>et al.</em> 1997</a>). The <em>ns</em> loci regulate the initiation of lateral cells in the maize shoot apical meristem. <em>ns1</em> and <em>ns2</em> encode duplicate homeodomain transcription factors, homologous to <em>PRESSED FLOWER</em> in <em>Arabidopsis thaliana</em> (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Nardmann <em>et al.</em> 2004</a>). We mutagenized pollen homozygous for a weak allele of <em>tb1</em> in the A619 inbred (<em>tb1-sh</em>), and <em>ns</em> in an unknown background obtained from Pioneer Hi-Bred Intl. (<em>ns1;ns2</em>), using established protocols (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Neuffer <em>et al.</em> 1997</a>). After mutagenesis, M1 progeny were selfed to generate M2 populations, where we identified the mutants <em>tb1 enhancer</em> (<em>ten</em>) and <em>very narrow sheath</em> (<em>vns</em>). <em>ten</em>, which arose in the A619 genetic background, was crossed to the B73 genetic background and then selfed to generate an F2 mapping population (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1a</a>). <em>vns</em> is seedling-lethal. Therefore a wild-type sibling, heterozygous at <em>vns</em>, was backcrossed to a <em>ns1</em> heterozygote fixed for <em>ns2 (ns1/NS1;ns2/ns2)</em>, and the progeny selfed to generate a mapping population (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1b</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/11/3583/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Crossing scheme to generate ten and vns mapping populations. (A) For ten, we crossed a homozygous ten tb1-sh double mutant (in A619) to a wild-type individual of a widely divergent genotype (B73), then selfed many plants in the resulting F1. We selected ten mutants in the F2 generation, regardless of the tb1 phenotype. We pooled tissue from 101 of these mutants into a single DNA extraction and a single library for whole genome shotgun sequencing. (B) For vns, we backcrossed a plant heterozygous at vns to a wild-type individual of the unmutagenized parental genotype. We selfed many individuals in the resulting F1 (50% heterozygous at vns). We selected 9 vns mutants in the F2 generation, regardless of the ns phenotype, and pooled tissue into a single DNA extraction and NGS library." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2116734167" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Crossing scheme to generate &lt;em&gt;ten&lt;/em&gt; and &lt;em&gt;vns&lt;/em&gt; mapping populations. (A) For &lt;em&gt;ten&lt;/em&gt;, we crossed a homozygous &lt;em&gt;ten tb1-sh&lt;/em&gt; double mutant (in A619) to a wild-type individual of a widely divergent genotype (B73), then selfed many plants in the resulting F1. We selected &lt;em&gt;ten&lt;/em&gt; mutants in the F2 generation, regardless of the &lt;em&gt;tb1&lt;/em&gt; phenotype. We pooled tissue from 101 of these mutants into a single DNA extraction and a single library for whole genome shotgun sequencing. (B) For &lt;em&gt;vns&lt;/em&gt;, we backcrossed a plant heterozygous at &lt;em&gt;vns&lt;/em&gt; to a wild-type individual of the unmutagenized parental genotype. We selfed many individuals in the resulting F1 (50% heterozygous at &lt;em&gt;vns&lt;/em&gt;). We selected 9 &lt;em&gt;vns&lt;/em&gt; mutants in the F2 generation, regardless of the &lt;em&gt;ns&lt;/em&gt; phenotype, and pooled tissue into a single DNA extraction and NGS library.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/11/3583/F1.medium.gif" width="440" height="225"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/8/11/3583/F1.medium.gif" width="440" height="225"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/11/3583/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/11/3583/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491418" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-10" class="first-child">Crossing scheme to generate <em>ten</em> and <em>vns</em> mapping populations. (A) For <em>ten</em>, we crossed a homozygous <em>ten tb1-sh</em> double mutant (in A619) to a wild-type individual of a widely divergent genotype (B73), then selfed many plants in the resulting F1. We selected <em>ten</em> mutants in the F2 generation, regardless of the <em>tb1</em> phenotype. We pooled tissue from 101 of these mutants into a single DNA extraction and a single library for whole genome shotgun sequencing. (B) For <em>vns</em>, we backcrossed a plant heterozygous at <em>vns</em> to a wild-type individual of the unmutagenized parental genotype. We selfed many individuals in the resulting F1 (50% heterozygous at <em>vns</em>). We selected 9 <em>vns</em> mutants in the F2 generation, regardless of the <em>ns</em> phenotype, and pooled tissue into a single DNA extraction and NGS library.</p><div class="sb-div caption-clear"></div></div></div><p id="p-11">For BSA-seq, tissue samples were collected from 101 <em>ten</em> mutant individuals, and 9 <em>vns</em> mutant individuals in the mapping populations. We used this small number of <em>vns</em> individuals because that was what we had available from a small mapping population. Mutants in the F2 mapping populations were sampled independent of their genotypes at <em>tb1</em> or <em>ns1</em> (<em>i.e.</em>, neither <em>ten</em> nor <em>vns</em> was dependent on <em>tb1</em> or <em>ns</em>, respectively). Because the <em>vns</em> genetic background is unknown, we also collected tissue samples for 9 wild-type siblings in the <em>vns</em> mapping population and for the parents of the EMS mutagenesis screen. To characterize the genetic interaction between <em>tb1</em> and <em>ten</em>, we counted tillers in families segregating <em>tb1-sh</em> and <em>ten</em>.</p></div><div id="sec-3" class="subsection"><h3>DNA extractions and NGS sequencing</h3><p id="p-12">Tissue samples were pooled by hole punching each leaf twice to ensure equal representation of individuals. We extracted DNA from the pools of tissue using a CTAB method (<a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">Gallavotti and Whipple 2015</a>). Libraries were prepared using the TruSeq DNA sample prep kit according to the manufacturer’s instructions (Illumina). We sequenced the <em>ten</em> mutant pool on an Illumina Hi-Seq 2500 at Brigham Young University. Sequenced reads were 125bp long with paired-ends. We sequenced <em>vns</em> mutants, <em>vns</em> wild-type siblings, and unmutagenized parents on an Illumina Hi-Seq 2500 at Cornell University. Sequenced reads from <em>vns</em> were 150bp long with paired ends. Different sequencing platforms were used because of differing availability at our respective institutions.</p></div><div id="sec-4" class="subsection"><h3>NGS quality control and read alignment</h3><p id="p-13">We used the genomic tools hosted on Galaxy web portal to process our Next-Gen sequencing data (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Afgan <em>et al.</em> 2016</a>). An overview of our process is shown in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>. We used FastQC (v. 0.69) to determine the quality of our sequencing data, and established a PHRED quality cutoff of 20 based on this analysis (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Ewing <em>et al.</em> 1998</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Andrews 2014</a>). We used FASTQ Groomer (v. 1.1.1) to convert our FASTQ files to Sanger format for input into downstream Galaxy tools (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Blankenberg <em>et al.</em> 2010</a>). Trimmomatic (v. 0.36.3) was used for sliding window trimming averaged across 4 base pairs with average quality &gt;20 and for adapter sequence removal (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Bolger <em>et al.</em> 2014</a>). The Galaxy tool cat, which concatenates datasets tail-to-head (cat) (v. 0.1.0) was used to join sequencing files together if data were split from multiple Illumina sequencing runs (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Afgan <em>et al</em>. 2018</a>). We used Bowtie2 (v. 2.3.2.2) to align sequencing reads to the B73 reference genome version 4, release 56 and to generate an index (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Langmead and Salzberg 2012</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Jiao <em>et al.</em> 2017</a>). Read alignment was assessed with Mtools Flagstat (v. 2.0) (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Li <em>et al.</em> 2009</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/11/3583/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A simplified overview of our BSA-Seq pipeline. (A) We analyzed our NGS data in Galaxy and in R to identify chromosomal regions that contained the mutant genes. (B) SNP filtering to identify candidate (EMS) SNPs." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2116734167" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;A simplified overview of our BSA-Seq pipeline. (A) We analyzed our NGS data in Galaxy and in R to identify chromosomal regions that contained the mutant genes. (B) SNP filtering to identify candidate (EMS) SNPs.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/11/3583/F2.medium.gif" width="440" height="313"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/8/11/3583/F2.medium.gif" width="440" height="313"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/11/3583/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/11/3583/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491404" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-14" class="first-child">A simplified overview of our BSA-Seq pipeline. (A) We analyzed our NGS data in Galaxy and in R to identify chromosomal regions that contained the mutant genes. (B) SNP filtering to identify candidate (EMS) SNPs.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-5" class="subsection"><h3>Variant calling</h3><p id="p-15">SAMtools mpileup (v. 2.1.3) was used to output variants from the indexed bam file into pileup or variant call format (vcf) files (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Li <em>et al.</em> 2009</a>). Pileup files were generated for identification of the mapping region and vcf files were generated for identification of candidate SNPs. To filter variant files for mapping region identification, we used SAMtools filter pileup (v. 1.0.2), a custom Galaxy tool, to filter based on read coverage and quality (<a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">Li <em>et al.</em> 2009</a>). We filtered out non-variant positions, and focused only on variant positions with coverage of 8 or more reads, each with a quality score of 20 or more. To generate vcf files for candidate SNP identification, we used Varscan (v. 0.1). We used a minimum homozygous calling frequency of 0.99, and filtered out SNPs with low read coverage (coverage &lt;8) (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Koboldt <em>et al.</em> 2012</a>). We used this coverage cutoff to reduce artifacts from sequencing and/or mapping error, and 8 was well below the coverage means for both of our datasets.</p><p id="p-16">We also used our mapping data to generate an A619 variant file. We used Varscan (v. 0.1) to call SNPs in <em>ten</em> and 4 other A619 mutants. We did not assign a coverage cutoff for these datasets, and retained only SNPs that had a paired read. We used the function findCommonVariants in the Rsubread package (Shi <em>et al.</em> 2013) to identify SNPs common to all 5 datasets (File S1).</p></div><div id="sec-6" class="subsection"><h3>Data processing for plotting and SNP filtering</h3><p id="p-17">Each filtered pileup file was split into 10 chromosomes. At each nucleotide position in the pileup file, we calculated variant allele frequency by dividing the number of reads that differed from the B73 reference sequence by the quality adjusted total number of reads at that position. This variant allele frequency was added as a new column in the pileup file. Nucleotide position, reference allele, alternate allele, coverage, and variant allele frequency were extracted from the pileup file to create a variant file for downstream analysis in R (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Wickham and Francois 2015</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Wickham 2016</a>).</p></div><div id="sec-7" class="subsection"><h3>Mapping region identification</h3><p id="p-18">We plotted our data against the B73 reference genome, to identify chromosomal regions enriched for homozygous SNPs, presumably in linkage with the causative lesions. Before plotting, we filtered positions with coverage &gt;100 to remove highly enriched positions in our data. High coverage SNPs are likely in repeat rich regions and will not be informative for mapping region identification (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Addo-Quaye <em>et al.</em> 2017</a>). We used ggplot2 to plot the number of homozygous variant positions that differed from the B73 reference genome, per 1Mbp chromosomal bin (<a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">Wickham and Francois 2015</a>; <a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Wickham 2016</a>). For these analyses, nucleotide positions that differed from the reference genome at a frequency greater than or equal to 0.99 were defined as homozygous variants. For <em>vns</em>, we went on to remove SNPs that were in the wild-type sibling and unmutagenized parent datasets, and plot only those homozygous SNPs that were unique to the mutant pool. For both <em>ten</em> and <em>vns</em>, we also plotted the number of homozygous canonical EMS variants (G to A and C to T transitions) per 1Mbp bin. We defined the limits of the <em>ten</em> and <em>vns</em> mapping intervals as those chromosomal coordinates where variant frequency returned to background levels.</p></div><div id="sec-8" class="subsection"><h3>SNP filtering for candidate SNP identification</h3><p id="p-19">Once we had identified likely mapping intervals for <em>ten</em> and <em>vns</em>, we searched for potentially causative lesions in each of these intervals. Here, we returned to the vcf file generated using Varscan. Although we could have investigated the filtered EMS SNPs from our pileup file used to determine the mapping region directly, we chose to use Varscan because it outputs a vcf file, needed for downstream candidate SNP analyses using SnpEff (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Cingolani <em>et al.</em> 2012</a>). We filtered out all background A619 SNPs (File S1) from the <em>ten</em> dataset, and all parental and homozygous wild-type SNPs from the <em>vns</em> dataset. We reasoned that it was highly unlikely that the causative SNPs could be any known SNP present in any maize inbred. Therefore, we applied an additional SNP filtering step, and removed all the maize Hapmap 3.2.1 SNPs from both datasets, and our A619 SNPs from <em>vns</em> (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Merchant <em>et al. 2016</em></a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Bukowski <em>et al. 2018</em></a><em>)</em>. Once we had a set of positions enriched for SNPs unique to each of our particular mutant pools, we filtered out SNPs that were not homozygous and were not canonical EMS changes (G to A or C to T) (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Till <em>et al.</em> 2004</a>). The final set of homozygous EMS SNPs unique to each dataset was the input for an analysis of likely SNP effects on gene function.</p><p id="p-20">To identify SNPs in our filtered datasets that might negatively affect gene function, we used SnpEff (version 4.3a). SnpEff identifies nonsense SNPs and splice site mutations as likely to have highly deleterious effects on gene function, and all missense SNPs as likely to have moderate effects on gene function (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Cingolani <em>et al.</em> 2012</a>). Functional annotations for the genes disrupted by likely moderate- and high-effect candidate SNPs were obtained from Gramene (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Tello-Ruiz <em>et al.</em> 2018</a>). Candidate SNPs were validated through Sanger sequencing and/or complementation crosses.</p></div><div id="sec-9" class="subsection"><h3>Indel marker design and fine mapping</h3><p id="p-21">To refine our <em>ten</em> mapping interval, we designed custom markers for fine mapping. We used Varscan (v. 0.1) to identify insertion and deletion polymorphisms (indels) in our mapping interval that differentiated our mutant pool from the B73 reference genome. We chose indels that were 15 base pairs in length or longer, and designed primers to flank these indels by 100-150bp (Table S1). DNA was extracted from each of the 101 F2 mutants originally pooled into one NGS library and used for Indel PCR (<a id="xref-ref-15-4" class="xref-bibr" href="#ref-15">Gallavotti and Whipple 2015</a>). Size differences between the resulting PCR products were detected on 3.5% agarose gels.</p></div><div id="sec-10" class="subsection"><h3>Data availability</h3><p id="p-22">Raw sequencing data available at NCBI SRA (<span class="underline"><a href="https://www.ncbi.nlm.nih.gov/bioproject/PRJNA476333">https://www.ncbi.nlm.nih.gov/bioproject/PRJNA476333</a></span>). A protocol for NGS data analysis and plotting in R is available at protocols.io (<span class="underline"><a href="https://www.protocols.io/view/bsa-seq-in-maize-qyedxte">https://www.protocols.io/view/bsa-seq-in-maize-qyedxte</a></span>). Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.7014851">https://doi.org/10.25387/g3.7014851</a>.</p></div></div><div class="section results" id="sec-11"><h2 class="">Results</h2><div id="sec-12" class="subsection"><h3>Mutant phenotypes</h3><p id="p-23">From an EMS mutagenesis screen of <em>tb1-sh</em> mutants in the A619 genetic background, we recovered a dwarf mutant with additional tillers that we called <em>tb1 enhancer</em> (<em>ten</em>) (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3a</a>). <em>very narrow sheath</em> (<em>vns</em>) was uncovered in an independent EMS mutagenesis screen. <em>vns</em> single mutants are seedling lethal and often fail to develop more than 3 leaves (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3c</a>). Although <em>vns</em> was isolated in an <em>ns</em> enhancer screen, <em>vns</em> did not enhance <em>ns</em>.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/11/3583/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="ten and vns were identified in EMS mutagenesis screens. (A) ten (left), tb1-sh ten (middle), tb1-sh (right). (B) Quantification of tillers in a 9:3:3:1 population of ten and tb1-sh shows a synergistic interaction between ten and tb1-sh. (C) Wild-type (left) and vns mutant (right) exhibiting reduced growth at the seedling stage." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2116734167" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;em&gt;ten&lt;/em&gt; and &lt;em&gt;vns&lt;/em&gt; were identified in EMS mutagenesis screens. (A) &lt;em&gt;ten&lt;/em&gt; (left), &lt;em&gt;tb1-sh ten&lt;/em&gt; (middle), &lt;em&gt;tb1-sh&lt;/em&gt; (right). (B) Quantification of tillers in a 9:3:3:1 population of &lt;em&gt;ten&lt;/em&gt; and &lt;em&gt;tb1-sh&lt;/em&gt; shows a synergistic interaction between &lt;em&gt;ten&lt;/em&gt; and &lt;em&gt;tb1-sh&lt;/em&gt;. (C) Wild-type (left) and &lt;em&gt;vns&lt;/em&gt; mutant (right) exhibiting reduced growth at the seedling stage.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/11/3583/F3.medium.gif" width="440" height="173"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/8/11/3583/F3.medium.gif" width="440" height="173"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/11/3583/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/11/3583/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491284" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-24" class="first-child"><em>ten</em> and <em>vns</em> were identified in EMS mutagenesis screens. (A) <em>ten</em> (left), <em>tb1-sh ten</em> (middle), <em>tb1-sh</em> (right). (B) Quantification of tillers in a 9:3:3:1 population of <em>ten</em> and <em>tb1-sh</em> shows a synergistic interaction between <em>ten</em> and <em>tb1-sh</em>. (C) Wild-type (left) and <em>vns</em> mutant (right) exhibiting reduced growth at the seedling stage.</p><div class="sb-div caption-clear"></div></div></div><p id="p-25">To determine the nature of the genetic interaction between <em>ten</em> and <em>tb1</em>, we counted tillers in a population segregating both <em>tb1-sh</em> and <em>ten</em>. Tiller number (per plant) was counted in 15 <em>tb1-sh</em> single mutants, 15 <em>ten</em> single mutants and 15 <em>tb1-sh</em>; <em>ten</em> double mutants. <em>tb1-sh</em> and <em>ten</em> single mutants had an average of 4 and 2 tillers respectively. <em>ten tb1-sh</em> double mutants had an average of 9 tillers, showing a synergistic interaction between <em>ten</em> and <em>tb1-sh</em> in tiller development (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3b</a>).</p></div><div id="sec-13" class="subsection"><h3>Both ten and vns are on chromosome one</h3><p id="p-26">To identify the causative lesions underlying the <em>ten</em> and <em>vns</em> mutant phenotypes, we sequenced pooled DNA from 101 <em>ten</em> individuals and 9 <em>vns</em> individuals from F2 mapping populations (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>). We sequenced <em>ten</em> pools and <em>vns</em> pools on an Illumina platform with outputs of 125 bp and 150bp paired end reads, respectively. For <em>ten</em>, we recovered 450,665,452 reads, and for <em>vns</em> we recovered 311,711,046 reads. For <em>ten</em>, 97% of our reads (438,746,898) mapped to version 4 of the maize genome, and 91% were properly paired (409,548,662). For <em>vns</em>, 90% of our reads (279,239,524) mapped, and 77.5% were properly paired (241,580,812). The average coverage of all <em>ten</em> and <em>vns</em> mapped reads was 26-fold and 17-fold respectively (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). The combined dataset that we used to generate the A619 SNP dataset included 1.1 billion mapped reads, for a mean coverage of 59-fold (File S1).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/491328/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/491328/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Sequencing data summary for <em>ten</em> and <em>vns</em></span><div class="sb-div caption-clear"></div></div></div><p id="p-27">To identify the chromosomal regions corresponding to <em>ten</em> and <em>vns</em>, we searched for regions of the genome enriched for homozygous variants in the mutant pools. For <em>ten</em>, we plotted the number of homozygous SNPs in the NGS data that differed from B73 (variant frequency &gt;=0.99), per 1 Mbp bin. This plotting quickly identified a tall peak on the short arm of chromosome 1, between 0 and 20 Mbp. Plotting only homozygous canonical EMS SNPs (G to A and C to T transitions) identified a peak at the same position, although this peak was shorter (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4a</a>). We also identified two much smaller peaks at the telomeres of chromosome 1 and 5 that contained regions with protein coding genes. These telomeric peaks, at regions of the genome where recombination is high (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Gore <em>et al.</em> 2009</a>), were very narrow, and had sharp boundaries. In contrast, the peak at the top of chromosome 1 was much wider and taller than any of the telomeric peaks, and had broad shoulders. These broad shoulders are what we expected from a segregating locus mapped by BSA, where each genome in the pool will have different recombination breakpoints surrounding the mutant lesion. We hypothesized that the telomeric peaks were likely caused by differences between our B73 stocks and those used to generate the B73 reference genome. These differences may be because of the histories of our B73 stocks in our own labs, or because of the stocks’ provenance (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Liang and Schnable 2016</a>). We focused on the 0-20 Mbp peak at the top of chromosome 1 as the initial <em>ten</em> mapping interval, which included 623 genes.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/11/3583/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Both ten and vns are on chromosome 1. (A) ten likely lies between 0 and 20 Mbp on chromosome 1. Plotting the number of homozygous positions that differ from the B73 reference genome (per 1 Mbp chromosomal bin) reveals a large peak on chromosome 1 (top). This peak is still visible, but smaller, if only homozygous EMS SNPs are plotted (bottom). (B) vns likely lies between 40 and 70 Mbp on chromosome 1. Plotting the number of homozygous positions that differ from the B73 reference genome (per 1 Mbp chromosomal bin) reveals no distinct peaks (top). After filtering out parental, wild-type sibling, Hapmap 3.2.1 and background SNPs, a large peak on the short arm of chromosome 1 is revealed (middle). This peak is still visible, but smaller, if only homozygous EMS SNPs are plotted (bottom). *indicates a region where our B73 stocks likely differed from those that were used to generate the B73 reference genome." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2116734167" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Both &lt;em&gt;ten&lt;/em&gt; and &lt;em&gt;vns&lt;/em&gt; are on chromosome 1. (A) &lt;em&gt;ten&lt;/em&gt; likely lies between 0 and 20 Mbp on chromosome 1. Plotting the number of homozygous positions that differ from the B73 reference genome (per 1 Mbp chromosomal bin) reveals a large peak on chromosome 1 (top). This peak is still visible, but smaller, if only homozygous EMS SNPs are plotted (bottom). (B) &lt;em&gt;vns&lt;/em&gt; likely lies between 40 and 70 Mbp on chromosome 1. Plotting the number of homozygous positions that differ from the B73 reference genome (per 1 Mbp chromosomal bin) reveals no distinct peaks (top). After filtering out parental, wild-type sibling, Hapmap 3.2.1 and background SNPs, a large peak on the short arm of chromosome 1 is revealed (middle). This peak is still visible, but smaller, if only homozygous EMS SNPs are plotted (bottom). *indicates a region where our B73 stocks likely differed from those that were used to generate the B73 reference genome.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/11/3583/F4.medium.gif" width="440" height="370"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/8/11/3583/F4.medium.gif" width="440" height="370"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/11/3583/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/11/3583/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491392" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-28" class="first-child">Both <em>ten</em> and <em>vns</em> are on chromosome 1. (A) <em>ten</em> likely lies between 0 and 20 Mbp on chromosome 1. Plotting the number of homozygous positions that differ from the B73 reference genome (per 1 Mbp chromosomal bin) reveals a large peak on chromosome 1 (top). This peak is still visible, but smaller, if only homozygous EMS SNPs are plotted (bottom). (B) <em>vns</em> likely lies between 40 and 70 Mbp on chromosome 1. Plotting the number of homozygous positions that differ from the B73 reference genome (per 1 Mbp chromosomal bin) reveals no distinct peaks (top). After filtering out parental, wild-type sibling, Hapmap 3.2.1 and background SNPs, a large peak on the short arm of chromosome 1 is revealed (middle). This peak is still visible, but smaller, if only homozygous EMS SNPs are plotted (bottom). *indicates a region where our B73 stocks likely differed from those that were used to generate the B73 reference genome.</p><div class="sb-div caption-clear"></div></div></div><p id="p-29">For <em>vns</em>, identifying a chromosomal location was not quite as simple because <em>vns</em> arose in an unknown genetic background, and a <em>vns</em> heterozygote was crossed to an unmutagenized parent, and not outcrossed to any reference genotype (<em>e.g.</em>, B73) (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1b</a>). Thus, plotting only homozygous <em>vns</em> variants revealed, as expected, extensive homozygosity distinct from the B73 reference. There was still no clear peak when we plotted only canonical EMS SNPs. However, after removing all unmutagenized parental and wild-type sibling SNPs, as well as all the HapMap 3.2.1 and A619 SNPs, we found a region of high homozygosity on the short arm of chromosome 1 between 40 and 70 Mbp (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4b</a>). This <em>vns</em> interval was larger compared to the <em>ten</em> region, perhaps due to the small number of individuals (9) included in each <em>vns</em> pool, or the increased recombination at the end of chr.1 where <em>ten</em> is located. As with <em>ten</em>, plotting only EMS SNPs still revealed a (shorter) peak. The presence of many non-EMS SNPs in the <em>vns</em> mapping interval is likely a result of heterozygosity in the parental <em>ns1 ns2</em> stock. The <em>vns</em> mapping interval included 651 genes.</p></div><div id="sec-14" class="subsection"><h3>Fine mapping of ten using custom indel markers quickly reduced the mapping interval</h3><p id="p-30">Both the <em>ten</em> and <em>vns</em> pools were sequenced fairly deeply (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>), and our sequencing data were thus likely to contain both causative lesions. However, in cases where mutant pools are sequenced to a shallower depth, and the causative lesions might be missed, we reasoned that the sequencing data could be used to design markers for fine mapping to reduce the mapping interval. This fine mapping will also be critical for reducing the mapping interval if there are multiple gene candidates in the mapping interval. Therefore, as a proof of concept, we used our NGS data to design five insertion/deletion (indel) markers to refine the <em>ten</em> interval (<a id="xref-ref-15-5" class="xref-bibr" href="#ref-15">Gallavotti and Whipple 2015</a>) (Table S1). We used these markers to genotype 101 <em>ten</em> individuals from the F2 mapping population and narrowed the <em>ten</em> interval to a region that includes 74 genes between 11.1 and 13.1 Mbp (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4a</a>). Thus, we were able to quickly reduce the <em>ten</em> mapping interval more than eightfold, and simplify the evaluation of candidate SNPs.</p></div><div id="sec-15" class="subsection"><h3>SNP filtering and candidate SNP identification</h3><p id="p-31">Next, we wanted to identify candidate SNPs in our BSA-seq mapping regions. We used a variant calling program, Varscan, to call genotypes at variant positions based on allele frequency (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Koboldt <em>et al.</em> 2012</a>). Varscan identified 36,186,873 and 36,790,695 SNPs genome-wide in the <em>ten</em> and <em>vns</em> datasets, respectively (minimum read depth of 8) (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). We removed putative background and non-causative SNPs from these datasets, as described in the Methods. After SNP filtering, we were left with 50 homozygous EMS SNPs in the <em>ten</em> mapping interval, and 427 homozygous EMS SNPs in the <em>vns</em> interval (allele frequency &gt; 0.99).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/491350/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/491350/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Varscan SNP data summary for <em>ten</em> and <em>vns</em> (coverage &gt; 8)</span><div class="sb-div caption-clear"></div></div></div><p id="p-32">To identify which SNPs in these final sets might have deleterious effects on gene function, we ran these SNPs through SnpEff (<a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Cingolani <em>et al.</em> 2012</a>). SnpEff identified 4 missense mutations in our <em>ten</em> mapping interval. The <em>vns</em> mapping interval included 1 splice site mutation, and 2 missense mutations (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>).</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/491360/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/491360/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Candidate SNPs for <em>ten</em> and <em>vns</em></span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-16" class="subsection"><h3>Candidate SNP validation</h3><p id="p-33">In our <em>ten</em> dataset, the four missense SNPs were in genes encoding a 3-deoxy-D-arabino-heptulosonate 7-phosphate synthase 1 (<em>DHS1</em>) (Zm00001d027725); an SH2 domain protein (Zm00001d027752); Histone H2A 8 (Zm00001d027775); and a known maize gene - <em>narrow odd dwarf</em> (<em>nod</em>) (Zm00001d027722). <em>nod</em> encodes a maize MID-COMPLEMENTING ACTIVITY protein (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Rosa <em>et al.</em> 2017</a>). <em>nod-1</em> is a loss-of-function allele of <em>nod</em> in the B73 background (<a id="xref-ref-40-3" class="xref-bibr" href="#ref-40">Rosa <em>et al.</em> 2017</a>). Compared to the B73 inbred, <em>nod-1</em> exhibits several pleiotropic developmental defects, such as short stature, small organs, a lack of ligules, compromised apical dominance, and increased tiller growth (<a id="xref-ref-40-4" class="xref-bibr" href="#ref-40">Rosa <em>et al.</em> 2017</a>). This pleiotropic phenotype strongly resembled <em>ten</em> single mutants. Combined with our NGS data, this led us to speculate that <em>ten</em> could be an allele of <em>nod</em>.</p><p id="p-34">To test whether <em>ten</em> was an allele of <em>nod</em>, we Sanger sequenced <em>nod</em> in one of our <em>ten</em> mutants, and performed complementation crosses. Sanger sequencing confirmed that <em>ten</em> harbored the same missense mutation in <em>nod</em> that was identified through NGS (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5b</a>). In our field, <em>nod-1</em> homozygous individuals failed to produce mature tassels or ears by the time that <em>ten</em> plants were ready for crossing. Therefore, we crossed <em>ten</em> to heterozygous <em>nod</em> plants and examined the F1 progeny. Both <em>ten</em> and <em>nod-1</em> are recessive mutations. Therefore, if <em>ten</em> were an allele of <em>nod</em>, we expected about half of the F1 progeny to show the <em>nod</em> phenotype, and half of the F1 progeny to show a wild-type phenotype. Otherwise, if <em>ten</em> were not an allele of <em>nod</em>, we expected all F1 progeny to show a wild-type phenotype. Indeed, we found that 12 of 26 F1 plants were short, and lacked ligules (<em>P</em> = 0.69, chi-squared test, one degree of freedom, <a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5d</a>), as is the case with <em>nod</em> mutants (<a id="xref-ref-40-5" class="xref-bibr" href="#ref-40">Rosa <em>et al.</em> 2017</a>). Therefore, we concluded that <em>ten</em> is an allele of <em>nod</em>, and henceforth will refer to it as <em>nod-ten</em>.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/11/3583/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="ten and vns encode alleles of nod and dek1, respectively. (A) Fine mapping of ten using custom indel markers reduced the mapping interval to a region containing 74 genes. The approximate position of nod-ten is marked in red. (B) The ten lesion in nod is a missense mutation. (C) The vns lesion in dek1 is a splice site mutation. (D) ten fails to complement nod. As in nod-1 mutants (Rosa et al. 2017), the ligule and auricle are both absent in the leaves of ten/nod-1 F1 plants (top). The black arrow indicates the ligule in a wild-type plant (bottom), and the lack of a ligule in a ten/nod-1 F1 plant. (E) vns fails to complement dek1." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2116734167" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;em&gt;ten&lt;/em&gt; and &lt;em&gt;vns&lt;/em&gt; encode alleles of &lt;em&gt;nod&lt;/em&gt; and &lt;em&gt;dek1&lt;/em&gt;, respectively. (A) Fine mapping of &lt;em&gt;ten&lt;/em&gt; using custom indel markers reduced the mapping interval to a region containing 74 genes. The approximate position of &lt;em&gt;nod-ten&lt;/em&gt; is marked in red. (B) The &lt;em&gt;ten&lt;/em&gt; lesion in &lt;em&gt;nod&lt;/em&gt; is a missense mutation. (C) The &lt;em&gt;vns&lt;/em&gt; lesion in &lt;em&gt;dek1&lt;/em&gt; is a splice site mutation. (D) &lt;em&gt;ten&lt;/em&gt; fails to complement &lt;em&gt;nod&lt;/em&gt;. As in &lt;em&gt;nod-1&lt;/em&gt; mutants (Rosa &lt;em&gt;et al.&lt;/em&gt; 2017), the ligule and auricle are both absent in the leaves of &lt;em&gt;ten/nod-1&lt;/em&gt; F1 plants (top). The black arrow indicates the ligule in a wild-type plant (bottom), and the lack of a ligule in a &lt;em&gt;ten/nod-1&lt;/em&gt; F1 plant. (E) &lt;em&gt;vns&lt;/em&gt; fails to complement &lt;em&gt;dek1&lt;/em&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/11/3583/F5.medium.gif" width="440" height="384"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/8/11/3583/F5.medium.gif" width="440" height="384"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/11/3583/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/11/3583/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491253" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-35" class="first-child"><em>ten</em> and <em>vns</em> encode alleles of <em>nod</em> and <em>dek1</em>, respectively. (A) Fine mapping of <em>ten</em> using custom indel markers reduced the mapping interval to a region containing 74 genes. The approximate position of <em>nod-ten</em> is marked in red. (B) The <em>ten</em> lesion in <em>nod</em> is a missense mutation. (C) The <em>vns</em> lesion in <em>dek1</em> is a splice site mutation. (D) <em>ten</em> fails to complement <em>nod</em>. As in <em>nod-1</em> mutants (<a id="xref-ref-40-6" class="xref-bibr" href="#ref-40">Rosa <em>et al.</em> 2017</a>), the ligule and auricle are both absent in the leaves of <em>ten/nod-1</em> F1 plants (top). The black arrow indicates the ligule in a wild-type plant (bottom), and the lack of a ligule in a <em>ten/nod-1</em> F1 plant. (E) <em>vns</em> fails to complement <em>dek1</em>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-36">The single likely high effect SNP in the <em>vns</em> mapping interval was in a donor splice site of a known maize gene, <em>defective kernel1</em> (<em>dek1</em>) (Zm00001d028818) (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5c</a>). <em>Dek1</em> encodes a membrane-spanning protein with a calpain protease domain (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Lid <em>et al.</em> 2002</a>). <em>dek1</em> mutants contain a root primordium but lack a shoot structure (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Neuffer <em>et al.</em> 1997</a>), similar to what was observed in <em>vns</em> mutants. This led us to suspect that <em>vns</em> might encode an allele of <em>dek1</em>.</p><p id="p-37">To determine if the SNP in <em>dek1</em> was responsible for the <em>vns</em> phenotype, we crossed heterozygous <em>vns</em> plants with heterozygous <em>dek1</em> plants. If <em>vns</em> were an allele of <em>dek1</em>, we expected about one quarter of the F1 progeny to show the <em>dek1</em> phenotype, and three quarters of the F1 progeny to show a wild-type phenotype. Otherwise, if <em>vns</em> were not an allele of <em>dek1</em>, we expected all F1 progeny to show a wild-type phenotype. In the F1 progeny, 9 of 36 plants were very small, and died shortly after producing only a small number of leaves (<em>P</em> = 1, chi-squared test, one degree of freedom, <a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5e</a>). Similarly, 7 of 36 F1 progeny of the heterozygous <em>dek1</em> self exhibited the same phenotype (<em>P</em> = 0.44, chi-squared test, one degree of freedom). Thus, <em>vns</em> failed to complement the <em>dek1</em> mutant phenotype, indicating that <em>vns</em> is an allele of <em>dek1</em>. Henceforth we will refer to <em>vns</em> as <em>dek1-vns</em>.</p></div></div><div class="section discussion" id="sec-17"><h2 class="">Discussion</h2><p id="p-38">Here, we report a BSA-Seq method for cloning EMS mutants in maize. Using this method, we identified and validated causative SNPs for two separate EMS-induced mutants. We have provided a detailed protocol for conducting these analyses using Galaxy, as well as a SNP variant file for the A619 genetic background (File S1). This has the potential to be useful for anyone working in A619 - including for making a synthetic A619 reference genome (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">McKenna <em>et al.</em> 2010</a>). In addition, we showed an interaction between the known maize genes <em>nod</em> and <em>tb1</em> (<a id="xref-ref-12-4" class="xref-bibr" href="#ref-12">Doebley <em>et al.</em> 1997</a>; <a id="xref-ref-40-7" class="xref-bibr" href="#ref-40">Rosa <em>et al.</em> 2017</a>).</p><p id="p-39">We discovered a synergistic interaction between <em>tb1</em> and <em>nod</em> in regulating tiller production (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3</a>). Although <em>nod</em> is predicted to encode a membrane-localized maize MID-COMPLEMENTING ACTIVITY homolog, and may function to coordinate plant development in response to intrinsic and extrinsic cues (<a id="xref-ref-40-8" class="xref-bibr" href="#ref-40">Rosa <em>et al.</em> 2017</a>), how exactly <em>nod</em> regulates plant development remains largely unknown. The homolog of <em>NOD</em> in <em>Arabidopsis thaliana</em>, <em>MCA1</em>, may be involved in Ca<sup>2+</sup> uptake and the cell wall stress response pathway (Yamanaka <em>et al.</em> 2010). In maize, hormone metabolism and cell division are overrepresented GO categories in <em>nod-1</em> mutant transcriptomes, relative to wild-type (<a id="xref-ref-40-9" class="xref-bibr" href="#ref-40">Rosa <em>et al.</em> 2017</a>). Interestingly, <em>tb1</em> is downregulated (2.4-fold) in <em>nod-1</em> shoot apices, relative to wild-type (<a id="xref-ref-40-10" class="xref-bibr" href="#ref-40">Rosa <em>et al.</em> 2017</a>), which suggests that <em>nod</em> might act upstream of <em>tb1</em> to regulate its expression. <em>nod</em> could represent a sensor of positional and environmental cues that regulate <em>tb1</em>-mediated tiller outgrowth. The exact mechanism through which <em>nod</em> and <em>tb1</em> synergistically regulate tiller outgrowth requires further investigation.</p><p id="p-40">In any BSA-Seq experiment, the key goals are to define the smallest mapping interval possible, and to identify a small number of potentially causative lesions. To achieve these goals, both the size of the mutant pool and sequencing depth must be considered. In our experiments, the <em>nod-ten</em> BSA-Seq mapping interval encompassed ∼20Mbp on the short arm of chromosome 1, and included 623 genes. The <em>dek1-vns</em> mapping interval encompassed ∼30Mbp on the short arm of chromosome 1, and included 651 genes (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4</a>). Thus, despite a much smaller mutant pool (9 <em>dek1-vns</em> individuals <em>vs.</em> 101 <em>nod-ten</em> individuals), and a lower recombination rate where <em>dek1-vns</em> is located (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Gore <em>et al.</em> 2009</a>), the <em>dek1-vns</em> mapping interval included only 28 (4%) more genes than the <em>nod-ten</em> mapping interval. Mean coverage in both datasets was fairly high (26-fold for <em>ten</em> and 17-fold for <em>vns</em>), allowing for both causative lesions to be captured. To identify just the genomic interval that contains a particular gene, sequencing depth could be reduced, but the chances of catching the causative lesion will be similarly reduced. In a case where no clear candidate lesion is recovered, and/or there are many genes in a BSA-Seq mapping interval, fine mapping would become essential. Thus, our results indicate that sequencing to the deepest level affordable is more important than a big mutant pool for a successful BSA-Seq experiment.</p><p id="p-41">The most important advantage of BSA-Seq over other methods is its simplicity, both in terms of sample collection and data analysis. While BSA-Seq is used extensively in other taxa (<a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Schneeberger and Weigel 2011</a>; <a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">Abe <em>et al.</em> 2012</a>; <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Mascher <em>et al.</em> 2014</a>; <a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">Woods <em>et al.</em> 2014</a>; <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Ding <em>et al.</em> 2017</a>; <a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Song <em>et al.</em> 2017</a>; <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Jiao <em>et al.</em> 2018</a>), gene mapping via Bulked-Segregant RNA-Seq (BSR-Seq) is more often used for cloning maize genes (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Liu <em>et al.</em> 2012</a>; <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Li <em>et al.</em> 2013</a>; <a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Nestler <em>et al.</em> 2014</a>; <a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Tang <em>et al.</em> 2014</a>). In BSR-Seq, RNA from a pool of mutants and RNA from a pool of non-mutants is used to make RNA-Seq libraries (<a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">Liu <em>et al.</em> 2012</a>). These libraries are sequenced, and the resulting reads mapped to the maize reference genome. BSR-Seq offers a very good method for genome reduction, thus increasing sequencing depth without increasing cost. If the RNA-Seq is performed using the right tissue at the right developmental stage, BSR-Seq offers the advantage of differential gene expression data as well as mapping information. In contrast, since BSA-seq relies on DNA extraction, sample collection can be done at any developmental stage, and from any tissue. Although bulk RNA extractions are not extraordinarily challenging, DNA extractions are far simpler, and can be performed by relatively inexperienced trainees in the lab. This technical simplicity offers the advantage of being able to involve high school students and junior undergraduates in authentic research experiences (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Lopatto <em>et al.</em> 2014</a>). In addition, the data analysis for BSR-Seq is not as straightforward as it is for BSA-Seq. In BSR-Seq, allele-specific expression must be accounted for, as well as differential expression of genes not linked to the mutant gene in mutant <em>vs.</em> wild-type pools (<a id="xref-ref-29-4" class="xref-bibr" href="#ref-29">Liu <em>et al.</em> 2012</a>). Here, too, simplicity offers speed for the experienced researcher, and excellent training opportunities for beginning scientists.</p><p id="p-42">For <em>dek1-vns</em>, we used a modification of BSA-Seq that has been called MutMap (<a id="xref-ref-1-4" class="xref-bibr" href="#ref-1">Abe <em>et al.</em> 2012</a>). In MutMap, instead of generating an F1 between contrasting genotypes, the F1 comes from a mutant backcrossed to a wild-type, unmutagenized individual in the same genetic background as the mutant. Thus, MutMap introduces no additional genetic variation in the initial F1 cross. Instead, MutMap relies on co-segregation of induced SNPs with the mutant phenotype, allowing for the identification of a region of increased homozygosity in the mutant pool. MutMap would likely also have been successful in identifying <em>nod-ten</em>. In both of our datasets, just mapping the EMS mutations reveals clear peaks corresponding to the mutant genes (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4</a>). One advantage that MutMap offers is that no additional genetic variation is introduced in the F1 cross. This genetic variation can suppress or enhance mutant phenotypes, which can make scoring F2 populations challenging. However, in these cases, conventional BSA-Seq could be used to identify both causative lesions and modifiers in one step (<a id="xref-ref-43-3" class="xref-bibr" href="#ref-43">Song <em>et al.</em> 2017</a>).</p><p id="p-43">Another advantage of BSA-Seq over MutMap is that even if the lesion is not captured, or if there are many candidate lesions in a mapping interval, a researcher is immediately poised to design indel markers and commence fine-mapping in an identified genomic interval. Numerous candidate lesions are more likely when the mutant under study is not from an EMS mutagenesis experiment, which is fairly common in maize (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Vollbrecht <em>et al.</em> 1991</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Thompson <em>et al.</em> 2009</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Whipple <em>et al.</em> 2011</a>). In these cases, lesions are not necessarily of a defined type (<em>e.g.</em>, G to A or C to T transitions), which makes SNP filtering challenging. However, high homozygosity that is polymorphic with a reference genome will still reveal the chromosomal location of the causative lesion, and streamline subsequent fine mapping.</p><p id="p-44">Technical and conceptual advances will eliminate some, but not all, sources of uncertainty when it comes to capturing the candidate lesions underlying mutant phenotypes. With shallow read depths, causative lesions may not be captured; but deep coverage is likely to become ever more attainable as NGS costs drop, thus eliminating this source of uncertainty. There may be a number of potentially causative lesions in a candidate region; but as the maize pan-genome is resolved (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Hirsch <em>et al.</em> 2014</a>; <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Bukowski <em>et al.</em> 2018</a>), the pool of background SNPs that can be eliminated gets ever-deeper when cloning clear null mutations unlikely to be present in natural variation. Other potential sources of uncertainty will likely never go away. For example, filtering out all of the HapMap and pan-genome SNPs will be less useful when trying to identify natural modifiers. BSA-Seq and fine mapping will be particularly helpful in narrowing down a candidate region for a natural modifier; when a causative lesion is not in the coding sequence of an annotated gene; or when mutant pools are contaminated with wild-type or heterozygous samples. Thus, BSA-Seq offers many advantages that are likely to prove useful for mapping and cloning many maize mutants.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-45">The authors thank Courtney Babbitt and Beth Thompson for helpful discussion; Sarah Hake for the kind gift of the <em>tb1-sh</em> allele; and Ed Wilcox at the BYU DNA sequencing facility for technical assistance. This work was supported by a USDA-Hatch grant to M.E.B. (MAS00501); the Biology Department at the University of Massachusetts Amherst; an NSF PGRP grant to C.J.W. (IOS-1339332); and an NSF Postdoctoral Research Fellowship in Biology to P.A.C. (IOS-1612235).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1"><a class="rev-xref" href="#xref-fn-1-1">↵</a><span class="fn-label">1</span> Contributed equally to the manuscript</p></li><li class="fn" id="fn-2"><p id="p-46">Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.7014851">https://doi.org/10.25387/g3.7014851</a>.</p></li><li class="fn" id="fn-3"><p id="p-47"><em>Communicating editor: J. Yang</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2018-06-13"><span class="received-label">Received </span>June 13, 2018.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2018-09-05"><span class="accepted-label">Accepted </span>September 5, 2018.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2018 Klein <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3583.1" data-doi="10.1038/nbt.2095"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Abe</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kosugi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yoshida</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Natsume</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Takagi</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genome sequencing reveals agronomically important loci in rice using MutMap.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">174</span>–<span class="cit-lpage">178</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nbt.2095</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DAbe%26rft.auinit1%253DA.%26rft.volume%253D30%26rft.issue%253D2%26rft.spage%253D174%26rft.epage%253D178%26rft.atitle%253DGenome%2Bsequencing%2Breveals%2Bagronomically%2Bimportant%2Bloci%2Bin%2Brice%2Busing%2BMutMap.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.2095%26rft_id%253Dinfo%253Apmid%252F22267009%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.2095&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22267009&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3583.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3583.2" data-doi="10.1534/g3.116.029660"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Addo-Quaye</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buescher</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Best</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chaikam</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baxter</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Forward Genetics by Sequencing EMS Variation-Induced Inbred Lines. G3: Genes, Genomes</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">413</span>–<span class="cit-lpage">425</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.116.029660</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D7%26rft.spage%253D413%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.116.029660%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/g3.116.029660&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3583.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Afgan</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baker</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Batut</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Den Beek</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bouvier</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Čech</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">The Galaxy platform for accessible, reproducible and collaborative biomedical analyses: 2018 update.</span> <abbr class="cit-jnl-abbrev">Nuc. acids research</abbr>, <span class="cit-vol">46</span>: <span class="cit-fpage">W537</span>–<span class="cit-lpage">W544</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNuc.%2Bacids%2Bresearch%26rft.volume%253D46%26rft.spage%253DW537%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3583.4" data-doi="10.1093/nar/gkw343"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Afgan</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baker</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van den Beek</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blankenberg</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bouvier</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The Galaxy platform for accessible, reproducible and collaborative biomedical analyses: 2016 update.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">W3</span>–<span class="cit-lpage">W10</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkw343</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkw343%26rft_id%253Dinfo%253Apmid%252F27137889%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkw343&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27137889&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3583.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-web" id="cit-8.11.3583.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Andrews</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> FastQC: a quality control tool for high throughput sequence data. Version 0.11. 2. Babraham Institute, Cambridge, UK <a href="http://www.bioinformatics.babraham.ac.uk/projects/fastqc">http://www. bioinformatics. babraham. ac.uk/projects/fastqc</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3583.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Becraft</span>  <span class="cit-name-given-names">P. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dey</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Asuncion-Crabb</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">The maize dek1 gene functions in embryonic pattern formation and cell fate specification.</span> <abbr class="cit-jnl-abbrev">Development</abbr> <span class="cit-vol">129</span>: <span class="cit-fpage">5217</span>–<span class="cit-lpage">5225</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopment%26rft.volume%253D129%26rft.spage%253D5217%26rft_id%253Dinfo%253Apmid%252F12399313%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=12399313&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3583.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000179693200011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3583.7" data-doi="10.1093/bioinformatics/btq281"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Blankenberg</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gordon</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Von Kuster</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coraor</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Manipulation of FASTQ data with Galaxy.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">1783</span>–<span class="cit-lpage">1785</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btq281</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq281%26rft_id%253Dinfo%253Apmid%252F20562416%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btq281&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20562416&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3583.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000279474400018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3583.8" data-doi="10.1093/bioinformatics/btu170"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bolger</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lohse</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Usadel</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Trimmomatic: a flexible trimmer for Illumina sequence data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">2114</span>–<span class="cit-lpage">2120</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btu170</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu170%26rft_id%253Dinfo%253Apmid%252F24695404%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu170&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24695404&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3583.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000340049100004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3583.9" data-doi="10.1093/gigascience/gix120"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bukowski</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zou</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Construction of the third-generation Zea mays haplotype map.</span> <abbr class="cit-jnl-abbrev">Gigascience</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">12</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/gigascience/gix134</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGigascience%26rft.volume%253D7%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgigascience%252Fgix120%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/gigascience/gix120&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3583.10" data-doi="10.4161/fly.19695"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cingolani</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Platts</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">L. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coon</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nguyen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A program for annotating and predicting the effects of single nucleotide polymorphisms, SnpEff: SNPs in the genome of Drosophila melanogaster strain w1118; iso-2; iso-3.</span> <abbr class="cit-jnl-abbrev">Fly (Austin)</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">80</span>–<span class="cit-lpage">92</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.4161/fly.19695</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DFly%26rft.aulast%253DCingolani%26rft.auinit1%253DP.%26rft.volume%253D6%26rft.issue%253D2%26rft.spage%253D80%26rft.epage%253D92%26rft.atitle%253DA%2Bprogram%2Bfor%2Bannotating%2Band%2Bpredicting%2Bthe%2Beffects%2Bof%2Bsingle%2Bnucleotide%2Bpolymorphisms%252C%2BSnpEff%253A%2BSNPs%2Bin%2Bthe%2Bgenome%2Bof%2BDrosophila%2Bmelanogaster%2Bstrain%2Bw1118%253B%2Biso-2%253B%2Biso-3.%26rft_id%253Dinfo%253Adoi%252F10.4161%252Ffly.19695%26rft_id%253Dinfo%253Apmid%252F22728672%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4161/fly.19695&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22728672&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3583.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000305965500003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3583.11" data-doi="10.1111/nph.14281"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ding</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mou</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peng</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">A dominant-negative actin mutation alters corolla tube width and pollinator visitation in Mimulus lewisii.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">213</span>: <span class="cit-fpage">1936</span>–<span class="cit-lpage">1944</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/nph.14281</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytol.%26rft.volume%253D213%26rft.spage%253D1936%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fnph.14281%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/nph.14281&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3583.12" data-doi="10.1038/386485a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Doebley</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stec</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hubbard</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">The evolution of apical dominance in maize.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">386</span>: <span class="cit-fpage">485</span>–<span class="cit-lpage">488</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/386485a0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DDoebley%26rft.auinit1%253DJ.%26rft.volume%253D386%26rft.issue%253D6624%26rft.spage%253D485%26rft.epage%253D488%26rft.atitle%253DThe%2Bevolution%2Bof%2Bapical%2Bdominance%2Bin%2Bmaize.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F386485a0%26rft_id%253Dinfo%253Apmid%252F9087405%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/386485a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9087405&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3583.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997WR25600049&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3583.13" data-doi="10.1101/gr.8.3.175"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ewing</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hillier</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wendl</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Green</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Base-Calling of Automated Sequencer Traces UsingPhred. I. Accuracy Assessment.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">175</span>–<span class="cit-lpage">185</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.8.3.175</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DEwing%26rft.auinit1%253DB.%26rft.volume%253D8%26rft.issue%253D3%26rft.spage%253D175%26rft.epage%253D185%26rft.atitle%253DBase-Calling%2Bof%2BAutomated%2BSequencer%2BTraces%2BUsing%2BPhred.%2B%2BI.%2BAccuracy%2BAssessment%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.8.3.175%26rft_id%253Dinfo%253Apmid%252F9521921%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjc6IjgvMy8xNzUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzExLzM1ODMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3583.14" data-doi="10.1242/dev.051748"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gallavotti</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Long</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stanfield</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jackson</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The control of axillary meristem fate in the maize ramosa pathway.</span> <abbr class="cit-jnl-abbrev">Development</abbr> <span class="cit-vol">137</span>: <span class="cit-fpage">2849</span>–<span class="cit-lpage">2856</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1242/dev.051748</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopment%26rft.stitle%253DDevelopment%26rft.aulast%253DGallavotti%26rft.auinit1%253DA.%26rft.volume%253D137%26rft.issue%253D17%26rft.spage%253D2849%26rft.epage%253D2856%26rft.atitle%253DThe%2Bcontrol%2Bof%2Baxillary%2Bmeristem%2Bfate%2Bin%2Bthe%2Bmaize%2Bramosa%2Bpathway%26rft_id%253Dinfo%253Adoi%252F10.1242%252Fdev.051748%26rft_id%253Dinfo%253Apmid%252F20699296%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZGV2ZWxvcCI7czo1OiJyZXNpZCI7czoxMToiMTM3LzE3LzI4NDkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzExLzM1ODMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3583.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gallavotti</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whipple</span>  <span class="cit-name-given-names">C. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-comment">Positional cloning in maize (Zea mays subsp. mays, Poaceae). Appl. Plant Sci. 3: apps.1400092</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3732/apps.1400092</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3583.16" data-doi="10.1371/journal.pone.0146492"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gillmor</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roeder</span>  <span class="cit-name-given-names">A. H. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sieber</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Somerville</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lukowitz</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">A Genetic Screen for Mutations Affecting Cell Division in the Arabidopsis thaliana Embryo Identifies Seven Loci Required for Cytokinesis.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">e0146492</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0146492</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D11%26rft.spage%253De0146492%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0146492%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0146492&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3583.17" data-doi="10.1126/science.1177837"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gore</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chia</span>  <span class="cit-name-given-names">J.-M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ersoz</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A first-generation haplotype map of maize.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">326</span>: <span class="cit-fpage">1115</span>–<span class="cit-lpage">1117</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1177837</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DGore%26rft.auinit1%253DM.%2BA.%26rft.volume%253D326%26rft.issue%253D5956%26rft.spage%253D1115%26rft.epage%253D1117%26rft.atitle%253DA%2BFirst-Generation%2BHaplotype%2BMap%2Bof%2BMaize%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1177837%26rft_id%253Dinfo%253Apmid%252F19965431%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMjYvNTk1Ni8xMTE1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMS8zNTgzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3583.18" data-doi="10.1534/g3.115.017665"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haase</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beissinger</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hirsch</span>  <span class="cit-name-given-names">C. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vaillancourt</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deshpande</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Shared Genomic Regions Between Derivatives of a Large Segregating Population of Maize Identified Using Bulked Segregant Analysis Sequencing and Traditional Linkage Analysis.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">1593</span>–<span class="cit-lpage">1602</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.115.017665</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.115.017665%26rft_id%253Dinfo%253Apmid%252F26038364%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjUvOC8xNTkzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMS8zNTgzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3583.19" data-doi="10.1105/tpc.113.119982"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hirsch</span>  <span class="cit-name-given-names">C. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foerster</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sekhon</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muttoni</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Insights into the maize pan-genome and pan-transcriptome.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">121</span>–<span class="cit-lpage">135</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1105/tpc.113.119982</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.113.119982%26rft_id%253Dinfo%253Apmid%252F24488960%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjg6IjI2LzEvMTIxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMS8zNTgzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3583.20" data-doi="10.3389/fpls.2017.02267"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jiao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burow</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gladman</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Acosta-Martinez</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Efficient Identification of Causal Mutations through Sequencing of Bulked F2 from Two Allelic Bloomless Mutants of Sorghum bicolor.</span> <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">2267</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fpls.2017.02267</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BPlant%2BSci.%26rft.volume%253D8%26rft.spage%253D2267%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffpls.2017.02267%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fpls.2017.02267&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3583.21" data-doi="10.1038/nature22971"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jiao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peluso</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liang</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stitzer</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Improved maize reference genome with single-molecule technologies.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">546</span>: <span class="cit-fpage">524</span>–<span class="cit-lpage">527</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature22971</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D546%26rft.spage%253D524%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature22971%26rft_id%253Dinfo%253Apmid%252F28605751%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature22971&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28605751&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3583.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3583.22" data-doi="10.1101/gr.129684.111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koboldt</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Larson</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McLellan</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">VarScan 2: somatic mutation and copy number alteration discovery in cancer by exome sequencing.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">568</span>–<span class="cit-lpage">576</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.129684.111</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.129684.111%26rft_id%253Dinfo%253Apmid%252F22300766%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIyLzMvNTY4IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMS8zNTgzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3583.23" data-doi="10.1038/nmeth.1923"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langmead</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Fast gapped-read alignment with Bowtie 2.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">357</span>–<span class="cit-lpage">359</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.1923</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DLangmead%26rft.auinit1%253DB.%26rft.volume%253D9%26rft.issue%253D4%26rft.spage%253D357%26rft.epage%253D359%26rft.atitle%253DFast%2Bgapped-read%2Balignment%2Bwith%2BBowtie%2B2.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1923%26rft_id%253Dinfo%253Apmid%252F22388286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1923&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22388286&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3583.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000302218500017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3583.24" data-doi="10.1371/journal.pone.0157942"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schnable</span>  <span class="cit-name-given-names">J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">RNA-Seq Based Analysis of Population Structure within the Maize Inbred B73.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">e0157942</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0157942</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D11%26rft.spage%253De0157942%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0157942%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0157942&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3583.25" data-doi="10.1073/pnas.042098799"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lid</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gruis</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jung</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lorentzen</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ananiev</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">The defective kernel 1 (dek1) gene required for aleurone cell development in the endosperm of maize grains encodes a membrane protein of the calpain gene superfamily.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">99</span>: <span class="cit-fpage">5460</span>–<span class="cit-lpage">5465</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.042098799</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.042098799%26rft_id%253Dinfo%253Apmid%252F11929961%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI5OS84LzU0NjAiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzExLzM1ODMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3583.26" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Handsaker</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wysoker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fennell</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The Sequence Alignment/Map format and SAMtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2078</span>–<span class="cit-lpage">2079</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btp352</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp352&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19505943&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3583.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268808600014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3583.27" data-doi="10.1371/journal.pone.0082333"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dietrich</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> The maize glossy13 gene, cloned via BSR-Seq and Seq-walking encodes a putative ABC transporter required for the normal accumulation of epicuticular <em>PLoS One</em> 8: <span class="cit-fpage">e82333</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0082333</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.volume%253D8%26rft.spage%253De82333%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0082333%26rft_id%253Dinfo%253Apmid%252F24324772%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0082333&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24324772&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3583.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-8.11.3583.28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smyth</span>  <span class="cit-name-given-names">G. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The Subread aligner: fast, accurate and scalable read mapping by seed-and-vote. Nuc. Acids Res.,</span> <span class="cit-vol">41</span>: <span class="cit-fpage">e108</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3583.29" data-doi="10.1371/journal.pone.0036406"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yeh</span>  <span class="cit-name-given-names">C.-T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nettleton</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schnable</span>  <span class="cit-name-given-names">P. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Gene Mapping via Bulked Segregant RNA-Seq (BSR-Seq).</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e36406</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0036406</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DLiu%26rft.auinit1%253DS.%26rft.volume%253D7%26rft.issue%253D5%26rft.spage%253De36406%26rft.epage%253De36406%26rft.atitle%253DGene%2Bmapping%2Bvia%2Bbulked%2Bsegregant%2BRNA-Seq%2B%2528BSR-Seq%2529.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0036406%26rft_id%253Dinfo%253Apmid%252F22586469%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0036406&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22586469&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3583.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3583.30" data-doi="10.1187/cbe.13-10-0200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lopatto</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hauser</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paetkau</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chandrasekaran</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A central support system can facilitate implementation and sustainability of a Classroom-based Undergraduate Research Experience (CURE) in Genomics.</span> <abbr class="cit-jnl-abbrev">CBE Life Sci. Educ.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">711</span>–<span class="cit-lpage">723</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1187/cbe.13-10-0200</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCBE-Life%2BSciences%2BEducation%26rft.stitle%253Dcellbioed%26rft.aulast%253DLopatto%26rft.auinit1%253DD.%26rft.volume%253D13%26rft.issue%253D4%26rft.spage%253D711%26rft.epage%253D723%26rft.atitle%253DA%2BCentral%2BSupport%2BSystem%2BCan%2BFacilitate%2BImplementation%2Band%2BSustainability%2Bof%2Ba%2BClassroom-Based%2BUndergraduate%2BResearch%2BExperience%2B%2528CURE%2529%2Bin%2BGenomics%26rft_id%253Dinfo%253Adoi%252F10.1187%252Fcbe.13-10-0200%26rft_id%253Dinfo%253Apmid%252F25452493%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToiY2VsbGJpb2VkIjtzOjU6InJlc2lkIjtzOjg6IjEzLzQvNzExIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMS8zNTgzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3583.31" data-doi="10.1186/gb-2014-15-6-r78"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mascher</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jost</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuon</span>  <span class="cit-name-given-names">J.-E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Himmelbach</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aßfalg</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Mapping-by-sequencing accelerates forward genetics in barley.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">R78</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/gb-2014-15-6-r78</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D15%26rft.spage%253DR78%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2014-15-6-r78%26rft_id%253Dinfo%253Apmid%252F24917130%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2014-15-6-r78&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24917130&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3583.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3583.32" data-doi="10.1101/gr.107524.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McKenna</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hanna</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sivachenko</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cibulskis</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">1297</span>–<span class="cit-lpage">1303</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.107524.110</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.107524.110%26rft_id%253Dinfo%253Apmid%252F20644199%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIwLzkvMTI5NyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzgvMTEvMzU4My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3583.33" data-doi="10.1371/journal.pbio.1002342"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Merchant</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lyons</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goff</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vaughn</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ware</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The iPlant Collaborative: Cyberinfrastructure for Enabling Data to Discovery for the Life Sciences.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">e1002342</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pbio.1002342</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BBiol.%26rft.volume%253D14%26rft.spage%253De1002342%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.1002342%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.1002342&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3583.34" data-doi="10.1073/pnas.88.21.9828"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Michelmore</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paran</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kesseli</span>  <span class="cit-name-given-names">R. V.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Identification of markers linked to disease-resistance genes by bulked segregant analysis: a rapid method to detect markers in specific genomic regions by using segregating populations.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">88</span>: <span class="cit-fpage">9828</span>–<span class="cit-lpage">9832</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.88.21.9828</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DMichelmore%26rft.auinit1%253DR.%2BW.%26rft.volume%253D88%26rft.issue%253D21%26rft.spage%253D9828%26rft.epage%253D9832%26rft.atitle%253DIdentification%2Bof%2Bmarkers%2Blinked%2Bto%2Bdisease-resistance%2Bgenes%2Bby%2Bbulked%2Bsegregant%2Banalysis%253A%2Ba%2Brapid%2Bmethod%2Bto%2Bdetect%2Bmarkers%2Bin%2Bspecific%2Bgenomic%2Bregions%2Bby%2Busing%2Bsegregating%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.88.21.9828%26rft_id%253Dinfo%253Apmid%252F1682921%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiODgvMjEvOTgyOCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzgvMTEvMzU4My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3583.35" data-doi="10.1111/j.1095-8339.2006.00489.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mueller</span>  <span class="cit-name-given-names">R. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Ask the plant: investigating and teaching plant structure.</span> <abbr class="cit-jnl-abbrev">Bot. J. Linn. Soc.</abbr> <span class="cit-vol">150</span>: <span class="cit-fpage">73</span>–<span class="cit-lpage">78</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1095-8339.2006.00489.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBot.%2BJ.%2BLinn.%2BSoc.%26rft.volume%253D150%26rft.spage%253D73%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1095-8339.2006.00489.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1095-8339.2006.00489.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3583.36" data-doi="10.1242/dev.01164"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nardmann</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ji</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Werr</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scanlon</span>  <span class="cit-name-given-names">M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">The maize duplicate genes narrow sheath1 and narrow sheath2 encode a conserved homeobox gene function in a lateral domain of shoot apical meristems.</span> <abbr class="cit-jnl-abbrev">Development</abbr> <span class="cit-vol">131</span>: <span class="cit-fpage">2827</span>–<span class="cit-lpage">2839</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1242/dev.01164</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopment%26rft.stitle%253DDevelopment%26rft.aulast%253DNardmann%26rft.auinit1%253DJ.%26rft.volume%253D131%26rft.issue%253D12%26rft.spage%253D2827%26rft.epage%253D2839%26rft.atitle%253DThe%2Bmaize%2Bduplicate%2Bgenes%2Bnarrow%2Bsheath1%2Band%2Bnarrow%2Bsheath2%2Bencode%2Ba%2Bconserved%2Bhomeobox%2Bgene%2Bfunction%2Bin%2Ba%2Blateral%2Bdomain%2Bof%2Bshoot%2Bapical%2Bmeristems%26rft_id%253Dinfo%253Adoi%252F10.1242%252Fdev.01164%26rft_id%253Dinfo%253Apmid%252F15169755%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZGV2ZWxvcCI7czo1OiJyZXNpZCI7czoxMToiMTMxLzEyLzI4MjciO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzExLzM1ODMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3583.37" data-doi="10.1016/j.devcel.2010.06.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nawy</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bayer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mravec</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Friml</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Birnbaum</span>  <span class="cit-name-given-names">K. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The GATA factor HANABA TARANU is required to position the proembryo boundary in the early Arabidopsis embryo.</span> <abbr class="cit-jnl-abbrev">Dev. Cell</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">103</span>–<span class="cit-lpage">113</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.devcel.2010.06.004</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bcell%26rft.stitle%253DDev%2BCell%26rft.aulast%253DNawy%26rft.auinit1%253DT.%26rft.volume%253D19%26rft.issue%253D1%26rft.spage%253D103%26rft.epage%253D113%26rft.atitle%253DThe%2BGATA%2Bfactor%2BHANABA%2BTARANU%2Bis%2Brequired%2Bto%2Bposition%2Bthe%2Bproembryo%2Bboundary%2Bin%2Bthe%2Bearly%2BArabidopsis%2Bembryo.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.devcel.2010.06.004%26rft_id%253Dinfo%253Apmid%252F20643354%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.devcel.2010.06.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20643354&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3583.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000280469100014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3583.38" data-doi="10.1111/tpj.12578"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nestler</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wen</span>  <span class="cit-name-given-names">T.-J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paschold</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marcon</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Roothairless5, which functions in maize (Zea mays L.) root hair initiation and elongation encodes a monocot-specific NADPH oxidase.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">79</span>: <span class="cit-fpage">729</span>–<span class="cit-lpage">740</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/tpj.12578</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BJ.%26rft.volume%253D79%26rft.spage%253D729%26rft_id%253Dinfo%253Adoi%252F10.1111%252Ftpj.12578%26rft_id%253Dinfo%253Apmid%252F24902980%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/tpj.12578&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24902980&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3583.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-book" id="cit-8.11.3583.39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Neuffer</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coe</span>  <span class="cit-name-given-names">E. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wessler</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-etal">et al</span></li></ol><cite>., <span class="cit-pub-date">1997</span> <em>Mutants of maize</em>. Cold Spring Harbor Laboratory Press, New York.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3583.40" data-doi="10.1105/tpc.16.00878"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rosa</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abraham-Juárez</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lewis</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fonseca</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tian</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">The Maize MID-COMPLEMENTING ACTIVITY Homolog CELL NUMBER REGULATOR13/NARROW ODD DWARF Coordinates Organ Growth and Tissue Patterning.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">474</span>–<span class="cit-lpage">490</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1105/tpc.16.00878</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.16.00878%26rft_id%253Dinfo%253Apmid%252F28254777%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjg6IjI5LzMvNDc0IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMS8zNTgzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3583.41"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Scanlon</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schneeberger</span>  <span class="cit-name-given-names">R. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Freeling</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">The maize mutant narrow sheath fails to establish leaf margin identity in a meristematic domain.</span> <abbr class="cit-jnl-abbrev">Development</abbr> <span class="cit-vol">122</span>: <span class="cit-fpage">1683</span>–<span class="cit-lpage">1691</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopment%26rft.stitle%253DDevelopment%26rft.aulast%253DScanlon%26rft.auinit1%253DM.%2BJ.%26rft.volume%253D122%26rft.issue%253D6%26rft.spage%253D1683%26rft.epage%253D1691%26rft.atitle%253DThe%2Bmaize%2Bmutant%2Bnarrow%2Bsheath%2Bfails%2Bto%2Bestablish%2Bleaf%2Bmargin%2Bidentity%2Bin%2Ba%2Bmeristematic%2Bdomain%26rft_id%253Dinfo%253Apmid%252F8674408%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZGV2ZWxvcCI7czo1OiJyZXNpZCI7czoxMDoiMTIyLzYvMTY4MyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzgvMTEvMzU4My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3583.42" data-doi="10.1016/j.tplants.2011.02.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schneeberger</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weigel</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Fast-forward genetics enabled by new sequencing technologies.</span> <abbr class="cit-jnl-abbrev">Trends Plant Sci.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">282</span>–<span class="cit-lpage">288</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tplants.2011.02.006</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bplant%2Bscience%26rft.stitle%253DTrends%2BPlant%2BSci%26rft.aulast%253DSchneeberger%26rft.auinit1%253DK.%26rft.volume%253D16%26rft.issue%253D5%26rft.spage%253D282%26rft.epage%253D288%26rft.atitle%253DFast-forward%2Bgenetics%2Benabled%2Bby%2Bnew%2Bsequencing%2Btechnologies.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tplants.2011.02.006%26rft_id%253Dinfo%253Apmid%252F21439889%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tplants.2011.02.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21439889&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3583.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000291191000008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3583.43" data-doi="10.3389/fpls.2017.00919"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qiu</span>  <span class="cit-name-given-names">L.-J.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Next-Generation Sequencing from Bulked-Segregant Analysis Accelerates the Simultaneous Identification of Two Qualitative Genes in Soybean.</span> <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">919</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fpls.2017.00919</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BPlant%2BSci.%26rft.volume%253D8%26rft.spage%253D919%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffpls.2017.00919%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fpls.2017.00919&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3583.44" data-doi="10.1111/tpj.12394"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hill-Skinner</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reed</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The maize brown midrib2 (bm2) gene encodes a methylenetetrahydrofolate reductase that contributes to lignin accumulation.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">77</span>: <span class="cit-fpage">380</span>–<span class="cit-lpage">392</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/tpj.12394</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BJ.%26rft.volume%253D77%26rft.spage%253D380%26rft_id%253Dinfo%253Adoi%252F10.1111%252Ftpj.12394%26rft_id%253Dinfo%253Apmid%252F24286468%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/tpj.12394&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24286468&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3583.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3583.45" data-doi="10.1093/nar/gkx1111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tello-Ruiz</span>  <span class="cit-name-given-names">M. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Naithani</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stein</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gupta</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Campbell</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Gramene 2018: unifying comparative genomics and pathway resources for plant research.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">D1181</span>–<span class="cit-lpage">D1189</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkx1111</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D46%26rft.spage%253DD1181%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkx1111%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkx1111&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3583.46" data-doi="10.1105/tpc.109.067751"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">B. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bartling</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whipple</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">D. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sakai</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">bearded-ear Encodes a MADS Box Transcription Factor Critical for Maize Floral Development.</span> <abbr class="cit-jnl-abbrev">The Plant Cell Online</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">2578</span>–<span class="cit-lpage">2590</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1105/tpc.109.067751</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2BCell%2BOnline%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.109.067751%26rft_id%253Dinfo%253Apmid%252F19749152%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjIxLzkvMjU3OCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzgvMTEvMzU4My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3583.47" data-doi="10.1186/1471-2229-4-12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Till</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reynolds</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weil</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Springer</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burtner</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Discovery of induced point mutations in maize genes by TILLING.</span> <abbr class="cit-jnl-abbrev">BMC Plant Biol.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">12</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2229-4-12</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bplant%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BPlant%2BBiol%26rft.aulast%253DTill%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D4%26rft.spage%253D12%26rft.epage%253D12%26rft.atitle%253DDiscovery%2Bof%2Binduced%2Bpoint%2Bmutations%2Bin%2Bmaize%2Bgenes%2Bby%2BTILLING.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2229-4-12%26rft_id%253Dinfo%253Apmid%252F15282033%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2229-4-12&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15282033&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3583.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3583.48" data-doi="10.1126/science.1248384"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vlad</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kierzkowski</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rast</span>  <span class="cit-name-given-names">M. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vuolo</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dello Ioio</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Leaf Shape Evolution Through Duplication, Regulatory Diversification, and Loss of a Homeobox Gene.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">343</span>: <span class="cit-fpage">780</span>–<span class="cit-lpage">783</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1248384</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DVlad%26rft.auinit1%253DD.%26rft.volume%253D343%26rft.issue%253D6172%26rft.spage%253D780%26rft.epage%253D783%26rft.atitle%253DLeaf%2BShape%2BEvolution%2BThrough%2BDuplication%252C%2BRegulatory%2BDiversification%252C%2Band%2BLoss%2Bof%2Ba%2BHomeobox%2BGene%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1248384%26rft_id%253Dinfo%253Apmid%252F24531971%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNDMvNjE3Mi83ODAiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzExLzM1ODMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3583.49" data-doi="10.1038/350241a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vollbrecht</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Veit</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sinha</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hake</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">The developmental gene Knotted-1 is a member of a maize homeobox gene family.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">350</span>: <span class="cit-fpage">241</span>–<span class="cit-lpage">243</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/350241a0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DVollbrecht%26rft.auinit1%253DE.%26rft.volume%253D350%26rft.issue%253D6315%26rft.spage%253D241%26rft.epage%253D243%26rft.atitle%253DThe%2Bdevelopmental%2Bgene%2BKnotted-1%2Bis%2Ba%2Bmember%2Bof%2Ba%2Bmaize%2Bhomeobox%2Bgene%2Bfamily.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F350241a0%26rft_id%253Dinfo%253Apmid%252F1672445%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/350241a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1672445&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F11%2F3583.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1991FC77900061&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3583.50" data-doi="10.1073/pnas.1102819108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Whipple</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kebrom</span>  <span class="cit-name-given-names">T. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weber</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">grassy tillers1 promotes apical dominance in maize and responds to shade signals in the grasses.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">E506</span>–<span class="cit-lpage">E512</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1102819108</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1102819108%26rft_id%253Dinfo%253Apmid%252F21808030%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA4LzMzL0U1MDYiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzExLzM1ODMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-book" id="cit-8.11.3583.51"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wickham</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-source">ggplot2: Elegant Graphics for Data Analysis</span>. <span class="cit-publ-name">Springer-Verlag</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-other" id="cit-8.11.3583.52"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wickham</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Francois</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> dplyr: A grammar of data manipulation. R package version 0. 4 3.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3583.53" data-doi="10.1534/genetics.114.166785"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Woods</span>  <span class="cit-name-given-names">D. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ream</span>  <span class="cit-name-given-names">T. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Minevich</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hobert</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amasino</span>  <span class="cit-name-given-names">R. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">PHYTOCHROME C is an essential light receptor for photoperiodic flowering in the temperate grass, Brachypodium distachyon.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">397</span>–<span class="cit-lpage">408</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.114.166785</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.166785%26rft_id%253Dinfo%253Apmid%252F25023399%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk4LzEvMzk3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMS8zNTgzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-8.11.3583.54"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yamanaka</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nakagawa</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mori</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nakano</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Imamura</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">MCA1 and MCA2 that mediate Ca2+ uptake have distinct and overlapping roles in Arabidopsis. Plant Physio.</span>, <span class="cit-vol">152</span>: <span class="cit-fpage">1284</span>–<span class="cit-lpage">1296</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-8.11.3583.55" data-doi="10.1111/pbi.12559"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zou</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Bulked sample analysis in genetics, genomics and crop improvement.</span> <abbr class="cit-jnl-abbrev">Plant Biotechnol. J.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">1941</span>–<span class="cit-lpage">1955</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/pbi.12559</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBiotechnol.%2BJ.%26rft.volume%253D14%26rft.spage%253D1941%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fpbi.12559%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/pbi.12559&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/8/11/3583.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/11/3577" title="Estimating Asian Contribution to the Brazilian Population: A New Application of a Validated Set of 61 Ancestry Informative Markers" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/11/3593" title="A Drosophila CRISPR/Cas9 Toolkit for Conditionally Manipulating Gene Expression in the Prothoracic Gland as a Test Case for Polytene Tissues" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/8/11"><span>Volume 8 Issue 11, November 2018</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/8/11.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/8/11.cover-source.jpg?itok=lFO8v6bg" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 8 (11)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/11/3583" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-fKGoyDJEyguWHoc2k4f-_GXR5kGgJKk1TJhuH1JSTY4" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/8/11/3583" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/8/11/3583" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/8/11/3583" class="active" data-icon-position="" data-hide-link-title="0">Bulked-Segregant Analysis Coupled to Whole Genome Sequencing (BSA-Seq) for Rapid Gene Cloning in Maize</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/491246" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-nnOVJjPucLjPB7TRzKUU4qBZk8kV5vuj8el1UTERUV0" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/11/3583.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/8/11/3583" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-wkiKhCmg7M4kJZ1f6A8WtWY3ycV5aGDx2bGvnsHlzFI" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/8/11/3583" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/11/3583.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="491246" id="node491246--2" data-pisa="ggg;8/11/3583" data-pisa-master="ggg;g3.118.200499" data-apath="/ggg/8/11/3583.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Bulked-Segregant Analysis Coupled to Whole Genome Sequencing (BSA-Seq) for Rapid Gene Cloning in Maize</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Harry Klein</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-0747-5856" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Yuguo Xiao</span>, <span class="highwire-citation-author" data-delta="2">Phillip A. Conklin</span>, <span class="highwire-citation-author" data-delta="3">Rajanikanth Govindarajulu</span>, <span class="highwire-citation-author" data-delta="4">Jacob A. Kelly</span>, <span class="highwire-citation-author" data-delta="5">Michael J. Scanlon</span>, <span class="highwire-citation-author" data-delta="6">Clinton J. Whipple</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0002-0369-8606" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Madelaine Bartlett</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3583-3592;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200499 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node491246--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Harry Klein </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Biology Graduate Program and Biology Department, University of Massachusetts Amherst, Amherst, MA 01003</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Harry%2BKlein%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Klein%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHarry%2BKlein%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Yuguo Xiao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biology, Brigham Young University, 4102 LSB, Provo, UT 84602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yuguo%2BXiao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xiao%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYuguo%2BXiao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-0747-5856" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Yuguo Xiao</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Phillip A. Conklin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Plant Biology, Cornell University, Ithaca, NY 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Phillip%2BA.%2BConklin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Conklin%20PA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APhillip%2BA.%2BConklin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Rajanikanth Govindarajulu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Biology, West Virginia University, Morgantown, WV 26506</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rajanikanth%2BGovindarajulu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Govindarajulu%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARajanikanth%2BGovindarajulu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Jacob A. Kelly </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biology, Brigham Young University, 4102 LSB, Provo, UT 84602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jacob%2BA.%2BKelly%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kelly%20JA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJacob%2BA.%2BKelly%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Michael J. Scanlon </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Plant Biology, Cornell University, Ithaca, NY 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BJ.%2BScanlon%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Scanlon%20MJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BJ.%2BScanlon%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Clinton J. Whipple </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biology, Brigham Young University, 4102 LSB, Provo, UT 84602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Clinton%2BJ.%2BWhipple%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Whipple%20CJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AClinton%2BJ.%2BWhipple%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Madelaine Bartlett </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Biology Graduate Program and Biology Department, University of Massachusetts Amherst, Amherst, MA 01003</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Madelaine%2BBartlett%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bartlett%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMadelaine%2BBartlett%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-0369-8606" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Madelaine Bartlett</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mbartlett@bio.umass.edu" class="" data-icon-position="" data-hide-link-title="0">mbartlett@bio.umass.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/8/11/3583" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/8/11/3583&amp;title=Bulked-Segregant%20Analysis%20Coupled%20to%20Whole%20Genome%20Sequencing%20%28BSA-Seq%29%20for%20Rapid%20Gene%20Cloning%20in%20Maize" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/8/11/3583&amp;title=Bulked-Segregant%20Analysis%20Coupled%20to%20Whole%20Genome%20Sequencing%20%28BSA-Seq%29%20for%20Rapid%20Gene%20Cloning%20in%20Maize" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/8/11/3583&amp;title=Bulked-Segregant%20Analysis%20Coupled%20to%20Whole%20Genome%20Sequencing%20%28BSA-Seq%29%20for%20Rapid%20Gene%20Cloning%20in%20Maize" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/8/11/3583&amp;text=Bulked-Segregant%20Analysis%20Coupled%20to%20Whole%20Genome%20Sequencing%20%28BSA-Seq%29%20for%20Rapid%20Gene%20Cloning%20in%20Maize" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/8/11/3583&amp;title=Bulked-Segregant%20Analysis%20Coupled%20to%20Whole%20Genome%20Sequencing%20%28BSA-Seq%29%20for%20Rapid%20Gene%20Cloning%20in%20Maize" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/8/11/3583&amp;t=Bulked-Segregant%20Analysis%20Coupled%20to%20Whole%20Genome%20Sequencing%20%28BSA-Seq%29%20for%20Rapid%20Gene%20Cloning%20in%20Maize" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/8/11/3583&amp;title=Bulked-Segregant%20Analysis%20Coupled%20to%20Whole%20Genome%20Sequencing%20%28BSA-Seq%29%20for%20Rapid%20Gene%20Cloning%20in%20Maize" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/8/11/3583&amp;title=Bulked-Segregant%20Analysis%20Coupled%20to%20Whole%20Genome%20Sequencing%20%28BSA-Seq%29%20for%20Rapid%20Gene%20Cloning%20in%20Maize" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/491246/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="491246" id="node491246--3" data-pisa="ggg;8/11/3583" data-pisa-master="ggg;g3.118.200499" data-apath="/ggg/8/11/3583.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Bulked-Segregant Analysis Coupled to Whole Genome Sequencing (BSA-Seq) for Rapid Gene Cloning in Maize</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Harry Klein</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-0747-5856" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Yuguo Xiao</span>, <span class="highwire-citation-author" data-delta="2">Phillip A. Conklin</span>, <span class="highwire-citation-author" data-delta="3">Rajanikanth Govindarajulu</span>, <span class="highwire-citation-author" data-delta="4">Jacob A. Kelly</span>, <span class="highwire-citation-author" data-delta="5">Michael J. Scanlon</span>, <span class="highwire-citation-author" data-delta="6">Clinton J. Whipple</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0002-0369-8606" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Madelaine Bartlett</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3583-3592;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200499 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node491246--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Harry Klein </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Biology Graduate Program and Biology Department, University of Massachusetts Amherst, Amherst, MA 01003</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Harry%2BKlein%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Klein%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHarry%2BKlein%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Yuguo Xiao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biology, Brigham Young University, 4102 LSB, Provo, UT 84602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yuguo%2BXiao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xiao%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYuguo%2BXiao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-0747-5856" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Yuguo Xiao</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Phillip A. Conklin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Plant Biology, Cornell University, Ithaca, NY 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Phillip%2BA.%2BConklin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Conklin%20PA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APhillip%2BA.%2BConklin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Rajanikanth Govindarajulu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Biology, West Virginia University, Morgantown, WV 26506</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rajanikanth%2BGovindarajulu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Govindarajulu%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARajanikanth%2BGovindarajulu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Jacob A. Kelly </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biology, Brigham Young University, 4102 LSB, Provo, UT 84602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jacob%2BA.%2BKelly%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kelly%20JA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJacob%2BA.%2BKelly%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Michael J. Scanlon </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Plant Biology, Cornell University, Ithaca, NY 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BJ.%2BScanlon%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Scanlon%20MJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BJ.%2BScanlon%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Clinton J. Whipple </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biology, Brigham Young University, 4102 LSB, Provo, UT 84602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Clinton%2BJ.%2BWhipple%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Whipple%20CJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AClinton%2BJ.%2BWhipple%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Madelaine Bartlett </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Biology Graduate Program and Biology Department, University of Massachusetts Amherst, Amherst, MA 01003</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Madelaine%2BBartlett%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bartlett%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMadelaine%2BBartlett%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-0369-8606" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Madelaine Bartlett</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mbartlett@bio.umass.edu" class="" data-icon-position="" data-hide-link-title="0">mbartlett@bio.umass.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3583&amp;rft.epage=3592&amp;rft.atitle=Bulked-Segregant%20Analysis%20Coupled%20to%20Whole%20Genome%20Sequencing%20%28BSA-Seq%29%20for%20Rapid%20Gene%20Cloning%20in%20Maize&amp;rft.volume=8&amp;rft.issue=11&amp;rft.date=2018-11-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Klein%2C+Harry&amp;rft.au=Xiao%2C+Yuguo&amp;rft.au=Conklin%2C+Phillip+A.&amp;rft.au=Govindarajulu%2C+Rajanikanth&amp;rft.au=Kelly%2C+Jacob+A.&amp;rft.au=Scanlon%2C+Michael+J.&amp;rft.au=Whipple%2C+Clinton+J.&amp;rft.au=Bartlett%2C+Madelaine"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/491246/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/491246/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/491246/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/491246/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/491246/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/491246/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/491246/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/491246/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/491246/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/491246/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/491246/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/491246/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/8/11/3583" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Methods</a></li><li class="odd"><a href="#sec-11" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-17" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/8/11/3583.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/8/11/3583.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/8/11/3583.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/8/11/3583.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
