<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/6/3/743" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/6/3/743.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/6/3/743.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/6/3/743.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/6/3/743/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/6/3/743.atom" />
<meta name="HW.pisa" content="ggg;6/3/743" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Implementation of the Realized Genomic Relationship Matrix to Open-Pollinated White Spruce Family Testing for Disentangling Additive from Nonadditive Genetic Effects" />
<meta name="DC.Identifier" content="10.1534/g3.115.025957" />
<meta name="DC.Date" content="2016-03-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2016 El-Dien et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="The open-pollinated (OP) family testing combines the simplest known progeny evaluation and quantitative genetics analyses as candidates’ offspring are assumed to represent independent half-sib families. The accuracy of genetic parameter estimates is often questioned as the assumption of “half-sibling” in OP families may often be violated. We compared the pedigree- vs. marker-based genetic models by analysing 22-yr height and 30-yr wood density for 214 white spruce [ Picea glauca (Moench) Voss] OP families represented by 1694 individuals growing on one site in Quebec, Canada. Assuming half-sibling, the pedigree-based model was limited to estimating the additive genetic variances which, in turn, were grossly overestimated as they were confounded by very minor dominance and major additive-by-additive epistatic genetic variances. In contrast, the implemented genomic pairwise realized relationship models allowed the disentanglement of additive from all nonadditive factors through genetic variance decomposition. The marker-based models produced more realistic narrow-sense heritability estimates and, for the first time, allowed estimating the dominance and epistatic genetic variances from OP testing. In addition, the genomic models showed better prediction accuracies compared to pedigree models and were able to predict individual breeding values for new individuals from untested families, which was not possible using the pedigree-based model. Clearly, the use of marker-based relationship approach is effective in estimating the quantitative genetic parameters of complex traits even under simple and shallow pedigree structure." />
<meta name="DC.Contributor" content="Omnia Gamal El-Dien" />
<meta name="DC.Contributor" content="Blaise Ratcliffe" />
<meta name="DC.Contributor" content="Jaroslav Klápště" />
<meta name="DC.Contributor" content="Ilga Porth" />
<meta name="DC.Contributor" content="Charles Chen" />
<meta name="DC.Contributor" content="Yousry A. El-Kassaby" />
<meta name="article:published_time" content="2016-03-01" />
<meta name="article:section" content="Genomic Selection" />
<meta name="citation_title" content="Implementation of the Realized Genomic Relationship Matrix to Open-Pollinated White Spruce Family Testing for Disentangling Additive from Nonadditive Genetic Effects" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The open-pollinated (OP) family testing combines the simplest known progeny evaluation and quantitative genetics analyses as candidates’ offspring are assumed to represent independent half-sib families. The accuracy of genetic parameter estimates is often questioned as the assumption of “half-sibling” in OP families may often be violated. We compared the pedigree- &lt;i&gt;vs.&lt;/i&gt; marker-based genetic models by analysing 22-yr height and 30-yr wood density for 214 white spruce [&lt;i&gt;Picea glauca&lt;/i&gt; (Moench) Voss] OP families represented by 1694 individuals growing on one site in Quebec, Canada. Assuming half-sibling, the pedigree-based model was limited to estimating the additive genetic variances which, in turn, were grossly overestimated as they were confounded by very minor dominance and major additive-by-additive epistatic genetic variances. In contrast, the implemented genomic pairwise realized relationship models allowed the disentanglement of additive from all nonadditive factors through genetic variance decomposition. The marker-based models produced more realistic narrow-sense heritability estimates and, for the first time, allowed estimating the dominance and epistatic genetic variances from OP testing. In addition, the genomic models showed better prediction accuracies compared to pedigree models and were able to predict individual breeding values for new individuals from untested families, which was not possible using the pedigree-based model. Clearly, the use of marker-based relationship approach is effective in estimating the quantitative genetic parameters of complex traits even under simple and shallow pedigree structure.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2016/03/01" />
<meta name="citation_mjid" content="ggg;6/3/743" />
<meta name="citation_id" content="6/3/743" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/6/3/743" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/6/3/743.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/6/3/743.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/6/3/743.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.115.025957" />
<meta name="citation_pmid" content="26801647" />
<meta name="citation_volume" content="6" />
<meta name="citation_issue" content="3" />
<meta name="citation_num_pages" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genomic Selection" />
<meta name="citation_firstpage" content="743" />
<meta name="citation_lastpage" content="753" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Omnia Gamal El-Dien" />
<meta name="citation_author_institution" content="Department of Forest and Conservation Sciences, Faculty of Forestry, University of British Columbia, Vancouver, V6T 1Z4, Canada" />
<meta name="citation_author_institution" content="Pharmacognosy Department, Faculty of Pharmacy, Alexandria University, 21521, Egypt" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-9112-8522" />
<meta name="citation_author" content="Blaise Ratcliffe" />
<meta name="citation_author_institution" content="Department of Forest and Conservation Sciences, Faculty of Forestry, University of British Columbia, Vancouver, V6T 1Z4, Canada" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-4469-2929" />
<meta name="citation_author" content="Jaroslav Klápště" />
<meta name="citation_author_institution" content="Department of Forest and Conservation Sciences, Faculty of Forestry, University of British Columbia, Vancouver, V6T 1Z4, Canada" />
<meta name="citation_author_institution" content="Department of Genetics and Physiology of Forest Trees, Faculty of Forestry and Wood Sciences, Czech University of Life Sciences Prague, 165 21 Praha 6, Czech Republic" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-5504-3735" />
<meta name="citation_author" content="Ilga Porth" />
<meta name="citation_author_institution" content="Department of Forest and Conservation Sciences, Faculty of Forestry, University of British Columbia, Vancouver, V6T 1Z4, Canada" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-9344-6348" />
<meta name="citation_author" content="Charles Chen" />
<meta name="citation_author_institution" content="Department of Biochemistry and Molecular Biology, Oklahoma State University, Stillwater, Oklahoma 74078" />
<meta name="citation_author" content="Yousry A. El-Kassaby" />
<meta name="citation_author_institution" content="Department of Forest and Conservation Sciences, Faculty of Forestry, University of British Columbia, Vancouver, V6T 1Z4, Canada" />
<meta name="citation_author_email" content="y.el-kassaby@ubc.ca" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-4887-8977" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=G. R. Askew;citation_author=Y. A. El-Kassaby;citation_title=Estimation of relationship coefficients among progeny derived from wind-pollinated orchard seeds.;citation_pages=267-272;citation_volume=88;citation_year=1994;citation_pmid=24185937" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=S. Avendano;citation_author=J. A. Woolliams;citation_author=B. Villanueva;citation_title=Mendelian sampling terms as a selective advantage in optimum breeding schemes with restrictions on the rate of inbreeding.;citation_pages=55-64;citation_volume=83;citation_year=2004;citation_issue=1;citation_pmid=15125067;citation_doi=10.1017/S0016672303006566" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=N. A. Baird;citation_author=P. D. Etter;citation_author=T. S. Atwood;citation_author=M. C. Currey;citation_author=A. L. Shiver;citation_title=Rapid SNP discovery and genetic mapping using sequenced RAD markers.;citation_pages=e3376-e3376;citation_volume=3;citation_year=2008;citation_issue=10;citation_pmid=18852878;citation_doi=10.1371/journal.pone.0003376" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. Beaulieu;citation_author=T. Doerksen;citation_author=B. Boyle;citation_author=S. Clément;citation_author=M. Deslauriers;citation_title=Association genetics of wood physical traits in the conifer white spruce and relationships with gene expression.;citation_volume=188;citation_year=2011;citation_pmid=21385726;citation_doi=10.1534/genetics.110.125781" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=J. Beaulieu;citation_author=T. Doerksen;citation_author=S. Clément;citation_author=J. Mackay;citation_author=J. Bousquet;citation_title=Accuracy of genomic selection models in a large population of open-pollinated families in white spruce.;citation_pages=343-352;citation_volume=113;citation_year=2014;citation_pmid=24781808;citation_doi=10.1038/hdy.2014.36" />
<meta name="citation_reference" content="citation_journal_title=Can. J. For. Res.;citation_author=H. D. Bradshaw;citation_author=G. S. Foster;citation_title=Marker-aided selection and propagation system in trees: advantages of cloning for studying quantitative inheritance.;citation_pages=1044-1049;citation_volume=22;citation_year=1992;citation_doi=10.1139/x92-139" />
<meta name="citation_reference" content="citation_journal_title=N. Z. J. For. Sci.;citation_author=R. D. Burdon;citation_author=C. J. A. Shelbourne;citation_title=Breeding populations for recurrent selection: conflicts and possible solutions.;citation_pages=174-193;citation_volume=1;citation_year=1971" />
<meta name="citation_reference" content="citation_journal_title=Unasylva;citation_author=R. Z. Callaham;citation_title=Provenance research: investigation of genetic diversity associated with geography.;citation_pages=40-50;citation_volume=18;citation_year=1964" />
<meta name="citation_reference" content="citation_journal_title=Tree Genet. Genomes;citation_author=C. Chen;citation_author=S. E. Mitchell;citation_author=R. J. Elshire;citation_author=E. S. Buckler;citation_author=Y. A. El-Kassaby;citation_title=Mining conifers’ mega-genome using rapid and efficient multiplexed high-throughput genotyping-by-sequencing (GBS) SNP discovery platform.;citation_pages=1537-1544;citation_volume=9;citation_year=2013;citation_doi=10.1007/s11295-013-0657-1" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=E. Costa;citation_author=J. Silva;citation_author=N. M. G. Borralho;citation_author=B. M. Botts;citation_title=Additive and non-additive genetic parameters from clonally replicated and seedling progenies of Eucalyptus globulus.;citation_pages=1113-1119;citation_volume=108;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=Journal of genetics;citation_journal_abbrev=J Genet;citation_author=J. F. Crow;citation_title=Maintaining evolvability.;citation_pages=349-353;citation_volume=87;citation_year=2008;citation_issue=4;citation_pmid=19147924;citation_doi=10.1007/s12041-008-0057-8" />
<meta name="citation_reference" content="citation_journal_title=Philosophical Transactions of the Royal Society B: Biological Sciences;citation_journal_abbrev=Phil Trans R Soc B;citation_author=J. F. Crow;citation_title=On epistasis: why it is unimportant in polygenic directional selection;citation_pages=1241-1244;citation_volume=365;citation_year=2010;citation_issue=1544;citation_pmid=20308099;citation_doi=10.1098/rstb.2009.0275" />
<meta name="citation_reference" content="citation_journal_title=Tree Physiol.;citation_author=Y. A. El-Kassaby;citation_title=Evaluation of the tree-improvement delivery system: factors affecting genetic potential.;citation_pages=545-550;citation_volume=15;citation_year=1993" />
<meta name="citation_reference" content="citation_journal_title=Can. J. Bot.;citation_author=Y. A. El-Kassaby;citation_author=H. J. Barclay;citation_title=Cost of reproduction in Douglas-fir.;citation_pages=1429-1432;citation_volume=70;citation_year=1992" />
<meta name="citation_reference" content="citation_journal_title=For. Ecol. Manage.;citation_author=Y. A. El-Kassaby;citation_author=O. Sziklai;citation_title=Genetic variation of allozyme and quantitative traits in a selected Douglas-fir [Pseudotsuga menziesii var. menziesii (Mirb.) Franco] population.;citation_pages=115-126;citation_volume=4;citation_year=1982;citation_doi=10.1016/0378-1127(82)90009-3" />
<meta name="citation_reference" content="citation_journal_title=Silvae Genet.;citation_author=Y. A. El-Kassaby;citation_author=A. M. K. Fashler;citation_author=O. Sziklai;citation_title=Reproductive phenology and its impact on genetically improved seed production in a Douglas-fir seed orchard.;citation_pages=120-125;citation_volume=33;citation_year=1984" />
<meta name="citation_reference" content="citation_journal_title=Silvae Genet.;citation_author=Y. A. El-Kassaby;citation_author=A. M. K. Fashler;citation_author=M. Crown;citation_title=Variation in fruitfulness in a Douglas-fir seed orchard and its effect on crop-management decisions.;citation_pages=113-121;citation_volume=38;citation_year=1989" />
<meta name="citation_reference" content="citation_journal_title=New For.;citation_author=Y. A. El-Kassaby;citation_author=J. Klápště;citation_author=R. D. Guy;citation_title=Breeding without breeding: selection using the genomic best linear unbiased predictor method (GBLUP).;citation_pages=631-637;citation_volume=43;citation_year=2012;citation_doi=10.1007/s11056-012-9338-4" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=R. J. Elshire;citation_author=J. C. Glaubitz;citation_author=Q. Sun;citation_author=J. A. Poland;citation_author=K. Kawamoto;citation_title=A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.;citation_pages=e19379-e19379;citation_volume=6;citation_year=2011;citation_issue=5;citation_pmid=21573248;citation_doi=10.1371/journal.pone.0019379" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=G. S. Foster;citation_author=D. V. Shaw;citation_title=Using clonal replications to explore genetic variation in a perennial plant species.;citation_pages=788-794;citation_volume=76;citation_year=1988;citation_pmid=24232359;citation_doi=10.1007/BF00303527" />
<meta name="citation_reference" content="citation_journal_title=P. Roy. Soc. B-Biol. Sci.;citation_author=F. D. Frentiu;citation_author=S. M. Clegg;citation_author=J. Chittock;citation_author=T. Burke;citation_author=M. W. Blows;citation_title=Pedigree-free animal models: the relatedness matrix reloaded;citation_pages=639-647;citation_volume=275;citation_year=2008;citation_doi=10.1098/rspb.2007.1032" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=O. Gamal El-Dien;citation_author=B. Ratcliffe;citation_author=J. Klápště;citation_author=C. Chen;citation_author=I. Porth;citation_title=Prediction accuracies for growth and wood attributes of interior spruce in space using genotyping-by-sequencing.;citation_pages=370;citation_volume=16;citation_year=2015;citation_pmid=25956247;citation_doi=10.1186/s12864-015-1597-y" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=L. Gay;citation_author=M. Siol;citation_author=J. Ronfort;citation_title=Pedigree-free estimates of heritability in the wild: promising prospects for selfing populations.;citation_pages=e66983;citation_volume=8;citation_year=2013;citation_pmid=23825602;citation_doi=10.1371/journal.pone.0066983" />
<meta name="citation_reference" content="citation_journal_title=Biometrics;citation_author=A. R. Gilmour;citation_author=R. Thompson;citation_author=B. R. Cullis;citation_title=Average information REML: an efficient algorithm for variance parameter estimation in linear mixed models.;citation_pages=1440-1450;citation_volume=51;citation_year=1995;citation_doi=10.2307/2533274" />
<meta name="citation_reference" content="citation_title=ASReml User Guide Release 3.0;citation_author=A. R. Gilmour;citation_author=B. Gogel;citation_author=B. Cullis;citation_author=R. Thompson;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. Habier;citation_author=R. L. Fernando;citation_author=J. C. M. Dekkers;citation_title=The Impact of Genetic Relationship Information on Genome-Assisted Breeding Values;citation_pages=2389-2397;citation_volume=177;citation_year=2007;citation_issue=4;citation_pmid=18073436;citation_doi=10.1534/genetics.107.081190" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=D. Habier;citation_author=J. Tetens;citation_author=F. R. Seefried;citation_author=P. Lichtner;citation_author=G. Thaller;citation_title=The impact of genetic relationship information on genomic breeding values in German Holstein cattle.;citation_pages=5-5;citation_volume=42;citation_year=2010;citation_pmid=20170500;citation_doi=10.1186/1297-9686-42-5" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. Habier;citation_author=R. L. Fernando;citation_author=D. J. Garrick;citation_title=Genomic BLUP decoded: a look into the black box of genomic prediction.;citation_volume=194;citation_year=2013;citation_pmid=23640517;citation_doi=10.1534/genetics.113.152207" />
<meta name="citation_reference" content="citation_journal_title=Tree Genet. Genomes;citation_author=H. R. Hallingback;citation_author=G. Jansson;citation_title=Genetic information from progeny trials: a comparison between progenies generated by open pollination and by controlled crosses.;citation_pages=731-740;citation_volume=9;citation_year=2013;citation_doi=10.1007/s11295-012-0588-2" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=B. J. Hayes;citation_author=P. M. Visscher;citation_author=M. E. Goddard;citation_title=Increased accuracy of artificial selection by using the realized relationship matrix.;citation_pages=47-60;citation_volume=91;citation_year=2009;citation_issue=1;citation_pmid=19220931;citation_doi=10.1017/S0016672308009981" />
<meta name="citation_reference" content="citation_journal_title=Biometrics;citation_journal_abbrev=Biometrics;citation_author=C. R. Henderson;citation_title=Best linear unbiased estimation and prediction under a selection model.;citation_pages=423-447;citation_volume=31;citation_year=1975;citation_issue=2;citation_pmid=1174616;citation_doi=10.2307/2529430" />
<meta name="citation_reference" content="citation_journal_title=Biometrics;citation_author=C. R. Henderson;citation_title=Simple method for computing inverse of a numerator relationship matrix used in prediction of breeding values.;citation_pages=69-83;citation_volume=32;citation_year=1976;citation_doi=10.2307/2529339" />
<meta name="citation_reference" content="citation_title=Applications of Linear Models in Animal Breeding;citation_author=C. R. Henderson;citation_year=1984" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=W. G. Hill;citation_author=M. E. Goddard;citation_author=P. M. Visscher;citation_title=Data and theory point to mainly additive genetic variance for complex traits.;citation_pages=e1000008-e1000008;citation_volume=4;citation_year=2008;citation_issue=2;citation_pmid=18454194;citation_doi=10.1371/journal.pgen.1000008" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=W. G. Hill;citation_author=B. S. Weir;citation_title=Variation in actual relationship as a consequence of Mendelian sampling and linkage.;citation_pages=47-64;citation_volume=93;citation_year=2011;citation_doi=10.1017/S0016672310000480" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. L. Jannink;citation_title=Identifying quantitative trait locus by genetic background interactions in association studies.;citation_volume=176;citation_year=2007;citation_pmid=17179077;citation_doi=10.1534/genetics.106.062992" />
<meta name="citation_reference" content="citation_journal_title=Silvae Genet.;citation_author=K. J. S. Jayawickrama;citation_author=M. J. Carson;citation_title=A breeding strategy for the New Zealand radiata pine breeding cooperative.;citation_pages=82-90;citation_volume=49;citation_year=2000" />
<meta name="citation_reference" content="citation_journal_title=Silvae Genet.;citation_author=G. R. Johnson;citation_title=Site-to-site genetic correlations and their implications on breeding zone size and optimum number of progeny test sites for coastal Douglas-fir.;citation_pages=280-285;citation_volume=46;citation_year=1997" />
<meta name="citation_reference" content="citation_journal_title=Can. J. For. Res.;citation_author=G. K. Kiss;citation_author=A. D. Yanchuk;citation_title=Preliminary evaluation of genetic variation of weevil resistance in interior spruce in British Columbia.;citation_pages=230-234;citation_volume=21;citation_year=1991;citation_doi=10.1139/x91-028" />
<meta name="citation_reference" content="citation_journal_title=Tree Genet. Genomes;citation_author=J. Klápště;citation_author=M. Lstibůrek;citation_author=Y. A. El-Kassaby;citation_title=Estimates of genetic parameters and breeding values from western larch open-pollinated families using marker-based relationship.;citation_pages=241-249;citation_volume=10;citation_year=2014;citation_doi=10.1007/s11295-013-0673-1" />
<meta name="citation_reference" content="citation_journal_title=Tree Genet. Genomes;citation_author=J. Korecky;citation_author=J. Klápště;citation_author=M. Lstibůrek;citation_author=J. Kobliha;citation_author=C. Dana Nelson;citation_title=Comparison of genetic parameters from marker-based relationship, sibship, and combined models in Scots pine multi-site open-pollinated tests.;citation_pages=1227-1235;citation_volume=9;citation_year=2013;citation_doi=10.1007/s11295-013-0630-z" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=S. H. Lee;citation_author=M. E. Goddard;citation_author=P. M. Visscher;citation_author=J. H. J. van der Werf;citation_title=Using the realized relationship matrix to disentangle confounding factors for the estimation of genetic variance components of complex traits.;citation_pages=22-22;citation_volume=42;citation_year=2010;citation_pmid=20546624;citation_doi=10.1186/1297-9686-42-22" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=C. Lippert;citation_author=G. Quon;citation_author=E. Y. Kang;citation_author=C. M. Kadie;citation_author=J. Listgarten;citation_title=The benefits of selecting phenotype-specific variants for applications of mixed models in genomics.;citation_pages=1815;citation_volume=3;citation_year=2013;citation_pmid=23657357" />
<meta name="citation_reference" content="citation_journal_title=J. Anim. Breed. Genet.;citation_author=A. Loberg;citation_author=J. W. Durr;citation_author=W. F. Fikse;citation_author=H. Jorjani;citation_author=L. Crooks;citation_title=Estimates of genetic variance and variance of predicted genetic merits using pedigree or genomic relationship matrices in six Brown Swiss cattle populations for different traits.;citation_year=2015;citation_doi=10.1111/jbg.12142" />
<meta name="citation_reference" content="citation_title=Genetics and Analysis of Quantitative Traits;citation_author=M. Lynch;citation_author=B. Walsh;citation_year=1998" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=T. F. C. Mackay;citation_title=Epistasis and quantitative traits: using model organisms to study gene-gene interactions.;citation_pages=22-33;citation_volume=15;citation_year=2014;citation_pmid=24296533;citation_doi=10.1038/nrg3627" />
<meta name="citation_reference" content="citation_title=Linear Models for the Prediction of Animal Breeding Values;citation_author=R. A. Mrode;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=P. R. Muñoz;citation_author=M. F. R. Resende;citation_author=S. A. Gezan;citation_author=M. D. V. Resende;citation_author=G. de los Campos;citation_title=Unraveling additive from nonadditive effects using genomic relationship matrices.;citation_volume=198;citation_year=2014;citation_pmid=25324160;citation_doi=10.1534/genetics.114.171322" />
<meta name="citation_reference" content="citation_journal_title=For. Sci.;citation_author=G. Namkoong;citation_title=Inbreeding effects on estimation of genetic additive variance.;citation_pages=8-13;citation_volume=12;citation_year=1966" />
<meta name="citation_reference" content="citation_title=Tree Breeding: Principles and Strategies.;citation_author=G. Namkoong;citation_author=H. C. Kang;citation_author=J. S. Brouard;citation_year=1988" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=B. K. Peterson;citation_author=J. N. Weber;citation_author=E. H. Kay;citation_author=H. S. Fisher;citation_author=H. E. Hoekstra;citation_title=Double digest RADseq: an inexpensive method for de novo SNP discovery and genotyping in model and non-model species.;citation_pages=e37135-e37135;citation_volume=7;citation_year=2012;citation_issue=5;citation_pmid=22675423;citation_doi=10.1371/journal.pone.0037135" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=J. Poland;citation_author=J. Endelman;citation_author=J. Dawson;citation_author=J. Rutkoski;citation_author=S. Wu;citation_title=Genomic selection in wheat breeding using genotyping-by-sequencing.;citation_pages=103-113;citation_volume=5;citation_year=2012;citation_doi=10.3835/plantgenome2012.06.0006" />
<meta name="citation_reference" content="citation_journal_title=New Phytol.;citation_author=I. Porth;citation_author=J. Klápště;citation_author=O. Skyba;citation_author=B. S. K. Lai;citation_author=A. Geraldes;citation_title=Populus trichocarpa cell wall chemistry and ultrastructure trait variation, genetic control and genetic correlations.;citation_pages=777-790;citation_volume=197;citation_year=2013;citation_pmid=23278123;citation_doi=10.1111/nph.12014" />
<meta name="citation_reference" content="citation_journal_title=Ann. For. Sci.;citation_author=B. Ratcliffe;citation_author=F. J. Hart;citation_author=J. Klápště;citation_author=B. Jaquish;citation_author=S. D. Mansfield;citation_title=Genetics of wood quality attributes in western larch.;citation_pages=415-424;citation_volume=71;citation_year=2014;citation_doi=10.1007/s13595-013-0349-x" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=B. Ratcliffe;citation_author=O. Gamal El-Dien;citation_author=J. Klápště;citation_author=I. Porth;citation_author=C. Chen;citation_title=A comparison of genomic selection models across time in interior spruce (Picea engelmannii x glauca) using unordered SNP imputation methods.;citation_pages=547-555;citation_volume=115;citation_year=2015;citation_pmid=26126540;citation_doi=10.1038/hdy.2015.57." />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=P. Rigault;citation_author=B. Boyle;citation_author=P. Lepage;citation_author=J. E. K. Cooke;citation_author=J. Bousquet;citation_title=A white spruce gene catalog for conifer genome analyses.;citation_pages=14-28;citation_volume=193;citation_year=2011;citation_issue=157" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=A. W. Santure;citation_author=J. Syapley;citation_author=A. D. Ball;citation_author=T. I. M. R. Birkhead;citation_author=T. Burke;citation_title=On the use of large marker panels to estimate inbreeding and relatedness: empirical and simulation studies of a pedigreed zebra finch population typed at 771 SNPs.;citation_pages=1439-1451;citation_volume=19;citation_year=2010;citation_issue=7;citation_pmid=20149098;citation_doi=10.1111/j.1365-294X.2010.04554.x" />
<meta name="citation_reference" content="citation_journal_title=Silvae Genet.;citation_author=A. E. Squillace;citation_title=Average genetic correlations among offspring from open-pollinated forest trees.;citation_pages=149-156;citation_volume=23;citation_year=1974" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=G. Su;citation_author=O. F. Christensen;citation_author=T. Ostersen;citation_author=M. Henryon;citation_author=M. S. Lund;citation_title=Estimating additive and non-additive genetic variances and predicting genetic merits using genome-wide dense single nucleotide polymorphism markers.;citation_pages=e45293-e45293;citation_volume=7;citation_year=2012;citation_issue=9;citation_pmid=23028912;citation_doi=10.1371/journal.pone.0045293" />
<meta name="citation_reference" content="citation_journal_title=J. Evol. Biol.;citation_author=S. C. Thomas;citation_author=D. W. Coltman;citation_author=J. M. Pemberton;citation_title=The use of marker-based relationship information to estimate the heritability of body weight in a natural population: a cautionary tale.;citation_pages=92-99;citation_volume=15;citation_year=2002;citation_doi=10.1046/j.1420-9101.2002.00372.x" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=H. T. Truong;citation_author=A. M. Ramos;citation_author=F. Yalcin;citation_author=M. de Ruiter;citation_author=H. J. A. van der Poel;citation_title=Sequence-based genotyping for marker discovery and co-dominant scoring in germplasm and populations.;citation_pages=e37565-e37565;citation_volume=7;citation_year=2012;citation_issue=5;citation_pmid=22662172;citation_doi=10.1371/journal.pone.0037565" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=P. M. VanRaden;citation_title=Efficient Methods to Compute Genomic Predictions;citation_pages=4414-4423;citation_volume=91;citation_year=2008;citation_issue=11;citation_pmid=18946147;citation_doi=10.3168/jds.2007-0980" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=P. M. Visscher;citation_author=S. E. Medland;citation_author=M. A. R. Ferreira;citation_author=K. I. Morley;citation_author=G. Zhu;citation_title=Assumption-free estimation of heritability from genome-wide identity-by-descent sharing between full siblings.;citation_pages=e41-e41;citation_volume=2;citation_year=2006;citation_issue=3;citation_pmid=16565746;citation_doi=10.1371/journal.pgen.0020041" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Z. G. Vitezica;citation_author=L. Varona;citation_author=A. Legarra;citation_title=On the additive and dominance and covariance of individuals within the genomic selection scope.;citation_volume=195;citation_year=2013;citation_pmid=24121775;citation_doi=10.1534/genetics.113.155176" />
<meta name="citation_reference" content="citation_journal_title=Am. Nat.;citation_author=S. Wright;citation_title=Coefficients of inbreeding and relationship.;citation_pages=330-338;citation_volume=56;citation_year=1922;citation_doi=10.1086/279872" />
<meta name="citation_reference" content="citation_journal_title=G3 Genes Genom. Genet.;citation_author=J. Zapata-Valenzuela;citation_author=R. W. Whetten;citation_author=D. Neale;citation_author=S. McKeand;citation_author=F. Isik;citation_title=Genomic estimated breeding values using genomic relationship matrices in a cloned population of loblolly pine;citation_pages=909-916;citation_volume=3;citation_year=2013" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Implementation of the Realized Genomic Relationship Matrix to Open-Pollinated White Spruce Family Testing for Disentangling Additive from Nonadditive Genetic Effects" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3.cover-source.jpg" />
<meta name="twitter:description" content="The open-pollinated (OP) family testing combines the simplest known progeny evaluation and quantitative genetics analyses as candidates’ offspring are assumed to represent independent half-sib families. The accuracy of genetic parameter estimates is often questioned as the assumption of “half-sibling” in OP families may often be violated. We compared the pedigree- vs. marker-based genetic models by analysing 22-yr height and 30-yr wood density for 214 white spruce [ Picea glauca (Moench) Voss] OP families represented by 1694 individuals growing on one site in Quebec, Canada. Assuming half-sibling, the pedigree-based model was limited to estimating the additive genetic variances which, in turn, were grossly overestimated as they were confounded by very minor dominance and major additive-by-additive epistatic genetic variances. In contrast, the implemented genomic pairwise realized relationship models allowed the disentanglement of additive from all nonadditive factors through genetic variance decomposition. The marker-based models produced more realistic narrow-sense heritability estimates and, for the first time, allowed estimating the dominance and epistatic genetic variances from OP testing. In addition, the genomic models showed better prediction accuracies compared to pedigree models and were able to predict individual breeding values for new individuals from untested families, which was not possible using the pedigree-based model. Clearly, the use of marker-based relationship approach is effective in estimating the quantitative genetic parameters of complex traits even under simple and shallow pedigree structure." />
<meta name="og-title" property="og:title" content="Implementation of the Realized Genomic Relationship Matrix to Open-Pollinated White Spruce Family Testing for Disentangling Additive from Nonadditive Genetic Effects" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/6/3/743" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="The open-pollinated (OP) family testing combines the simplest known progeny evaluation and quantitative genetics analyses as candidates’ offspring are assumed to represent independent half-sib families. The accuracy of genetic parameter estimates is often questioned as the assumption of “half-sibling” in OP families may often be violated. We compared the pedigree- vs. marker-based genetic models by analysing 22-yr height and 30-yr wood density for 214 white spruce [ Picea glauca (Moench) Voss] OP families represented by 1694 individuals growing on one site in Quebec, Canada. Assuming half-sibling, the pedigree-based model was limited to estimating the additive genetic variances which, in turn, were grossly overestimated as they were confounded by very minor dominance and major additive-by-additive epistatic genetic variances. In contrast, the implemented genomic pairwise realized relationship models allowed the disentanglement of additive from all nonadditive factors through genetic variance decomposition. The marker-based models produced more realistic narrow-sense heritability estimates and, for the first time, allowed estimating the dominance and epistatic genetic variances from OP testing. In addition, the genomic models showed better prediction accuracies compared to pedigree models and were able to predict individual breeding values for new individuals from untested families, which was not possible using the pedigree-based model. Clearly, the use of marker-based relationship approach is effective in estimating the quantitative genetic parameters of complex traits even under simple and shallow pedigree structure." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3.cover-source.jpg" />
    <title>Implementation of the Realized Genomic Relationship Matrix to Open-Pollinated White Spruce Family Testing for Disentangling Additive from Nonadditive Genetic Effects | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"rnI9sbT8y7w6eLINcoyeXIpKLH1g3L9U-23nDiqGR_s","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460098","apath":"\/ggg\/6\/3\/743.atom","pisa":"ggg;6\/3\/743","ac":{"\/ggg\/6\/3\/743.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/6\/3\/743.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;6\/3\/743"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;6\/3\/743"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/6\/3\/743.full":true,"\/content\/6\/3\/743":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022mIe02IYR3etFut\\\\\\\/dAo9jaHwLH1LyygV\\\\\\\/tS4zfVKVDmwAsRcX6VOEfiPKHSd+4X8CuE\\\\\\\/SER68Dp\\\\\\\/xSUYQ6vT3RPPlskUzFnd0ijSa63srRhBlyqj06ksFhLRmcLa8pFQwHD803+GjJMkhSpTqtyytCPqjNHIsh0p4JvcHawgK71IhQRhVDhMaGFNHUHPNvO9CLhevJUajxo0DZU7+TcrJoafa13gnf+VA0S6Cj+OE4aUd+lQ05VrR8EnUZvPuyTyFkpjVByciTf1JTEsYhwkBYdgx77G1OWd0\\\\\\\/mSi59Vqhbh3mHxJk0hsFZ7MNrOApWKLKBr\\\\\\\/aofsx8Hol35Lh9rW5Brxe9VXxhnSVUD2USJVcvh38CuxtmFYOipj07torvE8RxlDFhVrTgyKwgdgDtiUfvRcRxxoWCCy5hGiNwCiHjmF2DZ\\\\\\\/5as3+NM7940vRksOxujzR6PUz5AYSCRah3s450KFWfPdzafJw\\\\\\\/p7bBOi\\\\\\\/KR8tKzLuXUnXapQAMjmTFS5HvKAg9n3I+GOQqcz7IVC7fmhtUKeHH6xMGu4ANu9MLST4r0m9+OBebbE3\\\\\\\/7Xt4UTbJNyktrKHQhWN133i7wIgJHPVAhFolL+wkpEYfw2jdf8Kp1l\\\\\\\/6FzeUNzu3yrv0TlhXQs79i0HMAU\\\\\\\/fuEEzSuLN\\\\\\\/RBDqCX5yv+68vyyQte0C8Fu6s66jsmG6kioO3si\\\\\\\/7w2rfYGF2hGAxlKle3cZu3Wb1eXjWe3sCu9KKjXsceBQf8XZJwE3Ljoq4M9RYlXFPr6zZzfSW6zU+oiJS5DJyYnznEz\\\\\\\/JloHLVOVko5XGBFKIX1IMddwMd169Ujdq54QPBwlzQltp6IlrkMUDg2VKq2ovwJrPc7o7VK0BJ35fIv\\\\\\\/lUsfM6viR1iVdbkFwrbvj3q5E6xyDgS5pCulatZ8Dj5szpdnbF41lyVPiiROpGzNe5+RAOfO5u3mhVvrYh6WPVSwlnY\\\\\\\/41vA0b\\\\\\\/nIlXjAqcGcXzCkdNEaERDOFPRBK+Yu2Q8VggdkFOdtwbDD\\\\\\\/RksKUOHKZYUQF1Bp9AWXyRXQczW1Vwg7Cso5l5ZPlHakZU6QG+qZp9xWrltDt0xup0Ia4ELjSdXc88Nzjxa6SvPVtTYO\\\\\\\/Xs\\\\\\\/GH8BO30n\\\\\\\/lOzwwgPV4W9jEuv3xGlHg\\\\\\\/g6C3eWKKPKKASWjJkiuaKMOgBVADLBMhe9i9GpA6zE9E+kvsnDuNtKY7WoFHKXG2CNyFkXvv\\\\\\\/kefKuTgtttWZJ3xFHeJBkQZ3+grZFvJPTym1x\\\\\\\/X95mvJ\\\\\\\/+qVd47gHYyz\\\\\\\/62\\\u0022,\\\u0022iv\\\u0022:\\\u0022s\\\\\\\/PC+AvYpvJbgHkLM88G6Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00226332f5fb259084992e5de41fd2c89325c910097dc0850e0cbec5d74eda6b04f9\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022bebC\\\\\\\/c6fhjG4R9AOd3qv0xT4S0GTsLPq2So2+qx9mcZko2j28jVHNpx\\\\\\\/6DdX44BNDfC1xy84XQ0hGp0bVt7j6kkg0izMlQXaTUhNCxGErURhHv3oeolNp3wBWTWz5iFiQhrqxQjrQ+C8YnFZKXl96cw\\\\\\\/y4U08GZAP8mR\\\\\\\/GYTEw9Z8\\\\\\\/ABMvk2++jZL5WsnPEYplf4q0kBmWU6BGQgSMJDqw39bOthtoTvj6a4kWJ+TiSJHy63izHj7VynQvxBK5rN\\\\\\\/ufK4zNOM8oWOy0sqygJQ3CnvDY6l6Y5e30A+jLm3UvclN9u11Ao+5JbSk8AE3f60xbzLcsbpSCFL1vO7UeCcbT08GO3gJtVqGVtMtmvE\\\\\\\/+C63lTXioGAhI2ecYAUBc+gNppNjz4MPbX70\\\\\\\/TOYyTFsn5UxscKJct\\\\\\\/PDJChYF0OijbpCLxoezlbGWyCNbLl4i8nO6Df\\\\\\\/5zrYI+Q\\\\\\\/mNtsNJJ2J9GpkYK7QTC+StwEVchFWV0BdCrVnDIg1wqYStC\\\\\\\/LNsvJmraF46yM0NHtnVsCrK8F3Pur5BivovsxpHQw3qJ6Eubhb0trTB0IUO+u8Slgw99Ww+Gbus7frcbVdkZ1L7vljszd7CW5PFp7B8z5y57EDNp2TQ2Pe\\\\\\\/Q\\\\\\\/8DMXqw+lOEvaroMwGxqCChTZA\\\\\\\/GYqq6C5WhBUcrlesmqIf39Mp0zn2X+CU2zVTrs3DMkjN9vVYfFJYgz1Y7EH3zvjDsmmGCEog0Pd7SWW0wQxQPZ4Tb7lfZswcU4OuUXWS3bGCMfz4nwgpx7WWpOkhEqgD3blNn1L7gxAgGM\\\\\\\/Umlh78pemRTwXTswFxTuoYxrjkMSuYJHaiQ8Rd+ui\\\\\\\/8rwXPWtsgRJG214kJu3D+WMV+Fve05c58jA2ROgI4k7YPrxA1QM5\\\\\\\/mYQ\\\\\\\/mHt7eBxDxSLDWwNk8+KvZgCSZX59pjsGFHMpZZNoZp2NZQNqrMDgH5azp04f4eoR++KhOLU1Rg\\\\\\\/UhMnp8uF9CvnCb3pboga2L6q4exWVnGl2Wk82kPmgAPXQ2+sd6hRzBRKgXE484KM3ZC6mugDDu\\\\\\\/YQ8SBuPc+H4lmbVlKdX\\\\\\\/c1H6N55a8LD+7TPLJ8acRr9JXNH8ZiXrGNprchM08FcUdZ3HddgeF1bc1iqCAb1oZ1naoGiDff0MFEh\\\\\\\/imnx4TV3Jsk\\\\\\\/No3jS\\\\\\\/VDOH8LEUU4911w+tFsCTLVsmcjsspDM2BJLd+bJ0HbssqOxL0g2PDxvj5KIWcL83nddRpvDh6n8xzIMlUY5DP\\\\\\\/AnSE8igUddtvNj5bo3m3RAF2A3Ba\\\\\\\/CU09nW5dSyTKbf79hxHe1Emydwwew7LGuramE796mPoiybFTXcMuyRgNbKhTy+k6ye6vpJHx8iEbweI2uha7bEdptGxBudu9f2yg4wdvG+ms8v56e40vOyRi81emK\\\\\\\/sOmnfxCoBYNFMuSyONgCEcNdX9Gj3vjo+W0Cq1OnxDexukA2fpCDx0CoPILxh9iM0n\\\\\\\/UQW+sa1YmujXTAXjDwgpHIBgc00EqIUDTt8JHNyGkMJGrLz1Fqk3CbcgdRQ+OTdLBgrheztJzjGyI2spHo9ZdIM7MjAbK1wKwqnc\\\\\\\/3ZsXgFGNCoUoCNfX3SPeqnHbr5j+ej4qbp5k5ATWLNnpcwNFn2RiAoUE+7LcQdcOu5rQH4xba1ZPn0qiF+QvqCqc0cf5rm6B2XLl5GOgHEun8yY\\\\\\\/ljscV1j0PhxDAg2qECJiyOBoFcX01Vxg+i65mvjE3sSeO5+tSLD21\\\\\\\/2KP0ad7cYhaFpaoBTG20ZdNTB0i7Pga4jWDRc1wBzhZq92haThNg\\\\\\\/B37Up2+cJofVUqoeDwpaR\\\\\\\/qFB9nRnYp0aZHhguvWQA6\\\\\\\/f5Er1w5bRXjPpaJV7J+dbUFhSB6Oeh94swPxXLvGLipaEtG66l67I74oOIDmmhh2S45lgr54f5lPK9CqdZLD9GiszK0f632fQyV9CdZ3WS7kdsy6fdEnu7U2l5hL5pOfvOq3v10PQwgYf0liGI92QkLBuo7l9T89YfVjYFzX+i2lma0UPJr2DaK\\\\\\\/MwbBCTK6WKk1fU5kYj1oTdn\\\\\\\/ttzfvN72u5Wi8ug7qLIPSxXCpcvTazkg1rhnv+sFnuy\\\\\\\/CXXOxBsI7wbCXeMHzCEXd5r4KmKy\\\\\\\/8sstreE+QQlosnxzMlxuWifdAKCXGzpv+bdB\\\\\\\/9xexaqWlcumOFlaCR6jXeAcF+x+9m2oHVs6AMlHMWxRBQ6iU1tq7cMqQIdpPaV\\\\\\\/TkKylnzQt2AO\\\\\\\/wOsmfZupvouCyYCvZ9eqb0JBfKZRUwHP+3JjpUc0LVy1h21PepmZ\\\\\\\/wvatLHu4pI8DU+35EY+05Al9AW\\\\\\\/QGaD1JH+SRa1C\\\\\\\/ortxJ3fcvnvlbfPPZ6xEDwVjYw0pYYIE4fI09CvHZSFKs9xjR5+Ok6DklUyau3FbCQ3RzMGO7vxxis+8FmabgpqVitIcYY8eQwxf3GqJBe7YATLVrrQvsfNMSlYiDyTdiw0CnqGTunV8kudmgdMsU1ZlbU55LoyC4oiuVCbndzOpsK3mDXprXNE+ByHXH5m7rsc5HhTPoxql3nw31kma2moX5yqvmf30mb4ZnfdITCHjiKbS18805vajl+9QqP9Ql9bZmd1xiN4MJFfUBbz2D+87Yh1hVv+tupg7bqYLEveD+t0R6baHMqJ5CSFuNWqcOFDTvcdlgSzxZAB9gUAqn3Eb7zkBe1Cn08rrZBpoGMjL2eEDvp1M+5PXiXiWLxRB1TbeXawdRBTtkrwJ2jyyLZztvdAjL2+rZW72PFlycy4eZiyNG1NPtKtBVm+yNajLtfCST8PYlrP4XajZ+rx8bCk+vBsD4TnEQ2gHWF\\\\\\\/GVtTa1WKT1zbE2DVPwiv\\\\\\\/jW3LhRsSuupaALTqySqmB+Lu0VyAl2FTskkVHJbGEqiexivW7QStAnX\\\\\\\/ab1ZoWBkI\\\\\\\/Mv7Fwz\\\\\\\/2up81dhq14AgJdZHoy\\\\\\\/mnorIhqfJaGNZZR+rb+zeKKRm\\\\\\\/Y\\\\\\\/D63xYNO1X\\\\\\\/v9FHFT9XZfilqFvwaIEJjAHX\\\\\\\/9vet+33X14xLtvsNsMpmttGPGH8vab+yu3i0qEioEBGr\\\\\\\/77sFnDvxDLup8fEcZdEJ\\\\\\\/R\\\\\\\/PaeIb4w1MFbADvhZlbjqVfsqcaDtBN\\\\\\\/wIznSIj0hPc4Qmdbe82SpF7o2krxuiM6pZnUA5CALJOuf0EN98gTEvcf9JeTziHFQ\\\\\\\/SiKqJz4AnHIx3RhA0KjWRqmUeIFflsvG5N92SmcvWZv7ItCUIoYbvgsxwbRPUJdz80r\\\\\\\/BfbCJ3VU20anR33eopkqz7fCAu0QyryTMZu9\\\\\\\/keSdnNMPkEhL9lqQvnj4nsTuNiRhqRvVrOP0nSepp0poe2cnTJNlL2Qsn9miAxiVEuUVkEky7gPvxsee0HlyEJhrA0WeruE4Jln\\\\\\\/OZM3iL\\\\\\\/jTvx9ow\\\\\\\/IW4PgJlkcAtgHPRQ\\\\\\\/D91aDBFMlMH30GAYoAkkeuI89EefwgZ2ReJIyPj40WUqoPRSCYb6DJnWmM3s7aniqRrkT8bYN2L5F0pVK9l6cEVDLtwOPDvmdT\\\\\\\/a2QDcCq6W0qwjZMF5ARBPKpILRCRvPXVe4\\\\\\\/ynvLAuVulQc5ZSGNYuGGQcTvwAaR1w3UyvM4CfHh44iH5yY3y5faNMR8D0YnBTwhs6mDS\\\\\\\/+8WWhcCSEoYJbETMnh5wyWmNvEH6olT5zIF0b1eYTah6ZZ5h92z\\\\\\\/rb7wDL0z95M4vBkRfCOfhhnBGd\\\\\\\/ErdSd1N3z9+W8tebrQLTEumtBQposaraEXLq2vFs78BME6GPyJgBZhVrDdtZogOWIDo\\\\\\\/DOWLkvGA9F0TlC2a9Glm4MVThFzI1RbMbufHAkFFyzrUg8LjnBKXwpGN2L7kkaWRz8233XZXrBJWHd7bl3vX36HI0p03j8A+WAnMfcNtw5Zfsn\\\\\\\/AZTwzvPCzH3zimT1cKzAtGbxlfiPDZ6jGHQYuiiMN0bCq+04oeh0N05V\\\\\\\/CjPTFvUqChfIdx6MFTlcsjdB\\\\\\\/gk0NTKJxhb4KO7zUxklLe8yvm0OKX5eTfYy20fAIA9Wm92ny\\\\\\\/+PExeXeFRnZuA9XgcJDN2EvqTs\\\\\\\/LsfdlH0t+WPQUzwNZ6ye\\\\\\\/r3NYsNZhgvvT5PZJVOCB\\\\\\\/EnJiWMeP0Ir1fIDhcxwv5yfoRvL5yddbYtbrHhhUwu9ojV8KmneXycPHMKh1jfvJMS41pZ7KoaTo0NEJgO2USOAvbV6Zls0MJq4wm4Y7QJoZTc7I9iA16mMQuD4tsuODGXYX+sF33J0ZIvZIEszaN3XV3My3xoRT42UkUPenZFriVxTM80FxShWTGMZUCGM+jzDl2PnO1m5fp6tIHpeQQlte78\\\\\\\/W+e0ebcfnX34OFGW6PFrgZEIfP9ZOasFBw+Nfn+z29hvgknyc+baO0PIlq99KZ\\\\\\\/1MtunMZ2sVC9RMAv8bs1kVSoQL8JTGGkOB32axlYHfj\\\\\\\/aQIDbDrSQU\\\\\\\/Asm+XJvAtiId9\\\\\\\/7kU0TNYFHVA7HqH5ZCBQgE3\\\\\\\/9rOVR+grBg\\\\\\\/dzJGDN8xt\\\\\\\/1Z+\\\\\\\/HDA6YTT5Ym8ygd258wDy1ST8p9Rh6FoeSdCctncrpYLpf4xSfrIaG+kugO5BP2eu9bv052IBfv7GgKNpBTibIBW+LIX3ciXHM7TfViJBguBKtvgYf7Nq\\\\\\\/1mtEcnGSgrXwERnnYO\\\\\\\/0loRXAlq\\\\\\\/FwalWLfWB8uiPSTnGeK06c1tAnFU1AOJdeuZPGiHNifA3\\\\\\\/c1tdvcM3a61jktFCJ4qXrNgJ8XBsyinTSVoeK9JDbldoeIS4KzmVyNbcDz94ugkTd2iEY2e+KvLrbK+Ylq+gMYBZuPOx3zJjQvPwJIAr3ZOCfEeaV86mMU8iJsjfQaBj+EbKM\\\\\\\/8A9q9DFOvMltNz4WqbTnrH92AN8DZplCW3mDXBaZ\\\\\\\/wVb25Ha71Br3tIKZjttE+AD+JhukRvYl10AFLWCNAtmGXMLkkFrbI4wA5TyL44rwjA0\\\\\\\/Yia1fM5vO\\\\\\\/CCoAsKdUuNoyW0An9TfFcrzhq8y1rRqam0XbG2SM+FUhjQiFz5Okruu7BsALvFnjZfvJKYmFtoV\\\\\\\/bb3hwhef3pkK5f3Ub+ei8E2P947weqNDlQXWhLLXLtYUh2bXg8gIjjTZbkG+qTPY0r5nbYxsr5HDJY\\\\\\\/+C90llxBNiLmmpPY3bI5\\\\\\\/oNz+FTZJKaCOabMNQme3LH5yKR51EzqWGe71e+btw0p5CAEV+xsOwlfmxSPF3YNtLbxh3gLS3cAeJdJ+3u\\\\\\\/HVoXoM0X5+x4JyQKFKAdMo\\\\\\\/sBmUB28pag1D99ci48iT1N8etDz83EnTC+aYhq48Z5SpNZi8b2v5cpE8\\\\\\\/wfRTAiy3vyYI\\\\\\\/mlaFJ02kMFs6q27LqVK2WGCJtr2uzBJOgNdVB2L+VfwePOdAtG8id\\\\\\\/\\\\\\\/lgahb9\\\\\\\/FYwMg8+L1rbr4oxfkjhTVgp1xYvskMVv0xLtUjZiw23vjdNo98a97PvnuG0HY8vYvT9sTNM84sxuH\\\\\\\/ti\\\\\\\/NTJWzaoC7ws1tD4hLL89bjzsdx9X92dw7+lww0dx5pxgy6LcdDOjRbd6oNDC58cSbmvklnkCaocgaer5aAphNHEcaFeGSiq4ZdhzhhhNaj+VTnWAtkCTTtxmwT8QObpMPDZTpszFbYue+O0fV8lUrkC+2jsZMuRY3sEVf50b\\\\\\\/vcL6Xd0dJpzzal85MLYElz+ooi2hkSv1L3SHSSrvvzAx1ts4JK1A0Iqflk95NTYtEhUKJMlYtMnHKn4gOSsSTjOIeSL29fWmTW2McFX4ZS6jFdFH6IYIbcOJ7THEjq\\\\\\\/zrOldOqIothwqwh4DerOtrsrPplZPCL8OMs2AW3mzGaAcD5NXqmCPoPyEnrDLUinfns+dpy5N8KCWL1KqmsH\\\\\\\/BtjQgJmwCDXihXoR+LbHkk9lugWVZljbcbZtDXLsYe+Eb7olHz7kQ6h\\\\\\\/zAV9Z3mU0s1iz9JPmqNbUJZI\\\\\\\/cH\\\\\\\/5i26bt0Ud3ikm7ZSfgoyEfHYjIFygKuVkOJmpQQc47LLNJ6MPqus9nNO3+Fp6PdUTwL6brmAgu0IbR1SVGh6WlRZzYxRKgT8mgyaY5z4uCwh1QObPLB4wqidqn51GIWcDcbKPNrmtoYt98EMwU5iyMAL8HDRYWVjksQLUdYttUxxWziBII6grdxYLz\\\\\\\/uU+0ssVUn6Zvj\\\\\\\/yP9lbDEClC0yBQvRDE\\\\\\\/YhZs06I09W0fZk6sBpOakLK1VfFSokxQKQgjgfFzoiPEEzVGBIqzsQhYsWWfJ\\\\\\\/kYMETVLw8mOx6y6pUExf1IcBuULPoe9Ps9n6RtO9596dI3Yk8NRQDVxXK8mJBXpo9MMPIx\\\\\\\/cIxiFXfsC7oGjJP4xbw1BP19CQQSdu6gQ1tWkkZAH1h6pRnXCy6fTmp8eSne5tJ5mhHx3Uccb2OjXb7zgX4in6C3RnyOiBuCR7p92N3zgHRDTb8QA0M4OSOnIGb7\\\\\\\/BcRlciARA8it1D8vzF4mOy3c1usx9pd3+AbcZtbuxsdtdBZKRnnV+Ida\\\\\\\/cyeg3WPXxPi07zu7B8ARnBNJIAUOi+Mvdi60GeSpjnWi7al2G7DlYV8HsejVmrdH3uWpM2wTxICQoz9m+hzcDujjDwDAPvTwmVGPLENscHCYXYnOzOM1IvT1UgjW3g\\\\\\\/++t0Np7ykEBxVwv941ufXyKE6srJo6GEoA97sCO5heLEYEafre00w8K2izYIlfnTDBZAeMd\\\\\\\/DYB8D25Q8RjXLWw43wfyxHjS\\\\\\\/iV3H\\\\\\\/C616iPSS1O6m7\\\\\\\/syt65hiU5SCEhK8aGr47Z5pfP0K2Ohn7Qlq+GOfdGVKOCi5p6dEfFtqIAeCIb7uxLKK8z7N0dz\\\\\\\/hc6XIJcrKp\\\\\\\/vkOhsf7X88rmW6PM6ZyJjgBXuKWS72anvHmTIQgPKqD8zwOrPT1Tqq58\\\\\\\/Wqm5cPRrgXeVaMhWEcaIIlK1HTwGm8VrYH8MUdg+bmnWlI9Rbo41k7tsfGrK16W1bJTRJ+HVHT8\\\\\\\/CUJJrvlFNTaYY6nhGP2kiz1IUWzpUnugd2xvICuxXXQhAeVjqwr5fzG63bVRfkM28ltVNwFmmk0r6iSKUmcAumKqSGMLGzlJmI8dqlV6XIEwZZ80qdryiQw\\\\\\\/IByEjrbAV6luRpIluEuTEtmUanxv+ed+xSzGzlHyX5Zb4iG25Khs52KGeAsXYb9lnZNpHtihlU18EvkqzXsPUas69EDbepd8ZODVT0Lvf6\\\\\\\/\\\\\\\/u2zQrQu1PY+n0bSO1nO9PitRUncP1ZWz1Vh64Bf8uWLyTlscsRi18HJ+QrcnNbFRSJLFeoNmydsm9JbhamOr6h1OMlW5gGMJlS\\\\\\\/uJrI+OWgeOl7FZKIaiDKZ9TQVIeLlokgMqkzAWiUqNetAL6POgqFQIQhqbtpFrBBgT3wf\\\\\\\/VVcsoy7DhRYYhqWIl\\\\\\\/2dQPZ1LsE6KSmWCKzCQbcdSDkEvLauG4cERnSe6p9ifUAsrNSQmWrYaTRW9EW9+mG241eF5LD+HEsOck1j3TKbYBu\\\\\\\/51DYhbU0236\\\\\\\/dWQ064b\\\\\\\/7BfZla3vQL\\\\\\\/gInQann3Rtlq+NsL\\\\\\\/6VTOVujOVyZL7ls3VRTc2JdXVC07qx+Ves1prfm8puYSSzsaVxhTPcC\\\\\\\/OATei1gYNxk9mKFHbubK+tac6F+ycC8GDb\\\\\\\/9lerTJaqGCD5cp1qh93dERqx\\\\\\\/nqJ1wGoSKF+zaUyjUphcbe8bru4IVTvuB3VVowfd0LsJwyIsoMXhqbenS+SH\\\\\\\/J4f1aaYfhBXzWaLAsBrgePPeJXHkYqPywiaX2ZtfDMe8wbvXRAsf085y0vbgBdyd1ya\\\\\\\/Wr5frydgl4+UMPKJOahDyV454rwheMzlGfzKA9ZDkMk1nt9Fx\\\\\\\/TyqEeMcXZ5FVNGqsTePOjMVRZG5Ut5duXz0ICdWQl3J8Me6hLmE5H2q4pWI4\\\\\\\/hnr418G5Dam1TaNcksAZ6lP4bc+1ExQuz2fuWQYSQIUlvV+lTYD7+qWzs7weOwcy3xkFvXWM\\\\\\\/2DdO57DC2ifAJ8z+j+eGRDQpqKbD7k\\\\\\\/X2qxNs+2LabNHpA3Ql3H9Fwfm\\\\\\\/qZ8Mr3CZP2+pPx51B2gLKEyf2QkieVLE6eS7ksGc44\\\\\\\/nsiq+DE4uVacW4tZgcBW1rrzFO6mzmbzFRj1THxuYuuGS0X8qmavCZYwRZHT55z1dOAk+fAkwnT+x9k6pIUfZ916y8XuC8HwSyM\\\\\\\/EKhNZ\\\\\\\/llCZkiLfIrA2YicKuDY0o5kZ9xiRXRHBntrqa\\\\\\\/XUZeeN9Eojd2E3I10nT\\\\\\\/MVKUPjWVcZ8Tv2RTHYgnqAnnHqeoMR0Pu2ulsLyYBvfZJOxQdr1KywvPWBKMxY2hjC6OGIDFKRPzoeSAwTEM4q9NtKG4mNVbAcJSJt5WUcQsaAkMR68\\\\\\\/6nidNwNyVMoDxpDRRXRHzKHG5tnkBc6LaZZ3ErjsAu4GuQn6VEL3bZhTfnrA75mHaNA94wWK1qXCzoTt9MgjTmoKVf6lIS\\\\\\\/hYJ8+vB9paTADhwZchI+CuJl2qUBAIce8T8yybasVZtuEpDrtMI7BLS8LOoZUeFwWOTa7tGmhFS3CrAF9I6qfUOjM825WGVn8\\\\\\\/I3c1ETIIGfjXUHToTZnohZHGTyPc31ZGSEpAqwuOO9LAkdh+KIJ94CmJ1dMfKav\\\\\\\/0KT4PRVpuuMyHub42Y7e7gvO51LppDNqCGxtiC11m3sP8W9tjbuPAXw3JLZpzn5Ja5kxODQ5BA\\\\\\\/ydNA80v+ZUJbMU\\\\\\\/b3Dm3POFot0UHtY8ylz5E5WvgYUXNGAGI06BZJPJhWcN\\\\\\\/54lkw2pwRVRIFAcXftHoMr9LiJFXOqadqd4RgWc\\\\\\\/70LS4XXRrdFotKyGKhaZ\\\\\\\/V35nxsGKJ9a\\\\\\\/5rDkQ0PJ+J+X\\\\\\\/bjArOLp9yhnJsibHzftAEpf6O7W5Cs2yIjWHAO\\\\\\\/AKpMFyvG\\\\\\\/UJEgKKGnb1KERJBQYkznNH3yI10YPa8gwJJSc4nRgxFTkKLvdBverYbIrU9tMzLurBOgkw12deFh3+pY\\\\\\\/uInrsqAYnJHnRfnvVghARcssst719vbtXhKfuGNnhzjOGynLHBeiY6HvW+UmxBiNYnzCp0D1+49NcgDfhXXnjhEB2Ww9uLy5FIcbxyw3OHWQC8cghkfCs6osYOFeVcxqKyzrOKmq+JHOLp7fnqInPh6XH5cyhGEp0pmhdVqsaRJpb9o0ocUaw+ISqS3OqCrIH0N3HC5PrAAeDgIRWUhOeBUprUzPlmo93ztFMhtVgcVnX+Uxnjsmh0EEcbdCLJ3cNMvIxOTFoEZYXT5CYzM9oK1rrf8V1R8ln272Dn9dFzTrzQk5NnK9fK2s2y7DWl7fI7SHD2CmI4BW03EZi76zGgCB45\\\\\\\/dm3TjxS8EJVT4+xE\\\\\\\/mgTHbk9xt+bcxkR4XVOfr06ZGXJRarWE1nKyCe\\\\\\\/GMeKvy9HUr5RDfZydxdzBTTXrvNekDtjvuPOz7fP0ae2G7cGL4rBL8qLOxTg82CbjxBTopcMx2SY2C1rKq7Ex3xv5V2\\\\\\\/cmvyo40UUSJQ3TVP8RDQ4l4zZB91hikEwRK0VGGIaBSxTgnrLGR6FRq6yrHfua3ZMWpc+JpcYmw\\\\\\\/WAEat2oH5iaOpuag7LSRhxxWIhpQ7KN2yh\\\\\\\/y1wJJIIUD9opSdMSTjWJBjiz+NTng\\\\\\\/SKZfnhAy96rzU5hYzYdMJ\\\\\\\/6Y+G7ZPmeFgumJmhMDQ8UZ048C+Bye+\\\\\\\/p7MaSIJVZJ9NarJmKh6AJfCIlL+ncleDlhgwHTZvjCscMSrMo7wXJnu6h6ISmY1TYYuRXlJXznWqleQk6ic3ULytsgKY8eJXaSH5jEZj9Sv49ageF\\\\\\\/9QzLa+YTBhb1iwzNftMBSKsGyF4Q23CcxLInp6as+6uX+g7UdoAFeT41y0wW8IhKkJ1rGQDY\\\\\\\/G97p55+XG2EvNoPvsgDuuY2Ae45OJ5iMII8uS6sobkg9lJOglb2QZOMvS3C\\\\\\\/\\\\\\\/+MXnGkQ55kIBj9omA0q9mMy1begj8M+knrOfjnQWv\\\\\\\/KQRqNaiHlZx3qpdj4Um9XJjWyufd3NWtJiULP6rGDXeYufDSmHIz7iHkSjGoIWuoAehZ1o7CCtuPJkca7obkKYP2R\\\\\\\/Bpg6Gm2Fdyf+\\\\\\\/qe2Zxo4woC5twMrA9c8a\\\\\\\/FFSQYQU6wSTg7vnzT\\\\\\\/1woliqClbZIskTZnhGbs9Ac2KFl9\\\\\\\/a2776T8jLqPL4bDv8VY5n7w2icVJX7UmuFLH+qcQxbA7j6j9Bb\\\\\\\/3jZKH0MEewmFBIGT7BndyeL9lVgBL+OV7KXO2tZvd0Zzg3+c92rXorPkjUx0GLfw+zq8Nswnzxf0TB26kmNwjGi6hULko2eDkP+h+jMYWxDrTRHEHTLBhc+rzdkO7mCS\\\\\\\/LvV2\\\\\\\/T6i8v6KUGidFedACxYlCMzLMwiN87lf+YS1T8cVtRJlBfANQ4ZgbkrZCTR78TejhWBRz\\\\\\\/lgL0qJMzHdD2xjrJuBU+PIB\\\\\\\/NVDFDYJy7qAwO7uPdrEEYXrG3mX65ITN4zVXGq4t4C\\\\\\\/h7whIXDseqU3uHfZZbuprKT+3TDnf4XQ5MWBKJgVSW2Jrhxp3JooeAb67D+y2VOdbkaswyNOj4UMTBDonbRc7o0uI6QERBq9RALXx4h6ioxAelaE\\\\\\\/yHmPrBSJCho2C+2YZofviNS0C0a6htPFQQvv2bqcH4cdUTU6uCKnlldeuV1FJbtpNKj21lKhULDOKaaZ3JFG87+XN+8eqXOkpL1LQHI+EK8fOghh88kyJu6g+mEOZXqzGRYoTSmvmH3AXIJJlWPTvQ4Mgv19su5mk9BndPE23O\\\\\\\/Gf2fbei5F4OZL5q4wCiS2d+vA66oZgxIje4\\\\\\\/IQUzjXqbS3B5n5OLzQ6acFuPI2KzA51r4EFd1EStVRkP0FIMdNotp50GKsZCZJMVDbOZOKqtDBS4XN8dnXaDTLc50wMk\\\\\\\/+LLelH7IJKUtSiGaLBqRPmqGrBqcMqxMRlC9oLxGF1ttrpO8nMVJf9LAnRH4AeRvhe3rx4YuKUB6QUgiSKcMabv8vPSPmUmxFRwViUO71Yu1Db8n4kMoYnPXplUpOZKgqUiZoi\\\\\\\/p70HFUUuOC2BzIHfq+JIzGJ2nMcjktTo95TOlMqhKnJkzuBrh2rKig1E23d9tF5bup6JIsjY4OsoP6gvooEpLVwkIsJATnCDgmvPo9fF2zgNupFpAaSv6KDQ3+n3L6OBZmbqXeaJf3nw4JMMlKxQlS8Rsurjh90FUqD\\\\\\\/VD7zVO55F4pt0C4MO7tSBmtF4OrVlYiVg4vv0R7sUi7Hif23JDxQrIREubXBczsjpIYnGbgZndjBE+s050wBmGySKAZi5bTJFAxra4tFv1VDTRhVmsTIrORx25R7cmggS94tFh8Rfwom+\\\\\\\/zw5SuTLLS3FSIfrcT7rCZfMYQvCu6HJoqJaMxu+py4QSrKP6eqAiklipcgkKlTPYbw2cP7R+eFN4YYvhObAMiku0FHW6hSzZvooFlqf9GaO2FieP+Js2nuT\\\\\\\/JEKDuWYrdiDZFucFNAML7rTwGAMHy6lMxL1MQ4wCWLzAg7CfjJs2njnmWg8CQWXRE\\\\\\\/akqxhchBE5WrZ1R6tpvYDs2ODCAgvPzipgqzM8lIinaU6+UCxDIfe9ugTdpajYxARafhXqoYyzRw0ARNCIeGjInNhJky532Lj9qtLP4WfTpzb1hTWO7bnn2gGbT3PqovSTaM9N3GouYjTrjPJ7TdZv90foWFX9L9cplKrEPA25kLz7oaPlzMujlUwe7dYWKmKpkT5N7Glam24HiLz4wWzEmF3zmoPhjy0d80JzHgdS3cTq6dlQd2tizIpeZx9mtHuCtBISNfRUZR+PM8KREjKKsrOu+Qa2HwG1exd0IaGHInwMMEsPZBO96ruknGOoQJAQA2gzjcg1sewzqpIcvUGgOw4zPW2Nz+Kbr8WyeJOdrRSDqAsW1k8OK3FV\\\\\\\/AxcsKLqyRt3hZ309NsT9Gpg1wNBuucgRV6AkM\\\\\\\/++GyF33R5KrrhjY7W4pDh6DwR8fLJPYLG+dW0Hz79\\\\\\\/6LQR2zzR47+1AECp6vnvM1CALLWzc5JKZAF6v0fG86f44JcjsB0P\\\\\\\/dQTDjjNBSZpAjs2r1aVPqo5h02vpN4Pijv0VU5V3qbj8369\\\\\\\/ll7cfvJp0EsfpSgNCSU+TA5mj5J+BbJAmXwcmMFrY404FYFSZ+NcpXSt50BYahUPd3St6CHt5uqg9VsU39MTb2iaTnwi442peZUVRsfB4yjL4zJBcpc5NPdpuIjBJdaygoCMujbQ5i6qlAZ0vliDv5IGizV1fcvKnh9ib8bwLQ8BI\\\\\\\/U+n46yilh0K0jbr+6JZXmqFnZK+EYxPnUEYXIS8wLSv0vtLaSUsoAehX9XTHaJMx6LrN4oKMqv\\\\\\\/9G9QniyQzq0eiIiL\\\\\\\/4F0N4C2OoqJwEgyLVsVuQwG1\\\\\\\/RaeNJGUpLaaeT3YtgBvN2z2QJi0khDYWj\\\\\\\/6BYUnw18y9ROGfJiRr37h32rq06vk8NbfQfhqhkuOrqisBFUxhfw9ECUwdILwpXSuu0FnlZEuS9jDh6eG6vYfBxoU+UGy1hCTrd7DH\\\\\\\/Qfmk55fezLAjpUnOtGRYGtIjk6xQaXSKXhjKBoZxM7CM5TvTYqLlwS+CI9vdUjFuJ4TSy23AMLGofKxhhh8CKxt55Atr8AEsJ894BqjXlwwIQNhrN0dhuBK7v7xdepQyL9D7\\\\\\\/UDmebsT6n0b9mAWcrm4dQ+vPCNJtmkz2H\\\\\\\/3ik3Xi0PFWT2JSZQWMOumImwNFdE9ytb5+8i9umYnibtnYM9X3YEFK3\\\\\\\/MkQp+C4GIzfX\\\\\\\/E95Hbt0LjTXCAq4j3jnxXHYdT7AkSJXT12Pkxbgu94FpyaXEWkDgbBtyFoWwWmSOTqpfBZGSYIJRIVoEPbUiPzUWKjh7rx9dpSDEr6IBON6kaXT3\\\\\\\/dNBWqbZxAcRqNitnQ5b9Mg0Z5AAKFQ+1Y9+Tq37hqgzWTs3x88cE6Ih6i\\\\\\\/ILN1oW8FP+Z75EFKkBussQDDRARvuYXYNNHQcJSdsr2aQNp2A3OR9iOqK1IzyUQGd\\\\\\\/ZFNhLEfsWhRtjohawJvmJyzVO4HYHSa4iyI6WBw3fWIFHPJrzNqAf1j0oiuWS4GUmwdbmEVDqycfmFi+VHs1HvncE1BSurWYhkc3ePeof9JqGal\\\\\\\/hvGCCl8fY\\\\\\\/WZcVNDZXqnotEpTMDgIyoRiJSZ4DFKRBZ6\\\\\\\/w6eS8ti29K\\\\\\\/UhyCtBSmMgX+zVu2ecVK5xhxlIwRJeRAUDp\\\\\\\/lUpGmzPPxB5gx8XCJOztXSZDO8o0rfk8BIKDHU728ZBhAbF96T0YQP0x0QvCJRJMUwg26YsQJTqlPPO\\\\\\\/NnPuwKtHU+38JDDNMsHdUNfGqoi4I7V7IrmxAqZrqJzQep90x5Akj8Dmm\\\\\\\/8ovx0lc7iLRWcoGZrDvpKsGAJIuPfMnE4JgsHgTbjWEzQ4+hA3wXWbql8cT3wJ9mZaujotA4zeWTcgnPXsBa+IgpFsOvYC6nzpbGXDKUrgx77tLw+9JHiaPt1jbHIpFTQ3i1hSUsHas3gQ9cpmnVAh2ZswKorB4qMfzm9g+oZY6Azn9gCY\\\\\\\/21+KnYgCa95wzWBiJ4WEsUPTydF7JjjEpDFbS2gqdPoN+Nh\\\\\\\/l3wySiA0nnbw2TdZ5mF6mFClkML6UxmPgNPnAgb8QooPFtckKdvqQ+2pyN2dyDazZ7qNez2zXl0Za+snyRzBjAfgRrzi3qJJOqCo1yiP1pD9eYm971pOIBhhVnfxNcO56cGdjNuspDGorBVwi2wZeY0i3BTwbPcZ4HNUuhvf2MwLeRG\\\\\\\/qN9O2uUOYno31ZjuVM\\\\\\\/qQqOThfMHJB7Z8JU+8pF0uVmKeVgRXwfZc9xUwYc0\\\\\\\/H0fqCOGdGht1brHUExsZKz14gTpH5dX3Lux8Zf9fB6yFkhsz2yBmt6UlGT9XwEmbOI2KO\\\\\\\/24yG6DVwQUCVkJx4Xq4Isic\\\\\\\/EHd9CfiSzJO0IQKO6Q2o\\\\\\\/cc4wSwmiDRDQlzDxoZKNVE+TqWQlguTp1zzLAVjaoiPioTMdeY0zk7wkvZauk3riW1pCmyYSIYxR1cGaxKx8i+H+Y5ODArrat8qqdOtOEWJGBUUWlv6MjXempcF19ZVQPq3HfYDmUQsSUs+M\\\\\\\/aGKOtJhBkyvBWaNE1laiFOfYF1mfk6zk\\\\\\\/OnuM1I30qCVBMglLshfPaLFXR9B5RNO2x3+u+ElVP\\\\\\\/lSYtPyHprZms9V8XykGuQJo8LD1x0cm9ZKzt6u4U5avAIMqAhSaxk69k+f7stUeY2lSN3g9v\\\\\\\/1JnFTyNjF5cjiPU5zXXUfnWJphmbal6i2Oh\\\\\\\/uoj74LkX07A8OmKZjacEk05BM2VSt5QmWZKzncRWaJbOuNLgPm4L\\\\\\\/2ZR2\\\\\\\/q49c2ljIjPu210YzOrWxnvQeeJCMR72b\\\\\\\/LQXRfXnsAG3ywfFJtcWDQU5kyqGQjdwFVLq\\\\\\\/T0E3WTXOazcvNjlEK1ZlzVwJKrrG17H8r\\\\\\\/nu4XQoAiJK9mWO5I1s5UaDYMmB+sKQ38mLOys0EMqopqBnkPpIXZENRT5PLbcbARaUEM\\\\\\\/qeF9wDVV\\\\\\\/d43wiRF05kfxNzmzwhN7jIkKdcqbuaUb8zJUsEP2VEgVSWtSJU6msH9zvSzdWHqkJAGnaepzpxMNfnqUXToRpUJ6cpg23vE2WkNg6K7Iv6D1GindNKMOGsFsKdK7H5UFrK0h6eIjY8+KhHGVdC2KNER6nehegE2Y0dtGquxGszjCMqrHblLL4V+XtEpF4EA4IK7zvhHt4cEl5amZKpEh0nDDJZzzP9Kkj1YjsNBPfkKmwa3J1I18YxBk+CZPRjm+g1AvsRoO07njoaznA\\\\\\\/0Zd9QWa6cwtf1KZ4xZnD6WyVgGys+7OodWLei4uW\\\\\\\/NPFTONhU5XKE6XFVGXPxv72Ck\\\\\\\/SGZ5tXH7kaJnRwxk5LusVCK5mSJoBX59r3GSuvfzGzF+Dp2nEqN6ff83i01Jw+Q3Unt9LIREZ2oq8WqomUv0r4q8a65YzxLSlZYJC\\\\\\\/VP5ThLSo\\\\\\\/sEiTRW5XGLdOF4hPY25OaU4KRCorwQihIEgBelWjIp7Zc5\\\\\\\/jBuUNrQJeaAUj\\\\\\\/V0eNfdwhIk2uW4Wey6jPsona9A3KL+ySM5+l\\\\\\\/7xCGxodWl+iowP6a\\\\\\\/ugBlq5iwQWLJ1fubL3A4FRAnqpQYko\\\\\\\/N1mT3J+xHNn1qkm\\\\\\\/eFYmJN2\\\\\\\/N9k7vsec\\\\\\\/bQzrU2Y9EhlScKXl1hJIi5JoiUfIrOspxgmwIHwqiSJfmd69vnNujEu1GQMxN72qOSfxLflXzK5XbNt7vS9Jpyzf863bYZ71f1ivaMJeFRcvO85OgFF\\\\\\\/ag\\\\\\\/sPpYgQxtuHNQccMg6Zu40v6t+gz853ooBn8aHME41\\\\\\\/oEa\\\\\\\/Fik+xvClX9RN6BSP9sOqjTpNDt4y1XCJEzGUc81XuQX7+L8TXVg21tisd4aDQaqh08JT+Ng3Lvo8Yv9KoxQdnXJjYjWbHW8E4laS3N4zkHaDYfNeVZPMrKJDcjJVJlaH346KlwA9qbKUVit6bnRqciDJdTfN3mlXMQLU1n\\\\\\\/dZ7PnvmPukRFDS6RkNhSyWABezjz4ScxCFtHAYisBWVAFQusMlysKQ2TwPvS1tsoOtizvfXBja7xYXxahkQy+GnixIXXSgck6hxCp5Qvx7SHOn6OQRVL2N+kFHDQNuC3+3d5Vp5dV0NLQMq1Xl6ZbKZPJmAlLrtUsWLYOurkZ0cpVV7DRiez0ODl0jfNo4GZKk3xR4lmxdJs3BA50BNAkWsee9Lh42PfpOQ86RpzhYOARiP3vDqfq\\\\\\\/s0\\\\\\\/0qA+GWH+Fber20VcA+rG4ImyDwmT+LGUGK36daLtHeK1WTV1cSQ14HcXRNJlGNiYSvLB9udPZdZOCVPTL6EsQObLl8N+9qt3PnQRCjnKt6eqGyTr2yja7iTCvgmFiFJLzdB1j87dswd32uO1qMDT8W6nwU9eHa932V+NKw2feGUEDPB1Up+q85dP4xZ+HAskDBt3TdJAMteb3o92X9tmyjCxs\\\\\\\/9KY0Fm1zwuQR9Usd2mTOcFn8utvmHOmAJ3QMK4xaErrwmjBcL02XZ8iuiYSQKVT5JH6SkdiFFs7cvtUQ80LSIzIcyCiCisRsgMrtSYcCgKXg+s+4S8RZr2lek7bPZYWK5oywYCMJ8VilNagHN4Ry68UASA2hukxCB7Aq4rcxM7PqYSOdemSBonCzr1mNsYZKZG5VTraiQHPvNuDCzVrJ79jmjwDFOWLZLse1rvauJDycMx3K6Efk49XYCReyPkpkN+IJw9MvBlxoMbwc77V9rgllzz9cryCrX7sBdXoaUEZDd0qzlJ9tlo8HLOtN3UaNU4X0XopbrHFQwkf3\\\\\\\/stxI2zTWypxYzv9Am+7ZnR5cnSPN3IoCToSqhIYo2Dud0vlu8tJkhwLgRbWKZ5w4ONRcj+dCgTMs8hw5s80GOm6ltuEIHIAtRCnBUbWPTX9A4it66\\\\\\\/FyFttSGfpBUafrwJkqPaYs3ZLyR8alF+U89cayKNsy1vfGL0yCL5tAIMKHAlh+kghyiJcy52YtI4Oyga+JT5KT5JUvkEzBLGsXLhESSTaviAjpenuiILx1O29YvnNPzwGSeBeP+0nXPKt4mttmZ+9BDWUuItnnYx76FGgKrc4pj+UVh6seb9vFNKD3MMGO+kKCC4ol6ahAk8Yqggykp6tm2F8qic8U\\\\\\\/gBsuqvuyQb2Y6bESBjjgW2Wm\\\\\\\/PmlWnoZe3LUcXVQUuNgjMIx8OEWCVwmQoshXZh5vV3Pl7hA94DbTIWOXjlKHMUEdCrf4K5xr3AeQX0jw\\\\\\\/cunm42Q0WGI1rGYDzXWns7nrGrzSbZj0K\\\\\\\/PoIuwh0BcgzRQnNad8m+g3qjJ9GPmUqcGhSXOx4+9TTXodzXS+npN5u7su6JKWGw8lCdAptzy91KAqgwQhJDU9FAAATVkslBkL5JpsnfsFSBs61Pcx91nMf0ENecYDbFi3wiQP4FimDXVJiPRVI+zfxzLQ\\\\\\\/RAnQuRrA6lhAiMjmIv2Tt6GJMskpHJtZhu4XJ9F7+XWGise7UqGO+UZy4oRmDEUNXnuqkAt5erarj57Quo7gvs7vvvdVBsYt+lWnND3cePWwT39gyrA7alsYnSjYFUC3xpasYoQae3AhiEcYL\\\\\\\/\\\\\\\/H5\\\\\\\/EhwqkcbYUWMGTe5KtjFJNM8SMj4ujI7Nhscz9ua20DFxbEdFrU0jWiElee0JJzRalJakgV5bJGmizcHG4srBAL9ihe+SCpqxUgKIulXMApX4tAtxrHdyeuSWt6c4Iq\\\\\\\/5TXhsqbJgw8yYJe46ILTT3U3gxLvtsealYCWOmfsBUDci9uR2XHXyMUlZytHUDE+hW0hqLMavHJ84UNSpNQUwJBRfxTxr78jr2K8lb5wdfEoHyfjzP3kzf49MJ\\\\\\\/rXH4lc20kLEh2X\\\\\\\/ozSUxcTqDCdm3QIk3PjwCrmlYTNlzjCO63OQO+TBQX7j\\\\\\\/T6tmUaDDNeOa3HeE3z0FLSgCUC7NJnAC41EJFvH0tr+LQ3WTsv+rsYDTs3XJcAWvRWExFg6osTa+4jEvHTRhlaFS4m5CIvv5OTYJkwvchiE4ZHXPdUHQh1vbGqC15cMwSsAhRMDVWGDect969SmNqUOYESeUOdwHSCQD\\\\\\\/oIbuI53Fbs0jJ9crEsueC0fd7wl4S4rs4TzplYkyuZjvqk7yTuayQudpDQgt5XwIQlWgBywuDIPrfNneURH0fZ00sAuakBr5Llnep6LF6aJFzO6LTTAnxf6Y7KSEV3YLRgsV+IcCeNvWqI4FNSbIhwt7NDWmEffCL509e8p78dnRJ80aMUmhwlducZpjlzAehSF0bePDNp9YliunXphFF56kP7Yi+TUP\\\\\\\/BjERysOuXTKin4TiM4UUuaNKz+BHi4qrhGFVJ64XEd30EHQILyJxq\\\\\\\/DVhm0a\\\\\\\/WAMZpYM5imuDQ22acyJAvKSlu5wbH0md\\\\\\\/XKdllYaJ8Rhn\\\\\\\/NfNT\\\\\\\/48mKscWUlcbhKW9mGm8KT6Ad+FqqZlcTE7esl7my5HBfManULRp2AReV4UoeSaj1RXSmnpoMoMx\\\\\\\/z31LcnOF55c93Yy1B9292pGhmhHi8Rd4R30ghzwB4aLnT2WEf\\\\\\\/Yhvu3c47Qs0MzKcCwkJMNtnRS6mlq5Mq3CQxnKbe9vW1uKI5SB3x0qHO052gpXpGF9kd1NNgCIWAeYjvCH1cbhd0MuldDfRStNSRaJjTwTNfhwrlCHNbvvzKjMOr4SQZGwyXaZmP34cPjiZZSvDqSJKxAUjxWro2H+vCK2ObVmOdVjHxNwAKOcD8uqQCCq9ndhAiyuGY0OPy2q69J4L6I23tOV1\\\\\\\/jq9s44quI5nBrxr1JWukFBbSizkWJRuZ49O5vCnqxO6H4mj5JeKuidnva\\\\\\\/lb9otFAJGU3T6LrSAP1W8JEvNqGCYUySLDL8w0SMgyv98+x1+XehpJzl1p7Geb7FUO6Hl+kNG7mMWFJ73JsxUjRKOtJ7pPxLJnZTSBMwZ7OOyFAFa1imOXHMwEEPj5nmysEKxmEEhKMNbHQgnbhyd1lOmfLCDoO\\\\\\\/7cDZjd9uFS13FxBCCoqMp5fk04CDsgX\\\\\\\/hiN\\\\\\\/b8FrHEVifO8q52GRN59zFoT3sKoY\\\\\\\/GIj5ArXmDD6SElBK6AyAU2\\\\\\\/rUpTooy6WblRgEkxXBa2zGdzB80R8MZnUPYSVyXgiSxqkbhGDM93xn9y0iTSlZ\\\\\\\/lS3CDIIi5vQFAq6RCUiMqLqh+JzZrrjb4dNdQqZ+msFo7qSvhwZov2CQBfrYZZCw2Bgxg5KRoHTAJrXZWtP7eZpWg7zAkTPQ2FVN\\\\\\\/TdsT2aX5KKF1MhT5b2kYk48L0mvF963PDwnZOGTc5hOJ2uPKmS98jcWDcrBlz+9i3N23eMW8vVTTJXoyxUPQQ1E0O2EEcWd4UWZHV1gBiADLW2TGNqkIah3vBDc5EbwrhAO9WwoBeKYCl4wg0CUDiKLqLoe15AUNcqJbWEPiEv\\\\\\\/zFjENaCv\\\\\\\/hMKNGStm8eOMTf8Uz7mCObY1C0b+GMoh+L86BCI6qOHmX3nNFXeQP6soLVgP3WMGWDx7\\\\\\\/f3StkDK8HZjkH7iF8OuhxrodFlvB49pb40MGGUArtp5i90g6CFNe8HanqXoE1Y4BBDvlgCP9KSv8edLO165mBhmvqiQxtzwLgo+HpnJf9N390ZLQtTSTaA2h644HXo6oQrdINESG+I5kH761U3GyE2NviIity\\\\\\\/isxmQgd57blqPSknyWsSCpJzklveaxe5vc5WcaJ6TDkIXLVqU+cEjw3HBLg67dVLbjUcFb7r9H96IWtPBHT9my86Xu1ovcWNiBtox9dmSmISiusv\\\\\\\/aBubDF6DhUNFbK7nBS9DyOSESZL3M9PmgsvnjH0SkWZZhwvDh15TX49GgYmXsNgufJ3I87ADsesKH2US0pioW5YsTIu6lSWtzjItX1wh4RRkhQ\\\\\\\/e+AJ3xigg4SXP0KptGjX6WuZL6eT1o69TjxotxewsHDmex3FdRU5Utp+GVmssYmeEIofh9ylOmBn8husBzCfDpriP\\\\\\\/3R8Q2JshU2T+hgUlSv22pOmp8hHtnZp8CZviVYi3TOKTxjJdgits8lzSsV4xPjEkGUAbm6xaph12tvzwz6z9JiVPNgJHNMnWlobpa6Ne\\\\\\\/yxCG4jXDy4V8pmuWdCxxGJaIl5neAvmmmazWINH3XphDk73nhi+dSfp\\\\\\\/xsYGx5vOKEOY0D890dYdj8G3bJkCvqK+j0r8PLNGf775XSFrQSDJuOYre2qcF4R2rKoKGlFBUbFhkg2LpP7JmIfo9ejMf841W1Or+oo9TZAq\\\\\\\/NrM7UKoBa0CURY4pu0Vif6zVQVKxFa3iIlfXNBmi6nL1GXceyG7C7nrmx1if2sx8Yase4ljiNEgxBKoNrR4yZ1Ox0sOY8YHEan1lYKI+w\\\\\\\/NNvhPpAZ\\\\\\\/RWvT+RYs2Q71Gn6FOwUy9OZJB4RFW6guhBQ0+43VcFLPvbYjDEurq57yMOco6e++xQVM0va0B5x3Bwy+a4Yz3zaosbv9a81pRoAi1Z1LuVro6Xi1WETueJpmRaxit2ivmF9QoNM4IU+p5KEFFSqm+zDRL5yc70RjA0NYAnApjZjXB+1a1XPsXCqgYWVirX2CiZBTDWTq36Y+tpIjM17GSt36vjyJ0qyopWxOXsMEw79rxWRgGwEHazZdbx8btp4qLQONC1QyRTlm\\\\\\\/R\\\\\\\/UuH8d\\\\\\\/624buCI\\\\\\\/Tk6OJgBQMes8FEZC83nEYsw5YHjlKR\\\\\\\/p4c6hl8sdTQax5A+KGJSHHe9w3EQcPEkpbrUDOlPJfEUDbC2eN\\\\\\\/6LhPugiidCMwEbpg41v8FtwmkyYun0ybbaLtWrPIF8JUHUsI3Y7dmaauM2t5JBdJ8ZyOxHw91+JhVc\\\\\\\/+Wve0J7pTBLjpZ0dmuDLTOzjcl+FQYIRd44O4YfDPmipOCPuAOES+6BY+ZncNBTdRezoGtwOAWwEsfC1zfuQaNvyMJwxOUgz3UYt91nt1nQavxXH6CRMztHub+57oKtboD+cPGI2DB0viuHH6NZUZ922C3c1NXo\\\\\\\/xTFUY5cRYfqKVzaL+NP2JkDCddo+F\\\\\\\/zwVGoZcKLTGYktHYxHTLt+W3ecEs5pUPTsqVuN61z046PBvkQQ7vgTVNhguC4bt7F+hnkSrywz2+joC0AIA\\\\\\\/6fjl+R+4zPyaTHnUnAdpqGPc95dQH5VAvHrOLaft77LU6VbAY7ZcNZtwG7IfusA\\\\\\\/62y77imMatBYmu8OvOS6RV5lont96pMsVdJFQDzk5g\\\\\\\/aAM\\\\\\\/d73RjGDEk4pPG81zE\\\\\\\/HlawZ8y69y8rYen9PHL7fvgsH1JzsaUCj5uinEGpt3OFMbeH2t58uP6kNC6DtzdG2pPufUG7ZtuhHWNbSgeqOzjJ3Tm8+lp3xWc9e6w1pVD6rHCcbxTstUSbOuR5GCCH5rmSXPnyhg36vypK6J6AY4o4Ywle9YiDrz4ZFzl251t\\\\\\\/J5b359fQ6vQbBz583GyW3\\\\\\\/OoJMl1++Pvaccw6QRxTWT4ZZiUAWRl5IJdjvRCFpG7tdMlN6uxJ0\\\\\\\/1OMnAtFKYks1j5p\\\\\\\/1lJhkZeIaFgL1dhv+55MnNagNvqgBkWe9h2cGLSWcArYgsjTaZy99Joe+9G3voVpQIskSrZExo0HSlmt2+affR9MzRGmJznJ\\\\\\\/kQBYmcp+JKY\\\\\\\/CRUJs1UaFlwgF03NRqNoIZDe31HKnaEc\\\\\\\/EUGO0e+bY45XUEec\\\\\\\/lVsZj8mQhS+rq\\\\\\\/B+HInM\\\\\\\/SBgV1eW2CVLIrykPHn0QwDsTFOEJMjf3OW766VSc75NL0ksxliTswviP\\\\\\\/sojE3601cjILKxP6mNJmTInrLC0lIvMiT\\\\\\\/OUreQQWk5zzubP3u3ws6SNGKNhjH15nYbRumL7uK9eiVn39eK1Tbv9bfAccn+4HTwm8047IpWxD0OoOvTUkvKypSfZQ5KL7woI+QDizutNM5aymZl3GvC2StUtU+7G50NO0AE97n35b7aUNwvTL7dy3iXB93LqG4tTiRI9DdjcYf1VIkCqlIzgH33ExNqbsLAtI7\\\\\\\/K3kv3E7uF+BbH4db1POGfaarT\\\\\\\/TTyDEx4Ct\\\\\\\/eeZZEJc2SogZTJOd8iBFgCDMhj2gnJD8XRmZfGLJ5Br8XszjqfqFSafHB8V0\\\\\\\/H7EetPzDDV+5zOhLVmuepxI5MUBJ\\\\\\\/0NAfUnWWZVEktYrg3dP5RTYbLr7pKx\\\\\\\/87pQbAXWCvy+nZxDqmnbLBT9SIC\\\\\\\/EpSl0B2oXl7A96b8DMskVYy9CgObPxf86ai+LpIgT1xoqBif1sayOF0P7arkrp3nTFYs0lGwtllPM0qtlPo62M1zZz1CHwxl2yO3lPqC58dv5Qfj6\\\\\\\/jRPQARF8CHJsZDzmZOEU9dJXvrTjKPfHLmahsyUB66VjXLnwEjy\\\\\\\/zfxRGm6FOusaCEAz+8OnNH5xEcnQhkiUkEIyLgKCJbDbOm804V18zwLEcOP72mNzhFWoqeY7Cc3zY+cRX2FOUTFc0FCpXdvNJiOsWlVp9g67833SSyJjvaD9CrLzVd0Cwd08fInqNPrsXTlph+2tRFoC\\\\\\\/oJ8QM\\\\\\\/ZTQeieDW3CRNhMhH9HQfP9cwhchMbUiIbWXFhHY4inuuwr22gqPhXzwleGbeYLn9c\\\\\\\/bZ1la72X4POghtGGSpc7A67azzMs9SMbMMG39r2gvyWRn+bHJ1AkWQEtxC9COQHuU0SqAma+YMhjiKIPDGWQ9ixLfvyUxrk7vdpbcIvmhwCDUCVaEYPdhsFaDDOXu4ws4oI6lON8wmsoJgLsDbOZ6WWb3FQWP69Uh6GDcl37wobS36llHIiUAuUWQmSzbHhf5btPB8ksxk4tMr6vl6gDYtNZWaoO6hilNV6xg5IGaQn3aEMq5c+EGHQwHxVGZhYHuahZdUp6fbxbcgZ7wWQ3g3PDzOeX9XWwyB47xJP12rX+KiMG68PFGN0bJlPbxQkF3xXhscqSI3doGot66qRTxeEWm2Z+pHvg2PkeoEkJEjvC8yKNKZj1jcHAcpG4dR0D7Km9XwB6YTBGlAK1F0o+rpZmQBmbZsxxMCt1sVJVaNLgTJzZXOS4rdiGgWjvtL2i8S+YFU4uPNflpbwmwHRXq++OTYy0jfBrsG70pHiVbOiArD18UmXrronE9gwfhj8FDGBCvwfRvlRRRrijO0sszeon3VjtZK7sWUYIQl\\\\\\\/qrJaWRWoAVqslKU1I2\\\\\\\/yaK\\\\\\\/HmC3fMvn2BGbp\\\\\\\/AZpRb94+yDzxBXPxkTLReCExeZnvY16TQ9Kpa\\\\\\\/8zTVJP9i7nqxUVrgZKfio9Z95bR2hTqfziAobjvcEDv1vsYzY\\\\\\\/A2g4FPtG8tcOb0xB0NGGs9KiPDBY06L\\\\\\\/vialW9YP0D+\\\\\\\/eSD7SDCS4Sy1XuyXuhkpwCSmxLa1TWjr58Q21efV9MJql6VJoMYRUPflo0InISZPXzq+\\\\\\\/OjhzkBvmsV4611A9ClYVQ0ZbSs\\\\\\\/JBAuxeD27uQw+fmKidrC+Ew061SMp+Dz78Tdc2FX3E\\\\\\\/uZ+TOuCLW8KP5MmByIUI3AaVDNBlUsWQY6OUokL8H+v\\\\\\\/udCKVP7JFpAhMBFkowtjdGkcK3D4t47izFQKazZltSutqPQDY94IvnmFxisrosx2e96T\\\\\\\/e53EFbYzkwaJJQmPG\\\\\\\/7J3OTJGGSMJjlN9888SGPNgZcSZsedElEKMbPfwEh7HirWqTuP\\\\\\\/wkAFLNlvING48OJ5\\\\\\\/6z3z\\\\\\\/RYwPQiexw5d0\\\\\\\/pJ7lg\\\\\\\/w7fCov+vq6th8ZZuselHdhF3J5380sfigQOG8cyWbHpScD+7KeOyM1Z0Ug3K037SShWIe9MNqr9pwSJAgn4KI6o2zZ+9Y89UsyLAheIfNyykpnw\\\\\\\/ar4mum+J6YEHueqRc2zYSy0Cwd0T9ckoGLW0Dcq5SlMzDWrhBGBZIrz38jCH5Horbdo\\\\\\\/2BqsctJQz\\\\\\\/4Cf2oJFnkabqHK1Vgmg2\\\\\\\/KRo+hAcUqVLS2dhWpsBUddNmHx+DWz3tYbzZHZrCckEjMtFoHqxxsPU0jtvlW4Fwgs\\\\\\\/t59eFCaHnEoFn1LTY5huLW2vtVmKvqPJAuNGi38n\\\\\\\/\\\\\\\/AgNZwO66YT750M1xPYrOUyZApVx5kMMyJaMVqaS4+kSM2YfL8SBD0rDOu3cMfv+epbsdnJdMfB88PopQePicfGql\\\\\\\/GqLNJcEkSwmrLU00PpYDk0NZPgEQUStZr0NxaPGDUzyoE7gbUdUvEaba4s4Ne8wWdYIiprcx\\\\\\\/cr8p\\\\\\\/4sEdsBlhXT+JCbWkOa7UFLTJXjreULYMR9t8rdavHL39KFAyntpViH94uAdmAfyomnKe2z7SgBsbFax2ZQA0AxPCBw5X3rwhn59Mac53AtPwPwtw847FdVgNbbIEIua\\\\\\\/XWA+0M5d+m0Rczq4bJ7nTyRh+k\\\\\\\/mxKvIc8ON+z\\\\\\\/d3JF67a6Efy+u7hyQHAYsqOyZ6PZVsuBCzA29rxHZmyY8wDG5K5EGRLbWjmP4jJ4R1UPW3IAFz9W5ujgdo+GAh0r1ryhehYhOFA0+oX1e+pdPctB06VPK5x\\\\\\\/T5YkkR1O8Fg\\\\\\\/qfS+ks2DXMoog+9NUo\\\\\\\/4y5M32vNOQowBj4L8m0ReDsW7lq37vBT3IE5+ECzNYykDQHcFilfAM2+xB8zOzsAkNMBq2eU6Ngqbc5sEERbO4UbNFqHCK3zEah2XLO5luvdtdVDuHPJg96EqjOGGpGQTnVSvY2v6fGkT2974wYy22xg8q\\\\\\\/+fTguGA3u3wAV1AbVu9ZepZB2dw7uSEgdNehKdIEPiCACthHo4bFDwLUGbsTGD2BvwS0jg+hnfvfeUwmjWoKphNgRfIYrvicV\\\\\\\/4pB7778dUWtEQPjmNKAzlPdaZ7GcAjjk8xDmnCLl8ZdSz7rN0vtAb1LOts8vxKvpUErKnx1Buz2aiIhpbd6SbC87CxB9\\\\\\\/ulU9LeOCVUSIl89cPp8iQoXGJaiikz0jaXcDUWy1PdXXopKUBu3xQD53k0Qzmbylbhnlz2ehe5JC4\\\\\\\/LWFvoL1Uo3K5rTUyrh7I5nKOfGl0dmr85qbmVmKPPFSsEIH2aiGcXs9Q8mIEzEaDVVr+PLGe3LPAGELMF+r4hKDtKLO9H2uv3OeFtLoqZ1IrcfIwKcBd2OrGWTn\\\\\\\/HE2ujVVfWFQdHst2nEEz3lQQPCyTLzQ23gbS1Kk0iVrIurtYVk9FncsmXL2OHOFlzvJ95qOsL146aQOZMfgdGXhMsq+0+MjILGu7y6Rff8BWKG4Fp915NlzshPSPBd4TA1iteKz+SS49qy8m48o4sqfMd86bHRLKUYVeIy8gXO0IXO8fm3b5PsthM\\\\\\\/Mg8bQn1YMoqt1GrJ1VDbSdXvm54asqwSIXRakJkGVZSVXxObuL+1OptGOqZwWWNarLsyzYw5OncBeqAWQvDngfyaaMtDi+2xtL+X2VAQ8FcV\\\\\\\/GI4xqOWMhIHRp2yLaX9ePIC2rCCm31XD87OKUJkQBXRYzLL5xWhx0Q+XlOpZkqUHrTif\\\\\\\/Y8e7j9t0VoqTpezB6xUjzk2j66rpyWY6kOZmjAzMpJHrptwjcJca8RGWXGgAhee7nHdw5Z29195SGO6L6OKnEppfHdNGGYdMZNJuFbA8uOc2zdB57g2jyckdM+hEAQL0gqYMkVXGg62RHk\\\\\\\/evKdoQfHI+taBrunOufq3Is8dK1T3V+WgPCjUDM29kTuG7iT5iwTbk02PfnsWBLK5IngLw+gjx4\\\\\\\/bV5Jhc7UG8rFL7wOIXevXLdyN70oYGsAV\\\\\\\/NSOGdblizDazVfrJ2mxqBb+EKDwPk7PI+KEdOJUBYxGvqeASdWaOYDSrgw6ifva3ppcvsO6Yt\\\\\\\/O3z\\\\\\\/mZysKdvQ05k\\\\\\\/YCzUQEw0yR3mRn7roTO+PiKPFodSsRmVnmqE+3u2UHAy0R6GV5DOTUln6Ur24m\\\\\\\/GdMKaC5EdsC0g\\\\\\\/RE1rv16spZ71+q+36ypFTDPaB\\\\\\\/vBFNCuAyv8eNZvme9ao7FvpkAyYI5fPM9ZDg3+8UuUJPr4KnzhwJFm4u+80I\\\\\\\/kadOcWNjggM04mRd9+WvR8JCEDIsG2uHt1dD+\\\\\\\/e8WWs3JvVfUDu+oXHl8rRE6xnugwQZxPhtSpRmR6Vwg4dOBdRU08XrMjBkwROXR\\\\\\\/U9A9QIEWMivzBxir4NnoD92O8B9dZpT3kvjPdHlcs3aMMJGIJVSiTm+BzJxRmoiZHGtCP+Z3fHb11L8ShvvsyLc5ofAQS118r8ios18tQn1OijJ9muymxFbPHyeJfzkyVpCRlzHR6dQbsAlYI5xjZqUbTaIvuDVSRfkuBbBKujWLBu0l3qqkEcVEWVXJEt9ofkHOWR1xKu\\\\\\\/0oOkyEIkbsMo3pMk1X+ymEX4m1dy57LLh3Waz5UCXh3SFOW2FeF9PFjqG3Yo628ajltCLIj6For9i2rgNjDmQ2b73iQkETc6zZZX8X10cvjCgk2GnmnkjNCJi2YEtlGwS6YFw3hdj+vj5v49qb8YErDjc3Ezp3xq7r26+N972UPY8c+FkRxIqPaI1jhdpJdBvZKfSXb6t14Ru4\\\\\\\/xEbXgmUY3U6mfCF63kqZAjKI8lgjMAl0xzz5x4uRI2dYTRRIIrWa2tdZkXQw7ok4343548CsNMCTP49P7PZtkEbwsL8eWHeyQkATXWIh\\\\\\\/kNHcr62QsU9hJXRd6iGrvKa8y0j1lVVftbhtm8hnwwe7HbKX2ED+ykV\\\\\\\/zloPA9QZcxNsHcxhBnpoNu2\\\\\\\/ob9m7oz1IdZIXCNuyumrSzFeg+gjnybEyQrrO++UHVtWfObKb8WbTIVkYDvaeJHpRDUlT9zGLrHWqQhi+RR5qOU1HoJSuS8yXMCKa4Q395SE8axSrUwGt9TYkhZRHMuMXT0mddUqhAOli1YKwSWVi896RV9ZWNa9LY1cB2YltY8756sFMVkWJfC4mzOhWGXYGUUMZASb1+FTfz7aUmZJ+r9kfsugiXyNn5ylsQVof4KLmd6U1sGHpZJ0qqwQ9cGWoZs6D9vnqC4lk4pAyj\\\\\\\/P9DT43+3sAKGOnY3KO\\\\\\\/dzyF59kPDo0HjCLU6jfTDdtsJXvxNwQH5Kwmkqo4tlCzfP92WP3zLATaiIrQLznYWIymKyDqefkwOQ86227MzUQ61uU5\\\\\\\/fIFnPVeiOa\\\\\\\/0uJrR+AWlFN9YWp7Er5ZuhWesz0zAGrw49mfmWdKI7dMu8ECeG9xWbHIlPxZ9XQ3EdgbBQmDVKORDpgmFM\\\\\\\/EvC1Kc0hNcGainhiTkyGnTPgY+BQe0Ph5weGMG1gjfYt0a2JcRxcjK8YRpdnlxRw+pCqxGbwTXxdLXkLyzyZ5rnHLuTNqNmXzDz1gU3mEWCEdWadV0Qbnqh9v2oGyTtE1LI3txKd\\\\\\\/Z616mUYNzjaTDzxz3To1ZB514NWROxAL4BYsW+93MSfS\\\\\\\/+QOzuNrevBnjDcYeGS4gnoCOSvOQSm97ahDTNNHecO977l25A9WL5YNIdAz7Ex2GnFwfN6rtlTPEjRttspn2fRK18hQcyI+++3QZHtMkgIEowsGT76RiiE7cZkvsJOKTh2jf8eodc\\\\\\\/46fwBPPErg06CqHTc6omYUESm1kapNYIq82wkCIK5EswZwWLzS0qFTnRVcc5aT9B96UeQD6TtlN\\\\\\\/\\\\\\\/k65zf1EWCAqNPmLq\\\\\\\/kk+u\\\\\\\/gAioprhMgy7nFi+yWOH0JFlM0huQaAbtGOcEv\\\\\\\/OyT5BosC\\\\\\\/VyRM3Klu16oh0\\\\\\\/sTz4VlDJWVk2uYznOQAWRmted3uPx1r7GeepxFmYDCeyxyUiwlA14aVv7XCw\\\\\\\/xTRanjjlYTkTKvtotGe+aD6RhiZYU+qlo238ntMp55\\\\\\\/dpZYSsQECUGWRhYMh95JPOoxhPBYgX0u2keSHyj6PTAPQNsuMxsedhqpMh1ymRKjT7ka8KvOsHVO2n0jP1ye+L9QVKgshY9MR8aH+ms7j9QxmIlYSm5JnUAYzUYHfUAsLbtb8DH9CM4ExgCj8XHU9nzU9z9xcyXC6DDA93L5SqN356ehl1FWOe9wLKW8K1uU40WHCEuzsQMw018JUq3Wy3A8jqY8ndVY9sU1T5dPGXPRVqNcm4A39D4hHJbPnG6QGd+YRFVp7vfjEOTXa\\\\\\\/Pd4c0UmEKrvCT6bgxE07fWYInKkQ5nIsJ9IlpGC0\\\\\\\/5WGktZ0gnGnIq9NuNmIXyTxlzK8eIlGQNpB+H6Ye0\\\\\\\/Reif1i4j4JDidhKdC81usmg7BeQkteRhEUVLXfVJFPUXdTEmQSVeGnAAAqtHPTLsfhi3dHX22fanAigCFlySFf0wwC7N2FPhE3+NUjHmeviLd8RPjFTh5F3AB5H\\\\\\\/ifx3S1Vw+YC3sC2jc9hw9bn\\\\\\\/dNtOrhaLUKci77N2TseW30kA4i2ZTLINVkGjvPrOtMpo7mAZ+BojyhK02RZroLE+HjNgtx3P96h2pfXS1Br1pcC4NNQGA6k1SJ1iCEUi2hrM\\\\\\\/fzN3g6sqabkcWjDp0FBPp7FgkGMo8kchyaYvCWRWZLEgTJYzWRtc7xik1FMAFYl6ZBW8ieiXfduSKE9RoSaj7MlFscNqZCMafpzWyYxyMMoankZ5t7WE3nOG1+dzz5EQv7MKAPZ9HIvaMZaaTeekjDk5GjzUDsII9OATw8lcnpWmBQOezjv5638W5hcEV1XCBR8Ro2kdr2GSxN5UxaqsJnUfRjSez9IQQA+548cY9RzIl3mE\\\\\\\/OQqSJQc+ZmYbd6xdSe5fMl4iuCUzHrSMlhSe5HmZP\\\\\\\/BZDj9iL7PmtBuWtbOnELqPzM0xZ25+Eo6liPd\\\\\\\/pzPcAZLkacsM\\\\\\\/d7Gmb+w+2rgWe+B2PpZX33bU8MhwVOPORDNVKu+5W1CiLEz0o8hlT0WsgjkbhzUw+vrxpr1XAiKVtGWVUrRrEHouU6yhEQ8sXN5USpWxTj6AGXkpsjWgY9Zg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022HxGyYS49yRLG2gQF6hex9Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00222bea770cbee0c5f50dec512d7e1704de283987733269557e5742d629b8a20be1\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Up553OEqTbKkdWYtCABeLrKlURz8gjyKt\\\\\\\/AYdvM+9pSTYCkzeO69\\\\\\\/ATfLkRKETmtM2NgFq4mD1k+Ybph+E6pZH4JKKiyOVn90ugOmm\\\\\\\/ZUm6s4At9Nn65VFeU6CuRJFQ0XzOa3ypAcL1nHh2\\\\\\\/qbc6OdnrqGP+9wMfgJadsiaFYvP2fTWxouoCEsQFRdvZ9jgCGGY7HCPiLHiSl8bFy\\\\\\\/+7kTZEYW7iYnjp+YOBs49E8Pkf2B198MtSWtxKyCtV8guJBbsL3wJGD\\\\\\\/RvGMaZw71PJnpGPdvwUeGqGBIJSQRuocvJ7sZgjRP0rJ\\\\\\\/BPJYq6v3EPbdTH\\\\\\\/xUbNlUEZFtmzDuQrsZnqBYRCxp1l4IsIk7fGnVP5QNwIoTzvMIcJ+G8pScKNo0328tEnF+\\\\\\\/xxIEJ57H2shf5+nAgOcBULN7uRBaxm6zr5fJkXMzIasgIGVKY2t9zx1a2j+\\\\\\\/e\\\\\\\/raZuHJuvXC2X98CzhJJ0qQ1E+e5ooykzAd0vKsPT0L6pQ3yVqIhrwg9zhUXb0nqF+n0jtI+GbphUx8\\\\\\\/xB50fhY8Sz+\\\\\\\/vEmLgIe2ANSE25czp1+EArHJ+rxNzVxxsbZcyyAUkXC\\\\\\\/l+lWibWNvvrHTQjS8TIWBJxYa9Irgr1iiS8M6jncJwhUxEvgroLeCyYyPGiVgaoLGHMunMNJfAID5tO3mInT2Sm5N9zE4Jjg87bYlU5zo\\\\\\\/jFkRoaKQmBmjBPHZPTVTFpSj9w73V40EvLAf863tz9EkutUQopJXyuwPKCMD3TfIyNoAmfR02FQuMAgtOBO1dNn4MsfSQxwLRG0FGi7yTV8bLgptVACobibJeVgdxX1h4NmOHQ61E+BCcO0\\\\\\\/KK4kFzvQExtn45mLgDY3pc4JRD9P0+i4LVndt6cYH4YBMOK55kMefLElxR+yhsUHyz14Aqb0wlDzNJCpjU3KcY11EEGC1FZmBtSbS0EbW3zWGRB8WM2qsiue\\\\\\\/lV4cTsOcH7mPOHSss\\\\\\\/c6cc3lrHIezm3HDnkK3HR8Hpgj2fHAlLDlULRbR01xwgQRkhKthztfJVAnZl6lH6fhtiTgP2v49c3TgWx3o5jR4e34qobFbK3qQknQqC35SxhDYHgsD0W+EkjFAHsdxFTZO7Slll82tW\\\\\\\/RVRtHkEcfY7ojaCC0uJyOptMH46rSJN5v63tI15vlYJn\\\\\\\/31mdo9JJIS+Gbi6dmJcet4\\\\\\\/dD\\\\\\\/6ZYAJUJdhSe2EpjasksZtUr4rHHlnSyhljlWSFdvfFHoI8vbXjxPT1ppcvGcBXYykTky2D2wZDcNtJ\\\\\\\/+P+A8aZDVSuBARhLav4YTvEyOuCWnWs+toKrkenRryo6QJjNbNcakyYrjkk36XuQpKVmMW9jTlbrs+aPdgK+GfG0bzeRXFdRctDlhwNMRvFp8MJQ8TXwpaCBD156AusthNC5Yz7kGSmVZdUYdV2uYoGtQGO5LxuQPB0FzLz6ixRKRSzNPLRZxDlreJcJJKND8omhQcCVysqCqAGoYj3xucilS30Fgg2c4hRPVJvxeTA\\\\\\\/6ebAMzmylqy+kvjN10X75flF7avEVoxbs3x36PtpZp4CIB16XEmBhxyPykfWJ2zwH+glWueCHaEVm4JQSpf3w7vk92Mkn0BryI8UIy4QlsSQaiejyKzxEkS+THsJUffHT0aMRNay0aktveiSvHFIPxT5so1WLo7sSC76zSe0HmflY6ZRTQXkWetFHRTM3CLmTj9vdIoqvQJH3ruQE6Gd2XmN76qra\\\\\\\/1hfP\\\\\\\/dnBBVfgeTVWiAmyHGt1QnjvGESj8z2fEcOt29nKVqBS7pvSDP0CAjq8xWvu2MlLGqQpIKLDiPH4qzB66XH9hObqhDJVH3rv2\\\\\\\/RmVglNUxuXMulgtUuqJXB+46wvUHr9bFufzNCEVMvkW5TgEZekI\\\\\\\/ZfTtO9y3EEzg+1PDeHsKPZqqoX3oku\\\\\\\/NJgpRfPxClv7MaYrHF5uVf1M1UZG4Nu6tMxVqBvHDvCsSjrBbIPiNncCjGMGf8nzT1Q3zDmkscmqo848N9wxS\\\\\\\/aNQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022TyMlBWDDyq6wzmX0qHnt+Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022aeb1ee0e3444922563141d51a031fae167f17c0a6915abf290cb84aab9646d5a\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460098 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genomic-selection">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/3/743" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-x4YN1enO9yXqCKicq0C5FcIt4z-aniNk4Jmqsl3b1fQ" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/3/743" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-ZECT1zHvUAoA6s2j1zWGhpsWcbGSqWurAGiJC395pRo" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/3/731" title="Differential Expression of Genes Involved in Host Recognition, Attachment, and Degradation in the Mycoparasite Tolypocladium ophioglossoides" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/3/755" title="Genome-Wide Investigation of MicroRNAs and Their Targets in Response to Freezing Stress in Medicago sativa L., Based on High-Throughput Sequencing" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460098" id="node460098" data-pisa="ggg;6/3/743" data-pisa-master="ggg;g3.115.025957" data-apath="/ggg/6/3/743.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Implementation of the Realized Genomic Relationship Matrix to Open-Pollinated White Spruce Family Testing for Disentangling Additive from Nonadditive Genetic Effects</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-9112-8522" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Omnia Gamal El-Dien</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-4469-2929" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Blaise Ratcliffe</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-5504-3735" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jaroslav Klápště</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-9344-6348" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Ilga Porth</span>, <span class="highwire-citation-author" data-delta="4">Charles Chen</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0002-4887-8977" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Yousry A. El-Kassaby</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">743-753;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.025957 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460098" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Omnia Gamal El-Dien </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Forest and Conservation Sciences, Faculty of Forestry, University of British Columbia, Vancouver, V6T 1Z4, Canada</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Pharmacognosy Department, Faculty of Pharmacy, Alexandria University, 21521, Egypt</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Omnia%2BGamal%2BEl-Dien%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gamal%20El-Dien%20O&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AOmnia%2BGamal%2BEl-Dien%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9112-8522" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Omnia Gamal El-Dien</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Blaise Ratcliffe </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Forest and Conservation Sciences, Faculty of Forestry, University of British Columbia, Vancouver, V6T 1Z4, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Blaise%2BRatcliffe%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ratcliffe%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABlaise%2BRatcliffe%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4469-2929" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Blaise Ratcliffe</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jaroslav Klápště </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Forest and Conservation Sciences, Faculty of Forestry, University of British Columbia, Vancouver, V6T 1Z4, Canada</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Genetics and Physiology of Forest Trees, Faculty of Forestry and Wood Sciences, Czech University of Life Sciences Prague, 165 21 Praha 6, Czech Republic</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jaroslav%2BKl%C3%A1p%C5%A1t%C4%9B%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kl%C3%A1p%C5%A1t%C4%9B%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJaroslav%2BKl%25C3%25A1p%25C5%25A1t%25C4%259B%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5504-3735" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jaroslav Klápště</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Ilga Porth </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Forest and Conservation Sciences, Faculty of Forestry, University of British Columbia, Vancouver, V6T 1Z4, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ilga%2BPorth%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Porth%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AIlga%2BPorth%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9344-6348" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Ilga Porth</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Charles Chen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Biochemistry and Molecular Biology, Oklahoma State University, Stillwater, Oklahoma 74078</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Charles%2BChen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chen%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACharles%2BChen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Yousry A. El-Kassaby </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Forest and Conservation Sciences, Faculty of Forestry, University of British Columbia, Vancouver, V6T 1Z4, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yousry%2BA.%2BEl-Kassaby%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=El-Kassaby%20YA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYousry%2BA.%2BEl-Kassaby%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-4887-8977" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Yousry A. El-Kassaby</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:y.el-kassaby@ubc.ca" class="" data-icon-position="" data-hide-link-title="0">y.el-kassaby@ubc.ca</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/6/3/743" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460098" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460098/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/3/743.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460098" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460098/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/3/743.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460098" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460098/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/3/743.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460098" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460098/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/6/3/743.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460098" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460098/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-4">The open-pollinated (OP) family testing combines the simplest known progeny evaluation and quantitative genetics analyses as candidates’ offspring are assumed to represent independent half-sib families. The accuracy of genetic parameter estimates is often questioned as the assumption of “half-sibling” in OP families may often be violated. We compared the pedigree- <em>vs.</em> marker-based genetic models by analysing 22-yr height and 30-yr wood density for 214 white spruce [<em>Picea glauca</em> (Moench) Voss] OP families represented by 1694 individuals growing on one site in Quebec, Canada. Assuming half-sibling, the pedigree-based model was limited to estimating the additive genetic variances which, in turn, were grossly overestimated as they were confounded by very minor dominance and major additive-by-additive epistatic genetic variances. In contrast, the implemented genomic pairwise realized relationship models allowed the disentanglement of additive from all nonadditive factors through genetic variance decomposition. The marker-based models produced more realistic narrow-sense heritability estimates and, for the first time, allowed estimating the dominance and epistatic genetic variances from OP testing. In addition, the genomic models showed better prediction accuracies compared to pedigree models and were able to predict individual breeding values for new individuals from untested families, which was not possible using the pedigree-based model. Clearly, the use of marker-based relationship approach is effective in estimating the quantitative genetic parameters of complex traits even under simple and shallow pedigree structure.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/open-pollinated-families" class="hw-term hw-article-keyword hw-article-keyword-open-pollinated-families" rel="nofollow">open-pollinated families</a></li><li class="kwd"><a href="/keyword/genetic-variance-decomposition" class="hw-term hw-article-keyword hw-article-keyword-genetic-variance-decomposition" rel="nofollow">genetic variance decomposition</a></li><li class="kwd"><a href="/keyword/pedigree-and-marker-based-relationships" class="hw-term hw-article-keyword hw-article-keyword-pedigree--and-marker-based-relationships" rel="nofollow">pedigree- and marker-based relationships</a></li><li class="kwd"><a href="/keyword/mendelian-sampling-term" class="hw-term hw-article-keyword hw-article-keyword-mendelian-sampling-term" rel="nofollow">Mendelian sampling term</a></li><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">GenPred</a></li><li class="kwd"><a href="/keyword/genomic-selection" class="hw-term hw-article-keyword hw-article-keyword-genomic-selection" rel="nofollow">genomic selection</a></li><li class="kwd"><a href="/keyword/shared-data-resource" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resource" rel="nofollow">shared data resource</a></li></ul><p id="p-5">Open-pollinated (OP) (also known as wind-pollinated) family testing is, by far, the simplest and most economical means for screening, evaluating, and ranking large numbers of candidate parent trees. Thus, OP testing combines the simplest known field experimental design in pedigree testing as candidate trees enter the test as maternal parents and their offspring are assumed to represent independent half-sib families. OP testing has been widely implemented for several tree species throughout the world, <em>e.g.</em>, radiata pine (<em>Pinus radiata</em> D. Don) (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Burdon and Shelbourne 1971</a>), interior spruce [<em>Picea glauca</em> (Moench) Voss × <em>P. engelmannii</em> Parry ex Engelm.] (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Kiss and Yanchuk 1991</a>), Douglas-fir [<em>Pseudotsuga menziesii</em> (Mirb.) Franco] (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">El-Kassaby and Sziklai 1982</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Johnson 1997</a>), western larch (<em>Larix occidentalis</em> Nutt.) (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Ratcliffe <em>et al.</em> 2014</a>), and Scots pine (<em>P. sylvestris</em> L.) (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Korecky <em>et al.</em> 2013</a>), and it is often considered as a prelude to full-pedigree testing (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Jayawickrama and Carson 2000</a>).</p><p id="p-6">Genealogically speaking, OP testing (<em>i.e.</em>, partial pedigree) is positioned between the “no pedigree” provenance testing (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Callaham 1964</a>) and the “full-pedigree” mating design-based progeny testing that includes all higher levels of relatedness and connectivity among the created families (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Namkoong <em>et al.</em> 1988</a>). Thus, the accuracy of all OP testing-based estimated genetic parameters (<em>e.g.</em>, additive genetic variance, heritability, breeding values, <em>etc</em>.) is superior to the former yet somewhat limited compared to the latter (but also see, <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Hallingback and Jansson 2013</a>). In fact, doubts are often raised regarding the accuracy of OP family testing-derived genetic parameters as the assumption of “half-sibling” is hardly fulfilled (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Namkoong 1966</a>; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Squillace 1974</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Askew and El-Kassaby 1994</a>).</p><p id="p-7">The pedigree-based genetic relationships among individuals [based on the so-called <strong><em>A</em></strong>-matrix: average numerator relationship matrix (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Wright 1922</a>)] are often used to estimate the genetic variance components by using the Restricted Maximum Likelihood (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Gilmour <em>et al.</em> 1995</a>) and predict each individual’s breeding value using the Best Linear Unbiased Prediction algorithms (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Henderson 1975</a>, <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">1976</a>, <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">1984</a>). However, while effective, this method with its traditional pedigree-based approach does not adjust for the Mendelian sampling term, that is, this method ignores variation among family members of a half- or full-sib family around the family’s average relatedness [as all sibs are not alike (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Hill and Weir 2011</a>)]. Furthermore, the utilization of the <strong><em>A</em></strong>-matrix, specifically, in the case of the well-known “shallow” pedigree present within most forest tree breeding and testing populations does not permit detecting hidden coancestry and inbreeding. Consequently, individuals’ estimated breeding values are inflated by the overestimation of the additive genetic variance.</p><p id="p-8">With the affordability, scalability, and high-throughput nature of next generation sequencing technologies, tens of thousands of single nucleotide polymorphisms (SNPs) have become available for model and nonmodel species (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Baird <em>et al.</em> 2008</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Elshire <em>et al.</em> 2011</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Peterson <em>et al.</em> 2012</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Poland <em>et al.</em> 2012</a>; <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Truong <em>et al.</em> 2012</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Chen <em>et al.</em> 2013</a>). This technical advancement made it possible to ascertain, with a great level of accuracy, the actual fraction of alleles shared between individuals, and the estimates of the individuals’ pairwise realized relationship including potential inbreeding can be easily determined (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Santure <em>et al.</em> 2010</a>). Therefore, genomic fingerprinting data permit the accurate estimation of the realized relationships among any set of individuals, irrespective of their genealogy, to construct the realized genomic relationship matrix (<strong><em>G</em></strong>-matrix) which can be used to substitute the <strong><em>A</em></strong>-matrix (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">VanRaden 2008</a>). This advancement represents a clear quantitative genetics watershed as the complete dependency on known pedigree relationships (<em>i.e.</em>, <strong><em>A</em></strong>-matrix) for estimating genetic parameters can be circumvented in the so-called “pedigree-free models” using the <strong><em>G</em></strong>-matrix. As already demonstrated in earlier cases, the <strong><em>G</em></strong>-matrix can provide relatively accurate genetic variance components and breeding value estimates without the need for elaborate mating designs (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Thomas <em>et al.</em> 2002</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Frentiu <em>et al.</em> 2008</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Hayes <em>et al.</em> 2009</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">El-Kassaby <em>et al.</em> 2012</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Gay <em>et al.</em> 2013</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Porth <em>et al.</em> 2013</a>; <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Zapata-Valenzuela <em>et al.</em> 2013</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Klápště <em>et al.</em> 2014</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Muñoz <em>et al.</em> 2014</a>).</p><p id="p-9">The use of the <strong><em>G</em></strong>-matrix in OP family testing has several implications and is expected to: (1) overcome the drawback of the average numerator relationship matrix (<strong><em>A</em></strong>-matrix) as genomic data will unravel any undetectable hidden relatedness such as full-sibs, self-sibs, and self-halfs that inflates the estimated additive genetic variance (<a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Namkoong 1966</a>; <a id="xref-ref-58-2" class="xref-bibr" href="#ref-58">Squillace 1974</a>; <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Askew and El-Kassaby 1994</a>) (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>), (2) provide more accurate genetic covariances among relatives, thus accounting for the Mendelian sampling term (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Visscher <em>et al.</em> 2006</a>), and (3) provide higher flexibility in capturing the allele frequency segregation in quantitative trait loci (QTL) (present <em>vs.</em> absent QTL) (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Lippert <em>et al.</em> 2013</a>). Additionally, we hypothesize that also for OP families the use of genomic markers will create an opportunity to effectively decompose the genetic variance components, thus separating the additive and nonadditive genetic components through the definition of realized genomic relationship matrix related to specific variance components, a so far unattainable feat for OP family testing.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/3/743/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Representative histograms of the genomic pairwise relationship coefficients within (right panel) and among (left panel) members of the 214 white spruce OP families showing relationships clustering around the expected 0.25 with deviations from 0.25 as indicative of imperfect half-sib family (right panel) and clustering around 0.00 as indicative of no relationship (left panel)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2082364622" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Representative histograms of the genomic pairwise relationship coefficients within (right panel) and among (left panel) members of the 214 white spruce OP families showing relationships clustering around the expected 0.25 with deviations from 0.25 as indicative of imperfect half-sib family (right panel) and clustering around 0.00 as indicative of no relationship (left panel).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/3/743/F1.medium.gif" width="440" height="301"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/6/3/743/F1.medium.gif" width="440" height="301"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/3/743/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/3/743/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467845" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-10" class="first-child">Representative histograms of the genomic pairwise relationship coefficients within (right panel) and among (left panel) members of the 214 white spruce OP families showing relationships clustering around the expected 0.25 with deviations from 0.25 as indicative of imperfect half-sib family (right panel) and clustering around 0.00 as indicative of no relationship (left panel).</p><div class="sb-div caption-clear"></div></div></div><p id="p-11">Here, we used 214 white spruce OP families grown on one site [Mastigouche Arboretum, Quebec, Canada (Lat. 46° 38’ N, Long. 73° 13’ W, Elev. 230 m)] in a randomized complete block design, replicated over six blocks (replications) with each OP family represented by five-tree row plots within each of the six blocks [for complete details, see <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Beaulieu <em>et al.</em> 2014</a>)].We compared the genetic variance estimates generated from both the average numerator relationship matrix (the expected relationships) and the realized genomic relationship matrix (the observed relationships) to demonstrate the genomic markers’ utility in partitioning the genetic variance components into additive and nonadditive effects. To our knowledge, this study provides the first attempt of such an analysis approach in OP families.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>White spruce open-pollinated progeny test, phenotype data, and genotyping</h3><p id="p-12">The white spruce [<em>P. glauca</em> (Moench) Voss] phenotypic and genotypic data used are available online from the Dryad Digital Repository: doi:10.5061/dryad.6rd6f (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Beaulieu <em>et al.</em> 2014</a>). Briefly, the study site is part of a larger, three-site white spruce provenance-progeny test established in 1979 by the Canadian Forest Service in Quebec, Canada. Each site was planted as a randomized complete block design with six blocks and five-tree row plots at 1.2 and 2.4 m spacing within and between rows, respectively. The present study is based on a subset of the provenance-progeny test that includes eight individuals for each OP family from 214 families representing a total of 1694 individuals; the average family representation per block was 1.32 trees since not all families were present in all blocks. It is noteworthy to state that the 214 OP families were selected from 43 provenances throughout Quebec, thus a population effect might be present. <a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">Beaulieu <em>et al.</em> (2014)</a>, using principal component analysis, reported the presence of weak population structure with no defined geographical pattern. In fact, <a id="xref-ref-5-4" class="xref-bibr" href="#ref-5">Beaulieu <em>et al.</em> (2014)</a> estimated that 1.3% of the total variance was explained by the first two principal component analysis eigenvectors and indicated that their lack of population structure is concordant with previous studies using the same populations, thus population structure was not considered in the present study. Wood density was determined using X-ray densitometry from 12 mm increment cores collected 1.3 m from the ground (see <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Beaulieu <em>et al.</em> 2011</a>, for details). Trees were genotyped for 7338 SNP loci from 2814 genes using Illumina Infinium HD iSelect bead chip PgAS1 (Illumina, San Diego, CA) (for details see <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Rigault <em>et al.</em> 2011</a>). The data used are available from the Dryad Digital Repository: doi: 10.5061/ dryad.6rd6f (<a id="xref-ref-5-5" class="xref-bibr" href="#ref-5">Beaulieu <em>et al.</em> 2014</a>).</p></div><div id="sec-3" class="subsection"><h3>Relationship matrices</h3><p id="p-13">The additive relationship matrix was estimated as follows:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/743/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/743/embed/mml-math-1.gif"/></noscript></span><span class="disp-formula-label">[1]</span></span>where <strong><em>Z</em></strong> is the rescaled genotype matrix following <strong><em>M</em></strong> − <strong><em>P</em></strong>, <strong><em>M</em></strong> is the genotype matrix containing genotypes coded as 0, 1, and 2 according to the number of alternative alleles, and <strong><em>P</em></strong> is a vector of twice the allelic frequency, <em>p</em> (<a id="xref-ref-62-2" class="xref-bibr" href="#ref-62">VanRaden 2008</a>). The dominance genetic variance was fitted by including a marker-based dominance relationship matrix as follows:<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/743/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/743/embed/mml-math-2.gif"/></noscript></span><span class="disp-formula-label">[2]</span></span>where <strong><em>W</em></strong> is the matrix containing −2<em>q</em><sup>2</sup> for the alternative homozygote, 2<em>pq</em> for the heterozygote, and −2<em>p</em><sup>2</sup> for the reference allele homozygote (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Vitezica <em>et al.</em> 2013</a>). Similarly, epistatic variance was fitted by including several relationship matrices capturing first-order additive × additive, dominance × dominance, and additive × dominance interaction. The relationship matrices were constructed as the Hadamard product of the relationship matrices defined above: <strong><em>G</em><sub>add</sub></strong>#<strong><em>G</em><sub>add</sub></strong>, <strong><em>G</em><sub>dom</sub></strong>#<strong><em>G</em><sub>dom</sub></strong>, and <strong><em>G</em><sub>add</sub></strong>#<strong><em>G</em><sub>dom</sub></strong> (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Su <em>et al.</em> 2012</a>; <a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Muñoz <em>et al.</em> 2014</a>).</p><p id="p-14">The variance components from pedigree-based analysis (ABLUP) were obtained by solving the following mixed model:<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/743/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/743/embed/mml-math-3.gif"/></noscript></span><span class="disp-formula-label">[3]</span></span>where <strong><em>y</em></strong> is the vector of phenotypic measurements, <strong><em>β</em></strong> is the vector of fixed effects (overall mean), <strong><em>u</em></strong> is the vector of random additive genetic effects following <strong><em>u</em></strong> ∼ N(0, <strong><em>A<span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/743/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/743/embed/mml-math-4.gif"/></noscript></span></span></em></strong>), where <strong><em>A</em></strong> is the average numerator relationship matrix and <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/743/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/743/embed/mml-math-5.gif"/></noscript></span></span> is the additive genetic variance, <strong><em>r</em></strong> is the vector of random replication effect following <strong><em>r</em></strong> ∼ N(0, <strong><em>I<span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/743/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/743/embed/mml-math-6.gif"/></noscript></span></span></em></strong>), where <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/743/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/743/embed/mml-math-7.gif"/></noscript></span></span> is the replication variance,<sup><span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/743/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/743/embed/mml-math-8.gif"/></noscript></span></span></sup>is the vector of random replication × family interaction effects following <sup><span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/743/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/743/embed/mml-math-9.gif"/></noscript></span></span></sup> ∼ N(0, <strong><em>I<span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/743/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/743/embed/mml-math-10.gif"/></noscript></span></span></em></strong>), where <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/743/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/743/embed/mml-math-11.gif"/></noscript></span></span> is the replication × family interaction variance, and <strong><em>e</em></strong> is a vector of the random residual effects following <strong><em>e</em></strong> ∼ N(0, <strong><em>I<span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/743/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/743/embed/mml-math-12.gif"/></noscript></span></span></em></strong>), where <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/743/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/743/embed/mml-math-13.gif"/></noscript></span></span> is the residual error variance, <strong><em>X</em></strong>, <strong><em>Z<sub>i</sub></em></strong>, <strong><em>Z<sub>j</sub></em></strong>, and <strong><em>Z<sub>k</sub></em></strong> are incidence matrices relating fixed and random effects to measurements in vector <strong><em>y</em></strong>.</p><p id="p-15">The variance components from the analysis using the marker-based additive relationship matrix (GBLUP-A) were obtained from the model described above but the average numerator relationship matrix <strong><em>A</em></strong> is substituted by the marker-based relationship matrix <strong><em>G<sub>add</sub></em></strong>. The extended model for the dominance terms are performed as follows:<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/743/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/743/embed/mml-math-14.gif"/></noscript></span><span class="disp-formula-label">[4]</span></span>where <strong><em>d</em></strong> is the vector of the random dominance effect following <strong><em>d</em></strong> ∼ N(0, <strong><em>G<sub>dom<span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/743/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/743/embed/mml-math-15.gif"/></noscript></span></span></sub></em></strong>) where <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/743/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/743/embed/mml-math-16.gif"/></noscript></span></span> is the dominance variance. Additional model extension for epistatic terms is performed as follows:<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/743/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/743/embed/mml-math-17.gif"/></noscript></span><span class="disp-formula-label">[5]</span></span>where <strong><em>axa</em></strong> is the vector of random additive × additive epistatic interaction effects following <strong><em>axa</em></strong> ∼ N(0, <strong><em>G<sub>add#add<span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/743/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/743/embed/mml-math-18.gif"/></noscript></span></span></sub></em></strong>), where <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/743/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/743/embed/mml-math-19.gif"/></noscript></span></span> is the additive × additive epistatic interaction variance, <strong><em>dxd</em></strong> is the vector of random dominance × dominance epistatic interaction effects following <strong><em>dxd</em></strong> ∼ N(0, <strong><em>G<sub>dom#dom<span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/743/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/743/embed/mml-math-20.gif"/></noscript></span></span></sub></em></strong>), where <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/743/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/743/embed/mml-math-21.gif"/></noscript></span></span> is dominance × dominance epistatic interaction variance, and <strong><em>axd</em></strong> is the vector of random additive × dominance epistatic interaction effects following <strong><em>axd</em></strong> ∼ N(0, <strong><em>G<sub>add#dom<span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/743/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/743/embed/mml-math-22.gif"/></noscript></span></span></sub></em></strong>), where <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/743/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/743/embed/mml-math-23.gif"/></noscript></span></span> is the additive × dominance epistatic interaction variance.</p><p id="p-16">Narrow-sense heritability was estimated as <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/743/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/743/embed/mml-math-24.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/743/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/743/embed/mml-math-25.gif"/></noscript></span></span> represents the estimate of the additive variance and <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/743/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/743/embed/mml-math-26.gif"/></noscript></span></span> equals the sum of <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/743/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/743/embed/mml-math-27.gif"/></noscript></span></span> and all random model effect variance component estimates such as additive, dominance, additive × additive, additive × dominance, and dominance × dominance interactions following that of the ABLUP and GBLUPs (GBLUP-A, GBLUP-AD, and GBLUP-ADE) models, respectively (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). The analyses and the derived genetic and environmental parameters and their SEs for the ABLUP and GBLUPs were estimated using ASReml v. 3.0 software (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Gilmour <em>et al.</em>, 2009</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/467809/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/467809/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Estimates of genetic variance components and their SEs for height (HT) and wood density (WD) for the Québec white spruce population across the four genetic models</span><div class="sb-div caption-clear"></div></div></div><p id="p-19">Models were compared using the Akaike Information Criterion (AIC) estimates obtained from each analysis (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Gilmour <em>et al.</em> 2009</a>) and the precision of the estimated variance components and their dependence was assessed by investigation of accumulated eigenvalues of the asymptotic sampling correlation matrix of variance component estimates <strong><em>F</em></strong>, where <strong><em>F</em></strong> = <strong><em>L</em></strong><sup>-1/2</sup><strong><em>VL</em></strong><sup>-1/2</sup> using the asymptotic variance–covariance matrix of estimates of variance components <strong><em>V</em></strong> and its diagonal matrix <strong><em>L</em></strong> (<a id="xref-ref-48-3" class="xref-bibr" href="#ref-48">Muñoz <em>et al.</em> 2014</a>).</p><p id="p-20">A 10-fold cross-validation scenario with five replications was used to assess prediction accuracy and consistency within and between the various models, respectively. Folding of the training population was either random, block restricted, or family restricted. The latter scenario removes the genetic relatedness between the training and validation populations according to the pedigree information. That is, all individuals belonging to a single OP family were strictly assigned to either the training or validation population. Block restricted folding was performed as a leave one block out scenario. That is, all individuals belonging to a single block were assigned as the validation population, while the individuals belonging to the remaining five experimental blocks were randomly divided into 10 folds as the training population. Random folding had no prior restriction when assigning the folds.</p><p id="p-21">Prediction accuracy within and consistency between models was evaluated using the mean Pearson correlation from the five replications. Specifically, the correlation values for each replication were calculated as:<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/743/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/743/embed/mml-math-46.gif"/></noscript></span><span class="disp-formula-label">[6]</span></span>where EBV refers to the individual additive breeding value of the validation population obtained using the entire data set (1694 individuals) for the <em>l</em><sup>th</sup> model (ABLUP, GBLUP-A, GBLUP-AD, GBLUP-ADE), PBV is the individual additive breeding value of the validation population obtained using the <em>m</em><sup>th</sup> model (ABLUP, GBLUP-A, GBLUP-AD, GBLUP-ADE) and <em>n</em><sup>th</sup> cross-validation scenario (random, block, family), <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/743/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/743/embed/mml-math-47.gif"/></noscript></span></span> is the covariance, and <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/743/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/743/embed/mml-math-48.gif"/></noscript></span></span> is the standard deviation. SEM for the correlations was computed using the following equation:<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/743/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/743/embed/mml-math-49.gif"/></noscript></span><span class="disp-formula-label">[7]</span></span>where <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/743/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/743/embed/mml-math-50.gif"/></noscript></span></span> is the standard deviation of the Pearson correlations and <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/743/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/743/embed/mml-math-51.gif"/></noscript></span></span> is the number of replicates.</p></div><div id="sec-4" class="subsection"><h3>Data availability</h3><p id="p-22">The data used are available from the Dryad Digital Repository: doi: 10.5061/ dryad.6rd6f (<a id="xref-ref-5-6" class="xref-bibr" href="#ref-5">Beaulieu <em>et al.</em> 2014</a>).</p></div></div><div class="section results" id="sec-5"><h2 class="">Results</h2><p id="p-23">As expected, replication and family × replication interaction produced constant variance components across the four studied models for both height (4.7 and 22%) and wood density (1 and 2–5%), leaving most of the within-replication effects residing within the residual terms (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). The greatest observed difference between the pedigree-based (ABLUP) and the marker-based (GBLUP-A) models was the substantial discrepancy of the additive genetic variance estimates’ magnitude (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). The additive genetic variances estimated from GBLUP-A were 64.4 and 46.9% of those from the ABLUP for height and wood density, respectively (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>). Naturally, this change is reflected in the residual terms as they increased to 110.1 and 168.4% of that of the ABLUP and subsequently resulting in substantial reduction of narrow-sense heritability estimates (<span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/743/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/743/embed/mml-math-52.gif"/></noscript></span></span>: from 0.25 down to 0.16 for height and from 0.61 down to 0.30 for wood density comparing ABLUP <em>vs.</em> GBLUP-A, respectively). Overall, narrow-sense heritability was reduced by 65% for height and 50% for wood density when the genomic relationship matrix GBLUP-A was employed (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>), highlighting known caveats of OP progeny testing. Also, the inflation of additive genetic variance observed in ABLUP and the subsequent impact on heritability estimates were expected, thus it is more reasonable to use the results from the GBLUP-A as the basis for comparing the extended analyses that included dominance (GBLUP-AD), and epistasis and dominance (additive × additive, dominance × dominance, and additive × dominance first-order interaction) (GBLUP-ADE).</p><p id="p-24">The GBLUP-AD analysis produced identical results to that of the GBLUP-A confirming the existence of minuscule and nonsignificant dominance variance estimates, accounting for 1.13 and 2.84% of the total phenotypic variance for height and wood density, respectively (<a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>). This is not surprising considering the small sample size of the studied OP families (≈8 individuals/family) or simply the fact that these traits do not possess dominance genetic variance (see <em>Discussion</em>). Including dominance variance in the models increased the AIC values for the models, indicating that GBLUP-AD models were overfitted compared to GBLUP-A models (<a id="xref-table-wrap-1-6" class="xref-table" href="#T1">Table 1</a>), and that the simpler GBLUP-A models should be preferred.</p><p id="p-25">The GBLUP-ADE analysis produced the most striking results with further reduction as to the additive genetic and the residual variances compared to the ABLUP and GBLUP-AD models for height and wood density, respectively (<a id="xref-table-wrap-1-7" class="xref-table" href="#T1">Table 1</a>). This observed reduction in the additive genetic and residual variances was caused by the presence of significant additive × additive genetic variance within the total phenotypic variance (<a id="xref-table-wrap-1-8" class="xref-table" href="#T1">Table 1</a>). This observed additive × additive genetic variance in turn resulted in further reduction of the narrow-sense heritability estimates; from 0.16 to 0.13 and from 0.30 to 0.18 in GBLUP-AD compared to GBLUP-ADE for height and wood density, respectively. Again, the GBLUP-ADE analysis did not cause any change to the dominance variances (<a id="xref-table-wrap-1-9" class="xref-table" href="#T1">Table 1</a>). Small and not significant dominance × dominance and additive × dominance first-order interactions were observed for height and wood density in the GBLUP-ADE (<a id="xref-table-wrap-1-10" class="xref-table" href="#T1">Table 1</a>). The AIC statistics for this model produced the best fit with values lower than that observed for all tested models for wood density (−9,726.65), supporting the inclusion of the additional epistasis terms in the model, specifically that of the additive × additive (<a id="xref-table-wrap-1-11" class="xref-table" href="#T1">Table 1</a>). Unexpectedly, the AIC for GBLUP-A (17,465.80) produced the best fit for height (<a id="xref-table-wrap-1-12" class="xref-table" href="#T1">Table 1</a>).</p><p id="p-26">Comparing the SEs for the predictions (SEPs) of breeding values (BV) between the ABLUP and GBLUP-A models, all of the SEPs for height and wood density BVs were smaller for GBLUP-A compared to ABLUP as all SEPs were below the 45° reference lines, clearly indicating the superiority of the GBLUP-A model (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). GBLUP-A and GBLUP-AD models produced identical results owing to the lack of significant dominance effects and all SEPs for height and wood density BVs resided on the diagonal 45° reference lines. Additionally, SEPs for height and wood density BVs from the GBLUP-ADE model were smaller than the corresponding SEPs produced by the GBLUP-A model indicating the effectiveness of the GBLUP-ADE model (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>). When we compared the pedigree- and the marker-based models using the cumulative proportion of variance that was explained by eigenvalues of the sampling variance–covariance matrix of variance component estimates, we found that the GBLUP-A outperformed the pedigree-based (ABLUP) models as indicated by the closeness of their respective lines to the ideal scenario (straight line) where the variance components are completely independent (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). Finally, since the GBLUP-ADE model does not have a corresponding model in the pedigree method, GBLUP-ADE was plotted only against the 45° diagonal for reference (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/3/743/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="SEPs of BVs from the ABLUP (x-axis) against that from the GBLUP-A (y-axis) for height (left panel) and wood density (right panel) and that from the GBLUP-A against those from the GBLUP-AD and GBLUP-ADE." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2082364622" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;SEPs of BVs from the ABLUP (x-axis) against that from the GBLUP-A (y-axis) for height (left panel) and wood density (right panel) and that from the GBLUP-A against those from the GBLUP-AD and GBLUP-ADE.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/3/743/F2.medium.gif" width="420" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/6/3/743/F2.medium.gif" width="420" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/3/743/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/3/743/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467816" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-27" class="first-child">SEPs of BVs from the ABLUP (x-axis) against that from the GBLUP-A (y-axis) for height (left panel) and wood density (right panel) and that from the GBLUP-A against those from the GBLUP-AD and GBLUP-ADE.</p><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/3/743/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Cumulative proportion of the variance explained by eigenvalues for ABLUP vs. GBLUP-A (top panel) and GBLUP-ADE (bottom panel) for height (left) and wood density (right). Diagonal line represents an orthogonal correlation matrix." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2082364622" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Cumulative proportion of the variance explained by eigenvalues for ABLUP &lt;em&gt;vs.&lt;/em&gt; GBLUP-A (top panel) and GBLUP-ADE (bottom panel) for height (left) and wood density (right). Diagonal line represents an orthogonal correlation matrix.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/3/743/F3.medium.gif" width="440" height="385"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/6/3/743/F3.medium.gif" width="440" height="385"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/3/743/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/3/743/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467814" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-28" class="first-child">Cumulative proportion of the variance explained by eigenvalues for ABLUP <em>vs.</em> GBLUP-A (top panel) and GBLUP-ADE (bottom panel) for height (left) and wood density (right). Diagonal line represents an orthogonal correlation matrix.</p><div class="sb-div caption-clear"></div></div></div><p id="p-29">Cross-validation prediction accuracies (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>; diagonals) indicated that the ABLUP model was associated with the lowest values among all tested models for both random and block restricted folding (range: 0.451−0.475 and 0.439−0.449 for height and wood density, respectively), while the GBLUP models produced greater prediction accuracies under the same two folding scenarios (range: 0.735−0.772 and 0.748−0.783, for height and wood density, respectively). Prediction accuracies were lowest under the family restricted scenario for the GBLUP models (range: 0.683−0.698 and 0.651−0.658, for height and wood density, respectively), with random folding producing the greatest prediction accuracies. Comparison of prediction accuracies among the GBLUP models using random folding showed that differences between GBLUP-A and GBLUP-AD were not significant (based on SEs); however the two were significantly greater than GBLUP-ADE for both height and wood density. The family and random folding scenarios both produced no significant differences in prediction accuracy among the GBLUP models.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/467706/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/467706/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Correlations for height (HT) and wood density (WD) between estimated individual additive breeding values (EBVs) and predicted individual additive breeding values (PBVs) produced by 10-fold cross-validation for the four models (ABLUP, GBLUP-A, GBLUP-AD, and GBLUP-ADE) using random, block, and family based folding</span><div class="sb-div caption-clear"></div></div></div><p id="p-32">Pairwise model comparisons (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>; off-diagonals) showed high consistency between all GBLUP models within the individual folding scenarios. It is also noteworthy to mention that under the family folding scenario, the ability of ABLUP to produce across family prediction challenges the assumption of zero expected relatedness among OP families, thus predictions of individual additive BVs here would simply be equal to the overall mean.</p></div><div class="section discussion" id="sec-6"><h2 class="">Discussion</h2><p id="p-33">Traditionally, the pedigree-based average numerator relationship matrix (<strong><em>A</em></strong>-matrix) is used to estimate the genetic variance components for forest tree progeny test populations. The estimated genetic variance components (<em>e.g.</em>, additive and dominance genetic variances, <em>etc</em>.) often are mating design-dependent and the mating scheme determines which component can be obtained. In most cases, this approach is incapable of disentangling the within-family genetic from within-family coenvironment effects. This is even more problematic in OP family screening as separating additive from nonadditive genetic variances is limited by shallow pedigrees and lack of connectedness among the tested families; furthermore, as shown in <a id="xref-table-wrap-1-13" class="xref-table" href="#T1">Table 1</a>, the estimated additive genetic variance is inflated as the half-sib assumption is hardly fulfilled (<a id="xref-ref-49-3" class="xref-bibr" href="#ref-49">Namkoong 1966</a>; <a id="xref-ref-58-3" class="xref-bibr" href="#ref-58">Squillace 1974</a>; <a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">Askew and El-Kassaby 1994</a>). In fact, the estimated genomic pairwise relationships of the 214 OP families studied showed deviation from the expected 0.25 coefficient of relatedness for half-siblings, confirming causes for additive genetic variance overestimation, while the relationships among members of unrelated families clustered around the expected 0.0 (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>). The availability of dense genomic marker panels made it possible to genotype individuals for a large number of SNPs and obtain the realized genomic relationship matrix (<strong><em>G</em></strong>-matrix) among these individuals. In turn, the <strong><em>G</em></strong>-matrix can be used as a substitute to the <strong><em>A</em></strong>-matrices to estimate more accurate and precise genetic variance components as the <strong><em>G</em></strong>-matrix represents the realized pedigree as well as having the capacity to exploit the Mendelian sampling/segregation within families (<a id="xref-ref-62-3" class="xref-bibr" href="#ref-62">VanRaden 2008</a>; <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Hayes <em>et al.</em> 2009</a>). It is worthwhile to note that some of the recently reported gain increase in animal breeding programs is mainly due to exploiting the Mendelian sampling term (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Avendano <em>et al.</em> 2004</a>).</p><p id="p-34">The utility of the <strong><em>G</em></strong>-matrix in generating improved estimates of the genetic variance parameters from experimental populations of forest trees (<em>e.g.</em>, full-sib families) has recently been explored (<a id="xref-ref-66-2" class="xref-bibr" href="#ref-66">Zapata-Valenzuela <em>et al.</em> 2013</a>; <a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Klápště <em>et al.</em> 2014</a>; <a id="xref-ref-48-4" class="xref-bibr" href="#ref-48">Muñoz <em>et al.</em> 2014</a>). The present study, to our knowledge, represents the first attempt to implement the <strong><em>G</em></strong>-matrix in OP family testing, thus not only overcoming the common bias associated with the unfulfilled half-sib assumption, but also separating the additive from the nonadditive genetic variance components. It is well known that separating the additive from the nonadditive (dominance and epistatic variances) genetic components requires elaborate mating designs with large numbers of interconnected full-sib families coupled with the inclusion of replicated clonal material (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Foster and Shaw 1988</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bradshaw and Foster 1992</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Costa <em>et al.</em> 2009</a>). Our study accomplished a mixed-model approach for variance decomposition, providing realized estimates of the additive, dominance, and epistatic genetic variances without the need for mating designs to generate interconnected full-sib families or vegetative propagation for the production of replicated clonal material.</p><p id="p-35">It is interesting to note that the estimated additive genetic variances for the three realized genomic relationship matrix-based analyses (GBLUP-A, GBLUP-AD, and GBLUP-ADE) were lower than those of the average numerator relationship matrix (ABLUP) (<a id="xref-table-wrap-1-14" class="xref-table" href="#T1">Table 1</a>), an observation already reported for mice (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Lee <em>et al.</em> 2010</a>), loblolly pine (<a id="xref-ref-48-5" class="xref-bibr" href="#ref-48">Muñoz <em>et al.</em> 2014</a>), and Brown Swiss cattle populations (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Loberg <em>et al.</em> 2015</a>). The improved performance of the GBLUP-A compared to that of the ABLUP indicates that the former model took full advantage of: (1) the within-family variation (<em>i.e.</em>, Mendelian sampling term), (2) discerning whether full-sibs, self-sibs, and self-halfs existed within the studied 214 OP families, (3) the ability to estimate among-family relationships (even if they were as small as those seen in <a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a> and <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>) identifying pedigree errors if present, as shown in <a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1</a> (<em>i.e.</em>, some individuals have a coefficient of relationship of 0.0 within the same OP family). The observed reduction in the additive genetic variance between the two models (ABLUP <em>vs.</em> GBLUP-A) resulted in concomitant increase in the residual error terms and hence considerably reduced narrow-sense heritability estimates for height (0.25 <em>vs.</em> 0.16) and also for wood density (0.61 <em>vs.</em> 0.30) along with improvement in the model fit based on improved AIC values (<a id="xref-table-wrap-1-15" class="xref-table" href="#T1">Table 1</a>). Additionally, GBLUP-A produced greater precision for its estimated breeding value (EBV) as indicated by the EBV’s smaller SEs compared to the ABLUP (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>; ABLUP <em>vs.</em> GBLUP-A).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/3/743/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Ranking plots for the top 50 performing white spruce individuals for height (left) and wood density (right), respectively, comparing results of ABLUP vs. GBLUP-ADE assessments (note the number of highly ranked individuals in the ABLUP that dropped from the top 50 in the GBLUP-ADE)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2082364622" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Ranking plots for the top 50 performing white spruce individuals for height (left) and wood density (right), respectively, comparing results of ABLUP &lt;em&gt;vs.&lt;/em&gt; GBLUP-ADE assessments (note the number of highly ranked individuals in the ABLUP that dropped from the top 50 in the GBLUP-ADE).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/3/743/F4.medium.gif" width="440" height="266"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/6/3/743/F4.medium.gif" width="440" height="266"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/3/743/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/3/743/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467646" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-36" class="first-child">Ranking plots for the top 50 performing white spruce individuals for height (left) and wood density (right), respectively, comparing results of ABLUP <em>vs.</em> GBLUP-ADE assessments (note the number of highly ranked individuals in the ABLUP that dropped from the top 50 in the GBLUP-ADE).</p><div class="sb-div caption-clear"></div></div></div><p id="p-37">It is noteworthy to mention that the present study is based on data collected from one site, thus there is a chance that the estimated genetic parameters could be upwards biased due to the genotype × environment confounding effects specific to this particular site or year. However, results from similar analyses for the same species and attributes for a set of 25 OP families planted in replicated trials over three sites in British Columbia were consistent with those reported here with the added benefit of estimating the additive and dominance × site (environment) interactions (O. Gamal El-Dien, unpublished data).</p><p id="p-38">The observed overall trend in genetic variance decomposition persisted when the dominance genetic variance was estimated using the alternative genotypic approach proposed by <a id="xref-ref-59-2" class="xref-bibr" href="#ref-59">Su <em>et al.</em> (2012)</a> and discussed by <a id="xref-ref-64-2" class="xref-bibr" href="#ref-64">Vitezica <em>et al.</em> (2013)</a>; however, the dominance genetic variance of wood density showed a slight increase (Supporting Information, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025957/-/DC1/TableS1.pdf">Table S1</a>).</p><p id="p-39">Additionally, estimating the dominance genetic variance is only feasible when full-sib families are available (<a id="xref-ref-66-3" class="xref-bibr" href="#ref-66">Zapata-Valenzuela <em>et al.</em> 2013</a>; <a id="xref-ref-40-3" class="xref-bibr" href="#ref-40">Klápště <em>et al.</em> 2014</a>; <a id="xref-ref-48-6" class="xref-bibr" href="#ref-48">Muñoz <em>et al.</em> 2014</a>). This scenario is easily resolved when pedigree- and marker-based models are compared for mating design accommodating full-sib families (<a id="xref-ref-48-7" class="xref-bibr" href="#ref-48">Muñoz <em>et al.</em> 2014</a>). However, the utility of the GBLUP-AD model in OP family testing is still worth exploring to discern the dominance genetic variance – if existing – as well as separating the genetic variances from the confounding environment effects. It should be noted that the goodness-of-fit statistics (AIC) for the GBLUP-AD clearly indicated that adding the dominance genetic variance resulted in model over fit and this is expected due to the extremely small and nonsignificant dominance genetic variance (1.1 and 2.8% for height and wood density, respectively).</p><p id="p-40">The model that included the additive, dominance, and epistatic variances (GBLUP-ADE) offered better partitioning of the variance complements, as the additive × additive epistatic variance became extremely pronounced and accounted for 11 and 52% of the total variance for height and wood density, respectively (<a id="xref-table-wrap-1-16" class="xref-table" href="#T1">Table 1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025957/-/DC1/TableS1.pdf">Table S1</a>). When we removed the dominance genetic variance from the GBLUP-ADE model, the revised models (GBLUP-AE) produced better model fit for height (17,466.9 <em>vs.</em> 14,472.9) and wood density (−9,732.6 <em>vs.</em> −9,726.6), confirming that dominance variance was negligible (<a id="xref-table-wrap-1-17" class="xref-table" href="#T1">Table 1</a>). Interestingly, both models (GBLUP-ADE and GBLUP-AE) produced similar variance components apportionment and heritability estimates (<a id="xref-table-wrap-1-18" class="xref-table" href="#T1">Table 1</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025957/-/DC1/FigureS1.pdf">Figure S1</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025957/-/DC1/FigureS2.pdf">Figure S2</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025957/-/DC1/FigureS3.pdf">Figure S3</a>). Similar magnitude of the additive × additive epistatic variance to that of the additive variance, <em>per se</em>, was also observed in loblolly pine (<a id="xref-ref-48-8" class="xref-bibr" href="#ref-48">Muñoz <em>et al.</em> 2014</a>), a situation meeting theoretical expectations where the additive × additive epistatic variance is commonly absorbed by both the additive and the residual variances (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Lynch and Walsh 1998</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Jannink 2007</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Mackay 2014</a>). The power of the GBLUP-ADE and/or GBLUP-AE models in identifying and separating the additive × additive epistasis from the additive genetic variance lies in the genetic background of the tested families for providing a range of options to demonstrate all established interactions between the alleles at the various loci that are affecting the studied traits. The magnitude of the epistatic additive × additive genetic variance observed for height and wood density along with the AIC values produced from the tested models require some reflection. The observed AIC values support GBLUP-A and GBLUP-AE to be the best model for height and wood density, respectively (<a id="xref-table-wrap-1-19" class="xref-table" href="#T1">Table 1</a>). However, in wood density where the additive × additive is ≈3 times that of the additive variance, the prediction accuracy of the GBLUP-AE and GBLUP-A models were almost similar (<a id="xref-table-wrap-1-20" class="xref-table" href="#T1">Table 1</a> and <a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>). This indicates that the additive × additive and additive relationship matrices are in a “tug-of-war” state over the same variance. In fact, we estimated the correlation between these two relationship matrices and it was close to perfect correlation (r = 0.988), confirming our notion and makes us believe that while we observed exceedingly large epistatic additive × additive genetic variance, the impact on predicting the BVs between GBLUP-A and GBLUP-AE is similar (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025957/-/DC1/FigureS4.pdf">Figure S4</a>).</p><p id="p-41">The subject of genetic epistasis is controversial as all variance components, including epistasis, are dependent on the allele frequencies in the studied population. Thus, epistasis could have an allusive and unique effect across different scenarios (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Hill <em>et al.</em> 2008</a>; <a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Mackay 2014</a>). The role of epistasis on the genetic architecture of quantitative traits is still not clearly determined due to several discrepancies between statistical and functional definition of epistasis. The statistical approach considers the epistatic variance orthogonal to the additive genetic variance and assumes a clear determination (separation) of both components by the implementation of independent terms in the model. Moreover, the epistatic effects are transient and disappear by breaking of linkage disequilibrium (LD) (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Hill <em>et al.</em> 2008</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Crow 2008</a>, <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">2010</a>). The functional approach assumes that the allelic substitution effect depends on the genetic background. <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Hill <em>et al.</em> (2008)</a> based their empirical evidence on an exhaustive review across a wide range of species. This includes comparisons between narrow- and broad-sense heritability estimates, concluding that complex traits are mainly controlled by additive genetic variance as most studied cases supported the notion that the majority of the genetic variance appeared to be additive (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Crow 2010</a>). However, in the present study, if we utilized the heritability estimates derived from the GBLUP-A or GBLUP-AD models alone (without the application of the GBLUP-ADE and/or GBLUP-AE models), then the part of the genetic variance attributable to additive × additive interaction would have been excluded from the calculations, and thus our conclusion would have been mainly based on inflated additive genetic variance. Clearly, the utilization of the marker-based relationship method enabled disentangling the additive from the nonadditive genetic component, while effectively accounting for the proper environment variance through the removal of possible confounding effects. Such methodology provides much more realistic breeding value estimations for an individual.</p><p id="p-42"><a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Habier <em>et al.</em> (2007</a>, <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">2010</a>, <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">2013</a>) indicated that the realized genomic relationships do not only capture relatedness among individuals but also the LD between SNPs and QTL, the deviation from independent segregation of alleles on the same gamete if the loci are linked (cosegregation or classical linkage), as well as the additive genetic relationship. <a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Habier <em>et al.</em> (2013)</a> demonstrated that these types of information collectively have different effects on the accuracy of the EBV. As a result, it is safe to state that there is more to the realized genomic relationship than the straightforward accounting for the Mendelian sampling term, hence resulting in the superior decomposition of the genetic variance components and BV estimation.</p><p id="p-43">When ABLUP is used to estimate the genetic variance components from either half- or full-sib families, the above factors are barely considered, except those that were captured through common ancestry. As indicted above, the OP/half-sib structure is incapable of estimating the dominance and the epistatic genetic variances. This situation was clearly demonstrated in the study by <a id="xref-ref-48-9" class="xref-bibr" href="#ref-48">Muñoz <em>et al.</em> (2014)</a>, as more accurate breeding value estimates and effective partitioning of variance components were obtained from their single site, full-sib, and clonally replicated loblolly pine experiment. The present study, on the other hand, demonstrated the power of the realized genomic relationship in quantitative genetic analyses using a more challenging structure (OP families). As proof-of-concept we compared the rank order among the top 50 performing individuals based on the conventional ABLUP <em>vs.</em> the GBLUP-ADE/GBLUP-AE (see details in the interaction plots of <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>). Only 23 and 33 of the top 50 individuals persisted from ABLUP to GBLUP-ADE/GBLUP-AE for height and wood density, respectively, and overall, the individuals’ ranking among the top 50 trees dramatically changed from ABLUP to GBLUP-AE. Interestingly, for the 10 best performing trees, only 2 and 4 individuals persisted for height and wood density, respectively (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a>). The true EBV of an individual is commonly determined from experiments with deep pedigree with ample connectedness; however, when the ABLUP approach is used in forestry progeny testing experiments that are characterized by shallow and inadequate connectedness, then the obtained breeding value is expected to greatly deviate from its true value as the assumption of mixed models of error-free covariance matrices is not met (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Mrode 2014</a>). The greatest difference between the GBLUP and ABLUP models is the ability of the former to more precisely define the genetic relationship between any two individuals as compared to the latter (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1</a>). Our model’s cross-validation supports this notion as the prediction accuracy for the GBLUP models was greater than those produced by ABLUP, regardless of the folding scenario (<a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2</a>; diagonals). This difference is due to the quantity of realized pairwise genetic relationship information used for prediction, where in ABLUP only the information from the pedigreed OP family is used to predict the breeding value. Conversely in GBLUP all information from related individuals are used regardless of family assignment; this can be seen in the family folding scenario (<a id="xref-table-wrap-2-5" class="xref-table" href="#T2">Table 2</a>). Thus, we believe that the EBVs produced by the GBLUP models are closer to the true value as more information is used.</p><p id="p-44">We feel that our results have important and immediate implications for tree improvement programs in forestry as most programs are long-term and resource-dependent (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">El-Kassaby 1993</a>). The conventional genetic improvement in forestry follows the classical recurrent selection scheme with repeated cycles of selection, breeding, and testing over time and space (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Gamal El-Dien <em>et al.</em> 2015</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Ratcliffe <em>et al.</em> 2015</a>). These programs often include: (1) phenotypic selection of untested candidate parents from natural or managed forests, (2) propagation of the selected parents as grafts followed by a period of inactivity until sexual maturity, (3) the sexual production of structure pedigreed offspring from the selected parents using a specific mating design, (4) field testing over vast geographical areas for a reasonable period to attain meaningful data for target traits, (5) estimation of genetic parameters and ranking of individuals based on their BVs, and (6) genotypic selection of superior individuals for the second round of breeding and/or seed production from seed orchard populations. Obviously, the completion of a single breeding–testing–selection cycle is a protracted endeavor due to several uncontrollable biological factors; namely, the time needed for reaching sexual maturity for structured pedigree production and reproductive phenology and fertility variation that hinder the mating design completion (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">El-Kassaby <em>et al.</em> 1984</a>, <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">1989</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">El-Kassaby and Barclay 1992</a>). Therefore, the use of OP family testing, as demonstrated in the present study, allows immediate testing and the evaluation of a large number of individuals using their naturally produced offspring through wind pollination without the need for structured pedigree. The present study also demonstrated the utility of the realized genomic relationship approach in providing a simple and extremely efficient method for generating accurate genetic parameters from a simple OP testing that is characterized by shallow genealogy that is typical of most forest tree testing populations. It is noteworthy to mention that the use of the realized genomic relationship also allowed the generation of genetic parameters comparable to those generated only from elaborate mating designs coupled with cloning approaches. In conclusion, the utility of the realized genomic relationship in a simple, yet extremely efficient testing method, such as OP families, cannot be overlooked and calls for the re-evaluation of present-day conventional elaborate testing methods that are incapable of providing the genetic information produced in the present study.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-45">We thank <a id="xref-ref-5-7" class="xref-bibr" href="#ref-5">Beaulieu <em>et al.</em> (2014)</a> for making the data available on the Dryad Digital Repository: doi: 10.5061/ dryad.6rd6f and the two anonymous reviewers and J. Holland, Associate Editor, for helpful and constructive comments. This study was funded by the Johnson’s Family Forest Biotechnology Endowment, FPInnovations’ ForValueNet, and the Natural Sciences and Engineering Research Council of Canada (NSERC) Discovery Grant to Y.A.E. The authors declare no conflict of interest.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-supplementary-material" id="fn-7"><p id="p-46">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025957/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025957/-/DC1</a></p></li><li class="fn" id="fn-8"><p id="p-47"><em>Communicating editor: J. B. Holland</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2015-12-10"><span class="received-label">Received </span>December 10, 2015.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2016-01-14"><span class="accepted-label">Accepted </span>January 14, 2016.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2016 El-Dien <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-3">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.743.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Askew</span>  <span class="cit-name-given-names">G. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">El-Kassaby</span>  <span class="cit-name-given-names">Y. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Estimation of relationship coefficients among progeny derived from wind-pollinated orchard seeds.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">88</span>: <span class="cit-fpage">267</span>–<span class="cit-lpage">272</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D88%26rft.spage%253D267%26rft_id%253Dinfo%253Apmid%252F24185937%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=24185937&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F743.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.743.2" data-doi="10.1017/S0016672303006566"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Avendano</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Woolliams</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Villanueva</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Mendelian sampling terms as a selective advantage in optimum breeding schemes with restrictions on the rate of inbreeding.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">83</span>: <span class="cit-fpage">55</span>–<span class="cit-lpage">64</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DAvenda%25C3%25B1o%26rft.auinit1%253DS.%26rft.volume%253D83%26rft.issue%253D1%26rft.spage%253D55%26rft.epage%253D64%26rft.atitle%253DMendelian%2Bsampling%2Bterms%2Bas%2Ba%2Bselective%2Badvantage%2Bin%2Boptimum%2Bbreeding%2Bschemes%2Bwith%2Brestrictions%2Bon%2Bthe%2Brate%2Bof%2Binbreeding.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672303006566%26rft_id%253Dinfo%253Apmid%252F15125067%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672303006566&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15125067&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F743.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000220923200007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.743.3" data-doi="10.1371/journal.pone.0003376"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baird</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Etter</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Atwood</span>  <span class="cit-name-given-names">T. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Currey</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shiver</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Rapid SNP discovery and genetic mapping using sequenced RAD markers.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">e3376</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DBaird%26rft.auinit1%253DN.%2BA.%26rft.volume%253D3%26rft.issue%253D10%26rft.spage%253De3376%26rft.epage%253De3376%26rft.atitle%253DRapid%2BSNP%2Bdiscovery%2Band%2Bgenetic%2Bmapping%2Busing%2Bsequenced%2BRAD%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0003376%26rft_id%253Dinfo%253Apmid%252F18852878%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0003376&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18852878&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F743.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.743.4" data-doi="10.1534/genetics.110.125781"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beaulieu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doerksen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boyle</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clément</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deslauriers</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Association genetics of wood physical traits in the conifer white spruce and relationships with gene expression.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">188</span>: <span class="cit-fpage">197</span>–<span class="cit-lpage">214</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.125781%26rft_id%253Dinfo%253Apmid%252F21385726%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg4LzEvMTk3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8zLzc0My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.743.5" data-doi="10.1038/hdy.2014.36"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beaulieu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doerksen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clément</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bousquet</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Accuracy of genomic selection models in a large population of open-pollinated families in white spruce.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">113</span>: <span class="cit-fpage">343</span>–<span class="cit-lpage">352</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D113%26rft.spage%253D343%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2014.36%26rft_id%253Dinfo%253Apmid%252F24781808%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2014.36&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24781808&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F743.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000342564000008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.743.6" data-doi="10.1139/x92-139"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bradshaw</span>  <span class="cit-name-given-names">H. D.</span>  <span class="cit-name-suffix">Jr</span>.</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foster</span>  <span class="cit-name-given-names">G. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Marker-aided selection and propagation system in trees: advantages of cloning for studying quantitative inheritance.</span> <abbr class="cit-jnl-abbrev">Can. J. For. Res.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">1044</span>–<span class="cit-lpage">1049</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCan.%2BJ.%2BFor.%2BRes.%26rft.volume%253D22%26rft.spage%253D1044%26rft_id%253Dinfo%253Adoi%252F10.1139%252Fx92-139%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1139/x92-139&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.743.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burdon</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shelbourne</span>  <span class="cit-name-given-names">C. J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1971</span> <span class="cit-article-title">Breeding populations for recurrent selection: conflicts and possible solutions.</span> <abbr class="cit-jnl-abbrev">N. Z. J. For. Sci.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">174</span>–<span class="cit-lpage">193</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DN.%2BZ.%2BJ.%2BFor.%2BSci.%26rft.volume%253D1%26rft.spage%253D174%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.743.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Callaham</span>  <span class="cit-name-given-names">R. Z.</span></span></li></ol><cite>, <span class="cit-pub-date">1964</span> <span class="cit-article-title">Provenance research: investigation of genetic diversity associated with geography.</span> <abbr class="cit-jnl-abbrev">Unasylva</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">40</span>–<span class="cit-lpage">50</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DUnasylva%26rft.volume%253D18%26rft.spage%253D40%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.743.9" data-doi="10.1007/s11295-013-0657-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mitchell</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buckler</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">El-Kassaby</span>  <span class="cit-name-given-names">Y. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Mining conifers’ mega-genome using rapid and efficient multiplexed high-throughput genotyping-by-sequencing (GBS) SNP discovery platform.</span> <abbr class="cit-jnl-abbrev">Tree Genet. Genomes</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">1537</span>–<span class="cit-lpage">1544</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTree%2BGenet.%2BGenomes%26rft.volume%253D9%26rft.spage%253D1537%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11295-013-0657-1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11295-013-0657-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.743.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Costa</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Silva</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borralho</span>  <span class="cit-name-given-names">N. M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Botts</span>  <span class="cit-name-given-names">B. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Additive and non-additive genetic parameters from clonally replicated and seedling progenies of <em>Eucalyptus globulus</em>.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">1113</span>–<span class="cit-lpage">1119</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D108%26rft.spage%253D1113%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.743.11" data-doi="10.1007/s12041-008-0057-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crow</span>  <span class="cit-name-given-names">J. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Maintaining evolvability.</span> <abbr class="cit-jnl-abbrev">J. Genet.</abbr> <span class="cit-vol">87</span>: <span class="cit-fpage">349</span>–<span class="cit-lpage">353</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bgenetics%26rft.stitle%253DJ%2BGenet%26rft.aulast%253DCrow%26rft.auinit1%253DJ.%2BF.%26rft.volume%253D87%26rft.issue%253D4%26rft.spage%253D349%26rft.epage%253D353%26rft.atitle%253DMaintaining%2Bevolvability.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs12041-008-0057-8%26rft_id%253Dinfo%253Apmid%252F19147924%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s12041-008-0057-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19147924&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F743.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000262851900005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.743.12" data-doi="10.1098/rstb.2009.0275"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crow</span>  <span class="cit-name-given-names">J. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">On epistasis: why it is unimportant in polygenic directional selection.</span> <abbr class="cit-jnl-abbrev">Philos. Trans. R. Soc. London Ser. B.</abbr> <span class="cit-vol">365</span>: <span class="cit-fpage">1241</span>–<span class="cit-lpage">1244</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhilosophical%2BTransactions%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DPhil%2BTrans%2BR%2BSoc%2BB%26rft.aulast%253DCrow%26rft.auinit1%253DJ.%2BF.%26rft.volume%253D365%26rft.issue%253D1544%26rft.spage%253D1241%26rft.epage%253D1244%26rft.atitle%253DOn%2Bepistasis%253A%2Bwhy%2Bit%2Bis%2Bunimportant%2Bin%2Bpolygenic%2Bdirectional%2Bselection%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.2009.0275%26rft_id%253Dinfo%253Apmid%252F20308099%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoicm95cHRiIjtzOjU6InJlc2lkIjtzOjEzOiIzNjUvMTU0NC8xMjQxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8zLzc0My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.743.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">El-Kassaby</span>  <span class="cit-name-given-names">Y. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Evaluation of the tree-improvement delivery system: factors affecting genetic potential.</span> <abbr class="cit-jnl-abbrev">Tree Physiol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">545</span>–<span class="cit-lpage">550</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTree%2BPhysiol.%26rft.volume%253D15%26rft.spage%253D545%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.743.14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">El-Kassaby</span>  <span class="cit-name-given-names">Y. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barclay</span>  <span class="cit-name-given-names">H. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Cost of reproduction in Douglas-fir.</span> <abbr class="cit-jnl-abbrev">Can. J. Bot.</abbr> <span class="cit-vol">70</span>: <span class="cit-fpage">1429</span>–<span class="cit-lpage">1432</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCan.%2BJ.%2BBot.%26rft.volume%253D70%26rft.spage%253D1429%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.743.15" data-doi="10.1016/0378-1127(82)90009-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">El-Kassaby</span>  <span class="cit-name-given-names">Y. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sziklai</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">1982</span> <span class="cit-article-title">Genetic variation of allozyme and quantitative traits in a selected Douglas-fir [<em>Pseudotsuga menziesii</em> var. <em>menziesii</em> (Mirb.) Franco] population.</span> <abbr class="cit-jnl-abbrev">For. Ecol. Manage.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">115</span>–<span class="cit-lpage">126</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFor.%2BEcol.%2BManage.%26rft.volume%253D4%26rft.spage%253D115%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0378-1127%252882%252990009-3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0378-1127(82)90009-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.743.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">El-Kassaby</span>  <span class="cit-name-given-names">Y. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fashler</span>  <span class="cit-name-given-names">A. M. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sziklai</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title">Reproductive phenology and its impact on genetically improved seed production in a Douglas-fir seed orchard.</span> <abbr class="cit-jnl-abbrev">Silvae Genet.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">120</span>–<span class="cit-lpage">125</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSilvae%2BGenet.%26rft.volume%253D33%26rft.spage%253D120%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.743.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">El-Kassaby</span>  <span class="cit-name-given-names">Y. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fashler</span>  <span class="cit-name-given-names">A. M. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crown</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Variation in fruitfulness in a Douglas-fir seed orchard and its effect on crop-management decisions.</span> <abbr class="cit-jnl-abbrev">Silvae Genet.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">113</span>–<span class="cit-lpage">121</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSilvae%2BGenet.%26rft.volume%253D38%26rft.spage%253D113%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.743.18" data-doi="10.1007/s11056-012-9338-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">El-Kassaby</span>  <span class="cit-name-given-names">Y. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klápště</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guy</span>  <span class="cit-name-given-names">R. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Breeding without breeding: selection using the genomic best linear unbiased predictor method (GBLUP).</span> <abbr class="cit-jnl-abbrev">New For.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">631</span>–<span class="cit-lpage">637</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BFor.%26rft.volume%253D43%26rft.spage%253D631%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11056-012-9338-4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11056-012-9338-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.743.19" data-doi="10.1371/journal.pone.0019379"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawamoto</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e19379</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DElshire%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253De19379%26rft.epage%253De19379%26rft.atitle%253DA%2Brobust%252C%2Bsimple%2Bgenotyping-by-sequencing%2B%2528GBS%2529%2Bapproach%2Bfor%2Bhigh%2Bdiversity%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0019379%26rft_id%253Dinfo%253Apmid%252F21573248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0019379&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21573248&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F743.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.743.20" data-doi="10.1007/BF00303527"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Foster</span>  <span class="cit-name-given-names">G. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shaw</span>  <span class="cit-name-given-names">D. V.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">Using clonal replications to explore genetic variation in a perennial plant species.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">76</span>: <span class="cit-fpage">788</span>–<span class="cit-lpage">794</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D76%26rft.spage%253D788%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00303527%26rft_id%253Dinfo%253Apmid%252F24232359%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00303527&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24232359&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F743.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.743.21" data-doi="10.1098/rspb.2007.1032"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Frentiu</span>  <span class="cit-name-given-names">F. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clegg</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chittock</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burke</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blows</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Pedigree-free animal models: the relatedness matrix reloaded</span>. <abbr class="cit-jnl-abbrev">P. Roy. Soc. B-Biol. Sci.</abbr> <span class="cit-vol">275</span>: <span class="cit-fpage">639</span>–<span class="cit-lpage">647</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DP.%2BRoy.%2BSoc.%2BB-Biol.%2BSci.%26rft.volume%253D275%26rft.spage%253D639%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.2007.1032%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rspb.2007.1032&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.743.22" data-doi="10.1186/s12864-015-1597-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gamal El-Dien</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ratcliffe</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klápště</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Porth</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Prediction accuracies for growth and wood attributes of interior spruce in space using genotyping-by-sequencing.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">370</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D16%26rft.spage%253D370%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-015-1597-y%26rft_id%253Dinfo%253Apmid%252F25956247%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-015-1597-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25956247&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F743.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.743.23" data-doi="10.1371/journal.pone.0066983"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gay</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Siol</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ronfort</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Pedigree-free estimates of heritability in the wild: promising prospects for selfing populations.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e66983</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De66983%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0066983%26rft_id%253Dinfo%253Apmid%252F23825602%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0066983&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23825602&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F743.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.743.24" data-doi="10.2307/2533274"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gilmour</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cullis</span>  <span class="cit-name-given-names">B. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Average information REML: an efficient algorithm for variance parameter estimation in linear mixed models.</span> <abbr class="cit-jnl-abbrev">Biometrics</abbr> <span class="cit-vol">51</span>: <span class="cit-fpage">1440</span>–<span class="cit-lpage">1450</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrics%26rft.volume%253D51%26rft.spage%253D1440%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2533274%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2533274&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1995TQ15600022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-book" id="cit-6.3.743.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gilmour</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gogel</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cullis</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-source">ASReml User Guide Release 3.0</span>, <span class="cit-publ-name">VSN International Ltd</span>, <span class="cit-publ-loc">Hemel Hempstead, UK</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.743.26" data-doi="10.1534/genetics.107.081190"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Habier</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dekkers</span>  <span class="cit-name-given-names">J. C. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The impact of genetic relationship information on genome-assisted breeding values.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">177</span>: <span class="cit-fpage">2389</span>–<span class="cit-lpage">2397</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHabier%26rft.auinit1%253DD.%26rft.volume%253D177%26rft.issue%253D4%26rft.spage%253D2389%26rft.epage%253D2397%26rft.atitle%253DThe%2BImpact%2Bof%2BGenetic%2BRelationship%2BInformation%2Bon%2BGenome-Assisted%2BBreeding%2BValues%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.081190%26rft_id%253Dinfo%253Apmid%252F18073436%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Ny80LzIzODkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzMvNzQzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.743.27" data-doi="10.1186/1297-9686-42-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Habier</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tetens</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seefried</span>  <span class="cit-name-given-names">F. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lichtner</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thaller</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The impact of genetic relationship information on genomic breeding values in German Holstein cattle.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">5</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DHabier%26rft.auinit1%253DD.%26rft.volume%253D42%26rft.spage%253D5%26rft.epage%253D5%26rft.atitle%253DThe%2Bimpact%2Bof%2Bgenetic%2Brelationship%2Binformation%2Bon%2Bgenomic%2Bbreeding%2Bvalues%2Bin%2BGerman%2BHolstein%2Bcattle.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-42-5%26rft_id%253Dinfo%253Apmid%252F20170500%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-42-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20170500&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F743.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.743.28" data-doi="10.1534/genetics.113.152207"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Habier</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garrick</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomic BLUP decoded: a look into the black box of genomic prediction.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">194</span>: <span class="cit-fpage">597</span>–<span class="cit-lpage">607</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.152207%26rft_id%253Dinfo%253Apmid%252F23640517%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk0LzMvNTk3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8zLzc0My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.743.29" data-doi="10.1007/s11295-012-0588-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hallingback</span>  <span class="cit-name-given-names">H. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jansson</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genetic information from progeny trials: a comparison between progenies generated by open pollination and by controlled crosses.</span> <abbr class="cit-jnl-abbrev">Tree Genet. Genomes</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">731</span>–<span class="cit-lpage">740</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTree%2BGenet.%2BGenomes%26rft.volume%253D9%26rft.spage%253D731%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11295-012-0588-2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11295-012-0588-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.743.30" data-doi="10.1017/S0016672308009981"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Visscher</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Increased accuracy of artificial selection by using the realized relationship matrix.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">47</span>–<span class="cit-lpage">60</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DHayes%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D91%26rft.issue%253D1%26rft.spage%253D47%26rft.epage%253D60%26rft.atitle%253DIncreased%2Baccuracy%2Bof%2Bartificial%2Bselection%2Bby%2Busing%2Bthe%2Brealized%2Brelationship%2Bmatrix.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672308009981%26rft_id%253Dinfo%253Apmid%252F19220931%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672308009981&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19220931&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F743.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000264226300006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.743.31" data-doi="10.2307/2529430"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Henderson</span>  <span class="cit-name-given-names">C. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1975</span> <span class="cit-article-title">Best linear unbiased estimation and prediction under a selection model.</span> <abbr class="cit-jnl-abbrev">Biometrics</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">423</span>–<span class="cit-lpage">447</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrics%26rft.stitle%253DBiometrics%26rft.aulast%253DHenderson%26rft.auinit1%253DC.%2BR.%26rft.volume%253D31%26rft.issue%253D2%26rft.spage%253D423%26rft.epage%253D447%26rft.atitle%253DBest%2Blinear%2Bunbiased%2Bestimation%2Band%2Bprediction%2Bunder%2Ba%2Bselection%2Bmodel.%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2529430%26rft_id%253Dinfo%253Apmid%252F1174616%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2529430&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1174616&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F743.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1975AF09900010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.743.32" data-doi="10.2307/2529339"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Henderson</span>  <span class="cit-name-given-names">C. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1976</span> <span class="cit-article-title">Simple method for computing inverse of a numerator relationship matrix used in prediction of breeding values.</span> <abbr class="cit-jnl-abbrev">Biometrics</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">69</span>–<span class="cit-lpage">83</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrics%26rft.volume%253D32%26rft.spage%253D69%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2529339%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2529339&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1976BM07600004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-book" id="cit-6.3.743.33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Henderson</span>  <span class="cit-name-given-names">C. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-source">Applications of Linear Models in Animal Breeding</span>. <span class="cit-publ-name">University of Guelph</span>, <span class="cit-publ-loc">Ontario</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.743.34" data-doi="10.1371/journal.pgen.1000008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">W. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Visscher</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Data and theory point to mainly additive genetic variance for complex traits.</span> <abbr class="cit-jnl-abbrev">PLoS Genet</abbr>. <span class="cit-vol">4</span>: <span class="cit-fpage">e1000008</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DHill%26rft.auinit1%253DW.%2BG.%26rft.volume%253D4%26rft.issue%253D2%26rft.spage%253De1000008%26rft.epage%253De1000008%26rft.atitle%253DData%2Band%2Btheory%2Bpoint%2Bto%2Bmainly%2Badditive%2Bgenetic%2Bvariance%2Bfor%2Bcomplex%2Btraits.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000008%26rft_id%253Dinfo%253Apmid%252F18454194%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18454194&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F743.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.743.35" data-doi="10.1017/S0016672310000480"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">W. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weir</span>  <span class="cit-name-given-names">B. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Variation in actual relationship as a consequence of Mendelian sampling and linkage.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">47</span>–<span class="cit-lpage">64</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D93%26rft.spage%253D47%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672310000480%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672310000480&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.743.36" data-doi="10.1534/genetics.106.062992"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Identifying quantitative trait locus by genetic background interactions in association studies.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">176</span>: <span class="cit-fpage">553</span>–<span class="cit-lpage">561</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.062992%26rft_id%253Dinfo%253Apmid%252F17179077%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTc2LzEvNTUzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8zLzc0My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.743.37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jayawickrama</span>  <span class="cit-name-given-names">K. J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carson</span>  <span class="cit-name-given-names">M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">A breeding strategy for the New Zealand radiata pine breeding cooperative.</span> <abbr class="cit-jnl-abbrev">Silvae Genet.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">82</span>–<span class="cit-lpage">90</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSilvae%2BGenet.%26rft.volume%253D49%26rft.spage%253D82%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.743.38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">G. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Site-to-site genetic correlations and their implications on breeding zone size and optimum number of progeny test sites for coastal Douglas-fir.</span> <abbr class="cit-jnl-abbrev">Silvae Genet.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">280</span>–<span class="cit-lpage">285</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSilvae%2BGenet.%26rft.volume%253D46%26rft.spage%253D280%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.743.39" data-doi="10.1139/x91-028"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kiss</span>  <span class="cit-name-given-names">G. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yanchuk</span>  <span class="cit-name-given-names">A. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Preliminary evaluation of genetic variation of weevil resistance in interior spruce in British Columbia.</span> <abbr class="cit-jnl-abbrev">Can. J. For. Res.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">230</span>–<span class="cit-lpage">234</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCan.%2BJ.%2BFor.%2BRes.%26rft.volume%253D21%26rft.spage%253D230%26rft_id%253Dinfo%253Adoi%252F10.1139%252Fx91-028%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1139/x91-028&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.743.40" data-doi="10.1007/s11295-013-0673-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Klápště</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lstibůrek</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">El-Kassaby</span>  <span class="cit-name-given-names">Y. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Estimates of genetic parameters and breeding values from western larch open-pollinated families using marker-based relationship.</span> <abbr class="cit-jnl-abbrev">Tree Genet. Genomes</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">241</span>–<span class="cit-lpage">249</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTree%2BGenet.%2BGenomes%26rft.volume%253D10%26rft.spage%253D241%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11295-013-0673-1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11295-013-0673-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.743.41" data-doi="10.1007/s11295-013-0630-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Korecky</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klápště</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lstibůrek</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kobliha</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dana Nelson</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Comparison of genetic parameters from marker-based relationship, sibship, and combined models in Scots pine multi-site open-pollinated tests.</span> <abbr class="cit-jnl-abbrev">Tree Genet. Genomes</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">1227</span>–<span class="cit-lpage">1235</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTree%2BGenet.%2BGenomes%26rft.volume%253D9%26rft.spage%253D1227%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11295-013-0630-z%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11295-013-0630-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.743.42" data-doi="10.1186/1297-9686-42-22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Visscher</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van der Werf</span>  <span class="cit-name-given-names">J. H. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Using the realized relationship matrix to disentangle confounding factors for the estimation of genetic variance components of complex traits.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">22</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DLee%26rft.auinit1%253DS.%2BH.%26rft.volume%253D42%26rft.spage%253D22%26rft.epage%253D22%26rft.atitle%253DUsing%2Bthe%2Brealized%2Brelationship%2Bmatrix%2Bto%2Bdisentangle%2Bconfounding%2Bfactors%2Bfor%2Bthe%2Bestimation%2Bof%2Bgenetic%2Bvariance%2Bcomponents%2Bof%2Bcomplex%2Btraits.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-42-22%26rft_id%253Dinfo%253Apmid%252F20546624%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-42-22&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20546624&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F743.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.743.43"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lippert</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quon</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">E. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kadie</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Listgarten</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The benefits of selecting phenotype-specific variants for applications of mixed models in genomics.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">1815</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D3%26rft.spage%253D1815%26rft_id%253Dinfo%253Apmid%252F23657357%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=23657357&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F743.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.743.44" data-doi="10.1111/jbg.12142"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Loberg</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durr</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fikse</span>  <span class="cit-name-given-names">W. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jorjani</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crooks</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Estimates of genetic variance and variance of predicted genetic merits using pedigree or genomic relationship matrices in six Brown Swiss cattle populations for different traits.</span> <abbr class="cit-jnl-abbrev">J. Anim. Breed. Genet.</abbr><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/jbg.12142</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAnim.%2BBreed.%2BGenet.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fjbg.12142%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/jbg.12142&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-book" id="cit-6.3.743.45"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walsh</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-source">Genetics and Analysis of Quantitative Traits</span>. <span class="cit-publ-name">Sinauer Associates</span>, <span class="cit-publ-loc">Sunderland, MA</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.743.46" data-doi="10.1038/nrg3627"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>  <span class="cit-name-given-names">T. F. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Epistasis and quantitative traits: using model organisms to study gene-gene interactions.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">22</span>–<span class="cit-lpage">33</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D15%26rft.spage%253D22%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3627%26rft_id%253Dinfo%253Apmid%252F24296533%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3627&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24296533&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F743.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-book" id="cit-6.3.743.47"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mrode</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-source">Linear Models for the Prediction of Animal Breeding Values</span>. <span class="cit-publ-name">CAB International</span>, <span class="cit-publ-loc">Wallingford, Oxfordshire</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.743.48" data-doi="10.1534/genetics.114.171322"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Muñoz</span>  <span class="cit-name-given-names">P. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Resende</span>  <span class="cit-name-given-names">M. F. R.</span>  <span class="cit-name-suffix">Jr</span>.</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gezan</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Resende</span>  <span class="cit-name-given-names">M. D. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Unraveling additive from nonadditive effects using genomic relationship matrices.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">1759</span>–<span class="cit-lpage">1768</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.171322%26rft_id%253Dinfo%253Apmid%252F25324160%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5OC80LzE3NTkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzMvNzQzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.743.49"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Namkoong</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">1966</span> <span class="cit-article-title">Inbreeding effects on estimation of genetic additive variance.</span> <abbr class="cit-jnl-abbrev">For. Sci.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">8</span>–<span class="cit-lpage">13</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFor.%2BSci.%26rft.volume%253D12%26rft.spage%253D8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=A19667450900002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-book" id="cit-6.3.743.50"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Namkoong</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">H. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brouard</span>  <span class="cit-name-given-names">J. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-source">Tree Breeding: Principles and Strategies.</span> Monographs on Theoretical and Applied Genetics. <span class="cit-publ-name">Springer,</span> <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.743.51" data-doi="10.1371/journal.pone.0037135"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peterson</span>  <span class="cit-name-given-names">B. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weber</span>  <span class="cit-name-given-names">J. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kay</span>  <span class="cit-name-given-names">E. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fisher</span>  <span class="cit-name-given-names">H. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoekstra</span>  <span class="cit-name-given-names">H. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Double digest RADseq: an inexpensive method for de novo SNP discovery and genotyping in model and non-model species.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e37135</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DPeterson%26rft.auinit1%253DB.%2BK.%26rft.volume%253D7%26rft.issue%253D5%26rft.spage%253De37135%26rft.epage%253De37135%26rft.atitle%253DDouble%2Bdigest%2BRADseq%253A%2Ban%2Binexpensive%2Bmethod%2Bfor%2Bde%2Bnovo%2BSNP%2Bdiscovery%2Band%2Bgenotyping%2Bin%2Bmodel%2Band%2Bnon-model%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0037135%26rft_id%253Dinfo%253Apmid%252F22675423%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0037135&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22675423&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F743.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.743.52" data-doi="10.3835/plantgenome2012.06.0006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Endelman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dawson</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rutkoski</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic selection in wheat breeding using genotyping-by-sequencing.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">103</span>–<span class="cit-lpage">113</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D5%26rft.spage%253D103%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2012.06.0006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2012.06.0006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.743.53" data-doi="10.1111/nph.12014"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Porth</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klápště</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Skyba</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lai</span>  <span class="cit-name-given-names">B. S. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Geraldes</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Populus trichocarpa cell wall chemistry and ultrastructure trait variation, genetic control and genetic correlations.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">197</span>: <span class="cit-fpage">777</span>–<span class="cit-lpage">790</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytol.%26rft.volume%253D197%26rft.spage%253D777%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fnph.12014%26rft_id%253Dinfo%253Apmid%252F23278123%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/nph.12014&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23278123&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F743.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000313266000012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.743.54" data-doi="10.1007/s13595-013-0349-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ratcliffe</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hart</span>  <span class="cit-name-given-names">F. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klápště</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jaquish</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mansfield</span>  <span class="cit-name-given-names">S. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genetics of wood quality attributes in western larch.</span> <abbr class="cit-jnl-abbrev">Ann. For. Sci.</abbr> <span class="cit-vol">71</span>: <span class="cit-fpage">415</span>–<span class="cit-lpage">424</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BFor.%2BSci.%26rft.volume%253D71%26rft.spage%253D415%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs13595-013-0349-x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s13595-013-0349-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.743.55" data-doi="10.1038/hdy.2015.57."><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ratcliffe</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gamal El-Dien</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klápště</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Porth</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A comparison of genomic selection models across time in interior spruce (<em>Picea engelmannii</em> x <em>glauca</em>) using unordered SNP imputation methods.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">115</span>: <span class="cit-fpage">547</span>–<span class="cit-lpage">555</span><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/hdy.2015.57.</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D115%26rft.spage%253D547%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2015.57.%26rft_id%253Dinfo%253Apmid%252F26126540%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2015.57.&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26126540&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F743.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.743.56"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rigault</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boyle</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lepage</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cooke</span>  <span class="cit-name-given-names">J. E. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bousquet</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A white spruce gene catalog for conifer genome analyses.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">193</span>(<span class="cit-issue">157</span>): <span class="cit-fpage">14</span>–<span class="cit-lpage">28</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft.volume%253D193%26rft.spage%253D14%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.743.57" data-doi="10.1111/j.1365-294X.2010.04554.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Santure</span>  <span class="cit-name-given-names">A. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Syapley</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ball</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Birkhead</span>  <span class="cit-name-given-names">T. I. M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burke</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">On the use of large marker panels to estimate inbreeding and relatedness: empirical and simulation studies of a pedigreed zebra finch population types at 771 SNPs.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1439</span>–<span class="cit-lpage">1451</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DSanture%26rft.auinit1%253DA.%2BW.%26rft.volume%253D19%26rft.issue%253D7%26rft.spage%253D1439%26rft.epage%253D1451%26rft.atitle%253DOn%2Bthe%2Buse%2Bof%2Blarge%2Bmarker%2Bpanels%2Bto%2Bestimate%2Binbreeding%2Band%2Brelatedness%253A%2Bempirical%2Band%2Bsimulation%2Bstudies%2Bof%2Ba%2Bpedigreed%2Bzebra%2Bfinch%2Bpopulation%2Btyped%2Bat%2B771%2BSNPs.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2010.04554.x%26rft_id%253Dinfo%253Apmid%252F20149098%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2010.04554.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20149098&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F743.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000275761300016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.743.58"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Squillace</span>  <span class="cit-name-given-names">A. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1974</span> <span class="cit-article-title">Average genetic correlations among offspring from open-pollinated forest trees.</span> <abbr class="cit-jnl-abbrev">Silvae Genet.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">149</span>–<span class="cit-lpage">156</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSilvae%2BGenet.%26rft.volume%253D23%26rft.spage%253D149%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.743.59" data-doi="10.1371/journal.pone.0045293"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Su</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Christensen</span>  <span class="cit-name-given-names">O. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ostersen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henryon</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lund</span>  <span class="cit-name-given-names">M. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Estimating additive and non-additive genetic variances and predicting genetic merits using genome-wide dense single nucleotide polymorphism markers.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e45293</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DSu%26rft.auinit1%253DG.%26rft.volume%253D7%26rft.issue%253D9%26rft.spage%253De45293%26rft.epage%253De45293%26rft.atitle%253DEstimating%2Badditive%2Band%2Bnon-additive%2Bgenetic%2Bvariances%2Band%2Bpredicting%2Bgenetic%2Bmerits%2Busing%2Bgenome-wide%2Bdense%2Bsingle%2Bnucleotide%2Bpolymorphism%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0045293%26rft_id%253Dinfo%253Apmid%252F23028912%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0045293&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23028912&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F743.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.743.60" data-doi="10.1046/j.1420-9101.2002.00372.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coltman</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pemberton</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">The use of marker-based relationship information to estimate the heritability of body weight in a natural population: a cautionary tale.</span> <abbr class="cit-jnl-abbrev">J. Evol. Biol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">92</span>–<span class="cit-lpage">99</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BEvol.%2BBiol.%26rft.volume%253D15%26rft.spage%253D92%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1420-9101.2002.00372.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1420-9101.2002.00372.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000173865600011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.743.61" data-doi="10.1371/journal.pone.0037565"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Truong</span>  <span class="cit-name-given-names">H. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramos</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yalcin</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Ruiter</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van der Poel</span>  <span class="cit-name-given-names">H. J. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Sequence-based genotyping for marker discovery and co-dominant scoring in germplasm and populations.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e37565</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DTruong%26rft.auinit1%253DH.%2BT.%26rft.volume%253D7%26rft.issue%253D5%26rft.spage%253De37565%26rft.epage%253De37565%26rft.atitle%253DSequence-based%2Bgenotyping%2Bfor%2Bmarker%2Bdiscovery%2Band%2Bco-dominant%2Bscoring%2Bin%2Bgermplasm%2Band%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0037565%26rft_id%253Dinfo%253Apmid%252F22662172%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0037565&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22662172&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F743.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.743.62" data-doi="10.3168/jds.2007-0980"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient methods to compute genomic predictions.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">4414</span>–<span class="cit-lpage">4423</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVanRaden%26rft.auinit1%253DP.%2BM.%26rft.volume%253D91%26rft.issue%253D11%26rft.spage%253D4414%26rft.epage%253D4423%26rft.atitle%253DEfficient%2BMethods%2Bto%2BCompute%2BGenomic%2BPredictions%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2007-0980%26rft_id%253Dinfo%253Apmid%252F18946147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2007-0980&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18946147&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F743.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260277200035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.743.63" data-doi="10.1371/journal.pgen.0020041"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Visscher</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Medland</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferreira</span>  <span class="cit-name-given-names">M. A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morley</span>  <span class="cit-name-given-names">K. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Assumption-free estimation of heritability from genome-wide identity-by-descent sharing between full siblings.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">e41</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DVisscher%26rft.auinit1%253DP.%2BM.%26rft.volume%253D2%26rft.issue%253D3%26rft.spage%253De41%26rft.epage%253De41%26rft.atitle%253DAssumption-free%2Bestimation%2Bof%2Bheritability%2Bfrom%2Bgenome-wide%2Bidentity-by-descent%2Bsharing%2Bbetween%2Bfull%2Bsiblings.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.0020041%26rft_id%253Dinfo%253Apmid%252F16565746%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.0020041&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16565746&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F743.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.743.64" data-doi="10.1534/genetics.113.155176"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vitezica</span>  <span class="cit-name-given-names">Z. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Varona</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">On the additive and dominance and covariance of individuals within the genomic selection scope.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">195</span>: <span class="cit-fpage">1223</span>–<span class="cit-lpage">1230</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.155176%26rft_id%253Dinfo%253Apmid%252F24121775%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5NS80LzEyMjMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzMvNzQzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.743.65" data-doi="10.1086/279872"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1922</span> <span class="cit-article-title">Coefficients of inbreeding and relationship.</span> <abbr class="cit-jnl-abbrev">Am. Nat.</abbr> <span class="cit-vol">56</span>: <span class="cit-fpage">330</span>–<span class="cit-lpage">338</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BNat.%26rft.volume%253D56%26rft.spage%253D330%26rft_id%253Dinfo%253Adoi%252F10.1086%252F279872%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/279872&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000200856800029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.743.66"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zapata-Valenzuela</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whetten</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neale</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McKeand</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Isik</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomic estimated breeding values using genomic relationship matrices in a cloned population of loblolly pine</span>. <abbr class="cit-jnl-abbrev">G3 Genes Genom. Genet.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">909</span>–<span class="cit-lpage">916</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2BGenes%2BGenom.%2BGenet.%26rft.volume%253D3%26rft.spage%253D909%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/6/3/743.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/3/731" title="Differential Expression of Genes Involved in Host Recognition, Attachment, and Degradation in the Mycoparasite Tolypocladium ophioglossoides" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/3/755" title="Genome-Wide Investigation of MicroRNAs and Their Targets in Response to Freezing Stress in Medicago sativa L., Based on High-Throughput Sequencing" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/6/3"><span>Volume 6 Issue 3, March 2016</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/6/3.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/6/3.cover-source.jpg?itok=8EDC4x5t" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 6 (3)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/3/743.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-tmUju-gJLHhIZyx4LibOFyKLu7bd77iO_jN88cs9RvA" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genomic-selection" class="" data-icon-position="" data-hide-link-title="0">Genomic Selection</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/6/3/743" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/6/3/743" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/6/3/743" class="active" data-icon-position="" data-hide-link-title="0">Implementation of the Realized Genomic Relationship Matrix to Open-Pollinated White Spruce Family Testing for Disentangling Additive from Nonadditive Genetic Effects</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460098" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-Q-mOTbLhIcvTmVLa0JAY7U25A__4leQ69P0U2jaKNm0" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/3/743.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/6/3/743" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-FlOIZzqFhokBP7gIL0L7JDBrDV3uO9CoM_qaDGzmNjM" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/6/3/743" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/3/743.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460098" id="node460098--2" data-pisa="ggg;6/3/743" data-pisa-master="ggg;g3.115.025957" data-apath="/ggg/6/3/743.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Implementation of the Realized Genomic Relationship Matrix to Open-Pollinated White Spruce Family Testing for Disentangling Additive from Nonadditive Genetic Effects</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-9112-8522" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Omnia Gamal El-Dien</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-4469-2929" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Blaise Ratcliffe</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-5504-3735" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jaroslav Klápště</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-9344-6348" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Ilga Porth</span>, <span class="highwire-citation-author" data-delta="4">Charles Chen</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0002-4887-8977" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Yousry A. El-Kassaby</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">743-753;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.025957 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460098--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Omnia Gamal El-Dien </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Forest and Conservation Sciences, Faculty of Forestry, University of British Columbia, Vancouver, V6T 1Z4, Canada</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Pharmacognosy Department, Faculty of Pharmacy, Alexandria University, 21521, Egypt</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Omnia%2BGamal%2BEl-Dien%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gamal%20El-Dien%20O&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AOmnia%2BGamal%2BEl-Dien%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9112-8522" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Omnia Gamal El-Dien</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Blaise Ratcliffe </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Forest and Conservation Sciences, Faculty of Forestry, University of British Columbia, Vancouver, V6T 1Z4, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Blaise%2BRatcliffe%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ratcliffe%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABlaise%2BRatcliffe%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4469-2929" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Blaise Ratcliffe</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jaroslav Klápště </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Forest and Conservation Sciences, Faculty of Forestry, University of British Columbia, Vancouver, V6T 1Z4, Canada</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Genetics and Physiology of Forest Trees, Faculty of Forestry and Wood Sciences, Czech University of Life Sciences Prague, 165 21 Praha 6, Czech Republic</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jaroslav%2BKl%C3%A1p%C5%A1t%C4%9B%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kl%C3%A1p%C5%A1t%C4%9B%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJaroslav%2BKl%25C3%25A1p%25C5%25A1t%25C4%259B%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5504-3735" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jaroslav Klápště</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Ilga Porth </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Forest and Conservation Sciences, Faculty of Forestry, University of British Columbia, Vancouver, V6T 1Z4, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ilga%2BPorth%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Porth%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AIlga%2BPorth%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9344-6348" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Ilga Porth</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Charles Chen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Biochemistry and Molecular Biology, Oklahoma State University, Stillwater, Oklahoma 74078</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Charles%2BChen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chen%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACharles%2BChen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Yousry A. El-Kassaby </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Forest and Conservation Sciences, Faculty of Forestry, University of British Columbia, Vancouver, V6T 1Z4, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yousry%2BA.%2BEl-Kassaby%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=El-Kassaby%20YA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYousry%2BA.%2BEl-Kassaby%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-4887-8977" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Yousry A. El-Kassaby</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:y.el-kassaby@ubc.ca" class="" data-icon-position="" data-hide-link-title="0">y.el-kassaby@ubc.ca</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/6/3/743" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/6/3/743&amp;title=Implementation%20of%20the%20Realized%20Genomic%20Relationship%20Matrix%20to%20Open-Pollinated%20White%20Spruce%20Family%20Testing%20for%20Disentangling%20Additive%20from%20Nonadditive%20Genetic%20Effects" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/6/3/743&amp;title=Implementation%20of%20the%20Realized%20Genomic%20Relationship%20Matrix%20to%20Open-Pollinated%20White%20Spruce%20Family%20Testing%20for%20Disentangling%20Additive%20from%20Nonadditive%20Genetic%20Effects" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/6/3/743&amp;title=Implementation%20of%20the%20Realized%20Genomic%20Relationship%20Matrix%20to%20Open-Pollinated%20White%20Spruce%20Family%20Testing%20for%20Disentangling%20Additive%20from%20Nonadditive%20Genetic%20Effects" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/6/3/743&amp;text=Implementation%20of%20the%20Realized%20Genomic%20Relationship%20Matrix%20to%20Open-Pollinated%20White%20Spruce%20Family%20Testing%20for%20Disentangling%20Additive%20from%20Nonadditive%20Genetic%20Effects" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/6/3/743&amp;title=Implementation%20of%20the%20Realized%20Genomic%20Relationship%20Matrix%20to%20Open-Pollinated%20White%20Spruce%20Family%20Testing%20for%20Disentangling%20Additive%20from%20Nonadditive%20Genetic%20Effects" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/6/3/743&amp;t=Implementation%20of%20the%20Realized%20Genomic%20Relationship%20Matrix%20to%20Open-Pollinated%20White%20Spruce%20Family%20Testing%20for%20Disentangling%20Additive%20from%20Nonadditive%20Genetic%20Effects" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/6/3/743&amp;title=Implementation%20of%20the%20Realized%20Genomic%20Relationship%20Matrix%20to%20Open-Pollinated%20White%20Spruce%20Family%20Testing%20for%20Disentangling%20Additive%20from%20Nonadditive%20Genetic%20Effects" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/6/3/743&amp;title=Implementation%20of%20the%20Realized%20Genomic%20Relationship%20Matrix%20to%20Open-Pollinated%20White%20Spruce%20Family%20Testing%20for%20Disentangling%20Additive%20from%20Nonadditive%20Genetic%20Effects" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460098/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460098" id="node460098--3" data-pisa="ggg;6/3/743" data-pisa-master="ggg;g3.115.025957" data-apath="/ggg/6/3/743.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Implementation of the Realized Genomic Relationship Matrix to Open-Pollinated White Spruce Family Testing for Disentangling Additive from Nonadditive Genetic Effects</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-9112-8522" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Omnia Gamal El-Dien</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-4469-2929" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Blaise Ratcliffe</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-5504-3735" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jaroslav Klápště</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-9344-6348" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Ilga Porth</span>, <span class="highwire-citation-author" data-delta="4">Charles Chen</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0002-4887-8977" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Yousry A. El-Kassaby</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">743-753;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.025957 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460098--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Omnia Gamal El-Dien </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Forest and Conservation Sciences, Faculty of Forestry, University of British Columbia, Vancouver, V6T 1Z4, Canada</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Pharmacognosy Department, Faculty of Pharmacy, Alexandria University, 21521, Egypt</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Omnia%2BGamal%2BEl-Dien%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gamal%20El-Dien%20O&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AOmnia%2BGamal%2BEl-Dien%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9112-8522" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Omnia Gamal El-Dien</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Blaise Ratcliffe </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Forest and Conservation Sciences, Faculty of Forestry, University of British Columbia, Vancouver, V6T 1Z4, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Blaise%2BRatcliffe%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ratcliffe%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABlaise%2BRatcliffe%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4469-2929" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Blaise Ratcliffe</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jaroslav Klápště </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Forest and Conservation Sciences, Faculty of Forestry, University of British Columbia, Vancouver, V6T 1Z4, Canada</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Genetics and Physiology of Forest Trees, Faculty of Forestry and Wood Sciences, Czech University of Life Sciences Prague, 165 21 Praha 6, Czech Republic</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jaroslav%2BKl%C3%A1p%C5%A1t%C4%9B%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kl%C3%A1p%C5%A1t%C4%9B%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJaroslav%2BKl%25C3%25A1p%25C5%25A1t%25C4%259B%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5504-3735" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jaroslav Klápště</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Ilga Porth </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Forest and Conservation Sciences, Faculty of Forestry, University of British Columbia, Vancouver, V6T 1Z4, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ilga%2BPorth%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Porth%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AIlga%2BPorth%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9344-6348" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Ilga Porth</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Charles Chen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Biochemistry and Molecular Biology, Oklahoma State University, Stillwater, Oklahoma 74078</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Charles%2BChen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chen%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACharles%2BChen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Yousry A. El-Kassaby </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Forest and Conservation Sciences, Faculty of Forestry, University of British Columbia, Vancouver, V6T 1Z4, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yousry%2BA.%2BEl-Kassaby%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=El-Kassaby%20YA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYousry%2BA.%2BEl-Kassaby%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-4887-8977" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Yousry A. El-Kassaby</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:y.el-kassaby@ubc.ca" class="" data-icon-position="" data-hide-link-title="0">y.el-kassaby@ubc.ca</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=743&amp;rft.epage=753&amp;rft.atitle=Implementation%20of%20the%20Realized%20Genomic%20Relationship%20Matrix%20to%20Open-Pollinated%20White%20Spruce%20Family%20Testing%20for%20Disentangling%20Additive%20from%20Nonadditive%20Genetic%20Effects&amp;rft.volume=6&amp;rft.issue=3&amp;rft.date=2016-03-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Gamal+El-Dien%2C+Omnia&amp;rft.au=Ratcliffe%2C+Blaise&amp;rft.au=Kl%C3%A1p%C5%A1t%C4%9B%2C+Jaroslav&amp;rft.au=Porth%2C+Ilga&amp;rft.au=Chen%2C+Charles&amp;rft.au=El-Kassaby%2C+Yousry+A."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460098/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460098/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460098/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460098/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460098/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460098/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460098/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460098/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460098/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460098/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460098/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460098/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490720" id="node490720" data-pisa="ggg;8/9/3019" data-pisa-master="ggg;g3.118.200430" data-apath="/ggg/8/9/3019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">A Bayesian Decision Theory Approach for Genomic Selection</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490724" id="node490724" data-pisa="ggg;8/9/2889" data-pisa-master="ggg;g3.118.200311" data-apath="/ggg/8/9/2889.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/2889" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Optimising Genomic Selection in Wheat: Effect of Marker Density, Population Size and Population Structure on Prediction Accuracy</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490727" id="node490727" data-pisa="ggg;8/9/3039" data-pisa-master="ggg;g3.118.200435" data-apath="/ggg/8/9/3039.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3039" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">BGGE: A New Package for Genomic-Enabled Prediction Incorporating Genotype × Environment Interaction Models</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genomic%20Selection" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genomic Selection</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/6/3/743" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-5" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-6" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/6/3/743.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/6/3/743.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/6/3/743.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/6/3/743.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
