<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/6/3/695" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/6/3/695.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/6/3/695.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/6/3/695.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/6/3/695.atom" />
<meta name="HW.pisa" content="ggg;6/3/695" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Major Improvements to the Heliconius melpomene Genome Assembly Used to Confirm 10 Chromosome Fusion Events in 6 Million Years of Butterfly Evolution" />
<meta name="DC.Identifier" content="10.1534/g3.115.023655" />
<meta name="DC.Date" content="2016-03-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2016 Davey et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Relation" content="10.1534/g3.116.029256" />
<meta name="DC.Description" content="The Heliconius butterflies are a widely studied adaptive radiation of 46 species spread across Central and South America, several of which are known to hybridize in the wild. Here, we present a substantially improved assembly of the Heliconius melpomene genome, developed using novel methods that should be applicable to improving other genome assemblies produced using short read sequencing. First, we whole-genome-sequenced a pedigree to produce a linkage map incorporating 99% of the genome. Second, we incorporated haplotype scaffolds extensively to produce a more complete haploid version of the draft genome. Third, we incorporated ∼20x coverage of Pacific Biosciences sequencing, and scaffolded the haploid genome using an assembly of this long-read sequence. These improvements result in a genome of 795 scaffolds, 275 Mb in length, with an N50 length of 2.1 Mb, an N50 number of 34, and with 99% of the genome placed, and 84% anchored on chromosomes. We use the new genome assembly to confirm that the Heliconius genome underwent 10 chromosome fusions since the split with its sister genus Eueides , over a period of about 6 million yr." />
<meta name="DC.Contributor" content="John W. Davey" />
<meta name="DC.Contributor" content="Mathieu Chouteau" />
<meta name="DC.Contributor" content="Sarah L. Barker" />
<meta name="DC.Contributor" content="Luana Maroja" />
<meta name="DC.Contributor" content="Simon W. Baxter" />
<meta name="DC.Contributor" content="Fraser Simpson" />
<meta name="DC.Contributor" content="Richard M. Merrill" />
<meta name="DC.Contributor" content="Mathieu Joron" />
<meta name="DC.Contributor" content="James Mallet" />
<meta name="DC.Contributor" content="Kanchon K. Dasmahapatra" />
<meta name="DC.Contributor" content="Chris D. Jiggins" />
<meta name="article:published_time" content="2016-03-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Major Improvements to the Heliconius melpomene Genome Assembly Used to Confirm 10 Chromosome Fusion Events in 6 Million Years of Butterfly Evolution" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The &lt;i&gt;Heliconius&lt;/i&gt; butterflies are a widely studied adaptive radiation of 46 species spread across Central and South America, several of which are known to hybridize in the wild. Here, we present a substantially improved assembly of the &lt;i&gt;Heliconius melpomene&lt;/i&gt; genome, developed using novel methods that should be applicable to improving other genome assemblies produced using short read sequencing. First, we whole-genome-sequenced a pedigree to produce a linkage map incorporating 99% of the genome. Second, we incorporated haplotype scaffolds extensively to produce a more complete haploid version of the draft genome. Third, we incorporated ∼20x coverage of Pacific Biosciences sequencing, and scaffolded the haploid genome using an assembly of this long-read sequence. These improvements result in a genome of 795 scaffolds, 275&amp;nbsp;Mb in length, with an N50 length of 2.1&amp;nbsp;Mb, an N50 number of 34, and with 99% of the genome placed, and 84% anchored on chromosomes. We use the new genome assembly to confirm that the &lt;i&gt;Heliconius&lt;/i&gt; genome underwent 10 chromosome fusions since the split with its sister genus &lt;i&gt;Eueides&lt;/i&gt;, over a period of about 6&amp;nbsp;million yr.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2016/03/01" />
<meta name="citation_mjid" content="ggg;6/3/695" />
<meta name="citation_id" content="6/3/695" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/6/3/695" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/6/3/695.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/6/3/695.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/6/3/695.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.115.023655" />
<meta name="citation_pmid" content="26772750" />
<meta name="citation_volume" content="6" />
<meta name="citation_issue" content="3" />
<meta name="citation_num_pages" content="14" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="695" />
<meta name="citation_lastpage" content="708" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="John W. Davey" />
<meta name="citation_author_institution" content="Department of Zoology, University of Cambridge, CB2 3EJ, United Kingdom" />
<meta name="citation_author_email" content="jd626@cam.ac.uk" />
<meta name="citation_author_email" content="cj107@cam.ac.uk" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-1017-9775" />
<meta name="citation_author" content="Mathieu Chouteau" />
<meta name="citation_author_institution" content="Centre d’Ecologie Fonctionnelle et Evolutive, UMR 5175 CNRS - EPHE - Université de Montpellier - Université Paul Valéry, 34293 Montpellier 5, France" />
<meta name="citation_author" content="Sarah L. Barker" />
<meta name="citation_author_institution" content="Department of Zoology, University of Cambridge, CB2 3EJ, United Kingdom" />
<meta name="citation_author" content="Luana Maroja" />
<meta name="citation_author_institution" content="Department of Biology, Williams College, Williamstown, Massachusetts, 01267" />
<meta name="citation_author" content="Simon W. Baxter" />
<meta name="citation_author_institution" content="School of Biological Sciences, University of Adelaide, SA 5005 Australia" />
<meta name="citation_author" content="Fraser Simpson" />
<meta name="citation_author_institution" content="Department of Genetics, Evolution and Environment, University College London, Darwin Building, Gower Street, WC1E 6BT, United Kingdom" />
<meta name="citation_author" content="Richard M. Merrill" />
<meta name="citation_author_institution" content="Department of Zoology, University of Cambridge, CB2 3EJ, United Kingdom" />
<meta name="citation_author" content="Mathieu Joron" />
<meta name="citation_author_institution" content="Centre d’Ecologie Fonctionnelle et Evolutive, UMR 5175 CNRS - EPHE - Université de Montpellier - Université Paul Valéry, 34293 Montpellier 5, France" />
<meta name="citation_author" content="James Mallet" />
<meta name="citation_author_institution" content="Department of Genetics, Evolution and Environment, University College London, Darwin Building, Gower Street, WC1E 6BT, United Kingdom" />
<meta name="citation_author" content="Kanchon K. Dasmahapatra" />
<meta name="citation_author_institution" content="Department of Genetics, Evolution and Environment, University College London, Darwin Building, Gower Street, WC1E 6BT, United Kingdom" />
<meta name="citation_author" content="Chris D. Jiggins" />
<meta name="citation_author_institution" content="Department of Zoology, University of Cambridge, CB2 3EJ, United Kingdom" />
<meta name="citation_author_email" content="jd626@cam.ac.uk" />
<meta name="citation_author_email" content="cj107@cam.ac.uk" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=V. Ahola;citation_author=R. Lehtonen;citation_author=P. Somervuo;citation_author=L. Salmela;citation_author=P. Koskinen;citation_title=The Glanville fritillary genome retains an ancient karyotype and reveals selective chromosomal fusions in Lepidoptera.;citation_pages=4737;citation_volume=5;citation_year=2014;citation_pmid=25189940;citation_doi=10.1038/ncomms5737" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=S. W. Baxter;citation_author=N. J. Nadeau;citation_author=L. S. Maroja;citation_author=P. Wilkinson;citation_author=B. A. Counterman;citation_title=Genomic hotspots for adaptation: the population genetics of Mullerian mimicry in the Heliconius melpomene clade.;citation_pages=e1000794-e1000794;citation_volume=6;citation_year=2010;citation_issue=2;citation_pmid=20140188;citation_doi=10.1371/journal.pgen.1000794" />
<meta name="citation_reference" content="citation_journal_title=Biol. J. Linn. Soc. Lond.;citation_author=M. Beltrán;citation_author=C. D. Jiggins;citation_author=A. V. Z. Brower;citation_author=E. Bermingham;citation_author=J. Mallet;citation_title=Do pollen feeding, pupal-mating and larval gregariousness have a single origin in Heliconius butterflies? Inferences from multilocus DNA sequence data.;citation_pages=221-239;citation_volume=92;citation_year=2007;citation_doi=10.1111/j.1095-8312.2007.00830.x" />
<meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.;citation_author=K. Berlin;citation_author=S. Koren;citation_author=C.-S. Chin;citation_author=J. P. Drake;citation_author=J. M. Landolin;citation_title=Assembling large genomes with single-molecule sequencing and locality-sensitive hashing.;citation_pages=623-630;citation_volume=33;citation_year=2015;citation_pmid=26006009;citation_doi=10.1038/nbt.3238" />
<meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics;citation_author=M. Boetzer;citation_author=W. Pirovano;citation_title=SSPACE-LongRead: scaffolding bacterial draft genomes using long read sequence information.;citation_pages=211;citation_volume=15;citation_year=2014;citation_pmid=24950923;citation_doi=10.1186/1471-2105-15-211" />
<meta name="citation_reference" content="citation_journal_title=Gigascience;citation_author=K. R. Bradnam;citation_author=J. N. Fass;citation_author=A. Alexandrov;citation_author=P. Baranay;citation_author=M. Bechner;citation_title=Assemblathon 2: evaluating de novo methods of genome assembly in three vertebrate species.;citation_pages=1-31;citation_volume=2;citation_year=2013;citation_pmid=23587291;citation_doi=10.1186/2047-217X-2-1" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=Y. Brandvain;citation_author=A. M. Kenney;citation_author=L. Flagel;citation_author=G. Coop;citation_author=A. L. Sweigart;citation_title=Speciation and introgression between Mimulus nasutus and Mimulus guttatus.;citation_pages=e1004410;citation_volume=10;citation_year=2014;citation_pmid=24967630;citation_doi=10.1371/journal.pgen.1004410" />
<meta name="citation_reference" content="citation_journal_title=Hereditas;citation_journal_abbrev=Hereditas;citation_author=K. S. Brown;citation_author=T. C. Emmel;citation_author=P. J. Eliazar;citation_author=E. Suomalainen;citation_title=Evolutionary patterns in chromosome numbers in neotropical Lepidoptera. I. Chromosomes of the Heliconiini (family Nymphalidae: subfamily Nymphalinae).;citation_pages=109-125;citation_volume=117;citation_year=1992;citation_issue=2;citation_pmid=1459855" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=R. K. Butlin;citation_title=Recombination and speciation.;citation_pages=2621-2635;citation_volume=14;citation_year=2005;citation_issue=9;citation_pmid=16029465;citation_doi=10.1111/j.1365-294X.2005.02617.x" />
<meta name="citation_reference" content="citation_journal_title=G3&amp;#58; Genes|Genomes|Genetics;citation_journal_abbrev=G3;citation_author=J. M. Catchen;citation_author=A. Amores;citation_author=P. Hohenlohe;citation_author=W. Cresko;citation_author=J. H. Postlethwait;citation_title=Stacks: Building and Genotyping Loci De Novo From Short-Read Sequences;citation_pages=171-182;citation_volume=1;citation_year=2011;citation_issue=3;citation_pmid=22384329;citation_doi=10.1534/g3.111.000240" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=D. M. Church;citation_author=V. A. Schneider;citation_author=K. M. Steinberg;citation_author=M. C. Schatz;citation_author=A. R. Quinlan;citation_title=Extending reference assembly models.;citation_pages=13;citation_volume=16;citation_year=2015;citation_pmid=25651527;citation_doi=10.1186/s13059-015-0587-3" />
<meta name="citation_reference" content="citation_journal_title=Cell Reports;citation_title=Tiger swallowtail genome reveals mechanisms for speciation and caterpillar chemical defense.;citation_author=Q. Cong;citation_author=D. Borek;citation_author=Z. Otwinowski;citation_author=N. V. Grishin;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=Q. Cong;citation_author=D. Borek;citation_author=Z. Otwinowski;citation_author=N. V. Grishin;citation_title=Skipper genome sheds light on unique phenotypic traits and phylogeny.;citation_pages=94;citation_volume=16;citation_year=2015;citation_pmid=25765081;citation_doi=10.1186/s12864-015-1298-6" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=P. Danecek;citation_author=A. Auton;citation_author=G. Abecasis;citation_author=C. A. Albers;citation_author=E. Banks;citation_title=The variant call format and VCFtools.;citation_volume=27;citation_year=2011;citation_pmid=21653522;citation_doi=10.1093/bioinformatics/btr330" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=A. E. Darling;citation_author=B. Mau;citation_author=N. T. Perna;citation_title=progressiveMauve: multiple genome alignment with gene gain, loss and rearrangement.;citation_pages=e11147-e11147;citation_volume=5;citation_year=2010;citation_issue=6;citation_pmid=20593022;citation_doi=10.1371/journal.pone.0011147" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=M. A. DePristo;citation_author=E. Banks;citation_author=R. Poplin;citation_author=K. V. Garimella;citation_author=J. R. Maguire;citation_title=A framework for variation discovery and genotyping using next-generation DNA sequencing data.;citation_pages=491-498;citation_volume=43;citation_year=2011;citation_issue=5;citation_pmid=21478889;citation_doi=10.1038/ng.806" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=J. Duan;citation_author=R. Li;citation_author=D. Cheng;citation_author=W. Fan;citation_author=X. Zha;citation_title=SilkDB v2.0: a platform for silkworm (Bombyx mori) genome biology.;citation_volume=38;citation_year=2010;citation_pmid=19793867;citation_doi=10.1093/nar/gkp801" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=A. C. English;citation_author=S. Richards;citation_author=Y. Han;citation_author=M. Wang;citation_author=V. Vee;citation_title=Mind the Gap: Upgrading Genomes with Pacific Biosciences RS Long-Read Sequencing Technology.;citation_pages=e47768-e47768;citation_volume=7;citation_year=2012;citation_issue=11;citation_pmid=23185243;citation_doi=10.1371/journal.pone.0047768" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=A. C. English;citation_author=W. J. Salerno;citation_author=O. A. Hampton;citation_author=C. Gonzaga-Jauregui;citation_author=S. Ambreth;citation_title=Assessing structural variation in a personal genome—towards a human reference diploid genome.;citation_pages=286;citation_volume=16;citation_year=2015;citation_pmid=25886820;citation_doi=10.1186/s12864-015-1479-3" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=M. Farré;citation_author=D. Micheletti;citation_author=A. Ruiz-Herrera;citation_title=Recombination rates and genomic shuffling in human and chimpanzee–a new twist in the chromosomal speciation theory.;citation_volume=30;citation_year=2013;citation_pmid=23204393;citation_doi=10.1093/molbev/mss272" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_journal_abbrev=Evolution;citation_author=J. L. Feder;citation_author=P. Nosil;citation_title=Chromosomal inversions and species differences: when are genes affecting adaptive divergence and reproductive isolation expected to reside within inversions?;citation_pages=3061-3075;citation_volume=63;citation_year=2009;citation_issue=12;citation_pmid=19656182;citation_doi=10.1111/j.1558-5646.2009.00786.x" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=A. Fledel-Alon;citation_author=D. J. Wilson;citation_author=K. Broman;citation_author=X. Wen;citation_author=C. Ober;citation_title=Broad-scale recombination patterns underlying proper disjunction in humans.;citation_pages=e1000658-e1000658;citation_volume=5;citation_year=2009;citation_issue=9;citation_pmid=19763175;citation_doi=10.1371/journal.pgen.1000658" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_title=Butterfly genome reveals promiscuous exchange of mimicry adaptations among species.;citation_pages=94-98;citation_volume=487;citation_year=2012;citation_issue=7405;citation_pmid=22722851;citation_doi=10.1038/nature11041" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=S. Huang;citation_author=Z. Chen;citation_author=G. Huang;citation_author=T. Yu;citation_author=P. Yang;citation_title=HaploMerger: reconstructing allelic relationships for polymorphic diploid genome assemblies.;citation_volume=22;citation_year=2012;citation_pmid=22555592;citation_doi=10.1101/gr.133652.111" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=C. D. Jiggins;citation_author=J. Mavarez;citation_author=M. Beltrán;citation_author=W. O. McMillan;citation_author=J. S. Johnston;citation_title=A genetic linkage map of the mimetic butterfly Heliconius melpomene.;citation_volume=171;citation_year=2005;citation_pmid=15489522;citation_doi=10.1534/genetics.104.034686" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=M. Joron;citation_author=L. Frezal;citation_author=R. T. Jones;citation_author=N. L. Chamberlain;citation_author=S. F. Lee;citation_title=Chromosomal rearrangements maintain a polymorphic supergene controlling butterfly mimicry.;citation_pages=203-206;citation_volume=477;citation_year=2011;citation_issue=7363;citation_pmid=21841803;citation_doi=10.1038/nature10341" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=T. Kawakami;citation_author=L. Smeds;citation_author=N. Backström;citation_author=A. Husby;citation_author=A. Qvarnström;citation_title=A high-density linkage map enables a second-generation collared flycatcher genome assembly and reveals the patterns of avian recombination rate variation and chromosomal evolution.;citation_pages=4035-4058;citation_volume=23;citation_year=2014;citation_pmid=24863701;citation_doi=10.1111/mec.12810" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. Kirkpatrick;citation_author=N. Barton;citation_title=Chromosome inversions, local adaptation and speciation.;citation_volume=173;citation_year=2006;citation_pmid=16204214;citation_doi=10.1534/genetics.105.047985" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=M. Kirkpatrick;citation_author=B. Barrett;citation_title=Chromosome inversions, adaptive cassettes and the evolution of species’ ranges.;citation_pages=2046-2055;citation_volume=24;citation_year=2015;citation_doi=10.1111/mec.13074" />
<meta name="citation_reference" content="citation_journal_title=Syst. Biol.;citation_author=K. M. Kozak;citation_author=N. Wahlberg;citation_author=A. F. E. Neild;citation_author=K. K. Dasmahapatra;citation_author=J. Mallet;citation_title=Multilocus species trees show the recent adaptive radiation of the mimetic heliconius butterflies.;citation_volume=64;citation_year=2015;citation_pmid=25634098;citation_doi=10.1093/sysbio/syv007" />
<meta name="citation_reference" content="citation_journal_title=Mob. DNA;citation_author=C. A. Lavoie;citation_author=R. N. Platt;citation_author=P. A. Novick;citation_author=B. A. Counterman;citation_author=D. A. Ray;citation_title=Transposable element evolution in Heliconius suggests genome diversity within Lepidoptera.;citation_pages=21;citation_volume=4;citation_year=2013;citation_pmid=24088337;citation_doi=10.1186/1759-8753-4-21" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=G. Lunter;citation_author=M. Goodson;citation_title=Stampy: A statistical algorithm for sensitive and fast mapping of Illumina sequence reads.;citation_volume=21;citation_year=2011;citation_pmid=20980556;citation_doi=10.1101/gr.111120.110" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=S. H. Martin;citation_author=K. K. Dasmahapatra;citation_author=N. J. Nadeau;citation_author=C. Salazar;citation_author=J. R. Walters;citation_title=Genome-wide evidence for speciation with gene flow in Heliconius butterflies.;citation_volume=23;citation_year=2013;citation_pmid=24045163;citation_doi=10.1101/gr.159426.113" />
<meta name="citation_reference" content="citation_journal_title=Chromosome research :  an international journal on the molecular, supramolecular and evolutionary aspects of chromosome biology;citation_journal_abbrev=Chromosome Res;citation_author=D. P. Melters;citation_author=L. V. Paliulis;citation_author=I. F. Korf;citation_author=S. W. L. Chan;citation_title=Holocentric chromosomes: convergent evolution, meiotic adaptations, and genomic analysis.;citation_pages=579-593;citation_volume=20;citation_year=2012;citation_issue=5;citation_pmid=22766638;citation_doi=10.1007/s10577-012-9292-1" />
<meta name="citation_reference" content="citation_journal_title=Curr. Opin. Plant Biol.;citation_author=T. P. Michael;citation_author=R. VanBuren;citation_title=Progress, challenges and the future of crop genomes.;citation_pages=71-81;citation_volume=24;citation_year=2015;citation_pmid=25703261" />
<meta name="citation_reference" content="citation_journal_title=Philosophical Transactions of the Royal Society B: Biological Sciences;citation_journal_abbrev=Phil Trans R Soc B;citation_author=M. W. Nachman;citation_author=B. A. Payseur;citation_title=Recombination rate variation and speciation: theoretical predictions and empirical results from rabbits and mice;citation_pages=409-421;citation_volume=367;citation_year=2012;citation_issue=1587;citation_pmid=22201170;citation_doi=10.1098/rstb.2011.0249" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=N. J. Nadeau;citation_author=M. Ruiz;citation_author=P. Salazar;citation_author=B. Counterman;citation_author=J. A. Medina;citation_title=Population genomics of parallel hybrid zones in the mimetic butterflies, H. melpomene and H. erato.;citation_volume=24;citation_year=2014;citation_pmid=24823669;citation_doi=10.1101/gr.169292.113" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=K. Nam;citation_author=H. Ellegren;citation_title=Recombination drives vertebrate genome contraction.;citation_pages=e1002680-e1002680;citation_volume=8;citation_year=2012;citation_issue=5;citation_pmid=22570634;citation_doi=10.1371/journal.pgen.1002680" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=H. Nishikawa;citation_author=T. Iijima;citation_author=R. Kajitani;citation_author=J. Yamaguchi;citation_author=T. Ando;citation_title=A genetic mechanism for female-limited Batesian mimicry in Papilio butterfly.;citation_pages=405-409;citation_volume=47;citation_year=2015;citation_pmid=25751626;citation_doi=10.1038/ng.3241" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=M. A. Noor;citation_author=K. L. Grams;citation_author=L. A. Bertucci;citation_author=J. Reiland;citation_title=Chromosomal inversions and the reproductive isolation of species.;citation_volume=98;citation_year=2001;citation_pmid=11593019;citation_doi=10.1073/pnas.221274498" />
<meta name="citation_reference" content="citation_journal_title=Nat. Methods;citation_author=M. Pendleton;citation_author=R. Sebra;citation_author=A. W. C. Pang;citation_author=A. Ummat;citation_author=O. Franzen;citation_title=Assembly and diploid architecture of an individual human genome via single-molecule technologies.;citation_pages=780-786;citation_volume=12;citation_year=2015;citation_pmid=26121404;citation_doi=10.1038/nmeth.3454" />
<meta name="citation_reference" content="citation_journal_title=Appl. Math. Comput.;citation_author=W. Perkins;citation_author=M. Tygert;citation_author=R. Ward;citation_title=Computing the confidence levels for a root-mean-square test of goodness-of-fit.;citation_pages=9072-9084;citation_volume=217;citation_year=2011;citation_doi=10.1016/j.amc.2011.03.124" />
<meta name="citation_reference" content="citation_author=A. R. Quinlan;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=R. D. Reed;citation_author=R. Papa;citation_author=A. Martin;citation_author=H. M. Hines;citation_author=B. A. Counterman;citation_title=Optix drives the repeated convergent evolution of butterfly wing pattern mimicry.;citation_volume=333;citation_year=2011;citation_pmid=21778360;citation_doi=10.1126/science.1208227" />
<meta name="citation_reference" content="citation_journal_title=Curr. Opin. Insect Sci.;citation_author=S. Richards;citation_author=S. C. Murali;citation_title=Best practices in insect genome sequencing: what works and what doesn’t.;citation_pages=1-7;citation_volume=7;citation_year=2015;citation_pmid=26085980;citation_doi=10.1016/j.cois.2015.02.013" />
<meta name="citation_reference" content="citation_title=Lepidoptera Genetics;citation_author=R. Robinson;citation_year=1971" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=M. G. Ross;citation_author=C. Russ;citation_author=M. Costello;citation_author=A. Hollinger;citation_author=N. J. Lennon;citation_title=Characterizing and measuring bias in sequence data.;citation_pages=R51;citation_volume=14;citation_year=2013;citation_pmid=23718773;citation_doi=10.1186/gb-2013-14-5-r51" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=F. A. Simão;citation_author=R. M. Waterhouse;citation_author=P. Ioannidis;citation_author=E. V. Kriventseva;citation_author=E. M. Zdobnov;citation_title=BUSCO: assessing genome assembly and annotation completeness with single-copy orthologs.;citation_volume=31;citation_year=2015;citation_pmid=26059717;citation_doi=10.1093/bioinformatics/btv351" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=A. Tobler;citation_author=D. Kapan;citation_author=N. S. Flanagan;citation_author=C. Gonzalez;citation_author=E. Peterson;citation_title=First-generation linkage map of the warningly colored butterfly Heliconius erato.;citation_pages=408-417;citation_volume=94;citation_year=2005;citation_issue=4;citation_pmid=15592446;citation_doi=10.1038/sj.hdy.6800619" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=J. R. G. Turner;citation_author=P. M. Sheppard;citation_title=Absence of crossing-over in female butterflies (Heliconius).;citation_pages=265-269;citation_volume=34;citation_year=1975;citation_pmid=1055712;citation_doi=10.1038/hdy.1975.29" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=S. Via;citation_author=J. West;citation_title=The genetic mosaic suggests a new role for hitchhiking in ecological speciation.;citation_pages=4334-4345;citation_volume=17;citation_year=2008;citation_issue=19;citation_pmid=18986504;citation_doi=10.1111/j.1365-294X.2008.03921.x" />
<meta name="citation_reference" content="citation_journal_title=Plos. Bio.l;citation_author=R. W. R. Wallbank;citation_author=S. W. Baxter;citation_author=C. Pardo-Diaz;citation_author=J. J. Hanly;citation_author=S. H. Martin;citation_title=Evolutionary Novelty in a Butterfly Wing Pattern through Enhancer Shuffling.;citation_pages=e1002353;citation_volume=14;citation_year=2016;citation_pmid=26771987;citation_doi=10.1371/journal.pbio.1002353" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=J. Wang;citation_author=Y. Wurm;citation_author=M. Nipitwattanaphon;citation_author=O. Riba-Grognuz;citation_author=Y.-C. Huang;citation_title=A Y-like social chromosome causes alternative colony organization in fire ants.;citation_pages=664-668;citation_volume=493;citation_year=2013;citation_pmid=23334415;citation_doi=10.1038/nature11832" />
<meta name="citation_reference" content="citation_journal_title=Front. Genet.;citation_author=A. Warr;citation_author=C. Robert;citation_author=D. Hume;citation_author=A. L. Archibald;citation_author=N. Deeb;citation_title=Identification of low-confidence regions in the pig reference genome (Sscrofa10.2).;citation_pages=217;citation_volume=6;citation_year=2015;citation_pmid=26150828" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=D. Weetman;citation_author=C. S. Wilding;citation_author=K. Steen;citation_author=J. Pinto;citation_author=M. J. Donnelly;citation_title=Gene flow–dependent genomic divergence between Anopheles gambiae M and S forms.;citation_volume=29;citation_year=2012;citation_pmid=21836185;citation_doi=10.1093/molbev/msr199" />
<meta name="citation_reference" content="citation_journal_title=Int. Rev. Cytol.;citation_author=K. W. Wolf;citation_title=The unique structure of Lepidopteran spindles.;citation_pages=1-48;citation_volume=152;citation_year=1994;citation_doi=10.1016/S0074-7696(08)62553-X" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=Y. Wu;citation_author=P. R. Bhat;citation_author=T. J. Close;citation_author=S. Lonardi;citation_title=Efficient and accurate construction of genetic linkage maps from the minimum spanning tree of a graph.;citation_pages=e1000212-e1000212;citation_volume=4;citation_year=2008;citation_issue=10;citation_pmid=18846212;citation_doi=10.1371/journal.pgen.1000212" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=M. You;citation_author=Z. Yue;citation_author=W. He;citation_author=X. Yang;citation_author=G. Yang;citation_title=A heterozygous moth genome provides insights into herbivory and detoxification.;citation_pages=220-225;citation_volume=45;citation_year=2013;citation_pmid=23313953;citation_doi=10.1038/ng.2524" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=S. Zhan;citation_author=C. Merlin;citation_author=J. L. Boore;citation_author=S. M. Reppert;citation_title=The monarch butterfly genome yields insights into long-distance migration.;citation_pages=1171-1185;citation_volume=147;citation_year=2011;citation_issue=5;citation_pmid=22118469;citation_doi=10.1016/j.cell.2011.09.052" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Major Improvements to the Heliconius melpomene Genome Assembly Used to Confirm 10 Chromosome Fusion Events in 6 Million Years of Butterfly Evolution" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3.cover-source.jpg" />
<meta name="twitter:description" content="The Heliconius butterflies are a widely studied adaptive radiation of 46 species spread across Central and South America, several of which are known to hybridize in the wild. Here, we present a substantially improved assembly of the Heliconius melpomene genome, developed using novel methods that should be applicable to improving other genome assemblies produced using short read sequencing. First, we whole-genome-sequenced a pedigree to produce a linkage map incorporating 99% of the genome. Second, we incorporated haplotype scaffolds extensively to produce a more complete haploid version of the draft genome. Third, we incorporated ∼20x coverage of Pacific Biosciences sequencing, and scaffolded the haploid genome using an assembly of this long-read sequence. These improvements result in a genome of 795 scaffolds, 275 Mb in length, with an N50 length of 2.1 Mb, an N50 number of 34, and with 99% of the genome placed, and 84% anchored on chromosomes. We use the new genome assembly to confirm that the Heliconius genome underwent 10 chromosome fusions since the split with its sister genus Eueides , over a period of about 6 million yr." />
<meta name="og-title" property="og:title" content="Major Improvements to the Heliconius melpomene Genome Assembly Used to Confirm 10 Chromosome Fusion Events in 6 Million Years of Butterfly Evolution" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/6/3/695" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="The Heliconius butterflies are a widely studied adaptive radiation of 46 species spread across Central and South America, several of which are known to hybridize in the wild. Here, we present a substantially improved assembly of the Heliconius melpomene genome, developed using novel methods that should be applicable to improving other genome assemblies produced using short read sequencing. First, we whole-genome-sequenced a pedigree to produce a linkage map incorporating 99% of the genome. Second, we incorporated haplotype scaffolds extensively to produce a more complete haploid version of the draft genome. Third, we incorporated ∼20x coverage of Pacific Biosciences sequencing, and scaffolded the haploid genome using an assembly of this long-read sequence. These improvements result in a genome of 795 scaffolds, 275 Mb in length, with an N50 length of 2.1 Mb, an N50 number of 34, and with 99% of the genome placed, and 84% anchored on chromosomes. We use the new genome assembly to confirm that the Heliconius genome underwent 10 chromosome fusions since the split with its sister genus Eueides , over a period of about 6 million yr." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3.cover-source.jpg" />
    <title>Major Improvements to the Heliconius melpomene Genome Assembly Used to Confirm 10 Chromosome Fusion Events in 6 Million Years of Butterfly Evolution | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"vuoeDoH7niNSzc-RMvkvkNdMaDOWtRh5NUqQz1Mpwz8","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460075","apath":"\/ggg\/6\/3\/695.atom","pisa":"ggg;6\/3\/695","ac":{"\/ggg\/6\/3\/695.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/6\/3\/695.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;6\/3\/695"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;6\/3\/695"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/6\/3\/695":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022zUfZfSI9mggiqBHuOgeoTqJ893L4EO2MBKYLtZkdfYmEfhSW9Lxg9127GJwizKq2WdrRQqt7Z8mn9DL0A1ZSYwsxrgp+cVv9uPbVJUW4dzF12XX3t7pcMeM8TGxSW1HikQP3OGGWV\\\\\\\/\\\\\\\/6K0VEorcEOc27OWtVeu5p6p7nNCemvlykGlg1UwV6d3S90yxJsignHDaaRdixgbc9diw6MmWSpkRYXDXAf4Im2XXIrCQh51PKmQDsejVRepm62+QhZKEfQI5KbBrr05fO6BgIOE6MnjWTRuECpAmj+zS0+7fiXXpuCglpS\\\\\\\/9dJ0pz5m2SPPTGtWBhDp8ux2SK18GC1K7lDc3FsTrpY0wGLq3dOZpW6QYos77YTa4HW4lK7+OnSi8Dmv7R08id+mY7rekkdiZSwfQPrZHGbUtLABAAG84zmFresmzVRS0ymFaWCLZ40IWDOieRvHo8GDh5wG+tSQUzOog3IO++oIsnc63m6gCpjBwyg2D8pxR+W9hp406AT972nLgQf5ODvmB0nd5RukwpPRgMbXm88+pUnknXQNq8AgMQZM9SjHJow+Rd6fwP2mHpbRaaRDJM9W+a66Bct\\\\\\\/witZbbRxELj3B+fsg8XmC2YiiVIKS51gjdnIuN+aChB6h5NoEG277Teyz4Z7X3y3nUaa0C4Dc5jVERhtTKyHjK\\\\\\\/Vna+JOr\\\\\\\/42nVXZ1SIQ+Q9Sp0ciwXnk4fl08+DZ0+uyMQi\\\\\\\/C7HCieVY3eQ6CUgZ1bWfrWZgmc6gYMtvkGhkfOIWPUop430wbMfH32RHVoQEmo+8VOtPKO\\\\\\\/lwqsHzRzfGwXu0mIkh4q2DC+aNbfEeRhPbnfyBO9iIINW0xewY9O3XJ5FzCO6mDdPKyRGzwZZQZcvihIiahLA3gzLu5hRNf0QgtYNpVQwOk4HiIjQ2xDf7U65F9N9WVW+bJQCiyvS2uGD9o1YqbEv4AEPOzIw+iMcnOYGFCFqaVtSwS7Txuo0d4mYyFnuTofykKA\\\\\\\/a\\\\\\\/rsBTtF3572XI3tMax+uML1hsGVcQM2VYHQB1CbDfJ0GSZC9zSy7ZgIQ2x2H+64maUpzbOUyYqje9rz5+teAbs+RQOS2jfyI9JNqULA2I4Umqoa1X3kazSjJrHW9T77uHwq5fuQbZ6gmopgldvho6vJe4vR8W3Tc5dJxJvYQxC3E7ehTmR+R01u9OrJgDQBkDGM8B35clj0tc0rfVDWoA\\\\\\\/qFJrsWigMfzRdKFRGEcQ9bMYd704QUESTLSe0sx0Ywf9\\\\\\\/KJWeCOGHDaGNwEobprcXVMRwF\\\u0022,\\\u0022iv\\\u0022:\\\u0022U4EJxTg3owRiCZmkus4t9g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022593363cffd66f3526ee6cd1c8613794d1eafa8ae832089000b66b78083c46fbf\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022izx4GdIPeJqnPUikWRZbgGc+bA2TCByZXt+mK1lOGDYkC2vQZLvxoag+Rwt6vkibW8hZET0OdzEFbfWVzztzEiWGQxr\\\\\\\/YsWRbxwNcvtFgAtPD441kQ\\\\\\\/E2YMdAeogvaTXKZJ\\\\\\\/8FJnHep1G4O1b04EO8DDHQq7PbCirSDLNQx58Y6Q2qc8BOQxNhwV9vEmBtv0p5+5VM0E\\\\\\\/BHOXRJfPWz1kYICyx3o7abrBffoabIJYH6ilDYFelw\\\\\\\/votHpPQnuQvTtIwkqK0HLlUJw7iT6+6wr2d+VH7tI0cZ8bR2QFQQDtm3oacYFtBvQ4sLM0Na4Eo1nsuisa8Cr1\\\\\\\/LvrJdRERtzQVFLbvWhVewzOgMpb1cjNtpto+ZH8GtEhdBUYVwEdhfslLJ8RYw0RYnIFowVzqpuWNAJnMCwZQWjICQgrnEeL8qJzjNNUJWJ22jGECTyT8yoVHuhjTNJsvoT6utjQUkDjO35y7jkgiSMpT2bI3zWCOYnPOefvhMyaK6y4RpMMrf43EQWpm9R+FIzpxz9gF1tKnN9oTzhoZo3TCzG1213XkwuxVnTP0s4vrr3dSqFQ9vQEmxgTig+04BSIcCFJML8heIQtqaffFzkWvBRs112kzAwyP0t4d9XJ2XVn13SL0sGeyTx7EM2wmFJqQLMSYpwrQbxO02xFsRAM7m3EthJecfsAH42iSYJgLjC\\\\\\\/RW+CO7hTOSLoMlxY7Bg\\\\\\\/xxXe4MsxQMSKQpudacdoxtfKf7PSnBaujSdg7gnolmyOBiZF7sdLIxhpe+IwOmuptlTX83sk+aDsHhnKOIfHSEW7tsIERBLOA92TzLfJ0t1aoeatXkv4YaAdHd\\\\\\\/AX5cpw8cmnf7WWnun4HOzDEoPZyxThzPPbD6ZSbmx3OqG3uNB264xWTNeKpuG80NXa1KVsKT0oCmqFopFyJIBjgkhGbh+NDK3KbhCBsanaMdLZMaKJ7qb1sI+Crro9UBHqmsCaSXy6nxCfjNRsxrgi2jxYqNrnXGyTV5ATqUwfjM\\\\\\\/vHH3Awdn3EhKLiZtxQh7hGj1ufhXCSVmsyn2+7PbbqplgY97JUqj7x+f6SDN98HHIQpqcq6SKdwWYjJs\\\\\\\/wQYCRRwVp4N2yUiAOSDr3vfnLsyPU\\\\\\\/278Jces0UZ1ZGpACI2nq1I5txsus8fKzDSFnD\\\\\\\/BrJ8+b6PmqFerq58sLceAj2oNJ9F8+FQfp9ek99xo0b3K3X4Ss+Fe+VDK8TzM4sm+hHgMwQM\\\\\\\/1oN0ylGCL5lscwD14+dCSLRlLokH8jr5Rqyn\\\\\\\/yYhJ0MRL1NJWug2pDxGTBOAZhcKnvQdEFX2WM3fPLPZf2bLL+huW5Iwy\\\\\\\/Ic5JfA9UD\\\\\\\/t0urQ86zcCpdw4+YF80qrZCZx+cvstsvkUnaRLX0hPFkjtfzGagMW8\\\\\\\/oqDuJG+1ANOZAel+FNRWo8Wlj2HR7OKI8RSvirZyEIaHa+hsGkFhamsU1Ftu0ZtQCfYs911Dm9GlBfK9ES581+\\\\\\\/pevdILDAFe6u+8wJJ3sdPyzgEgXkQdL0aQaKcmo7GbfNOtA9JDUMy4qiTx\\\\\\\/\\\\\\\/s+gas5qDwFOjkSymTttR+rklMArn7xqFFDCflHmaWlLkGEZrNXDcdfti5pmfFJ2Rob5VJ2Cx9epaq1L34PXXnjM+oLWtGmpoOUS7OZNdzr9Q+tU2pm4y1hajB4wanU7Bw1UeMNKItvXI4G5k0evuaCvFjxxM+ZICbc5ysN20aSShxRY7YQcimqnLOSgmnWp99Hsu79t0lfUiN7+jRMGnven3JTk1qcQ3AYwHHSBqnqpv7X\\\\\\\/iuwjf9hV4BG4XsE5XUPSGR77vhmSYc6wOy8Wh6drlg6uEZFktzfK26Fy23N75a7Dv\\\\\\\/uSE062gSpktSCLvg9dKbJ3RvRJxJWmhCdVtmxdwOgbXD6lripNt5d8H+b0SBWj+s7Up\\\\\\\/7rOXsyEqHepgdqXBJSSDiqyA1+acCS6w9oBAGMUvhLXiL7cDlQJI2FmFUgXTlm3UpYO3bDnAORiOPY2N80F+0tIEMu93whUVIizSi4ENPFBKfqTFvfFEyF\\\\\\\/cqcnQB\\\\\\\/vSBNdz8ngUK\\\\\\\/EG\\\\\\\/bUuQaa4LTY3j3adeKmKOq4QIr1Br\\\\\\\/TCgFBL31lQnPZ19zz0TMHAYxi8eP49j8if+HmxUPNar0GsP9bMoFBbc2CslwrMffWD2ZUkBpvUyOVhJCjhvxSvcFinv8CFafPiSPserQfxlrRh4ZVclaYYUTE5PmmjrD5H6EUydgTR22URiLjpQoFJXMMjnJ3HSXAVAmKUOXBnND55laFtruys+fpYVq18CJum2XU5AP4M6630ElhbMGL+gnjNABiW2lfurBGQBKHCrrJHuWOLLm3gQCTH9YZG3hQhtaxWozkautnSstWGfBolsZGT7VrAeZvNOGEdW707V813xIGUlD0cE4DcjEXMlKzg0Jao6I1CFA\\\\\\\/Bs3tyPNMBmOIUzt1R9w7ZodfHZ8mopOYgd\\\\\\\/jH3HU6t0TWshOCv1L4kgrpALxIRCOQW67E7RqjY3hu\\\\\\\/SilCWeumBosHKHquHVCO9BAwoVNmVLE6rTEH4xVxSZ0Cn3zY\\\\\\\/\\\\\\\/YFj+0nZezIgCQc6oMxoguRtlnb5ra\\\\\\\/txbMxmOSqLOByoNeWA8mvmmxjc\\\\\\\/ie7W7wQisTHk3MfH2au9V\\\\\\\/FGpVQaWD6Qu\\\\\\\/oxfz6dj+kuRoYfrd\\\\\\\/K2BLUeB5sbqSil2LQ7BOUG\\\\\\\/k2kzHFg5cDM610Sauu\\\\\\\/iu6Y82C8ovV18LaCWO6I8kcFyHjg3SR0QF4PDeq8Lbb3HrzHDdmvjkRKx0n9fzFKfvCOPQHJjtFjTYgbsqV6WXLtX1+9sLOsXFF4UDTxrA1tqlkH+wWALSyJYhB2NcvNX\\\\\\\/Ul+DDeO\\\\\\\/d8B7B7WfiVHlyotzWpMEUmFBFlYHgQUQgs\\\\\\\/YMrGdYZbhuh6ZQua6qC8N8uT+EbG6\\\\\\\/9AElZIUA\\\\\\\/WEneAWBHNK\\\\\\\/YbSQqUWhnU6Nt7Tn0JygxvbHVZNsaw32b3bbTKcu51sO3N4lO7ri4BqODE\\\\\\\/ATSIhprxt5kOqCd9JgmCXAg1TaRI7DmgzNoiNZ59Uk2HZ1qQLL3ParXIz43IGGkxIHtfovoQJGlXGo2dBYmYObMWL62XOCHu88T91veTSAM0g113uemrreKa7OOneeTu9Wmn7vDQDH0JboddFNVEVrM\\\\\\\/wZJywAVoXrf1tUbfGFgxuFxrktF+KbMIPe6LRPWzIM5wF4f4b6YcD6Q68DRP5oxWGbeJb2FyJQvT0lYlOHxEA2yiVTvy+Tke1qdZ+2qI50qWwpcWxhcppwn1To73CQr4dGjc6XvDMNfyTrSDHwhR3sEZIY2VEQ32IW2\\\\\\\/lWoPxgg0YZSEy+JP4k+2Tp5y1ygslaaMgWRAKQzBtLt37dz3A5+972IMo36hWPSyyf11q\\\\\\\/OEbuooqQ37YPQe8RHvIFrmakndKwqSc8QpuOTfPnd9X3blZDo+4ajTdsdjT8DwcRndha+4KfzkrC4AgI1kKyDMdRSVX9jXtQS8cY+yxL\\\\\\\/tDkAWC2AX710DvYbKMUUmH1W+2vTzV65EDeXxzOUDAyI2nRv\\\\\\\/ZG7AFf+nOWsluDe7YFr9+FCb+1SwoNzY+gRs5NUQ+dgpjid3qW5c\\\\\\\/n0dSY19Q+edApm0Gcbo4kkz5dkwNQGd0fmy8Ed2vJn9wBlZy4UwbGAgKBI4ldAIkYtrSgo9kdP6fW+oim+QXUWcMC3i3I06YEXfF\\\\\\\/FkHKyyqdvuagcwyVt3i8bRoHO1SN0btX5j3VgI6qCglM7bVvyfCldHNBJrXo\\\\\\\/1W51Xy3K9J2++jWseTEXnrRkMr5oVmEYAg42HR1yf2NCBGnauZZQnx22QbLzkSQmQiPvoQhehxA03diCIpWxPq537OHkJOJlWc6zL9dEAY9FuXQ1eX\\\\\\\/K6HCefhTgW5TBBSBcdyP5SBdBb42ENeMi44Tr3F1IxashpPtQf6okZJpfj0H6eLgIvSDA4ArocpOpkxTLhYTf19KruFUPBXQ37Re68Y\\\\\\\/5yM4unZaek8GFBTS1dxw\\\\\\\/wAtuchF8lej++vm3kvhiHxnBMR8Bvn2EOMnsyyoyX2EMPmAfASxQTVVWyT4pZoVl7z2hc4mJa4cJJMUgIs\\\\\\\/AYn1b\\\\\\\/aciZhoU42yxdY+7BhWZQRWq8KsN9+\\\\\\\/o5UKnaZblpL1+PoPRFj\\\\\\\/KRbnsJYnH\\\\\\\/bRB5fPeQb6MA4PI9xEXwXQWjMWTSiy3pgqo9LES9GaBzoTOj3D3k4h1hzxLnxeoKvPFwfqu6zc9vsEgpG5p\\\\\\\/x9Hegcm09Rqk005WPZREzs\\\\\\\/1TqAEoPq9o9XeTW288Vv2t6OS\\\\\\\/GmtFnA6QpxNIF60Cw7l+hgwMbZv3eDIPjuGWG37N0bVBpjgyCdnbfKvcIPlZKe7jBhdGVb1SCaU8gXpLvO22dPxHRPMYKMBl7Lyrz0SVspDg+G52FQIyETafmVemCxgxxKfQLhozNoP6cKNk8cE3mP+O1GxUTtnSd08GFcJsLZbyGjuG8sdNF2IshGEbPZTVOcDjgZzOh4BIYCd1MoR9g7rkjIZUcX+9hUbJj7f\\\\\\\/lNVXc9t1nAHuGEtwBo2zew+FvVqPscT53RJVnocI\\\\\\\/j6\\\\\\\/Mw9WT14SjS2SHQ3MyCAu75NcwfRc1vX8xQD++Ig81bihdXtAtT9KMBAhvbOtMtW+GKcHslmUi4B911suWqYxIqu8n05nL1DSCJki7mQPwizs0gSK5VIA+adY0NA6TuvfH1eJN\\\\\\\/ePGxbHvHeHJvleK+KxowGKtNU+ydNa2Zl9fusm+9BwrT7XcYaILfFYPYBwn2KXvudLLkXxWhKVZ3MASrLdGEuVEGijwXj7TzRMFIZWDaTvmwdF2z3AveUVcZzuSkSHYlZyH\\\\\\\/GDT4PpNkJjNdoPz3b2LcYZpyX0d5dWkYAiP2n5XtA2KJi34aHiR0CIMeeW6bpxn42NaZDVUfHoayF8hQiBN+EYXbMzMGIjbVguor5Mc0D\\\\\\\/nhK9ZCKR2smhpkx5+LwJOqr6FqWALHtEgSgsiuKCOSUj3mLMS7itGiKmumUWox14j8lKJ+BuWu11udnK0U59NXNxbd\\\\\\\/8ba\\\\\\\/i\\\\\\\/NV0kMGy4uE38PU+4tDiq+0FnAyazKBQigCFboB\\\\\\\/IdATWHgkJ8YliT\\\\\\\/IZQP3Um3wjMJgKrBrMXCdECoXEMbyjeQdZi\\\\\\\/TIsKlsSYJdBp90jENBMv\\\\\\\/9+qGZ77Xs0x00t0XBDDDk+x7UgPEJKCcZTVNhL3bLHYpAk0E+NoKXeYdscwMvbi\\\\\\\/e2vXLqVnZEXbWrm3lYal5baBZ35K\\\\\\\/BfOz7rZuTmYLNcdvarfd6jyqqv3pkO6BrxwwV5rYges6fl\\\\\\\/mmDNEYlBmhhJ50DtFPjMNiAGHzYjgj5vjf4kTqJefc64jeZr9ONbIhawiIZmswQLPQw2LoLHAM2UCKNmdMTDH7UJGQlkGt9FBq7HMCjFiSx\\\\\\\/7RVxbvMu+JyMGueB7UItil1CNgFTrBYkat4v\\\\\\\/kiJUSTT6q9QoUmNIhWEZekq3fi9K4ZptRBOZ\\\\\\\/CV234o+YWn8tyl\\\\\\\/3G55BZNQrXe3GNPF45USYIMt77IEbgsx7l9BuvO4sebo5XBqbr+EpMneuV5qcmFJGT1UpcZ+pcEbzi2WETIvVkdGvmli\\\\\\\/F\\\\\\\/YFqsSrUC750H4BH4g2tCyJThS9tj8J6Xac7OOJ8RoagH3t\\\\\\\/Q4PK1T\\\\\\\/2PyWJNq844xDP++ED178PqggfUVSToIF7jpXf6LgpljCpp\\\\\\\/KGyAVpTA7hqkRRbY6WjM6p9mtdtt6KID3mniCIVq4LYiwD3mKQCaQbf3dYpzLjfBpcSPTUoRIuoCGTk9+0yJqePbXH3GJGR78RgdttA\\\\\\\/\\\\\\\/bmKUDAt1fsJEw5KrxBpL7nB7p9K\\\\\\\/Qrskk7aFikgn8nqeAIP\\\\\\\/dwF415RyuSQOF13mba05LxQijhOJQHa\\\\\\\/krBgcYB4\\\\\\\/ZYiMUg\\\\\\\/YxHwsb8WGFI6zmDEVqvh0qAZgCKUHWAR5e\\\\\\\/17m\\\\\\\/nMi28YM9hZ3Xd\\\\\\\/PTNPHDdouDHprZOwpm8elMmKlxjkzrQm980qz5jUVNHKuyCQ2bQyzbsY2XmVCdcyTgHw7ZCyJz5SDN9V0o+wt53xSYSx99edi8KObzgTnOf7Fp3lWq1ZuV8e7lUIw4ryVWs5v2nxr28DoE\\\\\\\/+YXM6qXLPydqM1hgLrKCDx5nThEBqa10jDgsS+76KLpi8jamG7BiE3K342+8evQX2PCcVxeKtlHYfwA2AWmJDlYNe5SiQLt6LKCo8YtBPcKRPyuxmXKs86PBCtdZqowCDQZwK0UDOUz+mw61QV+OJRK5rv09wJ0uPXRPzJss8YPu3J1S00On4FdIfpwJrUYOfzRVqkuRQ3ffVLVOHaNweiq2vgpaX6CHVaijOvzMuOA5eTwRgUa5iLim\\\\\\\/YkylSDPuF4yRY33ksy+T\\\\\\\/rFc+EE+4qPsWShpg2sd0Fcik1PC86aWdQ5N6YnPmXjs9eqmC+CeT6GeHXM8MBGygwDRumCCmJxiJ44fAZwdAGOIamaNflnvMhGM918FOV5RKv6IESebhT0vWNX9CfIjQ1fKyiQe+BstyGnsVxjlPfv0GDAoys5s4w+nqo3BowFMBzkkN9899CfgjV0PwtJfb860LPFSoybiJOTjcet8EIAzN0V83TRerF0ubNnibvKihfA81+NeQVb9OKRUujV7fz9RzvUMaWSPaOT4fYZmWf+shl7zYOZTKlOk8QoFyANz2XRzZZ4FVeRKxpoN2C0DOIsz0GIfp3AOuKzmKTYl+lKKkpAoLQcY01sVCHN5V+XqDxCNYaHho\\\\\\\/xK\\\\\\\/cyXIIHFmeiy2fJbV+Eh489DzrtwzDsmtNyeoiMKJz8XDOyWdzDh9YTRi8Cpso3RQAo7FHdj2vjV12lOrXfQ7EcBOex3r2kCnZqXkeHkMpaQQsy6rijbUS5NMx9KLcViZMDm5gTwSztwz4TBU2+GLCQG26YEHMquAadgwgUKrtms0UgzKLaFPp9m1TIra4IzRPeFLoFCuIuOv7tXLUfJrKIqt5UBFbYtPgndqzqIU4JfrZk3nwRKrxKIy1qzIUO0taBXWxMf44Z\\\\\\\/2apWKLCQkcUoSnvTzD7iL5qZ1ZCicKHkzHQ9psAvde9oqClkS+DV9iTTxqEpWZnTpGaug07ING30Kkl0g3AajMeeTZNt8vQrmLTyImlgLklXz4JlTRdqJTbNRSLWMDInCwocadl9uDXKsshDL9owqq2LNj\\\\\\\/jSoHLEKetbixzFBuI7oWM4l65GP4uvW7Lt44z\\\\\\\/8ed1N3r5X+uPTfcnlxnPT45sxojUvbWR\\\\\\\/upNAZcm20Zs9Ir6rixUqxQXyK\\\\\\\/CbMVLk+IwAn8HVBIuIJcAWK7pVq\\\\\\\/2I5yuGR5IfNKEQGKWbYKaN2J3P4KKmEbZ1w9QZm47c7OF5s3uvWmfAwDrxBigj0QGmx7srD+bH+i9KBb6OuZUFQAbsvnPYeQrMjX9K2vLCiC9X2vjgOuD1GZv5xK3e61P6UpQiXh67l6t2tpyhZmya47wUGo9i218oyTw\\\\\\\/+cp9t6AXBVA0TxpCb2EetdS4UTo6MrvVzJkONAN\\\\\\\/JbYUt+rlXLkvCGf3sMopxowdnjzX4xBgaig7azBrsoCucO63nA5IvCPpdsXeALOVqIKMgfaliZchz8zS7m9RrisArrX3HcGy1nQxgP775SRWrAmoiNnZ+\\\\\\\/VDGfTcgw3Jsjr50JOPzSTj3n1hbf5slH8vCzlMi0PaU35g8K01Ejsn\\\\\\\/zzyp3MiQdcbJZiuA8JwYBYJCRerOUoFqtk9wGoKWoMHBxyBQ\\\\\\\/eNNk94nz6ndDFCFcwat1KxOOXwZEI4+r1GajsId8tlr6tKd4Z0h0Q3y0cRUQk1NMYylfK2vZwMtdNscVA\\\\\\\/JiassD9Ts0UYbk2DyC\\\\\\\/F\\\\\\\/bde0q8b8cSx2lJOoB7oEIBjbkGzS4A8\\\\\\\/MC2YBS5TG5qnT27Q6yuR450Exr9PMZaZwmryqqZE9+ptd7vON5ffrp5pDuBEfwnv7L7uKRGS89aKR7kybqup1aPckaMzjJTIzPp23AvV5GjBsN\\\\\\\/J+8vIyofB0qIBClv9KP9eSF8y0iDnfYUyzMViqoutOac6bLs9f9EKQB15u+MMD9l4zD\\\\\\\/BfEcd3sWkX4Q+XUPWwHmXuk4+kKWQzp\\\\\\\/yFHry86UHl7PzDiDEFMjJ7cAuDnKIQWA8XABlDNgBfTD2xdYyD01KXuCpMXzLjP7vmrEQRoSBeDtXlbjouGzK80q1y4\\\\\\\/CjOkQ\\\\\\\/16pdhg0hKzMmAyNwXYHI4IuAm8xOeJI2j2rgz\\\\\\\/T8A0mSgsr94et5Bh0gDPkzizqZ4nGKoCHaRy3cTUmD3V53gvZBXGDqE4UwIwz3vILouqytxo0jtFk3Ln4Fb6bTuQYSESlrUzpuSk1nCYkA49jfoiTXapUVGqtI7zz2QqZkiebldiMOO26FOX2zLO\\\\\\\/yEE5PzaGxyCMdgX1eoOzkomupSITN6HWOYnyeInij+UQKls0KvrPjZERtf990YbEaR0cWQvMlu4RqTMlBZFLldnrKhSZDFCiZZtFCYhWPp9kYrQ9nhbBUamO2AR7UiykryiRxkSAol\\\\\\\/FvZh0xX0+saKPXnzbMhd6f3LkBPGAHE\\\\\\\/q7kxSbmw9HwdWZGFh0ZtSN9LfJ7gfWS4m0JW0tRbR66OpSrHNjhvygk8SmopngBWFsQwa2YRUjWMxucYpWsijxt9+aOIn6SDcDNcOXkfPLICpKDyM\\\\\\\/dtm4O9fQHVw41BJtIIsZ6D9ncE5rXy61mwcX5YW3bylFJ1Ga+6RVIrYkV\\\\\\\/eXo7zUsta1xSj+kS1cfvFWRemliUF8+Wd7H8x8Cyki2QpPJ5pkKO6zp2DcEadpQZpoyFMmgsufNOs+kSpeDkIPR6\\\\\\\/CYwvOB9WPecCMeo2EAA\\\\\\\/ql15BzawMd+PvHP5dxl1SsH1rIsBUOLU3a3kgzQzK2ND+Tu07\\\\\\\/6sn5xoPgToRoz\\\\\\\/MeP828lLGh00JnMA6kK+icnuhjvUvEGNn+b8qQ8vqImjBxy7YTLcKGSPNO13d8q11d+Ma6X8d\\\\\\\/GyqPfI1\\\\\\\/3rIDNX6Gx50XWJtcjC5eNcP34OyLy8DEknq3M3j99wMOw63UtWN81Z7maxY8qI1ReUIiNDda54sLFEYArXeNL8OeKm\\\\\\\/XP2mCT8ntP+IEDs2XMIj+OT5zrjJ6MxJ6sulS8QmJaUWr6k2RZavtVzJJ\\\\\\\/V38NKbg+LkgrKCqPyxP97OxlUrddA4+cj9K5+hWKDVtVit\\\\\\\/Q2nfweiSu8Un3q4gSKKfc\\\\\\\/fWAGcSqw89SIIqy25LgVB4Gu5b42d+YUeMOzIEb2a4ipEahm\\\\\\\/Jilx9Cg3LJBo1T517cwiyk+IFsa0SYUFRWi+G+SXeGYdxSZOjVWt0MYXjO8klypbtt\\\\\\\/zFlfbwCUo\\\\\\\/pE+ApaWyFgx4tnaOWinpPurkFzeYW4Be2+Ze2IoUUt1+rfxCa5Zb41wEg4i1Zwy1Jg4KCHBXd5fLArwEZrtcHEF5h0ZaEp\\\\\\\/DylfyAcRKQpxu5NlIb7LpzH0v0D7zrR2vfkVISj7K7ux0hZo9NWegfZk7zzryFKEjb5q8BFzvSWco982Bzedp1FiDJUCdy5Ezf0wNf1GOhihDsMIvYZNX1xdjzhHgNYUNg7C0IbjUi7bDIjGmfMEzym6DrWHV5bmdTTzO0h38L2tMq8fiXReeGPrpCn57rtP2bhJPgiV5RHUtam6d+fUuHGDTrawdO6zDGptKq2bA\\\\\\\/0xDxmF+9\\\\\\\/phG27RAGm8VmxWWBbnzyXur72vDnWe3jc9u5GSwHHmvppAZIfXEDW7GyPHz7nQOY+DuslGVdtBAxwYiYG1G3EBxpj9w1t3OgEob3aOXQ9IfMAa01bmEPIMWoVMNEtK09+LeHWAsHkD74O0DOVq1PqIWLOyuWOUYz\\\\\\\/RH2UBwa3mO4hTPJ2JBXuA9Z5ANwgDeWC1LJL4q38nsGKjEayvEBftUEhca5gjcQ7ocoH8w\\\\\\\/RRU+eaAnDkpC7PwZTpfl1oB21WrROwgl+DjkdfWEtKmj4bUF7dGc71mTfNm7tok+usx2NfZCVH7VO4o2eehiqw5Ic543uODQvjr013vbVhIFK8tNmA0WhMoS3uPpSo1L55TObYC+EAJ7uGYR\\\\\\\/8OJmajV3HWOxdp+0dYmEW5XGT9U9cu0zvK5fnQ8bPg\\\\\\\/jTcCW5\\\\\\\/1DcSNK4UcJVZ6XRKZM2zDGe\\\\\\\/76LTCle6\\\\\\\/JeiTG+xHJf0Vi8HKbkiLGGae31DzYoOid5Hn9Nh2Zem3d9A0Y+t5wn7kGaK3ghQnzrMyjSmhjNQQHwqX\\\\\\\/xyizr3TOY8+eOZNKCbw2HbcFKwCgEz0u9A0mIG8tidTuPS3AN\\\\\\\/AFJkQJDC+W4hB+Yg5r42bntmhRszdKsvDoB3uizfJGtsrHygQ8pj6ypd+68NcfISPWD1iqtplxa614rJKl386pBGFnp0gcuvwSysVtAk0HcC5NbAm0FXpwqS25ofb7IeoUgM21V3C74KBZGsbpVIX8ZM5tg+bgtJrecdkdtmo2D6\\\\\\\/UgOEtIPz0zzPbIe\\\\\\\/\\\\\\\/ejum3W02fmJud0BrBQiCx7wlKvYjLSJhUt3AbQHQ7GZ8DMDOaWFbTcIHwDnV0KFAMHmYl3n\\\\\\\/bPoWRuu4hEh5EPslblQD7NbMR4uawXyBnS3VAzAws2HkSbNoLe1vL1mFF2E3jeBBNOROjb+KjlIIHKO2H501qXiaAyt+OMNsfh3qnTM5fxMIwYVxVKYwXQAOlNeZ4XN0jqoYYV0dNFwf60CBbwGg0MWPXuyXp3uB4XjgSHkZF1fBu2zxR++Ozy+ugdTdrpuOAbjAGZuLwLU\\\\\\\/YT2jK4EXVeKOOxDCc8EBOzHsLfWv6W5J\\\\\\\/bJP+YxVgofgT70zefqEDat\\\\\\\/WYCpk0q1MP5O4\\\\\\\/vgsnRNMKyF8ZtPY574vl3qKd6ljPNJAfanSlW45RGwR2gtmYnf5VvJHCHB7elbwLhN6ZCp52Nt9dK8Sdjq1vnEAJbXjvQCDMWzA\\\\\\\/anqROx2DyTPlvT06pjGa8mmGu71niTqB55Am\\\\\\\/ifA6kncr7Rld+u+L6uL7iAmpl8ogQIRN\\\\\\\/eOyS6Yinj3siyT88D3hur\\\\\\\/maL4IoQi1eqok+QLUdgglMI36U2mE52\\\\\\\/eS4zlNKVJ\\\\\\\/7d3Z3V5KgecNkSqEB5zvBHfhlhRj8d2L+RbCDvoncTcekYsBi1UcS9t3wFTtKruCla66eOMcA9BTo\\\\\\\/T\\\\\\\/6FJ4tMZyw15WWZMB2qDweOsOqCP4q+oYSlEN+bmpoNzFQo9LE8AqVkRlCfp\\\\\\\/PxA42IEyLIUyS+2+Eh9Hk8QHhzuHsgjopBJmBD9H4EPvbauPM2Qj\\\\\\\/1YAf\\\\\\\/whtb8s6d3hUvlAdmMnR8YeMU8mkiVqU8Ewb+0V32Ex4oVwwMeR\\\\\\\/UTNQH41k1wuj1Z9w13\\\\\\\/5Bwgn3w20GcP8W\\\\\\\/110E6S\\\\\\\/zOy4p+A7XFEDFoeX4xcdhqLmHUzBa0gsRXKgy7TnYsGW6rnXGi5E3u06XYcT++zDZlb1ZEPhS4jdbw1pI9nGgo+wZFJS7QLWN2lgTaKOOM9LbS+qj5GXLXi2LQekK5mG7BEHwUH3aP0x7rWO5idSXi9k6IYy8eHxhF844Ec0cO5yanehUv0wPQrf3qmzi9jtD2ctmg05SKCTqoDkSpy7vR3WOfB9AhuOD8gkace1qmVo+\\\\\\\/eaNmp7gvivcOJvAe78VsyNSbbrhWuToxoRQ9Wvd\\\\\\\/AL0E\\\\\\\/t2KZTC6Y+JP7FHIDa4RuHpZmPTBVG+MMrBAaqC+j+Wi4+yonfbh1ZREcxZfT\\\\\\\/iE16BiKeBFMcG9TZxobcuIoIjtr4I7\\\\\\\/WSsYxpsgDDsoyQliwktUtZcpyWXGZYIX3nVlzs\\\\\\\/tdZCIiXKzYE2uYEwiZDjjVwI8tLCfzUg59Tt+5XBR2\\\\\\\/atRqDCHCwKt5Y8uzCePwaV\\\\\\\/Imy1LqL43fzjqJBltcnZhEXnGbmVx99BD29B0HVNviL9Q9QVR6PNTgRDHtPvt7MTG7d4CQL1+BGryepbnS6N43xQ\\\\\\\/CaQQAMRwWSx+v4BAbs7zNCptAnuv4C5NqxnDQy1I8CwcZEwTUAE8CbaLQDB56TLPaXjZu+ompXNjpZ4ktQ8waBcHHKPettFvBoprSe1psTAIn3XE03Rxrbsh3id+gCKVRNfCXy8fPkadvsO6ofZK6AbnBvN\\\\\\\/p090syyHwNmRgMf3Puj0CqS8GseCYitHCh\\\\\\\/hXMDaTNn6y9r5\\\\\\\/scqEi8suen43aueca9U1rOEJAlPGr75zoU9cnK70E6uXg4zO\\\\\\\/Og43q+YZLVEbosTNx6nyPYNhzylf9SWBsRkafUFmfZxX3LwaY4AKDm5x7Rm+84a56nQBZeXULhhC8BNKwSv5AO9SqIgBeqIB\\\\\\\/QQNJRvKG5Kijpjox6A5psBRNLZRfG0iFeUJen5tJn2NfvdniioBpGuYKm3J55M0odFOVFFF93a4EP8uWM9nacRVh6J+HBzMsPM1\\\\\\\/+1pbvyRLit4gnTl08NjYqxPlM7BFSU2bIcdHgKqC2doX+Oh0uXXle8VsQ\\\\\\\/3rmgazjUs1VOjpjjEU00hTKQ7oNo7hAOL\\\\\\\/LMboZIXvMmVtSScaCZC+tRzFedv36jmbauRw5jxwCr23OfO8jlAW4J9UOrKgPKqwRnsAx7AEe0ZK\\\\\\\/G1s31qme\\\\\\\/LgFNSf1iVQyAovNbrW6M9KELosPUgYe8xxZSj6rvzIiORTbclgyCsJKAT+h\\\\\\\/zjo3wUxMqlDx800FPrQSCq8k2Y\\\\\\\/qtT8NMWFI0FrUgsi69x4Y0+J+viEAC5Q0ChU2KSkoDuszCXlWncNMU1axglW12oqIPQ1brhfZCFP\\\\\\\/8vfg1Qb3XPY5nA3BQRtU8kdIE4zfn\\\\\\\/+QqxGkzWLHgyf3mLzhixYkvUoIQAYHE1LCYU5ImBu+Gq3SweoSGxCJyA\\\\\\\/2nS\\\\\\\/XqayFar4lA0yiYntNLmW27HHeTZ2U6v9RIq5W2EehgtE63koxTIyNtZFmGmxWjbaUKVWIZJt005+bk9k1uSAP96wEPqqsw3fHiqnAswjF7Bjgqlnc693\\\\\\\/0oINwrTcbNwZc5DjSD4E2cfy\\\\\\\/30\\\\\\\/8H6aDcmN8s7CR41dsaPRZQ+tSRxhgNv+nrnIdxi8tWefLW+GFnkipWVZK6VSYKlMQkXkIMzlCYUh5vUF8834YAXxSTFqsnftLzbfitMX+Cch2Ojvm1UDxWLPE\\\\\\\/mzOafStau7uBM0SDUIACmpAg6EQoZfcL6EcSkIY4xXB3f\\\\\\\/XnJSY90iJiWOr6WoA\\\\\\\/MLHMRIP\\\\\\\/euMEssb5aOWJ6Jl3CLu539pHJkAaElBxGujLPZh79XeSpdFXJhv+J+saER5fBjJqWivBuGJxKelTKJ0To5EfJXC\\\\\\\/PWY5Dja40W5EN6\\\\\\\/1NNSPTfqVRUeqyA7Hsky8lSKadJ7StADFkWpLuuaBWAghedck4TGioCIyXOm44tEqQ8TFP8MJiHG2cLymUsc0GEIKUDTF43SzovYVU+Hrt\\\\\\\/MBM6dcNwy\\\\\\\/bqkmb\\\\\\\/Fb89ncrVsbH6mt9iaZS0\\\\\\\/jPzzXKEcHyhIx5IEftTglw0GHOPNy5wSF3\\\\\\\/\\\\\\\/JUx56PPXg4gy4IaHj+H0tgfahIuvcQOMhqONUR+L8jz7yVZW\\\\\\\/pFD+clrkAgtsFuOk2pcBXeEoXlvo4IkwrEaLMgXTBeAAkbR7TD3qqDxDl\\\\\\\/qYN1IGRsJ7ira8951kB2YlCBQgmmEZB5ZzAM6mK75MisUwZQMJmYGEk6c7Of4J8XquG1JSEa08WeVDwOY3seiAGZiWegU3yuvPOkmCUGXmol1kPagUfjac2zFHCtP7fE2qUU6I+43UGOy0B60pO0tjScHjUzrZu7Qae2YrlSfANt05z9TK8tl5HwDst9tLNYeRMa6ZyM4vba4QJy+s49T8hrePoEz7jOiQz\\\\\\\/89ZV+f\\\\\\\/Ys\\\\\\\/puNm+gwAPC30rOcrxV+fjpSYKUUD8N9Y5W32tHW0ungwTbJTZwGCe0Lr\\\\\\\/iv5zYKBVig35haOFbrAwNKupB3tN7u40V7qc8DXhy4ZIP+biMCOzvhn3ROs2JRFZJYuH55UPUb9aQepDAtfS+\\\\\\\/\\\\\\\/pFSB\\\\\\\/dyWBbWDemJByEUjcquNpeqBaotLpGDp\\\\\\\/NYfx9IZUrwVrP80F\\\\\\\/D9zzivWmeGayYjw7rZotab1ocpZxrqN96GsphI6OXqSxK44s5mxAlYkKiyb\\\\\\\/526Qb99baMwGcotk3NcGR\\\\\\\/Tap\\\\\\\/T49rUd9SNdusYQd2ucgPLdRoJyQ7qHY2CETDT3S6jGHK7qEG7+QyQ2bOKbcXiBEGvU6T+hL3AOrA8FoIUEw7LUKH\\\\\\\/+d\\\\\\\/i44nyuKM4oWnahW1pS8S+Wo2HWvzDOheVPD+BTz8TdrudjrUGcTs+NgBmKRyWjEnnxaeSCv5lfo2N0IodVVK2ha+VdUwwtu5XXsnAPI1Gr13mPtWk5JRFCMwV\\\\\\\/nGLYFjKO7D5XV\\\\\\\/mdiDMtel6DQXa9ge9ZiQ1OaIG1WGvWpDyQtI6VdLF19i\\\\\\\/2DF+\\\\\\\/Tt4DBLRBtT\\\\\\\/1gNNf0wm1NxBte5Nn8dlOuU+n5lNAOZm8UXtQP946Ok6glmJYGEibuDNkS5g6cYOZEoIUS\\\\\\\/0HI558K\\\\\\\/cv8IDfi4VFWEmcmd2aQOHb2lQatYx0A78el2Y31PyyCDg5LUQFsAHpctY0UuHfWkpDoNLFTA5a4JveeoBB5Q7N2QX3X8lkoYitwX2GmUu0RAJOhm2EUog8kMl6qA6TlyKjYHldX6aDv2WdRwW4n5SWvExHYTLnIhA2kS3CsyldTYp1IG9l50hd5YutnKwdaJTnzuwesx65fTz4EvNymw3qJBUntfl4S2HY2mXINIRPGyoWqFhxghVbqqgmFbeYLdIxzIWbeRI6Dl\\\\\\\/Tz1xTiqjaGkRZOvTwjcsNRBT6ppFMrJqnPIAraQI02rmY22\\\\\\\/Z3gPiYKig2PzxKHsck4SRno2tP3WysItWbMXMDAm5Slr0AG\\\\\\\/GSHQhnOFFDHCQTNl1v1nByFF7Bv6koBbPpruWGmREl9HPrpyMcmcMJRsMkIiJJalRqeUp1XWfy+vM9QDvg4OFWk0+3kwj6gewleLfgvkxkLNlVGAXk8Dw7KkUMGPsp7OvHxgi1xbLuIcslQyiCY9mQ4j4EIK7x\\\\\\\/rm8DMGLw6VFD+FgjQnp8SjGvcpM1ZpGs8Ejt9rsWExXauQyfvvpuRKCUyizr\\\\\\\/7sysvLCQMSm4g36dxNTJk9KqFV9Z9nA1gS8pJOJhpEKWjKHRpJy0dc8B\\\\\\\/V6GzQ8Mmy\\\\\\\/zW44xLdwfqd0qMijNp5\\\\\\\/5tXyaNKxZmuEraOpeAeu6BZwFvSDskNf1bnheh9LgDmMh9XQX7d6Gi9vDaHS0dQTc53jSIBMWpF+bd5JFndO8NxPi+aF+8BYEESENQ5trMck+4gTY5D5FUMDRzmMiYPOWfe8+\\\\\\\/P9cx6nWT\\\\\\\/BukHY1HXknaLk3ZJZTNY1S6iQhqSf05Ryn6SayySobAgWWtSkL61zBqbuk8ExP\\\\\\\/vmGOBC72iEb3+kIOJSUuzwL+7oR9xclpEgGVQZwgv6m8\\\\\\\/NBRCMBgwtlesvEdYMezGEjrVicmjR+G6hP2kAIfNiDVux4Izq+5e3Rh7TNAfK5yGU4wHTxgCw0br2PYjro\\\\\\\/tXsjsEyKJ0j\\\\\\\/lnIa\\\\\\\/u\\\\\\\/taDGb4s1YlfxDy23DV5rXnyUTvsjTMp96Og4oIuwyE\\\\\\\/dXPeetc9r8qiumx1X4yacrc72Yx4mVU29hMlKKzQbuzDEtUS\\\\\\\/Bmm8lTKA5Zsprb76xymlsODHwhHKSy+08bPcKT+P3QUwicCAYUElQftMdom229ndiE9YZ7MvxiiwywLs9rDERSY\\\\\\\/sNibbZPabKGZiQkSZSkBxtjxW7ahz83a68dJe5avdZj6Xlh+77HI8MXztZJGFZqLouWUYdQDJXDv6s0iivGvNGEWjXTTf10yIdbe8y3YTYmBzfDygE6VzvwKgUWaYtJdb7+698mA1VW0Pmi1j\\\\\\\/lZAnNnoQAm0uUpbKrtzrwWvzHqLUzzNYwAN2\\\\\\\/rDjBW30XFiAZtPKmkH76yBMVjltvHFn5V+eWJTWpEeLcWMLZm7O0cU9P3pYkc\\\\\\\/Ei8QFwjOp3BsAtOjaqKFpZMlkYDFdqDhRFO3rkaRWrlX9rKdSdH+JbciXatqoLugfAR3yHbFNYbREb\\\\\\\/BTy4mcyYM\\\\\\\/62tGa9xjLj+DvMLjDYhMfJSAco0jGmbLBbgnQHPO2PUAhPzeNT2Aw1si7FGXktkW+yXA7FUIAnHEjC\\\\\\\/\\\\\\\/9jyes8jRpBhysyxKq0vyLdfoJjjf3yGQ5sjX23Ypi+qAS3\\\\\\\/cM5Qc4SuOMGjhttdSNwapU0mMTd+9kTwAayIJyECAZYCeWJNSacaZR+xYOFVMrRCUfwOEqsjdkFwBtlI+SGCAjNWApjv1Qxu57EC7ENAVaMn4Ukwe45bsnbo\\\\\\\/TkPlvnfMxC5MVHN5rgLY4EZ0HIl1I7T4ZGClWrD2tbAumuntGPAsycababmpj5UMuj1gANmQT\\\\\\\/UcWSsdbG1QaaP25JDfsEnFpxSdX3pXV+mJtkt6CfX7+u4+ve5XDWM9yMI8ASrQiBgU8eMDPhk2Mjwxk6R61aCnyuoiCxjoc4wvC0nhmlfqIjXHJUnyd3ZZDulZE1U8PwTXbPyqyAXCBxTrtS5LeymXhKkEHp4n8HiLKVGyr+xSmWmahILqR+BhGPyBtPoTbasSfl24jISC5XW2JHz0PA6hF+M0MgsvXEzbrL2J7Ni+mG32s+CU7\\\\\\\/JzoM64EyOrtafjzNxYd1\\\\\\\/hhV89hZBApK7KCGynIYDstCCmAjBKMBtSxhd2\\\\\\\/as4rNFd0pxC3JOqvRnDVW\\\\\\\/TPI0Chqm6n9dluhaL\\\\\\\/1eMJUYDYUq8S29Su6HVpnf9f5nm5IjN\\\\\\\/0mBZzcWtFYer4+jaPWyXsDxHvtTJkxJBmSlFmkec\\\\\\\/9sHCNc0JY9iPbLRIfICnwbD3HZXTrpiPwXIs8SbUOk4VRFz8LTRFlG5yyByzJ2QOrODKZTm9z8T+g+f19BcRzlHRyN2ZUmvuWvG4gJkuatcGgtXpl9rPNxsDMJkcu+c3WrW6rE+IwjIcfFftr1DfX5nPpToa7ZgtmBdP\\\\\\\/icqsTUG09G+UGMRmBGKO\\\\\\\/51edGbou8q5VThPWOYZlGIOSKroUz4wGI7hohbVK\\\\\\\/QH1doPiKUtk4GaJQk7Tfa43OqS04ES1Sa0pjC\\\\\\\/xpxNdJi6ALSUhImtQoEHj6\\\\\\\/7gGJRFM157u8ukBJU5YV2gLCk1uQ+605uO30fvgYtCeyNwKnTMNHfawbRPF\\\\\\\/2t0u1XYOuEOAWyT396gqmRUD1EDQcqzTvVzZqxoSNuqj8dTxHYi1eZyzlVhmnLul\\\\\\\/Ep58o+yMKQvjdJQorLf4JOZQIsr\\\\\\\/\\\\\\\/V+2bO71dGbOOajYPukp2GGw9k05WxwTED8TCNGO10djzq\\\\\\\/r5qvb8S1JrzKjDvhR4LbLJwVzkU9NwmA0hpq\\\\\\\/1Pi07lUasJOYmsMcNzRPkLu4hsiP3FrKpVkZ97On\\\\\\\/bFTmx\\\\\\\/OmzYO4CNfX8mld9BjbbtB1GLo1hc+HqorLZi\\\\\\\/0\\\\\\\/et1BDoe41qWusRasb2cu\\\\\\\/NggQrv4KIp5XoWzO\\\\\\\/boe6HmFZPaNUVsyGL7TFXCXgBPW9ZC2ZXgSdXpofW0u+IsHpbLZiD4qaN48pyR3cxb2ebUkvahL2AjDKyoEeTsynALXqPDfblMmWuwU+Ap0upESLKk6m1VVZU6NAzcUtbduGv\\\\\\\/OTdG7AwuvnCB02N3swH4Z8cLrQJuvp5CbnHh\\\\\\\/6vLD6US4IBZcqtd0WfSC4ensHPXCblOAlzHTV9FUjgFpTpUn3T\\\\\\\/2oZlP7jaaIncMhd\\\\\\\/q5l4EQnKk4G4NTFZBLOzZiL9TeQQjbAnT5xiRYZ23NumjVbFKQdUjIm2kkP7NvZXjaewK4QPs2d8Uktj8EfpoSP38YALAyrQG7QXI1XAxaWYoK8ITeKoTznbeitVa6CoA+GNw+bVqoaLlVLTa1zyf6+fue+9QILhVlLzaYPfSTIFAFPpIaAUiBsWWGhlIv5XVXjGidJMdJT4gmpGIu\\\\\\\/mBJCTRXrkc45H9LToI3A70Jo2a+L5kNtzj13Xvn1f\\\\\\\/0aWtFGuwbROk2tKr67BuX6+8D1jseWqKVwAz6onFlV3iJkocq2pTuCCDbpvCn8PbGMQryF00w2KxWf8nwutxMDhO4KWm3T1X9bFwBb7kUYbLMv+RBrasU+yK2e77RoTGpLArmb0PzAFm2xsPf8JoJU2gOJILvXEmx\\\\\\\/4QorkJu4dcJHO0P3Nqd1hGMO6NwLSnefkmL2oHeP0O32xW\\\\\\\/Y4CTdL3lo+FO3PbMOzCwPg+2MnSLX4cIFL92iN1YCJHfxlxz50oR2bISfSid1ZirBi2E2Ml+0zRX0of5lUoAVWNFQz5M3fVO0p3tBR98Oc3C3\\\\\\\/UtvYTh3lVGpmer8IjYOjQJcbRDlIaASLdoJVJqXHtid6E1SPyWXv0eFT5Do6OTQXmaj\\\\\\\/aQbX7Hll4OPVz6Ri6ewCDkrMljyRZxXuqPIltETB+X\\\\\\\/xsMmIEEDw+5UoXYnGM7\\\\\\\/97eYgMpvmrPh2Xn\\\\\\\/GeDirlI8yL4PbGwYSIUC0CAtZjG0BfKbaducC+keBDHgLnWFSg5OjOD2smKGn1pD1KLnwIgd51Kj32MzZFkKSWwqg6bRcmGCJVUDn23xV99CF1+r\\\\\\\/xmyCmcCA+UTLL8zCgFeP+jHkF+A1KDMbLQ4tW2bkL8GQGXoLDICln1WpU5Y2B4PduXfsZ+68L1wV8xHTuiOIInAKdu0UQEmFo+0pLMXOzoMlGmxcovSeQQyz7C3vj81udyVLFapwz3TSHZR81baM3SQU3iyextk618k27gCJz7bkpBMOBH4\\\\\\\/NY0KnNCf1wVDueJiPNpv3xak\\\\\\\/KjrZ+xK9F8DL1V0m2oeK48elVRrZjZFa2XOVHPctxwP4HY9LFRT7937gtl4MPm+lPxqlZjO6MoWX9f+MJhXQch0l4zOUMvOFKGKV5fyQ0Y8wRAuWuY+FM2QpCsoYx5XjoOG+d5lYzrOJfTmabGKTn2uywwp7rFsWBUmjCvjIWVOW05dVt9nH6Hg2aEDdOYFtvryzBqA6VlDt\\\\\\\/4KimaKTQ0AF5P\\\\\\\/\\\\\\\/mKoHBxn3YQXV+15uiUIccU4BBMJeqR7CbQr6ATFYz4lsHvUpp50itSevFtdrUh5FFLbsZ7+FSuVq7imAq3aePlf7eag8E2ZfmzHZ88YMAlOTNzXa2RB9C+GabwKmwdqliDG2jA1xwIT4t5F5aMY5lVpGJHqcBtzdAhxO0s1yIXjS4xAua+D0ZaYruZ3r6juGjH8bTpNxvpGyt7Zjk220uI7a6V1X\\\\\\\/+CY5BWiJp09OAsLfFjClCKJ40OxxDeVQMZ08aG\\\\\\\/QlPZ5kjkNSMnmRFfYy+Ao7LhJRREQZ\\\\\\\/S7\\\\\\\/cWKwJQVkADtKG46kd7zRkVpQRJY7+ASYSuubi6niyhKXcRt96annTKu86J\\\\\\\/P7nuifDphzPaktU7+zPj\\\\\\\/wzeaVejZsMLLZamIeBtYsDQCloWgdS62VFNtVUj7lPJxS7CrSlHbV7lOu5G+qZYjFAo4vml\\\\\\\/S3pZ8ohc9rmB3CLj0sj8X\\\\\\\/\\\\\\\/6VcDGbo92lVu4VY1nYH4if3NJDT+b9gNp1S\\\\\\\/8yGiS9WrXcI6mETl8dD9mANP4mqPJp6uedTs9jfb42BjeoRjtQWagUnUmQIGknw8qYYInIhDK44+lS\\\\\\\/bWtwyXyVVY5SOGwrSpgGIHbvHbukO9ofkQvYbSCjlQAorgGeJkOjCJyZ9ZDT6S5ck\\\\\\\/qKxHNFXfCe2GXgQuwzAP4cNjIixRCQ6hsXcNeFU9hNOzYryi7jALTtFCTBvt9ywFWsoboKAuYiN\\\\\\\/hI9ZaHeMgjYfoduLddGEZqw3onxvykLtKhOreMho6EjwprsvDGoZ6Gt36nYpYEHOMJWa8f4l5iXct1Oz0A4yclPz4Fh1grbTrrQj4TA1rnmyQz9Tr1XTVdKpZpGsyi5JTWa3L9I3swaxsynZkiTp7lyJqd\\\\\\\/q7eG6ceduh5a5xJbyZbHxjTqEWiha44o7liRiVaRg4pxD7v9Qc6MumHrrq\\\\\\\/v4V1dIQcF0OQ1bUxrEwPVaR9ZlfGfAy8cjOjf\\\\\\\/hoHVWhYQnPylD6WZexRA21pfzML3vgns5brKYbJ2YSMipQbEKNYC2QYxJFSk9DxkUTsTlLBUzlQ5XV2Ee0gxf0dCfB7NeCORqENBkUhUkhdC8jOGHrBqO89eNasA84s6X4j34IYHMxi\\\\\\\/N3aSQLhTYd8Hhj13JOuIvG4sLYZz68ct+dyMhMauP+4QbJUlijX\\\\\\\/1skN6Awzw69Celv\\\\\\\/GrRnWzPxFPkDCcjAhh+0eM25g51QvOKb6froWQ8PPoKsqGyyWG1fW1M28xWnq+djiqSwWnlks47YpbVrfitwcZl6oUJASLOp5YwooIgN7JFc38ZOoobtg9AACz9bBaRggHcKIMqHJhzlrRfMEXM3OcS4HuBcuW5tSw4szzN2H6gTFVbxazJBM\\\\\\\/kshgIRY4hXnz9Kpvo090DgynSDElVOAZp9gLpp+Q2qNKoIOeip1QeUkUEnKdRgNtPzvIAKfm78vuR2ajG79N0e5LYJpax7XtLDdv7+lC1zpeW\\\\\\\/6LMybqXKjI\\\\\\\/hdINH\\\\\\\/izQnKnpO5WrUFV0AlyxpHfxKUiY2ShlOLNnmYNMAxH18upAsu0qmsoUQ3p6d5H1uc8DiouPxkKpg7nZUpQJye71IeGvPu2iUzGxyXRI7409RSirbP5ffzzj0irqgY+uhwQ+rflhqEVR6Wp1c6u\\\\\\\/5U51k+wqWBuLd33tK24YlMuRu7NWP9JZwSy5mUE0+H1EImcgGkj7FXVI\\\\\\\/NKod3bdmbONVvcHl+iEFaToSQQPfBNm+lVY66VQES99ksZJvt4pYyHqrqcPJmKBT6LCG2qA69e1lnBH\\\\\\\/\\\\\\\/03wu5N8Z6IkLC\\\\\\\/ofTKdopwbFq+gGm7sk+RYEFts2Jc7oHW0ZdiHx6Yt47KCLXj9r58gyHg8TGOVhSerAZ8ne\\\\\\\/6uDd9vlaDMHo9WuIg1UafDH7OGX8rOOtIfSePZVviQbtgCgKq9rvNDQx\\\\\\\/mWXEyP9aKc\\\\\\\/+NCNb+21xcQMXA4jpJJm7QNVrJGb0XaA7zTEyRJT\\\\\\\/yC0s\\\\\\\/p9Aq\\\\\\\/tXcvu3hJ4EVHGCk7qC\\\\\\\/CWiBBdVobW7b51KvWsjdkJIJCpRd1L\\\\\\\/JVH6ot5qkv2e0jd8BUDrq7Lj0b4uZOjyprn94NW82ebJ\\\\\\\/VVW8f8I51DI1sAcZgcbUJrBNgVzR0jO4TWBsSHRJD1Omf5BNUkOXnlEsotvxQM9siV6x3LXBpQRYWQajC8bwg94Vq+AnMFvAn69uBQJ4iGyd6IRynFXx6wi\\\\\\\/lWPzV\\\\\\\/0gW2vtBRWakwdFHxElnODcxiWAch5lmzzcTNxyimaVcwxSHKNqOLOX+9N+aXBL6CgObqVyf6H37WyGPAGoPocdLG5I\\\\\\\/e8VpLtg18jjFbc7BZxusNC4Pu9EAKU4Ep2HZ343CjiBsvUh\\\\\\\/D\\\\\\\/STBL+XzGbh53G9vUTkG5uN1pZWX4SteNzAAzXH3eKEGu100lMn+IWc4SwtE1Y\\\\\\\/cWnwKNV\\\\\\\/Z9BmWuwVMUdyNPUzKHVIlai0hN6MMcs+tNyxv4LpYtzJZmuHtbbytftjl\\\\\\\/CYPHDsTLCyUNxqHByKjlJQ2JRQeUJiG4VZJF8WDYVfVOUaf3JM0gBuPKhuWdEp4kjDEHe+xSvzLxNGt3bUppjCN169jnSNAVzOwo\\\\\\\/Gmry3M69pBqFZE0KRY1mgJOEyjUezGYgsPyWalSfnDhZEniCj0UhTWaFzt6pmuxM3DiStR703K\\\\\\\/ibL84CIno63E0Ve33zdrzDcnLpMfZuSxD\\\\\\\/JzzkaL5TWvthslQC\\\\\\\/R2652FBj2GLB77+Fp4KhyslsH5wTLb+x88x5A5zw+yTr2qcDKMGtNK2OstUaXconFPyQm5dJcIH64x4gsvJtQT8G0AM9TzKLXDPJN3igXVVq+uAldpr6miwGOiCTSbHh\\\\\\\/5ZduP1Zf8g5pmz+IDpKh+CXLRuML2X5CvQEXKvhHovyIVCinzaZxolgcrtRsaApUhS7oH0\\\\\\\/vP4ebnUtykZZdAKXjls0w4tWBLUCTDRzLaHoMDFge0F59RQqaVl62R4AYGy988Qlyi3XtIRrIHArejmtHuGIbdHwq8w2fL8mZLAdc2q\\\\\\\/SD+qGX3WOOC9uPBp1rRg1MrbZGrMHTm70liRTsMXM5XghV7PrrVuYom0a5Yhn\\\\\\\/\\\\\\\/OToRsf054FZ7ke8nluVsShT8wsT+u7UKxNoKouCyP6H0z8uuJWUSaK7TzKuAgF07thiVrkWgAPUxp8JQi2T66nCSXe0XbbJOikFOY6xpBCZV689c05HGf2GlU+RZcTBn4I8bsNBlCp4evoXap\\\\\\\/SpGE1ozktRgvCX5yZi4Thv3mUmfVbtJSGnoqUa+cvZ\\\\\\\/\\\\\\\/2TJotM0b3alil1\\\\\\\/ZDDCd8UU9kV5gxQHamiWoBtW6sAgGvFawDafelKL7UF1nTLd8TJfQVwOrX5HSwcfSnCxjU5olJ9KYTOJ9w+MFhUhFZUQL6YLLIqiZQq3qcmFaKj6uXF553BJJHrmhdfIa3LSt9geRbzZuRUo41VACiht2xJKTU8pE7IC4htNJHiYxlIxPYqxroVJ\\\\\\\/uCe3dmkWISpOxNty0k5P9pYZN+dW5FWfl6cwq5PZpvH8KoJMuhg0X8OkGTr+6Q7\\\\\\\/3bXwQYlV+lMmVNS7+EDzHxy8zG7c4x5ERzIagebKZ9OvIvS8\\\\\\\/+qA9ShcEg1Rquqq+CmEbaNxOdkE0nSZyz6JA+IYezDZKJomRPNAgi+rkU45BHrzFtVQJJ2ERLTToH2Vsik8nOc0daHPITtLuH0FVh\\\\\\\/jc1SgAbAvKU0dN\\\\\\\/jPFluPwPSmyljrnqeb+J6GrQd8vkD0SpKIPLHUc05kCBbT0EkpacsVrKDgyYQxRk3\\\\\\\/WWmdJUJK7Zf5vgJORZe43vG9FbQTRuhYQoxfnfbT+GzERkZVTmcrtXe2muJMJTKamBrCbkpgbZYqDu856QwzMzkUz2Q7mkBmjP6QwzY02wD5ZvkEzUEz7pP3zQLjzMym3OlRzAKJdBTnS5TbeELQ53EQNTC5jfULj30Nhiwf6oHNVHVDp\\\\\\\/NG2CpHYbzcKjrNfb4\\\\\\\/vVjpj5gSS4HHYCw81+WlcL\\\\\\\/LbgGNZUSXaSEmHa2MF1V+FRjUCnIopNVZQxhwuRjrMN\\\\\\\/c5nfljXYZ8856bKcm4Qk8xSj9yloQfGgVEuUtIZuJXLfJqULHrSHU0RnAvwe\\\\\\\/yb3QJXMJMgQY\\\\\\\/wskV19+afMSJ4XPMEzyMk0Lyaf8DjI8cehHIq\\\\\\\/kp6Q9hEJwNzgKIUpsnhrZHWtCGWM0BfEuW81\\\\\\\/8ZTJ1g3at+ua3RYndaEpv5eubW9sDoCHOZL9q709RvysJWY3SEpbBjsPcnftH7DoB5vKS+7CXGtxtcM0cD8HEV7vA0VTwu1K3IOW47VB+pmtRUmN2f0U5SQWZqC4BoYukKkrVf5O6H7aDbBQGMe8zhfmYJrTU+iEZtK8y3DlOUVz9t69CfgmmyyPVSxrSttD2cu9j1mG3fTJyth67W4WryfTjC59TNmVStDd+mHb8K8pX7CCZ2w96i\\\\\\\/FofHnDOwgYCggtCjMWmEo5iFyalZL\\\\\\\/gwXZWLu+AvlylPyK4GC7nteb8tbLoLLMNVMKe69ifqDE7yYTolbohehgo0+jfofjWdcg3ZZ9bsGLmjlPufzW\\\\\\\/dl1sn8t3VeXGm\\\\\\\/1nyrdVJxeJfrp8fKMq89EYE3st3G6ivDHHtOZoicWva\\\\\\\/HKKyQGpn+dl2eKmI1OwqkD8MR02Y6y+pG3M75dOC93pliDMHcbrg5DIrPDfykc8ECYJGzT1+FBgGwU4sgD5r46tiFX0eTO7BSzUc3KLPmu89i6VY9lLa8zy\\\\\\\/fnApHqpLqIcH88Zr1RfGenxUlAHmlG8zL2SBGS2o7mGT\\\\\\\/piRYStCrFqbH\\\\\\\/NHgFsONSRPR\\\\\\\/pcw0sRlM9xCYBLTjXvhbQODDg6\\\\\\\/vHscqjAoPe1UBlAjrXNBI+xY7PYwPHjf6SVpYMs7v2QPoqBvlDExcWTOMAfVFjNxYXSEQV+3av3jRIupWMOiBzo8Glj7h5VGD0lMJT27EX0FXTad+aI6hwDPYuRUxlj3NyIdcyvZ0ZJQe5ZOVI1wFEWVKRIVQAHEgDQYHwtNHyY3TfKO4kMGCS+b4Mgc61ARPOcTUfdKpHXpATbjzUdMmXdvdzSG6aNAb0UkgHk43S23gwZAzLkiexzSrwDxs21SZ5W0YPdNrIkZmMZXyDh6y5QFnUQvwBTDenRsmb0zmOPieJ+CjHUIumoWP9fqqroXAVMU5aJMumZeAJk1rDgDFrqTQ3NS1BUanmjMX7kF3hWOcAEddAE\\\\\\\/v3BFJ2VzRl1H3nLJ7dYWzyUewwa07WFkM4cCrBR4nRZqgb+DruravEe81BULkxkBL\\\\\\\/XPArJ+oRk6ZWKywPyiR42gCh4AYRqa6MzR0Zd0K5bIV\\\\\\\/PWhY0l2cLs383ycd1Dwyn+X3cE6noLy1tHra6L5u+2\\\\\\\/kUFmxBwSkqH3lQo6YgTxLuJSloSzqLuHEZhDhyHtIbxYsBy4mEayJsdBUovfnYEQtwFjB6utIF\\\\\\\/6wMZQjcfXyVoXO\\\\\\\/eyeMathf\\\\\\\/6UA1+KLnfDo54x9D5FQA49snPXYz9apjGqloOD2bqrAU1xH57+duwtVRMNJ6OPDa0s8KHNsrOOnYab87pmPIH9Xu0ufoGv50IndPKS4\\\\\\\/lPWYDJkt1M\\\\\\\/USNgrMew+e6034HJ8dPNF2eqQvvBln55dWU\\\\\\\/hFQqkBb1nnckjwR7msPEY\\\\\\\/VpiAxol3Fq8Voa6p5j2yli4i\\\\\\\/4IFPX3y0oftlDnu7Z4KXcWOi+SQ2Y3lRCpP1pvxIG+bAVGG94BpFDleYOSB8WhA8lVM9m5qEUHkN0e3CS07O\\\\\\\/eKdmiYHjsVHPyYkEKM+yjA2gVoml4Zn3d2X0YIWE7M9EVoTBYja0YMiM0PE7+ex\\\\\\\/Ew51vOMNc5iEQJku7jnz5XuBC7lA8Y3wB2z2sUwepgVjltzo8e18qWeN5WhQa9xIpPFcYaFEkJL+25AJHZQERK12c0DxwITSewhY+7lvX4XSuFiOBZwtAa\\\\\\\/pQxo5Nc8h\\\\\\\/zkyPvaI835vxxB5+H2riSvM0s+bEQ1\\\\\\\/XX1VTChI26ICQju\\\\\\\/CHA7oGwDH57v6PT8P0C0QuYcxekK1YWLwfnFQiLkP+SKSkwM\\\\\\\/5d8cM7nZLJ69s8vGklrRLV7hrzugE65XbBZG4x8+\\\\\\\/pFqkg3nmTd0AI34rPg1l1O0S+3PEB\\\\\\\/5NcEot4+XphMFHF8DyzT93MwvXhTWrY57jE+d9r7SjflPerxBCy8o38dl0nJnfglDxnoJkDPw+fOAVacD780VzGfOJoFB4DG+Th0pQPYmBkm+m4YhEakmUDFoYsMDyh+u9VawTVtJUVn8zB15JmcNX70fyGHJyYvQJ9uS5BEEWwhaSTWYXjKu+KPLtR2+9sT8oCG0\\\\\\\/45ujH0fgzamRV+zesuMSV\\\\\\\/q868yB3bxRFVMS0B\\\\\\\/mGDno2LagTDk6gCnKU\\\\\\\/\\\\\\\/XradO5pJcLE2Kt7UfOwAM9gTQKnmF3382W6LyTIbevawcmdGarL4VwyKSl1Z6A5FcdQN6uPVoDl7M5gpwiVD\\\\\\\/GrPr8tzsa7tcBCKHsVZtM1VLUgnBjUYivcx3Jlou8I3GGIr7TDueqAa3eK0AJkiPGVpp4brOaIny+CXpP59vSH3m5QMu4bZzlFHbovIO3cQiu5RI2pppeu4CTbRtBVbITvEvNPilq5G7eUaBhay0tqzvbGDqP+dPsS35J1U32dC4J4GfVQKZe1WMnnzBYeJRTbuvfQ1t6hwlZGysaoOwSafl0lITdcNt97OsXxYwRSJxL\\\\\\\/WO4sy\\\\\\\/K0kLYV+udr2Cgxwe8A\\\\\\\/QmoSVS0rUMJ3DDM172K\\\\\\\/FcF0c8Ok8KGq1jM0zygCPwhPxJjvRiACPg65huZXLBHIRn9nTyXMVt2nlfQ8lf9IBCIYboupZNXsbW9NkzUIJx4GKMKrFUEqSRZ3rAMUY5ffgcuYs2nXpWk3O603PetUshXl8tWlBVhZJlkOu9\\\\\\\/K\\\\\\\/KjlYSBk69Kp0xQEn1a4rxAd45KS9ArrYALPPGI7wtX5VQ9BX5u2WatAc\\\\\\\/oKUtJbIK3aINtxsYp3L66aph+CiOBtYDgybgDebVNxOunwDEAgbOBYxULwO6yCGS3Ia2tzi2VYnojxmeMh5+tEqL7GUN+zwXyHBfPVql3s+l8dGvPK8AZDw85fqOsud7FhLq2WfpxG2jorN6CzQLBRUNMEG3tzZbw28bK3Dom5OYElKbMXbGoIN6E1Bfyywd5dE4oWKM1yuJay4VHQcd5d7xRxnHcuXBx+g9WS6qg8o1bDwRgFfYNpjbzbEIukmHmIut3q1d9tZxuhCT1jcRpI5yC6oKgzoOWoXss5cIHfabFeog83QGR04eppwuouS9q7k3Gwk5+VCxtAvsgGUXY8UbiLAb\\\\\\\/an+kbjmuJuThtPKQJG7MCbFFUokitVNAxaaqwLYJFQ6bR\\\\\\\/dXmmrRZ44jzmWtasIOdP0rWzmRTF+oX3AUoczz4\\\\\\\/Z7hvgWHGGDrI77QSNJcUVXQTKcxbcZxPjJsvyqoBp3KCWQmKI+DIJA0B6y3nv5atgEX4WtM13efdDfT\\\\\\\/GvJc9v5hKZA8XcHrn5RLWhULkX1lwEaXaie8voXjDcSuEs6oUETVwQ+GOk4YMZ2egaKs8IJkoCzlNrzhpNYnDMDwXoY8MX88CgvisYwi\\\\\\\/0jgLyZP6p3RXLZETeFAtlYCCKSVMdIz+MjzBfvDczmHuxEkq37jB3vnvpklimeC52aRWSsE2RZiPPffP\\\\\\\/ODexPoGSxlnP7+Vl8d435EdMEoU\\\\\\\/r73Dwfhj\\\\\\\/itsDjovJjCZYOJ2DJLaCRWhrGQuB5Y7S0QiCbBWYhBVyagjutbiOSDP1s3LNjZkVUFZWOLuXSHGBP2aGqi2mmj4jt5jelmYP+E+bSkdJbd5cmdznxGM3TPT3CQY1MktShWCcgAhLD5hrruKEzRlH72yPNCvl8DTKs7ztWnjRhcsnVh7B9suNKbXR+FRm\\\\\\\/PHbN7N1ij\\\\\\\/g7OjxaxurmD1ch1\\\\\\\/vd\\\\\\\/BPyS7C9poWdUIVTYkiMV\\\\\\\/HXFQB1pnzDm4xTXyDJM6h\\\\\\\/NhT79XadXdw5F8Ted9B32CDcCG6Ew8kmmHMofNXorhy5Z+TENbC3Cd\\\\\\\/\\\\\\\/\\\\\\\/HbHegHsbKcpmVrLTnYlMzP8R9u\\\\\\\/uMrZrxtViVSVXkr2v4uREhNHZ8wb\\\\\\\/HVGSmyclAQXfgIk0o0X8IuJE5T2AW8guHIc96gq60hibq8+rt9NqTyFmVzOCK0vyaVzYceFrCucLrHFuwXOpOa8k030aNm7VKBmGEP\\\\\\\/NL51rC+6yJN+SuBLAhWUBVszZvdPau2pA+mbFBmPcJQelVzNCVgqT8b\\\\\\\/2rbuWv9OD3kMAtCp+ou562Jb8mF5vgxjfV5n2QeaHgCNOhEPqSWVKxMqTCLkCJvq+2ZIryyFnoG1lsoT+iG9Zk9nOFbrXDp0l95GZxh1N\\\\\\\/gDwHt8\\\\\\\/IVkdqjLiyYN1h14sElo1XRfalQL2hPvDvydug\\\\\\\/OxEd6AXKjvPDEts1lbOAgagnVDLOrBqy\\\\\\\/H2aBzeEIHZQtnDCXxMMxtOQ7LaSSq5sY5b+o1AeiJ9r+1TZTjPm1q6dOMEp4RImebuGB8tDBYdiD4nFuAdmHkcE1nGLZ47JMbr5a1y9cOLOJ55qI7Ck5yVx4rPmTninK3g6zx2fxIO\\\\\\\/jTk+vuU+lmVMOulzwZzhJNfSMdUCj8g0nwYJxwacAGiOpwG7YsNnPZdzYDg4MkGv+jm8pxuxufg7hf93aslXnGwgZBjwQGoPxz4ycPrxoxsWklEUHvvK\\\\\\\/hECRI6XqaCzl0tNW32zI0z99Nfs3TEMPqKrrBuiasyd8uZTAUE2Q6hlNO0TVCUO5HmtWj6dvoBWMUO0bd5HS1vjgy4R0\\\\\\\/z00XSdEAoozGR5hn+C\\\\\\\/DN3mwXjTpqCOrmUPl0eOpJ3irvd3glCgE7lLYLBb+VNr4vDuQM1D6FAM4jkiAELk3uUkHre+M+u6PH6ccrWE2pkwDU8rm121xixGJLoTY+KNXwO0iLYOJf\\\\\\\/o+twrgwSRPAgI5HuHRv8x4zgqClotOpZk19Ytr7X\\\\\\\/sg+lQHDJSGNqozcaT3su+1gX3OGcyDHlGh37dbxxblgTGK8W7\\\\\\\/bdyiFsLot\\\\\\\/e0khYsGtf0mtW\\\\\\\/uELsEc86Non5lc37qvpO1FyjWTsWj3JX80QAvEbJSFIfkFaMsTGetzxt00FaCckWIRYDs7m8jtrlTp3xJEcYIr86q43jKPP9\\\\\\\/troBf4AVLdQA7IjsZRxqKTfCFqogb31HTdgnRzbNUC1SU1IhAk8gj\\\\\\\/wdt2UBJ\\\\\\\/b+XYgjMMk\\\\\\\/0r6lz8c1czaQ21Lpiw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022l+pVkxNIQLPH5vghhITj6g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00224938966bfb942e03a442fd67751f1da433533740c08a7007e19c9890a65b3bf2\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022nDRNmJTD3dlbXKKx6P4CZ9eT6B955XXPv5e7DJr1G34n0Um5F2m1e0mWmzP+A8sDzWUYpVLzufT881G1sn+BSR0RLlfuQoBMbFK317vPIZV4BA2JZDKpYT3zYToI0ZGeHBWUDBHohRutplwTQOGK0oTWLPSS0mGkD1HFCr7o43igDdb0JTVT1xRMlEaIhDIkGi+fZc\\\\\\\/OPWfFbrov\\\\\\\/2uVi0zBO+HvxAY3mnB16et9svs\\\\\\\/pdplMpb3Xd0L+X4GYoPkbKMHn2J2uyrhrqh1JWQNqmA0NgoFa+O46hXA6MJg0WrIWUaknDgJApsNWcLKjsL1iCoW6\\\\\\\/WONoj81osPSKWuLFpYkPfbknVVlmwGhJUQ\\\\\\\/AgFW6wIyCs15gfsoE\\\\\\\/aGpxxqLUIE0Krch1USEFdfuIl9bOm\\\\\\\/6wjr8lUxnqEnpHnouQJSotdOUofjy4RBrEjKdriJndHIFLmTGV7Z\\\\\\\/4oxR\\\\\\\/ij1bIo2hvqnXJcp8wNguuLR6eGGNqF0S\\\\\\\/iV03GjS7OXVVCv0OzZhg+UIB7o807no5w4KaAmKEH7AT6T+42vekGjBGGAM+xlW5zXt4MHeMZF11OUjkeiw2xMgk6GiJwpS5KGUtjqzc0ecA8\\\\\\\/tCPeNWYqmcH\\\\\\\/02DRMDaJSyW4jjKTo2q0do7sEOTN6Oh3gpsLlh88\\\\\\\/Oj1W5iysy92wBtA\\\\\\\/6HVFI0THNcYicmnuOROBs3TRLvXI+PBGNIMfDq+wItyG9yOuq\\\\\\\/mtHbMUdz2oCEDUwpM42ZGYM1b5SrC56UvpHP+lpivvFLabHCjs0892z5LkNaaIjYFTTxh\\\\\\\/wMCNl58Z79zJbahF6P2M\\\\\\\/7KDqPtZ2yWm7GCtB4k7Ez5DRiKTNWZi7xdH6TpypkRqwvGetgSS4e712c1iyBv7U11snHgec9pRdlGjpj0cI0Cb\\\\\\\/e+KXbyGeowWRSjehs07tbl3Y7jJwd83ezkGcLLSBmc52fdKPn+aJvKo6qgviUCXLi1Iv3LewNmLYG6Pt8bhk5D8k1XyeSGzq2TWcO2z0tkrGjcU6+saWBTqx0fFl5ddwyBzPqsoQRSe807esBEZFnOmdEwxJ+NkPkK+n2O4jVmL9wNL8YNY\\\\\\\/Zi6l31wgQdjWUyd\\\\\\\/AQM+yCe8qGvRg7QmRHJPYqudTWb6Ar9s8pi6EG+9V77oUoEhTTsf68+zoMQZD02KW1chzVSfXgEKW1k6G09qCWvygIAQHgSQbzYFIwCHbQtI6MieR1YpoGXkk+Rw203ZRjRTwhjVDeRq0BxOcOh0Sw3DIgZqfWxtsKgwjA+XtRB+i7\\\\\\\/6yrkobBVGszXu\\\\\\\/HSV3f\\\\\\\/FykMzDaImUFY7hj0xX4RKnAxUnpQTIa1xtQ0WnV4XiYqJbU4\\\\\\\/nZAhJrID7oJ+eQ1bOQipOxI9VQ6+OqgvLJWCaNPkjSdTUvw7hm3I56Cq8uqNBtuwKJAUEA+bPONurgDirTpKut\\\\\\\/tCFWqWh5vhM2L+Q1rVK96IvaRKV2pwBLOwPtxF57TO8f6bTeq4X+hkoEH3tX5ET\\\\\\\/Xu+iNWc68C66sYmPymM25e8AXRchB0strxnj5AWQql8czMmoU66gqb4cBX3M\\\\\\\/kt5vfMJumwjqoDBdz+V5T9ZX\\\\\\\/sFMt6M\\\\\\\/e0wOEcmD15LQZqQXtAQmwtaa7JQzrDEKcVCSg0oVF1oIU61io6jFnJ+GmfQT56tpxq8RsceDN2o\\\\\\\/BSSZurLQpG2M3yltrs2cpqZtG5ykLxpAT0jQxqJkRSYNBrwMmY6cdTpG665sbwAX769JGE6I6oGVqHCAUWe3YdWqn2prJ8EXs+Ais9M0+ojtKVtgq6uAN8q4gr4UBQyd8yRYsNx9or89OPajlqg9khTWT6rtsB7JNiCYOE7Q7lKW+WURDuZ6M4ctEnKqT+\\\\\\\/6Dib1\\\\\\\/ar4JCkVpFDgC9WyjCg3Nie31n4F3x3MGRpsMJPxZbTy+fnYlQGTNxaXW1IvSMc0Ym6ePPdhjf7QERg5uRPXZrEwsbNIBXQySwxPf7TtSNRJRRLNSSTUATvZ0AgCsA+NW4IllNMU9hFJVQ==\\\u0022,\\\u0022iv\\\u0022:\\\u00229QX2wZbY+594dPxiF9+7LA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00226097d6478cdab9d0dd72537e53e9cf689f0c793a4db347287e6b02e189da2856\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460075 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/3/695" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-6i6sFLP8-Tv16kaWjZpiKecT3rmnx52F0MAZoIT2ioc" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/3/695" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-UZH8uu38Zt-uu49M9piR1jYudodspN_19Wjb-LdZAFQ" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/3/683" title="Diverse Hormone Response Networks in 41 Independent Drosophila Cell Lines" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/3/709" title="The E2F-DP1 Transcription Factor Complex Regulates Centriole Duplication in Caenorhabditis elegans" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460075" id="node460075" data-pisa="ggg;6/3/695" data-pisa-master="ggg;g3.115.023655" data-apath="/ggg/6/3/695.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Major Improvements to the <em>Heliconius melpomene</em> Genome Assembly Used to Confirm 10 Chromosome Fusion Events in 6 Million Years of Butterfly Evolution</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-1017-9775" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>John W. Davey</span>, <span class="highwire-citation-author" data-delta="1">Mathieu Chouteau</span>, <span class="highwire-citation-author" data-delta="2">Sarah L. Barker</span>, <span class="highwire-citation-author" data-delta="3">Luana Maroja</span>, <span class="highwire-citation-author" data-delta="4">Simon W. Baxter</span>, <span class="highwire-citation-author" data-delta="5">Fraser Simpson</span>, <span class="highwire-citation-author" data-delta="6">Richard M. Merrill</span>, <span class="highwire-citation-author" data-delta="7">Mathieu Joron</span>, <span class="highwire-citation-author" data-delta="8">James Mallet</span>, <span class="highwire-citation-author" data-delta="9">Kanchon K. Dasmahapatra</span> and <span class="highwire-citation-author" data-delta="10">Chris D. Jiggins</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">695-708;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.023655 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460075" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">John W. Davey </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Zoology, University of Cambridge, CB2 3EJ, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BW.%2BDavey%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Davey%20JW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJohn%2BW.%2BDavey%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-1017-9775" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for John W. Davey</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jd626@cam.ac.uk" class="" data-icon-position="" data-hide-link-title="0">jd626@cam.ac.uk</a>
<a href="mailto:cj107@cam.ac.uk" class="" data-icon-position="" data-hide-link-title="0">cj107@cam.ac.uk</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Mathieu Chouteau </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Centre d’Ecologie Fonctionnelle et Evolutive, UMR 5175 CNRS - EPHE - Université de Montpellier - Université Paul Valéry, 34293 Montpellier 5, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mathieu%2BChouteau%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chouteau%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMathieu%2BChouteau%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Sarah L. Barker </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Zoology, University of Cambridge, CB2 3EJ, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sarah%2BL.%2BBarker%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Barker%20SL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASarah%2BL.%2BBarker%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Luana Maroja </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Biology, Williams College, Williamstown, Massachusetts, 01267</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Luana%2BMaroja%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Maroja%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALuana%2BMaroja%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Simon W. Baxter </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>School of Biological Sciences, University of Adelaide, SA 5005 Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Simon%2BW.%2BBaxter%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Baxter%20SW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASimon%2BW.%2BBaxter%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Fraser Simpson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Genetics, Evolution and Environment, University College London, Darwin Building, Gower Street, WC1E 6BT, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fraser%2BSimpson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Simpson%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFraser%2BSimpson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Richard M. Merrill </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Zoology, University of Cambridge, CB2 3EJ, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Richard%2BM.%2BMerrill%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Merrill%20RM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARichard%2BM.%2BMerrill%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Mathieu Joron </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Centre d’Ecologie Fonctionnelle et Evolutive, UMR 5175 CNRS - EPHE - Université de Montpellier - Université Paul Valéry, 34293 Montpellier 5, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mathieu%2BJoron%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Joron%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMathieu%2BJoron%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">James Mallet </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Genetics, Evolution and Environment, University College London, Darwin Building, Gower Street, WC1E 6BT, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=James%2BMallet%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mallet%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJames%2BMallet%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Kanchon K. Dasmahapatra </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Genetics, Evolution and Environment, University College London, Darwin Building, Gower Street, WC1E 6BT, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kanchon%2BK.%2BDasmahapatra%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dasmahapatra%20KK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKanchon%2BK.%2BDasmahapatra%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Chris D. Jiggins </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Zoology, University of Cambridge, CB2 3EJ, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chris%2BD.%2BJiggins%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jiggins%20CD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChris%2BD.%2BJiggins%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jd626@cam.ac.uk" class="" data-icon-position="" data-hide-link-title="0">jd626@cam.ac.uk</a>
<a href="mailto:cj107@cam.ac.uk" class="" data-icon-position="" data-hide-link-title="0">cj107@cam.ac.uk</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/6/3/695" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460075" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460075/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/3/695.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460075" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460075/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/3/695.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460075" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460075/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/3/695.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460075" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460075/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/6/3/695.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460075" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460075/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-corrections" >
  
      
  
  <div class="pane-content">
    <div class="messages highwire-corrections highwire-correction-type-correction"><h3 class="highwire-corrections-title">This article has a correction. Please see:</h3><ul class="highwire-corrections-list"><li class="relation-68154 first last"><a href="/content/6/5/1489" class="" data-icon-position="" data-hide-link-title="0">Corrigendum - May 01, 2016</a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-4">The <em>Heliconius</em> butterflies are a widely studied adaptive radiation of 46 species spread across Central and South America, several of which are known to hybridize in the wild. Here, we present a substantially improved assembly of the <em>Heliconius melpomene</em> genome, developed using novel methods that should be applicable to improving other genome assemblies produced using short read sequencing. First, we whole-genome-sequenced a pedigree to produce a linkage map incorporating 99% of the genome. Second, we incorporated haplotype scaffolds extensively to produce a more complete haploid version of the draft genome. Third, we incorporated ∼20x coverage of Pacific Biosciences sequencing, and scaffolded the haploid genome using an assembly of this long-read sequence. These improvements result in a genome of 795 scaffolds, 275 Mb in length, with an N50 length of 2.1 Mb, an N50 number of 34, and with 99% of the genome placed, and 84% anchored on chromosomes. We use the new genome assembly to confirm that the <em>Heliconius</em> genome underwent 10 chromosome fusions since the split with its sister genus <em>Eueides</em>, over a period of about 6 million yr.</p></div><ul class="kwd-group"><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/heliconius" class="hw-term hw-article-keyword hw-article-keyword-heliconius" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Heliconius</em></a></li><li class="kwd"><a href="/keyword/genome-assembly" class="hw-term hw-article-keyword hw-article-keyword-genome-assembly" rel="nofollow">genome assembly</a></li><li class="kwd"><a href="/keyword/linkage-mapping" class="hw-term hw-article-keyword hw-article-keyword-linkage-mapping" rel="nofollow">linkage mapping</a></li><li class="kwd"><a href="/keyword/chromosome-fusions" class="hw-term hw-article-keyword hw-article-keyword-chromosome-fusions" rel="nofollow">chromosome fusions</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/eueides" class="hw-term hw-article-keyword hw-article-keyword-eueides" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Eueides</em></a></li></ul><p id="p-5">Understanding evolution and speciation requires an understanding of genome architecture. Phenotypic variation within a population can be maintained by chromosome inversions (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Lowry and Willis 2010</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Joron <em>et al.</em> 2011</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Wang <em>et al.</em> 2013</a>), and may lead to species divergence (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Noor <em>et al.</em> 2001</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Feder and Nosil 2009</a>) or to the spread of phenotypes by introgression (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Kirkpatrick and Barrett 2015</a>). Genetic divergence and genome composition is affected by variation in recombination rate (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Nachman and Payseur 2012</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Nam and Ellegren 2012</a>). Gene flow between species can be extensive (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Martin <em>et al.</em> 2013</a>), and varies considerably across chromosomes (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Via and West 2008</a>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Weetman <em>et al.</em> 2012</a>).</p><p id="p-6">Describing chromosome inversions, recombination rate variation, and gene flow in full requires as close to chromosomal assemblies of the genomes of study species as possible. Recombination rate varies along chromosomes and is influenced by chromosome length (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Fledel-Alon <em>et al.</em> 2009</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Kawakami <em>et al.</em> 2014</a>), and inversions are often hundreds of kilobases to megabases long. However, many draft genomes generated with short-read technologies contain thousands of scaffolds, often without any chromosomal assignment (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bradnam <em>et al.</em> 2013</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Michael and VanBuren 2015</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Richards and Murali 2015</a>). Where scaffolds are assigned to chromosomes, often a substantial fraction of the genome is left unmapped, and scaffolds are often unordered or unoriented along the chromosomes.</p><p id="p-7">To date, there are nine published Lepidopteran genomes [<em>Bombyx mori</em> (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Duan <em>et al.</em> 2010</a>), <em>Danaus plexippus</em> (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Zhan <em>et al.</em> 2011</a>), <em>Heliconius melpomene</em> (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Heliconius Genome Consortium 2012</a>), <em>Plutella xylostella</em> (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">You <em>et al.</em> 2013</a>), <em>Melitaea cinxia</em> (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Ahola <em>et al.</em> 2014</a>), <em>Papilio glaucus</em> (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Cong <em>et al.</em> 2015a</a>), <em>Papilio polytes</em>, <em>Papilio xuthus</em> (both <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Nishikawa <em>et al.</em> 2015</a>), and <em>Lerema accius</em> (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Cong <em>et al.</em> 2015b</a>)], and several more available in draft (<em>Bicyclus anynana</em>, <em>Chilo suppressalis</em>, <em>Manduca sexta</em>, and <em>Plodia interpunctella</em>; see LepBase version 1.0 at <a href="http://ensembl.lepbase.org">http://ensembl.lepbase.org</a>). Of these genomes, only <em>B. mori</em>, <em>H. melpomene</em>, <em>P. xylostella</em>, and <em>M. cinxia</em> have scaffolds with chromosome assignments.</p><p id="p-8">The published <em>H. melpomene</em> genome (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Heliconius Genome Consortium 2012</a>; version 1.1 used throughout, referred to as Hmel1.1) contained 4309 scaffolds (“Hmel1.1”; <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> and <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>), 1775 of which were assigned to chromosomes based on a linkage map built using 43 RAD-Sequenced F2 offspring (Supplemental Information S4 in <a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">Heliconius Genome Consortium 2012</a>). The total length of the genome was 273 Mb, including 4 Mb of gaps, slightly smaller than the estimate of genome size by flow cytometry of 292 Mb +/− 2.4 Mb (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Jiggins <em>et al.</em> 2005</a>), with 226 Mb (83%) of the genome assigned to chromosomes. The resulting map has been good enough for many purposes, including estimation of introgression of 40% of the genome between <em>H. melpomene</em> and <em>H. cydno</em> (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Martin <em>et al.</em> 2013</a>), and identifying breakpoints between <em>Heliconius</em>, <em>M. cinxia</em>, and <em>B. mori</em> (<a id="xref-ref-23-4" class="xref-bibr" href="#ref-23">Heliconius Genome Consortium 2012</a>; <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Ahola <em>et al.</em> 2014</a>). However, for understanding these features and mapping inversions and recombinations, Hmel1.1 has several limitations.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/3/695/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genome assembly quality. A perfect assembly would appear as an almost straight vertical line. Horizontal plateaus indicate many very small scaffolds. The top right end of each curve shows the number of scaffolds and genome size in the whole assembly. See Table 1 for statistics." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1884001071" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Genome assembly quality. A perfect assembly would appear as an almost straight vertical line. Horizontal plateaus indicate many very small scaffolds. The top right end of each curve shows the number of scaffolds and genome size in the whole assembly. See Table 1 for statistics.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/3/695/F1.medium.gif" width="440" height="296"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/6/3/695/F1.medium.gif" width="440" height="296"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/3/695/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/3/695/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467613" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-9" class="first-child">Genome assembly quality. A perfect assembly would appear as an almost straight vertical line. Horizontal plateaus indicate many very small scaffolds. The top right end of each curve shows the number of scaffolds and genome size in the whole assembly. See <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a> for statistics.</p><div class="sb-div caption-clear"></div></div></div><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/467716/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/467716/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Statistics for genome assembly versions</span><div class="sb-div caption-clear"></div></div></div><p id="p-11">The original RAD Sequencing linkage map used to place scaffolds on chromosomes in Hmel1.1 was built using the restriction enzyme PstI (cut site CTGCAG), which cuts sites ∼10 kb apart in the <em>H. melpomene</em> genome (32% GC content). Scaffolds shorter than 10 kb often did not contain linkage map single nucleotide polymorphisms (SNPs), and could not be placed on chromosomes. Also, misassemblies could be identified, but only corrected to within ∼10 kb. With only 43 offspring used in the cross, the average physical distance between recombinations for Hmel1.1 was 320 kb. Scaffolds that could be mapped to a single linkage marker, but not more (and so did not span a recombination), could be placed on the linkage map but could not be anchored. Either only one scaffold would be placed at a single marker, and could not be oriented, or multiple scaffolds would be placed at a single marker, and could not be ordered or oriented. While 226 Mb (83%) of the genome was placed on chromosomes, only 73 Mb (27%) of the genome could be anchored (ordered and oriented). As 17% (46 Mb) of the genome could not be placed on the map, consecutive anchored scaffolds were not joined, as unplaced scaffolds may have been missing in between.</p><p id="p-12">Although the primary Hmel1.1 assembly contained 4309 scaffolds, an additional 8077 scaffolds (69 Mb) were identified as haplotypes and removed from the assembly (Supplemental Information S2.4 in <a id="xref-ref-23-5" class="xref-bibr" href="#ref-23">Heliconius Genome Consortium 2012</a>; “Hmel1.1 with haplotypes” in <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a> and <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). These scaffolds contained 2480 genes, and have been used in several cases to manually bridge primary scaffolds and assemble important regions of the genome (including the Hox cluster, Supplemental Information S10 in <a id="xref-ref-23-6" class="xref-bibr" href="#ref-23">Heliconius Genome Consortium 2012</a>). It seemed plausible that the assembly would be improved by better genome-wide incorporation of these haplotype scaffolds, rather than their removal.</p><p id="p-13">Since Hmel1.1 was published, long-read technologies have matured to the point where high coverage with long reads can be used to produce very high quality assemblies for small or haploid genomes (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Berlin <em>et al.</em> 2015</a>). Several tools are also available for scaffolding existing genomes with Pacific Biosciences (PacBio) sequence (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">English <em>et al.</em> 2012</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Boetzer and Pirovano 2014</a>). However, these methods are limited by requiring single reads to connect scaffolds, whereas it is likely that many gaps sequenced by PacBio sequencing, but missed by Illumina and 454 sequencing (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Ross <em>et al.</em> 2013</a>), are longer than single reads. An alternative approach is to assemble the PacBio sequence, so that PacBio-unique sequence is retained, and then combine the PacBio assembly with the existing assembly, but tools for doing this have previously been lacking.</p><p id="p-14">Here, we present Hmel2, the second version of the <em>H. melpomene</em> genome, which benefits from the use of three techniques to make substantial improvements to the genome assembly: whole genome sequencing of a pedigree, merging of haplotypic sequence, and incorporation of assembled PacBio sequence into the genome (more details on assembly strategy can be found in Supporting Information, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023655/-/DC1/FileS1.pdf">File S1</a>; see Supporting Methods).</p><p id="p-15">We have used Hmel2 to test the hypothesis that the <em>Heliconius</em> genome underwent 10 chromosome fusions since <em>Heliconius</em> split from the neighboring genus <em>Eueides</em> over a period of about 6 million yr. It has been known for several decades that all 11 <em>Eueides</em> species have 31 chromosomes, whereas <em>Heliconius</em> vary from 21 to 56 (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Brown <em>et al.</em> 1992</a>). It was previously thought that <em>Heliconius</em> gradually lost or fused 10 chromosomes via the <em>Laparus</em> and <em>Neruda</em> genera, which have chromosome numbers between 20 and 31, and had unresolved relationships with <em>Eueides</em> and <em>Heliconius</em> (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Beltrán <em>et al.</em> 2007</a>). However, the most recent molecular taxonomy of the Heliconiini (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Kozak <em>et al.</em> 2015</a>) places <em>Laparus</em> and <em>Neruda</em> as clades within <em>Heliconius</em>, implying that the ancestral chromosome number of <em>Heliconius</em> is 21 and suggesting there are no extant species with intermediate chromosome numbers between <em>Eueides</em> and <em>Heliconius</em>. The change in chromosome number is due to fusions rather than loss, because the 31 chromosomes of <em>M. cinxia</em> can be mapped to the 21 chromosomes of <em>H. melpomene</em> (<a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">Ahola <em>et al.</em> 2014</a>). As <em>Eueides</em> butterflies also have 31 chromosomes, it seems most likely that these fusions happened since the split between <em>Eueides</em> and <em>Heliconius</em>, but this has not yet been confirmed. Here, we use a small pedigree of <em>Eueides isabella</em> to test whether fusion points between <em>Eueides</em> and <em>Heliconius</em> match those between <em>Melitaea</em> and <em>Heliconius</em>.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Preparation of cross</h3><p id="p-16">The cross used to build a linkage map for Hmel2 was the same cross used in the original <em>H. melpomene</em> genome project (Supplemental Information section S4 in <a id="xref-ref-23-7" class="xref-bibr" href="#ref-23">Heliconius Genome Consortium 2012</a>). A fourth generation male <em>H. melpomene</em> from an inbred strain was crossed with a female <em>H. melpomene rosina</em> (F0 grandmother) from a laboratory strain, both raised in insectaries in Gamboa, Panama. The male was from the same lineage used to produce the Hmel1.1 genome sequence, to ensure the cross was close to the assembly; the female was from a different subspecies to ensure many SNPs were available for use as markers. Two siblings from this F1 were crossed to produce F2 progeny, many of which were frozen at a larval stage. Where possible, sex was determined from wing morphology of individuals that successfully eclosed. Sex of the larval offspring was determined later using sex-linked markers (identified using offspring with known sexes). DNA from the F0 grandmother (the F0 grandfather was lost), two F1 parents, and 69 of their F2 offspring was extracted using the DNeasy Blood and Tissue Kit (Qiagen). All samples were prepared as 300-bp insert size Illumina TruSeq libraries except for offspring 11, 16, 17, and 18, which were prepared as Nextera libraries due to low DNA quantities. Libraries were sequenced using 100-bp paired-end reads on an Illumina HiSeq2500 at the FAS Centre for Systems Biology genomics facility, Harvard University. Samples were sequenced over three HiSeq runs. Sequencing failed during sequencing of the second read for two libraries together containing 24 individuals; these libraries were resequenced, but the first run data were still used, with the second read truncated to 65 bases to include only bases of comparable quality to the first read. This truncation had no effect on the mapping efficiency of these samples (all samples had percentage of mapped reads within 1% of the percentage of mapped reads for the resequenced run).</p></div><div id="sec-3" class="subsection"><h3>Alignment and SNP calling</h3><p id="p-17">Reads for parents and offspring were aligned to Hmel1.1 using Stampy (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Lunter and Goodson 2011</a>) version 1.0.23 with options -substitution-rate = 0.01 and —gatkcigarworkaround and converted from SAM to BAM format with the SortSam tool from Picard version 1.117 (<a href="http://broadinstitute.github.io/picard">http://broadinstitute.github.io/picard</a>). Reads were aligned to the primary scaffolds (Hmel1-1_primaryScaffolds.fa), and haplotype scaffolds (Hmel_haplotype_scaffolds.fas) separately. Duplicate reads were removed using the Picard MarkDuplicates tool. Indels were realigned using the RealignerTargetCreator and IndelRealigner tools from GATK version 3.2.2 (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">DePristo <em>et al.</em> 2011</a>). SNPs were called for each individual using the GATK HaplotypeCaller and combined into one final VCF file using GATK GenotypeGVCFs with options–annotateNDA and -max_alternate_alleles 30. Statistics on VCF files (Table A in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023655/-/DC1/FileS2.xlsx">File S2</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023655/-/DC1/FileS3.tar.gz">File S3</a>) were calculated using VCFtools v0.1.11 (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Danecek <em>et al.</em> 2011</a>).</p></div><div id="sec-4" class="subsection"><h3>Linkage map construction from SNPs</h3><p id="p-18">Full methods for constructing linkage maps, and reasoning behind the construction strategy can be found in Supporting Methods in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023655/-/DC1/FileS1.pdf">File S1</a> (see also <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023655/-/DC1/FigureS1.pdf">Figure S1</a>). Briefly, SNPs were accepted only if they passed a set of filters, including Mendelian segregation according to a root mean square test (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Perkins <em>et al.</em> 2011</a>) for a set of valid marker types (Table B in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023655/-/DC1/FileS2.xlsx">File S2</a>), genotype quality, mapping quality and strand bias. Accepted SNPs were collapsed to a set of maternal and paternal markers. As recombination is absent in <em>Heliconius</em> females (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Turner and Sheppard 1975</a>), the maternal markers acted as chromosome prints for each of the 21 <em>H. melpomene</em> chromosomes. Paternal markers could then be assigned to chromosomes, where they were colocated with chromosome prints on genome scaffolds. MSTMap (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Wu <em>et al.</em> 2008</a>) was used to build linkage maps for each chromosome using the paternal markers.</p></div><div id="sec-5" class="subsection"><h3>Preprocessing and fixing misassemblies in Hmel1.1</h3><p id="p-19">The primary and haplotype scaffolds of Hmel1.1 were concatenated together into one FASTA file and then repeat masked using RepeatMasker 4.0.5 (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Smit <em>et al.</em> 2013–2015</a>), with the <em>H. melpomene</em> version 1.1 repeat library (Hmel.all.named.final.1-31.lib, <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Lavoie <em>et al.</em> 2013</a>) as input, and with options -xsmall and -no_is. Candidate misassemblies in Hmel1.1 were identified by detecting discontinuities in linkage map markers across genomic scaffolds, and then manually validated to identify the smallest possible breakpoint based on marker SNPs, including SNPs that were rejected from linkage map construction but could be assigned to one of the two markers around the breakpoint. Long misassembled regions (∼5 kb or greater) were retained as separate scaffolds, but most misassembled regions were discarded. Breakpoints that spanned two contigs or contained an entire contig were likely due to scaffolding errors; in these cases the scaffold was broken at the gap. If an entire contig was contained within a breakpoint, with no additional SNP to link it to the markers on either side, it was discarded.</p><p id="p-20">Misassemblies corrected in version 1.1 were also revisited (Supplementary Information S4.6 in <a id="xref-ref-23-8" class="xref-bibr" href="#ref-23">Heliconius Genome Consortium 2012</a>). The linkage map used to place scaffolds for version 1.1 was built using RAD Sequencing data, with samples cut with the PstI restriction enzyme. This produces sites roughly 10 kb apart, which meant that many breakpoints were not identified accurately. Each of the misassemblies was reconsidered here, with all of the previously broken scaffolds remerged, and new breakpoints defined based on the whole genome mapping data.</p><p id="p-21">Errors in the linkage map were identified during the merging and reassembly processes described below. A list of linkage map errors was constructed, and erroneous blocks removed and corrected using a script, clean_errors.py.</p></div><div id="sec-6" class="subsection"><h3>Merging genome</h3><p id="p-22">HaploMerger version 20120810 (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Huang <em>et al.</em> 2012</a>) was used to collapse haplotypes in the <em>H. melpomene</em> genome. A scoring matrix for LASTZ (as used within HaploMerger) was generated using the lastz_D_Wrapper.pl script with–identity = 94. This scoring matrix was used for all runs of HaploMerger, including for the PacBio genome (see below). HaploMerger was run with default settings, except for setting- size = 20 in all_lastz.ctl, targetSize = 5,000,000 and querySize = 400,000,000 in hm.batchA.initiation_and_all_lastz, and haploMergingMode=“updated” in hm.batchF.refine_haplomerger_connections_and_Ngap_fillings.</p><p id="p-23">Several scripts were written to make running HaploMerger easier. The new script runhm.pl executes one iteration of HaploMerger, running batch scripts A, B, C, E, F, and G, renaming output scaffolds with a given prefix, producing a final FASTA file concatenating merged scaffolds and unmerged scaffolds, and generating summary statistics (using summarizeAssembly.py in PBSuite 14.9.9, <a href="http://sourceforge.net/projects/pb-jelly/">http://sourceforge.net/projects/pb-jelly/</a>; <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">English <em>et al.</em> 2012</a>), and an AGP file for the final FASTA (using bespoke script agp_from_fasta.py). The HaploMerger script hm.batchG.refine_unpaired_sequences was not used for the initial Hmel1.1 and PacBio assembly merges, retaining all potentially redundant scaffolds in case they could be used for scaffolding later, but it was used to merge the haploid Hmel1.1 assembly with the haploid PacBio assembly. The new script batchhm.pl runs runhm.pl iteratively until HaploMerger fails to merge any further scaffolds. It also runs a set of additional new scripts, map_merge.py, transfer_merge.py, and transfer_features.py, that document where the original genome parts are in the new genome. The map_merge.py script takes HaploMerger output and documents where the input genome scaffolds are in the merged output genome. The transfer_merge.py script takes this transfer information and another transfer file, for example between the original version 1.1 <em>H. melpomene</em> genome and the input genome, and computes the transfer from the original genome to the output genome. The transfer_features.py script then transfers linkage map markers, genes, and misassembly information to the new genome.</p><p id="p-24">HaploMerger sometimes merges scaffolds incorrectly, but has several mechanisms for users to manually edit its output. The hm.nodes file, which contains detected overlaps between scaffolds, can be manually annotated, with incorrect merges marked to be rejected. The revised hm.nodes file is then passed through the batchE script to update the merged scaffolds to ignore the incorrect merges. Incorrect merges in the <em>Heliconius</em> genome could be detected by comparing against the linkage map data. A list of scaffolds that should not be merged was constructed over multiple merge attempts, and runhm.pl was used to edit the hm.nodes and run the batchE script automatically.</p><p id="p-25">HaploMerger merges scaffolds based on overlaps, and reports the parts contributing to merged scaffolds in the hm.new_scaffolds file, including which of the two overlapping parts has been included in the new genome. These choices sometimes broke genes, whereas choosing the other part would retain the annotated gene. runhm.pl can also take a GFF file as input, and check for broken genes in hm.nodes and hm.new_scaffolds, rejecting nodes if they break manually curated genes, and swapping parts in an overlap if it prevents gene breakage. It then runs the batchE and batchF to update the merged scaffolds. The Hmel1.1 GFF files (heliconius_melpomene_v1.1_primaryScaffs_wGeneSymDesc.gff3 and Hmel1-0_HaplotypeScaffolds.gff) were concatenated and passed to runhm.pl to avoid as many breakages of Hmel1.1 genes as possible.</p></div><div id="sec-7" class="subsection"><h3>Pacific Biosciences sequencing, error correction, and assembly</h3><p id="p-26">A pupa from the <em>H. melpomene</em> genome strain from Gamboa, Panama was dissected, and DNA extracted using the QIAGEN HMW MagAttract kit. This pupa was taken after four generations of inbreeding, and came from the same generation as the F0 father used to construct the pedigree reported here, and the generation before the individuals used for the genome sequence itself. A Pacific Biosciences (PacBio) SMRTbell 25kb needle sheared library was constructed, size-selected with 0.375x SPRI beads, and sequenced using P4/C2 chemistry (180-min movie).</p><p id="p-27">PacBio subreads were self-corrected with PBcR [in Celera assembly v8.3 (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Berlin <em>et al.</em> 2015</a>)], run with options -length 200, -genomeSize 292,000,000), and separately corrected with the original genome strain Illumina (Sequence Read Archive accession SRX124669), 454 shotgun (SRX124544), and 454 3 kb mate-pair (SRX124545) sequencing data (using option -genomeSize 292,000,000). Self-corrected and genome-strain-corrected reads were concatenated into one read set, and assembled with FALCON (<a href="https://github.com/PacificBiosciences/falcon">https://github.com/PacificBiosciences/falcon</a>, commit bb63f20d500efa77f930c373105edb5fbe37d74b, April 2, 2015) with options input_type = preads, length_cutoff = 500, length_cutoff_pr = 500, pa_HPCdaligner_option=“-v -dal4 -t16 -e.70 -l1000 -s1000”, ovlp_HPCdaligner_option=“-v -dal32 -t32 -h60 -e.95 -l500 -s1000, pa_DBsplit_option=“-x500 -s50”, ovlp_DBsplit_option=“-x500 -s50”, falcon_sense_option=“–output_multi–min_idt 0.70–min_cov 4–local_match_count_threshold 2–max_n_read 100–n_core 6”, overlap_filtering_setting=“–max_diff 40–max_cov 60–min_cov 2–bestn 10”.</p><p id="p-28">The FALCON assembly was merged iteratively to exhaustion using batchhm.pl, as with version 1.1 of the <em>H. melpomene</em> genome (see previous section). Misassemblies in the PacBio assembly were identified using the same methods as Hmel1.1, and the merge was repeated several times to remove these misassemblies.</p></div><div id="sec-8" class="subsection"><h3>Scaffolding and gap filling with PacBio assembly</h3><p id="p-29">The final, ‘haploid’ merged Hmel1.1 and PacBio genomes were merged together using runhm.pl. For this final merge, gap filling in hm.batchF.refine_haplomerger_connections_and_Ngap_fillings was turned on, and runhm.pl edited hm.new_scaffolds to always select portions from the Hmel1.1 genome over portions from the PacBio genome, to preserve as much of the Hmel1.1 genome as possible, and use the PacBio genome for scaffolding only. Also, hm.batchG.refine_unpaired_sequences was run, and the refined FASTA output used, to remove as many redundant sequences from the resulting merged genome as possible. Finally, runhm.pl was run on the merged Hmel1.1+PacBio genome, to generate a set of nodes for use in scaffolding. Linkage map markers and genes were transferred to this final merged genome with transfer_features.py.</p></div><div id="sec-9" class="subsection"><h3>Cleaning merged assembly and ordering scaffolds along chromosomes</h3><p id="p-30">The Hmel1.1+PacBio merged genome was cleaned and ordered with reference to the linkage map markers. Scaffolds coming from the PacBio assembly alone were removed. If HaploMerger incorporates some portion P of a scaffold S into a merged scaffold, it retains the remaining portions of the scaffold as new scaffolds. These remaining portions were labeled offcuts. Offcuts were removed from the genome if they contained no markers on the linkage map, or if they mapped to the same chromosomal location as the merged scaffold containing their original portion P, assuming that the offcut is part of a haplotype. However, some offcuts that mapped to different chromosomal locations were retained, as they were often long portions of scaffolds that had been misassembled. Scaffolds were also removed if they mapped to a marker that mapped within a larger scaffold that featured surrounding markers; for example, if scaffold A has markers 1,2,3, and scaffold B has marker 2 only, scaffold B was removed as an assumed haplotype.</p><p id="p-31">Scaffolds were ordered along chromosomes based on their linkage markers. Pools of scaffolds were defined as containing one or more scaffold. If a pool contained a single scaffold that bridged multiple consecutive markers, the scaffold could be ordered and oriented, and so was labeled ‘anchored’. A pool containing a single scaffold spanning only a single marker could be ordered on the chromosome but not oriented, and so was labeled ‘unoriented’. A pool containing multiple scaffolds at a single marker was labeled ‘unordered’, as the scaffolds could be neither ordered nor oriented against each other.</p><p id="p-32">This order was refined by using the nodes (overlaps between pairs of scaffolds) identified by HaploMerger in the merged Hmel1.1+PacBio genome. HaploMerger does not use all the nodes it identifies, relying on a scoring threshold to reject low-affinity overlaps. While this is sensible when merging over a whole genome, many of these nodes proved to be useful when considering single pools, or neighboring pools, of scaffolds. Scaffolds that had a connecting node in a scaffold in a neighboring pool, which would mean that the scaffold was completely contained in the neighboring scaffold, were removed as likely haplotypes, providing that candidate haplotype scaffolds longer than 10 kb had a percentage alignment &gt; 50%, and candidate haplotype scaffolds shorter than 10 kb had a percentage alignment &gt; 25%. If neighboring scaffolds had an overlapping node at their ends, or were bridged via nodes to a PacBio scaffold, they were ordered and oriented next to each other in the genome, connecting the scaffolds with a 100-bp gap.</p><p id="p-33">Consecutive anchored scaffolds were connected together into one scaffold. This was not done during scaffolding for Hmel1.1, as with only 86% of the genome scaffolded, it was assumed that large scaffolds may have been missing between anchored scaffolds. However, with 98% of the genome mapped for version 2, it was felt the connection of anchored scaffolds with a gap was reasonable.</p><p id="p-34">After each chromosome was assembled, a set of unmapped scaffolds remained. These scaffolds were retained if they had a maternally informative marker, but no paternally informative marker (and so could be placed on the chromosome but not ordered on it), or if they featured a gene. Otherwise, they were removed from the final genome.</p></div><div id="sec-10" class="subsection"><h3>Annotation transfer</h3><p id="p-35">Using transfer_features.py (see above), the Hmel1.1 gene annotation could be transferred directly to Hmel2. However, this revealed a number of avoidable gene breakages, where a haplotype scaffold had been incorporated in place of a primary scaffold, but the sequence was still the same or similar. CrossMap (version 0.1.8, <a href="http://crossmap.sourceforge.net">http://crossmap.sourceforge.net</a>) was used to transfer as many remaining annotations by alignment as possible, using HaploMerger to produce a chain map of Hmel1.1 against Hmel2 to use as input to CrossMap.</p></div><div id="sec-11" class="subsection"><h3>Identifying Eueides and Melitaea chromosome fusion points</h3><p id="p-36"><em>E. isabella</em> subspecies (male dissoluta, female eva) were crossed in insectaries in Tarapoto, Peru. Parents were whole-genome-sequenced, and 21 F1 offspring were RAD sequenced using the PstI restriction enzyme on an Illumina HiSeq 2500. Offspring were separated by barcode using process_radtags from version 1.30 of Stacks (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Catchen <em>et al.</em> 2011</a>). Parents and offspring were aligned to Hmel2 using the same alignment pipeline described above except using GATK version 3.4-0, and Picard tools version 1.135. UnifiedGenotyper was used for SNP calling rather than HaplotypeCaller, as HaplotypeCaller does not perform well with RAD sequencing data. SNPs where the father was homozygous, the mother was heterozygous (or, for the Z chromosome, had a different allele to the father), and the offspring all had genotypes, were identified. The resulting segregation patterns were sorted by number of SNPs. The most common segregation patterns and mirrors of these patterns were identified as chromosome prints, as no other patterns appeared at large numbers of SNPs, except for where all offspring were homozygous, or where the patterns were genotyping errors from the chromosome prints. The positions of the SNPs for each chromosome print were then examined to identify fusion points, with clear transitions from one segregation pattern to another visible for all 10 fused chromosomes.</p><p id="p-37">The fusion points in <em>Heliconius</em> relative to <em>M. cinxia</em> were identified by running HaploMerger on a merge of Hmel2, and the <em>M. cinxia</em> version 1 genome superscaffolds (Melitaea_cinxia_superscaffolds_v1.fsa.gz, downloaded from <a href="http://www.helsinki.fi/science/metapop/research/mcgenome2_downloads.html">http://www.helsinki.fi/science/metapop/research/mcgenome2_downloads.html</a> on July 14, 2015). Overlaps (nodes) detected by HaploMerger between Hmel2 scaffolds and <em>M. cinxia</em> superscaffolds were used to confirm synteny based on known chromosomal assignments of <em>M. cinxia</em> superscaffolds. All fusion points could be identified using this method except for <em>Heliconius</em> chromosome 20, which was confirmed using progressiveMauve [as used by <a id="xref-ref-1-4" class="xref-bibr" href="#ref-1">Ahola <em>et al.</em> (2014)</a> to confirm synteny between <em>H. melpomene</em>, <em>M. cinxia</em>, and <em>B. mori</em>; Mauve version 2.4.0 Linux snapshot 2015-02-13 used, <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Darling <em>et al.</em> 2010</a>)].</p></div><div id="sec-12" class="subsection"><h3>Lepidopteran genome statistics</h3><p id="p-38">Lepidopteran genomes were downloaded from LepBase v1.0 (<a href="http://ensembl.lepbase.org">http://ensembl.lepbase.org</a>; <em>B. mori</em> version GCA_000151625.1, <em>L. accius</em> version 1.1, <em>M. cinxia</em> version MelCinx1.0, <em>P. glaucus</em> version v1.1, and <em>P. xylostella</em> version DBM_FJ_v1.1) on October 2, 2015, except for <em>D. plexippus</em> version 3 (<a href="http://monarchbase.umassmed.edu/download/Dp_genome_v3.fasta.gz">http://monarchbase.umassmed.edu/download/Dp_genome_v3.fasta.gz</a>), <em>P. polytes</em> (<a href="http://papilio.nig.ac.jp/data/Ppolytes_genome.fa.gz">http://papilio.nig.ac.jp/data/Ppolytes_genome.fa.gz</a>), and <em>P. xuthus</em> (<a href="http://papilio.nig.ac.jp/data/Pxuthus_genome.fa.gz">http://papilio.nig.ac.jp/data/Pxuthus_genome.fa.gz</a>). Summary statistics were calculated using summarizeAssembly.py in PBSuite 14.9.9 (<a href="http://sourceforge.net/projects/pb-jelly/">http://sourceforge.net/projects/pb-jelly/</a>; <a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">English <em>et al.</em> 2012</a>), and bespoke script genome_kb_plot.pl, used to calculate N50 values and make plots of number of scaffolds against cumulative genome length. BUSCO values were calculated using BUSCO v1.1b1 with the set of 2675 arthropod genes (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Simão <em>et al.</em> 2015</a>) using generic Augustus parameters.</p></div><div id="sec-13" class="subsection"><h3>Genome size estimation from read alignments</h3><p id="p-39">To estimate the number of true bases in the genome, we followed <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Warr <em>et al.</em> (2015)</a> to calculate GC-content-adjusted read depths in 1-kb windows across Hmel2 (details on commands and scripts used can be found in the Dryad and GitHub repositories). BED files containing scaffold positions and gap positions were constructed with Unix tools operating on the Hmel2 scaffold, and chromosome AGP files (in the Hmel2 distribution). Reads for F1 father were aligned to Hmel2 using the alignment pipeline described above (Stampy, MarkDuplicates, IndelRealigner). Windows of 1 kb were constructed using BEDTools makewindows [using BEDTools v2.25.0 (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Quinlan 2014</a>)]; windows containing gaps were removed using BEDTools intersect, and per-base read coverage across Hmel2 for the F1 father was calculated with BEDTools genomecov using the -d option.</p><p id="p-40">Median read depth and GC content was calculated for each window using bespoke script calculate_read_depth_gc_windows.py, ignoring windows shorter than 1 kb. The bespoke script adjust_read_depth_windows.py was then used to adjust read depth for each 1-kb window <em>w</em> by a multiplying factor <em>f</em>, with <em>f</em> equal to the ratio of the overall median read depth across all windows, divided by the median read depth of all windows with the same GC percentage as window <em>w</em>. The same script estimates genome size as the sum across all windows of the number of bases in each window <em>w</em> multiplied by the GC-adjusted median read depth for <em>w</em> divided by the genome-wide median read depth.</p></div><div id="sec-14" class="subsection"><h3>Data availability</h3><p id="p-41">The Hmel2 genome is available from LepBase v1.0 (<a href="http://ensembl.lepbase.org">http://ensembl.lepbase.org</a>). A distribution containing the genome and many supplementary files is available from <a href="http://butterflygenome.org">http://butterflygenome.org</a>. Sequence reads from the <em>H. melpomene</em> and <em>E. isabella</em> crosses are available from the European Nucleotide Archive (ENA), accession PRJEB11288. Pacific Biosciences data are available from ENA accession ERP005954. All bespoke code is available in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023655/-/DC1/FileS3.tar.gz">File S3</a>, and on GitHub at <a href="https://github.com/johnomics/Heliconius_melpomene_version_2">https://github.com/johnomics/Heliconius_melpomene_version_2</a>. A Dryad repository containing the Hmel2 distribution, a frozen version of the GitHub repository, VCF files for the <em>H. melpomene</em> and <em>E. isabella</em> crosses, marker databases, GC content and read depths for 1-kb windows, and intermediate genome versions for Hmel1-1 and the PacBio assemblies is available at <a href="http://dx.doi.org/10.5061/dryad.3s795">http://dx.doi.org/10.5061/dryad.3s795</a>.</p></div></div><div class="section results" id="sec-15"><h2 class="">Results</h2><div id="sec-16" class="subsection"><h3>Whole genome sequence genetic map</h3><p id="p-42">A genetic map of a full-sib cross between <em>H. melpomene melpomene</em> × <em>H. melpomene rosina</em> was constructed to place scaffolds from version Hmel1.1 of the <em>H. melpomene</em> genome on to chromosomes. The F0 grandmother, F1 parents, and 69 offspring were whole-genome-sequenced and aligned to Hmel1.1 (Table A in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023655/-/DC1/FileS2.xlsx">File S2</a>). A total of 17.2 million raw SNPs (12,858,047 aligned to primary scaffolds, and 4,362,732 aligned to haplotype scaffolds) were filtered down to 2.9 million accepted SNPs (2,525,485 aligned to primary scaffolds, and 431,488 aligned to haplotype scaffolds; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023655/-/DC1/FigureS2.pdf">Figure S2</a>). The accepted SNPs were converted into 919 unique markers (full SNP counts and marker types shown in Table B in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023655/-/DC1/FileS2.xlsx">File S2</a>; see Supporting Methods in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023655/-/DC1/FileS1.pdf">File S1</a> for further details). Offspring prepared with the Nextera kit were sequenced to a similar standard to offspring prepared with the TruSeq kit (Table A in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023655/-/DC1/FileS2.xlsx">File S2</a>). The linkage map built from these markers has 21 linkage groups, and a total map length of 1364.23 cM (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>); 2749 of 4309 primary scaffolds, and 4062 of 8077 haplotype scaffolds contained marker SNPs, adding up to 268 Mb (98%) of the primary sequence, and 57 Mb (83%) of the haplotype sequence.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/3/695/F2/graphic-2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The Hmel2 genome assembly. Chromosome numbers shown on the left. Each chromosome has a genetic map and a physical map. Linkage markers (alternating blue and orange vertical lines) connect to physical ranges for each marker (alternating blue and orange horizontal lines) scaled to maximum chromosome length (x-axis at the bottom of each page). Scaffolds are shown in green (anchored), orange (one unoriented scaffold placed at a marker), and alternating light and dark red (multiple unordered scaffolds placed at one marker). Red scaffolds at each marker are arbitrarily ordered by length. Eueides chromosome synteny is shown above each chromosome (see Figure 4)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1884001071" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The Hmel2 genome assembly. Chromosome numbers shown on the left. Each chromosome has a genetic map and a physical map. Linkage markers (alternating blue and orange vertical lines) connect to physical ranges for each marker (alternating blue and orange horizontal lines) scaled to maximum chromosome length (x-axis at the bottom of each page). Scaffolds are shown in green (anchored), orange (one unoriented scaffold placed at a marker), and alternating light and dark red (multiple unordered scaffolds placed at one marker). Red scaffolds at each marker are arbitrarily ordered by length. &lt;em&gt;Eueides&lt;/em&gt; chromosome synteny is shown above each chromosome (see Figure 4).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/3/695/F2/graphic-2.medium.gif" width="309" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.g3journal.org/content/ggg/6/3/695/F2/graphic-2.medium.gif" width="309" height="440"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/3/695/F2/graphic-2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/3/695/F2/graphic-2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467694" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/3/695/F2/graphic-3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The Hmel2 genome assembly. Chromosome numbers shown on the left. Each chromosome has a genetic map and a physical map. Linkage markers (alternating blue and orange vertical lines) connect to physical ranges for each marker (alternating blue and orange horizontal lines) scaled to maximum chromosome length (x-axis at the bottom of each page). Scaffolds are shown in green (anchored), orange (one unoriented scaffold placed at a marker), and alternating light and dark red (multiple unordered scaffolds placed at one marker). Red scaffolds at each marker are arbitrarily ordered by length. Eueides chromosome synteny is shown above each chromosome (see Figure 4)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1884001071" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The Hmel2 genome assembly. Chromosome numbers shown on the left. Each chromosome has a genetic map and a physical map. Linkage markers (alternating blue and orange vertical lines) connect to physical ranges for each marker (alternating blue and orange horizontal lines) scaled to maximum chromosome length (x-axis at the bottom of each page). Scaffolds are shown in green (anchored), orange (one unoriented scaffold placed at a marker), and alternating light and dark red (multiple unordered scaffolds placed at one marker). Red scaffolds at each marker are arbitrarily ordered by length. &lt;em&gt;Eueides&lt;/em&gt; chromosome synteny is shown above each chromosome (see Figure 4).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/3/695/F2/graphic-3.medium.gif" width="309" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.g3journal.org/content/ggg/6/3/695/F2/graphic-3.medium.gif" width="309" height="440"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/3/695/F2/graphic-3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/3/695/F2/graphic-3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467694" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-43" class="first-child">The Hmel2 genome assembly. Chromosome numbers shown on the left. Each chromosome has a genetic map and a physical map. Linkage markers (alternating blue and orange vertical lines) connect to physical ranges for each marker (alternating blue and orange horizontal lines) scaled to maximum chromosome length (x-axis at the bottom of each page). Scaffolds are shown in green (anchored), orange (one unoriented scaffold placed at a marker), and alternating light and dark red (multiple unordered scaffolds placed at one marker). Red scaffolds at each marker are arbitrarily ordered by length. <em>Eueides</em> chromosome synteny is shown above each chromosome (see <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>).</p><div class="sb-div caption-clear"></div></div></div><p id="p-44">In addition to mapping the majority of the genome sequence to chromosomes, whole genome sequencing of a pedigree allows very accurate detection of crossovers and misassemblies. Identical SNPs could be concatenated into linkage blocks across scaffolds. For example, across the scaffold containing the B/D locus, which controls red patterning in <em>Heliconius</em> (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Baxter <em>et al.</em> 2010</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Reed <em>et al.</em> 2011</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Wallbank <em>et al.</em> 2016</a>), six crossovers were called, with an average gap of 344 bp between linkage blocks; a misassembly at the end of the scaffold was called with a gap of 2.9 kb (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). Across the genome, crossover and misassembly gaps have a mean size of 2.2 kb (SD 3.7 kb), all unmapped regions (crossover and misassembly gaps, unmapped scaffold ends, or whole unmapped scaffolds) have a mean size of 2.5 kb (SD 5.1 kb), whereas mapped regions have a mean size of 28.4 kb (SD 62.7 kb) (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023655/-/DC1/FigureS3.pdf">Figure S3</a> for distributions).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/3/695/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="SNPs across the B/D locus scaffold for the major marker types Maternal (F1 mother heterozygous, F1 father homozygous), Paternal (F1 father heterozygous, F1 mother homozygous), and Intercross (both F1 parents heterozygous); see Table B in File S2 for marker type details. Kinesin, Dennis, Rays and Optix are major features of the locus (Baxter et al. 2010; Reed et al. 2011; Wallbank et al. 2016). Vertical lines, SNPs; horizontal lines, linkage map marker ranges (cf. Figure 2). SNP colors: black, maternal pattern for chromosome 18; alternating blue and orange, linkage map markers from 1.45 cM to 11.6 cM on chromosome 18 (cf. Figure 2); gray; misassembly, now on chromosome 16." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1884001071" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;SNPs across the B/D locus scaffold for the major marker types Maternal (F1 mother heterozygous, F1 father homozygous), Paternal (F1 father heterozygous, F1 mother homozygous), and Intercross (both F1 parents heterozygous); see Table B in File S2 for marker type details. Kinesin, Dennis, Rays and Optix are major features of the locus (Baxter &lt;em&gt;et al.&lt;/em&gt; 2010; Reed &lt;em&gt;et al.&lt;/em&gt; 2011; Wallbank &lt;em&gt;et al.&lt;/em&gt; 2016). Vertical lines, SNPs; horizontal lines, linkage map marker ranges (cf. Figure 2). SNP colors: black, maternal pattern for chromosome 18; alternating blue and orange, linkage map markers from 1.45 cM to 11.6 cM on chromosome 18 (cf. Figure 2); gray; misassembly, now on chromosome 16.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/3/695/F3.medium.gif" width="440" height="150"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/6/3/695/F3.medium.gif" width="440" height="150"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/3/695/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/3/695/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467822" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-45" class="first-child">SNPs across the B/D locus scaffold for the major marker types Maternal (F1 mother heterozygous, F1 father homozygous), Paternal (F1 father heterozygous, F1 mother homozygous), and Intercross (both F1 parents heterozygous); see Table B in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023655/-/DC1/FileS2.xlsx">File S2</a> for marker type details. Kinesin, Dennis, Rays and Optix are major features of the locus (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Baxter <em>et al.</em> 2010</a>; <a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Reed <em>et al.</em> 2011</a>; <a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">Wallbank <em>et al.</em> 2016</a>). Vertical lines, SNPs; horizontal lines, linkage map marker ranges (cf. <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>). SNP colors: black, maternal pattern for chromosome 18; alternating blue and orange, linkage map markers from 1.45 cM to 11.6 cM on chromosome 18 (cf. <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>); gray; misassembly, now on chromosome 16.</p><div class="sb-div caption-clear"></div></div></div><p id="p-46">Based on this linkage information, 380 misassemblies were corrected in the genome. This included revisiting the 149 misassemblies fixed for Hmel1.1 (Supplementary Information S4.6 in <a id="xref-ref-23-9" class="xref-bibr" href="#ref-23">Heliconius Genome Consortium 2012</a>) to more accurately identify the breakpoints for these misassemblies, and fixing 231 newly discovered misassemblies.</p></div><div id="sec-17" class="subsection"><h3>Haplotype merging and scaffolding with PacBio sequencing</h3><p id="p-47">The Hmel1.1 primary and haplotype scaffolds were merged together using HaploMerger, iterating nine times until no further scaffolds could be merged, avoiding gene breakages where possible, and reverting merges where they conflicted with the linkage map. This produced a haploid genome containing 6689 scaffolds, length 289 Mb, N50 length 214 kb (“Hmel1.1 haploid”; <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a> and <a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>).</p><p id="p-48">A 23x coverage of the <em>H. melpomene</em> genome was generated using PacBio sequencing. These sequence reads were error-corrected once using the original Illumina and 454 data from the genome, and again using self-correction (Table C in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023655/-/DC1/FileS2.xlsx">File S2</a>). The two error-corrected read sets were combined and assembled together using FALCON to produce an initial assembly of 11,121 scaffolds, with N50 length 96 kb and total length 325 Mb (“PacBio FALCON”; <a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a> and <a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>).</p><p id="p-49">The initial PacBio assembly was merged to itself iteratively using HaploMerger to produce a haploid PacBio assembly (“PacBio haploid”; <a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1</a> and <a id="xref-table-wrap-1-6" class="xref-table" href="#T1">Table 1</a>). The haploid Hmel1.1 genome and haploid PacBio genome were then merged using HaploMerger to scaffold the two genomes together. This final merge was checked against the linkage map, and 470 misassemblies in the original PacBio assembly were fixed, requiring the two PacBio merging steps to be repeated several times. The final haploid PacBio genome had 4565 scaffolds, N50 length 178 kb, total length 256 Mb; the Hmel1.1+PacBio merged assembly had 2961 scaffolds, N50 length 629 kb, total length 283 Mb (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1</a> and <a id="xref-table-wrap-1-7" class="xref-table" href="#T1">Table 1</a>).</p></div><div id="sec-18" class="subsection"><h3>Ordering of scaffolds on chromosomes</h3><p id="p-50">Linkage information was transferred to the Hmel1.1+PacBio merged assembly, and used to place the resulting scaffolds on chromosomes, anchoring scaffolds wherever possible, connecting consecutive anchored scaffolds, and removing remaining haplotypic scaffolds (see <em>Materials and Methods</em> for details). Further scaffolds were joined by searching for connections to PacBio scaffolds unused by HaploMerger during the merge process. This left 641 scaffolds (274 Mb) placed on chromosomes (98.7% of the genome), with a further 869 scaffolds (3.6 Mb) unplaced; 154 (1.1 Mb) of the unplaced scaffolds were retained as they contained genes or had chromosome assignments (but no placement within the chromosome), and the remaining 715 scaffolds (2.5 Mb, 0.9%) were discarded.</p></div><div id="sec-19" class="subsection"><h3>Final assembly quality</h3><p id="p-51">The final genome assembly, Hmel2, has 795 scaffolds, length 275.2 Mb, N50 length 2.1 Mb (<a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1</a>, <a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a>, <a id="xref-table-wrap-1-8" class="xref-table" href="#T1">Table 1</a>, and <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>), with 231 Mb (84%) anchored, and 274 Mb (99%) placed on chromosomes (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023655/-/DC1/FigureS4.pdf">Figure S4</a>). This compares well with the other published Lepidopteran genome assemblies to date (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023655/-/DC1/FigureS5.pdf">Figure S5</a>). BUSCO results (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>) indicate that 5% of arthropod BUSCOs (134 out of 2675 genes) are missing in Hmel2. In fact, BUSCO found BLAST and HMMER matches falling below the expected score threshold for all but 11 of these missing BUSCOs (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023655/-/DC1/FigureS6.pdf">Figure S6</a>). Matches for missing BUSCOs were substantially shorter than complete BUSCOs (mean 31% of expected length, compared to mean 89% of expected length for complete BUSCOs; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023655/-/DC1/FigureS7.pdf">Figure S7</a>); while these matches may be spurious, it seems likely many of the missing BUSCOs are at least partially present in the assembly.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/467727/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/467727/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Genome assembly statistics for Hmel1.1, Hmel2, and other published Lepidopteran genomes</span><div class="sb-div caption-clear"></div></div></div><p id="p-53">The final Hmel2 genome size of 275.1 Mb with only 0.98 Mb of gaps is an improvement on Hmel1.1 (total size of 273.7 Mb with 4.1 Mb of gaps), adding 4.5 Mb of sequence to the assembly (<a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2</a>); 246.9 Mb has been carried over from Hmel1.1 directly, with 17.1 Mb added from the haplotype scaffolds, and 11.2 Mb added from PacBio scaffolds. Of the filled Hmel1.1 gaps, the average difference in size between the Hmel1.1 gap and the Hmel2 filled region is mean 75 bp, median –117.5 bp; 62% of gaps have reduced in size, with 38% increased in size (full distribution shown in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023655/-/DC1/FigureS8.pdf">Figure S8</a>).</p><p id="p-54">Hmel2 is still smaller than the flow cytometry estimate of 292 Mb +/− 2.4 Mb (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Jiggins <em>et al.</em> 2005</a>). One reason for this may be collapsed repeats across the genome. To test for this, we attempted to estimate the number of true bases in the genome for the F1 father by calculating the median per base read depth in 1-kb windows across the genome, and genome-wide (see <em>Materials and Methods</em> for full details). Assuming the genome-wide median read depth is the true diploid read depth, we adjusted the number of bases represented by each 1-kb window by multiplying 1000 by the ratio of window median read depth to genome-wide median read depth, adjusted for GC content. The sum of the estimates of true bases across the genome was 288.7 Mb, but only 270.9 Mb of the assembly was included in this analysis, as windows shorter than 1 kb, or containing gaps, were removed. Adjusting to the total length of the genome assembly, the estimate of true bases is 293.3 Mb. This is within the range of the flow cytometry estimates, and indicates that most of the missing genome sequence is in collapsed repeats and could be extended with more attention to these areas.</p></div><div id="sec-20" class="subsection"><h3>Improved assembly of major loci</h3><p id="p-55">The assembly of major adaptive loci is greatly improved in Hmel2, with all scaffolds containing known adaptive loci substantially extended and most gaps filled. The yellow color pattern locus Yb, previously on a 1.33-Mb scaffold, is now on a 1.96-Mb scaffold; the red pattern BD locus scaffold has increased from 602 kb to 1.89 Mb, and is now gap-free; the K locus, previously spread over two scaffolds totaling 173 kb, is now on a single 3-Mb scaffold; the Ac locus, previously on three scaffolds totaling 838 kb, is now on a single 7.4-Mb scaffold; and the Hox cluster, previously manually assembled into seven scaffolds covering 1.4 Mb (Supplementary Information S10 in <a id="xref-ref-23-10" class="xref-bibr" href="#ref-23">Heliconius Genome Consortium 2012</a>), is now a single scaffold covering 1.3 Mb, with some misassembled material reassigned elsewhere. Full details of major locus locations in Hmel1.1 and Hmel2 (based on loci from <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Nadeau <em>et al.</em> 2014</a>) can be found in Table D in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023655/-/DC1/FileS2.xlsx">File S2</a>, with three previously unmapped minor loci now placed on chromosomes.</p></div><div id="sec-21" class="subsection"><h3>Chromosome fusions between Eueides and Heliconius</h3><p id="p-56">To identify chromosome fusion points between <em>Eueides</em> and <em>Heliconius</em>, chromosome prints for the 31 <em>Eueides</em> chromosomes were discovered using RAD Sequencing data from an <em>E. isabella</em> cross aligned to the Hmel2 genome (Table E in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023655/-/DC1/FileS2.xlsx">File S2</a>). Synteny between <em>Heliconius</em> and <em>Eueides</em> is clear on all chromosomes, with 11 unfused and 10 fused <em>Heliconius</em> chromosomes (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>). The <em>Eueides</em> fusion points all fall within the <em>Melitaea</em> fusion points reported by <a id="xref-ref-1-5" class="xref-bibr" href="#ref-1">Ahola <em>et al.</em> (2014)</a> and confirmed against Hmel2 here (Table F in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023655/-/DC1/FileS2.xlsx">File S2</a>), indicating that these fusions occurred since the split between <em>Eueides</em> and <em>Heliconius</em>. Major color pattern loci, and other adaptive loci (<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Nadeau <em>et al.</em> 2014</a>), are not near to fusion points, with the exception of the <em>H. erato</em> locus Ro, which is 73 kb away from the chromosome 13 fusion point (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a>, and Table D in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023655/-/DC1/FileS2.xlsx">File S2</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/3/695/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Chromosome fusions in H. melpomene. Chromosomes of H. melpomene ordered by length. Unfused Heliconius chromosomes in pink; fused Eueides/Melitaea chromosomes in orange and blue, longest chromosome of each pair in blue. Melitaea chromosome numbers in white. Black line, beginning of H. melpomene chromosome in Hmel2. Black labels, loci known to be associated with color pattern features or altitude (alt) in H. melpomene or H. erato (Nadeau et al. 2014); see Table S4 for details." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1884001071" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Chromosome fusions in &lt;em&gt;H. melpomene&lt;/em&gt;. Chromosomes of &lt;em&gt;H. melpomene&lt;/em&gt; ordered by length. Unfused &lt;em&gt;Heliconius&lt;/em&gt; chromosomes in pink; fused &lt;em&gt;Eueides/Melitaea&lt;/em&gt; chromosomes in orange and blue, longest chromosome of each pair in blue. &lt;em&gt;Melitaea&lt;/em&gt; chromosome numbers in white. Black line, beginning of &lt;em&gt;H. melpomene&lt;/em&gt; chromosome in Hmel2. Black labels, loci known to be associated with color pattern features or altitude (alt) in &lt;em&gt;H. melpomene&lt;/em&gt; or &lt;em&gt;H. erato&lt;/em&gt; (Nadeau &lt;em&gt;et al.&lt;/em&gt; 2014); see Table S4 for details.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/3/695/F4.medium.gif" width="440" height="278"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/6/3/695/F4.medium.gif" width="440" height="278"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/3/695/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/3/695/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467606" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-57" class="first-child">Chromosome fusions in <em>H. melpomene</em>. Chromosomes of <em>H. melpomene</em> ordered by length. Unfused <em>Heliconius</em> chromosomes in pink; fused <em>Eueides/Melitaea</em> chromosomes in orange and blue, longest chromosome of each pair in blue. <em>Melitaea</em> chromosome numbers in white. Black line, beginning of <em>H. melpomene</em> chromosome in Hmel2. Black labels, loci known to be associated with color pattern features or altitude (alt) in <em>H. melpomene</em> or <em>H. erato</em> (<a id="xref-ref-38-3" class="xref-bibr" href="#ref-38">Nadeau <em>et al.</em> 2014</a>); see Table S4 for details.</p><div class="sb-div caption-clear"></div></div></div><p id="p-58">As noted by <a id="xref-ref-1-6" class="xref-bibr" href="#ref-1">Ahola <em>et al.</em> (2014)</a>, the shorter <em>Melitaea</em> chromosomes (22–31) are all involved in fusions. The longer <em>Melitaea</em> autosome in each fusion pair in <em>Heliconius</em> (<em>Melitaea</em> 2, 4, 6, 9–15; mean length 10.7 Mb, SD 688 kb) does not, on average, differ substantially in length to unfused autosomes (<em>Melitaea</em> 3, 5, 7, 8, 16–21; mean length 9.9 Mb, SD 894 kb). In contrast, the shorter <em>Melitaea</em> autosomes in each fusion pair in <em>Heliconius</em> (<em>Melitaea</em> 22–31) have mean length 5.4 Mb (SD 1.5 Mb), suggesting a bimodal distribution with the long <em>Melitaea</em> autosomes, both fused and unfused, clustering together into one group, and the short fused <em>Melitaea</em> autosomes clustering into a second group.</p></div></div><div class="section discussion" id="sec-22"><h2 class="">Discussion</h2><div id="sec-23" class="subsection"><h3>Genome assembly improvements</h3><p id="p-59">Many long-range technologies are now available for improvement of existing draft genomes. Deep coverage with long reads can be sufficient for producing almost complete <em>de novo</em> assemblies (<a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">Berlin <em>et al.</em> 2015</a>), and additional technologies such as optical mapping can substantially improve genome scaffolding and identify complex structural variants (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Pendleton <em>et al.</em> 2015</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">English <em>et al.</em> 2015</a>). However, it remains unclear how well these technologies will work with highly heterozygous nonmodel organisms.</p><p id="p-60">Here, we show that even a small amount of PacBio data (∼20x coverage) was sufficient to substantially improve the <em>H. melpomene</em> genome. Indeed, the assembly of the PacBio data alone was comparable in quality to our initial draft assembly constructed with Illumina, 454, and mate pair sequencing (<a id="xref-ref-23-11" class="xref-bibr" href="#ref-23">Heliconius Genome Consortium 2012</a>; compare lines “Hmel1.1 with haplotypes” and “PacBio FALCON” in <a id="xref-table-wrap-1-10" class="xref-table" href="#T1">Table 1</a> and <a id="xref-fig-1-8" class="xref-fig" href="#F1">Figure 1</a>). We expect that increasing this coverage could have produced a very high quality genome with no additional data.</p><p id="p-61">However, this does not deal with heterozygosity across the genome and the resulting generation of many haplotypic scaffolds—a problem for most species and particularly for insects (<a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Richards and Murali 2015</a>). As sequencing methods improve and true haplotypes can be assembled, it is hoped that full diploid genomes can be produced, and several efforts are already moving toward this (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Church <em>et al.</em> 2015</a>; <a href="https://github.com/ekg/vg">https://github.com/ekg/vg</a>). We hope that, in the near future, it will be possible to assemble a diploid reference graph for <em>H. melpomene</em>, perhaps with the haplotypes reported here. However, as we wanted to preserve contiguity with Hmel1.1, which was already a composite of both haplotypes, Hmel2 remains a composite haploid genome.</p><p id="p-62">HaploMerger has proved to be a very versatile assembly tool. In addition to having many options for varying the merging process, and for manually accepting or rejecting merges, HaploMerger is almost unique among similar tools in reporting where it has placed parts of the original genome in the new genome. This has allowed us to write scripts to transfer linkage map information and genes to new genome versions directly and automatically, without having to map the original genome scaffolds to the new genome separately and possibly erroneously (although we have used this approach to map genes that could not be transferred directly). We could then accept or reject merges where they introduced misassemblies that conflicted with the linkage map or broke genes, and iterate the use of HaploMerger to collapse as many scaffolds as possible. This allowed us to use HaploMerger to scaffold the existing <em>Heliconius</em> genome with our novel PacBio genome, by treating the two ‘haploid’ genomes as two haplotypes in one diploid genome. We could then modify the HaploMerger output to prefer the original Hmel1.1 genome over the PacBio genome, only using the PacBio genome for scaffolding, and so preserve our original assembly and annotation wherever possible.</p><p id="p-63">Hmel2 is not complete; it does not contain a W chromosome, and no chromosome is assembled into a single scaffold. The incomplete assemblies may be partially due to errors in haplotype merging. The detailed linkage mapping information available for most scaffolds increases our confidence that primary and haplotype scaffolds have been accurately placed, but it may be that merging haplotypes has collapsed or removed some repetitive material. The majority of Hmel1.1 gaps filled with haplotypic or PacBio sequence were reduced in size; these filled regions may be correct, but they may also indicate some reduction in repeat copy number. However, over 40% of gaps did increase in size, many substantially (for example, 6% of gaps increased by over 5 kb; see the section <em>Final assembly quality</em> in <em>Results</em>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023655/-/DC1/FigureS8.pdf">Figure S8</a>). Remaining gaps between scaffolds, and failures to order scaffolds, may be due to incorrect assembly of haplotypes at the ends of scaffolds, or due to genuine incompatibilities between the many individual butterflies that have contributed to the genome sequence, making it impossible to find overlaps or connections between these ends. Several hundred small scaffolds remain in the genome, which are likely to be misassemblies of repetitive elements, but no clear metric could be found that excluded or integrated these scaffolds. However, as the positions of removed haplotypes have been recorded, it may be possible to reintegrate this material with further analysis of particular regions of the genome. Finally, the assembly remains shorter than the flow cytometry estimate of the <em>H. melpomene</em> genome size, which appears to be due to collapsing of repetitive material (see the section <em>Final assembly quality</em> in <em>Results</em>). Further manual inspection of existing data, PCRs across scaffold ends, additional long-read sequencing, or additional cross sequencing or optical mapping will hopefully resolve many of these remaining assembly problems.</p></div><div id="sec-24" class="subsection"><h3>Is Heliconius speciation rate driven by chromosome fusions?</h3><p id="p-64">Chromosome number varies widely in the Lepidoptera (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Robinson 1971</a>), and gradual transitions from one number to another occur frequently. Lepidopteran chromosomes are believed to be holocentric (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Wolf 1994</a>), which may make it easier for chromosome fusions and fissions to spread throughout a population (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Melters <em>et al.</em> 2012</a>). However, the fusion of 20 chromosomes into 10 over 6 million yr (timing based on nodes in figure 1 of <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Kozak <em>et al.</em> 2015</a>) is the largest shift in chromosome number in such a short period across the Lepidoptera (<a id="xref-ref-1-7" class="xref-bibr" href="#ref-1">Ahola <em>et al.</em> 2014</a>; <a id="xref-fig-3-2" class="xref-fig" href="#F3">figure 3A</a>). Also, given the supposed ease of chromosome number transitions, it is unusual that chromosome number in the Nymphalinae and Heliconiinae is stable at 31 and 21 chromosomes respectively for the majority of species, in contrast to all other subfamilies, where chromosome number tends to fluctuate gradually and widely (<a id="xref-ref-1-8" class="xref-bibr" href="#ref-1">Ahola <em>et al.</em> 2014</a>; <a id="xref-fig-3-3" class="xref-fig" href="#F3">figure 3B</a>). While <em>Heliconius</em> species do vary in chromosome number, the majority still have 21 chromosomes, with substantial variations only found in derived clades (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Brown <em>et al.</em> 1992</a>; <a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">Kozak <em>et al.</em> 2015</a>). It is not just the transition in chromosome number but also the stability of chromosome number before and after the transition that requires explanation.</p><p id="p-65">The difference in chromosome number confirmed here is a major difference between the <em>Heliconius</em> and <em>Eueides</em> genera, which may make these genera an excellent system for studying macroevolution and speciation. <a id="xref-ref-30-4" class="xref-bibr" href="#ref-30">Kozak <em>et al.</em> (2015)</a> demonstrated that speciation rate in <em>Heliconius</em> is significantly higher than in <em>Eueides</em>, but the rate in both genera is more or less stable, and does not obviously relate to geological events or adaptive traits. The difference in chromosome number may contribute to explaining this difference in speciation rate.</p><p id="p-66">Restriction of recombination facilitates speciation in the presence of gene flow (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Butlin 2005</a>). One of the major mechanisms for restricting recombination are chromosome inversions, where opposing alleles can become linked together, and then become fixed in different populations (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Kirkpatrick and Barton 2006</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Farré <em>et al.</em> 2013</a>; <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Kirkpatrick and Barrett 2015</a>). However, other methods of restricting recombination may produce similar effects.</p><p id="p-67">Recombination rate is negatively correlated with chromosome length, although the relationship is complex (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Fledel-Alon <em>et al.</em> 2009</a>; <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Kawakami <em>et al.</em> 2014</a>). In many species, one obligate crossover is required for successful meiosis, inflating recombination rate in short chromosomes. However, beyond a certain length, recombination rate increases roughly linearly with chromosome length (<a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">Kawakami <em>et al.</em> 2014</a>). It is unclear whether these relationships will hold in Lepidoptera, which may have no obligate crossovers, as females do not recombine, and meiosis requires the formation of a synaptonemal complex rather than recombination (<a id="xref-ref-58-2" class="xref-bibr" href="#ref-58">Wolf 1994</a>).</p><p id="p-68">It is possible that recombination rate along fused chromosomes in <em>Heliconius</em> has decreased considerably compared to their shorter, unfused counterparts in <em>Eueides</em> (and <em>Melitaea</em>), particularly on the shorter chromosomes. This may have enabled linked pairs of divergently selected loci to accrue more easily in <em>Heliconius</em> than in <em>Eueides</em>, making the process of speciation more likely (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Nachman and Payseur 2012</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Brandvain <em>et al.</em> 2014</a>). This hypothesis could be tested by generating population sequence for <em>Eueides</em> species to compare to existing <em>Heliconius</em> population data (such as <a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">Martin <em>et al.</em> 2013</a>), and by modeling speciation rates in the face of different recombination rates. Such a model could predict speciation rate differences between the genera, but full testing would also require the generation of accurate recombination rates in both genera. The system is particularly well suited for testing speciation rate effects because the set of 10 unfused autosomes can act as a control; the hypothesis predicts that recombination rate will not have changed substantially on these chromosomes.</p><p id="p-69">This hypothesis demonstrates the pressing need to generate full, chromosomal genomes for <em>Eueides</em> and other <em>Heliconius</em> species; genome size in <em>H. erato</em> is ∼393 Mb (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Tobler <em>et al.</em> 2005</a>), very similar to <em>M. cinxia</em>, but roughly 100 Mb larger than <em>H. melpomene</em>. Unpublished draft genome sequences of <em>Eueides tales</em>, and other <em>Heliconius</em> species, suggest genome sizes similar to <em>H. erato</em> or larger, with <em>H. melpomene</em> being one of the smallest <em>Heliconius</em> genomes (data not shown). Measuring recombination rate for other species against the <em>H. melpomene</em> genome alone is therefore unlikely to be accurate, and may not allow for accurate model fitting. However, with additional genomes in hand, we believe these genera may provide a useful test case for the influence of genome architecture on speciation and molecular evolution.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-70">We thank Jenny Barna for computing support. <em>E. isabella</em> cross sequencing was carried out by Sylviane Moss at the Gurdon Institute. PacBio sequencing was carried out by Paul Coupland and Richard Durbin at the Sanger Institute, supported by European Research Council (ERC) grant number 339873, Wellcome Trust grant number 098051, and J.W.D.’s Herchel Smith funding. <em>H. melpomene</em> cross sequencing was carried out at the Harvard FAS Center for Systems Biology core facility, and funded by <em>Biotechnology and Biological Sciences Research Council</em> grant number G006903/1. <em>E. isabella</em> breeding was supported by ERC grant StG-MimEvol, and Agence Nationale de la Recherche grant JCJC-HybEvol. Alignment and SNP calling was performed using the Darwin Supercomputer of the University of Cambridge High Performance Computing Service (<a href="http://www.hpc.cam.ac.uk/">http://www.hpc.cam.ac.uk/</a>), provided by Dell Inc. using Strategic Research Infrastructure Funding from the Higher Education Funding Council for England, and funding from the Science and Technology Facilities Council. J.W.D. is funded by a Herchel Smith Postdoctoral Research Fellowship.</p><p id="p-71">Author contributions: J.W.D. and C.J. conceived the study. J.W.D. designed the analyses, wrote the software, extracted DNA for PacBio sequencing, and wrote the paper. F.S., K.K.D., and J.M. extracted DNA, prepared libraries, and whole genome sequenced the pedigree. L.M., R.M.M., and S.W.B. bred the <em>H. melpomene</em> cross. M.C. and M.J. bred the <em>Eueides isabella</em> cross. S.L.B. and R.M.M. extracted DNA and made RAD libraries for the <em>E. isabella</em> cross. All authors read and commented on the manuscript.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-supplementary-material" id="fn-5"><p id="p-72">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023655/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023655/-/DC1</a></p></li><li class="fn" id="fn-6"><p id="p-73">ENA project accessions: PRJEB11288, ERP005954</p></li><li class="fn" id="fn-7"><p id="p-74">Dryad DOI: 10.5061/dryad.3s795</p></li><li class="fn" id="fn-8"><p id="p-75"><em>Communicating editor: S. I. Wright</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2015-10-14"><span class="received-label">Received </span>October 14, 2015.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2016-01-06"><span class="accepted-label">Accepted </span>January 6, 2016.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2016 Davey <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-3">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.695.1" data-doi="10.1038/ncomms5737"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ahola</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lehtonen</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Somervuo</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salmela</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koskinen</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The Glanville fritillary genome retains an ancient karyotype and reveals selective chromosomal fusions in Lepidoptera.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">4737</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D5%26rft.spage%253D4737%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms5737%26rft_id%253Dinfo%253Apmid%252F25189940%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms5737&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25189940&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F695.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.695.2" data-doi="10.1371/journal.pgen.1000794"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baxter</span>  <span class="cit-name-given-names">S. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nadeau</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maroja</span>  <span class="cit-name-given-names">L. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilkinson</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Counterman</span>  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genomic hotspots for adaptation: the population genetics of Müllerian mimicry in the Heliconius melpomene clade.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e1000794</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DBaxter%26rft.auinit1%253DS.%2BW.%26rft.volume%253D6%26rft.issue%253D2%26rft.spage%253De1000794%26rft.epage%253De1000794%26rft.atitle%253DGenomic%2Bhotspots%2Bfor%2Badaptation%253A%2Bthe%2Bpopulation%2Bgenetics%2Bof%2BMullerian%2Bmimicry%2Bin%2Bthe%2BHeliconius%2Bmelpomene%2Bclade.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000794%26rft_id%253Dinfo%253Apmid%252F20140188%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000794&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20140188&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F695.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.695.3" data-doi="10.1111/j.1095-8312.2007.00830.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beltrán</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiggins</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brower</span>  <span class="cit-name-given-names">A. V. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bermingham</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mallet</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Do pollen feeding, pupal-mating and larval gregariousness have a single origin in Heliconius butterflies? Inferences from multilocus DNA sequence data.</span> <abbr class="cit-jnl-abbrev">Biol. J. Linn. Soc. Lond.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">221</span>–<span class="cit-lpage">239</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiol.%2BJ.%2BLinn.%2BSoc.%2BLond.%26rft.volume%253D92%26rft.spage%253D221%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1095-8312.2007.00830.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1095-8312.2007.00830.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000249562600003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.695.4" data-doi="10.1038/nbt.3238"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Berlin</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koren</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chin</span>  <span class="cit-name-given-names">C.-S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Drake</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Landolin</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Assembling large genomes with single-molecule sequencing and locality-sensitive hashing.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">623</span>–<span class="cit-lpage">630</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BBiotechnol.%26rft.volume%253D33%26rft.spage%253D623%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.3238%26rft_id%253Dinfo%253Apmid%252F26006009%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.3238&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26006009&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F695.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.695.5" data-doi="10.1186/1471-2105-15-211"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boetzer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pirovano</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">SSPACE-LongRead: scaffolding bacterial draft genomes using long read sequence information.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">211</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D15%26rft.spage%253D211%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-15-211%26rft_id%253Dinfo%253Apmid%252F24950923%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-15-211&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24950923&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F695.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.695.6" data-doi="10.1186/2047-217X-2-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bradnam</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fass</span>  <span class="cit-name-given-names">J. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alexandrov</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baranay</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bechner</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Assemblathon 2: evaluating de novo methods of genome assembly in three vertebrate species.</span> <abbr class="cit-jnl-abbrev">Gigascience</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">31</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGigascience%26rft.volume%253D2%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1186%252F2047-217X-2-1%26rft_id%253Dinfo%253Apmid%252F23587291%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/2047-217X-2-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23587291&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F695.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.695.7" data-doi="10.1371/journal.pgen.1004410"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brandvain</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kenney</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flagel</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coop</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sweigart</span>  <span class="cit-name-given-names">A. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Speciation and introgression between <em>Mimulus nasutus</em> and <em>Mimulus guttatus</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e1004410</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D10%26rft.spage%253De1004410%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1004410%26rft_id%253Dinfo%253Apmid%252F24967630%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1004410&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24967630&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F695.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.695.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">K. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Emmel</span>  <span class="cit-name-given-names">T. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eliazar</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Suomalainen</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Evolutionary patterns in chromosome numbers in neotropical Lepidoptera.</span> <abbr class="cit-jnl-abbrev">Hereditas</abbr> <span class="cit-vol">117</span>: <span class="cit-fpage">109</span>–<span class="cit-lpage">125</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHereditas%26rft.stitle%253DHereditas%26rft.aulast%253DBrown%26rft.auinit1%253DK.%2BS.%26rft.volume%253D117%26rft.issue%253D2%26rft.spage%253D109%26rft.epage%253D125%26rft.atitle%253DEvolutionary%2Bpatterns%2Bin%2Bchromosome%2Bnumbers%2Bin%2Bneotropical%2BLepidoptera.%2BI.%2BChromosomes%2Bof%2Bthe%2BHeliconiini%2B%2528family%2BNymphalidae%253A%2Bsubfamily%2BNymphalinae%2529.%26rft_id%253Dinfo%253Apmid%252F1459855%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=1459855&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F695.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1992KB55100002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.695.9" data-doi="10.1111/j.1365-294X.2005.02617.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Butlin</span>  <span class="cit-name-given-names">R. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Recombination and speciation.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">2621</span>–<span class="cit-lpage">2635</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DButlin%26rft.auinit1%253DR.%2BK.%26rft.volume%253D14%26rft.issue%253D9%26rft.spage%253D2621%26rft.epage%253D2635%26rft.atitle%253DRecombination%2Band%2Bspeciation.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2005.02617.x%26rft_id%253Dinfo%253Apmid%252F16029465%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2005.02617.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16029465&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F695.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000230573600001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.695.10" data-doi="10.1534/g3.111.000240"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Catchen</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amores</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hohenlohe</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cresko</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Postlethwait</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Stacks: building and genotyping loci de novo from short-read sequences.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">171</span>–<span class="cit-lpage">182</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2526%252358%253B%2BGenes%257CGenomes%257CGenetics%26rft.stitle%253DG3%26rft.aulast%253DCatchen%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D1%26rft.issue%253D3%26rft.spage%253D171%26rft.epage%253D182%26rft.atitle%253DStacks%253A%2BBuilding%2Band%2BGenotyping%2BLoci%2BDe%2BNovo%2BFrom%2BShort-Read%2BSequences%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.111.000240%26rft_id%253Dinfo%253Apmid%252F22384329%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjEvMy8xNzEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzMvNjk1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.695.11" data-doi="10.1186/s13059-015-0587-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Church</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schneider</span>  <span class="cit-name-given-names">V. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steinberg</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schatz</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quinlan</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Extending reference assembly models.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">13</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D16%26rft.spage%253D13%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-015-0587-3%26rft_id%253Dinfo%253Apmid%252F25651527%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-015-0587-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25651527&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F695.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.695.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cong</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borek</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Otwinowski</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grishin</span>  <span class="cit-name-given-names">N. V.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span><span class="cit-comment">a</span> <span class="cit-article-title">Tiger swallowtail genome reveals mechanisms for speciation and caterpillar chemical defense.</span> <abbr class="cit-jnl-abbrev">Cell Reports</abbr>.<strong>10:</strong> 910–919</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.695.13" data-doi="10.1186/s12864-015-1298-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cong</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borek</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Otwinowski</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grishin</span>  <span class="cit-name-given-names">N. V.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span><span class="cit-comment">b</span> <span class="cit-article-title">Skipper genome sheds light on unique phenotypic traits and phylogeny.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">94</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D16%26rft.spage%253D94%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-015-1298-6%26rft_id%253Dinfo%253Apmid%252F25765081%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-015-1298-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25765081&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F695.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.695.14" data-doi="10.1093/bioinformatics/btr330"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Danecek</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auton</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abecasis</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albers</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The variant call format and VCFtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">2156</span>–<span class="cit-lpage">2158</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr330%26rft_id%253Dinfo%253Apmid%252F21653522%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjcvMTUvMjE1NiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvMy82OTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.695.15" data-doi="10.1371/journal.pone.0011147"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Darling</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mau</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perna</span>  <span class="cit-name-given-names">N. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">progressiveMauve: Multiple genome alignment with gene gain, loss and rearrangement.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e11147</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DDarling%26rft.auinit1%253DA.%2BE.%26rft.volume%253D5%26rft.issue%253D6%26rft.spage%253De11147%26rft.epage%253De11147%26rft.atitle%253DprogressiveMauve%253A%2Bmultiple%2Bgenome%2Balignment%2Bwith%2Bgene%2Bgain%252C%2Bloss%2Band%2Brearrangement.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0011147%26rft_id%253Dinfo%253Apmid%252F20593022%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0011147&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20593022&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F695.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.695.16" data-doi="10.1038/ng.806"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">DePristo</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poplin</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garimella</span>  <span class="cit-name-given-names">K. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maguire</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A framework for variation discovery and genotyping using next-generation DNA sequencing data.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">491</span>–<span class="cit-lpage">498</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DDepristo%26rft.auinit1%253DM.%2BA.%26rft.volume%253D43%26rft.issue%253D5%26rft.spage%253D491%26rft.epage%253D498%26rft.atitle%253DA%2Bframework%2Bfor%2Bvariation%2Bdiscovery%2Band%2Bgenotyping%2Busing%2Bnext-generation%2BDNA%2Bsequencing%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.806%26rft_id%253Dinfo%253Apmid%252F21478889%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.806&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21478889&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F695.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000289972600023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.695.17" data-doi="10.1093/nar/gkp801"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Duan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fan</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zha</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">SilkDB v2.0: a platform for silkworm (<em>Bombyx mori</em>) genome biology.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">D453</span>–<span class="cit-lpage">D456</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkp801%26rft_id%253Dinfo%253Apmid%252F19793867%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjE1OiIzOC9zdXBwbF8xL0Q0NTMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzMvNjk1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.695.18" data-doi="10.1371/journal.pone.0047768"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">English</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Richards</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vee</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Mind the gap: upgrading genomes with Pacific Biosciences RS long-read sequencing technology.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e47768</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DEnglish%26rft.auinit1%253DA.%2BC.%26rft.volume%253D7%26rft.issue%253D11%26rft.spage%253De47768%26rft.epage%253De47768%26rft.atitle%253DMind%2Bthe%2BGap%253A%2BUpgrading%2BGenomes%2Bwith%2BPacific%2BBiosciences%2BRS%2BLong-Read%2BSequencing%2BTechnology.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0047768%26rft_id%253Dinfo%253Apmid%252F23185243%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0047768&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23185243&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F695.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.695.19" data-doi="10.1186/s12864-015-1479-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">English</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salerno</span>  <span class="cit-name-given-names">W. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hampton</span>  <span class="cit-name-given-names">O. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gonzaga-Jauregui</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ambreth</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Assessing structural variation in a personal genome—towards a human reference diploid genome.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">286</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D16%26rft.spage%253D286%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-015-1479-3%26rft_id%253Dinfo%253Apmid%252F25886820%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-015-1479-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25886820&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F695.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.695.20" data-doi="10.1093/molbev/mss272"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Farré</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Micheletti</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruiz-Herrera</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Recombination rates and genomic shuffling in human and chimpanzee–a new twist in the chromosomal speciation theory.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">853</span>–<span class="cit-lpage">864</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmss272%26rft_id%253Dinfo%253Apmid%252F23204393%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjMwLzQvODUzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8zLzY5NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.695.21" data-doi="10.1111/j.1558-5646.2009.00786.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Feder</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nosil</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Chromosomal inversions and species differences: when are genes affecting adaptive divergence and reproductive isolation expected to reside within inversions?</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">63</span>: <span class="cit-fpage">3061</span>–<span class="cit-lpage">3075</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DFeder%26rft.auinit1%253DJ.%2BL.%26rft.volume%253D63%26rft.issue%253D12%26rft.spage%253D3061%26rft.epage%253D3075%26rft.atitle%253DChromosomal%2Binversions%2Band%2Bspecies%2Bdifferences%253A%2Bwhen%2Bare%2Bgenes%2Baffecting%2Badaptive%2Bdivergence%2Band%2Breproductive%2Bisolation%2Bexpected%2Bto%2Breside%2Bwithin%2Binversions%253F%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1558-5646.2009.00786.x%26rft_id%253Dinfo%253Apmid%252F19656182%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1558-5646.2009.00786.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19656182&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F695.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000271973600003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.695.22" data-doi="10.1371/journal.pgen.1000658"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fledel-Alon</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wen</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ober</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Broad-scale recombination patterns underlying proper disjunction in humans.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e1000658</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DFledel-Alon%26rft.auinit1%253DA.%26rft.volume%253D5%26rft.issue%253D9%26rft.spage%253De1000658%26rft.epage%253De1000658%26rft.atitle%253DBroad-scale%2Brecombination%2Bpatterns%2Bunderlying%2Bproper%2Bdisjunction%2Bin%2Bhumans.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000658%26rft_id%253Dinfo%253Apmid%252F19763175%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000658&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19763175&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F695.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.695.23" data-doi="10.1038/nature11041"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">Heliconius Genome Consortium</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Butterfly genome reveals promiscuous exchange of mimicry adaptations among species.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">487</span>: <span class="cit-fpage">94</span>–<span class="cit-lpage">98</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DDasmahapatra%26rft.auinit1%253DK.%2BK.%26rft.volume%253D487%26rft.issue%253D7405%26rft.spage%253D94%26rft.epage%253D98%26rft.atitle%253DButterfly%2Bgenome%2Breveals%2Bpromiscuous%2Bexchange%2Bof%2Bmimicry%2Badaptations%2Bamong%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11041%26rft_id%253Dinfo%253Apmid%252F22722851%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11041&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22722851&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F695.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000305982900057&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.695.24" data-doi="10.1101/gr.133652.111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">HaploMerger: reconstructing allelic relationships for polymorphic diploid genome assemblies.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">1581</span>–<span class="cit-lpage">1588</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.133652.111%26rft_id%253Dinfo%253Apmid%252F22555592%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIyLzgvMTU4MSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvMy82OTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.695.25" data-doi="10.1534/genetics.104.034686"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jiggins</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mavarez</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beltrán</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McMillan</span>  <span class="cit-name-given-names">W. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnston</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">A genetic linkage map of the mimetic butterfly <em>Heliconius melpomene</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">171</span>: <span class="cit-fpage">557</span>–<span class="cit-lpage">570</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.034686%26rft_id%253Dinfo%253Apmid%252F15489522%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTcxLzIvNTU3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8zLzY5NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.695.26" data-doi="10.1038/nature10341"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Joron</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frezal</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">R. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chamberlain</span>  <span class="cit-name-given-names">N. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Chromosomal rearrangements maintain a polymorphic supergene controlling butterfly mimicry.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">477</span>: <span class="cit-fpage">203</span>–<span class="cit-lpage">206</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DJoron%26rft.auinit1%253DM.%26rft.volume%253D477%26rft.issue%253D7363%26rft.spage%253D203%26rft.epage%253D206%26rft.atitle%253DChromosomal%2Brearrangements%2Bmaintain%2Ba%2Bpolymorphic%2Bsupergene%2Bcontrolling%2Bbutterfly%2Bmimicry.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10341%26rft_id%253Dinfo%253Apmid%252F21841803%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10341&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21841803&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F695.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000294603900034&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.695.27" data-doi="10.1111/mec.12810"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kawakami</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smeds</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Backström</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Husby</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qvarnström</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A high-density linkage map enables a second-generation collared flycatcher genome assembly and reveals the patterns of avian recombination rate variation and chromosomal evolution.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">4035</span>–<span class="cit-lpage">4058</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D23%26rft.spage%253D4035%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12810%26rft_id%253Dinfo%253Apmid%252F24863701%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12810&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24863701&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F695.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000340419800010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.695.28" data-doi="10.1534/genetics.105.047985"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kirkpatrick</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barton</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Chromosome inversions, local adaptation and speciation.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">173</span>: <span class="cit-fpage">419</span>–<span class="cit-lpage">434</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.047985%26rft_id%253Dinfo%253Apmid%252F16204214%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTczLzEvNDE5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8zLzY5NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.695.29" data-doi="10.1111/mec.13074"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kirkpatrick</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barrett</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Chromosome inversions, adaptive cassettes and the evolution of species’ ranges.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">2046</span>–<span class="cit-lpage">2055</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D24%26rft.spage%253D2046%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.13074%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.13074&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.695.30" data-doi="10.1093/sysbio/syv007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kozak</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wahlberg</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neild</span>  <span class="cit-name-given-names">A. F. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dasmahapatra</span>  <span class="cit-name-given-names">K. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mallet</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Multilocus species trees show the recent adaptive radiation of the mimetic heliconius butterflies.</span> <abbr class="cit-jnl-abbrev">Syst. Biol.</abbr> <span class="cit-vol">64</span>: <span class="cit-fpage">505</span>–<span class="cit-lpage">524</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSyst.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fsysbio%252Fsyv007%26rft_id%253Dinfo%253Apmid%252F25634098%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoic3lzYmlvIjtzOjU6InJlc2lkIjtzOjg6IjY0LzMvNTA1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8zLzY5NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.695.31" data-doi="10.1186/1759-8753-4-21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lavoie</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Platt</span>  <span class="cit-name-given-names">R. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Novick</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Counterman</span>  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ray</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Transposable element evolution in Heliconius suggests genome diversity within Lepidoptera.</span> <abbr class="cit-jnl-abbrev">Mob. DNA</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">21</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMob.%2BDNA%26rft.volume%253D4%26rft.spage%253D21%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1759-8753-4-21%26rft_id%253Dinfo%253Apmid%252F24088337%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1759-8753-4-21&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24088337&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F695.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-book" id="cit-6.3.695.32"><div class="cit-metadata unstructured">Lowry, D. B., and J. H. Willis, 2010 A widespread chromosomal inversion polymorphism contributes to a major life-history transition, local adaptation, and reproductive isolation. PLoS Biol. <strong>8:</strong> e1000500.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.695.33" data-doi="10.1101/gr.111120.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lunter</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goodson</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Stampy: A statistical algorithm for sensitive and fast mapping of Illumina sequence reads.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">936</span>–<span class="cit-lpage">939</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.111120.110%26rft_id%253Dinfo%253Apmid%252F20980556%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIxLzYvOTM2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8zLzY5NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.695.34" data-doi="10.1101/gr.159426.113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dasmahapatra</span>  <span class="cit-name-given-names">K. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nadeau</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salazar</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walters</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genome-wide evidence for speciation with gene flow in Heliconius butterflies.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">1817</span>–<span class="cit-lpage">1828</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.159426.113%26rft_id%253Dinfo%253Apmid%252F24045163%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIyMy8xMS8xODE3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8zLzY5NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.695.35" data-doi="10.1007/s10577-012-9292-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Melters</span>  <span class="cit-name-given-names">D. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paliulis</span>  <span class="cit-name-given-names">L. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Korf</span>  <span class="cit-name-given-names">I. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chan</span>  <span class="cit-name-given-names">S. W. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Holocentric chromosomes: convergent evolution, meiotic adaptations, and genomic analysis.</span> <abbr class="cit-jnl-abbrev">Chromosome Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">579</span>–<span class="cit-lpage">593</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DChromosome%2Bresearch%2B%253A%2B%2Ban%2Binternational%2Bjournal%2Bon%2Bthe%2Bmolecular%252C%2Bsupramolecular%2Band%2Bevolutionary%2Baspects%2Bof%2Bchromosome%2Bbiology%26rft.stitle%253DChromosome%2BRes%26rft.aulast%253DMelters%26rft.auinit1%253DD.%2BP.%26rft.volume%253D20%26rft.issue%253D5%26rft.spage%253D579%26rft.epage%253D593%26rft.atitle%253DHolocentric%2Bchromosomes%253A%2Bconvergent%2Bevolution%252C%2Bmeiotic%2Badaptations%252C%2Band%2Bgenomic%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10577-012-9292-1%26rft_id%253Dinfo%253Apmid%252F22766638%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10577-012-9292-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22766638&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F695.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000306951100010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.695.36"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Michael</span>  <span class="cit-name-given-names">T. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">VanBuren</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Progress, challenges and the future of crop genomes.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Plant Biol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">71</span>–<span class="cit-lpage">81</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BOpin.%2BPlant%2BBiol.%26rft.volume%253D24%26rft.spage%253D71%26rft_id%253Dinfo%253Apmid%252F25703261%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=25703261&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F695.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.695.37" data-doi="10.1098/rstb.2011.0249"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nachman</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Payseur</span>  <span class="cit-name-given-names">B. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Recombination rate variation and speciation: theoretical predictions and empirical results from rabbits and mice.</span> <abbr class="cit-jnl-abbrev">Philos. Trans. R. Soc. Lond. B Biol. Sci.</abbr> <span class="cit-vol">367</span>: <span class="cit-fpage">409</span>–<span class="cit-lpage">421</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhilosophical%2BTransactions%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DPhil%2BTrans%2BR%2BSoc%2BB%26rft.aulast%253DNachman%26rft.auinit1%253DM.%2BW.%26rft.volume%253D367%26rft.issue%253D1587%26rft.spage%253D409%26rft.epage%253D421%26rft.atitle%253DRecombination%2Brate%2Bvariation%2Band%2Bspeciation%253A%2Btheoretical%2Bpredictions%2Band%2Bempirical%2Bresults%2Bfrom%2Brabbits%2Band%2Bmice%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.2011.0249%26rft_id%253Dinfo%253Apmid%252F22201170%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoicm95cHRiIjtzOjU6InJlc2lkIjtzOjEyOiIzNjcvMTU4Ny80MDkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzMvNjk1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.695.38" data-doi="10.1101/gr.169292.113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nadeau</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruiz</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salazar</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Counterman</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Medina</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Population genomics of parallel hybrid zones in the mimetic butterflies, <em>H. melpomene</em> and <em>H. erato</em>.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">1316</span>–<span class="cit-lpage">1333</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.169292.113%26rft_id%253Dinfo%253Apmid%252F24823669%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjI0LzgvMTMxNiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvMy82OTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.695.39" data-doi="10.1371/journal.pgen.1002680"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nam</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ellegren</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Recombination drives vertebrate genome contraction.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1002680</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DNam%26rft.auinit1%253DK.%26rft.volume%253D8%26rft.issue%253D5%26rft.spage%253De1002680%26rft.epage%253De1002680%26rft.atitle%253DRecombination%2Bdrives%2Bvertebrate%2Bgenome%2Bcontraction.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002680%26rft_id%253Dinfo%253Apmid%252F22570634%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002680&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22570634&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F695.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.695.40" data-doi="10.1038/ng.3241"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nishikawa</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iijima</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kajitani</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yamaguchi</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ando</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A genetic mechanism for female-limited Batesian mimicry in Papilio butterfly.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">405</span>–<span class="cit-lpage">409</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D47%26rft.spage%253D405%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3241%26rft_id%253Dinfo%253Apmid%252F25751626%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.3241&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25751626&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F695.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.695.41" data-doi="10.1073/pnas.221274498"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Noor</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grams</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bertucci</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reiland</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Chromosomal inversions and the reproductive isolation of species.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">98</span>: <span class="cit-fpage">12084</span>–<span class="cit-lpage">12088</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.221274498%26rft_id%253Dinfo%253Apmid%252F11593019%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiOTgvMjEvMTIwODQiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzMvNjk1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.695.42" data-doi="10.1038/nmeth.3454"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pendleton</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sebra</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pang</span>  <span class="cit-name-given-names">A. W. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ummat</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Franzen</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Assembly and diploid architecture of an individual human genome via single-molecule technologies.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">780</span>–<span class="cit-lpage">786</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D12%26rft.spage%253D780%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.3454%26rft_id%253Dinfo%253Apmid%252F26121404%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.3454&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26121404&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F695.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.695.43" data-doi="10.1016/j.amc.2011.03.124"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Perkins</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tygert</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ward</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Computing the confidence levels for a root-mean-square test of goodness-of-fit.</span> <abbr class="cit-jnl-abbrev">Appl. Math. Comput.</abbr> <span class="cit-vol">217</span>: <span class="cit-fpage">9072</span>–<span class="cit-lpage">9084</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAppl.%2BMath.%2BComput.%26rft.volume%253D217%26rft.spage%253D9072%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.amc.2011.03.124%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.amc.2011.03.124&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000290902600020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.695.44"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Quinlan</span>  <span class="cit-name-given-names">A. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> BEDTools: The Swiss-Army Tool for Genome Feature Analysis. Curr. Protoc. Bioinformatics <strong>47:</strong> 11.12.1–11.12.34.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.695.45" data-doi="10.1126/science.1208227"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Reed</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Papa</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hines</span>  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Counterman</span>  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Optix drives the repeated convergent evolution of butterfly wing pattern mimicry.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">333</span>: <span class="cit-fpage">1137</span>–<span class="cit-lpage">1141</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1208227%26rft_id%253Dinfo%253Apmid%252F21778360%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMzMvNjA0Ni8xMTM3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8zLzY5NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.695.46" data-doi="10.1016/j.cois.2015.02.013"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Richards</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murali</span>  <span class="cit-name-given-names">S. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Best practices in insect genome sequencing: what works and what doesn’t.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Insect Sci.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">7</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BOpin.%2BInsect%2BSci.%26rft.volume%253D7%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cois.2015.02.013%26rft_id%253Dinfo%253Apmid%252F26085980%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cois.2015.02.013&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26085980&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F695.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-book" id="cit-6.3.695.47"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1971</span> <span class="cit-source"><em>Lepidoptera Genetics</em></span>, <span class="cit-publ-name">Pergamon Press</span>, <span class="cit-publ-loc">Oxford</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.695.48" data-doi="10.1186/gb-2013-14-5-r51"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ross</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Russ</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Costello</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hollinger</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lennon</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Characterizing and measuring bias in sequence data.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">R51</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D14%26rft.spage%253DR51%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2013-14-5-r51%26rft_id%253Dinfo%253Apmid%252F23718773%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2013-14-5-r51&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23718773&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F695.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.695.49" data-doi="10.1093/bioinformatics/btv351"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Simão</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waterhouse</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ioannidis</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kriventseva</span>  <span class="cit-name-given-names">E. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zdobnov</span>  <span class="cit-name-given-names">E. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">BUSCO: assessing genome assembly and annotation completeness with single-copy orthologs.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">3210</span>–<span class="cit-lpage">3212</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtv351%26rft_id%253Dinfo%253Apmid%252F26059717%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMzEvMTkvMzIxMCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvMy82OTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-web" id="cit-6.3.695.50"><div class="cit-metadata unstructured">Smit, A. F. A., Hubley, R., and Green, P., 2013–2015 RepeatMasker Open-4.0. http://www.repeatmasker.org. Accessed: 4 February 2016.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.695.51" data-doi="10.1038/sj.hdy.6800619"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tobler</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kapan</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flanagan</span>  <span class="cit-name-given-names">N. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gonzalez</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peterson</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">First-generation linkage map of the warningly colored butterfly Heliconius erato.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">94</span>: <span class="cit-fpage">408</span>–<span class="cit-lpage">417</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DTobler%26rft.auinit1%253DA.%26rft.volume%253D94%26rft.issue%253D4%26rft.spage%253D408%26rft.epage%253D417%26rft.atitle%253DFirst-generation%2Blinkage%2Bmap%2Bof%2Bthe%2Bwarningly%2Bcolored%2Bbutterfly%2BHeliconius%2Berato.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.hdy.6800619%26rft_id%253Dinfo%253Apmid%252F15592446%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/sj.hdy.6800619&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15592446&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F695.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000227985000006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.695.52" data-doi="10.1038/hdy.1975.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Turner</span>  <span class="cit-name-given-names">J. R. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sheppard</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1975</span> <span class="cit-article-title">Absence of crossing-over in female butterflies (Heliconius).</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">265</span>–<span class="cit-lpage">269</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D34%26rft.spage%253D265%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.1975.29%26rft_id%253Dinfo%253Apmid%252F1055712%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.1975.29&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1055712&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F695.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1975AE95100010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.695.53" data-doi="10.1111/j.1365-294X.2008.03921.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Via</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">West</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The genetic mosaic suggests a new role for hitchhiking in ecological speciation.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">4334</span>–<span class="cit-lpage">4345</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DVia%26rft.auinit1%253DS.%26rft.volume%253D17%26rft.issue%253D19%26rft.spage%253D4334%26rft.epage%253D4345%26rft.atitle%253DThe%2Bgenetic%2Bmosaic%2Bsuggests%2Ba%2Bnew%2Brole%2Bfor%2Bhitchhiking%2Bin%2Becological%2Bspeciation.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2008.03921.x%26rft_id%253Dinfo%253Apmid%252F18986504%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2008.03921.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18986504&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F695.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000259525900014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.695.54" data-doi="10.1371/journal.pbio.1002353"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wallbank</span>  <span class="cit-name-given-names">R. W. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baxter</span>  <span class="cit-name-given-names">S. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pardo-Diaz</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hanly</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Evolutionary Novelty in a Butterfly Wing Pattern through Enhancer Shuffling.</span> <abbr class="cit-jnl-abbrev">Plos. Bio.l</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">e1002353</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlos.%2BBio.l%26rft.volume%253D14%26rft.spage%253De1002353%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.1002353%26rft_id%253Dinfo%253Apmid%252F26771987%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.1002353&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26771987&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F695.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.695.55" data-doi="10.1038/nature11832"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wurm</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nipitwattanaphon</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Riba-Grognuz</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">Y.-C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A Y-like social chromosome causes alternative colony organization in fire ants.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">493</span>: <span class="cit-fpage">664</span>–<span class="cit-lpage">668</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D493%26rft.spage%253D664%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11832%26rft_id%253Dinfo%253Apmid%252F23334415%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11832&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23334415&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F695.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000314219600055&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.695.56"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Warr</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robert</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hume</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Archibald</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deeb</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Identification of low-confidence regions in the pig reference genome (Sscrofa10.2).</span> <abbr class="cit-jnl-abbrev">Front. Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">217</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BGenet.%26rft.volume%253D6%26rft.spage%253D217%26rft_id%253Dinfo%253Apmid%252F26150828%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=26150828&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F695.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.695.57" data-doi="10.1093/molbev/msr199"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weetman</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilding</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steen</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pinto</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donnelly</span>  <span class="cit-name-given-names">M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Gene flow–dependent genomic divergence between <em>Anopheles gambiae</em> M and S forms.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">279</span>–<span class="cit-lpage">291</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsr199%26rft_id%253Dinfo%253Apmid%252F21836185%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjI5LzEvMjc5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8zLzY5NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.695.58" data-doi="10.1016/S0074-7696(08)62553-X"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wolf</span>  <span class="cit-name-given-names">K. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">The unique structure of Lepidopteran spindles.</span> <abbr class="cit-jnl-abbrev">Int. Rev. Cytol.</abbr> <span class="cit-vol">152</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">48</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInt.%2BRev.%2BCytol.%26rft.volume%253D152%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0074-7696%252808%252962553-X%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0074-7696(08)62553-X&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.695.59" data-doi="10.1371/journal.pgen.1000212"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bhat</span>  <span class="cit-name-given-names">P. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Close</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lonardi</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient and accurate construction of genetic linkage maps from the minimum spanning tree of a graph.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e1000212</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DWu%26rft.auinit1%253DY.%26rft.volume%253D4%26rft.issue%253D10%26rft.spage%253De1000212%26rft.epage%253De1000212%26rft.atitle%253DEfficient%2Band%2Baccurate%2Bconstruction%2Bof%2Bgenetic%2Blinkage%2Bmaps%2Bfrom%2Bthe%2Bminimum%2Bspanning%2Btree%2Bof%2Ba%2Bgraph.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000212%26rft_id%253Dinfo%253Apmid%252F18846212%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000212&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18846212&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F695.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.695.60" data-doi="10.1038/ng.2524"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">You</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yue</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A heterozygous moth genome provides insights into herbivory and detoxification.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">220</span>–<span class="cit-lpage">225</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D45%26rft.spage%253D220%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2524%26rft_id%253Dinfo%253Apmid%252F23313953%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2524&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23313953&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F695.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.695.61" data-doi="10.1016/j.cell.2011.09.052"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhan</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Merlin</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boore</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reppert</span>  <span class="cit-name-given-names">S. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The monarch butterfly genome yields insights into long-distance migration.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">147</span>: <span class="cit-fpage">1171</span>–<span class="cit-lpage">1185</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DZhan%26rft.auinit1%253DS.%26rft.volume%253D147%26rft.issue%253D5%26rft.spage%253D1171%26rft.epage%253D1185%26rft.atitle%253DThe%2Bmonarch%2Bbutterfly%2Bgenome%2Byields%2Binsights%2Binto%2Blong-distance%2Bmigration.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2011.09.052%26rft_id%253Dinfo%253Apmid%252F22118469%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2011.09.052&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22118469&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F695.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000297376600026&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/6/3/695.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/3/683" title="Diverse Hormone Response Networks in 41 Independent Drosophila Cell Lines" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/3/709" title="The E2F-DP1 Transcription Factor Complex Regulates Centriole Duplication in Caenorhabditis elegans" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/6/3"><span>Volume 6 Issue 3, March 2016</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/6/3.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/6/3.cover-source.jpg?itok=8EDC4x5t" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 6 (3)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/3/695" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-26-qyH9Tg13JHhuOtgrLmQ1IcVrAthxA5GKPqpEYIKU" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/6/3/695" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/6/3/695" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/6/3/695" class="active" data-icon-position="" data-hide-link-title="0">Major Improvements to the Heliconius melpomene Genome Assembly Used to Confirm 10 Chromosome Fusion Events in 6 Million Years of Butterfly Evolution</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460075" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-l2Kxt5bfX1WGPMP8KCM1CtX6DATVSbJ55gVZDEDkRug" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/3/695.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/6/3/695" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-rmEXoGoF9hxZDfmZJrctUkukxSx67ttRqJUVP69deco" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/6/3/695" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/3/695.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460075" id="node460075--2" data-pisa="ggg;6/3/695" data-pisa-master="ggg;g3.115.023655" data-apath="/ggg/6/3/695.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Major Improvements to the <em>Heliconius melpomene</em> Genome Assembly Used to Confirm 10 Chromosome Fusion Events in 6 Million Years of Butterfly Evolution</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-1017-9775" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>John W. Davey</span>, <span class="highwire-citation-author" data-delta="1">Mathieu Chouteau</span>, <span class="highwire-citation-author" data-delta="2">Sarah L. Barker</span>, <span class="highwire-citation-author" data-delta="3">Luana Maroja</span>, <span class="highwire-citation-author" data-delta="4">Simon W. Baxter</span>, <span class="highwire-citation-author" data-delta="5">Fraser Simpson</span>, <span class="highwire-citation-author" data-delta="6">Richard M. Merrill</span>, <span class="highwire-citation-author" data-delta="7">Mathieu Joron</span>, <span class="highwire-citation-author" data-delta="8">James Mallet</span>, <span class="highwire-citation-author" data-delta="9">Kanchon K. Dasmahapatra</span> and <span class="highwire-citation-author" data-delta="10">Chris D. Jiggins</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">695-708;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.023655 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460075--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">John W. Davey </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Zoology, University of Cambridge, CB2 3EJ, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BW.%2BDavey%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Davey%20JW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJohn%2BW.%2BDavey%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-1017-9775" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for John W. Davey</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jd626@cam.ac.uk" class="" data-icon-position="" data-hide-link-title="0">jd626@cam.ac.uk</a>
<a href="mailto:cj107@cam.ac.uk" class="" data-icon-position="" data-hide-link-title="0">cj107@cam.ac.uk</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Mathieu Chouteau </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Centre d’Ecologie Fonctionnelle et Evolutive, UMR 5175 CNRS - EPHE - Université de Montpellier - Université Paul Valéry, 34293 Montpellier 5, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mathieu%2BChouteau%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chouteau%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMathieu%2BChouteau%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Sarah L. Barker </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Zoology, University of Cambridge, CB2 3EJ, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sarah%2BL.%2BBarker%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Barker%20SL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASarah%2BL.%2BBarker%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Luana Maroja </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Biology, Williams College, Williamstown, Massachusetts, 01267</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Luana%2BMaroja%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Maroja%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALuana%2BMaroja%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Simon W. Baxter </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>School of Biological Sciences, University of Adelaide, SA 5005 Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Simon%2BW.%2BBaxter%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Baxter%20SW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASimon%2BW.%2BBaxter%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Fraser Simpson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Genetics, Evolution and Environment, University College London, Darwin Building, Gower Street, WC1E 6BT, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fraser%2BSimpson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Simpson%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFraser%2BSimpson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Richard M. Merrill </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Zoology, University of Cambridge, CB2 3EJ, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Richard%2BM.%2BMerrill%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Merrill%20RM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARichard%2BM.%2BMerrill%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Mathieu Joron </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Centre d’Ecologie Fonctionnelle et Evolutive, UMR 5175 CNRS - EPHE - Université de Montpellier - Université Paul Valéry, 34293 Montpellier 5, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mathieu%2BJoron%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Joron%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMathieu%2BJoron%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">James Mallet </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Genetics, Evolution and Environment, University College London, Darwin Building, Gower Street, WC1E 6BT, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=James%2BMallet%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mallet%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJames%2BMallet%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Kanchon K. Dasmahapatra </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Genetics, Evolution and Environment, University College London, Darwin Building, Gower Street, WC1E 6BT, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kanchon%2BK.%2BDasmahapatra%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dasmahapatra%20KK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKanchon%2BK.%2BDasmahapatra%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Chris D. Jiggins </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Zoology, University of Cambridge, CB2 3EJ, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chris%2BD.%2BJiggins%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jiggins%20CD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChris%2BD.%2BJiggins%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jd626@cam.ac.uk" class="" data-icon-position="" data-hide-link-title="0">jd626@cam.ac.uk</a>
<a href="mailto:cj107@cam.ac.uk" class="" data-icon-position="" data-hide-link-title="0">cj107@cam.ac.uk</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/6/3/695" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/6/3/695&amp;title=Major%20Improvements%20to%20the%20Heliconius%20melpomene%20Genome%20Assembly%20Used%20to%20Confirm%2010%20Chromosome%20Fusion%20Events%20in%206%C2%A0Million%20Years%20of%20Butterfly%20Evolution" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/6/3/695&amp;title=Major%20Improvements%20to%20the%20Heliconius%20melpomene%20Genome%20Assembly%20Used%20to%20Confirm%2010%20Chromosome%20Fusion%20Events%20in%206%C2%A0Million%20Years%20of%20Butterfly%20Evolution" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/6/3/695&amp;title=Major%20Improvements%20to%20the%20Heliconius%20melpomene%20Genome%20Assembly%20Used%20to%20Confirm%2010%20Chromosome%20Fusion%20Events%20in%206%C2%A0Million%20Years%20of%20Butterfly%20Evolution" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/6/3/695&amp;text=Major%20Improvements%20to%20the%20Heliconius%20melpomene%20Genome%20Assembly%20Used%20to%20Confirm%2010%20Chromosome%20Fusion%20Events%20in%206%C2%A0Million%20Years%20of%20Butterfly%20Evolution" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/6/3/695&amp;title=Major%20Improvements%20to%20the%20Heliconius%20melpomene%20Genome%20Assembly%20Used%20to%20Confirm%2010%20Chromosome%20Fusion%20Events%20in%206%C2%A0Million%20Years%20of%20Butterfly%20Evolution" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/6/3/695&amp;t=Major%20Improvements%20to%20the%20Heliconius%20melpomene%20Genome%20Assembly%20Used%20to%20Confirm%2010%20Chromosome%20Fusion%20Events%20in%206%C2%A0Million%20Years%20of%20Butterfly%20Evolution" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/6/3/695&amp;title=Major%20Improvements%20to%20the%20Heliconius%20melpomene%20Genome%20Assembly%20Used%20to%20Confirm%2010%20Chromosome%20Fusion%20Events%20in%206%C2%A0Million%20Years%20of%20Butterfly%20Evolution" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/6/3/695&amp;title=Major%20Improvements%20to%20the%20Heliconius%20melpomene%20Genome%20Assembly%20Used%20to%20Confirm%2010%20Chromosome%20Fusion%20Events%20in%206%C2%A0Million%20Years%20of%20Butterfly%20Evolution" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460075/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460075" id="node460075--3" data-pisa="ggg;6/3/695" data-pisa-master="ggg;g3.115.023655" data-apath="/ggg/6/3/695.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Major Improvements to the <em>Heliconius melpomene</em> Genome Assembly Used to Confirm 10 Chromosome Fusion Events in 6 Million Years of Butterfly Evolution</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-1017-9775" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>John W. Davey</span>, <span class="highwire-citation-author" data-delta="1">Mathieu Chouteau</span>, <span class="highwire-citation-author" data-delta="2">Sarah L. Barker</span>, <span class="highwire-citation-author" data-delta="3">Luana Maroja</span>, <span class="highwire-citation-author" data-delta="4">Simon W. Baxter</span>, <span class="highwire-citation-author" data-delta="5">Fraser Simpson</span>, <span class="highwire-citation-author" data-delta="6">Richard M. Merrill</span>, <span class="highwire-citation-author" data-delta="7">Mathieu Joron</span>, <span class="highwire-citation-author" data-delta="8">James Mallet</span>, <span class="highwire-citation-author" data-delta="9">Kanchon K. Dasmahapatra</span> and <span class="highwire-citation-author" data-delta="10">Chris D. Jiggins</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">695-708;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.023655 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460075--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">John W. Davey </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Zoology, University of Cambridge, CB2 3EJ, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BW.%2BDavey%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Davey%20JW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJohn%2BW.%2BDavey%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-1017-9775" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for John W. Davey</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jd626@cam.ac.uk" class="" data-icon-position="" data-hide-link-title="0">jd626@cam.ac.uk</a>
<a href="mailto:cj107@cam.ac.uk" class="" data-icon-position="" data-hide-link-title="0">cj107@cam.ac.uk</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Mathieu Chouteau </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Centre d’Ecologie Fonctionnelle et Evolutive, UMR 5175 CNRS - EPHE - Université de Montpellier - Université Paul Valéry, 34293 Montpellier 5, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mathieu%2BChouteau%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chouteau%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMathieu%2BChouteau%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Sarah L. Barker </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Zoology, University of Cambridge, CB2 3EJ, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sarah%2BL.%2BBarker%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Barker%20SL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASarah%2BL.%2BBarker%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Luana Maroja </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Biology, Williams College, Williamstown, Massachusetts, 01267</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Luana%2BMaroja%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Maroja%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALuana%2BMaroja%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Simon W. Baxter </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>School of Biological Sciences, University of Adelaide, SA 5005 Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Simon%2BW.%2BBaxter%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Baxter%20SW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASimon%2BW.%2BBaxter%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Fraser Simpson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Genetics, Evolution and Environment, University College London, Darwin Building, Gower Street, WC1E 6BT, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fraser%2BSimpson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Simpson%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFraser%2BSimpson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Richard M. Merrill </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Zoology, University of Cambridge, CB2 3EJ, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Richard%2BM.%2BMerrill%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Merrill%20RM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARichard%2BM.%2BMerrill%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Mathieu Joron </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Centre d’Ecologie Fonctionnelle et Evolutive, UMR 5175 CNRS - EPHE - Université de Montpellier - Université Paul Valéry, 34293 Montpellier 5, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mathieu%2BJoron%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Joron%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMathieu%2BJoron%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">James Mallet </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Genetics, Evolution and Environment, University College London, Darwin Building, Gower Street, WC1E 6BT, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=James%2BMallet%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mallet%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJames%2BMallet%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Kanchon K. Dasmahapatra </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Genetics, Evolution and Environment, University College London, Darwin Building, Gower Street, WC1E 6BT, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kanchon%2BK.%2BDasmahapatra%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dasmahapatra%20KK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKanchon%2BK.%2BDasmahapatra%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Chris D. Jiggins </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Zoology, University of Cambridge, CB2 3EJ, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chris%2BD.%2BJiggins%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jiggins%20CD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChris%2BD.%2BJiggins%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jd626@cam.ac.uk" class="" data-icon-position="" data-hide-link-title="0">jd626@cam.ac.uk</a>
<a href="mailto:cj107@cam.ac.uk" class="" data-icon-position="" data-hide-link-title="0">cj107@cam.ac.uk</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=695&amp;rft.epage=708&amp;rft.atitle=Major%20Improvements%20to%20the%20Heliconius%20melpomene%20Genome%20Assembly%20Used%20to%20Confirm%2010%20Chromosome%20Fusion%20Events%20in%206%C2%A0Million%20Years%20of%20Butterfly%20Evolution&amp;rft.volume=6&amp;rft.issue=3&amp;rft.date=2016-03-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Davey%2C+John+W.&amp;rft.au=Chouteau%2C+Mathieu&amp;rft.au=Barker%2C+Sarah+L.&amp;rft.au=Maroja%2C+Luana&amp;rft.au=Baxter%2C+Simon+W.&amp;rft.au=Simpson%2C+Fraser&amp;rft.au=Merrill%2C+Richard+M.&amp;rft.au=Joron%2C+Mathieu&amp;rft.au=Mallet%2C+James&amp;rft.au=Dasmahapatra%2C+Kanchon+K.&amp;rft.au=Jiggins%2C+Chris+D."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460075/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460075/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460075/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460075/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460075/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460075/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460075/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460075/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460075/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460075/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460075/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460075/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/6/3/695" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-15" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-22" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/6/3/695.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/6/3/695.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/6/3/695.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/6/3/695.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
