<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/6/3/589" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/6/3/589.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/6/3/589.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/6/3/589.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/6/3/589/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/6/3/589.atom" />
<meta name="HW.pisa" content="ggg;6/3/589" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="FW: An R Package for Finlay–Wilkinson Regression that Incorporates Genomic/Pedigree Information and Covariance Structures Between Environments" />
<meta name="DC.Identifier" content="10.1534/g3.115.026328" />
<meta name="DC.Date" content="2016-03-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2016 Lian and Campos. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="The Finlay–Wilkinson regression (FW) is a popular method among plant breeders to describe genotype by environment interaction. The standard implementation is a two-step procedure that uses environment (sample) means as covariates in a within-line ordinary least squares (OLS) regression. This procedure can be suboptimal for at least four reasons: (1) in the first step environmental means are typically estimated without considering genetic-by-environment interactions, (2) in the second step uncertainty about the environmental means is ignored, (3) estimation is performed regarding lines and environment as fixed effects, and (4) the procedure does not incorporate genetic (either pedigree-derived or marker-derived) relationships. Su et al. proposed to address these problems using a Bayesian method that allows simultaneous estimation of environmental and genotype parameters, and allows incorporation of pedigree information. In this article we: (1) extend the model presented by Su et al. to allow integration of genomic information [ e.g. , single nucleotide polymorphism (SNP)] and covariance between environments, (2) present an R package (FW) that implements these methods, and (3) illustrate the use of the package using examples based on real data. The FW R package implements both the two-step OLS method and a full Bayesian approach for Finlay–Wilkinson regression with a very simple interface. Using a real wheat data set we demonstrate that the prediction accuracy of the Bayesian approach is consistently higher than the one achieved by the two-step OLS method." />
<meta name="DC.Contributor" content="Lian Lian" />
<meta name="DC.Contributor" content="Gustavo de los Campos" />
<meta name="article:published_time" content="2016-03-01" />
<meta name="article:section" content="Genomic Selection" />
<meta name="citation_title" content="FW: An R Package for Finlay–Wilkinson Regression that Incorporates Genomic/Pedigree Information and Covariance Structures Between Environments" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The Finlay–Wilkinson regression (FW) is a popular method among plant breeders to describe genotype by environment interaction. The standard implementation is a two-step procedure that uses environment (sample) means as covariates in a within-line ordinary least squares (OLS) regression. This procedure can be suboptimal for at least four reasons: (1) in the first step environmental means are typically estimated without considering genetic-by-environment interactions, (2) in the second step uncertainty about the environmental means is ignored, (3) estimation is performed regarding lines and environment as fixed effects, and (4) the procedure does not incorporate genetic (either pedigree-derived or marker-derived) relationships. Su &lt;i&gt;et al.&lt;/i&gt; proposed to address these problems using a Bayesian method that allows simultaneous estimation of environmental and genotype parameters, and allows incorporation of pedigree information. In this article we: (1) extend the model presented by Su &lt;i&gt;et al.&lt;/i&gt; to allow integration of genomic information [&lt;i&gt;e.g.&lt;/i&gt;, single nucleotide polymorphism (SNP)] and covariance between environments, (2) present an R package (FW) that implements these methods, and (3) illustrate the use of the package using examples based on real data. The FW R package implements both the two-step OLS method and a full Bayesian approach for Finlay–Wilkinson regression with a very simple interface. Using a real wheat data set we demonstrate that the prediction accuracy of the Bayesian approach is consistently higher than the one achieved by the two-step OLS method.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2016/03/01" />
<meta name="citation_mjid" content="ggg;6/3/589" />
<meta name="citation_id" content="6/3/589" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/6/3/589" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/6/3/589.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/6/3/589.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/6/3/589.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.115.026328" />
<meta name="citation_pmid" content="26715095" />
<meta name="citation_volume" content="6" />
<meta name="citation_issue" content="3" />
<meta name="citation_num_pages" content="9" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genomic Selection" />
<meta name="citation_firstpage" content="589" />
<meta name="citation_lastpage" content="597" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Lian Lian" />
<meta name="citation_author_institution" content="Department of Epidemiology and Biostatistics, Michigan State University, East Lansing, Michigan 48824" />
<meta name="citation_author_email" content="lianl0501@gmail.com" />
<meta name="citation_author" content="Gustavo de los Campos" />
<meta name="citation_author_institution" content="Department of Epidemiology and Biostatistics, Michigan State University, East Lansing, Michigan 48824" />
<meta name="citation_author_institution" content="Department of Probability and Statistics, Michigan State University, East Lansing, Michigan 48824" />
<meta name="citation_reference" content="citation_journal_title=Am. Stat.;citation_author=G. Casella;citation_author=E. I. George;citation_title=Explaining the Gibbs sampler.;citation_pages=167-174;citation_volume=46;citation_year=1992;citation_doi=10.2307/2685208" />
<meta name="citation_reference" content="citation_journal_title=J. R. Stat. Soc., B;citation_author=J. B. Copas;citation_title=Regression, prediction and shrinkage.;citation_pages=311-354;citation_volume=45;citation_year=1983" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. Crossa;citation_author=G. de Los Campos;citation_author=P. Pérez;citation_author=D. Gianola;citation_author=J. Burgueño;citation_title=Prediction of genetic values of quantitative traits in plant breeding using pedigree and molecular markers.;citation_volume=186;citation_year=2010;citation_pmid=20813882;citation_doi=10.1534/genetics.110.118521" />
<meta name="citation_reference" content="citation_journal_title=Crop Pasture Sci.;citation_author=K. Finlay;citation_author=G. Wilkinson;citation_title=The analysis of adaptation in a plant-breeding programme.;citation_pages=742-754;citation_volume=14;citation_year=1963;citation_doi=10.1071/AR9630742" />
<meta name="citation_reference" content="citation_journal_title=Technometrics;citation_author=L. E. Frank;citation_author=J. H. Friedman;citation_title=A statistical view of some chemometrics regression tools.;citation_pages=109-135;citation_volume=35;citation_year=1993;citation_doi=10.2307/1269656" />
<meta name="citation_reference" content="citation_journal_title=IEEE Trans. Pattern Anal. Mach. Intell.;citation_author=S. Geman;citation_author=D. Geman;citation_title=Stochastic relaxation, Gibbs distributions, and the bayesian restoration of images.;citation_pages=721-741;citation_volume=6;citation_year=1984;citation_pmid=22499653;citation_doi=10.1109/TPAMI.1984.4767596" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=H.-R. Gregorius;citation_author=G. Namkoong;citation_title=Joint analysis of genotypic and environmental effects.;citation_pages=413-422;citation_volume=72;citation_year=1986;citation_pmid=24247951;citation_doi=10.1007/BF00288581" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=D. Jarquín;citation_author=J. Crossa;citation_author=X. Lacaze;citation_author=P. Du Cheyron;citation_author=J. Daucourt;citation_title=A reaction norm model for genomic selection using high-dimensional genomic and environmental data.;citation_pages=595-607;citation_volume=127;citation_year=2014;citation_pmid=24337101;citation_doi=10.1007/s00122-013-2243-1" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=P. Pérez;citation_author=G. de Los Campos;citation_author=J. Crossa;citation_author=D. Gianola;citation_title=Genomic-enabled prediction based on molecular markers and pedigree using the bayesian linear regression package in R.;citation_pages=106-116;citation_volume=3;citation_year=2010;citation_pmid=21566722;citation_doi=10.3835/plantgenome2010.04.0005" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=J. M. Perkins;citation_author=J. Jinks;citation_title=Environmental and genotype-environmental components of variability. 3. Multiple lines and crosses.;citation_pages=339-356;citation_volume=23;citation_year=1968;citation_issue=3;citation_pmid=5250121;citation_doi=10.1038/hdy.1968.48" />
<meta name="citation_reference" content="citation_journal_title=R News;citation_author=M. Plummer;citation_author=N. Best;citation_author=K. Cowles;citation_author=K. Vines;citation_title=CODA: convergence diagnosis and output analysis for MCMC.;citation_pages=7-11;citation_volume=6;citation_year=2006" />
<meta name="citation_reference" content="citation_title=R: A Language and Environment for Statistical Computing;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_abbrev=J Anim Breed Genet;citation_author=M. Shariati;citation_author=I. Korsgaard;citation_author=D. Sorensen;citation_title=Identifiability of parameters and behaviour of MCMC chains: a case study using the reaction norm model.;citation_pages=92-102;citation_volume=126;citation_year=2009;citation_issue=2;citation_pmid=19320765;citation_doi=10.1111/j.1439-0388.2008.00773.x" />
<meta name="citation_reference" content="citation_journal_title=Journal of Animal Science;citation_journal_abbrev=J ANIM SCI;citation_author=G. Su;citation_author=P. Madsen;citation_author=M. S. Lund;citation_author=D. Sorensen;citation_author=I. R. Korsgaard;citation_title=Bayesian analysis of the linear reaction norm model with unknown covariates;citation_pages=1651-1657;citation_volume=84;citation_year=2006;citation_issue=7;citation_pmid=16775048;citation_doi=10.2527/jas.2005-517" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="FW: An R Package for Finlay–Wilkinson Regression that Incorporates Genomic/Pedigree Information and Covariance Structures Between Environments" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3.cover-source.jpg" />
<meta name="twitter:description" content="The Finlay–Wilkinson regression (FW) is a popular method among plant breeders to describe genotype by environment interaction. The standard implementation is a two-step procedure that uses environment (sample) means as covariates in a within-line ordinary least squares (OLS) regression. This procedure can be suboptimal for at least four reasons: (1) in the first step environmental means are typically estimated without considering genetic-by-environment interactions, (2) in the second step uncertainty about the environmental means is ignored, (3) estimation is performed regarding lines and environment as fixed effects, and (4) the procedure does not incorporate genetic (either pedigree-derived or marker-derived) relationships. Su et al. proposed to address these problems using a Bayesian method that allows simultaneous estimation of environmental and genotype parameters, and allows incorporation of pedigree information. In this article we: (1) extend the model presented by Su et al. to allow integration of genomic information [ e.g. , single nucleotide polymorphism (SNP)] and covariance between environments, (2) present an R package (FW) that implements these methods, and (3) illustrate the use of the package using examples based on real data. The FW R package implements both the two-step OLS method and a full Bayesian approach for Finlay–Wilkinson regression with a very simple interface. Using a real wheat data set we demonstrate that the prediction accuracy of the Bayesian approach is consistently higher than the one achieved by the two-step OLS method." />
<meta name="og-title" property="og:title" content="FW: An R Package for Finlay–Wilkinson Regression that Incorporates Genomic/Pedigree Information and Covariance Structures Between Environments" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/6/3/589" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="The Finlay–Wilkinson regression (FW) is a popular method among plant breeders to describe genotype by environment interaction. The standard implementation is a two-step procedure that uses environment (sample) means as covariates in a within-line ordinary least squares (OLS) regression. This procedure can be suboptimal for at least four reasons: (1) in the first step environmental means are typically estimated without considering genetic-by-environment interactions, (2) in the second step uncertainty about the environmental means is ignored, (3) estimation is performed regarding lines and environment as fixed effects, and (4) the procedure does not incorporate genetic (either pedigree-derived or marker-derived) relationships. Su et al. proposed to address these problems using a Bayesian method that allows simultaneous estimation of environmental and genotype parameters, and allows incorporation of pedigree information. In this article we: (1) extend the model presented by Su et al. to allow integration of genomic information [ e.g. , single nucleotide polymorphism (SNP)] and covariance between environments, (2) present an R package (FW) that implements these methods, and (3) illustrate the use of the package using examples based on real data. The FW R package implements both the two-step OLS method and a full Bayesian approach for Finlay–Wilkinson regression with a very simple interface. Using a real wheat data set we demonstrate that the prediction accuracy of the Bayesian approach is consistently higher than the one achieved by the two-step OLS method." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3.cover-source.jpg" />
    <title>FW: An R Package for Finlay–Wilkinson Regression that Incorporates Genomic/Pedigree Information and Covariance Structures Between Environments | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"fkm25or48SCHfW6_0gEUC99GJVVHguwzM8eEIg6SieA","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460084","apath":"\/ggg\/6\/3\/589.atom","pisa":"ggg;6\/3\/589","ac":{"\/ggg\/6\/3\/589.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/6\/3\/589.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;6\/3\/589"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;6\/3\/589"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/6\/3\/589":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022IqRNbPhcgUv6R9JsVY+ye0MLPi1MmfwHwTqclN4RxRMYi7A9Vg4GdwJZ\\\\\\\/cLbwoBSAzsguG0fgMEb2QTK6oYtHEbKb84dqZNmImBAT0X6J2kVFiw\\\\\\\/bsiNovrcLkvpfYLiLZp9yjlqz7PzFnfAeFsuOnvumZI1WpeY9M0n6JqumhrKCCMdiOknqj36AXbsIb84GXW3tFZcnO96ImHNP\\\\\\\/tY5Ou2L1jwjxzJ6FBhTmkrjYBgyMgl9SGFMZHKH5Xh11OBMh+sq86sSF11Zw54RpHRFdrbXcNupOD4gPPHYak5s\\\\\\\/MpzE9K5HdpXca9hUTM0cdFZZ1Ma8a54R93m+C9GzeE7IZhvcicgTptth1gxziN+Ega7xDB\\\\\\\/DRBO3YwlBq1k6j5CI8Wu+BkV839zRnmqt25beEjCerCQ2fTluYtjGGl8E8QMMJC1ERv2dtl\\\\\\\/UvZuSf8+gxgu9AOYl5b2CzXPhZy6OkuXBOL\\\\\\\/kMsHvzbjAMP7Q6SHBBlNQSp2XDkOaoQCTSQRn2o3NvHrbUMaihi7sU7wJSDRnO56b806C8+fwYPwvjIsBCNliE1r1ROLiW3pDv0diTvieefUm3oUO7J9SuP0cWxOTzysD5mA1R4m1VB2WR+SgB05TwAViljfEFcSGaq5OEL6BYjuc2FCp1c7AbbDP+0MEblTvlDJvs5\\\\\\\/v0MqaFLXPW3hyCllgFkwM3GNMWra6ANIxct6ZDmc1X5hknkQqmu\\\\\\\/o4WBG\\\\\\\/PTHqvDefhRf6TxCTyJiCb6J2OjByfQ9roqkEzz1+\\\\\\\/HxIXl0JWE3nBc+sJI9R4rpNXdM6SvkbXeiJQcbh5jWBCo8tzaJklBmDmLMsaqAai0Bhh8Ep0X\\\\\\\/8mmk7xup9LkzlBJOEg5w7r1sxYZ9vk8iVUzo0zNJzIF\\\\\\\/CAmAJReHxTBjVIw87HN91uu4KjOrsQokUq1AYVX897RFhuaquAuhCaWIrlWJtwYBgOtHxzs8JpwAS4TkRtxRKvHHsrKe4vIJX72xXL0VouvaqA9CoBDK3mE0SdG6zGXInjh3iSqlj5jnZMTyfYEqE1\\\\\\\/8Vem687wqQbnwKu9UbVqJiiVb\\\\\\\/sPXH8MVCtDqJCaLb2UxLnBblVQZb5qZIy774DBEFAMGLUw4uXS7yBWwx7VNi08bAyPprsTkMMZ0+xI\\\\\\\/oGb2AiAfvGqKTTU9GjuVhxb1zCP\\\\\\\/Od3DcTp+QlNl6r00x6ipn4fcm+Iant9EmyZrdpWM9HrzipMPErnuAoC2a72YgHQPOcHE4XqY1PyQne8Hio6jP8DPM4Vp4mxF3V32gG\\\u0022,\\\u0022iv\\\u0022:\\\u0022vsyBWeQcYLuG4LmoWM61Fw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022b67d904580d21b1baebf780345e1f7702a381a525eb11231ef6af252c889f2b6\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Fyr\\\\\\\/yzZiNjnPiJFV4rx82hoVX1C4JKs3LOEpivb7ZEMjqmXilK2WQRSyIry88OoHo9NB5IFrmTg92srgqRAubl2u5VpS5w2PRXggxbs5vX7c271Q\\\\\\\/n2hspZXqG26FK0w7pRS\\\\\\\/06auTBx8tI8AoaFag1KxfO83B87eypbG46xDa7z6l1Uv7ZbuvK003Do6ra29tgml7PoUErj1YaynA5GgbqgTHPgYZksGT+dSP9SkhXz+Sdicd2oL7cpPbcntoJSJ2J1jUYJjrTWpCro2Up3HtsjaQYpC01CxHq5fG7FJ9ALdGxy\\\\\\\/3tA2W+cmVOP05mxylVzeVNPgNIKCpdbJB3pegEt832RIium06bd9boSFcfwez1IPwAoA28WeSQeJy4PGh5L92SiZxVUR2PXkbP+whAnQ8d2HyHnGHaMdwOKBEXTHbsPo6O1MbJW5PjFVG6EMrv9QwZz9+AYNwmCNkJdXeKP5yTIN2xaNL+fCcCLGhPdOGrteMUIdNiOdolH47FMGYhmlGSs0U4GiChcpKbqdA3ar\\\\\\\/f7uei7QLaWmlwRdjkFIixcaR6CrXEkwF+SvIcIOAE8Ws+ompcCLXBzSeYYxC92HM8Ne\\\\\\\/rA8VW\\\\\\\/BW74JdAUKKNf3F4AK9JLFGdUyoCWJHRY8MmWD5s5q2G3krHHHsSjmZWw3cvOkZnTKT4Ep7wsK\\\\\\\/COBZ+WcZNUZnRZpREmC\\\\\\\/TIsWrSP7XZP\\\\\\\/1WqqgdyhSAfVjWVe0MjrCdJfT2ggephpODBH3KedXNQbMshCzLktn76scdkMOCq9BIwVRQFIeBuHbf6spqlTgzAdbrq3M2ix4PdFk9ozm7kLeX3C9k3Thr7ZOpnkTQ7R1lb+nMVcRl97sI3c2B\\\\\\\/\\\\\\\/kavBryJJ9nBorxLCCbZROVKMw\\\\\\\/rl4IT5L+r8h13atNxY\\\\\\\/k88P4QNV\\\\\\\/IINTI2qHtk65ESljFjPadXqKdX5scfgiV3AbWaOmvmx4b9OA3PUC9Er8FTXD68e7zsbPeW\\\\\\\/syKzbkWyMKhOZh39pkNvuUNJkAMvWK8n4K4a+Pv73X2hE2Yj931J\\\\\\\/a6gE3UA+bSQTUaIKFLvEb9ZsyIIBS8qnrALs3yKlMESgILCX7vtll8vyh1wIlvDY2XF8k2Zq3F5El19jsht0ZmJrU2lPNvdAN+ZyiWamHK7mU2SFmoWFI6zLHnPwFQosgCiVOHr2jwj0\\\\\\\/vE5WV38tBcSzewPxhtoQvLyI4cFFE4ZOBYtQ53FD3ndQh3ejtvHhq7DQR2qje+4KNx+khuBYWbBPNw1Afu2IAlrugJPZLue4rkuWwhgNXktUXXJy6N41e6heRQd6OJ6K\\\\\\\/L0OCk\\\\\\\/5E9d13P3luRG+4gZCFjRILnyL3L0rNk1Rl4VOQqP9wGmLK2wx9ppysH8ZoLlVq6BCLFPPsze04rdcIWV0M0yvTfSanjKKhtXVCGDY8cRY6uwujj2jbGvGCjjeqOjTJ9i1+x+yZcu6NtCGhRTxn5FWVB1EJrOqP7uqmucjc17orxDKMu7N4XWGdVw2hHyd0X15AcPdstN+p6PJh8fXfRqFt\\\\\\\/waDrOR+n\\\\\\\/0H7+I3z0HfVcKAygTKIyJKc3D7B5SfqdUgjBv95wewI6fvyfhrB3otLNIZc0TUM87O\\\\\\\/IqKtSMQAbRigEBCKroIacKBV4hvNFJIf9+Wk+4mHd5AH+z3KQNrYzzB4m5d6ky3Yi5NTKu+VZ9SYDqJl1gEy0DKv9Pm3416zFiKZUFTDPqOuyS52\\\\\\\/YMjoNSotPQc4h5ylwPBO0xO5l0rLmEHE48GzBkbwUcVxgcU9MN0pf7myW5KGpIR4KgC\\\\\\\/v2zy4XUGaLvIFkmF0QPEkX9+m8az2fHqhqtv7hfXWWXqr8SyCZ6l2SllXWe1D7SddgXFtHSrd7aQz\\\\\\\/M6DU7YIRLOZWT21Nx1ZgsBenbQrjhUEm8shOfgDRfO8j8fVc3SOHCJ4MDH2QyedYp9unZN2Zbb\\\\\\\/tR1cr2mnvEs4L4B3SXkdE6SzNGm+1vJaVneJwb4mLizTnE1DRnOs4UF5ij\\\\\\\/S4q4STonrIvfvGZxAUpnuxiZYSP\\\\\\\/ZjS87JzuV4eJb4Ti5fLJ4ctiJfnEiAaUBCNHhZzeYVYQqqeFxLkIqB+w1j9+0kZ0x8UZM9ids5lwj9\\\\\\\/WKHJtbmxQ01AmfhpfN7W1r\\\\\\\/4Gxajkg8YxQ23ALA9Zry+6YKliFYwBeI20lvsqKdZoq0vf\\\\\\\/RSfTgE4l00A\\\\\\\/hv1kPZv9K4hTrA8HYAcR\\\\\\\/GyAoUIyfyhF\\\\\\\/WSRizh9DqJCAa79w5w2QOVJiJBEFoOcYUCdsPWgflT8gUGul1JMEv4SJLbYLvhV1rxaIJoRmJCuH08Zr4opJNVDl\\\\\\\/TihaFF52qMLzIMzsxg2kDvApJlNg2UDFJFInJgBhfOHcrZW45gJGBf2Q1BdYuIUIWWcqP6YuUgT79+O8+e65rYoAGc90f+rxt23tApvzuNFs1cSXfbVJNc\\\\\\\/XDdzE7CpcAM6hTEb9wcqs4H2g0UjLmRGTrgzWOiFMb+31MB9xAHIqV1WvNfhyUHwBSxapMf0J6Kgem8Abxvee2irPJAeeBgbOSz8h8VIbtgaH4YMytLz22lnMYxCbAIxPoP5tBGiup7zqrxn8iKmT6y7lmS9C+QXENZd4LLGKSCb6pH+\\\\\\\/oZPequlcLwSpl3WVH86LzUagBMqwhEYZHX2ZfVaQwNVxdzVXNSW8Hs+8wVRYg0\\\\\\\/9WZl6BroBt\\\\\\\/XVDCB08bvj555qWBzIT0M9OyIv\\\\\\\/Ih97enkRy1f1ntOd5SmeflAySiSQbMTKcykm\\\\\\\/3NHMVovYuuhvODeQhVN+FQYeuqZ6MMHgPWu6HTOcsLf8JlOuMlxoKAf9h\\\\\\\/E6b3B0txSVONYt21PwhqPOmUK+p6TJgT+K+PJdY9gTDyHJgX368FDPgllALCL3xOI0RahlVx27ptFfvtdT4XXcNKF0hua5Eo1wCYCRhyeNWIlpEHU1xc5dB9fTdwBnuUIFH535944kwArPVZc+ltGmZj8WL5oiqoT0TXYBh244knQS4Iv1FE81UJaXq1Ych4cfZ8u7LriDwiqcyYVTRfkX5Om3vmsHgKsliX9oqOzg8ueKSNK0MIil4y3HwolfXPsJvMlg3Qqv0Bnvypg3hbuuishr1eo2ZVKirs8S5Oq0POFP6B92rkjcDnaRplGy7svqpXJ9w3jn+c9bRIOrUXGjqgh4djUR1epdN0s64+DNaeeqLtjTAwCU8FdLFR2n4MtS3ZWTDJntgFNXKMcpEXlDhm\\\\\\\/L6c2bgyfSitRS1JPIIPn0QRfKLm6QeTAsO8nU1bqdYks9x6EY5vkQ6H6YR0vap5WKV0wpUGy7H9Z8gAL9zsUd1UXPJcV8z0UQP6pUGB3floEqzm7zh5JH\\\\\\\/AS1M312+\\\\\\\/JzscsBXqOTXt+ingSOZy5eU1tJ+KcvprCiS78SISnQdeMw1Ssw5NOuyqZiWs3wtcjQXey9Oe6WWwHNh7Q+OZJbBqDnXXNw277IS2wvrZpkxK3jGiQcIkZL7T9z7LsJxfd4nKD+62j1xoIWt92ZMgn\\\\\\\/7mPmli\\\\\\\/V6VaP5ZAHp1xFn0ZxqUAPvqhHxPTtqgJQ1oxlDNn2T62n9\\\\\\\/Q6Xx7Jb11ozjPF9bonYRjnedxGb1VWNRcKvQFuYqjbwjtWSQhJRJoOUQBkKxvXdj1caMwVmae8ZJxB2l6sTq548qZGs1PW34U6PzH8FyImsbdBMk\\\\\\\/Z4YyHIhMT0bPKJqVwEcXPaxqzE2pGnT9h9jTDJZR+DNUwX0nL6iOlrkA7yPZb9PWxDmGMRVevp4z4CABYw4\\\\\\\/0Yt++rbwr2+h2G2\\\\\\\/Xn2tBppzMXZTCDgCJv7Id5ray74zDCbkS+VFHKNI8se+RneGXfF2kcBKF7pJynqt4wDyyHOwWyXag3DAhQ6PR4VZsQNOt\\\\\\\/mBatkNOTDhwLyWZzugpSBpt1cFib17IxgATeP4pWrsZKKL5T7VBw56qTKEOmPcVavRM3ivNGIEGBJmu6o4bb0RSlLrf4tPTTTozLXK2rmzmCtyfGNArEw9nvrSiauCXs4CPpRtuwl\\\\\\\/30ksfNALB059SqgvIwtS6SKc+5DXQI\\\\\\\/BQ5k0kyOL3DXW\\\\\\\/iFGH8xVE3ALHven+ToKpeK2b3zvkgQBJ7gN++bKRHSRYUubZSYhVH1bc6n2BZP5eR2gzMeeQ1Z4IHyc876nRQ\\\\\\\/IRq\\\\\\\/qvyFvai3JgXjoJ3HHXEw6PSap85cgxrEL0yKPVHkK8N7ocdUpy+1w\\\\\\\/kDFPp2QLxb7qhXT7MQgkGXTStmUqqHV0VkLZppPsEpYp34poHXnKHpUQRhVnXRyQ2bi8eIr8JGSF0EA9Mkj1itpYQfYlAoUuvB66I988rSCI8+B0+kP9piPmwF2YZKz86n9ENvB03jv0+3IR2OYjtcO8VvGQVjNthbRaFDzzp29olo\\\\\\\/16zjqFGNmNmhRPXcO578seFhnf1f2miskzTJyiL9DdFbwmPqhl1EjgalLcNminfSGPwYbxkQsFpkX3Qpb2rUlf+1lWVW6zOUiYXmqea6cK+SStxa9Tu7lNbESd5YaNVDwh9G2yxBc+3T3qrKJ4qAyetSPkEHH+FrTEH5XhOUzrO0gIrV2pAkiigKXfSjQTBxnulzqWfXuR1AA9L49KeRkHSfmwBspnD1csZeEv6zO+si2idAL7Uyye972H2uC5zsv26f9P1lQNBpG5JzI\\\\\\\/2Xi0tfpfzcezmixLkjHEb\\\\\\\/z5R5ZYaH2t2CGRElyR85GCNX25hTs6S5+Dv0A1SbDL\\\\\\\/WlI1Z1j1HzcZiKzf6v\\\\\\\/4Hp+GuEoxIES5iLdaow1GiJzgYrA\\\\\\\/4KDcl5VZE6vibS9fn65unEBl\\\\\\\/XZpA2\\\\\\\/z0P\\\\\\\/8aZPASt9pjRFarzzGMkK+YtA5w4Mr9PMd9Qmy4eTJKOPbYlSD8EW1+BUPVS6Nl4jwG3K9J1d0y2AhWT0EraDkM3s\\\\\\\/iKSYeIJug5kLeuF9ex0ia+zt8YJWz9Wdw2j8RBr7FOoIvhOALLF2ERyw6e1KFf+RSXmNKrAty7XLwuH2O2kE4FfjT1GZaTHDxStmxOZl4h\\\\\\\/GtZVtSLWsSCZuEYOjlM3od3utaeRBnvG25i1gFmJdlxI+WfURXdq66mHwPxNpE\\\\\\\/ETjJqMQWgyps7FW0mgiAFPFOqYRqChO8+SIxmd1L8E1\\\\\\\/j+GnUUxoZCbYPpQiLNt+tdORdGn51NUCeRfkiw8CqLoMKAcijXmQBkaQ+wNIPN68K+ERW9EPMgqiU0qmh78+8aNbuhiOUSmYpg4zUreOMcuU8K6twTSsLOKOjKNVCLZuGlJwrjBIRrub29veZkyL0JIH7srjrbqB2nwB3Ig9Bhh4h85kfBYWFx9cZT4SP4ZQMq+NQHiIawd9PbkppZxdbSK9jhYBVtDMHao3hH2TN+RF3\\\\\\\/bv9Out3x3lGG1m5Xs1sxh3lIOaZ0rCJY5LKwcsxTy3mq1LiSzcTyM3OTjkzp2ejB+Lz0ZVQdBqy0gbBuuEqqVxS7lSoF6Swjg73NNQgrwZDmvX1\\\\\\\/5FTwLtdEHu7LryOI0oAHHtAsWzNEQZ4QE4K2SP7lfeHhLXpIM3JjUeETMktaLMChLXovv2xeCdu2j1+ekZApUnl7oxZlQ6sEvxicZcb5j3mZiKmKqgdUM78BLYZl\\\\\\\/HFHyxclFJZKURiL0FpqiGQEl6I4+bjTwgt+J+GYBFQSFrEexcqU5wP9jjALcuk\\\\\\\/sp5XEfzf3UfndWFyAVz5RXZiCnV3vRyBRX2b8UMzReL0\\\\\\\/MLJtlfp+uo8aqvHhzUwmlkc9GjD2MqLZ9UBn9n0IlR5H7JGWvzxYp3GcstOL7ZK2nW+7UyBcjTWEgsJXzJ0PDclMSheYIWFmKk1bea0G5MNCOyV20B0mE5hwfve8mqyPsd41a6o8EduW6jhePN\\\\\\\/jAtXXA6ImA1Gib7fCDV1sywEAFqy5Lu54cKufLujsqAjeWVwKWWyMU32YJb74CDPVMalQJWbkMZ9wOTrHQah4ekBgXFSgnhJzYLhO29yjPL5WUMl0BhI8JY9ycEm5IL0+1Z\\\\\\\/STQ2hvibns9J9jYE6VT1lC6OOHXjuaq8xPOJwfKJgsH7XzVfzW34RiHDEgmT8jFJ0CKP50HSdnpgdCpvYnsfie+JodMWPongq5fjjsM20Eflkdw2C\\\\\\\/V+qXGxZKUR66sOwTH67YrF1dlhdu\\\\\\\/peuRD4Xn7cecUjcNNvc\\\\\\\/P041uAMsQN6k1qkKx24ZJV5EZOWBx1Ws8Z3NITRXDYrkqSvi0dhcN4sju44+4HTrKYEgLmtMfeDTZOy1nDdVcVMLikQKoMPXbPpVwLgAyWzZ\\\\\\\/LmfWcuOK4L3SAjOTHyYPc8WJAH7nhN\\\\\\\/WKFMMCkHr4RTPII4izUwWwfjVWecheTY0qFL7xZPyOrPnPfpqmfZ9UnMtKqJRKurJumQMdhAuvzFjFOFAKVXI+ogavKlR6aJM\\\\\\\/McnsEC81afHhJDM+EKEbx3Dp43A\\\\\\\/X+hCc1jCkOwHukS+U\\\\\\\/\\\\\\\/iqLchyk35GFwlPGVYjjlwn\\\\\\\/Px2WpARr6BZxgIAQXGqBFvCoqzPxQkfbltC1ZcvhVGWx1tqC0rWay9fYLN8oiSb\\\\\\\/zrnwxxceQuuQoaxQuHd6N3DoC00FzncrdgZVvxniVUQITSkKB3rsn\\\\\\\/+nGZPwniEzDOYAweI6zgVvG2lZysSssjBVNkaMmr+qraxvRngszCEv0bRkYaINRs+fy4Cte\\\\\\\/zcNbVM+w2hSDi1zCspggp6Vq4yK6fy8XaRFTI6c1s0OSZOY53mi7pIfXSCrC\\\\\\\/at\\\\\\\/PCc0oRyYQK5K8QTgWDLokxY58IpKcojhrwSJNQig8Pm7gnNJwf8lncr1bDY7NLiAxTdtS2J5NP5skdTBr6cIeo9v9udAwEc29sJ4wt0m0af3XiXstdyQcOVkJIWkp2fOSjZ6nK9Rpk33CcF4BQGaS51\\\\\\\/qAkBxEGvhLRCIaxpgQNngeQ6OrZ7VwkuTKjjijOzLG6CADL1oPCo247h8eRC6nbZl4vPNH\\\\\\\/9UyeabvY8ndg4F59JvqEfqLyjDv8BOI+Py1ddnlhp0JpiuAZIJ4ayI1cvYCydTlC++9FZiOhvISryxk0tj5rajvnB6lyvLaBDJr7iaIbB5qDro5B9aB+YMkUZx1jHl1rm09h8d0\\\\\\\/MsY3mHmu9N8GbSYwvXDNrHfOMlSMO4yxgyYU\\\\\\\/owIhyBjLJE5Z\\\\\\\/EHdJkn4C\\\\\\\/5ZFAF6ZzVLKCV5mrntZbp1Ocd+JJhd5yRYFFISOiBDucn626FyXLG0xlZVbgCM149VMCVs1RbPyTjKl+Icz3ZrMgd8nV59iHfgrYAqhZm8gyRCjYgUCwsr\\\\\\\/qBuPf4JBDzTp6NWYY50u6aILlqNde5zdJEDLXMeop7TH10DI0v5mDRWZtMN3HH5uqymhF9RACk0YUgtrgMPRamKrXhgvdCfMOEYWEt1dQWtVAf4Dl0qZIXdty6vOC8ybzwFk0BaQ6KunxarRRdWyXBIZDJPh7DzYiZTD+4d2Wa6BNtK0UH3aoyFGpGIhVrwJe\\\\\\\/s\\\\\\\/5iBk4oX3K+wZT5ltbYDvXL8JYajjDU45kPzFFBWha6XglB5khwY7wuaXWNmw2nyI8aJZBSWjdN8XJ49YutgBbZiVaG7YTG4vcGP7isoDZ8neCFuMUA9fE3x1X9hYt1UzjSg8byuZ+ag9j1q+wfYZD4JFksPnTgWjVvw46VnWhHagaY8t0WciFIuFzdvGq4pNhUfIyBCgMSYYF+vxllgyGYd09fOaXPPG3bwBYDGi\\\\\\\/\\\\\\\/FSt2vxJFgjZUnowMVQd1H6BQ5x1sL9qenZrNNuVitFweGClhr1KtobVpBk1zFaWXRzag63TpmEeORrQKR8gkG9As3Q8z4zWe7gCMS\\\\\\\/94ZXubNCd4XDwTL2ZwNHhbGZIcXJo\\\\\\\/\\\\\\\/mhR39SNFbtsGiXadE8NomlYiOWV9e7nVzyOLVT33GjqyR\\\\\\\/7s0lQQKDXUDeZqr2AGduFwmki+f4bPPXDi2zmhV0mEriwHAVgNzda9AwXFhHw0vF3L3cqE0R1FWYvxF22ucu\\\\\\\/MQiozyXEg8YVu+MZXL+7Ef8H3YtrawokKk+U4jXpgj+qsgARZhqxrYgs0GUIofQrUNdoduyQs8lWjjupeWqLeJj2A\\\\\\\/enslG8HpyO4Rg3X8BnVRlwRsdJmK7gWzNJGPTSJkkssHH3HoQ9o1LVW6rFgWHlKfJW+ny14nmudP6eDgHM7YPHDH4G5HKy2+vkrZut+bjeP1yiORLZXJKJtVRoztEpL3tD8dahwgie9fFY0unc8mSuQGT3Vx4V\\\\\\\/JdsWpZoaeqNtgFQrMpQv70O2cWhf1x3IWuAnDlmhqRPmtUgGRXiKEVPxBqhfhc8OKiPXhFF35wE8x4\\\\\\\/WDbsQRTec2ysxcfMUoFUaZ8oYy+3hJEYZ\\\\\\\/WdmVsvYcrbOc5BKogxcTYoJYIGxns6RtyfdyacHbZJG4lTL\\\\\\\/5\\\\\\\/gf3L4jQC6G1V6BdtqsYmfYHWQweLJmKRBqgVB3R5woRgSWqspuTfsYFdEJmy\\\\\\\/HHslVYrVAghhzPO0dseY66ep\\\\\\\/duZQFaO25p82vfVDWjriuG+60wFN8mX1YijSPSHrYJWxqR\\\\\\\/XaAT5ycZvE62mY0YTj9viksG\\\\\\\/XlwZfx5uYj9HZZmFhzZreaJXJLLsKldap\\\\\\\/kxQSBP1tfiy1Eg+eDJeaPY\\\\\\\/TuDcowgjfrALZ\\\\\\\/nOcGL6lRs2+ONqE\\\\\\\/Nph+6nJvms1y6hTUhvmwt9vGO+zf4l65IR6dw4X2kff5dJJXSb4X4QLTshG0JVePdwZ6da3mPvteAu4fkfPWb4P\\\\\\\/YPSDow\\\\\\\/yUqVePZyq7FYs5RDlkY1oQ5mKl65VtZ4AdoIT6uM6wqMsVqe1t8q\\\\\\\/99vH8tCFpW29ZF+HsETetYedhGKbFkP2i6zD2M7mI8eya6sp8O5pOkHX5dJAjxtWVx\\\\\\\/Lwg4tu9hPrUzSIuzrZBRUizfsjySvy5D\\\\\\\/EIj\\\\\\\/D43YyGRijI0gsXUUgPhGf3prZqowooSNK\\\\\\\/VXzLWWwTfYim7nMel53qbKs2doUWavokctuLURa+2xp+\\\\\\\/1vqvcJ3bAgZ0tWgKUe4nV1o+mM7VaZa9AlxtDJ+pdYblsdrWmX714cQ0rLdHIQfonsxOrNFeKtL8Het0LY0v7rwM0tk\\\\\\\/Qn0uumWfbDIbO+qvL0z73geGN+84TA091IHyCJAN0ds7nS498TErHMkAdGxKoKaZ9y87eV1OCsK5HXXrl3\\\\\\\/udPdzSbApaekae+eygn8IDOfzGfOqUYwriLrH16m8e+98txAqYnwhoCfepzfjB+ZjzzKzMIR5deGCpLdcO0BlnAw74Q7Gpacde9cQ96HKbIB7OFwvgBGQlKDEIO09LC30bryW8LaA+E4TinYLz6D1gbrchUeD+JG3vQdzcp17VXrsneXt+5EZSjzTIuD7393DnaTcDZ38atsNo3PFj9+ot7K2DGa6aJdW80TXteAF7KER0ZLeuNT7Zrsy7x55vpWdujkf5yw8m6lySAPXjvo3HuFaJjmbUY9aFWb+V9\\\\\\\/+VvIBFecoZzN2YlJhIDQeiIaji9RcYYXD2WTxVkPqss+4sT4m18SwbR4fsCpYQXO6O47585jJNozdxBs00uRHSdlz5rvfAciNY712VHxQkWrntmF0QPD6YlVY5oWKeWzrTq3aaN24WN+spdGcktNq+iaWC+g9d0HOYRiBqQLDzrKgeuLLgvL7pFbSfdQ+VrdAHleBuLk9xmjR9t+y95ZCaayRAN3xG+ISXJtN0SPO0L4jMwHFjU553XNTl4uOPDvbKvQixw1Unh8ycDWvAbSreIH+510edjyCqn\\\\\\\/0HweA41uwUrB\\\\\\\/LrA5Mq9WNjW7He+GfpEMMojgJdU18T9gGUt3YQ4jsfu\\\\\\\/H\\\\\\\/W7VR58jqGTGGv8HymLHK2p\\\\\\\/bfvjneXcE\\\\\\\/EMDs860XMtxkw3IIxyMi6N59zVCTBGeXmOQR+SOCsoGnx37YkYf+uqjfDZvpM2mL0vasqlf5rDhoem0M6pfUKwtfGa6Klt6SqycA9nxTtlJ+yrkyVmgWcxZcQnnNAYkZIbBP38nItzAcL18stP\\\\\\\/zvjMOgd\\\\\\\/qKud09oqJYMZAzcY3\\\\\\\/hLx7UuFjjQhFWhIzPJlESOtYn8y7xHfG91ZX2icOh15XoihCbhrAhzVu1DS7Z4eBksNKlKIGdQT63ILMKfuw3X4e2QgHYopUf\\\\\\\/pnfk+z5oUkKv6dNtLXvaiK1YMU86tK8w+UQcTDqOZA+krothe0SpzjeUH7JzPrffaID+\\\\\\\/8B+wiEieukJZb5pCqzue1UQXmiqHidlhZGRr62v6aWPTb6yLNWJoBuVWlE54RudWupOcjv5D+s99K3gyHOBtfNq8nMM0AVZgFuDcQCld80yrBz3LF8hk6oDUF5J7DNxiQKtRQQwMwoVGZDl7n21g9gSye58EQAQQ6q2JG27RRPhb66aTU5qGPYG6aG0f5N28QKZhAjwCeGSKWfvZ4UNmg3YVJTBwpxHie6Lr3AS9qEoUbT\\\\\\\/c1fgyRqSlTIzzw0RRl+2NrRCmAtXVOq53TAtj+giYvehgm+t1TWoxrydKsq7z902hNBg\\\\\\\/VSaqIt1iP0WhS7T9NJGptiPTaDCd6HKjjUKCGjsSOA7p6hqgSQx4T4Qbkr1UjW\\\\\\\/VSOLePCuTr4vEpUKqrZmDLgfMW9MDSDsVE1cjJldcAwQTkGJ2VrZOkjfNaXRhGP1b0wUtwBhxsXPpr40mULV72J0UTu+RPxKTN6TsdZcDovih20iGz7iSlUANtZKBwxQCk66uRUCthU8Xt\\\\\\\/sY\\\\\\\/Hh7KbkBMXMnphdqx\\\\\\\/uCgBTfSQMu9khwgOGUCsyjpnsE7QEfm\\\\\\\/7hiu6HBlLVkueAWrPOolxsggMZ5Uuj82pwrjxb0nrZImUHZ3TMiwQ9aXvyn3DUzZO1Ojm1dAqFLvfMo3K1O7q4SbjfUYaIbO64DwfCBkI1MT1RpO6q7krcdwBR8eKKjMtcGuqzutbIKARad2c1FibIKOUvbg2I3O892j8jg2If7PWvInjhUDz\\\\\\\/l13DEbq0okmyJNZDbJfJQD2D+EyhHAPhICY409qPPTrlahZRWrJ6CyyMFOFBPu+ni7EfUwHpigI0eBQNF1Q9ioxqHlioYecV1hd1XWO6+ClXv7tfFn1jLHpzrXWkk+B8U\\\\\\\/5TZ4cIkaPliZ0x753280GRXXMXZoeIt9YECAoTvbR8X9u4g4t\\\\\\\/8dyBZhAxDPuPn3m1yEXVf8HtjtcUw3GaTrUWrRhpUDl8YZK9jKNxvsHU+7H3kO\\\\\\\/9kTzqiYMfAoOu7iaMfzVSraxVKCBbEZDbd7+a4zNSDBOKTpBe+LyFNl\\\\\\\/NqjE\\\\\\\/\\\\\\\/U4dHOIotRFrvJ32bnx6dyUzDxjasBAoq029Yzv70Rp0APYG3okCf04SznpJs4GYLdHfW5AzE8+7fr4NzeRrGPWtSzu36JGavfbOJLcyixYwc4ncHBsurWreaiyzWhYU33SjFXCg3nowT9r5ENVXgwFFJszmIWcgGAdCza5MXkiHVlS56tEfe1XlJLj9qEZWBajQfQ5\\\\\\\/3nM6vwxZfAMkmBl1elVJawmtgmGMyYw+CuL4k8NVzUhQ5qiRwBI71bb8iI8Wof22oZVSUtQEnkv3pb7aH3ovehR\\\\\\\/F9o6xpvytDj9eLs4rFrqmZ+l7wMoQJc1HxBCRNzxZF3+CngfyVQAZEbWRoVMwNk3M4lw6RalR+WpmmJTMQ02\\\\\\\/wghU6k1FJYQIZtxE4AMv3lPhawYgaWRA0DxcvPbD2u1PqEFKnhBJI8pNTJtFjxvB634Avh+hgKAzUuUpenc7xr\\\\\\\/tG61oNxQyU2j\\\\\\\/yY2vzkD80LBvIpOmmSWWXX2Rwbf4cbDi+YccRZ0PyPyVfiX\\\\\\\/dANrTBxjC0DJAvH2nmLJVtbMThyJClpfpKn7pGoOn8XAUAdsDOVKVhr8K7PSj4r9pWE21jmsG0WnmrXgShc+jEBsoE2sFTUH\\\\\\\/BysabfFTlJ6xeNrgt8VhOxpftCSEbrlMqlP1wDjP9f5SZTcHaQwMyeJjRTTQMssi71TUseU66dtM4XDVlHa4Wl8dDfv19DyyE0d64y4ZUG7qUF4Gyws1LrklPuD\\\\\\\/nlxQLmlKNuiBtHyqp9t1W3XndrPr3+ineYOZPTV8Qz5HhifooSmsyErC5QKjg9GHv+K2j8RfO+o8FJyTwFZUyYYmahcJU5YkhaWr0rn7GUq\\\\\\\/1j+tD5nlqBnjrqVfwStIx3IxNceA\\\\\\\/\\\\\\\/tW6Dc89jGMHyOwYsobDpJFZhElerxcfpa0b+1WWzYPQ4ewDtYR6piKxgvU8wQGQsv\\\\\\\/e2qfJipPlHlc1vtx240t3bBySbTzAgfz2C7Ob+pPfcvLov\\\\\\\/eCvJ+kmdnHef29QvG1TB6fB3wJtOk+52q9ECqCOSQZ4xRrAd8kyK\\\\\\\/Z41MSKOkOweER+R2gI9En7wsM2vWub9Dr5kydpfN71lVtZxm6JQTn3+tGxMmpQ+s9hDldbf7jIie9HNF1KlXp7TG6VC573GYwr74sQ6r6KFf9HgZeKPaq15AvyMp+\\\\\\\/lKCe1UiWj9AKk5N5MJSn5sOk28WUVDb13+jpzlXS7ZIdNdZRrO\\\\\\\/bTcRmUxXOK3CyqFadWbnw1B8iMkbMGhpYzqsM\\\\\\\/fad6SHAeOl\\\\\\\/mH\\\\\\\/++kDesSNbXLtDRSOv9EfOJUZc1LhHoVJZGFoc2GBKypB8A018ycCqqWyF7SVnioP7UEwTlWAxH9A6WRZxn9L8OjAHa7qY\\\\\\\/Ljb5b2+NbBWZfKbwynCYG6uTRPE92QJ9P6tMlZSm\\\\\\\/K005zPPJrPl5r+BDpJRI6RwjklzRPirDfFs4zCJEXXVc10p7Vd47p793OMLWF5OIMpT9uIrRnlsvgtA08Ur57qLoP3F+SoOwalE0bvYeWRvzJ3AFgPBhXRhU84\\\\\\\/mJFl4qCX3fVJfLNvSvA5+kXOtwqn+l8fpFBleJ97iOHeIy948fPQfI6a0WUECJZAowIqryZytmbLy68ogH7OKIOQ98JKJJUC+ibxTsyEsV64Q0rJI8inA+Fp1llrCJgJuEVKjRHTdpzeEON+A9hjjf5tUqmzMZG9ATscFYqMH+so9HUN5CnJChTD3ZrXEEM1Gl55f2Y6lux1w5mQtaR8cjzCtokhBHywZt8bkXk6aK+XaHmw9L0A7FSi5H\\\\\\\/Ggj0pa3ZDsdh5hjcyTHQqFqcFAyx9SdyW+7AFm5No+wDQC15mrZl0JvCgnyhautKBNcY4pxeeEDxZ39l0MPPcP2MSJAn4Pc+p\\\\\\\/LUaim498caKyelx0nE4or9JgrrpxUkXkfsVVLZnNk\\\\\\\/bIf0AsThL9kEUvFyszEZYX5Yc4Snh\\\\\\\/sy4Vs1tm3FsycUnBQhAbpIWADgqM4mxV5PRFn4Nmte1JL99GrpeKrHKg5DXXtecZ\\\\\\\/qODsY6aIwwCmh2dsaZov1DUBScfpH41\\\\\\\/VarNcqZyjLKRCxbX2ZVsXJPQ7tv9qL3SJhfshyOQ0iIyuM0OuEl7O2r2i5oVrOTRQqz7qRX3tYEuBh64SG36qNU1lQIu6jGzqm3Fy6MGJKWqXXr75tj6Q0Uth\\\\\\\/TFMuxYAjfjaB3GG3RHRGhtpde40jLKRpICBXNAksC5U\\\\\\\/SaoM4lEFOtyc4swY1QDXyCKErDDsj8gj+AdCxgG4gtoptNCcPj6\\\\\\\/YKevvInrctEFcVLAB8lN7u0C6CvB8KD4b+TH5ysMl3ZvvDwGay3hX2\\\\\\\/Z4LkXuNh0lOMQ7WO0IrUxqRAq+YWW5\\\\\\\/WmTROC1vLuBEzMFo9hwOV3u8JDCWtgIG8+edXncphVp2JLSUD1WrOZ4BMbXbaUHwTJIC0HHgcFJbeP2lG0h0eUTryIzMp2dkxNxQA1dDciXJFHW0FhJZQG2Rnr7WWMj0g6LMhKjzexWSF0YoxlAY5276pvfVP05vgvviM3aurdh8R75QFcWGHRMFuL2rVqLvYNZzedJaA3eTPcJErliXUVX1SKrSo9PqgpoR8hlfIlmhBEzqXMhNYsDhbwgr+KtdfkhTL5lzg46cI9igOvMy6qiY5AZkjaDexJK7qEHxVzi+l16UB+lfuuzXzNSWJvAz3jWPT7PWUIUs+9WV7h+7ma3ykNC95bl1Cr7NTJppvlBh7Ju9Nn0znyowk7YGvYvCKB+ljYccCaJ+wcnPwD2U4LtJi3c0ZmyyOvf6KLAMfXzN9zo88s5XR+oeRNAlQwuRGwe4tcBZtKKVoaydwQaRVmTHlU8iwRl3AFxcOVUTZw4v01wMzyKkJgtZ949z\\\\\\\/xPxMgVo5c\\\\\\\/BbhiUnK8N5jefccjucFA9wnLg01lOu7bRsOuPWTFpReGpYE2o89iJNYd2RM3bSSf87eYJW6Lty0nWn6RYbXRcExvlbYQqADXcQ64xPbFrTqTjwMzP3UrPh8gp0vglrLAHQYuIs6N9ByR4iG+BR3OmseMsmRdwH9incPSj1SCEMxbwNhqpftokmzKqMhSas0T6vm3SF6485Sq5MubG8+rfWeO87ymf6szVJBeH3EHJgb1P7E7B5zEExjIG3ArZKLPuDfndaiG54B0otsrRgPLoU8MbHvRDIus0+yUFD4iVxltJSUM7mTumRd\\\\\\\/Dq2XhYIegcLaRqK466m9DvLRW9iPUnpJQ0xSwzbRsSShrr4PPELWVMucoHWq457fblvfN\\\\\\\/waYDJpPSSCBpS6K3n0d0g6mCLMZ+rb9hh+abOZJ5Ha\\\\\\\/CrwrFEGLcLf1Aq5QHef1GX2bUBcD4jGS1gsEd06pelaIbDMfWwEcCMDI8D58cvYP+\\\\\\\/csD0+oQAXjhtuu9mcMQD1MzYFtlnwDsUicuTnx7p3Wpgzu8nt0SobfHg14BIFZKdDOuxAWOUfA3hjv1jaO1J7gFj4rzRICSKdyVKqL1lKRTHGPh1Bmc2oc5ERpYmTJyE2aAssACY1xj\\\\\\\/9xBWeGpS1J0dfp06LX5og99paKSRqwwZlc5CzCivb1nSzHeoDgDJdN9Iedu3NBXdvEm06837iKiaphIQP6kbrg10LAZYcXSj8Xh6nkbcjmW4DM8ZDtkDxABjwYXWvTUfYcjOuQeAzIlIXj8KdftsogvOWANngxazg6SbexLJwRU+bsCdwp7BY8HcxKrfMNH5yNI0EVr5ORpQ1DRIz3ziFpCid0lbQ6S11DAfzHgx3\\\\\\\/3pD6TFF08Z0mf9SvTKuJZDAsyDBWtWRewfJGpaF4YzQkFSQTowfRFQoDChjyeIDjkaBlgCubKIE0RD2IGEVP76oXgKSWj5n0\\\\\\\/mE11U5Y4tQV+Rap6wQ3QTcgKiVvQHF4icH4CIEjMMQhO5enxo1+zIfYFvDeJC1AQUxaAs8kogzww6QkEhOWuJPHgzhpyO9YZXXEtZi112IITs6JDrgr08vhT2M4WfBzN8Pw9JCDPjEMcbNgXFfrlUalulBFh4PjZaaF5Hb8EpG60NKDSOraiJg6rc5cE+ysuv4lUI6eNCiGvEjabtUSupj17G2gwph7gIE2wtbPHT7TPJLOXGxqDm90ZLqVH3Yxv240Vz36Lee\\\\\\\/ZG1gHQZpG6mz6VTYVrC0m3CHPLblB1M5gRvlaXtMz9Ir6GX+ADgL+EujQFpL67SITq93aUwGJZPMYtH6nasmjI869QUR7zZeb0TuwureCaimurmdqou6dglszca1LPYnImYEPK9HDLDHPBRwOgywJ6\\\\\\\/QHhm2eGUkqv+j7tSEwiYAWeVevVWX6EE8zczgp44FOlCfKQNstBUJ7UN0fX4OAIP5YgTdCxvay8WnjjqjSw3NsbFexxwU2WEx6IsAp1Tk4nZS5zT7z7DgtONDqSuYgQrTkKrXAAO2qU7K+QXsFMJQwVu\\\\\\\/M34d9YDgxCxticdQLELHp3kLDma9IGgXOZcx6iMZm0JnIkRfBaqi+3p3dVbRwUxnhsAejLyzT6vDpFWifInHqspZwAGB9PGm2Z1UXkbmUaGMVOFPTzEZKcYhBGtWxQebnn0j7GqfK0oJCN6\\\\\\\/5N0j4KKgq2DwaH1+35Qv34VZ3f5DH7CGiPunO\\\\\\\/izw8z\\\\\\\/fX44j2pouPpDfWCbmPj4YgC2R2TJgi57KqKOhAKeKCtBUptVj2anTAOjXdHSPhaNh3Bkq9nNf2Lp8rX7po9ys9P9U5\\\\\\\/+z\\\\\\\/S7HdSROGewseQyvi5EnCWlGB7+6rjrGFPickDPhLkOHTXVXxs2UcQv06l13jDki6asQd2Q6JbCKzz2Ea7og0kJnipp65updidPzPoengaXSRz3eAo5H+zClfbMgo+MRnd9zBtWle+EMaDmNmENHcReIMNh0UFa6sqWfOj7jDHTLlN+H16mqIFpzegYpBsT4aa8K\\\\\\\/5CSw5uTie7XL5nvdudSR7u8Lcvf6LqjZf7jBR09k876JqEdW9fAjGIjhwx5BKtsh0gCDybRi6n+Rag1RIDB\\\\\\\/BiGH3boW2MyNWN+f4HBT056YzTPlvR5ByS0dygU6mHSZgeNga\\\\\\\/R5L\\\\\\\/0h2GSaLdbMjTSXAnQUnPL6HEPGsUANPvhUgUnZ9nlKyyLW9lxV+vN\\\\\\\/wj28hPuJT\\\\\\\/MklTpsgiXjENhaecpJNfCFZRp7A6BWtYxmrB0yA\\\\\\\/wHQt7eNcehbXafkpNFriiPeLXpWP0ISr21gAagwlb5B5aFpMNXYC6+rCz3flxtlT01nj6m3s48K1Xs8Y6Xuwl\\\\\\\/8ykfMQS+iJGTzoaHi7pHW9NFe1jpu2zWktxNHGEkZve\\\\\\\/AkVFxURhsbTfQZX2IbWNcoHeJ60s3e9pIQYZity8OU+01DDduus+95XD1sIy+wOjUnATlbLuyPpWzeOoltFwcFMWcaIiBnAzZcrdDgbpfXcTBqId57aNyZ973fv9kpX8zx1ydqvyd5ytLi+\\\\\\\/3\\\\\\\/Wufy2GnNRoOAb2lrd\\\\\\\/gp6MmcQZzjtGGWSpA8P\\\\\\\/kR0ws+GN57b6fAa\\\\\\\/ZijGeEQknZtz4LQO92aGMHDUldFWVVizoW+ukRXbOU+tEYdXO7y+rAw\\\\\\\/d7qZgBLQZmgP5cdFi8GeS6nZ7l7UfiXD7IDzQ6tLNkb8bIpRPuirCrkOV6htHipOw3be1hwhARW\\\\\\\/E5zON2KB0CLsIvGsJKXiq4kQOtDRaaUY+wjQ\\\\\\\/hbeTklqgwIxExhLh2\\\\\\\/RqEVEJyGpxtjAxNGvf\\\\\\\/uJ\\\\\\\/Ee9c+Q1y0D3fdKRqpN1IZV0CJm5kdrjqvuVkYQAiTwKcuS5BnpR+T\\\\\\\/n6586X+ud5Zw056L1nnpwxaGh6jTmcqAVtI4DzUNPfsuWsYpnOrXVp7px6EZDNCQEHh08Hr6fsxiS9cZr9NQhWk+Gl+AGIKkFDMoZ0NOHHeg6ANRKA32P\\\\\\\/IAJFJ12Ve50FjYD5Atamoferc5qeJDb9m1KNLLcdi\\\\\\\/M2gO3CxiWz9tkt4wnGID20Qf6gIUXF12uoeMPm5o34yr\\\\\\\/d4bH8KA0hn82fDm\\\\\\\/GEu4sPumEhb8ZPgGG\\\\\\\/UQnaMa74ON\\\\\\\/a2HwTB15IJqcnJaYcDGGyWsLEta8OhPGpUC4FcYjwlizqYSfc0UT2J64r2+BKX1BadFT7FNV2ZO8Ne6UaN2McT3HtGB\\\\\\\/U\\\\\\\/Vyv2pI9KpDUiuOWtOR\\\\\\\/84JTc8BbCZqh8V0KVlj3bPlJ9cT2qCWItQTVHMlu0iUiA2udwTOzeMHrL6RNbbp8d28+w6h0SzAYR4\\\\\\\/9bn\\\\\\\/ScWbC7Q7axsihGSjdBpOywKCJ6tWMatC20CjIq0LtKdEmkRBky22dyg3SYN9NsjA1gS\\\\\\\/Rd3y9e6JrpIhWaCcR0hyksoS+oc8UfweNxQ2lAFOrlcfzykae\\\\\\\/27OwupOwpqEP0r8KUuhqXxERpq8iJv3OVKMwzLqdP6SkSDYmz8NfsnwSsmiWTeHEYUcM76kWqbwy3zaP0i5L8Sk4rg2pIq96sejpbonTgBdnTNzgGwWVgZnQ6Ni40MOXAjaA8HVBBizPYB9fkMPDW0mYdK424Ls\\\\\\\/EIalousoPl8\\\\\\\/\\\\\\\/1vh0SKnDXJqdN6tqyRs3F0AAKYYtaJi+7zBPKtJmAvNjW8DYSQDBUSQ9yefbMyhUaYAJ+vA4BboIOABI2nlyCyUSf8d21186IYjAv9KwHdFv8ub1e\\\\\\\/2zffaHRfSBQEjGcvyLs+3TDqRMeEdv0d1UsIcvAEmu\\\\\\\/PpdxsWbZQB98V74X3oWwVJv4y+McFn8JItM2Pwe2gjY3I5XAqVvnmeiICcIys48F9hLjL2VYN9UU62aVo0tsRCpC00+vhdStOeBoZJAwftUwuYVC6nVv6W1FLws7AgoGsmEeFibUTRmvJbroTLXJD6YscTGbSKDyaQDKmsIqgVDuy3E2w9IQTlH70BMY928J3OELU8L1bZ+gRJvI58J+DML7W+5d98m\\\\\\\/hUYQ1bcMLRDCmRfpcFwfUwhrp2voJRaZN1kmlugu+gT69x4rpMLN3UXHPvSYhJrt\\\\\\\/4VY0LsMpV7If49T2Wh95gHnBe6mBeEnm4QKeVdOBuw2cRToE0Yrp\\\\\\\/sPG\\\\\\\/J\\\\\\\/RyBtTcQPcZSY0d1obF4WDZvOQxm+sqJzJNuA0v9ie\\\\\\\/8J9X85j9mPVblJSptAaZtPCQ1EsA1l\\\\\\\/Gb3e2vCLjLBpkmLARuF+gdiUKZTwb5ts4j9Ff3215Q2AzVZMN3EBdbJGRjYZP\\\\\\\/yFZuhysljeyXGxe2Xm6WVFvwsH3+H42vYfhO4Zm8SrckWxdtvr5Y6Z6ImxK20lzWWXc+nUIUZgnFFunbmyChc\\\\\\\/eNTLrlc+rT2WLKrS+LE+Ca+NOr3IxEkNMo+Kxzv+tH7YH\\\\\\\/alnAyks2Hv\\\\\\\/4AZ+FLaDttbHt3vTx+Xbx2Jvqeq+S3ZiULoczMD2yfAoKsdoYTjBtFqX6KHnyKxRj+xODPaNy8ZPZ7cvaPH0G+Zp8ftMUwypw89raeHFfAFeTvlIKTRFj6Ry99usZse4u8NSYYmPvL2oC1c9kzYbXZl1HmHrChFcMDjp8kmQb00dpK5CeIUy+sCEKKaUTlwlQ12T7wPvkyppocY5mlnEp\\\\\\\/AHOSEgI+59btngKNLLAgxuJFEUVMHJojUskWIka8uZ2LNfoBQWwMCJOF4XKSN91Q42phI0pjDt2JAbiPeQKSMOitCvxpXdrPWdtgv+H\\\\\\\/nu+3RFctH1LL\\\\\\\/uzwDfXOL0koH0SHrXVA\\\\\\\/lO+8fW8KqDYa74bCj\\\\\\\/YlOP6wHmJAp8YKxhTaYKL5oj+s+Wh2uQ3GbBkbz9JARXHs2C5gRKiRCypRHY1zFQDHAnjakXe9DMLw3kfDdUUawaPyviw3r9NVCVNkDSficOClCoe9oqsik24cr9LsU+FuKc7F9toNMTpjutxCqkObmVDHiQbCXXhn8PFGIBbwLQSZcJyTRz+ifgp+a1u9Aq\\\\\\\/Xd2xZmF+WYI7HDcBinazYOPFpSnqf9AEvz9DrGWaffOVTu9R0zFCXzTQDq9bQUvs+hilqcdS3aqDbCI\\\\\\\/ZxcCb8zzlYrUQ36h4vBle0jqvd2qRJibXNs0iDsYOR33xAH6FoXxOGIMuxmcPaKRqQ2dP6YKWu0K0qmhQVhOh+YVPI65OnWdY6JZ6sICDpWhrnAQ6VkrRa3axdQscdj95ajghQ154ccmZPdJ+Fampe6afIC6B0QyFyPLKJbY2czKxbtiZ+5W7HhzZDko4kzGiOZEH\\\\\\\/fnaSVlzIQl+\\\\\\\/EJby933zwmUsVJ6N9gwYTAZhtiDKVwbymYcLzx3wskAEw\\\\\\\/nuBWF55qTr7uauPu\\\\\\\/geGwa\\\\\\\/fe6D4+lbCufojhz8eyxlIZOzG7jTghfmVG1LmYPKpJeLM4+48VidplePFCA+dCbVWGYHlTVoCp7FqcZ2fgd0paldH\\\\\\\/OdCDUPq3YDctHPUBnYa6AF3tHj8X2vXcq2WAAtz8GY37cW6nmMEH+suFJ6ONbTK+oQHl\\\\\\\/nPTIt+68qGL6lp\\\\\\\/2apS+BUPvu1sapXo6tKzJXx22MneEdDMo1RplLbFQ85kVX3og2tqRPrfRRxoORPe9v6Q3slD8\\\\\\\/jhj+QiMW\\\\\\\/DRaJTQnoz3TmqLI2k7Prw3X7bh1VoMikeD+iN8uicT\\\\\\\/BEz6RdyLQ3vx\\\\\\\/DSTzUcpl90\\\\\\\/veyK1+WZigPkLdKTLqJhLurtwBl0Pm5AJWzH402VXF\\\\\\\/IieSixsMK5BeraoZ8ZUdFFnjxpyt1Jx5FeJXZgVl8Pe0NX6Bi0Bc51k3yXGFxS0ZQ5CBZHQ+1jxN\\\\\\\/C6slYCOjlJ5Kbwvkt\\\\\\\/RTYLptGRkjTZT9QdfkW95MFE2s1pEoUXkUCBxv1z5GNdbKFVMT0TsD7ctjsZNAysdyUxYGXDPRtp57Cee\\\\\\\/i29wCsWe1TTdD439IZC9OFblUro0EpdAuUFTGeqola8r3FqUg+5FyBkQzlHe8DG8nhh3QnbRZz\\\\\\\/GcsLtJvdaSFcPkeRwwBFaBGr2nnt5uFwiAYUFiOfpgudxZgRe8j6xCGIpTAcZa3anxxuEzckmmYv38WqQDKYLRXH6zvHZzRCXtGAi9Tsxhpo5q\\\\\\\/OuIbfELZTTh\\\\\\\/aM9R3q6R3KWAV1SGLIuCSWqVPkPk\\\\\\\/+IRMs9ZfkdxQzLE9jAZyaqSgd5X8Eglr64UtLrQg21y8EXNGta7vbJiA7GhqVxg5K5qd65Jo+rBfRdJosF8RuocrZCm6GPChYmXpz+U54gLqEyGhgQ2920AM8Sm67aPYXHLDny34Ggo1ok3ZUgG8XM0OanvsYqtjfFo+wccJKZhaGdAPkzI\\\\\\\/fnQgrIYvM4HAKO5ctLyhPWEGWWg8qwTqiP3nMqmAsjZ8AAc3jCgr9tmbwH8P7mxQmksg8F6sNLhGTtpijh0zSf0nqtRq2QR0F7WJVPOwjD4ssdqj\\\\\\\/QU4GvRoYdLGOL2hd4kftT5yqteD0\\\\\\\/wsBy3YR2vG7YP0LGZL7VfMGcIusokAwv7WhRmIUtNdFv9vp84wKPcNK8ng4laEdoCxdCmcJPx44asXLvgw2c8P1c69TibiO1vQ+NbwlmsSOngzty6VmjAZ3Gv7HFENxp54D+d4eMim2O5y5YY31LL+ekBinIvxCvHng+x9dIf9xanMlFbRnRec+lu8aMjPhuaxlEFAZLBbNWc3DdRhjZ5JIfyRa+FG3zqTeAS92N2QcwjAVtHGZazRvYq5cZQakKGNpc19L9w5997ljowWs4wmXybr4G2NuIvMAZPWCLUwsCpr6P\\\\\\\/jUfUt5nCzngQZaKkqez4QKNYAxlfOpx970ScwoZZ0qnrHMYQlRMGrF56GVR3pyJyvQfeCNO60S3d3UDos0Kr6eUh+OcLRedyQgORmfvblgc2sBMfb0SgDH2U6r974LiDsb4VEsPyRHxWdn8MS7Nen2uJzdTq4abeZDwFEuXu+pG1oohMxwF5xFuyQwj\\\\\\\/jxLl92cc31O8DrU8kON+CY2v04q1Bk1eo\\\\\\\/Eqc6IFW6kK5nY6yVyDb7OmWPLssx7hO95kVGqUljqsA8r8sZ1nicv9+vGxZaJrlDLPD+LZLEoBLjkxpdYBumZKDYHFB8nK\\\\\\\/FD0UitaVTywbS726ygIRV1SeazejEY2ZxV3bJZwQxIWSO9juH+0haJ4wg6yaIZjEPin0Cd3k7MWuMtjZMuLOglj8RuXG5M1bpEFQVsyicIn0GjON5\\\\\\\/4WbwSBQ93Di\\\\\\\/B3nWvkYuqZaBD0GeR\\\\\\\/\\\\\\\/9bCe6k\\\\\\\/LPeL8wTyuzCIs2XqIO50HUzU+0N3c77o77r9MOOLw98MlAqNWhVQrzZQ1hAgOG1HkHRzTxT0LOpdHf2luKWhguMFjexNt9aPcxftu2ko\\\\\\\/dxeAcamamoTVZ3GojYSsBxWeWJr3jKI3IFtm\\\\\\\/fa7qPj5cfUv+H6pBplsE4\\\\\\\/8d7Ao6G0ejYZEoMLdVLR9gfDEFJbEaLS4blTS7\\\\\\\/66Vw9UARWpfn8dxLHLM\\\\\\\/lOq0o1LqWq43IW38sUrG6LGBGfXz3\\\\\\\/6r5BJ5PX8NKRK8rwZsJu0n0TAX1\\\\\\\/Bt+DildA3QKOkljBwVXTFYM6cCQX9y9nZ4KWeSO\\\\\\\/rP6sE\\\\\\\/PfY5oVwRmb75ceuaaOG54DRxtq+8QxjFCaySsKiEvF9aXgc9LZDZy8LpVPm1LYme1uk8TAD1RV0x9QdgjzYlSUPKvPCZWSx4p820Nid\\\\\\\/sLvmn5zrYy+Y2+BnoLIOlPT6Y0PndJtBFfj9hTVRe47BokQCf5kqFCXQEpohsMkSXEmN4oGF+UQboH\\\\\\\/v39fn9q1FtWII5SbDPL2qEbyJ7lXXjZgp+BrnMwaXKvFPgCvARgZIQrVuDVf7xfQc8IxHL1ttDZH1VeXzyDngoWYFP8PcBHmJlL9TUA18t80Tuj\\\\\\\/rXuHQJ+P4iGh\\\\\\\/V\\\\\\\/zemF33ZxgYWItVgg55UKGNHtapDCcLTmtjwN4jfOGQDj9+PjagXxxZiigFTP\\\\\\\/fQENfaYaE68ZeW6tBvos1QF8kZxLzS6+kO1vvGnNThAE54FixNsWVLT080imUmag0oqNi9X9VAob6Py\\\\\\\/EXKJ2PWGkM8gQi2JMAylP2+Zhso5A4jFW8k8P8MR2HzxrOGEIpjb4sHK1SH+RmwwL9JrOoPNmTbEha\\\\\\\/rT\\\\\\\/7hjOeOvYyoKxosowvPrSdsYNpdW+rGN288Uv9rP3c3067ujbv8xOoTMARsQVw9i1baBvY6vAT\\\\\\\/26z07MVklrPuwx6XBVi8JD3Re6ZtRlzDbYOX9Pv6dGzlaRaKKmNnAEGfI4jFPZN85zetFYP0Kh57tBwPq1Tf8Gtl\\\\\\\/9ByyVzqwCP2YXI2ZY9uRkhQqVIs+XwHFnxn2tSnvYT5xpmIH32AJmlHneuIX3z1zV5lr7UwHj2jC2FeK4lmmrSiYAo3o\\\\\\\/zaSvSBUWfjnXyot+Bh0rmlru7q8lSb8fyIIY+2gQXtifgE1S3LI0vTzRpLZBGjhWNeunnJWS55aLxRZtEni\\\\\\\/VYeAzWXRDbUSAsSMqSTRMMOV0SiSG\\\\\\\/HpTnZWnvV6iYScPKnc8ue5snGbbELKGBWju4WE4VtiOERVC2+oTvpLHXcs0rU7+OAWF+L9aEUhXYe\\\\\\\/axodUD7A9GNJl34YK\\\\\\\/6m2xAoksqU\\\\\\\/PSPBTcEBl7Hgjz9\\\\\\\/3urBRYjbVNGr3A8kuPGXBTtzsapL63B9g2DnIe7I1IKNtbQWqYFZH7sPQrC5JGvuBvYdLPmsrLpLd8VaAiGy+t\\\\\\\/xDoQRckmRd3lIFfgL42wNZGt3\\\\\\\/yXGr2FuObvs7PBUEPqL0WGQv0RQyD8EJDQS+DT0QXycV4Ef9Vd10nbTYQ3OLw6yqgNUq4v\\\\\\\/pADTiueQ9mfEaj1MqY\\\\\\\/AfiEEj6b2qAH8TC\\\\\\\/KaOHUlQLRBLtoxzNwJlPqml5yI+XvXdKtjfT9JAoSQrwtfSTV7WuBRGofVj4zFqoWD5KAappcd0ooZxrfYhRMpbzodpCTKmPODP+\\\\\\\/3GLat+QRSTRdEXSdJ6tmUS460GSmQZxPsPKmTweSWS9inwttxS2xpJEgiIsy4bkEKCe3sGFy0eGKlsS0BGdAf+TIRhbLN\\\\\\\/IdTC\\\\\\\/bbGoqrwtFteordRwZXVFhzY7WUTI1k3fObDb7lYzHXhdwvk5NU+a0BYNVez+JTMPta63AGZii9eRtGs7ZT7yoy36df9RFYWqrY7LdYZLxDJv8cBPLUVmrOwmOC6IDGmXgsBpAfk+fJq9J7VKIJ1lCANdBntdwnuNSf4+KWEPJ3iGFXQ9q\\\\\\\/uhnN6K8C2TqU8y0\\\\\\\/pYp9cHSrV82C2I\\\\\\\/i9c0m5hcM17Mfp1pzkvMn6SSwVWp1Uor7V0unzm4AX9y7CzVWhnsYhqbA6ANb9pAiy\\\\\\\/gG5P3xejTKFlnkbfVXubXj0a3sBVkGfmGXelBXZfELRE5RiuEHx4TT9V6Ky7aaoXCvIOsEwQkkSqCfUNcqx6W4YYZT1QDD6tUg2xT1CNxJE8cnpuZ7tlZ2sSH9Rh6diSz\\\\\\\/Q4D5y1dRYGOC+sXTYtbzHzYHXlSA3orxSlgrWzy7qD8zE3T3WB2M\\\\\\\/xCfCmKUmYqzEhLUQQ8sv76QOFGEs1u9aUEhIPJOPJw+XXtXi0WRzwN6UAvTyzsrcXRwsNdhPKD5HHHnJ4SAwTBwVMepcEfPwzhE3s2LI9J+rTLJYjRrpuTNJomaMtmi9smMnWKz4Lsa0QmerzvTZlB3TuY9YhZLMlCli2PFCntGMBSL1BLvvpfnvapoaNtZMs48FE5OnjH1cbQUeqnDHK3PI7WYBPtrQusuxsderklnHxXhIUWwzh70rDsIFg3AfPQfsCA9msGFTSigY+3IkzZrAw+rvrrQnKGTpGcmwQEYtHOtiVqPVS5OaUbdAGmKy5uiguBo4uQnxJm5BR3UATNxtKlJIuSs52u+CwKszuw72JeD6GKHB6L87EzFZ9bVkM9Cx62iQVge4B5Gi9CNNYX86ry+0IOH6SkrRRil5P9zGJuccBTVHmrPWrA5KgsxExQn38M0V\\\\\\\/fRyl8j8Ht5wtPRX3mLZww+HGghDqkvEOqoRHPJmdEcB4Ww\\\\\\\/Rl8xQhD02C3FcO90+sogxKUVQH4QgTk8TxmW4\\\\\\\/AmqyWF9mmL+4xywYbpBE7AKnrb08oXTfKVP\\\\\\\/sh0+5Tsc3BDMTj7xV5lTmKDh3ExpF\\\\\\\/TwuOcA\\\\\\\/is5hwCK\\\\\\\/vX7lDmWsb0DexOSkp3XxgDZofxCcV\\\\\\\/pZJ0tHsNiw5kz7WLq6IWgi02Q5gdLjLAP5CEV8O5jSr1E98It67pufJRrFiDshhtVYR26h2ietTbBa\\\\\\\/fNkUE+PemrtdTTugeCVxJdq+kV+QWoKDPLjxdXVao9AnENBN9DcLIAVncvrfJV2v76OAVd110KuqN\\\\\\\/HsrP9rwIWveOIQj5Uw6eHH\\\\\\\/mGabE5XlJgruAbbQM6VAXuBTPeGtDSH4uMu1Tn1X4Rm+up+gJVyEh9dHpIyl1PclQbI\\\\\\\/Ik8zvWX\\\\\\\/qOAFjjyRfms85g+utxfRsyWzAado9TizFFbdmD2yBv6aAJ4ccn37fRUoN5sb7ZyUxpobVtO9BXeOgIcRtsbSOqg62mYhj+pdDkPBrzcWx1H16vFQ+Ut9zKpARPulPMlOrlK7tvX1ySmvCeB2o+8wz34XhQtYzz1JAt1Z8SXygkiDToPN1dLRgS7pYECWE\\\\\\\/GPCnCD8J5U7JR38ow6rXqQYqePyxUVIRS4pptKmTRVctRbUraMtcv2HUXf8PmGNaBMymFCmUUmuUSEA2Zneyf8vQeAERTAK7lCgIN6DzfrCYWO0RIoWmp9AUG0rmznmj3Hfmmb0fdswUDsfxkbSw1cvqDmD\\\\\\\/onUgglehP9Ojnl9Vvq5umD7IvfNwekmwBN0dau86GmEQF1Ulou2SWGg\\\\\\\/iFUL5+fJLQz0ioKIlbLKnAncTXUJwqPTgggnEzNVF7BNjtml5Lv1H3uaLk5Vh5APWdBN7eyCLP6cDWhfNFQ9sccNJOVFM1SpWbXjY07OLX5FN263Pg2HZGgI6nAlbLKmsfvoXmNthhzz8HT67uuQRIsezH7EET1nyJVrcYvqd8w7bRy8VxGKS6HVP7GPEj6d0lSaIRu28l4afaJD1mwfgDIOmymz7u+SjfhaUSS4Eid5NarOUCJ+OHJ3jj3EDp9V2Xy7dfwOpikNzKLX3Ccli1QJBseM6Bz9+uhFdijKx1mt5hxUwVFhTe4H0D\\\\\\\/bskxpj7HNyt4kNuTEbvAqEfxkGCYi4XTn3voTY+CKpCG2HOphrotpChLZNSeoCaqkNbn2Rr28hev0vYfmbhqNVvHJh0yr\\\\\\\/7O9zZnNydmwdYdo\\\\\\\/w37ChTaAFp1nG+qBiogqWKeUDL9tTcwUOvBjBXJhGJ0\\\\\\\/59nYxtndWsgKOciunLGmKiLpjWetHxuHcG+nIr57s7h467SEBo9TVddxzCu9StpYbliQx6e8C2KUaaEQpaOWOmU4Y+80inoYKp9ewvBdQ15vJzvVZXzNGCVtaTMLO7kRl9MyDe8SuMOVayhu8B3VmNrHShAPP6\\\\\\\/b6FfQk\\\\\\\/cSNVIrcBe8NZBnQFMC8XY6QtabXqxOGyBuqF92UckiaHdvQw\\\\\\\/PzRuyWZXPMx3xtgmN\\\\\\\/NNGiqpHRYxTbC3G1BG09cROLeZYv9fhBOJpBCn5USr3y0smVQknXTWtmhQscsjqaHlruEbQ0SY9Shf8U\\\\\\\/gO4JmRYwNg10wEb4MWfaqL86pWL94ateNszcr5tkNuxC4d5l6BNdxyEgQjHpYBnu5D\\\\\\\/bsxlzLJpATrxkmo4mdnW6aDGbr5amJQWNI9yMtw0LY4Z2D3e\\\\\\\/+L+S1RqLnAdjcHn3wsi2urnea\\\\\\\/f0STInbO1nP7gq\\\\\\\/bk2NpZt\\\\\\\/y9Vjfv7PFuE+5KXc8J80LU2+9Xmks2mTnUp\\\\\\\/2g9TVonJoUSpY0m\\\\\\\/QIrHbe5ANIdHSrYXDZZnehCWnsoHUXzNtKlX4\\\\\\\/WWFFD8NiE5r0+1GzFmVY2ys\\\\\\\/msGm5jqnc5HZ87mHODaB9X7exN5YLshmowW8XIBOO5dQRnCgaUeunvh8z7q35Hq7WV0ClccmZ\\\\\\\/G1quLwAI0AvzQe25h2lZVwpquyr41HCzGzw74EqmcQjNhOCqHN7WsAOJIoTx0spFjjSsI\\\\\\\/AS2\\\\\\\/TMM9dNApm8ftpf3q\\\\\\\/4GQS3fDrJHJ7v1RCp6Zt0Kyz+cf+knXxF4dBykKsnvke75raD8aAX2IXqdiB0jnEJ8BhTCbmmtKH9IOCI0YeUIk9jiWL99IYWlpXMqOA8QpsMSLJGeB1UgNmo3iIsWa7dW6dQ45kVkmqHlM354+m7PGQ5SuO\\\\\\\/L2rh4HVsptmjq+e7WS9gTU7SIHQplFXd+tUV5iZO7MHcGhMIQ4KQW1P1jQ\\\\\\\/PEqMZjbbeTO\\\\\\\/rYc9nvSsnaznNiJsK+WNETNbHlrly8DQm3QLztWno5GZXKQ2TgKn6\\\\\\\/9tbsfJ2Pky03bK9uWwwArW\\\\\\\/XQQ3WdL0Y+jVG7kVJ+eomc2Vsq3VoQbhZXcO95iZoQV2jvT0\\\\\\\/3o2Xfi\\\\\\\/Z+Dx1DFaSXBtI3VNBc5SJZ\\\\\\\/kVy4YwqiO7soeAaeuRpcNK0iMlmhJCpgoOOqhRF6okt31UaJC1qvVMUFUnzQY2oNQwkjsNxYtskHCAL8qLaTCQ4rWeLepmagN6LEpVUNEULAYxo9TRnfRTFBGtCGcNCtr\\\\\\\/RDPrv1kOFtrmAD1YkSRyv8Xpg8T9Av9hNnyrSAHlsVSndatMJIGW6RDm10bzYiRLYq+IN+wdI66MWPzjugjWHvb\\\\\\\/0sJh9BISr6CBzoigcYYGBDMnAkPl7Hrp9rAh7mY6Z0wlxgCNoTc065BHAhYiwXQDi+HYTm5Xq2c2R2KJrwGAta1bs5ymrGOPOwe851RwhHDl9DriCi7IKeCDYJj2Fd7l11W8ySNe4K+G6HZUiQwesyaFUrPvR9adiKoX+XchxpCwbr2vZE4tKO\\\\\\\/H6JoFXFmZJzwonLe7kc7f1vmig4yPtuM5A6cOSabWn\\\\\\\/p72Lo\\\\\\\/5I6cEo8yhUMneuRwnVgrJR3\\\\\\\/aw9hTcS+6a0b2Ib0rew4GDofI2MnwcsyNk5qDmz+a3Oj\\\\\\\/PdV4IIBb091G\\\\\\\/DMGdDqTXI6mk2V\\\\\\\/CoBSTtgjzY+i5Z9Ap2U2xc0Fsc4r4gGOanVDmgMLfhCdygoaU6QI7tZQQ+odU\\\\\\\/NzG1u\\\\\\\/J\\\\\\\/DvJgRCZUoRcH2SSy\\\\\\\/0\\\\\\\/C3VGpsxvy4mfemzcMft4sReiauGkZSTQRk8rqEE9m35BLbxJJOGkRHj5sCeQmVK46B6tVItLr\\\\\\\/h0Mb6sINJve72muEw5dAvhs7236aMo0YoPHFMPbMsZUPJ64KcMoeCSBlId6bKvYqhOajzfNqDNVtzmsaiR1G+EXoPRS83PD6kYIl8qdz6Rb8RuOAxoOk+uBVgyUiRV6T6+4+XjXNwUQwby3vJJpvOvNQPpYWWn24FPqDTdIdzx72zt9BD+wKpnijIR6kdXFDqF3Me3PRWUHJAJpICkJ0wNa242aD4sbL7B1vQmzruLeM+Imb8SQDtd\\\\\\\/rF0vt\\\\\\\/FM4SgZnQuoUY2661PgS7vxuAPxLcw6u6Rwf5nz3mDZVlEsaGgbel5wFEd2GtV7TIv9\\\\\\\/XQ2pzJ2jU\\\\\\\/vNsp+3FwX6j7kmU4JWEh9XXT\\\\\\\/d1sbFxlumOny9xKv3fm9Vn1faPaE4sdjj+\\\\\\\/2PwdyIM5QjyoSU24YSkg+UJFeVDngS0FXEl\\\\\\\/08tqSWLbmxAveANW1e1CwTQ\\\\\\\/qlDpJ\\\\\\\/Exhm0wLAO+wCrJGEG0YGexOLD8dipDNYblYv5\\\\\\\/bRjUyr1CBi0T4O1GjdmVfrG9D7DHEK7Iwry6Ban3Gsme\\\\\\\/FWJ3P52PtVKhk\\\\\\\/tRDvgdW32JCJdmQUmNie5fcDDjzyEIjeCxSpKLKLRMOpNAB6jWy37RcJNwWkqA0nkALcneaF\\\\\\\/x8cZkulBZXHxh4InEaXAfA0L9JN26gbHPl3p25emfPenRfj\\\\\\\/lY4\\\\\\\/CzKMG2n3PnmXUiuFgm5yY31oCRcdYPIDy\\\\\\\/QAlTHQI8FoNGpfh1c7GMkfmhQMjp++7TzSmezIk1AxBP+P0WsDHx0EzroYvybjUeWZYB2SCDQmBZvKcOOWAwS1Ojii1pzDgupLdkSA\\\\\\\/ECl202fes30rFCO+5KDJaQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022jtqFq1Nx2khzmYzdN1T2WQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002286c7773e810d621badcb085d829fdb679769b828f5e61513ae05c2a21b38a4bd\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022N8Myl0Ra16yjIbKZcnx1CMtitMbeW2F86wv258gvJLQZrGBl9duy6qcPVYmSAj5DgaNOH5N3zFptjVpyPGNuo1necpSsVFeRXPv4jt93QSQVCyTLL7nyxBZR0mPw9wsr1eu4qTtCzqlF7Q9Sq6iLX1M5oLvYfErtnojAMDsh0EfE5\\\\\\\/xSxcCOqzZHeCh79bMuXrn1Q3LOoR+vjF7Ox1Xa9U8L4+DNl2opl7DsViufErJE\\\\\\\/zG74TO\\\\\\\/CSTcL+Y9MUcaJVy1hitfara0h2rFt\\\\\\\/6UneBEgs8kLlcE4JSY8dAhhMp\\\\\\\/wbbrxcijULEbOh27K9cTs\\\\\\\/jUkL0ngcWdPRGjObW4rTWlXYUHQB2+0rmleoKVNz6jCm1C0Ur7K3x5Z9ZtI62QZCdbPDUeX3c5LQbEuHbZO\\\\\\\/4wM4MQOXj7wiNUe0Fyz8gNkF+jJUiBPvtd5JUdYWwIn8Bbq10Jje4rJ+mlgPg4n\\\\\\\/3WYsNqqOvaCSC+Xy5tdfcou5ceHAUMN0TU1pgBhqF0oBM5FlAxpVGB3uYfN8Sjju+VlAnpebBNTIeq2oORggbfv54dQ2LJgno2HhijEnYdOlG00vWJcemGguKuC9PsA\\\\\\\/X\\\\\\\/aaSg66ucqqUGXW\\\\\\\/WdnDLQsMsRUJew03xTDfWKbKozzl86peSm6sKIF8fKwxHAyP+XSiAGZBJjAm72Mq2Fy6ulBmHO14PenaYP0EyZkbBI9KkxeKhWzCuvkTigv8Wga\\\\\\\/ppaLPRccvY15Cj+P8AxJaoCPS3SfJZ20gsrZISr\\\\\\\/qmZSE7aYzmptiDJgP4nacbKWrg5bbkqIrVW3pErlVG+yonhd5+YMBCnhjZT+O4pQnqteN7WI+OB4iWoen7cIuAPfx\\\\\\\/Hlgd06EnffEb0ujivfEZmU1+fqMSYyBT4kCgSmF5lasQHsn9pnmYP0S8wt5Xf+bNteQR6HJEzjchPoytVQAyiOQZucjluGGaJLn9EyXPddKRWHtfVtkQTigNUS4nHb+FsQ8kyK3TpGNEH4a5Ussaly7skg9xl74FrMtqDvPRa3AHe0KgLWj7n0II91pihea6cMrAyrBTYBmarLD51olMUA1RfuzIhx9l+sH4GJuS+A5PjfIaTiAfdXlQzcm6UzjnUFGqO5QSSUGyiKTrQoYGX4Ioa7qz8FQHLN3PTVj+nqVMXaOqTI\\\\\\\/BljWHZfQItPL9OrAmQJJ5DMF+LMKu7C8GVYWUnPxrdr9Zm+Av22Lo08LUxCuJ8NnpOt0wQ\\\\\\\/zyesIeqAjQGPQtpD8T0snZ\\\\\\\/mbiHusnn+miJ86czjCiuhqpoDV\\\\\\\/PT596oOED2GqmzZbsg0I\\\\\\\/IUY3ZpuXOYt5Id2YlXZvUckdCmgdollJwNXbL9HAEoiV821w02rLrfqfhWs2ctV0jR0Os1xOXkt9koWIY491EMlNE89t6t9IXB0JisxHDKUT831Rs32c1DZ0WqfjkBXrttXBpe3gXSirNM3pRjmyVLBWM6XbWo9uv5xy8xPQh\\\\\\\/4gCEp9RKUbvZknidm1zV6YUCMLom\\\\\\\/gALgpRWKG\\\\\\\/kikzhJznciGPYNQPUBapzp1Kd+lMcN3vo01ScWmm1fHQBjlRTt6oBMxOwCggB++h9SfKNiQhSuX2\\\\\\\/DsgA6Pgd+YtbdVpoAv37o3iaapq1oyXjVGZRdfGDq4+QBPgPAQnTJbu7ZPc2FXxgxW2OYgZOxVHfC9UUmv1XdhheWVD7qM1p+f1APntFUIYl\\\\\\\/p\\\\\\\/9Dyzs\\\\\\\/S4ks32aR9\\\\\\\/gLa6ZyOEDq+JF+vCxpZCoufDIlSxHWAC0RsQyD30vUpQ++WOwwjKDbMHlnTlzmYhYP6Add4VxOChPCjNcsUavzOa2uSC\\\\\\\/Sf0j7srUHiQWcxIj5XmxN71MK8F88dMKevHoJCysT02JKv+NhklUGbVEKydpeFOH8SMrlqz9VgOiUghACbGSO1N2xYAZ6UmFWbNldO6De4UjNbyEZTzA0AVdy+KuJqoWdfJT4Lhzr8RO\\\\\\\/A8vKfqKhhXtHuqSM9DwNwJBjpBlce+BGs7\\\\\\\/YvKqE\\\\\\\/xTaCkybPpwiQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022SBNQZ+\\\\\\\/jlfnDliFs9QioXA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022111a226154e8b9e613174c535cf320f6e1dd69a82d739c16bacf0db7a51cd77d\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460084 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genomic-selection">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/3/589" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Yppmy811_Ysmzyo9h3Wug0EhMH6I460w7WFalO6JSJw" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/3/589" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Ud6KF9fIaO9cRBvqPc9HIvSOmIofpx-wWCBVXY4eFFQ" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/3/579" title="Genetic Architecture of Conspicuous Red Ornaments in Female Threespine Stickleback" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/3/599" title="Genetic Correlates of Individual Differences in Sleep Behavior of Free-Living Great Tits (Parus major)" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460084" id="node460084" data-pisa="ggg;6/3/589" data-pisa-master="ggg;g3.115.026328" data-apath="/ggg/6/3/589.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">FW: An R Package for Finlay–Wilkinson Regression that Incorporates Genomic/Pedigree Information and Covariance Structures Between Environments</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Lian Lian</span> and <span class="highwire-citation-author" data-delta="1">Gustavo de los Campos</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">589-597;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.026328 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460084" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Lian Lian </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Epidemiology and Biostatistics, Michigan State University, East Lansing, Michigan 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lian%2BLian%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lian%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALian%2BLian%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:lianl0501@gmail.com" class="" data-icon-position="" data-hide-link-title="0">lianl0501@gmail.com</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Gustavo de los Campos </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Epidemiology and Biostatistics, Michigan State University, East Lansing, Michigan 48824</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Probability and Statistics, Michigan State University, East Lansing, Michigan 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gustavo%2Bde%2Blos%2BCampos%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=de%20los%20Campos%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGustavo%2Bde%2Blos%2BCampos%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/6/3/589" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460084" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460084/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/3/589.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460084" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460084/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/3/589.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460084" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460084/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/3/589.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460084" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460084/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/6/3/589.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460084" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460084/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">The Finlay–Wilkinson regression (FW) is a popular method among plant breeders to describe genotype by environment interaction. The standard implementation is a two-step procedure that uses environment (sample) means as covariates in a within-line ordinary least squares (OLS) regression. This procedure can be suboptimal for at least four reasons: (1) in the first step environmental means are typically estimated without considering genetic-by-environment interactions, (2) in the second step uncertainty about the environmental means is ignored, (3) estimation is performed regarding lines and environment as fixed effects, and (4) the procedure does not incorporate genetic (either pedigree-derived or marker-derived) relationships. Su <em>et al.</em> proposed to address these problems using a Bayesian method that allows simultaneous estimation of environmental and genotype parameters, and allows incorporation of pedigree information. In this article we: (1) extend the model presented by Su <em>et al.</em> to allow integration of genomic information [<em>e.g.</em>, single nucleotide polymorphism (SNP)] and covariance between environments, (2) present an R package (FW) that implements these methods, and (3) illustrate the use of the package using examples based on real data. The FW R package implements both the two-step OLS method and a full Bayesian approach for Finlay–Wilkinson regression with a very simple interface. Using a real wheat data set we demonstrate that the prediction accuracy of the Bayesian approach is consistently higher than the one achieved by the two-step OLS method.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/bayesian" class="hw-term hw-article-keyword hw-article-keyword-bayesian" rel="nofollow">Bayesian</a></li><li class="kwd"><a href="/keyword/finlay%E2%80%93wilkinson" class="hw-term hw-article-keyword hw-article-keyword-finlay&#x2013;wilkinson" rel="nofollow">Finlay–Wilkinson</a></li><li class="kwd"><a href="/keyword/genomicenvironment" class="hw-term hw-article-keyword hw-article-keyword-genomic-environment" rel="nofollow">genomic/environment</a></li><li class="kwd"><a href="/keyword/correlation" class="hw-term hw-article-keyword hw-article-keyword-correlation" rel="nofollow">correlation</a></li><li class="kwd"><a href="/keyword/genotype-environment-interaction-0" class="hw-term hw-article-keyword hw-article-keyword-genotype-by-environment-interaction" rel="nofollow">genotype by environment interaction</a></li><li class="kwd"><a href="/keyword/reaction-norm" class="hw-term hw-article-keyword hw-article-keyword-reaction-norm" rel="nofollow">reaction norm</a></li><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">GenPred</a></li><li class="kwd"><a href="/keyword/genomic-selection" class="hw-term hw-article-keyword hw-article-keyword-genomic-selection" rel="nofollow">genomic selection</a></li><li class="kwd"><a href="/keyword/shared-data-resource" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resource" rel="nofollow">shared data resource</a></li></ul><p id="p-3">Plant breeders use the Finlay–Wilkinson regression (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Finlay and Wilkinson 1963</a>) to assess stability of varieties across different environments. The FW aims at assessing how the expected performance of a genotype varies as a function of the environmental effects. Usually this is achieved by regressing the performance of each genotype on the environmental means. Compared with a completely unstructured genotype by environment interaction (G × E) model that fits every level of genotype and environment combination, the Finlay–Wilkinson regression is parsimonious and can reveal a trend of variety performance across environments. Breeders can use this model to select for plants either based on stability or on responsiveness to environment potential (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Walsh and Lynch 2014</a>).</p><p id="p-4">The standard implementation of Finlay–Wilkinson regression is a two-step procedure whereas in the first step environmental sample means are computed and in the second step intercepts and slopes of each line are estimated by regressing, within line, the performance of each line on the estimated environmental means. This procedure has at least four potential limitations: (1) in the first step environmental means are typically estimated without considering G × E; (2) in the second step, uncertainty about the environmental means is ignored; (3) the environmental means and the variety intercepts and slopes are regarded as fixed effects (this can lead to large sampling variance of estimates); and (4) the procedure does not offer a clear way of incorporating pedigree or molecular marker information when estimating the intercepts and slopes of the lines. These drawbacks can induce biases (especially in incomplete designs where a few lines are evaluated in each environment) and lead to large sampling variance of estimates.</p><p id="p-5"><a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Su <em>et al.</em> (2006)</a> proposed a Bayesian method that addresses the limitations of the standard two-step procedure. The methodology described by Su <em>et al.</em>: (1) uses a Gibbs sampler that allows estimating environmental and genotype parameters jointly; (2) fully accounts for confounding and uncertainty about environmental means; (3) treats environmental means and the intercepts and slopes of the lines as random – this treatment usually performs better than ordinary least squares in terms of mean-squared error and of prediction accuracy, especially when the number of parameters to be estimated is large relative to sample size (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Copas 1983</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Frank and Friedman 1993</a>); and (4) allows incorporating pedigree information into the model. Using simulations, <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Su <em>et al.</em> (2006)</a> reported better statistical performance of the Bayesian method for estimating model parameters. In this article we extend the model proposed by <a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Su <em>et al.</em> (2006)</a> in ways that allow incorporating genomic [<em>e.g.</em>, single nucleotide polymorphism (SNP)] information and covariance between the environment effects.</p><p id="p-6">To the best of our knowledge the methodology described by Su <em>et al.</em> for animal breeding applications has not been considered in plant breeding, and there is no publicly available user-friendly software for implementing a Bayesian Finlay-Wilkinson regression. Therefore, in this article we introduce an R package (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">R Development Core Team 2011</a>) that implements the Finlay–Wilkinson regression. The FW package implements both the two-step ordinary least squares (OLS) procedure and Bayesian single step procedure that allows incorporating covariance structure for varieties (<em>e.g.</em>, a pedigree or marker-derived kinship matrix) and environments. We describe the methods implemented in the package and show with examples how this package can be used to perform the Finlay–Wilkinson regression with both methods. Finally, we present an evaluation of prediction accuracy for the Bayesian and two-step OLS methods with a wheat data set.</p><div class="section other1" id="sec-1"><h2 class="">Model Specification and Algorithm</h2><p id="p-7">In a reaction norm model (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Gregorius and Namkoong 1986</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Perkins and Jinks 1968</a>) the phenotypic record of the <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-1.gif"/></noscript></span></span><sup>th</sup> replicate of the <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-2.gif"/></noscript></span></span><sup>th</sup> variety observed in the <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-3.gif"/></noscript></span></span><sup>th</sup> environment is modeled as follows:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-4.gif"/></noscript></span> <span class="disp-formula-label">[Equation 1]</span></span>where <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-5.gif"/></noscript></span></span> is the main effect of <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-6.gif"/></noscript></span></span><sup>th</sup> variety and <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-7.gif"/></noscript></span></span> is the main effect of the <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-8.gif"/></noscript></span></span><sup>th</sup> environment, and <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-9.gif"/></noscript></span></span> is an error term, usually assumed to be IID normal with mean zero and variance <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-10.gif"/></noscript></span></span>. When we reorganize Equation 1 into the form: <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-11.gif"/></noscript></span></span>, we can recognize that <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-12.gif"/></noscript></span></span>is the change of expected variety performance per unit change of the environment effect (<span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-13.gif"/></noscript></span></span>). If there are no replicates the index <em>k</em> can be removed. With this, the equation reduces to <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-14.gif"/></noscript></span></span>. The collection of parameters to be estimated from the model of Equation 1 includes the intercept and the vectors of effects: <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-15.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-16.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-17.gif"/></noscript></span></span>.</p><div id="sec-2" class="subsection"><h3>Estimation using two-steps methods</h3><p>The Finlay–Wilkinson regression requires regressing the observed phenotypes of the line on environment effects. In the standard Finlay–Wilkinson regression (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Finlay and Wilkinson 1963</a>) the environmental effects are computed from the sample environmental means. However, in incomplete designs the sample mean of an environment may not be an unbiased estimate of the true environment mean. Therefore, a better estimate of environment effects comes from a regression that accounts for both environment effects and genotype effects, that is:</p><ul class="list-simple " id="list-1"><li id="list-item-1"><p id="p-9"> <strong>Step 1</strong>– estimate the environmental effect using a main effects model:<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-18.gif"/></noscript></span> <span class="disp-formula-label">[Equation 2]</span></span></p></li></ul><p>The above regression yields estimates of environment effects (<span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-19.gif"/></noscript></span></span>); these can be used in the second step to estimate the intercepts and slopes of each line.</p><ul class="list-simple " id="list-2"><li id="list-item-2"><p id="p-10"> <strong>Step 2</strong>– replace <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-20.gif"/></noscript></span></span> with <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-21.gif"/></noscript></span></span> in Equation 1 yielding:<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-22.gif"/></noscript></span> <span class="disp-formula-label">[Equation 3]</span></span></p></li></ul><p>Both Equation 2 and Equation 3 can be implemented with either OLS or mixed models. The current FW package implemented both Step 1 and Step 2 with OLS. In Step 1, Equation 2 is fitted with the constraint that <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-23.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-24.gif"/></noscript></span></span>. Step 2 is implemented by fitting Equation 3 separately within each line with the constraint <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-25.gif"/></noscript></span></span>.</p></div><div id="sec-3" class="subsection"><h3>Bayesian approach</h3><p id="p-11">Bayesian inferences are based on the posterior distribution of unknown parameters given the data: <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-26.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-27.gif"/></noscript></span></span> represents the collection of the unknowns: <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-28.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-29.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-30.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-31.gif"/></noscript></span></span>,<span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-32.gif"/></noscript></span></span>,<span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-33.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-34.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-35.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-36.gif"/></noscript></span></span> is the conditional distribution of the data given the parameters, and <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-37.gif"/></noscript></span></span> is the joint prior distribution assigned to the model unknowns. According to Equation 1 and assuming IID normal residuals, we have:<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-38.gif"/></noscript></span>.</span>In the FW package, the prior density is assumed to have the following form: <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-39.gif"/></noscript></span></span>. The residual variance <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-40.gif"/></noscript></span></span> is assigned a scaled-inverse <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-41.gif"/></noscript></span></span> distribution: <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-42.gif"/></noscript></span></span>, with degrees of freedom <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-43.gif"/></noscript></span></span> (&gt;0) and scale parameter <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-44.gif"/></noscript></span></span> (&gt;0), in the parameterization used <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-45.gif"/></noscript></span></span>. The overall mean <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-46.gif"/></noscript></span></span> is assigned a flat prior. The prior distributions for <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-47.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-48.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-49.gif"/></noscript></span></span> are all multivariate normal: <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-50.gif"/></noscript></span></span>, <strong><span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-51.gif"/></noscript></span></span></strong>, <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-52.gif"/></noscript></span></span>, where <strong>H</strong> is a covariance structure describing covariances between the environment effects (this can be a covariance structure based on spatial information) and <strong>A</strong> is a covariance structure describing covariances between levels of the random effects <strong>g</strong> and <strong>b</strong> (<strong>A</strong> could be either a pedigree or marker-derived relationship matrix). Independence between the effects of the levels of any of the random effects can be obtained by setting either <strong>A</strong> or <strong>H</strong> to be an identity matrix. The variance components <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-53.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-54.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-55.gif"/></noscript></span></span> are assigned scaled-inverse-<span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-56.gif"/></noscript></span></span> distributions whose shape are controlled by variance-specific degree of freedom and scale hyper-parameters. The FW package offers users the possibility of specifying hyper-parameters (degree of freedom and scale parameters); however, if these are not specified, specific sets of rules similar to those described in <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Pérez <em>et al.</em> (2010</a>) are used to determine those parameters. Further details about this are given in Supporting Information, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.026328/-/DC1/FileS2.pdf">File S2</a>.</p><p id="p-12">In the model described above the posterior density does not have a closed form; however, estimates of features of the posterior distribution (<em>e.g.</em>, posterior means, posterior standard deviations, or credibility regions) can be derived using Monte Carlo methods. The FW package draws samples from the posterior distribution of the model using a Gibbs sampler (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Casella and George 1992</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Geman and Geman 1984</a>) similar to the one described in <a id="xref-ref-14-4" class="xref-bibr" href="#ref-14">Su <em>et al.</em> (2006</a>); details of the implementation of Gibbs sampler are provided in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.026328/-/DC1/FileS1.pdf">File S1</a>.</p></div></div><div class="section other2" id="sec-4"><h2 class="">Software</h2><p id="p-13">The FW package implements both a two-step OLS method and the Bayesian method described in the previous section. Typing the following command in R will install the package:</p><div class="boxed-text" id="boxed-text-1"><div id="T1" class="table pos-anchor"><div class="table-inline"><table frame="hsides" rules="groups" id="table-1"><tbody id="tbody-1"><tr id="tr-1"><td rowspan="1" colspan="1" id="td-1"><kbd>library(devtools)</kbd></td></tr><tr id="tr-2"><td rowspan="1" colspan="1" id="td-2"><kbd>install_github(“lian0090/FW”)</kbd></td></tr></tbody></table><div class="table-foot"></div></div></div></div><div id="sec-5" class="subsection"><h3>Wheat data set</h3><p id="p-14">The package includes a data set that can be used to run examples. The data set [originally made publicly available by <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Crossa <em>et al.</em> (2010</a>)] contains data for 599 wheat lines from CIMMYT’s Global Wheat Program evaluated for grain yield in four environments. The data set becomes available in the R environment by running the following R-code:</p><div class="boxed-text" id="boxed-text-2"><div id="T2" class="table pos-anchor"><div class="table-inline"><table frame="hsides" rules="groups" id="table-2"><tbody id="tbody-2"><tr id="tr-3"><td rowspan="1" colspan="1" id="td-3"><kbd>library(FW)</kbd></td></tr><tr id="tr-4"><td rowspan="1" colspan="1" id="td-4"><kbd>data(wheat)</kbd></td></tr></tbody></table><div class="table-foot"></div></div></div></div><p id="p-15">Function <kbd>library()</kbd> loads the package, and <kbd>data()</kbd> loads data sets included in the package into the environment. The above code loads the following objects into the environment: (1) <kbd>wheat.Y</kbd>, a data.frame (2396 × 3) containing the grain yield (average of two plot records, <kbd>$y</kbd>) of 599 wheat lines (<kbd>$VAR</kbd>) in four environments (<kbd>$ENV</kbd>) and (2) <kbd>wheat.G</kbd> (599 × 599) is a genomic relationship matrix computed from DArT markers. Further details about this data set can be found in <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Crossa <em>et al.</em> (2010)</a>.</p></div><div id="sec-6" class="subsection"><h3>User interface</h3><p id="p-16">All the arguments of the FW function have default values, except the response variable and the corresponding identifiers for varieties and environments. A basic call to the FW program is given in <a id="xref-boxed-text-1-1" class="xref-boxed-text" href="#boxed-text-1">Box 1</a>.</p><div class="boxed-text" id="boxed-text-3"><h3><kbd>Box 1 Basic call of the FW function</kbd></h3><div id="T3" class="table pos-anchor"><div class="table-inline"><table frame="hsides" rules="groups" id="table-3"><tbody id="tbody-3"><tr id="tr-5"><td rowspan="1" colspan="1" id="td-5"><kbd>1</kbd></td><td rowspan="1" colspan="1" id="td-6"><kbd>library(FW)</kbd></td></tr><tr id="tr-6"><td rowspan="1" colspan="1" id="td-7"><kbd>2</kbd></td><td rowspan="1" colspan="1" id="td-8"><kbd>data(wheat)</kbd></td></tr><tr id="tr-7"><td rowspan="1" colspan="1" id="td-9"><kbd>3</kbd></td><td rowspan="1" colspan="1" id="td-10"><kbd>attach(wheat.Y)</kbd></td></tr><tr id="tr-8"><td rowspan="1" colspan="1" id="td-11"><kbd>4</kbd></td><td rowspan="1" colspan="1" id="td-12"><kbd>lm1 = FW(y = y,VAR = VAR,ENV = ENV,method=”OLS”)</kbd></td></tr><tr id="tr-9"><td rowspan="1" colspan="1" id="td-13"><kbd>5</kbd></td><td rowspan="1" colspan="1" id="td-14"><kbd>lm2 = FW(y = y,VAR = VAR,ENV = ENV)</kbd></td></tr></tbody></table><div class="table-foot"></div></div></div></div><p id="p-17">When the call of the FW function is done using the code in line 4 of <a id="xref-boxed-text-1-2" class="xref-boxed-text" href="#boxed-text-1">Box 1</a>, FW fits a Finlay–Wilkinson regression with the two-step OLS method: <kbd>y</kbd> (numeric, <em>n</em>, NAs are allowed) is the response variable, <kbd>VAR</kbd> (character, <em>n</em>, NAs are not allowed) are the identifiers for the varieties which are treated as labels; <kbd>ENV</kbd> (character, <em>n</em>, NAs are not allowed) are the identifiers for the environments; method is used to describe what method to use: <kbd>“OLS”</kbd> for ordinary least squares. The default method (<kbd>“Gibbs”</kbd>) is the Bayesian regression; this can be invoked using the code in line 5 of <a id="xref-boxed-text-1-3" class="xref-boxed-text" href="#boxed-text-1">Box 1</a>. By default, a single chain of Gibbs sampler is run with a total of 5000 cycles and the samples from the first 3000 cycles are used for Burn-in, and samples from the remaining 2000 cycles for inference (the user is advised to run longer chains and to check convergence as well as the size of Monte Carlo errors). The FW function provides many additional arguments that can be used to specify the model (<em>e.g.</em>, providing covariance matrices for varieties and environments, user-defined values for hyperparameters) and the algorithm (number of chains, number of iterations, <em>etc</em>.); details can be found in the user manual and in the examples presented below.</p><p id="p-18">After fitting either OLS or Gibbs method, FW function returns a list with estimates and arguments used in the call; a brief description of the outputs follows.</p></div><div id="sec-7" class="subsection"><h3>Return</h3><p id="p-19"><a id="xref-boxed-text-2-1" class="xref-boxed-text" href="#boxed-text-2">Box 2</a> shows the structure of the object returned after calling the FW function with the default Gibbs method (see line 1 of <a id="xref-boxed-text-2-2" class="xref-boxed-text" href="#boxed-text-2">Box 2</a>). The first element <kbd>$y</kbd> of the list is the response vector used in the call to FW, <kbd>$whichNa</kbd> gives the position of the entries in y that were missing, <kbd>$mu</kbd> (vector), <kbd>$g</kbd> (matrix), <kbd>$b</kbd> (matrix), <kbd>$h</kbd> (matrix) are the estimated posterior means of <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-57.gif"/></noscript></span></span>, <strong>g</strong>, <strong>b</strong>, and <strong>h</strong>; <kbd>$yhat</kbd> (matrix) is the estimated posterior means of the predictor <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-58.gif"/></noscript></span></span>: <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-59.gif"/></noscript></span></span>; <kbd>$SD.mu</kbd> (vector), <kbd>$SD.g</kbd> (matrix), <kbd>$SD.b</kbd> (matrix), <kbd>$SD.h</kbd> (matrix), and <kbd>$SD.yhat</kbd> are the estimated posterior standard deviation for <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-60.gif"/></noscript></span></span>, <strong>g</strong>, <strong>b</strong>, <strong>h</strong>, and <strong><span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-61.gif"/></noscript></span></span></strong>, respectively.</p><p id="p-20">With the OLS method, <kbd>$g</kbd>, <kbd>$b</kbd>, <kbd>$h</kbd>, and <kbd>$yhat</kbd> all have only one column; with the Gibbs method each column provides estimates derived from one chain of Markov chain Monte Carlo (MCMC). Since the default behavior is to run only one chain the outputs in <a id="xref-boxed-text-2-3" class="xref-boxed-text" href="#boxed-text-2">Box 2</a> contain only one column; however, if multiple chains are run, estimates from different chains are provided in different columns.</p><div class="boxed-text" id="boxed-text-4"><h3><kbd>Box 2 Structure of the object returned by FW</kbd></h3><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/467768/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/467768/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div></div></div><p id="p-21">The outputs <kbd>$var_e</kbd>, <kbd>$var_g</kbd>, <kbd>$var_b</kbd>, and <kbd>$var_h</kbd> are the estimated posterior means for <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-62.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-63.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-64.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-65.gif"/></noscript></span></span> (only available for the Gibbs method). Each element of <kbd>$var_e</kbd>, <kbd>$var_g</kbd>, <kbd>$var_b</kbd>, and <kbd>$var_h</kbd> correspond to estimates derived from different chains; <kbd>$SD.var_e</kbd>, <kbd>$SD.var_g</kbd>, <kbd>$SD.var_b</kbd>, and <kbd>$SD.var_h</kbd> are the estimated posterior standard deviation for <span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-66.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-67.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-68.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-69.gif"/></noscript></span></span>, respectively.</p></div><div id="sec-8" class="subsection"><h3>Output files</h3><p id="p-22">No output files are generated for the OLS method. For the Gibbs method, samples for <span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-70.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-71.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-72.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-73.gif"/></noscript></span></span>, and (by default) the first two elements of <strong>g, b,</strong> and <strong>h</strong> will be saved; as the Gibbs sampler collects samples, these samples are saved to the hard drive (only the most recent samples are retained in the memory); by default, a thinning of 5 is used. Once the iteration process finishes, FW will read all the saved samples into a <kbd>mcmc</kbd> object, save the <kbd>mcmc</kbd> object into a file <kbd>samps.rda</kbd>, and remove the raw sample files. To prevent overloading the RAM with samples by default FW only save samples of the first two entries of the vectors of random effects; however the user can change this behavior by specifying which entries of the vectors are desired using the <kbd>saveVAR</kbd> (for <strong>g</strong> and <strong>b</strong>) and <kbd>saveENV</kbd> (for <strong>h</strong>) argument. These samples produced by FW can be used to assess convergence and to estimate Monte Carlo standard errors. The file <kbd>samps.rda</kbd> can be directly loaded into R using load(<kbd>’samps.rda’</kbd>). Once the object containing the samples is loaded in the R environment, the package coda (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Plummer <em>et al.</em>, 2006</a>) can be used to obtain plots of the chains and compute convergence diagnostics.</p></div></div><div class="section other3" id="sec-9"><h2 class="">Application examples</h2><p id="p-23">In this section we illustrate via examples some of the features of the FW package. Example 1 illustrates how the package can be used to fit Finlay–Wilkinson regression by the OLS method and Gibbs method with and without covariance structure and Example 2 describes how the package can be used for cross-validation analyses. Additional examples involving fine-tuning the Gibbs method (<em>e.g.</em>, hyperparameter setup, fitting more than two chains, specify saved samples) are provided in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.026328/-/DC1/FileS2.pdf">File S2</a>.</p><div id="sec-10" class="subsection"><h3>Example 1: fitting models with default setup for 599 wheat lines</h3><p id="p-24"><a id="xref-boxed-text-3-1" class="xref-boxed-text" href="#boxed-text-3">Box 3</a> shows the code used to fit a FW regression using three different approaches: (1) a two-step OLS method (code in line 3), (2) a Bayesian FW regression assuming independence of lines and of environments (code in lines 3–5), and (3) a Bayesian FW regression that incorporates genomic information (lines 6–8). In the Bayesian models, the seed for the random number generator can be specified using the argument seed (see lines 3–8) and the argument saveAt can be used to add a path and a prefix to be appended to <kbd>’samps.rda’</kbd> file.</p><div class="boxed-text" id="boxed-text-5"><h3><kbd>Box 3 Fit models by default parameters</kbd></h3><div id="T5" class="table pos-anchor"><div class="table-inline"><table frame="hsides" rules="groups" id="table-5"><tbody id="tbody-5"><tr id="tr-34"><td rowspan="1" colspan="1" id="td-63">1</td><td rowspan="1" colspan="1" id="td-64">library(FW); data(wheat); attach(wheat.Y)</td></tr><tr id="tr-35"><td rowspan="1" colspan="1" id="td-65">2</td><td rowspan="1" colspan="1" id="td-66">OLS = FW(y = y,VAR = VAR,ENV = ENV, method=”OLS”)</td></tr><tr id="tr-36"><td rowspan="1" colspan="1" id="td-67">3</td><td rowspan="1" colspan="1" id="td-68">GibbsI = FW(y = y,VAR = VAR,ENV = ENV,</td></tr><tr id="tr-37"><td rowspan="1" colspan="1" id="td-69">4</td><td rowspan="1" colspan="1" id="td-70">method=”Gibbs”,seed = 12345,saveAt=”GibbsI”,nIter = 50000</td></tr><tr id="tr-38"><td rowspan="1" colspan="1" id="td-71">5</td><td rowspan="1" colspan="1" id="td-72">,burnIn = 5000)</td></tr><tr id="tr-39"><td rowspan="1" colspan="1" id="td-73">6</td><td rowspan="1" colspan="1" id="td-74">GibbsA = FW(y = y,VAR = VAR,ENV = ENV,</td></tr><tr id="tr-40"><td rowspan="1" colspan="1" id="td-75">7</td><td rowspan="1" colspan="1" id="td-76">method=”Gibbs”,A = wheat.G,seed = 12345,</td></tr><tr id="tr-41"><td rowspan="1" colspan="1" id="td-77">8</td><td rowspan="1" colspan="1" id="td-78">saveAt=”GibbsA”,nIter = 50000,burnIn = 5000)</td></tr><tr id="tr-42"><td rowspan="1" colspan="1" id="td-79">9</td><td rowspan="1" colspan="1" id="td-80">load(“GibbsIsamps.rda”)</td></tr><tr id="tr-43"><td rowspan="1" colspan="1" id="td-81">10</td><td rowspan="1" colspan="1" id="td-82">HPDinterval(samps[,c(“var_e”,”var_g”,”var_b”,”var_h”)])</td></tr><tr id="tr-44"><td rowspan="1" colspan="1" id="td-83">11</td><td rowspan="1" colspan="1" id="td-84">load(“GibbsAsamps.rda”)</td></tr><tr id="tr-45"><td rowspan="1" colspan="1" id="td-85">12</td><td rowspan="1" colspan="1" id="td-86">HPDinterval(samps[,c(“var_e”,”var_g”,”var_b”,”var_h”)])</td></tr></tbody></table><div class="table-foot"></div></div></div></div><p id="p-25">Parameter estimates (estimated posterior means) can be directly extracted from the FW object as illustrated in <a id="xref-boxed-text-2-4" class="xref-boxed-text" href="#boxed-text-2">Box 2</a>. Other features of the posterior distribution (<em>e.g.</em>, 95% credibility intervals for the parameters) can be obtained by <em>post hoc</em> analyses of the samples included in the rda file generated by the program (see lines 9–12 of <a id="xref-boxed-text-3-2" class="xref-boxed-text" href="#boxed-text-3">Box 3</a>). In <a id="xref-table-wrap-6-1" class="xref-table" href="#T6">Table 1</a>, we listed the estimates of variance components from the three models. For the OLS method, only the residual variance <span class="inline-formula" id="inline-formula-70"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-74.gif"/></noscript></span></span> (weighted mean of residual variance for each within-line regression by its residual degree of freedom) is estimated. The estimated error variances are very similar across the three models. Also from <a id="xref-table-wrap-6-2" class="xref-table" href="#T6">Table 1</a>, we can see that the estimated variance of the main effects of the environments is large relative to both the error variance and the phenotypic variance.</p><div id="T6" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/467826/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/467826/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Estimated variance components (posterior 95% credibility intervals in parentheses) from different models</span><div class="sb-div caption-clear"></div></div></div><p id="p-26">The fitness of the models can be examined by the correlations between the observed values y and the fitted values <span class="inline-formula" id="inline-formula-75"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-79.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-79.gif"/></noscript></span></span> (line 1 of <a id="xref-boxed-text-4-1" class="xref-boxed-text" href="#boxed-text-4">Box 4</a>). The OLS model fitted the data better than both GibbsI and GibbsA: the correlation was 0.91 for the OLS method, 0.88 for GibbsI, and 0.86 for GibbsA.</p><div class="boxed-text" id="boxed-text-6"><h3><kbd>Box 4 Correlation between <em>y</em> and <span class="inline-formula" id="inline-formula-76"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-80.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-80.gif"/></noscript></span></span>, and correlations for <span class="inline-formula" id="inline-formula-77"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-81.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-81.gif"/></noscript></span></span> among different models</kbd></h3><div id="T7" class="table pos-anchor"><div class="table-inline"><table frame="hsides" rules="groups" id="table-7"><tbody id="tbody-7"><tr id="tr-51"><td rowspan="1" colspan="1" id="td-107"><kbd>1</kbd></td><td rowspan="1" colspan="1" id="td-108"><kbd>cor(y,OLS$yhat);cor(y,GibbsI$yhat);cor(y,GibbsA$yhat);</kbd></td></tr><tr id="tr-52"><td rowspan="1" colspan="1" id="td-109"><kbd>2</kbd></td><td rowspan="1" colspan="1" id="td-110"><kbd>cor(OLS$b,GibbsI$b);cor(OLS$b,GibbsA$b);cor(GibbsI$b,GibbsA$b)</kbd></td></tr></tbody></table><div class="table-foot"></div></div></div></div><p id="p-27">In <a id="xref-table-wrap-8-1" class="xref-table" href="#T8">Table 2</a>, we listed the correlations among parameter estimates from different models (code for <span class="inline-formula" id="inline-formula-78"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-82.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-82.gif"/></noscript></span></span> was provided in line 2 of <a id="xref-boxed-text-4-2" class="xref-boxed-text" href="#boxed-text-4">Box 4</a>), and noticed that correlations among parameter estimates from different models are high; this is expected considering that the data comes from a full factorial design where all lines are evaluated in all environments.</p><div id="T8" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/467840/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/467840/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Pearson’s product-moment correlation between parameter estimates derived by each of the three methods implemented in Box 3</span><div class="sb-div caption-clear"></div></div></div><p id="p-28">The pattern of variety performance in different environments can be visualized by plotting the observed and fitted values against the estimated environment effects. <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> was generated by the calling of plot function in line 2–3 of <a id="xref-boxed-text-5-1" class="xref-boxed-text" href="#boxed-text-5">Box 5</a>. Each line in this plot corresponds to a genotype. The comparison of the results from the OLS and GibbsA reveals interesting patterns: the OLS method predicts a much stronger extent of variability in intercepts and slopes (this is likely due to overfitting, see Example 2 below) than the Bayesian method. The Bayesian method yields ‘smoother’ predictions; this is a direct consequence of the shrinkage-toward-the-mean induced in the Bayesian method by treating effects as random and the use of correlations between genotypes (<em>e.g.</em>, genomic relationships).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/3/589/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Plot of variety performance vs. estimated environment values. Each line represents a different variety. Lines are fitted values and points are the cell means of genotype and environment combination. The horizontal axis displays the estimated environmental effects. The labels of these environments are also displayed; these labels can be removed by setting ENVlabel = F." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1968816158" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Plot of variety performance &lt;em&gt;vs.&lt;/em&gt; estimated environment values. Each line represents a different variety. Lines are fitted values and points are the cell means of genotype and environment combination. The horizontal axis displays the estimated environmental effects. The labels of these environments are also displayed; these labels can be removed by setting &lt;kbd&gt;ENVlabel = F&lt;/kbd&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/3/589/F1.medium.gif" width="440" height="181"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/6/3/589/F1.medium.gif" width="440" height="181"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/3/589/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/3/589/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467733" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-29" class="first-child">Plot of variety performance <em>vs.</em> estimated environment values. Each line represents a different variety. Lines are fitted values and points are the cell means of genotype and environment combination. The horizontal axis displays the estimated environmental effects. The labels of these environments are also displayed; these labels can be removed by setting <kbd>ENVlabel = F</kbd>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-30">The function plotVAR also allows users to display the curves for a few genotypes (see code in lines 5–9 of <a id="xref-boxed-text-5-2" class="xref-boxed-text" href="#boxed-text-5">Box 5</a>). Using this feature we display in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> the estimated regressions for five varieties. The slope in the plot corresponds to <span class="inline-formula" id="inline-formula-83"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-87.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-87.gif"/></noscript></span></span> and the dashed line corresponds to a slope equal to 1 (<span class="inline-formula" id="inline-formula-84"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-88.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-88.gif"/></noscript></span></span>); recall that <span class="inline-formula" id="inline-formula-85"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-89.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-89.gif"/></noscript></span></span> represents the expected change in performance of the i<sup>th</sup> variety per unit change in the environment effect. We observe from <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a> that line ID = 1081265 performs well in all environments and line ID = 13302 is better adapted to good environments.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/3/589/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Plot of the performance of five varieties on estimated environment values. Each color represents a different variety. Lines are fitted values and circles are the cell means of genotype by environment combination. The horizontal axis displays the estimated environmental effects. The labels of these environments are also displayed; these labels can be removed by setting ENVlabel = F." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1968816158" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Plot of the performance of five varieties on estimated environment values. Each color represents a different variety. Lines are fitted values and circles are the cell means of genotype by environment combination. The horizontal axis displays the estimated environmental effects. The labels of these environments are also displayed; these labels can be removed by setting &lt;kbd&gt;ENVlabel = F&lt;/kbd&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/3/589/F2.medium.gif" width="440" height="182"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/6/3/589/F2.medium.gif" width="440" height="182"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/3/589/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/3/589/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467738" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-31" class="first-child">Plot of the performance of five varieties on estimated environment values. Each color represents a different variety. Lines are fitted values and circles are the cell means of genotype by environment combination. The horizontal axis displays the estimated environmental effects. The labels of these environments are also displayed; these labels can be removed by setting <kbd>ENVlabel = F</kbd>.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>Fitting models with covariance between the environment effects</h3><p id="p-32">Covariance structures can be used to induce borrowing of information between levels of a random effect. For instance, pedigree-based or genomic-derived relationships can be used to induce borrowing of information between genotypes. Similarly a covariance structure between the environment effects could be used to induce borrowing of information between environments. Such covariance structures can be derived from previous knowledge about the correlation of the average performance of genotypes in pairs of environments or by using environmental covariates, as demonstrated in <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Jarquín <em>et al.</em> (2014</a>). The FW package allows incorporating covariance between the environment effects; an example of how this can be done is given in <a id="xref-boxed-text-6-1" class="xref-boxed-text" href="#boxed-text-6">Box 6</a>. In this example we compare three analyses. The first model (GibbsI) assumes that the environment effects are independent; this model was fitted previously using the code in <a id="xref-boxed-text-3-3" class="xref-boxed-text" href="#boxed-text-3">Box 3</a>. Subsequently, we modified this model by incorporating a covariance structure that assumes a covariance of 0.9 between environments 1 and 2, and null covariance among the other pairs of environments. This model was fitted using the entire data set (GibbsH) and after setting to NA all the records from the 2nd environment (GibbsH_NA).</p><div class="boxed-text" id="boxed-text-7"><h3><kbd>Box 5 Plot fitted models</kbd></h3><div id="T9" class="table pos-anchor"><div class="table-inline"><table frame="hsides" rules="groups" id="table-9"><tbody id="tbody-9"><tr id="tr-58"><td rowspan="1" colspan="1" id="td-127"><kbd>1</kbd></td><td rowspan="1" colspan="1" id="td-128"><kbd>par(mfrow = c(1,2))</kbd></td></tr><tr id="tr-59"><td rowspan="1" colspan="1" id="td-129"><kbd>2</kbd></td><td rowspan="1" colspan="1" id="td-130"><kbd>plot(OLS,main=”OLS”, cex = 0.2,lwd = 0.2)</kbd></td></tr><tr id="tr-60"><td rowspan="1" colspan="1" id="td-131"><kbd>3</kbd></td><td rowspan="1" colspan="1" id="td-132"><kbd>plot(GibbsA,main=”GibbsA”, cex = 0.2,lwd = 0.2) #cex controls point</kbd></td></tr><tr id="tr-61"><td rowspan="1" colspan="1" id="td-133"><kbd>4</kbd></td><td rowspan="1" colspan="1" id="td-134"><kbd>#size, lwd controls the line width</kbd></td></tr><tr id="tr-62"><td rowspan="1" colspan="1" id="td-135"><kbd>5</kbd></td><td rowspan="1" colspan="1" id="td-136"><kbd>plot(OLS, plotVAR = c(“1081265”,”1101307”,</kbd></td></tr><tr id="tr-63"><td rowspan="1" colspan="1" id="td-137"><kbd>6</kbd></td><td rowspan="1" colspan="1" id="td-138"><kbd>“1295736”, “13302”, “1343502”), main=”OLS”)</kbd></td></tr><tr id="tr-64"><td rowspan="1" colspan="1" id="td-139"><kbd>7</kbd></td><td rowspan="1" colspan="1" id="td-140"><kbd>plot(GibbsA, plotVAR = c(“1081265”,”1101307”,</kbd></td></tr><tr id="tr-65"><td rowspan="1" colspan="1" id="td-141"><kbd>8</kbd></td><td rowspan="1" colspan="1" id="td-142"><kbd>“1295736”, “13302”, “1343502”),</kbd></td></tr><tr id="tr-66"><td rowspan="1" colspan="1" id="td-143"><kbd>9</kbd></td><td rowspan="1" colspan="1" id="td-144"><kbd>main=”GibbsA”)</kbd></td></tr></tbody></table><div class="table-foot"></div></div></div></div><p id="p-33"><a id="xref-table-wrap-10-1" class="xref-table" href="#T10">Table 3</a> displays the estimated environment effects derived from each of the analyses. The estimated environment effects derived from GibbsI and GibbsH were almost identical. This happens because in these two examples the data available for each environment dominate over the prior distribution (which in case of GibssH assumes that the effects of environments 1 and 2 are highly correlated). However, when we set to NA all the entries of environment 2 (GibbsH_NA), the estimated effects for environments 1 and 2 are very close. This was entirely driven by the covariance structure H. An intermediate situation can emerge where one environment has records for a few genotypes. In such cases, nondiagonal covariance structures (H) may be used to borrow information between environments.</p><div id="T10" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/467834/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/467834/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Estimated environment effects from GibbsI and GibbsH</span><div class="sb-div caption-clear"></div></div></div><div class="boxed-text" id="boxed-text-8"><h3><kbd>Box 6 Including covariance matrix (H) for environments in FW</kbd></h3><div id="T11" class="table pos-anchor"><div class="table-inline"><table frame="hsides" rules="groups" id="table-11"><tbody id="tbody-11"><tr id="tr-72"><td rowspan="1" colspan="1" id="td-161"><kbd>1</kbd></td><td rowspan="1" colspan="1" id="td-162"><kbd>H = diag(1,4)</kbd></td></tr><tr id="tr-73"><td rowspan="1" colspan="1" id="td-163"><kbd>2</kbd></td><td rowspan="1" colspan="1" id="td-164"><kbd>H[1,2]=H[2,1]=0.9</kbd></td></tr><tr id="tr-74"><td rowspan="1" colspan="1" id="td-165"><kbd>3</kbd></td><td rowspan="1" colspan="1" id="td-166"><kbd>colnames(H)=rownames(H)=c(1,2,4,5)</kbd></td></tr><tr id="tr-75"><td rowspan="1" colspan="1" id="td-167"><kbd>4</kbd></td><td rowspan="1" colspan="1" id="td-168"><kbd>GibbsH = FW(y = y,VAR = VAR,ENV = ENV, method=“Gibbs”,H = H,seed =</kbd></td></tr><tr id="tr-76"><td rowspan="1" colspan="1" id="td-169"><kbd>5</kbd></td><td rowspan="1" colspan="1" id="td-170"><kbd> 12345,nIter = 50000,burnIn = 5000)</kbd></td></tr><tr id="tr-77"><td rowspan="1" colspan="1" id="td-171"><kbd>6</kbd></td><td rowspan="1" colspan="1" id="td-172"><kbd>yNA = y</kbd></td></tr><tr id="tr-78"><td rowspan="1" colspan="1" id="td-173"><kbd>7</kbd></td><td rowspan="1" colspan="1" id="td-174"><kbd>yNA[which(ENV==2)]=NA</kbd></td></tr><tr id="tr-79"><td rowspan="1" colspan="1" id="td-175"><kbd>8</kbd></td><td rowspan="1" colspan="1" id="td-176"><kbd>GibbsH_NA = FW(y = yNA,VAR = VAR,ENV = ENV, method=“Gibbs”,H = H,seed =</kbd></td></tr><tr id="tr-80"><td rowspan="1" colspan="1" id="td-177"><kbd>9</kbd></td><td rowspan="1" colspan="1" id="td-178"><kbd> 12345,nIter = 50000,burnIn = 5000)</kbd></td></tr><tr id="tr-81"><td rowspan="1" colspan="1" id="td-179"><kbd>10</kbd></td><td rowspan="1" colspan="1" id="td-180"><kbd>round(cbind(GibbsI$h,GibbsH$h,GibbsH_NA$h),2)</kbd></td></tr></tbody></table><div class="table-foot"></div></div></div></div><p id="p-34">Finally, the example provided by GibbsH_NA also illustrates how H allows predictions to be made about environments without records; if such environments are correlated with other environments for which we have data, in principle we can infer the effects for those environments. This of course will not be possible if H is diagonal.</p></div><div id="sec-12" class="subsection"><h3>Assessment of convergence for Bayesian FW regressions</h3><p id="p-35">The convergence of Gibbs sampler can be examined by plotting the samples collected by FW. The code in <a id="xref-boxed-text-7-1" class="xref-boxed-text" href="#boxed-text-7">Box 7</a> illustrates how to produce trace plots: lines 1–2 load and plot the samples from GibbsI and lines 3–4 do the same for GibbsA. Mixing was reasonably good (samples traverse through the sample space in relatively few steps, and can be verified by low average autocorrelation between samples: for example the average autocorrelation was 0.05 for <kbd>var_e</kbd> at lag 5, see line 5 of <a id="xref-boxed-text-7-2" class="xref-boxed-text" href="#boxed-text-7">Box 7</a>) in both cases for the variance components [<span class="inline-formula" id="inline-formula-86"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-90.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-90.gif"/></noscript></span></span> (<kbd>var_e</kbd>), <span class="inline-formula" id="inline-formula-87"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-91.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-91.gif"/></noscript></span></span> (<kbd>var_g</kbd>), <span class="inline-formula" id="inline-formula-88"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-92.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-92.gif"/></noscript></span></span> (<kbd>var_b</kbd>)], genotype main effects <strong>g</strong> (<kbd>g</kbd>), genotype slope <strong>b</strong> (<kbd>b</kbd>), and the function predictor <span class="inline-formula" id="inline-formula-89"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-93.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-93.gif"/></noscript></span></span>(<kbd>yhat</kbd>). There are many high peaks in the trace plot of <span class="inline-formula" id="inline-formula-90"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-94.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-94.gif"/></noscript></span></span> (<kbd>var_h</kbd>), which indicates that the distribution of <span class="inline-formula" id="inline-formula-91"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-95.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-95.gif"/></noscript></span></span> is skewed (this is also self-evident in the density plot). This should be expected since there are only four levels of environment effect and scaled-inverse chi-square distribution with few degrees of freedom is highly skewed. <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> reproduces the trace plot of the variance components (<kbd>var_e</kbd>, <kbd>var_g</kbd>, <kbd>var_b</kbd>, <kbd>var_h</kbd>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/3/589/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Trace and density plot of variance components from GibbsA." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1968816158" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Trace and density plot of variance components from GibbsA.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/3/589/F3.medium.gif" width="440" height="414"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/6/3/589/F3.medium.gif" width="440" height="414"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/3/589/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/3/589/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467764" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-36" class="first-child">Trace and density plot of variance components from GibbsA.</p><div class="sb-div caption-clear"></div></div></div><p id="p-37">The mixing for the intercept <span class="inline-formula" id="inline-formula-92"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-96.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-96.gif"/></noscript></span></span> and the environment effects (the entries of <strong>h</strong>) can be slow in multiplicative models (<em>e.g.</em>, <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Shariati <em>et al.</em>, 2009</a>). Therefore, the user is advised to check convergence to the posterior distribution and the magnitude of Monte Carlo standard errors. Convergence to the posterior distribution can be assessed graphically using a trace plot for single or more formally multiple chains. <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a> reproduces the trace plot for intercept <span class="inline-formula" id="inline-formula-93"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-97.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-97.gif"/></noscript></span></span> and the first two elements of environment effect, <kbd>h[1]</kbd> and <kbd>h[2]</kbd>; in all cases we used samples from model GibbsA. From <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>, we can see that even the mixing of <kbd>h[1]</kbd> and <kbd>h[2]</kbd> is slow; when running 50,000 iterations, the chain has converged to relative constant sample means. The Time-series standard error for the sample means of <kbd>h[1]</kbd> and <kbd>h[2]</kbd> are both around 0.0065, which is at a reasonable level (obtained by line 6 of <a id="xref-boxed-text-7-3" class="xref-boxed-text" href="#boxed-text-7">Box 7</a>). An example of how to assess convergence using multiple chains is provided in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.026328/-/DC1/FileS2.pdf">File S2</a>.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/3/589/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Trace plot of the intercept (mu) and the first two levels of environment effects (h[1]) and h[2]) from GibbsA." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1968816158" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Trace plot of the intercept (&lt;kbd&gt;mu&lt;/kbd&gt;) and the first two levels of environment effects (&lt;kbd&gt;h[1]&lt;/kbd&gt;) and &lt;kbd&gt;h[2]&lt;/kbd&gt;) from GibbsA.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/3/589/F4.medium.gif" width="143" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/6/3/589/F4.medium.gif" width="143" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/3/589/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/3/589/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467626" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-38" class="first-child">Trace plot of the intercept (<kbd>mu</kbd>) and the first two levels of environment effects (<kbd>h[1]</kbd>) and <kbd>h[2]</kbd>) from GibbsA.</p><div class="sb-div caption-clear"></div></div></div><div class="boxed-text" id="boxed-text-9"><h3><kbd>Box 7 Plot of Gibbs samples</kbd></h3><div id="T12" class="table pos-anchor"><div class="table-inline"><table frame="hsides" rules="groups" id="table-12"><tbody id="tbody-12"><tr id="tr-82"><td rowspan="1" colspan="1" id="td-181"><kbd>1</kbd></td><td rowspan="1" colspan="1" id="td-182"><kbd>load(“GibbsIsamps.rda”)</kbd></td></tr><tr id="tr-83"><td rowspan="1" colspan="1" id="td-183"><kbd>2</kbd></td><td rowspan="1" colspan="1" id="td-184"><kbd>plot(samps,ask = T)</kbd></td></tr><tr id="tr-84"><td rowspan="1" colspan="1" id="td-185"><kbd>3</kbd></td><td rowspan="1" colspan="1" id="td-186"><kbd>load(“GibbsAsamps.rda”)</kbd></td></tr><tr id="tr-85"><td rowspan="1" colspan="1" id="td-187"><kbd>4</kbd></td><td rowspan="1" colspan="1" id="td-188"><kbd>plot(samps,ask = T)</kbd></td></tr><tr id="tr-86"><td rowspan="1" colspan="1" id="td-189"><kbd>5</kbd></td><td rowspan="1" colspan="1" id="td-190"><kbd>autocorr(samps[[1]][,”var_e”])</kbd></td></tr><tr id="tr-87"><td rowspan="1" colspan="1" id="td-191"><kbd>6</kbd></td><td rowspan="1" colspan="1" id="td-192"><kbd>summary(samps)</kbd></td></tr></tbody></table><div class="table-foot"></div></div></div></div></div><div id="sec-13" class="subsection"><h3>Example 2: assessment of prediction accuracy in testing data sets</h3><p id="p-39">Example 1 suggests that the OLS method fitted the training data better than the Bayesian models; this is expected because shrinkage reduces fitness to the data used to train a model. However, better model fitness does not necessarily imply higher prediction accuracy in validation data sets. In the following example we illustrate how to use the FW for assessment of prediction accuracy using cross-validation.</p><p id="p-40">To assess the ability of different models for predicting new data, we modified the code in <a id="xref-boxed-text-3-4" class="xref-boxed-text" href="#boxed-text-3">Box 3</a> by setting NA to randomly selected entries of the phenotypic vector (<em>i.e.</em>, one record out of four per line was randomly selected and labeled as NA; see code in lines 1–5 in <a id="xref-boxed-text-8-1" class="xref-boxed-text" href="#boxed-text-8">Box 8</a>). The FW package produces estimates and predictions for all the lines, environments, and entries of the phenotypic vector, including those that had observed values and those that had NA. Therefore, predictions for entries with masked phenotypes can be used to assess prediction accuracy in validation data sets (see lines 14–16 in <a id="xref-boxed-text-8-2" class="xref-boxed-text" href="#boxed-text-8">Box 8</a>). We repeated the code in <a id="xref-boxed-text-8-3" class="xref-boxed-text" href="#boxed-text-8">Box 8</a> 100 times and generated 100 random partitions of the data into training and testing sets. Each partition renders an estimate of prediction accuracy for each of the models.</p><div class="boxed-text" id="boxed-text-10"><h3><kbd>Box 8 Correlation between y and <span class="inline-formula" id="inline-formula-94"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-98.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/589/embed/mml-math-98.gif"/></noscript></span></span> for training and validation data sets</kbd></h3><div id="T13" class="table pos-anchor"><div class="table-inline"><table frame="hsides" rules="groups" id="table-13"><tbody id="tbody-13"><tr id="tr-88"><td rowspan="1" colspan="1" id="td-193"><kbd>1</kbd></td><td rowspan="1" colspan="1" id="td-194"><kbd>yNA = y</kbd></td></tr><tr id="tr-89"><td rowspan="1" colspan="1" id="td-195"><kbd>2</kbd></td><td rowspan="1" colspan="1" id="td-196"><kbd>seed = 12345; set.seed(seed)</kbd></td></tr><tr id="tr-90"><td rowspan="1" colspan="1" id="td-197"><kbd>3</kbd></td><td rowspan="1" colspan="1" id="td-198"><kbd>#randomly masking one environment for each variety</kbd></td></tr><tr id="tr-91"><td rowspan="1" colspan="1" id="td-199"><kbd>4</kbd></td><td rowspan="1" colspan="1" id="td-200"><kbd>whichNa = seq(from = 0,to = 2392,by = 4)+sample(1:4,size = 599,replace = T)</kbd></td></tr><tr id="tr-92"><td rowspan="1" colspan="1" id="td-201"><kbd>5</kbd></td><td rowspan="1" colspan="1" id="td-202"><kbd>yNA[whichNa]=NA</kbd></td></tr><tr id="tr-93"><td rowspan="1" colspan="1" id="td-203"><kbd>6</kbd></td><td rowspan="1" colspan="1" id="td-204"><kbd>OLS = FW(y = yNA,VAR = VAR,ENV = ENV, method=”OLS”)</kbd></td></tr><tr id="tr-94"><td rowspan="1" colspan="1" id="td-205"><kbd>7</kbd></td><td rowspan="1" colspan="1" id="td-206"><kbd>GibbsI = FW(y = yNA,VAR = VAR,ENV = ENV,</kbd></td></tr><tr id="tr-95"><td rowspan="1" colspan="1" id="td-207"><kbd>8</kbd></td><td rowspan="1" colspan="1" id="td-208"><kbd>method=”Gibbs”,seed = seed,nIter = 50000, burnIn = 5000)</kbd></td></tr><tr id="tr-96"><td rowspan="1" colspan="1" id="td-209"><kbd>9</kbd></td><td rowspan="1" colspan="1" id="td-210"><kbd>GibbsA = FW(y = yNA,VAR = VAR,ENV = ENV,</kbd></td></tr><tr id="tr-97"><td rowspan="1" colspan="1" id="td-211"><kbd>10</kbd></td><td rowspan="1" colspan="1" id="td-212"><kbd>method=”Gibbs”,A = wheat.G,seed = seed,nIter = 50000,burnIn = 5000)</kbd></td></tr><tr id="tr-98"><td rowspan="1" colspan="1" id="td-213"><kbd>11</kbd></td><td rowspan="1" colspan="1" id="td-214"><kbd>cor(y[-whichNa],OLS$yhat[-whichNa,])</kbd></td></tr><tr id="tr-99"><td rowspan="1" colspan="1" id="td-215"><kbd>12</kbd></td><td rowspan="1" colspan="1" id="td-216"><kbd>cor(y[-whichNa],GibbsI$yhat[-whichNa,])</kbd></td></tr><tr id="tr-100"><td rowspan="1" colspan="1" id="td-217"><kbd>13</kbd></td><td rowspan="1" colspan="1" id="td-218"><kbd>cor(y[-whichNa],GibbsA$yhat[-whichNa,])</kbd></td></tr><tr id="tr-101"><td rowspan="1" colspan="1" id="td-219"><kbd>14</kbd></td><td rowspan="1" colspan="1" id="td-220"><kbd>cor(y[whichNa],OLS$yhat[whichNa,])</kbd></td></tr><tr id="tr-102"><td rowspan="1" colspan="1" id="td-221"><kbd>15</kbd></td><td rowspan="1" colspan="1" id="td-222"><kbd>cor(y[whichNa],GibbsI$yhat[whichNa,])</kbd></td></tr><tr id="tr-103"><td rowspan="1" colspan="1" id="td-223"><kbd>16</kbd></td><td rowspan="1" colspan="1" id="td-224"><kbd>cor(y[whichNa],GibbsA$yhat[whichNa,])</kbd></td></tr></tbody></table><div class="table-foot"></div></div></div></div><p id="p-41">The mean correlation (of the 100 replicates) between phenotypes and predictions in the training data set (<em>i.e.</em>, for the entries of y that did not have missing values) follows the same patterns as in Example 1, where OLS fitted the data best: 0.95 for OLS, 0.89 for GibbsI, and 0.86 for GibbsA. However, the mean prediction correlation (of the 100 replicates) for the entries of the validation set has reversed orders: 0.61 for OLS, 0.77 for GibbsI, and 0.80 for GibbsA.</p><p id="p-42">In <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>, we plotted the estimated prediction correlation between predictions and observations in training (1st row of plots) and testing (2nd row of plots) data sets. Plots in the 1st, 2nd, and 3rd column correspond to comparisons of: OLS <em>vs.</em> GibbsI, OLS <em>vs.</em> GibbsA, and GibbsI <em>vs.</em> GibbsA, respectively. Within each plot each point represents the accuracy obtained in a partition for the models represented in the vertical and horizontal axis. Points above (below) the 45° line indicate higher (lower) accuracy of the model in the vertical axis, relative to the one in the horizontal axis. We observed that OLS always fitted the data better than GibbsI and GibbsA in the training data sets; however, GibbsI and GibbsA always outperformed OLS by a sizable margin in terms of prediction accuracy in testing data sets. Finally, incorporating genetic information (GibbsA) always led to higher prediction accuracy than models that assumed independence between lines (GibbsI).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/3/589/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Prediction accuracy for training and validation sets for the three methods implemented in Box 8." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1968816158" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Prediction accuracy for training and validation sets for the three methods implemented in Box 8.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/3/589/F5.medium.gif" width="440" height="236"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/6/3/589/F5.medium.gif" width="440" height="236"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/3/589/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/3/589/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467660" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-43" class="first-child">Prediction accuracy for training and validation sets for the three methods implemented in Box 8.</p><div class="sb-div caption-clear"></div></div></div><p id="p-44">We also noted in <a id="xref-table-wrap-14-1" class="xref-table" href="#T14">Table 4</a> that the correlations (here we reported results only for the first replicate) for the parameter estimates among different models reduced compared to Example 1 due to the missing values. For example, the correlation for the estimated <strong>b</strong> among different models reduced to 0.85 between OLS and GibbsI, 0.64 between OLS and GibbsA, and 0.79 between GibbsI and GibbsA.</p><div id="T14" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/467758/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/467758/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Pearson’s product-moment correlation between parameter estimates derived by each of the three methods implemented in <a id="xref-boxed-text-8-4" class="xref-boxed-text" href="#boxed-text-8">Box 8</a> (results from the first replicate only)</span><div class="sb-div caption-clear"></div></div></div></div></div><div class="section other4" id="sec-14"><h2 class="">Computation Time for 599 Wheat Lines</h2><p id="p-45">We ran the FW function in an Intel Core i7 1867 MHz Processor (R was executed in a single thread) with 16 GB of RAM memories. We recorded the memory and time usage for Gibbs methods with 50000 iterations. With the full data set (599 varieties, 2396 observations) the process used approximately 50 M of RAM memory for GibbsA, 17 M of RAM for GibbsI, and 153 M for OLS. The time needed to finish the process was: 11 min for GibbsA, 3 min for GibbsI, and 2 sec for OLS.</p></div><div class="section other5" id="sec-15"><h2 class="">Concluding Remarks</h2><p id="p-46">The FW package allows fitting Finlay–Wilkinson regression with ordinary least square method and Bayesian method. For Bayesian method, covariance matrix among varieties and environments can be included in the model. The interface allows the user to fit the models (<em>e.g.</em>, OLS <em>vs.</em> Gibbs) and visualize the results easily. The algorithms for Gibbs sampler are implemented in C and the speed is high. The package also provided flexibility for changing the hyper-parameters and model output.</p><p id="p-47">For incomplete/unbalanced experimental design the Bayesian approach is expected to have better statistical performance and prediction accuracy than the traditional two-step OLS method. Furthermore, the Bayesian models implemented in FW allows incorporating pedigree and marker information as well as modeling environment covariance. A cross-validation study based on real wheat data confirmed those expectations; indeed, the Bayesian method incorporating relationships between lines had a prediction accuracy that was 30% greater than the two-step OLS method.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-48">We thank the collaborators in national agricultural research institutes who carried out the Elite Spring Wheat Yield Trials (ESWYT) and provided the phenotypic data analyzed in this article. G.d.l.C. and L.L. received financial support from NIH grants R01GM101219 and R01GM099992 and from Arvalis. G.d.l.C. received financial support from Arvalis and CIMMYT.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-supplementary-material" id="fn-1"><p id="p-49">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.026328/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.026328/-/DC1</a></p></li><li class="fn" id="fn-2"><p id="p-50"><em>Communicating editor: D. J. de Koning</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2015-10-29"><span class="received-label">Received </span>October 29, 2015.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2015-12-23"><span class="accepted-label">Accepted </span>December 23, 2015.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2016 Lian and Campos</li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.589.1" data-doi="10.2307/2685208"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Casella</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">George</span>  <span class="cit-name-given-names">E. I.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Explaining the Gibbs sampler.</span> <abbr class="cit-jnl-abbrev">Am. Stat.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">167</span>–<span class="cit-lpage">174</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BStat.%26rft.volume%253D46%26rft.spage%253D167%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2685208%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2685208&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1992JE81900001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.589.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Copas</span>  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-article-title">Regression, prediction and shrinkage.</span> <abbr class="cit-jnl-abbrev">J. R. Stat. Soc., B</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">311</span>–<span class="cit-lpage">354</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BR.%2BStat.%2BSoc.%252C%2BB%26rft.volume%253D45%26rft.spage%253D311%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.589.3" data-doi="10.1534/genetics.110.118521"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgueño</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Prediction of genetic values of quantitative traits in plant breeding using pedigree and molecular markers.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">186</span>: <span class="cit-fpage">713</span>–<span class="cit-lpage">724</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.118521%26rft_id%253Dinfo%253Apmid%252F20813882%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg2LzIvNzEzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8zLzU4OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.589.4" data-doi="10.1071/AR9630742"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Finlay</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilkinson</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">1963</span> <span class="cit-article-title">The analysis of adaptation in a plant-breeding programme.</span> <abbr class="cit-jnl-abbrev">Crop Pasture Sci.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">742</span>–<span class="cit-lpage">754</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BPasture%2BSci.%26rft.volume%253D14%26rft.spage%253D742%26rft_id%253Dinfo%253Adoi%252F10.1071%252FAR9630742%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1071/AR9630742&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.589.5" data-doi="10.2307/1269656"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Frank</span>  <span class="cit-name-given-names">L. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Friedman</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">A statistical view of some chemometrics regression tools.</span> <abbr class="cit-jnl-abbrev">Technometrics</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">109</span>–<span class="cit-lpage">135</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTechnometrics%26rft.volume%253D35%26rft.spage%253D109%26rft_id%253Dinfo%253Adoi%252F10.2307%252F1269656%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/1269656&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1993KZ16900001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.589.6" data-doi="10.1109/TPAMI.1984.4767596"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Geman</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Geman</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title">Stochastic relaxation, Gibbs distributions, and the bayesian restoration of images.</span> <abbr class="cit-jnl-abbrev">IEEE Trans. Pattern Anal. Mach. Intell.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">721</span>–<span class="cit-lpage">741</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DIEEE%2BTrans.%2BPattern%2BAnal.%2BMach.%2BIntell.%26rft.volume%253D6%26rft.spage%253D721%26rft_id%253Dinfo%253Adoi%252F10.1109%252FTPAMI.1984.4767596%26rft_id%253Dinfo%253Apmid%252F22499653%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1109/TPAMI.1984.4767596&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22499653&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F589.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1984TX36100006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.589.7" data-doi="10.1007/BF00288581"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gregorius</span>  <span class="cit-name-given-names">H.-R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Namkoong</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span> <span class="cit-article-title">Joint analysis of genotypic and environmental effects.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">72</span>: <span class="cit-fpage">413</span>–<span class="cit-lpage">422</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D72%26rft.spage%253D413%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00288581%26rft_id%253Dinfo%253Apmid%252F24247951%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00288581&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24247951&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F589.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1986C864600019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.589.8" data-doi="10.1007/s00122-013-2243-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jarquín</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lacaze</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Du Cheyron</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daucourt</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A reaction norm model for genomic selection using high-dimensional genomic and environmental data.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">127</span>: <span class="cit-fpage">595</span>–<span class="cit-lpage">607</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D127%26rft.spage%253D595%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-013-2243-1%26rft_id%253Dinfo%253Apmid%252F24337101%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-013-2243-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24337101&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F589.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.589.9" data-doi="10.3835/plantgenome2010.04.0005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pérez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genomic-enabled prediction based on molecular markers and pedigree using the bayesian linear regression package in R.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">106</span>–<span class="cit-lpage">116</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D3%26rft.spage%253D106%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2010.04.0005%26rft_id%253Dinfo%253Apmid%252F21566722%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2010.04.0005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21566722&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F589.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.589.10" data-doi="10.1038/hdy.1968.48"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Perkins</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jinks</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1968</span> <span class="cit-article-title">Environmental and genotype-environmental components of variability III. Multiple lines and crosses.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">339</span>–<span class="cit-lpage">356</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DPerkins%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D23%26rft.issue%253D3%26rft.spage%253D339%26rft.epage%253D356%26rft.atitle%253DEnvironmental%2Band%2Bgenotype-environmental%2Bcomponents%2Bof%2Bvariability.%2B3.%2BMultiple%2Blines%2Band%2Bcrosses.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.1968.48%26rft_id%253Dinfo%253Apmid%252F5250121%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.1968.48&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=5250121&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F589.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1968B818400002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.589.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Plummer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Best</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cowles</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vines</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">CODA: convergence diagnosis and output analysis for MCMC.</span> <abbr class="cit-jnl-abbrev">R News</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">7</span>–<span class="cit-lpage">11</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DR%2BNews%26rft.volume%253D6%26rft.spage%253D7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-book" id="cit-6.3.589.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">R Development Core Team</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-source">R: A Language and Environment for Statistical Computing</span>. <span class="cit-publ-name">R Foundation for Statistical Computing</span>, <span class="cit-publ-loc">Vienna, Austria</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.589.13" data-doi="10.1111/j.1439-0388.2008.00773.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shariati</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Korsgaard</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorensen</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Identifiability of parameters and behaviour of mcmc chains: a case study using the reaction norm model.</span> <abbr class="cit-jnl-abbrev">J. Anim. Breed. Genet.</abbr> <span class="cit-vol">126</span>: <span class="cit-fpage">92</span>–<span class="cit-lpage">102</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BAnim%2BBreed%2BGenet%26rft.aulast%253DShariati%26rft.auinit1%253DM.%2BM.%26rft.volume%253D126%26rft.issue%253D2%26rft.spage%253D92%26rft.epage%253D102%26rft.atitle%253DIdentifiability%2Bof%2Bparameters%2Band%2Bbehaviour%2Bof%2BMCMC%2Bchains%253A%2Ba%2Bcase%2Bstudy%2Busing%2Bthe%2Breaction%2Bnorm%2Bmodel.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1439-0388.2008.00773.x%26rft_id%253Dinfo%253Apmid%252F19320765%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1439-0388.2008.00773.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19320765&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F589.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.589.14" data-doi="10.2527/jas.2005-517"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Su</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Madsen</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lund</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorensen</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Korsgaard</span>  <span class="cit-name-given-names">I. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Bayesian analysis of the linear reaction norm model with unknown covariates.</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">84</span>: <span class="cit-fpage">1651</span>–<span class="cit-lpage">1657</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BAnimal%2BScience%26rft.stitle%253DJ%2BANIM%2BSCI%26rft.aulast%253DSu%26rft.auinit1%253DG.%26rft.volume%253D84%26rft.issue%253D7%26rft.spage%253D1651%26rft.epage%253D1657%26rft.atitle%253DBayesian%2Banalysis%2Bof%2Bthe%2Blinear%2Breaction%2Bnorm%2Bmodel%2Bwith%2Bunknown%2Bcovariates%26rft_id%253Dinfo%253Adoi%252F10.2527%252Fjas.2005-517%26rft_id%253Dinfo%253Apmid%252F16775048%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2527/jas.2005-517&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16775048&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F589.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-web" id="cit-6.3.589.15"><div class="cit-metadata unstructured">Walsh, J. B., and M. Lynch, 2014 Selection and G × E: advanced topics, Chapter 44 in Evolution and Selection of Quantitative Traits: II. Advanced Topics in Breeding and Evolution. Available at: <a href="http://nitro.biosci.arizona.edu/zbook/NewVolume_2/pdf/Chapter44.pdf">http://nitro.biosci.arizona.edu/zbook/NewVolume_2/pdf/Chapter44.pdf</a>. Accessed December 5, 2015.</div><div class="cit-extra"></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/6/3/589.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/3/579" title="Genetic Architecture of Conspicuous Red Ornaments in Female Threespine Stickleback" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/3/599" title="Genetic Correlates of Individual Differences in Sleep Behavior of Free-Living Great Tits (Parus major)" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/6/3"><span>Volume 6 Issue 3, March 2016</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/6/3.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/6/3.cover-source.jpg?itok=8EDC4x5t" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 6 (3)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/3/589" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-o0wjTATrhfxrhdbq4PLvau-vsXCEWE5-47ZxVe9rstQ" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genomic-selection" class="" data-icon-position="" data-hide-link-title="0">Genomic Selection</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/6/3/589" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/6/3/589" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/6/3/589" class="active" data-icon-position="" data-hide-link-title="0">FW: An R Package for Finlay–Wilkinson Regression that Incorporates Genomic/Pedigree Information and Covariance Structures Between Environments</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460084" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-AMw99bD5_VY2Tq0sTrtdSyKlc_g4KnMkByVcd9OCiG4" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/3/589.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/6/3/589" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-BpaFm10OAxv0bNhbLy3IfiyOx0HEhWOD5Zx07A2KHP4" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/6/3/589" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/3/589.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460084" id="node460084--2" data-pisa="ggg;6/3/589" data-pisa-master="ggg;g3.115.026328" data-apath="/ggg/6/3/589.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">FW: An R Package for Finlay–Wilkinson Regression that Incorporates Genomic/Pedigree Information and Covariance Structures Between Environments</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Lian Lian</span> and <span class="highwire-citation-author" data-delta="1">Gustavo de los Campos</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">589-597;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.026328 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460084--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Lian Lian </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Epidemiology and Biostatistics, Michigan State University, East Lansing, Michigan 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lian%2BLian%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lian%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALian%2BLian%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:lianl0501@gmail.com" class="" data-icon-position="" data-hide-link-title="0">lianl0501@gmail.com</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Gustavo de los Campos </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Epidemiology and Biostatistics, Michigan State University, East Lansing, Michigan 48824</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Probability and Statistics, Michigan State University, East Lansing, Michigan 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gustavo%2Bde%2Blos%2BCampos%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=de%20los%20Campos%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGustavo%2Bde%2Blos%2BCampos%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/6/3/589" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/6/3/589&amp;title=FW%3A%20An%20R%20Package%20for%20Finlay%E2%80%93Wilkinson%20Regression%20that%20Incorporates%20Genomic/Pedigree%20Information%20and%20Covariance%20Structures%20Between%20Environments" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/6/3/589&amp;title=FW%3A%20An%20R%20Package%20for%20Finlay%E2%80%93Wilkinson%20Regression%20that%20Incorporates%20Genomic/Pedigree%20Information%20and%20Covariance%20Structures%20Between%20Environments" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/6/3/589&amp;title=FW%3A%20An%20R%20Package%20for%20Finlay%E2%80%93Wilkinson%20Regression%20that%20Incorporates%20Genomic/Pedigree%20Information%20and%20Covariance%20Structures%20Between%20Environments" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/6/3/589&amp;text=FW%3A%20An%20R%20Package%20for%20Finlay%E2%80%93Wilkinson%20Regression%20that%20Incorporates%20Genomic/Pedigree%20Information%20and%20Covariance%20Structures%20Between%20Environments" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/6/3/589&amp;title=FW%3A%20An%20R%20Package%20for%20Finlay%E2%80%93Wilkinson%20Regression%20that%20Incorporates%20Genomic/Pedigree%20Information%20and%20Covariance%20Structures%20Between%20Environments" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/6/3/589&amp;t=FW%3A%20An%20R%20Package%20for%20Finlay%E2%80%93Wilkinson%20Regression%20that%20Incorporates%20Genomic/Pedigree%20Information%20and%20Covariance%20Structures%20Between%20Environments" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/6/3/589&amp;title=FW%3A%20An%20R%20Package%20for%20Finlay%E2%80%93Wilkinson%20Regression%20that%20Incorporates%20Genomic/Pedigree%20Information%20and%20Covariance%20Structures%20Between%20Environments" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/6/3/589&amp;title=FW%3A%20An%20R%20Package%20for%20Finlay%E2%80%93Wilkinson%20Regression%20that%20Incorporates%20Genomic/Pedigree%20Information%20and%20Covariance%20Structures%20Between%20Environments" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460084/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460084" id="node460084--3" data-pisa="ggg;6/3/589" data-pisa-master="ggg;g3.115.026328" data-apath="/ggg/6/3/589.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">FW: An R Package for Finlay–Wilkinson Regression that Incorporates Genomic/Pedigree Information and Covariance Structures Between Environments</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Lian Lian</span> and <span class="highwire-citation-author" data-delta="1">Gustavo de los Campos</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">589-597;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.026328 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460084--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Lian Lian </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Epidemiology and Biostatistics, Michigan State University, East Lansing, Michigan 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lian%2BLian%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lian%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALian%2BLian%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:lianl0501@gmail.com" class="" data-icon-position="" data-hide-link-title="0">lianl0501@gmail.com</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Gustavo de los Campos </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Epidemiology and Biostatistics, Michigan State University, East Lansing, Michigan 48824</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Probability and Statistics, Michigan State University, East Lansing, Michigan 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gustavo%2Bde%2Blos%2BCampos%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=de%20los%20Campos%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGustavo%2Bde%2Blos%2BCampos%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=589&amp;rft.epage=597&amp;rft.atitle=FW%3A%20An%20R%20Package%20for%20Finlay%E2%80%93Wilkinson%20Regression%20that%20Incorporates%20Genomic/Pedigree%20Information%20and%20Covariance%20Structures%20Between%20Environments&amp;rft.volume=6&amp;rft.issue=3&amp;rft.date=2016-03-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Lian%2C+Lian&amp;rft.au=de+los+Campos%2C+Gustavo"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460084/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460084/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460084/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460084/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460084/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460084/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460084/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460084/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460084/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460084/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460084/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460084/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490720" id="node490720" data-pisa="ggg;8/9/3019" data-pisa-master="ggg;g3.118.200430" data-apath="/ggg/8/9/3019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">A Bayesian Decision Theory Approach for Genomic Selection</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490724" id="node490724" data-pisa="ggg;8/9/2889" data-pisa-master="ggg;g3.118.200311" data-apath="/ggg/8/9/2889.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/2889" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Optimising Genomic Selection in Wheat: Effect of Marker Density, Population Size and Population Structure on Prediction Accuracy</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490727" id="node490727" data-pisa="ggg;8/9/3039" data-pisa-master="ggg;g3.118.200435" data-apath="/ggg/8/9/3039.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3039" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">BGGE: A New Package for Genomic-Enabled Prediction Incorporating Genotype × Environment Interaction Models</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genomic%20Selection" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genomic Selection</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/6/3/589" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Model Specification and Algorithm</a></li><li class="odd"><a href="#sec-4" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Software</a></li><li class="even"><a href="#sec-9" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Application examples</a></li><li class="odd"><a href="#sec-14" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Computation Time for 599 Wheat Lines</a></li><li class="even"><a href="#sec-15" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Concluding Remarks</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/6/3/589.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/6/3/589.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/6/3/589.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/6/3/589.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
